![li01857775e0](li01857775e0.jpg)

Two cells in mortal combat: a malaria parasite invading a human red blood cell. (Courtesy Dr. Stuart Ralph, University of Melbourne.)

Nothing in biology makes sense except in the light of evolution.
-Theodosius Dobzhansky, 1973, essay in American Biology Teacher 35:125-129

Biology is a science fundamentally different from physics or chemistry, which deal with unchanging properties of matter that can be described by mathematical equations. Biological systems, of course, follow the rules of chemistry and physics, but biology is a historical science, as the forms and structures of the living world today are the results of billions of years of evolution. Through evolution, all organisms are related in a family tree extending from primitive single-celled organisms that lived in the distant past to the diverse plants, animals, and microorganisms of the present era (Figure 1-1, Table 1-1). The great insight of Charles Darwin (Figure 1-2) was the principle of natural selection: organisms vary randomly and compete within their environment for resources. Only those that survive and reproduce are able to pass down their genetic traits.

## 1

## Molecular, Cells, and Model Organisms

At first glance, the biological universe does appear amazingly diverse-from tiny ferns to tall fir trees, from single-celled bacteria and protozoans visible only under a microscope to multicellular animals of all kinds. Indeed, cells come in an astonishing variety of sizes and shapes (Figure 1-3). Some move rapidly and have fast-changing structures, as we can see in movies of amoebae and rotifers. Others are largely stationary and structurally stable. Oxygen kills some cells but is an absolute requirement for others. Most cells in multicellular organisms are intimately involved with other cells. Although some unicellular organisms live in isolation (Figure 1-3a), others form colonies or live in close association with other types of organisms (Figure 1-3b, d), such as the bacteria that help plants to extract nitrogen from the air or the bacteria that live in our intestines and help us digest food.

Yet the bewildering array of outward biological forms overlies a powerful uniformity: thanks to our common ancestry, all biological systems are composed of cells containing the same types of chemical molecules and employing similar principles of organization at the cellular level. Although the

### 1.1 The Molecules of Life

1.2 Prokaryotic Cell Structure and Function
1.3 Eukaryotic Cell Structure and Function

### 1.4 Unicellular Eukaryotic Model Organisms

1.5 Metazoan Structure, Differentiation, and Model Organisms

![li02857775e0](li02857775e0.jpg)

FIGURE 1-1 All living organisms descended from a common ancestral cell. All organisms, from simple bacteria to complex mammals, probably evolved from a common single-celled ancestor. This family tree depicts the evolutionary relationships among the three major lineages of organisms. The structure of the tree was initially ascertained from morphological criteria: creatures that look alike were put close together. More recently, the sequences of DNA and proteins
found in organisms have provided more information-rich criteria for assigning relationships. The greater the similarities in these macromolecular sequences, the more closely related organisms are thought to be. The trees based on morphological comparisons and the fossil record generally agree well with those based on molecular data. [Data from J. R. Brown, 2005, "Universal tree of life," in Encyclopedia of Life Sciences, Wiley InterScience (online).]
basic kinds of biological molecules have been conserved during the billions of years of evolution, the patterns in which they are assembled to form functioning cells and organisms have undergone considerable change.

We now know that genes, which chemically are composed of deoxyribonucleic acid (DNA), ultimately define biological structure and maintain the integration of cellular function. Many genes encode proteins, the primary molecules that make up cell structures and carry out cellular activities. Alterations in the structure and organization of genes, or mutations, provide the random variation that can alter biological structure and function. While the vast majority of random mutations have no observable effect on a gene's or protein's function, many are deleterious, and only a few confer an evolutionary advantage on the organism. In all organisms, mutations in DNA are constantly occurring, allowing over time the small alterations in cellular structures and functions that may prove to be advantageous. Entirely new cellular structures are rarely created; more often, existing cellular structures undergo changes that better adapt the organism to new circumstances. Slight changes in a protein can cause important changes in its function or abolish its function entirely.

For instance, in a particular organism, one gene may randomly become duplicated, after which one copy of the
gene and its encoded protein retain their original function while, over time, the second copy of the gene mutates such that its protein takes on a slightly different or even a totally new function. During the evolution of some organisms, the entire genome became duplicated, allowing the second copies of many genes to undergo mutations and acquire new functions. The cellular organization of organisms plays a fundamental role in this process because it allows these changes to come about by small alterations in previously evolved cells, giving them new abilities. The result is that closely related organisms have very similar genes and proteins as well as similar cellular and tissue organizations.

Multicellular organisms, including the human body, consist of such closely interrelated elements that no single element can be fully appreciated in isolation from the others. Organisms contain organs, organs are composed of tissues, tissues consist of cells, and cells are formed from molecules (Figure 1-4). The unity of living systems is coordinated by many levels of interrelationship: molecules carry messages from organ to organ and cell to cell, and tissues are delineated and integrated with other tissues by molecules secreted by cells. Generally all the levels into which we fragment biological systems interconnect.

TABLE 1-1 Timeline for Evolution of Life on Earth, as Determined from the Fossil Record

| 4600 million years ago | The planet Earth forms from material revolving around the young Sun. |
| :--: | :--: |
| $\sim 3900-2500$ million years ago | Cells resembling prokaryotes appear. These first organisms are chemoautotrophs: they use carbon dioxide as a carbon source and oxidize inorganic materials to extract energy. |
| 3500 million years ago | Lifetime of the last universal ancestor; the split between Eubacteria and Archaea occurs. |
| 3000 million years ago | Photosynthesizing cyanobacteria evolve; they use water as a reducing agent, thereby producing oxygen as a waste product. |
| 1850 million years ago | Unicellular eukaryotes appear. |
| 1200 million years ago | Simple multicellular organisms evolve, mostly consisting of cell colonies of limited complexity. |
| $580-500$ million years ago | Most modern phyla of animals begin to appear in the fossil record during the Cambrian explosion. |
| 535 million years ago | Major diversification of living things in the oceans: chordates, arthropods (e.g., trilobites, crustaceans), echinoderms, mollusks, brachiopods, foraminifers, radiolarians, etc. |
| 485 million years ago | First vertebrates with true bones (jawless fishes) evolve. |
| 434 million years ago | First primitive plants arise on land. |
| 225 million years ago | Earliest dinosaurs (prosauropods) and teleost fishes appear. |
| 220 million years ago | Gymnosperm forests dominate the land; herbivores grow to huge sizes. |
| 215 million years ago | First mammals evolve. |
| 65.5 million years ago | The Cretaceous-Tertiary extinction event eradicates about half of all animal species, including all of the dinosaurs. |
| 6.5 million years ago | First hominids evolve. |
| 2 million years ago | First members of the genus Homo appear in the fossil record. |
| 350 thousand years ago | Neanderthals appear. |
| 200 thousand years ago | Anatomically modern humans appear in Africa. |
| 30 thousand years ago | Extinction of Neanderthals. |

![li03857775e0](li03857775e0.jpg)

FIGURE 1-2 Charles Darwin (1809-1882). Four years after his epic voyage on HMS Beagle, Darwin had already begun formulating in private notebooks his concept of natural selection, which would be published in his Origin of Species (1859). [Charles Darwin on the Galapagos Islands by Howat, Andrew (20th century)/Private Collection/© Look and Learn/ Bridgeman Images.]

![li04857775e0](li04857775e0.jpg)

FIGURE 1-3 Cells come in an astounding assortment of shapes and sizes. Some of the morphological variety of cells is illustrated in these photographs. In addition to morphology, cells differ in their ability to move, internal organization (prokaryotic versus eukaryotic cells), and metabolic activities. (a) Eubacteria: Lactococcus lactis, which are used to produce cheese such as Roquefort, Brie, and Camembert. Note the dividing cells. (b) A mass of archaeans (Methanosarcina) that produce their energy by converting carbon dioxide and hydrogen gas to methane. Some species that live in the rumens of cattle give rise to $>150$ liters of methane gas each day. (c) Human blood cells, shown in false color. The red cells are oxygen-bearing erythrocytes, the white cells (leukocytes) are part of the immune system and fight infection, and the green cells are platelets that plug wounds and contain substances to initiate blood clotting. (d) A colonial single-celled green alga,

Volvox aureus. The large spheres are made up of many individual cells, visible as blue or green dots. The yellow masses inside are daughter colonies, each made up of many cells. (e) A single Purkinje neuron of the cerebellum, which can form more than a hundred thousand connections with other cells through its branched network of dendrites. The cell was made visible by introduction of a green fluorescent protein; the cell body is the bulb at the upper right. (f) Plant cells are fixed firmly in place in vascular plants, supported by a rigid cellulose skeleton. Spaces between the cells are joined into tubes for transport of water and food. (Part (a) Gary Gaugler/Science Source. Part (b) Power and Syred/Science Source. Part (c) Science Source. Part (d) micro_photo//Stockphoto/Getty Images. Part (e) Courtesy of Dr. Helen M. Blau (Stanford University School of Medicine) and Dr. Clas B. Johansson (Karolinska Institutet). Part (f) Biophoto Associates/Science Source.]
molecules and macromolecules found in biological systems. Next we discuss the fundamental aspects of cell structure and function that are conserved in present-day organisms, focusing first on prokaryotic organisms-single-celled organisms without a nucleus-and their uses in studying the basic molecules of life. Then we discuss the structure and function of eukaryotic cells-cells with a defined nucleus-focusing on their many organelles. This discussion is followed by a section describing the use of unicellular eukaryotic organisms in investigations of molecular cell biology, focusing on yeasts and the parasite that causes malaria.

We now have the complete sequences of the genomes of several thousand metazoans (multicellular animals), and these sequences have provided considerable insight into the

(a)
![li05857775e0](li05857775e0.jpg)
(d)
![li06857775e0](li06857775e0.jpg)

FIGURE 1-4 Living systems such as the human body consist of closely interrelated elements. (a) The surface of the hand is covered by a living organ, skin, that is composed of several layers of tissue. (b) An outer covering of hard, dead skin cells protects the body from injury, infection, and dehydration. This layer is constantly renewed by living epidermal cells, which also give rise to hair and fur in animals. Deeper layers of muscle and connective tissue give skin its tone and firmness. (c) Tissues are formed through subcellular adhesion
(e)
![li07857775e0](li07857775e0.jpg)
structures (desmosomes and hemidesmosomes) that join cells to one another and to an underlying layer of supporting fibers. (d) At the heart of cell-cell adhesion are its structural components: phospholipid molecules that make up the cell-surface membrane, and large protein molecules. Protein molecules that traverse the cell membrane often form strong bonds with internal and external fibers made of multiple proteins.
evolution of genes and organisms. The final section in this chapter shows us how this information can be used to refine the evolutionary relationships among organisms as well as our understanding of human development. Indeed, biologists use evolution as a research tool: if a gene and its protein have been conserved in all metazoans but are not found in unicellular organisms, the protein probably has an important function in all metazoans and thus can be studied in whatever metazoan organism is most suitable for the investigation. Because the structure and function of many types of metazoan cells is also conserved, we now understand the structure and function of many cell types in considerable detail, including muscle and liver cells and the sheets of epithelial cells that line the intestine and form our skin. But other cells-especially the multiple types that form our nervous and immune systems-still remain mysterious; much important cell biological experimentation is needed on these and other cell systems and organs that form our bodies.

### 1.1 The Molecules of Life

While large polymers are the focus of molecular cell biology, small molecules are the stage on which all cellular processes are set. Water, inorganic ions, and a wide array of relatively small organic molecules (Figure 1-5) account for 75 to 80 percent of living matter by weight, and water accounts for about 75 percent of a cell's volume. These small molecules, including water, serve as substrates for many of the reactions that take place inside the cell, including energy metabolism and cell signaling. Cells acquire these small molecules in different ways. Ions, water, and many small organic molecules are imported into the cell (see Chapter 11); other small molecules are synthesized within the cell, often by a series of chemical reactions (see Chapter 12).

Even in the structures of many small molecules, such as sugars, vitamins, and amino acids, we see the footprint of evolution. For example, all amino acids save glycine have an

![li08857775e0](li08857775e0.jpg)

FIGURE 1-5 Some of the many small molecules found in cells. Only the L-forms of amino acids such as serine are incorporated into proteins, not their D-mirror images; only the D-form of glucose, not its L-mirror image, can be metabolized to carbon dioxide and water.
asymmetric carbon atom, yet only the l-stereoisomer, never the D-stereoisomer, is incorporated into proteins. Similarly, only the D-stereoisomer of glucose is invariably found in cells, never the mirror-image L-stereoisomer (see Figure 1-5). At an early stage of biological evolution, our common cellular ancestor evolved the ability to catalyze reactions with one
stereoisomer instead of the other. How these selections happened is unknown, but now these choices are locked in place.

An important and universally conserved small molecule is adenosine triphosphate (ATP), which stores readily available chemical energy in two of its chemical bonds (Figure 1-6). When one of these energy-rich bonds in ATP is broken, forming ADP (adenosine diphosphate), the released energy can be harnessed to power energy-requiring processes such as muscle contraction or protein biosynthesis. To obtain energy for making ATP, all cells break down food molecules. For instance, when sugar is degraded to carbon dioxide and water, the energy stored in the sugar molecule's chemical bonds is released, and much of it can be "captured" in the energy-rich bonds in ATP. Bacterial, plant, and animal cells can all make ATP by this process. In addition, plants and a few other organisms can harvest energy from sunlight to form ATP in photosynthesis.

Other small molecules (e.g., certain hormones and growth factors) act as signals that direct the activities of cells (see Chapters 15 and 16), and neurons (nerve cells) communicate with one another by releasing and sensing certain small signaling molecules (see Chapter 22). The powerful physiological effects of a frightening event, for example, come from the instantaneous flooding of the body with the small-molecule hormone adrenaline, which mobilizes the "fight or flight" response.

Certain small molecules (monomers) can be joined to form polymers (also called macromolecules) through
![li09857775e0](li09857775e0.jpg)

FIGURE 1-6 Adenosine triphosphate (ATP) is the most common molecule used by cells to capture, store, and transfer energy. ATP is formed from adenosine diphosphate (ADP) and inorganic phosphate
(P) by photosynthesis in plants and by the breakdown of sugars and fats in most cells. The energy released by the splitting (hydrolysis) of $P_{i}$ from ATP drives many cellular processes.

repetition of a single type of covalent chemical-linkage reaction. Cells produce three types of large macromolecules: polysaccharides, proteins, and nucleic acids. Sugars, for example, are the monomers used to form polysaccharides. Different polymers of D-glucose form cellulose, an important component of plant cell walls, and glycogen, a storage form of glucose found in liver and muscle. The cell is careful to provide the appropriate mix of small molecules needed as precursors for synthesis of macromolecules.

## Proteins Give Cells Structure and Perform Most Cellular Tasks

Proteins, the workhorses of the cell, are the most abundant and functionally versatile of the cellular macromolecules. Cells string together 20 different amino acids in linear chains, each with a defined sequence, to form proteins (see Figure 2-14), which commonly range in length from 100 to 1000 amino acids. During or just after its polymerization, a linear chain of amino acids folds into a complex shape, conferring a distinctive three-dimensional structure and function on the protein (Figure 1-7). Humans obtain amino acids either by synthesizing them from other molecules or by breaking down proteins that we eat.

Proteins have a variety of functions in the cell. Many proteins are enzymes, which accelerate (catalyze) chemical reactions involving small molecules or macromolecules (see Chapter 3). Certain proteins catalyze steps in the synthesis of all proteins; others catalyze synthesis of macromolecules such
as DNA and RNA. Cytoskeletal proteins serve as structural components of a cell; for example, by forming an internal skeleton. Other proteins associated with the cytoskeleton power the movement of subcellular structures such as chromosomes, and even of whole cells, by using energy stored in the chemical bonds of ATP (see Chapters 17 and 18). Still other proteins bind adjacent cells together or form parts of the extracellular matrix (see Figure 1-4). Proteins can be sensors that change shape as temperature, ion concentrations, or other properties of the cell change. Many proteins that are embedded in the cell-surface (plasma) membrane import and export a variety of small molecules and ions (see Chapter 11). Some proteins, such as insulin, are hormones; others are hormone receptors that bind their target protein or small molecule and then generate a signal that regulates a specific aspect of cell function. Other important classes of proteins bind to specific segments of DNA, turning genes on or off (see Chapter 9). In fact, much of molecular cell biology consists of studying the function of specific proteins in specific cell types.

## Nucleic Acids Carry Coded Information for Making Proteins at the Right Time and Place

The macromolecule that garners the most public attention is deoxyribonucleic acid (DNA), whose functional properties make it the cell's "master molecule." The three-dimensional structure of DNA, first proposed by James D. Watson and Francis H. C. Crick in 1953, consists of two long helical strands that are coiled around a common axis to form a
![li10857775e0](li10857775e0.jpg)

FIGURE 1-7 Models of some representative proteins drawn to a common scale and compared with a small portion of a lipid bilayer, a DNA molecule, and an RNA molecule. Each protein has a defined three-dimensional shape held together by numerous chemical bonds. The illustrated proteins include enzymes (glutamine synthetase and adenylate kinase), an antibody (immunoglobulin), a hormone (insulin), and the blood's oxygen carrier (hemoglobin). [Glutamine synthetase
data from H. S. Gill and D. Eisenberg, 2001, Biochemistry 40:1903-1912, PDB ID 1fpy. Insulin data from E. N. Baker et al., 1988, Phil. Trans. R. Soc. Lond. B Biol. Sci. 319:369-456, PDB ID 4ins. Hemoglobin data from G. Fermi et al., 1984, J. Mol. Biol. 175:159-174, PDB ID 2hhb. Immunoglobulin data from L. J. Harris et al., 1998, J. Mol. Biol. 275:861-872, PDB ID 1igy. Adenylate kinase data from G. Bunkoczi et al., PDB ID 2c9y.]

FIGURE 1-8 DNA consists of two complementary strands wound around each other to form a double helix. The double helix is stabilized by weak hydrogen bonds between the $A$ and $T$ bases and between the $C$ and $G$ bases. During replication, the two strands are unwound and used as templates to produce complementary strands. The outcome is two identical copies of the original double helix, each containing one of the original strands and one new daughter (complementary) strand.

Nucleotide (T)
![li11857775e0](li11857775e0.jpg)
double helix (Figure 1-8). The double-helical structure of DNA, one of nature's most magnificent constructions, is critical to the phenomenon of heredity, the transfer of genetically determined characteristics from one generation to the next.

DNA strands are composed of monomers called nucleotides; these monomers are often referred to as bases because they contain cyclic organic bases (see Chapter 5). Four different nucleotides, abbreviated A, T, C, and G, are joined to form a DNA strand, with the base parts projecting inward from the backbone of the strand. Two strands bind together via the bases and twist to form a double helix. Each DNA double helix has a simple construction: wherever one strand has an A , the other strand has a T , and each C is matched with a G (see Figure 1-8). This complementary matching of the two strands is so strong that if complementary strands are separated under the right salt concentration and temperature conditions, they will spontaneously zip back together. This property is critical for DNA replication and inheritance, as we will learn in Chapter 5, and also underlies many of the techniques for studying DNA molecules that are detailed in Chapter 6.

The genetic information carried by DNA resides in its sequence, the linear order of nucleotides along a strand. Specific segments of DNA, termed genes, carry instructions for making specific proteins. Commonly, genes contain two parts: the coding region specifies the amino acid sequence of a protein; the regulatory region binds specific proteins and controls when and in which cells the gene's protein is made.

Most bacteria have a few thousand protein-coding genes; yeasts and other unicellular eukaryotes have about 5000. Humans and other metazoans have between 13,000 and 23,000 , while many plants have more. As we discuss later in this chapter, many of the genes in bacteria specify the sequences of proteins that catalyze reactions that occur universally, such as the metabolism of glucose and the synthesis of nucleic acids and proteins. These genes, and the proteins encoded by them, are conserved throughout all living organisms, and thus studies on the functions of these genes and proteins in bacterial cells have yielded profound insights into these basic life processes. Similarly, many genes in unicellular eukaryotes such as yeasts encode proteins that are conserved throughout all eukaryotes; we will see how yeasts have been used in studies of processes such as cell division that have yielded profound insights into human diseases such as cancer.

How is information stored in the sequence of DNA used? Cells use two processes in series to convert the coded information in DNA into proteins (Figure 1-9). In the first process, called transcription, the protein-coding region of a gene is copied into a single-stranded ribonucleic acid (RNA) whose sequence is the same as one of the two in the double-stranded DNA. A large enzyme, RNA polymerase, catalyzes the linkage of nucleotides into an RNA chain using DNA as a template. In eukaryotic cells, the initial RNA product is processed into a smaller messenger RNA (mRNA) molecule, which moves out of the nucleus to the cytoplasm, the region of the cell outside of the nucleus. Here the ribosome, an enormously complex molecular machine composed of both RNA and proteins, carries out the second process, called translation. During translation, the ribosome assembles and links together amino acids in the precise order dictated by the mRNA sequence according to the nearly universal genetic code. We examine the cell components that carry out transcription and translation in detail in Chapter 5.

In addition to its role in transferring information from nucleus to cytoplasm, RNA can serve as a framework for building a molecular machine. The ribosome, for example, is built of four RNA chains that bind to more than 50 proteins to make a remarkably precise and efficient mRNA reader and protein synthesizer. While most chemical reactions in cells are catalyzed by proteins, a few, such as the formation by ribosomes of the peptide bonds that connect amino acids in proteins, are catalyzed by RNA molecules.

Well before the entire human genome was sequenced, it was apparent that only about 10 percent of human DNA consists of protein-coding genes, and for many years the remaining 90 percent was considered "junk DNA"! In recent years, we've learned that much of the so-called junk DNA is actually copied into thousands of RNA molecules that, though they do not encode proteins, serve equally important purposes in the cell (see Chapter 10). At present, however, we know the function of only a very few of these abundant noncoding RNAs.

Like enzymes, certain RNA molecules, termed ribozymes, catalyze chemical reactions, as exemplified by the RNA inside a ribosome. Many scientists support the RNA world hypothesis, which proposes that RNA molecules that could replicate themselves were the precursors of current life forms;

![li12857775e0](li12857775e0.jpg)

FIGURE 1-9 The information encoded in DNA is converted into the amino acid sequences of proteins by a multistep process.
Step $\mathbf{\square}$ Transcription factors and other proteins bind to the regulatory regions of the specific genes they control to activate those genes. Step $\mathbf{\square}$ RNA polymerase begins transcription of an activated gene at a specific location, the start site. The polymerase moves along the DNA, linking nucleotides into a single-stranded pre-mRNA transcript using one of the DNA strands as a template. Step $\mathbf{\square}$ The transcript is processed to remove noncoding sequences. Step $\mathbf{\square}$ In a eukaryotic cell, the mature mRNA moves to the cytoplasm, where it is bound by ribosomes that read its sequence and assemble a protein by chemically linking amino acids into a linear chain.
billions of years ago, the RNA world gradually evolved into the DNA, RNA, and protein world of today's organisms.

All organisms must control when and where their genes are transcribed. Nearly all the cells in our bodies contain the full set of human genes, but in each cell type only some of these genes are active, or turned on, and used to make proteins. For instance, liver cells produce some proteins that are not produced by kidney cells, and vice versa. Moreover, many cells respond to external signals or changes in external conditions by turning specific genes on or off, thereby adapting their repertoire of proteins to meet current needs. Such control of gene activity depends on DNA-binding proteins called transcription factors, which bind to specific sequences of DNA and act as switches, either activating or repressing transcription of particular genes, as discussed in Chapter 9.

## Phospholipids Are the Conserved Building Blocks of All Cellular Membranes

In all organisms, cellular membranes are composed primarily of a bilayer (two layers) of phospholipid molecules. Each of these bipartite molecules has a "water-loving" (hydrophilic) "head" and a "water-hating" (hydrophobic) "tail." The two phospholipid layers of a membrane are oriented with all the hydrophilic heads directed toward the inner or outer surfaces of the membrane and the hydrophobic tails buried within its interior (Figure 1-10). Smaller amounts of other lipids, such as cholesterol, are inserted into this phospholipid framework. Cellular membranes are extremely thin relative to the size of a cell. If you magnify a bacterium or yeast cell about 10,000 times to the size of a soccer ball, the plasma membrane is about as thick as a sheet of paper!

Phospholipid membranes are impermeable to water, all ions, and virtually all hydrophilic small molecules. Thus each membrane in each cell also contains groups of proteins that allow specific ions and small molecules to cross. Other membrane proteins serve to attach the cell to other cells or to polymers that surround it; still others give the cell its shape or allow its shape to change. We will learn more about membranes and how molecules cross them in Chapters 7 and 11.

New cells are always derived from parental cells by cell division. We've seen that the synthesis of new DNA molecules is templated by the two strands of the parental DNA such that each daughter DNA molecule has the same sequence as the parental one. In parallel, new membranes are made by incorporation of lipids and proteins into existing membranes in the parental cell and divided between daughter cells by fission. Thus membrane synthesis, like DNA synthesis, is templated by a parental structure.
![li13857775e0](li13857775e0.jpg)

FIGURE 1-10 The watery interior of cells is surrounded by the plasma membrane, a two-layered shell of phospholipids. The phospholipid molecules are oriented with their hydrophobic fatty acyl chains (black squiggly lines) facing inward and their hydrophilic head groups (white spheres) facing outward. Thus both sides of the membrane are lined by head groups, mainly charged phosphates, adjacent to the watery spaces inside and outside the cell. All biological membranes have the same basic phospholipid bilayer structure. Cholesterol (red) and various proteins are embedded in the bilayer. The interior space is actually much larger relative to the volume of the plasma membrane than is depicted here.

### 1.2 Prokaryotic Cell Structure and Function

The biological universe consists of two types of cells: prokaryotic and eukaryotic. Prokaryotic cells such as bacteria consist of a single closed compartment that is surrounded by a plasma membrane, lack a defined nucleus, and have a relatively simple internal organization (Figure 1-11). Eukaryotic cells contain a defined membrane-bounded nucleus and extensive internal membranes that enclose the organelles (see Figure 1-12).

## Prokaryotes Comprise Two Kingdoms: Archaea and Eubacteria

In recent years, detailed analysis of DNA sequences from a variety of prokaryotic organisms has revealed two distinct kingdoms: the Eubacteria, often simply called "bacteria," and the Archaea. Eubacteria are single-celled organisms; they include the cyanobacteria, or "blue-green algae," which can be unicellular or filamentous chains of cells. Figure 1-11 illustrates the general structure of a typical eubacterial cell; archaeal cells have a similar structure. Bacterial cells are commonly $1-2 \mu \mathrm{~m}$ in size and consist of a single closed compartment containing the cytoplasm and bounded by the plasma membrane. The genome is composed of a single circular DNA molecule; many prokaryotes contain additional small circular DNA molecules called plasmids. Although bacterial cells do not have a defined nucleus, the DNA is extensively folded and condensed into the central region of the cell, called the nucleoid. In contrast, most ribosomes are found in the cytoplasm. Some bacteria also have an invagination of the cell membrane, called a mesosome, which is
![li14857775e0](li14857775e0.jpg)

FIGURE 1-11 Prokaryotic cells are have a relatively simple structure. (Left) Electron micrograph of a thin section of Escherichia coli, a common intestinal bacterium. The nucleoid, consisting of the bacterial DNA, is not enclosed within a membrane. E. coli and other gramnegative bacteria are surrounded by two membranes separated by the periplasmic space. The thin cell wall is adjacent to the inner membrane.
associated with synthesis of DNA and secretion of proteins. Many proteins are precisely localized within the cytosol or in the plasma membrane, indicating the presence of an elaborate internal organization. Unlike those in eukaryotes (see Figure 1-9), bacterial mRNAs undergo limited if any processing. And because there is no membrane barrier between bacterial DNA and cytoplasm, ribosomes are able to bind to an mRNA as soon as part of it has been synthesized by RNA polymerase; thus in prokaryotes, transcription and translation occur contemporaneously.

Bacterial cells possess a cell wall, which lies adjacent to the external side of the plasma membrane. The cell wall is composed of layers of peptidoglycan, a complex of proteins and oligosaccharides; it helps protect the cell and maintain its shape. Some bacteria (e.g., E. coli) have a thin inner cell wall and an outer membrane separated from the inner cell wall by the periplasmic space. Such bacteria are not stained by the Gram technique and thus are classified as gramnegative. Other bacteria (e.g., Bacillus polymyxa) that have a thicker cell wall and no outer membrane take the Gram stain and thus are classified as Gram-positive.

In addition to DNA sequence distinctions that separate them from eubacteria, archaea have cell membranes that differ dramatically in composition from those of eubacteria and eukaryotes. Many archaeans grow in unusual, often extreme, environments that may resemble the ancient conditions that existed when life first appeared on Earth. For instance, halophiles ("salt lovers") require high concentrations of salt to survive, and thermoacidophiles ("heat and acid lovers") grow in hot $\left(80^{\circ} \mathrm{C}\right)$ sulfur springs, where a pH of less than 2 is common. Still other archaeans live in oxygen-free milieus and generate methane $\left(\mathrm{CH}_{4}\right)$ by combining water with carbon dioxide.
![li15857775e0](li15857775e0.jpg)
(Right) This artist's drawing shows the nucleoid (blue) and a magnification of the layers that surround the cytoplasm. Most of the cell is composed of water, proteins, ions, and other molecules that are too small to be depicted at the scale of this drawing. (Electron micrograph courtesy of I. D. J. Burdett and R. G. E. Murray.)

## Escherichia coli Is Widely Used in Biological Research

The bacterial lineage includes Escherichia coli, a favorite experimental organism, which in nature is common in soil and in animal intestines. E. coli and several other bacteria have a number of advantages as experimental organisms. They grow rapidly in a simple and inexpensive medium containing glucose and salts, in which they can synthesize all necessary amino acids, lipids, vitamins, and other essential small molecules. Like all bacteria, E. coli possesses elegant mechanisms for controlling gene activity that are now well understood
(see Chapter 9). Over time, researchers have developed powerful systems for genetic analysis of this organism. These systems are facilitated by the small size of bacterial genomes, the ease of obtaining mutants, the availability of techniques for transferring genes into bacteria, an enormous wealth of knowledge about bacterial gene control and protein functions, and the relative simplicity of mapping genes relative to one another in the bacterial genome. In Chapter 6 we see how E. coli is used in recombinant DNA research.

Bacteria such as E. coli that grow in environments as diverse as the soil and the human gut have about 4000 genes, encoding about the same number of proteins (Table 1-2).

TABLE 1-2 Genome Sizes of Organisms Used in Molecular Cell Biology Research That Have Been Completely Sequenced

|  | Base Pairs (Millions) | Approximate Number of <br> Encoded Proteins* | Chromosomes** | Reference |
| :-- | :--: | :--: | :--: | :--: |
| Eubacteria |  |  |  |  |
| Mycoplasma genitalium | 0.58 | 500 | 1 | a |
| Helicobacter pylori | 1.67 | 1,500 | 1 | a |
| Haemophilus influenza | 1.83 | 1,600 | 1 | a |
| Escherichia coli | 4.64 | 4,100 | 1 | a |
| Bacillus subtilis | 4.22 | 4,200 | 1 | a |
| Archaea |  |  |  |  |
| Methanococcus jannaschii | 1.74 | 1,800 | 1 | a |
| Sulfolobus solfataricus | 2.99 | 3,000 | 1 | a |
| Single-Celled Eukaryotes |  |  |  |  |
| Saccharomyces cerevisiae | 12.16 | 6,700 | 16 | b |
| Chlamydomonas reinhardtii | 120.4 | 14,400 | 17 | b |
| Plasmodium falciparum | 23.26 | 5,400 | 14 | b |
| Multicellular Eukaryotes (Metazoans) |  |  |  |  |
| Drosophila melanogaster | 168.74 | 13,900 | 6 | b |
| Caenorhabditis elegans | 100.29 | 20,500 | 6 | b |
| Schmidtea mediterranea (planarian) | 480 | $>20,000^{* * *}$ | 4 | c |
| Danio rerio (zebrafish) | 1412.46 | 26,500 | 25 | b |
| Gallus gallus (chicken) | 1072.54 | 15,500 | 33 | b |
| Mus musculus (mouse) | 3480.96 | 23,100 | 21 | b |
| Homo sapiens (human) | 3326.74 | 20,800 | 24 | b |
| Arabidopsis thaliana | 135.67 | 27,400 | 5 | b |

*Numbers of encoded proteins are current estimates rounded to the nearest 100 based on genome DNA sequences. They will likely change slightly in eubacteria and archaea because of the inclusion of newly discovered genes that code for very small proteins, and modestly in eukaryotes because of newly discovered small genes and because of pseudogenes that are not expressed.
**Only nuclear chromosomes are counted in eukaryotes, including distinct sex chromosomes in metazoans.
***Predicted value.
source: Table courtesy of Dr. Juan Alvarez-Dominguez. References: a, http://www.ncbi.nlm.nih.gov/genome/; b, http://ensemblgenomes.org/; c, http://www.genome.gov/12512286.

Parasitic bacteria such as the Mycoplasma species acquire amino acids and other nutrients from their host cells, and they lack the genes for enzymes that catalyze reactions in the synthesis of amino acids and certain lipids. Many bacterial genes encoding proteins essential for DNA, RNA, protein synthesis, and membrane function are conserved in all organisms, and much of our knowledge of these important cellular processes was uncovered first by studies in E. coli and other bacteria. For example, certain E. coli membrane proteins that import amino acids across the plasma membrane are closely related in sequence, structure, and function to membrane proteins in certain mammalian brain cells that import small nerve-to-nerve signaling molecules called neurotransmitters (see Chapters 11 and 22).

Because many of its genes and proteins, as well as their functions, are conserved in all organisms, E. coli has been chosen by scientists as a favorite model organism: an experimental system in which the study of specific genes or proteins, or aspects of cell or organismal function or regulation, can provide an understanding of similar molecules or processes in other species. Throughout this chapter, we will encounter other model organisms that have been chosen because, like E. coli, they are easy to grow and study. Of course, many bacteria cause serious diseases, and research on them is often focused on understanding their unique biology and on discovering antibiotics that selectively kill them but not their human or animal hosts.

### 1.3 Eukaryotic Cell Structure and Function

Eukaryotes comprise all members of the plant and animal kingdoms as well as protozoans (proto, "primitive"; zoan, "animal"), which are exclusively unicellular and include fungi and amoebae. Eukaryotic cells are commonly about $10-100 \mu \mathrm{~m}$ across, generally much larger than bacteria. A typical human fibroblast, a connective tissue cell, is about $15 \mu \mathrm{~m}$ across, with a volume and dry weight some thousands of times those of an E. coli cell. An amoeba, a singlecelled protozoan, can have a cell diameter of approximately 0.5 mm , more than 30 times that of a fibroblast.

Eukaryotic cells, like prokaryotic cells, are surrounded by a plasma membrane. However, unlike prokaryotic cells, most eukaryotic cells (the human red blood cell is an exception) also contain extensive internal membranes that enclose specific subcellular compartments, the organelles, and separate them from the cytoplasm (Figure 1-12). The cytosol, the organelle-free part of the cytoplasm, contains water, dissolved ions, small molecules, and proteins. Plant cells and most fungal cells are surrounded by a cell wall that gives the cell a rigid shape and also allows for rapid cell expansion.

All eukaryotic cells have many of the same organelles and other subcellular structures. Many organelles are surrounded by a single phospholipid membrane, but the nucleus, mitochondrion, and chloroplast are enclosed by two membranes. Each organelle membrane and each space in the interior of an organelle has a unique set of proteins that enable it to carry out its specific functions, including enzymes that catalyze requisite chemical reactions. The membranes defining these subcellular compartments contain proteins that control their internal ionic composition so that it generally differs from that of the surrounding cytosol as well as that of the other organelles. Here we describe the organelles common to all eukaryotic cells as well as several that are found only in certain types of eukaryotes. We begin with the proteins that give eukaryotic cells their shapes and organize the organelles.

## The Cytoskeleton Has Many Important Functions

The cytoplasm contains an array of fibrous proteins collectively called the cytoskeleton (see Chapters 17 and 18). Three classes of fibers compose the cytoskeleton: microtubules ( 20 nm in diameter), built of polymers of the protein tubulin; microfilaments ( 7 nm in diameter), built of the protein actin; and intermediate filaments ( 10 nm in diameter). All of these fibers are long chains of multiple copies of one or more small protein subunits (Figure 1-13). The cytoskeleton gives the cell strength and rigidity, thereby helping to maintain its shape; this is perhaps most obvious with neurons, in which microtubules and other fibers allow the formation of the long, slim protuberances-the axons and dendrites (see Figure 1-3e and Chapter 22)—that emanate from the cell body and allow each neuron to carry out its specialized functions. Cytoskeletal fibers also control movement of structures within the cell; for example, some cytoskeletal fibers connect to organelles or provide tracks along which organelles and chromosomes move. Other fibers play key roles in cell motility. Perhaps most important, cell division and the segregation of chromosomes and organelles into the two daughter cells could not occur without the organizational framework provided by the cytoskeleton and its associated proteins.

Cilia and flagella are similar extensions of the plasma membrane. They contain a bundle of microtubules that gives them shape and, together with motor proteins, allows them to beat rhythmically. They propel materials across epithelial surfaces (Figure 1-14), enable sperm to swim, and push eggs through the oviduct (see Chapter 18). As detailed in Chapter 16, most vertebrate cells contain at least one cilium that plays a key role in cell-cell signaling.

## The Nucleus Contains the DNA Genome, RNA Synthetic Apparatus, and a Fibrous Matrix

The nucleus, the largest organelle in animal cells, is surrounded by two membranes, each one a phospholipid bilayer containing many different types of proteins

![li16857775e0](li16857775e0.jpg)

Plasma membrane controls movement of molecules in and out of the cell and functions in cell-cell signaling and cell adhesion.

Mitochondria, which are surrounded by a double membrane, generate ATP by oxidation of glucose and fatty acids.

Lysosomes, which have an acidic lumen, degrade material internalized by the cell and worn-out cellular membranes and organelles.

Nuclear envelope, a double membrane, encloses the contents of the nucleus; the outer nuclear membrane is continuous with the rough ER.

Nucleolus is a nuclear subcompartment where most of the cell's rRNA is synthesized.

Nucleus is filled with chromatin composed of DNA and proteins; site of mRNA and tRNA synthesis.

Smooth endoplasmic reticulum (ER) contains enzymes that synthesize lipids and detoxify certain hydrophobic molecules.

Rough endoplasmic reticulum (ER) functions in the synthesis, processing, and sorting of secreted proteins, lysosomal proteins, and certain membrane proteins.

Golgi complex processes and sorts secreted proteins, lysosomal proteins, and membrane proteins synthesized on the rough ER.

Secretory vesicles store secreted proteins and fuse with the plasma membrane to release their contents.

Peroxisomes contain enzymes that break down fatty acids into smaller molecules used for biosynthesis and also detoxify certain molecules.

Cytoskeletal fibers form networks and bundles that support cellular membranes, help organize organelles, and participate in cell movement.

Microvilli increase surface area for absorption of nutrients from surrounding medium.

Cell wall, composed largely of cellulose, helps maintain the cell's shape and provides protection against mechanical stress.

Vacuole stores water, ions, and nutrients, degrades macromolecules, and functions in cell elongation during growth.

Chloroplasts, which carry out photosynthesis, are surrounded by a double membrane and contain a network of internal membrane-bounded sacs.

Plasmodesmata are tubelike cell junctions that span the cell wall and connect the cytoplasms of adjacent plant cells.
(b)
![li17857775e0](li17857775e0.jpg)

FIGURE 1-12 Subcellular organization of eukaryotic cells. (a) Schematic overview of a "typical" animal cell (top) and plant cell (bottom) and their major substructures. Not every cell type will contain all the organelles, granules, and fibrous structures shown here, and other substructures can be present in some cell types. Cells also differ considerably in shape and in the prominence of various organelles and substructures. (b) Electron micrograph of a plasma cell, a type of white blood cell that secretes antibodies, showing some of the larger organelles. [Part (b) courtesy of I. D. J. Burdett and R. G. E. Murray.]

![li18857775e0](li18857775e0.jpg)

FIGURE 1-13 The three types of cytoskeletal filaments have characteristic distributions within mammalian cells. Three views of the same cell. A cultured fibroblast was permeabilized and then treated with three different antibody preparations. Each antibody binds specifically to the protein monomers forming one type of filament and is chemically linked to a differently colored fluorescent
 dye (green, blue, or red). Visualization of the stained cell in a fluorescence microscope reveals the locations of filaments bound to a particular dye-antibody preparation. In this case, microtubules are stained blue; microfilaments, red; and intermediate filaments, green. All three fiber systems contribute to the shape and movements of cells. [Courtesy of V. Small.]
(Figure 1-15). The inner nuclear membrane defines the nucleus itself. In most cells, the outer nuclear membrane is continuous with the endoplasmic reticulum, and the space between the inner and outer nuclear membranes is continuous with the lumen of the endoplasmic reticulum (see Figure 1-15a). The two nuclear membranes appear to fuse at nuclear pore complexes, ringlike structures composed of specific membrane proteins through which material moves between the nucleus and the cytosol. The structure of the nuclear pores and the regulated transport of material through them are detailed in Chapters 10 and 13. Intermedi-ate-filament proteins called lamins form a two-dimensional network, called the nuclear lamina, along the inner surface of the inner membrane, giving it shape and rigidity. The breakdown of the lamina occurs early in cell division, as we detail in Chapter 19. In a growing or differentiating cell, the nucleus is metabolically active, as it is the site of DNA replication and the synthesis of ribosomal RNA, mRNA, and a large
![li19857775e0](li19857775e0.jpg)

FIGURE 1-14 Surface of the ciliated epithelium lining a mammalian trachea viewed in a scanning electron microscope. Beating cilia, which have a core of microtubules, propel mucus and foreign particles out of the respiratory tract, keeping the lungs and airways clear. [NIBSC/Science Source.]
variety of noncoding RNAs (see Chapters 5 and 9). Inside the nucleus one can often see a dense subcompartment, termed the nucleolus, where ribosomal RNA is synthesized and ribosomes are assembled (see Figure 1-15b and Chapter 10).

The total DNA in an organism is referred to as its genome. In most prokaryotic cells, most or all of the genetic information resides in a single circular DNA molecule about a millimeter in length; this molecule lies, folded back on itself many times, in the central region of the micrometersized cell (see Figure 1-11). In contrast, DNA in the nuclei of eukaryotic cells is distributed among multiple long linear structures called chromosomes. The length and number of chromosomes are the same in all cells of a particular species, but vary among different species (see Table 1-2). Each chromosome comprises a single DNA molecule associated with numerous histones and other proteins. In a nucleus that is not dividing, the chromosomes are dispersed and are not dense enough to be observed in the light microscope. Only during cell division are individual chromosomes visible by light microscopy. When nondividing cells are visualized in an electron microscope, the non-nucleolar regions of the nucleus, called the nucleoplasm, can be seen to have dark- and light-staining areas. The dark areas, which are often closely associated with the nuclear membrane, contain condensed, concentrated DNA that cannot be transcribed into RNA, called heterochromatin (see Figure 1-15b).

Chromosomes, which stain intensely with basic dyes, are visible in light and electron microscopes only during cell division, when the DNA becomes tightly compacted (Figure 1-16). Although the large genomic DNA molecule in prokaryotes is associated with proteins, the arrangement of DNA within a bacterial chromosome differs greatly from that within the linear chromosomes of eukaryotic cells; bacterial chromosomes are circular and are associated with different types of proteins than are eukaryotic chromosomes.

## Eukaryotic Cells Contain a Large Number of Internal Membrane Structures

We noted earlier that, unlike prokaryotic cells, most eukaryotic cells contain extensive internal membranes that enclose

![li20857775e0](li20857775e0.jpg)

FIGURE 1-15 Structure of the nucleus. (a) Schematic diagram of the structure of a typical cell nucleus and the connection of the outer nuclear membrane with the rough endoplasmic reticulum. The small black dots attached to the membrane of the rough endoplasmic reticulum are ribosomes that are synthesizing membrane and secreted
![li21857775e0](li21857775e0.jpg)

FIGURE 1-16 Individual chromosomes can be seen in cells during cell division (mitosis). (a) During the S phase of the cell cycle (see Figure 1-21) chromosomes are duplicated, and the daughter "sister chromatids," each with a complete copy of the chromosomal DNA, remain attached at the centromere. (b) During the actual cell division process (mitosis), the chromosomal DNA becomes highly compacted, and the pairs of sister chromatids can be seen in the electron micro-
(b)
![li22857775e0](li22857775e0.jpg)
proteins. (b) Electron micrograph of a pancreatic acinar cell from the bat Myotis lucifugus. The nucleolus is a subcompartment of the nucleus and is not surrounded by a membrane; most ribosomal RNA is produced in the nucleolus. Darkly staining areas in the nucleus outside the nucleolus are regions of heterochromatin. [Part (b) Don W. Fawcett/Science Source.]
(c)
![li23857775e0](li23857775e0.jpg)
scope, as depicted here. (c) Light-microscope image of a chromosomal spread from a cultured human male lymphoid cell arrested in the metaphase stage of mitosis by treatment with the microtubule-depolymerizing drug colcemid. There is a single copy of the duplicated $X$ and $Y$ chromosomes and two copies of each of the others. [Part (b) Medical RF/The Medical File/Peter Arnold Inc. Part (c) courtesy of Tatyana Pyntikova.]

specific subcellular compartments, termed organelles. Here we review the organelles and their functions.

Endoplasmic Reticulum and Golgi Complex Generally the largest membrane in a eukaryotic cell encloses the organelle termed the endoplasmic reticulum (ER)—an extensive network of closed, flattened membrane-bounded sacs called cisternae (Figure 1-17; see also Figure 1-15a). The endoplasmic reticulum has a number of functions in the cell but is particularly important in the synthesis of lipids, secreted proteins, and many types of membrane proteins. The smooth endoplasmic reticulum is smooth because it lacks ribosomes; it is the site of synthesis of fatty acids and phospholipids.

In contrast, the cytosolic side of the rough endoplasmic reticulum is studded with ribosomes; these ribosomes synthesize certain membrane and organelle proteins and virtually all proteins that are to be secreted from the cell (see Chapter 13). As a growing polypeptide emerges from a ribosome, it passes through the rough ER membrane with the help of specific transport proteins that are embedded in the membrane. Newly made membrane proteins remain associated with the rough ER membrane, and proteins to be secreted accumulate in the lumen, the aqueous interior of the organelle. Several minutes after proteins are synthesized in the rough ER, most of them leave the organelle within small membrane-bounded transport vesicles. These vesicles, which bud from regions of the rough ER not coated with ribosomes, carry the proteins to another membrane-bounded organelle, the Golgi complex (see Figure 1-17). As detailed in Chapter 14, secreted and membrane proteins undergo a series of enzyme-catalyzed chemical modifications in the Golgi complex that are essential for these proteins to function normally.

After proteins to be secreted and membrane proteins are modified in the Golgi complex, they are transported out of the complex by a second set of vesicles, which bud from one side of the Golgi complex. Some vesicles carry membrane
proteins destined for the plasma membrane or soluble proteins to be released from the cell into the extracellular space; others carry soluble or membrane proteins to lysosomes or other organelles. How intracellular transport vesicles "know" with which membranes to fuse and where to deliver their contents is also discussed in Chapter 14.
Endosomes Although transport proteins in the plasma membrane mediate the movement of ions and small molecules into the cell across the lipid bilayer, proteins and some other soluble macromolecules in the extracellular milieu are internalized by endocytosis. In this process, a segment of the plasma membrane invaginates into a coated pit, whose cytosolic face is lined by a specific set of proteins that cause vesicles to form. The pit pinches from the membrane into a small membrane-bounded vesicle that contains the extracellular material. The vesicle is delivered to and fuses with an endosome, a sorting station of membrane-limited tubules and vesicles (Figure 1-18). From this compartment, some membrane proteins are recycled back to the plasma membrane; other membrane proteins are transported in vesicles that eventually fuse with lysosomes for degradation. The entire endocytic pathway is described in detail in Chapter 14.
Lysosomes Lysosomes provide an excellent example of the ability of intracellular membranes to form closed compartments in which the composition of the lumen (the aqueous interior of the compartment) differs substantially from that of the surrounding cytosol. Found exclusively in animal cells, lysosomes are responsible for degrading many components that have become obsolete for the cell or organism. The process by which an aged organelle is degraded in a lysosome is called autophagy ("eating oneself"). Materials taken into a cell by endocytosis or phagocytosis may also be degraded in lysosomes (see Figure 1-18). In phagocytosis, large, insoluble particles (e.g., bacteria) are enveloped by the plasma membrane and internalized.

FIGURE 1-17 The Golgi complex and rough endoplasmic reticulum. An electron micrograph of a section of a human liver cell shows the abundant ribosome-studded rough endoplasmic reticulum and the Golgi complex, as well as many ribosomes free in the cytosol. [Courtesy George E. Palade EM Slide Collection, University of California, San Diego.]
![li24857775e0](li24857775e0.jpg)

FIGURE 1-18 Endosomes and other cellular structures deliver materials to lysosomes. Schematic overview of three pathways by which materials are moved to lysosomes. Soluble macromolecules and molecules bound to proteins on the cell surface are taken into the cell by invagination of segments of the plasma membrane and delivered to lysosomes through the endocytic pathway Whole cells and other large, insoluble particles move from the cell surface to lysosomes through the phagocytic pathway Worn-out organelles and bulk cytoplasm are delivered to lysosomes through the autophagic pathway Within the acidic lumen of a lysosome, hydrolytic enzymes degrade proteins, nucleic acids, lipids, and other large molecules.
![li25857775e0](li25857775e0.jpg)

Lysosomes contain a group of enzymes that degrade polymers into their monomeric subunits. For example, nucleases degrade RNA and DNA into their mononucleotide building blocks; proteases degrade a variety of proteins and peptides; phosphatases remove phosphate groups from mononucleotides, phospholipids, and other compounds; still other enzymes degrade complex polysaccharides and glycolipids into smaller units. All of these lysosomal enzymes, collectively termed acid hydrolases, work most efficiently at acidic pH values. The acidic pH helps to denature proteins, making them accessible to the action of the lysosomal hydrolases. These enzymes are less active at the neutral pH of cells and most extracellular fluids. Thus if a lysosome releases its enzymes into the cytosol, where the pH is between 7.0 and 7.3, they cause little degradation of cytosolic components. Cytosolic and nuclear proteins generally are not degraded in lysosomes, but rather in proteasomes, large multiprotein complexes in the cytosol (see Chapter 3).

Peroxisomes All animal cells (except erythrocytes) and many plant and fungal cells contain peroxisomes, a class of roughly spherical organelles $0.2-1.0 \mu \mathrm{~m}$ in diameter. Peroxisomes contain several oxidases: enzymes that use molecular oxygen to oxidize organic substances and in the process form hydrogen peroxide $\left(\mathrm{H}_{2} \mathrm{O}_{2}\right)$, a corrosive substance. Peroxisomes also contain copious amounts of the enzyme catalase, which degrades hydrogen peroxide to yield water and oxygen (see Chapter 12). Plant seeds contain glyoxisomes, small organelles that oxidize stored lipids as a source of carbon and energy for growth. They are similar to peroxisomes and contain many of the same types of enzymes as well as additional ones used to convert fatty acids into glucose precursors.

Plant Vacuoles Most plant cells contain at least one mem-brane-limited vacuole that accumulates and stores water, ions, and small-molecule nutrients such as sugars and amino acids. A variety of membrane proteins in the vacuolar membrane allow the transport of these molecules from the cyto-
sol and their retention in the vacuole lumen. The number and size of vacuoles depend on both the type of cell and its stage of development; a single vacuole may occupy as much as 80 percent of a mature plant cell (Figure 1-19). Like that of a lysosome, the lumen of a vacuole contains a battery of degradative enzymes and has an acidic pH , which is maintained by similar transport proteins in the vacuolar membrane. Thus plant vacuoles may also have a degradative function similar to that of lysosomes in animal cells. Similar
![li26857775e0](li26857775e0.jpg)

FIGURE 1-19 Electron micrograph of a thin section of a leaf cell. In this cell, a single large vacuole occupies much of the cell volume. Parts of five chloroplasts and the cell wall are also visible. Note the internal subcompartments in the chloroplasts. [Biophoto Associates/ Science Source.]

storage vacuoles are found in green algae and in many microorganisms such as fungi.

## Mitochondria Are the Principal Sites of ATP Production in Aerobic Cells

Most eukaryotic cells contain many mitochondria (Figure 1-20), which occupy up to 25 percent of the volume of the cytoplasm. These complex organelles, which are the main sites of ATP production during aerobic metabolism, are generally exceeded in size only by the nucleus, vacuoles, and chloroplasts. The two membranes that bound a mitochondrion differ in composition and function. The outer mitochondrial membrane contains proteins that allow many molecules to move from the cytosol to the intermembrane space between the inner and outer membrane. The inner mitochondrial membrane, which is much less permeable, is about 20 percent lipid and 80 percent protein-a proportion of protein that is higher than those in other cellular membranes. The surface area of the inner membrane is greatly increased by a large number of infoldings, or cristae, that protrude into the matrix, or central aqueous space.

In non-photosynthetic cells, the principal fuels for ATP synthesis are fatty acids and glucose. The complete aerobic degradation of 1 molecule of glucose to carbon dioxide and water is coupled to the synthesis of as many as 30 molecules of ATP from ADP and inorganic phosphate (see Figure 1-6). In eukaryotic cells, the initial stages of glucose degradation take place in the cytosol, where 2 ATP molecules per glucose molecule are generated. The terminal stages of oxidation and
![li27857775e0](li27857775e0.jpg)

FIGURE 1-20 Electron micrograph of a mitochondrion in a pancreas cell. The smooth outer membrane forms the outside boundary of the mitochondrion. The inner membrane is distinct from the outer membrane and is highly invaginated to form sheets and tubes called cristae; ATP is produced by proteins embedded in the membranes of the cristae. The aqueous space between the inner and outer membranes (the intermembrane space) and the space inside the inner membrane (the matrix) each contain specific proteins important for the metabolism of sugars, lipids, and other molecules. [Keith R. Porter/ Science Source.]

ATP synthesis are carried out by enzymes in the mitochondrial matrix and inner membrane (see Chapter 12); as many as 28 ATP molecules per glucose molecule are generated in mitochondria. Similarly, virtually all the ATP formed in the oxidation of fatty acids to carbon dioxide is generated in mitochondria. Thus mitochondria can be regarded as the "power plants" of the cell.

Mitochondria contain small DNA molecules that encode a small number of mitochondrial proteins; the majority of mitochondrial proteins are encoded by nuclear DNA. As discussed in Chapter 12, the popular endosymbiont hypothesis postulates that mitochondria originated by endocytosis of an ancient bacterium by the precursor of a eukaryotic cell; the bacterial plasma membrane evolved to become the inner mitochondrial membrane.

## Chloroplasts Contain Internal Compartments in Which Photosynthesis Takes Place

Except for vacuoles, chloroplasts are the largest and the most characteristic organelles in the cells of plants and green algae (see Figure 1-19). The endosymbiont hypothesis (see Chapter 12) posits that these organelles originated by endocytosis of a primitive photosynthetic bacterium. Chloroplasts can be as long as $10 \mu \mathrm{~m}$ and are typically $0.5-2 \mu \mathrm{~m}$ thick, but they vary in size and shape in different cells, especially among the algae. In addition to the inner and outer membranes that bound a chloroplast, this organelle also contains an extensive internal system of interconnected membranelimited vesicles called thylakoids, which are flattened to form disks. Thylakoids often form stacks called grana and are embedded in an aqueous matrix termed the stroma. The thylakoid membranes contain green pigments (chlorophylls) and other pigments that absorb light, as well as enzymes that generate ATP during photosynthesis. Some of the ATP is used to convert carbon dioxide into three-carbon intermediates by enzymes located in the stroma; the intermediates are then exported to the cytosol and converted into sugars.

The molecular mechanisms by which ATP is formed in mitochondria and chloroplasts are very similar, as explained in Chapter 12. Besides being surrounded by two membranes, chloroplasts and mitochondria have other features in common: both often migrate from place to place within cells, and both contain their own DNA, which encodes some of the key organelle proteins (see Chapter 12). The proteins encoded by mitochondrial or chloroplast DNA are synthesized on ribosomes within the organelles. However, most of the proteins in each organelle are encoded in nuclear DNA and are synthesized in the cytosol; these proteins are then incorporated into the organelles by processes described in Chapter 13.

## All Eukaryotic Cells Use a Similar Cycle to Regulate Their Division

Unicellular eukaryotes, animals, and plants all use essentially the same cell cycle, the series of events that prepares a cell to

![li28857775e0](li28857775e0.jpg)

FIGURE 1-21 During growth, all eukaryotic cells continually progress through the four phases of the cell cycle. In proliferating cells, the four phases of the cell cycle proceed successively. In humans, the cycle takes from 10 to 20 hours depending on cell type and developmental state. Yeasts divide much faster. During interphase, which consists of the $G_{1}, S$, and $G_{2}$ phases, the cell roughly doubles its mass. Replication of DNA during the $S$ phase leaves the cell with four copies of each type of chromosome. In the mitotic (M) phase, the chromosomes are evenly partitioned into two daughter cells, and in most cases the cytoplasm divides roughly in half. Under certain conditions, such as starvation or when a tissue has reached its final size, cells will stop cycling and remain in a waiting state called $G_{0}$. Some types of cells in $G_{0}$ can reenter the cell cycle if conditions change.
divide, and the same actual division process, called mitosis. The eukaryotic cell cycle is commonly divided into four phases (Figure 1-21). The chromosomes and the DNA they carry are duplicated during the $\mathbf{S}$ (synthesis) phase. The replicated chromosomes separate during the $\mathbf{M}$ (mitotic) phase, in which the cell divides, and each daughter cell gets a copy of each chromosome. The M and S phases are separated by two gap phases, the $G_{1}$ phase and the $G_{2}$ phase, during which mRNAs, proteins, lipids, and other cell constituents are made and the cell increases in size.

Under optimal conditions, some bacteria, such as E. coli, can divide to form two daughter cells once every 30 min utes. Most eukaryotic cells take considerably longer to grow and divide, generally several hours. Moreover, the cell cycle in eukaryotes is normally highly regulated (see Chapter 19). This tight control prevents imbalanced, excessive growth of cells and tissues if essential nutrients or certain hormonal signals are lacking. Some highly specialized cells in adult animals, such as neurons and striated muscle cells, divide rarely, if at all. However, an organism usually replaces worn-out cells or makes more cells in response to a new need, as exemplified by the generation of new muscle cells from undifferentiated stem cells in response to exercise or damage. Another example is the formation of additional red blood cells when a person ascends to a higher altitude and needs more capacity to capture oxygen. The fundamental defect in cancer is loss of the ability to control the growth and
division of cells. In Chapter 24 we examine the molecular and cellular events that lead to inappropriate, uncontrolled proliferation of cells.

### 1.4 Unicellular Eukaryotic Model Organisms

Our current understanding of the molecular functioning of eukaryotic cells largely rests on studies of just a few types of organisms, termed model organisms (Figure 1-22). Because of the evolutionary conservation of genes, proteins, organelles, cell types, and so forth, discoveries about biological structures and functions obtained with one experimental organism often apply to others. Thus researchers generally conduct studies with the organism that is most suitable for rapidly and completely answering the question being posed, knowing that the results obtained in one organism are likely to be broadly applicable. Indeed, many organisms, particularly rats, frogs, sea urchins, chickens, and slime molds, have been and continue to be immensely valuable for cell biology research. As more and more organisms have their entire genomes sequenced, a wide variety of other species are increasingly being used for investigations, especially for studies of the evolution of genes, cells, and organisms and of how organisms become adapted to diverse ecological niches.

As we have seen, bacteria are excellent models for studies of several cellular functions, but they lack the organelles found in eukaryotes. Unicellular eukaryotes such as yeasts are used to study many fundamental aspects of eukaryotic cell structure and function. Metazoan models such as the roundworm, fruit fly, and mouse are required to study more complex tissue and organ systems and development. As we will see in this section and the next, several eukaryotic model organisms are widely used to understand complex cell systems and mechanisms.

## Yeasts Are Used to Study Fundamental Aspects of Eukaryotic Cell Structure and Function

One group of single-celled eukaryotes, the yeasts, has proven exceptionally useful in molecular and genetic analysis of eukaryotic cell formation and function. Yeasts and their multicellular cousins, the molds, which collectively constitute the fungi, have an important ecological role in breaking down plant and animal remains for reuse. They also make numerous antibiotics and are used in the manufacture of bread, beer, and wine.

The common yeast used to make bread and beer, Saccharomyces cerevisiae, appears frequently in this book because it has proved to be an extremely useful experimental organism. Homologs of many of the approximately 6000 different proteins expressed in an S. cerevisiae cell (see Table 1-2) are found in most, if not all, eukaryotes and are important for cell division or for the functioning of individual eukaryotic organelles. Much of what we know of the proteins in the endoplasmic reticulum and Golgi complex

(a)
![li29857775e0](li29857775e0.jpg)

Yeast (Saccharomyces cerevisiae)
Control of cell cycle and cell division Protein secretion and membrane biogenesis
Function of the cytoskeleton Cell differentiation Aging Gene regulation and chromosome structure
(c)
![li30857775e0](li30857775e0.jpg)

Roundworm (Caenorhabditis elegans)

Development of the body plan Cell lineage
Formation and function of the nervous system
Control of programmed cell death Cell proliferation and cancer genes Aging
Behavior
Gene regulation and chromosome structure
(e)
![li31857775e0](li31857775e0.jpg)

Planarian (Schmidtea mediterranea)

Stem cells
Turnover of adult tissues
Wound healing
Regeneration
Pharynx
Photoreceptors
(g)
![li32857775e0](li32857775e0.jpg)

Mouse (Mus musculus), including cultured cells

Development of body tissues
Function of mammalian immune system
Formation and function of brain and nervous system
Models of cancers and other human diseases
Gene regulation and inheritance
Infectious disease
Behavior

FIGURE 1-22 Each eukaryotic organism used in cell biology has advantages for certain types of studies. The yeast Saccharomyces cerevisiae (a) has the cellular organization of a eukaryote but is a relatively simple single-celled organism that is easy to grow and to manipulate genetically. The green alga Chlamydomonas reinhardtii (b) is widely used to study photosynthesis and the structure and function of flagella. In the roundworm Caenorhabditis elegans (c), which has a small number of cells arranged in a nearly identical way in every worm, the formation of each individual cell can be traced. The fruit fly Drosophila melanogaster (d), first used to discover the properties of chromosomes, has been especially valuable in identifying genes that control embryonic development. Many of these genes are evolutionarily conserved in humans. Planaria (e) are flatworms that can regenerate any part of
(b)
![li33857775e0](li33857775e0.jpg)

Alga (Chlamydomonas reinhardtii)

Structure and function of flagella Chloroplasts and photosynthesis Organelle movement Phototaxis
(d)
![li34857775e0](li34857775e0.jpg)

Fruit fly (Drosophila melanogaster)
Development of the body plan Generation of differentiated cell lineages
Formation of the nervous system, heart, and musculature
Programmed cell death
Genetic control of behavior
Cancer genes and control of cell proliferation
Control of cell polarization
Effects of drugs, alcohol, pesticides
(f)
![li35857775e0](li35857775e0.jpg)

Zebrafish (Danio rerio)
Development of vertebrate body tissues
Formation and function of brain and nervous system
Birth defects
Cancer
(h)
![li36857775e0](li36857775e0.jpg)

Plant (Arabidopsis thaliana)
Development and patterning of tissues
Genetics of cell biology
Agricultural applications
Physiology
Gene regulation
Immunity
Infectious disease
the body that is cut off, including the head and the photoreceptors. The stem cells that give rise to their new cells and tissues are widely studied. The zebrafish Danio rerio (f) is used for rapid genetic screens to identify genes that control vertebrate development and organogenesis. Of the experimental animal systems, mice (Mus musculus) (g) are evolutionarily the closest to humans and have thus provided models for studying numerous human genetic and infectious diseases. The mustard-family weed Arabidopsis thaliana (h) has been used for genetic screens to identify genes involved in nearly every aspect of plant life. [Part (a) Scimat/Photo Researchers, Inc. Part (b) William Dentler University of Kansas. Part (c) Science Source. Part (d) Darwin Dale/Science Source. Part (e) Peter Reddien, MIT Whitehead Institute. Part (f) blickwinkel/Hartl/Alamy. Part (g) J. M. Labat/ Jacana/Photo Researchers, Inc. Part (h) Darwin Dale/Science Source.]

that promote protein secretion was elucidated first in yeasts (see Chapter 14). Yeasts were also essential for the identification of many proteins that regulate the cell cycle and catalyze DNA replication and transcription. S. cerevisiae (Figure 1-23a; see also Figure 1-22a) and other yeasts offer many advantages to molecular and cellular biologists:

- Vast numbers of yeast cells can be grown easily and cheaply in culture from a single cell; the cells in such clones are genetically identical and have the same biochemical properties. Individual proteins or multiprotein complexes can be purified from large amounts of cells and then studied in detail.
- Yeast cells may be either haploid (containing one copy of each chromosome) or diploid (containing two copies of each chromosome), and both forms can divide by mitosis; this ability makes isolating and characterizing mutations in genes encoding essential yeast cell proteins relatively straightforward.
- Yeasts, like many organisms, have a sexual cycle that allows exchange of genes between cells. Under starvation conditions, diploid cells undergo meiosis (see Chapter 19) to form haploid daughter cells, which are of two types, a and $\alpha$ cells. If haploid a and $\alpha$ cells encounter each other, they can fuse, forming an a/ $\alpha$ diploid cell that contains two copies of each chromosome, one from each parent cell (Figure 1-23b).

With the use of a single species such as S. cerevisiae as a model organism, results from studies carried out by tens of thousands of scientists worldwide, using multiple experimental techniques, can be combined to yield a deeper level of understanding of a single type of cell. As we will see many times in this book, conclusions based on studies of $S$. cerevisiae have often proved true for all eukaryotes and have formed the basis for exploring the evolution of more complex processes in multicellular animals and plants.

## Mutations in Yeast Led to the Identification of Key Cell Cycle Proteins

Biochemical studies can tell us much about an individual protein, but they cannot prove that it is required for cell division or any other cell process. The importance of a protein is demonstrated most firmly if a mutation that prevents its synthesis or makes it nonfunctional adversely affects the process under study.

In a classical genetics approach, scientists isolate and characterize mutants that lack the ability to do something a normal organism can do. Often large genetic "screens" are done to look for many different mutant individuals (e.g., fruit flies, yeast cells) that are unable to complete a certain process, such as cell division or muscle formation. Mutations are usually produced by treatment with a mutagen, a chemical or physical agent that promotes mutations in a largely random fashion. But how can we isolate and maintain mutant organisms or cells that are defective in some process, such as cell division or protein secretion, that is essential for survival?
(a)
![li37857775e0](li37857775e0.jpg)

Budding (S. cerevisiae)
(b)
![li38857775e0](li38857775e0.jpg)

FIGURE 1-23 The yeast Saccharomyces cerevisiae can be haploid or diploid and can reproduce sexually or asexually. (a) Scanning electron micrograph of the budding yeast Saccharomyces cerevisiae. These cells grow by an unusual type of mitosis termed mitotic budding. One daughter nucleus remains in the "mother" cell; the other daughter nucleus is transported into the bud, which grows in size and soon is released as a new cell. After each bud cell breaks free, a scar is left at the budding site, so the number of previous buds on the parent cell can be counted. The orange-colored cells are bacteria. (b) Haploid yeast cells can have different mating types, called a (blue) and $\alpha$ (orange). Both types contain a single copy of each yeast chromosome, half the usual number, and grow by mitotic budding. Two haploid cells that differ in mating type, one a and one $\alpha$, can fuse together to form an a/ $\alpha$ diploid cell that contains two copies of each chromosome; diploid cells can multiply by mitotic budding. Under starvation conditions, a diploid cell can undergo meiosis, a special type of cell division, to form four haploid ascospores. Rupture of an ascus releases four haploid spores, which can germinate into haploid a and $\alpha$ cells. These cells can also multiply asexually. [Part (a) SCIMAT/Science Source.]

One way is to isolate organisms with a temperaturesensitive mutation. These mutants are able to grow at the permissive temperature, but not at another, usually higher temperature, the nonpermissive temperature. Normal cells can grow at either temperature. In most cases, a tempera-ture-sensitive mutant produces an altered protein that works at the permissive temperature but unfolds and is nonfunctional at the nonpermissive temperature. Screens for temper-ature-sensitive mutations are most readily done with haploid

organisms such as yeasts because they have only one copy of each gene, and thus a mutation in it will immediately have a consequence.

By analyzing the effects of numerous different tempera-ture-sensitive mutations that altered the division of haploid yeast cells, geneticists discovered most of the genes necessary for cell division without knowing anything, initially, about which proteins they encode or how these proteins participate in the process. In general, the great power of genetics is to reveal the existence and relevance of all proteins required for a particular cell function without prior knowledge of their biochemical identity or molecular function. These "mutation-defined" genes can be isolated and replicated (cloned) with recombinant DNA techniques discussed in Chapter 6. With the isolated genes in hand, the encoded proteins can be produced in a test tube or in engineered bacteria or cultured cells. In this way, biochemists can investigate whether the genes necessary for cell division encode proteins that associate with other proteins or DNA or catalyze particular chemical reactions during cell division (see Chapter 19).

Most of these yeast cell cycle genes are found in human cells as well, and the encoded proteins have similar amino acid sequences. Proteins from different organisms, but with similar amino acid sequences, are said to be homologous; such proteins may have the same or similar functions. Remarkably, it has been shown that a human cell cycle protein, when expressed in a mutant yeast defective in the homologous yeast protein, is able to "rescue the defect" of the mutant yeast (that is, to allow the cell to grow normally), thus demonstrating the protein's ability to function in a very different type of eukaryotic cell. This experimental result, which garnered a Nobel Prize for Paul Nurse, was especially notable because the common ancestor of present-day yeasts, plants, and humans is thought to have lived over a billion years ago. Clearly the eukaryotic cell cycle and many of the genes and proteins that catalyze and regulate it evolved early in biological evolution and have remained quite constant over a very long period of evolutionary time. Subsequent studies showed that mutations in many yeast cell cycle proteins that allow uncontrolled cell growth also frequently occur in human cancers (see Chapter 24), again attesting to the important conserved functions of these proteins in all eukaryotes.

## Studies in the Alga Chlamydomonas reinhardtii Led to the Development of a Powerful Technique to Study Brain Function

The green unicellular alga Chlamydomonas reinhardtii (Figure 1-22b), which swims using its two long flagella, is widely used in studies of the structure, function, and assembly of this organelle. In part because of the powerful genetic techniques now available, Chlamydomonas is also used in studies of chloroplast formation and photosynthesis. The Chlamydomonas genome (see Table 1-2) encodes many more proteins than do those of yeasts, including flagellar proteins and proteins needed to build a chloroplast, organelles not found in yeasts.

One important outcome of the use of this experimental organism came from studies of phototaxis, the behavior in which an organism moves toward or away from a source of light. Chlamydomonas needs to move toward light to undergo photosynthesis and thus generate the energy it needs to grow and divide, but light that is too intense repels it, as it causes damage to the chloroplast. Studies of Chlamydomonas phototaxis led to the discovery of two proteins in its plasma membrane that, when they absorb light, open a "channel" in the membrane that allows ions such as $\mathrm{Ca}^{2+}$ to flow from the extracellular medium into the cytosol, triggering phototactic responses. As detailed in Chapter 22, recombinant DNA techniques have been used to express one such protein in specific neurons in the mouse brain, allowing investigators to activate just one or a few cells in the brain using a point source of light. Thus studies on this humble alga have led to the development of an important experimental system—optogenetics—for the study of brain function.

## The Parasite That Causes Malaria Has Novel Organelles That Allow It to Undergo a Remarkable Life Cycle

Whereas yeasts are used in the manufacture of bread, beer, wine, and cheese, some unicellular eukaryotes cause major human diseases and are widely studied in an attempt to develop drugs that will kill them but not injure their human host. Entamoeba histolytica causes dysentery; Trichomonas vaginalis, vaginitis; and Trypanosoma brucei, sleeping sickness. Each year the worst of these protozoans, Plasmodium falciparum and related species, cause more than 300 million new cases of malaria, a disease that kills 1.5 million to 3 million people annually. These protozoans inhabit mammals and mosquitoes alternately, changing their morphology and behavior in response to signals in each of these environments.

The complex life cycle of Plasmodium dramatically illustrates how a single cell can adapt to multiple challenges (Figure 1-24a). Additionally, the merozoite form that infects human red blood cells contains several organelles, not found in most eukaryotes, that enable the parasite to invade a red blood cell, including the rhoptry, polar ring, and microneme, as well as a fuzzy surface coat on the outside of the cell (Figure 1-24b, c). Entry of the parasite into a red blood cell is initiated by binding of certain parasite cell-surface proteins to proteins on the red blood cell surface, followed by the formation of a tight junction between the two plasma membranes, the loss of the "fuzzy coat," and secretion of proteins stored in the microneme and rhoptry.

All the transformations in cell type that occur during the Plasmodium life cycle are governed by instructions encoded in the genetic material of this parasite (see Table 1-2). The Plasmodium genome has about the same number of protein-coding genes as the yeast Saccharomyces cerevisiae, but about two-thirds of the Plasmodium genes appear to be unique to this and related parasites, attesting to the great evolutionary distance between these parasites, the

![li39857775e0](li39857775e0.jpg)

FIGURE 1-24 Plasmodium species, the parasites that cause malaria, are single-celled protozoans with a remarkable life cycle. Many Plasmodium species are known, and they can infect a variety of animals, cycling between insect and vertebrate hosts. The four species that cause malaria in humans undergo several dramatic transformations within their human and mosquito hosts. (a) Diagram of the life cycle. Step $\mathbf{\square}$ Sporozoites enter a human host when an infected Anopheles mosquito bites a person. Step $\mathbf{\square}$ They migrate to the liver, where they develop into merozoites, which are released into the blood. Merozoites differ substantially from sporozoites, so this transformation is a metamorphosis (Greek, "to transform" or "many shapes"). Step $\mathbf{\square}$ Circulating merozoites invade red blood cells (RBCs) and reproduce within them. Proteins produced by some Plasmodium species move to the surface of infected RBCs, causing the cells to adhere to the walls of blood vessels. This prevents infected RBCs from circulating to the spleen, where cells of the immune system would destroy the RBCs and the Plasmodium organisms they harbor. Step $\mathbf{\square}$ After growing and reproducing in RBCs for a period of time characteristic of each Plasmodium species, the merozoites suddenly burst forth in synchrony from large numbers of infected cells. It is this event that brings on the fevers and shaking chills that are the well-known symptoms of malaria. Some of the released merozoites infect additional RBCs, creating a cycle of production and infection. Step $\mathbf{\square}$ Eventually, some merozoites undergo meiosis and develop into male and female gametocytes, another metamorphosis. These cells, which contain half the usual number of
![li40857775e0](li40857775e0.jpg)
chromosomes, cannot survive for long unless they are transferred in blood to an Anopheles mosquito. Step $\mathbf{\square}$ In the mosquito's stomach, the gametocytes are transformed into sperm or eggs (gametes), yet another metamorphosis marked by development of long hairlike flagella on the sperm. Step $\mathbf{\square}$ Fusion of sperm and eggs generates zygotes, which implant into the cells of the stomach wall and grow into oocysts, essentially factories for producing sporozoites. Step $\mathbf{\square}$ Rupture of an oocyst releases thousands of sporozoites, which migrate to the salivary glands, setting the stage for infection of another human host. (b) Organelles of the Plasmodium vivax merozoite. Some of these organelles are found only in Plasmodium and related eukaryotic parasitic microorganisms. (c) Section of a Plasmodium vivax merozoite invading a human red blood cell. See A. Cowman and B. Crabb, 2006, Cell 124:755-766. [Part (c) Masamichi Aikawa.]

Apicomplexa (see Figure 1-1), and most other eukaryotes as well as the presence of unusual organelles required for their complex life cycles.

### 1.5 Metazoan Structure, Differentiation, and Model Organisms

The evolution of multicellular organisms most likely began when cells remained associated in small colonies after division instead of separating into individual cells. A few prokaryotes and several unicellular eukaryotes, such as Volvox (see Figure 1-3d), as well as many fungi and slime molds, exhibit such rudimentary social behavior. The full flowering of multicellularity, however, occurred in eukaryotic organisms whose cells became differentiated and organized into groups, or tissues, in which the different cells performed specialized functions.

## Multicellularity Requires Cell-Cell and Cell-Matrix Adhesions

The cells of higher plants are encased in a network of chambers formed by the interlocking cell walls surrounding the cells and are connected by cytoplasmic bridges called plasmodesmata (see Figure 1-12a). Animal cells are often "glued" together into a chain, a ball, or a sheet by celladhesion proteins on their surfaces, often called cell-adhesion molecules, or CAMs (see Figure 1-4d). Some CAMs bind cells to one another; other types bind cells to the extracellular matrix, forming a cohesive unit. In animals, the matrix cushions cells and allows nutrients to diffuse toward them and waste products to diffuse away. A specialized, especially tough matrix called the basal lamina, made up of polysaccharides and multiple proteins such as collagen, forms a supporting layer underlying cell sheets and prevents the cell aggregates from ripping apart (see Figure 1-4). Many CAMs and extracellular-matrix proteins found in humans also occur in invertebrates, indicating their importance during metazoan evolution. Similarly, many of the proteins and small molecules used by metazoans as signaling molecules are conserved in humans and many invertebrates, as are their receptors, the cellular proteins that bind to these signaling molecules and trigger an effect in the receiving cell. As one example, the signaling protein Wnt, discussed in Chapter 16, was discovered simultaneously as the gene mutated in the Drosophila Wingless mutation and as the site of integration of a cancer-causing virus in mice.

## Epithelia Originated Early in Evolution

Metazoans, which are thought to have evolved in an oceanlike, saline environment, had to solve a fundamental problem: separating the inside of the organism from the outside. The external surfaces of all metazoan animals, as well as the surfaces of their internal organs, are covered by a sheet-like layer of tissue called an epithelium. Epithelia commonly serve
as barriers and protective surfaces, as exemplified by the sheets of epidermal cells that form the skin (see Figure 1-4). Other epithelia are one cell layer thick and line internal organs such as the small intestine, where they are crucial for transport of the products of digestion (e.g., glucose and amino acids) into the blood (see Chapter 11). As discussed in Chapter 20, epithelia in different body locations have characteristic morphologies and functions.

Cells that form epithelial tissues are said to be polarized because their plasma membranes are organized into at least two discrete regions. Typically, the distinct surfaces of a polarized epithelial cell are the apical surface-the "top" of the cell facing the external world-and the basal and lateral (collectively, basolateral) surfaces that face the organism's interior. As shown in Figure 1-4, the basal surface usually contacts an underlying extracellular matrix, the basal lamina. Specialized junction proteins in the basolateral plasma membrane link adjacent cells together and also bind the cells to the basal lamina.

## Tissues Are Organized into Organs

Cells in metazoans do not work in isolation; specialized groups of differentiated cells often form tissues, which are themselves the major components of organs. For example, the lumen of a small blood vessel is lined with a sheet-like layer of endothelial cells, or endothelium, which prevents blood cells from leaking out (Figure 1-25). A layer of smooth muscle tissue encircles the endothelium and basal lamina and contracts to limit blood flow. During times of fright, constriction of smaller peripheral vessels forces more blood to the vital organs. The muscle layer of a blood vessel is wrapped in an outer layer of connective tissue, a network of fibers and cells that encases the vessel walls and protects them from stretching and rupture.

This hierarchy of tissues is copied in other blood vessels, which differ mainly in the thickness of the layers. The wall of a major artery must withstand much stress and is therefore thicker than that of a minor vessel. The strategy of grouping and layering different tissues is used to build other complex organs as well. In each case, the function of the organ is determined by the specific functions of its component tissues, and each type of cell in a tissue produces the specific groups of proteins that enable the tissue to carry out its functions.

## Genomics Has Revealed Important Aspects of Metazoan Evolution and Cell Function

Metazoans-be they invertebrates such as the fruit fly Drosophila melanogaster and the roundworm Caenorhabditis elegans, or vertebrates such as mice and humans-contain between 13,000 and 23,000 protein-coding genes, about three to four times as many as a yeast (see Table 1-2). Sequencing of entire genomes has shown that many of these genes are conserved among the metazoans, and genetic

![li41857775e0](li41857775e0.jpg)

FIGURE 1-25 All organs are organized arrangements of various tissues, as illustrated in this cross section of a small artery (arteriole). Blood flows through the vessel lumen, which is lined by a thin sheet of endothelial cells forming the endothelium and by the underlying basal lamina. This tissue adheres to the overlying layer of smooth muscle tissue; contraction of the muscle layer controls blood flow through the vessel. A fibrillar layer of connective tissue surrounds the vessel and connects it to other tissues. (SPL/Science Source.)
studies have shown that many of them are essential for the formation and function of specific tissues and organs. Thus many of the organisms listed in Table 1-2 are used to study the roles of these conserved proteins in cell development and function.

While the human and mouse genomes encode about the same number of proteins as those of the roundworm Caenorhabditis elegans, frogs, and fish, mammalian cells contain about 30 times the DNA of a roundworm and two to three times the DNA of frogs and fish. Only about 10 percent of human DNA encodes proteins. We know now that much of the remaining 90 percent has important functions. Many DNA segments bind proteins that regulate expression of nearby genes, allowing each mammalian gene to make the precise amount of mRNA and protein needed in each of many types of cells.

Other segments of DNA are used to synthesize thousands of RNA molecules whose function in regulating gene expression is only now being uncovered. As an example, hundreds of different micro-RNAs, 20 to 25 nucleotides long, are abundant in metazoan cells, where they bind to and repress the activity of target mRNAs. These small RNAs may indirectly regulate the activity of most or all genes, either by inhibiting the ability of mRNAs to be translated into proteins or by triggering the degradation of target mRNAs (see Chapter 10).

Some of this non-protein-coding DNA probably regulates expression of genes that make us uniquely human. Indeed, fish and humans have about the same number of protein-coding genes-about 20,000-yet as noted above, the human genome is over twice the size of that in fish (see Table 1-2). The human brain can perform complex mental processes such as reading and writing a textbook. Somehow these 20,000 human genes are exquisitely regulated such that humans produce a brain with about 100,000,000,000 neurons, which communicate with one another at about 100,000,000,000,000 interaction sites termed synapses.

Genomics-the study of the entire DNA sequences of or-ganisms-has shown us how close humans really are to our nearest relatives, the great apes (Figure 1-26). Human DNA is 99 percent identical in sequence to that of chimpanzees and bonobos; the 1 percent difference is about 3,000,000 base pairs, but it somehow explains the obvious differences between our species, such as the evolution of human brains during the past $5,000,000$ years since we last shared a common ancestor.

Genomics coupled with paleontological findings indicates that humans and mice descended from a common mammalian ancestor that probably lived about 75 million years ago. Nonetheless, both organisms contain about the same number of genes, and about 99 percent of mouse protein-coding genes have homologs in humans, and vice versa. Over 90 percent of mouse and human genomes can be partitioned into regions of synteny-that is, DNA segments that have the same order of unique DNA sequences and genes along a segment of a chromosome. This observation suggests that much of the gene order in the most recent common ancestor of humans and mice has been conserved in both species (Figure 1-27). Of course, mice are not people; relative to humans, mice have expanded families of genes related to immunity, reproduction, and olfaction, probably reflecting the differences between the human and mouse lifestyles.

It's not only human evolution that interests us! Polar bears live in the Arctic and eat a high-fat diet, mostly composed of seals. Recent genome sequencing allowed researchers to conclude that the most recent common ancestor of polar bears and their brown bear relatives, which live in temperate climates, was present about 500,000 years-or only about 20,000 bear generations-ago. But during that rather short evolutionary period the polar bear genome acquired changes in many genes regulating cardiovascular function, fat metabolism, and heart development, allowing it to consume a diet very rich in fats.

## Embryonic Development Uses a Conserved Set of Master Transcription Factors

The astute reader will note a paradox in the previous discussion: if indeed most human protein-coding genes are shared with apes and mice, and many with flies and worms, how is it that these organisms look and function so differently?

![li42857775e0](li42857775e0.jpg)

FIGURE 1-26 Evolutionary tree connecting monkeys, apes, and humans. The evolutionary tree of humans, great apes, a small ape, and an Old World monkey was estimated from the divergence among their genomic DNA sequences. Whole-genome DNA sequences were aligned, and the average nucleotide divergence in unique DNA
sequences was estimated. Estimates of the times the different species diverged from each other, indicated at each node, were calculated in millions of years (Myr) based on DNA sequence identity; $\sim 1$ Myr implies approximately 1 Myr or less. [Data from D. P. Locke et al., 2011, Nature 469:529-533.]

The answer to this question resides in the way genes are regulated during the development of all metazoans from a single cell, the fertilized egg. As we learn in Chapters 8 and 9 , each protein-coding gene is associated with regulatory DNA sequences that differ in different organisms. Many of these regulatory sequences bind proteins that direct the expression of the gene, and thus the amount of a protein it makes, in specific types of cells. Some of these proteins are termed master transcription factors; these proteins bind to regulatory DNA sequences, are conserved throughout evolution, and control the development of specific types of cells by activating or repressing groups of genes, often at different stages of development.

The early stages in the development of a human embryo are similar to those in the mouse. They are characterized by rapid cell divisions (Figure 1-28) followed by the
![li43857775e0](li43857775e0.jpg)

FIGURE 1-27 Conservation of synteny between human and mouse. Shown is a 510,000-base-pair (bp) segment of mouse chromosome 12 that shares common ancestry with a 600,000-bp section of human chromosome 14. Pink lines connect the reciprocal unique DNA sequences in the two genomes. Mb, 1 million base pairs. [Data from Mouse Genome Sequencing Consortium, 2002, Nature 420:S20.]
differentiation of cells into tissues. In all organisms, the embryonic body plan-the spatial pattern of cell types (tissues) and body parts-emerges from two influences: a program of genes that specifies the pattern of the body, and local cell interactions that induce different parts of the program.

With only a few exceptions, animals display axial symmetry; that is, their left and right sides mirror each other. This most basic of patterns is encoded in the genome. Developmental biologists have divided bilaterally symmetric animal phyla into two large groups depending on where the mouth and anus form in the early embryo. Protostomes develop a mouth close to a transient opening in the early embryo (the blastopore) and have a ventral nerve cord; protostomes include all worms, insects, and mollusks. Deuterostomes develop an anus close to this transient opening in the embryo and have a dorsal central nervous system; they include echinoderms (such as sea stars and sea urchins) and vertebrates. The bodies of both protostomes and deuterostomes are divided into discrete segments that form early in embryonic development. Protostomes and deuterostomes probably evolved from a common ancestor, termed Urbilateria, that lived approximately 600 million years ago (Figure 1-29a).

Many patterning genes encode master transcription factors that control expression of other genes and specify the general organization of an organism, beginning with the major body axes-anterior-posterior (head-to-tail), dorsalventral (back-to-belly), and left-right-and ending with body segments such as the head, chest, abdomen, and tail. The conservation of axial symmetry from the simplest worms to mammals is explained by the presence of conserved

![li44857775e0](li44857775e0.jpg)
(a)
![li45857775e0](li45857775e0.jpg)
(b)
![li46857775e0](li46857775e0.jpg)

Fly
(protostome)
![li47857775e0](li47857775e0.jpg)
(c)
![li48857775e0](li48857775e0.jpg)
(c)
![li49857775e0](li49857775e0.jpg)

Deurte
![li50857775e0](li50857775e0.jpg)

FIGURE 1-29 Similar master transcription factors, conserved during evolution, regulate early developmental processes in diverse animals. (a) Urbilateria is the presumed ancestor of all protostomes and deuterostomes that existed about 600 million years ago. The positions of its nerve cord (violet), surface ectoderm (mainly skin; white), and endoderm (mainly digestive tract and organs; light green) are shown. (b) Highly conserved master transcription factors called Hox proteins, which determine the identity of body segments during embryonic development, are found in both protostomes and deuterostomes. Hox genes are found in clusters on the chromosomes of most or all animals, and they encode related master transcription factors that control the activities of other genes. In many animals, different Hox genes direct the development of different segments along the head-to-tail axis, as indicated by corresponding colors. Each gene is activated (transcriptionally) in a specific region along the head-to-tail axis and controls the growth and development of tissues there. For example, in the mouse, a deuterostome, the Hox genes are responsible for the distinctive shapes of vertebrae. Mutations affecting Hox genes in the fruit fly, a protostome, cause body parts to form in the wrong locations, such as legs in lieu of antennae on the head. In both organisms, these genes provide a head-to-tail "address" and serve to direct the formation of structures in the appropriate places.
patterning genes in their genomes. Other patterning genes encode proteins that are important in cell adhesion or in cell signaling. This broad repertoire of patterning genes permits the integration and coordination of events in different parts of the developing embryo and gives each segment in the body its unique identity.

Remarkably, many patterning genes encoding master transcription factors are highly conserved in both protostomes and deuterostomes (Figure 1-29b). This conservation of body plan reflects evolutionary pressure to preserve the commonalities in the molecular and cellular mechanisms controlling development in different organisms. For instance, fly eyes and human eyes are very different in their structure, function, and nerve connections. Nonetheless, the master transcription factors that initiate eye developmenteyeless in the fly and Pax6 in the human-are highly related proteins that regulate the activities of other genes and are descended from the same ancestral gene. Mutations in the eyeless or Pax6 genes cause major defects in eye formation (Figure 1-30).

## Planaria Are Used to Study Stem Cells and Tissue Regeneration

In single-celled organisms, both daughter cells usually (though not always) resemble the parent cell. Similarly, in multicellular organisms, when many types of cells divide, the daughter cells look a lot like the parent cell-liver cells, for instance, divide to generate liver cells with the same characteristics and functions as their parent, as do insulinproducing cells in the pancreas. In contrast, stem cells and certain other undifferentiated cells can generate multiple types of differentiated descendant cells; these cells often divide in such a way that the two daughter cells are different. Such asymmetric cell division is characteristic of stem cells and is critical to the generation of different cell types in the body (see Chapter 21). Often one daughter cell resembles its parent in that it remains undifferentiated and retains its ability to give rise to multiple types of differentiated cells. The other daughter cell divides many times, and each of its daughter cells differentiates into a specific type of cell.

The planarian Schmidtea mediterranea is best known for its capacity to regenerate complete individuals-with a normal head-from minuscule body parts formed by dissection (see Figure 1-22e). Planaria contain stem cells that

![li51857775e0](li51857775e0.jpg)

FIGURE 1-30 Homologous genes regulate eye development in diverse animals. (a) Development of the large compound eyes in fruit flies requires a gene called eyeless (named for the mutant phenotype). (b) Flies with inactivated eyeless genes lack eyes. (c) Normal human eyes require the gene Pax6, the homolog of eyeless. (d) People lacking adequate Pax6 function have the genetic disease aniridia, a lack of irises in the eyes. Pax6 and eyeless, which encode highly related master transcription factors that regulate the activities of other genes, are homologs and presumably descended from the same ancestral gene. [Parts (a) and (b) Courtesy Andreas Hefti, Interdepartmental Electron Microscopy (IEM), Biocenter of the University of Basel. Part (c) © Simon Fraser/Science Source. Part (d) © Mediscan/Alamy.]
replace cells lost to normal turnover. In portions of a dissected animal, they will, after several cell divisions, generate any cell type needed during regeneration. These stem cells have served as a potent experimental system to discover how heads and tails, each built of many types of cells, are formed (see Chapters 16 and 21). The hormones that instruct stem cells in different parts of the body to generate specific types of cells are similar to those used in mammals, including humans, in development (see Chapter 16), and thus future studies on planarian regeneration may inform scientists how to regenerate human body parts such as a hand or an eye.

## Invertebrates, Fish, Mice, and Other Organisms Serve as Experimental Systems for Study of Human Development and Disease

Organisms with large-celled embryos that develop outside the mother's body (e.g., frogs, sea urchins, fish, and chickens) are extremely useful for tracing the fates of cells as they form different tissues, as well as for making extracts for biochemical studies. For instance, a key protein in regulating
cell division in all eukaryotes, including humans, was first identified in studies with sea stars and sea urchin embryos and subsequently purified from extracts prepared from these embryos (see Chapter 19).

Studies of cells in specialized tissues make use of animal and plant model organisms. Neurons and muscle cells, for instance, were traditionally studied in mammals or in creatures with especially large or accessible cells, such as the giant neural cells of the squid and sea hare or cells in the flight muscles of birds. More recently, muscle and nerve development have been extensively studied in fruit flies (Drosophila melanogaster), roundworms (Caenorhabditis elegans), and zebrafish (Danio rerio), in which mutations in genes required for muscle and nerve formation or function can be readily isolated (see Figure 1-22).

Mice have one enormous advantage over other experimental organisms: they are the most closely related to humans of any animal for which powerful genetic approaches have been available for many years. Mice and humans have shared living structures for millennia, have similar nervous systems, have similar immune systems, and are subject to infection by many of the same pathogens. As noted, both organisms contain about the same number of genes, and about 99 percent of mouse protein-coding genes have homologs in the human genome, and vice versa.

Using recombinant DNA techniques developed in the past few years, researchers can inactivate any desired gene, and thus abolish production of its encoded protein. Such specific mutations can be introduced into the genomes of worms, flies, frogs, sea urchins, chickens, mice, a variety of plants, and other organisms, permitting the effects of these mutations to be assessed. Using the Cas9 experimental system described in Chapter 6, this approach is being used extensively to produce animal versions of human genetic diseases, in mice as well as in other animals. As an example, people with autism spectrum disorder often have mutations in specific protein-coding genes. To understand the role of these mutations, these genes have been inactivated in mice; in many cases, the mice exhibit symptoms of the human disease, including repetitive actions such as excessive grooming, strongly suggesting that the human mutation indeed has a role in triggering the disorder. Within the past year, similar techniques have been used to produce monkeys in which the targeted gene has been inactivated. Such approaches can be useful in uncovering the role of specific genes in higherorder brain tasks such as learning and memory, or in studies of viruses that infect only humans and nonhuman primates. Once animal models of a human disease are available, further studies on the molecular defects causing the disease can be done and new treatments can be tested, thereby minimizing the testing of new drugs on humans.

## Genetic Diseases Elucidate Important Aspects of Cell Function

Many genetic diseases are caused by mutations in a single protein; studies on people with these diseases have shed light on the normal function of those proteins. As an example,

![li52857775e0](li52857775e0.jpg)

FIGURE 1-31 The dystrophin glycoprotein complex (DGC) in skeletal muscle cells. Dystrophin-the protein that is defective in Duchenne muscular dystrophy-links the actin cytoskeleton to the multiprotein sarcoglycan complex in the plasma membrane. Other proteins in the complex bind to components of the basal lamina, such as laminin, which in turn bind to the collagen fibers that give the basal lamina strength and rigidity. Thus dystrophin is an important member of a group of proteins that links the muscle cell and its internal actin cytoskeleton with the surrounding basal lamina. See D. E. Michele and K. P. Campbell, 2003, J. Biol. Chem. 278:15457.
consider Duchenne muscular dystrophy (DMD), the most common among the hereditary muscle-wasting diseases, collectively called muscular dystrophies. DMD, an X chromo-some-linked disorder that affects 1 in 3300 boys, results in cardiac or respiratory failure and death, usually in the late teens or early twenties. The first clue to understanding the molecular basis of this disease came from the discovery that people with DMD carry mutations in the gene encoding a protein named dystrophin. As detailed in Chapter 17, this very large protein was later found to be a cytosolic adapter protein that binds to actin filaments that are part of the cytoskeleton (see Figure 1-13) and to a complex of muscle plasma-membrane proteins termed the sarcoglycan complex (Figure 1-31). The resulting large multiprotein assemblage, the dystrophin glycoprotein complex (DGC), links the extracellular matrix protein laminin to the cytoskeleton within muscle cells. Mutations in dystrophin, other DGC components, or laminin can disrupt the DGC-mediated link between the exterior and interior of muscle cells and cause muscle weakness and eventual death. The first step in identifying the entire dystrophin glycoprotein complex involved cloning the dystrophin-encoding gene using DNA from normal individuals and from patients with Duchenne muscular dystrophy.

## The Following Chapters Present Much Experimental Data That Explains How We Know What We Know About Cell Structure and Function

In subsequent chapters of this book, we discuss cellular processes in much greater detail. We begin (in Chapter 2) with a discussion of the chemical nature of the building blocks
of cells and the basic chemical processes required to understand the macromolecular processes discussed in subsequent chapters. We go on to discuss the structure and function of proteins (in Chapter 3). Chapter 4 discusses many of the techniques biologists use to culture and fractionate cells and to visualize specific proteins and structures within cells. Chapter 5 describes how DNA is replicated, how segments of DNA are copied into RNA, and how proteins are synthesized on ribosomes. Chapter 6 describes many of the techniques used to study genes, gene expression, and protein function, including the generation of animals with specific genetic mutations. Biomembrane structure is the topic of Chapter 7. Gene and chromosome structure and the regulation of gene expression are covered in Chapters 8, 9, and 10. The transport of ions and small molecules across membranes is covered in Chapter 11, and Chapter 12 discusses cellular energetics and the functions of mitochondria and chloroplasts. Membrane biogenesis, protein secretion, and protein trafficking-the directing of proteins to their correct subcellular destinations-are the topics of Chapters 13 and 14. Chapters 15 and 16 discuss the many types of signals and signal receptors used by cells to communicate and regulate their activities. The cytoskeleton and cell movements are discussed in Chapters 17 and 18. Chapter 19 discusses the cell cycle and how cell division is regulated. The interactions among cells, and between cells and the extracellular matrix, that enable formation of tissues and organs are detailed in Chapter 20. Later chapters of the book discuss important types of specialized cells—stem cells (Chapter 21), neurons (Chapter 22), and cells of the immune system (Chapter 23). Chapter 24 discusses cancer and the multiple ways in which cell growth and differentiation can be altered by mutations.

this page left intentionally blank

![li53857775e0](li53857775e0.jpg)
"Star of David" catenane. Two triply entwined rings composed of carbon, hydrogen, and nitrogen are linked together with bridging iron atoms via a complex chemical synthetic pathway to cross each other six times and form a hexagram (six-pointed star). The chemical structure is indicated on the left, where the two independent rings are colored blue and orange. On the right is the three-dimensional structure determined by x-ray crystallography with the carbon atoms of one ring in blue and the other light gray; irons are pink and nitrogens purple. In the center is a noncovalently bound, negatively charged phosphorus hexafluoride (cyan and green). See D. A. Leigh, R. G. Pritchard, and A. J. Stephens, 2014, Nature Chem. 6:978-982.

The life of a cell depends on thousands of chemical interactions and reactions exquisitely coordinated with one another in time and space, influenced by the cell's genetic instructions and its environment. By understanding these interactions and reactions at a molecular level, we can begin to answer fundamental questions about cellular life: How does a cell extract nutrients and information from its environment? How does a cell convert the energy stored in nutrients into the work of movement or metabolism? How does a cell transform nutrients into the cellular components required for its survival? How does a cell link itself to other cells to form a tissue? How do cells communicate with one another so that a complex, efficiently functioning organism can develop

## OUTLINE

### 2.1 Covalent Bonds and Noncovalent Interactions

2.2 Chemical Building Blocks of Cells

### 2.3 Chemical Reactions and Chemical Equilibrium

2.4 Biochemical Energetics

![li54857775e0](li54857775e0.jpg)

Protein B
(c) Chemical equilibrium
![li55857775e0](li55857775e0.jpg)

FIGURE 2-1 Chemistry of life: four key concepts. (a) Molecular complementarity lies at the heart of all biomolecular interactions (see Section 2.1), as when two proteins with complementary shapes and chemical properties come together to form a tightly bound complex. (b) Small molecules serve as building blocks for larger structures (see Section 2.2). For example, to generate the information-carrying macromolecule DNA, four small nucleotide building blocks are covalently linked into long strings (polymers), which then wrap around each other to form the double helix. (c) Chemical reactions are reversible, and the distribution of the chemicals
larger macromolecules and macromolecular assemblies that make up a cell's machinery and architecture and thus the remaining mass of organisms. These small molecules include amino acids (the building blocks of proteins), nucleotides (the building blocks of DNA and RNA), lipids (the building blocks of biomembranes), and sugars (the building blocks of complex carbohydrates).

Many of the cell's biomolecules (such as sugars) readily dissolve in water; these molecules are referred to as hydrophilic ("water liking"). Others (such as cholesterol) are oily, fatlike substances that shun water; these molecules are said to be hydrophobic ("water fearing"). Still other biomolecules (such as phospholipids) contain both hydrophilic and hydrophobic regions; these molecules are said to be amphipathic or amphiphilic ("both liking"). The smooth functioning of cells, tissues, and organisms depends on all these molecules, from the smallest to the largest. Indeed, the chemistry of the simple proton $\left(\mathrm{H}^{+}\right)$can be as important to the survival of a human cell as that of each gigantic DNA molecule (the
(b) Chemical building blocks
![li56857775e0](li56857775e0.jpg)

Macromolecule
(d) Chemical bond energy
![li57857775e0](li57857775e0.jpg)
between starting reactants (left) and the products of the reactions (right) depends on the rate constants of the forward ( $k_{\mathrm{f}}$, upper arrow) and reverse ( $k_{\mathrm{r}}$, lower arrow) reactions. The ratio of these, $K_{\text {exp }}$ provides an informative measure of the relative amounts of products and reactants that will be present at equilibrium (see Section 2.3). (d) In many cases, the source of energy for chemical reactions in cells is the hydrolysis of the molecule ATP (see Section 2.4). This energy is released when a high-energy phosphoanhydride bond linking the b and g phosphates in the ATP molecule (red) is broken by the addition of a water molecule, forming ADP and $\mathrm{P}_{\mathrm{i}}$.
mass of the DNA molecule in human chromosome 1 is $8.6 \times 10^{10}$ times that of a proton!). The chemical interactions of all these molecules, large and small, with water and with one another define the nature of life.

Luckily, although many types of biomolecules interact and react in numerous and complex pathways to form functional cells and organisms, a relatively small number of chemical principles are necessary to understand cellular processes at the molecular level (Figure 2-1). In this chapter, we review these key principles, some of which you already know well. We begin with the covalent bonds that connect atoms into molecules and the noncovalent interactions that stabilize groups of atoms within and between molecules. We then consider the basic chemical building blocks of macromolecules and macromolecular assemblies. After reviewing those aspects of chemical equilibrium that are most relevant to biological systems, we end the chapter with the basic principles of biochemical energetics, including the central role of ATP (adenosine triphosphate) in capturing and transferring energy in cellular metabolism.

### 2.1 Covalent Bonds and Noncovalent Interactions

Strong and weak attractive forces between atoms are the "glue" that holds individual molecules together and permits interactions between different molecules. When two atoms share a single pair of electrons, the result is a covalent bond-a type of strong force that holds atoms together in molecules. Sharing of multiple pairs of electrons results in multiple covalent bonds (e.g., "double" or "triple" bonds). The weak attractive forces of noncovalent interactions are equally important in determining the properties and functions of biomolecules such as proteins, nucleic acids, carbohydrates, and lipids. In this section, we first review covalent bonds and then discuss the four major types of noncovalent interactions: ionic bonds, hydrogen bonds, van der Waals interactions, and the hydrophobic effect.

## The Electronic Structure of an Atom Determines the Number and Geometry of the Covalent Bonds It Can Make

Hydrogen, oxygen, carbon, nitrogen, phosphorus, and sulfur are the most abundant elements in biological molecules. These atoms, which rarely exist as isolated entities, readily form covalent bonds, using electrons in the outermost electron orbitals surrounding their nuclei (Figure 2-2). As a rule, each type of atom forms a characteristic number of covalent bonds with other atoms. These bonds have a well-defined geometry determined by the atom's size and by both the distribution of electrons around the nucleus and the number of electrons that it can share. In some cases, the number of stable covalent bonds an atom can make is fixed; carbon, for example, always forms four covalent bonds. In other cases, different numbers of stable covalent bonds are possible; for example, sulfur can form two, four, or six stable covalent bonds.

All the biological building blocks are organized around the carbon atom, which forms four covalent bonds. In these
![li58857775e0](li58857775e0.jpg)

FIGURE 2-2 Covalent bonds form by the sharing of electrons. Covalent bonds, the strong forces that hold atoms together in molecules, form when atoms share electrons from their outermost electron orbitals. Each atom forms a defined number and geometry of covalent bonds.
organic biomolecules, each carbon usually bonds to three or four other atoms. [Carbon can also bond to two other atoms, as in the linear molecule carbon dioxide, $\mathrm{CO}_{2}$, which has two carbon-oxygen double bonds $(\mathrm{O}=\mathrm{C}=\mathrm{O})$; however, such bond arrangements of carbon are not found in biological building blocks.] As illustrated in Figure 2-3a for formaldehyde, carbon can bond to three atoms, all in a common plane. The carbon atom forms two single bonds with two atoms and one double bond (two shared electron pairs) with the third atom. In the absence of other constraints, atoms joined by a single bond generally can rotate freely about the bond axis, whereas those connected by a double bond cannot. The rigid planarity imposed by double bonds has enormous significance for the shapes and flexibility of biomolecules such as phospholipids, proteins, and nucleic acids.

Carbon can also bond to four rather than three atoms. As illustrated by methane $\left(\mathrm{CH}_{4}\right)$, when carbon is bonded to four other atoms, the angle between any two bonds is $109.5^{\circ}$, and the positions of bonded atoms define the four points of a tetrahedron (Figure 2-3b). This geometry defines the structures of many biomolecules. A carbon (or any other) atom bonded to four dissimilar atoms or groups in a nonplanar
(a) Formaldehyde
![li59857775e0](li59857775e0.jpg)
(b) Methane
![li60857775e0](li60857775e0.jpg)

FIGURE 2-3 Geometry of bonds when carbon is covalently linked to three or four other atoms. (a) A carbon atom can be bonded to three atoms, as in formaldehyde $\left(\mathrm{CH}_{2} \mathrm{O}\right)$. In this case, the carbon-bonding electrons participate in two single bonds and one double bond, which all lie in the same plane. Unlike atoms connected by a single bond, which usually can rotate freely about the bond axis, those connected by a double bond cannot. (b) When a carbon atom forms four single bonds, as in methane $\left(\mathrm{CH}_{4}\right)$, the bonded atoms (all H in this case) are oriented in space in the form of a tetrahedron. The letter representations on the left clearly indicate the atomic composition of each molecule and its bonding pattern. The ball-and-stick models in the center illustrate the geometric arrangement of the atoms and bonds, but the diameters of the balls representing the atoms and their nonbonding electrons are unrealistically small compared with the bond lengths. The sizes of the electron clouds in the space-filling models on the right more accurately represent the structure in three dimensions.

![li61857775e0](li61857775e0.jpg)

FIGURE 2-4 Stereoisomers. Many molecules in cells contain at least one asymmetric carbon atom. The tetrahedral orientation of bonds formed by an asymmetric carbon atom can be arranged in three-dimensional space in two different ways, producing molecules that are mirror images, or stereoisomers, of each other. Shown here is the common structure of an amino acid, with its central asymmetric carbon and four attached groups, including the R group, discussed in Section 2.2. Amino acids can exist in two mirror-image forms, designated $L$ and $O$. Although the chemical properties of such stereoisomers are identical, their biological activities are distinct. Only L amino acids are found in proteins.
configuration is said to be asymmetric. The tetrahedral orientation of bonds formed by an asymmetric carbon atom can be arranged in three-dimensional space in two different ways, producing molecules that are mirror images of each other, a property called chirality ("handedness," from the Greek word cheir, meaning "hand") (Figure 2-4). Such molecules are called optical isomers, or stereoisomers. Many molecules in cells contain at least one asymmetric carbon atom, often called a chiral carbon atom. The different stereoisomers of a molecule usually have completely different biological activities because the arrangement of atoms within their structures, and thus their ability to interact with other molecules, differs.

5Some drugs are mixtures of the stereoisomers of small molecules in which only one stereoisomer has the biological activity of interest. The use of a pure single stereoisomer of the chemical in place of the mixture may result in a more potent drug with reduced side effects. For example, one stereoisomer of the antidepressant drug citalopram (Celexa) is 170 times more potent than the other. Some stereoisomers have very different activities. Darvon is a pain reliever, whereas its stereoisomer, Novrad (Darvon spelled backward), is a cough suppressant. One stereoisomer of ketamine is an anesthetic, whereas the other causes hallucinations.

The typical numbers of covalent bonds formed by other atoms common in biomolecules are shown in Table 2-1. A hydrogen atom forms only one covalent bond. An atom of oxygen usually forms only two covalent bonds but has two additional pairs of electrons that can participate in

|  |  |  |
| :-- | :-- | :-- |
| A |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |

noncovalent interactions. Sulfur forms two covalent bonds in hydrogen sulfide $\left(\mathrm{H}_{2} \mathrm{~S}\right)$ but can accommodate six covalent bonds, as in sulfuric acid $\left(\mathrm{H}_{2} \mathrm{SO}_{4}\right)$ and its sulfate derivatives. Nitrogen and phosphorus each have five electrons to share. In ammonia $\left(\mathrm{NH}_{3}\right)$, the nitrogen atom forms three covalent bonds; the pair of electrons around the atom not involved in a covalent bond can take part in noncovalent interactions. In the ammonium ion $\left(\mathrm{NH}_{4}{ }^{+}\right)$, nitrogen forms four covalent bonds, which have a tetrahedral geometry. Phosphorus commonly forms five covalent bonds, as in phosphoric acid $\left(\mathrm{H}_{3} \mathrm{PO}_{4}\right)$ and its phosphate derivatives, which form the backbone of nucleic acids. Phosphate groups covalently attached to proteins play a key role in regulating the activity of many proteins, and the central molecule in cellular energetics, ATP, contains three phosphate groups (see Section 2.4). A summary of common covalent linkages and functional groups, which confer distinctive chemical properties on the molecules of which they are a part, is provided in Table 2-2.

## Electrons May Be Shared Equally or Unequally in Covalent Bonds

The extent of an atom's ability to attract an electron is called its electronegativity. In a bond between atoms with identical or similar electronegativities, the bonding electrons are essentially shared equally between the two atoms, as is the case for most carbon-carbon single bonds ( $\mathrm{C}-\mathrm{C}$ ) and carbon-hydrogen single bonds ( $\mathrm{C}-\mathrm{H}$ ). Such bonds are called nonpolar. In many molecules, however, the bonded atoms have different electronegativities, resulting in unequal sharing of electrons. The bond between them is said to be polar.

One end of a polar bond has a partial negative charge $\left(\delta^{-}\right)$, and the other end has a partial positive charge $\left(\delta^{+}\right)$. In an $\mathrm{O}-\mathrm{H}$ bond, for example, the greater electronegativity of the oxygen atom relative to the hydrogen atom results in the electrons spending more time around the oxygen atom than around the hydrogen. Thus the $\mathrm{O}-\mathrm{H}$ bond possesses an

# Functional Groups 

![li62857775e0](li62857775e0.jpg)
electric dipole, a positive charge separated from an equal but opposite negative charge. The amount of $\delta^{+}$charge on the oxygen atom of an $\mathrm{O}-\mathrm{H}$ dipole is approximately 25 percent that of an electron, and there is an equivalent and opposite $\delta^{+}$charge on the H atom. A common quantitative measure of the extent of charge separation, or strength, of a dipole is called the dipole moment, $\mu$, which for a chemical bond is the product of the partial charge on each atom and the distance between the two atoms. For a molecule with multiple dipoles, the amount of charge separation for the molecule as a whole depends in part on the dipole moments of all of its individual chemical bonds and in part on the geometry of the molecule (the relative orientations of the individual dipole moments).

Consider the example of water $\left(\mathrm{H}_{2} \mathrm{O}\right)$, which has two $\mathrm{O}-\mathrm{H}$ bonds and thus two individual bond dipole moments. If water were a linear molecule with the two bonds on exact opposite sides of the O atom, the two dipoles on each end of the molecule would be identical in strength but would be oriented in opposite directions. The two dipole moments would cancel each other, and the dipole moment of molecule as a whole would be zero. However, because water is a V-shaped molecule, with the individual dipoles of its two $\mathrm{O}-\mathrm{H}$ bonds both pointing toward the oxygen, one end of the water molecule (the end with the oxygen atom) has a partial negative charge and the other end (the one with the two hydrogen atoms) has a partial positive charge. As a consequence, the molecule as a whole is a dipole with a well-defined dipole moment (Figure 2-5). This dipole moment and the electronic properties of the oxygen and hydrogen atoms allow water to form electrostatic, noncovalent interactions with other
![li63857775e0](li63857775e0.jpg)

FIGURE 2-5 The dipole nature of a water molecule. The symbol $\delta$ represents a partial charge (a weaker charge than the one on an electron or a proton). Because of the difference in the electronegativities of H and O , each of the polar $\mathrm{H}-\mathrm{O}$ bonds in water is a dipole. The sizes and directions of the dipoles of each of the bonds determine the net distance and amount of charge separation, or dipole moment, of the molecule.
water molecules and with molecules of other types. These interactions play a critical role in almost every biochemical interaction in cells and organisms, as we will see shortly.

Another important example of polarity is the $\mathrm{O}=\mathrm{P}$ double bond in $\mathrm{H}_{3} \mathrm{PO}_{4}$. In the structure of $\mathrm{H}_{3} \mathrm{PO}_{4}$ shown on the left below, lines represent single and double bonds and nonbonding electrons are shown as pairs of dots (each dot represents one electron):
![li64857775e0](li64857775e0.jpg)

FIGURE 2-6 Relative energies of covalent bonds and noncovalent interactions. Bond energies are defined as the energy required to break a particular type of linkage. Shown here are the energies required to break a variety of linkages, arranged on a log scale. Covalent bonds, including single $(\mathrm{C}-\mathrm{C})$ and double $(\mathrm{C}=\mathrm{C})$ carbon-carbon bonds, are one to two powers of 10 stronger than noncovalent interactions. Noncovalent interactions have energies somewhat greater than the thermal energy of the environment at normal room temperature $\left(25^{\circ} \mathrm{C}\right)$. Many biological processes are coupled to the energy released during hydrolysis of a phosphoanhydride bond in ATP.
![li65857775e0](li65857775e0.jpg)

Because of the polarity of the $\mathrm{O}=\mathrm{P}$ double bond, $\mathrm{H}_{3} \mathrm{PO}_{4}$ can also be represented by the structure on the right, in which one of the electrons from the $\mathrm{P}=\mathrm{O}$ double bond has accumulated around the O atom, giving it a negative charge and leaving the P atom with a positive charge. These charges are important in noncovalent interactions. Neither of these two models precisely describes the electronic state of $\mathrm{H}_{3} \mathrm{PO}_{4}$. The actual structure can be considered to be an intermediate, or hybrid, between these two representations, as indicated by the double-headed arrow between them. Such intermediate structures are called resonance hybrids.

## Covalent Bonds Are Much Stronger and More Stable Than Noncovalent Interactions

Covalent bonds are considered to be strong because the energies required to break them are much greater than the thermal energy available at room temperature $\left(25^{\circ} \mathrm{C}\right)$ or body temperature $\left(37^{\circ} \mathrm{C}\right)$. As a consequence, they are stable at these temperatures. For example, the thermal energy available at $25^{\circ} \mathrm{C}$ is approximately 0.6 kilocalorie per mole $(\mathrm{kcal} / \mathrm{mol})$, whereas the energy required to break the $\mathrm{C}-\mathrm{C}$ bond in ethane is about 140 times larger (Figure 2-6). Consequently, at room temperature $\left(25^{\circ} \mathrm{C}\right)$, fewer than 1 in $10^{12}$ ethane molecules is broken into a pair of $\cdot \mathrm{CH}_{3}$ molecules, each containing an unpaired, nonbonding electron (called a radical).

Covalent single bonds in biological molecules have energies similar to the energy of the $\mathrm{C}-\mathrm{C}$ bond in ethane. Because more electrons are shared between atoms in double bonds, they require more energy to break than single bonds. For instance, it takes $84 \mathrm{kcal} / \mathrm{mol}$ to break a single $\mathrm{C}-\mathrm{O}$ bond but $170 \mathrm{kcal} / \mathrm{mol}$ to break a $\mathrm{C}=\mathrm{O}$ double bond. The most common double bonds in biological molecules are $\mathrm{C}=\mathrm{O}, \mathrm{C}=\mathrm{N}, \mathrm{C}=\mathrm{C}$, and $\mathrm{P}=\mathrm{O}$.

In contrast, the energy required to break noncovalent interactions is only $1-5 \mathrm{kcal} / \mathrm{mol}$, much less than the bond energies of covalent bonds (see Figure 2-6). Indeed, noncovalent interactions are weak enough that they are constantly being formed and broken at room temperature. Although these interactions are weak and have a transient existence
at physiological temperatures $\left(25-37^{\circ} \mathrm{C}\right)$, multiple noncovalent interactions can, as we will see, act together to produce highly stable and specific associations between different parts of a large molecule or between different macromolecules. Protein-protein and protein-nucleic acid interactions are good examples of such noncovalent interactions. Below, we review the four main types of noncovalent interactions and then consider their roles in the binding of biomolecules to one another and to other molecules.

## Ionic Interactions Are Attractions Between Oppositely Charged lons

Ionic interactions result from the attraction between a positively charged ion-a cation-and a negatively charged ion-an anion. In sodium chloride $(\mathrm{NaCl})$, for example, the bonding electron contributed by the sodium atom is completely transferred to the chlorine atom (Figure 2-7a). Unlike covalent bonds, ionic interactions do not have fixed or specific geometric orientations because the electrostatic field around an ion-its attraction for an opposite charge-is uniform in all directions. In solid NaCl , oppositely charged ions pack tightly together in an alternating pattern, forming the highly ordered crystalline array, or lattice, that is typical of salt crystals (Figure 2-7b). The energy required to break an ionic interaction depends on the distance between the ions and the electrical properties of the environment of the ions.

When solid salts dissolve in water, the ions separate from one another and are stabilized by their interactions with water molecules. In aqueous solutions, simple ions of biological significance, such as $\mathrm{Na}^{+}, \mathrm{K}^{+}, \mathrm{Ca}^{2+}, \mathrm{Mg}^{2+}$, and $\mathrm{Cl}^{-}$, are hydrated, surrounded by a stable shell of water molecules held in place by ionic interactions between the ion at the center and the oppositely charged ends of the water molecules, which are dipoles (Figure 2-7c). Most ionic compounds dissolve readily in water because the energy of hydration-the energy released when ions tightly bind water molecules and spread out in an aqueous solution-is greater than the lattice energy that stabilizes the crystal structure. Parts or all of the aqueous hydration shell must be removed from ions in solution when they interact directly with proteins. For example,

![li66857775e0](li66857775e0.jpg)

FIGURE 2-7 Electrostatic interactions of the oppositely charged ions of salt ( NaCl ) in crystals and in aqueous solution. (a) In crystalline table salt, sodium atoms are positively charged ions $\left(\mathrm{Na}^{+}\right)$due to the loss of one electron each, whereas chloride atoms are correspondingly negatively charged $\left(\mathrm{Cl}^{-}\right)$by gaining one electron each. (b) In solid form, ionic compounds form neatly ordered arrays, or crystals, of tightly packed ions in which the positive and negatively charged ions
counterbalance each other. (c) When the crystals are dissolved in water, the ions separate, and their charges, no longer balanced by immediately adjacent ions of opposite charge, are stabilized by interactions with polar water. The water molecules and the ions are held together by electrostatic interactions between the charges on the ion and the partial charges on the water's oxygen and hydrogen atoms. In aqueous solutions, all ions are surrounded by a hydration shell of water molecules.
water of hydration is lost when ions pass through protein pores in the cell membrane during nerve conduction.

The relative strength of the interaction between two oppositely charged ions, $\mathrm{A}^{-}$and $\mathrm{C}^{+}$, depends on the concentration of other ions in a solution. The higher the concentration of other ions (e.g., $\mathrm{Na}^{+}$and $\mathrm{Cl}^{-}$), the more opportunities $\mathrm{A}^{-}$and $\mathrm{C}^{+}$have to interact ionically with those other ions, and thus the lower the energy required to break the interaction between $\mathrm{A}^{-}$and $\mathrm{C}^{+}$. As a result, increasing the concentrations of salts such as NaCl in a solution of biological molecules can weaken and even disrupt the ionic interactions holding the biomolecules together. This principle can be exploited to separate complex mixtures of interacting molecules such as proteins into their individual, pure components.

## Hydrogen Bonds Are Noncovalent Interactions That Determine the Water Solubility of Uncharged Molecules

A hydrogen bond is the interaction of a partially positively charged hydrogen atom in a dipole, such as water, with unpaired electrons from another atom, either in the same or in a different molecule. Normally, a hydrogen atom forms a covalent bond with only one other atom. However, a hydrogen atom covalently bonded to an electronegative donor atom D may form an additional weak association, the hydrogen bond, with an acceptor atom A , which must have a nonbonding pair of electrons available for the interaction:

$$
\mathrm{D}^{6-}-\mathrm{H}^{6+}+: \mathrm{A}^{6-} \rightleftharpoons \mathrm{D}^{6-}-\mathrm{H}^{6+} \ldots \ldots: \mathrm{A}^{6-}
$$

The length of the covalent $\mathrm{D}-\mathrm{H}$ bond is a bit longer than it would be if there were no hydrogen bond because the acceptor "pulls" the hydrogen away from the donor. An important feature of all hydrogen bonds is directionality. In the strongest hydrogen bonds, the donor atom, the hydrogen atom, and the acceptor atom all lie in a straight line. Nonlinear hydrogen bonds are weaker than linear ones; still, multiple nonlinear hydrogen bonds help to stabilize the three-dimensional structures of many proteins.

Hydrogen bonds are both longer and weaker than covalent bonds between the same atoms. In water, for example, the distance between the nuclei of the hydrogen and oxygen atoms of adjacent, hydrogen-bonded water molecules is about 0.27 nm , about twice the length of the covalent $\mathrm{O}-\mathrm{H}$ bonds within a single water molecule (Figure 2-8a). A hydrogen bond between water molecules (approximately $5 \mathrm{kcal} / \mathrm{mol}$ ) is much weaker than a covalent $\mathrm{O}-\mathrm{H}$ bond (roughly $110 \mathrm{kcal} / \mathrm{mol}$ ), although it is stronger than many other hydrogen bonds in biological molecules ( $1-2 \mathrm{kcal} / \mathrm{mol}$ ). Extensive intermolecular hydrogen bonding between water molecules accounts for many of water's key properties, including its unusually high melting and boiling points and its ability to dissolve many other molecules.

The solubility of uncharged substances in an aqueous environment depends largely on their ability to form hydrogen bonds with water. For instance, the hydroxyl group $(-\mathrm{OH})$ in alcohols $\left(-\mathrm{CH}_{2} \mathrm{OH}\right)$ and the amino group $\left(-\mathrm{NH}_{2}\right)$ in amines $\left(-\mathrm{CH}_{2} \mathrm{NH}_{2}\right)$ can form several hydrogen bonds with water, which allows these molecules to dissolve in water at high concentrations (Figure 2-8b). In general, molecules with polar bonds that easily form hydrogen bonds with water, as well as charged molecules and ions

![li67857775e0](li67857775e0.jpg)

FIGURE 2-8 Hydrogen bonding of water with itself and with other compounds. Each pair of nonbonding outer electrons in an oxygen or a nitrogen atom can accept a hydrogen atom in a hydrogen bond. The hydroxyl and the amino groups can also form hydrogen bonds with water. (a) In liquid water, each water molecule forms transient hydrogen bonds with several others, creating a dynamic network
that interact with the dipole in water, can readily dissolve in water; that is, they are hydrophilic. Many biological molecules contain, in addition to hydroxyl and amino groups, peptide and ester groups, which form hydrogen bonds with water via otherwise nonbonded electrons on their carbonyl oxygens (Figure 2-8c). X-ray crystallography combined with computational analysis permits an accurate depiction of the distribution of the outermost unbonded electrons of atoms that can participate in hydrogen bonds as well as the electrons in covalent bonds, as illustrated in Figure 2-9.

## Van der Waals Interactions Are Weak Attractive Interactions Caused by Transient Dipoles

When any two atoms approach each other closely, they create a weak, nonspecific attractive force called a van der Waals interaction. These nonspecific interactions result
![li68857775e0](li68857775e0.jpg)
of hydrogen-bonded molecules. (b) Water can also form hydrogen bonds with alcohols and amines, which accounts for the high solubility of these compounds. (c) The peptide group and the ester group, which are present in many biomolecules, commonly participate in hydrogen bonds with water or polar groups in other molecules.
from the momentary random fluctuations in the distribution of the electrons of any atom, which give rise to a transient unequal distribution of electrons. If two noncovalently bonded atoms are close enough, electrons of one atom will perturb the electrons of the other. This perturbation generates a transient dipole in the second atom, and the two dipoles attract each other weakly (Figure 2-10). Similarly, a polar covalent bond in one molecule attracts an oppositely oriented dipole in another.

Van der Waals interactions, involving either transient or permanent dipoles, occur in all types of molecules, both polar and nonpolar. In particular, van der Waals interactions are responsible for the cohesion between nonpolar molecules such as heptane, $\mathrm{CH}_{3}-\left(\mathrm{CH}_{2}\right)_{3}-\mathrm{CH}_{3}$, that cannot form hydrogen bonds or ionic interactions with each other. The strength of van der Waals interactions decreases rapidly with increasing distance; thus these noncovalent interactions

FIGURE 2-9 Distribution of bonding and outer nonbonding electrons in the peptide group. Shown here is a peptide bond linking two amino acids within a protein called crambin. No protein has been structurally characterized at higher resolution than crambin. The black lines represent the covalent bonds between atoms. The red (negative) and blue (positive) lines represent contours of charge determined using x-ray crystallography and computational methods. The greater the number of contour lines, the higher the charge. The high density of red contour lines between atoms represents the covalent bonds (shared electron pairs). The two sets of red contour lines emanating from the oxygen (O) and not falling on a covalent bond (black line) represent the two pairs of nonbonding electrons on the oxygen that are available to participate in hydrogen bonding. The high density of blue contour lines near the hydrogen $(\mathrm{H})$ bonded to nitrogen $(\mathrm{N})$ represents a partial positive charge, indicating that this H can act as a donor in hydrogen bonding. (From Proc. Natl. Acad. Sci. USA, 2000, 97(7):3171-3176, Fig. 3A. Accurate protein crystallography at ultra-high resolution: Valence electron distribution in crambin, by Christian Jelsch et al., Copyright (2000) National Academy of Sciences, USA.)

![li69857775e0](li69857775e0.jpg)

FIGURE 2-10 Two oxygen molecules in van der Waals contact. In this model, red indicates negative charge and blue indicates positive charge. Transient dipoles in the electron clouds of all atoms give rise to weak attractive forces, called van der Waals interactions. Each type of atom has a characteristic van der Waals radius at which van der Waals interactions with other atoms are optimal. Because atoms repel one another if they are close enough together for their outer electrons to overlap without being shared in a covalent bond, the van der Waals radius is a measure of the size of the electron cloud surrounding an atom. The covalent radius indicated here is for the double bond of $\mathrm{O}=\mathrm{O}$; the single-bond covalent radius of oxygen is slightly longer.
can form only when atoms are quite close to one another. However, if atoms get too close together, the negative charges of their electrons create a repulsive force. When the van der Waals attraction between two atoms exactly balances the repulsion between their two electron clouds, the atoms are said to be in van der Waals contact. The strength of the van der Waals interaction is about $1 \mathrm{kcal} / \mathrm{mol}$, so it is weaker than typical hydrogen bonds, and its energy is only slightly higher than the average thermal energy of molecules at $25^{\circ} \mathrm{C}$. Thus multiple van der Waals interactions, a van der Waals interaction together with other noncovalent interactions, or both are required to form van der Waals-mediated stable attractions within and between molecules.

## The Hydrophobic Effect Causes Nonpolar Molecules to Adhere to One Another

Because nonpolar molecules do not contain charged groups, do not possess a dipole moment, and do not become hydrated, they are insoluble, or almost insoluble, in water; that is, they are hydrophobic. The covalent bonds between two carbon atoms and between carbon and hydrogen atoms are the most common nonpolar bonds in biological systems. Hydrocarbons-molecules made up only of carbon and hydrogen-are virtually insoluble in water. Large triacylglycerols (also known as triglycerides), which make up animal fats and vegetable oils, also essentially are insoluble in water. As we will see later, the major part of these molecules consists of long hydrocarbon chains. After being shaken in water, triacylglycerols form a separate phase. A familiar example is the separation of oil from the water-based vinegar in an oil-and-vinegar salad dressing.

Nonpolar molecules or nonpolar parts of molecules tend to aggregate in water owing to a phenomenon called the hydrophobic effect. Because water molecules cannot form hydrogen bonds with nonpolar substances, they tend to form "cages" of relatively rigid hydrogen-bonded pentagons and hexagons around nonpolar molecules (Figure 2-11, left). This state is energetically unfavorable because it decreases the entropy, or randomness, of the population of water molecules. (The role of entropy in chemical systems is discussed in Section 2.4.) If nonpolar molecules in an aqueous environment aggregate with their hydrophobic surfaces facing each other, the net hydrophobic surface area exposed to water is reduced (Figure 2-11, right). As a consequence, less water is needed to form the cages surrounding the nonpolar molecules, entropy increases relative to the unaggregated state, and an energetically more favorable state is reached. In a sense, then, water squeezes the nonpolar molecules into aggregates. Rather than constituting an attractive force, as in hydrogen bonds, the hydrophobic effect results from an avoidance of an unstable state-that is, extensive water cages around individual nonpolar molecules.

Nonpolar molecules can also associate, albeit weakly, through van der Waals interactions. The net result of the hydrophobic effect and van der Waals interactions is a very powerful tendency for hydrophobic molecules to interact with one another, not with water. Simply put, like dissolves like. Polar molecules dissolve in polar solvents such as water; nonpolar molecules dissolve in nonpolar solvents such as hexane.

2
One well-known hydrophobic molecule is cholesterol (see the structure in Section 2.2). Cholesterol, triglycerides, and other poorly water-soluble molecules are called lipids. Unlike hydrophilic molecules such as glucose or
![li70857775e0](li70857775e0.jpg)

FIGURE 2-11 Schematic depiction of the hydrophobic effect. Cages of water molecules that form around nonpolar molecules in solution are more ordered than water molecules in the surrounding bulk liquid. Aggregation of nonpolar molecules reduces the number of water molecules involved in forming highly ordered cages, resulting in a higher-entropy, more energetically favorable state (right) compared with the unaggregated state (left).

amino acids, lipids cannot readily dissolve in the blood, the aqueous circulatory system that transports molecules and cells throughout the body. Instead, lipids such as cholesterol must be packaged into special hydrophilic carriers, called lipoproteins, that can themselves dissolve in the blood and be transported throughout the body. There can be hundreds to thousands of lipid molecules packed into the center, or core, of each lipoprotein. The hydrophobic core is surrounded by amphipathic molecules that have hydrophilic parts that interact with water and hydrophobic parts that interact with one another and the core. The packaging of lipids into lipoproteins (discussed in Chapter 14) permits their efficient transport in blood and is reminiscent of the containerization of cargo for efficient long-distance transport via cargo ships, trains, and trucks.

High-density lipoprotein (HDL) and low-density lipoprotein (LDL) are two such lipoprotein carriers that are associated with either reduced or increased heart disease, respectively, and are therefore often referred to as "good" and "bad" cholesterol. Actually, the cholesterol molecules and their derivatives that are carried by both HDL and LDL are essentially identical and in themselves are neither "good" nor "bad." However, HDL and LDL have different effects on cells, and as a consequence, LDL contributes to and HDL appears to protect from clogging of the arteries (known as atherosclerosis) and consequent heart disease and stroke. Thus LDL is known as "bad" cholesterol.

## Molecular Complementarity Due to Noncovalent Interactions Leads to a Lock-and-Key Fit Between Biomolecules

Both inside and outside cells, ions and molecules constantly collide. The higher the concentration of any two types of molecules, the more likely they are to encounter each other. When two molecules encounter each other, they are most likely to simply bounce apart because the noncovalent interactions that would bind them together are weak and have a transient existence at physiological temperatures. However, molecules that exhibit molecular complementarity, a lock-and-key kind of fit between their shapes, charges, or other physical properties, can form multiple noncovalent interactions at close range. When two such structurally complementary molecules bump into each other, these multiple interactions cause them to stick together, or bind.

Figure 2-12 illustrates how multiple, different weak interactions can cause two hypothetical proteins to bind together tightly. Numerous examples of such protein-to-protein molecular complementarity may be found throughout this book (see, for example, Figures 16-8, 16-9, and 16-11). Almost any other arrangement of the same groups of molecules on the two surfaces would not allow the molecules to bind so tightly. Such molecular complementarity between regions within a protein molecule allow it to fold into a unique three-dimensional shape (see Chapter 3); it is also what holds the two chains of DNA together in a double helix (see Chapter 5). Similar interactions underlie the association
![li71857775e0](li71857775e0.jpg)

FIGURE 2-12 Molecular complementarity permits tight protein bonding via multiple noncovalent interactions. The complementary shapes, charges, polarity, and hydrophobicity of two protein surfaces permit multiple weak interactions, which in combination produce a strong interaction and tight binding. Because deviations from molecular complementarity substantially weaken binding, a particular surface region of any given biomolecule usually can bind tightly to only one or a very limited number of other molecules. The complementarity of the two protein molecules on the left permits them to bind much more tightly than the two noncomplementary proteins on the right.
of groups of molecules into multimolecular assemblies, or complexes, leading, for example, to the formation of muscle fibers, to the gluelike associations between cells in solid tissues, and to numerous other cellular structures. The antibodies that help neutralize pathogens (see Chapter 23) bind to them using similar principles of complementary molecular shapes.

Depending on the number and strength of the noncovalent interactions between the two molecules and on their environment, their binding may be tight or loose and, as a consequence, either lasting or transient. The higher the affinity of two molecules for each other, the better the molecular "fit" between them, the more noncovalent interactions can form, and the more tightly they can bind together. An important quantitative measure of affinity is the binding dissociation constant $K_{\mathrm{d}}$, described in Section 2.3. It is important to note that many large biological molecules are not hard, rigid structures, but rather can be somewhat malleable. Thus the binding of a molecule to another has the potential to induce a change in the shape of its binding partner. When the molecular complementarity increases after such interactions, the process is called induced fit.

As we discuss in Chapter 3, nearly all the chemical reactions that occur in cells also depend on the binding properties of enzymes. These proteins not only speed up, or catalyze, reactions, but do so with a high degree of specificity, which is a reflection of their ability to bind tightly to only one or a few related molecules. The specificity of intermolecular interactions and reactions, which depends on molecular complementarity, is essential for many processes critical to life.

## KEY CONCEPTS OF SECTION 2.1

## Covalent Bonds and Noncovalent Interactions

- The terms hydrophilic, hydrophobic, and amphipathic/ amphiphilic refer to the tendency of molecules to be waterloving, incapable of interacting with water, and having features of or being tolerant of both, respectively. Hydrophilic molecules typically dissolve readily in water, whereas hydrophobic molecules are poorly soluble or insoluble in water.
- Covalent bonds consist of pairs of electrons shared by two atoms. Covalent bonds arrange the atoms of a molecule into a specific geometry.
- Many molecules in cells contain at least one asymmetric carbon atom, which is bonded to four dissimilar atoms. Such molecules can exist as stereoisomers (mirror images), designated D and L (see Figure 2-4), which have different biological activities. Nearly all amino acids are L isomers.
- Electrons may be shared equally or unequally in covalent bonds. Atoms that differ in electronegativity form polar covalent bonds, in which the bonding electrons are distributed unequally. One end of a polar bond has a partial positive charge and the other end has a partial negative charge (see Figure 2-5).
- Covalent bonds are stable in biological systems because the relatively high energies required to break them (50-200 $\mathrm{kcal} / \mathrm{mol}$ ) are much larger than the thermal kinetic energy available at room $\left(25^{\circ} \mathrm{C}\right)$ or body $\left(37^{\circ} \mathrm{C}\right)$ temperatures.
- Noncovalent interactions between atoms are considerably weaker than covalent bonds, with energies ranging from about $1-5 \mathrm{kcal} / \mathrm{mol}$ (see Figure 2-6).
- Four main types of noncovalent interactions occur in biological systems: ionic bonds, hydrogen bonds, van der Waals interactions, and interactions due to the hydrophobic effect.
- Ionic bonds result from the electrostatic attraction between the positive and negative charges of ions. In aqueous solutions, all cations and anions are surrounded by a shell of bound water molecules (see Figure 2-7c). Increasing the salt (e.g., NaCl ) concentration of a solution can weaken the relative strength of and even break the ionic bonds between biomolecules.
- In a hydrogen bond, a hydrogen atom covalently bonded to an electronegative atom associates with an acceptor atom whose nonbonding electrons attract the hydrogen (see Figure 2-8).
- Weak and relatively nonspecific van der Waals interactions result from the attraction between transient dipoles associated with all molecules. They can form when two atoms approach each other closely (see Figure 2-10).
- In an aqueous environment, nonpolar molecules or nonpolar parts of larger molecules are driven together by the
hydrophobic effect, thereby reducing the extent of their direct contact with water molecules (see Figure 2-11).
- Molecular complementarity is the lock-and-key fit between molecules whose shapes, charges, and other physical properties are complementary. Multiple noncovalent interactions can form between complementary molecules, causing them to bind tightly (see Figure 2-12), but not between molecules that are not complementary.
- The high degree of binding specificity that results from molecular complementarity is one of the features that underlies intermolecular interactions in biology and thus is essential for many processes critical to life.


### 2.2 Chemical Building Blocks of Cells

A common theme in biology is the construction of large macromolecules and macromolecular structures out of smaller molecular subunits, which can be thought of as building blocks. Often these subunits are similar or identical. The three main types of biological macromolecules-proteins, nucleic acids, and polysaccharides-are all polymers composed of multiple covalently linked small molecules, or monomers (Figure 2-13). Proteins are linear polymers containing up to several thousand amino acids linked by peptide bonds. Nucleic acids are linear polymers containing hundreds to millions of nucleotides linked by phosphodiester bonds. Polysaccharides are linear or branched polymers of monosaccharides (sugars) such as glucose linked by glycosidic bonds.

Although the actual mechanisms of covalent bond formation between monomers are complex, as we will see, the formation of a covalent bond between two monomers usually involves the net loss of a hydrogen $(\mathrm{H})$ from one monomer and a hydroxyl $(\mathrm{OH})$ from the other monomer-or the net loss of one water molecule-and can therefore be thought of as a dehydration reaction. The breakdown, or cleavage, of a covalent bond in a polymer that releases a monomeric subunit involves the reverse reaction, or the addition of water, called hydrolysis. The covalent bonds that link monomers together are normally stable under normal biological conditions (e.g., $37^{\circ} \mathrm{C}$, neutral pH ), so these biopolymers are stable and can perform a wide variety of jobs in cells, such as storing information, catalyzing chemical reactions, serving as structural elements that define cell shape and movement, and many others. Macromolecular structures can also be assembled using noncovalent interactions. The two-ply, or "bilayer," structure of cellular membranes is built up by the noncovalent assembly of many thousands of small molecules called phospholipids (see Figure 2-13).

In this chapter, we focus on the chemical building blocks making up cells-amino acids, nucleotides, sugars, and phospholipids. The structure, function, and assembly of

![li72857775e0](li72857775e0.jpg)

FIGURE 2-13 Overview of the cell's principal chemical building blocks. (Top) The three major types of biological macromolecules are each assembled by the polymerization of multiple small molecules (monomers) of a particular type: proteins from amino acids (see Chapter 3), nucleic acids from nucleotides (see Chapter 5), and
polysaccharides from monosaccharides (sugars). Each monomer is covalently linked into the polymer by a reaction whose net result is loss of a water molecule (dehydration). (Bottom) In contrast, phospholipid monomers noncovalently assemble into a bilayer structure, which forms the basis of all cellular membranes (see Chapter 7).
proteins, nucleic acids, polysaccharides, and biomembranes are discussed in subsequent chapters.

## Amino Acids Differing Only in Their Side Chains Compose Proteins

The monomeric building blocks of proteins are 20 amino acids, which-when incorporated into a protein poly-mer-are sometimes called residues. All amino acids have a characteristic structure consisting of a central alpha carbon atom $\left(\mathrm{C}_{\alpha}\right)$ bonded to four different chemical groups: an amino $\left(-\mathrm{NH}_{2}\right)$ group, a carboxyl or carboxylic acid $(-\mathrm{COOH})$ group (hence the name amino acid), a hydrogen $(\mathrm{H})$ atom, and one variable group, called a side chain or $R$
group. Because the $\alpha$ carbon in all amino acids except glycine is asymmetric, these molecules can exist in two mirrorimage forms, called by convention the D (dextro) and the L (levo) isomers (see Figure 2-4). The two isomers cannot be interconverted (one made identical to the other) without breaking and then re-forming a chemical bond in one of them. With rare exceptions, only the L forms of amino acids are found in proteins. However, D amino acids are prevalent in bacterial cell walls and other microbial products.

To understand the three-dimensional structures and functions of proteins, discussed in detail in Chapter 3, you must be familiar with some of the distinctive properties of amino acids, which are determined in part by their side chains. You need not memorize the detailed structure of each type of side chain

![li73857775e0](li73857775e0.jpg)
to understand how proteins work because amino acids can be classified into several broad categories based on the size, shape, charge, hydrophobicity (a measure of water solubility), and chemical reactivity of their side chains (Figure 2-14).

Amino acids with nonpolar side chains, called hydrophobic amino acids, are poorly soluble in water. The larger the nonpolar side chain, the more hydrophobic the amino acid. The side chains of alanine, valine, leucine, and isoleucine are linear or branched hydrocarbons that do not form a ring, and they are therefore called aliphatic amino acids. These amino acids are all nonpolar, as is methionine, which is similar to them except that it contains one sulfur atom. Phenylalanine, tyrosine, and tryptophan have large, hydrophobic,
aromatic rings in their side chains. In later chapters, we will see in detail how hydrophobic side chains under the influence of the hydrophobic effect often pack into the interior of proteins or line the surfaces of proteins that are embedded within hydrophobic regions of biomembranes.

Amino acids with polar side chains are called hydrophilic amino acids; the most hydrophilic of these amino acids is the subset with side chains that are charged (ionized) at the pH typical of biological fluids $(\sim 7)$ both inside and outside the cell (see Section 2.3). Arginine and lysine have positively charged side chains and are called basic amino acids; aspartic acid and glutamic acid have negatively charged side chains due to the carboxylic acid groups in their side chains

(their charged forms are called aspartate and glutamate) and are called acidic amino acids. A fifth amino acid, histidine, has a side chain containing a ring with two nitrogens, called imidazole, which can shift from being positively charged to uncharged in response to small changes in the acidity of its environment:
![li74857775e0](li74857775e0.jpg)

The activities of many proteins are modulated by shifts in environmental acidity $(\mathrm{pH})$ through protonation or deprotonation of histidine side chains. Asparagine and glutamine are uncharged but have polar side chains containing amide groups with extensive hydrogen-bonding capacities. Similarly, serine and threonine are uncharged but have polar hydroxyl groups, which also participate in hydrogen bonds with other polar molecules.

Finally, cysteine, glycine, and proline play special roles in proteins because of the unique properties of their side chains. The side chain of cysteine contains a reactive sulfhydryl group $(-S H)$. On release of a proton $\left(\mathrm{H}^{+}\right)$, a sulfhydryl group is converted into a thiolate anion $\left(\mathrm{S}^{-}\right)$. Thiolate anions can play important roles in catalysis, notably in certain enzymes that destroy proteins (proteases). In proteins, each of two adjacent sulfhydryl groups can be oxidized, each releasing a proton and an electron, to form a covalent disulfide bond $(-S-S-)$ :
![li75857775e0](li75857775e0.jpg)

Disulfide bonds serve to "cross-link" regions within a single polypeptide chain (intramolecular cross-linking) or between two separate chains (intermolecular cross-linking). Disulfide bonds stabilize the folded structure of some proteins. The smallest amino acid, glycine, has a single hydrogen atom as its R group. Its small size allows it to fit into tight spaces. Unlike those of the other common amino acids, the side chain of proline (pronounced pro-leen) bends around to form a ring by covalently bonding to the nitrogen atom in the amino group attached to the $\mathrm{C}_{\alpha}$. As a result, proline is very rigid, and its amino group is not available for typical
hydrogen bonding. The presence of proline in a protein creates a fixed kink in the polymer chain, limiting how it can fold in the vicinity of the proline residue.

Some amino acids are more abundant in proteins than others. Cysteine, tryptophan, and methionine are not common amino acids: together, they constitute approximately 5 percent of the amino acids in a typical protein. Four amino acids-leucine, serine, lysine, and glutamic acid-are the most abundant amino acids, constituting 32 percent of all the residues in a typical protein. However, the amino acid compositions of particular proteins may vary widely from these values.

5Humans and other mammals can synthesize 11 of the 20 amino acids. The other nine are called essential amino acids and must be included in the diet to permit normal protein production. These essential amino acids are phenylalanine, valine, threonine, tryptophan, isoleucine, methionine, leucine, lysine, and histidine. Adequate provision of these essential amino acids in feed is key to the livestock industry. Indeed, a genetically engineered variety of corn with a high lysine content is now in use as an "enhanced" feed to promote the growth of animals.

Although cells use the 20 amino acids shown in Figure 2-14 in the initial synthesis of proteins, analysis of cellular proteins reveals that they contain over 100 different amino acids. The difference is due to the chemical modification of some amino acids after they are incorporated into proteins by the addition of a variety of chemical groups (Figure 2-15). One important modification is the addition of acetyl groups $\left(\mathrm{CH}_{3} \mathrm{CO}\right)$ to amino acids, a process known as acetylation. Another is the addition of a phosphate $\left(\mathrm{PO}_{4}\right)$ to hydroxyl groups in serine, threonine, and tyrosine residues, a process known as phosphorylation. We will encounter numerous examples of proteins whose activity is regulated by reversible phosphorylation and dephosphorylation. Phosphorylation of nitrogen in the side chain of histidine is well known in bacteria, fungi, and plants, but less studiedperhaps because of the relative instability of phosphorylated histidine-and apparently rare in mammals. Methylation of arginine and lysine side chains on proteins called histones is an important regulator of gene expression in eukaryotes (see Chapter 9). Like phosphorylation and dephosphorylation, controlled methylation and demethylation are important regulatory processes. The side chains of asparagine, serine, and threonine are sites for glycosylation, the attachment of linear and branched carbohydrate chains. Many secreted proteins and membrane proteins contain glycosylated residues, and the reversible modification of hydroxyl groups on specific serines and threonines by a sugar called N -acetylglucosamine also regulates protein activities. Other amino acid modifications found in selected proteins include the hydroxylation of proline and lysine residues in collagen (see Chapter 19), the methylation of histidine residues in membrane receptors, and the $\gamma$-carboxylation of glutamate in blood-clotting factors such as prothrombin. Deamidation of asparagine and

![li76857775e0](li76857775e0.jpg)

FIGURE 2-15 Common modifications of amino acid side chains in proteins. These modified residues and numerous others are formed by addition of various chemical groups (red) to the amino acid side chains during or after synthesis of a polypeptide chain.
glutamine to form the corresponding acidic amino acids, aspartate and glutamate, is also a common occurrence.

Acetylation of the amino group of the N -terminal residue is the most common form of amino acid chemical modification, affecting an estimated 80 percent of all proteins:
![li77857775e0](li77857775e0.jpg)

This modification may play an important role in controlling the life span of proteins within cells because many nonacetylated proteins are rapidly degraded.

## Five Different Nucleotides Are Used to Build Nucleic Acids

Two types of chemically similar nucleic acids, DNA (deoxyribonucleic acid) and RNA (ribonucleic acid), are the cell's principal molecules that carry genetic information. The monomers from which DNA and RNA polymers are built, called nucleotides, all have a common structure: a phosphate group linked by a phosphoester bond to a pentose (five-carbon) sugar, which in turn is linked to a nitrogenand carbon-containing ring structure commonly referred to as a base (Figure 2-16a). In RNA, the pentose is ribose; in DNA, it is deoxyribose, which has a proton, rather than a hydroxyl group, at position 2' (Figure 2-16b). (We describe the structures of sugars in more detail below.) The bases adenine, guanine, and cytosine (Figure 2-17) are found in both DNA and RNA; thymine is found only in DNA, and uracil is found only in RNA.

Adenine and guanine are purines, which contain a pair of fused rings; cytosine, thymine, and uracil are pyrimidines, which contain a single ring (see Figure 2-17). The bases are often abbreviated A, G, C, T, and U, respectively; these same single-letter abbreviations are also commonly used to denote the entire nucleotides in nucleic acid polymers. In nucleotides, the $1^{\prime}$ carbon atom of the sugar (ribose or deoxyribose) is attached to the nitrogen at position 9 of a purine $\left(\mathrm{N}_{9}\right)$ or at position 1 of a pyrimidine $\left(\mathrm{N}_{1}\right)$. The acidic character of nucleotides is due to the phosphate group, which under normal intracellular conditions releases hydrogen
(a)
![li78857775e0](li78857775e0.jpg)
(b)
![li79857775e0](li79857775e0.jpg)

FIGURE 2-16 Common structure of nucleotides. (a) Adenosine 5'-monophosphate (AMP), a nucleotide present in RNA. By convention, the carbon atoms of the pentose sugar in nucleotides are numbered with primes. In natural nucleotides, the $1^{\prime}$ carbon is joined by a $\beta$ linkage to the base (in this case, adenine); both the base (blue) and the phosphate on the $5^{\prime}$ hydroxyl (red) extend above the plane of the sugar ring. (b) Ribose and deoxyribose, the pentoses in RNA and DNA, respectively.

![li80857775e0](li80857775e0.jpg)

FIGURE 2-17 Chemical structures of the principal bases in nucleic acids. In nucleic acids and nucleotides, nitrogen 9 of purines and nitrogen 1 of pyrimidines (red) are bonded to the $1^{\prime}$ carbon of ribose or deoxyribose. $U$ is found only in RNA, and $T$ is found only in DNA. Both RNA and DNA contain A, G, and C.
ions $\left(\mathrm{H}^{+}\right)$, leaving the phosphate negatively charged (see Figure 2-16a). Most nucleic acids in cells are associated with proteins, which form ionic interactions with the negatively charged phosphates.

Cells and extracellular fluids in organisms contain small concentrations of nucleosides, combinations of a base and a sugar without a phosphate. Nucleotides are nucleosides that have one, two, or three phosphate groups esterified at the $5^{\prime}$ hydroxyl. Esterification-the formation of an
ester-involves the covalent linking of an acid, such as a carboxylic acid or a phosphoric acid, with an alcohol accompanied by the release of an hydroxyl $(-\mathrm{OH})$ group from the acid and an H from the hydroxyl group on the other molecule, which together form a water molecule. Here, a phosphoric acid is esterified with the $5^{\prime}$ hydroxyl group of the ribose. Nucleoside monophosphates have a single esterified phosphate (see Figure 2-16a); nucleoside diphosphates contain a pyrophosphate group:
![li81857775e0](li81857775e0.jpg)
and nucleoside triphosphates have a third phosphate. Table 2-3 lists the names of the nucleosides and nucleotides in nucleic acids and the various forms of nucleoside phosphates. The nucleoside triphosphates are used in the synthesis of nucleic acids, which we cover in Chapter 5. Among their other functions in the cell, GTP participates in intracellular signaling and acts as an energy reservoir, particularly in protein synthesis, and ATP, discussed later in this chapter, is the most widely used biological energy carrier.

## Monosaccharides Covalently Assemble into Linear and Branched Polysaccharides

The building blocks of the polysaccharides are the simple sugars, or monosaccharides. Monosaccharides are carbohydrates, which are literally covalently bonded combinations of carbon and water in a one-to-one ratio $\left(\mathrm{CH}_{2} \mathrm{O}\right)_{n}$, where $n$ equals $3,4,5,6$, or 7 . Hexoses $(n=6)$ and pentoses $(n=5)$ are the most common monosaccharides. All monosaccharides

# TABLE 2-3 Terminology of Nucleosides and Nucleotides 

| Bases |  | Purines |  | Pyrimidines |  |
| :--: | :--: | :--: | :--: | :--: | :--: |
|  |  | Adenine (A) | Guanine (G) | Cytosine (C) | Uracil (U) <br> Thymine (T) |
| Nucleosides | in RNA | Adenosine | Guanosine | Cytidine | Uridine |
|  | in DNA | Deoxyadenosine | Deoxyguanosine | Deoxycytidine | Deoxythymidine |
| Nucleotides | in RNA | Adenylate | Guanylate | Cytidylate | Uridylate |
|  | in DNA | Deoxyadenylate | Deoxyguanylate | Deoxycytidylate | Deoxythymidylate |
| Nucleoside monophosphates |  | AMP | GMP | CMP | UMP |
| Nucleoside diphosphates |  | ADP | GDP | CDP | UDP |
| Nucleoside triphosphates |  | ATP | GTP | CTP | UTP |
| Deoxynucleoside mono-, di-, and triphosphates |  | dAMP, etc. | dGMP, etc. | dCMP, etc. | dTMP, etc. |

contain hydroxyl $(-\mathrm{OH})$ groups and either an aldehyde or a keto group:
![li82857775e0](li82857775e0.jpg)

Many biologically important sugars are hexoses, including glucose, mannose, and galactose (Figure 2-18). Mannose is identical to glucose except that the orientation of the groups bonded to carbon 2 is reversed. Similarly, galactose, another hexose, differs from glucose only in the orientation of the groups attached to carbon 4. Interconversion of glucose and mannose or galactose requires the breaking and making of covalent bonds; such reactions are carried out by enzymes called epimerases.

D-Glucose $\left(\mathrm{C}_{6} \mathrm{H}_{12} \mathrm{O}_{6}\right)$ is the principal external source of energy for most cells in complex multicellular organisms. It can exist in three different forms: a linear structure and two different hemiacetal ring structures (Figure 2-18a). If the aldehyde group on carbon 1 combines with the hydroxyl group on carbon 5, the resulting hemiacetal, D-glucopyranose, contains a six-member ring. In the $\alpha$ anomer of D-glucopyranose, the hydroxyl group attached
![li83857775e0](li83857775e0.jpg)

FIGURE 2-18 Chemical structures of hexoses. All hexoses have the same chemical formula $\left(\mathrm{C}_{6} \mathrm{H}_{12} \mathrm{O}_{6}\right)$ and contain an aldehyde or a keto group. (a) The ring forms of D-glucose are generated from the linear molecule by reaction of the aldehyde at carbon 1 with the hydroxyl on carbon 5 or carbon 4. The three forms are readily interconvertible, although the pyranose form (right) predominates in biological systems. (b) In D-mannose and D-galactose, the configuration of the H (green) and OH (blue) bound to one carbon atom differs from that in glucose. These sugars, like glucose, exist primarily as pyranoses (six-member rings).
to carbon 1 points "downward" from the ring, as shown in Figure 2-18a; in the $\beta$ anomer, this hydroxyl points "upward." In aqueous solution, the $\alpha$ and $\beta$ anomers readily interconvert spontaneously; at equilibrium there is about one-third $\alpha$ anomer and two-thirds $\beta$, with very little of the open-chain form. Because enzymes can distinguish between the $\alpha$ and $\beta$ anomers of D-glucose, these forms have distinct biological roles. Condensation of the hydroxyl group on carbon 4 of the linear glucose with its aldehyde group results in the formation of D-glucofuranose, a hemiacetal containing a five-member ring. Although all three forms of D-glucose exist in biological systems, the pyranose (six-member ring) form is by far the most abundant.

The pyranose ring in Figure 2-18a is depicted as planar. In fact, because of the tetrahedral geometry around carbon atoms, the most stable conformation of a pyranose ring has a nonplanar, chairlike shape. In this conformation, each bond from a ring carbon to a nonring atom (e.g., H or O ) is either nearly perpendicular to the ring, referred to as axial (a), or nearly in the plane of the ring, referred to as equatorial (e):
![li84857775e0](li84857775e0.jpg)

Disaccharides, formed from two monosaccharides, are the simplest polysaccharides. The disaccharide lactose, composed of galactose and glucose, is the major sugar in milk; the disaccharide sucrose, composed of glucose and fructose, is a principal product of plant photosynthesis and is refined into common table sugar (Figure 2-19).

Larger polysaccharides, containing dozens to hundreds of monosaccharide units, can function as reservoirs for glucose, as structural components, or as adhesives that help hold cells together in tissues. The most common storage carbohydrate in animal cells is glycogen, a very long, highly branched polymer of glucose. As much as 10 percent of the liver by weight can be glycogen. The primary storage carbohydrate in plant cells, starch, is also a glucose polymer. It occurs in an unbranched form (amylose) and a lightly branched form (amylopectin). Both glycogen and starch are composed of the $\alpha$ anomer of glucose. In contrast, cellulose, the major constituent of plant cell walls, which confers stiffness to many plant structures (see Chapter 19), is an unbranched polymer of the $\beta$ anomer of glucose. Human digestive enzymes can hydrolyze the $\alpha$ glycosidic bonds in starch but not the $\beta$ glycosidic bonds in cellulose. Many species of plants, bacteria, and molds produce cellulose-degrading enzymes. Cows and termites can break down cellulose because they harbor cellu-lose-degrading bacteria in their gut. Bacterial cell walls consist of peptidoglycan, a polysaccharide chain cross-linked by peptide cross-bridges, which confers rigidity and cell shape. Human tears and gastrointestinal fluids contains lysozyme, an enzyme capable of hydrolyzing peptidoglycan in the bacterial cell wall.

## FIGURE 2-19 Formation of the disaccharides lactose and sucrose. In any glycosidic linkage,

the anomeric carbon of one sugar molecule (in either the $\alpha$ or $\beta$ conformation) is linked to a hydroxyl oxygen on another sugar molecule. The linkages are named accordingly; thus lactose contains a $\beta(1 \rightarrow 4)$ bond, and sucrose contains an $\alpha(1 \rightarrow 2)$ bond.
![li85857775e0](li85857775e0.jpg)

The enzymes that make the glycosidic bonds linking monosaccharides into polysaccharides are specific for the $\alpha$ or $\beta$ anomer of one sugar and a particular hydroxyl group on the other. In principle, any two sugar molecules can be linked in a variety of ways because each monosaccharide has multiple hydroxyl groups that can participate in the formation of glycosidic bonds. Furthermore, any one monosaccharide has the potential to be linked to more than two other monosaccharides, thus generating a branch point and nonlinear polymers. Glycosidic bonds are usually formed between the growing polysaccharide chain and a covalently modified form of a monosaccharide. Such modifications include the addition of a phosphate (e.g., glucose-6-phosphate) or a nucleotide (e.g., UDP-galactose):
![li86857775e0](li86857775e0.jpg)

Glucose-6-phosphate
![li87857775e0](li87857775e0.jpg)

UDP-galactose

The epimerase enzymes that interconvert different monosaccharides often do so using the nucleotide sugars rather than the unmodified, or "free," sugars.

Many complex polysaccharides contain modified sugars that are covalently linked to various small groups, particularly amino, sulfate, and acetyl groups. Such modifications are abundant in glycosaminoglycans, major polysaccharide components of the extracellular matrix that we describe in Chapter 19.

## Phospholipids Associate Noncovalently to Form the Basic Bilayer Structure of Biomembranes

Biomembranes are large, flexible sheets with a two-ply, or bilayer, structure. They serve as the boundaries of cells and
their intracellular organelles and form the outer surfaces of some viruses. Membranes literally define what is a cell (the outer membrane and the contents within the membrane) and what is not (the extracellular space outside the membrane). Unlike proteins, nucleic acids, and polysaccharides, membranes are assembled by the noncovalent association of their component building blocks. The primary building blocks of all biomembranes are phospholipids, whose physical properties are responsible for the formation of the sheet-like bilayer structure of membranes. In addition to phospholipids, biomembranes can contain a variety of other molecules, including cholesterol, glycolipids, and proteins. The structure and functions of biomembranes will be described in detail in Chapter 7. Here we will focus on the phospholipids in biomembranes.

To understand the structure a phospholipid molecule, we have to understand each of its component parts and how it is assembled. As we will see shortly, a phospholipid molecule consists of two long-chain, nonpolar fatty acid groups linked (usually by an ester bond) to small, highly polar groups, including a short organic molecule such as glycerol (trihydroxy propane), a phosphate, and typically, a small organic molecule (Figure 2-20).

Fatty acids consist of a hydrocarbon chain attached to a carboxyl group ( -COOH ). Like glucose, fatty acids are an important energy source for many cells (see Chapter 12). They differ in length, although the predominant fatty acids in cells have an even number of carbon atoms, usually 14 , 16,18 , or 20 . The major fatty acids in phospholipids are listed in Table 2-4. Fatty acids are often designated by the abbreviation Cx:y, where $x$ is the number of carbons in the chain and $y$ is the number of double bonds. Fatty acids containing 12 or more carbon atoms are nearly insoluble in aqueous solutions because of their long hydrophobic hydrocarbon chains.

Fatty acids in which all the carbon-carbon bonds are single bonds-that is, the fatty acids have no carbon-carbon double bonds-are said to be saturated; those with at least one carbon-carbon double bond are called unsaturated.

![li88857775e0](li88857775e0.jpg)

FIGURE 2-20 Phosphatidylcholine, a typical phosphoglyceride. All phosphoglycerides are amphipathic phospholipids, having a hydrophobic tail (yellow) and a hydrophilic head (blue) in which glycerol is linked via a phosphate group to an alcohol. Either or both
of the fatty acyl side chains in a phosphoglyceride may be saturated or unsaturated. In phosphatidic acid (red), the simplest phospholipid, the phosphate is not linked to an alcohol.

Unsaturated fatty acids with more than one carbon-carbon double bond are referred to as polyunsaturated. Two "essential" polyunsaturated fatty acids, linoleic acid (C18:2) and linolenic acid (C18:3), cannot be synthesized by mammals and must be supplied in their diet. Mammals can synthesize other common fatty acids.

In phospholipids, fatty acids are covalently attached to another molecule by esterification. In the combined molecule formed by this reaction, the part derived from the fatty acid is called an acyl group, or fatty acyl group. This structure is illustrated by the most common forms of phospholipids: phosphoglycerides, which contain two acyl groups attached to two of the three hydroxyl groups of glycerol (see Figure 2-20).

In phosphoglycerides, one hydroxyl group of the glycerol is esterified to phosphate while the other two are normally esterified to fatty acids. The simplest phospholipid, phosphatidic acid, contains only these components. Phospholipids such as phosphatidic acids are not only membrane building blocks but also important signaling molecules. Lysophosphatidic acid, in which the acyl chain at the 2 position (attached to the hydroxyl group on the central carbon of the glycerol) has been removed, is relatively water soluble and can be a potent inducer of cell division (called a mitogen). In most phospholipids found in membranes, the phosphate group is also esterified to a hydroxyl group on another hydrophilic compound. In phosphatidylcholine, for example,
choline is attached to the phosphate (see Figure 2-20). The negatively charged phosphate, as well as the charged or polar groups esterified to it, can interact strongly with water. The phosphate and its associated esterified group constitute the "head" group of a phospholipid, which is hydrophilic, whereas the fatty acyl chains, the "tails," are hydrophobic. Other common phosphoglycerides and associated head groups are shown in Table 2-5. Molecules such as phospholipids that have both hydrophobic and hydrophilic regions are called amphipathic. In Chapter 7, we will see how the amphipathic properties of phospholipids allow their assembly into sheet-like bilayers in which the fatty acyl tails point into the center of the sheet and the head groups point outward toward the aqueous environment (see Figure 2-13).

Fatty acyl groups also can be covalently linked in other fatty molecules, including triacylglycerols, or triglycerides, which contain three acyl groups esterified to glycerol:
![li89857775e0](li89857775e0.jpg)

# TABLE 2-4 Fatty Acids That Predominate in Phospholipids 

| Common Name of Acid (ionized form in parentheses) | Abbreviation | Chemical Formula |
| :-- | :--: | :--: |
| Saturated Fatty Acids |  |  |
| Myristic (myristate) | $\mathrm{C} 14: 0$ | $\mathrm{CH}_{3}\left(\mathrm{CH}_{2}\right)_{12} \mathrm{COOH}$ |
| Palmitic (palmitate) | $\mathrm{C} 16: 0$ | $\mathrm{CH}_{3}\left(\mathrm{CH}_{2}\right)_{14} \mathrm{COOH}$ |
| Stearic (stearate) | $\mathrm{C} 18: 0$ | $\mathrm{CH}_{3}\left(\mathrm{CH}_{2}\right)_{16} \mathrm{COOH}$ |
| Unsaturated Fatty Acids |  |  |
| Oleic (oleate) | $\mathrm{C} 18: 1$ | $\mathrm{CH}_{3}\left(\mathrm{CH}_{2}\right)_{7} \mathrm{CH}=\mathrm{CH}\left(\mathrm{CH}_{2}\right)_{7} \mathrm{COOH}$ |
| Linoleic (linoleate) | $\mathrm{C} 18: 2$ | $\mathrm{CH}_{3}\left(\mathrm{CH}_{2}\right)_{4} \mathrm{CH}=\mathrm{CHCH}_{2} \mathrm{CH}=\mathrm{CH}\left(\mathrm{CH}_{2}\right)_{7} \mathrm{COOH}$ |
| Arachidonic (arachidonate) | $\mathrm{C} 20: 4$ | $\mathrm{CH}_{3}\left(\mathrm{CH}_{2}\right)_{4}\left(\mathrm{CH}=\mathrm{CHCH}_{2}\right)_{3} \mathrm{CH}=\mathrm{CH}\left(\mathrm{CH}_{2}\right)_{3} \mathrm{COOH}$ |

TABLE 2-5 Common Phosphoglycerides and Head Groups

| Common Phosphoglycerides | Head Group |
| :--: | :--: |
| Phosphatidylcholine | ![li90857775e0](li90857775e0.jpg) |
| Phosphatidylethanolamine | ![li91857775e0](li91857775e0.jpg) |
| Phosphatidylserine | ![li92857775e0](li92857775e0.jpg) |
|  | Serine |
| Phosphatidylinositol | ![li93857775e0](li93857775e0.jpg) |

They also can be covalently attached to the very hydrophobic molecule cholesterol, an alcohol, to form cholesteryl esters:
![li94857775e0](li94857775e0.jpg)

A cis double bond introduces a rigid kink in the otherwise flexible straight acyl chain of a saturated fatty acid (Figure 2-21). In general, the unsaturated fatty acids in biological systems contain only cis double bonds. Saturated fatty acids without the kink can pack together tightly and so have higher melting points than unsaturated fatty acids. The main fatty molecules in butter are triglycerides with saturated fatty acyl chains, which is why butter is usually solid at room temperature. Unsaturated fatty acids or fatty acyl chains with the cis double bond kink cannot pack as closely together as saturated fatty acyl chains. Thus vegetable oils, composed of triglycerides with unsaturated fatty acyl groups, usually are liquid at room temperature. Vegetable and similar oils may be partially hydrogenated to convert some of their unsaturated fatty acyl chains to saturated fatty acyl chains. As a consequence, the hydrogenated vegetable oil can be molded into solid sticks of margarine. A by-product of the hydrogenation reaction is the conversion of some of the fatty acyl chains into trans fatty acids, popularly called "trans fats." These "trans fats," found in partially hydrogenated margarine and other food products, are not natural. Saturated and trans fatty acids have similar physical properties; for example, they tend to be solids at room temperature. Their consumption, relative to the consumption of unsaturated fats, is associated with increased plasma cholesterol levels and is discouraged by some nutritionists.
![li95857775e0](li95857775e0.jpg)

![li96857775e0](li96857775e0.jpg)

FIGURE 2-21 The effect of a double bond on the shape of fatty acids. Shown are chemical structures of the ionized form of palmitic acid, a saturated fatty acid with 16 C atoms, and oleic acid,

## KEY CONCEPTS OF SECTION 2.2

## Chemical Building Blocks of Cells

- Macromolecules are polymers of monomer subunits linked together by covalent bonds via dehydration reactions. Three major types of macromolecules are found in cells: proteins, composed of amino acids linked by peptide bonds; nucleic acids, composed of nucleotides linked by phosphodiester bonds; and polysaccharides, composed of monosaccharides (sugars) linked by glycosidic bonds (see Figure 2-13). Phospholipids, the fourth major chemical building block, assemble noncovalently into biomembranes.
- Differences in the size, shape, charge, hydrophobicity, and reactivity of the side chains of the 20 common amino acids determine the chemical and structural properties of proteins (see Figure 2-14). The three general categories into which the side chains fall are hydrophobic, hydrophilic (basic, acidic, polar), and special (see Figure 2-14). It is helpful to remember which amino acids fall into each of these categories.
- The bases in the nucleotides composing DNA and RNA are carbon- and nitrogen-containing rings attached to a pentose sugar. They form two groups: the purines, with two rings-adenine (A) and guanine (G)-and the pyrimidines, with one ring-cytosine (C), thymine (T), and uracil (U) (see Figure 2-17). A, G, T, and C are found in DNA, and A, G, U , and C are found in RNA.
- Glucose and other hexoses can exist in three forms: an open-chain linear structure, a six-member (pyranose) ring, and a five-member (furanose) ring (see Figure 2-18). In biological systems, the pyranose form of D-glucose predominates.
- Glycosidic bonds are formed between either the $\alpha$ or the $\beta$ anomer of one sugar and a hydroxyl group on another
![li97857775e0](li97857775e0.jpg)
an unsaturated one with 18 C atoms. In saturated fatty acids, the hydrocarbon chain is often linear; the cis double bond in oleate creates a rigid kink in the hydrocarbon chain.
sugar, leading to formation of disaccharides and other polysaccharides (see Figure 2-19).
- Phospholipids are amphipathic molecules with a hydrophobic tail (often two fatty acyl chains) connected by a small organic molecule (often glycerol) to a hydrophilic head (see Figure 2-20).
- The long hydrocarbon chain of a fatty acid may be saturated (containing no carbon-carbon double bonds) or unsaturated (containing one or more double bonds). Fatty substances such as butter that have primarily saturated fatty acyl chains tend to be solid at room temperature, whereas unsaturated fats with cis double bonds have kinked chains that cannot pack closely together and so tend to be liquids at room temperature.


### 2.3 Chemical Reactions and Chemical Equilibrium

We now shift our discussion to chemical reactions in which bonds, primarily covalent bonds in reactant chemicals, are broken and new bonds are formed to generate reaction products. At any one time, several hundred different kinds of chemical reactions are occurring simultaneously in every cell, and many chemicals can, in principle, undergo multiple chemical reactions. Both the extent to which reactions can proceed and the rate at which they take place determine the chemical composition of cells. In this section, we discuss the concepts of equilibrium and steady state as well as dissociation constants and pH . These concepts will arise again and again throughout this text, so it is important for you to be familiar with them. In Section 2.4, we discuss how energy influences the extents and rates of chemical reactions.

## A Chemical Reaction Is in Equilibrium When the Rates of the Forward and Reverse Reactions Are Equal

When reactants first mix together-before any products have been formed-the rate of the forward reaction to form products is determined in part by the reactants' initial concentrations, which determine the likelihood of reactants bumping into one another and reacting (Figure 2-22). As the reaction products accumulate, the concentration of each reactant decreases, and so does the forward reaction rate. Meanwhile, some of the product molecules begin to participate in the reverse reaction, which re-forms the reactants. The ability of a reaction to go "backward" is called microscopic reversibility. The reverse reaction is slow at first but speeds up as the concentration of product increases. Eventually, the rates of the forward and reverse reactions become equal, so that the concentrations of reactants and products stop changing. The system is then said to be in chemical equilibrium (plural, equilibria).

The ratio of the concentrations of the products to the concentrations of the reactants when they reach equilibrium, called the equilibrium constant $\left(K_{\text {eq }}\right)$, is a fixed value. Thus $K_{\text {eq }}$ provides a measure of the extent to which a reaction occurs by the time it reaches equilibrium. The rate of a chemical reaction can be increased by a catalyst, but a catalyst does not change the equilibrium constant (see Section 2.4). A catalyst accelerates the making and breaking of covalent bonds but itself is not permanently changed during a reaction.
![li98857775e0](li98857775e0.jpg)

FIGURE 2-22 Time dependence of the rates of a chemical reaction. The forward and reverse rates of a reaction depend in part on the initial concentrations of reactants and products. The net forward reaction rate slows as the concentration of reactants decreases, whereas the net reverse reaction rate increases as the concentration of products increases. At equilibrium, the rates of the forward and reverse reactions are equal, and the concentrations of reactants and products remain constant.

## The Equilibrium Constant Reflects the Extent of a Chemical Reaction

For any chemical reaction, $K_{\text {eq }}$ depends on the chemical nature of the reactants and products, the temperature, and the pressure (particularly in reactions involving gases). Under standard physical conditions ( $25^{\circ} \mathrm{C}$ and 1 atm pressure for biological systems), $K_{\text {eq }}$ is always the same for a given reaction, whether or not a catalyst is present.

For the general reaction with three reactants and three products,

$$
\mathrm{aA}+\mathrm{bB}+\mathrm{cC} \rightleftharpoons \mathrm{zZ}+\mathrm{yY}+\mathrm{xX}
$$

where capital letters represent particular molecules or atoms and lowercase letters represent the number of each in the reaction, the formula for the equilibrium constant is given by

$$
K_{\mathrm{eq}}=\frac{[\mathrm{X}]^{a}[\mathrm{Y}]^{b}[\mathrm{Z}]^{a}}{[\mathrm{~A}]^{a}[\mathrm{~B}]^{b}[\mathrm{C}]^{c}}
$$

where brackets denote the concentrations of the molecules. In Equation 2-2, the concentrations of reactants and products are those present at equilibrium. The rate of the forward reaction (left to right in Equation 2-1) is

$$
\text { Rate }_{\text {forward }}=k_{t}[\mathrm{~A}]^{a}[\mathrm{~B}]^{b}[\mathrm{C}]^{c}
$$

where $k_{t}$ is the rate constant for the forward reaction. Similarly, the rate of the reverse reaction (right to left in Equation 2-1) is

$$
\text { Rate }_{\text {reverse }}=k_{r}[\mathrm{X}]^{a}[\mathrm{Y}]^{b}[\mathrm{Z}]^{a}
$$

where $k_{r}$ is the rate constant for the reverse reaction. These reaction rate equations apply whether or not the reaction has reached equilibrium. It is important to remember that the forward and reverse rates of a reaction can change because of changes in reactant or product concentrations, yet at the same time the forward and reverse rate constants do not change; hence the name "constant." Confusing rates and rate constants is a common error. At equilibrium the forward and reverse rates are equal, so Rate $_{\text {forward }} /$ Rate $_{\text {reverse }}=1$. By rearranging these equations, we can express the equilibrium constant as the ratio of the rate constants:

$$
K_{\mathrm{eq}}=\frac{k_{t}}{k_{r}}
$$

The concept of $K_{\text {eq }}$ is particularly helpful when we want to think about the energy that is released or absorbed when a chemical reaction occurs. We will discuss this concept in considerable detail in Section 2.4.

## Chemical Reactions in Cells Are at Steady State

Under appropriate conditions and given sufficient time, a single biochemical reaction carried out in a test tube eventually reaches equilibrium, at which the concentrations of reactants and products do not change with time because the

(a) Test tube equilibrium concentrations

$$
\mathrm{AAA} \rightleftharpoons \mathrm{BBB}
$$

(b) Intracellular steady-state concentrations

$$
\mathrm{AA} \rightleftharpoons \mathrm{BBB}
$$

FIGURE 2-23 Comparison of reactions at equilibrium and at steady state. (a) In the test tube, a biochemical reaction $(\mathrm{A} \rightarrow \mathrm{B})$ eventually reaches equilibrium, at which the rates of the forward and reverse reactions are equal (as indicated by the reaction arrows of equal length). (b) In metabolic pathways within cells, the product B is commonly consumed-in this example, by conversion to C. A pathway of linked reactions is at steady state when the rate of formation of the intermediates (e.g., B) equals their rate of consumption. As indicated by the unequal length of the arrows, the individual reversible reactions constituting a metabolic pathway do not reach equilibrium. Moreover, the concentrations of the intermediates at steady state can differ from what they would be at equilibrium.
rates of the forward and reverse reactions are equal. Within cells, however, many reactions are linked in pathways in which a product of one reaction is not simply reconverted via a reverse reaction to the reactants. For example, the product of one reaction might serve as a reactant in another, or it might be pumped out of the cell. In this more complex situation, the original reaction can never reach equilibrium because some of the products do not have a chance to be converted back to reactants. Nevertheless, in such nonequilibrium conditions, the rate of formation of a substance can be equal to the rate of its consumption, and as a consequence, the concentration of the substance remains constant over time. In such circumstances, the system of linked reactions for producing and consuming that substance is said to be in a steady state (Figure 2-23). One consequence of such linked reactions is that they prevent the accumulation of excess intermediates, protecting cells from the harmful effects of intermediates that are toxic at high concentrations. When the concentration of a product of an ongoing reaction is not changing over time, it might be a consequence of a state of equilibrium, or it might be a consequence of a steady state. In biological systems, when metabolite concentrations, such as blood glucose levels, are not changing with time-a condition called homeostasis-it is a consequence of a steady state rather than equilibrium.

## Dissociation Constants of Binding Reactions Reflect the Affinity of Interacting Molecules

The concept of equilibrium also applies to the binding of one molecule to another without covalent changes to either molecule. Many important cellular processes depend on such binding "reactions," which involve the making and breaking of various noncovalent interactions rather than covalent bonds, as discussed above. A common example is the binding
of a ligand (e.g., the hormone insulin or adrenaline) to its receptor on the surface of a cell, which triggers an intracellular signaling pathway (see Chapter 15). Another example is the binding of a protein to a specific sequence of bases in a molecule of DNA, which frequently causes the expression of a nearby gene to increase or decrease (see Chapter 9). If the equilibrium constant for a binding reaction is known, the stability of the resulting complex can be predicted.

To illustrate the general approach for determining the concentration of noncovalently associated complexes, let's calculate the extent to which a protein (P) is bound to DNA (D), forming a protein-DNA complex (PD):

$$
\mathrm{P}+\mathrm{D} \rightleftharpoons \mathrm{PD}
$$

Most commonly, binding reactions are described in terms of the dissociation constant $\left(K_{\mathrm{d}}\right)$, which is the reciprocal of the equilibrium constant. For this binding reaction, the dissociation constant is calculated from the concentrations of the three components when they are at equilibrium by

$$
K_{\mathrm{d}}=\frac{[\mathrm{P}][\mathrm{D}]}{[\mathrm{PD}]}
$$

It is worth noting that in such a binding reaction, when half of the DNA is bound to the protein ([PD] $=[\mathrm{D}]$ ), the concentration of P is equal to $K_{\mathrm{d}}$. The lower the $K_{\mathrm{d}}$, the lower the concentration of P needed to bind to half of D. In other words, the lower the $K_{\mathrm{d}}$, the tighter the binding (the higher the affinity) of P for D .

Typically, a protein's binding to a specific DNA sequence exhibits a $K_{\mathrm{d}}$ of $10^{-10} \mathrm{M}$, where M symbolizes molarity, or moles per liter ( $\mathrm{mol} / \mathrm{L}$ ). To relate the magnitude of this dissociation constant to the intracellular ratio of bound to unbound DNA, let's consider the simple example of a bacterial cell having a volume of $1.5 \times 10^{-15} \mathrm{~L}$ and containing 1 molecule of DNA and 10 molecules of the DNA-binding protein P. In this case, given a $K_{\mathrm{d}}$ of $10^{-10} \mathrm{M}$ and the total concentration of the P in the cell $\left(\sim 111 \times 10^{-10} \mathrm{M}\right.$, about a hundredfold higher than the $K_{\mathrm{d}}$ ), 99 percent of the time this specific DNA sequence will have a molecule of protein bound to it and 1 percent of the time it will not, even though the cell contains only 10 molecules of the protein! Clearly P and D have a high affinity for each other and bind tightly, as reflected by the low value of the dissociation constant for their binding reaction. For protein-protein and protein-DNA binding, $K_{\mathrm{d}}$ values of $\sim 10^{-9} \mathrm{M}$ (nanomolar) are considered to be tight, $\sim 10^{-6} \mathrm{M}$ (micromolar) modestly tight, and $\sim 10^{-3} \mathrm{M}$ (millimolar) relatively weak.

A large biological macromolecule, such as a protein, can have multiple binding surfaces for binding several molecules simultaneously (Figure 2-24). In some cases, these binding reactions are independent, with their own distinct $K_{\mathrm{d}}$ values that are independent of each other. In other cases, binding of a molecule at one site on a macromolecule can change the three-dimensional shape, or conformation, of a distant site, thus altering the binding interactions of that distant site with some other molecule. The modifications

![li99857775e0](li99857775e0.jpg)

FIGURE 2-24 Macromolecules can have distinct binding sites for multiple ligands. A large macromolecule (e.g., a protein, blue) with three distinct binding sites (A-C) is shown; each of the three binding sites exhibit molecular complementarity to three different binding partners (ligands $\mathrm{A}-\mathrm{C}$ ) with distinct dissociation constants $\left(K_{\text {dd }-\mathrm{C}}\right)$.
of amino acid side chains-mentioned above-often contribute to the molecular shapes required for such binding interactions. These covalent and noncovalent binding reactions are important mechanisms by which one molecule can alter, and thus regulate, the structure and binding activity of another. We examine this regulatory mechanism in more detail in Chapter 3.

## Biological Fluids Have Characteristic pH Values

The solvent inside cells and in all extracellular fluids is water. An important characteristic of any aqueous solution is the concentration of positively charged hydrogen ions $\left(\mathrm{H}^{+}\right)$and negatively charged hydroxyl ions $\left(\mathrm{OH}^{-}\right)$. Because these ions are the dissociation products of $\mathrm{H}_{2} \mathrm{O}$, they are constituents of all living systems, and they are liberated by many reactions that take place between molecules within cells. These ions can also be transported into or out of cells, as when highly acidic gastric juice is secreted by cells lining the walls of the stomach.

When a water molecule dissociates, one of its polar $\mathrm{H}-\mathrm{O}$ bonds breaks. The resulting hydrogen ion, referred to as a proton, has a short lifetime as a free ion and quickly combines with a water molecule to form a hydronium ion $\left(\mathrm{H}_{3} \mathrm{O}^{+}\right)$. For convenience, we refer to the concentration of hydrogen ions in a solution, $\left[\mathrm{H}^{+}\right]$, even though this quantity really represents the concentration of hydronium ions, $\left[\mathrm{H}_{3} \mathrm{O}^{+}\right]$. Dissociation of $\mathrm{H}_{2} \mathrm{O}$ generates one $\mathrm{OH}^{-}$ion along with each $\mathrm{H}^{+}$. The dissociation of water is a reversible reaction:

$$
\mathrm{H}_{2} \mathrm{O} \rightleftharpoons \mathrm{H}^{+}+\mathrm{OH}^{-}
$$

At $25^{\circ} \mathrm{C},\left[\mathrm{H}^{+}\right]\left[\mathrm{OH}^{-}\right]=10^{-14} \mathrm{M}^{2}$, so that in pure water, $\left[\mathrm{H}^{+}\right]=\left[\mathrm{OH}^{-}\right]=10^{-5} \mathrm{M}$.

The concentration of hydrogen ions in a solution is expressed conventionally as its pH , defined as the negative $\log$ of the hydrogen ion concentration. The pH of pure water at $25^{\circ} \mathrm{C}$ is 7 :

$$
\mathrm{pH}=-\log \left[\mathrm{H}^{+}\right]=\log \frac{1}{\left[\mathrm{H}^{+}\right]}=\log \frac{1}{10^{-7}}=7
$$

It is important to keep in mind that a one-unit difference in pH represents a tenfold difference in the concentration of protons. On the pH scale, 7.0 is considered neutral: pH values below 7.0 indicate acidic solutions (higher $\left[\mathrm{H}^{+}\right]$), and values above 7.0 indicate basic, or alkaline, solutions (Figure 2-25). For instance, gastric juice, which is rich in hydrochloric acid $(\mathrm{HCl})$, has a pH of about 1 . Its $\left[\mathrm{H}^{+}\right]$is roughly 1-million-fold greater than that of cytoplasm, which has a pH of about 7.2-7.4.

Although the cytosol of cells normally has a pH of about 7.2 , the interior of certain organelles in eukaryotic cells (see Chapter 1) can have a much lower pH . The internal (luminal) fluid in lysosomes, for example, has a pH of about 4.5 .
![li100857775e0](li100857775e0.jpg)

FIGURE 2-25 Some $\mathbf{p H}$ values for common solutions. The pH of an aqueous solution is the negative log of the hydrogen ion concentration. The pH values for most intracellular and extracellular biological fluids are near 7 and are carefully regulated to permit the proper functioning of cells, organelles, and cellular secretions. The pH values for solutions of ammonia and hydrochloric acid are for one normal ( 1 N ) solutions.

The many degradative enzymes within lysosomes function optimally in an acidic environment, whereas their action is inhibited in the near neutral pH environment of the cytoplasm. As this example illustrates, maintenance of a particular pH is essential for the proper functioning of some cellular structures. On the other hand, dramatic shifts in cellular pH may play an important role in controlling cellular activity. For example, the pH of the cytoplasm of an unfertilized egg of the sea urchin, an aquatic animal, is 6.6. Within 1 minute of fertilization, however, the pH rises to 7.2 ; that is, the $\left[\mathrm{H}^{+}\right]$ decreases to about one-fourth its original value, a change that is necessary for subsequent growth and division of the egg.

## Hydrogen Ions Are Released by Acids and Taken Up by Bases

In general, an acid is any molecule, ion, or chemical group that tends to release a hydrogen ion $\left(\mathrm{H}^{+}\right)$, such as the carboxyl group $(-\mathrm{COOH})$, which tends to dissociate to form the negatively charged carboxylate ion $\left(-\mathrm{COO}^{-}\right)$; or hydrochloric acid $(\mathrm{HCl})$. Conversely, a base is any molecule, ion, or chemical group that readily combines with an $\mathrm{H}^{+}$, such as the hydroxyl ion $\left(\mathrm{OH}^{-}\right)$; ammonia $\left(\mathrm{NH}_{3}\right)$, which forms an ammonium ion $\left(\mathrm{NH}_{4}{ }^{+}\right)$; or the amino group $\left(-\mathrm{NH}_{2}\right)$.

When an acid is added to an aqueous solution, the $\left[\mathrm{H}^{+}\right]$ increases, and the pH goes down. Conversely, when a base is added to a solution, the $\left[\mathrm{H}^{+}\right]$decreases, and the pH goes up. Because $\left[\mathrm{H}^{+}\right]\left[\mathrm{OH}^{-}\right]=10^{-14} \mathrm{M}^{2}$, any increase in $\left[\mathrm{H}^{+}\right]$is coupled with a commensurate decrease in $\left[\mathrm{OH}^{-}\right]$, and vice versa.

Many biological molecules contain both acidic and basic groups. For example, in neutral solutions $(\mathrm{pH}=7.0)$, many amino acids exist predominantly in the doubly ionized form, in which the carboxyl group has lost a proton and the amino group has accepted one:

$$
\mathrm{H}-\stackrel{\mathrm{NH}_{3}^{+}}{\stackrel{\mathrm{COO}^{-}}{\rightleftarrows}} \mathrm{COO}^{-}
$$

where R represents the uncharged side chain. Such a molecule, containing an equal number of positive and negative ions, is called a zwitterion. Zwitterions, having no net charge, are neutral. At extreme pH values, only one of these two ionizable groups of an amino acid is charged: the $-\mathrm{NH}_{2}{ }^{+}$at low pH and the $-\mathrm{COO}^{-}$at high pH .

The dissociation reaction for an acid (or acid group in a larger molecule) HA can be written as $\mathrm{HA} \rightleftharpoons \mathrm{H}^{+}+\mathrm{A}^{-}$. The equilibrium constant for this reaction, denoted $K_{\mathrm{a}}$ (the subscript $a$ stands for "acid"), is defined as $K_{\mathrm{a}}=\left[\mathrm{H}^{+}\right]\left[\mathrm{A}^{-}\right] /$ [HA]. Taking the logarithm of both sides and rearranging the result yields a very useful relation between the equilibrium constant and pH :

$$
\mathrm{pH}=\mathrm{p} K_{\mathrm{a}}+\log \frac{\left[\mathrm{A}^{-}\right]}{[\mathrm{HA}]}
$$

where $\mathrm{p} K_{\mathrm{a}}$ equals $-\log K_{\mathrm{a}}$.
![li101857775e0](li101857775e0.jpg)

FIGURE 2-26 The relationship between $\mathrm{pH}, \mathrm{p} K_{\mathrm{a}}$, and the dissociation of an acid. As the pH of a solution of carbonic acid rises from 0 to 8.5 , the percentage of the compound in the undissociated, or un-ionized, form $\left(\mathrm{H}_{2} \mathrm{CO}_{3}\right)$ decreases from 100 percent and that of the ionized form increases from 0 percent. When the $\mathrm{pH}(6.4)$ is equal to the acid's $\mathrm{p} K_{\mathrm{a}}$, half of the carbonic acid has ionized. When the pH rises to above 8 , virtually all of the acid has ionized to the bicarbonate form $\left(\mathrm{HCO}_{3}{ }^{-}\right)$.

From this expression, commonly known as the Henderson-Hasselbalch equation, it can be seen that the $\mathrm{p} K_{\mathrm{a}}$ of any acid is equal to the pH at which half the molecules are dissociated and half are neutral (undissociated). This is because when $\left[\mathrm{A}^{-}\right]=[\mathrm{HA}]$, then $\log \left(\left[\mathrm{A}^{-}\right] /[\mathrm{HA}]\right)=0$, and thus $\mathrm{p} K_{\mathrm{a}}=\mathrm{pH}$. The Henderson-Hasselbalch equation allows us to calculate the degree of dissociation of an acid-that is, the ratio of dissociated and undissociated forms-if both the pH of the solution and the $\mathrm{p} K_{\mathrm{a}}$ of the acid are known. Experimentally, by measuring the $\left[\mathrm{A}^{-}\right]$and $[\mathrm{HA}]$ as a function of the solution's pH , one can calculate the $\mathrm{p} K_{\mathrm{a}}$ of the acid and thus the equilibrium constant $K_{\mathrm{a}}$ for the dissociation reaction (Figure 2-26). Knowing the $\mathrm{p} K_{\mathrm{a}}$ of a molecule not only provides an important description of its properties, but also allows us to exploit these properties to manipulate the acidity of an aqueous solution and to understand how biological systems control this critical characteristic of their aqueous fluids.

## Buffers Maintain the pH of Intracellular and Extracellular Fluids

A living, actively metabolizing cell must maintain a constant pH in the cytoplasm of about $7.2-7.4$, and it must do so even as its metabolism is producing many acids. Cells have a reservoir of weak bases and weak acids, called buffers, which ensure that the cell's cytoplasmic pH remains relatively constant despite small fluctuations in the amounts of $\mathrm{H}^{+}$or $\mathrm{OH}^{-}$being generated by metabolism or by the uptake or secretion of molecules and ions by the cell. Buffers do this by "soaking up" excess $\mathrm{H}^{+}$or $\mathrm{OH}^{-}$when these ions are added to the cell or are produced by metabolism. As we shall see below, buffers are most effective at preventing changes in pH when the pH of the solution is similar to the $\mathrm{p} K_{\mathrm{a}}$ of the buffer.

![li102857775e0](li102857775e0.jpg)

FIGURE 2-27 The titration curve of the buffer acetic acid $\left(\mathrm{CH}_{3} \mathrm{COOH}\right)$. The $\mathrm{p} K_{\mathrm{a}}$ for the dissociation of acetic acid to hydrogen and acetate ions is 4.75 . At this pH , half the acid molecules are dissociated. Because pH is measured on a logarithmic scale, the solution changes from 91 percent $\mathrm{CH}_{3} \mathrm{COOH}$ at pH 3.75 to 9 percent $\mathrm{CH}_{3} \mathrm{COOH}$ at pH 5.75. The acid has maximum buffering capacity in this pH range.

If additional acid (or base) is added to a buffered solution whose pH is equal to the $\mathrm{p} K_{\mathrm{a}}$ of the buffer $\left([\mathrm{HA}]=\left[\mathrm{A}^{-}\right]\right)$, the pH of the solution changes, but it changes less than it would if the buffer had not been present. This is because protons released by the added acid are taken up by the ionized form of the buffer $\left(\mathrm{A}^{-}\right)$; likewise, hydroxyl ions generated by the addition of a base are neutralized by protons released by the undissociated buffer (HA). The capacity of a buffer or any other substance to release hydrogen ions or take them up depends partly on the extent to which the substance has already taken up or released protons, which in turn depends on the pH of the solution relative to the $\mathrm{p} K_{\mathrm{a}}$ of the substance. The ability of a buffer to minimize changes in pH , its buffering capacity, depends on the concentration of the buffer and the relationship between its $\mathrm{p} K_{\mathrm{a}}$ value and the pH , which is expressed by the Henderson-Hasselbalch equation.

The titration curve for acetic acid shown in Figure 2-27 illustrates the effect of pH on the fraction of molecules in the un-ionized (HA) and ionized forms $\left(\mathrm{A}^{-}\right)$. When the pH is equal to the $\mathrm{p} K_{\mathrm{a}}$, half of the acetic acid is dissociated (dashed lines). At one pH unit below the $\mathrm{p} K_{\mathrm{a}}$ of an acid, 91 percent of the molecules are in the HA form; at one pH unit above the $\mathrm{p} K_{\mathrm{a}}, 91$ percent are in the $\mathrm{A}^{-}$form. At pH values more than one unit above or below the $\mathrm{p} K_{\mathrm{a}}$ (unshaded regions in Figure 2-27), the buffering capacity of weak acids and bases declines rapidly. In other words, the addition of the same number of moles of base-for example, hydroxyl ions added as sodium hydroxide $(\mathrm{NaOH})$ —to a solution containing a mixture of HA and $\mathrm{A}^{-}$that is at a pH near the $\mathrm{p} K_{\mathrm{a}}$ will cause less of a pH change than it would if the HA and $\mathrm{A}^{-}$were not present or if the pH were far from the $\mathrm{p} K_{\mathrm{a}}$ value.

All biological systems contain one or more buffers. Phosphate ions, the ionized forms of phosphoric acid, are present in considerable quantities in cells and are important in maintaining, or buffering, the pH of the cytoplasm. Phosphoric acid $\left(\mathrm{H}_{3} \mathrm{PO}_{4}\right)$ has three protons that are capable of dissociating, but they do not dissociate simultaneously. Loss of each proton can be described by a discrete dissociation reaction and $\mathrm{p} K_{\mathrm{a}}$, as shown in Figure 2-28. When hydroxyl ions are added to a solution of phosphoric acid, the pH change is much less steep at pH values near the three $\mathrm{p} K_{\mathrm{a}}$ values (shaded region) than when the pH of the solution is not similar to any of the $\mathrm{p} K_{\mathrm{a}} \mathrm{s}$. The titration curve for phosphoric acid shows that the $\mathrm{p} K_{\mathrm{a}}$ for the dissociation of the second proton is 7.2 . Thus, at pH 7.2 , about 50 percent of cellular phosphate is $\mathrm{H}_{2} \mathrm{PO}_{4}{ }^{-}$and about 50 percent is $\mathrm{HPO}_{4}{ }^{2-}$ according to the Henderson-Hasselbalch equation. For this reason, phosphate is an excellent buffer at pH values around 7.2 , the approximate pH of the cytoplasm of cells, and at pH 7.4 , the pH of human blood. The amino (lysine), guanidinium (arginine), and carboxylate (aspartate, glutamate) portions of amino acid side chains of proteins as well as the amino and carboxylate groups at the N - and C-termini of proteins can also bind and release protons. Thus proteins that are present in high concentrations inside of cells and in many extracellular fluids can themselves serve as buffers.
![li103857775e0](li103857775e0.jpg)

FIGURE 2-28 The titration curve of phosphoric acid $\left(\mathrm{H}_{3} \mathrm{PO}_{4}\right)$, a common buffer in biological systems. This biologically ubiquitous molecule has three hydrogen atoms that dissociate at different pH values; thus phosphoric acid has three $\mathrm{p} K_{\mathrm{a}}$ values, as noted on the graph. The shaded areas denote the pH ranges-within one pH unit of the three $\mathrm{p} K_{\mathrm{a}}$ values-where the buffering capacity of phosphoric acid is high. In these regions, the addition of an acid (or base) will cause relatively small changes in the pH .

## KEY CONCEPTS OF SECTION 2.3

## Chemical Reactions and Chemical Equilibrium

- A chemical reaction is at equilibrium when the rate of the forward reaction is equal to the rate of the reverse reaction, and thus there is no net change in the concentration of the reactants or products.
- The equilibrium constant $K_{\text {eq }}$ of a reaction reflects the ratio of products to reactants at equilibrium and thus is a measure of the extent of the reaction and the relative stabilities of the reactants and products.
- The $K_{\text {eq }}$ depends on the temperature, pressure, and chemical properties of the reactants and products but is independent of the reaction rate and of the initial concentrations of reactants and products.
- For any reaction, the equilibrium constant $K_{\text {eq }}$ equals the ratio of the forward rate constant to the reverse rate constant $\left(k_{\mathrm{f}} / k_{\mathrm{r}}\right)$. The rates of conversion of reactants to products and vice versa depend on the rate constants and the concentrations of the reactants or products.
- Within cells, the linked reactions in metabolic pathways generally are not at equilibrium, but rather at steady state, at which the rate of formation of the intermediates equals their rate of consumption (see Figure 2-23) and thus the concentrations of the intermediates are not changing.
- The dissociation constant $K_{\mathrm{d}}$ for the noncovalent binding of two molecules is a measure of the stability of the complex formed between the molecules (e.g., ligand-receptor or pro-tein-DNA complexes). $K_{\mathrm{d}}$ values of $\sim 10^{-9} \mathrm{M}$ (nanomolar) are considered to be tight, $\sim 10^{-6} \mathrm{M}$ (micromolar) modestly tight, and $\sim 10^{-3} \mathrm{M}$ (millimolar) relatively weak.
- The pH is the negative logarithm of the concentration of hydrogen ions $\left(-\log \left[\mathrm{H}^{+}\right]\right)$. The pH of the cytoplasm is normally about $7.2-7.4$, whereas the interior of lysosomes has a pH of about 4.5 .
- Acids release protons $\left(\mathrm{H}^{+}\right)$, and bases bind them.
- Buffers are mixtures of a weak acid (HA) and its corresponding base form $\left(\mathrm{A}^{-}\right)$, which minimize the change in pH of a solution when an acid or base is added. Biological systems use various buffers to maintain their pH within a very narrow range.


### 2.4 Biochemical Energetics

The transformation of energy, its storage, and its use are central to the economy of the cell. Energy may be defined as the ability to do work, a concept that is as applicable to cells as to automobile engines and electric power plants. The energy stored within chemical bonds can be harnessed to support chemical work and the physical movements of cells.

In this section, we review how energy influences the extents of chemical reactions (chemical thermodynamics) and the rates of chemical reactions (chemical kinetics).

## Several Forms of Energy Are Important in Biological Systems

There are two principal forms of energy: kinetic and potential. Kinetic energy is the energy of movement-the motion of molecules, for example. Potential energy is stored energy-the energy stored in covalent bonds, for example. Potential energy plays a particularly important role in the energy economy of cells.

Thermal energy, or heat, is a form of kinetic energy-the energy of the motion of molecules. For heat to do work, it must flow from a region of higher temperature-where the average speed of molecular motion is greater-to one of lower temperature. Although differences in temperature can exist between the internal and external environments of cells, these thermal gradients do not usually serve as the source of energy for cellular activities. The thermal energy in warm-blooded animals, which have evolved a mechanism for thermoregulation, is used chiefly to maintain constant organismal temperatures. This is an important homeostatic function because the rates of many cellular activities are temperature dependent. For example, cooling mammalian cells from their normal body temperature of $37^{\circ} \mathrm{C}$ to $4^{\circ} \mathrm{C}$ can virtually "freeze" or stop many cellular processes (e.g., intracellular membrane movements).

Radiant energy, the kinetic energy of photons, or waves of light, is critical to biology. Radiant energy can be converted to thermal energy, for instance, when light is absorbed by molecules and the energy is converted to molecular motion. Radiant energy absorbed by molecules can also change the electronic structure of the molecules by moving electrons into higher-energy orbitals, whence it can later be recovered to perform work. For example, during photosynthesis, light energy absorbed by pigment molecules such as chlorophyll is subsequently converted into the energy of chemical bonds (see Chapter 12).

Mechanical energy, a major form of kinetic energy in biology, usually results from the conversion of stored chemical energy. For example, changes in the lengths of cytoskeletal filaments generate forces that push or pull on membranes and organelles (see Chapters 17 and 18).

Electric energy-the energy of moving electrons or other charged particles-is yet another major form of kinetic energy, one with particular importance to membrane function, as in electrically active neurons (see Chapter 22).

Several forms of potential energy are biologically significant. Central to biology is chemical potential energy, the energy stored in the bonds connecting atoms in molecules. Indeed, most of the biochemical reactions described in this book involve the making or breaking of at least one covalent chemical bond. In general, energy must be expended to make covalent bonds in typical biomolecules, and energy is released when those bonds are broken. For example, the

high potential energy in the covalent bonds of glucose can be released by controlled enzymatic combustion in cells (see Chapter 12). This energy is harnessed by the cell to do many kinds of work.

A second biologically important form of potential energy is the energy in a concentration gradient. When the concentration of a substance on one side of a barrier, such as a membrane, is different from that on the other side, a concentration gradient exists. All cells form concentration gradients between their interior and the external fluids by selectively exchanging nutrients, waste products, and ions with their surroundings. Furthermore, the fluids within organelles in cells (e.g., mitochondria, lysosomes) frequently contain different concentrations of ions and other molecules than the cytoplasm; the concentration of protons within a lysosome, as we saw in the last section, is about 500 times that in the cytoplasm. Concentration gradients of protons across membranes are an important driver of energy production in mitochondria.

A third form of potential energy in cells is an electric potential-the energy of charge separation. For instance, there is a gradient of electric charge of about 200,000 volts per centimeter across the plasma membranes of virtually all cells. We discuss how concentration gradients and electric potential gradients are generated and maintained in Chapter 11 and how they are converted to chemical potential energy in Chapter 12.

## Cells Can Transform One Type of Energy into Another

According to the first law of thermodynamics, energy is neither created nor destroyed, but can be converted from one form to another. (In nuclear reactions, mass is converted to energy, but this is irrelevant in biological systems.) Energy conversions are very important in biology. In photosynthesis, for example, the radiant energy of light is transformed into the chemical potential energy of the covalent bonds between the atoms in a sucrose or starch molecule. In muscles and nerves, chemical potential energy stored in covalent bonds is transformed, respectively, into the kinetic energy of muscle contraction and the electric energy of neural transmission. In all cells, potential energy-released by breaking certain chemical bonds-is used to generate potential energy in the form of concentration and electric potential gradients. Similarly, energy stored in chemical concentration gradients or electric potential gradients is used to synthesize chemical bonds or to transport molecules from one side of a membrane to another to generate a concentration gradient. The latter process occurs during the transport of nutrients such as glucose into certain cells and the transport of many waste products out of cells.

Because all forms of energy are interconvertible, they can be expressed in the same units of measurement. Although the standard unit of energy is the joule, biochemists have traditionally used an alternative unit, the calorie ( 1 joule $=$ 0.239 calorie). A calorie is the amount of energy required
to raise the temperature of one gram of water by $1^{\circ} \mathrm{C}$. Throughout this book, we use the kilocalorie to measure energy changes ( $1 \mathrm{kcal}=1000 \mathrm{cal}$ ). When you read or hear about the "Calories" in food (note the capital C), the reference is almost always to kilocalories as defined here.

## The Change in Free Energy Determines If a Chemical Reaction Will Occur Spontaneously

Chemical reactions can be divided into two types, depending on whether energy is absorbed or released in the process. In an exergonic ("energy-releasing") reaction, the products contain less energy than the reactants. Exergonic reactions take place spontaneously. The liberated energy is usually released as heat (the energy of molecular motion) and generally results in a rise in temperature, as in the oxidation (burning) of wood. In an endergonic ("energy-absorbing") reaction, the products contain more energy than the reactants, and energy is absorbed during the reaction. If there is no external source of energy to drive an endergonic reaction, it cannot take place. Endergonic reactions are responsible for the ability of the instant cold packs often used to treat injuries to rapidly cool below room temperature. Crushing the pack mixes the reactants, initiating the reaction.

A fundamentally important concept in understanding if a reaction is exergonic or endergonic, and therefore if it occurs spontaneously or not, is free energy (G), or Gibbs free energy, named after J. W. Gibbs. Gibbs, who received the first PhD in engineering in America in 1863, showed that "all systems change in such a way that free energy [G] is minimized." In other words, a chemical reaction occurs spontaneously when the free energy of the products is lower than the free energy of the reactants. In the case of a chemical reaction, reactants $\rightleftharpoons$ products, the free-energy change, $\Delta G$, is given by

$$
\Delta G=G_{\text {products }}-G_{\text {reactants }}
$$

The relation of $\Delta G$ to the direction of any chemical reaction can be summarized in three statements:

- If $\Delta G$ is negative, the forward reaction will tend to occur spontaneously, and energy usually will be released as the reaction takes place (exergonic reaction) (Figure 2-29). A reaction with a negative $\Delta G$ is referred to as thermodynamically favorable.
- If $\Delta G$ is positive, the forward reaction will not occur spontaneously; energy will have to be added to the system in order to force the reactants to become products (endergonic reaction).
- If $\Delta G$ is zero, both forward and reverse reactions will occur at equal rates, and there will be no spontaneous net conversion of reactants to products, or vice versa; the system is at equilibrium.

By convention, the standard free-energy change of a reaction $\left(\Delta G^{\circ \prime}\right)$ is the value of the change in free energy at

![li104857775e0](li104857775e0.jpg)

FIGURE 2-29 Changes in the free energy ( $\Delta G$ ) of exergonic and endergonic reactions. (a) In exergonic reactions, the free energy of the products is less than that of the reactants. Consequently, these reactions occur spontaneously, and energy is released as the reactions proceed. (b) In endergonic reactions, the free energy of the products is greater than that of the reactants, and these reactions do not occur spontaneously. An external source of energy must be supplied if the reactants are to be converted into products.
$298 \mathrm{~K}\left(25^{\circ} \mathrm{C}\right), 1 \mathrm{~atm}$ pressure, pH 7.0 (as in pure water), and initial concentrations of 1 M for all reactants and products except protons, which are kept at $10^{-7} \mathrm{M}(\mathrm{pH} 7.0)$. Most biological reactions differ from these standard conditions, particularly in the concentrations of reactants, which are normally less than 1 M .

The free energy of a chemical system can be defined as $G=H-T S$, where $H$ is the bond energy, or enthalpy, of the system; $T$ is its temperature in degrees Kelvin (K); and $S$ is the entropy, a measure of its randomness or disorder. According to the second law of thermodynamics, the natural tendency of any isolated system is to become more disordered-that is, for entropy to increase. A reaction can occur spontaneously only if the combined effects of changes in enthalpy and entropy lead to a lower $\Delta G$. That is, if temperature remains constant, a reaction proceeds spontaneously only if the free-energy change, $\Delta G$, in the following equation is negative:

$$
\Delta G=\Delta H-T \Delta S
$$

In an exothermic ("heat-releasing") chemical reaction, $\Delta H$ is negative. In an endothermic ("heat-absorbing") reaction, $\Delta H$ is positive. The combined effects of the changes in the enthalpy and entropy determine if the $\Delta G$ for a reaction is positive or negative, and thus if the reaction occurs spontaneously. An exothermic reaction $(\Delta H<0)$, in which entropy increases $(\Delta S>0)$, occurs spontaneously $(\Delta G<0)$. An endothermic reaction $(\Delta H>0)$ will occur spontaneously
if $\Delta S$ increases enough so that the $T \Delta S$ term can overcome the positive $\Delta H$.

Many biological reactions lead to an increase in order and thus a decrease in entropy $(\Delta S<0)$. An obvious example is the reaction that links amino acids to form a protein. A solution of protein molecules has a lower entropy than does a solution of the same amino acids unlinked because the free movement of any amino acid is more restricted (greater order) when it is bound into a long chain than when it is not. Thus, when cells synthesize polymers such as proteins from their constituent monomers, the polymerizing reaction will be spontaneous only if the cells can efficiently transfer energy to both generate the bonds that hold the monomers together and overcome the loss in entropy that accompanies polymerization. Often cells accomplish this feat by "coupling" such synthetic, entropy-lowering reactions with independent reactions that have a very highly negative $\Delta G$, such as the hydrolysis of nucleoside triphosphates (see below). In this way, cells can convert sources of energy in their environment into the highly organized structures and metabolic pathways that are essential for life.

The actual change in free energy during a reaction is influenced by temperature, pressure, and the initial concentrations of reactants and products, so it usually differs from the standard free-energy change $\Delta G^{\circ \prime}$. Most biological reactions-like others that take place in aqueous solutionsare also affected by the pH of the solution. We can estimate free-energy changes for temperatures and initial concentrations that differ from the standard conditions by using the equation
$\Delta G=\Delta G^{\circ \prime}+R T \ln Q=\Delta G^{\circ \prime}+R T \ln \frac{[\text { products }]}{[\text { reactants }]}$
where $R$ is the gas constant of $1.987 \mathrm{cal} /($ degree $\cdot \mathrm{mol}), T$ is the temperature (in degrees Kelvin), and $Q$ is the initial ratio of products to reactants. For a reaction $\mathrm{A}+\mathrm{B} \rightleftharpoons \mathrm{C}$, in which two molecules combine to form a third, $Q$ in Equation 2-7 equals $[\mathrm{C}] /[\mathrm{A}][\mathrm{B}]$. In this case, an increase in the initial concentration of either [A] or [B] will result in a larger negative value for $\Delta G$ and thus drive the reaction toward spontaneous formation of C.

Regardless of the $\Delta G^{\circ \prime}$ of a particular biochemical reaction, it will proceed spontaneously within cells only if $\Delta G$ is negative given the intracellular concentrations of reactants and products. For example, the conversion of glyceraldehyde 3-phosphate (G3P) to dihydroxyacetone phosphate (DHAP), two intermediates in the breakdown of glucose,

$$
\mathrm{G} 3 \mathrm{P} \rightleftharpoons \mathrm{DHAP}
$$

has a $\Delta G^{\circ \prime}$ of $-1840 \mathrm{cal} / \mathrm{mol}$. If the initial concentrations of G3P and DHAP are equal, then $\Delta G=\Delta G^{\circ \prime}$ because $R T$ $\ln =0$; in this situation, the reversible reaction G3P $\rightleftharpoons$ DHAP will proceed spontaneously in the direction of DHAP formation until equilibrium is reached. However, if the initial [DHAP] is 0.1 M and the initial [G3P] is 0.001 M , with

other conditions standard, then $Q$ in Equation 2-7 equals $0.1 / 0.001=100$, giving a $\Delta G$ of $+887 \mathrm{cal} / \mathrm{mol}$. Under these conditions, the reaction will proceed in the direction of formation of G3P.

The $\Delta G$ of a reaction is independent of the reaction rate. Indeed, under normal physiological conditions, few, if any, of the biochemical reactions needed to sustain life would occur without some mechanism for increasing reaction rates. As we describe below and in more detail in Chapter 3, the rates of reactions in biological systems are usually determined by the activity of enzymes, the protein catalysts that accelerate the formation of products from reactants without altering the value of $\Delta G$.

## The $\Delta G^{\circ \prime}$ of a Reaction Can Be Calculated from Its $K_{\text {eq }}$

A chemical mixture at equilibrium is in a stable state of minimal free energy. For a system at equilibrium ( $\Delta G=0$, $Q=K_{\text {eq }}$ ) under standard conditions, we can write

$$
\Delta G^{\circ \prime}=-2.3 R T \log K_{\mathrm{eq}}=-1362 \log K_{\mathrm{eq}}
$$

(note the change to base 10 logarithms). Thus, if we determine the concentrations of reactants and products at equilibrium (i.e., the $K_{\text {eq }}$ ), we can calculate the value of $\Delta G^{\circ \prime}$. For example, the $K_{\text {eq }}$ for the interconversion of glyceraldehyde 3-phosphate to dihydroxyacetone phosphate (G3P $\rightleftharpoons$ DHAP) is 22.2 under standard conditions. Substituting this value into Equation 2-8, we can easily calculate the $\Delta G^{\circ \prime}$ for this reaction as $-1840 \mathrm{cal} / \mathrm{mol}$.

By rearranging Equation 2-8 and taking the antilogarithm, we obtain

$$
K_{\mathrm{eq}}=10^{-(\Delta G^{\circ} / 2.3 R T)}
$$

From this expression, it is clear that if $\Delta G^{\circ \prime}$ is negative, the exponent will be positive, and hence $K_{\text {eq }}$ will be greater than 1 . Therefore, at equilibrium there will be more products than reactants; in other words, the formation of products from reactants is favored. Conversely, if $\Delta G^{\circ \prime}$ is positive, the exponent will be negative, and $K_{\text {eq }}$ will be less than 1 . The relationship between $K_{\text {eq }}$ and $\Delta G^{\circ \prime}$ further emphasizes the influence of the relative free energies of reactants and products on the extent to which a reaction will occur spontaneously.

## The Rate of a Reaction Depends on the Activation Energy Necessary to Energize the Reactants into a Transition State

As a chemical reaction proceeds, reactants approach each other; some bonds begin to form while others begin to break. One way to think of the state of the molecules during this transition is that there are strains in the electronic configurations of the atoms and their bonds. The collection of atoms
moves from the relatively stable state of the reactants to this transient, intermediate, and higher-energy state during the course of the reaction (Figure 2-30). The state during a chemical reaction at which the system is at its highest energy level is called the transition state, and the collection of reactants in that state is called the transition-state intermediate. The energy needed to excite the reactants to this higher-energy state is called the activation energy of the reaction. The activation energy is usually represented by $\Delta G^{\ddagger}$, which is analogous to the representation of the change in Gibbs free energy $(\Delta G)$ already discussed. From the transition state, the collection of atoms can either release energy as the reaction products are formed or release energy as the atoms go "backward" and re-form the original reactants. The velocity $(V)$ at which products are generated from reactants during the reaction under a given set of conditions (temperature, pressure, reactant concentrations) will depend on the concentration of material in the transition state, which in turn will depend on the activation energy, and on the characteristic rate constant $(\nu)$ at which the material in the transition state is converted to products. The higher the activation energy, the lower the fraction of reactants that reach the transition state, and the slower the overall rate of the reaction. The relationship between the concentration of reactants, $v$, and $V$ is

$$
V=v[\text { reactants }] \times 10^{-(\Delta G^{\ddagger} / 2.3 R T)}
$$

From this equation, we can see that lowering the activation energy-that is, decreasing the free energy of the transition
![li105857775e0](li105857775e0.jpg)

FIGURE 2-30 Activation energy of uncatalyzed and catalyzed chemical reactions. This hypothetical reaction pathway (blue) depicts the changes in free energy, $G$, as a reaction proceeds. A reaction will take place spontaneously if the free energy $(G)$ of the products is less than that of the reactants $(\Delta G<0)$. However, all chemical reactions proceed through one (shown here) or more high-energy transition states, and the rate of a reaction is inversely proportional to the activation energy $\left(\Delta G^{\ddagger}\right)$, which is the difference in free energy between the reactants and the transition state. In a catalyzed reaction (red), the free energies of the reactants and products are unchanged, but the free energy of the transition state is lowered, thus increasing the velocity of the reaction.

state $\Delta G^{\ddagger}$-leads to an acceleration of the overall reaction rate $V$. A reduction in $\Delta G^{\ddagger}$ of $1.36 \mathrm{kcal} / \mathrm{mol}$ leads to a tenfold increase in the rate of the reaction, whereas a $2.72 \mathrm{kcal} /$ mol reduction increases the rate a hundredfold. Thus relatively small changes in $\Delta G^{\ddagger}$ can lead to large changes in the overall rate of the reaction.

Catalysts such as enzymes (discussed further in Chapter 3) accelerate reaction rates by lowering the relative energy of the transition state and thus the activation energy required to reach it (see Figure 2-30). The relative energies of reactants and products determine if a reaction is thermodynamically favorable (negative $\Delta G$ ), whereas the activation energy determines how rapidly products form-that is, the reaction kinetics. Thermodynamically favorable reactions will not occur at appreciable rates if the activation energies are too high.

## Life Depends on the Coupling of Unfavorable Chemical Reactions with Energetically Favorable Ones

Many processes in cells are energetically unfavorable $(\Delta G>0)$ and will not proceed spontaneously. Examples include the synthesis of DNA from nucleotides and the transport of a substance across the plasma membrane from a lower to a higher concentration. Cells can carry out an energyrequiring, or endergonic, reaction $\left(\Delta G_{1}>0\right)$ by coupling it to an energy-releasing, or exergonic, reaction ( $\Delta G_{2}<0$ ) if the sum of the two reactions has an overall net negative $\Delta G$.

Suppose, for example, that the reaction $\mathrm{A} \rightleftharpoons \mathrm{B}+\mathrm{X}$ has a $\Delta G$ of $+5 \mathrm{kcal} / \mathrm{mol}$ and that the reaction $\mathrm{X} \rightleftharpoons \mathrm{Y}+\mathrm{Z}$ has a $\Delta G$ of $-10 \mathrm{kcal} / \mathrm{mol}$ :

$$
\begin{aligned}
& \text { (1) } \mathrm{A} \rightleftharpoons \mathrm{~B}+\mathrm{X} \quad \Delta G=+5 \mathrm{kcal} / \mathrm{mol} \\
& \text { (2) } \mathrm{X} \rightleftharpoons \mathrm{Y}+\mathrm{Z} \quad \Delta G=-10 \mathrm{kcal} / \mathrm{mol} \\
& \text { Sum: } \mathrm{A} \rightleftharpoons \mathrm{~B}+\mathrm{Y}+\mathrm{Z} \quad \Delta \mathrm{G}^{\circ \prime}=-5 \mathrm{kcal} / \mathrm{mol}
\end{aligned}
$$

In the absence of the second reaction, there would be much more A than B at equilibrium. However, because the conversion of X to $\mathrm{Y}+\mathrm{Z}$ is such a favorable reaction, it will pull the first process toward the formation of B and the consumption of A. Energetically unfavorable reactions in cells are often coupled to the energy-releasing hydrolysis of ATP.

## Hydrolysis of ATP Releases Substantial Free Energy and Drives Many Cellular Processes

In almost all organisms, the nucleoside triphosphate adenosine triphosphate, or ATP (Figure 2-31), is the most important molecule for capturing, transiently storing, and subsequently transferring energy to perform work (e.g., biosynthesis, mechanical motion). Commonly referred to as a cell's energy "currency," ATP is a type of usable potential energy that cells can "spend" in order to power their activities. The storied history of ATP begins with its discovery in 1929, apparently simultaneously by Kurt Lohmann, who
![li106857775e0](li106857775e0.jpg)

FIGURE 2-31 Hydrolysis of adenosine triphosphate (ATP).
The two phosphoanhydride bonds (red) in ATP (top), which link the three phosphate groups, each have a $\Delta G^{\circ}$ of about $-7.3 \mathrm{kcal} / \mathrm{mol}$ for hydrolysis. Hydrolysis of the terminal phosphoanhydride bond by the addition of water results in the release of phosphate and generation of ADP. Hydrolysis of the phosphoanhydride bonds of ATP, especially the terminal one, is the source of energy that drives many energy-requiring reactions in biological systems.
was working with the great biochemist Otto Meyerhof in Germany and who published first, and by Cyrus Fiske and Yellapragada SubbaRow in the United States. Muscle contractions were shown to depend on ATP in the 1930s. The proposal that ATP is the main intermediary for the transfer of energy in cells is credited to Fritz Lipmann around 1941. Many Nobel Prizes have been awarded for the study of ATP and its role in cellular energy metabolism, and its importance in understanding molecular cell biology cannot be overstated.

The useful energy in an ATP molecule is contained in phosphoanhydride bonds, which are covalent bonds formed from the condensation of two molecules of phosphate by the loss of water:
![li107857775e0](li107857775e0.jpg)

As shown in Figure 2-31, an ATP molecule has two key phosphoanhydride (also called phosphodiester) bonds. Forming these bonds (represented here by the symbol $\sim$ ) in ATP requires an input of energy. When these bonds are hydrolyzed, or broken by the addition of water, that energy is released. Hydrolysis of a phosphoanhydride bond in each of the following reactions has a highly negative $\Delta G^{\circ \prime}$ of about $-7.3 \mathrm{kcal} / \mathrm{mol}$ :

$$
\begin{aligned}
& \mathrm{Ap} \sim \mathrm{p} \sim \mathrm{p}+\mathrm{H}_{2} \mathrm{O} \rightarrow \mathrm{Ap} \sim \mathrm{p}+\mathrm{P}_{\mathrm{i}}+\mathrm{H}^{+} \\
& \text { (ATP) } \\
& \mathrm{Ap} \sim \mathrm{p} \sim \mathrm{p}+\mathrm{H}_{2} \mathrm{O} \rightarrow \mathrm{Ap}+\mathrm{PP}_{\mathrm{i}}+\mathrm{H}^{+} \\
& \text { (ATP) } \\
& \mathrm{Ap} \sim \mathrm{p}+\mathrm{H}_{2} \mathrm{O} \rightarrow \mathrm{Ap}+\mathrm{P}_{\mathrm{i}}+\mathrm{H}^{+} \\
& \text { (ADP) } \\
& \text { (AMP) }
\end{aligned}
$$

$\mathrm{P}_{\mathrm{i}}$ stands for inorganic phosphate $\left(\mathrm{PO}_{4}{ }^{3-}\right)$ and $\mathrm{PP}_{\mathrm{i}}$ for inorganic pyrophosphate, two phosphate groups linked by a phosphoanhydride bond. As the top two reactions show, the removal of a phosphate group from ATP leaves adenosine diphosphate (ADP), and the removal of a pyrophosphate group from ATP leaves adenosine monophosphate (AMP).

A phosphoanhydride bond or other "high-energy bond" (commonly denoted by $\sim$ ) is not intrinsically different from other covalent bonds. High-energy bonds simply release substantial amounts of energy when hydrolyzed. For instance, the $\Delta G^{\circ \prime}$ for hydrolysis of a phosphoanhydride bond in ATP $(\sim 7.3 \mathrm{kcal} / \mathrm{mol})$ is more than three times the $\Delta G^{\circ \prime}$ for hydrolysis of the phosphoester bond (red) in glycerol 3-phosphate $(\sim 2.2 \mathrm{kcal} / \mathrm{mol})$ :
![li108857775e0](li108857775e0.jpg)

A principal reason for this difference is that ATP and its hydrolysis products, ADP and $\mathrm{P}_{\mathrm{i}}$, are charged at neutral pH . During synthesis of ATP, a large amount of energy must be used to force the negative charges in ADP and $\mathrm{P}_{\mathrm{i}}$ together. Conversely, this energy is released when ATP is hydrolyzed to ADP and $\mathrm{P}_{\mathrm{i}}$. In comparison, formation of the phosphoester bond between an uncharged hydroxyl in glycerol and $\mathrm{P}_{\mathrm{i}}$ requires less energy, and less energy is released when this bond is hydrolyzed.

Cells have evolved protein-mediated mechanisms for transferring the free energy released by hydrolysis of phosphoanhydride bonds to other molecules, thereby driving reactions that would otherwise be energetically unfavorable. For example, if the $\Delta G$ for the reaction $\mathrm{B}+\mathrm{C} \rightarrow \mathrm{D}$ is positive but less than the $\Delta G$ for hydrolysis of ATP, the reaction can be driven to the right by coupling it to hydrolysis of the terminal phosphoanhydride bond in ATP. In one common mechanism of such energy coupling, some of the energy stored in this phosphoanhydride bond is transferred to one of the reactants (here, B) by the breaking of the bond
in ATP and the formation of a covalent bond between the released phosphate group and that reactant. The phosphorylated intermediate generated in this way can then react with reactant C to form product $\mathrm{D}+\mathrm{P}_{\mathrm{i}}$ in a reaction that has an overall negative $\Delta G$ :

$$
\begin{aligned}
\mathrm{B}+\mathrm{Ap} \sim \mathrm{p} \sim \mathrm{p} & \rightarrow \mathrm{~B} \sim \mathrm{p}+\mathrm{Ap} \sim \mathrm{p} \\
\mathrm{~B} \sim \mathrm{p}+\mathrm{C} & \rightarrow \mathrm{D}+\mathrm{P}_{\mathrm{i}}
\end{aligned}
$$

The overall reaction

$$
\mathrm{B}+\mathrm{C}+\mathrm{ATP} \rightleftharpoons \mathrm{D}+\mathrm{ADP}+\mathrm{P}_{\mathrm{i}}
$$

is energetically favorable $(\Delta G<0)$. Similarly, hydrolysis of GTP to GDP can provide energy to perform work, including the synthesis of ATP (see Chapter 12), but most often GTP hydrolysis is used to control cellular systems (e.g., protein synthesis, hormonal signaling) rather than as a source of energy.

An alternative mechanism of energy coupling is to use the energy released by ATP hydrolysis to change the conformation of a molecule to an "energy-rich" stressed state. In turn, the energy stored as conformational stress can be released as the molecule "relaxes" back into its unstressed conformation. If this relaxation process can be coupled to another reaction, the released energy can be harnessed to drive cellular processes.

As with many biosynthetic reactions, transport of molecules into or out of the cell often has a positive $\Delta G$ and thus requires an input of energy to proceed. Such simple transport reactions do not directly involve the making or breaking of covalent bonds; thus their $\Delta G^{\circ \prime}$ is 0 . In the case of a substance moving into a cell, Equation 2-7 becomes

$$
\Delta G=R T \ln \frac{\left[\mathrm{C}_{\mathrm{in}}\right]}{\left[\mathrm{C}_{\text {out }}\right]}
$$

where $\left[\mathrm{C}_{\mathrm{in}}\right]$ is the initial concentration of the substance inside the cell and $\left[\mathrm{C}_{\text {out }}\right]$ is its concentration outside the cell. We can see from Equation 2-10 that $\Delta G$ is positive for transport of a substance into a cell against its concentration gradient (when $\left[\mathrm{C}_{\mathrm{in}}\right]>\left[\mathrm{C}_{\text {out }}\right]$ ); the energy to drive such "uphill" transport is often supplied by the hydrolysis of ATP. Conversely, when a substance moves down its concentration gradient $\left(\left[\mathrm{C}_{\text {out }}\right]>\left[\mathrm{C}_{\text {in }}\right]\right), \Delta G$ is negative. Such "downhill" transport releases energy that can be coupled to an energyrequiring reaction, such as the movement of another substance uphill across a membrane or the synthesis of ATP itself (see Chapters 11 and 12).

## ATP Is Generated During Photosynthesis and Respiration

ATP is continuously being hydrolyzed to provide energy for many cellular activities. Some estimates suggest that humans daily hydrolyze a mass of ATP equal to their entire body weight. Clearly, to continue functioning, cells must

constantly replenish their ATP supply. Constantly replenishing ATP requires that cells obtain energy from their environment. For nearly all cells, the ultimate source of energy used to make ATP is sunlight. Some organisms can use sunlight directly. Through the process of photosynthesis, plants, algae, and certain photosynthetic bacteria trap the energy of sunlight and use it to synthesize ATP from ADP and $\mathrm{P}_{\mathrm{i}}$. Much of the ATP produced in photosynthesis is hydrolyzed to provide energy for the conversion of carbon dioxide to six-carbon sugars, a process called carbon fixation:

$$
6 \mathrm{CO}_{2}+6 \mathrm{H}_{2} \mathrm{O} \longrightarrow \mathrm{C}_{6} \mathrm{H}_{12} \mathrm{O}_{6}+6 \mathrm{O}_{2}+\text { energy }
$$

The sugars made during photosynthesis are a source of food, and thus energy, for the photosynthetic organisms making them and for the non-photosynthetic organisms, such as animals, that consume the plants either directly or indirectly by eating other animals that have eaten the plants. In this way, sunlight is the direct or indirect source of energy for most organisms (see Chapter 12).

In plants, animals, and nearly all other organisms, the free energy in sugars and other molecules derived from food is released in the processes of glycolysis and cellular respiration. During cellular respiration, energy-rich molecules in food (e.g., glucose) are oxidized to carbon dioxide and water. The complete oxidation of glucose,

$$
\mathrm{C}_{6} \mathrm{H}_{12} \mathrm{O}_{6}+6 \mathrm{O}_{2} \rightarrow 6 \mathrm{CO}_{2}+6 \mathrm{H}_{2} \mathrm{O}
$$

has a $\Delta G^{\circ \prime}$ of $-686 \mathrm{kcal} / \mathrm{mol}$ and is the reverse of photosynthetic carbon fixation. Cells employ an elaborate set of protein-mediated reactions to couple the oxidation of 1 molecule of glucose to the synthesis of as many as 30 molecules of ATP from 30 molecules of ADP. This oxygen-dependent (aerobic) degradation (catabolism) of glucose is the major pathway for generating ATP in all animal cells, all nonphotosynthetic plant cells, and many bacterial cells. Catabolism of fatty acids can also be an important source of ATP. We discuss the mechanisms of photosynthesis and cellular respiration in Chapter 12.

Although light energy captured in photosynthesis is the primary source of chemical energy for cells, it is not the only source. Certain microorganisms that live in or around deepsea hydrothermal vents, where adequate sunlight is unavailable, derive the energy for converting ADP and $\mathrm{P}_{\mathrm{i}}$ into ATP from the oxidation of reduced inorganic compounds. These reduced compounds originate deep in the earth and are released at the vents.

## NAD ${ }^{+}$and FAD Couple Many Biological Oxidation and Reduction Reactions

In many chemical reactions, electrons are transferred from one atom or molecule to another; this transfer may or may not accompany the formation of new chemical bonds or the release of energy that can be coupled to other reactions. The loss of electrons from an atom or a molecule is called
oxidation, and the gain of electrons by an atom or a molecule is called reduction. An example of oxidation is the removal of electrons from the sulfhydryl group-containing side chains of two cysteine amino acids to form a disulfide bond, described above in Section 2.2. Electrons are neither created nor destroyed in a chemical reaction, so if one atom or molecule is oxidized, another must be reduced. For example, oxygen draws electrons from $\mathrm{Fe}^{2+}$ (ferrous) ions to form $\mathrm{Fe}^{3+}$ (ferric) ions, a reaction that occurs as part of the process by which carbohydrates are degraded in mitochondria. Each oxygen atom receives two electrons, one from each of two $\mathrm{Fe}^{2+}$ ions:

$$
2 \mathrm{Fe}^{2+}+1 / 2 \mathrm{O}_{2} \rightarrow 2 \mathrm{Fe}^{3+}+\mathrm{O}^{2-}
$$

Thus $\mathrm{Fe}^{2+}$ is oxidized and $\mathrm{O}_{2}$ is reduced. Such reactions in which one molecule is reduced and another is oxidized are often referred to as redox reactions. Oxygen is an electron acceptor in many redox reactions in cells under aerobic conditions.

Many biologically important oxidation and reduction reactions involve the removal or addition of hydrogen atoms (protons plus electrons) rather than the transfer of isolated electrons on their own. The oxidation of succinate to fumarate, which occurs in mitochondria, is an example (Figure 2-32). Protons are soluble in aqueous solutions (as $\mathrm{H}_{3} \mathrm{O}^{+}$), but electrons are not, so they must be transferred directly from one atom or molecule to another without a water-dissolved intermediate. In this type of oxidation reaction, electrons are often transferred to small electroncarrying molecules, sometimes referred to as coenzymes. The most common of these electron carriers are $\mathrm{NAD}^{+}$(nicotinamide adenine dinucleotide), which is reduced to NADH, and FAD (flavin adenine dinucleotide), which is reduced to $\mathrm{FADH}_{2}$ (Figure 2-33). The reduced forms of these coenzymes can transfer protons and electrons to other molecules, thereby reducing them.

To describe redox reactions, such as the reaction of ferrous ion $\left(\mathrm{Fe}^{2+}\right)$ and oxygen $\left(\mathrm{O}_{2}\right)$, it is easiest to divide them into two half-reactions:

$$
\begin{aligned}
\text { Oxidation of } \mathrm{Fe}^{2+}: & 2 \mathrm{Fe}^{2+} \rightarrow 2 \mathrm{Fe}^{3+}+2 \mathrm{e}^{-} \\
\text {Reduction of } \mathrm{O}_{2}: & 2 \mathrm{e}^{-}+1 / 2 \mathrm{O}_{2} \rightarrow \mathrm{O}^{2-}
\end{aligned}
$$

![li109857775e0](li109857775e0.jpg)

FIGURE 2-32 Conversion of succinate to fumarate. In this oxidation reaction, which occurs in mitochondria as part of the citric acid cycle, succinate loses two electrons and two protons. These protons and electrons are transferred to FAD, reducing it to $\mathrm{FADH}_{2}$.

![li110857775e0](li110857775e0.jpg)

FIGURE 2-33 The electron-carrying coenzymes $\mathrm{NAD}^{+}$and FAD. (a) $\mathrm{NAD}^{+}$(nicotinamide adenine dinucleotide) is reduced to NADH by the addition of two electrons and one proton simultaneously. In many biological redox reactions, a pair of hydrogen atoms (two protons and two electrons) is removed from a molecule. In some cases, one of the protons and both electrons are transferred to $\mathrm{NAD}^{+}$; the other

In this case, the reduced oxygen $\left(\mathrm{O}^{2-}\right)$ readily reacts with two protons to form one water molecule $\left(\mathrm{H}_{2} \mathrm{O}\right)$. The readiness with which an atom or a molecule gains an electron is its reduction potential $(E)$. The tendency to lose electrons, the oxidation potential, has the same magnitude as the reduction potential for the reverse reaction, but has the opposite sign.

Reduction potentials are measured in volts (V) from an arbitrary zero point set at the reduction potential of the following half-reaction under standard conditions $\left(25^{\circ} \mathrm{C}\right.$, 1 atm , and reactants at 1 M$)$ :

$$
\mathrm{H}^{+}+\mathrm{e}^{-} \underset{\text { oxidation }}{\stackrel{\text { reduction }}{\rightleftharpoons}} \mathrm{H}_{2}
$$

The value of $E$ for a molecule or an atom under standard conditions is its standard reduction potential, $E^{\prime}{ }_{0}$. A molecule or an ion with a positive $E^{\prime}{ }_{0}$ has a higher affinity for electrons than the $\mathrm{H}^{+}$ion does under standard conditions. Conversely, a molecule or ion with a negative $E^{\prime}{ }_{0}$ has a lower affinity for electrons than the $\mathrm{H}^{+}$ion does under standard conditions. Like the values of $\Delta G^{\circ \prime}$, standard reduction potentials may differ somewhat from those found under the conditions in a cell because the concentrations of reactants in a cell are not 1 M .

In a redox reaction, electrons move spontaneously toward atoms or molecules having more positive reduction potentials. In other words, a molecule having a more negative reduction potential can transfer electrons spontaneously to, or reduce, a molecule with a more positive reduction potential. In this type of reaction, the change in electric potential $\Delta E$ is the sum of the reduction and oxidation potentials for the two half-reactions. The $\Delta E$ for a redox reaction is related to the change in free energy $\Delta G$ by the following expression:

$$
\Delta G(\mathrm{cal} / \mathrm{mol})=-n(23,064) \Delta E(\text { volts })
$$

(2-11)
proton is released into solution. (b) FAD (flavin adenine dinucleotide) is reduced to $\mathrm{FADH}_{2}$ by the addition of two electrons and two protons, as occurs when succinate is converted to fumarate (see Figure 2-32). In this two-step reaction, addition of one electron together with one proton first generates a short-lived semiquinone intermediate (not shown), which then accepts a second electron and proton.
where $n$ is the number of electrons transferred. Note that a redox reaction with a positive $\Delta E$ value will have a negative $\Delta G$ and thus will tend to proceed spontaneously from left to right.

## KEY CONCEPTS OF SECTION 2.4

## Biochemical Energetics

- The change in free energy, $\Delta G$, is the most useful measure for predicting the potential of chemical reactions to occur spontaneously in biological systems. Chemical reactions tend to proceed spontaneously in the direction for which $\Delta G$ is negative. The magnitude of $\Delta G$ is independent of the reaction rate. A reaction with a negative $\Delta G$ is referred to as thermodynamically favorable.
- The chemical free-energy change, $\Delta G^{\circ \prime}$, equals -2.3 RT $\log K_{\mathrm{eq}}$. Thus the value of $\Delta G^{\circ \prime}$ can be calculated from the experimentally determined concentrations of reactants and products at equilibrium.
- The rate of a reaction depends on the activation energy needed to energize reactants to a transition state. Catalysts such as enzymes speed up reactions by lowering the activation energy of the transition state.
- A chemical reaction having a positive $\Delta G$ can proceed if it is coupled with a reaction having a negative $\Delta G$ of larger magnitude.
- Many otherwise energetically unfavorable cellular processes are driven by the hydrolysis of phosphoanhydride bonds in ATP (see Figure 2-31).
- Directly or indirectly, light energy captured by photosynthesis in plants, algae, and photosynthetic bacteria is the

ultimate source of chemical energy for nearly all cells on Earth.

- An oxidation reaction (loss of electrons) is always coupled with a reduction reaction (gain of electrons).
- Biological oxidation and reduction reactions are often coupled by electron-carrying coenzymes such as $\mathrm{NAD}^{+}$and FAD (see Figure 2-33).
- Oxidation-reduction reactions with a positive $\Delta E$ have a negative $\Delta G$ and thus tend to proceed spontaneously.


## LaunchPad

macmillan learning
Visit LaunchPad to access study tools and to learn more about the content in this chapter.

- Analyze the Data
- Additional study tools, including videos, animations, and quizzes


## Key Terms

acid 55
adenosine triphosphate (ATP) 32
$\alpha$ carbon atom $\left(\mathrm{C}_{\alpha}\right) \quad 42$
amino acid 42
amphipathic 32
base 55
buffer 55
catalyst 52
chemical potential energy 57
covalent bond 33
dehydration reaction 41
dipole 35
dissociation constant $\left(K_{\mathrm{d}}\right) \quad 53$
disulfide bond 44
endergonic 58
endothermic 59
energy coupling 62
enthalpy (H) 59
entropy $(S) \quad 59$
equilibrium constant $\left(K_{\text {eq }}\right) \quad 52$
exergonic 58
exothermic 59
fatty acids 48
$\Delta G$ (free-energy change) 58
hydrogen bond 37
hydrophilic 32
hydrophobic 32
hydrophobic effect 39
ionic interactions 36
molecular
complementarity 40
monomer 41
monosaccharide 46
noncovalent
interactions 33
nucleoside 46
nucleotide 45
oxidation 63
$\mathrm{pH} \quad 54$
phosphoanhydride
bond 61
phosphoglyceride 49
phospholipid 48
polar 34
polymer 41
redox reaction 63
reduction 63
saturated 48
steady state 53
stereoisomer 34
transition state 60
unsaturated 48
van der Waals
interaction 38

## Review the Concepts

1. The gecko is a reptile with an amazing ability to climb smooth surfaces, including glass. Geckos appear to stick to smooth surfaces via van der Waals interactions between septa on their feet and the smooth surface. How is this method of stickiness advantageous over covalent interactions? Given that van der Waals forces are among the weakest molecular interactions, how can the gecko's feet stick so effectively?
2. The $\mathrm{K}^{+}$channel is an example of a transmembrane protein (a protein that spans the phospholipid bilayer of the plasma membrane). What types of amino acids are likely to be found (a) lining the channel through which $\mathrm{K}^{+}$passes, (b) in contact with the hydrophobic core of the phospholipid bilayer containing fatty acyl groups, (c) in the cytosolic domain of the protein, and (d) in the extracellular domain of the protein?
3. V-M-Y-F-E-N: This is the single-letter amino acid abbreviation for a peptide. What is the net charge of this peptide at pH 7.0 ? An enzyme called a protein tyrosine kinase can attach phosphates to the hydroxyl groups of tyrosine (Y). What is the net charge of the peptide at pH 7.0 after it has been phosphorylated by a tyrosine kinase? What is the likely source of phosphate used by the kinase for this reaction?
4. Disulfide bonds help to stabilize the three-dimensional structure of proteins. What amino acids are involved in the formation of disulfide bonds? Does the formation of a disulfide bond increase or decrease entropy $(\Delta S)$ ?
5. In the 1960 s , the drug thalidomide was prescribed to pregnant women to treat morning sickness. However, thalidomide caused severe limb defects in the children of some women who took the drug, and its use for morning sickness was discontinued. It is now known that thalidomide was administered as a mixture of two stereoisomeric compounds, one of which relieved morning sickness and the other of which was responsible for the birth defects. What are stereoisomers? Why might two such closely related compounds have such different physiological effects?
6. Name the compound shown below.
![li111857775e0](li111857775e0.jpg)

Is this nucleotide a component of DNA, RNA, or both? Name one other function of this compound.
7. The chemical basis of blood-group specificity resides in the carbohydrates displayed on the surfaces of red blood cells. Carbohydrates have the potential for great structural diversity. Indeed, the structural complexity of the oligosaccharides that can be formed from four sugars is greater than that of the oligopeptides that can be formed from four amino acids. What properties of carbohydrates make this great structural diversity possible?
8. Calculate the pH of 1 L of pure water at equilibrium. How will the pH change after 0.008 moles of the strong base sodium hydroxide $(\mathrm{NaOH})$ are dissolved in the water? Now, calculate the pH of a 50 mM aqueous solution of the weak acid 3-( N -morpholino) propane-1-sulfonic acid (MOPS) in which 61 percent of the solute is in its weak acid form and 39 percent is in the form of MOPS's corresponding base (the $\mathrm{p} K_{\mathrm{a}}$ for MOPS is 7.20 ). What is the final pH after 0.008 moles of NaOH are added to 1 L of this MOPS buffer?
9. Ammonia $\left(\mathrm{NH}_{3}\right)$ is a weak base that under acidic conditions becomes protonated to the ammonium ion in the following reaction:

$$
\mathrm{NH}_{3}+\mathrm{H}^{+} \rightarrow \mathrm{NH}_{4}^{+}
$$

$\mathrm{NH}_{3}$ freely permeates biological membranes, including those of lysosomes. The lysosome is a subcellular organelle with a pH of about $4.5-5.0$; the pH of cytoplasm is about 7.0 . What is the effect on the pH of the fluid content of lysosomes when cells are exposed to ammonia? Note: Ammonium $\left(\mathrm{NH}_{4}{ }^{+}\right)$does not diffuse freely across membranes.
10. Consider the binding reaction $\mathrm{L}+\mathrm{R} \rightarrow \mathrm{LR}$, where L is a ligand and R is its receptor. When $1 \times 10^{-3} \mathrm{M}$ of L is added to a solution containing $5 \times 10^{-2} \mathrm{M}$ of $\mathrm{R}, 90$ percent of the L binds to form LR. What is the $K_{\mathrm{eq}}$ of this reaction? How will the $K_{\text {eq }}$ be affected by the addition of a protein that facilitates (catalyzes) this binding reaction? What is the dissociation equilibrium constant $K_{\mathrm{d}}$ ?
11. What is the ionization state of phosphoric acid in the cytoplasm? Why is phosphoric acid such a physiologically important compound?
12. The $\Delta G^{\circ \prime}$ for the reaction $\mathrm{X}+\mathrm{Y} \rightarrow \mathrm{XY}$ is $-1000 \mathrm{cal} /$ mol. What is the $\Delta G$ at $25^{\circ} \mathrm{C}\left(298^{\circ} \mathrm{Kelvin}\right)$ starting with 0.01 M each of $\mathrm{X}, \mathrm{Y}$, and XY? Suggest two ways one could make this reaction energetically favorable.
13. According to health experts, saturated fatty acids, which come from animal fats, are a major factor contributing to coronary heart disease. What distinguishes a saturated fatty acid from an unsaturated fatty acid, and to what does the term saturated refer? Recently, trans unsaturated fatty acids,
or trans fats, which raise total cholesterol levels in the body, have also been implicated in heart disease. How does the cis stereoisomer differ from the trans configuration, and what effect does the cis configuration have on the structure of the fatty acid chain?
14. Chemical modifications of amino acids contribute to the diversity and function of proteins. For instance, $\gamma$-carboxylation of specific amino acids is required to make some proteins biologically active. What particular amino acid undergoes this modification, and what is its biological relevance? Warfarin, a derivative of coumarin, which is present in many plants, inhibits $\gamma$-carboxylation of this amino acid and was used in the past as a rat poison. At present, it is also used clinically in humans. What patients might be prescribed warfarin and why?

## References

Alberty, R. A., and R. J. Silbey. 2005. Physical Chemistry, 4th ed. Wiley.

Atkins, P., and J. de Paula. 2005. The Elements of Physical Chemistry, 4th ed. W. H. Freeman and Company.

Berg, J. M., J. L. Tymoczko, and L. Stryer. 2007. Biochemistry, 6th ed. W. H. Freeman and Company.

Cantor, P. R., and C. R. Schimmel. 1980. Biophysical Chemistry. W. H. Freeman and Company.

Davenport, H. W. 1974. ABC of Acid-Base Chemistry, 6th ed. University of Chicago Press.

Eisenberg, D., and D. Crothers. 1979. Physical Chemistry with Applications to the Life Sciences. Benjamin-Cummings.

Guyton, A. C., and J. E. Hall. 2000. Textbook of Medical Physiology, 10th ed. Saunders.

Hill, T. J. 1977. Free Energy Transduction in Biology. Academic Press.

Klotz, I. M. 1978. Energy Changes in Biochemical Reactions. Academic Press.

Murray, R. K., et al. 1999. Harper's Biochemistry, 25th ed. Lange.

Nicholls, D. G., and S. J. Ferguson. 1992. Bioenergetics 2. Academic Press.

Oxtoby, D., H. Gillis, and N. Nachtrieb. 2003. Principles of Modern Chemistry, 5th ed. Saunders.

Sharon, N. 1980. Carbohydrates. Sci. Am. 243(5):90-116.
Tanford, C. 1980. The Hydrophobic Effect: Formation of Micelles and Biological Membranes, 2d ed. Wiley.

Tinoco, I., K. Sauer, and J. Wang. 2001. Physical ChemistryPrinciples and Applications in Biological Sciences, 4th ed. Prentice Hall.

Van Holde, K., W. Johnson, and P. Ho. 1998. Principles of Physical Biochemistry. Prentice Hall.

Voet, D., and J. Voet. 2004. Biochemistry, 3d ed. Wiley.
Wood, W. B., et al. 1981. Biochemistry: A Problems Approach, 2d ed. Benjamin-Cummings.

![li112857775e0](li112857775e0.jpg)

Molecular ribbon model of a protein "needle" used by pathogenic bacteria to inject proteins into human cells to initiate infection. Many disease-causing bacteria, including Salmonella typhimurium (food poisoning) and Yersinia pestis (bubonic plague), use a syringe-like protein complex called a type III secretion system to inject proteins into their mammalian target cells. The structure of the needle portion of the syringe used by Salmonella typhimurium, determined using a combination of nuclear magnetic resonance (NMR), electron microscopy, and computational methods, is a long tube with many $\alpha$ helices (illustrated as coiled ribbons) forming the walls of the needle. [Data from A. Loquet et al., 2012, Nature 486:276, PDB ID 2lpz.]

Proteins, which are polymers of amino acids, come in many sizes and shapes. Their three-dimensional diversity principally reflects variations in their lengths and amino acid sequences. In general, the linear, unbranched polymer of amino acids composing any protein will fold into only one or a few closely related three-dimensional shapes-called conformations. The conformation of a protein, together with the distinctive chemical properties of its amino acid side chains, determines its function. In some cases, the conformation, and thus the function, of a protein can change when that protein noncovalently or covalently associates with other molecules. Because of their many different shapes and chemical properties, proteins can perform a dazzling array of distinct functions inside and outside cells that either are essential for life or provide a

## Protein Structure and Function

selective evolutionary advantage to the cell or organism that contains them. It is, therefore, not surprising that characterizing the structures and activities of proteins is a fundamental prerequisite for understanding how cells work. Much of this textbook is devoted to examining how proteins act together to allow cells to live and function properly.

Although their structures are diverse, most proteins can be grouped into one of a few broad functional classes. Structural proteins, for example, determine the shapes of cells and their extracellular environments and serve as guide wires or rails to direct the intracellular movement of molecules and organelles. They are usually formed by the assembly of multiple protein subunits into very large, long structures. Scaffold proteins bring other proteins together into ordered

### 3.1 Hierarchical Structure of Proteins

### 3.2 Protein Folding

### 3.3 Protein Binding and Enzyme Catalysis

### 3.4 Regulating Protein Function

### 3.5 Purifying, Detecting, and Characterizing Proteins

### 3.6 Proteomics

arrays to perform specific functions more efficiently than those proteins would if they were not assembled together. Enzymes are proteins that catalyze chemical reactions. Membrane transport proteins permit the flow of ions and molecules across cellular membranes. Regulatory proteins act as signals, sensors, and switches to control the activities of cells by altering the functions of other proteins and genes. Regulatory proteins include signaling proteins, such as the hormones and cell-surface receptors that transmit extracellular signals to the cell interior. Motor proteins are responsible for moving other proteins, organelles, cells-even whole organisms. Any one protein can be a member of more than one protein class, as is the case with some cell-surface signaling receptors that are both enzymes and regulator proteins because they transmit signals from outside to inside cells by catalyzing chemical reactions. To accomplish their diverse missions efficiently, some proteins assemble into large complexes, often called molecular machines.

How do proteins perform so many diverse functions? They do so by exploiting a few simple activities. Most fundamentally, proteins bind-to one another, to other macromolecules such as DNA, and to small molecules and ions. In many cases, such binding induces a conformational change (a change in the three-dimensional structure) in the protein and thus influences its activity. Binding is based on molecular complementarity between a protein and its binding partner, as described in Chapter 2. A second key activity is enzymatic catalysis. Appropriate folding of a protein will place some amino acid side chains and some carboxyl and amino groups of its backbone into positions that permit the catalysis of covalent bond rearrangements. A third activity is folding into a channel or pore within a membrane through which molecules and ions can flow. Although these are especially crucial protein activities, they are not the only ones. For example, fish that live in frigid waters-the Antarctic borchs and Arctic cods-have antifreeze proteins in their circulatory systems to prevent water crystallization.

A complete understanding of how proteins permit cells to live and thrive requires the identification and characterization of all the proteins used by a cell. In a sense, molecular cell biologists want to compile a complete protein "parts list" and construct a "user's manual" that describes how these proteins work. Compiling a comprehensive inventory of proteins has become feasible in recent years with the sequencing of the entire genomes-complete sets of genesof more and more organisms. From a computer analysis of a genome's sequence, researchers can deduce the amino acid sequences and approximate number of the proteins it encodes (see Chapter 6). The term proteome was coined to refer to the entire protein complement of an organism. The human genome contains some 20,000-23,000 genes that encode proteins. However, variations in mRNA production, such as alternative splicing (see Chapter 10), and more than a hundred types of protein modifications may generate hundreds of thousands of distinct human proteins. By comparing the sequences and structures of proteins of unknown function with those of proteins of known function, scientists can often deduce much about what the unknown
proteins do. In the past, characterization of protein function by genetic, biochemical, or physiological methods often preceded the identification of particular proteins. In the modern genomic and proteomic era, a protein is usually identified before its function is determined.

In this chapter, we begin our study of how the structure of a protein gives rise to its function, a theme that recurs throughout this book (Figure 3-1). The first section examines how linear chains of amino acid building blocks are arranged in a three-dimensional structural hierarchy. The next section discusses how proteins fold into these structures. We then turn to protein function, focusing on enzymes, those proteins that catalyze chemical reactions. Various mechanisms that cells use to control the activities and life spans of proteins are covered next. The chapter concludes with a discussion
![li113857775e0](li113857775e0.jpg)

FIGURE 3-1 Overview of protein structure and function. (a) Proteins have a hierarchical structure. A polypeptide's linear sequence of amino acids linked by peptide bonds (primary structure) folds into local helices or sheets (secondary structure) that pack into a complex three-dimensional shape (tertiary structure). Some individual polypeptides associate into multichain complexes (quaternary structure), which in some cases can be very large, consisting of tens to hundreds of subunits (supramolecular complexes). (b) Proteins perform numerous functions, including organizing the genome, organelles, cytoplasm, protein complexes, and membranes in three-dimensional space (structure); controlling protein activity (regulation); monitoring the environment and transmitting information (signaling); moving small molecules and ions across membranes (transport); catalyzing chemical reactions (via enzymes); and generating force for movement (via motor proteins). These functions and others arise from specific binding interactions and conformational changes in the structure of a properly folded protein.

of commonly used techniques for identifying, isolating, and characterizing proteins, and a discussion of the burgeoning field of proteomics.

### 3.1 Hierarchical Structure of Proteins

In many proteins, the polymer chain folds into a distinct three-dimensional shape that is stabilized primarily by noncovalent interactions between regions in the linear sequence of amino acids. A key concept in understanding how proteins work is that function is often derived from three-dimensional structure, and three-dimensional structure is determined by both a protein's amino acid sequence and intramolecular noncovalent interactions. The principles relating biological structure and function were initially formulated by the biologists Johann von Goethe (1749-1832), Ernst Haeckel (1834-1919), and D'Arcy Thompson (1860-1948), whose work has been widely influential in biology and beyond. Indeed, their ideas greatly influenced the school of "organic" architecture pioneered in the early twentieth century that is epitomized by the dicta "form follows function" (Louis Sullivan) and "form is function" (Frank Lloyd Wright). Here we consider the architecture of proteins at four levels of organization: primary, secondary, tertiary, and quaternary (Figure 3-2).
(a) Primary structure
-Ala-Glu-Val-Thr-Asp-Pro-Gly-
(b) Secondary structure
![li114857775e0](li114857775e0.jpg)
![li115857775e0](li115857775e0.jpg)

FIGURE 3-2 Four levels of protein hierarchy. (a) The linear sequence of amino acids linked together by peptide bonds is the primary structure. (b) Folding of the polypeptide chain into local $\alpha$ helices or $\beta$ sheets represents secondary structure. (c) Secondary structural elements, together with various loops and turns in a single polypeptide chain, pack into a larger, independently stable tertiary structure, which may include distinct domains. (d) Some proteins consist of more than one polypeptide associated together in a quaternary structure.

## The Primary Structure of a Protein Is Its Linear Arrangement of Amino Acids

As discussed in Chapter 2, proteins are polymers constructed out of 20 different types of amino acids. Individual amino acids are linked together in linear, unbranched chains by covalent amide bonds, called peptide bonds. Peptide bond formation between the amino group of one amino acid and the carboxyl group of another results in the net release of a water molecule and thus is a form of dehydration reaction (Figure 3-3a). The repeated amide $\mathrm{N}, \alpha$ carbon $\left(\mathrm{C}_{\alpha}\right)$, carbonyl C , and oxygen atoms of each amino acid residue form the backbone of a protein molecule from which the various sidechain groups project (Figure 3-3b, c). As a consequence of
(a)
![li116857775e0](li116857775e0.jpg)
(b)
![li117857775e0](li117857775e0.jpg)
(c)
![li118857775e0](li118857775e0.jpg)

FIGURE 3-3 Structure of a polypeptide. (a) Individual amino acids are linked together by peptide bonds, which form via reactions that result in a loss of water (dehydration). $R_{1}, R_{2}$, etc., represent the side chains ("R groups") of amino acids. (b) Linear polymers of peptide-bond-linked amino acids are called polypeptides, which have a free amino end ( N -terminus) and a free carboxyl end (C-terminus). (c) A ball-and-stick model shows peptide bonds (yellow) linking the amino nitrogen atom (blue) of one amino acid (aa) with the carbonyl carbon atom (gray) of an adjacent one in the chain. The R groups (green) extend from the $\alpha$ carbon atoms (black) of the amino acids. These side chains largely determine the distinct properties of individual proteins.

the peptide linkage, the backbone exhibits directionality, usually referred to as an N -to-C orientation, because all the amino groups are located on the same side of the $\mathrm{C}_{\alpha}$ atoms. Thus one end of a protein has a free (unlinked) amino group (the N -terminus), and the other end has a free carboxyl group (the C-terminus). The sequence of a protein chain is conventionally written with its N -terminal amino acid on the left and its C-terminal amino acid on the right, and the amino acids are numbered sequentially starting from the N -terminus.

The primary structure of a protein is simply the linear covalent arrangement, or sequence, of the amino acid residues that compose it. The first primary structure of a protein determined was that of insulin in the early 1950s. Today the number of known sequences exceeds 10 million and is growing daily. Many terms are used to denote the chains formed by the polymerization of amino acids. A short chain of amino acids linked by peptide bonds and having a defined sequence is called an oligopeptide, or simply a peptide; longer chains are referred to as polypeptides. Peptides generally contain fewer than 20-30 amino acid residues, whereas polypeptides are often 200-500 residues long. The longest protein described to date is the muscle protein titin, some forms of which can be more than 34,000 residues long. We generally reserve the term protein for a polypeptide (or complex of polypeptides) that has a well-defined threedimensional structure.

The size of a protein or a polypeptide is expressed either as its mass in daltons (a dalton is 1 atomic mass unit) or as its molecular weight (MW), which is a dimensionless number equal to the mass in daltons. For example, a 10,000-MW protein has a mass of 10,000 daltons ( Da ), or 10 kilodaltons ( kDa ). Later in this chapter, we will consider different methods for measuring the sizes and other physical characteristics of proteins. The precise molecular weight of a protein that has not been covalently modified is readily determined by summing up the weights of all of its constituent amino acids as determined from its amino acid sequence. The proteins encoded by the yeast genome, for example, have an average molecular weight of 52,728 and contain, on average, 466 amino acid residues. The average molecular weight of amino acids in proteins is 113 , taking into account their average relative abundances. This value can be used to estimate the number of residues in a protein of unknown sequence if you know its molecular weight or, conversely, to estimate from the number of residues in a protein its likely molecular weight. Covalent modification of one or more amino acids in a protein-for example, by phosphorylation or glycosylation (see Chapters 2 and 13)-alters the mass of those residues and thus the mass of the protein in which they reside.

How many proteins are there in a typical eukaryotic (nucleated) cell? Let's do a simple calculation for one such cell, a hepatocyte (a major type of cell in the mammalian liver). This type of cell, roughly a cube $15 \mu \mathrm{~m}(0.0015 \mathrm{~cm})$ on a side, has a volume of $3.4 \times 10^{-9} \mathrm{~cm}^{3}$ (or milliliters, ml ). Assuming a cell density of $1.03 \mathrm{~g} / \mathrm{ml}$, the cell would weigh $3.5 \times 10^{-9} \mathrm{~g}$. Since protein accounts for approximately 20 percent
of a cell's weight, the total weight of cellular protein is $7 \times 10^{-10} \mathrm{~g}$. Assuming that an average protein has a molecular weight of $52,728 \mathrm{~g} / \mathrm{mol}$, we can calculate the total number of protein molecules per hepatocyte as about $7.9 \times 10^{9}$ from the total protein weight and Avogadro's number, the number of molecules per mole of any chemical compound $\left(6.02 \times 10^{23}\right)$. To carry this calculation one step further, consider that a hepatocyte contains about 10,000 different proteins; thus each cell, on average, would contain close to a million molecules of each type of protein. In fact, the abundances of different proteins vary widely, from the quite rare insulin-binding receptor protein ( 20,000 molecules per cell) to the structural protein actin ( $5 \times 10^{8}$ molecules per cell). Every cell closely regulates the abundance of each protein such that each is present in the appropriate quantity for its cellular functions at any given time. We will learn more about the mechanisms used by cells to regulate protein levels later in this chapter and in Chapters 9 and 10.

## Secondary Structures Are the Core Elements of Protein Architecture

The second level in the hierarchy of protein structure is secondary structure. Secondary structures are stable spatial arrangements of segments of a polypeptide chain held together by hydrogen bonds between backbone amide and carbonyl groups and often involving repeating structural patterns. The propensity of a segment of a polypeptide chain to form any given secondary structure depends on its amino acid sequence (see Section 3.2 below). A single polypeptide may contain multiple types of secondary structure in various portions of the chain, depending on its sequence. The principal secondary structures are the alpha ( $\alpha$ ) helix, the beta $(\beta)$ sheet, and the short U-shaped beta $(\beta)$ turn. Parts of the polypeptide that don't form these structures but nevertheless have a well-defined, stable shape are said to have an irregular structure. The term random coil applies to highly flexible parts of a polypeptide chain that have no fixed threedimensional structure. In an average protein, 60 percent of the polypeptide chain exists as $\alpha$ helices and $\beta$ sheets; the remainder of the molecule is in irregular structures, coils, and turns. Thus $\alpha$ helices and $\beta$ sheets are the major internal supportive elements in most proteins. Here we explore the shapes of secondary structures and the forces that favor their formation. In later sections, we examine how arrays of secondary structure fold together into larger, more complex arrangements called tertiary structure.

The $\alpha$ Helix In a polypeptide segment folded into an $\alpha$ helix, the backbone forms a spiral structure in which the carbonyl oxygen atom of each peptide bond is hydrogen-bonded to the amide hydrogen atom of the amino acid four residues farther along the chain in the direction of the C-terminus (Figure 3-4). Within an $\alpha$ helix, all the backbone amino and carboxyl groups are hydrogen-bonded to one another except at the very beginning and end of the helix. This periodic arrangement of bonds confers an amino-to-carboxy-terminal

![li119857775e0](li119857775e0.jpg)

FIGURE 3-4 The $\alpha$ helix, a common secondary structure in proteins. The polypeptide backbone (seen as a ribbon) is folded into a spiral that is held in place by hydrogen bonds between backbone oxygen and hydrogen atoms. Only hydrogens involved in bonding are shown. The outer surface of the helix is covered by the side-chain $R$ groups (green).
directionality on the helix because all the hydrogen bond acceptors (i.e., the carbonyl groups) have the same orientation (pointing in the downward direction in Figure 3-4), resulting in a structure in which there is a complete turn of the spiral every 3.6 residues. An $\alpha$ helix 36 amino acids long has 10 turns of the helix and is 5.4 nm long ( 0.54 nm per turn).

The stable arrangement of hydrogen-bonded amino acids in the $\alpha$ helix holds the backbone in a straight, rodlike cylinder from which the side chains point outward. The relative hydrophobic or hydrophilic quality of a particular helix within a protein is determined entirely by the characteristics of the side chains. In water-soluble proteins, hydrophilic helices with polar side chains extending outward tend to be found on the outside surfaces, where they can interact with the aqueous environment, whereas hydrophobic helices with nonpolar, hydrophobic side chains tend to be buried within the core of the folded protein. Proteins embedded in the hydrophobic core of cellular membranes (see Chapter 7)
often use one or more hydrophobic helices that are 20-25 residues long to cross the membrane. The amino acid proline is usually not found in $\alpha$ helices because the covalent bonding of its amino group with a carbon in the side chain prevents its participation in stabilizing the backbone through normal hydrogen bonding. While the classic $\alpha$ helix is the most intrinsically stable and most common helical form in proteins, there are variations, such as more tightly or loosely twisted helices. For example, in a specialized helix called a coiled coil (described several sections farther on), the helix is more tightly wound ( 3.5 residues and 0.51 nm per turn).

The $\beta$ Sheet Another type of secondary structure, the $\beta$ sheet, consists of laterally packed $\beta$ strands. Each $\beta$ strand is a short (5-8-residue), nearly fully extended polypeptide segment. In contrast to the $\alpha$ helix, in which hydrogen bonds occur between the backbone amino and carboxyl groups of nearly adjacent residues, hydrogen bonds in the $\beta$ sheet occur between backbone atoms in separate, but adjacent, $\beta$ strands and are oriented perpendicularly to the chains of backbone atoms (Figure 3-5a). These distinct $\beta$ strands (indicated as green and blue arrows in the figure) may be either within a single polypeptide chain, with short or long loops between the $\beta$ strand segments, or on different polypeptide chains in a protein composed of multiple polypeptides. Figure 3-5b shows how two or more $\beta$ strands align into adjacent rows, forming a nearly two-dimensional $\beta$ pleated sheet (or simply pleated sheet), in which hydrogen bonds within the plane of the sheet hold the $\beta$ strands together as the side chains stick out above and below the plane. Like $\alpha$ helices, $\beta$ strands have a directionality defined by the orientation of the peptide bonds. Therefore, in a pleated sheet, adjacent $\beta$ strands can be oriented in alternating opposite (antiparallel) directions (see Figure 3-5a) or in the same (parallel) direction (Figure 3-5c). In some proteins, $\beta$ sheets form part of the hydrophobic core of the protein (described below) or the side of an open space that binds other molecules; in some proteins embedded in membranes, the $\beta$ sheets curve around and form a hydrophilic central pore through which ions and small molecules may flow (see Chapter 7).

The $\boldsymbol{\beta}$ Turn Composed of four residues, $\beta$ turns are located on the surface of a protein, forming sharp bends that reverse the direction of the polypeptide backbone, often toward the protein's interior. These short, U-shaped secondary structures are often stabilized by a hydrogen bond between their end residues (Figure 3-6). Glycine and proline are commonly found in $\beta$ turns. The lack of a large side chain in glycine and the presence of a built-in bend in proline allow the polypeptide backbone to fold into a tight $U$ shape. $\beta$ Turns help long polypeptides fold into highly compact structures. A reversal in the direction of the polypeptide backbone may also be mediated by segments of the polypeptide that are longer than four residues and that form bends or loops. In contrast to tight $\beta$ turns, which exhibit just a few well-defined conformations, longer loops can have many different conformations.

![li120857775e0](li120857775e0.jpg)

FIGURE 3-5 The $\beta$ sheet, another common secondary structure in proteins. (a) Top view of a three-stranded $\beta$ sheet. Each strand is highlighted by a ribbon-like arrow with alternating blue and green segments that is pointed with an N -to-C orientation, with the loops of connecting residues indicated by thick black lines. In this antiparallel $\beta$ sheet, each strand (arrow) points in the direction opposite to that of the adjacent strand. The stabilizing hydrogen bonds between the $\beta$ strands are indicated by green dashed lines. (b) Side view of an antiparallel $\beta$ sheet. The projection of the R groups (green) above and below the plane of the sheet is obvious in this view. The fixed bond angles in the polypeptide backbone produce a pleated contour represented in panel (a) by the alternating colored segments. (c) Top view of two $\beta$ sheets, whose individual strands ( N -to-C orientations represented by arrows) are either antiparallel, in which the strands alternately point in opposite directions (left), or parallel, in which all strands point in the same direction (right).

## Tertiary Structure Is the Overall Folding of a Polypeptide Chain

Tertiary structure refers to the overall conformation of a polypeptide chain-that is, the three-dimensional arrangement of all its amino acid residues. In contrast to secondary structures, which are stabilized only by hydrogen bonds, tertiary structure is stabilized primarily by hydrophobic interactions between nonpolar side chains, together with hydrogen
![li121857775e0](li121857775e0.jpg)

FIGURE 3-6 Structure of a $\boldsymbol{\beta}$ turn. Composed of four residues, $\beta$ turns reverse the direction of a polypeptide chain (resulting in a $180^{\circ} \mathrm{U}$ turn). The $C_{α}$ carbons of the first and fourth residues are usually less than 0.7 nm apart, and those residues are often linked by a hydrogen bond. $\beta$ turns facilitate the folding of long polypeptides into compact structures.
bonds involving polar side chains and backbone amino and carboxyl groups. These stabilizing forces hold together elements of secondary structure- $\alpha$ helices, $\beta$ strands, turns, and coils. Because the stabilizing interactions are often weak, however, the tertiary structure of a protein is not rigidly fixed, but undergoes continual minute fluctuations, and some segments within the tertiary structure of a protein can be so mobile that they are considered to be disordered-that is, lacking well-defined, stable, three-dimensional structure. This variation in structure has important consequences for the function and regulation of proteins.

The chemical properties of amino acid side chains help define tertiary structure. In some proteins-for example, those that are secreted from cells or are cell-surface proteins that face the extracellular environment-disulfide bonds between the side chains of cysteine residues can covalently link regions of the proteins, thus restricting the proteins' flexibility and increasing the stability of their tertiary structures. Amino acids with charged hydrophilic polar side chains tend to be on the outer surfaces of proteins; by interacting with water, they help to make the proteins soluble in aqueous solutions and can form noncovalent interactions with other water-soluble molecules, including other proteins. In contrast, amino acids with hydrophobic nonpolar side chains are usually sequestered away from the water-facing surfaces of a protein, in many cases forming a water-insoluble central core. This observation led to what's known as the "oil drop model" of protein conformation because the core of a protein is relatively hydrophobic, or "oily" (Figure 3-7). Uncharged hydrophilic polar side chains are found both on the surface and in the inner core of proteins.

## There Are Four Broad Structural Categories of Proteins

Proteins usually fall into one of four broad structural categories based on their tertiary structure: globular proteins, fibrous proteins, integral membrane proteins, and intrinsically disordered

![li122857775e0](li122857775e0.jpg)

FIGURE 3-7 The oil drop model of protein folding. The hydrophobic and hydrophilic residues of a polypeptide chain can be distributed throughout its linear sequence as illustrated in the unfolded protein (top). The color scale denotes the most most hydrophilic residues (blue) to the most hydrophobic (yellow). When the protein folds (bottom left), hydrophilic (charged and uncharged polar) side chains will often be exposed on the protein's surface, where they can form stabilizing interactions with surrounding water and ions. In contrast, the hydrophobic residues tend to cluster together in the inner core, somewhat like drops of oil in an aqueous liquid, driven away from the aqueous surroundings by the hydrophobic effect (see Chapter 2). These core residues are more easily seen when several surface residues are removed (bottom right). [Data from M. C. Vaney et al., 1996, Acta Crystallogr., Sect. D. 52:505, PDB ID 193I.]

Intrinsically disordered proteins are fundamentally distinct from the well-ordered proteins in the other three categories. Many proteins we consider in this book adopt only one or a few very closely related conformations when they are in their normal functional state, called the native state. Intrinsically disordered proteins, however, do not have well-ordered structures in their native, functional states; instead, their polypeptide chains are very flexible-indeed, disordered-with no fixed conformation. Sometimes only a segment of a polypeptide chain, rather than the entire chain, will be intrinsically disordered. The exceptional conformational flexibilities of intrinsically disordered proteins or protein segments appear to be key to their functional activities, such as the ability to interact with multiple partner proteins or to fold into a well-defined conformation only after binding to such partners (Figure 3-8a).

Intrinsically disordered proteins typically, but not exclusively, serve as signaling molecules, regulators of the activities of other molecules, or as scaffolds for multiple proteins, small molecules, and ions (e.g., binding ions via multiple charged residues). Regions of intrinsic disorder can provide flexible links, or tethers, between well-ordered regions of a protein; serve as sites of some types of post-translational protein modification [e.g., covalent addition of phosphate groups (phosphorylation) or sugars (glycosylation)]; serve as targets of protease digestion that regulates protein activity; inhibit the activity of the protein in which they are embedded (autoinhibition sites); or serve as signals for intracellular sorting of proteins (see Chapter 13). The activities of many proteins containing intrinsically disordered segments are described in subsequent chapters. For example, phosphorylation of the disordered C-terminal domain (CTD) of RNA polymerase II (see Figure 8-12), which is composed of multiple repeats of a seven-amino-acid sequence containing proline, threonine, and serine, regulates key steps in the synthesis of mRNA (see Chapters 9 and 10). The N-termini of histone proteins that control DNA organization in chromatin (see Chapter 8) are sites of important post-translational modifications, and the disordered, proline-rich FH1 region in the protein formin controls the assembly of actin filaments (see Chapter 17).

Intrinsically disordered proteins can be identified experimentally using various biochemical techniques, such as tests of sensitivity to protease digestion (disordered regions usually exhibit greater protease sensitivity), and a wide variety of biophysical techniques, including spectroscopy. The intrinsic disorder of these proteins apparently arises as a consequence of their having a sequence that, relative to well-ordered proteins, is richer in polar amino acids, proline, and net charge, and poorer in hydrophobic residues (Figure 3-8b). Algorithms primarily based on calculations of amino acid com-position-particularly net charge and hydrophobicity-are used to predict which proteins or segments of proteins are intrinsically disordered. By some estimates, about 30 percent or more of eukaryotic proteins are predicted to have at least one segment of 50 or more consecutive residues that is disordered.

![li123857775e0](li123857775e0.jpg)

EXPERIMENTAL FIGURE 3-8 Intrinsically disordered proteins: mechanisms of binding to well-ordered proteins and identification based on hydrophobicity and net charge. (a) The binding of an intrinsically disordered protein (PUMA, blue) to a well-ordered protein (MCL1, gray) results in the formation of a well-defined structure in the previously disordered protein. PUMA and MCL1 are intracellular proteins that can influence the regulated process of cell death called apoptosis (see Chapter 21). Two mechanisms have been proposed for generating a bound complex in which both proteins are structured: conformational selection (top pathway) and induced fit (bottom pathway). In conformational selection, the disordered protein (PUMA) occasionally and transiently adopts in solution the structure it would have in the bound state. The well-ordered binding partner (MLC1) can then bind to (select) PUMA in that transient, ordered conformation, forming a relatively stable bound complex. In induced fit, the disordered protein begins to bind to the well-ordered partner while still disordered and then, while bound, is induced to form the ordered conformation present in the relatively stable, heterodimeric complex. Recent experiments suggest

## Different Ways of Depicting the Conformation of Proteins Convey Different Types of Information

The simplest way to represent three-dimensional protein structure is to trace the course of the backbone atoms, sometimes only the $\mathrm{C}_{\alpha}$ atoms, with a solid line (called a $\mathrm{C}_{\alpha}$ backbone trace, Figure 3-9a); the most complex representation, called a ball-and-stick model, shows every atom (Figure 3-9b). The $\mathrm{C}_{\alpha}$ backbone trace shows the overall folding of the polypeptide chain without consideration of the amino acid side chains; the ball-and-stick model (with balls representing atoms and sticks representing bonds) details the interactions between side-chain atoms, including those that stabilize the protein's conformation and interact with other molecules, as well as the atoms of the backbone. Even though both views are useful, the elements of secondary structure are not always easily discerned in them. Another type of representation, called a ribbon diagram, uses common shorthand symbols for depicting secondary structure-for example,
that the induced fit mechanism best describes the binding of PUMA and MCL1. (b) The sequences of 275 well-ordered, monomeric globular proteins (gray squares) and 91 intrinsically disordered proteins (black and yellow circles) were used to calculate the mean hydrophobicity per residue in each protein using a scale of 0 (least hydrophobic) to 1 (most hydrophobic, $x$ axis), and the mean net charge per residue at pH 7.0 ( $y$ axis). With only three exceptions (black circles), the proteins define two distinct distributions: low hydrophobicity, high net charge (intrinsically disordered, yellow circles) and high hydrophobicity, low net charge (well-ordered, gray squares). The three disordered proteins (black circles) that overlap with the well-ordered population each contain substantial segments predicted to be disordered (low hydrophobicity, high net charge) that apparently overwhelm the rest of the proteins' sequences that might otherwise result in a well-ordered conformation. [Part (a) from Rogers, J. et al., 'Folding and Binding of an Intrinsically Disordered Protein: Fast, but Not 'Diffusion-Limited,'' J. Am. Chem. Soc, 2013, 135 (4), pp1415-1422. http://pubs.acs.org/doi/pdf/10.1021/ja309527h. Part (b) data from V. N. Uversky, J. R. Gillespie, and A. L. Fink, 2000, Protein 41:415-427]
coiled ribbons or solid cylinders for $\alpha$ helices, flat ribbons or arrows for $\beta$ strands, and flexible thin strands for $\beta$ turns, coils, and loops (Figure 3-9c). In a variation of the basic ribbon diagram, ball-and-stick or space-filling models of all or only a subset of side chains can be attached to the backbone ribbon. In this way, side chains that are of interest can be visualized in the context of the secondary structure that is especially clearly represented by the ribbons.

However, none of these three ways of representing protein structure conveys much information about the atoms that are on the protein's surface and in contact with the watery environment. The surface is of interest because it is where other molecules usually bind to a protein. Thus a useful alternative way to represent proteins is to show only the water-accessible surface and use colors to highlight regions having a common chemical character, such as hydrophobicity or hydrophilicity, and charge characteristics, such as positive (basic) or negative (acidic) side chains (Figure 3-9d). Such models reveal the topography of the protein surface

![li124857775e0](li124857775e0.jpg)
(b) Ball-and-stick model
![li125857775e0](li125857775e0.jpg)
(d) Water-accessible surface
![li126857775e0](li126857775e0.jpg)

FIGURE 3-9 Five ways to visualize the protein Ras with its bound GDP. (a) The $\mathrm{C}_{\alpha}$ backbone trace demonstrates how the polypeptide is tightly packed into a small volume. (b) A ball-and-stick representation reveals the locations of all atoms. (c) Turns and loops connect pairs of helices and strands. (d) A water-accessible surface reveals the numerous lumps, bumps, and crevices on the protein surface. Regions of positive charge are shaded purple; regions of negative charge are shaded red. (e) Hybrid model in which ribbon and transparent surface models are combined. [Data from E. F. Pai et al., 1990, EMBO J. 9:2351-2359, PDB ID 5p21.]
![li127857775e0](li127857775e0.jpg)
and the distribution of charge, both important features of binding sites, as well as clefts in the surface where other molecules may bind. This view represents a protein as it is "seen" by another molecule.

## Structural Motifs Are Regular Combinations of Secondary Structures

A particular combination of two or more secondary structures that form a distinct three-dimensional structure is called a structural motif when it appears in multiple proteins. A structural motif is often, but not always, associated with a specific function. Any particular structural motif will frequently perform a common function in different proteins, such as binding to a particular ion or small molecule-for example, calcium or ATP. Some structural motifs, when isolated from the rest of a protein, are stable, and are thus called structural domains, as we shall see shortly. However other structural motifs do not form thermodynamically stable structures in the absence of other portions of the protein and are thus not considered to be independent structural domains.

One common structural motif is the $\alpha$ helix-based coiled coil, or heptad repeat. Many proteins, including fibrous proteins and DNA-regulating proteins called transcription factors (see Chapter 9), assemble into dimers or trimers by using a coiled-coil motif, in which $\alpha$ helices from two, three, or even four separate polypeptide chains coil about one another-resulting in a coil of coils; hence the name (Figure 3-10a). The individual helices bind tightly to one
another because each helix has a strip of aliphatic (hydrophobic, but not aromatic) side chains (leucine, valine, etc.) running along one side of the helix that interacts with a similar strip in the adjacent helix, thus sequestering the hydrophobic groups away from water and stabilizing the assembly of multiple independent helices. These hydrophobic strips are generated along only one side of the helix because the primary structure of each helix is composed of repeating seven-amino-acid units, called heptads, in which the side chains of the first and fourth residues are aliphatic and the other side chains are often hydrophilic (see Figure 3-10a). Because hydrophilic side chains extend from one side of the helix and hydrophobic side chains extend from the opposite side, the overall helical structure is amphipathic. Because leucine frequently appears in the fourth positions and the hydrophobic side chains merge together like the teeth of a zipper, these structural motifs are also called leucine zippers.

Many other structural motifs contain $\alpha$ helices. A common calcium-binding motif called the EF hand contains two short helices connected by a loop (Figure 3-10b). This structural motif, one of several helix-turn-helix and helix-loophelix structural motifs, is found in more than a hundred proteins and is used for sensing calcium levels. The binding of a $\mathrm{Ca}^{2+}$ ion to oxygen atoms in conserved residues in the loop depends on the concentration of $\mathrm{Ca}^{2+}$ in the cell and sometimes induces a conformational change in the protein, altering its activity. Thus calcium concentrations can directly control proteins' structures and functions. Somewhat different helix-turn-helix and basic helix-loop-helix

![li128857775e0](li128857775e0.jpg)

FIGURE 3-10 Motifs of protein secondary structure. (a) This parallel two-stranded coiled-coil motif (left) is characterized by two $\alpha$ helices wound around each other. Helix packing is stabilized by interactions between hydrophobic side chains (red and blue) present at regular intervals along each strand and found along the seam of the intertwined helices. Each $\alpha$ helix exhibits a characteristic heptad repeat sequence with a hydrophobic residue often, but not always, at positions 1 and 4, as indicated. The coiled-coil nature of this structural motif is more apparent in long coiled coils containing many such motifs (right). (b) An EF hand, a type of helix-loop-helix motif, consists of two helices connected by a short loop in a specific conformation. This structural motif is common to many proteins, including many calcium-binding and DNA-binding regulatory proteins.
(bHLH) structural motifs are used for protein binding to DNA and, consequently, for the regulation of gene activity (see Chapter 9). Yet another structural motif commonly found in proteins that bind RNA or DNA is the zinc finger, which contains three secondary structures-an $\alpha$ helix and two $\beta$ strands with an antiparallel orientation-that form a fingerlike bundle held together by a zinc ion (Figure 3-10c).

The relationship between the primary structure of a polypeptide chain and the structural motifs into which it folds is not always straightforward. The amino acid sequences responsible for any given structural motif in different proteins may be very similar to one another. In other words, a common sequence motif can result in a common structural motif. This is the case for the heptad repeats that form coiled coils. However, it is also possible for seemingly unrelated amino acid sequences to fold into a common structural motif, so it is not always possible to predict which amino acid sequences will fold into a given structural motif. Conversely, it is possible that a commonly occurring sequence motif will not fold into a well-defined structural motif. Sometimes short sequence motifs that have an unusual abundance of a particular amino acid, such as proline or aspartate or glutamate, are called "domains"; however, these

In calcium-binding proteins such as calmodulin, oxygen atoms from five residues in the acidic glutamate- and aspartate-rich loop and one water molecule form ionic bonds with a $\mathrm{Ca}^{2+}$ ion. (c) The zinc-finger motif is present in many DNA-binding proteins that help regulate transcription. A $\mathrm{Zn}^{2+}$ ion is held between a pair of $\beta$ strands (blue) and a single $\alpha$ helix (red) by a pair of cysteine residues and a pair of histidine residues. The two invariant cysteine residues are usually at positions 3 and 6 , and the two invariant histidine residues are at positions 20 and 24 in this 25 -residue motif. (Part (a) data from L. Gonzalez, Jr., D. N. Woolfson, and T. Alber, 1996, Nat. Struct. Biol. 3:1011-1018, PDB IDs: 1clk and 2tma. Part (b) data from R. Chattopadhyaya et al., 1992, J. Mol. Biol. 228:1177-1192, PDB ID 1cll. Part (c) data from S. A. Wolfe, R. A. Grant, and C. O. Pabo, 2003, Biochemistry 42:13401-13409, PDB ID 1llm.]
and other short contiguous segments are more appropriately called "sequence motifs" than "domains," as the latter term has a distinct meaning that we will define shortly.

We will encounter numerous additional motifs in our discussions of proteins in this and other chapters. The presence of the same structural motif in different proteins with similar functions clearly indicates that these useful combinations of secondary structures have been conserved in evolution.

## Domains Are Modules of Tertiary Structure

Distinct regions of protein structure are often referred to as domains. There are three main classes of protein domains: functional, structural, and topological. A functional domain is a region of a protein that exhibits a particular activity characteristic of that protein, usually even when isolated from the rest of the protein. For instance, a particular region of a protein may be responsible for its catalytic activity (e.g., a kinase domain that covalently adds a phosphate group to another molecule) or its binding ability (e.g., a DNA-binding domain or a membrane-binding domain). Functional domains are often identified experimentally by whittling down a protein to its smallest active fragment with the aid of proteases,

![li129857775e0](li129857775e0.jpg)

FIGURE 3-11 Tertiary and quaternary levels of structure. The protein pictured here, hemagglutinin (HA), is found on the surface of the influenza virus. This long multimeric molecule has three identical subunits, each composed of two polypeptide chains, $\mathrm{HA}_{1}$ and $\mathrm{HA}_{2}$. (a) The tertiary structure of each HA subunit comprises the folding of its helices and strands into a compact structure that is 13.5 nm long and divided into two domains. The membrane-distal domain (silver) is folded into a globular conformation. The membrane-proximal domain (gold) has a fibrous, stemlike conformation owing to the alignment of two long $\alpha$ helices (cylinders) of $\mathrm{HA}_{2}$ with $\beta$ strands in $\mathrm{HA}_{1}$. Short turns and longer loops, many of them at the surface of the molecule, connect the helices and strands in each chain. (b) The quaternary structure of HA is stabilized by lateral interactions between the long helices (cylinders) in the fibrous domains of the three subunits (gold, blue, and green), forming a triple-stranded coiled-coil stalk. Each of the distal globular domains in HA binds sialic acid (red) on the surface of target cells. Like many membrane proteins, HA contains several covalently linked carbohydrate chains (not shown). [Data from S. J. Gamblin et al., 2004, Science 303:1838-1842, PDB ID 1ruz.]
enzymes that cleave one or more peptide bonds in a target polypeptide. Alternatively, the DNA encoding a protein can be modified so that when the modified DNA is used to generate a protein, only a particular region, or domain, of the full-length protein is made. Thus it is possible to determine if specific parts of a protein are responsible for particular activities exhibited by the protein. Indeed, functional domains are often also associated with corresponding structural domains.

A structural domain is a region about 40 or more amino acids in length, arranged in a single, stable, and distinct structure often comprising one or more secondary structures. Many structural domains can fold into their characteristic structures independently of the rest of the protein in which they are embedded. As a consequence, distinct structural domains can be linked together-sometimes by short or long spacers-to form a large multidomain protein. Each of the polypeptide chains in the trimeric flu virus hemagglutinin, for example, contains a globular domain and a fibrous domain (Figure 3-11a). Structural domains can be incorporated as modules into different proteins. The modular approach to protein architecture is particularly easy to recognize in large proteins, which tend to be mosaics of different domains that confer distinct activities and thus can perform different functions simultaneously. As many as 75 percent of the proteins in eukaryotes have multiple structural domains. Structural domains frequently are also functional domains in that they can have an activity independent of the rest of the protein.

The epidermal growth factor (EGF) domain is a structural domain that is present in several proteins (Figure 3-12). EGF
is a small, soluble peptide hormone that binds to cells in the embryo and in skin and connective tissue in adults, causing them to divide. It is generated by proteolytic cleavage (breaking of a peptide bond) between repeated EGF domains in the EGF precursor protein, which is anchored in the plasma membrane by a membrane-spanning domain. EGF domains with sequences similar to, but not identical to, that of the EGF peptide hormone are present in other proteins and can be liberated by proteolysis. These proteins include tissue plasminogen activator (TPA), a protease that is used to dissolve blood
![li130857775e0](li130857775e0.jpg)

FIGURE 3-12 Modular nature of protein domains. Epidermal growth factor (EGF) is generated by proteolytic cleavage of a precursor protein containing multiple EGF domains (green) and a membranespanning domain (blue). An EGF domain is also present in the Neu protein and in tissue plasminogen activator (TPA). These proteins also contain other widely distributed domains, indicated by shape and color. See I. D. Campbell and P. Bork, 1993, Curr. Opin. Struc. Biol. 3:385.

clots in heart attack victims; Neu protein, which takes part in embryonic differentiation; and Notch protein, a receptor protein in the plasma membrane that functions in developmentally important signaling (see Chapter 16). Besides the EGF domain, these proteins have other domains in common with other proteins. For example, TPA possesses a trypsin domain, a functional domain found in some proteases. It is estimated that there are about a thousand different types of structural domains in all proteins. Some of these are not very common, whereas others are found in many different proteins. Indeed, by some estimates, only nine major types of structural domains account for as much as a third of all the structural domains in all proteins. Structural domains can be recognized in proteins whose structures have been determined by x-ray crystallography or nuclear magnetic resonance (NMR) analysis or in images captured by electron microscopy.

Regions of proteins that are defined by their distinctive spatial relationships to the rest of the protein are topological domains. For example, some proteins associated with cellsurface membranes have a part extending inward into the cytoplasm (cytoplasmic domain), a part embedded within the phospholipid bilayer (membrane-spanning domain), and a part extending outward into the extracellular space (extracellular domain). Each of these parts can comprise one or more structural and functional domains.

In Chapter 8, we will consider the mechanism by which the gene segments that correspond to domains became shuffled in the course of evolution, resulting in their appearance in many proteins. Once a functional, structural, or topological domain has been identified and characterized in one protein, it is possible to use that information to search for similar domains in other proteins and to suggest potentially similar functions for those domains in those proteins.

## Multiple Polypeptides Assemble into Quaternary Structures and Supramolecular Complexes

Multimeric proteins consist of two or more polypeptide chains, which in this context are referred to as subunits. A fourth level of structural organization, quaternary structure, describes the number (stoichiometry) and relative positions of the subunits in multimeric proteins (Figure 3-2). Flu virus hemagglutinin, for example, is a trimer of three identical subunits (a homotrimer) held together by noncovalent bonds (Figure 3-11b). Other multimeric proteins are composed of various numbers of identical (homomeric) or different (heteromeric) subunits. Hemoglobin, the oxygen-carrying molecule in blood, is an example of a heteromeric multimeric protein, as it has two copies each of two different polypeptide chains (as discussed below). In many cases, the individual monomer subunits of a multimeric protein cannot function normally unless they are assembled into the multimeric protein. In other cases, assembly into a multimeric protein permits proteins that act sequentially in a pathway to increase their efficiency of operation owing to their juxtaposition in space, a phenomenon referred to as metabolic coupling. Classic examples of metabolic coupling are the fatty acid synthases, the enzymes in fungi that synthesize fatty acids, and the polyketide synthases,
the large multiprotein complexes in bacteria that synthesize a diverse set of pharmacologically relevant molecules called polyketides, including the antibiotic erythromycin.

The highest level in the hierarchy of protein structure is the association of proteins into supramolecular complexes. Typically, such structures are very large, in some cases exceeding 1 megadalton (MDa) in mass, approaching 30-300 nm in size, and containing tens to hundreds of polypeptide chains and sometimes other biopolymers such as nucleic acids. The capsid that encases the nucleic acids of the viral genome is an example of a supramolecular complex with a structural function. The bundles of cytoskeletal filaments that support and give shape to the plasma membrane are another example. Other supramolecular complexes act as molecular machines, carrying out the most complex cellular processes by integrating multiple proteins, each with distinct functions, into one large assembly. For example, a transcriptional machine is responsible for synthesizing messenger RNA (mRNA) using a DNA template. This transcriptional
![li131857775e0](li131857775e0.jpg)

FIGURE 3-13 A molecular machine: the transcription initiation complex. The core RNA polymerase, general transcription factors, a mediator complex containing about 20 subunits, and other protein complexes not depicted here assemble at a promoter in DNA. The polymerase carries out transcription of DNA; the associated proteins are required for initial binding of the polymerase to a specific promoter. The multiple components function together as a molecular machine.

machine, the operational details of which are discussed in Chapters 5 and 9, consists of RNA polymerase, itself a multimeric protein, and at least 50 additional components, including general transcription factors, promoter-binding proteins, helicase, and other protein complexes (Figure 3-13). Ribosomes, also discussed in Chapter 5, are complex multiprotein and multi-nucleic acid machines that synthesize proteins. One of the most complex multiprotein assemblies is the nuclear pore, a structure that allows communication and passage of macromolecules between the nucleus and the cytoplasm (see Chapter 14). It is composed of multiple copies of about 30 distinct proteins and forms an assembly with an estimated mass of 50 MDa . The fatty acid synthases and polyketide synthases referred to above are also molecular machines.

## Comparing Protein Sequences and Structures Provides Insight into Protein Function and Evolution

Analyses of many diverse proteins have conclusively established a relation between the amino acid sequence, threedimensional structure, and function of proteins. One of the earliest examples involved a comparison of two oxygencarrying proteins: myoglobin in muscle and hemoglobin in red blood cells. Myoglobin-a monomer (consisting of one polypeptide chain/protein molecule)—and hemoglobin-a
tetramer (consisting of two $\alpha$ and two $\beta$ polypeptides, or subunits, per protein)—both contain a heme group noncovalently attached to each polypeptide chain (Figure 3-14a). The heme group binds oxygen. A mutation in the gene encoding the $\beta$ chain of hemoglobin that results in the substitution of a valine for a glutamic acid disturbs this protein's folding and function and causes sickle-cell disease (also called sickle-cell anemia). The properly aligned sequences of the 141-residue myoglobin and the 153-residue $\beta$ subunit of hemoglobin have 40 residues in equivalent positions in the sequences that are identical and another 21 that have side chains that are chemically very similar. This high degree of identity and similarity ( 43 percent of the myoglobin residues) is consistent with their similar oxygen-binding functions. X-ray crystallographic analysis showed that the three-dimensional structures of myoglobin and of the $\alpha$ and $\beta$ subunits of hemoglobin, as well as that of the evolutionarily distant oxygen-carrying leghemoglobin from plants, are remarkably similar (see Figure 3-14a).

A good rule of thumb is that the greater the similarity of the sequences of two polypeptide chains, the more likely they are to have similar three-dimensional structures and similar functions. While this comparative approach is very powerful, caution must always be exercised when attributing to one protein, or a part of a protein, a function or structure similar to that of another protein based only on amino acid sequence
![li132857775e0](li132857775e0.jpg)

FIGURE 3-14 Evolution of the globin protein family.
(a) Hemoglobin is a tetramer of two $\alpha$ and two $\beta$ subunits. The structural similarity of these subunits to leghemoglobin and myoglobin, both of which are monomers, is evident. A heme molecule (red) noncovalently associated with each globin polypeptide is directly responsible for oxygen binding in these proteins. (b) A primitive monomeric oxygen-binding globin is thought to be the ancestor of modern-day blood hemoglobins, muscle myoglobins, and plant leghemoglobins. Sequence comparisons have revealed that the evolution of the globin
proteins parallels the evolution of animals and plants. Major changes occurred with the divergence of plant globins from animal globins and of myoglobin from hemoglobin. Later, gene duplication gave rise to the $\alpha$ and $\beta$ subunits of hemoglobin. See R. C. Hardison, 1996, P. Natl. Acad. Sci. USA 93:5675. [Part (a) data from G. Fermi et al., 1984, J. Mol. Biol. 175:159-174, PDB ID 2hbb (hemoglobin), H. C. Watson, 1969, Prog. Stereochem. 4:299, PDB ID 1mbn (myoglobin), and M. S. Hargrove et al., 1997, J. Mol. Biol. 266:1032-1042, PDB ID 1bin (leghemoglobin).]

similarities. There are examples in which proteins with similar overall structures display different functions, as well as cases in which functionally unrelated proteins with dissimilar amino acid sequences nevertheless have very similar folded tertiary structures, as will be explained below. Nevertheless, in many cases, such comparisons of sequences provide important insights into protein structure and function.

Use of sequence comparisons to deduce protein structure and function has expanded substantially in recent years as the genomes and messenger RNAs of more and more organisms have been sequenced, permitting a vast array of protein sequences to be deduced. Indeed, the molecular revolution in biology during the last decades of the twentieth century created a new scheme of biological classification based on similarities and differences in the amino acid sequences of proteins. Proteins that have a common ancestor are referred to as homologs. The main evidence for homology among proteins, and hence for their common ancestry, is similarity in their sequences, which is often reflected in similar structures. We can describe homologous proteins as belonging to a "family" and can trace their lineage-how closely or distantly they are related to one another in an evolutionary sense-from comparisons of their sequences. Generally, more closely related proteins exhibit greater sequence similarity than more distantly related proteins because, over evolutionary time, mutations accumulate in the genes encoding these proteins. The folded three-dimensional structures of homologous proteins may be similar even if some parts of their primary structure show little evidence of sequence homology. Initially, proteins with relatively high sequence similarities ( $>50$ percent exact amino acid matches, or "identities") and related functions or structures were defined as an evolutionarily related family, while a superfamily encompassed two or more families in which the interfamily sequences matched less well ( $\sim 30-40$ percent identities) than within one family. It is generally thought that proteins with about 30 percent sequence identity are likely to have similar three-dimensional structures; however, such high sequence identity is not required for proteins to share similar structures. Revised definitions of family and superfamily have been proposed, in which a family comprises proteins with a clear evolutionary relationship ( $>30$ percent identity or additional structural and functional information showing common descent but $<30$ percent identity), while a superfamily comprises proteins with only a probable common evolutionary origin-for example, lower sequence identities but one or more common motifs or domains.

The kinship among homologous proteins is most easily visualized by a tree diagram based on sequence analyses. For example, the amino acid sequences of globins-the proteins hemoglobin and myoglobin and their relatives from bacteria, plants, and animals-suggest that they evolved from an ancestral monomeric oxygen-binding protein (Figure 3-14b). With the passage of time, the gene for this ancestral protein slowly changed, initially diverging into lineages leading to animal and plant globins. Subsequent changes gave rise to myoglobin and to the $\alpha$ and $\beta$ subunits of the tetrameric hemoglobin molecule $\left(\alpha_{2} \beta_{2}\right)$ of the vertebrate circulatory system.

## KEY CONCEPTS OF SECTION 3.1

## Hierarchical Structure of Proteins

- Proteins are linear polymers of amino acids linked together by peptide bonds. A protein can have a single polypeptide chain or multiple polypeptide chains. The primary structure of a polypeptide chain is the sequence of covalently linked amino acids that compose the chain. Various, mostly noncovalent interactions between amino acids in the linear sequence stabilize a protein's specific folded three-dimensional structure, or conformation.
- The $\alpha$ helix, $\beta$ strand and sheet, and $\beta$ turn are the most prevalent elements of protein secondary structure. Secondary structures are stabilized by hydrogen bonds between atoms of the peptide backbone (see Figures 3-4-3-6).
- Protein tertiary structure results from hydrophobic interactions between nonpolar side groups and from hydrogen bonds and ionic interactions involving polar side groups and the polypeptide backbone. These interactions stabilize the folding of the protein, including its secondary structural elements, into an overall three-dimensional arrangement.
- Entire proteins or segments of proteins usually fall into one of four broad structural categories: globular proteins, fibrous proteins, integral membrane proteins, and intrinsically disordered proteins.
- The exceptional conformational flexibilities of intrinsically disordered proteins contribute to their functions as binding partners, signaling molecules, regulators of other molecules, scaffolds, flexible links between well-ordered regions of a protein, sites of post-translational protein modification, autoinhibitors, and signals for intracellular protein sorting.
- Certain combinations of secondary structures give rise to structural motifs, which are found in a variety of proteins and are often associated with specific functions (see Figure 3-10).
- Proteins often contain distinct domains, independently folded regions with characteristic structural, functional, and/ or topological properties.
- The incorporation of domains as modules in different proteins in the course of evolution has generated diversity in protein structure and function.
- The number and organization of individual polypeptide subunits in multimeric proteins define their quaternary structure.
- Cells contain large supramolecular assemblies, sometimes called molecular machines, in which all the necessary participants in complex cellular processes (e.g., DNA, RNA, and protein synthesis; photosynthesis; signal transduction) are bound together.
- Proteins with similar amino acid sequences generally can be assumed to have similar three-dimensional structures and similar functions. There are also examples of polypeptide chains with dissimilar sequences folding into similar threedimensional structures.

- Homologous proteins are proteins that evolved from a common ancestor and thus have similar sequences, structures, and functions. They can be classified into families and superfamilies.


### 3.2 Protein Folding

As noted above, when it comes to the architecture of proteins, "form follows function." Thus it is essential that a polypeptide be synthesized with the proper amino acid sequence, and that it fold into the proper three-dimensional conformation, with the appropriate secondary, tertiary, and possibly quaternary structure, if it is to fulfill its biological role within or outside cells. How is a protein with a proper sequence generated? A polypeptide chain is synthesized by a complex process called translation, which occurs in the cytoplasm on a large proteinnucleic acid complex called a ribosome. During translation, a sequence of messenger RNA (mRNA) serves as a template for the assembly of a corresponding amino acid sequence. The mRNA is initially generated by a process called transcription, whereby a nucleotide sequence in DNA is converted, by transcriptional machinery in the nucleus, into a sequence of mRNA. The intricacies of transcription and translation are considered in Chapter 5. Here we describe the key determinants of the proper folding of a newly formed or forming (nascent) polypeptide chain as it emerges from the ribosome.

## Planar Peptide Bonds Limit the Shapes into Which Proteins Can Fold

A critical structural feature of polypeptides that limits how the chain can fold is the planar structure of the peptide bond. Figure 3-3 illustrates the amide group in peptide bonds in a polypeptide chain. Because the peptide bond itself behaves somewhat like a planar double bond (center and right),
![li133857775e0](li133857775e0.jpg)
the portions of the polypeptide chain on either side of the peptide bond ( $\mathrm{P}_{1}$ and $\mathrm{P}_{2}$ ) can be oriented in either a trans (center) or cis (right) configuration relative to the peptide bond. We saw similar cis and trans isomers of carbon-carbon double bonds in unsaturated fatty acids in Chapter 2. Analysis of crystal structures indicates that in proteins, about 99.97 percent of the peptide bonds that have any residue other than proline at $\mathrm{P}_{2}$ are in the trans configuration. (We will consider those with proline at $\mathrm{P}_{2}$ shortly.) In a peptide bond, the carbonyl carbon and amide nitrogen and those atoms directly bonded to them must all lie in a fixed plane (Figure 3-15); little rotation about the peptide bond itself is possible. As a consequence, the only flexibility in a polypeptide chain, allowing it to twist and turn-and thus fold into different three-dimensional
![li134857775e0](li134857775e0.jpg)

FIGURE 3-15 Rotation between planar peptide groups in proteins. Rotation about the $\mathrm{C}_{\mathrm{n}}$-amino nitrogen bond (the $\Phi$ angle) and the $\mathrm{C}_{\mathrm{n}}$-carbonyl carbon bond (the $\Psi$ angle) permits polypeptide backbones, in principle, to adopt a very large number of potential conformations. However, steric restraints due to the structure of the polypeptide backbone and the properties of the amino acid side chains dramatically restrict the potential conformations that any given protein can assume.
shapes-is rotation of the fixed planes of adjacent peptide bonds with respect to one another about two bonds: the $\mathrm{C}_{\mathrm{n}}-$ amino nitrogen bond (rotational angle called $\Phi$ ) and the $\mathrm{C}_{\mathrm{n}}-$ carbonyl carbon bond (rotational angle called $\Psi$ ).

Yet a further constraint on the potential conformations that a polypeptide chain can adopt is the fact that only a limited number of $\Phi$ and $\Psi$ angles are possible because for most $\Phi$ and $\Psi$ angles, the backbone or side-chain atoms would come too close to one another, and thus the associated conformation would be highly unstable or even physically impossible to achieve.

## The Amino Acid Sequence of a Protein Determines How It Will Fold

While the constraints of backbone bond angles seem very restrictive, any polypeptide chain containing only a few residues could, in principle, still fold into many conformations. For example, if the $\Phi$ and $\Psi$ angles were limited to only eight combinations, an $n$-residue-long peptide would potentially have $8^{n}$ conformations; for even a small polypeptide of only 10 residues, that's about 8.6 million possible conformations! In general, however, the native state of any particular protein that is not intrinsically disordered adopts only one or a few very closely related conformations; for the vast majority of these proteins, the native state is a stably folded form of the molecule and the one that permits it to function normally. In thermodynamic terms, the native state is usually the conformation with the lowest free energy $(G)$ (see Chapter 2).

What features of natively well-ordered proteins limit their folding from so many potential conformations to just one or a few? The properties of the side chains (e.g., size, hydrophobicity, ability to form hydrogen and ionic bonds), together with their particular sequence along the polypeptide backbone, impose key restrictions. For example, a large side chain, such as that of tryptophan, might sterically block

one region of the chain from packing closely against another region, whereas a side chain with a positive charge, such as that of arginine, might attract a segment of the polypeptide that has a complementary negatively charged side chain (e.g., aspartic acid). Another example we have already discussed is the effect of the aliphatic side chains in heptad repeats in promoting the association of helices and the consequent formation of coiled coils. Thus a polypeptide's primary structure determines its secondary, tertiary, and quaternary structures.

The initial evidence that the information necessary for a protein to fold properly is encoded in its amino acid sequence came from in vitro studies (in test tubes) on the refolding of purified proteins, especially the Nobel Prize-winning studies in the 1960s by Christian Anfinsen of the refolding of ribonuclease A, an enzyme that cleaves RNA. Others had previously shown that various chemical and physical perturbations can disrupt the weak noncovalent interactions that stabilize the native conformation of a protein, leading to the loss of its normal tertiary structure. The disruption of a protein's structure (and this can include secondary as well as tertiary structure) is called denaturation. Denaturation can be induced by thermal energy from heat, extremes of pH that alter the charges on amino acid side chains, or exposure to denaturants such as urea or guanidine hydrochloride at concentrations of $6-8 \mathrm{M}$, all of which disrupt structure-stabilizing noncovalent interactions. Treatment with reducing agents, such as $\beta$-mercaptoethanol, that break disulfide bonds can further destabilize disulfide-containing proteins. Under denaturing conditions, a population of uniformly folded protein molecules is destabilized and converted into a collection of many unfolded, or denatured, molecules that have many different nonnative and biologically inactive conformations. As we have seen, a large number of possible non-native conformations exist (e.g., $8^{\circ}-1$ ). There are two broad classes of non-native conformations seen in proteins: (1) monomeric unfolded or denatured structures and (2) aggregates, which can either be amorphous or have a well-organized structure, as is the case for the disease-associated amyloid fibrils described later in this chapter. In principle, aggregates can comprise many copies of a single protein (homogeneous aggregates) or contain a mixture of distinct proteins (heterogeneous aggregates).

The spontaneous unfolding of proteins under denaturing conditions is not surprising, given the substantial increase in entropy that occurs because a denatured protein can adopt many non-native conformations (increased disorder). What is striking, however, is that when a pure sample of a single type of unfolded protein in a test tube is shifted back very carefully to normal conditions (body temperature, normal pH levels, reduction in the concentration of denaturants), some denatured polypeptides can spontaneously refold into their native, biologically active states, as in Anfinsen's experiments. This kind of refolding experiment, as well as studies showing that synthetic proteins made chemically can fold properly, established that the information contained in a protein's primary structure can be sufficient to direct correct refolding. Newly synthesized proteins appear to fold into their proper conformations just as denatured proteins
![li135857775e0](li135857775e0.jpg)

FIGURE 3-16 Hypothetical protein-folding pathway. Folding of a monomeric protein follows the structural hierarchy of primary (a) $\rightarrow$ secondary (b-d) $\rightarrow$ tertiary (e) structure. Formation of small structural motifs (c) appears to precede formation of domains (d) and the final tertiary structure (e).
do. The observed similarity in the folded, three-dimensional structures of proteins with similar amino acid sequences, noted in Section 3.1, provided additional evidence that the primary sequence also determines protein folding in vivo (in live organisms). It appears that formation of secondary structures and structural motifs occurs early in the folding process, followed by assembly of more complex structural domains, which then associate into more complex tertiary and quaternary structures (Figure 3-16).

## Folding of Proteins in Vivo Is Promoted by Chaperones

The conditions under which a purified, denatured protein refolds in a test tube differ markedly from the conditions

under which a newly synthesized polypeptide folds in a cell. The presence of other biomolecules, some of which are themselves nascent and in the process of folding, can potentially interfere with the autonomous, spontaneous folding of an otherwise natively well-ordered protein by forming aggregates. The cytosolic concentrations of some proteins are very high, and the total cytosolic protein concentration can be $\sim 300 \mathrm{mg} / \mathrm{ml}$ in mammalian cells. These high protein concentrations favor aggregate formation by increasing the chances a nascent protein will encounter other proteins prior to completing its folding. Unfolded and partly folded proteins tend to aggregate into large, often water-insoluble masses, from which it is extremely difficult for a protein to dissociate and then fold into its proper conformation. In part, this aggregation is due to the exposure of hydrophobic side chains that have not yet had a chance to be buried in the inner core of the folded protein. Exposed hydrophobic side chains on different molecules will stick to one another, owing to the hydrophobic effect (see Chapter 2), and thus promote aggregation. The risk of such aggregation is especially high for newly synthesized proteins that have not yet completed their proper folding. Intrinsically disordered proteins are much less likely to form aggregates because, at least in some cases, they have relatively fewer hydrophobic side chains that can mediate such aggregation. Although protein folding into a well-ordered native state can occur in vitro, this does not happen for all unfolded molecules in a timely fashion because of the very large number of potentially incorrect, intermediate conformations into which the protein might fold.

Given such impediments, cells require faster, more efficient mechanisms for folding natively well-ordered proteins into their correct shapes than sequence alone provides. Without such help, cells might waste much energy in the synthesis of improperly folded, nonfunctional proteins, which would have to be destroyed to prevent their disrupting cell function. Cells clearly have such mechanisms, since more than 95 percent of the proteins present within cells have been shown to be in their native conformations. Proteins that do not or cannot fold properly-for example, those encoded by genes with mutations that alter the amino acid sequence-are often recognized as unfolded and rapidly degraded (hydrolyzed) by enzymes. The explanation for the cell's remarkable efficiency in promoting proper protein folding is that cells make a set of proteins, called chaperones, that facilitate proper folding of nascent proteins. One way chaperones facilitate proper folding is to prevent aggregation by binding to the target polypeptide or sequestering it from other partially or fully unfolded proteins, thus giving the nascent protein time to fold properly. The importance of chaperones is highlighted by the observation that many are evolutionarily conserved. Chaperones are found in all organisms from bacteria to humans, and some are homologs with high sequence similarity that use almost identical mechanisms to assist protein folding.

Chaperones can fold newly made proteins into functional conformations, refold misfolded or unfolded proteins into functional conformations, disassemble potentially toxic protein aggregates that form due to protein misfolding,
assemble and dismantle large multiprotein complexes, and mediate transformations between inactive and active forms of some proteins. Chaperones, which in eukaryotes are located in every cellular compartment and organelle, bind to the target proteins-also called substrates or client proteins-whose folding they will assist. Chaperones use a cycle of ATP binding, ATP hydrolysis to ADP, and exchange of a new ATP molecule for the ADP to induce a series of conformational changes that are essential for their function. There are several different classes of chaperones with distinct structures, all of which use ATP binding and hydrolysis in a variety of ways, which include (1) enhancing the binding of the target protein and (2) switching their own conformation. This ATP-dependent conformational switching is used (1) to optimize folding, (2) to return the chaperone to its initial state so that it is available to help fold another molecule, and (3) to set the time permitted for refolding, which can be determined by the rate of ATP hydrolysis.

Two general families of chaperones have been identified:

- Molecular chaperones, which bind to a short segment of a protein substrate and stabilize unfolded or partly folded proteins, thereby preventing these proteins from aggregating and being degraded.
- Chaperonins, which form folding chambers into which all or part of an unfolded protein can be sequestered, giving it time and an appropriate environment to fold properly.

Molecular Chaperones The heat-shock protein Hsp70 in the cytosol and its homologs (Hsp70 in the mitochondrial matrix, BiP in the endoplasmic reticulum, and DnaK in bacteria) are molecular chaperones. They were first identified by their rapid appearance after a cell had been stressed by heat shock (Hsp stands for "heat-shock protein"). Hsp70 and its homologs are the major chaperones in all organisms that use an ATP-dependent cycle to fold their substrates (Figure 3-17a). When bound to ATP, the monomeric Hsp70 protein assumes an open conformation, in which an exposed hydrophobic substrate-binding pocket transiently binds to exposed hydrophobic regions of an incompletely folded or partially denatured target protein, and then rapidly releases this substrate, as long as ATP is bound (step in Figure 3-17a). Hydrolysis of the bound ATP causes the molecular chaperone to assume a closed form that binds its substrate protein much more tightly, and this tighter binding appears to facilitate the target protein's folding, in part by preventing it from aggregating with other unfolded proteins (step in Figure 3-17a). Next the exchange of ATP for the chaperone-bound ADP (step $\mathbf{\square}$ ) causes a conformational change in the chaperone that releases the target protein and regenerates an "empty," ATP-bound Hsp70 ready to help fold another protein (step $\mathbf{\square}$ ). If the target is now properly folded, it cannot rebind to an Hsp70. If it remains at least partially unfolded, it can bind again to give a chaperone another chance to help fold it properly. As we will see later in this chapter, a variety of proteins use a cycle of trinucleotide hydrolysis to a dinucleotide, followed by

![li136857775e0](li136857775e0.jpg)

FIGURE 3-17 Molecular chaperone-mediated protein folding.
(a) Hsp70. Many proteins fold into their proper three-dimensional structures with the assistance of Hsp70 or one of several Hsp70-like proteins. These molecular chaperones transiently bind to a nascent polypeptide as it emerges from a ribosome or to a protein that has otherwise unfolded. In the Hsp70 cycle, an unfolded substrate protein binds in rapid equilibrium (step $\mathbf{\square}$ ) to Hsp70's substrate-binding site (red) in the open conformation of its substrate-binding domain (light and dark orange) when an ATP (purple) is bound at Hsp70's nucleotide-binding domain (light blue). The substrate-binding domain comprises two subdomains (light and dark orange) that change relative positions and conformations during the cycle. Co-chaperone accessory proteins (DnaJ/Hsp40) stimulate the hydrolysis of ATP to ADP (yellow) that induces a large conformational change in the substrate-binding domain, resulting in the closed conformation, in which the substrate is locked into the substratebinding domain; here proper folding is facilitated (steps $\mathbf{\square}$ and $\mathbf{\square}$ ). Exchange of ATP for the bound ADP, stimulated by other accessory cochaperone proteins (GrpE/BAG1), converts the Hsp70 back to the open conformation (step $\mathbf{\square}$ ), releasing the properly folded substrate (step $\mathbf{\square}$ ) and regenerating the open conformation, which can then interact with additional substrates. (b) Three conformational states of the dimeric Hsp90 molecular chaperone thought to be involved in substrate (also called client) remodeling. Client proteins bind at the substrate-binding
dinucleotide/trinucleotide exchange, to control their activities. Later in this chapter, we will discuss a group of proteins called GTPases that depend on the exchange of GTP, rather than ATP, for bound GDP (instead of ADP) to induce conformational changes that dramatically influence the proteins' activities and the subsequent hydrolysis of the bound GTP to GDP.
![li137857775e0](li137857775e0.jpg)
site (red surface) shared by the substrate-binding (orange) and C-terminal dimerization (white) domains and are thought to be remodeled in response to ATP binding and hydrolysis. The Hsp90 cycle begins when there is no nucleotide bound to the nucleotide-binding domains (light blue) and the dimer is in a very flexible, open configuration (step $\mathbf{\square}$ ) that can bind a client. Rapid ATP binding leads to a conformational change (step $\mathbf{\square}$ ) in which the nucleotide-binding domains and the substratebinding domains move together (intermediate shown in step $\mathbf{\square}$ ) into a closed conformation in which the nucleotide-binding domains are dimerized (step $\mathbf{\square}$ ). The precise locations in Hsp90 at which clients bind apparently vary for different clients, but the binding surface, including the intersection of the substrate-binding domains and C-terminal dimerization domains (highlighted by red shading) binds a number of clients. ATP hydrolysis results in a conformational change in Hsp90 (step $\mathbf{\square}$ ) that may include a highly compact form, folding of the client, and client protein release. The ADP-bound form of Hsp90 can adopt several conformations, including a highly compact form. Release of ADP (step $\mathbf{\square}$ ) regenerates the initial flexible open state, which can then interact with additional clients. See E. D. Kirschke et al., 2014, Cell 157:1685 and M. Taipale, D. F. Jarosz, and S. Lindquist, 2010, Nat. Rev. Mol. Cell Biol. 11:515. (Solvent-accessible surface model of HSP90 courtesy of Elaine Kirschke and David A. Agard, UCSF. Open (ATP) PDB ID 2ior, closed (ATP) PDB ID 2cg9, closed (ADP) based on PDB ID 2cg9.)

Additional proteins, such as the co-chaperone Hsp40 in eukaryotes (DnaJ in bacteria), help increase the efficiency of the Hsp70-mediated folding of many proteins not only by stimulating the binding of substrate, but also by increasing the rate of hydrolysis of ATP by 100- to 1000-fold (see step $\mathbf{\square}$ in Figure 3-17a). Members of four different families of nucleotide

exchange factors (e.g., GrpE in bacteria; BAG, HspBP, and Hsp110 in eukaryotes) also interact with Hsp70 (or DnaK), promoting the exchange of ATP for ADP (see step $\mathbf{\square}$ ). Multiple molecular chaperones are thought to bind to all nascent polypeptide chains as they are being synthesized on ribosomes. In bacteria, 85 percent of the proteins are released from their chaperones and proceed to fold normally; an even higher percentage of proteins in eukaryotes follow this pathway.

The Hsp70 protein family is not the only class of molecular chaperones. Another distinct class of molecular chaperones is the Hsp90 family, whose members usually recognize partially folded substrate proteins. Evolutionarily related Hsp90 family members are present in all organisms except archaea. Their strong evolutionary conservation is seen in the high amino acid sequence similarity ( 55 percent) of the Hsp90 from the bacterium E. coli and human Hsp90. In most eukaryotes, there are four distinct Hsp90s, two of which are in the cytosol (at 1-2 percent of total protein, Hsp90 is one of the most abundant cytosolic proteins) and one each in the endoplasmic reticulum and the mitochondrion. Although the range of protein substrates for Hsp90 chaperones is not as broad as for some other chaperones (at least 10 percent of yeast proteins are thought to be Hsp90 substrates), the Hsp90s are essential in eukaryotes. The Hsp90s help cells cope with denatured proteins generated by stress (e.g., heat shock), and they ensure that some of their substrates, usually called "clients," can be converted from an inactive to an active state or otherwise held in a functional conformation. In some cases, an Hsp90 forms a relatively stable complex with a client until an appropriate signal causes its dissociation from the client, freeing the client to perform some regulated function in the cell. Hsp90 clients include transcription factors such as the receptors for the steroid hormones estrogen and testosterone. These steroid receptors regulate sexual development and function by controlling the activities of many genes (see Chapter 9). Another type of Hsp90 client is the set of enzymes called kinases, which control the activities of many proteins by phosphorylation (see Chapters 15 and 16).

Unlike monomeric Hsp70, Hsp90 functions as a dimer in a cycle in which ATP binding, hydrolysis, and ADP release are coupled to major conformational changes and to binding, folding or activation, and release of clients (Figure 3-17b). Although much about the mechanism of Hsp90 remains to be learned, it is clear that clients bind to the substrate-binding domains when the chaperone is in the "open" conformation (step $\mathbf{\square}$ in Figure 3-17b), that ATP binding leads to interaction of the ATP-binding domains and formation of a "closed" conformation (steps $\mathbf{\square}$ and $\mathbf{\square}$ in Figure 3-17b), and that hydrolysis of ATP plays an important role in activation of some client proteins and their subsequent release from the Hsp90 (step $\mathbf{\square})$. We also know that there are at least 20 co-chaperones that can have profound effects on the activity of Hsp90, including modulating its ATPase activity and determining which proteins will be clients (client specificity). Co-chaperones can also help coordinate the activities of Hsp90 and Hsp70. For example, Hsp70 can help begin the folding of a client that is then handed off by a co-chaperone to Hsp90 for additional
processing. Hsp90 activity can also be influenced by its covalent modification by small molecules. Finally, Hsp90 can help cells recognize misfolded proteins that are unable to refold and facilitate their degradation by mechanisms discussed later in this chapter. Thus, as part of the quality-control system in cells, molecular chaperones can help properly fold proteins or facilitate the destruction of those that cannot fold properly.

Chaperonins The proper folding of a large variety of newly synthesized proteins also requires the assistance of another class of proteins, the chaperonins, also called Hsp60s. These huge cylindrical supramolecular assemblies are formed from two rings of oligomers. There are two distinct groups of chaperonins that differ somewhat in their structures, detailed molecular mechanisms, and locations. Group I chaperonins, found in prokaryotes, chloroplasts, and mitochondria, are composed of two rings, each having seven subunits that interact with a homoheptameric co-chaperone "lid." The bacteria group I chaperonin, known as GroEL/GroES, is shown in Figure 3-18a. In the bacterium E. coli, GroEL is thought to participate in the folding of about 10 percent of all proteins. Group II chaperonins, which are found in the cytosol of eukaryotic cells (e.g., TriC in mammals) and in archaea, can have eight to nine either homomeric or heteromeric subunits in each ring, and the "lid" function is incorporated into those subunits themselves-no separate lid protein is needed. It appears that ATP hydrolysis triggers the closing of the lid of group II chaperonins.

Figure 3-18b illustrates the GroEL/GroES cycle of protein folding. A partially folded or misfolded polypeptide of less than 60 kDa in mass is captured by hydrophobic residues near the entrance of the GroEL chamber and enters one of the folding chambers (upper chamber in Figure 3-18b). The second chamber is blocked by a GroES lid. Each of the 14 subunits of GroEL can bind ATP, hydrolyze it, and subsequently release ADP. These reactions are concerted for each set of seven subunits in a single ring and lead to major conformational changes. These changes control both the binding of the GroES lid that seals the chamber and the environment of the chamber in which polypeptide folding takes place. The polypeptide remains encased in the chamber capped by the lid. There it can undergo folding until ATP hydrolysis in that chamber, which is the slowest, rate-limiting step in the cycle ( $t_{1 / 2} \sim 10 \mathrm{~s}$ ), induces binding of ATP and a different GroES to the other ring. This then causes the GroES lid and ADP bound to the peptide-containing ring to be released, opening the chamber and permitting the folded protein to diffuse out of the chamber. If the polypeptide is folded properly, it can proceed to function in the cell. If it remains partially folded or misfolded, it can rebind to an unoccupied GroEL and the cycle can be repeated. There is a reciprocal relationship between the two rings in one GroEL complex. The capping of one chamber by GroES to permit sequestered substrate folding in that chamber is accompanied by the release of substrate polypeptide from the chamber of the second ring (simultaneous binding, folding, and release from the second chamber is not illustrated in Figure 3-18b). There is a striking similarity between the capped-barrel design of GroEL/GroES, in which proteins are sequestered for

![li138857775e0](li138857775e0.jpg)

FIGURE 3-18 Chaperonin-mediated protein folding. Proper folding of some proteins depends on chaperonins such as the prokaryotic group I chaperonin GroEL. (a) GroEL is a barrel-shaped complex of fourteen identical 60,000-MW subunits, arranged in two stacked rings (blue) of seven subunits each that form two distinct internal polypeptide folding chambers. Homoheptameric lids (10,000-MW subunits), GroES (red), can bind to either end of the barrel and seal the chamber on that side. (b) The GroEL-GroES folding cycle. A partly folded or misfolded polypeptide enters one of the folding chambers (step 1). The second chamber is blocked by a GroES lid. Each ring of seven GroEL subunits binds seven ATPs, hydrolyzes them, and then releases the ADPs in a set order coordinated with GroES binding and release and polypeptide binding, folding, and release. The major conformational changes that take place in the GroEL rings control the binding of the GroES lid that seals the chamber (step 2). The polypeptide remains encased in the chamber capped by the lid, where it can undergo folding until ATP hydrolysis-the slowest, rate-limiting step in the cycle ( $t_{\mathrm{N}} \sim 10 \mathrm{~s}$ ) (step 3)—induces binding of ATP and a different GroES to the other ring (transient intermediate shown in brackets). This binding then causes the GroES lid and ADP bound to the peptide-containing ring to be released, opening the chamber and permitting the folded protein to diffuse out of the chamber (step 4). If the polypeptide has folded properly, it can proceed to function in the cell. If it remains partially folded or misfolded, it can rebind to an unoccupied GroEL and the cycle can be repeated. See D. L. Nelson and M. M. Cox, 2013, Lehninger Principles of Biochemistry, 6th ed., Macmillan. [Part (a) data from Z. Xu, A. L. Horwich, and P. B. Siegler, 1997, Nature 388:741-750, PDB ID 1aon.]

Folding, and the structure of the 26 S proteasome that participates in protein degradation (discussed in Section 3.4). In addition, a group of proteins that are part of the $\mathrm{AAA}^{+}$ family of ATPases are composed of hexameric rings with a central pore into which substrates can enter for folding or unfolding or in some cases proteolysis; examples of these will be discussed in Section 3.4 and in Chapter 13.

## Protein Folding Is Promoted by Proline Isomerases

As we learned earlier, the portions of the polypeptide chain on either side of a peptide bond ( $\mathrm{P}_{1}$ and $\mathrm{P}_{2}$ ) are almost always oriented in a trans configuration (Figure 3-19a).

However, the trans configuration is not dramatically more energetically favorable than a cis configuration when there is a proline at $\mathrm{P}_{2}$ (Figure 3-19b). Among those folded proteins whose structures have been determined, about 5 percent of peptide bonds with proline at $\mathrm{P}_{2}$ exhibit the cis configuration, as compared with 0.03 percent of all other peptide bonds without proline at $\mathrm{P}_{2}$. As the rate of isomerization between the cis and trans configurations is relatively slow, cells use proline isomerase proteins to catalyze these cis/trans isomerizations to facilitate the folding with the proper isomer. Prolyl isomerizations have been proposed to act as switches to alter the conformation, and thus the activity, of already stably folded proteins. Indeed, such isomerizations can substantially alter the structure of some proteins (Figure 3-19c).

![li139857775e0](li139857775e0.jpg)

FIGURE 3-19 Proline cis/trans isomerizations influence protein folding and structure. (a) The planar, double bond-like character of peptide bonds leads to the potential of the portions of the polypeptide chain on either side ( $\mathrm{P}_{1}$ and $\mathrm{P}_{2}$ ) having cis or trans configurations. The trans configuration is present in about 99.97 percent of all peptide bonds in well-ordered proteins when $\mathrm{P}_{2}$ is a residue other than proline. (b) When $\mathrm{P}_{2}$ is proline, about 5 percent of peptide bonds are in the cis configuration. Proline isomerases catalyze the cis/trans isomerization to facilitate protein folding. (c) The structure of a portion of a protein, here an SH2 protein domain (see Chapter 16), can be dramatically altered by the cis/trans isomerization of a single proline, and this structural change can influence the protein's activity. (Part (c) trans data from E. V. Pletneva et al., 2006, J. Mol. Biol. 357:550-561, PDB ID 2etz. Part (c) cis data from R. J. Mallis et al., 2002, Nat. Struct. Biol. 9:900-905. PDB ID 1lui.]

## Abnormally Folded Proteins Can Form Amyloids That Are Implicated in Diseases

After it is synthesized, a protein may fold into an alternative, abnormal three-dimensional structure as the result of mutations, inappropriate covalent modifications, or chemical (e.g., pH ) or physical (e.g., heat)
alterations in its environment. Misfolding or denaturation can lead to a loss of the normal function of the protein and can result in the protein being marked for destruction (proteolytic degradation), as described later in this chapter. However, when degradation is incomplete or fails to keep pace with the production of misfolded protein, the misfolded protein or its proteolytic fragments can accumulate either inside or outside of cells in aggregates, or plaques, in various organs, including joints between bones, the liver, and the brain. Even those proteins or protein fragments that are normally highly resistant to aggregation, as is the case for intrinsically disordered proteins or protein fragments, will form aggregates if their concentrations are sufficiently elevated or when there are changes in environmental conditions. As noted above, such aggregates can either be amorphous or have a well-organized structure, which most commonly is the amyloid state. Strikingly, many diverse proteins can each aggregate into amyloid fibrils that have a common structure, called a cross- $\beta$ sheet (Figure 3-20a). Short segments, generally $6-12$ residues long, in the unfolded or misfolded proteins hydrogen-bond to each other, forming a long array, or filament, of $\beta$ sheets. In these arrays, each $\beta$ strand is nearly perpendicular to the long axis of the filament, and two long, nearly flat $\beta$ sheets pack closely together and twist around each other to form protofilaments, which then assemble together into thicker filaments, called amyloid fibrils. Within each protofilament the $\beta$ strands can be either parallel or antiparallel (see Figure 3-5). Although some proteins form amyloid fibrils in their native, functional states, most amyloids are considered to be consequences of protein misfolding.

Amyloids were first recognized in protein aggregates that are deposited in tissues, are resistant to enzymatic degradation, and are associated with dozens of diseases, called amyloidoses. These diseases include neurodegenerative diseases such as Alzheimer's disease and Parkinson's disease in humans and transmissible spongiform encephalopathy ("mad cow" disease) in cows and sheep. Each of these diseases is characterized by the presence of filamentous plaques in a deteriorating brain (Figure 3-20b). Amyloidoses most commonly occur with aging; however, mutations in the genes encoding the aggregating protein can result in early amyloid formation and disease onset. The amyloid fibrils composing the plaques derive from abundant natural proteins. For example, fragments of the amyloid precursor protein, which is embedded in the plasma membrane, form the plaque found in the brains of patients with Alzheimer's disease; and prion protein, an "infectious" protein, forms fibrils in prion diseases. In Alzheimer's disease, a hyperphosphorylated form of the protein tau, normally a microtubulebinding protein (see Chapter 18), forms twisted fibers called "tangles." These amyloids, either as relatively short, watersoluble protofilaments or as long, insoluble fibrils, are thought to be toxic and to contribute directly to the pathology of amyloidoses.

![li140857775e0](li140857775e0.jpg)

FIGURE 3-20 Misfolded proteins can form ordered amyloid aggregates based on a cross- $\beta$ sheet structure. (a) In unfolded segments of proteins and polypeptides, exposed segments 6-12 residues long (short flat arrows) can assemble into $\beta$ sheets (see also Figure 3-5) in which each $\beta$ strand is oriented nearly perpendicularly to the long axis (vertical in this figure) of the resultant amyloid protofilament and hydrogen-bonded (light shading) to the strands above and below. Two long, nearly flat sheets pack closely together and twist around each other to form amyloid protofilaments, which then assemble together into thicker filaments called amyloid fibrils (b). Amyloid fibrils can be composed of varying numbers of protofilaments. A model of a four-protofilament-containing fibril fit into the electron density of acid-denatured insulin fibrils (left) and a cryoelectron microscopic image of two-protofilamentcontaining fibrils of fragments of transthyretin with an NMR-based model (yellow). Fibrils can aggregate into macroscopic plaques and tangles that are deposited in tissues and, when stained, are large enough to be visible using light microscopy. (c) Microscopic view of a section of human brain tissue from a patient with Alzheimer's disease with multiple amyloid plaques and fibrillary tangles. [Part (b, left) republished with permission of Elsevier, from Dobson, C.M., "Protein misfolding, evolution and disease," Trends in Biochemical Science 1999, 24(9):329-332. Fig. 3. Part (b, right) reprinted by permission from Macmillan Publishers Ltd: from Knowles et al., Nat. Rev. Mol. Cell Biol. 2014, 15(6):384-396. Fig. 3a. Part (c) Thomas Deerinck, NCMIR/Science Source.]

## KEY CONCEPTS OF SECTION 3.2

## Protein Folding

- The primary structure (amino acid sequence) of a protein determines its three-dimensional structure, which determines its function. In short, function derives from structure; structure derives from sequence.
- Because protein function derives from protein structure, newly synthesized proteins must fold into the correct shape to function properly.
- The planar structure of the peptide bond limits the number of conformations a polypeptide can have (see Figure 3-15).
- The amino acid sequence of a protein dictates its folding into a specific three-dimensional conformation, the native state. Proteins will unfold, or denature, if treated under conditions that disrupt the noncovalent interactions stabilizing their three-dimensional structures.
- There are two broad classes of non-native conformations seen in misfolded or denatured proteins: (1) monomeric unfolded or denatured structures and (2) aggregates, which can either be amorphous or have a well-organized structure.
- Protein folding in vivo occurs with assistance from ATPdependent chaperones. Chaperones can influence proteins in several ways, including preventing misfolding and

aggregation, facilitating proper folding, and maintaining an appropriate, stable structure required for subsequent protein activity (see Figure 3-17).

- There are two broad classes of chaperones: (1) molecular chaperones, which bind to a short segment of a substrate protein, and (2) chaperonins, which form folding chambers in which all or part of an unfolded protein can be sequestered, giving it time and an appropriate environment to fold properly. Cycles of ATP binding and hydrolysis, followed by exchange of the ADP produced with a new ATP molecule, play key roles in the mechanisms of protein folding by chaperones.
- Many misfolded or denatured proteins can form wellorganized aggregates, called amyloid fibrils, made by short stretches of polypeptide that form a long array of $\beta$ sheets nearly perpendicular to the fibril axis, called a cross- $\beta$ structure. Formation of amyloid fibrils that are resistant to degradation by diverse enzymes is associated with dozens of diseases called amyloidoses. Examples include the neurodegenerative diseases Alzheimer's disease and Parkinson's disease.


### 3.3 Protein Binding and Enzyme Catalysis

Proteins perform an extraordinarily diverse array of activities both inside and outside cells, yet most of these diverse functions are based on the ability of proteins to engage in a common activity: binding. Proteins bind to one another, to other macromolecules, to small molecules, and to ions. In this section, we describe some key features of protein binding and then turn to look at one group of proteins, enzymes, in greater detail. The activities of the other functional classes of proteins (structural, scaffold, transport, regulatory, motor) will be described in later chapters.

## Specific Binding of Ligands Underlies the Functions of Most Proteins

The molecule to which a protein binds is called its ligand. In some cases, ligand binding causes a change in the shape of a protein. Such conformational changes are integral to the mechanism of action of many proteins and are important in regulating protein activity.

Two properties of a protein characterize how it binds ligands. Specificity refers to the ability of a protein to bind one molecule or a very small group of molecules in preference to all other molecules. Affinity refers to the tightness or strength of binding, usually expressed as the dissociation constant $\left(K_{\mathrm{d}}\right)$. The $K_{\mathrm{d}}$ for a protein-ligand complex, which is the inverse of the equilibrium constant $K_{\text {eq }}$ for the binding reaction, is the most common quantitative measure of affinity (see Chapter 2). The stronger the interaction between a protein and ligand, the lower the value of $K_{\mathrm{d}}$. Both the specificity and the affinity of a protein for a ligand depend on the structure of the ligand-binding site. For high-affinity and highly specific interactions to take place, the shape and
chemical properties of the binding site must be complementary to those of the ligand molecule, a property termed molecular complementarity. As we saw in Chapter 2, molecular complementarity allows molecules to form multiple noncovalent interactions at close range and thus stick together.

One of the best-studied examples of protein-ligand binding, involving high affinity and exquisite specificity, is the binding of antibodies to antigens. Antibodies are proteins that circulate in the blood and are made by the immune system in response to antigens, which are usually macromolecules present in infectious agents (e.g., a bacterium or a virus) or other foreign substances (e.g., proteins or polysaccharides in pollens). Different antibodies are generated in response to different antigens, and these antibodies have the remarkable characteristic of binding specifically to ("recognizing") the part of the antigen, called an epitope, that initially induced the production of the antibody, and not to other molecules. Antibodies act as specific sensors for antigens, forming antibody-antigen complexes that initiate a cascade of protective reactions in cells of the immune system. Chapter 23 discusses antibodies and their roles in the immune system, and later in this chapter we will discuss techniques for studying proteins that exploit antibodies. Here we briefly introduce the structure of antibodies and their binding to epitopes.

Antibodies are Y-shaped molecules, often formed from two identical longer, or heavy, chains and two identical shorter, or light, chains. In IgG antibodies (also called immunoglobulins, shown in Figure 3-21a), there are four globular domains in each heavy chain and two in each light chain, all of which are called immunoglobulin (Ig) domains. Each of the two branching arms of an IgG antibody contains a single light chain linked to a heavy chain by a disulfide bond, and two disulfide bonds covalently link the two heavy chains together. Near the end of each arm are six highly variable loops, called complementarity-determining regions (CDRs), which form the antigen-binding sites. The sequences of the six loops are highly variable among antibodies, generating unique complementary ligand-binding sites that make them specific for different epitopes (Figure 3-21b). The intimate contact between antibody and epitope surfaces, stabilized by numerous noncovalent interactions, is responsible for the extremely precise binding specificity exhibited by an antibody.

The specificity of antibodies is so precise that they can distinguish between the cells of individual members of a species and in some cases between proteins that differ by only a single amino acid, or even between proteins with identical sequences that differ only in their post-translational modifications. Because of their specificity and the ease with which they can be produced (see Chapter 23), antibodies are highly useful reagents used in many of the experiments discussed in subsequent chapters.

We will see many examples of protein-ligand binding throughout this book, including binding of hormones to receptors (see Chapter 15), binding of regulatory molecules to DNA (see Chapter 9), and binding of cell-adhesion molecules to extracellular matrices (see Chapter 20), to name just a few. Here we focus on how the binding of one class of proteins, enzymes, to their ligands results in the catalysis of the chemical reactions essential for the survival and function of cells.

![li141857775e0](li141857775e0.jpg)

FIGURE 3-21 Protein-ligand binding of antibodies. (a) Hybrid (surface and ribbon) model of an antibody. Every antibody molecule of the immunoglobulin G (IgG) class consists of two identical heavy chains (medium and dark blue) and two identical light chains (light blue) covalently linked by disulfide bonds (yellow). The complementariydetermining regions (CDRs) that define the antigen-binding sites are represented by red shading. (b) The cartoon shows the overall structure

## Enzymes Are Highly Efficient and Specific Catalysts

Proteins that catalyze chemical reactions-the making and breaking of covalent bonds-are called enzymes, and the ligands of enzymes are called substrates. Enzymes make up a large and very important functional class of proteins-indeed, almost every chemical reaction in the cell is catalyzed by a specific catalyst, usually an enzyme. Another form of catalytic macromolecule in cells is made from RNA. These RNAs are called ribozymes (see Chapter 5).

Thousands of different types of enzymes, each of which catalyzes a single chemical reaction or a set of closely related reactions, have been identified. Certain enzymes are found in the majority of cells because they catalyze the synthesis of common cellular products (e.g., proteins, nucleic acids, and phospholipids) or take part in harvesting energy from nutrients (e.g., by the conversion of glucose and oxygen into carbon dioxide and water during cellular respiration). Other enzymes are present only in a particular type of cell because they catalyze chemical reactions unique to that cell type (e.g., the enzymes in neurons that convert tyrosine into dopamine, a neurotransmitter). Although most enzymes are located within cells, some are secreted and function at extracellular sites, such as the blood, the digestive tract, or even outside the organism (e.g., toxic enzymes in the venom of poisonous snakes).

Like all catalysts (see Chapter 2), enzymes increase the rate of a reaction, but they do not affect the extent of a reaction, which is determined by the change in free energy $(\Delta G)$ between reactants and products, and they are not themselves permanently changed as a consequence of the reaction they catalyze. Enzymes increase the reaction rate by lowering the energy of the transition state, and therefore the activation energy
containing the two heavy (longer) and two light (shorter) chains, with yellow bars representing disulfide bonds. (c) The hand-in-glove fit between an antibody and the site to which it binds (epitope) on its target antigen-in this case, chicken egg-white lysozyme. The antibody contacts the antigen with residues from its CDRs. (Part (a) data from L. J. Harris, et al., 1997, Biochemistry 36:1581-1597, PDB ID: 1igt. Part (b) data from E. A. Padlan et al., 1989, P. Natl. Acad. Sci. USA 86:5938-5942, PDB ID 3hfm.]
required to reach it (Figure 3-22). In the test tube, catalysts such as charcoal and platinum facilitate reactions, but usually only at high temperatures or pressures, at extremes of high or low pH , or in organic solvents. Within cells, however, enzymes must function effectively in an aqueous environment at $37^{\circ} \mathrm{C}$
![li142857775e0](li142857775e0.jpg)

FIGURE 3-22 Effect of an enzyme on the activation energy of a chemical reaction. This hypothetical reaction pathway depicts the changes in free energy, $G$, as a reaction proceeds. A reaction will take place spontaneously only if the total $G$ of the products is less than that of the reactants (negative $\Delta G$ ). However, all chemical reactions proceed through one or more high-energy transition states, and the rate of a reaction is inversely proportional to the activation energy $\left(\Delta G^{\mathrm{a}}\right)$, which is the difference in free energy between the reactants and the transition state (highest point along the pathway). Enzymes and other catalysts accelerate the rate of a reaction by reducing the free energy of the transition state and thus $\Delta G^{\mathrm{a}}$.

and 1 atmosphere of pressure and at physiological pH values, usually $6.5-7.5$ but sometimes lower. Remarkably, enzymes exhibit immense catalytic power, in some cases accelerating the rates of reactions to $10^{6}-10^{12}$ times those of the corresponding uncatalyzed reactions under otherwise similar conditions.

## An Enzyme's Active Site Binds Substrates and Carries Out Catalysis

Certain amino acids of an enzyme are particularly important in determining its specificity and catalytic power. In the native conformation of an enzyme, critically important amino acids (which usually come from different parts of the linear sequence of the polypeptide) are brought into proximity, forming a cleft in the enzyme surface called the active site (Figure 3-23). An active site usually makes up only a small part of the total protein; the remaining part is involved in the folding of the polypeptide, regulation of the active site, and interactions with other molecules.

An active site consists of two functionally important regions: the substrate-binding site, which recognizes and binds the substrate or substrates, and the catalytic site, which carries out the chemical reaction once the substrate has bound. The catalytic groups in the catalytic site are amino acid side chains and backbone carboxyl and amino groups. In some enzymes, the catalytic and substrate-binding sites overlap; in others, the two regions are structurally distinct.

The substrate-binding site is responsible for the remarkable specificity of enzymes. Alteration of the structure of an enzyme's substrate by only one or a few atoms, or a subtle change in the geometry (e.g., stereochemistry) of the substrate, can result in a variant molecule that is no longer a substrate of the enzyme. As with the specificity of antibodies for antigens described above, the specificity of enzymes for substrates is a
![li143857775e0](li143857775e0.jpg)

FIGURE 3-23 Active site of the enzyme trypsin. (a) An enzyme's active site (outlined by dashed line) is composed of a substrate-binding site (blue), which binds specifically to a substrate, and a catalytic site (purple), which carries out catalysis. (b) A hybrid surface/ribbon representation of a portion of the serine protease trypsin. Clearly visible are the active-site cleft containing the catalytic site (purple, includes the key catalytic triad of Ser-195, Asp-102, and His-57, see also Figure 3-27) and a portion of the substrate-binding site called the side-chain-specificity binding pocket (blue). [Data from B. Sandler, M. Murakami, and J. Clardy, 1998, J. Am. Chem. Soc. 120:595-596, PDB ID 1aq7.]
consequence of the precise molecular complementarity between an enzyme's substrate-binding site and the substrate. Usually only one or a few substrates can fit precisely into a binding site.

The idea that substrates might bind to enzymes in the manner of a key fitting into a lock was first suggested by Emil Fischer in 1894. A variation of this proposal by Daniel Koshland in 1958, called induced fit, posited that the substrate-binding site is not rigid, as a lock is, but flexible, and is induced to change shape for more optimal catalysis when the substrate binds. In 1913, Leonor Michaelis and Maud Leonora Menten provided crucial evidence supporting the enzyme-substrate binding hypothesis. They showed that the rate of an enzymatic reaction was proportional to the substrate concentration at low substrate concentrations, but that as substrate concentrations increased, the rate reached a plateau, or maximal velocity, $V_{\max }$, and became substrate concentration independent, with a value of $V_{\max }$ directly proportional to the amount of enzyme present in the reaction mixture (Figure 3-24).
![li144857775e0](li144857775e0.jpg)
(b)
![li145857775e0](li145857775e0.jpg)

FIGURE 3-24 $K_{m}$ and $V_{\max }$ for an enzyme-catalyzed reaction. $K_{m}$ and $V_{\max }$ are determined from analysis of the dependence of the initial reaction rate on substrate concentration. The shape of these hypothetical kinetic curves is characteristic of a simple enzyme-catalyzed reaction in which one substrate ( S ) is converted into product ( P ). The initial reaction velocity is measured immediately after addition of enzyme to substrate, before the substrate concentration changes appreciably. (a) Plots of initial reaction velocity at two different concentrations of enzyme [E] as a function of substrate concentration [S]. The [S] that yields a half-maximal reaction rate is the Michaelis constant $K_{m}$, a measure of the affinity of $E$ for turning $S$ into $P$. Quadrupling the enzyme concentration causes a proportional increase in the reaction rate, so the maximal velocity $V_{\max }$ is quadrupled; $K_{m}$, however, is unaltered. (b) Plots of initial reaction velocity versus substrate concentration with a substrate $S$ for which the enzyme has a high affinity and with a substrate $S^{\prime}$ for which the enzyme has a lower affinity. Note that $V_{\max }$ is the same with both substrates because [E] is the same, but that $K_{m}$ is higher for $S^{\prime}$, the low-affinity substrate.

Michaelis and Menten deduced that these characteristics were due to the binding of substrate molecules (S) to a fixed and limited number of sites on the enzymes (E), and they called the bound species the enzyme-substrate (ES) complex. At high concentrations of substrate, all the binding sites on the enzymes have substrate bound, and the substrate-binding sites are said to be saturated with substrate-no additional binding to active sites is possible, and the maximal velocity of the reaction is achieved. Michaelis and Menten proposed that the ES complex is in equilibrium with the unbound enzyme and substrate and is an intermediate step in the ultimately irreversible conversion of substrate to product (P) (Figure 3-25):

$$
\mathrm{E}+\mathrm{S} \rightleftharpoons \mathrm{ES} \rightarrow \mathrm{E}+\mathrm{P}
$$

and that the rate $V_{0}$ of formation of product at a particular substrate concentration [S] is given by what is now called the Michaelis-Menten equation:

$$
V_{0}=V_{\max } \frac{[\mathrm{S}]}{[\mathrm{S}]+K_{\mathrm{m}}}
$$

![li146857775e0](li146857775e0.jpg)

FIGURE 3-25 Schematic model of an enzyme's reaction mechanism. Enzyme kinetics suggest that enzymes (E) bind substrate molecules (S) at a fixed and limited number of sites -the enzymes' active sites. The bound species is known as an enzyme-substrate (ES) complex. The ES complex is in equilibrium with the unbound enzyme and substrate (double arrows) and is an intermediate step in the conversion of substrate to products (P).
where the Michaelis constant, $K_{\mathrm{m}}$, a measure of the affinity of an enzyme for its substrate, is the substrate concentration that yields a half-maximal reaction rate (i.e., $1 / 2 V_{\max }$ in Figure 3-24). The $K_{\mathrm{m}}$ is somewhat similar in nature, but not identical, to the dissociation constant, $K_{\mathrm{d}}$ (see Chapter 2). The smaller the value of $K_{\mathrm{m}}$, the more effective the enzyme is at making product from dilute solutions of substrate, and the lower the substrate concentration needed to reach halfmaximal velocity. The smaller the $K_{\mathrm{d}}$, the lower the ligand concentration needed to reach 50 percent of binding. The concentrations of the various small molecules in a cell vary widely, as do the $K_{\mathrm{m}}$ values for the different enzymes that act on them. A good rule of thumb is that the intracellular concentration of a substrate is often approximately the same as, or somewhat greater than, the $K_{\mathrm{m}}$ value of the enzyme to which it binds.

The rates of reaction at substrate saturation vary enormously among enzymes. The maximum number of substrate molecules converted to product at a single enzyme active site per second, called the turnover number, can be less than 1 for very slow enzymes. The turnover number for carbonic anhydrase, one of the fastest enzymes, is $6 \times 10^{3}$ molecules per second.

Many enzymes catalyze the conversion of substrates to products by dividing the process into multiple, discrete chemical reactions, in which the product of one reaction is the substrate for the subsequent reaction. These sequential reactions generate multiple, distinct enzyme-substrate complexes (ES, $\mathrm{ES}^{\prime}, \mathrm{ES}^{\prime \prime}$, etc.) prior to the final release of the products:

$$
\mathrm{E}+\mathrm{S} \rightleftharpoons \mathrm{ES} \rightleftharpoons \mathrm{ES}^{\prime} \rightleftharpoons \mathrm{ES}^{\prime \prime} \rightleftharpoons \ldots \mathrm{E}+\mathrm{P}
$$

The energy profiles for such multistep reactions involve multiple hills and valleys (Figure 3-26). Methods have been developed to trap the intermediates in such reactions to learn more about the details of how enzymes catalyze reactions.

## Serine Proteases Demonstrate How an Enzyme's Active Site Works

Serine proteases, a large family of protein-cleaving, or proteolytic, enzymes, are used throughout the biological world-to digest meals (the pancreatic enzymes trypsin, chymotrypsin, and elastase), to control blood clotting (the enzyme thrombin), even to help silk moths chew their way out of their cocoons (cocoonase). This class of enzymes usefully illustrates how an enzyme's substrate-binding site and catalytic site cooperate in multistep reactions to convert substrate to product. Here we consider how trypsin and its two evolutionarily closely related pancreatic proteases, chymotrypsin and elastase, catalyze cleavage of a peptide bond in a polypeptide substrate:
![li147857775e0](li147857775e0.jpg)

(a)
![li148857775e0](li148857775e0.jpg)

FIGURE 3-26 Free-energy reaction profiles of uncatalyzed and multistep enzyme-catalyzed reactions. (a) The free-energy reaction profile of a hypothetical simple uncatalyzed reaction converting substrate (S) to product (P) via a single high-energy transition state. (b) Many enzymes catalyze such reactions by dividing the process
where $P_{1}$ is the part of the protein on the N -terminal side of the peptide bond to be cleaved and $\mathrm{P}_{2}$ is the portion on the C-terminal side. We first consider how serine proteases bind specifically to their substrates and then show in detail how catalysis takes place.

Figure 3-27a shows how a substrate polypeptide binds to the substrate-binding site in the active site of trypsin. There are two key binding interactions. First, the substrate (black polypeptide backbone) and enzyme (blue polypeptide backbone) form hydrogen bonds that resemble those of a $\beta$ sheet. Second, a key side chain of the substrate that determines which peptide in the substrate is to be cleaved extends into the enzyme's side-chain-specificity binding pocket, at the bottom of which resides the negatively charged side chain of trypsin's Asp-189. Trypsin has a marked preference for hydrolyzing substrates at the carboxyl $(\mathrm{C}=\mathrm{O})$ side of an amino acid with a long, positively charged side chain (arginine or lysine) because the side chain is stabilized in the enzyme's side-chain-specificity binding pocket by the negative Asp-189.

FIGURE 3-27 Substrate binding in the active site of trypsin-like serine proteases. (a) The active site of trypsin (purple and blue molecule) with a bound substrate (black molecule). The substrate forms a two-stranded $\beta$ sheet with trypsin's substrate-binding site, and the side chain of an arginine $\left(R_{1}\right)$ in the substrate is bound in the side-chainspecificity binding pocket of the binding site. Its positively charged guanidinium group is stabilized by the negative charge on the side chain of the enzyme's Asp-189. This binding aligns the peptide bond of the arginine appropriately for hydrolysis catalyzed by the enzyme's active-site catalytic triad (side chains of Ser-195, His-57, and Asp-102). (b) The amino acids lining the side-chain-specificity binding pocket determine its shape and charge and thus its binding properties. Trypsin accommodates the positively charged side chains of arginine and lysine; chymotrypsin, large, hydrophobic side chains such as phenylalanine; and elastase, small side chains such as glycine and alanine. See J. J. Perona and C. S. Craik, 1997, J. Biol. Chem. 272:29987-29990.
![li149857775e0](li149857775e0.jpg)

$$
E+S \rightleftharpoons E S \rightleftharpoons E X^{1} \longrightarrow E+P
$$

into multiple discrete steps, in this case, the initial formation of an ES complex followed by conversion via a single transition state $\left(\mathrm{EX}^{1}\right)$ to the free enzyme (E) and P. The activation energy for each of these steps is significantly less than the activation energy for the uncatalyzed reaction; thus the enzyme dramatically enhances the reaction rate.

Slight differences in the structures of otherwise similar binding pockets help explain the differing substrate specificities of two serine proteases related to trypsin: chymotrypsin prefers large aromatic groups (as in Phe, Tyr, Trp), and elastase prefers the small side chains of Gly and Ala (Figure 3-27b). The uncharged Ser-189 in chymotrypsin
![li150857775e0](li150857775e0.jpg)

allows large, uncharged, hydrophobic side chains to bind stably in the binding pocket. The specificity of elastase is influenced by the replacement of glycines in the sides of the binding pocket in trypsin with the branched aliphatic side chains of valines (Val-216 and Val-190), which obstruct the binding pocket (see Figure 3-27b). As a consequence, large side chains in substrates are prevented from fitting into the binding pocket of elastase, whereas substrates with the short alanine or glycine side chains at this position can bind well and be subject to subsequent cleavage.

In the catalytic site, all three enzymes use the hydroxyl group on the side chain of a serine at position 195 to catalyze the hydrolysis of peptide bonds in substrate proteins. A catalytic triad formed by the three side chains of Ser-195, His-57, and Asp-102 participates in what is essentially a two-step hydrolysis reaction. Figure 3-28 shows how the
catalytic triad cooperates in breaking the peptide bond, with Asp-102 and His-57 supporting the attack of the hydroxyl oxygen of Ser-195 on the carbonyl carbon in the substrate. This attack initially forms an unstable transition state with four groups attached to the carbon (tetrahedral intermediate). Breaking of the $\mathrm{C}-\mathrm{N}$ peptide bond then releases one part of the substrate protein $\left(\mathrm{NH}_{3}-\mathrm{P}_{2}\right)$, while the other part remains covalently attached to the enzyme via an ester bond to the serine's oxygen, forming a relatively stable acyl enzyme intermediate. The subsequent replacement of this oxygen with one from water, in a reaction involving another unstable tetrahedral intermediate, leads to release of the final product $\left(\mathrm{P}_{1}-\mathrm{COOH}\right)$. The tetrahedral intermediate transition states are partially stabilized by hydrogen bonding with the enzyme's backbone amino groups in what is called the oxyanion hole. The large family of serine proteases and
(a) ES complex
(b) Tetrahedral intermediate (transition state)
![li151857775e0](li151857775e0.jpg)
![li152857775e0](li152857775e0.jpg)
(c) Acyl enzyme (ES' complex)
![li153857775e0](li153857775e0.jpg)
(f) EP complex
![li154857775e0](li154857775e0.jpg)
(e) Tetrahedral intermediate (transition state)
![li155857775e0](li155857775e0.jpg)
(d) Acyl enzyme (ES' complex)
![li156857775e0](li156857775e0.jpg)
![li157857775e0](li157857775e0.jpg)

FIGURE 3-28 Mechanism of serine protease-mediated hydrolysis of peptide bonds. The catalytic triad of Ser-195, His-57, and Asp-102 in the active sites of serine proteases employs a multistep mechanism to hydrolyze peptide bonds in target proteins. (a) After a polypeptide substrate binds to the active site (see Figure 3-27), forming an ES complex, the hydroxyl oxygen of Ser-195 attacks the carbonyl carbon of the substrate's targeted peptide bond (yellow). Movements of electrons are indicated by arrows. (b) This attack results in the formation of a transition state called the tetrahedral intermediate, in which the negative charge on the substrate's oxygen is stabilized by hydrogen bonds formed with the enzyme's oxyanion hole. (c) Additional electron movements result in the breaking of the
peptide bond, release of one of the reaction products $\left(\mathrm{NH}_{2}-\mathrm{P}_{2}\right)$, and formation of the acyl enzyme ( $\mathrm{ES}^{\prime}$ complex). (d) An oxygen from a solvent water molecule then attacks the carbonyl carbon of the acyl enzyme. (e) This attack results in the formation of a second tetrahedral intermediate. (f) Additional electron movements result in the breaking of the Ser-195-substrate bond (formation of the EP complex) and release of the final reaction product $\left(\mathrm{P}_{1}-\mathrm{COOH}\right)$. The side chain of His-57, which is held in the proper orientation by hydrogen bonding to the side chain of Asp-102, facilitates catalysis by withdrawing and donating protons throughout the reaction (inset). If the pH is too low and the side chain of His-57 is protonated, it cannot participate in catalysis and the enzyme is inactive.

related enzymes, all of which have an active-site serine, illustrates how an efficient reaction mechanism is used over and over by distinct enzymes to catalyze similar reactions.

The serine protease mechanism points out several key features of enzymatic catalysis. First, enzyme catalytic sites have evolved to stabilize the binding of a transition state, thus lowering the activation energy and accelerating the overall reaction. Second, multiple side chains, together with the polypeptide backbone, carefully organized in three dimensions, work together to chemically transform substrate into product, often by multistep reactions.

Third, acid-base catalysis mediated by one or more amino acid side chains is often used by enzymes, as when the imidazole group of His-57 in serine proteases acts as a base to remove the hydrogen from Ser-195's hydroxyl group. As a consequence, only a particular ionization state (protonated or nonprotonated) of one or more amino acid side chains in the catalytic site may be compatible with catalysis, and thus the enzyme's activity may be pH dependent. For example, the imidazole of His-57 in serine proteases, whose $\mathrm{p} K_{\mathrm{a}}$ is $\sim 6.8$, can help the Ser-195 hydroxyl attack the substrate only if it is not protonated. Thus the activity of the protease is low at $\mathrm{pH}<6.8$, at which the imidazole is protonated, and the shape of the pH activity profile in the pH range $4-8$ matches the titration of the His-57 side chain, which is governed by the Henderson-Hasselbalch equation, with an inflection near pH 6.8 (see chymotrypsin data in Figure 3-29 and see Chapter 2). The activity drops at higher pH values, generating a bell-shaped activity curve, because the proper folding of the protein is disrupted when the amino group at
![li158857775e0](li158857775e0.jpg)

FIGURE 3-29 The pH dependence of enzyme activity. In some cases, ionizable ( pH -titratable) groups in enzyme active sites or elsewhere in enzymes must be either protonated or deprotonated to permit proper substrate binding or catalysis, or to permit the enzyme to adopt the correct conformation. Measurement of enzyme activity as a function of pH can be used to identify the $\mathrm{p} K_{\mathrm{a}}$ 's of these groups. The pancreatic serine proteases, such as chymotrypsin, exhibit maximum activity at around pH 8 because of titration of the active-site His-57 (required for catalysis, $\mathrm{p} K_{\mathrm{a}} \sim 6.8$ ) and of the amino terminus of the protein (required for proper conformation, $\mathrm{p} K_{\mathrm{a}} \sim 9$ ). Many lysosomal hydrolases have evolved to exhibit a lower pH optimum ( 4.5) to match the low internal pH in the lysosomes in which they function. [Data from P. Lozano, T. De Diego, and J. L. Iborra, 1997, Eur. J. Biochem. 248:80-85, and W. A. Judice et al., 2004, Eur. J. Biochem. 271:1046-1053.]
the protein's amino terminus ( $\mathrm{p} K_{\mathrm{a}} \sim 9$ ) is deprotonated; the conformation near the active site changes as a consequence.

The pH sensitivity of an enzyme's activity can be due to changes in the ionization of catalytic groups, groups that participate directly in substrate binding, or groups that influence the conformation of the protein. Pancreatic serine proteases evolved to function in the neutral or slightly basic conditions in the intestines; hence their pH optima are $\sim 8$. Proteases and other hydrolytic enzymes that function in acidic conditions must employ a different catalytic mechanism. This is the case for enzymes within the stomach ( pH $\sim 1$ ), such as the protease pepsin, and for those within lysosomes ( $\mathrm{pH} \sim 4.5$ ), which play a key role in degrading macromolecules within cells (see the lysosomal hydrolase data in Figure 3-29). Indeed, lysosomal hydrolases, which degrade a wide variety of biomolecules (proteins, lipids, etc.), are relatively inactive at the pH in the cytosol ( $\sim 7$ ), which helps to protect a cell from self-digestion if these enzymes escape the confines of the membrane-bounded lysosome.

One key feature of enzymatic catalysis not seen in serine proteases but found in many other enzymes is a cofactor or prosthetic group. This "helper" group is a nonpolypeptide small molecule or ion (e.g., iron, zinc, copper, manganese) that is bound in the active site and plays an essential role in the reaction mechanism. Small organic prosthetic groups in enzymes are also called coenzymes. Some of these groups are chemically modified during the reaction and thus need to be replaced or regenerated after each reaction; others are not. Examples include $\mathrm{NAD}^{+}$(nicotinamide adenine dinucleotide), FAD (flavin adenine dinucleotide) (see Figure 2-33), and the heme groups that bind oxygen in hemoglobin or transfer electrons in some cytochromes (see Figure 12-20). Thus the chemical reactions catalyzed by enzymes are not restricted by the limited number of types of amino acids in polypeptide chains. Many of the vitamins [for example, the B vitamins thiamine $\left(\mathrm{B}_{1}\right)$, riboflavin $\left(\mathrm{B}_{2}\right)$, niacin $\left(\mathrm{B}_{3}\right)$, and pyridoxine $\left(\mathrm{B}_{6}\right)$, as well as vitamin C], which cannot be synthesized in mammalian cells, function as, or are used to generate, coenzymes. That is why supplements of vitamins must be added to the liquid medium in which mammalian cells are grown in the laboratory (see Chapter 4).

Small molecules that can bind to active sites and disrupt catalytic reactions are called enzyme inhibitors. Such inhibitors are useful tools for studying the roles of enzymes in cells and organisms. Inhibitors that bind directly to an enzyme's binding site and thus compete directly with the normal substrate are called competitive inhibitors. Noncompetitive inhibitors are those that interfere with enzyme activity in other ways-for example, by binding to some other site on the enzyme and changing its conformation. Enzyme inhibitors complement the use of genetic mutations and a technique called RNA interference (RNAi) for probing an enzyme's function in cells (see Chapter 6). In all three approaches, the cellular consequences of disrupting an enzyme's activity can be used to deduce the normal function of the enzyme. The same approaches can be used to study the functions of nonenzymatic macromolecules. Interpreting the results of inhibitor studies

can be complicated, however, if, as is often the case, the inhibitors block the activity of more than one protein.

5Small-molecule inhibition of protein activity is the basis for many drugs as well as for chemical warfare agents. Aspirin inhibits enzymes called cyclooxygenases, whose products can cause pain. Sarin and other nerve gases react with the active serine hydroxyl groups of both serine proteases and a related enzyme, acetylcholine esterase, which is a key enzyme in regulating nerve conduction (see Chapter 22).

## Enzymes in a Common Pathway Are Often Physically Associated with One Another

Enzymes taking part in a common metabolic process (e.g., the degradation of glucose to pyruvate during glycolysis; see Chapter 12) are generally located in the same cellular compartment, be it in the cytosol, at a membrane, or within a particular organelle. Within this compartment, products from one reaction can move by diffusion to the next enzyme in the metabolic pathway. Diffusion, however, entails random movement and can be a slow, relatively inefficient process for moving molecules between enzymes (Figure 3-30a).
![li159857775e0](li159857775e0.jpg)

FIGURE 3-30 Assembly of enzymes into efficient multienzyme complexes. In the hypothetical reaction pathways illustrated here, the initial reactants are converted into final products by the sequential action of three enzymes: $A, B$, and $C$. (a) When the enzymes are free in solution, or even constrained within the same cellular compartment, the intermediates in the reaction sequence must diffuse from one enzyme to the next, an inherently slow process. (b) Diffusion is greatly reduced or eliminated when the individual enzymes associate into multisubunit complexes, either by themselves or with the aid of a scaffold protein. (c) The closest integration of different catalytic activities occurs when the enzymes are fused at the genetic level, becoming domains in a single polypeptide chain.

To overcome this impediment, cells have evolved mechanisms for bringing enzymes in a common pathway into close proximity, a process called metabolic coupling.

In the simplest such mechanism, polypeptides with different catalytic activities cluster closely together as subunits of a multimeric enzyme or assemble on a common "scaffold" that holds them together (Figure 3-30b). This arrangement allows the product of one reaction to be channeled directly to the next enzyme in the pathway. In some cases, independent proteins have been fused together at the genetic level to create a single multidomain, multifunctional enzyme (Figure 3-30c). Metabolic coupling usually involves large multiprotein complexes, as described earlier in this chapter.

## KEY CONCEPTS OF SECTION 3.3

## Protein Binding and Enzyme Catalysis

- A protein's function depends on its ability to bind other molecules, known as ligands. For example, antibodies bind to a group of ligands known as antigens, and enzymes bind to reactants called substrates that will be converted by chemical reactions into products.
- The specificity of a protein for a particular ligand refers to the preferential binding of one or a few closely related ligands. The affinity of a protein for a particular ligand refers to the strength of binding, usually expressed as the dissociation constant $K_{\mathrm{d}}$.
- Proteins are able to bind to ligands because of molecular complementarity between the ligand-binding sites and the corresponding ligands.
- Enzymes are catalytic proteins that accelerate the rates of cellular reactions by lowering the activation energy and stabilizing transition-state intermediates (see Figure 3-22).
- An enzyme's active site, which is usually only a small part of the protein, comprises two functional parts: a substratebinding site and a catalytic site. The substrate-binding site is responsible for the exquisite specificity of enzymes owing to its molecular complementarity with the substrate.
- The initial binding of a substrate (S) to an enzyme (E) results in the formation of an enzyme-substrate complex (ES), which then undergoes one or more reactions catalyzed by the catalytic groups in the catalytic site until the final product $(\mathrm{P})$ is formed.
- From plots of reaction rate versus substrate concentration, two characteristic parameters of an enzyme can be determined: the Michaelis constant, $K_{\mathrm{m}}$, a rough measure of the enzyme's affinity for converting substrate into product, and the maximal velocity, $V_{\max }$, a measure of its catalytic power (see Figure 3-24).
- The rates of enzyme-catalyzed reactions vary enormously, with turnover numbers (numbers of substrate molecules converted to product at a single active site at substrate saturation) ranging from fewer than 1 to $6 \times 10^{5}$ molecules per second.

- Many enzymes catalyze the conversion of substrates to products by dividing the process into multiple discrete chemical reactions that involve multiple distinct enzyme-substrate complexes ( $\mathrm{ES}^{+}, \mathrm{ES}^{\circ}$ etc.).
- Serine proteases hydrolyze peptide bonds in substrate proteins using as catalytic groups the side chains of Ser-195, His-57, and Asp-102. Amino acids lining the side-chainspecificity binding pocket in the binding site of serine proteases determine the residue in a substrate protein whose peptide bond will be hydrolyzed and account for differences in protease specificity (for example, trypsin vs. chymotrypsin and elastase).
- Enzymes often use acid-base catalysis mediated by one or more amino acid side chains, such as the imidazole group of His-57 in serine proteases, to catalyze reactions. The pH dependence of protonation of catalytic groups $\left(\mathrm{p} K_{\mathrm{a}}\right)$ is often reflected in the pH -rate profile of the enzyme's activity.
- Nonpolypeptide small molecules or ions, called cofactors or prosthetic groups, bind to the active sites of some enzymes and play an essential role in enzymatic catalysis. Small organic prosthetic groups in enzymes are also called coenzymes; many vitamins, which cannot be synthesized in higher animal cells, function as or are used to generate coenzymes.
- Enzymes in a common metabolic pathway are often located within the same cellular compartments and may be further associated as domains of a monomeric protein, subunits of a multimeric protein, or components of a protein complex assembled on a common scaffold (see Figure 3-30).


### 3.4 Regulating Protein Function

Most processes in cells do not take place independently of one another or at a constant rate. The activities of all proteins and other biomolecules are regulated to integrate their functions for optimal performance for survival. For example, the catalytic activity of enzymes is regulated so that the amount of reaction product is just sufficient to meet the needs of the cell. As a result, the steady-state concentrations of substrates and products may vary depending on cellular conditions. Regulation of nonenzymatic proteins-the opening or closing of membrane channels or the assembly of a macromolecular complex, for example-is also essential.

In general, there are three ways to regulate protein activity. First, cells can increase or decrease the steady-state level of the protein by altering its rate of synthesis, its rate of degradation, or both. Second, cells can change the intrinsic activity, as distinct from the amount, of the protein. For example, through noncovalent and covalent interactions, cells can change the affinity of substrate binding, or the fraction of time the protein is in an active versus an inactive conformation. Third, there can be a change in the location or the concentration within the cell of the protein itself, of the target of the protein's activity (e.g., an enzyme's substrate), or of some other molecule required for the protein's
activity (e.g., an enzyme's cofactor). All three types of regulation play essential roles in the lives and functions of cells. In this section, we first discuss mechanisms for regulating the amount of a protein, then turn to noncovalent and covalent interactions that regulate protein activity.

## Regulated Synthesis and Degradation of Proteins Is a Fundamental Property of Cells

The rate of synthesis of a protein is determined by the rate at which the DNA encoding the protein is converted to mRNA (transcription), the steady-state amount of the active mRNA in the cell, and the rate at which the mRNA is converted into newly synthesized protein (translation). These important processes are described in detail in Chapter 5.

The life spans of intracellular proteins vary from as short as a few minutes for mitotic cyclins, which help regulate passage through the mitotic stage of cell division (see Chapter 19), to as long as the age of an organism for proteins in the lens of the eye. Protein life span is controlled primarily by regulated protein degradation.

Protein degradation plays two especially important roles in the cell. First, it removes proteins that are potentially toxic, improperly folded or assembled, or damaged-including the products of mutated genes and proteins damaged by chemically active cell metabolites or stress (e.g., heat shock). Despite the existence of chaperone-mediated protein folding, some newly made proteins are rapidly degraded because they are misfolded. This degradation might be necessary due to failure of timely engagement of the necessary chaperones to guide the folding of the proteins or due to their defective assembly into complexes. Most other proteins are degraded more slowly, undergoing about 1-2 percent degradation per hour in mammalian cells. Second, the controlled destruction of otherwise normal proteins, along with controlled rates of synthesis, provides a powerful mechanism for maintaining the appropriate levels of the proteins and their activities and for permitting rapid changes in these levels to help the cells respond to changing conditions.

Eukaryotic cells have several pathways for degrading proteins. One major pathway is degradation by enzymes within lysosomes, membrane-limited organelles whose acidic interior $(\mathrm{pH} \sim 4.5)$ is filled with a host of hydrolytic enzymes. Lysosomal degradation is directed primarily toward aged or defective organelles of the cell-a process called autophagy (see Chapter 14)-and toward extracellular proteins taken up by the cell. Lysosomes will be discussed at length in later chapters. Here we focus on another important degradation pathway: cytoplasmic protein degradation by proteasomes, which can account for up to 90 percent of the protein degradation in mammalian cells.

## The Proteasome Is a Molecular Machine Used to Degrade Proteins

Proteasomes are very large protein-degrading molecular machines that influence many different cellular functions,

including the cell cycle (see Chapter 19), transcription, DNA repair (see Chapter 5), programmed cell death, or apoptosis (see Chapter 21), recognition of and response to infection by foreign organisms (see Chapter 23), and removal of misfolded proteins. There are approximately 30,000 proteasomes in a typical mammalian cell.

Proteasomes consist of roughly 60 protein subunits and have a mass of about $2.4 \times 10^{9} \mathrm{Da}$. Proteasomes have a
cylindrical, barrel-like catalytic core (Figure 3-31a), called the 20 S proteasome (where $S$ is a Svedberg unit based on the sedimentation properties of the particle and is proportional to its size), which is approximately 14.8 nm tall and 11.3 nm in diameter. Bound to the ends of this core are either one or two 19 S cap complexes that regulate the activity of the 20 S catalytic core. When the core and one or two caps are combined, they are referred to collectively as the 26 S complex,
![li160857775e0](li160857775e0.jpg)

FIGURE 3-31 Ubiquitin- and proteasome-mediated proteolysis.
(a) Right: The 26 S proteasome has a cylindrical structure with a 19 S cap at one or both ends of the 20 S core particle. The nineteen different subunits of the 19 S cap (shown in multiple colors) include six AAAATPase subunits (Rpt1-6, red), which assemble into a heterohexameric ring; two ubiquitin (Ub) receptors (Rpn10 and Rpn13, yellow); and a deubiquitinase enzyme (DUB, Rpn11, green), which forms a heterodimer with its evolutionarily related counterpart Rpn8. Moreover, the 19 S cap contains scaffolding and other proteins (tan). The two 19 S caps shown are facing in opposite directions relative to the plane of the page. The 20 S core consists of four stacked heptameric rings ( $\sim 110 \AA$ diameter $\times 160 \AA$ long), each containing either $\alpha$ (outer rings) or $\beta$ (inner rings) subunits (blue). Left: Cutaway view of the 20 S core, showing the inner chambers. Proteolysis occurs within the central inner chamber of the core formed by the $\beta$ rings. (b) Proteins are targeted for proteasomal degradation by polyubiquitinylation. Enzyme E1 is activated by the ATP-dependent attachment of a ubiquitin (Ub) molecule
(step $\mathbf{\square}$ ) and then transfers this Ub molecule to a cysteine residue in E2 (step $\mathbf{\square}$ ). Ubiquitin ligase (E3) transfers the bound Ub molecule on E2 to the side-chain -NH2 of a lysine residue in a target protein, forming an isopeptide bond (step $\mathbf{\square}$ ). Additional Ub molecules are added to the Ub-modified target protein via isopeptide bonds to the previously added Ub by repeating steps $\mathbf{\square}-\mathbf{\square}$, forming a polyubiquitin chain (step $\mathbf{\square}$ ). The polyubiquitinylated target is recognized by Ub receptors in the proteasome's 19 S cap (step $\mathbf{\square}$ ), and the Ub groups are removed by the deubiquitinase enzyme (step $\mathbf{\square}$ ). In step $\mathbf{\square}$ ATP hydrolysis enables the six protein (hexameric) ATPase subunits (red) to unfold the substrate and transfer the unfolded protein via a pore in the hexamer into the proteolysis chamber in the 20 S core (step $\mathbf{\square}$ ), in some cases coordinately with step $\mathbf{\square}$, and the protein is cleaved into short peptide digestion fragments (step $\mathbf{\square}$ ) that are then released (step $\mathbf{\square}$ ). [Part (a) courtesy of Antje Aufderheide and Friedrich Foerster, data from P. Unverdorben et al., 2014, P. Natl. Acad. Sci. USA 111:(15):5544-5549, PDB ID 4cr2.]

even though the two-cap-containing complex is larger (30S). A 19 S cap has 19 protein subunits, six of which can hydrolyze ATP (i.e., they are AAA-type ATPases) to provide the energy needed to unfold protein substrates and selectively transfer them into the inner chamber of the proteasome's catalytic core. Genetic studies in yeast have shown that cells cannot survive without functional proteasomes, thus demonstrating their importance. Furthermore, proper proteasomal activity is so important that cells will expend as much as 30 percent of the energy needed to synthesize a protein to degrade it in a proteasome.

The 20S proteasomal catalytic core comprises two inner rings of seven $\beta$ subunits each, with three proteolytic active sites per ring facing toward the $\sim 1.7$-nm-diameter inner chamber formed by those rings, and two outer rings of seven $\alpha$ subunits each, which limit substrate access (see Figure 3-31a) via an entry channel that can be opened by the 19 S cap. Proteasomes can degrade most proteins thoroughly because the three active sites in each $\beta$ subunit ring can cleave peptide bonds at hydrophobic residues, acidic residues, or basic residues. Polypeptide substrates must enter the chamber via a regulated $\sim 1.3$-nm-diameter aperture at the center of the outer $\alpha$ subunit rings. In the 26 S proteasome, the opening of the aperture, which is narrow and often allows the entry of only unfolded proteins, is controlled by ATPases in the 19S cap. These ATPases are responsible for unfolding protein substrates and translocating those unfolded polypeptides into the inner chamber of the catalytic core (Figure 3-31b, bottom right). The short peptide products of proteasomal digestion (2-24 residues long) exit the chamber and are further degraded rapidly by cytosolic peptidases, eventually being converted to individual ("free") amino acids. One researcher has quipped that a proteasome is a "cellular chamber of doom" in which proteins suffer a "death by a thousand cuts."

BInhibitors of proteasome function have proved to be exceptionally useful in the laboratory and the clinic. Small-molecule proteasome inhibitors, such as MG132, are used to block proteasomal degradation in the lab and to help evaluate the role of the proteasome and, as we shall see below, polyubiquitinylation in a wide variety of processes. Other small-molecule proteasome inhibitors have been used therapeutically. Because of the global importance of proteasome-mediated protein breakdown in cells, continuous, complete inhibition of proteasomes kills cells. However, partial proteasome inhibition for short intervals is widely used as an approach to cancer chemotherapy, especially to treat multiple myeloma, a cancer involving the abnormal proliferation of antibody (immunoglobulin)-producing cells. The myeloma cells produce abnormally high levels of potentially toxic, aberrant immunoglobulin polypeptide chains, which are degraded by proteasomes. Proteasome inhibition in these cancer cells leads to the buildup of toxic, misfolded immunoglobulin polypeptides within the cells, and thus to cell death. In addition, to survive and grow, myeloma cells require the robust activity of a regulatory protein called $N F-\kappa B$ (see Chapter 16) as well as other "pro-survival" and
"pro-proliferation" proteins. In turn, NF- $\kappa$ B can function fully and promote survival and proliferation only when its inhibitor, $I-\kappa B$, is disengaged and degraded by proteasomes (see Chapter 16). Partial inhibition of proteasomal activity by a small-molecule inhibitor drug results in increased levels of I- $\kappa B$ and, consequently, reduced NF- $\kappa$ B activity (that is, loss of its protective activity). The cancer cells subsequently undergo less proliferation and die by apoptosis. Thus, multiple myeloma cells are more sensitive to proteasome inhibitors than normal cells. Consequently, controlled administration of proteasome inhibitors, at levels that kill the cancer cells but not normal cells, has proved to be an effective therapy for multiple myeloma.

## Ubiquitin Marks Cytosolic Proteins for Degradation in Proteasomes

If proteasomes are to rapidly degrade only those proteins that are either defective or scheduled to be removed, they must be able to distinguish between those proteins that need to be degraded and those that don't. Cells mark proteins that should be degraded by covalently attaching to them a linear chain of multiple copies of a 76 -residue polypeptide called ubiquitin (Ub) that is highly conserved from yeast to humans. This "polyubiquitin tail" serves as a cellular "kiss of death," marking the protein for destruction in the proteasome. The ubiquitinylation process (Figure 3-31b, steps $\mathbf{\square}$ ) involves three distinct steps:

1. Activation of ubiquitin-activating enzyme (E1) by the addition of a ubiquitin molecule, a reaction that requires ATP.
2. Transfer of this ubiquitin molecule to a cysteine residue in a ubiquitin-conjugating enzyme (E2).
3. Formation of a covalent bond between the carboxyl group of the C-terminal glycine 76 of the ubiquitin bound to E2 and the amino group of the side chain of a lysine residue in the target protein, a reaction catalyzed by a ubiquitin-protein ligase (E3). This type of bond is called an isopeptide bond because it covalently links a side-chain amino group, rather than the $\alpha$ amino group, to the carboxyl group. Subsequent ligase reactions covalently attach the C-terminal glycine of an additional ubiquitin molecule via an isopeptide bond to the side chain of lysine 48 of the previously added ubiquitin to generate a polyubiquitin chain covalently attached to the target protein. (We will discuss ubiquitin linkages via other lysine side chains shortly.)

Generally, following attachment of four or more ubiquitins in a polyubiquitin chain, the 19 S regulatory cap of the 26 S proteasome (sometimes with the help of accessory proteins) recognizes the polyubiquitin-labeled protein using its Ub receptors (see Figure 3-31a), uses ATPases to unfold it, and transports it into the proteasome core for degradation. As a polyubiquitinylated substrate is unfolded and passed into the core of the proteasome, enzymes called deubiquitinases (Dubs) hydrolyze the bonds between the individual ubiquitins and between the targeted protein and ubiquitin,

recycling the ubiquitins for additional rounds of protein modification (see Figure 3-31b). Analysis of the human genome sequence indicates the presence of about 90 distinct Dubs, about 80 percent of which use cysteine in a catalytic triad similar to that in the serine proteases described earlier (the sulfhydryl in the cysteine side chain is used in place of the hydroxyl in the side chain of the serine). In some Dubs, zinc is a key participant in the catalytic reactions.

Specificity of Degradation Targeting of specific proteins for proteasomal degradation is primarily achieved through the substrate specificity of E3 ligases (see Figure 3-31b, step ■). As a testament to their importance, there are an estimated 600 or more ubiquitin ligase genes in the human genome. The many E3 ligases in mammalian cells ensure that the wide variety of proteins to be polyubiquitinylated can be modified when necessary. Some E3 ligases are associated with chaperones that recognize unfolded or misfolded proteins; for example, the E3 ligase CHIP is a co-chaperone for Hsp70. These and other proteins (co-chaperones, escort factors, adapters) can mediate E3 ligase-catalyzed polyubiquitinylation of dysfunctional proteins that cannot be readily refolded properly and, consequently, mediate their delivery to proteasomes for degradation. In such cases, the chaperone-ubiquitinylation-proteasome system works in concert for protein quality control.

In addition to quality control, the ubiquitin-proteasome system can be used to regulate the activity of important cellular proteins. An example is the regulated degradation of proteins called cyclins, which control the cell cycle (see Chapter 19). Cyclins contain the internal sequence Arg-X-X-Leu-Gly-X-Ile-Gly-Asp/Asn (where X can be any amino acid), which is recognized by specific ubiquitinylating enzyme complexes. At a specific time in the cell cycle, each cyclin is phosphorylated by a cyclin kinase. This phosphorylation is thought to cause a conformational change that exposes the recognition sequence to the ubiquitinylating enzymes, leading to polyubiquitinylation and proteasomal degradation.

Other Functions of Ubiquitin and Ubiquitin-Related Molecules There are several close relatives of ubiquitin that employ similar E1-, E2-, and E3-dependent mechanisms of activation and transfer to acceptor substrates. These ubiquitin-like modifiers control processes as diverse as nuclear import, regulated by the ubiquitin-like modifier Sumo, and autophagy, regulated by the ubiquitin-like modifier Atg8/LC3 (see Chapter 14). Furthermore, the attachment of ubiquitin to a target protein can be used for purposes other than to mark the protein for degradation, as we will see later in this section, and some of these functions involve polyubiquitin linkages other than those via Lys-48.

Like ubiquitinylation, deubiquitinylation is involved in processes other than proteasome-mediated protein degradation. Large-scale, mass-spectrometry-based "proteomic" methods described later in this chapter, together with sophisticated computational approaches, have suggested that Dubs, which are often bound in multiprotein complexes, are involved in an extraordinarily wide range of cell processes.

These processes vary from cell division and cell cycle control (see Chapter 19) to membrane trafficking (see Chapter 14) to cell signaling pathways (see Chapters 15 and 16).

## Noncovalent Binding Permits Allosteric, or Cooperative, Regulation of Proteins

In addition to regulating the amount of a protein, cells can also regulate the intrinsic activity of a protein. One of the most important mechanisms for regulating protein function is through allosteric interactions. Broadly speaking, allostery (from the Greek, "other shape") refers to any change in a protein's tertiary or quaternary structure, or in both, induced by the noncovalent binding of a ligand. When a ligand binds to one site (A) in a protein and induces a conformational change that alters the activity of a different site (B), the ligand is called an allosteric effector of the protein, while site A is called an allosteric binding site, and the protein is called an allosteric protein. By definition, allosteric proteins have multiple binding sites, at least one for the allosteric effector and at least one for other molecules with which the protein interacts. The allosteric change in activity can be positive or negative; that is, it can be an increase or a decrease in protein activity. Negative allostery often involves the end product of a multistep biochemical pathway binding to, and reducing the activity of, an enzyme that catalyzes an early, rate-controlling step in that pathway. In this way, excessive buildup of the product is prevented. This kind of regulation of a metabolic pathway is also called end-product inhibition or feedback inhibition. Allosteric regulation is particularly prevalent in multimeric enzymes and other proteins in which conformational changes in one subunit are transmitted to an adjacent subunit.

Cooperativity, a term that is often used synonymously with allostery, usually refers to the influence (positive or negative) that the binding of a ligand at one site has on the binding of another molecule of the same type of ligand at a different site. Hemoglobin presents a classic example of positive cooperative binding in that the binding of a single ligand, molecular oxygen $\left(\mathrm{O}_{2}\right)$, increases the affinity of hemoglobin for the next oxygen molecule. Each of the four subunits in hemoglobin contains one heme molecule. The heme groups are the oxygen-binding components of hemoglobin (see Figure 3-14a). The binding of oxygen to the heme molecule in one of the four hemoglobin subunits induces a local conformational change whose effect spreads to the other subunits, lowering the $K_{\mathrm{d}}$ (increasing the affinity) for the binding of additional oxygen molecules to the remaining hemes and yielding a sigmoidal oxygen-binding curve (Figure 3-32). Because of the sigmoidal shape of the oxygen-saturation curve, it takes only a fourfold increase in oxygen concentration for the saturation of the oxygen-binding sites in hemoglobin to go from 10 to 90 percent. Conversely, if there were no cooperativity and the shape of the curve was typical of that for Michaelis-Menten (see Figure 3-24), or noncooperative, binding, it would take an eighty-one-fold increase in oxygen concentration to accomplish the same increase in loading of its binding sites in hemoglobin. This cooperativity permits hemoglobin to take up oxygen very efficiently in the lungs, where the oxygen

![li161857775e0](li161857775e0.jpg)

EXPERIMENTAL FIGURE 3-32 Hemoglobin binds oxygen cooperatively. Each tetrameric hemoglobin molecule has four oxygenbinding sites; at saturation, all the sites are loaded with oxygen. The oxygen concentration in tissues is commonly measured as the partial pressure of oxygen $\left(p \mathrm{O}_{2}\right)$ in torr units (a standard measure of pressure equivalent to 1 mm of mercury under standard conditions). $P_{50}$ is the $p \mathrm{O}_{2}$ at which half the oxygen-binding sites are occupied; it is somewhat analogous to the $K_{\mathrm{m}}$ for an enzymatic reaction. The large change in the amount of oxygen bound over a small range of $p \mathrm{O}_{2}$ values permits efficient unloading of oxygen in peripheral tissues such as muscle. The sigmoidal shape of a plot of saturation versus ligand concentration is indicative of cooperative binding, in which the binding of one oxygen molecule allosterically influences the binding of subsequent oxygens. In the absence of cooperative binding, a binding curve is a hyperbola, similar to the curves in Figure 3-24. See J. M. Berg et al., 2015, Biochemistry, 8th ed., Macmillan.
concentration is high, and unload it in tissues, where the concentration is low. Thus cooperativity amplifies the sensitivity of a system to changes in the concentration of its ligands, providing in many cases a selective evolutionary advantage.

## Noncovalent Binding of Calcium and GTP Are Widely Used as Allosteric Switches to Control Protein Activity

Unlike oxygen, which causes graded allosteric changes in the activity of hemoglobin, some other allosteric effectors act as switches, turning the activity of many different proteins on or off by binding to them noncovalently. Two important allosteric switches that we will encounter many times throughout this book, especially in the context of cell signaling pathways (see Chapters 15 and 16), are $\mathrm{Ca}^{2+}$ and GTP.
$\mathrm{Ca}^{2+}$ /Calmodulin-Mediated Switching The concentration of $\mathrm{Ca}^{2+}$ that is free in the cytosol (not bound to molecules other than water) is kept very low $\left(\sim 10^{-7} \mathrm{M}\right)$ by specialized membrane transport proteins that continually pump excess $\mathrm{Ca}^{2+}$ out of the cytosol (see Chapters 11 and 15). However, as we will learn in Chapters 11 and 15, the cytosolic $\mathrm{Ca}^{2+}$ concentration can increase tenfold to a hundredfold when $\mathrm{Ca}^{2+}$-permeable channels in the cell-surface membranes open
and allow extracellular $\mathrm{Ca}^{2+}$ to flow into the cell. This rise in cytosolic $\mathrm{Ca}^{2+}$ is sensed by specialized $\mathrm{Ca}^{2+}$-binding proteins, which alter cellular behavior by turning the activities of other proteins on or off. The importance of extracellular $\mathrm{Ca}^{2+}$ for cell activity was first documented by S. Ringer in 1883, when he discovered that isolated rat hearts suspended in an NaCl solution made with "hard" ( $\mathrm{Ca}^{2+}$-rich) London tap water contracted beautifully, whereas they beat poorly and stopped quickly if distilled, $\mathrm{Ca}^{2+}$-depleted, water was used.

Many $\mathrm{Ca}^{2+}$-binding proteins bind $\mathrm{Ca}^{2+}$ using the EF hand/helix-loop-helix structural motif discussed earlier (see Figure 3-10b). A well-studied EF hand protein, calmodulin, is found in all eukaryotic cells, where it may exist as an individual monomeric protein or as a subunit of a multimeric protein. This dumbbell-shaped molecule contains four $\mathrm{Ca}^{2+}$-binding EF hands with $K_{\mathrm{d}} \mathrm{s}$ of about $10^{-6} \mathrm{M}$. The binding of $\mathrm{Ca}^{2+}$ to calmodulin causes a conformational change that permits $\mathrm{Ca}^{2+} /$ calmodulin to bind to conserved sequences in various target proteins (Figure 3-33), thereby switching their activities on or off. Calmodulin and similar EF hand proteins thus function as switch proteins, acting in concert with changes in $\mathrm{Ca}^{2+}$ levels to modulate the activity of other proteins.
![li162857775e0](li162857775e0.jpg)
(b) $\mathrm{Ca}^{2+} /$ calmodulin bound to target peptide
![li163857775e0](li163857775e0.jpg)

FIGURE 3-33 Conformational changes induced by $\mathrm{Ca}^{2+}$ binding to calmodulin. Calmodulin is a widely distributed cytosolic protein that contains four $\mathrm{Ca}^{2+}$-binding sites, one in each of its EF hand (helix-loop-helix) motifs (EF1-ER4, see also Figure 3-10). At cytosolic $\mathrm{Ca}^{2+}$ concentrations above about $5 \times 10^{-7} \mathrm{M}$, binding of $\mathrm{Ca}^{2+}$ to calmodulin changes the protein's conformation from the dumbbell-shaped, $\mathrm{Ca}^{2+}$ free form (a) to one in which hydrophobic side chains become more exposed to solvent. The resulting $\mathrm{Ca}^{2+} /$ calmodulin complex can wrap around exposed helices (target peptides) with specialized sequences in various target proteins (b), thereby altering their activities. [Part (a) data from H. Kuboniwa et al., 1995, Nat. Struct. Biol. 2:768-776, PDB ID 1cfd. Part (b) data from W. E. Meador, A. R. Means, and F. A. Quiocho, 1992, Science 257:1251-1255, PDB ID 1cdl.]

Switching Mediated by Guanine Nucleotide-Binding Proteins Another group of intracellular switch proteins constitutes the GTPase superfamily. As the name suggests, these proteins are enzymes-GTPases-that can hydrolyze GTP (guanosine triphosphate) to GDP (guanosine diphosphate). They include the monomeric Ras protein (whose structure is shown in Figure 3-9, with bound GDP shown in blue) and the $\mathrm{G}_{\mathrm{q}}$ subunit of the trimeric G proteins, both discussed at length in Chapters 15 and 16. Both Ras and $\mathrm{G}_{\mathrm{q}}$ can bind to the plasma membrane, function in cell signaling, and play key roles in cell proliferation and differentiation. Other members of the GTPase superfamily function in protein synthesis, the transport of proteins between the nucleus and the cytoplasm, the formation of coated vesicles and their fusion with target membranes, and rearrangements of the actin cytoskeleton. Some GTPase proteins have a covalently attached lipid chain (see Figure 7-19) that mediates their binding to membranes. We examine the roles of various GTPase switch proteins in regulating intracellular signaling and other processes in several later chapters.

All the GTPase switch proteins exist in two forms, or conformations (Figure 3-34): (1) an active ("on") form with bound GTP, which can influence the activity of specific target proteins to which they bind, and (2) an inactive ("off") form with bound GDP. The switch is turned on-that is, the conformation of the protein changes from inactive to active-when a GTP molecule replaces a bound GDP in the inactive conformation. The switch is turned off when the relatively slow GTPase activity of the protein hydrolyzes bound GTP, converting it to GDP and leading the conformation to change to the inactive form. The amount of time any given GTPase switch remains in the active, GTP-bound form depends on the rate of its GTPase activity. Thus the GTPase activity acts as a timer to control this switch. Cells contain a variety of proteins that can modulate the baseline (or intrinsic) rate of GTPase activity for any given GTPase switch and so can control how long the switch remains on.

For example, GTPase-activating proteins, or GAPs, increase the rate of GTPase activity, thus reducing the time the GTPase is in the active form. Cells also have specific proteins whose function is to regulate the conversion of inactive GTPases to active ones-that is, to turn the switch on-by mediating the replacement of bound GDP with GTP (GDP/ GTP exchange). These proteins are called guanine nucleotide exchange factors, or GEFs. GTPases with lipid anchors are also regulated by proteins called guanine nucleotide dissociation inhibitors (GDIs) that bind to the lipid chain and thus influence interactions with cellular membranes.

■
The GAPs, GEFs, and GDIs are themselves subject to regulation and, together with their GTPases, participate in complex regulatory networks that control a vast array of cellular activities. It is, therefore, not surprising that disruptions of these finely tuned regulatory networks by mutations or pathogens are associated with a wide variety of diseases. Examples of genetic diseases affecting these networks include Noonan syndrome (a developmental disorder), retinitis pigmentosa (a degenerative eye disease), and X-linked mental retardation. Examples of disruptions of these networks by pathogens include bacterially induced food poisoning, dysenteries (inflammation of the intestines with diarrhea), Legionnaires' disease (a severe type of pneumonia that involves lung inflammation), and even the plague [also called the Black Death, which between 1347 and 1351 decimated the populations of China ( $\sim 50$ percent death rate) and Europe ( $\sim 33$ percent death rate)].

## Phosphorylation and Dephosphorylation Covalently Regulate Protein Activity

In addition to exploiting the noncovalent regulators described above, cells can use covalent modifications to regulate the intrinsic activity of a protein. One of the most common covalent mechanisms for regulating protein activity

FIGURE 3-34 The GTPase switch. GTPases are enzymes that bind to GTP and hydrolyze it to GDP. When bound to GTP, the GTPase protein adopts its active, or "on," conformation and can interact with target proteins to regulate their activities. When the bound GTP is hydrolyzed to GDP by the intrinsic GTPase activity of the protein, the GTPase with GDP bound assumes an inactive, or "off," conformation.
![li164857775e0](li164857775e0.jpg)

The GTPase switch can be turned back on when another protein, called a GEF (guanine nucleotide exchange factor), mediates the replacement (exchange) of the bound GDP with a GTP molecule from the surrounding fluid. GTPase-activating proteins, or GAPs, can influence the rates of GTP hydrolysis. The binding of the active form of the GTPase to its targets is a form of noncovalent regulation.

is phosphorylation, the reversible addition of phosphate groups to hydroxyl groups on the side chains of serine, threonine, or tyrosine residues. Phosphorylated proteins are called phosphoproteins. Phosphorylation is catalyzed by enzymes called protein kinases, while the removal of phosphates, known as dephosphorylation, is catalyzed by phosphatases. The counteracting activities of kinases and phosphatases provide cells with a "switch" that can turn on or turn off the function of various proteins that are the substrates (or targets) of these enzymes (Figure 3-35). Sometimes phosphorylation sites are masked transiently by reversible covalent modification with the sugar N -acetylglucosamine (called O-GlcNAcylation), which is an additional means of covalent regulation. Phosphorylation changes a protein's charge and to some extent its surface shape; it can also result in conformational changes. As a consequence, phosphorylation (or dephosphorylation) can influence the location of a protein within cells (e.g., its attachment to the inner surface of the plasma membrane), its intrinsic (e.g., enzymatic) activity, its ability to bind to other molecules, including metabolites, DNA, or other proteins, its ability to undergo further covalent modification, or its stability (rate of degradation). In addition, several conserved protein domains, such as the SH2 domain (see Figure 16-11), bind specifically to phosphorylated peptides. Thus phosphorylation can mediate the formation of protein complexes that can generate or extinguish a wide variety of cellular activities, discussed in many subsequent chapters.

Nearly 3 percent of all yeast proteins are protein kinases or phosphatases, indicating the importance of phosphorylation and dephosphorylation reactions even in these simple cells. Analysis of the human genome indicates there are approximately 500 human protein kinases (the human "kinome"). All classes of proteins-including structural proteins, scaffolds, enzymes, membrane channels, and signaling molecules-have members regulated by kinase/phosphatase modifications. Different protein kinases and phosphatases
![li165857775e0](li165857775e0.jpg)

FIGURE 3-35 Regulation of protein activity by phosphorylation and dephosphorylation. The cyclic phosphorylation and dephosphorylation of a protein is a common cellular mechanism for regulating protein activity. In this example, the target protein is active (top) when phosphorylated and inactive (bottom) when dephosphorylated; some proteins have the opposite response to phosphorylation.
are specific for different target proteins, often recognizing different linear sequences in which the residue to be phosphorylated is embedded, and so can regulate distinct cellular pathways, as discussed in later chapters. Some kinases have many targets, so that a single kinase can serve to integrate the activities of many targets simultaneously. Frequently, the target of a kinase or phosphatase is yet another kinase or phosphatase, creating a cascade effect. There are many examples of such kinase cascades, which permit amplification of a signal and many levels of fine-tuning (see Chapters 15 and 16).

## Ubiquitinylation and Deubiquitinylation Covalently Regulate Protein Activity

Both ubiquitin and ubiquitin-like proteins (of which there are more than a dozen in humans) can be covalently linked to a target protein in a regulated fashion, in a manner analogous to phosphorylation. Deubiquitinases can reverse ubiquitinylation in a manner analogous to the action of phosphatases. These ubiquitin modifications are structurally far more complex than phosphorylation, however, and so can mediate many distinct interactions between the ubiquitinylated protein and other cellular proteins. Ubiquitinylation can involve attachment of a single ubiquitin to a protein (monoubiquitinylation), addition of multiple, single ubiquitin molecules to different sites on one target protein (multiubiquitinylation), or addition of a polymeric chain of ubiquitins to a protein (polyubiquitinylation). An additional source of variation is that different amino groups in the ubiquitin molecule can be used to form an isopeptide bond with the C-terminal Gly-76 in another ubiquitin to form a polyubiquitin chain. All seven lysine residues in ubiquitin (Lys-6, Lys-11, Lys-27, Lys-29, Lys-33, Lys-48, and Lys-63) and its N-terminal amino group can participate in inter-ubiquitin linkages. Different ubiquitin ligases are specific both for targets (substrates) to be ubiquitinylated and for the lysine side chains on the ubiquitins that participate in the inter-ubiquitin isopeptide linkages (Lys-63 or Lys-48, etc.) (Figure 3-36). These multiple forms of ubiquitinylation result in the generation of a wide variety of recognition surfaces that can participate in many protein-protein interactions with the hundreds of proteins ( $>200$ in humans) that contain more than a dozen distinct ubiquitin-binding domains (UBD). In addition, any given polyubiquitin chain has the potential to bind simultaneously to more than one UBD-containing protein, leading to the formation of ubiquitinylation-dependent multiprotein complexes. Some deubiquitinases can remove an intact polyubiquitin chain from a modified protein ("anchored" chain) and thus generate a polyubiquitin chain not covalently linked to another protein ("unanchored" chain). Even these unanchored chains may serve a regulatory role. With this great structural diversity, it is not surprising that cells use ubiquitinylation and deubiquitinylation to control many different cellular functions.

We have already seen how polyubiquitinylation via Lys-48 residues is used to tag proteins for proteasomal degradation.

![li166857775e0](li166857775e0.jpg)

FIGURE 3-36 Determination of polyubiquitin function by the lysine used for inter-ubiquitin isopeptide bonds. Different ubiquitin ligases catalyze polyubiquitinylation of distinct target (substrate) proteins (colored ovals) using distinct lysine side chains of ubiquitin molecules (purple) to generate the inter-ubiquitin isopeptide linkages (blue) with Gly-76 of the adjacent ubiquitin. Dotted blue arrows represent additional ubiquitins in the chain that are not shown. The lysine used
for the isopeptide bonds determines the function of the polyubiquitinylation. For example, polyubiquitins with Lys-48:Gly-76 isopeptide bonds direct the target to proteasomes for degradation. Those that use Lys-63, Lys-33, and Lys-11 influence signaling, T-lymphocyte control, and cell division, respectively. Isopeptide bonds involving ubiquitin's Lys-6, Lys-27, and Lys-29 and bonds using its N-terminal amino group (not shown) can also be used to generate polyubiquitin chains.

There is evidence that polyubiquitinylation via other Lys residues (for example, Lys-11 and Lys-33, but not Lys-63) can also target proteins for proteasomal degradation. Strikingly, ubiquitinylation unrelated to protein degradation can also control diverse cell functions, including cellular internalization of molecules via endocytosis (see Chapter 14), repair of damaged DNA, metabolism, messenger RNA synthesis (transcription), defense against pathogens, cell division/cell cycle progression, cell signaling pathways, trafficking of proteins within a cell, and apoptosis. The lysine used to form the inter-ubiquitin isopeptide bonds can vary depending on the cellular system that is regulated (see Figure 3-36). For example, polyubiquitinylation with Lys-63 linkages is used in many cellular identification and signaling systems, such as recognition of the presence of intracellular viruses and bacteria and the consequent induction of a protective immune response, as well as direction of these pathogens to lysosomes for degradation. Lys-11-linked polyubiquitin chains regulate cell division. Lys-33-linked chains help suppress the activity of receptors on specialized white blood cells, called T lymphocytes (see Chapter 23), and so control the activity and function of those lymphocytes.

## Proteolytic Cleavage Irreversibly Activates or Inactivates Some Proteins

Unlike phosphorylation and ubiquitinylation, which are reversible, the activation or inactivation of protein function by proteolytic cleavage is an irreversible mechanism for regulating protein activity. For example, many polypeptide hormones, such as insulin, are synthesized as longer precursors, and prior to secretion from cells some of their peptide bonds must be hydrolyzed for them to fold properly. In some cases, a single long precursor prohormone polypeptide is cleaved into several distinct active hormones. To prevent the pancreatic serine proteases from inappropriately digesting proteins before they reach the small intestine, they are synthesized as zymogens, inactive precursor enzymes. Cleavage of a peptide bond near the N -terminus of trypsinogen (the zymogen of trypsin) by a highly specific protease in the small intestine generates a new N -terminal residue (Ile-16), whose amino group can form an ionic bond with the carboxylic acid side chain of an internal aspartic acid. This binding causes a conformational change that opens the substrate-binding site, activating the enzyme. The active trypsin can then activate

trypsinogen, chymotrypsinogen, and other zymogens. Similar but more elaborate protease cascades (with one protease activating inactive precursors of others) that can amplify an initial signal play important roles in several systems, such as the blood-clotting cascade and the complement system (see Chapter 23). The importance of carefully regulating such systems is clear-inappropriate clotting, for example, could fatally clog the circulatory system, while insufficient clotting could lead to uncontrolled bleeding.

An unusual and rare type of proteolytic processing, termed protein self-splicing, takes place in bacteria and some eukaryotes. This process is analogous to editing film: an internal segment of a polypeptide is removed and the ends of the polypeptide are rejoined (ligated). Unlike other forms of proteolytic processing, protein self-splicing is an autocatalytic process, which proceeds by itself without the participation of other enzymes. The excised peptide appears to eliminate itself from the protein by a mechanism similar to that used in the processing of some RNA molecules (see Chapter 10). In vertebrate cells, the processing of some proteins includes self-cleavage, but the subsequent ligation step is absent. One such protein is Hedgehog, a membranebound signaling molecule that is critical to a number of developmental processes (see Chapter 16).

## Higher-Order Regulation Includes Control of Protein Location

All the regulatory mechanisms heretofore described affect a protein locally at its site of action, altering the protein's concentration or turning its activity on or off. Normal functioning of a cell, however, also requires the segregation of proteins to particular compartments, such as the mitochondria, nucleus, or lysosomes. In regard to enzymes, compartmentation not only provides an opportunity for controlling the delivery of substrate or the exit of product, but also permits competing reactions to take place simultaneously in different parts of a cell. We describe the mechanisms that cells use to direct various proteins to different compartments in Chapters 13 and 14.

## KEY CONCEPTS OF SECTION 3.4

## Regulating Protein Function

- Proteins may be regulated at the level of protein synthesis, protein degradation, or the intrinsic activity of proteins through noncovalent or covalent interactions.
- The life span of intracellular proteins is largely determined by their susceptibility to proteolytic degradation.
- Many proteins are marked for destruction with a polyubiquitin tag by ubiquitin ligases and then degraded within proteasomes, large cylindrical complexes with multiple protease active sites in their interior chambers (see Figure 3-31).
- Ubiquitinylation of proteins is reversible due to the activity of deubiquitinylating enzymes.
- In allostery, the noncovalent binding of one ligand molecule, the allosteric effector, induces a conformational change that alters a protein's activity or affinity for other ligands. The allosteric effector can be identical in structure to or different from the other ligands, whose binding it affects. The allosteric effector can be an activator or an inhibitor.
- In multimeric proteins, such as hemoglobin, that bind multiple identical ligand molecules (e.g., oxygen), the binding of one ligand molecule may increase or decrease the protein's affinity for subsequent ligand molecules. This type of allostery is known as cooperativity (see Figure 3-32).
- Several allosteric mechanisms act as switches, turning protein activity on and off in a reversible fashion.
- Two classes of intracellular switch proteins regulate a wide variety of cellular processes: (1) $\mathrm{Ca}^{2+}$-binding proteins (e.g., calmodulin) and (2) members of the GTPase superfamily (e.g., Ras), which cycle between active GTP-bound and inactive GDP-bound forms (see Figure 3-34). GTPases participate in complex regulatory networks that include proteins (GAP, GEF, GDI) that regulate the cycling of the GTPase between its active and inactive forms.
- The phosphorylation and dephosphorylation of hydroxyl groups on serine, threonine, or tyrosine side chains by protein kinases and phosphatases provide reversible on/off regulation of numerous proteins (see Figure 3-35).
- Variations in the nature of the covalent attachment of ubiquitin to proteins (mono-, multi-, and polyubiquitinylation involving a variety of linkages between the ubiquitin monomers) are involved in a wide variety of cellular functions other than proteasome-mediated degradation, such as changes in the location or activity of proteins (see Figure 3-36).
- Many types of covalent and noncovalent regulation are reversible, but some forms of regulation, such as proteolytic cleavage, are irreversible.
- Higher-order regulation includes the intracellular location, or compartmentation, of proteins.


### 3.5 Purifying, Detecting, and Characterizing Proteins

A protein often must be purified before its structure and the mechanism of its action can be studied in detail. However, because proteins vary in size, shape, oligomerization state, charge, and water solubility, no single method can be used to isolate all proteins. To isolate one particular protein from the estimated 10,000 different proteins in a particular type of cell is a daunting task that requires methods both for separating proteins and for detecting the presence of specific proteins.

Any molecule, whether protein, carbohydrate, or nucleic acid, can be separated, or resolved, from other molecules on the basis of their differences in one or more physical or chemical characteristics. The larger and more numerous the differences between two proteins, the easier and more efficient their separation. As a practical matter, the more abundant a particular protein is in a biological sample, the easier it is to separate it from the other molecules in the sample. The three most widely used characteristics for separating proteins are size, defined as either length or mass; net electrical charge; and affinity for specific ligands. In this section, we briefly outline several important techniques for separating proteins; these separation techniques are also useful for the separation of nucleic acids and other biomolecules. (Specialized methods for removing membrane proteins from membranes are described in Chapter 7 after the unique properties of these proteins are discussed.) We then consider the use of radioactive compounds for tracking biological activity. Finally, we consider several techniques for characterizing a protein's mass, sequence, and three-dimensional structure.

## Centrifugation Can Separate Particles and Molecules That Differ in Mass or Density

The first step in a typical protein purification scheme is centrifugation. The principle behind centrifugation is that two types of particles in suspension (cells, cell fragments, organelles, or molecules) with different masses or densities will settle to the bottom of a test tube at different rates. Remember, mass is the weight of a sample (measured in daltons or molecular weight units), whereas density is the ratio of its mass to volume (often expressed as grams per liter because of the methods used to measure density). Proteins vary greatly in mass, but not in density. Unless a protein has an attached lipid or carbohydrate, its density will not vary by more than 15 percent from $1.37 \mathrm{~g} / \mathrm{cm}^{3}$, the average protein density. Heavier or denser molecules settle, or sediment, more quickly than lighter or less dense molecules.

A centrifuge speeds sedimentation by subjecting particles in suspension to centrifugal forces as great as 1 million times the force of gravity, $g$, which can sediment particles as small as 10 kDa . Modern ultracentrifuges achieve these forces by reaching speeds of 150,000 revolutions per minute (rpm) or greater. However, small particles with masses of 5 kDa or less will not sediment uniformly even at such remarkably high rotation rates. The extraordinary technical achievements of modern ultracentrifuges can be appreciated by considering that they can rotate a several-pound rotor (about the size of an American football) that holds the samples in tubes at rates as high as 2500 revolutions per second!

Centrifugation is used for two basic purposes: (1) as a preparative technique to separate one type of material from others with the goal of obtaining enough of the material to perform subsequent experiments and (2) as an analytical technique to measure physical properties (e.g., molecular
weight, density, shape, and equilibrium binding constants) of macromolecules. The sedimentation constant, $s$, of a protein is a measure of its sedimentation rate. The sedimentation constant is commonly expressed in Svedberg units (S), where a typical large protein complex is about $3-5 \mathrm{~S}$, a proteasome is 26 S , and a eukaryotic ribosome is 80 S .

Differential Centrifugation The most common initial step in protein purification from cells or tissues is the separation of water-soluble proteins from insoluble cellular material by differential centrifugation. A starting mixture, commonly a cell homogenate (mechanically broken cells), is poured into a tube and spun at a rotor speed, and for a period of time, that forces cell organelles such as nuclei as well as large unbroken cells or large cell fragments to collect as a pellet at the bottom; the soluble proteins remain in the supernatant (Figure 3-37a). The supernatant fraction then is poured off, and either it or the pellet can be subjected to other purification methods to separate the many different proteins that they contain.

Rate-Zonal Centrifugation On the basis of differences in their masses, water-soluble proteins can be separated by centrifugation through a solution of increasing density, called a density gradient. A concentrated sucrose solution is commonly used to form a density gradient in a centrifuge tube (with higher concentrations of sucrose, and thus a higher solution density, toward the bottom of the tube, Figure 3-37b). When a protein mixture is placed on top of a sucrose density gradient in a tube and subjected to centrifugation, each protein in the mixture migrates down the tube at a rate controlled by the protein's physical properties. All the proteins start from the thin layer of the sample that was placed at the top of the tube and separate into bands (actually, disks) of proteins of different masses as they travel at different rates through the gradient. In this separation technique, called rate-zonal centrifugation, samples are centrifuged just long enough to separate the molecules of interest into discrete bands, also called zones (see Figure 3-37b). If a sample is centrifuged for too short a time, the different protein molecules will not separate sufficiently. If a sample is centrifuged much longer than necessary, all the proteins will end up mixed together at the bottom of the tube.

Although the sedimentation rate is strongly influenced by particle mass, rate-zonal centrifugation is seldom effective in determining precise molecular weights because variations in shape also affect the sedimentation rate. The exact effects of shape are hard to assess, especially for proteins or other molecules, such as single-stranded nucleic acid molecules, that can assume many complex shapes. Nevertheless, ratezonal centrifugation has proved to be a practical method for separating many different types of polymers and particles. A second density-gradient technique, called equilibrium density-gradient centrifugation, is used mainly to separate DNA, lipoproteins that carry lipids through the circulatory system, or organelles (see Figure 4-37).

![li167857775e0](li167857775e0.jpg)

## Electrophoresis Separates Molecules on the Basis of Their Charge-to-Mass Ratio

Electrophoresis, a technique for separating molecules in a mixture under the influence of an applied electric field, is one of the most frequently used techniques to study proteins and nucleic acids. Dissolved molecules in an electric field move, or migrate, at a speed determined by their charge-to-mass (charge:mass) ratio and the physical properties of the medium through which they migrate. For example, if two molecules have the same mass and shape, the one with the greater net charge will move faster toward an electrode of the opposite polarity.

SDS-Polyacrylamide Gel Electrophoresis Because many proteins or nucleic acids that differ in size and shape have nearly identical charge:mass ratios, electrophoresis of these macromolecules in a liquid solution results in little or no separation of molecules of different lengths. However, successful separation of proteins and nucleic acids can be
accomplished by electrophoresis in various gels (semisolid suspensions in water similar to the congealed gelatin found in desserts). These gels are commonly cast into flat, relatively thin slabs between a pair of glass plates. When a mixture of proteins is placed in a gel and an electric current is applied, the gel acts as a sieve, allowing smaller species to maneuver more rapidly through its pores than larger species do. The shape of a molecule can also influence its rate of migration (long asymmetric molecules migrate more slowly than spherical ones of the same mass).

Electrophoretic separation of proteins is most commonly performed in polyacrylamide gels. These gels are made by polymerizing a solution of acrylamide monomers into polyacrylamide chains and simultaneously cross-linking the chains into a semisolid matrix. The pore size of a gel can be varied by adjusting the concentrations of polyacrylamide and the cross-linking reagent. The rate at which a protein moves through a gel is influenced by the gel's pore size and the strength of the electric field. By suitable adjustment of

these parameters, proteins of widely varying sizes can be resolved (separated from one another) by this technique, known as polyacrylamide gel electrophoresis (PAGE).

In the most powerful technique for resolving protein mixtures, proteins are exposed to the ionic detergent SDS (sodium dodecylsulfate) before and during gel electrophoresis (Figure 3-38). SDS denatures proteins, in part because
![li168857775e0](li168857775e0.jpg)

EXPERIMENTAL FIGURE 3-38 SDS-polyacrylamide gel electrophoresis (SDS-PAGE) separates proteins primarily on the basis of their masses. (a) Initial treatment with SDS, a negatively charged detergent, dissociates multimeric proteins and denatures all the polypeptide chains (step $\mathbf{\square}$ ). During electrophoresis, the SDSProtein complexes migrate through the polyacrylamide gel (step $\mathbf{\square}$ ). Small complexes are able to move through the pores faster than larger ones. Thus the proteins separate into bands according to their sizes as they migrate. The separated protein bands are visualized by staining with a dye (step $\mathbf{\square}$. (b) Example of SDS-PAGE separation of all the proteins in a whole-cell lysate (detergent-solubilized cells). (Left) The many separate stained proteins appear almost as a continuum. (Right) A single protein purified from the lysate by a single step of antibodyaffinity chromatography. The proteins were visualized by staining with a silver-based dye. (Part (b) data from B. Liu and M. Krieger, 2002, J. Biol. Chem. 277:3412S-3413S.)
it binds to hydrophobic side chains, destabilizing the hydrophobic interactions in the core of a protein that contribute to its stable conformation. SDS treatment is usually combined with heating, often in the presence of reducing agents that break disulfide bonds. Under these conditions, most multimeric proteins dissociate into their subunits. Typically, the amount of SDS that binds to the protein is proportional to the length of the polypeptide chain and relatively independent of the sequence. Two proteins of similar size will bind the same absolute quantity of SDS, whereas a protein twice that size will bind twice the amount of SDS. Denaturation of a complex protein mixture with SDS in combination with heat usually forces each polypeptide chain into an extended conformation and imparts on each of the proteins in the mixture a constant charge:mass ratio because the dodecylsulfate, which is negatively charged, is the major contributor of charge. As the SDS-bound proteins move through the polyacrylamide gel, they are separated according to size by the sieving action of the gel. SDS treatment thus eliminates the effect of differences in native conformation; therefore, chain length, which is proportional to mass, is the principal determinant of the migration rate of proteins in SDS-polyacrylamide electrophoresis (SDS-PAGE). Even chains that differ in molecular weight by less than 10 percent can be resolved by this technique. Moreover, the molecular weight of a protein can be estimated by comparing the distance that it migrates through a gel with the distances that proteins of known molecular weight (called molecular weight "standards") migrate in the same gel (there is a roughly linear relationship between migration distance and the log of the molecular weight). Proteins within the gels can be extracted for further analysis (e.g., identification by the methods described below).

If two or more polypeptides are cross-linked by disulfide bonds, the protein's migration rate in SDS-PAGE will depend on whether or not the protein has been reduced to break those bonds prior to electrophoresis. The cross-linked proteins will appear larger than the individual, reduced subunits. By examining samples with and without reduction, one can identify such proteins and their component polypeptides.

Two-Dimensional Gel Electrophoresis Electrophoresis of a mixture containing all cellular proteins by SDS-PAGE can separate proteins having relatively large differences in mass, but cannot readily resolve proteins having similar masses (e.g., a $41-\mathrm{kDa}$ protein versus a $42-\mathrm{kDa}$ protein). To separate proteins of similar masses, another physical characteristic must be exploited. Most commonly, this characteristic is electric charge, which is determined by the pH of the sample and by the relative number of the protein's positively and negatively charged groups, which is in turn dependent on the $\mathrm{p} K_{\mathrm{a}}$ 's of the ionizable groups (see Chapter 2) on the proteins (usually the amino and carboxyl termini and side chains such as those in lysine and aspartic acid). Two unrelated proteins having similar masses are unlikely to have identical net charges because their sequences, and thus the number of acidic and basic residues, are different.

![li169857775e0](li169857775e0.jpg)

EXPERIMENTAL FIGURE 3-39 Two-dimensional gel electrophoresis separates proteins on the basis of charge and mass. (a) In this technique, proteins are first separated into bands on the basis of their charges by isoelectric focusing (step 1). The resulting gel strip is applied to an SDS-polyacrylamide gel (step 2), and the proteins are separated into spots by mass (step 3). (b) In this two-dimensional
electrophoresis gel of a protein extract from cultured cells, each spot represents a single polypeptide. Polypeptides can be detected by dyes, as here, or by other techniques, such as autoradiography. Each polypeptide is characterized by its isoelectric point (pl) and molecular weight. [Part (b) Michael J. Dunn.]
In two-dimensional gel electrophoresis, proteins are separated sequentially, first by their charges and then by their masses (Figure 3-39a). In the first step, a cell or tissue extract is fully denatured by high concentrations ( 8 M ) of urea (and sometimes SDS) and then layered on a strip of gel that contains urea, which removes any bound SDS, and a continuous pH gradient. The pH gradient is formed by ampholytes, polyanionic and polycationic small molecules that are cast into the gel. When an electric field is applied to the gel, the ampholytes will migrate. Ampholytes with an excess of negative charges will migrate toward the anode, where they establish an acidic pH (many protons), while ampholytes with an excess of positive charges will migrate toward the cathode, where they establish an alkaline pH . The careful choice of the mixture of ampholytes and careful preparation of the gel allows the construction of stable pH gradients ranging from pH 3 to pH 10 . A charged protein placed at one end of such a gel will migrate through the gradient under the influence of the electric field until it reaches its isoelectric point ( $\mathbf{p l}$ ), the pH at which the net charge of the protein is zero. With no net charge, the protein will migrate no further. This technique, called isoelectric focusing (IEF), can resolve proteins that differ by only one charge unit. This method is sensitive enough to separate phosphorylated and nonphosphorylated versions of the same protein.

Proteins that have been separated on an IEF gel can then be separated in a second dimension on the basis of
their molecular weights. To accomplish this separation, the IEF gel strip is placed lengthwise on one outside edge of a square or rectangular slab of polyacrylamide gel, this time saturated with SDS to confer on each separated protein a more or less constant charge:mass ratio. When an electric field is imposed, the proteins will migrate from the IEF gel into the SDS gel and then separate according to their masses. The sequential resolution of proteins by charge and mass can achieve excellent separation of cellular proteins and provides a powerful visual representation of the complexity of proteins in cells (Figure 3-39b). Today sophisticated mass spectrometry methods, described below, are often used in place of two-dimensional gel electrophoresis, both to separate and to identify the protein components of a complex sample as well as to compare changes in the amounts of those components in different biological specimens.

## Liquid Chromatography Resolves Proteins by Mass, Charge, or Affinity

A third common technique for separating mixtures of proteins or fragments of proteins, as well as other molecules, is based on the principle that molecules in solution can differentially interact with (bind to and dissociate from) a particular solid surface, depending on the physical and chemical properties of the molecule and the surface. If the solution is allowed to flow across the surface, then molecules that

(a) Gel filtration chromatography
![li170857775e0](li170857775e0.jpg)

EXPERIMENTAL FIGURE 3-40 Three commonly used liquid chromatographic techniques separate proteins on the basis of mass, charge, or affinity for a specific binding partner. (a) Gel filtration chromatography separates proteins that differ in size. A mixture of proteins is carefully placed, or loaded, on the top of a cylinder packed with porous beads. Smaller proteins travel through the column more slowly than larger proteins. Thus the different proteins emerging in the eluate flowing out of the bottom of the column at different times (different elution volumes) can be collected in separate tubes, called fractions. (b) Ion-exchange chromatography separates proteins that differ in net charge in columns packed with beads that carry either a positive charge (shown here) or a negative charge. Proteins having the same net charge as the beads are repelled and flow through the column,
whereas proteins having the opposite charge bind to the beads more or less tightly, depending on their structures. Bound proteins-in this case, negatively charged proteins-are subsequently eluted by passing a salt gradient (usually of NaCl or KCl ) through the column. As the ions bind to the beads, they displace the proteins; more tightly bound proteins require higher salt concentrations in order to be released. (c) In antibodyaffinity chromatography, a mixture of proteins is passed through a column packed with beads to which a specific antibody is covalently attached. Only proteins with high affinity for the antibody are retained by the column; all the nonbinding proteins flow through. After the column is washed, the bound protein is eluted with an acidic solution or some other solution that disrupts the antigen-antibody complexes; the released protein then flows out of the column and is collected.
(Figure 3-40). The sample then flows down the column, driven by gravitational or hydrostatic forces alone or sometimes with the assistance of a pump. In some LC systems, the composition of the fluid flowing out of the column is monitored continuously (for example, by spectroscopy). Small aliquots of fluid flowing out of the column, called fractions,

are collected sequentially and can be analyzed subsequently for their contents and chemical activities (e.g., enzymatic activity). The nature of the beads in the column determines whether the separation of proteins depends on differences in their mass, charge, or other binding properties (e.g., affinity for substances attached to the beads).

Gel Filtration Chromatography Proteins that differ in mass can be separated on a column of porous beads made from polyacrylamide, dextran (a bacterial polysaccharide), or agarose (a seaweed derivative)-a technique called gel filtration chromatography. Although proteins flow around the beads, they spend some time within the large depressions that cover a bead's surface. Because smaller proteins can penetrate these depressions more readily than larger proteins can, they travel through a gel filtration column more slowly than larger proteins do (Figure 3-40a). (In contrast, proteins migrate through the pores in an electrophoretic gel; thus smaller proteins move faster than larger ones.) The total volume of liquid required to elute (or separate and remove) a protein from a gel filtration column depends on the protein's mass: the smaller its mass, the longer it is trapped on the beads, the longer it takes to traverse the column, and the greater the elution volume. If proteins of known mass are used as standards to calibrate the column, the elution volume can be used to estimate the mass of a protein in a mixture. A protein's shape as well as its mass can influence the elution volume.

Ion-Exchange Chromatography In ion-exchange chromatography, proteins are separated on the basis of differences in their charges. This technique makes use of specially modified beads whose surfaces are covered by amino groups or carboxyl groups and thus carry either a positive charge $\left(\mathrm{NH}_{3}{ }^{+}\right)$or a negative charge $\left(\mathrm{COO}^{-}\right)$at neutral pH .

The proteins in a mixture carry various net charges at any given pH . When a solution of mixed proteins flows through a column of positively charged beads, only proteins with a net negative charge (acidic proteins) adhere to the beads; neutral and positively charged (basic) proteins flow unimpeded through the column (Figure 3-40b). The acidic proteins are then eluted selectively from the column by passing a solution of increasing concentrations of salt (a salt gradient) through the column. At low salt concentrations, protein molecules and beads are attracted by their opposite charges. At higher salt concentrations, negatively charged salt ions bind to the positively charged beads, displacing the negatively charged proteins. In a gradient of increasing salt concentrations, weakly bound proteins-those with a relatively low chargeare eluted first, and highly charged proteins are eluted last. Similarly, a negatively charged column can be used to retain and fractionate basic (positively charged) proteins.

Affinity Chromatography The ability of proteins to bind specifically to other molecules is the basis of affinity chromatography. In this technique, ligands or other molecules that bind to the protein of interest are covalently attached to the beads used to form the column. Ligands can be enzyme substrates, inhibitors or their analogs, or other small molecules that bind
to specific proteins. In a widely used form of this technique-antibody-affinity, or immunoaffinity, chromatography-the molecule attached to the beads is an antibody specific for the desired protein (Figure 3-40c). (We discuss antibodies as tools for studying proteins next; see also Chapter 23, which describes how antibodies are made.)

In principle, an affinity column will retain only those proteins that bind the molecule attached to the beads; the remaining proteins, regardless of their charges or masses, will pass through the column because they do not bind. However, if a retained protein is in turn bound to other molecules, forming a complex, then the entire complex is retained on the column. The proteins bound to the affinity column are then eluted by adding an excess of a soluble form of the ligand, by exposure of bound materials to detergents, or by changing the salt concentration or pH such that the binding to the molecule on the column is disrupted. The ability of this technique to separate particular proteins depends on the selection of appropriate binding partners that bind more tightly to the protein of interest than to other proteins.

## Highly Specific Enzyme and Antibody Assays Can Detect Individual Proteins

The purification of a protein, or any other molecule, requires a specific assay that can detect the presence of that molecule as it is separated from other molecules (e.g., in column or density-gradient fractions or gel bands or spots). Such an assay capitalizes on some highly distinctive characteristic of a protein: the ability to bind a particular ligand, to catalyze a particular reaction, or to be recognized by a specific antibody. The assay must also be simple and fast to minimize errors and the possibility that the protein of interest will become denatured or degraded while the assay is being performed. The goal of any purification scheme is to isolate sufficient amounts of a given protein for study; thus a useful assay must also be sensitive enough that only a small proportion of the available material is consumed by it. Many common protein assays require just $10^{-9}$ to $10^{-12} \mathrm{~g}$ of material.

Chromogenic Enzyme Reactions Many assays are tailored to detect some functional aspect of a protein. For example, assays of enzymatic activity are based on the ability to detect the loss of substrate or the formation of product. Some enzymatic activity assays use chromogenic substrates, which change color in the course of the reaction. (Some substrates are naturally chromogenic; those that are not can be linked to a chromogenic molecule.) Because of the specificity of an enzyme for its substrate, only samples that contain the enzyme will change color in the presence of a chromogenic substrate; the rate of the change provides a measure of the quantity of enzyme present. Enzymes that catalyze chromogenic reactions can also be fused or chemically linked to an antibody and used to "report" the presence or location of an antigen to which the antibody binds (see below).

Antibody Assays As noted earlier, antibodies have the distinctive characteristic of binding tightly and specifically

to antigens. As a consequence, preparations of antibodies that recognize a protein antigen of interest can be generated and used to detect the presence of that protein, either in a complex mixture of other proteins (finding a needle in a haystack, as it were) or in a partially or completely purified preparation of a particular protein. The presence of the antigen can be detected by labeling the antibody with an enzyme, a fluorescent molecule, or a radioactive isotope, which can be detected using an enzyme assay, fluorescence microscopy or spectroscopy, or a radiation detector, respectively. For example, luciferase, an enzyme present in fireflies and some bacteria, can be linked to an antibody. In the presence of ATP and its substrate, luciferin, luciferase catalyzes a light-emitting reaction. In either case, after the antibody binds to the protein of interest (the antigen) and unbound antibody is washed away, substrates of the linked enzyme are added and the appearance of color or emitted light is monitored. The intensity is proportional to the amount of enzyme-linked antibody, and thus antigen, in the sample. Alternatively, after a first (or "primary") antibody that is not otherwise labeled binds to its target protein, a second ("secondary"), labeled antibody that can recognize the first antibody is used to bind to the complex of the first antibody and its target. This combination of primary and secondary antibodies (sometimes called an antibody "sandwich") permits very high sensitivity in the detection of a target protein because the labeled secondary antibody is often a mixture of antibodies that bind to multiple sites on the first antibody and thus results in a stronger signal than labeling of the primary antibody alone. It is important to remember that an antibody recognizes and binds to only its epitope on a target antigen. If that epitope is altered-for example, by partial unfolding or post-translational modifications-or is blocked when the antigen protein is bound to some other molecule, the ability of the antibody to bind may be reduced or completely lost. Thus the absence of antibody binding does not necessarily mean that the antigen is not present in a sample, only that the epitope portion of that antigen is not present or accessible for antibody binding.

To generate antibodies to a protein (discussed in detail in Chapter 23), the intact protein, or a fragment of the protein, is injected into an animal (usually a rabbit, mouse, or goat). Sometimes a short synthetic peptide of $10-15$ residues based on the sequence of the protein of interest is used as the antigen to induce antibody formation. Such a synthetic peptide, when coupled to a large protein carrier, can induce an animal to produce antibodies that bind specifically to that part (the epitope) of the full-sized, natural protein. Biosynthetically or chemically attaching the epitope to an unrelated protein is called epitope tagging. As we'll see throughout this book, antibodies generated using either synthetic peptide epitopes or intact proteins are extremely versatile reagents for isolating, detecting, and characterizing proteins.

## Detecting Proteins by Attaching Green Fluorescent

Protein An alternative to epitope tagging that is particularly useful in detecting specific proteins within live cells makes use of green fluorescent protein (GFP), a naturally fluorescent
protein found in jellyfish (see Figure 4-16). A chimeric protein containing both the protein of interest and GFP, linked together in one polypeptide chain, is expressed in cells by introducing into the cells a gene encoding the combined protein. The amounts and intracellular distribution of the chimeric protein can then be determined readily. This chimeric protein approach is described in Chapter 4.

Detecting Proteins in Gels Proteins embedded within a gel usually are not visible. The two general approaches to detecting proteins in gels are either to label or stain the proteins while they are still within the gel or to electrophoretically transfer the proteins to a membrane made of nitrocellulose or polyvinylidene difluoride and then detect them. Proteins within gels are usually stained with an organic dye or a silver-based stain, both of which are detectable with normal visible light, or with a fluorescent dye, which requires specialized detection equipment. Coomassie blue, the most commonly used organic dye, is typically used to detect about 1000 ng of protein, with a lower limit of detection of about $4-10 \mathrm{ng}$. Silver staining and fluorescence staining are more sensitive (with a lower limit of $\sim 1 \mathrm{ng}$ ). Coomassie and other stains can also be used to visualize proteins after transfer to membranes; however, the most common method of visualizing proteins in membranes is immunoblotting.

Immunoblotting, also called Western blotting, combines the resolving power of gel electrophoresis with the specificity of antibodies. This multistep procedure is commonly used to separate proteins and then identify a specific protein of interest. As shown in Figure 3-41, two different antibodies are used, one that is specific for the protein of interest (primary antibody) and a secondary antibody that binds to the first and is linked to an enzyme or other molecule that permits detection of the first antibody (and thus the protein of interest to which it binds). The enzyme to which the second antibody is attached can either generate a visible colored product or, by a process called chemiluminescence, produce light that can readily be recorded by film or a sensitive detector. An example of the results of an immunoblotting experiment can be seen in Figure 15-10. If an antibody to the protein of interest is not available, but the gene encoding the protein is available and can be used to express the protein, recombinant DNA methods (see Chapter 5) can incorporate a small peptide epitope into the normal sequence of the protein (epitope tagging) that can be detected by a commercially available antibody to that epitope.

Immunoprecipitation Immunoprecipitation, often abbreviated as IP, exploits the specificity of antibodies to separate a protein of interest from other molecules in a complex mixture-for example, all proteins extracted from a sample of cells or a sample of blood. An antibody to the protein of interest is added to a sample, and the antibody is given time to bind to epitopes on the target protein. An agent that binds to the antibody is then added to cause the antibody and its bound target to precipitate out of solution into particles that can be isolated by centrifugation. A detailed example of this technique is described in Chapter 15 (p. 684). The precipitate is then solubilized under denaturing conditions-for example, in

![li171857775e0](li171857775e0.jpg)
(b) Immunoblotting of lysed cells to detect intracellular receptors
![li172857775e0](li172857775e0.jpg)

EXPERIMENTAL FIGURE 3-41 Immunoblotting (IP, or Western blotting) and co-immunoprecipitation (co-IP) can detect specific proteins and their binding partners. (a) Immunoblotting method. Step After a protein mixture has been electrophoresed through an SDS gel, the separated bands (or spots, for two-dimensional gel electrophoresis) are transferred (blotted) from the gel onto a porous membrane from which the protein is not readily removed. Individual proteins (represented by blue ovals) are not visible at this stage. Step The membrane is flooded with a solution of an antibody $\left(\mathrm{Ab}_{1}\right)$ specific for the protein of interest and allowed to incubate for a while. $\mathrm{Ab}_{1}$ binds to the protein of interest (second from the top), but not to any other proteins attached to the membrane, forming a layer of antibody molecules coincident with the protein (whose position still cannot be seen at this point). Then the membrane is washed to remove unbound $\mathrm{Ab}_{1}$. Step The membrane is incubated with a second antibody $\left(\mathrm{Ab}_{2}\right)$ that specifically recognizes and binds to the first $\left(\mathrm{Ab}_{1}\right)$. This second antibody is covalently linked to an enzyme that catalyzes a chromogenic reaction or releases light (e.g., chemiluminescence), a radioactive isotope, or some other substance whose presence can be detected with great sensitivity. Step $\boldsymbol{\square}$ Finally, the location and amount of bound $\mathrm{Ab}_{2}$ are detected (e.g., by its color for a chromogenic reaction or by detectors or film that measure the light released by chemiluminescence), permitting the electrophoretic mobility (and therefore the mass) of the protein of interest to be determined as well as its quantity (based on band intensity). (b) Immunoblotting was used to detect intracellular receptors and the influence of exposure to a ligand for one of the receptors. In this experiment, cells that are precursors to red blood cells were maintained in vitro in petri dishes and then treated with no ligand (-, leftmost and rightmost lanes) or a ligand that binds to GR, the glucocorticoid receptor ( + , center lane). The cells were then lysed in detergent, and immunoblotting (Western blotting) was performed on the total cell lysates using three different antibodies that bind to GR (anti-GR), to a receptor called PPAR $\alpha$ (anti-PPAR $\alpha$ ), or to an abundant intracellular protein, actin, whose presence and abundance was not expected to be sensitive to treatment with the ligand.
(c) Immunoprecipitation (IP) followed by immunoblotting (co-IP)
![li173857775e0](li173857775e0.jpg)

The equal intensities of the immunoblotting bands detected using the anti-actin antibody (bottom box) provided a "loading control," which established that essentially equal amounts of cell lysate were applied (loaded) in each lane of the gel. The approximately equal intensities of the bands for both GR and PPAR $\alpha$ with or without prior incubation of the cells with the GR ligand showed that the ligand did not substantially alter the amounts of either of these proteins in the cells. Portions of the same cell lysates used for the immunoblotting in part (b) were also used for the immunoprecipitation/immunoblotting shown in part (c). (c) Immunoprecipitation (IP) followed by immunoblotting (together called co-IP) was used to determine if the GR ligand can induce formation of a stable complex that contains both GR and PPAR $\alpha$. Portions of the cell lysates were immunoprecipitated with an antibody to GR (left and center lanes) or a control antibody (right lane) that cannot bind to either GR or PPAR $\alpha$. The immunoprecipitates were separated from the rest of the lysates by centrifugation and then analyzed by immunoblotting with either anti-GR (top box) or anti-PPAR $\alpha$ (bottom box). As expected, the top box shows that the GR protein was detected in the immunoprecipitates generated using the $\alpha$-GR when the same anti-GR antibody was used for the immunoblotting, but not in the immunoprecipitates generated with the control antibody (no band observed). Strikingly, when one examines the immunoprecipitates by immunoblotting with the anti-PPAR $\alpha$ antibody (bottom box), a substantial amount of PPAR $\alpha$ is seen when the GR ligand is present (center lane), whereas little co-precipitates in the absence of the GR ligand (left lane) or in the control immunoprecipitate (right lane). These results indicate that the GR ligand induces formation of a complex containing both the glucocorticoid receptor and the PPAR $\alpha$ proteins. These results do not establish whether or not the GR and PPAR $\alpha$ proteins bind directly to each other when the GR ligand is present or if there are additional molecules in the complex that act as intermediates holding the GR and PPAR $\alpha$ tightly together when the ligand is present. [Parts (b) and (c) reprinted by permission from Macmillan Publishers Ltd, from Lee, H.Y. et al., "PPAR- $\alpha$ and glucocorticoid receptor synergize to promote erythroid progenitor self-renewal," Nature, 2015, 522:474-477.]
3.5 Purifying, Detecting, and Characterizing Proteins

a detergent-containing buffer-to separate the antibody from the protein, and the immunoprecipitated target protein can then be analyzed. If the immunoprecipitated target is tightly bound to one or more other molecules, those bound molecules may be precipitated along with the protein of interest (coimmunoprecipitation, sometimes abbreviated as co-IP). The co-IP method is used frequently to identify and characterize quaternary structures and supramolecular complexes.

## Radioisotopes Are Indispensable Tools for Detecting Biological Molecules

A sensitive method for tracking a protein or other biological molecule is by detecting the radioactivity emitted from a radiolabel introduced into the molecule. In a radiolabeled molecule, at least one atom is present in a radioactive form, called a radioisotope.

Radioisotopes Useful in Biological Research Hundreds of biological molecules (e.g., amino acids, nucleosides, and numerous other small-molecule metabolites) labeled with various radioisotopes are commercially available. These preparations vary considerably in their specific activity, which is the amount of radioactivity per unit of material, measured in disintegrations per minute ( dpm ) per millimole (mmol). The specific activity of a labeled compound depends on the radioisotope's half-life, the time required for half the atoms to undergo radioactive decay, which releases the detectable radiation. In general, the shorter the half-life of a radioisotope, the higher its specific activity (Table 3-1). The specific activity of a labeled compound must be high enough for accurate detection of its emitted radiation.

A common approach to radiolabeling macromolecules (proteins, RNA, DNA) in cells is to add a radiolabeled biosynthetic precursor to the extracellular medium [e.g., ${ }^{3} \mathrm{H}$ - or ${ }^{35} \mathrm{~S}$-labeled amino acids, ${ }^{32} \mathrm{P}$-labeled phosphate (precursor for ${ }^{32} \mathrm{P}$-labeled ATP), or ${ }^{3} \mathrm{H}$-labeled nucleic acid precursors such as deoxythymidine (also simply called thymidine) or ${ }^{32} \mathrm{P}$-labeled phosphate]. The precursor enters the cells via transporters (see Chapter 11) and is incorporated into newly synthesized macromolecules by the cells (see Chapter 5). For example, methionine and cysteine labeled

TABLE 3-1 Radioisotopes Commonly Used in Biological Research

| Isotope | Half-Life |
| :-- | :-- |
| Phosphorus-32 | 14.3 days |
| Iodine-125 | 60.4 days |
| Sulfur-35 | 87.5 days |
| Tritium (hydrogen-3) | 12.4 years |
| Carbon-14 | 5730.4 years |

with sulfur-35 $\left({ }^{35} \mathrm{~S}\right)$ are widely used to label cellular proteins because preparations of these amino acids with high specific activities ( $>10^{15} \mathrm{dpm} / \mathrm{mmol}$ ) are available. Kinases within cells (or used in vitro) can transfer a ${ }^{32} \mathrm{P}$-labeled phosphate from ${ }^{32} \mathrm{P}$-labeled ATP to label phosphoproteins. Likewise, commercial preparations of ${ }^{3} \mathrm{H}$-labeled nucleic acid precursors have much higher specific activities than those of the corresponding ${ }^{14} \mathrm{C}$-labeled preparations. In most experiments, the former are preferable because they allow RNA or DNA to be adequately labeled a shorter time after incorporation or require a smaller cell sample. Various phosphate-containing compounds in which the phosphorus atom is the radioisotope phosphorus- 32 are readily available. Because of their high specific activity, ${ }^{32} \mathrm{P}$-labeled nucleotides are routinely used to label nucleic acids in cellfree systems.

Labeled compounds in which a radioisotope replaces atoms normally present in the molecule have virtually the same chemical properties as the corresponding unlabeled compounds. Enzymes, for instance, generally cannot distinguish between substrates labeled in this way and their unlabeled substrates. The presence of such radioactive atoms is indicated with the isotope in brackets (no hyphen) as a prefix (e.g., $\left[{ }^{3} \mathrm{H}\right]$ leucine). In contrast, labeling of almost any biomolecule (e.g., protein or nucleic acid) with the radioisotope iodine-125 $\left({ }^{123} \mathrm{I}\right)$ requires the covalent addition of ${ }^{123} \mathrm{I}$ to a molecule that normally does not have iodine as part of its structure. Because this labeling procedure modifies the chemical structure, the biological activity of the labeled molecule may differ somewhat from that of the unlabeled form. The presence of such radioactive atoms is indicated with the isotope as a prefix followed a hyphen (no bracket) (e.g., ${ }^{123} \mathrm{I}$-trypsin). Standard methods for labeling proteins with ${ }^{123} \mathrm{I}$ result in covalent attachment of the ${ }^{123} \mathrm{I}$ primarily to the aromatic rings of tyrosine side chains (mono- and diiodotyrosine). Nonradioactive isotopes are finding increasing use in cell biology, especially in nuclear magnetic resonance studies and in mass spectroscopy applications, as will be explained below.

Labeling Experiments and Detection of Radiolabeled Molecules Whether labeled compounds are detected by autoradiography-exposure of the sample on a two-dimensional detector (photographic emulsion or electronic detector)—or their radioactivity is measured in an appropriate "counter," the amount of a radiolabeled compound in a sample can be determined with great precision.

In one use of autoradiography, a tissue, cell, or cell constituent is labeled with a radioactive molecule, unassociated radioactive material is washed away, and the structure of the sample is stabilized either by chemically cross-linking the macromolecules in the sample ("fixation") or by freezing it. The sample is then overlaid with a photographic emulsion that is sensitive to radiation. Development of the emulsion yields small silver grains whose distribution corresponds to that of the radioactive material and is usually detected by microscopy. Autoradiographic studies of whole

cells were crucial in determining the intracellular sites where various macromolecules are synthesized and the subsequent movements of those macromolecules within cells. Various techniques employing fluorescence microscopy, which we describe in Chapter 4, have largely supplanted autoradiography for studies of this type. However, autoradiography is sometimes used in various assays for detecting specific isolated DNA or RNA sequences at specific tissue locations (see Chapter 6) in a technique referred to as in situ hybridization.

Quantitative measurements of the amount of radioactivity in a labeled material are performed with several different instruments. A Geiger counter measures ions produced in a gas by the $\beta$ particles or $\gamma$ rays emitted from a radioisotope. These instruments are mostly handheld devices used to monitor radioactivity in the laboratory to protect investigators from excess exposure. In a scintillation counter, a radiolabeled sample is mixed with a liquid containing a fluorescent compound that emits a flash of light when it absorbs the energy of the $\beta$ particles or $\gamma$ rays released in the decay of the radioisotope; a phototube in the instrument detects and counts these light flashes. Phosphorimagers detect radioactivity using a two-dimensional array detector, storing digital data on the number of disintegrations per minute per small pixel of surface area. These instruments, which can be thought of as a kind of reusable electronic film, are commonly used to quantify radioactive molecules separated by gel electrophoresis and are replacing photographic emulsions for this purpose.

Combinations of labeling and biochemical techniques and of visual and quantitative detection methods are often employed in labeling experiments. For instance, to identify the major proteins synthesized by a particular cell type, a sample of the cells is incubated with a radiolabeled amino acid (e.g., $\left[{ }^{35} \mathrm{~S}\right]$ methionine) for a few minutes, during which time the labeled amino acid enters the cells and mixes with the cellular pool of unlabeled amino acids, and some of it is biosynthetically incorporated into newly synthesized proteins. Subsequently, unincorporated radiolabeled amino acid is washed away from the cells. The cells are harvested, and the mixture of cellular proteins is extracted from the cells (for example, by a detergent solution) and then separated by any of the methods commonly used to resolve complex protein mixtures into individual components. Gel electrophoresis in combination with autoradiography or phosphorimager analysis is often the method of choice. The radioactive bands in the gel correspond to newly synthesized proteins, which have incorporated the radiolabeled amino acid. To detect a specific protein of interest, rather than the entire ensemble of biosynthetically radiolabeled proteins, a specific protein can be isolated by immunoprecipitation. The precipitate is then solubilized, for example, in an SDS-containing buffer, and the sample is analyzed by SDS-PAGE followed by autoradiography to detect the protein that is radioactively labeled. In this type of experiment, a fluorescent compound that is activated by the radiation ("scintillator") may be infused into the gel on completion of the electrophoretic separation so that the light emitted can be used to detect the presence of
![li174857775e0](li174857775e0.jpg)
(b)
![li175857775e0](li175857775e0.jpg)

## EXPERIMENTAL FIGURE 3-42 Pulse-chase experiments can

track the pathway of protein modification within cells. (a) To follow the fate of a specific newly synthesized protein in cells, cells were incubated with $\left[{ }^{35} \mathrm{~S}\right]$ methionine for 0.5 hours (the pulse) to label all newly synthesized proteins, and any radioactive amino acid not incorporated into the cells was then washed away. The cells were further incubated (the chase) for varying times up to 24 hours, and samples from each time of chase were subjected to immunoprecipitation to isolate one specific protein (here the low-density lipoprotein receptor). SDS-PAGE of the immunoprecipitates followed by autoradiography permitted visualization of the target protein, which is initially synthesized as a small precursor ( $p$ ) and then rapidly modified to a larger mature form $(m)$ by addition of carbohydrates. About half of the labeled protein was converted from $p$ to $m$ during the pulse; the rest was converted after 0.5 hours of chase. The protein remained stable for 6-8 hours before it began to be degraded (as indicated by reduced band intensity). (b) The same experiment was performed in cells in which a mutant form of the protein is made. The mutant $p$ form cannot be properly converted to the $m$ form, and it is more quickly degraded than the normal protein. [© Kozarsky et al., The Journal of Cell Biology. 102: 1567-1575. doi:10.1083/ jcb.102.5.1567]
the labeled protein, using either film or a two-dimensional electronic detector. An example is shown in the experiment described below (Figure 3-42). This method is particularly useful for weak $\beta$ emitters such as ${ }^{3} \mathrm{H}$.

Pulse-chase experiments are particularly useful for tracing changes in the intracellular location of proteins or the modification of a protein or metabolite over time. In this experimental protocol, a cell sample is exposed to a radiolabeled compound that can be incorporated into or otherwise attached to a cellular molecule of interest-the "pulse"for a brief period. The pulse ends when the unincorporated radiolabeled molecules are washed away and the cells are exposed to a vast excess of the identical, but unlabeled, compound to dilute the radioactivity of any remaining, but unincorporated, radiolabeled compound. This procedure prevents any incorporation of significant amounts of radiolabel after the "pulse" period and initiates the "chase" period (see Figure 3-42). Samples taken periodically during the chase period are assayed to determine the location or chemical form of the radiolabel as a function of time. Pulse-chase

experiments in which the radiolabeled protein is detected by autoradiography after immunoprecipitation and SDS-PAGE are often used to follow the rate of synthesis, modification, and degradation of proteins. In these experiments, radiolabeled amino acid precursors are added during the pulse, and the amounts and characteristics of the radiolabeled target protein are detected during the chase. One can thus observe postsynthetic modifications of the protein, such as the covalent addition of sugars (see Chapters 13 and 14) or proteolytic cleavage, that change its electrophoretic mobility, as well as the rate of degradation of the protein, which is detected as the loss of signal with increasing time of chase. A classic use of the pulse-chase technique with autoradiography was in studies that elucidated the pathway traversed by secreted proteins from their site of synthesis in the endoplasmic reticulum to the cell surface (see Chapter 14).

## Mass Spectrometry Can Determine the Mass and Sequence of Proteins

Mass spectrometry (MS) is a powerful technique for characterizing proteins, especially for determining the mass of a protein or fragments of a protein. With such information in hand, it is also possible to determine part or all of the protein's sequence. This method permits the accurate direct determination of the ratio of the mass $(m)$ of a charged molecule (molecular ion) to its charge ( $z$ ), or $m / z$. Additional techniques are then used to deduce the absolute mass of the molecular ion.

All mass spectrometers have four key features. The first is an ion source, from which charge, usually in the form of protons, is transferred to the peptide or protein molecules under study (ionization). Their conversion to ions occurs in the presence of a high electric field, which then directs the charged molecular ions into the second key component, the mass analyzer. The mass analyzer, which is always in a high vacuum chamber, physically separates the ions on the basis of their differing mass-to-charge $(m / z)$ ratios. The separated ions are subsequently directed to strike a detector, the third key component, which provides a measure of the relative abundances of each of the ions in the sample. The fourth essential component is a computerized data system that is used to calibrate the instrument; to acquire, store, and process the resulting data; and often to direct the instrument to automatically collect additional specific types of data from the sample, based on the initial observations. This type of automated feedback is used for the tandem MS (MS/MS) peptide-sequencing methods described below.

The two most frequently used methods of generating ions of proteins and protein fragments are (1) matrix-assisted laser desorption/ionization (MALDI) and (2) electrospray (ES). In MALDI (Figure 3-43), the peptide or protein sample is mixed with a low-molecular-weight, UV-absorbing organic acid (the matrix) and then dried on a metal target. Energy from a laser ionizes and vaporizes the sample, producing singly charged molecular ions from the constituent molecules. In ES (Figure 3-44a), a sample of peptides or
![li176857775e0](li176857775e0.jpg)

EXPERIMENTAL FIGURE 3-43 Molecular mass can be determined by matrix-assisted laser desorption/ionization time-of-flight (MALDI-TOF) mass spectrometry. In a MALDI-TOF mass spectrometer, pulses of light from a laser ionize a protein or peptide mixture that is absorbed on a metal target (step 1). An electric field in the mass analyzer accelerates the ions in the sample toward the detector (steps $\mathbf{\square}$ and $\mathbf{\square}$ ). The time it takes an ion to reach the detector is proportional to the square root of the mass-to-charge ( $\mathrm{m} / \mathrm{z}$ ) ratio. Among ions having the same charge, the smaller ions move faster (shorter time to the detector). The molecular weight of each ion from the sample is calculated using the time of flight of a standard.
proteins in solution is converted into a fine mist of tiny droplets by spraying through a narrow capillary at atmospheric pressure. The droplets are formed in the presence of a high electric field, which renders them highly charged. The solvent evaporates from the droplets in their short flight ( mm ) to the entrance of the mass spectrometer's mass analyzer, forming multiply charged ions from the peptides and proteins. The gaseous ions are transferred into the mass analyzer region of the MS, where they are then accelerated by electric fields and separated by the mass analyzer on the basis of their $m / z$.

The two most frequently used types of mass analyzers are time-of-flight (TOF) instruments and ion traps. TOF instruments exploit the fact that the time it takes an ion to pass through the length of the mass analyzer before reaching the detector is proportional to the square root of $m / z$ (smaller ions move faster than larger ones with the same charge; see Figure 3-43). In ion-trap analyzers, tunable electric fields are used to capture, or "trap," ions with a specific $m / z$ and to sequentially pass the trapped ions out of the mass analyzer onto the detector (see Figure 3-44a). By varying the electric fields, researchers can examine ions with a wide range of $m / z$ values one by one, producing a mass spectrum, which is a graph of $m / z$ ( $x$ axis) versus relative abundance, determined by the intensity of the signal measured by the detector (y axis) (Figure 3-44b, top panel).

In tandem, or MS/MS, instruments, any given parent ion in the original mass spectrum (see Figure 3-44b, top panel) can be chosen (mass-selected) for further analysis. The chosen ions are transferred into a second chamber in which

![li177857775e0](li177857775e0.jpg)

EXPERIMENTAL FIGURE 3-44 Molecular mass of proteins and peptides can be determined by electrospray ionization iontrap mass spectrometry. (a) Electrospray (ES) ionization converts proteins and peptides in a solution into highly charged gaseous ions by passing the solution through a needle (forming the droplets) that has a high voltage across it (charging the droplets). Evaporation of the solvent produces gaseous ions that enter a mass spectrometer. The ions are analyzed by an ion-trap mass analyzer that then directs ions to the detector. (b) Top panel: Mass spectrum of a mixture of three major and several minor peptides from the mouse H-2 class I histocompatibility antigen Q10 $\alpha$ chain is presented as the relative abundance of the ions striking the detector ( $y$ axis) as a function of the mass-to-charge ( $\mathrm{m} / \mathrm{z}$ ) ratio (x axis). Bottom panel: In an MS/MS instrument such as the ion trap
shown in part (a), a specific peptide ion can be selected for fragmentation into smaller ions that are then analyzed and detected. The MS/ MS spectrum (also called the product-ion spectrum) provides detailed structural information about the parent ion, including sequence information for peptides. Here the ion with an $m / z$ of 836.47 was selected and fragmented and the $m / z$ mass spectrum of the product ions measured. Note there is no longer an ion with an $m / z$ of 836.47 present because it was fragmented. From the varying sizes of the product ions, the understanding that peptide bonds are often broken in such experiments, the known $m / z$ values for individual amino acid fragments, and database information, the sequence of the peptide, FIIVGYVDDTQFVR, can be deduced. [Part (b), unpublished data from S. Carr.]

they are broken into smaller fragment ions by collision with an inert gas, and then the $\mathrm{m} / \mathrm{z}$ and relative abundances of the resulting fragment ions are measured in a second MS analyzer (Figure 3-44b, bottom panel, see also Figure 3-47 later in this chapter). These multiple mass analysis and fragmentation steps all take place within the same machine in about 0.1 seconds per selected parent ion. The fragmentation and subsequent mass analysis permit the sequences of short peptides ( $<25$ amino acids) to be determined because collisional fragmentation occurs primarily at peptide bonds, so the differences in masses between the multiple ion fragments generated correspond to the in-chain masses of the individual amino acids, permitting deduction of the sequence in conjunction with database sequence information (see Figure 3-44b, bottom panel).

Mass spectrometry is highly sensitive, able to detect as little as $1 \times 10^{-16} \mathrm{~mol}$ ( 100 attomoles) of a peptide or $10 \times 10^{-15} \mathrm{~mol}$ ( 10 femtomoles) of a protein of 200,000 MW. Errors in mass measurement accuracy are dependent on the specific mass analyzer used, but are typically about 0.01 percent for peptides and $0.05-0.1$ percent for proteins. As described in Section 3.6 below, it is possible to use MS to analyze complex mixtures of proteins as well as purified proteins. MS can readily distinguish between two chemically identical peptides that differ only in that one of the peptides contains "heavy" stable (nonradioactive) isotopic forms of one or more elements (e.g., the isotopes ${ }^{2} \mathrm{H},{ }^{13} \mathrm{C}$, ${ }^{15} \mathrm{~N}$ ) whereas the other contains the most common, "light" isotopes (e.g., ${ }^{1} \mathrm{H},{ }^{12} \mathrm{C},{ }^{14} \mathrm{~N}$ ) because the masses of these peptides differ. Most commonly, protein samples are digested by proteases and the peptide digestion products are subjected to analysis. An especially powerfully application of MS is to take a complex mixture of proteins from a biological specimen, digest it with trypsin or other proteases, partially separate the components using liquid chromatography, and then transfer the solution flowing out of the chromatographic column directly into an ES tandem mass spectrometer. This technique, called $L C-M S / M S$, which permits the nearly continuous analysis of a very complex mixture of proteins, will be described in more detail below.

The abundances of ions determined by mass spectrometry in any given sample are relative, not absolute, values. Therefore, if one wants to use MS to compare the amounts of a particular protein in two different samples (e.g., from a normal versus a mutant organism), it is necessary to have an internal standard in the samples-a molecule whose amounts do not differ between the two samples. One then determines the amounts of the protein of interest relative to that of the standard in each sample. This approach permits quantitatively accurate inter-sample comparisons of protein levels. An alternative approach involves simultaneously comparing in a single MS analysis the amounts of proteins from two different cell or tissue samples that are mixed together. This mixing approach is possible provided the proteins in one of the samples contain different stable isotopes than those in the other. The masses of the otherwise chemically identical peptides from the two samples
will then differ (heavy vs. light) and can thus be distinguished by MS. Several methods can be used to chemically or enzymatically incorporate heavy or light isotopes into proteins isolated from cells for such analysis. Alternatively, cells or organisms can first be grown in the presence of amino acids containing either "heavy" or "light" isotope atoms so that these amino acids are biosynthetically incorporated into all the proteins of that sample. Cells are typically incubated with the heavy or light amino acids for five or more cell divisions to ensure that all proteins are thoroughly labeled. Proteins from the two samples are then mixed together, digested into peptides, and the peptides analyzed by mass spectrometry. Proteins and peptides derived from the "heavy" sample can be distinguished in the mass spectrometer from those from the other, "light," sample because of their higher masses. Thus a direct comparison can be made of the relative amounts of the equivalent proteins in each sample-for example, in tumor versus normal cells or in cells treated with and without a drug. When the samples are cells grown in the laboratory, the method is called stable isotope labeling with amino acids in cell culture (SILAC).

## Protein Primary Structure Can Be Determined by Chemical Methods and from Gene Sequences

The classic method for determining the amino acid sequence of a protein is Edman degradation. In this procedure, the free amino group of the N -terminal amino acid of a polypeptide is labeled, and the labeled amino acid is then cleaved from the polypeptide and identified by high-pressure liquid chromatography. The polypeptide is left one residue shorter, with a new amino acid at the N -terminus. The cycle is repeated on the ever-shortening polypeptide until all the residues have been identified.

Before about 1985, biologists commonly used Edman degradation for determining protein sequences. Now, however, complete protein sequences usually are determined primarily by analysis of genome and messenger RNA sequences. The complete genomes of many organisms have already been sequenced, and the database of genome sequences from humans and numerous model organisms is expanding rapidly. As discussed in Chapter 6, the sequences of proteins can be deduced from DNA sequences that are predicted to encode proteins.

A powerful approach for determining the primary structure of an isolated protein combines MS and the use of sequence databases. First, the peptide mass fingerprint of the protein is obtained by MS. A peptide mass fingerprint is the list of the molecular weights of peptides that are generated from the protein by digestion with a specific protease, such as trypsin. The molecular weights of the parent protein and its proteolytic fragments are then used to search genome databases for any similar-sized protein with identical or similar peptide mass fingerprints. Mass spectrometry can also be used to directly sequence peptides using MS/MS, as described above.

## Protein Conformation Is Determined by Sophisticated Physical Methods

In this chapter, we have emphasized that protein function is dependent on protein structure. Thus, to figure out exactly how a protein works, its three-dimensional structure must be determined. Determining a protein's conformation requires sophisticated physical methods and complex analyses of the experimental data. Here we briefly describe three methods used to generate three-dimensional models of proteins.

X-ray Crystallography The use of x-ray crystallography to determine the three-dimensional structures of proteins was pioneered by Max Perutz and John Kendrew in the 1950s. In this technique, beams of x-rays are passed through a protein crystal, in which millions of protein molecules are precisely aligned with one another in a rigid crystalline array. The wavelengths of x-rays are about $0.1-0.2 \mathrm{~nm}$, short enough to determine the positions of individual atoms in the protein. The electrons in the atoms of the crystal scatter the x-rays, which produce a diffraction pattern of discrete spots when they are intercepted by photographic film or an electronic detector (Figure 3-45). Such patterns are extremely complex-composed of as many as 25,000 diffraction spots, or reflections, whose measured intensities vary depending on the distribution of the electrons in the sample, which is, in turn, determined by the atomic structure and three-dimensional conformation of the protein. Elaborate calculations and modifications of the protein (such as the binding of heavy metals) must be made to interpret the diffraction pattern and calculate the distribution of electrons (called the electron density map). A portion of an electron density map of a protein can be seen in Figure 2-9. With the three-dimensional electron density map in hand, one then "fits" a molecular model of the protein to match the electron density, and it is these models that one sees in the various diagrams of proteins throughout this book (e.g., Figure 3-9). The process is analogous to reconstructing the precise shape of a rock from the ripples that it creates when thrown into a pond. Although sometimes the structures of parts of the protein cannot be clearly defined, using x-ray crystallography, researchers are systematically determining the structures of representative types of most proteins. To date, more than 90,000 detailed three-dimensional structures, including more than 35,000 distinct protein sequences, have been established using x-ray crystallography. These structures can be found in the Research Collaboratory for Structural Bioinformatics Protein Data Bank (http://www.rcsb.org/pdb/home/home. do), each with its own "PDB" entry.

Cryoelectron Microscopy Although some proteins readily crystallize, obtaining crystals of others-particularly large multisubunit proteins and membrane-associated proteinsrequires a time-consuming, often robot-assisted trial-anderror effort to find just the right conditions, if they can be found at all. (Growing crystals suitable for structural studies is as much an art as a science.) There are several ways to
(a)
![li178857775e0](li178857775e0.jpg)
(b)
![li179857775e0](li179857775e0.jpg)

EXPERIMENTAL FIGURE 3-45 X-ray crystallography provides diffraction data from which the three-dimensional structure of a protein can be determined. (a) Basic components of an x-ray crystallographic determination. When a narrow beam of x-rays strikes a crystal, part of it passes straight through and the rest is scattered (diffracted) in various directions. The intensity of the diffracted waves, which form periodic arrangements of diffraction spots, is recorded on an x-ray film or with a solid-state electronic detector. (b) X-ray diffraction pattern for a protein crystal collected on a solid-state detector. From complex analyses of patterns of spots like this one, the locations of the atoms in a protein can be determined. See J. M. Berg, J. L. Tymoczko, G. J. Gatto, and L. Stryer, 2015, Biochemistry, 8th ed., Macmillan. [Part (b) courtesy James M. Berger]
determine the structures of such difficult-to-crystallize proteins. One is cryoelectron microscopy (Figure 3-46). In this technique, a dilute protein sample in an aqueous solution is applied in a thin layer to an electron microscope sample holder (a "grid") and rapidly frozen in liquid helium to preserve its structure. It is then examined in the frozen, hydrated state in a cryoelectron microscope. Images of the protein are

![li180857775e0](li180857775e0.jpg)

## EXPERIMENTAL FIGURE 3-46 Cryoelectron microscopy

analysis of the structure of the human mitochondrial ribosome. The mitochondrion is a complex, multifunctional intracellular organelle best known for its ability to synthesize the energy carrier ATP (see Chapter 12). Human mitochondria can synthesize proteins encoded by mitochondrial DNA using large ( 1.7 MDa ), multi-protein (at least 78) and multi-RNA complexes called mitochondrial ribosomes that differ somewhat from cytoplasmic ribosomes. (a) Cryoelectron micrograph of isolated human mitochondrial ribosomes. The low contrast between the ribosomes and the buffer solution makes it difficult to clearly see individual, frozen ribosome particles, which are oriented randomly in the image. (b) Automated image processing of 323,292 individual particles permits their grouping into classes based on orientation and averaging of the images within each class to generate clearer images of the ribosome. (c) Additional computational analysis generates distinct structures, each based on tens of thousands of individual particles (the number of particles analyzed for each structure in thousands [K] is shown beneath each). The structures enclosed in boxes were selected for additional analysis, which produced the two very similar models shown in (d) containing virtually identical large subunits. (e) Color-coded, lowresolution model of the electron density of the large (blue) and small (yellow) subunits. The conformational heterogeneity of the small subunit prevented its high-resolution structure determination from the data shown here. (f) High-magnification view of the experimentally determined electron density (meshwork) from a portion of one of the proteins in the large subunit illustrates how the electron density is used to build the superimposed molecular model of polypeptide chains. In this very small portion of one protein within the large subunit, the side chains of proline (Pro), phenylalanine (Phe), valine (Val), and tyrosine (Tyr) residues are easily seen and demonstrate the power of cryoelectron microscopy to determine protein structures at very high resolutions. (g) Model of the 48 protein subunits (different colors) in the large subunit determined at $3.4 \AA$ resolution. [Republished with permission of American Association for the Advancement of Science, from Brown, A., et al, "Structure of the large ribosomal subunit from human mitochondria." Science, 2014, 346 (6210): 718-722; permission conveyed through Copyright Clearance Center, Inc.]

recorded on a very sensitive camera using a low dose of electrons to prevent radiation-induced damage to the structure. Since the individual proteins are in different orientations in the frozen sample, sophisticated computer algorithms analyze the images to sort them into groups with the same orientation. The average image of each orientation is calculated from images of the thousands of different molecules in each group, and then the computer assembles the average images, each of which show views of the protein from different orientations, to reconstruct the protein's structure in three dimensions. Recent advances in this technology have produced structures in which the polypeptide backbone and amino acid side chains can be discerned. These structures help provide insight into the mechanisms underlying the protein's function. The use of cryoelectron microscopy and other types of electron microscopy for visualizing cell structures is discussed in Chapter 4.

NMR Spectroscopy The three-dimensional structures of small proteins containing as many as 200 amino acids can be studied routinely with nuclear magnetic resonance (NMR) spectroscopy, and specialized approaches can be used to extend the size range to somewhat larger proteins. In this technique, a concentrated protein solution is placed in a magnetic field, and the effects of different radio frequencies on the nuclear spin states of different atoms are measured. The spin state of any atom is influenced by neighboring atoms in adjacent residues, with closely spaced residues having a greater effect than distant residues. From the magnitude of the effect, the distances between residues can be calculated by a triangulation-like process; these distances are then used to generate a model of the three-dimensional structure of the protein. An important distinction between x-ray crystallography and NMR spectroscopy is that the former method directly determines the locations of the atoms, while the latter directly determines the distances between the atoms, from which the structure is deduced.

Although NMR does not require the crystallization of a protein-a definite advantage-this technique is usually limited to proteins smaller than about 50 kDa (although new techniques permit analysis of the dynamics in much larger proteins). However, NMR analysis can provide information about the ability of a protein to adopt a set of closely related, but not exactly identical, conformations and to move between those conformations (protein dynamics). This is a common feature of proteins, which are not absolutely rigid structures, but can "breathe" or exhibit slight variations in the relative positions of their constituent atoms. In some cases, these variations can have functional significance; for example, they may influence how proteins bind to one another. NMR structural analysis has been particularly useful in studying isolated protein domains, which can often be obtained as stable structures and tend to be small enough for this technique. To date, there are more than 10,000 NMR-determined structures available in the Protein Data Bank.

Another powerful approach to studying protein dynamics and protein-protein interactions is hydrogen/deuterium
exchange mass spectrometry (HXMS). When a protein is placed in a deuterated water $\left(\mathrm{D}_{2} \mathrm{O}\right)$ solution, the rate at which deuterium is exchanged for hydrogen in the amides in the peptide bonds depends on the accessibility of an amide to the solvent. Those amides exposed on the protein's surface are highly accessible and exhibit rapid proton/deuterium exchange. Those amides buried in the center of the protein or in a protein-to-protein interface, as well as those participating in hydrogen bonds with other parts of the protein, exhibit slower proton/deuterium exchange rates. A change in protein conformation or binding to other molecules has the potential to alter the rate of hydrogen/deuterium exchange of one or more amides of a protein. MS analysis permits a hypersensitive assay of such conformational changes, allowing the identification of those parts of the protein that directly bind to other molecules or undergo such conformational changes.

## KEY CONCEPTS OF SECTION 3.5

## Purifying, Detecting, and Characterizing Proteins

- Proteins can be separated from other cell components and from one another on the basis of differences in their physical and chemical properties.
- Centrifugation separates proteins on the basis of their rates of sedimentation, which are influenced by their masses and shapes (see Figure 3-37).
- Electrophoresis separates proteins on the basis of their rates of movement in an applied electric field. SDSpolyacrylamide gel electrophoresis (SDS-PAGE) can resolve polypeptide chains differing in molecular weight by 10 percent or less (see Figure 3-38). Two-dimensional gel electrophoresis provides additional resolution by separating proteins first by charge (first dimension) and then by mass (second dimension).
- Liquid chromatography separates proteins on the basis of their rates of movement through a column packed with spherical beads. Proteins differing in mass are resolved on gel filtration columns; those differing in charge, on ionexchange columns; and those differing in ligand-binding properties, on affinity columns (see Figure 3-40).
- Various assays are used to detect and quantify proteins. Some assays use a light-producing reaction to generate a readily detected signal. Other assays produce an amplified colored signal with enzymes and chromogenic substrates.
- Antibodies are powerful reagents used to detect, quantify, and isolate proteins.
- Immunoblotting, also called Western blotting, is a frequently used method to study specific proteins that exploits the high specificity and sensitivity of protein detection by

antibodies and the high-resolution separation of proteins by SDS-PAGE (see Figure 3-41).

- Immunoprecipitation, often abbreviated as IP, permits the separation of a protein of interest from other proteins in a complex mixture using antibodies specific for the protein of interest. The antibodies are used to precipitate their target protein out of solution for subsequent analysis. Molecules tightly bound to the target protein can precipitate with it (co-immunoprecipitation).
- Isotopes, both radioactive and nonradioactive, play a key role in the study of proteins and other biomolecules. They can be incorporated into molecules without changing the chemical composition of the molecule or as add-on tags. They can be used to help detect the synthesis, location, processing, and stability of proteins.
- Autoradiography is a technique for detecting radioactively labeled molecules in cells, tissues, or electrophoretic gels using two-dimensional detectors (photographic emulsion or electronic detectors).
- Pulse-chase experiments can determine the intracellular fate of proteins and other metabolites (see Figure 3-42).
- Mass spectrometry is a very sensitive and highly precise method of detecting, identifying, and characterizing proteins and peptides.
- Three-dimensional structures of proteins are obtained by x-ray crystallography, cryoelectron microscopy, and NMR spectroscopy. X-ray crystallography provides the most detailed structures but requires protein crystallization. Cryoelectron microscopy is most useful for large protein complexes, which are difficult to crystallize. Only relatively small proteins are amenable to NMR three-dimensional structural analysis.


### 3.6 Proteomics

For most of the twentieth century, the study of proteins was restricted primarily to the analysis of individual proteins. For example, one would study an enzyme by determining its enzymatic activity (its substrates, products, rate of reaction, requirement for cofactors, pH , etc.), its structure, and its mechanism of action. In some cases, the relationships between a few enzymes that participate in a metabolic pathway might also be studied. On a broader scale, the localization and activity of an enzyme would be examined in the context of a cell or tissue. The effects of mutations, diseases, or drugs on the expression and activity of the enzyme might also be the subject of investigation. This multipronged approach provided deep insight into the function and mechanisms of action of individual proteins or relatively small numbers of interacting proteins. However, such a one-byone approach to studying proteins does not readily provide a global picture of what is happening in the proteome of a cell, tissue, or entire organism.

## Proteomics Is the Study of All or a Large Subset of Proteins in a Biological System

The advent of genomics (sequencing of genomic DNA and its associated technologies, such as simultaneous analysis of the levels of all mRNAs in cells and tissues) clearly showed that a global, or systems, approach to biology could provide unique and highly valuable insights. Many scientists recognized that a global analysis of the proteins in biological systems had the potential for equally valuable contributions to our understanding. Thus a new field was born-proteomics. Proteomics is the systematic study of the amounts, modifications, interactions, localization, and functions of all or subsets of proteins at the whole-organism, tissue, cellular, and subcellular levels.

A number of broad questions are addressed in proteomic studies:

- In a given sample (whole organism, tissue, cell, subcellular compartment), what fraction of the whole proteome is expressed (i.e., which proteins are present)?
- Of those proteins present in the sample, what are their relative abundances?
- What are the relative amounts of the different splice forms and chemically modified forms (e.g., phosphorylated, methylated, fatty acylated) of the proteins?
- Which proteins are present in large multiprotein complexes, and which proteins are in each complex? What are the functions of these complexes, and how do they interact?
- When the state (e.g., growth rate, stage of cell cycle, differentiation, stress level) of a cell changes, do the proteins in the cell, or those secreted from the cell, change in a characteristic (fingerprint-like) pattern? Which proteins change, and how (relative amounts, modifications, splice forms, etc.)? [Answering these questions requires a form of protein expression profiling that complements the transcriptional ( $m R N A$ ) profiling discussed in Chapter 9.]
- Can such fingerprint-like changes be used for diagnostic purposes? For example, do certain cancers or heart disease cause characteristic changes in blood proteins? Can the proteomic fingerprint help determine if a given cancer is resistant or sensitive to a particular chemotherapeutic drug? [Proteomic fingerprints can also be the starting point for studies of the mechanisms underlying the change of state. Proteins (and other biomolecules) that show changes that are diagnostic of a particular state are called biomarkers.]
- Can changes in the proteome help define targets for drugs or suggest mechanisms by which a drug might induce toxic side effects? (If so, it might be possible to engineer modified versions of the drug with fewer side effects.)

These are just a few of the questions that can be addressed using proteomics. The methods used to answer these questions are as diverse as the questions themselves, and their numbers are growing rapidly.

## Advanced Techniques in Mass Spectrometry Are Critical to Proteomic Analysis

Advances in proteomics technologies (e.g., mass spectrometry) profoundly affect the types of questions that can be practically studied. For many years, two-dimensional gel electrophoresis allowed researchers to separate, display, and characterize complex mixtures of proteins (see Figure 3-39). The spots on a two-dimensional electrophoresis gel could be excised, the protein fragmented by proteolysis (e.g., by trypsin digestion), and the fragments identified by MS. An alternative to this two-dimensional gel electrophoresis method is high-throughput LC-MS/MS. Figure 3-47 outlines the general LC-MS/MS approach, in which a complex mixture of proteins is digested with a protease; the myriad resulting peptides are fractionated by LC into multiple, less complex fractions, which are slowly but continuously injected by electrospray ionization into a tandem mass spectrometer. The fractions are then sequentially subjected to multiple
cycles of MS/MS until the sequences of many of the peptides have been determined and used to identify the proteins in the original biological sample. Detection of a substantial fraction of the proteins in whole cells or tissues currently requires samples containing more than $50 \mu \mathrm{~g}$ of protein, an amount equivalent to the protein content of some 70,000200,000 mammalian cells. Efforts are under way to increase the sensitivity of the method so that eventually one might be able to analyze the proteome of an individual cell.

An example of the use of LC-MS/MS to identify many of the proteins in each organelle is seen in Figure 3-48. Cells from murine (mouse) liver tissue were mechanically broken to release the organelles, and the organelles were partially separated by density-gradient centrifugation. The locations of the organelles in the gradient were determined using immunoblotting with antibodies that recognized previously identified, organelle-specific proteins. Fractions from the gradient were then subjected to LC-MS/MS to identify the proteins in each fraction, and the distributions in the gradient of many
![li181857775e0](li181857775e0.jpg)

EXPERIMENTAL FIGURE 3-47 LC-MS/MS is used to identify the proteins in a complex biological sample. A complex mixture of proteins in a biological sample (e.g., an isolated preparation of Golgi organelles) is digested with a protease. The mixture of resulting peptides is fractionated by liquid chromatography (LC) into multiple, less complex, fractions, which are slowly but continuously injected by
electrospray ionization into a tandem mass spectrometer. The fractions are then sequentially subjected to multiple cycles of MS/MS until the masses and sequences of many of the peptides have been determined and used to identify the proteins in the original biological sample through comparison with protein databases.

![li182857775e0](li182857775e0.jpg)

EXPERIMENTAL FIGURE 3-48 Density-gradient centrifugation and LC-MS/MS can be used to identify many of the proteins in organelles. (a) The cells in liver tissue were mechanically broken to release the organelles, and the organelles were partially separated by density-gradient centrifugation. The locations of the organelles-which were spread out through the gradient and somewhat overlapped with one another-were determined using immunoblotting with antibodies that recognized previously identified, organelle-specific proteins. Fractions from the gradient were
individual proteins were compared with the distributions of the organelles. This strategy permitted the assignment of many individual proteins to one or more organelles (organelle proteome profiling). More recently, a combination of organelle purification, MS, biochemical localization, and computational methods has been used to show that at least a thousand distinct proteins are localized in the mitochondria of humans and mice.

Proteomics methods combined with molecular genetics methods are currently being used to identify all the protein complexes in eukaryotic cells. For example, in the yeast Saccharomyces cerevisiae, approximately 500 complexes, with an average of 4.9 distinct proteins per complex, have been identified. These complexes, in turn, are involved in at least 400 complex-to-complex interactions. Such systematic proteomic studies are providing new insights into the organization of proteins within cells and into how proteins work together to permit cells to live and function.

Phosphoproteomics, the identification and quantification of phosphorylation sites on the proteins in a complex
![li183857775e0](li183857775e0.jpg)
subjected to proteolysis and LC-MS/MS to identify the peptides, and hence the proteins, in each fraction. Comparisons with the locations of the organelles in the gradient (called protein correlation profiling) permitted assignment of many individual proteins to one or more organelles (organelle proteome identification). (b) The hierarchical breakdown of data derived from the procedures in part (a). Note that not all proteins identified could be assigned to organelles and that some proteins were assigned to more than one organelle. [Data from L. J. Foster et al., 2006, Cell 125:187-199.]
mixture, is playing a growing role in the analysis of cell metabolism and regulation. As we have already learned, the reversible phosphorylation of proteins by kinases and phosphatases is a key mechanism for regulating proteins in cells. Phosphoproteomics permits the simultaneous determination of the phosphorylation states of many proteins and thus provides an important tool for analyzing complex cellular regulatory networks. Only a fraction-in some cases, only a small fraction-of a particular protein might be phosphorylated. Thus phosphoproteomic analysis can require 50-100 times more initial cell or tissue sample material (from about 2.5 to more than 20 mg of total cellular protein per sample) than does standard proteomic analysis. As a consequence, investigators usually use affinity chromatography methods with either metal-containing (e.g., $\mathrm{Fe}^{3+}$ or $\mathrm{TiO}_{2}$ ) or antibody-containing columns to separate phosphopeptides from nonphosphorylated peptides (phosphopeptide enrichment) prior to subjecting the phosphopeptides to LC-MS/ MS analysis.

## KEY CONCEPTS OF SECTION 3.6

## Proteomics

- Proteomics is the systematic study of the amounts (and changes in the amounts), modifications, interactions, localization, and functions of all or subsets of all proteins in biological systems at the whole-organism, tissue, cellular, and subcellular levels.
- Proteomics provides insights into the fundamental organization of proteins within cells and how that organization is influenced by the state of the cells (e.g., differentiation into distinct cell types; responses to stress, disease, and drugs).
- A wide variety of methods are used for proteomic analyses, including two-dimensional gel electrophoresis, densitygradient centrifugation, and mass spectrometry (particularly LC-MS/MS).
- Proteomics has helped begin to identify the proteomes of organelles ("organelle proteome profiling") as well as the organization of individual proteins into multiprotein complexes (see Figure 3-48).
- Phosphoproteomics is a specialized application of proteomics that identifies the collection of phosphorylated proteins (phosphoproteome) in cells and characterizes how the level of phosphorylation of these proteins varies as the state of the cells changes.


## LaunchPad

macmillan learning

Visit LaunchPad to access study tools and to learn more about the content in this chapter.

- Perspectives for the Future
- Analyze the Data
- Extended References
- Additional study tools, including videos, animations, and quizzes


## Key Terms

active site 91
allostery 100
$\alpha$ helix 70
amyloid fibril 87
autoradiography 114
$\beta$ sheet 70

| $\beta$ turn 70 |  |
| :--: | :--: |
| chaperone 83 |  |
| conformation 67 |  |
| cooperativity 100 |  |
| domain 76 |  |
| electrophoresis 107 |  |

enzyme 90
homology 80
kinase 103
$K_{\mathrm{m}} 92$
ligand 89
liquid
chromatography 110
peptide bond 69
phosphorylation 103
polypeptide 70
primary structure 70
proteasome 97
protein 70
proteome 68
proteomics 122
quaternary structure 78
rate-zonal
centrifugation 106
secondary structure 70
structural motif 75
tertiary structure 72
ubiquitin 99
$V_{\max } 91$
Western blotting 112
x-ray
crystallography 119

## Review the Concepts

1. The three-dimensional structure of a protein is determined by its primary, secondary, and tertiary structures. Define the primary, secondary, and tertiary structures. What are some of the common secondary structures? What are the forces that hold together the secondary and tertiary structures?
2. Proper folding of proteins is essential for their biological activity. In general, the functional conformation of a protein is the conformation with lowest energy. This means that if an unfolded protein is allowed to reach equilibrium, it should assemble automatically into its native, functioning folded state. Why then is there a need for molecular chaperones and chaperonins in cells? What different roles do molecular chaperones and chaperonins play in the folding of proteins?
3. Enzymes catalyze chemical reactions. What constitutes the active site of an enzyme? What are the turnover number $\left(k_{\text {cat }}\right)$, the Michaelis constant $\left(K_{\mathrm{m}}\right)$, and the maximal velocity $\left(V_{\max }\right)$ of an enzyme? The $k_{\text {cat }}$ (catalytic rate constant) for carbonic anhydrase is $5 \times 10^{5}$ molecules per second. This is a "rate constant," but not a "rate." What is the difference? By what concentration would you multiply this rate constant in order to determine an actual rate of product formation $(V)$ ? Under what circumstances would this rate become equal to the maximal velocity $\left(V_{\max }\right)$ of the enzyme?
4. The following reaction coordinate diagram charts the energy of a substrate molecule $(\mathrm{S})$ as it passes through a transition state $\left(\mathrm{X}^{3}\right)$ on its way to becoming a stable product (P) alone or in the presence of one of two different enzymes (E1 and E2). How does the addition of either enzyme affect the change in Gibbs free energy $(\Delta G)$ for the reaction? Which of the two enzymes binds with greater affinity to the substrate? Which enzyme better stabilizes the transition state? Which enzyme functions as a better catalyst?

![li184857775e0](li184857775e0.jpg)
5. A healthy immune system can raise antibodies that recognize and bind with high affinity to almost any stable molecule. The molecule to which an antibody binds is known as an antigen. Antibodies have been exploited by enterprising scientists to generate valuable tools for research, diagnosis, and therapy. One clever application is the generation of antibodies that function like enzymes to catalyze complicated chemical reactions. If you wished to produce such a "catalytic" antibody, what would you suggest using as the antigen? Should it be the substrate of the reaction? The product? Something else?
6. Proteins are degraded in cells. What is ubiquitin, and what role does it play in tagging proteins for degradation? What is the role of proteasomes in protein degradation? How might proteasome inhibitors serve as chemotherapeutic (cancer-treating) agents?
7. The function of proteins can be regulated in a number of ways. What is cooperativity, and how does it influence protein function? Describe how protein phosphorylation and proteolytic cleavage can modulate protein function.
8. A number of techniques can separate proteins on the basis of their differences in mass. Describe the use of two of these techniques, centrifugation and gel electrophoresis. The blood proteins transferrin (MW 76 kDa ) and lysozyme (MW 15 kDa ) can be separated by rate-zonal centrifugation or SDS-polyacrylamide gel electrophoresis. Which of the two proteins will sediment faster during centrifugation? Which will migrate faster during electrophoresis?
9. Liquid chromatography is an analytical method used to separate proteins. Describe the principles for separating proteins by gel filtration, ion-exchange, and affinity chromatography.
10. Various methods have been developed for detecting proteins. Describe how radioisotopes and autoradiography can be used for labeling and detecting proteins. How does Western blotting detect proteins?
11. Physical methods are often used to determine protein conformation. Describe how x-ray crystallography, cryoelectron microscopy, and NMR spectroscopy can be used to determine the shapes of proteins. What are the advantages
and disadvantages of each method? Which is better for small proteins? Large proteins? Huge macromolecular assemblies? 12. Mass spectrometry is a powerful tool in proteomics. What are the four key features of a mass spectrometer? Describe briefly how MALDI and two-dimensional polyacrylamide gel electrophoresis could be used to identify a protein expressed in cancer cells but not in normal healthy cells.

## References

## Web Sites

Entry site into proteins, structures, genomes, and taxonomy: http://www.ncbi.nlm.nih.gov/Entrez/

The protein 3-D structure database: http://www.rcsb.org/
Structural classifications of proteins: http://scop.berkeley.edu/
Sites containing general information about proteins: http:// www.expasy.ch; http://www.proweb.org/; http://scop.berkeley.edu/ intro.html

PROSITE database of protein families and domains: http:// www.expasy.org/prosite/

Domain organization of proteins and large collection of multiple sequence alignments: http://www.sanger.ac.uk/Software/Pfam/; http://people.cryst.bbk.ac.uk/,ubcg16z/cpn/elmovies.html

MitoCarta: An Inventory of Mammalian Mitochondrial Genes: http://www.broadinstitute.org/pubs/MitoCarta/index.html

Human protein atlas with expression of proteins in different tissues: http://www.proteinatlas.org/

## Hierarchical Structure of Proteins

Dunker, A. K., et al. 2015. Intrinsically disordered proteins and multicellular organisms. Semin. Cell Dev. Biol. 37:44-55.

Levitt, M. 2009. Nature of the protein universe. P. Natl. Acad. Sci. USA 106:11079-11084.

Patthy, L. 1999. Protein Evolution. Blackwell Science.
Vogel, C., and C. Chothia. 2006. Protein family expansions and biological complexity. PLoS Comput. Biol. 2(5):e48.

Yaffe, M. B. 2006. "Bits" and pieces. Sci. STKE 2006:pe28.

## Protein Folding

Brandvold, K. R., and R. I. Morimoto. 2015. The chemical biology of molecular chaperones-implications for modulation of proteostasis. J. Mol. Biol. 427:2931-2947.

Coulson, A. F., and J. Moult. 2002. A unifold, mesofold, and superfold model of protein fold use. Proteins 46:61-71.

Daggett, V, and A. R. Fersht. 2003. Is there a unifying mechanism for protein folding? Trends Biochem. Sci. 28:18-25.

Dobson, C. M. 1999. Protein misfolding, evolution, and disease. Trends Biochem. Sci. 24:329-332.

Jackrel, M. E., et al. 2014. Potentiated Hsp104 variants antagonize diverse proteotoxic misfolding events. Cell 156:170-182.

Knowles, T. P., M. Vendruscolo, and C. M. Dobson. 2014. The amyloid state and its association with protein misfolding diseases. Nat. Rev. Mol. Cell Biol. 15:384-396.

Lavery, L. A., et al. 2014. Structural asymmetry in the closed state of mitochondrial Hsp90 (TRAP1) supports a two-step ATP hydrolysis mechanism. Mol. Cell 53:330-343.

Saibil, H. 2013. Chaperone machines for protein folding, unfolding and disaggregation. Nat. Rev. Mol. Cell Biol. 14:630-642

Schmidpeter, P. A., and F. X. Schmid. 2015. Prolyl isomerization and its catalysis in protein folding and protein function. J. Mol. Biol. 427:1609-1631.

Taipale, M., D. F. Jarosz, and S. Lindquist. 2010. HSP90 at the hub of protein homeostasis: emerging mechanistic insights. Nat. Rev. Mol. Cell Biol. 11:515-528.

Valastyan, J. S., and S. Lindquist. 2014. Mechanisms of proteinfolding diseases at a glance. Dis. Model Mech. 7:9-14.

## Protein Binding and Enzyme Catalysis

Fersht, A. 1999. Enzyme Structure and Mechanism, 3d ed. W. H. Freeman and Company.

Martínez Cuesta, S., et al. 2015. The classification and evolution of enzyme function. Biophys. J. 109:1082-1086.

Radisky, E. S., et al. 2006. Insights into the serine protease mechanism from atomic resolution structures of trypsin reaction intermediates. P. Natl. Acad. Sci. USA 103:6835-6840.

## Regulating Protein Function

Bellelli, A., et al. 2006. The allosteric properties of hemoglobin: insights from natural and site directed mutants. Curr. Prot. Pep. Sci. $7: 17-45$.

Campbell, M. G., et al. 2015. 2.8 Å resolution reconstruction of the Thermoplasma acidophilum 20S proteasome using cryo-electron microscopy. eLife. 10.7554/eLife. 06380.

Glickman, M. H., and A. Ciechanover. 2002. The ubiquitinproteasome proteolytic pathway: destruction for the sake of construction. Physiol. Rev. 82:373-428.

Goldberg, A. L, S. J. Elledge, and J. W. Harper. 2001. The cellular chamber of doom. Sci. Am. 284:68-73.

Goldberg, A. L. 2003. Protein degradation and protection against misfolded or damaged proteins. Nature 426:895-899.

Kern, D., and E. R. Zuiderweg. 2003. The role of dynamics in allosteric regulation. Curr. Opin. Struc. Biol. 13:748-757.

Kisselev, A. F., A. Callard, and A. L. Goldberg. 2006. Importance of the different proteolytic sites of the proteasome and the efficacy of inhibitors varies with the protein substrate. J. Biol. Chem. 281:8582-8590.

Lim, W. A. 2002. The modular logic of signaling proteins: building allosteric switches from simple binding domains. Curr. Opin. Struc. Biol. 12:61-68.

Sahtoe, D. D., and T. K. Sixma. 2015. Layers of DUB regulation. Trends Biochem. Sci. 40(8):456-467.

Sowa, M. E., et al. 2009. Defining the human deubiquitinating enzyme interaction landscape. Cell 138:389-403.

## Purifying, Detecting, and Characterizing Proteins

Engen, J. R., et al. 2013. Partial cooperative unfolding in proteins as observed by hydrogen exchange mass spectrometry. Int. Rev. Phys. Chem. 32:96-127.

Hames, B. D. A Practical Approach. Oxford University Press. A methods series that describes protein purification methods and assays.

Liao, M, et al. 2014. Single particle electron cryo-microscopy of a mammalian ion channel. Curr. Opin. Struc. Biol. 27:1-7.

Nogales, E., and S. H. Scheres. 2015. Cryo-EM: a unique tool for the visualization of macromolecular complexity. Mol. Cell 58:677-689.

Rosenzweig, R., and L. E. Kay. 2014. Bringing dynamic molecular machines into focus by methyl-TROSY NMR. Annu. Rev. Biochem. 83:291-315.

Zhang, G., et al. 2014. Overview of peptide and protein analysis by mass spectrometry. Curr. Protoc. Mol. Biol. 108:10.21.110.21.30.

## Proteomics

Azimifar, S. B., et al. 2014. Cell-type-resolved quantitative proteomics of murine liver. Cell Metab. 20:1076-1087.

Calvo, S. E., and V. K. Mootha. 2010. The mitochondrial proteome and human disease. Annu. Rev. Genomics Hum. Genet. 11:25-34.

Cox, J., and M. Mann. 2011. Quantitative, high-resolution proteomics for data-driven systems biology. Annu. Rev. Biochem. 80:273-299.

Foster, L. J., et al. 2006. A mammalian organelle map by protein correlation profiling. Cell 125:187-199.

Krogan, N. J., et al. 2006. Global landscape of protein complexes in the yeast Saccharomyces cerevisiae. Nature 440:637-643.

Rifai, N., M. A. Gillette, and S. A. Carr. 2006. Protein biomarker discovery and validation: the long and uncertain path to clinical utility. Nature Biotechnol. 24:971-983.

Roux, P. P., and P. Thibault. 2013. The coming of age of phosphoproteomics-from large data sets to inference of protein functions. Mol Cell Proteomics:12:3453-3464.

Walther, T. C., and M. Mann. 2010. Mass spectrometry-based proteomics in cell biology. J. Cell Biol. 190:491-500.

this page left intentionally blank

![li185857775e0](li185857775e0.jpg)

Fluorescence microscopy showing the locations of DNA (green), microtubules (yellow), and microfilaments (purple) in two cultured cells. The cells were chemically fixed and then rendered permeable to antibodies using a gentle detergent. Microtubules were stained with an antibody to tubulin; microfilaments were stained with a labeled toxin, phalloidin, that binds selectively to F-actin; and DNA was visualized with a DNAbinding dye. [Torsten Wittman, University of California, San Francisco.]

It is difficult to believe that just 200 years ago it was not yet appreciated that all living things are made of cells. In 1655, Robert Hooke used a primitive microscope to examine a piece of cork and saw an orderly arrangement of rectangles-the walls of the dead plant cells-that reminded him of monks' cells in a monastery, so he coined the term cell. Shortly after this, Antoni van Leeuwenhoek's observations of the microorganisms he saw in his simple microscope became the first description of live cells. Two hundred years later, Matthias Schleiden and Theodore Schwann proposed that cells constitute the fundamental unit of life in plants, animals, and single-celled organisms. Collectively, these discoveries were some of the greatest in biology and posed the question of how cells are organized and function.

Even today, many technical constraints hamper studies of cells in intact animals and plants. One alternative is the use of intact organs that are removed from animals and treated

## 4

## Culturing and Visualizing Cells

to maintain their physiological integrity and function. However, the organization of organs, even isolated ones, is sufficiently complex to pose numerous problems for research. Thus molecular cell biologists often conduct experimental studies on cells isolated from an organism. In Section 4.1, we learn how to maintain and grow diverse cell types and how to isolate specific types of cells from complex mixtures.

In many cases, isolated cells can be maintained in the laboratory under conditions that permit their survival and growth, a procedure known as culturing. Cultured cells have several advantages over intact organisms for cell biology research. Cells of a single specific type can be grown in culture, experimental conditions can be better controlled, and in many cases a single cell can be readily grown into a colony of many identical cells. The resulting strain of cells, which is genetically homogeneous, is called a clone. However, cultured cells are not in their native setting, so researchers are

### 4.1 Growing and Studying Cells in Culture

### 4.2 Light Microscopy: Exploring Cell Structure and Visualizing Proteins Within Cells

### 4.3 Electron Microscopy: High-Resolution Imaging

### 4.4 Isolation of Cell Organelles

now growing and examining cells in three-dimensional environments to more closely mimic their situation in an animal.

Discoveries about cellular organization have been intimately tied to developments in microscopy. This is as true today as it was 400 years ago. Light microscopy initially revealed the beautiful internal organization of cells, and today highly sophisticated microscopes are continually being improved to probe deeper and deeper into the molecular mechanisms by which cells function. In Section 4.2, we discuss light microscopy and the long-standing but still valuable techniques that are available, and then examine several clever methods that have been developed more recently, culminating with the newest, cutting-edge technologies. A major advance came in the 1960s and 1970s with the development of immunofluorescence microscopy, which allows the localization of specific proteins within fixed cells, thus providing a static image of their location, as illustrated in the chapter-opening figure. Such studies led to the important understanding that the membranes and interior spaces of each type of organelle contain a distinctive group of proteins that are necessary for the organelle to carry out its unique functions. Another major advance came in the mid-1990s with the simple idea of expressing chimeric proteins-consisting of a protein of interest covalently linked to a naturally fluorescent protein-which enabled biologists to visualize the movements of individual proteins in live cells. Suddenly, the dynamic nature of cells could be appreciated, which changed the view of cells from the previously available static images. In addition, it presented a technological challenge: the more sensitive a microscope could be made to detect the fluorescent protein, the more information the investigator could glean from the data. It also opened up the development of fluorescent techniques to monitor protein-protein interactions in live cells, as well as a myriad of other sophisticated molecular technologies, some of which we also discuss in this section. For decades, light microscopy was constrained by the resolution of the light microscope-to about 200 nmdue to the limitations imposed by the wavelength of visible light. We discuss methods that have been developed over the last few years to "beat" this resolution barrier with the development of super-resolution microscopy.

Despite the amazing developments in light microscopy, visible light still provides too low a resolution to examine cells in fine ultrastructural detail. The electron microscope gives a much higher resolution, but the technology generally requires that the cell be fixed and sectioned, and therefore all cell movements are frozen in time. Electron microscopy also allows investigators to examine the structure of macromolecular complexes or single macromolecules. In Section 4.3, we outline the various approaches for preparing specimens for observation in the electron microscope and describe the types of information that can be derived from them.

Light and electron microscopy revealed that all eukaryotic cells-whether of fungal, plant, or animal origin—contain a similar repertoire of membrane-limited compartments termed organelles. In parallel with the developments in microscopy,
subcellular fractionation methods were developed that have enabled cell biologists to isolate individual organelles to a high degree of purity. These techniques, detailed in Section 4.4, continue to provide important information about the protein composition and biochemical function of organelles.

### 4.1 Growing and Studying Cells in Culture

The study of cells is greatly facilitated by growing them in culture, so that they can be examined by microscopy and subjected to specific treatments under controlled conditions. It is generally quite easy to grow unicellular bacteria, fungi, or protists, for example, by placing them in a rich medium that supports their growth. However, animal cells come from multicellular organisms, which makes it more difficult to culture single cells or small groups of cells. In this section, we discuss how animal cells are grown in culture and how different cell types can be purified for study.

## Culture of Animal Cells Requires Nutrient-Rich Media and Special Solid Surfaces

To permit the survival and normal function of cultured tissues or cells, the temperature, pH , ionic strength, and access to essential nutrients must simulate as closely as possible the conditions within an intact organism. Isolated animal cells are typically placed in a nutrient-rich liquid, called the culture medium, within specially coated plastic dishes or flasks. The cultures are kept in incubators in which the temperature, atmosphere, and humidity can be controlled. To reduce the chances of bacterial or fungal contamination, antibiotics are often added to the culture medium. To further guard against contamination, investigators usually transfer cells between dishes, add reagents to the culture medium, and otherwise manipulate the specimens within special sterile cabinets containing circulating air that is filtered to remove microorganisms and other airborne contaminants.

Media for culturing animal cells must supply the nine amino acids (phenylalanine, valine, threonine, tryptophan, isoleucine, methionine, leucine, lysine, and histidine) that cannot be synthesized by adult vertebrate animal cells. In addition, most cultured cells require three other amino acids (cysteine, tyrosine, and arginine) that are synthesized only by specialized cells in intact animals, as well as glutamine, which serves as a nitrogen source. The other necessary components of a medium for culturing animal cells are vitamins, various salts, fatty acids, glucose, and serum-the fluid remaining after the noncellular part of blood (plasma) has been allowed to clot. Serum contains various protein factors that are needed for the proliferation of mammalian cells in culture, including the polypeptide hormone insulin; transferrin, which supplies iron in a bioaccessible form; and numerous growth factors. In addition, certain cell types require specialized protein growth factors not present in

serum. For instance, progenitors of red blood cells require erythropoietin, and T lymphocytes require interleukin 2 (see Chapter 16). A few mammalian cell types can be grown in a chemically defined, serum-free medium containing amino acids, glucose, vitamins, and salts plus certain trace minerals, specific protein growth factors, and other components.

Unlike bacterial and yeast cells, which can be grown in suspension, most animal cell types will grow only when attached to a solid surface. This requirement highlights the importance of the cell-surface proteins, called cell-adhesion molecules (CAMs), that cells use to bind to adjacent cells and to components of the extracellular matrix such as collagen, laminin, or fibronectin (see Chapter 20). The solid growth surface (usually glass or plastic) is either pre-coated with these extracellular-matrix proteins, or they come from the serum or are secreted by the cells in culture. A single cell cultured on a glass or plastic dish proliferates to form a visible mass, or colony, containing thousands of genetically identical cells in 4 to 14 days, depending on the growth rate. Although most normal animal cells require a surface to grow on, some specialized blood cells, and especially tumor cells, can be grown in suspension as single cells.

## Primary Cell Cultures and Cell Strains Have a Finite Life Span

Primary cells are cells isolated directly from tissues. Normal animal tissues (e.g., skin, kidney, liver) or whole embryos are commonly used to establish primary cell cultures. To prepare individual tissue cells for a primary culture, the cell-cell and cell-matrix interactions must be broken. To do so, tissue fragments are treated with a combination of a protease (e.g., trypsin, the collagen-hydrolyzing enzyme collagenase, or both) and a divalent cation chelator (e.g., EDTA) that depletes the medium of free calcium $\left(\mathrm{Ca}^{2+}\right)$. Many CAMs require calcium and are thus inactivated when calcium is removed; other CAMs that are not calcium dependent need to be cleaved by a protease for the cells to separate. The released cells are then placed in a nutrient-rich, serum-supplemented medium in dishes, where they can adhere to the surface and to one another. The same protease-chelator solution is used to remove adherent cells from a culture dish for biochemical studies or subculturing (transfer to another dish).

Fibroblasts are the predominant cells in connective tissue and normally produce extracellular-matrix components such as collagen that bind to CAMs, thereby anchoring cells to a surface. In culture, fibroblasts usually divide more rapidly than other cells from a tissue, eventually becoming the predominant cell type in a primary culture unless special precautions are taken to remove them when isolating other types of cells.

When cells removed from an embryo or an adult animal are cultured, most of the adherent cells will divide a finite number of times and then cease growing (a phenomenon called cell senescence). For instance, human fetal fibroblasts divide about 50 times before they cease growth (Figure 4-1a). Starting with $10^{6}$ cells, 50 doublings has the potential to
produce $10^{6} \times 2^{50}$, or more than $10^{20}$, cells, whose weight would be equivalent to that of about a thousand people. Normally, only a very small fraction of these cells are used in any one experiment. Thus, even though its lifetime is limited, a single culture, if carefully maintained, can be studied through many cell generations. Such a lineage of cells originating from one initial primary culture is called a cell strain.

One important exception to the finite life span of normal cells is the embryonic stem cell, which, as its name implies, is derived from an embryo and will divide and give rise to all tissues during development. As we discuss in Chapter 21, embryonic stems cells can be cultured indefinitely under the appropriate conditions.
![li186857775e0](li186857775e0.jpg)
(b) Mouse cells
![li187857775e0](li187857775e0.jpg)

FIGURE 4-1 Stages in the establishment of a cell culture.
(a) When cells isolated from human tissue are initially cultured, some cells die and others (mainly fibroblasts) start to grow; overall, the growth rate increases (phase I). If the remaining cells are harvested, diluted, and replated into dishes again and again, the cell strain continues to divide at a constant rate for about 50 cell generations (phase II), after which the growth rate falls rapidly. In the ensuing period (phase III), all the cells in the culture stop growing (senescence). (b) In a culture prepared from mouse or other rodent cells, initial cell death (not shown) is coupled with the emergence of healthy, growing cells. As these dividing cells are diluted and allowed to continue growth, they soon begin to lose growth potential, and most stop growing (i.e., the culture goes into senescence). Very rare cells undergo oncogenic mutations that allow them to survive and continue dividing until their progeny overgrow the culture. These cells constitute a cell line, which will grow indefinitely if it is appropriately diluted and fed with nutrients. Such cells are said to be immortal.

Research with cell strains is simplified by our ability to freeze them and successfully thaw them at a later time for experimental analysis. Cell strains can be frozen in a state of suspended animation and stored for extended periods at liquid nitrogen temperature, provided that a preservative that prevents the formation of damaging ice crystals is used. Although not all cells survive thawing, many do survive and resume growth.

## Transformed Cells Can Grow Indefinitely in Culture

To be able to clone individual cells, modify cell behavior, or select mutants, biologists often want to maintain cell cultures for many more than 50 doublings. Such prolonged growth is exhibited by cells derived from some tumors. In addition, rare cells in a population of primary cells may undergo spontaneous oncogenic mutations, leading to oncogenic transformation (see Chapter 24). Such cells, said to be oncogenically transformed, or simply transformed, are able to grow indefinitely. A culture of cells with an indefinite life span is considered immortal and is called a cell line.

Primary cultures of normal rodent cells commonly undergo spontaneous transformation into a cell line. After rodent cells are grown in culture for several generations, the culture goes into senescence (Figure 4-1b). During this period, most of the cells stop growing, but often a rapidly dividing transformed cell arises spontaneously and takes over, or overgrows, the culture. A cell line derived from such a transformed variant will grow indefinitely if provided with the necessary nutrients. In contrast to rodent cells, normal human cells rarely undergo spontaneous transformation into a cell line. The HeLa cell line, the first human cell line established, was originally obtained in 1952 from a malignant tumor (carcinoma) of the uterine cervix and is still used extensively today. Many other human cell lines are derived from cancers, and biologists have rendered others immortal by transforming them to express oncogenes.

Regardless of their source, cells in immortal lines often have chromosomes with abnormal DNA sequences. In addition, the number of chromosomes in such cells is usually greater than that in the normal cell from which they arose, and the chromosome number changes as the cells continue to divide in culture. A noteworthy exception, and exciting development, is a recently described human cell line of hematopoietic origin that is haploid for all chromosomes except chromosome 8. Since inactivation of one of the two copies of a gene in a diploid cell generally does not generate a phenotype, a line with a single copy of most genes should be very useful for genetic analysis, making possible the types of genetic screens employed in model organisms (see Chapter 6). Cells with an abnormal number of chromosomes are said to be aneuploid.

## Flow Cytometry Separates Different Cell Types

Some cell types differ sufficiently in density that they can be separated on the basis of this physical property. White blood cells (leukocytes) and red blood cells (erythrocytes),
for instance, have very different densities because erythrocytes have no nucleus; thus these cells can be separated by equilibrium density-gradient centrifugation (described in Section 4.4). Most cell types cannot be differentiated so easily, so other techniques, such as flow cytometry, must be used to separate them.

To separate one type of cell from a complex mixture, it is necessary to have some way to mark and then sort out the desired cells. As we will see below, it is possible to mark cells by expressing a fluorescent protein in them, but if only a few cells in the population express the protein, how can we sort them from the nonfluorescent ones? The cells can be analyzed in a flow cytometer. This machine flows cells past a laser beam that measures the light that they scatter and the fluorescence that they emit; thus it can quantify the cells expressing the fluorescent protein in a mixture. A fluorescence-activated cell sorter (FACS), which is based on flow cytometry, can both analyze the cells and select the few fluorescent cells from thousands of others and sort them into a separate culture dish (Figure 4-2). To achieve this, the cells are mixed with a buffer and forced through a vibrating nozzle to generate tiny droplets. The concentration of cells is adjusted so that most of the droplets do not contain cells, and the ones that do contain only one. Just before the nozzle, the stream of cells passes through a laser beam so that the presence and size of a cell can be recorded from the scattered light using one detector, and the amount of fluorescent light emitted can be quantified using a second, fluorescent light detector. If a cell is present in a droplet, the droplet is given a negative electric charge as it emerges from the nozzle. The stream of droplets then passes through two plates that generate an electric field proportional to the fluorescence detected from the cell in the droplet. This field generates a force that moves charged droplets out of the stream of uncharged droplets and into a collection tube. Since the amount of force applied is proportional to the fluorescence emitted by the cell in the droplet, cells with different levels of fluorescence can be collected. Having been sorted from other cells, the selected cells can be grown in culture.

The FACS procedure is commonly used to purify the different types of white blood cells, each of which bears on its surface one or more distinctive proteins and so will bind monoclonal antibodies specific for its proteins. If a cell mixture is incubated with a fluorescent dye linked to the antibody to a specific cell-surface protein, only the desired cells will be fluorescent. Only the T cells of the immune system, for instance, have both CD3 and Thy1.2 proteins on their surfaces. The presence of these surface proteins allows T cells to be separated easily from other types of blood cells or spleen cells (Figure 4-3).

Other uses of flow cytometry include the measurement of a cell's DNA and RNA content and the determination of its general shape and size. The FACS can make simultaneous measurements of the size of a cell (from the amount of scattered light) and the amount of DNA that it contains (from the amount of fluorescence emitted from a DNA-binding dye). Measurements of the DNA content of individual cells

![li188857775e0](li188857775e0.jpg)

FIGURE 4-2 A fluorescence-activated cell sorter (FACS) separates cells having different levels of fluorescence. Step A concentrated suspension of labeled cells is mixed with a buffer (the sheath fluid) so that the cells pass single file through a laser light beam. Step $\mathbf{B}$ Both the fluorescent light emitted and the light scattered by each cell are measured; from measurements of the scattered light, the size and shape of the cell can be determined. Step $\mathbf{B}$ The suspension is then forced through a nozzle, which forms tiny droplets containing at most a single cell. At the time of formation at the nozzle tip, each droplet containing a cell is given a negative electric charge proportional to the fluorescence of that cell determined from the earlier measurement. Step $\mathbf{B}$ Droplets now pass through an electric field, so that those with no charge are discarded, whereas those with different electric charges are separated and collected. Because it takes only milliseconds to sort each droplet, as many as 10 million cells per hour can pass through the machine.
are used to follow replication of DNA as the cells progress through the cell cycle (see Chapter 19).

An alternative method for separating specific types of cells uses small magnetic beads coupled to antibodies to a specific cell-surface molecule. For example, to isolate T cells, the beads are coated with a monoclonal antibody specific for a surface protein such as CD3 or Thy1.2. Only cells with these proteins will stick to the beads, which can be recovered from the preparation by adhesion to a small magnet on the side of a test tube.

## Growth of Cells in Two-Dimensional and Three-Dimensional Culture Mimics the In Vivo Environment

While much has been learned using cells grown on plastic or glass surfaces, these surfaces are far removed from cells' normal tissue environment. As detailed in Chapter 20, many cell types function only when closely linked to other cells. Key examples are the sheet-like layers of epithelial tissue, called epithelia (singular, epithelium), that cover the external
![li189857775e0](li189857775e0.jpg)

EXPERIMENTAL FIGURE 4-3 T cells bound to fluorescencetagged antibodies to two cell-surface proteins are separated from other white blood cells by FACS. Spleen cells from a mouse were treated with a red fluorescent monoclonal antibody specific for the CD3 cell-surface protein and with a green fluorescent monoclonal antibody specific for a second cell-surface protein, Thy1.2. As the cells were passed through a FACS, the intensity of the green and red fluorescence emitted by each cell was recorded. Each dot represents a single cell. This plot of green fluorescence (vertical axis) versus red fluorescence (horizontal axis) for thousands of spleen cells shows that about half of them-the T cells-express both CD3 and Thy1.2 proteins on their surfaces (upper-right quadrant). The remaining cells, which exhibit low fluorescence (lower-left quadrant), express only background levels of these proteins and are other types of white blood cells. Note the logarithmic scale on both axes. [Data from Chengcheng Zhang, Whitehead Institute.]

![li190857775e0](li190857775e0.jpg)

FIGURE 4-4 Madin-Darby canine kidney (MDCK) cells grown in specialized containers provide a useful experimental system for studying epithelial cells. MDCK cells form a polarized epithelium when grown on a porous membrane filter coated on one side with collagen and other components of the basal lamina. With the use of the special culture dish shown here, the medium on each side of the filter (apical and basal sides of the monolayer) can be experimentally manipulated and the movement of molecules across the layer monitored. Several cell junctions that interconnect the cells form only if the growth medium contains sufficient calcium.
and internal surfaces of organs. Typically, epithelial cells have distinct surfaces, called the apical (top), basal (base or bottom), and lateral (side) surfaces (see Figure 20-11).
(a)
![li191857775e0](li191857775e0.jpg)

EXPERIMENTAL FIGURE 4-5 MDCK cells can form cysts in culture. (a) MDCK cells grown on a supported extracellular matrix will form groups of cells that polarize to form a tubular single layer of cells with a lumen in the middle, called a cyst. (b) By examining the localization of proteins found in the apical (red) and basolateral membranes

The basal surface usually contacts an underlying extracellular matrix called the basal lamina, whose composition and function are discussed in Section 20.3. Epithelial cells often function to transport specific classes of molecules across the epithelial sheet; for example, the epithelial lining of the intestine transports nutrients into the cell through the apical surface and out toward the bloodstream across the basolateral surface. When grown on plastic or glass, epithelial cells cannot easily perform this function. Therefore, special containers have been designed with a porous surface that acts as a basal lamina to which epithelial cells attach and form a uniform two-dimensional sheet (Figure 4-4). A commonly used cultured cell line derived from dog kidney epithelium, called the Madin-Darby canine kidney (MDCK) cell line, is often used to study the formation and function of epithelial sheets.

However, even a two-dimensional sheet often does not allow cells to fully mimic behavior in their normal environment. Methods have now been developed to grow cells in three dimensions by providing a support infiltrated with appropriate components of the extracellular matrix (discussed in Chapter 20). If MDCK cells are cultured under appropriate conditions, they will form a tubular sheet mimicking a tubular organ or the duct of a secretory gland. In these three-dimensional structures, the apical side of the epithelial sheet lines the lumen, whereas the basal side of each cell is in contact with the extracellular matrix (Figure 4-5).
(b)
![li192857775e0](li192857775e0.jpg)
(green), we can see that these cells are fully polarized, with the apical side facing the lumen, which recapitulates their organization in the kidney tubules from which they are derived. The nuclear DNA is stained blue. [Institute of Cancer Sciences/CRUK Beatson Institute, University of Glasgow, Garscube Estate, Switchback Road.]

5. If we can grow an epithelial tube in culture, can we grow a whole organ that could be transplanted into a patient? Recent advances in biomedical engineering are developing promising strategies to do this, initially in experimental animals. In an example of one approach, a 3-D printer is used to help make a replacement ear. First, an exact computer image of an ear is generated. This image is used to program a 3-D printer to assemble a pliable matrixcontaining support material that is biodegradable, together with appropriate components of the extracellular matrix-in the precise shape of an ear. This matrix provides support for the growth of skin cells, either in culture or after transplantation under the skin, so that ultimately the synthetic organ can be surgically attached to a living animal. Other approaches make use of 3-D printers to assemble the matrix and seed it with appropriate cells. An exciting and ambitious goal of this technology is to generate synthetic organs containing many different types of cells by printing each of several layers with appropriate matrix and cells to generate complex three-dimensional organs that might one day be used to replace defective ones in patients. Many hurdles still need to be overcome, but the ability to generate stem cells from patients and then induce differentiation in culture (described in Chapter 21) is overcoming the major obstacle of immunological rejection and will probably be key to providing cells for the assembly of synthetic organs.

## Hybridomas Produce Abundant Monoclonal Antibodies

In addition to serving as research models for studies of cell function, cultured cells can be converted into "factories" for producing specific proteins. For example, special cultured cells can be used to generate monoclonal antibodies, which are experimental tools widely used in many aspects of cell biological research. They are also used for diagnostic and therapeutic purposes in medicine, as we discuss in later chapters.

To understand the challenge of generating monoclonal antibodies, we must briefly review how mammals produce antibodies; more detail is provided in Chapter 23. Recall that antibodies are proteins secreted by white blood cells that bind with high affinity to their antigen (see Figure 3-19). Each normal antibody-producing B lymphocyte in a mammal is capable of producing a single type of antibody that can bind to a particular determinant, or epitope, on an antigen molecule. An epitope is generally a small region on the antigen, consisting, for example, of just a few amino acids. If an animal is injected with an antigen, the B lymphocytes that make antibodies recognizing that antigen are stimulated to grow and secrete those antibodies. Each antigen-activated B lymphocyte forms a clone of cells in the spleen or lymph nodes, with each cell producing the identical antibody-that is, a monoclonal antibody. Because most natural antigens contain multiple epitopes, exposure of an animal to an antigen usually stimulates the formation of multiple B-lymphocyte clones, each producing a different antibody. The resulting mixture of antibodies from the many B-lymphocyte clones that recognize different epitopes on the antigen is said to be
polyclonal. Such polyclonal antibodies circulate in the blood and can be isolated as a group.

Although polyclonal antibodies are very useful, monoclonal antibodies are more suitable for the many types of experiments and medical applications in which we need a reagent that binds to just one site on a protein; for example, one that competes with a ligand on a cell-surface receptor. Unfortunately, the biochemical purification of any one type of monoclonal antibody from blood is not feasible for two main reasons: the concentration of any given antibody is quite low, and all antibodies have the same basic molecular architecture (see Figure 3-19).

To produce and then purify monoclonal antibodies, one first needs to be able to grow the appropriate B-lymphocyte clone. However, primary cultures of normal B lymphocytes are of limited usefulness for the production of monoclonal antibodies because they have a limited life span. Thus the first step in producing a monoclonal antibody is to generate immortal antibody-producing cells (Figure 4-6). Immortality is achieved by fusing normal B lymphocytes from an immunized animal with transformed, immortal lymphocytes called myeloma cells that themselves do not synthesize antibodies (see Figure 3-19). Treatment with certain viral glycoproteins or the chemical polyethylene glycol encourages the plasma membranes of two cells to fuse, allowing their cytosols and organelles to intermingle. Some of the fused cells undergo division, and their nuclei eventually coalesce, producing viable hybrid cells with a single nucleus that contains chromosomes from both parent cells. The fusion of two cells that are genetically different can yield a hybrid cell with novel characteristics. For instance, the fusion of a myeloma cell with a normal antibody-producing cell from a rat or mouse spleen yields a hybrid that proliferates into a clone called a hybridoma. Like myeloma cells, hybridoma cells grow rapidly and are immortal. Each hybridoma produces the monoclonal antibody encoded by its B-lymphocyte parent.

The second step in this procedure for producing a monoclonal antibody is to separate, or select, the hybridoma cells from the unfused parent cells and the cells fused with another of the same type. This selection is usually performed by incubating the mixture of cells in a special culture medium, called a selection medium, that permits the growth of only the hybridoma cells because of their novel characteristics. The myeloma cells used for the fusion carry a mutation that blocks a metabolic pathway, so a selection medium can be used that is lethal to them and not to their B-lymphocyte fusion partners that do not have the mutation. In the immortal hybrid cells, the functional gene from the lymphocyte can supply the missing gene product. The lymphocytes used in the fusion are not immortal and will not be able to grow in the selection medium either. Thus the hybridoma cells will be the only ones able to proliferate rapidly in the selection medium and so can be readily isolated from the initial mixture of cells. Finally, each selected hybridoma clone is tested for the production of the desired antibody; any clone producing that antibody is then grown in large cultures, from which a substantial quantity of pure monoclonal antibody can be obtained.

![li193857775e0](li193857775e0.jpg)

Test each well for antibody to antigen $X$
FIGURE 4-6 Use of cell fusion and selection to obtain hybridomas producing a monoclonal antibody to a specific protein. Step $\mathbf{1}$ Immortal myeloma cells that cannot synthesize purines under special conditions because they lack thymidine kinase are fused with normal antibody-producing spleen cells from an animal that was immunized with antigen X. Step $\mathbf{2}$ When cultured in a special selective medium, unfused and self-fused cells do not grow: the myeloma cells do not grow because the selective medium does not contain purines, and the spleen cells do not grow because they have a limited life span in culture. Thus only fused cells formed from a myeloma cell and a spleen cell survive in the selective medium, proliferating into clones called hybridomas. Each hybridoma produces a single antibody. Step Testing of individual clones identifies those that recognize antigen X. After a hybridoma that produces a desired antibody has been identified, the clone can be cultured to yield large amounts of that antibody.

Monoclonal antibodies have become very valuable as specific research tools. They are commonly employed in affinity chromatography to isolate and purify proteins from complex mixtures (see Figure 3-38c). As we discuss later in this chapter, they can also be employed in immunofluorescence microscopy to bind to and so locate a particular protein within cells. They can also be used to identify specific proteins in cell fractions with the use of immunoblotting (see Figure 3-39). Monoclonal antibodies have become important diagnostic and therapeutic tools in medicine as well; for example, monoclonal antibodies that bind to and inactivate toxins secreted by bacterial pathogens are used to treat diseases. Other monoclonal antibodies are specific for cell-surface proteins expressed by certain types of tumor cells. Several of these anti-tumor antibodies are widely used in cancer therapy, including a monoclonal antibody against a mutant form of the Her2 receptor that is overexpressed in some breast cancers.

## A Wide Variety of Cell Biological Processes Can Be Studied with Cultured Cells

As we discussed in the introduction to this chapter, studying animal cells in culture is much easier than studying cells in intact animals, partly because they can easily be subjected to a variety of manipulations. Cultured cells are particularly useful for the elucidation of fundamental processes. One way to understand a biological process is to interfere with a specific constituent in the cell and assess the outcome-this approach is like trying to understand how a car works by removing components and seeing what goes wrong. In some cases, human diseases associated with genetic defects in specific cell components can be analyzed using cells cultured from the patients. For example, analysis of cultured cells from patients with a genetic defect resulting in hypercholesterolemia-who have elevated blood cholesterol leading to heart disease and stroke-was critical in elucidating the basic steps of receptormediated endocytosis (see Chapter 14). In addition to relying on naturally occurring genetic lesions, we can manipulate cultured cells to interfere with expression of specific components. As we will see in Chapter 6, it is possible to decrease the expression of a specific protein in cultured cells by selectively "knocking down" the corresponding mRNA and then assess what effect this change has on particular processes in the cell. Chapter 6 also describes very recently developed techniques that can be used to inactivate specific genes in the genomes of cultured cells and thereby assess how the complete loss of specific RNAs and proteins affects cell functions.

## Drugs Are Commonly Used in Cell Biological Research

Another powerful way to analyze biological processes is to treat cells with drugs that bind to specific cell components and inactive or activate them. In this section, we discuss some of the common drugs used for this purpose and how new drugs affecting specific cell processes can be developed.

Naturally occurring drugs have been used for centuries, but how they worked was often unknown. For example, extracts of meadow saffron were used to treat gout, a painful disease resulting from inflammation of joints. Today we know that this plant contains colchicine, a drug that depolymerizes microtubules and interferes with the ability of white blood cells to move to sites of inflammation (see Chapter 18). Alexander Fleming discovered that certain fungi secrete compounds that kill bacteria (antibiotics), and his discovery resulted in the development of penicillin. Only later was it was discovered that penicillin inhibits cell division by blocking the assembly of the cell walls of certain bacteria.

Discoveries like these have resulted in a wide range of drugs that can inhibit specific and essential processes of cells. In most cases, researchers have eventually been able to identify the molecular targets of these drugs. For example, there are many antibiotic drugs that affect aspects of prokaryotic protein synthesis. A selection of some of the drugs most commonly used in cell biological research are listed in Table 4-1, grouped according to the process they inhibit.

How does one discover a new drug? One widely used approach is to search chemical libraries, consisting of tens of thousands to hundreds of thousands of different compounds, for chemicals that inhibit a specific process. The screening

TABLE 4-1 Selected Drugs Used in Cell Biological Research

| DNA replication inhibitors | Aphidicolin (eukaryotic DNA polymerase inhibitor); camptothecin, etoposide (eukaryotic topoisomerase inhibitors) |
| :--: | :--: |
| Transcription inhibitors | $\alpha$-Amanitin (eukaryotic RNA polymerase II inhibitor); actinomycin D, 5,6-Dichloro-1-$\beta$-D-ribofuranosylbenzimidazole (DRB) (eukaryotic transcription elongation inhibitor); rifampicin (bacterial RNA polymerase inhibitor); thiolutin (bacterial and yeast RNA polymerase inhibitor) |
| Protein synthesis inhibitors: block general protein production; toxic after extended exposure | Cycloheximide, anisomycin (translation inhibitors in eukaryotes); geneticin/ G418, hygromycin, puromycin (translation inhibitors in bacteria and eukaryotes); chloramphenicol (translation inhibitor in bacteria and mitochondria); tetracycline (translation inhibitor in bacteria) |
| Protease inhibitors: block protein degradation | MG-132, lactacystin (proteasome inhibitors); E-64, leupeptin (serine and/or cysteine protease inhibitors); phenylmethanesulfonylfluoride (PMSF) (serine protease inhibitor); tosyl-L-lysine chloromethyl ketone (TLCK) (trypsin-like serine protease inhibitor) |
| Compounds affecting the cytoskeleton | Phalloidin, jasplakinolide (F-actin stabilizer); latrunculin, cytochalasin (F-actin polymerization inhibitors); taxol (microtubule stabilizer); colchicine, nocodazole, vinblastine, podophyllotoxin (microtubule polymerization inhibitors); monastrol (kinesin-5 inhibitor) |
| Compounds affecting membrane traffic, intracellular movement, and the secretory pathway, protein glycosylation | Brefeldin A (secretion inhibitor); leptomycin B (nuclear protein export inhibitor); dynasore (dynamin inhibitor); tunicamycin ( N -linked glycosylation inhibitor) |
| Kinase inhibitors | Genistein, rapamycin, gleevec (tyrosine kinase inhibitors with various specificities); wortmannin, LY294002 (PI3 kinase inhibitors); staurosporine (protein kinase inhibitor); roscovitine (cell cycle CDK1 and CDK2 inhibitors); U0126 (MEK inhibitor) |
| Phosphatase inhibitors | Cyclosporine A, FK506, calyculin (protein phosphatase inhibitors with various specificities); okadaic acid (general inhibitor of serine/threonine phosphatases); phenylarsine oxide, sodium orthovanadate (tyrosine phosphatase inhibitors) |
| Compounds affecting intracellular cAMP levels | Forskolin (adenylate cyclase activator) |
| Compounds affecting ions (e.g., $\mathrm{K}^{+}, \mathrm{Ca}^{2+}$ ) | A23187 ( $\mathrm{Ca}^{2+}$ ionophore); valinomycin ( $\mathrm{K}^{+}$ionophore); BAPTA (divalent cation (e.g., $\mathrm{Ca}^{2+}$ ) binding/sequestering agent); thapsigargin (endoplasmic reticulum $\mathrm{Ca}^{2+}$ ATPase inhibitor); ouabain $\left(\mathrm{Na}^{+} / \mathrm{K}^{+}\right.$ATPase inhibitor) |
| Some drugs used in medicine | Propranolol ( $\beta$-adrenergic receptor antagonist), statins (HMG-CoA reductase inhibitors, block cholesterol synthesis); omeprazole (a gastric proton pump inhibitor) |

[^0]
[^0]:    Note: Some of these molecules have broad specificity, whereas others are highly specific. More information about many of these compounds can be found in the relevant chapters in this text.

FIGURE 4-7 Screening for drugs that affect specific biological processes. (a) In this example, a chemical library of 16,320 different chemicals was subjected to a series of screens for inhibitors of mitosis. Since such an inhibitor is expected to arrest cells at the mitotic stage of the cell cycle, the first screen (step $\mathbf{\square}$ ) was to see if any of the chemicals enhanced the level of a marker specific for mitotic cells; this screen yielded 139 candidates. Microtubules make up the structure of the mitotic spindle, and the researchers were not interested in new drugs that target microtubules, so in the second screen (step $\mathbf{\square}$ ) they tested the 139 compounds for their ability to affect microtubule assembly; this test eliminated 53 candidates. Immunofluorescence microscopy with antibodies to tubulin (the major subunit of microtubules), together with a stain for DNA, was then used in the third screen (step $\mathbf{\square}$ ) to identify compounds that disrupt the structure of the spindle. (b) Localization of tubulin (green) and DNA (blue) for an untreated mitotic spindle (top) and one treated with one of the recovered compounds, now called monastrol. Monastrol inhibits a microtubule-based motor protein called kinesin-5, discussed in Chapter 18, that is necessary to separate the poles of the mitotic spindle. When kinesin-5 is inhibited, the two poles remain associated to give a monopolar spindle. [Part (b) T. U. Mayer et al., 1999, Science 286:971-974]
(a)
![li194857775e0](li194857775e0.jpg)
of chemical libraries in conjunction with high-throughput microscopic techniques has now become one of the major routes for new leads in drug discovery. Here we give just one case to illustrate how this type of approach works.

In our example (Figure 4-7a), researchers wanted to identify compounds that inhibit mitosis, the process by which duplicated chromosomes are accurately segregated by a microtubule-based machine called the mitotic spindle (discussed in Chapter 18). It was known that if spindle assembly is compromised, cells are arrested in mitosis. Therefore, the screen first used an automated robotic method to look for compounds that arrest cells in mitosis. The basis for the inhibition of mitosis by the candidate compounds was then explored to see if they affected assembly of the microtubules. Since inhibition of microtubule assembly was not of interest, the effect of the remaining candidate compounds on the structure of the spindle was determined by immunofluorescence microscopy using antibodies to tubulin, the major protein of microtubules. Over 16,000 compounds were screened, and a compound was identified that resulted in cells with abnormal spindlesinstead of having two asters, they had a single aster, resulting in what is called a mono-astral array (Figure 4-7b). This drug, now called monastrol, was found to interfere with the assembly of the spindle by inhibiting a microtubule-based motor protein called kinesin-5. (See Chapter 18 for more details about the mitotic spindle.) Derivatives of monastrol are now being tested as anti-tumor agents for the treatment of certain cancers.

## KEY CONCEPTS OF SECTION 4.1

## Growing and Studying Cells in Culture

- Animal cells have to be grown in culture under conditions that mimic their natural environment, which generally requires them to be supplied with necessary amino acids and growth factors.
- Most animal cells need to adhere to a solid surface to grow.
- Primary cells-those isolated directly from tissue-have a finite life span.
- Transformed cells, such as cells derived from tumors, can grow indefinitely in culture.
- Cells that can be grown indefinitely are called a cell line.
- Many cells lines are aneuploid, having a different number of chromosomes than the parent cell from which they were derived.
- Cells expressing a fluorescent protein can be sorted on a machine called a fluorescence-activated cell sorter (FACS).
- Different cell types express different marker proteins on their cell surfaces, which can be labeled with fluorescent markers, allowing them to be sorted on a FACS machine.

- Epithelial cells are often grown is special containers to mimic their functional polarity. Cells can also be grown on three-dimensional matrices to more accurately reflect their normal environment.
- Monoclonal antibodies, which bind one epitope on an antigen, can be secreted by cultured cells called hybridomas. These hybrid cells are made by fusing antibody-producing B lymphocytes with immortal myeloma cells and then identifying those clones that produce the desired antibody. Monoclonal antibodies are important for basic research and as therapeutic agents.
- Cells in culture can be much more easy manipulated than cells in an intact animal.
- Basic biological processes can be studied by interfering with specific cell components, either through genetic mechanisms or by the application of specific drugs.
- Large chemical libraries can be screened for compounds that target specific processes to study those processes and to identify new drugs.


### 4.2 Light Microscopy: Exploring Cell Structure and Visualizing Proteins Within Cells

The cellular basis of life was first appreciated using primitive light microscopes. Since then, progress in cell biology has paralleled, and has often been driven by, technological advances in light microscopy (Figure 4-8). Here we discuss each of these major developments and how they advanced the study of cellular processes. First we describe basic uses
of a light microscope to observe unstained cells and structures. Next we describe the development of fluorescence microscopy and its use to localize specific proteins in fixed cells. By using molecular genetic techniques to fuse a protein of interest with a naturally fluorescent protein and express the resulting chimeric protein in cells, it is possible to follow the movement of specific proteins in live cells-an ability that has revealed how dynamic the organization of live cells is. In parallel with these advances in specimen preparation, optical advances were being made to enhance and sharpen the images provided by fluorescence microscopy to reveal cellular structure with unprecedented clarity. Many specialized technologies have emerged from these advances, and we describe some of the more important ones.

Many of the techniques we describe allow one to examine live cells in a microscope. These advances not only permit video microscopy, but also allow one to examine the responses of live cells or their components to specific stimuli or their interactions with other cells. As we discuss in this section, they have provided scientists with the ability to probe the functioning of individual components in live cells.

## The Resolution of the Conventional Light Microscope Is About $0.2 \mu \mathrm{~m}$

All microscopes produce a magnified image of a small object, but the nature of the image depends on the type of microscope employed and on the way the specimen is prepared. The compound microscope, used in conventional brightfield light microscopy, contains several lenses that magnify the image of a specimen under study (Figure 4-9a, b). The total magnification is a product of the magnification of the individual lenses: if the objective lens, the lens closest to the specimen, magnifies 100 -fold (a $100 \times$ lens, the maximum usually employed), and the projection lens that focuses the
(a)
![li195857775e0](li195857775e0.jpg)
(b)
![li196857775e0](li196857775e0.jpg)
(c)
![li197857775e0](li197857775e0.jpg)

FIGURE 4-8 Development of the light microscope. (a) Early microscopes, like the ones used by Robert Hooke in the 1660s, used lenses or a mirror to illuminate the specimen. (b) Optics in general, and light microscopes in particular, developed enormously during the nineteenth century. By the middle of the twentieth century, highly
sophisticated microscopes limited only by the resolution of light were common. (c) In the second half of the twentieth century, fluorescence microscopy and digital imaging, together with confocal techniques, were developed to yield the versatile microscopes of today. (Part (a) SSPL/Getty Images; parts (b) and (c) courtesy of A. Bretscher.]

![li198857775e0](li198857775e0.jpg)

FIGURE 4-9 Optical microscopes are commonly configured for bright-field, phase-contrast, or fluorescence microscopy. (a) In a typical light microscope, the specimen is usually mounted on a transparent glass slide and positioned on the movable specimen stage. (b) In bright-field light microscopy, light from a tungsten lamp is focused on the specimen by a condenser lens below the stage; the light travels the pathway shown in yellow. (c) In phase-contrast microscopy, incident light passes through an annular diaphragm, which focuses a circular annulus (ring) of light on the specimen. Light that passes unobstructed through the specimen is focused by the objective lens onto the thicker gray ring of the phase plate, which absorbs some of the direct light and alters its phase by one-quarter of a wavelength. If a specimen refracts (bends) or diffracts the light, the phase of some light waves is altered (green lines), and the light waves pass through the clear region of the phase plate. The refracted and unrefracted light is recombined at the image plane to form the image. (d) In fluorescence microscopy, a beam of light from a mercury lamp (gray lines) is directed to the excitation filter, which allows only the correct wavelength of light to pass through (green lines). The light is then reflected off a dichroic mirror and through the objective lens, which focuses it on the specimen. The fluorescent light emitted by the specimen (red lines) passes up through the objective lens, then through the dichroic mirror, and is focused and recorded on the detector at the image plane.
image on a camera magnifies 10 -fold, the final magnification will be 1000 -fold. Alternatively, if the light is directed to an ocular or eyepiece lens that magnifies 10 -fold, the final magnification recorded by the human eye will also be 1000 -fold.

The most important property of any microscope, however, is not its magnification, but its resolving power, or resolution: the ability to distinguish between two very closely positioned objects. Merely enlarging the image of a specimen accomplishes nothing if the image is blurred. The resolution of a microscope lens is numerically equivalent to $D$, the minimum distance between two distinguishable objects. The smaller the value of $D$, the better the resolution. The value of $D$ is given by the equation

$$
D=\frac{0.61 \lambda}{\mathrm{~N} \sin \alpha}
$$

where $\alpha$ is the angular aperture, or half-angle, of the cone of light entering the objective lens from the specimen (Figure 4-9a), $N$ is the refractive index of the medium between the specimen and the objective lens (i.e., the relative velocity of light in the medium compared with the velocity in air), and $\lambda$ is the wavelength of the incident light. Resolution is improved by using shorter wavelengths of light (decreasing the value of $\lambda$ ) or by gathering more light (increasing either $N$ or $\alpha$ ). Lenses for high-resolution microscopy are designed to work with oil between the lens and the specimen since oil has a higher refractive index (1.56, compared with 1.0 for air and 1.3 for water). To maximize the angle $\alpha$, and hence $\sin$ $\alpha$, the lenses are also designed to focus very close to the thin coverslip covering the specimen. The term $N \sin \alpha$ is known as the numerical aperture ( $N A$ ) and is usually marked on the objective lens. A good high-magnification lens has an NA of about 1.4 , and the very best lenses-which cost as much as
a medium-sized car-have a value approaching 1.5. Notice that the magnification is not part of this equation.

Owing to limitations in the values of $\alpha, \lambda$, and $N$ based on the physical properties of light, the limit of resolution of a light microscope using visible light is about $0.2 \mu \mathrm{~m}$ $(200 \mathrm{~nm})$. No matter how many times the image is magnified, a conventional light microscope can never resolve objects that are closer than about $0.2 \mu \mathrm{~m}$ apart or reveal details smaller than about $0.2 \mu \mathrm{~m}$ in size. However, some new and sophisticated technologies devised to "beat" this resolution barrier can resolve objects just a few nanometers apart; we discuss such super-resolution microscopes in a later section.

Despite its lack of resolution, a conventional microscope can track a single object to within a few nanometers. If we know the precise size and shape of an object-say, a $5-\mathrm{nm}$ sphere of gold that is attached to an antibody that is in turn bound to a cell-surface protein on a live cell-and if we use a camera to rapidly take multiple digital images, then a computer can calculate the average position to reveal the center of the object to within a few nanometers. In this way, computer algorithms can be used to track a single object-in this case, the location and movement with time of a cell-surface protein labeled with the gold-tagged antibody-more precisely than would be possible based on the light microscope's resolution alone. This technique has been used to measure nanometer-sized steps as molecules and vesicles move along cytoskeletal filaments (see Figures 17-28 and 17-29).

## Phase-Contrast and Differential-InterferenceContrast Microscopy Visualize Unstained Live Cells

Cells are about 70 percent water, 15 percent protein, 6 percent RNA, and contain smaller amounts of lipids, DNA, and small molecules. Since none of these major classes of molecules are colored, and since they hardly impede the transmission of light, special methods must be used to see cells in a microscope. For example, the simplest microscopes view cells under bright-field optics (Figure 4-9b), and little detail can be seen (Figure 4-10). Two common methods for imaging live cells and unstained tissues to generate contrast take advantage of differences in the refractive index and thickness of cellular materials. These methods, called phasecontrast microscopy and differential-interference-contrast (DIC) microscopy (or Nomarski interference microscopy), produce images that differ in appearance and reveal different features of cell architecture. Figure 4-10 compares images of live, cultured cells obtained with these two methods and with standard bright-field microscopy. Since optical microscopes are expensive, they are often set up to perform many different types of microscopy on the same microscope stand (see Figure 4-9a-d).

Phase-contrast microscopy generates an image in which the degree of darkness or brightness of a region of a specimen depends on the refractive index of that region. Light moves more slowly in a medium with a higher refractive index. Thus

![li199857775e0](li199857775e0.jpg)

FIGURE 4-10 Live cells can be visualized by microscopy techniques that generate contrast by interference. These micrographs show live, cultured macrophage cells viewed by bright-field microscopy (left), phase-contrast microscopy (middle), and differential-interference-contrast (DIC) microscopy (right). In a phase-contrast
image, cells are surrounded by alternating dark and light bands; in-focus and out-of-focus details are simultaneously imaged in a phasecontrast microscope. In a DIC image, cells appear in pseudorelief. Because only a narrow in-focus region is imaged, a DIC image is an optical slice through the object. [Courtesy of N. Watson and James Evans.]
a beam of light is refracted (bent) once as it passes from the medium into a transparent object and again when it departs. In a phase-contrast microscope, a cone of light generated by an annular diaphragm in the condenser lens illuminates the specimen (see Figure 4-9c). The light passes through the specimen into the objective lens, and the unobstructed direct light passes through a region of the phase plate that both transmits only a small percentage of the light and changes its phase slightly. The part of a light wave that passes through a specimen will be refracted and will be out of phase (out of synchrony) with the part of the wave that does not pass through the specimen. How much their phases differ depends on the difference in refractive index along the two paths and on the thickness of the specimen. The refracted and unrefracted light is recombined at the image plane to form the image. If the two parts of the light wave are recombined, the resultant light will be brighter if they are in phase and less bright if they are out of phase. Phase-contrast microscopy is suitable for observing single cells or thin cell layers, but not thick tissues. It is particularly useful for examining the location and movement of larger organelles in live cells.

DIC microscopy, which is based on splitting the light into two perpendicular components before passing them through the specimen and then recombining them to observe their interference pattern, is the method of choice for visualizing extremely small details and thick objects. Contrast is generated by differences in the refractive index of the object and of its surrounding medium. In DIC images, objects appear to cast a shadow to one side. The "shadow" primarily represents a difference in the refractive index of a specimen rather than its topography. DIC microscopy easily defines the outlines of large organelles, such as the nucleus and vacuole. In addition to having a "relief"-like appearance, a DIC image is a thin optical section, or slice, through the object (Figure 4-10, right). Thus details of the nucleus in thick specimens (e.g., an intact Caenorhabditis elegans roundworm; see Figure 21-25d) can be observed in a series of such optical sections, and the three-dimensional structure of the object can be reconstructed by combining the individual DIC images.

Both phase-contrast and DIC microscopy can be used in time-lapse microscopy, in which the same cell is photographed at regular intervals over time to generate a movie. This procedure allows the observer to study cell movement, provided the microscope's stage can control the temperature of the specimen and the appropriate environment.

## Imaging Subcellular Details Often Requires That Specimens Be Fixed, Sectioned, and Stained

As we have seen, live cells and tissues generally do not absorb light, so they are nearly invisible in a light microscope. Although cells can be visualized by the special techniques we have just discussed, these methods do not reveal the fine details of structure.

Specimens for light microscopy are commonly fixed with a solution containing chemicals that cross-link most proteins and nucleic acids. Formaldehyde, a common fixative, crosslinks amino groups on adjacent molecules; these covalent bonds stabilize protein-protein and protein-nucleic acid interactions and render the molecules insoluble and stable for subsequent procedures. After fixation, a tissue sample for examination by light microscopy is usually embedded in paraffin and cut into sections about $50 \mu \mathrm{~m}$ thick (Figure 4-11a). Cultured cells growing on glass coverslips, as described above, are thin enough so they can be fixed in situ and visualized by light microscopy without the need for sectioning.

A final step in preparing a specimen for light microscopy is to stain it so as to visualize the main structural features of the cell or tissue. Many chemical stains bind to molecules that have specific features. For example, histological samples are often stained with hematoxylin and eosin ("H\&E stain"). Hematoxylin binds to basic amino acids (lysine and arginine) on many different kinds of proteins, whereas eosin binds to acidic molecules (such as DNA and side chains of the amino acids aspartate and glutamate). Because of their different binding properties, these dyes stain various cell types sufficiently differently that they are distinguishable

![li200857775e0](li200857775e0.jpg)

FIGURE 4-11 Tissues for light microscopy are commonly fixed, embedded in a solid medium, and cut into thin sections. (a) A fixed tissue is dehydrated by soaking in a series of alcohol-water solutions, ending with an organic solvent compatible with the embedding medium. To embed the tissue for sectioning, the tissue is placed in liquid paraffin. After the block containing the specimen has hardened, it is mounted on the arm of a microtome, and slices are cut with a knife. Typical sections cut for light microscopy are 0.5 to $50 \mu \mathrm{~m}$ thick. The sections are collected on microscope slides and stained with an appropriate agent. (b) Sections of normal (top) and cancerous (adenocarcinoma, bottom) human colon stained with H\&E stain. Notice the disorganization of the cells in the cancer tissue. [Part (b) courtesy of Dr. Alexander Nikitin, Cornell University.]
visually (Figure 4-11b). If an enzyme catalyzes a reaction that produces a colored or otherwise visible precipitate from a colorless precursor, that enzyme can be detected in cell sections by their colored reaction products. Such staining techniques, although once quite common, have been largely replaced by other techniques for visualizing particular proteins, as we discuss next.

## Fluorescence Microscopy Can Localize and Quantify Specific Molecules in Live Cells

Perhaps the most versatile and powerful technique for localizing molecules within a cell by light microscopy is fluorescent staining of cells and observation by fluorescence microscopy. A chemical is said to be fluorescent if it absorbs light at one wavelength (the excitation wavelength) and emits light (fluoresces) at a specific longer wavelength. Modern microscopes used for observing fluorescent samples are configured to pass the excitation light through the objective lens into the sample and then selectively observe the emitted fluorescent light coming back through the objective lens from the sample. This is achieved by reflecting the excitation light with a special type of filter, called a dichroic mirror, into the sample and allowing the light emitted at the longer wavelength to pass through to the observer (see Figure 4-9d). Here we discuss several ways in which fluorescence microscopy can be used to examine specific molecules in cells.

## Intracellular Ion Concentrations Can Be Determined with Ion-Sensitive Fluorescent Dyes

The concentration of $\mathrm{Ca}^{2+}$ or $\mathrm{H}^{+}$within live cells can be measured with the aid of fluorescent dyes, or fluorochromes, whose fluorescence depends on the concentration of these ions. As discussed in later chapters, intracellular $\mathrm{Ca}^{2+}$ and $\mathrm{H}^{+}$concentrations have pronounced effects on many cellular processes. For instance, many hormones and other stimuli cause a rise in cytosolic $\mathrm{Ca}^{2+}$ from the resting level of about $10^{-7} \mathrm{M}$ to $10^{-6} \mathrm{M}$, which induces various cellular responses such as the contraction of muscle.

The fluorochrome fura-2, which is sensitive to $\mathrm{Ca}^{2+}$, contains five carboxylate groups that form ester linkages with ethanol. The resulting fura-2 ester is lipophilic and can diffuse from the medium across the plasma membrane into cells. Within the cytosol, esterases hydrolyze the fura-2 ester, yielding fura-2, whose free carboxylate groups render the molecule nonlipophilic and thus unable to cross cellular membranes, so it remains in the cytosol. Inside cells, each fura- 2 molecule can bind a single $\mathrm{Ca}^{2+}$ ion, but no other cellular cation. This binding, which is proportional to the cytosolic $\mathrm{Ca}^{2+}$ concentration over a certain range, increases the fluorescence of fura-2 at one particular wavelength. At a second wavelength, the fluorescence of fura- 2 is the same whether or not $\mathrm{Ca}^{2+}$ is bound and thus provides a measure of the total amount of fura-2 in a region of the cell. By examining cells continuously in the fluorescence microscope and measuring rapid changes in the ratio of fura-2 fluorescence

![li201857775e0](li201857775e0.jpg)

EXPERIMENTAL FIGURE 4-12 Fura-2, a $\mathrm{Ca}^{2+}$-sensitive fluorochrome, can be used to monitor the relative concentrations of cytosolic $\mathrm{Ca}^{2+}$ in different regions of live cells. (Left) In a moving leukocyte, a $\mathrm{Ca}^{2+}$ gradient is established. The highest concentrations (green) are at the rear of the cell, where cortical contractions take place, and the lowest concentrations (blue) are at the cell front, where actin undergoes polymerization. (Right) When a pipette filled with chemotactic molecules placed to the side of the cell induces the cell to turn, the $\mathrm{Ca}^{2+}$ concentration momentarily increases throughout the cytoplasm, and a new gradient is established. The gradient is oriented such that the region of lowest $\mathrm{Ca}^{2+}$ (blue) lies in the direction that the cell will turn, whereas a region of high $\mathrm{Ca}^{2+}$ (yellow) always forms at the site that will become the rear of the cell. (From R. A. Brundage et al., 1991, Science 254:703; courtesy of F. Fay.)
at the two wavelengths, one can quantify rapid changes in the fraction of fura-2 that has bound a $\mathrm{Ca}^{2+}$ ion, and thus in the concentration of cytosolic $\mathrm{Ca}^{2+}$ (Figure 4-12).

Fluorescent dyes (e.g., SNARF-1) that are sensitive to $\mathrm{H}^{+}$concentrations can be used similarly to monitor the cytosolic pH of live cells. Other useful probes consist of a fluorochrome linked to a weak base that is only partially protonated at neutral pH and thus can freely permeate cellular membranes. In acidic organelles, however, these probes become protonated; because the protonated probes cannot recross the organelle membrane, they accumulate in the lumen at concentrations much greater than in the cytosol. Thus this type of fluorescent dye can be used to specifically stain particular organelles in live cells (Figure 4-13).

## Immunofluorescence Microscopy Can Detect Specific Proteins in Fixed Cells

The common chemical dyes mentioned above stain nucleic acids or broad classes of proteins, but it is much more informative to detect the presence and location of specific proteins. Immunofluorescence microscopy, the most widely used method of detecting specific proteins, uses an antibody to which a fluorescent dye has been covalently attached. To use this method, one must first generate antibodies to the specific protein of interest. As discussed briefly in Section 4.1 and in detail in Chapter 23, as part of the response to infection, the vertebrate immune system generates proteins called antibodies that bind specifically to the infectious agent. Cell
biologists have made use of this immunological response to generate antibodies to specific proteins. Consider you have purified protein X and then inject it into an experimental animal so that it responds to the protein as a foreign molecule. Over a period of weeks, the animal will mount an immune response and make antibodies to protein X (the "antigen"). If you collect the blood from the animal, it will have antibodies to protein X mixed in with antibodies to many other different antigens, together with all the other blood proteins. You can now covalently bind protein X to a resin and, using affinity chromatography, bind and selectively retain just those antibodies specific to protein X. The antibodies can be eluted from the resin, and now you have a reagent that binds specifically to protein X. This approach generates polyclonal antibodies since many different cells in the animal have contributed the antibodies, and the antibodies are likely to bind to many different epitopes on protein X. Alternatively, as we described earlier in this chapter, it is possible to generate a clonal cell line that secretes antibodies to a specific epitope on protein X; these are called monoclonal antibodies.

To use either type of antibody to localize a protein, the cells or tissue must first be fixed to ensure that all components remain in place, and the cell must be permeabilized to allow entry of the antibody, which is commonly done by incubating the cells with a non-ionic detergent or by
![li202857775e0](li202857775e0.jpg)

EXPERIMENTAL FIGURE 4-13 Location of lysosomes and mitochondria in a cultured living bovine pulmonary artery endothelial cell. The cell was stained with a green-fluorescing dye that is specifically bound to mitochondria and a red-fluorescing dye that is specifically incorporated into lysosomes. The image was sharpened using a deconvolution computer program discussed later in the chapter. N, nucleus. [© 2015 Thermo Fisher Scientific, Inc. Used under permission.]

extracting the lipids with an organic solvent. In one version of immunofluorescence microscopy, the antibody is covalently linked to a fluorochrome. Classically used fluorochromes include rhodamine and Texas red, which emit red light; Cy3, which emits orange light; and fluorescein, which emits green light; but newer and more photostable fluorochromes, with emission wavelengths from blue to far-red, have now been developed. When a fluorochromeantibody complex is added to a permeabilized cell or tissue section, the complex will bind to the corresponding antigen, then light up when illuminated at the excitation wavelength. Staining a specimen with different dyes that fluoresce at different wavelengths allows multiple proteins as well as DNA to be localized within the same cell (see the chapter-opening figure).

The most commonly used variation of this technique is called indirect immunofluorescence microscopy because the antibody specific to the protein of interest is detected indirectly. In this technique, an unlabeled monoclonal or polyclonal antibody is applied to the specimen, followed by a second, fluorochrome-tagged antibody that binds to the constant ( Fc ) segment of the first antibody. For example, a "secondary" antibody can be generated by immunizing a goat with the Fc segment that is common to all rabbit IgG antibodies; when coupled to a fluorochrome, this second antibody preparation (called "goat anti-rabbit") will detect any rabbit antibody used to stain a tissue or cell (Figure 4-14). Because several
![li203857775e0](li203857775e0.jpg)

Mount specimen and observe in fluorescence microscope
FIGURE 4-14 A specific protein can be localized in fixed tissue sections by indirect immunofluorescence microscopy. (a) To localize a protein by immunofluorescence microscopy, a tissue section, or sample of cells, must be chemically fixed and made permeable to antibodies (step $\mathbf{\square}$ ). The sample is then incubated with a primary antibody that binds specifically to the antigen of interest, and unbound antibody is then removed by washing (step $\mathbf{\square}$ ). The sample is next incubated with a fluorochrome-labeled secondary antibody that specifically binds to the primary antibody, and again, excess secondary antibody is removed by washing (step $\mathbf{\square}$ ). The sample is then mounted in specialized mounting
goat anti-rabbit antibody molecules can bind to a single rabbit antibody molecule in a specimen, the fluorescence is generally much brighter than if a single fluorochrome-tagged antibody were used. This approach is often extended to do doublelabel fluorescence microscopy, in which two proteins can be visualized simultaneously. For example, two proteins can be visualized by indirect immunofluorescence microscopy using primary antibodies made in different animals (e.g., rabbit and chicken) and secondary antibodies (e.g., goat-anti-rabbit and sheep-anti-chicken) labeled with different fluorochromes. In another variation, one protein can be visualized by indirect immunofluorescence microscopy and the second protein by a dye that specifically binds to it. Once the individual images are taken on the fluorescence microscope, they can be merged electronically (Figure 4-15).

In another widely used version of this technology, molecular genetic techniques are used to make a cDNA encoding a recombinant protein to which is fused a short sequence of amino acids called an epitope tag. When expressed in cells, this cDNA will generate the protein linked to the specific tag. Two commonly used epitope tags are FLAG, which encodes the amino acid sequence DYKDDDDK (single-letter amino acid code), and myc, which encodes the sequence EQKLISEEDL. Commercial fluorochrome-coupled monoclonal antibodies to the FLAG or myc epitopes can then be used to detect the recombinant protein in the cell. In an extension of this technology to allow the simultaneous visualization
![li204857775e0](li204857775e0.jpg)
medium and examined in a fluorescence microscope (step $\mathbf{\square}$ ). (b) In this example, a section of the rat intestinal wall was stained with Evans blue, which generates a nonspecific red fluorescence, and GLUT2, a glucose transport protein, was localized by indirect immunofluorescence microscopy. GLUT2 (yellow) is seen to be present in the basal and lateral sides of the intestinal cells, but is absent from the brush border, composed of closely packed microvilli on the apical surface facing the intestinal lumen. Capillaries run through the lamina propria, a loose connective tissue beneath the epithelial layer. [Part (b) from B. Thorens et al., 1990, Am. J. Physiol. 259:C279; courtesy of B. Thorens.]

![li205857775e0](li205857775e0.jpg)

Primary antibodies (rabbit, black) that recognize microtubules and fluorochrome-conjugated secondary antibodies (goat-anti-rabbit, green)
(b)
![li206857775e0](li206857775e0.jpg)

EXPERIMENTAL FIGURE 4-15 Double-label fluorescence microscopy can visualize the relative distributions of two proteins. In double-label fluorescence microscopy, each protein must be labeled with a different fluorochrome. (a) A cultured cell was fixed and permeabilized and then incubated with Rhodamine-labeled phalloidin, a reagent that specifically binds to filamentous actin. It was also incubated with rabbit antibodies to tubulin, the major component of microtubules, followed by a fluorescein-labeled secondary goat-anti-rabbit antibody. (b) The upper panels show the fluorescein-stained tubulin (left) and Rhodamine-stained actin (right), and the lower panel shows the electronically merged images. [Part (b) courtesy of A. Bretscher.]
of two proteins, one protein can be tagged with FLAG and a different protein with myc. Each tagged protein is then visualized with a different color, for example, with an Alexa-488-labeled antibody (emitting green light) to the myc epitope and an Alexa-568-labeled antibody (emitting red light) to the FLAG epitope.

## Tagging with Fluorescent Proteins Allows the Visualization of Specific Proteins in Live Cells

The jellyfish Aequorea victoria expresses a naturally fluorescent protein, called green fluorescent protein (GFP, $\sim 27 \mathrm{kDa}$ ). GFP contains a serine, tyrosine, and glycine sequence whose side chains spontaneously cyclize to form a green-fluorescing fluorochrome when illuminated with blue light. Using recombinant DNA technology, it is possible to make a DNA construct in which the coding sequence of GFP is fused to the coding sequence of a protein of interest. When this construct is introduced into and expressed in cells, a GFP"tagged" protein is made in which the protein of interest is covalently linked to GFP as part of the same polypeptide. Although GFP is a medium-sized protein, the function of the protein of interest is often not changed by fusing it to GFP. This technique allows one to visualize GFP—and hence the protein of interest. One can not only see the location of the GFP-tagged protein immediately, but can also view its distribution in a live cell over time and thereby assess its dynamics or track its localization following various cell treatments. The simple idea of tagging specific proteins with GFP has revolutionized cell biology and has led to the development of many different fluorescent proteins (Figure 4-16). Use of this colorful variety of fluorescent proteins allows one to visualize two
(a)
![li207857775e0](li207857775e0.jpg)
(b)
![li208857775e0](li208857775e0.jpg)

FIGURE 4-16 Many different colors of fluorescent proteins are now available. (a) Tubes show the emission colors and names of many different fluorescent proteins. (b) An agar dish is illuminated to show growing bacteria expressing several different-colored fluorescent proteins. [Courtesy of Roger Tsien.]

or more proteins simultaneously if they are each tagged with a different-colored fluorescent protein. We describe additional techniques that exploit fluorescent proteins in the following sections.

## Deconvolution and Confocal Microscopy Enhance Visualization of Three-Dimensional Fluorescent Objects

Conventional fluorescence microscopy has two major limitations. First, the fluorescent light emitted by a sample comes not only from the plane of focus, but also from molecules above and below it; thus the observer sees a blurred image caused by the superposition of fluorescent images from molecules at many depths in the cell. The blurring effect makes it difficult to determine the actual spatial arrangements. Second, to visualize thick specimens, consecutive (serial) images at various depths throughout the sample must be collected and then aligned to reconstruct structures in the original thick tissue. Two general approaches have been developed to obtain high-resolution three-dimensional information. Both of these methods require that the image be collected electronically so that it can then be computationally manipulated as necessary.

The first approach, called deconvolution microscopy, uses computational methods to remove fluorescence contributed by out-of-focus parts of the sample. Consider a three-dimensional sample in which images from three different focal planes are recorded. Since the whole sample is illuminated, the image from plane 2 will contain out-of-focus fluorescence from planes 1 and 3. If we knew exactly how out-of-focus fluorescence from planes 1 and 3 contributed to the light collected in plane 2 , we could computationally remove it. To obtain this information for a particular microscope, we can make a series of images of focal planes from a test slide containing tiny fluorescent beads. Each bead represents a pinpoint of light that becomes a blurred object outside its focal plane; from these images we can determine a point spread function that enables us to calculate the distribution of fluorescent point sources that contributed to the "blur" when out of focus. Once we have calibrated the microscope in this manner, the experimental series of images can be computationally deconvolved. Microscopes with automated stages to collect the images, and associated software programs to deconvolve the images, are available commercially. Images restored by deconvolution display impressive detail without any blurring, as illustrated in Figure 4-17.

The second approach to obtaining better threedimensional information is called confocal microscopy because it uses optical methods to obtain images from a specific focal plane and exclude light from other planes. Confocal microscopes collect a series of images focused through the vertical depth of the sample, from which an accurate three-dimensional representation can be computationally generated. Two types of confocal microscopes are in common use today, a point-scanning confocal
(a)
![li209857775e0](li209857775e0.jpg)
(b)
![li210857775e0](li210857775e0.jpg)

## EXPERIMENTAL FIGURE 4-17 Deconvolution fluorescence microscopy yields high-resolution optical sections that can be reconstructed to create one three-dimensional image. A

macrophage cell was stained with fluorochrome-labeled reagents specific for DNA (blue), microtubules (green), and actin microfilaments (red). The series of fluorescent images obtained at consecutive focal planes (optical sections) through the cell were recombined in three dimensions. (a) In this three-dimensional reconstruction of the raw images, the DNA, microtubules, and actin appear as diffuse zones in the cell. (b) After application of the deconvolution algorithm to the images, the fibrillar organization of microtubules and the localization of actin to adhesions are readily visible in the reconstruction. (Courtesy of James Evans, PhenolVista Biosciences.)
microscope (also known as a laser-scanning confocal microscope, or LSCM) and a spinning disk confocal microscope. The idea behind each microscope is to both illuminate and

collect emitted fluorescent light from just one small area of a focal plane at a time in such a way that out-of-focus light is excluded. This can be achieved by collecting the emitted light through a pinhole before it reaches the detectorlight from the illuminated focal plane passes through, whereas light from other focal planes is largely excluded. The illuminated area is then moved across the whole focal plane to build up the image electronically. The two types of microscopes differ in how they cover the image. The point-scanning microscope uses a point laser light source at the excitation wavelength to rapidly scan the focal plane in a raster pattern, collects the emitted fluorescence in a photomultiplier tube, and thereby builds up an image (Figure 4-18a). It can then take a series of images at different depths in the sample to generate a three-dimensional reconstruction. A point-scanning confocal microscope can provide exceptionally high-resolution images in both two and three dimensions (Figure 4-19), although it has two minor limitations. First, it can take significant time to
![li211857775e0](li211857775e0.jpg)

FIGURE 4-18 Light paths for two types of confocal microscopy. Both types of microscopy are assembled around a conventional fluorescence microscope (yellow shading). (a) Light path in a point-scanning confocal microscope. A single-wavelength point of light from an appropriate laser is reflected off a dichroic mirror and bounces off two scanning mirrors and from there passes through the objective lens to illuminate a spot in the specimen. The scanning mirrors rock back and forth in such a way that the light scans the specimen in a raster fashion (see green lines in the specimen). The fluorescence emitted by the specimen passes back through the objective lens and is bounced off the scanning mirrors onto the dichroic mirror. This allows the light to pass through toward the pinhole. This pinhole excludes light from out-of-focus focal planes, so the light reaching the photomultiplier tube
scan each focal plane, so if a very dynamic process is being imaged, the microscope may not be able to collect images fast enough to follow the dynamics. Second, it illuminates each spot with intense laser light, which can bleach the fluorochrome being imaged and damage live cells by phototoxicity, thereby limiting the number of images that can be collected.

The spinning disk microscope circumvents these two problems (Figure 4-18b). The excitation light from a laser is spread out and illuminates a small part of a disk spinning at high speed; for example, at 3000 rpm . The disk in fact consists of two linked disks: one with 20,000 lenses that precisely focuses the laser light on 20,000 pinholes of the second disk. The pinholes are arranged in such a way that they completely scan the focal plane of the sample several times with each turn of the disk. The emitted fluorescent light returns through the pinholes of the second disk and is reflected by a dichroic mirror and focused onto a highly sensitive digital camera. In this way, the sample is scanned
(b) Spinning disk confocal microscope
![li212857775e0](li212857775e0.jpg)
comes almost exclusively from the illuminated spot in the focal plane. A computer then takes these signals and reconstructs the image. (b) Light path in a spinning disk confocal microscope. Here, instead of using two scanning mirrors, the beam from the laser is spread to illuminate pinholes on the coupled spinning disks, the first consisting of microlenses to focus the light on pinholes in the second disk. The excitation light passes through the objective lens to provide point illumination of a number of spots in the specimen. The fluorescence emitted passes back through the objective lens and through the holes in the spinning disk, and is then bounced off a dichroic mirror into a sensitive digital camera. The pinholes in the disk are arranged so that as it spins, it rapidly illuminates all parts of the specimen several times. As the disk spins as fast as 3000 rpm , very dynamic events in live cells can be recorded.

![li213857775e0](li213857775e0.jpg)

FIGURE 4-19 Confocal microscopy produces an in-focus optical section through thick cells. A mitotic fertilized egg from a sea urchin (Psammechinus) was lysed with a detergent, exposed to an tubulin antibody, and then exposed to a fluorescein-tagged antibody that binds to the anti-tubulin antibody. (a) When viewed by conventional fluorescence microscopy, the mitotic spindle is blurred. This blurring occurs
because background fluorescence is detected from tubulin above and below the focal plane as depicted in the sketch. (b) The confocal microscopic image is sharp, particularly in the center of the mitotic spindle. In this case, fluorescence is detected only from molecules in the focal plane, generating a very thin optical section. [Micrographs © 1987 White et al., 1987, The Journal of Cell Biology 105:41-48. doi10.1083/jcb.105.1.41]
in less than a millisecond, so the real-time location of a fluorescent reporter can be captured even if it is highly dynamic (Figure 4-20). A current limitation of the spinning disk microscope is that the pinhole size is generally fixed and has to be matched to the magnification of the objective lens, so it is generally configured for use with a $63 \times$ or $100 \times$ objective and is less useful for the lower-magnification imaging that might be required in tissue sections. However, within the last year, spinning disk head units have become available in which the pinhole size can be changed. Thus the pointscanning and spinning disk confocal microscopes have overlapping and complementary strengths.

## Two-Photon Excitation Microscopy Allows Imaging Deep into Tissue Samples

We have just seen how point-scanning confocal microscopy can help reduce fluorescence from out-of-focus planes. To achieve this, it focuses a cone of laser light on a spot that scans across the focal plane. Regions above and below the focal plane are also illuminated by this cone of light,
generating out-of-focus signal that must be removed by collecting the light through a pinhole. This intense cone of light can also lead to photobleaching (rendering the fluorescent protein inactive) or damage to the sample by phototoxicity. If the sample is very thin, these are not significant problems, but as the sample gets thicker, they become more relevant. To circumvent these problems, use was made of the finding that a fluorochrome can be excited either by a single photon-for example, at 488 nm -or by two photons of half the energy at 960 nm , either of which will generate the same emission wavelength (Figure 4-21a). Thus if a $960-\mathrm{nm}$ cone of laser light is focused on a spot in one plane so that only at the focal point is there sufficient density of photons to excite the fluorochrome (Figure 4-21b), no out-of-focus signal will be obtained and less photobleaching or phototoxicity will occur. Because only fluorochromes in the focal plane are excited, two-photon microscopy can be used to explore much thicker samples, and there is no need for a pinhole to exclude out-of-focus light. However, very high laser intensity is required for two-photon excitation microscopy, as the two photons must arrive within about a femtosecond
![li214857775e0](li214857775e0.jpg)

EXPERIMENTAL FIGURE 4-20 The dynamics of microtubules can be imaged on the spinning disk confocal microscope. Six
frames from a movie of GFP-tubulin in two rod-shaped cells of fission yeast are shown. [Courtesy of Fred Chang.]

![li215857775e0](li215857775e0.jpg)

FIGURE 4-21 Two-photon excitation microscopy restricts illumination to the focal plane to allow deep penetration for intravital imaging. (a) A diagram illustrating the different excitation methods used for conventional point-scanning confocal microscopy and for two-photon excitation microscopy. In the conventional system, absorption of a single photon of the appropriate wavelength (here at 488 nm , shown by the blue arrow) results in an electron jumping to the excited state. After undergoing vibrational relaxation (black dashed arrow), the electron falls back to the ground state with emission of one photon at a longer (lower-energy) wavelength, in this case 507 nm (green arrow). In two-photon excitation, when two photons of the appropriate wavelength (shown here at 960 nm , red arrows) arrive almost instantaneously, they can both be absorbed and induce the electron to jump to the excited state. As in the previous case, the electron undergoes some vibrational relaxation (black dashed arrow) and falls back to the
of each other, which is achieved using rapid laser pulses. If individual cells in an animal express different color variants of fluorescent proteins, two-photon microscopy can be used to observe events in living animals (called intravital imaging) within about 1 mm of the surface (Figure 4-21c,d).

## TIRF Microscopy Provides Exceptional Imaging in One Focal Plane

The confocal microscopes we have just described provide amazing and informative images, but they are not perfect. Scientists continue to develop systems that are optimized
(c)
![li216857775e0](li216857775e0.jpg)
(d)
![li217857775e0](li217857775e0.jpg)
ground state with the emission of a photon ( 507 nm ). (b) A cuvette of fluorescent material is illuminated with $488-\mathrm{nm}$ light (left), as in conventional confocal microscopy, or with intense $960-\mathrm{nm}$ light, as in two-photon microscopy. Notice that the conventional system produces a bright cone of excitation outside the focal plane, whereas two-photon excitation illuminates just one spot in the focal plane.
(c) Because two-photon microscopy does not excite fluorochromes outside the plane of focus, it can be used to observe cells up to 1 mm deep within a living animal ("intravital imaging"). To image a living animal, it has to be immobilized on the microscope stage and access given for the objective lens to come close to the region being imaged. (d) An example of intravital imaging in which labeled neurons in a lobster were imaged. [Part (b) from W. Zipfel et al. 2003, Macmillan Publishers Ltd: Nature Biotechnol. 21:1369, courtesy Warren R. Zipfel; part (d) unpublished data from Peter Kloppenburg and Warren R. Zipfel.]
for special circumstances. Some experimental situations call for fluorescence imaging in a thin focal plane adjacent to a surface, where it would be optimal to minimize out-of-focus background. For example, confocal imaging is not ideal for exploring the details of proteins at adhesion sites between a cell and a coverslip, or for following the kinetics of assembly of microtubules attached to a coverslip. Both of these situations can be imaged at high sensitivity using total internal reflection fluorescence (TIRF) microscopy, in which only the portion of the specimen immediately adjacent to the coverslip is illuminated. In the most common configuration of TIRF microscopy, the

![li218857775e0](li218857775e0.jpg)

EXPERIMENTAL FIGURE 4-22 Fluorescent samples in a restricted focal plane can be imaged by total internal reflection fluorescence (TIRF) microscopy. (a) In TIRF microscopy, only about $50-100 \mathrm{~nm}$ of the specimen adjacent to the coverslip is illuminated, so that fluorescent molecules in the rest of the sample are not excited. This limited illumination is achieved by directing the illuminating light at an angle at which it is reflected from the glass-water interface of the coverslip rather than passing through it. Whereas most of the light is reflected, it also generates a very small region of illumination called the evanescent wave (depicted in light green). (b) Immunofluorescence microscopy with tubulin antibody was used to visualize microtubules viewed by conventional fluorescence microscopy (top) and by TIRF (middle), and a merged image was created from the two views (bottom). The two images were collected and false-colored red and green so that the merge could highlight those microtubules that are close to the coverslip (green). [Part (b) © 2010 J. B. Manneville et al., 2010, The Journal of Cell Biology 191:585-598. doi: 10.1083/jcb.201002151.]
excitation light comes through the objective lens (Figure 4-22a). However, the angle at which the light arrives at the coverslip is adjusted so that the light is reflected off the coverslip and returns up through the objective. This generates a narrow band of light, called an evanescent wave, that illuminates only about $50-100 \mathrm{~nm}$ of the sample adjacent to the coverslip ( $2-4$ times the thickness of a microtubule), with no illumination of the rest of the sample. Thus if you have a complex mixture of fluorescent structures in a specimen, the TIRF microscope will show you only those that are within $50-100 \mathrm{~nm}$ of the coverslip. TIRF has been exceptionally useful in identifying structures on the bottoms of cells grown on a coverslip (Figure 4-22b) and for measuring the kinetics of assembly and disassembly of structures such as microtubules and actin filaments (see Chapters 17 and 18).

## FRAP Reveals the Dynamics of Cellular Components

Live cell fluorescence imaging reveals the locations and bulk dynamics of populations of fluorescent molecules, but it doesn't tell you how dynamic individual molecules are. For example, if we see that a GFP-labeled protein forms a patch at the surface of a cell, does this represent a stable collection of fluorescent protein molecules or a dynamic equilibrium, with fluorescent proteins coming in and out of the patch? We can investigate this question by observing the dynamics of the molecules in the patch (Figure 4-23).
(b)
![li219857775e0](li219857775e0.jpg)
![li220857775e0](li220857775e0.jpg)

If we use a high-intensity light to permanently bleach the fluorochrome (e.g., GFP) in the patch, there will initially be no fluorescence coming from it, and it will look dark in the fluorescence microscope. However, if the components in the patch are in dynamic equilibrium with unbleached molecules elsewhere in the cell, the bleached molecules will be replaced by unbleached ones, and the fluorescence will begin to come back. The rate of fluorescence recovery is a measure of the dynamics of the molecules. This technique, known as fluorescence recovery after photobleaching (FRAP), has revealed how very dynamic many components of cells are. For example, it has been used to determine the diffusion coefficient of membrane proteins (see Figure 7-10) and the dynamics of specific components of the secretory pathway. In another approach to measuring the dynamics of tagged proteins, variants of fluorescent proteins have been developed that can be switched, using a laser of an appropriate wavelength, from emitting green light to emitting red light. In this way, the dynamics of the switched population of red-emitting molecules can be imaged in live cells.

![li221857775e0](li221857775e0.jpg)

## FRET Measures Distance Between Fluorochromes

Fluorescence microscopy can also be used to determine if two proteins interact in vivo by taking advantage of a phenomenon called Förster resonance energy transfer (FRET). This technique uses a pair of fluorescent proteins in which the emission wavelength of the first is close to the excitation wavelength of the second (Figure 4-24). For example, when cyan fluorescent protein (CFP) is excited with $433-\mathrm{nm}$ light, it fluoresces and emits light at 475 nm . If yellow fluorescent protein (YFP) is close by, however, instead of emitting $475-\mathrm{nm}$ light, CFP transfers energy to YFP by FRET, and YFP emits light at 530 nm . The efficiency of FRET is proportional to $R^{-6}$, where $R$ is the distance between the fluorochromes; it is therefore very sensitive to small changes in distance and in practice is not detectable at distances greater than 10 nm . Thus, by illuminating an appropriately prepared sample with $433-\mathrm{nm}$ light and observing at 530 nm , one can tell if proteins separately tagged with CFP and YFP are in very close proximity. For example, FRET sensors have been developed to determine where signaling between a small GTP-binding regulatory protein and its effector occurs in the cell (Figure 4-24b).
![li222857775e0](li222857775e0.jpg)

EXPERIMENTAL FIGURE 4-23 Fluorescence recovery after photobleaching (FRAP) reveals the dynamics of molecules. In a live cell, following the distribution of a GFP-labeled protein provides a view of the overall distribution of the protein, but it doesn't tell us how dynamic populations of individual molecules might be. (a) In FRAP, the GFP fluorochrome is bleached by a short burst of strong laser light focused on the region of interest (ROI). The light rapidly bleaches the molecules irreversibly, so they are not detected again. Restoration of fluorescence in the region tells us that unbleached molecules have moved into the ROI. (b) FRAP was used to determine the exchange rate of EBP50 and ezrin, two components of microvilli (seen as white lines), on the apical surface of epithelial cells. In cells expressing either GFPEBP50 or ezrin-GFP, the GFP in a small region indicated by the green box was bleached, and recovery by exchange with unbleached protein was followed. The fluorescence of GFP-EBP50 returns very fast, indicating it has a fast exchange rate, whereas the fluorescence of ezrin-GFP returns slowly, indicating a slower exchange rate. (c) By quantifying the recovery, the dynamic properties of EBP50 and ezrin can be established. [Parts (b) and (c) © 2012 Garbett et al., 2012, The Journal of Cell Biology 198:195-203. doi: 10.1083/jcb.201204008.]

A clever application of FRET, called a FRET biosensor, can be used to sense local biochemical environments in live cells. The idea is to express a single polypeptide containing both CFP and YFP separated by a region that undergoes a conformational change when it senses a biochemical signal. In the absence of the signal, CFP and YFP are too far apart to undergo significant FRET. However, when the signal is detected, the conformational change brings CFP and YFP close enough together to generate FRET. A version of this technique can be used to measure the local activity of a specific protein kinase. In this case, between the CFP and YFP lies a region of polypeptide containing the substrate for the protein kinase-the sensor domain-and a domain that binds specifically to the phosphorylated substrate-the ligand domain (Figure 4-25a). When the sensor domain is phosphorylated by the kinase, the ligand domain binds to it and brings the CFP and YFP sufficiently close to undergo FRET. Since protein phosphorylation is a dynamic process, dephosphorylation of the sensor domain by the appropriate phosphatase will deactivate the FRET biosensor. Thus the FRET signal will reflect the regions of the cell where there is excess kinase over phosphatase activity. As an example, scientists have developed a FRET biosensor for protein kinase A activity, which is activated by

(a)
![li223857775e0](li223857775e0.jpg)
(b)
![li224857775e0](li224857775e0.jpg)

FIGURE 4-24 Protein-protein interactions can be visualized by FRET. The idea behind FRET is to use two different fluorescent proteins so that when one is excited, energy will be transferred to the second one by FRET, provided that they are sufficiently close. (a) In this example, cyan fluorescent protein (CFP) is fused to protein $X$, yellow fluorescent protein (YFP) is fused to protein $Y$, and both proteins are expressed in a live cell. If the cell is now illuminated with $433-\mathrm{nm}$ light, the CFP will emit a fluorescent signal at 475 nm . If YFP is not close by (left), energy transfer will not occur, and no 530-nm light will be emitted. However, if protein $X$ interacts with protein $Y$ (right), it will bring CFP close to YFP, energy will be transferred to YFP by FRET, and YFP will emit light at 530 nm . (b) In this mouse fibroblast, FRET has been used to reveal that the interaction between an active regulatory protein (Rac) and its binding partner is localized to the front of the migrating cell. [Part (b) © 2003 R. B. Sekar, A. Periasamy, et al. 2003, The Journal of Cell Biology 160:629-633. doi: 10.1083/jcb.200210140.]
elevation of the signaling molecule cAMP (see Section 15.1). In cells expressing the protein kinase A biosensor, pharmacological elevation of cAMP induces rapid FRET (Figure 4-25b). Creative researchers are developing FRET biosensors to illuminate many different types of local environments; for example, FRET biosensors exist to measure the concentration and location of $\mathrm{Ca}^{2+}$ and the activation state of GTPase switch proteins (see Figure 3-32).

## Super-Resolution Microscopy Can Localize Proteins to Nanometer Accuracy

As we discussed earlier, the theoretical resolution limit of the fluorescence microscope is about $0.2 \mu \mathrm{~m}(200 \mathrm{~nm})$. Two new general approaches, collectively known as superresolution microscopy, have been developed to get around this limitation. In the first type of approach, the illuminating light is patterned in such a way that higher-resolution images are obtained. In structured illumination microscopy (SIM), the specimen is illuminated with a pattern of light and
(a)
![li225857775e0](li225857775e0.jpg)
(b)
![li226857775e0](li226857775e0.jpg)

EXPERIMENTAL FIGURE 4-25 FRET biosensors can detect local biochemical environments. (a) A FRET biosensor is a fusion protein containing two fluorescent proteins linked by a region sensitive to the environment under study. In this example, a protein construct consists of CFP linked to YFP by a region that contains a particular sequence that can be phosphorylated by a specific kinase (the "sensor domain") and a region (the "ligand domain") that binds the sensor domain when it is phosphorylated. In the absence of kinase activity, the two fluorescent proteins are too far apart to undergo FRET, whereas when locally phosphorylated by the active kinase, the sensor domain becomes phosphorylated, the ligand domain binds to it, and CFP and YFP are brought sufficiently close to undergo FRET. The sensor can also be deactivated when it encounters the appropriate phosphatase that removes the added phosphate. Thus the biosensor reports on the ratio of kinase to phosphatase activity in the local environment. (b) An example of the use of a FRET biosensor for protein kinase A, which is activated by elevation of cAMP. In this example, forskolin, a drug that induces the generation of cAMP, was added to cells at $t=0$ and the images collected at various times thereafter. Imaging shows both the rate of activation and localization of the active kinase. [Part (b) from J. Zhang et al., 2001, Proc. Natl. Acad. Sci. USA 98:14997-15005. © 2001 National Academy of Sciences, U.S.A.]
dark stripes, and several images are taken as the illuminating pattern is rotated to generate Moiré fringes. Computational analysis of the images gives a resolution of about 100 nm , twice that of a conventional confocal microscope, as can be seen in a micrograph of part of the nuclear envelope (Figure 4-26a). SIM is especially good for live cell imaging, as three-dimensional images can be collected every 4 seconds. In stimulated emission depletion (STED) microscopy, the sample is scanned just as in point-scanning microscopy, but with an important difference: the focused excitation laser point is surrounded by a donut-shaped "depletion beam," which effectively makes the area excited much smaller. Since the computer records the precise position of the spot excited and can record the emission from it, it can build up an impressive image, optimally yielding a resolution of 30 nm , greatly enhancing the detail one can see, for example, in an image of actin fibers (Figure 4-26b).

(a) Structured illumination microscopy (SIM)
![li227857775e0](li227857775e0.jpg)

C
Collect images with
illumination pattern in different orientations
![li228857775e0](li228857775e0.jpg)

Generate interference patterns that can be computationally reconstructed to create an image
(b) Stimulated emission depletion microscopy (STED)
![li229857775e0](li229857775e0.jpg)

Excitation laser point surrounded by donut-shaped "depletion beam," making area excited much smaller
(c) Photo-activated localization microscopy (PALM)
![li230857775e0](li230857775e0.jpg)

Collect thousands of images, in each of which only a few GFP molecules are excited
![li231857775e0](li231857775e0.jpg)

Point-scanning image
![li232857775e0](li232857775e0.jpg)

PALM image
![li233857775e0](li233857775e0.jpg)
the right shows part of a cell stained for actin fibers after imaging by point-scanning microscopy and by STED. (c) In photoactivated localization microscopy (PALM), use is made of a variant of GFP that can be photoactivated by a wavelength different from its excitation wavelength. When a small number of GFP molecules are activated, and then excited, each will emit thousands of photons that can be collected. This generates a Gaussian curve centered on the location of the emitting GFP; the center provides the location of the GFP to nanometer accuracy. This process is reiterated hundreds of times to excite other GFP molecules, and a high-resolution image emerges. At the right, a confocal image of microtubules is compared with a corresponding super-resolution image in which the three-dimensional arrangement of the microtubules is color coded. [Micrographs in (a) from Schermelleh et al., 2008, Science 320:1332-1336.; in (b) from Dr. Elise Stanley, Toronto Western Research Institute; (c) from B. Huang et al., 2008, Science 319:810-813.]

The second general approach uses single-molecule detection and localization. To understand how this works, consider two fluorescent spots separated by 75 nm . When you try to image them, they each generate a Gaussian distribution of fluorescence, which overlap so much that they look like one spot. However, if you could image each spot individually and find the center of each Gaussian curve, you could "beat" the resolution limit and detect the two spots 75 nm apart. One way to do this, called photoactivated localization microscopy (PALM), relies on the ability of a variant of GFP to be photoactivated; that is, it can become fluorescent only after being activated by a specific wavelength of light, different from its excitation wavelength. Consider what happens when we activate just one such GFP molecule. When we then excite the sample, the one activated GFP emits many hundreds of photons, giving rise to a Gaussian distribution (Figure 4-26c). Although analysis of each photon does not tell us precisely where the GFP is, the center of the distribution can tell us where the GFP is located with nanometer accuracy. If we now activate another GFP, we can localize it individually with the same precision. In PALM, a small percentage of GFPs are activated and each localized with high precision, and then another set is activated and localized, and as additional cycles of activation and localization are
![li234857775e0](li234857775e0.jpg)
recorded, a high-resolution image emerges. For example, the three-dimensional distribution of microtubules can be seen with much greater clarity than with conventional fluorescence microscopy (see Figure 4-26c). These types of images can take significant time to generate, so their use on samples of live cells is so far restricted. Nevertheless, because of the tremendous benefits of super-resolution microscopy, enormous efforts are being made to improve its sensitivity, speed, and spatial resolution, and we can expect rapid progress in the development of these approaches.

## Light-Sheet Microscopy Can Rapidly Image Cells in Living Tissue

Most of the confocal imaging approaches we have discussed above illuminate and detect fluorochromes through the same objective lens; this ensures that the excitation of the sample and the imaging of the emitted fluorescence occur in the same focal plane. As a result of this approach, there are limitations on the depth of sample that can be imaged. Very recently, a new technology has been developed to get around this limitation. In light-sheet microscopy, the sample is illuminated from the side and then viewed in an orthogonal direction (Figure 4-27a). A focused laser beam sweeps back and forth across the sample,
![li235857775e0](li235857775e0.jpg)

FIGURE 4-27 Light-sheet microscopy can image rapid events in living tissue. (a) In light-sheet microscopy, a tissue sample is illuminated from the side by a focused laser beam that scans the sample to generate a sheet of light. The sample is observed in the orthogonal direction though the detection objective. To get a three-dimensional image, the illuminating and detection objectives are moved coordinately, taking images throughout the depth of the sample. (b) The $\mathrm{Ca}^{2+}$ biosensor known as GCaMP. This biosensor is made using recombinant DNA techniques to generate a polypeptide to which the N - and C-termini of GFP are fused, and the middle interrupted. On one side of the interruption is the sensor domain, consisting of the $\mathrm{Ca}^{2+}$-binding protein calmodulin. On the other side is the ligand domain, consisting of a target sequence to which calmodulin will bind in the presence of $\mathrm{Ca}^{2+}$. In the absence of $\mathrm{Ca}^{2+}$, the GFP is not functional due to the interruption. In the presence of $\mathrm{Ca}^{2+}$, calmodulin binds four $\mathrm{Ca}^{2+}$ ions and undergoes a conformational change that allows it to bind the ligand domain. This conformational change brings the two parts of GFP into close proximity so that the fluorescent protein is functional. (c) $\mathrm{Ca}^{2+}$ transients, false-colored red, in cells of the brain of a living zebrafish. [Part (c) from M. B. Ahrens et al., 2013, Nature Methods 10:413-420.]

illuminating a single plane. A detection objective, at right angles to the illuminated plane, then images the sample. To generate a three-dimensional image, all the planes in the sample have to be imaged. This is achieved by coordinately stepping the illuminating sheet and detection objective through the sample to generate a stack of images, which can then be assembled computationally into a three-dimensional rendering.

In one example of the use of this technology, scientists have imaged concentrations of $\mathrm{Ca}^{2+}$ in the cells of a living zebrafish brain. The neurons in the brain of the zebrafish were made to express a $\mathrm{Ca}^{2+}$ biosensor, called GCaMP, which is based on GFP that does not fluoresce in the absence of $\mathrm{Ca}^{2+}$, but is designed to fluoresce in its presence (Figure 4-27b). This biosensor consists of GFP in which the N - and C-terminal domains are connected, but the middle of the protein is interrupted, and a $\mathrm{Ca}^{2+}$ sensor domain is attached to one end and a ligand domain to the other. Because the structure of GFP is interrupted, the protein no longer fluoresces when excited by the appropriate illuminating light. The sensor domain is derived from the small $\mathrm{Ca}^{2+}$-binding protein calmodulin (see Figure 3-31), which changes its conformation when it binds four $\mathrm{Ca}^{2+}$ ions. The ligand domain is a target sequence to which calmodulin binds only when it is activated by binding $\mathrm{Ca}^{2+}$. When active calmodulin binds the ligand domain, it changes the conformation of the interrupted GFP in such a way that it can now be excited and fluoresce. In this way, a fluorescent signal is reported whenever $\mathrm{Ca}^{2+}$ levels rise sufficiently to activate calmodulin. As we discuss in Chapter 22, when neurons communicate, part of this communication involves an elevation in $\mathrm{Ca}^{2+}$ levels. This neuronal communication can be nicely imaged by light-sheet microscopy in vivo in the brain of a zebrafish expressing the GCaMP biosensor (Figure 4-27c).

## KEY CONCEPTS OF SECTION 4.2

## Light Microscopy: Exploring Cell Structure and Visualizing Proteins Within Cells

- The resolution of the light microscope, about $0.2 \mu \mathrm{~m}$, is limited by the wavelength of light.
- Differences in refractive index can be used to observe parts of single cells by employing phase-contrast and direct-interference-contrast microscopy.
- Tissues generally have to be fixed, sectioned, and stained for cells and subcellular structures to be observed.
- Fluorescence microscopy makes use of compounds that absorb light at one wavelength and emit it at a longer wavelength.
- Ion-sensitive fluorescent dyes can measure intracellular concentrations of ions, such as $\mathrm{Ca}^{2+}$.
- Immunofluorescence microscopy makes use of antibodies to localize specific components in fixed and permeabilized cells.
- Indirect immunofluorescence microscopy uses an unlabeled primary antibody, followed by a fluorescently labeled secondary antibody that recognizes the primary one and allows it to be localized.
- Short sequences encoding epitope tags can be appended to protein-coding sequences to allow localization of the expressed protein using an antibody to the epitope tag.
- Green fluorescence protein (GFP) and its derivatives are naturally occurring fluorescent proteins.
- Fusing GFP to a protein of interest allows its localization and dynamics to be explored in a live cell.
- Deconvolution and confocal microscopy provide greatly improved clarity in fluorescent images by removing out-offocus fluorescent light.
- Total internal reflection fluorescence (TIRF) microscopy allows fluorescent samples adjacent to a coverslip to be seen with great clarity.
- Fluorescence recovery after photobleaching (FRAP) allows the dynamics of a population of molecules to be analyzed.
- Förster resonance energy transfer (FRET) is a technique in which light energy is transferred from one fluorescent protein to another when the proteins are very close, thereby revealing when two molecules are close in the cell.
- Super-resolution microscopy allows for detailed fluorescent images at nanometer resolution.
- Light-sheet microscopy can provide fluorescent images of thick samples by illuminating the sample with a sheet of light from the side.


## 4.3 Electron Microscopy: High-Resolution Imaging

Electron microscopic imaging of biological samples, such as single proteins, organelles, cells, and tissues, offers a much higher resolution of ultrastructure than can be obtained by light microscopy. The short wavelength of electrons means that the limit of resolution for a transmission electron microscope is theoretically 0.005 nm (much less than the diameter of a single atom), or 40,000 times better than that of a light microscope and 2 million times better than that of the unaided human eye. However, the effective resolution of the transmission electron microscope in the study of biological systems is considerably less than this ideal. Under optimal conditions, a resolution of 0.10 nm can be obtained with transmission electron microscopes, about 2000 times better than with conventional light microscopes and at least 300 times better than with the current super-resolution microscopes.

The fundamental principles of electron microscopy are similar to those of light microscopy; the major difference is that in electron microscopes, electromagnetic lenses focus a high-velocity electron beam instead of the visible light used by

optical lenses. In the transmission electron microscope (TEM), electrons are emitted from a filament and accelerated in an electric field (Figure 4-28, left). A condenser lens focuses the electron beam onto the sample; objective and projector lenses focus the electrons that pass through the specimen and project them onto a viewing screen or other detector. Because atoms in air absorb electrons, the entire tube between the electron source and the detector is maintained under an ultrahigh vacuum. Thus living material cannot be imaged by electron microscopy.

In this section, we describe various approaches to viewing biological material by electron microscopy. The most widely used instrument is the transmission electron microscope, but also in common use is the scanning electron microscope (SEM), which provides complementary information, as we discuss at the end of this section (Figure 4-28, right).
![li236857775e0](li236857775e0.jpg)

FIGURE 4-28 In electron microscopy, images are formed from electrons that pass through a specimen or are scattered from a metal-coated specimen. In a transmission electron microscope (TEM, left), electrons are extracted from a heated filament, accelerated by an electric field, and focused on the specimen by a magnetic condenser lens. Electrons that pass through the specimen are focused by a series of magnetic objective and projector lenses to form a magnified image of the specimen on a detector, which may be a fluorescent viewing screen, a photographic film, or a charged-couple-device (CCD) camera. In a scanning electron microscope (SEM), electrons are focused by condenser and objective lenses on a metal-coated specimen. Scanning coils move the beam across the specimen, and electrons scattered from the metal are collected by a photomultiplier tube detector. In both types of microscopes, because electrons are easily scattered by air molecules, the entire column is maintained at a very high vacuum.

## Single Molecules or Structures Can Be Imaged Using a Negative Stain or Metal Shadowing

It is common in biology to explore the detailed shapes of single macromolecules, such as proteins or nucleic acids, or of structures, such as viruses and the filaments that make up the cytoskeleton. It is relatively easy to view these objects in the transmission electron microscope, provided they are stained with a heavy metal that scatters the incident electrons. To prepare a sample, it is first absorbed to a 3-mm electron microscope grid (Figure 4-29a), which is
![li237857775e0](li237857775e0.jpg)
(c)
![li238857775e0](li238857775e0.jpg)

FIGURE 4-29 Transmission electron microscopy of negatively stained samples reveals fine features. (a) Samples for transmission electron microscopy (TEM) are usually mounted on a small copper or gold grid. The grid is usually covered with a very thin film of plastic and carbon to which a sample can adhere. (b) The specimen is then incubated in a heavy metal, such as uranyl acetate, and excess stain is removed. (c) The sample excludes the stain, so when it is observed in the TEM, it is seen in negative outline. The example in (c) is a negative stain of rotaviruses. [Part (c) ISM/Phototake.]

coated with a thin film of plastic and carbon. The sample is then bathed in a solution of a heavy metal, such as uranyl acetate, and excess solution is removed (Figure 4-29b). As a result of this procedure, the uranyl acetate coats the grid, but is excluded from the regions where the sample has adhered. When we view the sample in the TEM, we see where the stain has been excluded, so the sample is said to be negatively stained. Because the stain can precisely reveal the topology of the sample, a high-resolution image can be obtained (Figure 4-29c).

Samples can also be prepared by metal shadowing (Figure 4-30). In this technique, the sample is absorbed to a small piece of mica, then coated with a thin film of platinum by evaporation of the metal, then dissolved with acid or bleach, leaving the platinum coating (known as a replica). The platinum coating can be generated from a fixed angle or at a low angle as the sample is rotated, in which case it is called low-angle rotary shadowing. When the replica is transferred to a grid and examined in the TEM, it provides information about the three-dimensional topology of the sample.
![li239857775e0](li239857775e0.jpg)

FIGURE 4-30 Metal shadowing makes surface details on very small objects visible by transmission electron microscopy. (a) The sample is spread on a mica surface and then dried in a vacuum evaporator (step $\mathbf{\square}$ ). The sample grid is coated with a thin film of a heavy metal, such as platinum or gold, evaporated from an electrically heated metal filament (step $\mathbf{\square}$ ). To stabilize the replica, the specimen is then coated with a carbon film evaporated from an overhead elec-

## Cells and Tissues Are Cut into Thin Sections for Viewing by Electron Microscopy

Single cells and pieces of tissue are too thick to be viewed directly in the standard transmission electron microscope. To overcome this problem, methods were developed to prepare and cut thin sections of cells and tissues. When these sections were examined in the electron microscope, the organization, beauty, and complexity of the cell interior was revealed and led to a revolution in cell biology-for the first time, new organelles and the first glimpses of the cytoskeleton were seen.

To prepare thin sections, it is necessary to chemically fix the sample, dehydrate it, impregnate it with a liquid plastic that hardens (similar to Plexiglas), and then cut sections of about 5 to 100 nm in thickness. For structures to be seen, the sample has to be stained with heavy metals such as uranium and lead salts, which can be done either before embedding in the plastic or after sections are cut. Examples of cells and tissues viewed by thin-section electron microscopy appear here and throughout this book (Figure 4-31). It is important
![li240857775e0](li240857775e0.jpg)
trode (step $\mathbf{\square}$ ). The biological material is then dissolved by acid and bleach (step $\mathbf{\square}$ ), and the remaining metal replica is viewed in a TEM. In electron micrographs of such preparations, the carbon-coated areas appear light-the reverse of micrographs of simple metal-stained preparations, in which the areas of heaviest metal staining appear the darkest. (b) A platinum-shadowed replica of poliovirus particles. [Part (b) Science Source]

![li241857775e0](li241857775e0.jpg)

FIGURE 4-31 Example of a thin section viewed by transmission electron microscopy. Section through a pancreatic cell showing the extensive rough endoplasmic reticulum involved in the synthesis and secretion of digestive enzymes. [Keith R. Porter Archive, University of Maryland, Baltimore County.]
to realize that the images obtained represent just a thin slice through a cell, so to get a three-dimensional view, it is necessary to cut serial sections through the sample and reconstruct the sample from a series of sequential images (Figure 4-32).

## Immunoelectron Microscopy Localizes Proteins at the Ultrastructural Level

Just as immunofluorescence microscopy is used for localizing proteins at the light-microscope level, methods have been developed to use antibodies to localize proteins in thin sections at the electron microscope level. However, the harsh procedures used to prepare traditional thin sections-chemical fixation and embedding in plastic-can denature or modify the antigens so that they are no longer recognized by specific antibodies.
![li242857775e0](li242857775e0.jpg)

FIGURE 4-32 Model of the Golgi complex based on threedimensional reconstruction of electron microscopy images. Transport vesicles (white spheres) that have budded off the rough ER fuse with the cis membranes (light blue) of the Golgi complex. By mechanisms described in Chapter 14, proteins move from the cis region to the medial region and finally to the trans region of the Golgi complex. Eventually, vesicles bud off the trans-Golgi membranes (orange and red); some move to the cell surface and others move to lysosomes. The Golgi complex, like the rough endoplasmic reticulum, is especially prominent in secretory cells. [B. J. Marsh \& K. E. Howell, 2002, Nature Rev. Mol. Cell Biol. 3789-795.]

Gentler methods, such as light fixation, sectioning of material after freezing at the temperature of liquid nitrogen, and finally incubation with antibody at room temperature, have been developed. To make the antibody visible in the electron microscope, it must be attached to an electron-dense marker. One way to do this is to use electron-dense gold particles coated with protein A, a bacterial protein that binds the Fc segment of all antibody molecules (Figure 4-33). Because the gold particles diffract incident electrons, they appear as dark spots.
(a)
![li243857775e0](li243857775e0.jpg)
(b)
![li244857775e0](li244857775e0.jpg)

FIGURE 4-33 Gold particles coated with protein A are used to detect an antibody-bound protein by transmission electron microscopy. (a) First, antibodies are allowed to interact with their specific antigen in a section of fixed tissue. Then the section is treated with electron-dense gold particles coated with protein A from the bacterium S. aureus. Binding of the bound protein A to the Fc domains of the antibody molecules makes the location of the target protein visible in the electron microscope. (b) HIV particle budding from an infected HeLa cell. A cryosection of the specimen was prepared and first incubated with an antibody to capsid protein, then with protein A-coated $5-\mathrm{nm}$ gold particles to localize the internal capsid protein. The unoccupied sites in the protein A were inactivated, and the specimen was incubated with antibody to the membrane-bound Env protein, followed by protein A-coated $10-\mathrm{nm}$ gold particles. The distinct localization of the $5-\mathrm{nm}$ gold labeling the capsid protein and the $10-\mathrm{nm}$ gold labeling the Env protein can be seen. Scale bar is 100 nm . [Courtesy of Annegret Pelchen-Matthews and Mark Marsh, MRC-Laboratory for Molecular Cell Biology, University College London, UK.]

## Cryoelectron Microscopy Allows Visualization of Specimens Without Fixation or Staining

Standard transmission electron microscopy cannot be used to study live cells, and the absence of water in samples causes macromolecules to become denatured and nonfunctional. However, hydrated, unfixed, and unstained biological specimens can be viewed directly in a transmission electron microscope if the samples are frozen. In cryoelectron microscopy, an aqueous suspension of a sample is applied to a grid in an extremely thin film, frozen in liquid nitrogen, and maintained in this state by means of a special mount. The frozen sample is then placed in the electron microscope. The very low temperature $\left(-196^{\circ} \mathrm{C}\right)$ keeps water from evaporating,
![li245857775e0](li245857775e0.jpg)
(b)
![li246857775e0](li246857775e0.jpg)
(e)
![li247857775e0](li247857775e0.jpg)

FIGURE 4-34 Structure of the nuclear pore complex (NPC) imaged by cryoelectron tomography. (a) In cryoelectron tomography, a semicircular series of two-dimensional projection images is recorded from the three-dimensional specimen that is located at the center; the specimen is tilted while the electron optics and detector remain stationary. The three-dimensional structure is then computed from the collected two-dimensional images. (b) Isolated nuclei from the cellular slime mold Dictyostelium discoideum were quick-frozen in liquid nitrogen and maintained in this state as the sample was observed in the electron
even in a vacuum. Thus the sample can be observed in detail in its native, hydrated state without fixing or heavy metal staining. By computer-based averaging of hundreds of images, a three-dimensional model can be generated almost to atomic resolution. For example, this method has been used to generate models of ribosomes, the muscle calcium pump discussed in Chapter 11, and the three-dimensional structure of a poliovirus capsid (see Figure 5-43b).

An extension of this technique, cryoelectron tomography, allows researchers to determine the three-dimensional architecture of organelles or even whole cells embedded in ice; that is, in a state close to life. A single picture is a two-dimensional representation of a structure that lacks depth information. However, looking at the same structure from different angles gives us a three-dimensional perspective. In cryoelectron tomography, the specimen holder is tilted in small increments around the axis perpendicular to the electron beam; thus images of the object viewed from different directions are obtained (Figure 4-34a, b). The images are then merged computationally
(d)
![li248857775e0](li248857775e0.jpg)
microscope. The panel shows three sequential tilted images. Different orientations of NPCs (arrows) are shown in top view (left and center) and side view (right). Ribosomes connected to the outer nuclear membrane are visible, as is a patch of rough ER (arrowheads). (c) Computergenerated surface-rendered representation of a segment of the nuclear envelope membrane (yellow) studded with NPCs (blue). (d) By averaging the images of multiple nuclear pores, much more detail can be discerned. See S. Nickell et al., 2006, Nature Rev. Mol. Cell Biol. 7:225. [Parts (b), (c), and (d) from: M. Beck et al., 2004, Science 306:1387]

into a three-dimensional reconstruction termed a tomogram (Figure 4-34c, d). A disadvantage of cryoelectron tomography is that the samples must be relatively thin, about 200 nm ; this is much thinner than the samples ( $200 \mu \mathrm{~m}$ thick) that can be studied by confocal light microscopy.

## Scanning Electron Microscopy of Metal-Coated Specimens Reveals Surface Features

Scanning electron microscopy (SEM) allows investigators to view the surfaces of unsectioned metal-coated specimens. An intense electron beam inside the microscope scans rapidly over the sample. Molecules in the coating are excited and release secondary electrons that are focused onto a scintillation detector; the resulting signal is displayed on a cathode-ray tube much like a conventional television (see Figure 4-28, right). The resulting scanning electron micrograph has a three-dimensional appearance because the number of secondary electrons produced by any one point on the sample depends on the angle of the electron beam in relation to the surface (Figure 4-35). The resolving power of scanning electron microscopes, which is limited by the thickness of the metal coating, is only about 10 nm , much less than that of transmission instruments.

## KEY CONCEPTS OF SECTION 4.3

## Electron Microscopy: High-Resolution Imaging

- Electron microscopy provides very high-resolution images because of the short wavelength of the high-energy electrons used to image the sample.
- Simple specimens, such as proteins or viruses, can be negatively stained or shadowed with heavy metals for examination in a transmission electron microscope (TEM).
- Thicker sections generally must be fixed, dehydrated, embedded in plastic, sectioned, and then stained with electrondense heavy metals before viewing by TEM.
- Specific proteins can be localized by TEM by employing specific antibodies associated with a heavy metal marker, such as small gold particles.
- Cryoelectron microscopy allows examination of hydrated, unfixed, and unstained biological specimens in the TEM by maintaining them at very low temperatures.
- Scanning electron microscopy (SEM) of metal-shadowed material reveals the surface features of specimens.


### 4.4 Isolation of Cell Organelles

The examination of cells by light and electron microscopy led to the appreciation that eukaryotic cells contain a common set of organelles, introduced in Chapter 1 (see Figure 1-12a). However, observing organelles and documenting their detailed structure by microscopy does not clearly reveal the roles they play and how they work. For this, it is necessary to isolate organelles in their native state and identify and dissect the function of each component. For this reason, methods to isolate and characterize organelles were developed in parallel with advances in microscopy. Lysosomes, for example, are organelles in which biological molecules are degraded, as described in Chapter 1. Lysosomes had been seen by microscopy, but their function was discovered only after a method was developed to isolate
![li249857775e0](li249857775e0.jpg)

FIGURE 4-35 Scanning electron microscopy (SEM) produces a threedimensional image of the surface of an unsectioned specimen. Seen here is an SEM image of cells of the trachea. In the middle is a goblet cell, which secretes mucus. On either side of the goblet cell are epithelial cells with abundant cilia on their apical surfaces. [Steve Gschmeissner/Science Source.]

them. When a method is developed to purify a type of organelle, it is possible to begin to catalogue all of its components and probe the function of each one. In another example, electron microscopy revealed that some ribosomes are associated with the endoplasmic reticulum, suggesting that the ER is a site of protein synthesis. We now know that proteins to be secreted from the cell are made by these ribosomes, and that the nascent secretory protein is translocated across the membrane into the lumen of the endoplasmic reticulum. As we describe in Chapter 13, understanding the mechanism by which this occurs depended on the isolation of the endoplasmic reticulum and the development of in vitro assays for the synthesis and translocation of secretory proteins. Thus, before one can begin to fully understand organelles, biochemical assays need to be established to probe the functions of each component, with the eventual goal of reconstituting functional organelles from purified components.

Most organelles are enclosed in a lipid bilayer and perform a specific function. Each type of organelle has a recognizable structure and contains a specific set of proteins to perform its function. Cell biologists use this fact to identify specific organelles. For example, as discussed in Chapter 12, most of the ATP in a cell is made by ATP synthase, which converts ADP to ATP in mitochondria, so ATP synthase is a good marker for mitochondria. As we will discuss below, the availability of specific markers for organelles has helped in the development of organelle purification.

In this section, we discuss methods that are used to open up cells for the purification of organelles. We end with recent advances in proteomics aimed at defining the complete protein inventories of organelles.

## Disruption of Cells Releases Their Organelles and Other Contents

The initial step in purifying subcellular structures is to release the cell's contents by rupturing the plasma membrane and the cell wall, if present. To do this, the cells are suspended in a solution of appropriate pH and salt content, usually isotonic sucrose $(0.25 \mathrm{M})$ or a combination of salts similar in composition to those in the cell's interior. Many cells can then be broken by stirring the cell suspension in a high-speed blender or by exposing it to ultrahigh-frequency sound (sonication). Alternatively, plasma membranes can be sheared by special pressurized tissue homogenizers in which cells are forced through a very narrow space between a plunger and the wall of the vessel; the pressure of being forced between the vessel wall and the plunger ruptures the cell.

Recall that water flows into cells when they are placed in a hypotonic solution; that is, one with a lower concentration of ions and small molecules than is found inside the cell. This osmotic flow can be used to cause cells to swell, weakening the plasma membrane and facilitating its rupture. Generally the cell solution is best kept at $0^{\circ} \mathrm{C}$ to preserve enzymes and other constituents after their release from the stabilizing forces of the cell.

Disrupting the cell produces a mix of suspended cellular components, the homogenate, from which the desired organelles can be retrieved. Because rat liver contains an abundance of a single cell type, this tissue has been used in many classic studies of cell organelles. However, the same isolation principles apply to virtually all cells and tissues, and modifications of these cell-fractionation techniques can be used to separate and purify any desired components.

## Centrifugation Can Separate Many Types of Organelles

In Chapter 3, we considered the principles of centrifugation and the uses of centrifugation techniques for separating proteins and nucleic acids. Similar approaches are used for separating and purifying various organelles, which differ in both size and density and thus undergo sedimentation at different rates.

Most cell-fractionation procedures begin with differential centrifugation of a filtered cell homogenate at increasingly higher speeds (Figure 4-36). After centrifugation at each speed for an appropriate time, the liquid that remains at the top of the vessel, called the supernatant, is poured off and centrifuged at higher speed. The pelleted fractions obtained by differential centrifugation generally contain a mixture of organelles, although nuclei and viral particles can sometimes be purified completely by this procedure.

An impure organelle fraction obtained by differential centrifugation can be further purified by equilibrium densitygradient centrifugation, which separates cellular components according to their density. After the fraction is resuspended, it is layered on top of a solution that contains a gradient of a dense non-ionic substance (e.g., sucrose or glycerol). The tube is centrifuged at a high speed (about $40,000 \mathrm{rpm}$ ) for several hours, allowing each particle to migrate to an equilibrium position where the density of the surrounding liquid is equal to the density of the particle (Figure 4-37). The different layers of the gradient are then recovered by pumping out the contents of the centrifuge tube through a narrow piece of tubing and collecting the fractions (see Classic Experiment 4-1).

Because each organelle has unique morphological features, the purity of organelle preparations can be assessed by examination in an electron microscope. Alternatively, organelle-specific marker molecules can be quantified. For example, the protein cytochrome $c$ is present only in mitochondria, so the presence of this protein in a fraction of lysosomes would indicate its contamination by mitochondria. Similarly, catalase is present only in peroxisomes; acid phosphatase, only in lysosomes; and ribosomes, only in the rough endoplasmic reticulum or the cytosol.

## Organelle-Specific Antibodies Are Useful in Preparing Highly Purified Organelles

Cell fractions remaining after differential and equilibrium density-gradient centrifugation usually contain more than

![li250857775e0](li250857775e0.jpg)

FIGURE 4-36 Differential centrifugation is a common first step in fractionating a cell homogenate. The homogenate that results from disrupting cells is usually filtered to remove unbroken cells and then centrifuged at a fairly low speed to selectively pellet the nuclei-the largest organelle. The undeposited material (the supernatant) is next centrifuged at a higher speed to sediment the mitochondria, chloroplasts, lysosomes, and peroxisomes. Subsequent centrifugation in an ultracentrifuge at $100,000 \mathrm{~g}$ for 60 minutes results in deposition of the plasma membrane, fragments of the endoplasmic reticulum, and large polyribosomes. The recovery of ribosomal subunits, small polyribosomes, and particles such as complexes of enzymes requires additional centrifugation at still higher speeds. Only the cytosol-the soluble aqueous part of the cytoplasmremains in the supernatant after centrifugation at $300,000 \mathrm{~g}$ for 2 hours.
![li251857775e0](li251857775e0.jpg)

FIGURE 4-37 A mixed-organelle fraction can be further separated by equilibrium density-gradient centrifugation. In this example, using rat liver, material in the pellet from centrifugation at $15,000 \mathrm{~g}$ (see Figure 4-36) is resuspended and layered on a gradient of increasingly dense sucrose solutions in a centrifuge tube. During centrifugation for several hours, each organelle migrates to its appropriate equilibrium density and remains there. To obtain a good separation of lysosomes from mitochondria, the liver is perfused with a solution containing a small amount of detergent before the tissue is disrupted. During this perfusion period, detergent is taken into the cells by endocytosis and transferred to the lysosomes, making them less dense than they would normally be and permitting a "clean" separation of lysosomes from mitochondria.
one type of organelle. Monoclonal antibodies to various organelle-specific membrane proteins are a powerful tool for further purifying such fractions. One example is the purification of vesicles whose outer surface is covered with the protein clathrin; these coated vesicles are derived from coated pits at the plasma membrane during receptor-mediated endocytosis, a topic we will discuss in detail in Chapter 14. An antibody to clathrin, bound to a dead bacterial cell that expressed protein A on its surface, can selectively bind these vesicles in a crude preparation of membranes, and the whole antibody complex can then be isolated by low-speed centrifugation (Figure 4-38). A related technique uses tiny metallic beads coated with specific antibodies. Organelles that bind to the antibodies, and are thus linked to the metallic beads, are recovered from the preparation by adhesion to a small magnet on the side of the test tube.

All cells contain a dozen or more different types of small membrane-limited vesicles of about the same size (50-100 nm in diameter) and density, which makes them difficult to separate from one another by centrifugation techniques. Immunological techniques are particularly useful for purifying specific classes of such vesicles. Fat and muscle cells, for instance, contain a particular glucose transporter (GLUT4) that is localized to the membrane of one of these vesicle types. When insulin is added to the cells, these vesicles fuse with the plasma membrane and increase the number of glucose transporters able to take up glucose from the blood. As we will see in Chapter 15, this process is critical to maintaining the appropriate concentration of sugar in the blood. The GLUT4-containing vesicles

![li252857775e0](li252857775e0.jpg)

FIGURE 4-38 Coated vesicles can be purified by binding to an antibody specific for a vesicle surface protein and linkage to bacterial cells. In this example, a suspension of membranes from rat liver is incubated with an antibody specific for clathrin, a protein that coats the outer surface of certain cytosolic vesicles. To this mixture is added a suspension of killed Staphylococcus aureus bacteria, whose surface membrane contains protein A, which binds to the constant ( Fc )
can be purified by using an antibody that binds to a segment of the GLUT4 protein that faces the cytosol. Likewise, the various transport vesicles discussed in Chapter 14 are characterized by unique surface proteins that permit their separation with the aid of specific antibodies.

A variation of this technique is employed when no antibody specific for the organelle under study is available. A gene encoding an organelle-specific membrane protein is modified by the addition of a segment encoding an epitope tag; the tag is placed on a segment of the protein that faces the cytosol. Following stable expression of the recombinant protein in the cell under study, an anti-epitope monoclonal antibody (described above) can be used to purify the organelle.

## Proteomics Reveals the Protein Composition of Organelles

We introduced this section by emphasizing how important it is to isolate organelles to identify their components. For many years, the technology was not sufficiently sophisticated to generate a complete inventory of the proteins in each organelle, but recent advances in genomics and mass
![li253857775e0](li253857775e0.jpg)
region of antibodies. (a) Interaction of protein A with antibodies bound to clathrin-coated vesicles links the vesicles to the bacterial cells. The vesicle-bacteria complexes can then be recovered by low-speed centrifugation. (b) A thin-section electron micrograph reveals clathrincoated vesicles bound to an S. aureus cell. See E. Merisko et al., 1982, J. Cell Biol. 93:846. [Micrograph courtesy of George Palade.]
spectrometry have now made it possible. This approach combines organelle isolation with the proteomics techniques discussed in Chapter 3.

Identifying all the proteins in an organelle requires three steps. First, one has to be able to isolate the organelle to a high degree of purity. Second, one has to have a way to identify all the sequences of the proteins in the organelle. This identification is generally done by digesting all the proteins with a protease such as trypsin, which cleaves all polypeptides at lysine and arginine residues, and then determining the mass and sequence of all the resulting peptides by mass spectrometry. Third, one has to have a genomic sequence to identify the proteins from which all the peptides came. In this way, the proteomes of many organelles have been determined. As one example, a recent proteomic study on mitochondria purified from mouse brain, heart, kidney, and liver revealed 591 mitochondrial proteins, including 163 proteins not previously known to be associated with this organelle. Several proteins were found in mitochondria only from specific cell types. Determining the functions associated with these newly identified mitochondrial proteins is a major objective of current research on this organelle.

## KEY CONCEPTS OF SECTION 4.4

## Isolation of Cell Organelles

- Microscopy has revealed a common set of organelles that are present in eukaryotic cells.
- Disruption of cells by vigorous homogenization, sonication, or other techniques releases their organelles. Swelling of cells in a hypotonic solution weakens the plasma membrane, making it easier to rupture.
- Sequential differential centrifugation of a cell homogenate yields fractions of partly purified organelles that differ in mass and density.
- Equilibrium density-gradient centrifugation, which separates cellular components according to their densities, can further purify cell fractions obtained by differential centrifugation.
- Immunological techniques using antibodies against organelle-specific membrane proteins are particularly useful in purifying organelles and vesicles of similar sizes and densities.
- Proteomic analysis can identify all the protein components in a preparation of a purified organelle.


## LaunchPad

macmillan learning

Visit LaunchPad to access study tools and to learn more about the content in this chapter:

- Perspectives for the Future
- Analyze the Data
- Classic Experiment 4-1: Separating Organelles
- Extended References
- Additional study tools, including videos, animations, and quizzes


## Key Terms

| bright-field |  |
| :-- | :-- |
| light microscopy | 141 |
| cell line | 132 |
| cell strain | 131 |
| chimeric proteins | 130 |
| clone | 129 |
| confocal microscopy | 147 |
| cryoelectron |  |
| microscopy | 160 |

culturing 129
deconvolution microscopy 147
differential
centrifugation 162
differential-interferencecontrast (DIC)
microscopy 141
equilibrium density-gradient centrifugation 162
fluorescence-activated cell sorter (FACS) 132
fluorescence recovery after photobleaching (FRAP) 151
fluorescent staining 143
Förster resonance energy transfer (FRET) 152
hybridoma 135
immunofluorescence microscopy 130
indirect immunofluorescence microscopy 145
metal shadowing 158
monoclonal antibody 135
phase-contrast microscopy 141
photoactivated
localization microscopy
(PALM) 155
polyclonal antibody 144
resolution 141
scanning electron microscope
(SEM) 157
total internal reflection
fluorescence (TIRF)
microscopy 150
transmission electron microscope (TEM) 157

# Review the Concepts 

1. Both light and electron microscopy are commonly used to visualize cells, cell structures, and the location of specific molecules. Explain why a scientist may choose one or the other microscopy technique for use in research.
2. The magnification possible with any type of microscope is an important property, but its resolution, the ability to distinguish between two very closely apposed objects, is even more critical. Describe why the resolving power of a microscope is more important for seeing finer details than its magnification. What is the formula used to describe the resolution of a microscope lens, and what are the limitations placed on the values in the formula?
3. Why are chemical stains required for visualizing cells and tissues with the basic light microscope? What advantage do fluorescent dyes and fluorescence microscopy provide in comparison to the chemical dyes used to stain specimens for light microscopy? What advantages do confocal and deconvolution microscopy provide in comparison to conventional fluorescence microscopy?
4. In certain electron microscopy methods, the specimen is not directly imaged. How do these methods provide information about cellular structure, and what types of structures do they visualize? What limitation applies to most forms of electron microscopy?
5. What is the difference between a cell strain, a cell line, and a clone?
6. Explain why the process of cell fusion is necessary to produce monoclonal antibodies used for research.
7. Much of what we know about cell function depends on experiments using specific cells and specific parts (e.g., organelles) of cells. What techniques do scientists commonly use to isolate cells and organelles from complex mixtures, and how do these techniques work?
8. Hoechst 33258 is a chemical dye that binds specifically to DNA in live cells, and when excited by UV light, it fluoresces

in the visible spectrum. Name and describe one specific method, employing Hoechst 33258, an investigator would use to isolate fibroblasts in the $\mathrm{G}_{2}$ phase of the cell cycle from those fibroblasts in interphase.

## References

## Growing and Studying Cells in Culture

Bissell, M. J., A. Rizki, and I. S. Mian. 2003. Tissue architecture: the ultimate regulator of breast epithelial function. Curr. Opin. Cell Biol. 15:753-762.

Davis, J. M., ed. 1994. Basic Cell Culture: A Practical Approach. IRL Press.

Edwards, B., et al. 2004. Flow cytometry for high-throughput, high-content screening. Curr. Opin. Chem. Biol. 8:392-398.

Eggert, U. S., and T. J. Mitchison. 2006. Small molecule screening by imaging. Curr. Opin. Chem. Biol. 10:232-237.

Goding, J. W. 1996. Monoclonal Antibodies: Principles and Practice. Production and Application of Monoclonal Antibodies in Cell Biology, Biochemistry, and Immunology, 3d ed. Academic Press.

Griffith, L. G., and M. A. Swartz. 2006. Capturing complex 3D tissue physiology in vitro. Nature Rev. Mol. Cell Biol. 7:211-224.

Herzenberg, L. A., et al. 2002. The future and history of fluorescence activated cell sorter and flow cytometry: a view from Stanford. Clin. Chem. 48:1819-1827.

## Light Microscopy: Exploring Cell Structure and Visualizing Proteins Within Cells

Egner, A., and S. Hell. 2005. Fluorescence microscopy with super-resolved optical sections. Trends Cell Biol. 15:207-215.

Giepmans, B. N. G., et al. 2006. The fluorescent toolbox for assessing protein location and function. Science 312:217-224.

Huang, B., H. Babcock, and X. Zhuang. 2010. Breaking the diffraction barrier: super-resolution imaging of cells. Cell 143: 1047-1058.

Inoué, S., and K. Spring. 1997. Video Microscopy, 2d ed. Plenum Press.

Komatsu, N., et al. 2011. Development of an optimized backbone of FRET biosensors for kinases and GTPases. Mol. Biol. Cell 22:4647-4656.

Lakadamyali, M. 2014. Super-resolution microscopy: going live and going fast. Chem. Phys. Chem. 15:630636.

Lippincott-Schwartz, J. 2010. Imaging: visualizing the possibilities. J. Cell Sci. 123:3619-3620.

Lippincott-Schwartz, J. 2011. Emerging in vivo analyses of cell function using fluorescence imaging. Ann. Rev. Biochem. 80:327-332.

Matsumoto, B., ed. 2002. Methods in Cell Biology. Vol. 70: Cell Biological Applications of Confocal Microscopy. Academic Press.

Mayer, T. U., et al. 1999. Small molecule inhibitor of mitotic spindle bipolarity identified in a phenotype-based screen. Science 286:971-974.

Mayor, S., and S. Bilgrami. 2007. Fretting about FRET in cell and structural biology. In D. Zuk, ed., Evaluating Techniques in Biochemical Research. Cell Press.

Newman, R. H., M. Fosbrink, and J. Zhang. 2011. Genetically encoded fluorescent biosensors for tracking signaling dynamics in living cells. Chem. Biol. 115:3614-3666.

Roukos, V., T. Misteli, and C. K. Schmidt. 2010. Descriptive no more: the dawn of high-throughput microscopy. Trends Cell Biol. 20:503-506.

Schermelleh, L., R. Heintzmann, and H. Leonhardt. 2010. A guide to super-resolution fluorescence microscopy. J. Cell Biol. 190:165-175.

Tsien, R. Y. 2009. Indicators based on fluorescence resonance energy transfer (FRET). Cold Spring Harbor Protoc., doi:10.1101/ pdb.top57.

Weigert, R., N. Porat-Shliom, and P. Amornphimoltham. 2013. Imaging cell biology in live animals: ready for prime time. J. Cell Biol. 201:969-979.

Zipfel, W. R., R. M. Williams, and W. W. Webb. 2003. Nonlinear magic: multiphoton microscopy in the biosciences. Nature Biotechnol. 21:1369-1377.

## Electron Microscopy: High-Resolution Imaging

Beck, M., et al. 2004. Nuclear pore complex structure and dynamics revealed by cryoelectron tomography. Science 306: $1387-1390$.

Frey, T. G., G. A. Perkins, and M. H. Ellisman. 2006. Electron tomography of membrane-bound cellular organelles. Ann. Rev. Biophys. Biomol. Struc. 35:199-224.

Hyatt, M. A. 2000. Principles and Techniques of Electron Microscopy, 4th ed. Cambridge University Press.

Koster, A., and J. Klumperman. 2003. Electron microscopy in cell biology: integrating structure and function. Nature Rev. Mol. Cell Biol. 4:SS6-SS10.

## Isolation of Cell Organelles

Bainton, D. 1981. The discovery of lysosomes. J. Cell Biol. 91:66s-76s.
de Duve, C. 1975. Exploring cells with a centrifuge. Science 189:186-194. The Nobel Prize lecture of a pioneer in the study of cellular organelles.
de Duve, C., and H. Beaufay. 1981. A short history of tissue fractionation. J. Cell Biol. 91:293s-299s.

Foster, L. J., et al. 2006. A mammalian organelle map by protein correlation profiling. Cell 125:187-199.

Holtzman, E. 1989. Lysosomes. Plenum Press.
Palade, G. 1975. Intracellular aspects of the process of protein synthesis. Science 189:347-358. The Nobel Prize lecture of a pioneer in the study of cellular organelles.

![li254857775e0](li254857775e0.jpg)

Colored transmission electron micrograph of one ribosomal RNA transcription unit from a Xenopus oocyte. Transcription proceeds from left to right, with nascent ribosomal ribonucleoprotein complexes (rRNPs) growing in length as each successive RNA polymerase I molecule moves along the DNA template at the center. In this preparation, each rRNP is oriented either above or below the central strand of DNA being transcribed, so that the overall shape is similar to a feather. In the nucleolus of a living cell, the nascent rRNPs extend in all directions, like a bottlebrush. (Professor Oscar L. Miller/Science Photo Library.)

The extraordinary versatility of proteins as the components of cellular structures, cellular catalysts, and molecular switches and machines was described in Chapter 3. In this chapter, we consider the process by which proteins are made as well as other cellular processes that are critical for the survival of an organism and its descendants. Our focus will be on the vital molecules known as nucleic acids, and how they ultimately are responsible for governing all cellular function. As we saw in Chapter 2, nucleic acids are linear polymers of four types of nucleotides (see Figures 2-13, 2-16, and 2-17). These macromolecules (1) contain in the precise sequence of their nucleotides the information for determining the amino acid sequence, and hence the structure and function, of all the proteins of a cell; (2) are critical functional components of the cellular macromolecular factories

## FUdamental Molecular Genetic Mechanisms

that select amino acids and align them in the correct order as a polypeptide chain is being synthesized; (3) catalyze a number of fundamental chemical reactions in cells, including formation of peptide bonds between amino acids during protein synthesis; and (4) regulate the expression of genes.

Deoxyribonucleic acid (DNA) is an informational molecule that contains in the sequence of its nucleotides the information required to build all the proteins and RNAs of an organism, and hence the cells and tissues of that organism. Chemically, it is ideally suited to perform this function. It is extraordinarily stable under most terrestrial conditions, as exemplified by our ability to recover DNA sequence from bones and tissues that are tens of thousands of years old. Because of this, and because of the repair mechanisms that operate in living cells, the long polymers that make up a

### 5.1 Structure of Nucleic Acids

### 5.2 Transcription of Protein-Coding Genes and Formation of Functional mRNA

### 5.3 The Decoding of mRNA by tRNAs

5.4 Stepwise Synthesis of Proteins on Ribosomes

### 5.5 DNA Replication

### 5.6 DNA Repair and Recombination

5.7 Viruses: Parasites of the Cellular Genetic System