![li012239539b](li012239539b.jpg)

Colored transmission electron micrograph of one ribosomal RNA transcription unit from a Xenopus oocyte. Transcription proceeds from left to right, with nascent ribosomal ribonucleoprotein complexes (rRNPs) growing in length as each successive RNA polymerase I molecule moves along the DNA template at the center. In this preparation, each rRNP is oriented either above or below the central strand of DNA being transcribed, so that the overall shape is similar to a feather. In the nucleolus of a living cell, the nascent rRNPs extend in all directions, like a bottlebrush. (Professor Oscar L. Miller/Science Photo Library.)

The extraordinary versatility of proteins as the components of cellular structures, cellular catalysts, and molecular switches and machines was described in Chapter 3. In this chapter, we consider the process by which proteins are made as well as other cellular processes that are critical for the survival of an organism and its descendants. Our focus will be on the vital molecules known as nucleic acids, and how they ultimately are responsible for governing all cellular function. As we saw in Chapter 2, nucleic acids are linear polymers of four types of nucleotides (see Figures 2-13, 2-16, and 2-17). These macromolecules (1) contain in the precise sequence of their nucleotides the information for determining the amino acid sequence, and hence the structure and function, of all the proteins of a cell; (2) are critical functional components of the cellular macromolecular factories

## FUdamental Molecular Genetic Mechanisms

that select amino acids and align them in the correct order as a polypeptide chain is being synthesized; (3) catalyze a number of fundamental chemical reactions in cells, including formation of peptide bonds between amino acids during protein synthesis; and (4) regulate the expression of genes.

Deoxyribonucleic acid (DNA) is an informational molecule that contains in the sequence of its nucleotides the information required to build all the proteins and RNAs of an organism, and hence the cells and tissues of that organism. Chemically, it is ideally suited to perform this function. It is extraordinarily stable under most terrestrial conditions, as exemplified by our ability to recover DNA sequence from bones and tissues that are tens of thousands of years old. Because of this, and because of the repair mechanisms that operate in living cells, the long polymers that make up a

### 5.1 Structure of Nucleic Acids

### 5.2 Transcription of Protein-Coding Genes and Formation of Functional mRNA

### 5.3 The Decoding of mRNA by tRNAs

5.4 Stepwise Synthesis of Proteins on Ribosomes

### 5.5 DNA Replication

### 5.6 DNA Repair and Recombination

5.7 Viruses: Parasites of the Cellular Genetic System

DNA molecule can be up to $10^{9}$ nucleotides long. Virtually all the information required for the development of a fertilized human egg into an adult made of trillions of cells with specialized functions can be stored in the sequence of the four types of nucleotides that make up the roughly $3 \times 10^{9}$ base pairs in the human genome. Because of the principles of base pairing discussed in the following sections, this information is readily copied with an error rate of only about 1 in $2.5 \times 10^{8}$ nucleotides per generation. The exact replication of this information in any species ensures its genetic continuity from generation to generation and is critical to the normal development of individuals. DNA fulfills these functions so well that it is the vessel for genetic information in all known forms of life (excluding RNA viruses, which are limited to extremely short genomes because of the relative instability of RNA compared with DNA, as we will see). The discovery that virtually all forms of life use DNA to encode their genetic information and use a nearly identical genetic code implies that all forms of life descended from a common ancestor whose genetic information was stored in nucleic acid sequence. This information is accessed and replicated by specific base pairing between nucleotides. The information stored in DNA is arranged in hereditary units, known as genes, that control identifiable traits of an organism. In the process of transcription, the information stored in DNA is copied into ribonucleic acid (RNA), which has three distinct roles in protein synthesis, in addition to its more recently discovered functions in the regulation of chromatin structure, transcription, and protein synthesis, which we will discuss in Chapters 8,9 , and 10 .

Portions of the DNA nucleotide sequence are copied into messenger RNA (mRNA) molecules that direct the synthesis of a specific protein. The nucleotide sequence of an mRNA molecule contains information that specifies the correct order of amino acids during the synthesis of a protein. The remarkably accurate, stepwise assembly of amino acids into proteins occurs by translation of mRNA. In this process, the nucleotide sequence of an mRNA molecule is "read" by a second type of RNA called transfer RNA (tRNA) with the aid of a third type of RNA, ribosomal RNA (rRNA), and associated proteins. As the correct amino acids are brought into sequence by tRNAs, they are linked by peptide bonds to make proteins. RNA synthesis is called transcription because the four-base sequence "language" of DNA is precisely copied, or transcribed, into the nucleotide sequence of an RNA molecule. Protein synthesis is referred to as translation because the four-base sequence "language" of DNA and RNA is translated into the twenty-amino acid sequence "language" of proteins.

Discovery of the structure of DNA in 1953 and the subsequent elucidation of how DNA directs synthesis of RNA, which then directs assembly of proteins-the so-called central dogma-were monumental achievements marking the early days of molecular biology. However, the simplified representation of the central dogma as $\mathrm{DNA} \rightarrow \mathrm{RNA} \rightarrow$ protein does not reflect the role of proteins in the synthesis
of nucleic acids. Moreover, as discussed here for bacteria and in later chapters for eukaryotes, proteins are largely responsible for regulating gene expression, the entire process whereby the information encoded in DNA is decoded into proteins in the correct cells at the correct times in development. As a consequence, hemoglobin is expressed only in cells in the bone marrow (erythroid progenitors) destined to develop into circulating red blood cells (erythrocytes), and developing neurons make the proper synapses (connections) with $10^{11}$ other developing neurons in the human brain. The fundamental molecular genetic processes of DNA replication, transcription, and translation must be carried out with extraordinary fidelity, speed, and accurate regulation for the normal development of organisms as complex as bacteria, archaea, and eukaryotes (see Figure 1-1). This is achieved by chemical processes that operate with extraordinary accuracy coupled with multiple layers of checkpoint or surveillance mechanisms that test whether critical steps in these processes have occurred correctly before the next step is initiated. The highly regulated expression of genes necessary for the development of a multicellular organism requires integration of information from signals sent by distant cells in the developing organism, as well as from neighboring cells, and an intrinsic developmental program determined by earlier steps in embryogenesis taken by each cell's progenitors. All of this regulation is dependent on control sequences in the DNA that function with proteins called transcription factors to coordinate the expression of every gene. The RNA sequences we discuss in Chapters 8,9 , and 10 also serve to regulate chromatin structure, transcription, RNA processing, and translation. Nucleic acids function as the "brains and central nervous system" of the cell, while proteins carry out most of the functions they specify.

In this chapter, we first review the structures and properties of DNA and RNA and explore how the different characteristics of these two types of nucleic acids make them suited for their respective functions in the cell. In the next several sections, we discuss the basic processes summarized in Figure 5-1: transcription of DNA into RNA precursors, processing of these precursors to make functional RNA molecules, translation of mRNAs into proteins, and the replication of DNA. Proteins regulate cell structure and most of the biochemical reactions in cells, so we first consider how the amino acid sequences of proteins, which determine their three-dimensional structures and hence their functions, are encoded in DNA and translated. After outlining the functions of mRNA, tRNA, and rRNA in protein synthesis, we present a detailed description of the components and biochemical steps in translation. Understanding these processes gives us a deep appreciation of the need to copy the nucleotide sequence of DNA precisely. Consequently, we next consider the molecular problems involved in DNA replication and the complex cellular machinery that ensures accurate copying of the genetic material. Along the way, we compare these processes in prokaryotes and eukaryotes. The next section describes how damage to DNA is repaired and how

![li022239539b](li022239539b.jpg)

FIGURE 5-1 Overview of four basic molecular genetic processes. In this chapter, we cover the three processes that lead to production of proteins $\boldsymbol{\square}$ and the process for replicating DNA. Because viruses utilize host-cell machinery, they have been important models for studying these processes. During transcription of a protein-coding gene by RNA polymerase $\boldsymbol{\square}$, the four-base DNA code specifying the amino acid sequence of a protein is copied, or transcribed, into a precursor messenger RNA (pre-mRNA) by the polymerization of ribonucleoside triphosphate monomers (rNTPs). Removal of noncoding sequences and other modifications to the pre-mRNA $\boldsymbol{\square}$ collectively known as RNA processing, produce a functional mRNA, which is transported to the cytoplasm.
regions of different DNA molecules are exchanged in the process of recombination to generate new combinations of traits in the individual organisms of a species. The final section of the chapter presents basic information about viruses, parasites that exploit the cellular machinery for DNA replication, transcription, and protein synthesis. In addition to being significant pathogens, viruses are important model organisms for studying these cellular mechanisms of macromolecular synthesis and other cellular processes. Viruses have relatively simple structures compared with cells, and their small genomes made them tractable for historic early studies of these fundamental cellular processes. Viruses continue to teach important lessons in molecular cell biology today and have been adapted as experimental tools for introducing genes into cells, tools that are currently being tested for their effectiveness in human gene therapy.

During translation $\boldsymbol{\square}$, the four-base code of the mRNA is decoded into the 20-amino acid language of proteins. Ribosomes, the macromolecular machines that translate the mRNA code, are composed of two subunits assembled in the nucleolus from ribosomal RNAs (rRNAs) and multiple proteins (left). After transport to the cytoplasm, ribosomal subunits associate with an mRNA and carry out protein synthesis with the help of transfer RNAs (tRNAs) and translation factor proteins. During DNA replication $\boldsymbol{\square}$ which occurs only in cells preparing to divide, deoxyribonucleoside triphosphate monomers (dNTPs) are polymerized to yield two identical copies of each chromosomal DNA molecule. Each daughter cell receives one of the identical copies.

### 5.1 Structure of Nucleic Acids

DNA and RNA are chemically very similar. The primary structures of both are linear polymers composed of monomers called nucleotides. DNA and messenger RNA function primarily as informational molecules, carrying information in the exact sequence of their nucleotides. Cellular RNAs range in length from about 22 to many thousands of nucleotides. Cellular DNA molecules can be as long as several hundred million nucleotides. These large DNA units in association with proteins can be stained with dyes and visualized in the light microscope as chromosomes, so named because of their stainability. Though chemically similar, DNA and RNA exhibit some very important differences. For example, RNA can also function as a catalytic molecule. As we will see, the different and unique properties

![li032239539b](li032239539b.jpg)

FIGURE 5-2 Chemical directionality of a nucleic acid strand.
Shown here are alternative representations of a single strand of DNA containing only three bases: cytosine (C), adenine (A), and guanine (G). (a) The chemical structure shows a hydroxyl group at the $3^{\prime}$ end and a phosphate group at the $5^{\prime}$ end. Note also that two phosphoester bonds link adjacent nucleotides; this two-bond linkage is commonly referred to as a phosphodiester bond. (b) In the "stick" diagram (top), the sugars are indicated as vertical lines and the phosphodiester bonds as slanting lines; the bases are denoted by their single-letter abbreviations. In the simplest representation (bottom), only the bases are indicated. By convention, a polynucleotide sequence is always written in the $5^{\prime} \rightarrow 3^{\prime}$ direction (left to right) unless otherwise indicated.
of DNA and RNA make them each suited for their specific functions in the cell.

## A Nucleic Acid Strand Is a Linear Polymer with End-to-End Directionality

In all organisms, DNA and RNA are each made up of only four different nucleotides. Recall from Chapter 2 that all nucleotides consist of an organic base linked to a fivecarbon sugar that has a phosphate group attached to the $5^{\prime}$ carbon. In RNA, the sugar is ribose; in DNA, deoxyribose (see Figure 2-16). The nucleotides used in synthesis of DNA and RNA contain five different bases. The bases adenine (A) and guanine (G) are purines, which contain a pair of fused rings; the bases cytosine (C), thymine (T), and uracil (U) are pyrimidines, which contain a single ring (see Figure 2-17). Three of these bases-A, G, and C-are found in both DNA and RNA; however, T is found only in DNA and U only
in RNA. (Note that the single-letter abbreviations for these bases are also commonly used to denote the entire nucleotides in nucleic acid polymers.)

A single nucleic acid strand has a backbone composed of repeating pentose-phosphate units from which the purine and pyrimidine bases extend as side groups. Like a polypeptide, a nucleic acid strand has an end-to-end chemical orientation: the $5^{\prime}$ end has a hydroxyl or phosphate group on the $5^{\prime}$ carbon of its terminal sugar; the $3^{\prime}$ end usually has a hydroxyl group on the $3^{\prime}$ carbon of its terminal sugar (Figure 5-2). This directionality, plus the fact that synthesis proceeds $5^{\prime}$ to $3^{\prime}$, has given rise to the convention that polynucleotide sequences are written and read in the $5^{\prime} \rightarrow 3^{\prime}$ direction (from left to right); for example, the sequence AUG is assumed to be $\left(5^{\prime}\right) \mathrm{AUG}\left(3^{\prime}\right)$. As we will see, the $5^{\prime} \rightarrow 3^{\prime}$ directionality of a nucleic acid strand is an important property of the molecule. The chemical linkage between adjacent nucleotides, commonly called a phosphodiester bond, actually consists of two phosphoester bonds, one on the $5^{\prime}$ side of the phosphate and another on the $3^{\prime}$ side.

The linear sequence of nucleotides linked by phosphodiester bonds constitutes the primary structure of a nucleic acid molecule. Like polypeptides, polynucleotides can twist and fold into three-dimensional conformations stabilized by noncovalent bonds. Although the primary structures of DNA and RNA are generally similar, their three-dimensional conformations are quite different. These structural differences are critical to the different functions of the two types of nucleic acids.

## Native DNA Is a Double Helix of Complementary Antiparallel Strands

The modern era of molecular biology began in 1953 when James D. Watson and Francis H. C. Crick proposed that DNA has a double-helical structure. Their proposal was based on analysis of x-ray diffraction patterns of DNA fibers generated by Rosalind Franklin and Maurice Wilkins, which showed that the structure was helical, and analyses of the base composition of DNA from multiple organisms by Erwin Chargaff and colleagues. Chargaff's studies revealed that while the base composition of DNA (percentages of A, T, G, and C) varies greatly between distantly related organisms, the percentage of A always equals the percentage of T, and the percentage of G always equals the percentage of C , in all organisms. Based on these discoveries and the structures of the four nucleotides, Watson and Crick performed careful molecular model building, proposing a double helix, with A always hydrogen-bonded to T and G always hydro-gen-bonded to C at the axis of the double helix, as the structure of DNA. The Watson and Crick model proved correct and paved the way for our modern understanding of how DNA functions as the genetic material. Today our most accurate models for DNA structure come from high-resolution x-ray diffraction studies of crystals of DNA, made possible by the chemical synthesis of large amounts of short DNA molecules of uniform length and sequence that are amenable to crystallization (Figure 5-3a).

![li042239539b](li042239539b.jpg)

FIGURE 5-3 The DNA double helix. (a) Space-filling model of $B$ DNA, the most common form of DNA in cells. The bases (light shades) project inward from the sugar-phosphate backbones (dark red and blue) of each strand, but their edges are accessible through major and minor grooves. Arrows indicate the $5^{\prime} \rightarrow 3^{\prime}$ direction of each strand. Hydrogen bonds between the bases are in the center of the structure. The major and minor grooves are lined by potential hydrogen bond donors and acceptors (highlighted in yellow). (b) Chemical structure of DNA double helix. This extended schematic shows the two sugar-phosphate backbones and hydrogen bonding between the Watson-Crick base pairs, A-T and G-C. See R. E. Dickerson, 1983, Sci. Am. 249:94. [Part (a) data from R. Wing et al., 1980, Nature 287:755, PDB ID 1bna.]

DNA consists of two associated polynucleotide strands that wind together to form a double helix. The two sugarphosphate backbones are on the outside of the double helix, and the bases project into the interior. The adjoining bases in each strand stack on top of one another in parallel planes (see Figure 5-3a). The orientation of the two strands is antiparallel; that is, their $5^{\prime} \rightarrow 3^{\prime}$ directions are opposite. The strands are held in precise register by formation of base pairs between the two strands: A is paired with T through two hydrogen bonds; G is paired with C through three hydrogen bonds (Figure 5-3b). This basepair complementarity is a consequence of the size, shape, and chemical composition of the bases. The presence of thousands of such hydrogen bonds in a DNA molecule contributes greatly to the stability of the double helix. Hydrophobic and van der Waals interactions between the stacked adjacent base pairs further stabilize the doublehelical structure.

In natural DNA, A always hydrogen-bonds with T and G with C , forming $\mathrm{A} \cdot \mathrm{T}$ and $\mathrm{G} \cdot \mathrm{C}$ base pairs as shown in Figure 5-3b. These associations, always between a larger purine and a smaller pyrimidine, are often called WatsonCrick base pairs. Two polynucleotide strands, or regions thereof, in which all the nucleotides form such base pairs are said to be complementary. However, in theory and in synthetic DNAs, other base pairs can form. For example, guanine (a purine) could theoretically form hydrogen bonds with thymine (a pyrimidine), causing only a minor distortion in the helix. The space available in the helix would also allow pairing between the two pyrimidines cytosine and
thymine. Although the nonstandard G$\cdot$T and C$\cdot$T base pairs are not normally found in DNA, G$\cdot$U base pairs are quite common in double-helical regions that form within otherwise single-stranded RNA. Nonstandard base pairs do not occur naturally in double-stranded (duplex) DNA because the DNA copying enzyme, described later in this chapter, does not permit them.

Most DNA in cells takes the form of a right-handed helix. The x-ray diffraction pattern of DNA indicates that the stacked bases are regularly spaced 0.34 nm apart along the helix axis. The helix makes a complete turn every 3.4 to 3.6 nm , depending on the sequence; thus there are about $10-10.5$ base pairs per turn. This helical form, referred to as the $B$ form of DNA, is the normal form present in most DNA stretches in cells. On the outside of the helix, the spaces between the intertwined strands form two helical grooves of different widths, described as the major groove and the minor groove (see Figure 5-3a). As a consequence, the atoms on the edges of each base within these grooves are accessible from outside the helix, forming two types of binding surfaces. DNA-binding proteins can "read" the sequence of bases in duplex DNA by contacting atoms in either the major or the minor grooves.

Under laboratory conditions in which most of the water is removed from DNA, the crystallographic structure of DNA changes to the $A$ form, which is wider and shorter than B-form DNA, with a wider and deeper major groove and a narrower and shallower minor groove (Figure 5-4). RNA-DNA and RNA-RNA helices also exist in this form in cells and in vitro.

![li052239539b](li052239539b.jpg)

FIGURE 5-4 Comparison of A-Form and B-Form DNA. The sugarphosphate backbones of the two polynucleotide strands, which are on the outside in both structures, are shown in red and blue; the bases (lighter shades) are oriented inward. (a) The B form of DNA has about 10.5 base pairs per helical turn. Adjacent stacked base pairs are 0.34 nm apart. (b) The more compact A form of DNA has 11 base pairs per turn, with a much deeper major groove and a much shallower minor groove than B-form DNA. [Part (a) data from R. Wing et al., 1980, Nature 287:755, PDB ID 1bna. Part (b) data from B. N. Conner et al., 1984, J. Mol. Biol. 174:663, PDB ID 1ana.]

Important modifications in the structure of standard Bform DNA come about as a result of protein binding to specific DNA sequences. Although the multitude of hydrogen and hydrophobic bonds between the bases provides stability to DNA, the double helix is flexible about its long axis. Unlike the $\alpha$ helix in proteins (see Figure 3-4), it has no hydrogen bonds parallel to the axis of the helix. This property allows DNA to bend when complexed with a DNA-binding protein, such as the transcription factor TBP (Figure 5-5).
![li062239539b](li062239539b.jpg)

FIGURE 5-5 Interaction with a protein can bend DNA. The conserved C-terminal domain of the TATA box-binding protein (TBP) binds to the minor groove of specific DNA sequences rich in $A$ and $T$, untwisting and sharply bending the double helix. Transcription of most eukaryotic genes requires participation of TBP. [Data from D. B. Nikolov and S. K. Burley, 1997, Proc. Natl. Acad. Sci. USA 94:15, PDB ID 1cdw.]

Bending of DNA is also critical to the dense packing of DNA in chromatin, the protein-DNA complex in which nuclear DNA occurs in eukaryotic cells (see Chapter 8).

Why did DNA, rather than RNA, evolve to be the carrier of genetic information in cells? The hydrogen at the $2^{\prime}$ position in the deoxyribose of DNA makes it a far more stable molecule than RNA, which instead has a hydroxyl group at the $2^{\prime}$ position of ribose (see Figure 2-16). The $2^{\prime}$-hydroxyl groups in RNA participate in the slow, $\mathrm{OH}^{-}$-catalyzed hydrolysis of phosphodiester bonds at neutral pH (Figure 5-6). The absence of $2^{\prime}$-hydroxyl groups in DNA prevents this process. Therefore, the presence of deoxyribose in DNA makes it a more stable molecule-a characteristic that is critical to its function in the long-term storage of genetic information.

## DNA Can Undergo Reversible Strand Separation

During replication and transcription of DNA, the strands of the double helix must separate to allow the internal edges of the bases to pair with the bases of the nucleotides being polymerized into new complementary polynucleotide chains. In later sections, we describe the cellular mechanisms that separate and subsequently reassociate DNA strands during replication and transcription. Here we discuss the fundamental factors that influence the separation and reassociation of DNA strands. These properties of DNA were elucidated by in vitro experiments.

The unwinding and separation of DNA strands, referred to as denaturation, or melting, can be induced experimentally by increasing the temperature of a solution of DNA. As the thermal energy increases, the resulting increase in molecular motion eventually breaks the hydrogen bonds and other forces that stabilize the double helix. The strands then separate, driven apart by the electrostatic repulsion of the negatively charged deoxyribose-phosphate backbones of the two strands. Near the denaturation temperature, a small increase in temperature causes a rapid, nearly simultaneous loss of the multiple weak interactions holding the strands together along the entire length of the DNA molecules. Because the stacked base pairs in duplex DNA absorb less ultraviolet (UV) light than the unstacked bases in single-stranded DNA, this change leads to an abrupt increase in the absorption of UV light. This phenomenon, known as hyperchromicity (Figure 5-7a), is useful for monitoring DNA denaturation.

The melting temperature $\left(T_{\mathrm{m}}\right)$ at which DNA strands separate depends on several factors. Molecules that contain a greater proportion of $\mathrm{G} \cdot \mathrm{C}$ pairs require higher temperatures to denature because the three hydrogen bonds in $\mathrm{G} \cdot \mathrm{C}$ pairs make these base pairs more stable than $\mathrm{A} \cdot \mathrm{T}$ pairs, which have only two hydrogen bonds. Indeed, the percentage of $\mathrm{G} \cdot \mathrm{C}$ base pairs in a DNA sample can be estimated from its $T_{\mathrm{m}}$ (Figure 5-7b). The ion concentration of the solution also influences the $T_{\mathrm{m}}$ because the negatively charged phosphate groups in the two strands are shielded by positively charged ions. When the ion concentration is low, this shielding is decreased, thus increasing the repulsive forces between the strands and reducing the $T_{\mathrm{m}}$. Agents that

![li072239539b](li072239539b.jpg)

FIGURE 5-6 Base-catalyzed hydrolysis of RNA. The $2^{\prime}$-hydroxyl group in RNA can act as a nucleophile, attacking the phosphodiester bond. The $2^{\prime}, 3^{\prime}$ cyclic monophosphate derivative is further hydro-
destabilize hydrogen bonds, such as formamide or urea, also lower the $T_{\mathrm{m}}$. Finally, extremes of pH denature DNA at low temperatures. At low (acid) pH , the bases become protonated and thus positively charged, repelling each other. At high (alkaline) pH , the bases lose protons and become negatively charged, again repelling each other because of their similar charges. In cells, pH and temperature are, for the most part, maintained at a constant level. These features of
![li082239539b](li082239539b.jpg)

EXPERIMENTAL FIGURE 5-7 G-C content of DNA affects melting temperature. The temperature at which DNA denatures increases with the proportion of G-C pairs. (a) Melting of doublestranded DNA can be monitored by its absorption of UV light at 260 nm . As regions of double-stranded DNA unpair, the absorption of light by those regions increases almost twofold. Light absorption
lyzed to a mixture of $2^{\prime}$ and $3^{\prime}$ monophosphates. This mechanism of phosphodiester bond hydrolysis cannot occur in DNA, which lacks $2^{\prime}$-hydroxyl groups.

DNA denaturation are most useful for manipulating DNA in a laboratory setting.

The single-stranded DNA molecules that result from denaturation form random coils without an organized structure. Lowering the temperature, increasing the ion concentration, or neutralizing the pH causes two complementary strands to reassociate into a perfect double helix. The extent of such renaturation is dependent on time, the DNA
(b)
![li092239539b](li092239539b.jpg)
by single-stranded DNA changes much less as the temperature is increased. (b) The temperature at which half the bases in a doublestranded DNA sample have denatured is denoted $T_{\mathrm{m}}$ (for "temperature of melting"). The $T_{\mathrm{m}}$ is a function of the G-C content of the DNA; the greater the $\mathrm{G}+\mathrm{C}$ percentage, the higher the $T_{\mathrm{m}}$.

concentration, and the ion concentration. Two DNA strands that are not related in sequence will remain as random coils and will not renature, but they will not inhibit complementary DNA partner strands from finding each other and renaturing. Denaturation and renaturation of DNA are the basis of nucleic acid hybridization, a powerful technique used to study the relatedness of two DNA samples and to detect and isolate specific DNA molecules in a mixture containing numerous different DNA sequences (see Chapter 6).

## Torsional Stress in DNA Is Relieved by Enzymes

Many bacterial genomic DNAs and many viral DNAs are circular molecules. Circular DNA molecules also occur in mitochondria, which are present in almost all eukaryotic cells, and in chloroplasts, which are present in plants and some unicellular eukaryotes. Although eukaryotic nuclear DNA is linear, long loops of DNA are fixed in place within chromosomes (see Chapter 8). Each of the two strands in a circular DNA molecule or in a fixed loop of a eukaryotic chromosome forms a closed structure without free ends and is therefore subject to torsional stress.

Most bacterial DNA in chromosomes and DNA isolated from viruses containing circular double-stranded DNA is underwound, meaning that it has fewer helical turns than B-form linear DNA of the same length. As a result, the DNA molecule twists back on itself like a twisted rubber band, forming supercoils (Figure 5-8a). Localized unwinding or overwinding of a DNA molecule, which occurs during DNA replication and transcription, induces torsional stress into the remaining portion of the molecule because the ends of the strands are not free to rotate. All cells, however, contain topoisomerase I, which can relieve the torsional stress that
develops in cellular and viral DNA molecules during replication and transcription. This enzyme binds to DNA at random sites and breaks a phosphodiester bond in one strand. Such a one-strand break in DNA is called a nick. The broken end then winds around the uncut strand, leading to loss of supercoils (Figure 5-8b). Finally, the same enzyme joins (ligates) the two ends of the broken strand. Another type of enzyme, topoisomerase II, makes breaks in both strands of a double-stranded DNA and then religates them. As a result, topoisomerase II can both relieve torsional stress and link together two circular DNA molecules as in the links of a chain.

## Different Types of RNA Exhibit Various Conformations Related to Their Functions

The primary structure of RNA is generally similar to that of DNA, with two exceptions: the sugar component of RNA, ribose, has a hydroxyl group at the $2^{\prime}$ position (see Figure 2-16b), and thymine in DNA is replaced by uracil in RNA. The presence of thymine rather than uracil in DNA is important to the long-term stability of DNA because of its function in DNA repair (see Section 5.6). As noted earlier, the hydroxyl group on the $2^{\prime}$ carbon of ribose makes RNA more chemically labile than DNA. As a result of this lability, RNA is cleaved into mononucleotides by an alkaline solution (see Figure 5-6), whereas DNA is not. The $2^{\prime}$-hydroxyl group of RNA also provides a chemically reactive group that takes part in RNAmediated catalysis. Like DNA, RNA is a long polynucleotide that can be double-stranded or single-stranded, linear or circular. It can also participate in a hybrid helix composed of one RNA strand and one DNA strand. As mentioned above, RNA-RNA and RNA-DNA double helices have a compact conformation like the A form of DNA (see Figure 5-4b).
![li102239539b](li102239539b.jpg)

EXPERIMENTAL FIGURE 5-8 Topoisomerase I relieves torsional stress on DNA.
(a) Electron micrograph of SV40 viral DNA. When the circular DNA of the SV40 virus is isolated and separated from its associated protein, the DNA duplex is underwound and assumes the supercoiled configuration. (b) If a supercoiled DNA is nicked (i.e., one strand cleaved), the strands can rewind, leading to loss of a supercoil. Topoisomerase I catalyzes this reaction and also reseals the broken ends. All the supercoils in isolated SV40 DNA can be removed by the sequential action of this enzyme, producing the relaxed-circle conformation. For clarity, the shapes of the molecules at the bottom have been simplified. (Photos courtesy of Laurien Polder, from A. Kornberg (1980) DNA Replication, p. 29, W.H. Freeman, New York.)

![li112239539b](li112239539b.jpg)

FIGURE 5-9 RNA secondary and tertiary structures. (a) Hairpins, stem-loops, and other secondary structures can form by base pairing between distant complementary segments of an RNA molecule. In stem-loops, the single-stranded loop between the base-paired helical stem may be hundreds or even thousands of nucleotides long, whereas in hairpins, the short turn may contain as few as four nucleotides. (b) Pseudoknots, one type of RNA tertiary structure, are formed by interaction of loops through base pairing between complementary bases. The structure shown forms the core domain of the human telomerase RNA. Left: Secondary-structure diagram with base-paired nucleotides in green and blue and single-stranded regions in red. Middle: Sequence of the telomerase RNA core domain, colored to correspond to the secondary-structure diagram at the left. Right: Diagram of the telomerase core domain structure determined by 2D-NMR, showing paired bases only and a tube for the sugar phosphate backbone, colored to correspond to the diagrams at left. [Part (b) middle and right data from C. A. Theimer et al., 2005, Mol. Cell 17:671, PDB ID 1ymo.]

Unlike DNA, which exists primarily as a very long double helix, most cellular RNAs are single-stranded, and they exhibit a variety of conformations (Figure 5-9). Differences in the sizes and conformations of the various types of RNA permit them to carry out specific functions in a cell. The simplest secondary structures in single-stranded RNAs are formed by pairing of complementary bases. "Hairpins" are formed by pairing of bases within about five to ten nucleotides of each other, and "stem-loops" by pairing of bases that are separated by eleven to several hundred nucleotides. These simple folds can cooperate to form more complicated tertiary structures, one of which is termed a "pseudoknot."

As discussed in detail later, tRNA molecules adopt a well-defined three-dimensional architecture in solution that is crucial to protein synthesis. Larger rRNA molecules also have locally well-defined three-dimensional structures with
more flexible linkers in between. Secondary and tertiary structures have also been recognized in mRNA, particularly near the ends of molecules. Clearly, then, RNA molecules are like proteins in that they have structured domains connected by less structured, flexible stretches.

The folded domains of RNA molecules not only are structurally analogous to the $\alpha$ helices and $\beta$ strands found in proteins, but in some cases also have catalytic capacities. Such catalytic RNAs are called ribozymes. Although ribozymes are usually associated with proteins that stabilize the ribozyme structure, it is the RNA that acts as a catalyst. Some ribozymes can catalyze splicing, a remarkable process in which an internal RNA sequence is cut and removed and the two resulting chains are then ligated. This process occurs during the formation of the majority of functional mRNA molecules in multicellular eukaryotes, and it also occurs in single-celled eukaryotes such as yeasts, bacteria, and archaea. Remarkably, some RNAs carry out self-splicing, with the catalytic activity residing in the sequence that is removed. The mechanisms of splicing and self-splicing are discussed in detail in Chapter 10. As noted later in this chapter, rRNA plays a catalytic role in the formation of peptide bonds during protein synthesis.

In this chapter, we focus on the functions of mRNA, tRNA, and rRNA in gene expression. In later chapters, we will encounter other RNAs, often associated with proteins, that participate in other cell functions.

## KEY CONCEPTS OF SECTION 5.1

## Structure of Nucleic Acids

- Deoxyribonucleic acid (DNA), the genetic material, carries information to specify the amino acid sequences of proteins. It is transcribed into several types of ribonucleic acid (RNA), including messenger RNA (mRNA), transfer RNA (tRNA), and ribosomal RNA (rRNA), all of which function in protein synthesis (see Figure 5-1).
- All DNAs and most RNAs are long, unbranched polymers of nucleotides. A nucleotide consists of a phosphorylated pentose linked to an organic base, either a purine or a pyrimidine.
- The purines adenine (A) and guanine (G) and the pyrimidine cytosine $(\mathrm{C})$ are present in both DNA and RNA. The pyrimidine thymine ( T ) present in DNA is replaced by the pyrimidine uracil (U) in RNA.
- Adjacent nucleotides in a polynucleotide are linked by phosphodiester bonds. The entire strand has a chemical directionality with $5^{\prime}$ and $3^{\prime}$ ends (see Figure 5-2).
- Natural DNA (B DNA) contains two complementary antiparallel polynucleotide strands wound together into a regular right-handed double helix with the bases on the inside and the two sugar-phosphate backbones on the outside

(see Figure 5-3). Base pairing between the strands and hydrophobic interactions between adjacent base pairs stacked perpendicular to the helix axis stabilize this native structure.

- The bases in nucleic acids can interact via hydrogen bonds. The standard Watson-Crick base pairs are G$\cdot$C, A-T (in DNA), and G$\cdot$C, A-U (in RNA). Base pairing stabilizes the native three-dimensional structures of DNA and RNA.
- Binding of protein to DNA can deform its helical structure, causing local bending or unwinding of the DNA molecule.
- Heat causes the DNA strands to separate (denature). The melting temperature ( $T_{\mathrm{m}}$ ) of DNA increases with the percentage of G$\cdot$C base pairs. Under suitable conditions, separated complementary nucleic acid strands will renature.
- Circular DNA molecules can be twisted on themselves, forming supercoils (see Figure 5-8). Enzymes called topoisomerases can relieve torsional stress and remove supercoils from circular DNA molecules. Long linear DNA can also experience torsional stress because long loops are fixed in place within chromosomes.
- Cellular RNAs are single-stranded polynucleotides, some of which form well-defined secondary and tertiary structures (see Figure 5-9). Some RNAs, called ribozymes, have catalytic activity.


### 5.2 Transcription of Protein-Coding Genes and Formation of Functional mRNA

The simplest definition of a gene is "a unit of DNA that contains the information to specify synthesis of a single polypeptide chain or functional RNA (such as a tRNA)." The DNA molecules of small viruses contain only a few genes, whereas the single DNA molecule in each of the chromosomes of higher animals and plants may contain several thousand genes. The vast majority of genes carry information used to build protein molecules, and it is the RNA copies of such protein-coding genes that constitute the mRNA molecules of cells.

During synthesis of RNA, the four-base language of DNA containing $\mathrm{A}, \mathrm{G}, \mathrm{C}$, and T is simply copied, or transcribed, into the four-base language of RNA, which is identical except that $U$ replaces $T$. In contrast, during protein synthesis, the four-base language of DNA and RNA is translated into the 20 -amino acid language of proteins. In this section, we focus on the formation of functional mRNAs from proteincoding genes (see Figure 5-1, step ■). A similar process yields precursors of rRNAs and tRNAs, encoded by rRNA and tRNA genes; these precursors are then further modified to yield functional rRNAs and tRNAs (see Chapters 9 and 10). Similarly, thousands of micro-RNAs (miRNAs), which regulate the translation and stability of specific target mRNAs,
are transcribed into precursors by RNA polymerases and processed into functional miRNAs (see Chapter 10). Other non-protein-coding (or simply noncoding) RNAs help to regulate the transcription of specific protein-coding genes. Regulation of transcription allows distinct sets of genes to be expressed in the multiple different types of cells that make up a multicellular organism. It also allows different amounts of mRNA to be transcribed from different genes, resulting in differences in the amounts of the encoded proteins in a cell. Regulation of transcription is addressed in Chapter 9.

## A Template DNA Strand Is Transcribed into a Complementary RNA Chain by RNA Polymerase

During transcription of DNA, one DNA strand acts as a template, determining the order in which ribonucleoside triphosphate (rNTP) monomers are linked together to form a complementary RNA chain. Bases in the template DNA strand base-pair with complementary incoming rNTPs, which are then joined in a polymerization reaction catalyzed by RNA polymerase. The polymerization reaction involves a nucleophilic attack by the $3^{\prime}$ oxygen in the growing RNA chain on the $\alpha$ phosphate of the next nucleotide precursor to be added, which results in the formation of a phosphodiester bond and the release of pyrophosphate $\left(\mathrm{PP}_{\mathrm{i}}\right)$. As a consequence of this mechanism, RNA molecules are always synthesized in the $5^{\prime} \rightarrow 3^{\prime}$ direction (Figure 5-10a).

The energetics of the polymerization reaction strongly favor the addition of ribonucleotides to the growing RNA chain because the high-energy bond between the $\alpha$ and $\beta$ phosphates of rNTP monomers is replaced by the lowerenergy phosphodiester bond between nucleotides. The equilibrium for the reaction is driven further toward chain elongation by pyrophosphatase, an enzyme that catalyzes cleavage of the released PP, into two molecules of inorganic phosphate. Like the two strands in DNA, the template DNA strand and the growing RNA strand that is base-paired to it have opposite $5^{\prime} \rightarrow 3^{\prime}$ directionality.

By convention, the site on the DNA template at which RNA polymerase begins transcription is numbered +1 (Figure 5-10b). Downstream denotes the direction in which a template DNA strand is transcribed; upstream denotes the opposite direction. Nucleotide positions in the DNA sequence downstream from a start site are indicated by a positive $(+)$ sign; those upstream, by a negative $(-)$ sign. Because RNA is synthesized $5^{\prime} \rightarrow 3^{\prime}$, RNA polymerase moves down the template DNA strand in a $3^{\prime} \rightarrow 5^{\prime}$ direction. The newly synthesized RNA is complementary to the template DNA strand; therefore, it is identical to the nontemplate DNA strand, with uracil in place of thymine.

Stages in Transcription To carry out transcription, RNA polymerase performs several distinct functions, as depicted in Figure 5-11. During transcription initiation, RNA polymerase, with the help of initiation factors (discussed later), recognizes and binds to a specific sequence of doublestranded DNA called a promoter (step ■). After binding,

![li122239539b](li122239539b.jpg)
(b)
![li132239539b](li132239539b.jpg)

Nontemplate strand 5' CTGCCATTGTCAGACATGTATACCCCGTACGTCTTCCCGAGCGAAAACGATCTGCGCTGC 3'
Template strand 3' GACGGTAACAGTCTGTACATATGGGGCATGCAGAAGGGCTCGCTTTTGCTAGACGCGACG 5' $\leftarrow$ DNA
5' CUGCCAUUGUCAGACAUGUAUACCCCGUACGUCUUCCCGAGCGAAAACGAUCUGCGCUGC 3' Primary RNA transcript

![li142239539b](li142239539b.jpg)

FIGURE 5-11 Three stages in transcription. During initiation of transcription, RNA polymerase forms a transcription bubble and begins polymerization of ribonucleotides (rNTPs) at the start site, which is located within the promoter region. Once a DNA region has been transcribed, the separated strands reassociate into a double helix. The nascent RNA is displaced from its template strand except at its $3^{\prime}$ end. The $5^{\prime}$ end of the RNA strand exits the RNA polymerase through a channel in the enzyme. Termination occurs when the polymerase encounters a specific termination sequence (stop site). See the text for details. For simplicity, the diagram depicts transcription of four turns of the DNA helix encoding some 40 nucleotides of RNA. Most RNAs are considerably longer, requiring transcription of a longer region of DNA.
strand elongation stage, RNA polymerase moves along the template DNA, opening the double-stranded DNA in front of its direction of movement and guiding the strands back together so that they reassociate at the upstream end of the transcription bubble (step $\mathbf{\square}$ ). One ribonucleotide at a time is added by the polymerase to the $3^{\prime}$ end of the growing (nascent) RNA chain. During strand elongation, the enzyme maintains a melted region of approximately 14-20 base pairs in the transcription bubble. Approximately eight nucleotides at the $3^{\prime}$ end of the growing RNA strand remain base-paired to the template DNA strand in the transcription bubble. The elongation complex, comprising RNA polymerase, template DNA, and the nascent RNA strand, is extraordinarily stable. For example, RNA polymerase transcribes the longest known mammalian gene, containing about 2 million base pairs, without dissociating from the DNA template or releasing the nascent RNA. RNA synthesis occurs at a rate of about $1000-2000$ nucleotides per minute at $37^{\circ} \mathrm{C}$, so the
elongation complex must remain intact for more than 24 hours to ensure continuous synthesis of pre-mRNA from this very long gene.

During transcription termination, the final stage in RNA synthesis, the completed RNA molecule is released from the RNA polymerase and the polymerase dissociates from the template DNA (step $\mathbf{\square}$ ). Once it is released, an RNA polymerase is free to transcribe the same gene again or another gene.

Structure of RNA Polymerases The RNA polymerases of bacteria, archaea, and eukaryotic cells are fundamentally similar in structure and function. Bacterial RNA polymerases are composed of two related large subunits ( $\beta^{\prime}$ and $\beta$ ), two copies of a smaller subunit ( $\alpha$ ), and one copy of a fifth subunit ( $\omega$ ) that is not essential for transcription or cell viability, but that stabilizes the enzyme and assists in the assembly of its subunits. Archaeal and eukaryotic RNA

![li152239539b](li152239539b.jpg)

FIGURE 5-12 Bacterial RNA polymerase. This structure corresponds to the polymerase molecule in the elongation stage (step 5) of Figure 5-11. In this diagram, transcription is proceeding in the rightward direction. Arrows indicate where downstream DNA enters the polymerase and upstream DNA exits at an angle from the downstream DNA. The template strand is light violet, the nontemplate strand, dark violet; the nascent RNA, red. The RNA polymerase $\beta^{\prime}$ subunit is gold; the $\beta$ subunit, light yellow; and the $\alpha$ subunits visible from this angle, brown. Nucleotides complementary to the template DNA are added to the $3^{\prime}$ end of the nascent RNA strand on the right side of the transcription bubble. The newly synthesized nascent RNA exits the polymerase at the upstream side through a channel formed by the $\beta$ subunit. The $\omega$ subunit is also visible from this angle. (Data courtesy of Seth Darst; see N. Korzheva et al., 2000, Science 289:619-625, and N. Opalka et al., 2003, Cell 114:335-345.)
polymerases have several additional small subunits associated with this core complex, which we describe in Chapter 9. Schematic diagrams of the transcription process generally show RNA polymerase bound to an unbent DNA molecule, as in Figure 5-11. However, x-ray crystallography and other studies of an elongating bacterial RNA polymerase indicate that the DNA bends at the transcription bubble (Figure 5-12).

## Organization of Genes Differs in Prokaryotic and Eukaryotic DNA

Having outlined the process of transcription, we now briefly consider the large-scale arrangement of information in DNA and how this arrangement dictates the requirements for RNA synthesis so that information transfer goes smoothly. In recent years, sequencing of entire genomes from multiple organisms has revealed not only large variations in the number of protein-coding genes, but also differences in their organization in bacteria and in eukaryotes.

The most common arrangement of protein-coding genes in bacteria has a powerful and appealing logic: genes encoding proteins that function together-for example, the enzymes required to synthesize the amino acid tryptophanare most often found in a contiguous array in the DNA. Such an arrangement of genes in a functional group is called an operon because it operates as a unit from a single promoter. Transcription of an operon produces a continuous strand of mRNA that carries the message for a related series of proteins (Figure 5-13a). Each section of the mRNA represents the unit (or gene) that encodes one of the proteins in the series. This arrangement results in the coordinate expression of all the genes in the operon. Every time an RNA polymerase molecule initiates transcription at the promoter of the operon, all the genes of the operon are transcribed and translated. In prokaryotic DNA the genes are closely packed with very few noncoding gaps, and the DNA is transcribed directly into mRNA. Because DNA is not sequestered in a nucleus in prokaryotes, ribosomes have immediate access to the translation start sites in the mRNA as they emerge from the surface of the RNA polymerase. Consequently, translation of the mRNA begins even while the $3^{\prime}$ end of the mRNA is still being synthesized at the active site of the RNA polymerase.

This economical clustering of genes devoted to a single metabolic function is rarely found in eukaryotes, even simple ones such as yeasts, which can be metabolically similar to bacteria. Rather, eukaryotic genes encoding proteins that function together are most often physically separated in the DNA; indeed, such genes are usually located on different chromosomes. Each gene is transcribed from its own promoter, producing one mRNA, which is generally translated to yield a single polypeptide (Figure 5-13b).

Early research on the structure of eukaryotic genes involved studies of viruses that infect animals. When researchers analyzed the regions of a viral DNA molecule that encode viral mRNAs, they were surprised to observe that the sequence of a single viral mRNA was encoded in several regions of the viral DNA separated by DNA sequences that are not present in the mRNA. Later, the development of gene cloning and DNA sequencing (see Chapter 6) allowed researchers to compare the genomic DNA sequences of multicellular organisms with the sequences of their mRNAs. This research revealed that most cellular mRNAs are also encoded in several separate regions of genomic DNA, called exons, separated by sequences of DNA called introns. Further studies showed that a gene is first transcribed into a long primary transcript that includes both exon sequences and the intron sequences that separate them. Subsequently, the introns are removed and the exons are spliced together (see Chapter 10). Although introns are common in multicellular eukaryotes, they are extremely rare in bacteria and archaea and uncommon in many unicellular eukaryotes, such as baker's yeast.

![li162239539b](li162239539b.jpg)

FIGURE 5-13 Gene organization in prokaryotes and in eukaryotes. (a) The tryptophan (trp) operon is a continuous segment of the E. coli chromosome containing five genes (blue) that encode the enzymes necessary for the stepwise synthesis of tryptophan. The entire operon is transcribed from one promoter into one long continuous trp mRNA (red). Translation of this mRNA begins at five different start sites, yielding five proteins (green). The order of the genes in the bacterial

## Eukaryotic Precursor mRNAs Are Processed to Form Functional mRNAs

In bacterial cells, which have no nuclei, translation of an mRNA into protein can begin at the $5^{\prime}$ end of the mRNA even while the $3^{\prime}$ end is still being synthesized by RNA polymerase. In other words, transcription and translation occur concurrently in bacteria. In eukaryotic cells, however, the site of RNA synthesis-the nucleus-is separated from the site of translation-the cytoplasm. Furthermore, the primary transcripts of protein-coding genes are precursor mRNAs (pre-mRNAs) that must undergo several modifications, collectively termed RNA processing, to yield a functional mRNA (see Figure 5-1, step ■). This mRNA then must be exported to the cytoplasm before it can be translated into protein. Thus transcription and translation cannot occur concurrently in eukaryotic cells.

All eukaryotic pre-mRNAs are initially modified at the two ends, and these modifications are retained in mRNAs. As the $5^{\prime}$ end of a nascent RNA chain emerges from the surface of RNA polymerase, it is immediately acted on
(b) Eukaryotes
![li172239539b](li172239539b.jpg)
genome parallels the sequential function of the encoded proteins in the tryptophan synthesis pathway. (b) The five genes encoding the enzymes required for tryptophan synthesis in baker's yeast (Saccharomyces cerevisiae) are carried on four different chromosomes. Each gene is transcribed from its own promoter to yield a primary transcript that is processed into a functional mRNA encoding a single protein. The lengths of the various chromosomes are given in kilobases ( $10^{5}$ bases).
by several enzymes that together synthesize the $5^{\prime}$ cap, a 7-methylguanylate that is connected to the terminal nucleotide of the RNA by an unusual $5^{\prime}, 5^{\prime}$ triphosphate linkage (Figure 5-14). The cap protects an mRNA from enzymatic degradation and assists in its export to the cytoplasm. The cap is also bound by a protein factor required to begin translation in the cytoplasm.

Processing at the $3^{\prime}$ end of a pre-mRNA involves cleavage by an endonuclease to yield a free $3^{\prime}$-hydroxyl group, to which a string of adenylic acid residues is added one at a time by an enzyme called poly $(A)$ polymerase. The resulting poly $(A)$ tail contains 100-250 bases, being shorter in yeasts and invertebrates than in vertebrates. Poly(A) polymerase is part of a complex of proteins that can locate and cleave a transcript at a specific site and then add the correct number of A residues, in a process that does not require a template. As discussed further in Section 5.4 and in Chapter 10, the poly(A) tail has important functions both in translation of mRNA and in stabilizing premRNAs in the nucleus and fully processed mRNAs in the nucleus and cytoplasm.

![li182239539b](li182239539b.jpg)

FIGURE 5-14 Structure of the $5^{\prime}$ methylated cap. The distinguishing chemical features of the $5^{\prime}$ methylated cap on eukaryotic mRNA are (1) the $5^{\prime} \rightarrow 5^{\prime}$ linkage of 7-methylguanylate to the initial nucleotide of the mRNA molecule and (2) the methyl group on the $2^{\prime}$ hydroxyl of the ribose of the first nucleotide (base 1). Both of these features occur in all animal cells and in cells of higher plants; yeasts lack the methyl group on nucleotide 1. The ribose of the second nucleotide (base 2) is also methylated in vertebrates. See A. J. Shatkin, 1976, Cell 9:645.

Another step in the processing of many different eukaryotic mRNA molecules is RNA splicing: the internal cleavage of a transcript to excise the introns and stitch together the coding exons. Figure 5-15 summarizes the basic steps in eukaryotic mRNA processing using the $\beta$-globin gene as an example. We examine the cellular machinery for carrying out processing of mRNA, as well as tRNA and rRNA, in Chapter 10.

The functional eukaryotic mRNAs produced by RNA processing retain noncoding regions, referred to as untranslated regions (UTRs), at each end. In mammalian mRNAs, the $5^{\prime}$ UTR may be a hundred or more nucleotides long, and the $3^{\prime}$ UTR may be several kilobases in length. Bacterial mRNAs also usually have $5^{\prime}$ and $3^{\prime}$ UTRs, but these regions are much shorter than those in eukaryotic mRNAs, generally containing fewer than 10 nucleotides. As discussed in Chapter 10, the $5^{\prime}$ UTR and $3^{\prime}$ UTR sequences participate in
![li192239539b](li192239539b.jpg)

FIGURE 5-15 Overview of RNA processing. RNA processing produces functional mRNA in eukaryotes. The $\beta$-globin gene contains three protein-coding exons (constituting the coding region) and two intervening noncoding introns. The introns interrupt the proteincoding sequence between the codons for amino acids 31 and 32 and 105 and 106. Transcription of eukaryotic protein-coding genes starts before the sequence that encodes the first amino acid and extends beyond the sequence that encodes the last amino acid, resulting in noncoding regions at the ends of the primary transcript. These untranslated regions (UTRs) are retained during processing. The $5^{\prime}$ cap ( $\mathrm{m}^{7} \mathrm{Gppp}$ ) is added during formation of the primary RNA transcript, which extends beyond the poly(A) site. After cleavage at the poly(A) site and addition of multiple A residues to the $3^{\prime}$ end, splicing removes the introns and joins the exons. The small numbers refer to positions in the 147-amino acid sequence of $\beta$-globin.
regulation of mRNA translation and stability, and $3^{\prime}$ UTRs also function in the localization of many mRNAs to specific regions of the cytoplasm.

## Alternative RNA Splicing Increases the Number of Proteins Expressed from a Single Eukaryotic Gene

In contrast to bacterial and archaeal genes, the vast majority of genes in multicellular eukaryotes contain multiple introns. As noted in Chapter 3, many proteins from higher eukaryotes have a multidomain tertiary structure (see Figure 3-11). Individual repeated protein domains are often encoded by one exon or by a small number of exons that are repeated in genomic DNA and encode identical or nearly identical amino acid sequences. Such repeated exons are thought to have evolved from multiple duplications of a length of DNA lying between two sites in

![li202239539b](li202239539b.jpg)

FIGURE 5-16 Alternative splicing. The $\sim 75-\mathrm{kb}$ fibronectin gene (top) contains multiple exons; splicing of the fibronectin transcript varies by cell type. The EIIIB and EIIIA exons (green) encode binding domains for specific proteins on the surface of fibroblasts. The fibronectin mRNA
produced in fibroblasts includes the EIIIA and EIIIB exons, whereas these exons are spliced out of fibronectin mRNA in hepatocytes. In this diagram, introns (black lines in the top diagram of the fibronectin gene) are not drawn to scale; most of them are much longer than any of the exons.
introns on either side of an exon, resulting in insertion of a string of repeated exons separated by introns. The presence of multiple introns in many eukaryotic genes permits expression of multiple, related proteins from a single gene by means of alternative splicing. In higher eukaryotes, alternative splicing is an important mechanism for production of different forms of a protein, called isoforms, by different types of cells.

Fibronectin, a multidomain protein found in mammals, provides a good example of alternative splicing (Figure 5-16). Fibronectin is a long, adhesive protein secreted into the extracellular space that can bind other proteins together. What and where it binds depends on which domains are spliced together. The fibronectin gene contains numerous exons, grouped into several regions corresponding to specific domains of the protein. Fibroblasts produce fibronectin mRNAs that contain exons EIIIA and EIIIB; these exons encode a protein domain that binds tightly to proteins in the fibroblast plasma membrane. Consequently, this fibronectin isoform adheres fibroblasts to the extracellular matrix. Alternative splicing of the fibronectin primary transcript in hepatocytes, the major type of cell in the liver, yields mRNAs that lack the EIIIA and EIIIB exons. As a result, the fibronectin secreted by hepatocytes into the blood does not adhere tightly to fibroblasts or to most other cell types, which allows it to circulate. During formation of blood clots, however, other fibrin-binding domains of hepatocyte fibronectin bind to fibrin, one of the principal constituents of blood clots. Yet another domain of the bound fibronectin then interacts with integrins on the membranes of passing platelets, thereby expanding the clot by addition of platelets.

More than 20 different isoforms of fibronectin have been identified, each encoded by a different, alternatively spliced mRNA composed of a unique combination of fibronectin gene exons. Sequencing of large numbers of mRNAs isolated from various tissues and comparison of their sequences with genomic DNA has revealed that nearly 90 percent of all human genes are expressed as alternatively spliced mRNAs. Clearly alternative RNA splicing greatly expands the number of proteins encoded by the genomes of higher, multicellular organisms.

- Eukaryotic primary transcripts must undergo RNA processing to yield functional RNAs. During processing, the ends of nearly all primary transcripts from protein-coding genes are modified by addition of a $5^{\prime}$ cap and $3^{\prime}$ poly(A) tail. Transcripts from genes containing introns undergo splicing, the removal of the introns and joining of the exons (see Figure 5-15).
- The individual domains of multidomain proteins found in higher eukaryotes are often encoded by individual exons or a small number of exons. Distinct isoforms of such proteins are often expressed in specific cell types as the result of alternative splicing of exons.


### 5.3 The Decoding of mRNA by tRNAs

Although DNA stores the information for protein synthesis and mRNA conveys the instructions encoded in DNA, most biological activities are carried out by proteins. As we saw in Chapter 3, the linear order of amino acids in each protein determines its three-dimensional structure and activity. For this reason, assembly of amino acids in their correct order, as encoded in DNA, is critical to production of functional proteins and hence the proper functioning of cells and organisms.

Translation is the whole process by which the nucleotide sequence of an mRNA is used as a template to join the amino acids of a polypeptide chain in the correct order (see Figure 5-1, step ). In eukaryotic cells, protein synthesis occurs in the cytoplasm, where three types of RNA molecules come together to perform different but cooperative functions (Figure 5-17):

1. Messenger RNA (mRNA) carries the genetic information transcribed from DNA in a linear form. The mRNA is read in sets of three-nucleotide sequences, called codons, each of which specifies a particular amino acid.
2. Transfer RNA (tRNA) is the key to deciphering the codons in mRNA. Each type of amino acid has its own subset of tRNAs, which are covalently bound to that amino acid and carry it to the growing end of a polypeptide chain when the next codon in the mRNA calls for it. The correct tRNA with its attached amino acid is selected at each step because each specific tRNA molecule contains a threenucleotide sequence, an anticodon, that can base-pair with its complementary codon in the mRNA.
3. Ribosomal RNA (rRNA) associates with a set of proteins to form ribosomes. These complex structures, which physically move along an mRNA molecule, catalyze the assembly of amino acids into polypeptide chains. They also bind tRNAs and various accessory proteins necessary for protein synthesis. Ribosomes are composed of a large and a small subunit, each of which contains its own rRNA molecule or molecules.
![li212239539b](li212239539b.jpg)

FIGURE 5-17 Three roles of RNA in protein synthesis. Messenger RNA (mRNA) is translated into protein by the joint action of transfer RNA (tRNA) and the ribosome, which is composed of numerous proteins and three (bacterial) or four (eukaryotic) ribosomal RNA (rRNA) molecules (not shown). Note the base pairing between tRNA anticodons and complementary codons in the mRNA. Formation of a peptide bond between the amino-group N on the incoming aa-tRNA and the carboxy-terminal C on the growing protein chain (green) is catalyzed by one of the rRNAs. aa = amino acid; $R=$ side group. Note that these are simplified diagrams of tRNAs and the ribosomal subunits. Their actual structures are shown in Figure 5-20b and Figure 5-22.

These three types of RNA participate in the synthesis of proteins in all organisms. In this section, we focus on the decoding of mRNA by tRNAs and how the structure of each of these RNAs relates to its specific task. How they work together with ribosomes and protein factors to synthesize proteins is detailed in the following section. Because translation is essential for protein synthesis, the two processes are commonly referred to interchangeably. However, the polypeptide chains resulting from translation must undergo posttranslational folding and often other changes (e.g., chemical modifications, association with other chains) that are required for the production of mature, functional proteins (see Chapter 3).

## Messenger RNA Carries Information from DNA in a Three-Letter Genetic Code

As noted above, the genetic code used by cells is a triplet code, in which every three-nucleotide sequence, or codon, is "read" from a specified starting point in the mRNA. Of the 64 possible codons in the genetic code (one of four

TABLE 5-1 The Genetic Code (Codons to Amino Acids)*

|  |  | Second position |  |  |  |  |
| :--: | :--: | :--: | :--: | :--: | :--: | :--: |
|  |  | U | C | A | G |  |
|  | U | Phe | Ser | Tyr | Cys | U |
|  |  | Phe | Ser | Tyr | Cys | C |
|  |  | Leu | Ser | Stop | Stop | A |
|  |  | Leu | Ser | Stop | Trp | G |
|  | C | Leu | Pro | His | Arg | U |
|  |  | Leu | Pro | His | Arg | C |
|  |  | Leu | Pro | Gln | Arg | A |
|  |  | Leu (Met)* | Pro | Gln | Arg | G |
|  | A | Ile | Thr | Asn | Ser | U |
|  |  | Ile | Thr | Asn | Ser | C |
|  |  | Ile | Thr | Lys | Arg | A |
|  |  | Met (Start) | Thr | Lys | Arg | G |
|  | G | Val | Ala | Asp | Gly | U |
|  |  | Val | Ala | Asp | Gly | C |
|  |  | Val | Ala | Glu | Gly | A |
|  |  | Val (Met)* | Ala | Glu | Gly | G |

*AUG is the most common initiation codon; GUG usually codes for valine and CUG for leucine, but rarely, these codons can also code for methionine to initiate a protein chain.
nucleotides at each of the three positions of a codon yields $4 \times 4 \times 4=64$ possible codons), 61 specify individual amino acids, and three are stop codons. Table 5-1 shows that most amino acids are encoded by more than one codon. Only two-methionine and tryptophan-have a single codon; at the other extreme, leucine, serine, and arginine are each specified by six different codons. The different codons for a given amino acid are said to be synonymous. The code itself is termed degenerate, meaning that a particular amino acid can be specified by several codons.

Synthesis of all polypeptide chains in prokaryotic and eukaryotic cells begins with the amino acid methionine. In bacteria, a specialized form of methionine with a formyl group linked to its amino group is used. In most mRNAs, the start (initiation) codon specifying this amino-terminal methionine is AUG. In a few bacterial mRNAs, GUG is used as the initiation codon, and CUG is occasionally used as an initiation codon for methionine in eukaryotes. The three codons UAA, UGA, and UAG do not specify amino acids, but rather constitute stop (termination) codons that mark the carboxyl terminus of polypeptide chains in almost all cells. The sequence of codons that runs from a specific start codon to a stop codon is called a reading frame. This precise linear
array of ribonucleotides in groups of three in mRNA specifies the precise linear sequence of amino acids in a polypeptide chain and also signals where synthesis of the chain starts and stops.

Because the genetic code is a non-overlapping triplet code without divisions between codons, a particular mRNA theoretically could be translated in three different reading frames. Indeed, some mRNAs have been shown to contain overlapping information that can be translated in different reading frames, yielding different polypeptides (Figure 5-18). The vast majority of mRNAs, however, can be read in only one frame because stop codons encountered in the other two possible reading frames terminate translation before a functional protein is produced. Very rarely, another unusual coding arrangement occurs because of frame shifting. In this case, the protein-synthesizing machinery may read four nucleotides as one amino acid and then continue reading triplets, or it may back up one base and read all succeeding triplets in the new reading frame until termination of the chain occurs. Only a few dozen such instances are known.

The meaning of each codon is the same in most known organisms-strong evidence that life on Earth evolved only

Frame 1
5' GCU UGU UUA CGA AUUAA - mRNA

Alu Cys Leu Arg Phe Polypeptide 1
Frame 2
5' -G CUU GUU UAC GAA UUA A - mRNA

Leu Val Tyr Glu Leu Polypeptide 2
Frame 3
5'-GC UUG UUU ACG AAU UAA - mRNA

Leu Phe Phe Phe Phe Polypeptide 3

FIGURE 5-18 Multiple reading frames in an mRNA sequence. If translation of the mRNA sequence shown begins at three different upstream start sites (not shown), then three overlapping reading frames are possible. In this example, the codons are shifted one base to the right in the middle frame and two bases to the right in the third frame, which ends in a stop codon. As a result, the same mRNA nucleotide sequence can specify different amino acids. Although regions of sequence that are translated in more than one of the three possible reading frames are rare, there are examples in both prokaryotes and eukaryotes, and especially in their viruses, in which the same sequence is used in two alternative mRNAs expressed from the same region of DNA, and the sequence is read in one reading frame in one mRNA and in an alternative reading frame in the other mRNA. There are even a few instances in which the same short sequence is read in all three possible reading frames.
once. In fact, the genetic code shown in Table 5-1 is known as the universal code. However, the genetic code has been found to differ for a few codons in many mitochondria, in ciliated protozoans, and in Acetabularia, a single-celled plant. As shown in Table 5-2, most of these differences involve the reading of normal stop codons as amino acids, not an exchange of one amino acid for another. These exceptions to the universal code probably were later evolutionary developments; that is, at no single time was the code immutably fixed, although massive changes were not tolerated once a general code began to function early in evolution.

## The Folded Structure of tRNA Promotes Its Decoding Functions

Translation, or decoding, of the four-nucleotide language of DNA and mRNA into the twenty-amino acid language of proteins requires both tRNAs and enzymes called aminoacyl-tRNA synthetases. To participate in protein synthesis, a tRNA molecule must become chemically linked to a particular amino acid via a high-energy bond, forming an aminoacyl-tRNA (Figure 5-19). The anticodon in the tRNA then base-pairs with a codon in mRNA so that the activated amino acid can be added to the growing polypeptide chain (see Figure 5-17).

Some 30-40 different tRNAs have been identified in bacterial cells and as many as $50-100$ in animal and plant cells. Thus the number of tRNAs in most cells is more than the number of amino acids used in protein synthesis (20) and also differs from the number of amino acid codons in the genetic code (61). Consequently, many amino acids have more than one tRNA to which they can attach (explaining how there can be more tRNAs than amino acids); in addition, many tRNAs can pair with more than one codon (explaining how there can be more codons than tRNAs).

The function of tRNA molecules, which are 70-80 nucleotides long, depends on their precise three-dimensional structures. In solution, all tRNA molecules fold into a similar stem-loop arrangement that resembles a cloverleaf when drawn in two dimensions (Figure 5-20a). The four stems are short double helices stabilized by Watson-Crick base pairing; three of the four stems have loops containing seven or eight bases at their ends, while the remaining, unlooped stem contains the free $3^{\prime}$ and $5^{\prime}$ ends of the chain. The three nucleotides composing the anticodon are located at the center of the middle loop, in an accessible position that facilitates codon-anticodon base pairing. In all tRNAs, the $3^{\prime}$ end of the unlooped acceptor stem, to which a specific amino acid is attached, has the sequence CCA, which in most cases is added after synthesis and processing of the tRNA are complete. Several bases in most tRNAs are also modified after transcription, creating nonstandard nucleotides such as inosine, dihydrouridine, and pseudouridine. As we will see shortly, some of these modified bases are known to play an important role in protein

TABLE 5-2 Known Deviations from the Universal Genetic Code

| Codon | Universal code | Unusual code* | Occurrence |
| :-- | :--: | :--: | :-- |
| UGA | Stop | Trp | Mycoplasma, Spiroplasma, mitochondria of many species |
| CUG | Leu | Thr | Mitochondria in yeasts |
| UAA, UAG | Stop | Gln | Acetabularia, Tetrahymena, Paramecium, etc. |
| UGA | Stop | Cys | Euplotes |

*Found in nuclear genes of the listed organisms and in mitochondrial genes as indicated.
source: Data from S. Osawa et al., 1992, Microbiol. Rev. 56:229.

![li222239539b](li222239539b.jpg)
synthesis. Viewed in three dimensions, the folded tRNA molecule has an L shape, with the anticodon loop and acceptor stem forming the ends of the two arms (Figure 5-20b).

## Nonstandard Base Pairing Often Occurs Between Codons and Anticodons

If perfect Watson-Crick base pairing between codons and anticodons were required, cells would have to contain at least 61 different types of tRNAs, one for each codon that specifies an amino acid. As noted above, however, many cells contain fewer than 61 tRNAs. The explanation for the smaller number lies in the capability of a single tRNA anticodon to recognize more than one, but not necessarily every, codon corresponding to a given amino acid. This broader recognition can occur because of nonstandard pairing between bases in the so-called wobble position: that is, the third ( $3^{\prime}$ ) base in an mRNA codon and the corresponding first ( $5^{\prime}$ ) base in its tRNA anticodon.

The first and second bases of a codon almost always form standard Watson-Crick base pairs with the third and second bases, respectively, of the corresponding anticodon, but four nonstandard interactions can occur between bases

FIGURE 5-19 Translating nucleic acid sequence into amino acid sequence. (a) The process for translating nucleic acid sequences in mRNA into amino acid sequences in proteins involves two steps. Step (1) An aminoacyl-tRNA synthetase first couples a specific amino acid, via a high-energy ester bond (yellow), to either the $2^{\prime}$ or $3^{\prime}$ hydroxyl of the terminal adenosine in the corresponding tRNA. Step $\mathbf{2}$ A threebase sequence in the tRNA (the anticodon) then base-pairs with a codon in the mRNA specifying the attached amino acid. If an error occurs in either step, the wrong amino acid may be incorporated into a polypeptide chain. Phe = phenylalanine. (Note that this is a simplified diagram of tRNA ${ }^{\text {Phe }}$; its actual structure is shown in Figure 5-20b.) (b) Molecular model of the human mitochondrial aminoacyl-tRNA synthetase for Phe in complex with tRNA ${ }^{\text {Phe }}$. [Data from Klipcan L., et al., 2012. J. Mol. Biol. 415:527, PDB ID 3tup.]
in the wobble position. Particularly important is the G$\cdot$U base pair, which fits into the short, 3-bp RNA-RNA doublestranded region formed between the codon and the anticodon almost as well as the standard G$\cdot$C pair. Thus a tRNA anticodon with G in the first (wobble) position can basepair with the two corresponding codons that have either pyrimidine ( C or U ) in the third position (Figure 5-21). For example, the phenylalanine codons UUU and UUC $\left(5^{\prime} \rightarrow 3^{\prime}\right)$ are both recognized by the tRNA that has GAA $\left(5^{\prime} \rightarrow 3^{\prime}\right)$ as its anticodon. In fact, any two codons of the type NNPyr ( $\mathrm{N}=$ any base; $\mathrm{Pyr}=$ pyrimidine) encode a single amino acid and are decoded by a single tRNA with G in the first (wobble) position of the anticodon.

Although adenine is rarely found in the anticodon wobble position, many tRNAs in plants and animals contain inosine (I), a deaminated product of adenine, at this position. Inosine can form nonstandard base pairs with A, C, and U. A tRNA with inosine in the wobble position thus can recognize the corresponding mRNA codons with $\mathrm{A}, \mathrm{C}$, or U in the third (wobble) position (see Figure 5-21). For this reason, inosine-containing tRNAs are heavily employed in translation of the synonymous codons that specify a single amino acid. For example, four of the six codons for leucine (CUA,

(a)
![li232239539b](li232239539b.jpg)

FIGURE 5-20 Structure of tRNAs. (a) Although the exact nucleotide sequence varies among tRNAs, they all fold into four base-paired stems and three loops. The CCA sequence at the $3^{\prime}$ end is also found in all tRNAs. Attachment of an amino acid to the $3^{\prime}$ A yields an aminoacyltRNA. Some of the A, C, G, and U residues are modified post-transcriptionally in most tRNAs (see key). Dihydrouridine (D) is nearly always present in the D loop; likewise, ribothymidine (T) and pseudouridine
![li242239539b](li242239539b.jpg)

If these bases are in
first, or wobble, position of anticodon

| C | A | G | U | I |  |
| :--: | :--: | :--: | :--: | :--: | :--: |
| G | U | C | A | C | then the tRNA may |
|  |  | U | G | A | recognize codons in |
|  |  |  |  | U | mRNA having these |
|  |  |  |  |  | bases in third position |

(b)
![li252239539b](li252239539b.jpg)
$(\psi)$ are almost always present in the TΨCG loop. Yeast alanine tRNA, represented here, also contains other modified bases. The triplet at the tip of the anticodon loop base-pairs with the corresponding codon in mRNA. See R. W. Holly et al., 1965, Science 147:1462. (b) Threedimensional model of the generalized backbone of all tRNAs. Note the L shape of the molecule. (Part (b) data from J. G. Arnez and D. Moras, 1997, Trends Biochem. Sci. 22:211, PDB ID 1vtr4.)

## FIGURE 5-21 Nonstandard base pairing at the wobble position.

The base in the third (or wobble) position of an mRNA codon often forms a nonstandard base pair with the base in the first (or wobble) position of a tRNA anticodon. Wobble pairing allows a tRNA to recognize more than one mRNA codon (top); conversely, it allows a codon to be recognized by more than one kind of tRNA (bottom), although each of those tRNAs will bear the same amino acid. Note that a tRNA with I (inosine) in the wobble position can "read" (become paired with) three different codons, and a tRNA with G or U in the wobble position can read two codons. Although A is theoretically possible in the wobble position of the anticodon, it is almost never found in nature. (Note that this is a simplified diagram of a tRNA. The actual structure of a tRNA is shown in Figure 5-20b.)

CUC, CUU, and UUA) are all recognized by the same tRNA with the anticodon $3^{\prime}$-GAI- $5^{\prime}$; the inosine in the wobble position forms nonstandard base pairs with the third base in each of these four codons. In the case of the UUA codon, a nonstandard G-U pair also forms between position 3 of the anticodon and position 1 of the codon.

## Amino Acids Become Activated When Covalently Linked to tRNAs

Recognition of the codon or codons specifying a given amino acid by a particular tRNA is actually the second step in decoding the genetic message. The first step, attachment of the appropriate amino acid to a tRNA, is catalyzed by a specific aminoacyl-tRNA synthetase. Each of the 20 different synthetases recognizes one amino acid and all its compatible, or cognate, tRNAs. These coupling enzymes link an amino acid to the free $2^{\prime}$ or $3^{\prime}$ hydroxyl of the adenosine at the $3^{\prime}$ terminus of the tRNA molecule by an ATP-requiring reaction. In this reaction, the amino acid is linked to the tRNA by a highenergy bond and is thus said to be activated. The energy of this bond subsequently drives the formation of the peptide bonds linking adjacent amino acids in a growing polypeptide chain. The equilibrium of the aminoacylation reaction is driven further toward activation of the amino acid by hydrolysis of the high-energy phosphoanhydride bond in the released pyrophosphate (see Figure 5-19).

Aminoacyl-tRNA synthetases recognize their cognate tRNAs primarily by interacting with the anticodon loop and acceptor stem, although interactions with other regions of a tRNA also contribute to recognition in some cases. Furthermore, specific bases in incorrect tRNAs that are structurally similar to a cognate tRNA will inhibit charging of the incorrect tRNA. Thus recognition of the correct tRNA depends on both positive interactions and the absence of negative interactions. Still, because some amino acids are so similar structurally, aminoacyl-tRNA synthetases sometimes make mistakes. These mistakes are corrected, however, by the enzymes themselves, which have a proofreading activity that checks the fit in their amino acid-binding pocket. If the wrong amino acid becomes attached to a tRNA, the bound synthetase catalyzes removal of the amino acid from the tRNA. This crucial function helps guarantee that a tRNA delivers the correct amino acid to the protein-synthesizing machinery. The overall error rate for translation in E. coli is very low, approximately 1 per 50,000 codons, evidence of both the fidelity of tRNA recognition and the importance of proofreading by aminoacyl-tRNA synthetases.

## KEY CONCEPTS OF SECTION 5.3

## The Decoding of mRNA by tRNAs

- Genetic information from DNA is carried by mRNA in the form of a non-overlapping, degenerate triplet code.
- Each amino acid is encoded by one or more threenucleotide sequences (codons) in mRNA. Each codon specifies one amino acid, but most amino acids are encoded by multiple codons (see Table 5-1).
- The AUG codon for methionine is the most common start codon, specifying the amino acid at the amino-terminus of a protein chain. Three codons (UAA, UAG, UGA) function as stop codons and specify no amino acids.
- A reading frame, the uninterrupted sequence of codons in mRNA from a specific start codon to a stop codon, is translated into the linear sequence of amino acids in a polypeptide chain.
- Decoding of the nucleotide sequence in mRNA into the amino acid sequence of proteins depends on tRNAs and aminoacyl-tRNA synthetases.
- All tRNAs have a similar three-dimensional structure that includes an acceptor stem for attachment of a specific amino acid and a stem-loop with a three-base anticodon sequence at its end (see Figure 5-20). The anticodon can base-pair with its corresponding codon in mRNA.
- Because of nonstandard interactions, a tRNA may basepair with more than one mRNA codon; conversely, a particular codon may base-pair with multiple tRNAs. In each case, however, only the proper amino acid is inserted into a growing polypeptide chain.
- Each of the 20 aminoacyl-tRNA synthetases recognizes a single amino acid and covalently links it to a cognate tRNA, forming an aminoacyl-tRNA (see Figure 5-19). This reaction activates the amino acid so that it can participate in peptide bond formation.


### 5.4 Stepwise Synthesis of Proteins on Ribosomes

The previous sections have introduced two of the major participants in protein synthesis: mRNA and aminoacyl-tRNA. Here we first describe the third key player in protein syn-thesis-the rRNA-containing ribosome-before taking a detailed look at how all three components are brought together to carry out the biochemical events leading to the formation of polypeptide chains by ribosomes. Like transcription, the complex process of translation can be divided into three stages-initiation, elongation, and termination-which we consider in order. We focus our description on translation in eukaryotic cells, but the mechanism of translation is fundamentally the same in all cells.

## Ribosomes Are Protein-Synthesizing Machines

If the many components that participate in translating mRNA had to interact in free solution, the likelihood of their

coming together would be so low that the rate of amino acid polymerization would be very slow. The efficiency of translation is greatly increased by the binding of mRNA and the individual aminoacyl-tRNAs within a ribosome. The ribosome, the most abundant RNA-protein complex in the cell, directs elongation of polypeptides at a rate of 3 to 5 amino acids added per second. Small proteins of 100-200 amino acids are therefore made in a minute or less. On the other hand, it takes $2-3$ hours to make the largest known protein, titin, which is found in muscle and contains about 30,000 amino acid residues. The cellular machine that accomplishes this task must be precise and persistent.

With the aid of the electron microscope, ribosomes were first discovered as small, discrete, RNA-rich particles in cell types that secrete large amounts of protein. However, their role in protein synthesis was not recognized until reasonably pure ribosome preparations were obtained. In vitro radiolabeling experiments with such preparations showed that radioactive amino acids were incorporated into growing polypeptide chains associated with ribosomes before appearing in finished chains.

Although there are differences between the ribosomes of bacteria, archaea, and eukaryotes, the great structural and functional similarities between ribosomes from all species reflect the common evolutionary origin of the most basic constituents of living cells. A ribosome is composed of three (in bacteria and archaea) or four (in eukaryotes) different rRNA molecules and as many as 80 proteins, organized into a large subunit and a small subunit (Figure 5-22 and Table 5-3). The ribosomal subunits and the rRNA molecules are commonly designated in svedberg units (S), a measure of
![li262239539b](li262239539b.jpg)

FIGURE 5-22 Structure of the bacterial ribosome. Model of the T. thermophilus ribosome viewed along the interface between the large (50S) and small (30S) subunits. The 16S rRNA and proteins in the small subunit are dark gray. RNA is depicted as a tube model and protein surfaces are shown; the 23S rRNA and proteins in the large subunit are light gray; and the 5S rRNA is an intermediate shade of gray. The surface of the ribosome is made partially transparent to display the positions of tRNAs in the A, P, and E sites. Note that the ribosomal proteins are located primarily on the surface of the ribosome. [Data from A. Korostelev et al., 2006, Cell 126:1065-1077, PDB ID 4v4L]

TABLE 5-3 Ribosome Components

| Common core | E. coli | S. cerevisiae | Human |
| :-- | :-- | :-- | :-- |
| 2.0 MDa | 2.3 MDa | 3.3 MDa | 4.3 MDa |
| 34 proteins | 54 proteins | 79 proteins | 80 proteins |
| 3 rRNAs | 3 rRNAs | 4 rRNAs | 4 rRNAs |
| Large subunit |  |  |  |
|  | 50S | 60S | 60S |
| 19 proteins | 33 proteins | 46 proteins | 47 proteins |
| 23S rRNA: 2843 bases | 23S rRNA: 2904 bases | 25S rRNA: 3396 bases* | 28S rRNA: 5034 bases* |
|  |  | 5.8S rRNA: 158 bases* | 5.8S rRNA: 156 bases* |
| 5s rRNA: 121 bases | 5S rRNA: 121 bases | 5S rRNA: 121 bases | 5S rRNA: 121 bases |
| Small subunit |  |  |  |
|  | 30S | 40S | 40S |
| 15 proteins | 21 proteins | 33 proteins | 33 proteins |
| 16S rRNA: 1458 bases | 16S rRNA: 1542 bases | 18S rRNA: 1800 bases | 18S rRNA: 1870 bases |

[^0]
[^0]:    *5.8S rRNA in eukaryotes is base-paired to 25 S or 28 S rRNA.
    source: Data from G. Yusupov and M. Yusupov, Ann. Rev. Biochem., 2014, 83:467.

![li272239539b](li272239539b.jpg)

FIGURE 5-23 Comparison of the common core structure at the center of ribosomes from all domains of life and bacterial, yeast, and human ribosomes. (a) RNA in the common core structure is shown in light blue and protein domains common to all ribosomes are shown in pink. Additions to the common core structure are shown in dark
the sedimentation rate of macromolecules centrifuged under standard conditions-essentially, a logarithmic measure of size. The small ribosomal subunit contains a single rRNA molecule, referred to as small rRNA. The large subunit contains a molecule of large rRNA and one molecule of 5 S rRNA, plus an additional molecule of 5.8 S rRNA in vertebrates. The lengths of the rRNA molecules, the numbers of proteins in each subunit, and consequently, the sizes of the subunits differ between bacterial and eukaryotic cells (see Table 5-3). The assembled ribosome is 70 S in bacteria and 80 S in vertebrates.

The sequences of the small and large rRNAs from several thousand organisms are now known. Although the primary nucleotide sequences of these rRNAs vary considerably, the same parts of each type of rRNA theoretically can form base-paired hairpins, stem-loops, and loop-loop interactions (see Figure 5-9), which would generate a similar three-dimensional core structure in the rRNAs and proteins of all organisms. The three-dimensional structures of bacterial and yeast ribosomes (see Figure 5-23) and of the large subunit of an archaeal ribosome have been determined by x-ray crystallography. The structures of human (see Figure 5-23) and plant ribosomes have also been determined by cryoelectron microscopy. The structure of the rRNAs in the common core, where mRNAs and tRNAs are bound and where peptide bond formation is catalyzed, is similar in all three domains of life. However, archaeal rRNAs and proteins are more similar to those of eukaryotic ribosomes than to those of bacterial ribosomes, reflecting their later divergence from a common ancestor (see Figure 1-1). For the most part, the multiple ribosomal proteins are much smaller than the rRNAs and associate with the surface of the ribosomes. Although the number of protein molecules in ribosomes greatly exceeds the number of RNA molecules,
(c) Lower eukaryotes
(S. cerevisiae)
![li282239539b](li282239539b.jpg)
blue for RNA and red for proteins in ribosomes from T. thermophilus (b) and S. cerevisiae (c). Human ribosome structure (d) from cryoelectron microscopy. A tRNA visible in the E site is shown in green. [Data for (a, b, c) from G. Yusupova and M. Yusupov, 2014, Annu. Rev. Biochem. 83:467; data for (d) from H. Khatter et al., 2015, Nature 520:640; PDB ID 4ug0.]

RNA constitutes about 60 percent of the mass of a bacterial ribosome, and about 50 percent of the mass of a human ribosome. Eukaryotic ribosomes are generally similar to bacterial ribosomes, but are larger because of eukaryote-specific insertions of RNA segments into regions of the common core rRNAs as well as the presence of a larger number of proteins (see Figure 5-23 and Table 5-3). Basic aspects of protein synthesis are thought to be similar among all three domains, although initiation of translation in eukaryotes, discussed later, is more complex and subject to additional mechanisms of regulation.

The high-resolution structures of ribosomes are providing new insights into the mechanism by which many antibiotics inhibit bacterial protein synthesis without affecting the function of mammalian ribosomes. These insights are providing important clues for the design and synthesis of new antibiotics. Such research is desperately needed as the occurrence of bacteria resistant to currently available antibiotics becomes increasingly more common, especially in hospitals, where antibiotic-resistant bacteria are under positive selection.

## Methionyl-tRNA ${ }_{i}{ }^{\text {Met }}$ Recognizes the AUG Start Codon

As noted earlier, the AUG codon for methionine functions as the start codon in the vast majority of mRNAs. A critical aspect of translation initiation is to begin protein synthesis at the start codon, thereby establishing the correct reading frame for the entire mRNA. Both bacteria and eukaryotes contain two different methionine tRNAs: $\mathrm{rRNA}_{\mathrm{i}}{ }^{\text {Met }}$ can initiate protein synthesis, and $\mathrm{rRNA}^{\text {Met }}$ can incorporate methionine only into a growing protein chain. The same

aminoacyl-tRNA synthetase (MetRS) charges both tRNAs with methionine. But only Met-tRNA ${ }_{i}{ }^{\text {Met }}$ (i.e., activated methionine attached to $\mathrm{tRNA}_{\mathrm{i}}{ }^{\text {Met }}$ ) can bind at the appropriate site on the small ribosomal subunit, the $P$ site, to begin synthesis of a polypeptide chain. The regular Met-tRNA ${ }^{\text {Met }}$, and all other charged tRNAs, bind only to the $A$ site, as described later. tRNAs enter the exit or $E$ site after transferring their covalently bound amino acids to the growing polypeptide chain.

## Eukaryotic Translation Initiation Usually Occurs at the First AUG Downstream from the $5^{\prime}$ End of an mRNA

During the first stage of translation, the small and large ribosomal subunits assemble around an mRNA that has a Met-tRNA ${ }_{i}{ }^{\text {Met }}$ correctly positioned at the start codon in the ribosomal P site. In eukaryotes, the assembly of this complex is mediated by a special set of proteins known as eukaryotic translation initiation factors (eIFs). As each individual component joins the complex, it is guided by interactions with specific eIFs. Several of the initiation factors bind GTP, and the hydrolysis of GTP to GDP functions as a proofreading switch that allows subsequent steps to proceed only if the preceding step has occurred correctly. Before GTP hydrolysis, the complex is unstable, allowing dissociation of the components and a second attempt at complex formation until the correct complex assembles, resulting in GTP hydrolysis and stabilization of the appropriate complex.

Considerable progress has been made in the past few years in understanding translation initiation in vertebrates. The current model for initiation of translation in vertebrates is depicted in Figure 5-24. Large and small ribosomal subunits released from a previous round of translation are kept apart by the binding of eIFs 1, 1A, and 3 to the small 40 S subunit (Figure 5-24, top). The first step of translation initiation is formation of a 43 S preinitiation complex. This preinitiation complex is formed when the 40 S subunit with eIFs 1,1A, and 3 associates with eIF5 and a ternary (three-part) complex consisting of the Met-tRNA ${ }_{i}{ }^{\text {Met }}$ and eIF2 bound to GTP (Figure 5-24, steps $\mathbf{\square}$ and $\mathbf{\square}$ ). The initiation factor eIF2 alternates between association with GTP and GDP; it can bind Met-tRNA ${ }_{i}{ }^{\text {Met }}$ only when it is associated with GTP. Cells can inhibit protein synthesis by phosphorylating a serine residue on the eIF2 bound to GDP; the phosphorylated complex is unable to exchange the bound GDP for GTP and cannot bind Met-tRNA ${ }_{i}{ }^{\text {Met }}$ so protein synthesis cannot occur.

The mRNA to be translated is bound by the multisubunit eIF4 complex, which interacts with both the $5^{\prime}$ cap and the cytoplasmic poly(A)-binding protein (PABPC) bound in multiple copies to the mRNA poly(A) tail. Both interactions are required for translation of most mRNAs. This binding results in the formation of a circular complex (Figure 5-24, step $\mathbf{\square}$ ). The eIF4 cap-binding complex consists of several subunits with different functions. The eIF4E subunit binds
the $5^{\prime}$ cap on mRNAs (see Figure 5-14). The large eIF4G subunit binds cooperatively to several PABPC proteins bound to the mRNA poly(A) tail, and also forms a scaffold to which the other eIF4 subunits bind. The mRNA-eIF4 complex then associates with the preinitiation complex through an interaction between eIF4G and eIF3 (step $\mathbf{\square}$ ).

The initiation complex then slides along, or scans, the associated mRNA as the helicase activity of eIF4A, stimulated by eIF4B, uses energy from ATP hydrolysis to unwind the RNA secondary structure (step $\mathbf{\square}$ ). Scanning stops when the tRNA ${ }_{i}{ }^{\text {Met }}$ anticodon recognizes the start codon, which is the first AUG downstream from the $5^{\prime}$ end in most eukaryotic mRNAs. Recognition of the start codon leads to hydrolysis of the GTP associated with eIF2, an irreversible step that prevents further scanning, resulting in formation of the $48 S$ initiation complex (step $\mathbf{\square}$ ). This commitment to the correct initiation codon is facilitated by eIF5, an eIF2 GTPaseactivating protein (GAP, see Figure 3-34). Selection of the initiating AUG is facilitated by specific surrounding nucleotides called the Kozak sequence, for Marilyn Kozak, who defined it: $\left(5^{\prime}\right) \mathrm{ACCAUGG}\left(3^{\prime}\right)$. The A preceding the AUG (in bold) and the G immediately following it are the most important nucleotides affecting translation initiation efficiency.

Association of the large (60S) subunit with the small subunit, which is mediated by eIF5B bound to GTP, results in displacement of many of the initiation factors (step $\mathbf{\square}$ ). Correct association between the ribosomal subunits results in hydrolysis of the eIF5B-bound GTP to GDP and the release of eIF5B-GDP and eIF1A (step $\mathbf{\square}$ ), completing the formation of an 80 S initiation complex. Coupling of the ribosome subunit-joining reaction to GTP hydrolysis by eIF5B allows the initiation process to continue only when the subunit interaction has occurred correctly. It also makes this an irreversible step, so that the ribosomal subunits do not dissociate until the entire mRNA is translated and protein synthesis is terminated.

The eukaryotic protein-synthesizing machinery begins translation of most cellular mRNAs within about 100 nucleotides of the $5^{\prime}$-capped end as just described. However, some cellular mRNAs contain an internal ribosome entry site (IRES) located far downstream from the $5^{\prime}$ end. It is thought that cellular IRESs form RNA structures that interact with a complex of eIF4A and eIF4G, which then associates with eIF3 bound to a 40 S subunit with eIF1 and eIF1A. This assembly then binds an eIF2 ternary complex to assemble an initiation complex directly on a neighboring AUG codon. In addition, translation of some positive-stranded viral RNAs, which lack a $5^{\prime}$ cap, is initiated at viral IRES sequences. These RNAs fall into different classes depending on how many of the standard eIFs are required for initiation. In the case of cricket paralysis virus, the $\sim 200$-nt-long IRES folds into a complex structure that interacts directly with the 40 S ribosomal subunit and leads to initiation without any of the eIFs or even the initiator Met-tRNA ${ }_{i}{ }^{\text {Met }}$ !

In bacteria, binding of the small ribosomal subunit to an initiation site occurs by a different mechanism that

![li292239539b](li292239539b.jpg)

FIGURE 5-24 Initiation of translation in eukaryotes. The current model of eukaryotic initiation involves eight steps. Step $\mathbf{E}$ An eIF2 ternary complex forms when eIF2-GTP binds a tRNA ${ }^{\text {Met }}$. Step $\mathbf{E}$ When a ribosome dissociates at the termination of translation, the 40 S subunit is bound by eIF1, eIF1A, and eIF3. A 43 S preinitiation complex forms when this subunit associates with an eIF2 ternary complex and eIF5. Step $\mathbf{E}$ An mRNA is activated when a multisubunit eIF4 complex binds: subunit eIF4E binds to the $5^{\prime}$ cap, and subunit eIF4G binds multiple copies of the cytoplasmic poly(A)-binding protein (PABPC) bound to the mRNA poly(A) tail. For simplicity, binding of only one PABPC to eIF4G is shown. Then eIF4B, which stimulates eIF4A helicase activity, also joins this circular complex in which both the mRNA $5^{\prime}$ cap and poly(A) tail are associated with the eIF4 complex. Step The 43S preinitiation complex binds an eIF4-mRNA complex. Step The RNA helicase activity of subunit eIF4A unwinds any RNA secondary structure at the $5^{\prime}$ end of the mRNA as the 40 S subunit scans in the $5^{\prime} \rightarrow 3^{\prime}$ direction until it recognizes the initiation codon. For simplicity, eIF4E is diagrammed as releasing from the remainder of the eIF4 complex, but in reality, it remains associated, forming a loop in the mRNA between the $5^{\prime}$ cap and the scanning eIF4 complex. Step $\mathbf{E}$ Recognition of the initiation codon causes eIF5 to stimulate hydrolysis of eIF2-bound GTP. This switches the conformation of the scanning complex to a 48 S initiation complex with the anticodon of tRNA ${ }^{\text {Met }}$ base-paired to the initiator AUG in the 40S-subunit P site. Step The 60S subunit joins the 40 S subunit, leading to the release of most of the earlier-acting eIFs as eIF5B-GTP binds to eIF1A in the ribosomal A site. The released eIF4 complex and eIF4B associate with the $5^{\prime}$ cap and PABPC as shown in step $\mathbf{B}$ to prepare for interaction with another 43 S preinitiation complex. For simplicity, this association is not shown. Step Correct association of the 40 S and 60 S subunits results in hydrolysis of eIF5B-bound GTP, release of eIF5B-GDP and eIF1A, and formation of the 80 S initiation complex with tRNA ${ }^{\text {Met }}$ basepaired to the initiation codon in the ribosomal P site. See R. J. Jackson et al., 2010, Nature Rev. Mol. Cell Biol. 11:113.

allows initiation at internal sites in the polycistronic mRNAs transcribed from operons. In bacterial mRNAs, a $\sim 6-\mathrm{bp}$ sequence complementary to the $3^{\prime}$ end of the small rRNA precedes the AUG start codon by 4-7 nucleotides. Base pairing between this sequence in the mRNA, called the ShineDalgarno sequence after its discoverers, and the small rRNA places the small ribosomal subunit in the proper position for initiation. Initiation factors comparable to eIF1A, eIF2, eIF3, and f-Met-tRNA ${ }_{i}{ }^{\text {Met }}$ then associate with the small subunit, followed by association of the large subunit to form the complete bacterial ribosome.

## During Chain Elongation Each Incoming Aminoacyl-tRNA Moves Through Three Ribosomal Sites

The correctly positioned ribosome-Met-tRNA ${ }_{i}{ }^{\text {Met }}$ complex is now ready to begin the task of stepwise addition of amino acids by in-frame translation of the mRNA. As is the case with initiation, a set of specialized proteins, termed translation elongation factors (EFs), is required to carry out this process of chain elongation. The key steps in elongation are the entry of each succeeding aminoacyl-tRNA with an anticodon complementary to the next codon, the formation of a peptide bond, and the movement, or translocation, of the ribosome one codon at a time along the mRNA.

At the completion of translation initiation, as noted already, Met-tRNA ${ }_{i}{ }^{\text {Met }}$ is bound to the P site on the assembled 80 S ribosome (Figure 5-25, top). This region of the ribosome is called the $P$ site because the tRNA chemically linked to the growing polypeptide chain is located here. The second aminoacyl-tRNA is brought into the ribosome as a ternary complex in association with EF1 $\alpha$-GTP and becomes bound to the $A$ site, so named because it is where aminoacyl-tRNAs bind (step 1). EF1 $\alpha$-GTP bound to various aminoacyltRNAs diffuse into the A site, but the next step in translation proceeds only when the tRNA anticodon base-pairs with the second codon in the coding region. When that occurs properly, the GTP in the associated EF1 $\alpha$-GTP is hydrolyzed. The hydrolysis of GTP promotes a conformational change

FIGURE 5-25 Chain elongation in eukaryotes. Once the 80 S ribosome with Met-tRNA ${ }_{i}{ }^{\text {Met }}$ in the ribosome $P$ site is assembled (top), a ternary complex bearing the second amino acid $\left(a a_{2}\right)$ coded by the mRNA binds to the A site (step 1). Following a conformational change in the ribosome induced by hydrolysis of GTP in EF1 $\alpha$-GTP (step 1), the large rRNA catalyzes peptide bond formation between Met, and $\mathrm{aa}_{2}$ (step 1). Hydrolysis of GTP in EF2-GTP causes another conformational change in the ribosome that results in its translocation one codon along the mRNA and shifts the unacylated tRNA ${ }_{i}^{\text {Met }}$ to the E site and the tRNA with the bound peptide to the P site (step 1). The cycle can begin again with binding of a ternary complex bearing $a a_{2}$ to the now open A site. In the second and subsequent elongation cycles, the tRNA at the E site is ejected during step as a result of the conformational change induced by hydrolysis of GTP in EF1 $\alpha$-GTP.
in EF1 $\alpha$ that leads to release of the resulting EF1 $\alpha$-GDP complex and tight binding of the aminoacyl-tRNA in the A site (step 1). This conformational change also positions the aminoacylated $3^{\prime}$ end of the tRNA in the A site close to the $3^{\prime}$ end of the Met-tRNA ${ }_{i}{ }^{\text {Met }}$ in the P site. GTP hydrolysis, and hence tight binding, does not occur if the anticodon of the incoming aminoacyl-tRNA cannot base-pair with the
![li302239539b](li302239539b.jpg)

codon at the A site. In this case, the ternary complex diffuses away, leaving an empty A site that can associate with other aminoacyl-tRNA-EF1 $\alpha \cdot$ GTP complexes until a correctly base-paired tRNA is bound. Thus GTP hydrolysis by EF1 $\alpha$ is another proofreading step that allows protein synthesis to proceed only when the correct aminoacyl-tRNA is bound to the A site. This phenomenon contributes to the fidelity of protein synthesis.

With the initiating Met-tRNA ${ }_{i}^{\text {Met }}$ at the P site and the second aminoacyl-tRNA tightly bound at the A site, the $\alpha$-amino group of the second amino acid reacts with the "activated" (ester-linked) methionine on the initiator tRNA, forming a peptide bond (Figure 5-25, step $\mathbf{B}$; see Figure 5-17). This peptidyltransferase reaction is catalyzed by the large rRNA, which precisely orients the interacting atoms, permitting the reaction to proceed. The $2^{\prime}$-hydroxyl of the terminal A of the peptidyl-tRNA in the P site also participates in catalysis. The catalytic ability of the large rRNA in bacteria has been demonstrated by carefully removing the vast majority of the protein from large ribosomal subunits. The nearly pure bacterial 23 S rRNA can catalyze a peptidyltransferase reaction between analogs of aminoacyl-tRNA and peptidyl-tRNA. Further support for the catalytic role of large rRNA in protein synthesis came from high-resolution crystallographic studies showing that no proteins lie near the site of peptide bond synthesis in the crystal structure of the bacterial large subunit.

Following peptide bond synthesis, the ribosome translocates a distance equal to one codon along the mRNA. This translocation step is monitored by hydrolysis of the GTP in eukaryotic EF2-GTP. Once translocation has occurred correctly, the bound GTP is hydrolyzed, another irreversible process that prevents the ribosome from moving along the RNA in the wrong direction or from translocating an incorrect number of nucleotides. As a result of conformational changes in the ribosome that accompany proper translocation and the resulting GTP hydrolysis by EF2, tRNA ${ }_{i}^{\text {Met }}$, now without its activated methionine, is moved to the $E$ (exit) site on the ribosome; concurrently, the second tRNA, now covalently bound to a dipeptide (a peptidyl-tRNA), is moved to the P site (Figure 5-25, step $\mathbf{B}$ ). Translocation thus returns the ribosome conformation to a state in which the A site is open and able to accept another aminoacyl-tRNA complexed with EF1 $\alpha \cdot$ GTP, beginning another cycle of chain elongation.

Repetition of the elongation cycle depicted in Figure 5-25 adds amino acids one at a time to the carboxyl terminus of the growing polypeptide as directed by the mRNA sequence until a stop codon is encountered. In subsequent cycles, the conformational change that occurs in step $\mathbf{B}$ ejects the unacylated tRNA from the E site. As the nascent polypeptide chain becomes longer, it threads through a channel in the large ribosomal subunit, exiting at a position opposite the side that interacts with the small subunit (Figures 5-22, 5-26).

In the absence of the ribosome, the three-base-pair RNARNA hybrid between the tRNA anticodons and the mRNA codons in the A and P sites would not be stable; RNA-RNA duplexes between separate RNA molecules must be considerably
![li312239539b](li312239539b.jpg)

FIGURE 5-26 Termination of translation in eukaryotes. When a ribosome bearing a nascent protein chain reaches a stop codon (UAA, UGA, UAG), release factor eRF1 enters the A site together with eRF3-GTP. Hydrolysis of the bound GTP is accompanied by cleavage of the peptide chain from the tRNA in the P site and ejection of the tRNA in the E site, forming a post-termination complex. The ribosomal subunits are separated by the action of the ABCE1 ATPase together with eIF1, eIF1A, and eIF3. The 40S subunit is released bound to these eIFs, ready to initiate another cycle of translation (see Figure 5-24).
longer to be stable under physiological conditions. However, multiple interactions between the large and small rRNAs and the general domains of tRNAs (e.g., the D and T $\psi$ CG loops,

see Figure 5-20) stabilize the tRNAs in the A and P sites, while other RNA-RNA interactions sense correct codon-anticodon base pairing, ensuring that the genetic code is read properly. Then, interactions between rRNAs and the general domains of all tRNAs result in the movement of the tRNAs between the $\mathrm{A}, \mathrm{P}$, and E sites as the ribosome translocates along the mRNA one three-nucleotide codon at a time.

## Translation Is Terminated by Release Factors When a Stop Codon Is Reached

The final stages of translation, like initiation and elongation, require highly specific molecular signals that decide the fate of the mRNA-ribosome-peptidyl-tRNA complex. Two types of specific protein release factors (RFs) have been discovered. Eukaryotic eRF1, whose shape is similar to that of tRNAs, acts by binding to the ribosomal A site and recognizing stop codons directly. Like some of the initiation and elongation factors discussed previously, the second eukaryotic release factor, eRF3, is a GTP-binding protein. The eRF3-GTP complex acts in concert with eRF1 to promote cleavage of the peptidyl-tRNA bond, thus releasing the completed protein chain and terminating translation (Figure 5-26). Bacteria have two release factors (RF1 and RF2) that are functionally analogous to eRF1 and a GTPbinding factor (RF3) that is analogous to eRF3. Once again, the eRF3 GTPase monitors the correct recognition of a stop codon by eRF1. The peptidyl-tRNA bond of the tRNA in the P site is not cleaved until one of the three stop codons is correctly recognized by eRF1, another example of a proofreading step in protein synthesis.

Release of the completed protein leaves a free tRNA in the P site and the mRNA still associated with the 80 S ribosome, to which eRF1 and eRF3-GDP are still bound in the A site. In eukaryotes, ribosome recycling occurs when this posttermination complex is bound by a protein called ABCE1, which uses energy from ATP hydrolysis to separate the subunits and release the mRNA and tRNA in the P site. Initiation factors eIF1, eIF1A, and eIF3, which are also required for separation of the subunits, load onto the 40 S subunit, making it ready for another round of initiation (see Figure $5-24, \operatorname{top}$ ). In reality, a free mRNA is never released as diagrammed in Figure 5-26 for simplicity. Rather, the mRNA has other ribosomes associated with it in various stages of elongation, PABPC bound to the poly(A) tail, and the eIF4 complex associated with the $5^{\prime}$ cap, ready to associate with another 43 S preinitiation complex (see Figure 5-24).

In addition to these functions in protein synthesis, ribosomes also associate transiently with protein chaperones that assist in folding the polypeptide chain as it emerges from the ribosome surface (see Figure 3-17). As we will see in Chapter 13, ribosomes that synthesize proteins destined to be inserted into the endoplasmic reticulum (ER), transported into the ER lumen and later secreted from the cell, or introduced into other organelles such as lysosomes, also associate with a ribonucleoprotein complex called SRP (signal recognition particle) that arrests protein synthesis until
the nascent polypeptide encounters specialized channels for insertion into the ER. SRP also assists with the insertion and threading of these proteins through these ER channels when protein synthesis is permitted to resume.

## Polysomes and Rapid Ribosome Recycling Increase the Efficiency of Translation

Translation of a single eukaryotic mRNA molecule to yield a typical-sized protein takes 1 to 2 minutes. Two phenomena significantly increase the overall rate at which cells can synthesize a protein: the simultaneous translation of a single mRNA molecule by multiple ribosomes, and rapid recycling of ribosomal subunits after they disengage from a stop codon. Simultaneous translation of an mRNA by multiple ribosomes is readily observable in electron micrographs and by sedimentation analysis, revealing mRNA molecules attached to multiple ribosomes bearing nascent growing polypeptide chains. These structures, referred to as polyribosomes or polysomes, were seen to be circular in electron micrographs of some tissues. Subsequent studies with purified initiation factors explained the circular shape of polyribosomes and suggested the mechanism by which ribosomes recycle efficiently.

These studies revealed that multiple copies of the cytoplasmic poly(A)-binding protein (PABPC) interact with both an mRNA poly(A) tail and the eIF4G subunit of eIF4. Since the eIF4E subunit of eIF4 binds to the cap structure on the $5^{\prime}$ end of an mRNA, the two ends of an mRNA molecule are bridged by the intervening proteins, forming a "circular" mRNA (Figure 5-27a). Because the two ends of a polysome are relatively close together, ribosomal subunits that disengage from a stop codon are positioned near the $5^{\prime}$ end, facilitating reinitiation by the interaction of the 40 S subunit and its associated initiation factors with eIF4 bound to the $5^{\prime}$ cap. The circular pathway depicted in Figure 5-27b is thought to enhance ribosome recycling and thus increase the efficiency of protein synthesis.

## GTPase-Superfamily Proteins Function in Several Quality-Control Steps of Translation

We can now see that one or more GTP-binding proteins participate in each stage of translation. These proteins belong to the GTPase superfamily of switch proteins that cycle between a GTP-bound active form and a GDP-bound inactive form (see Figure 3-34). Hydrolysis of the bound GTP causes a conformational change in the GTPase itself and in other associated proteins that are critical to various complex molecular processes. In translation initiation, for instance, hydrolysis of eIF2-GTP to eIF2-GDP prevents further scanning of the mRNA once the start site is encountered and allows binding of the large ribosomal subunit to the small subunit (see Figure 5-24, step $\mathbf{\square}$ ). Similarly, hydrolysis of eIF5B-GTP monitors successful association of the large and small ribosomal subunits (Figure 5-24 step $\mathbf{\square}$ ). Recall that if the correct complex does not form, eIF5B-GTP does not hydrolyze the GTP and the complex is unstable, free to diffuse apart

![li322239539b](li322239539b.jpg)

## EXPERIMENTAL FIGURE 5-27 Circular structure of mRNA

increases translation efficiency. Eukaryotic mRNA forms a circular structure owing to interactions of three proteins. (a) In the presence of purified yeast poly(A)-binding protein [PABP; there is only one PABP in S. cerevisiae, rather than a nuclear (PABPN) and cytoplasmic (PABPC) protein as in higher eukaryotes], eIF4E, and eIF4G, eukaryotic mRNAs form circular structures, visible in this force-field electron micrograph. In these structures, protein-protein and protein-mRNA interactions form a bridge between the $5^{\prime}$ and $3^{\prime}$ ends of the mRNA. (b) Model of protein synthesis on circular polysomes and recycling of ribosomal subunits. Multiple individual ribosomes can simultaneously translate a eukaryotic mRNA, shown here in a circular form stabilized by interactions between proteins bound at the $3^{\prime}$ and $5^{\prime}$ ends. When a ribosome completes translation and dissociates from the $3^{\prime}$ end, the separated subunits can rapidly find the nearby $5^{\prime}$ cap $\left(\mathrm{m}^{7} \mathrm{G}\right)$ and PABPC-bound poly(A) tail and initiate another round of synthesis. (Part (a) courtesy of Alan Sachs and Sandra Wells.)
and try again. When the precise alignment of the subunits required for elongation occurs, eIF5B hydrolyzes the GTP to GDP, locking the correct complex in place. Energy released from the high energy $\beta-\gamma$ bond in GTP drives the reaction in one direction. In another example, hydrolysis of EF1 $\alpha$-GTP to EF1 $\alpha$-GDP during chain elongation occurs only when the A site is occupied by a charged tRNA with an anticodon that base-pairs with the codon in that site. GTP hydrolysis causes a conformational change in EF1 $\alpha$ that results in the release of its bound tRNA, allowing the aminoacylated $3^{\prime}$ end of the charged tRNA to move into the position required for peptide bond formation (see Figure 5-25, step $\mathbf{B}$ ). Hydrolysis of EF2$\cdot$GTP to EF2$\cdot$GDP leads to correct translocation of the ribosome along the mRNA (see Figure 5-25, step $\mathbf{\square}$ ), and hydrolysis of eRF3$\cdot$GTP to eRF3$\cdot$GDP ensures correct termination of translation (see Figure 5-26).

## Nonsense Mutations Cause Premature Termination of Protein Synthesis

One kind of mutation that can inactivate a gene in any organism is a base-pair change that converts a codon normally encoding an amino acid into a stop codon, such as a change from UAC (encoding tyrosine) to UAG (stop). When such a mutation occurs early in the reading frame, the resulting truncated protein is usually nonfunctional. Such mutations are called nonsense mutations because when the genetic code equating each triplet codon sequence with a single amino acid was being deciphered by researchers, the three stop codons were found not to encode any amino acid-they did not "make sense."

In genetic studies with the bacterium E. coli, it was discovered that the effect of a nonsense mutation can be suppressed by a second mutation in a tRNA gene. This occurs when the sequence in a tRNA gene that encodes the anticodon is changed to a triplet that is complementary to the original mutant stop codon. For example, if a mutation in tRNA ${ }^{\text {Tyr }}$ changes its anticodon from GUA to CUA, which can base-pair with the UAG stop codon, then the mutant tRNA ${ }^{\text {Tyr }}$ can still be recognized by the tyrosine aminoacyltRNA synthetase and coupled to tyrosine. Cells that have both the original nonsense mutation and the second mutation in the anticodon of the tRNA ${ }^{\text {Tyr }}$ gene consequently can insert a tyrosine at the position of the mutant stop codon, allowing protein synthesis to continue past the original nonsense mutation. This mechanism is not highly efficient, so translation of normal mRNAs with a UAG stop codon terminates at the normal position in most instances. If enough of the protein encoded by the original gene with the nonsense mutation is produced to provide its essential functions, the effect of the first mutation is said to be suppressed by the second mutation in the anticodon of the tRNA gene.

This mechanism of nonsense suppression is a powerful tool in genetic studies of bacteria. For example, it allows us to isolate mutant bacterial viruses that cannot grow in normal cells but can grow in cells expressing a nonsense-suppressing tRNA because the mutant virus has a nonsense mutation in an essential gene. Such mutant viruses grown on nonsensesuppressing cells can then be used in experiments to analyze

the function of the mutant gene by infecting normal cells that do not suppress the mutation and analyzing what step in the viral life cycle is defective in the absence of the mutant protein.

## KEY CONCEPTS OF SECTION 5.4

## Stepwise Synthesis of Proteins on Ribosomes

- Bacterial, archaeal, and eukaryotic ribosomes-the large ribonucleoprotein complexes on which translation occurs-consist of a small and a large subunit (see Table 5-3). Each subunit contains numerous different proteins and one major rRNA molecule (small or large). The large subunit also contains one accessory 5S rRNA in bacteria and archaea and two accessory rRNAs in eukaryotes ( 5 S and 5.8 S ).
- Analogous rRNAs from many different species fold into quite similar three-dimensional structures containing numerous stem-loops and binding sites for proteins, mRNA, and tRNAs. Much smaller ribosomal proteins are associated with the periphery of the rRNAs.
- Of the two methionine tRNAs found in all cells, only one (tRNA ${ }_{i}^{\text {Met }}$ ) functions in initiation of translation.
- Each stage of translation-initiation, chain elongation, and termination-requires specific protein factors, including GTP-binding proteins that hydrolyze their bound GTP to GDP when a step has been completed successfully.
- During initiation, the ribosomal subunits assemble near the translation start site in an mRNA molecule on which a tRNA carrying the amino-terminal methionine (Met-tRNA ${ }_{i}{ }^{\text {Met }}$ ) has base-paired with the start codon (see Figure 5-24).
- Chain elongation entails a repetitive four-step cycle: loose binding of an incoming aminoacyl-tRNA to the A site on the ribosome; tight binding of the correct aminoacyltRNA to the A site accompanied by release of the previously used tRNA from the E site; transfer of the growing polypeptide chain to the incoming amino acid catalyzed by large rRNA; and translocation of the ribosome to the next codon, thereby moving the peptidyl-tRNA in the A site to the P site and the now unacylated tRNA in the P site to the E site (see Figure 5-25).
- In each cycle of chain elongation, the ribosome undergoes two conformational changes monitored by GTP-binding proteins. The first of these proteins (EF1 $\alpha$ ) permits tight binding of the incoming aminoacyl-tRNA to the A site and ejection of a tRNA from the E site, and the second (EF2) monitors translocation.
- Termination of translation is carried out by two types of termination factors: those that recognize stop codons and those that promote hydrolysis of peptidyl-tRNA (see Figure 5-26). Once again, correct recognition of a stop codon is monitored by a GTPase (eRF3).
- The efficiency of protein synthesis is increased by the simultaneous translation of a single mRNA by multiple ribosomes, forming a polyribosome or polysome. In eukaryotic cells, protein-mediated interactions bring the two ends of a polyribosome close together, thereby promoting the rapid recycling of ribosomal subunits, which further increases the efficiency of protein synthesis (see Figure 5-27b).


### 5.5 DNA Replication

Now that we have seen how the genetic information encoded in the nucleotide sequence of DNA is translated into the proteins and RNAs that perform most cell functions, we can appreciate the necessity for precisely copying DNA sequences during DNA replication, in preparation for cell division (see Figure 5-1, step ■). The regular pairing of bases in the double-helical DNA structure suggested to Watson and Crick that new DNA strands are synthesized by using the existing (parent) strands as templates in the formation of new, daughter strands that are complementary to the parent strands.

This base-pairing template model theoretically could proceed by either a conservative or a semiconservative mechanism. If a conservative mechanism were used, the two daughter strands would form a new double-stranded DNA molecule and the parent duplex would remain intact. If a semiconservative mechanism were used, the parent strands would be permanently separated and each would form a duplex molecule with the newly synthesized daughter strand base-paired to it. Definitive evidence that duplex DNA is replicated by a semiconservative mechanism came from a now classic experiment conducted by M. Meselson and W. F. Stahl, outlined in Figure 5-28.

Copying of a DNA template strand into a complementary strand is thus a common feature of DNA replication, transcription of DNA into RNA, and as we will see later in this chapter, DNA repair and recombination. In all cases, the information in the template, in the form of the specific sequence of nucleotides, is preserved. In some viruses, singlestranded RNA molecules function as templates for the synthesis of complementary RNA or DNA strands. However, the vast preponderance of RNA and DNA in cells is synthesized from preexisting duplex DNA.

## DNA Polymerases Require a Primer to Initiate Replication

DNA is synthesized from deoxyribonucleoside $5^{\prime}$-triphosphate precursors (dNTPs) in a manner analogous to RNA synthesis. Also like RNA synthesis, DNA synthesis always proceeds in the $5^{\prime} \rightarrow 3^{\prime}$ direction because chain growth results from formation of a phosphoester bond between the $3^{\prime}$

![li332239539b](li332239539b.jpg)

EXPERIMENTAL FIGURE 5-28 The Meselson-Stahl experiment. This experiment showed that DNA replicates by a semiconservative mechanism. E. coli cells were initially grown in a medium containing ammonium salts prepared with "heavy" nitrogen $\left({ }^{15} \mathrm{~N}\right)$ until all the cellular DNA was labeled. After the cells were transferred to a medium containing the normal "light" isotope $\left({ }^{14} \mathrm{~N}\right)$, samples were removed periodically from the cultures, and the DNA in each sample was analyzed by equilibrium density-gradient centrifugation, a procedure that separates macromolecules on the basis of their density. This technique can separate heavy-heavy (H-H), light-light (L-L), and heavy-light (H-L) duplexes into distinct bands. (a) Expected composition of daughter duplex molecules synthesized from ${ }^{15} \mathrm{~N}$-labeled DNA after cells are shifted to ${ }^{14} \mathrm{~N}$-containing medium if DNA replication occurs by a conservative or by a semiconservative mechanism. Parent heavy (H) strands are in red; light (L) strands synthesized after shift to ${ }^{14} \mathrm{~N}$-containing medium are in blue. Note that the conservative mechanism would never generate H-L DNA, and that the semiconservative mechanism would never generate H-H DNA but would generate H-L DNA during the first and subsequent doublings. With additional replication cycles, the ${ }^{15} \mathrm{~N}$ labeled (H) strands from the original DNA would be diluted, so that the vast bulk of the DNA would consist of L-L duplexes with either mecha-
nism. (b) Actual banding patterns of DNA subjected to equilibrium density-gradient centrifugation before and after ${ }^{15} \mathrm{~N}$-labeled cells were shifted to ${ }^{14} \mathrm{~N}$-containing medium. DNA bands were visualized under UV light and photographed. The traces on the left are a measure of the density of the photographic signal, and hence the DNA concentration, along the length of the centrifuged cells from left to right. The number of generations (far left) following the shift to ${ }^{14} \mathrm{~N}$-containing medium was determined by counting the concentration of cells in the culture. This value corresponds to the number of DNA replication cycles that had occurred at the time each sample was taken. After one generation of growth, all the extracted DNA had the density of H-L DNA. After 1.9 generations, approximately half the DNA had the density of H-L DNA; the other half had the density of L-L DNA. With additional generations, a larger and larger fraction of the extracted DNA consisted of L-L duplexes; H-H duplexes never appeared. These results match the predicted pattern for the semiconservative replication mechanism depicted in (a). The bottom two centrifuge cells contained mixtures of H-H DNA and DNA isolated at 1.9 and 4.1 generations in order to clearly show the positions of H-H, H-L, and L-L DNA in the density gradient. See M. Meselson and F. W. Stahl, 1958, Proc. Nat'I Acad. Sci. USA 44:671. [Part (b) photo courtesy of M. Meselson and F. W. Stahl, 1958.]
oxygen of a growing strand and the $\alpha$ phosphate of a dNTP (see Figure 5-10a). As discussed earlier, an RNA polymerase can find an appropriate transcription start site on duplex DNA and initiate the synthesis of an RNA complementary to the template DNA strand (see Figure 5-11). In contrast, DNA polymerases cannot initiate chain synthesis de novo; instead, they require a short, preexisting RNA or DNA strand, called a primer, to begin chain growth. With a primer base-paired to the template strand, a DNA polymerase adds
deoxyribonucleotides to the free hydroxyl group at the $3^{\prime}$ end of the primer as directed by the sequence of the template strand:
![li342239539b](li342239539b.jpg)

When RNA is the primer, the daughter strand that is formed is RNA at the $5^{\prime}$ end and DNA at the $3^{\prime}$ end.

## Duplex DNA Is Unwound, and Daughter Strands Are Formed at the DNA Replication Fork

In order for duplex DNA to function as a template during replication, the two intertwined strands must be unwound, or melted, to make their bases available for pairing with the bases of the dNTPs that are polymerized into the newly synthesized daughter strands. This unwinding of the parent DNA strands is performed by enzymes called helicases. Unwinding begins at segments in a DNA molecule called replication origins, or simply origins. The nucleotide sequences of origins from different organisms vary greatly, although they usually contain AT-rich sequences. Once helicases have unwound the parent DNA at an origin, a specialized RNA polymerase called primase forms a short ( $\sim 12$-nucleotide) RNA primer complementary to the unwound template strands. The primer, still base-paired to its complementary DNA strand, is then elongated by DNA polymerase $\alpha$ for another 25 nucleotides or so, forming a primer made of RNA at the $5^{\prime}$ end and DNA at the $3^{\prime}$ end. This primer is further extended by DNA polymerase $\delta$, thereby forming a new daughter strand.

The DNA region at which all these proteins come together to carry out the synthesis of daughter strands is called the replication fork. As replication proceeds, the replication fork and the associated proteins move away from the origin. As noted earlier, local unwinding of duplex DNA produces torsional stress, which is relieved by topoisomerase I. In order for DNA polymerases to move along and copy a duplex DNA, helicase must sequentially unwind the duplex and topoisomerase must remove the supercoils that form.

A major complication in the operation of a DNA replication fork arises from two properties of DNA: the two strands of the parent DNA duplex are antiparallel, and DNA polymerases (like RNA polymerases) can add nucleotides to the growing daughter strands only in the $5^{\prime} \rightarrow 3^{\prime}$ direction. Synthesis of one daughter strand, called the leading strand, can proceed continuously from a single RNA primer in the $5^{\prime} \rightarrow 3^{\prime}$ direction, the same direction as movement of the replication fork (Figure 5-29). The problem comes in synthesis of the other daughter strand, called the lagging strand.

Because growth of the lagging strand must occur in the $5^{\prime} \rightarrow 3^{\prime}$ direction, copying of its template strand must somehow proceed in the opposite direction from the movement of the replication fork. A cell accomplishes this feat by synthesizing a new primer every 100 to 200 nucleotides on that template strand as more of the strand is exposed by unwinding. Each of these primers, base-paired to the template strand, is elongated in the $5^{\prime} \rightarrow 3^{\prime}$ direction, forming discontinuous segments named Okazaki fragments after their discoverer, Reiji Okazaki (see Figure 5-29). The RNA primer of each Okazaki fragment is removed and replaced by DNA chain growth from the neighboring Okazaki fragment; finally, an enzyme called DNA ligase joins the adjacent fragments.
![li352239539b](li352239539b.jpg)

FIGURE 5-29 Leading-strand and lagging-strand DNA synthesis. Nucleotides are added by a DNA polymerase to each growing daughter strand in the $5^{\prime} \rightarrow 3^{\prime}$ direction (indicated by arrowheads). The leading strand is synthesized continuously from a single RNA primer (red) at its $5^{\prime}$ end. The lagging strand is synthesized discontinuously from multiple RNA primers that are formed periodically as each new region of the parent duplex is unwound. Elongation of these primers initially produces Okazaki fragments. As each growing fragment approaches the previous primer, that primer is removed and the fragments are ligated. Repetition of this process eventually results in synthesis of the entire lagging strand.

## Several Proteins Participate in DNA Replication

Detailed understanding of the eukaryotic proteins that participate in DNA replication initially came largely from studies with small viral DNAs, particularly SV40 DNA, the circular genome of a virus that infects monkeys. Virusinfected cells replicate large numbers of the simple viral genome in a short period of time, which makes them an ideal model system for studying basic aspects of DNA replication. Because simple viruses such as SV40 depend largely on the DNA replication machinery of their host cells (in this case monkey cells), they offer a unique opportunity to study the replication of multiple identical small DNA molecules by cellular proteins. Figure 5-30 depicts the multiple proteins that coordinate the copying of SV40 DNA at a replication fork. The assembled proteins at a replication fork further illustrate the concept of molecular machines introduced in Chapter 3. These multicomponent complexes permit the cell to carry out an ordered sequence of events that accomplishes essential cell functions.

The molecular machine that replicates SV40 DNA contains only one viral protein; all other proteins involved in SV40 DNA replication are provided by the host cell. This viral protein, large T-antigen, forms a hexameric replicative helicase, a protein that uses energy from ATP hydrolysis to unwind the parent strands at a replication fork. Primers for the leading and lagging daughter strands are synthesized by a complex of primase, which synthesizes a short RNA primer ( $\sim 12$ nucleotides), and DNA polymerase $\alpha$ (Pol $\alpha$ ),

![li362239539b](li362239539b.jpg)

FIGURE 5-30 Model of an SV40 DNA replication fork. (a) A hexamer of large T-antigen, a viral protein, functions as a helicase to unwind the parent DNA strands. The leading strand is extended by DNA polymerase $\varepsilon$ (Pol $\varepsilon$ ) up to the replication fork. Pol $\varepsilon$ is bound to a ring of PCNA that surrounds the daughter double-stranded DNA so that the Pol $\varepsilon$-PCNA complex remains stably associated with the replication fork. The single-stranded region of the lagging strand template generated by T-antigen helicase is bound by multiple copies of the heterotrimeric protein RPA. Primers for lagging-strand synthesis (red, RNA; light green, DNA) are synthesized by a complex of primase and DNA polymerase $\alpha$ (Pol $\alpha$ ). The $3^{\prime}$ end of each primer synthesized by Pol $\alpha$-primase is then bound by a PCNA-Pol $\delta$ complex, which extends the primer and synthesizes most of each Okazaki fragment. (b) The helicase domain of SV40 T-antigen forms a hexameric replicative helicase. Subunits are shown in alternating light and dark orange. (c) Model of DNA polymerase $\varepsilon$ extending the $3^{\prime}$-end of the leading strand. (d) The three subunits of PCNA, shown in different shades of yellow, form a circular structure with a central hole through which daughter double-stranded DNA passes. (e) RFC, the pentameric "clamp-loader" (monomers shown in different shades of green) is shown bound to a circular trimer PCNA before the PCNA "clamp" is opened. (f) The large subunit of RPA contains two domains that bind single-stranded DNA. Note that the single DNA strand is extended, with the bases oriented in an optimal conformation for replication by Pol $\delta$. See M. O'Donnell, L. Langston, and B. Stillman, 2013, Cold Spring Harbor Perspect. Biol. 5:a010108. [Part (b) data from D. Li et al., 2003, Nature 423:512, PDB ID 1n25. Part (c) data from M. Hogg et al., 2014, Nat. Struct. Mol. Biol., 21:49, PDB ID 4m8o. Part (d) data from J. M. Gulbis et al., 1996, Cell 87:297, PDB ID 1axc. Part (e) data from G. D. Bowman, M. O'Donnell, and J. Kuriyan, 2004, Nature 429:724, PDB ID 1sxj. Part (f) data from A. Bochkarev et al., 1997, Nature 385:176, PDB ID 1jmc.]
which extends the RNA primer with deoxyribonucleotides for another 25 nucleotides or so, forming a mixed RNADNA primer.

The primer is extended into daughter-strand DNA by DNA polymerase $\delta$ (Pol $\delta$ ), which is less likely to make errors during copying of the template strand than is Pol $\alpha$ because of its proofreading mechanism (see Section 5.6). During the replication of cellular DNA, Pol $\delta$ synthesizes laggingstrand DNA, while DNA polymerase $\varepsilon$ (Pol $\varepsilon$ ) synthesizes most of the length of the leading strand. Pol $\delta$ and Pol $\varepsilon$ each form a complex with PCNA (proliferating cell nuclear antigen), which displaces the primase-Pol $\alpha$ complex following primer synthesis. As illustrated in Figure 5-30d, PCNA is a homotrimeric protein that has a central hole through which the daughter duplex DNA passes, thereby preventing the PCNA-Pol $\delta$ and PCNA-Pol $\varepsilon$ complexes from dissociating from the template. As such, PCNA is known as a sliding clamp that enables Pol $\delta$ and Pol $\varepsilon$ to remain stably associated with a single template strand for thousands of nucleotides. A pentameric protein called RFC (replication factor C)

functions to open the PCNA ring so that it can encircle the short region of double-stranded DNA synthesized by Pol $\alpha$. Consequently, RFC is called a clamp loader.

After parent DNA is separated into single-stranded templates at the replication fork, the leading strand is extended by Pol $\varepsilon$, which can extend the growing strand up to the replication fork. The single-stranded template for laggingstrand synthesis is bound by multiple copies of RPA (replication protein $A$ ), a heterotrimeric protein (Figure 5-30c). Binding of RPA maintains the template in a uniform conformation that is optimal for copying by Pol $\delta$. Bound RPA proteins are dislodged from the parent strand by Pol $\delta$ as it synthesizes the complementary strand base-paired with the parent strand.

Several other eukaryotic proteins that function in DNA replication are not depicted in Figure 5-30. For example, topoisomerase I associates with the parent DNA ahead of the replicative helicase (i.e., to the left of large T-antigen in Figure 5-30) to remove torsional stress introduced by the unwinding of the parent strands (see Figure 5-8a). Ribonuclease H and FEN I remove the ribonucleotides at the $5^{\prime}$ ends of Okazaki fragments; these ribonucleotides are replaced by deoxyribonucleotides added by Pol $\delta$ as it extends the upstream Okazaki fragment. Successive Okazaki fragments are coupled by DNA ligase through standard $5^{\prime} \rightarrow 3^{\prime}$ phosphoester bonds. Other specialized DNA polymerases are involved in the repair of mismatches and lesions in DNA (see Section 5.6).

## DNA Replication Occurs Bidirectionally from Each Origin

As indicated in Figures 5-29 and 5-30, both parent DNA strands that are exposed by local unwinding at a replication fork are copied into daughter strands. In theory, DNA replication from a single origin could involve one replication fork that moves in one direction. Alternatively, two replication forks might assemble at a single origin and then move in opposite directions, leading to bidirectional growth of both daughter strands. Several types of experiments, including the one shown in Figure 5-31, provided early evidence in support of bidirectional strand growth.

The general consensus is that all bacterial, archaeal, and eukaryotic cells employ a bidirectional mechanism of DNA replication. In the case of SV40 DNA, replication is initiated by the binding of two large T-antigen hexameric helicases to the single SV40 origin and the assembly of other proteins to form two replication forks. These forks then move away from the SV40 origin in opposite directions, and leadingand lagging-strand synthesis occurs at both forks. As shown in Figure 5-32, the left replication fork extends DNA synthesis in the leftward direction; similarly, the right replication fork extends DNA synthesis in the rightward direction.

Unlike SV40 DNA, eukaryotic chromosomal DNA molecules contain multiple replication origins separated by tens to hundreds of kilobases. A six-subunit protein called $O R C$, for origin recognition complex, binds to each origin and associates with other proteins required to load cellular
![li372239539b](li372239539b.jpg)

EXPERIMENTAL FIGURE 5-31 Bidirectional replication in SV40 DNA. Electron microscopy of replicating SV40 DNA indicates bidirectional growth of DNA strands from an origin. Replicating viral DNA from SV40infected cells was cut by the restriction enzyme EcoRI, which recognizes one site in the circular viral DNA. This was done to provide a landmark in the SV40 genome: the EcoRI recognition sequence could now be easily recognized as the ends of the linear DNA molecules visualized by electron microscopy. Electron micrographs of EcoRI-cut, replicating SV40 DNA molecules showed a collection of cut molecules with increasingly longer replication "bubbles," whose centers were a constant distance from each end of the cut molecules. This finding is consistent with chain growth in two directions from a common origin located at the center of a bubble, as illustrated in the corresponding diagrams. See G. C. Fareed et al., 1972, J. Virol. 10:484. (Micrographs republished with permission of American Society for Microbiology-Journals, from Journal of Virology, Fareed et al., 10, 3, 1972; permission conveyed through Copyright Clearance Center, Inc.)
hexameric helicases composed of six homologous MCM proteins. Two MCM helicases, oriented in opposite directions, separate the parent strands at an origin, and RPA proteins bind to the resulting single-stranded DNA. Synthesis of primers and subsequent steps in the replication of cellular DNA are thought to be analogous to those in SV40 DNA replication (see Figures 5-31 and 5-32).

Replication of cellular DNA and other events leading to the proliferation of cells must be tightly regulated so that

![li382239539b](li382239539b.jpg)
the appropriate numbers of cells constituting each tissue will be produced during embryonic development and throughout the life of an organism. Control of the initiation step is the primary mechanism for regulating cellular DNA replication. Activation of MCM helicase, which is required to initiate cellular DNA replication, is regulated by a specific protein kinase ( $D D K$ ), which in turn is regulated by S-phase cyclindependent kinases. Other cyclin-dependent kinases regulate additional aspects of cell proliferation, including the complex process of mitosis by which a eukaryotic cell divides

FIGURE 5-32 Bidirectional mechanism of DNA replication.
The left replication fork here is comparable to the replication fork diagrammed in Figure 5-30 (although that figure also shows proteins other than large T-antigen, which are not shown here). Top: Two large T-antigen hexameric helicases first bind at the replication origin in opposite orientations. Step $\mathbf{\square}$ Using energy provided by ATP hydrolysis, the helicases move in opposite directions, unwinding the parent DNA and generating single-stranded templates, which are bound by RPA proteins. Step $\mathbf{\square}$ Primase-Pol $\alpha$ complexes synthesize short primers (red) base-paired to each of the separated parent strands. Step $\mathbf{\square}$ PCNA-Rfc-Pol $\varepsilon$ complexes replace the primase-Pol $\alpha$ complexes and extend the short primers, generating the leading strands (dark green) at each replication fork. Step $\mathbf{\square}$ The helicases further unwind the parent strands, and RPA proteins bind to the newly exposed single-stranded regions. Step PCNA-Rfc-Pol $\varepsilon$ complexes extend the leading strands farther. Step $\mathbf{\square}$ Primase-Pol $\alpha$ complexes synthesize primers for lagging-strand synthesis at each replication fork. Step PCNA-RfcPol $\delta$ complexes displace the primase-Pol $\alpha$ complexes and extend the lagging-strand Okazaki fragments (light green), which are eventually ligated to the $5^{\prime}$ ends of the leading strands. The position where ligation occurs is represented by a circle. Replication continues by further unwinding of the parent strands and synthesis of leading and lagging strands as in Steps $\boldsymbol{\square}$ Although depicted as individual steps for clarity, unwinding and synthesis of leading and lagging strands occur concurrently.
into two daughter cells. Mitosis and another specialized type of cell division called meiosis, which generates haploid sperm and egg cells, are discussed in Chapter 6. We discuss the various regulatory mechanisms that determine the rate of cell division in Chapter 19.

## KEY CONCEPTS OF SECTION 5.5

## DNA Replication

- Each strand in a parent duplex DNA acts as a template for synthesis of a daughter strand and remains base-paired to the new strand, forming a daughter duplex (semiconservative replication). New strands are formed in the $5^{\prime} \rightarrow 3^{\prime}$ direction.
- Replication begins at a sequence called an origin. Each eukaryotic chromosomal DNA molecule contains multiple replication origins.
- DNA polymerases, unlike RNA polymerases, cannot unwind the strands of duplex DNA and cannot initiate synthesis of new strands complementary to the template strands.
- At a replication fork, one daughter strand (the leading strand) is elongated continuously. The other daughter strand (the lagging strand) is formed as a series of discontinuous Okazaki fragments from primers synthesized every 100 to 200 nucleotides (Figure 5-29).
- The ribonucleotides at the $5^{\prime}$ end of each Okazaki fragment are removed and replaced by elongation of the $3^{\prime}$ end

of the next Okazaki fragment. Finally, adjacent Okazaki fragments are joined by DNA ligase.

- Helicases use energy from ATP hydrolysis to separate the parent (template) DNA strands, which are initially bound by multiple copies of a single-stranded DNA-binding protein, RPA. Primase synthesizes a short RNA primer, which remains base-paired to the template DNA. This primer is initially extended at the $3^{\prime}$ end by DNA polymerase $\alpha$ (Pol $\alpha$ ), resulting in a short ( $5^{\prime}$ )-RNA-(3')DNA daughter strand.
- Most of the DNA in eukaryotic cells is synthesized by Pol $\delta$ and $\operatorname{Pol} \varepsilon$, which take over from $\operatorname{Pol} \alpha$ and continue elongation of the daughter strands in the $5^{\prime} \rightarrow 3^{\prime}$ direction. Pol $\delta$ synthesizes most of the length of the lagging strand, while Pol $\varepsilon$ synthesizes the leading strand. Pol $\delta$ and Pol $\varepsilon$ remain stably associated with the template by binding to PCNA, a trimeric protein that encircles the daughter duplex DNA, functioning as a sliding clamp (see Figure 5-30).
- DNA replication generally occurs by a bidirectional mechanism in which two replication forks form at an origin and move in opposite directions, and both template strands are copied at each fork (see Figure 5-32).
- MCM helicases initiate eukaryotic DNA replication in vivo at multiple origins spaced along chromosomal DNA. Synthesis of eukaryotic DNA is regulated by controls on the binding and activity of these helicases.


### 5.6 DNA Repair and Recombination

Damage to DNA is unavoidable and arises in many ways. DNA damage can be caused by spontaneous cleavage of chemical bonds in DNA, by environmental agents such as ultraviolet and ionizing radiation, and by reaction with genotoxic chemicals that are by-products of normal cellular metabolism or occur in the environment. A change in the normal DNA sequence, called a mutation, can occur during replication when a DNA polymerase inserts the wrong nucleotide as it reads a damaged template. Mutations also occur at a low frequency as the result of copying errors introduced by DNA polymerases when they replicate an undamaged template. If such mutations were left uncorrected, cells might accumulate so many mutations that they could no longer function properly. In addition, the DNA in germ cells might incur too many mutations for viable offspring to be formed. Thus the prevention of DNA sequence errors in all types of cells is important for their survival, and several cellular mechanisms for repairing damaged DNA and correcting sequence errors have evolved. One of these mechanisms for repairing breaks in double-stranded DNA is also used by eukaryotic cells to generate new combinations of maternal and paternal genes on each chromosome through the exchange of segments of the chromosomes during the production of germ cells (e.g., sperm and eggs), a process known as genetic recombination.

Significantly, defects in DNA repair mechanisms and cancer are closely related. When repair mechanisms are compromised, mutations accumulate in the cell's DNA. If these mutations affect genes that are normally involved in the careful regulation of cell division, cells can begin to divide uncontrollably, leading to tumor formation and cancer. Chapter 24 outlines in detail how cancer arises from defects in DNA repair mechanisms. We will encounter a few examples in this section as well as we first consider the ways in which DNA integrity can be compromised, then discuss the repair mechanisms that cells have evolved to ensure the fidelity of this very important molecule.

## DNA Polymerases Introduce Copying Errors and Also Correct Them

The first line of defense in preventing mutations is DNA polymerase itself. Occasionally, when replicative DNA polymerases (Pol $\delta$ and Pol $\varepsilon$ ) progress along the template DNA, an incorrect nucleotide is added to the growing $3^{\prime}$ end of the daughter strand. E. coli DNA polymerases, for instance, introduce about 1 incorrect nucleotide per $10^{4}$ (ten thousand) polymerized nucleotides. Yet the measured mutation rate in bacterial cells is much lower: about 1 mistake in $10^{9}$ (one billion) nucleotides incorporated into a growing strand. This remarkable accuracy is largely due to proofreading by $E$. coli DNA polymerases. Eukaryotic Pol $\delta$ and Pol $\varepsilon$ employ a similar mechanism.

Proofreading depends on the $3^{\prime} \rightarrow 5^{\prime}$ exonuclease activity of some DNA polymerases. When an incorrect base is incorporated during DNA synthesis, base pairing between the $3^{\prime}$ nucleotide of the nascent strand and the template strand does not occur. As a result, the polymerase pauses, then transfers the $3^{\prime}$ end of the growing chain to its exonuclease site, where the incorrect mispaired base is removed (Figure 5-33). Then the $3^{\prime}$ end is transferred back to the polymerase site, where this region is copied correctly. All three E. coli DNA polymerases have proofreading activity, as do the two eukaryotic DNA polymerases, $\delta$ and $\varepsilon$, used for replication of most chromosomal DNA in animal cells. It seems likely that proofreading is indispensable for all cells to avoid excessive mutations.

## Chemical and Radiation Damage to DNA Can Lead to Mutations

DNA is continually subjected to a barrage of damaging chemical reactions; estimates of the number of DNA damage events in a single human cell range from $10^{4}$ to $10^{6}$ per day! Even if DNA were not exposed to damaging chemicals, certain aspects of DNA structure are inherently unstable. For example, the bond connecting a purine base to deoxyribose is prone to hydrolysis at a low rate under physiological conditions, leaving a sugar without an attached base. Thus coding information is lost, and this loss can lead to a mutation during DNA replication. Normal cellular reactions, including the movement of electrons along the electron-transport

![li392239539b](li392239539b.jpg)

FIGURE 5-33 Proofreading by DNA polymerase. All DNA polymerases have a similar three-dimensional structure, which resembles a half-opened right hand. The "fingers" bind the single-stranded segment of the template strand, and the polymerase catalytic activity (Pol) lies in the junction between the fingers and palm. As long as the correct nucleotides are added to the $3^{\prime}$ end of the growing strand, it remains in the polymerase site. Incorporation of an incorrect base at
chain in mitochondria and lipid oxidation in peroxisomes (see Chapter 12), produce several chemicals that react with and damage DNA, including hydroxyl radicals and superoxide $\left(\mathrm{O}_{2}^{-}\right)$. These chemicals can also cause mutations, including mutations that lead to cancers.

Many spontaneous mutations are point mutations, which involve a change in a single base pair in the DNA sequence. A point mutation can introduce a stop codon, causing a nonsense mutation as discussed earlier, or a change in the amino acid sequence of an encoded protein, called a missense mutation. Silent mutations do not change the amino acid sequence (e.g., GAG to GAA; both encode glutamine). Point mutations can also occur in non-protein-coding DNA sequences that function in the regulation of a gene's transcription, as discussed in Chapter 9. One of the most frequent causes of
the $3^{\prime}$ end causes melting of the newly formed end of the duplex. As a result, the polymerase pauses, and the $3^{\prime}$ end of the growing strand is transferred to the $3^{\prime} \rightarrow 5^{\prime}$ exonuclease site (Exo) about 3 nm away, where the mispaired base, and probably other bases, are removed. Subsequently, the $3^{\prime}$ end flips back into the polymerase site and elongation resumes. See C. M. Joyce and T. T. Steitz, 1995, J. Bacteriol. 177:6321, and S. Bell and T. Baker, 1998, Cell 92:295.
point mutations is deamination of a cytosine (C) base, which converts it into a uracil (U) base. Another is deamination of the common modified base 5-methyl cytosine, which forms thymine when it is deaminated. If these alterations are not corrected before the DNA is replicated, the cell will use the strand containing $U$ or $T$ as a template to form a $U-A$ or $T-A$ base pair, thus creating a permanent change in the DNA sequence (Figure 5-34).

## High-Fidelity DNA Excision-Repair Systems Recognize and Repair Damage

In addition to proofreading, cells have other systems for preventing mutations due to copying errors and exposure to chemicals and radiation. Several DNA excision-repair
![li402239539b](li402239539b.jpg)

FIGURE 5-34 Deamination leads to point mutations. A spontaneous point mutation can be caused by deamination of 5-methylcytosine (C) to form thymine (T). If the resulting T-G base pair is not restored to the normal C-G base pair by base excision-repair mechanisms It will lead to a permanent change in sequence (i.e., a mutation) following DNA replication After one round of replication, one daughter DNA molecule will have the mutant T-A base pair and the other will have the wild-type C-G base pair.

systems that normally operate with a high degree of accuracy have been well studied. These systems were first elucidated through a combination of genetic and biochemical studies in E. coli. Homologs of the key bacterial proteins exist in eukaryotes from yeasts to humans, indicating that these error-free mechanisms arose early in evolution to protect DNA integrity. Each of these systems functions in a similar manner: a segment of the damaged DNA strand is excised, and the gap is filled by DNA polymerase and ligase using the complementary DNA strand as a template.

We will now take a closer look at some of the mechanisms of DNA repair, ranging from repair of single-base mutations to repair of DNA broken across both strands. Some of these mechanisms accomplish their repairs with great accuracy; others are less precise.

## Base Excision Repairs T-G Mismatches and Damaged Bases

In humans, the most common type of point mutation is a change from a C to a T, which is caused by deamination of 5-methyl C (see Figure 5-34). The conceptual problem with base excision repair in this case is determining which is the normal and which is the mutant DNA strand. But since a G-T mismatch is almost invariably caused by chemical conversion of C to U or 5-methyl C to T , the repair system evolved to remove the T and replace it with a C .

The G-T mismatch is recognized by a DNA glycosylase that flips the thymine base out of the helix and then hydrolyzes the bond that connects it to the sugar-phosphate DNA backbone. Following this initial incision, an endonuclease, APE1, cuts the DNA strand near the now abasic site. The deoxyribose phosphate lacking the base is then removed and replaced with a C by a specialized DNA polymerase that reads the G in the template strand (Figure 5-35).

As mentioned earlier, this repair must take place prior to DNA replication because the incorrect base in this pair, T, occurs naturally in normal DNA. Consequently, it would be able to engage in normal Watson-Crick base pairing during replication, generating a stable point mutation that would no longer be recognized by repair mechanisms (see Figure 5-34, step ■).

Human cells contain a battery of glycosylases, each of which is specific for a different set of chemically modified DNA bases. For example, one glycosylase removes 8 -oxyguanine, an oxidized form of guanine, allowing its replacement by an undamaged G, and others remove bases modified by alkylating agents. The resulting abasic nucleotide is then replaced by the repair mechanism described above. A similar mechanism functions in the repair of lesions resulting from depurination, the loss of a guanine or adenine base from DNA caused by hydrolysis of the glycosylic bond between deoxyribose and the base. Depurination occurs spontaneously and is fairly common in mammals and birds because of their warm body temperatures. The resulting abasic sites, if left unrepaired, generate mutations during DNA replication because they cannot specify the appropriate paired base.
![li412239539b](li412239539b.jpg)

FIGURE 5-35 Base excision repair of a T-G mismatch. A DNA glycosylase specific for G-T mismatches, which are usually formed by deamination of 5-methyl C (see Figure 5-34), flips the thymine base out of the helix and then cuts it away from the sugar-phosphate DNA backbone (step $\mathbf{\square}$ ), leaving just the deoxyribose phosphate (black dot). An endonuclease specific for the resultant abasic site (apurinic endonuclease I, APE1) then cuts the DNA backbone (step $\mathbf{\square}$, and the deoxyribose phosphate is removed by an endonuclease, apurinic lyase (AP lyase) associated with DNA polymerase $\beta$, a specialized DNA polymerase used in repair (step $\mathbf{\square}$ ). The gap is then filled in by DNA Pol $\beta$ and sealed by DNA ligase (step $\mathbf{\square}$ ), restoring the original G-C base pair. See O. Schärer, 2003, Angewandte Chemie 42:2946.

## Mismatch Excision Repairs Other Mismatches and Small Insertions and Deletions

Another process, also conserved in organisms from bacteria to humans, principally eliminates base-pair mismatches and insertions or deletions of one or a few nucleotides that are accidentally introduced by DNA polymerases during replication. As with base excision repair of a T in a T-G mismatch, the conceptual problem with mismatch excision repair is determining which is the normal and which is the mutant DNA strand. How this happens in human cells is not known with certainty. It is thought that the proteins that bind to the mismatched segment of DNA distinguish the template and daughter strands; then the mispaired segment of the daughter strand-the one with the replication error-is excised and repaired to produce an exact complement of the template strand (Figure 5-36). In contrast to base excision repair, mismatch excision repair occurs after DNA replication.

Predisposition to a colon cancer known as hereditary nonpolyposis colorectal cancer results from an inherited loss-of-function mutation in one copy of either the MLH1 or the MSH2 gene. The MSH2 and MLH1 proteins

![li422239539b](li422239539b.jpg)
are essential for DNA mismatch repair (see Figure 5-36). Cells with at least one functional copy of each of these genes exhibit normal mismatch repair. However, tumor cells frequently arise from individual cells that have experienced a random mutation in the second copy; when both copies of
![li432239539b](li432239539b.jpg)
(b)
![li442239539b](li442239539b.jpg)

Thymine-thymine dimer residue

FIGURE 5-36 Mismatch excision repair in human cells. The mismatch excision-repair pathway corrects errors introduced during replication. A complex of the MSH2 and MSH6 proteins (bacterial MutS homologs 1 and 6) binds to a mispaired segment of DNA in such a way as to distinguish between the template and the newly synthesized daughter strand (step $\mathbf{D}$ ). This binding triggers binding of MLH1 and PMS2 (both homologs of bacterial MutL). The resulting DNA-protein complex then binds an endonuclease that cuts the newly synthesized daughter strand. Next a DNA helicase unwinds the helix, and an exonuclease removes several nucleotides from the cut end of the daughter strand, including the mismatched base (step $\mathbf{D}$ ). Finally, as with base excision repair, the gap is filled in by a DNA polymerase (Pol $\delta$, in this case) and sealed by DNA ligase (step $\mathbf{D}$ ).
one gene are nonfunctional, the mismatch repair system is lost. Inactivating mutations in these genes are also common in noninherited forms of colon cancer.

## Nucleotide Excision Repairs Chemical Adducts that Distort Normal DNA Shape

Cells use nucleotide excision repair to fix DNA regions containing chemically modified bases, often called chemical adducts, that distort the normal shape of DNA locally. A key to this type of repair is the ability of certain proteins to slide along the surface of a double-stranded DNA molecule looking for bulges or other irregularities in the shape of the double helix. For example, this mechanism repairs thymine-thymine dimers, a common type of chemical adduct caused by UV light (Figure 5-37); these dimers interfere with both replication and transcription of DNA. Figure 5-38
![li452239539b](li452239539b.jpg)

FIGURE 5-37 Formation of thymine-thymine dimers. (a) The most common type of DNA damage caused by UV irradiation is the formation of thymine-thymine dimers. (b) These lesions can be repaired by an excision-repair mechanism that recognizes the distortion they create in the shape of the DNA double helix. The red lines in (b) represent the UV-induced C-C bonds shown in (a). [Part (b) data from K. McAteer et al., 1998, J Mol Biol. 282:1013, PDB ID 1ttd.]

![li462239539b](li462239539b.jpg)

FIGURE 5-38 Nucleotide excision repair in human cells. A DNA lesion that causes distortion of the double helix, such as a thymine thymine dimer, is initially recognized by a complex of the XP-C (xeroderma pigmentosum $C$ protein) and 23B proteins (step $\mathbf{\square}$ ). This complex then recruits transcription factor TFIIH, whose helicase subunits, powered by ATP hydrolysis, partially unwind the double helix. XP-G and RPA proteins then bind to the complex and further unwind and stabilize the helix until a bubble of about 25 bases is formed (step $\mathbf{\square}$ ). Then XP-G (now acting as an endonuclease) and XP-F, a second endonuclease, cut the damaged strand at points 24-32 bases apart on each side of the lesion (step $\mathbf{\square}$ ). This releases the DNA fragment with the damaged bases, which is degraded to mononucleotides. Finally the gap is filled by DNA polymerase exactly as in DNA replication, and the remaining nick is sealed by DNA ligase (step $\mathbf{\square}$ ). See J. Hoeijmakers, 2001, Nature 411:366, and O. Schärer, 2003, Angewandte Chemie 42:2946.
illustrates how the nucleotide excision-repair system repairs damaged DNA.

$\square$Some 30 proteins are involved in the nucleotide exci-sion-repair process, the first of which were identified through a study of the defects in DNA repair in cultured cells from individuals with xeroderma pigmentosum, a hereditary disease associated with a predisposition to cancer. Individuals with this disease frequently develop skin cancers called melanomas and squamous cell carcinomas if their skin is exposed to the UV rays in sunlight. The cells of affected patients lack a functional nucleotide excision-repair system. Mutations in any of at least seven different genes, called $X P-A$ through $X P-G$, lead to inactivation of this repair system and cause xeroderma pigmentosum; all produce the same phenotype and have the same consequences. The functions of most of these XP proteins in nucleotide excision repair are now well understood (see Figure 5-38).

Remarkably, five polypeptide subunits of TFIIH, a general transcription factor required for transcription of all genes (see Figure 9-19), are also required for nucleotide excision repair in eukaryotic cells. Two of these subunits have homology to helicases, as shown in Figure 5-38. In transcription, the helicase activity of TFIIH helps to unwind the DNA helix at the start site, allowing RNA polymerase to initiate the process (see Figure 9-19). It appears that nature has used a similar protein assembly in two different cellular processes that require helicase activity.

The use of shared subunits in transcription and in DNA repair may help explain the observation that DNA damage in higher eukaryotes is repaired at a much faster rate in regions of the genome being actively transcribed than in nontranscribed regions-a phenomenon called transcriptioncoupled repair. Since only a small fraction of the genome is transcribed in any one cell, transcription-coupled repair efficiently directs repair efforts to the most critical regions. In this system, if an RNA polymerase becomes stalled at a lesion on DNA (e.g., a thymine-thymine dimer), a complex of proteins CSA and CSB is recruited to the RNA polymerase, where they trigger opening of the DNA helix, recruitment of TFIIH, and the reactions of steps $\mathbf{\square}$ through $\mathbf{\square}$ depicted in Figure 5-38. CSA and CSB are named after the rare inherited developmental disorder called Cockayne syndrome that results from mutations in these proteins.

## Two Systems Use Recombination to Repair Double-Strand Breaks in DNA

Ionizing radiation (e.g., x - and $\gamma$-radiation) and some anticancer drugs cause double-strand breaks in DNA. These lesions are particularly severe because incorrect rejoining of double strands of DNA can lead to gross chromosomal rearrangements that can affect the functioning of genes. For example, incorrect joining could create a "hybrid" gene that codes for the N-terminal portion of one amino acid sequence fused to the C-terminal portion of a completely different

amino acid sequence; or a chromosomal rearrangement could bring the promoter of one gene close to the coding region of another gene, changing the level or cell type in which that gene is expressed.

Two systems have evolved to repair double-strand breaks: homologous recombination, discussed below, and nonhomologous end joining ( $N H E J$ ), which is error-prone, since several nucleotides are invariably lost at the point of repair.

Error-Prone Repair by Nonhomologous End Joining The predominant mechanism for repairing double-strand breaks
in multicellular organisms involves rejoining of the nonhomologous ends of two DNA molecules. Even if the joined DNA fragments come from the same chromosome, the repair process results in the loss of several base pairs at the joining point (Figure 5-39). The formation of such a possibly mutagenic deletion is one example of how repair of DNA damage can introduce mutations.

Since diffusion of DNA within the viscous nucleoplasm is fairly slow, the correct ends of a broken chromosome are generally rejoined together, albeit with a loss of base pairs, before they diffuse too far apart for nonhomologous
![li472239539b](li472239539b.jpg)

FIGURE 5-39 Nonhomologous end joining. (a) When sister chromatids are not available to help repair double-strand breaks, nucleotide sequences are butted together that were not apposed in the unbroken DNA. These DNA ends are usually from the same chromosome locus, and when linked together, several base pairs are lost. Occasionally, ends from different chromosomes are accidentally joined together. A complex of two proteins, Ku and DNA-dependent protein kinase (DNAPK), binds to the ends of a double-strand break (step $\mathbf{\square}$ ). After formation of a synapse, the ends are further processed by nucleases, resulting in removal of a few bases (step $\mathbf{\square}$ ), and the two double-stranded molecules are ligated together (step $\mathbf{\square}$. As a result, the double-strand break is repaired, but several base pairs at the site of the break are removed. See G. Chu, 1997, J. Biol. Chem. 272:24097; M. Lieber et al., 1997, Curr. Opin. Genet. Devel. 7:99; and D. van Gant et al., 2001, Nature Rev. Genet. 2:196. (b) Structure of KU70/KU80 bound to a duplex DNA end. The complex is shown in a view down the DNA axis (left) and from the side (right). KU80 is light green, KU70 dark green, DNA blue. [Part (b) data from J. R. Walker, R. A. Corpina, and J. Goldberg, 2001, Nature 412:607, PDB ID 1jey.]

end joining to be efficient. Since most of the length of most human genes consists of introns that are spliced out of the processed mRNA, in most cases the small deletion generated is not harmful. Occasionally, however, broken ends from different chromosomes are joined together, leading to translocation of pieces of DNA from one chromosome to another. Such translocations may generate chimeric genes that can have drastic effects on normal cell function, such as uncontrollable cell growth, which is the hallmark of cancer (see Figure 8-38). The devastating effects of double-strand breaks make these the "most unkindest cuts of all," to borrow a phrase from Shakespeare's Julius Caesar.

## Homologous Recombination Can Repair DNA Damage and Generate Genetic Diversity

At one time homologous recombination was thought to be a minor repair process in human cells. This view changed when it was realized that several human cancers are potentiated by inherited mutations in genes that are essential for homologous recombination repair (see Table 24-1). For example, some women with an inherited susceptibility to breast cancer have a mutation in one allele of either the BRCA-1 or the BRCA-2 gene, both of which encode proteins participating in this repair process. Loss or inactivation of the second allele inhibits the homologous recombination repair pathway and thus tends to induce cancer in mammary or ovarian epithelial cells. Yeasts can use homologous recombination to repair double-strand breaks induced by $\gamma$-irradiation. Isolation and analysis of radiation-sensitive ( $R A D$ ) mutants that are deficient in this repair system facilitated study of the process. Virtually all the yeast Rad proteins have homologs in the human genome, and the human and yeast proteins function in an essentially identical fashion.

A variety of DNA lesions that are not repaired by the mechanisms discussed earlier can be repaired by mechanisms in which the damaged DNA sequence is copied from an undamaged copy of the same or a highly homologous sequence on the homologous chromosome in diploid organisms or the sister chromosome following DNA replication in haploid and diploid organisms. These mechanisms involve an exchange of strands between separate DNA molecules and hence are referred to as recombination.

In addition to providing a mechanism for DNA repair, similar recombination mechanisms generate genetic diversity among the individuals of a species by causing the exchange of large regions of chromosomes between the maternal and paternal pair of homologous chromosomes during meiosis, the special type of cell division that generates germ cells (sperm and eggs) (see Figure 6-3). In fact, the exchange of regions of homologous chromosomes, called crossing over, is required for proper segregation of chromosomes during the first meiotic cell division. Meiosis and the consequences of generating new combinations of maternal and paternal genes by recombination are discussed further in Chapter 6, and the mechanisms leading to proper segregation of chromosomes
during meiosis are discussed in Chapter 19. Here we focus on the molecular mechanisms of DNA recombination, highlighting the exchange of DNA strands between two recombining DNA molecules.

Repair of a Collapsed Replication Fork An example of recombinational DNA repair is the repair of a "collapsed" replication fork. If a break in the phosphodiester backbone of one DNA strand (called a "nick") is not repaired before a replication fork passes, the replicated portions of the daughter chromosomes become separated when the replication helicase reaches the nick in the parent strand because there are no covalent bonds between the two fragments of the parent strand on either side of the nick. This process is called replication fork collapse (Figure 5-40, step $\mathbf{\square}$ ). If the break in the double-stranded daughter DNA molecule is not repaired, it is generally lethal to at least one daughter cell following cell division because of the loss of genetic information between the break and the end of the chromosome. The recombination process that repairs the resulting double-strand break and regenerates a replication fork involves multiple enzymes and other proteins, only some of which are mentioned here.

The first step in the repair of the double-strand break is exonucleolytic digestion of the strand with its $5^{\prime}$ end at the broken end of DNA, leaving the strand with a $3^{\prime}$ end at the break single-stranded (Figure 5-40, step $\mathbf{\square}$ ). The lagging nascent strand (pink) base-paired to the unbroken parent strand (dark blue) is ligated to the unreplicated portion of the parent chromosome (light blue), as shown in Figure 5-40, step $\mathbf{\square}$. A protein required for the next step is RecA in bacteria, or the homologous Rad51 in S. cerevisiae and other eukaryotes. Multiple RecA/Rad51 molecules bind to the single-stranded DNA and catalyze its hybridization to a perfectly or nearly perfectly complementary sequence in another, homologous, double-stranded DNA molecule. The complementary strand of this target double-stranded DNA (dark blue) is displaced as a single-stranded loop of DNA over the region of hybridization to the invading strand (Figure 5-40, step $\mathbf{\square}$ ). This RecA/Rad51-catalyzed invasion of a duplex DNA by a single-stranded complement of one of the strands is key to the recombination process. This process is called strand invasion, and because there is no change in the number of base pairs, it does not require an input of energy.

Next the hybrid region between target DNA and the invading strand is extended in the direction away from the break by proteins that use energy from ATP hydrolysis. This process is called branch migration (Figure 5-40, step $\mathbf{\square}$ ) because the position where the target DNA strand crosses from one complementary strand (dark blue) to its complement in the broken DNA molecule (dark red)-that is, the pink diagonal line after step $\mathbf{\square}$-is called a branch in the DNA structure. In this diagram, the diagonal lines represent only one phosphodiester bond. Molecular modeling and other studies show that the first base on either side of the branch is base-paired to a complementary nucleotide. As this branch migrates to the left, the number of base pairs remains

![li482239539b](li482239539b.jpg)

FIGURE 5-40 Recombinational repair of a collapsed replication fork. Parent strands are light and dark blue. The leading daughter strand is dark red, and the lagging daughter strand pink. Diagonal lines in step $\mathbf{B}$ and beyond represent a single phosphodiester bond from the DNA strand of the corresponding color. Small black arrows following step represent cleavage of the phosphodiester bonds at the crossover of DNA strands in the Holliday structure. See http://www.sheffield. ac.uk/mbb/ruva for an animation of branch migration catalyzed by E. coli proteins RuvA and RuvB. See the text for a discussion.
constant; one new base pair formed with the (red) invading strand is matched by the loss of one base pair with the parent (blue) strand.

When the hybrid region extends beyond the $5^{\prime}$ end of the broken strand that was digested by the $5^{\prime}$-exonuclease in step $\mathbf{B}$ (light blue), the single-stranded parent DNA strand generated (light blue) base-pairs with the complementary
region of the other parent strand (dark blue), which becomes single-stranded as the branch migrates to the left (Figure 5-40, step $\mathbf{B}$ ). The resulting structure is called a Holliday structure, after Robin Holliday, the geneticist who first proposed it as an intermediate in genetic recombination. Again, the diagonal lines in the diagram following step represent single phosphodiester bonds, and all bases in the Holliday structure are base-paired to complementary bases in the parent strands. Cleavage of the phosphodiester bonds that cross over from one parent strand to the other (step $\mathbf{B}$ ) and ligation of the $5^{\prime}$ and $3^{\prime}$ ends base-paired to the same parent strands (step $\mathbf{B}$ ) result in the generation of a structure similar to a replication fork. Rebinding of replication fork proteins results in extension of the leading strand past the point of the original strand break and re-initiation of lagging-strand synthesis (step $\mathbf{B}$ ), thus regenerating a replication fork. The overall process allows the ligated upper strand in the lower molecule following step $\mathbf{B}$ to serve as a template for extension of the leading strand in step $\mathbf{B}$.

Repair of a Double-Strand Break by Homologous Recombination A similar mechanism, called homologous recombination, can repair a double-strand break in a chromosome and can also exchange large segments of two double-stranded DNA molecules (Figure 5-41). First, the broken ends of the DNA molecule are digested by $5^{\prime}$-exonucleases, leaving a single-stranded region of DNA with a $3^{\prime}$ end (step $\mathbf{B}$ ). RecA in bacteria or Rad51 in eukaryotes then catalyzes invasion of one of these $3^{\prime}$ ends into the homologous region of the homologous chromosome, as described above for repair of a collapsed replication fork (step $\mathbf{B}$ ). The $3^{\prime}$ end of the invading DNA strand is then extended by a DNA polymerase, displacing the parent strand as an enlarging single-stranded loop of DNA (dark blue) (step $\mathbf{B}$ ). When the loop extends to a sequence that is complementary to the other broken and $5^{\prime}$-exonuclease -digested end of DNA (the fragment on the left following step $\mathbf{B}$ ), the complementary sequences basepair (diagram following step $\mathbf{B}$ ). This $3^{\prime}$ end is then extended by a DNA polymerase using the displaced single-stranded loop of parent DNA (dark blue) as a template (step $\mathbf{B}$ ).

Next the new $3^{\prime}$ ends are ligated (step $\mathbf{B}$ ) to the exonu-clease-digested $5^{\prime}$ ends. This generates two Holliday structures in the paired molecules (step $\mathbf{B}$ ). Branch migration of these Holliday structures can occur in either direction (not diagrammed). Finally, cleavage of the strands at the positions shown by the arrows, and ligation of the alternative $5^{\prime}$ and $3^{\prime}$ ends at each cleaved Holliday structure, generates two recombinant chromosomes that contain the DNA of one parent DNA molecule on one side of the initial break point (pink and red strands), and the DNA of the other parent DNA molecule on the other side of the break point (light and dark blue) (step $\mathbf{B}$ ). The region in the immediate vicinity of the initial break point forms a heteroduplex, in which one strand from one parent is base-paired to the complementary strand of the other parent (pink or red strand base-paired to light or dark blue strand). Base-pair mismatches between the two parent strands are usually repaired by the repair mechanisms discussed above to generate

![li492239539b](li492239539b.jpg)

FIGURE 5-41 Repair of a double-strand break by homologous recombination. For simplicity, each DNA double helix is represented by two parallel lines with the polarities of the strands indicated by arrowheads at their $3^{\prime}$ ends. The upper molecule has a double-strand break.
a complementary base pair. In the process, sequence differences between the two parents are lost, a process referred to as gene conversion.

Figure 5-42 diagrams how cleavage of one or the other pair of strands at the four-way strand junction in the Holliday structure generates parent or recombinant molecules. This process, called resolution of the Holliday structure, separates DNA molecules initially joined by RecA/Rad51catalyzed strand invasion. Each Holliday structure in the intermediate following Figure 5-41, step $\mathbf{B}$, can be cleaved

Note that in the diagram of the upper DNA molecule, the strand with its $3^{\prime}$ end at the right is on the top, while in the diagram of the lower DNA molecule, this strand is drawn on the bottom. See the text for discussion. See T. L. Orr-Weaver and J. W. Szostak, 1985, Microbiol. Rev. 49:33.
and religated in the two possible ways shown by the two sets of small black arrows in Figure 5-42 (step $\mathbf{B}$ or step $\mathbf{B}$ ). Consequently, there are four possible products of the recombination process shown in Figure 5-41. After ligation of the cleaved ends, two of these products regenerate the parent chromosomes [with the exception of the heteroduplex region at the break point that is repaired into the sequence of one parent or the other (gene conversion)]. The other two possible products generate recombinant chromosomes as shown in Figure 5-41.

![li502239539b](li502239539b.jpg)

FIGURE 5-42 Alternative resolution of a Holliday structure. Diagonal and vertical lines represent a single phosphodiester bond. It is simplest to diagram the process by rotating the diagram of the bottom molecule $180^{\circ}$ so that the top and bottom molecules have the same
strand orientations. Cutting the bonds as shown in $\boldsymbol{\square}$ and ligating the ends as indicated regenerates the original chromosomes. Cutting the strands as shown in $\boldsymbol{\square}$ and religating as shown at the bottom generates recombinant chromosomes.

## KEY CONCEPTS OF SECTION 5.6

## DNA Repair and Recombination

- Changes in the DNA sequence result from copying errors and the effects of various physical and chemical agents.
- Many copying errors that occur during DNA replication are corrected by the proofreading function of DNA polymerases, which can recognize incorrect (mispaired) bases at the $3^{\prime}$ end of the growing strand and then remove them by means of an inherent $3^{\prime} \rightarrow 5^{\prime}$ exonuclease activity (see Figure 5-33).
- Eukaryotic cells have three excision-repair systems for correcting mispaired bases and for removing chemical adducts from DNA. Base excision repair, mismatch repair, and nucleotide excision repair operate with high accuracy and generally do not introduce errors.
- Repair of double-strand breaks by the nonhomologous end joining pathway can link segments of DNA from different chromosomes, possibly forming an oncogenic chromosomal translocation. This repair mechanism also produces a small deletion, even when segments from the same chromosome are joined.
- Error-free repair of double-strand breaks in DNA is accomplished by homologous recombination using the undamaged sister chromatid as a template.
- Inherited defects in the nucleotide excision-repair pathway, as in individuals with xeroderma pigmentosum, predispose them to skin cancer. Inherited colon cancer is frequently associated with mutant forms of proteins essential for the mismatch repair pathway. Defects in repair by homologous recombination are associated with inheritance of one mutant allele of the BRCA-1 or BRCA-2 gene and result in predisposition to breast and uterine cancer.


### 5.7 Viruses: Parasites of the Cellular Genetic System

Viruses are obligate, intracellular parasites. They cannot reproduce by themselves and must commandeer a host cell's machinery to synthesize viral proteins and, in some cases, to replicate the viral genome. RNA viruses, which usually replicate in the host-cell cytoplasm, have an RNA genome, and DNA viruses, which commonly replicate in the host-cell nucleus, have a DNA genome (see Figure 5-1). Viral genomes may be single- or double-stranded, depending on the specific type of virus. The entire infectious virus particle, called a virion, consists of nucleic acid and an outer shell of protein that both protects the viral nucleic acid and functions in the process of host-cell infection. The simplest viruses contain only enough RNA or DNA to encode four proteins; the most complex can encode some two hundred proteins. In addition to their obvious importance as causes of disease, viruses are extremely useful as research tools in the study of basic biological processes such as those discussed in this chapter.

## Most Viral Host Ranges Are Narrow

The surface of a virion contains many copies of one type of protein that binds specifically to multiple copies of a receptor protein on a host-cell surface. This interaction determines the host range-the group of cell types that a virus can infect-and begins the infection process. Most viruses have a rather limited host range.

A virus that infects only bacteria is called a bacteriophage, or simply a phage. Viruses that infect animal or plant cells are referred to generally as animal viruses or plant viruses. A few viruses can grow in both plants or animals and the insects that feed on them. The highly mobile insects serve as vectors for transferring such viruses between susceptible animal or plant hosts. Wide host ranges are also characteristic of some strictly animal viruses, such as vesicular stomatitis

virus, which grows in insect vectors and in many different types of mammals. Most animal viruses, however, do not cross phyla, and some (e.g., poliovirus) infect only closely related species, such as primates. The host-cell range of some animal viruses is further restricted to a limited number of cell types because only those cells have surface receptors to which the virions can attach. One example is poliovirus, which infects only cells in the intestine and, unfortunately for its host, motor neurons in the spinal cord, causing paralysis. Another is HIV-1, discussed further below, which infects cells called CD4 ${ }^{+} \mathrm{T}$ lymphocytes that are essential for the immune response (see Chapter 23) as well as certain neurons and other cells of the central nervous system called glial cells.

## Viral Capsids Are Regular Arrays of One or a Few Types of Protein

The nucleic acid of a virion is enclosed within a protein coat, or capsid, composed of multiple copies of one protein or a few different proteins, each of which is encoded by a single viral gene. Because of this structure, a virus is able to encode all the information for making a relatively large capsid in a small number of genes. This efficient use of genetic information is important because only a limited amount of DNA or RNA, and therefore a limited number of genes, can fit into a virion capsid.

Nature has found two basic ways of arranging the multiple capsid protein subunits and the viral genome into a virion. In some viruses, multiple copies of a single capsid protein form a helical structure that encloses and protects the viral RNA or DNA, which runs in a helical groove within the protein tube. Viruses with such a helical structure, such as tobacco mosaic virus, have a rodlike shape (Figure 5-43a). The other major structural type is based on the icosahedron, a solid, approximately spherical object built of 20 identical faces, each of which is an equilateral triangle (Figure 5-43b). During infection, some icosahedral viruses interact with host-cell receptors via clefts in between the capsid subunits. Others interact via long fiberlike proteins extending from the vertices of the icosahedron.

In many DNA bacteriophages, the viral DNA is located within an icosahedral "head" that is attached to a rodlike "tail." During infection, viral proteins at the tip of the tail bind to host-cell receptors, and the viral DNA then passes down the tail into the cytoplasm of the host cell (Figure 5-43c).

In some viruses, a symmetrically arranged nucleocapsid composed of the viral genome associated with multiple copies of one or a few proteins is covered by an external membrane, or envelope, which consists mainly of a phospholipid bilayer but also contains one or more types of virus-encoded glycoproteins (Figure 5-43d). The phospholipids in the viral envelope are similar to those in the plasma membrane of an infected host cell. The viral envelope is, in fact, derived by budding from that membrane, but contains mainly viral glycoproteins, as we will discuss shortly.

## Viruses Can Be Cloned and Counted in Plaque Assays

The number of infectious viral particles in a sample can be quantified by a plaque assay. This assay is performed by culturing a dilute sample of viral particles on a plate covered with host cells and then counting the number of local lesions, called plaques, that develop (Figure 5-44). A plaque develops on the plate wherever a single virion initially infects a single cell. The virus replicates in this initial host cell and then lyses (ruptures) the cell, releasing many progeny virions that infect the neighboring cells on the plate. After a few such cycles of infection, enough cells are lysed to produce a visible clear area, or plaque, in the layer of remaining uninfected cells. Since all the progeny virions in a plaque are derived from a single parent virus, they constitute a viral clone.

This type of plaque assay is in standard use for bacterial and animal viruses. Plant viruses can be assayed similarly by counting local lesions on plant leaves inoculated with viruses. Analysis of viral mutants, which are commonly isolated by plaque assays, has contributed extensively to our current understanding of molecular cellular processes.

## Lytic Viral Growth Cycles Lead to Death of Host Cells

Although details vary among different types of viruses, those that exhibit a lytic cycle of growth proceed through the following general stages:

1. Adsorption-Virion interacts with a host cell by binding of multiple copies of capsid protein to specific receptors on the cell surface.
2. Penetration-Viral genome crosses the host plasma membrane. For some viruses, viral proteins packaged inside the capsid also enter the host cell.
3. Replication-Viral mRNAs are produced with the aid of the host-cell transcription machinery (DNA viruses) or by viral enzymes (RNA viruses). For both types of viruses, viral mRNAs are translated by the host-cell translation machinery. Production of multiple copies of the viral genome is carried out either by viral proteins alone or with the help of host-cell proteins.
4. Assembly-Viral proteins and replicated genomes associate to form progeny virions.
5. Release-The host cell either ruptures suddenly (lysis), releasing all the newly formed virions at once, or disintegrates gradually, releasing the virions slowly. Both types of release lead to the death of the infected cell.

Figure 5-45 illustrates the lytic cycle for T4 bacteriophage, a nonenveloped DNA virus that infects E. coli. Viral capsid proteins generally are made in large amounts because many copies of them are required for the assembly of each progeny virion. In each infected E. coli cell, about 100-200 T4 progeny virions are produced and released by lysis.

(a)
![li512239539b](li512239539b.jpg)
(c)
![li522239539b](li522239539b.jpg)

The lytic cycle is somewhat more complicated for DNA viruses that infect eukaryotic cells. In most such viruses, the DNA genome is transported (with some associated proteins) into the cell nucleus. Once inside the nucleus, the viral DNA is transcribed into RNA by the host's transcription machinery. Processing of the viral RNA primary transcript by host-cell enzymes yields viral mRNA, which is transported to the cytoplasm and translated into viral proteins by host-cell ribosomes, tRNA, and translation factors. The viral proteins are then transported back into the nucleus, where some of them either replicate the viral DNA directly or direct cellular proteins to replicate the viral DNA, as in the case of SV40, discussed earlier. Association
(b)
![li532239539b](li532239539b.jpg)
(d)
![li542239539b](li542239539b.jpg)

FIGURE 5-43 Virion structures. (a) Helical tobacco mosaic virus. (b, left) Diagram of the structure of poliovirus, a small icosahedral virus, made of 20 equilateral triangular faces, one of which is outlined in red. Each face is composed of three outlined structural elements called capsomeres. The numbers show how five capsomeres associate at the 12 vertices of the icosahedron. (b, right) Space-filling model of poliovirus based on x-ray crystallography. The model is color-coded according to distance from the center of the virion, red furthest, blue closest. The virion binds to host cell receptors (not shown), which are long narrow cell surface proteins that enter the blue "canyons" around each vertex. (c) Bacteriophage T4. (d) Influenza virus, an example of an enveloped virus. [Part (a): Omikron/Science Source. Part (b) data from D. J. Filman et al., 1989. EMBO J. 8:1567, PDB ID 2plv. Part (c) Department of Microbiology, Biozentrum, University of Basel/Science Source. Part (d) James Cavallini/Science Source.]
of the capsid proteins with the newly replicated viral DNA occurs in the nucleus, yielding thousands to hundreds of thousands of progeny virions. Since mammalian cells are about a thousand times larger than bacterial cells, on the order of a thousand times as many virions are produced per cell. However, since many more proteins must be synthesized and viral genomes replicated to assemble this large number of virions, one cycle of infection takes much longer for an animal virus than for bacteriophage of comparable complexity. Bacteriophage T4 replicates in and lyses E. coli cells in about 20 minutes, whereas replication of poliovirus and lysis of host cells requires about 8 hours. Many animal viruses are considerably slower than poliovirus, requiring

![li552239539b](li552239539b.jpg)

Each plaque represents cell lysis initiated by one viral particle (agar restricts movement so that virus can infect only contiguous cells)
(b)
![li562239539b](li562239539b.jpg)

Blaque
EXPERIMENTAL FIGURE 5-44 Plaque assay. The plaque assay determines the number of infectious particles in a viral suspension. (a) A lesion, or plaque, develops where a single virion initially infected a single cell. Each plaque constitutes a pure viral clone. (b) Plaques on a lawn of Pseudomonas fluorescens bacteria made by bacteriophage $\phi 51$. [Part (b) P. Rossi, 1994, Advances in biological tracer techniques for hydrology and hydrogeology using bacteriophages, Ph. D. thesis, University of Neuchatel, 200 pp. http://doc.rero.ch/record/2576/files/these_RossiP.]
2 days or more for lysis of an infected cell and release of progeny virions.

Most plant and animal viruses with an RNA genome do not require nuclear functions for lytic replication. In some of these viruses, a virus-encoded enzyme that enters the host cell during penetration transcribes the viral genomic RNA into mRNA in the cell cytoplasm. The mRNA is directly translated into viral proteins by the host-cell translation machinery. One or more of these proteins then produces additional copies of the viral RNA genome. Finally, progeny genomes are associated with newly synthesized capsid proteins to form progeny virions in the cytoplasm.

After the synthesis of hundreds to hundreds of thousands of new virions has been completed, depending on the type of virus and host cell, most infected bacterial cells and some
![li572239539b](li572239539b.jpg)

FIGURE 5-45 Lytic replication cycle of a nonenveloped bacterial virus. E. coli bacteriophage T4 has a double-stranded DNA genome and lacks a membrane envelope. After viral capsid proteins at the tip of the tail in T4 interact with specific receptor proteins on the exterior of the host cell, the viral genome is injected into the host (step $\mathbf{\square}$ ). Host-cell enzymes then transcribe viral "early" genes into mRNAs and subsequently translate these mRNAs into viral "early" proteins (step $\mathbf{\square}$ ). The early proteins replicate the viral DNA and induce expression of viral "late" proteins by host-cell enzymes (step $\mathbf{\square}$ ). The viral late proteins include capsid and assembly proteins and enzymes that degrade the host-cell DNA, supplying nucleotides for synthesis of more viral DNA. Progeny virions are assembled in the cell (step $\mathbf{\square}$ ) and released (step $\mathbf{\square}$ ) when viral proteins lyse the cell. Newly liberated viruses initiate another cycle of infection in other host cells.
infected plant and animal cells are lysed, releasing all the virions at once. In many plant and animal viral infections, however, no discrete lytic event occurs; rather, the dead host cell releases the virions as it gradually disintegrates.

As noted previously, enveloped animal viruses are surrounded by an outer phospholipid bilayer, derived from the plasma membrane of a host cell, that contains abundant viral glycoproteins. The processes of adsorption and release of enveloped viruses differ substantially from these processes for nonenveloped viruses. To illustrate lytic replication of enveloped viruses, we consider the rabies virus, whose nucleocapsid consists of a single-stranded RNA genome bound by multiple copies of a nucleocapsid protein. Like other lytic RNA viruses, the rabies virus is replicated in the cytoplasm and does not require host-cell nuclear enzymes. As shown in

![li582239539b](li582239539b.jpg)

FIGURE 5-46 Lytic replication cycle of an enveloped animal virus. The rabies virus is an enveloped virus with a single-stranded RNA genome. The structural components of this virus are depicted at the top. After a virion adsorbs to multiple copies of a specific host membrane protein (step 5), the cell engulfs it in an endosome (step 5). A cellular protein in the endosome membrane pumps $\mathrm{H}^{+}$ions from the cytosol into the endosome interior. The resulting decrease in endosomal pH induces a conformational change in the viral glycoprotein, leading to fusion of the viral envelope with the endosomal lipid bilayer and release of the nucleocapsid into the cytosol (steps $\mathbf{1}$ and $\mathbf{2}$ ). Viral RNA polymerase uses ribonucleoside triphosphates in the cytosol to replicate the viral RNA genome (step 5) and to synthesize viral mRNAs (step 5). One of the viral mRNAs encodes the viral transmembrane glycoprotein, which is inserted into the membrane of the endoplasmic
reticulum (ER) as it is synthesized on ER-bound ribosomes (step 5). Carbohydrate is added to the large folded domain inside the ER lumen and is modified as the membrane and the associated glycoproteins pass through the Golgi apparatus (step 5). Vesicles with mature glycoprotein fuse with the host-cell plasma membrane, depositing viral glycoprotein on the cell surface with the large receptor-binding domain outside the cell (step 5). Meanwhile, other viral mRNAs are translated on host-cell ribosomes into nucleocapsid protein, matrix protein, and viral RNA polymerase (step 5). These proteins are assembled with replicated viral genomic RNA (bright red) into progeny nucleocapsids (step 5), which then associate with the cytosolic domain of viral transmembrane glycoproteins in the host-cell plasma membrane (step 5). The plasma membrane is folded around the nucleocapsid, forming a "bud" that is eventually released (step 5).
however, the viral DNA is integrated into the host-cell chromosome rather than being replicated. The integrated viral DNA, called a prophage, is replicated as part of the host cell's DNA from one host-cell generation to the next. This phenomenon is referred to as lysogeny. If the host cell suffers extensive damage to its DNA from UV light, the prophage DNA is activated, leading to its excision from the host-cell chromosome, entrance into the lytic cycle, and subsequent production and release of progeny virions before the host cell dies.

The genomes of a number of animal viruses can also integrate into the host-cell genome. Among the most important of these are the retroviruses, which are enveloped viruses with a genome consisting of two identical strands of RNA. These viruses are so named because their RNA genome acts

![li592239539b](li592239539b.jpg)

EXPERIMENTAL FIGURE 5-47 Release of progeny virions by budding. Progeny virions of enveloped viruses are released by budding from infected cells. In this transmission electron micrograph of a cell infected with measles virus, virion buds are clearly visible protruding from the cell surface. Measles virus is an enveloped RNA virus with a helical nucleocapsid, like rabies virus, and replicates as illustrated in Figure 5-46. [Thomas Deerinck, NCMIR/Science Source.]
as a template for the formation of a DNA molecule-a flow of genetic information that is opposite to the more common transcription of DNA into RNA. In the retroviral life cycle (Figure 5-48), a viral enzyme called reverse transcriptase initially copies the viral RNA genome into single-stranded DNA that is complementary to the viral RNA; the same enzyme then catalyzes the synthesis of a complementary DNA strand. (This complex reaction is detailed in Chapter 8 , where we consider closely related intracellular parasites called retrotransposons.) The resulting double-stranded DNA is integrated into the chromosomal DNA of the infected cell by an integrase enzyme in the virion. Finally, the integrated DNA, called a provirus, is transcribed by the host cell's own RNA polymerase into RNA, which is either translated into viral proteins or packaged within virion capsid proteins to form progeny virions that are released by budding from the host-cell membrane. Because most retroviruses do not kill their host cells, infected cells can replicate, producing daughter cells with integrated proviral DNA. These daughter cells continue to transcribe the proviral DNA and bud progeny virions.

Some retroviruses contain cancer-causing genes (oncogenes), and cells infected by such retroviruses are oncogenically transformed into tumor cells. Studies of oncogenic retroviruses (mostly viruses of birds and mice) have revealed a great deal about the processes that lead to transformation of a normal cell into a cancer cell (see Chapter 24).
![li602239539b](li602239539b.jpg)

FIGURE 5-48 Retroviral life cycle. Retroviruses have a genome of two identical copies of single-stranded RNA and an outer envelope. Step After viral glycoproteins in the retroviral envelope interact with a specific host-cell membrane protein, the envelope fuses directly with the plasma membrane, allowing entry of the nucleocapsid into the cytoplasm of the cell. Step Viral reverse transcriptase and other proteins copy the viral ssRNA genome into a double-stranded DNA. Step The viral dsDNA is transported into the nucleus and integrated
into one of many possible sites in the host-cell chromosomal DNA. For simplicity, only one host-cell chromosome is depicted. Step $\mathbf{B}$ : The integrated viral DNA (provirus) is transcribed by the host-cell RNA polymerase, generating viral mRNAs (dark red) and viral genomic RNA molecules (bright red). The host-cell machinery translates the viral mRNAs into glycoproteins and nucleocapsid proteins. Step Progeny virions then assemble and are released by budding, as illustrated in Figure 5-46.

5Among the known human retroviruses are human Tcell lymphotrophic virus (HTLV), which causes a form of leukemia, and human immunodeficiency virus (HIV-1), which causes acquired immune deficiency syndrome (AIDS). Both of these viruses can infect only specific cell types, primarily certain cells of the immune system and, in the case of HIV-1, some central nervous system neurons and glial cells. Only those cells have cell-surface receptors that interact with viral envelope proteins. Unlike most other retroviruses, HIV-1 eventually kills its host cells. The eventual death of large numbers of immune-system cells results in the defective immune response characteristic of AIDS.

Some DNA viruses can also integrate into a host-cell chromosome. One example is the human papillomaviruses (HPVs), which most commonly cause warts and other benign skin lesions. These viruses replicate without integrating into a host-cell chromosome. However, the genomes of certain HPV serotypes occasionally integrate into the chromosomal DNA of infected cervical epithelial cells. Unlike retroviruses and temperate bacteriophages, this integration is not carried out by viral proteins, but rather is the consequence of host-cell DNA repair processes. Integration is a dead-end for HPV. The integrated viral DNA cannot replicate and generate progeny virions. But oncogenic (cancercausing) viral proteins can be expressed from the integrated viral genome, initiating development of cervical cancer. Routine Pap smears can detect cells in the early stages of the transformation process initiated by HPV integration, permitting effective treatment before cancer develops. A vaccine for the types of HPV associated with cervical cancer has been developed and can protect against the initial infection by these viruses, and consequently, against development of cervical cancer. However, once an individual is infected with these HPVs, the "window of opportunity" is missed, and the vaccine does not protect against the development of cancer. Because the vaccine is not 100 percent effective, even vaccinated women should have regular Pap smears.

## KEY CONCEPTS OF SECTION 5.7

## Viruses: Parasites of the Cellular Genetic System

- Viruses are small parasites that can replicate only in host cells. Viral genomes may be either DNA (DNA viruses) or RNA (RNA viruses) and may be either single- or doublestranded.
- The capsid, which surrounds the viral genome, is composed of multiple copies of one or a small number of virusencoded proteins. Some viruses also have an outer envelope, which is similar to the plasma membrane of the host cell but contains viral transmembrane proteins.
- Most animal and plant DNA viruses require host-cell nuclear enzymes to carry out transcription of the viral genome into mRNA and production of progeny genomes. In contrast, most RNA viruses encode enzymes that can transcribe the RNA genome into viral mRNA and produce new copies of the RNA genome.
- Host-cell ribosomes, tRNAs, and translation factors are used in the synthesis of all viral proteins in infected cells.
- Lytic viral infection entails adsorption, penetration, synthesis of viral proteins and progeny genomes (replication), assembly of progeny virions, and release of hundreds to thousands of virions, leading to death of the host cell (see Figure 5-45). Release of enveloped viruses occurs by budding through the host-cell plasma membrane (see Figure 5-46).
- Nonlytic infection occurs when the viral genome is integrated into the host-cell DNA and generally does not lead to cell death.
- Retroviruses are enveloped animal viruses containing a single-stranded RNA genome. After a host cell is penetrated, reverse transcriptase, a viral enzyme carried in the virion, converts the viral RNA genome into double-stranded DNA, which is integrated into chromosomal DNA by an integrase enzyme that enters the cell inside the virion (see Figure 5-48).
- Unlike infection by other retroviruses, HIV-1 infection eventually kills host cells, causing the defects in the immune response characteristic of AIDS.
- Tumor viruses, which contain oncogenes, may have an RNA genome (e.g., human T-cell lymphotrophic virus) or a DNA genome (e.g., human papillomaviruses). Integration of the genomes of these viruses into a host-cell chromosome can cause transformation of the host cell into a tumor cell.


## LaunchPad

macmillan learning

Visit LaunchPad to access study tools and to learn more about the content in this chapter:

- Perspectives for the Future
- Analyze the Data
- Extended References
- Additional study tools, including videos, animations, and quizzes

## Key Terms

anticodon 183
codon 183
complementary 171
DNA polymerase 198
double helix 170
excision-repair systems
gene conversion 211
genetic code 183
Holliday structure 210
homologous recombination 208
isoform 182
lagging strand 199
leading strand 199
messenger RNA
(mRNA) 168
mutation 203

Okazaki fragments 199
phosphodiester bond 170
polyribosome 195
primer 198
promoter 176
204 reading frame 184
recombination 209
replication fork 199
retroviruses 216
reverse transcriptase 217
ribosomal RNA
(rRNA) 168
ribosome 183
RNA polymerase 176
transcription 168
transfer RNA (tRNA) 168
translation 168

## Review the Concepts

1. What are Watson-Crick base pairs? Why are they important?
2. Preparing plasmid (double-stranded, circular) DNA for sequencing involves annealing a complementary, short, single-stranded oligonucleotide DNA primer to one strand of the plasmid template. This is routinely accomplished by heating the plasmid DNA and primer to $90^{\circ} \mathrm{C}$ and then slowly bringing the temperature down to $25^{\circ} \mathrm{C}$. Why does this protocol work?
3. What difference between RNA and DNA helps to explain the greater stability of DNA? What implications does this have for the function of DNA?
4. What are the major differences in the synthesis and structure of prokaryotic and eukaryotic mRNAs?
5. While investigating the function of a specific growth factor receptor gene from humans, researchers found that two types of proteins are synthesized from this gene. A larger protein containing a membrane-spanning domain recognizes growth factors at the cell surface, stimulating a specific downstream signaling pathway. In contrast, a related, smaller protein is secreted from the cell and binds available growth factor circulating in the blood, thus inhibiting the downstream signaling pathway. Speculate on how the cell synthesizes these disparate proteins.
6. The transcription of many bacterial genes relies on functional groups called operons, such as the tryptophan operon (see Figure 5-13a). What is an operon? What advantages are there to having genes arranged in an operon, compared with the arrangement in eukaryotes?
7. How would a mutation in the poly(A)-binding protein gene affect translation? How would an electron micrograph of polyribosomes from such a mutant differ from the normal pattern?
8. What characteristic of DNA results in the requirement that some DNA synthesis be discontinuous? How are Okazaki fragments and DNA ligase used by the cell?
9. Eukaryotes have repair systems that prevent mutations due to copying errors and exposure to mutagens. What are the three excision-repair systems found in eukaryotes, and which one is responsible for correcting thymine-thymine dimers that form as a result of UV light damage to DNA?
10. DNA repair systems are responsible for maintaining genomic fidelity in normal cells despite the high frequency with which mutational events occur. What type of DNA mutation is generated by (a) UV radiation and (b) ionizing radiation? Describe the system responsible for repairing each of these types of mutations in mammalian cells. Postulate why a loss of function in one or more DNA repair systems typifies many cancers.
11. What is the name given to the process that can repair DNA damage and generate genetic diversity? Briefly describe the similarities and differences of the two processes.
12. The genome of a retrovirus can integrate into the hostcell genome. What gene is unique to retroviruses, and why is the protein encoded by this gene absolutely necessary for maintaining the retroviral life cycle? A number of retroviruses can infect certain human cells. List two of them, briefly describe the medical implications resulting from these infections, and describe why only certain cells are infected.
13. a. Which of the following DNA strands, the top or bottom, would serve as a template for RNA transcription if the DNA molecule were to unwind in the indicated direction?

5' ACGGACTGTACCGCTGAAGTCATGGACGCTCGA 3' 3' TGCCTGACATGGCGACTTCAGTACCTGCGAGCT 5'

Direction of DNA unwinding
b. What would be the resulting RNA sequence (written $5^{\prime} \rightarrow 3^{\prime}$ )?
14. Contrast prokaryotic and eukaryotic gene characteristics. 15. You have learned about the events surrounding DNA replication and the central dogma. Identify the steps associated with these processes that would be adversely affected in the following scenarios.
a. Helicases unwind the DNA, but stabilizing proteins are mutated and cannot bind to the DNA.
b. The mRNA molecule forms a hairpin loop on itself via complementary base pairing in an area spanning the AUG start site.
c. The cell is unable to produce functional tRNA ${ }_{i}^{\text {Met }}$.

16. Use the key provided below to determine the amino acid sequence of the polypeptide produced from the following DNA sequence. Intron sequences are highlighted. Note: Not all amino acids in the key will be used.

5'TTCTAAACGCATGAAGCACCGTCTCAGAGCCAGTGA 3' 3' AAGATTTGCGTACTTCGTGGCAGAGTCTCGGTCACT 5'
Direction of DNA unwinding
Asn = AAU Cys = TCG Gly = CAG His = CAU Lys = AAG
Met $=$ AUG Phe $=$ UUC Ser $=$ AGC Tyr $=$ UAC Val $=$ GUC; GUA
![li612239539b](li612239539b.jpg)
17. a. Look at the figure above. Explain why it is necessary for Okazaki fragments to be formed as the lagging strand is produced (instead of a continuous strand).
b. If the DNA polymerase in the figure above could bind only to the lower template strand, under what condition(s) would it be able to produce a leading strand?
18. The DNA repair systems preferentially target the newly synthesized strand. Why is this important?
19. Identify the specific types of point mutations below (you are viewing the direct DNA version of the RNA sequence).

Original sequence: 5' AUG TCA GGA CGT CAC TCA GCT 3' Mutation A: 5' AUG TCA GGA CGT CAC TGA GCT 3' Mutation B: 5' AUA TCA GGA CGT CAC TCA GCT 3'
20. a. Detail the key differences between lytic and nonlytic viral infection and provide an example of each.
b. Which of the following processes occurs in both lytic and nonlytic viral infections?
(i) Infected cell ruptures to release viral particles.
(ii) Viral mRNAs are transcribed by the host-cell translation machinery.
(iii) Viral proteins and nucleic acids are packaged to produce virions.

## References

## Structure of Nucleic Acids

Arnott, S. 2006. Historical article: DNA polymorphism and the early history of the double helix. Trends Biochem. Sci. 31:349-354.

Berger, J. M., and J. C. Wang. 1996. Recent developments in DNA topoisomerase II structure and mechanism. Curr. Opin. Struc. Biol. 6:84-90.

Cech, T. R. 2009. Evolution of biological catalysis: ribozyme to RNP enzyme. Cold Spring Harbor Symp. Quant. Biol. 74:11-16.

Dickerson, R. E. 1992. DNA Structure from A to Z. Meth. Enzymol. 211:67-111.

Kornberg, A., and T. A. Baker. 2005. DNA Replication. University Science, chap. 1.

Lilley, D. M. 2005. Structure, folding and mechanisms of ribozymes. Curr. Opin. Struc. Biol. 15:313-323.

Vicens, Q., and T. R. Cech. 2005. Atomic level architecture of group I introns revealed. Trends Biochem. Sci. 31:41-51.

Wang, J. C. 1980. Superhelical DNA. Trends Biochem. Sci. 5:219-221.

Wigley, D. B. 1995. Structure and mechanism of DNA topoisomerases. Ann. Rev. Biophys. Biomol. Struc. 24:185-208.

## Transcription of Protein-Coding Genes and Formation of Functional mRNA

Brenner, S., F. Jacob, and M. Meselson. 1961. An unstable intermediate carrying information from genes to ribosomes for protein synthesis. Nature 190:576-581.

Brueckner F., J. Ortiz, and P. Cramer. 2009. A movie of the RNA polymerase nucleotide addition cycle. Curr. Opin. Struc. Biol. 19:294-299.

Murakami, K. S., and S. A. Darst. 2003. Bacterial RNA polymerases: the whole story. Curr. Opin. Struc. Biol. 13:31-39.

Okamoto K., Y. Sugino, and M. Nomura. 1962. Synthesis and turnover of phage messenger RNA in E. coli infected with bacteriophage T4 in the presence of chloromycetin. J. Mol. Biol. 5:527-534.

Steitz, T. A. 2006. Visualizing polynucleotide polymerase machines at work. EMBO J. 25:3458-3468.

## The Decoding of mRNA by tRNAs

Alexander, R. W., and P. Schimmel. 2001. Domain-domain communication in aminoacyl-tRNA synthetases. Prog. Nucl. Acid Res. Mol. Biol. 69:317-349.

Hatfield, D. L., and V. N. Gladyshev. 2002. How selenium has altered our understanding of the genetic code. Mol. Cell Biol. 22:3565-3576.

Hoagland, M. B., et al. 1958. A soluble ribonucleic acid intermediate in protein synthesis. J. Biol. Chem. 231:241-257.

Ibba, M., and D. Soll. 2004. Aminoacyl-tRNAs: setting the limits of the genetic code. Genes \& Dev. 18:731-738.

Khorana, G. H., et al. 1966. Polynucleotide synthesis and the genetic code. Cold Spring Harbor Symp. Quant. Biol. 31:39-49.

Nakanishi, K., and O. Nureki. 2005. Recent progress of structural biology of tRNA processing and modification. Mol. Cells 19:157-166.

Nirenberg, M., et al. 1966. The RNA code in protein synthesis. Cold Spring Harbor Symp. Quant. Biol. 31:11-24.

Rich, A., and S.-H. Kim. 1978. The three-dimensional structure of transfer RNA. Sci. Am. 240(1):52-62 (offprint 1377).

## Stepwise Synthesis of Proteins on Ribosomes

Ban N., et al. 2000. The complete atomic structure of the large ribosomal subunit at 2.4 Å resolution. Science 289:905-920.

Belousoff, M. J., et al. 2010. Ancient machinery embedded in the contemporary ribosome. Biochem. Soc. Trans. 38:422-427.

Frank, J., and R. L. Gonzalez, Jr. 2010. Structure and dynamics of a processive Brownian motor: the translating ribosome. Ann. Rev. Biochem. 79:381-412.

Jackson, R.J., et al. 2010. The mechanism of eukaryotic translation initiation and principles of its regulation. Nature Rev. Mol. Cell Biol. 11:113-127.

Korostelev, A., D. N. Ermolenko, and H. F. Noller. 2008. Structural dynamics of the ribosome. Curr. Opin. Chem. Biol. 12:674-683.

Livingstone, M., et al. 2010. Mechanisms governing the control of mRNA translation. Phys. Biol. 7(2):021001.

Sarnow, P., R. C. Cevallos, and E. Jan. 2005. Takeover of host ribosomes by divergent IRES elements. Biochem. Soc. Trans. 33:1479-1482.

Steitz, T. A. 2008. A structural understanding of the dynamic ribosome machine. Nature Rev. Mol. Cell Biol. 9:242-253.

Wimberly, B. T., et al. 2000. Structure of the 30S ribosomal subunit. Nature 407:327-339.

Yusupova, G. and M. Yusupov. 2014. High-resolution structure of the eukaryotic 80S ribosome. Ann. Rev. Biochem. 83:467-486. http://www.annualreviews.org/doi/suppl/10.1146/annurev-bio-chem-060713-035445.

## DNA Replication

DePamphilis, M. L., ed. 2006. DNA Replication and Human Disease. Cold Spring Harbor Laboratory Press.

Gai, D., Y. P. Chang, and X. S. Chen. 2010. Origin DNA melting and unwinding in DNA replication. Curr. Opin. Struc. Biol. 20(6):756-762.

Kornberg, A., and T. A. Baker. 2005. DNA Replication. University Science.

Langston, L. D., C. Indiani, and M. O'Donnell. 2009. Whither the replisome: emerging perspectives on the dynamic nature of the DNA replication machinery. Cell Cycle 8:2686-2691.

Langston, L. D., and M. O'Donnell. 2006. DNA replication: keep moving and don't mind the gap. Mol. Cells 23:155-160.

Schoeffler, A. J., and J. M. Berger. 2008. DNA topoisomerases: harnessing and constraining energy to govern chromosome topology. Quart. Rev. Biophys. 41:41-101.

Stillman, B. 2008. DNA polymerases at the replication fork in eukaryotes. Cell 30:259-260.

## DNA Repair and Recombination

Andressoo, J. O., and J. H. Hoeijmakers. 2005. Transcriptioncoupled repair and premature aging. Mutat. Res. 577:179-194.

Barnes, D. E., and T. Lindahl. 2004. Repair and genetic consequences of endogenous DNA base damage in mammalian cells. Ann. Rev. Genet. 38:445-476.

Bell, C. E. 2005. Structure and mechanism of Escherichia coli RecA ATPase. Mol. Microbiol. 58:358-366.

Friedberg, E. C., et al. 2006. DNA repair: from molecular mechanism to human disease. DNA Repair 5:986-996.

Haber, J. E. 2000. Partners and pathways repairing a doublestrand break. Trends Genet. 16:259-264.

Jiricny, J. 2006. The multifaceted mismatch-repair system. Nature Rev. Mol. Cell Biol. 7:335-346.

Khuu, P. A., et al. 2006. The stacked-X DNA Holliday junction and protein recognition. J. Mol. Recog. 19:234-242.

Lilley, D. M., and R. M. Clegg. 1993. The structure of the four-way junction in DNA. Ann. Rev. Biophys. Biomol. Struc. 22:299-328.

Mirchandani, K. D., and A. D. D'Andrea. 2006. The Fanconi anemia/BRCA pathway: a coordinator of cross-link repair. Exp. Cell Res. 312:2647-2653.

Mitchell, J. R., J. H. Hoeijmakers, and L. J. Niedernhofer. 2003. Divide and conquer: nucleotide excision repair battles cancer and aging. Curr. Opin. Cell Biol.15:232-240.

Orr-Weaver, T. L., and J. W. Szostak. 1985. Fungal recombination. Microbiol. Rev. 49:33-58.

Shin, D. S., et al. 2004. Structure and function of the double strand break repair machinery. DNA Repair 3:863-873.

Wood, R. D., M. Mitchell, and T. Lindahl. Human DNA repair genes. Mutat. Res. 577:275-283.

Yoshida, K., and Y. Miki. 2004. Role of BRCA1 and BRCA2 as regulators of DNA repair, transcription, and cell cycle in response to DNA damage. Cancer Sci. 95:866-871.

## Viruses: Parasites of the Cellular Genetic System

Flint, S. J., et al. 2000. Principles of Virology: Molecular Biology, Pathogenesis, and Control. ASM Press.

Hull, R. 2002. Mathews' Plant Virology. Academic Press.
Klug, A. 1999. The tobacco mosaic virus particle: structure and assembly. Phil. Trans. R. Soc. Lond. B Biol. Sci. 354:531-535.

Knipe, D. M., and P. M. Howley, eds. 2012. Fields Virology, 6th ed. Lippincott Williams \& Wilkins.

Kornberg, A., and T. A. Baker. 1992. DNA Replication, 2d ed. W. H. Freeman and Company. Good summary of bacteriophage molecular biology.

this page left intentionally blank

![li622239539b](li622239539b.jpg)

The mouse on the right has cataracts because it carries a single copy of a dominant mutant allele of the gene for gamma-crystallin. The mouse on the left, with normal eyes, was produced from a zygote in which the mutant allele was corrected back to the wild-type state by the CRISPRCas9 genome editing method. [Jinsong Li]

In the field of molecular cell biology, reduced to its most basic elements, we seek an understanding of the biological behavior of cells in terms of the underlying chemical and molecular mechanisms. Often the investigation of a new molecular process focuses on the function of a particular protein or set of proteins. There are three fundamental questions that cell biologists usually ask about a newly discovered protein: What is the function of the protein in the context of a living cell? What is the biochemical function of the purified protein? Where is the protein located? To answer these questions, investigators employ three molecular genetic tools: the gene that encodes the protein, a mutant cell line or organism that lacks the functional protein, and a source of the purified protein itself. In this chapter, we consider various aspects of two basic experimental strategies for obtaining all three tools (Figure 6-1).

The first strategy, often referred to as classical genetics, begins with isolation of a mutant that appears to be defective in some process of interest. Genetic methods are then used to identify and isolate the affected gene. The isolated gene can

## Molecular Genetic Techniques

be manipulated to produce large quantities of the protein it encodes for biochemical experiments and to design probes for studies of where and when the protein is expressed in an organism. The second strategy follows essentially the same steps as the classical approach, but in the reverse order, beginning with isolation of an interesting protein or its identification based on analysis of an organism's genomic sequence. Once the corresponding gene has been isolated, that gene can be altered and then reinserted into an organism. With both strategies, by examining the phenotypic consequences of mutations that inactivate a particular gene, geneticists are able to connect knowledge about the sequence, structure, and biochemical activity of the encoded protein to its function in the context of a living cell or multicellular organism.

An important component in both strategies for studying a protein and its biological function is isolation of the corresponding gene. Thus we discuss various techniques by which researchers can isolate, sequence, and manipulate specific regions of an organism's DNA. Next we introduce

### 6.1 Genetic Analysis of Mutations to Identify and Study Genes

### 6.2 DNA Cloning and Characterization

### 6.3 Using Cloned DNA Fragments to Study Gene Expression

### 6.4 Locating and Identifying Human Disease Genes

### 6.5 Inactivating the Function of Specific Genes in Eukaryotes

![li632239539b](li632239539b.jpg)

FIGURE 6-1 Overview of two strategies for relating the function, location, and structure of gene products. A mutant organism is the starting point for the classical genetic strategy (green arrows). The reverse strategy (orange arrows) usually begins with identification of a protein-coding sequence by analysis of genomic sequence databases. In both strategies, the actual gene is isolated either from a DNA library or by specific amplification of the gene sequence from genomic DNA. Once a cloned gene is isolated, it can be used to produce the encoded protein in bacterial or eukaryotic expression systems. Alternatively, a cloned gene can be inactivated by one of various techniques and used to generate mutant cells or organisms.
a variety of techniques that are commonly used to analyze where and when a particular gene is expressed and where in the cell its protein is localized. In some cases, knowledge of protein function can lead to significant medical advances, and the first step in developing treatments for an inherited disease is to identify and isolate the affected gene, a process we describe here. Finally, we discuss techniques that abolish normal protein function in order to analyze the role of the protein in the cell.

### 6.1 Genetic Analysis of Mutations to Identify and Study Genes

As described in Chapter 5, the information encoded in the DNA sequence of genes specifies the sequence-and therefore the structure and function-of every protein molecule in a cell. The power of genetics as a tool for studying cells and organisms lies in the ability of researchers to selectively alter every copy of one specific type of protein in a cell by making a change in the gene for that protein. Genetic analyses of mutants defective in a particular process can reveal (1) genes required for the process to occur, (2) the order in which gene products act in the process, and (3) whether and how the proteins encoded by different genes interact with one another. Before we see how genetic studies of this type can provide insights into the mechanism of a complicated cellular or developmental process, let's first review some basic genetic terms used throughout our discussion.

The different forms or variants of a gene are referred to as alleles. Geneticists commonly refer to the numerous naturally occurring genetic variants that exist in populations, particularly human populations, as alleles. The term mutation is usually reserved for instances in which an allele is known to have been newly formed, such as after treatment of an experimental organism with a mutagen, an agent that causes a heritable change in the DNA sequence.

Strictly speaking, the particular set of alleles for all the genes carried by an individual constitutes its genotype. However, this term is most often used in a more restricted sense to denote the alleles of a particular gene or genes under examination. For experimental organisms, the term wild type is often used to designate a standard genotype for use as a reference in breeding experiments. Thus the normal, nonmutant allele is usually designated as the wild type. Because of the enormous allelic variation that naturally exists in human populations, the term wild type usually denotes an allele that is present at a much higher frequency than any of the other possible alternatives.

Geneticists draw an important distinction between the genotype and the phenotype of an organism. The term phenotype refers to all the physical attributes or traits of an individual that are the consequence of a given genotype. In practice, however, the term phenotype is usually used to denote the consequences that result from the particular alleles that are under experimental study. Readily observable phenotypic characteristics are critical in the genetic analysis of mutations.

## Recessive and Dominant Mutant Alleles Generally Have Opposite Effects on Gene Function

A fundamental genetic difference between experimental organisms is whether their cells carry two copies of each chromosome or only one copy of each chromosome. The former are referred to as diploid; the latter as haploid. Most complex multicellular organisms (e.g., fruit flies, mice, humans) are diploid, whereas some simple unicellular organisms are haploid. Some organisms, notably the yeast Saccharomyces cerevisiae, can exist in either haploid or diploid states. The normal cells of some organisms, both plants and animals, carry more than two copies of each chromosome and are thus designated polyploid. Moreover, cancer cells begin as diploid cells, but through the process of transformation into

| DIPLOID <br> GENOTYPE | Wild type |  |  | Recessive | Recessive |
| :--: | :--: | :--: | :--: | :--: | :--: |
| DIPLOID <br> PHENOTYPE | Wild type | Mutant | Mutant | Wild type | Mutant |

FIGURE 6-2 Effects of dominant and recessive mutant alleles on phenotype in diploid organisms. A single copy of a dominant mutant allele is sufficient to produce a mutant phenotype, whereas both
cancer cells, can gain extra copies of one or more chromosomes and are thus designated as aneuploid. However, our discussion of genetic techniques and analysis relates to diploid organisms, including diploid yeasts.

Although many different alleles of a gene might occur in different organisms in a population, any individual diploid organism will carry two copies of each gene and thus at most can have two different alleles. A diploid individual with two different alleles is heterozygous for a gene, whereas a diploid individual that carries two identical alleles is homozygous for a gene. A recessive mutant allele is defined as one in which both alleles must be mutant in order for the mutant phenotype to be observed; that is, the individual must be homozygous for the mutant allele to show the mutant phenotype. In contrast, the phenotypic consequences of a dominant mutant allele can be observed in a heterozygous individual carrying one mutant and one wild-type allele (Figure 6-2).

Whether a mutant allele is recessive or dominant provides valuable information about the function of the affected gene and the nature of the causative mutation. Recessive alleles usually result from a mutation that inactivates the affected gene, leading to a partial or complete loss of function. Such recessive mutations may remove part of the gene or the entire gene from the chromosome, disrupt expression of the gene, or alter the structure of the encoded protein, thereby altering its function. Conversely, dominant alleles are often the consequence of a mutation that causes some kind of gain of function. Such dominant mutations may increase the activity of the encoded protein, confer a new function on it, or lead to a new spatial or temporal pattern of expression.

In some rare cases, dominant mutations are associated with a loss of function. For instance, some genes are haploinsufficient, in that removing or inactivating one of the two alleles of such a gene leads to a mutant phenotype because not enough gene product is made. In other rare instances, a dominant mutation in one allele may lead to a structural change in the protein that interferes with the function of the wild-type protein encoded by the other allele. This type of mutation, referred to as a dominant-negative, produces a phenotype similar to that obtained from a loss-of-function mutation.

Some alleles can exhibit both recessive and dominant properties. In such cases, statements about whether an allele is dominant or recessive must specify the phenotype. For example, the allele of the hemoglobin gene in humans designated $H b^{s}$ has more than one phenotypic consequence. Individuals who are homozygous for this allele $\left(H b^{s} / H b^{s}\right)$ copies of a recessive mutant allele must be present to cause a mutant phenotype. Recessive mutations usually cause a loss of function; dominant mutations usually cause a gain of function or an altered function.
have debilitating anemia caused by sickle-cell disease, but heterozygous individuals $\left(H b^{s} / H b^{s}\right)$ do not have the disease. Therefore, $H b^{s}$ is recessive for the trait of sickle-cell disease. On the other hand, heterozygous $\left(H b^{s} / H b^{s}\right)$ individuals are more resistant to malaria than are homozygous $\left(H b^{s} / H b^{s}\right)$ individuals, revealing that $H b^{s}$ is dominant for the trait of malaria resistance.

A mutagen commonly used in experimental organisms is ethylmethane sulfonate (EMS). Although this mutagen can alter DNA sequences in several ways, its most common effect is to chemically modify guanine bases in DNA, ultimately leading to the conversion of a G-C base pair into an A$\cdot$T base pair. Such an alteration in the sequence of a gene that involves only a single base pair is known as a point mutation. A silent point mutation causes no change in the amino acid sequence or activity of a gene's encoded protein. However, observable phenotypic consequences due to changes in a protein's activity can arise from point mutations that result in substitution of one amino acid for another (missense mutations) or introduction of a premature stop codon (nonsense mutations). Addition or deletion of bases may change the reading frame of a gene (frameshift mutations). Because alterations in the DNA sequence leading to a decrease in protein activity are much more likely than alterations leading to an increase or qualitative change in protein activity, mutagenesis usually produces many more recessive mutations than dominant mutations.

## Segregation of Mutations in Breeding Experiments Reveals Their Dominance or Recessivity

Geneticists exploit the normal life cycle of an organism to test for the dominance or recessivity of alleles. To see how this is done, we must first review the type of cell division that gives rise to gametes (sperm and egg cells in higher plants and animals). Whereas the body (somatic) cells of most multicellular organisms divide by mitosis, the germ cells that give rise to gametes undergo meiosis. Like somatic cells, premeiotic germ cells are diploid, containing two homologs of each morphological type of chromosome. The two homologs that constitute each pair of homologous chromosomes are descended from different parents, and thus their genes may exist in different allelic forms. Figure 6-3 depicts the major events in mitotic and meiotic cell division. In mitosis, DNA replication is always followed by cell division, yielding two

![li642239539b](li642239539b.jpg)

Somatic cell $(2 n)$
![li652239539b](li652239539b.jpg)

Daughter cells $(2 n)$

FIGURE 6-3 Comparison of mitosis and meiosis. Both somatic cells and pre-meiotic germ cells have two copies of each chromosome $(2 n)$, one maternal and one paternal. In mitosis, the replicated chromosomes, each composed of two sister chromatids, align at the cell center in such a way that both daughter cells receive a maternal and a paternal homolog of each morphological type of chromosome. During the first meiotic division, however, each replicated chromosome pairs with its homologous partner at the cell center; this pairing off is referred to as synapsis, and crossing over between homologous chromosomes is evident at this stage. One replicated
![li662239539b](li662239539b.jpg)
chromosome of each morphological type then goes into each daughter cell. The resulting cells undergo a second division without intervening DNA replication, so that one of the sister chromatids of each morphological type is apportioned to the daughter cells. In the second meiotic division, the alignment of chromatids and their equal segregation into daughter cells is the same as in mitotic division. The alignment of pairs of homologous chromosomes in metaphase I is random with respect to other chromosome pairs, resulting in a mix of paternally and maternally derived chromosomes in each daughter cell.

diploid daughter cells. In meiosis, one round of DNA replication is followed by two separate cell divisions, yielding haploid ( $1 n$ ) cells known as gametes that contain only one chromosome of each homologous pair. The apportionment, or segregation, of the replicated homologous chromosomes to daughter cells during the first meiotic division is random and different chromosomes segregate independently of one another, yielding gametes with different mixes of paternal and maternal chromosomes.

As a way to avoid unwanted complexity, geneticists usually strive to begin breeding experiments with strains of organisms that are homozygous for the genes under examination. In such true-breeding strains, every individual will receive the same allele from each parent, and therefore the composition of alleles will not change from one generation to the next. When a true-breeding mutant strain is mated to a true-breeding wild-type strain, all the first filial $\left(\mathrm{F}_{1}\right)$ progeny will be heterozygous (Figure 6-4). If the $\mathrm{F}_{1}$ progeny exhibit the mutant trait, then the mutant allele is dominant; if the $\mathrm{F}_{1}$ progeny exhibit the wild-type trait, then the mutant allele is recessive. Further crossing between $\mathrm{F}_{1}$ individuals will also reveal different patterns of inheritance according to whether the mutation is dominant or recessive. When $\mathrm{F}_{1}$ individuals that are heterozygous for a dominant allele are crossed among themselves, three-fourths of the resulting $\mathrm{F}_{2}$ progeny will exhibit the mutant trait. In contrast, when $\mathrm{F}_{1}$ individuals that are heterozygous for a recessive allele are crossed among themselves, only one-fourth of the resulting $\mathrm{F}_{2}$ progeny will exhibit the mutant trait.

As noted earlier, the yeast $S$. cerevisiae, an important experimental organism, can exist in either a haploid or a diploid state. In these unicellular eukaryotes, crosses between haploid cells can determine whether a mutant allele is dominant or recessive. Haploid yeast cells, which carry one copy of each chromosome, can be of two different mating types, known as a and $\alpha$. Haploid cells of opposite mating type can mate to produce a/ $\alpha$ diploids, which carry two copies of each chromosome. If a new mutation with an observable phenotype is isolated in a haploid strain, the mutant strain can be mated to a wild-type strain of the opposite mating type to produce a/ $\alpha$ diploids that are heterozygous for the mutant allele. If these diploids exhibit the mutant trait, then the mutant allele is dominant, but if the diploids exhibit the wild-type trait, then the mutant allele is recessive. When a/ $\alpha$ diploids are placed under starvation conditions, the cells undergo meiosis, each giving rise to a tetrad of four haploid spores, two of type a and two of type $\alpha$. Sporulation of a heterozygous diploid cell yields two spores carrying the mutant allele and two carrying the wild-type allele (Figure 6-5). Under appropriate conditions, yeast spores will germinate, producing vegetative haploid strains of both mating types.

## Conditional Mutations Can Be Used to Study Essential Genes in Yeast

The procedures used to identify and isolate mutants, referred to as genetic screens, depend on whether the experimental
(a) Segregation of dominant mutation
![li672239539b](li672239539b.jpg)
(b) Segregation of recessive mutation
![li682239539b](li682239539b.jpg)

FIGURE 6-4 Segregation patterns of dominant and recessive mutations in crosses between true-breeding strains of diploid organisms. All the offspring in the first $\left(F_{1}\right)$ generation are heterozygous. If the mutant allele is dominant, the $F_{1}$ offspring will exhibit the mutant phenotype, as in part (a). If the mutant allele is recessive, the $F_{1}$ offspring will exhibit the wild-type phenotype, as in part (b). Crossing of the $F_{1}$ heterozygotes among themselves also produces different segregation ratios for dominant and recessive mutant alleles in the $F_{2}$ generation.

![li692239539b](li692239539b.jpg)

FIGURE 6-5 Segregation of alleles in yeast. Haploid Saccharomyces cells of opposite mating type (i.e., one of mating type a and one of mating type $\alpha$ ) can mate to produce an a/ $\alpha$ diploid. If one haploid carries a dominant wild-type allele and the other carries a recessive mutant allele of the same gene, the resulting heterozygous diploid will express the dominant trait. Under certain conditions, a diploid cell will form a tetrad of four haploid spores. Two of the spores in the tetrad will express the recessive trait and two will express the dominant trait.
organism is haploid or diploid and, if the latter, whether the mutation is recessive or dominant. Genes that encode proteins essential for life are among the most interesting and important ones to study. Since phenotypic expression of a mutation in an essential gene leads to death of the individual, clever genetic screens are needed to isolate and maintain organisms with a lethal mutation.

In haploid yeast cells, essential genes can be studied through the use of conditional mutations. Among the most common conditional mutations are temperature-sensitive mutations, which are useful in organisms, such as bacteria and lower eukaryotes, that can grow at a range of temperatures. For instance, a single missense mutation may cause the resulting mutant protein to have reduced thermal stability, such that the protein is fully functional at one temperature (e.g., $23^{\circ} \mathrm{C}$ ) but begins to denature and is thus inactive at another temperature (e.g., $36^{\circ} \mathrm{C}$ ), whereas the normal protein would be fully stable and functional at both temperatures. A temperature at which the mutant phenotype is observed is called nonpermissive; a permissive temperature is one at which the mutant phenotype is not observed even though the mutant allele is present. Thus mutant strains can be maintained at a permissive temperature and then subcultured at a nonpermissive temperature for analysis of the mutant phenotype.

An example of a particularly important screen for temperature-sensitive mutants in the yeast S. cerevisiae comes from the studies of L. H. Hartwell and colleagues in the late 1960s and early 1970s. They set out to identify genes important in regulation of the cell cycle (during which a cell synthesizes proteins, replicates its DNA, and then undergoes mitotic cell division). Exponential growth of a single yeast
cell for 20-30 cell divisions forms a visible yeast colony on solid agar medium. Because mutants with a complete block in the cell cycle would not be able to form colonies, conditional mutants were required to study mutations that affect this basic cellular process. To screen for such mutants, the researchers first exposed yeast cells to mutagens and then identified mutant yeast cells that could grow normally at $23^{\circ} \mathrm{C}$, but could not form a colony when placed at $36^{\circ} \mathrm{C}$ (Figure 6-6a).

Once temperature-sensitive mutants were isolated, further analysis revealed that some indeed were defective in cell division. In S. cerevisiae, cell division occurs through a budding process, and the size of the bud, which is easily visualized by light microscopy, indicates a cell's position in the cell cycle. Each of the mutants that could not grow at $36^{\circ} \mathrm{C}$ was examined by microscopy after several hours at the nonpermissive temperature. Examination of many different temper-ature-sensitive mutants revealed a set that exhibited a distinct block in the cell cycle. These mutants were therefore designated $c d c$ (cell division cycle) mutants. Importantly, these yeast mutants did not simply fail to grow, as they might have if they carried a mutation affecting general cellular metabolism. Rather, at the nonpermissive temperature, the mutants grew normally for part of the cell cycle, but then arrested at a particular stage of the cell cycle, so that many cells at that stage were seen (Figure 6-6b). Most cdc mutations in yeast are recessive; that is, when haploid cdc strains are mated to wild-type haploids, the resulting heterozygous diploids are neither temperature sensitive nor defective in cell division.

## Recessive Lethal Mutations in Diploids Can Be Identified by Inbreeding and Maintained in Heterozygotes

In diploid organisms, phenotypes resulting from recessive mutations can be observed only in individuals that are homozygous for the mutant alleles. Since mutagenesis in a diploid organism typically changes only one allele of a gene, yielding heterozygous mutants, genetic screens must include inbreeding steps to generate progeny that are homozygous for the mutant allele. The geneticist H. Muller developed a general and efficient procedure for carrying out such inbreeding experiments in the fruit fly Drosophila. By using such procedures, recessive lethal mutations in Drosophila and other diploid organisms can be maintained in heterozygous individuals and their phenotypic consequences analyzed in homozygotes.

The Muller approach was used to great effect by C. Nüsslein-Volhard and E. Wieschaus, who systematically screened for recessive lethal mutations affecting embryogenesis in Drosophila. Dead homozygous embryos carrying recessive lethal mutations identified by this screen were examined under the microscope for specific morphological defects. Current understanding of the molecular mechanisms underlying the development of multicellular organisms is based, in large part, on the detailed picture of embryonic development revealed by characterization of these Drosophila mutants.

![li702239539b](li702239539b.jpg)
(b)

Wild type
![li712239539b](li712239539b.jpg)
cdc28 mutants
![li722239539b](li722239539b.jpg)
$c d c 7$ mutants
![li732239539b](li732239539b.jpg)

EXPERIMENTAL FIGURE 6-6 Haploid yeast cells carrying temperature-sensitive lethal mutations can be maintained at permissive temperature and analyzed at nonpermissive temperature. (a) Genetic screen for temperature-sensitive cell division cycle (cdc) mutants in S. cerevisiae. Yeast cells that grow and form colonies at $23^{\circ} \mathrm{C}$ (permissive temperature) but not at $36^{\circ} \mathrm{C}$ (nonpermissive temperature) may carry a lethal mutation that blocks cell division. See L. H. Hartwell, 1967, J. Bacteriol. 93:1662. (b) Assay of temperature-sensitive colonies for blocks at specific stages in the cell cycle. Shown here are micrographs of wild-type yeast and two different temperature-sensitive mutants after incubation at the nonpermissive temperature for 6 hours. Wild-type cells, which continue to grow, can be seen with all different sizes of buds, reflecting different stages of the cell cycle. In contrast, cells in the lower two micrographs exhibit a block at a specific stage in the cell cycle. The $c d c 28$ mutants arrest at a point before emergence of a new bud and therefore appear as unbudded cells. The $c d c 7$ mutants, which arrest just before separation of the mother cell and bud (emerging daughter cell), appear as cells with large buds. (Part (b) republished with permission of Elsevier, from Herefor, L. M, and Hartwell, L. H., "Sequential gene function in the initiation of Saccharomyces cerevisiae DNA synthesis," Journal of Molecular Biology, 1974, 84:3, pps 445-456; permission conveyed through Copyright Clearance Center, Inc.)

## Complementation Tests Determine Whether Different Recessive Mutations Are in the Same Gene

Researchers using the classical genetic approach to studying a particular cellular process often isolate multiple recessive mutations that produce the same phenotype. A common test for determining whether these mutations are in the same gene or in different genes exploits the phenomenon of genetic complementation; that is, restoration of the wild-type phenotype by mating two different mutants. If two recessive mutations, $a$ and $b$, are in the same gene, then a diploid organism carrying one $a$ allele and one $b$ allele will exhibit the mutant phenotype because neither allele provides a functional copy of the gene. In contrast, if mutations $a$ and $b$ are in separate genes, then heterozygotes carrying a single copy of each mutant allele will not exhibit the mutant phenotype because a wild-type allele of each gene is also present. In this case, the mutations are said to complement each other. Complementation analysis cannot be performed on dominant mutations because the phenotype conferred by the mutant allele is displayed even in the presence of a wild-type allele of the gene.

Complementation analysis of a set of mutants exhibiting the same phenotype can distinguish the individual genes in a set of functionally related genes, all of which must function to produce a given phenotypic trait. For example, the screen for cdc mutations in Saccharomyces described previously yielded many recessive temperature-sensitive mutants that appeared to be arrested at the same cell cycle stage. To determine how many genes were affected by these mutations, Hartwell and his colleagues performed complementation

tests on all of the pair-wise combinations of their cdc mutants, following the general protocol outlined in Figure 6-7. These tests organized more than 100 cdc mutations into about 20 different $C D C$ genes. The subsequent molecular characterization of the $C D C$ genes and their encoded proteins, as described in detail in Chapter 19, has provided a framework for understanding how cell division is regulated in organisms ranging from yeast to humans.

## Double Mutants Are Useful in Assessing the Order in Which Proteins Function

By careful analysis of mutant phenotypes associated with a particular cellular process, researchers can often deduce the order in which a set of genes and their protein products function. Two general types of processes are amenable to such analysis: (1) biosynthetic pathways in which a precursor material is converted via one or more intermediates to a final product, and (2) signaling pathways that regulate other processes and involve the flow of information rather than chemical intermediates.

Ordering of Biosynthetic Pathways A simple example of the first type of process is the biosynthesis of a metabolite such as the amino acid tryptophan in bacteria. In this case, each of the enzymes required for synthesis of tryptophan catalyzes the conversion of one of the intermediates in the biosynthetic pathway to the next. In E. coli, the genes encoding these enzymes lie adjacent to one another in the genome, constituting the trp operon (see Figure 5-13). The order of action of the different genes for these enzymes, and hence the order of the biochemical reactions in the pathway, was initially deduced from the types of intermediate compounds that accumulated in each mutant. In the case of complex synthetic pathways, however, phenotypic analysis of mutants defective in a single step may give ambiguous results that do not permit conclusive ordering of the steps. Double mutants defective in two steps in the pathway are particularly useful in ordering such pathways (Figure 6-8a).

In Chapter 14, we discuss the classic use of the doublemutant strategy to help elucidate the secretory pathway. In this pathway, proteins to be secreted from the cell move from their site of synthesis on the rough endoplasmic reticulum

## EXPERIMENTAL FIGURE 6-7 Complementation analysis determines whether recessive mutations are in the same or different genes. Complementation tests in yeast are performed by mating haploid $\mathbf{a}$ and $\alpha$ cells carrying different recessive mutations to produce diploid cells. In the analysis of cdc mutations, pairs of different haploid temperature-sensitive cdc strains were systematically mated and the resulting diploids tested for growth at the permissive and nonpermissive temperatures. In this hypothetical example, the $c d c X$ and $c d c Y$ mutants complement each other, and thus have mutations in different genes, whereas the $c d c X$ and $c d c Z$ mutants have mutations in the same gene.

![li742239539b](li742239539b.jpg)

(a) Analysis of a biosynthetic pathway

A mutation in A accumulates intermediate 1.
A mutation in B accumulates intermediate 2.

| PHENOTYPE OF | A double mutation in A and B accumulates |
| :-- | :-- |
| DOUBLE MUTANT: | intermediate 1. |

INTERPRETATION: The reaction catalyzed by $A$ precedes the reaction catalyzed by $B$.

| 1 | A | B | 3 |
| :-- | :-- | :-- | :-- | :-- | :-- | :-- | :-- |

(b) Analysis of a signaling pathway

A mutation in A gives repressed reporter expression.
A mutation in B gives constitutive reporter expression.

| PHENOTYPE OF |  |
| :--: | :--: |
| DOUBLE MUTANT: | A double mutation in A and B gives repressed reporter expression. |
| INTERPRETATION: | A positively regulates reporter expression and is negatively regulated by $B$. |
|  |  |

PHENOTYPE OF
DOUBLE MUTANT: A double mutation in $A$ and $B$ gives constitutive reporter expression.
INTERPRETATION: B negatively regulates reporter expression and is negatively regulated by $A$.
![li752239539b](li752239539b.jpg)

FIGURE 6-8 Analysis of double mutants can often order the steps in biosynthetic or signaling pathways. When mutations in two different genes affect the same cellular process but produce distinctly different phenotypes, the phenotype of the double mutant can often reveal the order in which the two genes must function. (a) In the case of mutations that affect the same biosynthetic pathway, a double mutant will accumulate the intermediate immediately preceding the step catalyzed by the protein that acts earlier in the wild-type organism. (b) Double-mutant analysis of a signaling pathway is possible if two mutations have opposite effects on expression of a reporter gene. In this case, the observed phenotype of the double mutant provides information about the order in which the proteins act and whether they are positive or negative regulators.
(ER) to the Golgi complex, then to secretory vesicles, and finally to the cell surface.

Ordering of Signaling Pathways As we will learn in later chapters, the expression of many eukaryotic genes is regulated by signaling pathways that are initiated by extracellular hormones, growth factors, or other signals. Such signaling pathways may include numerous components, and doublemutant analysis can often provide insight into the functions and interactions of these components. The only prerequisite for obtaining useful information from this type of analysis is that the two mutations must have very different, or even
opposite, effects on the output of the same regulated pathway as measured by expression of a reporter gene. Most commonly, one mutation represses expression of a particular reporter gene even when the signal is present, while another mutation results in reporter gene expression even when the signal is absent (i.e., constitutive expression). As illustrated in Figure 6-8b, two simple regulatory mechanisms are consistent with such single mutants, but the double-mutant phenotype can distinguish between them. This general approach has enabled geneticists to delineate many of the key steps in a variety of different regulatory pathways, setting the stage for more specific biochemical assays.

Note that this technique differs from the complementation analysis just described in that both dominant and recessive mutants can be subjected to double-mutant analysis. When two recessive mutations are tested, the double mutant created must be homozygous for both mutations.

## Genetic Suppression and Synthetic Lethality Can Reveal Interacting or Redundant Proteins

Two other types of genetic analysis can provide additional clues about how proteins that function in the same cellular process may interact with one another in the living cell. Both of these methods, which are applicable in many experimental organisms, involve the use of double mutants in which the phenotypic effects of one mutation are changed by the presence of a second mutation.

Suppressor Mutations The first type of analysis is based on genetic suppression. To understand this phenomenon, suppose that point mutations lead to structural changes in one protein (A) that disrupt its ability to associate with another protein (B) involved in the same cellular process. Similarly, mutations in protein B lead to small structural changes that inhibit its ability to interact with protein A. Assume, furthermore, that the normal functioning of proteins A and B depends on their interacting. In theory, a specific structural change in protein A might be suppressed by compensatory changes in protein B, allowing the mutant proteins to interact. In the rare cases in which such suppressor mutations occurred, strains carrying both mutant alleles would be normal, whereas strains carrying only one or the other mutant allele would have a mutant phenotype (Figure 6-9a).

The observation of genetic suppression in yeast strains carrying a mutant actin allele (act1-1) and a second mutation in another gene (sac6) provided early evidence for a direct interaction in vivo between the proteins encoded by the two genes. Later biochemical studies showed that these two proteins-Act1 and Sac6-do indeed interact in the construction of functional actin structures within the cell.

Synthetic Lethal Mutations Another phenomenon, called synthetic lethality, produces a phenotypic effect opposite to that of suppression. In this case, the deleterious effect of one mutation is greatly exacerbated (rather than suppressed) by a second mutation in a related gene. One situation in which

(a) Suppression
![li762239539b](li762239539b.jpg)
(b) Synthetic lethality 1
![li772239539b](li772239539b.jpg)
(c) Synthetic lethality 2
![li782239539b](li782239539b.jpg)

FIGURE 6-9 Mutations that result in genetic suppression or synthetic lethality reveal interacting or redundant proteins. (a) The observation that double mutants with two defective proteins (A and B) have a wild-type phenotype but that single mutants have a mutant phenotype indicates that the function of each protein depends on interaction with the other. (b) The observation that double mutants have a more severe phenotypic defect than single mutants is also evidence that two proteins (e.g., subunits of a heterodimer) must interact to function normally. (c) The observation that a double mutant is nonviable but that the corresponding single mutants have the wild-type phenotype indicates that two proteins function in redundant pathways to produce an essential product.
such synthetic lethal mutations can occur is illustrated in Figure 6-9b. In this example, a heterodimeric protein is partially, but not completely, inactivated by mutations in either one of its nonidentical subunits. However, in double mutants carrying specific mutations in the genes encoding both subunits, little interaction between subunits occurs, resulting in severe phenotypic effects. Synthetic lethal mutations can also reveal nonessential genes whose encoded proteins function in redundant pathways for producing an essential cell component. As depicted in Figure 6-9c, if either pathway alone is inactivated by a mutation, the other pathway will be able to supply the needed product. However, if both pathways are inactivated at the same time, the essential product cannot be synthesized, and the double mutants will be nonviable.

## Genes Can Be Identified by Their Map Position on the Chromosome

We will now consider a fundamentally different type of genetic analysis based on gene position. Studies designed to determine the position of a gene on a chromosome, often referred to as genetic mapping studies, can be used to identify the gene affected by a particular mutation or to determine whether two mutations are in the same gene.

In many organisms, genetic mapping studies rely on exchanges of genetic information that occur during meiosis. As shown in Figure 6-10a, genetic recombination takes place in germ cells after the chromosomes of each homologous pair have replicated, but before the first meiotic cell division. At this time, homologous DNA sequences on maternally and paternally derived chromatids can be exchanged with each other in a process known as crossing over. We now know that the resulting crossovers between homologous chromosomes provide structural links that are important for the proper segregation of pairs of homologous chromatids to opposite poles during the first meiotic cell division (for further discussion, see Chapter 19).

Consider an individual with two different mutations, one inherited from each parent, that are located close to each other on the same chromosome. That individual can produce two different types of gametes according to whether a crossover occurs between the mutations during meiosis. If no crossover occurs between them, gametes known as parental types, which contain either one or the other mutation, will be produced. In contrast, if a crossover occurs between the two mutations, gametes known as recombinant types will be produced. In this example, recombinant chromosomes would contain either both mutations or neither of them. Recombination events occur more or less at random along the length of chromosomes; thus the closer together two genes are, the less likely that recombination will happen to occur between them during meiosis. Thus the less frequently recombination is observed to occur between two genes on the same chromosome, the closer together they must be. Two genes that are on the same chromosome and that are sufficiently close together that significantly fewer recombinant gametes than parental gametes are produced are considered to exhibit genetic linkage. If the number of recombinant gametes produced is not significantly less than the number of parental gametes, the two loci under consideration are considered to be unlinked and could be far apart on the same chromosome, or they could be on different chromosomes.

The technique of recombination mapping was devised in 1911 by A. Sturtevant while he was an undergraduate working in the laboratory of T. H. Morgan at Columbia University. Originally used in studies on Drosophila, this technique is still used today to assess the distance between two genetic loci on the same chromosome in many experimental organisms. A typical experiment designed to determine the map distance between two genetic positions involves two steps. In the first step, a strain is constructed that carries a different mutation at each of two positions, or loci. In the second step, the progeny of this strain are assessed to determine

![li792239539b](li792239539b.jpg)

FIGURE 6-10 Recombination during meiosis can be used to map the positions of genes. (a) Consider the gametes produced by an individual that carries two mutations, designated $m 1$ (yellow) and $m 2$ (green), that are on the maternal and paternal versions of the same chromosome, respectively. If crossing over occurs at an interval between $m 1$ and $m 2$ before the first meiotic division, then two types of recombinant gametes are produced; one carries both $m 1$ and $m 2$, whereas the other carries neither mutation. The longer the distance between two
the relative frequency of inheritance of parental or recombinant types. A typical way to determine the frequency of recombination between two genes is to cross one of these heterozygous progeny with another individual that is homozygous for each gene. For such a cross, the proportion of recombinant progeny is readily determined because recombinant phenotypes will differ from the parental phenotypes. By convention, one genetic map unit is defined as the distance between two positions along a chromosome that results in 1 recombinant individual in 100 total progeny. The distance corresponding to this 1 percent recombination frequency is called a centimorgan (cM) in honor of Sturtevant's mentor, Morgan (Figure 6-10b).

A complete discussion of the methods of genetic mapping is beyond the scope of this introductory discussion; however, two features of recombination mapping need particular emphasis. First, the frequency of genetic exchange between two loci is proportional to the physical distance in base pairs separating them only for loci that are relatively close together (say, less than about 10 cM ). For linked loci that are farther apart than this, a distance measured by the frequency of genetic exchange tends to underestimate the physical distance because of the possibility of two or more crossovers occurring within an interval.
(b) Consider two linked genes $A$ and $B$ with recessive alleles $a$ and $b$. Cross of two mutants to construct a doubly heterozygous strain:
![li802239539b](li802239539b.jpg)

Cross of double heterozygote to test strain:
![li812239539b](li812239539b.jpg)

Genetic distance between $A$ and $B$ can be determined from frequency of parental and recombinant gametes:

Genetic distance in $\mathrm{cM}=100 \times \frac{\text { recombinant gametes }}{\text { total gametes }}$
mutations on a chromatid, the more likely they are to be separated by recombination, and the greater the proportion of recombinant gametes produced. (b) In a typical mapping experiment, a strain that is heterozygous for two different genes is constructed. The frequency of parental or recombinant gametes produced by this strain can be determined from the phenotypes of the progeny in a testcross to a homozygous recessive strain. The genetic map distance in centimorgans (cM) is given as the percentage of the gametes that are recombinant.

A second important concept needed for interpreting genetic mapping experiments in different types of organisms is that although genetic distance is defined in the same way for different organisms, the relationship between recombination frequency (i.e., genetic map distance) and physical distance varies between organisms. For example, a 1 percent recombination frequency (i.e., a genetic distance of 1 cM ) represents a physical distance of about 2.8 kb in yeast, compared with a distance of about 400 kb in Drosophila and about 780 kb in humans.

One of the chief uses of genetic mapping studies is to locate the gene that is affected by a mutation of interest. The presence of many different already mapped genetic traits, or genetic markers, distributed along the length of a chromosome permits the position of an unmapped mutation to be determined by assessing its segregation with respect to these marker genes during meiosis. Thus the more markers that are available, the more precisely a mutation can be mapped. In Section 6.4, we will see how the genes affected in inherited human diseases can be identified using such methods. A second general use of mapping experiments is to determine whether two different mutations are in the same gene. If two mutations are in the same gene, they will exhibit tight linkage in mapping experiments, but if they are in different genes, they will usually be unlinked or exhibit weak linkage.

## KEY CONCEPTS OF SECTION 6.1

## Genetic Analysis of Mutations to Identify and Study Genes

- Diploid organisms carry two copies (alleles) of each gene, whereas haploid organisms carry only one copy.
- Recessive mutations often lead to a loss of function, which is masked if a wild-type allele of the gene is present. For the mutant phenotype to be observed, both alleles must carry the mutation.
- Dominant mutations lead to a mutant phenotype in the presence of a wild-type allele of the gene. The phenotypes associated with dominant mutations often represent a gain of function.
- In meiosis, a diploid cell undergoes one DNA replication and two cell divisions, yielding four haploid cells in which maternal and paternal chromosomes and their associated alleles are randomly assorted (see Figure 6-3).
- Dominant and recessive mutations exhibit characteristic segregation patterns in genetic crosses (see Figure 6-4).
- In haploid yeast, temperature-sensitive mutations are particularly useful for identifying and studying genes essential to survival.
- The number of functionally related genes involved in a process can be defined by complementation analysis (see Figure 6-7).
- The order in which genes function in a biosynthetic or signaling pathway can be deduced from the phenotype of double mutants defective in two steps in the affected process.
- Functionally significant interactions between proteins can be deduced from the phenotypic effects of allele-specific suppressor mutations or synthetic lethal mutations.
- Genetic mapping experiments make use of crossing over between homologous chromosomes during meiosis to measure the genetic distance between two different mutations on the same chromosome.


### 6.2 DNA Cloning and Characterization

Detailed studies of the structure and function of a gene at the molecular level require large quantities of the individual gene in pure form. A variety of techniques, often referred to as recombinant DNA technology, are used in DNA cloning, which permits researchers to prepare large numbers of identical DNA molecules. Recombinant DNA is simply any DNA molecule composed of sequences derived from different sources.

The key to cloning a DNA fragment of interest is to link it to a vector DNA molecule that can replicate within a host cell. After a single recombinant DNA molecule, composed of a vector plus an inserted DNA fragment, is introduced into a host cell, the inserted DNA is replicated along with
the vector, generating a large number of identical DNA molecules. The basic scheme can be summarized as follows:

$$
\begin{gathered}
\text { Vector }+ \text { DNA fragment } \\
\downarrow \\
\text { Recombinant DNA } \\
\downarrow
\end{gathered}
$$

Replication of recombinant DNA within host cells
Isolation, sequencing, and manipulation of purified DNA fragment

Although investigators have devised numerous experimental variations, this flow diagram indicates the essential steps in DNA cloning.

In this section, we first describe methods for isolating a specific sequence of DNA from a sea of other DNA sequences. This process often involves cutting the genome into fragments and then placing each fragment in a vector so that the entire collection can be propagated as recombinant molecules in separate host cells. While many different types of vectors exist, our discussion will mainly focus on plasmid vectors in E. coli host cells, which are commonly used. Various techniques can then be employed to identify the sequence of interest from this collection of DNA fragments. Once a specific DNA fragment is isolated, the exact sequence of nucleotides in the fragment can be determined. We end with a discussion of the polymerase chain reaction (PCR). This powerful and versatile technique can be used in many ways to generate large quantities of a specific sequence and to manipulate DNA in the laboratory. The various uses of cloned DNA fragments are discussed in subsequent sections.

## Restriction Enzymes and DNA Ligases Allow Insertion of DNA Fragments into Cloning Vectors

A major objective of DNA cloning is to obtain discrete, small regions of an organism's DNA that constitute specific genes. In addition, only relatively small DNA molecules can be inserted into any of the available vectors. For these reasons, the very long DNA molecules that compose an organism's genome must be cleaved into fragments that can be inserted into the vector DNA. Two types of enzymes-restriction enzymes and DNA ligases-facilitate production of such recombinant DNA molecules.

Cutting DNA Molecules into Small Fragments Restriction enzymes are endonucleases produced by bacteria that typically recognize specific $4-8$-bp sequences, called restriction sites, and cleave both DNA strands at these sites. Restriction sites commonly are short palindromic sequences; that is, the restriction-site sequence is the same on each DNA strand when read in the $5^{\prime}$ to $3^{\prime}$ direction (Figure 6-11).

For each restriction enzyme, bacteria also produce a modification enzyme, which protects a host bacterium's own DNA from cleavage by modifying the host DNA at or near each potential cleavage site. The modification

![li822239539b](li822239539b.jpg)

FIGURE 6-11 Cleavage of DNA by the restriction enzyme EcoRI.
This restriction enzyme from E. coli makes staggered cuts at the specific 6 -bp palindromic sequence shown, yielding fragments with singlestranded, complementary 4-base "sticky" ends. Many other restriction enzymes also produce fragments with sticky ends.
enzyme adds a methyl group to one or two bases, usually within the restriction site. When a methyl group is present there, the restriction endonuclease is prevented from cutting the DNA. Together with the restriction endonuclease, the modification enzyme forms a restriction-modification system that protects the host DNA while it destroys incoming foreign DNA (e.g., bacteriophage DNA or DNA taken
up during transformation) by cleaving it at all the available restriction sites.

Many restriction enzymes make staggered cuts in the two DNA strands at the corresponding restriction site, generating fragments that have a single-stranded "tail" at both ends (see Figure 6-11). The tails on the fragments generated at a given restriction site are complementary to those on all other fragments generated by the same restriction enzyme. At room temperature, these sticky ends can transiently base-pair with those on other DNA fragments generated with the same restriction enzyme.

The DNA isolated from an individual organism has a specific sequence that, purely by chance, contains a specific set of restriction sites. Thus a given restriction enzyme will cut the DNA from a particular source into a reproducible set of fragments called restriction fragments. The frequency with which a restriction enzyme cuts DNA, and thus the average size of the resulting restriction fragments, depends largely on the length of the recognition site. For example, a restriction enzyme that recognizes a 4 -bp site will cleave DNA an average of once every $4^{4}$, or 256 , base pairs, whereas an enzyme that recognizes an 8 -bp sequence will cleave DNA an average of once every $4^{8}$ base pairs $(65 \mathrm{~kb})$. The hundreds of different restriction enzymes that have been identified from different species of bacteria allow DNA molecules to be cut at a large number of different sequences corresponding to the recognition sites of these enzymes (Table 6-1).

TABLE 6-1 Selected Restriction Enzymes and Their Recognition Sequences

| Enzyme | Source Microorganism | Recognition Site* | Ends Produced |
| :--: | :--: | :--: | :--: |
| BamHI | Bacillus amyloliquefaciens | $\downarrow$ <br> -G-G-A-T-C-C- <br> -C-C-T-A-G-G- <br> $\uparrow$ | Sticky |
| Sau3AI | Staphylococcus aureus | $\begin{aligned} & \downarrow \\ & -\mathrm{G}-\mathrm{A}-\mathrm{T}-\mathrm{C}- \\ & -\mathrm{C}-\mathrm{T}-\mathrm{A}-\mathrm{G}- \\ & \uparrow \end{aligned}$ | Sticky |
| EcoRI | Escherichia coli | $\begin{gathered} \downarrow \\ \text {-G-A-A-T-T-C- } \\ \text {-C-T-T-A-A-G } \\ \uparrow \end{gathered}$ | Sticky |
| HindIII | Haemophilus influenzae | $\begin{gathered} \downarrow \\ \text {-A-A-G-C-T-T- } \\ \text {-T-T-C-G-A-A- } \\ \uparrow \end{gathered}$ | Sticky |
| SmaI | Serratia marcescens | $\begin{gathered} \downarrow \\ \text {-C-C-C-G-G-G- } \\ \text {-G-G-G-C-C-C- } \\ \uparrow \end{gathered}$ | Blunt |
| NotI | Nocardia otitidis-caviarum | $\begin{gathered} \downarrow \\ \text {-G-C-G-G-C-C-G-C- } \\ \text {-C-G-C-C-G-G-C-G- } \\ \uparrow \end{gathered}$ | Sticky |

[^0]
[^0]:    *Many of these recognition sequences are included in a common polylinker sequence (see Figure 6-13).

Inserting DNA Fragments into Vectors DNA fragments with either sticky ends or blunt ends can be inserted into vector DNA with the aid of DNA ligase. During normal DNA replication, DNA ligase catalyzes the end-to-end joining (ligation) of short fragments of DNA. For purposes of DNA cloning, purified DNA ligase is used to covalently join the ends of a restriction fragment and vector DNA that have complementary ends (Figure 6-12). The vector DNA and restriction fragment are covalently ligated together through the standard phosphodiester bonds of DNA. In addition to ligating complementary sticky ends, the DNA ligase from bacteriophage T4 can ligate any two blunt DNA ends. However, blunt-end ligation is inherently inefficient and requires a higher concentration of both DNA and DNA ligase than does ligation of sticky ends.

## Isolated DNA Fragments Can Be Cloned into E. coli Plasmid Vectors

Plasmids are circular, double-stranded DNA (dsDNA) molecules that replicate separately from a cell's chromosomal
![li832239539b](li832239539b.jpg)

FIGURE 6-12 Ligation of restriction fragments with complementary sticky ends. In this example, vector DNA cut with EcoRI is mixed with a sample containing restriction fragments produced by cleaving genomic DNA with several different restriction enzymes. The short base sequences composing the sticky ends of each fragment type are shown. The sticky end on the cut vector DNA ( $a^{\prime}$ ) base-pairs only with the complementary sticky ends on the EcoRI fragment (a) in the genomic sample. The adjacent $3^{\prime}$ hydroxyl and $5^{\prime}$ phosphate groups (red) on the base-paired fragments are then covalently joined (ligated) by T4 DNA ligase.

DNA. These extrachromosomal DNAs, which occur naturally in bacteria and in lower eukaryotic cells (e.g., yeast), exist in a symbiotic relationship with their host cell. Like the host-cell chromosomal DNA, plasmid DNA is duplicated before every cell division. During cell division, copies of the plasmid DNA segregate to each daughter cell, ensuring continued propagation of the plasmid through successive generations of the host cell.

The plasmids most commonly used in recombinant DNA technology are those that replicate in E. coli. Investigators have engineered these plasmids to optimize their use as vectors in DNA cloning. For instance, removal of unneeded portions from naturally occurring E. coli plasmids yields plasmid vectors about $1.2-3 \mathrm{~kb}$ in circumferential length that contain three regions essential for DNA cloning: a replication origin (ORI); a marker that permits selection, usually a drug-resistance gene; and a region in which exogenous DNA fragments can be inserted (Figure 6-13).

Figure 6-14 outlines the general procedure for cloning a DNA fragment using E. coli plasmid vectors. When E. coli cells are mixed with recombinant vector DNA and subjected to a stress such as heat shock, a small fraction of the cells will take up the plasmid DNA, a process known as transformation. Typically, 1 cell in about 10,000 incorporates a single plasmid DNA molecule and thus becomes transformed. The rare transformed cells can be easily selected by use of a selectable marker. For instance, if the plasmid carries a gene that confers resistance to the antibiotic ampicillin, transformed cells can be selected by growing them in an ampicillin-containing medium. All the antibiotic-resistant progeny cells that arise from the initial transformed cell will contain plasmids with the same inserted DNA. Since all the cells in a colony arise from a single transformed parent cell,
![li842239539b](li842239539b.jpg)

FIGURE 6-13 Basic components of a plasmid cloning vector that can replicate within an E. coli cell. Plasmid vectors contain a selectable gene, such as amp ${ }^{\prime}$, which encodes the enzyme $\beta$-lactamase and confers resistance to ampicillin. Exogenous DNA can be inserted into the bracketed region without disturbing the ability of the plasmid to replicate or express the amp ${ }^{\prime}$ gene. Plasmid vectors also contain a replication origin (ORI) sequence where DNA replication can be initiated by host-cell enzymes. Inclusion of a synthetic polylinker containing the recognition sequences for several different restriction enzymes increases the versatility of a plasmid vector. The vector is designed so that each site in the polylinker is unique on the plasmid.

they constitute a clone of cells, and the initial fragment of DNA inserted into the parental plasmid is referred to as cloned DNA or a DNA clone.

The versatility of an E. coli plasmid vector is increased by the addition of a polylinker, a synthetically generated sequence containing one copy of each of several different restriction sites that are not present elsewhere in the plasmid sequence (see Figure 6-13). Typically, the polylinker is cleaved with two different restriction enzymes in preparation to accept a DNA fragment prepared with two different sticky ends corresponding to the same two enzymes. Such a strategy eliminates unwanted by-products such as reclosing
![li852239539b](li852239539b.jpg)
of the cleaved plasmid vector and greatly increases the efficiency of cloning DNA fragments.

Plasmid cloning vectors are useful for propagating DNA fragments up to about 20 kb in length, but fragments longer than this cannot be reliably replicated within one celldivision cycle. For some purposes, such as the isolation and manipulation of large segments of the human genome, it is desirable to clone DNA segments as large as several megabases [ 1 megabase $(\mathrm{Mb})=1$ million base pairs]. For this purpose, specialized plasmid vectors known as BACs (bacterial artificial chromosomes) have been developed. One type of BAC uses a replication origin derived from an endogenous plasmid of E. coli known as the F factor. The F factor, and cloning vectors derived from it, can be stably maintained at a single copy per E. coli cell even when they contain inserted sequences of up to about 2 Mb . Production of BAC libraries requires special methods for the isolation, ligation, and transformation of large segments of DNA because segments of DNA larger than about 20 kb are highly vulnerable to mechanical breakage even by standard manipulations such as pipetting.

## Yeast Genomic Libraries Can Be Constructed with Shuttle Vectors and Screened by Functional Complementation

A collection of DNA molecules each cloned into a vector molecule is known as a DNA library. When genomic DNA from a particular organism is the source of the starting DNA, the set of clones that collectively represent all the DNA sequences in the genome is known as a genomic library. In some cases, a DNA library can be screened for the ability to express a functional protein that complements a recessive mutation. Such a screening strategy would be an efficient way to isolate a cloned gene that corresponds to an interesting recessive mutation identified in an experimental organism. To illustrate this method, referred to as functional complementation, we describe how yeast genes cloned in special E. coli plasmids can be introduced into mutant yeast cells to identify the wild-type gene that is defective in the mutant strain. Because Saccharomyces genes do not contain multiple introns, they are sufficiently compact that the entire sequence of as many as 10 genes can be included in a genomic DNA fragment inserted into a plasmid vector.

To construct a plasmid genomic library that is to be screened by functional complementation in yeast cells,

## EXPERIMENTAL FIGURE 6-14 DNA cloning in a plasmid

vector permits amplification of a DNA fragment. A fragment of DNA to be cloned is first inserted into a plasmid vector containing an ampicillin-resistance gene ( $a m p^{\dagger}$ ), such as that shown in Figure 6-13. Only the few cells transformed by incorporation of a plasmid will survive on ampicillin-containing medium. In transformed cells, the plasmid DNA replicates and segregates into daughter cells, resulting in the formation of an ampicillin-resistant colony in which each cell contains the cloned DNA.

the plasmid vector must be capable of replication in both E. coli cells and yeast cells. This type of vector, capable of propagation in two different hosts, is called a shuttle vector. The structure of a typical yeast shuttle vector is shown in Figure 6-15a. This vector contains the basic elements that permit cloning of DNA fragments in E. coli as well as sequences required for its propagation in yeast.

To increase the probability that all regions of the yeast genome will be successfully cloned and represented in the plasmid library, the genomic DNA is usually only partially digested to yield overlapping restriction fragments of $\sim 10 \mathrm{~kb}$. These fragments are then ligated into a shuttle
![li862239539b](li862239539b.jpg)

Assay yeast genomic library by functional complementation
vector in which the polylinker has been cleaved with a restriction enzyme that produces sticky ends complementary to those on the yeast DNA fragments (Figure 6-15b). Because the $10-\mathrm{kb}$ restriction fragments of yeast DNA are incorporated into the shuttle vectors randomly, at least $10^{3}$ E. coli colonies, each containing a particular recombinant shuttle vector, are necessary to ensure that each region of yeast DNA has a high probability of being represented in the library at least once.

Figure 6-16 outlines how such a yeast genomic library can be screened to isolate the wild-type gene corresponding to one of the temperature-sensitive cdc mutations mentioned earlier in this chapter. The starting yeast strain is a double mutant that requires uracil for growth due to a $u r a 3^{-}$mutation and is temperature sensitive due to a $c d c 28$ mutation identified by its phenotype (see Figure 6-6b). Recombinant plasmids isolated from the yeast genomic library are mixed with yeast cells under conditions that promote transformation of the cells with foreign DNA. Since transformed yeast cells carry a plasmid-borne copy of the wild-type URA3 gene, they can be selected by their ability to grow in the absence of uracil. Typically, about 20 petri dishes, each containing about 500 yeast transformants, are sufficient to represent the entire yeast genome. This collection of yeast transformants can be maintained at $23^{\circ} \mathrm{C}$, a temperature permissive for growth of the $c d c 28$ mutant. The entire collection on 20 plates is then transferred to replica plates, which are maintained at $36^{\circ} \mathrm{C}$, a nonpermissive temperature for cdc mutants. Yeast colonies that carry recombinant plasmids expressing a wild-type copy of the CDC28 gene will be able to grow at $36^{\circ} \mathrm{C}$. Once temperature-resistant yeast colonies have been identified, plasmid DNA can be extracted from the cultured yeast cells and analyzed by DNA sequencing, a topic we take up shortly.

## cDNA Libraries Represent the Sequences of Protein-Coding Genes

Genomic libraries are ideal for representing the genetic content of relatively simple organisms such as bacteria or yeast,

## EXPERIMENTAL FIGURE 6-15 A yeast genomic library can be constructed in a plasmid shuttle vector that can replicate in

yeast and in E. coli. (a) Components of a typical plasmid shuttle vector for cloning Saccharomyces genes. The presence of a yeast replication origin (ARS) and a yeast centromere (CEN) allows stable replication and segregation in yeast. Also included is a yeast selectable marker such as URA3, which allows a ura3 mutant to grow on medium lacking uracil. Finally, the vector contains sequences for replication and selection in E. coli (ORI and amp ${ }^{\text {T }}$ ) and a polylinker for easy insertion of yeast DNA fragments. (b) Typical protocol for constructing a yeast genomic library. Partial digestion of total yeast genomic DNA with Sau3A is adjusted to generate fragments with an average size of $\sim 10 \mathrm{~kb}$. The vector is prepared to accept the genomic fragments by digestion with BamHI, which produces the same sticky ends as Sau3A. Each transformed clone of E. coli that grows after selection for ampicillin resistance contains a single type of yeast DNA fragment.

![li872239539b](li872239539b.jpg)

EXPERIMENTAL FIGURE 6-16 Screening of a yeast genomic library by functional complementation can identify clones carrying the normal form of a mutant yeast gene. In this example, a wild-type CDC gene is isolated by complementation of a cdc yeast mutant. The yeast genomic library prepared as shown in Figure 6-15 is transformed into a ura3 ${ }^{-}$, temperature-sensitive cdc mutant strain. The relatively few transformed yeast cells, which contain recombinant plasmid DNA, can grow in the absence of uracil at $23^{\circ} \mathrm{C}$. When these colonies are replica-plated and incubated at $36^{\circ} \mathrm{C}$ (a nonpermissive temperature), only clones carrying a library plasmid that contains the wild-type copy of the CDC gene will survive. LiOAC $=$ lithium acetate; PEG = polyethylene glycol.
but present certain experimental difficulties for higher eukaryotes. First, the genes of eukaryotes usually contain extensive intron sequences and can therefore be too large to be inserted intact into plasmid vectors. As a result, the sequences of individual genes are broken apart and carried in more than one clone. Moreover, the presence of introns and long intergenic regions in genomic DNA often makes it difficult to identify the important parts of a gene that actually encode protein sequences. For example, only about 1.5 percent of the human genome actually represents protein-coding gene sequences. Thus for many studies, cellular mRNAs, which lack the noncoding regions present in genomic DNA, are a more useful starting material for generating a DNA library. In this approach, DNA copies of mRNAs, called complementary DNAs (cDNAs), are synthesized and cloned into plasmid vectors. A large collection of the resulting cDNA clones, representing all the mRNAs expressed in a cell type, is called a cDNA library.

The first step in preparing a cDNA library is to isolate the total mRNA from the cell type or tissue of interest. Because of their poly(A) tails, mRNAs are easily separated from the much more prevalent rRNAs and tRNAs present in a cell extract by use of a matrix to which short strings of thymidylate (oligo-dTs) are linked. The general procedure for preparing a cDNA library from a mixture of cellular mRNAs is outlined in Figure 6-17. The enzyme reverse transcriptase, which is found in retroviruses, is used to synthesize a strand of DNA complementary to each mRNA molecule, starting from an oligo-dT primer (steps $\mathbf{\square}$ and $\mathbf{\square}$ ). The resulting cDNA-mRNA hybrid molecules are converted in several steps into double-stranded cDNA molecules corresponding to all the mRNA molecules in the original preparation (steps $\mathbf{\square}-\mathbf{\square})$. Each double-stranded cDNA contains an oligo-dC$\cdot$oligo-dG double-stranded region at one end and an oligo-dT-oligo-dA double-stranded region at the other end. Methylation of the cDNA protects it from subsequent restriction enzyme cleavage (step $\mathbf{\square}$ ).

To prepare double-stranded cDNAs for cloning, short double-stranded DNA molecules containing the recognition site for a particular restriction enzyme (called linkers) are ligated to both ends of the cDNAs using DNA ligase from bacteriophage T4 (Figure 6-17, step $\mathbf{\square}$ ). As noted earlier, this ligase can join "blunt-ended" double-stranded DNA molecules lacking sticky ends. The resulting molecules are then treated with the restriction enzyme specific for the attached linker, generating cDNA molecules with sticky ends (step $\mathbf{\square}$ ). In a separate procedure, plasmid DNA is treated with the same restriction enzyme to produce the appropriate sticky ends (step $\mathbf{\square \bullet}$ ).

The plasmid vector and the collection of cDNAs, all containing complementary sticky ends, are then mixed and joined covalently by DNA ligase (Figure 6-17, step $\mathbf{\square}$ ). The resulting DNA molecules are introduced into E. coli cells to generate individual clones; each clone carries a cDNA derived from a single mRNA.

Because different genes are transcribed at very different rates, cDNA clones corresponding to abundantly transcribed genes will be represented many times in a cDNA library, whereas cDNAs corresponding to infrequently transcribed genes will be extremely rare or not present at all. To have a reasonable chance of including clones corresponding to slowly transcribed genes, mammalian cDNA libraries must contain $10^{6}-10^{7}$ individual recombinant clones.

## The Polymerase Chain Reaction Amplifies a Specific DNA Sequence from a Complex Mixture

If the nucleotide sequences at the ends of a particular DNA region are known, the intervening fragment can be amplified directly by the polymerase chain reaction (PCR). Here we describe the basic PCR technique and three situations in which it is used.

![li882239539b](li882239539b.jpg)

The PCR depends on the ability to alternately denature (melt) double-stranded DNA molecules and hybridize complementary single strands in a controlled fashion. As outlined in Figure 6-18, a typical PCR procedure begins with
heat-denaturation of a DNA sample into single strands at $95^{\circ} \mathrm{C}$. Next, two synthetic oligonucleotides complementary to the $3^{\prime}$ ends of the DNA segment of interest (the target sequence) are added in great excess to the denatured DNA,

![li892239539b](li892239539b.jpg)

FIGURE 6-18 The polymerase chain reaction (PCR) is widely used to amplify DNA regions with known flanking sequences. To amplify a specific region of DNA, an investigator chemically synthesizes two different oligonucleotide primers complementary to sequences of approximately 18 bases flanking the region of interest (shown here as light blue and dark blue bars). The complete reaction is composed of a complex mixture of double-stranded DNA (usually genomic DNA containing the target sequence of interest), a stoichiometric excess of both primers, the four dNTPs, and a heat-stable DNA polymerase known as Taq polymerase. During each PCR cycle, the reaction mixture is first heated to separate the strands and then cooled to allow the primers to bind to complementary sequences flanking the region to be amplified. Taq polymerase then extends each primer from its $3^{\prime}$ end, generating newly synthesized strands that extend in the $3^{\prime}$ direction to the $5^{\prime}$ end of the template strand. During the third cycle, two double-stranded DNA molecules are generated equal in length to the sequence of the region to be amplified. In each successive cycle, the target sequence, which anneals to the primers, is duplicated, so it eventually vastly outnumbers all other DNA sequences in the reaction mixture. Successive PCR cycles can be automated by cycling the reaction at timed intervals between a high temperature for DNA melting and a lower temperature for the annealing and elongation parts of the cycle. A reaction that cycles 20 times will amplify the specific target sequence 1-million-fold.
and the temperature is lowered to $50-60^{\circ} \mathrm{C}$. These specific oligonucleotides, which are present at a very high concentration, hybridize to their complementary sequences in the DNA sample, whereas the long strands of the sample DNA remain apart because of their comparatively low concentration. The hybridized oligonucleotides then serve as primers for DNA chain synthesis in the presence of deoxynucleotides (dNTPs) and a temperature-resistant DNA polymerase such as that from Thermus aquaticus, a bacterium that lives in hot springs. This enzyme, called Taq polymerase, can
remain active even after being heated to $95^{\circ} \mathrm{C}$ and can extend the primers at temperatures up to $72^{\circ} \mathrm{C}$. When synthesis is complete, the whole mixture is reheated to $95^{\circ} \mathrm{C}$ to denature the newly formed DNA duplexes. After the temperature is lowered again, another cycle of synthesis takes place because excess primer is still present. Repeated cycles of denaturation (heating) followed by hybridization and synthesis (cooling) quickly amplify the sequence of interest. At each cycle, the number of copies of the sequence between the primer sites is doubled; therefore, the target sequence increases exponentially—about 1-million-fold after 20 cycles-whereas all other sequences in the original DNA sample remain unamplified.

Direct Isolation of a Specific Segment of Genomic DNA For organisms in which all or most of the genome has been sequenced, PCR amplification starting with the total genomic DNA is often the easiest way to obtain a specific DNA region of interest for cloning. In this application of the PCR, the two oligonucleotide primers are designed to hybridize to sequences flanking the genomic region of interest and to include sequences that are recognized by specific restriction enzymes (Figure 6-19). For an oligonucleotide to be useful as a PCR primer, it must be long enough for its sequence to occur uniquely in the genome. For most purposes, this condition is satisfied by oligonucleotides containing about 20 nucleotides. Any given 20-nucleotide sequence occurs once in every $4^{20}\left(\sim 10^{12}\right)$ nucleotides by chance, so it is usually possible to identify two specific 20-nucleotide sequences flanking the target sequence that each occur only once in the genome.

After amplification of the target sequence for about 20 PCR cycles, cleavage with the appropriate restriction

![li902239539b](li902239539b.jpg)
enzymes produces sticky ends that allow efficient ligation of the fragment to a plasmid vector whose polylinker has been cleaved by the same restriction enzymes. The resulting recombinant plasmids, all carrying the identical genomic DNA segment, can then be cloned in E. coli cells. With certain refinements of the PCR, even DNA segments greater than 10 kb in length can be amplified and cloned in this way.

Note that this method does not require the cloning of large numbers of restriction fragments derived from genomic DNA and their subsequent screening to identify the specific fragment of interest. In effect, the PCR method inverts this traditional approach and so avoids its most tedious aspects. The PCR method is useful for isolating gene sequences to be manipulated in a variety of useful ways described later. In addition, the PCR method can be used to isolate gene sequences from mutant organisms to determine how they differ from the wild type.

EXPERIMENTAL FIGURE 6-19 A specific target sequence in genomic DNA can be amplified by PCR for use in cloning. Each primer for PCR is complementary to one end of the target sequence and includes the recognition site for a restriction enzyme that does not have a recognition site within the target region. In this example, primer 1 contains a BamHI recognition sequence, whereas primer 2 contains a HindIII recognition sequence. (Note that for clarity, in any round, amplification of only one of the two strands-the one in brackets-is shown.) After amplification, the target segments are treated with appropriate restriction enzymes, generating fragments with sticky ends. These fragments can be incorporated into complementary plasmid vectors and cloned in E. coli by the usual procedure (see Figure 6-14).

A variation on the PCR method allows PCR amplification of a specific cDNA sequence made from cellular mRNAs. This method, known as reverse transcriptase-PCR ( $R T-P C R$ ), begins with the same procedure described previously for isolation of cDNA from a collection of cellular mRNAs. Typically, an oligo-dT primer, which hybridizes to the $3^{\prime}$ poly $(\mathrm{A})$ tail of the mRNA, is used as the primer for the first strand of cDNA synthesis by reverse transcriptase. A specific cDNA can then be isolated from this complex mixture of cDNAs by PCR amplification using two oligonucleotide primers designed to match sequences at the $5^{\prime}$ and $3^{\prime}$ ends of the corresponding mRNA. As described previously, these primers can be designed to include restriction sites to facilitate the insertion of the amplified cDNA into a suitable plasmid vector.

RT-PCR can be performed so that the starting amount of a particular cellular mRNA can be determined accurately. To carry out quantitative RT-PCR, the amount of double-stranded DNA sequence produced by each amplification cycle is determined as the amplification of a particular mRNA sequence proceeds. By extrapolation from these amounts, an estimate of the starting amount of the mRNA sequence can be obtained. Such quantitative RT-PCR analyses carried out on tissues or whole organisms using primers targeted to genes of interest provide one of the most accurate means to follow changes in gene expression.

Tagging of Genes by Insertion Mutations Another useful application of the PCR is the amplification of a "tagged" gene from the genomic DNA of a mutant strain. This approach is a simpler method for identifying genes associated with a particular mutant phenotype than the screening of a library by functional complementation (see Figure 6-16).

The key to this use of the PCR is the ability to produce mutations by inserting a known DNA sequence into the genome of an experimental organism. Such insertion mutations can be generated by the use of DNA transposons, which can move (or transpose) from one chromosomal site

to another. As discussed in more detail in Chapter 8, these DNA sequences occur naturally in the genomes of most organisms and may give rise to loss-of-function mutations if they transpose into a protein-coding region.

For example, researchers have modified a Drosophila DNA transposon, known as the $P$ element, to optimize its use in the experimental generation of insertion mutations. Once it has been demonstrated that insertion of a P element causes a mutation with an interesting phenotype, the genomic sequences adjacent to the insertion site can be amplified by a variation of the standard PCR protocol that uses synthetic primers complementary to the known P-element sequence, but allows unknown neighboring sequences to be amplified. One such method, depicted in Figure 6-20, begins by cleaving Drosophila genomic DNA containing a P-element insertion with a restriction enzyme that makes a single cut within the P-element DNA. The resulting collection of cleaved DNA fragments, when treated with DNA ligase, yields circular molecules, some of which contain P-element DNA. The chromosomal region flanking the P element can then be amplified by PCR using primers that match P-element sequences and are elongated in opposite directions. The sequence of the resulting amplified fragment can then be determined using a third DNA primer. The crucial sequence for identifying the site of P-element insertion is
the junction between the end of the P-element and genomic sequences. Overall, this approach avoids the cloning of large numbers of DNA fragments and their screening to detect a cloned DNA corresponding to a mutated gene of interest.

Similar methods have been applied to other organisms for which insertion mutations can be generated using either DNA transposons or viruses with sequenced genomes that can insert randomly into the genome.

## Cloned DNA Molecules Can Be Sequenced Rapidly by Methods Based on PCR

The complete characterization of any cloned DNA fragment requires determination of its nucleotide sequence. The technology used to determine the sequence of a DNA segment represents one of the most rapidly developing fields in molecular biology. In the 1970s, F. Sanger and his colleagues developed the chain-termination procedure, which served as the basis for most DNA sequencing methods for the next 30 years. The idea behind this method is to synthesize from the DNA fragment to be sequenced a set of daughter strands that are labeled at one end and terminate at one of the four nucleotides. Separation of the truncated daughter strands by gel electrophoresis, which can resolve strands that differ in length by a single nucleotide, can then reveal the lengths of
![li912239539b](li912239539b.jpg)

FIGURE 6-20 The genomic sequence at the insertion site of a DNA transposon is revealed by PCR amplification and sequencing. To obtain the DNA sequence of the insertion site of a P element, it is necessary to amplify the junction between known transposon sequences and unknown flanking chromosomal sequences. One way to achieve this is to cleave genomic DNA with a restriction enzyme that cuts once within the transposon sequence. Ligation of the resulting restriction fragments will generate circular DNA molecules. By using appropriately designed DNA primers that match transposon sequences, it is possible to amplify the desired junction fragment using PCR. Finally, a DNA sequencing reaction (see Figures 6-21 and 6-22) is performed using the amplified fragment as a template and an oligonucleotide primer that matches sequences near the end of the transposon to obtain the sequence of the junction between the transposon and chromosome.

all strands ending in G, A, T, or C. From these collections of strands of different lengths, the nucleotide sequence of the original DNA fragment can be established. The Sanger method has undergone many refinements and can now be fully automated, but because each DNA fragment requires a separate individual sequencing reaction, the overall rate at which new DNA sequences can be produced by this method is limited by the total number of reactions that can be performed at one time.

A breakthrough in sequencing technology occurred when methods were devised to allow a single sequencing instrument to carry out billions of sequencing reactions simultaneously by localizing them in tiny clusters on the surface of a solid substratum. In 2007, when these so-called next-generation
![li922239539b](li922239539b.jpg)

EXPERIMENTAL FIGURE 6-21 Generation of clusters of identical DNA fragments attached to a solid support. A large collection of DNA fragments to be sequenced is ligated to double-stranded linkers, which become attached to each end of each fragment. The DNA is then amplified by PCR using primers matching the sequences of the linkers that are covalently attached to a solid substratum. Ten cycles of amplification yield about a thousand identical copies of each DNA fragment localized in a small cluster and attached at both ends to the solid substratum. These reactions are optimized to produce as many as $3 \times 10^{9}$ discrete, non-overlapping clusters that are ready to be sequenced.
sequences became commercially available, the capacity for new sequence production increased enormously, and since then, because of improvements in the technology, it has been further increasing at an amazing pace-doubling every few months. In one popular sequencing method, billions of different DNA fragments to be sequenced are prepared by ligating double-stranded linkers to their ends (Figure 6-21). Next the DNA fragments are amplified by PCR using primers that match the linker sequences. This reaction differs from the standard PCR amplification shown in Figure 6-19 in that the primers used are covalently attached to a solid substratum. Thus, as the PCR amplification proceeds, one end of each daughter DNA strand is covalently linked to the substratum, and at the end of the amplification about a thousand identical PCR products are linked to the surface in in a tight cluster.

These clusters can then be sequenced by using a special microscope to image fluorescently labeled deoxyribonucleotide triphosphates (dNTPs) as they are incorporated by DNA polymerase one at a time into a growing DNA chain (Figure 6-22). First, one strand is cut and washed out, leaving a single-stranded DNA template. Then sequencing is carried out on the thousand or so identical templates in each cluster, one nucleotide at a time. All four dNTPs are fluorescently labeled and added to the sequencing reaction. After they are allowed to anneal, the substratum is imaged and the color of each cluster is recorded. Next the fluorescent tag is chemically removed and a new dNTP is allowed to bind. This cycle is repeated about a hundred times, resulting in billions of hundred-nucleotide-long sequences. The entire procedure may take about one day and yield some $10^{11}$ bases of sequence information.

In order to sequence a long continuous region of genomic DNA, or even the entire genome of an organism, researchers usually employ one of the strategies outlined in Figure 6-23. The first method requires the isolation of a collection of cloned DNA fragments whose sequences overlap. Once the sequence of one of these fragments is determined, oligonucleotides based on that sequence can be chemically synthesized for use as primers in sequencing the adjacent overlapping fragments. In this way, the sequence of a long stretch of DNA is determined incrementally by sequencing of the overlapping cloned DNA fragments that compose it. A second method, which is called whole genome shotgun sequencing, bypasses the time-consuming step of isolating an ordered collection of DNA segments that span the genome. This method involves simply sequencing random clones from a genomic library. A sufficient number of clones are chosen for sequencing so that on average, each segment of the genome is sequenced about 10 times. This degree of coverage ensures that each segment of the genome is sequenced more than once. The entire genomic sequence is then assembled using a computer algorithm that aligns all the segments using their regions of overlap. Whole genome shotgun sequencing is the fastest and most cost-effective method for sequencing long stretches of DNA, and most genomes, including the human genome, have been sequenced by this method.

(a)
1 Cut one DNA strand, denature, and wash, leaving single strand

2 Add new primer, then fluorescently labeled dNTPs; one dNTP binds, wash away excess

3 Fluorescent imaging to determine which dNTP bound

4 Chemically remove bound fluorophore and wash
![li932239539b](li932239539b.jpg)
(b)
![li942239539b](li942239539b.jpg)

EXPERIMENTAL FIGURE 6-22 Using fluorescent-tagged deoxyribonucleotide triphosphates for sequence determination.
(a) The reaction begins with the cleaving of one strand of the amplified, clustered DNA (see Figure 6-21). After denaturation, a single DNA strand remains attached to the substratum. A synthetic oligodeoxynucleotide is used as the primer for the polymerization reaction, which also contains dNTPs, each fluorescently tagged with a different color. The fluorescent tag is designed to block the $3^{\prime} \mathrm{OH}$ group on the dNTP so that once the fluorescent dNTP has been incorporated, further elongation is not possible. Because DNA polymerase will incorporate the same fluorescently labeled dNTP into each of the thousand or so identical DNA copies in a cluster, the entire cluster will be uniformly labeled with the same fluorescent color, which can be imaged in a special microscope. (b) Five images from the same field of view, each corresponding to an individual cycle of dNTP addition. Each colored dot represents a cluster of identical DNA fragments. After each image is made, the fluorescent tags are removed by a chemical reaction that leaves a new primer terminus available for the next cycle of dNTP addition. As can be seen for the circled colored dot, the color changes in each reaction cycle according to which nucleotide is added to the DNA fragments. A typical sequencing reaction may carry out a hundred polymerization cycles, allowing a hundred bases of sequence for each cluster to be determined. Thus a total sequencing reaction of this type may generate as much as $3 \times 10^{11}$ bases of sequence information in about two days. [Part (b) A. Loehr and A.W. Zaranek for the Harvard Personal Genome Project.]

EXPERIMENTAL FIGURE 6-23 Two strategies for assembling whole genome sequences. One method (left) depends on isolating and assembling a set of cloned DNA fragments that span the genome. This can be done by matching cloned fragments by hybridization or by alignment of restriction-site maps. The DNA sequence of the ordered clones can then be assembled into a complete genomic sequence. The alternative method (right) depends on the relative ease of automated DNA sequencing and bypasses the laborious step of ordering a DNA library. By sequencing enough random clones from the library so that each segment of the genome is represented from 3 to 10 times, it is possible to reconstruct the genomic sequence by computer alignment of the very large number of sequence fragments.
![li952239539b](li952239539b.jpg)

## KEY CONCEPTS OF SECTION 6.2

## DNA Cloning and Characterization

- In DNA cloning, recombinant DNA molecules are formed in vitro by inserting DNA fragments into vector DNA molecules. The recombinant DNA molecules are then introduced into host cells, where they replicate, producing large numbers of recombinant DNA molecules.
- Restriction enzymes (endonucleases) typically cut DNA at specific $4-8$-bp palindromic sequences, producing defined fragments that often have self-complementary singlestranded tails (sticky ends).
- Two restriction fragments with complementary ends can be joined with DNA ligase to form a recombinant DNA molecule (see Figure 6-12).
- E. coli cloning vectors are small circular DNA molecules (plasmids) that include three functional regions: a replication origin, a selectable marker gene, and a site where a DNA fragment can be inserted. Transformed cells carrying a vector grow into colonies on the selection medium (see Figure 6-14).
- A genomic library is a set of clones carrying restriction fragments produced by cleavage of the entire genome.
- Shuttle vectors that can replicate in both yeast and E. coli can be used to construct a yeast genomic library. Specific genes can be isolated by their ability to complement the corresponding mutant genes in yeast cells (see Figure 6-16).
- In cDNA cloning, expressed mRNAs are reverse-transcribed into complementary DNAs, or cDNAs. A cDNA library is a set of cDNA clones prepared from the mRNAs isolated from a particular type of cell or tissue (see Figure 6-17).
- The polymerase chain reaction (PCR) permits exponential amplification of a specific segment of DNA from a single initial template DNA molecule if the sequence flanking the DNA region to be amplified is known (see Figure 6-19).
- PCR is a highly versatile method that can be programmed to amplify a specific genomic DNA sequence, a cDNA, or a sequence at the junction between a DNA transposon and flanking chromosomal sequences.
- DNA fragments up to about 100 nucleotides long can be sequenced by generating clusters of identical fragments by PCR and imaging fluorescently labeled nucleotide precursors incorporated by DNA polymerase (see Figures 6-21 and 6-22).
- Whole genome sequences can be assembled from the sequences of a large number of overlapping clones from a genomic library (see Figure 6-23).


### 6.3 Using Cloned DNA Fragments to Study Gene Expression

In the last section, we described the basic techniques for using recombinant DNA technology to isolate specific DNA clones and ways in which those clones can be further characterized.

Here we consider how an isolated DNA clone can be used to study gene expression. We discuss several widely used general techniques that rely on nucleic acid hybridization to elucidate when and where genes are expressed, as well as methods for generating large quantities of protein and otherwise manipulating amino acid sequences to determine their expression patterns, structure, and function. More specific applications of all these basic techniques are examined in the following sections.

## Hybridization Techniques Permit Detection of Specific DNA Fragments and mRNAs

Hybridization depends on the ability of complementary single-stranded DNA or RNA molecules to associate (hybridize) specifically with each other via base pairing. As discussed in Chapter 4, double-stranded (duplex) DNA can be denatured (melted) into single strands by heating in a dilute salt solution. If the temperature is then lowered and the ion concentration raised, complementary single strands will reassociate (hybridize) into duplexes. In a mixture of nucleic acids, only complementary single strands (or strands containing complementary regions) will reassociate; moreover, the extent of their reassociation is virtually unaffected by the presence of noncomplementary strands.

Two very sensitive methods for detecting a particular DNA or RNA sequence within a complex mixture combine separation by gel electrophoresis and hybridization to a complementary DNA probe that is either radioactively or fluorescently labeled. A third method involves hybridizing labeled probes directly to a prepared tissue sample. We will encounter references to all three of these techniques, which have numerous applications, in other chapters.

Southern Blotting The first hybridization technique developed to detect DNA fragments of a specific sequence is known as Southern blotting, after its originator, E. M. Southern. This technique is capable of detecting a single specific restriction fragment in the highly complex mixture of fragments produced by cleavage of the entire human genome with a restriction enzyme. A common procedure for separating DNA fragments of different sizes is gel electrophoresis. Near neutral pH , DNA molecules carry a large negative charge and therefore move toward the positive electrode during gel electrophoresis. Because the gel matrix restricts random diffusion, molecules of the same length migrate together as distinct bands. Smaller molecules move through the gel matrix more readily than larger molecules. Larger DNA molecules from about 200 bp to more than 20 kb can be separated electrophoretically on agarose gels, and smaller DNA molecules from about 10 to 2000 bp can be separated on polyacrylamide gels.

When DNA from a whole genome is fragmented by digestion with a restriction enzyme, the mixture of DNA fragments that is produced is so complex that even after gel electrophoresis, so many different fragments of nearly the same length are present that it is not possible to resolve any particular DNA fragment as a discrete band on the gel. Nevertheless, it is possible to identify a particular fragment migrating as a band on the gel by its ability to hybridize to

![li962239539b](li962239539b.jpg)

EXPERIMENTAL FIGURE 6-24 Southern blotting can detect a specific DNA fragment in a complex mixture of restriction fragments. The diagram depicts three different restriction fragments in the gel, but the procedure can be applied to a mixture of millions of DNA fragments. Only fragments that hybridize to a labeled probe will give a signal on an autoradiogram. A similar technique, called Northern blotting, detects specific mRNAs within a mixture. See E. M. Southern, 1975, J. Mol. Biol. 98:508.
a specific DNA probe. To accomplish this, the restriction fragments present in the gel are denatured with alkali and transferred onto a nitrocellulose filter or nylon membrane by blotting (Figure 6-24). This procedure preserves the distribution of the fragments in the gel, creating a replica of the gel on the filter. (The blot is used because probes do not readily diffuse into the original gel.) The filter is then incubated under hybridization conditions with a specific labeled DNA probe, which is usually generated from a cloned restriction fragment. The DNA restriction fragment that is complementary to the probe hybridizes to it, and its location on the filter can be revealed by autoradiography (for a radiolabeled probe) or by fluorescent imaging (for a fluorescently labeled probe). Although PCR is now more commonly used to detect the presence of a particular sequence in a complex mixture, Southern blotting is still useful for reconstructing the relationship between genomic sequences that are too far apart to be amplified by PCR in a single reaction.

Northern Blotting One of the most basic ways to characterize a cloned gene is to determine when and where in an organism the gene is expressed. The expression of a particular gene can be followed by assaying for the corresponding mRNA by Northern blotting, named, in a play on words, after the related method of Southern blotting. An RNA sample, often the total cellular RNA, is denatured by treatment with an agent such as formaldehyde that disrupts the hydrogen bonds between base pairs, ensuring that all the RNA molecules have an unfolded, linear conformation. The individual RNAs are separated according to size by gel electrophoresis and transferred to a nitrocellulose filter to which the extended denatured RNAs adhere. As in Southern blotting, the filter is then exposed to a labeled DNA probe that is complementary to the gene of interest; finally, the labeled filter is subjected to autoradiography. Because the amount of a specific RNA in a sample can be estimated from a Northern blot, the procedure is widely used to compare the amounts of a particular mRNA in cells under different conditions.

In Situ Hybridization Northern blotting requires extracting the mRNA from a cell or mixture of cells, which means that the cells are removed from their normal location within an organism or tissue. As a result, the location of a cell and its relation to its neighbors is lost. To retain such positional information in precise studies of gene expression, a whole or sectioned tissue, or even a whole permeabilized embryo, may be subjected to in situ hybridization to detect the mRNA encoded by a particular gene. This technique allows gene transcription to be monitored in both time and space (Figure 6-25).

## DNA Microarrays Can Be Used to Evaluate the Expression of Many Genes at One Time

Monitoring the expression of thousands of genes simultaneously is possible with DNA microarray analysis, another technique based on the concept of nucleic acid hybridization. A DNA microarray consists of an organized array of thousands of individual, closely packed, gene-specific sequences attached to the surface of a glass microscope slide. By coupling microarray analysis with the results from genomic sequencing projects, researchers can analyze the global patterns of gene expression in an organism during specific physiological responses or developmental processes.

Preparation of DNA Microarrays In one method of preparing microarrays, a DNA segment of about 1 kb corresponding to part of the coding region of each gene to be analyzed is individually amplified by PCR. A robotic device is used to apply each amplified DNA sample to the surface of a glass microscope slide, which is then chemically processed to permanently attach the DNA sequences to the glass surface and to denature them. A typical array might contain some 6000 spots of DNA in a $2 \times 2-\mathrm{cm}$ grid.

In an alternative method, multiple DNA oligonucleotides, usually at least 20 nucleotides in length, are synthesized from an initial nucleotide that is covalently bound to the surface of a glass slide. The synthesis of an oligonucleotide of specific

![li972239539b](li972239539b.jpg)

EXPERIMENTAL FIGURE 6-25 In situ hybridization can detect activity of specific genes in whole and sectioned embryos. The specimen is permeabilized by treatment with detergent and a protease to expose the mRNA to the probe. A DNA or RNA probe, specific for the mRNA of interest, is made with nucleotide analogs containing chemical groups that can be recognized by antibodies. After the permeabilized specimen has been incubated with the probe under conditions that promote hybridization, the excess probe is removed with a series of washes. The specimen is then incubated in a solution containing an antibody that binds to the probe. This antibody is covalently joined to a reporter enzyme (e.g., horseradish peroxidase or alkaline phosphatase) that produces a colored reaction product. After excess antibody
has been removed, substrate for the reporter enzyme is added. A colored precipitate forms where the probe has hybridized to the mRNA being detected. (a) A whole mouse embryo at about 10 days of development probed for Sonic hedgehog mRNA. The stain marks the notochord (red arrow), a rod of mesoderm running along the future spinal cord. (b) A cross section of a mouse embryo similar to that in part (a). The dorsal/ventral axis of the neural tube (NT) can be seen, with the Sonic hedgehog-expressing notochord (red arrow) below it and the endoderm (blue arrow) still farther ventral. (c) A whole Drosophila embryo probed for an mRNA produced during trachea development. The repeating pattern of body segments is visible. Anterior (head) is up; ventral is to the left. [Ljiljana Milenkovic and Matthew P. Scott.]
sequence can be programmed in a small region on the surface of the slide. Several oligonucleotide sequences from a single gene are thus synthesized in neighboring regions of the slide. With this method, oligonucleotides representing thousands of genes can be produced on a single glass slide. Because the methods for constructing these arrays of synthetic oligonucleotides were adapted from methods for manufacturing microscopic integrated circuits used in computers, these types of oligonucleotide microarrays are often called DNA chips.

Using Microarrays to Compare Gene Expression Under Different Conditions The initial step in a microarray expression study is to prepare cDNAs from the mRNAs expressed by the cells under study and attach fluorescent labels to them. When the cDNA preparation is applied to a microarray under appropriate conditions, DNA spots representing genes that are expressed will hybridize to their complementary cDNAs in the labeled probe mix and can subsequently be detected in a scanning laser microscope.

Figure 6-26 depicts how this method can be applied to examine the changes in gene expression observed after starved human fibroblasts are transferred to a rich, serum-containing growth medium. In this type of experiment, the separate cDNA preparations from starved and from serum-grown fibroblasts are labeled with differently colored fluorescent dyes. A DNA microarray comprising 8600 mammalian genes is then incubated with a mixture containing equal amounts of the two cDNA preparations under hybridization conditions. After unhybridized cDNA is washed away, the
intensity of green and red fluorescence at each DNA spot is measured using a fluorescence microscope and is recorded under the name of each gene according to its known position on the slide. The relative intensities of red and green fluorescence signals at each spot provide a measure of the relative level of expression of that gene in response to serum. Genes that are not transcribed under these growth conditions give no detectable signal. Genes that are transcribed at the same level under both conditions hybridize equally to both redand green-labeled cDNA preparations. Microarray analysis of gene expression in fibroblasts showed that transcription of about 500 of the 8600 genes examined changed substantially after addition of serum.

## Cluster Analysis of Multiple Expression Experiments Identifies Co-regulated Genes

Firm conclusions about whether genes that exhibit similar changes in expression are co-regulated, and hence likely to be closely related functionally, rarely can be drawn from a single microarray experiment. For example, many of the observed differences in gene expression just described in fibroblasts could be indirect consequences of the many different changes in cell physiology that occur when cells are transferred from one medium to another. In other words, genes that appear to be co-regulated in a single microarray expression experiment may undergo changes in expression for very different reasons and may actually have very different biological functions. A solution to this problem is to

![li982239539b](li982239539b.jpg)

A If a spot is green, expression of that gene decreases in cells after serum addition

B If a spot is red, expression of that gene increases in cells after serum addition
![li992239539b](li992239539b.jpg)
combine the information from a set of microarray expression experiments to find genes that are similarly regulated under a variety of conditions or over a period of time.

EXPERIMENTAL FIGURE 6-26 DNA microarray analysis can reveal differences in gene expression in fibroblasts under different experimental conditions. (a) In this example, cDNA prepared from mRNA isolated from fibroblasts either starved for serum or after serum addition is labeled with different fluorescent dyes. A microarray composed of DNA spots representing 8600 mammalian genes is exposed to an equal mixture of the two cDNA preparations under hybridization conditions. The ratio of the intensities of red and green fluorescence over each spot, detected with a scanning confocal laser microscope, indicates the relative expression of each gene in response to serum. (b) A micrograph of a small segment of an actual DNA microarray. Each spot in this $16 \times 16$ array contains DNA from a different gene hybridized to control and experimental cDNA samples labeled with red and green fluorescent dyes. (A yellow spot indicates equal hybridization of green and red fluorescence, indicating no change in gene expression.) [Part (b) Alfred Pasieka/Science Source.]

This more informative use of multiple microarray expression experiments is illustrated by an examination of the relative expression of the 8600 genes mentioned above at different times after serum addition to fibroblasts, which generated more than $10^{4}$ individual pieces of data. A computer program, related to the one used to determine the relatedness of different protein sequences, can organize these data and cluster genes that show similar expression over the time course after serum addition. Remarkably, such cluster analysis groups sets of genes whose encoded proteins participate in a common cellular process, such as cholesterol biosynthesis or the cell cycle (Figure 6-27).

Microarray analysis is a powerful diagnostic tool in medicine. For instance, particular sets of mRNAs have been found to distinguish tumors with a poor prognosis from those with a good prognosis. Previously indistinguishable disease variations are now detectable. Analysis of tumor biopsies for these distinguishing mRNAs will help physicians to select the most appropriate treatment. As more patterns of gene expression characteristic of various diseased tissues are recognized, the diagnostic use of DNA microarrays will be extended to other conditions.

## E. coli Expression Systems Can Produce Large Quantities of Proteins from Cloned Genes

Many protein hormones and other signaling or regulatory proteins are normally expressed at very low concentrations, which precludes their isolation and purification in large quantities by standard biochemical techniques. Widespread therapeutic use of such proteins, as well as basic research on their structure and functions, depends on efficient procedures for producing them in large amounts at reasonable cost. Recombinant DNA techniques that turn E. coli cells into factories for synthesizing low-abundance proteins are now used to produce granulocyte colony-stimulating factor (G-CSF), insulin, growth hormone, erythropoietin, and other human proteins with therapeutic uses commercially. For example, G-CSF stimulates the production of

Each column represents a different gene at times after addition of serum
![li1002239539b](li1002239539b.jpg)

## EXPERIMENTAL FIGURE 6-27 Cluster analysis of data from multiple microarray expression experiments can identify co-regulated genes. The expression of 8600 mammalian genes was detected by microarray analysis at time intervals over a 24-hour period after serum-starved fibroblasts were provided with serum. The cluster diagram shown here is based on a computer algorithm that groups genes showing similar changes in expression compared with a serum-starved control sample over time. Each column of colored boxes represents a single gene, and each row represents a time point. A red box indicates an increase in expression relative to the control; a green box, a decrease

granulocytes, the phagocytic white blood cells that are critical to defense against bacterial infections. Administration of G-CSF to cancer patients helps offset the reduction in granulocyte production caused by chemotherapeutic agents, thereby protecting patients against serious infection while they are receiving chemotherapy.

The first step in producing large amounts of a lowabundance protein is to obtain a cDNA clone encoding the full-length protein by the methods discussed previously. The second step is to engineer plasmid vectors that will express large amounts of the encoded protein when they are inserted into E. coli cells. The key to designing such expression vectors is inclusion of a promoter, a DNA sequence from which transcription of the cDNA can begin. Consider, for example, the relatively simple system for expressing G-CSF shown in Figure 6-28. In this case, G-CSF is expressed in E. coli transformed with plasmid vectors that contain the lac promoter

## EXPERIMENTAL FIGURE 6-28 Some eukaryotic proteins can be produced in E. coli cells from plasmid vectors containing the

lac promoter. (a) This plasmid expression vector contains a fragment of the E. coli chromosome containing the lac promoter and the neighboring lacZ gene. In the presence of the lactose analog IPTG, RNA polymerase normally transcribes the lacZ gene, producing lacZ mRNA, which is translated into the encoded protein, $\beta$-galactosidase. (b) The lacZ gene can be cut out of the expression vector with restriction enzymes and replaced by a cloned cDNA, in this case, one encoding granulocyte colony-stimulating factor (G-CSF). When the resulting plasmid is inserted into E. coli cells, addition of IPTG and subsequent transcription from the lac promoter produce G-CSF mRNA, which is translated into G-CSF protein.
in expression; and a black box, no significant change in expression. The "tree" diagram at the top shows how the expression patterns for individual genes can be organized in a hierarchical fashion to group together the genes with the greatest similarity in their patterns of expression over time. Five clusters of coordinately regulated genes were identified in this experiment, as indicated by the bars at the bottom. Each cluster contains multiple genes whose encoded proteins function in a particular cellular process: cholesterol biosynthesis (A), the cell cycle (B), the immediateearly response (C), signaling and angiogenesis (D), and wound healing and tissue remodeling (E). [Michael B. Eisen, University of California, Berkeley.]
adjacent to the cloned cDNA encoding G-CSF. Transcription from the lac promoter occurs at high rates only when lactose, or a lactose analog such as isopropylthiogalactoside (IPTG),
![li1012239539b](li1012239539b.jpg)

is added to the culture medium. Even larger quantities of a desired protein can be produced in more complicated E. coli expression systems.

To aid in purification of a eukaryotic protein produced in an E. coli expression system, researchers often modify the cDNA encoding the recombinant protein to facilitate its separation from endogenous E. coli proteins. A commonly used modification of this type is the addition of a short nucleotide sequence to the end of the cDNA, so that the expressed protein will have six histidine residues at the C-terminus. Proteins modified in this way bind tightly to an affinity matrix that contains chelated nickel atoms, whereas E. coli proteins do not bind to such a matrix. The bound proteins can be released from the nickel atoms by decreasing the pH of the surrounding medium. In most cases, this procedure yields a pure recombinant protein that is functional, since addition of short amino acid sequences to either the C-terminus or the N -terminus of a protein usually does not interfere with the protein's biochemical activity.

## Plasmid Expression Vectors Can Be Designed for Use in Animal Cells

Although bacterial expression systems can be used successfully to create large quantities of some proteins, bacteria cannot be used in all cases. Many experiments designed to examine the function of a protein in an appropriate cellular context require expression of a genetically modified protein in cultured animal cells. To accomplish this, genes are cloned into specialized eukaryotic expression vectors and are introduced into cultured animal cells by a process called transfection. Two common methods for transfecting animal cells differ in whether the recombinant vector DNA is or is not integrated into the host-cell genomic DNA.

In both methods, cultured animal cells must be treated to facilitate their initial uptake of the recombinant plasmid vector. This can be done by exposing cells to a preparation of lipids that penetrates the plasma membrane, increasing its permeability to DNA. Alternatively, subjecting cells to a brief electric shock of several thousand volts, a technique known as electroporation, makes them transiently permeable to DNA. Usually the plasmid DNA is added in sufficient concentration to ensure that a large proportion of the cultured cells will receive at least one copy. Researchers have also harnessed viruses for transfection; viruses can be modified to contain DNA of interest, which is then introduced into host cells by simply infecting them with the recombinant virus.

Transient Transfection The simpler of the two transfection methods, called transient transfection, employs a plasmid vector similar to the yeast shuttle vectors described previously. For use in mammalian cells, plasmid vectors are engineered to carry a replication origin derived from a virus that infects mammalian cells, a strong promoter recognized by mammalian RNA polymerase, and the cloned cDNA encoding the protein to be expressed adjacent to the promoter (Figure 6-29a). Once such a plasmid vector enters a mammalian cell, the viral
![li1022239539b](li1022239539b.jpg)

Protein is expressed from cDNA in plasmid DNA
(b) Stable transfection (transformation)
![li1032239539b](li1032239539b.jpg)

EXPERIMENTAL FIGURE 6-29 Transient and stable transfection with specially designed plasmid vectors permits expression of cloned genes in cultured animal cells. Both methods employ plasmid vectors that contain the usual elements-replication origin, selectable marker (e.g., amp'), and polylinker-that permit propagation in E. coli as well as a cloned cDNA with an adjacent animal promoter. For simplicity, these elements are not depicted. (a) In transient transfection, the plasmid vector contains a replication origin from a virus that can replicate in the cultured animal cells. Since the vector is not incorporated into the genome of the cultured cells, production of the cDNA-encoded protein continues for only a limited time. (b) In stable transfection, the vector carries a selectable marker such as neo', which confers resistance to G-418. The relatively few transfected animal cells that integrate the exogenous DNA into their genomes are selected on medium containing G-418. Because the vector is integrated into the genome, these stably transfected, or transformed, cells will continue to produce the cDNA-encoded protein as long as the culture is maintained. See the text for discussion.

replication origin allows it to replicate efficiently, generating numerous plasmids from which the protein is expressed. However, during cell division, such plasmids are not faithfully segregated into both daughter cells, and in time, a substantial fraction of the cells in a culture will not contain a plasmid, hence the name transient transfection.

Stable Transfection (Transformation) If an introduced vector integrates into the genome of the host cell, that genome is permanently altered, and the cell is said to be transformed. Integration is most likely accomplished by endogenous enzymes that normally function in DNA repair and recombination. A commonly used selectable marker is the gene for neomycin phosphotransferase (designated neo'), which confers resistance to a toxic compound chemically related to neomycin, known as G-418. The basic procedure for expressing a cloned cDNA by stable transfection is outlined in Figure 6-29b. Only those cells that have integrated the expression vector into the host chromosome will survive and give rise to a clone in the presence of a high concentration of G-418. Because integration occurs at random sites in the genome, individual transformed G-418-resistant clones will differ in their rates of transcribing the inserted cDNA. Therefore, the stable transfectants are usually screened to identify those that produce the protein of interest at the highest levels.

Retroviral Expression Systems Researchers have exploited the basic mechanisms used by viruses for introduction of their genetic material into animal cells and its subsequent insertion into chromosomal DNA to greatly increase the efficiency by which a modified gene can be stably expressed in animal cells. One such viral expression system is derived from a class of retroviruses known as lentiviruses. As shown
![li1042239539b](li1042239539b.jpg)
in Figure 6-30, three different plasmids, introduced into cells by transient transfection, are used to produce recombinant lentivirus particles suitable for efficient introduction of a cloned gene into target animal cells. The first plasmid, known as the vector plasmid, contains a cloned gene of interest next to a selectable marker such as neo' flanked by lentivirus LTR sequences. The left LTR sequence directs synthesis of an RNA molecule that carries lentiviral LTR sequences at either end and thus has many of the properties of native retroviral RNA. In an appropriate host, this LTR-bearing RNA can be packaged into viral particles and then introduced into a target cell by viral infection. In the target cell, the LTR sequences direct the copying of the RNA into double-stranded DNA by reverse transcription and the integration of that DNA into chromosomal DNA following a sequence of events depicted in Figure 8-14. A second plasmid, known as the packaging plasmid, carries all the viral genes except for the major viral envelope protein, necessary for packaging LTR-containing viral RNA into a functional lentivirus particle. The final plasmid allows expression of a viral envelope protein that, when incorporated into a recombinant lentivirus, allows the resulting hybrid virus particles to infect a desired target cell type. A common envelope protein used in this context is the glycoprotein of the vesicular stomatitis virus (VSV-G protein), which can readily replace the normal lentivirus envelope protein on the surface of completed virus particles and allows the resulting virus particles to infect a wide variety of mammalian cell types, including hematopoietic stem cells, neurons, and muscle and liver cells. After cell infection, the cloned gene flanked by the viral LTR sequences is reverse-transcribed into DNA, which is transported into the nucleus and then integrated into the host genome. If necessary, as in the case of stable transfection, cells with a stably integrated cloned gene and neo marker can be selected by resistance to G-418. Many of the techniques for inactivating the function of specific genes (see Section 6.5) require that an entire population of cultured cells be genetically modified simultaneously. Engineered lentiviruses are particularly useful for such experiments because they infect cells with such high efficiency that every cell in a population will receive at least one copy of the lentivirus-borne plasmid.

Gene and Protein Tagging Expression vectors can provide a way to study the expression and intracellular localization of eukaryotic proteins. Such studies often rely on the use of a reporter protein, such as green fluorescent protein (GFP),

EXPERIMENTAL FIGURE 6-30 Retroviral vectors can be used for efficient integration of cloned genes into the mammalian genome. See text for discussion.

that can be conveniently detected in cells (see Figure 4-16). Here we describe two ways to create a hybrid gene that connects expression of the reporter protein to that of the protein of interest. When the hybrid gene is reintroduced into cells, either by transfection with a plasmid expression vector containing the modified gene or by creation of a transgenic animal as described in Section 6.5, the expression of the reporter protein can be used to determine where and when a gene is expressed. This method provides data similar to that from the in situ hybridization experiments described previously, but often with greater resolution and sensitivity.

Figure 6-31 illustrates the use of two different types of GFP-tagging experiments to study the expression of an odorant receptor protein in C. elegans. When the promoter for the
(a) Promoter-fusion; ODR10 promoter fused to GFP
![li1052239539b](li1052239539b.jpg)
(b) Protein-fusion; ODR10-GFP fusion protein
![li1062239539b](li1062239539b.jpg)

EXPERIMENTAL FIGURE 6-31 Gene and protein tagging facilitate cellular localization of proteins expressed from cloned genes. In this experiment, the gene encoding a chemical odorant receptor, Odr10, of C. elegans was fused to the gene encoding green fluorescent protein (GFP). (a) A promoter-fusion was generated by linking just the promoter of Odr10 to the coding sequence for GFP. The result is that GFP is expressed in the cytoplasm of the same specific sensory neurons in the head of C. elegans where Odr10 is expressed. (b) A protein-fusion was constructed by linking GFP to the end of the full-length Odr10 coding sequence. In this case, the Odr10-GFP fusion protein is targeted to the membrane at the tip of the sensory neurons and is apparent only at the distal end of the sensory cilia. The observed distribution can be inferred to reflect the normal location of Odr10 protein in specific neurons. Because the promoter-fusion shown in (a) lacks the Odr10 localization sequences, the expressed GFP fills the entire cell cytoplasm rather than being localized just to the distal tip of the sensory cilia. [Courtesy Ashish Maurya.]
odorant receptor is linked directly to the coding sequence of GFP in a configuration usually known as a promoter-fusion, GFP is expressed in specific neurons, filling the cytoplasm of those neurons. In contrast, when the hybrid gene is constructed by linking GFP to the coding sequence of the odorant receptor, the resulting protein-fusion can be localized by GFP fluorescence at the distal cilia in sensory neurons, the site at which the receptor protein is normally located.

An alternative to GFP tagging for detecting the intracellular location of a protein is to modify the gene of interest by appending to it a short DNA sequence that encodes a short stretch of amino acids recognized by a known monoclonal antibody. Such a short peptide that can be bound by an antibody is called an epitope; hence this method is known as epitope tagging. After transfection of cells with a plasmid expression vector containing the modified gene, the expressed epitope-tagged form of the protein can be detected by immunofluorescence labeling of the cells with the monoclonal antibody specific for the epitope. The choice of whether to use a short epitope or GFP to tag a given protein often depends on what types of modification a cloned gene can tolerate and still remain functional.

## KEY CONCEPTS OF SECTION 6.3

## Using Cloned DNA Fragments to Study Gene Expression

- Southern blotting, which can detect a single specific DNA fragment within a complex mixture, combines gel electrophoresis, transfer (blotting) of the separated bands to a filter, and hybridization with a complementary labeled DNA probe (see Figure 6-24). The similar technique of Northern blotting can detect a specific RNA within a mixture.
- The presence and distribution of specific mRNAs can be detected in living cells by in situ hybridization.
- DNA microarray analysis simultaneously detects the relative levels of expression of thousands of genes in different types of cells or in the same cells under different conditions (see Figure 6-26).
- Cluster analysis of the data from multiple microarray expression experiments can identify genes that are similarly regulated under various conditions. Such co-regulated genes commonly encode proteins that have biologically related functions.
- Expression vectors derived from plasmids allow the production of abundant amounts of a protein from a cloned gene.
- Eukaryotic expression vectors can be used to express cloned genes in yeast or mammalian cells. An important application of these methods is the tagging of proteins with GFP or an epitope for antibody detection.

### 6.4 Locating and Identifying Human Disease Genes

5 Inherited human diseases are the phenotypic consequence of defective human genes. Table 6-2 lists several of the most commonly occurring inherited diseases. Although a "disease" gene may result from a new mutation that arose in the preceding generation, most cases of inherited diseases are caused by preexisting mutant alleles that have been passed from one generation to the next for many generations.

The typical first step in deciphering the underlying cause of any inherited human disease is to identify the affected gene and its encoded protein. Comparison of the sequences of a disease gene and its product with those of genes and proteins whose sequence and function are known can provide clues to the molecular and cellular cause of the disease. Historically, researchers have used whatever phenotypic clues might be relevant to make guesses about the molecular basis of inherited diseases. An early example of successful guesswork was the hypothesis that sickle-cell disease, known to be a disease of blood cells, might be caused by defective hemoglobin. This idea led to the identification of a specific amino acid substitution in hemoglobin that causes polymerization of the
defective hemoglobin molecules, causing the sickle-like deformation of red blood cells in individuals who have inherited two copies of the $H b^{\prime}$ allele for sickle-cell hemoglobin.

Most often, however, the genes responsible for inherited diseases must be found without any prior knowledge or reasonable hypotheses about the nature of the affected gene or its encoded protein. In this section, we see how human geneticists can find the gene responsible for an inherited disease by following the segregation of the disease in families. The segregation of the disease can be correlated with the segregation of many other genetic markers, eventually leading to identification of the chromosomal position of the affected gene. This information, along with knowledge of the sequence of the human genome, can ultimately allow the affected gene and the disease-causing mutations to be pinpointed.

## Monogenic Diseases Show One of Three Patterns of Inheritance

Human genetic diseases that result from mutation in one specific gene, referred to as monogenic diseases, display different inheritance patterns depending on the nature and chromosomal location of the alleles that cause them. One

TABLE 6-2 Common Inherited Human Diseases

| Disease | Molecular and Cellular Defect | Incidence |
| :--: | :--: | :--: |
| Autosomal Recessive |  |  |
| Sickle-cell disease | Abnormal hemoglobin causes deformation of red blood cells, which can become lodged in capillaries; also confers resistance to malaria. | $1 / 625$ of sub-Saharan African origin |
| Cystic fibrosis | Defective chloride channel (CFTR) in epithelial cells leads to excessive mucus in lungs. | $1 / 2500$ of European origin |
| Phenylketonuria (PKU) | Defective enzyme in phenylalanine metabolism (tyrosine hydroxylase) results in excess phenylalanine leading to mental retardation, unless restricted by diet. | $1 / 10,000$ of European origin |
| Tay-Sachs disease | Defective hexosaminidase enzyme leads to accumulation of excess sphingolipids in the lysosomes of neurons, impairing neural development. | $1 / 1000$ eastern European Jews |
| Autosomal Dominant |  |  |
| Huntington's disease | Defective neural protein (huntingtin) may assemble into aggregates, causing damage to neural tissue. | $1 / 10,000$ of European origin |
| Hypercholesterolemia | Defective LDL receptor leads to excessive cholesterol in blood and early heart attacks. | $1 / 122$ French Canadians |
| X-Linked Recessive |  |  |
| Duchenne muscular dystrophy (DMD) | Defective cytoskeletal protein (dystrophin) leads to impaired muscle function. | $1 / 3500$ males |
| Hemophilia A | Defective blood clotting factor VIII leads to uncontrolled bleeding. | $1-2 / 10,000$ males |

characteristic pattern is that exhibited by a dominant allele in an autosome (that is, one of the 22 human chromosomes that is not a sex chromosome). Because an autosomal dominant allele is expressed in the heterozygote, usually at least one of the parents of an affected individual will also have the disease. Diseases caused by dominant alleles often appear later in life, after reproductive age. If this were not the case, natural selection would have eliminated these alleles during human evolution. An example of an autosomal dominant disease is Huntington's disease, a neural degenerative disease that generally strikes in mid- to late life. If either parent carries a mutant HD allele, each of his or her children (regardless of sex) has a 50 percent chance of inheriting the mutant allele and being affected (Figure 6-32a).

A recessive allele in an autosome exhibits quite a different segregation pattern. Both parents must be heterozygous carriers of an autosomal recessive allele in order for their children to be at risk of the disease. Each child of heterozygous
![li1072239539b](li1072239539b.jpg)

FIGURE 6-32 Three common inheritance patterns for human monogenic diseases. Wild-type autosomes ( $A$ ) and sex chromosomes ( $X$ and $Y$ ) are indicated by superscript plus signs. (a) In an autosomal dominant disorder such as Huntington's disease, only one mutant allele is needed to confer the disease. If either parent is heterozygous for the mutant $H D$ allele, his or her children have a 50 percent chance of inheriting the mutant allele and getting the disease. (b) In an autosomal recessive disorder such as cystic fibrosis, two mutant alleles must be present to confer the disease. Both parents must be heterozygous carriers of the mutant CFTR gene for their children to be at risk of being affected or being carriers. (c) An X-linked recessive disease such as Duchenne muscular dystrophy is caused by a recessive mutation on the X chromosome and exhibits the typical sex-linked segregation pattern. Males born to mothers heterozygous for a mutant DMD allele have a 50 percent chance of inheriting the mutant allele and being affected. Females born to heterozygous mothers have a 50 percent chance of being carriers.
parents has a 25 percent chance of receiving both recessive alleles, and thus being affected; a 50 percent chance of receiving one normal and one mutant allele, and thus being a carrier; and a 25 percent chance of receiving two normal alleles. A clear example of an autosomal recessive disease is cystic fibrosis, which results from a defective chloride-channel gene known as CFTR (Figure 6-32b). Related individuals (e.g., first or second cousins) have a relatively high probability of being carriers for the same recessive alleles. Thus children born to related parents are much more likely than those born to unrelated parents to be homozygous for, and therefore affected by, a rare autosomal recessive disorder.

The third common pattern of inheritance is that of an X-linked recessive allele. A recessive allele on the X chromosome will most often be expressed in males, who receive only one X chromosome from their mother, but not in females, who receive an X chromosome from both their mother and their father. This leads to a distinctive sex-linked segregation pattern in which the disease is exhibited much more frequently in males than in females. For example, Duchenne muscular dystrophy (DMD), a muscle degenerative disease that specifically affects males, is caused by a recessive allele on the X chromosome. DMD exhibits the typical sex-linked segregation pattern in which mothers who are heterozygous, and therefore phenotypically normal, can act as carriers, transmitting the DMD allele, and therefore the disease, to 50 percent of their male progeny (Figure 6-32c).

## DNA Polymorphisms Are Used as Markers for Linkage Mapping of Human Mutations

Once the mode of inheritance has been determined, the next step in finding a disease allele is to map its position with respect to known genetic markers using the basic principle of genetic linkage, as described in Section 6.1. The presence of many different already mapped genetic traits, or markers, distributed along the length of a chromosome facilitates the mapping of a new mutation. The more markers that are available, the more precisely a mutation can be mapped. The density of genetic markers needed for a high-resolution human genetic map is about one marker every 5 centimorgans (cM) (as discussed previously, one genetic map unit, or centimorgan, is defined as the distance between two positions along a chromosome that results in 1 recombinant individual in 100 progeny). Thus a high-resolution genetic map requires 25 or so genetic markers of known position distributed along the length of each human chromosome.

In the experimental organisms commonly used in genetic studies, numerous markers with easily detectable phenotypes are available for genetic mapping of mutations. For humans, there are not nearly enough phenotypic markers to carry out genetic mapping studies. Instead, recombinant DNA technology has made available a wealth of useful DNA-based molecular markers. Because most of the human genome does not encode proteins, a large amount of phenotypically inconsequential sequence variation exists between individuals. Indeed, it has been estimated that nucleotide

differences between unrelated individuals occur on an average of 1 of every $10^{3}$ nucleotides. Since variations in DNA sequence, referred to as DNA polymorphisms, can be followed from one generation to the next by sequencing the DNA of individuals, they can serve as ideal genetic markers for linkage studies. Currently, a panel of as many as $10^{4}$ different known polymorphisms whose locations have been mapped in the human genome is used for genetic linkage studies in humans.

Single-nucleotide polymorphisms (SNPs) constitute the most abundant type of DNA polymorphism and are therefore useful for constructing genetic maps of maximum resolution (Figure 6-33). Another useful type of DNA polymorphism consists of a variable number of repetitions of a two-, three-, or four-base sequence. Such polymorphisms, known as short tandem repeats (STRs) or microsatellites, presumably are formed by recombination or by slippage of either the template or newly synthesized strand during DNA replication. A useful property of STRs is that different individuals often have different numbers of repeats. The existence of multiple versions of an STR makes it more likely to produce an informative segregation pattern in a given pedigree and therefore to be of more general use in mapping the positions of disease genes. These polymorphisms can be detected by PCR amplification and DNA sequencing.

## Linkage Studies Can Map Disease Genes with a Resolution of About 1 Centimorgan

Without going into all the technical considerations, let's see how the allele conferring a particular dominant disease trait (e.g., familial hypercholesterolemia) might be mapped. The first step is to obtain DNA samples from all the members of a family containing individuals that exhibit the disease. The DNA from each affected and unaffected individual is then analyzed to determine that individual's genotype for a large number of known DNA polymorphisms (either STR or SNP markers can be used). The segregation pattern of each DNA polymorphism within the family is then compared with the segregation pattern of the disease under study. Polymorphisms that are not linked to the disease allele will not show any significant tendency to co-segregate along with the disease, whereas polymorphisms that are closely linked to it will almost always co-segregate with the disease, because recombination events that would separate the disease allele and the polymorphism will be rare. Computer analysis of the segregation data is used to calculate the likelihood of linkage between each DNA polymorphism and the disease-causing allele. Under ideal circumstances, the segregation patterns from at least 10 individuals in a family are needed to establish statistically significant evidence for linkage.

In practice, segregation data are usually collected from different families exhibiting the same disease and pooled. The more families that can be examined, the greater the statistical significance of any evidence for linkage that can be obtained, and the greater the precision with which the distance between a linked DNA polymorphism and a disease
![li1082239539b](li1082239539b.jpg)

EXPERIMENTAL FIGURE 6-33 Single-nucleotide polymorphisms (SNPs) can be followed like genetic markers. A hypothetical pedigree based on SNP analysis of the DNA from a region of a chromosome. In this family, the SNP exists as an A, T, or C nucleotide. Each individual has two alleles: some contain an A on both chromosomes, and others are heterozygous at this site. Circles indicate females; squares indicate males. Blue indicates unaffected individuals; orange indicates individuals with the trait of interest. Analysis reveals that the trait segregates with a $C$ at the SNP.
allele can be measured. Most family studies have a maximum of about 100 individuals in whom linkage between a disease gene and a panel of DNA polymorphisms can be tested. This number of individuals sets the practical upper limit on the resolution of such a mapping study to about 1 centimorgan, or a physical distance of about $7.5 \times 10^{5} \mathrm{bp}$.

A phenomenon called linkage disequilibrium is the basis for an alternative mapping strategy, which can often afford a higher degree of resolution. This approach can be applied to a genetic disease commonly found in a particular population that results from a single mutation that occurred many generations in the past. The DNA polymorphisms carried by the ancestral chromosome in which the mutation occurred are collectively known as the haplotype of that chromosome. As the disease allele is passed from one generation to the next, only the polymorphisms that are closest to the disease gene will not be separated from it by recombination. After many generations, the region that contains the disease gene will be evident because it will be the only region of the chromosome that will carry the haplotype of the ancestral chromosome conserved through many generations (Figure 6-34). By assessing the distribution of specific markers in all affected individuals in a population, geneticists can identify DNA markers tightly associated with the disease, thus localizing the diseaseassociated gene to a relatively small region. Under ideal circumstances, linkage-disequilibrium studies can improve the resolution of mapping studies to less than 0.1 centimorgans. The resolving power of this method comes from its ability to determine whether a polymorphism and the disease allele were ever separated by a meiotic recombination event at any time since the disease allele first appeared on the ancestral chromosome-in some cases, this can amount to finding markers that are so closely linked to the disease gene that even after hundreds of meioses, they have never been separated from it by recombination.

![li1092239539b](li1092239539b.jpg)

FIGURE 6-34 Linkage-disequilibrium studies of human populations can be used to map genes at high resolution. A new disease mutation arises in the context of an ancestral chromosome among a set of polymorphisms known as the haplotype of that chromosome (indicated by red shading; the blue segments of chromosomes represent general haplotypes derived from the general population and not from the ancestral haplotype in which the mutation originally arose). After many generations, chromosomes that carry the disease mutation will also carry segments of the ancestral haplotype that have not been separated from the disease mutation by recombination. The regions closest to the disease mutation are the most likely to be the ancestral haplotype. This phenomenon is known as linkage disequilibrium. The position of the disease mutation can be located by scanning chromosomes that contain it for highly conserved polymorphisms corresponding to the ancestral haplotype.

## Further Analysis Is Needed to Locate a Disease Gene in Cloned DNA

Although linkage mapping can usually locate a human disease gene to a region containing about $10^{3} \mathrm{bp}$, as many as 10 different genes may be located in a region of this size. The ultimate objective of a mapping study is to locate the gene of interest within a cloned segment of DNA and then to determine the nucleotide sequence of this fragment. The relative scales of a chromosomal genetic map and physical maps corresponding to ordered sets of plasmid clones and the nucleotide sequence are shown in Figure 6-35.

One strategy for further localizing a disease gene within the genome is to identify mRNA encoded by DNA in the region under study. Comparison of gene expression in tissues from normal and affected individuals may suggest tissues in which a particular disease gene is normally expressed. For instance, a mutation that phenotypically affects muscle, but no other tissue, might be in a gene that is expressed only in muscle tissue. The expression of mRNA in both normal and affected individuals is generally determined by Northern blotting, microarray analysis, or in situ hybridization
of labeled DNA or RNA to tissue sections. Northern blots, in situ hybridization, or microarray experiments permit comparison of both the level of expression and the sizes of mRNAs in mutant and wild-type tissues. Although the sensitivity of in situ hybridization is lower than that of Northern blot analysis, it can be very helpful in identifying an mRNA that is expressed at low levels in a given tissue but at very high levels in a subclass of cells within that tissue. An mRNA that is altered or missing in various individuals affected with a disease compared with wild-type individuals would be an excellent candidate for encoding the protein whose disrupted function causes that disease.

In many cases, point mutations that give rise to diseasecausing alleles result in no detectable change in the level of expression or electrophoretic mobility of mRNAs. Thus, if comparison of the mRNAs expressed in normal and affected individuals reveals no detectable differences in the candidate mRNAs, a search for point mutations in the DNA regions encoding the mRNAs is undertaken. Now that highly efficient methods for sequencing DNA are available, researchers frequently determine the sequences of candidate regions of DNA isolated from affected individuals to identify point mutations. The overall strategy is to search for a coding sequence that consistently shows possibly deleterious alterations in DNA from individuals that exhibit the disease. A limitation of this approach is that the region near the affected gene may carry naturally occurring polymorphisms unrelated to the gene of interest. Such polymorphisms, not functionally related to the disease, can lead to misidentification of the DNA fragment carrying the gene of interest. For this reason, the more mutant alleles available for analysis, the more likely that a gene will be correctly identified.

## Many Inherited Diseases Result from Multiple Genetic Defects

Most of the inherited human diseases that are now understood at the molecular level are monogenic diseases; that is, a clearly discernible disease state is produced by a defect in a single gene. Monogenic diseases caused by mutation in one specific gene exhibit one of the characteristic inheritance patterns shown in Figure 6-32. The genes associated with most of the common monogenic diseases have already been mapped using DNA-based markers as described previously.

However, many other inherited diseases show more complicated patterns of inheritance, making the identification of the underlying genetic cause much more difficult. One type of added complexity that is frequently encountered is genetic heterogeneity. In such cases, mutations in any one of several different genes can cause the same disease. For example, retinitis pigmentosa, which is characterized by degeneration of the retina usually leading to blindness, can be caused by mutations in any one of more than 60 different genes. In human linkage studies, data from multiple families must usually be combined to determine whether a statistically significant linkage exists between a disease gene and known molecular markers. Genetic heterogeneity such as that exhibited by

![li1102239539b](li1102239539b.jpg)

FIGURE 6-35 Relationship between genetic and physical maps of a human chromosome. The diagram depicts a human chromosome analyzed at different levels of detail. The chromosome as a whole can be viewed in the light microscope when it is in a condensed state that occurs at metaphase, and the approximate location of specific sequences can be determined by fluorescence in situ hybridization
(FISH). At the next level of detail, genetic traits can be mapped relative to DNA-based genetic markers. Local segments of the chromosome can be analyzed at the level of DNA sequences identified by Southern blotting or PCR. Finally, important genetic differences can be most precisely defined by differences in the nucleotide sequence of the chromosomal DNA.
retinitis pigmentosa can confound such an approach because any statistical trend in the mapping data from one family tends to be canceled out by the data obtained from another family with an unrelated causative gene.

Human geneticists used two different approaches to identify the many genes associated with retinitis pigmentosa. The first approach relied on mapping studies of exceptionally large families that contained a sufficient number of affected individuals to provide statistically significant evidence for linkage between known DNA polymorphisms and a single causative gene. The genes identified in such studies showed that several of the mutations that cause retinitis pigmentosa lie within genes that encode proteins that are abundant in the retina. Following up on this clue, geneticists concentrated their attention on those genes that are highly expressed in the retina when screening other individuals with retinitis pigmentosa. This approach of using additional information to focus screening efforts on a subset of candidate genes led to
identification of additional rare causative mutations in many different genes encoding retinal proteins.

A further complication in the genetic dissection of human diseases is posed by diabetes, heart disease, obesity, predisposition to cancer, and a variety of mental disorders that have at least some heritable properties. These and many other diseases can be considered to be polygenic diseases in the sense that alleles of multiple genes, acting together within an individual, contribute to both the occurrence and the severity of disease. How to systematically map complex polygenic traits in humans is one of the most important and challenging problems in human genetics today.

One of the most promising methods of studying diseases that exhibit genetic heterogeneity or are polygenic is to seek a statistical correlation between inheritance of a particular region of a chromosome and the propensity to have a disease using a procedure known as a genome-wide association study (GWAS). The identification of disease-causing genes

by GWAS relies on the phenomenon of linkage disequilibrium described previously. If an allele that causes, or even predisposes an individual to, a disease has originated relatively recently during human evolution, that disease-causing allele will tend to remain associated with the particular set of DNA-based markers in the neighborhood of its chromosomal location. By examining a large number of DNA markers in populations of individuals with a particular disease as well as in control populations of individuals without the disease, researchers can identify chromosomal regions that tend to be correlated with occurrence of the disease. The power of this approach lies in computer algorithms that scan data from large numbers of individuals to identify small but significant correlations between a disease and inheritance of a particular region of the genome. Genomic sequencing and other methods can then be used to identify possible diseasecausing mutations in these regions. In this way, alleles that cause a predisposition to the disease in some, but not necessarily all, individuals with the disease can be identified. Although GWAS can be a powerful tool to identify candidate disease genes, much further work is needed to determine how an individual carrying a particular mutation might be predisposed to the disease.

Models of human disease in experimental organisms may also contribute to unraveling the genetics of complex traits such as obesity or diabetes. For instance, large-scale controlled breeding experiments in mice can identify mouse genes associated with diseases analogous to those in humans. The human orthologs of the mouse genes identified in such studies would be likely candidates for involvement in the corresponding human disease. DNA from human populations can then be examined to determine if particular alleles of the candidate genes show a tendency to be present in individuals affected with the disease but absent from unaffected individuals. This "candidate gene" approach is currently being used intensively to search for genes that may contribute to the major polygenic diseases in humans.

## KEY CONCEPTS OF SECTION 6.4

## Locating and Identifying Human Disease Genes

- Inherited diseases and other traits in humans show three major patterns of inheritance: autosomal dominant, autosomal recessive, and X-linked recessive (see Figure 6-32).
- Genes for human diseases and other traits can be mapped by determining their co-segregation with markers whose locations in the genome are known. The closer a gene is to a particular marker, the more likely they are to co-segregate.
- Mapping of human genes with great precision requires thousands of molecular markers distributed along the chromosomes. The most useful markers are differences in the DNA sequence (polymorphisms) between individuals in noncoding regions of the genome.
- DNA polymorphisms useful in mapping human genes include single-nucleotide polymorphisms (SNPs) and short tandem repeats (STRs).
- Linkage mapping can often locate a human disease gene to a chromosomal region that includes as many as 10 genes. To identify the gene of interest within this candidate region typically requires expression analysis and comparison of DNA sequences between wild-type and disease-affected individuals.
- Some inherited diseases can result from mutations in different genes in different individuals (genetic heterogeneity). The occurrence and severity of other diseases depend on the presence of mutant alleles of multiple genes in the same individual (polygenic traits). The genes associated with such diseases can be mapped by finding a statistical correlation between the disease and a particular chromosomal location in a genome-wide association study.


### 6.5 Inactivating the Function of Specific Genes in Eukaryotes

The elucidation of DNA and protein sequences in recent years has led to the identification of many genes, using sequence patterns in genomic DNA and the sequence similarity of the encoded proteins with proteins of known function. As discussed in Chapter 8, the general functions of proteins identified by sequence searches can be predicted by analogy with known proteins. However, the precise in vivo roles of such "new" proteins may be unclear in the absence of mutant forms of the corresponding genes. In this section, we describe several ways of disrupting the normal function of a specific gene in the genome of an organism. Analysis of the resulting mutant phenotype often helps reveal the in vivo function of the normal gene and its encoded protein.

Three basic approaches underlie these gene inactivation techniques: (1) replacing a normal gene with other sequences, (2) introducing an allele whose encoded protein inhibits the functioning of the expressed normal protein, and (3) promoting destruction of the mRNA transcribed from a gene. The normal endogenous gene is modified in techniques based on the first approach, but is not modified in the other approaches.

Often researchers desire to study the effect of a particular allele of a gene rather than observing the effect of complete inactivation of the gene. For example, testing the effect of a dominant allele of an oncogene on cell division might require replacing one of the normal copies of the gene with that dominant allele. Until recently, this type of precise genome editing was nearly impossible to achieve, but now a variation on a bacterial system for making sequence-specific cuts in phage DNA has been adapted to allow small deletions, or even specific sequence alterations, to be made in cells from a wide variety of organisms, including mammals, as we'll see at the end of this section.

## Normal Yeast Genes Can Be Replaced with Mutant Alleles by Homologous Recombination

Modifying the genome of the yeast S. cerevisiae is particularly easy for two reasons: yeast cells readily take up exogenous DNA under certain conditions, and the introduced DNA is efficiently exchanged for the homologous chromosomal site in the recipient cell. This specific, targeted recombination of identical stretches of DNA allows any gene in yeast chromosomes to be replaced with a mutant allele. (As we saw in Section 6.1, recombination between homologous chromosomes also occurs naturally during meiosis.)

In one popular method for disrupting yeast genes in this fashion, PCR is used to generate a disruption construct containing a selectable marker, which is subsequently transfected into yeast cells. As shown in Figure 6-36a, the two primers for PCR amplification of the selectable marker are each designed to include about 20 nucleotides identical to sequences flanking the yeast gene to be replaced. The resulting amplified construct comprises the selectable marker (e.g., the kanMX gene, which, like neo ${ }^{\mathrm{r}}$, confers resistance to G-418) flanked by about 20 bp at each end that match the ends of the target yeast gene. Transformed diploid yeast cells in which one of the two copies of the target endogenous gene has been replaced by the disruption construct can be identified by their resistance to G-418 or other selectable phenotype. These heterozygous diploid yeast cells generally grow normally regardless of the function of the target gene, but half the haploid spores derived from these cells will carry only the disrupted allele (Figure 6-36b). If a gene is essential for viability, then spores carrying a disrupted allele will not survive.

Disruption of genes by this method is proving particularly useful in assessing the roles of proteins identified by analysis of the entire genomic sequence of $S$. cerevisiae (see Chapter 8). Each of the approximately 6000 genes has been disrupted with the kanMX construct in diploids, and gene disruptions in haploid spores have also been produced. These analyses have shown that about 4500 of the 6000 yeast gene disruptions can reside in viable haploid spores, revealing an unexpectedly large number of apparently nonessential genes. In some cases, disruption of a particular gene may give rise to subtle defects that do not compromise the viability of yeast cells growing under laboratory conditions. Alternatively, cells carrying a disrupted gene may be viable because of the operation of backup or compensatory pathways. To investigate this possibility, yeast geneticists are currently testing all possible double-mutant combinations for synthetic lethal effects that might reveal nonessential genes with redundant functions (see Figure 6-9c).

## Genes Can Be Placed Under the Control of an Experimentally Regulated Promoter

Although disruption of an essential gene required for cell growth will yield nonviable spores, this method provides
(a)
![li1112239539b](li1112239539b.jpg)
(b)
![li1122239539b](li1122239539b.jpg)

EXPERIMENTAL FIGURE 6-36 Homologous recombination with transfected disruption constructs can inactivate specific target genes in yeast. (a) A suitable construct for disrupting a target gene can be prepared using PCR. The two primers designed for this purpose each contain a sequence of about 20 nucleotides (nt) that is homologous to one end of the target yeast gene as well as sequences needed to amplify a segment of DNA carrying a selectable marker gene such as kanMX, which confers resistance to G-418. (b) When recipient diploid Saccharomyces cells are transformed with the disruption construct, homologous recombination between the ends of the construct and the corresponding chromosomal sequences integrates the marker gene into the chromosome, replacing the target-gene sequence. The recombinant diploid cells will grow on a medium containing G-418, whereas untransformed cells will not. If the target gene is essential for viability, half the haploid spores that form after sporulation of recombinant diploid cells will be nonviable.

little information about what the encoded protein actually does in yeast cells. To learn more about how a specific gene contributes to cell growth and viability, investigators must be able to selectively inactivate the gene in a population of growing cells. One method for doing this employs a regulated promoter to selectively shut off transcription of an essential gene.

A useful promoter for this purpose is the yeast GAL1 promoter, which is active in cells grown on galactose but completely inactive in cells grown on glucose. In this approach, the coding sequence of an essential gene (gene X) is ligated to the GAL1 promoter and inserted into a yeast shuttle vector (see Figure 6-15a). The recombinant vector is then introduced into haploid yeast cells in which gene X has been disrupted. Haploid cells that are transformed will grow on galactose medium because the normal copy of gene X on the vector is expressed in the presence of galactose. When the cells are transferred to a glucose-containing medium, however, gene X no longer is transcribed; as the cells divide, the amount of the encoded protein X declines, eventually reaching a state of depletion that mimics a loss-of-function mutation. The observed changes in the phenotype of these cells after the shift to glucose medium may suggest which cellular processes depend on the protein encoded by the essential gene X .

In an early application of this method, researchers explored the function of Hsp70 genes in yeast. Haploid cells with disruptions in all four redundant Hsp70 genes were nonviable unless the cells carried a vector containing a copy of a Hsp70 gene that could be expressed from the GAL1 promoter on galactose medium. On transfer to glucose medium, the vector-carrying cells eventually stopped growing because of insufficient Hsp70 activity. Careful examination of these dying cells revealed that their secretory proteins could no longer enter the endoplasmic reticulum (ER). This study provided the first evidence for the unexpected role of Hsp70 proteins in translocation of secretory proteins into the ER, a process examined in detail in Chapter 13.

## Specific Genes Can Be Permanently Inactivated in the Germ Line of Mice

Many of the methods for disrupting genes in yeast can be applied to the genes of higher eukaryotes. These altered genes can be introduced into the germ line via homologous recombination to produce animals with a gene knockout, or simply "knockout." Knockout mice in which a specific gene is disrupted are powerful experimental systems for studying mammalian development, behavior, and physiology. They are also useful for studying the molecular basis of certain human genetic diseases.

Knockout mice are generated by a two-stage procedure. In the first stage, a DNA construct containing a disrupted allele of a particular target gene is introduced into embryonic stem (ES) cells. These cells, which are derived from the blastocyst, can be grown in culture through many generations
(see Figure 21-7). In a small fraction of transfected cells, the introduced DNA undergoes homologous recombination with the target gene, although recombination at nonhomologous chromosomal sites occurs much more frequently. To enable selection for cells in which homologous recombination and gene-targeted insertion occurs, the recombinant DNA construct introduced into ES cells includes two selectable marker genes (Figure 6-37). One of these genes (neo'), which confers resistance to G-418, is inserted within the target gene (X), thereby disrupting it. The other selectable gene, the thymidine kinase gene from herpes simplex virus $\left(t k^{H S V}\right)$, is inserted into the construct outside the target-gene sequence. ES cells that undergo recombination between the recombinant DNA construct and the homologous site on the chromosome will contain neo' but will not incorporate $t k^{H S V}$. Because $t k^{H S V}$ confers sensitivity to the cytotoxic nucleotide analog ganciclovir, the desired recombinant ES cells can be selected by their ability to survive in the presence of both G-418 and ganciclovir. In these cells, one allele of gene X will be disrupted.

In the second stage in the production of knockout mice, ES cells heterozygous for a knockout mutation in gene X are injected into a recipient wild-type mouse blastocyst, which subsequently is transferred into a pseudopregnant female mouse (Figure 6-38). The resulting progeny will be chimeras, containing tissues derived from both the transplanted ES cells and the host cells. If the ES cells are also homozygous for a visible marker trait (e.g., coat color), then chimeric progeny in which the ES cells have survived and proliferated can be identified easily. Chimeric mice are then mated with mice that are homozygous for another allele of the marker trait to determine if the knockout mutation has been incorporated into the germ line. Finally, mating of mice, each heterozygous for the knockout allele, will produce progeny homozygous for the knockout mutation.

The development of knockout mice that mimic certain human diseases can be illustrated by cystic fibrosis. By the methods discussed in Section 6.4, the recessive mutation that causes this disease was shown to be located in a gene known as CFTR, which encodes a chloride channel. Using the cloned wild-type human CFTR gene, researchers isolated the homologous mouse gene and subsequently introduced mutations in it. The gene-knockout technique was then used to produce homozygous mutant mice, which showed symptoms (i.e., a phenotype), including disturbances to the functioning of epithelial cells, similar to those of humans with cystic fibrosis. These knockout mice are currently being used as a model system for studying this genetic disease and developing effective therapies.

## Somatic Cell Recombination Can Inactivate Genes in Specific Tissues

Investigators are often interested in examining the effects of knockout mutations in a particular tissue of the mouse,

(a) Formation of ES cells carrying a knockout mutation
![li1132239539b](li1132239539b.jpg)
(b) Positive and negative selection of recombinant ES cells
![li1142239539b](li1142239539b.jpg)

ES cells with targeted disruption in gene $X$
at a specific stage in development, or both. However, mice carrying a germ-line knockout may have defects in numerous tissues or die before the developmental stage of interest. To address this problem, mouse geneticists have devised a clever technique to inactivate target genes in specific types of somatic cells or at particular times during development.

This technique employs site-specific DNA recombination sites (called loxP sites) and the enzyme Cre that catalyzes recombination between them. The loxP-Cre recombination system is derived from bacteriophage P1, but this site-specific

EXPERIMENTAL FIGURE 6-37 Isolation of mouse ES cells with a gene-targeted disruption is the first stage in production of knockout mice. (a) When a recombinant DNA construct is introduced into embryonic stem (ES) cells, random insertion via nonhomologous recombination occurs much more frequently than gene-targeted insertion via homologous recombination. Recombinant cells in which one allele of gene $X$ (orange and white) is disrupted can be obtained by using a recombinant vector that carries gene $X$ disrupted with neo ${ }^{\prime}$ (green), which confers resistance to G-418, and, outside the region of homology, $t k^{H S V}$ (yellow), the thymidine kinase gene from herpes simplex virus. The viral thymidine kinase, unlike the endogenous mouse enzyme, can convert the nucleotide analog ganciclovir into the monophosphate form; this is then modified to the triphosphate form, which inhibits cellular DNA replication in ES cells. Thus ganciclovir is cytotoxic for recombinant ES cells carrying the $t k^{H S V}$ gene. Nonhomologous insertion includes the $t k^{H S V}$ gene, whereas homologous insertion does not; therefore, only cells with nonhomologous insertion are sensitive to ganciclovir. (b) Recombinant cells are selected by treatment with G-418, since cells that fail to pick up the construct or integrate it into their genome are sensitive to this cytotoxic compound. The surviving recombinant cells are treated with ganciclovir. Only cells with a targeted disruption in gene $X$, and therefore lacking the $t k^{H S V}$ gene and its accompanying cytotoxicity, will survive. See S. L. Mansour et al., 1988, Nature 336:348.
recombination system also functions when placed in mouse cells. An essential feature of this technique is that expression of Cre is controlled by a cell-type-specific promoter. In loxPCre mice generated by the procedure depicted in Figure 6-39, inactivation of the gene of interest ( X ) occurs only in cells in which the promoter controlling the cre gene is active.

An early application of this technique provided strong evidence that a particular neurotransmitter receptor is important for learning and memory. Previous pharmacological and physiological studies had indicated that normal learning requires the NMDA class of glutamate receptors in the hippocampus, a region of the brain. But mice in which the gene encoding an NMDA receptor subunit was knocked out died neonatally, precluding analysis of the receptor's role in learning. Following the protocol in Figure 6-39, researchers generated mice in which the receptor subunit gene was inactivated in the hippocampus but expressed in other tissues. These mice survived to adulthood and showed learning and memory defects, confirming a role for these receptors in the ability of mice to encode their experiences into memory.

## Dominant-Negative Alleles Can Inhibit the Function of Some Genes

In diploid organisms, as noted in Section 6.1, the phenotypic effect of a recessive allele is expressed only in homozygous individuals, whereas dominant alleles are expressed in heterozygotes. Thus an individual must carry two copies of a recessive allele, but only one copy of a dominant allele, to exhibit the corresponding phenotypes.

![li1152239539b](li1152239539b.jpg)

We have seen how strains of mice that are homozygous for a given recessive knockout mutation can be produced by crossing individuals that are heterozygous for the same knockout mutation (see Figure 6-38). In experiments with cultured animal cells, however, it is usually difficult to disrupt both copies of a gene in order to produce a mutant phenotype. Moreover, the difficulty of producing strains with both copies of a gene mutated is often compounded by the presence of related genes of similar function that

EXPERIMENTAL FIGURE 6-38 ES cells heterozygous for a disrupted gene are used to produce knockout mice. Step $\mathbf{\square}$ Embryonic stem (ES) cells heterozygous for a knockout mutation in a gene of interest $(X)$ and homozygous for a dominant allele of a marker gene (here, brown coat color, A) are transplanted into the blastocoel cavity of 4.5-day blastocysts that are homozygous for a recessive allele of the marker (here, black coat color, a). Step $\mathbf{\square}$ The early embryos are then implanted into a pseudopregnant female. Those progeny containing ES-derived cells are chimeras, as indicated by their mixed black and brown coats. Step $\mathbf{\square}$ Chimeric mice are then backcrossed to black mice; brown progeny from this mating have ES-derived cells in their germ line. Steps $\mathbf{\square} \mathbf{\square}$ Analysis of DNA isolated from a small amount of tail tissue can identify brown mice heterozygous for the knockout allele. Intercrossing of these mice produces some individuals homozygous for the disrupted allele-that is, knockout mice. See M. R. Capecchi, 1989, Trends Genet. 5:70.
must also be inactivated in order to reveal an observable phenotype.

For certain genes, the difficulties of producing homozygous knockout mutants can be avoided by use of an allele carrying a dominant-negative mutation. These alleles are genetically dominant; that is, they produce a mutant phenotype even in cells carrying a wild-type copy of the gene. However, unlike other types of dominant alleles, dominantnegative alleles produce a phenotype equivalent to that of a loss-of-function mutation.

Useful dominant-negative alleles have been identified for a variety of genes and can be introduced into cultured cells by transfection or into the germ line of mice or other organisms. In both cases, the introduced gene is integrated into the genome by nonhomologous recombination. Such randomly inserted genes are called transgenes; the cells or organisms carrying them are referred to as transgenic. Transgenes carrying a dominant-negative allele are usually engineered so that the allele is controlled by a regulated promoter, which allows expression of the mutant protein in particular tissues or at particular times. As noted above, the random integration of exogenous DNA via nonhomologous recombination occurs at a much higher frequency than insertion via homologous recombination. Therefore, the production of transgenic mice is an efficient and straightforward process (Figure 6-40).

Among the genes that can be functionally inactivated by introduction of a dominant-negative allele are those encoding small (monomeric) GTP-binding proteins belonging to the GTPase superfamily. As we will see in several later chapters, these proteins (e.g., Ras, Rac, and Rab) act as intracellular switches. Conversion of these small GTPases from an inactive GDP-bound state to an active GTP-bound state depends on their interacting with a corresponding guanine nucleotide exchange factor (GEF). A mutant small GTPase that permanently binds to the GEF protein will block conversion of endogenous wild-type small GTPases to the active

![li1162239539b](li1162239539b.jpg)

EXPERIMENTAL FIGURE 6-39 The loxP-Cre recombination system can knock out genes in specific cell types. A loxP site (purple) is inserted on each side of an essential exon (exon 2) of the target gene (gene $X$; blue) by homologous recombination, producing a loxP mouse. Since the loxP sites are in introns, they do not disrupt the function of $X$. A Cre mouse carries one gene $X$ knockout allele and an introduced cre gene (orange) from bacteriophage P1 linked to a cell-type-specific promoter (yellow). The cre gene is incorporated into the mouse genome by nonhomologous recombination and does not affect the function of other genes. In the loxP-Cre mice that result from crossing these two types of mice, Cre protein is produced only in those cells in which the promoter is active. Thus these are the only cells in which recombination between the loxP sites catalyzed by Cre occurs, leading to deletion of exon 2. Since the other allele is a constitutive gene $X$ knockout, deletion between the loxP sites results in complete loss of function of gene $X$ in all cells expressing Cre. By using different promoters, researchers can study the effects of knocking out gene $X$ in various types of cells.

## Inject foreign DNA into one of the pronuclei

![li1172239539b](li1172239539b.jpg)

Fertilized mouse egg prior to fusion of male and female pronuclei
![li1182239539b](li1182239539b.jpg)

About 10-30\% of offspring will contain foreign DNA in chromosomes of all their tissues and germ line
![li1192239539b](li1192239539b.jpg)

GTP-bound state, thereby inhibiting them from performing their switching function (Figure 6-41).

## RNA Interference Causes Gene Inactivation by Destroying the Corresponding mRNA

The phenomenon known as RNA interference provides what may be the most straightforward method to inhibit the function of specific genes. This approach to disrupting genes is technically simpler than the methods described above. First observed in the roundworm C. elegans, RNA interference (RNAi) refers to the ability of double-stranded RNA to block expression of its corresponding single-stranded mRNA, but not that of mRNAs with a different sequence.

As described in Chapter 10, the phenomenon of RNAi rests on the general ability of eukaryotic cells to cleave double-stranded RNA into short (23-nt) double-stranded segments known as small inhibitory RNAs (siRNAs). The RNA endonuclease that catalyzes this reaction, known as Dicer, is found in all metazoans, but not in simpler

EXPERIMENTAL FIGURE 6-40 Transgenic mice are produced by random integration of a foreign gene into the mouse germ line. Foreign DNA injected into one of the two pronuclei (the male and female haploid nuclei contributed by the parents) has a good chance of being randomly integrated into the chromosomes of the diploid zygote. Because a transgene is integrated into the recipient genome by nonhomologous recombination, it does not disrupt endogenous genes. See R. L. Brinster et al., 1981, Cell 27:223.

(a) Cells expressing only wild-type alleles of a small GTPase

Wild type
(b) Cells expressing both wild-type alleles and a dominant-negative allele

Dominant-negative mutant
![li1202239539b](li1202239539b.jpg)

FIGURE 6-41 Inactivation of the function of a wild-type GTPase by the action of a dominant-negative mutant allele. (a) Small (monomeric) GTPases (purple) are activated by their interaction with a guanine nucleotide exchange factor (GEF), which catalyzes the exchange of GDP for GTP. (b) Introduction of a dominant-negative allele of a small GTPase gene into cultured cells or transgenic animals leads to expression of a mutant GTPase that binds to and inactivates the GEF. As a result, endogenous wild-type copies of the same small GTPase are trapped in the inactive GDP-bound state. A single dominant-negative allele thus causes a loss-of-function phenotype in heterozygotes similar to that seen in homozygotes carrying two recessive loss-of-function alleles.
eukaryotes such as yeast. The siRNA molecules, in turn, can cause cleavage of mRNA molecules of matching sequence, in a reaction catalyzed by a protein complex known as RISC. RISC mediates recognition and hybridization between one strand of the siRNA and its complementary sequence on the target mRNA; subsequently, specific nucleases in the RISC complex cleave the mRNA-siRNA hybrid. This model accounts for the specificity of RNAi, since it depends on base pairing, and for its potency in silencing gene function, since the complementary mRNA is permanently destroyed by nucleolytic degradation. The normal function of both Dicer and RISC is to allow for gene regulation by small endogenous RNA molecules known as micro-RNAs, or miRNAs.

Researchers can exploit the micro-RNA pathway to intentionally silence a C. elegans gene of interest by using either of two general methods for generating siRNAs of defined sequence. In the first method, a double-stranded RNA corresponding to the target-gene sequence is produced by in vitro transcription of both sense and antisense copies of that sequence (Figure 6-42a). This dsRNA is then injected into the gonad of an adult worm, where it is converted to siRNA by Dicer in the developing embryos. In conjunction with the RISC complex, the siRNA molecules cause the corresponding mRNA molecules to be destroyed rapidly. The resulting worms display a phenotype similar to the one that would result from disruption of the target gene itself. In some cases, insertion of just a few molecules of a particular dsRNA into a cell is sufficient to inactivate many copies of the corresponding mRNA. Figure 6-42b illustrates the ability of an
(a) In vitro production of double-stranded RNA
![li1212239539b](li1212239539b.jpg)
(b)
![li1222239539b](li1222239539b.jpg)
(c) In vivo production of double-stranded RNA
![li1232239539b](li1232239539b.jpg)

EXPERIMENTAL FIGURE 6-42 RNA interference (RNAi) can inhibit gene function in C. elegans and other organisms. (a) In vitro production of double-stranded RNA (dsRNA) for interference with a specific target gene. The coding sequence of the gene, derived from either a cDNA clone or a segment of genomic DNA, is placed in two orientations in a plasmid vector adjacent to a strong promoter. Transcription of both constructs in vitro using RNA polymerase and ribonucleoside triphosphates yields many RNA copies in both the sense orientation (identical to the mRNA sequence) and the complementary antisense orientation. Under suitable conditions, these complementary RNA molecules hybridize to form dsRNA. When the dsRNA is injected into cells, it is cleaved by Dicer into siRNAs. (b) Inhibition of mex3 RNA expression in C. elegans embryos by RNAi (see the text for the mechanism). Expression of mex3 RNA in embryos was assayed by in situ hybridization to a probe specific for this mRNA, linked to an enzyme that produces a colored (purple) product. (Left) Wild-type embryo. (Right) Embryo derived from a worm injected with double-stranded mex3 RNA. Each four-cell-stage embryo is $\sim 50 \mathrm{~mm}$ in length. (c) In vivo production of double-stranded RNA via an engineered plasmid introduced directly into cells. The synthetic gene construct is a tandem arrangement of both sense and antisense sequences of the target gene. When it is transcribed, doublestranded small hairpin RNA (shRNA) forms. The shRNA is cleaved by Dicer to form siRNA. [Part (b) reprinted by permission from Macmillan Publishers Ltd: from, Fire, A., "Potent and specific genetic interference by double-stranded RNA in Caenorhabditis elegans," Nature, 1998, 391(6669):806-811; permission conveyed through Copyright Clearance Center, Inc.]

injected dsRNA to interfere with the production of the corresponding endogenous mRNA in C. elegans embryos. In this experiment, the mRNA levels in embryos were determined by in situ hybridization, as described earlier, using a fluorescently labeled probe.

The second method is to produce a specific doublestranded RNA in vivo. An efficient way to do this is to express a synthetic gene that is designed to contain tandem segments of both sense and antisense sequences corresponding to the target gene (Figure 6-42c). When this gene is transcribed, a double-stranded RNA "hairpin" structure forms, known as small hairpin RNA, or shRNA. The shRNA will then be cleaved by Dicer to form siRNA molecules. The lentiviral expression vectors are particularly useful for introducing synthetic genes for the expression of shRNA constructs into animal cells.

Both RNAi methods lend themselves to systematic studies in which researchers inactivate each of the known genes in an organism and observe what goes wrong. For example, in initial studies with C. elegans, RNA interference with 16,700 genes (about 86 percent of the genome) yielded 1722 visibly abnormal phenotypes. The genes whose functional inactivation causes particular abnormal phenotypes can be grouped into sets; each member of a set presumably controls the same signals or events. The regulatory relations between the genes in a set-for example, the genes that control muscle development-can then be worked out.

RNAi-mediated gene inactivation has been successful in many organisms, including Drosophila, many kinds of plants, zebrafish, the frog Xenopus, and mice, and these organisms are now the subjects of large-scale RNAi screens. For example, lentiviral vectors have been designed to inactivate by RNAi more than 10,000 different genes expressed in cultured mammalian cells. The functions of the inactivated genes can be inferred from defects in the growth or morphology of cell clones transfected with lentiviral vectors.

## Engineered CRISPR-Cas9 Systems Allow Precise Genome Editing

A recently developed technique allows precise alterations to be made in genomic DNA sequences by a process known as genome editing. This method is based on a natural mechanism that evolved to protect bacterial cells against foreign DNA such as phage DNA. This mechanism, called CRISPR (clustered regularly interspaced short palindromic repeats), is named after the curious arrays of tandem repeated sequences found in about half of the bacterial genomes that have been sequenced. The arrays of repeated sequences are flanked by a conserved set of genes, known as Cas (CRISPR-associated) genes, that show similarity to genes encoding nucleases.

The breakthrough in understanding the function of CRISPR sequences came from the observation that one set of repeated sequence elements in the array often matched short segments in phage genomes. It was subsequently shown that the CRISPR element and associated Cas genes confer on bacterial cells the ability to cleave phage DNA at precisely the
site that corresponds to repeated sequences in the CRISPR array. The acquisition of immunity to a phage by means of CRISPR takes place in two stages. In the first stage, a phageinfected bacterium carrying a CRISPR system cleaves the phage DNA into short segments and adds those segments to the CRISPR array so that they are interspersed with highly conserved repeats. In the second stage, transcription of the CRISPR array and processing of the resulting RNA yields mature bipartite RNA molecules that carry both the conserved repeat sequences and a phage-derived spacer sequence. The repeat sequence assembles with a second RNA molecule called tracrRNA that provides a scaffold for binding to Cas proteins to form an interference complex, and the phage-derived sequence guides the complex to a specific target sequence through base pairing. Once targeted to a specific DNA sequence, nucleases in the Cas proteins cleave both strands of the target DNA molecule at a site adjacent to the region base-paired with the guide sequence.

Although CRISPR elements have been found only in prokaryotic organisms, researchers proposed that the guide RNA and Cas proteins should be able to perform their functions if expressed in eukaryotic cells. To adapt CRISPR to function in virtually any cell type, minimal systems consisting of the nuclease Cas 9 and an engineered guide RNA have been developed (Figure 6-43a). Cas9 contains all the enzymatic activities necessary for genome editing, including two separate endonuclease activities-one for each strand of DNA. The guide RNA is composed of two regions: the first is made up of two complementary sequences that form a double-stranded hairpin structure designed to bind to Cas9, and the second region of about 20 nt provides the targeting activity and is designed to perfectly match a specific site in genomic DNA. When both Cas9 and the guide RNA are expressed in a recipient cell, Cas9 will cleave both DNA strands at the chromosomal site specified by the guide RNA sequence (Figure 6-43b,c). Transfection with expression plasmids for Cas9 and a specific guide RNA has been shown to give rise to specific DNA cleavages in cells derived from a variety of organisms, including Drosophila, C. elegans, zebrafish, mouse, rat, and human. A particularly efficient way to modify the germ line of an organism begins with microinjecting Cas 9 mRNA and a guide RNA into a mouse zygote. The resulting double-strand break in the specific target sequence is typically repaired by a set of enzymes that ligate the free DNA ends back together in a process known as nonhomologous end joining (Figure 6-43d). Typically, a few base pairs are deleted at the site of cleavage because nucleases can remove bases from the free DNA ends before the end joining reaction is completed. If the cleavage site is within a gene coding sequence, the deletion of several bases will usually cause a frameshift mutation, thus inactivating the targeted gene.

An important refinement of this method of genome editing is the addition of a segment of DNA, typically about 100 nt long, that matches the sequences flanking the cleavage site. When homologous DNA is present, the free ends can be repaired by homologous recombination using the added

![li1242239539b](li1242239539b.jpg)

FIGURE 6-43 Single-nucleotide mutations can be introduced into the genome using an engineered CRISPR-Cas9 system. (a) The genome of a target cell can be modified by expression of the double-stranded DNA endonuclease Cas9 and a guide RNA. Expression of these components can be achieved by transfection with plasmids carrying genes for Cas9 and the guide RNA or by direct injection of Cas9 mRNA and guide RNA. The guide RNA is composed of two parts: a sequence that folds into a hairpin scaffold structure that binds to Cas9, and a sequence of approximately 20 nt corresponding to the targeted site in the genome. Expression of these components can be achieved by transfection with plasmids carrying genes for Cas9 and the guide RNA or by direct injection of Cas9 mRNA and guide RNA. (b) A complex of guide RNA bound to Cas9 is targeted to the genome by base pairing of the guide RNA with the complementary genomic DNA sequence. This structure allows the two distinct nuclease active sites of Cas9 to cleave both strands of the target DNA adjacent to the heteroduplex formed with the guide RNA. (c) By this mechanism, the expression of both Cas9 and a bipartite guide RNA designed to target a specific gene sequence leads to a double-strand cleavage of the target gene. (d) Cleaved DNA can be repaired via a nonhomologous end joining (NHEJ) process, which usually removes a small number of bases at the cleavage site. If the cleavage occurs in a coding sequence, NHEJ will usually inactivate gene function by producing a frameshift mutation. If a $\sim 100$-nt single-stranded DNA segment that spans the sequences flanking the cleavage site is injected along with Cas9 mRNA and the guide RNA, the cleaved DNA can be repaired by homologous recombination (homology-directed repair, HDR). By this mechanism, single base changes can be introduced into the repaired genomic DNA. [Part (b) data from C. Anders et al., 2014, Nature 513:569-573, PDB ID 4un3.]

DNA segment (see Figure 6-43d). By appropriate design of the guide RNA to specify the cleavage site, and by introduction of appropriate sequence alterations in the added homologous DNA segment, exact single-base changes in the target DNA can be produced. In a dramatic demonstration of this genome editing method, a single-base mutation in Crygc, a gene for gamma-crystallin, that leads to the formation of cataracts was corrected back to the wild-type form by injection of a mutant mouse zygote with Cas 9 mRNA, a guide RNA that directs cleavage at the site of the mutation, and a $90-\mathrm{nt}$ DNA segment spanning the cleavage site and containing the wild-type Crygc sequence (see the chapter-opening photo). This experiment also illustrates the remarkable selectivity of CRISPR-Cas9 targeting, since correction of the mutant allele requires that the guide RNA discriminate between the mutant and wild-type alleles, which differ by only one base pair.

In Section 6.4, we saw that human genetic mapping and association studies have identified a large number of genetic variants, many representing single-nucleotide polymorphisms that predispose people to one or another inherited diseases. The ability to use CRISPR-Cas9 systems to produce specific single-base changes in the mammalian germ line now opens the possibility of creating the equivalent alleles in a well-controlled experimental organism such as the mouse.

## KEY CONCEPTS OF SECTION 6.5

## Inactivating the Function of Specific Genes in Eukaryotes

- Once a gene has been cloned, important clues about its normal function in vivo can be deduced from the observed phenotypic effects of mutating the gene.
- Genes can be disrupted in yeast by inserting a selectable marker gene into one copy of a wild-type gene via homologous recombination, producing a heterozygous mutant. When such a heterozygote forms spores, two nonviable haploid spores will be produced (see Figure 6-36).
- A yeast gene can be inactivated in a controlled manner by using the GAL1 promoter to shut off transcription of a gene when cells are transferred to glucose medium.
- In mice, modified genes can be incorporated into the germ line at their original genomic location by homologous recombination, producing knockouts (see Figures 6-38 and 6-39). Knockout mice can provide models for human genetic diseases such as cystic fibrosis.
- The loxP-Cre recombination system permits production of mice in which a gene is knocked out in a specific tissue.
- In the production of transgenic cells or organisms, exogenous DNA is integrated into the host genome by nonhomologous recombination (see Figure 6-40). Introduction of a dominant-negative allele in this way can functionally inactivate a gene without altering its sequence.
- In many organisms, including the roundworm C. elegans, double-stranded RNA triggers destruction of the all the mRNA molecules with the same sequence (see Figure 6-42). This phenomenon, known as RNAi (RNA interference), provides a specific and potent means of functionally inactivating genes without altering their structure.
- A bacterial system that evolved to precisely target and cleave foreign DNA, known as CRISPR, has been adapted for use in many organisms to enable specific changes to be introduced into genomic DNA. Cleavage of chromosomal DNA at a specific site by CRISPR-Cas9 usually results in a short deletion at the cleavage site. If an appropriately designed DNA segment is provided by transfection, specific DNA changes such as point mutations can be introduced at the cleavage site (see Figure 6-43).


## LaunchPad

macmillan learning
Visit LaunchPad to access study tools and to learn more about the content in this chapter.

- Perspectives for the Future
- Analyze the Data
- Extended References
- Additional study tools, including videos, animations, and quizzes


## Key Terms

alleles 224
clone 237
complementary DNAs
(cDNAs) 239
CRISPR 266
DNA cloning 234
DNA library 237
DNA microarray 247
dominant 225
functional complementation 237
gene knockout 261
genotype 224
heterozygous 225
homozygous 225
hybridization 246
in situ hybridization 247
linkage 232
mutagen 224
mutation 224

Northern blotting 247
phenotype 224
plasmids 236
point mutation 225
polymerase chain reaction
(PCR) 239
recessive 225
recombinant DNA 234
recombination 232
restriction enzymes 234
RNA interference
(RNAi) 264
segregation 227
Southern blotting 246
transfection 251
transformation 236
transgenes 263
vector 234
wild type 224

## Review the Concepts

1. Genetic mutations can provide insights into the mechanisms of complex cellular or developmental processes. How might your analysis of a genetic mutation be different depending on whether a particular mutation is recessive or dominant?
2. What is a temperature-sensitive mutation? Why are temperature-sensitive mutations useful for uncovering the function of a gene?
3. Describe how complementation analysis can be used to reveal whether two mutations are in the same or in different genes. Explain why complementation analysis will not work with dominant mutations.
4. Jane has isolated a mutant strain of yeast that forms red colonies instead of the normal white when grown on a plate. To determine the mutant gene, she decides to use functional complementation with a DNA library containing a lysine selection marker. In addition to the unknown gene mutation, the yeast are lacking the gene required to synthesize the amino acids leucine and lysine. What media will Jane grow her yeast on to ensure that they have acquired the library plasmids? How will she know when a library plasmid has complemented her yeast mutation?
5. Restriction enzymes and DNA ligase play essential roles in DNA cloning. How is it that a bacterium that produces a restriction enzyme does not cut its own DNA? Describe some general features of restriction sites. What are the three types of DNA ends that can be generated after cutting DNA with restriction enzymes? What reaction is catalyzed by DNA ligase?
6. Bacterial plasmids often serve as cloning vectors. Describe the essential features of a plasmid vector. What are the advantages and applications of plasmids as cloning vectors? 7. A DNA library is a collection of clones, each containing a different fragment of DNA, inserted into a cloning vector. What is the difference between a cDNA library and a genomic DNA library? Suppose you would like to clone gene X , a gene expressed only in neurons, into a vector using a library as the source of the insert. You have the following libraries at your disposal: a genomic library from skin cells, a cDNA library from skin cells, a genomic library from neurons, and a cDNA library from neurons. Which could you use, and why?
7. In 1993, Kary Mullis won the Nobel Prize in Chemistry for his invention of the PCR process. Describe the three steps in each cycle of a PCR reaction. Why was the discovery of a thermostable DNA polymerase so important for the development of PCR?
8. Southern and Northern blotting are powerful molecular biological tools based on hybridization of nucleic acids. How are these techniques the same? How do they differ? Give some specific applications for each blotting technique. 10. A number of foreign proteins have been expressed in bacterial and mammalian cells. Describe the essential features of a recombinant plasmid that are required for
expression of a foreign gene. How can the foreign protein be modified to facilitate its purification? What is the advantage of expressing a protein in mammalian cells versus bacteria?
9. Northern blotting, RT-PCR, and microarrays can be used to analyze gene expression. A lab studies yeast cells, comparing their growth in two different sugars, glucose and galactose. One student is comparing expression of the gene HMG2 under these two conditions. Which technique(s) could he use and why? Another student wants to compare expression of all the genes on chromosome 4 , of which there are approximately 800 . What technique(s) could she use and why?
10. In determining the identity of the protein that corresponds to a newly discovered gene, it often helps to know the pattern of tissue expression for that gene. For example, researchers have found that a gene called SERPINA6 is expressed in the liver, kidney, and pancreas but not in other tissues. What techniques might researchers use to find out which tissues express a particular gene?
11. DNA polymorphisms can be used as DNA markers. Describe the differences between SNPs and STR polymorphisms. How can these markers be used for DNA-mapping studies?
12. How can linkage-disequilibrium mapping sometimes provide a much higher resolution of gene location than classical linkage mapping?
13. Genetic linkage studies can usually locate the chromosomal position of a disease gene only roughly. How can expression analysis and DNA sequence analysis help locate a disease gene within the region identified by linkage mapping?
14. The ability to selectively modify the genome in the mouse has revolutionized mouse genetics. Outline the procedure for generating a knockout mouse at a specific genetic locus. How can the loxP-Cre system be used to conditionally knock out a gene? What is an important medical application of knockout mice?
15. Two methods for functionally inactivating a gene without altering the gene sequence involve dominant-negative alleles and RNA interference (RNAi). Describe how each method can inhibit expression of a gene.

## References

Genetic Analysis of Mutations to Identify and Study Genes
Hartwell, L. H. 1967. Macromolecular synthesis of temperature-sensitive mutants of yeast. J. Bacteriol. 93:1662.

Nüsslein-Volhard, C., and E. Wieschaus. 1980. Mutations affecting segment number and polarity in Drosophila. Nature 287:795-801.

Tong, A. H., et al. 2001. Systematic genetic analysis with ordered arrays of yeast deletion mutants. Science 294:2364-2368.

## DNA Cloning and Characterization

Maniatis, T., et al. 1978. The isolation of structural genes from libraries of eucaryotic DNA. Cell 15:687-701.

Nasmyth, K. A., and S. I. Reed. 1980. Isolation of genes by complementation in yeast: molecular cloning of a cell-cycle gene. P. Natl. Acad. Sci. USA 77:2119-2123.

Nathans, D., and H. O. Smith. 1975. Restriction endonucleases in the analysis and restructuring of DNA molecules. Annu. Rev. Biochem. 44:273-293.

## Using Cloned DNA Fragments to Study Gene Expression

Pellicer, A., et al. 1978. The transfer and stable integration of the HSV thymidine kinase gene into mouse cells. Cell 41:133-141.

Saiki, R. K., et al. 1988. Primer-directed enzymatic amplification of DNA with a thermostable DNA polymerase. Science 239:487-491.

## Locating and Identifying Human Disease Genes

Botstein, D., et al. 1980. Construction of a genetic linkage map in man using restriction fragment length polymorphisms. Am. J. Genet. 32:314-331.

Donis-Keller, H., et al. 1987. A genetic linkage map of the human genome. Cell 51:319-337.

Hastbacka, T., et al. 1994. The diastrophic dysplasia gene encodes a novel sulfate transporter: positional cloning by fine-structure linkage disequilibrium mapping. Cell 78:1073.

Inactivating the Function of Specific Genes in Eukaryotes
Capecchi, M. R. 1989. Altering the genome by homologous recombination. Science 244:1288-1292.

Fire, A., et al. 1998. Potent and specific genetic interference by double-stranded RNA in Caenorbabditis elegans. Nature 391: 806-811.

Jinek, M., et al. 2012. A programmable dual-RNA-guided DNA endonuclease in adaptive bacterial immunity. Science 337: 816-821.

Zamore, P. D., et al. 2000. RNAi: double-stranded RNA directs the ATP-dependent cleavage of mRNA at 21 to 23 nucleotide intervals. Cell 101:25-33.

Zimmer, A. 1992. Manipulating the genome by homologous recombination in embryonic stem cells. Annu. Rev. Neurosci. 15:115.

![li1252239539b](li1252239539b.jpg)

Molecular model of a lipid bilayer with embedded membrane proteins. Integral membrane proteins have distinct exoplasmic, cytosolic, and membrane-spanning domains. Shown here are portions of the insulin receptor, which regulates cell metabolism. [Ramon Andrade 3Dciencia/ Science Photo Library/Science Source.]

Membranes participate in many aspects of cell structure and function. The plasma membrane defines the cell and separates the inside from the outside. In eukaryotes, membranes also define intracellular organelles such as the nucleus, mitochondrion, and lysosome. These biomembranes all have the same basic architecture: a phospholipid bilayer in which proteins are embedded (Figure 7-1). By preventing the unassisted movement of most water-soluble substances across the membrane, the phospholipid bilayer serves as a permeability barrier, helping to maintain the characteristic differences between the inside and the outside of the cell or organelle; in turn, the embedded proteins endow the membrane with specific functions, such as regulated transport of substances from one side to the other. Each cellular membrane has its own set of proteins that allow it to carry out many different functions.

Prokaryotes, the simplest and smallest cells, are about $1-2 \mu \mathrm{~m}$ in length and are surrounded by a single plasma membrane; in most cases, they contain no internal membranelimited subcompartments (see Figure 1-11). However, this single plasma membrane contains hundreds of different types of proteins that are integral to the function of the cell.

## Biorembrane Structure

Some of these proteins catalyze ATP synthesis and initiation of DNA replication, for instance. Others include the many types of membrane transport proteins that enable specific ions, sugars, amino acids, and vitamins to cross the otherwise impermeable phospholipid bilayer to enter the cell and that allow specific metabolic products to exit. Receptor proteins in the plasma membrane are proteins that allow the cell to recognize chemical signals present in its environment and adjust its metabolism or pattern of gene expression in response.

Eukaryotes also have a plasma membrane studded with many different proteins that perform a variety of functions, including membrane transport, cell signaling, and connecting cells into tissues. In addition, eukaryotic cells-which are generally much larger than prokaryotes-have a variety of internal membrane-bounded organelles (see Figure 1-12). Each type of organelle membrane has a unique complement of proteins that enable it to carry out its characteristic cellular functions, such as ATP generation (in mitochondria) and DNA synthesis (in the nucleus). Many plasma-membrane proteins also bind components of the cytoskeleton, a dense network of protein filaments that crisscrosses the cytosol to

### 7.1 The Lipid Bilayer: Composition and Structural Organization

### 7.2 Membrane Proteins: Structure and Basic Functions

### 7.3 Phospholipids, Sphingolipids, and Cholesterol: Synthesis and Intracellular Movement

![li1262239539b](li1262239539b.jpg)

FIGURE 7-1 Fluid mosaic model of biomembranes. A bilayer of phospholipids about 3 nm thick provides the basic architecture of all cellular membranes; membrane proteins give each cellular membrane its unique set of functions. Individual phospholipids can move laterally and spin within the plane of the membrane, giving the membrane a fluidlike consistency similar to that of olive oil. Noncovalent interactions between phospholipids, and between phospholipids and proteins, lend strength and resilience to the membrane, while the hydrophobic core of the bilayer prevents the
unassisted movement of water-soluble substances from one side to the other. Integral membrane proteins (transmembrane proteins) span the bilayer and often form dimers and higher-order oligomers. Lipid-anchored proteins are tethered to one leaflet by a covalently attached hydrocarbon chain. Peripheral proteins associate with the membrane primarily by specific noncovalent interactions with integral membrane proteins or membrane lipids. Proteins in the plasma membrane also make extensive contact with the cytoskeleton. See D. Engelman, 2005, Nature 438:578-580.
provide mechanical support for cellular membranes. These interactions are essential for the cell to assume its specific shape and for many types of cell movements.

Despite playing a structural role in cells, membranes are not rigid structures. They can bend and flex in three dimensions while still maintaining their integrity, in part because of the abundant noncovalent interactions that hold the lipids and proteins together. Moreover, there is considerable mobility of individual lipids and proteins within the plane of the membrane. According to the fluid mosaic model of biomembranes, first proposed by researchers in the 1970s, the phospholipid bilayer behaves in some respects like a two-dimensional fluid, with individual lipid molecules able to move past one another as well as spin in place. Such fluidity and flexibility not only allows organelles to assume their typical shapes, but also confers on the membrane the dynamic property that enables membrane budding and fusion, as occurs when viruses are released from an infected cell (Figure 7-2a) or when the internal cellular membranes of the

## FIGURE 7-2 Eukaryotic cellular membranes are dynamic

structures. (a) An electron micrograph of the plasma membrane of an HIV-infected cell, showing HIV particles budding into the culture medium. As the virus core buds from the cell, it becomes enveloped by a membrane, derived from the cell's plasma membrane, that contains specific viral proteins. (b) Stacked membranes of the Golgi complex with budding vesicles. Note the irregular shape and curvature of these membranes. [Part (a) from W. Sundquist and U. von Schwedler, University of Utah; part (b) Biology Pics/Science Source.]
![li1272239539b](li1272239539b.jpg)
(b)
![li1282239539b](li1282239539b.jpg)

Golgi complex bud into vesicles in the cytosol (Figure 7-2b) and then fuse with other membranes to transport their contents from one organelle to another (see Chapter 14).

We begin our examination of biomembranes by considering their lipid components. These molecules not only affect membrane shape and function, but also help anchor proteins to the membrane, modify membrane protein activities, and transduce signals to the cytoplasm. We then consider the structure of membrane proteins. Many of these proteins have large segments that are embedded in the hydrocarbon core of the phospholipid bilayer, and we focus on the principal classes of such transmembrane proteins. Finally, we consider how lipids such as phospholipids and cholesterol are synthesized in cells and distributed to their many membranes and organelles.

### 7.1 The Lipid Bilayer: Composition and Structural Organization

In Chapter 2, we learned that phospholipids are the principal building blocks of biomembranes. The most common phospholipids in membranes are the phosphoglycerides (see Figure 2-20), but as we will see in this chapter, there are multiple types of phospholipids. All phospholipids are amphipathic molecules that consist of two segments with very different chemical properties: a fatty acid-based (fatty acyl) hydrocarbon "tail" that is hydrophobic ("water fearing") and partitions away from water, and a polar "head group" that is strongly hydrophilic ("water loving") and tends to interact with water molecules. The interactions of phospholipids with one another and with water largely determine the structure of biomembranes.

Besides phospholipids, biomembranes contain smaller amounts of other amphipathic lipids, such as glycolipids and cholesterol, which contribute to membrane function in important ways. We first consider the structure and properties of pure phospholipid bilayers and then discuss the composition and behavior of natural cellular membranes. Then we consider how the precise lipid composition of a given membrane influences its physical properties.

FIGURE 7-3 The bilayer structure of biomembranes. (a) Electron micrograph of a thin section through an erythrocyte membrane stained with osmium tetroxide. The characteristic "railroad track" appearance of the membrane indicates the presence of two polar layers, consistent with the bilayer structure of phospholipid membranes. (b) Schematic interpretation of the phospholipid bilayer, in which polar groups face outward to shield the hydrophobic fatty acyl tails from water. The hydrophobic effect and van der Waals interactions between the fatty acyl tails drive the assembly of the bilayer (see Chapter 2). (c) Cross-sectional views of two other structures formed by dispersal of phospholipids in water. A spherical micelle has a hydrophobic interior composed entirely of fatty acyl chains; a spherical liposome consists of a phospholipid bilayer surrounding an aqueous center. (d) Under certain circumstances, lipids can assume yet other forms of organization. Shown here is the cubic phase of lipids, a highly regular recurring structure that has helped the formation of crystals of membrane proteins that were otherwise difficult to crystallize. [Part (a) Warren Rosenberg/Fundamental Photographs.]

## Phospholipids Spontaneously Form Bilayers

The amphipathic nature of phospholipids, which governs their interactions, is critical to the structure of biomembranes. When a suspension of phospholipids is mechanically dispersed in an aqueous solution, the phospholipids aggregate into one of three structures: spherical micelles or liposomes or sheet-like phospholipid bilayers that are two molecules thick (Figure 7-3). The type of structure formed
![li1292239539b](li1292239539b.jpg)

by pure phospholipids or a mixture of phospholipids in the absence of added proteins depends on several factors, including the length of the fatty acyl chains in the hydrophobic tails, their degree of saturation (i.e., the number of $\mathrm{C}-\mathrm{C}$ and $\mathrm{C}=\mathrm{C}$ bonds), and temperature. In all three structures, the hydrophobic effect (see Chapter 2) causes the fatty acyl chains to aggregate and exclude water molecules from the "core" of the structure. Micelles are rarely formed from natural phospholipids, whose fatty acyl chains are generally too bulky to fit into the interior of a micelle. Micelles are formed, however, if one of the two fatty acyl chains that make up the tail of a phospholipid is removed by hydrolysis to form a lysophospholipid, as occurs upon treatment with the enzyme phospholipase. In aqueous solutions, common detergents and soaps form micelles that behave like the balls in tiny ball bearings, thus giving soap solutions their slippery feel and lubricating properties.

Phospholipid mixtures of the composition present in cells spontaneously form a symmetric phospholipid bilayer. Each phospholipid layer in this lamellar structure is called a leaflet. The hydrophobic fatty acyl chains in each leaflet minimize their contact with water by aligning themselves tightly together in the center of the bilayer, forming a hydrophobic core that is about 3-4 nm thick (see Figure 7-3b). The close packing of these nonpolar tails is stabilized by van der Waals interactions between the hydrocarbon chains. Ionic and hydrogen bonds stabilize the interactions of the polar head groups with one another and with water. Electron microscopy of thin sections of cells stained with osmium tetroxide, which binds strongly to the polar head groups of phospholipids, shows the bilayer structure (see Figure 7-3a). A cross section of a single membrane stained with osmium tetroxide looks like a railroad track: two thin dark lines (the stained head group complexes) with a uniform light space of about 2 nm between them (the hydrophobic tails).

A phospholipid bilayer can be of almost unlimited sizefrom micrometers ( $\mu \mathrm{m}$ ) to millimeters ( mm ) in length or width-and can contain tens of millions of phospholipid molecules. The phospholipid bilayer is the basic structural unit of nearly all biological membranes. Its hydrophobic core prevents most water-soluble substances from crossing from one side of the membrane to the other. Although biomembranes contain other molecules (e.g., cholesterol, glycolipids, proteins), it is the phospholipid bilayer that separates two

## EXPERIMENTAL FIGURE 7-4 Formation and study of pure

phospholipid bilayers. (Top) A preparation of biological membranes is treated with an organic solvent, such as a mixture of chloroform and methanol (3:1), which selectively solubilizes the phospholipids and cholesterol. Proteins and carbohydrates remain in an insoluble residue. The solvent is removed by evaporation. (Bottom left) If the extracted materials are mechanically dispersed in water, they spontaneously form a liposome, shown in cross section, with an internal aqueous compartment. (Bottom right) A planar bilayer, also shown in cross section, can form over a small hole in a partition separating two aqueous phases; such a system can be used to study the physical properties of bilayers, such as their permeability to solutes.
aqueous solutions and acts as a permeability barrier. The lipid bilayer thus defines cellular compartments and allows a separation of the cell's interior from the outside world.

The three structures mentioned above are not the only forms that lipids can assume in an aqueous environment. Unusual configurations of lipids have been instrumental in enforcing order on otherwise difficult-to-crystallize membrane proteins, including G protein-coupled receptors, enabling crystallographic analysis of membrane proteins in a true lipid environment (Figure 7-3d).

## Phospholipid Bilayers Form a Sealed Compartment Surrounding an Internal Aqueous Space

Phospholipid bilayers can be generated in the laboratory by simple means, using either chemically pure phospholipids or lipid mixtures of the composition found in cellular membranes (Figure 7-4). Such synthetic bilayers possess three important properties. First, they are virtually impermeable
![li1302239539b](li1302239539b.jpg)

![li1312239539b](li1312239539b.jpg)

FIGURE 7-5 The faces of cellular membranes. The plasma membrane, a single bilayer, encloses the cell. In this highly schematic representation, internal cytosol (tan) and external environment (white) define the cytosolic (red) and exoplasmic (gray) faces of the bilayer. Vesicles and some organelles have a single membrane, and their internal aqueous space (white) is topologically equivalent to the outside of the cell. Three organellesthe nucleus, mitochondrion, and chloroplast (which is not shown)—are enclosed by two membranes separated by a small intermembrane space. The exoplasmic faces of the inner and outer membranes around these organelles border the intermembrane space between them. For simplicity, the hydrophobic membrane interior is not indicated in this diagram.
to water-soluble (hydrophilic) solutes, which do not readily diffuse across the bilayer. These solutes include salts, sugars, and most other small hydrophilic molecules-including water itself. The second property of a bilayer is its stability. The hydrophobic and van der Waals interactions between the fatty acyl chains maintain the integrity of the interior of the bilayer structure. Even though the exterior aqueous environment may vary widely in ionic strength and pH , the bilayer has the strength to retain its characteristic architecture. Third, all synthetic phospholipid bilayers can spontaneously form sealed closed compartments in which the aqueous space on the inside is separated from the exterior environment. An "edge" of a phospholipid bilayer, as depicted in Figure 7-3b, with the hydrocarbon core of the bilayer exposed to an aqueous solution, would be unstable; the exposed fatty acyl chains would be in an energetically much more stable state if they were not adjacent to water molecules but rather surrounded by other fatty acyl chains. Thus, in an aqueous solution, sheets of phospholipid bilayers spontaneously seal their edges, forming a spherical bilayer that encloses an aqueous central compartment. The liposome depicted in Figure 7-3c is an example of such a structure viewed in cross section.

This physical chemical property of a phospholipid bilayer has important implications for cellular membranes: no membrane in a cell can have an "edge" with exposed hydrocarbon fatty acyl chains. All membranes form closed compartments, similar in basic architecture to liposomes. Because all cellular membranes enclose an entire cell or an internal compartment, they have an internal face (the surface
oriented toward the interior of the compartment) and an external face (the surface presented to the environment). More commonly, we designate the two surfaces of a cellular membrane as the cytosolic face and the exoplasmic face. This nomenclature is useful in highlighting the topological equivalence of the faces in different membranes, as diagrammed in Figures 7-5 and 7-6. For example, the exoplasmic face of the plasma membrane is directed away from the cytosol, toward the extracellular space or external environment, and defines the outer limit of the cell. The cytosolic face of the plasma membrane faces the cytosol. Similarly for organelles and vesicles surrounded by a single membrane, the cytosolic face faces the cytosol. The exoplasmic face is always directed away from the cytosol, and in this case it is on the inside of the organelle, in contact with the internal aqueous space, or lumen. The lumen of a vesicle is topologically equivalent to the extracellular space, a concept most easily understood for vesicles that arise by invagination (endocytosis) of the plasma membrane. The external face of the plasma membrane becomes the internal face of the vesicle membrane, while in the vesicle the cytosolic face of the plasma membrane still faces the cytosol (see Figure 7-6).

Three organelles-the nucleus, mitochondrion, and chloroplast-are surrounded not by a single membrane, but by two. The exoplasmic surface of each membrane faces the space between the two membranes (the intermembrane space). This relationship can perhaps best be understood by reference to the endosymbiont hypothesis, discussed in Chapter 12, which posits that mitochondria and chloroplasts arose early in the evolution of eukaryotic cells by the

![li1322239539b](li1322239539b.jpg)

FIGURE 7-6 The faces of cellular membranes are conserved during membrane budding and fusion. Red membrane surfaces are cytosolic faces; gray membrane surfaces are exoplasmic faces. During endocytosis, a segment of the plasma membrane buds inward toward the cytosol and eventually pinches off a separate vesicle. During this process, the cytosolic face of the plasma membrane remains facing the cytosol, and the exoplasmic face of the new vesicle membrane faces the vesicle lumen. During exocytosis, an intracellular vesicle fuses with the plasma membrane, and the lumen of the vesicle (exoplasmic face) connects with the extracellular medium. Proteins that span the membrane retain their asymmetric orientation during vesicle budding and fusion; in particular, the same segment always faces the cytosol.
engulfment of bacteria capable of oxidative phosphorylation or photosynthesis, respectively (see Figure 12-7).

Natural membranes from different cell types exhibit a variety of shapes, which complement a cell's function. The smooth, flexible surface of the erythrocyte plasma membrane allows the cell to squeeze through narrow blood capillaries (Figure 7-7a). Some cells have a long, slender extension of the plasma membrane, called a cilium or flagellum, which beats in a whiplike manner (Figure 7-7b). This motion causes fluid to flow across the surface of a sheet of cells, or a sperm cell to swim toward an egg. The differing shapes and properties of biomembranes raise a key question in cell biology, namely, how the composition of biological membranes is regulated to establish and maintain the identity of the different membrane structures and membrane-delimited compartments. We return to this question in Section 7.3 and in Chapter 14.

## Biomembranes Contain Three Principal Classes of Lipids

The term phospholipid is a somewhat generic term. It refers to any amphipathic lipid with a phosphate-based head group and a two-chain hydrophobic tail. A typical biomembrane is not composed of phospholipids alone, but actually contains three classes of amphipathic lipids: phosphoglycerides, sphingolipids, and sterols, which differ in their chemical structures, abundance, and functions in the membrane
![li1332239539b](li1332239539b.jpg)

FIGURE 7-7 Variation in biomembranes in different cell types.
(a) A smooth, flexible membrane covers the surface of the discoid erythrocyte cell, as seen in this scanning electron micrograph. (b) Tufts of cilia project from the ependymal cells that line the brain ventricles. [Part (a) Omikron/Science Source. Part (b) iophoto Associates/Science Source.]
(Figure 7-8). While all phosphoglycerides are phospholipids, only certain sphingolipids are, and no sterols are.

Phosphoglycerides, the most abundant class of phospholipids in most membranes, are derivatives of glycerol 3-phosphate (see Figure 7-8a). A typical phosphoglyceride molecule consists of a hydrophobic tail composed of two fatty acid-based (acyl) chains esterified to the two hydroxyl groups in glycerol phosphate and a polar head group attached to the phosphate group. The structure comprising the glycerol moiety and the two fatty acyl chains is referred to as a diacylglycerol. The two fatty acyl chains may differ in the number of carbons that they contain (commonly 16 or 18) and their degree of saturation ( 0,1 , or 2 double bonds). A phosphoglyceride is classified according to the nature of its head group. In phosphatidylcholines, the most abundant phospholipids in the plasma membrane, the head group consists of choline, a positively charged alcohol,

![li1342239539b](li1342239539b.jpg)
(b) Sphingolipids
![li1352239539b](li1352239539b.jpg)

FIGURE 7-8 Three classes of membrane lipids. (a) Most phosphoglycerides are derivatives of glycerol 3-phosphate (red), which contains two esterified fatty acyl chains that constitute the hydrophobic "tail" and a polar "head group" esterified to the phosphate. The fatty acids can vary in length and be saturated (no double bonds) or unsaturated (one, two, or three double bonds). In phosphatidylcholine (PC), the head group is choline. Also shown are the molecules attached to the phosphate group in three other common phosphoglycerides: phosphatidylethanolamine (PE), phosphatidylserine (PS), and phosphatidylinositol (PI). Plasmalogens contain one fatty acyl chain attached to glycerol by an ester linkage and one attached by an ether linkage; they contain the same head groups as other phosphoglycerides. (b) Sphingolipids are derivatives of sphingosine (red), an amino alcohol with a long hydrocarbon
chain. Various fatty acyl chains are connected to sphingosine by an amide bond. The sphingomyelins (SM), which contain a phosphocholine head group, are phospholipids. Other sphingolipids are glycolipids in which a single sugar residue or branched oligosaccharide is attached to the sphingosine backbone. For instance, the simple glycolipid glucosylcerebroside (GlcCer) has a glucose head group. (c) The major sterols in animals (cholesterol), fungi (ergosterol), and plants (stigmasterol) differ slightly in structure, but all serve as key components of cellular membranes. The basic structure of sterols is a four-ring hydrocarbon (yellow). Like other membrane lipids, sterols are amphipathic. The single hydroxyl group is equivalent to the polar head group in other lipids; the conjugated ring and short hydrocarbon chain form the hydrophobic tail. See H. Sprong et al., 2001, Nature Rev. Mol. Cell Biol. 2:504.

esterified to the negatively charged phosphate. In other phosphoglycerides, an OH -containing molecule such as ethanolamine, serine, or the sugar derivative inositol is linked to the phosphate group. In the case of the inositol head group, the hydroxyl groups may be further modified with phosphates, yielding a class of phospholipids called phosphoinositides. The phosphoinositides fulfill an important function in signal transduction. The negatively charged phosphate group and the positively charged groups or hydroxyl groups on the head group interact strongly with water. At neutral pH , some phosphoglycerides (e.g., phosphatidylcholine and phosphatidylethanolamine) carry no net electric charge, whereas others (e.g., phosphatidylinositol and phosphatidylserine) carry a single net negative charge. Nonetheless, the polar head groups in all these phospholipids can pack together into the characteristic bilayer structure. When phospholipases act on phosphoglycerides, they produce lysophospholipids, which lack one of the two acyl chains. Lysophospholipids are not only important signaling molecules, released from cells and recognized by specific receptors; their presence can also affect the physical properties of the membranes in which they reside.

The plasmalogens are a group of phosphoglycerides that contain one fatty acyl chain attached to carbon 2 of glycerol by an ester linkage and one long hydrocarbon chain attached to carbon 1 of glycerol by an ether ( $\mathrm{C}-\mathrm{O}-\mathrm{C}$ ) rather than an ester linkage. Plasmalogens are particularly abundant in human brain and heart tissue. The greater chemical stability of the ether linkage in plasmalogens compared with the ester linkage, and the subtle differences in three-dimensional structure between plasmalogens and other phosphoglycerides, may have as yet unrecognized physiological significance.

A second class of membrane lipids is the sphingolipids. All these compounds are derived from sphingosine, an amino alcohol with a long hydrocarbon chain, and contain a long-chain fatty acid attached in amide linkage to the sphingosine amino group (see Figure 7-8b). Like phosphoglycerides, some sphingolipids have a phosphate-based polar head group. In sphingomyelin, the most abundant sphingolipid, phosphocholine is attached to the terminal hydroxyl group of sphingosine (see Figure 7-8b, SM). Thus sphingomyelin is a phospholipid, and its overall structure is quite similar to that of phosphatidylcholine. Sphingomyelins are similar in shape to phosphoglycerides and can form mixed bilayers with them. Other sphingolipids are amphipathic glycolipids whose polar head groups are sugars that are not linked to the tails via a phosphate group (so they are technically not phospholipids). Glucosylcerebroside, the simplest glycosphingolipid, contains a single glucose unit attached to sphingosine. In the complex glycosphingolipids called gangliosides, one or two branched sugar chains (oligosaccharides) containing sialic acid groups are attached to sphingosine. Glycolipids constitute $2-10$ percent of the total lipid content of plasma membranes; they are most abundant in nervous tissue.

Cholesterol and its analogs constitute the third important class of membrane lipids, the sterols. The basic structure of sterols is a four-ring isoprenoid-based hydrocarbon. The structures of the principal yeast sterol (ergosterol) and
plant phytosterols (e.g., stigmasterol) differ slightly from that of cholesterol, the major animal sterol (see Figure 7-8c). The small differences in the biosynthetic pathways and structures of fungal and animal sterols are the basis of most antifungal drugs currently in use. Cholesterol, like the two other sterols, has a hydroxyl substituent on one ring. Although cholesterol is almost entirely hydrocarbon in composition, it is amphipathic because its hydroxyl group can interact with water. Because it lacks a phosphate-based head group, it is not a phospholipid. Cholesterol is especially abundant in the plasma membranes of mammalian cells but is absent from most prokaryotic and all plant cells. As much as $30-50$ percent of the lipids in plant plasma membranes consist of certain steroids unique to plants. Between 50 and 90 percent of the cholesterol in most mammalian cells is present in the plasma membrane and associated vesicles. Cholesterol and other sterols are too hydrophobic to form a bilayer structure on their own. Instead, at the concentrations found in natural membranes, these sterols must intercalate between phospholipid molecules to be incorporated into biomembranes. When so intercalated, sterols provide structural support to membranes, preventing too close a packing of the phospholipids' acyl chains to maintain a significant measure of membrane fluidity, and at the same time conferring the necessary rigidity required for mechanical support. Some of these effects can be highly local, as in the case of lipid rafts, discussed below.

In addition to its structural role in membranes, cholesterol is the precursor for several important bioactive molecules. They include bile acids, which are made in the liver and help emulsify dietary fats for digestion and absorption in the intestines; steroid hormones produced by endocrine cells (e.g., adrenal gland, ovary, testes); and vitamin D produced in the skin and kidneys. Another critical function of cholesterol is its covalent addition to Hedgehog protein, a key signaling molecule in embryonic development (see Chapter 16).

## Most Lipids and Many Proteins Are Laterally Mobile in Biomembranes

In the two-dimensional plane of a bilayer, thermal motion permits lipid molecules to rotate freely around their long axes and to diffuse laterally within each leaflet. Because such movements are lateral or rotational, the fatty acyl chains remain in the hydrophobic interior of the bilayer. In both natural and artificial membranes, a typical lipid molecule exchanges places with its neighbors in a leaflet about $10^{7}$ times per second and diffuses several micrometers per second at $37^{\circ} \mathrm{C}$. These diffusion rates indicate that the bilayer is 100 times more viscous than water-about the same as the viscosity of olive oil. Even though lipids diffuse more slowly in the bilayer than in an aqueous solvent, a membrane lipid could diffuse the length of a typical bacterial cell $(1 \mu \mathrm{~m})$ in only 1 second and the length of an animal cell in about 20 seconds. When artificial pure phospholipid membranes are cooled below $37^{\circ} \mathrm{C}$, the lipids can undergo a phase transition from a liquid-like (fluid) state to a gel-like (semisolid) state, analogous to the liquid-solid transition when liquid water freezes (Figure 7-9). Below the

![li1362239539b](li1362239539b.jpg)

FIGURE 7-9 Gel-like and fluidlike forms of the phospholipid bilayer. (Top) Depiction of gel-to-fluid transition. Phospholipids with long saturated fatty acyl chains tend to assemble into a highly ordered, gel-like bilayer in which there is little overlap of the nonpolar tails in the two leaflets. Heat disorders the nonpolar tails and induces a transition from a gel to a fluid within a temperature range of only a few degrees. As the chains become disordered, the bilayer also decreases in thickness. (Bottom) Molecular models of phospholipid monolayers in gel-like and fluidlike states, as determined by molecular dynamics calculations. [Data from H. Heller et al., 1993, J. Phys. Chem. 97:8343.]
phase-transition temperature, the rate of diffusion of the lipids drops precipitously. At usual physiological temperatures, the hydrophobic interior of natural membranes generally has a low viscosity and a fluidlike consistency, in contrast to the gel-like consistency observed at lower temperatures.

In pure lipid bilayers (i.e., in the absence of protein), phospholipids and sphingolipids rotate and move laterally, but they do not spontaneously migrate, or flip-flop, from one leaflet to the other. The energetic barrier is too high; migration would require moving the polar head group from its aqueous environment through the hydrocarbon core of the bilayer to the aqueous solution on the other side. Special membrane proteins discussed in Chapter 11 are required to flip membrane lipids and other polar molecules from one leaflet to the other.

The lateral movements of specific plasma-membrane proteins and lipids can be quantified by a technique called fluorescence recovery after photobleaching (FRAP). Phospholipids containing a fluorescent substituent are used to monitor lipid movement. For proteins, a fragment of a monoclonal antibody that is specific for the exoplasmic domain of the desired protein and that has only a single antigen-binding site is tagged with a fluorescent dye. With this method, described in Figure 7-10, the rate at which membrane molecules move-the diffusion coefficient-can be determined, as well as the proportion of the molecules that are laterally mobile.

The results of FRAP studies with fluorescence-labeled phospholipids have shown that in fibroblast plasma membranes, all the phospholipids are freely mobile over distances
of about $0.5 \mu \mathrm{~m}$, but most cannot diffuse over much longer distances. These findings suggest that protein-rich regions of the plasma membrane about $1 \mu \mathrm{~m}$ in diameter separate lipidrich regions containing the bulk of the membrane phospholipids. Phospholipids are free to diffuse within such regions, but not from one lipid-rich region to an adjacent one. Furthermore, the rate of lateral diffusion of lipids in the plasma membrane is nearly an order of magnitude slower than in pure phospholipid bilayers: diffusion constants of $10^{-4} \mathrm{~cm}^{2}$ per second and $10^{-7} \mathrm{~cm}^{2}$ per second are characteristic of the plasma membrane and a pure phospholipid bilayer, respectively. This difference suggests that lipids may be tightly but not irreversibly bound to certain integral proteins in some membranes, as indeed has recently been demonstrated (see the discussion of annular phospholipids below).

## Lipid Composition Influences the Physical Properties of Membranes

A typical cell contains many different types of membranes, each with unique properties derived from its particular mix of lipids and proteins. The data in Table 7-1 illustrate the variation in lipid composition in different biomembranes. Several phenomena contribute to these differences. For instance, the relative abundances of phosphoglycerides and sphingolipids differ between membranes in the endoplasmic reticulum (ER), where phospholipids are synthesized, and the Golgi complex, where sphingolipids are synthesized. The proportion of sphingomyelin as a percentage of total membrane lipid phosphorus is about six times as high in Golgi membranes as it is in ER membranes. In other cases, the movement of membranes from one cellular compartment to another can selectively enrich certain membranes in lipids such as cholesterol. In responding to differing environments throughout an organism, different types of cells generate membranes with differing lipid compositions. In the cells that line the intestinal tract, for example, the membranes that face the harsh environment in which dietary nutrients are digested have a sphingolipid-to-phosphoglyceride-tocholesterol ratio of 1:1:1, rather than the 0.5:1.5:1 ratio found in cells subject to less stress. The relatively high concentration of sphingolipids in these intestinal membranes may increase their stability because of extensive hydrogen bonding by the free -OH group in the sphingosine moiety (see Figure 7-8).

The degree of bilayer fluidity depends on lipid composition, the structure of the phospholipid hydrophobic tails, and temperature. As already noted, van der Waals interactions and the hydrophobic effect cause the nonpolar tails of phospholipids to aggregate. Long, saturated fatty acyl chains have the greatest tendency to aggregate, packing tightly together into a gel-like state. Phospholipids with short fatty acyl chains, which have less surface area and therefore fewer van der Waals interactions, form more fluid bilayers. Likewise, the kinks in cis-unsaturated fatty acyl chains (see Chapter 2) result in their forming less stable van der Waals interactions with other lipids, and hence more fluid bilayers,

![li1372239539b](li1372239539b.jpg)

EXPERIMENTAL FIGURE 7-10 Fluorescence recovery after photobleaching (FRAP) experiments can quantify the lateral movement of proteins and lipids within the plasma membrane. (a) Experimental protocol. Step ■ Cells are first labeled with a fluorescent reagent that binds uniformly to a specific membrane lipid or protein. Step ■ A laser light is then focused on a small area of the cell surface, irreversibly bleaching the bound reagent and thus reducing the fluorescence in the illuminated area. Step ■ In time, the fluorescence of the bleached patch increases as unbleached fluorescent surface molecules diffuse into it and bleached ones diffuse outward. The extent of recovery of fluorescence in the bleached
and the amount of the fluorescence per second of the bleached area indicates that 50 percent of the receptor molecules in the illuminated membrane patch were mobile and 50 percent were immobile. Because the rate of fluorescence recovery is proportional to the rate at which labeled molecules move into the bleached region, the diffusion coefficient of a protein or lipid in the membrane can be calculated from such data. See Y. I. Henis et al., 1990, J. Cell Biol. 111:1409.
than do straight saturated chains, which can pack more tightly together.

Cholesterol is important in maintaining the appropriate fluidity of natural membranes, a property that appears to be essential for normal cell growth and reproduction. Cholesterol restricts the random movement of phospholipid head groups at the outer surfaces of the leaflets, but its effect on the movement of long phospholipid tails depends on its concentration. At the cholesterol concentrations normally present in the plasma membrane, the interaction of the steroid ring with the long hydrophobic tails of phospholipids tends to immobilize those lipids and thus decreases biomembrane fluidity. It is this property that can help organize the plasma membrane into discrete subdomains of unique lipid and protein composition. At lower cholesterol concentrations, however, the steroid ring separates and disperses phospholipid tails, causing the inner regions of the membrane to become slightly more fluid.

The lipid composition of a bilayer also influences its thickness, which in turn may influence the distribution of
and the distribution of the total cholesterol and the cholesterol in the plasma membrane. (b) Results of a FRAP experiment with human hepatoma cells treated with a fluorescent antibody specific for the asialoglycoprotein receptor protein. The finding that 50 percent of the fluorescence returned to the bleached area indicates that 50 percent of the receptor molecules in the illuminated membrane patch were mobile and 50 percent were immobile. Because the rate of fluorescence recovery is proportional to the rate at which labeled molecules move into the bleached region, the diffusion coefficient of a protein or lipid in the membrane can be calculated from such data. See Y. I. Henis et al., 1990, J. Cell Biol. 111:1409.
other membrane components, such as proteins, in a particular membrane. It has been argued that relatively short transmembrane segments of certain Golgi-resident enzymes (glycosyltransferases) are an adaptation to the lipid composition of the Golgi membrane and contribute to the retention of these enzymes in the Golgi apparatus. The results of biophysical studies on artificial membranes demonstrate that sphingomyelin associates into a more gel-like and thicker bilayer than phosphoglycerides do (Figure 7-11a). Cholesterol and other molecules that decrease membrane fluidity also increase membrane bilayer thickness. Because sphingomyelin tails are already optimally stabilized, the addition of cholesterol has no effect on the thickness of a sphingomyelin bilayer.

Another property dependent on the lipid composition of a bilayer is its curvature, which depends on the relative sizes of the polar head groups and nonpolar tails of its constituent phospholipids. Lipids with long tails and large head groups are cylindrical in shape; those with small head groups are cone-shaped (Figure 7-11b). As a result, bilayers composed

TABLE 7-1 Major Lipid Components of Selected Biomembranes

| Composition (mol \%) |  |  |  |  |
| :--: | :--: | :--: | :--: | :--: |
| Source/Location | PC | $\mathrm{PE}+\mathrm{PS}$ | SM | Cholesterol |
| Plasma membrane (human erythrocytes) | 21 | 29 | 21 | 26 |
| Myelin membrane (human neurons) | 16 | 37 | 13 | 34 |
| Plasma membrane (mung bean) | 47 | 43 | 0 | 0 |
| Inner mitochondrial membrane (cauliflower) | 42 | 38 | 0 | 0 |
| Outer mitochondrial membrane (cauliflower) | 47 | 27 | 0 | 0 |
| Plasma membrane (E. coli) | 0 | 85 | 0 | 0 |
| Endoplasmic reticulum membrane (rat) | 60 | 25 | 3 | 7 |
| Golgi membrane (rat) | 51 | 26 | 8 | 13 |
| Inner mitochondrial membrane (rat) | 40 | 37 | 2 | 7 |
| Outer mitochondrial membrane (rat) | 54 | 31 | 2 | 11 |
| Primary leaflet location | Exoplasmic | Cytosolic | Exoplasmic | Both |

$\mathrm{PC}=$ phosphatidylcholine; $\mathrm{PE}=$ phosphatidylethanolamine; $\mathrm{PS}=$ phosphatidylserine, $\mathrm{SM}=$ sphingomyelin.
source: Data from S. E. Horvath and G. Daum, 2013, Lipids of mitochondria, Progress in Lipid Research 52:590-614.
of cylindrical lipids are relatively flat, whereas those containing large numbers of cone-shaped lipids form curved bilayers (Figure 7-11c). This effect of lipid composition on bilayer curvature may play a role in the formation of highly curved membranes, such as sites of viral budding (see Figure 7-2)
![li1382239539b](li1382239539b.jpg)
and of formation of internal vesicles from the plasma membrane (see Figure 7-6), and in specialized stable membrane structures such as microvilli. Several proteins bind to the surface of a phospholipid bilayer and cause the membrane to curve; such proteins are important in formation of transport vesicles that bud from a donor membrane (see Chapter 14).

## Lipid Composition Is Different in the Exoplasmic and Cytosolic Leaflets

A characteristic of all biomembranes is an asymmetry in lipid composition across the bilayer. Although most phospholipids are present in both membrane leaflets, some are commonly more abundant in one or the other leaflet. For instance, in plasma membranes from human erythrocytes and MadinDarby canine kidney (MDCK) cells grown in culture, almost all the sphingomyelin and phosphatidylcholine, both of which form less fluid bilayers, are found in the exoplasmic leaflet. In contrast, phosphatidylethanolamine, phosphatidylserine,

FIGURE 7-11 Effect of lipid composition on bilayer thickness and curvature. (a) A pure sphingomyelin (SM) bilayer is thicker than one formed from a phosphoglyceride such as phosphatidylcholine (PC). Cholesterol has a lipid-ordering effect on phosphoglyceride bilayers that increases their thickness, but it does not affect the thickness of the more ordered SM bilayer. (b) Phospholipids such as PC have a cylindrical shape and form essentially flat monolayers, whereas those with smaller head groups, such as phosphatidylethanolamine (PE), have a conical shape. (c) A bilayer enriched with PC in the exoplasmic leaflet and with PE in the cytosolic face, as in many plasma membranes, would have a natural curvature. See H. Sprong et al., 2001, Nature Rev. Mol. Cell Biol. 2:504.

and phosphatidylinositol, which form more fluid bilayers, are preferentially located in the cytosolic leaflet. Because phosphatidylserine and phosphatidylinositol carry a net negative charge, the stretch of amino acids on the cytosolic face of a membrane protein with a single transmembrane segment, is often enriched in positively charged (Lys, Arg) residues in close proximity to the transmembrane segment (this distribution of charged amino acids is referred to as the "inside positive" rule). This segregation of lipids across the bilayer may influence membrane curvature (see Figure 7-11c). Unlike particular phospholipids, cholesterol is relatively evenly distributed in both leaflets of cellular membranes.

The relative abundances of a particular phospholipid in the two leaflets of a plasma membrane can be determined experimentally on the basis of the susceptibility of phospholipids to hydrolysis by phospholipases, enzymes that cleave the ester bonds via which acyl chains and head groups are connected to the lipid molecule (Figure 7-12). When added to the external medium, phospholipases cannot cross the membrane, and thus they cleave off the head groups of only those lipids present in the exoplasmic face; phospholipids in the cytosolic leaflet are resistant to hydrolysis because the enzymes cannot penetrate to the cytosolic face of the plasma membrane.

How the asymmetric distribution of phospholipids in membrane leaflets arises is still unclear. As noted, in pure bilayers, phospholipids do not spontaneously migrate, or flip-flop, from one leaflet to the other. In part, the asymmetry in phospholipid distribution may reflect where these lipids are synthesized in the endoplasmic reticulum and Golgi. Sphingomyelin is synthesized on the luminal (exoplasmic) face of the Golgi, which becomes the exoplasmic face of the plasma membrane. In contrast, phosphoglycerides are synthesized on the cytosolic face of the
![li1392239539b](li1392239539b.jpg)

FIGURE 7-12 Specificity of phospholipases. Each type of phospholipase cleaves one of the susceptible bonds shown in red. The glycerol carbon atoms are indicated by small numbers. In intact cells, only phospholipids in the exoplasmic leaflet of the plasma membrane are cleaved by phospholipases in the surrounding medium. Phospholipase C, a cytosolic enzyme, cleaves certain phospholipids in the cytosolic leaflet of the plasma membrane.

ER membrane, which is topologically equivalent to the cytosolic face of the plasma membrane (see Figure 7-5). Clearly, however, this explanation does not account for the preferential location of phosphatidylcholine (a phosphoglyceride) in the exoplasmic leaflet. Movement of this phosphoglyceride, and perhaps others, from one leaflet to the other in some natural membranes is most likely catalyzed by ATP-powered transport proteins called flippases, which are discussed in Chapter 11.

The preferential location of lipids on one face of the bilayer is necessary for a variety of membrane-based functions. For example, the head groups of all phosphorylated forms of phosphatidylinositol (PI; see Figure 7-8), an important source of second messengers, face the cytosol. Stimulation of many cell-surface receptors by their corresponding ligands results in activation of the cytosolic enzyme phospholipase C, which can then hydrolyze the bond within PI connecting the phosphoinositols to the diacylglycerol. As we will see in Chapter 15, both water-soluble phosphoinositols and membrane-embedded diacylglycerol participate in intracellular signaling pathways that affect many aspects of cellular metabolism. Phosphatidylserine too is normally most abundant in the cytosolic leaflet of the plasma membrane. In the initial stages of platelet stimulation by serum, phosphatidylserine is briefly translocated to the exoplasmic face, presumably by a flippase enzyme, where it activates enzymes participating in blood clotting. When cells die, lipid asymmetry is no longer maintained, and phosphatidylserine, normally enriched in the cytosolic leaflet, is increasingly found in the exoplasmic one. This increased exposure is detected experimentally by use of a labeled version of annexin V, a protein that specifically binds to phosphatidylserine, to measure the onset of programmed cell death (apoptosis). The increased exposure of phosphatidylserine on dying or dead cells is recognized by phagocytic cells, which initiate engulfment of such apoptotic bodies and thus ensure timely and safe disposal of cell remnants.

## Cholesterol and Sphingolipids Cluster with Specific Proteins in Membrane Microdomains

Membrane lipids are not randomly distributed (evenly mixed) in each leaflet of a bilayer. One hint that lipids may be organized within the leaflets was the discovery that the lipids remaining after the extraction (solubilization) of plasma membranes with non-ionic detergents such as Triton X-100 predominantly contain two species: cholesterol and sphingomyelin. Because these two lipids are found in more ordered, less fluid bilayers, researchers hypothesized that they form microdomains, termed lipid rafts, surrounded by other, more fluid phospholipids that are more readily extracted by non-ionic detergents. (We discuss more fully the role of ionic and non-ionic detergents in extracting membrane proteins in Section 7.2.)

Some biochemical and microscopic evidence supports the existence of lipid rafts, which in natural membranes

are typically 50 nm in diameter. Rafts can be disrupted by methyl- $\beta$-cyclodextrin, which specifically extracts cholesterol from membranes, or by antibiotics such as filipin that sequester cholesterol into aggregates within the membrane. Such findings indicate the importance of cholesterol in maintaining the integrity of lipid rafts. These raft fractions, defined by their insolubility in non-ionic detergents, contain a subset of plasma-membrane proteins, many of which are implicated in sensing extracellular signals and transmitting them into the cytosol. Because raft fractions are enriched in glycolipids, an important tool for microscopic visualization of raft-type structures in intact cells is the use of fluorescently labeled cholera toxin, a protein that specifically binds to certain gangliosides. By bringing many key proteins into close proximity and stabilizing their interactions, lipid rafts may facilitate signaling by cell-surface receptors and the subsequent activation of cytosolic events. However, much remains to be learned about the structure and biological function of lipid rafts. The unique properties of some of the raft-associated lipids, such as glycolipids, may permit interactions of their tails across the hydrophobic core and help organize lipids of the cytosolic leaflet in the formation of signaling platforms.
![li1402239539b](li1402239539b.jpg)

FIGURE 7-13 Lipid droplets form by budding and scission from the ER membrane. Lipid droplet formation begins with the accumulation of cholesterol esters and triglycerides (yellow) within the hydrophobic core of the lipid bilayer. The resulting delamination of the two lipid monolayers causes a "lens" to form, the further growth of which creates a spherical droplet that is then released by scission at the neck. The newly formed droplet is surrounded by a lipid monolayer, derived from the cytosolic leaflet of the ER membrane.

## Cells Store Excess Lipids in Lipid Droplets

Lipid droplets are vesicular structures, composed of triglycerides and cholesterol esters, that originate from the ER and serve a lipid-storage function. When a cell's supply of lipids exceeds the immediate need for membrane construction, excess lipids are relegated to these lipid droplets, readily visualized in live cells by staining with a lipophilic dye such as Congo red. Feeding cells with oleic acid, a type of fatty acid, enhances lipid droplet formation. Lipid droplets are not only storage compartments for triglycerides and cholesterol esters, but may also serve as platforms for storage of proteins targeted for degradation. The biogenesis of lipid droplets starts with delamination of the lipid bilayer of the ER through insertion of triglycerides and cholesterol esters (Figure 7-13). The lipid "lens" continues to grow by insertion of more lipid, until finally a lipid droplet is hatched by scission from the ER. The resulting cytoplasmic droplet is thereby surrounded by a phospholipid monolayer. The details of lipid droplet biogenesis, as well as its functions, remain to be defined more clearly.

## KEY CONCEPTS OF SECTION 7.1

## The Lipid Bilayer: Composition and Structural Organization

- Membranes are crucial to cell structure and function. The eukaryotic cell is demarcated from the external environment by the plasma membrane and organized into membranelimited internal compartments (organelles and vesicles).
- The phospholipid bilayer, the basic structural unit of all biomembranes, is a two-layered lipid sheet with hydrophilic faces and a hydrophobic core, which is impermeable to water-soluble molecules and ions. Proteins embedded in the bilayer endow the membrane with specific functions (see Figure 7-1).
- Phospholipids spontaneously form bilayers and sealed compartments surrounding an aqueous space (see Figure 7-3).
- As bilayers, all biological membranes have an internal (cytosolic) face and an external (exoplasmic) face (see Figure 7-5). Some organelles are surrounded by two, rather than one, membrane bilayer.
- The primary lipid components of biomembranes are phosphoglycerides, sphingolipids, and sterols such as cholesterol. The term "phospholipid" applies to any amphipathic lipid molecule with a fatty acyl hydrocarbon tail and a phosphatebased polar head group (see Figure 7-8).
- Biomembranes can undergo phase transitions from fluidlike to gel-like states depending on the temperature and the composition of the membrane (see Figure 7-9).
- Most lipids and many proteins are laterally mobile in biomembranes (see Figure 7-10).

- Different cellular membranes vary in lipid composition (see Table 7-1). Phospholipids and sphingolipids are asymmetrically distributed in the two leaflets of the bilayer, whereas cholesterol is fairly evenly distributed in both leaflets.
- Natural biomembranes generally have a viscous consistency with fluidlike properties. In general, membrane fluidity is decreased by sphingolipids and cholesterol and increased by phosphoglycerides. The lipid composition of a membrane also influences its thickness and curvature (see Figure 7-11).
- Lipid rafts are microdomains containing cholesterol, sphingolipids, and certain membrane proteins that form in the plane of the bilayer. These lipid-protein aggregates might facilitate signaling by certain plasma-membrane receptors.
- Lipid droplets are storage vesicles for lipids, originating in the ER (see Figure 7-13).


### 7.2 Membrane Proteins: Structure and Basic Functions

Membrane proteins are defined by their location within or at the surface of a phospholipid bilayer. Although every biological membrane has the same basic bilayer structure, the proteins associated with a particular membrane are responsible for its distinctive activities. The kinds and amounts of proteins associated with biomembranes vary depending on cell type and subcellular location. For example, the inner mitochondrial membrane is 76 percent protein; the myelin membrane that surrounds nerve axons, only 18 percent. The high phospholipid content of myelin allows it to electrically insulate the nerve from its environment, as we discuss in Chapter 22. The importance of membrane proteins is evident from the finding that approximately a third of all yeast genes encode a membrane protein. The relative abundance of genes for membrane proteins is even greater in multicellular organisms, in which membrane proteins have additional functions in cell adhesion and in communication between different cell types (cell-cell interactions).

The lipid bilayer presents a distinctive two-dimensional hydrophobic environment for membrane proteins. Some proteins contain segments that are embedded within the hydrophobic core of the phospholipid bilayer; other proteins are associated with the exoplasmic or cytosolic leaflet of the bilayer. Protein domains on the extracellular surface of the plasma membrane generally bind to extracellular molecules, including external signaling proteins, ions, and small metabolites (e.g., glucose, fatty acids), as well as proteins on other cells or in the external environment. Segments of proteins within the plasma membrane perform multiple functions, such as forming the channels and pores through which molecules and ions move into and out of cells. These intramembrane segments also serve to organize multiple membrane proteins into larger assemblies within the plane of the membrane. Domains lying along the cytosolic face of the plasma membrane have a wide range of
functions, from anchoring cytoskeletal proteins to the membrane to triggering intracellular signaling pathways.

In many cases, the function of a membrane protein and the topology of its polypeptide chain in the membrane can be predicted on the basis of its similarity with other wellcharacterized proteins. In this section, we examine the characteristic structural features of membrane proteins and some of their basic functions. We describe the structures of several proteins to help you get a feel for the way membrane proteins interact with membranes. More complete characterizations of the properties of various types of membrane proteins are presented in later chapters that focus on their structures and activities in the context of their cellular functions.

## Proteins Interact with Membranes in Three Different Ways

Membrane proteins can be classified into three categoriesintegral, lipid-anchored, and peripheral-on the basis of their position with respect to the membrane (see Figure 7-1). Integral membrane proteins, also called transmembrane proteins, span a phospholipid bilayer and comprise three domains. The cytosolic and exoplasmic domains have hydrophilic exterior surfaces that interact with the aqueous environment on the cytosolic and exoplasmic faces of the membrane. These domains resemble segments of other water-soluble proteins in their amino acid composition and structure. In contrast, the membrane-spanning segments usually contain many hydrophobic amino acids whose side chains protrude outward and interact with the hydrophobic hydrocarbon core of the phospholipid bilayer. In all transmembrane proteins examined to date, the membrane-spanning domains consist of one or more $\alpha$ helices or of multiple $\beta$ strands. We discuss the ribosomal synthesis and post-translational processing of soluble cytosolic proteins in Chapters 5 and 10; the process by which integral membrane proteins are inserted into membranes as part of their synthesis is discussed in Chapter 13.

Lipid-anchored membrane proteins are bound covalently to one or more lipid molecules. The hydrophobic tail of the attached lipid is embedded in one leaflet of the membrane and anchors the protein to the membrane. The polypeptide chain itself does not enter the phospholipid bilayer.

Peripheral membrane proteins do not directly contact the hydrophobic core of the phospholipid bilayer. Instead, they are bound to the membrane either indirectly by interactions with integral or lipid-anchored membrane proteins or directly by interactions with lipid head groups. Peripheral proteins can be bound to either the cytosolic or the exoplasmic face of the plasma membrane. In addition to these proteins, which are closely associated with the bilayer, cytoskeletal filaments can be more loosely associated with the cytosolic face, usually through one or more peripheral adapter proteins. Such associations with the cytoskeleton provide support for various cellular membranes, helping to determine the cell's shape and mechanical properties, and play a role in the two-way communication between the cell interior and the exterior, as we learn in Chapter 17. Finally, peripheral proteins on the outer surface of the plasma membrane and

the exoplasmic domains of integral membrane proteins are often attached to components of the extracellular matrix or to the cell walls surrounding bacterial and plant cells, providing a crucial interface between the cell and its environment.

## Most Transmembrane Proteins Have MembraneSpanning a Helices

Soluble proteins exhibit hundreds of distinct localized folded structures, or motifs (see Figure 3-10). In comparison, the repertoire of folded structures in the transmembrane domains of integral membrane proteins is quite limited, with the hydrophobic $\alpha$ helix predominating. Proteins containing membranespanning $\alpha$-helical domains are stably embedded in membranes because of energetically favorable hydrophobic and van der Waals interactions of the hydrophobic side chains in the domain with specific lipids and probably also because of ionic interactions with the polar head groups of the phospholipids.

A single $\alpha$-helical domain is sufficient to incorporate an integral membrane protein into a membrane. However, many such proteins have more than one transmembrane $\alpha$ helix. Typically, a membrane-embedded $\alpha$ helix is composed of a continuous segment of 20-25 hydrophobic (uncharged)
amino acids (see Figure 2-14). The predicted length of such an $\alpha$ helix $(3.75 \mathrm{~nm})$ is just sufficient to span the hydrocarbon core of a phospholipid bilayer. In many membrane proteins, these helices are perpendicular to the plane of the membrane, whereas in others, the helices traverse the membrane at an oblique angle. The hydrophobic side chains protrude outward from the helix and form van der Waals interactions with the fatty acyl chains in the bilayer. In contrast, the hydrophilic amide peptide bonds are in the interior of the $\alpha$ helix (see Figure 3-4); each carbonyl $(\mathrm{C}=\mathrm{O})$ group forms a hydrogen bond with the amide hydrogen atom of the amino acid four residues toward the C-terminus of the helix. These polar groups are shielded from the hydrophobic interior of the membrane.

To help you get a better sense of the structures of proteins with $\alpha$-helical domains, we will briefly discuss four different kinds of such proteins: glycophorin A, G proteincoupled receptors, aquaporins (water/glycerol channels), and the T cell receptor for antigen.

Glycophorin A, the major protein in the erythrocyte plasma membrane, is a representative single-pass transmembrane protein, which contains only one membrane-spanning $\alpha$ helix (Figure 7-14a). The 23-residue membrane-spanning
![li1412239539b](li1412239539b.jpg)

FIGURE 7-14 Structure of glycophorin A, a typical single-pass transmembrane protein. (a) Diagram of dimeric glycophorin, showing its major sequence features and its relation to the membrane. The single 23-residue membrane-spanning $\alpha$ helix in each monomer is composed of amino acids with hydrophobic (uncharged) side chains (red and green spheres). By binding negatively charged phospholipid head groups, the positively charged arginine and lysine residues (blue spheres) near the cytosolic side of the helix help anchor glycophorin in the membrane. Both the extracellular and the cytosolic domains are rich in charged residues and polar uncharged residues; the extracellular domain is heavily
glycosylated, with carbohydrate chains (green diamonds) attached to specific serine, threonine, and asparagine residues. (b) Molecular model of the transmembrane domain of dimeric glycophorin A corresponding to residues 73-96. The hydrophobic side chains of the a helix in one monomer are shown in pink; those of the other monomer, in green. Residues depicted as space-filling structures participate in van der Waals interactions that stabilize the coiled-coil dimer. Note how the hydrophobic side chains project outward from the helix, toward what would be the surrounding fatty acyl chains. [Part (b) data from K. R. MacKenzie et al., 1997, Science 276:131, PDB ID 1afz.]

$\alpha$ helix is composed of amino acids with hydrophobic (uncharged) side chains, which interact with the fatty acyl chains in the surrounding bilayer. In cells, glycophorin A typically forms dimers: the transmembrane helix of one glycophorin A polypeptide associates with the corresponding transmembrane helix in a second glycophorin A to form a coiled-coil structure (Figure 7-14b). Such interactions of membrane-spanning $\alpha$ helices are a common mechanism for creating dimeric membrane proteins, and many membrane proteins form oligomers (two or more polypeptides bound together noncovalently) by interactions between their membrane-spanning helices.

A large and important group of integral membrane proteins is defined by the presence of seven membrane-spanning $\alpha$ helices. This group includes the large family of G proteincoupled cell-surface receptors discussed in Chapter 15, many of which have been crystallized. One such multipass transmembrane protein of known structure is bacteriorhodopsin, a protein found in the membranes of certain photosynthetic bacteria; it illustrates the general structure of all these proteins (Figure 7-15a). Absorption of light by the retinal group covalently attached to this protein causes a conformational change in the protein that results in the pumping of protons from the cytosol across the bacterial membrane to the extracellular space. The proton concentration gradient thus generated across the membrane is used to synthesize ATP during photosynthesis (see Chapter 12). In the high-resolution structure of bacteriorhodopsin, the positions of all the individual amino acids, the retinal group, and the surrounding
lipids are clearly defined. As might be expected, virtually all the amino acids on the exterior of the membrane-spanning segments of bacteriorhodopsin are hydrophobic, permitting energetically favorable interactions with the hydrocarbon core of the surrounding lipid bilayer.

The aquaporins are a large family of highly conserved proteins that transport water, glycerol, and other hydrophilic molecules across biomembranes. They illustrate several aspects of the structure of multipass transmembrane proteins. Aquaporins are tetramers of four identical subunits. Each of the four subunits has six membrane-spanning $\alpha$ helices, some of which traverse the membrane at oblique angles rather than perpendicularly. Because all aquaporins have similar structures, we will focus on one, the glycerol channel Glpf, whose structure has been especially well defined by x-ray diffraction studies (Figure 7-15b). This aquaporin has one long transmembrane helix with a bend in the middle, and more strikingly, there are two $\alpha$ helices that penetrate only halfway through the membrane. The N -termini of these helices face each other (yellow Ns in the figure), and together they span the membrane at an oblique angle. Thus some membraneembedded helices-and other, nonhelical, structures we will encounter later-do not traverse the entire bilayer. As we will see in Chapter 11, these short helices in aquaporins form part of the glycerol/water-selective pore in the middle of each subunit. This structure highlights the considerable diversity in the ways membrane-spanning $\alpha$ helices interact with the lipid bilayer and with other segments of the protein.
![li1422239539b](li1422239539b.jpg)

FIGURE 7-15 Structural models of two multipass membrane proteins. (a) Bacteriorhodopsin, a photoreceptor in certain bacteria. The seven hydrophobic a helices in bacteriorhodopsin traverse the lipid bilayer roughly perpendicular to the plane of the membrane. A retinal molecule (black) covalently attached to one helix absorbs light. The large class of G protein-coupled receptors in eukaryotic cells also has seven membrane-spanning a helices; their three-dimensional structure is thought to be similar to that of bacteriorhodopsin. (b) Two views of the glycerol channel Glpf, rotated $180^{\circ}$ with respect to each other along an axis perpendicular to the plane of the membrane. Note the
several membrane-spanning a helices that are at oblique angles, the two helices that penetrate only halfway through the membrane (purple with yellow arrows), and the one long membrane-spanning helix with a "break" or distortion in the middle (purple with yellow line). The glycerol molecule in the hydrophilic "core" is colored red. The protein structure was approximately positioned in the hydrocarbon core of the membrane by finding the most hydrophobic $3-\mu \mathrm{m}$ slab of the protein perpendicular to the membrane plane. (Part (a) data from H. Luecke et al., 1999, J. Mol. Biol. 291:899. Part (b) data from J. Bowie, 2005, Nature 438:581589, PDB ID 1c3w and D. Fu et al., 2000, Science 290:481-486, PDB ID 1fxd.)

The specificity of phospholipid-protein interactions is evident from the structure of a different aquaporin, aquaporin 0 (Figure 7-16). Aquaporin 0 is the most abundant protein in the plasma membrane of the fiber cells that make up the bulk of the lens of the mammalian eye. Like other aquaporins, it is a tetramer of identical subunits. The protein's surface is not covered by a set of uniform binding sites for phospholipid molecules. Instead, fatty acyl chains pack tightly against the irregular hydrophobic outer surface of the protein. The lipids involved in this interaction are referred to as annular phospholipids because they form a tight ring (annulus) of lipids around the protein that are not easily exchanged with bulk phospholipids in the bilayer. Some of the fatty acyl chains are straight, in the trans conformation (see Chapter 2), whereas others are kinked in order to interact with bulky hydrophilic side chains on the surface of the protein. Some of the lipid head groups are parallel to the surface of the membrane, as is the case in pure phospholipid bilayers. Others, however, are oriented almost at right angles to the plane of the membrane. Thus there can be specific interactions between phospholipids and membrane-spanning proteins, and the function of many membrane proteins can be affected by the specific types of phospholipids present in the bilayer.

In addition to the predominantly hydrophobic (uncharged) residues that serve to embed integral membrane
![li1432239539b](li1432239539b.jpg)

FIGURE 7-16 Annular phospholipids. Side view of the threedimensional structure of one subunit of the lens-specific aquaporin 0 homotetramer, crystallized in the presence of the phospholipid dimyristoylphosphatidylcholine, a phospholipid with 14 carbon-saturated fatty acyl chains. Note the lipid molecules forming a bilayer shell around the protein. The protein is shown as a surface plot (the lighter background molecule). The lipid molecules are shown in space-filling format; the polar lipid head groups (gray and red) and the lipid fatty acyl chains (the extended black and gray structures) form a bilayer with almost uniform thickness around the protein. Presumably, in the membrane, lipid fatty acyl chains cover the whole of the hydrophobic surface of the protein; only the most ordered of the lipid molecules would be resolved in the crystallographic structure. [Data from A. Lee, 2005, Nature 438:569-570, and T. Gonen et al., 2005, Nature 438:633-688, PDB ID 2b6o.]
proteins in the bilayer, many $\alpha$-helical transmembrane segments contain polar or charged residues. Their amino acid side chains can be used to guide the assembly and stabilization of multimeric membrane proteins. The T cell receptor for antigen is a case in point: it is composed of four separate dimers, the interactions of which are driven by charge-charge interactions between $\alpha$ helices at the appropriate "depth" in the hydrocarbon core of the lipid bilayer (Figure 7-17). The electrostatic attraction of positive and negative charges on each dimer helps the dimers to "find each other." Thus charged residues in otherwise hydrophobic transmembrane segments can help guide assembly of multimeric membrane proteins.
![li1442239539b](li1442239539b.jpg)
![li1452239539b](li1452239539b.jpg)

FIGURE 7-17 Charged residues can orchestrate the assembly of multimeric membrane proteins. The T cell receptor (TCR) for antigen is composed of four separate dimers: an $\alpha \beta$ pair directly responsible for antigen recognition, and accessory subunits collectively referred to as the CD3 complex. These accessory subunits include the $\gamma, \delta, \varepsilon$, and $\zeta$ subunits. The $\zeta$ subunits form a disulfide-linked homodimer. The $y$ and $\delta$ subunits occur in complex with an $\varepsilon$ subunit, to generate a $\gamma \varepsilon$ and a $\delta \varepsilon$ pair. The transmembrane segments of the TCR $\alpha$ and $\beta$ chains each contain positively charged residues (blue). These residues allow recruitment of corresponding $\delta \varepsilon$ and $\gamma \varepsilon$ heterodimers, which carry negative charges (red) at the appropriate depth in the hydrophobic core of the bilayer. The $\zeta$ homodimer docks onto the charges in the TCR $\alpha$ chain (dark green), while the $\gamma \varepsilon$ and $\delta \varepsilon$ subunit pairs find their corresponding partners deeper down in the hydrophobic core on both the TCR $\alpha$ and TCR $\beta$ chain (light green). Charged residues in otherwise nonpolar transmembrane segments can thus guide assembly of higher-order structures. [Data from K. W. Wucherpfennig et al., 2010, Cold Spring Harb. Perspect. Biol., 2:a005140, PDB ID 1xmw; M. E. Call et al., 2006, Cell. 127:355, PDB ID 2hac; and L. Kjer-Nielsen et al., 2003, Immunity. 18:53, PDB ID 1mi5.]

## Multiple $\beta$ Strands in Porins Form MembraneSpanning "Barrels"

The porins are a class of transmembrane proteins whose structure differs radically from that of other integral membrane proteins based on $\alpha$-helical transmembrane domains. Several types of porins are found in the outer membranes of gram-negative bacteria such as E. coli and in the outer membranes of mitochondria and chloroplasts. The outer membrane protects an intestinal bacterium from harmful agents (e.g., antibiotics, bile salts, and proteases), but permits the uptake and disposal of small hydrophilic molecules, including nutrients and waste products. Different types of porins in the outer membrane of an E. coli cell provide channels for the passage of specific types of disaccharides or other small molecules as well as for ions such as phosphate. The amino acid sequences of porins contain none of the long, continuous hydrophobic segments typical of integral membrane proteins with $\alpha$-helical membrane-spanning domains. Rather, it is the entire outer surface of the fully folded porin that displays its hydrophobic character to the hydrocarbon core of the lipid bilayer. X-ray crystallography shows that porins are trimers of identical subunits. In each subunit, $16 \beta$ strands form a sheet that twists into a barrel-shaped structure with a pore in the center (Figure 7-18). Unlike a typical water-soluble
![li1462239539b](li1462239539b.jpg)

FIGURE 7-18 Structural model of one subunit of OmpX, a porin found in the outer membrane of E. coli. All porins are trimeric transmembrane proteins. Each subunit is barrel shaped, with $\beta$ strands forming the wall and a transmembrane pore in the center. A band of aliphatic (hydrophobic and noncyclic) side chains (yellow) and a border of aromatic (ring-containing) side chains (red) position the protein in the bilayer. See G. E. Schulz, 2000, Curr. Opin. Struct. Biol. 10:443. [Data from J. Vogt and G. E. Schulz, 1999, Structure 7:1301, PDB ID 1qj8.]
globular protein, a porin has a hydrophilic interior and a hydrophobic exterior; in this sense, porins are inside out. In a porin monomer, the outward-facing side chains on each of the $\beta$ strands are hydrophobic and form a nonpolar ribbonlike band that encircles the outside of the barrel. This hydrophobic band interacts with the fatty acyl groups of the membrane lipids or with other porin monomers. The side chains facing the inside of a porin monomer are predominantly hydrophilic; they line the pore through which small water-soluble molecules cross the membrane. (Note that the aquaporins discussed above, despite their name, are not porins and contain multiple transmembrane $\alpha$ helices.)

## Covalently Attached Lipids Anchor Some Proteins to Membranes

In eukaryotic cells, covalently attached lipids anchor some otherwise typically water-soluble proteins to one or the other leaflet of the membrane. In such lipid-anchored proteins, the hydrocarbon chains of the lipid anchor are embedded in the bilayer, but the protein itself does not enter the bilayer. The lipid anchors used to anchor proteins to the cytosolic face are not used for the exoplasmic face, and vice versa.

One group of cytosolic proteins is anchored to the cytosolic face of a membrane by a fatty acyl group (e.g., myristate or palmitate) covalently attached to an N-terminal glycine residue, a modification called acylation (Figure 7-19a). Retention of such proteins at the membrane by the N-terminal acyl anchor may play an important role in a membraneassociated function. For example, v-Src, a mutant form of a cellular tyrosine kinase, induces abnormal cellular growth that can lead to cancer, but does so only when it has a myristylated N -terminus.

A second group of cytosolic proteins are anchored to membranes by a hydrocarbon chain attached to a cysteine residue at or near the C-terminus, a modification called prenylation (Figure 7-19b). Prenyl anchors are built from 5 -carbon isoprene units, which, as detailed in the following section, are also used in the synthesis of cholesterol. In prenylation, a 15 -carbon farnesyl or 20 -carbon geranylgeranyl group is bound by a thioether bond to the -SH group of a C-terminal cysteine residue of the protein, usually part of a C-terminal Cys-Ala-Ala-X ( $\mathrm{X}=$ any of a number of amino acids) or CAAX box. Once prenylation has occurred, the C-terminal Ala-Ala-X motif may be removed by proteolysis. In some cases, a second geranylgeranyl group or a fatty acyl palmitate group is linked to a nearby cysteine residue. The additional hydrocarbon anchor is thought to reinforce the attachment of the protein to the membrane. For example, Ras, a GTPase superfamily protein that functions in intracellular signaling (see Chapter 15), is recruited to the cytosolic face of the plasma membrane by such a double anchor. Rab proteins, which also belong to the GTPase superfamily, are similarly bound to the cytosolic surface of intracellular vesicles by prenyl anchors; these proteins are required for the fusion of vesicles with their target

![li1472239539b](li1472239539b.jpg)

FIGURE 7-19 Anchoring of plasma-membrane proteins to the phospholipid bilayer by covalently linked hydrocarbon groups. (a) Cytosolic proteins such as v-Src are associated with the plasma membrane through a single fatty acyl chain attached to the N -terminal glycine (Gly) residue of the polypeptide. Myristate (C14) and palmitate (C16) are common acyl anchors. (b) Other cytosolic proteins (e.g., Ras and Rab proteins) are anchored to the membrane by prenylation of one or two cysteine (Cys) residues at or near the C-terminus. The anchors are farnesyl (C15) and geranylgeranyl (C20) groups, both of which are unsaturated. (c) The lipid anchor on the exoplasmic surface of the plasma membrane is glycosylphosphatidylinositol (GPI). The phosphatidylinositol part (red) of this anchor contains two fatty acyl chains that extend into the bilayer. The phosphoethanolamine unit (purple) in the anchor links it to the protein. The two green hexagons represent sugar units, which vary in number, nature, and arrangement in different GPI anchors. The complete structure of a yeast GPI anchor is shown in Figure 13-15. See H. Sprong et al., 2001, Nature Rev. Mol. Cell Biol. 2:504.
membranes (see Chapter 14). Yet other proteins are palmitoylated on membrane-proximal cysteine residues in the absence of other lipid modifications.

Some cell-surface proteins, and some specialized proteins with distinctive covalently attached polysaccharides called proteoglycans (see Chapter 20), are bound to the exoplasmic face of the plasma membrane by a third anchor group, glycosylphosphatidylinositol (GPI) anchors. The exact structures of GPI anchors vary greatly among different cell types, but they always contain phosphatidylinositol, whose two fatty acyl chains extend into the lipid bilayer just like those of typical membrane phospholipids; phosphoethanolamine, which covalently links the anchor to the C-terminus of a protein; and several sugar residues (Figure 7-19c). Therefore, GPI anchors are glycolipids. The GPI anchor is both necessary and sufficient for binding proteins to the membrane. For instance, treatment of cells with phospholipase C, which cleaves the phosphate-glycerol bond in phospholipids
and in GPI anchors (see Figure 7-12), releases GPI-anchored proteins such as Thy-1 and placental alkaline phosphatase (PLAP) from the cell surface.

## All Transmembrane Proteins and Glycolipids Are Asymmetrically Oriented in the Bilayer

Every type of transmembrane protein has a specific orientation, known as its topology, with respect to the membrane faces. Its cytosolic segments always face the cytosol, and its exoplasmic segments always face the opposite side of the membrane. This asymmetry in protein orientation gives the two membrane faces their different properties. The orientations of different types of transmembrane proteins are established during their synthesis, as we will see in Chapter 13. Membrane proteins have never been observed to flipflop across a membrane; such movement, requiring a transient movement of hydrophilic amino acid residues through the hydrophobic interior of the membrane, would be energetically unfavorable. Accordingly, the asymmetric topology of a transmembrane protein is maintained throughout the protein's lifetime. As Figure 7-6 shows, membrane proteins retain their asymmetric orientation during membrane budding and fusion events; the same segment always faces the cytosol and the same segment is always exposed to the exoplasmic face. In multipass membrane proteins, the orientation of individual transmembrane segments can be affected by changes in the membrane's phospholipid composition.

Many transmembrane proteins contain carbohydrate chains covalently linked to serine, threonine, or asparagine side chains of the polypeptide. Such transmembrane glycoproteins are always oriented so that all the carbohydrate chains are in the exoplasmic domain (see Figure 7-14 for the example of glycophorin A). Likewise, glycolipids, in which a carbohydrate chain is attached to the glycerol or sphingosine backbone of a membrane lipid, are always located in the exoplasmic leaflet, with the carbohydrate chain protruding from the membrane surface. The biosynthetic basis for the asymmetric glycosylation of proteins is described in Chapter 14. Both glycoproteins and glycolipids are especially abundant in the plasma membranes of eukaryotic cells and in the membranes of the intracellular compartments that establish the secretory and endocytic pathways; they are absent from the inner mitochondrial membrane, chloroplast lamellae, and several other intracellular membranes. Because the carbohydrate chains of glycoproteins and glycolipids in the plasma membrane extend into the extracellular space, they are available to interact with components of the extracellular matrix as well as with lectins (proteins that bind specific sugars), growth factors, and antibodies.

One important consequence of interactions involving membrane glycoproteins and glycolipids is illustrated by the ABO blood-group antigens. These three structurally related oligosaccharide components of certain glycoproteins and glycolipids are expressed on the surfaces of human red

![li1482239539b](li1482239539b.jpg)

FIGURE 7-20 Human ABO blood group antigens. These antigens are oligosaccharide chains covalently attached to glycolipids or glycoproteins in the plasma membrane. The terminal oligosaccharide sugars distinguish the three antigens. The presence or absence of the glycosyltransferases that add galactose (Gal) or N -acetylgalactosamine (GalNAc) to O antigen determine a person's blood type.
blood cells and many other cell types (Figure 7-20). All humans have the enzymes for synthesizing O antigen. Persons with type A blood also have a glycosyltransferase enzyme that adds an extra modified monosaccharide called N -acetylgalactosamine to O antigen to form A antigen. Those with type B blood have a different transferase that adds an extra galactose to O antigen to form B antigen. People with both transferases produce both A and B antigen (AB blood type); those who lack these transferases produce O antigen only (O blood type).

People whose erythrocytes lack the A antigen, the B antigen, or both on their surface normally have antibodies against the missing antigen(s) in their serum. Thus if a type A or O person receives a transfusion of type B blood, antibodies against the B antigen will bind to the introduced red cells and trigger their destruction. To prevent such harmful
reactions, blood group typing and appropriate matching of blood donors and recipients are required in all transfusions (Table 7-2).

## Lipid-Binding Motifs Help Target Peripheral Proteins to the Membrane

Many water-soluble enzymes whose substrates are phospholipids must bind to membrane surfaces. Phospholipases, for example, hydrolyze various bonds in the head groups of phospholipids (see Figure 7-12) and thereby play a variety of roles in cells-helping to degrade damaged or aged cellular membranes, generating precursors to signaling molecules, and even serving as the active components in many snake venoms. Many peripheral proteins, including phospholipases, initially bind to the polar head groups of membrane phospholipids to carry out their catalytic functions. The mechanism of action of phospholipase $\mathrm{A}_{2}$ illustrates how such enzymes can reversibly interact with membranes and catalyze reactions at the interface of an aqueous solution and a lipid surface. When this enzyme is in aqueous solution, its $\mathrm{Ca}^{2+}$-containing active site is buried in a channel lined with hydrophobic amino acids. The enzyme binds with greatest affinity to bilayers composed of negatively charged phospholipids (e.g., phosphatidylserine). This observation suggests that the rim of positively charged lysine and arginine residues around the entrance to the catalytic channel is particularly important in binding (Figure 7-21a) and constitutes a lipid binding motif. Binding induces a conformational change in phospholipase $\mathrm{A}_{2}$ that strengthens its binding to the phospholipid heads and opens the hydrophobic channel. As a phospholipid molecule moves from the bilayer into the channel, the enzyme-bound $\mathrm{Ca}^{2+}$ binds to the phosphate in the head group, thereby positioning the ester bond to be cleaved in the catalytic site (Figure 7-21b), releasing the acyl chain.

## Proteins Can Be Removed from Membranes by Detergents or High-Salt Solutions

Membrane proteins are often difficult to purify and study, mostly because of their tight association with membrane lipids and other membrane proteins. Detergents, which are amphipathic molecules that disrupt membranes by

TABLE 7-2 ABO Blood Groups

| Blood Group | Antigens on RBCs* | Serum Antibodies | Can Receive Blood Types |
| :-- | :--: | :--: | :--: |
| A | A | Anti-B | A and O |
| B | B | Anti-A | B and O |
| AB | A and B | None | All |
| O | O | Anti-A and anti-B | O |

*See Figure 7-20 for antigen structures.

![li1492239539b](li1492239539b.jpg)

FIGURE 7-21 Lipid-binding surface and mechanism of action of phospholipase $\mathbf{A}_{2}$. (a) A structural model of the enzyme, showing the surface that interacts with a membrane. This lipid-binding surface contains a rim of positively charged arginine and lysine residues, shown in blue surrounding the cavity of the catalytic active site, in which a substrate lipid (red ball-and-stick structure) is bound. (b) Diagram of catalysis by phospholipase $\mathrm{A}_{2}$. When docked on a lipid membrane, positively charged residues of the binding site bind to negatively charged polar groups at the membrane surface. This binding triggers a small conformational change, opening a channel lined with hydrophobic amino acids that leads from the bilayer to the catalytic site. As a phospholipid moves into the channel, an enzyme-bound $\mathrm{Ca}^{2+}$ ion (green) binds to the head group, positioning the ester bond to be cleaved (red) next to the catalytic site. See D. Blow, 1991, Nature 351:444, and M. H. Gelb et al., 1999, Curr. Opin. Struct. Biol. 9:428. [Part (a) data from D. L. Scott et al., 1990, Science 250:1563, PDB ID 1poc.]
intercalating into phospholipid bilayers, can be used to solubilize lipids and many membrane proteins. The hydrophobic part of a detergent molecule is attracted to the phospholipid hydrocarbons and mingles with them readily; the hydrophilic part is strongly attracted to water. Some detergents, such as bile salts, are natural products, but most are synthetic molecules developed for cleaning and for dispersing mixtures of oil and water in the food industry (e.g., creamy peanut butter) (Figure 7-22). Ionic detergents, such as sodium deoxycholate (a bile salt) and sodium dodecylsulfate (SDS), contain a charged group; non-ionic detergents, such as Triton X-100 and octylglucoside, lack a charged group. At very low concentrations, detergents dissolve in pure water as isolated molecules. As the concentration increases, the molecules begin to form micelles-small spherical aggregates in which the hydrophilic parts of the molecules face outward and the hydrophobic parts cluster in the center (see Figure 7-3c). The critical micelle concentration (CMC) at which micelles form is characteristic of each detergent and is a function of the structures of its hydrophobic and hydrophilic parts.

Ionic and non-ionic detergents interact differently with proteins and have different uses in the lab. Ionic detergents bind to the exposed hydrophobic regions of membrane proteins as well as to the hydrophobic cores of water-soluble proteins. Because of their charge, these detergents can disrupt ionic and hydrogen bonds. At high concentrations, sodium dodecylsulfate, for example, completely denatures proteins by binding to every side chain, a property that is exploited in SDS gel electrophoresis (see Figure 3-38). Non-ionic detergents generally do not denature proteins and are thus useful in extracting proteins in their folded and active form from membranes before the proteins are purified. Protein-protein interactions, especially the weaker ones, can be sensitive to both ionic and non-ionic detergents.

At high concentrations (above the CMC), non-ionic detergents solubilize biological membranes by forming
![li1502239539b](li1502239539b.jpg)

FIGURE 7-22 Structures of four common detergents. The hydrophobic part of each molecule is shown in yellow; the hydrophilic part, in blue. The bile salt sodium deoxycholate is a natural product; the others are synthetic. Although ionic detergents commonly cause denaturation of proteins, non-ionic detergents do not and are thus useful in solubilizing integral membrane proteins.

FIGURE 7-23 Solubilization of integral membrane proteins by non-ionic detergents. At a concentration higher than its critical micelle concentration (CMC), a detergent solubilizes lipids and integral membrane proteins, forming mixed micelles containing detergent, protein, and lipid molecules. At concentrations below the CMC, non-ionic detergents (e.g., octylglucoside, Triton X-100) can dissolve membrane proteins without forming micelles by coating the membrane-spanning regions.
![li1512239539b](li1512239539b.jpg)
mixed micelles of detergent, phospholipids, and integral membrane proteins, bulky hydrophobic structures that do not dissolve in water (Figure 7-23, top). At low concentrations (below the CMC), these detergents bind to the hydrophobic regions of most integral membrane proteins, but without forming micelles, allowing the proteins to remain soluble (Figure 7-23, bottom). Creating such an aqueous solution of integral membrane proteins is a necessary first step in protein purification.

Treatment of cultured cells with a buffered salt solution containing a non-ionic detergent such as Triton X-100 extracts water-soluble proteins as well as integral membrane proteins from cellular membranes. As noted earlier, the exoplasmic and cytosolic domains of integral membrane proteins are generally hydrophilic and soluble in water. The membrane-spanning domains, however, are rich in hydrophobic and uncharged residues (see Figure 7-14). When separated from membranes, these exposed hydrophobic segments tend to interact with one another, causing the protein molecules to aggregate and precipitate from aqueous solutions. The hydrophobic parts of non-ionic detergent molecules preferentially bind to the hydrophobic segments of transmembrane proteins, preventing protein aggregation and allowing the proteins to remain in aqueous solution. Detergent-solubilized transmembrane proteins can then be purified by affinity chromatography and other techniques used in purifying water-soluble proteins (see Chapter 3).

As discussed previously, most peripheral membrane proteins are bound to specific transmembrane proteins or membrane phospholipids by ionic or other weak noncovalent interactions. Generally, peripheral proteins can be removed from the membrane by solutions of high ionic strength (high salt concentrations), which disrupt ionic bonds, or by chemicals that bind divalent cations such as $\mathrm{Mg}^{2+}$. Unlike integral membrane proteins, most peripheral proteins are soluble in water and need not be solubilized by non-ionic detergents.

## KEY CONCEPTS OF SECTION 7.2

## Membrane Proteins: Structure and Basic Functions

- Biological membranes usually contain integral (transmembrane) proteins as well as lipid-anchored proteins and peripheral membrane proteins, which do not enter the hydrophobic core of the bilayer (see Figure 7-1).
- Most integral membrane proteins contain one or more membrane-spanning hydrophobic $\alpha$ helices bracketed by hydrophilic domains that extend into the aqueous environment surrounding the cytosolic and exoplasmic faces of the membrane (see Figures 7-14, 7-15, and 7-17).
- Fatty acyl side chains as well as the polar head groups of membrane lipids pack tightly and irregularly around the hydrophobic segments of integral membrane proteins (see Figure 7-16).
- The porins, unlike other integral membrane proteins, contain membrane-spanning $\beta$ sheets that form a barrel-like channel through the bilayer (see Figure 7-18).
- Lipids attached to certain amino acids anchor some proteins to one or the other membrane leaflet (see Figure 7-19).
- All transmembrane proteins and glycolipids are asymmetrically oriented in the bilayer. Invariably, carbohydrate chains are present only on the exoplasmic surface of a glycoprotein or glycolipid.
- Many water-soluble enzymes (e.g., phospholipases) whose substrates are phospholipids must bind to the membrane surface to carry out their function. Such binding is often due to the attraction between positive charges on lysine and arginine residues in the protein and negative charges on phospholipid head groups in the bilayer.
- Transmembrane proteins can be selectively extracted from membranes with the use of non-ionic detergents.

### 7.3 Phospholipids, Sphingolipids, and Cholesterol: Synthesis and Intracellular Movement

In this section, we consider some of the special challenges that a cell faces in synthesizing and transporting lipids, which are poorly soluble in the aqueous interior of a cell. The focus of our discussion will be the biosynthesis and movement of the major lipids found in cellular membranes-phospholipids, sphingolipids, and cholesterol-and their precursors. In lipid biosynthesis, water-soluble precursors are assembled into membrane-associated intermediates that are then converted into membrane lipid products. The movement of these lipids, especially membrane components, between different organelles is critical for maintaining the proper composition and properties of membranes and overall cell structure.

A fundamental principle of membrane biosynthesis is that cells synthesize new membranes only by the expansion of existing membranes. [The one exception may be autophagy, in which new membrane is formed first through the formation of an autophagic crescent, the construction of which involves modification of phosphatidylethanolamine with the ubiquitin-like modifier Atg8 (see Figure 14-35).] Although some early steps in the synthesis of membrane lipids take place in the cytosol, the final steps are catalyzed by enzymes bound to preexisting cellular membranes, and the products are incorporated into the membranes as they are generated. Evidence for this process is seen when cells are briefly exposed to radioactively labeled precursors (e.g., phosphates or fatty acids): all the phospholipids and sphingolipids incorporating these labeled precursors are associated with intracellular membranes; as expected from the hydrophobicity of the fatty acyl chains, none are found free in the cytosol.

After they are formed, membrane lipids must be distributed appropriately both between the leaflets of a given membrane and among the independent membranes of different organelles in eukaryotic cells, as well as the plasma membrane. Here we consider how this precise lipid distribution is accomplished; in Chapters 13 and 14, we discuss how membrane proteins are inserted into cellular membranes and trafficked to their appropriate location within the cell.

## Fatty Acids Are Assembled from Two-Carbon Building Blocks by Several Important Enzymes

Fatty acids (see Chapter 2) play a number of important roles in cells. In addition to being a cellular fuel source
(see the discussion of aerobic oxidation in Chapter 12), fatty acids are key components of both the phospholipids and the sphingolipids that make up cellular membranes; they also anchor some proteins to cellular membranes (see Figure 7-19). Thus the regulation of fatty acid synthesis plays a key role in the regulation of membrane synthesis as a whole. The major fatty acids in phospholipids contain 14, 16, 18, or 20 carbon atoms and include both saturated and unsaturated chains. The fatty acyl chains found on sphingolipids can be longer than those in the phosphoglycerides, containing up to 26 carbon atoms, and may bear other chemical modifications (e.g., hydroxylation) as well.

Fatty acids are synthesized from the two-carbon building block acetate $\left(\mathrm{CH}_{3} \mathrm{COO}^{-}\right)$. In cells, both acetate and the intermediates in fatty acid biosynthesis are esterified to a large water-soluble molecule called coenzyme A (CoA), as exemplified by the structure of acetyl CoA (below).

Acetyl CoA is an important intermediate in the metabolism of glucose, fatty acids, and many amino acids, as detailed in Chapter 12. It also contributes acetyl groups in many biosynthetic pathways. Saturated fatty acids (with no carbon-carbon double bonds) containing 14 or 16 carbon atoms are made from acetyl CoA by two enzymes, acetylCoA carboxylase and fatty acid synthase. In animal cells, these enzymes are found in the cytosol; in plants, they are found in chloroplasts. Palmitoyl CoA (a 16-carbon fatty acyl group linked to CoA) can be elongated to 18-24 carbons by the sequential addition of two-carbon units in the endoplasmic reticulum (ER) or sometimes in the mitochondrion. Desaturase enzymes, also located in the ER, introduce double bonds at specific positions in some fatty acids, yielding unsaturated fatty acids. Oleyl CoA (oleate linked to CoA; see Table 2-4), for example, is formed by removal of two H atoms from stearyl CoA. In contrast to free fatty acids, fatty acyl CoA derivatives are soluble in aqueous solutions because of the hydrophilicity of the CoA segment.

## Small Cytosolic Proteins Facilitate Movement of Fatty Acids

In order to be transported through the cell cytosol, free, or unesterified, fatty acids (those unlinked to CoA) are commonly bound by fatty acid-binding proteins (FABPs), which belong to a group of small cytosolic proteins that act as chaperones to facilitate the intracellular movement of many lipids. These proteins contain a hydrophobic pocket lined by $\beta$ sheets (Figure 7-24). A long-chain fatty acid can fit into this pocket and interact noncovalently with the surrounding protein.
![li1522239539b](li1522239539b.jpg)

![li1532239539b](li1532239539b.jpg)

FIGURE 7-24 Binding of a fatty acid to the hydrophobic pocket of a fatty acid-binding protein (FABP). The crystal structure of adipocyte FABP (ribbon diagram) reveals that the hydrophobic binding pocket is generated from two $\beta$ sheets that are nearly at right angles to each other, forming a clam shell-like structure. A fatty acid (carbons yellow; oxygens red) interacts noncovalently with hydrophobic amino acid residues within this pocket. See A. Reese-Wagoner et al., 1999, Biochim. Biophys. Acta 23:1441(2-3):106-116. [Data from Z. Xu, D. A. Bernlohr, and L. J. Banaszak, 1993, J. Biol. Chem. 268:7874, PDB ID 1lid.]

The expression of cellular FABPs is regulated coordinately with cellular requirements for the uptake and release of fatty acids. Thus FABP levels are high in active muscles that are using fatty acids for generation of ATP, and in adipocytes (fat-storing cells) when they are either taking up
fatty acids to be stored as triglycerides or releasing fatty acids for use by other cells. The importance of FABPs in fatty acid metabolism is highlighted by the observations that they can compose as much as 5 percent of all cytosolic proteins in the liver, and that genetic inactivation of cardiac muscle FABP converts the heart from a muscle that primarily burns fatty acids for energy into one that primarily burns glucose.

## Fatty Acids Are Incorporated into Phospholipids Primarily on the ER Membrane

Fatty acids are not directly incorporated into phospholipids; rather, in eukaryotic cells, they are first converted into CoA esters. The subsequent synthesis of phospholipids such as the phosphoglycerides is carried out by enzymes associated with the cytosolic face of the ER membrane, usually the smooth ER, in animal cells; through a series of steps, fatty acyl CoAs, glycerol 3-phosphate, and polar head group precursors are linked together and then inserted into the ER membrane (Figure 7-25). The fact that the enzymes involved in this process are located on the cytosolic side of the membrane means that there is an inherent asymmetry in membrane biogenesis: new membranes are initially synthesized only on one leaflet-a fact with important consequences for the asymmetric distribution of lipids in membrane leaflets. Once synthesized on the ER, phospholipids are transported to other organelles and to the plasma membrane. Mitochondria synthesize some of their own membrane lipids and import others.
![li1542239539b](li1542239539b.jpg)

FIGURE 7-25 Phospholipid synthesis in the ER membrane. Because phospholipids are amphipathic molecules, the last steps of their multistep synthesis take place at the interface between a membrane and the cytosol and are catalyzed by membrane-associated enzymes. Step $\boldsymbol{\square}$ Two fatty acids from fatty acyl CoA are esterified to the phosphorylated glycerol backbone, forming phosphatidic acid, whose two long hydrocarbon
chains anchor the molecule to the membrane. Step $\boldsymbol{\square}$ A phosphatase converts phosphatidic acid into diacylglycerol. Step $\boldsymbol{\square}$ A polar head group (e.g., phosphorylcholine) is transferred from cytosine diphosphocholine (CDP-choline) to the exposed hydroxyl group. Step $\boldsymbol{\square}$ Flippase proteins catalyze the movement of phospholipids from the cytosolic leaflet in which they are initially formed to the exoplasmic leaflet.

Sphingolipids are also synthesized indirectly from multiple precursors. Sphingosine, the building block of these lipids, is made in the ER, beginning with the coupling of a palmitoyl group from palmitoyl CoA to serine; the subsequent addition of a second fatty acyl group to form N -acyl sphingosine (ceramide) also takes place in the ER. Later, in the Golgi, a polar head group is added to ceramide, yielding sphingomyelin, whose head group is phosphorylcholine, and various glycosphingolipids, in which the head group may be a monosaccharide or a more complex oligosaccharide (see Figure 7-8b). Some sphingolipid synthesis can also take place in mitochondria. In addition to serving as the backbone for sphingolipids, ceramide and its metabolic products are important signaling molecules that can influence cell growth, proliferation, endocytosis, resistance to stress, and programmed cell death (apoptosis).

After their synthesis is completed in the Golgi, sphingolipids are transported to other cellular compartments through vesicle-mediated mechanisms similar to those for the transport of proteins, discussed in Chapter 14. Any type of vesicular transport results in movement not only of the protein payload, but also of the lipids that compose the vesicular membrane. Phospholipids such as phosphoglycerides, as well as cholesterol, can move between organelles by additional mechanisms, described below.

## Flippases Move Phospholipids from One Membrane Leaflet to the Opposite Leaflet

Even though phospholipids are initially incorporated into the cytosolic leaflet of the ER membrane, various phospholipids are asymmetrically distributed in the two leaflets of the ER membrane and of other cellular membranes. As noted above, phospholipids do not readily flip-flop from one leaflet to the other. For the ER membrane to expand by growth of both leaflets and to have asymmetrically distributed phospholipids, its phospholipid components must be able to move from one leaflet to the other. Although the mechanisms employed to generate and maintain membrane phospholipid asymmetry are not well understood, it is clear that flippases play a key role. As described in Chapter 11, these integral membrane proteins use the energy of ATP hydrolysis to facilitate the movement of phospholipid molecules from one leaflet to the other (see Figure 11-16).

## Cholesterol Is Synthesized by Enzymes in the Cytosol and ER Membrane

Next we focus on cholesterol, the principal sterol in animal cells. Cholesterol is synthesized mainly in the liver. The first steps of cholesterol synthesis (Figure 7-26)—conversion of
![li1552239539b](li1552239539b.jpg)

FIGURE 7-26 Cholesterol biosynthetic pathway. The regulated rate-controlling step in cholesterol biosynthesis is the conversion of $\beta$-hydroxy- $\beta$-methylglutaryl CoA (HMG-CoA) into mevalonate by HMG-CoA reductase, an ER-membrane protein. Mevalonate is then converted into isopentenyl pyrophosphate (IPP), which has the basic
five-carbon isoprenoid structure. IPP can be converted into cholesterol and into many other lipids, often through the polyisoprenoid intermediates shown here. Some of the numerous compounds derived from isoprenoid intermediates and cholesterol itself are indicated.

three acetyl groups linked to CoA (acetyl CoA) forming the six-carbon molecule $\beta$-hydroxy- $\beta$-methylglutaryl linked to CoA (HMG-CoA)—take place in the cytosol. The conversion of HMG-CoA into mevalonate, the key rate-controlling step in cholesterol biosynthesis, is catalyzed by HMG-CoA reductase, an ER integral membrane protein, even though both its substrate and its product are water soluble. The water-soluble catalytic domain of HMG-CoA reductase extends into the cytosol, but its eight transmembrane $\alpha$ helices firmly embed the enzyme in the ER membrane. Five of the transmembrane $\alpha$ helices compose the so-called sterolsensing domain and regulate enzyme stability. When levels of cholesterol in the ER membrane are high, binding of cholesterol to this domain causes the enzyme to bind to two other integral ER membrane proteins, Insig-1 and Insig-2. This binding, in turn, induces ubiquitinylation (see Figure 3-31) of HMG-CoA reductase and its degradation by the proteasome pathway, reducing the production of mevalonate, the key intermediate in cholesterol biosynthesis.

5Atherosclerosis, frequently called cholesterol-dependent clogging of the arteries, is characterized by the progressive deposition of cholesterol and other lipids, cells, and extracellular matrix material in the inner layer of the wall of an artery. The resulting distortion of the artery's wall can lead, either alone or in combination with a blood clot, to major blockage of blood flow. Atherosclerosis accounts for 75 percent of deaths due to cardiovascular disease in the United States.

Perhaps the most successful anti-atherosclerosis medications are the statins. These drugs bind to HMG-CoA reductase and directly inhibit its activity, thereby lowering cholesterol biosynthesis. As a consequence, the concentration of low-density lipoproteins (see Figure 14-27)—the small, membrane-enveloped particles containing cholesterol esterified to fatty acids that often and rightly are called "bad cholesterol"-drops in the blood, reducing the formation of atherosclerotic plaques.

Mevalonate, the six-carbon product formed by HMGCoA reductase, is converted in several steps into the fivecarbon isoprenoid compound isopentenyl pyrophosphate (IPP) and its stereoisomer, dimethylallyl pyrophosphate (DMPP) (see Figure 7-26). These reactions are catalyzed by cytosolic enzymes, as are the subsequent reactions in the cholesterol synthesis pathway, in which six IPP units condense to yield squalene, a branched-chain 30-carbon intermediate. Enzymes bound to the ER membrane catalyze the multiple reactions that convert squalene into cholesterol in mammals or into related sterols in other species. One of the intermediates in this pathway, farnesyl pyrophosphate, is the precursor of the prenyl lipid that anchors Ras and related proteins to the cytosolic surface of the plasma membrane (see Figure 7-19) as well as other important biomolecules (see Figure 7-26).

## Cholesterol and Phospholipids Are Transported Between Organelles by Several Mechanisms

As already noted, the final steps in the synthesis of cholesterol and phospholipids take place primarily in the ER. Thus the plasma membrane and the membranes bounding other organelles must obtain these lipids by means of one or more intracellular transport processes. Membrane lipids can and do accompany both soluble and membrane proteins along the secretory pathway described in Chapter 14: membrane vesicles bud from the ER and fuse with membranes in the Golgi complex, and other membrane vesicles bud from the Golgi complex and fuse with the plasma membrane (Figure 7-27a). However, several lines of evidence suggest that there is substantial inter-organelle movement of cholesterol and phospholipids through other mechanisms. For example, chemical inhibitors of the classic secretory pathway and mutations that impede vesicular traffic along this pathway do not prevent cholesterol or phospholipid transport between membranes.
![li1562239539b](li1562239539b.jpg)
(b)
![li1572239539b](li1572239539b.jpg)
(c)
![li1582239539b](li1582239539b.jpg)

FIGURE 7-27 Proposed mechanisms of transport of cholesterol and phospholipids between membranes. In mechanism (a), vesicles transfer lipids between membranes. In mechanism (b), lipid transfer is a consequence of direct contact between membranes that is
mediated by membrane-embedded proteins. In mechanism (c), transfer is mediated by small, soluble lipid-transfer proteins. See F. R. Maxfield and D. Wustner, 2002, J. Clin. Invest. 110:891.

A second proposed mechanism of lipid movement entails direct protein-mediated contact of ER or ER-derived membranes with the membranes of other organelles (Figure 7-27b). In a third proposed mechanism, small lipid-transfer proteins facilitate the exchange of phospholipids or cholesterol between different membranes (Figure 7-27c). Although such lipid-transfer proteins have been identified in assays in vitro, their role in the intracellular movements of most phospholipids is not well defined. For instance, mice with a knockout mutation in the gene encoding the phosphatidylcholinetransfer protein appear to be normal in most respects, indicating that this protein is not essential for cellular phospholipid metabolism.

As noted earlier, the lipid compositions of different organelle membranes vary considerably (see Table 7-1). Some of these differences are due to different sites of synthesis. For example, a phospholipid called cardiolipin, which is localized to the mitochondrial membrane, is made only in mitochondria, and little of it is transferred to other organelles. Differential transport of lipids also plays a role in determining the lipid compositions of different cellular membranes. For instance, even though cholesterol is made in the ER, the cholesterol concentration (cholesterol-to-phospholipid molar ratio) is $1.5-13$-fold higher in the plasma membrane than in other organelles (ER, Golgi, mitochondrion, lysosome). Although the mechanisms responsible for establishing and maintaining these differences are not well understood, we have seen that the distinctive lipid composition of each membrane has a major influence on its physical and biological properties.

## KEY CONCEPTS OF SECTION 7.3

## Phospholipids, Sphingolipids, and Cholesterol: Synthesis and Intracellular Movement

- Saturated and unsaturated fatty acids of various chain lengths are components of phospholipids and sphingolipids.
- Fatty acids are synthesized from acetyl CoA by watersoluble enzymes and modified by elongation and desaturation in the endoplasmic reticulum (ER).
- Free fatty acids are transported within cells by fatty acidbinding proteins (FABPs).
- Fatty acids are incorporated into phospholipids through a multistep process. The final steps in the synthesis of phosphoglycerides and sphingolipids are catalyzed by membraneassociated enzymes primarily on the cytosolic face of the ER (see Figure 7-25).
- Each type of newly synthesized lipid is incorporated into the preexisting membranes on which it is made; thus, membranes are themselves the platform for the synthesis of new membrane material.
- Most membrane phospholipids are preferentially distributed in either the exoplasmic or the cytosolic leaflet. This asymmetry results in part from the action of flippases, which flip phospholipids from one leaflet to the other.
- The initial steps in cholesterol biosynthesis take place in the cytosol, whereas the last steps are catalyzed by enzymes associated with the ER membrane.
- The rate-controlling step in cholesterol biosynthesis is catalyzed by HMG-CoA reductase, whose transmembrane segments are embedded in the ER membrane and contain a sterol-sensing domain.
- Considerable evidence indicates that vesicular transport, direct protein-mediated contacts between different membranes, soluble lipid-transfer proteins, or all three may account for some inter-organelle transport of cholesterol and phospholipids (see Figure 7-27).


## LaunchPad

macmillan learning
Visit LaunchPad to access study tools and to learn more about the content in this chapter.

- Perspectives for the Future
- Analyze the Data
- Additional study tools, including videos, animations, and quizzes


## Key Terms

amphipathic 273
aquaporin 286
atherosclerosis 296
cholesterol 278
cilium 276
cytoskeleton 271
cytosolic face 275
exoplasmic face 275
flagellum 276
flippase 282
glycolipid 278
glycoprotein 289
hydrophilic 273
hydrophobic 273
integral membrane
protein 284
lectin 289
lipid-anchored membrane
protein 284
lipid droplet 283
lipid raft 282
liposome 273
lumen 275
membrane transport
protein 271
micelle 273
peripheral membrane
protein 284
phosphoglyceride 276
phospholipase 282
phospholipid bilayer 273
plasma membrane 271
porin 288
receptor protein 271
saturated 293
sphingolipid 278
statin 296
sterol 278
unsaturated 293

## Review the Concepts

1. When viewed by electron microscopy, the lipid bilayer is often described as looking like a railroad track. Explain how the structure of the bilayer creates this image.
2. Explain the following statement: The structure of all biomembranes depends on the chemical properties of phospholipids, whereas the function of each specific biomembrane depends on the specific proteins associated with that membrane.
3. Biomembranes contain many different types of lipid molecules. What are the three main types of lipid molecules found in biomembranes? How are the three types similar, and how are they different?
4. Lipid bilayers are said to behave like two-dimensional fluids. What does this mean? What drives the movement of lipid molecules and proteins within the bilayer? How can such movement be measured? What factors affect the degree of membrane fluidity?
5. Why are water-soluble substances unable to freely cross the lipid bilayer of the plasma membrane? How does the cell overcome this permeability barrier?
6. Name the three groups into which membrane-associated proteins may be classified. Explain the mechanism by which each group associates with a biomembrane.
7. Identify the following membrane-associated proteins based on their structure: (a) tetramers of identical subunits, each with six membrane-spanning $\alpha$ helices; (b) trimers of identical subunits, each with $16 \beta$ sheets forming a barrellike structure.
8. Proteins may be bound to the exoplasmic or cytosolic face of the plasma membrane by way of covalently attached lipids. What are the three types of lipid anchors responsible for tethering proteins to the plasma-membrane bilayer? Which type is used by cell-surface proteins that face the external medium? By glycosylated proteoglycans?
9. Although both faces of a biomembrane are composed of the same general types of macromolecules, principally lipids and proteins, the two faces of the bilayer are not identical. What accounts for the asymmetry between the two faces?
10. What are detergents? How do ionic and non-ionic detergents differ in their ability to disrupt biomembrane structure? 11. What is the likely identity of these membrane-associated proteins: (a) a protein that is released from a membrane treated with a high-salt solution, which causes disruption of ionic linkages; (b) a protein that is not released from the membrane upon its exposure to a high-salt solution alone, but is released when the membrane is incubated with an enzyme that cleaves phosphate-glycerol bonds and covalent linkages are disrupted; (c) a protein that is not released from the membrane upon exposure to a high-salt solution, but is released after the addition of the detergent sodium dodecylsulfate (SDS). Will the activity of the protein released in part (c) be preserved following its release?
11. Following the production of membrane extracts using the non-ionic detergent Triton X-100, you analyze the membrane lysates via mass spectrometry and note a high content of cholesterol and sphingolipids. Furthermore, biochemical analysis of the lysates reveals potential kinase activity. What have you probably isolated?
12. Phospholipid biosynthesis at the interface between the endoplasmic reticulum (ER) and the cytosol presents a number of challenges that must be solved by the cell. Explain how each of the following is handled.
a. The substrates for phospholipid biosynthesis are all water soluble, yet the end products are not.
b. The immediate site of incorporation of all newly synthesized phospholipids is the cytosolic leaflet of the ER membrane, yet phospholipids must be incorporated into both leaflets.
c. Many membrane systems in the cell, such as the plasma membrane, are unable to synthesize their own phospholipids, yet these membranes must also expand if the cell is to grow and divide.
13. What are the common fatty acid chains in phosphoglycerides, and why do these fatty acid chains differ in their number of carbon atoms by multiples of 2 ?
14. Fatty acids must associate with lipid chaperones in order to move within the cell. Why are these chaperones needed, and what is the name given to a group of proteins that are responsible for this intracellular trafficking of fatty acids? What is the key distinguishing feature of these proteins that allows fatty acids to move within the cell?
15. The biosynthesis of cholesterol is a highly regulated process. What is the key regulated enzyme in cholesterol biosynthesis? This enzyme is subject to feedback inhibition. What is feedback inhibition? How does this enzyme sense cholesterol levels in a cell?
16. Phospholipids and cholesterol must be transported from their site of synthesis to various membrane systems within cells. One way of doing this is through vesicular transport, as is the case for many proteins in the classic secretory pathway (see Chapter 14). However, phospholipid and cholesterol membrane-to-membrane transport in cells does not occur solely by vesicular transport. What is the evidence for this statement? What appear to be the major mechanisms for phospholipid and cholesterol transport?
17. Explain the mechanism by which statins lower "bad" cholesterol.

## References

The Lipid Bilayer: Composition and Structural Organization McMahon, H., and J. L. Gallop. 2005. Membrane curvature and mechanisms of dynamic cell membrane remodeling. Nature 438:590-596.

Mukherjee, S., and F. R. Maxfield. 2004. Membrane domains. Annu. Rev. Cell Dev. Biol. 20:839-866.

Ploegh, H. 2007. A lipid-based model for the creation of an escape hatch from the endoplasmic reticulum. Nature 448:435-438.

Simons, K., and D. Toomre. 2000. Lipid rafts and signal transduction. Nat. Rev. Mol. Cell Biol. 1:31-41.

Simons, K., and W. L. C. Vaz. 2004. Model systems, lipid rafts, and cell membranes. Annu. Rev. Biophys. Biomol. Struct. 33:269-295.

Tamm, L. K., V. K. Kiessling, and M. L. Wagner. 2001. Membrane dynamics. Encyclopedia of Life Sciences. Nature Publishing Group.

Vance, D. E., and J. E. Vance. 2002. Biochemistry of Lipids, Lipoproteins, and Membranes, 4th ed. Elsevier.

Van Meer, G. 2006. Cellular lipidomics. EMBO J. 24:3159-3165.
Yeager, P. L. 2001. Lipids. Encyclopedia of Life Sciences. Nature Publishing Group.

Zimmerberg, J., and M. M. Kozlov. 2006. How proteins produce cellular membrane curvature. Nat. Rev. Mol. Cell Biol. 7:9-19.

## Membrane Proteins: Structure and Basic Functions

Bowie, J. 2005. Solving the membrane protein folding problem. Nature 438:581-589.

Cullen, P. J., G. E. Cozier, G. Banting, and H. Mellor. 2001. Modular phosphoinositide-binding domains: their role in signalling and membrane trafficking. Curr. Biol. 11:R882-R893.

Engelman, D. 2005. Membranes are more mosaic than fluid. Nature 438:578-580.

Lanyi, J. K., and H. Luecke. 2001. Bacteriorhodopsin. Curr. Opin. Struct. Biol. 11:415-519.

Lee, A. G. 2005. A greasy grip. Nature 438:569-570.
MacKenzie, K. R., J. H. Prestegard, and D. M. Engelman. 1997. A transmembrane helix dimer: structure and implications. Science 276:131-133.

McIntosh, T. J., and S. A. Simon. 2006. Roles of bilayer material properties in function and distribution of membrane proteins. Annu. Rev. Biophys. Biomol. Struct. 35:177-198.

Schulz, G. E. 2000. $\beta$-Barrel membrane proteins. Curr. Opin. Struct. Biol. 10:443-447.

Wucherpfennig, K. W., et al. 2010. Cold Spring Harb. Perspect. Biol. 2:a005140.

Phospholipids, Sphingolipids, and Cholesterol: Synthesis and Intracellular Movement

Bloch, K. 1965. The biological synthesis of cholesterol. Science 150:19-28.

Daleke, D. L., and J. V. Lyles. 2000. Identification and purification of aminophospholipid flippases. Biochim. Biophys. Acta 1486:108-127.

Futerman, A., and H. Riezman. 2005. The ins and outs of sphingolipid synthesis. Trends Cell Biol. 15:312-318.

Hajri, T., and N. A. Abumrad. 2002. Fatty acid transport across membranes: relevance to nutrition and metabolic pathology. Annu. Rev. Nutr. 22:383-415.

Henneberry, A. L., M. M. Wright, and C. R. McMaster. 2002. The major sites of cellular phospholipid synthesis and molecular determinants of fatty acid and lipid head group specificity. Mol. Biol. Cell 13:3148-3161.

Holthuis, J. C. M., and T. P. Levine. 2005. Lipid traffic: floppy drives and a superhighway. Nat. Rev. Mol. Cell Biol. 6:209-220.

Ioannou, Y. A. 2001. Multidrug permeases and subcellular cholesterol transport. Nature Rev. Mol. Cell Biol. 2:657-668.

Kent, C. 1995. Eukaryotic phospholipid biosynthesis. Annu. Rev. Biochem. 64:315-343.

Maxfield, F. R., and I. Tabas. 2005. Role of cholesterol and lipid organization in disease. Nature 438:612-621.

Stahl, A., et al. 2001. Fatty acid transport proteins: a current view of a growing family. Trends Endocrin. Met. 12(6):266-273. van Meer, G., and H. Sprong. 2004. Membrane lipids and vesicular traffic. Curr. Opin. Cell Biol. 16:373-378.

this page left intentionally blank

![li1592239539b](li1592239539b.jpg)

These brightly colored FISH-painted chromosomes are not only beautiful, but also useful in revealing chromosome anomalies and in comparing karyotypes of different species. [L. Willatt/Science Source.]

In previous chapters, we learned how the structure and composition of proteins allow them to perform a wide variety of cellular functions. We also examined another vital component of cells, the nucleic acids, and the process by which information encoded in the sequence of DNA is translated into protein. In this chapter, our focus again is on DNA and proteins as we consider the characteristics of eukaryotic nuclear genomes: the features of genes and the other DNA sequences that constitute the genome, and how this DNA is structured and organized by proteins within the cell.

By the beginning of the twenty-first century, molecular biologists had completed the sequencing of the entire genomes of hundreds of viruses, scores of bacteria, and one unicellular eukaryote, the budding yeast S. cerevisiae. By now, the vast majority of the genomic sequence is also known for the fission yeast $S$. pombe, the simple plant $A$. thaliana, rice, and multiple multicellular animals (metazoans), including the roundworm C. elegans, the fruit fly D. melanogaster, mice, humans, and at least one representative each of the 35 or so metazoan phyla. Detailed analysis of these sequencing data has revealed insights into evolution, genome organization,

## G

## Genes, Genomics, and Chromosomes

and gene function. It has also allowed researchers to identify previously unknown genes and to estimate the total number of protein-coding genes in each genome. Comparisons between gene sequences often provide insight into possible functions of newly identified genes. Comparisons of genomic sequence and organization between species also help us understand the evolution of organisms.

Surprisingly, DNA sequencing revealed that large portions of the genomes of metazoans and plants do not encode mRNAs or any other RNAs required by the organism. Remarkably, such noncoding DNA constitutes about 98.5 percent of human chromosomal DNA! The noncoding DNA in multicellular organisms contains many regions that are similar, but not identical, to one another. Variations within some stretches of this repetitious DNA between individuals are so great that every person can be distinguished by a DNA "fingerprint" based on these sequence variations. Moreover, some repetitious DNA sequences are not found in the same positions in the genomes of different individuals of the same species. At one time, all noncoding DNA was collectively termed "junk DNA" and was considered to serve

### 8.1 Eukaryotic Gene Structure

8.2 Chromosomal Organization of Genes and Noncoding DNA

### 8.3 Transposable (Mobile) DNA Elements

8.4 Genomics: Genome-Wide Analysis of Gene Structure and Function

### 8.5 Structural Organization of Eukaryotic Chromosomes

8.6 Morphology and Functional Elements of Eukaryotic Chromosomes

no purpose. We now understand the evolutionary basis of all this extra DNA, and of the variation in location of certain sequences between individuals. Cellular genomes harbor transposable (mobile) DNA elements that can copy themselves and move throughout the genome. Although transposable DNA elements seem to have little function in the life cycle of an individual organism, over evolutionary time they have shaped our genomes and contributed to the rapid evolution of multicellular organisms. Moreover, we now understand that much of the DNA that does not encode proteins or stable RNAs functions as binding sites for protein complexes that regulate gene transcription.

The sheer length of cellular DNA is a significant problem with which cells must contend. The DNA in a single human cell, which measures about 2 m in total length, must be contained within nuclei with diameters of less than $10 \mu \mathrm{~m}$, a compaction ratio of greater than $10^{5}$ to 1 . In relative terms, if a cell were 1 cm in diameter (about the size of a pea), the length of DNA packed into its nucleus would be about 2 km ( 1.2 miles)! Specialized eukaryotic proteins associated with nuclear DNA exquisitely fold and organize the DNA so that it fits into nuclei. And yet, at the same time, any given portion of this highly compacted DNA can be accessed readily for transcription, replication, and repair of damage without the long DNA molecules becoming tangled or broken. Furthermore, the integrity of DNA must be maintained during the process of cell division when it is partitioned into daughter cells. In eukaryotes, the complex of DNA and the
proteins that organize it, called chromatin, can be visualized as individual chromosomes during mitosis. As we will see in this and the following chapter, the organization of DNA into chromatin allows a mechanism for regulation of gene expression that is not available in bacteria.

In the first four sections of this chapter, we provide an overview of the landscape of eukaryotic genes and genomes. First we discuss the structure of eukaryotic genes and the complexities that arise in higher organisms from the processing of mRNA precursors into alternatively spliced mRNAs. Next we discuss the main classes of eukaryotic DNA, including the special properties of transposable DNA elements and how they have shaped contemporary genomes. This background prepares us to discuss genomics, computer-based methods for analyzing and interpreting vast amounts of sequence data. The final two sections of the chapter address how DNA is physically organized in eukaryotic cells. We consider the packaging of DNA and histone proteins into the compact complexes called nucleosomes that are the fundamental building blocks of chromatin, the large-scale structure of chromosomes, and the functional elements required for chromosome duplication and segregation. Figure 8-1 provides an overview of these interrelated subjects. The understanding of genes, genomics, and chromosomes gained in this chapter will prepare us to explore current knowledge about how the synthesis and concentration of each protein and functional RNA in a cell is regulated in the following two chapters.

FIGURE 8-1 Overview of the structure of genes and chromosomes. DNA of higher eukaryotes consists of both unique and repeated sequences. Only about 1.5 percent of human DNA encodes proteins and functional RNAs. The remainder includes regulatory sequences that control gene expression, which are scattered through intergenic DNA between genes and in introns within genes. Much of this intergenic DNA, about 45 percent in humans, is derived from mobile DNA elements, genetic symbionts that have contributed to the evolution of contemporary genomes. Each chromosome consists of a single long molecule of DNA (as long as 280 Mb in humans), organized into increasing levels of condensation by the histone and nonhistone proteins with which it is intricately complexed. Each chromosome occupies its own "territory" in the nucleus.
![li1602239539b](li1602239539b.jpg)

### 8.1 Eukaryotic Gene Structure

In molecular terms, a gene is commonly defined as the entire nucleic acid sequence that is necessary for the synthesis of a functional gene product (polypeptide or RNA). According to this definition, a gene includes more than the nucleotides encoding an amino acid sequence or a functional RNA, referred to as the coding region. A gene also includes all the DNA sequences required for synthesis of a particular RNA transcript, no matter where those sequences are located in relation to the coding region. For example, in eukaryotic genes, transcription-control regions known as enhancers can lie 50 kb or more from the coding region. As we learned in Chapter 5, other critical noncoding regions in eukaryotic genes include not only the promoter, but also sequences that specify 3' cleavage and polyadenylation, known as poly(A) sites, and splicing of primary RNA transcripts, known as splice sites (see Figure 5-15). Mutations in these sequences, which control transcription initiation and RNA processing, affect the normal expression and function of RNAs, producing distinct phenotypes in mutant organisms. We examine these various control elements of genes in greater detail in Chapters 9 and 10.

Although most genes are transcribed into mRNAs, which encode proteins, some DNA sequences are transcribed into RNAs that do not encode proteins [e.g., tRNAs and rRNAs, described in Chapters 5 and 10; miRNAs and siRNAs that regulate mRNA translation and stability, discussed in Chapter 10; and long noncoding RNAs (lncRNAs) that regulate transcription, discussed in Chapter 9]. Because the DNA sequences that encode tRNAs, rRNAs, miRNAs, siRNAs, and lncRNAs can cause specific phenotypes when they are mutated, these DNA regions are generally referred to as tRNA, rRNA, miRNA, siRNA, and lncRNA genes, even though the final products of these genes are RNA molecules and not proteins.

In this section, we will examine the structure of genes in bacteria and eukaryotes and discuss how their respective gene structures influence their gene expression and evolution.

## Most Eukaryotic Genes Contain Introns and Produce mRNAs Encoding Single Proteins

As discussed in Chapter 5, many bacterial mRNAs (e.g., the mRNA encoded by the trp operon) include the coding region for several proteins that function together in a biological process. Such mRNAs are said to be polycistronic. (A cistron is a genetic unit encoding a single polypeptide.) In contrast, most eukaryotic mRNAs are monocistronic; that is, each mRNA molecule encodes a single protein. This difference between polycistronic and monocistronic mRNAs correlates with a fundamental difference in their translation.

Within a bacterial polycistronic mRNA, a ribosomebinding site is located near the start site for each of the protein-coding regions, or cistrons, in the mRNA. Translation initiation can begin at any of these multiple internal sites, producing multiple proteins (see Figure 5-13a). In most eukaryotic mRNAs, however, the $5^{\prime}$ cap directs ribosome
binding, and translation begins at the closest AUG start codon (see Figure 5-13b). As a result, translation begins only at this site. In many cases, the primary transcripts of eukaryotic protein-coding genes are processed into a single type of mRNA, which is translated to give a single type of polypeptide (see Figure 5-15).

Unlike bacterial and yeast genes, which generally lack introns, most genes in multicellular animals and plants contain introns, which are removed during RNA processing in the nucleus before the fully processed mRNA is exported to the cytosol for translation. In many cases, the introns in a gene are considerably longer than the exons. The median intron length in human genes is 3.3 kb . Some, however, are much longer: the longest known human intron is $17,106 \mathrm{bp}$ and lies within the titin gene, which encodes a structural protein in muscle cells. In comparison, most human exons contain only $50-200 \mathrm{bp}$. The typical human gene encoding an average-sized protein is about $50,000 \mathrm{bp}$ long, but more than 95 percent of that sequence consists of introns and flanking noncoding $5^{\prime}$ and $3^{\prime}$ regions.

Many large proteins in higher organisms that have repeated domains are encoded by genes consisting of repeats of similar exons separated by introns of variable length. An example is fibronectin, a component of the extracellular matrix. The fibronectin gene contains multiple copies of five types of exons (see Figure 5-16). Such genes evolved by tandem duplication of the DNA encoding the repeated exon, probably by unequal crossing over during meiosis, as shown in Figure 8-2a.

## Simple and Complex Transcription Units Are Found in Eukaryotic Genomes

The cluster of genes that forms a bacterial operon constitutes a single transcription unit, which is transcribed from a specific promoter in the DNA sequence to a termination site, producing a single primary transcript. In other words, genes and transcription units are often distinguishable in prokaryotes, since a single transcription unit contains several genes when they are part of an operon. In contrast, most eukaryotic genes are expressed from separate transcription units, so that each mRNA is translated into a single protein.

Eukaryotic transcription units, however, are classified into two types, depending on the fate of the primary transcript. The primary transcript produced from a simple transcription unit, such as the one encoding $\beta$-globin (see Figure 5-15), is processed to yield a single type of mRNA, encoding a single protein. Mutations in exons, introns, and transcription-control regions may all influence expression of the protein encoded by a simple transcription unit (Figure 8-3a). In humans, simple transcription units such as the one encoding $\beta$-globin are rare. Approximately 90 percent of human transcription units are complex. In these cases, the primary RNA transcript can be processed in more than one way, leading to formation of mRNAs containing different exons. Each alternative mRNA, however, is monocistronic: it is translated into a single polypeptide, with translation usually initiating at the first AUG in the mRNA.

![li1612239539b](li1612239539b.jpg)

FIGURE 8-2 Exon and gene duplication. (a) Exon duplication results from unequal crossing over during meiosis. Each parental chromosome (top) contains one ancestral gene containing three exons (blue) and two introns (orange). Homologous noncoding sequences called L1 long interspersed elements lie $5^{\prime}$ and $3^{\prime}$ of the gene as well as in the intron between exons 2 and 3. As we will see later in the chapter, L1 elements have been repeatedly transposed to new sites in the genome over the course of human evolution, so that all chromosomes are peppered with them. The parental chromosomes are shown displaced relative to each other, so that the L1 elements are aligned. Homologous recombination between these L1 elements as shown
would generate one recombinant chromosome in which the gene now has four exons (two copies of exon 3) and one chromosome in which the gene is missing exon 3. (b) The same process can generate duplications of entire genes. Each parental chromosome (top) contains one ancestral $\beta$-globin gene. After unequal recombination between L1 elements, subsequent independent mutations in the resulting duplicated genes could lead to slight changes in sequence that might result in slightly different functional properties of the encoded proteins. Unequal crossing over can also result from rare recombinations between unrelated sequences. See D. H. A. Fitch et al., 1991, Proc. Natl. Acad. Sci. USA 88:7396.

Multiple mRNAs can arise from a primary transcript in three ways, as shown in Figure 8-3b. Examples of all three types of alternative RNA processing occur in the genes that regulate sexual differentiation in Drosophila (see Figure 10-18). Commonly, one mRNA is produced from a complex transcription unit in some cell types, and a different mRNA is made in other cell types. For example, alternative splicing of the primary fibronectin transcript in fibroblasts and hepatocytes determines whether or not the secreted protein includes domains that adhere to cell surfaces (see Figure 5-16). The phenomenon of alternative splicing greatly expands the number of proteins encoded in the genomes of higher organisms. It is estimated that about 90 percent of human genes are contained within complex transcription units that give rise to alternatively spliced mRNAs encoding proteins with distinct functions, as for the fibroblast and hepatocyte forms of fibronectin.

The relationship between a mutation and a gene is not always straightforward when it comes to complex transcription units. A mutation in the control region or in an exon shared by alternatively spliced mRNAs will affect all the alternative proteins encoded by a given complex transcription unit. On the other hand, a mutation in an exon present in only one of the alternative mRNAs will affect only the protein encoded by that mRNA. As explained in Chapter 6, genetic complementation tests are commonly used to determine if two mutations are in the same or different genes (see Figure 6-7). However, in the complex transcription unit shown in Figure 8-3b (middle), mutations $d$ and $e$ would complement each other in a genetic complementation test, even though they occur in the same gene, because a chromosome with mutation $d$ can express a normal protein encoded by $\mathrm{mRNA}_{2}$ and a chromosome with mutation $e$ can express a normal protein encoded by $\mathrm{mRNA}_{1}$. Both mRNAs

(a) Simple transcription unit
![li1622239539b](li1622239539b.jpg)
produced from this gene would be present in a diploid cell carrying both mutations, generating both protein products and hence a wild-type phenotype. However, a chromosome with mutation $c$ in an exon common to both mRNAs would not complement either mutation $d$ or mutation $e$. In other words, mutation $c$ would be in the same complementation groups as mutations $d$ and $e$, even though $d$ and $e$ themselves would not be in the same complementation group! Given these complications with the genetic definition of a gene, the genomic definition outlined at the beginning of this section is commonly used. In the case of protein-coding genes, a gene is the DNA sequence transcribed into a pre-mRNA precursor, equivalent to a transcription unit, plus any other

FIGURE 8-3 Simple and complex eukaryotic transcription units. (a) A simple transcription unit includes a region that encodes one protein, extending from the $5^{\prime}$ cap site to the $3^{\prime}$ poly(A) site, and associated control regions. Introns lie between exons (light blue rectangles) and are removed during processing of the primary transcripts (dashed red lines); thus they do not occur in the functional monocistronic mRNA. Mutations in a transcription-control region $(a, b)$ may reduce or prevent transcription, thus reducing or eliminating synthesis of the encoded protein. A mutation within an exon (c) may result in an abnormal protein with diminished activity. A mutation within an intron (d) that introduces a new splice site results in an abnormally spliced mRNA encoding a nonfunctional protein. (b) Complex transcription units produce primary transcripts that can be processed in alternative ways. (Top) If a primary transcript contains alternative splice sites, it can be processed into mRNAs with the same $5^{\prime}$ and $3^{\prime}$ exons but different internal exons. (Middle) If a primary transcript has two poly(A) sites, it can be processed into mRNAs with alternative $3^{\prime}$ exons. (Bottom) If alternative promoters ( $f$ or $g$ ) are active in different cell types, $\mathrm{mRNA}_{1}$, produced in a cell type in which $f$ is activated, has a different first exon (1A) than $\mathrm{mRNA}_{2}$, which is produced in a cell type in which $g$ is activated (and in which exon 1B is used). Mutations in control regions ( $a$ and $b$ ) and in regions within exons shared by the alternative mRNAs (designated $c$ ) affect the proteins encoded by both alternatively processed mRNAs. In contrast, mutations (designated $d$ and $e$ ) within exons unique to one of the alternatively processed mRNAs affect only the protein translated from that mRNA. For genes that are transcribed from different promoters in different cell types (bottom), mutations in different control regions ( $f$ and $g$ ) affect expression only in the cell type in which that control region is active.
regulatory elements required for synthesis of the primary transcript. The various proteins encoded by the alternatively spliced mRNAs expressed from one gene are called isoforms.

## Protein-Coding Genes May Be Solitary or Belong to a Gene Family

The nucleotide sequences within chromosomal DNA can be classified on the basis of their structure and function, as shown in Table 8-1. Here we examine the properties of each class, beginning with protein-coding genes, which comprise two groups.

In multicellular organisms, roughly $25-50$ percent of the protein-coding genes are represented only once in the haploid genome and thus are termed solitary genes. A wellstudied example of a solitary protein-coding gene is the chicken lysozyme gene. Lysozyme, an enzyme that cleaves the polysaccharides in bacterial cell walls, is an abundant component of chicken egg-white protein and is also found in human tears. Its activity helps to keep the egg and the surface of the eye sterile. The $15-\mathrm{kb}$ DNA sequence encoding chicken lysozyme constitutes a simple transcription unit containing four exons and three introns. The flanking regions, extending about 20 kb upstream and downstream from the transcription unit, do not encode any detectable mRNAs, and are thus examples of intergenic regions.

Duplicated genes constitute the second group of protein-coding genes. These genes have close but nonidentical

| Class | Length | Copy Number in Human Genome | Fraction of Human Genome (\%)* |
| :--: | :--: | :--: | :--: |
| Protein-coding genes | $0.5-2200 \mathrm{~kb}$ | $\sim 21,000$ | $\sim 40^{* *}(2.0 \dagger)$ |
| Long noncoding RNA genes | $0.2-50 \mathrm{~kb}$ | $\sim 10,000$ | $\sim 15(0.9 \dagger)$ |
| Tandemly repeated genes |  |  |  |
| U2 snRNA | $6.1 \mathrm{~kb} \ddagger$ | $\sim 20$ | $<0.0001$ |
| rRNAs | $43 \mathrm{~kb} \ddagger$ | $\sim 300$ | 0.4 |
| Repetitious DNA |  |  |  |
| Simple-sequence DNA | $1-500 \mathrm{bp}$ | Variable | $\sim 6$ |
| Interspersed repeats (mobile DNA elements) |  |  |  |
| DNA transposons | $2-3 \mathrm{~kb}$ | 300,000 | 3 |
| LTR retrotransposons | $6-11 \mathrm{~kb}$ | 440,000 | 8 |
| Non-LTR retrotransposons |  |  |  |
| LINEs | $6-8 \mathrm{~kb}$ | 860,000 | 21 |
| SINEs | $100-400 \mathrm{bp}$ | $1,600,000$ | 13 |
| Processed pseudogenes | Variable | $\sim 12,500$ | $\sim 0.4$ |
| Intergenic regions | Variable | n.a. | $\sim 25$ |

*The sum of "Fraction of the Human Genome (\%)" totals more than $100 \%$ because mobile DNA elements are counted twice: once to show the different classes of human mobile DNA elements; and second as part of the intergenic regions and protein coding genes where they are located in introns and 3' untranslated regions of terminal exons.
**Complete transcription units including exons and introns.
$\dagger$ Total length of all exons. Protein-coding regions total 1.2 percent of the genome.
$\ddagger$ Length of each repeat in a tandemly repeated sequence.
source: Data from International Human Genome Sequencing Consortium, 2001, Nature 409:860 and 2004, Nature 431:931.
sequences and are often located within $5-50 \mathrm{~kb}$ of one another. A set of duplicated genes that encodes proteins with similar but nonidentical amino acid sequences is called a gene family; the encoded, closely related, homologous proteins constitute a protein family. A few protein families, such as protein kinases, vertebrate immunoglobulins, and olfactory receptors, include hundreds of members. Most protein families, however, include from just a few to 30 or so members; common examples are cytoskeletal proteins, the myosin heavy chain, and the $\alpha$-likes and $\beta$-likeas globins in vertebrates.

The genes encoding the $\beta$-likeas globins are a good example of a gene family. As shown in Figure 8-4a, the $\beta$-likeas globin gene family contains five functional genes, designated $H B B$ (encoding the most abundant adult $\beta$-globin), $H B D$ (a minor adult $\beta$-globin), $H B G 1$ and $H B G 2$ (fetal $\beta$-globins), and HBE1 (embryonic $\beta$-globin). Two identical $\beta$-likeas globin polypeptides combine with two identical $\alpha$-likes globin polypeptides (encoded by another gene family expressed during embryonic, fetal, and adult stages of development) and four heme prosthetic groups to form a hemoglobin molecule (see Figures 3-14 and 12-20). All the hemoglobins formed from the different $\alpha$-likes and $\beta$-likeas globins carry oxygen in the blood, but they exhibit somewhat different properties that are suited to their specific functions in human physiology.

For example, hemoglobins containing either the HBG1- or HBG2-encoded polypeptides are expressed only during fetal life. Because these fetal hemoglobins have a higher affinity for oxygen than adult hemoglobins, they can effectively extract oxygen from the maternal circulation in the placenta. The lower oxygen affinity of adult hemoglobins, which are expressed after birth, permits better release of oxygen to the tissues, especially muscles, which have a high demand for oxygen during exercise. The embryonic hemoglobin assembled from polypeptides encoded by the HBE1 gene and the embryonic $\alpha$-likes globin gene $H B Z$ has an even higher affinity for oxygen than the fetal and adult hemoglobins.

The different $\beta$-like globin genes arose by duplication of an ancestral gene, most likely as the result of unequal crossing over during meiotic recombination in a developing germ cell (egg or sperm) (see Figure 8-2b). Over evolutionary time, the two copies of the gene that resulted accumulated random mutations, resulting in sequence drift. Beneficial mutations that conferred some refinement in the basic oxygen-carrying function of hemoglobin were retained by natural selection. Repeated gene duplications and subsequent sequence drift and selection are thought to have generated the contemporary $\beta$-likeas globin genes observed in humans and other mammals today.

![li1632239539b](li1632239539b.jpg)
(b) S. cerevisiae (chromosome III)
![li1642239539b](li1642239539b.jpg)

FIGURE 8-4 Comparison of gene density in higher and lower eukaryotes. (a) In this diagram of the $\beta$-globin gene cluster on human chromosome 11, the green boxes represent exons of $\beta$-globin-related genes. Exons spliced together to form one mRNA are connected by caret-like spikes. The human $\beta$-globin gene cluster contains two pseudogenes (white); these regions are related to the functional $\beta$-globin genes but are not transcribed. Each red arrow indicates the location of an Alu sequence, a roughly 300-bp noncoding repeated sequence that is abundant in the human genome. See F. S. Collins and S. M. Weissman, 1984, Prog. Nucl. Acid Res. Mol. Biol. 31:315. (b) In this diagram of yeast DNA from chromosome III, the green boxes indicate open reading frames. Most of these potential protein-coding sequences are functional genes without introns. Note the much higher proportion of noncoding to coding sequences in the human DNA than in the yeast DNA. See S. G. Oliver et al., 1992, Nature 357:28.

Two regions in the human $\beta$-like globin gene cluster contain nonfunctional sequences, called pseudogenes, that are similar to the functional $\beta$-like globin genes (see Figure 8-4a). Sequence analysis shows that these pseudogenes have the same apparent exon-intron structure as the functional $\beta$-like globin genes, suggesting that they arose by duplication of the same ancestral gene. However, there was little selective pressure to maintain the function of these genes. Consequently, sequence drift during evolution generated sequences that either terminate translation or block mRNA processing, rendering these regions nonfunctional. Because such pseudogenes are not deleterious, they remain in the genome and mark the location of a gene duplication that occurred in one of our ancestors.

Duplications of segments of a chromosome (called segmental duplication) occurred fairly often during the evolution of multicellular plants and animals. As a result, a large fraction of the genes in these organisms today have been duplicated, allowing the process of sequence drift to generate gene families and pseudogenes. The extent of sequence divergence between duplicated copies of the genome and characterization of the homologous genomic sequences in related organisms allow us to estimate the time in evolutionary history when the duplication occurred. For example, the human fetal globin genes (HGB1 and HGB2) evolved following the duplication of a $5.5-\mathrm{kb}$ region in the $\beta$-globin locus that included the single $H G B$-globin gene in the common ancestor of catarrhine primates (Old World monkeys, apes, and humans) and platyrrhine primates (New World monkeys) about 50 million years ago.

Although members of gene families that arose relatively recently in evolution, such as the genes of the human $\beta$-globin family, are often found near one another on the same chromosome, members of gene families may also be found on different chromosomes in the same organism. This is the case for the human $\alpha$-like globin genes, which were separated from the $\beta$-globin genes by an ancient chromosomal translocation. Both the $\alpha$ - and $\beta$-globin genes evolved from a single ancestral globin gene that was duplicated (see Figure 8-2b) to generate the predecessors of the contemporary $\alpha$ - and $\beta$-globin genes in mammals. Both the primordial $\alpha$ - and $\beta$-globin genes then underwent further duplications to generate the different genes of the $\alpha$ - and $\beta$-globin gene clusters found in mammals today.

Several different gene families encode the various proteins that make up the cytoskeleton. These proteins are present in varying amounts in almost all cells. In vertebrates, the major cytoskeletal proteins are the actins, tubulins, and intermediate filament proteins such as the keratins, discussed in Chapters 17, 18, and 20. We examine the origin of one such family, the tubulin family, in Section 8.4. Although the physiological rationale for the cytoskeletal protein families is not as obvious as it is for the globins, the different members of a family probably have similar but subtly different functions suited to the particular type of cell in which they are expressed.

## Heavily Used Gene Products Are Encoded by Multiple Copies of Genes

In vertebrates and invertebrates, the genes encoding ribosomal RNAs and some other nonprotein-coding RNAs, such as those involved in RNA splicing, occur as tandemly repeated arrays. These multiple tandemly repeated genes are distinguished from the duplicated genes of gene families in that they encode identical, or nearly identical, proteins or functional RNAs. Most often, copies of these sequences appear one after the other, in a head-to-tail fashion, over a long stretch of DNA. Within a tandem array of rRNA genes, each copy is nearly exactly like all the others. Although the transcribed portions of the genes are the same, the nontranscribed regions between the transcribed regions can vary in length and sequence.

These tandemly repeated rRNA genes have evolved to meet the great cellular demand for their transcripts. To understand why, consider that a fixed maximal number of rRNA molecules can be produced from a single gene during one cell generation when the gene is fully loaded with RNA polymerase molecules. If more RNA is required than can be transcribed from one gene, multiple copies of the gene are necessary. For example, during early embryonic development in humans, many embryonic cells have a doubling time of about 24 hours and contain 5 million-10 million ribosomes. To produce enough rRNA to form this many ribosomes, an embryonic human cell needs at least 100 copies of genes encoding the large and small rRNA subunits, and most of these genes must be close to maximally active for the cell to divide every 24 hours; that is, multiple RNA polymerases must be transcribing each rRNA gene at the same time (see Figure 10-39). Indeed, all eukaryotes, including yeasts, contain 100 or more copies of the 5 S rRNA gene and of the genes encoding the other rRNAs.

The genes encoding tRNA and the genes encoding the histone proteins are also present in multiple copies. As we will see later in this chapter, histones bind and organize nuclear DNA. Just as the cell requires multiple rRNA and tRNA genes to produce sufficient numbers of ribosomes and tRNAs, multiple copies of the histone genes are required to produce sufficient histone protein to bind the large amount of nuclear DNA produced in each round of cell replication. While tRNA
and histone genes often occur in clusters, they generally do not occur in tandem arrays in the human genome.

## Nonprotein-Coding Genes Encode Functional RNAs

In addition to rRNA and tRNA genes, there are thousands of additional genes that are transcribed into nonproteincoding RNAs, some with various known functions and many whose functions are not yet known. For example, small nuclear RNAs (snRNAs) function in RNA splicing, and small nucleolar RNAs (snoRNAs) function in rRNA processing and base modification in the nucleolus. The RNase P RNA functions in tRNA processing, and a large family ( $\sim 2000$ in humans) of short micro-RNAs (miRNAs) regulates the translation and stability of specific mRNAs. The functions of these nonprotein-coding RNAs are discussed in Chapter 10. An RNA found in telomerase (see Figure 8-44 below) functions in maintaining the sequence at the ends of chromosomes, and the 7SL RNA functions in the transport of secreted proteins and most membrane proteins into the endoplasmic reticulum (see Chapter 13). These and other nonprotein-coding RNAs encoded in the human genome, and their functions when known, are listed in Table 8-2. Recent advances in DNA sequencing have led to the discovery of about 10,000 long noncoding RNAs (lncRNAs) in nuclei of mammalian cells. Some of these have been found

TABLE 8-2 Known Nonprotein-Coding RNAs and Their Functions

| RNA | Number of Genes in Human Genome | Function |
| :--: | :--: | :--: |
| rRNAs | $\sim 300$ | Protein synthesis |
| tRNAs | $\sim 500$ | Protein synthesis |
| snRNAs | $\sim 40$ | mRNA splicing |
| U7 snRNA | 1 | Histone mRNA $3^{\prime}$ processing |
| snoRNAs | $\sim 85$ | Pre-rRNA processing and rRNA modification |
| miRNAs | $\sim 2000$ | Regulation of gene expression |
| Xist | 1 | X-chromosome inactivation |
| 7SK | 1 | Transcription control |
| RNase P RNA | 1 | tRNA $5^{\prime}$ processing |
| 7SL RNA | 3 | Protein secretion (component of signal recognition particle, SRP) |
| RNase MRP RNA | 1 | rRNA processing, mtDNA* replication |
| Telomerase RNA | 1 | Template for addition of telomeres |
| Vault RNAs | 3 | Components of Vault ribonucleoproteins (RNPs), function unknown |
| hY1, hY3, hY4, hY5 | $\sim 30$ | Components of ribonucleoproteins (RNPs), function unknown |
| H19 | 1 | Unknown |

[^0]
[^0]:    *Mitochondrial DNA
    source: Data from International Human Genome Sequencing Consortium, 2001, Nature 409:860, and P. D. Zamore and B. Haley, 2005, Science 309:1519.

to function in regulating the expression of specific proteincoding genes. Pursuing the functions of lncRNAs is currently a highly active area of research.

## KEY CONCEPTS OF SECTION 8.1

## Eukaryotic Gene Structure

- In molecular terms, a gene is the entire DNA sequence required for synthesis of a functional protein or RNA molecule. In addition to the coding regions (exons), a gene includes control regions, and in multicellular animals and plants, most genes include introns.
- A simple eukaryotic transcription unit produces a single monocistronic mRNA, which is translated into a single protein.
- A complex eukaryotic transcription unit is transcribed into a primary transcript that can be processed into two or more different monocistronic mRNAs depending on the choice of splice sites or polyadenylation sites. A complex transcription unit with alternative promoters generates two or more different mRNAs in different cell types (see Figure 8-3b).
- Many complex transcription units (e.g., the fibronectin gene) express one mRNA in one cell type and an alternative mRNA in a different cell type.
- About half the protein-coding genes in vertebrate genomic DNA are solitary genes, each occurring only once in the haploid genome. The remainder are duplicated genes, which arose by duplication of an ancestral gene and subsequent independent mutations (see Figure 8-2b). The proteins encoded by a gene family have homologous but nonidentical amino acid sequences and exhibit similar but slightly different properties.
- In invertebrates and vertebrates, rRNAs are encoded by multiple copies of genes located in tandem arrays in genomic DNA. Multiple copies of rRNA, snRNA, and histone genes also occur, often in clusters, but not generally in tandem arrays.
- Many genes also encode functional RNAs that are not translated into proteins but nonetheless perform significant functions, such as rRNA, tRNA, and snRNA. Among these are micro-RNAs, whose biological significance in regulating gene expression has only recently been appreciated. Understanding the function of thousands of newly discovered nuclear long noncoding RNAs (lncRNAs) is currently an intense area of research.


### 8.2 Chromosomal Organization of Genes and Noncoding DNA

Having reviewed the relationship between transcription units and genes, we now consider the organization of genes on chromosomes and the relationship of noncoding DNA sequences to coding sequences.

## Genomes of Many Organisms Contain Nonfunctional DNA

Comparisons of the total chromosomal DNA per cell in various species first suggested that much of the DNA in certain organisms does not encode functional RNA or have any apparent regulatory function. For example, yeasts, fruit flies, chickens, and humans have successively more DNA in their haploid chromosome sets ( $12.5,180,1300$, and 3300 Mb , respectively), in keeping with what we perceive to be the increasing complexity of these organisms. Yet the vertebrates with the greatest amount of DNA per cell are amphibians, which are surely less complex than humans in their structure and behavior. Even more surprising, the unicellular protozoan Amoeba dubia has 200 times more DNA per cell than humans. Many plant species also have considerably more DNA per cell than humans have; tulips, for example, have 10 times as much DNA per cell as humans. The DNA content per cell also varies considerably between closely related species. All insects or all amphibians would appear to be similarly complex, but the amount of haploid DNA in species within each of these phylogenetic classes varies by a factor of 100 .

Sequencing and identification of exons in chromosomal DNA have provided direct evidence that the genomes of higher eukaryotes contain large amounts of noncoding DNA. For instance, only a small portion of the $\beta$-globin gene cluster of humans, which is about 80 kb long, encodes protein (see Figure 8-4a). In contrast, a typical $80-\mathrm{kb}$ stretch of DNA from the yeast $S$. cerevisiae, a single-celled eukaryote, contains many closely spaced protein-coding sequences without introns and relatively much less noncoding DNA (see Figure 8-4b). Moreover, the introns in globin genes are considerably shorter than those in most human genes. Globin proteins comprise about 50 percent of the total protein in developing red blood cells (erythroid progenitors), and the globin genes are expressed at maximum rates (i.e., a new RNA polymerase initiates transcription as soon as the previous polymerase transcribes far enough from the promoter to allow it to do so). Consequently, there has been selective pressure on globin genes for small introns that are compatible with the required high rate of globin mRNA transcription and processing. However, the vast majority of human genes are expressed at much lower levels, which require production of one encoded mRNA on a time scale of only tens of minutes or hours. Consequently, there has been little selective pressure to reduce the sizes of introns in most human genes.

The density of genes varies among regions of human chromosomal DNA, from "gene-rich" regions, where a few hundred base pairs separate transcription units, to large gene-poor "gene deserts," where intergenic regions are a few million base pairs long. Of the 96 percent of human genomic DNA that has been sequenced, only about 2.9 percent corresponds to exons, and only about 1.2 percent encodes proteins. (The fraction of the genome that corresponds to exons is much larger than the fraction that encodes proteins because many protein-coding genes include exons for long $3^{\prime}$ untranslated regions and because there are many

exons in nonprotein-coding lncRNAs; see Chapter 9.) We learned in the previous section that the intron sequences of most human genes are significantly longer than the exon sequences. Approximately 55 percent of human genomic DNA is thought to be transcribed into pre-mRNAs, prelncRNAs, or other nonprotein-coding RNAs in one cell or another, but some 95 percent of this sequence is intronic and is thus removed by RNA splicing. The remaining 45 percent of human DNA constitutes noncoding DNA between genes as well as the regions of repeated DNA sequences that make up the centromeres and telomeres of the human chromosomes. Consequently, about 97 percent of human DNA does not encode proteins, functional noncoding RNAs, or potentially functional lncRNAs.

Different selective pressures may account, at least in part, for the remarkable difference in the amount of nonfunctional DNA in different organisms. For example, many microorganisms must compete with other species of microorganisms in the same environment for limited amounts of available nutrients, and metabolic economy is thus a critical characteristic for these organisms. Because synthesis of nonfunctional (i.e., noncoding) DNA requires time, nutrients, and energy, presumably there was selective pressure to lose nonfunctional DNA during the evolution of rapidly growing microorganisms such as the yeast S. cerevisiae. On the other hand, natural selection in vertebrates depends largely on their behavior. The energy invested in DNA synthesis is trivial compared with the metabolic energy required for the movement of muscles and the function of the nervous system; thus there may have been little selective pressure on vertebrates to eliminate nonfunctional DNA. Furthermore, the replication time of cells in most vertebrates and plants is much longer than in rapidly growing microorganisms, so there may have been little selective pressure to eliminate nonfunctional DNA in order to permit rapid cellular replication.

## Most Simple-Sequence DNAs Are Concentrated in Specific Chromosomal Locations

Besides duplicated protein-coding genes and tandemly repeated genes, eukaryotic cells contain multiple copies of other DNA sequences, generally referred to as repetitious DNA (see Table 8-1). Of the two main types of repetitious DNA, the less prevalent is simple-sequence DNA, or satellite DNA. This type of DNA, which constitutes about 6 percent of the human genome, is composed of perfect or nearly perfect repeats of relatively short sequences. The second, more common type of repetitious DNA, collectively called interspersed repeats, is composed of much longer sequences. These sequences, consisting of several types of transposable elements, are discussed in Section 8.3.

The length of each repeat in simple-sequence DNA can range from 1 to 500 base pairs. DNA sequences in which the repeats each contain $1-13 \mathrm{bp}$ are often called microsatellites. Most microsatellite DNA has a repeat length of $1-4 \mathrm{bp}$, and the repeats usually occur in tandem sequences of 150 repeats or fewer. Microsatellites are thought to have
(a) Normal replication
![li1652239539b](li1652239539b.jpg)
(c) Second replication

Daughter DNA with one extra repeat
![li1662239539b](li1662239539b.jpg)

FIGURE 8-5 Generation of microsatellite repeats by backward slippage of the nascent daughter strand during DNA replication. If, during replication (a), the nascent daughter strand "slips" backward relative to the template strand by one repeat, one new copy of the repeat is added to the daughter strand when DNA replication continues (b). An extra copy of the repeat forms a single-stranded loop in the daughter strand of the daughter duplex DNA molecule. If this single-stranded loop is not removed by DNA repair proteins before the next round of DNA replication (c), the extra copy of the repeat is added to one of the double-stranded daughter DNA molecules.
originated by "backward slippage" of a daughter strand on its template strand during DNA replication so that the same short sequence was copied twice (Figure 8-5).

5
Microsatellites occasionally occur within transcription units. Some individuals are born with a larger number of repeats in specific genes than are observed in the general population, presumably because of daughter-strand slippage during DNA replication in the germ cells from which they and their forebears developed. Such expanded microsatellites have been found to cause at least 14 different types of neuromuscular diseases, depending on the gene in which they occur. In some cases, expanded microsatellites behave like a recessive mutation because they interfere with just the function or expression of the gene in which they

occur. But in the more common types of diseases associated with expanded microsatellites, the expanded microsatellites behave like dominant mutations. In some of these diseases, such as Huntington disease, triplet repeats occur within a coding region, resulting in the formation of long polymers of a single amino acid that may aggregate over time in longlived neuronal cells, eventually interfering with normal cellular function. For example, expansion of a CAG repeat in the first exon of the gene involved in Huntington disease leads to synthesis of long stretches of polyglutamine, which over several decades form toxic aggregates resulting in neuronal cell death in patients with the disease.

Pathogenic expanded repeats can also occur in the noncoding regions of some genes, where they are thought to function as dominant mutations because they interfere with the processing of a subset of mRNAs in the muscle cells and neurons where the affected genes are expressed. For example, in patients with myotonic dystrophy type 1, transcripts of the $D M P K$ gene contain between 50 and 1500 repeats of the sequence CUG in the $3^{\prime}$ untranslated region, compared with 5-34 repeats in unaffected individuals. The extended stretch of CUG repeats in affected individuals is thought to form a long RNA hairpin (see Figure 5-9), which binds and sequesters nuclear RNA-binding proteins that normally regulate alternative RNA splicing of a subset of pre-mRNAs essential for muscle and nerve cell function.

Most simple-sequence satellite DNA is composed of repeats of $14-500 \mathrm{bp}$ in tandem arrays $20-100 \mathrm{~kb}$ long. In situ hybridization studies with metaphase chromosomes have localized this simple-sequence DNA to specific chromosomal regions. Much of this DNA lies near centromeres, the discrete chromosomal regions that attach to spindle microtubules during mitosis and meiosis (Figure 8-6). Experiments in the fission yeast $S$. pombe indicate that these sequences are required to form a specialized chromatin structure called centromeric heterochromatin, necessary for the proper segregation of chromosomes to daughter cells during mitosis. Simple-sequence DNA is also found in long tandem repeats at the ends of chromosomes, the telomeres, where it functions to maintain those chromosome ends and prevent their joining to the ends of other DNA molecules, as discussed further in the last section of this chapter.

## DNA Fingerprinting Depends on Differences in Length of Simple-Sequence DNAs

Within a species, the nucleotide sequences of the repeat units composing a simple-sequence DNA tandem array are highly conserved among individuals. In contrast, the number of repeats, and thus the length of simple-sequence tandem arrays containing the same repeat unit, is quite variable among individuals. These differences in length are thought to result from unequal crossing over within regions of simple-sequence DNA during meiosis. As a consequence of this unequal crossing over, the lengths of some tandem arrays are unique in each individual.
![li1672239539b](li1672239539b.jpg)

EXPERIMENTAL FIGURE 8-6 Simple-sequence DNA is localized at the centromere in mouse chromosomes. Purified simple-sequence DNA from mouse cells was copied in vitro using E. coli DNA polymerase I and fluorescently labeled dNTPs to generate a fluorescently labeled DNA "probe" for mouse simple-sequence DNA. Chromosomes from cultured mouse cells were fixed and denatured on a microscope slide, and the chromosomal DNA was then hybridized in situ to the labeled probe (light blue). The slide was also stained with DAPI, a DNA-binding dye, to visualize the full length of the chromosomes (dark blue). Fluorescence microscopy shows that the simple-sequence probe hybridizes primarily to one end of the telocentric mouse chromosomes (i.e., chromosomes in which the centromeres are located near one end). [Courtesy of Sabine Mal, Ph.D., Manitoba Institute of Cell Biology, Canada.]

In humans and other mammals, some simple-sequence DNA exists in relatively short $1-5-\mathrm{kb}$ regions made up of 20-50 repeat units, each containing 14-100 bp. These regions are called minisatellites, in contrast to microsatellites made up of tandem repeats of 1-13 bp. Even slight differences in the total lengths of various minisatellites from different individuals can be detected by Southern blotting (see Figure 6-24). This technique was exploited in the first application of DNA fingerprinting, which was developed to detect DNA polymorphisms (i.e., differences in sequence between individuals of the same species) (Figure 8-7). Today the far more sensitive polymerase chain reaction (PCR) technique (see Figure 6-18) is generally used in forensic genetic testing. Microsatellites consisting of tandem repeats of four bases in 30-50 copies are usually analyzed today. The exact number of repeats at a specific location in the genome generally varies between the two homologous chromosomes of an individual (one inherited from the mother and one from the father) and between the Y chromosomes of different males. A mixture of pairs of PCR primers that hybridize to unique sequences flanking 13 of these short tandem repeats and a Y-chromosome short tandem repeat are used to amplify

![li1682239539b](li1682239539b.jpg)

FIGURE 8-7 Distinguishing individuals by DNA fingerprinting. (a) In this analysis of paternity, several minisatellite repeat lengths were determined by Southern blot analysis of restriction enzyme-digested genomic DNA and hybridization with a probe for a sequence shared by several minisatellite sequences. This method generated hypervariable multiband patterns for each individual called "DNA fingerprints." Lane $M$ shows the pattern of restriction fragment bands using the mother's DNA; C, using the child's DNA; and F1 and F2 using DNA from two potential fathers. The child has minisatellite repeat lengths inherited from either the mother or F1, demonstrating that F1 is the father. Arrows indicate restriction fragments from F1, but not F2, found in the child's DNA. (b) In these "DNA fingerprints" of a specimen isolated from a rape victim and three men suspected of the crime, it is clear that minisatellite repeat lengths in the specimen match those of suspect 1. The victim's DNA was included in the analysis to ensure that the specimen DNA was not contaminated with DNA from the victim.

DNA in a sample from one individual. The resulting mixture of PCR product lengths is unique in the human population, except for identical twins. The use of PCR allows analysis of minute amounts of DNA, and individuals can be distinguished more precisely and reliably than by conventional fingerprinting.

## Unclassified Intergenic DNA Occupies a Significant Portion of the Genome

About 45 percent of human DNA lies between transcription units. Much of this sequence is not repeated anywhere else in the genome. Enhancers on the order of $50-200 \mathrm{bp}$ in length that help to regulate transcription at distant promoters occur in these long stretches of intergenic DNA, as well as in introns. These enhancers (discussed in Chapter 9) are often conserved during evolution, while the neighboring intergenic sequences are not conserved. Other conserved
intergenic regions may perform significant functions that are not yet understood. For example, they may contribute to the structures of chromosomes discussed in Section 8.6.

## KEY CONCEPTS OF SECTION 8.2

## Chromosomal Organization of Genes and Noncoding DNA

- In the genomes of prokaryotes and most lower eukaryotes, which contain few nonfunctional sequences, coding regions are densely arrayed along the genomic DNA.
- In contrast, genomes of multicellular animals and plants contain many sequences that do not apparently code for functional RNAs or have any regulatory function. Much of this nonfunctional DNA is composed of repeated sequences. In humans, only about 2.9 percent of total DNA (the exons) actually encodes proteins or functional RNAs.
- Variation in the amount of nonfunctional DNA in the genomes of different species is largely responsible for the lack of a consistent relationship between the amount of DNA in the haploid chromosomes of an animal or plant and its phylogenetic complexity.
- Simple-sequence DNA, consisting of short sequences repeated in long tandem arrays, is preferentially located in centromeres and telomeres.
- The length of a particular simple-sequence tandem array is quite variable between individuals in a species, probably because of unequal crossing over during meiosis. Differences in the lengths of some simple-sequence tandem arrays form the basis for DNA fingerprinting (see Figure 8-7).


### 8.3 Transposable (Mobile) DNA Elements

Interspersed repeats, the second type of repetitious DNA in eukaryotic genomes, is composed of a very large number of copies of relatively few sequence families (see Table 8-1). Also known as moderately repeated DNA, or intermediaterepeat DNA, these sequences are interspersed throughout mammalian genomes and make up $25-50$ percent of mammalian DNA ( $\sim 45$ percent of human DNA).

Because interspersed repeats have the unique ability to "move" in the genome, they are collectively referred to as transposable DNA elements or mobile DNA elements (we use these terms interchangeably). Although transposable DNA elements were originally discovered in eukaryotes, they are also found, although less frequently, in prokaryotes. The process by which these sequences are copied and inserted into a new site in the genome is called transposition. Transposable DNA elements are essentially molecular symbionts that in most cases appear to have no specific function in the biology of their host organisms, but exist only to maintain themselves. For this reason, Francis Crick referred to them as "selfish DNA."

When transposition occurs in germ cells, the transposed sequences at their new sites are passed on to succeeding generations. In this way, mobile elements have multiplied and slowly accumulated in eukaryotic genomes over evolutionary time. Since mobile elements are eliminated from eukaryotic genomes very slowly, they now constitute a significant portion of the genomes of many eukaryotes.

Mobile elements are not only the source for much of the DNA in our genomes, but also provide a second mechanism, in addition to meiotic recombination, for bringing about chromosomal DNA rearrangements during evolution (see Figure 8-2). One reason for this is that during transposition of a particular mobile element, adjacent DNA is sometimes also mobilized (see Figure 8-19 below). Transpositions occur rarely: in humans, there is about one new germ-line transposition for every eight individuals. Since 97 percent of our DNA is noncoding, most transpositions have no deleterious effects. But over time, they have played an essential part in the evolution of genes that have multiple exons and of genes whose expression is restricted to specific cell types or developmental periods. In other words, although transposable elements probably evolved as cellular symbionts, they have had an important function in the evolution of complex multicellular organisms.

Transposition may also occur within a somatic cell; in this case, the transposed sequence is transmitted only to the daughter cells derived from that cell. In rare cases, such somatic-cell transposition may lead to a somatic-cell mutation with detrimental phenotypic effects, such as the inactivation of a tumor-suppressor gene (see Chapter 24). In
this section, we first describe the structure and transposition mechanisms of the major types of transposable DNA elements and then consider their likely role in evolution.

## Movement of Mobile Elements Involves a DNA or an RNA Intermediate

Barbara McClintock discovered the first mobile elements while doing classical genetic experiments in maize (corn) during the 1940s. She characterized genetic entities that could move into and back out of genes, changing the phenotypes of corn kernels. Her theories were very controversial until similar mobile elements were discovered in bacteria, where they were characterized as specific DNA sequences, and the molecular basis of their transposition was deciphered.

As research on mobile elements progressed, they were found to fall into two categories: (1) those that transpose directly as DNA and (2) those that transpose via an RNA intermediate transcribed from the mobile element by an RNA polymerase and then converted back into double-stranded DNA by a reverse transcriptase (Figure 8-8). Mobile elements that transpose directly as DNA are generally referred to as DNA transposons, or simply transposons. Eukaryotic DNA transposons excise themselves from one place in the genome, leaving that site and moving to another. Mobile elements that transpose to new sites in the genome via an RNA intermediate are called retrotransposons. Retrotransposons make an RNA copy of themselves and introduce this new copy into another site in the genome, while also remaining
![li1692239539b](li1692239539b.jpg)

FIGURE 8-8 Two major classes of mobile elements. (a) Eukaryotic DNA transposons (orange) move via a DNA intermediate, which is excised from the donor site. (b) Retrotransposons (green) are first transcribed into an RNA molecule, which is then reverse-transcribed into double-stranded DNA. In both cases, the double-stranded DNA intermediate is integrated into the target-site DNA to complete movement. Thus DNA transposons move by a cut-and-paste mechanism, whereas retrotransposons move by a copy-and-paste mechanism.

at their original location. The movement of retrotransposons is analogous to the infectious process of retroviruses (see Figure 5-49). Indeed, retroviruses can be thought of as retrotransposons that evolved genes encoding viral coats that allowed them to transpose between cells. Retrotransposons can be further classified on the basis of their specific mechanism of transposition. To summarize, DNA transposons can be thought of as transposing by a "cut-and-paste" mechanism, while retrotransposons move by a "copy-and-paste" mechanism in which the copy is an RNA intermediate.

## DNA Transposons Are Present in Prokaryotes and Eukaryotes

Most mobile elements in bacteria transpose directly as DNA. In contrast, most mobile elements in eukaryotes are retrotransposons, but eukaryotic DNA transposons also occur. Indeed, the original mobile elements discovered by Barbara McClintock are DNA transposons.

Bacterial Insertion Sequences The first molecular understanding of mobile elements came from the study of certain E. coli mutations caused by the spontaneous insertion of a DNA sequence, about $1-2 \mathrm{~kb}$ long, into the middle of a gene. These inserted stretches of DNA are called insertion sequences, or IS elements. So far, more than 1000 different IS elements have been found in E. coli and other bacteria.

Transposition of an IS element is a very rare event, occurring in only one in $10^{5}-10^{7}$ cells per generation, depending on the particular IS element. Often transpositions inactivate essential genes, killing the host cell and the IS elements it carries. Therefore, higher rates of transposition would probably result in too great a mutation rate for the host organism to survive. However, since IS elements transpose more or less randomly, some transposed sequences enter nonessential regions of the host genome (e.g., regions between genes), allowing the host cell to survive. At a very low rate of transposition, most host cells survive and therefore propagate the symbiotic IS element. IS elements can also insert themselves into plasmids or lysogenic viruses and can thus be transferred to other cells. In this way, IS elements can transpose into the chromosomes of new host cells.

The general structure of IS elements is diagrammed in Figure 8-9. An inverted repeat of $10-40 \mathrm{bp}$ is invariably present at each end of an IS element. In an inverted repeat, the $5^{\prime} \rightarrow 3^{\prime}$ sequence on one strand is repeated on the other strand, such as
![li1702239539b](li1702239539b.jpg)

Between the inverted repeats is a region that encodes a transposase, an enzyme required for transposition of the IS element to a new site. The transposase is expressed very rarely, accounting for the very low frequency of transposition. An important hallmark of IS elements is the presence of a short direct repeat sequence, containing $5-11 \mathrm{bp}$, depending on the particular IS element, immediately adjacent to both ends
![li1712239539b](li1712239539b.jpg)

FIGURE 8-9 General structure of bacterial IS elements. The relatively large central region of an IS element, which encodes one or two enzymes required for transposition, is flanked by an inverted repeat at each end. The sequences of the inverted repeats are nearly identical, but they are oriented in opposite directions. The sequence is characteristic of a particular IS element. The $5^{\prime}$ and $3^{\prime}$ short direct (as opposed to inverted) repeats are not transposed with the insertion element; rather, they are insertion-site sequences that become duplicated, with one copy at each end, during insertion of a mobile element. The length of the direct repeats is constant for a given IS element, but their sequence depends on the site of insertion and therefore varies with each transposition of the IS element. Arrows indicate sequence orientation. The regions in this diagram are not to scale; the coding region makes up most of the length of an IS element.
of the inserted element. The length of the direct repeat is characteristic of each type of IS element, but its sequence depends on the target site where a particular copy of the IS element inserted. When the sequence of a mutated gene containing an IS element is compared with the wild-type gene sequence, only one copy of the direct repeat is found in the wild-type gene. Duplication of this target-site sequence to create the second direct repeat adjacent to an IS element occurs during the insertion process.

As depicted in Figure 8-10, transposition of an IS element occurs by a "cut-and-paste" mechanism. Transposase performs three functions in this process: it (1) precisely excises the IS element from the donor DNA, (2) makes staggered cuts in a short sequence in the target DNA, and (3) ligates the $3^{\prime}$ termini of the IS element to the $5^{\prime}$ ends of the cut donor DNA. Finally, a host-cell DNA polymerase fills in the single-stranded gaps, generating the short direct repeats that flank IS elements, and DNA ligase joins the free ends.

Eukaryotic DNA Transposons McClintock's original discovery of mobile elements came from her observation of spontaneous mutations in maize that affect the production of enzymes required to make anthocyanin, a purple pigment in maize kernels. Mutant kernels are white and wild-type kernels are purple. One class of these mutations is revertible at high frequency, whereas a second class does not revert unless they occur in the presence of the first class of mutations. McClintock called the agents responsible for the first class of mutations activator (Ac) elements and those responsible for the second class dissociation ( $D s$ ) elements because they also tended to be associated with chromosome breaks.

Many years after McClintock's pioneering discoveries, cloning and sequencing revealed that Ac elements are equivalent to bacterial IS elements. Like IS elements, they contain inverted terminal repeat sequences that flank a region encoding a transposase, which recognizes the terminal repeats and catalyzes transposition to a new site in the host DNA.

![li1722239539b](li1722239539b.jpg)

FIGURE 8-10 Model for transposition of bacterial insertion sequences. Step $\mathbf{\square}$ Transposase, which is encoded by the IS element (IS10 in this example), cleaves both strands of the donor DNA next to the inverted repeats (dark red), excising the IS10 element. At a largely random target site, transposase makes staggered cuts in the target DNA. In the case of IS10, the two cuts are 9 bp apart. Step $\mathbf{\square}$ Ligation of the $3^{\prime}$ ends of the excised IS element to the staggered sites in the target DNA is also catalyzed by transposase. Step $\mathbf{\square}$ The 9-bp gaps of single-stranded DNA left in the resulting intermediate are filled in by a cellular DNA polymerase; finally, cellular DNA ligase forms the $3^{\prime} \rightarrow 5^{\prime}$ phosphodiester bonds between the $3^{\prime}$ ends of the extended target DNA strands and the $5^{\prime}$ ends of the IS10 strands. This process results in duplication of the target-site sequence on each side of the inserted IS element. Note that the lengths of the target site and IS10 are not to scale. See H. W. Benjamin and N. Kleckner, 1989, Cell 59:373; and 1992, Proc. Natl. Acad. Sci. USA 89:4648.

Ds elements are deleted forms of Ac elements in which a portion of the sequence encoding transposase is missing. Because it does not encode a functional transposase, a Ds element cannot move by itself. However, in plants that carry Ac elements and thus express a functional transposase, Ds elements can be transposed because they retain the inverted terminal repeats recognized by the transposase.

Since McClintock's early work on mobile elements in corn, DNA transposons have been identified in other
eukaryotes. For instance, approximately half of all the spontaneous mutations observed in Drosophila are due to the insertion of mobile elements. Although most of the mobile elements in Drosophila function as retrotransposons, at least one-the $P$ element-functions as a DNA transposon, moving by a mechanism similar to that used by bacterial insertion sequences. Current methods for constructing transgenic Drosophila depend on engineered, high-level expression of the P-element transposase and use of the P-element inverted terminal repeats as targets for transposition, as discussed in Chapter 6 (see Figure 6-20).

DNA transposition by the cut-and-paste mechanism can result in an increase in the copy number of a transposon if it occurs during the S phase of the cell cycle (see Figure 1-21), when DNA synthesis occurs. Such an increase happens when the donor DNA is in one of the two daughter DNA molecules in a region of a chromosome that has replicated but the target DNA is in a region that has not yet replicated. When DNA replication is complete at the end of the $S$ phase, the target DNA in its new location has also been replicated, resulting in a net increase in the total number of transposon copies in the cell (Figure 8-11). When such a transposition occurs during the S phase preceding meiosis, one of the four germ cells produced contains the extra copy of the transposon. Repetition of this process over evolutionary time has resulted in the accumulation of large numbers of DNA transposons in the genomes of some organisms. Human DNA contains about 300,000 copies of full-length and deleted DNA transposons, amounting to about 3 percent of human DNA. As we will see shortly, this mechanism can lead to the transposition of genomic DNA as well as the transposon itself.
![li1732239539b](li1732239539b.jpg)

FIGURE 8-11 Mechanism for increasing DNA-transposon copy number. If a DNA transposon, which transposes by a cut-and-paste mechanism (see Figure 8-10), transposes during S phase from a region of the chromosome that has replicated to a region that has not yet replicated, then when chromosomal replication is completed, one of the two daughter chromosomes will have a net increase of one copy of the transposon.

## LTR Retrotransposons Behave Like Intracellular Retroviruses

The genomes of all eukaryotes studied, from yeast to humans, contain retrotransposons, mobile DNA elements that transpose through an RNA intermediate using a reverse transcriptase (see Figure 8-8b). These mobile elements are divided into two major categories: those containing and those lacking long terminal repeats (LTRs). LTR retrotransposons, which we discuss here, are common in yeast (e.g., Ty elements) and in Drosophila (e.g., copia elements). Although less abundant in mammals than non-LTR retrotransposons, LTR retrotransposons nonetheless constitute about 8 percent of human genomic DNA. Non-LTR retrotransposons are the most common type of mobile element in mammals; these retrotransposons are described in the next section.

The general structure of LTR retrotransposons found in eukaryotes is depicted in Figure 8-12. In addition to the short $5^{\prime}$ and $3^{\prime}$ direct repeats that are typical of all transposons, these retrotransposons are marked by the presence of LTRs flanking the central protein-coding region. These long direct terminal repeats, containing $250-600 \mathrm{bp}$ depending on the particular LTR retrotransposon, are characteristic of integrated retroviral DNA and are critical to the life cycle of retroviruses. In addition to sharing LTRs with retroviruses, LTR retrotransposons encode all the proteins of the most common type of retroviruses, except for the envelope proteins. Lacking these envelope proteins, LTR retrotransposons cannot bud from their host cell and infect other cells; however, they can transpose to new sites in the DNA of their host cell. Because of their clear relationship with retroviruses, LTR retrotransposons are often called retrovirus-like elements.

A key step in the retroviral life cycle is the formation of retroviral genomic RNA from integrated retroviral DNA (see Figure 5-48). We describe this process in some detail here because it serves as a model for the generation of the RNA intermediate during the transposition of LTR retrotransposons. As depicted in Figure 8-13, the leftward retroviral LTR functions as a promoter that directs host-cell RNA polymerase to initiate transcription at the $5^{\prime}$ nucleotide of the roughly 20 -base $R$ sequence that is repeated at each end of the retroviral RNA. After the entire downstream retroviral

LTR retrotransposon ( $\mathbf{- 6 - 1 1} \mathbf{~ k b}$ )
![li1742239539b](li1742239539b.jpg)

FIGURE 8-12 General structure of eukaryotic LTR retrotransposons. The central protein-coding region is flanked by two long terminal repeats (LTRs), which are element-specific direct repeats. Like other mobile elements, integrated retrotransposons have short target-site direct repeats at each end. Note that the different regions are not drawn to scale. The protein-coding region constitutes 80 percent or more of a retrotransposon and encodes reverse transcriptase, integrase, and other retroviral proteins.
![li1752239539b](li1752239539b.jpg)

FIGURE 8-13 Generation of retroviral genomic RNA from integrated retroviral DNA. The left LTR directs cellular RNA polymerase to initiate transcription at the first nucleotide of the left $R$ region. The resulting primary transcript extends beyond the right LTR. The right LTR, now present in the RNA primary transcript, directs cellular enzymes to cleave the primary transcript at the last nucleotide of the right $R$ region and to add a poly(A) tail, yielding a retroviral RNA genome with the structure shown at the top of Figure 8-14. The $R$ sequence is repeated precisely at the $5^{\prime}$ and $3^{\prime}$ end (before the poly(A) tail) of the viral genomic RNA. U5 and U3 refer to sequences at the $5^{\prime}$ and $3^{\prime}$ ends of the viral RNA that are not repeated in the genomic retroviral RNA and hence are unique (see Figure 8-14). A similar mechanism is thought to generate the RNA intermediate during transposition of retrotransposons. The short direct repeat sequences (black) of target-site DNA are generated during integration of the retroviral DNA into the host-cell genome.

DNA has been transcribed, the RNA sequence corresponding to the rightward LTR directs host-cell RNA-processing enzymes to cleave the primary transcript and add a poly(A) tail at the $3^{\prime}$ end of the R sequence. The resulting retroviral RNA genome, which lacks a complete LTR, exits the nucleus and is packaged into a virion that buds from the host cell.

After a retrovirus infects a cell, reverse transcription of its RNA genome by the retrovirus-encoded reverse transcriptase yields a double-stranded DNA containing complete LTRs (Figure 8-14). This DNA synthesis takes place

FIGURE 8-14 Model for reverse transcription of retroviral genomic RNA into DNA. In this model, a complicated series of nine events generates a double-stranded DNA copy of the single-stranded RNA genome of a retrovirus. The genomic RNA is packaged in the virion with a retrovirus-specific cellular tRNA hybridized to a complementary sequence near its $5^{\prime}$ end, called the primer-binding site (PBS). The retroviral RNA has a short direct repeat terminal sequence ( $R$ ) at each end. The overall reaction is carried out by reverse transcriptase, which catalyzes polymerization of deoxyribonucleotides. RNaseH, also encoded in the viral RNA and packaged into the virion particle, digests the RNA strand in a DNA-RNA hybrid. The entire process yields a double-stranded DNA molecule that is longer than the template RNA and has a long terminal repeat (LTR) at each end. The different regions are not shown to scale. The PBS and $R$ regions are actually much shorter than the U5 and U3 regions, and the central coding region is very much longer than the other regions. See E. Gilboa et al., 1979, Cell 18:93.

![li1762239539b](li1762239539b.jpg)

in the cytosol. The double-stranded DNA, with an LTR at each end, is then transported into the nucleus in a complex with integrase, another enzyme encoded by retroviruses. Retroviral integrases are closely related to the transposases encoded by DNA transposons and use a similar mechanism to insert the double-stranded retroviral DNA into the host-cell genome. In this process, short direct repeats of the target-site sequence are generated at either end of the inserted viral DNA sequence. Although the mechanism of reverse transcription is complex, it is a critical aspect of the retrovirus life cycle. The process generates the complete $5^{\prime}$ LTR that functions as a promoter for initiation of transcription precisely at the $5^{\prime}$ nucleotide of the R sequence, while the complete $3^{\prime}$ LTR functions as a poly(A) site leading to polyadenylation precisely at the $3^{\prime}$ nucleotide of the R sequence. Consequently, no nucleotides are lost from an LTR retrotransposon as it undergoes successive rounds of insertion, transcription, reverse transcription, and reinsertion at a new site.

As noted above, LTR retrotransposons encode reverse transcriptase and integrase. By analogy with retroviruses, these mobile elements move by a "copy-and-paste" mechanism whereby reverse transcriptase converts an RNA copy of a donor element into DNA, which is inserted into a target site by integrase. The experiments depicted in Figure 8-15 provided strong evidence for the role of an RNA intermediate in the transposition of Ty elements in yeast.

The most common LTR retrotransposons in humans are called $E R V s$, for endogenous retroviruses. Most of the 443,000 ERV-related DNA sequences in the human genome consist only of isolated LTRs. These sequences are derived from full-length proviral DNA by homologous recombination between two LTRs, resulting in deletion of the internal retroviral sequences. Isolated LTRs such as these cannot be transposed to a new position in the genome, but recombination between homologous LTRs at different positions in the genome has probably contributed to the chromosomal DNA rearrangements leading to gene and exon duplications, the evolution of proteins with new combinations of exons, and, as we will see in Chapter 9, the evolution of complex control of gene expression.

## Non-LTR Retrotransposons Transpose by a Distinct Mechanism

The most abundant mobile elements in mammals are retrotransposons that lack LTRs, sometimes called nonviral retrotransposons. These moderately repeated DNA sequences form two classes in mammalian genomes: long interspersed elements (LINEs) and short interspersed elements (SINEs). In humans, full-length LINEs are about 6 kb long, and SINEs are about 300 bp long (see Table 8-1). Repeated sequences with the characteristics of LINEs have been observed in protozoans, insects, and plants, but for unknown reasons, they are particularly abundant in the genomes of mammals. SINEs too are found primarily in
![li1772239539b](li1772239539b.jpg)

Results in galactosecontaining medium

1. Ty mRNA synthesis increased
2. Transposition of Ty elements increased
![li1782239539b](li1782239539b.jpg)

Results in galactosecontaining medium

1. Ty mRNAs lack intron
2. Transposed Ty elements lack intron
![li1792239539b](li1792239539b.jpg)

EXPERIMENTAL FIGURE 8-15 The yeast Ty element transposes through an RNA intermediate. When yeast cells are transformed with a Ty-containing plasmid, the Ty element can transpose to new sites, although normally this occurs at a low rate. Using the elements diagrammed at the top, researchers engineered two different recombinant plasmid vectors containing recombinant Ty elements adjacent to a galactose-sensitive promoter. Yeast cells transformed with these plasmids were grown in a galactose-containing and a galactose-free medium. In experiment 1, growth of cells in galactose-containing medium resulted in many more transpositions than in galactose-free medium, indicating that transcription into an mRNA intermediate is required for Ty transposition. In experiment 2, an intron from an unrelated yeast gene was inserted into the putative protein-coding region of the recombinant galactose-responsive Ty element. The observed absence of the intron in transposed Ty elements is strong evidence that transposition involves an mRNA intermediate from which the intron was removed by RNA splicing, as depicted in the box on the right. In contrast, eukaryotic DNA transposons, such as the Ac element of maize, contain introns within the transposase gene, indicating that they do not transpose via an RNA intermediate. See J. Boeke et al., 1985, Cell 40:491.

![li1802239539b](li1802239539b.jpg)

FIGURE 8-16 General structure of a LINE. The length of the targetsite direct repeats varies among the copies of a LINE at different sites in the genome. Although the full-length L1 element is about 6 kb long, variable amounts of the left end are absent at over 90 percent of the sites where this mobile element is found. The shorter open reading frame (ORF1), about 1 kb in length, encodes an RNA-binding protein. The longer ORF2, about 4 kb in length, encodes a bifunctional protein with reverse transcriptase and DNA endonuclease activity. Note that LINEs lack the long terminal repeats found in LTR retrotransposons.
mammalian DNA. Large numbers of LINEs and SINEs in higher eukaryotes have accumulated over evolutionary time by repeated copying of sequences at a few positions in the genome and insertion of the copies into new positions.

LINEs Human DNA contains three major families of LINEs that are similar in their mechanism of transposition but differ in their sequences: L1, L2, and L3. Only members of the L1 family transpose in the contemporary human genome; apparently there are no remaining functional copies of L2 or L3. LINE sequences are present at roughly 900,000 sites in the human genome, accounting for a staggering 21 percent of total human DNA. The general structure of a complete LINE is diagrammed in Figure 8-16. LINEs are usually flanked by short direct repeats, the hallmark of mobile elements, and contain two long open reading frames (ORFs, which are protein-coding regions; see Section 8.4). ORF1, about 1 kb long, encodes an RNA-binding protein. ORF2, about 4 kb long, encodes a protein that has a long region of homology with the reverse transcriptases of retroviruses and LTR retrotransposons, but also exhibits DNA endonuclease activity.

Evidence for the mobility of L1 elements first came from analysis of DNA cloned from patients with certain genetic diseases such as hemophilia and myotonic dystrophy. DNA from these patients was found to carry mutations resulting from insertion of an L1 element into a gene, whereas no such element occurred within that gene in either parent. About 1 in 600 mutations that cause significant disease in humans are due to L1 transpositions or SINE transpositions that are catalyzed by L1-encoded proteins. Later experiments similar to those just described with yeast Ty elements (see Figure 8-15) confirmed that L1 elements transpose through an RNA intermediate. In these experiments, an intron was introduced into a cloned mouse L1 element, and the recombinant L1 element was stably transfected into cultured hamster cells. After several cell doublings, a DNA fragment corresponding to the L1 element but lacking the inserted intron was detected in the cells. This finding strongly
suggests that, over time, the recombinant L1 element containing the inserted intron had transposed to new sites in the hamster genome through an RNA intermediate that underwent RNA splicing to remove the intron.

Since LINEs do not contain LTRs, their mechanism of transposition through an RNA intermediate differs from that of LTR retrotransposons. The proteins encoded by ORF1 and ORF2 are translated from a LINE RNA. In vitro studies indicate that transcription by RNA polymerase is directed by promoter sequences at the left end of integrated LINE DNA. LINE RNA is polyadenylated by the same post-transcriptional mechanism that polyadenylates other mRNAs. The LINE RNA is then exported into the cytosol, where it is translated into ORF1 and ORF2 proteins. Multiple copies of ORF1 protein then bind to the LINE RNA, and ORF2 protein binds to the poly(A) tail. The LINE RNA is then transported back into the nucleus as a complex with ORF1and ORF2 proteins, where it is reverse-transcribed into LINE DNA by ORF2. The mechanism involves staggered cleavage of cellular DNA at the insertion site, followed by priming of reverse transcription by the resulting cleaved cellular DNA, as detailed in Figure 8-17. The complete process results in insertion of a copy of the original LINE retrotransposon into a new site in chromosomal DNA. A short direct repeat is generated at the insertion site because of the initial staggered cleavage of the two chromosomal DNA strands.

As noted already, the DNA form of an LTR retrotransposon is synthesized from its RNA form in the cytosol using a cellular tRNA as a primer for reverse transcription of the first strand of DNA (see Figure 8-14). The resulting doublestranded DNA with long terminal repeats is then transported into the nucleus, where it is integrated into chromosomal DNA by a retrotransposon-encoded integrase. In contrast, the DNA form of a non-LTR retrotransposon is synthesized in the nucleus. The synthesis of the first strand of the nonLTR retroviral DNA by ORF2, a reverse transcriptase, is primed by the $3^{\prime}$ end of cleaved chromosomal DNA, which base-pairs with the poly(A) tail of the non-LTR RNA (see Figure 8-17, step 1). Since its synthesis is primed by the cut end of a cleaved chromosome, and since synthesis of the other strand of the non-LTR retrotransposon DNA is primed by the $3^{\prime}$ end of chromosomal DNA on the other side of the initial cut (step 1), the mechanism of synthesis results in integration of the non-LTR retrotransposon DNA. There is no need for an integrase to insert the non-LTR retrotransposon DNA. Because its synthesis begins with reverse transcription of a poly(A) tail on the LINE RNA, one end of a non-LTR retrotransposon is AT-rich.

The vast majority of LINEs in the human genome are truncated at their $5^{\prime}$ end, suggesting that reverse transcription was terminated before completion and that the resulting fragments, extending variable distances from the poly(A) tail, were inserted. Because of this shortening, the average size of LINE elements is only about 900 bp , even though the full-length sequence is about 6 kb long. Truncated LINE

FIGURE 8-17 Proposed mechanism of LINE reverse transcription and integration. Only ORF2 protein is represented here. Newly synthesized LINE DNA is shown in black. ORF1 and ORF2 proteins, produced by translation of LINE RNA in the cytoplasm, bind to LINE RNA and transport it into the nucleus. Step $\mathbf{\square}$ In the nucleus, ORF2 makes staggered cuts in AT-rich target-site DNA, generating the DNA 3'-OH ends indicated by blue arrowheads. Step $\mathbf{\square}$ The $3^{\prime}$ end of the T-rich DNA strand hybridizes to the poly(A) tail of the LINE RNA and primes DNA synthesis by ORF2. Step $\mathbf{\square}$ ORF2 extends the DNA strand using the LINE RNA as a template. Steps $\mathbf{\square}$ and $\mathbf{\square}$ When synthesis of the LINE DNA bottom strand reaches the $5^{\prime}$ end of the LINE RNA template, ORF2 extends the newly synthesized LINE DNA using as a template the top-strand cellular DNA generated by the initial ORF2 staggered cleavage. Step $\mathbf{\square}$ A cellular DNA polymerase extends the $3^{\prime}$ end of the top strand generated by the initial ORF2 staggered cut, using the newly synthesized bottom-strand LINE DNA as a template. The LINE RNA is digested as the DNA polymerase extends the upperstrand DNA, just as occurs during removal of lagging-strand primer RNA during cellular DNA synthesis (see Figure 5-29). The $3^{\prime}$ ends of the newly synthesized DNA strands are ligated to the $5^{\prime}$ ends of the cellular DNA strands as in lagging-strand cellular DNA synthesis. See D. D. Luan et al., 1993, Cell 72:595.
elements, once formed, probably are not further transposed because they lack a promoter for formation of the RNA intermediate. In addition to the fact that most L1 insertions are truncated, nearly all the full-length elements contain stop codons and frameshift mutations in ORF1 and ORF2; these mutations have probably accumulated in most LINE sequences over evolutionary time. As a result, only about 0.01 percent of the LINE sequences in the human genome, or about 60 in total number, are full-length, with intact open reading frames for ORF1 and ORF2.

SINEs The most abundant class of mobile elements in the human genome, SINEs constitute about 13 percent of total human DNA. Varying in length from about 100 to 400 base pairs, these retrotransposons do not encode protein, but most contain a $3^{\prime}$ AT-rich sequence similar to that in LINEs. SINEs are transcribed by the same nuclear RNA polymerase that transcribes genes encoding tRNAs, 5S rRNAs, and other small stable RNAs. Most likely, the ORF1 and ORF2 proteins expressed from full-length LINEs mediate reverse transcription and integration of SINEs by the mechanism depicted in Figure 8-17. Consequently, SINEs can be viewed as parasites of the LINE symbionts, competing with LINE RNAs for binding, reverse transcription, and integration by LINE-encoded ORF1 and ORF2.

SINEs occur at about 1.6 million sites in the human genome. Of these, about 1.1 million are Alu elements, so named because most of them contain a single recognition site for the restriction enzyme AluL. Alu elements exhibit considerable sequence homology with, and probably evolved from, 7SL RNA, a cytosolic RNA in a ribonucleoprotein complex called the signal recognition particle. This abundant cytosolic ribonucleoprotein particle aids in targeting certain polypeptides to the membranes of the endoplasmic reticulum (see Chapter 13). Alu elements are scattered throughout
![li1812239539b](li1812239539b.jpg)

the human genome at sites where their insertion has not disrupted gene expression: between genes, within introns, and in the $3^{\prime}$ untranslated regions of some mRNAs. For instance, nine Alu elements are located within the human $\beta$-globin gene cluster (see Figure 8-4a). Of the new germ-line nonLTR retrotranspositions that are estimated to occur about once in every eight individuals, about 40 percent involve L1 elements and 60 percent involve SINEs, of which about 90 percent are Alu elements.

Like other mobile elements, most SINEs have accumulated mutations from the time of their insertion in the germ line of an ancient ancestor of modern humans. Like LINEs, many SINEs are truncated at their $5^{\prime}$ ends.

## Other Retroposed RNAs Are Found in Genomic DNA

In addition to the mobile elements listed in Table 8-1, DNA copies of a wide variety of mRNAs appear to have become integrated into chromosomal DNA. Since these sequences lack introns and do not have flanking sequences similar to those of functional gene copies, they clearly are not simply duplicated genes that have drifted into nonfunctionality and become pseudogenes, as discussed earlier (see Figure 8-4a). Instead, these DNA segments appear to be retrotransposed copies of spliced and polyadenylated mRNA. Compared with normal genes encoding mRNAs, these inserted segments generally contain multiple mutations, which are thought to have accumulated since they were first reverse-transcribed and randomly integrated into the genome of a germ cell in an ancient ancestor. These nonfunctional genomic copies of mRNAs are referred to as processed pseudogenes. Most processed pseudogenes are flanked by short direct repeats, supporting the hypothesis that they were generated by rare retrotransposition events involving cellular mRNAs.

Other interspersed repeats representing partial or mutant copies of genes encoding small nuclear RNAs (snRNAs) and tRNAs are found in mammalian genomes. Like processed pseudogenes derived from mRNAs, these nonfunctional copies of small RNA genes are flanked by short direct repeats and most likely result from rare retrotransposition events that have accumulated through the course of evolution. Enzymes expressed from a LINE are thought to have carried out all these retrotransposition events involving mRNAs, snRNAs, and tRNAs.

## Mobile DNA Elements Have Significantly Influenced Evolution

Although mobile DNA elements appear to have no direct function other than to maintain their own existence, their presence has had a profound effect on the evolution of modern-day organisms. As mentioned earlier, about half the spontaneous mutations in Drosophila result from insertion of a mobile DNA element into or near a transcription unit. In mammals, mobile elements cause a much smaller proportion of spontaneous mutations: about 10 percent in mice, and only $0.1-0.2$ percent in humans. Still, mobile elements have been found in mutant alleles associated with several human genetic diseases. For example, insertions into the clotting factor IX gene cause hemophilia, and insertions into the gene encoding the muscle protein dystrophin lead to Duchenne muscular dystrophy. The genes encoding clotting factor IX and dystrophin are both on the X chromosome. Consequently, disease resulting from a transposition into these genes occurs primarily in males, in which there is no second copy of the normal gene to complement the resulting mutation.

In lineages leading to higher eukaryotes, homologous recombination between mobile DNA elements dispersed throughout ancestral genomes may have generated gene duplications and other DNA rearrangements during evolution (see Figure 8-2b). For instance, cloning and sequencing of the $\beta$-globin gene clusters from various primate species has provided strong evidence that the human HGB1 and HGB2 genes encoding fetal $\beta$-globins arose from an unequal homologous crossover between two L1 elements flanking an ancestral globin gene. Subsequent divergence of such duplicated genes could lead to the acquisition of distinct, beneficial functions by each member of a gene family. Unequal crossing over between mobile elements located within introns of a particular gene could lead to the duplication of exons within that gene (see Figure 8-2a). This process most likely influenced the evolution of genes that contain multiple copies of similar exons encoding similar protein domains, such as the fibronectin gene (see Figure 5-16).

Some evidence suggests that during the evolution of higher eukaryotes, recombination between mobile DNA elements (e.g., Alu elements) in introns of two separate genes also occurred, generating new genes made from novel combinations of preexisting exons (Figure 8-18). This evolutionary
![li1822239539b](li1822239539b.jpg)

FIGURE 8-18 Exon shuffling via recombination between homologous interspersed repeats. Recombination between interspersed repeats in the introns of separate genes produces transcription units with new combinations of exons. In the example shown here, a double crossover between two sets of Alu repeats results in an exchange of exons between two genes.

FIGURE 8-19 Exon shuffling by transposition. (a) Transposition of an exon flanked by homologous DNA transposons into an intron on a second gene. As we saw in Figure 8-10, step $\mathbf{1}$, transposase can recognize and cleave the DNA at the ends of the transposon inverted repeats. In gene 1, if the transposase cleaves at the left end of the transposon on the left and at the right end of the transposon on the right, it can transpose all the intervening DNA, including the exon from gene 1 , to a new site in an intron of gene 2. The net result is an insertion of the exon from gene 1 into gene 2. (b) Integration of an exon into another gene via LINE transposition. Some LINEs have weak poly(A) signals. If such a LINE is in the $3^{\prime}$-most intron of gene 1, during transposition its transcription may continue beyond its own poly(A) signals and extend into the $3^{\prime}$ exon, transcribing the cleavage and polyadenylation signals of gene 1 itself. This RNA can then be reverse-transcribed and integrated by the LINE ORF2 protein (see Figure 8-17) into an intron on gene 2, introducing a new $3^{\prime}$ exon (from gene 1) into gene 2.
![li1832239539b](li1832239539b.jpg)
process, termed exon shuffling, may have occurred during the evolution of the genes encoding tissue plasminogen activator, the Neu receptor, and epidermal growth factor, all of which contain an EGF domain (see Figure 3-12). In this case, exon shuffling presumably resulted in the insertion of an EGF domain-encoding exon into an intron of the ancestral form of each of these genes.

Both DNA transposons and LINE retrotransposons have been shown to occasionally carry unrelated flanking sequences-including exons-when they transpose to new sites by the mechanisms diagrammed in Figure 8-19. These mechanisms probably also contributed to exon shuffling during the evolution of contemporary genes.

In addition to causing changes in coding sequences, recombination between mobile elements and transposition of DNA adjacent to mobile elements probably played a significant role in the evolution of regulatory sequences that control gene expression. As noted earlier, eukaryotic genes have transcription-control regions called enhancers that can operate over distances of tens of thousands of base pairs. The transcription of many genes is controlled through the combined effects of several enhancer elements. Insertion of mobile elements near such transcription-control regions probably contributed to the evolution of new combinations of enhancer sequences. These combinations, in turn, control which specific genes are expressed in particular cell types
and the amount of the encoded protein produced in modern organisms, as we discuss in the next chapter.

These considerations suggest that the early view of mobile DNA elements as completely selfish molecular parasites misses the mark. Rather, these elements have contributed profoundly to the evolution of higher organisms by promoting (1) the generation of gene families via gene duplication, (2) the creation of new genes via shuffling of preexisting exons, and (3) the formation of more complex regulatory regions that provide multifaceted control of gene expression. Today researchers are attempting to harness transposition mechanisms to insert therapeutic genes into patients as a form of gene therapy.

$\mathbf{E}$A process analogous to that shown in Figure 8-19a is largely responsible for the rapid spread of antibiotic resistance among pathogenic bacteria, a major problem in modern medicine. Bacterial genes encoding enzymes that inactivate antibiotics (drug resistance genes) have become flanked by insertion sequences, generating drug resistance transposons. The widespread use of antibiotics in medicine, often unnecessarily in the treatment of viral infections, on which they have no effect, and to prevent infections in healthy agricultural animals has led to positive selection on such drug resistance transposons, which have inserted into conjugating plasmids. Conjugating plasmids encode proteins

that result in their replication and transfer to other bacterial cells-even cells of other related bacterial species-through a complex macromolecular tube called a pilus. These plasmids, called R factors (for drug resistance), can contain multiple drug resistance genes introduced by transposition and selected in environments, such as hospitals, where antibiotics are used to sterilize surfaces. These R factors have led to the rapid spread of resistance to multiple antibiotics between pathogenic bacteria. Coping with the spread of R factors is a major challenge for modern medicine.

## KEY CONCEPTS OF SECTION 8.3

## Transposable (Mobile) DNA Elements

- Transposable DNA elements are moderately repeated sequences interspersed at multiple sites throughout the genomes of higher eukaryotes. They are present less frequently in prokaryotic genomes.
- DNA transposons move to new sites directly as DNA; retrotransposons are first transcribed into an RNA copy of the element, which is then reverse-transcribed into DNA (see Figure 8-8).
- A common feature of all mobile elements is the presence of short direct repeats flanking the element, generated as the result of staggered cuts in the target-site DNA that are filled in by a DNA polymerase during transposition (see Figure 8-10).
- Enzymes encoded by transposons themselves catalyze insertion of these sequences at new sites in genomic DNA.
- Although DNA transposons, similar in structure to bacterial IS elements, occur in eukaryotes (e.g., the Drosophila P element), retrotransposons are generally much more abundant, especially in vertebrates.
- LTR retrotransposons are flanked by long terminal repeats (LTRs) similar to those in retroviral DNA; like retroviruses, they encode reverse transcriptase and integrase. They move in the genome by being transcribed into RNA, which then undergoes reverse transcription in the cytosol, nuclear import of the resulting DNA with LTRs, and integration into a host-cell chromosome (see Figure 8-14).
- Non-LTR retrotransposons, including long interspersed elements (LINEs) and short interspersed elements (SINEs), lack LTRs and have an AT-rich stretch at one end. They are thought to move by a nonviral retrotransposition mechanism mediated by LINE-encoded proteins involving priming of reverse transcription by chromosomal DNA (see Figure 8-17).
- SINE sequences exhibit extensive homology with small cellular RNAs and are transcribed by the same RNA polymerase. Alu elements, the most common SINEs in humans, are sequences of about 300 bp found scattered throughout the human genome at about 1.6 million sites (see Figure 8-4a).
- Some interspersed repeats are derived from cellular RNAs that were reverse-transcribed and inserted into genomic

DNA at some time in evolutionary history. Processed pseudogenes derived from mRNAs lack introns, a feature that distinguishes them from pseudogenes, which arose by sequence drift of duplicated genes.

- Mobile DNA elements most likely influenced evolution significantly by serving as sites for homologous recombination during unequal crossing over, leading to gene and exon duplication (see Figure 8-2) and exon shuffling (see Figure 8-18), and by mobilizing adjacent DNA sequences (see Figure 8-19).


### 8.4 Genomics: Genome-Wide Analysis of Gene Structure and Function

By using automated DNA sequencing techniques and computer algorithms to piece together sequence data, researchers have determined vast amounts of DNA sequence, including nearly the entire genomic sequence of humans and of many key experimental organisms. This enormous volume of data, which is growing at a rapid pace, has been stored and organized by the National Center for Biotechnology Information (NCBI), US National Institutes of Health, the European Bioinformatics Institute at the European Molecular Biology Laboratory in Heidelberg, Germany, and the DNA Data Bank of Japan. These databases continuously exchange newly reported sequences and make them available to scientists throughout the world on the Internet. By now, the genomic sequences have been completely, or nearly completely, determined for hundreds of viruses and bacteria; scores of archaea; yeasts (eukaryotes); plants, including rice and maize; important model multicellular eukaryotes such as the roundworm C. elegans, the fruit fly Drosophila melanogaster, and mice; humans; and representatives of all of the 35 or so metazoan phyla. The cost of sequencing a megabase of DNA has fallen so low that the entire genomes of cancer cells have been sequenced and compared with the genomes of normal cells from the patients from which they came in order to determine all the mutations that have accumulated in that patient's tumor cells. This approach is revealing genes that are commonly mutated in all cancers, as well as genes that are commonly mutated in tumors from different patients with the same type of cancer (e.g., breast or colon cancer). This approach may eventually lead to highly individualized cancer treatments tailored to the specific mutations in the tumor cells of a particular patient. The latest automated DNA sequencing techniques are so powerful that a project known as the "1000 Genomes Project" is currently under way, with the goal of sequencing most of the genomes of 2500 randomly chosen individuals from 25 populations around the world in order to determine the extent of human genetic variation as a basis for investigating the relationship between genotype and phenotype in humans. Moreover, privately owned companies have been founded that will sequence much of an individual's genome for about $\$ 100$ in

order to search for sequence variations that may influence that individual's probability of developing specific diseases.

In this section, we examine some of the ways in which researchers are mining this treasure trove of data to provide insights about gene function and evolutionary relationships, to identify new genes whose encoded proteins have never been isolated, and to determine when and where genes are expressed. This use of computers to analyze sequence data has led to the emergence of a new field of biology: bioinformatics.

## Stored Sequences Suggest Functions of Newly Identified Genes and Proteins

As discussed in Chapter 3, proteins with similar functions often contain similar amino acid sequences that correspond to important functional domains in the three-dimensional structure of the proteins. By comparing the amino acid sequence of the protein encoded by a newly cloned gene with the sequences of proteins of known function, an investigator can look for sequence similarities that provide clues to the function of the encoded protein. Because of the degeneracy in the genetic code, related proteins invariably exhibit more sequence similarity than the genes encoding them. For this reason, protein sequences, rather than the corresponding DNA sequences, are usually compared.

The most widely used computer program for this purpose is known as BLAST (basic local alignment search tool). The BLAST algorithm divides the "new" protein sequence (known as the query sequence) into shorter segments and then searches the database for significant matches to any of
the stored sequences. The matching program assigns a high score to identically matched amino acids and a lower score to matches between amino acids that are related (e.g., hydrophobic, polar, positively charged, negatively charged) but not identical. When a significant match is found for a segment, the BLAST algorithm searches locally to extend the region of similarity. After searching is completed, the program ranks the matches between the query protein and various known proteins according to their p-values. This parameter is a measure of the probability of finding such a degree of similarity between two protein sequences by chance. The lower the $p$-value, the greater the sequence similarity between two sequences. A $p$-value less than about $10^{-3}$ is usually considered significant evidence that two proteins share a common ancestor. Many alternative computer programs have been developed that can detect relationships between proteins that are more distantly related to each other than can be detected by BLAST. The development of such methods is currently an active area of bioinformatics research.

To illustrate the power of this sequence comparison approach, let's consider the human gene NF1. Mutations in NF1 are associated with the inherited disease neurofibromatosis 1 , in which multiple tumors develop in the peripheral nervous system, causing large protuberances in the skin. After a cDNA clone of NF1 was isolated and sequenced, the deduced sequence of the NF1 protein was checked against all other protein sequences in GenBank. A region of NF1 protein was discovered to have considerable homology to a portion of the yeast protein called Ira (Figure 8-20). Previous studies had shown that Ira is a GTPase-activating protein

NF1
Ira 1500 I 我 I A 我 L R 望 F I D 我 V . . . 我 N Y P V N P 我 K HEM 我 K M L A I D D F L K Y I I K N P I 我 A F F 1546

891 MA 我 我 N V V P C S QW 我 E L 我 R V L V T L 我 D S 我 H L L Y Q 我 L WN M F S K 我 V E L 我 D S M Q T L 940 1547 Q S 我 我 . . C S P A D V 我 L Y 我 G G F L N A 我 D T 我 N A S H I 我 V T E L L K Q 我 I K R 我 A R S D D I 1594

941 F 我 G N 我 L 我 S K I M T F C F K V Y 我 A T 我 Q 我 L 我 D 我 L 我 R I V I T S S D W Q H V 我 我 E V 我 P T 990 1595 L 我 R N 我 C 我 T R A L S L Y T R S R Q N K 我 我 I 我 T 我 R 我 V 我 Q G I V D N K E . . . 我 我 E I D . . 1638

991 R L E 我 S E S L E 我 N Q R N L 我 QM T 我 K F . . . . F H 我 我 I 我 S S S E 我 P Q 我 R S V Q H C L 兑 Q 1036 1639 K M K 我 G . . . S 我 N S E K M 我 D L F 我 K Y M T R L I D 我 我 T 我 S I D D 我 P I E 我 V D I 我 K T I 兑 N 1685

1037 V V 我 Q R 我 P Q N S I G 我 兑 這 我 A M 我 我 我 我 N 我 我 I 兑 這 我 Y E A Q 我 L D K K P P P R I E R G L K L 1086 1686 A A 我 V N 我 我 E Y A Y I 我 兑 這 我 F V 我 我 我 我 G 我 我 L 兑 這 我 D S E N 我 I . I V T H A H D R K P F I T 1734

1087 M S K I L Q 我 I 我 N . . . . . . . . . H V 我 F T K 我 E H M R P F N 我 . . . . 我 V K S N F D A A R R F 我 1124 1735 L A K V I Q 我 L 我 N Q R E N I F K K D I 我 V S K 我 我 F L K T C S Q K I F N 我 L S E L C K I P T N N 1784

1125 L D I A S D C 我 我 S D A V N H 我 L . . . . . . . . . . . . . . . . S F I S D G N V L A L H R L 我 W N N . 1159 1785 T V N V R E D 我 我 P I S F D Y 我 F L H K F F Y L N E F T I R K E I 我 N E S K L P G E F S F 我 K N T V 1834

1160 . . . Q E 我 我 G Q Y 我 S S N R D H K A V G R R 我 F . . . . D 我 M A T 我 L A Y L Q P P E H K P 兑 A 1200 1835 M L N D 我 我 L G V 我 G Q P S M E I K N E I P 我 E V V E N R E 兑 Y P S 我 Y E F M S R Y A F 兑 K 兑 D 1882

FIGURE 8-20 Comparison of the regions of human NF1 protein and S. cerevisiae Ira protein that show significant sequence similarity. The NF1 and the Ira sequences are shown on the top and bottom lines of each row, respectively, in the one-letter amino acid code (see Figure 2-14). Amino acids that are identical in the two proteins are highlighted in dark blue. Amino acids with chemically similar but
nonidentical side chains are highlighted in light blue. Black dots indicate "gaps" in the upper and lower protein sequences, inserted in order to maximize the alignment of homologous amino acids. The BLAST $p$-value for these two sequences is $10^{-28}$, indicating a high degree of similarity. [Data from G. Xu et al., 1990, Cell 62:599.]

(GAP) that modulates the GTPase activity of the monomeric G protein called Ras (see Figure 3-34). As we examine in detail in Chapter 16, GAP and Ras proteins normally function to control cell replication and differentiation in response to signals from neighboring cells. Functional studies on the normal NF1 protein, obtained by expression of the cloned wildtype gene, showed that it did, indeed, regulate Ras activity, as suggested by its homology with Ira. These findings suggest that patients with neurofibromatosis express a mutant NF1 protein in cells of the peripheral nervous system, leading to abnormally high signaling through RAS protein leading to excessive cell division and formation of the tumors characteristic of the disease.

Even when the BLAST algorithm finds no significant similarities, a query sequence may nevertheless share a short sequence with known proteins that is functionally important. Such short segments recurring in many different proteins, referred to as structural motifs, generally have similar functions. Several such motifs are described in Chapter 3 and illustrated in Figure 3-10. To search for these and other motifs in a new protein, researchers compare the query protein sequence with a database of known motif sequences.

## Comparison of Related Sequences from Different Species Can Give Clues to Evolutionary Relationships Among Proteins

BLAST searches for related protein sequences may reveal that proteins belong to a protein family. Earlier, we considered
![li1842239539b](li1842239539b.jpg)

FIGURE 8-21 Generation of diverse tubulin sequences during the evolution of eukaryotes. (a) Probable mechanism giving rise to the tubulin genes found in existing species. It is possible to deduce that a gene duplication event occurred before speciation because the $\alpha$-tubulin sequences from different species (e.g., humans and yeast) are more alike than are the $\alpha$-tubulin and $\beta$-tubulin sequences within a species. (b) A phylogenetic tree representing the relationship between the tubulin sequences. The branch points (nodes), indicated by small numbers, represent common ancestral genes at the time that two
gene families in a single organism, using the $\beta$-globin genes in humans as an example (see Figure 8-4a). But in a database that includes the genomic sequences of multiple organisms, protein families can also be recognized as being shared among related organisms. Consider, for example, the tubulin proteins, the basic subunits of microtubules, which are important components of the cytoskeleton (see Chapter 18). According to the simplified scheme in Figure 8-21a, the earliest eukaryotic cells are thought to have contained a single tubulin gene that was duplicated early in evolution; subsequent divergence of the different copies of the original tubulin gene formed the ancestral versions of the $\alpha$ - and $\beta$-tubulin genes. As different species diverged from these early eukaryotic cells, each of these gene sequences further diverged, giving rise to the slightly different forms of $\alpha$-tubulin and $\beta$-tubulin now found in each species.

All the different members of the tubulin family of genes (and proteins) are sufficiently similar in sequence to suggest a common ancestral sequence. Thus all these sequences are considered to be homologous. More specifically, sequences that presumably diverged as a result of gene duplication (e.g., the $\alpha$ - and $\beta$-tubulin sequences) are described as paralogous. Sequences that arose because of speciation (e.g., the $\alpha$-tubulin genes in different species) are described as orthologous. From the degree of sequence relatedness of the tubulins present in different organisms today, evolutionary relationships can be deduced, as illustrated in Figure 8-21b. Of the three types of sequence relationships, orthologous sequences are the most likely to share the same function.
![li1852239539b](li1852239539b.jpg)
sequences diverged. For example, node 1 represents the duplication event that gave rise to the $\alpha$-tubulin and $\beta$-tubulin families, and node 2 represents the divergence of yeast from multicellular species. Braces and arrows indicate, respectively, the orthologous tubulin genes, which differ as a result of speciation, and the paralogous genes, which differ as a result of gene duplication. This diagram is simplified somewhat because flies, worms, and humans actually contain multiple $\alpha$-tubulin and $\beta$-tubulin genes that arose from later gene duplication events.

## Genes Can Be Identified Within Genomic DNA Sequences

The complete genomic sequence of an organism contains within it the information needed to deduce the sequence of every protein made by the cells of that organism. For organisms such as bacteria and yeast, whose genomes have few introns and short intergenic regions, most protein-coding sequences can be found simply by scanning the genomic sequence for open reading frames (ORFs) of significant length. An ORF is usually defined as a stretch of DNA containing at least 100 codons that begins with a start codon and ends with a stop codon. Because the probability that a random DNA sequence will contain no stop codons for 100 codons in a row is very small, most ORFs encode proteins.

ORF analysis correctly identifies more than 90 percent of the genes in yeast and bacteria. Some of the very shortest genes, however, are missed by this method, and occasionally long open reading frames that are not actually genes arise by chance. Both types of mis-assignments can be corrected by more sophisticated analysis of the sequence and by genetic tests for gene function. Of the Saccharomyces genes identified in this manner, about half were already known by some functional criterion such as mutant phenotype. The functions of some of the proteins encoded by the remaining putative (suspected) genes identified by ORF analysis have been assigned based on their sequence similarity to known proteins in other organisms.

Identification of genes in organisms with a more complex genome structure requires more sophisticated algorithms than searching for open reading frames. Because most genes in higher eukaryotes are composed of multiple, relatively short exons separated by often quite long noncoding introns, scanning for ORFs is a poor method for finding genes in these organisms. The best gene-finding algorithms combine all the available data that might suggest the presence of a gene at a particular genomic site. Relevant data include alignment of the query sequence to
a full-length cDNA sequence; alignment to a partial cDNA sequence, generally 200-400 bp in length, known as an expressed sequence tag (EST); fitting to models for exon, intron, and splice-site sequences; and sequence similarity to genes from other organisms. Using these computer-based bioinformatic methods, computational biologists have identified approximately 21,000 protein-coding genes in the human genome.

A particularly powerful method for identifying human genes is to compare the human genomic sequence with that of the mouse. Humans and mice are sufficiently related to have most genes in common, although largely nonfunctional DNA sequences, such as intergenic regions and introns, tend to be very different because these sequences are not under strong selective pressure. Thus corresponding segments of the human and mouse genome that exhibit high sequence similarity are likely to be functionally important: exons, transcription-control regions, or sequences with other functions that are not yet understood.

## The Number of Protein-Coding Genes in an Organism's Genome Is Not Directly Related to Its Biological Complexity

The combination of genomic sequencing and gene-finding computer algorithms has yielded the complete inventory of protein-coding genes for a variety of organisms. Figure 8-22 shows the total number of protein-coding genes in several eukaryotic genomes that have been completely sequenced. The functions of about half the proteins encoded in these genomes are known or have been predicted on the basis of sequence comparisons. One of the surprising features of this comparison is that the number of protein-coding genes within different organisms does not seem proportional to our intuitive sense of their biological complexity. For example, the roundworm C. elegans apparently has more genes than the fruit fly Drosophila, which has a much more complex body
![li1862239539b](li1862239539b.jpg)

FIGURE 8-22 Comparison of the number and types of proteins encoded in the genomes of different eukaryotes. For each organism, the area of the entire pie chart represents the total number of protein-coding genes, all shown at roughly the same scale. In most cases, the functions of the proteins encoded by about half the genes are still unknown (light blue). The functions of the remainder are
known or have been predicted by sequence similarity to genes of known function. (Data from ENCODE Project Consortium, 2012, Nature 489:57; J. D. Hollister, 2014, Chromosome Res. 22:103; L. W. Hillier et al., 2005, Genome Res. 15:1651; FlyBase: FB2015_02 Release Notes, http://flybase.org/ static_pages/docs/release_notes.html; Saccharomyces Genome Data Base 2015, http://www.yeastgenome.org/genomesnapshot.J

plan and more complex behavior. And humans have only about 5 percent more protein-coding genes than C. elegans. When it first became apparent that humans have so few more protein-coding genes than the simple roundworm, it was difficult to understand how such a small increase in the number of proteins could generate such a staggering difference in complexity.

Clearly, simple quantitative differences in the number of protein-coding genes in the genomes of different organisms are inadequate for explaining differences in biological complexity. However, several phenomena can generate more complexity in the expressed proteins of higher eukaryotes than is predicted from their genomes. First, alternative splicing of a pre-mRNA can yield multiple functional mRNAs corresponding to a particular gene (see Chapter 10). In humans, the mean number of alternatively spliced mRNAs expressed per gene is about 6 . Second, variations in the posttranslational modification of many proteins may produce functional differences. Finally, increased biological complexity results from increased numbers of cells built of the same kinds of proteins. Larger numbers of cells can interact in more complex combinations, as we can see by comparing the cerebral cortices of mouse and human. Similar cells are present in the mouse and in the human cerebral cortex, but in humans more of them make more complex connections. Evolution of the increasing biological complexity of multicellular organisms probably required increasingly complex regulation of cell replication and temporal and spatial regulation of gene expression in the cells that make up the organisms, leading to increasing complexity of embryological development.

The specific functions of many genes and proteins identified by analysis of genomic sequences still have not been determined. As researchers unravel the functions of individual proteins in different organisms and further detail their interactions with other proteins, the resulting advances will become immediately applicable to all homologous proteins in other organisms. When the function of every protein is known, no doubt, a more sophisticated understanding of the molecular basis of complex biological systems will emerge.

## KEY CONCEPTS OF SECTION 8.4

## Genomics: Genome-Wide Analysis of Gene Structure and Function

- The function of a protein that has not been isolated (a query sequence) can often be predicted on the basis of similarity of its amino acid sequence to the sequences of proteins of known function.
- A computer algorithm known as BLAST rapidly searches databases of known protein sequences to find those with significant similarity to a query protein.
- Proteins with common functional motifs, which can often be quite short, may not be identified in a typical BLAST
search. Such short sequences may be located by searches of structural motif databases.
- A protein family comprises multiple proteins all derived from the same ancestral protein. The genes encoding these proteins, which constitute the corresponding gene family, arose by an initial gene duplication event and subsequent divergence during speciation (see Figure 8-21).
- Related genes and their encoded proteins expressed in one organism that derive from a gene duplication event are paralogous, such as the $\alpha$ - and $\beta$-globins that combine in hemoglobin $\left(\alpha_{2} \beta_{2}\right)$. Those that derive from mutations that accumulated during speciation are orthologous. Proteins that are orthologous usually have a similar function in different organisms.
- Open reading frames (ORFs) are regions of genomic DNA containing at least 100 codons located between a start codon and stop codon.
- Computer searching of the entire bacterial and yeast genomic sequences for open reading frames (ORFs) correctly identifies most protein-coding genes. Several types of additional data must be used to identify probable (putative) genes in the genomic sequences of humans and other higher eukaryotes because of their more complex gene structure, in which relatively short coding exons are separated by relatively long noncoding introns.
- Analysis of the complete genomic sequences of several different organisms indicates that biological complexity is not directly related to the number of protein-coding genes (see Figure 8-22).


### 8.5 Structural Organization of Eukaryotic Chromosomes

Now that we have examined the various types of DNA sequences found in eukaryotic genomes and how they are organized within genomes, we turn to the question of how DNA molecules as a whole are organized within eukaryotic cells. Because the total length of cellular DNA is up to a hundred thousand times a cell's diameter, the packing of DNA is crucial to cell architecture. It is also essential to prevent the long DNA molecules from getting knotted or tangled with each other during cell division, when they must be precisely segregated to daughter cells. The task of compacting and organizing chromosomal DNA is performed by abundant nuclear proteins called histones. The complex of histones and DNA is called chromatin.

Chromatin, which is about half DNA and half protein by mass, is dispersed throughout much of the nucleus in interphase cells (those that are not undergoing mitosis). Further folding and compaction of chromatin during mitosis (see Figure 6-3) produces the visible metaphase chromosomes whose morphology and staining characteristics were detailed by early cytogeneticists. Although every eukaryotic

![li1872239539b](li1872239539b.jpg)

EXPERIMENTAL FIGURE 8-23 The extended and condensed forms of extracted chromatin have very different appearances in electron micrographs. (a) Chromatin isolated in low-ionic-strength buffer has an extended "beads-on-a-string" appearance. The "beads" are nucleosomes ( 10 nm in diameter) and the "string" is connecting (linker) DNA. (b) Chromatin isolated in buffer with a physiological ionic strength ( 0.15 M KCl ) appears as a condensed fiber 30 nm in diameter. [Part (a) courtesy of Steven McKnight and Oscar Miller, Jr. Part (b) courtesy of Barbara Hamkalo and J. B. Rattner.]
chromosome includes millions of individual protein molecules, each chromosome contains just one, extremely long, linear DNA molecule. The longest DNA molecules in human chromosomes, for instance, are $2.8 \times 10^{8} \mathrm{bp}$, or almost 10 cm , in length! The structural organization of chromatin allows this vast length of DNA to be compacted into the microscopic constraints of a cell nucleus (see Figure 8-1). Yet chromatin is organized in such a way that specific DNA sequences within the chromatin are readily available for cellular processes such as the transcription, replication, repair, and recombination of DNA molecules. In this section, we consider the properties of chromatin and its organization into chromosomes. Important features of chromosomes in their entirety are covered in the next section.

## Chromatin Exists in Extended and Condensed Forms

Histones, the most abundant proteins in chromatin, constitute a family of small, basic proteins. The five major types of histone proteins-termed $H 1, H 2 A, H 2 B, H 3$, and $H 4$-are rich in positively charged basic amino acids, which interact with the negatively charged phosphate groups in DNA.

When chromatin is extracted from nuclei and examined in the electron microscope, its appearance depends on the salt concentration to which it is exposed. At low salt concentrations and in the absence of divalent cations such as $\mathrm{Mg}^{2+}$, isolated chromatin resembles beads on a string (Figure 8-23a). In this extended form, the "string" is composed of free DNA, called "linker" DNA, connecting beadlike structures termed nucleosomes. Composed of DNA and histones, nucleosomes are about 10 nm in diameter and are the primary structural units of chromatin. If chromatin is isolated at a physiological salt concentration, it assumes a more condensed fiberlike form that is about 30 nm in diameter (Figure 8-23b).

Structure of Nucleosomes The DNA component of nucleosomes is much less susceptible to nuclease digestion than is the linker DNA between them. If nuclease treatment is carefully controlled, all the linker DNA can be digested,
releasing individual nucleosomes with their DNA component. A nucleosome consists of a protein core with DNA wound around its surface like thread around a spool. The core is an octamer containing two copies each of histones H2A, H2B, H3, and H4. X-ray crystallography has shown that the octameric histone core is a roughly disk-shaped structure made of interlocking histone subunits (Figure 8-24). Histones H3 and H4 fold into a tetramer in the absence of histones H2A and H2B. Two heterodimers of H2A and H2B then associate with the H3-H4 tetramer. Positive charges on the surface of the histone octamer in the region interacting with DNA hold the negatively charged DNA against the surface of the histone octamer at the center of the nucleosome. Nucleosomes from all eukaryotes contain about 147 bp of DNA wrapped one and two-thirds turns around the globular histone core. The length of the linker DNA is more variable among species, and even between different cells of the same organism, ranging from about 10 to 90 base pairs. During cell replication, DNA is assembled into nucleosomes shortly after the replication fork passes (see Figure 5-32). This process depends on specific chaperone molecules that bind to histones and assemble them, together with newly replicated DNA, into nucleosomes.

Structure of the 30-nm Fiber When extracted from cells in isotonic buffers (i.e., buffers with the same salt concentration found in cells, about $0.15 \mathrm{M} \mathrm{KCl}, 0.004 \mathrm{M} \mathrm{MgCl}_{2}$ ), most chromatin appears as fibers about 30 nm in diameter (see Figure 8-23b). Despite many years of study, the arrangement of nucleosomes in the $30-\mathrm{nm}$ chromatin fiber remains controversial. In one widely accepted model with considerable experimental support, nucleosomes are arranged in a helical conformation forming a solenoid with six or more nucleosomes per turn of the helix (Figure 8-25a). In a second strongly supported model, the two-start helix, nucleosomes alternately stack like coins into two "strands," and those strands wind into a left-handed double helix (Figure 8-25b). The $30-\mathrm{nm}$ fibers also include H 1 , the fifth major histone. H 1 is bound to the DNA as it enters and exits the nucleosome core, but its structure in the $30-\mathrm{nm}$ fiber is not known at atomic resolution.

![li1882239539b](li1882239539b.jpg)

FIGURE 8-24 Structure of the nucleosome based on x-ray crystallography. (a) Nucleosome with space-filling model of the histones. The sugar-phosphate backbones of the DNA strands are represented as white tubes to allow better visualization of the histones. Nucleosome shown from the top (left) and from the side (right, rotated clockwise $90^{\circ}$ ). H2A subunits are yellow; H2Bs are red; H3s are blue; H4s are green. The N-terminal tails of the eight histones and the H2A and H2B C-terminal tails, involved in condensation of the chromatin, are not visible because they are disordered in the crystal. One H2A, H2B heterodimer projects out of the page on the lower right of the side view, while the
other H2A, H2B heterodimer projects into the page, on the lower left of the side view. Only one H2A, H2B heterodimer is visible in the top view. The other H2A, H2B dimer is not visible in this view because it is behind the H3, H4 tetramer, on the lower right. (See also the ribbon diagram of the histone polypeptide chains in Figure 8-26, where only one H2A, H2B heterodimer is clearly visible on the lower left of the top view of the nucleosome.) (b) Space-filling model of histones and DNA (white) viewed from the side of the nucleosome. See also http://lugerlab.org for a rotating movie of the nucleosome core. (Data from K. Luger et al., 1997, Nature 389:251, PDB ID 1aoi.)
The chromatin in chromosomal regions that are not being transcribed or replicated exists predominantly in the condensed, $30-\mathrm{nm}$ fiber form and in higher-order folded structures whose detailed conformation is not currently understood. The regions of chromatin actively being transcribed and replicated are thought to assume the extended beads-on-a-string form.

Conservation of Chromatin Structure The general structure of chromatin is remarkably similar in the cells of all eukaryotes, including fungi, plants, and animals, indicating that the structure of chromatin was optimized early in the evolution of eukaryotic cells. The amino acid sequences for the four core histones (H2A, H2B, H3, and H4) are highly conserved between distantly related species. For example, the sequences of histone H 3 from sea urchin tissue and calf thymus differ by only a single amino acid, and H 3 from the garden pea and calf thymus differ by only four amino acids. Apparently, significant deviations from the histone amino acid sequences were selected against strongly during evolution. The amino acid sequence of H 1 , however, varies more from organism to organism than do the sequences of the other major histones. The similarity in sequence among histones from all eukaryotes indicates that they fold into very
similar three-dimensional conformations, which were optimized for histone function early in evolution in a common ancestor of all modern eukaryotes.

Minor histone variants encoded by genes that differ from the highly conserved major types also exist, particularly in vertebrates. For example, a special form of H2A, designated H2AX, is incorporated into nucleosomes in place of H 2 A in a small fraction of nucleosomes in all regions of chromatin. At sites of double-stranded breaks in chromosomal DNA, H2AX becomes phosphorylated and participates in the chromosome-repair process, probably by functioning as a binding site for repair proteins. In the nucleosomes at centromeres, H 3 is replaced by another variant histone called CENP-A, which participates in the binding of spindle microtubules during mitosis. A variant of histone H 3 , known as H 3.3 , replaces the major histone H 3 in transcribed regions of DNA, probably when the histone octamer must be moved out of the way by histone chaperones as RNA polymerase transcribes the DNA in chromatin. Most minor histone variants differ only slightly in sequence from the major histones. These slight changes in histone sequence may influence the stability of the nucleosome as well as its tendency to fold into the $30-\mathrm{nm}$ fiber and other higher-order structures.

![li1892239539b](li1892239539b.jpg)

FIGURE 8-25 Models of the structure of the 30-nm chromatin fiber. (a) In the solenoid model, nucleosomes are arranged in a lefthanded helix with six or more nucleosomes per turn. See M. Kruithof et al., 2009, Nature Struc. Mol. Biol. 16:534. (b) In the two-start helix model, a "zigzag ribbon" of nucleosomes (top) folds into a two-start
helix (bottom). For simplicity, DNA is not represented in the two-start helix. See C. L. F. Woodcock et al., 1984, J. Cell Biol. 99:42. (Part (b) top data from K. Luger et al., 1997, Nature 389:251, PDB ID 1ace. Part (b) bottom data from T. Schalch et al., 2005, Nature 436:138, PDB ID 1zbb.]

## Modifications of Histone Tails Control Chromatin Condensation and Function

Each of the histone proteins making up the nucleosome core contains an intrinsically disordered, flexible N -terminus (see Figure 3-8) of 19-39 residues extending from the globular structure of the nucleosome; the H2A and H2B proteins also contain a flexible C-terminus extending from the globular histone octamer core. These termini, called histone tails, are represented in the model shown in Figure 8-26a. The histone tails are required for chromatin to condense from the beads-on-a-string conformation into the $30-\mathrm{nm}$ fiber. For example, recent experiments indicate that the N -terminal tails of
histone H 4 , particularly lysine 16 , are critical for forming the $30-\mathrm{nm}$ fiber. This positively charged lysine interacts with a negative patch at the H2A-H2B interface of the next nucleosome in the stacked nucleosomes of the $30-\mathrm{nm}$ fiber.

Histone tails are subject to multiple post-translational modifications such as acetylation, methylation, phosphorylation, and ubiquitinylation. Figure 8-26b summarizes the types of post-translational modifications observed in human histones. A particular histone protein never has all of these modifications simultaneously, but the histones in a single nucleosome usually contain several of these modifications simultaneously. The particular combinations of post-transcriptional modifications found in different regions

![li1902239539b](li1902239539b.jpg)

FIGURE 8-26 Post-translational modifications observed on human histones. (a) Model of a nucleosome viewed from the top with histones shown as ribbon diagrams. This model depicts the lengths of the histone tails (dotted lines), which are not visible in the crystal structure (see Figure 8-24). The H2A N-terminal tails are at the bottom, and the H2A C-terminal tails are at the top. The H2B N-terminal tails are on the right and left, and the H2B C-terminal tails are at the bottom center. Histones H3 and H4 have short C-terminal tails that are not modified. (b) Summary of post-translational modifications observed
in human histones. Histone-tail sequences are shown in the one-letter amino acid code (see Figure 2-14). The main portion of each histone is depicted as an oval. These modifications do not all occur simultaneously on a single histone molecule. Rather, specific combinations of a few of these modifications are observed on any one histone. See K. Luger and T. J. Richmond, 1998, Curr. Opin. Genet. Devel. 8:140. [Part (a) data from K. Luger et al., 1997, Nature 389:251, PDB ID :1ack. Part (b) data from R. Margueron et al., 2005, Curr. Opin. Genet. Devel. 15:163.]
of chromatin constitute a histone code that influences chromatin function by creating or removing binding sites for chromatin-associated proteins depending on the specific combinations of these modifications present. Here we describe the most abundant kinds of modifications found in histone tails and how these modifications control chromatin condensation and function. We end with a discussion of a special case of chromatin condensation, the inactivation of X chromosomes in female mammals.

Histone Acetylation Histone-tail lysines undergo reversible acetylation and deacetylation by enzymes that act on specific lysines in the histone N-termini. In the acetylated form, the positive charge of the lysine $\varepsilon$-amino group is neutralized. As mentioned above, lysine 16 in histone H 4 is particularly important for the folding of the $30-\mathrm{nm}$ fiber because it interacts with a negatively charged patch on the surface of the neighboring nucleosome in the fiber. Consequently, when H 4 lysine 16 is acetylated, the chromatin tends to form the less condensed "beads-on-a-string" conformation conducive for transcription and replication.

Histone acetylation at other sites in H 4 and in other histones (see Figure 8-26b) is correlated with increased
sensitivity of chromatin DNA to digestion by nucleases-an observation indicating that histone acetylation is associated with a less condensed form of chromatin. This correlation can be demonstrated by conducting experiments in which isolated nuclei are digested with DNase I. Following digestion, the DNA is completely separated from chromatin protein, digested to completion with a restriction enzyme, and analyzed by Southern blotting. An intact gene treated with a restriction enzyme yields fragments of characteristic sizes. If the nuclei are exposed first to DNase, however, the gene may be cleaved at random sites within the boundaries of the restriction enzyme cut sites. Consequently, any Southern blot bands normally seen with that gene will be lost. This method was first used to show that the $\beta$-globin gene in non-erythroid cells, in which it is transcriptionally inactive and in which it is associated with relatively nonacetylated histones, is much more resistant to DNase I than it is in erythroid progenitor cells, in which it is actively transcribed and in which it is associated with acetylated histones (Figure 8-27). These results indicate that the chromatin structure of nontranscribed DNA in hypoacetylated chromatin makes the DNA less accessible to DNase I than it is in transcribed, hyperacetylated chromatin. This is thought to

![li1912239539b](li1912239539b.jpg)

EXPERIMENTAL FIGURE 8-27 Nontranscribed genes are less susceptible to DNase I digestion than active genes. Chick embryo erythroblasts at 14 days actively synthesize globin, whereas undifferentiated chicken lymphoblastic leukemia (MSB) cells do not. (a) Nuclei from each type of cell were isolated and exposed to increasing concentrations of DNase I. The nuclear DNA was then extracted and treated with the restriction enzyme BamHI, which cleaves the DNA around the globin sequence and normally releases a $4.6-\mathrm{kb}$ globin fragment. (b) The DNase I- and BamHI-digested DNA was subjected to Southern blot analysis with a probe of labeled cloned adult globin DNA, which hybridizes to the 4.6-kb BamHI fragment. If the globin gene is susceptible to the initial DNase digestion, it should be cleaved repeatedly and would not be expected to show this fragment. As seen in the Southern blot, the transcriptionally active DNA from the 14-day globin-synthesizing cells was sensitive to DNase I digestion, indicated by the absence of the $4.6-\mathrm{kb}$ band at high nuclease concentrations. In contrast, the inactive DNA from MSB cells was resistant to digestion. These results suggest that the inactive DNA is in a more condensed form of chromatin in which the globin gene is shielded from DNase digestion. See J. Stalder et al., 1980, Cell 19:973. [Part (b) republished with permission of Elsevier, from Stalder, J. et al., "Hb switching in chickens," Cell, 1980; 19(4):973-80; permission conveyed through Copyright Clearance Center, Inc.]
be because chromatin containing the repressed gene is folded into condensed structures that sterically inhibit access to the associated DNA by the nuclease. In contrast, the transcribed gene is associated with a more unfolded form of chromatin, which allows better access of the nuclease to the associated

DNA. Presumably, the condensed chromatin structure in non-erythroid cells also sterically inhibits access of the promoter and other transcription-control sequences in DNA to the proteins involved in transcription, contributing to transcriptional repression (see Chapter 9).

Genetic studies in yeast indicated that histone acetyl transferases (HATs), which acetylate specific lysine residues in histones, are required for the full activation of transcription of a number of genes. These enzymes are now known to have other substrates that influence gene expression in addition to histones. Consequently, they are more generally known as nuclear lysine acetyl transferases, or KATs, because $K$ represents lysine in the single-letter code for amino acids (see Figure 2-14). Conversely, early genetic studies in yeast indicated that complete repression of many yeast genes requires the action of histone deacetylases (HDACs) that remove acetyl groups of acetylated lysines from histone tails, as discussed further in Chapter 9.

Other Histone Modifications As shown in Figure 8-26b, histone tails in chromatin can undergo a variety of other covalent modifications at specific amino acids. Lysine $\varepsilon$-amino groups can be methylated, a process that prevents acetylation, thus maintaining their positive charge. Moreover, the N of lysine $\varepsilon$-amino groups can be methylated once, twice, or three times. Arginine side chains can also be methylated. The O in hydroxyl groups $(-\mathrm{OH})$ of serine and threonine side chains can be reversibly phosphorylated, introducing two negative charges. Each of these post-translational modifications contributes to the binding of chromatin-associated proteins that participate in the control of chromatin folding and the ability of DNA and RNA polymerases to replicate or transcribe the associated DNA. Finally, a single 76-aminoacid ubiquitin molecule can be reversibly added to a lysine in the C-terminal tails of H2A and H2B. Recall that addition of multiple linked ubiquitin molecules to a protein can mark it for degradation by the proteasome (see Figure 3-31). In this case, however, the addition of a single ubiquitin molecule does not affect the stability of a histone, although it does influence chromatin structure.

As mentioned previously, it is the precise combination of modified amino acids in histone tails that helps control the condensation, or compaction, of chromatin and its ability to be transcribed, replicated, and repaired. The extent of DNA compaction can be observed by electron microscopy and by light microscopy using dyes that bind DNA. Condensed regions of chromatin, known as heterochromatin, stain much more darkly than less condensed chromatin, known as euchromatin (Figure 8-28a). Heterochromatin does not fully decondense following mitosis, remaining in a compacted state during interphase and usually associating with the nuclear envelope, nucleoli, and additional distinct foci. Heterochromatin includes centromeres and telomeres of chromosomes as well as transcriptionally inactive genes. In contrast, areas of euchromatin, which are in a less compacted state during interphase, stain lightly with DNA dyes. Most transcribed

![li1922239539b](li1922239539b.jpg)

FIGURE 8-28 Heterochromatin versus euchromatin. (a) In this electron micrograph of a bone marrow stem cell, the dark-staining areas in the nucleus ( N ) outside the nucleolus ( n ) are heterochromatin. The light-staining, whitish areas are euchromatin. (b) The modifications of histone N -terminal tails in heterochromatin and euchromatin differ, as illustrated here for histone H3. Note in particular that histone tails are generally much more extensively acetylated in euchromatin than in heterochromatin. Heterochromatin is much more condensed (thus less accessible to proteins) and is much less transcriptionally active than is euchromatin. [Part (a) Don W Fawcett/Science Source/Getty Images. Part (b) data from T. Jenuwein and C. D. Allis, 2001, Science 293:1074.]
regions of DNA are found in euchromatin. Heterochromatin usually contains histone H 3 modified by methylation of lysine 9 , while euchromatin generally contains histone H 3 extensively acetylated on lysine 9 and other H3 lysines, especially at promoters and enhancers where there is also methylation of lysine 4 , and phosphorylation of serine 10 (Figure 8-28b). Other histone tails are also differently modified in euchromatin and in heterochromatin. For example, H4 lysine 16 is generally nonacetylated in heterochromatin, which allows it to interact with neighboring nucleosomes and stabilize chromatin folding into the $30-\mathrm{nm}$ fiber (see Figure 8-25).

Reading the Histone Code The histone code of modified amino acids in the histone tails is "read" by proteins that bind to the modified tails and in turn promote condensation or decondensation of chromatin. Eukaryotes express a number of proteins containing a so-called chromodomain that binds to histone tails when they are methylated at specific lysines. One example of such a protein is heterochromatin protein 1 (HP1). The chromodomain of HP1 binds the H3 N -terminal tail only when it is di- or trimethylated at lysine 9 (see Figure 8-28b). HP1 also contains a second domain called a chromoshadow domain because it is frequently found in proteins that contain a chromodomain. The chromoshadow domain binds to other chromoshadow domains. Consequently, chromatin containing H3 di- and trimethylated at lysine 9 ( $\mathrm{H} 3 \mathrm{~K} 9 \mathrm{Me}_{2 / 3}$ ) is assembled into a condensed chromatin structure by HP1, although the structure of this chromatin is not well understood.

In addition to binding to itself, the chromoshadow domain of HP1 also binds the enzyme that methylates H3 lysine 9, an H3K9 histone methyl transferase (HMT) (Figure 8-29a). As a consequence, nucleosomes adjacent to a region of HP1containing heterochromatin also become methylated at lysine 9 (Figure 8-29b). This methylation creates a binding site for another HP1 that can bind the H3K9 HMT, resulting in "spreading" of the heterochromatin structure along the chromosome until a boundary element is encountered that blocks further spreading. Boundary elements so far characterized are generally regions in chromatin where several nonhistone proteins bind to DNA, possibly blocking histone methylation on the other side of the boundary.

Epigenetic Memory Significantly, the model of heterochromatin formation in Figure 8-29b provides an explanation for how heterochromatic regions of a chromosome are reestablished following DNA replication during the S phase of the cell cycle. When DNA in heterochromatin is replicated, the histone octamers that are di- or trimethylated at H3 lysine 9 become distributed to both daughter chromosomes along with an equal number of newly assembled histone octamers. The H3K9 HMT associated with the H3K9 di- and trimethylated nucleosomes methylates lysine 9 of the newly assembled nucleosomes, regenerating the heterochromatin in both daughter chromosomes. Consequently, heterochromatin is marked with an epigenetic code, so called because it does not depend on the sequence of bases in DNA, that maintains the repression of associated genes in replicated daughter cells.

Other protein domains associate with histone-tail modifications typical of euchromatin. For example, the bromodomain binds to acetylated histone tails and therefore is associated with transcriptionally active chromatin. Several proteins involved in stimulating gene transcription contain bromodomains, such as the largest subunit of TFIID (see Chapter 9). This transcription factor contains two closely spaced bromodomains that probably help TFIID to associate with transcriptionally active chromatin (i.e., euchromatin). TFIID and other bromodomain-containing proteins also have histone acetylase activity, which helps to maintain

![li1932239539b](li1932239539b.jpg)

FIGURE 8-29 Model for the formation of heterochromatin by the binding of HP1 to histone H3 trimethylated at lysine 9. (a) HP1 contributes to the condensation of heterochromatin by binding to histone H3 N -terminal tails trimethylated at lysine 9 , then associating with other histone-bound HP1 molecules. (b) Heterochromatin condensation can spread along a chromosome because HP1 binds the histone methyltransferase (HMT) that methylates lysine 9 of histone H 3 . This creates a binding site for HP1 on the neighboring nucleosome. The spreading process continues until a "boundary element" is encountered. See G. Thiel et al., 2004, Eur. J. Biochem. 271:2855, and A. J. Bannister et al., 2001, Nature 410:120.
the chromatin in a hyperacetylated state conducive to transcription. Consequently, an epigenetic code associated with euchromatin helps to maintain the transcriptional activity of genes in euchromatin through successive cell divisions. These epigenetic codes for heterochromatin and euchromatin help to maintain the patterns of gene expression established in different cell types during early embryonic development as specific differentiated cell types increase in numbers by cell division. Importantly, abnormal alterations in these epigenetic codes have been found to contribute to the pathogenic replication and behavior of cancer cells (see Chapter 24).

In summary, multiple types of covalent modifications of histone tails can influence chromatin structure by altering nucleosome-nucleosome interactions and interactions with additional proteins that participate in or regulate processes such as transcription and DNA replication. The mechanisms and molecular processes governing chromatin modifications that regulate transcription are discussed in greater detail in the next chapter.

X-Chromosome Inactivation in Mammalian Females One important example of epigenetic gene control through repression by heterochromatin is the random inactivation and condensation of one of the two X chromosomes in female mammals. Each female mammal has two X chromosomes, one contributed by the egg from which it developed $\left(\mathrm{X}_{\mathrm{m}}\right)$ and one contributed by the sperm $\left(\mathrm{X}_{\mathrm{p}}\right)$. Early during embryonic development, random inactivation of either the $\mathrm{X}_{\mathrm{m}}$ or the $\mathrm{X}_{\mathrm{p}}$ chromosome occurs in each somatic cell. In the female embryo, about half the cells have an inactive $\mathrm{X}_{\mathrm{m}}$, and the other half have an inactive $\mathrm{X}_{\mathrm{p}}$. All subsequent daughter cells maintain the same inactive X chromosomes as their parent cells. As a result, the adult female is a mosaic of clones, some expressing the genes from the $\mathrm{X}_{\mathrm{m}}$ and some expressing the genes from the $\mathrm{X}_{\mathrm{p}}$. This inactivation of one X chromosome in female mammals results in dosage compensation, a process that ensures that cells of females express proteins encoded on the X chromosome at the same levels as the cells of males, which have only one X chromosome.

Histones associated with the inactive X chromosome have post-translational modifications characteristic of other regions of heterochromatin: hypoacetylation of lysines, di- and trimethylation of histone H3 lysine 9, trimethylation of H3 lysine 27, and a lack of methylation at histone H3 lysine 4 (see Figure 8-28b). X-chromosome inactivation at an early stage in embryonic development is controlled by the X -inactivation center, a complex locus on the X chromosome that determines which of the two X chromosomes will be inactivated and in which cells. The X-inactivation center also contains the XIST gene, which encodes a remarkable long nonprotein-coding RNA that coats only the X chromosome it was transcribed from, thereby triggering silencing of that chromosome.

Although the mechanism of X-chromosome inactivation is not fully understood, it involves several processes, including the action of Polycomb protein complexes, which are discussed further in Chapter 9. One subunit of the Polycomb

![li1942239539b](li1942239539b.jpg)
complex contains a chromodomain that binds to histone H3 tails when they are trimethylated at lysine 27. The Polycomb complex also contains a histone methyl transferase specific for H3 lysine 27. This finding helps to explain how the X-inactivation process spreads along large regions of the X chromosome and how it is maintained through DNA replication, apparently in a manner similar to heterochromatization by the binding of HP1 to histone H3 tails methylated at lysine 9 (see Figure 8-29b).

X-chromosome inactivation is another example of an epigenetic process; that is, a process that affects the expression of specific genes and is inherited by daughter cells, but is not the result of a change in DNA sequence. In other words, the activity of genes on the X chromosome in female mammals is controlled by chromatin structure, rather than by the nucleotide sequence of the underlying DNA. And the inactivated X chromosome (either $\mathrm{X}_{\mathrm{n}}$, or $\mathrm{X}_{\mathrm{p}}$ ) is maintained as the inactive chromosome in the progeny of all future cell divisions because the histones are modified in a specific, repressing manner that is faithfully inherited through each cell division.

## Nonhistone Proteins Organize Long Chromatin Loops

Although histones are the predominant proteins in chromatin, other, less abundant, nonhistone chromatin-associated

EXPERIMENTAL FIGURE 8-30 Fluorescent-labeled probes hybridized to interphase chromosomes demonstrate chromatin loops and permit their measurement. In situ hybridization to interphase cells was carried out with several different probes specific for sequences separated by known distances in linear, cloned DNA. Lettered circles represent probes. Measurement of the distances between different hybridized probes, which could be distinguished by their color, showed that some sequences (e.g., $A$ and $B$ ), separated from one another by millions of base pairs, appear located near one another within nuclei. For some sets of sequences, the measured distances in nuclei between one probe (e.g., C) and sequences successively farther away initially appear to increase (e.g., D, E, and F) and then appear to decrease (e.g., G and H). See H. Yokota et al., 1995, J. Cell Biol. 130:1239.
proteins, as well as the DNA molecule itself, are also crucial to chromosome structure. Recent results indicate that it is not protein alone that gives a metaphase chromosome its structure. Micromechanical studies of large metaphase chromosomes from newts in the presence of proteases or nucleases indicate that DNA, not protein, is responsible for the mechanical integrity of a metaphase chromosome when it is pulled from its ends. These results are inconsistent with a continuous protein scaffold at the chromosome axis. Rather, the integrity of chromosome structure requires the complete chromatin complex of DNA, histone octamers, and nonhistone chromatin-associated proteins.

Experiments in which several different fluorescentlabeled probes were hybridized to the DNA of one chromosome in human interphase cells support a model in which chromatin is arranged in large loops. In these experiments, some sequences separated by millions of base pairs in linear DNA appeared very close to one another in interphase nuclei (Figure 8-30). These closely spaced probe sites are postulated to lie at the bases of the chromatin loops. Loops of chromatin can also be directly visualized by light microscopy in the active chromatin of growing amphibian oocytes ("lampbrush chromosomes") (Figure 8-31). These cells are enormous compared to most cells ( $\sim 1 \mathrm{~mm}$ in diameter) because they stockpile all of the nuclear and cytoplasmic material required for division of the fertilized egg into the thousands of differentiated cells required to generate a feeding
![li1952239539b](li1952239539b.jpg)

EXPERIMENTAL FIGURE 8-31 Loop of chromatin in a lampbrush chromosome. A short segment of a lampbrush chromosome in the nucleus of an oocyte from the newt Notophthalmus viridescens. (a) Differential interference contrast (DIC) microscopy of a portion of the lampbrush chromosome and a loop with transcribed RNA associated with hnRNP proteins (arrow; see Chapter 10). (b) The same field observed by immunofluorescence after staining with antibody to RNA polymerase II. [Republished with permission of Elsevier, from Gall, J.G. et al., "Structure in the amphibian germinal vesicle," Exp. Cell Res., 2004, 296(1):28-34; permission conveyed through Copyright Clearance Center, Inc.]

embryo that can ingest additional nutrients. The nuclei of these amphibian oocytes are 50 times the diameter of most vertebrate somatic cells. The chromatin of these cells is not constrained by a closely associated nuclear envelope, but rather "floats" in a volume of nucleoplasm about $10^{5}$ times greater than in most somatic cells. Microscopy of chromatin from these cells stained with antibody to RNA polymerase II shows long loops of chromatin densely packed with polymerase extending from a central core of highly condensed, transcriptionally inactive chromatin. As discussed below, the loops are tethered at their bases by a mechanism that does not interrupt the duplex DNA molecule that extends the entire length of the chromosome.

Ringlike Structure of SMC Protein Complexes The bases of chromatin loops (see Figure 8-30) in interphase chromosomes may be held in place by proteins called structural maintenance of chromosome (SMC) proteins. These nonhistone proteins are critical for maintaining the structure of condensed chromosomes during mitosis. In extracts prepared from the large nuclei of Xenopus laevis (African frog) eggs, chromosomes can be induced to condense as they do in intact cells as they enter prophase of mitosis. This condensation fails to occur when one type of SMC protein is depleted from the extract with specific antibodies. Yeast with mutations in another type of SMC protein fail to properly associate sister chromatids following DNA replication in the S phase. As a result, chromosomes do not properly segregate to daughter cells during mitosis. Related SMC proteins are required for proper segregation of chromosomes in bacteria and archaea, indicating that SMCs are an ancient class of proteins vital to chromosome structure and segregation in all kingdoms of life.

Each SMC monomer contains a hinge region where the polypeptide folds back on itself, forming a very long coiledcoil region and bringing the N - and C -termini together so they can interact to form a globular head domain (Figure 8-32a). The hinge domain of one monomer (blue in Figure 8-32) binds to the hinge domain of a second monomer (red), forming a roughly U-shaped dimeric complex. The head domains of the monomers have ATPase activity and are linked by members of another small protein family called kleisins. The overall SMC complex is a ring with a diameter large enough to accommodate two $30-\mathrm{nm}$ chromatin fibers (Figure 8-32b) and is capable of linking two circular DNA molecules in vitro. SMC proteins are proposed to form the bases of chromatin loops by forming topologically constrained knots in 30-nm chromatin fibers, as diagrammed in Figure 8-32c. This model can explain why cleavage of the DNA at a relatively small number of sites leads to rapid dissolution of condensed metaphase chromosome structure, whereas protease cleavage of proteins has only a minor effect on chromosome structure until most of the protein is digested. When the DNA is cut anywhere in a long region of chromatin containing several chromatin loops, the broken ends can slip through the SMC protein rings, "untying" the topological knots that constrain the loops of chromatin. In contrast, most of the individual rings of SMC proteins must be broken
![li1962239539b](li1962239539b.jpg)
![li1972239539b](li1972239539b.jpg)

FIGURE 8-32 Model of SMC complexes bound to chromatin.
(a) Model of an SMC protein complex. (b) Model of SMC complex topologically linking two $30-\mathrm{nm}$ chromatin fibers (represented by cylinders). (c) Model for the binding of SMC complexes to the base of a loop of transcribed chromatin. See K. Nasmyth and C. H. Haering, 2005, Ann. Rev. Biochem. 74:595.
before the topological constraints holding the bases of the loops together are released.

Interphase Chromosome Territories In the small nuclei of most cells, individual interphase chromosomes, which are less condensed than metaphase chromosomes, cannot be resolved by standard microscopy or electron microscopy. Nonetheless, the chromatin of an interphase chromosome is not spread throughout the nucleus. Rather, interphase chromatin is organized into chromosome territories. As illustrated in Figure 8-33, in situ hybridization of interphase nuclei with chromosome-specific fluorescent-labeled probes shows that the probes are localized within restricted regions of the nucleus, rather than appearing throughout the nucleus. Use of probes specific for different chromosomes shows that there is little overlap between chromosomes in interphase nuclei. The precise positions of chromosomes are not reproducible between cells, although large chromosomes tend to lie at the periphery of the nucleus and small chromosomes toward the center. Also, repeats of rRNA transcription units on chromosomes $13,14,15,21$, and 22 , known as nucleolar organizers, associate with nucleoli found near the center of the nucleus.

Topological Domains Within Chromosome Territories A group of related methods referred to as chromosome conformation capture (" 3 C ") methods have been made possible by the advent of massively parallel DNA sequencing

(a)
![li1982239539b](li1982239539b.jpg)

## EXPERIMENTAL FIGURE 8-33 During interphase, human

chromosomes remain in non-overlapping territories in the nucleus. (a) Fixed interphase human fibroblasts were hybridized in situ to fluorescently labeled probes specific for sequences along the full length of human chromosomes 7 (cyan) and 8 (purple). DNA is stained blue with DAPI. In the diploid cell, each of the two chromosome 7 s and two chromosome 8 s is restricted to a territory or domain within the nucleus, rather than stretching throughout the entire nucleus.
and the ability to sequence tens of millions of $50-100-\mathrm{bp}$ DNA fragments (see Chapter 6). The goal of these methods is to determine the three-dimensional spatial organization of chromatin within nuclei of interphase cells. In human interphase cells, about 2 m of DNA is folded and refolded to fit within a nucleus only about $20 \mu \mathrm{~m}$ in diameter! The general strategy of chromosome conformation capture is illustrated in Figure 8-34, where associations between distant regions of chromatin are represented in (a), and regions of chromatin not specifically associated with another region are shown in (b). First (step $\mathbf{\square}$ ), intact cells are treated with chemical cross-linkers such as formaldehyde that diffuse through cell membranes and covalently cross-link protein to protein and protein to DNA. Next $\mathbf{\square}$, the cross-linked chromatin is isolated and either digested with a restriction enzyme or subjected to intense sonication to mechanically shear the DNA into fragments of 200-600 bp. $\mathbf{\square}$ Short oligonucleotide linkers are then ligated to the ends of the DNA fragments. These linkers contain a cytosine linked to a biotin at the $5^{\prime}$ position (see Figure 2-17). ■ After removal of excess linker, the preparation is diluted considerably and subjected to treatment with DNA ligase. Because the preparation is diluted, ligations occur preferentially between the ends of fragments held in close proximity by cross-links. After ligation, the protein-DNA cross-linking reaction is reversed, the protein is digested with proteases, and the DNA is isolated and further sonicated. Ligated fragments are separated from other fragments using streptavidin, which binds the biotin in the linkers added in step $\mathbf{\square}$.
(b)
![li1992239539b](li1992239539b.jpg)
(b) This image from a fixed interphase fibroblast from a human male was made with a method similar to that used in (a), except that chromosome paint probes specific for each chromosome were hybridized to the cell to reveal the locations of nearly all the chromosomes. Some of the chromosomes are not observed in this confocal slice through the nucleus. [Part (a) courtesy of Dr. Irina Solovei. Part (b) from A. Bolzer et al., 2005, Three-dimensional maps of all chromosomes in human male fibroblast nuclei and prometaphase rosettes. PLoS Biol. 3:826.]

The purified, ligated fragments are then sequenced. Ligation points are marked by the duplicated sequence of the oligonucleotide linker (see Figure 8-34, step $\mathbf{\square}$ ). Sequences on each side of the ligation point are mapped to the genome. In cases in which the two ends of a single fragment were linked together (Figure 8-34b), the sequences on either side of the oligonucleotide linker sequence will map within a few hundred bases of each other on the genome, since the ligated fragments were only a few hundred bases long. But in cases in which fragments that are distant in the genomic sequence were ligated together because they were cross-linked to proteins holding them together (Figure 8-34a), the sequences will map far apart. The observation of distant ( $>10 \mathrm{~kb}$ ) sequences repeatedly ligated to each other implies that their respective regions of chromatin were associated with each other in vivo.

The data from a chromatin conformation capture assay can be plotted on a two-dimensional heat map (Figure 8-34c). In this plot, the sequence of the same portion of the genome is plotted on both the $x$ and $y$ axes, with each pixel equivalent to 10 kb . The pixel at coordinates $x, y$ is colored red where a sequence at coordinate $x$ was ligated to a sequence at coordinate $y$. The intensity of the red color is proportional to the numbers of ligation events observed that linked a sequence in the $10-\mathrm{kb}$ interval $x$ with a sequence in the $10-\mathrm{kb}$ interval $y$.

Figure 8-34c shows the plot generated for a roughly $5.5-\mathrm{Mb}$ region of chromosome 6 in mouse embryonic stem cells. It is immediately apparent that the genome is divided into

![li2002239539b](li2002239539b.jpg)
regions called topological domains, in which a chromosomal region is far more likely to be ligated to another sequence within the same topological domain than it is to be ligated to a sequence in another topological domain. These topological domains are on the order of 200 kb to 1.5 Mb in length, with
![li2012239539b](li2012239539b.jpg)

FIGURE 8-34 Chromosome conformation capture. (a, b) Strategy of chromatin conformation capture methods. See text for explanation. Gray and blue tubes represent regions of DNA separated by $>10 \mathrm{~kb}$ in the genome sequence. Biotin is represented by red flags. See E. Lieberman-Aiden, 2009, Science 326:289. (c) Heat map of chromosome conformation capture data for a region of chromosome 6 in mouse embryonic stem cells. The sequence from 49 to 54 Mb from the left end of chromosome 6 is represented on both axes. Each pixel shows data from a 10 kb sequence. The number of times a sequence from one $10-\mathrm{kb}$ region indicated on the $x$ axis was ligated to a sequence from a second $10-\mathrm{kb}$ region on the $y$ axis is indicated by the intensity of red color, as shown in the key at the lower left. A value of 100 (dark red) indicates that a sequence anywhere within the $10-\mathrm{kb}$ region on the $x$ axis was found ligated to a sequence from anywhere in the $10-\mathrm{kb}$ region on the $y$ axis 100 times. Since the probability that two ends generated by sonication will be ligated together is higher for ends that are close together than for ends that are far apart, the intensity of the red color in any pixel indicates the relative proximity of the sequences in the two $10-\mathrm{kb}$ intervals in the nuclei at the time of crosslinking. Inset shows a model of chromatin folding that is consistent with these results. [Part (c) data from J. R. Dixon, 2012, Nature 485:376.]
a median size of 880 kb . For example, sequences in the interval of chromosome 6 between 50.9 Mb and 51.3 Mb (see Figure 8-34c, topological domain A) are much more likely to be ligated to each other than to sequences in the interval from 51.3 Mb to 52.2 Mb (topological domain B), or to sequences from any of the other topological domains that are apparent. In situ hybridization studies showed that sequences within a topological domain lie much closer to each other in the fixed cell nucleus than to sequences the same distance away in base pairs, but in a neighboring topological domain. These results have been interpreted to indicate that the chromatin fiber is folded into topological domains, as represented in the inset of Figure 8-34c. The topological domains are separated by shorter regions of chromatin, called boundary elements, that do not interact with distant regions of chromatin. Since the topological domains are on the order of $200 \mathrm{~kb}-1.5 \mathrm{Mb}$ in length, they are long enough to contain several averagesized genes. The topological domains identified by these chromatin conformation capture assays may correspond to

the loops of chromatin observed in the lampbrush chromosomes described above, which are not constrained by the nuclear envelope of a vastly smaller nucleus and have an opportunity to unfold (see Figure 8-31). Current research is exploring what protein-DNA interactions might be responsible for establishing boundary elements between topological domains. As we will see in Chapter 9, related chromosome conformation capture techniques have provided strong evidence that proteins bound to enhancers interact with proteins bound to promoters many kilobases away.

Metaphase Chromosome Structure Condensation of chromosomes during prophase (see Figure 18-37) may involve the formation of many more loops of chromatin, so that the length of each loop is greatly reduced compared with chromatin loops in interphase cells. As a result, chromosomes condense into structures of much greater width than interphase chromosomes and decrease in length severalfold, generating the condensed chromosomes observed during metaphase (Figure 8-35).

The geometry of chromatin in metaphase chromosomes is not well understood. Experiments with frog egg extracts have shown that a protein complex called condensin, composed of SMC subunits (see Figure 8-32 and Chapter 19), contributes to chromosome condensation using energy from ATP hydrolysis. Microscopic analysis of mammalian chromosomes as they condense during prophase indicates that in the initial period of prophase, the $30-\mathrm{nm}$ chromatin fiber folds into a $100-130-\mathrm{nm}$ chromonema fiber associated with the nuclear envelope (Figure 8-36). Chromonema fibers then fold into structures with a diameter of 200-250 nm, called middle prophase chromatids (Figure 8-36a, ■), which then fold into the $500-750-\mathrm{nm}$-diameter chromatids observed during metaphase when the nuclear envelope retracts into the endoplasmic reticulum (Figure 8-36a, $\mathbf{\square}$ ) (see also Chapter 19). Ultimately, the full lengths of the two associated daughter chromosomes generated by DNA replication during the previous S phase of the cell cycle (see Figure 1-21) condense into bar-shaped structures (chromatids) that in most eukaryotes are linked at the central constriction called the centromere (see Figure 8-35). An electron micrograph of a section through a metaphase chromosome stained with anti-SMC antibodies linked to small gold spheres (Figure 8-36b) shows that condensin, proposed to be at the bases of chromatin loops (see Figure 8-32c), occupies approximately one-third of the chromatid diameter (Figure 8-36c, right), where it contributes to the shaping of each chromatid.

## Additional Nonhistone Proteins Regulate Transcription and Replication

As we have seen, the total mass of the histones associated with DNA in chromatin is about equal to that of the DNA. Interphase chromatin and metaphase chromosomes also contain small amounts of a complex set of other proteins. For instance, thousands of different transcription factors are associated with interphase chromatin. The structure and
![li2022239539b](li2022239539b.jpg)

FIGURE 8-35 Typical metaphase chromosome. As seen in this scanning electron micrograph, the chromosome has replicated and comprises two chromatids, each containing one of two identical DNA molecules. The centromere, where the chromatids are attached at a constriction, is required for their separation late in mitosis. Special telomere sequences at the ends function in preventing chromosome shortening. [Andrew Syred/Science Source]
function of these critical nonhistone proteins, which regulate transcription, are examined in Chapter 9. Other lowabundance nonhistone proteins associated with chromatin regulate DNA replication during the eukaryotic cell cycle (see Chapter 19).

A few other nonhistone DNA-binding proteins are present in much larger amounts than the transcription or replication factors. Some of these proteins exhibit high mobility during electrophoretic separation and thus have been designated HMG (high-mobility group) proteins. When genes encoding the most abundant HMG proteins are deleted from yeast cells, normal transcription is disturbed in most genes examined. Some HMG proteins have been found to assist in the cooperative binding of several transcription factors to specific DNA sequences that are close to each other, stabilizing multiprotein complexes that regulate transcription of a neighboring gene, as discussed in Chapter 9.

![li2032239539b](li2032239539b.jpg)

FIGURE 8-36 Model for mitotic chromosome condensation. (a) Stages of chromosome condensation during mitosis. Changes in large-scale chromatin folding (blue) versus distribution of Smc2, a subunit of condensin (red), from early prophase $\mathbf{\square}$ to middle prophase $\mathbf{\square}$ to late prophase $\mathbf{\square}$ to metaphase $\mathbf{\square}$
(b) Transmission electron micrograph of immunogold staining of Smc2 in a section through a metaphase chromosome reveals axial staining of Smc2 of about $0.15-0.2 \mu \mathrm{~m}$ in width. (c) "Hierarchical folding, axial glue" model of metaphase chromosome structure. (Left) 30-nm fiber folds into 100-130-nm chromonema fiber, which folds into 200-250-nm middle prophase chromatid, which folds into 500-750-nm metaphase chromatid. Only one chromatid is shown. (Right) Axial condensin distribution (red) occupies approximately one-third of the chromatid diameter, acting as a cross-linking "glue" to stabilize the structure of the metaphase chromosome. [Part (b) © 2004 Kireeva et al., The Journal of Cell Biology, 166:775-785. doi: 10.1083/ jcb.200406049.]

## KEY CONCEPTS OF SECTION 8.5

## Structural Organization of Eukaryotic Chromosomes

- In eukaryotic cells, DNA is associated with about an equal mass of histone proteins in a highly condensed nucleoprotein complex called chromatin. The building block of chromatin is the nucleosome, consisting of a histone octamer around which is wrapped about 147 bp of DNA (see Figure 8-24).
- The chromatin in transcriptionally inactive regions of DNA within cells is thought to exist in a condensed, $30-\mathrm{nm}$ fiber form and higher-order structures built from it (see Figure 8-25 and 8-36).
- The chromatin in transcriptionally active regions of DNA within cells is thought to exist in an open, extended form.
- The flexible, intrinsically disordered N-terminal tails of histones, particularly H4 lysine 16, are required for
beads-on-a-string chromatin (the 10-nm chromatin fiber) to fold into a $30-\mathrm{nm}$ fiber.
- Histone tails can be modified by acetylation, methylation, phosphorylation, and ubiquitinylation (see Figure 8-26). These modifications influence chromatin structure by regulating the binding of histone tails to other, less abundant chromatin-associated proteins.
- The reversible acetylation and deacetylation of lysine residues in the N -terminal tails of the core histones regulate chromatin condensation. Proteins involved in transcription, replication, and repair, and enzymes such as DNase I, can more easily access chromatin with hyperacetylated histone tails (euchromatin) than chromatin with hypoacetylated histone tails (heterochromatin).
- When metaphase chromosomes decondense during interphase, areas of heterochromatin remain much more condensed than regions of euchromatin.

- Heterochromatin protein 1 (HP1) uses a chromodomain to bind to histone H 3 trimethylated at lysine 9 . The chromoshadow domain of HP1 associates with itself and with the histone methyl transferase that methylates H3 lysine 9. These interactions cause condensation of the $30-\mathrm{nm}$ chromatin fiber and spreading of the heterochromatic structure along the chromosome until a boundary element is encountered (see Figure 8-29).
- One X chromosome in nearly every cell of mammalian females consists of highly condensed heterochromatin, resulting in repression of expression of nearly all genes on that inactive chromosome. This inactivation results in dosage compensation so that genes on the X chromosome are expressed at the same level in both males and females.
- Each eukaryotic chromosome contains a single DNA molecule packaged into nucleosomes and folded into a $30-\mathrm{nm}$ chromatin fiber, which is associated with structural maintenance of chromosome (SMC) proteins thought to organize it into the megabase loops observed by hybridization to fluorescently labeled DNA probes and in lampbrush chromosomes observed in oocytes (see Figures 8-30, 8-31, and $8-32 \mathrm{c}$ ). Additional folding of the chromosomes further compacts the structure into the highly condensed form of metaphase chromosomes (see Figure 8-36).
- In interphase cells, chromosomes are localized to largely non-overlapping "territories" in the nucleus (see Figure 8-33).
- Chromosome conformation capture methods indicate that chromatin is organized into topological domains separated by boundary elements (see Figure 8-34c). These topological domains may correspond to the loops in lampbrush chromosomes observed in the giant nuclei of oocytes (see Figure 8-31) and inferred by studies of fluorescently labeled DNA probes hybridized to interphase nuclei (see Figure 8-30).
- During mitosis, chromosomes condense greatly, decreasing their lengths severalfold and increasing their diameter to generate metaphase chromosomes visible by light microscopy. The geometry of the $30-\mathrm{nm}$ chromatin fiber in metaphase chromosomes is not well understood, but intermediates of increasing diameter and decreasing length have been observed during prophase.


### 8.6 Morphology and Functional Elements of Eukaryotic Chromosomes

Having examined the detailed structural organization of chromosomes in the previous section, we now view them from a more global perspective. Early microscopic observations on the number and size of chromosomes and their staining patterns led to the discovery of many important general characteristics of chromosome structure. Researchers subsequently identified specific regions of chromosomes that are critical to their replication and segregation to
daughter cells during cell division. In this section, we discuss these functional elements of chromosomes and consider how chromosomes evolved through rare rearrangements of ancestral chromosomes.

## Chromosome Number, Size, and Shape at Metaphase Are Species-Specific

In interphase cells, as noted previously, chromosome territories can be visualized with chromosome-specific fluorescently labeled hybridization probes (see Figure 8-33), but the detailed structure of individual chromosomes cannot be observed, even with the aid of electron microscopy. During mitosis and meiosis, however, the chromosomes condense and become visible in the light microscope. Therefore, almost all cytogenetic work (i.e., studies of chromosome morphology) has been done with condensed metaphase chromosomes obtained from dividing cells-either somatic cells in mitosis or dividing gametes during meiosis.

The condensation of metaphase chromosomes probably results from several orders of folding of $30-\mathrm{nm}$ chromatin fibers (see Figure 8-36). At the time of mitosis, cells have already progressed through the $S$ phase of the cell cycle and have replicated their DNA. Consequently, the chromosomes that become visible during metaphase are duplicated structures. Each metaphase chromosome consists of two sister chromatids, which are linked at a constricted region, the centromere (see Figure 8-35).

The number, sizes, and shapes of the metaphase chromosomes constitute the karyotype, which is distinctive for each species. In most organisms, all somatic cells have the same karyotype. However, species that appear quite similar can have very different karyotypes, indicating that similar genetic potential can be organized on chromosomes in very different ways. For example, two species of small deer-the Indian muntjac and Reeves muntjac-contain about the same total amount of genomic DNA. In one species, however, this DNA is organized into 22 pairs of homologous autosomes and two physically separate sex chromosomes. In contrast, the other species contains the smallest number of chromosomes of any mammal, only three pairs of autosomes; one sex chromosome is physically separate, but the other is joined to the end of one autosome.

## During Metaphase, Chromosomes Can Be Distinguished by Banding Patterns and Chromosome Painting

Certain dyes selectively stain some regions of metaphase chromosomes more intensely than other regions, producing characteristic banding patterns that are specific for individual chromosomes. The regularity of chromosome bands provides useful visible landmarks along the length of each chromosome and can help to distinguish chromosomes of similar size and shape, as we will see later in this section.

Today the method of chromosome painting greatly simplifies the identification and differentiation of individual

![li2042239539b](li2042239539b.jpg)

EXPERIMENTAL FIGURE 8-37 Human chromosomes are readily identified by chromosome painting. (a) Image of human chromosomes from a male cell in mitosis made by fluorescence in situ
hybridization (FISH) using chromosome paint probes. (b) Alignment of these painted chromosomes by computer graphics to reveal the normal human male karyotype. [Courtesy of Dr. Michael R. Speicher.]
chromosomes within a karyotype, many of which have similar sizes and shapes. This technique, a variation of fluorescence in situ hybridization (FISH), makes use of probes specific for sites scattered along the length of each chromosome. The probes are labeled with several different fluorescent dyes with distinct excitation and emission wavelengths. Probes specific for each chromosome are labeled with a predetermined fraction of each of the dyes. After the probes are hybridized to chromosomes and the excess removed, the sample is observed with a fluorescence microscope in which a detector determines the fraction of each dye present at each fluorescing position in the microscopic field. This information is conveyed to a computer, and a special program assigns a false-color image to each type of chromosome (Figure 8-37a). Computer graphics allows the two homologs of each chromosome to be placed next to each other and numbered according to their decreasing size. Such an image clearly displays the cell's karyotype (Figure 8-37b).

Chromosome painting is a powerful method for detecting an abnormal number of chromosomes, such as chromosome 21 trisomy in patients with Down syndrome, or chromosomal translocations that occur in rare individuals and in cancer cells (Figure 8-38). The use of probes with different ratios of fluorescent dyes that hybridize to distinct positions along each normal human chromosome allows finer structural analysis of the chromosomes that can more readily reveal deletions or duplications of chromosomal regions. The chapter-opening figure illustrates the use of such multicolor FISH in analysis of the karyotype of a normal human female.

## Chromosome Painting and DNA Sequencing Reveal the Evolution of Chromosomes

Analysis of chromosomes from different species has provided considerable insight into how chromosomes evolved. For example, hybridization of chromosome paint probes for chromosome 16 of the tree shrew (Tupaia belangeri) to tree shrew metaphase chromosomes revealed the two copies of chromosome 16, as expected (Figure 8-39a). However, when the same chromosome paint probes were hybridized to human metaphase chromosomes, most of the probes hybridized to the long arm of chromosome 10 (Figure 8-39b). Further, when multiple probes for the long arm of human chromosome 10 with different fluorescent dye labels were hybridized to tree shrew metaphase chromosomes, these probes bound to sequences along tree shrew chromosome 16 in the same order in which they bind to human chromosome 10.

These results indicate that during the evolution of humans and tree shrews from a common ancestor that lived as recently as 85 million years ago, a long, continuous DNA sequence on one of the ancestral chromosomes became chromosome 16 in tree shrews, but evolved into the long arm of chromosome 10 in humans. The phenomenon of genes occurring in the same order on a chromosome in two different species is referred to as conserved synteny (derived from Latin for "on the same ribbon"). The presence of two or more genes in a common chromosomal region in two or more species indicates a conserved syntenic segment.

The relationships between the chromosomes of many primates have been determined by cross-species application of chromosome paint probes, as shown for human and tree

![li2052239539b](li2052239539b.jpg)

EXPERIMENTAL FIGURE 8-38 Chromosomal translocations can be analyzed using chromosome painting. Characteristic chromosomal translocations are associated with certain genetic disorders and specific types of cancers. For example, in nearly all patients with chronic myelogenous leukemia, the leukemic cells contain the Philadelphia chromosome, a shortened chromosome 22 [der (22)], and an
![li2062239539b](li2062239539b.jpg)
abnormally long chromosome 9 [der (9)] ("der" stands for derivative). These forms result from a translocation between normal chromosomes 9 and 22. This translocation can be detected (a) by classical banding analysis or (b) by chromosome painting. [Part (b) courtesy of J. Rowley and R. Espinosa.]
shrew in Figure 8-39a, b. Using these relationships, as well as higher-resolution analyses of regions of synteny by DNA sequencing and other methods, it has been possible to propose the karyotype of the common ancestor of all primates based on the minimum number of chromosomal rearrangements necessary to generate the regions of synteny in chromosomes of contemporary primates.

Human chromosomes are thought to have been derived from a common primate ancestor with 23 autosomes plus the X and Y sex chromosomes by several different mechanisms (Figure 8-39c). Some human chromosomes were derived without large-scale rearrangements of chromosome structure. Others are thought to have evolved by breakage of an ancestral chromosome into two chromosomes or, conversely, by fusion of two ancestral chromosomes. Still other human chromosomes appear to have been generated by exchanges of parts of the arms of distinct chromosomes; that is, by reciprocal translocation involving two ancestral chromosomes. Analysis of regions of conserved synteny between the chromosomes of many mammals indicates that chromosomal rearrangements by breakage, fusion, and translocations occurred rarely in mammalian evolution, about once every 5 million years. When such chromosomal rearrangements did occur, they very likely contributed to the evolution of new species that could not interbreed with the species from which they evolved.

Chromosomal rearrangements similar to those inferred for the primate lineage have been inferred for other groups of related organisms, including the invertebrate, plant, and fungus lineages. The excellent agreement between predictions of evolutionary relationships based on analysis of
syntenic regions of chromosomes from organisms with related morphology (i.e., among mammals, among insects with similar body organization, among similar plants, etc.) and evolutionary relationships based on the fossil record and on the extent of divergence of DNA sequences for homologous genes is a strong argument for the validity of evolution as the process that generated the diversity of contemporary organisms.

## Interphase Polytene Chromosomes Arise by DNA Amplification

The larval salivary glands of Drosophila species and other dipteran insects contain enlarged interphase chromosomes that are visible in the light microscope. When fixed and stained with a dye that stains DNA, these polytene chromosomes are characterized by a large number of reproducible, well-demarcated bands, which have been assigned standardized numbers (Figure 8-40a). The densely staining bands represent regions where the chromatin is more condensed, and the light interband areas are regions where the chromatin is less condensed. Although the molecular mechanisms that control the formation of bands in polytene chromosomes are not yet understood, the highly reproducible banding pattern seen in Drosophila salivary gland chromosomes provides an extremely powerful method for locating specific DNA sequences along the chromosomes of this species. Not only are chromosomal translocations and inversions readily detectable in polytene chromosomes, but specific chromosomal proteins can be localized on interphase polytene chromosomes by immunostaining with specific antibodies raised

![li2072239539b](li2072239539b.jpg)
![li2082239539b](li2082239539b.jpg)

FIGURE 8-39 Evolution of primate chromosomes. (a) Chromosome paint probes (yellow) for chromosome 16 of the tree shrew (T. belangeri, distantly related to humans) hybridized to tree shrew metaphase chromosomes (red). (b) The same tree shrew chromosome 16 paint probes hybridized to human metaphase chromosomes. (c) Proposed evolution of human chromosomes (bottom) from the chromosomes of the common ancestor of all primates (top). The proposed common primate ancestor chromosomes are numbered according to their sizes, with each chromosome represented by a different color. The human chromosomes are also numbered according to their relative sizes and labeled with colors taken from the colors of the proposed common primate ancestor chromosomes from which they were derived. Small numbers to the left of the colored regions of the human chromosomes indicate the number of the ancestral chromosome from which the region was derived. Various human chromosomes were derived from the proposed chromosomes of the common primate ancestor without significant rearrangements (e.g., human chromosome 1); by fusion (e.g., human chromosome 2 by fusion of ancestral chromosomes 9 and 11); by breakage (e.g., human chromosomes 14 and 15 by breakage of ancestral chromosome 5); or by chromosomal translocations (e.g., a reciprocal translocation between ancestral chromosomes 14 and 21 generated human chromosomes 12 and 22). (Parts (a) and (b) republished with permission of Springer, from Muller, S., et al., "Defining the ancestral karyotype of all primates by multidirectional chromosome painting between tree shrews, lemurs and humans," Chromosoma, 1999, 108(6):393-400; permission conveyed through Copyright Clearance Center. Part (c) data from L. Froenicke, 2005, Cytogenet. Genome Res. 108:122.)

Drosophila. This process, termed polytenization, occurs when the DNA repeatedly replicates everywhere except at the telomeres and centromere, but the daughter chromosomes do not separate. The result is an enlarged chromosome composed of many parallel copies of itself, 1024 resulting from ten such replications in Drosophila melanogaster salivary

![li2092239539b](li2092239539b.jpg)
(b)
![li2102239539b](li2102239539b.jpg)

EXPERIMENTAL FIGURE 8-40 Banding on Drosophila polytene salivary gland chromosomes. (a) In this light micrograph of Drosophila melanogaster larval salivary gland chromosomes, four chromosomes can be observed ( $X, 2,3$, and 4), with a total of approximately 5000 distinguishable bands. The banding pattern results from reproducible patterns of DNA and protein packing within each site along the chromosome. Dark bands are regions of more highly compacted chromatin. The centromeres of all four chromosomes often appear fused at the chromocenter. The tips of chromosomes 2 and 3 are labeled ( $L=$ left arm; $R=$ right arm), as is the tip of the $X$ chromosome. (b) The pattern of amplification of chromosome 4 during five replications. Double-stranded DNA is represented by a single line. Telomere and centromere DNA are not amplified. In salivary gland polytene chromosomes, each parental chromosome undergoes about 10 replications ( $2^{10}=1024$ strands). See C. D. Laird et al., 1973, Cold Spring Harbor Symp. Quant. Biol. 38:311. [Part (a) courtesy of Joseph Gall, Carnegie Institution for Science.]
glands (Figure 8-40b). The amplification of chromosomal DNA greatly increases gene copy number, presumably to supply sufficient mRNA for protein synthesis in the massive salivary gland cells. The bands in Drosophila polytene chromosomes each represent some 50,000-100,000 bp, and the banding pattern reveals that the condensation of DNA varies greatly along these relatively short regions of an interphase chromosome.

## Three Functional Elements Are Required for Replication and Stable Inheritance of Chromosomes

Although eukaryotic chromosomes differ in length and number among species, cytogenetic studies have shown that they all behave similarly at the time of cell division. Moreover, any eukaryotic chromosome must contain three functional elements in order to replicate and segregate correctly: (1) replication origins at which DNA polymerases and other proteins initiate synthesis of DNA (see Figures 5-31 and 5-33); (2) the centromere, the constricted region required for proper segregation of daughter chromosomes; and (3) the two ends, or telomeres. The yeast transformation studies depicted in Figure 8-41 demonstrated the functions of these three chromosomal elements and established their importance for chromosome function.

As discussed in Chapter 5, replication of DNA begins from sites that are scattered throughout eukaryotic chromosomes. The yeast genome contains many 100-bp sequences, called autonomously replicating sequences (ARSs), that act as replication origins. The observation that insertion of an ARS into a circular plasmid allows the plasmid to replicate in yeast cells provided the first functional identification of replication origins in eukaryotic DNA (Figure 8-41a).

Even though circular ARS-containing plasmids can replicate in yeast cells, only about 5-20 percent of progeny cells contain the plasmid because mitotic segregation of the plasmids is faulty. However, plasmids that also carry a CEN sequence, derived from the centromeres of yeast chromosomes, segregate equally, or nearly so, to both mother and daughter cells during mitosis (Figure 8-41b).

If circular plasmids containing an ARS and a CEN sequence are cut once with a restriction enzyme, the resulting linear plasmids do not transform yeast cells generating LEU* colonies that grow on medium lacking leucine unless they contain special telomeric (TEL) sequences ligated to their ends (Figure 8-41c). The first successful experiments involving transfection of yeast cells with linear plasmids were achieved by using the ends of a DNA molecule that was known to replicate as a linear molecule in the ciliated protozoan Tetrahymena. During part of the life cycle of Tetrahymena, much of the nuclear DNA is repeatedly copied in short pieces to form a so-called macronucleus. One of these repeated fragments was identified as a dimer of ribosomal DNA, the ends of which contained a repeated sequence $\left(\mathrm{G}_{4} \mathrm{~T}_{2}\right)_{\mathrm{n}}$. When a section of this repeated TEL sequence was ligated to the ends of linear yeast plasmids containing ARS and CEN, replication and good segregation of the linear plasmids occurred. This first cloning and characterization of telomeres garnered the Nobel Prize in Physiology or Medicine in 2009.

## Centromere Sequences Vary Greatly in Length and Complexity

Once the yeast centromere regions that confer mitotic segregation were cloned, their sequences could be determined and

![li2112239539b](li2112239539b.jpg)

EXPERIMENTAL FIGURE 8-41 Yeast transformation experiments were used to identify the functional chromosomal elements necessary for normal chromosome replication and segregation. In these experiments, plasmids containing the LEU gene from normal yeast cells are constructed and introduced into leu- cells by transfection. If the plasmid is maintained in the leu- cells, they are transformed to $L E U^{+}$cells by the $L E U$ gene on the plasmid and can form colonies on medium lacking leucine. (a) Sequences that allow autonomous replication (ARS) of a plasmid were identified because their insertion into a plasmid vector containing a cloned LEU gene resulted in a high frequency of transformation to $L E U^{+}$. However, even plasmids with ARS exhibit poor segregation during mitosis and therefore do not appear in each of the daughter cells. (b) When randomly broken pieces of yeast DNA are inserted into plasmids containing ARS and LEU, some of the subsequently transfected cells produce large colonies, indicating that a high rate of mitotic segregation among their plasmids is facilitating the continuous growth of daughter cells. The DNA recovered from plasmids in these large colonies contains yeast centromere (CEN) sequences. (c) When leu- yeast cells are transfected with linearized plasmids containing LEU, ARS, and CEN, no colonies grow. Addition of telomere (TEL) sequences to the ends of the linear DNA gives the linearized plasmids the ability to replicate as new chromosomes that behave very much like a normal chromosome in both mitosis and meiosis. See A. W. Murray and J. W. Szostak, 1983, Nature 305:89, and L. Clarke and J. Carbon, 1985, Ann. Rev. Genet. 19:29.
compared. The results revealed three regions (I, II, and III) that are conserved among the centromeres on different yeast chromosomes (Figure 8-42a). Short, fairly well-conserved nucleotide sequences are present in regions I and III. Region II does not have a specific sequence, but is AT-rich with a fairly constant length, probably so that regions I and III will lie on the same side of a specialized centromere-associated histone octamer. This specialized centromere-associated histone octamer contains the usual histones H2A, H2B, and H4, but a variant form of histone H3. Centromeres from all eukaryotes similarly contain nucleosomes with a specialized, centromere-specific form of histone H3, called CENP-A in humans. In the simple kinetochore of $S$. cerevisiae, a protein
complex called CBF3 associates with this specialized nucleosome. The CBF3 complex, in turn, associates with several copies of an elongated multiprotein complex called Ndc80 (Figure 8-42b). The Ndc80 complexes initially make lateral interactions with a spindle microtubule and subsequently interact with a Dam1 complex, which forms a ring around the end of the microtubule (Figure 8-42c). This interaction results in an end-on attachment of the centromere to the spindle microtubule. S. cerevisiae has by far the simplest centromere known in nature.

In the fission yeast $S$. pombe, centromeres are $40-100 \mathrm{~kb}$ in length and are composed of repeated copies of sequences similar to those in S. cerevisiae centromeres. Multiple

(a)
![li2122239539b](li2122239539b.jpg)
(b)
![li2132239539b](li2132239539b.jpg)

FIGURE 8-42 Kinetochore-microtubule interaction in S. cerevisiae. (a) Sequence of the simple centromeres of S. cerevisiae. See L. Clarke and J. Carbon, 1985, Ann. Rev. Genet. 19:29. (b) Ndc80 complexes associate with both the microtubule and the CBF3 complex. (c) Diagram of the centromere-associated CBF3 complex and its associated Ndc80 complexes, which associate with a ring of Dam1 proteins at the end of a spindle microtubule. The Ndc80 complexes initially make lateral interactions with the side of a spindle microtubule (top) and then associate with the Dam1 ring, making an end-on attachment (bottom) to the microtubule. See T. U. Tanaka, 2010, EMBO J. 29:4070.
copies of proteins homologous to those that interact with S. cerevisiae centromeres bind to these complex S. pombe centromeres, and in turn bind the much longer S. pombe chromosomes to several microtubules of the mitotic spindle apparatus. In plants and animals, centromeres are megabases in length and are composed of multiple repeats of simple-sequence DNA. In humans, centromeres contain $2-4-\mathrm{Mb}$ arrays of a $171-\mathrm{bp}$ simple-sequence DNA called alphoid DNA, which is bound by nucleosomes containing the CENP-A histone H3 variant, as well as other repeated simple-sequence DNAs.

In higher eukaryotes, a complex protein structure called the kinetochore assembles at centromeres and associates with multiple mitotic spindle fibers during mitosis (see Figure 18-40). Homologs of many of the centromereassociated proteins found in the yeasts occur in humans and other higher eukaryotes. For those yeast proteins for which clear homologs are not evident in higher cells based on amino acid sequence comparisons (such as the Dam1 complex), alternative complexes with similar properties have been proposed to function at kinetochores. The functions of the centromere and of the kinetochore proteins that bind to it during the segregation of sister chromatids in mitosis and meiosis are described in Chapters 18 and 19.

## Addition of Telomeric Sequences by Telomerase Prevents Shortening of Chromosomes

Sequencing of telomeres from multiple organisms, including humans, has shown that most are repetitive oligomers with a high G content located in the strand with its $3^{\prime}$ end at the end of the chromosome. The telomere repeat sequence in humans and other vertebrates is TTAGGG. These simple sequences are repeated at the very termini of chromosomes for a total of a few hundred base pairs in yeasts and protozoans and a few thousand base pairs in vertebrates. The $3^{\prime}$ end of the G-rich strand extends 12-16 nucleotides beyond the $5^{\prime}$ end of the complementary C-rich strand. This region is bound by specific proteins that protect the ends of linear chromosomes from attack by exonucleases.

The need for a specialized region at the ends of eukaryotic chromosomes is apparent when we consider that all known DNA polymerases elongate DNA chains at the $3^{\prime}$ end, and all require an RNA or DNA primer. As the replication fork approaches the end of a linear chromosome, synthesis of the leading strand continues to the end of the DNA template strand, completing one daughter DNA double helix. However, because the lagging-strand template is copied in a discontinuous fashion, it cannot be replicated in its entirety (Figure 8-43). When the final RNA primer is removed, there is no upstream strand onto which DNA polymerase can build to fill the resulting gap. Without some special mechanism, the daughter DNA strand resulting from lagging-strand synthesis would be shortened at each cell division.

The problem of telomere shortening is solved by an enzyme that adds telomeric repeat sequences to the ends of each chromosome. The enzyme is a protein-RNA complex called telomere terminal transferase, or telomerase. Because the sequence of the telomerase-associated RNA, as we will see, serves as the template for addition of deoxyribonucleotides to the ends of telomeres, the source of the enzyme, and not the source of the telomeric DNA primer, determines the sequence added. This was proved by transforming Tetrahymena with a mutated form of the gene encoding

![li2142239539b](li2142239539b.jpg)

FIGURE 8-43 Standard DNA replication leads to loss of DNA at the $5^{\prime}$ end of each strand of a linear DNA molecule. Replication of the right end of a linear DNA is shown; the same process occurs at the left end (as can be shown by inverting the figure). As the replication fork approaches the end of the parental DNA molecule, the leading strand can be synthesized all the way to the end of the template strand without the loss of deoxyribonucleotides. However, since synthesis of the lagging strand requires RNA primers, the right end of the lagging daughter DNA strand would remain as ribonucleotides, which are removed and therefore cannot serve as the template for a replicative DNA polymerase. Alternative mechanisms must be used to prevent successive shortening of the lagging strand with each round of replication.
the telomerase-associated RNA. The resulting telomerase added a DNA sequence complementary to the mutated RNA sequence to the ends of telomeric primers. Thus telomerase is a specialized form of a reverse transcriptase that carries its own internal RNA template to direct DNA synthesis. These experiments also earned the Nobel Prize in Physiology or Medicine for the structure and function of telomeres in 2009.

Figure 8-44 depicts how telomerase, by reverse transcription of its associated RNA, elongates the $3^{\prime}$ end of the singlestranded DNA at the end of the G-rich strand mentioned above. Cells from knockout mice that cannot produce the telomerase-associated RNA exhibit no telomerase activity, and their telomeres shorten successively with each cell generation. Such mice can breed and reproduce normally for three generations before the long telomere repeats become substantially eroded. Then, the absence of telomere DNA
![li2152239539b](li2152239539b.jpg)

FIGURE 8-44 Mechanism of action of telomerase. The singlestranded $3^{\prime}$ terminus of a telomere is extended by telomerase, counteracting the inability of the DNA replication mechanism to synthesize the extreme terminus of linear DNA. Telomerase elongates this singlestranded end by a reiterative reverse-transcription mechanism. The action of the telomerase from the protozoan Tetrahymena, which adds a $\mathrm{T}_{2} \mathrm{G}_{4}$ repeat unit, is depicted here; other telomerases add slightly different sequences. The telomerase contains an RNA template (red) that base-pairs to the $3^{\prime}$ end of the lagging-strand template. The telomerase catalytic site then adds deoxyribonucleotides TTG (blue), using the RNA molecule as a template (step $\mathbf{\square}$ ). The strands of the resulting DNA-RNA duplex are then thought to slip (translocate) relative to each other so that the TTG sequence at the $3^{\prime}$ end of the replicating DNA base-pairs to the complementary RNA sequence in the telomerase RNA (step $\mathbf{\square}$ ). The $3^{\prime}$ end of the replicating DNA is then again extended by telomerase (step $\mathbf{\square}$. Telomerases can add multiple repeats by repetition of steps $\mathbf{\square}$ and $\mathbf{\square}$. DNA polymerase $\alpha$-primase can prime synthesis of new Okazaki fragments on this extended template strand. The net result prevents shortening of the lagging strand at each cycle of DNA replication. See C. W. Greider and E. H. Blackburn, 1989, Nature 337:331.
results in adverse effects, including fusion of chromosome termini and chromosome loss. By the fourth generation, the reproductive potential of these knockout mice declines, and they cannot produce offspring after the sixth generation.

The human genes expressing the telomerase protein and the telomerase-associated RNA are active in germ cells and stem cells, but are turned off in most cells of adult

tissues that replicate only a limited number of times, or will never replicate again (such cells are called postmitotic). However, these genes are activated in most human cancer cells, where telomerase is required for the multiple cell divisions necessary to form a tumor. This phenomenon has stimulated a search for inhibitors of human telomerase as potential therapeutic agents for treating cancer.

While telomerase prevents telomere shortening in most eukaryotes, some organisms use alternative strategies. Drosophila species maintain telomere lengths by the regulated insertion of non-LTR retrotransposons into telomeres. This is one of the few instances in which a mobile element has a specific function in its host organism.

## KEY CONCEPTS OF SECTION 8.6

## Morphology and Functional Elements of Eukaryotic Chromosomes

- During metaphase, eukaryotic chromosomes become sufficiently condensed that they can be visualized individually in the light microscope.
- The chromosomal karyotype is characteristic of each species. Closely related species can have dramatically different karyotypes, indicating that similar genetic information can be organized on chromosomes in different ways.
- Banding analysis and chromosome painting are used to identify the different human metaphase chromosomes and to detect translocations and deletions (see Figure 8-37 and 8-38).
- Analysis of chromosomal rearrangements and regions of conserved synteny between related species allows scientists to make predictions about the evolution of chromosomes (see Figure 8-39c). The evolutionary relationships between organisms indicated by these studies are consistent with proposed evolutionary relationships based on the fossil record and DNA sequence analysis.
- The highly reproducible banding patterns of polytene chromosomes make it possible to visualize chromosomal deletions and rearrangements as changes in the normal pattern of bands.
- Three types of DNA sequences are required for a long linear DNA molecule to function as a chromosome: a replication origin, called ARS in yeast; a centromere (CEN) sequence; and two telomere (TEL) sequences at the ends of the DNA (see Figure 8-41).
- Telomerase, a protein-RNA complex, has a special reverse transcriptase activity that completes replication of telomeres during DNA synthesis (see Figure 8-44). In the absence of telomerase, the daughter DNA strand resulting from lagging-strand synthesis would be shortened at each cell division in most eukaryotes (see Figure 8-43).


## LaunchPad

macmillan learning

Visit LaunchPad to access study tools and to learn more about the content in this chapter.

- Perspectives for the Future
- Analyze the Data
- Extended References
- Additional study tools, including videos, animations, and quizzes


## Key Terms

centromere 345
chromatid 341
chromatin 327
DNA transposon 313
euchromatin 332
exon shuffling 322
fluorescence in situ
hybridization (FISH)
gene family 306
genomics 302
heterochromatin 332
histones 327
karyotype 341
LINEs 318
long terminal repeats
(LTRs) 316
nucleosome 328
open reading frame
(ORF) 326
polytene chromosome 343
protein family 306
pseudogene 307
retrotransposon 313
simple-sequence (satellite)
DNA 310
SINEs 318
SMC proteins 336
telomere 345
transcription unit 303
transposable (mobile) DNA
element 312

# Review the Concepts 

1. Genes can be transcribed into mRNA, in the case of protein-coding genes, or into RNA, in the case of genes such as those that encode ribosomal or transfer RNAs. Define a gene. For the following characteristics, state whether they apply to (a) continuous, (b) simple, or (c) complex transcription units.
i. Found in eukaryotes
ii. Contain introns
iii. Capable of making only a single protein from a given gene
2. Sequencing of the human genome has revealed much about the organization of genes. Describe the differences between solitary genes, gene families, pseudogenes, and tandemly repeated genes.
3. Much of the human genome consists of repetitious DNA. Describe the difference between microsatellite and minisatellite DNA. How is this repetitious DNA useful for identifying individuals by the technique of DNA fingerprinting?

4. Mobile DNA elements that can move or transpose to a new site directly as DNA are called DNA transposons. Describe the mechanism by which a bacterial DNA transposon, called an insertion sequence, can transpose.
5. Retrotransposons are a class of mobile elements that transpose via an RNA intermediate. Contrast the mechanism of transposition between retrotransposons that contain long terminal repeats (LTRs) and those that lack LTRs.
6. Discuss the role that transposons may have played in the evolution of modern organisms. What is exon shuffling? What role do transposons play in the process of exon shuffling?
7. What are paralogous and orthologous genes? What are some of the explanations for the finding that humans are a much more complex organism than the roundworm C. elegans, yet have only about 5 percent more proteincoding genes ( 21,000 versus 20,000$)$ ?
8. The DNA in a cell associates with proteins to form chromatin. What is a nucleosome? What role do histones play in nucleosomes? How are nucleosomes arranged in condensed 30-nm fibers?
9. How do chromatin modifications regulate transcription? What modifications are observed in regions of the genome that are being actively transcribed? In regions that are not actively transcribed?
10. What is FISH? Briefly describe how it works. How is FISH used to characterize chromosomal translocations associated with certain genetic disorders and specific types of cancers?
11. What is chromosome painting, and how is this technique useful? How can chromosome paint probes be used to analyze the evolution of mammalian chromosomes?
12. Certain organisms contain cells that possess polytene chromosomes. What are polytene chromosomes, where are they found, and what function do they serve?
13. Replication and segregation of eukaryotic chromosomes require three functional elements: replication origins, a centromere, and telomeres. How would a chromosome be affected if it lacked (a) replication origins or (b) a centromere?
14. Describe the problem that occurs during DNA replication at the ends of chromosomes. How are telomeres related to this problem?

## References

## Eukaryotic Gene Structure

Black, D. L. 2003. Mechanisms of alternative pre-messenger RNA splicing. Ann. Rev. Biochem. 72:291-336.

Davuluri, R. V., et al. 2008. The functional consequences of alternative promoter use in mammalian genomes. Trends Genet. 24:167-177.

Wang, E. T., et al. 2008. Alternative isoform regulation in human tissue transcriptomes. Nature 456:470-476.

Chromosomal Organization of Genes and Noncoding DNA
Celniker, S. E., and G. M. Rubin. 2003. The Drosophila melanogaster genome. Ann. Rev. Genomics Hum. Genet. 4:89-117.

Crook, Z. R., and D. Housman. 2011. Huntington's disease: can mice lead the way to treatment? Neuron 69:423-435.

Feuillet, C., et al. 2011. Crop genome sequencing: lessons and rationales. Trends Plant Sci. 16:77-88.

Giardina, E., A. Spinella, and G. Novelli. 2011. Past, present and future of forensic DNA typing. Nanomedicine (Lond.) 6:257-270.

Hannan, A. J. 2010. TRPing up the genome: tandem repeat polymorphisms as dynamic sources of genetic variability in health and disease. Discov. Med. 10:314-321.

International Human Genome Sequencing Consortium. 2004. Finishing the euchromatic sequence of the human genome. Nature 431:931-945.

Jobling, M. A., and P. Gill. 2004. Encoded evidence: DNA in forensic analysis. Nature Rev. Genet. 5:739-751.

Lander, E. S., et al. 2001. Initial sequencing and analysis of the human genome. Nature 409:860-921.

Todd, P. K., and H. L. Paulson. 2010. RNA-mediated neurodegeneration in repeat expansion disorders. Ann. Neurol. 67:291-300.

Venter, J. C., et al. 2001. The sequence of the human genome. Science 291:1304-1351.

## Transposable (Mobile) DNA Elements

Curcio, M. J., and K. M. Derbyshire. 2003. The outs and ins of transposition: from mu to kangaroo. Nature Rev. Mol. Cell Biol. 4:865-877.

Goodier, J. L., and H. H. Kazazian, Jr. 2008. Retrotransposons revisited: the restraint and rehabilitation of parasites. Cell 135:23-35.

Jones, R. N. 2005. McClintock's controlling elements: the full story. Cytogenet. Genome Res. 109:90-103.

Lisch, D. 2009. Epigenetic regulation of transposable elements in plants. Ann. Rev. Plant Biol. 60:43-66.

Genomics: Genome-Wide Analysis of Gene Structure and Function
BLAST Information can be found at: http://blast.ncbi.nlm.nih. gov/Blast.cgi.

1000 Genomes Project Consortium. 2010. A map of human genome variation from population-scale sequencing. Nature 467:1061-1073.

Alkan, C., B. P. Coe, and E. E. Eichler. 2011. Genome structural variation discovery and genotyping. Nature Rev. Genet. 12: 363-376.

Chimpanzee Sequencing and Analysis Consortium. 2005. Initial sequence of the chimpanzee genome and comparison with the human genome. Nature 437:69-87.
du Plessis, L., N. Skunca, and C. Dessimoz. 2011. The what, where, how and why of gene ontology-a primer for bioinformaticians. Brief Bioinform. 12:723-735.

Ideker, T., J. Dutkowski, and L. Hood. 2011. Boosting signal-to-noise in complex biology: prior knowledge is power. Cell 144:860-863.

Lander, E. S. 2011. Initial impact of the sequencing of the human genome. Nature 470:187-197.

Mills, R. E., et al. 2011. Mapping copy number variation by population-scale genome sequencing. Nature 470:59-65.

Picardi, E., and G. Pesole. 2010. Computational methods for ab initio and comparative gene finding. Meth. Mol. Biol. 609:269-284.

Ramskold, D., et al. 2009. An abundance of ubiquitously expressed genes revealed by tissue transcriptome sequence data. PLoS Comput. Biol. 5:e1000598.

Raney, B. J., et al. 2011. ENCODE whole-genome data in the UCSC genome browser (2011 update). Nucl. Acids Res. 39: D871-D875.

Sleator, R. D. 2010. An overview of the current status of eukaryote gene prediction strategies. Gene 461:1-4.

Sonah, H., et al. 2011. Genomic resources in horticultural crops: status, utility and challenges. Biotechnol. Adv. 29:199-209.

Stratton, M. R. 2011. Exploring the genomes of cancer cells: progress and promise. Science 331:1553-1558.

Venter, J. C. 2011. Genome-sequencing anniversary. The human genome at 10: successes and challenges. Science 331:546-547.

## Structural Organization of Eukaryotic Chromosomes

Bannister, A. J., and T. Kouzarides. 2011. Regulation of chromatin by histone modifications. Cell Res. 21:381-395.

Bernstein, B. E., A. Meissner, and E. S. Lander. 2007. The mammalian epigenome. Cell 128:669-681.

Horn, P. J., and C. L. Peterson. 2006. Heterochromatin assembly: a new twist on an old model. Chromosome Res. 14:83-94.

Kurdistani, S. K. 2011. Histone modifications in cancer biology and prognosis. Prog. Drug Res. 67:91-106.

Luger, K. 2006. Dynamic nucleosomes. Chromosome Res. 14:5-16.

Luger, K., and T. J. Richmond. 1998. The histone tails of the nucleosome. Curr. Opin. Genet. Devel. 8:140-146.

Nasmyth, K., and C. H. Haering. 2005. The structure and function of SMC and kleisin complexes. Ann. Rev. Biochem. 74:595-648.

Schalch, T., et al. 2005. X-ray structure of a tetranucleosome and its implications for the chromatin fibre. Nature 436:138-141.

Woodcock, C. L., and R. P. Ghosh. 2010. Chromatin higherorder structure and dynamics. Cold Spring Harbor Perspect. Biol. 2:a000596.

## Morphology and Functional Elements of Eukaryotic Chromosomes

Armanios, M., and C. W. Greider. 2005. Telomerase and cancer stem cells. Cold Spring Harbor Symp. Quant. Biol.70:205-208.

Belmont, A. S. 2006. Mitotic chromosome structure and condensation. Curr. Opin. Cell Biol. 18:632-638.

Blackburn, E. H. 2005. Telomeres and telomerase: their mechanisms of action and the effects of altering their functions. FEBS Lett. 579:859-862.

Cvetic, C., and J. C. Walter. 2005. Eukaryotic origins of DNA replication: could you please be more specific? Semin. Cell Dev. Biol. 16:343-353.

Froenicke, L. 2005. Origins of primate chromosomes as delineated by Zoo-FISH and alignments of human and mouse draft genome sequences. Cytogenet. Genome Res. 108:122-138.

MacAlpine, D. M., and S. P. Bell. 2005. A genomic view of eukaryotic DNA replication. Chromosome Res. 13:309-326.

Ohta, S., et al. 2011. Building mitotic chromosomes. Curr. Opin. Cell Biol. 23:114-121.

Tanaka, T. U. 2010. Kinetochore-microtubule interactions: steps towards bi-orientation. EMBO J. 29:4070-4082.

this page left intentionally blank

![li2162239539b](li2162239539b.jpg)

Drosophila polytene chromosomes stained with antibodies against a chromatin-remodeling ATPase called Kismet (blue), RNA polymerase II with low CTD phosphorylation (red), and RNA polymerase II with high CTD phosphorylation (green). [Reproduced with permission of The Company of Biologists, from Srinivasan, S., et al., "The Drosophila trithorax group protein Kismet facilitates an early step in transcriptional elongation by RNA Polymerase II," Development, 2005, 132(7):1623-1635; permission conveyed through Copyright Clearance Center, Inc.]

In previous chapters, we have seen that the properties and functions of each cell type are determined by the proteins it contains. In this chapter and the next, we consider how the kinds and amounts of the various proteins produced by a particular cell type in a multicellular organism are regulated. This regulation of gene expression is the fundamental process that controls the development of multicellular organisms such as ourselves from a single fertilized egg cell into the thousands of cell types of which we are made. When gene expression goes awry, cellular properties are altered, a process that all too often leads to the development of cancer. As discussed further in Chapter 24, genes encoding proteins that restrain cell growth are abnormally repressed in cancer cells, whereas genes encoding proteins that promote cell growth and replication are inappropriately activated in cancer cells. Abnormalities in gene expression also result in developmental defects such as cleft palate, tetralogy of Fallot (a serious developmental defect of the heart that can be treated surgically), and many others. Regulation of gene expression

## OUTLINE

### 9.1 Control of Gene Expression in Bacteria

9.2 Overview of Eukaryotic Gene Control

### 9.3 RNA Polymerase II Promoters and General Transcription Factors

9.4 Regulatory Sequences in Protein-Coding Genes and the Proteins Through Which They Function

## Transcriptional Control of Gene Expression

also plays a vital role in bacteria and other single-celled microorganisms, in which it allows cells to adjust their enzymatic machinery and structural components in response to their changing nutritional and physical environment. Consequently, to understand how microorganisms respond to their environment and how multicellular organisms normally develop, as well as how pathological abnormalities of gene expression occur, it is essential to understand the molecular interactions that control protein production.

The basic steps in gene expression-that is, the entire process whereby the information encoded in a particular gene is decoded into a particular protein-are reviewed in Chapter 5. Synthesis of mRNA requires that an RNA polymerase initiate transcription (initiation), polymerize ribonucleoside triphosphates complementary to the DNA coding strand (elongation), and then terminate transcription (termination) (see Figure 5-11). In bacteria, ribosomes and translation initiation factors have immediate access to newly formed RNA transcripts, which function as mRNA without further modification.

### 9.5 Molecular Mechanisms of Transcription Repression and Activation

9.6 Regulation of Transcription-Factor Activity

### 9.7 Epigenetic Regulation of Transcription

9.8 Other Eukaryotic Transcription Systems

In eukaryotes, however, the initial RNA transcript is subjected to processing that yields a functional mRNA (see Figure 5-15). The mRNA then is transported from its site of synthesis in the nucleus to the cytoplasm, where it is translated into protein with the aid of ribosomes, rRNAs, and translation factors (see Figures 5-23, 5-24, and 5-26).

Regulation may occur at several of the various steps in gene expression outlined above: transcription initiation, elongation, RNA processing, and mRNA export from the nucleus, as well as through control of mRNA degradation, mRNA translation into protein, and protein degradation. This regulation results in differential protein expression in different cell types or developmental stages or in response to external conditions. Although examples of regulation at each step in gene expression have been found, control of transcription initiation and of elongationthe first two steps-are the most important mechanisms for determining whether most genes are expressed and how much of the encoded mRNAs and, consequently, proteins are produced (Figure 9-1). The molecular mechanisms that regulate transcription initiation and elongation are critical to numerous biological phenomena, including the development of a multicellular organism, as mentioned above, the immune responses that protect us from pathogenic microorganisms, and neurological processes such as learning and memory. When these regulatory mechanisms controlling transcription function improperly, pathological processes may occur. For example, dominant mutations of the HOXD13 gene result in polydactyly, the embryological development of extra digits of the feet, hands, or both (Figure 9-2a). HOXD13 encodes a transcription factor that normally regulates the transcription of multiple genes involved in development of the extremities. Other mutations affecting the function or expression of transcription factors cause an extra pair of wings to develop in Drosophila (Figure 9-2b),
![li2172239539b](li2172239539b.jpg)

FIGURE 9-1 Contributions of the major processes that regulate protein concentrations. The concentration of a protein is controlled by regulation of the frequency with which the mRNA encoding the protein is synthesized (gene transcription), the rate at which that mRNA is degraded, the rate at which that mRNA is translated into protein, and the rate at which that protein is degraded. The relative contributions of these four rates to determining the concentrations of thousands of proteins in cultured mouse fibroblasts were determined by mass spectrometry to measure protein concentrations (see Chapter 3), mRNA sequencing (RNA-seq) to measure mRNA levels (see Chapter 6), protection of mRNA from ribonuclease digestion by associated ribosomes (ribosome footprinting) to estimate translation rates, stable isotope labeling to determine degradation rates, and statistical analysis of the data to correct for inherent biases and errors in these methods. [Data from J. J. Li and M. D. Biggin, 2014, Science 347:1066.]
alter the structures of flowers in plants (Figure 9-2c), and are responsible for multiple other developmental abnormalities.

Transcription is a complex process involving many layers of regulation. In this chapter, we focus on the molecular events that determine when transcription of a gene occurs. First, we consider the mechanisms of gene expression in bacteria, in which DNA is not bound by histones and packaged into nucleosomes. Repressor and activator proteins recognize and bind to specific DNA sequences to control the transcription of a nearby gene, and in many cases, specific tertiary structures in nascent mRNAs, called riboswitches, bind metabolites to regulate transcription elongation. The remainder of the chapter focuses on eukaryotic regulation of transcription and how the basic tenets of bacterial regulation are applied in more complex ways in higher organisms. In addition, eukaryotic regulation mechanisms make use of the association of DNA with histone octamers, forming chromatin structures with varying degrees of condensation, and of post-translational modifications of histone tails such as acetylation and methylation (see Figure 8-26). Figure 9-3 provides an overview of transcriptional regulation in metazoans (multicellular animals) and of the processes outlined in this chapter. We discuss how the RNA polymerases responsible for the transcription of different classes of eukaryotic genes bind to promoter sequences to initiate the synthesis of an RNA molecule, and how specific DNA sequences function as transcription-control regions by serving as the binding sites for the transcription factors that regulate transcription. Next we consider how eukaryotic activators and repressors influence transcription through interactions with large multiprotein complexes. Some of these multiprotein complexes modify chromatin condensation, altering the accessibility of chromosomal DNA to transcription factors and RNA polymerases. Other complexes directly influence the frequency at which RNA polymerases bind to promoters and initiate transcription. Very recent research has revealed that, for many genes in multicellular animals, the RNA polymerase pauses after transcribing a short RNA, and that one transcriptional regulation mechanism involves a release of the paused polymerase, allowing it to transcribe the rest of the gene. We discuss how transcription of specific genes can be specified by particular combinations of the roughly 1400 transcription factors encoded in the human genome, giving rise to cell-type-specific gene expression. We consider the various ways in which the activities of transcription factors themselves are controlled to ensure that genes are expressed only in the correct cell types and at the appropriate time during their differentiation.

We also discuss recent studies revealing that RNA-protein complexes in the nucleus can regulate transcription. New methods for sequencing DNA, coupled with reverse transcription of RNA into DNA in vitro, have revealed that much of the genome of eukaryotes is transcribed into lowabundance RNAs that do not encode proteins. Several nuclear long noncoding RNAs (lncRNAs) have recently been discovered to regulate the transcription of other proteincoding genes. This finding raises the possibility that transcriptional control by such noncoding RNAs may be much more general than is currently understood. Recent advances in mapping the association of transcription factors with

specific regions of chromatin across the entire genome in a variety of cell types have provided the first glimpses of how transcription factors regulate embryonic development from the pluripotent stem cells of the early embryo to the fully differentiated cells that make up most of our tissues. RNA processing and various post-transcriptional mechanisms
![li2182239539b](li2182239539b.jpg)
(b)
![li2192239539b](li2192239539b.jpg)

Normal
(c)
![li2202239539b](li2202239539b.jpg)

Normal

FIGURE 9-2 Phenotypes of mutations in genes encoding transcription factors. (a) A dominant mutation in the human HOXD13 gene results in the development of extra digits, a condition known as polydactyly. (b) Homozygous recessive mutations that prevent expression of the Ubx gene in the third thoracic segment of Drosophila result in transformation of that segment, which normally has a balancing organ called a haltere, into a second copy of the thoracic segment that develops wings. (c) Mutations in Arabidopsis thaliana that inactivate both copies of three floral organ-identity genes transform the normal parts of the flower into leaflike structures. In each case, these mutations
for controlling eukaryotic gene expression are covered in Chapter 10. Subsequent chapters, particularly Chapters 15, 16, and 21, provide examples of how transcription is regulated by interactions between cells and how the resulting gene control contributes to the development and function of specific types of cells in multicellular organisms.
![li2212239539b](li2212239539b.jpg)

Dominant HOXD13 mutation
![li2222239539b](li2222239539b.jpg)

Ubx mutation
![li2232239539b](li2232239539b.jpg)

Homozygous recessive mutations in ap2-1, pi-1, and ag-1 genes
affect master regulatory transcription factors that regulate multiple genes, including many genes encoding other transcription factors. (Part (a), left, Lightvision, LLC/Moment Open/Getty Images; right, Goodman, F. R. and Scrambler, P. J., Human HOX gene mutations. Clinical Genetics, 2001, 59:1, pages 1-11. Part (b) from "The bithorax complex: the first fifty years," by Edward B. Lewis, reproduced with permission from The International Journal of Developmental Biology, 1998, Vol 42(403-15), Figures 4a and 4b. Part (c) republished with permission of Elsevier, from Weigel, D. and Meyerowitz, M., "The ABCs of floral homeotic genes," Cell, 1994, 78(2):203-209; permission conveyed through Copyright Clearance Center, Inc.)

![li2242239539b](li2242239539b.jpg)

### 9.1 Control of Gene Expression in Bacteria

Because the structure and function of a cell are determined by the proteins it contains, the control of gene expression is a fundamental aspect of molecular cell biology. Most commonly, the "decision" to transcribe the gene encoding a particular protein is the major mechanism for controlling production of the encoded protein in a cell. By controlling transcription, a cell can regulate which proteins it produces

FIGURE 9-3 Overview of eukaryotic transcriptional control. Inactive genes are assembled into regions of condensed chromatin that inhibit RNA polymerases and their associated general transcription factors from interacting with promoters. A pioneer transcription factor is able to bind to a specific regulatory sequence within the condensed chromatin and interact with chromatin-remodeling enzymes and histone acetylases that decondense the chromatin, making it accessible to RNA polymerase II and the general transcription factors. Additional activator proteins then bind to specific transcriptioncontrol elements in both promoter-proximal sites and distant enhancers, where they interact with one another and with the multisubunit Mediator complex to assemble RNA polymerase II (Pol II) and general transcription factors on promoters. Alternatively, repressor proteins bind to other transcription-control elements to inhibit transcription initiation by Pol II and interact with multiprotein co-repressor complexes to condense chromatin. During transcriptional activation, Pol II initiates transcription, but pauses after transcribing fewer than 100 nucleotides due to the action of the elongation inhibitor NELF associated with DSIF. Activators promote the association of the Pol II-NELF-DSIF complex with elongation factor P-TEFb, which releases NELF and allows productive elongation through the gene. DSIF is the DRB sensitivity-inducing factor, NELF is the negative elongation factor, and P-TEFb is a protein kinase made up of CDK9 and cyclin T. See S. Malik and R. G. Roeder, 2010, Nat. Rev. Genet. 11:761.
and how rapidly they are synthesized. When transcription of a gene is repressed, the corresponding mRNA and encoded protein or proteins are synthesized at low rates. Conversely, when transcription of a gene is activated, both the mRNA and encoded protein or proteins are produced at much higher rates.

In most bacteria and other single-celled organisms, gene expression is highly regulated in order to adjust the cell's enzymatic machinery and structural components to changes in the nutritional and physical environment. Thus at any given

time, a bacterial cell normally synthesizes only those proteins that are required for its survival under the current conditions. Here we describe the basic features of transcriptional control in bacteria, using the lac operon and the glutamine synthetase gene in E. coli and the $x p t-p b u X$ operon in Bacillus subtilis as our primary examples. Many of the same features are involved in eukaryotic transcriptional control, which will be the subject of the remainder of this chapter.

## Transcription Initiation by Bacterial RNA Polymerase Requires Association with a Sigma Factor

In E. coli, about half the genes are clustered into operons, each of which encodes enzymes involved in a particular metabolic pathway or proteins that interact to form one multisubunit protein complex. For instance, the trp operon discussed in Chapter 5 encodes five polypeptides needed in the biosynthesis of tryptophan (see Figure 5-13). Similarly, the lac operon encodes three proteins required for the metabolism of lactose, a sugar present in milk. Because a bacterial operon is transcribed from one start site into a single mRNA, all the genes within an operon are coordinately regulated; that is, they are all activated or repressed at the same time to the same extent.

The transcription of operons, as well as that of isolated genes, is controlled by interplay between RNA polymerase and specific repressor and activator proteins. In order to initiate transcription, E. coli RNA polymerase must associate with one of a small number of $\sigma$ (sigma) factors. The most common one in eubacterial cells is $\sigma^{70}$. This $\sigma$-factor binds to both RNA polymerase and promoter DNA sequences, bringing the RNA polymerase enzyme to the promoter. It recognizes and binds to both a six-base-pair sequence centered at about 10 bp and a seven-base-pair sequence centered at about 35 bp upstream from the +1 transcription start. Consequently, the -10 sequence and the -35 sequence together constitute a promoter for E. coli RNA polymerase associated with $\sigma^{70}$ (see Figure 5-10b). Although the promoter sequences contacted by $\sigma^{70}$ are located at -35 and -10 , E. coli RNA polymerase binds to the promoter-region DNA from roughly -50 to +20 through interactions with DNA that do not depend on the sequence. The $\sigma$-factor also assists the RNA polymerase in separating the DNA strands at the transcription start site and in inserting the coding strand into the active site of the polymerase so that transcription starts at +1 (see Figure 5-11, step $\mathbf{D}$ ). The optimal $\sigma^{70}$-RNA polymerase promoter sequence, determined as the "consensus sequence" of multiple strong promoters, is

$$
\begin{aligned}
& -35 \text { region } \quad-10 \text { region } \\
& \text { TTGACAT-15-17 bp- TATAAT }
\end{aligned}
$$

This consensus sequence shows the most commonly occurring base at each of the positions in the -35 and -10 regions. The size of the font indicates the importance of the base at that position, as determined by the influence of mutations of these bases on the frequency of transcription
initiation (i.e., the number of times per minute that RNA polymerases initiate transcription). The sequence shows the strand of DNA that has the same $5^{\prime} \rightarrow 3^{\prime}$ orientation as the transcribed RNA (i.e., the nontemplate strand). However, the $\sigma^{70}$-RNA polymerase initially binds to double-stranded DNA. After the polymerase transcribes a few tens of base pairs, $\sigma^{70}$ is released. Thus $\sigma^{70}$ acts as an initiation factor that is required for transcription initiation, but not for RNA strand elongation once initiation has taken place.

## Initiation of lac Operon Transcription Can Be Repressed or Activated

When E. coli is in an environment that lacks lactose, synthesis of lac mRNA is repressed so that cellular energy is not wasted synthesizing enzymes the cell does not require. In an environment containing both lactose and glucose, E. coli cells preferentially metabolize glucose, the central molecule of carbohydrate metabolism. The cells metabolize lactose at a high rate only when lactose is present and glucose is largely depleted from the medium. They achieve this metabolic adjustment by repressing transcription of the lac operon until lactose is present and allowing synthesis of only low levels of lac mRNA until the cytosolic concentration of glucose falls to low levels. Transcription of the lac operon under different conditions is controlled by lac repressor protein and catabolite activator protein (CAP) (also called CRP, for cAMP receptor protein), each of which binds to a specific DNA sequence in the lac transcription-control region; these two sequences are called the operator and the CAP site, respectively (Figure 9-4, top).

For transcription of the lac operon to begin, the $\sigma^{70}$ subunit of the RNA polymerase must bind to the lac promoter at the -35 and -10 promoter sequences. When no lactose is present, the lac repressor binds to the lac operator, which overlaps the transcription start site. Therefore, the lac repressor bound to the operator site blocks $\sigma^{70}$ binding and hence transcription initiation by RNA polymerase (Figure 9-4a). When lactose is present, it binds to specific binding sites in each subunit of the tetrameric lac repressor, causing a conformational change in the protein that makes it dissociate from the lac operator. As a result, the polymerase can bind to the promoter and initiate transcription of the lac operon. However, when glucose is also present, the frequency of transcription initiation is very low, resulting in the synthesis of only low levels of lac mRNA and thus of the proteins encoded by the lac operon (Figure 9-4b). The frequency of transcription initiation is low because the -35 and -10 sequences in the lac promoter differ from the ideal $\sigma^{70}$-binding sequences shown previously.

Once glucose is depleted from the medium and the intracellular glucose concentration falls, E. coli cells respond by synthesizing cyclic AMP (cAMP). As the concentration of cAMP increases, it binds to a site in each subunit of the dimeric CAP protein, causing a conformational change that allows the protein to bind to the CAP site in the lac transcription-control region. The bound CAP-cAMP complex interacts with the polymerase bound to the promoter, greatly increasing the

![li2252239539b](li2252239539b.jpg)

FIGURE 9-4 Regulation of transcription from the lac operon of E. coli. (Top) The transcription-control region, composed of roughly a hundred base pairs, includes three protein-binding regions: the CAP site, which binds catabolite activator protein; the lac promoter, which binds the $\sigma^{70}$-RNA polymerase complex; and the lac operator, which binds lac repressor. The lacZ gene encoding the enzyme $\beta$-galactosidase, the first of the three genes in the operon, is shown to the right. (a) In the absence of lactose, very little lac mRNA is produced because the lac repressor binds to the operator, inhibiting transcription initiation by $\sigma^{70}$-RNA polymerase. (b) In the presence of glucose and lactose, lac repressor binds lactose and dissociates from the operator, allowing $\sigma^{70}$-RNA polymerase to initiate transcription at a low rate. (c) Maximal transcription of the lac operon occurs in the presence of lactose and the absence of glucose. In this situation, cAMP increases in response to the low glucose concentration and forms a CAP-cAMP complex, which binds to the CAP site, where it interacts with RNA polymerase to increase the rate of transcription initiation. (d) The tetrameric lac repressor binds to the primary lac operator (O1) and one of two secondary operators (O2 or O3) simultaneously. The two structures are in equilibrium. See B. Muller-Hill, 1998, Curr. Opin. Microbiol. 1:145. [Part (d) data from M. Lewis et al., 1996, Science 271:1247-1254, PDB IDs 1lbh and 1lbg; and R. Daber et al., 2007, J. Mol. Biol. 370:609-619, PDB ID 2pe5.]
frequency of transcription initiation. This activation leads to synthesis of high levels of lac mRNA and subsequently of the enzymes encoded by the lac operon (Figure 9-4c).

In fact, the lac operon is more complex than depicted in the simplified model in Figure 9-4a-c. The tetrameric lac repressor actually binds to two DNA sequences simultaneously, one at the primary operator (lacO1), which overlaps the region of DNA bound by RNA polymerase at the promoter, and the other at one of two secondary operators centered at +412 (lacO2), within the lacZ protein-coding region, and -82 (lacO3) (Figure 9-4d). The lac repressor tetramer is a dimer of dimers. Each dimer binds to one operator (Figure 9-4d). Simultaneous binding of the tetrameric lac repressor to the primary lac operator and one of the two secondary operators is possible because DNA is quite flexible, as we saw in the wrapping of DNA around the surface of a histone octamer in the nucleosomes of eukaryotes (see Figure 8-24). The secondary operators function to increase the local concentration of lac repressor in the micro-vicinity of the primary operator where repressor binding blocks RNA polymerase binding. Since the equilibrium of binding reactions depends on the concentrations of the binding partners, the resulting increased local concentration of lac repressor in the vicinity of O 1 increases repressor binding to O 1 . There are approximately 10 lac repressor tetramers per E. coli cell. Because of binding to O 2 and O 3 , there is nearly always a lac repressor tetramer much closer to O 1 than would otherwise be the case if the 10 repressor tetramers were diffusing randomly through the cell. If both O 2 and O 3 are mutated so that the lac repressor no longer binds to them with high affinity, repression at the lac promoter is reduced by a factor of 70 . Mutation of only O 2 or only O 3 reduces repression twofold, indicating that either one of these secondary operators can provide most of the increase in repression.

Although the promoters for different E. coli genes exhibit considerable homology, their exact sequences differ. The promoter sequence determines the intrinsic frequency at which RNA polymerase- $\sigma$ complexes initiate transcription of a gene in the absence of a repressor or activator protein. Promoters that support a high frequency of transcription initiation have -10 and -35 sequences similar to the ideal promoter shown previously and are called strong promoters. Those that support a low frequency of transcription initiation differ from this ideal sequence and are called weak promoters. The lac operon, for instance, has a weak promoter whose sequence differs from the consensus strong promoter at several positions. Its low intrinsic frequency of initiation is further reduced by the lac repressor and substantially increased by the cAMP-CAP complex.

## Small Molecules Regulate Expression of Many Bacterial Genes via DNA-Binding Repressors and Activators

Transcription of most E. coli genes is regulated by processes similar to those described for the lac operon, although the detailed interactions differ at each promoter. The general mechanism involves a specific repressor that binds to the operator

region of a gene or operon, thereby blocking transcription initiation. A small-molecule ligand binds to the repressor controlling its DNA-binding activity, and consequently the frequency of transcription initiation and therefore the rate of synthesis of the mRNA and encoded proteins as appropriate for the needs of the cell. As for the lac operon, many eubacterial transcription-control regions contain one or more secondary operators that contribute to the level of repression.

Specific activator proteins, such as CAP in the lac operon, also control transcription of a subset of bacterial genes that have binding sites for the activator. Like CAP, other activators bind to DNA together with RNA polymerase, stimulating transcription from a specific promoter. The DNA-binding activity of an activator can be modulated in response to cellular needs by the binding of specific smallmolecule ligands (e.g., cAMP) or by post-translational modifications, such as phosphorylation, that alter the conformation of the activator.

## Transcription Initiation from Some Promoters Requires Alternative Sigma Factors

Most E. coli promoters interact with $\sigma^{70}$-RNA polymerase, the major initiating form of the bacterial enzyme. The transcription of certain groups of genes, however, is initiated by E. coli RNA polymerases containing one of several
alternative sigma factors that recognize different consensus promoter sequences than $\sigma^{70}$ does (Table 9-1). These alternative $\sigma$-factors are required for the transcription of sets of genes with related functions, such as those involved in the response to heat shock or nutrient deprivation, motility, or sporulation in gram-positive eubacteria. In E. coli, there are 6 alternative $\sigma$-factors in addition to the major "housekeeping" $\sigma$-factor, $\sigma^{70}$. The genome of the gram-positive, sporulating bacterium Streptomyces coelicolor encodes 63 $\sigma$-factors, the current record, based on sequence analysis of hundreds of eubacterial genomes. Most are structurally and functionally related to $\sigma^{70}$. Transcription initiation by RNA polymerases containing $\sigma^{70}$-like factors is regulated by repressors and activators that bind to DNA near the region where the polymerase binds. But one class, represented in E. coli by $\sigma^{34}$, is unrelated to $\sigma^{70}$ and functions differently.

## Transcription by $\sigma^{54}$-RNA Polymerase Is Controlled by Activators That Bind Far from the Promoter

The sequence of $\sigma^{34}$ is distinctly different from that of all the $\sigma^{70}$-like factors. Transcription of genes by RNA polymerases containing $\sigma^{34}$ is regulated solely by activators whose binding sites in DNA, referred to as enhancers, are generally located $80-160 \mathrm{bp}$ upstream from the transcription start site.

TABLE 9-1 Sigma Factors of E. coli

| Sigma Factor | Promoters Recognized | Promoter Consensus |  |
| :--: | :--: | :--: | :--: |
|  |  | -35 Region | -10 Region |
| $\sigma^{70}\left(\sigma^{\mathrm{D}}\right)$ | Housekeeping genes, most genes in exponentially replicating cells | TTGACA | TATAAT |
| $\sigma^{\mathrm{S}}\left(\sigma^{38}\right)$ | Stationary-phase genes and general stress response | TTGACA | TATAAT |
| $\sigma^{32}\left(\sigma^{\mathrm{H}}\right)$ | Induced by unfolded proteins in the cytoplasm; genes encoding chaperones that refold unfolded proteins and protease systems leading to the degradation of unfolded proteins in the cytoplasm | TCTCNCCCTTGAA | CCCCATNTA |
| $\sigma^{\mathrm{P}}\left(\sigma^{24}\right)$ | Activated by unfolded proteins in the periplasmic space and cell membrane; genes encoding proteins that restore integrity to the cellular envelope | GAACTT | TCTGA |
| $\sigma^{\mathrm{F}}\left(\sigma^{28}\right)$ | Genes involved in flagellum assembly | CTAAA | CCGATAT |
| $\operatorname{Fecl}\left(\sigma^{18}\right)$ | Genes required for iron uptake | TTGGAAA | GTAATG |
|  |  | -24 Region | -12 Region |
| $\sigma^{54}\left(\sigma^{5}\right)$ | Genes for nitrogen metabolism and other functions | CTGGNA | TTGCA |

Data from T. M. Gruber and C. A. Gross, 2003, Annu. Rev. Microbiol. 57:441, and B. K. Cho et al., 2014, BMC Biol. 12:4.

Even when enhancers are moved more than a kilobase away from a start site, $\sigma^{54}$-activators can activate transcription.

The best-characterized $\sigma^{54}$-activator-the NtrC protein (nitrogen regulatory protein C)—stimulates transcription of the $\operatorname{gln} A$ gene. The $\operatorname{gln} A$ gene encodes the enzyme glutamine synthetase, which synthesizes the amino acid glutamine, the central molecule of nitrogen metabolism, from glutamic acid and ammonia. The $\sigma^{54}$-RNA polymerase binds to the $\operatorname{gln} A$ promoter but does not melt the DNA strands and initiate transcription until it is activated by NtrC, a dimeric protein. NtrC, in turn, is regulated by a protein kinase called NtrB. In response to low levels of glutamine, NtrB phosphorylates dimeric NtrC, which then binds to an enhancer upstream of the $\operatorname{gln} A$ promoter. Enhancer-bound phosphorylated NtrC then stimulates the $\sigma^{54}$-polymerase bound at the promoter to separate the DNA strands and initiate transcription.

Electron microscopy studies have shown that phosphorylated NtrC bound at enhancers and $\sigma^{54}$-polymerase bound at the promoter interact directly, forming a loop in the DNA between the binding sites (Figure 9-5). As discussed
later in this chapter, this activation mechanism resembles the predominant mechanism of transcriptional activation in eukaryotes.

NtrC has ATPase activity, and ATP hydrolysis is required for activation of bound $\sigma^{54}$-RNA polymerase by phosphorylated NtrC. Mutants with an NtrC that is defective in ATP hydrolysis are invariably defective in stimulating the $\sigma^{54}$-RNA polymerase to melt the DNA strands at the transcription start site. It is postulated that ATP hydrolysis supplies the energy required for melting the DNA strands. In contrast, the $\sigma^{70}$-polymerase does not require ATP hydrolysis to separate the strands at a start site.

## Many Bacterial Responses Are Controlled by Two-Component Regulatory Systems

As we have just seen, control of the E. coli $\operatorname{gln} A$ gene depends on two proteins, NtrC and NtrB. Such two-component regulatory systems control many responses of bacteria to changes in their environment. At high concentrations of glutamine, glutamine binds to a sensor domain of NtrB, causing a
![li2262239539b](li2262239539b.jpg)

EXPERIMENTAL FIGURE 9-5 DNA looping permits interaction of bound NtrC and $\sigma^{54}$-RNA polymerase. (a) Drawing (left) and electron micrograph (right) of DNA restriction fragment with phosphorylated NtrC dimers bound to the enhancer region near one end and $\sigma^{54}$-RNA polymerase bound to the $\operatorname{gln} A$ promoter near the other end.
(b) Drawing (left) and electron micrograph (right) of the same fragment preparation, showing NtrC dimers and $\sigma^{54}$-RNA polymerase bound to each other, with the intervening DNA forming a loop between them. See W. Su et al., 1990, Proc. Natl. Acad. Sci. USA 87:5504. [Micrographs courtesy Harrison Echols and Carol Gross.]

conformational change in the protein that inhibits its histidine kinase activity (Figure 9-6a). At the same time, the regulatory domain of NtrC blocks its DNA-binding domain from binding the $g l n A$ enhancers. At low concentrations of glutamine, glutamine dissociates from the sensor domain in the NtrB protein, leading to activation of a histidine kinase transmitter domain in NtrB that transfers the $\gamma$-phosphate of ATP to a histidine residue $(\mathrm{H})$ in the transmitter domain. This phosphohistidine then transfers the phosphate to an aspartic acid residue (D) in the NtrC protein. This causes a conformational change in NtrC that unmasks the NtrC DNA-binding domain so that it can bind to the $g l n A$ enhancers.

Many other bacterial responses are regulated by two proteins with homology to NtrB and NtrC (Figure 9-6b).
(a) Two-component system regulating response to low Gln
![li2272239539b](li2272239539b.jpg)
(b) General two-component signaling system
![li2282239539b](li2282239539b.jpg)

FIGURE 9-6 Two-component regulatory systems. (a) At low cytoplasmic concentrations of glutamine, glutamine dissociates from NtrB, resulting in a conformational change that activates a protein kinase transmitter domain that transfers an ATP $\gamma$-phosphate to a conserved histidine $(\mathrm{H})$ in the transmitter domain. This phosphate is then transferred to an aspartic acid (D) in the regulatory domain of the response regulator NtrC. This converts NtrC into its activated form, which binds the enhancer sites upstream of the $g l n A$ promoter (see Figure 9-5). (b) General organization of two-component histidyl-aspartyl phospho-relay regulatory systems in bacteria and plants. See A. H. West and A. M. Stock, 2001, Trends Biochem. Sci. 26:369.

In each of these regulatory systems, one protein, called a histidine kinase sensor, contains a latent histidine kinase transmitter domain that is regulated in response to environmental changes detected by a sensor domain. When activated, the transmitter domain transfers the $\gamma$-phosphate of ATP to a histidine residue in the transmitter domain. The second protein, called a response regulator, contains a receiver domain homologous to the region of NtrC containing the aspartic acid residue that is phosphorylated by activated NtrB. The response regulator contains a second functional domain that is regulated by phosphorylation of the receiver domain. In many cases, this domain of the response regulator is a sequence-specific DNA-binding domain that binds to related DNA sequences and functions either as a repressor, like the lac repressor, or as an activator, like CAP or NtrC, regulating the transcription of specific genes. However, the effector domain can have other functions as well, such as controlling the direction in which the bacterium swims in response to a concentration gradient of nutrients. Although all transmitter domains are homologous (as are receiver domains), the transmitter domain of a specific sensor protein will phosphorylate only the receiver domains of specific response regulators, allowing specific responses to different environmental changes. Similar two-component histidyl-aspartyl phospho-relay regulatory systems are also found in plants.

## Expression of Many Bacterial Operons Is Controlled by Regulation of Transcriptional Elongation

In addition to regulation of transcription initiation by activators and repressors, expression of many bacterial operons is controlled by regulation of transcriptional elongation in the promoter-proximal region. This mechanism of control was first discovered in studies of trp operon transcription in E. coli (see Figure 5-13). Transcription of the $t r p$ operon is repressed by the trp repressor when the concentration of tryptophan in the cytoplasm is high. But the low level of transcription initiation that still occurs is further controlled by a process called attenuation when the concentration of charged $\operatorname{tRNA}^{\text {Trp }}$ is sufficient to support a high rate of protein synthesis. The first 140 nt of the trp operon does not encode proteins required for tryptophan biosynthesis, but rather consists of a short peptide "leader sequence," as diagrammed in Figure 9-7a. Region 1 of this leader sequence contains two successive Trp codons. Region 3 can base-pair with either region 2 or region 4. A ribosome follows closely behind the RNA polymerase, initiating translation of the leader peptide shortly after the $5^{\prime}$ end of the $t r p$ leader sequence emerges from the RNA polymerase. When the concentration of $\mathrm{tRNA}^{\text {Trp }}$ is sufficient to support a high rate of protein synthesis, the ribosome translates quickly through region 1 into region 2 , blocking the ability of region 2 to base-pair with region 3 as it emerges from the surface of the transcribing RNA polymerase (Figure 9-7b, left). Instead, region 3 base-pairs with region 4 as soon as it emerges from the surface of the polymerase, forming a stem-loop (see Figure 5-9a) followed by several uracils, which is a signal for

![li2292239539b](li2292239539b.jpg)

FIGURE 9-7 Transcriptional control by regulation of RNA polymerase elongation and termination in the E. coli trp operon. (a) Diagram of the 140-nucleotide trp leader RNA. The numbered regions are critical to attenuation. (b) Translation of the trp leader sequence begins near the $5^{\prime}$ end soon after it is transcribed, while transcription of the rest of the polycistronic trp mRNA molecule continues.
bacterial RNA polymerase to pause transcription and terminate. As a consequence, the remainder of the long trp operon is not transcribed, and the cell does not waste the energy required for tryptophan synthesis, or for the translation of the encoded proteins, when the concentration of tryptophan is high.

However, when the concentration of $\mathrm{tRNA}^{\mathrm{Trp}}$ is not sufficient to support a high rate of protein synthesis, the ribosome stalls at the two successive Trp codons in region 1 (Figure 9-7b, right). As a consequence, region 2 base-pairs with region 3 as soon as it emerges from the transcribing RNA polymerase. This prevents region 3 from base-pairing with region 4 , so the $3-4$ hairpin does not form and does not cause RNA polymerase pausing or transcription termination. As a result, the proteins required for tryptophan synthesis are translated by ribosomes that initiate translation at the start codons for each of these proteins in the long polycistronic trp mRNA.

Attenuation of transcription elongation also occurs at some operons and single genes encoding enzymes involved in the biosynthesis of other amino acids and metabolites through the function of riboswitches. Riboswitches are sequences of RNA most commonly found in the $5^{\prime}$ untranslated region of bacterial mRNAs. They fold into complex tertiary structures called aptamers that bind small-molecule metabolites when those metabolites are present at sufficiently high concentrations. In some cases, this binding results in the formation of stem-loop structures that lead to early termination of transcription, as in the Bacillus subtilis xpt-pbuX operon, which encodes enzymes involved in purine synthesis (Figure 9-8). When the concentration of small-molecule metabolites is lower, the metabolites are

Low tryptophan
Ribosome is stalled at trp codons in region 1
![li2302239539b](li2302239539b.jpg)

At high concentrations of charged tRNA ${ }^{\text {Trp }}$, formation of the 3-4 stemloop followed by a series of uracils causes termination of transcription. At low concentrations of charged tRNA ${ }^{\text {Trp }}$, region 3 is sequestered in the 2-3 stem-loop and cannot base-pair with region 4. In the absence of the stem-loop structure required for termination, transcription of the trp operon continues. See C. Yanofsky, 1981, Nature 289:751.
not bound by the aptamers, and alternative RNA structures form that do not induce transcription termination, allowing transcription of genes encoding enzymes involved in the synthesis of the metabolites. As we will see below, although the mechanism in eukaryotes is different, regulation of promoterproximal transcriptional pausing and termination has recently been discovered to occur frequently in the regulation of gene expression in multicellular organisms as well.

## KEY CONCEPTS OF SECTION 9.1

## Control of Gene Expression in Bacteria

- Gene expression in both prokaryotes and eukaryotes is regulated primarily by mechanisms that control gene transcription.
- The first step in the initiation of transcription in E. coli is the binding of a $\sigma$-factor complexed with an RNA polymerase to a promoter.
- The nucleotide sequence of a promoter determines its strength, that is, how frequently different RNA polymerase molecules can bind and initiate transcription per minute.
- Repressors are proteins that bind to operator sequences that overlap or lie adjacent to promoters. Binding of a repressor to an operator inhibits transcription initiation or elongation.
- The DNA-binding activity of most bacterial repressors is modulated by small-molecule ligands. This allows bacterial cells to regulate transcription of specific genes in response

![li2312239539b](li2312239539b.jpg)

FIGURE 9-8 Riboswitch control of transcription termination in B. subtilis. (a) During transcription of the Bacillus subtilis xpt-pbuX operon, which encodes enzymes involved in purine synthesis, the $5^{\prime}$ untranslated region of the mRNA can fold into alternative structures depending on the concentration of purines in the cytoplasm, forming the "purine riboswitch." At high concentrations of purines, the riboswitch folds into an aptamer that binds a purine ligand (cyan circle), allowing formation of a stem-loop transcription termination signal similar to the termination signal that forms in the E. coli trp operon mRNA at high
tryptophan concentrations (see Figure 9-7), i.e., a stem loop followed by a run of Us. At low purine concentrations, an alternative RNA structure forms that prevents formation of the transcription termination signal, permitting transcription of the operon. Note the alternative base pairing of the red and blue regions of the RNA. (b) Structure of the purine riboswitch bound to a purine (cyan) as determined by X-ray crystallography. See A. D. Garst, A. L. Edwards, and R. T. Batey, 2011, Cold Spring Harb. Perspect. Biol. 3:a003533. [Part (b) data from R. T. Batey, S. D. Gilbert, and R. K. Montagne, 2004, Nature 432:411, PDB ID 4fe5.]

- Genes transcribed by $\sigma^{34}$-RNA polymerase are regulated by activators that bind to enhancers located about 100 base pairs upstream from the start site. When the activator and $\sigma^{34}$-RNA polymerase interact, the DNA between their binding sites forms a loop (see Figure 9-5).
- In two-component regulatory systems, one protein acts as a sensor, monitoring the level of nutrients or other components in the environment. Under appropriate conditions, the
$\gamma$-phosphate of an ATP is transferred first to a histidine in the sensor protein and then to an aspartic acid in a second protein, the response regulator. The phosphorylated response regulator then performs a specific function in response to the stimulus, such as binding to DNA regulatory sequences, thereby stimulating or repressing transcription of specific genes (see Figure 9-6).
- Transcription in bacteria can also be regulated by control of transcriptional elongation in the promoter-proximal region. This control can be exerted by ribosome binding to the nascent mRNA, as in the case of the E. coli trp operon (see Figure 9-7), or by riboswitches, RNA sequences that bind small molecules, as for the B. subtilis xpt-pbuX operon (see Figure 9-8), to determine whether a stem-loop followed by a string of uracils forms, causing the bacterial RNA polymerase to pause and terminate transcription.


### 9.2 Overview of Eukaryotic Gene Control

In bacteria, gene control serves mainly to allow a single cell to adjust to changes in its environment so that its growth and division can be optimized. In multicellular organisms, environmental changes also induce changes in gene expression. An example is the response to low oxygen concentrations

(hypoxia), in which a specific set of genes is rapidly induced that helps the cell survive under the hypoxic conditions. These genes include those encoding secreted angiogenic proteins that stimulate the growth and penetration of new capillaries into the surrounding tissue. However, the most characteristic and biologically far-reaching purpose of gene control in multicellular organisms is execution of the genetic program that underlies embryological development. Generation of the many different cell types that collectively form a multicellular organism depends on the right genes being activated in the right cells at the right time during the developmental period.

In most cases, once a developmental step has been taken by a cell, it is not reversed. Thus these decisions are fundamentally different from the reversible activation and repression of bacterial genes in response to environmental conditions. In executing their genetic programs, many differentiated cells (e.g., skin cells, red blood cells, and antibody-producing cells) march down a pathway to final cell death, leaving no progeny behind. The fixed patterns of gene control leading to differentiation serve the needs of the whole organism and not the survival of an individual cell.

Despite the differences in the purposes of gene control in bacteria and eukaryotes, two key features of transcriptional control first discovered in bacteria and described in the previous section also apply to eukaryotic cells. First, protein-binding regulatory DNA sequences, or transcription-control regions, are associated with genes. Second, specific proteins that bind to a gene's transcription-control regions determine where transcription will start and either activate or repress transcription. One fundamental difference between transcriptional control in bacteria and in eukaryotes is a consequence of the association of eukaryotic chromosomal DNA with histone octamers, forming nucleosomes that associate into chromatin fibers that further associate into chromatin of varying degrees of condensation (see Figures 8-24, 8-25, 8-27, and 8-28). Eukaryotic cells exploit chromatin structure to regulate transcription, a mechanism of transcriptional control that is not available to bacteria. In multicellular eukaryotes, many inactive genes are assembled into condensed chromatin, which inhibits binding of the RNA polymerases and general transcription factors required for transcription initiation (see Figure 9-3). Activator proteins, which bind to transcription-control regions near the transcription start site of a gene as well as kilobases away, promote chromatin decondensation, binding of RNA polymerase to the promoter, and transcriptional elongation. Repressor proteins, which bind to alternative control elements, cause condensation of chromatin and inhibition of polymerase binding or elongation. In this section, we discuss the general principles of eukaryotic gene control and point out some similarities and differences between bacterial and eukaryotic systems. Subsequent sections of this chapter will address specific aspects of eukaryotic transcription in greater detail.

## Regulatory Elements in Eukaryotic DNA Are Found Both Close to and Many Kilobases Away from Transcription Start Sites

Direct measurements of the transcription rates of multiple genes in different cell types have shown that regulation of transcription, either at the initiation step or during elongation in the promoter-proximal region, is the most widespread form of gene control in eukaryotes, as it is in bacteria. In eukaryotes, as in bacteria, a DNA sequence that specifies where RNA polymerase binds and initiates transcription of a gene is called a promoter. Transcription from a particular promoter is controlled by DNA-binding proteins that are functionally equivalent to bacterial repressors and activators. However, eukaryotic transcriptional regulatory proteins can often function either to activate or to repress transcription, depending on their associations with other proteins. Consequently, they are more generally called transcription factors.

The DNA control elements in eukaryotic genomes to which transcription factors bind are often located much farther from the promoter they regulate than is the case in bacterial genomes. In some cases, transcription factors bind at regulatory sites tens of thousands of base pairs either upstream (opposite to the direction of transcription) or downstream (in the same direction as transcription) from the promoter. As a result of this arrangement, transcription of a single gene may be regulated by the binding of multiple different transcription factors to alternative control elements, which direct expression of the same gene in different types of cells and at different times during development.

For example, several separate transcription-control regions regulate expression of the mammalian gene encoding the transcription factor Pax6. As mentioned in Chapter 1, Pax6 protein is required for development of the eye. Pax6 is also required for the development of certain regions of the brain and spinal cord, and the cells in the pancreas that secrete hormones such as insulin. As also mentioned in Chapter 1, heterozygous humans with only one functional Pax6 gene are born with aniridia, a lack of irises in the eyes (see Figure 1-30d). In mammals, the Pax6 gene is expressed from at least three alternative promoters that function in different cell types and at different times during embryogenesis (Figure 9-9a).

Researchers often analyze transcription-control regions by preparing recombinant DNA molecules that combine a fragment of DNA to be tested with the coding region for a reporter gene whose expression is easily assayed. Typical reporter genes include the gene that encodes luciferase, an enzyme that generates light that can be assayed with great sensitivity and over many orders of magnitude of intensity using a luminometer. Other frequently used reporter genes encode green fluorescent protein (GFP), which can be visualized by fluorescence microscopy (see Figures 4-9d and 4-16), and E. coli $\beta$-galactosidase, which generates an intensely blue insoluble precipitate when incubated with the colorless soluble lactose analog X-gal. When transgenic mice (see Figure 6-40) containing a $\beta$-galactosidase

![li2322239539b](li2322239539b.jpg)

FIGURE 9-9 Transcription-control regions of the mouse Pax6 gene and the orthologous human PAX6 gene. (a) Three alternative Pax6 promoters are used at distinct times during embryogenesis in different tissues of the developing mouse embryo. Transcription-control regions regulating expression of Pax6 in different tissues are indicated by colored rectangles. These control regions are some 200-500 bp in length. (b) Expression of a $\beta$-galactosidase reporter transgene fused to the 8 kb of mouse DNA upstream from exon 0. A transgenic mouse embryo 10.5 days after fertilization was stained with X-gal to reveal $\beta$-galactosidase. Lens pit (LP) is the tissue that will develop into the lens of the eye. Expression was also observed in tissue that will develop into the pancreas (P). (c) Expression in a mouse embryo at 13.5 days after fertilization of a $\beta$-galactosidase reporter gene linked to the sequence in part (a) between exons 4 and 5 marked Retina. Arrow points to nasal and temporal regions of the developing retina. (d) Human PAX6 control regions identified in the $600-\mathrm{kb}$ region of human DNA between the upstream gene RCN1 and the promoter of the downstream ELP4 gene. RCN1 and ELP4 are transcribed in the opposite direction from PAX6, as represented by the leftward-pointing arrows associated with their first exons. RCN1 and ELP1 exons are shown as black rectangles
below the line representing this region of human DNA. PAX6 exons are diagrammed as red rectangles above the line. The three PAX6 promoters first characterized in the mouse are shown by rightward arrowheads, and the control regions shown in (a) are represented by gray rectangles. Regions flanking the gene where the sequence is partially conserved in most vertebrates (as in Figure 9-10a) are shown as ovals. Colored ovals represent sequences that cause expression of the transgene in specific neuroanatomical locations in the zebrafish central nervous system. Ovals with the same color stimulated expression in the same region. Gray ovals represent conserved sequences that did not stimulate reporter-gene expression in the developing zebrafish embryo, or were not tested. Such conserved regions may function only in combination, or they may have been conserved for some reason other than regulation of transcription, such as proper folding of the chromosome into topological domains (see Figure 8-34). [Part (a) data from B. Kammendal et al., 1999, Devel. Biol. 205:79. Part (b) republished with permission of Elsevier, B. Kammendal et al., "Distinct cis-essential modules direct the time-space pattern of the Pax6 gene activity," Developmental Biology, 1999, 205(1): 79-97; permission conveyed through Copyright Clearance Center, Inc. Part (c) courtesy of Peter Gruss and Birgitta Kammandel. Part (d) data from S. Batia et al., 2014, Devel. Biol. 387:214.]

reporter gene fused to 8 kb of DNA upstream from Pax6 exon 0 were produced, $\beta$-galactosidase was observed in the developing lens, cornea, and pancreas of the embryo halfway through gestation (Figure 9-9b). Analysis of transgenic mice with smaller fragments of DNA from this region allowed the mapping of the separate transcription-control regions regulating transcription in the pancreas, and in both the lens and cornea. Transgenic mice with other reporter gene constructs revealed additional transcription-control regions (see Figure 9-9a). These regions control transcription in the developing retina and in different regions of the developing brain (encephalon). Some of these transcription-control regions are in introns between exons 4 and 5 and between exons 7 and 8 . For example, a reporter gene under control of the region labeled Retina in Figure 9-9a between exons 4 and 5 led to reporter-gene expression specifically in the retina (Figure 9-9c).

Control regions for many genes are found hundreds of kilobases away from the coding exons of the gene. One method for identifying such distant control regions is to compare the sequences of distantly related organisms. Transcription-control regions for a conserved gene are also often conserved and can be recognized in the background of nonfunctional sequences that diverge during evolution.
![li2332239539b](li2332239539b.jpg)
(b) Mouse egg microinjection
![li2342239539b](li2342239539b.jpg)
(c) E11.5 reporter staining
![li2352239539b](li2352239539b.jpg)

For example, there is a human DNA sequence, which is highly conserved between humans, mice, chickens, frog, and fish, about 500 kb downstream of the SALL1 gene (Figure 9-10a). SALL1 encodes a transcription factor required for normal development of the limbs. When transgenic mice were produced containing this conserved DNA sequence linked to a $\beta$-galactosidase reporter gene (Figure 9-10b), the transgenic embryos expressed a very high level of $\beta$-galactosidase in the developing limb buds (Figure 9-10c). Human patients with deletions in this region of the genome develop with limb abnormalities. These results indicate that this conserved region directs transcription of the SALL1 gene in the developing limb. Presumably, other transcription-control regions control expression of this gene in other types of cells, where it functions in the normal development of the ears, the lower intestine, and kidneys.

Because the sequences and functions of transcriptioncontrol regions are often conserved through evolution, the transcription factors that bind to these transcription-control regions to regulate gene expression in specific cell types are presumably conserved during evolution as well. This has made it possible to assay control regions in human DNA by

FIGURE 9-10 The human SALL1 enhancer activates expression of a reporter gene in limb buds of the developing mouse embryo. (a) Graphic representation of the conservation of DNA sequence in a region of the human genome (in the interval of chromosome 16 from 50214 kb to 50220.5 kb ) about 500 kb downstream from the SALL1 gene, which encodes a zinc-finger transcription repressor. A region of roughly 500 bp of nonprotein-coding sequence is conserved from zebrafish to human. Nine hundred base pairs of human DNA including this conserved region were inserted into a plasmid next to the coding region for E. coli $\beta$-galactosidase. (b) The plasmid was microinjected into a pronucleus of a fertilized mouse egg and implanted in the uterus of a pseudopregnant mouse to generate a transgenic mouse embryo with the reporter-gene-containing plasmid incorporated into its genome (see Figure 5-43). (c) After 11.5 days of development, at the time when limb buds develop, the fixed and permeabilized embryo was incubated in X-gal, which is converted by $\beta$-galactosidase into an insoluble, intensely blue compound. The results showed that the conserved region contains an enhancer that stimulates strong transcription of the $\beta$-galactosidase reporter gene specifically in limb buds. [Part (a) data from A. Visel et al., 2007. VISTA Enhancer Browser-a database of tissue-specific human enhancers. Nucleic Acids Res. 35:D88-92. Part (b) ©Deco/Alamy. Part (c) republished with permission of Nature, from Pennacchio, L.A., et al., "In vivo enhancer analysis of human conserved noncoding sequences", Nature, 444, 499-506, 2006; permission conveyed through Copyright Clearance Center, Inc.]

reporter-gene expression in transgenic zebrafish, a procedure that is far simpler, faster, and less expensive than preparing transgenic mice (Figure 9-9d). After discussing the proteins that function with RNA polymerase to carry out transcription in eukaryotic cells and eukaryotic promoters, we will return to a discussion of how such distant transcriptioncontrol regions, called enhancers, are thought to function.

## Three Eukaryotic RNA Polymerases Catalyze Formation of Different RNAs

The nuclei of all eukaryotic cells examined so far (e.g., vertebrate, Drosophila, yeast, and plant cells) contain three different RNA polymerases, designated I, II, and III. These enzymes are eluted at different salt concentrations during ion-exchange chromatography, reflecting the differences in their net charges. The three nuclear RNA polymerases also differ in their sensitivity to $\alpha$-amanitin, a poisonous cyclic octapeptide produced by some mushrooms (Figure 9-11). RNA polymerase I is insensitive to $\alpha$-amanitin, but RNA polymerase II is very sensitivethe drug binds near the active site of the enzyme and inhibits translocation of the enzyme along the DNA template. RNA polymerase III has intermediate sensitivity.

Each eukaryotic RNA polymerase catalyzes transcription of genes encoding different classes of RNA (Table 9-2). RNA polymerase I (Pol I), located in the nucleolus, transcribes genes encoding precursor rRNA (pre-rRNA), which is processed into 28S, 5.8S, and 18S rRNAs. RNA polymerase III (Pol III) transcribes genes encoding tRNAs, 5S rRNA, and an array of small stable RNAs, including one involved in RNA splicing (U6) and the RNA component of the signal recognition particle (SRP) involved in directing nascent proteins to the endoplasmic reticulum (see Chapter 13). RNA polymerase II (Pol II) transcribes all protein-coding genes: that is, it functions in production of mRNAs. RNA polymerase II
![li2362239539b](li2362239539b.jpg)

EXPERIMENTAL FIGURE 9-11 Liquid chromatography separates and identifies the three eukaryotic RNA polymerases, each with its own sensitivity to $\alpha$-amanitin. A protein extract from the nuclei of cultured eukaryotic cells was passed through a DEAE Sephadex column and adsorbed protein eluted (black curve) with a solution of constantly increasing NaCl concentration. An aliquot of each fraction of eluate collected from the column was assayed for RNA polymerase activity without (red curve) and with (green shading) $1 \mu \mathrm{~g} / \mathrm{ml} \alpha$-amanitin. This concentration of $\alpha$-amanitin inhibits polymerase II activity but has no effect on polymerases I and III. Polymerase III is inhibited by $10 \mu \mathrm{~g} / \mathrm{ml}$ of $\alpha$-amanitin, whereas polymerase I is unaffected even at this higher concentration. See R. G. Roeder, 1974, J. Biol. Chem. 249:241.
also produces four of the five small nuclear RNAs (snRNAs) that take part in RNA splicing and micro-RNAs (miRNAs) involved in translation control, as well as the closely related endogenous small interfering RNAs (siRNAs) (see Chapter 10).

TABLE 9-2 Classes of RNA Transcribed by the Three Eukaryotic Nuclear RNA Polymerases and Their Functions

| Polymerase | RNA Transcribed | RNA Function |
| :--: | :--: | :--: |
| RNA polymerase I | Pre-rRNA (28S, 18S, 5.8S rRNAs) | Ribosome components, protein synthesis |
| RNA polymerase II | mRNA <br> snRNAs <br> siRNAs <br> miRNAs | Encodes protein <br> RNA splicing <br> Chromatin-mediated repression, translation control <br> Translation control |
| RNA polymerase III | tRNAs <br> 5S rRNA <br> snRNA U6 <br> 7S RNA <br> Other small stable RNAs | Protein synthesis <br> Ribosome component, protein synthesis <br> RNA splicing <br> Signal recognition particle for insertion of polypeptides into the endoplasmic reticulum <br> Various functions, unknown for many |

![li2372239539b](li2372239539b.jpg)

FIGURE 9-12 Comparison of three-dimensional structures of bacterial and eukaryotic RNA polymerases. (a, b) These space-filling models are based on x-ray crystallographic analysis. (a) RNA polymerase from the bacterium T. aquaticus. The five subunits of the bacterial enzyme are distinguished by color. Only the N -terminal domains of the $\alpha$ subunits are included in this model. (b) Core RNA polymerase II from S. cerevisiae. Ten of the 12 subunits constituting yeast RNA polymerase II are shown in this model. Subunits that are similar in conformation to those in the bacterial enzyme are shown in the same colors. The C-terminal domain of the large subunit RPB1 was not observed in the crystal structure, but it is known to extend from the position
marked with a red arrow. (RPB is the abbreviation for "RNA polymerase $B$," which is an alternative way of referring to RNA polymerase II.) DNA entering the polymerases as they transcribe to the right is diagrammed. (c) Space-filling model of yeast RNA polymerase II including subunits 4 and 7. These subunits extend from the core portion of the enzyme shown in (b) near the region of the C-terminal domain of the large subunit. [Part (a) data courtesy of Seth Darst; see N. Korzheva et al., 2000, Science 289:619-625. Part (b) data from P. Cramer et al., 2001, Science 292:1863, PDB ID 1I50. Part (c) data from K. J. Armache et al., 2003, P. Natl. Acad. Sci. USA 100:6964, and D. A. Bushnell and R. D. Kornberg, 2003, P. Natl. Acad. Sci. USA 100:6969.]
to one another and are similar to the E. coli $\beta^{\prime}$ and $\beta$ subunits, respectively (see Figure 9-12a, b). Each of the eukaryotic RNA polymerases also contains an $\omega$-like and two nonidentical $\alpha$-like subunits (Figure 9-13). The extensive similarity in the structures of these core subunits in RNA polymerases from various sources indicates that RNA polymerase arose early in evolution and was largely conserved. This seems logical for an enzyme catalyzing a process as fundamental as the copying of RNA from DNA. In addition to the core subunits that are related to the E. coli RNA polymerase subunits, all three yeast RNA polymerases contain four additional small subunits, common to them but not to the bacterial RNA polymerase. Finally, each eukaryotic nuclear RNA polymerase has several enzyme-specific subunits that are not present in the other two (see Figure 9-13). Three of these additional subunits of Pol I and Pol III are homologous to the three additional Pol II-specific subunits. The other two Pol I-specific subunits are homologous to the Pol II general transcription factor TFIIF, discussed later, and the four additional subunits of Pol III are homologous to the Pol II general transcription factors TFIIF and TFIIE. These are likely stably associated with Pol III in the cell, and do not dissociate from it during purification.

![li2382239539b](li2382239539b.jpg)

FIGURE 9-13 Schematic representation of the subunit structure of the E. coli RNA core polymerase and yeast nuclear RNA polymerases. All three yeast polymerases have five core subunits homologous to the $\beta$, $\beta^{\prime}$, two $\alpha$, and $\omega$ subunits of E. coli RNA polymerase. The largest subunit (RPB1) of RNA polymerase II also contains an essential C-terminal domain (CTD). RNA polymerases I and III contain the same two nonidentical $\alpha$-like subunits, whereas RNA polymerase II contains two other nonidentical $\alpha$-like subunits. All three polymerases share the same $\omega$-like subunit and four other common subunits. In addition, each yeast polymerase contains three to seven unique smaller subunits.

The clamp domain of subunit RPB1 is so designated because it has been observed in two different positions in crystals of free Pol II (Figure 9-14a) and in a complex that mimics the elongating form of the enzyme (Figure 9-14b). This domain rotates on a hinge that is probably open when downstream DNA is inserted into this region of the polymerase, and then swings shut when the enzyme is in its elongation mode. It is postulated that when the $8-9-\mathrm{bp}$ RNA-DNA hybrid region near the active site (where RNA is base-paired to the template strand; see Figure 9-14b) is bound between RBP1 and RBP2, the clamp is locked in its closed position, anchoring the polymerase to the downstream double-stranded DNA. Furthermore, a transcription elongation factor called DSIF, discussed later, associates with the elongating polymerase, holding the clamp in its closed conformation. As a consequence, the polymerase is extraordinarily processive, which is to say that it continues to polymerize ribonucleotides until transcription is terminated. After termination and release of RNA from the exit
(a) Free RNA polymerase II
![li2392239539b](li2392239539b.jpg)
(b) Transcribing RNA polymerase II
![li2402239539b](li2402239539b.jpg)

FIGURE 9-14 The clamp domain of RPBI. The structures of the free (a) and transcribing (b) RNA polymerase II differ mainly in the position of a clamp domain in the RPB1 subunit (orange), which swings over the cleft between the jaws of the polymerase during formation of the transcribing complex, trapping the template DNA strand and transcript. Binding of the clamp domain to the 8-9-bp RNA-DNA hybrid may help couple clamp closure to the presence of RNA, stabilizing the closed, elongating complex. RNA is shown in red, and the template strand in light purple. For clarity, downstream nontemplate DNA is not shown. The clamp closes over the incoming downstream DNA. Portions of RBP2 that form one side of the cleft have been removed so that the nucleic acids can be better visualized. The $\mathrm{Mg}^{2+}$ ion that participates in catalysis of phosphodiester bond formation is shown in green. Wall is the domain of RPB2 that forces the template DNA entering the jaws of the polymerase to bend before it exits the polymerase. The bridge $\alpha$ helix, shown in green, extends across the cleft in the polymerase (see Figure 9-12b) and is postulated to bend and straighten as the polymerase translocates one base down the template strand. The nontemplate strand is thought to form a flexible single-stranded region above the cleft (not shown), extending from three bases downstream of the template base-paired to the $3^{\prime}$ base of the growing RNA to where the template strand exits the polymerase, where it hybridizes with the template strand to generate the transcription bubble. JPart (a) data from P. Cramer, D. A. Bushnell, and R. D. Kornberg, 2001, Science 292:1863, PDB ID 1150. Part (b) data from A. L. Gnatt et al., 2001, Science 292:1876, PDB ID 116h.)

channel, the clamp can swing open, releasing the enzyme from the template DNA. This mechanism can explain how human RNA polymerase II can transcribe the longest human gene, encoding dystrophin (DMD; see Figure 1-31), which is some 2 million base pairs in length, without dissociating and terminating transcription. Since transcription elongation proceeds at $1-2 \mathrm{~kb}$ per minute, transcription of the DMD gene requires approximately one day!

Gene-knockout experiments in yeast indicate that most of the subunits of the three nuclear RNA polymerases are essential for cell viability. Disruption of the genes encoding the few polymerase subunits that are not essential for viability (e.g., subunits 4 and 7 of RNA polymerase II) nevertheless results in very poorly growing cells. Thus all the subunits are necessary for eukaryotic RNA polymerases to function normally. Archaea, like eubacteria, have a single type of RNA polymerase involved in gene transcription, but archaeal RNA polymerases, like eukaryotic nuclear RNA polymerases, have on the order of a dozen subunits. Archaea also have general transcription factors, discussed later, that are related to those of eukaryotes, consistent with the closer evolutionary relationship between archaea and eukaryotes than between eubacteria and eukaryotes (see Figure 1-1).

## The Largest Subunit in RNA Polymerase II Has an Essential Carboxy-Terminal Repeat

The carboxyl end of RPB1, the largest subunit of RNA polymerase II, contains a stretch of seven amino acids that is nearly precisely repeated multiple times. Neither RNA polymerase I nor III contains these repeating units. This heptapeptide repeat, with a consensus sequence of Tyr-Ser-Pro-Thr-Ser-Pro-Ser, is known as the carboxy-terminal domain (CTD) (see Figure 9-12b, red arrow). Yeast RNA polymerase II contains 26 or more repeats, vertebrate enzymes have 52 repeats, and an intermediate number of repeats occur in RNA polymerase II from nearly all other eukaryotes. The CTD is critical for viability, and at least 10 copies of the repeat must be present for yeast to survive.

In vitro experiments with model promoters first showed that RNA polymerase II molecules that initiate transcription have a nonphosphorylated CTD. Once the polymerase initiates transcription and begins to move away from the promoter, many of the serine and some tyrosine residues in the CTD are phosphorylated. Analysis of polytene chromosomes from Drosophila salivary glands prepared just before molting of the larva, a time of active transcription, indicates that the CTD is also phosphorylated during in vivo transcription. The large chromosomal "puffs" induced at this time in development are regions where the genome is very actively transcribed. Staining with antibodies specific for the phosphorylated or nonphosphorylated CTD demonstrated that RNA polymerase II associated with the highly transcribed puffed regions contains a phosphorylated CTD (Figure 9-15).
![li2412239539b](li2412239539b.jpg)

EXPERIMENTAL FIGURE 9-15 Antibody staining demonstrates that the carboxy-terminal domain of RNA polymerase II is phosphorylated during in vivo transcription. Salivary-gland polytene chromosomes were prepared from Drosophila larvae just before they molted. The preparation was treated with a rabbit antibody specific for phosphorylated CTD and with a goat antibody specific for nonphosphorylated CTD. The preparation was then stained with fluorescein-labeled anti-goat antibody (green) and rhodamine-labeled anti-rabbit antibody (red). Thus polymerase molecules with a nonphosphorylated CTD stained green, and those with a phosphorylated CTD stained red. The molting hormone ecdysone induces very high rates of transcription in the puffed regions labeled 74 EF and 75 B ; note that only phosphorylated CTD is present in these regions. Smaller puffed regions transcribed at high rates are also visible. Nonpuffed sites that stained red (up arrow) or green (horizontal arrow) are also indicated, as is a site staining both red and green, producing a yellow color (down arrow). (From J. R. Weeks et al., "Locus-specific variation in phosphorylation state of RNA polymerase II in vivo: correlations with gene activity and transcript processing," Genes \& Development, 1993, 7(12A):2329-44; courtesy of J. R. Weeks and A. L. Greenleaf; republished with permission from Cold Spring Harbor Press.)

## KEY CONCEPTS OF SECTION 9.2

## Overview of Eukaryotic Gene Control

- The primary purpose of gene control in multicellular organisms is the execution of precise developmental programs so that the proper genes are expressed in the proper cells at the proper times during embryologic development and cellular differentiation.
- Transcriptional control is the primary means of regulating gene expression in eukaryotes, as it is in bacteria.
- In eukaryotic genomes, DNA transcription-control elements may be located many kilobases away from the promoter they regulate. Different control elements can control transcription of the same gene in different cell types.
- Eukaryotes contain three types of nuclear RNA polymerases. All three contain two large and three smaller core

subunits with homology to the $\beta^{\prime}, \beta, \alpha$, and $\omega$ subunits of E. coli RNA polymerase, as well as several additional small subunits (see Figure 9-13).

- RNA polymerase I synthesizes only pre-rRNA. RNA polymerase II synthesizes mRNAs, some of the small nuclear RNAs that participate in mRNA splicing, and micro- and small interfering RNAs (miRNAs and siRNAs) that regulate the translation and stability of mRNAs. RNA polymerase III synthesizes tRNAs, 5S rRNA, and several other small stable RNAs (see Table 9-2).
- The carboxy-terminal domain (CTD) in the largest subunit of RNA polymerase II becomes phosphorylated during transcription initiation and remains phosphorylated as the enzyme transcribes the DNA template.


### 9.3 RNA Polymerase II Promoters and General Transcription Factors

The mechanisms that regulate transcription initiation and elongation by RNA polymerase II have been studied extensively because this polymerase is the one that transcribes mRNAs. Transcription initiation and elongation by RNA polymerase II are the initial biochemical processes required for the expression of protein-coding genes and are the steps in gene expression that are most frequently regulated to determine when and in which cells specific proteins are synthesized. As noted in the previous section, the expression of eukaryotic protein-coding genes is regulated by multiple protein-binding DNA sequences, generically referred to as transcription-control regions. These sequences include promoters, which determine where transcription of the DNA template begins, and other types of control elements located near transcription start sites, as well as sequences located far from the genes they regulate, called enhancers, which control the type of cell in which the gene is transcribed and how frequently it is transcribed. In this section, we take a closer look at the properties of various transcription-control elements found in eukaryotic protein-coding genes and some techniques used to identify them.

## RNA Polymerase II Initiates Transcription at DNA Sequences Corresponding to the 5' Cap of mRNAs

In vitro transcription experiments using purified RNA polymerase II, a protein extract prepared from the nuclei of cultured cells, and DNA templates containing sequences encoding the $5^{\prime}$ ends of mRNAs for a number of abundantly expressed genes revealed that the transcripts produced always contained a cap structure at their $5^{\prime}$ ends identical to that present at the $5^{\prime}$ end of the spliced mRNA normally expressed from the gene in vivo (see Figure 5-14). In these
experiments, the $5^{\prime}$ cap was added to the $5^{\prime}$ end of the nascent RNA by enzymes in the nuclear extract, which can add a cap only to an RNA that has a $5^{\prime}$ tri- or diphosphate. Because a $5^{\prime}$ end generated by cleavage of a longer RNA would have a $5^{\prime}$ monophosphate, it would not be capped. Consequently, researchers concluded that the capped nucleotides generated in the in vitro transcription reactions must have been the nucleotides with which transcription was initiated. Sequence analysis revealed that, for any given gene, the sequence at the $5^{\prime}$ end of the RNA transcripts produced in vitro is the same as that at the $5^{\prime}$ end of the mRNAs isolated from cells, confirming that the capped nucleotide of eukaryotic mRNAs coincides with the transcription start site. Today the transcription start site for a newly characterized mRNA is generally determined simply by identifying the DNA sequence encoding the $5^{\prime}$-capped nucleotide of the encoded mRNA.

## The TATA Box, Initiators, and CpG Islands Function as Promoters in Eukaryotic DNA

Several different types of DNA sequences can function as promoters for RNA polymerase II, telling the polymerase where to initiate transcription of an RNA complementary to the template strand of a double-stranded DNA molecule. These sequences include TATA boxes, initiators, and CpG islands.

TATA Boxes The first genes to be sequenced and studied through in vitro transcription systems were viral genes and cellular protein-coding genes that are very actively transcribed, either at particular times of the cell cycle or in specific differentiated cell types. In all these highly transcribed genes, a conserved sequence called the TATA box was found about $26-31 \mathrm{bp}$ upstream of the transcription start site (Figure 9-16). Mutagenesis studies have shown that a singlebase change in this nucleotide sequence drastically decreases
![li2422239539b](li2422239539b.jpg)

FIGURE 9-16 Core promoter elements of non-CpG island promoters in metazoans. The sequence of each element is shown with the $5^{\prime}$ end at the left and the $3^{\prime}$ end at the right. The most frequently observed bases in TATA box promoters are shown in larger font. $A^{+1}$ is the base at which transcription starts, $Y$ is a pyrimidine ( $C$ or $T$ ), $N$ is any of the four bases. [Data from S. T. Smale and J. T. Kadonaga, 2003, Annu. Rev. Biochem. 72:449.]

in vitro transcription of the gene adjacent to it. If the base pairs between the TATA box and the normal transcription start site are deleted, transcription of the altered, shortened template begins at a new site about 25 bp downstream from the TATA box. Consequently, the TATA box acts similarly to an E. coli promoter to position RNA polymerase II for transcription initiation (see Figure 5-12).

Initiator Sequences Instead of a TATA box, some eukaryotic genes contain an alternative promoter element called an initiator. Most naturally occurring initiator elements have a cytosine (C) at the -1 position and an adenine (A) residue at the transcription start site $(+1)$. Directed mutagenesis of mammalian genes with an initiator-containing promoter revealed that the nucleotide sequence immediately surrounding the start site determines the strength of such promoters. In contrast to the conserved TATA box sequence, however, only an extremely degenerate initiator consensus sequence has been defined:

$$
\left(5^{\prime}\right) \mathrm{Y}-\mathrm{Y}-\mathrm{A}^{+1}-\mathrm{N}-\mathrm{T} / \mathrm{A}-\mathrm{Y}-\mathrm{Y}-\mathrm{Y}\left(3^{\prime}\right)
$$

where $\mathrm{A}^{+1}$ is the base at which transcription starts, Y is a pyrimidine ( C or T ), N is any of the four bases, and T/A is T or A at position +3 . As we will see, other promoter elements, designated BRE and DPE (see Figure 9-16), can be bound by general transcription factors and influence promoter strength.

CpG Islands Transcription of genes with promoters containing a TATA box or initiator element begins at a well-defined initiation site. However, the transcription of most proteincoding genes in mammals ( $\sim 70$ percent) occurs at a lower rate than at TATA box-containing and initiator-containing promoters and begins at any of several alternative start sites within regions of about 100-1000 bp that have an unusually high frequency of CG sequences. Many such genes encode proteins that are not required in large amounts (e.g., genes encoding enzymes involved in basic metabolic processes required in all cells, often called "housekeeping genes"). These promoter regions are called CpG islands (where "p" represents the phosphate between the C and G nucleotides) because they occur relatively rarely in the genome sequences of mammals.

In mammals, most Cs followed by a G that are not associated with CpG island promoters are methylated at position 5 of the pyrimidine ring (5-methyl C, represented $\mathrm{C}^{\mathrm{Me}}$; see Figure 2-17). CG sequences are thought to be underrepresented in mammalian genomes because spontaneous deamination of 5-methyl C generates thymidine. Over the time scale of mammalian evolution, this is thought to have led to the conversion of most CGs to TG by DNArepair mechanisms. As a consequence, the frequency of CG in the human genome is only 21 percent of that expected if Cs were randomly followed by any base. However, the

Cs in active CpG island promoters are unmethylated. Consequently, when they deaminate spontaneously, they are converted to U , a base that is recognized by DNA-repair enzymes and converted back to C. As a result, the frequency of CG sequences within CpG island promoters is close to that expected if C were followed by any of the other three nucleotides randomly.

CG-rich sequences are bound by histone octamers more weakly than CG-poor sequences because more energy is required to bend them into the small-diameter loops required to wrap around the histone octamer forming a nucleosome (see Figure 8-24). As a consequence, CpG islands coincide with nucleosome-free regions of DNA. Much remains to be learned about the molecular mechanisms that control transcription from CpG island promoters, but a current hypothesis is that the general transcription factors discussed in the next section can bind to them because CpG islands exclude nucleosomes.

## Divergent Transcription from CpG Island Promoters

Another remarkable feature of CpG islands is that transcription from these elements is initiated in both directions, even though only transcription of the sense strand yields an mRNA. By a mechanism(s) that remains to be fully elucidated, most RNA polymerase II molecules transcribing in the "wrong" direction-that is, transcribing the antisense strand-pause or terminate transcription about $1-3 \mathrm{~kb}$ from the transcription start site. This phenomenon was discovered by taking advantage of the stability conferred on the elongation complex by the RNA polymerase II clamp domain when an RNA-DNA hybrid is bound near the active site (see Figure 9-14b, c).

Nuclei were isolated from cultured human fibroblasts and incubated in a buffered solution containing salt and mild detergent, which removes RNA polymerases except for those in the process of elongation because of their stable association with template DNA. Nucleotide triphosphates were then added, with UTP replaced by bromo-UTP, containing uracil with a Br atom at position 5 on the pyrimidine ring (see Figure 2-17). The nuclei were then incubated at $30^{\circ} \mathrm{C}$ long enough for about 100 nucleotides to be polymerized by the RNA polymerase II (Pol II) molecules that were in the process of elongation at the time the nuclei were isolated. RNA was then isolated, and RNA containing bromo-U was immunoprecipitated with an antibody specific for BrU-labeled RNA. Thirty-three nucleotides at the $5^{\prime}$ ends of these RNAs were then sequenced by massively parallel DNA sequencing (see Chapter 6) of reverse transcripts, and the sequences were mapped on the human genome.

Figure 9-17 shows a plot of the number of sequence reads per kilobase of total BrU-labeled RNA relative to the major transcription start sites (TSS) of all currently known human protein-coding genes. The results show that approximately equal numbers of RNA polymerase molecules transcribed most promoters (mostly CpG island promoters) in the sense direction, toward the gene (blue, plotted

![li2432239539b](li2432239539b.jpg)

EXPERIMENTAL FIGURE 9-17 Analysis of elongating RNA polymerase II molecules in human fibroblasts. Nuclei from cultured fibroblasts were isolated and incubated in a buffer with a non-ionic detergent that prevents RNA polymerase II from initiating transcription. Treated nuclei were then incubated with ATP, CTP, GTP, and Br -UTP for 5 minutes at $30^{\circ} \mathrm{C}$, a time sufficient to incorporate about 100 nucleotides. RNA was then isolated and broken into fragments of about 100 nucleotides each by controlled incubation at high pH . Specific RNA oligonucleotides were ligated to the $5^{\prime}$ and $3^{\prime}$ ends of the RNA fragments, which were then subjected to reverse transcription. The resulting DNA was amplified by the polymerase chain reaction and subjected to massively parallel DNA sequencing. The sequences determined were aligned to the transcription start sites (TSS) of all known human genes, and the number of sequence reads per kilobase of total sequenced DNA was plotted for 10-bp intervals of sense transcripts (blue) and antisense transcripts (purple). See text for discussion. [Data from L. J. Core, J. J. Waterfall, and J. T. Lis, 2008, Science 322:1845.]
upward to indicate transcription in the sense direction), and in the antisense direction, away from the gene (purple, plotted downward to represent transcription of the complementary DNA strand in the opposite, antisense direction). A peak of sense transcripts was observed at about +50 relative to the major transcription start site (TSS), indicating that Pol II pauses in the +50 to +200 region before elongating further. A peak at -250 to -500 relative to the major transcription start site of Pol II transcribing in the opposite direction was also observed, revealing paused RNA polymerase II molecules at the other ends of the nucleosome-free regions in CpG island promoters. Note that the number of sequence reads, and therefore the number of elongating polymerases, is lower for polymerases transcribing in the antisense direction more than 1 kb from the transcription start site compared with polymerases transcribing more than 1 kb from the transcription start site in the sense direction. The molecular mechanism(s) potentially accounting for this difference is presented in Figure 10-15, in which transcription termination is discussed. Note that a low number of sequence reads was also observed resulting from transcription upstream of the major transcription start sites (blue sequence reads to the left of 0 and purple sequence reads to the right of 0 ), indicating
that there is a low level of transcription from seemingly random sites throughout the genome. These recent discoveries of divergent transcription from CpG island promoters and low-level transcription of most of the genomes of eukaryotes have been a great surprise to most researchers.

Chromatin Immunoprecipitation The technique of chromatin immunoprecipitation outlined in Figure 9-18a, using an antibody to RNA polymerase II, provided additional data supporting the occurrence of divergent transcription from most CpG island promoters in mammals. The data from this analysis are reported as the number of times a specific sequence from this region of the genome was identified per million total sequences analyzed (Figure 9-18b). At divergently transcribed genes, such as the Hsd17b12 gene encoding an enzyme involved in intermediary metabolism, two peaks of immunoprecipitated DNA were detected, corresponding to Pol II transcribing in the sense and antisense directions and then pausing. However, Pol II was detected more than 1 kb from the start site only in the sense direction. The number of counts per million from this region of the genome was very low because the gene is transcribed at low frequency. However, the number of counts per million at the transcription start site regions for both sense and antisense transcription was much higher, reflecting the fact that Pol II molecules had initiated transcription in both directions at this promoter, but paused before transcribing farther than 500 bp from the start sites in each direction. In contrast, the Rpl6 gene, encoding a large ribosomal subunit protein that was abundantly transcribed in the proliferating mouse embryonic stem cells used in the study, was transcribed almost exclusively in the sense direction. The peak in counts per million less than 250 bp from the transcription start site again results from a long pause in transcription in the promoter-proximal region before the polymerase is released to transcribe into the gene. The number of sequence counts per million more than 1 kb downstream from the transcription start site was much higher than for sense-direction transcription of the Hsd17b12 gene, reflecting the high rate of transcription of the Rpl6 gene.

## General Transcription Factors Position RNA Polymerase II at Start Sites and Assist in Initiation

Initiation of transcription by RNA polymerase II requires several initiation factors. These initiation factors position Pol II molecules at transcription start sites and help to separate the DNA strands so that the template strand can enter the active site of the enzyme. They are called general transcription factors because they are required at most, if not all, promoters of genes transcribed by RNA polymerase II. These proteins are designated TFIIA, TFIIB, and so on, and most are multimeric proteins. The largest is TFIID, which consists

![li2442239539b](li2442239539b.jpg)
of a single $38-\mathrm{kDa}$ TATA box-binding protein (TBP) and 13 TBP-associated factors (TAFs). General transcription factors with similar activities and homologous sequences are found in all eukaryotes. The complex of Pol II and its general transcription factors bound to a promoter and ready to initiate transcription is called a preinitiation complex (PIC). Figure 9-19 summarizes the current model for the stepwise assembly of the Pol II transcription preinitiation complex on a promoter containing a TATA box.

The TBP subunit of TFIID is the first protein to bind to a TATA box promoter. All eukaryotic TBPs analyzed to date have very similar C-terminal domains of 180 residues. This domain of TBP folds into a saddle-shaped structure; the two halves of the molecule exhibit an overall dyad symmetry but are not identical. TBP interacts with the minor groove in

EXPERIMENTAL FIGURE 9-18 The chromatin immunoprecipitation technique localizes where a protein of interest associates with the genome. (a) step Live cultured cells or tissues are incubated in 1 percent formaldehyde to covalently cross-link proteins to DNA and proteins to proteins. Step The preparation is then subjected to sonication to solubilize chromatin and shear it into fragments of 200-500 bp of DNA. Step $\mathbf{\square}$ An antibody to a protein of interest, here RNA polymerase II, is added, and DNA covalently linked to the protein of interest is immunoprecipitated. Step The covalent cross-linking is then reversed and the DNA is isolated. The isolated DNA can be analyzed by PCR with primers for a sequence of interest. Alternatively, total recovered DNA can be amplified, labeled by incorporation of a fluorescently labeled nucleotide, and hybridized to a microarray (see Figure 6-27) or subjected to massively parallel DNA sequencing. See A. Hecht and M. Grunstein, 1999, Method. Enzymol. 304:399. (b) Results from DNA sequencing of chromatin from mouse embryonic stem cells immunoprecipitated with antibody to RNA polymerase II are shown for a gene that is divergently transcribed (left) and a gene that is transcribed only in the sense direction (right). Data are plotted as the number of times a DNA sequence in a 50-bp interval was observed per million base pairs sequenced. The region encoding the $5^{\prime}$ end of the gene is shown below, with exons shown as rectangles and introns as lines. (Part (b) data from P. B. Rahl et al., 2010, Cell 141:452.)

DNA, bending the helix considerably (see Figure 5-5). The DNA-binding surface of TBP is conserved in all eukaryotes, explaining the high conservation of the TATA box promoter element (see Figure 9-16).

Once TFIID has bound to the TATA box, TFIIA and TFIIB can bind. TFIIA is a heterodimer larger than TBP, and TFIIB is a monomeric protein, slightly smaller than TBP. TFIIA associates with TBP and DNA on the upstream side of the TBP-TATA box complex. The C-terminal domain of TFIIB makes contact with both TBP and DNA on either side of the TATA box. During transcription initiation, its N-terminal domain is inserted into the RNA exit channel of RNA polymerase II (see Figure 9-12c). The TFIIB N-terminal domain assists Pol II in melting the DNA strands at the transcription start site and interacts with the template strand near the Pol II active site. Following TFIIB binding, a preformed complex of TFIIF (a heterodimer of two different subunits in mammals) and Pol II binds, positioning the polymerase over the start site. Two more general transcription factors must bind before the DNA duplex can be separated to expose the template strand. First to bind is TFIIE, a heterodimer of two different subunits. TFIIE creates a docking site for TFIIH, another multimeric factor containing 10 different subunits. Binding of TFIIH completes assembly of the transcription preinitiation complex (see Figure 9-19).

Figure 9-20 shows a cryoelectron microscopic image of a yeast (S. cerevisiae) preinitiation complex assembled in vitro from purified RNA polymerase II and general transcription factors with TBP in place of the complete TFIID complex-a total of thirty-three polypeptides with a mass

![li2452239539b](li2452239539b.jpg)

FIGURE 9-19 Model for the sequential assembly of an RNA polymerase II preinitiation complex. The indicated general transcription factors and purified RNA polymerase II (Pol II) bind sequentially to TATA box DNA to form a preinitiation complex (PIC). ATP hydrolysis then provides the energy for the unwinding of DNA at the transcription start site by a TFIIH helicase subunit that pushes downstream DNA into the polymerase. The DNA is held in position in the PIC by binding of the TATA box by the TBP subunit of TFIID, and the resulting strain on the structure of the duplex DNA assists the N -terminal region of TFIIB and Pol II to melt the DNA at the transcription start site, forming the transcription bubble. As Pol II initiates transcription in the resulting open complex, the polymerase transcribes away from the promoter, its CTD becomes phosphorylated by the TFIIH kinase domain, and the general transcription factors dissociate from the promoter. See S. Sainsbury, C. Berrnecky, and P. Cramer, 2015, Nat. Rev. Mol. Cell Biol. 16:129.
of 1.5 megadaltons (MDa)—about the size of a ribosomal subunit. Such elaborate preinitiation complexes assemble at the promoters of every protein-coding gene expressed by a eukaryotic cell.

The helicase activity of one of the core TFIIH subunits (Ssl2 in yeast; see Figure 9-20d) uses energy from ATP hydrolysis to help unwind the DNA duplex at the start site, allowing Pol II to form an open complex in which the DNA duplex surrounding the start site is melted and the template strand is bound at the polymerase active site. As the polymerase transcribes away from the promoter region, the N -terminal domain of TFIIB is released from the RNA exit channel as the $5^{\prime}$ end of the nascent RNA enters it. Three TFIIH subunits form a kinase module (TFIIH kinase in Figure 9-19) that phosphorylates the Pol II CTD multiple times on serine 5 (underlined) of the Tyr-Ser-Pro-Thr-Ser-Pro-Ser repeat that constitutes the CTD. As we will discuss further in Chapter 10, a multiply phosphorylated CTD is a docking site for the enzymes that form the cap structure (see Figure 5-14) on the $5^{\prime}$ end of an RNA transcribed by RNA polymerase II. In the minimal in vitro transcription assay with TBP substituted for the full TFIID complex and purified RNA polymerase II, TBP remains bound to the TATA box as the polymerase transcribes away from the promoter region, but the other general transcription factors dissociate.

$\square$ Remarkably, the first subunits of TFIIH to be cloned from humans were identified because mutations in them cause defects in the repair of damaged DNA, such as a base with a covalently linked mutagen or a UV-induced thy-mine-thymine dimer (see Figure 5-37). In normal individuals, when a transcribing RNA polymerase becomes stalled at a region of damaged template DNA, the core TFIIH complex, lacking the three subunits of the kinase domain (see Figure 9-19) but including the helicase subunit mentioned above, recognizes the stalled polymerase and then associates with other proteins that function with TFIIH in repairing the damaged DNA region. In patients with mutant forms of these TFIIH subunits, such repair of damaged DNA in

![li2462239539b](li2462239539b.jpg)

FIGURE 9-20 Model of the yeast preinitiation complex based on cryoelectron microscopy and fitting of known protein x-ray crystal structures. (a-c) Three views of the nearly complete PIC. The relative positions of Pol II and most of the GTFs are observed, but only about $50 \%$ of the mass of TFIIH is depicted because a large part of the mass of TFIIH is highly flexible and consequently could not be accurately determined by cryo-EM. Also high resolution structures have not been determined for many of the TFIIH subunits, and consequently could not be fitted to the TFIIH mass detected by cryo-EM. However, the interaction between DNA at the downstream side of the Pol II
transcriptionally active genes is impaired. As a result, affected individuals have extreme skin sensitivity to sunlight (a common cause of DNA damage through the generation of thymine-thymine dimers) and exhibit a high incidence of cancer. Consequently, these subunits of TFIIH serve two functions in the cell, one in the process of transcription initiation and a second in the repair of DNA. Depending on the severity of the defect in TFIIH function, these individuals may suffer from diseases such as xeroderma pigmentosum (see Chapter 24) and Cockayne syndrome (see Chapter 5).

The TAF subunits of TFIID function in initiating transcription from promoters that lack a TATA box. For instance, some TAF subunits contact the initiator element in promoters in which it occurs; their function probably explains how such sequences can replace a TATA box (see Figure 9-16). Additional TFIID TAF subunits can bind to a consensus sequence, A/G-G-A/T-C/T-G/A/C, that is centered about 30 bp downstream from the transcription start site in many genes that lack a TATA box promoter. Because of its position, this regulatory sequence is called the downstream promoter element (DPE) (see Figure 9-16). The DPE facilitates transcription of
cleft and the TFIIH Ssl2 helicase subunit required to melt promoter DNA is clearly visualized in (b) and (c). In (c), the interaction between TFIIH and TFIIE is not visualized because of the low resolution of the complex in this region. TFIIS is a Pol II elongation factor added to stabilize the PIC. (d) Model of entry of the template strand into the floor of the cleft where RNA polymerization is catalyzed. The Ssl2 helicase pushes DNA that is bound upstream to TBP, TFIIB, and TFIIA, creating torsional stress that contributes to transcription bubble melting. [Data from K. Murakami, et al. 2015. Proc. Natl. Acad. Sci. USA, 112:13543, PDB ID 5fmf.]

TATA-less genes that contain it by increasing TFIID binding. In addition, an $\alpha$ helix of TFIIB binds to the major groove of DNA upstream of the TATA box, and the strongest promoters contain the optimal sequence for this interaction, called the TFIIB recognition element (BRE) (see Figure 9-16).

Chromatin immunoprecipitation assays (see Figure 9-18) using antibodies to TBP show that it binds in the region between the divergent transcription start sites in CpG island promoters. Consequently, the same general transcription factors are probably required for initiation from the weaker CpG island promoters as for initiation from promoters containing a TATA box. The absence of the promoter elements summarized in Figure 9-16 may account for the divergent transcription from multiple transcription start sites observed from CpG island promoters, since cues from the DNA sequence are not present to correctly orient the preinitiation complex. TFIID and the other general transcription factors may choose among alternative, nearly equivalent weak binding sites in CpG island promoters, which may explain the low frequency of transcription initiation as well as the alternative transcription start sites in divergent directions generally observed from this class of promoters.

## Elongation Factors Regulate the Initial Stages of Transcription in the Promoter-Proximal Region

In metazoans, at most promoters, Pol II pauses after transcribing fewer than 100 nucleotides, due to the binding of a five-subunit protein called NELF (negative elongation factor). NELF binds to Pol II along with a two-subunit elongation factor called DSIF (DRB sensitivity-inducing factor, so named because an ATP analog called DRB inhibits further transcription elongation in its presence). The inhibition of elongation that results from NELF binding to Pol II is relieved when DSIF, NELF, and serine 2 of the Pol II CTD (Tyr-Ser-Pro-Thr-Ser-Pro-Ser) are phosphorylated by a protein kinase with two subunits, cyclin T-CDK9, also called $P-T E F b$, which associates with the Pol II-NELF-DSIF complex. The same elongation factors regulate transcription from CpG island promoters. These factors that regulate elongation in the promoter-proximal region provide a mechanism for controlling gene transcription in addition to the regulation of transcription initiation. This overall strategy for regulating transcription at both the initiation and elongation steps in the promoter-proximal region is similar to the regulation of the trp operon in E. coli (see Figure 9-7), although the molecular mechanisms involved are distinct.

8Transcription of HIV (human immunodeficiency virus), the cause of AIDS, is dependent on the activation of cyclin T-CDK9 by a small viral protein called Tat. Cells experimentally infected with tat mutants produce short viral transcripts about 50 nucleotides long. In contrast, cells infected with wild-type HIV synthesize long viral transcripts that encompass the entire integrated proviral genome (see Figure 5-48 and Figure 8-13). Thus Tat functions as an antitermination factor, permitting RNA polymerase II to read through a transcriptional block. (Tat is initially made by rare transcripts that fail to terminate when the HIV promoter is transcribed at a high rate in "activated" T-lymphocytes; see Chapter 23.) Tat is a sequence-specific RNA-binding protein. It binds to the RNA copy of a sequence called TAR, which forms a stemloop structure near the $5^{\prime}$ end of the HIV transcript (Figure 9-21). TAR also binds cyclin T, holding the cyclin T-CDK9 complex close to the polymerase, where it efficiently phosphorylates its substrates, resulting in transcription elongation. Chromatin immunoprecipitation assays done after treating cells with specific inhibitors of CDK9 indicate that the transcription of some 30 percent of mammalian genes is regulated by controlling the activity of cyclin T-CDK9 (P-TEFb), although this is probably done most frequently by sequence-specific DNA-binding transcription factors rather than by an RNA-binding protein, as in the case of HIV Tat.
![li2472239539b](li2472239539b.jpg)

FIGURE 9-21 Model of antitermination complex composed of HIV Tat protein and several cellular proteins. The TAR element in the HIV transcript contains sequences recognized by Tat and the cellular protein cyclin T. Cyclin T activates and helps position the protein kinase CDK9 near its substrates, the CTD of RNA polymerase II, NELF, and DSIF. CTD phosphorylation at serine 2 of the Pol II CTD heptad repeat is required for transcription elongation. Cellular proteins DSIF and the NELF complex are also involved in regulating Pol II elongation, as discussed in the text. See T. Wada et al., 1998, Gene Dev. 12:343; Y. Yamaguchi et al., 1999, Cell 97:451; and T. Yamada et al., 2006, Mol. Cell 21:227.

## KEY CONCEPTS OF SECTION 9.3

## RNA Polymerase II Promoters and General Transcription Factors

- RNA polymerase II initiates transcription of genes at the nucleotide in the DNA template that corresponds to the $5^{\prime}$ nucleotide that is capped in the encoded mRNA.
- Three principal types of promoter sequences have been identified in eukaryotic DNA. The TATA box is prevalent in highly transcribed genes. Initiator promoters are found in some genes, and CpG islands, the promoters for about 70 percent of protein-coding genes in vertebrates, are characteristic of genes transcribed at a low rate.
- Transcription of protein-coding genes by Pol II is initiated by sequential binding of the following in the indicated order: TFIID, which contains the TBP subunit that binds to TATA box DNA; TFIIA and TFIIB; a complex of Pol II and TFIIF; TFIIE; and finally, TFIIH (see Figure 9-19).
- The helicase activity of a TFIIH subunit helps to separate the DNA strands at the transcription start site in most promoters, a process that requires hydrolysis of ATP. As Pol II begins transcribing away from the start site, its CTD is phosphorylated on serine 5 by the TFIIH kinase domain.
- In metazoans, NELF and DSIF associate with Pol II after initiation, inhibiting elongation fewer than 100 bp from the transcription start site. Inhibition of elongation is relieved when cyclin T-CDK9 (also called P-TEFb) associates with the elongation complex and CDK9 phosphorylates subunits of NELF, DSIF, and serine 2 of the Pol II CTD.

### 9.4 Regulatory Sequences in ProteinCoding Genes and the Proteins Through Which They Function

As noted in the previous section, expression of eukaryotic protein-coding genes is regulated by multiple protein-binding DNA sequences, generically referred to as transcriptioncontrol regions. These regions include promoters and other types of control elements located near transcription start sites, as well as sequences located far from the genes they regulate. In this section, we take a closer look at the properties of various control elements found in eukaryotic proteincoding genes and the proteins that bind to them.

## Promoter-Proximal Elements Help Regulate Eukaryotic Genes

Recombinant DNA techniques have been used to systematically mutate the nucleotide sequences of various eukaryotic genes in order to identify transcription-control regions. The use of linker scanning mutagenesis, for example, can pinpoint the sequences within a regulatory region that function to control transcription. In this approach, a set of constructs with contiguous overlapping mutations are assayed for their effect on expression of a reporter gene or production of a specific mRNA (Figure 9-22a). This type of analysis
identified promoter-proximal elements of the thymidine kinase ( $t k$ ) gene from herpes simplex type I virus (HSV-I). The results demonstrated that the DNA region upstream of the HSV-I $t k$ gene contains three separate transcription-control sequences: a TATA box in the interval from -32 to -16 and two other control elements farther upstream (Figure 9-22b). Experiments using mutants containing single-base-pair changes in promoter-proximal control elements revealed that these elements are generally about 6-10 bp long. Recent results indicate that in human genes, they are found both upstream and downstream of the transcription start site at equal frequency. While, strictly speaking, the term promoter refers to the DNA sequence that determines where a polymerase initiates transcription, the term is often used to refer to both a promoter and its associated promoter-proximal control elements.

To test the spacing constraints on control elements in the HSV-I $t k$ promoter region identified by analysis of linker scanning mutations, researchers prepared and assayed constructs containing small deletions and insertions between the elements. Changes in spacing between the promoter and promoter-proximal control elements of 20 bp or fewer had little effect. However, insertions of $30-50 \mathrm{bp}$ between a HSV-I $t k$ promoter-proximal element and the TATA box was equivalent to deleting the element. Similar analyses of other eukaryotic promoters have also indicated that considerable flexibility in the spacing
(a)
![li2482239539b](li2482239539b.jpg)

EXPERIMENTAL FIGURE 9-22 Linker scanning mutations identify transcriptioncontrol elements. (a) In linker scanning mutagenesis, a region of eukaryotic DNA (tan) that supports high-level expression of a reporter gene (light purple) is cloned in a plasmid vector as diagrammed at the top. Overlapping linker scanning (LS) mutations (crosshatched areas) are introduced from one end of the region being analyzed to the other. These mutations are created by scrambling the nucleotide sequence in a short stretch of the DNA. After the mutant plasmids are transfected separately into cultured cells, the activity of the reportergene product is assayed. In the example shown here, the sequence from -120 to +1 of the herpes simplex virus thymidine kinase gene, LS mutations 1, 4, 6, 7, and 9 have little or no effect on expression of the reporter gene, indicating that the regions altered in these mutants contain no control elements. Reporter-gene expression is significantly reduced in mutants $2,3,5$, and 8 , indicating that control elements (brown) lie in the intervals shown at the bottom. (b) Analysis of these LS mutations identified a TATA box and two promoter-proximal elements (PE-1 and PE-2). See S. L. McKnight and R. Kingsbury, 1982, Science 217:316.

between promoter-proximal elements is generally tolerated, but that separations of several tens of base pairs may decrease transcription.

## Distant Enhancers Often Stimulate Transcription by RNA Polymerase II

As noted earlier, transcription from many eukaryotic promoters can be stimulated by control elements located thousands of base pairs away from the transcription start site. Such long-distance transcription-control elements, referred to as enhancers, are common in eukaryotic genomes but fairly rare in bacterial genomes. Procedures such as linker scanning mutagenesis have indicated that enhancers, usually on the order of 200 bp long, are, like promoter-proximal elements, composed of several functional sequence elements of about 6-10 bp each. As discussed later, each of these regulatory elements is a binding site for a sequence-specific DNAbinding transcription factor.

Analyses of many different metazoan enhancers have shown that they can occur with equal probability upstream from a promoter or downstream from a promoter within an intron, or even downstream from the final exon of a gene, as in the case of the SALL1 gene (see Figure 9-10a). Many enhancers are cell-type-specific. For example, an enhancer controlling Pax6 expression in the retina was characterized in the intron between exons 4 and 5 (see Figure 9-9a), whereas an enhancer controlling Pax6 expression in the
hormone-secreting cells of the pancreas is located in a roughly 200-bp region upstream of exon 0 (so named because it was discovered after the exon called "exon 1").

## Most Eukaryotic Genes Are Regulated by Multiple Transcription-Control Elements

Initially, enhancers and promoter-proximal elements were thought to be distinct types of transcription-control elements. However, as more enhancers and promoter-proximal elements were analyzed, the distinctions between them became less clear. For example, both types of elements can generally stimulate transcription even when inverted, and both types are often cell-type-specific. The general consensus now is that a spectrum of control elements regulates transcription by RNA polymerase II. At one extreme are enhancers, which can stimulate transcription from a promoter tens of thousands of base pairs away. At the other extreme are promoter-proximal elements, such as the upstream elements controlling the HSV-1 $t k$ gene, which lose their influence when moved $30-50 \mathrm{bp}$ farther from the promoter. Researchers have identified a large number of transcription-control elements that can stimulate transcription from distances between these two extremes.

Figure 9-23a summarizes the locations of transcriptioncontrol sequences for a hypothetical mammalian gene with a promoter containing a TATA box. The transcription start site encodes the first $\left(5^{\prime}\right)$ nucleotide of the first exon of an
![li2492239539b](li2492239539b.jpg)

FIGURE 9-23 General organization of control elements that regulate gene expression in multicellular eukaryotes and yeast.
(a) Mammalian genes with a TATA box promoter are regulated by promoter-proximal elements and enhancers. The promoter elements shown in Figure 9-16 position RNA polymerase II to initiate transcription at the start site and influence the rate of transcription. Enhancers may be either upstream or downstream and as far away as hundreds of kilobases from the transcription start site. In some cases, enhancers lie within introns. Promoter-proximal elements are found upstream and downstream of transcription start sites at equal frequency in mammalian genes. (b) For
mammalian genes with a CpG island promoter, transcription initiates at several sites in both the sense and antisense directions from the ends of the CpG-rich region. Transcripts in the sense direction are elongated and are processed into mRNAs by RNA splicing. These genes express mRNAs with alternative $5^{\prime}$ exons determined by the transcription start site. Genes with CpG island promoters contain promoter-proximal control elements. Currently, it is not clear whether they are also regulated by distant enhancers. (c) Most S. cerevisiae genes contain only one regulatory region, called an upstream activating sequence (UAS), and a TATA box, which is about 90 bp upstream from the transcription start site.

mRNA, the nucleotide that is capped. In addition to the TATA box at about -31 to -26 , promoter-proximal elements, which are relatively short ( 6-10 bp), are located within the first 200 bp either upstream or downstream of the start site. Enhancers, in contrast, are usually about $50-200 \mathrm{bp}$ long and are composed of multiple elements of about $6-10 \mathrm{bp}$. Enhancers may be located up to 50 kb or more upstream or downstream from the start site or within an intron. Like the Pax6 gene, many mammalian genes are controlled by multiple enhancer regions that function in different types of cells.

Figure 9-23b summarizes the promoter region of a mammalian gene with a CpG island promoter. About 70 percent of mammalian genes are expressed from CpG island promoters, usually at much lower levels than genes with TATA box promoters. Multiple alternative transcription start sites are used, generating mRNAs with alternative $5^{\prime}$ ends for the first exon derived from each start site. Transcription occurs in both directions, but Pol II molecules transcribing in the sense direction are elongated to 1 kb or more, much more efficiently than transcripts in the antisense direction.

In the important model organism Saccharomyces cerevisiae (budding yeast), genes are closely spaced (see Figure 8-4b), and few genes contain introns. In this organism, enhancers, which are referred to as upstream activating sequences (UASs), usually lie within 200 bp upstream of the promoters of the genes they regulate. Most yeast genes contain only one UAS. In addition, S. cerevisiae genes contain a TATA box about 90 bp upstream from the transcription start site (Figure 9-23c).

## DNase I Footprinting and EMSA Detect Protein-DNA Interactions

The various transcription-control elements found in eukaryotic DNA are binding sites for regulatory proteins called transcription factors. The simplest eukaryotic cells encode hundreds of transcription factors, and the human genome encodes at least 1400. The transcription of each gene in the genome is independently regulated by combinations of specific transcription factors that bind to its transcription-control regions. The number of possible combinations of this many transcription factors is astronomical, sufficient to generate unique controls for every gene encoded in the genome.

In yeast, Drosophila, and other genetically tractable eukaryotes, numerous genes encoding transcription activators and repressors have been identified by classical genetic analyses like those described in Chapter 6. However, in mammals and other vertebrates, which are less amenable to such genetic analysis, most transcription factors have been detected initially and subsequently purified by biochemical techniques. In this approach, a DNA regulatory element that has been identified by the kinds of mutational analyses described above is used to identify cognate proteins-those proteins that bind specifically
to it. Two common techniques for detecting such cognate proteins are DNase I footprinting and the electrophoretic mobility shift assay.

DNase I footprinting takes advantage of the fact that when a protein is bound to a region of DNA, it protects that DNA sequence from digestion by nucleases. As illustrated in Figure 9-24a, samples of a DNA fragment that has been labeled with a radioactive atom at one end of one strand are digested under carefully controlled conditions in the presence and absence of a DNA-binding protein, then denatured and electrophoresed, and the resulting gel is subjected to autoradiography. The region protected by the bound protein appears as a gap, or "footprint," in the array of bands resulting from digestion in the absence of the protein. When footprinting is performed with a DNA fragment containing a known transcription-control element, the appearance of a footprint indicates the presence of a transcription factor that binds that control element in the protein sample being assayed. Footprinting also identifies the specific DNA sequence to which the transcription factor binds.

For example, DNase I footprinting of the strong adenovirus late promoter shows a protected region over the TATA box when TBP is added to the labeled DNA before DNase I digestion (Figure 9-24b). DNase I does not digest all phosphodiester bonds in a duplex DNA at equal rate. Consequently, in the absence of added protein (lanes 1, 6 , and 9), a particular pattern of bands is observed that depends on the DNA sequence and results from cleavage at some phosphodiester bonds and not others. However, when increasing amounts of TBP are incubated with the end-labeled DNA before digestion with DNase I, TBP binds to the TATA box, and when sufficient TBP is added to bind all the labeled DNA molecules, it protects the region between about -35 and -20 from digestion (lanes 2-5). In contrast, increasing amounts of TFIID (lanes 7 and 8) protect not only the TATA box region, but also regions near $-7,+1$ to $+5,+10$ to +15 , and +20 from digestion, producing a different "footprint" from TBP. Results such as this tell us that other subunits of TFIID (the TBP-associated factors, or TAFs) also bind to the DNA in the region downstream from the TATA box.

The electrophoretic mobility shift assay (EMSA), also called the gel-shift or band-shift assay, is more useful than the footprinting assay for quantitative analysis of DNAbinding proteins. In general, the electrophoretic mobility of a DNA fragment is reduced when it is complexed with protein, causing a shift in the location of the fragment band. EMSA can be used to detect a transcription factor in protein fractions incubated with a radiolabeled DNA fragment (the probe) containing a known control element (Figure 9-25). The more transcription factor is added to the binding reaction, the more labeled probe is shifted to the position of the DNA-protein complex.

In the biochemical isolation of a transcription factor, an extract of cell nuclei is commonly subjected sequentially to several

![li2502239539b](li2502239539b.jpg)

EXPERIMENTAL FIGURE 9-24 DNase I footprinting reveals the region of a DNA sequence where a transcription factor binds. (a) A DNA fragment known to contain a transcription-control element is labeled at one end with ${ }^{32} \mathrm{P}$ (red dot). Portions of the labeled DNA sample are then digested with DNase I in the presence and in the absence of protein samples containing a sequence-specific DNAbinding protein. DNase I hydrolyzes the phosphodiester bonds of DNA between the $3^{\prime}$ oxygen on the deoxyribose of one nucleotide and the $5^{\prime}$ phosphate of the next nucleotide. A low concentration of DNase I is used so that, on average, each DNA molecule is cleaved just once (vertical arrows). If the protein sample does not contain a protein that binds to a specific sequence in the labeled DNA, the DNA fragment is cleaved at multiple positions between the labeled and unlabeled ends of the original fragment, as in sample A (left). If the protein sample does contain such a protein, as in sample B (right), the protein binds to its
types of liquid chromatography (see Chapter 3). Fractions eluted from the columns are assayed by DNase I footprinting or EMSA using DNA fragments containing an identified regulatory element (see Figure 9-22). Fractions containing a protein that binds to the regulatory element in these assays contain a putative transcription factor. A powerful technique that is commonly used for the final step in purifying transcription factors is sequence-specific DNA affinity chromatography, a particular type of affinity chromatography in which long DNA strands containing multiple copies of the transcription-factorbinding site are coupled to a column matrix.

Once a transcription factor has been isolated and purified, its partial amino acid sequence can be determined and used to clone the gene or cDNA encoding it, as outlined in
cognate sequence in the DNA, thereby protecting a portion of the fragment from digestion. Following DNase treatment, the DNA is separated from protein, denatured to separate the strands, and electrophoresed. Autoradiography of the resulting gel detects only labeled strands and reveals fragments extending from the labeled end to the site of cleavage by DNase I. Cleavage fragments containing the transcription-control element show up on the gel for sample $A$ but are missing in sample $B$ because the bound cognate protein has blocked cleavages within that sequence and thus production of the corresponding fragments. The missing bands on the gel constitute the footprint. (b) Footprints produced by increasing amounts of TBP (indicated by the triangle) and of TFIID on the strong adenovirus major late promoter. [Part (b) from Zhou, Q. et al., "Holo-TFIID supports transcriptional stimulation by diverse activators and from a TATA-less promoter," Genes \& Development, 11/1992; 6(10):1964-74; republished with permission from Cold Spring Harbor Laboratory Press.]

Chapter 6. The isolated gene can then be used to test the ability of the encoded protein to activate or repress transcription in an in vivo transfection assay (Figure 9-26).

## Activators Are Composed of Distinct Functional Domains

Studies with a yeast transcription activator called Gal4 provided early insight into the domain structure of transcription factors. The gene encoding Gal4, which promotes expression of enzymes needed to metabolize galactose, was identified by complementation analysis of gal4 mutants that cannot form colonies on an agar medium in which galactose is the only source of carbon and energy (see Chapter 6).

![li2512239539b](li2512239539b.jpg)

EXPERIMENTAL FIGURE 9-25 The electrophoretic mobility shift assay can be used to detect transcription factors during purification. In this example, protein fractions separated by column chromatography were assayed for their ability to bind to a radiolabeled DNA-fragment probe containing a known regulatory element. After an aliquot of the protein sample was loaded onto the column (ON) and successive column fractions (numbers) were incubated with the labeled probe, the samples were electrophoresed under conditions that do not disrupt protein-DNA interactions. The free probe not bound to protein migrated to the bottom of the gel. A protein in the preparation applied to the column and in fractions 7 and 8 bound to the probe, forming a DNA-protein complex that migrated more slowly than the free probe. These fractions are therefore likely to contain the regulatory protein being sought. [From Yoshinaga, S. et al., "Purification and characterization of transcription factor IIIC2," J. Biol. Chem., 1989, 264:10726 (01989 American Society for Biochemistry and Molecular Biology.]

Directed mutagenesis studies like those described previously identified UASs for the genes activated by Gal4. Each of these UASs was found to contain one or more copies of a 17-bp sequence called UAS ${ }_{G A L}$. DNase I footprinting assays with recombinant Gal4 protein produced in E. coli from the yeast GAL4 gene showed that Gal4 binds to UAS ${ }_{G A L}$ sequences. When a copy of UAS ${ }_{G A L}$ was cloned upstream of a TATA box followed by a $\beta$-galactosidase reporter gene, and that construct was introduced into yeast cells, expression of $\beta$-galactosidase was activated in galactose media in wildtype cells, but not in gal4 mutants. These results showed that UAS $_{G A L}$ is a transcription-control element activated by the Gal4 transcription factor in galactose media.

A remarkable set of experiments with gal4 deletion mutants demonstrated that the Gal4 transcription factor is composed of separable functional domains: an N-terminal DNA-binding domain, which binds to specific DNA sequences, and a C-terminal activation domain, which interacts with other proteins to stimulate transcription from a nearby promoter (Figure 9-27). When the N-terminal DNA-binding domain of Gal4 was fused directly to various portions of its own C-terminal region, deleting internal sequences, the resulting truncated proteins retained the ability to stimulate expression of a reporter gene in an in vivo assay like that depicted in Figure 9-26. Thus the internal portion of the protein is not required for the functioning of Gal4 as a transcription factor. Similar experiments with another yeast activator, Gcn4, which regulates genes required for the synthesis of many amino acids, indicated that it contains a roughly 50-amino-acid DNA-binding domain at its C-terminus and a roughly 20-amino-acid activation domain near the middle of its sequence.

Further evidence for the existence of distinct activation domains in Gal4 and Gcn4 came from experiments in which their activation domains were fused to a DNA-binding domain from an entirely unrelated E. coli DNA-binding protein. When these fusion proteins were assayed in vivo, they activated transcription of a reporter gene containing the cognate site for the E. coli protein. Thus functional transcription factors can be constructed from entirely novel combinations of prokaryotic and eukaryotic elements.

Studies such as these have now been carried out with many eukaryotic transcription factors. The structural model of eukaryotic activators that has emerged from these studies is a modular one in which one or more activation domains are connected to a sequence-specific DNA-binding domain by intrinsically disordered, flexible protein domains (Figure 9-28). In some cases, amino acids included in the DNA-binding domain also contribute to transcriptional activation. As discussed in a later section, activation domains

![li2522239539b](li2522239539b.jpg)

EXPERIMENTAL FIGURE 9-26 An in vivo transfection assay measures transcription activity to evaluate proteins believed to be transcription factors. The assay system requires two plasmids. One plasmid contains the gene encoding the putative transcription factor (protein X). The second plasmid contains a reporter gene (e.g., luciferase) and one or more binding sites for protein $X$. Both plasmids are simultaneously introduced into cells that lack the gene encoding protein $X$. The production of reporter-gene RNA transcripts is measured; alternatively, the activity of the encoded protein can be assayed. If reporter-gene transcription is greater in the presence of the X-encoding plasmid than in its absence, then the protein is an activator; if transcription is less, then it is a repressor. By use of plasmids encoding a mutated or rearranged transcription factor, important domains of the protein can be identified.

(a) Reporter-gene construct
![li2532239539b](li2532239539b.jpg)

EXPERIMENTAL FIGURE 9-27 Deletion mutants of the GAL4 gene in yeast with a UAS ${ }_{\text {GAL }}$ reporter-gene construct demonstrate the separate functional domains in a transcription activator.
(a) Diagram of DNA construct containing a lacZ reporter gene (encoding $\beta$-galactosidase) and TATA box ligated to $\mathrm{UAS}_{\mathrm{GAL}}$, a regulatory element that contains several Gal4-binding sites. The reporter-gene construct and DNA encoding wild-type or mutant (deleted) Gal4 were simultaneously introduced into mutant (gal4) yeast cells, and the activity of $\beta$-galactosidase expressed from lacZ was assayed. Activity should be high if the introduced GAL4 DNA encodes a functional protein. (b) Schematic diagrams of wild-type Gal4 and various mutant forms. Small numbers refer to positions in the wild-type sequence. Deletion of 50 amino acids
from the N-terminal end destroyed the ability of Gal4 to bind to UAS ${ }_{\text {GAL }}$ and to stimulate expression of $\beta$-galactosidase from the reporter gene. Proteins with extensive deletions from the C-terminal end still bound to UAS $_{\text {GAL }}$. These results localize the DNA-binding domain to the N-terminal end of Gal4. The ability to activate $\beta$-galactosidase expression was not entirely eliminated unless somewhere between 126 and 189 or more amino acids were deleted from the C-terminal end. Thus the activation domain lies in the C-terminal region of Gal4. Proteins with internal deletions (bottom) were also able to stimulate expression of $\beta$-galactosidase, indicating that the central region of Gal4 is not crucial for its function in this assay. See J. Ma and M. Ptashne, 1987, Cell 48:847; I. A. Hope and K. Struhl, 1986, Cell 46:885; and R. Brent and M. Ptashne, 1985, Cell 43:729.
are thought to function by binding other proteins involved in transcription. The presence of flexible, intrinsically disordered protein domains (see Figure 3-8) connecting the DNA-binding domain to the activation domains may explain why alterations in the spacing between control elements are so well tolerated in eukaryotic control regions. Thus even when the positions of transcription factors bound to DNA are shifted relative to each other, their activation domains may still be able to interact because they are attached to their DNA-binding domains through flexible protein regions.

## Repressors Are the Functional Converse of Activators

Eukaryotic transcription is regulated by repressors as well as activators. For example, geneticists have identified mutations
in yeast that result in continuously high expression of certain genes. This type of unregulated, abnormally high expression, called constitutive expression, results from the inactivation of a repressor that normally inhibits the transcription of these genes. Similarly, mutants of Drosophila melanogaster and Caenorhabditis elegans have been isolated that are defective in embryonic development because they express genes in embryonic cells where those genes are normally repressed. The mutations in these mutants inactivate repressors, leading to abnormal development.

Repressor-binding sites in DNA have been identified by systematic linker scanning mutation analyses similar to the one depicted in Figure 9-22. In this type of analysis, whereas mutation of an activator-binding site leads to decreased expression of the linked reporter gene, mutation of a repressorbinding site leads to increased expression of a reporter gene.

![li2542239539b](li2542239539b.jpg)

FIGURE 9-28 Schematic diagrams illustrating the modular structure of eukaryotic transcription activators. Transcription factors may contain more than one activation domain but rarely contain more than one DNA-binding domain. Gal4 and Gcn4 are yeast transcription activators. The glucocorticoid receptor (GR) promotes transcription of target genes when certain hormones are bound to the C-terminal activation domain. SP1 binds to GC-rich promoter elements in a large number of mammalian genes.

The repressor proteins that bind such sites can be purified and assayed using the same biochemical techniques described earlier for activator proteins.

Eukaryotic transcription repressors are the functional converse of activators. They can inhibit transcription of a gene they do not normally regulate when their cognate binding sites are placed within tens of base pairs to many kilobases of the gene's transcription start site. Like activators, most eukaryotic repressors are modular proteins that have two functional domains: a DNA-binding domain and a repression domain. Like activation domains, repression domains continue to function when fused to another type of DNA-binding domain. If binding sites for this second DNAbinding domain are inserted within a few hundred base pairs of a promoter, expression of the fusion protein inhibits transcription from the promoter. Also like activation domains, repression domains function by interacting with other proteins, as discussed later in this chapter.

## DNA-Binding Domains Can Be Classified into Numerous Structural Types

The DNA-binding domains of eukaryotic transcription factors contain a variety of structural motifs that bind specific DNA sequences. The ability of DNA-binding proteins to bind to specific DNA sequences commonly results from noncovalent interactions between atoms in an $\alpha$ helix in the DNA-binding domain and atoms on the edges of the bases within the major groove in the DNA. Ionic interactions between positively charged residues arginine and lysine and
negatively charged phosphates in the sugar-phosphate backbone, and in some cases, interactions with atoms in the DNA minor groove, also contribute to binding.

The principles of specific protein-DNA interactions were first discovered during the study of bacterial repressors. Many bacterial repressors are dimeric proteins in which an $\alpha$ helix from each monomer inserts into the major groove in the DNA helix and makes multiple, specific interactions with the atoms there (Figure 9-29). This $\alpha$ helix is referred to as the recognition helix or sequence-reading helix because most of the amino acid side chains that contact bases in the DNA extend from this helix. The recognition helix, which protrudes from the surface of a bacterial repressor, is usually supported in the protein structure in part by hydrophobic interactions with a second $\alpha$ helix just N -terminal to it. This entire structural element, which is present in many bacterial repressors, is called a helix-turnhelix motif.

Many additional structural motifs that can present an $\alpha$ helix to the major groove of DNA are found in eukaryotic transcription factors, which are often classified according to the type of DNA-binding domain they contain. Because most of these motifs have characteristic consensus amino acid sequences, potential transcription factors can be recognized among the cDNA sequences from various tissues that have
![li2552239539b](li2552239539b.jpg)

FIGURE 9-29 Interaction of bacteriophage 434 repressor with DNA. Ribbon diagram of 434 repressor bound to its specific operator DNA. The recognition helices are shown in green. The $\alpha$ helices N -terminal to the recognition helix and the turn in the polypeptide backbone between the helices in the helix-turn-helix structural motif are shown in yellow and red, respectively. The protein interacts intimately with one side of the DNA molecule over a length of 1.5 turns. [Data from A. K. Aggarwal et al., 1988, Science 242:899, PDB ID 2ori.]

been characterized in humans and other species. Here we introduce several common classes of DNA-binding proteins whose three-dimensional structures have been determined. In all these examples, and in many other transcription factors, at least one $\alpha$ helix is inserted into the major groove of DNA. However, some transcription factors contain alternative structural motifs (e.g., $\beta$ strands and loops; see NFAT in Figure 9-33 below as an example) that interact with DNA.

Homeodomain Proteins Many eukaryotic transcription factors that function during development contain a conserved 60-residue DNA-binding motif, called a homeodomain, that is similar to the helix-turn-helix motif of bacterial repressors. These transcription factors were first identified in Drosophila mutants in which one body part was transformed into another during development (see Figure 9-2b). The conserved homeodomain sequence has also been found in vertebrate transcription factors, including those that have similar master-control functions in human development.

Zinc-Finger Proteins A number of different eukaryotic proteins have regions that fold around a central $\mathrm{Zn}^{2+}$ ion, producing a compact domain from a relatively short length of polypeptide chain. Termed a zinc finger, this structural motif was first recognized in DNA-binding domains, but is now known to occur
in other proteins that do not bind to DNA. Here we describe two of the several classes of zinc-finger motifs that have been identified in eukaryotic transcription factors.

The $\mathrm{C}_{2} \mathrm{H}_{2}$ zinc finger is the most common DNA-binding motif encoded in the human genome and the genomes of other mammals. It is also common in multicellular plants, but is not the dominant type of DNA-binding domain in plants, as it is in animals. This motif has a 23-26-residue consensus sequence containing two conserved cysteine (C) and two conserved histidine $(\mathrm{H})$ residues, whose side chains bind one $\mathrm{Zn}^{2+}$ ion (see Figure 3-10c). The name "zinc finger" was coined because a two-dimensional diagram of the structure resembles a finger. When the three-dimensional structure was solved, it became clear that the binding of the $\mathrm{Zn}^{2+}$ ion by the two cysteine and two histidine residues folds the relatively short polypeptide sequence into a compact domain, which can insert its $\alpha$ helix into the major groove of DNA. Many transcription factors contain multiple $\mathrm{C}_{2} \mathrm{H}_{2}$ zinc fingers, which interact with successive groups of base pairs, within the major groove, as the protein wraps around the DNA double helix (Figure 9-30a).

A second type of zinc-finger structure, designated the $C_{4}$ zinc finger (because it has four conserved cysteines in contact with the $\mathrm{Zn}^{2+}$ ), is found in some 50 human
![li2562239539b](li2562239539b.jpg)

FIGURE 9-30 Eukaryotic DNA-binding domains that use an $\alpha$ helix to interact with the major groove of specific DNA sequences. (a) The GL1 DNA-binding domain is monomeric and contains five $\mathrm{C}_{2} \mathrm{H}_{2}$ zinc fingers. The $\alpha$ helices are shown as cylinders, the $\mathrm{Zn}^{2+}$ ions as spheres. Finger 1 does not interact with DNA, whereas the other four fingers do. (b) The glucocorticoid receptor is a homodimeric $\mathrm{C}_{4}$ zinc-finger protein, one monomer in green, one in yellow. The $\alpha$ helices are shown as cylinders, the $\beta$ strands as white arrows, the $\mathrm{Zn}^{2+}$ ions as spheres. Two $\alpha$ helices (darker shade), one in each monomer, interact with the DNA. Like all $\mathrm{C}_{4}$ zinc-finger homodimers, this transcription factor has twofold rotational symmetry. (c) In leucine-zipper proteins, basic residues in the extended $\alpha$-helical regions of the monomers interact with the DNA backbone at adjacent sites in the major groove. The coiled-coil dimerization domain is stabilized by hydrophobic interactions between the monomers. (d) In bHLH proteins, the DNA-binding helices at the right ( N -termini of the monomers) are separated by nonhelical loops from a leucine zipper-like region containing a coiled-coil dimerization domain. [Part (a), see N. P. Pavletich and C. O. Pabo, 1993, Science 261:1701, PDB ID 2gll. Part (b), see B. F. Luisi et al., 1991, Nature 352:497 PDB ID 1glu. Part (c), data from T. E. Ellenberger et al., 1992, Cell 71:1223, PDB ID 1ysa. Part (d), data from P. Brownlie et al., 1997, Structure 5:509, PDB ID 1hlo.]

transcription factors. The first members of this class were identified as specific intracellular high-affinity binding proteins, or "receptors," for steroid hormones, which led to the name steroid receptor superfamily. Because similar intracellular receptors for nonsteroid hormones were subsequently found, these transcription factors are now commonly called nuclear receptors. The characteristic feature of $\mathrm{C}_{4}$ zinc fingers is the presence of two groups of four critical cysteines, one toward each end of the 55-56-residue domain. Although the $\mathrm{C}_{4}$ zinc finger was initially named by analogy with the $\mathrm{C}_{2} \mathrm{H}_{2}$ zinc finger, the three-dimensional structures of proteins containing these DNA-binding motifs were later found to be quite distinct. A particularly important difference between the two is that $\mathrm{C}_{2} \mathrm{H}_{2}$ zinc-finger proteins generally contain three or more repeating finger units and bind as monomers, whereas $\mathrm{C}_{4}$ zinc-finger proteins generally contain only two finger units and generally bind to DNA as homodimers or heterodimers. Homodimers of $\mathrm{C}_{4}$ zinc-finger DNA-binding domains have twofold rotational symmetry (Figure 9-30b). Consequently, homodimeric nuclear receptors bind to consensus DNA sequences that are inverted repeats.

Leucine-Zipper Proteins Another structural motif present in the DNA-binding domains of a large class of transcription factors contains the hydrophobic amino acid leucine at every seventh position in the sequence. These proteins bind to DNA as dimers, and mutagenesis of the leucines showed that they were required for dimerization. Consequently, the name leucine zipper was coined to denote this structural motif of a coiled coil of two $\alpha$ helixes.

The DNA-binding domain of the yeast Gcn4 transcription factor mentioned earlier is a leucine-zipper domain. X-ray crystallographic analysis of complexes between DNA and the Gcn4 DNA-binding domain has shown that the dimeric protein contains two extended $\alpha$ helices that "grip" the DNA molecule, much like a pair of scissors, at two adjacent sites in the major groove separated by about half a turn of the double helix (Figure 9-30c). The portions of the $\alpha$ helices contacting the DNA include positively charged (basic) residues that interact with phosphates in the DNA backbone and additional residues that interact with specific bases in the major groove.

Gcn4 forms dimers via hydrophobic interactions between the C-terminal regions of the $\alpha$ helices, forming a coiled-coil structure. This structure is common in proteins containing amphipathic $\alpha$ helices in which hydrophobic amino acid residues are regularly spaced alternately three or four positions apart in the sequence, forming a stripe down one side of the $\alpha$ helix. These hydrophobic stripes make up the interacting surfaces between the $\alpha$-helical monomers in a coiled-coil dimer (see Figure 3-10a).

Although the first leucine-zipper transcription factors to be analyzed contained leucine residues at every seventh position in the dimerization region, additional DNA-binding proteins containing other hydrophobic amino acids in these positions were subsequently identified. Like leucine-zipper proteins, they form dimers containing a C-terminal coiledcoil dimerization region and an N-terminal DNA-binding
domain. The term basic zipper (bZIP) is now frequently used to refer to all proteins with these common structural features. Many basic-zipper transcription factors are heterodimers of two different polypeptide chains, each containing one basic-zipper domain.

Basic Helix-Loop-Helix (bHLH) Proteins The DNA-binding domain of another class of dimeric transcription factors contains a structural motif that is very similar to the basiczipper motif except that a nonhelical loop of the polypeptide chain separates two $\alpha$-helical regions in each monomer (Figure 9-30d). Termed a basic helix-loop-helix (bHLH), this motif was predicted from the amino acid sequences of these proteins, which contain an N -terminal $\alpha$ helix with basic residues that interact with DNA, a middle loop region, and a C-terminal region, with hydrophobic amino acids spaced at intervals characteristic of an amphipathic $\alpha$ helix, that dimerizes into a coiled coil. As with basic-zipper proteins, different bHLH proteins can form heterodimers.

## Structurally Diverse Activation and Repression Domains Regulate Transcription

Experiments with fusion proteins composed of the Gal4 DNA-binding domain and random segments of E. coli proteins demonstrated that a diverse group of amino acid sequences ( $\sim 1$ percent of all E. coli sequences) can function as activation domains, even though they evolved to perform other functions. Many transcription factors contain activation domains marked by an unusually high percentage of particular amino acids. Gal4, Gcn4, and most other yeast transcription factors, for instance, have activation domains that are rich in acidic amino acids (aspartic and glutamic acids). These so-called acidic activation domains are generally capable of stimulating transcription in nearly all types of eukaryotic cells-fungal, animal, and plant cells. Activation domains from some Drosophila and mammalian transcription factors are glutamine-rich, and some are proline-rich; still others are rich in the closely related amino acids serine and threonine, both of which have hydroxyl groups. However, some strong activation domains are not particularly rich in any specific amino acid.

Biophysical studies indicate that acidic activation domains have an unstructured, random-coil, intrinsically disordered conformation. These domains stimulate transcription when they are bound to a protein co-activator. The interaction with a co-activator causes the activation domain to assume a more structured $\alpha$-helical conformation in the activation domain-co-activator complex. A well-studied example of a transcription factor with an acidic activation domain is the mammalian CREB protein, which is phosphorylated in response to increased levels of cAMP. This regulated phosphorylation is required for CREB to bind to its co-activator CBP (CREB-binding protein), resulting in the transcription of genes whose control regions contain a CREB-binding site (see Figure 15-30). When the phosphorylated random-coil activation domain of CREB interacts with CBP, it undergoes a conformational change to form two $\alpha$ helices linked by a

short loop, which wrap around the interacting domain of CBP (Figure 9-31a).

Some activation domains are larger and more highly structured than acidic activation domains. For example, the ligand-binding domains of nuclear receptors function as activation domains when they bind their specific hormone ligand (Figure 9-31b, c). Binding of ligand induces a large
![li2572239539b](li2572239539b.jpg)

FIGURE 9-31 Activation domains may be random coils until they interact with co-activator proteins or folded protein domains. (a) The acidic activation domain of CREB (cyclic AMP response element-binding protein) is activated by phosphorylation at serine 123. It is a random coil until it interacts with a domain of its co-activator, CBP (shown as a spacefilling surface model with negatively charged regions in red and positively charged regions in blue). When the CREB activation domain binds to CBP, it folds into two amphipathic $\alpha$ helices. Side chains in the activation domain that interact with the surface of the CBP domain are labeled. (b) The ligand-binding activation domain of the estrogen receptor is a folded-protein domain. When estrogen is bound to the domain, the green $\alpha$ helix interacts with the ligand, generating a hydrophobic groove in the ligand-binding domain (dark brown helices), which binds an amphipathic $\alpha$ helix in a co-activator subunit (blue). (c) The conformation of the estrogen receptor in the absence of hormone is stabilized by binding of the estrogen antagonist tamoxifen. In this conformation, the green helix of the receptor folds into a conformation that interacts with the co-activatorbinding groove of the active receptor, sterically blocking binding of coactivators. [Part (a) data from I. Radhakrishnan et al., 1997, Cell 91:741, PDB ID 1kdx. Parts (b) and (c) data from A. K. Shiau et al., 1998, Cell 95:927, PDB ID 3erd and 3ert.]
conformational change in the nuclear receptor that allows the ligand-binding domain with bound hormone to interact with a short $\alpha$ helix in a co-activator; the resulting complex can then activate transcription of genes whose control regions bind the nuclear receptor.

Thus the acidic activation domain in CREB and the ligand-binding activation domains in nuclear receptors represent two structural extremes. The CREB acidic activation domain is an intrinsically disordered random coil that folds into two $\alpha$ helices when it binds to the surface of a globular domain in a co-activator. In contrast, the nuclear-receptor ligand-binding activation domain is a structured globular domain that interacts with a short $\alpha$ helix in a co-activator, which probably is a random coil before it is bound. In both cases, however, specific proteinprotein interactions between a co-activator and the activation domain permit the transcription factor to stimulate gene expression.

Currently, less is known about the structure of repression domains. The globular ligand-binding domains of some nuclear receptors function as repression domains in the absence of their specific hormone ligand. Like activation domains, repression domains may be relatively short, comprising 15 or fewer amino acids. Biochemical and genetic studies indicate that repression domains also mediate protein-protein interactions and bind to co-repressor proteins, forming a complex that inhibits transcription initiation by mechanisms that are discussed later in the chapter.

## Transcription Factor Interactions Increase Gene-Control Options

Two types of DNA-binding proteins discussed previouslybZIP and bHLH proteins-often exist in alternative heterodimeric combinations of monomers. Other classes of transcription factors not discussed here also form heterodimeric proteins. In some heterodimeric transcription factors, each monomer recognizes the same sequence. In these cases, the formation of alternative heterodimers does not increase the number of different sites on which the monomers can act, but rather allows the activation domains associated with each monomer to be brought together in alternative combinations that bind to the same site (Figure 9-32a). As we will see later, and in subsequent chapters, the activities of individual transcription factors can be regulated by multiple mechanisms. Consequently, a single bZIP- or bHLH-binding DNA regulatory element in the transcription-control region of a gene may elicit different transcriptional responses depending on which bZIP or bHLH monomers are expressed in the cell and how their activities are regulated.

In some heterodimeric transcription factors, however, each monomer has a different DNA-binding specificity. The resulting combinatorial possibilities increase the number of potential DNA sequences that a family of transcription factors can bind. Three different transcription-factor monomers could theoretically combine to form six different homo- and heterodimeric transcription factors, as illustrated in Figure 9-32b. Four different monomers could form a total of ten

![li2582239539b](li2582239539b.jpg)

FIGURE 9-32 Combinatorial possibilities due to formation of heterodimeric transcription factors. (a) In some heterodimeric transcription factors, the activation domain of each monomer recognizes the same DNA sequence. In the hypothetical example shown, transcriptionfactor monomers $A, B$, and $C$ can all interact with one another, creating six different alternative combinations of activation domains that can all bind at the same site. Each composite binding site is divided into two half-sites, and each heterodimeric factor contains the activation domains of its two constituent monomers. (b) When transcription-factor monomers recognize different DNA sequences, six alternative combinations of the transcription-factor monomers $A, B$, and $C$, each with a unique pair of activation domains, can bind to six different DNA sequences (sites 1-6). (c) Expression of an inhibitory factor (red) that interacts only with the dimerization domain of $A$ inhibits binding; hence transcriptional activation at sites 1,4 , and 5 is inhibited, but activation at sites 2,3 , and 6 is unaffected.
dimeric factors; five monomers, sixteen dimeric factors; and so forth. In addition, inhibitory factors are known that bind to some bZIP and bHLH monomers, thereby blocking their binding to DNA. When these inhibitory factors are expressed, they repress transcriptional activation by the factors with which they interact (Figure 9-32c). Thus the rules governing the interactions of members of a heterodimeric transcription factor family are complex. This combinatorial complexity expands both the number of DNA sites from which these factors can activate transcription and the ways in which they can be regulated.

Similar combinatorial transcription regulation is achieved through the interaction of structurally unrelated
transcription factors bound to closely spaced binding sites in DNA. An example is the interaction of two transcription factors, NFAT and AP1, that bind to neighboring sites in a composite promoter-proximal element regulating the gene encoding interleukin-2 (IL-2). Expression of the IL-2 gene is critical to the immune response, but abnormal expression of IL-2 can lead to autoimmune diseases such as rheumatoid arthritis (see Chapter 23). Neither NFAT nor AP1 binds to its site in the $I L-2$ control region in the absence of the other. The affinities of these factors for these particular DNA sequences are too low for the individual factors to form a stable complex with DNA. However, when both NFAT and AP1 are present, protein-protein interactions between them stabilize the ternary complex composed of NFAT, AP1, and DNA (Figure 9-33a). Such cooperative DNA binding by various transcription factors results in considerable combinatorial complexity of transcriptional control. As a result, the 1400 or so transcription factors encoded in the human genome can bind to DNA through a much larger number of cooperative interactions, resulting in unique transcriptional control for each of the roughly 21,000 human genes. In the case of $I L-2$, transcription occurs only when NFAT is activated, which results in its transport from the cytoplasm to the nucleus, and the two subunits of AP1 are synthesized. These two events are controlled by distinct signal transduction pathways (see Chapters 15 and 16), allowing stringent control of IL-2 expression.

Cooperative binding by NFAT and AP1 occurs only when their weak binding sites are positioned quite close to each other in DNA. The sites must be located at a precise distance from each other for effective binding. The requirements for cooperative binding are not so stringent in the case of some other transcription factors and transcription-control regions. For example, the $E G R-1$ control region contains a composite binding site to which the SRF and SAP1 transcription factors bind cooperatively (Figure 9-33b). Because SAP1 has a long, flexible domain that interacts with SRF, the two proteins can bind cooperatively when their individual sites in DNA are separated by any distance up to about 30 bp or are inverted relative to each other.

## Multiprotein Complexes Form on Enhancers

As noted previously, enhancers generally range in length from about 50 to 200 bp and include binding sites for several transcription factors. Analysis of the roughly 50-bp enhancer that regulates expression of $\beta$-interferon, an important protein in defense against viral infections in vertebrates, provides a good example of the structure of the DNA-binding domains of several transcription factors bound to the several transcription-factor-binding sites that constitute an enhancer (Figure 9-34). The term enhanceosome has been coined to describe such large DNA-protein complexes that assemble from transcription factors as they bind to the multiple binding sites in an enhancer.

Because of the presence of flexible regions connecting the DNA-binding domains and activation or repression domains in transcription factors (see Figure 9-28), and because

![li2592239539b](li2592239539b.jpg)

FIGURE 9-33 Cooperative binding of two unrelated transcription factors to neighboring sites in a composite control element. (a) By themselves, both monomeric NFAT and heterodimeric AP1 transcription factors have low affinity for their respective binding sites in the IL-2 promoter-proximal region. Protein-protein interactions between NFAT and AP1 add to the overall stability of the NFAT-AP1-DNA complex, so that the two proteins bind to the composite site cooperatively. (b) Cooperative DNA binding by dimeric SRF and monomeric SAP1 can occur when their binding sites are separated by 5-30 bp and when the SAP1 binding site is inverted because the domain of SAP1 that interacts with SRF is connected to the DNA-binding domain of SAP1 by a flexible linker region of the SAP1 polypeptide chain (dotted line). [Part (a) data from L. Chen et al., 1998, Nature 392:42, PDB ID 1al02; part (b) data from M. Hassler and T. J. Richmond, 2001, EMBO J. 20:3018, PDB ID 1hbx.]
of the ability of interacting proteins bound to distant sites to produce loops in the DNA between their binding sites (see Figure 9-5), considerable leeway in the spacing between regulatory elements in transcription-control regions is permissible. This tolerance for variable spacing between binding sites for specific transcription factors, and between promoter binding sites for the general transcription factors and for Pol II, probably contributed to rapid evolution of gene
control in eukaryotes. Transposition of DNA sequences and recombination between repeated sequences over evolutionary time probably created new combinations of control elements that were subjected to natural selection and retained if they proved beneficial. The latitude in spacing between regulatory elements probably allowed many more functional combinations to be subjected to this evolutionary experimentation than would be the case if constraints on the spacing between regulatory elements were strict, as for most genes in bacteria.
![li2602239539b](li2602239539b.jpg)

FIGURE 9-34 Model of the enhanceosome that forms on the $\beta$-interferon enhancer. Two heterodimeric factors, Jun/ATF-2 and p50/RelA (NF-кB), and two copies each of the monomeric transcription factors IRF-3 and IRF-7, bind to the six overlapping binding sites in this enhancer. See D. Penne, T. Manniatis, and S. Harrison, 2007, Cell 129:1111.

## KEY CONCEPTS OF SECTION 9.4

## Regulatory Sequences in Protein-Coding Genes and the Proteins Through Which They Function

- Expression of eukaryotic protein-coding genes is generally regulated through multiple protein-binding transcriptioncontrol regions that are located close to or distant from the transcription start site (see Figure 9-23).
- Promoters direct binding of RNA polymerase II to DNA, determine the site of transcription initiation, and influence the frequency of transcription initiation.
- Promoter-proximal elements occur within about 200 bp of a start site. Several such elements, containing 6-10 bp, may help regulate a particular gene.

- Enhancers, which contain multiple short control elements, may be located from 200 bp to tens of kilobases upstream or downstream from a promoter, within an intron, or downstream from the final exon of a gene.
- Promoter-proximal elements and enhancers are often cell-type-specific, functioning only in specific differentiated cell types.
- Transcription factors, which activate or repress transcription, bind to promoter-proximal regulatory elements and enhancers in eukaryotic DNA.
- Transcription activators and repressors are generally modular proteins containing a single DNA-binding domain and one or a few activation domains (for activators) or repression domains (for repressors). The different domains are frequently linked by flexible, intrinsically disordered polypeptide regions (see Figure 9-28).
- Among the most common structural motifs found in the DNA-binding domains of eukaryotic transcription factors are the homeodomain, C2H2 zinc finger, basic zipper (leucine zipper), and basic helix-loop-helix (bHLH). All these and many other DNA-binding motifs contain one or more $\alpha$ helices that interact with the major groove in their cognate site in DNA.
- Activation and repression domains in transcription factors exhibit a variety of amino acid sequences and threedimensional structures. In general, these functional domains interact with co-activators or co-repressors, which are critical to the ability of transcription factors to modulate gene expression.
- The transcription-control regions of most genes contain binding sites for multiple transcription factors. Transcription of such genes varies depending on the particular repertoire of transcription factors that are expressed and activated in a particular cell at a particular time.
- Combinatorial complexity in transcriptional control results from alternative combinations of monomers that form heterodimeric transcription factors (see Figure 9-32) and from cooperative binding of transcription factors to composite control sites (see Figure 9-33).
- Binding of multiple transcription factors to multiple sites in an enhancer forms a DNA-protein complex called an enhanceosome (see Figure 9-34).


### 9.5 Molecular Mechanisms of Transcription Repression and Activation

The repressors and activators that bind to specific sites in DNA and regulate expression of the associated protein-coding genes do so by three general mechanisms. First, these regulatory proteins act in concert with other proteins to modulate
chromatin structure, inhibiting or stimulating the ability of general transcription factors to bind to promoters. Recall from Chapter 8 that the DNA in eukaryotic cells is not free, but is associated with a roughly equal mass of protein in the form of chromatin. The basic structural unit of chromatin is the nucleosome, which is composed of about 147 bp of DNA wrapped tightly around a disk-shaped core of histone proteins. Residues within the N-terminal region of each histone, and the C-terminal regions of histones H2A and H 2 B , called histone tails, extend from the surface of the nucleosome and can be reversibly modified (see Figure $8-26 b$ ). Such modifications influence the relative condensation of chromatin and thus its accessibility to proteins required for transcription initiation. Second, activators and repressors interact with a large multiprotein complex called the mediator of transcription complex, or simply Mediator. This complex, in turn, binds to Pol II and directly regulates assembly of the preinitiation complex. In addition, some activation domains interact with TFIID-TAF subunits or other components of the preinitiation complex, and these interactions contribute to preinitiation complex assembly. Finally, activation domains may also interact with the elongation factor P-TEFb (cyclin T-CDK9) and other as yet unknown factors to stimulate elongation by Pol II away from the promoter region.

In this section, we review the current understanding of how repressors and activators control chromatin structure and preinitiation complex assembly. In the next section of the chapter, we discuss how the concentrations and activities of activators and repressors themselves are controlled, so that gene expression is precisely attuned to the needs of the cell and organism.

## Formation of Heterochromatin Silences Gene Expression at Telomeres, near Centromeres, and in Other Regions

For many years it has been clear that inactive genes in eukaryotic cells are often associated with heterochromatin, regions of chromatin that are more highly condensed and stain more darkly with DNA dyes than euchromatin, in which most transcribed genes are located (see Figure 8-28a). Regions of chromosomes near the centromeres and telomeres, as well as additional specific regions that vary in different cell types, are organized into heterochromatin. The DNA in heterochromatin is less accessible to externally added proteins than is DNA in euchromatin and consequently is often referred to as "closed" chromatin. For instance, in an experiment described in Chapter 8, the DNA of inactive genes was found to be far more resistant to digestion by DNase I than the DNA of transcribed genes (see Figure 8-27).

Study of DNA regions in S. cerevisiae that behave like the heterochromatin of higher eukaryotes provided early insight into the chromatin-mediated repression of transcription. This yeast can grow either as haploid or diploid cells. Haploid cells exhibit one of two possible mating types, called a and $\alpha$. Cells of different mating type can "mate," or fuse,

to generate a diploid cell (see Figure 1-23). When a haploid cell divides by budding, the larger "mother" cell switches its mating type. Genetic and molecular analyses have revealed that three genetic loci on yeast chromosome III control the mating type of yeast cells (Figure 9-35). The central matingtype locus, termed MAT-the only one of the three that is actively transcribed-encodes transcription factors (a1, or $\alpha 1$ and $\alpha 2$ ) that regulate genes that determine the mating type. In any one cell, either an a or $\alpha$ DNA sequence is located at the MAT. The two additional loci, termed $H M L$ and $H M R$, near the left and right telomere, respectively, contain "silent" (nontranscribed) copies of the a or $\alpha$ genes. These sequences are transferred alternately from $H M L \alpha$ or $H M R a$ into the MAT locus by a type of nonreciprocal recombination between homologous sequences during cell division. When the MAT locus contains the DNA sequence from $H M L \alpha$, the cells behave as $\alpha$ cells. When the MAT locus contains the DNA sequence from $H M R a$, the cells behave like a cells.

Our interest here is in how transcription of the silent mating-type genes at $H M L$ and $H M R$ is repressed. If these genes are expressed, as they are in yeast mutants with defects in the repressing mechanism, both a and $\alpha$ proteins are expressed, causing the cells to behave like diploid cells, which cannot mate. The promoters and UASs controlling transcription of the a and $\alpha$ genes lie near the center of the DNA sequence that is transferred and are identical whether the sequences are at the MAT locus or at one of the silent loci. This arrangement indicates that the function of the transcription factors that interact with these sequences must somehow be blocked at $H M L$ and $H M R$, but not at the MAT locus. This repression of the silent loci depends on silencer sequences located next to the region of transferred DNA at HML and HMR (see Figure 9-35). If the silencer is deleted, the adjacent locus is transcribed. Remarkably, any gene placed near the yeast mating-type silencer sequence by recombinant DNA techniques is repressed, or "silenced," even a tRNA gene transcribed by RNA polymerase III, which uses a different
set of general transcription factors than RNA polymerase II uses, as discussed later.

Several lines of evidence indicate that repression of the $H M L$ and $H M R$ loci results from a condensed chromatin structure that sterically blocks transcription factors from interacting with the DNA. In one telling experiment, the gene encoding an E. coli enzyme that methylates adenine residues in the sequence GATC was introduced into yeast cells under the control of a yeast promoter so that the enzyme was expressed. Researchers found that GATC sequences within the MAT locus and most other regions of the genome in these cells were methylated, but not those within the HML and HMR loci. These results indicate that the DNA of the silent loci is inaccessible to the E. coli methylase, and presumably to proteins in general, including transcription factors and RNA polymerase. Similar experiments conducted with various yeast histone mutants indicated that specific interactions involving the histone tails of H 3 and H 4 are required for formation of a fully repressed chromatin structure. Other studies have shown that the telomeres of every yeast chromosome also behave like silencer sequences. For instance, when a gene is placed within a few kilobases of any yeast telomere, its expression is repressed. In addition, this repression is relieved by the same mutations in the H3 and H4 histone tails that interfere with repression at the silent mating-type loci.

Genetic studies led to identification of several proteins, RAP1 and three SIR proteins, that are required for repression of the silent mating-type loci and the telomeres in yeast. RAP1 was found to bind within the DNA silencer sequences associated with $H M L$ and $H M R$ and to a sequence that is repeated multiple times at each yeast-chromosome telomere. Further biochemical studies showed that the SIR2 protein is a histone deacetylase; it removes acetyl groups on lysines of the histone tails. Furthermore, the RAP1 and SIR2, 3, and 4 proteins bind to one another, and SIR3 and SIR4 bind to the N-terminal tails of histones H3 and H4, which are maintained in a largely nonacetylated state by the deacetylase
![li2612239539b](li2612239539b.jpg)

FIGURE 9-35 Arrangement of mating-type loci on chromosome III in the yeast S. cerevisiae. Silent (unexpressed) mating-type genes (either a or $\alpha$ ) are located at the HML locus. The opposite mating-type gene is present at the silent $H M R$ locus. When the $\alpha$ or $\mathbf{a}$ sequences are present at the MAT locus, they can be transcribed into mRNAs whose
encoded proteins specify the mating-type phenotype of the cell. The silencer sequences near HML and HMR bind proteins that are critical for repression of these silent loci. Haploid cells can switch mating types in a process that transfers the DNA sequence from HML or HMR to the transcriptionally active MAT locus.

![li2622239539b](li2622239539b.jpg)

EXPERIMENTAL FIGURE 9-36 Antibody and DNA probes colocalize SIR3 protein with telomeric heterochromatin in yeast nuclei.
(a) Confocal micrograph 0.3 mm thick through three diploid yeast cells, each containing 68 telomeres. Telomeres were labeled by hybridization to a fluorescent telomere-specific probe (yellow). DNA was stained red to reveal the nuclei. The 68 telomeres coalesce into a much smaller number of regions near the nuclear periphery. (b, c) Confocal micrographs of yeast cells labeled with a telomere-specific hybridization probe (b) and a fluorescent-labeled antibody specific for SIR3 (c). Note that SIR3 is localized in the repressed telomeric heterochromatin. Similar experiments with RAP1, SIR2, and SIR4 have shown that these proteins also colocalize with the repressed telomeric heterochromatin. (d) Schematic model of the silencing mechanism at yeast telomeres. (Top left) Multiple copies of RAP1
bind to a simple repeated sequence at each telomere region that lacks nucleosomes. SIR3 and SIR4 bind to RAP1, and SIR2 binds to SIR4. SIR2 is a histone deacetylase that deacetylates the tails on the histones neighboring the repeated RAP1-binding site. (Middle) The hypoacetylated histone tails are also binding sites for SIR3 and SIR4, which in turn bind additional SIR2, deacetylating neighboring histones. Repetition of this process results in spreading of the region of hypoacetylated histones with associated SIR2, SIR3, and SIR4. (Bottom) Interactions between complexes of SIR2, SIR3, and SIR4 cause the chromatin to condense and several telomeres to associate, as shown in a-c. The higher-order chromatin structure generated sterically blocks other proteins from interacting with the underlying DNA. See M. Grunstein, 1997, Curr. Opin. Cell Biol. 9:383. [Parts (a)-(c) ©1996 Gotta et al., The Journal of Cell Biology, 134: 1349-1363. doi:10.1083/jcb.134.6.134.]

activity of SIR2. A series of experiments using fluorescence confocal microscopy on yeast cells either stained with fluorescent-labeled antibody to any one of the SIR proteins or RAP1 or hybridized to a labeled telomere-specific DNA probe revealed that these proteins form large, condensed telomeric nucleoprotein structures resembling the heterochromatin found in higher eukaryotes (Figure 9-36a, b, c).

Figure 9-36d depicts a model for the chromatin-mediated silencing at yeast telomeres based on these and other studies. Formation of heterochromatin at telomeres is nucleated by multiple RAP1 protein molecules bound to repeated sequences in a nucleosome-free region at the extreme end of a telomere. A network of protein-protein interactions involving telomere-bound RAP1, three SIR proteins (2, 3, and 4), and hypoacetylated histones H 3 and H 4 creates a higherorder nucleoprotein complex that includes several telomeres and in which the DNA is largely inaccessible to external proteins. One additional protein, SIR1, is also required for silencing of the mating-type loci. It binds to the silencer regions associated with HML and HMR together with RAP1 and other proteins to initiate assembly of a similar multiprotein silencing complex that encompasses $H M L$ and $H M R$.

An important feature of this model is the dependence of repression on hypoacetylation of the histone tails. This dependence was demonstrated in experiments with yeast mutants expressing histones in which lysines in histone N termini were replaced with arginines, glutamines, or glycines. Arginine is positively charged, like lysine, but cannot be acetylated. Glutamine, on the other hand, is neutral and simulates the neutral charge of acetylated lysine, and glycine, with no side chain, also mimics the absence of a positively charged lysine. Repression at telomeres and at the silent mating-type loci was defective in the mutants with glutamine and glycine substitutions for lysine in the H3 or H4 histone tails, but not in the mutants with arginine substitutions. Further, acetylation of H3 and H4 lysines interferes with binding by Sir3 and Sir4 and consequently prevents repression at the silent loci and telomeres. Finally, chromatin immunoprecipitation experiments (see Figure 9-18a) using antibodies specific for acetylated lysines at particular positions in the histone N-terminal tails (see Figure 8-26a) confirmed that histones in repressed regions near telomeres and at the silent mating loci are hypoacetylated, but become hyperacetylated in sir mutants when genes in these regions are derepressed.

## Repressors Can Direct Histone Deacetylation at Specific Genes

The importance of histone deacetylation in chromatinmediated gene repression was further supported by studies of eukaryotic repressors that regulate genes at internal chromosomal positions. These proteins are now known to act in part by causing deacetylation of histone tails in nucleosomes that encompass the TATA box and promoter-proximal region of the genes they repress. In vitro studies have shown that when promoter DNA is part of a nucleosome with nonacetylated histones, the general transcription factors cannot bind to the TATA box and promoter-proximal
region. In nonacetylated histones, the N -terminal lysines are positively charged and may interact with DNA phosphates. The nonacetylated histone tails also interact with neighboring histone octamers and other chromatin-associated proteins, favoring the folding of chromatin into condensed higher-order structures whose precise conformation is not well understood. The net effect is that general transcription factors cannot assemble into a preinitiation complex on a promoter associated with hypoacetylated histones. In contrast, binding of general transcription factors is repressed much less by histones with hyperacetylated tails, in which the positively charged lysines are neutralized and electrostatic interactions are eliminated.

The connection between histone deacetylation and repression of transcription at specific yeast promoters became clearer when the cDNA encoding a human histone deacetylase was found to have high homology to the yeast RPD3 gene, known to be required for the normal repression of a number of yeast genes. Further work showed that the yeast Rpd3 protein has histone deacetylase activity. The ability of Rpd3 to deacetylate histones at a number of promoters depends on two other proteins: Ume6, a repressor that binds to a specific upstream regulatory sequence (URS1), and Sin3, which is part of a large multiprotein complex called Rpd3L that also contains Rpd3 (Figure 9-37a). Sin3 also binds to the repression domain of Ume6, thus positioning the Rpd3 histone deacetylase in the complex so that it can interact with nearby promoter-associated nucleosomes and remove acetyl groups from histone-tail lysines. Additional experiments, using the chromatin immunoprecipitation technique outlined in Figure 9-18a and antibodies to specific histone acetylated lysines, demonstrated that in wild-type yeast, one or two nucleosomes in the immediate vicinity of Ume6-binding sites are hypoacetylated. These sites include the promoters of genes repressed by Ume6. In sin3 and rpd3 deletion mutants, not only were these promoters derepressed, but the nucleosomes near the Ume6-binding sites were hyperacetylated. All these findings provide considerable support for the model of repressor-directed deacetylation shown in Figure 9-37a.

In yeast, the Sin3-Rpd3 complex (Rpd3L) functions as a co-repressor, a protein or complex of proteins that binds to a repression domain and interacts with chromatin, Pol II, or the general transcription factors to repress transcription. Co-repressor complexes containing histone deacetylases have also been found associated with many repressors from mammalian cells. Some of these complexes contain the mammalian homolog of Sin3 (mSin3), which interacts with the repression domain of repressors, as in yeast. Other histone deacetylase complexes identified in mammalian cells contain additional or different repression domain-binding proteins. These various repressor and co-repressor combinations mediate histone deacetylation at specific promoters by a mechanism similar to the yeast mechanism (see Figure 9-37a). In addition to repressing transcription through the formation of "closed" chromatin structures, some repression domains have also been found to inhibit the assembly of preinitiation complexes in in vitro experiments with purified general

![li2632239539b](li2632239539b.jpg)

FIGURE 9-37 Proposed mechanism of histone deacetylation and hyperacetylation In yeast transcriptional control. (a) Repressordirected deacetylation of histone N-terminal tails. The DNA-binding domain (DBD) of the repressor Ume6 interacts with a specific upstream control element of the genes it regulates, called URS1. The Ume6 repression domain (RD) binds Sin3, a subunit of a multiprotein complex that includes Rpd3, a histone deacetylase. Deacetylation of histone N-terminal tails on nucleosomes in the region of the Ume6-binding site inhibits binding of general transcription factors at the TATA box, thereby repressing gene expression. (b) Activatordirected hyperacetylation of histone N-terminal tails. The DNA-binding domain of the activator Gcn4 interacts with specific upstream activating sequences (UAS) of the genes it regulates. The Gcn4 activation domain (AD) then interacts with a multiprotein histone acetylase complex that includes the Gcn5 catalytic subunit. Subsequent hyperacetylation of histone N-terminal tails on nucleosomes in the vicinity of the Gcn4-binding site facilitates access by the general transcription factors required for initiation. Repression and activation of many genes in higher eukaryotes occur by similar mechanisms.
transcription factors in the absence of histones. This activity probably contributes to the repression of transcription by these repression domains in vivo as well.

## Activators Can Direct Histone Acetylation at Specific Genes

Just as repressors function through co-repressors that bind to their repression domains, the activation domains of DNA-binding activators function by binding multisubunit co-activator complexes, protein complexes that interact with or modify chromatin, Pol II, or general transcription factors to activate transcription. One of the first co-activator complexes to be characterized was the yeast SAGA complex, which functions with the Gcn4 activator protein described in Section 9.4. Early genetic studies indicated that full activity of the Gcn4 activator required a protein called Gcn5. The clue to Gcn5's function came from biochemical studies of a histone acetylase purified from the protozoan Tetrahymena, the first histone acetylase to be purified. Sequence analysis revealed homology between the Tetrahymena protein and yeast Gcn5, which was soon shown to have histone acetylase activity as well. Further genetic and biochemical studies revealed that Gcn5 is one subunit of a multiprotein
co-activator complex, named the SAGA complex after genes encoding some of the subunits. Another subunit of this histone acetylase complex binds to activation domains in multiple yeast activator proteins, including Gcn4. The model shown in Figure 9-37b is consistent with the observation that nucleosomes near the promoter region of a gene regulated by the Gcn4 activator are specifically hyperacetylated compared with most histones in the cell. This activatordirected hyperacetylation of nucleosomes near a promoter region opens the chromatin structure so as to facilitate the binding of other proteins required for transcription initiation. The chromatin structure is less condensed than most chromatin, as indicated by its sensitivity to digestion with nucleases in isolated nuclei.

In addition to leading to the decondensation of chromatin, the acetylation of specific histone lysines generates binding sites for proteins containing bromodomains. A bromodomain is a sequence of about 110 amino acids that folds into a domain that binds acetylated lysine. One or more bromodomains are found in several chromosome-associated proteins that contribute to transcriptional activation. For example, a subunit of the general transcription factor TFIID contains two bromodomains, which bind to acetylated nucleosomes with high affinity. Recall that TFIID binding to a

promoter initiates assembly of an RNA polymerase II preinitiation complex (see Figure 9-19). Nucleosomes at promoter regions of virtually all active genes have acetylated lysines in their H3 and H4 histone tails.

A similar activation mechanism operates in higher eukaryotes. Mammalian cells contain multisubunit histone acetylase co-activator complexes that are homologous to the yeast SAGA complex. They also express two related $300-\mathrm{kDa}$, multidomain proteins called CBP and p300, which function similarly. As noted earlier, one domain of CBP binds the phosphorylated acidic activation domain in the CREB transcription factor. Other domains of CBP interact with different activation domains in other activators. Yet another domain of CBP has histone acetylase activity, and another CBP domain associates with additional multisubunit histone acetylase complexes. CREB and many other mammalian activators function in part by directing CBP and the associated histone acetylase complex to specific nucleosomes, where they acetylate histone tails, facilitating the interaction of general transcription factors with promoter DNA.

## Chromatin-Remodeling Complexes Help Activate or Repress Transcription

In addition to histone acetylase complexes, multiprotein chromatin-remodeling complexes are required for activation at many promoters. The first of these complexes characterized was the yeast SWI/SNF chromatin-remodeling complex. One of the SWI/SNF subunits has homology to DNA helicases, enzymes that use energy from ATP hydrolysis to disrupt interactions between base-paired nucleic acids or between nucleic acids and proteins. In vitro, the SWI/SNF complex is thought to pump or push DNA into the nucleosome so that DNA bound to the surface of the histone octamer transiently dissociates from the surface and translocates, causing the nucleosomes to "slide" along the DNA. The net result of such chromatin remodeling is to facilitate the binding of transcription factors to specific DNA sequences in chromatin. Many activation domains bind to such chromatinremodeling complexes, and this binding stimulates in vitro transcription from chromatin templates in which the DNA is associated with histone octamers. Thus the SWI/SNF complex represents another type of co-activator complex. The experiment shown in Figure 9-38 demonstrates dramatically how an activation domain can cause decondensation of a region of chromatin. This decondensation results from association of the activation domain with chromatin-remodeling and histone acetylase complexes.

Chromatin-remodeling complexes are required for many processes involving DNA in eukaryotic cells, including transcriptional control, DNA replication, recombination, and DNA repair. Several types of chromatin-remodeling complexes are found in eukaryotic cells, all with homologous DNA helicase domains. SWI/SNF complexes and related chromatin-remodeling complexes in multicellular organisms contain subunits with bromodomains that bind to acetylated
![li2642239539b](li2642239539b.jpg)

FIGURE 9-38 Expression of fusion proteins demonstrates chromatin decondensation in response to an activation domain. A cultured hamster cell line was engineered to contain multiple copies of a tandem array of $E$. coli lac operator sequences integrated into a chromosome in a region of heterochromatin. (a) When an expression vector for the lac repressor (LacI) was transfected into these cells, lac repressor bound to the lac operator sites could be visualized in a region of condensed chromatin using an antibody against the lac repressor (red). DNA was visualized by staining with DAPI (blue), revealing the nucleus. A diagram of condensed chromatin is shown below. (b) When LacI fused to an activation domain was transfected into these cells, staining as in (a) revealed that the activation domain causes this region of chromatin to decondense into a thinner chromatin fiber that fills a much larger volume of the nucleus. A diagram of a region of decondensed chromatin with bound LacI fusions to the VP16 activation domain (AD) and associated chromatin remodeling and histone acetylase complexes is shown below. [Photos ©1999 Dr. Andrew S. Belmont et al., The Journal of Cell Biology, 145:1341-1354. doi: 10.1083/jcb.145.7.1341.]
histone tails. Consequently, SWI/SNF complexes remain associated with activated, acetylated regions of chromatin, presumably maintaining them in a decondensed conformation. Chromatin-remodeling complexes can also participate in transcriptional repression. These complexes bind to the repression domains of repressors and contribute to repression, presumably by folding chromatin into condensed structures. Much remains to be learned about how this important class of proteins alters chromatin structure to influence gene expression and other processes.

## Pioneer Transcription Factors Initiate the Process of Gene Activation During Cellular Differentiation

As cells differentiate during embryogenesis and during differentiation from stem cells in adult organisms (see Chapter 21), many of the genes induced during the

process are initially in repressed regions of heterochromatin in undifferentiated progenitor cells. Activation of these genes requires that the chromatin environment of their transcription-control regions become decondensed so that transcription factors can bind to enhancers and promoterproximal control elements and so that the general transcription factors and Pol II can bind to promoters. In many cases, this decondensation is initiated by special pioneer transcription factors that can bind to their cognate binding sites in DNA even when those sites are within repressed heterochromatic regions of chromatin. These factors have a DNA-binding domain that binds to one side of the DNA helix in a manner similar to the bacteriophage 434 repressor (see Figure 9-29). This domain allows these factors to bind to their specific binding sites while the DNA is wrapped around a histone octamer with the opposite side of the DNA against the surfaces of histones.
![li2652239539b](li2652239539b.jpg)

FIGURE 9-39 Structure of yeast and human Mediator complexes. (a) Subunits of the S. cerevisiae and human Mediator complexes. The subunits constituting the head, middle, and tail modules of Mediator are indicated, as well as the subunits of the CDK8-kinase module (CKM) that associates with some Mediator complexes, blocking Pol II binding. (b) Cryoelectron microscopic structure of the yeast Mediator without the CKM. (Left) The head, middle, and tail modules composed of the subunits listed above are color-coded. (Right) The structure of a complex of Mediator with Pol II, called the holoenzyme, suggests that the Mediator modules rotate relative to one another as shown to create a surface that binds Pol II. (Part (b) republished with permission of Elsevier, from Tsai, K.L., "Subunit architecture and functional modular rearrangements of the transcriptional mediator complex," Cell, 2014,157(6): 1430-1444; permission conveyed through Copyright Clearance Center, Inc.)

One example of pioneer transcription factors initiating the process of transcriptional activation involves the liver-specific gene Alb1, encoding serum albumin, a major constituent of blood serum that is secreted into the blood by hepatocytes. In the developing mouse, the FoxA and GATA-4 or GATA-6 transcription factors are the first transcription factors to bind to an Alb1 enhancer in undifferentiated gut endodermal cells destined to develop into the liver. FoxA has a "winged helix" DNA-binding domain that binds to one side of the DNA helix containing the FoxA-binding site. GATA factors are also able to bind to their specific sites in DNA when those sites are included in nucleosomal DNA wrapped around a histone octamer. The FoxA and GATA-4/6 activation domains may then interact with chromatin remodeling complexes and histone acetylase complexes to decondense the chromatin of the 120-bp Alb1 enhancer, allowing the observed subsequent binding of four additional transcription factors in the nascent liver bud that develops later.

## The Mediator Complex Forms a Molecular Bridge Between Activation Domains and Pol II

Once the interaction of activation domains with histone acetylase complexes and chromatin remodeling complexes converts the chromatin of a promoter region to an "open" structure that allows the binding of general transcription factors, activation domains interact with another multisubunit co-activator complex, the Mediator complex (Figure 9-39). Activation domain-Mediator interactions stimulate assembly of the preinitiation complex on the promoter. Recent cryoelectron microscopy studies show that the head and middle domains of the Mediator complex interact directly with Pol II. Several Mediator subunits bind to activation domains in various activator proteins. Thus Mediator can form a molecular bridge between an activator bound to its cognate site in DNA and Pol II bound to a promoter.

Experiments with temperature-sensitive yeast mutants indicate that some Mediator subunits are required for transcription of virtually all yeast genes. These subunits help maintain the overall structure of the Mediator complex or bind to Pol II; they are therefore required for activation by all activators. In contrast, other Mediator subunits are required for normal activation or repression of specific subsets of genes. DNA microarray analysis (see Figure 6-26) of yeast gene expression in mutants with defects in these nonessential Mediator subunits have indicated that each one influences transcription of $3-10$ percent of all genes to the extent that its deletion either increases or decreases mRNA expression by a factor of twofold or more. In many cases, these Mediator subunits have been discovered to interact with specific activation domains; thus when one Mediator subunit is defective, transcription of genes regulated by activators that bind to that subunit is severely depressed, but

transcription of other genes is unaffected. Recent cryoelectron microscopy studies suggest that when activation domains interact with Mediator, the head, middle, and tail domains depicted in Figure 9-39 rotate relative to one another, creating a binding surface for RNA polymerase II. The surface of the polymerase that interacts with general transcription factors in the preinitiation complex (see Figure 9-20) remains exposed in the proposed model of the polymerase-Mediator complex, referred to as the boloenzyme.

The various experimental results indicating that individual Mediator subunits bind to specific activation domains suggest that multiple activators may influence transcription from a single promoter by interacting with a Mediator complex simultaneously or in rapid succession (Figure 9-40). Activators bound at enhancers or promoter-proximal elements can interact with Mediator associated with a promoter because chromatin, like DNA, is flexible and can form a loop, bringing the regulatory regions and the promoter close together, as observed for the E. coli NtrC activator and $\sigma^{54}$-RNA polymerase (see Figure 9-5). The multiprotein complexes that form on eukaryotic promoters may comprise more than 100 polypeptides with a total mass of 3-5 megadaltons (MDa)—as large as a ribosome.

In vivo, assembly of a preinitiation complex on a promoter and initiation of transcription is a highly cooperative process generally requiring that several transcription factors bound to transcription-control elements interact with co-activators that in turn interact with Pol II and general transcription factors. A cell must produce the specific set of activators required for transcription of a particular gene in order to express that gene.

## KEY CONCEPTS OF SECTION 9.5

## Molecular Mechanisms of Transcription Repression and Activation

- Eukaryotic transcription activators and repressors exert their effects largely by binding to multisubunit co-activators or co-repressors that influence the assembly of preinitiation complexes either by modulating chromatin structure or by interacting with Pol II and general transcription factors.
- The DNA in condensed regions of chromatin (heterochromatin) is relatively inaccessible to transcription factors and other proteins, so that gene expression in these regions is repressed.
- The interactions of several proteins with one another and with the hypoacetylated N -terminal tails of histones H 3 and H 4 are responsible for the chromatin-mediated repression of transcription that occurs in the telomeres and the silent mating-type loci in S. cerevisiae (see Figure 9-36).
![li2662239539b](li2662239539b.jpg)

FIGURE 9-40 Model of several DNA-bound activators interacting with a single Mediator complex. The ability of different Mediator subunits to interact with specific activation domains may contribute to the integration of signals from several activators at a single promoter. See the text for discussion.

- Some repression domains function by interacting with co-repressors that are histone deacetylase complexes. The subsequent deacetylation of histone N -terminal tails in nucleosomes near the repressor-binding site inhibits interaction between the promoter DNA and general transcription factors, thereby repressing transcription initiation (see Figure 9-37a).
- Activation domains function by binding multiprotein coactivator complexes such as histone acetylase complexes. The subsequent hyperacetylation of histone N-terminal tails in nucleosomes near the activator-binding site facilitates interactions between the promoter DNA and general transcription factors, thereby stimulating transcription initiation (see Figure 9-37b).
- SWI/SNF chromatin-remodeling factors constitute another type of co-activator. These multisubunit complexes can transiently dissociate DNA from histone cores in an ATPdependent reaction and may also decondense regions of chromatin, thereby promoting the binding of DNA-binding proteins needed for transcription initiation.
- The Mediator complex, another type of co-activator, is a roughly 30 -subunit complex that forms a molecular bridge between activation domains and RNA polymerase II by binding directly to the polymerase and activation domains. By binding to several different activators either simultaneously or in rapid succession, Mediator probably helps integrate the effects of multiple activators on a single promoter (see Figure 9-40).
- Activators bound to a distant enhancer can interact with transcription factors bound to a promoter because chromatin is flexible and the intervening chromatin can form a large loop.
- The highly cooperative assembly of preinitiation complexes in vivo generally requires several activators. A cell must produce the specific set of activators required for transcription of a particular gene in order to express that gene.

### 9.6 Regulation of TranscriptionFactor Activity

We have seen in the preceding discussion how combinations of transcription factors that bind to specific DNA regulatory sequences control the transcription of eukaryotic genes. Whether or not a specific gene in a multicellular organism is expressed in a particular cell at a particular time is largely a consequence of the nuclear concentrations and activities of the transcription factors that interact with the transcription-control regions of that gene. (Exceptions are due to the "transcriptional memory" that results from the epigenetic mechanisms discussed in the next section.) Which transcription factors are expressed in a particular cell type, and the amounts produced, are determined by multiple regulatory interactions between transcription factors and control regions in genes encoding transcription factors that occur during the development and differentiation of that cell type. Recent advances in the analysis of transcription-factor-binding sites through identification of DNase I hypersensitive sites on a genomic scale have given us the first high-resolution view of how transcription-factor binding changes during the development and differentiation of multiple human cell types.

## DNase I Hypersensitive Sites Reflect the Developmental History of Cellular Differentiation

In Chapter 8, we learned that an expressed gene is far more sensitive to digestion by DNase I (a bovine pancreatic enzyme) than the same gene in a different cell type in which it is not expressed (see Figure 8-34). In addition to this general increase in DNase I sensitivity over long regions, researchers later found that specific short regions of the genome, on the order of a hundred base pairs in length, are extremely sensitive to DNase I digestion and are the first regions cut when isolated nuclei are treated with low levels of DNase I. These sites are known as DNase I hypersensitive sites (DHSs). High-throughput sequencing methods have allowed mapping of DHSs across the genome in multiple differentiated and embryonic cell types. Briefly, after digestion of isolated nuclei with low levels of DNase I, DNA is isolated from the treated chromatin. Oligonucleotide linkers of a known sequence are ligated to the DNA ends generated by DNase I digestion. Then the DNA is sheared into small fragments by sonication, amplified by PCR, and sequenced. Human DNA sequences adjacent to the known sequence of the oligonucleotide linker were thus identified as DHSs.

Figure 9-41a shows plots of the number of times a DHS was sequenced—a measure of the DNase sensitivity of the site-in samples from the human cell types indicated at the left. A roughly $600-\mathrm{kb}$ region of the genome on chromosome 12, located $96.2-96.8 \mathrm{Mb}$ from the left end of the chromosome, is shown. The height of each vertical bar represents the degree of sensitivity of the DNA sequence at that position to digestion in nuclei isolated from each of the cell types.

Mapping of binding sites for specific transcription factors by chromatin immunoprecipitation (see Figure 9-18) has shown that most transcription-factor-binding sites are coincident with DHSs. This may be because the DNA-binding domain of the bound transcription factor exposes DNA flanking the binding site to DNase I digestion, or because the transcription-factor activation domain interacts with chromatin-remodeling complexes that destabilize the interaction of DNA with histone octamers in neighboring nucleosomes, causing the DNA to be more sensitive to DNase I digestion. Because DHSs are coincident with bound transcription factors, the DHS pattern in a region of chromatin represents the positions of bound transcription factors, although the transcription factors bound are not directly identified.

In Figure 9-41a, the type of tissue from which the DHS data were determined is shown on the left, and the embryonic tissues from which these tissue types developed are colorcoded as indicated in Figure 9-41b. It is apparent that more closely related cell types, such as fibroblasts from different regions of the body, or endothelial cells that line the inner surfaces of blood vessels from different organs, have more similar DHSs than more distantly related cell types. With computer methods, it is possible to compare the similarity of the DHS maps for each of these cell types across the entire genome. With these computational methods, a dendrogram can be generated showing how closely the DHS map from one cell type resembles those of other cell types (see Figure 9-41b). This dendrogram is similar to the dendrograms used to show the relatedness, and hence the evolution, of gene sequences (see Figure 8-21b).

Importantly, the DHS pattern of embryonic stem cells is at the root of the DHS dendrogram for all cell types (see Figure 9-41b). These cells from the inner cell mass of the early mammalian embryo, discussed in Chapter 21 (see Figure 21-5), are the progenitors of all cells in the adult organism. Embryonic stem cells appear to have the most complex transcriptional control of all cells in that they have the largest number of DHSs: about 257,000 in one study, compared with $90,000-150,000$ in differentiated cells. This difference probably reflects the developmental potential of embryonic stem cells. Approximately 30 percent of the DHSs observed in adult differentiated cells are also observed in embryonic stem cells, but a different 30 percent is retained in each adult cell type. An additional 50,000-100,000 new DHSs not found in embryonic stem cells arise during development, but a different set of DHSs arises in each cell type. These DHS patterns reveal the complexity of the combinations of transcription factors that regulate each gene. Approximately a million distinct DHSs were characterized in the cell types shown in Figure 9-41, suggesting that on average, combinations of four or five enhancers regulate the transcription of each of the roughly 21,000 genes in the human genome. This analysis excluded the central nervous system, probably the most complex organ system of all, so the total number of human enhancers may be much larger. But in the tissues analyzed, the maps of DHSs reveal where binding of early embryonic

![li2672239539b](li2672239539b.jpg)

FIGURE 9-41 Maps of DNase I hypersensitive sites in embryonic and adult cells reflect their developmental history. (a) DHSs from each of the human cell types shown at the left are mapped in the interval on chromosome 12 between 96.2 and 96.8 Mb from the left end. The height of each vertical bar in the figure represents the number of times a sequence in a 50-bp interval at that position was sequenced after following the protocol described in the text to ligate a linker of known sequence to DNA ends resulting from low-level DNase I digestion of chromatin. The plots are color-coded according to the embryonic tissue from which they developed, as shown in (b). (b) Dendrogram showing the relationships among the DHS maps for each cell type across the entire genome. The embryonic tissue from which each of these cell types develops is shown at the right. Embryonic stem cells form the root of the dendrogram. The DHS maps for all other cell types are derived from those for the embryonic stem cell by loss of some DHSs and the acquisition of other DHSs. The dendrogram, based on how closely DHS maps from two cell types are related, parallels the developmental relationships among the cell types. [Republished with permission of Elsevier, Stergachis, A.B., et al., "Developmental Fate and Cellular Maturity Encoded in Human Regulatory DNA Landscapes," Cell, 2013, 154: 888-903; permission conveyed through Copyright Clearance Center, Inc.]
(b)
![li2682239539b](li2682239539b.jpg)

transcription factors is lost and where new cell-type-specific combinations of transcription factors bind as a cell differentiates from the embryonic stem cell. Even this estimate fails to capture the complexity of transcriptional control, since many transcription-factor-binding sites detected as one DHS are bound by different related transcription factors expressed in different cell types. Often different related transcription factors bind to the same transcription-control region in different cell types to regulate the appropriate level of transcription for that cell type.

## Nuclear Receptors Are Regulated by Extracellular Signals

In addition to controlling the expression of transcription factors, cells also regulate the activities of many of the transcription factors expressed in a particular cell type. For example, many transcription factors are regulated by intercellular signals. Interactions between the extracellular domains of transmembrane receptor proteins on the surface of the cell and specific protein ligands for these receptors secreted by other cells or expressed on the surfaces of neighboring cells activate the intracellular domains of these transmembrane proteins, transducing the signal received on the outside of the cell to a signal on the inside of the cell. The intracellular signal then regulates the activities of enzymes that modify transcription factors by phosphorylation, acetylation, and other types of post-translational protein modifications. These post-translational modifications activate or inhibit transcription factors in the nucleus. In Chapter 16, we describe the major types of cell-surface receptors for protein ligands and the intracellular signaling pathways that regulate transcription-factor activity.

Here we discuss another major group of extracellular signals that regulate the activities of transcription factors: small, lipid-soluble hormones including many different steroid hormones, retinoids, and thyroid hormones. These lipid-soluble hormones can diffuse through the plasma and nuclear membranes and interact directly with the transcription factors they control (Figure 9-42). As noted earlier, transcription factors regulated by lipid-soluble hormones include the nuclear-receptor superfamily. These transcription
factors function as transcription activators only when bound to their ligands.

## All Nuclear Receptors Share a Common Domain Structure

Sequencing of cDNAs derived from mRNAs encoding various nuclear receptors has revealed remarkable conservation in their amino acid sequences. It has also revealed that each of these receptors has three functional regions (Figure 9-43). The first is a unique N-terminal region of variable length (100-500 amino acids). Portions of this variable region function as activation domains in most nuclear receptors. The second is a DNA-binding domain that maps near the center of the primary sequence and contains a repeat of the $\mathrm{C}_{4}$ zinc-finger motif (see Figure 9-30b). The third region, the hormonebinding domain, located near the C-terminal end, contains a hormone-dependent activation domain (see Figure 9-31b, c). In some nuclear receptors, the hormone-binding domain functions as a repression domain in the absence of ligand.

## Nuclear-Receptor Response Elements Contain Inverted or Direct Repeats

The DNA sites to which nuclear receptors bind are called response elements. The characteristic nucleotide sequences of several response elements have been determined. The consensus sequences of response elements for two steroid hormone receptors, the glucocorticoid receptor response element (GRE) and the estrogen receptor response element (ERE) are 6-bp inverted repeats separated by any three base pairs (Figure 9-44a, b). This finding suggested that the cognate steroid hormone receptors would bind to DNA as symmetric dimers (i.e., dimers with twofold rotational symmetry), as was later confirmed by x-ray crystallographic analysis of the homodimeric glucocorticoid receptor's $\mathrm{C}_{4}$ zinc-finger DNA-binding domain (see Figure 9-30b).

Some nuclear-receptor response elements, such as those for the receptors that bind nonsteroids such as vitamin $\mathrm{D}_{3}$, thyroid hormone, and retinoic acid, are direct repeats of the same sequence that is recognized by the estrogen receptor, separated by three, four, or five base pairs (Figure 9-44c-e).
![li2692239539b](li2692239539b.jpg)

FIGURE 9-42 Examples of hormones that bind to nuclear receptors. These and related lipid-soluble hormones diffuse through the plasma and nuclear membranes and bind to receptors located in the cytosol or nucleus. The ligand-receptor complex functions as a transcription activator.

![li2702239539b](li2702239539b.jpg)

FIGURE 9-43 General design of transcription factors in the nuclear-receptor superfamily. The centrally located DNA-binding domain exhibits considerable sequence homology among different receptors and contains two copies of the $\mathrm{C}_{4}$ zinc-finger motif
(see Figure 9-30b). The C-terminal hormone-binding domain exhibits somewhat less homology. The N -terminal regions of various receptors vary in length, have unique sequences, and may contain one or more activation domains. See R. M. Evans, 1988, Science 240:889.
(a)

GRE
$5^{\prime}$ AGAACA(N) $)_{3}$ TGTTCT $3^{\prime}$
$3^{\prime}$ TCTTGT(N) $)_{3}$ ACAAGA $5^{\prime}$
(b)

ERE
$5^{\prime}$ AGGTCA(N) $)_{3}$ TGACCT $3^{\prime}$
$3^{\prime}$ TCCAGT(N) $)_{3}$ ACTGGA $5^{\prime}$
(c)

VDRE
$5^{\prime}$ AGGTCA(N) $)_{3}$ AGGTCA $3^{\prime}$
$3^{\prime}$ TCCAGT(N) $)_{3}$ TCCAGT $5^{\prime}$
(d)

TRE
$5^{\prime}$ AGGTCA(N) $)_{4}$ AGGTCA $3^{\prime}$
$3^{\prime}$ TCCAGT(N) $)_{4}$ TCCAGT $5^{\prime}$
(e)

RARE
$5^{\prime}$ AGGTCA(N) $)_{6}$ AGGTCA $3^{\prime}$
$3^{\prime}$ TCCAGT(N) $)_{5}$ TCCAGT $5^{\prime}$
(f)
![li2712239539b](li2712239539b.jpg)

FIGURE 9-44 Consensus sequences of DNA response elements that bind five nuclear receptors. (a, b) The glucocorticoid and estrogen receptors are twofold symmetric dimers that bind, respectively, to the glucocorticoid receptor response element (GRE) and the estrogen receptor response element (ERE). Each of these response elements contains inverted repeats separated by three base pairs. (c-e) The heterodimeric nuclear receptors each contain one RXR subunit associated with another nuclear-receptor subunit that defines the hormone response. RXR-VDR mediates responses to vitamin $\mathrm{D}_{3}$ by binding to a direct repeat separated by three base pairs (a VDRE). RXR-TR mediates responses to thyroid hormone by binding to the same DNA bases in a direct repeat separated by four base pairs (a TRE). Similarly, RXR-RAR mediates a response to retinoic acid by binding to the same direct repeat separated by five base pairs, comprising a RARE. The repeat sequences bound by the reading helices of these receptors are indicated by red arrows. (f) Crystal structures of the glucocorticoid receptor bound to DNA containing a GRE (top) and of the RXR-TR heterodimer bound to DNA containing a TRE (bottom). Red arrows indicate the orientation from N to C of the helices below them. Note that in the twofold symmetric glucocorticoid receptor, the reading helices are inverted relative to each other so that they "read" an AGAACA on the top strand of the left half-site and on the bottom strand of the right half-site, separated by 3 base pairs. Consequently, the binding site for the glucocorticoid receptor and other twofold symmetric homodimers such as the estrogen receptor is an inverted repeat (see a and b). In contrast, the reading helices in the RXR-TR heterodimer are in the same orientation. Consequently, they read an AGGTCA sequence in the same orientation in the two-half sites separated by four base pairs, a direct-repeat binding site. The interface between the RXR subunit and the vitamin $\mathrm{D}_{3}$ receptor (VDR) subunit bound to a VDRE brings the two reading helices closer together so that they bind to the same half-sites separated by three rather than four base pairs. Similarly, the interface between the RXR and RAR subunits bound to a RARE positions the two reading helices in the heterodimer farther apart than in the RXR-TR, so that they bind the same AGGTCA sequences separated by five base pairs. See K. Umesono et al., 1991, Cell 65:1255, and A. M. Naar et al., 1991, Cell 65:1267. [Part (f) top data from B. F. Luisi et al., 1991, Nature 352:497-505, PDB ID 1glu. Part (f) bottom data from F. Rastinejad et al., 1995, Nature 375:203, PDB ID 2nll.]

The specificity of these response elements is determined by the spacing between the repeats. The nuclear receptors that bind to these direct-repeat response elements do so as heterodimers, all of which share a monomer called RXR. The vitamin $\mathrm{D}_{3}$ response element (VDRE), for example, is bound by the RXR-VDR heterodimer, and the retinoic acid response element (RARE) is bound by RXR-RAR. The monomers composing these heterodimers interact with each other in such a way that the two DNA-binding domains lie in the same rather than inverted orientation, allowing the RXR heterodimers to bind to direct repeats of the binding site for each monomer (Figure 9-44f). In contrast, the monomers in homodimeric nuclear receptors (e.g., GRE and ERE) have an inverted orientation.

## Hormone Binding to a Nuclear Receptor Regulates Its Activity as a Transcription Factor

The mechanism whereby hormone binding controls the activity of nuclear receptors differs between heterodimeric and homodimeric receptors. Heterodimeric nuclear receptors (e.g., RXR-VDR, RXR-TR, and RXR-RAR) are located exclusively in the nucleus. In the absence of their hormone ligand, they repress transcription when bound to their cognate sites in DNA. They do so by directing histone deacetylation at nearby nucleosomes by associating with histone deacetylase complexes, as described earlier for other repressors (see Figure 9-37a). When heterodimeric nuclear receptors bind their ligand, they undergo a conformational change, and as a consequence, they bind histone acetylase complexes, thereby reversing their own repressing effects. In the presence of ligand, the ligand-bound conformation of the receptor also binds Mediator, stimulating preinitiation complex assembly.

In contrast to heterodimeric nuclear receptors, homodimeric receptors are found in the cytoplasm in the absence of their ligands. Hormone binding to these receptors leads to their translocation to the nucleus. The hormone-dependent translocation of the homodimeric glucocorticoid receptor (GR) was demonstrated in the transfection experiments shown in Figure 9-45a-c. The GR hormone-binding domain alone mediates this transport. Subsequent studies showed that in the absence of hormone, GR cannot be transported into the nucleus because its ligand-binding domain is partially unfolded by the major cellular chaperone Hsp70. As long as the receptor is confined to the cytoplasm, it cannot interact with target genes and hence cannot activate transcription. Hormone binding promotes a "handoff" of GR from Hsp70 to Hsp90, which, with coupled hydrolysis of ATP, refolds the GR ligandbinding domain, increasing the affinity for hormone and releasing GR from Hsp70 so that it can enter the nucleus. Once in the nucleus in the conformation induced by ligand binding, it can bind to response elements associated with target genes (Figure 9-45d). Once the receptor with bound hormone binds to a response element, it activates transcription by interacting with chromatin-remodeling and histone acetylase complexes and Mediator.

## Metazoans Regulate the RNA Polymerase II Transition from Initiation to Elongation

A recent unexpected discovery that resulted from application of the chromatin immunoprecipitation technique (see Figure 9-18) is that a large fraction of genes in metazoans have a paused elongating RNA polymerase II within about 100 bp of the transcription start site. Thus expression of the encoded protein is controlled not only by transcription initiation, but also by transcription elongation early in the transcription unit. The first genes discovered to be regulated by control of transcription elongation were heat-shock genes (e.g., hsp70), which encode molecular chaperones that help to refold denatured proteins and other proteins that help the cell to deal with the effects of heat shock. When heat shock occurs, the heat-shock transcription factor (HSTF) is activated. Binding of activated HSTF to specific sites in the promoter-proximal region of heat-shock genes stimulates the paused polymerase to continue chain elongation and promotes rapid reinitiation by additional Pol II molecules, leading to many transcription initiations per minute. This mechanism of transcriptional control permits a rapid response: these genes are always paused in a state of suspended transcription and therefore, when an emergency arises, no time is required to remodel and acetylate chromatin at the promoter and assemble a transcription preinitiation complex.

Another transcription factor shown to regulate transcription by controlling elongation by Pol II paused near the transcription start site is MYC, which functions in the regulation of cell growth and division. MYC is often expressed at high levels in cancer cells and is a key transcription factor in the reprogramming of somatic cells into pluripotent stem cells capable of differentiation into any cell type. The ability to induce differentiated cells to convert to pluripotent stem cells has elicited enormous research interest because of its potential for the development of therapeutic treatments for traumatic injuries to the nervous system and degenerative diseases (see Chapter 21).

## Termination of Transcription Is Also Regulated

Once Pol II has transcribed about 200 nucleotides from the transcription start site, elongation through most genes is highly processive. Chromatin immunoprecipitation with antibody to Pol II, however, indicates that the amount of Pol II at various positions in a transcription unit in a population of cells varies greatly (see Figure 9-18b, right). This finding indicates that the enzyme can elongate through some regions much more rapidly than others. In most cases, Pol II does not terminate transcription until after a sequence is transcribed that directs cleavage and polyadenylation of the RNA at the sequence that forms the $3^{\prime}$ end of the encoded mRNA. Pol II can then terminate transcription at any of multiple sites located $0.5-2 \mathrm{~kb}$ beyond this poly $(\mathrm{A})$ addition site. Experiments with mutant genes show that termination is coupled to the process that cleaves and polyadenylates the $3^{\prime}$ end of a transcript, which is discussed in the next chapter.

![li2722239539b](li2722239539b.jpg)

EXPERIMENTAL FIGURE 9-45 Fusion proteins demonstrate that the hormone-binding domain of the glucocorticoid receptor mediates translocation to the nucleus in the presence of hormone. Cultured animal cells were transfected with expression vectors encoding the proteins diagrammed at the bottom. Immunofluorescence with a labeled antibody specific for $\beta$-galactosidase was used to detect the expressed proteins in transfected cells. (a) In cells that expressed $\beta$-galactosidase alone, the enzyme was localized to the cytoplasm in the presence and absence of the glucocorticoid hormone dexamethasone (Dex). (b) In cells that expressed a fusion protein consisting of $\beta$-galactosidase and the entire glucocorticoid receptor (GR), the fusion protein was present in the cytoplasm in the absence of hormone but was transported to the nucleus in the presence of hormone. (c) Cells that expressed a fusion protein composed of $\beta$-galactosidase and only the GR ligand-binding domain (light purple) also exhibited hormone-dependent transport of the fusion protein to the nucleus. (d) Model of hormone-dependent gene activation by a homodimeric nuclear receptor. In the absence of hormone, the receptor is kept in the cytoplasm by interaction between its ligand-binding domain (LBD) and chaperone proteins. When hormone is present, it diffuses through the plasma membrane and binds to the ligand-binding domain, causing a conformational change that releases the receptor from the chaperone proteins. The receptor with bound ligand is then translocated into the nucleus, where its DNA-binding domain (DBD) binds to response elements, allowing the ligandbinding domain and an additional activation domain (AD) at the N -terminus to stimulate transcription of target genes. [Parts (a)-(c) from Picard, D. and Yamamoto, K. R., "Two signals mediate hormone-dependent nuclear localization of the glucocorticoid receptor," EMBO J., 1987, 6(11):3333-3340; courtesy of the authors.]

## KEY CONCEPTS OF SECTION 9.6

## Regulation of Transcription-Factor Activity

- The activities of many transcription factors are indirectly regulated by binding of extracellular proteins and peptides to cell-surface receptors. These receptors activate intracellular signal transduction pathways that regulate specific transcription factors through a variety of mechanisms discussed in Chapter 16.
- Nuclear receptors constitute a superfamily of dimeric C4 zinc-finger transcription factors that bind lipid-soluble hormones and interact with specific response elements in DNA (see Figures 9-42 and 9-44).
- Hormone binding to nuclear receptors induces conformational changes that modify the interactions of these receptors with other proteins (see Figure 9-31b, c).
- Heterodimeric nuclear receptors (e.g., those for retinoids, vitamin D, and thyroid hormone) are found only in the

nucleus. In the absence of hormone, they repress transcription of target genes with the corresponding response element. When bound to their ligands, they activate transcription.

- Steroid hormone receptors are homodimeric nuclear receptors. In the absence of hormone, they are trapped in the cytoplasm by molecular chaperones. When bound to their ligands, they can translocate to the nucleus and activate transcription of target genes (see Figure 9-45).
- DNase I hypersensitive sites (DHSs) indicate the positions of transcription-factor binding in chromatin, although they do not indicate which transcription factor is bound. Nonetheless, mapping of DHSs in differentiating cells gives an overview of how transcription-factor-binding sites change as a cell differentiates into a specific cell type.
- In metazoans, RNA polymerase II often pauses during elongation within approximately 50-100 base pairs from the transcription start site. Release from this pause contributes to the regulation of gene transcription.
- Resumption of elongation by Pol II paused in the promoterproximal region is also required for gene transcription and is a regulated step.
- In most cases, Pol II does not terminate transcription until after a sequence is transcribed that directs cleavage and polyadenylation of the RNA.


### 9.7 Epigenetic Regulation of Transcription

The term epigenetics refers to the study of inherited changes in the phenotype of a cell that do not result from changes in DNA sequence. For example, during the differentiation of bone marrow stem cells into the several different types of blood cells, a hematopoietic stem cell divides into two daughter cells, one of which continues to have the properties of a hematopoietic stem cell, including the potential to differentiate into all the different types of blood cells. But the other daughter cell becomes either a lymphoid progenitor cell or a myeloid progenitor cell (see Figure 21-17). Lymphoid progenitor cells generate daughter cells that differentiate into lymphocytes, which perform many of the functions involved in immune responses to pathogens (see Chapter 23). Myeloid progenitor cells divide into daughter cells that are committed to differentiating into red blood cells, different kinds of phagocytic white blood cells, or the cells that generate platelets involved in blood clotting. Lymphoid and myeloid progenitor cells both have the same DNA sequence as the zygote (generated by fertilization of an egg cell by a sperm cell) from which they developed, but they have restricted developmental potential because of epigenetic differences between them.

Such epigenetic changes are initially the consequence of the expression of specific master transcription factors that are regulators of cellular differentiation, controlling the expression of other genes that encode transcription factors and proteins involved in cell-cell communication in complex networks of gene control, and which are currently the subject of intense investigation. Changes in gene expression initiated by transcription factors are often reinforced and maintained over multiple cell divisions by post-translational modifications of histones and methylations of DNA at position 5 of the cytosine pyrimidine ring (see Figure 2-17) that are maintained and propagated to daughter cells when cells divide. Consequently, the term epigenetic marks is used to refer to such post-translational modifications of histones and 5 -methyl C modification of DNA.

## DNA Methylation Represses Transcription

As mentioned earlier, most promoters in mammals fall into the CpG island class. Active CpG island promoters have Cs in their CG sequences that are unmethylated. Unmethylated CpG island promoters have reduced affinity for histone octamers, but nucleosomes immediately neighboring the unmethylated promoters are modified by histone H3 lysine 4 di- or trimethylation and are associated with Pol II molecules that are paused during transcription of both the sense and antisense template DNA strands, as discussed earlier (see Figures 9-18 and 9-19). Recent research indicates that methylation of histone H3 lysine 4 occurs in mouse cells because a protein named Cfp1 (CXXC finger protein 1) binds unmethylated CpG-rich DNA through a zinc-finger domain (CXXC) and associates with a histone methylase specific for histone H3 lysine 4 (Setd1). Chromatin-remodeling complexes and the general transcription factor TFIID, which initiates Pol II preinitiation complex assembly (see Figure 9-19), associate with nucleosomes bearing the H3 lysine 4 trimethyl mark, promoting Pol II transcription initiation.

In differentiated cells, however, a small percentage of specific CpG island promoters, depending on the cell type, have CpGs marked by 5 -methyl C. This modification of CpG island DNA triggers chromatin condensation. A family of proteins that bind to DNA that is rich in 5 -methyl C-modified CpGs (called methyl CpG-binding proteins, or MBDs) bind to the marked promoters and associate with histone deacetylases and repressive chromatin-remodeling complexes that condense chromatin, resulting in transcriptional repression. The 5 -methyl C is added to the CpGs by DNA methyl transferases named DNMT3a and DNMT3b. They are referred to as de novo DNA methyl transferases because they methylate an unmethylated C. Much remains to be learned about how DNMT3a and b are directed to specific CpG islands. But once they have methylated a DNA sequence, methylation at that C is passed on through DNA replication through the action of the ubiquitous maintenance methyl transferase DNMT1:

![li2732239539b](li2732239539b.jpg)
(red indicates daughter strands). As a consequence, once a CpG island promoter is methylated by DNMT3a or b, it continues to be methylated by DNMT1 in subsequent daughter cells. Consequently, the promoter remains repressed in all subsequent daughter cells through interactions with MBDs, even after the stimulus for the initial C-methylation by DNMT3a or b has ceased. Therefore, repression of C-methylated promoters is inherited through cell division. This mechanism of epigenetic repression is being intensely investigated because tumor-suppressor genes encoding proteins that function to suppress the development of cancer are often inactivated in cancer cells by abnormal CpG methylation of their promoter regions, as discussed further in Chapter 24.

## Methylation of Specific Histone Lysines Is Linked to Epigenetic Mechanisms of Gene Repression

Figure 8-26b summarized the different types of posttranslational modifications that are found on histones, including acetylation of lysines and methylation of lysines on the nitrogen atom of the terminal $\varepsilon$-amino group of the lysine side chain (see Figure 2-14). Lysines can be modified by the addition of one, two, or three methyl groups to this terminal nitrogen atom, generating mono-, di-, and trimethylated lysine, all of which carry a single positive charge.

The acetylation state at a specific histone lysine on a particular nucleosome results from a dynamic equilibrium between acetylation and deacetylation by histone acetylases and histone deacetylases, respectively. Acetylation of histones in a localized region of chromatin predominates when local DNAbound activators transiently bind histone acetylase complexes. Deacetylation predominates when repressors transiently bind histone deacetylase complexes. Pulse-chase radiolabeling experiments have shown that acetyl groups on histone lysines turn over rapidly through the sequential actions of histone acetylases and histone deacetylases. In contrast, methyl groups on histones are much more stable. Histone lysine methyl groups can be removed by histone lysine demethylases. But the resulting turnover of histone lysine methyl groups is much slower than the turnover of histone lysine acetyl groups, which makes methylation the more appropriate post-translational modification for propagating epigenetic information.

Several other post-translational modifications of histones have been characterized (see Figure 8-26b). These modifications all have the potential to positively or negatively regulate the binding of proteins that interact with the chromatin fiber to regulate transcription as well as other processes, such as chromosome folding into the highly condensed structures that form during mitosis (see Figures 8-35 and 8-36). A picture of chromatin has emerged in which histone tails extending as random coils from the chromatin fiber are
post-translationally modified to generate one of many possible combinations of modifications that regulate transcription and other processes by regulating the binding of a large number of different protein complexes. This control of the interactions of proteins with specific regions of chromatin that results from the combined influences of various posttranslational modifications of histones has been called a histone code. Some of these modifications, such as histone lysine acetylation, are rapidly reversible, whereas others, such as histone lysine methylation, can be templated through chromatin replication, generating epigenetic inheritance in addition to inheritance of DNA sequence. Table 9-3 summarizes the influence that post-translational modifications of specific histone amino acid residues usually have on transcription.

Histone H3 Lysine 9 Methylation in Heterochromatin In most eukaryotes, some co-repressor complexes contain histone methyl transferase subunits that methylate histone H3 at lysine 9, generating di- and trimethyl lysines. These methylated lysines are binding sites for isoforms of HP1 protein that function in the condensation of heterochromatin, as discussed in Chapter 8 (see Figure 8-29). For example, the KAP1 co-repressor complex functions with a class of more than 200 zinc-finger transcription factors encoded in the human genome. This co-repressor complex includes an H3 lysine 9 methyl transferase that methylates nucleosomes over the promoter regions of repressed genes, leading to HP1 binding and repression of transcription. An integrated transgene in cultured mouse fibroblasts that was repressed through the action of the KAP1 co-repressor was associated with heterochromatin in most cells, whereas the active form of the same transgene was associated with euchromatin (Figure 9-46). Chromatin immunoprecipitation assays (see Figure 9-18) showed that the repressed gene was associated with histone H3 methylated at lysine 9 and with HP1, whereas the active gene was not.

Importantly, H3 lysine 9 methylation is maintained following chromosome replication by the mechanism diagrammed in Figure 9-47. When a methylated region of DNA is replicated in $S$ phase, the histone octomers associated with the parent DNA are randomly distributed to the daughter DNA molecules. New histone octamers that are not methylated on lysine 9 also associate randomly with the new daughter chromosomes, but since the parent histone octomers are associated with both daughter chromosomes, approximately half of the daughter chromosomes' nucleosomes are methylated on lysine 9. Association of histone H3 lysine methyl transferases (directly or indirectly) with the parent methylated nucleosomes leads to methylation of the newly assembled histone octamers. Repetition of this process with each cell division results in maintenance of H3 lysine 9 methylation of this region of the chromosome.

| Modification | Sites of Modification | Effect on Transcription |
| :--: | :--: | :--: |
| Acetylated lysine | H3 (K9, K14, K18, K27, K56) | Activation |
|  | H4 (K5, K8, K13, K16) | Activation |
|  | H2A (K5, K9, K13) | Activation |
|  | H2B (K5, K12, K15, K20) | Repression |
| Hypoacetylated lysine |  | Repression |
| Phosphorylated serine/threonine | H3 (T3, S10, S28) | Activation |
|  | H2A (S1, T120) | Activation |
|  | H2B (S14) | Activation |
| Methylated arginine | H3 (R17, R23) | Activation |
|  | H4 (R3) | Activation |
| Methylated lysine | H3 (K4) Me3 in promoter region |  |
|  | H3 (K4) Me1 in enhancers | Activation |
|  | H3 (K36, K79) in transcribed region | Elongation |
|  | H3 (K9, K27) | Repression |
|  | H4 (K20) | Repression |
| Ubiquitinylated lysine | H2B (K120 in mammals, K123 in S. cerevisiae) | Activation |
|  | H2A (K119 in mammals) | Repression |

![li2742239539b](li2742239539b.jpg)

FIGURE 9-46 Association of a repressed transgene with heterochromatin. Mouse fibroblasts were stably transformed with a transgene that contained binding sites for an engineered repressor. The repressor was a fusion between a DNA-binding domain, a repression domain that interacts with the KAP1 co-repressor complex, and the ligand-binding domain of a nuclear receptor that allows the nuclear import of the fusion protein to be controlled experimentally (see Figure 9-45). DNA was stained blue with the dye DAPI. Brighter-staining regions are regions of heterochromatin, where the DNA concentration is higher than in euchromatin. The transgene was detected by hybridization of a fluorescently labeled complementary probe (green). When the recombinant

## Epigenetic Control by Polycomb and Trithorax Complexes

Another type of epigenetic mark that is essential for repression of genes in specific cell types in multicellular animals and plants involves a set of proteins known collectively as
repressor was retained in the cytoplasm, the transgene was transcribed (left) and was associated with euchromatin in most cells. When hormone was added so that the recombinant repressor entered the nucleus, the transgene was repressed (right) and associated with heterochromatin. Chromatin immunoprecipitation assays (see Figure 9-18) showed that the repressed gene was associated with histone H 3 methylated at lysine 9 and HP1, whereas the active gene was not. [From Ayyanathan, K. et al., "Regulated recruitment of HP1 to a euchromatic gene induces mitotically heritable, epigenetic gene silencing: a mammalian cell culture model of gene variegation," Genes and Development, 2003,17:1855-1869. Courtesy of Frank Rauscher; republished with permission from Cold Spring Harbor Laboratory Press.]

Polycomb proteins and a counteracting set of proteins known as Trithorax proteins. These names were derived from the phenotypes of mutations in the genes encoding these proteins in Drosophila, in which they were first discovered. The Polycomb repression mechanism is essential for maintaining the repression of genes in specific types of cells, and in

![li2752239539b](li2752239539b.jpg)

FIGURE 9-47 Maintenance of histone H3 lysine 9 methylation during chromosome replication. When chromosomal DNA is replicated, the parent histones randomly associate with the two daughter chromosomes, while unmethylated histones synthesized during S phase are assembled into other nucleosomes in those same daughter chromosomes. Association of histone H3 lysine 9 methyl transferases (H3K9 HMT) with parent nucleosomes bearing the histone 3 lysine 9 di- or trimethylation mark methylates the newly added unmodified nucleosomes. Consequently, histone H3 lysine 9 methylation marks are maintained during repeated cell divisions unless they are specifically removed by a histone demethylase.
all the subsequent cells that develop from them, throughout the life of an organism. Important genes regulated by Polycomb proteins include the Hox genes, which encode master regulatory transcription factors. Different combinations of Hox transcription factors help to direct the development of specific tissues and organs in a developing embryo. Early in embryogenesis, expression of Hox genes is controlled by typical activator and repressor proteins. However, the expression of these activators and repressors stops at an early point in embryogenesis. Correct expression of the Hox genes in the descendants of the early embryonic cells is then maintained throughout the remainder of embryogenesis and on into adult life by the Polycomb proteins, which maintain the repression of specific Hox genes. Trithorax proteins perform the opposite function, maintaining the expression of the Hox genes that were expressed in a specific cell early in embryogenesis in all the subsequent descendants of that cell. Polycomb and Trithorax proteins control thousands of genes, including genes that regulate cell growth and division (i.e., the cell cycle, as discussed in Chapter 19). Polycomb and Trithorax genes are often mutated in cancer cells, contributing importantly to the abnormal properties of these cells (see Chapter 24).

Remarkably, virtually all cells in the developing embryo and adult express a similar set of Polycomb and Trithorax proteins, and all cells contain the same set of Hox genes. Yet only the Hox genes in cells where they were initially
repressed in early embryogenesis remain repressed, even though the same Hox genes in other cells remain active in the presence of the same Polycomb proteins. Consequently, as in the case of the yeast silent mating-type loci, the expression of Hox genes is regulated by a process that involves more than specific DNA sequences interacting with proteins that diffuse through the nucleoplasm.

A current model for repression by Polycomb proteins is depicted in Figure 9-48. Most Polycomb proteins are subunits of one of two classes of multiprotein Polycomb repressive complexes: PRC1 and PRC2. The PRC2 complexes are thought to act initially by associating with the repression domains of specific repressors bound to their cognate DNA sequences early in embryogenesis, or with ribonucleoprotein complexes containing long noncoding RNAs, as discussed in a later section. The PRC2 complexes contain histone deacetylases that inhibit transcription, as discussed above. They also contain a subunit [E(z) in Drosophila, EZH2 in mammals] with a SET domain, which is the catalytic domain of several histone methyl transferases. This SET domain in PRC2 complexes methylates histone H3 on lysine 27, generating di- and trimethyl lysines. A PRC1 complex then binds the methylated nucleosomes through dimeric Pc subunits (CBXs in mammals), each containing a methyl lysine-binding domain (called a chromodomain) specific for methylated H3 lysine 27. Binding of the dimeric Pc to neighboring nucleosomes is proposed to condense the chromatin into a structure that inhibits transcription. This proposal is supported by electron microscopy studies showing that PRC1 complexes cause nucleosomes to associate in vitro (Figure 9-48d, e).

PRC1 complexes also repress transcription through additional mechanisms. The PRC1 complex contains a ubiquitin ligase that monoubiquitinylates histone H2A at lysine 119 in the H2A C-terminal tail (see Figure 8-26). This modification of H2A inhibits elongation by inhibiting a histone chaperone that removes histone octamers from DNA as Pol II transcribes through a nucleosome, then replaces them as the polymerase passes. PRC1 also associates with a histone demethylase that specifically removes methyl groups from lysine 4 of histone H 3 , an activating mark discussed above.

PRC2 complexes associate with nucleosomes bearing the histone H3 lysine 27 trimethylation mark, maintaining methylation of H3 lysine 27 in nucleosomes in the region. This methylation results in association of the chromatin with PRC1 and PRC2 complexes even after expression of the initial repressor proteins shown in Figure 9-48a, b has ceased. This association maintains H3 lysine 27 methylation by a mechanism analogous to that diagrammed in Figure 9-47. This mechanism is a key feature of Polycomb repression, which is maintained through successive cell divisions for the life of an organism ( 100 years for some vertebrates, 2000 years for a sugar cone pine!).

Trithorax proteins counteract the repressive mechanism of Polycomb proteins, as shown in studies of expression of the Hox transcription factor Abd-B in the Drosophila embryo (Figure 9-49). Abd-B is normally expressed only in posterior segments of the developing embryo. When the

![li2762239539b](li2762239539b.jpg)

FIGURE 9-48 Model for repression by Polycomb complexes.
(a) During early embryogenesis, repressors associate with the PRC2 complex. (b) This association results in methylation (Me) of neighboring nucleosomes on histone H3 lysine 27 (K27) by the SET domaincontaining subunit E(z). (c) The PRC1 complex binds nucleosomes methylated at H3 lysine 27 through a dimeric, chromodomaincontaining subunit Pc. The PRC1 complex condenses the chromatin into a repressed chromatin structure. PRC2 complexes associate with PRC1 complexes to maintain H3 lysine 27 methylation of neighboring
histones. As a consequence, PRC1 and PRC2 association with the region is maintained when expression of the repressor proteins in (a) ceases. (d, e) Electron micrograph of a 1-kb fragment of DNA bound by four nucleosomes in the absence (d) and presence (e) of one PRC1 complex per five nucleosomes. See A. H. Lund and M. van Lohuizen, 2004, Curr. Opin. Cell Biol. 16:239; and N. J. Francis, R. E. Kingston, and C. L. Woodcock, 2004, Science 306:1574. [Parts (d) and (e) republished with permission of AAAS, from Francis, N.J. et al., "Chromatin compaction by a polycomb group protein complex," Science, 2004, 306(5701):1574-7; permission conveyed through Copyright Clearance Center, Inc.]

![li2772239539b](li2772239539b.jpg)

FIGURE 9-49 Opposing influence of Polycomb and Trithorax complexes on expression of the Hox transcription factor Abd-B in Drosophila embryos. At the stage of Drosophila embryogenesis shown, Abd-B is normally expressed only in posterior segments of the developing embryo, as shown at the top (wt) by immunostaining with a specific anti-Abd-B antibody. In embryos with homozygous mutations of Scm, a Polycomb gene (PcG) encoding a protein associated with the PRC1 complex, Abd-B expression is derepressed in all embryo segments. In contrast, in homozygous mutants of $t r x$, a Trithorax gene ( $t r x G$ ), Abd-B repression is increased so that the protein is expressed at high concentrations only in the most posterior segment. [From Klymenko, T., and Muller, J., "The histone methyltransferases Trithorax and Ash1 prevent transcriptional silencing by Polycomb group proteins," EMBO Reports ©2004 John Wiley and Sons. Reproduced with permission of Wiley-VCH.]

Polycomb system is defective, Abd-B is expressed in all cells of the embryo. When the Trithorax system is defective and cannot counteract repression by the Polycomb system, Abd-B is repressed in most cells, except those in the very posterior of the embryo. Trithorax complexes include a histone methyl transferase that trimethylates histone H3 lysine 4, a histone methylation that is associated with the promoters of actively transcribed genes. This histone modification creates a binding site for histone acetylase and for chromatinremodeling complexes that promote transcription, as well as for TFIID, the general transcription factor that initiates preinitiation-complex assembly (see Figure 9-19). Nucleosomes with H3 lysine 4 methylation are also binding sites for specific histone demethylases that remove H3 histone K9 and K27 methylation, preventing the binding of HP1 and the Polycomb repressive complexes. Nucleosomes marked with H3 lysine 4 methylation are also thought to be distributed to both daughter DNA molecules during DNA replication, resulting in maintenance of this epigenetic mark by a strategy similar to that diagrammed in Figure 9-47.

## Long Noncoding RNAs Direct Epigenetic Repression in Metazoans

Repressive complexes have been discovered that are composed of multiple repressing proteins bound to RNAs many kilobases in length that do not contain long open reading frames and are consequently called long noncoding RNAs or lncRNAs. In some cases, these lncRNA-protein complexes repress genes on the same chromosome from which the RNA is transcribed, as in the case of X-chromosome inactivation in female mammals. In other cases, these repressive RNA-protein complexes act in trans, repressing genes on chromosomes other than those from which the lncRNA is transcribed.

X-Chromosome Inactivation in Mammals The phenomenon of X-chromosome inactivation in female mammals (see Chapter 8) is one of the most intensely studied examples of epigenetic repression mediated by a lncRNA. X inactivation is controlled by a roughly $100-\mathrm{kb}$ domain on the X chromosome called the X-inactivation center. Remarkably, this region encodes several lncRNAs required for the random inactivation of one entire X chromosome early in the development of female mammals. The functions of these lncRNAs are only partially understood. The most intensively studied are transcribed from the complementary DNA strands near the middle of the X-inactivation center: the $40-\mathrm{kb}$ TSIX lncRNA and the XIST RNA, which is spliced and polyadenylated into an RNA of about 17 kb that is not exported to the cytoplasm (Figure 9-50a).

In differentiated female cells, the inactive X chromosome is associated with XIST RNA-protein complexes along its entire length (Figure 9-50b). Targeted deletion of the Xist gene (see Figure 6-39) in cultured embryonic stem cells showed that it is required for X inactivation. Unlike most protein-coding genes on the inactive X chromosome, the Xist gene is actively transcribed. The XIST RNA-protein complexes do not diffuse to interact with the active X or other chromosomes, but remain associated with the inactive X chromosome. Since the full length of the inactive X becomes coated by XIST RNA-protein complexes (see Figure 9-50b), these complexes must spread along the chromosome from the X-inactivation center where XIST is transcribed. In contrast to XIST, TSIX is transcribed from the active X chromosome, not from the inactive X chromosome.

In the early female mouse embryo, made up of embryonic stem cells capable of differentiating into all cell types (see Chapter 21), genes on both X chromosomes are transcribed, and the $40-\mathrm{kb}$ TSIX lncRNA (see Figure 9-50a) is transcribed from both copies of the X chromosome. Experiments employing engineered deletions in the X-inactivation center showed that TSIX transcription prevents significant transcription of the XIST RNA from the complementary DNA strand. Later in development, as cells begin to differentiate, TSIX transcription is repressed on one of the X chromosomes. This repression occurs randomly in different cells on the X chromosome derived from the sperm $\left(\mathrm{X}_{\mathrm{p}}\right)$ or on the X chromosome derived from the egg $\left(\mathrm{X}_{\mathrm{m}}\right)$. This inhibition of TSIX transcription determines which of the X chromosomes will be inactivated as the cells differentiate further because inhibition of TSIX transcription allows transcription of the XIST lncRNA on that chromosome.

The transcribed XIST RNA contains RNA sequences that, by unknown mechanisms, cause it to spread along the X chromosome. Recent studies indicate that XIST lncRNA-protein complexes first associate with regions of the X chromosome localized near the X -inactivation center in the three-dimensional, folded structure of the future inactive X (Figure 9-50c), as shown by chromosome conformation capture assays (see Figure 8-34). These initial sites of XIST association are in gene-rich regions of the X chromosome and are postulated to serve as "entry sites" where additional copies of the XIST lncRNA-protein complexes first bind and then spread to neighboring regions. The mechanism of spreading is not currently understood. The inactive X chromosome also becomes associated with PRC2 complexes, which catalyze the trimethylation of histone H3 lysine 27. This methylation results in association of the PRC1 complex and transcriptional repression, as discussed above. These mechanisms of transcriptional repression must be redundant, however, because repression still occurs in the absence of the Polycomb proteins essential for the assembly of PRC1 and PRC2. At the same time, continued transcription of TSIX from the other, active X chromosome continues, represses XIST transcription from that X chromosome, and consequently prevents XIST-mediated repression of the active X. XIST and PRC1 and 2 complexes are then observed to associate with gene-poor regions of the inactive X chromosome as well as with gene-rich regions.

Recent analysis by protein mass spectrometry (see Chapter 3) of proteins associated with XIST lncRNA during the initiation phase of X inactivation in cultured mouse embryonic stem cells revealed that SMRT, a protein first characterized as a co-repressor that interacts with the thyroid hormone nuclear receptor in the absence of hormone, is part of the protein complex that interacts with XIST RNA. SMRT, in turn, interacts with a histone deacetylase (HDAC3). Subsequent knockdown experiments with siRNAs directed against SMRT

![li2782239539b](li2782239539b.jpg)
and HDAC3 showed that they are required for X inactivation, as are other identified RNA- and chromatin-binding proteins that link SMRT to XIST RNA and are required for the association of XIST RNA and PRC2 with the inactive X chromosome (Figure 9-50d). A short time later in development, the DNA of the inactive X also becomes methylated at most of its CpG island promoters. Specialized histone octamers in which histone H 2 A is replaced by a paralog of H 2 A called macroH 2 A also become associated with the inactive X. DNA methylation and macroH2A contribute to the stable repression of the inactive

FIGURE 9-50 The Xist long noncoding RNA encoded in the X -inactivation center coats the inactive $X$ chromosome in cells of mammalian females, repressing transcription of most genes on the inactive $X$. (a) The region of the human $X$-inactivation center encoding the noncoding RNAs Xist (transcribed from the inactive X), and Tsix (transcribed from the active X). Numbers are base pairs from the left end of the X chromosome. (b) A cultured fibroblast from a human female was analyzed by in situ hybridization with a probe complementary to Xist RNA labeled with a red fluorescent dye (left), a chromosome paint set of probes for the X chromosome labeled with a green fluorescent dye (center), and an overlay of the two fluorescent micrographs. The condensed inactive $X$ chromosome is associated with Xist RNA. (c) Model for the spreading of the Xist lncRNA-protein complex on the inactive $X$ chromosome during early differentiation of female embryonic stem cells. See E. Heard and A.-V. Gendrel, 2014, Annu. Rev. Cell Dev. Biol. 30:561. (d) Proteins associated with Xist IncRNA. Question marks indicate that it is not yet known how PRC2 complexes associate with HDAC3 and the RNA-binding protein SHARP. See C. A. McHugh et al., 2015, Nature 521:232. [Part (b) ©1996 C. M. Clemson et al., The Journal of Cell Biology, 132:259-275. doi: 10.1083/jcb.132.3.259.]

X through the multiple cell divisions that occur later during embryogenesis and throughout adult life.

Trans Repression by Long Noncoding RNAs Another example of transcriptional repression by a long noncoding RNA was discovered recently by researchers studying the function of noncoding RNAs transcribed from a region encoding a cluster of Hox genes, the HOXC locus, in cultured human fibroblasts. Depletion of a $2.2-\mathrm{kb}$ noncoding RNA expressed from the HOXC locus by siRNA (see Figure 6-42) unexpectedly led to derepression of the HOXD locus, a roughly $40-\mathrm{kb}$ region on another chromosome encoding several Hox proteins and multiple other noncoding RNAs, in these cells. Assays similar to chromatin immunoprecipitation showed that this noncoding RNA, named HOTAIR (for Hox Antisense Intergenic RNA), associates with the HOXD loci and with PRC2 complexes. This association results in histone H3 lysine 27 di- and trimethylation, PRC1 association, histone H3 lysine 4 demethylation, histone H2A monoubiquitinylation, and transcriptional repression. This process is similar to the recruitment of Polycomb complexes by Xist RNA, except that Xist RNA functions in cis, remaining in association with the chromosome from which it is transcribed, whereas HOTAIR leads to Polycomb repression in trans on both copies of another chromosome. Once again, redundant mechanisms for repression of these HOXD loci must exist, because extensive, but less complete, repression at the HOXD locus continues in the appropriate cells in mouse embryos with homozygous HOTAIR knockout mutations.

Cis Activation by Long Noncoding RNAs Examples of lncRNAs involved in gene activation have been characterized recently. For example, HOTTIP lncRNA, which is transcribed from the $5^{\prime}$ end of the HOXA locus, is proposed to coordinate the activation of HOXA genes by binding to a histone H3 lysine 4 methylase. In addition, nascent

transcripts of lncRNA genes have been reported to activate transcription from promoters several kilobases away by interacting with the Mediator complex and delivering it to the promoter by looping of the intervening chromatin.

In humans, but not in mice, a lncRNA called XACT has been discovered to associate with multiple sites along the full length of the active X chromosome and is postulated to contribute to maintenance of gene activity on that chromosome. XACT is also remarkable for being one of the longest characterized RNAs: 252 kb ! It is mostly unspliced.

In Drosophila, equal expression of genes encoded on the X chromosome in males and females (dosage compensation) does not result from inactivating one X chromosome in females. Rather, a generalized twofold increase in transcriptional activation of genes on the single X chromosome in males is controlled by two lncRNAs, roX1 and roX2, transcribed from the X chromosome in males only. The roX1 and roX2 RNAs associate with several proteins encoded by MSL (male-specific-lethal) genes and spread over the X chromosome specifically, much as Xist lncRNA-protein complexes spread over the inactive X in mammals.

Recently, sequencing of total cellular RNA in multiple types of human cells identified roughly 15,000 human lncRNAs. Many of these lncRNAs have sequences that are evolutionarily conserved in most mammals, and about 5000 are found only in primates. This conservation of sequence strongly suggests that these lncRNAs, like XIST, HOTAIR, and HOTTIP, have important functions. Multiple lncRNAs are expressed only in specific cell types at specific times during development. For example, multiple lncRNAs are expressed primarily in differentiating red blood cells. Knockdown (see Figure 6-42 and Chapter 10) of several of these lncRNAs inhibits normal red blood cell development, but precisely how these lncRNAs perform their essential functions is not yet clear. The study of these conserved long noncoding RNAs and how they influence gene expression is another area of intense current investigation.

ENCODE (Encyclopedia of DNA Elements) encompasses a consortium of international research groups organized and funded by the US National Human Genome Research Institute with the goal of building a comprehensive, publically available database of human DNA control elements and the transcription factors that bind to them in different cell types, histone post-translational modifications mapped by ChIP-seq and other related methods, DNase I hypersensitive sites, and regulatory lncRNAs and their sites of association in the genome, as well as newly discovered regulatory elements "that control cells and circumstances in which a gene is active." Data sets from human cells and cells of model organisms that are too large to be published are also made publically available at a site called GEO (Gene Expression Omnibus) maintained by the US National Center for Bioinformatics (NCBI). Most journals that publish research based on genomic methods such as RNA-seq and ChIP-seq require that authors upload their original data to GEO. Worldwide public access to these data sets is greatly accelerating the pace of discovery in the area of gene regulation.

## KEY CONCEPTS OF SECTION 9.7

## Epigenetic Regulation of Transcription

- Epigenetic control of transcription refers to repression or activation that is maintained after cells replicate as the result of DNA methylation or post-translational modification of histones, especially histone methylation.
- Methylation of CpG sequences in CpG island promoters in mammals generates binding sites for a family of methylbinding proteins (MBDs) that associate with histone deacetylases, inducing hypoacetylation of the promoter regions and transcriptional repression.
- Histone H3 lysine 9 di- and trimethylation creates binding sites for the heterochromatin-associated protein HP1, which results in the condensation of chromatin and transcriptional repression. These post-translational modifications are perpetuated following chromosome replication because the methylated histones are randomly associated with the daughter DNA molecules and associate with histone H3 lysine 9 methyl transferases that methylate histone 3 lysine 9 on newly synthesized histone octamers assembled on the daughter DNA.
- Polycomb complexes maintain repression of genes initially repressed by sequence-specific repressors expressed early during embryogenesis. One class of Polycomb repressive complexes, PRC2 complexes, associates with these repressors in early embryonic cells, resulting in methylation of histone H3 lysine 27. This methylation creates binding sites for subunits in the PRC2 complex as well as for PRC1 complexes, which condense chromatin, inhibit the assembly of preinitiation complexes, and inhibit elongation. Since parent histone octamers with H3 methylated at lysine 27 are distributed to both daughter DNA molecules following DNA replication, PRC2 complexes that associate with these nucleosomes maintain histone H3 lysine 27 methylation through cell division.
- Trithorax complexes oppose repression by Polycomb complexes by methylating H3 at lysine 4 and maintaining this activating mark through chromosome replication.
- X-chromosome inactivation in female mammals requires a long noncoding RNA (lncRNA) called Xist that is transcribed from the X-inactivation center of one X chromosome and then spreads by a poorly understood mechanism along the length of the same chromosome. Xist interacts with a co-repressor that binds a histone deacetylase and PRC2 complexes at an early stage of embryogenesis, initiating X inactivation. X inactivation is maintained throughout the remainder of embryogenesis and adult life by continued association with Polycomb complexes and DNA methylation of CpG island promoters on the inactive X.
- Some lncRNAs have been discovered that lead to repression of genes in trans, as opposed to the cis inactivation imposed by Xist. Repression is initiated by their interaction with PRC2 complexes.

- Some lncRNAs are associated with gene activation. Much remains to be learned about how lncRNAs are targeted to specific chromosomal regions, but the discovery of about 15,000 nuclear lncRNAs expressed in specific types of human cells during specific stages of their differentiation suggests that lncRNAs are central to widely used mechanisms of transcription regulation.


### 9.8 Other Eukaryotic Transcription Systems

We conclude this chapter with a brief discussion of transcription initiation by the other two eukaryotic nuclear RNA polymerases, Pol I and Pol III. The distinct polymerases that transcribe mitochondrial and chloroplast DNA will be discussed in Chapter 12, on cellular energetics. Although these systems, and particularly their regulation, are less thoroughly understood than transcription by RNA polymerase II, they are equally fundamental to the life of eukaryotic cells.

## Transcription Initiation by Pol I and Pol III Is Analogous to That by Pol II

The formation of transcription initiation complexes involving Pol I and Pol III is similar in some respects to assembly of Pol II initiation complexes (see Figure 9-19). However, each of the three eukaryotic nuclear RNA polymerases requires its own polymerase-specific general transcription factors and recognizes different DNA control elements. Moreover, neither Pol I nor Pol III requires ATP hydrolysis by a DNA helicase to help melt the DNA template strands to initiate transcription, whereas Pol II does. Transcription initiation by Pol I, which synthesizes pre-rRNA, and by Pol III, which synthesizes
![li2792239539b](li2792239539b.jpg)
tRNAs, 5S rRNA, and other small stable RNAs (see Table 9-2), is tightly coupled to the rate of cell growth and proliferation.

Initiation by Pol I The regulatory elements directing Pol I initiation are similarly located relative to the transcription start site in yeast and in mammals. A core element spanning the transcription start site from -40 to +5 is essential for Pol I transcription. An additional upstream control element extending from roughly -155 to -60 increases in vitro Pol I transcription tenfold. In humans, assembly of the Pol I preinitiation complex (Figure 9-51) is initiated by the cooperative binding of UBF (upstream binding factor) and SL1 (selectivity factor), a multisubunit factor containing TBP and four Pol I-specific TBP-associated factors $\left(\mathrm{TAF}_{I} \mathrm{~S}\right)$, to the Pol I promoter region. The $\mathrm{TAF}_{I}$ subunits interact directly with Pol I-specific subunits, directing this specific nuclear RNA polymerase to the transcription start site. TIF-1A, the mammalian homolog of $S$. cerevisiae RRN3, is another required factor, as are the abundant nuclear protein kinase CK2 (casein kinase 2), nuclear actin, nuclear myosin, the protein deacetylase SIRT7, and topoisomerase I, which prevents DNA supercoils (see Figure 5-8) from forming during rapid Pol I transcription of the $14-\mathrm{kb}$ transcription unit.

Transcription of the $14-\mathrm{kb}$ precursor of $18 \mathrm{~S}, 5.8 \mathrm{~S}$, and 28 S rRNAs (see Chapter 10) is highly regulated to coordinate ribosome synthesis with cell growth and division. This coordination is achieved through regulation of the activities of the Pol I initiation factors by post-translational modifications, including phosphorylation and acetylation at specific sites, control of the rate of Pol I elongation, and control of the number of the 300 or so human rRNA genes that are transcriptionally active by epigenetic mechanisms that assemble inactive copies into heterochromatin. Switching between the active and heterochromatic silent states of rRNA genes is accomplished by a multisubunit chromatin-remodeling complex called NoRC

FIGURE 9-51 Transcription of the rRNA precursor RNA by RNA polymerase L (Top) Electron micrograph of RNA-protein complexes transcribed from one copy of the repeated rRNA genes. (Middle) A single Pol I transcription unit. Enhancers that stimulate Pol I transcription from a single transcription start site are represented by blue boxes. Pol I transcription termination sites $\left(T_{0}, T_{1}-T_{10}\right)$ bound by the Pol I-specific termination factor TTF-1 are shown as red rectangles. pRNA indicates transcription of the noncoding pRNA required for transcriptional silencing. The sequences of regions of DNA shown as yellow rectangles are retained during processing of 18S, 5.8 S and 28 S rRNAs. The other regions transcribed from the black arrow to the red termination sites are removed and degraded. (Bottom) The core promoter element and upstream control element are shown with the location of Pol I and its general transcription factors UBF, SL1, and TIF-1A represented, as well as other proteins required for Pol I elongation and control. See I. Grummt, 2010, FEBS J. 277:4626. [Electron micrograph courtesy Ann L. Beyer.]

("No" for nucleolus, the site of rRNA transcription within nuclei). NoRC localizes a nucleosome over the Pol I transcription start site, blocking preinitiation complex assembly. It also interacts with a DNA methyl transferase that methylates a critical CpG in the upstream control element, inhibiting binding by UBF, as well as with histone methyl transferases that di- and trimethylate histone H3 lysine 9, creating binding sites for heterochromatic HP1, and with histone deacetylases. Moreover, a roughly 250 -nt noncoding RNA called pRNA (promoter-associated RNA) transcribed by Pol I from about 2 kb upstream of the rRNA transcription unit (red arrow in Figure 9-51) is bound by a subunit of NoRC and is required for transcriptional silencing. The pRNA is believed to target NoRC to Pol I promoter regions by forming an RNA:DNA triplex with the $\mathrm{T}_{0}$ terminator sequence. This creates a binding site for the DNA methyl transferase DNMT3b, which methylates the critical CpG in the upstream promoter element.

Initiation by Pol III Unlike those of protein-coding genes and pre-rRNA genes, the promoter regions of tRNA and 5S-rRNA genes lie entirely within the transcribed sequence (Figure 9-52a, b). Two such internal promoter elements, termed the $A$ box and the $B$ box, are present in all tRNA genes. These highly conserved sequences not only function as promoters, but also encode two invariant portions of eukaryotic tRNAs that are required for protein synthesis. In 5S-rRNA genes, a single internal control region, the $C$ box, acts as a promoter.

Three general transcription factors are required for Pol III to initiate transcription of tRNA and 5S-rRNA genes in vitro. Two multimeric factors, TFIIIC and TFIIIB, participate in initiation at both tRNA and 5S-rRNA promoters; a third factor, TFIIIA, is required for initiation at 5S-rRNA promoters. As with assembly of Pol I and Pol II initiation complexes, the Pol III general transcription factors bind to promoter DNA in a defined sequence.

The N-terminal half of one TFIIIB subunit, called BRF (for TFIIB-related factor), is similar in sequence to TFIIB (a Pol II factor). This similarity suggests that BRF and TFIIB perform a similar function in initiation, namely, to assist in separating the template DNA strands at the transcription start site. Once TFIIIB has bound to either a tRNA or a 5SrRNA gene, Pol III can bind and initiate transcription in the presence of ribonucleoside triphosphates. The BRF subunit of TFIIIB interacts specifically with one of the polymerase subunits unique to Pol III, accounting for initiation by this specific nuclear RNA polymerase.

Another of the three subunits composing TFIIIB is TBP, which we can now see is a component of a general transcription factor for all three eukaryotic nuclear RNA polymerases. The finding that TBP participates in transcription initiation by Pol I and Pol III was surprising, since the promoters recognized by these enzymes often do not contain TATA boxes. Nonetheless, in the case of Pol III transcription, the TBP subunit of TFIIIB interacts with DNA about 30 bp upstream of the transcription start site similarly to the way it interacts with TATA boxes.

Pol III also transcribes genes for small stable RNAs with upstream promoters containing a TATA box. One example is the gene for U6 snRNA, which is involved in pre-mRNA splicing, as discussed in Chapter 10. In mammals, this gene contains an upstream promoter element called the PSE in addition to the TATA box (Figure 9-52c). The PSE is bound by a multisubunit complex called SNAP $_{ C}$, while the TATA box is bound by the TBP subunit of a specialized form of TFIIIB containing an alternative BRF subunit.

MAF1 is a specific inhibitor of Pol III transcription that functions by interacting with the BRF subunit of TFIIIB and with Pol III. Its function is regulated by control of its import from the cytoplasm into the nucleus by phosphorylations at specific sites in response to signal transduction protein kinase cascades that respond to cell stress and nutrient deprivation
(a)
![li2802239539b](li2802239539b.jpg)
(b)
![li2812239539b](li2812239539b.jpg)
(c)
![li2822239539b](li2822239539b.jpg)

FIGURE 9-52 Transcription-control elements in genes transcribed by RNA polymerase III. Both tRNA (a) and 5S-rRNA (b) genes contain internal promoter elements (yellow) located downstream from the start site and named $A, B$, and $C$ boxes, as indicated. Assembly of transcription initiation complexes on these genes begins with the binding of Pol III-specific general transcription factors TFIIIA, TFIIIB, and TFIIIC to these control elements. Green arrows indicate strong, sequence-specific protein-DNA interactions. Blue arrows indicate interactions between general transcription factors. Purple arrows indicate interactions between general transcription factors and Pol III. (c) Transcription of the U6 snRNA gene in mammals is controlled by an upstream promoter with a TATA box bound by the TBP subunit of a specialized form of TFIIIB with an alternative BRF subunit and an upstream regulatory element called the PSE bound by a multisubunit factor called SNAP $_{C}$. See L. Schramm and N. Hernandez, 2002, Gene Dev. 16:2593.

(see Chapters 16 and 24). In mammals, Pol III transcription is also repressed by the critical tumor suppressors p53 and the retinoblastoma ( Rb ) family. In humans, there are two genes encoding RNA polymerase III subunit RPC32. One of these is expressed specifically in replicating cells, and its forced expression can contribute to oncogenic transformation of cultured human fibroblasts.

## KEY CONCEPTS OF SECTION 9.8

## Other Eukaryotic Transcription Systems

- The process of transcription initiation by Pol I and Pol III is similar to that by Pol II but requires different general transcription factors, is directed by different promoter elements, and does not require hydrolysis of ATP $\beta-\gamma$ phosphodiester bonds to separate the DNA strands at the start site as Pol II transcription does.
- Pol I transcribes only a single RNA, the 45 S precursor of 18S, 5.8 S , and 28 S rRNA, from multiple copies of the prerRNA gene.
- Pol III transcribes tRNAs from promoters within the genes that encode the tRNA regions common to all tRNAs. This internal promoter is bound by transcription factor TFIIIC, which in turn binds TFIIIB, a multisubunit factor that includes the TATA box-binding protein, TBP, which associates with the tRNA gene about 30 bp upstream of the transcription start site.
- Pol III transcribes 5 s rRNA directed by a promoter within the 5 S-rRNA coding region that is bound by transcription factor TFIIIA. TFIIIA then associates with TFIIIC and TFIIIB, which interact with Pol III in a manner similar to their interactions in tRNA transcription.
- Additional small stable RNAs, several with as yet unknown functions, are transcribed by Pol III as directed by TBP-containing transcription factors that bind immediately upstream of the genes (see Figure 9-52).
- Pol III transcription is regulated by a specific inhibitor, MAF1, whose transport from the cytoplasm into the nucleus is controlled in response to nutrient availability.


## LaunchPad

macmillan learning

Visit LaunchPad to access study tools and to learn more about the content in this chapter.

- Perspectives for the Future
- Analyze the Data
- Extended References
- Additional study tools, including videos, animations, and quizzes


## Key Terms

activation domain 382
activators 354
antitermination
factor 377
bromodomain 394
carboxy-terminal domain
(CTD) 370
chromatin-mediated
repression 390
chromodomain 407
co-activator 386
co-repressor 393
DNase I footprinting 380
enhanceosome 388
enhancers 359
general transcription
factors 373
heat-shock genes 402
histone deacetylation 393
leucine zipper 386
MAT locus (in yeast) 391
Mediator 390
nuclear receptors 386
promoter 364
promoter-proximal elements 378
repression domain 384
repressors 354
RNA polymerase II 367
silencer sequences 391
specific transcription
factors 380
TATA box 371
TATA box-binding protein (TBP) 374
upstream activating sequence (UAS) 380
zinc finger 385

## Review the Concepts

1. Describe the molecular events that occur at the lac operon when E. coli cells are shifted from a glucose-containing medium to a lactose-containing medium.
2. The concentration of free glutamine affects transcription of the enzyme glutamine synthetase in E. coli. Describe the mechanism of this effect.
3. Recall that the $t r p$ repressor binds to a site in the operator region of tryptophan-producing genes when tryptophan is abundant, thereby preventing transcription. What would happen to the expression of the tryptophan biosynthetic enzyme genes in the following scenarios? Fill in the blanks with one of the following phrases:
never be expressed/always (constitutively) be expressed
a. The cell produces a mutant trp repressor that cannot bind to the operator. The enzyme genes will
b. The cell produces a mutant trp repressor that binds to its operator site even if no tryptophan is present. The enzyme genes will $\qquad$ .
c. The cell produces a mutant sigma factor that cannot bind the promoter region. The enzyme genes will
d. Elongation of the leader sequence is always stalled after transcription of region 1 . The enzyme genes will
4. Compare and contrast bacterial and eukaryotic gene expression mechanisms.

5. What types of genes are transcribed by RNA polymerases I, II, and III? Design an experiment to determine whether a specific gene is transcribed by RNA polymerase II.
6. The CTD of the largest subunit of RNA polymerase II can be phosphorylated at multiple serine residues. What are the conditions that lead to the phosphorylated versus nonphosphorylated RNA polymerase II CTD?
7. What do TATA boxes, initiators, and CpG islands have in common? Which was the first of these to be identified? Why?
8. Describe the methods used to identify the location of transcription-control elements in promoter-proximal regions of genes.
9. What is the difference between a promoter-proximal element and a distal enhancer? What are the similarities?
10. Describe the methods used to identify the location of DNA-binding proteins in the regulatory regions of genes.
11. Describe the structural features of transcription activator and repressor proteins.
12. Give two examples of how gene expression may be repressed without altering the coding sequence.
13. Using CREB and nuclear receptors as examples, compare and contrast the structural changes that take place when these transcription factors bind to their coactivators.
14. What general transcription factors associate with an RNA polymerase II promoter in addition to the polymerase? In what order do they bind in vitro? What structural change occurs in the DNA when an "open" transcription initiation complex is formed?
15. Expression of recombinant proteins in yeast is an important tool for biotechnology companies that produce new drugs for human use. In an attempt to get a new gene $X$ expressed in yeast, a researcher has integrated gene $X$ into the yeast genome near a telomere. Will this strategy result in good expression of gene $X$ ? Why or why not? Would the outcome of this experiment differ if the experiment had been performed in a yeast line containing mutations in the H3 or H 4 histone tails?
16. You have isolated a new protein called STICKY. You can predict from comparisons with other known proteins that STICKY contains a bHLH domain and a Sin3-interacting domain. Predict the function of STICKY and explain the importance of these domains in STICKY function.
17. Prokaryotes and lower eukaryotes such as yeast have transcription-control elements called upstream activating sequences. What are the comparable sequences found in higher eukaryotic species?
18. You are curious to identify the region of the gene $X$ sequence that serves as an enhancer for gene expression. Design an experiment to investigate this issue.
19. Some organisms have mechanisms in place that will override transcription termination. One such mechanism
using the Tat protein is employed by the HIV retrovirus. Explain why Tat is therefore a good target for HIV vaccination.
20. Upon identification of the DNA regulatory sequence responsible for translating a given gene, you note that it is enriched with CG sequences. Is the corresponding gene likely to be a highly expressed transcript?
21. Name four major classes of DNA-binding proteins that are responsible for controlling transcription, and describe their structural features.

## References

## Control of Gene Expression in Bacteria

Bush, M., and R. Dixon. 2012. The role of bacterial enhancer binding proteins as specialized activators of $\sigma 54$-dependent transcription. Microbiol. Mol. Biol. R. 76:497-529.

Casino, P., V. Rubio, and A. Marina. 2010. The mechanism of signal transduction by two-component systems. Curr. Opin. Struc. Biol. 20:763-771.

Fürtig, B., et al. 2015. Multiple conformational states of riboswitches fine-tune gene regulation. Curr. Opin. Struc. Biol. 30:112-124.

Muller-Hill, B. 1998. Some repressors of bacterial transcription. Curr. Opin. Microbiol. 1:145-151.

## Overview of Eukaryotic Gene Control

Djebali, S., et al. 2012. Landscape of transcription in human cells. Nature 489:101-108.

Kellis, M., et al. 2014. Defining functional DNA elements in the human genome. P. Natl. Acad. Sci. USA 111:6131-6138.

RNA Polymerase II Promoters and General Transcription Factors
Sainsbury, S., C. Bernecky, and P. Cramer. 2015. Structural basis of transcription initiation by RNA polymerase II. Nat. Rev. Mol. Cell Biol. 16:129-143.

Regulatory Sequences in Protein-Coding Genes and the Proteins Through Which They Function
de Wit, E., and W. de Laat. 2012. A decade of 3C technologies: insights into nuclear organization. Genes Dev. 26:11-24.

ENCODE Project Consortium. 2012. An integrated encyclopedia of DNA elements in the human genome. Nature 489:57-74.

Vaquerizas, J. M., et al. 2009. A census of human transcription factors: function, expression and evolution. Nat. Rev. Genet. 10:252-263.

Molecular Mechanisms of Transcription Repression and Activation
Berger, S. L. 2007. The complex language of chromatin regulation during transcription. Nature 447:407-412.

Malladi, V. S., et al. 2015. Ontology application and use at the ENCODE DCC. Database (Oxford). doi: 10.1093/database/bav010.

Plaschka, C., et al. 2015. Architecture of the RNA polymerase II-Mediator core initiation complex. Nature 518:376-380.

Rothbart, S. B., and B. D. Strahl. 2014. Interpreting the language of histone and DNA modifications. Biochim. Biophys. Acta 1839:627-643.

Zaret, K. S., and J. S. Carroll. 2011. Pioneer transcription factors: establishing competence for gene expression. Genes Dev. 25:2227-2241.

## Regulation of Transcription-Factor Activity

Kirschke, E., et al. 2014. Glucocorticoid receptor function regulated by coordinated action of the Hsp90 and Hsp70 chaperone cycles. Cell 157:1685-1697.

## Epigenetic Regulation of Transcription

Derrien, T., et al. 2012. The GENCODE v7 catalog of human long noncoding RNAs: analysis of their gene structure, evolution, and expression. Genome Res. 22:1775-1789.

Gendrel, A. V., and E. Heard. 2014. Noncoding RNAs and epigenetic mechanisms during X-chromosome inactivation. Annu. Rev. Cell Dev. Biol. 30:561-580.

Klose, R. J., and A. P. Bird. 2006. Genomic DNA methylation: the mark and its mediators. Trends Biochem. Sci. 31:89-97.

McHugh, C. A., et al. 2015. The Xist lncRNA interacts directly with SHARP to silence transcription through HDAC3. Nature 521:232-236.

## Other Eukaryotic Transcription Systems

Moir, R. D., and I. M. Willis. 2015. Regulating maf1 expression and its expanding biological functions. PLoS Genet. 11:e1004896.

![li2832239539b](li2832239539b.jpg)

Portion of a "lampbrush chromosome" from an oocyte of the newt Nophthalmus viridescens. The hnRNP protein associated with nascent RNA transcripts fluoresces red after staining with a monoclonal antibody. [Courtesy of M. Roth and J. Gall.]

In the previous chapter, we saw that most genes are regulated at the first step in gene expression, transcription, by regulation of the assembly of the transcription preinitiation complex on a promoter DNA sequence and of transcription elongation in the promoter-proximal region. Once transcription has been initiated, synthesis of the encoded RNA requires that RNA polymerase transcribe the entire gene and not terminate transcription prematurely. Moreover, the initial primary transcripts produced from eukaryotic genes must undergo various processing reactions to yield the corresponding functional RNAs. For mRNAs, the $5^{\prime}$ cap structure necessary for translation must be added (see Figure 5-14), introns must be spliced out of pre-mRNAs, and the $3^{\prime}$ end must be polyadenylated (see Figure 5-15). Once formed in the nucleus, mature, functional RNAs are exported to the cytoplasm as components of ribonucleoproteins. Both the processing of RNAs and their export from the nucleus offer opportunities for further regulation of gene expression after the initiation of transcription.

## OUTLINE

### 10.1 Processing of Eukaryotic Pre-mRNA

### 10.2 Regulation of Pre-mRNA Processing

10.3 Transport of mRNA Across the Nuclear Envelope

## Posttranscriptional Gene Control

Recently, the vast amount of sequence data on human mRNAs expressed in different tissues and at various times during embryogenesis and cellular differentiation has revealed that some 95 percent of human genes give rise to alternatively spliced mRNAs. These alternatively spliced mRNAs encode related proteins with differences in their sequences that are limited to specific functional domains. In many cases, alternative RNA splicing is regulated to meet the need for a specific protein isoform in a specific cell type. Given the complexity of pre-mRNA splicing, it is not surprising that mistakes are occasionally made, giving rise to mRNA precursors with improperly spliced exons. However, eukaryotic cells have evolved RNA surveillance mechanisms that prevent the export of incorrectly processed RNAs to the cytoplasm or lead to their degradation if they are exported.

Additional control of gene expression can occur in the cytoplasm. In the case of protein-coding genes, for instance, the amount of protein produced depends on the stability of the corresponding mRNAs in the cytoplasm and the

### 10.4 Cytoplasmic Mechanisms of Post-transcriptional Control

### 10.5 Processing of rRNA and tRNA

![li2842239539b](li2842239539b.jpg)

FIGURE 10-1 Overview of RNA processing and posttranscriptional gene control. Nearly all cytoplasmic RNAs are processed from primary transcripts in the nucleus before they are exported to the cytoplasm. For protein-coding genes transcribed by RNA polymerase II, gene control can be exerted through step $\mathbf{B}$ the choice of alternative exons during pre-mRNA splicing and step $\mathbf{B}$ the choice of alternative poly(A) sites. Improperly processed mRNAs are blocked from export to the cytoplasm and degraded step $\mathbf{B}$ by a large complex called the exosome that contains multiple ribonucleases. Once the mRNA has been exported to the cytoplasm, step translation initiation factors bind to the $5^{\prime}$ cap cooperatively with poly(A)-binding protein I bound to the poly(A) tail and initiate translation (see Figure 4-28). Step $\mathbf{B}$ mRNA is degraded in the cytoplasm by deadenylation and decapping followed by degradation by cytoplasmic exosomes.

These processes occur rapidly in dense regions of the cytoplasm called P bodies that function in translational repression. The degradation rate of each mRNA is controlled, thereby regulating the mRNA concentration and, consequently, the amount of protein translated. Some mRNAs are synthesized without long poly(A) tails. Their translation is regulated by step control of the synthesis of a long poly(A) tail by a cytoplasmic poly(A) polymerase. Step $\mathbf{B}$ Translation is also regulated by other mechanisms, including miRNAs. When expressed, these $\sim 22$-nucleotide RNAs inhibit translation of mRNAs to which they hybridize, usually in the $3^{\prime}$ untranslated region. tRNAs and rRNAs are also synthesized as precursor RNAs that must be step processed before they are functional. Regions of precursors cleaved from the mature RNAs are degraded by nuclear exosomes step $\mathbf{B}$. See Houseley et al., 2006, Nat. Rev. Mol. Cell Biol. 7:529.

rate of their translation. For example, during an immune response, lymphocytes communicate by secreting polypeptide hormones called cytokines that signal neighboring lymphocytes through cytokine receptors that span their plasma membranes (see Chapter 23). It is important for lymphocytes to synthesize and secrete cytokines in short bursts. This is possible because cytokine mRNAs are extremely unstable; consequently, the concentration of these mRNAs in the cytoplasm falls rapidly once their synthesis is stopped. In contrast, mRNAs encoding proteins required in large amounts that function over long periods, such as ribosomal proteins, are extremely stable, so that multiple polypeptides are transcribed from each mRNA.

Just as pre-mRNA processing, nuclear export, and translation are regulated, so is the cellular localization of many, if not most, mRNAs, so that newly synthesized protein is concentrated where it is needed. Particularly striking examples of this type of regulation occur in the nervous systems of multicellular animals. Some neurons in the human brain generate more than a thousand separate synapses with other neurons. During the process of learning, synapses that fire more frequently than others increase in size many times, while other synapses made by the same neuron do not. This can occur because mRNAs encoding proteins critical for synapse enlargement are stored at all synapses, but translation of these localized, stored mRNAs is regulated at each synapse independently by the frequency at which the synapse signals. In this way, synthesis of synapse-associated proteins can be regulated independently at each of the many synapses made by the same neuron (see Chapter 22).

Another type of gene regulation involves micro-RNAs (miRNAs), which regulate the translation and stability of specific target mRNAs in multicellular animals and plants. Analyses of these short miRNAs in various human tissues indicate that about 1900 miRNAs are expressed in the multiple types of human cells. Although some have recently been discovered to function through inhibition of target-gene expression in the appropriate tissue and at the appropriate time in development, the functions of the vast majority of human miRNAs are unknown and are the subject of a growing new area of research. If most miRNAs do indeed have significant functions, miRNA genes constitute an important subset of the 25,000 or so human genes. A closely related process, called RNA interference (RNAi), leads to the degradation of viral RNAs in infected cells and the degradation of transposon-encoded RNAs in many eukaryotes. This discovery is of tremendous significance to biological researchers because it is possible to design short interfering RNAs (siRNAs) to inhibit the translation of specific mRNAs experimentally by a process called RNA knockdown. This method makes it possible to inhibit the function of any desired gene, even in organisms that are not amenable to classical genetic methods for isolating mutants.

We refer to all the mechanisms that regulate gene expression following transcription as post-transcriptional gene control (Figure 10-1). Because the stability and translation rate of an mRNA contribute to the amount of protein expressed from a gene, these post-transcriptional processes are important components of gene control. Indeed, the protein
output of a gene is regulated at every step in the life of an mRNA, from the initiation of its synthesis to its degradation. Thus genetic regulatory processes act on RNA as well as on DNA. In this chapter, we consider the events in the processing of mRNA that follow transcription initiation and promoter-proximal elongation as well as the various mechanisms that are known to regulate these events. In the last section, we briefly discuss the processing of primary transcripts produced from genes encoding $r$ RNAs and $t$ RNAs.

### 10.1 Processing of Eukaryotic Pre-mRNA

In this section, we take a closer look at how eukaryotic cells convert the initial primary transcript synthesized by RNA polymerase II into a functional mRNA. Three major events occur during the process: $5^{\prime}$ capping, $3^{\prime}$ cleavage and polyadenylation, and RNA splicing (Figure 10-2). Adding these specific modifications to the $5^{\prime}$ and $3^{\prime}$ ends of the pre-mRNA protects it from enzymes that quickly digest uncapped RNAs generated by RNA processing, such as spliced-out introns and RNA transcribed downstream from a polyadenylation site. Thus the $5^{\prime}$ cap and $3^{\prime}$ poly $(\mathrm{A})$ tail distinguish pre-mRNA molecules from the many other kinds of RNAs in the nucleus (Table 10-1). Pre-mRNA molecules are bound by nuclear proteins that function in mRNA export to the cytoplasm. Prior to nuclear export, introns must be removed to generate the correct coding region of the mRNA. In higher eukaryotes, including humans, alternative splicing is intricately regulated in order to substitute different functional domains into proteins, producing a considerable expansion of the proteome of these organisms.

The pre-mRNA processing events of capping, polyadenylation, and splicing occur in the nucleus as the nascent mRNA precursor is being transcribed. Thus pre-mRNA processing is co-transcriptional. As the RNA emerges from the surface of RNA polymerase II, its $5^{\prime}$ end is immediately modified by the addition of the $5^{\prime}$ cap structure found on all mRNAs (see Figure 5-14). As the nascent pre-mRNA continues to emerge from the surface of the polymerase, it is immediately bound by members of a complex group of RNA-binding proteins that assist in RNA splicing and export of the fully processed mRNA through nuclear pore complexes into the cytoplasm. Some of these proteins remain associated with the mRNA in the cytoplasm, but most either remain in the nucleus or shuttle back into the nucleus shortly after the mRNA is exported to the cytoplasm. Cytoplasmic RNA-binding proteins are exchanged for the nuclear ones. Consequently, mRNAs never occur as free RNA molecules in the cell, but are always associated with proteins as ribonucleoprotein (RNP) complexes, first as nascent pre-mRNPs that are capped and spliced as they are transcribed. Then, following cleavage and polyadenylation, they are referred to as nuclear $m R N P$ s. Following the exchange of proteins that accompanies export to the cytoplasm, they are called cytoplasmic $m R N P$ s. Although we frequently refer to premRNAs and mRNAs, it is important to remember that they are always associated with proteins as RNP complexes.

![li2852239539b](li2852239539b.jpg)

FIGURE 10-2 Overview of mRNA processing in eukaryotes. Shortly after RNA polymerase II initiates transcription at the first nucleotide of the first exon of a gene, the $5^{\prime}$ end of the nascent RNA is capped with 7-methylguanylate (step $\mathbf{\square}$ ). Transcription by RNA polymerase II terminates at any one of multiple termination sites downstream from the poly(A) site, which is located at the $3^{\prime}$ end of the final exon. After the primary transcript is cleaved at the poly(A) site (step $\mathbf{\square}$ ), a string of adenosine (A) residues is added (step $\mathbf{\square}$ ). The poly(A)
tail contains $\sim 250$ A residues in mammals, $\sim 150$ in insects, and $\sim 100$ in yeasts. For short primary transcripts with few introns, splicing (step $\mathbf{\square}$ ) usually follows cleavage and polyadenylation, as shown. For large genes with multiple introns, introns are often spliced out of the nascent RNA during its transcription, before transcription of the gene is complete. Note that the $5^{\prime}$ cap and the sequence adjacent to the poly(A) tail are retained in mature mRNAs. The diagram shown represents processing of human $\beta$-globin RNA.

## The 5' Cap Is Added to Nascent RNAs Shortly After Transcription Initiation

As a nascent eukaryotic RNA transcript emerges from the RNA exit channel of RNA polymerase II (see Figure 9-12) and reaches a length of about 25 nucleotides, a protective cap composed of 7-methylguanosine and methylated riboses
is added to the $5^{\prime}$ end of the mRNA (see Figure 5-14). This $5^{\prime}$ cap marks RNA molecules as mRNA precursors and protects them from RNA-digesting enzymes ( $5^{\prime}$-exoribonucleases) in the nucleus and cytoplasm. This initial step in RNA processing is catalyzed by a dimeric capping enzyme, which associates with the phosphorylated carboxy-terminal domain (CTD) of RNA polymerase II. Recall that the TFIIH general

# TABLE 10-1 RNAs Discussed in Chapter 10 

mRNA Fully processed messenger RNA with $5^{\prime}$ cap, introns removed by RNA splicing, and a poly(A) tail.
pre-mRNA An mRNA precursor containing introns and not cleaved at the poly(A) site.
hnRNA Heterogeneous nuclear RNAs. These RNAs include pre-mRNAs and RNA-processing intermediates containing one or more introns.
snRNA Five small nuclear RNAs that function in the removal of introns from pre-mRNAs by RNA splicing, plus two small nuclear RNAs that substitute for the first two at rare introns.
pre-tRNA A tRNA precursor containing additional transcribed bases at the $5^{\prime}$ and $3^{\prime}$ ends compared with the mature tRNA. Some pre-tRNAs also contain an intron in the anticodon loop.
pre-rRNA The precursor to mature 18S, 5.8S, and 28S ribosomal RNAs. The mature rRNAs are processed from this long precursor RNA molecule by cleavage, removal of bases from the ends of the cleaved products, and modification of specific bases.
snoRNA Small nucleolar RNAs. These RNAs base-pair with complementary regions of the pre-rRNA molecule, directing cleavage of the RNA chain and modification of bases during maturation of the rRNAs.
siRNA Short interfering RNAs, $\sim 22$ bases long, that are each perfectly complementary to a sequence in an mRNA. Together with associated proteins, siRNAs cause cleavage of the "target" RNA, leading to its rapid degradation.
miRNA Micro-RNAs, $\sim 22$ bases long, that base-pair extensively, but not completely, with mRNAs, especially over bases 2 to 7 at the $5^{\prime}$ end of the miRNA (the "seed" sequence). This pairing inhibits translation of the "target" mRNA and targets it for degradation.

transcription factor phosphorylates the CTD multiple times on serine 5 of the CTD heptapeptide repeat during transcription initiation (see Figure 19-20). Binding of the capping enzyme to the serine 5-phosphorylated CTD stimulates the activity of the enzyme so that it is focused on RNAs containing a $5^{\prime}$ triphosphate that emerge from RNA polymerase II, and not on RNAs transcribed by RNA polymerases I or III, which do not have a CTD. This is important because pre-mRNA synthesis accounts for only about 80 percent of the total RNA synthesized in replicating cells. About 20 percent is preribosomal RNA, which is transcribed by RNA polymerase I, and 5S rRNA, rRNAs, and other small stable RNAs, which are transcribed by RNA polymerase III. These two mechanisms, (1) binding of the capping enzyme to RNA polymerase II specifically through its unique CTD phosphorylated on serine 5 of the heptapeptide repeat during transcription initiation by TFIIH, and (2) activation of the capping enzyme by the serine 5 -phosphorylated CTD, result in specific capping of RNAs transcribed by RNA polymerase II.

One subunit of the capping enzyme removes the $\gamma$ phosphate from the $5^{\prime}$ end of the nascent RNA (Figure 10-3). Another domain of this subunit transfers the GMP moiety from GTP to the $5^{\prime}$ diphosphate of the nascent transcript,
![li2862239539b](li2862239539b.jpg)

FIGURE 10-3 Synthesis of the 5' cap on eukaryotic mRNAs. The $5^{\prime}$ end of a nascent RNA contains a $5^{\prime}$ triphosphate from the initiating rNTP. The $\gamma$ phosphate is removed in the first step of capping, while the remaining $\alpha$ and $\beta$ phosphates (orange) remain associated with the cap. The third phosphate of the $5^{\prime}, 5^{\prime}$ triphosphate bond is derived from the $\alpha$ phosphate of the GTP that donates the guanine. The methyl donor for methylation of the cap guanine and the first one or two riboses of the mRNA is $S$-adenosylmethionine (S-Ado-Met). See S. Venkatesan and B. Moss, 1982, Proc. Natl. Acad. Sci. USA 79:340.
creating the unusual guanosine $5^{\prime}-5^{\prime}$ triphosphate structure. In the final steps, separate enzymes transfer methyl groups from $S$-adenosylmethionine to the $\mathrm{N}^{\prime}$ position of the guanine and to the $2^{\prime}$ oxygens of riboses of the first one or two nucleotides at the $5^{\prime}$ end of the nascent RNA.

Considerable evidence indicates that capping of the nascent transcript is coupled to elongation by RNA polymerase II so that all of its transcripts are capped during the earliest phase of elongation. As discussed in Chapter 9, in metazoans, during the initial phase of transcription, the polymerase elongates the nascent transcript very slowly due to the association of NELF (negative elongation factor) with RNA polymerase II in the promoter-proximal region (see Figure 9-21). Once the $5^{\prime}$ end of the nascent RNA is capped, phosphorylation of the RNA polymerase CTD at serine 2 in the heptapeptide repeat and of NELF and DSIF (DRB-sensitivity-inducing factor) by the cyclin T-CDK9 protein kinase (also known as P-TEFb) causes the release of NELF. (DRB is an analog of ATP that inhibits CDK9, preventing transcription elongation from the promoter-proximal region.) This allows RNA polymerase II to enter into a faster mode of elongation that rapidly transcribes away from the promoter. The net effect of this mechanism is that the polymerase waits for the nascent RNA to be capped before elongating at a rapid rate.

## A Diverse Set of Proteins with Conserved RNABinding Domains Associate with Pre-mRNAs

As noted earlier, neither nascent RNA transcripts of proteincoding genes nor the intermediates of mRNA processing, collectively referred to as pre-mRNA, exist as free RNA molecules in the nuclei of eukaryotic cells. From the time nascent transcripts first emerge from RNA polymerase II until mature mRNAs are transported into the cytoplasm, the RNA molecules are associated with an abundant set of nuclear proteins. These proteins are the major protein components of heterogeneous ribonucleoprotein particles (hnRNPs), which contain heterogeneous nuclear RNA (hnRNA), a collective term referring to pre-mRNA and other nuclear RNAs of various sizes. These hnRNP proteins contribute to further steps in RNA processing, including splicing, polyadenylation, and export through nuclear pore complexes to the cytoplasm.

Researchers identified hnRNP proteins by first exposing cultured cells to high-dose UV irradiation, which causes covalent cross-links to form between RNA bases and closely associated proteins. Chromatography of nuclear extracts from treated cells on an oligo-dT cellulose column, which binds RNAs with a poly $(\mathrm{A})$ tail, was used to recover the proteins that had become cross-linked to nuclear polyadenylated RNA. Subsequent treatment of cell extracts from nonirradiated cells with monoclonal antibodies specific for the major proteins identified by this cross-linking technique revealed a complex set of abundant hnRNP proteins ranging in size from 30 to 120 kDa .

Like transcription factors, most hnRNP proteins have a modular structure. They contain one or more RNA-binding

domains and at least one other domain that interacts with other proteins. Several different RNA-binding motifs have been identified by creating hnRNP proteins with missing amino acid sequences and testing their ability to bind RNA.

Functions of hnRNP Proteins The association of pre-mRNAs with hnRNP proteins prevents the pre-mRNAs from forming short secondary structures by base pairing of complementary regions, thereby making the pre-mRNAs accessible for interaction with other RNA molecules or proteins. Pre-mRNAs associated with hnRNP proteins present a more uniform substrate for subsequent processing steps than would free, unbound pre-mRNAs, each of which would form a unique secondary structure due to its specific sequence.

Binding studies with purified hnRNP proteins indicate that different hnRNP proteins associate with different regions of a newly made pre-mRNA molecule. For example, the hnRNP proteins A1, C, and D bind preferentially to the pyrimidine-rich sequences at the $3^{\prime}$ ends of introns (see Figure 10-7 below). Some hnRNP proteins interact with the RNA sequences that specify RNA splicing or cleavage/ polyadenylation and contribute to the structure recognized by RNA-processing factors. Finally, cell-fusion experiments have shown that some hnRNP proteins remain localized in the nucleus, whereas others cycle in and out of the cytoplasm, suggesting that they function in the export of mRNA from the nucleus to the cytoplasm (Figure 10-4).

Conserved RNA-Binding Motifs The RNA recognition motif ( $R R M$ ), also called the RNP motif and the RNA-binding domain (RBD), is the most common RNA-binding domain
in hnRNP proteins. This 80 -residue domain, which occurs in many other RNA-binding proteins as well, contains two highly conserved sequences (RNP1 and RNP2) that are found across organisms ranging from yeast to humans-indicating that, like many DNA-binding domains, it evolved early in eukaryotic evolution.

Structural analyses have shown that the RRM domain consists of a four-stranded $\beta$ sheet flanked on one side by two $\alpha$ helices. To interact with the negatively charged RNA phosphates, the $\beta$ sheet forms a positively charged surface. The conserved RNP1 and RNP2 sequences lie side by side on the two central $\beta$ strands, and their side chains make multiple contacts with a single-stranded region of RNA that lies across the surface of the $\beta$ sheet (Figure 10-5).

The 45 -residue $K H$ motif is found in the hnRNP K protein and several other RNA-binding proteins. The three-dimensional structure of representative KH domains is similar to that of the RRM domain but smaller, consisting of a three-stranded $\beta$ sheet supported from one side by two $\alpha$ helices. Nonetheless, the KH domain interacts with RNA much differently than does the RRM domain. RNA binds to the KH domain by interacting with a hydrophobic surface formed by the $\alpha$ helices and one $\beta$ strand. The $R G G$ box, another RNA-binding motif found in hnRNP proteins, contains five Arg-Gly-Gly (RGG) repeats with several interspersed aromatic amino acids. A recent structural analysis indicates that in one example of RNA binding, an RGG-containing peptide binds in the major groove of a G-rich RNA duplex region (see Figure 5-4b). KH domains and RGG repeats are often interspersed in two or more sets in a single RNA-binding protein.
![li2872239539b](li2872239539b.jpg)

FIGURE 10-4 Human hnRNP A1 protein can cycle in and out of the nucleus, but human hnRNP C protein cannot. Cultured HeLa cells and Xenopus cells were fused by treatment with polyethylene glycol, producing heterokaryons containing nuclei from each cell type. These hybrid cells were treated with cycloheximide immediately after fusion to prevent protein synthesis. After 2 hours, the cells were fixed and stained with fluorescent-labeled antibodies specific for human hnRNP C and A1 proteins. These antibodies do not bind to the homologous Xenopus proteins. (a) A fixed preparation viewed by phase-contrast microscopy includes unfused HeLa cells (arrowhead) and Xenopus cells (dotted arrow), as well as fused heterokaryons (solid arrow). In the heterokaryon in this micrograph, the round HeLa-cell nucleus is to the right of the oval-shaped Xenopus nucleus. (b, c) When the same preparation
was viewed by fluorescence microscopy, the stained hnRNP C protein appeared green and the stained hnRNP A1 protein appeared red. Note that the unfused Xenopus cell on the left is unstained, confirming that the antibodies are specific for the human proteins. In the heterokaryon, hnRNP C protein appears only in the HeLa-cell nucleus (b), whereas the A1 protein appears in both the HeLa-cell nucleus and the Xenopus nucleus (c). Since protein synthesis was blocked after cell fusion, some of the human hnRNP A1 protein must have left the HeLa-cell nucleus, moved through the cytoplasm, and entered the Xenopus nucleus in the heterokaryon. (Reprinted by permission of Nature Publishing Group, from: Piñol-Roma S., and Dreyfuss, G., "Shuttling of pre-mRNA binding proteins between nucleus and cytoplasm," Nature, 1992, 355(6362):730-2; permission conveyed through the Copyright Clearance Center, Inc.)

![li2882239539b](li2882239539b.jpg)

FIGURE 10-5 Structure of the RRM domain and its interaction with RNA. (a) Ribbon diagram of the RRM domain found in hnRNP proteins, showing the two $\alpha$ helices (green) and four $\beta$ strands (red) that characterize this motif. The conserved RNP1 and RNP2 regions are located in the two central $\beta$ strands. (b, c) Ribbon diagram and surface representation of the two RRM domains in Drosophila Sex-lethal (Sxl) protein (b) and the polypyrimidine tract-binding protein (PTB) (c). In both (b) and (c), positively charged regions are shown in shades of blue; negatively charged regions, in shades of red; RNA is yellow. The two RRMs in Sxl are oriented like the two parts of an open pair of castanets, with the $\beta$ sheets of the RRMs facing toward each other. The pre-mRNA is bound to the surfaces of the positively charged $\beta$ sheets, making most of its contacts with the RNP1 and RNP2 regions of each RRM. PTB has a strikingly different orientation of RRM domains,

## Splicing Occurs at Short, Conserved Sequences in Pre-mRNAs via Two Transesterification Reactions

During the formation of a mature, functional mRNA, the introns are removed and the exons are spliced together. For short transcription units, RNA splicing often follows cleavage and polyadenylation of the $3^{\prime}$ end of the primary transcript, as depicted in Figure 10-2 for the processing of human $\beta$-globin mRNA. For long transcription units containing multiple exons, however, splicing of exons in the nascent RNA begins before transcription of the gene is complete.

Early pioneering research on the nuclear processing of mRNAs revealed that mRNAs are initially transcribed as molecules that are much longer than the mature mRNAs in the cytoplasm. It was also shown that RNA sequences near the $5^{\prime}$ cap added shortly after transcription initiation are retained in the mature mRNA, and that RNA sequences near the polyadenylated ends of mRNA-processing intermediates are retained in the mature mRNAs in the cytoplasm. The solution to this apparent conundrum came from the discovery of introns by electron microscopy of RNA-DNA hybrids of adenovirus DNA and the mRNA encoding hexon, a major virion capsid protein (Figure 10-6). Other studies revealed nuclear viral RNAs that were colinear with the viral DNA (primary transcripts), and others with one or two of the introns removed (processing intermediates). Together,
illustrating that RRMs are oriented in different relative positions in different hnRNPs. The $p(Y)$-tract is a polypyrimidine tract. In PTB, the two RRMs associate through their $\alpha$ helices so that the positively charged $\beta$ sheets face away from each other, upward for RRM3 and downward for RRM4. The structure of CUCUCU single-stranded RNA bound to each of the two RRMs was determined, explaining how PTB can bind to two tracts of six pyrimidines in a single RNA if they are separated by a loop of 15 or more nucleotides. This ability of PTB to form a small loop in a pre-mRNA probably contributes to its ability to function as a splicing repressor at exons where the upstream $3^{\prime}$ splice site or the downstream $5^{\prime}$ splice site is flanked by two polypyrimidine tracts. See K. Nagai et al., 1995, Trends Biochem. Sci. 20:235. [Part (b) data from N. Harada et al., 1999, Nature 398:579, PDB ID 1b7f, Part (c) data from F. C. Oberstrass et al., 2006, Science 309:2054, PDB ID 2adb, 2adc.]
these results led to the realization that introns are removed from primary transcripts as exons are spliced together.

The locations of splice sites-that is, exon-intron junctions-in a pre-mRNA can be determined by comparing the sequence of genomic DNA with that of cDNA prepared from the corresponding mRNA (see Figure 6-17). Sequences that are present in the genomic DNA but absent from the cDNA represent introns and indicate the positions of splice sites. Such analyses of a large number of different mRNAs revealed moderately conserved, short consensus sequences at the splice sites flanking introns in eukaryotic pre-mRNAs, including a polypyrimidine tract just upstream of the $3^{\prime}$ splice site (Figure 10-7). Studies of mutant genes with deletions introduced into introns have shown that much of the central portion of an intron can be removed without affecting splicing; generally only 30-40 nucleotides at each end of an intron are necessary for splicing to occur at normal rates.

Analysis of the intermediates formed during the splicing of pre-mRNAs in vitro led to the discovery that splicing of exons proceeds via two sequential transesterification reactions (Figure 10-8). Introns are removed as a lariat structure in which the $5^{\prime}$ guanine of the intron is joined in an unusual $2^{\prime}, 5^{\prime}$-phosphodiester bond to an adenosine near the $3^{\prime}$ end of the intron. This A residue is called the branch-point $A$ because it forms an RNA branch in the lariat structure. In each transesterification reaction, one phosphoester bond is

![li2892239539b](li2892239539b.jpg)
(b)
![li2902239539b](li2902239539b.jpg)

EXPERIMENTAL FIGURE 10-6 Electron microscopy of mRNA-template DNA hybrids shows that introns are spliced out during pre-mRNA processing. (a) Diagram of the EcoRI A fragment of adenovirus DNA, which extends from the left end of the genome to just before the end of the final exon of the hexon gene. The hexon gene consists of three short exons and one long ( $\sim 3.5 \mathrm{~kb}$ ) exon separated by three introns of $\sim 1,2.5$, and 9 kb . (b) Electron micrograph (left) and schematic drawing (right) of a hybrid between an EcoRI A DNA fragment and a hexon mRNA. The loops marked A, B, and C correspond to the introns indicated in (a). Since these intron sequences in the viral genomic DNA are not present in the mature hexon mRNA, they loop out between the exon sequences that hybridize to their complementary sequences in the mRNA. [Micrograph courtesy of Phillip A. Sharp.]
![li2912239539b](li2912239539b.jpg)

FIGURE 10-7 Consensus sequences around splice sites in vertebrate pre-mRNAs. The only nearly invariant bases are the $5^{\prime}$ GU and the $3^{\prime}$ AG of the intron (blue), although the flanking bases indicated are found at frequencies higher than expected based on a random distribution. A polypyrimidine tract (hatched area) near the $3^{\prime}$ end of the intron is found in most introns. The branch-point
adenosine, also invariant, is usually 20-50 bases from the $3^{\prime}$ splice site. The central region of the intron, which may range from 40 bases to 50 kilobases in length, is generally unnecessary for splicing to occur. See R. A. Padgett et al., 1986, Annu. Rev. Biochem. 55:1119, and E. B. Keller and W. A. Noon, 1984, Proc. Natl. Acad. Sci. USA 81:7417.
exchanged for another. Since the number of phosphoester bonds in the molecule is not changed in either reaction, no energy is consumed. The net result of these two reactions is that two exons are ligated and the intervening intron is released as a branched lariat structure.

## During Splicing, snRNAs Base-Pair with Pre-mRNA

Splicing requires the presence of small nuclear RNAs (snRNAs), which base-pair with the pre-mRNA, and some 170 associated proteins. Five U-rich snRNAs, designated U1, U2, U4, U5, and U6, participate in pre-mRNA splicing. Ranging in length from 107 to 210 nucleotides, these snRNAs are associated with 6-10 proteins each in the many small nuclear ribonucleoprotein particles (snRNPs) in the nuclei of eukaryotic cells.

Definitive evidence for the role of U1 snRNA in splicing came from experiments indicating that base pairing between the $5^{\prime}$ splice site of a pre-mRNA and the $5^{\prime}$ region of U1 snRNA is required for RNA splicing (Figure 10-9a). In vitro experiments
showed that a synthetic oligonucleotide that hybridizes with the $5^{\prime}$-end region of U 1 snRNA blocks RNA splicing. In vivo experiments showed that base pairing-disrupting mutations in the pre-mRNA $5^{\prime}$ splice site also block RNA splicing; in this case, however, splicing can be restored by expression of a U1 snRNA with a compensating mutation that restores base pairing to the mutant pre-mRNA $5^{\prime}$ splice site (Figure 10-9b). Involvement of U2 snRNA in splicing was initially suspected when it was found to have an internal sequence that is largely complementary to the consensus sequence flanking the branch point in pre-mRNAs (see Figure 10-7). Compensating mutation experiments, similar to those conducted with U1 snRNA and $5^{\prime}$ splice sites, demonstrated that base pairing between U2 snRNA and the branch-point sequence in pre-mRNA is also critical to splicing.

Figure 10-9a illustrates the general structures of the U1 and U2 snRNAs and how they base-pair with pre-mRNA during splicing. Significantly, the branch-point A itself, which is not base-paired to U2 snRNA, "bulges out" (Figure 10-10a), which allows its $2^{\prime}$ hydroxyl to participate in the first transesterification reaction of RNA splicing (see Figure 10-8).

![li2922239539b](li2922239539b.jpg)

Excised lariat intron

FIGURE 10-8 Two transesterification reactions result in the splicing of exons in pre-mRNA. In the first reaction, the ester bond between the $5^{\prime}$ phosphorus of the intron and the $3^{\prime}$ oxygen (dark red) of exon 1 is exchanged for an ester bond with the $2^{\prime}$ oxygen (blue) of the branch-point A residue. In the second reaction, the ester bond between the $5^{\prime}$ phosphorus of exon 2 and the $3^{\prime}$ oxygen (orange) of the intron is exchanged for an ester bond with the $3^{\prime}$ oxygen of exon 1 , releasing the intron as a lariat structure and joining the two exons. Arrows show where activated hydroxyl oxygens react with phosphorus atoms.

FIGURE 10-9 (below) Base pairing between pre-mRNA, U1 snRNA, and U2 snRNA early in the splicing process. (a) In this diagram, secondary structures in the snRNAs that are not altered during splicing are depicted schematically. The yeast branch-point sequence is shown here. Note that U2 snRNA base-pairs with a sequence that includes the branch-point A, although this residue is not base-paired. For unknown reasons, antisera from patients with the autoimmune disease systemic lupus erythematosus (SLE) contain antibodies to snRNP proteins, which have been useful in characterizing components of the splicing reaction; the purple rectangles represent sequences that bind snRNP proteins recognized by these anti-Sm antibodies. (b) Only the $5^{\prime}$ ends of U1 snRNAs and $5^{\prime}$ splice sites in pre-mRNAs are shown. (Left) A mutation (A) in a pre-mRNA splice site that interferes with base pairing to the $5^{\prime}$ end of U1 snRNA blocks splicing. (Right) Expression of a U1 snRNA with a compensating mutation (U) that restores base pairing also restores splicing of the mutant pre-mRNA. See M. J. Moore et al., 1993, in R. Gesteland and J. Atkins, eds., The RNA World, Cold Spring Harbor Press, pp. 303-357; see also Y. Zhuang and A. M. Weiner, 1986, Cell 46:827.
(a)
![li2932239539b](li2932239539b.jpg)
(b)

Mutant U1 snRNA $3^{\prime} \leftrightarrows \underset{\text { Mutant pre-mRNA } 5^{\prime}}{ }$ Exon 1 CAGGAAA

(a) Self-complementary sequence with bulging A
![li2942239539b](li2942239539b.jpg)
(c) Spliceosome structure
![li2952239539b](li2952239539b.jpg)

Similar studies with other snRNAs demonstrated that base pairing between the snRNAs themselves also occurs during splicing. Moreover, rearrangements in these RNA-RNA interactions are critical in the splicing pathway.

8As mentioned above, a synthetic oligonucleotide that base pairs with the $5^{\prime}$ end of U1 snRNA was found to inhibit RNA splicing in vitro, supporting the importance of U1 snRNA base pairing to a $5^{\prime}$ splice site for the first step in pre-mRNA splicing. Currently, a similar strategy is being used in clinical trials for the treatment of Duchenne muscular dystrophy (DMD). This disorder is the most common human genetic disease due to new mutations in the genome. It is caused by mutations in the DMD gene, especially chainterminating mutations due to a base-pair change in an exon that generates a stop codon. Alternatively, short deletions or insertions that change the reading frame of the message result in translation of abnormal amino acids, generally followed by a stop codon in the altered reading frame. These mutations eliminate the C-terminus of the encoded protein, dystrophin, which is essential to its function (see Figure 17-20, bottom). The DMD gene is the longest human gene ( $\sim 2$ million base pairs; half the length of the entire E. coli genome!), which makes it a large target for random mutations. Since the DMD gene is on the X chromosome, there is no second wild-type copy to complement the mutation in males. Synthetic oligonucleotides have been developed that are modified to permeate cell membranes, but have normal Watson-Crick base-pairing properties. By hybridizing with the terminus of a mutant exon, they can cause the abnormal exon to be "skipped" during pre-mRNA splicing, and can be designed so that the normal exon upstream of the mutation splices to an in-frame downstream exon. This results in expression of a protein with an internal deletion, but one that, potentially, has sufficient function to alleviate what are otherwise devastating symptoms.

## Spliceosomes, Assembled from snRNPs and a Pre-mRNA, Carry Out Splicing

The five splicing snRNPs and other proteins involved in splicing assemble on a pre-mRNA, forming a

FIGURE 10-10 Structures of a bulged A in an RNA-RNA helix and an intermediate in the splicing process. (a) Diagram of RNA duplex used for determining the structure of a bulged A. Bulged As at position 5 (red) are excluded from duplex RNA-RNA hybrid formed by complementary bases (blue and green). (b) X-ray crystallography of the structure showed that the bulged A residues extend from the side of an A-form RNA-RNA helix. The phosphate backbone of one strand is shown in green and that of the other strand in blue. The structure on the right is turned 90 degrees for a view down the axis of the helix. (c) $40 \AA$ resolution structure of a spliceosomal splicing intermediate containing U2, U4, U5, and U6 snRNPs, determined by cryoelectron microscopy and image reconstruction. The U4/U6/U5 tri-snRNP complex has a structure similar to the triangular body of this complex below the neck, suggesting that these snRNPs are at the bottom of the structure shown here and that the head is composed largely of U2 snRNP. See H. Stark and R. Luhrmann, 2006, Annu. Rev. Biophys. Biomol. Struct. 35:435. (Parts (a) and (b) data from J. A. Berglund et al., 2001, RNA 7:682, PDB ID 1IRx. Part (c) from E. Wolf et al., "Exon, intron and splice site locations in the spliceosomal B complex," EMBO J., 2009, 28(15):2283-2292; doi:10.1038/emboj.2009.171.)
large ribonucleoprotein complex called a spliceosome (Figure 10-11). The spliceosome has a mass similar to that of a ribosome. Assembly of a spliceosome begins with the base pairing of the U1 snRNA to the $5^{\prime}$ splice site as well as the cooperative binding of protein SF1 (splicing factor 1) to the branch-point A and of the heterodimeric protein U2AF (U2-associated factor) to the polypyrimidine tract and the $3^{\prime}$ AG of the intron via its large and small subunits, respectively. The U2 snRNP then base-pairs with the branch-point region (see Figure 10-9a) as SF1 is released. Extensive base pairing between the snRNAs in the U4 and U6 snRNPs forms a complex that associates with U5 snRNP. This U4/U6/U5 "tri-snRNP" then associates with the previously formed U1/ U2/pre-mRNA complex to generate a spliceosome.

After formation of the spliceosome, extensive rearrangements in the pairing of snRNAs and the pre-mRNA lead to the release of the U1 snRNP. Figure 10-10c shows the structure of this intermediate in the splicing process. A further rearrangement of spliceosomal components occurs with the loss of the U4 snRNP. Its release generates a complex that catalyzes the first transesterification reaction that forms the $2^{\prime}, 5^{\prime}$-phosphodiester bond between the $2^{\prime}$ hydroxyl on the branch-point A and the phosphate at the $5^{\prime}$ end of the intron (see Figure 10-8). Following another rearrangement of the snRNPs, the second transesterification reaction ligates the two exons in a standard $3^{\prime}, 5^{\prime}$-phosphodiester bond, releasing the intron as a lariat structure associated with the snRNPs. This final intron-snRNP complex rapidly dissociates, and the individual snRNPs released can participate in a new cycle of splicing. The excised intron is then rapidly degraded by a debranching enzyme and other nuclear RNases discussed later.

![li2962239539b](li2962239539b.jpg)

As mentioned above, a spliceosome is roughly the size of a ribosome and is composed of about 170 proteins, including about 100 "splicing factors" in addition to the proteins associated with the five snRNPs. This makes RNA splicing comparable in complexity to initiation of transcription and protein synthesis. Some of the splicing factors are associated with snRNPs, but others are not. For instance, the

FIGURE 10-11 Model of spliceosome-mediated splicing of pre-mRNA. Step After U1 base-pairs with the consensus 5' splice site, SF1 (splicing factor 1) binds the branch-point A; U2AF (U2 snRNP associated factor) associates with the polypyrimidine tract and 3' splice site; and the U2 snRNP associates with the branch-point A via base-pairing interactions shown in Figure 10-9, displacing SF1. Step A trimeric snRNP complex of U4, U5, and U6 joins the initial complex to form the spliceosome. Step B Rearrangements of base-pairing interactions between snRNAs convert the spliceosome into a catalytically active conformation and destabilize the U1 and U4 snRNPs, which are released. Step The catalytic core, thought to be formed by U6 and U2, then catalyzes the first transesterification reaction, forming the intermediate containing a $2^{\prime}, 5^{\prime}$-phosphodiester bond, as shown in Figure 10-8. Step Following further rearrangements between the snRNPs, the second transesterification reaction joins the two exons by a standard $3^{\prime}, 5^{\prime}$-phosphodiester bond and releases the intron as a lariat structure as well as the remaining snRNPs. Step $\mathbf{B}$ The excised lariat intron is converted into a linear RNA by a debranching enzyme. See T. Villa et al., 2002, Cell 109:149.

65-kDa subunit of U2AF binds to the polypyrimidine tract near the $3^{\prime}$ end of an intron and to the U2 snRNP. The $35-\mathrm{kDa}$ subunit of U2AF binds to the AG dinucleotide at the $3^{\prime}$ end of the intron and also interacts with the larger U2AF subunit bound nearby. These two U2AF subunits act together with SF1 to help specify the $3^{\prime}$ splice site by promoting interaction of the U2 snRNP with the branch point (see Figure 10-11, step $\mathbf{B}$ ). Some splicing factors also exhibit sequence homologies to known RNA helicases; these factors are probably necessary for the base-pairing rearrangements that occur among snRNAs during the spliceosomal splicing cycle. Several splicing factors associate with the CTD of RNA polymerase II when it is phosphorylated at serine 2 of the heptapeptide repeat by the cyclin T-CDK9 transcription elongation factor (see Figure 9-21). This association concentrates these splicing factors near the RNA exit site of RNA polymerase II so that they can rapidly assemble a spliceosome at a splice site as it emerges from the polymerase.

Following RNA splicing, a specific set of hnRNP proteins remains bound to the spliced RNA approximately 20 nucleotides $5^{\prime}$ to each exon-exon junction, thus forming an exon-junction complex. One of the hnRNP proteins associated with the exon-junction complex is the RNA export factor (REF), which functions in the export of fully processed mRNPs from the nucleus to the cytoplasm, as discussed in Section 10.3. Other proteins associated with the exon-junction complex function in a quality-control mechanism in the cytoplasm that leads to the degradation of improperly spliced mRNAs, known as nonsense-mediated decay (see Section 10.4).

A small fraction of pre-mRNAs ( 1 percent in humans) contain introns whose splice sites do not conform to the standard consensus sequence. This class of introns begins with AU and ends with AC rather than following the usual "GU-AG rule" (see Figure 10-7). Splicing of this special class of introns occurs via a splicing cycle analogous to that shown in Figure 10-11, except that four novel, low-abundance snRNPs, together with the standard U5 snRNP, are involved.

Nearly all functional mRNAs in vertebrate, insect, and plant cells are derived from a single molecule of the corresponding pre-mRNA by removal of internal introns and splicing of exons. However, in two types of protozoanstrypanosomes and euglenoids-mRNAs are constructed by splicing together separate RNA molecules. This process, referred to as trans-splicing, is also used in the synthesis of 10-15 percent of the mRNAs in the nematode (roundworm) Caenorhabditis elegans, an important model organism for studying embryonic development. Trans-splicing is carried out by snRNPs by a process similar to the splicing of exons in a single pre-mRNA.

## Chain Elongation by RNA Polymerase II Is Coupled to the Presence of RNA-Processing Factors

How is RNA processing efficiently coupled with the transcription of a pre-mRNA? The key lies in the long carboxy-terminal domain (CTD) of RNA polymerase II, which, as discussed in Chapter 9, is composed of multiple repeats of a seven-residue (heptapeptide) sequence. When fully extended, the CTD domain in the human RNA polymerase II is about 130 nm long (Figure 10-12). The remarkable length of the CTD apparently allows multiple proteins to associate simultaneously with a single RNA polymerase II molecule. For instance, the enzymes that add the $5^{\prime}$ cap to nascent transcripts associate
![li2972239539b](li2972239539b.jpg)

FIGURE 10-12 Schematic diagram of human RNA polymerase II with the CTD extended. The length of the human RNA polymerase II carboxy-terminal domain (CTD) and the linker region that connects it to the polymerase is shown relative to the globular domain of the polymerase. In its extended form, the CTD can associate with multiple RNA-processing factors simultaneously. See P. Cramer, D. A. Bushnell, and R. D. Kornberg, 2001, Science 292:1863.
with the serine 5-phosphorylated CTD, as mentioned above, as do splicing and polyadenylation factors. As a consequence, these processing factors are present at high local concentrations when splice sites and polyadenylation signals are transcribed by the polymerase, enhancing the rate and specificity of RNA processing. In a reciprocal fashion, the association of hnRNP proteins with the nascent RNA enhances the interaction of RNA polymerase II with elongation factors such as DSIF and cyclin T-CDK9 (see Figure 9-21), increasing the rate of transcription. As a consequence, the rate of transcription is coordinated with the rate of nascent RNA association with hnRNPs and RNA-processing factors. This mechanism may ensure that a pre-mRNA is not synthesized unless the machinery for processing it is properly positioned.

## SR Proteins Contribute to Exon Definition in Long Pre-mRNAs

The average length of an exon in the human genome is about 150 bases, whereas the average length of an intron is about 3500 bases, and the longest introns exceed 500 kb ! Because the sequences of $5^{\prime}$ and $3^{\prime}$ splice sites and branch points are so degenerate, multiple copies of those sequences are likely to occur randomly in long introns. Consequently, additional sequence information is required to define the exons that should be spliced together in higher organisms with long introns.

The information for defining the splice sites that demarcate exons is encoded within the sequences of the exons. A family of RNA-binding proteins, the $S R$ proteins, interact with sequences within exons called exonic splicing enhancers. SR proteins are a subset of the hnRNP proteins discussed earlier that contain one or more RRM RNA-binding domains. They also contain several protein-protein interaction domains rich in arginine (R) and serine (S) residues, called RS domains. When bound to exonic splicing enhancers, SR proteins mediate the cooperative binding of U1 snRNP to a true $5^{\prime}$ splice site and U2 snRNP to a branch point through a network of protein-protein interactions that span an exon (Figure 10-13). The complex of SR proteins, snRNPs, and other splicing factors (e.g., U2AF and SF1) that assemble across an exon, which has been called a cross-exon recognition complex, permits precise specification of exons in long pre-mRNAs.

8Mutations that interfere with the binding of an SR protein to an exonic splicing enhancer, even if they do not change the encoded amino acid sequence, prevent formation of the cross-exon recognition complex. As a result, the affected exon is "skipped" during splicing and is not included in the final processed mRNA. The truncated mRNA produced in this case is either degraded or translated into a mutant, abnormally functioning protein. This type of mutation occurs in some human genetic diseases. For example, spinal muscular atrophy is one of the most common genetic causes of childhood mortality. This disease results from mutations in a region of the genome containing two closely related genes, SMN1 and SMN2, that arose by gene duplication. The two genes encode identical proteins, but SMN2 is expressed

![li2982239539b](li2982239539b.jpg)

FIGURE 10-13 Exon recognition through cooperative binding of SR proteins and splicing factors to pre-mRNA. The correct $5^{\prime}$ GU and $3^{\prime}$ AG splice sites are recognized by splicing factors on the basis of their proximity to exons. The exons contain exonic splicing enhancers (ESEs) that are binding sites for SR proteins. When bound to ESEs, the SR proteins interact with one another and promote the cooperative binding of the U1 snRNP to the $5^{\prime}$ splice site of the downstream intron, SF1 and then the U2 snRNP to the branch point of the upstream intron, the 65 - and $35-\mathrm{kDa}$ subunits of U2AF to the polypyrimidine tract and AG $3^{\prime}$ splice site of the upstream intron, and other splicing factors (not shown). The resulting RNA-protein cross-exon recognition
at a much lower level because a silent mutation in one exon interferes with the binding of an SR protein. This mutation leads to exon skipping in most of the SMN2 mRNAs. The homologous $S M N$ gene in the mouse, in which there is only a single copy, is essential for cell viability. Spinal muscular atrophy in humans results from homozygous mutations that inactivate SMN1. The small amount of protein translated from the small fraction of SMN2 mRNAs that are correctly spliced is sufficient to maintain cell viability during embryogenesis and fetal development, but it is not sufficient to maintain the viability of spinal cord motor neurons in childhood, resulting in their death and the associated disease.

Approximately 15 percent of the single-base mutations that cause human genetic diseases interfere with proper exon definition. Some of these mutations occur in $5^{\prime}$ or $3^{\prime}$ splice sites, often resulting in the use of nearby alternative "cryptic" splice sites that are present in the normal gene sequence. In the absence of the normal splice site, the cross-exon recognition complex recognizes these alternative sites. Other mutations that cause abnormal splicing result in a new consensus splice-site sequence that becomes recognized in place of the normal splice site. Finally, some mutations can interfere with the binding of specific SR proteins to pre-mRNAs. These mutations inhibit splicing at normal splice sites, as in the case of the SMN2 gene, and thus lead to exon skipping.

Strategies involving membrane-permeant synthetic oligonucleotide derivatives similar to those discussed above for causing skipping of mutant exons in DMD are being developed for the treatment of these genetic diseases. Such molecules can hybridize to a mutant sequence that creates an abnormal splice site, sterically blocking access of U1 or U2 snRNAs to that site. In the case of spinal muscular atrophy, researchers are experimenting with modified oligonucleotides that base-pair to a region in the SMN2 pre-mRNA close to the missing exonic splicing enhancer.
complex spans an exon and activates the correct splice sites for RNA splicing. Note that the U1 and U2 snRNPs in this unit do not become part of the same spliceosome. The U2 snRNP on the right forms a spliceosome with the U1 snRNP bound to the $5^{\prime}$ end of the same intron. The U1 snRNP shown on the right forms a spliceosome with the U2 snRNP bound to the branch point of the downstream intron (not shown), and the U2 snRNP on the left forms a spliceosome with a U1 snRNP bound to the $5^{\prime}$ splice site of the upstream intron (not shown). Double-headed arrows indicate protein-protein interactions. See T. Maniatis, 2002, Nature 418:236; see also S. M. Berget, 1995, J. Biol. Chem. 270:2411.

A non-hybridizing region that remains single-stranded and can bind an abundant SR protein may help to assemble a cross-exon recognition complex to increase correct splicing of exons in pre-mRNAs expressed from the SMN2 gene.

## Self-Splicing Group II Introns Provide Clues to the Evolution of snRNAs

Under certain unphysiological in vitro conditions, pure preparations of some RNA transcripts slowly splice out introns in the absence of any protein. This observation led to the recognition that some introns are self-splicing. Two types of self-splicing introns have been discovered: group I introns, present in nuclear rRNA genes of protozoans, and group II introns, present in protein-coding genes and some rRNA and tRNA genes in mitochondria and chloroplasts of plants and fungi. Discovery of the catalytic activity of self-splicing introns revolutionized our thinking about the functions of RNA. As discussed in Chapter 5, RNA is now known to catalyze peptide-bond formation during protein synthesis in ribosomes. Here we discuss the probable role of group II introns, now found only in mitochondrial and chloroplast DNA, in the evolution of snRNAs; the functioning of group I introns is considered in the later section on rRNA processing.

Even though their precise sequences are not highly conserved, all group II introns fold into a conserved, complex secondary structure containing numerous stem-loops (Figure 10-14a). Self-splicing by a group II intron occurs via two transesterification reactions involving intermediates and products analogous to those found in nuclear pre-mRNA splicing. The mechanistic similarities between group II intron self-splicing and spliceosomal splicing led to the hypothesis that snRNAs function analogously to the stemloops in the secondary structure of group II introns. According to this hypothesis, snRNAs interact with $5^{\prime}$ and $3^{\prime}$ splice sites of pre-mRNAs and with one another to produce a

![li2992239539b](li2992239539b.jpg)

FIGURE 10-14 Comparison of group II self-splicing introns and the spliceosome. These schematic diagrams compare the secondary structures of (a) group II self-splicing introns and (b) U snRNAs present in the spliceosome. The first transesterification reaction is indicated by light green arrows; the second reaction, by blue arrows. The branchpoint $A$ is boldfaced. The similarity in these structures suggests that the spliceosomal snRNAs evolved from group II introns, and that the trans-acting snRNAs are functionally analogous to the corresponding domains in group II introns. The colored bars flanking the introns in (a) and (b) represent exons. See P. A. Sharp, 1991, Science 254:663.
three-dimensional RNA structure that is functionally analogous to that of group II self-splicing introns (Figure 10-14b).

An extension of this hypothesis is that introns in ancient pre-mRNAs evolved from group II self-splicing introns through the progressive loss of internal RNA structures, which concurrently evolved into trans-acting snRNAs that perform the same functions. Support for this type of evolutionary model comes from experiments with group II intron mutants in which domain V and part of domain I are deleted. RNA transcripts containing such mutant introns are defective in self-splicing, but when RNA molecules equivalent to the deleted regions are added to the in vitro reaction, self-splicing occurs. This finding demonstrates that these domains in group II introns can be trans-acting, like snRNAs.

The similarity in the mechanisms of group II intron selfsplicing and of spliceosomal splicing of pre-mRNAs also suggests that the splicing reaction is catalyzed by the snRNA, not the protein, components of spliceosomes. Although group II introns can self-splice in vitro at elevated temperatures and $\mathrm{Mg}^{2+}$ concentrations, under in vivo conditions, proteins called maturases, which bind to group II intron RNA, are required for rapid splicing. Maturases are thought to stabilize the precise three-dimensional interactions of the intron RNA required to catalyze the two splicing transesterification reactions. By analogy, snRNP proteins in spliceosomes are thought to stabilize the precise geometry of snRNAs and intron nucleotides required to catalyze pre-mRNA splicing.

The evolution of snRNAs may have been an important step in the rapid evolution of higher eukaryotes. As sequences involved in self-splicing were lost from introns and their functions supplanted by trans-acting snRNAs, the remaining intron sequences would have become free to diverge. This in turn probably facilitated the evolution of new genes through exon shuffling, since there would be few constraints on the sequences of new introns generated
in the process (see Figures 8-18 and 8-19). It also permitted the increase in protein diversity that results from alternative RNA splicing and an additional level of gene control resulting from regulated RNA splicing.

## 3' Cleavage and Polyadenylation of Pre-mRNAs Are Tightly Coupled

In eukaryotic cells, all mRNAs, except histone mRNAs,* have a 3' poly(A) tail. Early studies of pulse-labeled adenovirus and SV40 RNA demonstrated that the viral primary transcripts extend beyond the site from which the poly(A) tail extends. These results suggested that A residues are added to a 3' hydroxyl generated by endonucleolytic cleavage of a longer transcript, but the predicted downstream RNA fragments were never detected in vivo, presumably because of their rapid degradation. However, both predicted cleavage products were observed in in vitro processing reactions performed with nuclear extracts of cultured human cells. The cleavage/polyadenylation process and degradation of the RNA downstream of the cleavage site occurs much more slowly in these in vitro reactions, simplifying detection of the downstream cleavage product.

Early sequencing of cDNA clones from animal cells showed that nearly all mRNAs contain the sequence AAUAAA 15-30 nucleotides upstream from the poly(A) tail (Figure 10-15). Polyadenylation of RNA transcripts is virtually eliminated when the corresponding sequence in the template DNA is mutated to any other sequence except one encoding a closely related sequence (AUUAAA). The unprocessed RNA transcripts produced from such mutant templates do not accumulate in nuclei, but are rapidly degraded. Further mutagenesis studies revealed that a second signal downstream from the cleavage site is required for efficient cleavage and polyadenylation of most pre-mRNAs in animal cells. This downstream signal is not a specific sequence, but rather a GU-rich or simply a U-rich region within about 20 nucleotides of the cleavage site.

Identification and purification of the proteins required for cleavage and polyadenylation of pre-mRNA have led to the model shown in Figure 10-15. A $360-\mathrm{kDa}$ cleavage and polyadenylation specificity factor (CPSF), composed of five different polypeptides, first forms an unstable complex with the upstream AAUAAA polyadenylation signal. Then at least three additional proteins bind to the CPSF-RNA complex: a $200-\mathrm{kDa}$ heterotrimer called cleavage stimulatory factor (CStF), which interacts with the G/U-rich sequence; a $150-\mathrm{kDa}$ heterotetramer called cleavage factor I (CFI); and a second heterodimeric cleavage factor (CFII). A $150-\mathrm{kDa}$ protein called symplekin is thought to form a scaffold on which these cleavage/polyadenylation factors assemble. Finally,

[^0]
[^0]:    *The major histone mRNAs are transcribed from repeated genes in prodigious amounts in replicating cells during the S phase. They undergo a special form of 3'-end processing that involves cleavage but not polyadenylation. Specialized RNA-binding proteins that help to regulate histone mRNA translation bind to the 3' end generated by this specialized system.

![li3002239539b](li3002239539b.jpg)

FIGURE 10-15 Model for cleavage and polyadenylation of pre-mRNAs in mammalian cells. Cleavage and polyadenylation specificity factor (CPSF) binds to the upstream AAUAAA polyadenylation signal. CStF interacts with a downstream GU- or U-rich sequence and with bound CPSF, forming a loop in the RNA; binding of CFI and CFII helps stabilize the complex. Binding of poly(A) polymerase (PAP) then stimulates cleavage at a poly(A) cleavage site, which usually is $15-30$ nucleotides $3^{\prime}$ of the upstream polyadenylation signal. The cleavage factors are released, as is the downstream RNA cleavage product, which is rapidly degraded. Bound PAP then adds about 12 A residues at a slow rate to the $3^{\prime}$-hydroxyl group generated by the cleavage reaction. Binding of nuclear poly(A)-binding protein (PABPN1) to the initial short poly(A) tail accelerates the rate of addition by PAP. After 200-250 A residues have been added, PABPN1 signals PAP to stop polymerization.
poly(A) polymerase (PAP) must bind to the complex before cleavage can occur. This requirement for PAP binding links cleavage and polyadenylation, so that the free $3^{\prime}$ end generated is rapidly polyadenylated and no essential information is lost to exonuclease degradation of an unprotected $3^{\prime}$ end.

Assembly of this large multiprotein cleavage/polyadenylation complex around the AU-rich polyadenylation signal in a pre-mRNA is analogous in many ways to formation of the transcription preinitiation complex at the AT-rich TATA box of a template DNA molecule (see Figure 9-19). In both cases, multiprotein complexes assemble cooperatively through a network of specific protein-nucleic acid and protein-protein interactions.

Following cleavage at the poly(A) site, polyadenylation proceeds in two phases: addition of the first 12 or so A residues occurs slowly, followed by rapid addition of up to 200-250 more A residues. The rapid phase requires the binding of multiple copies of a poly(A)-binding protein containing the RRM motif. This protein is designated PABPN1 to distinguish it from the poly(A)-binding protein that is present in the cytoplasm in humans, PABPC1. PABPN1 binds cooperatively to the short A tail initially added by PAP and to CPSF bound to the AAUAAA polyadenylation signal. This binding stimulates the PAP to extend the short poly(A) tail rapidly and processively; that is, without releasing the growing poly(A) tail from the complex of PABPN1 and CPSF. Once the poly(A) tail reaches a length of about 250 adenines, this processivity is lost, and PAP dissociates from the poly(A)-PABPN1 complex, terminating A addition (see Figure 10-15). Binding of PABPN1 to the poly(A) tail is essential for mRNA export into the cytoplasm. As for splicing factors, several of the subunits of the proteins involved in cleavage and polyadenylation associate with the serine 2-phosphorylated CTD of RNA polymerase II, which concentrates them in the region where polyadenylation signals in the RNA emerge from the elongating polymerase.

In wild-type genes, RNA polymerase II terminates transcription at any one of multiple possible sites within about 2 kb of the polyadenylation signal. Experiments with SV40 and adenovirus (both DNA viruses) showed that when the polyadenylation signal is mutated, RNA polymerase II does not terminate transcription, but continues transcription until the next poly(A) site in the viral genome is encountered. Similar results were soon shown for a recombinant

human $\beta$-globin gene inserted into an adenovirus. These experiments showed that transcription termination by RNA polymerase II is coupled to cleavage and polyadenylation of the transcript. It is hypothesized that this is due to the de-protection of the $5^{\prime}$ end of the nascent RNA. Because no cap is present on the $5^{\prime}$ end of the cleaved RNA, it is susceptible to the XRN1 $5^{\prime} \rightarrow 3^{\prime}$ exoribonuclease. It is thought that when this exoribonuclease reaches the still-transcribing polymerase, it triggers termination, either by pulling the $3^{\prime}$ end of the nascent RNA out of the polymerase active site or by inducing a conformational change in the polymerase that causes transcription termination. Once the nascent RNA is removed from the elongating polymerase, the contacts between the RNA polymerase II clamp and the RNA-DNA hybrid within the polymerase (see Figure 9-15) are lost, allowing the clamp to open and releasing the polymerase from the DNA template. More recent chromatin immunoprecipitation studies (ChIP-seq) (see Figure 9-18) with antibody to RNA polymerase II indicate that the polymerase may be removed from the template DNA at multiple possible sites within about 2 kb downstream from the poly(A) site.

## Nuclear Exoribonucleases Degrade RNA That Is Processed Out of Pre-mRNAs

Because the human genome contains long introns, only about 5 percent of the nucleotides that are polymerized by RNA polymerase II during transcription are retained in mature, processed mRNAs. Although this process appears inefficient, it probably evolved in multicellular organisms because the process of exon shuffling facilitated the evolution of new genes in organisms with long introns (see Chapter 8). The introns that are spliced out and the RNA downstream from the cleavage/polyadenylation site are degraded by nuclear exoribonucleases.

As mentioned earlier, the $2^{\prime}, 5^{\prime}$-phosphodiester bond in excised introns is hydrolyzed by a debranching enzyme (see Figure 10-11, step 11), yielding a linear molecule with unprotected ends. Such linear RNA molecules can be attacked by exoribonucleases, which hydrolyze one base at a time from the $5^{\prime}$ or $3^{\prime}$ end (as opposed to endoribonucleases, which digest internal phosphodiester bonds). The predominant mechanism of RNA decay is digestion by a large ( $\sim 400-\mathrm{kDa}$ ) protein complex called the exosome, which contains an internal $3^{\prime} \rightarrow 5^{\prime}$ exoribonuclease (Figure 10-16). (Exosomes also function in the cytoplasm, as discussed later.) The exosome is in many ways analogous to the proteasome (see Figure 3-31) that digests polyubiquitinylated proteins in both the nucleus and the cytoplasm. The predominant active site of the exosome lies on the inside of the complex, where it can digest only single-stranded RNAs that are threaded into the pore at the top of the complex (Figure 10-16b). This pore is too small to allow the entry of double-stranded or other structured regions of RNAs. Other proteins that associate with the complex include an RNA helicase, which disrupts base pairing and RNA-protein interactions that would otherwise prevent the entry of RNA into the pore.

In addition to introns, the exosome also degrades premRNAs that have not been properly spliced or polyadenylated, although at present, it is not yet clear how the exosome recognizes improperly processed pre-mRNAs. But in yeast cells with temperature-sensitive mutant PAP (see Figure 10-15), pre-mRNAs are retained at their sites of transcription in the nucleus at the nonpermissive temperature. These abnormally processed pre-mRNAs are released in cells with a second mutation in a subunit of the exosome found only in nuclear and not in cytoplasmic exosomes (Rrp6, see Figure 10-16). In addition, exosomes are found concentrated at sites of transcription in Drosophila polytene chromosomes, where they are associated with RNA polymerase II elongation factors. These results suggest that the exosome participates in an as yet poorly understood quality-control mechanism in the nucleus that recognizes aberrantly processed pre-mRNAs, preventing their export to the cytoplasm and ultimately leading to their degradation.

To avoid being degraded by nuclear exonucleases, nascent transcripts, pre-mRNA-processing intermediates, and mature mRNAs in the nucleus must have their ends protected. As discussed above, the $5^{\prime}$ end of a nascent transcript is protected by addition of the $5^{\prime}$ cap structure as soon as the $5^{\prime}$ end emerges from the polymerase. The $5^{\prime}$ cap is protected because it is bound by a heterodimeric nuclear cap-binding complex (CBC), which protects it from $5^{\prime}$ exonucleases and also functions in export of the mRNA to the cytoplasm. The $3^{\prime}$ end of a nascent transcript lies within the RNA polymerase and is thus inaccessible to exonucleases (see Figure 5-12). As discussed previously, the free $3^{\prime}$ end generated by cleavage of a pre-mRNA downstream from the polyadenylation signal is rapidly polyadenylated by the PAP associated with the other $3^{\prime}$ processing factors, and the resulting poly(A) tail is bound by PABPN1 (see Figure 10-15). This tight coupling of cleavage and polyadenylation, followed by PABPN1 binding, protects the $3^{\prime}$ end from exonuclease attack.

## RNA Processing Solves the Problem of Pervasive Transcription of the Genome in Metazoans

As discussed in Chapter 9, analysis of the location of transcribing RNA polymerase II in metazoan cells revealed the surprising result that the polymerase transcribes in the downstream direction, into coding regions, and in the upstream direction, away from coding regions, at nearly equal frequency from most promoters (see Figure 9-18). This finding was confirmed by deep sequencing of small RNAs isolated from metazoan cells, which revealed low levels of short, capped RNAs transcribed from both the sense and antisense strands at CpG island promoters, which account for some 70 percent of mammalian promoters. Indeed, deep sequencing of all cellular RNAs showed that both strands of nearly the entire genome are transcribed, although much of the resulting RNA is present at extremely low concentrations of less than one molecule per cell. This finding raised the question of how the cell deals with such "pervasive transcription."

Sequence analysis of these low-abundance short, capped RNAs indicates that they are probably prevented from

![li3012239539b](li3012239539b.jpg)

FIGURE 10-16 Structure of the exosome. (a) Catalytically inactive exosome core. A nine-subunit, 286-kDa human exosome core was assembled in vitro from subunits Rrp41, Rrp45, Rrp42, Mtr3, Rrp43, Rrp46, Rrp4, Rrp40, and Csl4 expressed at a high level in E. coli (see Figure 6-29). Its structure was determined to a resolution of $3.35 \AA$ by x-ray crystallography. (b) The 10 -subunit, catalytically active cytoplasmic exosome. The orientation is similar to that of the upper right image in part (a), but rotated slightly counterclockwise. Processive $3^{\prime} \rightarrow 5^{\prime}$ exonuclease activity is provided by the tenth subunit, Rrp44 (pink), associated with the bottom of the core. The C-terminus of an eleventh subunit, Rrp6, in the nuclear exosome is shown in maroon. RNA with a double-stranded region at the top and a $3^{\prime}$ single-stranded region that enters the core pore is shown in black. (c) Diagram of the 14-subunit nuclear exosome. Exo-10 represents the 10 -subunit complex shown in (b). A heterodimer
of Rrp6 and Rrp47 associates with Csl4 at the top of the exosome core through the C-terminal domain of Rrp6, as shown in (b). The N-terminus of an RNA helicase, Mtr4 (blue), associates with the heterodimerization domain of Rrp6 and Rrp47. Another subunit associated with the top, Mpp6, also associates with the Mtr4 RNA helicase in the human nuclear exosome, but its structure and the details of the Mpp6-Mtr4 interaction remain to be determined. The path of single-stranded RNA through the exosome is diagrammed in red. The exonuclease active site in the processive exonuclease Rrp44 is indicated by a pink circle. An endonuclease active site in Rrp44 is represented by the pink oval. A non-processive $3^{\prime} \rightarrow 5^{\prime}$ exonuclease active site in Rrp6 is represented by a maroon oval. See B. Schuch et al., 2014. EMBO J. 33:2829. [Part (a) data from Q. Liu, J. C. Greimann, and C. D. Lima, 2006, Cell 127:1223. Part (b) data from D. L. Makino, M. Braumgartner, and E. Conti, 2013, Nature 495:70. PDB ID 4ifd.]
reaching high concentrations by RNA processing and nuclear surveillance for abnormally processed RNAs. Sequencing of RNAs from several cell types has revealed that the antisense RNAs have a higher frequency of AAUAAA polyadenylation signal sequences transcribed from the AT-rich DNA of most metazoans ( 60 percent AT in mammals) than do transcripts transcribed in the sense direction into coding regions. Because of the high AT composition of mammalian DNA, an AAUAAA sequence in an antisense transcript is frequently followed by a U-rich sequence that may function as the downstream element of a bona fide pre-mRNA cleavage/ polyadenylation signal (see Figure 10-15). These cleavage/ polyadenylation signals occur much less frequently in transcripts going into coding regions. Where they do occur in the sequence of pre-mRNAs, in either exons or introns, they usually lie downstream of consensus base-pairing sites for U1 snRNA, which has been found to suppress cleavage/polyadenylation following nearby AAUAAA sequences. This function of U1 snRNA may help to explain why the U1 snRNP is much more abundant than the other spliceosomal snRNPs.

This is not the case for cleavage/polyadenylation signals used in the processing of $3^{\prime}$ ends of mRNAs because U1 snRNA associates with the $5^{\prime}$ end of the terminal intron, far from the poly(A) site. In addition, as discussed above, transcription by RNA polymerase II usually terminates within $\sim 2 \mathrm{~kb}$ following cleavage and polyadenylation of a pre-mRNA. Consequently, the enrichment of poly $(\mathrm{A})$ sites, and the relative lack of binding sites for U1 snRNA, in antisense transcripts may lead to cleavage of most of these transcripts within $\sim 2 \mathrm{~kb}$ of the transcription start site by cleavage/polyadenylation factors (see Figure 10-15), followed by termination of transcription (Figure 10-17). Cleaved antisense transcripts are probably degraded by the same nuclear exonucleases that degrade introns spliced out of pre-mRNAs and sequences downstream of pre-mRNA cleavage/polyadenylation sites, as well as sequences processed out of rRNA and tRNA precursors, discussed in a later section (see Figure 10-1). As a result, even though a large number of polymerases transcribe in the "wrong" direction, most of the transcripts generated in this way are rapidly degraded.

![li3022239539b](li3022239539b.jpg)

FIGURE 10-17 RNA transcribed in the "wrong" direction from most promoters in metazoans has a high frequency of polyadenylation signals and a low frequency of binding sites for U1 snRNA. This pattern may account for the termination of transcription in the "wrong" direction after about 2 kb for most of these transcripts. PAS represents polyadenylation signals encoded in the DNA that are transcribed into RNA. Cleavage of transcripts transcribed in the upstream direction (scissors) is proposed to generate free RNA ends that

## KEY CONCEPTS OF SECTION 10.1

## Processing of Eukaryotic Pre-mRNA

- In the nucleus of eukaryotic cells, pre-mRNAs are associated with hnRNP proteins and processed by $5^{\prime}$ capping, $3^{\prime}$ cleavage and polyadenylation, and splicing before being transported to the cytoplasm (see Figure 10-2).
- Shortly after transcription initiation, capping enzymes associate with the carboxy-terminal domain (CTD) of RNA polymerase II, phosphorylated multiple times at serine 5 of the heptapeptide repeat by TFIIH during transcription initiation. These enzymes then rapidly add the $5^{\prime}$ cap to the nascent transcript when it reaches a length of about 25 nucleotides. Other RNA-processing factors involved in RNA splicing and in $3^{\prime}$ cleavage and polyadenylation associate with the CTD when it is phosphorylated at serine 2 of the heptapeptide repeat, increasing the rate of transcription elongation. Consequently, transcription does not proceed at a high rate until RNA-processing factors become associated with the CTD, where they are poised to interact with the nascent pre-mRNA as it emerges from the surface of the polymerase.
- Five different snRNPs interact via base pairing with one another and with pre-mRNA to form the spliceosome (see Figure 10-11). This very large ribonucleoprotein complex catalyzes two transesterification reactions that join two exons and remove the intron as a lariat structure, which is subsequently degraded (see Figure 10-8).
- SR proteins that bind to exonic splicing enhancer sequences in exons are critical in defining exons in the large
are digested by the nuclear exosome and a nuclear $5^{\prime} \rightarrow 3^{\prime}$ exonuclease, XRN1. In contrast, pre-mRNAs synthesized by RNA polymerase II transcribing into coding regions have evolved to have few polyadenylation signals. Where they do occur, these signals are usually preceded by a binding site for U1 snRNP, which inhibits cleavage at a nearby PAS (stop sign). However, the PAS used to generate the $3^{\prime}$ end of an mRNA does not have a closely associated U1 RNP binding site. See A. E. Almada et al., 2013, Nature 499:360.
pre-mRNAs of higher organisms. A network of interactions between SR proteins, snRNPs, and splicing factors forms a cross-exon recognition complex that specifies correct splice sites (see Figure 10-13).
- The snRNAs in the spliceosome are thought to have an overall tertiary structure similar to that of group II self-splicing introns.
- For long transcription units in higher organisms, splicing of exons usually begins as the pre-mRNA is still being formed. Cleavage and polyadenylation to form the $3^{\prime}$ end of the mRNA occur after the poly(A) cleavage site is transcribed.
- In most protein-coding genes, a conserved AAUAAA polyadenylation signal lies slightly upstream from a poly(A) site where cleavage and polyadenylation occur. A GU- or U-rich sequence downstream from the poly(A) site contributes to the efficiency of cleavage and polyadenylation.
- A multiprotein complex that includes poly(A) polymerase (PAP) carries out the cleavage and polyadenylation of a pre-mRNA. A nuclear poly(A)-binding protein, PABPN1, stimulates addition of A residues by PAP and stops their addition once the poly(A) tail reaches about 250 residues (see Figure 10-15).
- Excised introns and RNA downstream from the cleavage/ polyadenylation site are degraded primarily by exosomes, multiprotein complexes that contain an internal $3^{\prime} \rightarrow 5^{\prime}$ exonuclease. Exosomes also degrade improperly processed pre-mRNAs.

### 10.2 Regulation of Pre-mRNA Processing

Now that we've seen how pre-mRNAs are processed into mature, functional mRNAs, let's consider how regulation of this process can contribute to gene control. Recall from Chapter 8 that higher eukaryotes have both simple and complex transcription units encoded in their DNA. The primary transcripts produced from the former contain one poly(A) site and exhibit only one pattern of RNA splicing, even if multiple introns are present; thus simple transcription units encode a single mRNA. In contrast, the primary transcripts produced from complex transcription units (which constitute about 95 percent of all human transcription units) can be processed in alternative ways to yield different mRNAs that encode distinct proteins (see Figure 8-3).

## Alternative Splicing Generates Transcripts with Different Combinations of Exons

The discovery that a large fraction of transcription units in higher organisms encode alternatively spliced mRNAs and that differently spliced mRNAs are expressed in different cell types revealed that regulation of RNA splicing is an important gene-control mechanism in higher eukaryotes. Although many examples of cleavage at alternative poly(A) sites in pre-mRNAs are known, alternative splicing of different exons is the more common mechanism for expressing different proteins from one complex transcription unit. In Chapter 5, for example, we mentioned that fibroblasts produce one type of the extracellular protein fibronectin,
whereas hepatocytes produce another type. Both fibronectin isoforms are encoded by the same transcription unit, but the transcript is spliced differently in the two cell types to yield two different mRNAs (see Figure 5-16). In other cases, alternative processing of the same transcript may occur simultaneously in the same cell type in response to different developmental or environmental signals. We first discuss one of the best-understood examples of regulated RNA processing, then briefly consider the consequences of RNA splicing in the development of the nervous system.

## A Cascade of Regulated RNA Splicing Controls Drosophila Sexual Differentiation

One of the earliest examples of regulated alternative splicing of pre-mRNA came from studies of sexual differentiation in Drosophila. The genes required for normal Drosophila sexual differentiation were first characterized by isolating Drosophila mutants defective in the process. When the proteins encoded by the wild-type genes were characterized biochemically, two of them were found to regulate a cascade of alternative RNA splicing in Drosophila embryos. More recent research has provided insight into how these proteins regulate RNA processing and ultimately lead to the creation of two different sex-specific transcriptional repressors that suppress the development of characteristics of the opposite sex.

The Sex-lethal (Sxl) protein, encoded by the sex-lethal gene, is the first protein to act in the cascade (Figure 10-18). The Sxl protein is present only in female embryos. Early in
![li3032239539b](li3032239539b.jpg)

FIGURE 10-18 A cascade of regulated splicing controls sex determination in Drosophila embryos. For clarity, only the exons (boxes) and introns (black lines) where regulated splicing occurs are shown. Splicing is indicated by red dashed lines above (female) and blue dashed lines below (male) the pre-mRNAs. Vertical red lines in exons indicate in-frame stop codons, which prevent synthesis of functional protein. Only female embryos produce functional Sxl protein, which represses splicing between exons 2 and 3 in $s x l$ pre-mRNA (a) and between exons 1 and 2 in tra pre-mRNA (b). (c) In contrast, the cooperative binding of Tra protein and two SR proteins, Rbp1 and Tra2, activates splicing between exons 3 and 4 and cleavage/polyadenylation $\left(\mathrm{A}_{1}\right)$ at the $3^{\prime}$ end of exon 4 in dsx pre-mRNA in female embryos. In male embryos, which lack functional Tra, the SR proteins do not bind to exon 4, and consequently exon 3 is spliced to exon 5. The distinct Dsx proteins produced in female and male embryos as the result of this cascade of regulated splicing repress transcription of genes required for sexual differentiation of the opposite sex. See M. J. Moore et al., 1993, in R. Gesteland and J. Atkins, eds., The RNA World, Cold Spring Harbor Press, pp. 303-357.

development, the Sxl gene is transcribed from a promoter that functions only in female embryos. Later in development, this female-specific promoter is shut off, and another promoter for sex-lethal becomes active in both male and female embryos. In male embryos, however, in the absence of early Sxl protein, exon 2 of the sex-lethal pre-mRNA is spliced to exon 3 to produce an mRNA that contains a stop codon early in the sequence. The net result is that male embryos produce no functional Sxl protein either early or later in development.

In contrast, the Sxl protein expressed in early female embryos regulates splicing of the sex-lethal pre-mRNA so that a functional sex-lethal mRNA is produced (Figure 10-18a). Sxl accomplishes this by binding to a sequence in the pre-mRNA near the $3^{\prime}$ end of the intron between exon 2 and exon 3, thereby blocking the proper association of U2AF and U2 snRNP with the adjacent $3^{\prime}$ splice site used in males (see Figure 10-11). As a consequence, the U1 snRNP bound to the $5^{\prime}$ end of the intron between exons 2 and 3 assembles into a spliceosome with U2 snRNP bound to the branch point at the $3^{\prime}$ end of the intron between exons 3 and 4 , leading to the splicing of exon 2 to exon 4 and the skipping of exon 3. The binding site for Sxl in the sex-lethal pre-mRNA is called an intronic splicing silencer because of its location in an intron and its function in blocking, or "silencing," the use of a splice site. The resulting female-specific sex-lethal mRNA is translated into functional Sxl protein, which reinforces its own expression in female embryos by continuing to cause skipping of exon 3 . The absence of Sxl protein in male embryos allows the inclusion of exon 3 and, consequently, of the stop codon near the $5^{\prime}$ end of exon 3 that prevents translation of functional Sxl protein (see Figure 10-18a).

Sxl protein also regulates alternative splicing of the pre-mRNA transcribed from the transformer gene (Figure 10-18b). In male embryos, in which no Sxl is expressed, exon 1 is spliced to exon 2, which contains a stop codon that prevents synthesis of a functional Transformer (Tra) protein. In female embryos, however, binding of Sxl protein to an intronic splicing silencer at the $3^{\prime}$ end of the intron between exons 1 and 2 blocks binding of U2AF at this site. The interaction of Sxl with transformer pre-mRNA is mediated by two adjacent RRM domains in the protein (see Figure 10-5). When Sxl is bound, U2AF binds to a lower-affinity site farther $3^{\prime}$ in the pre-mRNA; as a result, exon 1 is spliced to this alternative $3^{\prime}$ splice site, causing skipping of exon 2 with its stop codon. The resulting female-specific transformer mRNA, which contains additional constitutively spliced exons, is translated into functional Tra protein.

Finally, Tra protein regulates the alternative processing of pre-mRNA transcribed from the doublesex (dsx) gene (Figure 10-18c). In female embryos, a complex of Tra and two constitutively expressed SR proteins, Rbp1 and Tra2, directs the splicing of exon 3 to exon 4 and also promotes cleavage/polyadenylation at the alternative poly $(\mathrm{A})$ site at the $3^{\prime}$ end of exon 4 , leading to a short, female-specific version of the Dsx protein. In male embryos, which produce
no Tra protein, exon 4 is skipped, so that exon 3 is spliced to exon 5. Exon 5 is constitutively spliced to exon 6 , which is polyadenylated at its $3^{\prime}$ end-leading to a longer, male-specific version of the Dsx protein. The RNA sequence to which Tra binds in exon 4 is called an exonic splicing enhancer because it enhances splicing at a nearby splice site.

As a result of the cascade of regulated RNA processing depicted in Figure 10-18, different Dsx proteins are expressed in male and female embryos. The two proteins are transcription factors that share the N -terminal sequence encoded in exons $1-3$, including a common DNA-binding domain, but have different C-terminal sequences, encoded by exon 4 in females and exon 5 plus additional downstream exons in males. The unique C-terminal end of the female protein functions as a strong activation domain, while the C-terminal end of the male protein is a strong repression domain. Consequently, the female Dsx protein activates genes with binding sites for the transcription factor, including genes that induce development of female characteristics, while the male Dsx protein represses the same target genes.

Figure 10-19 illustrates how the Tra/Tra2/Rbp1 complex is thought to interact with doublesex pre-mRNA. Rbp1 and Tra2 are SR proteins, but they do not interact with exon 4 in the absence of the Tra protein. The interaction of the Tra protein with Rbp1 and Tra2 results in the cooperative binding of all three proteins to six exonic splicing enhancers in exon 4. The bound Tra2 and Rbp1 proteins then promote the binding of U2AF and the U2 snRNP to the $3^{\prime}$ end of the intron between exons 3 and 4 , just as other SR proteins do for constitutively spliced exons (see Figure 10-13). The Tra/ Tra2/Rbp1 complexes also enhance binding of the cleavage/ polyadenylation complex to the $3^{\prime}$ end of exon 4 because the U2 snRNP plus associated proteins bound to a $3^{\prime}$ splice site enhance binding of cleavage/polyadenylation factors (see Figure 10-15) to an appropriately spaced polyadenylation signal through cooperative binding interactions.
![li3042239539b](li3042239539b.jpg)

FIGURE 10-19 Model of splicing activation by Tra protein and the SR proteins Rbp1 and Tra2. In female Drosophila embryos, splicing of exons 3 and 4 in dsx pre-mRNA is activated by the binding of Tra/Tra2/Rbp1 complexes to six exonic splicing enhancers in exon 4. Because Rbp1 and Tra2 cannot bind to the pre-mRNA in the absence of Tra, exon 4 is skipped in male embryos. See the text for discussion. $A_{n}=$ polyadenylation. See T. Maniatis and B. Tasic, 2002, Nature 418:236.

## Splicing Repressors and Activators Control Splicing at Alternative Sites

As is evident from Figure 10-18, the Drosophila Sxl protein and Tra protein have opposite effects: Sxl prevents splicing, causing exons to be skipped, whereas Tra promotes splicing. The action of similar proteins may explain the cell-type-specific expression of fibronectin isoforms in humans (see Figure 5-16). For instance, an Sxl-like splicing repressor expressed in hepatocytes might bind to splice sites for the EIIIA and EIIIB exons in the fibronectin pre-mRNA, causing them to be skipped during RNA splicing. Alternatively, a Tra-like splicing activator expressed in fibroblasts might activate the splice sites associated with those exons, leading to their inclusion in the mature mRNA. Experimental examination of some systems has revealed that the inclusion of an exon in some cell types and the skipping of the same exon in other cell types results from the combined influence of several splicing repressors (usually hnRNP proteins) and enhancers (usually SR proteins). RNA binding sites for repressors can also occur in exons, where they are called exonic splicing silencers. And binding sites for splicing activators can also occur in introns, where they are called intronic splicing enhancers.

Alternative splicing of exons is especially common in the nervous system, where it generates multiple isoforms of many proteins required for neuronal development and function in both vertebrates and invertebrates. The primary transcripts of the genes encoding these proteins often show complex splicing patterns that can generate several different mRNAs, which are expressed in different anatomic locations within the central nervous system. Here we consider two remarkable examples that illustrate the critical role of this process in neural function.

Expression of $\mathrm{K}^{+}$-Channel Proteins in Vertebrate Hair Cells In the inner ear of vertebrates, individual hair cells, which are ciliated neurons, respond most strongly to a specific frequency of sound. Cells tuned to low frequencies ( $\sim 50 \mathrm{~Hz}$ ) are found at one end of the tubular cochlea that makes up the inner ear; cells responding to high frequencies ( $\sim 5000 \mathrm{~Hz}$ ) are found at the other end (Figure 10-20a). Cells in between the two ends respond to a gradient of frequencies between these extremes. One component in the tuning of hair cells in reptiles and birds is the opening of $\mathrm{K}^{+}$ion channels in response to increased intracellular $\mathrm{Ca}^{2+}$ concentrations. The $\mathrm{Ca}^{2+}$ concentration at which the channel opens determines the frequency with which the membrane potential oscillates and hence the frequency to which the cell is tuned.

The gene encoding this $\mathrm{Ca}^{2+}$-activated $\mathrm{K}^{+}$channel is expressed as multiple, alternatively spliced mRNAs, which encode proteins that open at different $\mathrm{Ca}^{2+}$ concentrations. Hair cells with different response frequencies express different isoforms of the channel protein depending on their position along the length of the cochlea. The sequence variation in the protein is very complex: there are at least eight regions in the mRNA where one of several alternative exons is utilized, permitting the expression of 576 possible isoforms (Figure 10-20b).
(a)
![li3052239539b](li3052239539b.jpg)
(b)
![li3062239539b](li3062239539b.jpg)

FIGURE 10-20 Role of alternative splicing in the perception of sounds of different frequencies. (a) The chicken cochlea, a 5-mmlong tube, contains an epithelium of auditory hair cells that are tuned to a gradient of vibrational frequencies from 50 Hz at the apical end (left) to 5000 Hz at the basal end (right). (b) The $\mathrm{Ca}^{2+}$-activated $\mathrm{K}^{+}$ channel contains seven transmembrane $\alpha$ helices (50-56), which associate to form the channel. The cytosolic domain, which includes four hydrophobic regions (57-510), regulates opening of the channel in response to $\mathrm{Ca}^{2+}$. Isoforms of the channel, encoded by alternatively spliced mRNAs produced from the same primary transcript, open at different $\mathrm{Ca}^{2+}$ concentrations and thus respond to different frequencies. Red numbers refer to regions where alternative splicing produces different amino acid sequences in the various isoforms. See K. P. Rosenblatt et al., 1997, Neuron 19:1061.

PCR analysis of mRNAs from individual hair cells has shown that each hair cell expresses a mixture of different $\mathrm{K}^{+}$-channel mRNAs, with different isoforms predominating in different cells according to their position along the cochlea. This remarkable arrangement suggests that splicing of the $\mathrm{K}^{+}$-channel pre-mRNA is regulated in response to extracellular signals that inform the cell of its position along the cochlea.

Other studies have demonstrated that splicing at one of the alternative splice sites in the $\mathrm{Ca}^{2}+$-activated $\mathrm{K}+$-channel pre-mRNA in the rat is suppressed when a specific protein kinase is activated by neuron depolarization in response to synaptic activity from interacting neurons. This observation raises the possibility that a splicing repressor specific for this splice site may be activated when it is phosphorylated by this protein kinase, whose activity in turn is regulated by synaptic activity. Since hnRNP and SR proteins are extensively modified by phosphorylation and other post-translational

modifications, it seems likely that complex regulation of alternative RNA splicing through post-translational modifications of splicing factors plays a significant role in modulating neuron function.

Many examples of genes similar to those that encode the cochlear $\mathrm{K}^{+}$channel have been observed in vertebrate neurons; in these cases, alternatively spliced mRNAs co-expressed from a specific gene in one type of neuron are expressed at different relative concentrations in different regions of the central nervous system. Expansions in the number of microsatellite repeats within the transcribed regions of genes expressed in neurons can alter the relative concentrations of alternatively spliced mRNAs transcribed from multiple genes. In Chapter 8, we discussed how backward slippage during DNA replication can lead to expansion of a microsatellite repeat (see Figure 8-5). At least 14 different types of neurological diseases result from expansion of microsatellite regions within transcription units expressed in neurons. The resulting long regions of repeated simple sequences in nuclear pre-mRNAs of these neurons result in
abnormalities in the relative concentrations of alternatively spliced mRNAs. For example, the most common of these types of diseases, myotonic dystrophy, results from increased copies of either CUG repeats in one transcript, in some patients, or CCUG repeats in another transcript, in other patients. When the number of these repeats increases to 10 or more times the normal number of repeats, abnormalities are observed in the functions of two hnRNP proteins that bind to these repeated sequences. The abnormalities probably result because the hnRNPs are bound by the abnormally high concentrations of the repeats in the nuclei of neurons in these patients and cannot associate with other pre-mRNAs. This sequestration of the hnRNPs leads to alterations in the rate of splicing of different alternative splice sites in multiple pre-mRNAs that are normally regulated by these hnRNP proteins. Because of the importance of the proper regulation of alternative splicing for the normal function of neurons, multiple human neurological disorders are associated with abnormalities in the function of nuclear RNA-binding proteins and the expansion of microsatellite repeats that generate binding sites for splicing factors (Table 10-2).

# TABLE 10-2 Neurological Disorders with Links to Abnormalities in Alternative RNA Splicing 

| Disease | Link to Alternative Splicing |
| :--: | :--: |
| Ataxia telangiectasia | Point mutations within the ATM gene cause aberrant splicing of ATM transcripts |
| Fascioscapulohumoral dystrophy (FSHD) | Loss of FRG1, a nuclear RNA-binding protein, leads to altered splicing of many pre-mRNAs |
| Fragile-X-associated tremor/ataxia syndrome (FXTAS) | Premutation CGG repeat expansions in the FMR1 gene result in the sequestration of RNA-binding splicing factors |
| Frontotemporal dementia with Parkinsonism linked to chromosome 17 (FTDP-17) | Point mutations within the MAPT gene result in altered levels of MAPT transcripts containing the alternatively spliced exon 10 |
| Duchenne muscular dystrophy; Becker's muscular dystrophy | Altered splicing of dystrophin transcripts due to deletions and mutations in the dystrophin gene |
| MYOTONIC DYSTROPHY (DM) |  |
| DM1 | CUG expansion in the $3^{\prime}$ UTR of $D M P K$ results in the misregulation of the MBNL splicing factor and consequent missplicing of MBNL target pre-mRNAs |
| DM2 | CCUG expansion in ZNF9 intron leading to misregulation of the CUG-BP1 splicing factor and missplicing of CUG-BP1 target pre-mRNAs |
| Neurofibromatosis type 1 (NF1) | Numerous mutations in the NF1 gene, including mutations that result in aberrant splicing |
| PARANEOPLASTIC NEUROLOGIC DISORDERS (PND) |  |
| Paraneoplastic opsoclonus-myoclonus-ataxia (POMA) | Autoimmune antibodies recognize the Nova family of neuronspecific RNA-binding splicing factors; Nova knockout mice phenocopy POMA |
| Hu syndrome (PEM/SN; paraneoplastic encephalomyelitis/sensory neuronopathy) | Autoimmune antibodies recognize the Hu family of RNA-binding factors related to the Drosophila splicing factor ELAV |
| Prader-Willi syndrome | Loss of a splicing regulatory snoRNA that is complementary to a splicing silencer element implicated in regulating the alternative splicing of serotonin receptor $5-\mathrm{HT}_{2 \mathrm{~s}} R$ transcripts |

(Continued)

| Psychiatric disorders | Accumulation of aberrantly spliced transcripts in schizophrenic patients |
| :--: | :--: |
| Retinitis pigmentosa | Mutation of genes encoding U snRNP-associated proteins |
| Rett syndrome | Mutation of the gene encoding MeCP2, which interacts with the YB-1 RNA-binding protein; mouse model of Rett syndrome shows aberrant pre-mRNA splicing |
| Spinal muscular atrophy | Deletion/mutation of the SMN1 gene and the loss of a splicing regulatory element in SMN2 results in insufficient levels of SMN, which is involved in snRNP biogenesis |
| SPINOCEREBELLAR ATAXIAS |  |
| SCA2, SCA8, SCA10, and SCA12 | Possible RNA gain of function due to triplet repeat expansions; direct and indirect interactions with RNA-binding splicing factors |

source: Republished by permission of Elsevier, from Licatalosi, D. and Darnel, R., "Splicing regulation in neurologic disease," Neuron, 2006, 52:1, 93-101. Permission conveyed through the Copyright Clearance Center, Inc.

## Expression of Dscam Isoforms in Drosophila Retinal Neurons

The most extreme example of regulated alternative RNA processing yet uncovered occurs in expression of the Dscam gene in Drosophila. Mutations in this gene interfere with the normal synaptic connections made between retinal axons and dendrites during fly development. Analysis of the Dscam gene showed that it contains four groups of exons within which one of several possible exons is included in the final mature mRNA. The gene contains a total of 95 exons (Figure 10-21), generating 38,016 possible alternatively spliced isoforms! Drosophila mutants with a version of the gene that can be spliced in only about 22,000 different ways have specific defects in connectivity between neurons. These results indicate that expression of most of the possible Dscam isoforms through regulated RNA splicing helps to specify the tens of millions of different specific
![li3072239539b](li3072239539b.jpg)

FIGURE 10-21 The Drosophila Dscam gene is processed into a vast number of alternative isoforms. Dscam encodes a cell-surface protein on neurons. The protein (bottom) is composed of ten different immunoglobulin (Ig) domains (ovals), six different fibronectin type III domains (rectangles), one transmembrane domain (yellow), and a C-terminal cytoplasmic domain (dark gray). The fully processed mRNA is shown as rectangles representing each exon, with the length of the rectangle corresponding to the length of the exons, and a green circle representing the $5^{\prime}$ cap. Each mRNA contains one of the 12 Ig 2 exons shown in light blue (top), one of the 48 Ig 3 exons shown in green, one of the 33 Ig 7 exons shown in dark blue, and one of the 2 transmembrane exons shown in yellow. The exons shown in pink are spliced into each of the messages. Thus alternative splicing can generate $12 \times 48 \times 33 \times 2=38,016$ possible isoforms. See M. R. Sawaya et al., 2008, Cell 134:1007.
synaptic connections between neurons in the Drosophila brain. In other words, the correct wiring of neurons in the brain requires regulated RNA splicing.

## RNA Editing Alters the Sequences of Some Pre-mRNAs

In the mid-1980s, sequencing of numerous cDNA clones and corresponding genomic DNAs from multiple organisms led to the unexpected discovery of another type of pre-mRNA processing. In this type of processing, called RNA editing, the sequence of a pre-mRNA is altered; as a result, the sequence of a mature mRNA differs from that of the exons encoding it in genomic DNA.

RNA editing is widespread in the mitochondria of protozoans and plants as well as in chloroplasts. In the mitochondria of certain pathogenic trypanosomes, more than half the sequence of some mRNAs is altered from the sequence of the corresponding primary transcripts. Additions and deletions of specific numbers of Us follow templates provided by base-paired short "guide" RNAs. These RNAs are encoded by thousands of small circular DNA molecules concatenated to many fewer large DNA molecules. The reason for this baroque mechanism for encoding mitochondrial proteins in such protozoans is not clear. But this system does represent a potential target for drugs to inhibit the complex processing enzymes essential to the microbe that do not exist in the cells of its human or other vertebrate hosts.

In higher eukaryotes, RNA editing is much rarer, and thus far, only single-base changes have been observed. Such minor editing, however, turns out to have significant functional consequences in some cases. An important example of RNA editing in mammals involves the APOB gene, which encodes two alternative forms of a serum protein that is central to the uptake and transport of cholesterol. Consequently, it is important in the pathogenic processes that lead to atherosclerosis, the arterial disease that is the major cause of death in the developed world. The APOB gene encodes both the serum protein apolipoprotein B-100 (apoB-100), which is expressed in hepatocytes, the major cell type in the liver, and apoB-48, which is expressed in intestinal epithelial cells. The $240-\mathrm{kDa}$ apoB-48 corresponds to the N -terminal region of the $500-\mathrm{kDa}$

![li3082239539b](li3082239539b.jpg)

FIGURE 10-22 RNA editing of APOB pre-mRNA. The APOB mRNA produced in the liver has the same sequence as the exons in the primary transcript. This mRNA is translated into apoB-100, which has two functional domains: an N-terminal domain (green) that associates with lipids and a C-terminal domain (orange) that binds to LDL receptors on
apoB-100. Both ApoB proteins are components of the large lipoprotein complexes we described in Chapter 7, which transport lipids in the serum. However, only low-density lipoprotein (LDL) complexes, which contain apoB-100 on their surface, deliver cholesterol to body tissues by binding to the LDL receptor that is present on all cells (see Figures 14-27 and 14-29).

The cell-type-specific expression of the two forms of ApoB results from editing of ApoB pre-mRNA so as to change the nucleotide at position 6666 in the sequence from a C to a U. This alteration, which occurs only in intestinal cells, converts a CAA codon for glutamine to a UAA stop codon, leading to synthesis of the shorter apoB-48 (Figure 10-22). Studies with the partially purified enzyme that performs the post-transcriptional deamination of $\mathrm{C}_{6666}$ to $U$ (see Figure 2-17) shows that it can recognize and edit an RNA as short as 26 nucleotides containing the sequence surrounding $\mathrm{C}_{6666}$ in the ApoB primary transcript.

## KEY CONCEPTS OF SECTION 10.2

## Regulation of Pre-mRNA Processing

- Because of alternative splicing of primary transcripts, the use of alternative promoters, and cleavage at different poly(A) sites, different mRNAs may be expressed from the same gene in different cell types or at different developmental stages (see Figure 10-18).
- Alternative splicing can be regulated by RNA-binding proteins that bind to specific sequences near regulated splice sites. Splicing repressors may sterically block the binding of splicing factors to specific sites in pre-mRNAs or inhibit their function. Splicing activators enhance splicing by interacting with splicing factors, thus promoting their association with a regulated splice site. The RNA sequences bound by splicing repressors are called intronic or exonic splicing silencers, depending on their location in an intron or exon.
cell membranes. In the APOB mRNA produced in the intestine, however, the CAA codon in exon 26 is edited to a UAA stop codon. As a result, intestinal cells produce apoB-48, which corresponds to the N-terminal domain of apoB-100. See P. Hodges and J. Scott, 1992, Trends Biochem. Sci. 17:77.

RNA sequences bound by splicing activators are called intronic or exonic splicing enhancers.

- In RNA editing, the nucleotide sequence of a pre-mRNA is altered in the nucleus. In vertebrates, this process is relatively rare, and only single-base C to U changes have been observed, but those changes can have important consequences by altering the amino acid encoded by an edited codon (see Figure 10-22).


### 10.3 Transport of mRNA Across the Nuclear Envelope

Fully processed mRNAs in the nucleus remain bound by hnRNP proteins in complexes referred to as nuclear mRNPs. Before an mRNA can be translated into its encoded protein, it must be exported from the nucleus into the cytoplasm. The nuclear envelope is a double membrane that separates the nucleus from the cytoplasm (see Figure 1-12). Like the plasma membrane surrounding a cell, each nuclear membrane consists of a water-impermeable phospholipid bilayer and multiple associated proteins. mRNPs and other macromolecules, including tRNAs and ribosomal subunits, traverse the nuclear envelope through nuclear pore complexes (NPCs). This section focuses on the export of mRNPs through NPCs and the mechanisms that allow some level of regulation of this step. Transport of mRNPs, proteins, and other cargoes through NPCs is discussed in greater detail in Chapter 13.

Embedded in the nuclear envelope, NPCs are cylindrical in shape with a diameter of about 30 nm . Proteins and RNPs larger than $40-60 \mathrm{kDa}$ must be selectively transported across the nuclear envelope with the assistance of transporter proteins that bind them and also interact reversibly with components in the central channel of the NPC. mRNPs are transported through the NPC by the $m R N P$ exporter,

a heterodimer consisting of a large subunit, called nuclear export factor 1 (NXF1), and a small subunit, nuclear export transporter 1 (NXT1). NXF1 binds nuclear mRNPs through associations with both RNA and proteins in the mRNP complex. One of the most important of these proteins is REF (RNA export factor), a component of the exon-junction complexes discussed earlier, which is bound approximately 20 nucleotides $5^{\prime}$ to each exon-exon junction (Figure 10-23). The mRNP exporter also associates with SR proteins bound to exonic splicing enhancers. Thus SR proteins associated with exons function to direct both the splicing of pre-mRNAs and the export of fully processed mRNAs through NPCs to the cytoplasm. mRNPs are probably bound along their length by multiple mRNP exporters, which interact reversibly with unstructured protein domains that fill the NPC central channel (see Chapter 13).

Protein filaments extend from the core NPC scaffold into the nucleoplasm, forming an NPC nuclear basket (see Figure 10-23). Other protein filaments extend from the cytoplasmic face of the NPC into the cytoplasm. Both sets of filaments assist in mRNP export. Gle2, an adapter protein that reversibly binds both NXF1 and a protein in the nuclear basket, brings nuclear mRNPs to the NPC in preparation for export. A protein in the cytoplasmic filaments of the NPC binds an RNA helicase (Dbp5) that functions in
![li3092239539b](li3092239539b.jpg)

FIGURE 10-23 Remodeling of mRNPs during nuclear export.
Some mRNP proteins (rectangles) dissociate from nuclear mRNP complexes before their export through an NPC. Others (ovals) are exported through the NPC with the mRNP, but dissociate from it in the cytoplasm and are shuttled back into the nucleus through an NPC. In the cytoplasm, translation initiation factor elF4E replaces CBC bound to the $5^{\prime}$ cap, and PABPC1 replaces PABPN1.
the dissociation of NXF1/NXT1 and other hnRNP proteins from the mRNP as it reaches the cytoplasm.

In a process called $m R N P$ remodeling, the proteins associated with an mRNA in the nuclear mRNP are exchanged for a different set of proteins as the mRNP is transported through the NPC (see Figure 10-23). Some nuclear mRNP proteins dissociate early in transport, remaining in the nucleus to bind to newly synthesized nascent pre-mRNA. Other nuclear mRNP proteins remain with the mRNP as it traverses the NPC and do not dissociate from the mRNP until the complex reaches the cytoplasm. Proteins in this category include the NXF1/NXT1 mRNP exporter, the nuclear cap-binding complex (CBC) bound to the $5^{\prime}$ cap, and PABPN1 bound to the poly(A) tail. These proteins dissociate from the mRNP on the cytoplasmic side of the NPC through the action of the Dbp5 RNA helicase that associates with the cytoplasmic NPC filaments, as discussed above. These proteins are then imported back into the nucleus, as described for other nuclear proteins in Chapter 13, where they can function in the export of another mRNP. In the cytoplasm, the cap-binding translation initiation factor eIF4E replaces the CBC bound to the $5^{\prime}$ cap of nuclear mRNPs (see Figure 5-23). In vertebrates, the nuclear poly(A)-binding protein PABPN1 is replaced with the cytoplasmic poly(A)-binding protein PABPC1 (so named to distinguish it from the nuclear PABPN1). Only a single PABP is found in budding yeast, in both the nucleus and the cytoplasm.

## Phosphorylation and Dephosphorylation of SR Proteins Improses Directionality on mRNP Export Across the Nuclear Pore Complex

Studies of S. cerevisiae indicate that the direction of mRNP export from the nucleus into the cytoplasm is controlled by the phosphorylation and dephosphorylation of mRNP adapter proteins, such as REF, that assist in the binding of the NXF1/NXT1 mRNP exporter to mRNPs. In one case, a yeast SR protein (Npl3) functions as an adapter protein that promotes the binding of the yeast mRNP exporter (Figure 10-24). In its phosphorylated form, the SR protein initially binds to nascent pre-mRNA. When 3' cleavage and polyadenylation are completed, the adapter protein is dephosphorylated by a specific nuclear protein phosphatase that is essential for mRNP export. Only the dephosphorylated adapter protein can bind the mRNP exporter, thereby coupling mRNP export to correct polyadenylation. This mechanism is one form of mRNA "quality control." If the nascent mRNP is not correctly processed, it is not recognized by the phosphatase that dephosphorylates Npl3, and consequently, it is not bound by the mRNP exporter and is not exported from the nucleus. Instead, it is degraded by exosomes, the multiprotein complexes that degrade unprotected RNAs in the nucleus and cytoplasm (see Figures 10-1 and 10-16).

Following export to the cytoplasm, the Npl3 SR protein is phosphorylated by a specific cytoplasmic protein kinase.

![li3102239539b](li3102239539b.jpg)

FIGURE 10-24 Reversible phosphorylation and direction of mRNP nuclear export. Step The yeast SR protein Npl3 binds nascent pre-mRNAs in its phosphorylated form. Step When polyadenylation has occurred successfully, the Glc7 nuclear phosphatase dephosphorylates Npl3, promoting the binding of the mRNP exporter, NXF1/NXT1. Step The mRNP exporter allows diffusion of the mRNP complex through the central channel of the nuclear pore complex (NPC). Step The cytoplasmic protein kinase Sky1 phosphorylates

Npl3 in the cytoplasm, causing step dissociation of the phosphorylated Npl3 from the mRNP exporter, probably through the action of an RNA helicase associated with NPC cytoplasmic filaments step . The mRNA transporter and phosphorylated Npl3 are transported back into the nucleus through NPCs. Step Transported mRNA is available for translation in the cytoplasm. See E. Izaurralde, 2004, Nat. Struct. Mol. Biol. 11:210-212; see also W. Gilbert and C. Guthrie, 2004, Mol. Cell 13:201-212.

This phosphorylation causes it to dissociate from the mRNP, along with the mRNP exporter. In this way, dephosphorylation of mRNP adapter proteins in the nucleus once RNA processing is complete and their phosphorylation and resulting dissociation in the cytoplasm result in a higher concentration of mRNP exporter-mRNP complexes in the nucleus, where they form, and a lower concentration of these complexes in the cytoplasm, where they dissociate. As a result, the direction of mRNP export may be driven by simple diffusion down a concentration gradient of the mRNP exportermRNP complex across the NPC, from high in the nucleus to low in the cytoplasm.

## Balbiani Rings in Insect Larval Salivary Glands Allow Direct Visualization of mRNP Export Through NPCs

The larval salivary glands of the insect Chironomus tentans provide a good model system for electron microscopic studies of the formation of hnRNPs and their export through NPCs. In these larvae, genes in large chromosomal puffs called Balbiani rings are abundantly transcribed into nascent pre-mRNAs that associate with hnRNP proteins and are processed into coiled mRNPs with a final mRNA length of about 75 kb (Figure 10-25a, b). These giant mRNAs encode large

glue proteins that adhere the developing larva to a leaf. After processing of the pre-mRNA in Balbiani ring hnRNPs, the resulting mRNPs move through NPCs to the cytoplasm. Electron micrographs of sections of these cells show mRNPs that appear to uncoil during their passage through NPCs and then bind to ribosomes as they enter the cytoplasm. This uncoiling is probably a consequence of the remodeling of mRNPs as the result of phosphorylation of mRNP proteins by cytoplasmic kinases and the action of the RNA helicase associated with NPC cytoplasmic filaments, as discussed in the previous section. The observation that mRNPs become associated with ribosomes during transport indicates that the $5^{\prime}$ end leads the way through the NPC. Detailed electron microscopic studies of the transport of Balbiani ring mRNPs through nuclear pore complexes led to the model depicted in Figure 10-25c.

## Pre-mRNAs in Spliceosomes Are Not Exported from the Nucleus

It is critical that only fully processed mature mRNAs be exported from the nucleus because translation of incompletely processed pre-mRNAs containing introns would produce defective proteins that might interfere with the functioning of the cell. To prevent this, pre-mRNAs associated with
snRNPs in spliceosomes are usually prevented from being transported to the cytoplasm.

In one type of experiment demonstrating this restriction, a gene encoding a pre-mRNA with a single intron that is normally spliced out was mutated to introduce deviations from the consensus splice-site sequences. Mutation of either the $5^{\prime}$ or the $3^{\prime}$ invariant splice-site bases at the ends of the intron resulted in pre-mRNAs that were bound by snRNPs to form spliceosomes; however, RNA splicing was blocked, and the pre-mRNA was retained in the nucleus. In contrast, mutation of both the $5^{\prime}$ and $3^{\prime}$ splice sites in the same pre-mRNA resulted in export of the unspliced pre-mRNA, although less efficiently than for the spliced mRNA, probably because of the absence of an exon-junction complex. When both splice sites were mutated, the pre-mRNAs were not efficiently bound by snRNPs, and consequently, their export was not blocked.

Studies in yeast have shown that a protein component of the NPC nuclear basket is required to retain pre-mRNAs associated with snRNPs in the nucleus. If either this protein or the nuclear basket protein to which it binds is deleted, unspliced pre-mRNAs are exported. Consequently, these proteins prevent hnRNPs associated with snRNPs from traversing the NPC.
![li3112239539b](li3112239539b.jpg)

FIGURE 10-25 Formation of heterogeneous ribonucleoprotein particles (hnRNPs) and export of mRNPs from the nucleus.
(a) Model of a single chromatin transcription loop and assembly of Balbiani ring (BR) mRNP in Chironomus tentans. Nascent RNA transcripts produced from the template DNA rapidly associate with proteins, forming hnRNPs. The gradual increase in the size of the hnRNPs reflects the increasing length of RNA transcripts at greater distances from the transcription start site. The model was reconstructed from electron micrographs of serial thin sections of salivary gland cells. (b) Schematic diagram of the biogenesis of hnRNPs. Following processing of the pre-mRNA, the resulting ribonucleoprotein particle is referred to as an mRNP. (c) Model for the transport of BR mRNPs through the nuclear pore complex (NPC) based on electron microscopic studies. Note that the curved mRNPs appear to uncoil as they pass through NPCs. As the mRNA enters the cytoplasm, it rapidly associates with ribosomes, indicating that the $5^{\prime}$ end passes through the NPC first. Parts (b) and (c), see B. Daneholt, 1997, Cell 88:585. See also B. Daneholt, 2001, Proc. Natl. Acad. Sci. USA 98:7012. [Part (a) republished with permission from Elsevier, from Erricson, C. et al., "The ultrastructure of upstream and downstream regions of an active Balbiani ring gene," Cell, 1989, 56(4): 631-9; courtesy of B. Daneholt. Permission conveyed through the Copyright Clearance Center, Inc.]
![li3122239539b](li3122239539b.jpg)

5 Many cases of thalassemia, an inherited disease that results in abnormally low levels of globin proteins, are due to mutations in globin-gene splice sites that decrease the efficiency of splicing but do not prevent association of the pre-mRNA with snRNPs. The resulting unspliced globin pre-mRNAs are retained in the nuclei of erythroid progenitors (see Figure 16-7) and are rapidly degraded.

## HIV Rev Protein Regulates the Transport of Unspliced Viral mRNAs

As discussed earlier, transport of mRNPs containing mature, functional mRNAs through NPCs from the nucleus to the cytoplasm entails a complex mechanism that is crucial to gene expression (see Figures 10-23, 10-24, and 10-25). Regulation of this transport theoretically could provide another means of gene control, although it appears to be relatively rare. Indeed, the only known examples of regulated mRNA export occur during the cellular response to conditions (e.g., heat shock) that cause protein denaturation or during viral infection, when virus-induced alterations in nuclear export of mRNPs maximize viral replication. Here we describe the regulation of mRNP export mediated by a protein encoded by human immunodeficiency virus (HIV).

HIV, which is a retrovirus, integrates a DNA copy of its RNA genome into the host-cell DNA (see Figure 5-48). The integrated viral DNA, or provirus, contains a single transcription unit, which is transcribed into a single primary transcript by cellular RNA polymerase II. The HIV transcript can be spliced in alternative ways to yield three classes of
mRNAs: a $9-\mathrm{kb}$ unspliced mRNA; $4-\mathrm{kb}$ mRNAs formed by removal of one intron; and $2-\mathrm{kb}$ mRNAs formed by removal of two or more introns (Figure 10-26). After their synthesis in the host-cell nucleus, all three classes of HIV mRNAs are transported to the cytoplasm and translated into viral proteins; some of the $9-\mathrm{kb}$ unspliced RNA is used as the viral genome in progeny virions that bud from the cell surface.

Since the $9-\mathrm{kb}$ and $4-\mathrm{kb}$ HIV mRNAs contain splice sites, they can be viewed as incompletely spliced mRNAs. As discussed earlier, association of such incompletely spliced mRNAs with snRNPs in spliceosomes normally blocks their export from the nucleus. Thus HIV, as well as other retroviruses, must have some mechanism for overcoming this block, permitting export of the longer viral mRNAs. Some retroviruses have evolved an RNA sequence within their genome called the constitutive transport element (CTE), which binds to the NXF1/NXT1 mRNP exporter with high affinity. This strong interaction with the mRNP exporter allows export of unspliced retroviral RNA into the cytoplasm. HIV solved the problem differently.

Studies with HIV mutants showed that transport of unspliced $9-\mathrm{kb}$ and singly spliced $4-\mathrm{kb}$ viral mRNAs from the nucleus to the cytoplasm requires the virus-encoded Rev protein. Subsequent biochemical experiments demonstrated that Rev binds to a specific Rev-response element (RRE) that is present in HIV RNA. In cells infected with HIV mutants lacking the RRE, unspliced and singly spliced viral mRNAs remain in the nucleus, demonstrating that the RRE is required for Rev-mediated stimulation of nuclear export. Early in an infection, before any Rev protein is synthesized, only multiply spliced $2-\mathrm{kb}$ mRNAs that do not retain any splice
![li3132239539b](li3132239539b.jpg)

FIGURE 10-26 Transport of HIV mRNAs from the nucleus to the cytoplasm. The HIV genome, which contains several coding regions, is transcribed into a single $9-\mathrm{kb}$ primary transcript. Several $4-\mathrm{kb}$ mRNAs result from the splicing out of any one of several introns (dashed lines), and several $2-\mathrm{kb}$ mRNAs result from the splicing out of two or more alternative
introns. After transport to the cytoplasm, these various RNA species are translated into different viral proteins. Rev protein, encoded by a $2-\mathrm{kb}$ mRNA, interacts with the Rev-response element (RRE) in the unspliced and singly spliced mRNAs, stimulating their transport to the cytoplasm. See B. R. Cullen and M. H. Malim, 1991, Trends Biochem. Sci. 16:346.

sites can be exported. One of these alternatively spliced 2-kb mRNAs encodes Rev, which contains a leucine-rich nuclearexport signal that interacts with the transporter exportin 1 (see Chapter 13) rather than with the NXF1/NXT1 mRNP exporter. Translation of Rev in the cytoplasm, followed by its import into the nucleus, results in export of the larger unspliced and singly spliced HIV mRNAs through the NPC.

## KEY CONCEPTS OF SECTION 10.3

## Transport of mRNA Across the Nuclear Envelope

- Most mRNPs are exported from the nucleus by a heterodimeric mRNP exporter that interacts with unstructured protein domains that fill the central channel of the nuclear pore complex (NPC). The direction of transport (nucleus to cytoplasm) results from dissociation of the mRNP exporter-mRNP complex in the cytoplasm due to the phosphorylation of mRNP adapter proteins by cytoplasmic kinases and the action of an RNA helicase associated with cytoplasmic filaments of the nuclear pore complexes. As a result, mRNP exporter-mRNP complexes diffuse down a concentration gradient across the NPC from the nucleus to the cytoplasm.
- The mRNP exporter binds to most mRNAs cooperatively with SR proteins bound to exonic splicing enhancers and with REF associated with exon-junction complexes as well as with additional mRNP proteins.
- Pre-mRNAs bound by a spliceosome normally are not exported from the nucleus, ensuring that only fully processed, functional mRNAs reach the cytoplasm for translation.


### 10.4 Cytoplasmic Mechanisms of Post-transcriptional Control

Before proceeding, let's quickly review the steps in gene expression at which control is exerted. We saw in Chapter 9 that regulation of transcription initiation and transcription elongation in the promoter-proximal region are the initial mechanisms for controlling the expression of genes in the DNA $\rightarrow$ RNA $\rightarrow$ protein pathway. In the preceding sections of this chapter, we learned that the expression of protein isoforms is controlled by the regulation of alternative RNA splicing and of cleavage and polyadenylation at alternative poly(A) sites. Although nuclear export of fully and correctly processed mRNPs to the cytoplasm is rarely regulated, the export of improperly processed or aberrantly remodeled pre-mRNPs is prevented, and such abnormal transcripts are degraded by exosomes. However, retroviruses, including HIV, have evolved mechanisms that permit pre-mRNAs that retain splice sites to be exported and translated.

In this section, we consider other mechanisms of posttranscriptional control that contribute to regulating the
expression of many genes. Most of these mechanisms operate in the cytoplasm, controlling the stability or localization of mRNA or its translation into protein. The concentration of an mRNA in the cytoplasm is determined by its rate of synthesis and its rate of degradation. The most stable mRNAs, which encode proteins required in large amounts (such as the ribosomal proteins), can accumulate to very high copy numbers per cell. In contrast, highly unstable mRNAs, which encode proteins expressed in short bursts (such as cytokines, secreted proteins that regulate the immune response), rarely achieve such high concentrations even when transcribed, processed, and exported from the nucleus at high rates. We begin by discussing the major pathways that degrade mRNAs.

Next we discuss two related mechanisms of gene control that provide powerful new techniques for manipulating the expression of specific genes for experimental and therapeutic purposes. These mechanisms are controlled by short ( 22-nucleotide) single-stranded RNAs called micro-RNAs (miRNAs) and short interfering RNAs (siRNAs). Both base-pair with specific target mRNAs, causing their rapid degradation (siRNAs) or inhibiting their translation and inducing a slower form of degradation (miRNAs). Many miRNAs can target more than one mRNA. Consequently, these mechanisms contribute significantly to the regulation of gene expression. Short interfering RNAs, involved in a process called RNA interference, are an important cellular defense against viral infection and excessive transposition by retrotransposons. We also discuss mechanisms that control the overall rate of protein synthesis, as well as highly specific mechanisms that regulate the translation and stability of particular mRNAs. Finally, we discuss mechanisms that control the localization of mRNAs in the cytoplasm of asymmetric cells so that the encoded protein is translated at sites in the cell where it is needed.

## Degradation of mRNAs in the Cytoplasm Occurs by Several Mechanisms

As mentioned above, the concentration of an mRNA is a function of both its rate of synthesis and its rate of degradation. For this reason, if two genes are transcribed at the same rate, the steady-state concentration of the corresponding mRNA that is more stable will be higher than the concentration of the other. The stability of an mRNA also determines how rapidly synthesis of the encoded protein can be shut down. For a stable mRNA, synthesis of the encoded protein persists long after transcription of the gene is repressed. Most bacterial mRNAs are unstable, decaying exponentially with a typical half-life of a few minutes. For this reason, a bacterial cell can rapidly adjust the synthesis of proteins to accommodate changes in the cellular environment. Most cells in multicellular organisms, on the other hand, exist in a fairly constant environment and carry out a specific set of functions over days to months or even the lifetime of the organism (neurons, for example). Accordingly, most mRNAs of higher eukaryotes have half-lives of many hours.

However, some proteins in eukaryotic cells are required only for short periods and must be expressed in bursts. For example, as discussed above, certain signaling molecules called cytokines, which are involved in regulating the immune response of mammals, are synthesized and secreted in short bursts (see Chapter 23). Similarly, many of the transcription factors that regulate the onset of the $S$ phase of the cell cycle, such as Fos and Jun, are synthesized only for brief periods (see Chapter 19). The expression of such proteins occurs in short bursts because transcription of their genes can be rapidly turned on and off, and their mRNAs have unusually short half-lives, on the order of 30 minutes or less.

Cytoplasmic mRNAs are degraded by one of the three pathways shown in Figure 10-27. For most mRNAs, the deadenylation-dependent pathway is followed: the length of the poly $(\mathrm{A})$ tail gradually decreases with time through the action of a deadenylating nuclease complex. When the tail has been shortened sufficiently, PABPC1 molecules can no longer bind to it and stabilize the interaction of the $5^{\prime}$ cap and translation initiation factors (see Figure 5-23, which summarizes the steps of translation initiation). The exposed cap is then removed by a decapping enzyme (DCP1/DCP2), leaving the unprotected mRNA susceptible to degradation
(a) Deadenylation-dependent mRNA decay
![li3142239539b](li3142239539b.jpg)

FIGURE 10-27 Pathways for degradation of eukaryotic mRNAs. (a) In the most common pathway of mRNA degradation, the deadenylation-dependent pathway, the poly(A) tail is progressively shortened by a deadenylase complex until it reaches a length of 20 or fewer A residues, at which point the interaction between PABPC1 and the remaining poly(A) is destabilized, leading to weakened interactions between the $5^{\prime}$ cap and translation initiation factors (see Figure 5-23). The deadenylated mRNA then may either (1) be decapped by the DCP1/DCP2 deadenylation complex and degraded by XRN1, a $5^{\prime} \rightarrow 3^{\prime}$ exonuclease, or (2) be degraded by XRN1, a $5^{\prime} \rightarrow 3^{\prime}$ exonibonuclease. Removal of the poly(A) tail also makes mRNAs susceptible to degradation by cytoplasmic exosomes containing $3^{\prime} \rightarrow 5^{\prime}$ exonucleases. The $5^{\prime} \rightarrow 3^{\prime}$ exonuclease pathway predominates in yeast, and the $3^{\prime} \rightarrow 5^{\prime}$ exosome pathway predominates in mammalian cells. The decapping enzymes and $5^{\prime} \rightarrow 3^{\prime}$ exonuclease are concentrated in P bodies (processing bodies, described below), regions of the cytoplasm with unusually high concentrations of RNPs.

Some mRNAs are degraded primarily by a deadenylationindependent decapping pathway (Figure 10-27b). Certain sequences at the $5^{\prime}$ end of an mRNA make the cap sensitive to the decapping enzyme. For these mRNAs, the rate at which they are decapped controls the rate at which they are degraded because once the $5^{\prime}$ cap is removed, the RNA is rapidly hydrolyzed by the $5^{\prime} \rightarrow 3^{\prime}$ exoribonuclease XRN1.

Other mRNAs are degraded by an endonucleolytic pathway that does not involve decapping or significant deadenylation (Figure 10-27c). One example of this type of pathway is the RNA interference pathway discussed below. Each siRNA-RISC complex can degrade thousands of targeted RNA molecules. The fragments generated by internal cleavage are then degraded by exonucleases.
(b) Deadenylation-independent mRNA decay
![li3152239539b](li3152239539b.jpg)
(c) Endonuclease-mediated mRNA decay
![li3162239539b](li3162239539b.jpg)
$3^{\prime} \rightarrow 5^{\prime}$ exonucleases in cytoplasmic exosomes. (b) Other mRNAs are decapped before they are deadenylated and then degraded by the XRN1 $5^{\prime} \rightarrow 3^{\prime}$ exonuclease. In the example shown from yeast, an RNAbinding protein Rps28B binds a sequence in the $3^{\prime}$-UTR of its own mRNA, which then interacts with Edc3 (enhancer of decapping 3). Edc3 then recruits the DCP1/2 decapping enzyme to the mRNA, auto regulating expression of Rps28B. (c) Some mRNAs are cleaved internally by an endonuclease and the fragments degraded by a cytoplasmic exosome and the XRN1 exonuclease. See N. L. Garneau, J. Wilusz, and C. J. Wilusz, 2007, Nat. Rev. Mol. Cell Biol. 8:113.

The rate of mRNA deadenylation varies inversely with the frequency of translation initiation for an mRNA: the higher the frequency of initiation, the slower the rate of deadenylation. This relationship is probably due to the reciprocal interactions between translation initiation factors bound at the $5^{\prime}$ cap and PABPC1 bound to the poly(A) tail. For an mRNA that is translated at a high rate, initiation factors are bound to the cap much of the time, stabilizing the binding of PABPC1 and thereby protecting the poly(A) tail from deadenylating nuclease complexes.

Many short-lived mRNAs in mammalian cells-those encoding proteins such as cytokines and transcription factors whose concentrations must change rapidly-contain multiple, sometimes overlapping copies of the sequence AUUUA in their $3^{\prime}$ untranslated region. These sequences are known as $A U$-rich elements. Specific RNA-binding proteins have been found that bind to these $3^{\prime} \mathrm{AU}$-rich sequences and also interact with a deadenylating enzyme and with the exosome, causing rapid deadenylation and subsequent $3^{\prime} \rightarrow 5^{\prime}$ degradation of these mRNAs. This mechanism uncouples the rate of mRNA degradation from the frequency of translation. Thus mRNAs containing AU-rich elements can be translated at high frequency yet can also be degraded rapidly, allowing the encoded proteins to be expressed in short bursts.

P bodies are dense cytoplasmic domains many times the size of a ribosome. They are sites of translational repression that contain no ribosomes or translation factors. They are also major sites of mRNA degradation in the cytoplasm. These dense regions of cytoplasm contain the decapping enzyme (DCP1/DCP2), activators of decapping (DHH, PAT1, LSM1-7), and the major $5^{\prime} \rightarrow 3^{\prime}$ exoribonuclease XRN1, as well as densely associated mRNAs. P bodies are dynamic structures that grow and shrink in size depending on the rate at which mRNPs associate with them, the rate at which mRNAs are degraded, and the rate at which mRNPs exit P bodies and reenter the pool of translated mRNPs. Those mRNAs whose translation is inhibited by imperfect base pairing of miRNAs are major components of P bodies, as we will see shortly.

## Adenines in mRNAs and IncRNAs May Be Posttranscriptionally Modified by $\mathrm{N}^{6}$ Methylation

Like DNA, which can be modified after synthesis by C-methylation (which generally leads to transcriptional repression through methyl CpG-binding proteins; see page 404), pre-mRNAs, mRNAs, and lncRNAs can undergo base modifications following their transcription. The functions of the most frequent post-transcriptional base modification of mRNA, methylation of the $\mathrm{N}^{6}$ position of adenine ( $\mathrm{m}^{6} \mathrm{~A}$ ), are currently intense areas of investigation. In mammalian cells, about one in every 2000 bases in mRNAs and long noncoding RNAs (lncRNAs) are $\mathrm{m}^{6} \mathrm{As}$, amounting to $3-5 \mathrm{~m}^{6} \mathrm{As}$ per mRNA, on average. Sites of $\mathrm{m}^{6} \mathrm{~A}$ are found in all rRNAs, in all snRNAs, and in the T $\Psi$ CG loop of all tRNAs (see Figure 5-20). In contrast, among mRNAs and lncRNAs, only a fraction of all molecules contain $\mathrm{m}^{6} \mathrm{~A}$, ranging from 10 to

70 percent for the few mRNAs and lncRNAs thus far analyzed. But $\mathrm{m}^{6} \mathrm{~A}$ has been detected in transcripts of over 7000 human protein-coding genes and some 300 lncRNAs. It has also been detected in introns, indicating that it can be added to pre-mRNAs co-transcriptionally. In mRNAs, a high percentage of $\mathrm{m}^{6} \mathrm{As}$ are located near stop codons, in $3^{\prime}$ untranslated regions, and in unusually long internal exons.

As for DNA, specific enzymes add methyl groups from S-adenosylmethionine (a common donor of methyl groups in many biochemical reactions) to specific sites, and enzymes have been identified that can remove RNA methyl groups. These observations raise the possibility that $\mathrm{m}^{6} \mathrm{~A}$ modification of a particular RNA molecule may be dynamically regulated. However, these enzymes are primarily nuclear, so once an mRNA is modified with $\mathrm{m}^{6} \mathrm{~A}$, it is probably not demethylated in the cytoplasm. Importantly, proteins have been identified that bind $\mathrm{m}^{6} \mathrm{~A}$-modified RNAs preferentially over RNAs lacking $\mathrm{m}^{6} \mathrm{~A}$. By analogy with DNA C-methylation, this class of proteins may carry out the function(s) of $\mathrm{m}^{6} \mathrm{~A}$ modification.

Recent research indicates that $\mathrm{m}^{6} \mathrm{~A}$ may affect many aspects of the "life cycle" of specific mRNAs, including RNA splicing, nuclear export, translation, and degradation. In Drosophila and the plant A. thaliana, there is a single mRNA $\mathrm{m}^{6} \mathrm{~A}$ methyl transferase, and knockouts of these genes are embryonic lethal in both organisms, attesting to the functional importance of $\mathrm{m}^{6} \mathrm{~A}$ modification. RNA molecules containing $\mathrm{m}^{6} \mathrm{~A}$ are less stable than the same unmethylated RNAs. In this regard, $\mathrm{m}^{6} \mathrm{~A}$-binding proteins have been reported to induce association of $\mathrm{m}^{6} \mathrm{~A}$-containing mRNAs with P bodies, potentially accounting for how this base modification affects mRNA translation and stability. In addition to $\mathrm{m}^{6} \mathrm{~A}$, more than a hundred other modifications of the four bases have been characterized in RNAs. Obviously, much remains to be learned about the functions of these base modifications.

## Micro-RNAs Repress Translation and Induce Degradation of Specific mRNAs

Micro-RNAs (miRNAs) were first discovered during analysis of mutations in the lin-4 and let-7 genes of the nematode C. elegans, which influence the development of that organism. Cloning and analysis of wild-type lin-4 and let-7 revealed that they do not encode protein products, but rather RNAs only 21 and 22 nucleotides long, respectively. These RNAs hybridize to the $3^{\prime}$ untranslated regions ( $3^{\prime}$ UTRs) of specific target mRNAs. For example, the lin-4 miRNA, which is expressed early in embryogenesis, hybridizes to the $3^{\prime}$ UTRs of both the lin-14 and lin-28 mRNAs in the cytoplasm, thereby repressing their translation. Expression of lin-4 miRNA ceases later in development, allowing the translation of newly synthesized lin-14 and lin-28 mRNAs at that time. Expression of let-7 miRNA occurs at comparable times during embryogenesis in all bilaterally symmetric animals.

Regulation of translation by miRNAs appears to be widespread in all multicellular plants and animals. In the

past few years, small RNAs of 20-26 nucleotides have been isolated, cloned, and sequenced from various tissues of multiple model organisms. Recent estimates suggest that the expression of one-third of all human genes is regulated by the roughly 1900 human miRNAs isolated from various tissues. The potential for regulation of multiple mRNAs by one miRNA is great because base pairing between the miRNAs and the $3^{\prime}$ ends of the mRNAs that they regulate need not be perfect (Figure 10-28). In fact, considerable experimentation with synthetic miRNAs has shown that complementarity between bases $2-7$ at the $5^{\prime}$ end of an miRNA (called the "seed" sequence) and its target-mRNA 3' UTR is most critical for target-mRNA selection.

Most miRNAs are processed from RNA polymerase II transcripts that are several hundred to thousands of nucleotides in length, called pri-miRNAs (for primary transcript) (Figure 10-29). A pri-miRNA can contain the sequence of one or more miRNAs. Some miRNAs are also processed from excised introns and from $3^{\prime}$ UTRs of some pre-mRNAs. Within these long transcripts are sequences that fold into hairpin structures about 70 nucleotides in length with imperfect base pairing in the stem. A nuclear RNase specific for double-stranded RNA, called Drosha, acts with a nuclear double-stranded RNA-binding protein, called DGCR8 (DiGeorge syndrome chromosomal region 8, named for its association with this genetic disease) in humans (Pasha in Drosophila) to cleave the hairpin region out of the long precursor RNA, generating a pre-miRNA. Pre-miRNAs are recognized and bound by a specific nuclear export factor, exportin 5, which allows them to diffuse through the inner channel of the nuclear pore complex. Once it reaches the cytoplasm, a cytoplasmic double-stranded RNA-specific

RNase, called Dicer, acts with a cytoplasmic doublestranded RNA-binding protein, called TRBP in humans (for Tar binding protein; called Loquacious in Drosophila), to further process the pre-miRNA into a double-stranded miRNA. The double-stranded miRNA is approximately two turns of an A-form RNA helix in length, with strands 21-23 nucleotides long and two unpaired $3^{\prime}$ nucleotides at each end. Finally, one of the two strands is selected for assembly into a mature RNA-induced silencing complex (RISC), which contains a single-stranded mature miRNA bound by a multidomain Argonaute protein, a member of a protein family with a recognizable conserved sequence, as well as additional proteins. Several Argonaute proteins are expressed in some organisms, especially plants, and are found in distinct RISC complexes with different functions. Humans express four Argonaute proteins. AGO2 is the human Argonaute protein in miRNA-containing RISC complexes. The other human Argonaute proteins have partially overlapping functions because knockout of all four human Argonaute proteins is lethal to human embryonic stem cells, but any one of the four is sufficient for viability. The specific functions of the other Argonaute proteins during mouse development are currently under study.

The miRNA-RISC complexes associate with target mRNPs by base pairing between the Argonaute-bound mature miRNA and complementary regions in the $3^{\prime}$ UTRs of target mRNAs (see Figure 10-28). Inhibition of targetmRNA translation requires the binding of two or more RISC complexes to distinct complementary regions in the target-mRNA 3' UTR. Generally the more RISC complexes bound to the $3^{\prime}$ UTR of an mRNA, the greater the repression of translation. This mechanism allows combinatorial
(a) miRNA $\rightarrow$ translation inhibition
![li3172239539b](li3172239539b.jpg)

Target RNA
![li3182239539b](li3182239539b.jpg)

5'-UAGGUAGUUUCAUGUUGUUGGG-3' 3' -CUUAUCCGUCAAAGUACAACAACCUUCU-5' miR-196a and HOXB8 mRNA (H. sapiens)
![li3192239539b](li3192239539b.jpg)
miR-166 and PHAVOLUTA mRNA (A. thaliana)

FIGURE 10-28 Base pairing with target RNAs distinguishes miRNA and siRNA. (a) miRNAs hybridize imperfectly with their target mRNA, repressing translation of the mRNA. Nucleotides 2-7 of an miRNA (highlighted blue) are the most critical for targeting it to a specific mRNA. The CXCR4 miRNA shown at the bottom is a synthetic
oligonucleotide introduced into cells by transfection. (b) siRNA hybridizes perfectly with its target mRNA, causing cleavage of the mRNA at the position indicated by the red arrow, triggering its rapid degradation. See P. D. Zamore and B. Haley, 2005, Science 309:1519.

![li3202239539b](li3202239539b.jpg)

FIGURE 10-29 Processing of miRNA. This diagram shows transcription and processing of the miR-1-1 miRNA. The primary miRNA transcript (pri-miRNA) is transcribed by RNA polymerase II. The nuclear double-stranded RNA-specific endoribonuclease Drosha, with its partner, double-stranded RNA-binding protein DGCR8 (Pasha in Drosophila), makes the initial cleavages in the pri-miRNA, generating a $\sim 70$-nucleotide pre-miRNA that is exported to the cytoplasm by nuclear transporter exportin 5. The pre-miRNA is further processed in the cytoplasm by Dicer, in conjunction with the double-stranded RNA-binding protein TRBP (Loquacious in Drosophila), into a doublestranded miRNA with two-base single-stranded 3' ends. Finally, one of the two strands is incorporated into a RISC complex, where it is bound by an Argonaute protein. See P. D. Zamore and B. Haley, 2005, Science 309:1519.
regulation of mRNA translation by separately regulating the transcription of two or more different pri-miRNAs, which are processed into miRNAs required in combination to suppress the translation of a specific target mRNA.

The mechanism by which the binding of several RISC complexes to an mRNA inhibits translation initiation is currently being analyzed. Binding of RISC complexes causes the bound mRNPs to associate with P bodies. Since P bodies are major sites of mRNA degradation where the decapping complex DPC1/DPC2, the $5^{\prime} \rightarrow 3^{\prime}$ exonuclease XRN1, and cytoplasmic exosomes are concentrated, mRNAs bound by several RISC complexes are degraded.

As mentioned earlier, approximately 1900 different human miRNAs have been observed, most of which are expressed only in specific cell types at particular times during embryogenesis and after birth. Determining the function of these miRNAs is currently a highly active area of research. In one example, a specific miRNA, called miR-133, is induced when myoblasts differentiate into muscle cells. This miRNA suppresses the translation of PTB, a regulatory splicing factor that functions similarly to Sxl in Drosophila (see Figure 10-18). PTB binds to $3^{\prime}$ splice sites in the pre-mRNAs of many genes, leading to exon skipping or use of alternative $3^{\prime}$ splice sites. When miR-133 is expressed in differentiating myoblasts, the PTB concentration falls. As a result, alternative isoforms of multiple proteins important for muscle-cell function are expressed in the differentiated cells.

Other examples of miRNA regulation are being discovered at a rapid pace in various organisms. Knocking out the dicer gene eliminates the generation of miRNAs in mammals. This manipulation causes embryonic death early in mouse development. When dicer is knocked out only in limb primordia, however, the influence of miRNAs on the development of the nonessential limbs can be observed (Figure 10-30). Although all major cell types in the limb differentiate and the fundamental aspects of limb patterning are maintained, development is abnormal—demonstrating
![li3212239539b](li3212239539b.jpg)

EXPERIMENTAL FIGURE 10-30 The function of miRNAs in limb development. Micrographs comparing normal (left) and Dicerknockout (right) limbs of 13-day mouse embryos immunostained for the GdS protein, a marker of joint formation. Dicer is knocked out in the limbs of developing mouse embryos by conditional expression of Cre to induce deletion of the dicer gene in only those cells (see Figure 6-40). [From Harfe, B.D., et al., "The RNaseIII enzyme Dicer is required for morphogenesis but not patterning of the vertebrate limb," Proc. Natl. Acad. Sci. USA, 2005, 102(31):10898-903. Copyright (2005) National Academy of Sciences, USA.]

the importance of miRNAs in regulating the proper level of translation of multiple mRNAs. In effect, miRNAs "finetune" gene expression to the appropriate level for gene function in various cell types. Of the 1900 human miRNAs, 53 appear to be unique to primates. It seems likely that new miRNAs have arisen readily during evolution by the duplication of a pri-miRNA gene followed by mutation of bases encoding the mature miRNA. miRNAs are particularly abundant in plants-more than 1.5 million distinct miRNAs have been characterized in Arabidopsis thaliana!

## Alternative Polyadenylation Increases miRNA Control Options

In addition to alternative splicing, which occurs in one tissue or another for some 95 percent of human genes, alternative polyadenylation occurs for some 50 percent of human mRNAs. Alternative polyadenylation results from the use of two or more alternative polyadenylation signals in different cell types. In some cases, this appears to be due to different concentrations of cleavage/polyadenylation factors in different cell types coupled with alternative poly(A) signals that have higher or lower affinity for the CStF complex that binds the downstream G/U-rich portion of the cleavage/polyadenylation signal (see Figure 10-15). In these cases, when the concentration of CStF is low, only the highest-affinity polyadenylation signals are used. But in alternative cell types where the CStF concentration is higher, an upstream low-affinity site is used preferentially because once the premRNA is cleaved, the downstream site cannot be used. In other cases, sequence-specific RNA-binding proteins may block or enhance binding of the cleavage/polyadenylation factors, as in the case of splicing repressors and activators.

When multiple mRNAs expressed from the same gene use alternative polyadenylation sites, additional miRNAbinding sites may be located in the mRNA with the longer $3^{\prime}$ exon. As a consequence, mRNAs with the same proteincoding sequence may be regulated differently in different cell types depending on the length of the $3^{\prime}$ UTR and the miRNAs expressed in those cells. Consequently, alternative polyadenylation can regulate the translation of mRNAs encoding the same protein as a consequence of miRNA control of translation and mRNA stability.

Alternative sites of polyadenylation can also be coupled to alternative splicing of the final exon in an mRNA. As a consequence, protein isoforms can be expressed that have different C-terminal amino acid sequences. This type of variation is observed in the expression of alternative immunoglobulin molecules during B-lymphocyte development (see Figure 23-19). Initially, an immunoglobulin antibody is produced with a transmembrane domain, which anchors the antibody in the plasma membrane, and a cytoplasmic domain, which signals when the antigen-binding extracellular domain encounters antigen-the molecule bound by an antibody. When antigen is bound, processing of the premRNA is modified so that an alternative $3^{\prime}$ exon is included in the mRNA. The antibody molecules translated from this
alternatively processed mRNA lack the transmembrane domain, and as a consequence, they are secreted into the extracellular space, where they can neutralize pathogens (see Chapters 14 and 23).

## RNA Interference Induces Degradation of Precisely Complementary mRNAs

RNA interference (RNAi) was discovered unexpectedly during attempts to experimentally manipulate the expression of specific genes. Researchers tried to inhibit the expression of a gene in C. elegans by microinjecting a single-stranded, complementary RNA that would hybridize to the encoded mRNA and prevent its translation, a method called antisense inhibition. But in control experiments, a perfectly base-paired double-stranded RNA a few hundred base pairs long was much more effective at inhibiting expression of the gene than the antisense strand alone (see Figure 6-42). Similar inhibition of gene expression by introduced double-stranded RNA was soon observed in plants. In each case, the double-stranded RNA induced degradation of all cellular RNAs containing a sequence that was exactly the same as that of one strand of the double-stranded RNA. Because of the specificity of this technique in targeting mRNAs for destruction, it has become a powerful experimental tool for studying gene function.

Subsequent biochemical studies with extracts of Drosophila embryos showed that a long double-stranded RNA that mediates RNA interference is initially processed into a double-stranded short interfering RNA (siRNA). The strands in siRNAs contain 21-23 nucleotides hybridized to each other so that the two bases at the $3^{\prime}$ end of each strand are unpaired. Further studies revealed that the cytoplasmic double-stranded RNA-specific ribonuclease that cleaves long double-stranded RNA into siRNAs is the same Dicer enzyme involved in processing pre-miRNAs after their export to the cytoplasm (see Figure 10-29). This discovery led to the realization that RNA interference and miRNAmediated repression of translation and target-mRNA degradation are related processes. In both cases, the mature short single-stranded RNAs, either siRNAs or miRNAs, are assembled into RISC complexes in which they are bound by an Argonaute protein. What distinguishes a RISC complex containing an siRNA from one containing an miRNA is that the siRNA base-pairs perfectly with its target RNA and induces its cleavage, whereas a RISC complex associated with an miRNA recognizes its target through imperfect base pairing and results in inhibition of translation and a slower form of target-mRNA degradation (see Figure 10-28).

AGO2 is the protein responsible for the cleavage of target RNA. One domain of the protein is homologous to the RNase H enzymes that degrade the RNA of an RNA-DNA hybrid (see Figure 8-14). When the $5^{\prime}$ end of the siRNA of a RISC complex base-pairs precisely with a target mRNA over a distance of one turn of an RNA helix (10-12 base pairs), this domain of AGO2 cleaves the phosphodiester bond of the target RNA across from nucleotides 10 and 11 of the siRNA (see Figure 10-28b). The cleaved RNAs are released

and subsequently degraded by cytoplasmic exosomes and the XRN1 $5^{\prime} \rightarrow 3^{\prime}$ exoribonuclease. If base pairing is not perfect, the AGO2 domain does not cleave or release the target mRNA. Instead, if several miRNA-RISC complexes associate with a target mRNA, its translation is inhibited, and the mRNA becomes associated with P bodies, where, as mentioned earlier, it is degraded by a different and slower mechanism than the degradation pathway initiated by RISC cleavage of a perfectly complementary target RNA.

When double-stranded RNA is introduced into the cytoplasm of eukaryotic cells, it enters the pathway for the assembly of siRNAs into a RISC complex because it is recognized by Dicer and TRBP (see Figure 10-29). This process of RNA interference is believed to be an ancient cellular defense against certain viruses and mobile genetic elements in both plants and animals. Plants with mutations in the genes encoding the Dicer and RISC proteins exhibit increased sensitivity to infection by RNA viruses and increased movement of transposons within their genomes. The double-stranded RNA intermediates generated during replication of RNA viruses are thought to be recognized by Dicer, inducing an RNAi response that ultimately degrades the viral mRNAs. During transposition, transposons are inserted into cellular genes in a random orientation, and their transcription from different promoters produces complementary RNAs that can hybridize with each other, initiating the RNAi system, which then interferes with the expression of transposon proteins required for additional transpositions.

In plants and C. elegans, RNA interference can be induced in all cells of the organism by introduction of double-stranded RNA into just a few cells. Such organism-wide induction requires production of a protein that is homologous to the RNA replicates of RNA viruses. It has been revealed that doublestranded siRNAs are replicated and then transferred to other cells in these organisms. In plants, the transfer of siRNAs might occur through plasmodesmata, the cytoplasmic connections between plant cells that traverse the cell walls between them (see Figure 20-42). Organism-wide induction of RNA interference does not occur in Drosophila or mammals, presumably because their genomes do not encode RNA replicase homologs.

In mammalian cells, the introduction of long RNARNA duplex molecules into the cytoplasm results in generalized inhibition of protein synthesis via the PKR pathway, discussed further below. This response greatly limits the use of long double-stranded RNAs to experimentally induce RNA interference against a specific targeted mRNA. Fortunately, researchers discovered that double-stranded siRNAs 21-23 nucleotides long with two-base 3' singlestranded regions lead to the generation of single-stranded RNAs that are incorporated into functional siRNA RISC complexes without inducing the generalized inhibition of protein synthesis. This discovery has allowed researchers to use synthetic double-stranded siRNAs to knock down the expression of specific genes in human cells as well as in other mammals. This siRNA knockdown method is now widely used in studies of diverse processes, including the RNAi pathway itself!

## Cytoplasmic Polyadenylation Promotes Translation of Some mRNAs

In addition to miRNAs, several protein-mediated translational controls help regulate the expression of some genes. Regulatory elements in mRNAs that interact with specific proteins to control translation are generally present in the UTR at the $3^{\prime}$ or $5^{\prime}$ end of an mRNA. Here we discuss a type of protein-mediated translational control involving $3^{\prime}$ regulatory elements. A different mechanism, involving RNA-binding proteins that interact with $5^{\prime}$ regulatory elements, is discussed later.

Translation of many eukaryotic mRNAs is regulated by sequence-specific RNA-binding proteins that bind cooperatively to neighboring sites in $3^{\prime}$ UTRs. This allows them to function in a combinatorial manner similar to the cooperative binding of transcription factors to regulatory sites in an enhancer or promoter region. In most cases studied, translation is repressed by protein binding to $3^{\prime}$ regulatory elements, and regulation results from derepression at the appropriate time or place in a cell or developing embryo. The mechanism of such repression is best understood for mRNAs that must undergo cytoplasmic polyadenylation before they can be translated.

Cytoplasmic polyadenylation is a critical aspect of gene expression in the early embryos of animals. The egg cells (oocytes) of multicellular animals contain many mRNAs, encoding numerous different proteins, that are not translated until after the egg is fertilized by a sperm cell. Some of these "stored" mRNAs have a short poly(A) tail, consisting of only 20-40 A residues, to which only a few molecules of cytoplasmic poly(A)-binding protein (PABPC1) can bind. As discussed in Chapter 5, multiple PABPC1 molecules bound to the long poly(A) tail of an mRNA interact with the eIF4G initiation factor, thereby stabilizing the interaction of the mRNA $5^{\prime}$ cap with eIF4E, which is required for translation initiation (see Figure 5-23). Because this stabilization cannot occur with mRNAs that have short poly(A) tails, such mRNAs are not translated efficiently. At the appropriate time during oocyte maturation or after fertilization, usually in response to an external signal, approximately 150 A residues are added to the short poly(A) tails on these mRNAs in the cytoplasm, stimulating their translation.

Studies with mRNAs stored in Xenopus oocytes have helped elucidate the mechanism of this type of translational control. Experiments in which short-tailed mRNAs were injected into oocytes have shown that two sequences in their $3^{\prime}$ UTRs are required for their polyadenylation in the cytoplasm: the AAUAAA polyadenylation signal that is also required for the nuclear polyadenylation of pre-mRNAs, and one or more copies of an upstream U-rich cytoplasmic polyadenylation element (CPE). This regulatory element is bound by a highly conserved CPE-binding protein (CPEB) that contains an RRM domain and a zinc-finger domain.

In the absence of a stimulatory signal, CPEB bound to the U-rich CPE interacts with the protein Maskin, which in turn binds to the eIF4E associated with the mRNA $5^{\prime}$ cap (Figure 10-31, left). As a result, eIF4E cannot interact with other initiation factors or the small ribosomal subunit, so translation initiation is blocked. During oocyte maturation,

![li3222239539b](li3222239539b.jpg)

FIGURE 10-31 Model for control of cytoplasmic polyadenylation and translation initiation. (Left) In immature oocytes, mRNAs containing the U-rich cytoplasmic polyadenylation element (CPE) have short poly(A) tails. CPE-binding protein (CPEB) mediates repression of translation through the interactions depicted, which prevent assembly of an initiation complex at the 5' end of the mRNA. (Right) Hormonal stimulation of oocytes activates a protein kinase that phosphorylates CPEB, causing it to release Maskin. The cleavage and polyadenylation specificity factor (CPSF) then binds to the poly(A) site, interacting with both bound CPEB and the cytoplasmic form of poly(A) polymerase (PAP). After the poly(A) tail is lengthened, multiple copies of cytoplasmic poly(A)-binding protein 1 (PABPC1) can bind to it and interact with eIF4G, which functions with other initiation factors to bind the 40S ribosomal subunit and initiate translation. See R. Mendez and J. D. Richter, 2001, Nat. Rev. Mol. Cell Biol. 2:521.
a specific CPEB serine is phosphorylated, causing Maskin to dissociate from the complex. This allows cytoplasmic forms of the cleavage and polyadenylation specificity factor (CPSF) and poly(A) polymerase (PAP) to bind to the mRNA cooperatively with CPEB. Once PAP catalyzes the addition of A residues, PABPC1 can bind to the lengthened poly(A) tail, leading to the stabilized interaction of all the factors needed to initiate translation (Figure 10-31, right; see also Figure 5-23). In the case of Xenopus oocyte maturation, the protein kinase that phosphorylates CPEB is activated in response to the hormone progesterone. Thus timing of the translation of stored mRNAs encoding proteins needed for oocyte maturation is regulated by this external signal.

Considerable evidence indicates that a similar mechanism of translational control plays a role in learning and memory. In the central nervous system, the axons from a thousand or so neurons can make connections (synapses) with the dendrites of a single postsynaptic neuron (see Figure 22-31). When one of these axons is stimulated, the postsynaptic neuron "remembers" which one of these thousands of synapses was stimulated. The next time that synapse is stimulated, the strength of the response triggered in the postsynaptic cell differs from the first time. This change in response has been shown to result largely from the translational activation of mRNAs stored in the region of the synapse, leading to the local synthesis of new proteins that increase the size and alter the neurophysiological characteristics of the synapse. The finding that CPEB is present in neuronal dendrites has led to the proposal that cytoplasmic polyadenylation stimulates translation of specific mRNAs in dendrites, much as it does in oocytes. In this case, presumably, synaptic activity (rather than a hormone) is the signal that induces phosphorylation of CPEB and subsequent activation of translation.

## Protein Synthesis Can Be Globally Regulated

Like proteins involved in other processes, translation initiation factors and ribosomal proteins can be regulated by posttranslational modifications such as phosphorylation. Such mechanisms affect the translation rates of most mRNAs and hence the overall rates of cellular protein synthesis.

TOR Pathway The TOR pathway was discovered through research into the mechanism of action of rapamycin, an antibiotic produced by a strain of Streptomyces bacteria, which is useful for suppressing the immune response in patients who have undergone organ transplants. The target of rapamycin (TOR) was identified by isolating yeast mutants resistant to rapamycin inhibition of cell growth. TOR is a large ( $\sim 2400$-amino-acid) protein kinase that regulates several cellular processes in yeast cells in response to nutritional status. In mammals, $m$ TOR (mammalian TOR) responds to multiple signals from cell-surface signaling proteins to coordinate cell growth with developmental programs as well as with nutritional status.

In mammals, mTOR is assembled into two types of multiprotein complexes, mTOR complexes 1 and 2 (mTORC1 and mTORC2). The protein kinase activity of mTORC1 increases in response to the presence of amino acids in lysosomes. Its protein kinase activity is also increased by levels of ATP sufficient for cell growth, by oxygen, and by signaling from growth-factor receptors (see Chapter 16). mTORC1 is inhibited by various types of cellular stress, including hypoxia and low levels of ATP and nutrients. It is also the mTOR complex inhibited by rapamycin. Active mTORC1 regulates cellular metabolism to promote cell growth and stimulates ribosome synthesis and translation. It also inhibits autophagy, a process in which large portions of the

cytoplasm, including whole ribosomes, mitochondria, and other organelles, are surrounded by a double membrane, forming an autophagosome that then fuses with lysosomes, in which the contents are digested to provide essential nutrients in times of stress and when nutrient supply is low. The other complex, mTORC2, is insensitive to rapamycin. When active, it regulates the actin cytoskeleton that controls cell shape and movement (see Chapter 17), and it inhibits apoptosis, a highly organized and regulated pathway to cell death that recycles breakdown products of macromolecules and membranes, making them available for uptake by phagocytic cells (see Chapter 21).

Our current understanding of mTORC1 function is summarized in Figure 10-32. Active mTORC1 increases the overall rate of protein synthesis by phosphorylating two critical types of proteins that regulate translation directly. Recall that the first step in translation of a eukaryotic mRNA is binding of the eIF4 initiation complex to the $5^{\prime}$ cap via its eIF4E cap-binding subunit (see Figure 5-23).

The concentration of active eIF4E is regulated by a small family of homologous eIF4E-binding proteins (4E-BPs) that inhibit the interaction of eIF4E with mRNA 5' caps. 4E-BPs are direct targets of mTORC1. When phosphorylated by mTORC1, 4E-BPs release eIF4E, stimulating translation initiation. mTORC1 also phosphorylates and activates another protein kinase, S6 kinase (S6K), that phosphorylates the small ribosomal subunit protein S6 and additional substrates, leading to a further increase in the rate of protein synthesis.

Translation of a specific subset of mRNAs that have a string of pyrimidines in their 5' UTR, called TOP mRNAs (for tract of oligopyrimidine), is stimulated particularly strongly by mTORC1. The TOP mRNAs encode ribosomal proteins and translation elongation factors. S6K activated by mTORC1 activates the RNA polymerase I transcription factor TIF-1A, stimulating transcription of the large rRNA precursor (see Figure 9-51). mTORC1 also phosphorylates and inhibits the RNA polymerase III inhibitor MAF1, thereby
![li3232239539b](li3232239539b.jpg)

FIGURE 10-32 The mTORC1 pathway. mTORC1 is an active protein kinase when bound by a complex of Rheb and an associated GTP (lower left). In contrast, mTORC1 is inactive when bound by a complex of Rheb associated with GDP (lower right). When active, the TSC1/ TSC2 Rheb-GTPase activating protein (Rheb-GAP) causes hydrolysis of Rheb-bound GTP to GDP, thereby inactivating mTORC1. The TSC1/ TSC2 Rheb-GAP is activated (arrows) by phosphorylation by AMP kinase (AMPK) when cellular energy is low and by other cellular stress responses. Signal transduction pathways activated by cell-surface growth-factor receptors lead to phosphorylation of inactivating sites on TSC1/TSC2, inhibiting its GAP activity. Consequently, they leave a higher fraction of cellular Rheb in the GTP conformation that activates
mTORC1 protein kinase activity. Low nutrient concentrations also regulate Rheb GTPase activity by a mechanism that does not require TSC1/ TSC2. Active mTORC1 phosphorylates 4E-BP, causing it to release eIF4E, stimulating translation initiation. It also phosphorylates and activates S6 kinase (S6K), which in turn phosphorylates ribosomal proteins, stimulating translation. Activated mTORC1 also activates transcription factors for RNA polymerases I, II, and III, leading to synthesis and assembly of ribosomes, tRNAs, and translation factors. In the absence of mTORC1 activity, all of these processes are inhibited. In contrast, activated mTORC1 inhibits autophagy, which is stimulated in cells with inactive mTORC1. See S. Wullschleger et al., 2006, Cell 124:471.

stimulating synthesis of 5 S rRNA and tRNAs. In addition, mTORC1 activates two RNA polymerase II activators that stimulate transcription of genes encoding ribosomal proteins and translation factors. Finally, mTORC1 stimulates processing of the large rRNA precursor (see Section 10.5). As a consequence of the phosphorylation of these several mTORC1 substrates, the synthesis and assembly of ribosomes, as well as the synthesis of translation factors and tRNAs, is greatly increased. Alternatively, when mTORC1 kinase activity is inhibited, these substrates become dephosphorylated, which greatly decreases the rate of protein synthesis and the production of ribosomes, translation factors, and tRNAs, thus halting cell growth.

The activity of mTORC1 is regulated by a monomeric G protein in the Ras protein family, called Rheb. Like other small monomeric G proteins, Rheb is in its active conformation when it is bound to GTP (see Figures 15-4 and 15-5). Rheb-GTP binds the mTORC1 complex, stimulating mTORC1 kinase activity, probably by inducing a conformational change in its kinase domain. Rheb, in turn, is regulated by a heterodimer composed of subunits TSC1 and TSC2, named for their involvement in the medical syndrome tuberous sclerosis complex, described below. In the active conformation, the TSC1/TSC2 heterodimer functions as a GTPase-activating protein (GAP) for Rheb, causing hydrolysis of the Rheb-bound GTP to GDP. This converts Rheb to its GDP-bound conformation, which does not activate mTORC1 kinase.

The activity of the TSC1/TSC2 Rheb-GAP is regulated by several inputs, allowing the cell to integrate information from different cellular signaling pathways to control the overall rate of protein synthesis. Signaling from cell-surface growth-factor receptors leads to phosphorylation of TSC1/ TSC2 at inhibitory sites, causing an increase in Rheb-GTP and activation of mTORC1 kinase activity. This type of regulation through cell-surface receptors links the control of cell growth to developmental processes controlled by cellcell interactions.

When energy from nutrients is not sufficient for cell growth, the resulting fall in the ratio of ATP to AMP concentrations is detected by AMP-activated kinase (AMPK). The activated AMPK phosphorylates TSC1/TSC2 at activating sites, stimulating its Rheb-GAP activity and consequently inhibiting mTORC1 kinase activity and the global rate of translation. Hypoxia and other cellular stresses also activate the TSC1/TSC2 Rheb-GAP.

Activation of mTORC1 depends on the regulated association of mTORC1 with lysosomes. Much of the Rheb in the cell is associated with the outer lysosomal membrane, and other proteins that help Rheb-GTP to associate with mTORC1 are restricted to the outer lysosomal membrane. As mentioned previously, regulation of mTORC1 activity is controlled by the lysosomal concentration of amino acids. The mechanism by which this occurs is currently an active area of investigation.

In contrast to mTORC1, mTORC2 is insensitive to nutrients. However, mTORC2 is activated by insulin binding
to the insulin receptor, which regulates carbohydrate uptake and metabolism (see Section 16.8). mTORC2 also phosphorylates and activates protein kinase B (also called Akt) (see Figure 16-29), protein kinase C (see page 714), and serum- and glucocorticoid-induced protein kinase 1 (SGK1). These protein kinases, in turn, regulate metabolism, apoptosis, and cell shape through regulation of the actin cytoskeleton (see Chapter 17).

8Genes encoding components of the mTORC1 pathway are mutated in many human cancers, resulting in cell growth in the absence of normal growth signals. TSC1 and TSC2 (see Figure 10-32) were initially identified because one or the other is mutated in a rare human genetic syndrome: tuberous sclerosis complex. Patients with this disorder develop benign tumors in multiple tissues. The disease results because inactivation of either TSC1 or TSC2 eliminates the Rheb-GAP activity of the TSC1/TSC2 heterodimer, resulting in an abnormally high and unregulated level of Rheb-GTP and thus high, unregulated mTOR activity. Mutations in components of cell-surface receptor signal transduction pathways that lead to inhibition of TSC1/ TSC2 Rheb-GAP activity are also common in human tumors and contribute to cell growth and replication in the absence of normal signals for growth and proliferation. High mTORC1 protein kinase activity in tumors correlates with a poor clinical prognosis. Consequently, mTOR inhibitors are currently in clinical trials to test their effectiveness for treating cancers in conjunction with other modes of therapy. Rapamycin and other structurally related mTORC1 inhibitors are potent suppressors of the immune response because they inhibit activation and replication of T lymphocytes in response to foreign antigens (see Chapter 23). Several viruses encode proteins that activate mTORC1 soon after viral infection. The resulting stimulation of translation has an obvious selective advantage for these cellular parasites. $\square$
eIF2 Kinases eIF2 kinases also regulate the global rate of cellular protein synthesis. As Figure 5-23 shows, the translation initiation factor eIF2 brings the charged initiator tRNA to the small ribosomal subunit P site. eIF2 is a trimeric G protein and consequently exists in either a GTP-bound or a GDP-bound conformation. Only the GTPbound form of eIF2 is able to bind the charged initiator tRNA and associate with the small ribosomal subunit. The small ribosomal subunit, with bound initiation factors and charged initiator tRNA, then interacts with the eIF4 complex bound to the $5^{\prime}$ cap of an mRNA via its eIF4E subunit. The small ribosomal subunit then scans down the mRNA in the $3^{\prime}$ direction until it reaches an AUG initiation codon that can base-pair with the initiator tRNA in its P site. When this occurs, the GTP bound by eIF2 is hydrolyzed to GDP and the resulting eIF2-GDP complex is released. GTP hydrolysis results in an irreversible "proofreading" step that prepares the small ribosomal subunit to associate with the large subunit only when an initiator tRNA is properly

bound in the P site and is properly base-paired with the AUG start codon. Before eIF2 can participate in another round of initiation, its bound GDP must be replaced with a GTP. This process is catalyzed by the translation initiation factor eIF2B, a guanine nucleotide exchange factor (GEF) specific for eIF2.

A global mechanism for inhibiting protein synthesis in stressed cells involves phosphorylation of the eIF2 $\alpha$ subunit at a specific serine. Phosphorylation at this site does not interfere with eIF2 function in protein synthesis directly. Rather, phosphorylated eIF2 has very high affinity for the eIF2 guanine nucleotide exchange factor, eIF2B, which cannot release the phosphorylated eIF2 and is consequently blocked from catalyzing GTP exchange by additional eIF2 factors. Since there is an excess of eIF2 over eIF2B, phosphorylation of a fraction of eIF2 results in inhibition of all the cellular eIF2B. The remaining eIF2 accumulates in its GDP-bound form, which cannot participate in protein synthesis, thereby inhibiting nearly all protein synthesis in the cell. However, some mRNAs have $5^{\prime}$ regions that allow translation initiation at the low eIF2-GTP concentration that results from eIF2 phosphorylation. These mRNAs include those for chaperone proteins that function to refold cellular proteins denatured as the result of cellular stress, additional proteins that help the cell to cope with stress, and transcription factors that activate transcription of the genes encoding these stress-induced proteins.

Humans express four eIF2 kinases that all phosphorylate the same inhibitory eIF2 $\alpha$ serine. Each of these kinases is regulated by a different type of cellular stress, and each one inhibits protein synthesis, allowing cells to divert the large fraction of their resources usually devoted to protein synthesis when they are growing for use in responding to the stress.

The GCN2 (general control non-derepressible 2) eIF2 kinase is activated by binding uncharged tRNAs. The concentration of uncharged tRNAs increases when cells are starved for amino acids, activating GCN2 eIF2 kinase and greatly inhibiting protein synthesis.

PEK (pancreatic eIF2 kinase) is activated when proteins translocated into the endoplasmic reticulum (ER) do not fold properly because of abnormalities in the ER lumen environment. Inducers of PEK include abnormal carbohydrate concentrations, which inhibit the glycosylation of many ER proteins. Inactivating mutations in an ER chaperone required for proper folding of many ER proteins (see Chapters 13 and 14) also result in PEK activation.

Heme-regulated inhibitor (HRI) is an eIF2 kinase activated in developing red blood cells when the supply of the heme prosthetic group is too low to accommodate the rate of globin protein synthesis. This negative feedback loop lowers the rate of globin protein synthesis until it matches the rate of heme synthesis. HRI is also activated in other types of cells in response to oxidative stress or heat shock.

Finally, protein kinase RNA-activated (PKR) is activated by double-stranded RNAs longer than about 30 base pairs. Under normal circumstances in mammalian cells, such double-stranded RNAs are produced only during a
viral infection. Long regions of double-stranded RNA are generated as replication intermediates of RNA viruses or by hybridization of complementary regions of RNA transcribed from both strands of DNA virus genomes. Inhibition of protein synthesis prevents the production of progeny virions, protecting neighboring cells from infection. Interestingly, adenoviruses have evolved a defense against PKR: they express prodigious amounts of a 160-nucleotide virusassociated (VA) RNA with long double-stranded hairpin regions. VA RNA is transcribed by RNA polymerase III and exported from the nucleus by exportin 5, the exportin for pre-miRNAs (see Figure 10-29). VA RNA binds to PKR with high affinity, inhibiting its protein kinase activity and preventing the inhibition of protein synthesis observed in cells infected with a mutant adenovirus from which the VA gene had been deleted.

## Sequence-Specific RNA-Binding Proteins Control Translation of Specific mRNAs

In contrast to the global mRNA regulation we have just described, other mechanisms have evolved for controlling the translation of certain specific mRNAs. These mechanisms usually rely on sequence-specific RNA-binding proteins that bind to a particular sequence or structure in the mRNA. When such proteins bind to the $5^{\prime}$ UTR of an mRNA, the small ribosomal subunit's ability to scan to the first initiation codon is blocked, inhibiting translation initiation. Binding in other regions can either promote or inhibit mRNA degradation.

Control of intracellular iron concentrations by the ironresponse element-binding protein (IRE-BP) is an elegant example of a system in which a single protein regulates the translation of one mRNA and the degradation of another. Precise regulation of cellular iron ion concentrations is critical to the cell. Multiple enzymes and proteins contain $\mathrm{Fe}^{2+}$ as a cofactor, such as enzymes of the citric acid cycle (see Figure 12-16) and electron-carrying proteins involved in the generation of ATP by mitochondria and chloroplasts (see Chapter 12). On the other hand, excess $\mathrm{Fe}^{2+}$ generates free radicals that react with and damage cellular macromolecules. When intracellular iron stores are low, a dual-control system operates to increase the level of cellular iron; when iron is in excess, the system operates to prevent accumulation of toxic levels of free ions.

One component of this system is regulation of the production of ferritin, an intracellular protein that binds and stores excess cellular iron. The $5^{\prime}$ UTR of ferritin mRNA contains iron-response elements (IREs) that have a stemloop structure. IRE-BP recognizes five specific bases in the IRE loop and the duplex nature of the stem. At low iron concentrations, IRE-BP is in an active conformation that binds to the IREs (Figure 10-33a). The bound IRE-BP blocks the small ribosomal subunit from scanning for the AUG start codon (see Figure 5-23), thereby inhibiting translation initiation. The resulting decrease in ferritin means that less iron is complexed with ferritin, and therefore more iron is available

![li3242239539b](li3242239539b.jpg)

FIGURE 10-33 Iron-dependent regulation of mRNA translation and degradation. The iron-response element-binding protein (IREBP) controls (a) translation of ferritin mRNA and (b) degradation of transferrin-receptor (TfR) mRNA. At low intracellular iron concentrations, IRE-BP binds to iron-response elements (IREs) in the $5^{\prime}$ or $3^{\prime}$ UTR of these mRNAs. At high iron concentrations, IRE-BP undergoes a conformational change and cannot bind either mRNA. The dual control by IRE-BP precisely regulates the level of free iron ions within cells. See the text for discussion.
to iron-requiring enzymes. At high iron concentrations, IREBP is in an inactive conformation that does not bind to the $5^{\prime}$ IREs, so translation initiation can proceed. The newly synthesized ferritin then binds free iron ions, preventing their accumulation to harmful levels.

The other part of this regulatory system controls the import of iron into cells. In vertebrates, ingested iron is carried through the circulatory system bound to a protein called transferrin. After binding to the transferrin receptor (TfR) in the plasma membrane, the transferrin-iron complex is brought into cells by receptor-mediated endocytosis (see Figure 14-31). The $3^{\prime}$ UTR of TfR mRNA contains IREs whose stems have destabilizing AU-rich elements (Figure 10-33b). At high iron concentrations, when IRE-BP is in its inactive, nonbinding conformation, these AU-rich elements promote degradation of TfR mRNA by the mechanism described earlier in this section that leads to rapid degradation of other short-lived mRNAs with AU-rich elements.

The resulting decrease in production of the transferrin receptor quickly reduces iron import, thus protecting the cell from excess iron. At low iron concentrations, however, IREBP is active and can bind to the $3^{\prime}$ IREs in TfR mRNA. The bound IRE-BP blocks recognition of the AU-rich elements by the proteins that would otherwise lead to rapid degradation of the mRNAs. As a result, production of the transferrin receptor increases, and more iron is transported into the cell.

Other regulated RNA-binding proteins function to control the translation or degradation of specific mRNAs in a similar manner. For example, a heme-sensitive RNAbinding protein controls translation of the mRNA encoding aminolevulinate (ALA) synthase, a key enzyme in the synthesis of heme. Similarly, in vitro studies have shown that the mRNA encoding the milk protein casein is stabilized by the hormone prolactin and rapidly degraded in its absence.

## Surveillance Mechanisms Prevent Translation of Improperly Processed mRNAs

Translation of an improperly processed mRNA could lead to production of an abnormal protein that interferes with the gene's normal function. This effect would be equivalent to that of a dominant-negative mutation, discussed in Chapter 6 (see Figure 6-41). Several mechanisms, collectively termed mRNA surveillance, help cells avoid the translation of improperly processed mRNA molecules. We have previously mentioned two such surveillance mechanisms: the recognition of improperly processed pre-mRNAs in the nucleus and their degradation by nuclear exosomes, and the general restriction against nuclear export of incompletely spliced pre-mRNAs that remain associated with a snRNP.

Another surveillance mechanism, called nonsense-mediated decay (NMD), causes degradation of mRNAs in which one or more exons have been incorrectly spliced. Such incorrect splicing often alters the open reading frame of the mRNA $3^{\prime}$ to the improper exon-exon junction, resulting in the introduction of an out-of-frame missense mutation and an incorrect stop codon. For nearly all properly spliced mRNAs, the stop codon is in the last exon. Nonsense-mediated decay results in the rapid degradation of mRNAs with stop codons that occur before the last exon-exon junction, since in most cases, such mRNAs arise from errors in RNA splicing. However, NMD can also result from a mutation creating a stop codon within a gene or a frame-shifting deletion or insertion. NMD was initially discovered during the study of patients with $\beta^{0}$-thalassemia, who produce a low level of $\beta$-globin protein associated with a low level of $\beta$-globin mRNA (Figure 10-34).

A search for possible molecular signals that might indicate the positions of exon-exon junctions in a processed mRNA led to the discovery of exon-junction complexes. As noted already, these complexes of several proteins (including Y14, Magoh, eIF4IIIA, UPF2, UPF3, and REF) bind about 20 nucleotides $5^{\prime}$ to an exon-exon junction

(a)
![li3252239539b](li3252239539b.jpg)

FIGURE 10-34 Discovery of nonsense-mediated decay (NMD).
(a) Patients with $\beta^{0}$-thalassemia express very low levels of $\beta$-globin mRNA. A common cause of this syndrome is a single-base-pair deletion in exon 1 or exon 2 of the $\beta$-globin gene. Ribosomes translating the mutant mRNA read out of frame following the deletion and encounter a stop codon in the wrong reading frame before they translate across the last exon-exon junction in the mRNA. Consequently, they leave an exon-junction complex (EJC) in place on the mRNA. Cytoplasmic proteins associate with the EJC and induce degradation of the mRNA. (b) Bone marrow was obtained from a patient with a wild-type $\beta$-globin gene and from a patient with $\beta^{0}$-thalassemia. RNA was isolated from
following RNA splicing and stimulate export of mRNPs from the nucleus by interacting with the mRNP exporter (see Figure 10-23). Analysis of yeast mutants indicated that one of the proteins in exon-junction complexes (UPF3) functions in nonsense-mediated decay. In the cytoplasm, UPF3 interacts with a protein (UPF1) and a protein kinase (SMG1) that phosphorylates it, causing the mRNA to associate with P bodies, repressing translation and inducing degradation of the mRNA. An additional protein (UPF2) associated with the exon-junction complex binds a P bodyassociated deadenylase complex that rapidly removes the poly(A) tail from the associated mRNA, leading to its decapping and degradation by the P body-associated $5^{\prime} \rightarrow 3^{\prime}$ exoribonuclease XRN1 (see Figure 10-24). In the case of a properly spliced mRNA, the mRNP exporter associated with the nuclear cap-binding complex is exported through a nuclear pore complex, thereby protecting the mRNA from degradation. The exon-junction complexes are thought to be dislodged from the mRNA by passage of the first "pioneer" ribosome to translate the mRNA. However, for mRNAs with a stop codon before the final exon-exon junction, one or more exon-junction complexes remain associated with the mRNA, resulting in nonsensemediated decay (Figure 10-35a). Alternative mechanisms lead to the inhibition of translation and degradation of mRNAs that were polyadenylated prematurely (non-stop decay) (Figure 10-35b) or that contain damaged bases or stable secondary structures that block ribosomal translocation along the mRNA (no-go decay) (Figure 10-35c).

## Localization of mRNAs Permits Production of Proteins at Specific Regions Within the Cytoplasm

Many cellular processes depend on localization of particular proteins to specific structures or regions of the cell. In later chapters, we examine how some proteins are transported after their synthesis to their proper cellular location.

(b)
![li3262239539b](li3262239539b.jpg)
the bone marrow cells shortly after collection and again 30 minutes after incubation in media with actinomycin D, a drug that inhibits transcription. The amount of $\beta$-globin RNA was measured using the S1-nuclease protection method (arrow). The patient with $\beta^{0}$-thalassemia had much less $\beta$-globin mRNA than the patient with a wild-type $\beta$-globin gene ( $\sim$ Act D). The mutant $\beta$-globin mRNA decayed rapidly when transcription was inhibited ( + Act D), whereas the wild-type $\beta$-globin mRNA remained stable. (Part (b) republished with permission of Elsevier, from Maquat, L.E., et al., "Unstable $\beta$-globin mRNA in mRNA-deficient $\beta^{0}$ thalassemia," Cell, 1981, 27(3 Pt2):543-53; permission conveyed through Copyright Clearance Center, Inc.)
Alternatively, protein localization can be achieved by localization of mRNAs to the specific regions of the cytoplasm in which their encoded proteins function. In most cases examined thus far, such mRNA localization is specified by sequences in the $3^{\prime}$ UTR of the mRNA. A recent genomiclevel study of mRNA localization in Drosophila embryos revealed that some 70 percent of the 3000 mRNAs analyzed were localized to specific subcellular regions, raising the possibility that mRNA localization is a much more general phenomenon than previously appreciated.

Localization of mRNAs to the Bud in S. cerevisiae The most thoroughly understood example of mRNA localization occurs in the budding yeast $S$. cerevisiae. As discussed in Chapter 9, whether a haploid yeast cell exhibits the a or $\alpha$ mating type is determined by whether a or $\alpha$ genes are present at the expressed MAT locus on chromosome III (see Figure 9-35). The process that transfers a or $\alpha$ genes from the silent mating-type locus to the expressed MAT locus is initiated by a sequence-specific endonuclease called HO. Transcription of the HO gene is dependent on the SWI/SNF chromatin-remodeling complex (see Section 9.5). Daughter yeast cells that arise by budding from mother cells contain a transcriptional repressor called Ash1 (for Asymmetric synthesis of HO) that prevents recruitment of the SWI/SNF complex to the $H O$ gene, thereby preventing its transcription. The absence of Ash1 from mother cells allows them to transcribe the $H O$ gene. As a consequence, mother cells switch their mating type, while daughter cells generated by budding do not (Figure 10-36a).

Ash1 protein accumulates only in daughter cells because the mRNA encoding it is localized to daughter cells. The localization process requires three proteins: She2 (for SWIdependent $H O$ expression), an RNA-binding protein that binds specifically to a localization signal with a specific RNA structure in the ASH1 mRNA; Myo4, a myosin motor protein that moves cargoes along actin filaments (see Chapter 17);

![li3272239539b](li3272239539b.jpg)

FIGURE 10-35 Mechanisms of RNA surveillance in the cytoplasm. (a) Nonsense-mediated decay. PTC = premature termination (stop) codon; SURF complex = complex of protein kinase SMG1, UPF1, and release factors eRF1 and eRF3. Formation of the SURF complex leads to phosphorylation of UPF1 by SMG1. The phosphorylated UPF1 associates with the UPF2-UPF3 complex bound to any exon-exon junction complexes that were not displaced from the mRNA by the first, pioneer ribosome to translate the message. This association leads to the association of the PTC-containing mRNA with P bodies, removal of the poly(A) tail, and degradation of the mRNA. (b) Non-stop decay. mRNAs that were prematurely cleaved and polyadenylated do not contain a stop codon before the poly(A) tail. When such mRNAs are translated, the ribosome translates the poly(A) tail and stalls at the $3^{\prime}$ end of the abnormal mRNA because the stop codon required for release factors eRF1 and eRF3 to associate with the ribosome A site is absent (see Figure 5-26). In higher eukaryotes, the factor Ski7 binds to the stalled ribosome and recruits the cytoplasmic exosome, which degrades the abnormal RNA step. Alternatively, in S. cerevisiae step the displacement of PABPC1 from the poly(A) tail by the elongating ribosome leads to decapping and $5^{\prime} \rightarrow 3^{\prime}$ degradation by the XRN1 exonuclease. (c) No-go decay. If a base of an mRNA is damaged so that a ribosome stalls there, or if an improperly processed mRNA has a stable stem-loop region with a long duplex stem that blocks elongation by the ribosome, the Dom34-Hbs1 complex binds the abnormal mRNA and makes an endonucleolytic cut in it. This cut generates a free $3^{\prime}$ end on the $5^{\prime}$ fragment, which is degraded by a cytoplasmic exosome, and a free $5^{\prime}$ end on the $3^{\prime}$ fragment, which is digested in the $5^{\prime} \rightarrow 3^{\prime}$ direction by the XRN1 exonuclease. See N. L. Garneau, J. Wilusz, and C. J. Wilusz, 2007, Nat. Rev. Mol. Cell Biol. 8:113.

![li3282239539b](li3282239539b.jpg)
and She3, which links She2, and therefore ASH1 mRNA, to Myo4 (Figure 10-36b). ASH1 mRNA is transcribed in the nucleus of the mother cell before mitosis. Movement of Myo4, with its bound ASH1 mRNA, along actin filaments that extend from the mother cell into the bud carries the ASH1 mRNA into the growing bud before cell division.

At least 23 other mRNAs were found to be transported by the She2/She3/Myo4 system. All have an RNA localization signal to which She2 binds, usually in the $3^{\prime}$ UTR. The transport process can be visualized in live cells by the experiment shown in Figure 10-37. RNAs can be fluorescently labeled by including in their sequence high-affinity binding sites for RNA-binding proteins, such as bacteriophage MS2 coat protein and bacteriophage $\lambda \mathrm{N}$ protein, which bind to different stem-loops with specific sequences (Figure 10-37a). When such engineered mRNAs are expressed in budding yeast cells, along with the bacteriophage proteins fused to proteins that fluoresce different colors, the fusion proteins bind to their specific RNA binding sites, thereby labeling the RNAs that contain those sites with different colors. In the experiment shown in Figure 10-37b, ASH1 mRNA was labeled by the binding of green fluorescent protein fused to $\lambda \mathrm{N}$. Another mRNA localized to the

FIGURE 10-36 Switching of mating type in haploid yeast cells. (a) Division by budding forms a larger mother cell (M) and smaller daughter cell (D), both of which have the same mating type as the original cell ( $\alpha$ in this example). The mother cell can switch mating type during $G_{1}$ of the next cell cycle and then divide again, producing two cells of the opposite type (a in this example). Switching depends on transcription of the $H O$ gene, which occurs only in the absence of Ash1 protein. The smaller daughter cells, which produce Ash1 protein, cannot switch; after growing in size through interphase, they divide to form a mother cell and daughter cell. (b) Model for restriction of mating-type switching to mother cells in S. cerevisiae. Ash1 protein prevents a cell from transcribing the $H O$ gene, whose encoded protein initiates the DNA rearrangement that results in mating-type switching from a to $\alpha$ or $\alpha$ to a. Switching occurs only in the mother cell, after it separates from a newly budded daughter cell, because the Ash1 protein is present only in the daughter cell. The molecular basis for this differential localization of Ash1 is the one-way transport of ASH1 mRNA into the bud. A linking protein, She2, binds to specific $3^{\prime}$ untranslated sequences in the ASH1 mRNA and also binds to She3 protein. This protein, in turn, binds to a myosin motor, Myo4, which moves along actin filaments into the bud. See S. Koon and B. J. Schnapp, 2001, Curr. Biol. 11:R166.
bud by the same transport system, the IST2 mRNA, which encodes a component of the growing bud membrane, was labeled by the binding of red fluorescent protein fused to MS2 coat protein. Video of a budding cell showed that the differently labeled ASH1 and IST2 mRNAs accumulated in the same large cytoplasmic RNP particle, containing multiple mRNAs, in the mother-cell cytoplasm, as can be seen from the merge of the green and red fluorescent signals. The RNP particle was then transported into the bud within about one minute.

Formation of large cytoplasmic RNP particles like those observed in Figure 10-37b, and in other examples of transported RNA in cells of higher eukaryotes, requires low-complexity amino acid sequences, such as sequences composed of repeats of [G/S]Y[G/S], in the RNA-binding protein. Peptides containing these low-complexity sequences spontaneously associate in vitro, forming a semi-permeable gel. These gels can be dissociated by phosphorylation of serines within them. Such complexes are probably involved in the formation of the large RNP complexes transported on actin cables in yeast and on microtubules in large asymmetric cells in higher eukaryotes (see Chapter 17 and 18), such as the neurons described in the next section. Regulated phosphorylation of these low-complexity sequences in RNAbinding proteins associated with RNP particles may well account for the regulated formation and dissociation of RNP particles such as those observed in Figure 10-37b.

Localization of mRNAs to Synapses in the Mammalian Nervous System As mentioned earlier, localization of specific mRNAs at synapses far from the nucleus of a neuron plays an essential role in learning and memory (Figure 10-38). Like the localized mRNAs in yeast, these mRNAs contain RNA localization signals in their $3^{\prime}$ UTR. Some of these mRNAs are initially synthesized with short

![li3292239539b](li3292239539b.jpg)

EXPERIMENTAL FIGURE 10-37 Transport of mRNP particles from a yeast mother cell into the bud. (a) Yeast cells were engineered to express an ASH1 mRNA with binding sites for the bacteriophage $\lambda \mathrm{N}$ protein in its $5^{\prime}$ UTR, and an IST2 mRNA with binding sites for bacteriophage MS2 coat protein in its $3^{\prime}$ UTR. A fusion of green fluorescent protein to $\lambda \mathrm{N}$ protein (GFP- $\lambda \mathrm{N}$ ) and a fusion of red fluorescent protein to MS2 coat protein (RFP-MS2) were also expressed in the same cells. In other experiments, these fluorescently tagged sequencespecific RNA-binding proteins were shown to bind to their own specific binding sites engineered into the ASH1 and IST2 mRNAs, and not to each other's binding sites. Both fluorescently tagged bacteriophage proteins also contained a nuclear localization signal so that those proteins that were not bound to their high-affinity binding sites in these mRNAs were transported into nuclei through nuclear pore complexes (see Chapter 13). This step was necessary to prevent high fluorescence from excess GFP- $\lambda \mathrm{N}$ and RFP-MS2 in the cytoplasm. (b) Frames from a video of fluorescing cells. GFP- $\lambda \mathrm{N}$ and RFP-MS2 were independently visualized by using millisecond alternating laser excitation of GFP and RFP. The nucleus next to the large vacuole in the mother cell near the center of each micrograph, as well as nuclei in neighboring cells, was observed by green and red fluorescence, as shown in the top and middle rows. A merge of the two images is shown in the bottom row, which also indicates the time elapsed between images. An RNP particle containing both the ASH1 mRNA with $\lambda \mathrm{N}$-binding sites and the IST2 mRNA with MS2-binding sites was observed in the mother-cell cytoplasm in the left column of images (arrow). The particle increased in intensity between 0.00 and 46.80 seconds, indicating that more of these mRNAs joined the RNP particle. The RNP particle was transported into the bud between 46.80 and 85.17 seconds and then became localized to the bud tip. [Republished with permission of John Wiley \& Sons, Inc., from Lange, S. et al., "Simultaneous transport of different localized mRNA species revealed by live-cell imaging," 2008, Traffic, 9(8)1256-67; permission conveyed through Copyright Clearance Center, Inc. See this paper to view the video.]
![li3302239539b](li3302239539b.jpg)

EXPERIMENTAL FIGURE 10-38 A specific neuronal mRNA localizes to synapses. Sensory neurons from the sea slug Aplysia californica were cultured with target motor neurons so that processes from the sensory neurons formed synapses with processes from the motor neurons. The micrograph at the left shows motor neuron processes visualized with a blue fluorescent dye. GFP-VAMP (green) was expressed in sensory neurons and marks the location of synapses formed between sensory and motor neuron processes (arrows). The micrograph at the right shows red fluorescence from in situ hybridization of an antisensorin mRNA probe. Sensorin is a neurotransmitter expressed by the sensory neuron only; sensory neuron processes are not otherwise visualized in this preparation, but they lie adjacent to the motor neuron processes. The in situ hybridization results indicate that sensorin mRNA is localized to synapses. [Republished with permission from Elsevier, from Lyles, V., et al., "Synapse formation and mRNA localization in cultured Aplysia neurons," Neuron, 2006, 49(3):349-356; permission conveyed through Copyright Clearance Center, Inc.]
poly(A) tails that do not allow translation initiation. Once again, large RNP particles containing multiple mRNAs bearing localization signals form in the cytoplasm near the nucleus. In this case, the RNP particles are then transported down the axon to synapses by kinesin motor proteins, which travel down microtubules extending the length of the axon (see Chapter 18). Electrical activity at a given synapse may then stimulate polyadenylation of the mRNAs in the region of that synapse, activating the translation of encoded proteins that increase the size and alter the neurophysiological properties of that synapse, while leaving unaffected the hundreds to thousands of other synapses made by the neuron.

## KEY CONCEPTS OF SECTION 10.4

## Cytoplasmic Mechanisms of Posttranscriptional Control

- Most mRNAs are degraded as the result of the gradual shortening of the poly(A) tail (deadenylation) followed by exosome-mediated $3^{\prime} \rightarrow 5^{\prime}$ digestion, or removal of the $5^{\prime}$ cap and digestion by a $5^{\prime} \rightarrow 3^{\prime}$ exoribonuclease (see Figure 10-27).
- Eukaryotic mRNAs encoding proteins that are expressed in short bursts generally have repeated copies of an AU-rich sequence (AU-rich element) in their $3^{\prime}$ UTR. Specific proteins that bind to these elements also interact with a deadenylating enzyme complex and cytoplasmic exosomes, promoting rapid RNA degradation.

- Translation can be repressed by micro-RNAs (miRNAs), which form imperfect hybrids with sequences in the $3^{\prime}$ untranslated region (UTR) of specific target mRNAs. mRNAs bound by several miRNAs are concentrated in P bodies in the cytoplasm, where they are degraded by decapping followed by digestion by the cytoplasmic exosome.
- The related phenomenon of RNA interference, which probably evolved as an early defense system against viruses and transposons, leads to rapid degradation of mRNAs that form perfect hybrids with short interfering RNAs (siRNAs).
- Both miRNAs and siRNAs contain 21-23 nucleotides, are generated from longer precursor molecules, and are bound by an Argonaute protein and assembled into a multiprotein RNA-induced silencing complex (RISC). RISC complexes either repress translation of target mRNAs and induce their localization to P bodies, where they are degraded (miRNAs), or cleave them (siRNAs), generating unprotected ends that are rapidly degraded by cytoplasmic exosomes and the $5^{\prime} \rightarrow 3^{\prime}$ exonuclease XRN1 (see Figures 10-28 and 10-29).
- Cytoplasmic polyadenylation is required for the translation of mRNAs with a short poly(A) tail. Binding of a specific protein to regulatory elements in the $3^{\prime}$ UTRs represses translation of these mRNAs. Phosphorylation of this RNAbinding protein, induced by an external signal, leads to lengthening of the $3^{\prime}$ poly(A) tail and thus translation (see Figure 10-31).
- Binding of various proteins to regulatory elements in the $3^{\prime}$ or $5^{\prime}$ UTRs of mRNAs regulates the translation or degradation of many mRNAs in the cytoplasm.
- Translation of ferritin mRNA and degradation of transferrin receptor (TfR) mRNA are both regulated by the same iron-sensitive RNA-binding protein, IRE-BP. At low iron concentrations, this protein has an active conformation that binds to specific sequences that form stem-loops in the mRNAs, inhibiting ferritin mRNA translation and degradation of TfR mRNA (see Figure 10-33). This dual control precisely regulates the iron level within cells.
- Nonsense-mediated decay and other mRNA surveillance mechanisms prevent the translation of improperly processed mRNAs encoding abnormal proteins that might interfere with the functioning of the corresponding normal proteins.
- Many mRNAs are transported to specific subcellular locations by sequence-specific RNA-binding proteins that bind localization sequences usually found in the $3^{\prime}$ UTR. These RNA-binding proteins then associate, directly or via intermediary proteins, with motor proteins that carry large RNP particles, containing many mRNAs bearing the localization signals, on actin or microtubule fibers to specific locations in the cytoplasm.


### 10.5 Processing of rRNA and tRNA

Approximately 80 percent of the total RNA in rapidly growing mammalian cells (e.g., cultured HeLa cells) is rRNA, and 15 percent is tRNA; protein-coding mRNA thus constitutes only a small portion of the total RNA. The primary transcripts produced from most rRNA genes and from tRNA genes, like pre-mRNAs, are extensively processed to yield the mature, functional forms of these RNAs.

The ribosome is a highly evolved, complex structure (see Figure 5-22), optimized for its function in protein synthesis. Ribosome synthesis requires the function and coordination of all three nuclear RNA polymerases. The 28 S and 5.8 S rRNAs associated with the large ribosomal subunit and the single 18 S rRNA of the small subunit are transcribed by RNA polymerase I. The 5 S rRNA of the large subunit is transcribed by RNA polymerase III, and the mRNAs encoding the ribosomal proteins are transcribed by RNA polymerase II. In addition to the four rRNAs and some 70 ribosomal proteins, at least 150 other RNAs and proteins interact transiently with the two ribosomal subunits during their assembly through a series of coordinated steps. Furthermore, multiple specific bases and riboses of the mature rRNAs are modified to optimize their function in protein synthesis. Although most of the steps in ribosomal subunit synthesis and assembly occur in the nucleolus (a subcompartment of the nucleus not bounded by a membrane), some occur in the nucleoplasm during passage from the nucleolus to nuclear pore complexes. A quality-control step occurs before nuclear export so that only fully functional subunits are exported to the cytoplasm, where the final steps of ribosomal subunit maturation occur. tRNAs are also processed from precursor primary transcripts in the nucleus and modified extensively before they are exported to the cytoplasm and used in protein synthesis. We begin this section by discussing the processing and modification of rRNA and the assembly and nuclear export of ribosomes. Then we consider the processing and modification of tRNAs.

## Pre-rRNA Genes Function as Nucleolar Organizers

The 28 S and 5.8 S rRNAs associated with the large (60S) ribosomal subunit and the 18 S rRNA associated with the small (40S) ribosomal subunit in higher eukaryotes (and the functionally equivalent rRNAs in all other eukaryotes) are all encoded by a single pre-rRNA transcription unit. In human cells, its transcription by RNA polymerase I yields a $45 \mathrm{~S}(-13.7-\mathrm{kb})$ primary transcript (pre-rRNA), which is cleaved and processed into the mature $28 \mathrm{~S}, 18 \mathrm{~S}$, and 5.8 S rRNAs found in cytoplasmic ribosomes. The fourth rRNA, 5 S , is encoded separately and transcribed outside the nucleolus. Sequencing of the DNA encoding the 45 S pre-rRNA from many species showed that this DNA shares several properties in all eukaryotes. First, the pre-rRNA genes are arranged in long tandem arrays separated by nontranscribed

spacer regions ranging in length from 2 kb in frogs to 30 kb in humans (Figure 10-39). Second, the genomic regions corresponding to the three mature rRNAs are always arranged in the same $5^{\prime} \rightarrow 3^{\prime}$ order: 18S, 5.8 S , and 28 S . Third, in all eukaryotic cells (and even in bacteria), the pre-rRNA gene codes for regions that are removed during processing and rapidly degraded. These regions probably contribute to proper folding of the rRNAs but are not required once that folding has occurred. The general structure of pre-rRNA transcription units is diagrammed in Figure 10-40.

The synthesis and most of the processing of pre-rRNA occurs in the nucleolus. When pre-rRNA genes were initially identified in the nucleolus by in situ hybridization, it was not known whether any other DNA was required to form the nucleolus. Subsequent experiments with transgenic Drosophila strains demonstrated that a single complete pre-rRNA transcription unit induces formation of a small nucleolus. Thus a single pre-rRNA gene is sufficient to be a nucleolar organizer, and all the other components of the ribosome diffuse to the newly formed pre-rRNA. The structure of the nucleolus observed by light and electron microscopy results from
![li3312239539b](li3312239539b.jpg)

EXPERIMENTAL FIGURE 10-39 Electron micrograph of pre-rRNA transcription units from the nucleolus of a frog oocyte. Each "feather" represents multiple pre-rRNA molecules associated with protein in a pre-ribonucleoprotein complex (pre-rRNP) emerging from a transcription unit. Note the dense "knob" at the $5^{\prime}$ end of each nascent pre-RNP, which is thought to be a processome. Pre-rRNA transcription units are arranged in tandem, separated by nontranscribed spacer regions of nucleolar chromatin. [Courtesy of Y. Osheim and O. J. Miller, Jr.]
![li3322239539b](li3322239539b.jpg)

FIGURE 10-40 General structure of eukaryotic pre-rRNA transcription units. The three coding regions (red) encode the 18S, 5.8 S , and 28 S rRNAs found in ribosomes of higher eukaryotes, or their equivalents in other species. The order of these coding regions in the genome is always $5^{\prime} \rightarrow 3^{\prime}$. Variations in the lengths of the transcribed spacer regions (blue) account for most of the difference in the lengths of pre-rRNA transcription units among different organisms.
the processing of pre-RNA and the assembly of ribosomal subunits.

## Small Nucleolar RNAs Assist in Processing Pre-rRNAs

Ribosomal subunit assembly, maturation, and export to the cytoplasm are best understood in the yeast S. cerevisiae. However, nearly all the proteins and RNAs involved are highly conserved in multicellular eukaryotes, in which the fundamental aspects of ribosome biosynthesis are likely to be the same. Like pre-mRNAs, nascent pre-rRNA transcripts are immediately bound by proteins, forming pre-ribosomal ribonucleoprotein particles (pre-rRNPs). For reasons not yet known, cleavage of the pre-rRNA does not begin until its transcription is nearly complete. In yeast, it takes approximately 6 minutes for a pre-rRNA to be transcribed. Once transcription is complete, the pre-rRNA is cleaved, and bases and riboses are modified, in about 10 seconds. In a rapidly growing yeast cell, about 40 pairs of ribosomal subunits are synthesized, processed, and transported to the cytoplasm every second. This extremely high rate of ribosome synthesis, despite the seemingly long period required to transcribe a pre-rRNA, is possible because pre-rRNA genes are packed with RNA polymerase I molecules all transcribing the same gene simultaneously (see Figure 10-39) and because there are 100-200 such genes on chromosome XII, the yeast nucleolar organizer.

In yeast, the primary transcript of $\sim 6.6 \mathrm{~kb}$ is cut in a series of cleavage and exonucleolytic steps that ultimately yield the mature rRNAs found in ribosomes (Figure 10-41). During processing, pre-rRNA is also extensively modified, mostly by methylation of the $2^{\prime}$-hydroxyl group of specific riboses and conversion of specific uridine residues to pseudouridine. These post-transcriptional modifications of rRNA are probably important for protein synthesis because they are highly conserved. Virtually all of these modifications occur in the

![li3332239539b](li3332239539b.jpg)

FIGURE 10-41 Pre-rRNA processing in yeast. Endoribonucleases that make internal cleavages are represented as scissors. Exoribonucleases that digest from one end, either $5^{\prime}$ or $3^{\prime}$, are shown as Pac-Men. Most $2^{\prime}$-O-ribose methylation $\left(\mathrm{CH}_{3}\right)$ and generation of pseudouridines
most conserved core structure of the ribosome, which is directly involved in protein synthesis.

The positions of the specific sites of $2^{\prime}$-O-methylation and pseudouridine formation are determined by approximately 150 different small nucleolus-restricted RNA species, called small nucleolar RNAs (snoRNAs), which hybridize transiently to pre-rRNA molecules. Like the snRNAs that function in pre-mRNA processing, snoRNAs associate with proteins, forming ribonucleoprotein particles called snoRNPs. One class of more than 40 snoRNPs (containing box C+D snoRNAs) positions a methyl transferase enzyme near methylation sites in the pre-rRNA. Multiple different box C+D snoRNAs direct methylation at multiple sites through a similar mechanism. They share common sequences and structural features and are bound by a common set of
( $\Psi$ ) in the rRNAs occurs following the initial cleavage at the $3^{\prime}$ end, before the initial cleavage at the $5^{\prime}$ end. Proteins and snoRNPs known to participate in these steps are indicated. See J. Venema and D. Tollervey, 1999, Annu. Rev. Genet. 33:261.
proteins. One or two regions of each of these snoRNAs are precisely complementary to sites on the pre-rRNA and direct the methyl transferase to specific riboses in the sequences with which they hybridize (Figure 10-42a). A second major class of snoRNPs (containing box $\mathrm{H}+\mathrm{ACA}$ snoRNAs) positions the enzyme that converts uridine to pseudouridine (Figure 10-42b). This conversion involves rotation of the pyrimidine ring (Figure 10-42c). Bases on either side of the uridine to be modified in the pre-rRNA pair with bases in the bulge of a stem in the $\mathrm{H}+\mathrm{ACA}$ snoRNA, leaving the uridine bulged out of the helical double-stranded region, like the branch-point A in pre-mRNA spliceosomal splicing (see Figure 10-10). Other modifications of pre-rRNA nucleotides, such as adenine dimethylation, are carried out by specific proteins without the assistance of guiding snoRNAs.

![li3342239539b](li3342239539b.jpg)
(b)
![li3352239539b](li3352239539b.jpg)
![li3362239539b](li3362239539b.jpg)

FIGURE 10-42 snoRNP-directed modification of pre-rRNA.
(a) A class of snoRNAs called box C + D snoRNAs is involved in ribose $2^{\prime}$-O-methylation. Sequences in the snoRNA illustrated here hybridize to two different regions in the pre-rRNA, directing methylation at the indicated sites. (b) Box H+ACA snoRNAs fold into two stem-loops with
internal single-stranded bulges in the stems. Pre-rRNA hybridizes to the single-stranded bulges, demarcating a site of pseudouridylation. (c) Conversion from uridine to pseudouridine involves rotation of the pyrimidine ring. See T. Kiss, 2001, EMBO J. 20:3617.

The U3 snoRNA is assembled into a large snoRNP containing some 72 proteins, called the small subunit (SSU) processome, which specifies cleavage at site $\mathrm{A}_{0}$, the initial cut near the $5^{\prime}$ end of the pre-rRNA (see Figure 10-41). The U3 snoRNA base-pairs with an upstream region of the pre-rRNA to specify the location of the cleavage. The processome is thought to form the " $5^{\prime}$ knob" visible in electron micrographs of pre-rRNPs (see Figure10-39). Base pairing of other snoRNPs specifies additional cleavage reactions that remove transcribed spacer regions. The first cleavage to initiate processing of the yeast 5.8 S and 25 S rRNAs of the large subunit is performed by RNase MRP, a complex of nine proteins with an RNA. Once cleaved from pre-rRNAs, the spacer sequences are degraded by the same exosomeassociated $3^{\prime} \rightarrow 5^{\prime}$ nuclear exonucleases that degrade introns spliced from pre-mRNAs. Nuclear $5^{\prime} \rightarrow 3^{\prime}$ exoribonucleases (Rat1 in yeast; XRN1 in humans) also remove some regions of $5^{\prime}$ spacer.

Some snoRNAs are expressed from their own promoters by RNA polymerase II or III. Remarkably, however, the large majority of snoRNAs are processed from spliced-out introns of genes encoding functional mRNAs for proteins involved in ribosome synthesis or translation. Some snoRNAs are processed from introns spliced from apparently nonfunctional mRNAs. The genes encoding these mRNAs seem to exist only to express snoRNAs from excised introns.

Unlike 18S, 5.8S, and 28S rRNA genes, 5S rRNA genes are transcribed by RNA polymerase III in the nucleoplasm outside the nucleolus. With only minor additional processing to remove nucleotides at the $3^{\prime}$ end, 5 S rRNA diffuses to the nucleolus, where it assembles with the pre-rRNA and
remains associated with the region that is cleaved into the precursor of the large ribosomal subunit.

Most of the ribosomal proteins of the small (40S) ribosomal subunit associate with the nascent pre-rRNA during transcription (Figure 10-43). Cleavage of the full-length prerRNA in the 90 S RNP precursor of that subunit releases a pre-40S particle that requires only a few more remodeling steps before it is transported to the cytoplasm. Once the pre-40S particle leaves the nucleolus, it traverses the nucleoplasm quickly and is exported through nuclear pore complexes (NPCs), as discussed below. The final steps in the maturation of the small ribosomal subunit occur in the cytoplasm: exonucleolytic processing of the 20 S rRNA into mature small subunit 18 S rRNA by the cytoplasmic $5^{\prime} \rightarrow 3^{\prime}$ exoribonuclease XRN1, and the dimethylation of two adjacent adenines near the $3^{\prime}$ end of 18 S rRNA by the cytoplasmic enzyme Dim1.

In contrast to the pre-40S particle, the precursor of the large subunit requires considerable remodeling through many more transient interactions with nonribosomal proteins before it is sufficiently mature for export to the cytoplasm. Consequently, it takes a considerably longer time for the maturing 60 S subunit to exit the nucleus ( 30 minutes, compared with 5 minutes for export of the 40 S subunit, in cultured human cells). Multiple presumptive RNA helicases and small G proteins are associated with the maturing pre-60S subunits. Some RNA helicases are necessary to dislodge the snoRNPs, which base-pair perfectly with pre-rRNA over up to 30 base pairs. Other RNA helicases may function in the disruption of protein-RNA interactions. The requirement for so many GTPases suggests that there

![li3372239539b](li3372239539b.jpg)

FIGURE 10-43 Ribosomal subunit assembly. Ribosomal proteins and RNAs in the maturing small and large ribosomal subunits are depicted in blue, with a shape similar to the icons for the mature subunits
in the cytoplasm. Other factors that associate transiently with the maturing subunits are depicted in different colors, as shown in the key. See H. Tschochner and E. Hurt, 2003, Trends Cell Biol.13:255.
are many quality-control checkpoints in the assembly and remodeling of the large subunit RNP, in which one step must be completed before a GTPase is activated to allow the next step to proceed. Members of the AAA ATPase family are also bound transiently. This class of proteins is often involved in large molecular movements and may be required to fold the large, complex rRNA into the proper conformation. Some steps in 60 S subunit maturation occur in the nucleoplasm, during passage from the nucleolus to nuclear pore complexes (see Figure 10-43). Much remains to be learned about the complex, fascinating, and essential remodeling processes that occur during formation of the ribosomal subunits.

The large ribosomal subunit is one of the largest structures to pass through nuclear pore complexes. Maturation of the large subunit in the nucleoplasm leads to the generation of binding sites for a nuclear export adapter called Nmd3. Nmd3 is bound by the nuclear transporter exportin 1 (also called Crm1). This binding is another quality-control step because only correctly assembled subunits can bind Nmd3
and be exported. The small subunit of the mRNP exporter (Nxt1) also becomes associated with the nearly mature large ribosomal subunit. These nuclear transporters permit diffusion of the large subunit through the central channel of the NPC, which is filled with a cloud of unstructured protein domains that extend from the structured parts of the proteins that line the wall of the channel (see Chapter 13). Several additional subunits that form the walls of the NPC central channel are also required for ribosomal subunit export and may have additional functions specific for this task. The dimensions of ribosomal subunits ( $\sim 25-30 \mathrm{~nm}$ in diameter) and the central channel of the NPC are comparable, so passage may not require distortion of either the ribosomal subunit or the channel. Final maturation of the large subunit in the cytoplasm includes removal of these export factors. Like the export of most macromolecules from the nucleus, including tRNAs and pre-miRNAs (but not most mRNPs), ribosomal subunit export requires the function of a small G protein called Ran, as discussed in Chapter 13.

## Self-Splicing Group I Introns Were the First Examples of Catalytic RNA

During the 1970s, the pre-rRNA genes of the protozoan Tetrahymena thermophila were discovered to contain an intron. Careful searches failed to uncover even one pre-rRNA gene without the extra sequence, indicating that splicing is required to produce mature rRNAs in these organisms. In 1982, in vitro studies showing that the pre-rRNA is spliced at the correct sites in the absence of any protein provided the first indication that RNA can function as a catalyst, as enzymes do.

A whole raft of self-splicing sequences were subsequently found in pre-rRNAs from other single-celled organisms, in mitochondrial and chloroplast pre-rRNAs, in several premRNAs from certain E. coli bacteriophages, and in some bacterial tRNA primary transcripts. The self-splicing sequences in all these precursors, referred to as group I introns, use guanosine as a cofactor and can fold by internal base pairing to juxtapose the two exons that must be joined. As discussed earlier, certain mitochondrial and chloroplast pre-mRNAs and tRNAs contain a second type of self-splicing intron, designated group II introns.

The splicing mechanisms used by group I introns, group II introns, and spliceosomes are generally similar, involving two
transesterification reactions, which require no input of energy (Figure 10-44). Structural studies of the group I intron from Tetrahymena pre-rRNA, combined with mutational and biochemical experiments, have revealed that the RNA folds into a precise three-dimensional structure that, like an enzyme, contains deep grooves for binding substrates and solvent-inaccessible regions that function in catalysis. The group I intron functions like a metalloenzyme to precisely place the atoms that participate in the two transesterification reactions adjacent to catalytic $\mathrm{Mg}^{2+}$ ions. Considerable evidence now indicates that splicing by group II introns and by snRNAs in the spliceosome also involves bound catalytic $\mathrm{Mg}^{2+}$ ions. In both group I and group II self-splicing introns, and probably in the spliceosome, RNA functions as a ribozyme, an RNA sequence with catalytic ability.

## Pre-tRNAs Undergo Extensive Modification in the Nucleus

Mature cytosolic tRNAs, which average 75-80 nucleotides in length, are produced from larger precursors (pre-tRNAs) synthesized by RNA polymerase III in the nucleoplasm. Mature tRNAs contain numerous modified bases that are not present in tRNA primary transcripts. Cleavage and base modification occur during processing of all pre-tRNAs; some
![li3382239539b](li3382239539b.jpg)

FIGURE 10-44 Splicing mechanisms in group I and group II selfsplicing introns and in spliceosome-catalyzed splicing of pre-mRNA. The intron is shown in gray, the exons to be joined in red. In group I introns, a guanosine cofactor (G) that is not part of the RNA chain associates with the active site. The $3^{\prime}$-hydroxyl group of this guanosine participates in a transesterification reaction with the phosphate at the $5^{\prime}$ end of the intron; this reaction is analogous to that involving the

2'-hydroxyl groups of the branch-point As in group II introns and premRNA introns spliced in spliceosomes (see Figure 10-8). The subsequent transesterification that links the $5^{\prime}$ and $3^{\prime}$ exons is similar in all three splicing mechanisms. Note that spliced-out group I introns are linear structures, unlike the branched intron products in the other two cases. See P. A. Sharp, 1987, Science 235:769.

pre-tRNAs are also spliced during processing. All of these processing and modification events occur in the nucleus.

A $5^{\prime}$ sequence of variable length that is absent from mature tRNAs is present in all pre-tRNAs (Figure 10-45). These extra $5^{\prime}$ nucleotides are present because the $5^{\prime}$ end of a mature tRNA is generated by an endonucleolytic cleavage specified by the tRNA three-dimensional structure, rather than by the start site of transcription. The extra nucleotides are removed by ribonuclease P (RNase P), a ribonucleoprotein endonuclease. Studies with E. coli RNase P indicate that at high $\mathrm{Mg}^{2+}$ concentrations, its RNA component alone can recognize and cleave E. coli pre-tRNAs. The RNase P polypeptide increases the rate of cleavage by the RNA, allowing cleavage to proceed at physiological $\mathrm{Mg}^{2+}$ concentrations. A comparable RNase P functions in eukaryotes.

About 10 percent of the bases in pre-tRNAs are modified enzymatically during processing. Three classes of base modifications occur (see Figure 10-45):

1. U residues at the $3^{\prime}$ end of pre-tRNA are replaced with a CCA sequence. The CCA sequence is found at the $3^{\prime}$ end of all tRNAs and is required for their charging by aminoacyltRNA synthetases during protein synthesis. This step in tRNA synthesis probably functions as a quality-control point, since only properly folded tRNAs are recognized by the CCA addition enzyme.
2. Methyl and isopentenyl groups are added to the heterocyclic ring of purine bases, and the $2^{\prime}-\mathrm{OH}$ groups in the ribose of specific residues are methylated.
3. Specific uridines are converted to dihydrouridine, pseudouridine, or ribothymidine residues. The functions of these base and ribose modifications are not well understood, but since they are highly conserved, they probably have a positive influence on protein synthesis.

As shown in Figure 10-45, the pre-tRNA expressed from the yeast tyrosine tRNA (tRNA ${ }^{\mathrm{Tyr}}$ ) gene contains a 14-base intron that is not present in mature tRNA ${ }^{\mathrm{Tyr}}$. Some other eukaryotic tRNA genes and some archaeal tRNA genes also contain introns. The introns in nuclear pre-tRNAs are shorter than those in pre-mRNAs and lack the consensus splice-site sequences found in pre-mRNAs (see Figure 10-7). Pre-tRNA introns are also clearly distinct from the much longer self-splicing group I and group II introns found in chloroplast and mitochondrial pre-rRNAs. The mechanism of pre-tRNA splicing differs in three fundamental ways from the mechanisms used by self-splicing introns and spliceosomes (see Figure 10-44). First, splicing of pre-tRNAs is catalyzed by proteins, not by RNAs. Second, a pre-tRNA intron is excised in one step that entails simultaneous cleavage at both ends of the intron. Finally, hydrolysis of GTP and ATP is required to join the two tRNA halves generated by cleavage on either side of the intron.

After pre-tRNAs are processed in the nucleoplasm, the mature tRNAs are transported to the cytoplasm through nuclear pore complexes by exportin-t, an exportin (see Chapter 13) dedicated to the nuclear export of tRNAs. In the cytoplasm, tRNAs are passed between aminoacyltRNA synthetases, elongation factors, and ribosomes during
![li3392239539b](li3392239539b.jpg)

FIGURE 10-45 Changes that occur during the processing of tyrosine pre-tRNA. A 14-nucleotide intron (blue) in the anticodon loop is removed by splicing. A 16-nucleotide sequence (green) at the $5^{\prime}$ end is cleaved by RNase P. $U$ residues at the $3^{\prime}$ end are replaced by the CCA sequence (red) found in all mature tRNAs. Numerous bases
in the stem-loops are converted to characteristic modified bases (yellow). Not all pre-tRNAs contain introns that are spliced out during processing, but they all undergo the other types of changes shown here. $D=$ dihydrouridine; $\Psi=$ pseudouridine.

protein synthesis (see Chapter 5). Thus tRNAs are generally associated with proteins and spend little time free in the cell, as is also the case for mRNAs and rRNAs.

## Nuclear Bodies Are Functionally Specialized Nuclear Domains

High-resolution visualization of plant- and animal-cell nuclei by electron microscopy and by staining with fluorescently labeled antibodies has revealed domains in nuclei in addition to chromosome territories and nucleoli. These specialized nuclear domains, called nuclear bodies, are not surrounded by membranes. Nonetheless, they are regions of high concentrations of specific proteins and RNAs that form distinct, often roughly spherical structures within the nucleus (Figure 10-46). The most prominent nuclear bodies are nucleoli, the sites of ribosomal subunit synthesis and assembly discussed earlier. Several other types of nuclear bodies have also been described in structural studies.

Experiments with fluorescently labeled nuclear proteins have shown that the nucleus is a highly dynamic
(a)
![li3402239539b](li3402239539b.jpg)
environment, in which proteins diffuse rapidly through the nucleoplasm. Proteins associated with nuclear bodies are often also observed at lower concentrations in the nucleoplasm outside the nuclear bodies, and fluorescence studies indicate that they diffuse into and out of the nuclear bodies. Based on measurements of molecular mobility in live cells, nuclear bodies can be mathematically modeled as the expected steady state for specific diffusing proteins that interact with sufficient affinity to form self-organized regions of high concentrations, but with low enough affinity for one another to be able to diffuse into and out of these structures. In electron micrographs, these structures appear as a heterogeneous, spongelike network of interacting components. We discuss a few examples of nuclear bodies here.

Cajal Bodies Cajal bodies are $0.2-1-\mu \mathrm{m}$ spherical structures that have been observed in large nuclei for more than a century (Figure 10-46a). Current research indicates that, like nucleoli, Cajal bodies are centers of RNP-complex assembly for spliceosomal snRNPs and other RNPs. Like rRNAs, snRNAs undergo specific post-transcriptional modifications, such as the conversion of specific uridine residues to pseudouridine and the addition of methyl groups to the $2^{\prime}$-hydroxyl groups of specific riboses, that are important for the proper assembly and function of snRNPs in pre-mRNA splicing. These modifications occur in Cajal bodies, where they are directed by a class of snoRNA-like guide RNA molecules called scaRNAs (small Cajal body-associated RNAs). There is

FIGURE 10-46 Examples of nuclear bodies. (a) Cajal bodies and nucleoli in a HeLa cell nucleus. A DIC microscopic image (left) shows four nucleoli and three Cajal bodies (arrowheads). When the same nucleus is immunostained (right) with antibodies against coilin (green) and fibrillarin (red), the three Cajal bodies appear yellow because they stain with both antibodies. The nucleoli stain only for fibrillarin, which is the methyl transferase for $2^{\prime}$-O-methylation of rRNA in the nucleoli and snRNAs in the Cajal bodies. (b) Transmission electron micrograph of nuclear bodies in a single Xenopus oocyte nucleus. Only a portion of the extraordinarily large oocyte nucleus is shown. Histone locus bodies are sites of histone mRNA transcription and processing. They are larger in oocytes, which produce prodigious amounts of histone mRNAs, than in most vertebrate cells. A speckle is a region of concentrated splicing factors. (c) HeLa cell stained with DAPI (blue); antibody to SC35 (red), a splicing factor stored in speckles; and antibody to PSPC1, a protein found in nuclear bodies called paraspeckles (white arrows) because they are most often observed close to speckles. (d) PML nuclear bodies in the nucleus of an H1299 cell (a lung carcinoma cell line). DNA was stained with DAPI (blue) and PML nuclear bodies were immunostained with antibody to the major protein in these bodies, PML. (Part (a) reprinted by permission from Macmillan Publishers Ltd., from Gall, J. G., "The centennial of the Cajal body," Nat. Rev. Mol. Cell Biol., 2003, 4(12):975-980; permission conveyed through Copyright Clearance Center, Inc. Part (b) republished with permission of Elsevier, from Handwerger, K. E. and Gall, J. G., "Subnuclear organelles: new insights into form and function," Trends Cell Biol. 2006, 16(1):19-26; permission conveyed through Copyright Clearance Center, Inc. Part (c) from Fox, A. H., and Lamond, A. I., "Paraspeckles," Cold Spring Harb. Perspect. Biol., 2010, 2(7):a000687. Part (d) republished with permission of American Society for Microbiology, from Pennella, M. A., et al., "Adenovirus E1B 55-kilodalton protein is a p53-SUMO1 E3 ligase that represses p53 and stimulates its nuclear export through interactions with promyelocytic leukemia nuclear bodies," J. Virol., 2010, 84(23):12210-25.]

also evidence that Cajal bodies are sites of reassembly of the U4/U6/U5 tri-snRNP complex from the free U4, U5, and U6 snRNPs released during the removal of introns from mRNAs (see Figure 10-11).

Histone Locus Bodies Histone locus bodies (Figure 10-46b) are sites of histone mRNA synthesis. They contain a high concentration of the $U 7 \mathrm{snRNP}$ involved in the specialized $3^{\prime}$-end processing of the major histone mRNAs, which do not have a poly(A) tail.

Nuclear Speckles Nuclear speckles have been observed, using fluorescently labeled antibodies to snRNP proteins and other proteins involved in pre-mRNA splicing, as approximately $25-50$ irregular, amorphous structures $0.5-2 \mu \mathrm{~m}$ in diameter distributed through the nucleoplasm of a vertebrate cell (Figure 10-46c). Because speckles are not located at sites of co-transcriptional pre-mRNA splicing, which are associated closely with chromatin, they are thought to be storage regions for snRNPs and proteins involved in pre-mRNA splicing that are released into the nucleoplasm when required.

Nuclear Paraspeckles Paraspeckles are composed of RNPs formed by the interaction between a long nonprotein-coding RNA species (lncRNA), NEAT1, and members of the DBHS (Drosophila Behavior Human Splicing) family of proteins, P54NRB/NONO, PSPC1, and PSF/SFPQ. Paraspeckles are critical to the control of gene expression through the nuclear retention of RNA containing double-stranded RNA regions that have been subjected to adenosine-to-inosine editing. In this way, they may function in the poorly understood mRNA quality-control mechanisms that operate in the nucleus.

Promyelocytic Leukemia (PML) Nuclear Bodies The PML gene was originally discovered when chromosomal translocations within it were observed in the leukemic cells of patients with a rare disease called promyelocytic leukemia (PML). When antibodies specific for the PML protein were used in immunofluorescence microscopy studies of mammalian cells, the protein was found to localize to $10-30$ roughly spherical regions $0.3-1 \mu \mathrm{~m}$ in diameter in the cell nuclei. Multiple functions have been proposed for these PML nuclear bodies, but a consensus is emerging that they function as sites for the assembly and modification of protein complexes involved in DNA repair and the induction of apoptosis. For example, the important p53 tumor suppressor protein appears to be post-translationally modified by phosphorylation and acetylation in PML nuclear bodies in response to DNA damage, increasing its ability to activate the expression of genes whose products mitigate that damage. PML nuclear bodies are also required for cellular defenses against DNA viruses that are induced by interferons, proteins secreted by virus-infected cells and T-lymphocytes involved in the immune response (see Chapter 23).

PML nuclear bodies are also sites of post-translational modification of proteins through the addition of a small, ubiquitin-like protein called SUMO1 (small ubiquitin-like
moiety-1), which can control the activity and subcellular localization of the modified protein. Many transcriptional activators are inhibited when they are sumoylated, and mutation of their site of sumoylation increases their activity in stimulating transcription. These observations indicate that PML nuclear bodies are involved in a mechanism of transcriptional repression that remains to be thoroughly understood.

Nucleolar Functions in Addition to Ribosomal Subunit Synthesis The first nuclear bodies to be observed, the nucleoli, have specialized regions of substructure (see Figure 10-46b) that are dedicated to functions other than ribosome biogenesis. There is evidence that the signal recognition particles involved in protein secretion and ER membrane insertion (see Chapter 13) are assembled in nucleoli and then exported to the cytoplasm, where their final maturation takes place. The Cdc14 protein phosphatase that regulates processes in the final stages of mitosis in yeast is sequestered in nucleoli until chromosomes have been properly segregated into the bud (see Chapter 19). In addition, a tumor suppressor protein called ARF, which is involved in the regulation of the protein encoded by the most frequently mutated gene in human cancers, the $p 53$ gene, is sequestered in nucleoli and released in response to DNA damage (see Chapter 24). Furthermore, heterochromatin often forms on the surfaces of nucleoli (see Figure 8-28), suggesting that proteins associated with nucleoli participate in the formation of this transcriptionrepressing chromatin structure.

## KEY CONCEPTS OF SECTION 10.5

## Processing of rRNA and tRNA

- A large precursor pre-rRNA ( 13.7 kb in humans) transcribed by RNA polymerase I undergoes cleavage, exonucleolytic digestion, and base modifications to yield mature 28S, 18S, and 5.8 S rRNAs, which associate with ribosomal proteins into ribosomal subunits.
- Transcription and processing of pre-rRNA occur in the nucleolus. The 5S rRNA component of the large ribosomal subunit is synthesized in the nucleoplasm by RNA polymerase III.
- Approximately 150 snoRNAs, associated with proteins in snoRNPs, base-pair with specific sites in pre-rRNA, where they direct ribose methylation, modification of uridine to pseudouridine, and cleavage at specific sites during rRNA processing in the nucleolus.
- Group I and group II self-splicing introns, and probably snRNAs in spliceosomes, all function as ribozymes, or catalytically active RNA sequences, that carry out splicing by analogous transesterification reactions requiring bound $\mathrm{Mg}^{2+}$ ions (see Figure 10-44).
- Pre-tRNAs synthesized by RNA polymerase III in the nucleoplasm are processed by removal of the $5^{\prime}$-end sequence,

addition of CCA to the $3^{\prime}$ end, and modification of multiple internal bases (see Figure 10-45).

- Some pre-tRNAs contain a short intron that is removed by a protein-catalyzed mechanism distinct from the splicing mechanisms used by pre-mRNAs and self-splicing introns.
- All species of RNA molecules are associated with proteins in various types of ribonucleoprotein particles, both in the nucleus and after export to the cytoplasm.
- Nuclear bodies are functionally specialized regions in the nucleus where interacting proteins form self-organized structures. Many of these bodies, including the nucleolus, are regions of assembly of RNP complexes.


## LaunchPad

macmillan learning

Visit LaunchPad to access study tools and to learn more about the content in this chapter.

- Perspectives for the Future
- Analyze the Data
- Extended References
- Additional study tools, including videos, animations, and quizzes


## Key Terms

alternative splicing 417
cleavage/polyadenylation complex 431
cross-exon recognition complex 428
Dicer 448
Drosha 448
exosome 432
$5^{\prime}$ cap 419
group I
introns 429
group II introns 429
iron-response element-binding protein (IRE-BP) 455
micro-RNAs
(miRNAs) 445
mRNA surveillance 456
mRNP exporter 440
nuclear pore complex
(NPC) 440
poly(A) tail 430
pre-mRNA 421
pre-rRNA 461
ribozyme 466
RNA editing 439
RNA-induced silencing complex (RISC) 448
RNA interference (RNAi) 450
RNA splicing 419
short interfering RNAs (siRNA) 445
siRNA knockdown 451
small nuclear RNAs (snRNAs) 424
small nucleolar RNAs (snoRNAs) 463
spliceosome 426
SR proteins 428

## Review the Concepts

1. Describe three types of post-transcriptional regulation of protein-coding genes.
2. True or false?: The CTD is responsible for mRNAprocessing steps that are specific for mRNA and not for other forms of RNA. Explain why you chose true or false.
3. There are a number of conserved sequences found in an mRNA that dictate where splicing occurs. Where are these sequences found relative to the exon-intron junctions? What is the significance of these sequences in the splicing process? One of these important regions is the branch-point A found in the intron. What is the role of the branch-point A in the splicing process, and can this be accomplished with the OH group on either the $2^{\prime}$ or the $3^{\prime}$ carbon?
4. What are the differences between hnRNAs, snRNAs, miRNAs, siRNAs, and snoRNAs?
5. What are the mechanistic similarities between group II intron self-splicing and spliceosomal splicing? What is the evidence that there may be an evolutionary relationship between the two?
6. You obtain the sequence of a gene containing 10 exons, 9 introns, and a $3^{\prime}$ UTR containing a polyadenylation consensus sequence. The fifth intron also contains a polyadenylation site. To test whether both polyadenylation sites are used, you isolate mRNA and find a longer transcript from muscle tissue and a shorter transcript from all other tissues. Speculate about the mechanism involved in the production of these different transcripts.
7. RNA editing is a common process in the mitochondria of trypanosomes and plants as well as in chloroplasts, and in rare cases it occurs in higher eukaryotes. What is RNA editing, and what benefit does it demonstrate in the documented example of ApoB in humans?
8. Because DNA is found in the nucleus, transcription is a nuclear-localized process. Ribosomes responsible for protein synthesis are found in the cytoplasm. Why is hnRNP trafficking to the cytoplasm restricted to the nuclear pore complexes?
9. A protein complex in the nucleus is responsible for transporting mRNA molecules into the cytoplasm. Describe the proteins that form this exporter. What two protein groups are probably behind the mechanism involved in the directional movement of the mRNP and exporter into the cytosol?
10. RNA knockdown has become a powerful tool in the arsenal of methods used to repress gene expression. Briefly describe how gene expression can be knocked down. What effect would introducing siRNAs to TSC1 have on human cells?
11. Speculate about why plants deficient in Dicer activity show increased sensitivity to infection by RNA viruses.
12. mRNA stability is a key regulator of protein levels in a cell. Briefly describe the three mRNA degradation pathways. Suppose that a yeast cell has a mutation in the DCP1 gene,

resulting in decreased uncapping activity. Would you expect to see a change in the P bodies found in this mutant cell? 13. mRNA localization now appears to be a common phenomenon. What benefit does mRNA localization have for a cell? What is the evidence that some mRNAs are directed to accumulate in specific subcellular locations?

## References

## Processing of Eukaryotic Pre-mRNA

Bergkessel, M., G. M. Wilmes, and C. Guthrie. 2009. SnapShot: formation of mRNPs. Cell 136:794.

Hocine, S., R. H. Singer, and D. Grünwald. 2010. RNA processing and export. Cold Spring Harb. Perspect. Biol. 2(12):a000752.

Houseley, J., and D. Tollervey. 2009. The many pathways of RNA degradation. Cell 136:763-776.

Lambowitz, A. M., and S. Zimmerly. 2004. Mobile group II introns. Annu. Rev. Genet. 38:1-35.

Moore, M. J., and N. J. Proudfoot. 2009. Pre-mRNA processing reaches back to transcription and ahead to translation. Cell 136:688-700.

Sharp, P. A. 2005. The discovery of split genes and RNA splicing. Trends Biochem. Sci. 30:279-281.

Shi, Y., and J. L. Manley. 2015. The end of the message: multiple protein-RNA interactions define the mRNA polyadenylation site. Genes Dev. 29:889-897.

Valadkhan, S. 2010. Role of the snRNAs in spliceosomal active site. RNA Biol. 7:345-353.

Wahl, M. C., C. L. Will, and R. Lührmann. 2009. The spliceosome: design principles of a dynamic RNP machine. Cell 136:701-718.

## Regulation of Pre-mRNA Processing

Licatalosi, D. D., and R. B. Darnell. 2010. RNA processing and its regulation: global insights into biological networks. Nat. Rev. Genet. 11:75-87.

Maniatis, T., and B. Tasic. 2002. Alternative pre-mRNA splicing and proteome expansion in metazoans. Nature 418:236-243.

Raponi, M., and D. Baralle. 2010. Alternative splicing: good and bad effects of translationally silent substitutions. FEBS J. 277:836-840.

Wang, E. T., et al. 2008. Alternative isoform regulation in human tissue transcriptomes. Nature 456:470-476.

Zheng, S., and D. L. Black. 2013. Alternative pre-mRNA splicing in neurons: growing up and extending its reach. Trends Genet. 29:442-448.

Zhong, X. Y., et al. 2009. SR proteins in vertical integration of gene expression from transcription to RNA processing to translation. Curr. Opin. Genet. Dev. 19:424-436.

Transport of mRNA Across the Nuclear Envelope
Field, M. C., L. Koreny, and M. P. Rout. 2014. Enriching the pore: splendid complexity from humble origins. Traffic 15: $141-156$.

Folkmann, A. W., et al. 2011. Dbp5, Gle1-IP6 and Nup159: a working model for mRNP export. Nucleus 2:540-548.

Grünwald, D., R. H. Singer, and M. Rout. 2011. Nuclear export dynamics of RNA-protein complexes. Nature 475:333-341.

## Cytoplasmic Mechanisms of Post-transcriptional Control

Ambros, V. 2004. The functions of animal microRNAs. Nature 431:350-355.

Bar-Peled, L., and D. M. Sabatini. 2014. Regulation of mTORC1 by amino acids. Trends Cell Biol. 24:400-406.

Buchan, J. R., and R. Parker. 2009. Eukaryotic stress granules: the ins and outs of translation. Mol. Cell 36:932-941.

Doma, M. K., and R. Parker. 2007. RNA quality control in eukaryotes. Cell 131:660-668.

Ghildiyal, M., and P. D. Zamore. 2009. Small silencing RNAs: an expanding universe. Nat. Rev. Genet. 10:94-108.

Ivshina, M., P. Lasko, and J. D. Richter. 2014. Cytoplasmic polyadenylation element binding proteins in development, health, and disease. Annu. Rev. Cell Dev. Biol. 30:393-415.

Jonas, S., and E. Izaurralde. 2015. Towards a molecular understanding of microRNA-mediated gene silencing. Nat. Rev. Genet. 16:421-433.

Kato, M., et al. 2012. Cell-free formation of RNA granules: low complexity sequence domains form dynamic fibers within hydrogels. Cell 149:753-767.

Martin, K. C., and A. Ephrussi. 2009. mRNA localization: gene expression in the spatial dimension. Cell 136:719-730.

## Processing of rRNA and tRNA

Handwerger, K. E., and J. G. Gall. 2006. Subnuclear organelles: new insights into form and function. Trends Cell Biol. 16:19-26.

Hopper, A. K., and H. Y. Huang. 2015. Quality control pathways for nucleus-encoded eukaryotic tRNA biosynthesis and subcellular trafficking. Mol. Cell Biol. 35:2052-2058.

Januszyk, K., and C. D. Lima. 2014. The eukaryotic RNA exosome. Curr. Opin. Struct. Biol. 24:132-140.

Kressler, D., E. Hurt, and J. Bassler. 2010. Driving ribosome assembly. Biochim. Biophys. Acta 1803:673-683.

Stahley, M. R., and S. A. Strobel. 2006. RNA splicing: group I intron crystal structures reveal the basis of splice site selection and metal ion catalysis. Curr. Opin. Struct. Biol. 16:319-326.

Turowski, T. W., and D. Tollervey. 2015. Cotranscriptional events in eukaryotic ribosome synthesis. Wiley Interdiscip. Rev. RNA 6:129-139

this page left intentionally blank

![li3412239539b](li3412239539b.jpg)

Outside-in view of a bacterial aquaporin protein, which transports water and glycerol into and out of the cell, embedded in a phospholipid membrane (yellow). The four identical monomers are colored in light and dark purple; each has a channel in its center. [Data from D. Fu et al., 2000, Science 290:481-486, PDB ID 1fx8.]

In all cells, the plasma membrane forms the barrier that separates the cytoplasm from the exterior environment, thus defining a cell's physical and chemical boundaries. By preventing the unimpeded movement of molecules and ions into and out of the cell, the plasma membrane maintains essential differences between the composition of the extracellular fluid and that of the cytosol. For example, the concentration of sodium chloride $(\mathrm{NaCl})$ in the blood and extracellular fluids of animals is generally above 150 mM , similar to the $-450 \mathrm{mM} \mathrm{Na}^{+}$found in the seawater, in which all cells are thought to have evolved. In contrast, the sodium ion $\left(\mathrm{Na}^{+}\right)$ concentration in the cytosol is tenfold lower, about 15 mM , while the potassium ion $\left(\mathrm{K}^{+}\right)$concentration is higher in the cytosol than outside.

Organelle membranes, which separate the cytosol from the interior of the organelle, also form permeability barriers. For example, the proton concentration in the lysosome interior, pH 5 , is about a hundredfold greater than that of the cytosol, and many specific metabolites accumulate at higher

## OUTLINE

### 11.1 Overview of Transmembrane Transport

11.2 Facilitated Transport of Glucose and Water

### 11.3 ATP-Powered Pumps and the Intracellular Ionic Environment

concentrations in the interior of other organelles, such as the endoplasmic reticulum or the Golgi complex, than in the cytosol.

All cellular membranes, both plasma membranes and organelle membranes, consist of a bilayer of phospholipids in which other lipids and specific types of proteins are embedded. It is this combination of lipids and proteins that gives cellular membranes their distinctive permeability qualities. If cellular membranes were pure phospholipid bilayers (see Figure 10-4), they would be excellent chemical barriers, impermeable to virtually all ions, amino acids, sugars, and other water-soluble molecules. In fact, only a few gases and small, uncharged, water-soluble molecules can readily diffuse across a pure phospholipid bilayer (Figure 11-1). But cellular membranes must serve not only as barriers, but also as conduits, selectively transporting molecules and ions from one side of the membrane to the other. Energy-rich glucose, for example, must be imported into the cell, and wastes must be shipped out.

### 11.4 Nongated Ion Channels and the Resting Membrane Potential

### 11.5 Cotransport by Symporters and Antiporters

11.6 Transcellular Transport