![li014542480e](li014542480e.jpg)

Outside-in view of a bacterial aquaporin protein, which transports water and glycerol into and out of the cell, embedded in a phospholipid membrane (yellow). The four identical monomers are colored in light and dark purple; each has a channel in its center. [Data from D. Fu et al., 2000, Science 290:481-486, PDB ID 1fx8.]

In all cells, the plasma membrane forms the barrier that separates the cytoplasm from the exterior environment, thus defining a cell's physical and chemical boundaries. By preventing the unimpeded movement of molecules and ions into and out of the cell, the plasma membrane maintains essential differences between the composition of the extracellular fluid and that of the cytosol. For example, the concentration of sodium chloride $(\mathrm{NaCl})$ in the blood and extracellular fluids of animals is generally above 150 mM , similar to the $-450 \mathrm{mM} \mathrm{Na}^{+}$found in the seawater, in which all cells are thought to have evolved. In contrast, the sodium ion $\left(\mathrm{Na}^{+}\right)$ concentration in the cytosol is tenfold lower, about 15 mM , while the potassium ion $\left(\mathrm{K}^{+}\right)$concentration is higher in the cytosol than outside.

Organelle membranes, which separate the cytosol from the interior of the organelle, also form permeability barriers. For example, the proton concentration in the lysosome interior, pH 5 , is about a hundredfold greater than that of the cytosol, and many specific metabolites accumulate at higher

## OUTLINE

### 11.1 Overview of Transmembrane Transport

11.2 Facilitated Transport of Glucose and Water

### 11.3 ATP-Powered Pumps and the Intracellular Ionic Environment

concentrations in the interior of other organelles, such as the endoplasmic reticulum or the Golgi complex, than in the cytosol.

All cellular membranes, both plasma membranes and organelle membranes, consist of a bilayer of phospholipids in which other lipids and specific types of proteins are embedded. It is this combination of lipids and proteins that gives cellular membranes their distinctive permeability qualities. If cellular membranes were pure phospholipid bilayers (see Figure 10-4), they would be excellent chemical barriers, impermeable to virtually all ions, amino acids, sugars, and other water-soluble molecules. In fact, only a few gases and small, uncharged, water-soluble molecules can readily diffuse across a pure phospholipid bilayer (Figure 11-1). But cellular membranes must serve not only as barriers, but also as conduits, selectively transporting molecules and ions from one side of the membrane to the other. Energy-rich glucose, for example, must be imported into the cell, and wastes must be shipped out.

### 11.4 Nongated Ion Channels and the Resting Membrane Potential

### 11.5 Cotransport by Symporters and Antiporters

11.6 Transcellular Transport

![li024542480e](li024542480e.jpg)

FIGURE 11-1 Relative permeability of a pure phospholipid bilayer to various molecules and ions. A pure phospholipid bilayer is permeable to many gases and to small, uncharged, water-soluble (polar) molecules. It is slightly permeable to water, and essentially impermeable to ions and to large polar molecules.

Movement of virtually all small molecules and ions across cellular membranes is mediated by membrane transport proteins-integral membrane proteins with multiple transmembrane domains embedded in cellular membranes. These membrane-spanning proteins act variously as shuttles, channels, or pumps for transporting molecules and ions through a membrane's hydrophobic interior. In some cases, molecules or ions are transported from a higher to a lower concentration, a thermodynamically favored process powered by an increase in entropy. Examples include the transport of water or glucose from the blood into most body cells. In other cases, molecules or ions must be pumped from a lower to a higher concentration, a thermodynamically unfavorable process that can occur only when an external source of energy is available to push the molecules "uphill" against a concentration gradient. An example of such a process is the concentration of protons within lysosomes to generate a low pH in the lumen. Often the required energy is provided by mechanistic coupling of the energy-releasing hydrolysis of the terminal phosphoanhydride bond in ATP with the movement of a molecule or ion across the membrane. Other proteins couple the movement of one molecule or ion against its concentration gradient with the movement of another down its gradient, using the energy released by the downhill movement of one molecule or ion to drive the uphill movement of another. Proper functioning of any cell relies on a precise balance between such import and export of various molecules and ions.

We begin our discussion of membrane transport proteins by reviewing some of the general principles of transport
across membranes and distinguishing between three major classes of such proteins. In subsequent sections, we describe the structure and operation of specific examples of each class and show how members of families of homologous transport proteins have different properties that enable different cell types to function appropriately. We also explain how specific combinations of transport proteins in both the plasma membrane and organelle membranes enable cells to carry out essential physiological processes, including the maintenance of cytosolic pH , the accumulation of sucrose and salts in plant cell vacuoles, and direction of the flow of water in both plants and animals. The cell's resting membrane potential is an important consequence of selective ion transport across membranes, and we consider how this potential arises. Epithelial cells, such as those lining the small intestine, use a combination of membrane transport proteins to transport ions, sugars and other small molecules, and water from one side of the cell to the other. We will see how our understanding of this process has led to the development of sports drinks as well as therapies for cholera and other diarrheal diseases.

Note that in this chapter we cover only transport of small molecules and ions; transport of larger molecules, such as proteins and oligosaccharides, is covered in Chapters 13 and 14 .

### 11.1 Overview of Transmembrane Transport

In this section, we first describe the factors that influence the permeability of lipid membranes, then briefly describe the three major classes of membrane transport proteins that allow molecules and ions to cross them. Different kinds of membrane-embedded proteins accomplish the task of moving molecules and ions in different ways.

## Only Gases and Small Uncharged Molecules Cross Membranes by Simple Diffusion

With its dense hydrophobic core, a phospholipid bilayer is largely impermeable to water-soluble molecules and ions. Only gases, such as $\mathrm{O}_{2}$ and $\mathrm{CO}_{2}$, and small uncharged polar molecules, such as urea and ethanol, can readily move across an artificial membrane composed of pure phospholipid or of phospholipid and cholesterol (see Figure 11-1). Such molecules can also diffuse across cellular membranes without the aid of transport proteins. No metabolic energy is expended during simple diffusion because movement is from a high to a low concentration of the molecule, down its chemical concentration gradient. As noted in Chapter 2, such movements are spontaneous because they have a positive $\Delta S$ value (increase in entropy) and thus a negative $\Delta G$ (decrease in free energy).

The diffusion rate of any substance across a pure phospholipid bilayer is proportional to its concentration gradient across the bilayer and to its hydrophobicity and size; the

movement of charged molecules is also affected by any electric potential across the membrane. When a pure phospholipid bilayer separates two aqueous spaces, or "compartments," membrane permeability can be easily determined by adding a small amount of labeled material to one compartment and measuring its rate of appearance in the other compartment. The label can be radioactive or nonradioactive-for example, a fluorescent label whose light emission can be measured. The greater the concentration gradient of the substance, the faster its rate of movement across a bilayer.

The hydrophobicity of a substance is determined by measuring its partition coefficient $K$, the equilibrium constant for its partition between oil and water. The higher a substance's partition coefficient (the greater the fraction found in oil relative to water), the more lipid soluble it is, and therefore, the faster its rate of movement across a bilayer. The first and rate-limiting step in transport by simple diffusion is movement of a molecule from the aqueous solution into the hydrophobic interior of the phospholipid bilayer, which resembles olive oil in its chemical properties. This is the reason that the more hydrophobic a molecule is, the faster it diffuses across a pure phospholipid bilayer. For example, diethylurea, with an ethyl group attached to each nitrogen atom:

$$
\mathrm{CH}_{3}-\mathrm{CH}_{2}-\mathrm{NH}-\stackrel{\mathrm{O}}{\mathrm{C}}-\mathrm{NH}-\mathrm{CH}_{2}-\mathrm{CH}_{3}
$$

has a $K$ of 0.01 , whereas urea

$$
\mathrm{NH}_{2}-\stackrel{\mathrm{O}}{\mathrm{C}}-\mathrm{NH}_{2}
$$

has a $K$ of 0.0002 . Diethylurea, which is 50 times (0.01/0.0002) more hydrophobic than urea, will therefore diffuse through a pure phospholipid bilayer about 50 times faster than urea. Similarly, fatty acids with longer hydrocarbon chains are more hydrophobic than those with shorter
chains and at all concentrations will diffuse more rapidly across a pure phospholipid bilayer.

If a substance carries a net charge, its movement across a membrane is influenced by both its concentration gradient and the membrane potential, the electric potential (voltage) across the membrane. The combination of these two forces, called the electrochemical gradient, determines the energetically favorable direction of movement of a charged molecule across a membrane. The electric potential that exists across most cellular membranes results from a small imbalance in the concentrations of positively and negatively charged ions on the two sides of the membrane. We discuss how this ionic imbalance, and the resulting potential, arise and are maintained in Sections 11.4 and 11.5.

## Three Main Classes of Membrane Proteins Transport Molecules and Ions Across Cellular Membranes

As is evident from Figure 11-1, very few molecules and no ions can cross a pure phospholipid bilayer at appreciable rates by simple diffusion. Thus transport of most molecules into and out of cells requires the assistance of specialized membrane proteins. Even in the cases of molecules with relatively large partition coefficients (e.g., urea, fatty acids) and certain gases, such as $\mathrm{CO}_{2}$ (carbon dioxide) and $\mathrm{NH}_{3}$ (ammonia), transport is frequently accelerated by specific proteins because simple diffusion does not occur rapidly enough to meet cellular needs.

All membrane transport proteins are transmembrane proteins containing multiple membrane-spanning segments that are generally $\alpha$ helices. By forming a protein-lined pathway across the membrane, transport proteins are thought to allow hydrophilic substances to move through the membrane without coming into contact with its hydrophobic interior. Here we introduce the three main types of membrane transport proteins covered in this chapter (Figure 11-2).
![li034542480e](li034542480e.jpg)

FIGURE 11-2 Overview of membrane transport proteins. Gradients are indicated by triangles with the tip pointing toward lower concentration, electric potential, or both. $\square$ Channels permit movement of specific ions (or water) down their electrochemical gradient. $\square$ Transporters, which fall into three groups, facilitate movement of specific small molecules or ions. Uniporters transport a single type of molecule down its concentration gradient $\boldsymbol{\square}$. Cotransport proteins (symporters. $\boldsymbol{\square}$.
and antiporters, $\boldsymbol{\Delta}$ catalyze the movement of one molecule against its concentration gradient (black circles), driven by movement of one or more ions down an electrochemical gradient (red circles). Pumps use the energy released by ATP hydrolysis to power movement of specific ions or small molecules (red circles) against their electrochemical gradient. Differences in the mechanisms of transport by these three major classes of proteins account for their varying rates of solute movement.

Channels transport water, specific ions, or hydrophilic small molecules across membranes down their concentration or electric potential gradients. Because this process requires transport proteins but not energy, it is sometimes referred to as passive transport or facilitated diffusion, but it is more properly called facilitated transport. Channels form a hydrophilic "tube" or passageway across the membrane through which multiple water molecules or ions move simultaneously, single file, at a very rapid rate. Some channels are open much of the time; they are referred to as nongated channels. Most ion channels, however, open only in response to specific chemical or electrical signals. These channels are referred to as gated channels because a protein "gate" alternatively blocks the channel or moves out of the way to open the channel (see Figure 11-2). Channels, like all transport proteins, are very selective for the type of molecule they transport.

Transporters (also called carriers) move a wide variety of ions and molecules across cellular membranes, but at a much slower rate than channels. Three types of transporters have been identified. Uniporters transport a single type of molecule down its concentration gradient. Glucose and amino acids cross the plasma membrane into most mammalian cells with the aid of uniporters. Collectively, channels and uniporters are sometimes called facilitated transporters, indicating movement down a concentration or electrochemical gradient.

In contrast, antiporters and symporters couple the movement of one type of ion or molecule against its concentration gradient with the movement of one or more different ions down its concentration gradient, in the same (symporter) or different (antiporter) directions. These proteins are often called cotransporters because of their ability to transport two or more different solutes simultaneously.

ATP-powered pumps (or simply pumps) are ATPases that use the energy of ATP hydrolysis to move ions or small molecules across a membrane against a chemical concentration gradient, an electric potential, or both. This process, referred to as active transport, is an example of coupled chemical reactions (see Chapter 2). In this case, transport of ions or small molecules "uphill" against an electrochemical gradient, which requires energy, is coupled to the hydrolysis of ATP, which releases energy. The overall reaction-ATP hydrolysis and the "uphill" movement of ions or small molecules-is energetically favorable.

Like cotransporters, pumps mediate coupled reactions in which an energetically unfavorable reaction (i.e., the uphill movement of one type of molecule or ion) is coupled to an energetically favorable reaction (i.e., the downhill movement of another). Note, however, that the nature of the energy-supplying reaction driving active transport by these two classes of proteins differs. ATP pumps use energy from hydrolysis of ATP, whereas cotransporters use the energy stored in an electrochemical gradient. The latter process is sometimes referred to as secondary active transport.

Conformational changes are essential to the function of all transport proteins. ATP-powered pumps and transporters
undergo a cycle of conformational change exposing a binding site (or sites) to one side of the membrane in one conformation and to the other side in a second conformation. Because each such cycle results in the movement of only one substrate molecule (or, at most, a few), these proteins are characterized by relatively slow rates of transport, ranging from $10^{0}$ to $10^{4}$ ions or molecules per second (see Figure 11-2). Most ion channels shuttle between a closed state and an open state, but many ions can pass through an open channel without any further conformational change. For this reason, channels are characterized by very fast rates of transport, up to $10^{8}$ ions per second.

Frequently, several different types of transport proteins work in concert to achieve a physiological function. An example is seen in Figure 11-3, where an ATPase pumps $\mathrm{Na}^{+}$out of the cell and $\mathrm{K}^{+}$inward; this pump, which is found in virtually all metazoan cells, establishes the oppositely directed concentration gradients of $\mathrm{Na}^{+}$and $\mathrm{K}^{+}$ions across the plasma membrane (relatively high concentrations of $\mathrm{K}^{+}$inside and $\mathrm{Na}^{+}$outside cells) that are used to power the import of amino acids. The human genome encodes hundreds of different types of transport proteins that use the energy stored across the plasma membrane in the $\mathrm{Na}^{+}$
![li044542480e](li044542480e.jpg)

FIGURE 11-3 Multiple membrane transport proteins function together in the plasma membrane of metazoan cells. Gradients are indicated by triangles with the tip pointing toward lower concentration. The $\mathrm{Na}^{+} / \mathrm{K}^{+}$ATPase in the plasma membrane uses energy released by ATP hydrolysis to pump $\mathrm{Na}^{+}$(red circles) out of the cell and $\mathrm{K}^{+}$(blue squares) inward; this creates a concentration gradient of $\mathrm{Na}^{+}$ that is greater outside the cell than inside, and one of $\mathrm{K}^{+}$that is greater inside than outside. Movement of positively charged $\mathrm{K}^{+}$ions out of the cell through membrane $\mathrm{K}^{+}$channels creates an electric potential across the plasma membrane-the cytosolic face is negative with respect to the extracellular face. A $\mathrm{Na}^{+}$/lysine transporter, a typical sodium/amino acid cotransporter, moves two $\mathrm{Na}^{+}$ions together with one lysine from the extracellular medium into the cell. "Uphill" movement of the amino acid is powered by "downhill" movement of $\mathrm{Na}^{+}$ions, which in turn is powered both by the outside-greater-than-inside $\mathrm{Na}^{+}$concentration gradient and by the negative charge on the inside of the plasma membrane, which attracts the positively charged $\mathrm{Na}^{+}$ions. The ultimate source of the energy to power amino acid uptake comes from the ATP hydrolyzed by the $\mathrm{Na}^{+} / \mathrm{K}^{+}$ATPase, since this pump creates both the $\mathrm{Na}^{+}$ion concentration gradient and, via the $\mathrm{K}^{+}$channels, the membrane potential, which together power the influx of $\mathrm{Na}^{+}$ions.

TABLE 11-1 Mechanisms for Transporting Ions and Small Molecules Across Cellular Membranes

| Property | Simple Diffusion | Facilitated Transport | Active Transport | Cotransport* |
| :--: | :--: | :--: | :--: | :--: |
| Requires specific protein | $-$ | $+$ | $+$ | $+$ |
| Solute transported against its gradient | $-$ | $-$ | $+$ | $+$ |
| Coupled to ATP hydrolysis | $-$ | $-$ | $+$ | $-$ |
| Driven by movement of a cotransported ion down its gradient | $-$ | $-$ | $-$ | $+$ |
| Examples of molecules transported | $\mathrm{O}_{2}, \mathrm{CO}_{2}$, steroid hormones, many drugs | Glucose and amino acids (uniporters); ions and water (channels) | Ions, small hydrophilic molecules, lipids (ATP-powered pumps) | Glucose and amino acids (symporters); various ions and sucrose (antiporters) |

*Also called secondary active transport.
concentration gradient and its associated electric potential to transport a wide variety of molecules into cells against their concentration gradients.

Table 11-1 summarizes the four mechanisms by which small molecules and ions are transported across cellular membranes. In the next section, we consider some of the simplest membrane transport proteins, those responsible for the transport of glucose and water.

## KEY CONCEPTS OF SECTION 11.1

## Overview of Transmembrane Transport

- Cellular membranes regulate the traffic of molecules and ions into and out of cells and their organelles. The rate of simple diffusion of a substance across a membrane is proportional to its concentration gradient and hydrophobicity.
- With the exception of gases (e.g., $\mathrm{O}_{2}$ and $\mathrm{CO}_{2}$ ) and small, uncharged, water-soluble molecules, most molecules cannot diffuse across a pure phospholipid bilayer at rates sufficient to meet cellular needs.
- Membrane transport proteins provide a hydrophilic passageway for molecules and ions to travel through the hydrophobic interior of a membrane.
- Three classes of transmembrane proteins mediate transport of ions, sugars, amino acids, and other metabolites across cellular membranes: channels, transporters, and ATPpowered pumps (see Figure 11-2).
- Channels form a hydrophilic "tube" through which water or ions move down a concentration gradient, a process known as facilitated transport.
- Transporters fall into three groups. Uniporters transport a molecule down its concentration gradient (facilitated transport); symporters and antiporters couple movement of a substrate against its concentration gradient to the movement of a second substrate down its concentration gradient, a process known as secondary active transport or cotransport (see Table 11-1).
- ATP-powered pumps couple the movement of a substrate against its concentration gradient to ATP hydrolysis, a process known as active transport.
- Conformational changes are essential to the function of all membrane transport proteins; speed of transport depends on the number of substrate molecules or ions that can pass through a protein at once.


### 11.2 Facilitated Transport of Glucose and Water

Most animal cells use glucose as a substrate for ATP production; they usually employ a glucose uniporter to take up glucose from the blood or other extracellular fluid. Many cells use channel-like membrane transport proteins called aquaporins to increase the rate of water movement across their plasma membranes. Here we discuss the structure and function of these and other facilitated transporters.

## Uniport Transport Is Faster and More Specific than Simple Diffusion

The protein-mediated transport of a single type of molecule, such as glucose or another small hydrophilic molecule, down a concentration gradient across a cellular membrane is

known as uniport. Several features distinguish uniport from simple diffusion:

1. The rate of substrate movement by uniporters is far higher than simple diffusion through a pure phospholipid bilayer.
2. Because the transported molecule never enters the hydrophobic core of the phospholipid bilayer, its partition coefficient $K$ is irrelevant.
3. Transport occurs via a limited number of uniporter molecules. Consequently, there is a maximum transport rate, $V_{\text {max }}$, which depends on the number of uniporters in the membrane. $V_{\text {max }}$ is achieved when the concentration gradient across the membrane is very large and each uniporter is working at its maximal rate.
4. Transport is reversible, and the direction of transport will change if the direction of the concentration gradient changes.
5. Transport is specific. Each uniporter transports only a single type of molecule or a single group of closely related molecules. A measure of the affinity of a transporter for its substrate is the Michaelis constant, $K_{\mathrm{m}}$, which is the concentration of substrate at which transport is half $V_{\text {max }}$.

These properties also apply to transport mediated by the other classes of proteins depicted in Figure 11-2.

One of the best-understood uniporters is the glucose transporter called GLUT1, found in the plasma membrane of most mammalian cells. GLUT1 is especially abundant in the erythrocyte (red blood cell) plasma membrane. Because erythrocytes have a single membrane and no nucleus or other internal organelles (see Figure 7-7a), it is relatively simple to isolate and purify their plasma-membrane transport proteins. As a result, the properties of GLUT1 and many other transport proteins from mature erythrocytes have been extensively studied. In addition, the three-dimensional structure of human GLUT1 was solved in 2014, providing further molecular insights into the details of GLUT1 function.

Figure 11-4 shows that glucose uptake by erythrocytes and liver cells exhibits kinetics similar to those of a simple enzyme-catalyzed reaction involving a single substrate. The kinetics of transport reactions mediated by other types of proteins are more complicated than those for uniporters. Nonetheless, all protein-assisted transport reactions occur faster than simple diffusion across the bilayer, are substratespecific, and exhibit a maximal rate $\left(V_{\max }\right)$.

## The Low $K_{\mathrm{m}}$ of the GLUT1 Uniporter Enables It to Transport Glucose into Most Mammalian Cells

Like other uniporters, GLUT1 alternates between two conformational states: in one, a glucose-binding site faces the outside of the cell; in the other, a glucose-binding site faces the cytosol. The latter conformation has been solved at high resolution, as shown Figure 11-5a. Since the glucose concentration is usually higher in the extracellular medium (blood, in the case of
![li054542480e](li054542480e.jpg)

EXPERIMENTAL FIGURE 11-4 Cellular uptake of glucose mediated by GLUT proteins exhibits simple enzyme kinetics. The initial rate of glucose uptake, $v$ (measured as micromoles per milliliter of cells per hour), in the first few seconds is plotted as a percentage of the maximum rate, $V_{\text {max }}$, against increasing glucose concentration in the extracellular medium. In this experiment, the initial concentration of glucose in the cells is always zero. Both GLUT1, expressed by erythrocytes, and GLUT2, expressed by liver cells, catalyze glucose uptake. Like enzyme-catalyzed reactions, GLUT-facilitated uptake of glucose exhibits a maximum rate $\left(V_{\max }\right) . K_{\mathrm{m}}$ is the concentration at which the rate of glucose uptake is half maximal. GLUT2, with a $K_{\mathrm{m}}$ of about 20 mM (not shown), has a much lower affinity for glucose than GLUT1, with a $K_{\mathrm{m}}$ of about 1.5 mM .
erythrocytes) than in the cell, the GLUT1 uniporter generally catalyzes the net import of glucose from the extracellular medium into the cell. Figure 11-5b depicts the sequence of events during the unidirectional transport of glucose from the cell exterior inward to the cytosol through a mechanism known as the alternating access model; note the conformational changes in several of the membrane-spanning $\alpha$ helices during this process. GLUT1 can also catalyze the net export of glucose from the cytosol to the extracellular medium when the glucose concentration is higher inside the cell than outside.

The kinetics of the unidirectional transport of glucose from the outside of a cell inward via GLUT1 can be described by the same type of equation used to describe a simple enzyme-catalyzed chemical reaction. For simplicity, let's assume that the substrate (glucose), S, is present initially only on the outside of the cell; this can be achieved by first incubating cells in a medium lacking glucose so that their internal stores are depleted. In this case, we can write

$$
\mathrm{S}_{\text {out }}+\mathrm{GLUT} 1 \stackrel{K_{\mathrm{m}}}{\rightleftharpoons} \mathrm{~S}_{\text {out }}-\mathrm{GLUT} 1 \stackrel{V_{\max }}{\rightleftharpoons} \mathrm{~S}_{\text {in }}+\mathrm{GLUT} 1
$$

where $\mathrm{S}_{\text {out }}-$ GLUT1 represents GLUT1 in the outward-facing conformation with a bound glucose. This equation is similar to the one describing the path of a simple enzyme-catalyzed reaction in which the protein binds a single substrate and then transforms it into a different molecule. Here, however, no chemical modification of the GLUT1-bound glucose molecule occurs; rather, it is moved across a cellular membrane.

![li064542480e](li064542480e.jpg)

FIGURE 11-5 The human GLUT1 uniporter transports glucose across cellular membranes. (a) Structural model (side view) of the full-length human GLUT1 protein in an inward-open conformation. The transporter consists of 12 transmembrane $\alpha$-helical segments, which are organized into amino-terminal and carboxy-terminal domains, each of which consists of a pair of three transmembrane $\alpha$ helices. The corresponding transmembrane segments in one set of the four three-helix repeats are colored orange in the model on the left. The amino-terminal and carboxy-terminal domains are connected by intracellular and extracellular $\alpha$ helices, which are colored green and purple, respectively. A section of a cut-open view of the surface electrostatic potential highlights the central cavity that transports glucose (red) across the membrane. The colors represent the

Nonetheless, the kinetics of this transport reaction are similar to those of simple enzyme-catalyzed reactions, and we can use the same derivation as that of the Michaelis-Menten equation in Chapter 3 to derive the following expression for $v_{0}$, the initial transport rate for $S$ into the cell catalyzed by GLUT1:

$$
v_{0}=\frac{V_{\max }}{1+\frac{K_{\mathrm{m}}}{C}}
$$

where $C$ is the concentration of $\mathrm{S}_{\text {out }}$ (initially, the concentration of $\mathrm{S}_{\mathrm{m}}=0$ ). $V_{\text {max }}$, the rate of transport when all molecules of GLUT1 contain a bound S, occurs at an infinitely high $\mathrm{S}_{\text {out }}$ concentration. The lower the value of $K_{\mathrm{m}}$, the more tightly the substrate binds to the transporter. Equation 11-1 describes the curve for glucose uptake by erythrocytes shown in Figure 11-4 as well as similar curves for other uniporters.

For GLUT1 in the human erythrocyte membrane, the $K_{\mathrm{m}}$ for glucose transport is 1.5 mM . Thus when the extracellular glucose concentration is 1.5 mM , roughly half the GLUT1 transporters with outward-facing binding sites will have a bound glucose, and transport will occur at 50 percent of the maximal rate. Blood glucose is normally 5 mM , so the erythrocyte glucose transporter is usually functioning at 77 percent of its maximal rate, as can be seen from Equation 11-1. The GLUT1 transporter (or the very similar GLUT3 glucose transporter) is expressed by all body cells that need to take
![li074542480e](li074542480e.jpg)
hydrophobicity of the amino acids, with hydrophobic in yellow and hydrophilic in blue. (b) A working model for GLUT1. In this alternating access model, the outward-open conformation of GLUT1 binds glucose (step $\mathbf{E}$ ) and moves to a ligand-bound occluded conformation (step $\mathbf{E}$ ) before changing to its inward-open conformation (step $\mathbf{E}$ ) when it delivers glucose to the cytoplasm, then moves through a ligandfree occluded conformation (step $\mathbf{E}$ ) before beginning another round of glucose transport from outside to inside the cell. If the concentration of glucose is higher inside the cell than outside, the cycle will work in reverse (step $\mathbf{E} \rightarrow$ step $\mathbf{E}$ ), resulting in net movement of glucose out of the cell. The actual conformational changes are probably smaller than those depicted here. [Part (a) data from D. Deng et al., 2014, Nature 510:121-125, PDB ID:4pyp.]
up glucose from the blood continuously at high rates. The rate of glucose uptake by such cells remains high regardless of small changes in the concentration of blood glucose because the blood concentration remains much higher than the $K_{\mathrm{m}}$ and the intracellular glucose concentration is kept low by metabolism.

In addition to glucose, the isomeric sugars D-mannose and D-galactose, which differ from D-glucose in their configuration at only one carbon atom, are transported by GLUT1 at measurable rates. However, the $K_{\mathrm{m}}$ for glucose $(1.5 \mathrm{mM})$ is much lower than it is for D-mannose $(20 \mathrm{mM})$ or D-galactose ( 30 mM ). Thus GLUT1 is quite specific, having a much higher affinity (indicated by a lower $K_{\mathrm{m}}$ ) for its normal substrate D-glucose than for other substrates.

GLUT1 accounts for 2 percent of the protein in the plasma membrane of erythrocytes. After glucose is transported into the erythrocyte, it is rapidly phosphorylated, forming glucose-6-phosphate, which cannot leave the cell. Because this reaction, the first step in the metabolism of glucose (see Figure 12-3), is rapid and occurs at a constant rate, the intracellular concentration of glucose is kept low even when glucose is imported from the extracellular environment. Consequently, the concentration gradient of glucose (outside greater than inside the cell) is kept sufficiently high to support continuous, rapid import of additional glucose molecules and provide sufficient glucose for cellular metabolism.

## The Human Genome Encodes a Family of Sugar-Transporting GLUT Proteins

The human genome encodes at least 14 highly homologous GLUT proteins, GLUT1-GLUT14, that are all thought to contain 12 membrane-spanning $\alpha$ helices, suggesting that they evolved from a single ancestral transport protein. In the human GLUT1 protein, the transmembrane $\alpha$ helices are predominantly hydrophobic; several helices, however, bear amino acid residues (e.g., serine, threonine, asparagine, and glutamine) whose side chains can form hydrogen bonds with the hydroxyl groups on glucose. These residues are thought to form the inward-facing and outwardfacing glucose-binding sites in the interior of the protein (see Figure 11-5).

The structures of all GLUT isoforms are thought to be quite similar, and all of them transport sugars. Nonetheless, their differential expression in various cell types, the regulation of their numbers on cell surfaces, and isoform-specific functional properties enable different body cells to regulate glucose metabolism differently and at the same time allow a constant concentration of glucose in the blood to be maintained. For instance, GLUT3 is found in neuronal cells of the brain. Neurons depend on a constant influx of glucose for metabolism, and the low $K_{\mathrm{m}}$ of GLUT3 for glucose ( 1.5 mM ), like that of GLUT1, ensures that these cells incorporate glucose from brain extracellular fluids at a high and constant rate.

GLUT2, expressed in liver cells and in the insulin-secreting $\beta$ islet cells of the pancreas, has a $K_{\mathrm{m}}$ of $\sim 20 \mathrm{mM}$, about 13 times higher than the $K_{\mathrm{m}}$ of GLUT1. As a result, when blood glucose rises after a meal from its basal level of 5 mM to 10 mM or so, the rate of glucose influx will almost double in GLUT2-expressing cells, whereas it will increase only slightly in GLUT1-expressing cells (see Figure 11-4). In the liver, the "excess" glucose brought into the cell is stored as the polymer glycogen. In $\beta$ islet cells, the rise in glucose triggers secretion of the hormone insulin (see Figure 16-39), which in turn lowers blood glucose by increasing glucose uptake and metabolism in muscle and by inhibiting glucose production in the liver (see Figure 15-37). Indeed, cell-specific inactivation of GLUT2 in pancreatic $\beta$ islet cells prevents glucose-stimulated insulin secretion and disrupts the regulated expression of glucose-sensitive genes in liver cells (hepatocytes).

Another GLUT isoform, GLUT4, is expressed only in fat and muscle cells, which respond to insulin by increasing their uptake of glucose, thereby removing glucose from the blood. In the absence of insulin, GLUT4 resides in intracellular membranes, not the plasma membrane, and is unable to facilitate glucose uptake from the extracellular fluid. By a process detailed in Figure 16-40, insulin causes these GLUT4rich internal membranes to fuse with the plasma membrane, increasing the number of GLUT4 molecules present on the cell surface and thus the rate of glucose uptake. This is one principal mechanism by which insulin lowers blood glucose; defects in the movement of GLUT4 to the plasma membrane are one of the causes of adult-onset, or type II, diabetes, a disease marked by continuously high blood glucose.

GLUT5 is the only GLUT protein with a high specificity (preference) for fructose; its principal site of expression is the apical membrane of intestinal epithelial cells, where it transports dietary fructose from the intestinal lumen to the inside of the cells.

## Transport Proteins Can Be Studied Using Artificial Membranes and Recombinant Cells

There are a variety of approaches to studying the intrinsic properties of transport proteins, such as the $V_{\max }$ and $K_{\mathrm{m}}$ parameters and the key residues responsible for binding. Most cellular membranes contain many different types of transport proteins but a relatively low concentration of any particular one, which makes functional studies of a single protein difficult. To facilitate such studies, researchers use two approaches to enrich a transport protein of interest so that it predominates in the membrane: purification and insertion into artificial membranes, and overexpression in recombinant cells.

In the first approach, a specific transport protein is extracted from its membrane with detergent and purified. Although transport proteins can be isolated from membranes and purified, their functional properties (i.e., their role in the movement of substrates across membranes) can be studied only when they are associated with a membrane. Thus the purified proteins are usually reincorporated into pure phospholipid bilayer membranes, such as liposomes (see Figure 7-3), across which substrate transport can be readily measured. One good source of GLUT1 is erythrocyte membranes. Another is recombinant cultured mammalian cells that express a GLUT1 transgene, often one that encodes a modified GLUT1 that contains an epitope tag [a portion of a molecule to which a monoclonal antibody (see Chapter 4) can bind] fused to its N - or C -terminus. All of the integral membrane proteins in either of these two types of cells can be extracted by using a non-ionic detergent such as octylglucoside, which solubilizes the membrane without significantly denaturing the membrane proteins. The glucose uniporter GLUT1 can be purified from the solubilized mixture by antibody affinity chromatography (see Chapter 3) on a column containing either a GLUT1-specific monoclonal antibody or an antibody specific for the epitope tag, then incorporated into liposomes made of pure phospholipids.

Alternatively, the gene encoding a specific transport protein can be expressed at high levels in a cell type that normally does not express it. The difference between the transport rate of a substance by the transfected cells and by control nontransfected cells will be due to the expressed transport protein. In these systems, the functional properties of the various membrane proteins can be examined without ambiguity caused, for instance, by partial protein denaturation during isolation and purification procedures. As an example, overexpressing GLUT1 in lines of cultured fibroblasts increases their rate of uptake of glucose severalfold, and expression of mutant GLUT1 proteins with specific amino acid alterations can identify residues important for substrate binding.

## Osmotic Pressure Causes Water to Move Across Membranes

Movement of water into and out of cells is an important feature of the life of all organisms. The aquaporins are a family of membrane proteins that allow water and a few other small uncharged molecules, such as glycerol, to cross cellular membranes efficiently. But before discussing these transport proteins, we need to review osmosis, the force that powers the movement of water across membranes.

Water spontaneously moves "downhill" across a semipermeable membrane from a solution of lower solute concentration (relatively high water concentration) to one of higher solute concentration (relatively low water concentration), a process termed osmosis, or osmotic flow. In effect, osmosis is equivalent to "diffusion" of water across a semipermeable membrane. Osmotic pressure is defined as the hydrostatic pressure required to stop the net flow of water across a membrane separating solutions of different water concentrations (Figure 11-6). In other words, osmotic pressure balances the entropy-driven thermodynamic force of the water concentration gradient. In this context, a "membrane" may be a layer of cells or a plasma membrane that is permeable to water but not to the solutes it contains. The osmotic pressure is directly proportional to the difference in the concentrations of the total numbers of solute molecules on the two sides of the membrane. For example, a 0.5 M NaCl solution is actually $0.5 \mathrm{M} \mathrm{Na}^{+}$ions and $0.5 \mathrm{M} \mathrm{Cl}^{-}$ions and has the same osmotic pressure as a 1 M solution of glucose or sucrose.

The movement of water across the plasma membrane determines the volume of an individual cell, which must be regulated to avoid damage to the cell. Small changes in extracellular osmotic conditions cause most animal cells to swell or shrink rapidly. When placed in a hypotonic solution (i.e., one
![li084542480e](li084542480e.jpg)

FIGURE 11-6 Osmotic pressure. Solutions $A$ and $B$ are separated by a membrane that is permeable to water but impermeable to all solutes. If $C_{B}$ (the total concentration of solutes in solution B) is greater than $C_{A}$, water will tend to flow across the membrane from solution A to solution B. Osmotic pressure $n$ is the hydrostatic pressure that would have to be applied to solution $B$ to prevent this water flow. From the van't Hoff equation, osmotic pressure is given by $\pi=R T\left(C_{B}-C_{A}\right)$, where $R$ is the gas constant and $T$ is the absolute temperature.
in which the concentration of non-membrane-penetrating solutes is lower than in the cytosol), animal cells swell owing to the osmotic flow of water inward. Conversely, when placed in a hypertonic solution (i.e., one in which the concentration of non-membrane-penetrating solutes is higher than in the cytosol), animal cells shrink as cytosolic water leaves the cell by osmotic flow. Consequently, cultured animal cells must be maintained in an isotonic medium, which has a solute concentration, and thus osmotic strength, similar to that of the cell cytosol.

$12 \int$In vascular plants, water and minerals are absorbed from the soil by the roots and move up the plant through conducting tubes (the xylem); water loss from the plant, mainly by evaporation from the leaves, drives this movement of water. Unlike animal cells, plant, algal, fungal, and bacterial cells are surrounded by a rigid cell wall, which resists the expansion of the volume of the cell when the intracellular osmotic pressure increases. Without such a wall, animal cells expand when internal osmotic pressure increases; if that pressure rises too much, the cells burst like overinflated balloons. Because of the cell wall, the osmotic influx of water that occurs when plant cells are placed in a hypotonic solution (even pure water) leads to an increase in intracellular pressure, but not in cell volume.

In plant cells, the concentration of solutes (e.g., sugars and salts) is usually higher in the vacuole (see Figure 1-12a) than in the cytosol, which in turn has a higher solute concentration than the extracellular space. The osmotic pressure generated by the entry of water into the cytosol and then into the vacuole, called turgor pressure, pushes the cytosol and the plasma membrane against the resistant cell wall. Plant cells can harness this pressure to help them stand upright and grow. Cell elongation during growth occurs by means of a hormone-induced, localized loosening of a defined region of the cell wall followed by an influx of water into the vacuole, increasing its size and thus the size of the cell.

Although most protozoans (like animal cells) do not have a rigid cell wall, many contain a contractile vacuole that permits them to avoid osmotic lysis. A contractile vacuole takes up water from the cytosol and, unlike a plant vacuole, periodically discharges its contents through fusion with the plasma membrane. Thus even though water continuously enters the protozoan cell by osmotic flow, the contractile vacuole prevents too much water from accumulating in the cell and swelling it to the bursting point.

## Aquaporins Increase the Water Permeability of Cellular Membranes

The natural tendency of water to flow across cellular membranes as a result of osmotic pressure raises an obvious question: why don't the cells of freshwater animals burst in water? Frogs, for example, lay their eggs in pond water

(a hypotonic solution), but frog oocytes and eggs do not swell with water even though their internal salt (mainly KCl ) concentration is comparable to that of other cells ( $\sim 150 \mathrm{mM} \mathrm{KCl}$ ). These observations were what first led investigators to suspect that the plasma membranes of most cell types, but not of frog oocytes, contain water-channel
proteins that accelerate the osmotic flow of water. The experimental results shown in Figure 11-7 demonstrate that an aquaporin from the erythrocyte plasma membrane functions as a water channel.

In its functional form, an aquaporin is a tetramer of identical $28-\mathrm{kDa}$ subunits (Figure 11-8a). Each subunit
![li094542480e](li094542480e.jpg)

EXPERIMENTAL FIGURE 11-7 Expression of aquaporin by frog oocytes increases their permeability to water. Frog oocytes, which normally are impermeable to water and do not express an aquaporin protein, were microinjected with mRNA encoding aquaporin. These photographs show control oocytes (bottom cell in each panel) and microinjected oocytes (top cell in each panel) at the indicated times after transfer from an isotonic salt solution ( 0.1 M )
![li104542480e](li104542480e.jpg)
to a hypotonic salt solution ( 0.035 M ). The volume of the control oocytes remained unchanged because they are not very permeable to water. In contrast, the microinjected oocytes expressing aquaporin swelled and then burst because of an osmotic influx of water, indicating that aquaporin is a water-channel protein. See L. S. King, D. Kozono, and P. Agre, 2004, Nat. Rev. Mol. Cell Biol. 5:687-698. [Courtesy of Gregory M. Preston and Peter Agre.]
![li114542480e](li114542480e.jpg)

FIGURE 11-8 Structure of an aquaporin. (a) Structural model of the tetrameric protein comprising four identical subunits. Each subunit forms a water channel, as seen in this view looking down on the protein from the exoplasmic side. One of the monomers is shown as a water-accessible surface model, in which the pore entrance can be seen. (b) Schematic diagram of the topology of a single aquaporin subunit in relation to the membrane. Three pairs of homologous transmembrane $\alpha$ helices ( $A$ and $A^{\prime}, B$ and $B^{\prime}$, and $C$ and $C^{\prime}$ ) are oriented in the opposite direction with respect to the membrane and are connected by two hydrophilic loops containing short non-membrane-spanning helices and conserved asparagine ( N ) residues. The loops bend into the cavity formed by the six transmembrane helices, meeting in the middle to form part of the
water-selective gate. (c) Side view of the pore in a single aquaporin subunit, in which several water molecules (blue spheres) are seen within the 2-nm-long water-selective gate that separates the waterfilled cytosolic and extracellular vestibules. The gate contains highly conserved hydrophilic amino acid residues whose side chains form hydrogen bonds with transported water molecules. The amino acids lining the pore are colored from hydrophilic (blue) to hydrophobic (yellow). The arrangement of these hydrogen bonds and the narrow pore diameter of 0.28 nm prevent passage of protons (i.e., $\mathrm{H}_{3} \mathrm{O}^{+}$) or other ions. See T. Zeuthen, 2001, Trends Biochem. Sci. 26:77, and K. Murata et al., 2000, Nature 407:599. [Data from H. Sui et al., 2001, Nature 414:872, PDB ID 1j4n.]

contains six membrane-spanning $\alpha$ helices that form a central pore through which water can move in either direction, depending on the osmotic gradient (Figure 11-8b, c). The $\sim 2$-nm-long water-selective channel, or pore, at the center of each monomer is only 0.28 nm in diameter-only slightly larger than the diameter of a water molecule. The molecular sieving properties of the channel are determined by several conserved hydrophilic amino acid residues whose side-chain and carbonyl groups extend into the middle of the channel and by a relatively hydrophobic wall that lines one side of the channel. Several water molecules can move simultaneously through the channel, each molecule sequentially forming specific hydrogen bonds with the channel-lining amino acids and displacing another water molecule downstream. Aquaporins do not undergo conformational changes during water transport, so they transport water orders of magnitude faster than GLUT1 transports glucose. The formation of hydrogen bonds between the oxygen atom of water and the amino groups of two amino acid side chains ensures that only uncharged water (i.e., $\mathrm{H}_{2} \mathrm{O}$, but not $\mathrm{H}_{3} \mathrm{O}^{+}$) passes through the channel; the orientations of the water molecules in the channel prevent protons from jumping from one to the next and thus prevent the net movement of protons through the channel. As a consequence, ionic gradients are maintained across membranes even when water is flowing across them through aquaporins.

5 Mammals express a family of aquaporins; 11 such genes are known in humans. Aquaporin 1 is expressed in abundance in erythrocytes, and the homologous aquaporin 2 is found in the kidney epithelial cells that resorb water from the urine, thus controlling the amount of water in the body. The activity of aquaporin 2 is regulated by vasopressin, also called antidiuretic hormone, in a manner that resembles the regulation of GLUT4 activity in fat and muscle. When the cells are in their resting state and water is being excreted to form urine, aquaporin 2 is sequestered in intracellular vesicle membranes and so is unable to mediate water import into the cell. When the polypeptide hormone vasopressin binds to the cell-surface vasopressin receptor, it activates a signaling pathway using cAMP as the intracellular signal (detailed in Chapter 15) that causes these aquaporin 2-containing vesicles to fuse with the plasma membrane, increasing the rate of water uptake and return to the circulation. Inactivating mutations in either the vasopressin receptor or the aquaporin 2 gene cause diabetes insipidus, a disease marked by excretion of large volumes of dilute urine. This finding demonstrates that the level of aquaporin 2 is rate limiting for water resorption from urine being formed by the kidney.

Other members of the aquaporin family transport hydroxyl-containing molecules such as glycerol rather than water. Human aquaporin 3, for instance, transports glycerol and is similar in amino acid sequence and structure to the Escherichia coli glycerol transport protein GlpF.

## KEY CONCEPTS OF SECTION 11.2

## Facilitated Transport of Glucose and Water

- Protein-catalyzed transport of biological solutes across a membrane occurs much faster than simple diffusion, exhibits a $V_{\max }$ when the limited number of transporter molecules are saturated with substrate, and is highly specific for substrate (see Figure 11-4).
- Uniport proteins, such as the glucose transporters (GLUTs), are thought to shuttle between two conformational states, one in which the substrate-binding site faces outward and one in which the binding site faces inward (see Figure 11-5).
- All members of the GLUT protein family transport sugars and have similar structures. Differences in their $K_{\mathrm{m}}$ values, expression in different cell types, and substrate specificities are important for proper sugar metabolism in the body.
- Two common experimental systems for studying the functions of transport proteins are liposomes containing a purified transport protein and cells transfected with the gene encoding a particular transport protein.
- Most cellular membranes are semipermeable, more permeable to water than to ions or most other solutes. Water moves by osmosis across membranes from a solution of lower solute concentration to one of higher solute concentration.
- The rigid cell wall surrounding plant cells prevents their swelling and leads to generation of turgor pressure in response to the osmotic influx of water.
- Aquaporins are water-channel proteins that specifically increase the permeability of cellular membranes to water (see Figure 11-8).
- Aquaporin 2 in the plasma membrane of certain kidney cells is essential for resorption of water from urine being formed; the absence of aquaporin 2 leads to the medical condition diabetes insipidus.


### 11.3 ATP-Powered Pumps and the Intracellular Ionic Environment

In the previous sections, we focused on transport proteins that move molecules down their concentration gradients (facilitated transport). Here we focus our attention on the class of proteins-the ATP-powered pumps-that use the energy released by hydrolysis of the terminal phosphoanhydride bond of ATP to transport ions and various small molecules across membranes against their concentration gradients.

All ATP-powered pumps are transmembrane proteins with one or more binding sites for ATP located on subunits or segments of the protein that face the cytosol. These proteins are ATPases, but they normally do not hydrolyze ATP into ADP and $\mathrm{P}_{\mathrm{i}}$ unless ions or other molecules are simultaneously transported. Because of this tight coupling between ATP hydrolysis and transport, the energy stored in the phosphoanhydride bond is not dissipated as heat, but rather is used to move ions or other molecules uphill against an electrochemical gradient.

## There Are Four Main Classes of ATP-Powered Pumps

The general structures of the four classes of ATP-powered pumps are depicted in Figure 11-9, with specific examples in each class listed below the figure. Note that the members of three of the classes ( $\mathrm{P}, \mathrm{F}$, and V ) transport only ions, as do some members of the fourth class, the ABC superfamily. Most members of the ABC superfamily, however, transport

## Exterior

![li124542480e](li124542480e.jpg)

## P-class pumps

Plasma membrane of plants and fungi ( $\mathrm{H}^{+}$pump)
Plasma membrane of higher eukaryotes ( $\mathrm{Na}^{+} / \mathrm{K}^{+}$pump)
Apical plasma membrane of mammalian stomach ( $\mathrm{H}^{+} / \mathrm{K}^{+}$pump)
Plasma membrane of all eukaryotic cells ( $\mathrm{Ca}^{2+}$ pump)
Sarcoplasmic reticulum membrane in muscle cells ( $\mathrm{Ca}^{2+}$ pump)

## FIGURE 11-9 The four classes of ATP-powered transport

proteins. The locations of specific examples are indicated below each class. P-class pumps are composed of two catalytic $\alpha$ subunits, which become phosphorylated as part of the transport cycle. Two $\beta$ subunits, present in some of these pumps, may regulate transport. Only one $\alpha$ and one $\beta$ subunit are depicted here. V-class and F-class pumps do not form phosphoprotein intermediates, and almost all transport only protons. Their structures are similar and contain similar proteins, but none of their subunits are related to those of P-class pumps. V-class pumps couple ATP hydrolysis to transport of protons against a concentration gradient, whereas F-class pumps
small molecules such as amino acids, sugars, peptides, lipids, and many types of drugs.

All P-class pumps possess two identical catalytic $\alpha$ subunits, each of which contains an ATP-binding site. Most also have two smaller $\beta$ subunits, which usually have regulatory functions. During transport, at least one of the $\alpha$ subunits becomes phosphorylated (hence the name "P" class), and the transported ions move through the phosphorylated subunit. The amino acid sequences around the phosphorylated residues are homologous in different pumps. This class includes the $\mathrm{Na}^{+} / \mathrm{K}^{+}$ATPase in the plasma membrane, which generates the low cytosolic $\mathrm{Na}^{+}$and high cytosolic $\mathrm{K}^{+}$concentrations typical of animal cells (see Figure 11-3). Certain $\mathrm{Ca}^{2+}$ ATPases pump $\mathrm{Ca}^{2+}$ ions out of the cytosol into the external medium; others pump $\mathrm{Ca}^{2+}$ from the cytosol into the endoplasmic reticulum or into the specialized ER called the sarcoplasmic reticulum that is found in muscle cells. Another member of the P class, found in acid-secreting cells of the mammalian stomach, transports protons ( $\mathrm{H}^{+}$ions) out of and $\mathrm{K}^{+}$ions into the cell.
![li134542480e](li134542480e.jpg)
![li144542480e](li144542480e.jpg)

## F-class pumps

Bacterial plasma membrane
Inner mitochondrial membrane
Thylakoid membrane of chloroplast

ABC superfamily
Bacterial plasma membranes (amino acid, sugar, and peptide transporters)
Mammalian plasma membranes (transporters of phospholipids, small lipophilic drugs, cholesterol, other small molecules)
normally operate in the reverse direction and use the energy in a proton concentration or voltage gradient to synthesize ATP. All members of the large ABC superfamily of proteins contain two transmembrane (T) domains and two cytosolic ATP-binding (A) domains, which couple ATP hydrolysis to solute movement. These core domains are present as separate subunits in some ABC proteins (as depicted here) but are fused into a single polypeptide in other ABC proteins. See T. Nishi and M. Forgac, 2002, Nat. Rev. Mol. Cell Biol. 3:94; C. Toyoshima et al., 2000, Nature 405:647; D. McIntosh, 2000, Nat. Struct. Biol. 7:532; and T. Elston, H. Wang, and G. Oster, 1998, Nature 391:510.

The structures of V-class and F-class pumps are similar to one another but are unrelated to, and more complicated than, those of P-class pumps. V- and F-class pumps contain several different transmembrane and cytosolic subunits. Virtually all known V and F pumps transport only protons and do so in a process that does not involve a phosphoprotein intermediate. V-class pumps generally function to generate the low pH of plant vacuoles and of lysosomes and other acidic vesicles in animal cells by pumping protons from the cytosolic to the exoplasmic face of the membrane against a proton electrochemical gradient. In contrast, the proton pumps that generate and maintain the plasma-membrane electric potential in plant, fungal, and many bacterial cells belong to the P class of pumps.

F-class pumps are found in bacterial plasma membranes and in mitochondria and chloroplasts. In contrast to V-class pumps, they generally function as reverse proton pumps, in which the energy released by the energetically favored movement of protons from the exoplasmic to the cytosolic face of the membrane down the proton electrochemical gradient is used to power the energetically unfavorable synthesis of ATP from ADP and $\mathrm{P}_{\mathrm{i}}$. Because of their importance in ATP synthesis in chloroplasts and mitochondria, F-class proton pumps, commonly called ATP synthases, are treated separately in Chapter 12 (Cellular Energetics).

The final class of ATP-powered pumps is a large family with multiple members that are more diverse in function than those of the other classes. Referred to as the ABC (ATP-binding cassette) superfamily, this class includes several hundred different transport proteins found in organisms ranging from bacteria to humans. As detailed below, some of these transport proteins were first identified as multidrugresistance proteins that, when overexpressed in cancer cells, export anticancer drugs and render tumors resistant to their action. Each ABC protein is specific for a single substrate or group of related substrates, which may be ions, sugars, amino acids, phospholipids, cholesterol, peptides, polysaccharides, or even proteins. All ABC membrane transport proteins share a structural organization consisting of four "core" domains: two transmembrane (T) domains, which form the passageway through which transported molecules cross the membrane, and two cytosolic ATP-binding (A) domains. In some ABC proteins, mostly those in bacteria, the core domains are present as four separate polypeptides; in others, the core domains are fused into one or two multidomain polypeptides. ATP binding and hydrolysis drives the transport process in most ABC membrane transporters. However, the energy from ATP binding and hydrolysis can also be used to regulate the opening and closing of a continuous channel, as described below for the cystic fibrosis transmembrane conductance regulator protein (CFTR).

## ATP-Powered Ion Pumps Generate and Maintain Ionic Gradients Across Cellular Membranes

The specific ionic composition of the cytosol usually differs greatly from that of the surrounding extracellular fluid. In virtually all cells-including microbial, plant, and animal cells-the cytosolic pH is kept near 7.2 regardless of the extracellular pH .

TABLE 11-2 Typical Intracellular and Extracellular Ion Concentrations

| Ion | Cell (mM) | Blood (mM) |
| :--: | :--: | :--: |
| Squid Giant Axon (marine invertebrate)* |  |  |
| $\mathrm{K}^{+}$ | 400 | 20 |
| $\mathrm{Na}^{+}$ | 50 | 440 |
| $\mathrm{Cl}^{-}$ | $40-150$ | 560 |
| $\mathrm{Ca}^{2+}$ | 0.0003 | 10 |
| $\mathrm{X}^{-1}$ | $300-400$ | $5-10$ |
| Mammalian Cell (vertebrate) |  |  |
| $\mathrm{K}^{+}$ | 139 | 4 |
| $\mathrm{Na}^{+}$ | 12 | 145 |
| $\mathrm{Cl}^{-}$ | 4 | 116 |
| $\mathrm{HCO}_{3}^{-}$ | 12 | 29 |
| $\mathrm{X}^{-}$ | 138 | 9 |
| $\mathrm{Mg}^{2+}$ | 0.8 | 1.5 |
| $\mathrm{Ca}^{2+}$ | $<0.0002$ | 1.8 |

*The large nerve axon of the squid has been widely used in studies of the mechanism of conduction of electric impulses.
${ }^{1} \mathrm{X}^{-}$represents proteins, which have a net negative charge at the neutral pH of blood and cells.

In the most extreme case, there is a 1-million-fold difference in $\mathrm{H}^{+}$concentration between the cytosol of the epithelial cells lining the stomach and the stomach contents after a meal. Furthermore, the cytosolic concentration of $\mathrm{K}^{+}$is much higher than that of $\mathrm{Na}^{+}$. In both invertebrates and vertebrates, the concentration of $\mathrm{K}^{+}$is $20-40$ times higher in the cytosol than in the blood, while the concentration of $\mathrm{Na}^{+}$is $8-12$ times lower in the cytosol than in the blood (Table 11-2).

Some $\mathrm{Ca}^{2+}$ in the cytosol is bound to the negatively charged groups in ATP and in proteins and other molecules, but it is the concentration of unbound (or "free") $\mathrm{Ca}^{2+}$ that is critical to its functions in signaling pathways and muscle contraction. The concentration of free $\mathrm{Ca}^{2+}$ in the cytosol is generally less than 0.2 micromolar $\left(2 \times 10^{-7} \mathrm{M}\right)$, a thousand or more times lower than that in the blood. Plant cells and many microorganisms maintain similarly high cytosolic concentrations of $\mathrm{K}^{+}$and low concentrations of $\mathrm{Ca}^{2+}$ and $\mathrm{Na}^{+}$, even if the cells are cultured in very dilute salt solutions.

The ion pumps discussed in this section are largely responsible for establishing and maintaining the usual ionic

gradients across the plasma and intracellular membranes. In carrying out this task, cells expend considerable energy. For example, up to 25 percent of the ATP produced by nerve and kidney cells is used for ion transport, and human erythrocytes consume up to 50 percent of their available ATP for this purpose; in both cases, most of this ATP is used to power the $\mathrm{Na}^{+} / \mathrm{K}^{+}$pump (see Figure 11-3). The resultant $\mathrm{Na}^{+}$and $\mathrm{K}^{+}$gradients in neurons are essential for their ability to conduct electrical signals rapidly and efficiently, as we detail in Chapter 22. Certain enzymes required for protein synthesis in all cells require a high $\mathrm{K}^{+}$concentration and are inhibited by high concentrations of $\mathrm{Na}^{+}$; these enzymes would cease to function without the operation of the $\mathrm{Na}^{+} / \mathrm{K}^{+}$pump. In cells treated with poisons that inhibit the production of ATP (e.g., 2,4-dinitrophenol in aerobic cells), the pumping stops, and the ion concentrations inside the cell gradually approach those of the exterior environment as ions spontaneously move through channels in the plasma membrane down their electrochemical gradients. Eventually the treated cells die, partly because protein synthesis requires a high concentration of $\mathrm{K}^{+}$ions and partly because, in the absence of a $\mathrm{Na}^{+}$ gradient across the plasma membrane, a cell cannot import certain nutrients such as amino acids (see Figure 11-3). Studies on the effects of such poisons provided early evidence for the existence and significance of ion pumps.

## Muscle Relaxation Depends on $\mathrm{Ca}^{2+}$ ATPases That Pump $\mathrm{Ca}^{2+}$ from the Cytosol into the Sarcoplasmic Reticulum

In skeletal muscle cells, $\mathrm{Ca}^{2+}$ ions are concentrated and stored in the sarcoplasmic reticulum (SR), a specialized type of endoplasmic reticulum (ER). The release (via ion channels) of stored $\mathrm{Ca}^{2+}$ ions from the SR lumen into the cytosol causes muscle contraction, as discussed in Chapter 17. A Pclass $\mathrm{Ca}^{2+}$ ATPase located in the SR membrane pumps $\mathrm{Ca}^{2+}$ from the cytosol back into the lumen of the SR, thereby inducing muscle relaxation.

In the cytosol of muscle cells, the free $\mathrm{Ca}^{2+}$ concentration ranges from $10^{-7} \mathrm{M}$ (resting cells) to more than $10^{-6} \mathrm{M}$ (contracting cells), whereas the total $\mathrm{Ca}^{2+}$ concentration in the SR lumen can be as high as $10^{-2} \mathrm{M}$. The lumen of the SR contains two abundant proteins, calsequestrin and the socalled high-affinity $\mathrm{Ca}^{2+}$ binding protein, each of which binds multiple $\mathrm{Ca}^{2+}$ ions at high affinity. By binding much of the $\mathrm{Ca}^{2+}$ in the SR lumen, these proteins reduce the concentration of "free" $\mathrm{Ca}^{2+}$ ions in the SR vesicles. This reduction, in turn, reduces the $\mathrm{Ca}^{2+}$ concentration gradient between the cytosol and the SR lumen and consequently reduces the energy needed to pump $\mathrm{Ca}^{2+}$ ions into the SR from the cytosol. The activity of the muscle $\mathrm{Ca}^{2+}$ ATPase increases as the free $\mathrm{Ca}^{2+}$ concentration in the cytosol rises. In skeletal muscle cells, the calcium pump in the SR membrane works in concert with a similar $\mathrm{Ca}^{2+}$ pump located in the plasma membrane to ensure that the cytosolic concentration of free $\mathrm{Ca}^{2+}$ in resting muscle remains below $0.1 \mu \mathrm{M}$.

## The Mechanism of Action of the $\mathrm{Ca}^{2+}$ Pump Is Known in Detail

Because the calcium pump constitutes more than 80 percent of the integral membrane protein in muscle SR membranes, it is easily purified and has been studied extensively. Determination of the three-dimensional structure of this protein in several conformational states, representing different steps in the pumping process, has revealed much about its mechanism of action, which serves as a paradigm for understanding many P-class ATPases.

The current model for the mechanism of action of the $\mathrm{Ca}^{2+}$ ATPase in the SR membrane involves multiple conformational states. For simplicity, we group these into E1 states, in which the two binding sites for $\mathrm{Ca}^{2+}$, located in the center of the membrane-spanning domain, face the cytosol, and E2 states, in which these binding sites face the exoplasmic face of the membrane, pointing into the lumen of the SR. The coupling of ATP hydrolysis with ion pumping involves several conformational changes in the protein that must occur in a defined order, as shown in Figure 11-10. When the protein is in the E1 conformation, two $\mathrm{Ca}^{2+}$ ions bind to two high-affinity binding sites accessible from the cytosolic side; even though the cytosolic $\mathrm{Ca}^{2+}$ concentration is low (see Table 11-2), calcium ions still fill these sites.

Next an ATP binds to a site on the cytosolic surface (step $\mathbf{\square}$ ). The bound ATP is hydrolyzed to ADP in a reaction that requires $\mathrm{Mg}^{2+}$, and the liberated phosphate is transferred to a specific aspartate residue in the protein, forming the high-energy acyl phosphate bond denoted by E1-P (step $\mathbf{\square}$ ). The protein then undergoes a conformational change that generates E2, in which the affinity of the two $\mathrm{Ca}^{2+}$-binding sites is reduced (shown in detail in the next figure) and in which these sites are now accessible to the SR lumen (step $\mathbf{\square}$ ). The free energy of hydrolysis of the aspartyl-phosphate bond in E1-P is greater than that in E2-P, and this reduction in the free energy of the aspartylphosphate bond can be said to power the E1 $\rightarrow$ E2 conformational change.

The $\mathrm{Ca}^{2+}$ ions spontaneously dissociate from the binding sites to enter the SR lumen because even though the $\mathrm{Ca}^{2+}$ concentration there is higher than in the cytosol, it is lower than the $K_{\mathrm{d}}$ for $\mathrm{Ca}^{2+}$ binding in the low-affinity state (step $\mathbf{\square}$ ). Finally, the aspartyl-phosphate bond is hydrolyzed (step $\mathbf{\square}$ ). This dephosphorylation, coupled with subsequent binding of cytosolic $\mathrm{Ca}^{2+}$ to the high-affinity $\mathrm{E} 1 \mathrm{Ca}^{2+}$ binding sites, stabilizes the E1 conformational state relative to E2; furthermore, it can be said to power the E2 $\rightarrow$ E1 conformational change (step $\mathbf{\square}$ ). Now E1 is ready to transport two more $\mathrm{Ca}^{2+}$ ions. Thus the cycle is complete, and hydrolysis of one phosphoanhydride bond in ATP has powered the pumping of two $\mathrm{Ca}^{2+}$ ions against a concentration gradient into the SR lumen.

Much structural and biophysical evidence supports the model depicted in Figure 11-10. For instance, the muscle calcium pump has been isolated with phosphate linked to the key aspartate residue, and spectroscopic studies have detected slight alterations in protein conformation during

![li154542480e](li154542480e.jpg)

FIGURE 11-10 Operational model of the $\mathrm{Ca}^{2+}$ ATPase in the SR membrane of skeletal muscle cells. Only one of the two catalytic $\alpha$ subunits of this P-class pump is depicted. E1 and E2 are alternate conformations of the protein in which the $\mathrm{Ca}^{2+}$-binding sites are accessible from the cytosolic and exoplasmic (SR lumen) faces of the membrane, respectively. An ordered sequence of steps, as diagrammed here, is essential for coupling ATP hydrolysis with the transport of $\mathrm{Ca}^{2+}$ ions
the E1 $\rightarrow$ E2 conversion. The two phosphorylated states can also be distinguished biochemically; addition of ADP to phosphorylated E1 results in synthesis of ATP, the reverse of step $\mathbf{1}$ in Figure 11-10, whereas addition of ADP to phosphorylated E2 does not. Each principal conformational state of the reaction cycle can also be characterized by a different susceptibility to various proteolytic enzymes such as trypsin.

Figure 11-11 shows the three-dimensional structure of the $\mathrm{Ca}^{2+}$ pump in the E1 state. As can be seen in Figure 11-11c, the 10 membrane-spanning $\alpha$ helices in the catalytic subunit form the passageway through which $\mathrm{Ca}^{2+}$ ions move. Amino acids in four of these helices form the two high-affinity E1 $\mathrm{Ca}^{2+}$-binding sites (Figure 11-11a, left). One site is formed from negatively charged oxygen atoms from the carboxyl groups $\left(\mathrm{COO}^{-}\right)$of glutamate and aspartate side chains, as well as from bound water molecules. The other site is formed from side- and main-chain oxygen atoms. Thus as $\mathrm{Ca}^{2+}$ ions bind to the $\mathrm{Ca}^{2+}$ pump, they lose the water molecules that normally surround a $\mathrm{Ca}^{2+}$ ion in aqueous solution (see Figure 2-7), but these waters are replaced by oxygen atoms with a similar geometry that are part of the transport protein. In contrast, in the E2 state (Figure 11-11a, right), several of these binding side chains have moved fractions of a nanometer and are unable to interact with bound $\mathrm{Ca}^{2+}$ ions, accounting for the low affinity of the E2 state for $\mathrm{Ca}^{2+}$ ions.

The binding of $\mathrm{Ca}^{2+}$ ions to the $\mathrm{Ca}^{2+}$ pump illustrates a general principle of ion binding to membrane transport
across the membrane. In the figure, -P indicates a high-energy aspartyl phosphate bond; -P indicates a low-energy bond. Because the affinity of $\mathrm{Ca}^{2+}$ for the cytosolic-facing binding sites in E1 is 1000-fold greater than its affinity for the exoplasmic-facing sites in E2, this pump transports $\mathrm{Ca}^{2+}$ unidirectionally from the cytosol to the SR lumen. See the text and Figure 11-11 for more details. See C. Toyoshima and G. Inesi, 2004, Annu. Rev. Biochem. 73:269-292.
proteins that we will encounter repeatedly in this chapter: as ions bind, they lose most of their waters of hydration, but interact with oxygen atoms in the transport protein that have a geometry similar to that of the water oxygens that are bound to them in aqueous solution. This reduces the thermodynamic barrier for ion binding to the protein and allows tight binding of the ion even from solutions of relatively low concentrations.

The cytosolic region of the $\mathrm{Ca}^{2+}$ pump consists of three domains that are well separated from one another in the E1 state (Figure 11-11b). Each of these domains is connected to the membrane-spanning helices by short segments of amino acids. Movements of these cytosolic domains during the pumping cycle cause movements of the connecting segments, which are transmitted into movements of the attached membrane-spanning $\alpha$ helices. For example, the phosphorylated residue, Asp 351, is located in the phosphorylation (P) domain. The adenosine moiety of ATP binds to the nucleotide-binding ( N ) domain, but the $\gamma$ phosphate of ATP binds to specific residues on the P domain, requiring movements of both the N and P domains. Thus, following ATP and $\mathrm{Ca}^{2+}$ binding, the $\gamma$ phosphate of the bound ATP sits adjacent to the aspartate on the P domain that is to receive the phosphate. Although the precise details of these and other protein conformational changes are not yet clear, the movements of the N and P domains are transmitted by leverlike motions of the connecting segments into rearrangements of several membrane-spanning $\alpha$ helices. These changes are

![li164542480e](li164542480e.jpg)

FIGURE 11-11 Structure of the catalytic $\alpha$ subunit of the muscle $\mathrm{Ca}^{2+}$ ATPase. (a) $\mathrm{Ca}^{2+}$-binding sites in the E 1 state (left), with two bound calcium ions, and the low-affinity E2 state (right), without bound ions. Side chains of key amino acids are white, and the oxygen atoms on the glutamate and aspartate side chains are red. In the high-affinity E1 conformation, $\mathrm{Ca}^{2+}$ ions bind at two sites between helices 4,5 , 6 , and 8 inside the membrane. One site is formed out of negatively charged oxygen atoms from glutamate and aspartate side chains and from water molecules (not shown), and the other is formed out of sideand main-chain oxygen atoms. Seven oxygen atoms surround the $\mathrm{Ca}^{2+}$ ion in both sites. (b) Three-dimensional model of the protein in the E1 state based on the structure determined by x-ray crystallography. There are 10 transmembrane $\alpha$ helices, four of which (purple) contain residues that participate in $\mathrm{Ca}^{2+}$ binding. The cytosolic segment forms
especially apparent in the four helices that contain the two $\mathrm{Ca}^{2+}$-binding sites: the changes prevent the bound $\mathrm{Ca}^{2+}$ ions from moving back into the cytosol when released, but enable them to dissociate into the exoplasmic space (lumen).

All P-class ATP-powered pumps, regardless of which ion they transport, are phosphorylated on a highly conserved aspartate residue during the transport process.
three domains: the nucleotide-binding domain ( N , blue), the phosphorylation domain ( $P$, green), and the actuator domain ( $A$, beige), which connects two of the membrane-spanning helices. (c) Models of the pump in the E1 state (left) and in the E2 state (right). Note the differences between the E1 and E2 states in the conformations of the N and A domains. Movements of these domains power the conformational changes of the membrane-spanning $\alpha$ helices (purple) that constitute the $\mathrm{Ca}^{2+}$-binding sites, converting them from a conformation in which the $\mathrm{Ca}^{2+}$-binding sites are accessible from the cytosolic face (E1 state) to one in which the now loosely bound $\mathrm{Ca}^{2+}$ ions gain access to the exoplasmic face (E2 state). (Data from C. Toyoshima and G. Inesi, 2004, Annu. Rev. Biochem. 73:269-292, PDB ID 1su4; and K. Obara et al., 2005, P. Natl. Acad. Sci. USA 102:14489-14496, PDB ID 1agv.)

As deduced from cDNA sequences, the catalytic $\alpha$ subunits of all the P-class pumps examined to date have similar amino acid sequences and thus are presumed to have similar arrangements of transmembrane $\alpha$ helices and cytosol-facing A (actuator), P, and N domains (see Figure 11-11). These findings strongly suggest that all such proteins evolved from a common precursor, although they now transport

different ions. This suggestion is borne out by the similarities of the three-dimensional structures of the membranespanning segments of the $\mathrm{Na}^{+} / \mathrm{K}^{+}$ATPase and the $\mathrm{Ca}^{2+}$ pump (Figure 11-12); the molecular structures of the three cytoplasmic domains are also very similar. Thus the operational model in Figure 11-11 is generally applicable to all of the P-class ATP-powered pumps.

## Calmodulin Regulates the Plasma-Membrane Pumps That Control Cytosolic $\mathrm{Ca}^{2+}$ Concentrations

As we explain in Chapter 15, in muscle cells and in many other types of cells, small increases in the concentration of free $\mathrm{Ca}^{2+}$ ions in the cytosol trigger a variety of cellular responses. In order for $\mathrm{Ca}^{2+}$ to function in intracellular signaling, the concentration of $\mathrm{Ca}^{2+}$ ions free in the cytosol usually must be kept below $0.1-0.2 \mu \mathrm{M}$. Animal, yeast, and probably plant cells express plasma-membrane $\mathrm{Ca}^{2+}$ ATPases that transport $\mathrm{Ca}^{2+}$ out of the cell against its electrochemical gradient. The catalytic $\alpha$ subunit of these P -class pumps is similar in structure and sequence to that of the muscle SR $\mathrm{Ca}^{2+}$ pump.

The activity of plasma-membrane $\mathrm{Ca}^{2+}$ ATPases is regulated by calmodulin, a cytosolic $\mathrm{Ca}^{2+}$-binding protein (see Figure 3-33). A rise in cytosolic $\mathrm{Ca}^{2+}$ induces the binding of $\mathrm{Ca}^{2+}$ ions to calmodulin, which triggers activation of the $\mathrm{Ca}^{2+}$ ATPase. As a result, the export of $\mathrm{Ca}^{2+}$ ions from the cell accelerates, quickly restoring the low concentration of free cytosolic $\mathrm{Ca}^{2+}$ characteristic of the resting cell.
![li174542480e](li174542480e.jpg)

FIGURE 11-12 Structural comparison of $\mathrm{Na}^{+} / \mathrm{K}^{+}$ATPase and muscle $\mathrm{Ca}^{2+}$ ATPase. Three-dimensional structure of the $\mathrm{Na}^{+} / \mathrm{K}^{+}$ ATPase (gold) compared with that of the muscle $\mathrm{Ca}^{2+}$ ATPase (purple), as seen from the cytoplasmic surface. $\alpha \mathrm{M} 1-\alpha \mathrm{M} 10$ denote the 10 membrane-spanning $\alpha$ helices of the $\mathrm{Na}^{+} / \mathrm{K}^{+}$ATPase. [Data from J. P. Morth et al., 2007, Nature 450:1043, PDB ID 3b8e; and C. Toyoshima, H. Nomura, and T. Tsuda, 2004, Nature 432:361-368, PDB ID 1wpg.]

## The $\mathrm{Na}^{+} / \mathrm{K}^{+}$ATPase Maintains the Intracellular $\mathrm{Na}^{+}$and $\mathrm{K}^{+}$Concentrations in Animal Cells

An important P-class pump that is present in the plasma membranes of all animal cells is the $\mathrm{Na}^{+} / \mathrm{K}^{+}$ATPase. This ion pump is a tetramer of subunit composition $\alpha_{2} \beta_{2}$ and shares structural homology with the $\mathrm{Ca}^{2+}$ pump (see Figure 11-12). The small, glycosylated $\beta$ transmembrane polypeptide apparently is not involved directly in ion pumping. During its catalytic cycle, the $\mathrm{Na}^{+} / \mathrm{K}^{+}$ATPase moves three $\mathrm{Na}^{+}$ions out of and two $\mathrm{K}^{+}$ions into the cell per ATP molecule hydrolyzed. The mechanism of action of the $\mathrm{Na}^{+} / \mathrm{K}^{+}$ATPase, outlined in Figure 11-13, is similar to that of the muscle SR calcium pump, except that ions are pumped in both directions across the membrane, with each ion moving against its concentration gradient. In its E1 conformation, the $\mathrm{Na}^{+} / \mathrm{K}^{+}$ATPase has three high-affinity $\mathrm{Na}^{+}$-binding sites and two low-affinity $\mathrm{K}^{+}$-binding sites accessible from the cytosolic surface of the protein. The $K_{\mathrm{m}}$ for binding of $\mathrm{Na}^{+}$to these cytosolic sites is 0.6 mM , a value considerably lower than the intracellular $\mathrm{Na}^{+}$concentration of $\sim 12 \mathrm{mM}$; as a result, $\mathrm{Na}^{+}$ ions normally fully occupy these sites. Conversely, the affinity of the cytosolic $\mathrm{K}^{+}$-binding sites is low enough that $\mathrm{K}^{+}$ions, transported inward through the protein, dissociate from E1 and enter the cytosol despite the high intracellular $\mathrm{K}^{+}$concentration. During the E1 $\rightarrow$ E2 transition, the three bound $\mathrm{Na}^{+}$ions gain access to the exoplasmic face, and simultaneously, the affinity of the three $\mathrm{Na}^{+}$-binding sites drops. The three $\mathrm{Na}^{+}$ions, now bound to low-affinity $\mathrm{Na}^{+}$sites, dissociate one at a time and enter the extracellular medium despite the high extracellular $\mathrm{Na}^{+}$concentration. Transition to the E2 conformation also generates two high-affinity $\mathrm{K}^{+}$sites accessible from the exoplasmic face. Because the $K_{\mathrm{m}}$ for $\mathrm{K}^{+}$binding to these sites $(0.2 \mathrm{mM})$ is lower than the extracellular $\mathrm{K}^{+}$concentration $(4 \mathrm{mM})$, these sites will fill with $\mathrm{K}^{+}$ions as the $\mathrm{Na}^{+}$ions dissociate. Similarly, during the subsequent $\mathrm{E} 2 \rightarrow \mathrm{E} 1$ transition, the two bound $\mathrm{K}^{+}$ ions are transported inward and then released into the cytosol.

Certain drugs (e.g., ouabain and digoxin) bind to the exoplasmic domain of the plasma-membrane $\mathrm{Na}^{+} / \mathrm{K}^{+}$ ATPase and specifically inhibit its ATPase activity. The resulting disruption in the $\mathrm{Na}^{+} / \mathrm{K}^{+}$balance of cells is strong evidence for the critical role of this ion pump in maintaining the normal $\mathrm{K}^{+}$and $\mathrm{Na}^{+}$ion concentration gradients. Classic Experiment 11-1 describes the discovery of this important pump, which is required for life.

## V-Class H+ ATPases Maintain the Acidity of Lysosomes and Vacuoles

All V-class ATPases transport only $\mathrm{H}^{+}$ions. These proton pumps, present in the membranes of lysosomes, endosomes, and plant vacuoles, function to acidify the lumina of these organelles. The pH of the lysosomal lumen can be measured precisely in live cells by use of particles labeled with a pH -sensitive fluorescent dye. When these particles are added to the extracellular fluid, the cells engulf and internalize them (phagocytosis; see Figure 1-18 and Chapter 17), ultimately transporting them into lysosomes. The lysosomal pH can be calculated from the

![li184542480e](li184542480e.jpg)

FIGURE 11-13 Operational model of the plasma-membrane $\mathrm{Na}^{+} / \mathrm{K}^{+}$ATPase. Only one of the two catalytic $\alpha$ subunits of this P -class pump is depicted. It is not known whether just one or both subunits in a single ATPase molecule transport ions. Ion pumping by the $\mathrm{Na}^{+} / \mathrm{K}^{+}$ATPase involves phosphorylation, dephosphorylation, and conformational changes similar to those in the muscle $\mathrm{Ca}^{2+}$ ATPase
(see Figure 11-11). In this case, hydrolysis of the E2-P intermediate powers the E2 $\rightarrow$ E1 conformational change and concomitant transport of two $\mathrm{K}^{+}$ions inward. $\mathrm{Na}^{+}$ions are indicated by red circles; $\mathrm{K}^{+}$ions, by purple squares; high-energy acyl phosphate bond, by -P; low-energy phosphoester bond, by -P.
spectrum of the fluorescence emitted. The DNA encoding a naturally fluorescent protein whose fluorescence depends on the pH can be modified (by adding DNA segments encoding "signal sequences," detailed in Chapters 13 and 14) such that the protein is targeted to the lysosome lumen; fluorescence measurements can then be used to determine the pH in the organelle lumen. Maintenance of the hundredfold or more proton gradient between the lysosomal lumen ( $\mathrm{pH} \sim 4.5-5.0$ ) and the cytosol ( $\mathrm{pH} \sim 7.0$ ) depends on a V-class ATPase and thus on ATP production by the cell. The low lysosomal pH is necessary for optimal function of the many proteases, nucleases, and other hydrolytic enzymes in the lumen; on the other hand, a cytosolic pH of 5 would disrupt the functions of many proteins optimized to act at pH 7 and lead to death of the cell.

Pumping of relatively few protons is required to acidify an intracellular vesicle. To understand why, recall that a solution of pH 4 has a $\mathrm{H}^{+}$ion concentration of $10^{-4}$ moles per liter, or $10^{-7}$ moles of $\mathrm{H}^{+}$ions per milliliter. There are $6.02 \times 10^{23}$ atoms of H per mole (Avogadro's number), so a milliliter of a pH 4 solution contains $6.02 \times 10^{16} \mathrm{H}^{+}$ions. Thus at pH 4 , a primary spherical lysosome with a volume of $4.18 \times 10^{-15} \mathrm{ml}$ (diameter of $0.2 \mu \mathrm{~m}$ ) would contain just 252 protons. At pH 7 , the same organelle would have an average of only 0.2 protons in its lumen, and thus pumping of only about 250 protons would be necessary for lysosome acidification.

By themselves, V-class proton pumps cannot acidify the lumen of an organelle (or the extracellular space) because these pumps are electrogenic; that is, a net movement of
electric charge occurs during transport. Pumping of just a few protons causes a buildup of positively charged $\mathrm{H}^{+}$ions on the exoplasmic (inside) face of the organelle membrane. For each $\mathrm{H}^{+}$pumped across, a negative ion (e.g., $\mathrm{OH}^{-}$or $\mathrm{Cl}^{-}$) will be "left behind" on the cytosolic face, causing a buildup of negatively charged ions there. These oppositely charged ions attract each other on opposite faces of the membrane, generating a charge separation, or electric potential, across the membrane. The lysosome membrane thus functions as a capacitor in an electric circuit, storing opposing charges (anions and cations) on opposite sides of a barrier impermeable to the movement of charged particles.

As more and more protons are pumped and build up excess positive charge on the exoplasmic face, the energy required to move additional protons against this rising electric potential gradient increases dramatically and prevents pumping of additional protons long before a significant transmembrane $\mathrm{H}^{+}$concentration gradient is established (Figure 11-14a). In fact, this is the way that P-class proton pumps generate a cytosol-negative potential across plant and yeast plasma membranes.

In order for an organelle lumen or an extracellular space (e.g., the lumen of the stomach) to become acidic, movement of protons must be accompanied either by (1) movement of an equal number of anions (e.g., $\mathrm{Cl}^{-}$) in the same direction or by (2) movement of equal numbers of a different cation in the opposite direction. The first process occurs in lysosomes and plant vacuoles, whose membranes contain V-class $\mathrm{H}^{+}$ATPases and anion channels through which accompanying $\mathrm{Cl}^{-}$ions

move (Figure 11-14b). The second process occurs in the lining of the stomach, which contains a P-class $\mathrm{H}^{+} / \mathrm{K}^{+}$ATPase that is not electrogenic and pumps one $\mathrm{H}^{+}$outward and one $\mathrm{K}^{+}$inward. Operation of this pump is discussed later in the chapter.

The V-class proton pumps in lysosomal and vacuolar membranes have been solubilized, purified, and incorporated into liposomes. As shown in Figure 11-9, these pumps contain two discrete domains, a cytosolic hydrophilic domain $\left(\mathrm{V}_{1}\right)$ and a transmembrane domain $\left(\mathrm{V}_{0}\right)$, with multiple subunits forming each domain. Binding and hydrolysis of ATP by the B subunits in $\mathrm{V}_{1}$ provide the energy for the pumping of $\mathrm{H}^{+}$ions through the proton-conducting channel formed by the c and a subunits in $\mathrm{V}_{0}$. Unlike P-class ion pumps, V-class proton pumps are not phosphorylated and dephosphorylated during proton transport.

## ABC Proteins Export a Wide Variety of Drugs and Toxins from the Cell

As noted earlier, all members of the very large and diverse ABC superfamily of membrane transport proteins contain two transmembrane (T) domains and two cytosolic ATP-binding (A) domains (see Figure 11-9). The T domains, each built of 10 membrane-spanning $\alpha$ helices, form the pathway through
![li194542480e](li194542480e.jpg)

FIGURE 11-14 Effect of V-class proton pumps on $\mathrm{H}^{+}$concentration gradients and electric potential gradients across cellular membranes. (a) If an intracellular organelle contains only V-class pumps, proton pumping generates an electric potential across the membrane (the cytosolic face becomes negative and the luminal face positive), but no significant change in the intraluminal pH . (b) If the organelle membrane also contains $\mathrm{Cl}^{-}$channels, anions passively follow the pumped protons, resulting in an accumulation of $\mathrm{H}^{+}$and $\mathrm{Cl}^{-}$ions in the lumen (low luminal pH ) but no electric potential across the membrane.
which the transported substance (substrate) crosses the membrane (Figure 11-15a) and determine the substrate specificity of each ABC protein. The sequences of the A domains are approximately $30-40$ percent homologous in all members of this superfamily, indicating a common evolutionary origin.

The first eukaryotic ABC protein to be recognized was discovered during studies on tumor cells and cultured cells that exhibited resistance to several drugs with unrelated chemical structures. Such cells were eventually shown to express elevated levels of a multidrug-resistance (MDR) transport protein originally called MDR1 and now known as ABCB1. This protein uses the energy derived from ATP hydrolysis to export a large variety of drugs from the cytosol to the extracellular medium. The Mdr1 gene is frequently amplified in multidrug-resistant cells, resulting in a large overproduction of the MDR1 protein. In contrast to bacterial ABC proteins, which are built of four discrete subunits, all four domains of mammalian ABCB1 are fused into a single $170-\mathrm{kDa}$ protein.

The substrates of mammalian ABCB1 are primarily planar, lipid-soluble molecules with one or more positive charges; they all compete with one another for transport, which suggests that they bind to the same or overlapping sites on the protein. Many drugs transported by ABCB1 diffuse from the extracellular medium across the plasma membrane, unaided by transport proteins, into the cell cytosol, where they block various cellular functions. Two such drugs are colchicine and vinblastine, which block assembly of microtubules (see Chapter 18). ATP-powered export of such drugs by MDR1 reduces their concentration in the cytosol. As a result, a much higher extracellular drug concentration is required to kill cells that express ABCB1 than those that do not. That ABCB1 is an ATP-powered small-molecule pump has been demonstrated with liposomes containing the purified protein. Different drugs enhance the ATPase activity of these liposomes in a dose-dependent manner corresponding to their ability to be transported by ABCB1.

The three-dimensional structure of ABCB1, together with those of homologous bacterial ABC proteins, revealed the protein's mechanism of transport as well as its ability to bind and transport a wide array of hydrophilic and hydrophobic substrates (see Figure 11-15). The two T domains form a binding site in the center of the membrane that alternates between an inward-facing (Figure 11-15b) and an outward-facing (Figure 11-15c) orientation, conforming to the alternating access model. The alternation between these two conformational states is powered by ATP binding to the two A subunits and its subsequent hydrolysis to ADP and $\mathrm{P}_{\mathrm{i}}$, but precisely how this happens is not known.

The substrate-binding cavity formed by ABCB1 is large. Some of the amino acids that line the cavity-mainly tyrosine and phenylalanine-have aromatic side chains, allowing ABCB1 to bind multiple types of hydrophobic ligands. Other segments of the cavity are lined with hydrophilic residues, allowing hydrophilic or amphipathic molecules to bind. In the inward-facing conformation, the binding site is open directly to the surrounding aqueous solution, allowing hydrophilic molecules to enter the binding site directly

FIGURE 11-15 The multidrug transporter ABCB1 (MDR1): Structure and model of ligand export. (a) Cross-sectional view through the center of an ABCB1 protein bound to two molecules of a drug analog, qz59-sss (black), reveals the central location of the ligand-binding site in relation to the phospholipid bilayer: the central ligand-binding cavity is close to the leaflet-leaflet interface of the membrane. During transport, this binding cavity is alternately exposed to the exoplasmic and the cytosolic surface of the membrane. Serines 289 and 290 affect the ligand specificity of the transporter; they are shown as red spheres to highlight their juxtaposition to the bound ligand. Surface residues are colored yellow to denote hydrophobic and blue to denote hydrophilic amino acids. (b) Three-dimensional structure of ABCB1 with its ligand-binding site facing inward toward the cytosol. In this conformation, a hydrophilic ligand can bind directly from the cytosol. A more hydrophobic ligand can partition into the inner leaflet of the plasma membrane bilayer and then enter the ligand-binding site through a gap in the protein that is accessible directly from the hydrophobic core of the inner leaflet. (c) Model for the structure of ABCB1 with its ligandbinding site facing outward, based on the structures of homologous bacterial ABC proteins. When the protein assumes this conformation, the ligand can either diffuse into the exoplasmic leaflet or directly into the aqueous extracellular medium. See D. Gutman et al., 2009, Trends Biochem. Sci. 35:36-42. [Data from S. G. Aller et al., 2009, Science 323:1718-1722, PDB ID 3g61]
from the cytosol. In addition, a gap in the protein is accessible directly from the hydrophobic core of the inner leaflet of the membrane bilayer; this allows hydrophobic molecules to enter the binding site directly from the inner leaflet (see Figure 11-15b). After the ATP-powered change to the outward-facing conformation, molecules can exit the binding site into the outer membrane leaflet or directly into the extracellular medium (see Figure 11-15c).

About 50 different mammalian ABC proteins are now recognized (Table 11-3). In eukaryotic cells, ABC proteins localize not only to the plasma membrane, but also to the membranes of many intracellular organelles. Several are
(a)
![li204542480e](li204542480e.jpg)
(b)
![li214542480e](li214542480e.jpg)
expressed in abundance in the liver, intestines, and kidneysites where natural toxic and waste products are removed from the body. Substrates for these ABC proteins include sugars, amino acids, cholesterol, bile acids, phospholipids, peptides, proteins, toxins, and foreign substances. The normal function of ABCB1 most likely is to transport various

TABLE 11-3 Selected Human ABC Proteins

| Protein | Tissue Expression | Function | Disease Caused by Defective Protein |
| :--: | :--: | :--: | :--: |
| ABCB1 (MDR1) | Adrenal, kidney, brain | Exports lipophilic drugs |  |
| ABCB4 (MDR2) | Liver | Exports phosphatidylcholine into bile |  |
| ABCB11 | Liver | Exports bile salts into bile |  |
| CFTR | Exocrine tissue | Transports Cl ions | Cystic fibrosis |
| ABCDI | Ubiquitous in peroxisomal | Influences activity of peroxisomal enzyme that oxidizes very long chain fatty acids | Adrenoleukodystrophy (ADL) |
| ABCG5/8 | Liver, intestine | Exports cholesterol and other sterols | $\beta$-Sitosterolemia |
| ABCA1 | Ubiquitous | Exports cholesterol and phospholipid for uptake into high-density lipoprotein (HDL) | Tangier's disease |
| ABCA4 | Retina | Transports N-retinyl-phosphatidylethanolamine | Stargardt's disease in photoreceptor cells (juvenile macular degeneration) |

natural and metabolic toxins into the bile or intestinal lumen for excretion or into the urine being formed in the kidney. During the course of its evolution, ABCB1 appears to have acquired the ability to transport drugs whose structures are similar to those of these endogenous toxins. Tumors derived from MDR-expressing cell types, such as hepatomas (liver cancers), are frequently resistant to virtually all chemotherapeutic agents and are thus difficult to treat, presumably because the tumors exhibit increased expression of ABCB1 or a related ABC protein.

## Certain ABC Proteins "Flip" Phospholipids and Other Lipid-Soluble Substrates from One Membrane Leaflet to the Other

As shown in Figure 11-15, parts (b) and (c), ABCB1 can move, or "flip," a hydrophobic or amphipathic substrate molecule from the inner leaflet of the membrane to the outer leaflet. This otherwise energetically unfavorable reaction is powered by the ATPase activity of the protein. Support for this so-called flippase model of transport by ABCB1 comes
![li224542480e](li224542480e.jpg)

## EXPERIMENTAL FIGURE 11-16 An in vitro fluorescencequenching assay revealed the phospholipid flippase activity of ABCB4. A homogeneous population of secretory vesicles containing ABCB4 protein was obtained by introducing the cDNA encoding mammalian ABCB4 into a temperature-sensitive yeast sec mutant such that ABCB4 was localized to intracellular endoplasmic reticulum vesicles in its normal orientation and with the cytosolic face of the vesicles facing outward (see Figure 14-4). Step $\mathbf{\square}$ When synthetic phospholipids containing a fluorescently modified head group (blue) were added to the medium surrounding the purified vesicles, they were incorporated primarily into the outer, cytosolic leaflets of the vesicles. Step $\mathbf{\square}$ If ABCB4 acted as a flippase, then on addition of ATP to the medium, a small fraction of the outward-facing labeled phospholipids would be flipped to the inside leaflet. Step $\mathbf{\square}$ Flipping was detected by adding a non-membrane-permeating quenching compound called
dithionite to the medium. Dithionite reacts with the fluorescent head groups, destroying their ability to fluoresce (gray). In the presence of the quencher, only labeled phospholipids in the protected environment of the inner leaflet will fluoresce. Subsequent to the addition of the quenching agent, the total fluorescence decreases with time until it plateaus at the point at which all external fluorescence is quenched and only the internal phospholipid fluorescence can be detected. The observation of greater fluorescence (less quenching) in the presence of ATP than in its absence indicates that ABCB4 has flipped some of the labeled phospholipid to the inside leaflet. Not shown here are "control" vesicles isolated from cells that did not express ABCB4 and that exhibited no flippase activity. Step $\mathbf{\square}$ Addition of detergent to the vesicles generates micelles and makes all fluorescent lipids accessible to the quenching agent, lowering the fluorescence to baseline values. See S. Ruetz and P. Gros, 1994, Cell 77:1071.

from experiments on ABCB4 (originally called MDR2), a protein homologous to ABCB1 that is present in the region of the liver-cell plasma membrane that faces the bile canaliculi. ABCB4 moves phosphatidylcholine from the cytosolic to the exoplasmic leaflet of the plasma membrane for subsequent release into the bile in combination with cholesterol and bile acids, which themselves are transported by other ABC superfamily members. Still other ABC superfamily members participate in the cellular export of various lipids, presumably by mechanisms similar to that of ABCB1 (see Table 11-3).

ABCB4 was first suspected of having phospholipid flippase activity because mice with homozygous loss-of-function mutations in the $A B C B 4$ gene exhibited defects in the secretion of phosphatidylcholine into bile. To determine directly if ABCB4 was in fact a flippase, researchers performed experiments on a homogeneous population of purified vesicles isolated from special mutant yeast cells with ABCB4 in the membrane and with the cytosolic face directed outward (Figure 11-16). After purifying these vesicles, investigators labeled them in vitro with a fluorescent phosphatidylcholine derivative. The fluorescence-quenching assay outlined in Figure 11-16 was used to demonstrate that the vesicles containing ABCB4 exhibited an ATP-dependent flippase activity.

## The ABC Cystic Fibrosis Transmembrane Regulator Is a Chloride Channel, Not a Pump

Several human genetic diseases are associated with defective ABC proteins (see Table 11-3). The best-studied and most widespread is cystic fibrosis (CF), caused by a mutation in the gene encoding the cystic fibrosis transmembrane regulator (CFTR, also called $A B C C 7$ ). Like other ABC proteins, CFTR has two transmembrane T domains and two cytosolic A, or ATP-binding, domains. CFTR contains an additional R (regulatory) domain on the cytosolic face; R links the two homologous halves of the protein, creating an overall domain organization of T1-A1-R-T2-A2. But CFTR is a
$\mathrm{Cl}^{-}$channel, not a pump. It is expressed in the apical plasma membranes of epithelial cells in the lungs, sweat glands, pancreas, and other tissues. For instance, CFTR protein is important for reuptake into the cells of sweat glands of $\mathrm{Cl}^{-}$lost by sweating; babies with cystic fibrosis, if licked, often taste "salty" because this reuptake is inhibited.

The $\mathrm{Cl}^{-}$channel of CFTR is normally closed. Channel opening is activated by phosphorylation of the R domain by a protein kinase (PKA, discussed in Chapter 15), which in turn is activated by an increase in cyclic AMP (cAMP), a small intracellular signaling molecule. Opening of the channel also requires sequential binding of two ATP molecules to the two A domains (Figure 11-17).

About two-thirds of all CF cases can be attributed to a single mutation in CFTR: deletion of Phe 508 in the ATP-binding A1 domain. At body temperature, the mutant protein fails to fold properly and to move to the cell surface, where it normally functions. Interestingly, if cells expressing the mutant protein are incubated at room temperature, the protein folds and accumulates normally on the plasma membrane, where it functions nearly as well as the wild-type CFTR channel. Recently a small molecule has been chemically synthesized that binds to this mutant CFTR protein in CF patients and stabilizes the folded form at $37^{\circ} \mathrm{C}$, allows it to traffic normally to the cell surface, and partially reverses the effects of the disease. Another CFTR mutation, Gly 551 to Asp, accounts for approximately 5 percent of CF cases and results in a channel that has normal surface expression but is defective in $\mathrm{Cl}^{-}$ transport because the mutation disrupts ATP binding. Small molecules that increase the flow of $\mathrm{Cl}^{-}$ions through the mutant channel, called CFTR potentiators, are currently used to treat CF patients whose disease is caused by this mutation. These drugs represent some of the first successful personalized therapies that are based on a molecular understanding of the disease-causing protein.

FIGURE 11-17 Structure and function of the cystic fibrosis transmembrane regulator (CFTR). The regulatory (R) domain (not depicted) must be phosphorylated before ATP is able to power channel opening. Upon phosphorylation, one ATP (yellow circle) becomes tightly bound to the A1 domain (green). Binding of a second ATP to the A2 domain (blue) is followed by formation of a tight intramolecular A1-A2 heterodimer and slow channel opening. The relatively stable open state becomes destabilized by hydrolysis of the ATP bound at A2 to ADP (red crescent) and $P_{i}$. The ensuing disruption of the tight A1-A2 dimer interface leads to channel closure. $T=$ transmembrane domain; $\mathrm{A}=$ cytosolic ATP-binding domain. See D. C. Gadsby et al., 2006, Nature 440:477.
![li234542480e](li234542480e.jpg)

## KEY CONCEPTS OF SECTION 11.3

## ATP-Powered Pumps and the Intracellular Ionic Environment

- Four classes of transmembrane proteins couple the energyreleasing hydrolysis of ATP with the energy-requiring transport of substances against their concentration gradients: P-, V-, and F-class pumps and ABC proteins (see Figure 11-9).
- The combined action of P -class $\mathrm{Na}^{+} / \mathrm{K}^{+}$ATPases in the plasma membrane and homologous $\mathrm{Ca}^{2+}$ ATPases in the plasma membrane or sarcoplasmic reticulum creates the usual ionic milieu of animal cells: high $\mathrm{K}^{+}$, low $\mathrm{Ca}^{2+}$, and low $\mathrm{Na}^{+}$ in the cytosol; low $\mathrm{K}^{+}$, high $\mathrm{Ca}^{2+}$, and high $\mathrm{Na}^{+}$in the extracellular fluid.
- In P-class pumps, phosphorylation of the $\alpha$ (catalytic) subunit and changes in conformational states are essential for coupling ATP hydrolysis to transport of $\mathrm{H}^{+}, \mathrm{Na}^{+}, \mathrm{K}^{+}$, or $\mathrm{Ca}^{2+}$ ions (see Figures 11-10 through 11-13).
- V- and F-class ATPases, which transport protons exclusively, are large, multisubunit complexes with a protonconducting channel in the transmembrane domain and ATPbinding sites in the cytosolic domain.
- V-class proton pumps in animal lysosomal and endosomal membranes and plant vacuolar membranes are responsible for maintaining a lower pH inside the organelles than in the surrounding cytosol (see Figure 11-14).
- All members of the large and diverse ABC superfamily of membrane transport proteins contain four core domains: two transmembrane domains, which form a pathway for solute movement and determine substrate specificity, and two cytosolic ATP-binding domains (see Figure 11-15).
- The two T domains of the multidrug transporter ABCB1 form a ligand-binding site in the middle of the plane of the membrane; ligands can bind directly from the cytosol or from the inner membrane leaflet through a gap in the protein.
- The ABC superfamily includes about 50 mammalian proteins (e.g., ABCB1, ABCA1) that transport a wide array of substrates, including toxins, drugs, phospholipids, peptides, and proteins, into or out of the cell.
- Biochemical experiments directly demonstrate that ABCB4 (MDR2) possesses phospholipid flippase activity (see Figure 11-16).
- CFTR, an ABC protein, is a $\mathrm{Cl}^{-}$channel, not a pump. Channel opening is triggered by protein phosphorylation and by binding of ATP to the two A domains (see Figure 11-17).


## 11.4 Nongated Ion Channels and the Resting Membrane Potential

In addition to ATP-powered ion pumps, which transport ions against their concentration gradients, the plasma membrane contains channel proteins that allow the principal cellular ions $\left(\mathrm{Na}^{+}, \mathrm{K}^{+}, \mathrm{Ca}^{2+}\right.$, and $\left.\mathrm{Cl}^{-}\right)$to move through them at different rates down their concentration gradients. Ion concentration gradients generated by pumps and selective movements of ions through channels constitute the principal mechanism by which a difference in voltage, or electric potential, is generated across the plasma membrane. In other words, ATPpowered ion pumps generate differences in ion concentrations across the plasma membrane, and ion channels use these concentration gradients to generate a tightly controlled electric potential across the membrane (see Figure 11-3).

In all cells, the magnitude of this electric potential is generally $\sim 70$ millivolts ( mV ), with the inside cytosolic face of the plasma membrane always negative with respect to the outside exoplasmic face. This value does not seem like much until we consider that the thickness of the plasma membrane is only $\sim 3.5 \mathrm{~nm}$. Thus the voltage gradient across the plasma membrane is 0.07 V per $3.5 \times 10^{-7} \mathrm{~cm}$, or 200,000 volts per centimeter! (To appreciate what this means, consider that high-voltage transmission lines for electricity use gradients of about 200,000 volts per kilometer, $10^{5}$-fold less!)

The ionic gradients and electric potential across the plasma membrane play crucial roles in many biological processes. As noted previously, a rise in the cytosolic $\mathrm{Ca}^{2+}$ concentration is an important regulatory signal, initiating contraction in muscle cells and triggering in many cells secretion of proteins, such as digestive enzymes from pancreatic cells. In many animal cells, the combined force of the $\mathrm{Na}^{+}$concentration gradient and the membrane electric potential drives the uptake of amino acids and other molecules against their concentration gradients by symporters and antiporters (see Figure 11-3 and Section 11.5). Furthermore, electrical signaling by neurons depends on the opening and closing of ion channels in response to changes in the membrane electric potential (see Chapter 22).

Here we discuss the origin of the membrane electric potential in resting non-neuronal cells (often called the cell's resting membrane potential); how ion channels mediate the selective movement of ions across a membrane; and useful experimental techniques for characterizing the functional properties of channel proteins.

## Selective Movement of Ions Creates a Transmembrane Electric Gradient

To help explain how an electric potential across the plasma membrane can arise, we first consider a set of simplified experimental systems in which a membrane separates a 150 mM $\mathrm{NaCl} / 15 \mathrm{mM} \mathrm{KCl}$ solution (similar to the extracellular medium surrounding metazoan cells) on the right from a 15 mM $\mathrm{NaCl} / 150 \mathrm{mM} \mathrm{KCl}$ solution (similar to that of the cytosol)

(a) Membrane impermeable to $\mathrm{Na}^{+}, \mathrm{K}^{+}$, and $\mathrm{Cl}^{-}$
![li244542480e](li244542480e.jpg)
(b) Membrane permeable only to $\mathrm{Na}^{+}$
![li254542480e](li254542480e.jpg)
(c) Membrane permeable only to $\mathrm{K}^{+}$
![li264542480e](li264542480e.jpg)

## EXPERIMENTAL FIGURE 11-18 Generation of a transmembrane electric potential (voltage) depends on the selective movement of ions across a semipermeable membrane. In this experimental

system, a membrane separates a $15 \mathrm{mM} \mathrm{NaCl} / 150 \mathrm{mM} \mathrm{KCl}$ solution (left) from a $150 \mathrm{mM} \mathrm{NaCl} / 15 \mathrm{mM} \mathrm{KCl}$ solution (right); these ion concentrations are similar to those in cytosol and blood, respectively. If the membrane separating the two solutions is impermeable to all ions (a), no ions can move across the membrane, and no electric potential is registered on the potentiometer connecting the two solutions. If the membrane is selectively permeable only to $\mathrm{Na}^{+}(\mathrm{b})$ or only to $\mathrm{K}^{+}$(c), then diffusion of these ions through their respective channels leads to a separation of charge across the membrane. At equilibrium, the membrane potential caused by the charge separation becomes equal to the Nernst potential $E_{\mathrm{Na}}$ or $E_{\mathrm{K}}$ registered on the potentiometer. See the text for further explanation.
on the left (Figure 11-18a). A potentiometer (voltmeter) is connected to both solutions to measure any electric potential across the membrane. Initially, both solutions contain an equal number of positive and negative ions. If the membrane is impermeable to all ions, no ions will flow across it. Furthermore, there will be no difference in voltage, or electric potential gradient, across the membrane, as shown in Figure 11-18a.

Now suppose that the membrane contains $\mathrm{Na}^{+}$channels that accommodate $\mathrm{Na}^{+}$ions but exclude $\mathrm{K}^{+}$and $\mathrm{Cl}^{-}$ions (Figure 11-18b). $\mathrm{Na}^{+}$ions then tend to move down their concentration gradient from the right side to the left, leaving an excess of negative $\mathrm{Cl}^{-}$ions compared with $\mathrm{Na}^{+}$ions on the right side and generating an excess of positive $\mathrm{Na}^{+}$ions compared with $\mathrm{Cl}^{-}$ions on the left side. The excess $\mathrm{Na}^{+}$on the left and $\mathrm{Cl}^{-}$on the right remain near the respective surfaces of the membrane because the excess positive charges on one side of the membrane are attracted to the excess negative charges on the other side. The resulting separation of charge across the membrane constitutes an electric potential, or voltage, and the left (cytosolic) side of the membrane has excess positive charge with respect to the right.

As more and more $\mathrm{Na}^{+}$ions move through channels across the membrane, the magnitude of this charge difference (i.e., voltage) increases. However, continued right-toleft movement of the $\mathrm{Na}^{+}$ions is eventually inhibited by the mutual repulsion between the excess positive $\left(\mathrm{Na}^{+}\right)$charges accumulated on the left side of the membrane and by the attraction of $\mathrm{Na}^{+}$ions to the excess negative charges built up on the right side. The system soon reaches an equilibrium point at which the two opposing factors that determine the movement of $\mathrm{Na}^{+}$ions-the membrane electric potential and the ion concentration gradient-balance each other out. At equilibrium, there is no net movement of $\mathrm{Na}^{+}$ions across the membrane. Thus this membrane, like all cellular membranes, acts as a capacitor-a device consisting of a thin sheet of nonconducting material (the hydrophobic interior) surrounded on both sides by electrically conducting material (the polar phospholipid head groups and the ions in the surrounding aqueous solution) that can store positive charges on one side and negative charges on the other.

If a membrane is permeable only to $\mathrm{Na}^{+}$ions, then at equilibrium, the measured electric potential across the membrane equals the sodium equilibrium potential in volts, $E_{\mathrm{Na}}$.

The magnitude of $E_{\mathrm{Na}}$ is given by the Nernst equation, which is derived from basic principles of physical chemistry:

$$
E_{\mathrm{Na}}=\frac{R T}{Z F} \ln \frac{\left[\mathrm{Na}_{\text {right }}\right]}{\left[\mathrm{Na}_{\text {left }}\right]}
$$

where $R$ (the gas constant) $=1.987 \mathrm{cal} /($ degree $\cdot \mathrm{mol})$, or 8.28 joules/(degree $\cdot \mathrm{mol}) ; T$ (the absolute temperature in degrees Kelvin) $=293^{\circ} \mathrm{K}$ at $20^{\circ} \mathrm{C} ; Z$ (the charge, also called the valence) is here equal to $+1 ; F$ (the Faraday constant) $=23,062 \mathrm{cal} /(\mathrm{mol} \cdot \mathrm{V})$, or $96,000 \mathrm{coulombs} /(\mathrm{mol} \cdot \mathrm{V}) ;$ and $\left[\mathrm{Na}_{\text {left }}\right]$ and $\left[\mathrm{Na}_{\text {right }}$ ] are the $\mathrm{Na}^{+}$concentrations on the left and right sides, respectively, at equilibrium. By convention, the potential is expressed as the cytosolic face of the membrane relative to the exoplasmic face, and the equation is written with the ion concentration of the extracellular solution (here the right side of the membrane) placed in the numerator and that of the cytosol in the denominator.

At $20^{\circ} \mathrm{C}$ (room temperature), Equation 11-2 reduces to

$$
E_{\mathrm{Na}}=0.059 \log _{10} \frac{\left[\mathrm{Na}_{\text {right }}\right]}{\left[\mathrm{Na}_{\text {left }}\right]}
$$

If $\left[\mathrm{Na}_{\text {right }}\right] /\left[\mathrm{Na}_{\text {left }}\right]=10$, a tenfold ratio of concentrations as in Figure 11-18b, then $E_{\mathrm{Na}}=+0.059 \mathrm{~V}$ (or +59 mV ), with the left, cytosolic side positive with respect to the right, exoplasmic side.

If the membrane is permeable only to $\mathrm{K}^{+}$ions and not to $\mathrm{Na}^{+}$or $\mathrm{Cl}^{-}$ions, then a similar equation describes the potassium equilibrium potential $E_{\mathrm{K}}$ :

$$
E_{\mathrm{K}}=0.059 \log _{10} \frac{\left[\mathrm{~K}_{\text {right }}\right]}{\left[\mathrm{K}_{\text {left }}\right]}
$$

The magnitude of the membrane electric potential is the same ( 59 mV , for a tenfold difference in ion concentrations), but the left, cytosolic side is now negative with respect to the right, exoplasmic (Figure 11-18c), so the polarity is opposite to that obtained across a membrane selectively permeable to $\mathrm{Na}^{+}$ions.

## The Resting Membrane Potential in Animal Cells Depends Largely on the Outward Flow of $\mathrm{K}^{+}$Ions Through Open $\mathrm{K}^{+}$Channels

The plasma membranes of animal cells contain many open $\mathrm{K}^{+}$channels but few open $\mathrm{Na}^{+}, \mathrm{Cl}^{-}$, or $\mathrm{Ca}^{2+}$ channels. As a result, the major ionic movement across the plasma membrane is the movement of $\mathrm{K}^{+}$from the inside outward, powered by the $\mathrm{K}^{+}$concentration gradient. This movement leaves an excess of negative charge on the cytosolic face of the plasma membrane and creates an excess of positive charge on the exoplasmic face, as in the experimental system shown in Figure 11-18c. This outward flow of $\mathrm{K}^{+}$ions is the major determinant of the inside-negative membrane potential. The channels through which the $\mathrm{K}^{+}$ions flow, called resting $\mathrm{K}^{+}$ channels, alternate, like all channels, between an open and a closed state (see Figure 11-2), but since their opening and
closing is not affected by the membrane potential or by small signaling molecules, these channels are referred to as nongated. In contrast, the various gated channels in neurons and other excitable cells (see Chapter 22) open only in response to specific ligands or to changes in membrane potential.

Quantitatively, the usual resting membrane potential of -60 to $-70 \mathrm{mV}$ is close to the potassium equilibrium potential, calculated from the Nernst equation and the $\mathrm{K}^{+}$concentrations in cells and surrounding media (see Table 11-2). Usually the resting membrane potential is slightly lower (less negative) than that calculated from the Nernst equation because of the presence of a few open $\mathrm{Na}^{+}$channels. These channels allow the net inward flow of $\mathrm{Na}^{+}$ions, making the cytosolic face of the plasma membrane more positive-that is, less negative-than predicted by the Nernst equation for $\mathrm{K}^{+}$. The $\mathrm{K}^{+}$concentration gradient that drives the flow of ions through resting $\mathrm{K}^{+}$channels is generated by the $\mathrm{Na}^{+} / \mathrm{K}^{+}$ ATPase described previously (see Figures 11-3 and 11-13). In the absence of this pump, or when it is inhibited, the $\mathrm{K}^{+}$ concentration gradient cannot be maintained, the membrane potential falls to zero, and the cell eventually dies.

Although resting $\mathrm{K}^{+}$channels play the dominant role in generating the electric potential across the plasma membranes of animal cells, this is not the case in bacterial, plant, and fungal cells. The inside-negative membrane potential in plant and fungal cells is generated by transport of positively charged protons $\left(\mathrm{H}^{+}\right)$out of the cell by ATP-powered proton pumps, a process similar to what occurs in lysosomal membranes lacking $\mathrm{Cl}^{-}$channels (see Figure 11-14a): each $\mathrm{H}^{+}$pumped out of the cell leaves behind a $\mathrm{Cl}^{-}$ion, generating an inside-negative electric potential across the membrane. In aerobic bacterial cells, an inside-negative potential is generated by outward pumping of protons during electron transport, a process similar to proton pumping in mitochondrial inner membranes that will be discussed in detail in Chapter 12 (see Figure 12-22).

The electric potential across the plasma membrane of a cell can be measured with a microelectrode inserted into the cell and a reference electrode placed in the extracellular fluid. The two electrodes are connected to a potentiometer capable of measuring small potential differences (Figure 11-19). The potential across the plasma membrane of most animal cells generally does not vary with time. In contrast, neurons and muscle cells-the principal types of electrically active cellsundergo controlled changes in their membrane potential, as we discuss in Chapter 22.

## Ion Channels Are Selective for Certain Ions by Virtue of a Molecular "Selectivity Filter"

All ion channels exhibit specificity for particular ions: $\mathrm{K}^{+}$ channels allow $\mathrm{K}^{+}$ions, but not closely related $\mathrm{Na}^{+}$ions, to enter them, whereas $\mathrm{Na}^{+}$channels admit $\mathrm{Na}^{+}$, but not $\mathrm{K}^{+}$. Determination of the three-dimensional structure of a bacterial $\mathrm{K}^{+}$channel first revealed how this exquisite ion selectivity is achieved. Comparisons of the sequences and structures of other $\mathrm{K}^{+}$channels from organisms as diverse as bacteria, fungi, and humans established that all share a common structure and probably evolved from a single type of channel protein.

![li274542480e](li274542480e.jpg)

EXPERIMENTAL FIGURE 11-19 The electric potential across the plasma membrane of a live cell can be measured. A microelectrode, constructed by filling a glass tube of extremely small diameter with a conducting fluid such as a KCl solution, is inserted into a cell in such a way that the plasma membrane seals itself around the tip of the electrode. A reference electrode is placed in the extracellular medium. A potentiometer connecting the two electrodes registers the potential-in this case, -60 mV , with the cytosolic face negative with respect to the exoplasmic face of the membrane. A potential difference is registered only when the microelectrode is inserted into the cell; no potential is registered if the microelectrode is in the extracellular fluid.
(a) Single subunit
(b) Tetrameric channel
![li284542480e](li284542480e.jpg)

FIGURE 11-20 Structure of a resting $\mathrm{K}^{+}$channel from the bacterium Streptomyces lividans. All $\mathrm{K}^{+}$channels are tetramers comprising four identical subunits, each containing two conserved membranespanning $\alpha$ helices, called by convention S5 and S6, and a shorter P, or pore, segment. (a) One of the subunits, viewed from the side, with key structural features indicated. (b-d) The complete tetrameric channel viewed from the side (b and c) and from the top, or extracellular, end (d). The P segments (green) are located near the exoplasmic surface and connect the S5 and S6 $\alpha$ helices [yellow in (a), yellow and lavender

Like all other $\mathrm{K}^{+}$channels, bacterial $\mathrm{K}^{+}$channels are built of four identical transmembrane subunits symmetrically arranged around a central pore (Figure 11-20). Each subunit contains two membrane-spanning $\alpha$ helices (S5 and S6) and a short P (pore) segment that partly penetrates the membrane bilayer from the exoplasmic surface. In the tetrameric $\mathrm{K}^{+}$ channel, the eight transmembrane $\alpha$ helices (two from each subunit) form an inverted cone, generating a water-filled cavity called the vestibule in the central portion of the channel that extends halfway through the membrane toward the cytosolic side. Four extended loops that are part of the four P segments form the actual selectivity filter in the narrow part of the pore near the exoplasmic surface, above the vestibule.

Several related pieces of evidence support the role of P segments in ion selection. First, the amino acid sequences of the P segments in all known $\mathrm{K}^{+}$channels are highly homologous and are different from those in other ion channels. Second, mutations of certain amino acids in this segment alter the ability of a $\mathrm{K}^{+}$channel to distinguish $\mathrm{Na}^{+}$from $\mathrm{K}^{+}$. Finally, replacing the P segment of a bacterial $\mathrm{K}^{+}$channel with the homologous segment from a mammalian $\mathrm{K}^{+}$channel yields a chimeric protein that exhibits normal selectivity for $\mathrm{K}^{+}$over other ions. Thus all $\mathrm{K}^{+}$channels are thought to use the same mechanism to distinguish $\mathrm{K}^{+}$from other ions.
$\mathrm{Na}^{+}$ions are smaller than $\mathrm{K}^{+}$ions. How, then, can a channel protein exclude smaller $\mathrm{Na}^{+}$, yet allow passage of larger $\mathrm{K}^{+}$? The ability of the selectivity filter in $\mathrm{K}^{+}$channels to select $\mathrm{K}^{+}$over $\mathrm{Na}^{+}$is due mainly to the backbone carbonyl oxygens on residues located in a Gly-Tyr-Gly sequence that is found in an analogous position in the P segment in every known $\mathrm{K}^{+}$channel. As a $\mathrm{K}^{+}$ion enters the selectivity
![li294542480e](li294542480e.jpg)
in (b-d)]; they consist of a nonhelical "turret," which lines the upper part of the pore; a short $\alpha$ helix; and an extended loop that protrudes into the narrowest part of the pore and forms the selectivity filter. This filter allows $\mathrm{K}^{+}$(purple spheres), but not other ions, to pass. Below the filter is the central cavity, or vestibule, lined by the inner, or S6, $\alpha$ helices. The subunits in gated $\mathrm{K}^{+}$channels, which open and close in response to specific stimuli, contain additional transmembrane helices not shown here; these channels are discussed in Chapter 22. [Data from Y. Zhou et al., 2001, Nature 414:43, PDB ID 1k4c.]

(a) $\mathrm{K}^{+}$and $\mathrm{Na}^{+}$ions in the pore of a $\mathrm{K}^{+}$channel (top view)
![li304542480e](li304542480e.jpg)
(b) $\mathrm{K}^{+}$ions in the pore of a $\mathrm{K}^{+}$channel (side view)
![li314542480e](li314542480e.jpg)

State 1
State 2
filter-the narrow space between the P-segment filter sequences contributed by the four adjacent subunits-it loses its eight waters of hydration, but becomes bound in the same geometry to eight backbone carbonyl oxygens, two from the extended loop in each of the four P segments lining the channel (Figure 11-21a, bottom left). Thus little energy is required to strip off the eight waters of hydration of a $\mathrm{K}^{+}$ion, and as a result, a relatively low activation energy is required for passage of $\mathrm{K}^{+}$ions into the channel from an aqueous solution. A dehydrated $\mathrm{Na}^{+}$ion is too small to bind to all eight carbonyl oxygens that line the selectivity filter with the same geometry as a $\mathrm{Na}^{+}$ion surrounded by its normal eight water molecules in aqueous solution. As a result, $\mathrm{Na}^{+}$ions "prefer" to remain in water rather than enter the selectivity filter, and thus the change in free energy for entry of $\mathrm{Na}^{+}$ions into the channel is relatively high (Figure 11-21a, right). This difference in free energies favors passage of $\mathrm{K}^{+}$ions through the channel over $\mathrm{Na}^{+}$ions by a factor of 1000 . Like $\mathrm{Na}^{+}$, the dehydrated $\mathrm{Ca}^{2+}$ ion is smaller than the dehydrated $\mathrm{K}^{+}$ion and cannot interact properly with the oxygen atoms in the selectivity filter. Furthermore, because a $\mathrm{Ca}^{2+}$ ion has two positive charges and binds water oxygens more tightly than does a single-positive $\mathrm{Na}^{+}$or $\mathrm{K}^{+}$ion, more energy is required to strip the waters of hydration from $\mathrm{Ca}^{2+}$ than from $\mathrm{K}^{+}$or $\mathrm{Na}^{+}$.

FIGURE 11-21 Mechanism of ion selectivity and transport in resting $\mathrm{K}^{+}$channels. (a) Schematic diagrams of $\mathrm{K}^{+}$and $\mathrm{Na}^{+}$ions hydrated in solution and in the pore of a $\mathrm{K}^{+}$channel. As $\mathrm{K}^{+}$ions pass through the selectivity filter, they lose their bound water molecules and become bound instead to eight backbone carbonyl oxygens (four of which are shown) that are part of the conserved amino acid sequence in the channel-lining selectivity filter loop of each P segment. The smaller $\mathrm{Na}^{+}$ions, with their tighter shell of water molecules, cannot perfectly bind to the channel oxygen atoms and therefore pass through the channel only rarely. (b) High-resolution electron density map obtained from x-ray crystallography showing $\mathrm{K}^{+}$ions (purple spheres) passing through the selectivity filter. Only two of the diagonally opposed channel subunits are shown. Within the selectivity filter, each unhydrated $\mathrm{K}^{+}$ion interacts with eight carbonyl oxygen atoms (red sticks) lining the channel, two from each of the four subunits, as if to mimic the eight waters of hydration. (c) Interpretation of the electron density map, showing the two alternating states by which $\mathrm{K}^{+}$ions move through the channel. Ion positions are numbered top to bottom from the exoplasmic side of the channel inward. In state 1 , one sees a hydrated $\mathrm{K}^{+}$ ion with its eight bound water molecules, $\mathrm{K}^{+}$ions at positions 1 and 3 within the selectivity filter, and a fully hydrated $\mathrm{K}^{+}$ion within the vestibule. During $\mathrm{K}^{+}$movement, each ion in state 1 moves one step inward, forming state 2 . Thus in state 2 , the $\mathrm{K}^{+}$ion on the exoplasmic side of the channel has lost four of its eight waters, the ion at position 1 in state 1 has moved to position 2, and the ion at position 3 in state 1 has moved to position 4. In going from state 2 to state 1 , the $\mathrm{K}^{+}$at position 4 moves into the vestibule and picks up eight water molecules, while another hydrated $\mathrm{K}^{+}$ion moves into the channel opening, and the other $\mathrm{K}^{+}$ions move down one step. Note that $\mathrm{K}^{+}$ions are shown here moving from the exoplasmic side of the channel to the cytosolic side because that is the normal direction of movement in bacteria. In animal cells, the direction of $\mathrm{K}^{+}$movement is typically the reverse-from inside to outside. See C. Armstrong, 1998, Science 280:56. [Parts (b) and (c) data from Y. Zhou et al., 2001, Nature 414:43, PDB ID 1k4c.]

![li324542480e](li324542480e.jpg)

Cell-attached patch measures effect of extracellular solutes on channels within membrane patch on intact cell
![li334542480e](li334542480e.jpg)

Cytosolic face
Inside-out detached patch measures effects of intracellular solutes on channels within isolated patch
![li344542480e](li344542480e.jpg)

Whole-cell patch measures effect of intracellular solutes on channels in cell body
![li354542480e](li354542480e.jpg)

Exoplasmic face
Outside-out detached patch measures effects of extracellular solutes on channels within isolated patch

Recent x-ray crystallographic studies reveal that both when open and when closed, the $\mathrm{K}^{+}$channel contains $\mathrm{K}^{+}$ ions within the selectivity filter; without these ions, the channel would probably collapse. The $\mathrm{K}^{+}$ions are thought to be present either at positions 1 and 3 or at positions 2 and 4 , each surrounded by eight carbonyl oxygen atoms (Figure 1121 b and c). Several $\mathrm{K}^{+}$ions move simultaneously through the channel such that when the ion on the exoplasmic face that has been partially stripped of its water of hydration moves into position 1, the ion at position 2 jumps to position 3, and the one at position 4 exits the channel (Figure 11-21c).

The first three-dimensional structure of a sodium channel was determined in late 2011, and its mechanisms of ion selectivity are only now being determined. In contrast to potassium channels, all of which share a conserved selectivity filter, studies of sodium channels have revealed that distinct types of sodium channels in diverse species possess different types of selectivity filters, each of which uses a distinct mechanism to achieve selectivity for $\mathrm{Na}^{+}$ions.

EXPERIMENTAL FIGURE 11-22 Current flow through individual ion channels can be measured by patch clamping.
(a) Basic experimental arrangement for measuring current flow through individual ion channels in the plasma membrane of a live cell. The patch electrode, filled with a current-conducting saline solution, is applied, with a slight suction, to the plasma membrane. The $0.5-\mu \mathrm{m}$-diameter tip covers a region that contains only one or a few ion channels. A recording device measures current flow only through the channel or channels in that region. (b) Photomicrograph of the cell body of a cultured neuron and the tip of a patch pipette touching the plasma membrane. (c) Different patch-clamping configurations. In cell-attached patches, the electrode forms a tight seal around a patch of membrane; it records the effects on channels within this patch of different concentrations of ions and solutes such as extracellular hormones. In whole-cell patch-clamp recording, suction is used to create a hole in the membrane so that the micropipette gains access to the inside of the cell. Reagents can be delivered to the cytoplasm through the micropipette, and their effects on channels in the entire cell body can be recorded. One can also study the effects of different reagents on channels in isolated, detached patches; these are the best configurations for studying the effects on channels of different ion concentrations. In inside-out patches, one can measure the effect of intracellular reagents on channel function by exposing the tip of the micropipette to solutions containing defined reagents. Outside-out patches are made from a whole-cell patch configuration by pulling the pipette away from the cell to create a bleb of membrane such that the exoplasmic face of the membrane has access to the solutions to which the tip of the micropipette is exposed. In this way, one can measure the effect of extracellular reagents on channel function. An inside-out patch, in which a cell-attached patch is formed and then the cell pulled away, is used in the experiment in Figure 11-24. [Part (b) republished with permission of Elsevier, from Sakmann, B., "Elementary steps in synaptic transmission revealed by currents through single ion channels," Neuron, 1992, 8:4, pp 613-629; permission conveyed through Copyright Clearance Center, Inc.]

## Patch Clamps Permit Measurement of Ion Movements Through Single Channels

Once it was realized that in most cells there are only one or a few ion channels per square micrometer of plasma membrane, it became possible to record ion movements through single ion channels, and to measure the rates at which these channels open and close and conduct specific ions, using a technique known as patch clamping. As illustrated in Figure 11-22, a tiny glass pipette is tightly sealed, using suction, to the surface of a cell; the segment of the plasma membrane within the tip of the pipette will contain only one or a few ion channels. The only current that crosses through the patch of membrane must pass through these channels as ions flow. An electrical recording device detects this ion flow, measured as electric current, through the channels; this ion flow usually occurs in small bursts when a channel is open (Figure 11-23). The electrical device also "clamps" or locks the electric potential across the membrane at a predetermined value (hence the term patch clamping). The inward or outward movement of ions across a patch of membrane is quantified from the amount of electric current needed to maintain the membrane potential at a particular "clamped" value (Figure 11-22a). Four common patch-clamp configurations are used (Figure 11-22c). In cell-attached patch clamping, the pipette forms a tight seal around a small area of membrane to record the current through channels within that small area. In whole-cell patch clamping, the plasma membrane is ruptured

![li364542480e](li364542480e.jpg)

EXPERIMENTAL FIGURE 11-23 Ion flux through individual $\mathrm{Na}^{+}$channels can be calculated from patch-clamp tracings. Two inside-out patches of muscle plasma membrane were clamped at a potential of slightly less than the resting membrane potential. The patch pipette contained NaCl . The transient pulses of electric current (in picoamperes), recorded as large downward deviations (blue arrows), indicate the opening of a $\mathrm{Na}^{+}$channel and the movement of positive charges ( $\mathrm{Na}^{+}$ions) inward across the membrane. The smaller deviations in current represent background noise. The average current through an open channel is 1.6 pA , or $1.6 \times 10^{-12}$ amperes. Since 1 ampere $=1$ coulomb (C) of charge per second, this current is equivalent to the movement of about $9900 \mathrm{Na}^{+}$ions per channel per millisecond: $\left(1.6 \times 10^{-12} \mathrm{C} /\right.$ s) $\left(10^{-3} \mathrm{~s} / \mathrm{ms}\right)\left(6 \times 10^{23}\right.$ molecules $\left./ \mathrm{mol}\right) \div 96,500$ $\mathrm{C} / \mathrm{mol}$. See F. J. Sigworth and E. Neher, 1980, Nature 287:447.
so that the glass pipette has access to the cytoplasm to record currents from the entire cell body. In addition, small patches of membrane can be torn away from the cell, with either the inside or the outside of the plasma membrane surface exposed to the extracellular solution. Addition of specific molecules and compounds to the solution within the glass pipette can be used to probe the effects of these reagents on ion channel function.

The tracings in Figure 11-23 illustrate the use of patch clamping to study the properties of voltage-gated $\mathrm{Na}^{+}$channels (which open in response to changes in membrane potential) in the plasma membrane of muscle cells. As we discuss in Chapter 22, these channels are normally closed in resting muscle cells and open following neuronal stimulation. Patches of muscle-cell plasma membrane, each containing an average of one $\mathrm{Na}^{+}$channel, were clamped at a predetermined voltage that, in this study, was slightly less than the resting membrane potential. Under these circumstances, transient pulses of positive charges $\left(\mathrm{Na}^{+}\right.$ions) cross the membrane from the exoplasmic to the cytosolic face as individual $\mathrm{Na}^{+}$channels open and then close. Each channel is either fully open or completely closed. From such tracings, it is possible to determine how long a channel is open and the ion flux through it. For the channels measured in Figure 11-23, the flux is about 10 million $\mathrm{Na}^{+}$ions per channel per second, a typical value for ion channels. Replacement of the NaCl within the patch pipette (corresponding to the outside of the cell) with KCl or choline chloride abolishes current through the channels, confirming that they conduct only $\mathrm{Na}^{+}$ions, not $\mathrm{K}^{+}$or other ions.

## Novel Ion Channels Can Be Characterized by a Combination of Oocyte Expression and Patch Clamping

Cloning of human-disease-causing genes and sequencing of the human genome have identified many genes encoding putative channel proteins, including 67 putative $\mathrm{K}^{+}$ channels. One way of characterizing the function of these
![li374542480e](li374542480e.jpg)

EXPERIMENTAL FIGURE 11-24 The oocyte expression assay is useful in comparing the function of normal and mutant forms of a channel protein. An oocyte from the follicle of a frog is first treated with collagenase to remove the surrounding follicle cells, leaving a denuded oocyte, which is then microinjected with mRNA encoding the channel protein under study. See T. P. Smith, 1988, Trends Neurosci. 11:250.
proteins is to transcribe a cloned cDNA in a cell-free system to produce the corresponding mRNA. Injecting this mRNA into frog oocytes and taking patch-clamp measurements of the newly synthesized channel protein can often reveal its function (Figure 11-24). This experimental approach is especially useful because frog oocytes normally do not express any channel proteins on their plasma membranes, so only the channel under study is present in the membrane. In addition, because of the large size of frog oocytes, patch-clamping studies are technically easier to perform on them than on smaller cells.

## KEY CONCEPTS OF SECTION 11.4

## Nongated Ion Channels and the Resting Membrane Potential

- An inside-negative electric potential (voltage) of about 60 to 70 mV exists across the plasma membrane of all cells.
- The electric potential generated by the selective flow of ions across a membrane can be calculated using the Nernst equation (see Equation 11-2).
- The resting membrane potential in animal cells is the result of the combined action of the ATP-powered $\mathrm{Na}^{+} / \mathrm{K}^{+}$

pump, which establishes $\mathrm{Na}^{+}$and $\mathrm{K}^{+}$concentration gradients across the membrane, and resting $\mathrm{K}^{+}$channels, which permit selective movement of $\mathrm{K}^{+}$ions back down their concentration gradient to the external medium (see Figure 11-3).

- Unlike the more common gated ion channels, which open only in response to various signals, the nongated resting $\mathrm{K}^{+}$ channels are usually open.
- In plants and fungi, the membrane potential is maintained by the ATP-driven pumping of protons from the cytosol to the exterior of the cell.
- $\mathrm{K}^{+}$channels are assembled from four identical subunits, each of which has at least two conserved membrane-spanning $\alpha$ helices and a nonhelical $P$ segment that lines the ion pore and forms the selectivity filter (see Figure 11-20).
- The ion specificity of $\mathrm{K}^{+}$channels is due mainly to binding of the $\mathrm{K}^{+}$ion with eight carbonyl oxygen atoms of specific amino acids in the P segments, which lowers the activation energy for the passage of $\mathrm{K}^{+}$compared with $\mathrm{Na}^{+}$or other ions (see Figure 11-21).
- Patch-clamping techniques, which permit measurement of ion movements through single channels, are used to determine the ion conductivity of a channel and the effect of various reagents on its activity (see Figure 11-22).
- Recombinant DNA techniques and patch clamping allow the expression and functional characterization of channel proteins in frog oocytes (see Figure 11-24).


### 11.5 Cotransport by Symporters and Antiporters

In previous sections, we saw how ATP-powered pumps generate ion concentration gradients across cellular membranes and how $\mathrm{K}^{+}$channels use the $\mathrm{K}^{+}$concentration gradient to establish an electric potential across the plasma membrane. In this section, we see how cotransporters use the energy stored in electric potentials and concentration gradients of $\mathrm{Na}^{+}$or $\mathrm{H}^{+}$ions to power the uphill movement of another substance, which may be a small organic molecule such as glucose, or an amino acid, or a different ion. An important feature of such cotransport is that neither substance can move alone; movement of both substances together is obligatory, or coupled.

Cotransporters share features with uniporters such as the GLUT proteins. The two types of transporters exhibit certain structural similarities, operate at equivalent rates, and undergo cyclical conformational changes during transport of their substrates. They differ in that uniporters can only accelerate thermodynamically favorable transport down a concentration gradient, whereas cotransporters can harness the energy released when one substance moves down its concentration gradient to drive the movement of another substance against its concentration gradient.

When the transported molecule and cotransported ion move in the same direction, the process is called symport; when they move in opposite directions, the process is called antiport (see Figure 11-2). Some cotransporters transport only positive ions (cations), while others transport only negative ions (anions). Yet other cotransporters mediate movement of both cations and anions together. Cotransporters are present in all organisms, including bacteria, plants, and animals. In this section, we describe the operation and function of several physiologically important symporters and antiporters.

## $\mathrm{Na}^{+}$Entry into Mammalian Cells Is Thermodynamically Favored

Mammalian cells express many types of $\mathrm{Na}^{+}$-linked symporters. The human genome encodes literally hundreds of different types of transporters that use the energy stored in the $\mathrm{Na}^{+}$concentration gradient and in the inside-negative electric potential across the plasma membrane to transport a wide variety of molecules into cells against their concentration gradients. To see how such transporters allow cells to accumulate substrates against a considerable concentration gradient, we first need to calculate the change in free energy $(\Delta G)$ that occurs during $\mathrm{Na}^{+}$entry. As mentioned earlier, two forces govern the movement of ions across a selectively permeable membrane: the voltage across the membrane and the ion concentration gradient across the membrane. The sum of these forces constitutes the electrochemical gradient. To calculate the free-energy change, $\Delta G$, corresponding to the transport of any ion across a membrane, we need to consider the independent contributions from each of these forces to the electrochemical gradient.

For example, when $\mathrm{Na}^{+}$moves from the outside to the inside of a cell, the free-energy change generated from the $\mathrm{Na}^{+}$concentration gradient is given by

$$
\Delta G_{\mathrm{c}}=R T \ln \frac{\left[\mathrm{Na}_{\mathrm{in}}\right]}{\left[\mathrm{Na}_{\text {out }}\right]}
$$

At the concentrations of $\mathrm{Na}_{\text {in }}$ and $\mathrm{Na}_{\text {out }}$ shown in Figure 11-25, which are typical for many mammalian cells, $\Delta G_{\mathrm{c}}$, the change in free energy due to the concentration gradient, is -1.45 kcal for transport of 1 mole of $\mathrm{Na}^{+}$ions from the outside to the inside of a cell, assuming there is no membrane electric potential. Note that the free energy is negative, indicating spontaneous movement of $\mathrm{Na}^{+}$into the cell down its concentration gradient.

The free-energy change generated from the membrane electric potential is given by

$$
\Delta G_{\mathrm{m}}=F E
$$

where $F$ is the Faraday constant [ $=23,062 \mathrm{cal} /(\mathrm{mol} \cdot \mathrm{V})]$; and $E$ is the membrane electric potential. If $E=-70 \mathrm{mV}$, then $\Delta G_{\mathrm{m}}$, the free-energy change due to the membrane potential,

![li384542480e](li384542480e.jpg)

FIGURE 11-25 Transmembrane forces acting on $\mathrm{Na}^{+}$ions. As with all ions, the movement of $\mathrm{Na}^{+}$ions across the plasma membrane is governed by the sum of two separate forces: the ion concentration gradient and the membrane electric potential. At the internal and external $\mathrm{Na}^{+}$concentrations typical of mammalian cells, these forces usually act in the same direction, making the inward movement of $\mathrm{Na}^{+}$ ions energetically favorable.
is -1.61 kcal for transport of 1 mole of $\mathrm{Na}^{+}$ions from the outside to the inside of a cell, assuming there is no $\mathrm{Na}^{+}$concentration gradient. Since both forces do in fact act on $\mathrm{Na}^{+}$ ions, the total $\Delta G$ is the sum of the two partial values:
$\Delta G=\Delta G_{\mathrm{c}}+\Delta G_{\mathrm{m}}=(-1.45)+(-1.61)=-3.06 \mathrm{kcal} / \mathrm{mol}$
In this example, the $\mathrm{Na}^{+}$concentration gradient and the membrane electric potential contribute almost equally to the total $\Delta G$ for transport of $\mathrm{Na}^{+}$ions. Since $\Delta G$ is less than 0 , the inward movement of $\mathrm{Na}^{+}$ions is thermodynamically favored. As we will see next, the inward movement of $\mathrm{Na}^{+}$ is used to power the uphill movement of other ions and several types of small molecules into or out of animal cells. The rapid, energetically favorable movement of $\mathrm{Na}^{+}$ions through gated $\mathrm{Na}^{+}$channels is also critical in generating action potentials in neurons and muscle cells, as discussed in Chapter 22.

## $\mathrm{Na}^{+}$-Linked Symporters Enable Animal Cells to Import Glucose and Amino Acids Against High Concentration Gradients

Most body cells import glucose from the blood down a concentration gradient of glucose, using GLUT proteins to facilitate this transport. However, certain cells, such as those lining the small intestine and the kidney tubules, need to import glucose from extracellular fluids (digestive products or
urine) against a very large concentration gradient (glucose concentration is higher inside the cell). Such cells use a two$\mathrm{Na}^{+}$/one-glucose symporter, a protein that couples the import of one glucose molecule to the import of two $\mathrm{Na}^{+}$ions:

$$
2 \mathrm{Na}_{\text {out }}^{+}+\text {glucose }_{\text {out }} \rightleftharpoons 2 \mathrm{Na}_{\text {in }}^{+}+\text {glucose }_{\text {in }}
$$

Quantitatively, the free-energy change for the symport of two $\mathrm{Na}^{+}$ions and one glucose molecule can be written
$\Delta G=R T \ln \frac{\left[\text { glucose }_{\text {in }}\right]}{\left[\text { glucose }_{\text {out }}\right]}+2 R T \ln \frac{\left[\mathrm{Na}_{\text {in }}^{+}\right]}{\left[\mathrm{Na}_{\text {out }}^{+}\right]}+2 F E$
Thus the $\Delta G$ for the overall reaction is the sum of the freeenergy changes generated by the glucose concentration gradient ( 1 molecule transported), the $\mathrm{Na}^{+}$concentration gradient ( $2 \mathrm{Na}^{+}$ions transported), and the membrane potential ( $2 \mathrm{Na}^{+}$ions transported). As illustrated in Figure 11-25, the movement of $\mathrm{Na}^{+}$ions into a mammalian cell down their electrochemical gradient has a free-energy change, $\Delta G$, of about -3 kcal per mole of $\mathrm{Na}^{+}$transported. Thus the $\Delta G$ for the transport of two moles of $\mathrm{Na}^{+}$inward would be twice this amount, or about -6 kcal . This negative freeenergy change for sodium import is coupled to the uphill transport of glucose, a process with a positive $\Delta G$. We can calculate the glucose concentration gradient (inside greater than outside) that can be established by the action of this $\mathrm{Na}^{+}$-powered symporter by realizing that at equilibrium for $\mathrm{Na}^{+}$-linked glucose import, $\Delta G=0$. By substituting the values for sodium import into Equation 11-7 and setting $\Delta G=0$, we see that

$$
0=R T \ln \frac{\left[\text { glucose }_{\text {in }}\right]}{\left[\text { glucose }_{\text {out }}\right]}-6 \mathrm{kcal}
$$

and we can calculate that at equilibrium, the ratio of glucose $_{\text {in }} /$ glucose $_{\text {out }}=\sim 30,000$. Thus the inward flow of two moles of $\mathrm{Na}^{+}$can generate an intracellular glucose concentration that is $\sim 30,000$ times greater than the exterior concentration. If only one $\mathrm{Na}^{+}$ion were imported ( $\Delta G$ of approximately $-3 \mathrm{kcal} / \mathrm{mol}$ ) per glucose molecule, then the available energy could generate a glucose concentration gradient (inside/outside) of only about 170 -fold. Thus by coupling the transport of two $\mathrm{Na}^{+}$ions to the transport of one glucose molecule, the two- $\mathrm{Na}^{+}$/one-glucose symporter permits cells to accumulate a very high concentration of glucose relative to the external concentration. This means that glucose that is present even at very low concentrations in the lumen of the intestine or in the kidney tubules can be efficiently transported into the lining cells and not lost from the body.

The two- $\mathrm{Na}^{+}$/one-glucose symporter is thought to contain 14 transmembrane $\alpha$ helices with both its N - and C -termini extending into the cytosol. Figure 11-26 depicts the current model of transport by $\mathrm{Na}^{+} /$glucose symporters. This model entails conformational changes in the protein analogous to those that occur in uniporters, such as GLUT1, that do not require a cotransported ion (compare with Figure 11-5). Binding of all substrates to their sites on the extracellular domain

![li394542480e](li394542480e.jpg)

FIGURE 11-26 Operational model for the two- $\mathrm{Na}^{+} /$one-glucose symporter. Simultaneous binding of $\mathrm{Na}^{+}$and glucose to the conformation with outward-facing binding sites (step $\mathbf{\square}$ causes a conformational change in the protein such that the bound substrates are transiently occluded, unable to dissociate into either medium (step $\mathbf{\square}$. In step $\mathbf{\square}$, the protein assumes a third conformation with inward-facing
sites. Dissociation of the bound $\mathrm{Na}^{+}$and glucose (step $\mathbf{\square}$ allows the protein to revert to its original outward-facing conformation (step $\mathbf{\square}$ ), ready to transport additional substrate. See H. Krishnamurthy et al., 2009, Nature 459:347-355 for details on the structure and function of this and related $\mathrm{Na}^{+}$-linked symporters.
is required before the protein undergoes the change that converts the substrate-binding sites from the outward- to the inward-facing conformation; this ensures that inward transport of glucose and $\mathrm{Na}^{+}$ions are coupled.

$\square$There are two human $\mathrm{Na}^{+} /$glucose symporters. SGLT1 is found in the absorptive cells lining the small intestine as well as in the epithelial cells lining part of the kidney tubules. SGLT2 is found only in kidney tubules, where, together with SGLT1, it resorbs glucose into the blood from the forming urine. Inhibition of SGLT2 leads to excretion of glucose in the urine and a reduction in blood glucose levels; therefore, SGLT2 inhibitors have potential use in the treatment of type II diabetes. Indeed, several drug candidates that selectively inhibit SGLT2 and not SGLT1 have been developed or are currently undergoing clinical trials, including one approved for use in the United States and Canada.

Cells use $\mathrm{Na}^{+}$-powered symporters to transport substances other than glucose into the cell against high concentration gradients. For example, several types of $\mathrm{Na}^{+} /$amino acid symporters allow cells to import many amino acids. As another example, $\mathrm{Na}^{+} /$neurotransmitter symporters couple the import of $\mathrm{Na}^{+}$to the reuptake and recycling of neurotransmitters, and they are the targets of many therapeutic drugs, including many antidepressants. They are also the targets of several drugs of abuse, including cocaine and amphetamines.

## A Bacterial $\mathrm{Na}^{+} /$Amino Acid Symporter Reveals How Symport Works

No three-dimensional structure has yet been determined for any mammalian $\mathrm{Na}^{+}$-linked symporter, but the structures of several homologous bacterial $\mathrm{Na}^{+} /$substrate symporters have provided considerable information about symporter function. The bacterial two- $\mathrm{Na}^{+} /$one-leucine symporter shown in Figure 11-27a consists of 12 membrane-spanning $\alpha$ helices. Two of the helices (numbers 1 and 6) have nonhelical segments in the middle of the membrane that form part of the leucine-binding site.

The amino acid residues involved in binding the leucine and the two $\mathrm{Na}^{+}$ions are located in the middle of the membrane-spanning segment (as depicted for the two- $\mathrm{Na}^{+} /$ one-glucose symporter in Figure 11-26) and are close together in three-dimensional space. This proximity suggests that the coupling of amino acid and ion transport in these transporters is the consequence of direct or nearly direct physical interactions of the substrates. Indeed, one of the $\mathrm{Na}^{+}$ions is bound to the carboxyl group of the transported leucine (Figure 11-27b). Thus neither substance can bind to the transporter without the other. Each of the two $\mathrm{Na}^{+}$ ions is bound to six oxygen atoms in the transporter. Sodium 1, for example, is bound to carbonyl oxygens of several transporter amino acids as well as to carbonyl oxygens and the hydroxyl oxygen of one threonine. Equally importantly, there are no water molecules surrounding either of the bound $\mathrm{Na}^{+}$atoms, as is the case for $\mathrm{K}^{+}$ions in potassium channels (see Figure 11-21). Thus as the $\mathrm{Na}^{+}$ions lose their water of hydration in binding to the transporter, they bind to six oxygen atoms with a similar geometry. This reduces the energy change required for the binding of $\mathrm{Na}^{+}$ions and prevents other ions, such as $\mathrm{K}^{+}$, from binding in place of $\mathrm{Na}^{+}$.

One striking feature of the structure depicted in Figure 11-27 is that the bound $\mathrm{Na}^{+}$ions and leucine are occluded-that is, they cannot diffuse out of the protein to either the surrounding extracellular or cytoplasmic media. This structure represents an intermediate in the transport process (see Figure 11-26) in which the transporter appears to be changing from a conformation with an exoplasmicfacing to one with a cytosolic-facing binding site.

## A $\mathrm{Na}^{+}$-Linked $\mathrm{Ca}^{2+}$ Antiporter Regulates the Strength of Cardiac Muscle Contraction

In all muscle cells, a rise in the cytosolic $\mathrm{Ca}^{2+}$ concentration triggers contraction. In cardiac muscle cells, a three- $\mathrm{Na}^{+} /$ one- $\mathrm{Ca}^{2+}$ antiporter, rather than the plasma-membrane $\mathrm{Ca}^{2+}$ ATPase discussed earlier, plays the principal role in maintaining a low concentration of $\mathrm{Ca}^{2+}$ in the cytosol. The transport reaction mediated by this cation antiporter can be written

![li404542480e](li404542480e.jpg)

FIGURE 11-27 Three-dimensional structure of the two- $\mathrm{Na}^{+} /$ one-leucine symporter from the bacterium Aquifex aeolicus. (a) The bound L-leucine, two $\mathrm{Na}^{+}$ions, and a $\mathrm{Cl}^{-}$ion are shown in yellow, purple, and green, respectively. The three membrane-spanning $\alpha$ helices that bind the $\mathrm{Na}^{+}$or the leucine are colored brown, blue, and orange. (b, c) Binding of the two $\mathrm{Na}^{+}$ions to carbonyl main-chain or

$$
3 \mathrm{Na}^{+}{ }_{\text {out }}+\mathrm{Ca}_{\mathrm{in}}^{2+} \rightleftharpoons 3 \mathrm{Na}^{+}{ }_{\text {in }}+\mathrm{Ca}^{2}{ }_{\text {out }}^{+}
$$

Note that the inward movement of three $\mathrm{Na}^{+}$ions is required to power the export of one $\mathrm{Ca}^{2+}$ ion from the cytosol, which has a $\left[\mathrm{Ca}^{2+}\right]$ of $-2 \times 10^{-7} \mathrm{M}$, to the extracellular medium, which has a $\left[\mathrm{Ca}^{2+}\right]$ of $-2 \times 10^{-3} \mathrm{M}$-a concentration gradient of some 10,000 -fold (higher on the outside). By lowering cytosolic $\mathrm{Ca}^{2+}$, operation of the $\mathrm{Na}^{+} / \mathrm{Ca}^{2+}$ antiporter reduces the strength of heart muscle contraction.

5The $\mathrm{Na}^{+} / \mathrm{K}^{+}$ATPase in the plasma membrane of cardiac muscle cells, as in other body cells, creates the $\mathrm{Na}^{+}$concentration gradient necessary for export of $\mathrm{Ca}^{2+}$ by the $\mathrm{Na}^{+}$-linked $\mathrm{Ca}^{2+}$ antiporter. As mentioned earlier, inhibition of the $\mathrm{Na}^{+} / \mathrm{K}^{+}$ATPase by the drugs ouabain and digoxin lowers the cytosolic $\mathrm{K}^{+}$concentration and, more importantly here, simultaneously increases cytosolic $\mathrm{Na}^{+}$. The resulting reduced $\mathrm{Na}^{+}$electrochemical gradient across the membrane causes the $\mathrm{Na}^{+}$-linked $\mathrm{Ca}^{2+}$ antiporter to function less efficiently. As a result, fewer $\mathrm{Ca}^{2+}$ ions are exported, and the cytosolic $\mathrm{Ca}^{2+}$ concentration increases, causing the muscle to contract more strongly. Because of their ability to increase the force of heart muscle contractions, drugs such as ouabain and digoxin that inhibit the $\mathrm{Na}^{+} / \mathrm{K}^{+}$ATPase are widely used in the treatment of congestive heart failure.
carboxyl side-chain oxygen atoms (red) that are part of helices 1 (brown), 6 (blue), or 8 (orange). It is important that one of the $\mathrm{Na}^{+}$ ions is also bound to the carboxyl group of the transported leucine (part b). See H. Krishnamurthy et al., 2009, Nature 459:347-355 for details on the structure and function of this and related $\mathrm{Na}^{+}$-linked symporters. [Data from A. Yamashita et al., 2005, Nature 437:215, PDB ID 2af5.]

## Several Cotransporters Regulate Cytosolic pH

The anaerobic metabolism of glucose yields lactic acid, and aerobic metabolism yields $\mathrm{CO}_{2}$, which combines with water to form carbonic acid $\left(\mathrm{H}_{2} \mathrm{CO}_{3}\right)$. These weak acids dissociate, yielding $\mathrm{H}^{+}$ions (protons); if these excess protons were not removed from cells, the cytosolic pH would drop precipitously, endangering cellular functions. Two types of cotransporters help remove some of the excess protons generated during metabolism in animal cells. One is a $\mathrm{Na}^{+} \mathrm{HCO}_{3}^{-} / \mathrm{Cl}^{-}$ antiporter, which imports one $\mathrm{Na}^{+}$ion, together with one $\mathrm{HCO}_{3}{ }^{-}$, in exchange for export of one $\mathrm{Cl}^{-}$ion. The cytosolic enzyme carbonic anhydrase catalyzes the dissociation of the imported $\mathrm{HCO}_{3}{ }^{-}$ion into $\mathrm{CO}_{2}$ and an $\mathrm{OH}^{-}$(hydroxyl) ion:

$$
\mathrm{HCO}_{3}^{-} \stackrel{\text { Carbonic }}{\rightleftharpoons} \mathrm{CO}_{2}+\mathrm{OH}^{-}
$$

The $\mathrm{OH}^{-}$ions combine with intracellular protons, forming water, and the $\mathrm{CO}_{2}$ diffuses out of the cell. Thus the overall action of this transporter is to consume cytosolic $\mathrm{H}^{+}$ions, thereby raising the cytosolic pH . Also important in raising cytosolic pH is a $\mathrm{Na}^{+} / \mathrm{H}^{+}$antiporter, which couples the movement of one $\mathrm{Na}^{+}$ion into the cell down its concentration gradient to the export of one $\mathrm{H}^{+}$ion.

Under certain circumstances, the cytosolic pH can rise beyond the normal range of 7.2-7.5. To cope with the excess

![li414542480e](li414542480e.jpg)

FIGURE 11-28 Carbon dioxide transport in blood requires a $\mathrm{Cl}^{-} / \mathrm{HCO}_{3}{ }^{-}$antiporter. (a) In systemic capillaries, carbon dioxide gas diffuses across the erythrocyte plasma membrane and is converted into soluble $\mathrm{HCO}_{3}{ }^{-}$by the enzyme carbonic anhydrase; at the same time, oxygen leaves the cell and hemoglobin binds a proton. The anion antiporter AE1 (purple) catalyzes the reversible exchange of $\mathrm{Cl}^{-}$and
$\mathrm{OH}^{-}$ions associated with elevated pH , many animal cells use an anion antiporter that catalyzes a one-for-one exchange of $\mathrm{HCO}_{3}{ }^{-}$for $\mathrm{Cl}^{-}$across the plasma membrane. At high pH , this $\mathrm{Cl}^{-} / \mathrm{HCO}_{3}{ }^{-}$antiporter exports one molecule of $\mathrm{HCO}_{3}{ }^{-}$(which can be viewed as a "complex" of $\mathrm{OH}^{-}$ and $\mathrm{CO}_{2}$ ) in exchange for the import of one molecule of $\mathrm{Cl}^{-}$, thus lowering the cytosolic pH . The movement of $\mathrm{Cl}^{-}$ down its concentration gradient ( $\mathrm{Cl}^{-}$exterior $>\mathrm{Cl}^{-}$cytosol; see Table 11-2) powers the export of $\mathrm{HCO}_{3}{ }^{-}$.

The activity of all three of these antiporters is regulated by the cytosolic pH , providing cells with a finely tuned mechanism for controlling cytosolic pH . The two antiporters that operate to increase cytosolic pH are activated when the pH of the cytosol falls. Similarly, a rise in pH above 7.2 stimulates the $\mathrm{Cl}^{-} / \mathrm{HCO}_{3}{ }^{-}$antiporter, leading to a more rapid export of $\mathrm{HCO}_{3}{ }^{-}$and a drop in the cytosolic pH . In this manner, the cytosolic pH of growing cells is maintained very close to pH 7.4 .

## An Anion Antiporter Is Essential for Transport of $\mathrm{CO}_{2}$ by Erythrocytes

Transmembrane anion exchange is essential for an important function of erythrocytes: the transport of waste $\mathrm{CO}_{2}$ from peripheral tissues to the lungs for exhalation. Waste $\mathrm{CO}_{2}$ released from cells into the capillary blood freely diffuses across the erythrocyte membrane (Figure 11-28a). In its gaseous form, $\mathrm{CO}_{2}$ dissolves poorly in aqueous solutions such as the cytosol or blood plasma, as is apparent to anyone who has opened a bottle of a carbonated beverage. However, the large amount of the potent enzyme carbonic anhydrase in the erythrocyte combines $\mathrm{CO}_{2}$ with hydroxyl ions $\left(\mathrm{OH}^{-}\right)$to form water-soluble bicarbonate $\left(\mathrm{HCO}_{3}{ }^{-}\right)$anions. This process occurs while erythrocytes are in systemic (tissue) capillaries and
![li424542480e](li424542480e.jpg)
$\mathrm{HCO}_{3}{ }^{-}$ions across the membrane. The overall reaction causes $\mathrm{HCO}_{3}{ }^{-}$ to be released from the cell, which is essential for maximal $\mathrm{CO}_{2}$ transport from the tissues to the lungs and for maintaining pH neutrality in the erythrocyte. (b) In the lungs, where carbon dioxide is excreted, the overall reaction is reversed. See text for additional discussion.
releasing oxygen into the blood plasma. The release of oxygen from hemoglobin induces a change in its conformation that enables a histidine side chain of a globin polypeptide to bind a proton. Thus when erythrocytes are in systemic capillaries, water is split into a proton that binds hemoglobin and an $\mathrm{OH}^{-}$that reacts with $\mathrm{CO}_{2}$ to form an $\mathrm{HCO}_{3}{ }^{-}$anion.

In a reaction catalyzed by the antiporter $A E 1$, cytosolic $\mathrm{HCO}_{3}{ }^{-}$is transported out of the erythrocyte in exchange for an entering $\mathrm{Cl}^{-}$anion:

$$
\mathrm{HCO}_{3 \text { in }}^{-}+\mathrm{Cl}^{-}{ }_{\text {out }} \rightleftharpoons \mathrm{HCO}_{3 \text { out }}^{-}+\mathrm{Cl}^{-}{ }_{\text {in }}
$$

(see Figure 11-28a). The entire anion-exchange process is completed within 50 milliseconds (ms), during which time $5 \times 10^{9} \mathrm{HCO}_{3}{ }^{-}$ions are exported from each cell down their concentration gradient. If anion exchange did not occur, then during periods such as exercise, when much $\mathrm{CO}_{2}$ is generated, $\mathrm{HCO}_{3}{ }^{-}$would accumulate inside the erythrocyte to toxic levels, as the cytosol would become alkaline. The exchange of $\mathrm{HCO}_{3}{ }^{-}$(equal to $\mathrm{OH}^{-}+\mathrm{CO}_{2}$ ) for $\mathrm{Cl}^{-}$causes the cytosolic pH to remain nearly neutral. Normally, about 80 percent of the $\mathrm{CO}_{2}$ in blood is transported as $\mathrm{HCO}_{3}{ }^{-}$ generated inside erythrocytes; anion exchange allows about two-thirds of this $\mathrm{HCO}_{3}{ }^{-}$to be transported by blood plasma external to the cells, increasing the amount of $\mathrm{CO}_{2}$ that can be transported from tissues to the lungs. In the lungs, where $\mathrm{CO}_{2}$ leaves the body, the overall direction of this anion-exchange process is reversed (Figure 11-28b).

AE1 catalyzes the precise one-for-one sequential exchange of anions on opposite sides of the plasma membrane required to preserve electroneutrality in the cell; only once every 10,000 or so transport cycles does an anion move unidirectionally from one side of the membrane to the other.

![li434542480e](li434542480e.jpg)

FIGURE 11-29 Concentration of ions and sucrose by the plant vacuole. The vacuolar membrane contains two types of proton pumps (orange): a V-class $\mathrm{H}^{+}$ATPase (left) and a pyrophosphate-hydrolyzing proton pump (right) that differs from all other ion pumps and is probably unique to plants. These pumps generate a low luminal pH as well as an inside-positive electric potential across the vacuolar membrane by their inward pumping of $\mathrm{H}^{+}$ions. The inside-positive potential powers the movement of $\mathrm{Cl}^{-}$and $\mathrm{NO}_{3}{ }^{-}$from the cytosol through separate channel proteins (purple). Proton antiporters (green), powered by the $\mathrm{H}^{+}$gradient, accumulate $\mathrm{Na}^{+}, \mathrm{Ca}^{2+}$, and sucrose inside the vacuole. See B. J. Barkla and O. Pantoja, 1996, Annu. Rev. Plant Phys. 47:159-184 and P. A. Rea et al., 1992, Trends Biochem. Sci. 17:348.

AE1 is composed of a membrane-embedded domain, folded into at least 12 transmembrane $\alpha$ helices, that catalyzes anion transport, and a cytosolic-facing domain that anchors certain cytoskeletal proteins to the membrane (see Figure 17-21).

## Numerous Transport Proteins Enable Plant Vacuoles to Accumulate Metabolites and Ions

The lumen of a plant vacuole is much more acidic ( $\mathrm{pH} 3-6$ ) than is the cytosol ( pH 7.5 ). The acidity of vacuoles is maintained by a V-class ATP-powered proton pump (see Figure 11-9) and by a pyrophosphate-powered proton pump that is unique to plants. Both of these pumps, located in the vacuolar membrane, import $\mathrm{H}^{+}$ions into the vacuolar lumen against a concentration gradient. The vacuolar membrane also contains $\mathrm{Cl}^{-}$and $\mathrm{NO}_{3}{ }^{-}$channels that transport these anions from the cytosol into the vacuole. Entry of these anions against their concentration gradients is driven by the inside-positive electric potential generated by the proton pumps. The combined operation of these proton pumps and anion channels produces an inside-positive electric potential of about 20 mV across the vacuolar membrane as well as a substantial pH gradient (Figure 11-29).

The proton electrochemical gradient across the plant vacuole membrane is used in much the same way as the $\mathrm{Na}^{+}$ electrochemical gradient across the animal-cell plasma membrane: to power the selective uptake or extrusion of ions
and small molecules by various antiporters. In the leaf, for example, excess sucrose generated by photosynthesis during the day is stored in the vacuole; during the night, the stored sucrose moves into the cytoplasm and is metabolized to $\mathrm{CO}_{2}$ and $\mathrm{H}_{2} \mathrm{O}$ with concomitant generation of ATP from ADP and $\mathrm{P}_{\mathrm{i}}$. A proton/sucrose antiporter in the vacuolar membrane operates to accumulate sucrose in plant vacuoles. The inward movement of sucrose is powered by the outward movement of $\mathrm{H}^{+}$, which is favored by its concentration gradient (lumen > cytosol) and by the cytosolic-negative potential across the vacuolar membrane (see Figure 11-29). Uptake of $\mathrm{Ca}^{2+}$ and $\mathrm{Na}^{+}$ into the vacuole from the cytosol against their concentration gradients is similarly mediated by proton antiporters.

Q1Our understanding of the transporters in plant vacuolar membranes has the potential for increasing agricultural production in soils with a high salt $(\mathrm{NaCl})$ concentration, which are found throughout the world. Because most agriculturally useful crops cannot grow in such saline soils, agricultural scientists have long sought to develop salt-tolerant plants by traditional breeding methods. With the availability of the cloned gene encoding the vacuolar $\mathrm{Na}^{+} / \mathrm{H}^{+}$antiporter, researchers can now produce transgenic plants that overexpress this transporter, leading to increased sequestration of $\mathrm{Na}^{+}$in the vacuole. For instance, transgenic tomato plants that overexpress the vacuolar $\mathrm{Na}^{+} / \mathrm{H}^{+}$antiporter can grow, flower, and produce fruit in the presence of soil NaCl concentrations that kill wild-type plants. Interestingly, although the leaves of these transgenic tomato plants accumulate large amounts of salt, the fruit has a very low salt content.

## KEY CONCEPTS OF SECTION 11.5

## Cotransport by Symporters and Antiporters

- The electrochemical gradient across a semipermeable membrane determines the direction of ion movement through transmembrane proteins. The two forces constituting the electrochemical gradient-the membrane electric potential and the ion concentration gradient-may act in the same or opposite directions (see Figure 11-25).
- Cotransporters use the energy released by movement of an ion (usually $\mathrm{H}^{+}$or $\mathrm{Na}^{+}$) down its electrochemical gradient to power the import or export of a small molecule or different ion against its concentration gradient.
- The cells lining the small intestine and kidney tubules contain symporters that couple the energetically favorable entry of $\mathrm{Na}^{+}$to the import of glucose against its concentration gradient (see Figure 11-26). Amino acids also enter cells by means of $\mathrm{Na}^{+}$-linked symporters.
- The molecular structure of a bacterial $\mathrm{Na}^{+}$/amino acid symporter reveals how binding of $\mathrm{Na}^{+}$and leucine are coupled and provides a snapshot of an occluded transport intermediate in which the bound substrates cannot diffuse out of the protein (see Figure 11-27).

- In cardiac muscle cells, the export of $\mathrm{Ca}^{2+}$ is coupled to and powered by the import of $\mathrm{Na}^{+}$by a cation antiporter, which transports three $\mathrm{Na}^{+}$ions inward for each $\mathrm{Ca}^{2+}$ ion exported.
- Two cotransporters that are activated at low pH help maintain the cytosolic pH in animal cells very close to 7.4 despite metabolic production of carbonic and lactic acids. One, a $\mathrm{Na}^{+} / \mathrm{H}^{+}$antiporter, exports excess protons. The other, a $\mathrm{Na}^{+} \mathrm{HCO}_{3}^{-} / \mathrm{Cl}^{-}$cotransporter, imports $\mathrm{HCO}_{3}^{-}$, which dissociates in the cytosol to yield pH -raising $\mathrm{OH}^{-}$ions.
- A $\mathrm{Cl}^{-} / \mathrm{HCO}_{3}{ }^{-}$antiporter that is activated when the cytosolic pH rises above normal decreases pH by exporting $\mathrm{HCO}_{3}{ }^{-}$.
- AE1, a $\mathrm{Cl}^{-} / \mathrm{HCO}_{3}{ }^{-}$antiporter in the erythrocyte membrane, increases the ability of blood to transport $\mathrm{CO}_{2}$ from tissues to the lungs (see Figure 11-28).
- Uptake of sucrose, $\mathrm{Na}^{+}, \mathrm{Ca}^{2+}$, and other substances into plant vacuoles is carried out by proton antiporters in the vacuolar membrane. Ion channels and proton pumps in the membrane are critical in generating a large enough proton concentration gradient to power these proton antiporters (see Figure 11-29).


### 11.6 Transcellular Transport

The previous sections have illustrated how several types of transporters function together to carry out important cellular functions. Here we extend this concept by focusing on the transport of several types of molecules and ions across polarized cells, which are cells that are asymmetric (have different "sides") and thus have biochemically distinct regions of the plasma membrane. A particularly well-studied class of polarized cells includes many of the epithelial cells that form sheet-like layers (epithelia) covering most external and internal surfaces of body organs. (Epithelial cells are discussed in greater detail in Chapter 20.) Like many epithelial cells, an intestinal epithelial cell involved in absorbing nutrients from the gastrointestinal tract has a plasma membrane organized into two major discrete regions: the surface that faces the outside of the organism, called the apical, or top, surface, and the surface that faces the inside of the organism (or the bloodstreamfacing side), called the basolateral surface, which is composed of the basal and lateral surfaces of the cell (see Figure 20-11).

Specialized regions of the epithelial-cell plasma membrane, called tight junctions, separate the apical and basolateral membranes and prevent many, but not all, water-soluble substances on one side from moving across to the other side through the extracellular space between cells. For this reason, absorption of many nutrients from the intestinal lumen across the epithelial cell layer and eventually into the blood occurs by a two-stage process called transcellular transport: import of molecules through the plasma membrane on the apical side of intestinal epithelial cells and their export through the plasma membrane on the basolateral (blood-facing) side
(Figure 11-30). The apical portion of the plasma membrane, which faces the intestinal lumen, is specialized for absorption of sugars, amino acids, and other molecules that are produced from food by multiple digestive enzymes.

## Multiple Transport Proteins Are Needed to Move Glucose and Amino Acids Across Epithelia

Figure 11-30, which depicts the proteins that mediate absorption of glucose from the intestinal lumen into the blood, illustrates the important concept that different types of proteins are localized to the apical and basolateral membranes of epithelial cells. In the first stage of this process, a two$\mathrm{Na}^{+}$/one-glucose symporter located in the apical membrane imports glucose, against its concentration gradient, from the intestinal lumen across the apical surface of the epithelial cells. As noted above, this symporter couples the energetically unfavorable inward movement of one glucose molecule to the energetically favorable inward transport of two $\mathrm{Na}^{+}$ ions (see Figure 11-26). In the steady state, all the $\mathrm{Na}^{+}$ions transported from the intestinal lumen into the cell during $\mathrm{Na}^{+}$/glucose symport, or the similar process of $\mathrm{Na}^{+}$/amino acid symport that also takes place on the apical membrane, are pumped out across the basolateral membrane, which faces the blood. Thus the low intracellular $\mathrm{Na}^{+}$concentration is maintained. The $\mathrm{Na}^{+} / \mathrm{K}^{+}$ATPase that accomplishes this pumping is found exclusively in the basolateral membrane of intestinal epithelial cells. The coordinated operation of these two transport proteins allows uphill movement of
![li444542480e](li444542480e.jpg)

FIGURE 11-30 Transcellular transport of glucose from the intestinal lumen into the blood. The $\mathrm{Na}^{+} / \mathrm{K}^{+}$ATPase in the basolateral surface membrane generates $\mathrm{Na}^{+}$and $\mathrm{K}^{+}$concentration gradients (step $\mathbf{E}$ ). The outward movement of $\mathrm{K}^{+}$ions through nongated $\mathrm{K}^{+}$ channels generates an inside-negative membrane potential across the entire plasma membrane. Both the $\mathrm{Na}^{+}$concentration gradient and the membrane potential are used to drive the uptake of glucose from the intestinal lumen by the two- $\mathrm{Na}^{+}$/one-glucose symporter located in the apical surface membrane (step $\mathbf{E}$ ). Glucose leaves the cell via facilitated transport catalyzed by GLUT2, a glucose uniporter located in the basolateral membrane (step $\mathbf{E}$ ).

glucose and amino acids from the intestine into the cell. This first stage in transcellular transport is ultimately powered by ATP hydrolysis by the $\mathrm{Na}^{+} / \mathrm{K}^{+}$ATPase.

In the second stage, the glucose and amino acids concentrated inside intestinal cells by apical symporters are exported down their concentration gradients into the blood via uniport proteins in the basolateral membrane. In the case of glucose, this movement is mediated by GLUT2 (see Figure 11-30). As noted earlier, this GLUT isoform has a relatively low affinity for glucose but increases its rate of transport substantially when the glucose gradient across the membrane rises (see Figure 11-4).

The net result of this two-stage process is movement of $\mathrm{Na}^{+}$ions, glucose, and amino acids from the intestinal lumen across the intestinal epithelium into the extracellular medium that surrounds the basolateral surface of intestinal epithelial cells, and eventually into the blood. Tight junctions between the epithelial cells prevent these molecules from diffusing back into the intestinal lumen. The increased osmotic pressure created by transcellular transport of salt, glucose, and amino acids across the intestinal epithelium draws water from the intestinal lumen, mainly through the tight junctions, into the extracellular medium that surrounds the basolateral surface; aquaporins do not appear to play a major role. In a sense, salts, glucose, and amino acids "carry" the water along with them.

## Simple Rehydration Therapy Depends on the Osmotic Gradient Created by Absorption of Glucose and $\mathrm{Na}^{+}$

An understanding of osmosis and the intestinal absorption of salt and glucose forms the basis for a simple therapy that saves millions of lives each year, particularly in developing countries. In these countries, cholera and other intestinal pathogens are major causes of death for young children. A toxin released by these bacteria activates chloride secretion from the apical surfaces of intestinal epithelial cells into the lumen; water follows osmotically, and the resultant massive loss of water causes diarrhea, dehydration, and ultimately death. A cure demands not only killing the bacteria with antibiotics but also rehydration: replacement of the water that is lost from the blood and other tissues.

Simply drinking water does not help because it is excreted from the gastrointestinal tract almost as soon as it enters. However, as we have just learned, the coordinated transport of glucose and $\mathrm{Na}^{+}$across the intestinal epithelium creates a transepithelial osmotic gradient, forcing water to move from the intestinal lumen across the epithelial cell layer and ultimately into the blood. Thus giving affected children a solution of sugar and salt to drink (but not sugar or salt alone) causes increased sodium and sugar transepithelial transport and, consequently, increased osmotic flow of water into the blood from the intestinal lumen, leading to rehydration. Similar sugar-salt solutions are the basis of popular drinks used by athletes to get sugar as well as water into the body quickly and efficiently.

## Parietal Cells Acidify the Stomach Contents While Maintaining a Neutral Cytosolic pH

The mammalian stomach contains a 0.1 M solution of hydrochloric acid $(\mathrm{HCl})$. This strongly acidic medium kills many ingested pathogens and denatures many ingested proteins so that they can be degraded by proteolytic enzymes (e.g., pepsin) that function at acidic pH . Hydrochloric acid is secreted into the stomach by specialized epithelial cells called parietal cells (also known as oxyntic cells) in the stomach lining. These cells contain a $\mathrm{H}^{+} / \mathrm{K}^{+}$ATPase in the apical membrane (which faces the stomach lumen) that generates a 1-million-fold $\mathrm{H}^{+}$concentration gradient: $\mathrm{pH} \sim 1.0$ in the stomach lumen versus $\mathrm{pH} \sim 7.2$ in the cell cytosol. This P-class ATP-powered ion pump is similar in structure and function to the plasma-membrane $\mathrm{Na}^{+} / \mathrm{K}^{+}$ATPase discussed earlier. The numerous mitochondria in parietal cells produce abundant ATP for use by the $\mathrm{H}^{+} / \mathrm{K}^{+}$ATPase.

If parietal cells simply exported $\mathrm{H}^{+}$ions in exchange for $\mathrm{K}^{+}$ions, the loss of protons would lead to a rise in the concentration of $\mathrm{OH}^{-}$ions in the cytosol and thus a marked increase in cytosolic pH . (Recall that $\left[\mathrm{H}^{+}\right] \times\left[\mathrm{OH}^{-}\right]$is always is a constant, $10^{-14} \mathrm{M}^{2}$.) Parietal cells avoid this rise in cytosolic pH in conjunction with acidification of the stomach lumen by using $\mathrm{Cl}^{-} / \mathrm{HCO}_{3}{ }^{-}$antiporters in the basolateral membrane to export the excess $\mathrm{OH}^{-}$ions from the cytosol to the blood. As noted earlier, these anion antiporters are activated at high cytosolic pH .

The overall process by which parietal cells acidify the stomach lumen is illustrated in Figure 11-31. In a reaction catalyzed by carbonic anhydrase, the excess cytosolic $\mathrm{OH}^{-}$ combines with $\mathrm{CO}_{2}$ that diffuses in from the blood, forming
![li454542480e](li454542480e.jpg)

FIGURE 11-31 Acidification of the stomach lumen by parietal cells in the gastric lining. The apical membrane of parietal cells contains a $\mathrm{H}^{+} / \mathrm{K}^{+}$ATPase (a P -class pump) as well as $\mathrm{Cl}^{-}$and $\mathrm{K}^{+}$channels. Note the cyclic $\mathrm{K}^{+}$transport across the apical membrane: $\mathrm{K}^{+}$ions are pumped inward by the $\mathrm{H}^{+} / \mathrm{K}^{+}$ATPase and exit via a $\mathrm{K}^{+}$channel. The basolateral membrane contains an anion antiporter that exchanges $\mathrm{HCO}_{3}{ }^{-}$and $\mathrm{Cl}^{-}$ions. The combined operation of these four different transport proteins and carbonic anhydrase acidifies the stomach lumen while maintaining the neutral pH of the cytosol.

$\mathrm{HCO}_{3}{ }^{-}$. This bicarbonate ion is exported across the basolateral membrane (and ultimately into the blood) by the $\mathrm{Cl}^{-} /$ $\mathrm{HCO}_{3}{ }^{-}$antiporter in exchange for a $\mathrm{Cl}^{-}$ion. The $\mathrm{Cl}^{-}$ions then exit through $\mathrm{Cl}^{-}$channels in the apical membrane, entering the stomach lumen. To preserve electroneutrality, each $\mathrm{Cl}^{-}$ion that moves into the stomach lumen across the apical membrane is accompanied by a $\mathrm{K}^{+}$ion that moves outward through a separate $\mathrm{K}^{+}$channel. In this way, the excess $\mathrm{K}^{+}$ions pumped inward by the $\mathrm{H}^{+} / \mathrm{K}^{+}$ATPase are returned to the stomach lumen, thus maintaining the normal intracellular $\mathrm{K}^{+}$concentration. The net result is secretion of equal amounts of $\mathrm{H}^{+}$and $\mathrm{Cl}^{-}$ions (i.e., HCl ) into the stomach lumen, while the pH of the cytosol remains neutral and the excess $\mathrm{OH}^{-}$ions, as $\mathrm{HCO}_{3}{ }^{-}$, are transported into the blood, where the change in pH is minimal.

## Bone Resorption Requires the Coordinated Function of a V-Class Proton Pump and a Specific Chloride Channel

Net bone growth in mammals subsides just after puberty, but a finely balanced, highly dynamic process of disassembly (resorption) and reassembly (bone formation) goes on throughout adulthood. Such continual bone remodeling permits the repair of damaged bones and can release calcium, phosphate, and other ions from mineralized bone into the blood for use elsewhere in the body.

Osteoclasts, the bone-dissolving cells, are macrophages, a type of cells best known for their role in protecting the body from infections. Osteoclasts are polarized cells that adhere to bone and form specialized, very tight seals between themselves and the bone, creating an enclosed extracellular space (Figure 11-32). An adhered osteoclast then secretes into this space a corrosive mixture of HCl and proteases that dissolves the inorganic components of the bone into $\mathrm{Ca}^{2+}$ and phosphate and digests its protein components. The mechanism of HCl secretion is similar to that used by the stomach to generate digestive juice (see Figure 11-31). As in gastric HCl secretion, carbonic anhydrase and an anion antiporter are important for osteoclast function. Osteoclasts employ a V-class proton pump to export $\mathrm{H}^{+}$ions into the bone-facing space, rather than the P -class $\mathrm{H}^{+} / \mathrm{K}^{+}$pump used by gastric epithelial cells.

The rare hereditary disease osteopetrosis, marked by increased bone density, is due to abnormally low bone resorption. Many patients have mutations in the gene encoding TCIRG1, a subunit of the osteoclast V-class proton pump, whose action is required to acidify the space between the osteoclast and the bone. Other patients have mutations in the gene encoding CIC-7, the chloride channel localized to the domain of the osteoclast plasma membrane that faces the space near the bone. As with lysosomes (see Figure 11-14), in the absence of a chloride channel, the proton pump cannot acidify the enclosed extracellular space, and thus bone resorption is defective.
![li464542480e](li464542480e.jpg)

FIGURE 11-32 Dissolution of bone by polarized osteoclast cells requires a V-class proton pump and the CIC-7 chloride channel. The osteoclast plasma membrane is divided into two domains separated by the tight seal between a ring of membrane and the bone surface. The membrane domain facing the bone contains V-class proton pumps and CIC-7 $\mathrm{Cl}^{-}$channels. The opposing membrane domain contains anion antiporters that exchange $\mathrm{HCO}_{3}{ }^{-}$and $\mathrm{Cl}^{-}$ions. The combined operation of these three transport proteins and carbonic anhydrase acidifies the enclosed space and allows bone resorption while maintaining the neutral pH of the cytosol. See R. Planells-Cases and T. Jentsch, 2009, Biochim. Biophys. Acta 1792:173 for discussion of CIC-7.

## KEY CONCEPTS OF SECTION 11.6

## Transcellular Transport

- The apical and basolateral plasma-membrane regions of epithelial cells contain different transport proteins and carry out quite different transport processes.
- In intestinal epithelial cells, the coordinated operation of $\mathrm{Na}^{+}$-linked symporters in the apical membrane and $\mathrm{Na}^{+} / \mathrm{K}^{+}$ ATPases and uniporters in the basolateral membrane mediates transcellular transport of amino acids and glucose from the intestinal lumen to the blood (see Figure 11-30).
- The increased osmotic pressure created by transcellular transport of salt, glucose, and amino acids across the intestinal epithelium draws water from the intestinal lumen into the body, a phenomenon that serves as the basis for rehydration therapy using sugar-salt solutions.
- The combined action of carbonic anhydrase and four different transport proteins permits parietal cells in the stomach lining to secrete HCl into the lumen while maintaining their cytosolic pH near neutrality (see Figure 11-31).
- Bone resorption requires coordinated function in osteoclasts of a V-class proton pump and the CIC-7 chloride channel (see Figure 11-32).

## LaunchPad

macmillan learning
Visit LaunchPad to access study tools and to learn more about the content in this chapter.

- Perspectives for the Future
- Classic Experiment 11-1: Stumbling upon Active Transport
- Analyze the Data
- Extended References
- Additional study tools, including videos, animations, and quizzes


## Key Terms

ABC superfamily 484
active transport 476
antiport 502
aquaporins 481
ATP-powered pump 476
cotransport 476
electrochemical gradient 475
facilitated transport 476
F-class pump 485
flippase 493
gated channel 476
GLUT proteins 480
hypertonic 481
hypotonic 481
isotonic 481
membrane potential 475
$\mathrm{Na}^{+} / \mathrm{K}^{+}$ATPase 489
patch clamping 500
P-class pump 484
resting membrane
potential 495
resting $\mathrm{K}^{+}$channel 497
sarcoplasmic reticulum
(SR) 486
simple diffusion 474
symport 502
tight junction 508
transcellular transport 508
transporter 476
uniport 478
V-class pump 485

## Review the Concepts

1. Nitric oxide (NO) is a gaseous molecule with lipid solubility similar to that of $\mathrm{O}_{2}$ and $\mathrm{CO}_{2}$. Endothelial cells lining arteries use NO to signal surrounding smooth muscle cells to relax, thereby increasing blood flow. What mechanism or mechanisms would transport NO from where it is produced in the cytoplasm of an endothelial cell into the cytoplasm of a smooth muscle cell, where it acts?
2. Acetic acid (a weak acid with a $\mathrm{p} K_{\mathrm{a}}$ of 4.75 ) and ethanol (an alcohol) are each composed of two carbons, hydrogen, and oxygen, and both enter cells by passive diffusion. At pH 7 , one is much more able to permeate a cellular membrane than the other. Which is more membrane permeable, and why? Predict how the membrane permeability of each is altered when the extracellular pH is reduced to 1.0 , a value typical of the stomach.
3. Uniporters and ion channels support facilitated transport across cellular membranes. Although both are examples of facilitated transport, the rates of ion movement via an ion
channel are roughly $10^{4}$ - to $10^{5}$-fold faster than the rates of molecule movement via a uniporter. What key mechanistic difference results in this large difference in transport rate? What contribution to free energy $(\Delta \mathrm{G})$ determines the direction of transport?
4. Name the three classes of membrane transport proteins. Explain which one or ones of these classes is able to move glucose and which can move bicarbonate $\left(\mathrm{HCO}_{3}^{-}\right)$against an electrochemical gradient. In the case of bicarbonate, but not glucose, the $\Delta G$ of the transport process has two terms. What are these two terms, and why does the second not apply to glucose? Why are cotransporters often referred to as examples of secondary active transport?
5. An $\mathrm{H}^{+}$ion is smaller than an $\mathrm{H}_{2} \mathrm{O}$ molecule, and a glycerol molecule, a three-carbon alcohol, is much larger. Both readily dissolve in $\mathrm{H}_{2} \mathrm{O}$. Why do aquaporins fail to transport $\mathrm{H}^{+}$whereas some can transport glycerol?
6. GLUT1, found in the plasma membrane of erythrocytes, is a classic example of a uniporter.
a. Design a set of experiments to prove that GLUT1 is indeed a glucose-specific uniporter rather than a galactoseor mannose-specific uniporter.
b. Glucose is a six-carbon sugar, and ribose is a five-carbon sugar. Despite its smaller size, ribose is not efficiently transported by GLUT1. How can this be explained?
c. A drop in blood sugar from 5 mM to 2.8 mM or below can cause confusion and fainting. Calculate the effect of this drop on glucose transport into cells expressing GLUT1.
d. How do liver and muscle cells maximize glucose uptake without changing $V_{\max }$ ?
e. Tumor cells expressing GLUT1 often have a higher $V_{\max }$ for glucose transport than do normal cells of the same type. How could these cells increase the $V_{\max }$ ?
f. Fat and muscle cells modulate the $V_{\max }$ for glucose uptake in response to insulin signaling. How?
7. Name the four classes of ATP-powered pumps that produce active transport of ions and molecules. Indicate which of these classes transport ions only and which transport primarily small organic molecules. The initial discovery of one class of these ATP-powered pumps came from studying the transport not of a natural substrate, but rather of artificial substrates used as cancer chemotherapy drugs. What do investigators now think are common examples of the natural substrates of this particular class of ATP-powered pumps?
8. Explain why the coupled reaction $\mathrm{ATP} \rightarrow \mathrm{ADP}+\mathrm{P}_{\mathrm{i}}$ in the P -class ion pump mechanism does not involve direct hydrolysis of the phosphoanhydride bond.
9. Describe a negative feedback mechanism for controlling a rising cytosolic $\mathrm{Ca}^{2+}$ concentration in cells that require rapid changes in $\mathrm{Ca}^{2+}$ concentration for normal functioning. How would a drug that inhibits calmodulin activity affect cytosolic $\mathrm{Ca}^{2+}$ concentration regulation by this mechanism? What would be the effect on the function of, for example, a skeletal muscle cell?

10. Certain proton pump inhibitors that inhibit secretion of stomach acid are among the most widely sold drugs in the world today. What pump does this type of drug inhibit, and where is this pump located?
11. The membrane potential in animal cells, but not in plants, depends largely on resting $\mathrm{K}^{+}$channels. How do these channels contribute to the resting membrane potential? Why are these channels considered to be nongated channels? How do these channels achieve selectivity for $\mathrm{K}^{+}$versus $\mathrm{Na}^{+}$, which is smaller than $\mathrm{K}^{+}$?
12. Patch clamping can be used to measure the conductance properties of individual ion channels. Describe how patch clamping can be used to determine whether or not the gene coding for a putative $\mathrm{K}^{+}$channel actually codes for a $\mathrm{K}^{+}$or a $\mathrm{Na}^{+}$channel.
13. Plants use the proton electrochemical gradient across the vacuole membrane to power the accumulation of salts and sugars in the organelle. This accumulation creates hypertonic conditions in the vacuole. Why does this not result in the plant cell swelling and bursting? Even under isotonic conditions, there is a slow leakage of ions into animal cells. How does the plasma-membrane $\mathrm{Na}^{+} / \mathrm{K}^{+}$ ATPase enable animal cells to avoid osmotic lysis under isotonic conditions?
14. In the case of the bacterial two- $\mathrm{Na}^{+} /$one-leucine symporter, what is the key distinguishing feature of the bound $\mathrm{Na}^{+}$ions that ensures that other ions, particularly $\mathrm{K}^{+}$, do not bind?
15. Describe the symport process by which cells lining the small intestine import glucose. What ion is responsible for the transport, and what two particular features facilitate the energetically favored movement of this ion across the plasma membrane?
16. Movement of glucose from one side to the other side of the intestinal epithelium is a major example of transcellular transport. How does the $\mathrm{Na}^{+} / \mathrm{K}^{+}$ATPase power the process? Why are tight junctions essential for the process? Why is localization of the transporters specifically in the apical or basolateral membrane crucial for transcellular transport? Rehydration supplements such as sport drinks include a sugar and a salt. Why are both important to rehydration?

## References

Facilitated Transport of Glucose and Water
Chen, L. Q., et al. 2015. Transport of sugars. Annu. Rev. Biochem. 84:865-894.

Deng, D., et al. N. 2014. Crystal structure of the human glucose transporter GLUT1. Nature 510:121-126.

Gonen, T., and T. Walz. 2006. The structure of aquaporins. Quart. Rev. Biophys. 39:361-396.

ATP-Powered Pumps and the Intracellular Ionic Environment
Aller, S., et al. 2009. Structure of P-glycoprotein reveals a molecular basis for poly-specific drug binding. Science 323: $1718-1722$.

Oldham, M. L., A. L. Davidson, and J. Chen. 2008. Structural insights into ABC transporter mechanism. Curr. Opin. Struct. Biol. 18:726-733.

Penmatsa, A., and E. Gouaux. 2014. How LeuT shapes our understanding of the mechanisms of sodium-coupled neurotransmitter transporters. J. Physiol. 592:863-869.

Ramsey, B. W., et al. 2011. A CFTR potentiator in patients with cystic fibrosis and the G551D mutation. N. Engl. J. Med. 365:1663-1672.

Shinoda, T., et al. 2009. Crystal structure of the sodium-potassium pump at $2.4 \AA$ resolution. Nature 459:446-450.

## Nongated Ion Channels and the Resting Membrane Potential

Gouaux, E., and R. Mackinnon. 2005. Principles of selective ion transport in channels and pumps. Science 310:1461-1465.

Hibino, H., et al. 2010. Inwardly rectifying potassium channels: their structure, function, and physiological roles. Physiol. Rev. 90:291-366.

Hille, B. 2001. Ion Channels of Excitable Membranes, 3d ed. Sinauer Associates.

## Cotransport by Symporters and Antiporters

Alper, S. L. 2009. Molecular physiology and genetics of $\mathrm{Na}^{+}$independent SLC4 anion exchangers. J. Exp. Biol. 212:1672-1683.

Gao, X., et al. 2009. Structure and mechanism of an amino acid antiporter. Science 324:1565-1568.

Krishnamurthy, H., C. L. Piscitelli, and E. Gouaux. 2009. Unlocking the molecular secrets of sodium-coupled transporters. Nature 459:347-355.

Wright, E. M. 2004. The sodium/glucose cotransport family SLC5. Pflug. Arch. 447:510-518.

## Transcellular Transport

Anderson, J. M., and C. M. Van Itallie. 2009. Physiology and function of the tight junction. Cold Spring Harb. Perspect. Biol. 1:a002584.

Rao, M. 2004. Oral rehydration therapy: new explanations for an old remedy. Annu. Rev. Physiol. 66:385-417.

Sobacchi, C., et al. 2013. Osteopetrosis: genetics, treatment and new insights into osteoclast function. Nat. Rev. Endocrinol. 9:522-536.

## New Perspectives

Liao, M., et al. 2014. Single particle electron cryo-microscopy of a mammalian ion channel. Curr. Opin. Struct. Biol. 27:1-7.

Stansfeld, P. J. and M. S. P. Sansom. 2011. Molecular simulation approaches to membrane proteins. Structure 19:1562-1572.

Wisedchaisri, G., S. L. Reichow, and T. Gonen. 2011. Advances in structural and functional analysis of membrane proteins by electron crystallography. Structure 19:1381-1393.

![li474542480e](li474542480e.jpg)

Fluorescence micrograph showing the yellow tubelike network of mitochondria in a human bone cancer (osteosarcoma) cell along with the DNA in the nucleus (aqua) and actin cytoskeletal fibers (purple). [Dylan Burnette and Jennifer Lippincott-Schwartz.]

From the growth and division of a cell to the beating of a heart to the electrical activity of a neuron that underlies thinking, life requires energy. Energy is defined as the capacity to do work, and on a cellular level, that work includes conducting and regulating a multitude of chemical reactions and transport processes, growing and dividing, generating and maintaining a highly organized structure, and interacting with other cells. This chapter describes the molecular mechanisms by which cells use sunlight or chemical nutrients as sources of energy, with a special focus on how cells convert these external sources of energy into a biologically universal intracellular chemical energy carrier, adenosine triphosphate, or ATP (Figure 12-1). ATP, found in all types of organisms and presumably present in the earliest life forms, is generated by the chemical addition of inorganic phosphate $\left(\mathrm{HPO}_{4}{ }^{2-}\right.$, often abbreviated as $\mathrm{P}_{\mathrm{i}}$ ) to adenosine diphosphate, or ADP, a process called phosphorylation. Cells use the energy released during hydrolysis of the terminal phosphoanhydride bond in ATP (see Figure 2-31) to power many otherwise energetically unfavorable processes. Examples include the synthesis of proteins from amino acids and of nucleic acids from nucleotides (see Chapter 4), the transport of molecules against a concentration gradient by ATP-powered pumps (see Chapter 11), the

## 12

## Cellular Energetics

contraction of muscles (see Chapter 17), and the beating of cilia (see Chapter 18). A key theme of cellular energetics is that proteins use, or "couple," energy released from one process (e.g., ATP hydrolysis) to drive another process (e.g., movement of molecules across membranes) that otherwise would be thermodynamically unfavorable.

The energy to drive ATP synthesis from ADP ( $\Delta G^{\circ \prime}$ $=7.3 \mathrm{kcal} / \mathrm{mol}$ ) derives primarily from two sources: the energy in the chemical bonds of nutrients and the energy in sunlight (see Figure 12-1). The two processes primarily responsible for converting these energy sources into ATP are aerobic oxidation (also known as aerobic respiration), which occurs in mitochondria in nearly all eukaryotic cells (see Figure 12-1, top), and photosynthesis, which occurs only in chloroplasts, found in the leaf cells of plants (see Figure 12-1, bottom) and in certain single-celled organisms, such as algae and cyanobacteria. Two additional processes, glycolysis and the citric acid cycle (see Figure 12-1, top), are also important direct or indirect sources of ATP in both animal and plant cells.

In aerobic oxidation, breakdown products of sugars (carbohydrates) and fatty acids (hydrocarbons)-both derived from the digestion of food in animals-are converted by oxidation with oxygen $\left(\mathrm{O}_{2}\right)$ to carbon dioxide $\left(\mathrm{CO}_{2}\right)$ and water

### 12.1 First Step of Harvesting Energy from Glucose: Glycolysis

### 12.2 The Structure and Functions of Mitochondria

12.3 The Citric Acid Cycle and Fatty Acid Oxidation

### 12.4 The Electron-Transport Chain and Generation of the Proton-Motive Force

### 12.5 Harnessing the Proton-Motive Force to Synthesize ATP

### 12.6 Photosynthesis and Light-Absorbing Pigments

12.7 Molecular Analysis of Photosystems

### 12.8 $\mathrm{CO}_{2}$ Metabolism During Photosynthesis

![li484542480e](li484542480e.jpg)

FIGURE 12-1 Overview of aerobic oxidation and photosynthesis. Eukaryotic cells use two fundamental mechanisms to convert external sources of energy into ATP. (Top) In aerobic oxidation, "fuel" molecules [primarily sugars and fatty acids (lipids)] undergo preliminary processing in the cytosol, such as breakdown of glucose to pyruvate (stage I), and are then transferred into mitochondria, where they are converted by oxidation with $\mathrm{O}_{2}$ to $\mathrm{CO}_{2}$ and $\mathrm{H}_{2} \mathrm{O}$ (stages II and III) and ATP is generated (stage IV). (Bottom) In photosynthesis, which occurs in chloroplasts, the radiant energy of light is absorbed by specialized pigments (stage 1);
the absorbed energy is used both to oxidize $\mathrm{H}_{2} \mathrm{O}$ to $\mathrm{O}_{2}$ and to establish conditions (stage 2) necessary for the generation of ATP (stage 3) and of carbohydrates from $\mathrm{CO}_{2}$ (carbon fixation, stage 4). Both mechanisms involve the production of reduced high-energy electron carriers (NADH, NADPH, $\mathrm{FADH}_{2}$ ) and the movement of electrons down an electric potential gradient in an electron-transport chain through specialized membranes. Energy released from these electrons is captured as a proton electrochemical gradient (proton-motive force) that is then used to drive ATP synthesis. Bacteria use comparable processes.
$\left(\mathrm{H}_{2} \mathrm{O}\right)$. The energy released from this overall reaction is transformed into the chemical energy of phosphoanhydride bonds in ATP. This process is analogous to burning wood (carbohydrates) or oil (hydrocarbons) to generate heat in furnaces or motion in automobile engines: both consume $\mathrm{O}_{2}$ and generate $\mathrm{CO}_{2}$ and $\mathrm{H}_{2} \mathrm{O}$. The key difference is that cells break the overall reaction down into many intermediate steps, with the amount of energy released in any given step closely matched to the amount of energy that can be stored—for example, as ATP-or that is required for the next intermediate step. If there were not such a close match, excess released energy would be lost as heat (which would be very inefficient), or not enough energy would be released to generate energy storage molecules such as ATP or to drive the next step in the process (which would be ineffective).

In photosynthesis, the radiant energy of light is absorbed by pigments such as chlorophyll and used to make ATP and carbohydrates-primarily sucrose and starch. Unlike aerobic oxidation, which uses carbohydrates and $\mathrm{O}_{2}$ to generate $\mathrm{CO}_{2}$, photosynthesis uses $\mathrm{CO}_{2}$ as a substrate and generates $\mathrm{O}_{2}$ and carbohydrates as products.

This reciprocal relationship between aerobic oxidation in mitochondria and photosynthesis in chloroplasts underlies a profound symbiotic relationship between photosynthetic and non-photosynthetic organisms. The oxygen generated during photosynthesis is the source of
virtually all the oxygen in the air, and the carbohydrates produced are the ultimate source of energy for virtually all non-photosynthetic organisms on earth. (An exception is bacteria living in deep-sea hydrothermal vents-and the organisms that feed on them-which obtain energy for converting $\mathrm{CO}_{2}$ into carbohydrates by oxidation of geologically generated reduced inorganic compounds released by the vents.)

At first glance, it might seem that the molecular mechanisms of photosynthesis and aerobic oxidation have little in common, besides the fact that they both produce ATP. However, a revolutionary discovery in cell biology established that bacteria, mitochondria, and chloroplasts all use the same mechanism, known as chemiosmosis, to generate ATP from ADP and $\mathrm{P}_{\mathrm{i}}$. In chemiosmosis (also known as chemiosmotic coupling), a proton electrochemical gradient across a membrane is first generated by energy released as electrons travel down their electric potential gradient through an electron-transport chain. The energy stored in this proton electrochemical gradient, called the protonmotive force, is then used to power the synthesis of ATP (Figure 12-2) or other energy-requiring processes. As protons move down their electrochemical gradient through the ATP synthesizing enzyme called ATP synthase, ATP is generated from ADP and $\mathrm{P}_{\mathrm{i}}$, a process that is the reverse of that mediated by the ATP-powered ion pumps discussed in Chapter 11.

![li494542480e](li494542480e.jpg)

FIGURE 12-2 The proton-motive force powers ATP synthesis.
Transmembrane proton concentration and electrical (voltage) gradients, collectively called the proton-motive force, are generated during aerobic oxidation and photosynthesis in both eukaryotes and prokaryotes (bacteria). High-energy electrons generated by light absorption by pigments (e.g., chlorophyll), or held in the reduced form of electron carriers (e.g., NADH, FADH ${ }_{2}$ ) made during the catabolism of sugars and lipids, pass down an electron-transport chain (blue arrows), releasing energy throughout the process. The released energy is used to pump protons across the membrane (red arrows), generating the protonmotive force. In chemiosmotic coupling, the energy released when protons flow down the gradient through ATP synthase drives the synthesis of ATP. The proton-motive force can also power other processes, such as the transport of metabolites across the membrane against their concentration gradient and rotation of bacterial flagella.

In this chapter, we explore the molecular mechanisms of the two processes that share this central mechanism, focusing first on aerobic oxidation and then on photosynthesis.

### 12.1 First Step of Harvesting Energy from Glucose: Glycolysis

In an automobile engine, hydrocarbon fuel is oxidatively and explosively converted in an essentially one-step process to mechanical work (i.e., driving a piston) plus the products $\mathrm{CO}_{2}$ and $\mathrm{H}_{2} \mathrm{O}$. The process is relatively inefficient in that substantial amounts of the chemical energy stored in the fuel are wasted, as they are converted to unused heat, and substantial amounts of fuel are only partially oxidized and are released as carbonaceous, sometimes toxic, exhaust. In the competition to survive, organisms cannot afford to squander their sometimes limited energy sources on an equivalently inefficient process and have therefore evolved a more efficient mechanism for converting fuel into
work. That mechanism, known as aerobic oxidation, provides the following advantages:

- By dividing the energy conversion process into multiple steps that generate several energy-carrying intermediates, chemical bond energy is efficiently channeled into the synthesis of ATP, with little energy lost as heat.
- Different fuels (sugars and fatty acids) are reduced to common intermediates that can then share subsequent pathways for combustion and ATP synthesis.
- Because the total energy stored in the bonds of the initial fuel molecules is substantially greater than that required to drive the synthesis of a single ATP molecule ( $\sim 7.3 \mathrm{kcal} / \mathrm{mol}$ ), many ATP molecules are produced.

An important feature of ATP production from the breakdown of nutrient fuels into $\mathrm{CO}_{2}$ and $\mathrm{H}_{2} \mathrm{O}$ (see Figure 12-1, top) is a set of reactions, called respiration, involving a series of oxidation and reduction reactions called an electron-transport chain. The combination of these reactions with phosphorylation of ADP to form ATP is called oxidative phosphorylation and occurs in mitochondria in nearly all eukaryotic cells. When oxygen is available and is used as the final recipient of the electrons transported via the electron-transport chain, the respiratory process that converts nutrient energy into ATP is called aerobic oxidation or aerobic respiration. Aerobic oxidation is an especially efficient way to maximize the conversion of nutrient energy into ATP because $\mathrm{O}_{2}$ is a relatively strong oxidant. If some molecule other than $\mathrm{O}_{2}$-for example, the weaker oxidants sulfate $\left(\mathrm{SO}_{4}{ }^{2-}\right)$ or nitrate $\left(\mathrm{NO}^{3-}\right)$-is the final recipient of the electrons in the electron-transport chain, the process is called anaerobic respiration. Anaerobic respiration is typical of some prokaryotic microorganisms. Although there are exceptions, most known multicellular (metazoan) eukaryotic organisms use aerobic oxidation to generate most of their ATP.

In our discussion of aerobic oxidation, we will be tracing the fate of the two main cellular fuels: sugars (principally glucose) and fatty acids. Under certain conditions-for example, starvation conditions-amino acids also feed into these metabolic pathways. We first consider glucose oxidation, then turn to fatty acids.

The complete aerobic oxidation of one molecule of glucose yields 6 molecules of $\mathrm{CO}_{2}$, and the energy released is coupled to the synthesis of as many as 30 molecules of ATP. The overall reaction is

$$
\begin{aligned}
& \mathrm{C}_{6} \mathrm{H}_{12} \mathrm{O}_{6}+6 \mathrm{O}_{2}+30 \mathrm{P}_{i}^{2-}+30 \mathrm{ADP}^{3-}+30 \mathrm{H}^{+} \rightarrow \\
& 6 \mathrm{CO}_{2}+30 \mathrm{ATP}^{4-}+36 \mathrm{H}_{2} \mathrm{O}
\end{aligned}
$$

Glucose oxidation in eukaryotes takes place in four stages (see Figure 12-1, top):
Stage I: Glycolysis In the cytosol, one 6-carbon glucose molecule is converted by a series of reactions to two 3-carbon

pyruvate molecules; a net of 2 ATPs are produced for each glucose molecule.

Stage II: Citric Acid Cycle In the mitochondrion, pyruvate oxidation to $\mathrm{CO}_{2}$ is coupled to the generation of the highenergy electron carriers NADH and $\mathrm{FADH}_{2}$, which store the energy for later use. These two carriers can be considered the sources of high-energy electrons.

Stage III: Electron-Transport Chain High-energy electrons flow down their electric potential gradient from NADH and $\mathrm{FADH}_{2}$ to $\mathrm{O}_{2}$ via membrane proteins that convert the energy released into a proton-motive force ( $\mathrm{H}^{+}$gradient). The energy released from the electrons pumps protons across a membrane, thus generating the gradient.

Stage IV: ATP Synthesis The proton-motive force powers the synthesis of ATP as protons flow down their concentration and voltage gradients through the ATP-synthesizing enzyme ATP synthase, which is embedded in a mitochondrial membrane. For each original glucose molecule, an estimated 28 additional ATPs are produced by this mechanism of oxidative phosphorylation.

In this section, we discuss stage I: the biochemical pathways that break down glucose into pyruvate in the cytosol. We also discuss how these pathways are regulated, and we contrast the metabolism of glucose under anaerobic and aerobic conditions. The ultimate fate of pyruvate, once it enters mitochondria, is discussed in Section 12.3.

## During Glycolysis (Stage I), Cytosolic Enzymes Convert Glucose to Pyruvate

Glycolysis, the first stage of glucose oxidation, occurs in the cytosol in both eukaryotes and prokaryotes; it does not require molecular oxygen $\left(\mathrm{O}_{2}\right)$ and is thus an anaerobic process. Glycolysis is an example of catabolism, the biological breakdown of complex substances into simpler ones. A set of 10 water-soluble cytosolic enzymes catalyze the reactions constituting the glycolytic pathway (glyco, "sweet"; lysis, "split"), in which one molecule of glucose is converted to two molecules of pyruvate (Figure 12-3). All the reaction intermediates produced by these enzymes are water-soluble, phosphorylated compounds called metabolic intermediates. In addition to chemically converting one glucose molecule into two pyruvates, the glycolytic pathway generates four ATP molecules by phosphorylation of four ADPs (steps $\mathbf{\square}$ and $\mathbf{\square}$ ). ATP is formed directly through the enzymecatalyzed joining of ADP with a P , that is derived from phosphorylated metabolic intermediates; this process is called substrate-level phosphorylation (to distinguish it from the oxidative phosphorylation that generates ATP in stages III and IV). Substrate-level phosphorylation in glycolysis, which does not involve the use of a proton-motive force, requires the prior addition (in steps $\mathbf{\square}$ and $\mathbf{\square}$ ) of two phosphates from two ATPs. These additions can be thought of as "pump priming" reactions, which introduce a little energy up front
in order to effectively recover more energy downstream. Thus glycolysis yields the net production of only two ATP molecules per glucose molecule.

The balanced chemical equation for the conversion of glucose to pyruvate shows that four hydrogen atoms (four protons and four electrons) are also released:
![li504542480e](li504542480e.jpg)
(For convenience, we show pyruvate here in its un-ionized form, pyruvic acid, although at physiological pH it would be largely dissociated.) All four electrons and two of the four protons are transferred (see Figure 12-3, step $\mathbf{\square}$ ) to two molecules of the oxidized form of nicotinamide adenine dinucleotide $\left(\mathrm{NAD}^{+}\right)$to produce the reduced form of the coenzyme, NADH (see Figure 2-33a):

$$
2 \mathrm{H}^{+}+4 \mathrm{e}^{-}+2 \mathrm{NAD}^{+} \rightarrow 2 \mathrm{NADH}
$$

Later we will see that the energy carried by the electrons in NADH and the analogous electron carrier $\mathrm{FADH}_{2}$, the reduced form of the coenzyme flavin adenine dinucleotide (FAD) (see Figure 2-33b), can be used to make additional ATPs via the electron-transport chain. The overall chemical equation for this first stage of glucose metabolism is

$$
\begin{gathered}
\mathrm{C}_{6} \mathrm{H}_{12} \mathrm{O}_{6}+2 \mathrm{NAD}^{+}+2 \mathrm{ADP}^{3-}+2 \mathrm{P}_{i}^{2-} \rightarrow \\
2 \mathrm{C}_{3} \mathrm{H}_{4} \mathrm{O}_{3}+2 \mathrm{NADH}+2 \mathrm{ATP}^{4-}
\end{gathered}
$$

After glycolysis, only a fraction of the energy available in glucose has been extracted and converted to ATP and NADH. The rest remains trapped in the covalent bonds of the two pyruvate molecules. The ability to efficiently convert the energy remaining in pyruvate to ATP depends on the presence of molecular oxygen. As we will see, energy conversion is substantially more efficient under aerobic conditions than under anaerobic conditions.

## The Rate of Glycolysis Is Adjusted to Meet the Cell's Need for ATP

To maintain appropriate levels of ATP, cells must control the rate of glucose catabolism. The operation of the glycolytic pathway (stage I), as well as the citric acid cycle (stage II), is continuously regulated, primarily by allosteric mechanisms (see Chapter 3 for general principles of allosteric control). Three allosteric enzymes involved in glycolysis play key roles in regulating the entire glycolytic pathway. Hexokinase (see Figure 12-3, step $\mathbf{\square}$ ) is inhibited by its reaction product, glucose 6 -phosphate. Pyruvate kinase (step $\mathbf{\square}$ ) is inhibited by ATP, so glycolysis slows down if too much ATP is present. The third enzyme, phosphofructokinase-1 (step $\mathbf{\square}$ ), is the principal rate-limiting enzyme of the glycolytic pathway. In a manner that is emblematic of its critical role in regulating the

FIGURE 12-3 The glycolytic pathway. A series of ten reactions degrades glucose to pyruvate. Two reactions consume ATP, forming ADP and phosphorylated sugars (red), two generate ATP from ADP by substrate-level phosphorylation (green), and one yields NADH by reduction of $\mathrm{NAD}^{+}$ (yellow). Note that all the intermediates between glucose and pyruvate are phosphorylated compounds. Steps 1, 2 and 3, with single arrows, are essentially irreversible (have large negative $\Delta G$ values) under ordinary conditions in cells.
![li514542480e](li514542480e.jpg)
rate of glycolysis, this enzyme is allosterically controlled by several molecules (Figure 12-4).

For example, phosphofructokinase-1 is allosterically inhibited by ATP and allosterically activated by adenosine monophosphate (AMP). As a result, the rate of glycolysis is very sensitive to the cell's energy charge, a measure of the fraction of total adenosine phosphates that have "high-energy" phosphoanhydride bonds, which is equal to $[(\mathrm{ATP})+0.5(\mathrm{ADP})] /[(\mathrm{ATP})+(\mathrm{ADP})+(\mathrm{AMP})]$. The allosteric inhibition of phosphofructokinase-1 by ATP may seem unusual because ATP is also a substrate of this enzyme. But the affinity of the substrate-binding site for ATP is much higher (has a lower $K_{\mathrm{m}}$ ) than that of the allosteric site. Thus at low concentrations, ATP binds to the catalytic site, but not to the inhibitory allosteric site, and enzymatic catalysis proceeds at near-maximal rates. At high concentrations, ATP also binds to the allosteric site, inducing a conformational change that reduces the affinity of the enzyme for its other substrate, fructose 6-phosphate, and thus reduces the rate of this reaction and the overall rate of glycolysis.

Another important allosteric activator of phosphofructokinase-1 is fructose 2,6-bisphosphate. This metabolite is formed from fructose 6-phosphate by an enzyme called phosphofructokinase-2. Fructose 6-phosphate accelerates the formation of fructose 2,6-bisphosphate, which in turn activates phosphofructokinase-1. This type of control is known as feed-forward activation, in which a high abundance of a metabolite (here, fructose 6-phosphate) accelerates its subsequent metabolism. Fructose 2,6-bisphosphate allosterically
activates phosphofructokinase-1 in liver cells by decreasing the inhibitory effect of high ATP concentrations and by increasing the affinity of phosphofructokinase-1 for one of its substrates, fructose 6-phosphate.

The three glycolytic enzymes that are regulated by allostery catalyze reactions with large negative $\Delta G^{\circ \prime}$ valuesreactions that are essentially irreversible under ordinary conditions. These enzymes are therefore particularly suitable for

![li524542480e](li524542480e.jpg)

FIGURE 12-4 Allosteric regulation of glucose metabolism. The key regulatory enzyme in glycolysis, phosphofructokinase-1, is allosterically activated by AMP and fructose 2,6-bisphosphate, which are elevated when the cell's energy stores are low. The enzyme is inhibited by ATP and citrate, both of which are elevated when the cell is actively oxidizing glucose to $\mathrm{CO}_{2}$ (i.e., when energy stores are high). Later we will see that citrate is generated during stage II of glucose oxidation. Phosphofructokinase-2 is
a bifunctional enzyme: its kinase activity forms fructose 2,6-bisphosphate from fructose 6-phosphate, and its phosphatase activity catalyzes the reverse reaction. Insulin, which is released by the pancreas when blood glucose levels are high, promotes phosphofructokinase-2 kinase activity and thus stimulates glycolysis. At low blood glucose, glucagon is released by the pancreas and promotes phosphofructokinase-2 phosphatase activity in the liver, indirectly slowing down glycolysis.
regulating the entire glycolytic pathway. Additional control is exerted by glyceraldehyde 3-phosphate dehydrogenase, which catalyzes the reduction of $\mathrm{NAD}^{+}$to NADH (see Figure 12-3, step $\mathbf{\square}$ ). As we shall see, NADH is a high-energy electron carrier used subsequently during oxidative phosphorylation in mitochondria. If cytosolic NADH builds up owing to a slowdown in mitochondrial oxidation, step $\mathbf{\square}$ becomes thermodynamically less favorable.

Glucose metabolism is controlled differently in various mammalian tissues to meet the metabolic needs of the organism as a whole. During periods of carbohydrate starvation, for instance, it is necessary for the liver to release glucose into the bloodstream. To do this, the liver converts the polymer glycogen, a storage form of glucose (see Chapter 2), directly to glucose 6-phosphate (without involvement of hexokinase, step $\mathbf{\square}$ ). Under these conditions, there is a reduction in fructose 2,6-bisphosphate levels and decreased phosphofructokinase-1 activity (see Figure 12-4). As a result, glucose 6-phosphate derived from glycogen is not metabolized to pyruvate; rather, it is converted to glucose by a phosphatase and released into the blood to nourish the brain and red blood cells, which depend primarily on glucose for their energy. In all cases, the activity of these regulated enzymes is controlled by the level of small-molecule metabolites, generally by allosteric interactions, or by hormone-mediated phosphorylation and dephosphorylation reactions. (Chapter 15 gives a more detailed discussion of hormonal control of glucose metabolism in liver and muscle.)

## Glucose Is Fermented When Oxygen Is Scarce

Many eukaryotes, including humans, are obligate aerobes: they grow only in the presence of molecular oxygen and can metabolize glucose (or related sugars) completely to $\mathrm{CO}_{2}$, with the concomitant production of a large amount of ATP. Most eukaryotes, however, can generate some

ATP by anaerobic metabolism. A few eukaryotes are facultative anaerobes: they grow in either the presence or the absence of oxygen. Annelids (segmented worms), mollusks, and some yeasts, for example, can survive without oxygen, relying on the ATP produced by fermentation.

In the absence of oxygen, yeasts convert the pyruvate produced by glycolysis to one molecule each of ethanol and $\mathrm{CO}_{2}$; in these reactions, two NADH molecules are oxidized to $\mathrm{NAD}^{+}$for every two pyruvates converted to ethanol, thereby regenerating the supply of $\mathrm{NAD}^{+}$, which is necessary for glycolysis to continue (Figure 12-5a, left). This anaerobic catabolism of glucose, called fermentation, is the basis of beer and wine production.

Fermentation also occurs in animal cells, although lactic acid, rather than alcohol, is the product. During prolonged contraction of mammalian skeletal muscle cells-for example, during exercise-oxygen can become scarce within the muscle tissue. As a consequence, glucose catabolism is limited to glycolysis, and muscle cells convert pyruvate to two molecules of lactic acid by a reduction reaction that also oxidizes two NADHs to two $\mathrm{NAD}^{+}$s (Figure 12-5a, right). Although the lactic acid is released from the muscle into the blood, if the contractions are sufficiently rapid and strong, the lactic acid can transiently accumulate in the tissue and contribute to muscle and joint pain during exercise. Once it is secreted into the blood, some of the lactic acid passes into the liver, where it is reoxidized to pyruvate and either further metabolized to $\mathrm{CO}_{2}$ aerobically or converted back to glucose. Much lactate is metabolized to $\mathrm{CO}_{2}$ by the heart, which is highly perfused by blood and can continue aerobic metabolism at times when exercising, oxygen-poor skeletal muscles secrete lactate. If too much lactic acid accumulates in the blood, the acid causes an unhealthy decrease in the pH of the blood (lactic acidosis). Lactic acid bacteria (the organisms that spoil milk) and other prokaryotes also generate ATP by the fermentation of glucose to lactic acid.

![li534542480e](li534542480e.jpg)

Overall reaction of aerobic metabolism:
Glucose $+6 \mathrm{O}_{2}+-30 \mathrm{ADP}+-30 \mathrm{P}_{\mathrm{i}} \longrightarrow$ $6 \mathrm{CO}_{2}+36 \mathrm{H}_{2} \mathrm{O}+-30 \mathrm{ATP}$

FIGURE 12-5 Anaerobic versus aerobic metabolism of glucose. The ultimate fate of pyruvate formed during glycolysis depends on the presence or absence of oxygen. (a) In the absence of oxygen, pyruvate is only partially degraded and no further ATP is made. However, two electrons are transferred from each NADH molecule produced during glycolysis to an acceptor molecule to regenerate $\mathrm{NAD}^{+}$, which is required for continued glycolysis. In yeast (left), acetaldehyde is the electron acceptor and ethanol is the product. This process is called alcoholic fermentation. When oxygen is scarce in muscle cells (right), NADH reduces pyruvate to form lactic acid, regenerating $\mathrm{NAD}^{+}$, a process called lactic acid fermentation. (b) In the presence of oxygen, pyruvate is transported into mitochondria, where it is first converted by pyruvate dehydrogenase into one molecule of $\mathrm{CO}_{2}$ and one of acetic acid, the latter linked to coenzyme A (CoA-SH) to form acetyl CoA, concomitant with reduction of one molecule of $\mathrm{NAD}^{+}$to NADH. Further metabolism of acetyl CoA and NADH generates approximately an additional 28 molecules of ATP per glucose molecule oxidized.

Fermentation is a much less efficient way to generate ATP than aerobic oxidation and therefore occurs in animal cells only when oxygen is scarce. In the presence of oxygen, pyruvate formed by glycolysis is transported into mitochondria, where it is oxidized by $\mathrm{O}_{2}$ to $\mathrm{CO}_{2}$ and $\mathrm{H}_{2} \mathrm{O}$ via the series of reactions outlined in Figure 12-5b. This aerobic metabolism of glucose, which occurs in stages II-IV of the process outlined in Figure 12-1, generates an estimated 28 additional ATP molecules per original glucose molecule, far outstripping the ATP yield from anaerobic glucose metabolism (fermentation).

To understand how ATP is generated so efficiently by aerobic oxidation, we must consider first the structure and function of the organelle responsible, the mitochondrion. Mitochondria, and the reactions that take place within them, are the subjects of the next section.

## KEY CONCEPTS OF SECTION 12.1

## First Step of Harvesting Energy from Glucose: Glycolysis

- In a process known as aerobic oxidation, cells convert the energy released by the oxidation of glucose or fatty acids into the terminal phosphoanhydride bond of ATP.
- The complete aerobic oxidation of each molecule of glucose produces 6 molecules of $\mathrm{CO}_{2}$ and approximately 30 ATP molecules. The entire process, which starts in the cytosol and is completed in the mitochondrion, can be divided into four stages: (I) degradation of glucose to pyruvate in the cytosol (glycolysis); (II) pyruvate oxidation to $\mathrm{CO}_{2}$ in the mitochondrion coupled to generation of the high-energy electron carriers NADH and $\mathrm{FADH}_{2}$ (via the citric acid cycle); (III) electron transport to generate a proton-motive force together with conversion of molecular oxygen to water; and (IV) ATP synthesis (see Figure 12-1). From each glucose molecule, two ATPs are generated by stage I and approximately 28 from stages II-IV.
- In glycolysis (stage I), cytosolic enzymes convert glucose to two molecules of pyruvate and generate two molecules each of NADH and ATP (see Figure 12-3).
- The rate of glucose oxidation via glycolysis is regulated by the inhibition or stimulation of several enzymes, depending on the cell's need for ATP (see Figure 12-4). Glucose is stored, for example, as glycogen, when ATP is abundant.
- In the absence of oxygen (anaerobic conditions), cells can metabolize pyruvate to lactic acid or (in the case of yeast) to ethanol and $\mathrm{CO}_{2}$, in the process converting NADH back to $\mathrm{NAD}^{+}$, which is necessary for continued glycolysis. In the presence of oxygen (aerobic conditions), pyruvate is transported into the mitochondrion, where it is metabolized to $\mathrm{CO}_{2}$, in the process generating abundant ATP (see Figure 12-5).


### 12.2 The Structure and Functions of Mitochondria

Oxygen-producing photosynthetic cyanobacteria first appeared about 2.7 billion years ago. The subsequent buildup of oxygen in Earth's atmosphere during the next approximately 1 billion years opened the way for organisms to evolve the very efficient aerobic oxidation pathway, which in turn permitted the evolution, especially during what is called the Cambrian explosion, of large and complex body forms and associated metabolic activities. In eukaryotic cells, aerobic oxidation is carried out by mitochondria (stages II-IV). In effect, mitochondria are ATP-generating factories, taking full advantage of this plentiful oxygen. We first describe their structure, dynamics, and diverse functions, and then the reactions they employ to degrade pyruvate and make ATP.

## Mitochondria Are Multifunctional Organelles

Mitochondria (Figure 12-6) are among the larger organelles in a eukaryotic cell. An individual mitochondrion is about the size of an E. coli bacterium, which is not surprising because, as we will see below, bacteria are thought to be the evolutionary precursors of mitochondria. Most eukaryotic cells contain many mitochondria, which may be fused to one another and may collectively occupy as much as 25 percent of the volume of the cytoplasm. The numbers of mitochondria in a cell-hundreds to thousands in mammalian cells-are regulated to match the cell's requirements for ATP (e.g., specialized stomach cells that use a lot of ATP for acid secretion have many mitochondria).

Although our focus in this chapter is on the mechanisms by which mitochondria generate ATP, it is important to recognize that mitochondria participate in a wide variety of critical cellular processes (Table 12-1). Mitochondria are involved with the biosynthesis of many small molecules. In several cases, some steps in these biosynthetic pathways take place within the mitochondria and some outside (extramitochondrial steps), requiring precursors and products to be shuttled into and out of the mitochondria via transport proteins. For example, in many eukaryotes, mitochondria host the first rate-limiting step in heme biosynthesis. The product of this first step is then exported for additional modification in the cytoplasm, producing a precursor that must then be imported into the mitochondria for the terminal steps. The biosynthesis in the cytoplasm of a wide variety of small molecules depends on mitochondria. Mitochondria provide to the cytoplasm small organic molecules-for example, citrate, isocitrate, malate, formate, and $\alpha$-ketoglutarate-that can be used to generate NADPH (an energy source), acetyl CoA (a carbon source), or other precursors for the extramitochondrial biosynthesis of molecules such as glutathione, purines, fatty acids, and cholesterol.

## Mitochondria Have Two Structurally and Functionally Distinct Membranes

The details of mitochondrial structure (see Figure 12-6) can be observed with electron microscopy (see Figure 1-20).

![li544542480e](li544542480e.jpg)

FIGURE 12-6 Internal structure of a mitochondrion.
(a) Schematic diagram showing the principal membranes and compartments. The smooth outer membrane forms the outside boundary of the mitochondrion. The inner membrane is apparently a single continuous membrane that has three distinct domains: boundary membrane, cristae, and crista junctions. The boundary membrane is flat and lies immediately below and adjacent to the outer membrane. The cristae are sheet-like and tubelike invaginations that extend from the boundary membrane into the center of the mitochondrion. The sharp bends that form the connection between the boundary membrane and the cristae are called crista junctions. The intermembrane space is continuous with the lumen of each crista. The $F_{0} F_{1}$ complexes (small red spheres), which synthesize ATP, are intramembrane particles that protrude from the cristae and inner membrane into the matrix. The matrix contains the mitochondrial DNA (blue strands), ribosomes (small blue spheres), and granules (large yellow spheres).
(b) Computer-generated model of a section of a mitochondrion from chicken brain. This model is based on a three-dimensional electron
microscopic image calculated from a series of two-dimensional electron micrographs recorded at regular intervals. This technique is analogous to a three-dimensional x-ray tomogram or CAT scan used in medical imaging. Note the tightly packed cristae (yellow-green), the inner membrane (light blue), and the outer membrane (dark blue). (c) Cristae and crista junctions from human fibroblasts were visualized and modeled using electron microscopy and tomography. The right panel shows one of the multiple sections through the mitochondrion imaged using transmission electron microscopy, with the mitochondrial membranes clearly distinguished. The sharp bends in the inner membrane at the junctions (dotted circles) that separate the crista membranes from the inner boundary membrane are seen clearly. The left panel shows a three-dimensional tomographic model of the laminar cristae seen edge on (green) and crista junctions (orange spheres) overlaid with the EM image. [Part (b) T. G. Frey and G. A. Perkins. Part (c) from: Proc. Natl. Acad. Sci. USA 2013. 110 (22): 8936-8941, Fig. 6. Fig. 6A and C, "STED super-resolution microscopy reveals an array of MINOS clusters along human mitochondria," by Jans et al.]

# Biosynthesis or processing of small molecules 

Fatty acids
Steroid hormones
Pyrimidines
Iron-sulfur clusters
Heme
Phospholipids (phosphatidylethanolamine, phosphatidylglycerol, cardiolipin)
Ubiquinone
Amino acids (synthesis, interconversion, and catabolism)

## Other mitochondrial functions

Oxidative phosphorylation and ATP synthesis
Reactive oxygen species (ROS) homeostasis
Ion homeostasis (e.g., calcium)
Ammonia detoxification
Fatty acid oxidation
Thermogenesis (heat generation) in brown fat
Contributions to innate immunity and inflammation
Regulated cell death pathways (e.g., apoptosis)

## Cellular processes influenced by mitochondria-associated membranes (MAMs)

Mitochondrial shape and dynamics
PINK1/Parkin-dependent mitophagy (initiated at MAMs)
Calcium transport into the mitochondria
Calcium homeostasis and calcium-mediated signaling
Glucose and energy metabolism
Mitochondrial import from the ER of lipids, including phosphatidylserine and possibly cholesterol
Mitochondrial biosynthesis of lipids, including phosphatidylethanolamine and steroid hormones
Responses to stress
Cell survival via regulated cell death (see Chapter 21)
Inflammatory responses via the inflammasome and innate immune responses (see Chapter 23)
Pathways implicated in viral infections (cytomegalovirus, hepatitis C virus)
Neurodegenerative pathology (Alzheimer's and Parkinson's diseases)

Each mitochondrion has two distinct, concentric membranes: the inner and outer mitochondrial membranes. The outer mitochondrial membrane defines the smooth outer perimeter of the mitochondrion. The inner mitochondrial membrane lies immediately underneath the outer membrane. The inner mitochondrial membrane is a single continuous membrane that itself can be considered to have three compositionally and structurally distinct domains. The boundary membrane is the flat inner mitochondrial membrane that lies immediately inside and adjacent to the outer membrane. The cristae are the numerous invaginations that extend from the boundary membrane at the perimeter into the center of the mitochondrion. The connection between the inner boundary membrane and a crista is called a crista junction (Figure 12-6a and c).

The lengths of the cristae and their structures (which may be tubular in shape or flat and pancake-like) can vary within a mitochondrion. The crista junctions and the edges and tips of the cristae are highly curved. The curvature of the crista junctions (see Figure 12-6c) is due to a protein complex called MICOS (mitochondrial contact site and cristae organizing system), which has an integral membrane protein subunit that homo-oligomerizes and bends the inner membrane to produce high curvature. MICOS also mediates close juxtaposition of the outer membrane and inner membrane by binding to outer membrane-associated proteins. Additionally, MICOS appears to function as a diffusion barrier to prevent mixing of the distinct proteins and lipids in the boundary membrane and cristal membranes. We will discuss

the molecular basis of the curvature at the edges and tips of the cristae in Section 12.5.

The outer and inner membranes topologically define two submitochondrial compartments: the intermembrane space, between the outer and inner membranes, and the matrix, or central compartment, which forms the lumen within the inner membrane (see Figure 12-6a). Many of the proteins directly involved with transforming the energy of nutrients into the energy stored in ATP, such as the proteins of the electron-transport chain and ATP synthase, are located in the inner mitochondrial membrane. The invaginating cristae greatly expand the surface area of the inner mitochondrial membrane, thus increasing the mitochondrion's capacity to synthesize ATP. In typical liver mitochondria, for example, the area of the inner membrane, including cristae, is about five times that of the outer membrane. In fact, the total area of all inner mitochondrial membranes in liver cells is about 17 times that of the plasma membranes. The mitochondria in heart and skeletal muscle contain three times as many cristae as are found in typical liver mitochondria-presumably reflecting the greater demand for ATP by muscle cells.

Fractionation and purification of mitochondrial membranes and compartments have made it possible to determine their protein, DNA, and phospholipid compositions and to localize each enzyme-catalyzed reaction to a specific membrane or compartment. Over a thousand different types of polypeptides are required to make and maintain mitochondria and permit them to function. Detailed biochemical analysis has established that there are at least 1098 proteins in mammalian mitochondria and perhaps as many at 1500. Defective functioning of these mitochondria-associated proteins-due, for example, to inherited genetic mutationsleads to over 250 human diseases. The most common of these are electron-transport chain diseases, which result from mutations in any one of 150 genes and exhibit a very wide variety of clinical abnormalities affecting muscles, the heart, the nervous system, and the liver, among other physiological systems. Other mitochondria-associated diseases include Miller syndrome, which results in multiple anatomic malformations, and connective tissue defects.

The most abundant protein in the outer mitochondrial membrane is a mitochondrial $\beta$-barrel porin called VDAC (voltage-dependent anion channel), a multifunctional transmembrane channel protein that is similar in structure to bacterial porins (see Figure 7-18). Ions and most small hydrophilic molecules (up to about 5000 Da ) can readily pass through these channel proteins when they are open. Although there may be metabolic regulation of the opening of mitochondrial porins and thus of the flow of metabolites across the outer membrane, the inner membrane is the major permeability barrier between the cytosol and the mitochondrial matrix, controlling the rate of mitochondrial oxidation and ATP generation.

Proteins constitute 76 percent of the total mass of the inner mitochondrial membrane-a higher fraction than in any other cellular membrane. Many of these proteins are key participants in oxidative phosphorylation. They include ATP synthase, proteins responsible for electron transport, and a wide
variety of transport proteins that permit the movement of metabolites between the cytosol and the mitochondrial matrix. The human genome encodes 48 members of one family of mitochondrial transport proteins. One of these, the ADP/ATP carrier, is an antiporter that moves newly synthesized ATP out of the matrix and into the inner membrane space (and subsequently the cytosol) in exchange for ADP originating from the cytosol. Without this essential antiporter, the energy trapped in the chemical bonds of mitochondrial ATP made in the matrix would not be available to the rest of the cell.

Keep in mind that plants, as well as animals, have mitochondria and perform aerobic oxidation. In plants, stored carbohydrates, mostly in the form of starch, are hydrolyzed to glucose. Glycolysis then produces pyruvate that is transported into mitochondria, as in animal cells. Mitochondrial oxidation of pyruvate and concomitant formation of ATP occur in photosynthetic cells during dark periods when photosynthesis is not possible, and in roots and other non-photosynthetic tissues at all times.

The inner mitochondrial membrane and matrix are the sites of most reactions involved in the oxidation of pyruvate and fatty acids to $\mathrm{CO}_{2}$ and $\mathrm{H}_{2} \mathrm{O}$ and the coupled synthesis of ATP from ADP and $\mathrm{P}_{\mathrm{i}}$. Each of these reactions occurs in a discrete membrane or space in the mitochondrion (see Figure 12-16 below).

## Mitochondria Contain DNA Located in the Matrix

Although the vast majority of DNA in most eukaryotes is found in the nucleus, some DNA is present within the mitochondria of animals, plants, and fungi and within the chloroplasts of plants. Many lines of evidence indicate that mitochondria and chloroplasts evolved from eubacteria that were engulfed into ancestral cells containing a eukaryotic nucleus, forming endosymbionts (Figure 12-7). Over evolutionary time, most of the bacterial genes were lost from organelle DNA. Some, such as genes encoding proteins involved in nucleotide, lipid, and amino acid biosynthesis, were lost because their functions were provided by genes in the nucleus of the host cell. Other genes encoding components of the present-day organelles were transferred to the nucleus. However, mitochondria and chloroplasts in today's eukaryotes retain DNA encoding some proteins essential for organelle function as well as the ribosomal and transfer RNAs required for synthesis of those proteins. Thus eukaryotic cells have multiple genetic systems: a predominant nuclear system and secondary systems with their own DNA, ribosomes, and tRNAs in mitochondria and chloroplasts.

The mitochondrial DNA (mtDNA) is located in the mitochondrial matrix (see Figure 12-6). As judged by the number of yellow fluorescent "dots" of mtDNA, a Euglena gracilis cell—a simple, single-celled eukaryote-contains at least 30 mtDNA molecules (Figure 12-8). Replication of mtDNA and division of the mitochondria can be followed in live cells using time-lapse microscopy. Such studies show that in most organisms, mtDNA replicates throughout interphase. At mitosis, each daughter cell receives approximately the same

![li554542480e](li554542480e.jpg)

FIGURE 12-7 Endosymbiont hypothesis for the evolutionary origin of mitochondria and chloroplasts. Endocytosis of a bacterium by an ancestral eukaryotic cell (step (1) would generate an organelle with two membranes, the outer membrane derived from the eukaryotic plasma membrane and the inner one from the bacterial membrane (step (2). Proteins localized to the ancestral bacterial membrane would retain their orientation, such that the portion of the protein once facing the extracellular space would now face the intermembrane space. For example, the
![li564542480e](li564542480e.jpg)

EXPERIMENTAL FIGURE 12-8 Dual staining reveals the multiple mitochondrial DNA molecules in a growing Euglena gracilis cell. Cells were treated with a mixture of two dyes: ethidium bromide, which binds to DNA and emits a red fluorescence, and DiOC6, which is incorporated specifically into mitochondria and emits a green fluorescence. Thus the nucleus emits a red fluorescence, and areas rich in mitochondrial DNA fluoresce yellow-a combination of red DNA and green mitochondrial fluorescence. (Reproduced with permission of The Journal of Cell Science, from Hayashi, Y. and Ueda, K., "The shape of mitochondria and the number of mitochondrial nucleoids during the cell cycle of Euglena gracilis," Journal of Cell Science, 1989, 93, pp 565-570.)
$F_{1}$ subunit of ATP synthase, localized to the cytosolic face of the bacterial membrane, would face the matrix of the evolving mitochondrion (left) or chloroplast (right). Budding of vesicles from the inner chloroplast membrane, such as occurs during development of chloroplasts in contemporary plants, would generate the thylakoid membranes with the $F_{1}$ subunit remaining on the cytosolic face, facing the chloroplast stroma (step (1). The organelle DNAs are indicated. Membrane surfaces facing a shaded area are cytosolic faces; surfaces facing an unshaded area are exoplasmic faces.
number of mitochondria, but because there is no mechanism for apportioning exactly equal numbers of mitochondria to the daughter cells, some cells contain more mtDNA than others. Thus the total amount of mtDNA in a cell depends on the number of mitochondria, the size of the mtDNA molecules, and the number of mtDNA molecules per mitochondrion. Each of these parameters varies greatly between cell types. In a typical human cell, there are about 1000-2000 mtDNA molecules per nucleus; however, a human egg has about 500,000 mtDNA molecules, and a sperm has only about 100.

Studies of mutants in yeasts and other single-celled organisms first indicated that mitochondria exhibit cytoplasmic inheritance (Figure 12-9). For instance, yeast cells with the petite mutation exhibit structurally abnormal mitochondria and are incapable of oxidative phosphorylation. As a result, petite cells grow more slowly than wild-type cells and form smaller colonies. Genetic crosses between different (haploid) yeast strains showed that the petite mutation does not segregate with any known nuclear gene or chromosome. In later studies, most petite mutants were found to contain deletions of mtDNA.

In the mating by fusion of haploid yeast cells, both parents contribute equally to the cytoplasm of the resulting diploid; thus inheritance of mitochondria is biparental (see Figure 12-9a). In mammals and most other multicellular organisms, however, the sperm contributes little (if any) cytoplasm to the zygote, and virtually all the mitochondria in the embryo are derived from those in the egg, not the sperm. Studies in mice have shown that 99.99 percent of mtDNA is maternally inherited, but a small part ( 0.01 percent) is

![li574542480e](li574542480e.jpg)

All haploid cells respiratory-proficient

FIGURE 12-9 Cytoplasmic inheritance of an mtDNA petite mutation in yeast. Petite-strain mitochondria are defective in oxidative phosphorylation owing to a deletion in mtDNA. (a) Haploid yeast cells fuse to produce a diploid cell that undergoes meiosis, during which random segregation of parental chromosomes and mitochondria containing mtDNA occurs. Note that alleles for genes in nuclear DNA (represented by large and small nuclear chromosomes colored red and blue) segregate 2:2 during meiosis (see Figure 6-5).
inherited from the male parent. In higher plants, mtDNA is inherited exclusively in a uniparental fashion through the female parent (egg), not the male (pollen).

## The Size, Structure, and Coding Capacity of mtDNA Vary Considerably Among Organisms

Surprisingly, the size of the mtDNA, the number and nature of the proteins it encodes, and even the mitochondrial genetic code itself vary greatly between different organisms. The mtDNAs of most multicellular animals are approximately $16-\mathrm{kb}$ circular molecules that encode intron-less genes compactly arranged on both DNA strands. Vertebrate mtDNAs encode the two rRNAs found in mitochondrial ribosomes, the 22 tRNAs used to translate mitochondrial mRNAs, and 13 proteins involved in electron transport and ATP synthesis. The smallest mitochondrial genomes known are found in Plasmodium, a genus of single-celled obligate intracellular parasites that cause malaria in humans. Plasmodium
(b)
![li584542480e](li584542480e.jpg)

In contrast, since yeast normally contain some 50 mtDNA molecules per cell, most products of meiosis contain both normal and petite mtDNAs and are capable of respiration. (b) As these haploid cells grow and divide mitotically, the cytoplasm (including the mitochondria) is randomly distributed to the daughter cells. Occasionally, a cell is generated that contains only petite mtDNA and yields a petite colony. Thus formation of such petite cells is independent of any nuclear genetic marker.
mtDNAs are only about 6 kb , encoding three proteins and the mitochondrial rRNAs.

The mitochondrial genomes of a number of different metazoans have now been sequenced, revealing that mtDNAs from all these sources encode essential mitochondrial proteins that are synthesized on mitochondrial ribosomes (Figure 12-10). Most mitochondrially synthesized polypeptides identified thus far are subunits of multimeric complexes used in electron transport or ATP synthesis. However, most of the proteins localized in mitochondria, such as those involved in the processes listed at the top of Figure 12-10 and Table 12-1, are encoded by nuclear genes, synthesized on cytosolic ribosomes, and imported into the organelle by processes discussed in Chapter 13.

Plant mitochondrial genomes are many times larger than those of metazoans. For instance, Arabidopsis thaliana, a member of the mustard weed family, has 366 kb of mtDNA. The largest known mitochondrial genome, about 2 Mb , is found in cucurbit plants (e.g., melon and cucumber). Most plant

![li594542480e](li594542480e.jpg)

FIGURE 12-10 Proteins encoded in mitochondrial DNA and their involvement in mitochondrial processes. Only the mitochondrial matrix and inner membrane are depicted. Most mitochondrial components are encoded by the nucleus (blue); those highlighted in pink are encoded by mtDNA in some eukaryotes but by the nuclear genome in other eukaryotes, whereas a small portion are invariably specified by mtDNA (orange). Mitochondrial processes that have exclusively nucleus-encoded components are listed at the top. Complexes I-V are involved in electron transport and oxidative phosphorylation.
mtDNA does not encode proteins, but rather consists of long introns, pseudogenes, mobile DNA elements restricted to the mitochondrial compartment, and pieces of foreign (chloroplast, nuclear, and viral) DNA that were probably inserted into plant mitochondrial genomes during their evolution. Duplicated sequences also contribute to the greater length of plant mtDNAs.

Differences in the numbers of genes in the mtDNA from various organisms most likely reflect the movement of DNA between mitochondria and the nucleus during evolution. Direct evidence for this movement comes from the observation that several proteins encoded by mtDNA in some species are encoded by nuclear DNA in other, closely related species. A striking example of this phenomenon involves the coxII gene, which encodes subunit 2 of cytochrome $c$ oxidase, which constitutes complex IV in the mitochondrial electrontransport chain (described in detail below). This gene is found in mtDNA in all multicellular plants studied except for certain related species of legumes, including the mung bean and the soybean, in which the coxII gene is nuclear. The coxII gene is completely missing from mung bean mtDNA, but a defective coxII pseudogene that has accumulated many mutations can still be recognized in soybean mtDNA.

Many RNA transcripts of plant mitochondrial genes are edited, mainly by the enzyme-catalyzed conversion of selected C residues to U, and occasionally of U to C. (RNA editing is discussed in Chapter 10.) Indeed, the nuclear coxII gene of the mung bean corresponds more closely to the

Tim, Sec, Tat, and Oxa1 translocases are involved in protein import and export and in the insertion of proteins into the inner membrane (see Chapter 13). RNase P is a ribozyme that processes the $5^{\prime}$ end of tRNAs (discussed in Chapter 10). It should be noted that the majority of eukaryotes have a multisubunit complex I as depicted, with three subunits invariantly encoded by mtDNA. However, in a few organisms (Saccharomyces, Schizosaccharomyces, and Plasmodium), this complex is replaced by a nucleus-encoded, single-polypeptide enzyme. See G. Burger et al., 2003, Trends Genet. 19:709.
edited coxII mtDNA-encoded mRNA transcripts in other legumes with functional coxII mtDNA than to their unedited mtDNA-encoded coxII genes. These observations are strong evidence that the coxII gene moved from the mitochondrion to the nucleus during mung bean evolution by a process that involved an edited, mRNA intermediate. Presumably this movement involved a reverse-transcription mechanism and insertion into a nuclear chromosome. This process would be similar to that by which processed pseudogenes are generated in the nuclear genome from nucleus-encoded mRNAs.

In addition to the large differences in the sizes of mitochondrial genomes among eukaryotes, the structure of the mtDNA also varies greatly. As mentioned above, mtDNA in most animals is a circular molecule of $6-16 \mathrm{~kb}$. However, the mtDNA of many organisms, such as the protist Tetrahymena, exists as linear head-to-tail repeats. In the most extreme examples, the mtDNA of the protist Amoebidium parasiticum is composed of several hundred distinct short linear molecules. And the mtDNA of Trypanosoma is composed of multiple maxicircles concatenated (interlocked) to thousands of minicircles encoding guide RNAs involved in editing the sequence of the mitochondrial mRNAs encoded in the maxicircles.

## Products of Mitochondrial Genes Are Not Exported

As far as is known, all RNA transcripts of mtDNA and their translation products remain in the mitochondrion in which they

are produced, and all mtDNA-encoded proteins are synthesized on mitochondrial ribosomes. Mitochondrial DNA encodes the rRNAs that form mitochondrial ribosomes, although most of the ribosomal proteins are imported from the cytosol. In animals and fungi, all the tRNAs used for protein synthesis in mitochondria are also encoded by mtDNAs. However, in plants and many protozoans, most mitochondrial tRNAs are encoded by the nuclear DNA and imported into the mitochondrion.

3Reflecting the bacterial ancestry of mitochondria, mitochondrial ribosomes resemble bacterial ribosomes and differ from eukaryotic cytosolic ribosomes in their RNA and protein compositions, their size, and their sensitivity to certain antibiotics (see Table 5-3). For instance, chloramphenicol blocks protein synthesis by bacterial and mitochondrial ribosomes from most organisms, but cycloheximide, which inhibits protein synthesis on eukaryotic cytosolic ribosomes, does not affect mitochondrial ribosomes. This sensitivity of mitochondrial ribosomes to the important aminoglycoside class of antibiotics, which includes chloramphenicol, is the main cause of the toxicity in patients that these antibiotics can cause.

## Mitochondria Evolved from a Single Endosymbiotic Event Involving a Rickettsia-Like Bacterium

Analysis of mtDNA sequences from various eukaryotes, including single-celled protists that diverged from other eukaryotes early in evolution, provides strong support for the idea that the mitochondrion had a single origin. Mitochondria most likely arose from a bacterial symbiote whose closest contemporary relatives are in the Rickettsiaceae group. Bacteria in this group are obligate intracellular parasites. Thus the ancestor of the mitochondrion probably also had an intracellular lifestyle, which placed it in a good position to evolve into an intracellular symbiote. The mtDNA with the largest number of encoded genes so far found is from the protist species

Reclinomonas americana. All other mitochondrial genomes contain a subset of the R. americana genes, which strongly implies that they evolved from a common ancestor shared with R. americana, losing different groups of mitochondrial genes by deletion or transfer to the nucleus, or both, over time.

In organisms whose mtDNA includes only a limited number of genes, the same set of mitochondrial genes is retained (see Figure 12-10, orange proteins), regardless of the phyla that include these organisms. One hypothesis for why these genes were never successfully transferred to the nuclear genome is that their encoded polypeptides are too hydrophobic to cross the outer mitochondrial membrane, and therefore would not be imported back into the mitochondria if they were synthesized in the cytosol. Similarly, the large size of rRNAs may interfere with their transport from the nucleus through the cytosol into mitochondria. Alternatively, these genes may not have been transferred to the nucleus during evolution because regulation of their expression in response to conditions within individual mitochondria may be advantageous. If these genes were located in the nucleus, conditions within each mitochondrion could not influence the expression of proteins found in that mitochondrion.

## Mitochondrial Genetic Codes Differ from the Standard Nuclear Code

The genetic code used in animal and fungal mitochondria is different from the standard code used in all prokaryotic and eukaryotic nuclear genes; remarkably, the code even differs among mitochondria from different species (Table 12-2). Why and how these differences arose during evolution is a mystery. UGA, for example, is normally a stop codon, but is read as tryptophan by human and fungal mitochondrial translation systems; however, in plant mitochondria, UGA is still recognized as a stop codon. AGA and AGG, the standard nuclear codons for arginine, also code for arginine in fungal and plant mtDNA, but they are stop codons in mammalian mtDNA and serine codons in Drosophila mtDNA.

TABLE 12-2 Alterations in the Standard Genetic Code in Mitochondria

| Codon | Mitochondria |  |  |  |  |  |
| :--: | :--: | :--: | :--: | :--: | :--: | :--: |
|  | Standard Code* | Mammals | Drosophila | Neurospora | Yeasts | Plants |
| UGA | Stop | Trp | Trp | Trp | Trp | Stop |
| AGA, AGG | Arg | Stop | Ser | Arg | Arg | Arg |
| AUA | Ile | Met | Met | Ile | Met | Ile |
| AUU | Ile | Met | Met | Met | Met | Ile |
| CUU, CUC, CUA, CUG | Leu | Leu | Leu | Leu | Thr | Leu |

*For nuclear-encoded proteins.
source: Data from S. Anderson et al., 1981, Nature 290:457; P. Borst, in International Cell Biology 1980-1981, H. G. Schweiger, ed., Springer-Verlag, p. 239; C. Breitenberger and U. L. Raj Bhandary, 1985, Trends Biochem. Sci. 10:478; V. K. Eckenrode and C. S. Levings, 1986, In Vitro Cell. Dev. B. 22:169; and J. M. Gualber et al., 1989, Nature 341:660.

(2) As shown in Table 12-2, plant mitochondria appear to use the standard genetic code. However, comparisons of the amino acid sequences of plant mitochondrial proteins with the nucleotide sequences of plant mtDNAs suggested that CGG could code for either arginine (the "standard" amino acid) or tryptophan. This apparent nonspecificity of the plant mitochondrial code is explained by editing of mitochondrial RNA transcripts, which can convert cytosine residues to uracil residues. If a CGG sequence is edited to UGG, the codon specifies tryptophan, the standard amino acid for UGG, whereas unedited CGG codons encode the standard arginine. Thus the translation system in plant mitochondria does use the standard genetic code.

## Mutations in Mitochondrial DNA Cause Several Genetic Diseases in Humans

The severity of disease caused by a mutation in mtDNA depends on the nature of the mutation and on the proportion of mutant and wild-type mtDNAs present in a particular cell type. Generally, when mutations in mtDNA are found, cells contain mixtures of wild-type and mutant mtDNAs-a condition known as heteroplasmy. Each time a mammalian somatic or germ-line cell divides, the mutant and wild-type mtDNAs segregate randomly into the daughter cells, as occurs in yeast cells (see Figure 12-9b). Thus the mtDNA genotype, which fluctuates from one generation and from one cell division to the next, can drift toward predominantly wild-type or predominantly mutant mtDNAs. Since all enzymes required for the replication and growth of mammalian mitochondria, such as the mitochondrial DNA and RNA polymerases, are encoded in the nucleus and imported from the cytosol, a mutant mtDNA should not be at a "replication disadvantage"; mutants that have large deletions of mtDNA might even be at a selective advantage because they can replicate faster.

Recent research suggests that the accumulation of mutations in mtDNA is an important component of aging in mammals. Mutations in mtDNA have been observed to accumulate over time, probably because mammalian mtDNA is not repaired in response to DNA damage. To study this hypothesis, researchers used gene "knock-in" techniques in mice to replace the nuclear gene encoding mitochondrial DNA polymerase with normal proofreading activity (see Figure 5-33) with a mutant gene encoding a polymerase that is defective in proofreading. Mutations in mtDNA accumulated much more rapidly in homozygous mutant mice than in wild-type mice, and the mutant mice aged at a highly accelerated rate and died earlier than wild-type mice (Figure 12-11). It has been proposed that the loss of mitochondrial function that accompanies aging, due in part to accumulation of mutations and damage induced by reactive oxygen species, might contribute to aging and limit the life span. However, additional studies will be required to determine how mitochondrial dysfunction, aging, and longevity are related.

3With few exceptions, all human cells have mitochondria, yet mutations in mtDNA affect only some tissues. Those most commonly affected are tissues that have a high
![li604542480e](li604542480e.jpg)

EXPERIMENTAL FIGURE 12-11 Mice with a mitochondrial DNA polymerase defective for proofreading exhibit premature aging. A line of "knock-in" mice were prepared by methods discussed in Chapter 6 with an aspartic acid-to-alanine mutation in the gene encoding mitochondrial DNA polymerase (D257A), which inactivated the polymerase's proofreading function. (a) Wild-type and homozygous mutant mice at 390 days old ( 13 months). The mutant mouse displays many of the features of an aged mouse ( $>720$ days, or 24 months, of age). (b) Plot of survival versus time of wild-type ( $+/+$ ), heterozygous (D257A/+), and homozygous (D257A/D257A) mice. [Part (a) Jeff Miller/University of Wisconsin-Madison. Part (b) data from G. C. Kujoth et al., 2005, Science 309:481.]
requirement for the ATP produced by oxidative phosphorylation and tissues that require most or all of the mtDNA in the cell to synthesize sufficient amounts of functional mitochondrial proteins. For instance, Leber's hereditary optic neuropathy (degeneration of the optic nerve) is caused by a missense mutation in the mtDNA gene encoding subunit 4 of the NADH-CoQ reductase (complex I), a protein required for ATP production by mitochondria (see below). Several large deletions in mtDNA cause another set of diseases, including chronic progressive external ophthalmoplegia, characterized by eye defects, and Kearns-Sayre syndrome, characterized by eye defects, an abnormal heartbeat, and central nervous system degeneration. A third condition, causing "ragged-red" muscle fibers (with improperly assembled mitochondria) and associated uncontrolled jerky movements, is due to a single mutation in the T $\Psi$ CG loop of the mitochondrial lysine tRNA. As a result of this mutation, the translation of several mitochondrial proteins is apparently inhibited.

## Mitochondria Are Dynamic Organelles That Interact Directly with One Another

Analysis of fluorescently labeled mitochondria in live cells has shown that mitochondria in many different types of cells

are highly dynamic. They undergo frequent fusions (merging) and fissions (breaking apart) that generate tubular, sometimes branched networks (Figures 12-12a and b), which may account for some of the wide variety of mitochondrial morphologies seen in different types of cells. When individual mitochondria fuse, each of the two membranes fuses (inner with inner, and outer with outer) and each of their distinct compartments intermix (matrix with matrix, intermembrane space with intermembrane space). A set of four evolutionarily conserved GTP-hydrolyzing (GTPase) enzymes-MFN1 (mitofusin 1), MFN2 (mitofusin 2), OPA1, and DRP1—plays critical roles in mediating these membrane fusions and fissions (Figure 12-12c). These enzymes are members of the dynamin family of GTPases. Dynamin, the first-identified member of this family, mediates a comparable membrane fission reaction required during the pinching off of endocytic vesicles from the plasma membrane (see Chapter 14). Mutations in several of the genes encoding these GTPases can disrupt mitochondrial functions, such as maintenance of proper inner membrane electric potential, and structure, resulting in truncated or enlarged mitochondria, and cause human disease. The inherited, autosomal dominant neuromuscular disease Charcot-MarieTooth subtype 2 A is caused by loss-of-function mutations in MFN2 that lead to defects in peripheral nerve function and progressive muscle weakness, mainly in the feet and hands. Mutations in OPA1 are associated with autosomal dominant optic atrophy that influences nerves in the eye's retina.

What is the value of mitochondrial fission and fusion? It has been suggested that fusion helps to maintain a relatively homogeneous population of mitochondria within a cell. Should some subset of individual mitochondria suffer deleterious modification or loss of important components, fusion with other mitochondria would permit restoration by sharing of those components. Studies of cells and organisms with mutations in genes encoding the fusion machinery suggest that fusion also plays a role in the proper localization of mitochondria within cells, maintaining the proper morphology and cristal organization, distribution of mitochondrial DNA, and maintenance of fully functional electron transport.

A number of functions are served by mitochondrial fission. For example, mitochondrial fission is particularly active when cells divide (particularly during the $\mathrm{G}_{2}$ and M phases of the cell cycle; see Chapter 19). As a consequence, the multiple discrete mitochondria generated by fission are readily distributed evenly into the daughter cells. Mitochondria can be transported by motor proteins along cytoskeletal filaments, including microtubules and microfilaments (see Chapter 17), to establish their proper intracellular distribution. In addition, fission provides a powerful mechanism of quality control by culling defective segments of mitochondria from the interconnected, healthy mitochondrial network. Should a portion of a large mitochondrial network become damaged or dysfunctional-for example, by the generation of high levels of reactive oxygen species (discussed later) or by mutations in the mitochondrial DNA-fission can separate the compromised segments from healthy segments.

Cells have the capacity to recognize damaged or dysfunctional segments of the mitochondrial network and, after
they detach from the network, can surround them with a membrane and then deliver them to lysosomes for degradation. This destruction of mitochondria, which is called mitophagy ("eating mitochondria"), is a subset of the general process called autophagy ("self eating") by which cells engulf in membranes and degrade organelles and portions of the cytosol (see Chapter 14). Strikingly, hereditary early-onset Parkinson's disease is caused by mutations in two genes encoding proteins that can mediate mitophagy. The proteins are PINK1 (a kinase) and Parkin (an E3 ubiquitin ligase that covalently links the small protein ubiquitin to nearby proteins to target them for destruction by proteasomes; see Chapter 3). In healthy mitochondria, PINK1 is imported into the mitochondrial matrix. When the mitochondrion is damaged or dysfunctional, PINK1 cannot enter the matrix and remains at the outer mitochondrial membrane, where it recruits cytosolic Parkin to the damaged segment. The Parkin is activated and polyubiquitinylates outer-membrane proteins, thus targeting them for degradation and inducing mitophagy. When the environment of a cell has inadequate oxygen (hypoxia), certain enzymes modify proteins on the outer mitochondrial membrane, inducing mitophagy. The influences of PINK1 and Parkin on mitochondrial homeostasis, which may include mitophagy-independent as well as mitophagy-dependent pathways, were first identified in genetic studies of the fly Drosophila melanogaster.

Mitochondrial dynamism has additional features that are worth mentioning. Mitochondrial structure and function can change in response to the metabolic state of the cells. For example, the isolation of rat liver cells (hepatocytes) from the body and their transfer into cell culture stresses the cells and results in their depolarization (loss of some of their epithelial cell properties; see Chapter 20). These cells exhibit fragmentation of their mitochondria (Figure 12-12d, left) accompanied by low generation of ATP via oxidative phosphorylation. As the cells adjust to growth in cell culture, they become polarized (have a morphology and biochemistry more like that of hepatocytes in a liver), increase mitochondrial generation of ATP via oxidative phosphorylation, and exhibit an extensive network of fused mitochondria (Figure 12-12d, right). Remarkably, recent studies have suggested that mtDNA, and indeed intact mitochondria, can be transferred from one cell to another via membrane tubules called tunneling nanotubes, which are described in Chapter 20.

## Mitochondria Are Influenced by Direct Contacts with the Endoplasmic Reticulum

Mitochondrial dynamics, and indeed, many mitochondrial functions, are influenced by direct contacts between mitochondria and the endoplasmic reticulum (ER). The portions of the ER that form special contact regions with the mitochondria, called mitochondria-associated membranes, or MAMs, can be visualized using electron microscopy and fluorescence microscopy (Figure 12-13). Their lipid and protein composition differs somewhat from that of the rest of the ER. In yeast, a protein complex called ERMES (ERmitochondria encounter structure) has been proposed to mediate the reversible tethering of MAMs to mitochondria.

![li614542480e](li614542480e.jpg)

EXPERIMENTAL FIGURE 12-12 Mitochondria undergo rapid fusion and fission. (a) A human HeLa cell labeled with a mitochon-drion-specific fluorescent dye (MitoTracker Green) was imaged using three-dimensional structured illumination fluorescence microscopy (a 6.1- $\mu \mathrm{m}$-thick section through the cell is shown). The network of fused and branched mitochondria is seen in the cytoplasm, with only a few mitochondria observed above or below the nucleus (unstained central dark oval). The identity of the striations seen within the mitochondria is not known. The mitochondria are shown in artificial colors to indicate their positions relative to the surface to which the cell is attached (blue is closest to and red farthest from the surface). (b) Mitochondria labeled with a fluorescent protein in a live normal mouse embryonic fibroblast were observed using time-lapse fluorescence microscopy. Several mitochondria undergoing fusion
(top) or fission (bottom) are artificially highlighted in blue and with arrows. (c) Mitochondrial fusion (top) and fission (bottom) are mediated by a set of GTPase enzymes (MFN1, MFN2, OPA1, and DRP1). The integral membrane proteins MFN1 and MFN2 (MFN1/2) mediate outer mitochondrial membrane (OMM) fusion, which is followed by fusion of the inner mitochondrial membranes (IMM) mediated by the integral membrane protein OPA1. The matrix and inner membrane space (IMS) remain distinct. The soluble cytosolic GTPase DRP1 is recruited to a constricted site on the surface of a mitochondrion, where DRP1 polymers sever the membrane, resulting in fission. A variety of post-translational modifications of DRP1 regulate fission. (d) (Left) Rat liver cells (hepatocytes) one day after being removed from the liver and placed in cell culture, are stressed and depolarized (lack some of the morphological and biochemical properties of epithelial cells; see Chapter 20), have low levels of oxidative phosphorylation and ATP production, and have fragmented mitochondria (visualized by staining with MitoTracker Green). (Right) After growth in culture for six days, the hepatocytes become polarized, their mitochondria fuse, forming an extensive network, and the cells exhibit high levels of oxidative phosphorylation and ATP production. Insets show higher-magnification views of the mitochondria. (la) Reprinted by permission from Macmillan Publishers Ltd: Shao et al., "Super-resolution 3D microscopy of live whole cells using structured illumination," Nature Methods, B12, 1044-1046, Fig. 54, 2011, courtesy of Mats Gustafsson. (b) Republished with permission from Elsevier. Modified from Chan D. C., "Mitochondria: Dynamic Organelles in Disease, Aging, and Development," Cell, 2006, 125(7):1241-52. Permission conveyed through Copyright Clearance Center, Inc. (c) Information from P. Mishra and D. C. Chan, 2014, Nat. Rev. Mol. Cell Biol. 15:634-646. (d) From Proc. Natl. Acad. Sci. USA 2013. 110(18):7288-7293, Fig. 3 Day 1 and Day 6. "Coordinated elevation of mitochondrial oxidative phosphorylation and autophagy help drive hepatocyte polarization," by Fu, D. et al. Courtesy Jennifer Lippincott-Schwartz.]

![li624542480e](li624542480e.jpg)

EXPERIMENTAL FIGURE 12-13 Specialized regions of the endoplasmic reticulum called mitochondria-associated membranes (MAMs) directly contact mitochondria and influence mitochondrial shape, function, and sites of fission. (a) Transmission electron microscopic (EM) image of a section through rat brown adipose (fat) tissue. The lumen of the endoplasmic reticulum (ER) is false colored to show a MAM (yellow) and the non-MAM, bulk ER (blue). The MAM is closely apposed to the outer mitochondrial membrane. (b) Three-dimensional model of a segment of a mitochondrion (red, only outer membrane shown) and the adjacent MAM (yellow) determined from a line of cultured avian lymphoma cells using EM tomography (assembly of a three-dimensional image from consecutive individual sections). (c) A three-dimensional model of a mitochondrion (red) and adjacent MAMs (green) from a yeast cell using EM tomography. The two MAM domains are derived from ER tubules that in some cases can wrap around the mitochondrion, in the top case forming a clamp-like structure that appears to constrict the mitochondrion in preparation for fission. (d) Live cell fluorescence
microscopic images of a Cos-7 monkey cell, showing a mitochondrion (white in the top panels, same mitochondrion in red in the bottom panels) and MAM (green in bottom panels), taken from a single field of view at 10 -second intervals. The arrow points to the site of constriction and fission on the mitochondrion and to the MAM at the constriction/fission site. The MAM directs constriction and subsequent DRP1-mediated fission at this site. To visualize the mitochondria and ER, the Cos-7 cells were transfected with cDNA vectors encoding two fluorescent proteins that specifically accumulate in either the mitochondrion (red fluorescence) or the ER (green fluorescence). [Part (a) de Meis L., Ketzer, L. A., da Costa R. M., de Andrade I. R., Berchimol M. (2010) Fusion of the Endoplasmic Reticulum and Mitochondrial Outer Membrane in Rats Brown Adipose Tissue: Activation of Thermogenesis by $\mathrm{Ca}^{2+}$, PLoS ONE 5(3): e9439.doi:10.1371/journal.pone.0009439. Part (b) ©(2006. Csordas et al. The Journal of Cell Biology. 174:915-921. doi:10.1083/jcb.200604016. Parts (c) and (d) republished with permission from AAAS, from Friedman, J. R., et al., "ER tubules mark sites of mitochondrial division," Science, 2011, 334(6054):358-62; permission conveyed through the Copyright Clearance Center, Inc.]

The ERMES complex is not present in mammals; the proteins that mediate the tethering of MAMs to mitochondria in complex multicellular organisms are as yet unknown. Tethering proteins hold the MAM and the outer mitochondrial membrane about $10-30 \mathrm{~nm}$ apart.

MAMs contribute significantly to many cellular processes (see Table 12-1), including mitochondrial fission. MAM-mitochondrial contacts can initiate mitochondrial constriction and help recruit DPR1, which completes membrane fission (see Figure 12-12c). In yeast, MAM tubules have been seen to loop completely around mitochondria, forming a clamp that constricts the mitochondrion (Figure 12-13c). In mammalian cells, the MAMs contact the mitochondria at fission sites, but they have not been shown to loop fully around the mitochondria (Figure 12-13d).

MAMs also play an integral role in intracellular calcium and energy metabolism. Variations in the concentrations of calcium ions in intracellular compartments-cytosolic calcium $\left(\left[\mathrm{Ca}^{2+}\right]_{\mathrm{c}}\right)$, mitochondrial calcium $\left(\left[\mathrm{Ca}^{2+}\right]_{\mathrm{m}}\right)$, and calcium in the ER $\left(\left[\mathrm{Ca}^{2+}\right]_{\mathrm{er}}\right)$-are employed to control a wide variety of activities within cells, a process called calcium signaling (see Chapter 15). Calcium is also important for extracellular processes, such as the activity of some blood-clotting proteins. Intramitochondrial calcium ions play an important role in controlling mitochondrial function, and MAMs mediate this control by delivering calcium from the ER to mitochondria. For example, an increase in $\left[\mathrm{Ca}^{2+}\right]_{\mathrm{m}}$ in the matrix can increase mitochondrial production of ATP. Increased $\left[\mathrm{Ca}^{2+}\right]_{\mathrm{m}}$ directly increases the activities of three mitochondrial enzymes that produce NADH from $\mathrm{NAD}^{+}$: pyruvate dehydrogenase (see Figure 12-5) and $\alpha$-ketoglutarate and isocitrate dehydrogenases (see Figure 12-16 below). As we shall see later in this chapter, NADH provides high-energy electrons for ATP synthesis. Thus continuous low-level release of $\mathrm{Ca}^{2+}$ from MAMs into mitochondria is necessary for ATP synthesis when cells are in a basal, or resting, state. Increased delivery of $\mathrm{Ca}^{2+}$ via MAMs can occur when cells require more ATP-for example, when muscle cells are stimulated to contract. Strikingly, calcium signaling is used both to induce muscle contraction (see Chapter 17) and coordinately to increase mitochondrial ATP synthesis to provide the energy to fuel that contraction. When $\left[\mathrm{Ca}^{2+}\right]_{\mathrm{m}}$ is elevated, mitophagy can be induced. Indeed, mitochondrial calcium overload can activate regulated cell death pathways. Thus the control of $\left[\mathrm{Ca}^{2+}\right]_{\mathrm{m}}$ can literally control the life and death of cells.

## KEY CONCEPTS OF SECTION 12.2

## The Structure and Functions of Mitochondria

- In eukaryotic cells, mitochondria use aerobic oxidation to generate ATP. These multifunctional organelles are also responsible for many other key activities (see Table 12-1), including biosynthesis and metabolism of a wide variety of small molecules and regulated cell death.
- The mitochondrion has two distinct membranes (outer and inner) and two distinct subcompartments (the intermembrane space between the two membranes, and the matrix surrounded by the inner membrane) (see Figure 12-6). Aerobic oxidation occurs in the mitochondrial matrix and on the inner mitochondrial membrane.
- The inner mitochondrial membrane is a single continuous membrane with three compositionally, structurally, and functionally distinct domains: boundary membrane, cristae, and crista junctions.
- There are at least 1100 proteins associated with mammalian mitochondria, most of which are encoded by nuclear genes. The mechanisms by which proteins enter the mitochondria are described in Chapter 13.
- Mitochondria and chloroplasts most likely evolved from bacteria that formed a symbiotic relationship with ancestral cells containing a eukaryotic nucleus (see Figure 12-7).
- Most of the genes originally within mitochondria and chloroplasts were either lost because their functions were redundant with nuclear genes or moved to the nuclear genome over evolutionary time, leaving different gene sets in the organelle DNAs of different organisms (see Figure 12-10).
- Because most mtDNA is inherited from egg cells rather than sperm, mutations in mtDNA exhibit a maternal cytoplasmic pattern of inheritance. Similarly, chloroplast DNA is exclusively inherited from the maternal parent.
- Animal mtDNAs are circular molecules, reflecting their probable bacterial origin. Plant mtDNAs and chloroplast DNAs are generally longer than mtDNAs from other eukaryotes, largely because they contain more noncoding regions and repetitive sequences.
- Mitochondrial DNA (mtDNA) in the mitochondrial matrix and chloroplast DNAs encode rRNAs and some of the proteins involved in mitochondrial or photosynthetic electron transport and ATP synthesis. Mammalian mtDNA encodes only 13 proteins. Most animal mtDNAs and chloroplast DNAs also encode the rRNAs necessary to translate the organelle mRNAs.
- Mitochondrial ribosomes resemble bacterial ribosomes in their structure and in their sensitivity to drugs such as chloramphenicol (sensitive) and cycloheximide (resistant).
- The genetic code of animal and fungal mtDNA differs slightly from that of bacterial and nuclear genomes and varies among different animals and fungi (see Table 12-2). In contrast, plant mtDNAs appear to conform to the standard genetic code.
- Several human neuromuscular disorders result from mutations in mtDNA. Patients generally have a mixture of wildtype and mutant mtDNA in their cells (heteroplasmy): the higher the fraction of mutant mtDNA, the more severe the mutant phenotype.
- Mitochondria are dynamic organelles, undergoing fusion and fission reactions that are regulated by the state of the cell.

In many cells, the fused mitochondria form a large, interconnected branched tubular network. A family of GTPases mediate mitochondrial membrane fusion and fission (see Figure 12-12). Mutations in the genes encoding some of these GTPases cause human diseases.

- Mitochondrial fission and fusion are thought to play roles in maintaining a relatively homogeneous population of mitochondria, distributing mitochondria among the daughter cells during cell division, and establishing a system of quality control to permit culling of defective mitochondria from healthy mitochondria. Defective mitochondria or segments of mitochondria are destroyed by a processed called mitophagy.
- Two proteins that can mediate mitophagy, PINK1 and Parkin, are encoded by genes that, when mutated, are responsible for hereditary early-onset Parkinson's disease.
- Mitochondria-associated membranes (MAMs) (see Figure 12-13), are specialized regions of the endoplasmic reticulum that closely contact mitochondria via protein tethers.
- The MAM/mitochondrial interface significantly influences many cellular functions, including mitochondrial shape and dynamics (see Table 12-1).
- The MAM/mitochondrial interface plays a key role in moving calcium from the ER into the mitochondria. Calcium influx into mitochondria from MAMs can stimulate ATP synthesis and, in the context of mitochondrial calcium overload, initiates a program of regulated cell death.


### 12.3 The Citric Acid Cycle and Fatty Acid Oxidation

We now continue our detailed discussion of glucose oxidation and ATP generation, exploring what happens to the pyruvate generated during glycolysis (stage I, see Figures 12-1 and 12-3) after it is transported into the mitochondrial matrix. The last three of the four stages of glucose oxidation (Figure 12-14) are

- Stage II. Stage II can be subdivided into two distinct parts: (1) the conversion of pyruvate to acetyl CoA, followed by (2) oxidation of acetyl CoA to $\mathrm{CO}_{2}$ in the citric acid cycle. These oxidations are coupled to reduction of $\mathrm{NAD}^{+}$to NADH and of FAD to $\mathrm{FADH}_{2}$. These two carriers can be considered the sources of high-energy electrons. (Fatty acid oxidation follows a similar route, with conversion of fatty acyl CoA to acetyl CoA.) Most of the reactions occur in or on the inner membrane facing the matrix.
- Stage III. Electron transfer from NADH and $\mathrm{FADH}_{2}$ to $\mathrm{O}_{2}$ via an electron-transport chain within the inner membrane converts the energy carried in those electrons into an electrochemical gradient across that membrane, called the proton-motive force.
- Stage IV. The energy of the proton-motive force is harnessed for ATP synthesis in the inner mitochondrial membrane. Stages III and IV are together called oxidative phosphorylation.


## In the First Part of Stage II, Pyruvate Is Converted to Acetyl CoA and High-Energy Electrons

Within the mitochondrial matrix, pyruvate reacts with coenzyme A, forming $\mathrm{CO}_{2}$, acetyl CoA, and NADH (Figure 12-14, stage II, left). This reaction, catalyzed by pyruvate dehydrogenase, is highly exergonic $\left(\Delta G^{\circ \prime}=-8.0 \mathrm{kcal} / \mathrm{mol}\right)$ and essentially irreversible. Influx of calcium from the MAM into the mitochondrion increases the activity of pyruvate dehydrogenase, driving the formation of acetyl CoA.

Acetyl CoA is a molecule consisting of a two-carbon acetyl group covalently linked to a longer molecule known as coenzyme A (CoA) (Figure 12-15). It plays a central role in the oxidation of pyruvate, fatty acids, and amino acids. In addition, it is an intermediate in numerous biosynthetic reactions, including the transfer of an acetyl group to histone and many other mammalian proteins and the synthesis of lipids such as cholesterol. In respiring mitochondria, however, the two-carbon acetyl group of acetyl CoA is almost always oxidized to $\mathrm{CO}_{2}$ via the citric acid cycle. Note that the two carbons in the acetyl group come from pyruvate; the third carbon of pyruvate is released as carbon dioxide.

## In the Second Part of Stage II, the Citric Acid Cycle Oxidizes the Acetyl Group in Acetyl CoA to $\mathrm{CO}_{2}$ and Generates High-Energy Electrons

Nine sequential reactions operate in a cycle to oxidize the acetyl group of acetyl CoA to $\mathrm{CO}_{2}$ (Figure 12-14, stage II, right). This cycle is referred to by several names: the citric acid cycle, the tricarboxylic acid (TCA) cycle, and the Krebs cycle. The net result is that for each acetyl group entering the cycle as acetyl CoA, two molecules of $\mathrm{CO}_{2}$, three of NADH, and one each of $\mathrm{FADH}_{2}$ and GTP are produced. NADH and $\mathrm{FADH}_{2}$ are high-energy electron carriers that will play a major role in stage III of mitochondrial oxidation: electron transport.

As shown in Figure 12-16, the cycle begins with condensation of the two-carbon acetyl group from acetyl CoA and the four-carbon molecule oxaloacetate to yield the six-carbon citric acid (hence the name citric acid cycle). Reactions step $\mathbf{1}$ and step $\mathbf{1}$ each release a $\mathrm{CO}_{2}$ molecule and reduce $\mathrm{NAD}^{+}$ to NADH. The source of the oxygen for generating the $\mathrm{CO}_{2}$ molecules in these reactions is water $\left(\mathrm{H}_{2} \mathrm{O}\right)$, not molecular oxygen $\left(\mathrm{O}_{2}\right)$, and the enzymatic activities of the enzymes catalyzing reactions step $\mathbf{1}$ and step $\mathbf{1}$ are increased by the influx of calcium into the mitochondrion from the MAM. Reduction of $\mathrm{NAD}^{+}$to NADH also occurs during reaction step $\mathbf{1}$, thus three NADHs are generated per turn of the cycle. In reaction step $\mathbf{1}$, two electrons and two protons are transferred to FAD, yielding the reduced form of this coenzyme, $\mathrm{FADH}_{2}$. Reaction step $\mathbf{1}$ is distinctive not only because it is an intrinsic part of the citric acid cycle (stage II), but also because it is catalyzed by a membrane-attached enzyme that, as we shall see, also plays an important role in stage III. In reaction step $\mathbf{1}$, hydrolysis of the high-energy thioester bond in succinyl CoA is coupled to synthesis of one GTP by substrate-level phosphorylation. Because GTP and ATP are interconvertible,

$$
\mathrm{GTP}+\mathrm{ADP} \rightleftharpoons \mathrm{GDP}+\mathrm{ATP}
$$

![li634542480e](li634542480e.jpg)

FIGURE 12-14 Summary of aerobic oxidation of glucose and fatty acids. Stage I: In the cytosol, glucose is converted to pyruvate (glycolysis) and fatty acid to fatty acyl CoA. Pyruvate and fatty acyl CoA then move into the mitochondrion. Mitochondrial porins make the outer membrane permeable to these metabolites, but specific transport proteins (colored ovals) in the inner membrane are required to import pyruvate (yellow) and fatty acids (blue) into the matrix. Fatty acyl groups are transferred from fatty acyl CoA to an intermediate carrier, transported across the inner membrane, and then reattached to CoA on the matrix side. Stage II: In the mitochondrial matrix, pyruvate and fatty acyl CoA are converted to acetyl CoA and then oxidized, releasing $\mathrm{CO}_{2}$. Pyruvate is converted to acetyl CoA with the formation of NADH and $\mathrm{CO}_{2}$; two carbons from fatty acyl CoA are converted to acetyl CoA with the formation of $\mathrm{FADH}_{2}$ and NADH. Oxidation of acetyl CoA in the citric acid cycle generates NADH and $\mathrm{FADH}_{2}$, GTP, and $\mathrm{CO}_{2}$.
this can be considered an ATP-generating step. Reaction step regenerates oxaloacetate, so the cycle can begin again. Note that molecular $\mathrm{O}_{2}$ does not participate in the citric acid cycle.

Most enzymes and small molecules involved in the citric acid cycle are soluble in the aqueous mitochondrial matrix. These include CoA, acetyl CoA, succinyl CoA, $\mathrm{NAD}^{+}$, and NADH, as well as most of the citric acid cycle enzymes. Succinate dehydrogenase (reaction step $\mathbf{D}$ ), however, is a

Stage III: Electron transport reduces $\mathrm{O}_{2}$ to $\mathrm{H}_{2} \mathrm{O}$ and generates a protonmotive force. Electrons (blue) from reduced coenzymes are transferred via electron-transport complexes (blue boxes) to $\mathrm{O}_{2}$ concomitant with transport of $\mathrm{H}^{+}$ions (red) from the matrix to the intermembrane space, generating the proton-motive force. Electrons from NADH flow directly from complex I to complex III, bypassing complex II. Electrons from $\mathrm{FADH}_{2}$ flow directly from complex II to complex III, bypassing complex I. Stage IV: ATP synthase, also called the $\mathrm{F}_{0} \mathrm{~F}_{1}$ complex (orange), harnesses the proton-motive force to synthesize ATP in the matrix. Antiporter proteins (purple and green ovals) transport ADP and $P_{i}$ into the matrix and export hydroxyl groups and ATP. NADH generated in the cytosol is not transported directly to the matrix because the inner membrane is impermeable to $\mathrm{NAD}^{+}$and NADH; instead, a shuttle system (red) transports electrons from cytosolic NADH to $\mathrm{NAD}^{+}$in the matrix. $\mathrm{O}_{2}$ diffuses into the matrix, and $\mathrm{CO}_{2}$ diffuses out.
component of an integral membrane protein in the inner membrane, with its active site facing the matrix. When mitochondria are disrupted by gentle ultrasonic vibration or by osmotic lysis, the non-membrane-bound enzymes of the citric acid cycle are released as very large multiprotein complexes. It is believed that within such complexes, the reaction product of one enzyme passes directly to the next enzyme without diffusing through the solution (see Figure 3-30).
![li644542480e](li644542480e.jpg)

FIGURE 12-15 The structure of acetyl CoA. This compound, consisting of an acetyl group covalently linked to a coenzyme A (CoA) molecule, is an important intermediate in the aerobic oxidation of
pyruvate, fatty acids, and many amino acids. It also contributes acetyl groups to many biosynthetic pathways.

![li654542480e](li654542480e.jpg)

FIGURE 12-16 The citric acid cycle. Acetyl CoA is metabolized to $\mathrm{CO}_{2}$ and the high-energy electron carriers NADH and $\mathrm{FADH}_{2}$. In reaction $\boldsymbol{\square}$, a two-carbon acetyl residue from acetyl CoA condenses with the four-carbon molecule oxaloacetate to form the six-carbon citrate. In the remaining reactions ( $\boldsymbol{\square}$- $\boldsymbol{\square}$ ), each molecule of citrate is eventually converted back to oxaloacetate, losing two $\mathrm{CO}_{2}$ molecules in the process. In each turn of the cycle, four pairs of electrons are removed from carbon atoms, forming three molecules of NADH, one molecule

Because glycolysis of one glucose molecule generates two pyruvate molecules, and thus two acetyl CoA molecules, the reactions in the glycolytic pathway and citric acid cycle produce six $\mathrm{CO}_{2}$ molecules, ten NADH molecules, and two $\mathrm{FADH}_{2}$ molecules per glucose molecule (Table 12-3). Although these reactions also generate four high-energy phosphoanhydride bonds in the form of two ATP and two GTP molecules, this represents only a small fraction of the available energy released in the complete aerobic oxidation of glucose. The remaining energy is stored as high-energy electrons in the reduced coenzymes NADH and $\mathrm{FADH}_{2}$, which can be
of $\mathrm{FADH}_{2}$, and one molecule of GTP. The two carbon atoms that enter the cycle with acetyl CoA are highlighted in blue through succinyl CoA. In succinate and fumarate, which are symmetric molecules, they can no longer be specifically denoted. Isotope-labeling studies have shown that these carbon atoms are not lost in the turn of the cycle in which they enter; on average, one will be lost as $\mathrm{CO}_{2}$ during the next turn of the cycle and the other in subsequent turns.
thought of as high-energy electron carriers. The goal of stages III and IV is to recover this energy in the form of ATP.

## Transporters in the Inner Mitochondrial Membrane Help Maintain Appropriate Cytosolic and Matrix Concentrations of $\mathrm{NAD}^{+}$and NADH

In the cytosol, $\mathrm{NAD}^{+}$is required for step $\boldsymbol{\square}$ of glycolysis (see Figure 12-3), and in the mitochondrial matrix, $\mathrm{NAD}^{+}$is required for the conversion of pyruvate to acetyl CoA and for three steps in the citric acid cycle (step $\boldsymbol{\square}$, step $\boldsymbol{\square}$, and step $\boldsymbol{\square}$ in

TABLE 12-3 Net Result of the Glycolytic Pathway and the Citric Acid Cycle

| Reaction | $\mathrm{CO}_{2}$ Molecules <br> Produced | $\mathrm{NAD}^{+}$Molecules <br> Reduced to NADH | FAD Molecules <br> Reduced to $\mathrm{FADH}_{2}$ | ATP (or GTP) |
| :-- | :--: | :--: | :--: | :--: |
| 1 glucose molecule to 2 pyruvate molecules | 0 | 2 | 0 | 2 |
| 2 pyruvates to 2 acetyl CoA molecules | 2 | 2 | 0 | 0 |
| 2 acetyl CoA to $4 \mathrm{CO}_{2}$ molecules | 4 | 6 | 2 | 2 |
| Total | 6 | 10 | 2 | 4 |

Figure 12-16). In each case, NADH is a product of the reaction. If glycolysis and oxidation of pyruvate are to continue, $\mathrm{NAD}^{+}$ must be regenerated by oxidation of NADH to ensure that this substrate is available. (Similarly, the $\mathrm{FADH}_{2}$ generated in stage II reactions must be reoxidized to FAD if FAD-dependent reactions are to continue.) As we will see in the next section, the electron-transport chain within the inner mitochondrial membrane converts NADH to $\mathrm{NAD}^{+}$and $\mathrm{FADH}_{2}$ to FAD as it reduces $\mathrm{O}_{2}$ to water and converts the energy stored in the highenergy electrons in the reduced forms of these molecules into a proton-motive force (stage III). Even though $\mathrm{O}_{2}$ is not involved in any reaction of the citric acid cycle, in the absence of $\mathrm{O}_{2}$ this cycle soon stops operating because in such anaerobic conditions, the mitochondria cannot regenerate the required $\mathrm{NAD}^{+}$ and FAD substrates. $\mathrm{NAD}^{+}$and FAD dwindle due to the inability of the electron-transport chain within the mitochondrion to oxidize NADH and $\mathrm{FADH}_{2}$. These observations raise the question of how a supply of $\mathrm{NAD}^{+}$in the cytosol is regenerated.

If the NADH from the cytosol could move into the mitochondrial matrix and be oxidized by the electron-transport chain, and if the $\mathrm{NAD}^{+}$product could be transported back into the cytosol, regeneration of cytosolic $\mathrm{NAD}^{+}$would be simple when $\mathrm{O}_{2}$ is available. However, the inner mitochondrial membrane is impermeable to NADH. To bypass this problem and permit the electrons from cytosolic NADH to be transferred indirectly to $\mathrm{O}_{2}$ via the mitochondrial electron-transport chain, cells use several electron shuttles to
transfer electrons from NADH in the cytoplasm to $\mathrm{NAD}^{+}$in the matrix. The operation of the most widespread shuttlethe malate-aspartate shuttle-is depicted in Figure 12-17.

For every complete cycle of the shuttle, there is no overall change in the numbers of NADH and $\mathrm{NAD}^{+}$molecules or the intermediates aspartate or malate. In the cytosol, however, NADH is oxidized to $\mathrm{NAD}^{+}$, which can be used for glycolysis, and in the matrix, $\mathrm{NAD}^{+}$is reduced to NADH, which can be used for electron transport:

$$
\mathrm{NADH}_{\text {cytosol }}+\mathrm{NAD}_{{ }^{+}{ }_{\text {matrix }}} \rightarrow \mathrm{NAD}_{{ }^{+}{ }_{\text {cytosol }}}^{\text {NADH }}+\mathrm{NADH}_{\text {matrix }}
$$

## Mitochondrial Oxidation of Fatty Acids Generates ATP

Up to now, we have focused mainly on the oxidation of carbohydrates, namely glucose, for ATP generation. Fatty acids are another important source of cellular energy. Cells can take up either glucose or fatty acids from the extracellular space with the help of specific transporter proteins (see Chapter 11). Should a cell not need to burn these molecules immediately, it can store them as a polymer of glucose called glycogen (especially in muscle or liver) or as a trimer of fatty acids covalently linked to glycerol, called a triacylglycerol or triglyceride (see below). In some cells, excess glucose is converted into fatty acids and then triacylglycerols for storage. However, unlike microorganisms, animals are unable to convert fatty acids to
![li664542480e](li664542480e.jpg)

FIGURE 12-17 The malate-aspartate shuttle. This cyclical series of reactions transfers electrons from NADH in the cytosol (via the intermembrane space) across the inner mitochondrial membrane, which is impermeable to NADH itself, to $\mathrm{NAD}^{+}$in the matrix. The net result is the replacement of cytosolic NADH with $\mathrm{NAD}^{+}$and matrix $\mathrm{NAD}^{+}$with NADH. Step - Cytosolic malate dehydrogenase transfers electrons from cytosolic NADH to oxaloacetate, forming malate.
Step $\mathbf{B}$ An antiporter (blue oval) in the inner mitochondrial membrane transports malate into the matrix in exchange for $\alpha$-ketoglutarate. Step $\mathbf{B}$ Mitochondrial malate dehydrogenase converts malate back to oxaloacetate, reducing $\mathrm{NAD}^{+}$in the matrix to NADH in the process.

Step $\mathbf{B}$ Oxaloacetate, which cannot directly cross the inner membrane, is converted to aspartate by addition of an amino group from glutamate. In this transaminase-catalyzed reaction in the matrix, glutamate is converted to $\alpha$-ketoglutarate. Step $\mathbf{B}$ A second antiporter (red oval) exports aspartate to the cytosol in exchange for glutamate.
Step $\mathbf{B}$ A cytosolic transaminase converts aspartate to oxaloacetate and $\alpha$-ketoglutarate to glutamate, completing the cycle. The blue arrows reflect the movement of the $\alpha$-ketoglutarate, the red arrows the movement of glutamate, and the black arrows that of aspartate/malate. It is noteworthy that as aspartate and malate cycle clockwise, glutamate and $\alpha$-ketoglutarate cycle in the opposite direction.

glucose. When the cells need to burn these energy stores to make ATP (e.g., when a resting muscle begins to do work and needs to burn glucose or fatty acids as fuel), enzymes break down glycogen to glucose or hydrolyze triacylglycerols to fatty acids, which are then oxidized to generate ATP:
![li674542480e](li674542480e.jpg)

Fatty acids are the major energy source for some tissues, particularly adult heart muscle. In humans, in fact, more ATP is generated by the oxidation of fats than by the oxidation of glucose. The oxidation of 1 g of triacylglycerol to $\mathrm{CO}_{2}$ generates about six times as much ATP as does the oxidation of 1 g of hydrated glycogen. Thus, considering the mass of stored fuel an organism must carry, triglycerides are more efficient than carbohydrates for storage of energy, in part because they are stored in anhydrous form and can yield more energy when oxidized, and in part because they are intrinsically more reduced (have more hydrogens) than carbohydrates. In mammals, the primary site of storage of triacylglycerol is fat (adipose) tissue, whereas the primary sites for glycogen storage are muscle and the liver. In animals, when tissues need to generate a lot of ATP, as in exercising muscle, signals are sent to adipose tissue to hydrolyze triacylglycerols and to release the fatty acids into the circulatory system so that they can move to and be transported into the ATP-requiring tissues.

Just as there are four stages in the oxidation of glucose, there are four stages in the oxidation of fatty acids. To optimize the efficiency of ATP generation, part of stage II (citric acid cycle oxidation of acetyl CoA) and all of stages III and IV of fatty acid oxidation are identical to those of glucose oxidation. The differences lie in cytosolic stage I and in the first part of mitochondrial stage II. In stage I, fatty acids are converted to a fatty acyl CoA in the cytosol in a reaction coupled to the hydrolysis of ATP to AMP and $\mathrm{PP}_{\mathrm{i}}$ (inorganic pyrophosphate) (see Figure 12-14):
![li684542480e](li684542480e.jpg)

Subsequent hydrolysis of $\mathrm{PP}_{\mathrm{i}}$ to two molecules of $\mathrm{P}_{\mathrm{i}}$ releases energy that drives this reaction to completion. To enter the mitochondrial matrix, the fatty acyl group must be covalently transferred to a molecule called carnitine and moved across the inner mitochondrial membrane by an acylcarnitine transporter protein (see Figure 12-14, blue oval); then, on the matrix side, the fatty acyl group is released from carnitine
and reattached to another CoA molecule. The activity of the acylcarnitine transporter is regulated to prevent oxidation of fatty acids when cells have adequate energy (ATP) supplies.

In the first part of stage II, each molecule of a fatty acyl CoA in the mitochondrion is oxidized in a cyclical sequence of four reactions in which all the carbon atoms are converted, two at a time, to acetyl CoA with generation of $\mathrm{FADH}_{2}$ and NADH (Figure 12-18a). For example, mitochondrial oxidation of each molecule of the 18 -carbon stearic acid, $\mathrm{CH}_{3}\left(\mathrm{CH}_{2}\right)_{16} \mathrm{COOH}$, yields nine molecules of acetyl CoA and eight molecules each of NADH and $\mathrm{FADH}_{2}$. In the second part of stage II, as with acetyl CoA generated from pyruvate, these acetyl groups enter the citric acid cycle and are oxidized to $\mathrm{CO}_{2}$. As will be described in detail in the next section, the reduced NADH and $\mathrm{FADH}_{2}$ with their high-energy electrons will be used in stage III to generate a proton-motive force, which in turn is used in stage IV to power ATP synthesis.

## Peroxisomal Oxidation of Fatty Acids Generates No ATP

Mitochondrial oxidation of fatty acids is the major source of ATP in mammalian liver cells, and biochemists at one time believed this was true in all cell types. However, rats treated with clofibrate, a drug that affects many features of lipid metabolism, were found to exhibit an increased rate of fatty acid oxidation and a large increase in the number of peroxisomes in their liver cells. This finding suggested that peroxisomes, as well as mitochondria, can oxidize fatty acids. These small organelles, $0.2-1 \mu \mathrm{~m}$ in diameter, are lined by a single membrane (see Figure 1-12). They are present in all mammalian cells except erythrocytes and are also found in plant cells, yeasts, and probably most other eukaryotic cells.

Mitochondria preferentially oxidize short-chain [fewer than 8 carbons ( $<$ C8)], medium-chain (C8-C12), and longchain (C14-C20) fatty acids, whereas peroxisomes preferentially oxidize very long chain fatty acids (VLCFAs, $>\mathrm{C}_{20}$ ), which cannot be oxidized by mitochondria. Most dietary fatty acids have long chains, which means that they are oxidized mostly in mitochondria. In contrast to mitochondrial oxidation of fatty acids, which is coupled to generation of ATP, peroxisomal oxidation of fatty acids is not linked to ATP formation, and energy is released as heat.

The reaction pathway by which fatty acids are degraded to acetyl CoA in peroxisomes is similar to that used in mitochondria (Figure 12-18b). However, peroxisomes lack an electrontransport chain, and electrons from the $\mathrm{FADH}_{2}$ produced during the oxidation of fatty acids are immediately transferred to $\mathrm{O}_{2}$ by oxidases, regenerating FAD and forming hydrogen peroxide $\left(\mathrm{H}_{2} \mathrm{O}_{2}\right)$. In addition to oxidases, peroxisomes contain abundant catalase, which quickly decomposes the $\mathrm{H}_{2} \mathrm{O}_{2}$, a highly cytotoxic metabolite. NADH produced during peroxisomal oxidation of fatty acids is exported and reoxidized in the cytosol; there is no need for a malate-aspartate shuttle here. Peroxisomes also lack the citric acid cycle, so acetyl CoA generated during peroxisomal degradation of fatty acids cannot be oxidized further; instead, it is transported into the cytosol for use in the synthesis of cholesterol (see Chapter 7) and other metabolites.

FIGURE 12-18 Oxidation of fatty acids in mitochondria and in peroxisomes. In both mitochondrial oxidation (a) and peroxisomal oxidation (b), fatty acids are converted to acetyl CoA by a series of four enzyme-catalyzed reactions (shown down the center of the figure). A fatty acyl CoA molecule is converted to acetyl CoA and a fatty acyl CoA shortened by two carbon atoms. Concomitantly, one FAD molecule is reduced to $\mathrm{FADH}_{2}$ and one $\mathrm{NAD}^{+}$molecule is reduced to NADH. The cycle is repeated on the shortened acyl CoA until fatty acids with an even number of carbon atoms are completely converted to acetyl CoA. In mitochondria, electrons from $\mathrm{FADH}_{2}$ and NADH enter the electron-transport chain and are ultimately used to generate ATP; the acetyl CoA generated is oxidized in the citric acid cycle, resulting in the release of $\mathrm{CO}_{2}$ and ultimately the synthesis of additional ATP. Because peroxisomes lack the protein complexes composing the electrontransport chain and the enzymes of the citric acid cycle, oxidation of fatty acids in these organelles yields no ATP.
![li694542480e](li694542480e.jpg)

## KEY CONCEPTS OF SECTION 12.3

## The Citric Acid Cycle and Fatty Acid Oxidation

- In stage II of glucose oxidation, the three-carbon pyruvate molecule is first oxidized to generate one molecule each of $\mathrm{CO}_{2}, \mathrm{NADH}$, and acetyl CoA. The acetyl group of acetyl CoA is then oxidized to $\mathrm{CO}_{2}$ by the citric acid cycle (see Figure 12-14).
- Each turn of the citric acid cycle releases two molecules of $\mathrm{CO}_{2}$ and generates three NADH molecules, one $\mathrm{FADH}_{2}$ molecule, and one GTP (see Figure 12-16).
- Most of the energy released in stages I and II of glucose oxidation is temporarily stored in the reduced coenzymes NADH and $\mathrm{FADH}_{2}$, which carry high-energy electrons that subsequently drive the electron-transport chain (stage III).
- Neither glycolysis nor the citric acid cycle directly uses molecular oxygen $\left(\mathrm{O}_{2}\right)$.
- The malate-aspartate shuttle regenerates the supply of cytosolic $\mathrm{NAD}^{+}$necessary for continued glycolysis (see Figure 12-17).
- Like glucose oxidation, the oxidation of fatty acids takes place in four stages. In stage I, fatty acids are converted to fatty acyl CoA in the cytosol. In stage II, the fatty acyl CoA is first converted into multiple acetyl CoA molecules, with generation of NADH and $\mathrm{FADH}_{2}$. Then, as in glucose oxidation, the acetyl CoA enters the citric acid cycle. Stages III and IV are identical for fatty acid and glucose oxidation (see Figure 12-14).
- In most eukaryotic cells, oxidation of short- to long-chain fatty acids occurs in mitochondria with production of ATP, whereas oxidation of very long chain fatty acids occurs primarily in peroxisomes and is not linked to ATP production (see Figure 12-18); the energy released during peroxisomal oxidation of fatty acids is converted to heat.

### 12.4 The Electron-Transport Chain and Generation of the Proton-Motive Force

Most of the energy released during the oxidation of glucose and fatty acids to $\mathrm{CO}_{2}$ (stages I and II) is converted into high-energy electrons in the reduced coenzymes NADH and $\mathrm{FADH}_{2}$. We now turn to stage III, in which the energy transiently stored in these reduced coenzymes is converted by an electron-transport chain, also known as the respiratory chain, into the proton-motive force. We first describe the logic and components of the electron-transport chain. Next we follow the path of electrons as they flow through the chain and describe the mechanism of proton pumping across the inner mitochondrial membrane. We conclude this section with a discussion of the magnitude of the proton-motive force produced by electron transport and proton pumping. In Section 12.5, we will see how the proton-motive force is used to synthesize ATP.

## Oxidation of NADH and $\mathrm{FADH}_{2}$ Releases a Significant Amount of Energy

During electron transport, electrons are released from NADH and $\mathrm{FADH}_{2}$ and eventually transferred to $\mathrm{O}_{2}$, forming $\mathrm{H}_{2} \mathrm{O}$, according to the following overall reactions:

$$
\begin{aligned}
\mathrm{NADH}+\mathrm{H}^{+}+1 / 2 \mathrm{O}_{2} & \rightarrow \mathrm{NAD}^{+}+\mathrm{H}_{2} \mathrm{O} \\
\Delta G & =-52.6 \mathrm{kcal} / \mathrm{mol} \\
\mathrm{FADH}_{2}+1 / 2 \mathrm{O}_{2} & \rightarrow \mathrm{FAD}+\mathrm{H}_{2} \mathrm{O} \\
\Delta G & =-43.4 \mathrm{kcal} / \mathrm{mol}
\end{aligned}
$$

Recall that the conversion of 1 glucose molecule to $\mathrm{CO}_{2}$ via the glycolytic pathway and citric acid cycle yields 10 NADH and $2 \mathrm{FADH}_{2}$ molecules (see Table 12-3). Oxidation of these reduced coenzymes has a total $\Delta G^{* \prime}$ of -613 $\mathrm{kcal} / \mathrm{mol}[10(-52.6)+2(-43.4)]$. Thus of the total potential free energy present in the chemical bonds of glucose $(-686 \mathrm{kcal} / \mathrm{mol})$, about 90 percent is conserved in the reduced coenzymes. Why should there be two different coenzymes, NADH and $\mathrm{FADH}_{2}$ ? Although many of the reactions involved in glucose and fatty acid oxidation are sufficiently energetic to reduce $\mathrm{NAD}^{+}$, several are not. To capture the energy released by those reactions, they are coupled to reduction of FAD, which requires less energy.

The energy carried in the reduced coenzymes can be released by oxidizing them. The biochemical challenge faced by the mitochondrion is to transfer, as efficiently as possible, the energy released by this oxidation into the energy in the terminal phosphoanhydride bond in ATP.

$$
\begin{gathered}
\mathrm{P}_{\mathrm{i}}^{2-}+\mathrm{H}^{+}+\mathrm{ADP}^{3-} \rightarrow \mathrm{ATP}^{4-}+\mathrm{H}_{2} \mathrm{O} \\
\Delta G=+7.3 \mathrm{kcal} / \mathrm{mol}
\end{gathered}
$$

A relatively simple one-to-one reaction involving reduction of one coenzyme molecule and synthesis of one ATP molecule would be terribly inefficient because the $\Delta \mathrm{G}^{* \prime}$ for ATP
generation from ADP and $\mathrm{P}_{\mathrm{i}}$ is substantially less than that for the coenzyme oxidation, and much energy would be lost as heat. To efficiently recover that energy, the mitochondrion converts the energy of coenzyme oxidation into a protonmotive force using a series of electron carriers, all but one of which are integral components of the inner membrane (see stage III in Figure 12-14). The proton-motive force can then be used to generate ATP very efficiently.

## Electron Transport in Mitochondria Is Coupled to Proton Pumping

During electron transport from NADH and $\mathrm{FADH}_{2}$ to $\mathrm{O}_{2}$, protons from the mitochondrial matrix are pumped across the inner membrane. This pumping raises the pH of the mitochondrial matrix relative to the intermembrane space and cytosol and also makes the matrix more electrically negative with respect to the intermembrane space. In other words, the free energy released during the oxidation of NADH or $\mathrm{FADH}_{2}$ is stored both as a proton concentration gradient and as an electrical gradient across the membrane-collectively known as the proton-motive force (see Figure 12-2). As we will see in Section 12.5, the movement of protons back across the inner membrane, driven by this force, is coupled to the synthesis of ATP from ADP and $\mathrm{P}_{\mathrm{i}}$ by ATP synthase (stage IV).

The synthesis of ATP from ADP and $\mathrm{P}_{\mathrm{i}}$, driven by the energy released by transfer of electrons from NADH or $\mathrm{FADH}_{2}$ to $\mathrm{O}_{2}$, is the major source of ATP in aerobic non-photosynthetic cells. Much evidence shows that in mitochondria and bacteria, this process of oxidative phosphorylation depends on the generation of a proton-motive force across the inner membrane (in mitochondria) or bacterial plasma membrane, with electron transport, proton pumping, and ATP formation occurring simultaneously. In the laboratory, for instance, addition of $\mathrm{O}_{2}$ and an oxidizable substrate such as pyruvate or succinate to isolated intact mitochondria results in net synthesis of ATP if the inner mitochondrial membrane is intact. In the presence of minute amounts of detergents that make the membrane leaky, electron transport and the oxidation of these metabolites by $\mathrm{O}_{2}$ still occurs. However, no ATP is made under these conditions because the proton leak prevents the maintenance of the proton-motive force.

The coupling between electron transport from NADH (or $\mathrm{FADH}_{2}$ ) to $\mathrm{O}_{2}$ and proton transport across the inner mitochondrial membrane can be demonstrated experimentally with isolated, intact mitochondria (Figure 12-19). As soon as $\mathrm{O}_{2}$ is added to a suspension of mitochondria in an otherwise $\mathrm{O}_{2}$-free solution that contains NADH, the medium outside the mitochondria transiently becomes more acidic (increased proton concentration) because the mitochondrial outer membrane is freely permeable to protons. (Remember that the malate-aspartate shuttle and other shuttles can convert the NADH in the solution into NADH in the mitochondrial matrix.) Once the $\mathrm{O}_{2}$ is depleted by its reduction, the excess protons in the medium slowly leak back into the matrix. By measuring the pH change in such experiments, one can

EXPERIMENTAL FIGURE 12-19 Electron transfer from NADH to $\mathrm{O}_{2}$ is coupled to proton transport across the mitochondrial membrane. If NADH is added to a suspension of mitochondria depleted of $\mathrm{O}_{2}$, no NADH is oxidized. When a small amount of $\mathrm{O}_{2}$ is added to the system (arrow), there is a sharp rise in the concentration of protons in the surrounding medium outside the mitochondria (decrease in pH ). Thus the oxidation of NADH by $\mathrm{O}_{2}$ is coupled to the movement of protons out of the matrix. Once the $\mathrm{O}_{2}$ is depleted, the excess protons slowly move back into the mitochondria (powering the synthesis of ATP), and the pH of the extracellular medium returns to its initial value.
![li704542480e](li704542480e.jpg)
calculate that about 10 protons are transported out of the matrix for every electron pair transferred from NADH to $\mathrm{O}_{2}$.

To obtain numbers for $\mathrm{FADH}_{2}$, the above experiment can be repeated using succinate instead of NADH as the substrate. (Recall that oxidation of succinate to fumarate in the citric acid cycle generates $\mathrm{FADH}_{2}$; see Figure 12-16.) The amount of succinate added can be adjusted so that the amount of $\mathrm{FADH}_{2}$ generated is equivalent to the amount of NADH in the first experiment. As in the first experiment, addition of $\mathrm{O}_{2}$ causes the medium outside the mitochondria to become acidic, but less so than with NADH. This difference is not surprising, because electrons in $\mathrm{FADH}_{2}$ have less potential energy ( $43.4 \mathrm{kcal} / \mathrm{mol}$ ) than electrons in NADH ( $52.6 \mathrm{kcal} / \mathrm{mol}$ ), and thus $\mathrm{FADH}_{2}$ drives the translocation of fewer protons from the matrix and a smaller change in pH .

## Electrons Flow "Downhill" Through a Series of Electron Carriers

Let's examine more closely the energetically favored movement of electrons from NADH and $\mathrm{FADH}_{2}$ to the final electron acceptor, $\mathrm{O}_{2}$. For simplicity, we will focus our discussion on NADH. In respiring mitochondria, each NADH molecule releases two electrons to the electron-transport chain; these electrons ultimately reduce one oxygen atom (half of an $\mathrm{O}_{2}$ molecule), forming one molecule of water:

$$
\begin{aligned}
& \mathrm{NADH} \rightarrow \mathrm{NAD}^{+}+\mathrm{H}^{+}+2 \mathrm{e}^{-} \\
& 2 \mathrm{e}^{-}+2 \mathrm{H}^{+}+1 / 2 \mathrm{O}_{2} \rightarrow \mathrm{H}_{2} \mathrm{O}
\end{aligned}
$$

As electrons move from NADH to $\mathrm{O}_{2}$, their electric potential declines by 1.14 V , which corresponds to $26.2 \mathrm{kcal} / \mathrm{mol}$ of electrons transferred, or about $53 \mathrm{kcal} / \mathrm{mol}$ for a pair of electrons. As noted earlier, much of this energy is conserved in the proton-motive force generated across the inner mitochondrial membrane.

Four large multiprotein complexes (complexes I-IV) compose the electron-transport chain in the inner mitochondrial membrane that is responsible for the generation of
the proton-motive force (see Figure 12-14, stage III). Each complex contains several prosthetic groups that participate in the process of moving electrons from donor molecules to acceptor molecules in coupled oxidation-reduction reactions (see Chapter 2). These small nonpeptide organic molecules or metal ions are tightly and specifically associated with the multiprotein complexes (Table 12-4).

Heme and the Cytochromes Several types of heme, an ironcontaining prosthetic group similar to that found in hemoglobin and myoglobin (Figure 12-20a), are tightly bound (covalently or noncovalently) to a set of mitochondrial proteins called cytochromes. Each cytochrome is designated by

| TABLE 12-4 Electron-Carrying Prosthetic Groups in the Electron-Transport Chain |  |
| :--: | :--: |
| Protein Component | Prosthetic Groups* |
| NADH-CoQ reductase (complex I) | FMN <br> $\mathrm{Fe}-\mathrm{S}$ |
| Succinate-CoQ reductase (complex II) | FAD <br> $\mathrm{Fe}-\mathrm{S}$ |
| $\mathrm{CoQH}_{2}$-cytochrome $c$ reductase (complex III) | Heme $b_{1}$ <br> Heme $b_{\mathrm{H}}$ <br> Fe-S <br> Heme $c_{1}$ |
| Cytochrome $c$ | Heme $c$ |
| Cytochrome $c$ oxidase (complex IV) | $\mathrm{Cu}_{2}{ }^{2+}$ <br> Heme $a$ <br> $\mathrm{Cu}_{3}{ }^{2+}$ <br> Heme $a_{3}$ |

[^0]
[^0]:    *Not included is coenzyme Q , an electron carrier that is not permanently bound to a protein complex.
    source: Data from J. W. De Pierre and L. Ernster, 1977, Annu. Rev. Biochem. 46:201.

![li714542480e](li714542480e.jpg)

FIGURE 12-20 Heme and iron-sulfur prosthetic groups in the electron-transport chain.
(a) Heme portion of cytochromes $b_{r}$ and $b_{m}$, which are components of $\mathrm{CoQH}_{2}$-cytochrome c reductase (complex III). The same porphyrin ring (yellow) is present in all hemes. The chemical substituents attached to the porphyrin ring differ in the other cytochromes in the electron-transport chain. All hemes accept and release one electron at a time. (b) Dimeric iron-sulfur cluster (Fe-S). Each Fe atom is bonded to four $S$ atoms: two are inorganic sulfur, and two are in cysteine side chains of the associated protein. All Fe-S clusters accept and release one electron at a time.
a letter, such as $a, b, c$, or $c_{1}$. Electron flow through the cytochromes occurs by oxidation and reduction of the Fe atom in the center of the heme molecule:

$$
\mathrm{Fe}^{3+}+\mathrm{e}^{-} \rightleftharpoons \mathrm{Fe}^{2+}
$$

Because the heme ring in cytochromes consists of alternating double- and single-bonded atoms, a large number of resonance hybrid forms exist. These forms allow the extra electron delivered to the cytochrome to be spread throughout the heme carbon and nitrogen atoms as well as the Fe ion.

The various cytochromes each have slightly different heme groups and surrounding atoms (called axial ligands), which generate different environments for the Fe ion. Therefore, each cytochrome has a different reduction potential, or tendency to accept an electron-an important property that dictates the unidirectional, energetically "downhill" electron flow along the chain. Just as water spontaneously flows downhill from a higher to a lower potential energy state-but not uphill-electrons flow in only one direction from one heme (or other prosthetic group) to another due to their differing reduction potentials. (For more on the concept of reduction potential, $E$, see Chapter 2.) All the cytochromes except cytochrome $c$ are components of integral membrane multiprotein complexes in the inner mitochondrial membrane.

Iron-Sulfur Clusters Iron-sulfur clusters are nonheme, ironcontaining prosthetic groups consisting of Fe atoms bonded both to inorganic sulfur (S) atoms and to S atoms on cysteine residues in a protein (Figure 12-20b). Some Fe atoms in the cluster bear a +2 charge; others have a +3 charge. However, the net charge of each Fe atom is actually between +2 and +3 , because electrons in their outermost orbitals, together with the extra electron delivered via the transport chain, are dispersed among the Fe atoms and move rapidly from one atom to another. Iron-sulfur clusters accept and release electrons one at a time.

Coenzyme Q Coenzyme $Q$ (CoQ), also called ubiquinone, is the only small-molecule electron carrier in the electrontransport chain that is not an essentially irreversibly proteinbound prosthetic group (Figure 12-21). It is a carrier of both protons and electrons. The oxidized quinone form of

CoQ can accept a single electron to form a semiquinone, a charged free radical denoted by $\mathrm{CoQ}^{*-}$. Addition of a second electron and two protons (thus a total of two hydrogen atoms) to $\mathrm{CoQ}^{*-}$ forms dihydroubiquinone $\left(\mathrm{CoQH}_{2}\right)$, the fully reduced form. Both CoQ and $\mathrm{CoQH}_{2}$ are soluble in phospholipids and diffuse freely in the hydrophobic center of the inner mitochondrial membrane. These properties underlie ubiquinone's role in the electron-transport chain: carrying electrons and protons between the membrane-embedded protein complexes of the chain.

Next we consider in detail the multiprotein complexes that use these prosthetic groups and the paths taken by electrons and protons as they pass through these complexes.
![li724542480e](li724542480e.jpg)

FIGURE 12-21 Oxidized and reduced forms of coenzyme $Q$ (CoQ), which can carry two protons and two electrons. Because of its long hydrocarbon "tail" of isoprene units, CoQ, also called ubiquinone, is soluble in the hydrophobic core of phospholipid bilayers and is very mobile. Reduction of CoQ to the fully reduced form, $\mathrm{QH}_{2}$ (dihydroquinone), occurs in two steps with a half-reduced free-radical intermediate, called semiquinone.

## Four Large Multiprotein Complexes Couple Electron Transport to Proton Pumping Across the Inner Mitochondrial Membrane

As electrons flow downhill from one electron carrier to the next in the electron-transport chain, the energy released is used to power the pumping of protons against their electrochemical gradient across the inner mitochondrial membrane. Four large multiprotein complexes (Figure 12-22) directly or indirectly couple the movement of electrons to proton pumping: NADH-CoQ reductase (complex I, $>40$ subunits), succinate-CoQ reductase (complex II, 4 subunits), $\mathrm{CoQH}_{2}$-cytochrome c reductase (complex III, 11 subunits), and cytochrome c oxidase (complex IV, 13 subunits). The electrons follow one of two routes through these complexes: $\mathrm{I} \rightarrow \mathrm{III} \rightarrow \mathrm{IV}$ or $\mathrm{II} \rightarrow \mathrm{III} \rightarrow \mathrm{IV}$. Complexes I, III, and IV all pump protons directly across the inner membrane, whereas complex II does not (see Figure 12-22). Electrons from NADH flow from complex I via $\mathrm{CoQ} /$ $\mathrm{CoQH}_{2}$ to complex III and then, via the soluble protein cytochrome $c$ (cyt $c$ ), to complex IV to reduce molecular oxygen (complex II is bypassed) (see Figure 12-22a); electrons from $\mathrm{FADH}_{2}$ flow from complex II via $\mathrm{CoQ} / \mathrm{CoQH}_{2}$ to complex III (see Figure 12-22b) and then via cytochrome
c to complex IV to reduce molecular oxygen (complex I is bypassed).

As shown in Figure 12-22, CoQ accepts electrons released from NADH-CoQ reductase (complex I) or succinate-CoQ reductase (complex II) and donates them to $\mathrm{CoQH}_{2}$-cytochrome $c$ reductase (complex III). Protons are simultaneously transported from the matrix side of the membrane (also called the cytosolic side) to the intermembrane space (also called the exoplasmic side). Whenever CoQ accepts electrons, it does so at a binding site on the matrix side of a protein complex, always picking up protons from the medium there. Whenever $\mathrm{CoQH}_{2}$ releases its electrons, it does so at a site on the intermembrane space side of a protein complex, releasing protons into the fluid of the intermembrane space. Thus the transport of each pair of electrons by CoQ is obligately coupled to the movement of two protons from the matrix to the intermembrane space.

NADH-CoQ Reductase (Complex I) Electrons are transferred from NADH to CoQ by NADH-CoQ reductase (see Figure 12-22a). Electron microscopy and x-ray crystallography of complex I from bacteria (mass $\sim 500 \mathrm{kDa}$, with 14 subunits) and from eukaryotes ( $\sim 1 \mathrm{MDa}$, with 14 highly conserved core
![li734542480e](li734542480e.jpg)

FIGURE 12-22 The mitochondrial electron-transport chain. Electrons (blue arrows) flow through four major multiprotein complexes (I-IV). Electron movement between complexes is mediated either by the lipid-soluble molecule coenzyme Q (CoQ, oxidized form; $\mathrm{CoQH}_{2}$, reduced form) or the water-soluble protein cytochrome $c$ (cyt c). Three of the multiprotein complexes use the energy released from the electrons to pump protons (red arrows) from the matrix (cytosolic compartment in bacteria) to the intermembrane space (exoplasmic space of bacteria). (a) Pathway from NADH. Electrons from NADH (2/NADH oxidized) flow through complex I, initially via a flavin mononucleotide (FMN) and then sequentially via seven iron-sulfur clusters (Fe-S), to CoQ, to which two protons bind, forming $\mathrm{CoQH}_{2}$. Conformational changes in complex I that accompany the electron flow drive proton pumping from the matrix to the intramembrane space. Electrons then flow via the released (and subsequently recycled) $\mathrm{CoQH}_{2}$ to complex III, and then via cyt $c$ to
complex IV. Each of four cyt c molecules carrying four electrons originating from two NADHs transfers individual electrons to complex IV for the four-electron reduction of one $\mathrm{O}_{2}$ molecule to two $\mathrm{H}_{2} \mathrm{O}$ molecules and transport of four protons. Thus, for every two NADHs oxidized and one $\mathrm{O}_{2}$ molecule reduced, a total of 20 protons are translocated out of the matrix into the intermembrane space. (b) Pathway from succinate. Two electrons flow from each succinate to complex II via FAD/FADH ${ }_{2}$ and iron-sulfur clusters (Fe-S), from complex II to complex III via $\mathrm{CoQ} / \mathrm{CoQH}_{2}$, and then to complex IV via cyt $c$. Electrons released during oxidation of succinate to fumarate in complex II are used to reduce CoQ to $\mathrm{CoQH}_{2}$ without translocating additional protons. The remainder of electron transport from $\mathrm{CoQH}_{2}$ proceeds by the same pathway as for the NADH pathway in (a). Thus, for every two succinates oxidized and one $\mathrm{O}_{2}$ molecule reduced, 12 protons ( 8 via the Q -cycle of complex III, 4 via complex IV) are translocated.

![li744542480e](li744542480e.jpg)

FIGURE 12-23 Electron and proton transport through complexes I and II. (a) Model of complex I based on its three-dimensional structure. The outline of the shape of the core complex I from the yeast $Y$. lipolytica, as determined by x-ray crystallography, is shown in light blue, and the borders separating several distinct structural subunits are indicated by thin dashed black lines. From NADH, electrons flow first to a flavin mononucleotide (FMN) and then, via iron-sulfur clusters (Fe-S, red and yellow balls), to CoQ, to which two protons from the matrix bind (red arrow) to form $\mathrm{CoQH}_{2}$. Conformational changes due to the electron flow, which may be induced by changes in the charge of the CoQ and may include
a piston-like horizontal movement of the t-helix, drive proton pumping through the transmembrane subunits from the matrix to the intramembrane space (red arrows). (b) Model of complex II based on its threedimensional structure. Electrons flow through complex II from succinate to CoQ via FAD/FADH 2 and iron-sulfur clusters (Fe-S), and from complex II to complex III via $\mathrm{CoQ} / \mathrm{CoQH}_{2}$. Electrons released during oxidation of succinate to fumarate in complex II are used to reduce CoQ to $\mathrm{CoQH}_{2}$ without translocating additional protons. [Part (a) data from V. Zickermann et al., 2015, Science 347:44-49, PDB ID 3m9s. Part (b) data from F. Sun et al., 2005, Cell 121:1043-1057, PDB ID 1zoy.]
subunits shared with bacteria plus about 26-32 accessory subunits) has established that it is L-shaped (Figure 12-23a). The membrane-embedded arm of the L is slightly curved, about $180 \AA$ long, and comprises proteins with more than 60 transmembrane $\alpha$ helices. This arm has four subdomains, three of which contain proteins that are members of a family of cation antiporters. The hydrophilic peripheral arm extends over $130 \AA$ away from the membrane into the matrix (cytosolic) space.
$\mathrm{NAD}^{+}$is exclusively a two-electron carrier: it accepts or releases a pair of electrons simultaneously. In NADHCoQ reductase, the NADH-binding site is at the tip of the peripheral arm (see Figure 12-23a); electrons released from NADH first flow to FMN (flavin mononucleotide), a prosthetic group, or cofactor, related to FAD, then are shuttled about $95 \AA$ down the peripheral arm through a series of iron-sulfur clusters and finally to CoQ , which is bound at a site at least partially in the plane of the membrane. FMN, like FAD, can accept two electrons, but does so one electron at a time.

Each transported electron undergoes a drop in potential of about 360 mV , equivalent to a $\Delta \mathrm{G}^{\circ \prime}$ of $-16.6 \mathrm{kcal} /$ mol for the two electrons transported. Much of this released energy is used to transport four protons across the inner membrane per molecule of NADH oxidized by complex I.

Those four protons are distinct from the two protons that are transferred to the CoQ as illustrated in Figures 12-21, 12-22a, and 12-23a. The precise mechanism by which the energy released by electron transport in the peripheral arm is used to change the conformation of subunits in the membrane arm and thus mediate the movement of four protons across the membrane is uncertain. Three protons are likely to pass through the three cation antiporter domains via a zigzag series of polar side chains that can be protonated and that span the membrane. A similar series of side chains is the likely conduit for the fourth proton. A transverse $\alpha$ helix (t-helix) in the membrane arm runs parallel to the plane of the membrane, potentially mechanically linking the antiporter-like domains to the peripheral arm (see Figure 12-23a) and possibly contributing directly to the conformational changes required to convert the energy released by the electron transport in the peripheral arm into proton transport by the transmembrane domains.

The overall reaction catalyzed by this complex is
$\mathrm{NADH}+\mathrm{CoQ}+6 \mathrm{H}_{-\mathrm{in}}^{+}$
(Reduced) (Oxidized)

$$
\underset{(\text { Oxidized) }}{\mathrm{NAD}^{+}}+\mathrm{H}_{-\mathrm{in}}^{+} \mathrm{CoQH}_{2}+4 \mathrm{H}_{-\text {out }}^{+} \text { (Reduced) }
$$

Succinate-CoQ Reductase (Complex II) Succinate dehydrogenase, the enzyme that oxidizes a molecule of succinate to fumarate in the citric acid cycle (and in the process generates the reduced coenzyme $\mathrm{FADH}_{2}$ ), is one of the four subunits of complex II (Figure 12-23b). Thus the citric acid cycle is physically as well as functionally linked to the electron-transport chain. The two electrons released in the conversion of succinate to fumarate are transferred first to FAD in succinate dehydrogenase, then to iron-sulfur clusters-regenerating FAD-and finally to CoQ, which binds to a cleft on the matrix side of the transmembrane portions of complex II (see Figures 12-22b and 12-23b). The pathway is somewhat reminiscent of that in complex I (see Figure 12-23a).

The overall reaction catalyzed by this complex is

$$
\begin{aligned}
& \text { Succinate }+\mathrm{CoQ} \rightarrow \text { fumarate }+\mathrm{CoQH}_{2} \\
& \text { (Reduced) (Oxidized) (Oxidized) (Reduced) }
\end{aligned}
$$

Although the $\Delta G^{\circ \prime}$ for this reaction is negative, the released energy is insufficient for proton pumping in addition to reduction of CoQ to form $\mathrm{CoQH}_{2}$. Thus no protons are translocated directly across the membrane by succinate-CoQ reductase, and no proton-motive force is generated in this part of the electron-transport chain. We will see shortly how the protons and electrons in the $\mathrm{CoQH}_{2}$ molecules generated by complexes I and II contribute to the generation of the proton-motive force.

Complex II generates $\mathrm{CoQH}_{2}$ from succinate via FAD/ $\mathrm{FADH}_{2}$-mediated redox reactions. Another set of proteins in the matrix and inner mitochondrial membrane performs a comparable set of FAD/FADH ${ }_{2}$-mediated redox reactions, producing $\mathrm{CoQH}_{2}$ from fatty acid oxidation and contributing electrons to the electron-transport chain (see Figure 12-18). Fatty acyl-CoA dehydrogenase, which is a water-soluble enzyme, catalyzes the first step of the oxidation of fatty acyl CoA in the mitochondrial matrix. There are several fatty acyl-CoA dehydrogenase enzymes with specificities for fatty acyl chains of different lengths. These enzymes mediate the initial step in a four-step process that removes two carbons from the fatty acyl group by oxidizing the carbon in the $\beta$ position of the fatty acyl chain (thus the entire process is often referred to as $\beta$-oxidation). These reactions generate acetyl CoA, which in turn enters the citric acid cycle. They also generate an $\mathrm{FADH}_{2}$ intermediate and NADH. The $\mathrm{FADH}_{2}$ generated remains bound to the enzyme during the redox reaction, as is the case for complex II. A water-soluble protein called electron transfer flavoprotein (ETF) transfers the high-energy electrons from the $\mathrm{FADH}_{2}$ in the fatty acyl-CoA dehydrogenase to electron transfer flavoprotein:ubiquinone oxidoreductase (ETF:QO), a membrane protein that reduces CoQ to $\mathrm{CoQH}_{2}$ in the inner membrane. This $\mathrm{CoQH}_{2}$ intermixes in the membrane with the other $\mathrm{CoQH}_{2}$ molecules generated by complexes I and II, all contributing to proton transport out of the matrix by complex III.

CoQH $_{2}$-Cytochrome c Reductase (Complex III) A $\mathrm{CoQH}_{2}$ generated by complex I, complex II, or ETF:QO donates two
electrons to $\mathrm{CoQH}_{2}$-cytochrome $c$ reductase (complex III), regenerating oxidized CoQ. Concomitantly, it releases into the intermembrane space two protons previously picked up by CoQ on the matrix face, generating part of the protonmotive force (see Figure 12-22). Within complex III, the released electrons are first transferred to an iron-sulfur cluster within the complex and then to cytochrome $c_{1}$ or to two $b$-type cytochromes ( $b_{L}$ and $b_{\mathrm{H}}$, see the description of the Q cycle below). Finally, the two electrons are transferred sequentially to two molecules of the oxidized form of cytochrome $c$, a water-soluble peripheral protein that diffuses in the intermembrane space. For each pair of electrons transferred, the overall reaction catalyzed by complex III is
$\mathrm{CoQH}_{2}+2 \mathrm{Cyt} c^{3+}+2 \mathrm{H}^{+}{ }_{\text {in }} \rightarrow \mathrm{CoQ}+4 \mathrm{H}^{+}{ }_{\text {out }}+2 \mathrm{Cyt} c^{2+}$
(Reduced) (Oxidized) (Oxidized) (Reduced)
The $\Delta G^{\circ \prime}$ for this reaction is sufficiently negative that two protons in addition to those from $\mathrm{CoQH}_{2}$ are translocated from the mitochondrial matrix across the inner membrane for each pair of electrons transferred; this transfer involves the proton-motive Q cycle, discussed below. The heme protein cytochrome $c$ and the small lipid-soluble molecule CoQ play similar roles in the electron-transport chain in that they both serve as mobile electron shuttles, transferring electrons (and thus energy) between the complexes of the electrontransport chain.

The Q Cycle Experiments have shown that four protons are translocated across the inner mitochondrial membrane per electron pair transported from $\mathrm{CoQH}_{2}$ through complex III. These four protons are those carried on two $\mathrm{CoQH}_{2}$ molecules, which are converted to two CoQ molecules during the cycle. However, another CoQ molecule receives two other protons from the matrix and is converted to one $\mathrm{CoQH}_{2}$ molecule. Thus the net overall reaction involves the conversion of only one $\mathrm{CoQH}_{2}$ molecule to CoQ as two electrons are transferred, one at a time, to two molecules of the acceptor cytochrome $c$. An evolutionarily conserved mechanism, called the $Q$ cycle, is responsible for the two-for-one transport of protons and electrons by complex III (Figure 12-24).

The substrate for complex III, $\mathrm{CoQH}_{2}$, is generated by several enzymes, including NADH-CoQ reductase (complex I), succinate-CoQ reductase (complex II), ETF:QO (during $\beta$-oxidation), and as we shall see, by complex III itself.

As shown in Figure 12-24, in one turn of the Q cycle, two molecules of $\mathrm{CoQH}_{2}$ are oxidized to CoQ at the Q , site in complex III and release a total of four protons into the intermembrane space, but at the Q , site, one molecule of $\mathrm{CoQH}_{2}$ is regenerated from CoQ and two additional protons from the matrix. The translocated protons are all derived from $\mathrm{CoQH}_{2}$, which obtained its protons from the matrix, as described above. Although seemingly cumbersome, the Q cycle optimizes the number of protons pumped per pair of electrons moving through complex III. The Q cycle is found in all plants and animals as well as in bacteria. Its formation at a very early stage of cellular evolution was probably

![li754542480e](li754542480e.jpg)

FIGURE 12-24 The $Q$ cycle. The $Q$ cycle of complex III uses the net oxidation of one $\mathrm{CoQH}_{2}$ molecule to transfer four protons into the intermembrane space and two electrons to two cytochrome $c$ molecules. The cycle begins when a molecule from the combined pool of reduced $\mathrm{CoQH}_{2}$ in the inner mitochondrial membrane binds to the $\mathrm{Q}_{\mathrm{o}}$ site on the intermembrane space (exoplasmic) side of the transmembrane portion of complex III (step $\mathbf{M}$ ). There $\mathrm{CoQH}_{2}$ releases two protons into the intermembrane space (step $\mathbf{M}$ ), and two electrons and the resulting CoQ dissociate (step $\mathbf{M}$ ). One of the electrons is transported, via an iron-sulfur protein and cytochrome $c_{1}$, directly to cytochrome $c$ (step MII). (Recall that each cytochrome $c$ shuttles one electron from complex III to complex IV.) The other electron moves through cytochromes $b_{1}$ and $b_{0}$, and partially reduces an oxidized CoQ molecule bound to the second, $\mathrm{Q}_{\mathrm{o}}$ site on the matrix (cytosolic) side of the complex, forming a CoQ semiquinone anion, $\mathrm{Q}^{--}$(step M). The process is repeated with the binding of a second $\mathrm{CoQH}_{2}$ at the $\mathrm{Q}_{\mathrm{o}}$ site (step M), proton release (step MII), reduction of another cytochrome $c$ (step MII), and addition of the other electron to the $\mathrm{Q}^{--}$bound at the $Q_{i}$ site (step M). There the addition of two protons from the matrix yields a fully reduced $\mathrm{CoQH}_{2}$ molecule at the $\mathrm{Q}_{i}$ site, which then dissociates (steps $\mathbf{M}$ and $\mathbf{M}$ ), freeing the $Q_{i}$ to bind a new molecule of CoQ (step MII) and begin the Q cycle over again. See B. Trumpower, 1990, J. Biol. Chem. 265:11409, and E. Darrouzet et al., 2001, Trends Biochem. Sci. 26:445.

Per $2 e^{-}$transferred through complex III to cytochrome $c, 4 \mathrm{H}^{+}$ released to the intermembrane space
essential for the success of all life forms as a way of converting the potential energy in reduced coenzyme Q into the maximum proton-motive force across a membrane. In turn, this process maximizes the number of ATP molecules synthesized from each electron that moves down the electrontransport chain from NADH or $\mathrm{FADH}_{2}$ to $\mathrm{O}_{2}$.

How are the two electrons released from $\mathrm{CoQH}_{2}$ at the $\mathrm{Q}_{\mathrm{o}}$ site directed to different acceptors, either to $\mathrm{Fe}-\mathrm{S}$, cytochrome $c_{1}$, and then cytochrome $c$ (upward pathway in Figure 12-24), or alternatively to cytochrome $b_{1}$, cytochrome $b_{\mathrm{H}}$, and then CoQ at the $\mathrm{Q}_{\mathrm{i}}$ site (downward pathway in Figure 12-24)? The mechanism involves a flexible hinge in the $\mathrm{Fe}-\mathrm{S}$-containing protein subunit of complex III. Initially, the $\mathrm{Fe}-\mathrm{S}$ cluster is close enough to the $\mathrm{Q}_{\mathrm{o}}$ site to pick up an electron from $\mathrm{CoQH}_{2}$ bound there. Once this happens, the subunit containing this $\mathrm{Fe}-\mathrm{S}$ cluster swings the cluster away from the $\mathrm{Q}_{\mathrm{o}}$ site to a position near enough to the heme on cytochrome $c_{1}$ for electron transfer to occur. With the $\mathrm{Fe}-\mathrm{S}$-containing subunit in this alternate conformation, the second electron released from the $\mathrm{CoQH}_{2}$ bound to the $\mathrm{Q}_{\mathrm{o}}$ site cannot move to the $\mathrm{Fe}-\mathrm{S}$ cluster-it is too far away, so it takes an alternative path open to it via a somewhat less thermodynamically favored route to cytochrome $b_{1}$ and through cytochrome $b_{\mathrm{H}}$ to the CoQ at the $\mathrm{Q}_{\mathrm{i}}$ site.

Cytochrome c Oxidase (Complex IV) Cytochrome $c$, after being reduced by one electron from complex III, is reoxidized as it transports its electron to cytochrome $c$ oxidase (complex IV) (see Figure 12-22a). Mitochondrial cytochrome $c$ oxidases contain 13 different subunits, but the catalytic core of the enzyme consists of only three. The functions of the remaining subunits are not well understood. Bacterial cytochrome $c$ oxidases contain only the three catalytic subunits. In both mitochondria and bacteria, four molecules of reduced cytochrome $c$ bind, one at a time, to the oxidase. An electron is transferred from the heme of each cytochrome $c$, first to the pair of copper ions labeled $\mathrm{Cu}_{\mathrm{x}}{ }^{2+}$, then to the heme in cytochrome $a_{x}$ and next to the oxygen reduction center, composed of $\mathrm{Cu}_{\mathrm{x}}{ }^{2+}$ and the heme in cytochrome $a_{3}$, which together bind in a sandwich fashion and sequester the $\mathrm{O}_{2}$ molecule that will be reduced to two $\mathrm{H}_{2} \mathrm{Os}$. Several lines of evidence suggest that $\mathrm{O}_{2}$ reaches the reduction center via one or more hydrophobic channels through the protein, into which $\mathrm{O}_{2}$ flows from the hydrophobic core of the membrane, where it is more soluble than in aqueous solution (see Figure 12-22a).

The four electrons are finally passed by complex IV to $\mathrm{O}_{2}$, the ultimate electron acceptor, yielding two $\mathrm{H}_{2} \mathrm{O}$, which together with $\mathrm{CO}_{2}$ is one of the end products of the overall

oxidation pathway. Proposed intermediates in oxygen reduction include the peroxide anion $\left(\mathrm{O}_{2}{ }^{2-}\right)$ and the hydroxyl radical $\left(\mathrm{OH}^{\bullet}\right)$, as well as unusual complexes of iron and oxygen atoms. These intermediates would be harmful to the cell if they escaped from complex IV, but they do so only rarely (see the discussion of reactive oxygen species below) because they are sequestered by the $\mathrm{Cu}_{0}{ }^{2+}$ and the heme in cytochrome $a_{3}$. To generate $\mathrm{H}_{2} \mathrm{O}$ from the reduced $\mathrm{O}_{2}$, protons are channeled to the reduction center. In addition, other protons are transported across the membrane from the matrix to the intermembrane space. During the transport of four electrons through the cytochrome $c$ oxidase complex, four protons from the matrix are translocated across the membrane. Thus complex IV transports only one proton per electron transferred, whereas complex III, using the Q cycle, transports two protons per electron transferred. However, the mechanism by which complex IV translocates these protons and energetically couples the translocation to $\mathrm{O}_{2}$ reduction is not fully understood.

For each four electrons transferred, the overall reaction catalyzed by cytochrome $c$ oxidase is

$$
\begin{aligned}
& 4 \mathrm{cyt} c^{2+}+8 \mathrm{H}^{+} \mathrm{in}+\mathrm{O}_{2} \rightarrow 4 \mathrm{cyt} c^{3+}+2 \mathrm{H}_{2} \mathrm{O}+4 \mathrm{H}^{+} \text {out } \\
& \text { (Reduced) } \\
& \text { (Oxidized) }
\end{aligned}
$$

The poison cyanide, which has been used as a chemical warfare agent, by spies to commit suicide when captured, in gas chambers to execute prisoners, and by the Nazis (Zyklon B gas) for the mass murder of Jews and others, is toxic because it binds to the heme $a_{3}$ in mitochondrial cytochrome $c$ oxidase (complex IV), inhibiting electron transport and thus oxidative phosphorylation and production of ATP. Cyanide is one of many toxic small molecules that interfere with energy production in mitochondria.

## The Reduction Potentials of Electron Carriers in the Electron-Transport Chain Favor Electron Flow from NADH to $\mathrm{O}_{2}$

As we saw in Chapter 2, the reduction potential $(E)$ for a partial reduction reaction

$$
\text { Oxidized molecule }+\mathrm{e}^{-} \rightleftharpoons \text { reduced molecule }
$$

is a measure of the equilibrium constant of that partial reaction. With the exception of the $b$ cytochromes in complex III ( $\mathrm{CoQH}_{2}$-cytochrome $c$ reductase), the standard reduction potential $E^{\circ \prime}$ of the electron carriers in the electron-transport chain increases steadily from NADH to $\mathrm{O}_{2}$. For instance, for the partial reaction

$$
\mathrm{NAD}^{+}+\mathrm{H}^{+}+2 \mathrm{e}^{-} \rightleftharpoons \mathrm{NADH}
$$

the value of the standard reduction potential is -320 mV , which is equivalent to a $\Delta \mathrm{G}^{\circ \prime}$ of $+14.8 \mathrm{kcal} / \mathrm{mol}$ for transfer of two electrons. Thus this partial reaction tends to
proceed toward the left; that is, toward the oxidation of NADH to $\mathrm{NAD}^{+}$.

In contrast, the standard reduction potential for the partial reaction

$$
\text { Cytochrome } c_{\mathrm{ox}}\left(\mathrm{Fe}^{3+}\right)+\mathrm{e}^{-} \rightleftharpoons \text { cytochrome } c_{\mathrm{red}}\left(\mathrm{Fe}^{2+}\right)
$$

is $+220 \mathrm{mV}\left(\Delta G^{\circ \prime}=-5.1 \mathrm{kcal} / \mathrm{mol}\right)$ for transfer of one electron. Thus this partial reaction tends to proceed toward the right; that is, toward the reduction of cytochrome $c\left(\mathrm{Fe}^{3+}\right)$ to cytochrome $c\left(\mathrm{Fe}^{2+}\right)$.

The final reaction in the electron-transport chain, the reduction of $\mathrm{O}_{2}$ to $\mathrm{H}_{2} \mathrm{O}$

$$
2 \mathrm{H}^{+}+1 / 2 \mathrm{O}_{2}+2 \mathrm{e}^{-} \rightarrow \mathrm{H}_{2} \mathrm{O}
$$

has a standard reduction potential of $+816 \mathrm{mV}\left(\Delta G^{\circ \prime}=\right.$ $-37.8 \mathrm{kcal} / \mathrm{mol}$ for transfer of two electrons), the most positive in the whole series; thus this reaction also tends to proceed toward the right.

As illustrated in Figure 12-25, the steady increase in $E^{\circ \prime}$ values, and the corresponding decrease in $\Delta G^{\circ \prime}$ values, of the carriers in the electron-transport chain favors the flow of electrons from NADH and $\mathrm{FADH}_{2}$ (generated from succinate) to $\mathrm{O}_{2}$. The energy released as electrons flow energetically "downhill" through the electron-transport chain complexes drives the pumping of protons against their concentration gradient across the inner mitochondrial membrane.

## The Multiprotein Complexes of the ElectronTransport Chain Assemble into Supercomplexes

Over 50 years ago, Britton Chance proposed that the electron-transport complexes might assemble into large supercomplexes. Doing so would bring the complexes into close and highly organized proximity, which might improve the speed and efficiency of the overall electron-transport process. Indeed, genetic, biochemical, and biophysical studies have provided very strong evidence for the existence of electron-transport chain supercomplexes. These studies involved polyacrylamide gel electrophoretic methods called blue native (BN)-PAGE and colorless native (CN)-PAGE, which permit separation of very large macromolecular protein complexes, and electron microscopic analysis of their three-dimensional structures. One such supercomplex contains one copy of complex I, a dimer of complex III (III2), and one or more copies of complex IV (Figure 12-26). When this I/III ${ }_{2} /$ IV supercomplex was isolated with ubiquinone (CoQ) and cytochrome $c$ from BN-PAGE gels, it was shown to transfer electrons from NADH to $\mathrm{O}_{2 \mathrm{I}}$ in other words, this supercomplex can respire-it is a respirasome. The precise function of supercomplex formation in the context of the very high protein concentration in the inner mitochondrial membrane remains to be established with certainty, but is thought to involve improving the speed and efficiency of electron transport, stabilizing individual multiprotein complexes, or preventing inappropriate protein aggregates.

![li764542480e](li764542480e.jpg)

FIGURE 12-25 Changes in reduction potential and free energy during the stepwise flow of electrons through the electron-transport chain. Blue arrows indicate electron flow; red arrows, translocation of protons across the inner mitochondrial membrane. Electrons pass through the multiprotein complexes from those with a lower reduction potential to those with a higher (more positive) reduction potential (left scale), with a corresponding reduction in free energy (right scale). The energy released as electrons flow through three of the complexes is sufficient to power the pumping of $\mathrm{H}^{+}$ions across the membrane, establishing a proton-motive force.

The unique phospholipid cardiolipin (diphosphatidyl glycerol) appears to play an important role in the assembly and function of these supercomplexes.
![li774542480e](li774542480e.jpg)

Generally not observed in other membranes of eukaryotic cells, cardiolipin has been observed to bind to integral membrane proteins of the inner mitochondrial membrane (e.g., complex II). Genetic and biochemical studies in yeast mutants in which cardiolipin synthesis is blocked have
established that cardiolipin contributes to the formation and activity of mitochondrial supercomplexes; thus it has been called the glue that holds together the electron-transport chain, though the precise mechanism by which it does so remains to be defined. In addition, there is evidence that cardiolipin may influence the inner membrane's binding and permeability to protons and consequently the proton-motive force. Barth's syndrome is a human X-linked genetic disease caused by defects in an enzyme that determines the structures of the acyl chains on cardiolipin. The reduction in the amounts of cardiolipin in patients with Barth's syndrome and its abnormal structure result in heart and skeletal muscle defects, growth retardation, and other abnormalities.

## Reactive Oxygen Species Are By-Products of Electron Transport

About 1-2 percent of the oxygen metabolized by aerobic organisms, rather than being converted to water, is partially reduced to the superoxide anion radical $\left(\mathrm{O}_{2}^{--}\right.$, where the "dot" represents an unpaired electron).

![li784542480e](li784542480e.jpg)

EXPERIMENTAL FIGURE 12-26 Electrophoresis and electron microscopic imaging have identified an electron-transport chain supercomplex containing complexes I, III, and IV. (a) Membrane proteins in isolated bovine heart mitochondria were solubilized with a detergent, and the complexes and supercomplexes were separated by gel electrophoresis using the blue native (BN)-PAGE method. Each blue-stained band within the gel represents the indicated protein complex or supercomplex. The intensity of the blue stain is approximately proportional to the amount of complex or supercomplex present. (b) Supercomplex I/III ${ }_{2}$ IV was extracted from a BN-PAGE gel, frozen, and visualized by cryoelectron tomography. The left image shows the three-dimensional surface structure viewed from an orientation
parallel to the presumptive plane of the membrane. The right image is the same structure into which were fit models of the structures of the individual complexes: complex I (blue), dimer of complex III (III ${ }_{2}$, orange), and complex IV (green). Colored dashed lines represent the approximate outlines of these complexes. The complex I structure is based on essentially the entire complex I from the yeast $Y$. lipolytica, not just the 14 core subunits. (Part (a) from Schafer, E., et al., "Architecture of active mammalian respiratory chain supercomplexes," J. Biol. Chem. 2006 Jun 2; 281(22):15370-5. Epub 2006 Mar 20. Part (b) from Proc. Natl. Acad. Sci. USA 2011. 108(37):15196-15200, Fig. 2A and 3A, "Interaction of complexes I, III, and IV within the bovine respirasome by single particle cryoelectron tomography," by Dudkina et al.)
![li794542480e](li794542480e.jpg)

FIGURE 12-27 Generation and inactivation of toxic reactive oxygen species. Electrons from the electron-transport chains of mitochondria and chloroplasts, as well as some generated through other enzymatic reactions, reduce molecular oxygen $\left(\mathrm{O}_{2}\right)$, forming the highly reactive radical anion superoxide $\left(\mathrm{O}_{2}^{-}\right)$. Superoxide is rapidly converted by superoxide dismutase (SOD) to hydrogen peroxide $\left(\mathrm{H}_{2} \mathrm{O}_{2}\right)$, which in turn can be converted by metal ions such as $\mathrm{Fe}^{3+}$ to hydroxyl radicals $\left(\mathrm{OH}^{\circ}\right)$ or inactivated to $\mathrm{H}_{2} \mathrm{O}$ by enzymes such as catalase. Because of their high chemical reactivity, $\mathrm{O}_{2}^{-}, \mathrm{H}_{2} \mathrm{O}_{2}, \mathrm{OH}^{\circ}$, and similar molecules are called reactive oxygen species (ROS). They cause oxidative and free-radical damage to many biomolecules, including lipids, proteins, and DNA. This damage leads to cellular oxidative stress that can cause disease and, if sufficiently severe, can kill cells. In addition, ROS can function as intra- and intercellular signaling molecules.

electron carriers (iron, $\mathrm{FMN}, \mathrm{CoQH}_{2}$ ) in the chain. Usually such contact is prevented by sequestration of the carriers within the proteins involved. However, there are some sites (particularly in complex I and $\mathrm{CoQ}^{\bullet-}$, see Figure 12-21) and some conditions (e.g., high NADH/NAD ${ }^{+}$ratio in the matrix, high proton-motive force when ATP is not generated) when electrons can more readily "leak" out of the chain and reduce $\mathrm{O}_{2}$ to $\mathrm{O}_{2}^{\circ-}$.

The superoxide anion is an especially unstable and reactive ROS. Mitochondria have evolved several defense mechanisms that help protect against $\mathrm{O}_{2}^{2-}$ toxicity, including the use of enzymes that inactivate superoxide, first by converting it to $\mathrm{H}_{2} \mathrm{O}_{2}$ (Mn-containing superoxide dismutase, called SOD) and then to $\mathrm{H}_{2} \mathrm{O}$ (catalase) (see Figure 12-27). Because $\mathrm{O}_{2}^{2-}$ is so highly reactive and toxic, SOD and catalase are some of the fastest enzymes known so that they prevent the buildup of these ROS. SOD is found within mitochondria and other cellular compartments. Hydrogen peroxide itself is a ROS that can diffuse readily across membranes and react with molecules throughout the cell. It can also be converted by certain metals, such as $\mathrm{Fe}^{2+}$, into the even more dangerous hydroxyl radical $\left(\mathrm{OH}^{\bullet}\right)$. Thus cells depend on the inactivation of $\mathrm{H}_{2} \mathrm{O}_{2}$ by catalase and other enzymes, such as peroxiredoxin and glutathione peroxidase, which also detoxify the lipid hydroperoxide products formed when ROS react with unsaturated fatty acyl groups. Small-molecule antioxidant radical scavengers, such as vitamin E and $\alpha$-lipoic acid, also protect against oxidative stress. Although in many cells catalase is located only in peroxisomes, in heart muscle cells it is found in mitochondria. This is not surprising because the heart is the most oxygen-consuming organ per gram in mammals.

As the rate of ROS production by mitochondria and chloroplasts reflects the metabolic state of these organelles (e.g., strength of proton-motive force, $\mathrm{NADH} / \mathrm{NAD}^{+}$ratio), cells have developed ROS-sensing systems, such as ROS/ redox-sensitive transcription factors, to monitor the metabolic state of these organelles and respond accordingly-for example, by changing the rate of transcription of nuclear genes that encode organelle-specific proteins. There are also reports that $\mathrm{H}_{2} \mathrm{O}_{2}$ can function as a physiologically relevant intra- and intercellular signaling molecule. ROS have been reported to participate in cell processes as diverse as adaptation to low oxygen levels (hypoxia) and stress, growth factor and nutrient regulation of cell proliferation, cell differentiation, regulated cell death, and autophagy.

## Experiments Using Purified Electron-Transport Chain Complexes Established the Stoichiometry of Proton Pumping

The multiprotein complexes of the electron-transport chain that are responsible for proton pumping have been identified by selectively extracting mitochondrial membranes with detergents, isolating each of the complexes in nearly pure form, and then preparing artificial phospholipid vesicles (liposomes) containing each complex. When an appropriate
electron donor and electron acceptor are added to such liposomes, a change in the pH of the medium will occur if the embedded complex transports protons (Figure 12-28). Studies of this type indicate that NADH-CoQ reductase (complex I) translocates four protons per pair of electrons transported, whereas cytochrome $c$ oxidase (complex IV) translocates two protons per pair of electrons transported.

Current evidence suggests that a total of ten protons are transported from the matrix across the inner mitochondrial membrane for every electron pair that is transferred from
![li804542480e](li804542480e.jpg)

EXPERIMENTAL FIGURE 12-28 Electron transfer from reduced cytochrome $c$ to $\mathrm{O}_{2}$ via cytochrome $c$ oxidase (complex IV) is coupled to proton transport. The cytochrome $c$ oxidase complex is incorporated into liposomes with the binding site for cytochrome $c$ positioned on the outer surface. (a) When $\mathrm{O}_{2}$ and reduced cytochrome $c$ are added, electrons are transferred to $\mathrm{O}_{2}$ to form $\mathrm{H}_{2} \mathrm{O}$, and protons are transported from the inside to the medium outside the vesicles. A drug called valinomycin is added to the medium to dissipate the voltage gradient generated by the translocation of $\mathrm{H}^{+}$, which would otherwise reduce the number of protons moved across the membrane. (b) Monitoring of the medium's pH reveals a sharp drop in pH following addition of $\mathrm{O}_{2}$. As the reduced cytochrome $c$ becomes fully oxidized, protons leak back into the vesicles, and the pH of the medium returns to its initial value. Measurements show that two protons are transported per O atom reduced. Two electrons are needed to reduce one O atom, but cytochrome $c$ transfers only one electron; thus two molecules of cytochrome $c^{2+}$ are oxidized for each O reduced. See B. Reynafarje et al., 1986, J. Biol. Chem. 261:8254.

NADH to $\mathrm{O}_{2}$ (see Figure 12-22). Because succinate-CoQ reductase (complex II) does not transport protons, and because complex I is bypassed when the electrons come from succinate-derived $\mathrm{FADH}_{2}$, only six protons are transported across the membrane for every electron pair that is transferred from this $\mathrm{FADH}_{2}$ to $\mathrm{O}_{2}$.

## The Proton-Motive Force in Mitochondria Is Due Largely to a Voltage Gradient Across the Inner Membrane

The main result of the electron-transport chain is the generation of the proton-motive force, which is the sum of a transmembrane proton concentration ( pH ) gradient and an electric potential, or voltage gradient, across the inner mitochondrial membrane. The relative contributions of these two components to the total proton-motive force have been shown to depend on the permeability of the membrane to ions other than $\mathrm{H}^{+}$. A significant voltage gradient can develop only if the membrane is poorly permeable to other cations and to anions. Otherwise, anions would leak across the membrane from the matrix to the intermembrane space along with the protons and prevent a voltage gradient from forming. Similarly, if cations other than $\mathrm{H}^{+}$could leak across the membrane in a direction opposite to that of the $\mathrm{H}^{+}$(from the intermembrane space to the matrix), that leakage would counterbalance the charge delivered to the intermembrane space by the protons, short-circuiting voltage-gradient formation. Indeed, the inner mitochondrial membrane is poorly permeable to ions other than $\mathrm{H}^{+}$. Thus proton pumping generates a voltage gradient that makes it energetically difficult for additional protons to move across the membrane because of charge repulsion. As a consequence, proton pumping by the electron-transport chain establishes a robust voltage gradient in the context of what turns out to be a rather small pH gradient.

Because mitochondria are much too small to be impaled with electrodes, the electric potential and pH gradient across the inner mitochondrial membrane cannot be directly measured. However, the electric potential can be measured indirectly by adding radioactive ${ }^{42} \mathrm{~K}^{+}$ions and a trace amount of valinomycin to a suspension of respiring mitochondria and measuring the amount of radioactivity that accumulates in the matrix. Although the inner membrane is normally impermeable to $\mathrm{K}^{+}$, valinomycin is an ionophore, a small lipidsoluble molecule that selectively binds a specific ion (in this case, $\mathrm{K}^{+}$) and carries it across otherwise impermeable membranes. In the presence of valinomycin, ${ }^{42} \mathrm{~K}^{+}$equilibrates across the inner membrane of isolated mitochondria in accordance with the electric potential: the more negative the matrix side of the membrane, the more ${ }^{42} \mathrm{~K}^{+}$will be attracted to and accumulate in the matrix.

At equilibrium, the measured concentration of radioactive $\mathrm{K}^{+}$ions in the matrix, $\left[\mathrm{K}_{\mathrm{m}}\right]$, is about 500 times greater than that in the surrounding medium, $\left[\mathrm{K}_{\text {out }}\right]$. Substitution of this value into the Nernst equation (see Chapter 11) shows that the electric potential $E$ (in mV ) across the inner
membrane in respiring mitochondria is -160 mV , with the matrix (inside) negative:

$$
E=-59 \log \frac{\left[\mathrm{~K}_{\mathrm{m}}\right]}{\left[\mathrm{K}_{\text {out }}\right]}=-59 \log 500=-160 \mathrm{mV}
$$

Researchers can measure the matrix (inside) pH by trapping pH -sensitive fluorescent dyes inside vesicles formed from the inner mitochondrial membrane, with the matrix side of the membrane facing inward. They can also measure the pH outside the vesicles (equivalent to the intermembrane space) and thus determine the pH gradient ( $\Delta \mathrm{pH}$ ), which turns out to be about one pH unit. A difference of one pH unit represents a tenfold difference in $\mathrm{H}^{+}$concentration, so according to the Nernst equation, a pH gradient of one unit across a membrane is equivalent to an electric potential of 59 mV at $20^{\circ} \mathrm{C}$. Thus, knowing the voltage and pH gradients, we can calculate the proton-motive force (pmf) as

$$
\mathrm{pmf}=\Psi-\left(\frac{R T}{F} \times \Delta \mathrm{pH}\right)=\Psi-59 \Delta \mathrm{pH}
$$

where $R$ is the gas constant of $1.987 \mathrm{cal} /($ degree $\cdot \mathrm{mol}), T$ is the temperature (in degrees Kelvin), $F$ is the Faraday constant [23,062 cal/(V $\cdot$ mol)] and $\Psi$ is the transmembrane electric potential; $\Psi$ and pmf are measured in millivolts. The electric potential $\Psi$ across the inner membrane is -160 mV (negative inside), and $\Delta \mathrm{pH}$ is equivalent to about 60 mV . Thus the total proton-motive force is -220 mV , with the transmembrane electric potential responsible for about 73 percent of the total.

## KEY CONCEPTS OF SECTION 12.4

## The Electron-Transport Chain and Generation of the Proton-Motive Force

- By the end of the citric acid cycle (stage II), much of the energy originally present in the covalent bonds of glucose and fatty acids has been converted into high-energy electrons in the reduced coenzymes NADH and $\mathrm{FADH}_{2}$. The energy from these electrons is used to generate the proton-motive force.
- In the mitochondrion, the proton-motive force is generated by coupling electron flow (from NADH and $\mathrm{FADH}_{2}$ to $\mathrm{O}_{2}$ ) to the energetically uphill transport of protons from the matrix across the inner membrane to the intermembrane space. This process, together with the synthesis of ATP from ADP and P , driven by the proton-motive force, is called oxidative phosphorylation.
- As electrons flow from $\mathrm{FADH}_{2}$ and NADH to $\mathrm{O}_{2}$, they pass through multiprotein complexes. The four major complexes are NADH-CoQ reductase (complex I), succinateCoQ reductase (complex II), $\mathrm{CoQH}_{2}$-cytochrome $c$ reductase (complex III), and cytochrome $c$ oxidase (complex IV) (see Figure 12-22).

- Each complex contains one or more electron-carrying prosthetic groups, which include iron-sulfur clusters, flavins, heme groups, and copper ions (see Table 12-4). Cytochrome $c$, which contains heme, and coenzyme Q (CoQ), a lipidsoluble small molecule, are mobile carriers that shuttle electrons between the complexes.
- Complexes I, III, and IV pump protons from the matrix into the intermembrane space. Complexes I and II reduce CoQ to $\mathrm{CoQH}_{2}$, which carries protons and high-energy electrons to complex III. The heme protein cytochrome $c$ carries electrons from complex III to complex IV, which uses them to pump protons and reduce molecular oxygen to water.
- The high-energy electrons from NADH enter the electrontransport chain through complex I, whereas the high-energy electrons from $\mathrm{FADH}_{2}$ (derived from succinate in the citric acid cycle) enter the electron-transport chain through complex II. Additional electrons derived from $\mathrm{FADH}_{2}$ by the initial step of fatty acyl-CoA $\beta$-oxidation increase the supply of $\mathrm{CoQH}_{2}$ available for electron transport.
- The Q cycle allows four protons to be translocated per pair of electrons moving through complex III (see Figure 12-24).
- Each electron carrier in the chain accepts an electron or electron pair from a carrier with a less positive reduction potential and transfers the electron to a carrier with a more positive reduction potential. Thus the reduction potentials of electron carriers favor unidirectional, "downhill," electron flow from NADH and $\mathrm{FADH}_{2}$ to $\mathrm{O}_{2}$ (see Figure 12-25).
- Within the inner mitochondrial membrane, electrontransport complexes assemble into supercomplexes held together by cardiolipin, a specialized phospholipid. Supercomplex formation may enhance the speed and efficiency of generation of the proton-motive force or play other roles.
- Reactive oxygen species (ROS) are toxic by-products of the electron-transport chain that can modify and damage proteins, DNA, and lipids. Specific enzymes (e.g., glutathione peroxidase, catalase) and small-molecule antioxidants (e.g., vitamin E) help protect against ROS-induced damage (see Figure 12-27). ROS can also be used as intracellular signaling molecules.
- A total of $10 \mathrm{H}^{+}$ions are translocated from the matrix across the inner membrane per electron pair flowing from NADH to $\mathrm{O}_{2}$ (see Figure 12-22), whereas $6 \mathrm{H}^{+}$ions are translocated per electron pair flowing from $\mathrm{FADH}_{2}$ to $\mathrm{O}_{2}$.
- The proton-motive force is largely due to a voltage gradient across the inner membrane produced by proton pumping; the pH gradient plays a quantitatively less important role.


### 12.5 Harnessing the Proton-Motive Force to Synthesize ATP

The hypothesis that a proton-motive force across the inner mitochondrial membrane is the immediate source of energy for ATP synthesis was proposed in 1961 by Peter Mitchell. Virtually all
researchers studying oxidative phosphorylation and photosynthesis initially rejected his proposal (called the chemiosmotic hypothesis). They favored a mechanism similar to the then well-elucidated substrate-level phosphorylation in glycolysis, in which chemical transformation of a substrate molecule (like phosphoenolpyruvate in glycolysis) is directly coupled to ATP synthesis. Despite intense efforts by a large number of investigators, however, compelling evidence for such a substrate-level phosphorylation-mediated mechanism was never observed.

Definitive evidence supporting Mitchell's hypothesis depended on developing techniques to purify and reconstitute organelle membranes and membrane proteins. An experiment with vesicles made from chloroplast thylakoid membranes (equivalent to the inner membranes of mitochondria) containing ATP synthase, outlined in Figure 12-29, was one of several demonstrating that ATP synthase is an
![li814542480e](li814542480e.jpg)

EXPERIMENTAL FIGURE 12-29 Synthesis of ATP by ATP synthase depends on a pH gradient across the membrane. Isolated chloroplast thylakoid vesicles containing ATP synthase ( $\mathrm{F}_{\mathrm{II}} \mathrm{F}_{1}$ particles) were equilibrated in the dark with a buffered solution at pH 4.0 . When the pH in the thylakoid lumen reached 4.0 , the vesicles were rapidly mixed with a solution at pH 8.0 containing ADP and $\mathrm{P}_{\mathrm{i}}$. A burst of ATP synthesis accompanied the transmembrane movement of protons driven by the 10,000 -fold $\mathrm{H}^{+}$concentration gradient $\left(10^{-4} \mathrm{M}\right.$ versus $\left.10^{-8} \mathrm{M}\right)$. In similar experiments using "inside-out" preparations of mitochondrial membrane vesicles, an artificially generated membrane electric potential also resulted in ATP synthesis.

ATP-generating enzyme and that ATP generation is dependent on proton movement down an electrochemical gradient. It turns out that the protons actually move through ATP synthase as they traverse the membrane.

As we shall see, ATP synthase is a multiprotein complex that can be subdivided into two subcomplexes, called $\mathrm{F}_{0}$ (containing the transmembrane portions of the complex) and $\mathrm{F}_{1}$ (containing the globular portions of the complex that sit above the membrane and point into the matrix in mitochondria). Thus ATP synthase is often called the $\mathrm{F}_{0} \mathrm{~F}_{1}$ complex; we will use the two terms interchangeably.

## The Mechanism of ATP Synthesis Is Shared Among Bacteria, Mitochondria, and Chloroplasts

Although bacteria lack internal membranes, aerobic bacteria nonetheless carry out oxidative phosphorylation by the same processes that occur in eukaryotic mitochondria and chloroplasts (Figure 12-30). Enzymes that catalyze the reactions of both the glycolytic pathway and the citric acid cycle are present in the cytosol of bacteria; enzymes that oxidize NADH to $\mathrm{NAD}^{+}$and transfer the electrons to the ultimate acceptor $\mathrm{O}_{2}$ reside in the bacterial plasma membrane. The movement of electrons through these membrane carriers is coupled to the pumping of protons out of the cell. The movement of protons back into the cell, down their concentration gradient through ATP synthase, drives the synthesis of ATP. The bacterial ATP synthase ( $\mathrm{F}_{0} \mathrm{~F}_{1}$ complex) is essentially identical in structure and function to the mitochondrial and chloroplast ATP synthases, but is simpler to purify and study.

Why is the mechanism of ATP synthesis shared among both prokaryotic organisms and eukaryotic organelles? Primitive aerobic bacteria were probably the progenitors of both mitochondria and chloroplasts in eukaryotic cells (see Figure 12-7). According to this endosymbiont hypothesis, the inner mitochondrial membrane was derived from the bacterial plasma membrane, with its cytosolic face pointing toward what became the matrix of the mitochondrion. Similarly, in plants, the progenitor bacterium's plasma membrane became the chloroplast's thylakoid membrane, and its cytosolic face pointed toward what became the stromal space of the chloroplast (chloroplast structure will be described in Section 12.6). In all cases, ATP synthase is positioned with the globular $\mathrm{F}_{1}$ domain, which catalyzes ATP synthesis, on the cytosolic face of the membrane, so ATP is always formed on the cytosolic face (see Figure 12-30). Protons always flow through ATP synthase from the exoplasmic to the cytosolic face of the membrane. This flow is driven by the proton-motive force. Invariably, the cytosolic face has a negative electric potential relative to the exoplasmic face.

In addition to ATP synthesis, the proton-motive force across the bacterial plasma membrane is used to power other processes, including the uptake of nutrients such as sugars (using proton/sugar symporters) and the rotation of bacterial flagella. Chemiosmotic coupling thus illustrates an important principle introduced in our discussion of active transport in Chapter 11: the membrane potential, the concentration gradients of protons (and other ions) across a membrane, and the phosphoanhydride
![li824542480e](li824542480e.jpg)

FIGURE 12-30 ATP synthesis by chemiosmosis is similar in bacteria, mitochondria, and chloroplasts. In chemiosmosis, a protonmotive force generated by proton pumping across a membrane is used to power ATP synthesis. The mechanism and membrane orientation of the process are similar in bacteria, mitochondria, and chloroplasts. In each illustration, the membrane surface facing a shaded area is a cytosolic face; the surface facing an unshaded, white area is an exoplasmic face. Note that the cytosolic face of the bacterial plasma membrane, the matrix face of the inner mitochondrial membrane, and the stromal face of the thylakoid membrane are all equivalent. During electron transport, protons are always pumped from the cytosolic face to the exoplasmic face, creating a proton concentration gradient (exoplasmic face $>$ cytosolic face) and an electric potential (negative cytosolic face and positive exoplasmic face) across the membrane. During the synthesis of ATP, protons flow in the reverse direction (down their electrochemical gradient) through ATP synthase ( $\mathrm{F}_{0} \mathrm{~F}_{1}$ complex), which protrudes in a knob at the cytosolic face in all cases.

bonds in ATP are equivalent and interconvertible forms of potential energy. Indeed, ATP synthesis through ATP synthase can be thought of as active transport in reverse.

## ATP Synthase Comprises $F_{0}$ and $F_{1}$ Multiprotein Complexes

With general acceptance of Mitchell's chemiosmotic mechanism, researchers turned their attention to the structure and operation of ATP synthase. The complex has two principal subcomplexes, $\mathrm{F}_{0}$ and $\mathrm{F}_{1}$, both of which are multimeric
![li834542480e](li834542480e.jpg)

FIGURE 12-31 Structure of ATP synthase (the $\mathrm{F}_{0} \mathrm{~F}_{1}$ complex) in the bacterial plasma membrane and mechanism of proton translocation across the membrane. (a) The $F_{0}$ membrane-embedded subcomplex of ATP synthase is built of three integral membrane proteins: one copy of a, two copies of $\mathbf{b}$, and an average of ten copies of $\mathbf{c}$ arranged in a ring in the plane of the membrane. Two proton half-channels near the interfaces of subunit a with the c subunits mediate proton movement across the membrane (proton path is indicated by red arrows). Half-channel I allows protons to move one at a time from the exoplasmic medium (equivalent to intermembrane space in mitochondria) to the negatively charged side chain of Asp-61 in the center of a c subunit near the middle of the membrane. The proton-binding site in each $\mathbf{c}$ subunit is represented as a white circle with a blue "-" representing the negative charge on the side chain of Asp-61. Half-channel II permits protons to move from the Asp-61 of an adjacent c subunit into the cytosolic medium. The detailed structure of the $\mathbf{c}$ ring and a portion of the adjacent $\mathbf{a}$ subunit is shown in Figure 12-34. The $F_{1}$ subcomplex of ATP synthase contains three copies each of subunits $\alpha$ and $\beta$, which form a hexamer resting atop the single rod-shaped $\gamma$ subunit, which is inserted into the $\mathbf{c}$ ring of $F_{0}$. The $\varepsilon$ subunit is rigidly attached to the $\gamma$ subunit and also to several of the $\mathbf{c}$ subunits. The $\delta$ subunit permanently links one of the $\alpha$ subunits
proteins (Figure 12-31a). The $\mathrm{F}_{0}$ subcomplex contains three types of integral membrane proteins, designated $\mathbf{a}, \mathbf{b}$, and $\mathbf{c}$. In bacteria and in yeast mitochondria, the most common subunit stoichiometry is $\mathbf{a}_{1} \mathbf{b}_{2} \mathbf{c}_{10}$, but the number of $\mathbf{c}$ subunits per subcomplex varies among different eukaryotes from 8 to 15 . In all cases, the c subunits form a doughnut-shaped ring ("c ring") in the plane of the membrane. The one $\mathbf{a}$ and two $\mathbf{b}$ subunits are rigidly linked to one another, but not to the c ring, a critical feature of the protein to which we will return shortly.

The $F_{1}$ subcomplex is a water-soluble complex of five distinct polypeptides with the composition $\alpha_{3} \beta_{3} \gamma \delta \varepsilon$ that is
![li844542480e](li844542480e.jpg)
in the $F_{1}$ subcomplex to the $\mathbf{b}$ subunit of $F_{0}$. Thus the $F_{0} \mathbf{a}$ and $\mathbf{b}$ subunits and the $F_{1} \delta$ subunit and $(\alpha \beta)_{3}$ hexamer form a rigid structure (orange) anchored in the membrane. During proton flow, the $\mathbf{c}$ ring and the attached $F_{1} \varepsilon$ and $\gamma$ subunits rotate as a unit (green), causing conformational changes in the $F_{1} \beta$ subunits, leading to ATP synthesis. (b) Potential mechanism of proton translocation. Step A proton from the exoplasmic space enters half-channel I and moves toward the "empty" (unprotonated) Asp-61 proton-binding site. The negative charge (blue " - ") on the unprotonated side chain Asp-61 is balanced, in part, by a positive charge on the side chain of Arg-210 (red " + "). Step The proton fills the empty proton-binding site and simultaneously displaces the positively charged Arg-210 side chain, which swings over to the filled proton-binding site on the adjacent $\mathbf{c}$ subunit (curved arrow). As a consequence, the proton bound at that adjacent site is displaced. Step The displaced adjacent proton moves through half-channel II and is released into the cytosolic space, leaving an empty proton-binding site on Asp-61. Step Counterclockwise rotation of the entire c ring moves the "empty" c subunit over half-channel I. Step The process is repeated. See M. J. Schnitzer, 2001, Nature 410:878; P. D. Boyer, 1999, Nature 402:247; and C. von Ballmoos, A. Wiedenmann, and P. Dimroth, 2009, Annu. Rev. Biochem. 78:649.

normally firmly bound to the $\mathrm{F}_{0}$ subcomplex at the surface of the membrane. The lower end of the rodlike $\gamma$ subunit of the $\mathrm{F}_{1}$ subcomplex is a coiled coil that fits into the center of the c -subunit ring of $\mathrm{F}_{0}$ and appears rigidly attached to it. Thus when the c-subunit ring rotates, the rodlike $\gamma$ subunit moves with it. The $F_{1} \varepsilon$ subunit is rigidly attached to $\gamma$ and also forms tight contacts with several of the c subunits of $\mathrm{F}_{0}$. The $\alpha$ and $\beta$ subunits are responsible for the overall globular shape of the $\mathrm{F}_{1}$ subcomplex and associate in alternating order to form a hexamer, $\alpha \beta \alpha \beta \alpha \beta$, or $(\alpha \beta)_{3}$, which rests atop the single long $\gamma$ subunit. The $F_{1} \delta$ subunit is permanently linked to one of the $\mathrm{F}_{1} \alpha$ subunits and also binds to the $\mathbf{b}$ subunit of $\mathrm{F}_{0}$. Thus the $\mathbf{a}$ and $\mathbf{b}$ subunits of the $\mathrm{F}_{0}$ subcomplex and the $\delta$ subunit and $(\alpha \beta)_{3}$ hexamer of the $\mathrm{F}_{1}$ subcomplex form a rigid structure anchored in the membrane. The rodlike $\mathbf{b}$ subunits form a "stator" that prevents the $(\alpha \beta)_{3}$ hexamer from moving while it rests on the $\gamma$ subunit, whose rotation, together with that of the c subunits of $\mathrm{F}_{0}$, plays an essential role in the ATP synthesis mechanism described below.

When ATP synthase is embedded in a membrane, the $\mathrm{F}_{1}$ subcomplex forms a knob that protrudes from the cytosolic face (the matrix face in the mitochondrion). Because $\mathrm{F}_{1}$ separated from membranes is capable of catalyzing ATP hydrolysis (ATP conversion to ADP plus $\mathrm{P}_{\mathrm{i}}$ ) in the absence of the $\mathrm{F}_{0}$ subcomplex, it has been called the $\mathrm{F}_{1}$ ATPase; however, its function in cells is the reverse, to synthesize ATP. ATP hydrolysis is a spontaneous process $(\Delta G<0)$; thus energy is required to drive the ATPase "in reverse" and generate ATP.

## Rotation of the $\mathrm{F}_{1} \gamma$ Subunit, Driven by Proton Movement Through $\mathrm{F}_{0}$, Powers ATP Synthesis

Each of the three $\beta$ subunits in the globular $F_{1}$ subcomplex of $\mathrm{F}_{0} \mathrm{~F}_{1}$ can bind ADP and $\mathrm{P}_{\mathrm{i}}$ and catalyze the endergonic synthesis of ATP when coupled to the flow of protons from the exoplasmic medium (the intermembrane space in the mitochondrion) to the cytosolic (matrix) medium. However, the energetic coupling of proton flow and ATP synthesis does not take place in the same portions of the protein, because the nucleotide-binding sites on the $\beta$ subunits of $\mathrm{F}_{1}$, where ATP synthesis occurs, are $9-10 \mathrm{~nm}$ from the surface of the membrane-embedded portion of $\mathrm{F}_{0}$ through which the protons flow. The most widely accepted model for ATP synthesis by the $\mathrm{F}_{0} \mathrm{~F}_{1}$ complex-the binding-change mechanism -posits an indirect coupling (Figure 12-32).

According to this mechanism, energy released by the "downhill" movement of protons through $\mathrm{F}_{0}$ directly powers rotation of the c -subunit ring together with its attached $\gamma$ and $\varepsilon$ subunits (see Figure 12-31a). The $\gamma$ subunit acts as a cam, or nonsymmetrical rotating shaft, whose c ring-driven rotation within the center of the static $(\alpha \beta)_{3}$ hexamer of $\mathrm{F}_{1}$ causes it to push sequentially against each of the $\beta$ subunits and thus cause cyclical changes in their conformations between three different states. As schematically depicted in a view of the bottom of the $(\alpha \beta)_{3}$ hexamer's globular structure in Figure 12-32, rotation of the $\gamma$ subunit relative to the fixed $(\alpha \beta)_{3}$ hexamer causes the nucleotide-binding site of each $\beta$ subunit to cycle through three conformational states in the following order:

1. An O (open) state that binds ATP very poorly and ADP and $\mathrm{P}_{\mathrm{i}}$ weakly
2. An L (loose) state that binds ADP and $\mathrm{P}_{\mathrm{i}}$ more strongly but cannot bind ATP
3. A T (tight) state that binds ADP and $\mathrm{P}_{\mathrm{i}}$ so tightly that they spontaneously react and form ATP
In the T state, the ATP produced is bound so tightly that it cannot readily dissociate from the site-it is trapped until another rotation of the $\gamma$ subunit returns that $\beta$ subunit to the O state, thereby releasing ATP and beginning the cycle again. ATP or ADP also binds to regulatory or allosteric sites on the three $\alpha$ subunits; this binding modifies the rate of ATP synthesis according to the level of ATP and ADP in the matrix, but is not directly involved in the catalytic step that synthesizes ATP from ADP and $\mathrm{P}_{\mathrm{i}}$.

Several types of evidence support the binding-change mechanism. First, biochemical studies showed that on isolated $\mathrm{F}_{1}$ particles, one of the three $\beta$ subunits can tightly bind ADP and $\mathrm{P}_{\mathrm{i}}$ and then form ATP, which remains tightly bound. The measured $\Delta G$ for this reaction is near zero, indicating that once ADP and $\mathrm{P}_{\mathrm{i}}$ are bound to the T state of a $\beta$ subunit, they spontaneously form ATP. Importantly, dissociation of the bound ATP from the $\beta$ subunit on isolated $\mathrm{F}_{1}$ particles occurs extremely slowly. This finding suggested that dissociation of ATP would have to be powered by a conformational change in the $\beta$ subunit, which in turn would be due to c ring rotation caused by proton movement.

X-ray crystallographic analysis of the $(\alpha \beta)_{3}$ hexamer yielded a striking conclusion: although the three $\beta$ subunits are identical in sequence and overall structure, the ADP/ ATP-binding sites have different conformations in each subunit. The most reasonable conclusion was that the three $\beta$ subunits cycle in an energy-dependent reaction between three conformational states ( $\mathrm{O}, \mathrm{L}, \mathrm{T}$ ), in which the nucleo-tide-binding site has substantially different structures.

In other studies, intact $\mathrm{F}_{0} \mathrm{~F}_{1}$ complexes were treated with chemical cross-linking agents that covalently linked the $\gamma$ and $\varepsilon$ subunits and the c -subunit ring. The observation that such treated complexes could synthesize ATP or use ATP to power proton pumping indicates that the cross-linked proteins normally rotate together.

Finally, rotation of the $\gamma$ subunit relative to the fixed $(\alpha \beta)_{3}$ hexamer, as proposed in the binding-change mechanism, was observed directly in the clever experiment depicted in Figure 12-33. In one modification of this experiment in which tiny gold particles, rather than an actin filament, were attached to the $\gamma$ subunit, rotation rates of 134 revolutions per second were observed. Hydrolysis of three ATPs, which you recall is the reverse reaction catalyzed by the same enzyme, is thought to power one revolution; this result is close to the experimentally determined rate of ATP hydrolysis by $\mathrm{F}_{0} \mathrm{~F}_{1}$ complexes: about 400 ATPs per second. In a related experiment, a $\gamma$ subunit linked to an $\varepsilon$ subunit and a ring of c subunits was seen to rotate relative to the fixed $(\alpha \beta)_{3}$ hexamer. Rotation of the $\gamma$ subunit in these experiments was powered by ATP hydrolysis. These observations established that the $\gamma$ subunit, along with the attached

![li854542480e](li854542480e.jpg)

FIGURE 12-32 The binding-change mechanism of ATP synthesis from ADP and $P_{i}$. This view is looking up at $F_{1}$ from the membrane surface (see Figure 12-31). As the $\gamma$ subunit rotates by $120^{\circ}$ in the center, each of the otherwise identical $F_{1} \beta$ subunits alternates between three conformational states ( O , open, with oval representation of the binding site; L, loose, with a rectangular binding site; T, tight, with a triangular site) that differ in their binding affinities for ATP, ADP, and $P_{i}$. The cycle begins (upper left) when ADP and $P_{i}$ bind loosely to one of the three $\beta$ subunits (here, arbitrarily designated $\beta_{1}$ ) whose nucleotide-binding site is in the O (open) conformation. Proton flux through the $F_{0}$ portion of the protein powers a $120^{\circ}$ rotation of the $\gamma$ subunit (relative to the fixed $\beta$ subunits) (step $\mathbf{\square}$ ). This causes the rotating $\gamma$ subunit, which is asymmetric, to push differentially against the $\beta$ subunits, resulting in a conformational change and an increase in the binding affinity of the $\beta_{1}$ subunit for ADP and $P_{i}(O \rightarrow L)$, an increase in the binding affinity of the $\beta_{2}$ subunit for ADP and $P_{i}$ that were previously bound ( $L \rightarrow T$ ),
c ring and $\varepsilon$ subunit, does indeed rotate, thereby driving the conformational changes in the $\beta$ subunits that are required for the binding of ADP and $P_{i}$, followed by synthesis and subsequent release of ATP.
![li864542480e](li864542480e.jpg)
and a decrease in the binding affinity of the $\beta_{2}$ subunit for a previously bound ATP ( $\mathrm{T} \rightarrow \mathrm{O}$ ), causing release of the bound ATP. Step $\boldsymbol{\square}$. Without additional rotation, the ADP and $P_{i}$ in the $T$ site (here, in the $\beta_{2}$ subunit) form ATP, a reaction that does not require an input of additional energy due to the special environment in the active site of the $T$ state. At the same time, a new ADP and $P_{i}$ bind loosely to the unoccupied $O$ site on $\beta_{2}$. Step $\boldsymbol{\square}$. Proton flux powers another $120^{\circ}$ rotation of the $\gamma$ subunit, consequent conformational changes in the binding sites ( $L \rightarrow T$, $\mathrm{O} \rightarrow \mathrm{L}, \mathrm{T} \rightarrow \mathrm{O}$ ), and release of ATP from $\beta_{2}$. Step $\boldsymbol{\square}$. Without additional rotation, the ADP and $P_{i}$ in the $T$ site of $\beta_{1}$ form ATP, and additional ADP and $P_{i}$ bind to the unoccupied $O$ site on $\beta_{2}$. The process continues with rotation (step $\boldsymbol{\square}$ ) and ATP formation (step $\boldsymbol{\square}$ ) until the cycle is complete, with three ATPs having been produced for every $360^{\circ}$ rotation of $\gamma$. See P. Boyer, 1989, FASEB J. 3:2164; Y. Zhou et al., 1997, Proc. Natl. Acad. Sci. USA 94:10583; and M. Yoshida, E. Muneyuki, and T. Hisabori, 2001, Nat. Rev. Mol. Cell Biol. 2:669.

## Multiple Protons Must Pass Through ATP Synthase to Synthesize One ATP

A simple calculation indicates that the passage of more than one proton is required to synthesize one molecule of ATP from ADP and $P_{i}$. Although the $\Delta G$ for this reaction under standard conditions is $+7.3 \mathrm{kcal} / \mathrm{mol}$, at the concentrations of reactants in the mitochondrion, $\Delta G$ is probably higher ( +10 to $+12 \mathrm{kcal} / \mathrm{mol}$ ). We can calculate the amount of free energy released by the passage of 1 mol of protons down an

## EXPERIMENTAL FIGURE 12-33 The $\gamma$ subunit of the $F_{1}$ subcomplex rotates relative to the $(\alpha \beta)_{1}$ hexamer. $F_{1}$ subcomplexes were engineered to contain $\beta$ subunits with an additional His-6 sequence, which causes them to adhere to a glass plate coated with a metal reagent that binds polyhistidine. The $\gamma$ subunit in the engineered $F_{1}$ subcomplexes was linked covalently to a fluorescently labeled actin filament. When viewed in a fluorescence microscope, the actin filament was seen to rotate counterclockwise in discrete $120^{\circ}$ steps in the presence of ATP due to ATP hydrolysis by the $\beta$ subunits. See H. Noji et al., 1997, Nature 386:299, and R. Yasuda et al., 1998, Cell 93:1117.

electrochemical gradient of $220 \mathrm{mV}(0.22 \mathrm{~V})$ from the Nernst equation, setting $n=1$ and measuring $\Delta E$ in volts:

$$
\begin{aligned}
\Delta G(\mathrm{cal} / \mathrm{mol}) & =-n F \Delta E=-\left(23,062 \mathrm{cal} \cdot \mathrm{~V}^{-1} \cdot \mathrm{~mol}^{-1}\right) \Delta E \\
& =\left(23,062 \mathrm{cal} \cdot \mathrm{~V}^{-1} \cdot \mathrm{~mol}^{-1}\right)(0.22 \mathrm{~V}) \\
& =-5074 \mathrm{cal} / \mathrm{mol}, \text { or }-5.1 \mathrm{kcal} / \mathrm{mol}
\end{aligned}
$$

Because the downhill movement of 1 mol of protons releases just over 5 kcal of free energy, the passage of at least two protons is required for synthesis of each molecule of ATP from ADP and $\mathrm{P}_{\mathrm{i}}$.

## $\mathrm{F}_{0} \mathrm{c}$ Ring Rotation Is Driven by Protons Flowing Through Transmembrane Channels

Each copy of the c subunit contains two membrane-spanning $\alpha$ helices that form a hairpin-like structure. An aspartate residue, Asp-61 (E. coli ATPase numbering), in the center of one of these helices in each c subunit is thought to play a key role in proton movement by binding and releasing protons as they traverse the membrane. Chemical modification of this aspartate by the poison dicyclohexylcarbodiimide, or its mutation to alanine, specifically blocks proton movement through $\mathrm{F}_{0}$. According to one current model, the protons traverse the membrane via two staggered half-channels, I and II (see Figure 12-31a and b). They are called half-channels because each extends only halfway across the membrane; the intramembrane termini of the channels are at the level of Asp-61 in the middle of the membrane. Half-channel I is open only to the exoplasmic face, and half-channel II is open only to the cytosolic face. Prior to rotation, each of the Asp-61 carboxylate side chains in the c subunits is bound to a proton, except that on the c subunit in contact with half-channel I. The negative charge on that unprotonated carboxylate (the "empty" proton-binding site; see Figure 12-31b, bottom) is neutralized by interaction with the positively charged side chain of Arg-210 from the a subunit. Proton translocation across the membrane begins when a proton from the exoplasmic medium moves upward through half-channel I (Figure 12-31b, step $\mathbf{\square}$ ). As that proton moves into the empty proton-binding site, it displaces the Arg-210 side chain, which swings toward the filled protonbinding site of the adjacent c subunit in contact with halfchannel II (step $\mathbf{\square}$ ). As a consequence, the positive side chain of Arg-210 displaces the proton bound to Asp-61 of the adjacent c subunit. This displaced proton is now free to travel up half-channel II and out into the cytosolic medium (step $\mathbf{\square}$ ). Thus when one proton entering from half-channel I binds to the c ring, a different proton is released to the opposite side of the membrane via half-channel II. Rotation of the entire c ring due to thermal/Brownian motion (step $\mathbf{\square}$ ) then allows the newly unprotonated c subunit to move into alignment above half-channel I as an adjacent, protonated c subunit rotates in to take its place under half-channel II. The entire cycle is then repeated (step $\mathbf{\square}$ ) as additional protons move down their electrochemical gradient from the exoplasmic medium to the cytosolic medium. During each partial rotation ( $360^{\circ}$ divided by the number of c subunits in the ring), the c ring rotation is ratcheted, in that net movement of the ring occurs in only one direction. The energy driving the protons across the membrane, and thus the rotation of the c ring, comes from the electrochemical gradient across the membrane. If the direction of proton flow is reversed, which can be done by experimentally reversing the direction of the proton gradient and the protonmotive force, the direction of c ring rotation is reversed.

Because the $\gamma$ subunit of $\mathrm{F}_{1}$ is tightly attached to the c ring of $F_{0}$, rotation of the $c$ ring associated with proton movement causes rotation of the $\gamma$ subunit. According to the bindingchange mechanism, a $120^{\circ}$ rotation of $\gamma$ powers synthesis of one ATP (see Figure 12-32). Thus complete rotation of the c ring by $360^{\circ}$ would generate three ATPs. In E. coli, where the $\mathrm{F}_{0}$ composition is $\mathrm{a}_{1} \mathrm{~b}_{2} \mathrm{c}_{10}$, movement of 10 protons drives one complete rotation and thus synthesis of three ATPs. This value is consistent with experimental data on proton flux during ATP synthesis, providing indirect support for the model coupling proton movement to c ring rotation depicted in Figure 12-31. The $F_{0}$ from chloroplasts contains 14 c subunits per ring, and movement of 14 protons would be needed for synthesis of three ATPs. Why these otherwise similar $\mathrm{F}_{0} \mathrm{~F}_{1}$ complexes have evolved to have different $\mathrm{H}^{+}$:ATP ratios is not clear.

High-resolution electron microscopic tomography (Figure 12-34) has provided additional insights into the structure of the c ring/a subunit interface and other features of $\mathrm{F}_{0} \mathrm{~F}_{1}$ structure and function. The experiments were performed using $\mathrm{F}_{0} \mathrm{~F}_{1}$ either dissolved in detergent, then incorporated into artificial phospholipid bilayers, or in isolated mitochondrial membranes. Figures 12-34a and b show two views of the two membrane-spanning $\alpha$ helices in each copy of the c subunit (green) that together form the c ring. In a portion of the a subunit (orange), a bundle of four $\alpha$ helices that are almost parallel to and embedded within the inner mitochondrial membrane forms the interface with the c ring and positions the side chain of Arg 210 adjacent to the c ring so that it can mediate proton displacement from Asp 61 as shown in Figure 12-31. The c ring/a subunit interface also forms the two proton half-channels through which protons flow out of the intermembrane space (red arrow), around the c ring (black arrows in Figure 12-34b), and then out into the matrix (red arrow). Each $\mathrm{F}_{0} \mathrm{~F}_{1}$ monomer bends the membrane by approximately $43^{\circ}$ (Figure 12-34c). The monomers dimerize to impart high membrane curvature $\left(\sim 86^{\circ}\right)$ and then align in long rows, contributing to the formation of the edges and tips of the pancake-like (flat) and tubular cristae (Figure 12-34d).

## ATP-ADP Exchange Across the Inner Mitochondrial Membrane Is Powered by the Proton-Motive Force

The proton-motive force is used to power multiple energy-requiring processes in cells. In addition to powering ATP synthesis, the proton-motive force across the inner mitochondrial membrane powers the exchange of ATP formed by oxidative phosphorylation inside the mitochondrion for ADP and $\mathrm{P}_{\mathrm{i}}$ in the cytosol. This exchange, which is required to supply ADP and $\mathrm{P}_{\mathrm{i}}$ substrates for

![li874542480e](li874542480e.jpg)
![li884542480e](li884542480e.jpg)

EXPERIMENTAL FIGURE 12-34 High-resolution electron microscopy-based mechanism of proton translocation and bending of cristae membranes by ATP synthase. (a) and (b) The interface between the $\mathbf{c}$ ring (green) and a subunit (orange) of detergent-solubilized mitochondrial ATP synthase from the alga Polytomella sp., imaged by single-particle cryoelectron microscopy ( $\sim 0.62 \mathrm{~nm}$ resolution), is shown (a) from within the plane of the inner mitochondrial membrane (side view) and (b) after a $90^{\circ}$ rotation (top view). The movement of protons through half-channels I and II and the rotation of the c ring are described in detail in Figure 12-31. (a) Cross section through the c ring (right) shows that each $\boldsymbol{c}$ subunit is a transmembrane helical hairpin - two adjacent transmembrane $\alpha$ helices connected by a short nonhelical linker on the matrix side of the membrane. The negative side chain of the $\mathbf{c}$ subunit's Asp61 in the middle of the membrane is thought to both serve as a binding site for translocating protons and interact with the side chain of the a subunit's Arg210. (c) A model of the bovine heart mitochondrial ATP synthase is based on cryoelectron tomography and
electron crystallographic image processing from crystalline ATP synthase in artificial membranes. Each $\mathrm{F}_{0} \mathrm{~F}_{1}$ monomer bends the membrane by $\sim 43^{\circ}$ toward the intermembrane space (IMS), resulting in dimers bending the membrane by $\sim 86^{\circ}$. The rotating c ring and $\gamma$ and $\varepsilon$ subinits are colored green, and the remaining static portions of the enzyme are shown in orange. (d) Cryoelectron tomographic image of frozen membranes from purified Saccharomyces cerevisiae (yeast) mitochondria. The surfaces of the ATP synthase complexes (orange) and the membrane (gray) show that the enzymes dimerize as in (c) and align into long rows that bend the membranes into characteristic tubular and flat, pancakelike cristae. [Parts (a) and (b) reprinted by permission from Macmillan Publishers Ltd., from Allegretti, M., et al., "Horizontal membrane-intrinsic $\alpha$-helices in the stator a-subunit of an F-type ATP synthase," Nature, 2015, 521, pp 237-240, 2015; permission conveyed through the Copyright Clearance Center, Inc. Part (c) data from C. Jiko et al., 2015, eLife 4:e06119. Part (d) from Proc. Natl. Acad. Sci. USA 2012. 109(34):13602-13607, Fig. 4C and D. "Structure of the yeast F1Fo-ATP synthase dimer and its role in shaping the mitochondrial cristae."]

oxidative phosphorylation to continue, is mediated by two proteins in the inner membrane: a phosphate transporter $\left(\mathrm{HPO}_{4}{ }^{2-}\right)$ $\mathrm{OH}^{-}$antiporter), which mediates the import of one $\mathrm{HPO}_{4}{ }^{2-}$ coupled to the export of one $\mathrm{OH}^{-}$, and an ATP/ADP antiporter (Figure 12-35). The ATP/ADP antiporter allows one molecule of ADP to enter the matrix only if one molecule of ATP exits simultaneously. The ATP/ADP antiporter, a dimer of two 30,000Da subunits, makes up $10-15$ percent of the protein in the inner mitochondrial membrane, so it is one of the more abundant mitochondrial proteins. The functioning of the two antiporters together produces an influx of one $\mathrm{ADP}^{3-}$ and one $\mathrm{P}_{\mathrm{i}}{ }^{2-}$ and an efflux of one $\mathrm{ATP}^{4-}$ together with one $\mathrm{OH}^{-}$. Each $\mathrm{OH}^{-}$transported outward combines with a proton, translocated during electron transport to the intermembrane space, to form $\mathrm{H}_{2} \mathrm{O}$. Thus proton translocation via electron transport drives the overall reaction in the direction of ATP export and ADP and $\mathrm{P}_{\mathrm{i}}$ import.

Because some of the protons translocated out of the mitochondrion during electron transport provide the power (by combining with the exported $\mathrm{OH}^{-}$) for the ATP-ADP exchange, fewer protons are available for ATP synthesis. It is estimated that for every four protons translocated out, three are used to synthesize one ATP molecule and one is used to power the export of ATP from the mitochondrion in exchange for ADP and $\mathrm{P}_{\mathrm{i}}$. This expenditure of energy from the proton concentration
![li894542480e](li894542480e.jpg)

FIGURE 12-35 The phosphate and ATP/ADP transport system in the inner mitochondrial membrane. The coordinated action of two antiporters (purple and green), which results in the uptake of one $\mathrm{ADP}^{3-}$ and one $\mathrm{HPO}_{4}{ }^{2-}$ in exchange for one $\mathrm{ATP}^{4-}$ and one hydroxyl, is powered by the outward translocation of one proton (mediated by the proteins of the electron-transport chain, blue) during electron transport. The outer membrane is not shown here because it is permeable to molecules smaller than 5000 Da .
gradient to export ATP from the mitochondrion in exchange for ADP and $\mathrm{P}_{\mathrm{i}}$ ensures a high ratio of ATP to ADP in the cytosol, where hydrolysis of the high-energy phosphoanhydride bond of ATP is used to power many energy-requiring reactions.

Studies of what turned out to be ATP/ADP antiporter activity were first recorded about 2000 years ago, when Dioscorides ( -AD 40-90) described the effects of a poisonous herb from the thistle Atractylis gummifera, found commonly in the Mediterranean region. The same agent is found in the traditional Zulu multipurpose herbal remedy impila (Callilepis laureola). In Zulu, impila means "health," although this herb has been associated with numerous poisonings. In 1962, the active agent in the herbs, atractyloside, which inhibits the ATP/ADP antiporter, was shown to inhibit oxidative phosphorylation of extramitochondrial ADP, but not intramitochondrial ADP. This finding demonstrated the importance of the ATP/ADP antiporter and has provided a powerful tool to study the mechanism by which this transporter functions.

Dioscorides lived near Tarsus, at the time a province of Rome in southeastern Asia Minor, in what is now Turkey. His five-volume De Materia Medica (The Materials of Medicine) "on the preparation, properties, and testing of drugs" described the medicinal properties of about 1000 natural products and 4740 medicinal usages of them. For approximately 1600 years, it was the basic reference in medicine from northern Europe to the Indian Ocean, comparable to today's Physicians' Desk Reference as a guide for using drugs.

## The Rate of Mitochondrial Oxidation Normally Depends on ADP Levels

If intact isolated mitochondria are provided with NADH (or a source of $\mathrm{FADH}_{2}$ such as succinate) plus $\mathrm{O}_{2}$ and $\mathrm{P}_{\mathrm{i}}$, but not ADP, the oxidation of NADH and the reduction of $\mathrm{O}_{2}$ rapidly cease as the amount of endogenous ADP is depleted by ATP formation. If ADP is then added, the oxidation of NADH is rapidly restored. Thus mitochondria can oxidize $\mathrm{FADH}_{2}$ and NADH only as long as there is a source of ADP and $\mathrm{P}_{\mathrm{i}}$ to generate ATP. This phenomenon, termed respiratory control, occurs because oxidation of NADH and succinate $\left(\mathrm{FADH}_{2}\right)$ is obligatorily coupled to proton transport across the inner mitochondrial membrane. If the resulting proton-motive force is not dissipated during the synthesis of ATP from ADP and P , (or during other energy-requiring processes), both the transmembrane proton concentration gradient and the membrane electric potential will increase to very high levels. At this point, pumping of additional protons across the inner membrane requires so much energy that it eventually ceases, blocking the coupled oxidation of NADH and other substrates.

## Mitochondria in Brown Fat Use the ProtonMotive Force to Generate Heat

Brown-fat tissue, whose color is due to the presence of abundant mitochondria, is specialized for the generation of heat. In contrast, white-fat tissue is specialized for the storage of fat and contains relatively few mitochondria.

The inner membranes of brown-fat mitochondria contain thermogenin, a protein that functions as a natural uncoupler of oxidative phosphorylation and generation of a protonmotive force. Thermogenin, or UCP1, is one of several uncoupling proteins (UCPs) found in most eukaryotes (but not in fermentative yeasts). Thermogenin dissipates the protonmotive force by rendering the inner mitochondrial membrane permeable to protons. As a consequence, the energy released by NADH oxidation in the electron-transport chain and used to create a proton gradient is not then used to synthesize ATP via ATP synthase. Instead, when protons move back into the matrix down their concentration gradient via thermogenin, the energy is released as heat. Thermogenin is a proton transporter, not a proton channel, and shuttles protons across the membrane at a rate that is 1-millionfold slower than that of typical ion channels (see Figure 11-2). Thermogenin is similar in sequence to the mitochondrial ATP/ADP transporter, as are many other mitochondrial transporter proteins that compose the ATP/ADP transporter family. Certain small-molecule poisons also function as uncouplers by rendering the inner mitochondrial membrane permeable to protons. One example is the lipid-soluble chemical 2,4-dinitrophenol (DNP), which can reversibly bind to and release protons and shuttle them across the inner membrane from the intermembrane space into the matrix.

Environmental conditions regulate the amount of thermogenin in brown-fat mitochondria. For instance, when rats adapt to cold temperatures, the ability of their tissues to generate heat is increased by the induction of thermogenin synthesis. In cold-adapted animals, thermogenin may constitute up to 15 percent of the total protein in the inner membranes of brown-fat mitochondria.

For many years, it was known that small animals and human infants expressed significant amounts of brown fat, but there was scant evidence for it playing a significant role in adult humans. In the newborn human, thermogenesis by brown-fat mitochondria is vital to survival, as it is in hibernating mammals. In fur seals and other animals naturally acclimated to the cold, muscle-cell mitochondria contain thermogenin; as a result, much of the proton-motive force is used for generating heat, thereby maintaining body temperature. Recently investigators have used sophisticated functional imaging methods (such as positron-emission tomography) to definitively establish the presence of brown fat in adult humans in the neck, clavicle, and other sites, the levels of which are significantly increased upon exposure to cold. Furthermore, detailed analyses of the biochemical properties and developmental origins of thermogenic fat cells have uncovered the existence of at least two subtypes of such cells: classic brown-fat cells that develop from precursor cells also used to generate skeletal muscle cells, and beige-fat cells whose detailed properties (gene expression pattern, responses to hormonal signals) differ from those of brown-fat cells. Further characterization of beige-fat cells and their influence on normal metabolism and disease may lead to new approaches to treat or prevent some metabolic disorders.

## KEY CONCEPTS OF SECTION 12.5

## Harnessing the Proton-Motive Force to Synthesize ATP

- Peter Mitchell proposed the chemiosmotic hypothesis that a proton-motive force across the inner mitochondrial membrane is the immediate source of energy for ATP synthesis.
- Bacteria, mitochondria, and chloroplasts all use the same chemiosmotic mechanism and a similar ATP synthase to generate ATP (see Figure 12-30).
- ATP synthase (also called the $\mathrm{F}_{0} \mathrm{~F}_{1}$ complex) catalyzes ATP synthesis as protons flow through the inner mitochondrial membrane (the plasma membrane in bacteria) down their electrochemical proton gradient.
- $\mathrm{F}_{0}$ contains a ring of $8-14 \mathrm{c}$ subunits, depending on the organism, that is rigidly linked to the rod-shaped $\gamma$ subunit and the $\varepsilon$ subunit of $\mathrm{F}_{1}$. These subunits rotate during ATP synthesis. Resting atop the $\gamma$ subunit is the hexameric knob of $\mathrm{F}_{1}\left[(\alpha \beta)_{3}\right]$, which protrudes into the mitochondrial matrix (cytosol in bacteria). The three $\beta$ subunits are the sites of ATP synthesis (see Figure 12-31 and 12-34a and b).
- Rotation of the $\mathrm{F}_{1} \gamma$ subunit, which is inserted in the center of the nonrotating $(\alpha \beta)_{3}$ hexamer and operates like a camshaft, leads to changes in the conformation of the nucleotidebinding sites in the three $\mathrm{F}_{1} \beta$ subunits (see Figure 12-32). By means of this binding-change mechanism, the $\beta$ subunits bind ADP and $\mathrm{P}_{\mathrm{i}}$, condense them to form ATP, and then release the ATP. Three ATPs are made for each revolution of the assembly of $\mathrm{c}, \gamma$, and $\varepsilon$ subunits.
- Movement of protons across the membrane via two halfchannels at the interface of the $\mathrm{F}_{0}$ a subunit and the c ring powers rotation of the c ring with its attached $\mathrm{F}_{1} \varepsilon$ and $\gamma$ subunits.
- The $\mathrm{F}_{0} \mathrm{~F}_{1}$ complex bends the inner mitochondrial membrane, contributing to its characteristic high curvature and to the tubular and pancake-like structures of the cristae (see Figure 12-34c and d).
- The proton-motive force also powers the uptake of $\mathrm{P}_{\mathrm{i}}$ and ADP from the cytosol in exchange for mitochondrial ATP and $\mathrm{OH}^{-}$, thus reducing the energy available for ATP synthesis. The ATP/ADP antiporter that participates in this exchange is one of the most abundant proteins in the inner mitochondrial membrane (see Figure 12-35).
- Continued mitochondrial oxidation of NADH and reduction of $\mathrm{O}_{2}$ are dependent on sufficient ADP being present in the matrix. This phenomenon, termed respiratory control, is an important mechanism for coordinating oxidation and ATP synthesis in mitochondria.
- In brown fat, the inner mitochondrial membrane contains the uncoupler protein thermogenin, a proton transporter that dissipates the proton-motive force into heat. Certain chemicals also function as uncouplers (e.g., DNP) and have the same effect, uncoupling oxidative phosphorylation from electron transport. There are two distinct types of thermogenic fat cells: brown-fat and beige-fat cells.

### 12.6 Photosynthesis and LightAbsorbing Pigments

We now shift our attention to photosynthesis, the second key process for synthesizing ATP. In plants, photosynthesis occurs in chloroplasts, large organelles found mainly in leaf cells. During photosynthesis, chloroplasts capture the energy of sunlight, convert it into chemical energy in the form of ATP and NADPH, and then use this energy to make complex carbohydrates out of carbon dioxide and water. The principal carbohydrates produced are polymers of hexose (six-carbon) sugars: sucrose, a glucose-fructose disaccharide (see Figure 2-19), and starch, a mixture of two types of large, insoluble glucose polymers called amylose and amylopectin. Starch is the primary storage carbohydrate in plants (Figure 12-36). Starch is synthesized and stored in the chloroplast. Sucrose is synthesized in the leaf cytosol from three-carbon precursors generated in the chloroplast; it is transported to non-photosynthetic (nongreen) plant tissues (e.g., roots and seeds), which metabolize it for energy by the pathways described in the previous sections.

Photosynthesis in plants, as well as in eukaryotic singlecelled algae and in several photosynthetic bacteria (e.g., the cyanobacteria and prochlorophytes), also generates oxygen. The overall reaction of oxygen-generating photosynthesis,

$$
6 \mathrm{CO}_{2}+6 \mathrm{H}_{2} \mathrm{O} \rightarrow 6 \mathrm{O}_{2}+\mathrm{C}_{6} \mathrm{H}_{12} \mathrm{O}_{6}
$$

is the reverse of the overall reaction by which carbohydrates are oxidized to $\mathrm{CO}_{2}$ and $\mathrm{H}_{2} \mathrm{O}$. In effect, photosynthesis in chloroplasts produces energy-rich sugars that are broken down and harvested for energy by mitochondria using oxidative phosphorylation.

Although green and purple bacteria also carry out photosynthesis, they use a process that does not generate oxygen. As discussed in Section 12.7, detailed analysis of the photosynthetic system in these bacteria has helped elucidate the first stages in the more common process of oxygen-generating photosynthesis. In this section, we provide an overview of the stages in oxygen-generating photosynthesis and introduce the main molecular components of the process, including the chlorophylls, the principal light-absorbing pigments.
![li904542480e](li904542480e.jpg)

FIGURE 12-36 Structure of starch. This large glucose polymer and the disaccharide sucrose (see Figure 2-19) are the principal end products of photosynthesis. Both are built of six-carbon sugars (hexoses).

## Thylakoid Membranes in Chloroplasts Are the Sites of Photosynthesis in Plants

Chloroplasts are lens-shaped organelles with a diameter of approximately $5 \mu \mathrm{~m}$ and a width of approximately $2.5 \mu \mathrm{~m}$. They contain about 3000 different proteins, 95 percent of which are encoded in the nucleus, made in the cytosol, imported into the organelle, and then transported to their appropriate membrane or space (see Chapter 13). Chloroplasts are bounded by two membranes, which do not contain chlorophyll and do not participate directly in the generation of ATP and NADPH driven by light (Figures 12-37 and 12-38). Like that of mitochondria, the outer membrane of chloroplasts contains porins and thus is permeable to metabolites of small molecular weight. The inner membrane forms a permeability barrier that contains transport proteins for regulating the movement of metabolites into and out of the organelle.

Unlike mitochondria, chloroplasts contain a third membrane-the thylakoid membrane-on which the lightdriven generation of ATP and NADPH occurs. The chloroplast thylakoid membrane is believed to constitute a single sheet that forms numerous small, interconnected flattened structures, the thylakoids, which are commonly arranged in stacks termed grana (see Figure 12-37). The spaces within all the thylakoids constitute a single continuous compartment, the thylakoid lumen (see Figure 12-38). The thylakoid membrane contains a number of integral membrane proteins to which are bound several important prosthetic groups and light-absorbing pigments, most notably chlorophylls. Starch synthesis and storage occurs in the stroma, the aqueous compartment between the thylakoid membrane and the inner membrane. In photosynthetic bacteria, extensive invaginations of the plasma membrane form a set of internal membranes, also termed thylakoid membranes, where photosynthesis occurs.

## Chloroplasts Contain Large DNAs Often Encoding More Than a Hundred Proteins

Like mitochondria, chloroplasts are thought to have evolved from an ancestral endosymbiotic photosynthetic bacterium (see Figure 12-7). However, the endosymbiotic event that gave rise to chloroplasts occurred more recently ( 1.2 billion 1.5 billion years ago) than the event that led to the evolution of mitochondria ( 1.5 billion2.2 billion years ago). Consequently, contemporary chloroplast DNAs show less structural diversity than do mtDNAs. Also like mitochondria, chloroplasts contain multiple copies of the organelle DNA as well as ribosomes, which synthesize some chloroplast DNA-encoded proteins using the standard genetic code. Like plant mtDNA, chloroplast DNA is inherited exclusively in a uniparental fashion through the female parent (egg). Other chloroplast proteins are encoded by nuclear genes, synthesized on cytosolic ribosomes, and then incorporated into the organelle (see Chapter 13).

In higher plants, chloroplast DNA molecules are 120-160 kb long, depending on the species. Plant chloroplast DNAs are

![li914542480e](li914542480e.jpg)

FIGURE 12-37 Structure of leaf and chloroplast. Like mitochondria, plant chloroplasts are bounded by two membranes separated by an intermembrane space. Photosynthesis occurs on a third membrane, the thylakoid membrane, which is surrounded by the inner membrane and forms a series of flattened vesicles (thylakoids) that enclose a single interconnected luminal space. The green color of plants is due to the green color of chlorophyll, all of which is located within the thylakoid membrane. A granum is a stack of adjacent thylakoids. The stroma is the space between the inner membrane and the thylakoids. [From Katherine Esau, D-120, Special Collections, University of California Library, Davis.]
long head-to-tail linear concatemers plus recombination intermediates between these long linear molecules. They contain 120-135 genes, 130 in the important model plant Arabidopsis thaliana. A. thaliana chloroplast DNA encodes 76 protein-coding genes and 54 genes with RNA products such as rRNAs and tRNAs. Chloroplast DNAs encode the subunits of a bacterialike RNA polymerase, and they express many of their genes from polycistronic operons, as in bacteria (see Figure 5-13a). Some chloroplast genes contain introns, but these introns are similar to the specialized introns found in some bacterial genes and in mitochondrial genes from fungi and protozoans, rather than the introns of nuclear genes. Many genes essential for chloroplast function have been transferred to the nuclear genome of plants over evolutionary time. Recent estimates from sequence analysis of the A. thaliana and cyanobacterial genomes indicate that somewhat less than 4500 genes have been transferred from the original endosymbiont to the nuclear genome.

$\square$Methods similar to those used for the transformation of yeast cells (see Chapter 6) have been developed for stably introducing foreign DNA into the chloroplasts of higher plants. The large number of chloroplast DNA molecules per cell permits the introduction of thousands of copies of an engineered gene into each cell, resulting in extraordinarily high levels of foreign protein production, comparable with that achieved with engineered bacteria. Chloroplast transformation has led to the engineering of plants that are resistant to bacterial and fungal infections, drought, and herbicides as well as to plants that can be used to make human pharmaceutical drugs (called pharming). The first such pharming drug, approved in the United States for use in adults in 2012 and children in 2014, is an enzyme to treat Gaucher's disease, a genetic disorder. This approach might also be used for the engineering of food crops containing high levels of all the amino acids essential to humans.

## Three of the Four Stages in Photosynthesis Occur Only During Illumination

The photosynthetic process in plants can be divided into four stages (see Figure 12-38), each localized to a defined area of the chloroplast: (1) absorption of light, generation of highenergy electrons, and formation of $\mathrm{O}_{2}$ from $\mathrm{H}_{2} \mathrm{O}$; (2) electron transport leading to reduction of $\mathrm{NADP}^{+}$to NADPH, and generation of a proton-motive force; (3) synthesis of ATP; and (4) conversion of $\mathrm{CO}_{2}$ into carbohydrates, commonly referred to as carbon fixation. The enzymes that incorporate $\mathrm{CO}_{2}$ into chemical intermediates and then convert them to starch are soluble constituents of the chloroplast stroma; the enzymes that form sucrose from three-carbon intermediates are in the cytosol. All four stages of photosynthesis are tightly coupled and controlled so as to produce the amount of carbohydrate required by the plant. All the reactions in stages $1-3$ are catalyzed by multiprotein complexes in the thylakoid membrane. The generation of a proton-motive force and the use of that proton-motive force to synthesize ATP resemble stages III and IV of mitochondrial oxidative phosphorylation.

![li924542480e](li924542480e.jpg)

FIGURE 12-38 Overview of the four stages of photosynthesis. In stage 1, light is absorbed by light-harvesting complexes (LHCs) and the reaction center of photosystem II (PSII). The LHCs transfer the absorbed energy to the reaction centers, which use it, or the energy absorbed directly from a photon, to oxidize water to molecular oxygen and generate high-energy electrons (electron paths shown by blue arrows). In stage 2, these electrons move down an electron-transport chain, which uses either lipid-soluble $\left(\mathrm{Q} / \mathrm{QH}_{2}\right)$ or water-soluble (plastocyanin, PC) electron carriers to shuttle electrons between multiple protein complexes. As electrons move down the chain, they release energy that the complexes use to generate a proton-motive force and, after additional energy is introduced by absorption of light in photosystem I (PSI), to synthesize the high-energy electron carrier NADPH. In stage 3, flow of protons down their concentration and voltage gradient through the
$F_{0} F_{1}$ ATP synthase drives ATP synthesis. Stages 1-3 in plants take place in the thylakoid membrane of the chloroplast. In stage 4, in the chloroplast stroma, the energy stored in NADPH and ATP is used to incorporate $\mathrm{CO}_{2}$ into the three-carbon molecule glyceraldehyde 3-phosphate, the first step in a process known as carbon fixation. These molecules are then transported to the cytosol of the cell for conversion to hexose sugars in the form of sucrose. Glyceraldehyde 3-phosphate is also used to make starch within the chloroplast. Inset: Three-dimensional reconstruction from cryoelectron tomography of a chloroplast in the unicellular green alga Chlamydomonas reinhardtii, showing thylakoid membranes (dark green), thylakoid lumen (light green), inner and outer membranes (blue), and one small starch granule (tan). (Inset from Engel, B. D., et al., "Native architecture of the Chlamydomonas chloroplast revealed by in situ cryo-electron tomography," eLIFE, 2015; 4: e04889.)
chlorophylls contain a central $\mathrm{Mg}^{2+}$ ion (rather than $\mathrm{Fe}^{2+}$ ) and have an additional five-member ring. The energy of the absorbed light is ultimately used to remove electrons from a donor (water in the case of green plants), forming oxygen:

$$
2 \mathrm{H}_{2} \mathrm{O} \xrightarrow{\text { Light }} \mathrm{O}_{2}+4 \mathrm{H}^{+}+4 \mathrm{e}^{-}
$$

![li934542480e](li934542480e.jpg)

FIGURE 12-39 Structure of chlorophyll a, the principal pigment that traps light energy. Electrons are delocalized among three of chlorophyll $a$ 's four central rings (yellow) and the atoms that interconnect them. In chlorophyll, a $\mathrm{Mg}^{2+}$ ion, rather than the $\mathrm{Fe}^{2+}$ ion found in heme, sits at the center of the porphyrin ring, and an additional fivemember ring (blue) is present; otherwise, the structure of chlorophyll is similar to that of heme, found in molecules such as hemoglobin and cytochromes (see Figure 12-20a). The hydrocarbon phytol "tail" facilitates the binding of chlorophyll to hydrophobic regions of chlorophyllbinding proteins. The $\mathrm{CH}_{3}$ group (green) is replaced by a formaldehyde (CHO) group in chlorophyll $b$.

The electrons are transferred to a primary electron acceptor, a quinone designated Q , which is similar to CoQ in mitochondria. In plants, the oxidation of water takes place in a multiprotein complex called photosystem II (PSII).

Quantum mechanics established that light, a form of electromagnetic radiation, has properties of both waves and particles. When light interacts with matter, it behaves as discrete packets of energy (quanta) called photons. The energy of a photon is proportional to the frequency of the light wave, and thus inversely proportional to its wavelength. Thus photons of shorter wavelengths have higher energies. The energy of visible light is considerable. Light with a wavelength of $550 \mathrm{~nm}\left(550 \times 10^{-7} \mathrm{~cm}\right)$, typical of sunlight, has about 52 kcal of energy per mole of photons. This is enough energy to synthesize several moles of ATP from ADP and $\mathrm{P}_{\mathrm{i}}$ if all the energy were used for this purpose.

Stage 2: Electron Transport and Generation of a ProtonMotive Force Electrons move from the quinone primary electron acceptor through a series of electron carriers until they reach the ultimate electron acceptor, usually the oxidized form of nicotinamide adenine dinucleotide phosphate $\left(\mathrm{NADP}^{+}\right)$, reducing it to NADPH. The structure of $\mathrm{NADP}^{+}$ is identical to that of $\mathrm{NAD}^{+}$except for the presence of an additional phosphate group. Both molecules gain and lose
electrons in the same way (see Figure 2-33). In plants, the reduction of $\mathrm{NADP}^{+}$takes place in a complex called photosystem I (PSI) (Figure 12-38). The transport of electrons in the thylakoid membrane is coupled to the movement of protons from the stroma to the thylakoid lumen, forming a pH gradient across the membrane $\left(\mathrm{pH}_{\text {lumen }}<\mathrm{pH}_{\text {stroma }}\right)$. This process is analogous to the generation of a proton-motive force across the inner mitochondrial membrane and in bacterial membranes during electron transport (see Figure 12-29).

Thus the overall reaction of stages 1 and 2 can be summarized as

$$
2 \mathrm{H}_{2} \mathrm{O}+2 \mathrm{NADP}^{+} \xrightarrow{\text { Light }} 2 \mathrm{H}^{+}+2 \mathrm{NADPH}+\mathrm{O}_{2}
$$

Stage 3: Synthesis of ATP Protons move down their concentration gradient from the thylakoid lumen to the stroma through the chloroplast $\mathrm{F}_{0} \mathrm{~F}_{1}$ complex (ATP synthase), which couples proton movement to the synthesis of ATP from ADP and $\mathrm{P}_{\mathrm{i}}$, as we have seen for the ATP synthases in mitochondria and bacteria (see Figures 12-31, 12-32, and 12-34).

Stage 4: Carbon Fixation The NADPH and ATP generated by stages 2 and 3 of photosynthesis provide the energy and the electrons to drive the synthesis of polymers of six-carbon sugars from $\mathrm{CO}_{2}$ and $\mathrm{H}_{2} \mathrm{O}$. The overall chemical equation is written as

$$
\begin{aligned}
& 6 \mathrm{CO}_{2}+18 \mathrm{ATP}^{4-}+12 \mathrm{NADPH}+12 \mathrm{H}_{2} \mathrm{O} \rightarrow \\
& \quad \mathrm{C}_{6} \mathrm{H}_{12} \mathrm{O}_{6}+18 \mathrm{ADP}^{3-}+18 \mathrm{P}_{\mathrm{i}}^{2-}+12 \mathrm{NADP}^{+}+6 \mathrm{H}^{+}
\end{aligned}
$$

The reactions that generate the ATP and NADPH used in carbon fixation are directly dependent on light energy; thus stages 1-3 are called the light reactions of photosynthesis. The reactions in stage 4 are indirectly dependent on light energy; they are sometimes called the dark reactions of photosynthesis because they can occur in the dark, using the supplies of ATP and NADPH generated by light energy (see Figure 12-38). However, the reactions in stage 4 are not confined to the dark; in fact, they occur primarily during illumination.

## Photosystems Comprise a Reaction Center and Associated Light-Harvesting Complexes

The absorption oflightenergy and its conversion into chemical energy occurs in multiprotein complexes called photosystems. Found in all photosynthetic organisms, both eukaryotic and prokaryotic, photosystems consist of two closely linked components: a reaction center, where the primary events of photosynthesis-light absorption and generation of high-energy electrons-occur; and an antenna complex consisting of numerous protein complexes, including internal antenna proteins. Each photosystem is also associated with external antenna complexes termed light-barvesting complexes (LHCs), made up of specialized proteins

that capture light energy and efficiently transmit it to the reaction center to generate high-energy electrons (see Figure 12-38).

Both reaction centers and antennas contain tightly bound light-absorbing pigment molecules. Chlorophyll $a$, the principal pigment involved in photosynthesis, is present in both reaction centers and antennas. In addition to chlorophyll $a$, antennas contain other light-absorbing pigments: chlorophyll $b$ in vascular plants and carotenoids in both plants and photosynthetic bacteria. Carotenoids consist of long branched hydrocarbon chains with alternating single and double bonds; they are similar in structure to the visual pigment retinal (see Figure 15-19), which absorbs light in the eye. The presence of various antenna pigments, which absorb light at different wavelengths, greatly extends the range of light that can be absorbed and used for photosynthesis.

One of the strongest pieces of evidence for the involvement of chlorophylls and carotenoids in photosynthesis is that the absorption spectrum of these pigments is similar to the action spectrum of photosynthesis (Figure 12-40). The latter is a measure of the relative ability of light of different wavelengths to support photosynthesis.
![li944542480e](li944542480e.jpg)

EXPERIMENTAL FIGURE 12-40 The rate of photosynthesis is greatest at the wavelengths of light absorbed by three plant pigments. The action spectrum of photosynthesis in plants (the relative ability of light of different wavelengths to support photosynthesis) is shown in black. The energy from light can be converted into ATP only if it can be absorbed by pigments in the chloroplast. Absorption spectra (showing how well light of different wavelengths is absorbed) for three photosynthetic pigments present in the antennas of plant photosystems are shown in color. Comparison of the action spectrum of photosynthesis with the individual absorption spectra of these pigments suggests that photosynthesis at 680 nm is primarily due to light absorbed by chlorophyll $a$; at 650 nm , to light absorbed by chlorophyll $b$; and at shorter wavelengths, to light absorbed by chlorophylls $a$ and $b$ and by carotenoid pigments, including $\beta$-carotene.

When chlorophyll $a$ (or any other molecule) absorbs visible light, the absorbed light energy raises electrons in the chlorophyll $a$ to a higher-energy (excited) state. This state differs from the ground (unexcited) state largely in the distribution of the electrons around the C and N atoms of the porphyrin ring. Excited states are unstable, and the electrons return to the ground state by one of several competing processes. For chlorophyll $a$ molecules dissolved in organic solvents such as ethanol, the principal reactions that dissipate the excited-state energy are the emission of light (fluorescence and phosphorescence) and thermal emission (heat). However, when the same chlorophyll $a$ is bound in the unique protein environment of the reaction center, dissipation of excited-state energy occurs by a different process, which is the key to photosynthesis.

## Photoelectron Transport from Energized Reaction-Center Chlorophyll a Produces a Charge Separation

Within the reaction center, two adjacent chlorophyll $a$ molecules, referred to as the special-pair chlorophylls, lie close to the luminal face of the thylakoid membrane (Figure 12-41).
![li954542480e](li954542480e.jpg)

FIGURE 12-41 Photoelectron transport, the primary event in photosynthesis. After absorption of a photon of light, one of the excited special-pair chlorophyll $a$ molecules in the reaction center (left) donates, via several intermediates (not shown), an electron to a loosely bound acceptor molecule, the quinone Q , on the stromal surface of the thylakoid membrane, creating an essentially irreversible charge separation across the membrane (right). Subsequent transfers of this electron release energy that is used to generate ATP and NADPH (see Figures 12-43 and 12-44). The positively charged chlorophyll $a^{+}$generated when the light-excited electron moves to Q is eventually neutralized by the transfer to the chlorophyll $a^{+}$of another electron. In plants, the oxidation of $\mathrm{H}_{2} \mathrm{O}$ to $\mathrm{O}_{2}$ provides this neutralizing electron and takes place in a multiprotein complex called photosystem II (see Figure 1244). Photosystem I uses a similar photoelectron transport pathway, but instead of oxidizing water, it receives an electron from a protein carrier called plastocyanin to neutralize the positive charge on chlorophyll $a^{+}$ (see Figure 12-44).

![li964542480e](li964542480e.jpg)

FIGURE 12-42 Light-harvesting complexes and photosystems in cyanobacteria and plants. (a) Diagram of the membrane of a cyanobacterium, in which each multiprotein light-harvesting complex (LHC) contains 90 chlorophyll molecules and 31 other small molecules, all held in a specific geometric arrangement for optimal light absorption and energy transfer. Of the six chlorophyll molecules in the reaction center, two constitute the special-pair chlorophylls that can initiate photoelectron transport (blue arrow) when excited. Resonance transfer of energy (red arrows) rapidly funnels energy from absorbed light to one of two "bridging" chlorophylls and thence to
the special-pair chlorophylls in the reaction center. (b) Three-dimensional organization of photosystem I (PSI) and its associated LHCs from Pisum sativum (garden pea), as determined by x-ray crystallography, seen from the plane of the membrane. Only the chlorophylls and the reaction-center electron carriers are shown. (c) Expanded view of the reaction center from (b), rotated $90^{\circ}$ about a vertical axis. See W. Kühlbrandt, 2001, Nature 411:896, and P. Jordan et al., 2001, Nature 411:909. [Parts (b) and (c) data from A. Ben-Sham et al., 2003, Nature 426:630, PDB ID 1qvz; and Y. Mazor, A. Borovikova, and N. Nelson, 2015, $£ 88$ : 4:e07433, PDB ID 4y28.]

When a photon of light with a wavelength of about 680 nm is absorbed by one of these two chlorophyll $a$ molecules, the energy of that chlorophyll $a$ molecule increases by $42 \mathrm{kcal} /$ mol (the first excited state). This energized molecule rapidly donates an electron to the adjacent chlorophyll, which passes it on to a series of intermediate acceptors. In this manner, the electron is rapidly passed on to the primary electron acceptor, quinone Q , near the stromal surface of the thylakoid membrane. This light-driven electron transfer, called photoelectron transport, depends on the unique environment of both the chlorophylls and the acceptor within the reaction center. Photoelectron transport, which occurs nearly every time a photon is absorbed, leaves a positive charge on the chlorophyll $a$ close to the luminal surface of the thylakoid membrane (on the opposite side from the stroma) and generates a reduced, negatively charged acceptor $\left(\mathrm{Q}^{-}\right)$near the stromal surface.

The $\mathrm{Q}^{-}$produced by photoelectron transport is a powerful reducing agent with a strong tendency to transfer an electron to another molecule, ultimately to $\mathrm{NADP}^{+}$. The positively charged chlorophyll $a^{+}$, a strong oxidizing agent, attracts an electron from an electron donor on the luminal surface to regenerate the original chlorophyll $a$. In plants, the oxidizing power of four chlorophyll $a^{+}$molecules is used, by way of intermediates, to remove four electrons from two $\mathrm{H}_{2} \mathrm{O}$ molecules bound to a site on the luminal surface to form $\mathrm{O}_{2}$ :
$2 \mathrm{H}_{2} \mathrm{O}+4$ chlorophyll $a^{+} \rightarrow 4 \mathrm{H}^{+}+\mathrm{O}_{2}+4$ chlorophyll $a$
These potent biological reductants and oxidants provide all the energy needed to drive all subsequent reactions of photosynthesis: electron transport (stage 2), ATP synthesis (stage 3), and $\mathrm{CO}_{2}$ fixation (stage 4).

Chlorophyll $a$ also absorbs light at discrete wavelengths shorter (and therefore of higher energy) than 680 nm (see Figure 12-40). Such absorption raises the molecule into one of several excited states whose energies are higher than that of the first excited state described above, and which decay by releasing energy within $2 \times 10^{-12}$ seconds ( 2 picoseconds, ps ) to the lower-energy first excited state, with loss of the extra energy as heat. Because photoelectron transport and the resulting charge separation occur only from the first excited state of the reaction-center chlorophyll $a$, the quantum yield-the amount of photosynthesis per absorbed photonis the same for all wavelengths of visible light shorter than 680 nm . How closely the wavelength of light matches the absorption spectrum of the pigment determines how likely it is that the photon will be absorbed. Once absorbed, the photon's exact wavelength is not critical, provided it is at least energetic enough to push the chlorophyll $a$ into the first excited state.

## Internal Antennas and Light-Harvesting Complexes Increase the Efficiency of Photosynthesis

Although the special-pair chlorophyll $a$ molecules within the reaction center that are involved directly in charge separation and electron transfer are capable of directly absorbing light and initiating photosynthesis, they are most commonly energized indirectly by energy transferred to them from other light-absorbing and energy-transferring pigments. These other pigments, which include many other chlorophylls, absorb photons and pass the energy to the specialpair chlorophylls (Figure 12-42). Some of these pigments are bound to protein subunits that are considered to be intrinsic components of the photosystem, which is made up of several distinct protein chains, and thus are called internal antennas. Others are incorporated into protein complexes that bind to, but are distinct from, the photosystem core proteins and are called light-barvesting complexes (LHCs). Even at the maximum light intensity encountered by photosynthetic organisms (tropical noontime sunlight), each reaction-center chlorophyll $a$ molecule absorbs only about one photon per second, which is not enough to support photosynthesis sufficient for the needs of the plant. The involvement of internal antennas and LHCs greatly increases the efficiency of photosynthesis, especially at more typical light intensities, by increasing absorption of $680-\mathrm{nm}$ light and by extending the range of wavelengths of light that can be absorbed by other antenna pigments.

Photosystem core proteins and LHC proteins maintain the pigment molecules in the precise orientations and positions that are optimal for light absorption and rapid ( $<10^{-9}$ seconds) energy transfer, called resonance transfer, to one of the special-pair chlorophyll $a$ molecules in the associated reaction center. Resonance energy transfer does not involve the transfer of an electron. Studies on one of the two photosystems in cyanobacteria, which are similar to those in multicellular, seed-bearing plants, suggest that energy from
absorbed light is funneled first to a "bridging" chlorophyll in each LHC and then to the special-pair chlorophylls in the reaction center (see Figure 12-42a). Surprisingly, however, the molecular structures of LHCs from plants and cyanobacteria are completely different from those from green and purple bacteria, even though both types contain carotenoids and chlorophylls. Figure 12-42b shows the distribution of the chlorophyll pigments in photosystem I from Pisum sativum (garden pea) together with those from peripheral LHC antennas. The large number of internal and LHC antenna chlorophylls that surround the reaction center permit efficient transfer of absorbed light energy to the special-pair chlorophylls in the reaction center.

Although LHC antenna chlorophylls can transfer light energy absorbed from a photon, they cannot release an electron. As we've seen already, this function resides in the two reaction-center chlorophylls. To understand their electronreleasing ability, we examine the structure and function of the reaction center in bacterial and plant photosystems in the next section.

## KEY CONCEPTS OF SECTION 12.6

## Photosynthesis and Light-Absorbing Pigments

- The principal end products of photosynthesis in plants are molecular oxygen and polymers of six-carbon sugars (starch and sucrose).
- The light-capturing and ATP-generating reactions of photosynthesis occur in the thylakoid membrane located within chloroplasts. The permeable outer membrane and the inner membrane surrounding chloroplasts do not participate directly in photosynthesis (see Figures 12-37 and 12-38).
- There are four stages in photosynthesis: (1) absorption of light, generation of high-energy electrons, and formation of $\mathrm{O}_{2}$ from $\mathrm{H}_{2} \mathrm{O}$; (2) electron transport leading to reduction of $\mathrm{NADP}^{+}$to NADPH, and to generation of a proton-motive force; (3) synthesis of ATP; and (4) conversion of $\mathrm{CO}_{2}$ into carbohydrates (carbon fixation).
- In stage 1 of photosynthesis, light energy is absorbed by one of two "special-pair" chlorophyll $a$ molecules bound to reaction-center proteins in the thylakoid membrane. The energized chlorophylls donate, via intermediates, an electron to a quinone on the opposite side of the membrane, creating a charge separation (see Figure 12-41). In green plants, the positively charged chlorophylls then remove electrons from water, forming molecular oxygen $\left(\mathrm{O}_{2}\right)$.
- In stage 2, electrons are transported from the reduced quinone via intermediates associated with the thylakoid membrane until they reach the ultimate electron acceptor, usually $\mathrm{NADP}^{+}$, reducing it to NADPH. Electron transport

is coupled to movement of protons across the membrane from the stroma to the thylakoid lumen, forming a pH gradient (proton-motive force) across the thylakoid membrane.

- In stage 3, movement of protons down their electrochemical gradient through $\mathrm{F}_{0} \mathrm{~F}_{1}$ complexes (ATP synthase) powers the synthesis of ATP from ADP and $\mathrm{P}_{\mathrm{i}}$.
- In stage 4, the NADPH and ATP generated in stages 2 and 3 provide the energy and the electrons to drive the fixation of $\mathrm{CO}_{2}$, which results in the synthesis of carbohydrates. These reactions occur in the thylakoid stroma and cytosol.
- Associated with each reaction center are multiple internal antennas and external light-harvesting complexes, which contain chlorophylls $a$ and $b$, carotenoids, and other pigments that absorb light at multiple wavelengths. Energy, but not an electron, is transferred from the internal antenna and LHC chlorophyll molecules to the special-pair chlorophylls in the reaction center by resonance energy transfer (see Figure 12-42).


### 12.7 Molecular Analysis of Photosystems

As noted in the previous section, photosynthesis in the green and purple bacteria does not generate oxygen, whereas photosynthesis in cyanobacteria, algae, and plants does. ${ }^{*}$ This difference is attributable to the presence of two types of photosystems (PS) in the latter organisms: PSI reduces NADP ${ }^{+}$ to NADPH, and PSII forms $\mathrm{O}_{2}$ from $\mathrm{H}_{2} \mathrm{O}$. In contrast, the green and purple bacteria have only one type of photosystem, which cannot form $\mathrm{O}_{2}$. We first discuss the simpler photosystem of purple bacteria and then consider the more complicated photosynthetic machinery in chloroplasts.

## The Single Photosystem of Purple Bacteria Generates a Proton-Motive Force but No $\mathrm{O}_{2}$

The mechanism of charge separation in the photosystem located in the plasma membrane of purple bacteria (Figure 12-43) is identical to that in plants outlined earlier. The reaction-center protein contains the prosthetic groups that absorb light and transport electrons during photosynthesis. These prosthetic groups include "special-pair" bacteriochlorophyll $a$ molecules equivalent to the reaction-center

[^0]chlorophyll $a$ molecules in plants, as well as several other pigments and two quinones, termed $Q_{A}$ and $Q_{B}$, that are structurally similar to mitochondrial ubiquinone. The energy from absorbed light is used to strip an electron from a reaction-center bacteriochlorophyll $a$ molecule and transfer it to $\mathrm{Q}_{\mathrm{A}}$ via several different pigments in a series of very rapid reactions taking between 4 and 200 ps . Then, in the slowest step, in about 200 microseconds ( $\mu \mathrm{s}$ ), the electron moves from $\mathrm{Q}_{\mathrm{A}}$ to the primary electron acceptor $\mathrm{Q}_{\mathrm{B}}$, which is loosely bound to a site on the cytosolic membrane face. The chlorophyll thereby acquires a positive charge, and $\mathrm{Q}_{\mathrm{B}}$ acquires a negative charge.

After the primary electron acceptor, $\mathrm{Q}_{\mathrm{B}}$, in the bacterial reaction center accepts one electron, forming $\mathrm{Q}_{\mathrm{B}}^{--}$, it accepts a second electron from the same reaction-center chlorophyll following its re-excitation (e.g., by absorption of a second photon or by transfer of energy from antenna molecules). The quinone then binds two protons from the cytosol, forming reduced quinone $\left(\mathrm{QH}_{2}\right)$, which is released from the reaction center (see Figure 12-43). $\mathrm{QH}_{2}$ diffuses within the bacterial membrane to the $\mathrm{Q}_{\mathrm{o}}$ site on the exoplasmic face of a cytochrome $b c_{1}$ electron-transport complex, which is similar in structure to complex III in mitochondria. There it releases its two protons into the periplasmic space (the space between the plasma membrane and the bacterial cell wall), generating a proton-motive force across the plasma membrane. Simultaneously, $\mathrm{QH}_{2}$ releases its two electrons, which move through the cytochrome $b c_{1}$ complex exactly as depicted for the mitochondrial complex III $\left(\mathrm{CoQH}_{2}-\right.$ cytochrome $c$ reductase) in Figure 12-24. The Q cycle in the bacterial reaction center, like the Q cycle in mitochondria, pumps additional protons from the cytosol to the periplasmic space, thereby increasing the proton-motive force.

The acceptor for electrons transferred from the reaction center and through the cytochrome $b c_{1}$ complex is a soluble cytochrome, a one-electron carrier, in the periplasmic space, which is reduced from the $\mathrm{Fe}^{3+}$ to the $\mathrm{Fe}^{2+}$ state (see Figure 12-43). The reduced cytochrome (analogous to cytochrome $c$ in mitochondria) then diffuses to a reaction center, where it releases its electron to a positively charged chlorophyll $a^{+}$, returning that chlorophyll to the uncharged ground state and the cytochrome to the $\mathrm{Fe}^{3+}$ state. This cyclic electron flow generates no oxygen and no reduced coenzymes, but it has generated a proton-motive force.

As in other systems, this proton-motive force is used both by the $\mathrm{F}_{0} \mathrm{~F}_{1}$ complex located in the bacterial plasma membrane to synthesize ATP and by transport proteins to move molecules across the membrane against a concentration gradient.

## Chloroplasts Contain Two Functionally and Spatially Distinct Photosystems

In the 1940s, biophysicist R. Emerson discovered that the rate of plant photosynthesis generated by light of wavelength 700 nm could be greatly enhanced by adding light of shorter wavelengths (higher energy). He found that a combination


[^0]:    * A very different type of mechanism used to harvest the energy of light, which occurs only in certain archaeans, is not discussed here because it is very different from the reaction-center mechanisms described here. In this other mechanism, the plasma-membrane protein that absorbs a photon of light, called bacteriorhodopsin, also pumps one proton from the cytosol to the extracellular space for every photon of light absorbed.

![li974542480e](li974542480e.jpg)

FIGURE 12-43 Cyclic electron flow in the single photosystem of purple bacteria. Cyclic electron flow generates a proton-motive force but no $\mathrm{O}_{2}$. Blue arrows indicate flow of electrons; red arrows indicate proton movement. (Left) Energy absorbed directly from light or funneled from an associated LHC (not illustrated here) energizes one of the special-pair chlorophylls in the reaction center. Photoelectron transport from the energized chlorophyll, via an accessory chlorophyll, pheophytin $(\mathrm{Ph})$, and quinone $\mathrm{A}\left(\mathrm{Q}_{6}\right)$, to quinone $\mathrm{B}\left(\mathrm{Q}_{6}\right)$, which forms the semiquinone $\mathrm{Q}^{--}$and leaves a positive charge on the chlorophyll. Following absorption of a second photon and transfer of a second electron to the semiquinone, the quinone rapidly picks up two protons from the
cytosol to form $\mathrm{QH}_{2}$. (Center) After diffusing through the membrane and binding to the $\mathrm{Q}_{6}$ site on the periplasmic (exoplasmic) face of the cytochrome $b c_{1}$ complex, $\mathrm{QH}_{2}$ donates two electrons and simultaneously gives up two protons to the external medium in the periplasmic space, generating a proton electrochemical gradient (proton-motive force) that drives ATP synthesis (right). Electrons are transported back to a reaction-center chlorophyll via a soluble cytochrome, which diffuses in the periplasmic space. Note the cyclical path (blue) of electrons. Operation of a Q cycle in the cytochrome $b c_{1}$ complex pumps additional protons across the membrane to the external medium, as in mitochondria. See J. Deisenhofer and H. Michael, 1991, Annu. Rev. Cell Biol. 7:1.
of light at, say, 600 and 700 nm supports a rate of photosynthesis higher than the sum of the rates for the two separate wavelengths. This so-called Emerson effect led researchers to conclude that photosynthesis in plants involves the interaction of two separate photosystems, referred to as PSI and PSII. PSI is driven by light of 700 nm or less; PSII, only by shorter-wavelength light ( $<680 \mathrm{~nm}$ ).

In chloroplasts, the special-pair reaction-center chlorophylls that initiate photoelectron transport in PSI and in PSII differ in their light-absorption maxima because of differences in their protein environments. For this reason, these chlorophylls are often denoted $\mathrm{P}_{680}$ (PSII) and $\mathrm{P}_{700}$ (PSI). Like a bacterial reaction center, each chloroplast reaction center is associated with multiple internal antennas and light-harvesting complexes; the LHCs associated with PSII (e.g., LHCII) and with PSI (e.g., LHCI) contain different proteins. Furthermore, the two photosystems are distributed differently in thylakoid membranes: PSII primarily in regions of stacked thylakoids (grana, see Figure 12-37) and PSI primarily in unstacked regions. The stacking of thylakoid membranes may be due to the binding properties of the proteins associated with PSII, especially LHCII.

Finally, and most important, the two chloroplast photosystems differ significantly in their functions (Figure 12-44): only PSII oxidizes water to form molecular oxygen, whereas only PSI transfers electrons to the final electron acceptor,
$\mathrm{NADP}^{+}$. Photosynthesis in chloroplasts can follow a linear or a cyclical pathway through these photosystems. The linear pathway, which we discuss first, can support carbon fixation as well as ATP synthesis. In contrast, the cyclical pathway supports only ATP synthesis and generates no reduced NADPH for use in carbon fixation. Photosynthetic algae and cyanobacteria contain two photosystems analogous to those in chloroplasts. Similar proteins and pigments compose photosystems I and II in plants and in photosynthetic bacteria.

## Linear Electron Flow Through Both Plant Photosystems Generates a Proton-Motive Force, $\mathrm{O}_{2}$, and NADPH

Linear electron flow in chloroplasts involves PSII and PSI in an obligate series in which electrons are transferred from $\mathrm{H}_{2} \mathrm{O}$ to $\mathrm{NADP}^{+}$(see Figure 12-44). The process begins with absorption of a photon by PSII, causing an electron to move from a $\mathrm{P}_{680}$ chlorophyll $a$ to an acceptor plastoquinone $\left(\mathrm{Q}_{8}\right)$ on the stromal surface. The resulting oxidized $\mathrm{P}_{680}{ }^{+}$strips one electron from the relatively unwilling donor $\mathrm{H}_{2} \mathrm{O}$, forming an intermediate in $\mathrm{O}_{2}$ formation, as we shall see shortly, and a proton, which remains in the thylakoid lumen and contributes to the proton-motive force. After $\mathrm{P}_{680}$ absorbs a second photon, the semiquinone $\mathrm{Q}^{--}$accepts a second electron and picks up two protons from the stromal space, generating $\mathrm{QH}_{2}$.

![li984542480e](li984542480e.jpg)

FIGURE 12-44 Linear electron flow in plants, which requires both chloroplast photosystems, PSI and PSII. Blue arrows indicate flow of electrons; red arrows indicate proton movement. LHCs are not shown. (Left) In the PSII reaction center, two sequential light-induced excitations of the same special-pair $\mathrm{P}_{080}$ chlorophyll result in a two-step reduction of the primary electron acceptor $\mathrm{Q}_{\mathrm{B}}$ to $\mathrm{QH}_{2}$. On the luminal side of PSII, electrons removed from $\mathrm{H}_{2} \mathrm{O}$ by the oxygen-evolving complex are transferred to $\mathrm{P}_{080}{ }^{+}$, restoring the reaction-center chlorophylls to the ground state after each excitation. The oxygen-evolving complex contains a cluster of four manganese ions (Mn, violet), a $\mathrm{Ca}^{2+}$ ion (green), and a $\mathrm{Cl}^{-}$ion (teal). These bound ions function in the splitting of $\mathrm{H}_{2} \mathrm{O}$ and maintain the environment essential for high rates of $\mathrm{O}_{2}$ evolution. Four sequential light-induced excitations of the $\mathrm{P}_{080}$ (double the number illustrated here) are required to oxidize two water molecules and release four protons and one molecule of molecular oxygen
$\left(\mathrm{O}_{2}\right)$. A tyrosine on the protein helps conducts electrons from the Mn ions to the oxidized reaction-center chlorophyll $\left(\mathrm{P}_{080}{ }^{+}\right)$, reducing it to the ground state $\left(\mathrm{P}_{080}\right)$ after excitation by each photon. (Center) The cytochrome $b f$ complex then accepts electrons from $\mathrm{QH}_{2}$ and transports two protons into the lumen. Operation of a Q cycle in the cytochrome $b f$ complex translocates additional protons across the membrane to the thylakoid lumen, increasing the proton-motive force. (Right) In the PSI reaction center, each electron released from light-excited $\mathrm{P}_{700}$ chlorophylls moves via a series of carriers in the reaction center to the stromal surface, where soluble ferredoxin (an Fe-S protein) transfers the electron to ferredoxin-NADP ${ }^{+}$reductase (FNR). This enzyme uses the prosthetic group flavin adenine dinucleotide (FAD) and a proton to reduce NADP ${ }^{+}$, forming NADPH. $\mathrm{P}_{700}{ }^{+}$is restored to its ground state by addition of an electron carried from PSII via the cytochrome $b f$ complex and plastocyanin, a soluble electron carrier.

After diffusing in the membrane, $\mathrm{QH}_{2}$ binds to the $\mathrm{Q}_{\mathrm{o}}$ site on a cytochrome $b f$ complex that is analogous to the bacterial cytochrome $b c_{1}$ complex and to the mitochondrial complex III. As in those systems, a Q cycle (see Figure 12-24) operates, thereby increasing the proton-motive force generated by electron transport. After the cytochrome $b f$ complex accepts electrons from $\mathrm{QH}_{2}$, it transfers them, one at a time, to the $\mathrm{Cu}^{2+}$ form of the soluble electron carrier plastocyanin (analogous to cytochrome c), reducing it to the $\mathrm{Cu}^{1+}$ form. Reduced plastocyanin then diffuses in the thylakoid lumen, carrying the electron to PSI.

Absorption of a photon by PSI leads to removal of an electron from the reaction-center chlorophyll $a, \mathrm{P}_{700}$ (see Figure 12-44). The resulting oxidized $\mathrm{P}_{700}{ }^{+}$is reduced by an electron from plastocyanin that originated in PSII. Again, this process is analogous to the electron-transport chain in mitochondria (see Figure 12-22). The electron taken up at the luminal surface by the $\mathrm{P}_{700}$ and energized by photon absorption moves within PSI via several carriers to the stromal surface of the thylakoid membrane, where it is accepted by ferredoxin, an iron-sulfur ( $\mathrm{Fe}-\mathrm{S}$ ) protein. In linear electron flow, electrons excited in PSI are transferred from ferredoxin via the enzyme ferredoxin-NADP ${ }^{+}$reductase (FNR). This
enzyme uses the prosthetic group FAD as an electron carrier to reduce $\mathrm{NADP}^{+}$, forming, together with one proton picked up from the stroma, the reduced molecule NADPH. The linear electron flow pathway is now completed.
$\mathrm{F}_{0} \mathrm{~F}_{1}$ complexes in the thylakoid membrane use the pro-ton-motive force generated during linear electron flow to synthesize ATP on the stromal side of the membrane. Thus this pathway exploits the energy from multiple photons absorbed by both PSII and PSI and their antennas to generate both NADPH and ATP in the stroma of the chloroplast, where they are used for $\mathrm{CO}_{2}$ fixation.

## An Oxygen-Evolving Complex Is Located on the Luminal Surface of the PSII Reaction Center

Photoelectron transport in the PSII reaction center (see Figure 12-44) resembles that in the purple bacterial reaction center described earlier (see Figure 12-43). Excitation by a photon with a wavelength of $<680 \mathrm{~nm}$ triggers the loss of an electron from a $\mathrm{P}_{680}$ chlorophyll. That electron is transported rapidly to a quinone $\left(Q_{A}\right)$ and then to the primary electron acceptor, $\mathrm{Q}_{\mathrm{B}}$, on the stromal surface of the thylakoid

membrane (see Figure 12-44). PSII differs from the bacterial reaction center, however, in that it has an additional threeprotein subunit, the oxygen-evolving complex, that faces the thylakoid lumen (in this context, "oxygen-evolving" refers to generating $\mathrm{O}_{2}$ from water).

The photochemically oxidized special-pair reactioncenter chlorophyll $a$ of PSII, $\mathrm{P}_{680}{ }^{+}$, is the strongest biological oxidant known. The reduction potential of $\mathrm{P}_{680}{ }^{+}$is more positive than that of $\mathrm{H}_{2} \mathrm{O}$, and thus it can oxidize $\mathrm{H}_{2} \mathrm{O}$ to generate $\mathrm{O}_{2}$ and $\mathrm{H}^{+}$ions. The oxidation of $\mathrm{H}_{2} \mathrm{O}$ provides the electrons for reduction of $\mathrm{P}_{680}{ }^{+}$in PSII, thus replacing the electron released by light absorption. The oxygen-evolving complex is a protein that contains a manganese $(\mathrm{Mn})$, calcium, and oxygen cluster $\left(\mathrm{Mn}_{4} \mathrm{CaO}_{5}\right)$ that directly mediates the conversion of $\mathrm{H}_{2} \mathrm{O}$ to $\mathrm{O}_{2}$, as well as a bound $\mathrm{Cl}^{-}$ion that influences the reaction rate (see Figure 12-44). The Mn ions cycle through five different oxidation states during $\mathrm{O}_{2}$ generation. A nearby tyrosine side chain helps transfer electrons from $\mathrm{H}_{2} \mathrm{O}$ to the $\mathrm{P}_{680}{ }^{+}$. The protons released from $\mathrm{H}_{2} \mathrm{O}$ remain in the thylakoid lumen. The oxidation of two molecules of $\mathrm{H}_{2} \mathrm{O}$ to form $\mathrm{O}_{2}$ requires the removal of four electrons, but absorption of each photon by PSII results in the transfer of just one electron. Thus a single PSII must lose an electron and then oxidize the oxygen-evolving complex four times in a row for an $\mathrm{O}_{2}$ molecule to be formed. Channels in the protein of the oxygen-evolving complex have been proposed to serve as conduits through the complex for the delivery of $\mathrm{H}_{2} \mathrm{O}$ to and the removal of $\mathrm{O}_{2}$ from the active site.

The herbicide atrazine is one of the most commonly used weed killers in US agriculture. Atrazine binds to PSII, blocks the binding of oxidized $\mathrm{Q}_{\mathrm{B}}$, and thus blocks downstream electron transport.

## Multiple Mechanisms Protect Cells Against Damage from Reactive Oxygen Species During Photoelectron Transport

As we saw earlier in the case of mitochondria, reactive oxygen species (ROS) generated during electron transport (see Figure 12-35) can both serve as signals to regulate organelle function and cause damage to a variety of biomolecules. The same is true of chloroplasts. For example, hydrogen peroxide $\left(\mathrm{H}_{2} \mathrm{O}_{2}\right)$ regulates gas-exchange pores (stomata) in plants subjected to drought stress to prevent dehydration and controls cyclic electron flow, a process we will describe shortly.

Even though the PSI and PSII photosystems, with their associated light-harvesting complexes, are remarkably efficient at converting radiant energy to useful chemical energy in the form of ATP and NADPH, they are not perfect. Depending on the intensity of the light and the physiological conditions of the cells, a relatively small—but significant—amount of the energy absorbed by chlorophyll in the light-harvesting antennas and reaction centers results in the chlorophyll being converted to an activated state called triplet chlorophyll. In this state, the chlorophyll can transfer some of its energy to molecular oxygen $\left(\mathrm{O}_{2}\right)$, converting it from its normal, relatively unreactive ground state, called triplet oxygen $\left({ }^{3} \mathrm{O}_{2}\right)$, to
a very highly reactive (ROS) singlet state, ${ }^{1} \mathrm{O}_{2}$. Some of this ${ }^{1} \mathrm{O}_{2}$ can be used for signaling to the nucleus to communicate the metabolic state of the chloroplast to the rest of the cell. However if the majority of the ${ }^{1} \mathrm{O}_{2}$ is not quickly quenched by reacting with specialized ${ }^{1} \mathrm{O}_{2}$ "scavenger molecules," it will react with, and usually damage, nearby molecules. This damage, called photoinhibition, can suppress the efficiency of thylakoid activity.

Carotenoids (polymers of unsaturated isoprene groups, including $\beta$-carotene, which gives carrots their orange color) and $\alpha$-tocopherol (a form of vitamin E) are hydrophobic small molecules that are known to play important roles as ${ }^{1} \mathrm{O}_{2}$ quenchers to protect plants. For example, inhibition of tocopherol synthesis in the unicellular green alga Chlamydomonas reinhardtii by the herbicide pyrazolynate can result in greater light-induced photoinhibition. The carotenoids, which very efficiently siphon off energy from dangerous triplet chlorophyll when they are in close proximity to it, are the quantitatively most important molecules for preventing ${ }^{1} \mathrm{O}_{2}$ formation. The PSII monomer from the cyanobacterium Thermosynechococcus elongatus contains about 11 carotenoid molecules compared with its 35 chlorophylls, indicating the importance of carotenoid-mediated quenching of ${ }^{1} \mathrm{O}_{2}$.

Under intense illumination, photosystem PSII is especially prone to generating ${ }^{1} \mathrm{O}_{2}$, whereas PSI will produce other ROS, including superoxide, hydrogen peroxide, and hydroxyl radicals. One of the protein subunits in the PSII reaction center, called D1, is subjected to almost constant ${ }^{1} \mathrm{O}_{2}$-mediated damage even under low light conditions. A damaged reaction center moves from the grana to the unstacked regions of the thylakoid, where the D1 subunit is degraded by a protease and replaced by newly synthesized D1 protein in what is called the D1 protein damage-repair cycle. This rapid replacement of damaged D1, which requires a high rate of D1 synthesis, helps PSII recover from photoinhibition and maintain sufficient activity. An important component in the damage-repair cycle is a chaperone of the HSP70 family (see Chapter 3) called HSP70B. This chaperone binds to the damaged PSII and helps prevent loss of the other components of the complex as the D1 subunit is replaced. The extent of photoinhibition can depend on the amount of HSP70B available to the chloroplasts.

## Cyclic Electron Flow Through PSI Generates a Proton-Motive Force but No NADPH or $\mathrm{O}_{2}$

As we've seen, electrons from reduced ferredoxin in PSI are transferred to NADP ${ }^{+}$during linear electron flow, resulting in production of NADPH (see Figure 12-44). In some circumstances, however, especially when plants are stressed by conditions such as drought, high light intensity, or low carbon dioxide levels, cells must generate greater amounts of ATP relative to NADPH than they can produce by linear electron flow. To do this, they photosynthetically produce ATP from PSI without concomitant NADPH production. This is accomplished by a PSI-dependent and PSIIindependent process called cyclic photophosphorylation, or

![li994542480e](li994542480e.jpg)

FIGURE 12-45 Cyclic electron flow in plants, which generates a proton-motive force and ATP but no oxygen or NADPH. In cyclic electron flow, light energy is used by PSI to transport electrons in a cycle to generate a proton-motive force and ATP without oxidizing water or generating NADPH. High-energy electrons are transferred via the ferredoxin of PSI either to a PGR5-PGRL1 heterodimer (red, left pathway) or to the NADH dehydrogenase-like complex (blue, right pathway), where they then reduce plastoquinone (Q) to $\mathrm{QH}_{2}$. Each of these
two electron acceptors forms independent supercomplexes with PSI via light harvesting complex (LHC) subunits (yellow). (The PGR5-PGRL1 heterodimer and NADH dehydrogenase-like complex are not found together in the same supercomplex, but here are drawn together with only one PSI to emphasize the similarities of the two mechanisms of cyclic electron flow.) $\mathrm{QH}_{2}$ then transfers the electrons to the cytochrome $b f$ complex, then to plastocyanin, and finally back to PSI, as is the case for the linear electron flow pathway (see Figure 12-44).
cyclic electron flow (Figure 12-45). In this process, electrons cycle between PSI, ferredoxin, plastoquinone (Q), and the cytochrome $b f$ complex, bypassing the ferredoxin-NADP ${ }^{+}$ reductase at PSI that normally generates NADPH. Thus, during cyclic electron flow, proton pumping permits additional ATP synthesis, but no net NADPH is generated, and there is no oxidization of $\mathrm{H}_{2} \mathrm{O}$ to produce $\mathrm{O}_{2}$.

In higher plants, there are two cyclic electron flow pathways that control the ATP:NADPH ratio (see Figure 12-45). The major pathway is the PGR5-PGRL1-dependent pathway, which we shall describe shortly; this pathway ensures efficient photosynthesis and protects against stress. The minor pathway is the NADH dehydrogenase-like complexdependent pathway, which appears to respond to stress and to be a target of $\mathrm{H}_{2} \mathrm{O}_{2}$-mediated regulation. The NADH dehydrogenase-like complex is a very large multiprotein complex that is very similar in shape and composition to mitochondrial complex I (see Figure 12-22), which oxidizes NADPH or NADH while reducing Q to $\mathrm{QH}_{2}$. The NADH dehydrogenase-like complex, however, appears to lack the subunit necessary for NADPH oxidation.

During cyclic electron flow, high-energy electrons generated by light absorption and photoelectric transport in PSI are transferred either to the PGR5-PGRL1 heterodimer or to the NADH dehydrogenase-like complex from the ferredoxin subunit of PSI. Indeed, there is evidence that each of these two electron acceptors independently associates with PSI in
supercomplexes mediated by LHC subunits. Both of these electron acceptors then reduce Q to $\mathrm{QH}_{2}$, which then delivers protons and electrons to the cytochrome $b f$ complex via a Q cycle, as we described earlier for linear electron flow (see Figure 12-44). Protons are transported across the thylakoid membrane into the lumen by the cytochrome $b f$ complex and possibly by the NADH dehydrogenase-like complex. Finally, plastocyanin returns the electrons from the cytochrome $b f$ complex to PSI to complete the cycle. This cyclic electron flow is similar to the cyclical process that occurs in the single photosystem of purple bacteria (see Figure 12-43). The proton-motive force generated by cyclic electron flow drives ATP synthesis by the $\mathrm{F}_{0} \mathrm{~F}_{1}$ complex (ATP synthase) and thus increases the ATP:NADPH ratio.

## Relative Activities of Photosystems I and II Are Regulated

Chloroplasts respond to changes in the wavelengths and intensities of ambient light (as a consequence of the time of day, cloudiness, etc.) by altering the relative outputs of PSI and PSII to maintain the appropriate balance of ATP and NADPH production. In order for PSII, which is preferentially located in the stacked grana, and PSI, which is preferentially located in the unstacked thylakoid membranes, to act in sequence during linear electron flow, the amount of light energy delivered to the two reaction centers must be controlled so that each center

![li1004542480e](li1004542480e.jpg)

FIGURE 12-46 Phosphorylation of LHCII and the regulation of linear versus cyclic electron flow. (a) (Top) In normal sunlight, PSI and PSII are equally activated, and the photosystems are organized in state 1. In this arrangement, light-harvesting complex II (LHCII) is not phosphorylated, and six copies of LHCII trimers, together with several other lightharvesting proteins, encircle a dimeric PSII reaction center in a tightly associated supercomplex in the grana (for clarity, molecular details of the supercomplexes are not shown). As a result, PSII and PSI can function in parallel in linear electron flow. (Bottom) When light excitation of the two photosystems is unbalanced (e.g., too much light via PSII), LHCII becomes
phosphorylated, dissociates from PSII, and diffuses into the unstacked membranes, where it associates with PSI and its permanently associated LHCI. In this alternative supramolecular organization (state 2), most of the absorbed light energy is transferred to PSI, supporting cyclic electron flow and ATP production, but no formation of NADPH and thus no $\mathrm{CO}_{2}$ fixation. (b) Model of a PSI "super-supercomplex" involved with PGRL1dependent cyclic electron flow that was isolated from green algae in stage 2. The super-supercomplex contains multiple complexes, including the integral membrane protein PGRL1 (but not PGRS). See F. A. Wollman, 2001, EMBO J. 20:3623; and M. Iwai, et al., 2010, Nature 464:1210.
activates the same number of electrons. This balanced condition is called state 1 (Figure 12-46a). If the two photosystems are not equally excited, then cyclic electron flow occurs in PSI, and PSII becomes less active (state 2).

One mechanism underlying the regulation of the relative activities of PSI and PSII is the phosphorylation and dephosphorylation of thylakoid membrane proteins, including PSII and LHCII, mediated by several protein kinases and protein phosphatases. Changes in phosphorylation, particularly for LHCII, can alter the intra-thylakoid membrane distribution (grana vs. unstacked membranes) of this antenna complex and thus its differential interactions with PSI and PSII. The more LHCII is associated with a particular photosystem, the more efficiently that system will be activated by light, and the greater its contribution to electron flow.

Under certain conditions, LHCII's nonphosphorylated form is preferentially associated with PSII, and the phosphorylated form diffuses in the thylakoid membrane from the grana to the unstacked region and associates with PSI more than the nonphosphorylated form does. Light conditions in which there is preferential absorption of light by PSII result in
activation of an LHCII kinase, increased LHCII phosphorylation, compensatory increased activation of PSI relative to PSII, and thus an increase in cyclic electron flow in state 2 (see Figure 12-46a). When the green alga Chlamydomonas reinhardtii was forced into state 2, it was possible to isolate a "super-supercomplex" containing PSI, LHCI, LHCII, Cyt $b f$, ferredoxin (Fd), ferredoxin-NADP ${ }^{+}$reductase (FNR), and the integral membrane protein PGRL1 that participates in cyclic electron flow-but not PGR5 (Figure 12-46b). Thus it appears that the efficient operation of electron-transport chains has involved the evolution of functional complexes of increasing size and complexity, from individual proteins to complexes to supercomplexes to super-supercomplexes.

Regulating the supramolecular organization of the photosystems in plants has the effect of directing them toward ATP production (state 2) or toward the generation of reducing equivalents (NADPH) and ATP (state 1), depending on ambient light conditions and the metabolic needs of the plant. Both NADPH and ATP are required to convert $\mathrm{CO}_{2}$ to sucrose or starch, the fourth stage in photosynthesis, which we cover in the last section of this chapter.

## KEY CONCEPTS OF SECTION 12.7

## Molecular Analysis of Photosystems

- In the single photosystem of purple bacteria, cyclic electron flow from light-excited special-pair chlorophyll $a$ molecules in the reaction center generates a proton-motive force, which is used mainly to power ATP synthesis by the $\mathrm{F}_{0} \mathrm{~F}_{1}$ complex in the plasma membrane (see Figure 12-43).
- Plants contain two photosystems, PSI and PSII, which have different functions and are physically separated in the thylakoid membrane. PSII converts $\mathrm{H}_{2} \mathrm{O}$ into $\mathrm{O}_{2}$, and PSI reduces $\mathrm{NADP}^{+}$to NADPH. Cyanobacteria have two analogous photosystems.
- In chloroplasts, light energy absorbed by light-harvesting complexes (LHCs) is transferred to chlorophyll $a$ molecules in the reaction centers ( $\mathrm{P}_{680}$ in PSII and $\mathrm{P}_{700}$ in PSI).
- Electrons flow through PSII via the same carriers that are present in the bacterial photosystem. In contrast to the bacterial system, photochemically oxidized $\mathrm{P}_{680}{ }^{+}$in PSII is regenerated to $\mathrm{P}_{680}$ by electrons derived from the formation of $\mathrm{O}_{2}$ from $\mathrm{H}_{2} \mathrm{O}$ (see Figure 12-44, left).
- In linear electron flow, photochemically oxidized $\mathrm{P}_{700}{ }^{+}$ in PSI is reduced, regenerating $\mathrm{P}_{700}$, by electrons transferred from PSII via the cytochrome $b f$ complex and soluble plastocyanin. Electrons released from $\mathrm{P}_{700}$ following excitation of PSI are transported via several carriers ultimately to $\mathrm{NADP}^{+}$, generating NADPH (see Figure 12-44, right).
- The absorption of light by pigments in the chloroplast can generate reactive oxygen species (ROS), including singlet oxygen, ${ }^{1} \mathrm{O}_{2}$, and hydrogen peroxide, $\mathrm{H}_{2} \mathrm{O}_{2}$. In small amounts, they are used as intracellular signaling molecules; in larger amounts, they can be toxic. Small-molecule scavengers, such as carotenoids, and antioxidant enzymes help to protect against ROS-induced damage; however, singlet oxygen damage to the D1 subunit of PSII still occurs, causing photoinhibition. An HSP70 chaperone helps PSII recover from the damage.
- In contrast to linear electron flow, which requires both PSII and PSI, cyclic electron flow in plants involves only PSI. In this pathway, neither NADPH nor $\mathrm{O}_{2}$ is formed, although a proton-motive force is generated.
- Reversible phosphorylation and dephosphorylation of the light-harvesting complex II (LHCII) control the functional organization of the photosynthetic apparatus in thylakoid membranes. State 1 favors linear electron flow, whereas state 2 favors cyclic electron flow (see Figure 12-46). Very large supersupercomplexes appear to be involved in cyclic electron flow.


### 12.8 $\mathrm{CO}_{2}$ Metabolism During Photosynthesis

Chloroplasts perform many metabolic reactions in green leaves. In addition to $\mathrm{CO}_{2}$ fixation-incorporation of
gaseous $\mathrm{CO}_{2}$ into small organic molecules and then sugarsthe synthesis of almost all amino acids, all fatty acids and carotenes, all pyrimidines, and probably all purines occurs in chloroplasts. However, the synthesis of sugars from $\mathrm{CO}_{2}$ is the most extensively studied biosynthetic pathway in plant cells. We first consider the unique pathway, known as the Calvin cycle (after discoverer Melvin Calvin), that fixes $\mathrm{CO}_{2}$ into three-carbon compounds, powered by energy released during ATP hydrolysis and oxidation of NADPH.

## Rubisco Fixes $\mathrm{CO}_{2}$ in the Chloroplast Stroma

The enzyme ribulose 1,5-bisphosphate carboxylase, or rubisco, incorporates $\mathrm{CO}_{2}$ into precursor molecules that are subsequently converted into carbohydrates. Rubisco is located in the stroma of the chloroplast. This enzyme adds $\mathrm{CO}_{2}$ to the five-carbon sugar ribulose 1,5-bisphosphate to form two molecules of the three-carbon compound 3-phosphoglycerate (Figure 12-47). Rubisco is a large enzyme ( $\sim 500 \mathrm{kDa}$ ) whose most common form is composed of eight identical large and eight identical small subunits. One subunit is encoded in chloroplast DNA; the other, in nuclear DNA. Because the catalytic rate of rubisco is quite low, many copies of the enzyme are needed to fix sufficient $\mathrm{CO}_{2}$. Indeed, this enzyme makes up almost 50 percent of the soluble protein in a chloroplast and is believed to be the most abundant protein on Earth. It is estimated that rubisco fixes more than $10^{11}$ tons of atmospheric $\mathrm{CO}_{2}$ each year.

When photosynthetic algae are exposed to a brief pulse of ${ }^{14} \mathrm{C}$-labeled $\mathrm{CO}_{2}$ and the cells are then quickly disrupted, 3-phosphoglycerate is radiolabeled most rapidly, and all the radioactivity is found in the carboxyl group. Because $\mathrm{CO}_{2}$ is initially incorporated into a three-carbon compound, the Calvin cycle is also called the $\mathrm{C}_{3}$ pathway of carbon fixation (Figure 12-48).

The fate of 3-phosphoglycerate formed by rubisco is complex: some is converted to hexoses incorporated into starch or sucrose, but some is used to regenerate ribulose 1,5-bisphosphate. At least nine enzymes are required to regenerate ribulose 1,5-bisphosphate from 3-phosphoglycerate. For every 12 molecules of 3-phosphoglycerate generated by rubisco (a total of 36 C atoms), 2 of them ( 6 C atoms) are converted to 2 molecules of glyceraldehyde 3-phosphate (and later to 1 hexose), whereas 10 of them ( 30 C atoms) are converted to 6 molecules of ribulose 1,5-bisphosphate (Figure 12-48, top). The fixation of $6 \mathrm{CO}_{2}$ molecules and the net formation of 2 glyceraldehyde 3-phosphate molecules require the consumption of 18 ATPs and 12 NADPHs generated by the light-requiring processes of photosynthesis.

## Synthesis of Sucrose Using Fixed $\mathrm{CO}_{2}$ Is Completed in the Cytosol

After its formation in the chloroplast stroma, glyceraldehyde 3-phosphate is transported to the cytosol in exchange for phosphate. The final steps of sucrose synthesis (Figure 12-48, bottom) occur in the cytosol of leaf cells.

![li1014542480e](li1014542480e.jpg)

FIGURE 12-47 The initial reaction of rubisco that fixes $\mathrm{CO}_{2}$ into organic compounds. In this reaction, catalyzed by ribulose 1,5-bisphosphate carboxylase (rubisco), $\mathrm{CO}_{2}$ condenses with the five-carbon

A phosphate/glyceraldehyde 3-phosphate antiporter in the inner chloroplast membrane brings fixed $\mathrm{CO}_{2}$ (as glyceraldehyde 3-phosphate) into the cytosol when the cell is exporting sucrose vigorously. No fixed $\mathrm{CO}_{2}$ leaves the chloroplast unless phosphate is fed into it to replace the phosphate carried out of the stroma in the form of glyceraldehyde 3-phosphate. During the synthesis of sucrose from glyceraldehyde 3-phosphate, inorganic phosphate groups are released (Figure 12-48, bottom left). Thus the synthesis of sucrose facilitates the transport of additional glyceraldehyde 3-phosphate from the chloroplast to the cytosol by providing phosphate for the antiporter. It is worth noting that glyceraldehyde 3-phosphate is a glycolytic intermediate and that the mechanism of the conversion of glyceraldehyde 3-phosphate to hexoses is almost the reverse of that in glycolysis.

The synthesis of starch is more complex. The key monomer substrate used to build large starch polymers is ADPglucose. This polymerization takes place in the stroma, and starch polymers are stored there in densely packed crystalline aggregates called granules (see Figure 12-37). The enzymes that generate ADP-glucose from glucose 1-phosphate and ATP are found in both the stroma and the cytosol, indicating that hexoses of various structures are imported from the cytosol into the stroma for starch synthesis.

## Light and Rubisco Activase Stimulate $\mathrm{CO}_{2}$ Fixation

The Calvin cycle enzymes that catalyze $\mathrm{CO}_{2}$ fixation are rapidly inactivated in the dark, thereby conserving ATP that is generated in the dark (for example, by the breakdown of starch) for other synthetic reactions, such as lipid and amino acid biosynthesis. One mechanism that contributes to this control is the pH dependence of several Calvin cycle enzymes. Because protons are transported from the stroma into the thylakoid lumen during photoelectron transport (see Figure 12-44), the pH of the stroma increases from $\sim 7$ in the dark to $\sim 8$ in the light. The increased activity of several Calvin cycle enzymes at the higher pH promotes $\mathrm{CO}_{2}$ fixation in the light.

A stromal protein called thioredoxin (Tx) also plays a role in controlling some Calvin cycle enzymes. In the dark, thioredoxin contains a disulfide bond; in the light, electrons are transferred from PSI, via ferredoxin, to thioredoxin, reducing its disulfide bond:
![li1024542480e](li1024542480e.jpg)

Reduced thioredoxin then activates several Calvin cycle enzymes by reducing disulfide bonds in those enzymes. In the dark, when thioredoxin becomes reoxidized, these enzymes are reoxidized and so inactivated. Thus the activities of these enzymes are sensitive to the redox state of the stroma, which in turn is light sensitive-an elegant mechanism for the regulation of enzymatic activity by light.

Rubisco is one such light/redox-sensitive enzyme, although its regulation is very complex and not yet fully understood. Rubisco is spontaneously activated in the presence of high $\mathrm{CO}_{2}$ and $\mathrm{Mg}^{2+}$ concentrations. The activating reaction entails the covalent addition of $\mathrm{CO}_{2}$ to the side-chain amino group of a lysine in the enzyme's active site, forming a carbamate group that then binds a $\mathrm{Mg}^{2+}$ ion, which is required for enzymatic activity. Under normal conditions, however, with ambient levels of $\mathrm{CO}_{2}$, this reaction is slow and usually requires catalysis by rubisco activase, a member of the $\mathrm{AAA}^{+}$family of ATPases. Rubisco activase hydrolyzes ATP and uses the energy released to clear the active site of rubisco so that $\mathrm{CO}_{2}$ can be added to its active site lysine. Rubisco activase also accelerates an activating conformational change in rubisco (from an inactive-closed to an active-opened state). The regulation of rubisco activase by thioredoxin is, at least in part in some species, responsible for rubisco's light/redox sensitivity. Furthermore, rubisco activase's activity is sensitive to the ratio of ATP to ADP. If that ratio is low (relatively high ADP), then the activase will not activate rubisco (and so the cell will expend less

![li1034542480e](li1034542480e.jpg)

FIGURE 12-48 The pathway of carbon during photosynthesis. (Top) Six molecules of $\mathrm{CO}_{2}$ are converted into two molecules of glyceraldehyde 3-phosphate. These reactions, which constitute the Calvin cycle, occur in the stroma of the chloroplast. Via the phosphate/ triosephosphate antiporter, some glyceraldehyde 3-phosphate is transported to the cytosol in exchange for phosphate. (Bottom) In
the cytosol, an exergonic series of reactions converts glyceraldehyde 3-phosphate to fructose 1,6-bisphosphate. Two molecules of fructose 1,6-bisphosphate are used to synthesize one molecule of the disaccharide sucrose. Some glyceraldehyde 3-phosphate (not shown here) is also converted to amino acids and fats, compounds essential for plant growth.

of its scarce ATP to fix carbon). Photosynthesis is sensitive to a variety of typical plant stresses-moderate heat, cool temperatures, drought (limited water), high salt, high light intensity, and UV radiation. At least some of these stresses influence $\mathrm{CO}_{2}$ fixation by reducing the activity of rubisco activase and thus rubisco. Inhibition of $\mathrm{CO}_{2}$ fixation reduces consumption of NADPH. Under strong light conditions, the high NADPH/NADP ${ }^{+}$ratio can reduce electron flow to $\mathrm{NADP}^{+}$and increase leakage to $\mathrm{O}_{2}$, resulting in increased ROS formation, which can both initiate cellular signaling pathways and interfere with a variety of cellular processes. Given the key role of rubisco in controlling energy utilization and carbon flux-both in individual chloroplasts and, in a sense, throughout the entire biosphere-it is not surprising that its activity is tightly regulated.

## Photorespiration Competes with Carbon Fixation and Is Reduced in $\mathrm{C}_{4}$ Plants

As noted above, rubisco catalyzes the incorporation of $\mathrm{CO}_{2}$ into ribulose 1,5-bisphosphate as part of photosynthesis. It can catalyze a second, distinct, and competing reaction with the same substrate-ribulose 1,5-bisphosphate-but with $\mathrm{O}_{2}$ in place of $\mathrm{CO}_{2}$ as a second substrate, in a process known as photorespiration (Figure 12-49). The products of this second reaction are one molecule of 3-phosphoglycerate and one molecule of the two-carbon compound phosphoglycolate. The carbon-fixing reaction is favored when the ambient $\mathrm{CO}_{2}$ concentration is relatively high, whereas photorespiration is favored when $\mathrm{CO}_{2}$ is low and $\mathrm{O}_{2}$ is relatively
high. Photorespiration takes place in light, consumes $\mathrm{O}_{2}$, and converts ribulose 1,5-bisphosphate in part to $\mathrm{CO}_{2}$. As Figure 12-49 shows, photorespiration is wasteful to the energy economy of the plant: it consumes ATP and $\mathrm{O}_{2}$, and it generates $\mathrm{CO}_{2}$ without fixing carbon. Indeed, when $\mathrm{CO}_{2}$ is low and $\mathrm{O}_{2}$ is high, much of the $\mathrm{CO}_{2}$ fixed by the Calvin cycle is lost as the result of photorespiration. This surprising, wasteful alternative reaction catalyzed by rubisco may be a consequence of the inherent difficulty the enzyme has in specifically binding the relatively featureless $\mathrm{CO}_{2}$ molecule and of the ability of both $\mathrm{CO}_{2}$ and $\mathrm{O}_{2}$ to react and form distinct products with the same initial enzyme/ribulose 1,5-bisphosphate intermediate.

Excessive photorespiration could become a problem for plants in a hot, dry environment because they must keep the gas-exchange pores (stomata) in their leaves closed much of the time to prevent excessive loss of moisture. As a consequence, the $\mathrm{CO}_{2}$ level inside the leaf can fall below the $K_{\mathrm{m}}$ of rubisco for $\mathrm{CO}_{2}$. Under these conditions, the rate of photosynthesis is slowed, photorespiration is greatly favored, and the plant might be in danger of fixing inadequate amounts of $\mathrm{CO}_{2}$. Corn, sugarcane, crabgrass, and other plants that can grow in hot, dry environments have evolved a way to avoid this problem by using a two-step pathway of $\mathrm{CO}_{2}$ fixation in which a $\mathrm{CO}_{2}$-hoarding step precedes the Calvin cycle. This pathway has been named the $\mathrm{C}_{4}$ pathway because $\left[{ }^{14} \mathrm{C}\right] \mathrm{CO}_{2}$ labeling showed that the first radioactive molecules formed during photosynthesis in this pathway are four-carbon compounds, such as oxaloacetate and malate, rather than the three-carbon molecules that initiate the Calvin cycle ( $\mathrm{C}_{3}$ pathway).
![li1044542480e](li1044542480e.jpg)

FIGURE 12-49 $\mathrm{CO}_{2}$ fixation and photorespiration. These competing pathways are both initiated by ribulose 1,5-bisphosphate carboxylase (rubisco), and both use ribulose 1,5-bisphosphate. $\mathrm{CO}_{2}$ fixation (pathway $\mathbf{\square}$ ) is favored by high $\mathrm{CO}_{2}$ and low $\mathrm{O}_{2}$ concentrations; photorespiration (pathway $\mathbf{\square}$ ) occurs at low $\mathrm{CO}_{2}$ and high $\mathrm{O}_{2}$ concentrations (that is, under normal atmospheric conditions). The phosphoglycolate
produced by photorespiration is recycled via a complex set of reactions that take place in peroxisomes and in mitochondria, as well as in chloroplasts. The net result: for every two molecules of phosphoglycolate formed by photorespiration (four C atoms), one molecule of 3-phosphoglycerate is ultimately formed and recycled, and one molecule of $\mathrm{CO}_{2}$ is lost.

![li1054542480e](li1054542480e.jpg)

FIGURE 12-50 Leaf anatomy of $C_{4}$ plants and the $C_{4}$ pathway. (a) In $\mathrm{C}_{4}$ plants, bundle sheath cells line the vascular bundles containing the xylem and phloem. Mesophyll cells, which are adjacent to the substomal air spaces, can assimilate $\mathrm{CO}_{2}$ into four-carbon molecules at low ambient $\mathrm{CO}_{2}$ concentrations and deliver those molecules to the interior bundle sheath cells. The bundle sheath cells contain abundant chloroplasts and are the sites of photosynthesis and sucrose synthesis.

The $\mathrm{C}_{4}$ pathway involves two types of cells: mesophyll cells, which are adjacent to the air spaces in the leaf interior, and bundle sheath cells, which surround the vascular tissue and are sequestered away from the high oxygen levels to which mesophyll cells are exposed (Figure 12-50a). In the mesophyll cells of $\mathrm{C}_{4}$ plants, phosphoenolpyruvate, a three-carbon molecule derived from pyruvate, reacts with $\mathrm{CO}_{2}$ to generate oxaloacetate, a four-carbon compound (Figure 12-50b). The enzyme that catalyzes this reaction, phosphoenolpyruvate carboxylase, is found almost exclusively in $\mathrm{C}_{4}$ plants and, unlike rubisco, is insensitive to $\mathrm{O}_{2}$. The overall reaction

Sucrose is carried to the rest of the plant via the phloem. In $\mathrm{C}_{3}$ plants, which lack bundle sheath cells, the Calvin cycle operates in the mesophyll cells to fix $\mathrm{CO}_{2}$. (b) The key enzyme in the $\mathrm{C}_{4}$ pathway is phosphoenolpyruvate carboxylase, which assimilates $\mathrm{CO}_{2}$ to form oxaloacetate in mesophyll cells. Decarboxylation of malate or other $\mathrm{C}_{4}$ intermediates in bundle sheath cells releases $\mathrm{CO}_{2}$, which enters the standard Calvin cycle (see Figure 12-48, top).
that forms oxaloacetate from pyruvate involves the hydrolysis of one ATP and has a negative $\Delta G$. Therefore, $\mathrm{CO}_{2}$ fixation will proceed even when the $\mathrm{CO}_{2}$ concentration is low. The oxaloacetate formed in the mesophyll cells is reduced to malate, which is transferred by a special transporter to the bundle sheath cells, where the $\mathrm{CO}_{2}$ is released by decarboxylation and enters the Calvin cycle (see Figure 12-50b).

Because of the transport of $\mathrm{CO}_{2}$ from mesophyll cells, the $\mathrm{CO}_{2}$ concentration in the bundle sheath cells of $\mathrm{C}_{4}$ plants is much higher than it is in the normal atmosphere. Bundle sheath cells are also unusual in that they lack PSII and carry

out only cyclic electron flow catalyzed by PSI, so no $\mathrm{O}_{2}$ is evolved. The high $\mathrm{CO}_{2}$ and reduced $\mathrm{O}_{2}$ concentrations in the bundle sheath cells favor the fixation of $\mathrm{CO}_{2}$ by rubisco to form 3-phosphoglycerate and suppress photorespiration.

In contrast, the high $\mathrm{O}_{2}$ concentration in the atmosphere favors photorespiration in the mesophyll cells of $\mathrm{C}_{3}$ plants (pathway 2 in Figure 12-49); as a result, as much as 50 percent of the carbon fixed by rubisco may be reoxidized to $\mathrm{CO}_{2}$ in $\mathrm{C}_{3}$ plants. $\mathrm{C}_{4}$ plants are superior to $\mathrm{C}_{3}$ plants in their ability to use the available $\mathrm{CO}_{2}$ because the $\mathrm{C}_{4}$ enzyme phosphoenolpyruvate carboxylase has a higher affinity for $\mathrm{CO}_{2}$ than does rubisco. However, one ATP is converted to one AMP in the cyclical $\mathrm{C}_{4}$ process (to generate phosphoenolpyruvate from pyruvate); thus the overall efficiency of the photosynthetic production of sugars from NADPH and ATP is lower than it is in $\mathrm{C}_{3}$ plants, which use only the Calvin cycle for $\mathrm{CO}_{2}$ fixation. Nonetheless, the net rates of photosynthesis for $\mathrm{C}_{4}$ grasses, such as corn or sugarcane, can be two to three times the rates for otherwise similar $\mathrm{C}_{3}$ grasses, such as wheat, rice, or oats, owing to the elimination of losses from photorespiration.

Of the two carbohydrate products of photosynthesis, starch remains in the mesophyll cells of $\mathrm{C}_{3}$ plants and in the bundle sheath cells of $\mathrm{C}_{4}$ plants. In these cells, starch is subjected to glycolysis, mainly in the dark, forming ATP, NADH, and small molecules that are used as building blocks for the synthesis of amino acids, lipids, and other cellular constituents. Sucrose, in contrast, is exported from the photosynthetic cells and transported throughout the plant.

## KEY CONCEPTS OF SECTION 12.8

## $\mathrm{CO}_{2}$ Metabolism During Photosynthesis

- In the Calvin cycle, $\mathrm{CO}_{2}$ is incorporated, or fixed, into organic molecules in a series of reactions that occur in the chloroplast stroma. The initial reaction, catalyzed by rubisco, forms a three-carbon intermediate, 3-phosphoglycerate. Some of the glyceraldehyde 3-phosphate generated in the cycle is transported to the cytosol and converted to sucrose (see Figure 12-48).
- The light-dependent activation of several Calvin cycle enzymes and other mechanisms increase fixation of $\mathrm{CO}_{2}$ in the light. The redox state of the stroma plays a key role in this regulation, as does regulation of the activity of rubisco by rubisco activase.
- In $\mathrm{C}_{3}$ plants, a substantial fraction of the $\mathrm{CO}_{2}$ fixed by the Calvin cycle can be lost as the result of photorespiration, a wasteful reaction catalyzed by rubisco that is favored at low $\mathrm{CO}_{2}$ and high $\mathrm{O}_{2}$ levels (see Figure 12-49).
- In $\mathrm{C}_{4}$ plants, $\mathrm{CO}_{2}$ is fixed initially in the outer mesophyll cells by reaction with phosphoenolpyruvate. The fourcarbon molecules so generated are shuttled to the interior bundle sheath cells, where the $\mathrm{CO}_{2}$ is released and then used in the Calvin cycle. The rate of photorespiration in $\mathrm{C}_{4}$ plants is much lower than it is in $\mathrm{C}_{3}$ plants.


## 5 LaunchPad

macmillan learning

Visit LaunchPad to access study tools and to learn more about the content in this chapter.

- Perspectives for the Future
- Analyze the Data
- Extended References
- Additional study tools, including videos, animations, and quizzes


## Key Terms

aerobic oxidation 513
ATP synthase 551
binding-change
mechanism 554
Calvin cycle 573
carbon fixation 561
catabolism 516
$\mathrm{C}_{4}$ pathway 576
chemiosmosis 514
chlorophylls 560
citric acid cycle 533
coenzyme Q 541
cytochrome 540
electron-transport
chain 514
endosymbiont
hypothesis 552
fermentation 518
flavin adenine dinucleotide (FAD) 516
$\mathrm{F}_{0} \mathrm{~F}_{1}$ complex 552
glycolysis 516
nicotinamide adenine dinucleotide $\left(\mathrm{NAD}^{+}\right) \quad 516$
oxidative
phosphorylation 515
photoelectron
transport 565
photorespiration 576
photosynthesis 513
photosystem 563
prosthetic group 540
proton-motive force 515
Q cycle 544
reactive oxygen species 548
reduction potential 546
respiration 515
respiratory control 558
rubisco 573
substrate-level
phosphorylation 516
uncoupler 559

## Review the Concepts

1. The proton-motive force is essential for both mitochondrial and chloroplast function. What produces the protonmotive force, and what is its relationship to ATP? The compound 2,4-dinitrophenol (DNP), which was used in diet pills in the 1930s but later shown to have dangerous side effects, allows protons to diffuse across membranes. Why is it dangerous to consume DNP?
2. Mitochondria and chloroplasts are thought to have evolved from symbiotic bacteria present in nucleated cells. What is the experimental evidence from this chapter that supports this hypothesis?
3. The inner mitochondrial membrane exhibits all of the fundamental characteristics of a typical cell membrane, but

it also has several unique characteristics that are closely associated with its role in oxidative phosphorylation. What are these unique characteristics? How does each contribute to the function of the inner membrane?
4. Maximal production of ATP from glucose involves the reactions of glycolysis, the citric acid cycle, and the electrontransport chain. Which of these reactions requires $\mathrm{O}_{2}$, and why? Which, in certain organisms or physiological conditions, can proceed in the absence of $\mathrm{O}_{2}$ ?
5. Fermentation permits the continued extraction of energy from glucose in the absence of oxygen. If glucose catabolism is anaerobic, why is fermentation necessary for glycolysis to continue?
6. Describe the step-by-step process by which electrons from glucose catabolism in the cytoplasm are transferred to the electron-transport chain in the inner mitochondrial membrane. In your answer, note whether the electron transfer at each step is direct or indirect.
7. Mitochondrial oxidation of fatty acids is a major source of ATP, yet fatty acids can be oxidized elsewhere. What organelle, besides the mitochondrion, can oxidize fatty acids? What is the fundamental difference between oxidation occurring in this organelle and mitochondrial oxidation?
8. Each of the cytochromes in the mitochondrion contains prosthetic groups. What is a prosthetic group? Which type of prosthetic group is associated with the cytochromes? What property of the various cytochromes ensures unidirectional electron flow along the electron-transport chain?
9. The electron-transport chain consists of a number of multiprotein complexes, which work in conjunction to pass electrons from an electron carrier, such as NADH, to $\mathrm{O}_{2}$. What is the role of these complexes in ATP synthesis? It has been demonstrated that respiration supercomplexes contain all the protein components necessary for respiration. Why is this beneficial for ATP synthesis, and what is one way that the existence of supercomplexes has been demonstrated experimentally? Coenzyme $\mathrm{Q}(\mathrm{CoQ})$ is not a protein, but a small, hydrophobic molecule. Why is it important for the functioning of the electrontransport chain that CoQ is a hydrophobic molecule?
10. It is estimated that each electron pair donated by NADH leads to the synthesis of approximately three ATP molecules, whereas each electron pair donated by $\mathrm{FADH}_{2}$ leads to the synthesis of approximately two ATP molecules. What is the underlying reason for the difference in yield for electrons donated by $\mathrm{FADH}_{2}$ versus NADH?
11. Describe the main functions of the different components of the ATP synthase enzyme in the mitochondrion. A structurally similar enzyme is responsible for the acidification of lysosomes and endosomes. Given what you know about the mechanism of ATP synthesis, explain how this acidification might occur.
12. Much of our understanding of ATP synthase is derived from research on aerobic bacteria. What makes these organisms useful for this research? Where do the reactions of glycolysis, the citric acid cycle, and the electron-transport chain occur in these organisms? Where is the proton-motive force
generated in aerobic bacteria? What other cellular processes depend on the proton-motive force in these organisms?
13. An important function of the inner mitochondrial membrane is to provide a selectively permeable barrier to the movement of water-soluble molecules and thus to generate different chemical environments on either side of the membrane. However, many of the substrates and products of oxidative phosphorylation are water soluble and must cross the inner membrane. How does this transport occur?
14. The Q cycle plays a major role in the electron-transport chain of mitochondria, chloroplasts, and bacteria. What is the function of the Q cycle, and how does it carry out this function? What electron-transport components participate in the Q cycle in mitochondria, in purple bacteria, and in chloroplasts? 15. True or false: Since ATP is generated in chloroplasts, cells capable of undergoing photosynthesis do not require mitochondria. Explain. Name and describe the idea that explains how mitochondria and chloroplasts are thought to have originated in eukaryotic cells.
16. Write the overall reaction of oxygen-generating photosynthesis. Explain the following statement: The $\mathrm{O}_{2}$ generated by photosynthesis is simply a by-product of the pathway's generation of carbohydrates and ATP.
17. Photosynthesis can be divided into multiple stages. What are the stages of photosynthesis, and where does each occur within the chloroplast? Where is the sucrose produced by photosynthesis generated?
18. The photosystems responsible for absorption of light energy are each composed of two linked components, the reaction center and an antenna complex. What is the pigment composition and role of each component in the process of light absorption? What evidence exists that the pigments found in these components are involved in photosynthesis?
19. Photosynthesis in green and purple bacteria does not produce $\mathrm{O}_{2}$. Why? How can these organisms still use photosynthesis to produce ATP? What molecules serve as electron donors in these organisms?
20. Chloroplasts contain two photosystems. What is the function of each? For linear electron flow, diagram the flow of electrons from photon absorption to NADPH formation. What does the energy stored in the form of NADPH synthesize?
21. The Calvin cycle reactions that fix $\mathrm{CO}_{2}$ do not function in the dark. What are the likely reasons for this? How are these reactions regulated by light?
22. Rubisco, which may be the most abundant protein on Earth, plays a key role in the synthesis of carbohydrates in organisms that use photosynthesis. What is rubisco, where is it located, and what function does it serve?

## References

First Step of Harvesting Energy from Glucose: Glycolysis
Berg, J., J. Tymoczko, and L. Stryer. 2012. Biochemistry, 7th ed. W. H. Freeman and Company. Chaps. 16-20.

Dasgupta, T., et al. 2014. A fundamental trade-off in covalent switching and its circumvention by enzyme bifunctionality in glucose homeostasis. J. Biol. Chem. 289:13010-13025.

Depre, C., M. Rider, and L. Hue. 1998. Mechanisms of control of heart glycolysis. Eur. J. Biochem. 258:277-290.

Fersht, A. 1999. Structure and Mechanism in Protein Science: A Guide to Enzyme Catalysis and Protein Folding. W. H. Freeman and Company.

Forbergill-Gilmore, L. A., and P. A. Michels. 1993. Evolution of glycolysis. Prog. Biophys. Mol. Biol. 59:105-135.

Nelson, D. L., and M. M. Cox. 2000. Lehninger Principles of Biochemistry. Worth. Chaps. 14-17, 19.

Pilkis, S. J., et al. 1995. 6-Phosphofructo-2-kinase/fructose-2,6bisphosphatase: a metabolic signaling enzyme. Annu. Rev. Biochem. 64:799-835.

## The Structure and Functions of Mitochondria

Ahn, C. S., and C. M. Metallo. 2015. Mitochondria as biosynthetic factories for cancer proliferation. Cancer Metab. J3:1.

Bonawitz, N. D., D. A. Clayton, and G. S. Shadel. 2006. Initiation and beyond: multiple functions of the human mitochondrial transcription machinery. Mol. Cell 24:813-825.

Canfield, D. E. 2005. The early history of atmospheric oxygen: homage to Robert M. Garrels. Annu. Rev. Earth Pl. Sc. 33:1-36.

Friedman J. R., et al. 2011. ER tubules mark sites of mitochondrial division. Science 334:358.

Giorgi, C., et al. 2015. Mitochondria-associated membranes: composition, molecular mechanisms, and physiopathological implications. Antioxid. Redox. Signal. 22:995-1019.

Kamer, K. J., Y. Sancak, and V. K. Mootha. 2014. The uniporter: from newly identified parts to function. Biochem. Bioph. Res. Co. 449:370-372.

Kaufman, R. J., and J. D. Malhotra. 2014. Calcium trafficking integrates endoplasmic reticulum function with mitochondrial bioenergetics. Biochim. Biophys. Acta 1843:2233-2239.

Las, G., and O. S. Shirihai. 2014. Miro1: new wheels for transferring mitochondria. EMBO J. 33:939-941.

Mishra, P., and D. C. Chan. 2014. Mitochondrial dynamics and inheritance during cell division, development and disease. Nat. Rev. Mol. Cell Biol. 15:634-646.

Miyawaki, A., et al.1997. Fluorescent indicators for $\mathrm{Ca}^{2+}$ based on green fluorescent proteins and calmodulin. Nature 388:882-887.

Song, M., and G. W. Dorn II. 2015. Mitoconfusion: noncanonical functioning of dynamism factors in static mitochondria of the heart. Cell Metab. 21:195-205.

Spät, A., et al. 2008. High- and low-calcium-dependent mechanisms of mitochondrial calcium signalling. Cell Calcium 44:51-63.

Tan, A. S., et al. 2015. Mitochondrial genome acquisition restores respiratory function and tumorigenic potential of cancer cells without mitochondrial DNA. Cell Metab. 21:81-94.

Vance, J. E. 2014. MAM (mitochondria-associated membranes) in mammalian cells: lipids and beyond. Biochim. Biophys. Acta 1841:595-609.
van der Merwe, C., et al. 2015. Evidence for a common biological pathway linking three Parkinson's disease-causing genes: parkin, PINK1 and DJ-1. Eur. J. Neurosci. 41:1113-1125.

Wang, X., and H. H. Gerdes. 2015. Transfer of mitochondria via tunneling nanotubes rescues apoptotic PC12 cells. Cell Death Differ. 22:1181-1191.

## The Citric Acid Cycle and Fatty Acid Oxidation

Canfield, D. E. 2005. The early history of atmospheric oxygen: homage to Robert M. Garrels. Annu. Rev. Earth Pl. Sc. 33:1-36.

Chan, D. C. 2006. Mitochondria: dynamic organelles in disease, aging, and development. Cell 125(7):1241-1252.

Eaton, S., K. Bartlett, and M. Pourfarzam. 1996. Mammalian mitochondrial beta-oxidation. Biochem. J. 320 (Part 2):345-557.

Guest, J. R., and G. C. Russell. 1992. Complexes and complexities of the citric acid cycle in Escherichia coli. Curr. Top. Cell. Regul. 33:231-247.

Krebs, H. A. 1970. The history of the tricarboxylic acid cycle. Perspect. Biol. Med. 14:154-170.

Rasmussen, B., and R. Wolfe. 1999. Regulation of fatty acid oxidation in skeletal muscle. Annu. Rev. Nutr. 19:463-484.

Velot, C., et al. 1997. Model of a quinary structure between Krebs TCA cycle enzymes: a model for the metabolon. Biochemistry 36:14271-14276.

Wanders, R. J., and H. R. Waterham. 2006. Biochemistry of mammalian peroxisomes revisited. Annu. Rev. Biochem. 75:295-332.

## The Electron-Transport Chain and Generation of the ProtonMotive Force

Acin-Pérez, R., et al. 2008. Respiratory active mitochondrial supercomplexes. Mol. Cell 32:529-539.

Babcock, G. 1999. How oxygen is activated and reduced in respiration. P. Natl. Acad. Sci. USA 96:12971-12973.

Beinert, H., R. Holm, and E. Münck. 1997. Iron-sulfur clusters: nature's modular, multipurpose structures. Science 277:653-659.

Brandt, U. 2006. Energy converting NADH:quinone oxidoreductase (complex I). Annu. Rev. Biochem. 75:165-187.

Brandt, U., and B. Trumpower. 1994. The protonmotive Q cycle in mitochondria and bacteria. Crit. Rev. Biochem. Mol. 29:165-197.

Daiber, A. 2010. Redox signaling (cross-talk) from and to mitochondria involves mitochondrial pores and reactive oxygen species. Biochim. Biophys. Acta 6-7:897-906.

Darrouzet, E., et al. 2001. Large scale domain movement in cytochrome bc1: a new device for electron transfer in proteins. Trends Biochem. Sci. 26:445-451.

Dickinson, B. C., D. Srikun, and C. J. Chang. 2010. Mitochondrialtargeted fluorescent probes for reactive oxygen species. Curr. Opin. Chem. Biol. 14:50-56.

Dudkina, N. V., I. M. Folea, and E. J. Boekema. 2015. Towards structural and functional characterization of photosynthetic and mitochondrial supercomplexes. Micron 72:39-51.

Efremov, R. G., R. Baradaran, and L. A. Sazanov. 2010. The architecture of respiratory complex I. Nature 465:441-445.

Finkel, T. 2011. Signal transduction by reactive oxygen species. J. Cell Biol. 194:7-15.

Grigorieff, N. 1999. Structure of the respiratory
NADH:ubiquinone oxidoreductase (complex I). Curr. Opin. Struct. Biol. 9:476-483.

Hosler, J. P., S. Ferguson-Miller, and D. A. Mills. 2006. Energy transduction: proton transfer through the respiratory complexes. Annu. Rev. Biochem. 75:165-187.

Hunte, C., V. Zickermann, and U. Brandt. 2010. Functional modules and structural basis of conformational coupling in mitochondrial complex I. Science 329:448-451.

Hyde, B. B., G. Twig, and O. S. Shirihai. 2010. Organellar vs cellular control of mitochondrial dynamics. Semin. Cell Dev. Biol. 21:575-581.

Koopman, W. J., et al. 2010. Mammalian mitochondrial complex I: biogenesis, regulation, and reactive oxygen species generation. Antioxid. Redox Signal. 12:1431-1470.

Michel, H., et al. 1998. Cytochrome c oxidase. Annu. Rev. Bioph. Biom. 27:329-356.

Mitchell, P. 1979. Keilin's respiratory chain concept and its chemiosmotic consequences. Science 206:1148-1159. (Nobel Prize lecture.)

Murphy, M. P. 2009. How mitochondria produce reactive oxygen species. Biochem. J. 417:1-13.

Ramirez, B. E., et al. 1995. The currents of life: the terminal electron-transfer complex of respiration. P. Natl. Acad. Sci. USA 92:11949-11951.

Ruitenberg, M., et al. 2002. Reduction of cytochrome $c$ oxidase by a second electron leads to proton translocation. Nature 417:99-102.

Saraste, M. 1999. Oxidative phosphorylation at the fin de siècle. Science 283:1488-1492.

Schafer, E., et al. 2006. Architecture of active mammalian respiratory chain supercomplexes. J. Biol. Chem. 281(22):15370-15375.

Schultz, B., and S. Chan. 2001. Structures and proton-pumping strategies of mitochondrial respiratory enzymes. Annu. Rev. Bioph. Biom. 30:23-65.

Sheeran, F. L., and S. Pepe. 2006. Energy deficiency in the failing heart: linking increased reactive oxygen species and disruption of oxidative phosphorylation rate. Biochim. Biophys. Acta 1757(5-6):543-552.

Sies, H. 2014. Role of metabolic $\mathrm{H}_{2} \mathrm{O}_{2}$ generation: redox signaling and oxidative stress. J. Biol. Chem. 289:8735-8741.

Tsukihara, T., et al. 1996. The whole structure of the 13-subunit oxidized cytochrome $c$ oxidase at $2.8 \AA$. Science 272:1136-1144.

Walker, J. E. 1995. Determination of the structures of respiratory enzyme complexes from mammalian mitochondria. Biochim. Biophys. Acta 1271:221-227.

Wallace, D. C. 2005. A mitochondrial paradigm of metabolic and degenerative diseases, aging, and cancer: a dawn for evolutionary medicine. Annu. Rev. Genet. 39:359-407.

Xia, D., et al. 1997. Crystal structure of the cytochrome $b c_{1}$ complex from bovine heart mitochondria. Science 277:60-66.

Zaslavsky, D., and R. Gennis. 2000. Proton pumping by cytochrome oxidase: progress and postulates. Biochim. Biophys. Acta 1458:164-179.

Zhang, M., E. Mileykovskaya, and W. Dowhan. 2005. Cardiolipin is essential for organization of complexes III and IV into a supercomplex in intact yeast mitochondria. J. Biol. Chem. 280(33):29403-29408.

Zhang, Z., et al. 1998. Electron transfer by domain movement in cytochrome $b c_{1}$. Nature 392:677-684.

## Harnessing the Proton-Motive Force to Synthesize ATP

Aksimentiev, A., et al. 2004. Insights into the molecular mechanism of rotation in the $\mathrm{F}_{0}$ sector of ATP synthase. Biophys. J. 86(3):1332-1344.

Allegretti, M., et al. 2015. Horizontal membrane-intrinsic $\alpha$-helices in the stator $\alpha$-subunit of an F-type ATP synthase. Nature 521:237-240.

Bianchet, M. A., et al. 1998. The $2.8 \AA$ structure of rat liver $\mathrm{F}_{1}$-ATPase: configuration of a critical intermediate in ATP synthesis/ hydrolysis. P. Natl. Acad. Sci. USA 95:11065-11070.

Boyer, P. D. 1997. The ATP synthase-a splendid molecular machine. Annu. Rev. Biochem. 66:717-749.

Capaldi, R., and R. Aggeler. 2002. Mechanism of the $\mathrm{F}_{0} \mathrm{~F}_{1}$-type ATP synthase-a biological rotary motor. Trends Biochem. Sci. 27:154-160.

Elston, T., H. Wang, and G. Oster. 1998. Energy transduction in ATP synthase. Nature 391:510-512.

Hinkle, P. C. 2005. P/O ratios of mitochondrial oxidative phosphorylation. Biochim. Biophys. Acta 1706(1-2):1-11.

Junge, W., and N. Nelson. 2015. ATP synthase. Annu. Rev. Biochem. 84:631-657.

Junge, W., S. Hendrik, and S. Engelbrecht. 2009. Torque generation and elastic power transmission in the rotary $\mathrm{F}_{0} \mathrm{~F}_{1}$-ATPase. Nature 459:364-370.

Kinosita, K., et al. 1998. $\mathrm{F}_{1}$-ATPase: a rotary motor made of a single molecule. Cell 93:21-24.

Klingenberg, M., and S. Huang. 1999. Structure and function of the uncoupling protein from brown adipose tissue. Biochim. Biophys. Acta 1415:271-296.

Nury, H., et al. 2006. Relations between structure and function of the mitochondrial ADP/ATP carrier. Annu. Rev. Biochem. 75:713-741.

Oliveira, A. S., et al. 2014. Exploring $\mathrm{O}_{2}$ diffusion in A-type cytochrome $c$ oxidases: molecular dynamics simulations uncover two alternative channels towards the binuclear site. PLoS Comput. Biol. 10:e1004010.

Rosen, E. D., and B. M. Spiegelman. 2014. What we talk about when we talk about fat. Cell 156:20-44.

Sharma, V., et al. 2015. Role of subunit III and its lipids in the molecular mechanism of cytochrome $c$ oxidase. Biochim. Biophys. Acta 1847:690-697.

Tsunoda, S., et al. 2001. Rotation of the c subunit oligomer in fully functional $\mathrm{F}_{0} \mathrm{~F}_{1}$ ATP synthase. P. Natl. Acad. Sci. USA 98:898-902.

Vercesi, A. E., et al. 2006. Plant uncoupling mitochondrial proteins. Annu. Rev. Plant Biol. 57:383-404.
von Ballmoos, C., A. Wiedenmann, and P. Dimroth. 2009. Essentials for ATP synthesis by $\mathrm{F}_{1} \mathrm{~F}_{0}$ ATP synthases. Annu. Rev. Biochem. 78:649-672.

Wu, J., H. Jun, and J. R. McDermott. 2015. Formation and activation of thermogenic fat. Trends Genet. 31:232-238.

Yasuda, R., et al. 2001. Resolution of distinct rotational substeps by submillisecond kinetic analysis of $\mathrm{F}_{1}$-ATPase. Nature 410:898-904.

## Photosynthesis and Light-Absorbing Pigments

Bendich, A. J. 2004. Circular chloroplast chromosomes: the grand illusion. Plant Cell 16:1661-1666.

Ben-Shem, A., F. Frolow, and N. Nelson. 2003. Crystal structure of plant photosystem I. Nature 426(6967):630-635.

Blankenship, R. E. 2002. Molecular Mechanisms of Photosynthesis. Blackwell.

Deisenhofer, J., and J. R. Norris, eds. 1993. The Photosynthetic Reaction Center. Vols. 1 and 2. Academic Press.

McDermott, G., et al. 1995. Crystal structure of an integral membrane light-harvesting complex from photosynthetic bacteria. Nature 364:517.

Nelson, N., and C. F. Yocum. 2006. Structure and function of photosystems I and II. Annu. Rev. Plant Biol. 57:521-565.

Prince, R. 1996. Photosynthesis: the Z-scheme revisited. Trends Biochem. Sci. 21:121-122.

Wollman, F. A. 2001. State transitions reveal the dynamics and flexibility of the photosynthetic apparatus. EMBO J. 20:3623-3630.

## Molecular Analysis of Photosystems

Allen, J. F. 2002. Photosynthesis of ATP—electrons, proton pumps, rotors, and poise. Cell 110:273-276.

Amunts, A., et al. 2010. Structure determination and improved model of plant photosystem I. J. Biol. Chem. 285:3478-3486.

Aro, E. M., I. Virgin, and B. Andersson. 1993. Photoinhibition of photosystem II: Inactivation, protein damage, and turnover. Biochim. Biophys. Acta 1143:113-134.

Deisenhofer, J., and H. Michel. 1989. The photosynthetic reaction center from the purple bacterium Rhodopseudomonas viridis. Science 245:1463-1473. (Nobel Prize lecture.)

Deisenhofer, J., and H. Michel. 1991. Structures of bacterial photosynthetic reaction centers. Annu. Rev. Cell Biol. 7:1-23.

Dekker, J. P., and E. J. Boekema. 2005. Supramolecular organization of thylakoid membrane proteins in green plants. Biochim. Biophys. Acta 1706(1-2):12-39.

Finazzi, G. 2005. The central role of the green alga Chlamydomonas reinhardtii in revealing the mechanism of state transitions J. Exp. Bot. 56(411):383-388.

Guskov, A., et al. 2010. Recent progress in the crystallographic studies of photosystem II. ChemPhysChem. 11(6):1160-1171.

Haldrup, A., et al. 2001. Balance of power: a view of the mechanism of photosynthetic state transitions. Trends Plant Sci. 6:301-305.

Hänkamer, B., J. Barber, and E. Boekema. 1997. Structure and membrane organization of photosystem II from green plants. Annu. Rev. Plant Phys. 48:641-672.

Heathcote, P., P. Fyfe, and M. Jones. 2002. Reaction centres: the structure and evolution of biological solar power. Trends Biochem. Sci. 27:79-87.

Horton, P., A. Ruban, and R. Walters. 1996. Regulation of light harvesting in green plants. Annu. Rev. Plant Phys. 47:655-684.

Iwai, M., et al. 2010. Isolation of the elusive supercomplex that drives cyclic electron flow in photosynthesis. Nature 464: $1210-1213$.

Joliot, P., and A. Joliot. 2005. Quantification of cyclic and linear flows in plants. P. Natl. Acad. Sci. USA 102(13):4913-4918.

Jordan, P., et al. 2001. Three-dimensional structure of cyanobacterial photosystem I at $2.5 \AA$ resolution. Nature 411:909-917.

Kühlbrandt, W. 2001. Chlorophylls galore. Nature 411:896-898.
Martin, J. L., and M. H. Vos. 1992. Femtosecond biology. Annu. Rev. Bioph. Biom. 21:199-222.

Nelson, N., and W. Junge. 2015. Structure and energy transfer in photosystems of oxygenic photosynthesis. Annu. Rev. Biochem. 84:659-683.

Penner-Hahn, J. 1998. Structural characterization of the Mn site in the photosynthetic oxygen-evolving complex. Struct. Bond. 90:1-36.

Shikanai, T. 2014. Central role of cyclic electron transport around photosystem I in the regulation of photosynthesis. Curr. Opin. Biotech. 26:25-30.

Suga, M., et al. 2015. Native structure of photosystem II at 1.95 Å resolution viewed by femtosecond X-ray pulses. Nature 517:99-103.

Tomizioli, M., et al. 2014. Deciphering thylakoid subcompartments using a mass spectrometry-based approach. Mol. Cell. Proteomics 13:2147-2167.

Tommos, C., and G. Babcock. 1998. Oxygen production in nature: a light-driven metalloradical enzyme process. Accounts Chem. Res. 31:18-25.

## $\mathrm{CO}_{2}$ Metabolism During Photosynthesis

Buchanan, B. B. 1991. Regulation of $\mathrm{CO}_{2}$ assimilation in oxygenic photosynthesis: the ferredoxin/thioredoxin system. Perspective on its discovery, present status, and future development. Arch. Biochem. Biophys. 288:1-9.

Gutteridge, S., and J. Pierce. 2006. A unified theory for the basis of the limitations of the primary reaction of photosynthetic $\mathrm{CO}_{2}$ fixation: was Dr. Pangloss right? P. Natl. Acad. Sci. USA 103:72037204.

Mueller-Cajar, O., M. Stotz, and A. Bracher. 2014. Maintaining photosynthetic $\mathrm{CO}_{2}$ fixation via protein remodelling: the Rubisco activases. Photosynth. Res. 119:191-201.

Portis, A. 1992. Regulation of ribulose 1,5-bisphosphate carboxylase/oxygenase activity. Annu. Rev. Plant Phys. 43:415-437.

Rawsthorne, S. 1992. Towards an understanding of $\mathrm{C}_{3}-\mathrm{C}_{4}$ photosynthesis. Essays Biochem. 27:135-146.

Rokka, A., I. Zhang, and E.-M. Aro. 2001. Rubisco activase: an enzyme with a temperature-dependent dual function? Plant J. 25:463-472.

Sage, R., and J. Colemana. 2001. Effects of low atmospheric $\mathrm{CO}_{2}$ on plants: more than a thing of the past. Trends Plant Sci. 6:18-24.

Schneider, G., Y. Lindqvist, and C. I. Branden. 1992. Rubisco: structure and mechanism. Annu. Rev. Bioph. Biom. 21:119-153.

Tcherkez, G. G., G. D. Farquhar, and T. J. Andrews. 2006. Despite slow catalysis and confused substrate specificity, all ribulose bisphosphate carboxylases may be nearly perfectly optimized. $P$. Natl. Acad. Sci. USA 103(19):7246-7251.

Wolosiuk, R. A., M. A. Ballicora, and K. Hagelin. 1993. The reductive pentose phosphate cycle for photosynthetic $\mathrm{CO}_{2}$ assimilation: enzyme modulation. FASEB J. 7:622-637.

![li1064542480e](li1064542480e.jpg)

A three-dimensional reconstruction of the internal membranes of a yeast cell using scanning electron microscopy. The cell wall has been removed and the organelles highlighted with false color to reveal the endoplasmic reticulum (yellow), mitochondria (red), and nucleus (blue). Cell diameter is $3.5 \mu \mathrm{~m}$. (From Wei, D. et al., "High-resolution threedimensional reconstruction of a whole yeast cell using focused-ion beam scanning electron microscopy," Biotechniques, 2012, 53(1):41-48.)

A typical mammalian cell contains up to 10,000 different kinds of proteins; a yeast cell, about 5000 . The vast majority of these proteins are synthesized by cytosolic ribosomes, and many remain within the cytosol (see Chapter 5). However, as many as half of the different kinds of proteins produced in a typical cell are delivered to one or another of the various membrane-bounded organelles within the cell or to the cell surface. For example, many receptor proteins and transport proteins must be delivered to the plasma membrane, some water-soluble enzymes such as RNA and DNA polymerases must be targeted to the nucleus, and components of the extracellular matrix, as well as digestive enzymes and polypeptide signaling molecules, must be directed to the cell surface for secretion from the cell. These and all the other proteins produced by a cell

## Moving Proteins into Membranes and Organelles

must reach their correct locations for the cell to function properly.

The delivery of newly synthesized proteins to their proper cellular destinations, usually referred to as protein targeting or protein sorting, encompasses two very different kinds of processes: signal-based targeting and vesicle-based trafficking. The first kind of process involves the targeting of a newly synthesized protein from the cytoplasm to an intracellular organelle. Targeting can occur during translation or soon after synthesis of the protein is complete. For membrane proteins, targeting leads to insertion of the protein into the lipid bilayer of the membrane, whereas for watersoluble proteins, targeting leads to translocation of the entire protein across the membrane into the aqueous interior of the organelle. Proteins are sorted to the endoplasmic reticulum

### 13.1 Targeting Proteins To and Across the ER Membrane

### 13.2 Insertion of Membrane Proteins into the ER

13.3 Protein Modifications, Folding, and Quality Control in the ER

### 13.4 Targeting of Proteins to Mitochondria and Chloroplasts

13.5 Targeting of Peroxisomal Proteins
13.6 Transport Into and Out of the Nucleus

![li1074542480e](li1074542480e.jpg)

FIGURE 13-1 Overview of major protein-sorting pathways in eukaryotes. All nuclear DNA-encoded mRNAs are translated on cytosolic ribosomes. Right (nonsecretory pathways): Synthesis of proteins lacking an ER signal sequence is completed on free ribosomes (step 1). Those proteins that contain no targeting sequence are released into the cytosol and remain there (step 2). Proteins with an organelle-specific targeting sequence (pink) are first released into the cytosol (step 3) but are then imported into mitochondria, chloroplasts, peroxisomes, or the nucleus (step 4-5). Mitochondrial and chloroplast proteins typically pass through the outer and inner membranes to enter the matrix or stromal space, respectively.
(ER), mitochondria, chloroplasts, peroxisomes, and nucleus by this general process (Figure 13-1).

The second general sorting process, known as the secretory pathway, involves transport of proteins from the ER to their final destination within membrane-enclosed vesicles. For many proteins, including those that make up the extracellular matrix, the final destination is the outside of the cell (hence the name); integral membrane proteins are also

Other proteins are sorted to other subcompartments of these organelles by additional sorting steps. Nuclear proteins enter and exit through visible pores in the nuclear envelope. Left (secretory pathway): Ribosomes synthesizing nascent proteins in the secretory pathway are directed to the rough endoplasmic reticulum (ER) by an ER signal sequence (pink; steps 1 and 2). After translation is completed on the ER, these proteins can move via transport vesicles to the Golgi complex (step 3). Further sorting delivers proteins either to the plasma membrane or to lysosomes (step 4 or 5). The vesicle-based processes underlying the secretory pathway (steps 6 and 7, shaded box) are discussed in Chapter 14.
transported to the Golgi complex, lysosomes, and plasma membrane by this process. The secretory pathway begins in the ER; thus all proteins slated to enter the secretory pathway are initially targeted to this organelle.

Targeting to the ER usually involves nascent proteins still in the process of being synthesized on a ribosome. Newly made proteins are thus extruded from the ribosome directly into the ER membrane. Once translocated across the ER membrane,

proteins are assembled into their native conformation by protein-folding catalysts present in the lumen of the ER. Indeed, the ER is the location where about one-third of the proteins in a typical cell fold into their native conformations, and most of the resident ER proteins either directly or indirectly contribute to the folding process. As part of the folding process, proteins also undergo specific post-translational modifications in the ER. These processes are monitored carefully, and only after their folding and assembly is complete are proteins permitted to be transported out of the ER to other destinations. Proteins whose final destination is the Golgi complex, lysosomes, plasma membrane, or cell exterior are transported along the secretory pathway by the action of small vesicles that bud from the membrane of one organelle and then fuse with the membrane of another (see Figure 13-1, shaded box). We discuss vesicle-based protein trafficking in the next chapter because mechanistically it differs significantly from non-vesiclebased protein targeting to intracellular organelles.

In this chapter, we examine how proteins are targeted to five intracellular organelles: the ER, mitochondria, chloroplasts, peroxisomes, and nucleus. Two features of this protein-targeting process were initially quite baffling: how a given protein could be directed to only one specific membrane, and how relatively large hydrophilic protein molecules could be translocated across a hydrophobic membrane without disrupting the function of the bilayer as a barrier to ions and small molecules. Using a combination of biochemical purification methods and genetic screens for identifying mutants unable to execute particular translocation steps, cell biologists have identified many of the cellular components required for translocation across each of the different intracellular membranes. In addition, many of the major translocation processes in the cell have been reconstituted by incorporating their purified protein components into artificial lipid bilayers. Such in vitro systems can be freely manipulated experimentally.

These studies have shown that, despite some variations, the same basic mechanisms govern protein sorting to all the various intracellular organelles. We now know, for instance, that the information to target a protein to a particular organelle destination is encoded within the amino acid sequence of the protein itself, usually within a sequence of about 20 amino acids, known generically as a targeting sequence (see Figure 13-1); these sequences are also called signal sequences or signal peptides. Such targeting sequences are usually located at the N -terminus of a protein and are thus the first part of the protein to be synthesized. More rarely, targeting sequences are located at either the C-terminus or within the interior of a protein sequence. Each organelle carries a set of receptor proteins that bind (directly or indirectly) only to specific kinds of targeting sequences, thus ensuring that the information encoded in a targeting sequence governs the specificity of targeting. Once a protein containing a targeting sequence has interacted with the corresponding receptor, the polypeptide chain is transferred to some kind of translocation channel that allows the protein to pass into or through the membrane bilayer. The unidirectional transfer of a protein
into an organelle, without its sliding back out into the cytoplasm, is usually achieved by coupling translocation to an energetically favorable process such as hydrolysis of GTP or ATP. Some proteins are subsequently sorted further to reach a subcompartment within the target organelle; such sorting depends on yet other signal sequences and other receptor proteins. Finally, targeting sequences may be removed from the mature protein by specific proteases.

For each of the protein-targeting events discussed in this chapter, we will seek to answer four fundamental questions:

1. What is the nature of the targeting sequence, and what distinguishes it from other types of targeting sequences?
2. What is the receptor for the targeting sequence?
3. What is the structure of the translocation channel that allows transfer of proteins across the membrane bilayer? In particular, is the channel so narrow that proteins can pass through only in an unfolded state, or will it accommodate folded protein domains?
4. What is the source of energy that drives unidirectional transfer across the membrane?

In the first part of the chapter, we cover targeting of proteins to the ER, including the post-translational modifications that proteins undergo as they enter the secretory pathway. Targeting of proteins to the ER is the best-understood example of protein targeting and will serve as an exemplar of the process in general. We then describe targeting of proteins to mitochondria, chloroplasts, and peroxisomes. Finally, we cover the transport of proteins into and out of the nucleus through nuclear pores.

### 13.1 Targeting Proteins To and Across the ER Membrane

All eukaryotic cells have an endoplasmic reticulum (ER). The ER is a convoluted organelle, made up of tubules and flattened sacs, whose membrane is continuous with the membrane of the nucleus. The ER usually has a very large surface area, and its membrane is where cellular lipids are synthesized (see Chapter 7). The ER is also where most membrane proteins are assembled, including those of the plasma membrane and the membranes of the lysosomes, ER, and Golgi complex. In addition, all soluble proteins that will eventually be secreted from the cell-as well as those destined for the lumen of the ER, Golgi complex, or lysosomes-are initially delivered to the ER lumen (see Figure 13-1). Since the ER plays such an important role in protein secretion, we refer to the pathway of protein trafficking that flows through the ER as the secretory pathway. For simplicity, we will refer to all proteins initially targeted to the ER as secretory proteins, but keep in mind that not all proteins that are targeted to the ER are actually secreted from the cell.

In this first section of the chapter, we discuss how proteins are initially identified as secretory proteins and how such proteins are translocated across the ER membrane.

![li1084542480e](li1084542480e.jpg)

FIGURE 13-2 Structure of the rough ER. (a) Electron micrograph of ribosomes attached to the rough ER in a pancreatic acinar cell. Most of the proteins synthesized by this type of cell are secretory proteins and are formed on membrane-attached ribosomes. A few unattached (free) ribosomes are evident; presumably, these ribosomes are synthesizing cytosolic or other nonsecretory proteins. (b) Schematic representation of protein synthesis on the ER. Note that membrane-bound and free cytosolic ribosomes are identical. Membrane-bound ribosomes are recruited to the ER during synthesis of a polypeptide containing an ER signal sequence. [Part (a) courtesy of Dr. Marilyn G. Farquhar, University of California, San Diego.]

We deal first with soluble proteins-those that pass all the way through the ER membrane, into the lumen. In the next section, we discuss integral membrane proteins, which are inserted into the ER membrane.

## Pulse-Chase Experiments with Purified ER Membranes Demonstrated That Secreted Proteins Cross the ER Membrane

Although all cells secrete a variety of proteins (e.g., extracellular matrix proteins), certain types of cells are specialized for secretion of large amounts of specific proteins. Pancreatic acinar
cells, for instance, synthesize large quantities of several digestive enzymes, which are secreted into ductules that lead to the intestine. Because such secretory cells contain the organelles of the secretory pathway (e.g., ER and Golgi complex) in great abundance, they have been widely used in studying this pathway, including the initial steps that occur at the ER membrane.

The sequence of events that occurs immediately after the synthesis of a secretory protein was first elucidated by pulse-chase experiments with pancreatic acinar cells. In these experiments, radioactively labeled amino acids were incorporated into secretory proteins as they were synthesized on ribosomes bound to the surface of the ER. The portion of the ER that receives proteins entering the secretory pathway is known as the rough $E R$ because it is so densely studded with ribosomes that its surface appears morphologically distinct from other ER membranes (Figure 13-2). From these experiments, it became clear that during or immediately after their synthesis on the ribosome, secretory proteins translocate across the ER membrane into the lumen of the ER.

To delineate the steps in the translocation process, it was necessary to isolate the ER from the rest of the cell. Isolation of intact ER, with its delicate lacelike structure and its interconnectedness with other organelles, is not feasible. However, scientists discovered that when cells are homogenized, the rough ER breaks up into small closed vesicles with ribosomes on the outside, termed microsomes, which retain most of the biochemical properties of the ER, including the capability of protein translocation. The experiments depicted in Figure 13-3, in which microsomes isolated from pulse-labeled cells were treated with a protease, demonstrate that although secretory proteins are synthesized on ribosomes bound to the cytosolic face of the ER membrane, the polypeptides produced by these ribosomes end up within the lumen of a microsome. Experiments such as these raised the question of how polypeptides are recognized as secretory proteins shortly after their synthesis begins and how a nascent secretory protein is threaded across the ER membrane.

## A Hydrophobic N-Terminal Signal Sequence Targets Nascent Secretory Proteins to the ER

After synthesis of a secretory protein begins on free ribosomes in the cytosol, a 16-30-residue ER targeting sequence in the nascent protein directs the ribosome to the ER membrane and initiates translocation of the growing polypeptide across the ER membrane (see Figure 13-1, left). An ER targeting sequence, typically located at the N -terminus of the protein, is usually known as a signal sequence. The signal sequences of different secretory proteins all contain one or more positively charged amino acids adjacent to a continuous stretch of 6-12 hydrophobic residues (known as the hydrophobic core), but otherwise have little in common. The signal sequence is cleaved from most secretory proteins while they are still elongating on the ribosome; thus signal sequences are usually not present in the mature proteins found in cells.

The hydrophobic core of an ER signal sequence is essential for its function. For instance, the specific deletion of several of the hydrophobic amino acids from a signal

![li1094542480e](li1094542480e.jpg)

FIGURE 13-3 Secretory proteins enter the ER lumen. Labeling experiments demonstrated that secretory proteins are localized to the ER lumen shortly after synthesis. Cells are incubated for a brief time with radiolabeled amino acids so that only newly synthesized proteins become labeled. The cells are then homogenized, fracturing the plasma membrane and shearing the rough ER into small vesicles called microsomes. Because they have bound ribosomes, microsomes have a much greater buoyant density than other membranous organelles and can be separated from them by a combination of differential and sucrose density-gradient centrifugation (see Chapter 4). The purified microsomes are treated with a protease in the presence or absence of a detergent. The labeled secretory proteins associated with the microsomes are digested by the protease only if the microsomal membrane is first destroyed by treatment with detergent. This finding indicates that the newly made proteins are inside the microsomes, equivalent to the lumen of the rough ER.
sequence or the introduction of charged amino acids into the hydrophobic core by mutation can abolish the ability of the N -terminus of a protein to function as a signal sequence. As a consequence, the modified protein remains in the cytosol, unable to cross the ER membrane into the lumen. Conversely, signal sequences can be added to normally
(a) Cell-free protein synthesis; no microsomes present
![li1104542480e](li1104542480e.jpg)
(b) Cell-free protein synthesis; microsomes present
![li1114542480e](li1114542480e.jpg)

FIGURE 13-4 Translation and translocation occur simultaneously. Cell-free experiments demonstrate that translocation of secretory proteins into microsomes is coupled to translation. Treatment of microsomes with EDTA, which chelates $\mathrm{Mg}^{2+}$ ions, strips them of associated ribosomes, allowing isolation of ribosome-free microsomes, which are equivalent to ER membranes (see Figure 13-3). Protein synthesis is carried out in a cell-free system containing functional ribosomes, tRNAs, ATP, GTP, and cytosolic enzymes, to which mRNA encoding a secretory protein is added. The secretory protein is synthesized in the absence of microsomes (a) but is translocated across the vesicle membrane and loses its signal sequence (resulting in a decrease in molecular weight) only if microsomes are present during protein synthesis (b).
cytosolic proteins using recombinant DNA techniques. Provided the added sequence is sufficiently long and hydrophobic, such a modified cytosolic protein can acquire the ability to be translocated to the ER lumen. The hydrophobic residues in the core of an ER signal sequence form a binding site that is critical for the interaction of the signal sequence with the machinery responsible for targeting the protein to the ER membrane.

Biochemical studies using a cell-free protein-synthesizing system, mRNA encoding a secretory protein, and microsomes stripped of their own bound ribosomes have elucidated the function and fate of ER signal sequences. Initial experiments with this system demonstrated that a typical secretory protein is incorporated into microsomes and has its signal sequence removed only if the microsomes are present during protein synthesis. If microsomes are added to the system after protein synthesis is completed, no protein transport into the microsomes occurs (Figure 13-4).

![li1124542480e](li1124542480e.jpg)
(b) Ffh, GTPase-domain (homolog of SRP P54 subunit)
![li1134542480e](li1134542480e.jpg)

FtsY (homolog of SRP receptor $\alpha$ subunit)
between these proteins is controlled by GTP binding and hydrolysis. Ffh and FtsY each can bind to one molecule of GTP, and when they bind to each other, the two bound molecules of GTP fit in the interface between the protein subunits and stabilize the dimer. Assembly of the pseudosymmetric dimer allows formation of two active sites for the hydrolysis of both bound GTP molecules. Hydrolysis to GDP destabilizes the interface, causing disassembly of the dimer. [Part (a) data from R. J. Keenan et al., 1998, Cell 94:181, PDB ID 2ffh. Part (b) data from P. J. Focia et al., 2004, Science 303:373, PDB ID 1okk.]

Subsequent experiments were designed to determine the precise stage of protein synthesis at which microsomes must be present in order for translocation to occur. In these experiments, microsomes were added to the reaction mixtures at different times after protein synthesis had begun. These experiments showed that microsomes must be added before the first 70 or so amino acids are translated in order for the completed secretory protein to be localized in the microsomal lumen. At this point, the first 40 or so amino acids protrude from the ribosome, including the signal sequence that will later be cleaved off, and the next 30 or so amino acids are still buried within a channel in the ribosome (see Figure 5-26). Thus the transport of most secretory proteins into the ER lumen begins while the incompletely synthesized (nascent) protein is still bound to the ribosome, a process referred to as cotranslational translocation.

## Cotranslational Translocation Is Initiated by Two GTP-Hydrolyzing Proteins

Given that secretory proteins are synthesized in association with the ER membrane but not with any other cellular
membrane, a signal-sequence recognition mechanism must target them there. The two key components in this targeting are the signal recognition particle (SRP) and its receptor. The SRP is a cytosolic ribonucleoprotein particle that transiently binds to both the ER signal sequence in a nascent protein and the large (60S) ribosomal subunit, forming a large complex. The SRP then targets the nascent protein-ribosome complex to the ER membrane by binding to the SRP receptor, which is located in the membrane.

The SRP is made up of six proteins bound to a 300 nucleotide RNA, which acts as a scaffold for the hexamer. One of the SRP proteins (P54) can be chemically cross-linked to ER signal sequences, which shows that this subunit is the one that binds to the signal sequence in a nascent secretory protein. A region of P54 known as the M domain, containing many methionine and other amino acid residues with hydrophobic side chains, contains a cleft whose inner surface is lined by hydrophobic side chains (Figure 13-5a). The hydrophobic core of the signal sequence binds to this cleft via hydrophobic interactions. Other polypeptides in the SRP interact with the ribosome or are required for protein translocation into the ER lumen.

![li1144542480e](li1144542480e.jpg)

FIGURE 13-6 Cotranslational translocation. Steps 1-4 Once the ER signal sequence emerges from the ribosome, it is bound by a signal recognition particle (SRP). Step 1. The SRP and the nascent polypeptide chain-ribosome complex bind to the SRP receptor in the ER membrane. This interaction is strengthened by the binding of GTP to both the SRP and its receptor. Step 2 Transfer of the nascent polypeptide-ribosome to the translocon leads to opening of this translocation channel to admit the growing polypeptide adjacent to the signal sequence, which is transferred to a hydrophobic binding site next to the central pore. Both the SRP and SRP receptor, once dissociated from the translocon, hydrolyze their bound GTP and
then are ready to initiate the insertion of another polypeptide chain. Step 3 As the polypeptide chain elongates, it passes through the translocon channel into the ER lumen, where the signal sequence is cleaved by signal peptidase and is rapidly degraded. Step 4. The peptide chain continues to elongate as the mRNA is translated toward the $3^{\prime}$ end. Because the ribosome is attached to the translocon, the growing chain is extruded through the translocon into the ER lumen. Steps 5-6 Once translation is complete, the ribosome is released, the remainder of the protein is drawn into the ER lumen, the translocon closes, and the protein assumes its native folded conformation.

The SRP and the nascent polypeptide chain-ribosome complex bind to the ER membrane by docking with the SRP receptor, an integral protein of the ER membrane made up of two subunits: an $\alpha$ subunit and a smaller $\beta$ subunit. Interaction of the SRP-nascent chain-ribosome complex with the SRP receptor is strengthened when both the P54 subunit of the SRP and the $\alpha$ subunit of the SRP receptor are bound to GTP. The structure of the SRP P54 subunit and the SRP receptor $\alpha$ subunit (FtsY) from the archaean Thermus aquaticus provides insight into how a cycle of GTP binding and hydrolysis can drive the binding and dissociation of these proteins. Figure 13-5b shows that P54 and FtsY, each bound to a single molecule of GTP, come together to form a pseudosymmetric heterodimer. Neither subunit alone contains a complete active site for the hydrolysis of GTP, but when the two proteins come together, they form two complete active sites that are capable of hydrolyzing both bound GTP molecules.

Figure 13-6 summarizes our current understanding of secretory protein synthesis and the role of the SRP and its receptor in this process. Hydrolysis of the bound GTP accompanies disassembly of the SRP and SRP receptor and initiates transfer of the nascent chain and ribosome to a site on the ER membrane, where translocation can take place. After dissociating from each other, the SRP and its receptor each release their bound GDP, SRP is recycled back to the cytosol, and both are ready to initiate another round of interaction between ribosomes synthesizing nascent secretory proteins and the ER membrane.

## Passage of Growing Polypeptides Through the Translocon Is Driven by Translation

Once the SRP and its receptor have targeted a ribosome synthesizing a secretory protein to the ER membrane, the ribosome and nascent polypeptide chain are rapidly

![li1154542480e](li1154542480e.jpg)

FIGURE 13-7 Sec61 $\alpha$ is a translocon component. Cross-linking experiments show that Sec61 $\alpha$ is a translocon component that contacts nascent secretory proteins as they pass into the ER lumen. An mRNA encoding the N -terminal 70 amino acids of the secreted protein prolactin was translated in a cell-free system containing microsomes (see Figure 13-4b). The mRNA lacked a chain-termination codon and contained one lysine codon, near the middle of the sequence. The reaction mixtures contained a chemically modified lysyl-tRNA in which a light-activated cross-linking reagent was attached to the lysine side chain. Although the entire mRNA was translated, the completed polypeptide could not be released from the ribosome without a chain-termination codon and thus became "stuck" crossing the ER membrane. The reaction mixtures were then exposed to intense light, which caused the nascent polypeptide chain to become covalently bound to whatever proteins were near it in the translocon. When the experiment was performed using microsomes from mammalian cells, the nascent chain became covalently linked to Sec61 $\alpha$. Different versions of the prolactin mRNA were created so that the modified lysine residue would be placed at different distances from the ribosome; cross-linking to Sec61 $\alpha$ was observed only when the modified lysine was positioned within the translocation channel. See T. A. Rapoport, 1992, Science 258:931 and D. Görlich and T. A. Rapoport, 1993, Cell 75:615.
transferred to the translocon, a complex of proteins that forms a channel embedded within the ER membrane. As translation continues, the elongating chain passes directly from the large ribosomal subunit into the central pore of the translocon. The large ribosomal subunit is aligned
with the pore of the translocon in such a way that the growing chain is never exposed to the cytoplasm and is prevented from folding until it reaches the ER lumen (see Figure 13-6).

The translocon was first identified through mutations in the yeast gene encoding a protein called Sec61 $\alpha$, which caused a block in the translocation of secretory proteins into the lumen of the ER. Subsequently, three proteins, collectively called the Sec61 complex, were found to form the mammalian translocon: Sec61 $\alpha$, an integral membrane protein with 10 membrane-spanning $\alpha$ helices, and two smaller proteins, termed Sec61 $\beta$ and Sec61 $\gamma$. Chemical cross-linking experiments in a cell-free translocation system-in which amino acid side chains from a nascent secretory protein became covalently attached to the Sec61 $\alpha$ subunit—demonstrated that the translocating polypeptide chain comes into contact with the Sec61 $\alpha$ protein, confirming its identity as the translocon pore (Figure 13-7).

When microsomes in the cell-free translocation system were replaced with reconstituted phospholipid vesicles containing only the SRP receptor and the Sec61 complex, nascent secretory proteins were translocated from their SRP-ribosome complex into the vesicles. This finding indicates that the SRP receptor and the Sec61 complex are the only ER-membrane proteins that are absolutely required for translocation. Because neither of these proteins can hydrolyze ATP or otherwise provide energy to drive ongoing translocation, the energy derived from chain elongation at the ribosome appears to be sufficient to push the polypeptide chain across the membrane in one direction.

The translocon must be able to allow passage of a polypeptide chain while remaining sealed to small molecules, such as ATP, in order to maintain the permeability barrier of the ER membrane. Furthermore, there must be some way to regulate the translocon so that it is closed in its default state, opening only when a nascent polypeptide chainribosome complex is bound. A high-resolution structural model of the archaeal Sec61 complex shows how the translocon preserves the integrity of the membrane (Figure 13-8). The 10 transmembrane helices of Sec61 $\alpha$ form a central channel through which the translocating polypeptide chain passes. Two different gating steps are required for Sec61 $\alpha$ to accept a translocating polypeptide. The 10 transmembrane helices are organized into two 5-helix bundles. In the first gating step, the bundles hinge apart like an opening clamshell to expose a hydrophobic binding pocket for the hydrophobic core of the signal sequence at the open edge. The signal sequence binds to Sec61 $\alpha$ with its N -terminus facing the cytosol and the elongating polypeptide doubling back through the central channel. The structural model of the Sec61 complex, which was isolated without a translocating peptide and is therefore presumed to be in a closed conformation, reveals a short helical peptide plugging the central channel. Biochemical studies of the Sec61 complex have shown that, in the absence of a translocating polypeptide, the peptide that forms the plug effectively seals the translocon to prevent the passage of ions and small molecules.

![li1164542480e](li1164542480e.jpg)

In the second gating step, after the signal sequence has bound to the opened channel, the translocating peptide enters the central pore of the channel, forcing away the plug peptide and allowing translocation to proceed. The middle of the central pore is lined with hydrophobic isoleucine residues that in effect form a gasket, preventing leakage of small polar molecules around the translocating peptide even as translocation proceeds.

As the growing polypeptide chain enters the lumen of the ER, the signal sequence is cleaved by signal peptidase, which is a transmembrane ER protein associated with the translocon (see Figure 13-6, step $\mathbf{\square}$ ). Signal peptidase recognizes a sequence on the C-terminal end of the hydrophobic core of the signal peptide and cleaves the chain specifically at this sequence once it has emerged into the luminal space of the ER. After the signal sequence has been cleaved, the growing polypeptide moves through the translocon into

FIGURE 13-8 Structure of an archaeal Sec61 complex. The structure of the detergent-solubilized Sec61 complex from the archaeon M. jannaschii (also known as the SecY complex) was determined by x-ray crystallography. (a) A side view shows the hourglass-shaped channel through the center of the pore. A ring of isoleucine residues at the constricted waist of the pore forms a gasket that keeps the channel sealed to small molecules even as a translocating polypeptide passes through the channel. When no translocating peptide is present, the channel is closed by a short helical plug (red). This plug moves out of the channel during translocation. In this view, the front half of protein has been removed to better show the pore. (b) A view looking through the center of the channel shows a region (on the left side) where helices may separate, allowing lateral passage of a hydrophobic transmembrane domain into the lipid bilayer. [Data from B. Van den Berg et al., 2004, Nature 427:36-44, PDB ID 1rhz and 1rh5.]
the ER lumen. The translocon remains open until translation is complete and the entire polypeptide chain has moved into the ER lumen. After translocation is complete, the plug peptide returns to the pore to reseal the translocon channel.

## ATP Hydrolysis Powers Post-translational Translocation of Some Secretory Proteins in Yeast

In most eukaryotes, secretory proteins enter the ER by cotranslational translocation. In yeast, however, some secretory proteins enter the ER lumen after translation has been completed. In such post-translational translocation, the translocating protein passes through the same Sec61 translocon that is used in cotranslational translocation. However, the SRP and SRP receptor are not involved in post-translational translocation, and in such cases a direct interaction between the translocon and the signal sequence of the completed protein appears to be sufficient for targeting to the ER membrane. In addition, the driving force for unidirectional translocation across the ER membrane is provided by an additional protein complex known as the Sec63 complex and a member of the Hsp70 family of molecular chaperones known as BiP (see Chapter 3 for further discussion of molecular chaperones). The tetrameric Sec63 complex is embedded in the ER membrane in the vicinity of the translocon, whereas BiP is within the ER lumen. Like other members of the Hsp70 family, BiP has a peptide-binding domain and an ATPase domain. These chaperones bind and stabilize unfolded or partially folded proteins (see Figure 3-17).

The current model for post-translational translocation of a protein into the ER is outlined in Figure 13-9. Once the N -terminal segment of the protein enters the ER lumen, signal peptidase cleaves the signal sequence just as in cotranslational translocation (step $\mathbf{\square}$ ). Interaction of BiP-ATP with the luminal portion of the Sec63 complex causes hydrolysis of the bound ATP, producing a conformational change in BiP that promotes its binding to an exposed polypeptide chain (step $\mathbf{\square}$ ). Since the Sec63 complex is located near

![li1174542480e](li1174542480e.jpg)

FIGURE 13-9 Post-translational translocation. This mechanism is fairly common in yeast and probably occurs occasionally in higher eukaryotes. Small arrows inside the translocon represent random sliding of the translocating polypeptide inward and outward. Successive binding of BiP-ATP to entering segments of the polypeptide prevents the chain from sliding out toward the cytosol. See K. E. Matlack et al., 1997, Science 277:938.
the translocon, BiP is thus activated at sites where nascent polypeptides can enter the ER. Certain experiments suggest that, in the absence of binding to BiP, an unfolded polypeptide can freely slide back and forth within the translocon channel. Such random sliding motions rarely result in the entire polypeptide's crossing the ER membrane. Binding of a molecule of BiP$\cdot$ADP to the luminal portion of the polypeptide prevents backsliding of the polypeptide out of the ER. As further inward random sliding exposes more of the polypeptide on the luminal side of the ER membrane, successive binding of BiP-ADP molecules to the polypeptide chain acts as a ratchet, ultimately drawing the entire polypeptide into the ER within a few seconds (steps $\mathbf{\square}$ and $\mathbf{\square}$ ). On a slower time scale, the BiP molecules spontaneously exchange their bound ADP for ATP, leading to release of the polypeptide, which can then fold into its native conformation (steps $\mathbf{\square}$ and $\mathbf{\square}$ ). The recycled BiP-ATP is then ready for another interaction with Sec63. BiP and the Sec63 complex are also required for cotranslational translocation. The details of their role in this process are not well understood, but they are thought to act at an early stage of the process,
such as the threading of the signal peptide into the pore of the translocon.

The overall reaction carried out by BiP is an important example of how the chemical energy released by the hydrolysis of ATP can power the mechanical movement of a protein across a membrane. Bacterial cells also use an ATP-driven process for translocating completed proteins across the plasma membrane-in this case to be released from the cell. In bacteria, the driving force for translocation comes from a cytosolic ATPase known as the SecA protein. SecA binds to the cytoplasmic side of the translocon and hydrolyzes cytosolic ATP. By a mechanism that resembles the needle on a sewing machine, the SecA protein pushes segments of the polypeptide through the membrane in a mechanical cycle coupled to the hydrolysis of ATP.

As we will see, translocation of proteins across other eukaryotic organelle membranes, such as those of mitochondria and chloroplasts, also typically occurs by posttranslational translocation. This explains why ribosomes are typically not found bound to these other organelles, as they are to the rough ER.

## KEY CONCEPTS OF SECTION 13.1

## Targeting Proteins To and Across the ER Membrane

- Synthesis of secreted proteins, integral plasma-membrane proteins, and proteins destined for the ER, Golgi complex, or lysosome begins on cytosolic ribosomes, which become attached to the membrane of the ER, forming the rough ER (see Figure 13-1, left).
- The ER signal sequence on a nascent secretory protein is located at the N -terminus and contains a sequence of hydrophobic amino acids.
- In cotranslational translocation, the signal recognition particle (SRP) first recognizes and binds the ER signal sequence on a nascent secretory protein, then is bound in turn by an SRP receptor on the ER membrane, thereby targeting the nascent polypeptide chain-ribosome complex to the ER.
- The SRP and SRP receptor then mediate insertion of the nascent secretory protein into the translocon (Sec61 complex). Hydrolysis of two molecules of GTP by the SRP and its receptor cause the dissociation of SRP (see Figures 13-5 and 13-6). As the ribosome attached to the translocon continues translation, the unfolded protein chain is extruded into the ER lumen. No additional energy is required for translocation.
- The translocon contains a central channel lined with hydrophobic residues that allows transit of an unfolded protein chain while remaining sealed to ions and small hydrophilic molecules. In addition, the channel is gated so that it is open only when a polypeptide is being translocated.
- In post-translational translocation, a completed secretory protein is targeted to the ER membrane by interaction of the signal sequence with the translocon. The polypeptide chain is then pulled into the ER by a ratcheting mechanism that requires ATP hydrolysis by the chaperone BiP, which stabilizes the entering polypeptide (see Figure 13-9). In bacteria, the driving force for post-translational translocation comes from SecA, a cytosolic ATPase that pushes polypeptides through the translocon channel.
- In both cotranslational and post-translational translocation, a signal peptidase in the ER membrane cleaves the ER signal sequence from a secretory protein soon after the N -terminus enters the lumen.


### 13.2 Insertion of Membrane Proteins into the ER

In previous chapters, we have encountered many of the vast array of integral membrane (transmembrane) proteins that are present throughout the cell. Each such protein has a unique orientation with respect to the membrane's phospholipid bilayer. Integral membrane proteins located in the ER, Golgi complex, and lysosomes, as well as in the
plasma membrane, which are all synthesized on the rough ER, remain embedded in the membrane as they move to their final destinations along the same pathway that is followed by soluble secretory proteins (see Figure 13-1, left). During this transport, the orientation of a membrane protein is preserved; that is, the same segments of the protein always face the cytosol, whereas other segments always face in the opposite direction. Thus the final orientation of these membrane proteins is established during their biosynthesis on the ER membrane. In this section, we first see how integral membrane proteins interact with membranes and then examine how several types of sequences, known collectively as topogenic sequences, direct the membrane insertion and orientation of various classes of integral membrane proteins. These processes occur via modifications of the basic mechanism used to translocate soluble secretory proteins across the ER membrane.

## Several Topological Classes of Integral Membrane Proteins Are Synthesized on the ER

The topology of a membrane protein refers to the number of times its polypeptide chain spans the membrane and the orientation of those membrane-spanning segments within the membrane. The key elements of a protein that determine its topology are the membrane-spanning segments themselves, which are usually $\alpha$ helices containing 20-25 hydrophobic amino acids that contribute to energetically favorable interactions within the hydrophobic interior of the phospholipid bilayer.

Most integral membrane proteins fall into one of the five topological classes illustrated in Figure 13-10. Topological classes I, II, III, and the tail-anchored proteins are singlepass membrane proteins, which have only one membranespanning $\alpha$-helical segment. Type I proteins have a cleaved N -terminal ER signal sequence and are anchored in the membrane with their hydrophilic N -terminal region on the luminal face (also known as the exoplasmic face) and their hydrophilic C-terminal region on the cytosolic face. Type II proteins do not contain a cleavable ER signal sequence and are oriented with their hydrophilic N -terminal region on the cytosolic face and their hydrophilic C-terminal region on the exoplasmic face (i.e., opposite to type I proteins). Type III proteins have a hydrophobic membrane-spanning segment at their N -terminus and thus have the same orientation as type I proteins, but do not contain a cleavable signal sequence. Finally, tail-anchored proteins have a hydrophobic segment at their C -terminus that spans the membrane. These different topologies reflect distinct mechanisms used by the cell to establish the orientation of transmembrane segments, as we will see shortly.

The proteins forming topological class IV contain two or more membrane-spanning segments and are sometimes called multipass membrane proteins. For example, many of the membrane transport proteins discussed in Chapter 11 and the numerous G protein-coupled receptors covered in Chapter 15 belong to this class.

![li1184542480e](li1184542480e.jpg)

FIGURE 13-10 Classes of ER membrane proteins. Five topological classes of integral membrane proteins are synthesized on the rough ER, as is a sixth type tethered to the membrane by a phospholipid anchor. These membrane proteins are classified by their orientation in the membrane and the types of signals they contain to direct them there. In the integral membrane proteins, hydrophobic segments of the protein chain form $\alpha$ helices embedded in the membrane bilayer; the regions outside the membrane are hydrophilic
and fold into various conformations. All type IV proteins have multiple transmembrane $\alpha$ helices. The type IV topology depicted here corresponds to that of G protein-coupled receptors: seven $\alpha$ helices, the N -terminus on the exoplasmic side of the membrane, and the C -terminus on the cytosolic side. Other type IV proteins may have a different number of helices and various orientations of the N -terminus and C-terminus. See E. Hartmann et al., 1989, P. Natl. Acad. Sci. USA 86:5786, and C. A. Brown and S. D. Black, 1989, J. Biol. Chem. 264:4442.

Some lipid-anchored membrane proteins are also synthesized on the ER. These membrane proteins lack a hydrophobic membrane-spanning segment altogether; instead, they are linked to an amphipathic phospholipid anchor that is embedded in the membrane (Figure 13-10, right).

## Internal Stop-Transfer Anchor and Signal-Anchor Sequences Determine Topology of Single-Pass Proteins

We begin our discussion of how membrane protein topology is determined with the insertion of integral membrane proteins that contain a single hydrophobic membrane-spanning segment. As we will see, three main types of topogenic sequences are used to direct proteins to the ER membrane and to orient them within it. We have already introduced one, the N -terminal signal sequence. The other two, introduced here, are internal sequences known as stop-transfer anchor sequences and signal-anchor sequences. Unlike signal sequences, these two types of internal topogenic sequences end up in the mature protein as membrane-spanning segments. However, the two types differ in their final orientation in the membrane.

Type I Proteins In addition to an N-terminal signal sequence that targets them to the ER, all type I transmembrane proteins possess an internal hydrophobic sequence of approximately

22 amino acids that becomes the membrane-spanning $\alpha$ helix. The N-terminal signal sequence of a nascent type I protein, like that of a soluble secretory protein, initiates cotranslational translocation of the protein through the combined action of the SRP and SRP receptor. Once the N-terminus of the growing polypeptide enters the lumen of the ER, the signal sequence is cleaved, and the growing polypeptide chain continues to be extruded across the ER membrane. However, when the sequence that will become a transmembrane domain enters the translocon, it stops transfer of the protein through the channel by allowing the transmembrane segment to move laterally from the channel into the membrane (Figure 13-11). The gating mechanism that allows lateral movement is the same as that for the opening of the translocon to accept a signal sequence: two five-helix bundles of Sec61 $\alpha$ hinge open to allow the hydrophobic transmembrane segment to move laterally past the hydrophobic signal-sequence binding site through the opened edge of the translocon (see Figure 13-8). When the peptide exits the translocon in this manner, the hydrophobicity of the transmembrane segment anchors it in the hydrophobic interior of the membrane. Because such a sequence functions both to stop passage of the polypeptide chain through the translocon and to become a hydrophobic transmembrane segment in the membrane bilayer, it is called a stop-transfer anchor sequence.

Once translocation is interrupted, translation continues at the ribosome, which is still anchored to the now

![li1194542480e](li1194542480e.jpg)

FIGURE 13-11 Membrane insertion and orientation of type I single-pass transmembrane proteins. Step After the nascent polypeptide chain-ribosome complex becomes associated with a translocon in the ER membrane, the N -terminal signal sequence is cleaved. This process occurs by the same mechanism as the one for soluble secretory proteins (see Figure 13-6). Steps - The chain is elongated until the hydrophobic stop-transfer anchor sequence is synthesized and enters the translocon, where it prevents the nascent chain from extruding farther into the ER lumen. Step $\mathbf{\square}$ The
stop-transfer anchor sequence moves laterally through a hydrophobic cleft between translocon subunits and ultimately becomes anchored in the phospholipid bilayer. At this time, the translocon probably closes. Step $\mathbf{\square}$ As synthesis continues, the elongating chain may loop out into the cytosol through the small space between the ribosome and translocon. Step When synthesis is complete, the ribosomal subunits are released into the cytosol, leaving the protein free to diffuse laterally in the membrane. See H. Do et al., 1996, Cell 85:369, and W. Mothes et al., 1997, Cell 89:523.
unoccupied and closed translocon. As the C-terminus of the protein chain is synthesized, it loops out on the cytosolic side of the membrane. When translation is complete, the ribosome is released from the translocon, and the C-terminus of the newly synthesized type I protein remains in the cytosol.

Support for this mechanism has come from studies in which cDNAs encoding various mutant receptors for human growth hormone (HGH) were expressed in cultured mammalian cells. The wild-type HGH receptor, a typical type I protein, is transported normally to the plasma membrane. However, a mutant receptor that has charged residues inserted into the single membrane-spanning segment, or that is missing most of this segment altogether, is translocated entirely into the ER lumen and is eventually secreted from the cell as a soluble protein. These kinds of experiments have established that the hydrophobic membrane-spanning segment of the HGH receptor, and of other type I proteins, functions both as a stop-transfer sequence and as a membrane anchor that prevents the C-terminus of the protein from crossing the ER membrane.

Type II and Type III Proteins Unlike type I proteins, type II and type III proteins lack a cleavable N-terminal ER signal sequence. Instead, both possess a single internal hydrophobic
signal-anchor sequence that functions as both an ER signal sequence and a membrane anchor. Recall that type II and type III proteins have opposite orientations in the membrane (see Figure 13-10); this difference depends on the orientation that their respective signal-anchor sequences assume within the translocon. The internal signal-anchor sequence in type II proteins directs insertion of the nascent polypeptide chain into the ER membrane so that the N -terminus of the chain faces the cytosol, using the same SRP-dependent mechanism described for signal sequences (Figure 13-12a). However, the internal signal-anchor sequence is not cleaved and eventually moves laterally from the signal-sequence binding site at the edge of the translocon directly into the phospholipid bilayer, where it functions as a membrane anchor. As elongation continues, the C-terminal region of the growing chain is extruded through the translocon into the ER lumen by cotranslational translocation.

In the case of type III proteins, the signal-anchor sequence, which is located near the N -terminus, directs insertion of the nascent chain into the ER membrane with its N -terminus facing the lumen, in an orientation opposite to that of the signal-anchor in type II proteins. The signalanchor sequence of type III proteins also functions like a

![li1204542480e](li1204542480e.jpg)

FIGURE 13-12 Membrane insertion and orientation of type II and type III single-pass transmembrane proteins. (a) Type II proteins. Step After the internal signal-anchor sequence is synthesized on a cytosolic ribosome, it is bound by an SRP (not shown), which binds the SRP receptor on the ER membrane. This process is similar to the targeting of soluble secretory proteins except that the hydrophobic signal sequence is not located at the N -terminus and is not subsequently cleaved. The nascent polypeptide chain becomes oriented in the translocon with its N -terminal portion toward the cytosol. This orientation is dictated by the positively charged residues shown N -terminal to the signal-anchor sequence. Step As the chain is elongated and extruded into the lumen, the internal signal-anchor sequence moves laterally
through a hydrophobic cleft between translocon subunits and anchors the chain in the phospholipid bilayer. Step Once protein synthesis is complete, the C-terminus of the polypeptide is released into the lumen, and the ribosomal subunits are released into the cytosol. (b) Type III proteins. Step Insertion is by a process similar to that of type II proteins, except that positively charged residues on the C-terminal side of the signal-anchor sequence cause the transmembrane segment to be oriented within the translocon with its C-terminal portion toward the cytosol and the N -terminal end in the ER lumen. Steps - Elongation of the C-terminal portion of the polypeptide chain is completed in the cytosol, and the ribosomal subunits are released. See M. Spiess and H. F. Lodish, 1986, Cell 44:177, and H. Do et al., 1996, Cell 85:369.
stop-transfer sequence and prevents further extrusion of the elongating chain into the ER lumen (Figure 13-12b). Continued elongation of the chain C-terminal to the signal-anchor sequence proceeds as it does for type I proteins, with the hydrophobic sequence eventually moving laterally out of the translocon to anchor the polypeptide in the ER membrane (see Figure 13-11).

The key difference between type II and type III proteins is the orientation of the hydrophobic transmembrane segment as it binds to the hydrophobic signal-sequence binding site at the edge of Sec61 $\alpha$. The most important feature of signal-anchor sequences that determines their orientation is a high density of positively charged amino acids adjacent to one end of the hydrophobic segment. These positively charged residues tend to remain on the cytosolic side of the membrane, rather than traversing the membrane into the ER lumen. Thus the position of the charged residues dictates the orientation of the signal-anchor sequence within the translocon as well as whether the rest of the polypeptide chain continues to pass into the ER lumen: type II proteins tend
to have positively charged residues on the N -terminal side of their signal-anchor sequence, orienting the N -terminus in the cytosol and allowing passage of the C-terminal side into the ER (see Figure 13-12a), whereas type III proteins tend to have positively charged residues on the C-terminal side of their signal-anchor sequence, which restrict the C-terminus to the cytosol (see Figure 13-12b). Note that the hydrophobic segment of a type II signal-anchor sequence assumes the same orientation in Sec61 $\alpha$ as the signal sequence of a secreted protein, and that in most respects these signal-anchor sequences behave exactly like signal sequences, although they are not cleaved.

A striking experimental demonstration of the importance of the flanking charge in determining orientation in the membrane is provided by neuraminidase, a type II protein in the surface coat of influenza virus. Three arginine residues are located just N -terminal to the internal signal-anchor sequence in neuraminidase. Mutation of these three positively charged residues to negatively charged glutamate residues causes neuraminidase to acquire the reverse orientation.

![li1214542480e](li1214542480e.jpg)

FIGURE 13-13 Insertion of tail-anchored proteins. For C-terminal tail-anchored proteins, the hydrophobic C-terminus is not available for membrane insertion until protein synthesis is complete and the protein has been released from the ribosome. Step $\mathbf{\square}$ Get3 in an ATPbound state binds to the hydrophobic C-terminal tail of the protein. This binding reaction is facilitated by a complex of three other proteins, Sgt2, Get4, and Get5, which sequester the hydrophobic C-terminal tail before transferring it to Get3-ATP (not shown). Step The ternary complex Get3-ATP bound to the protein docks onto the dimeric Get1/ Get2 receptor, which is embedded in the ER membrane. Step $\mathbf{\square}$ In succession, ATP is hydrolyzed and ADP is released from Get3. At the same time, the hydrophobic C-terminal tail is released from Get3 and ultimately becomes embedded in the ER membrane in a process that is facilitated by Get1/Get2. Step $\mathbf{\square}$ Get3 binds to ATP and Get3-ATP is released from Get1/Get2 in a soluble form, ready for another round of binding to a hydrophobic C-terminal tail.

Similar experiments have shown that other proteins, with either type II or type III orientation, can be made to "flip" their orientation in the ER membrane by mutating charged residues that flank the internal signal-anchor segment.

Tail-Anchored Proteins For all the topological classes of proteins we have considered so far, membrane insertion begins when the SRP recognizes a hydrophobic topogenic sequence as it emerges from the ribosome. Recognition of tail-anchored proteins, which have a single hydrophobic topogenic sequence at the C-terminus, presents a unique challenge because the hydrophobic C-terminus becomes available for recognition only after translation has been completed and the protein has been released from the ribosome. Insertion of tail-anchored proteins into the ER membrane does not employ an SRP, SRP receptor, or translocon, but instead depends on a pathway dedicated to this purpose, as depicted in Figure 13-13. This pathway involves an ATPase known as Get3, which binds to the C-terminal hydrophobic segment of a tail-anchored protein. The complex of Get3 bound to a tail-anchored protein is recruited to the

ER by a dimeric integral membrane receptor known as Get1/ Get2. The tail-anchored protein is released from Get3, and the transmembrane portion of Get1/Get2 participates in the insertion of the tail-anchor into the ER membrane. This process is mechanistically similar to the targeting of type II and type III signal-anchor sequences to the ER by the SRP and SRP receptor. A major difference between the two targeting processes is that Get3 couples targeting and transfer of tailanchored proteins to ATP hydrolysis, whereas SRP couples protein targeting to GTP hydrolysis. Moreover, the SRP receptor recruits the SRP-ribosome complex to the ER, and in a separate step, the translocon inserts the signal-anchor sequence into the membrane, whereas Get1/Get2 evidently performs both functions, recruiting Get3 to the ER membrane and catalyzing insertion of the tail-anchor into the membrane bilayer.

## Multipass Proteins Have Multiple Internal Topogenic Sequences

Figure 13-14 summarizes the arrangements of topogenic sequences in single-pass and multipass transmembrane proteins. In multipass (type IV) proteins, each of the mem-brane-spanning $\alpha$ helices acts as a topogenic sequence in the ways that we have already discussed: it can act to direct the protein to the ER, to anchor the protein in the ER membrane, or to stop transfer of the protein through the membrane. Multipass proteins fall into one of two types, depending on whether the N -terminus extends into the cytosol or the exoplasmic space (e.g., the ER lumen or cell exterior). This N-terminal topology is usually determined by the hydrophobic segment closest to the N -terminus and the charge of the sequences flanking it. If a type IV protein has an even number of transmembrane $\alpha$ helices, its N -terminus and C -terminus will be oriented toward the same side of the membrane (Figure 13-14d). Conversely, if a type IV protein has an odd number of $\alpha$ helices, its two ends will have opposite orientations (Figure 13-14e).

Type IV Proteins with N-Terminus in the Cytosol Among the multipass proteins whose N -terminus extends into the cytosol are the various glucose transporters (GLUTs) and most ion-channel proteins, discussed in Chapter 11. In these proteins, the hydrophobic segment closest to the N -terminus initiates insertion of the nascent polypeptide chain into the ER membrane with the N -terminus oriented toward the cytosol; thus this $\alpha$-helical segment functions like the internal signalanchor sequence of a type II protein (see Figure 13-12a). As the nascent chain following the first $\alpha$ helix elongates, it moves through the translocon until the second hydrophobic $\alpha$ helix is formed. This helix prevents further extrusion of the nascent chain through the translocon; thus its function is similar to that of the stop-transfer anchor sequence in a type I protein (see Figure 13-11).

After synthesis of the first two transmembrane $\alpha$ helices, both ends of the nascent chain face the cytosol, and the loop between them extends into the ER lumen. The C-terminus of

![li1224542480e](li1224542480e.jpg)
the nascent chain then continues to grow into the cytosol, as it does in synthesis of type I and type III proteins. The third transmembrane $\alpha$ helix acts as another type II signal-anchor sequence and the fourth as another stop-transfer anchor sequence (see Figure 13-14d). Apparently, once the first topogenic sequence of a multipass polypeptide initiates association with the translocon, the ribosome remains attached to the translocon, and topogenic sequences that subsequently emerge from the ribosome are threaded into the translocon without the need for the SRP and the SRP receptor. In a manner that is not well understood, as new hydrophobic topogenic sequences engage the translocon, the previously engaged sequences move laterally out of the translocon using the same mechanism as for type I, type II, and type III membrane proteins.

Experiments that use recombinant DNA techniques to exchange hydrophobic $\alpha$ helices have provided insight into the functioning of the topogenic sequences in type IV-A multipass proteins. These experiments indicate that the order of the hydrophobic $\alpha$ helices relative to one another in the growing chain largely determines whether a given helix functions as a signal-anchor sequence or a stop-transfer anchor sequence. Other than its hydrophobicity, the specific amino acid sequence of a particular helix has little bearing on its function. Thus the first N -terminal $\alpha$ helix and the subsequent odd-numbered ones function as signal-anchor sequences, whereas the intervening even-numbered helices function as stop-transfer anchor sequences. This odd-even relationship among signal-anchor and stop-transfer anchor sequences is dictated by the fact that the transmembrane $\alpha$ helices assume alternating orientations as a multipass protein is woven back and forth across the membrane; signalanchor sequences are oriented with their N-termini toward the cytoplasmic side of the bilayer, whereas stop-transfer anchor sequences have their N-termini oriented toward the exoplasmic side of the bilayer.

Type IV Proteins with N-Terminus in the Exoplasmic Space The large family of G protein-coupled receptors, all of which contain seven transmembrane $\alpha$ helices, constitute the most numerous type IV-B proteins, whose N -terminus extends into the exoplasmic space. In these proteins, the hydrophobic $\alpha$ helix closest to the N -terminus is often followed by a cluster of positively charged amino acids, like a type III signal-anchor sequence (see Figure 13-12b). As a result, the nascent polypeptide chain is inserted into the translocon with the N-terminus extending into the lumen (see Figure 13-14e). As the chain is elongated, it is inserted into the ER membrane by alternating type II signal-anchor sequences and stop-transfer sequences, as just described for type IV-A proteins.

## A Phospholipid Anchor Tethers Some CellSurface Proteins to the Membrane

Some cell-surface proteins are anchored to the phospholipid bilayer not by a sequence of hydrophobic amino

![li1234542480e](li1234542480e.jpg)

FIGURE 13-15 GPI-anchored proteins. (a) Structure of a glycosylphosphatidylinositol (GPI) molecule from yeast. The hydrophobic portion of the molecule is composed of fatty acyl chains, whereas the polar (hydrophilic) portion is composed of carbohydrate residues and phosphate groups. In other organisms, both the length of the acyl chains and the carbohydrate moieties may vary somewhat from the structure shown. (b) Formation of GPI-anchored proteins in the ER membrane. The protein is synthesized and initially inserted into the ER membrane like a type I transmembrane protein, as shown in Figure 13-11. A specific transamidase simultaneously cleaves the precursor protein within the exoplasmic-facing domain, near the stop-transfer anchor sequence (red), and transfers the carboxyl group of the new C-terminus to the terminal amino group of a preformed GPI anchor. See C. Abeijon and C. B. Hirschberg, 1992, Trends Biochem. Sci. 17:32, and K. Kodukula et al., 1992, P. Natl. Acad. Sci. USA 89:4982.
acids, but by a covalently attached amphipathic molecule, glycosylphosphatidylinositol (GPI) (Figure 13-15a; see also Chapter 7). These proteins are synthesized and initially anchored to the ER membrane exactly like type I transmembrane proteins, with a cleaved N -terminal signal sequence and an internal stop-transfer anchor sequence directing the process (see Figure 13-11). However, a short sequence of amino acids in the luminal domain, adjacent to the membrane-spanning domain, is recognized by a transamidase located within the ER membrane. This enzyme simultaneously cleaves off the original stoptransfer anchor sequence and transfers the luminal portion
of the protein to a preformed GPI anchor in the membrane (Figure 13-15b).

Why change one type of membrane anchor for another? Attachment of the GPI anchor, which results in removal of the cytosol-facing hydrophilic domain from the protein, can have several consequences. Proteins with GPI anchors, for example, can diffuse relatively rapidly in the plane of the phospholipid bilayer. In contrast, many proteins anchored by membrane-spanning $\alpha$ helices are impeded from moving laterally in the membrane because their cytosolfacing segments interact with the cytoskeleton. In addition, the GPI anchor targets the attached protein to the apical domain of the plasma membrane (instead of the basolateral domain) in certain polarized epithelial cells, as we discuss in Chapter 14.

## The Topology of a Membrane Protein Can Often Be Deduced from Its Sequence

As we have seen, various topogenic sequences in integral membrane proteins synthesized on the ER govern the interaction of the nascent polypeptide chain with the translocon. When scientists begin to study a protein of unknown function, the identification of potential topogenic sequences within the corresponding gene sequence can provide important clues about the protein's topological class and function. Suppose, for example, that the gene for a protein known to be required for a cell-to-cell signaling pathway contains nucleotide sequences that encode an apparent N -terminal signal sequence and an internal hydrophobic sequence. These findings suggest that the protein is a type I integral membrane protein and therefore may be a cell-surface receptor for an extracellular ligand. Furthermore, the implied type I topology suggests that the N -terminal segment that lies between the signal sequence and the internal hydrophobic sequence constitutes the extracellular domain, which probably has a part in ligand binding, whereas the C-terminal segment that lies after the internal hydrophobic sequence is probably cytosolic and may have a part in intracellular signaling.

Identification of topogenic sequences requires a way to scan sequence databases for segments that are sufficiently hydrophobic to be either signal sequences or transmembrane anchor sequences. Topogenic sequences can often be identified with the aid of computer programs that generate a hydropathy profile for the protein of interest. The first step is to assign a value known as the hydropathic index to each amino acid in the protein. By convention, hydrophobic amino acids are assigned positive values and hydrophilic amino acids negative values. Although different scales for the hydropathic index exist, all assign the most positive values to amino acids with side chains made up of mostly hydrocarbon residues (e.g., phenylalanine and methionine) and the most negative values to charged amino acids (e.g., arginine and aspartate). The second step is to identify long segments of sufficient overall hydrophobicity to be N-terminal signal sequences or internal stop-transfer anchor sequences

![li1244542480e](li1244542480e.jpg)
(b) Asialoglycoprotein receptor (type II)
![li1254542480e](li1254542480e.jpg)

FIGURE 13-16 Hydropathy profiles. Hydropathy profiles can identify likely topogenic sequences in integral membrane proteins. They are generated by plotting the total hydrophobicity of each segment of 20 contiguous amino acids along the length of a protein. Positive values indicate relatively hydrophobic portions of the protein;
(c) GLUT1 (type IV)
![li1264542480e](li1264542480e.jpg)
negative values, relatively polar portions. Probable topogenic sequences are marked. The complex profiles for multipass (type IV) proteins, such as GLUT1 in part (c), must often be supplemented with other analyses to determine the topology of these proteins.
and signal-anchor sequences. To accomplish this, the total hydropathic index for each segment of 20 consecutive amino acids is calculated along the entire length of the protein. Plots of these calculated values against position in the amino acid sequence yield a hydropathy profile.

Figure 13-16 shows the hydropathy profiles for three different membrane proteins. The prominent peaks in such plots identify probable topogenic sequences as well as their positions and approximate lengths. For example, the hydropathy profile of the human growth hormone receptor reveals the presence of both a hydrophobic signal sequence at the extreme N-terminus of the protein and an internal hydrophobic stop-transfer anchor sequence (Figure 13-16a). On the basis of this profile, we can deduce, correctly, that the HGH receptor is a type I integral membrane protein. The hydropathy profile of the asialoglycoprotein receptor, a cell-surface protein that mediates removal of abnormal extracellular glycoproteins, reveals a prominent internal hydrophobic signal-anchor sequence, but gives no indication of a hydrophobic N-terminal signal sequence (Figure 13-16b). Thus we can predict that the asialoglycoprotein receptor is a type II or type III membrane protein. The distribution of charged residues on either side of the signal-anchor sequence can often differentiate between these possibilities, since positively charged amino acids flanking a membranespanning segment are usually oriented toward the cytosolic face of the membrane. For instance, in the case of the asialoglycoprotein receptor, examination of the residues flanking
the signal-anchor sequence reveals that the residues on the N-terminal side carry a net positive charge, thus correctly predicting that this is a type II protein.

The hydropathy profile of the GLUT1 glucose transporter, a multipass transmembrane protein, shows the presence of many segments that are sufficiently hydrophobic to be membrane-spanning helices (Figure 13-16c). The complexity of this profile illustrates the difficulty both in unambiguously identifying all the membrane-spanning segments in a multipass protein and in predicting the topology of individual signal-anchor and stop-transfer anchor sequences. More sophisticated computer algorithms have been developed that take into account the presence of positively charged amino acids adjacent to hydrophobic segments as well as the length of and spacing between segments. Using all this information, the best algorithms can predict the complex topology of multipass proteins with an accuracy of greater than 75 percent.

Finally, sequence homology to a known protein may permit accurate prediction of the topology of a newly discovered multipass protein. For example, the genomes of multicellular organisms encode a very large number of multipass proteins with seven transmembrane $\alpha$ helices. The similarities between the sequences of these proteins strongly suggest that all have the same topology as the well-studied G protein-coupled receptors, which have the N-terminus oriented to the exoplasmic side and the C-terminus oriented to the cytosolic side of the membrane.

## KEY CONCEPTS OF SECTION 13.2

## Insertion of Membrane Proteins into the ER

- Proteins synthesized on the rough ER include five topological classes of integral membrane proteins as well as a lipid-anchored type (see Figure 13-10).
- Topogenic sequences-N-terminal signal sequences, internal stop-transfer anchor sequences, and internal signalanchor sequences-direct the insertion of nascent proteins into the ER membrane and their orientation within it. This orientation is retained during transport of the completed membrane protein to its final destination-e.g., the plasma membrane.
- Single-pass membrane proteins contain one or two topogenic sequences. In multipass membrane proteins, each $\alpha$-helical segment can function as an internal topogenic sequence, depending on its location in the polypeptide chain and the presence of adjacent positively charged residues (see Figure 13-14).
- Some cell-surface proteins are initially synthesized as type I proteins, but then cleaved, and their luminal domains transferred to a GPI anchor (see Figure 13-15).
- The topology of membrane proteins can often be correctly predicted by computer programs that identify hydrophobic topogenic segments within the amino acid sequence and generate hydropathy profiles (see Figure 13-16).


### 13.3 Protein Modifications, Folding, and Quality Control in the ER

Membrane and soluble secretory proteins synthesized on the rough ER undergo four principal modifications before they reach their final destinations: (1) covalent addition and processing of carbohydrates (glycosylation) in the ER and Golgi complex, (2) formation of disulfide bonds in the ER, (3) proper folding of polypeptide chains and assembly of multisubunit proteins in the ER, and (4) specific proteolytic cleavages in the ER, Golgi complex, and secretory vesicles. Generally speaking, these modifications promote the folding of secretory proteins into their native structures and add structural stability to proteins exposed to the extracellular environment. Modifications such as glycosylation also allow the cell to produce a vast array of chemically distinct molecules at the cell surface that are the basis of specific molecular interactions used in cell-to-cell adhesion and communication.

The majority of proteins that are synthesized on the rough ER and enter the lumen of the ER are modified by the addition of one or more carbohydrate chains. Proteins with attached carbohydrates are known as glycoproteins. Carbohydrate chains in glycoproteins attached to the hydroxyl
$(-\mathrm{OH})$ group in serine and threonine residues are referred to as O-linked oligosaccharides, and carbohydrate chains attached to the amide nitrogen of asparagine are referred to as $N$-linked oligosaccharides. The various types of $O$-linked oligosaccharides include the mucin-type O-linked chains (named after the abundant glycoproteins found in mucus) and the carbohydrate modifications on proteoglycans described in Chapter 20. O-linked chains typically consist of only one to four sugar residues in a linear chain, which are added to proteins by enzymes known as glycoslytransferases, located in the lumen of the Golgi complex. The more common N -linked oligosaccharides are larger and more complex, containing several branches. In this section, we focus on N -linked oligosaccharides, whose initial synthesis occurs in the ER. After the initial N -glycosylation of a protein in the ER, the oligosaccharide chain is modified in the ER and commonly in the Golgi complex as well.

Disulfide bond formation, protein folding, and assembly of multimeric proteins, which take place exclusively in the rough ER, are also discussed in this section. Only properly folded and assembled proteins are transported from the rough ER to the Golgi complex and ultimately to the cell surface or other final destination through the secretory pathway. Unfolded, misfolded, or partly folded and assembled proteins are selectively retained in the rough ER and marked for degradation. We consider several features of such "quality control" in the latter part of this section.

As discussed previously, N-terminal ER signal sequences are cleaved from soluble secretory proteins and type I membrane proteins in the ER. Some proteins also undergo other specific proteolytic cleavages in the Golgi complex or secretory vesicles. We cover these cleavages, as well as carbohydrate modifications that occur primarily or exclusively in the Golgi complex, in the next chapter.

## A Preformed $N$-Linked Oligosaccharide Is Added to Many Proteins in the Rough ER

Biosynthesis of all N -linked oligosaccharides begins in the rough ER with a preformed oligosaccharide precursor containing 14 residues (Figure 13-17). The structure of this precursor is the same in plants, animals, and single-celled eukaryotes: a branched oligosaccharide, containing three glucose (Glc), nine mannose (Man), and two N -acetylglucosamine (GlcNAc) molecules, which can be written as $\mathrm{Glc}_{7} \mathrm{Man}_{9}(\mathrm{GlcNAc})_{2}$. Once added to a protein, this branched carbohydrate structure is modified by addition or removal of monosaccharides in the ER and Golgi complex. The modifications to N -linked chains differ from one glycoprotein to another and among different organisms, but a core of 5 of the 14 residues is conserved in the structures of all N -linked oligosaccharides on secretory and membrane proteins.

Prior to transfer to a nascent chain in the lumen of the ER, the oligosaccharide precursor is assembled on a mem-brane-attached anchor called dolichol phosphate, a longchain polyisoprenoid lipid (see Chapter 7). After the first

![li1274542480e](li1274542480e.jpg)

FIGURE 13-17 Biosynthesis of the oligosaccharide precursor.
Dolichol phosphate is a strongly hydrophobic lipid, containing 75-95 carbon atoms, that is embedded in the ER membrane. Two N -acetylglucosamine (GlcNAc) and five mannose residues are added one at a time to a dolichol phosphate on the cytosolic face of the ER membrane (steps $\mathbf{\square} \mathbf{\square}$. The nucleotide-sugar donors in these and later reactions are synthesized in the cytosol. Note that the first sugar residue is attached to dolichol by a high-energy pyrophosphate linkage. Tunicamycin, which blocks the first enzyme in this pathway, inhibits the synthesis
of all N -linked oligosaccharides in cells. After the seven-residue dolichol pyrophosphoryl intermediate is flipped to the luminal face (step $\mathbf{\square}$ ), the remaining four mannose residues and all three glucose residues are added one at a time (steps $\mathbf{\square} \mathbf{\square}$. In the later reactions, the sugar to be added is first transferred from a nucleotide sugar to a carrier dolichol phosphate on the cytosolic face of the ER; the carrier is then flipped to the luminal face, where the sugar is transferred to the growing oligosaccharide, after which the "empty" carrier is flipped back to the cytosolic face. See C. Abeijon and C. B. Hirschberg, 1992, Trends Biochem. Sci. 17:32.
sugar, GlcNAc, is attached to the dolichol phosphate by a pyrophosphate bond, the other sugars are added by glycosidic bonds in a complex set of reactions catalyzed by enzymes attached to the cytosolic or luminal faces of the rough ER membrane (see Figure 13-17). The final dolichol pyrophosphoryl oligosaccharide is oriented so that the oligosaccharide portion faces the ER lumen.

The entire 14-residue precursor is transferred from the dolichol carrier to an asparagine residue on a nascent polypeptide as it emerges into the ER lumen (Figure 13-18, step $\mathbf{\square}$ ). Only asparagine residues in the tripeptide sequences Asn-X-Ser and Asn-X-Thr (where X is any amino acid except proline) are substrates for oligosaccharyl transferase, the enzyme that catalyzes this reaction. Two of the three subunits of this enzyme are ER membrane proteins whose cytosol-facing domains bind to the ribosome, localizing a third subunit of the transferase, the catalytic subunit, near the growing polypeptide chain in the ER lumen. Not all Asn-X-Ser/Thr sequences become glycosylated, and it is not possible to predict from the amino acid sequence alone which potential N -linked glycosylation sites will be modified; for instance, rapid folding of a segment of a protein containing an Asn-X-Ser/Thr sequence may prevent transfer of the oligosaccharide precursor to it.

Immediately after the entire precursor, $\mathrm{Glc}_{3} \mathrm{Man}_{9}(\mathrm{GlcNAc})_{2}$, is transferred to a nascent polypeptide, three different enzymes, called glycosidases, remove all three glucose residues and one particular mannose residue (Figure 13-18, steps $\mathbf{\square} \mathbf{\square}$ ).
The three glucose residues, which are the last residues added during synthesis of the precursor on the dolichol carrier, appear to act as a signal that the oligosaccharide is complete and ready to be transferred to a protein.

## Oligosaccharide Side Chains May Promote Folding and Stability of Glycoproteins

The oligosaccharides attached to glycoproteins serve various functions. For example, some proteins require N -linked oligosaccharides in order to fold properly in the ER. This function has been demonstrated in studies with the antibiotic tunicamycin, which blocks the first step in the formation of the dolichollinked oligosaccharide precursor and therefore inhibits synthesis of all N -linked oligosaccharides in cells (see Figure 13-17, top left). For example, in the presence of tunicamycin, the influenza virus hemagglutinin precursor polypeptide $\left(\mathrm{HA}_{0}\right)$ is synthesized, but it cannot fold properly and form a normal trimer; in this case, the protein remains, misfolded, in the rough ER. Moreover, mutation of a particular asparagine in the hemagglutinin sequence to a glutamine residue prevents addition of an N -linked oligosaccharide to that site and causes the protein to accumulate in the ER in an unfolded state.

In addition to promoting proper folding, N -linked oligosaccharides confer stability on many secreted glycoproteins. Many secretory proteins fold properly and are transported to their final destination even if the addition of all N -linked

![li1284542480e](li1284542480e.jpg)

FIGURE 13-18 Addition and initial processing of $N$-linked oligosaccharides. In the rough ER of vertebrate cells, the $\mathrm{Glc}_{2} \mathrm{Man}_{5}(\mathrm{GlcNAc})_{2}$ precursor is transferred from the dolichol carrier to a susceptible asparagine residue on a nascent protein as soon as the asparagine crosses to the luminal side of the ER (step 1). In three separate reactions, first one glucose residue (step 1), then two glucose residues (step 1), and finally one mannose residue (step 1)
are removed. Re-addition of one glucose residue (step 1) plays a role in the correct folding of many proteins in the ER, as discussed later. The process of N -linked glycosylation of a soluble secretory protein is shown here, but the luminal portions of an integral membrane protein can be modified on asparagine residues by the same mechanism. See R. Kornfeld and S. Kornfeld, 1985, Annu. Rev. Biochem. 45:631, and M. Sousa and A. J. Parodi, 1995, EMBO J. 14:4196.
oligosaccharides is blocked, for example, by treatment with tunicamycin. However, such nonglycosylated proteins have been shown to be less stable than their glycosylated forms. For instance, glycosylated fibronectin, a normal component of the extracellular matrix, is degraded much more slowly by tissue proteases than is nonglycosylated fibronectin.

Oligosaccharides on certain cell-surface glycoproteins also play a role in cell-cell adhesion. For example, the plasma membrane of white blood cells (leukocytes) contains cell-adhesion molecules (CAMs) that are extensively glycosylated. The oligosaccharides in these molecules interact with a sugar-binding domain in certain other CAMs found on endothelial cells lining blood vessels. This interaction tethers the leukocytes to the endothelium and assists in their movement into tissues during an inflammatory response to infection (see Figure 20-40). Other cell-surface glycoproteins possess oligosaccharide side chains that can induce an immune response. A common example is the ABO blood group antigens, which are O-linked oligosaccharides attached to glycoproteins and glycolipids on the surfaces of erythrocytes and other cell types (see Figure 7-20). In both cases, oligosaccharides are added to the luminal face of these membrane proteins, in a manner similar to what is shown in Figure 13-18 for soluble proteins. The luminal face of these membrane proteins is topologically equivalent to the exterior face of the plasma membrane, where these proteins eventually end up.

## Disulfide Bonds Are Formed and Rearranged by Proteins in the ER Lumen

In Chapter 3, we learned that both intramolecular and intermolecular disulfide bonds ( -S-S-) help stabilize the tertiaryi
and quaternary structure of many proteins. These covalent bonds form by the oxidative linkage of sulfhydryl groups (-SH), also known as thiol groups, on two cysteine residues in the same or different polypeptide chains. This reaction can proceed only when a suitable oxidant is present. In eukaryotic cells, disulfide bonds are formed only in the lumen of the rough ER. Thus disulfide bonds are found only in soluble secretory proteins and in the exoplasmic domains of membrane proteins. Cytosolic proteins and organelle proteins synthesized on free ribosomes (i.e., those destined for mitochondria, chloroplasts, peroxisomes, etc.) usually lack disulfide bonds.

The efficient formation of disulfide bonds in the lumen of the ER depends on the enzyme protein disulfide isomerase (PDI), which is present in all eukaryotic cells. This enzyme is especially abundant in the ER of secretory cells in organs such as the liver and pancreas, where large quantities of proteins that contain disulfide bonds are produced. As shown in Figure 13-19a, the disulfide bond in the active site of PDI can be readily transferred to a protein by two sequential thiol-disulfide transfer reactions. The reduced PDI generated by this reaction is returned to an oxidized form by the action of an ER-resident protein, called Ero1, which carries a disulfide bond that can be transferred to PDI. Ero1 itself becomes oxidized by reaction with molecular oxygen that has diffused into the ER.

In proteins that contain more than one disulfide bond, the proper pairing of cysteine residues is essential for normal structure and activity. Disulfide bonds are commonly formed between cysteines that occur sequentially in the amino acid sequence while a polypeptide is still growing on the ribosome. Such sequential formation, however,

sometimes yields disulfide bonds between the wrong cysteines. For example, proinsulin, a precursor to the peptide hormone insulin, has three disulfide bonds that link cysteines 1 and 4,2 and 6 , and 3 and 5 . In this case, a disulfide bond that initially formed sequentially (e.g., between cysteines 1 and 2) would have to be rearranged for the protein to achieve its proper folded conformation. In cells, the rearrangement of disulfide bonds is also accelerated by PDI, which acts on a broad range of protein substrates, allowing them to reach their thermodynamically most stable conformations (Figure 13-19b). Disulfide bonds generally form in a specific order, first stabilizing small domains of a polypeptide, then stabilizing the interactions of more distant segments; this phenomenon is illustrated by the folding of the influenza hemagglutinin (HA) protein, discussed in the next section.

## Chaperones and Other ER Proteins Facilitate Folding and Assembly of Proteins

Although many denatured proteins can spontaneously refold into their native state in vitro, such refolding usually requires hours to reach completion. Yet proteins produced in the ER generally fold into their proper conformation within minutes after their synthesis. The rapid folding of these newly synthesized proteins in cells depends on the sequential action of several proteins present within the ER lumen. We have already seen how the molecular chaperone BiP can drive post-translational translocation in yeast by binding fully synthesized polypeptides as they enter the ER (see Figure 13-9). BiP can also bind transiently to nascent polypeptide chains as they enter the ER during cotranslational translocation. Bound BiP is thought to prevent
![li1294542480e](li1294542480e.jpg)
(b) Rearrangement of disulfide bonds
![li1304542480e](li1304542480e.jpg)

FIGURE 13-19 Action of protein disulfide isomerase (PDI). PDI forms and rearranges disulfide bonds via an active site with two closely spaced cysteine residues that are easily interconverted between the reduced dithiol form and the oxidized disulfide form. Numbered red arrows indicate the sequence of electron transfers. Yellow bars represent disulfide bonds. (a) In the formation of disulfide bonds, the ionized ( $-5^{-}$) form of a cysteine thiol in the substrate protein reacts with the disulfide (S-S) bond in oxidized PDI to form a disulfidebonded PDI-substrate protein intermediate. A second ionized thiol
in the substrate protein then reacts with this intermediate, forming a disulfide bond within the substrate protein and releasing reduced PDI. PDI, in turn, transfers electrons to a disulfide bond in the luminal protein Ero1, thereby regenerating the oxidized form of PDI. (b) Reduced PDI can catalyze rearrangement of improperly formed disulfide bonds by similar thiol-disulfide transfer reactions. In this case, reduced PDI both initiates and is regenerated in the reaction pathway. These reactions are repeated until the most stable conformation of the protein is achieved. See M. M. Lyles and H. F. Gilbert, 1991, Biochemistry 30:619.

segments of a nascent chain from misfolding or forming aggregates, thereby promoting folding of the entire polypeptide into the proper conformation. PDI also contributes to proper folding because the correct three-dimensional conformation is stabilized by disulfide bonds in many proteins.

As illustrated in Figure 13-20, two other ER proteins, the homologous lectins (carbohydrate-binding proteins) calnexin and calreticulin, bind selectively to certain N -linked oligosaccharides on growing polypeptide chains. The ligand for these two proteins, which resembles the N -linked oligosaccharide precursor but has only a single glucose residue $\left[\mathrm{Glc}_{1} \mathrm{Man}_{9}(\mathrm{GlcNAc})_{2}\right]$, is generated by a specific glucosyltransferase in the ER lumen (see Figure 13-18, step 1).

This enzyme acts only on polypeptide chains that are unfolded or misfolded, and in this respect, the glucosyltransferase acts as one of the primary surveillance mechanisms to ensure quality control of protein folding in the ER. Unfolded proteins often expose hydrophobic segments that in a properly folded state are buried in the hydrophobic core of the protein. The glucosyltransferase probably recognizes unfolded proteins by binding to these exposed hydrophobic segments. Binding of calnexin and calreticulin to unfolded nascent chains marked with glucosylated N -linked oligosaccharides prevents aggregation of adjacent segments of a protein as it is being made on the ER. Thus calnexin and calreticulin, like BiP, help prevent premature, incorrect folding of segments of a newly made protein.
![li1314542480e](li1314542480e.jpg)

FIGURE 13-20 Hemagglutinin folding and assembly. (a) Mechanism of $\left(\mathrm{HA}_{0}\right)$ trimer assembly. Transient binding of the chaperone BiP (step 1) to the nascent polypeptide chain and of two lectins, calnexin and calreticulin, to certain oligosaccharide chains (step 1) promotes proper folding of adjacent segments of $\mathrm{HA}_{0}$. A total of seven $N$-linked oligosaccharide chains are added to the luminal portion of the nascent chain during cotranslational translocation, and PDI catalyzes the formation of six disulfide bonds per monomer. Completed $\mathrm{HA}_{0}$ monomers are anchored in the membrane by a single membrane-spanning $\alpha$ helix with the N -terminus in the lumen (step 1). Interaction of three $\mathrm{HA}_{0}$ chains with one another, initially via their transmembrane $\alpha$ helices, apparently triggers formation of a long stem containing one $\alpha$ helix from the luminal part of each $\mathrm{HA}_{0}$ polypeptide. Finally, interactions occur among the three globular heads, generating a stable $\mathrm{HA}_{0}$ trimer (step 1). (b) Electron micrograph (false color) of a complete influenza virion showing trimers of HA protein protruding as spikes from the surface of the viral membrane. See U. Tatu et al., 1995, EMBO J. 14:1340, and D. Hebert et al., 1997, J. Cell Biol. 139:613. [Part (b), Chris Bjornberg/Science Source.]
(b)
![li1324542480e](li1324542480e.jpg)

Other important protein-folding catalysts in the ER lumen are peptidyl-prolyl isomerases, a family of enzymes that accelerate the rotation about peptidyl-prolyl bonds at proline residues in unfolded segments of a polypeptide:
![li1334542480e](li1334542480e.jpg)

Such isomerizations are sometimes the rate-limiting step in the folding of protein domains. Many peptidyl-prolyl isomerases can catalyze the rotation of exposed peptidyl-prolyl bonds indiscriminately in numerous proteins, but some have very specific protein substrates.

Many important soluble secretory and membrane proteins synthesized on the ER are built of two or more polypeptide subunits. In all cases, the assembly of the subunits constituting these multisubunit (multimeric) proteins occurs in the ER. The immunoglobulins, which contain two heavy $(\mathrm{H})$ and two light (L) chains, all linked by intrachain disulfide bonds, are assembled in this way. Hemagglutinin (HA) is another multimeric protein that provides a good illustration of folding and subunit assembly (see Figure 13-20). This trimeric protein forms the spikes that protrude from the surface of an influenza virus particle. The HA trimer is formed within the ER of an infected host cell from three copies of a precursor protein termed $\mathrm{HA}_{0}$, which has a single membrane-spanning $\alpha$ helix. In the Golgi complex, each of the three $\mathrm{HA}_{0}$ proteins is cleaved to form two polypeptides, $\mathrm{HA}_{1}$ and $\mathrm{HA}_{2}$; thus each HA molecule that eventually resides on the viral surface contains three copies of $\mathrm{HA}_{1}$ and three of $\mathrm{HA}_{2}$ (see Figure 3-11). The trimer is stabilized by interactions between the large exoplasmic domains of the constituent polypeptides, which extend into the ER lumen; after HA is transported to the cell surface, these domains extend into the extracellular space. Interactions between the smaller cytosolic and membrane-spanning portions of the HA subunits also help stabilize the trimeric protein. Studies have shown that it takes just 10 minutes for the $\mathrm{HA}_{0}$ polypeptides to fold and assemble into their proper trimeric conformation.

## Improperly Folded Proteins in the ER Induce Expression of Protein-Folding Catalysts

Wild-type proteins that are synthesized on the rough ER cannot exit this compartment until they achieve their completely folded conformation. Likewise, almost any mutation that prevents proper folding of a protein in the ER also blocks the movement of that protein from the ER lumen or
![li1344542480e](li1344542480e.jpg)

FIGURE 13-21 The unfolded-protein response. Ire1, a transmembrane protein in the ER membrane, has a binding site for BiP on its luminal domain; the cytosolic domain contains a specific RNA endonuclease. Step $\boldsymbol{\square}$ Accumulating unfolded proteins in the ER lumen bind BiP molecules, releasing them from monomeric Ire1. Dimerization of Ire1 then activates its endonuclease activity. Steps $\boldsymbol{\square}$ The unspliced mRNA precursor encoding the transcription factor Hac1 is cleaved by dimeric Ire1, and the two exons are joined to form functional Hac1 mRNA. Current evidence indicates that this processing occurs in the cytosol, although pre-mRNA processing generally occurs in the nucleus. Step $\boldsymbol{\square}$ Hac1 is translated into Hac1 protein, which then moves back into the nucleus and activates transcription of genes encoding several protein-folding catalysts. See U. Ruegsegger et al., 2001, Cell 107:103; A. Bertolotti et al., 2000, Nat. Cell Biol. 2:326; and C. Sidrauski and P. Walter, 1997, Cell 90:1031.
membrane to the Golgi complex. The mechanisms that retain unfolded or incompletely folded proteins within the ER probably increase the overall efficiency of folding by keeping intermediate forms in proximity to folding catalysts, which are most abundant in the ER. Improperly folded proteins retained within the ER are generally seen bound to the ER chaperones BiP and calnexin. Thus these luminal folding catalysts perform two related functions: assisting in the folding of normal proteins by preventing their aggregation and binding to misfolded proteins to retain them in the ER.

Both mammalian cells and yeasts respond to the presence of unfolded proteins in the rough ER by increasing transcription of several genes that encode ER chaperones and other folding catalysts. A key participant in this unfoldedprotein response is Ire1, an ER membrane protein that exists both as a monomer and as a dimer. The dimeric form, but

![li1354542480e](li1354542480e.jpg)

FIGURE 13-22 Modifications of $\boldsymbol{N}$-linked oligosaccharides are used to monitor folding and for quality control. After three glucose residues are removed from $N$-linked oligosaccharides in the ER, a single glucose can be re-added by a glucosyltransferase to form $\mathrm{Glc}_{1} \mathrm{Man}_{9}(\mathrm{GlcNAc})_{2}$ (see Figure 13-18, step 13). This modified $N$-linked oligosaccharide binds the lectins calnexin (CNX) and calreticulin (CRT)
for retention in the ER and engagement of folding chaperones. Proteins that cannot fold properly, and are therefore retained in the ER for longer times, undergo mannose trimming by ER $\alpha$-mannosidases to form $\mathrm{Man}_{5-\mathrm{a}}(\mathrm{GlcNAc})_{2}$, which is recognized by OS-9. Recognition by OS-9 leads to dislocation of the misfolded protein out of the ER, ubiquitinylation, and degradation by proteasomes.
not the monomeric form, promotes formation of Hac1, a transcription factor in yeast that activates expression of the genes induced in the unfolded-protein response. As depicted in Figure 13-21, binding of BiP to the luminal domain of monomeric Ire1 prevents formation of the Ire1 dimer. Thus the quantity of free BiP in the ER lumen determines the relative proportions of monomeric and dimeric Ire1. Accumulation of unfolded proteins within the ER lumen sequesters BiP molecules, making them unavailable for binding to Ire1. As a result, the level of dimeric Ire1 increases, leading to an increase in the level of Hac1 and production of proteins that assist in protein folding.

Mammalian cells contain an additional regulatory pathway that operates in response to unfolded proteins in the ER. In this pathway, accumulation of unfolded proteins in the ER triggers proteolysis of ATF6, a transmembrane protein in the ER membrane, at a site within the membrane-spanning segment. The cytosolic domain of ATF6 released by proteolysis then moves to the nucleus, where it stimulates transcription of the genes encoding ER chaperones. Activation of a transcription factor by such regulated intramembrane proteolysis also occurs in the Notch signaling pathway and during activation of the cholesterol-responsive transcription factor SREBP (see Figures 16-36 and 16-38).

A hereditary form of emphysema illustrates the detrimental effects that can result from misfolding of proteins in the ER. This disease is caused by a point mutation in $\alpha_{1}$-antitrypsin, which is normally secreted by hepatocytes and macrophages. The wild-type protein binds to and inhibits trypsin as well as the blood protease elastase. In the absence of normal $\alpha_{1}$-antitrypsin, elastase degrades the fine tissue in the lung that participates in the absorption of oxygen, eventually producing the symptoms of emphysema. Although the mutant $\alpha_{1}$-antitrypsin is synthesized on the rough ER, it does not fold
properly, forming an almost crystalline aggregate that is not exported from the ER. In hepatocytes, the secretion of other proteins also becomes impaired as the rough ER is filled with aggregated $\alpha_{1}$-antitrypsin.

## Unassembled or Misfolded Proteins in the ER Are Often Transported to the Cytosol for Degradation

Misfolded soluble secretory and membrane proteins, as well as the unassembled subunits of multimeric proteins, are often degraded within an hour or two after their synthesis in the rough ER. Initially, it was thought that proteolytic enzymes within the ER lumen catalyzed degradation of misfolded or unassembled polypeptides, but such proteases were never found. More recent studies have shown that misfolded secretory proteins are recognized by specific ER membrane proteins and are targeted for transport from the ER lumen into the cytosol by a process known as dislocation.

The dislocation and degradation of misfolded proteins depends on a set of proteins located in the ER membrane and in the cytosol that perform three basic functions. The first function is recognition of misfolded proteins. One mechanism for recognition involves the trimming of N linked carbohydrate chains by mannosidases located in the ER (Figure 13-22). Trimmed glycans with the structure $\mathrm{Man}_{5-\mathrm{a}}(\mathrm{GlcNAc})_{2}$ are recognized by a protein known as OS-9, which targets the trimmed glycoprotein for dislocation. Both EDEM and OS-9 target the trimmed glycoprotein for dislocation. It is not known precisely how the ER $\alpha$-mannosidases distinguish proteins that cannot fold properly, and are thus legitimate substrates for the dislocation process, from normal proteins that have transient partially folded states as they acquire their fully folded conformation. One possibility is that the ER $\alpha$-mannosidases act slowly, so that only those

glycoproteins that remain misfolded in the ER lumen for a sufficiently long time are trimmed and therefore targeted for degradation. Luminal proteins that lack oligosaccharides altogether can also be targeted for degradation, indicating that other processes for the recognition of unfolded proteins must also exist. Those other mechanisms cannot involve trimming of N -linked oligosaccharides because misfolded membrane proteins that lack N -linked oligosaccharides altogether can nevertheless be targeted for degradation.

The second function required for dislocation of misfolded proteins is the transport of the marked proteins from the lumen of the ER across the ER membrane to the cytosol. A complex of at least four integral membrane proteins, known as the ERAD ( $E R$-associated degradation) complex, enables dislocation of misfolded proteins across the ER membrane. The mechanism by which misfolded proteins traverse the ER membrane is not yet known, but there is no evidence that the ERAD complex forms a protein channel for dislocation, and it may be that dislocation involves powerful pulling and unfolding protein machines in the cytosol that drag misfolded proteins directly through the lipid bilayer.

Finally, as segments of a dislocated polypeptide are exposed to the cytosol, they encounter cytosolic enzymes that effect their degradation. One of these enzymes is p97, a member of a protein family, known as the AAA ATPase family, that couples the energy of ATP hydrolysis to disassembly of protein complexes. In dislocation of polypeptides from the ER, hydrolysis of ATP by p97 may provide the driving force to pull misfolded proteins from the ER membrane into the cytosol. As the misfolded proteins enter the cytosol, specific ubiquitin ligase enzymes that are components of the ERAD complex add ubiquitin residues to the dislocated peptides. Like the action of p97, the ubiquitinylation reaction is coupled to ATP hydrolysis; this release of energy may also contribute to trapping the proteins in the cytosol. The resulting polyubiquitinylated polypeptides, now fully in the cytosol, are removed from the cell altogether by degradation in proteasomes. The role of polyubiquitinylation in targeting proteins to proteasomes is discussed more fully in Chapter 3 (see Figure 3-31 and Figure 3-36).

## KEY CONCEPTS OF SECTION 13.3

## Protein Modifications, Folding, and Quality Control in the ER

- All N -linked oligosaccharides, which are bound to asparagine residues, contain a core of two N -acetylglucosamine and at least three mannose residues and usually have several branches. O-linked oligosaccharides, which are bound to serine or threonine residues, are generally short, often containing only one to four sugar residues.
- Formation of N -linked oligosaccharides begins with assembly of a conserved 14-residue high-mannose oligosaccharide precursor on dolichol, a lipid in the membrane of the rough ER (see Figure 13-17). After this preformed oligosaccharide is transferred to specific asparagine residues of nascent polypeptide chains in the ER lumen, three glucose residues and one mannose residue are removed (see Figure 13-18).
- Oligosaccharide side chains may assist in the proper folding of glycoproteins, help protect the mature proteins from proteolysis, participate in cell-cell adhesion, and function as antigens.
- Disulfide bonds are added to many soluble secretory proteins and to the exoplasmic domain of membrane proteins in the ER. Protein disulfide isomerase (PDI), present in the ER lumen, catalyzes both the formation and the rearrangement of disulfide bonds (see Figure 13-19).
- The chaperone BiP, the lectins calnexin and calreticulin, and peptidyl-prolyl isomerases work together to ensure proper folding of newly made secretory and membrane proteins in the ER. The subunits of multimeric proteins also assemble in the ER (see Figure 13-20).
- Only properly folded proteins and assembled subunits are transported from the rough ER to the Golgi complex in vesicles.
- The accumulation of abnormally folded proteins and unassembled subunits in the ER can induce increased expression of ER protein-folding catalysts via the unfolded-protein response (see Figure 13-21).
- Unassembled or misfolded proteins in the ER are often transported back to the cytosol, where they are degraded in the ubiquitin-proteasome pathway (see Figure 13-22).


### 13.4 Targeting of Proteins to Mitochondria and Chloroplasts

In the remainder of this chapter, we examine how proteins synthesized on cytosolic ribosomes are sorted to the discrete organelles: mitochondria, chloroplasts, peroxisomes, and the nucleus (see Figure 13-1). Mitochondria and chloroplasts are closely related organelles that contain an internal lumen, called the matrix, that is surrounded by a double membrane. In contrast, peroxisomes, which we discuss next, are bounded by a single membrane and have a single luminal matrix compartment. The mechanism of protein transport into and out of the nucleus, which differs in many respects from sorting to other organelles, is discussed in the last section of the chapter.

In addition to being bounded by two membranes, mitochondria and chloroplasts share similar types of electrontransporting proteins and use an F-class ATPase to synthesize

ATP (see Figure 12-30). Remarkably, these characteristics are shared by gram-negative bacteria. Also like bacterial cells, mitochondria and chloroplasts contain their own DNA, which encodes organelle rRNAs, tRNAs, and some proteins (see Chapter 8). Moreover, growth and division of mitochondria and chloroplasts are not coupled to nuclear division. Rather, these organelles grow by the incorporation of cellular proteins and lipids, and new organelles form by division of preexisting organelles. These numerous similarities to free-living bacterial cells have led to the understanding that mitochondria and chloroplasts arose when bacteria were incorporated into ancestral eukaryotic cells, forming endosymbiotic organelles (see Figure 12-7). The sequence similarity of the many membrane translocation proteins shared by mitochondria, chloroplasts, and bacteria provides the most striking evidence for this ancient evolutionary relationship. In this section, we examine these membrane translocation proteins in detail.

Proteins encoded by mitochondrial DNA or chloroplast DNA are synthesized on ribosomes within the organelles and directed to the correct subcompartment within the parent organelle immediately after synthesis. However, the majority of proteins located in mitochondria and chloroplasts are encoded by genes in the nucleus and are imported into the organelles after their synthesis in the cytosol. Apparently, as eukaryotic cells evolved over a billion years, much of the genetic information from the ancestral bacterial DNA in these endosymbiotic organelles moved, by an unknown mechanism, to the nucleus. Precursor proteins synthesized in the cytosol that are destined for the matrix of mitochondria or the equivalent space in chloroplasts, the stroma, usually contain specific N -terminal targeting sequences that specify binding to receptor proteins on the organelle surface. Generally these sequences are cleaved once the protein reaches the matrix or stroma. Clearly these targeting sequences are similar in their location and general function to the signal
sequences that direct nascent proteins to the ER lumen. Although the three types of signals share some sequence features, their specific sequences differ considerably, as summarized in Table 13-1.

In both mitochondria and chloroplasts, protein import requires energy and occurs at points where the outer and inner organelle membranes are in direct contact. Because mitochondria and chloroplasts contain multiple membranes and membrane-limited spaces, the sorting of many proteins to their correct locations requires the sequential action of two targeting sequences and two membrane-bound translocation systems: one to direct the protein into the organelle and the other to direct it into the correct organelle subcompartment or membrane. As we will see, the mechanisms for sorting various proteins to mitochondria and chloroplasts are related to some of the mechanisms discussed previously.

## Amphipathic N-Terminal Targeting Sequences Direct Proteins to the Mitochondrial Matrix

All proteins that travel from the cytosol to the same mitochondrial destination have targeting signals that share common motifs, although the signal sequences are not identical. Thus the receptors that recognize such signals are able to bind to a number of different but related sequences. The most extensively studied sequences for localizing proteins to mitochondria are the matrix-targeting sequences. These sequences, located at the N -terminus, are usually $20-50$ amino acids in length. They are rich in hydrophobic amino acids, positively charged basic amino acids (arginine and lysine), and hydroxylated ones (serine and threonine), but tend to lack negatively charged acidic residues (aspartate and glutamate).

Mitochondrial matrix-targeting sequences are thought to assume an $\alpha$-helical conformation in which positively charged amino acids predominate on one side of the helix

TABLE 13-1 Targeting Sequences That Direct Proteins from the Cytosol to Organelles*

| Target Organelle | Location of Sequence Within Protein | Removal of Sequence | Nature of Sequence |
| :--: | :--: | :--: | :--: |
| Endoplasmic reticulum (lumen) | N-terminus | Yes | Core of 6-12 hydrophobic amino acids, often preceded by one or more basic amino acids (Arg, Lys) |
| Mitochondrion (matrix) | N-terminus | Yes | Amphipathic helix, 20-50 residues in length, with Arg and Lys residues on one side and hydrophobic residues on the other |
| Chloroplast (stroma) | N-terminus | Yes | No common motifs; generally rich in Ser, Thr, and small hydrophobic residues and poor in Glu and Asp |
| Peroxisome (matrix) | C-terminus (most proteins); N-terminus (few proteins) | No | PTS1 signal (Ser-Lys-Leu) at extreme C-terminus; PTS2 signal at N-terminus |
| Nucleus (nucleoplasm) | Varies | No | Multiple different kinds; a common motif includes a short segment rich in Lys and Arg residues |

[^0]
[^0]:    *Different or additional sequences target proteins to organelle membranes and subcompartments.

![li1364542480e](li1364542480e.jpg)

FIGURE 13-23 Import of mitochondrial precursor proteins is assayed in a cell-free system. Mitochondrial precursor proteins with attached uptake-targeting signals can be synthesized on ribosomes in a cell-free reaction. When respiring mitochondria are added to these synthesized mitochondrial precursor proteins (top), the proteins are taken up by mitochondria. Inside mitochondria, the proteins are protected from the action of proteases such as trypsin. When no mitochondria are present (bottom), the mitochondrial proteins are degraded by added protease. Only energized (respiring) mitochondria, which have a proton electrochemical gradient (proton-motive force) across the inner membrane, take up these proteins. Furthermore, the imported proteins must contain an appropriate uptake-targeting sequence. Uptake also requires ATP and a cytosolic extract containing chaperone proteins that maintain the precursor proteins in an unfolded conformation. This assay has been used to study targeting sequences and other features of the translocation process.
and hydrophobic amino acids on the other. Sequences such as these that contain both hydrophobic and hydrophilic regions are said to be amphipathic. Mutations that disrupt the amphipathic character of these sequences usually disrupt targeting to the matrix, although many other amino acid substitutions do not. These findings indicate that the amphipathicity of matrix-targeting sequences is critical to their function.

The cell-free assay outlined in Figure 13-23 has been widely used to define the biochemical steps in the import of mitochondrial precursor proteins. Respiring (energized) mitochondria extracted from cells can incorporate mitochondrial precursor proteins that have been synthesized in the absence of mitochondria if they are carrying appropriate targeting sequences. Successful incorporation of the precursor into the organelle can be assayed by resistance to digestion by an added protease such as trypsin. In other assays, successful import of a precursor protein can be shown by the proper cleavage of the N -terminal targeting sequences by specific mitochondrial proteases. The uptake of completely presynthesized mitochondrial precursor proteins by the organelle in this system contrasts with the cell-free cotranslational translocation of secretory proteins into the ER, which generally occurs only when microsomal (ER-derived) membranes are present during synthesis (see Figure 13-4).

## Mitochondrial Protein Import Requires OuterMembrane Receptors and Translocons in Both Membranes

Figure 13-24 presents an overview of protein import from the cytosol into the mitochondrial matrix, the route into the mitochondrion followed by most imported proteins.

Here we discuss in detail each step of protein transport into the matrix, then consider how some proteins are subsequently targeted to other compartments of the mitochondrion.

After synthesis in the cytosol, the soluble precursors of mitochondrial proteins (including hydrophobic integral membrane proteins) can interact directly with the mitochondrial membrane. Import of an unfolded mitochondrial precursor protein is initiated by the binding of a mitochondrial targeting sequence to an import receptor in the outer mitochondrial membrane. These import receptors were first identified by experiments in which antibodies to specific proteins of the outer mitochondrial membrane were shown to inhibit protein import into isolated mitochondria. Subsequent genetic experiments in which the genes for specific mitochondrial outer-membrane proteins were mutated showed that specific receptor proteins were responsible for the import of different classes of mitochondrial proteins. For example, N -terminal matrix-targeting sequences are recognized by Tom20 and Tom22. (Proteins in the outer mitochondrial membrane involved in targeting and import are designated Tom proteins, for translocon of the outer membrane.)

Many proteins can be imported into the mitochondrion only in an unfolded state. Chaperone proteins such as cytosolic Hsp70 and Hsp90 use energy derived from ATP hydrolysis to keep nascent and newly made proteins in a disaggregated state so that they are available to be taken up by mitochondria. For some mitochondrial precursor proteins, the mitochondrial outer-membrane protein Tom70 serves as an import receptor by binding to both Hsp90 and portions of the unfolded precursor protein.

The import receptors subsequently transfer the precursor protein to an import channel in the outer membrane.

![li1374542480e](li1374542480e.jpg)

FIGURE 13-24 Protein import into the mitochondrial matrix. Precursor proteins synthesized on cytosolic ribosomes are maintained in an unfolded or partially folded state by bound chaperones, such as cytosolic Hsp70 (step 1). After a precursor protein binds to an import receptor near a site of contact with the inner membrane (step 2), it is transferred into the general import pore (step 3). The translocating protein then moves through this channel and an adjacent channel in the inner membrane (steps 4-5). Note that translocation occurs at rare "contact sites" at which the inner and outer membranes appear to touch. Binding of the translocating protein by matrix Hsp70 and subsequent ATP hydrolysis by Hsp70 helps drive import into the matrix. Once the targeting sequence is removed by a matrix protease and Hsp70 is released from the newly imported protein (step 6), the protein folds into its mature, active conformation within the matrix (step 7). Folding of some proteins depends on matrix chaperonins. See G. Schatz, 1996, J. Biol. Chem. 271:31763, and N. Pfanner et al., 1997, Annu. Rev. Cell Dev. Biol. 13:25.

This channel, composed mainly of the Tom 40 protein, is known as the general import pore because all known mitochondrial precursor proteins gain access to the interior compartments of the mitochondrion through it. When Tom 40 is purified and incorporated into liposomes, it forms a transmembrane channel with a pore wide enough to accommodate an unfolded polypeptide chain. The general import pore forms a largely passive channel through the outer mitochondrial membrane; the driving force for unidirectional transport comes from within the mitochondrion, as we will see shortly. In the case of precursors destined for the mitochondrial matrix, transfer through the outer membrane occurs simultaneously with transfer through an inner-membrane channel composed of the Tim23 and Tim17 proteins. (Tim stands for translocon of the inner membrane.) Translocation into the matrix thus occurs at
"contact sites" where the outer and inner membranes are in close proximity.

Soon after the N-terminal matrix-targeting sequence of a protein enters the mitochondrial matrix, it is removed by a protease that resides within the matrix. The emerging protein is also bound by matrix Hsp70, a chaperone that is localized to the translocation channels in the inner mitochondrial membrane by interaction with transmembrane protein Tim44. This binding stimulates ATP hydrolysis by matrix Hsp70, and together, Tim44 and Hsp70 are thought to power translocation of proteins into the matrix.

Some imported proteins can fold into their final, active conformation without further assistance. Final folding of many matrix proteins, however, requires chaperonins. As discussed in Chapter 3, chaperonin proteins actively

facilitate protein folding by a process that depends on ATP. Yeast mutants defective in Hsc60, a chaperonin in the mitochondrial matrix, can import matrix proteins and cleave their targeting sequences normally, but the imported polypeptides fail to fold and assemble into their native tertiary and quaternary structures.

## Studies with Chimeric Proteins Demonstrate Important Features of Mitochondrial Protein Import

Dramatic evidence for the ability of mitochondrial matrixtargeting sequences to direct the import of proteins was obtained with chimeric proteins produced by recombinant DNA techniques. For example, the matrix-targeting sequence of alcohol dehydrogenase can be fused to the N -terminus of dihydrofolate reductase (DHFR), which normally resides in the cytosol. Cell-free translocation assays show that in the presence of chaperones, which prevent the C-terminal DHFR segment from folding in the cytosol, the chimeric protein is transported into the matrix (Figure 13-25a). The inhibitor
methotrexate, which binds tightly to the active site of DHFR and greatly stabilizes its folded conformation, renders the chimeric protein resistant to unfolding by cytosolic chaperones. When translocation assays are performed in the presence of methotrexate, the chimeric protein does not completely enter the matrix. This finding demonstrates that a precursor must be unfolded in order to traverse the import pores in both mitochondrial membranes.

Additional studies revealed that if a sufficiently long spacer sequence separates the N -terminal matrix-targeting sequence and the DHFR portion of the chimeric protein, then, in the presence of methotrexate, a translocation intermediate that spans both membranes can be trapped if enough of the polypeptide protrudes into the matrix to prevent the polypeptide chain from sliding back into the cytosol, possibly by stably associating with matrix Hsp70 (Figure 13-25b). In order for such a stable translocation intermediate to form, the spacer sequence must be long enough to span both membranes; a spacer of 50 amino acids extended to its maximum possible length is adequate to do so. If the chimera contains a shorter spacer-say,
![li1384542480e](li1384542480e.jpg)

EXPERIMENTAL FIGURE 13-25 Experiments with chimeric proteins elucidate mitochondrial protein import processes. These experiments show that a matrix-targeting sequence alone directs proteins to the mitochondrial matrix and that only unfolded proteins are translocated across both mitochondrial membranes. The chimeric protein in these experiments contained a matrix-targeting signal at its N -terminus (red), followed by a spacer sequence of no particular function (black), and then by dihydrofolate reductase (DHFR), an enzyme normally present only in the cytosol. (a) When the DHFR segment is unfolded, the chimeric protein moves across both membranes to the matrix of an energized mitochondrion, and the matrix-targeting signal is then removed. (b) When the C-terminus of the chimeric protein is locked in the folded state by binding of methotrexate, translocation
is blocked. If the spacer sequence is long enough to extend across both transport channels, a stable translocation intermediate, with the targeting sequence cleaved off, is generated in the presence of methotrexate, as shown here. (c) The C-terminus of the translocation intermediate in (b) can be detected by incubating the mitochondria with antibodies that bind to the DHFR segment, followed by gold particles coated with bacterial protein A, which binds nonspecifically to antibody molecules (see Figure 4-33). An electron micrograph of a sectioned sample reveals gold particles (red arrowhead) bound to the translocation intermediate at a contact site between the inner and outer membranes. Other contact sites (black arrows) are also evident. See J. Rassow et al., 1990, FEBS Lett. 275:190. [Part (c) ©1987 M. Schweiger et al., The Journal of Cell Biology 105:235-246. doi: 10.1083/jcb.105.1.235]

35 amino acids-no stable translocation intermediate is obtained because the spacer cannot span both membranes. These observations provide further evidence that translocated proteins can span both inner and outer mitochondrial membranes and can traverse these membranes only in an unfolded state.

Microscopy studies of stable translocation intermediates show that they accumulate at sites where the inner and outer mitochondrial membranes are close together (Figure 13-25c). Since roughly a thousand stuck chimeric proteins can be observed at these contact sites in a typical yeast mitochondrion, it is thought that mitochondria have approximately a thousand general import pores for the uptake of mitochondrial proteins.

## Three Energy Inputs Are Needed to Import Proteins into Mitochondria

As noted previously and as indicated in Figure 13-24, ATP hydrolysis by Hsp70 chaperone proteins in both the cytosol and the mitochondrial matrix is required for the import of mitochondrial proteins. Cytosolic Hsp70 expends energy to maintain bound precursor proteins in an unfolded state so that they can be translocated into the matrix. The importance of ATP to this function was demonstrated in studies in which a mitochondrial precursor protein was purified and then denatured (unfolded) by urea. When tested in the cell-free mitochondrial translocation system, the denatured protein was incorporated into the matrix in the absence of ATP. In contrast, the same precursor protein in its native, undenatured state was not imported in the absence of ATP, even in the presence of cytosolic chaperones.

The sequential binding to and ATP-driven release of multiple matrix Hsp70 molecules from a translocating protein may simply trap the unfolded protein in the matrix. Alternatively, the matrix Hsp70, anchored to the membrane by the Tim44 protein, may act as a molecular motor to pull the protein into the matrix (see Figure 13-24). In this case, the functions of matrix Hsp70 and Tim44 would be analogous to those of the chaperone BiP and Sec63 complex, respectively, in post-translational translocation into the ER lumen (see Figure 13-9).

The third energy input required for mitochondrial protein import is a $\mathrm{H}^{+}$electrochemical gradient, or protonmotive force, across the inner membrane. Recall from Chapter 12 that protons are pumped from the matrix into the intermembrane space during electron transport, creating an electric potential across the inner membrane. In general, only mitochondria that are actively undergoing respiration, and therefore have generated a proton-motive force across the inner membrane, are able to translocate precursor proteins from the cytosol into the mitochondrial matrix. Treatment of mitochondria with inhibitors or uncouplers of oxidative phosphorylation, such as cyanide or dinitrophenol, dissipates this proton-motive force. Although precursor proteins can still bind tightly to receptors on such poisoned mitochondria, the proteins cannot be imported,
either in intact cells or in cell-free systems, even in the presence of ATP and chaperone proteins. Scientists do not fully understand how the proton-motive force is used to facilitate entry of a precursor protein into the matrix. Once a protein is partially inserted into the inner membrane, it is subjected to a membrane potential of 200 mV (matrix negative). This seemingly small potential is established across the very narrow hydrophobic core of the lipid bilayer, which gives an enormous electrochemical gradient, equivalent to about $400,000 \mathrm{~V} / \mathrm{cm}$. One hypothesis is that the positive charges in the amphipathic matrix-targeting sequence are simply "electrophoresed," or pulled, into the matrix by the inside-negative membrane potential.

## Multiple Signals and Pathways Target Proteins to Submitochondrial Compartments

Unlike targeting to the matrix, targeting of proteins to the intermembrane space, inner membrane, or outer membrane of mitochondria generally requires more than one targeting sequence and occurs via one of several pathways. Figure 13-26 summarizes the organization of targeting sequences in proteins sorted to different mitochondrial locations.

Inner-Membrane Proteins Three separate pathways are known to target proteins to the inner mitochondrial membrane. One pathway makes use of the same machinery that is used for the targeting of matrix proteins (Figure 13-27, path A). A cytochrome oxidase subunit called CoxVa is one protein transported by this pathway. The precursor form of CoxVa, which contains an N-terminal matrix-targeting sequence recognized by the Tom20/22 import receptor, is transferred through the Tom40 general import pore of the outer membrane and the inner-membrane Tim23/17 translocation complex. In addition to the matrix-targeting sequence, which is cleaved during import, CoxVa contains a hydrophobic stop-transfer anchor sequence. As the protein passes through the Tim23/17 channel, the stop-transfer anchor sequence blocks translocation of the C-terminus across the inner membrane. The membrane-anchored intermediate is then transferred laterally into the bilayer of the inner membrane, much as type I integral membrane proteins are incorporated into the ER membrane (see Figure 13-11).

A second pathway to the inner membrane is followed by proteins (e.g., ATP synthase subunit 9) whose precursors contain both a matrix-targeting sequence and internal hydrophobic domains recognized by an inner-membrane protein termed Oxal. This pathway is thought to involve translocation of at least a portion of the precursor into the matrix via the Tom40 and Tim23/17 channels. After cleavage of the matrix-targeting sequence, the protein is inserted into the inner membrane by a process that requires interaction with Oxal and perhaps other innermembrane proteins (Figure 13-27, path B). Oxal is related to a protein involved in inserting some plasma-membrane proteins in bacteria. This relatedness suggests that Oxal may have descended from the translocation machinery

| Location <br> of imported <br> protein | Imported protein | Locations of targeting sequences in preprotein | FIGURE 13-26 Targeting sequences in imported mitochondrial proteins. Most mitochondrial proteins have an N-terminal matrix-targeting sequence (pink) that is similar, though not identical, in different proteins. Proteins destined for the inner membrane, the intermembrane space, or the outer membrane have one or more additional targeting sequences that function to direct the proteins to these locations by several different pathways. The lettered pathways correspond to those illustrated in Figures 13-27 and 13-28. See W. Neupert, 1997, Annu. Rev. Biochem. 66:863. |
| :--: | :--: | :--: | :--: |
| Matrix | Alcohol dehydrogenase III | Matrix-targeting sequence |  |
| Inner membrane | Cytochrome oxidase subunit CoxVa | ![li1394542480e](li1394542480e.jpg) |  |
| Path A | ATP synthase subunit 9 | ![li1404542480e](li1404542480e.jpg) |  |
|  |  | Internal sequences recognized by Tom70 receptor and Tim22 complex |  |
|  |  | ![li1414542480e](li1414542480e.jpg) |  |
| Intermembrane space |  | ![li1424542480e](li1424542480e.jpg) |  |
| Path A | Cytochrome $b_{2}$ |  |  |
|  |  | Targeting sequence for the general import pore |  |
| Path B | Tim9, Tim10 |  |  |
| Outer membrane |  | Stop-transfer and outer-membrane localization sequence |  |
|  | Porin (P70) |  |  |

in the endosymbiotic bacterium that eventually became the mitochondrion. However, the proteins that form the inner-membrane channels in mitochondria are not related to the proteins in bacterial translocons. Oxa1 also participates in the inner-membrane insertion of certain proteins (e.g., subunit II of cytochrome oxidase) that are encoded by mitochondrial DNA and synthesized in the matrix by mitochondrial ribosomes.

The final pathway for insertion in the inner mitochondrial membrane is followed by multipass proteins that contain six membrane-spanning domains, such as the ATP/ADP antiporter. These proteins, which lack the usual N-terminal matrix-targeting sequence, contain multiple
internal mitochondrial targeting sequences. After the internal sequences are recognized by a second import receptor composed of outer-membrane proteins Tom70 and Tom22, the imported protein passes through the outer membrane via the general import pore (Figure 13-27, path C). The protein then is transferred to a second translocation complex in the inner membrane composed of the Tim22, Tim18, and Tim54 proteins. Transfer to the Tim22/18/54 complex depends on a multimeric complex of two small proteins, Tim9 and Tim10, which reside in the intermembrane space. These small Tim proteins are thought to act as chaperones, guiding imported protein precursors from the general import pore to the Tim22/18/54 complex in

Path A
Path B
Path C
![li1434542480e](li1434542480e.jpg)

FIGURE 13-27 Three pathways to the inner mitochondrial membrane from the cytosol. Proteins with different targeting sequences are directed to the inner membrane via different pathways. In all three pathways, proteins cross the outer membrane via the Tom40 general import pore. Proteins delivered by pathways A and B contain an N -terminal matrix-targeting sequence that is recognized by the Tom20/22 import receptor in the outer membrane. Although both these pathways use the Tim23/17 inner-membrane channel, they differ in that the entire precursor protein enters the matrix and is then redirected
to the inner membrane in pathway B. Matrix Hsp70 plays a role similar to its role in the import of soluble matrix proteins (see Figure 13-23). Proteins delivered by pathway C contain internal sequences that are recognized by the Tom70/Tom22 import receptor; a different innermembrane translocation channel (Tim22/54) is used in this pathway. Two intermembrane proteins (Tim9 and Tim10) facilitate transfer between the outer and inner channels. See the text for discussion. See R. E. Dalbey and A. Kuhn, 2000, Annu. Rev. Cell Dev. Biol. 16:51, and N. Pfanner and A. Geissler, 2001, Nat. Rev. Mol. Cell Biol. 2:339.
the inner membrane by binding to their hydrophobic regions, preventing them from forming insoluble aggregates in the aqueous environment of the intermembrane space. Ultimately the Tim22/18/54 complex is responsible for incorporating the multiple hydrophobic segments of the imported protein into the inner membrane.

Intermembrane-Space Proteins Two pathways deliver cytosolic proteins to the space between the inner and outer mitochondrial membranes. The major pathway is followed
by proteins, such as cytochrome $b_{2}$, whose precursors carry two different N -terminal targeting sequences, both of which are ultimately cleaved. The most N -terminal of the two sequences is a matrix-targeting sequence, which is removed by the matrix protease. The second targeting sequence is a hydrophobic segment that blocks complete translocation of the protein across the inner membrane (Figure 13-28, path A). After the resulting membrane-embedded intermediate diffuses laterally away from the Tim23/17 translocation channel, a protease in the membrane cleaves the protein near the

![li1444542480e](li1444542480e.jpg)

FIGURE 13-28 Two pathways to the mitochondrial intermembrane space. Pathway A, the major one for delivery of proteins from the cytosol to the intermembrane space, is similar to pathway A for delivery to the inner membrane (see Figure 13-26). The major difference is that the internal targeting sequence in these proteins, such as cytochrome $b_{2}$, is recognized by an inner-membrane protease, which cleaves the protein on the intermembrane-space side of the membrane. The released protein then folds and binds to its heme cofactor within the intermembrane
hydrophobic transmembrane segment, releasing the mature protein in a soluble form into the intermembrane space. Except for the second proteolytic cleavage, this pathway is similar to that of inner-membrane proteins such as CoxVa (see Figure 13-27, path A).

The small Tim9 and Tim10 proteins, which reside in the intermembrane space, illustrate a second pathway for targeting to the intermembrane space. Proteins imported by this pathway do not contain an N-terminal matrix-targeting sequence and are delivered directly to the intermembrane space via the general import pore without involvement of any inner-membrane translocation factors (Figure 13-28, path B). Translocation through the Tom40 general import pore does not seem to be coupled to any energetically favorable process; however, once located in the intermembrane space, Tim9 and Tim10 proteins acquire two disulfide bonds each and fold into compact, stable structures. Apparently, the mechanism that drives their unidirectional translocation through the outer membrane involves passive diffusion through the outer membrane followed by folding and disulfide bond formation, which irreversibly traps the proteins in the intermembrane space. In many respects, the process of disulfide bond formation in the intermembrane space resembles that in the
space. Pathway B is a specialized pathway for delivery of the proteins Tim9 and Tim10 to the intermembrane space. These proteins readily pass through the Tom40 general import pore, and once they are in the intermembrane space, they fold and form disulfide bonds that prevent reverse translocation through Tom40. The disulfide bonds are generated by Erv1 and are transferred to Tim9 and Tim10 by Mia40. See R. E. Dalbey and A. Kuhn, 2000, Annu. Rev. Cell Dev. Biol. 16:51; N. Pfanner and A. Geissler, 2001, Nat. Rev. Mol. Cell Biol. 2:339; and K. Tokatlidis, 2005, Cell 121:965-967.

ER lumen and involves a disulfide bond-generating protein, Erv1, and a disulfide transfer protein, Mia40.

Outer-Membrane Proteins Many of the proteins that reside in the mitochondrial outer membrane, including the Tom40 pore itself and mitochondrial porin, have a $\beta$-barrel structure in which antiparallel $\beta$ strands form hydrophobic transmembrane segments surrounding a central channel. Such proteins are incorporated into the outer membrane by first interacting with the general import pore, Tom40; they are then transferred to the SAM (sorting and assembly machinery) complex, which is composed of at least three outer-membrane proteins. One of these three proteins, Sam50, is closely related to the bacterial protein BamA, which is necessary for insertion of $\beta$-barrel proteins into the outer membrane of gram-negative bacteria. This relatedness provides another example of the conservation of the mechanism of membrane protein insertion between bacteria and mitochondria. Presumably it is the very stable hydrophobic nature of $\beta$-barrel proteins that ultimately causes them to be stably incorporated into the outer membrane, but precisely how the SAM complex facilitates this process is not known.

## Import of Chloroplast Stromal Proteins Is Similar to Import of Mitochondrial Matrix Proteins

Among the proteins found in the chloroplast stroma are the enzymes of the Calvin cycle, which function in fixing carbon dioxide during photosynthesis (see Chapter 12). The large (L) subunit of ribulose 1,5-bisphosphate carboxylase (rubisco) is encoded by chloroplast DNA and synthesized on chloroplast ribosomes in the stromal space. The small (S) subunit of rubisco and all the other Calvin cycle enzymes are encoded by nuclear genes and transported to chloroplasts after their synthesis in the cytosol. The precursor forms of these stromal proteins contain an N-terminal stromal-import sequence (see Table 13-1).

Experiments with isolated chloroplasts, similar to those with mitochondria illustrated in Figure 13-23, have shown that they can import the rubisco S-subunit precursor after its synthesis. After the unfolded precursor enters the stromal space, it binds transiently to a stromal Hsp70 chaperone, and the N-terminal stromal-import sequence is cleaved. In reactions facilitated by Hsp60 chaperonins that reside within the stromal space, eight $S$ subunits combine with eight $L$ subunits to yield the active rubisco enzyme.

The general process of stromal import appears to be very similar to that of protein import into the mitochondrial matrix (see Figure 13-24). At least three chloroplast outer-membrane proteins, including a receptor that binds the stromal-import sequence and a translocation channel protein, and five chloroplast inner-membrane proteins are known to be essential for directing proteins to the stroma. Although these proteins are functionally analogous to the receptor and channel proteins in the mitochondrial membrane, they are not structurally homologous. The lack of sequence similarity between these chloroplast and mitochondrial proteins suggests that they may have arisen independently during evolution.

The available evidence suggests that chloroplast stromal proteins, like mitochondrial matrix proteins, are imported in the unfolded state. Import into the stroma depends on ATP hydrolysis catalyzed by a stromal Hsp70 chaperone whose function is similar to that of Hsp70 in the mitochondrial matrix and BiP in the ER lumen. Unlike mitochondria, chloroplasts do not generate an electrochemical gradient (proton-motive force) across their inner membrane. Thus protein import into the chloroplast stroma appears to be powered solely by ATP hydrolysis.

## Proteins Are Targeted to Thylakoids by Mechanisms Related to Bacterial Protein Translocation

In addition to the double membrane that surrounds them, chloroplasts contain a series of internal interconnected membranous sacs, the thylakoids (see Figure 12-37). All of the chemical reactions of photosynthesis take place in the thylakoid membrane or lumen and are catalyzed by the proteins that are localized to this specialized subcompartment. Many of these proteins are synthesized in the cytosol as precursors
containing multiple targeting sequences. For example, plastocyanin and other proteins destined for the thylakoid lumen require the successive action of two targeting sequences. The first is an N-terminal stromal-import sequence that directs the protein to the stroma by the same pathway that imports the rubisco $S$ subunit. The second sequence targets the protein from the stroma to the thylakoid lumen. The role of these targeting sequences has been shown in experiments measuring the uptake of mutant proteins generated by recombinant DNA techniques into isolated chloroplasts. For instance, mutant plastocyanin that lacks the thylakoidtargeting sequence but contains an intact stromal-import sequence accumulates in the stroma and is not transported into the thylakoid lumen.

Four separate pathways for transporting proteins from the stroma into the thylakoid have been identified. All four pathways have been found to be closely related to analogous transport mechanisms in bacteria, illustrating the close evolutionary relationship between the stromal membrane and the bacterial plasma membrane. Transport of plastocyanin and related proteins into the thylakoid lumen from the stroma occurs by an SRP-dependent pathway that uses a translocon similar to SecY, the bacterial version of the Sec61 complex (Figure 13-29, left). A second pathway for transporting proteins into the thylakoid lumen involves a protein related to bacterial protein SecA, which uses the energy from ATP hydrolysis to drive protein translocation through the SecY translocon. A third pathway, which targets proteins to the thylakoid membrane, depends on a protein related to the mitochondrial Oxa1 protein and the homologous bacterial protein (see Figure 13-27, path B). Some proteins encoded by chloroplast DNA and synthesized in the stroma or transported into the stroma from the cytosol are inserted into the thylakoid membrane via this pathway.

Finally, thylakoid proteins that bind metal-containing cofactors follow another pathway into the thylakoid lumen (Figure 13-29, right). The unfolded precursors of these proteins are first targeted to the stroma, where the N -terminal stromal-import sequence is cleaved off, and the protein then folds and binds its cofactor. A set of thylakoid-membrane proteins assists in translocating the folded protein and bound cofactor into the thylakoid lumen. This process is powered by the $\mathrm{H}^{+}$electrochemical gradient normally maintained across the thylakoid membrane. The thylakoid-targeting sequence that directs a protein to this pathway includes two closely spaced arginine residues that are crucial for recognition. Bacterial cells also have a mechanism for translocating folded proteins with a similar arginine-containing sequence across the plasma membrane, known as the Tat (twin-arginine translocation) pathway. The molecular mechanism whereby these large folded globular proteins are translocated across the thylakoid membrane is not fully understood, but the presence of a folded protein with an appropriate twin arginine signal appears to induce the oligomerization of Tat proteins in the membrane to form pore-like structures. In this respect, the Tat pathway resembles the pathway for the import of folded proteins into peroxisomes, described in the next section.

![li1454542480e](li1454542480e.jpg)

FIGURE 13-29 Transporting proteins to chloroplast thylakoids. Two of the four pathways for transporting proteins from the cytosol to the thylakoid lumen are shown here. In these pathways, unfolded precursors are delivered to the stroma via the same outer-membrane proteins that import stromal-localized proteins. Cleavage of the N -terminal stromal-import sequence by a stromal protease then reveals the thylakoid-targeting sequence (step 11). At this point the two pathways diverge. In the SRP-dependent pathway (left), plastocyanin and similar proteins are kept unfolded in the stromal space by a set of chaperones (not shown), and the thylakoid-targeting sequence binds to proteins that are closely related to the bacterial SRP, SRP receptor, and SecY translocon, which mediate movement into the thylakoid lumen (step 12). After the thylakoid-targeting sequence is removed in
the thylakoid lumen by a separate endoprotease, the protein folds into its mature conformation (step 13). In the pH -dependent pathway (right), metal-binding proteins fold in the stroma, and complex redox cofactors are added (step 14). Two arginine residues (RR) at the N -terminus of the thylakoid-targeting sequence and a pH gradient across the inner membrane are required for transport of the folded protein into the thylakoid lumen (step 15). The translocon in the thylakoid membrane is composed of at least four proteins related to proteins in the bacterial plasma membrane. The thylakoid-targeting sequence containing the two arginine residues is cleaved in the thylakoid lumen (step 16). See R. Dalbey and C. Robinson, 1999, Trends Biochem. Sci. 24:17; R. E. Dalbey and A. Kuhn, 2000, Annu. Rev. Cell Dev. Biol. 16:51; and C. Robinson and A. Bolhuis, 2001, Nat. Rev. Mol. Cell Biol. 2:350.

## KEY CONCEPTS OF SECTION 13.4

## Targeting of Proteins to Mitochondria and Chloroplasts

- Most mitochondrial and chloroplast proteins are encoded by nuclear genes, synthesized on cytosolic ribosomes, and imported post-translationally into the organelles.
- All the information required to target a precursor protein from the cytosol to the mitochondrial matrix or chloroplast stroma is contained within its N -terminal targeting sequence. After protein import, the targeting sequence is removed by proteases within the matrix or stroma.
- Cytosolic chaperones maintain the precursors of mitochondrial and chloroplast proteins in an unfolded state. Only unfolded proteins can be imported into the organelles. Translocation in mitochondria occurs at sites where the outer and inner membranes of the organelles are close together.
- Proteins destined for the mitochondrial matrix bind to receptors on the outer mitochondrial membrane and are then transferred to the general import pore (Tom40) in the outer membrane. Translocation through the outer and inner membranes occurs concurrently. Translocation is driven by ATP hydrolysis by Hsp70 in the matrix (see Figure 13-24) and by the proton-motive force across the inner membrane.
- Proteins sorted to mitochondrial destinations other than the matrix usually contain two or more targeting sequences, one of which may be an N -terminal matrix-targeting sequence (see Figure 13-26).
- Some mitochondrial proteins destined for the intermembrane space or inner membrane are first imported into the matrix and then redirected; others never enter the matrix, but go directly to their final location.
- Protein import into the chloroplast stroma occurs through outer-membrane and inner-membrane translocation channels that are analogous in function to mitochondrial channels, but composed of proteins unrelated in sequence to the corresponding mitochondrial proteins.
- Proteins destined for the thylakoid have secondary targeting sequences. After entry of these proteins into the stroma, cleavage of the stromal-targeting sequences reveals the thylakoid-targeting sequences.
- The four known pathways for moving proteins from the chloroplast stroma to the thylakoid closely resemble translocation across the bacterial plasma membrane (see Figure 13-29). One of these systems can translocate folded proteins.


### 13.5 Targeting of Peroxisomal Proteins

Peroxisomes are small organelles bounded by a single membrane. Unlike mitochondria and chloroplasts, peroxisomes lack DNA and ribosomes. Thus all luminal peroxisomal proteins
are encoded by nuclear genes, synthesized on free ribosomes in the cytosol, and then incorporated into preexisting or newly generated peroxisomes. As peroxisomes are enlarged by addition of protein (and lipid), they eventually divide, forming new ones, as is the case with mitochondria and chloroplasts.

The size and enzyme content of peroxisomes vary considerably among different kinds of cells. However, all peroxisomes contain enzymes that use molecular oxygen to oxidize various substrates such as amino acids and fatty acids, breaking them down into smaller components for use in biosynthetic pathways. The hydrogen peroxide $\left(\mathrm{H}_{2} \mathrm{O}_{2}\right)$ generated by these oxidation reactions is extremely reactive and potentially harmful to cellular components; however, the peroxisome contains other enzymes, such as catalase, that efficiently convert $\mathrm{H}_{2} \mathrm{O}_{2}$ into $\mathrm{H}_{2} \mathrm{O}$. In mammals, peroxisomes are most abundant in liver cells, where they constitute about $1-2$ percent of the cell volume.

## A Cytosolic Receptor Targets Proteins with an SKL Sequence at the C-Terminus to the Peroxisomal Matrix

Peroxisomal-targeting sequences were first identified by testing of peroxisomal proteins with deletions for a specific defect in peroxisomal targeting. In one early study, the gene for firefly luciferase was expressed in cultured insect cells, and the resulting protein was shown to be properly targeted to the peroxisome. However, expression of a gene missing a small portion of the sequence encoding the C -terminus of the protein led to luciferase that failed to be targeted to the peroxisome and remained in the cytoplasm. By testing various mutant luciferase proteins in this system, researchers discovered that the sequence Ser-Lys-Leu (SKL in one-letter code) or a related sequence at the C-terminus is necessary for peroxisomal targeting. Furthermore, addition of the SKL sequence to the C-terminus of a normally cytosolic protein leads to uptake of the altered protein by peroxisomes in cultured cells. All but a few of the many different peroxisomal proteins bear a sequence of this type, known as peroxisomaltargeting sequence 1, or simply PTS1.

The pathway for the import of catalase and other PTS1bearing proteins into the peroxisomal matrix is depicted in Figure 13-30. In the cytosol, PTS1 binds to a receptor called Pex5. Pex 5 has the remarkable property of being able to switch from a monomeric soluble form to an oligomeric form embedded in the peroxisomal membrane in a complex with the membrane protein Pex14. In a manner that is not well understood, the PTS1-bearing protein is released from the oligomeric form of Pex 5 into the interior of the peroxisome. This peroxisome import machinery, unlike most systems that mediate protein import into the ER, mitochondria, and chloroplasts, can translocate folded proteins across the membrane. For example, catalase assumes a folded conformation and binds to heme in the cytoplasm before traversing the peroxisomal membrane. Cell-free studies have shown that the peroxisome import machinery can transport large macromolecular objects, including gold particles about 9 nm in diameter, as long as they have a PTS1 tag attached to them.

![li1464542480e](li1464542480e.jpg)

FIGURE 13-30 PTS1-directed import of peroxisomal matrix proteins. Step 1 Most peroxisomal matrix proteins contain a C-terminal PTS1 targeting sequence (red), which binds to the cytosolic receptor Pex5. Step Pex5 with the bound matrix protein forms a multimeric complex with the Pex14 receptor located on the peroxisomal membrane. Step After assembly of the matrix protein-Pex5-Pex14 complex, the matrix protein dissociates from Pex5 and is released into the peroxisomal matrix. Steps $\mathbf{1}$ and $\mathbf{2}$ Pex5 is then returned to the cytosol by a process that involves ubiquitinylation by the membrane proteins Pex2, Pex10, and Pex12, followed by ATP-dependent removal from the membrane by the AAA-ATPase proteins Pex1 and Pex6. Note that folded proteins can be imported into peroxisomes and that the targeting sequence is not removed in the matrix. See P. E. Purdue and P. B. Lazarow, 2001, Annu. Rev. Cell Dev. Biol. 17:701; S. Subramani et al., 2000, Annu. Rev. Biochem. 69:399; and V. Dammai and S. Subramani, 2001, Cell 105:187.

There is evidence that the size of oligomers of Pex5 bound to PTS1-bearing cargo molecules and Pex14 adjusts according to the size of the PTS1-bearing cargo molecules. The dynamic formation of oligomers is apparently the key mechanism by which PTS1-bearing cargo molecules can be accommodated without the formation of large stable pores that would disrupt the integrity of the peroxisomal membrane.

Once the PTS1-bearing cargo molecule is released into the interior of the peroxisome, the oligomeric complex of Pex5 and Pex14 is actively disassembled, thus releasing Pex5 back into the cytoplasm in a soluble state. Pex5 recycling involves modification of membrane-bound Pex5 by ubiquitinylation. A complex of the peroxisomal membrane proteins Pex10, Pex12, and Pex2 transfers a ubiquitin moiety to Pex5. The AAA-ATPases Pex1 and Pex6, anchored to the peroxisomal membrane by Pex15, recognize ubiquitinylated Pex5 and use the energy from ATP
![li1474542480e](li1474542480e.jpg)
(b) Pex12 mutants (deficient in matrix protein import)
![li1484542480e](li1484542480e.jpg)
(c) Pex3 mutants (deficient in membrane biogenesis)
![li1494542480e](li1494542480e.jpg)

EXPERIMENTAL FIGURE 13-31 Studies reveal different pathways for incorporation of peroxisomal membrane and matrix proteins. Cells were stained with fluorescent antibodies to PMP70, a peroxisomal membrane protein, or with fluorescent antibodies to catalase, a peroxisomal matrix protein, then viewed in a fluorescence microscope. (a) In wild-type cells, both peroxisomal membrane and matrix proteins are visible as bright foci in numerous peroxisomal bodies. (b) In cells from a Pex12-deficient patient, catalase is distributed uniformly throughout the cytosol, whereas PMP70 is localized normally to peroxisomal bodies. (c) In cells from a Pex3-deficient patient, peroxisomal membranes cannot assemble, and as a consequence, peroxisomal bodies do not form. Thus both catalase and PMP70 are mis-localized to the cytosol. [Courtesy of Stephen Gould, Johns Hopkins University School of Medicine.]
hydrolysis to remove it from the oligomeric complex with Pex14 and release it into the cytosol. After removal of the ubiquitin modification, cytosolic Pex5 is ready to carry out another cycle of binding to a PTS1-bearing protein.

Peroxisomal import studies with purified components have shown that the binding of Pex5 to a PTS1-bearing protein, the assembly of an oligomeric complex of Pex5 and Pex14, and release of the PTS1-bearing protein into the interior of the peroxisome can all occur spontaneously without a source of chemical energy such as ATP. In contrast, both ubiquitin modification of Pex5 and the recycling of Pex5 by the AAA-ATPase are powered by ATP hydrolysis. Evidently, the recycling step in the import process uses energy to power

![li1504542480e](li1504542480e.jpg)

FIGURE 13-32 Model of peroxisomal biogenesis and division. The first stage in the de novo formation of peroxisomes is the incorporation of peroxisomal membrane proteins into precursor membranes derived from the ER. Pex19 acts as the receptor for membranetargeting sequences. A complex of Pex3 and Pex16 is required for proper insertion of proteins (e.g., PMP70) into the forming peroxisomal membrane. Insertion of all peroxisomal membrane proteins produces a peroxisomal ghost, which is capable of importing proteins targeted
unidirectional translocation of cargo molecules across the peroxisomal membrane.

A few peroxisomal matrix proteins, such as thiolase, are synthesized as precursors with an N-terminal targeting sequence known as PTS2. These proteins bind to a different cytosolic receptor protein, but otherwise their import is thought to occur by the same mechanism as for PTS1containing proteins.

## Peroxisomal Membrane and Matrix Proteins Are Incorporated by Different Pathways

Autosomal recessive mutations that cause defective peroxisome assembly occur naturally in the human population. Such mutations can lead to severe developmental defects often associated with craniofacial abnormalities. In Zellweger syndrome and related disorders, for example, the transport of many or all proteins into the peroxisomal matrix is impaired: newly synthesized peroxisomal enzymes remain in the cytosol and are eventually degraded. Genetic analyses of cultured cells from patients with Zellweger syndrome and of yeast cells carrying similar mutations have identified more than 20 genes that are required for peroxisome biogenesis.

Studies with peroxisome-assembly mutants have shown that different pathways are used for importing peroxisomal matrix proteins and for inserting proteins into the peroxisomal membrane (Figure 13-31). For example, analysis of cells from some patients with Zellweger syndrome led to the identification of the gene encoding Pex5 as well as many of the Pex genes needed for recycling of Pex5. Mutant cells that are defective in any one of these proteins cannot incorporate matrix proteins into peroxisomes; nonetheless, the cells contain empty peroxisomes that have a normal complement of peroxisomal membrane proteins (Figure 13-31b).
to the matrix. The pathways for importing PTS1- and PTS2-bearing matrix proteins differ only in the identity of the cytosolic receptor (Pex5 and Pex7, respectively) that binds the targeting sequence (see Figure 13-30). Complete incorporation of matrix proteins yields a mature peroxisome. Although peroxisomes can form de novo as just described, under most conditions, the proliferation of peroxisomes involves the division of mature peroxisomes, a process that depends on the Pex11 protein.

Mutations in any one of three other genes were found to block insertion of peroxisomal membrane proteins as well as import of matrix proteins (Figure 13-31c). These findings demonstrate that one set of proteins translocates soluble proteins into the peroxisomal matrix, but a different set is required for insertion of proteins into the peroxisomal membrane. This situation differs markedly from that of the ER, mitochondrion, and chloroplast, whose membrane proteins and soluble proteins share many of the same components for their insertion into these organelles.

Although most peroxisomes are generated by the division of preexisting organelles, these organelles can arise de novo by the three-stage process depicted in Figure 13-32. In this case, peroxisome assembly begins in the ER. At least two peroxisomal membrane proteins, Pex3 and Pex16, are inserted into the ER membrane by the mechanisms described in Section 13.2. Pex3 and Pex16 then recruit Pex19 to form a specialized region of the ER membrane that can bud off of the ER to form a peroxisomal precursor membrane. Peroxisomal membrane protein assembly into mature peroxisomes also requires Pex19, which appears to act as a soluble receptor for peroxisomal membrane protein targeting sequences, as well as Pex3 and Pex16, which serve as membrane receptors for Pex19. The insertion of a complete set of peroxisomal membrane proteins produces membranes that have all the components necessary for import of matrix proteins, leading to the formation of mature, functional peroxisomes.

Division of mature peroxisomes, which largely determines the number of peroxisomes within a cell, depends on still another protein, Pex11. Overexpression of the Pex11 protein causes a large increase in the number of peroxisomes, suggesting that this protein controls the extent of peroxisome division. The small peroxisomes generated by division can be enlarged by incorporation of additional matrix and membrane proteins via the same pathways described previously.

## KEY CONCEPTS OF SECTION 13.5

## Targeting of Peroxisomal Proteins

- All luminal peroxisomal proteins are synthesized on free cytosolic ribosomes and incorporated into the organelle post-translationally.
- Most peroxisomal matrix proteins contain a C-terminal targeting sequence known as PTS1; a few have an N-terminal PTS2 targeting sequence. Neither targeting sequence is cleaved after import.
- All proteins destined for the peroxisomal matrix bind to a cytosolic receptor protein, which differs for PTS1- and PTS2-bearing proteins, and then are directed to common translocation machinery on the peroxisomal membrane (see Figure 13-30).
- Translocation of matrix proteins across the peroxisomal membrane depends on ATP hydrolysis. Unlike proteins imported to the ER, mitochondrion, or chloroplast, many peroxisomal matrix proteins fold in the cytosol and traverse the membrane in a folded conformation.
- Proteins destined for the peroxisomal membrane contain different targeting sequences than peroxisomal matrix proteins and are imported by a different pathway.
- Unlike mitochondria and chloroplasts, peroxisomes can arise de novo from precursor membranes derived from the ER as well as by division of preexisting organelles (see Figure 13-32).


### 13.6 Transport Into and Out of the Nucleus

The nucleus is separated from the cytoplasm by two membranes, which form the nuclear envelope (see Figure 1-12a). The nuclear envelope is continuous with the ER and forms a part of it. Transport of proteins from the cytoplasm into the nucleus and movement of macromolecules, including mRNAs, tRNAs, and ribosomal subunits, out of the nucleus occur through nuclear pores, which span both membranes of the nuclear envelope. Import of proteins into the nucleus shares some fundamental features with protein import into other organelles. For example, imported nuclear proteins carry specific targeting sequences known as nuclear-localization signals (NLSs). However, proteins are imported into the nucleus in a folded state, and thus nuclear import differs fundamentally from protein translocation across the membranes of the ER, mitochondrion, and chloroplast, during which proteins are unfolded. In this section, we discuss the main mechanism by which proteins enter and exit the nucleus. We also discuss the process by which mRNAs and other ribonuclear protein complexes are exported from the nucleus, which differs mechanistically from nuclear protein import.

## Large and Small Molecules Enter and Leave the Nucleus via Nuclear Pore Complexes

Numerous nuclear pores perforate the nuclear envelope in all eukaryotic cells. Each nuclear pore is formed from an elaborate structure termed the nuclear pore complex (NPC), which is one of the largest protein assemblages in the cell. The total mass of the pore structure is $60,000-80,000 \mathrm{kDa}$ in vertebrates, which is about 16 times larger than a ribosome. An NPC is made up of multiple copies of some 30 different proteins called nucleoporins. Electron micrographs of nuclear pore complexes reveal a membrane-embedded ring structure that surrounds a largely aqueous pore (Figure 13-33). Eight approximately $100-\mathrm{nm}$-long filaments extend into the nucleoplasm with the distal ends of these filaments joined by a terminal ring, forming a structure called the nuclear basket. Cytoplasmic filaments extend from the cytoplasmic side of the NPC into the cytosol.

Ions, small metabolites, and globular proteins up to about 40 kDa can diffuse passively through the central aqueous region of the nuclear pore complex. However, large proteins and ribonucleoprotein complexes cannot diffuse in and out of the nucleus. Rather, these macromolecules are actively transported through the NPC with the assistance of soluble transport proteins that bind macromolecules and also interact with nucleoporins. The capacity and efficiency of the NPC for such active transport is remarkable. In one minute, each NPC is estimated to import 60,000 protein molecules into the nucleus, while exporting $50-250 \mathrm{mRNA}$ molecules, 10-20 ribosomal subunits, and 1000 tRNAs out of the nucleus.

In general terms, the nucleoporins are of three types: structural nucleoporins, membrane nucleoporins, and FGnucleoporins. The structural nucleoporins form the scaffold of the nuclear pore, which is a ring of eightfold rotational symmetry that traverses both membranes of the nuclear envelope, creating an annulus. The inner and outer membranes of the nuclear envelope are connected at the NPC by a highly curved region of membrane that contains the embedded membrane nucleoporins (Figure 13-33b). A set of seven structural nucleoporins forms a Y-shaped structure about the size of the ribosome, known as the Y-complex. Sixteen copies of the Y-complex form the basic structural scaffold of the pore, which has bilateral symmetry across the nuclear envelope and eightfold rotational symmetry in the plane of the envelope (Figure 13-33c). A structural motif repeated several times within the Y-complex is closely related to a structure found in the COPII proteins that drive the formation of coated vesicles within cells (see Chapter 14). This primordial relationship between structural nucleoproteins and vesicle coat proteins suggests that the two types of membrane coat complexes share a common origin. The basic function of this element may be to form a protein lattice that, in a complex with membrane nucleoporins, deforms the membrane into a highly curved structure.

The FG-nucleoporins, which line the channel of the nuclear pore complex and are also found associated with the nuclear basket and the cytoplasmic filaments, contain multiple repeats of short hydrophobic sequences that are rich in phenylalanine ( F ) and glycine (G) residues (FG-repeats).

![li1514542480e](li1514542480e.jpg)

FIGURE 13-33 Nuclear pore complex at different levels of resolution. (a) Nuclear envelopes from the large nuclei of Xenopus oocytes, visualized by scanning electron microscopy. Top: View of the cytoplasmic face reveals the octagonal shape of the membraneembedded portion of nuclear pore complexes. Bottom: View of the nucleoplasmic face shows the nuclear basket that extends from the membrane-embedded portion. (b) Cutaway model of the nuclear pore complex, showing the major structural features formed by membrane nucleoporins, structural nucleoporins, and FG-nucleoporins. (c) Sixteen copies of the Y-complex form a major part of the structural scaffold of the nuclear pore complex. The three-dimensional structure of the Y-complex is modeled into the pore structure. Note the twofold symmetry across the double membrane of the nucleus (left) and the eightfold rotational symmetry around the axis of the pore (right). (d) The FG-nucleoporins have extended disordered structures that are composed of repeats of the sequence Phe-Gly interspersed with hydrophilic regions (left). The FG-nucleoporins are most abundant in the central part of the pore, and the FG-repeat sequences are thought to fill the central channel with a gel-like matrix (right), See K. Ribbeck and D. Görlich, 2001, EMBO J. 20:1320-1330 and M. P. Rout and J. D. Atchison, 2001, J. Biol. Chem. 276:16593. [Part (a) republished with permission of Elsevier, from Doye, V. and Hurt, E., "Nucleoporins to Nuclear Pore Complexes," Curr. Opin. Cell Biol. 9(3):401-411, 1997; permission conveyed through the Copyright Clearance Center, Inc.]

The hydrophobic FG-repeats are thought to occur in regions of extended, otherwise hydrophilic polypeptide chains that fill the central transporter channel. The FG-nucleoporins are essential for the function of the NPC; however, the NPC remains functional even if up to half of the FG-repeats have been deleted. The FG-nucleoporins are thought to form a flexible gel-like matrix with bulk properties that allow the diffusion of small molecules while excluding unchaperoned hydrophilic proteins larger than 40 kDa (Figure 13-33d).

## Nuclear Transport Receptors Escort Proteins Containing Nuclear-Localization Signals into the Nucleus

All proteins found in the nucleus-such as histones, transcription factors, and DNA and RNA polymerases-are synthesized in the cytoplasm and imported into the nucleus through nuclear pore complexes. Such proteins contain a nuclear-localization signal that directs their selective transport into the nucleus. NLSs were first discovered through the analysis of mutations of the gene for large T-antigen encoded by simian virus 40 (SV40). The wild-type form of large T-antigen is localized to the nucleus in virus-infected cells, whereas some mutated forms of large T-antigen accumulate in the cytoplasm (Figure 13-34). The mutations responsible for this altered cellular localization all occur within a specific seven-residue sequence rich in basic amino acids near
![li1524542480e](li1524542480e.jpg)

EXPERIMENTAL FIGURE 13-34 Nuclear-localization signals (NLSs) direct proteins to the cell nucleus. Cytoplasmic proteins can be transported to the nucleus if they are fused to a nuclear-localization signal. (a) Normal pyruvate kinase, here visualized by immunofluorescence after cultured cells were treated with a specific antibody (yellow), is localized to the cytoplasm. This very large cytosolic protein functions in carbohydrate metabolism. (b) When a chimeric pyruvate kinase containing the SV40 NLS at its N -terminus was expressed in cells, it was localized to the nucleus. The chimeric protein was expressed from a transfected engineered gene produced by fusing a viral gene fragment encoding the SV40 NLS to the pyruvate kinase gene. [Republished with permission of Elsevier, from D. Kalderon et al., "A Short Amino Acid Sequence Able to Specify Nuclear Location," Cell 39(3 Pt 2): 499-509, 1984; permission conveyed through the Copyright Clearance Center Inc.]
the C-terminus of the protein: Pro-Lys-Lys-Lys-Arg-Lys-Val. Experiments with chimeric proteins in which this sequence was fused to a cytosolic protein demonstrated that it directs transport into the nucleus and consequently functions as an NLS. NLS sequences were subsequently identified in numerous other proteins imported into the nucleus. Many of these sequences are similar to the basic NLS in SV40 large T-antigen, whereas others are chemically quite different. For instance, an NLS in the RNA-binding protein hnRNP A1 is hydrophobic. Accordingly, there are multiple mechanisms for the recognition of these very different sequences.

Early work on the mechanism of nuclear import showed that proteins containing a basic NLS, similar to the one in SV40 large T-antigen, were efficiently transported into isolated nuclei only if they were provided with a cytosolic extract (Figure 13-35). Using this assay system, researchers purified two required cytosolic components: Ran and a nuclear transport receptor. Ran is a small monomeric G protein that exists
(a) Effect of digitonin
![li1534542480e](li1534542480e.jpg)
(b) Nuclear import by permeabilized cells
![li1544542480e](li1544542480e.jpg)

EXPERIMENTAL FIGURE 13-35 Cytosolic proteins are required for nuclear transport. The failure of nuclear transport to occur in permeabilized cultured cells in the absence of cell lysate demonstrates the involvement of soluble cytosolic components in the process. (a) Phase-contrast micrographs of untreated and digitoninpermeabilized HeLa cells. Treatment of a monolayer of cultured cells with the mild non-ionic detergent digitonin permeabilizes the plasma membrane so that cytosolic constituents leak out, but leaves the nuclear envelope and NPCs intact. (b) Fluorescence micrographs of digitonin-permeabilized HeLa cells incubated with a fluorescent protein chemically coupled to a synthetic SV40 T-antigen NLS peptide in the presence and absence of cell lysate (cytosol). Accumulation of the transport substrate in the nucleus occurred only when cytosol was present (right). [01990 S. Adam et al., The Journal of Cell Biology 111:807-816. doi:10.1083/jcb.111.3.807]

![li1554542480e](li1554542480e.jpg)

FIGURE 13-36 Mechanism for nuclear import of proteins. In the cytoplasm (top), free importin binds to the NLS of a cargo protein, forming an importin-cargo complex. The importin-cargo complex diffuses through the NPC by transiently interacting with FG-nucleoporins. In the nucleoplasm, Ran-GTP binds to the importin, causing a conformational change that decreases the importin's affinity for the NLS and releasing the cargo. To support another cycle of import, the importin-Ran-GTP complex is transported back to the cytoplasm. A GTPaseactivating protein (GAP) associated with the cytoplasmic filaments of the NPC stimulates Ran to hydrolyze the bound GTP. This generates a conformational change that causes Ran to dissociate from the nuclear transport receptor, which can then initiate another round of import. Ran-GDP is returned to the nucleoplasm, where a guanine nucleotide exchange factor (GEF) causes release of GDP and rebinding of GTP.
in either a GTP-bound or a GDP-bound conformation (see Figure 3-34). It is the cycling of Ran between GTP-bound and GDP-bound conformations, leading to the net hydrolysis of GTP to GDP, that ultimately provides the energy to drive unidirectional transport of macromolecules through the nuclear pore. A nuclear transport receptor binds to the NLS on a cargo protein to be transported into the nucleus and to FG-repeats on nucleoporins. By a physical process that is not well understood, by binding transiently to FG-repeats, nuclear transport receptors have the ability to rapidly traverse the FG-repeat-containing matrix in the central channel of the nuclear pore, whereas proteins of similar size that lack this property are excluded from the central channel. Nuclear transport receptors can be monomeric, consisting of a single polypeptide that can bind both to an NLS and to FG-repeats, or they can be dimeric, with one subunit binding to the NLS and the other binding to FG-repeats.

The mechanism for the import of cytoplasmic cargo proteins mediated by a nuclear transport receptor known as importin is shown in Figure 13-36. Free importin in the cytoplasm binds to its cognate NLS in a cargo protein, forming an importin-cargo complex. The cargo complex then translocates through the NPC channel as the importin interacts with FGrepeats. The cargo complex rapidly reaches the nucleoplasm, and there the importin interacts with Ran-GTP, which causes a conformational change in the importin that displaces the NLS, releasing the cargo protein into the nucleoplasm. The importin-Ran-GTP complex then diffuses back through the NPC. Once the importin-Ran-GTP complex reaches the cytoplasmic side of the NPC, Ran interacts with a specific GTPase-activating protein (Ran-GAP) that is a component of the NPC cytoplasmic filaments. This interaction stimulates Ran to hydrolyze its bound GTP to GDP, which causes it to convert to a conformation that has low affinity for the importin, so that the importin is released into the cytoplasm, where it can participate in another cycle of import. Ran-GDP travels back through the pore to the nucleoplasm, where it encounters a specific guanine nucleotide exchange factor (Ran-GEF) that causes Ran to release its bound GDP in favor of GTP. The net result of this series of reactions is the coupling of the hydrolysis of GTP to the transfer of an NLSbearing protein from the cytoplasm to the nuclear interior, thus providing the energy to drive nuclear import.

Although the importin-cargo complex travels through the pore by random diffusion, the overall process of transport of cargo into the nucleus is unidirectional. Because of the rapid dissociation of the complex when it reaches the nucleoplasm, there is a concentration gradient of importin-cargo complex across the NPC: high in the cytoplasm, where the complex assembles, and low in the nucleoplasm, where it dissociates. This concentration gradient is responsible for the unidirectional nature of nuclear import. A similar concentration gradient is responsible for driving importin from the nucleus back into the cytoplasm. The concentration of the importin-Ran-GTP complex is higher in the nucleoplasm, where it assembles, than on the cytoplasmic side of the NPC, where it dissociates. Ultimately, the direction of the transport processes depends on the localization of the Ran-GEF predominantly in the nucleoplasm and the Ran-GAP predominantly in the cytoplasm. Ran-GEF in the nucleoplasm maintains Ran in the Ran-GTP state, where it promotes dissociation of the cargo complex. Ran-GAP on the cytoplasmic side of the NPC converts Ran-GTP to Ran-GDP, dissociating the importin-Ran-GTP complex and releasing free importin into the cytosol.

## A Second Type of Nuclear Transport Receptor Escorts Proteins Containing Nuclear-Export Signals Out of the Nucleus

A mechanism very similar to the one we have just described is used to export proteins, tRNAs, and ribosomal subunits from the nucleus to the cytoplasm. This mechanism was initially elucidated by studies of certain ribonuclear protein

![li1564542480e](li1564542480e.jpg)
(b)
![li1574542480e](li1574542480e.jpg)

FIGURE 13-37 Ran-dependent and Ran-independent nuclear export. (a) Randependent mechanism for nuclear export of cargo proteins containing a leucine-rich nuclear-export signal (NES). In the nucleoplasm (bottom), the protein exportin 1 binds cooperatively to the NES of the cargo protein to be transported and to Ran-GTP. After the resulting cargo complex diffuses through an NPC via transient interactions with FG-repeats in FG-nucleoporins, the GAP associated with the NPC cytoplasmic filaments stimulates GTP hydrolysis, converting Ran-GTP to Ran-GDP. The accompanying conformational change in Ran leads to dissociation of the complex. The NES-containing cargo protein is released into the cytosol, whereas exportin 1 and Ran-GDP are transported back into the nucleus through an NPC. Ran-GEF in the nucleoplasm then stimulates conversion of Ran-GDP to Ran-GTP. (b) Ran-independent nuclear export of mRNAs. The heterodimeric NXF1/NXT1 complex binds to mRNA-protein complexes (mRNPs) in the nucleus. NXF1/NXT1 acts as a nuclear transport receptor and directs the associated mRNP to the central channel of the NPC by transiently interacting with FGnucleoporins. An RNA helicase (Dbp5) located on the cytoplasmic side of the NPC removes NXF1 and NXT1 from the mRNA in a reaction that is powered by ATP hydrolysis. Free NXF1 and NXT1 proteins are recycled back into the nucleus by the Ran-dependent import process depicted in Figure 13-36.
complexes that "shuttle" between the nucleus and the cytoplasm. Such "shuttling" proteins contain a nuclear-export signal (NES) that stimulates their export from the nucleus to the cytoplasm through nuclear pores, in addition to an NLS that results in their uptake into the nucleus. Experiments with engineered hybrid genes encoding a nucleus-restricted
protein fused to various segments of a shuttling protein have identified at least three different types of NESs: a leucine-rich sequence found in PKI (an inhibitor of protein kinase A) and in the Rev protein of human immunodeficiency virus (HIV), and two other sequences identified in two different heterogeneous ribonucleoprotein particles (hnRNPs). The precise

structural features that determine the recognition of each type of sequence for nuclear export remain poorly understood.

The mechanism whereby shuttling proteins are exported from the nucleus is best understood for those containing a leucine-rich NES. According to the current model, shown in Figure 13-37a, a specific nuclear transport receptor, called exportin 1, first forms a complex with Ran-GTP in the nucleus and then binds the NES in a cargo protein. Binding of exportin 1 to Ran-GTP causes a conformational change in exportin 1 that increases its affinity for the NES, so that a trimolecular cargo complex is formed. Like other nuclear transport receptors, exportin 1 interacts transiently with FGrepeats in FG-nucleoporins and diffuses through the NPC. The cargo complex dissociates when it encounters the RanGAP associated with the NPC cytoplasmic filaments, which stimulates Ran to hydrolyze the bound GTP, shifting it into a conformation that has low affinity for exportin 1. After Ran-GDP dissociates from the trimolecular cargo complex, exportin 1 changes its conformation to one that has low affinity for the NES, releasing the cargo into the cytosol. The direction of the export process is driven by this dissociation of the cargo from exportin 1 in the cytoplasm, which causes a concentration gradient of the cargo complex across the NPC that is high in the nucleoplasm and low in the cytoplasm. Exportin 1 and Ran-GDP are then transported back into the nucleus through the NPC.

By comparing this model for nuclear export with that in Figure 13-36 for nuclear import, we can see one obvious difference: Ran-GTP is part of the cargo complex during export, but not during import. Apart from this difference, the two transport processes are remarkably similar. In both processes, association of a nuclear transport receptor with Ran-GTP in the nucleoplasm causes a conformational change that affects its affinity for the transport signal. During import, the interaction causes release of the cargo, whereas during export, the interaction promotes association with the cargo. In both export and import, stimulation of Ran-GTP hydrolysis in the cytoplasm by Ran-GAP produces a conformational change in Ran that releases the nuclear transport receptor. During nuclear export, the cargo is also released. Localization of Ran-GAP and Ran-GEF to the cytoplasm and nucleus, respectively, is the basis for the unidirectional import and export of cargo proteins across the NPC.

In keeping with their similarity in function, the two types of nuclear transport receptors-importins and exportinsare highly homologous in sequence and structure. The family of nuclear transport receptors has 14 members in yeast and more than 20 in mammalian cells. The NESs or NLSs to which they bind have been determined for only a fraction of them. Some individual nuclear transport receptors function in both import and export.

A similar shuttling mechanism has been shown to export other cargoes from the nucleus. For example, exportin-t functions to export tRNAs. Exportin-t binds fully processed tRNAs in a complex with Ran-GTP that diffuses through NPCs and dissociates when it interacts with Ran-GAP in the NPC cytoplasmic filaments, releasing the tRNA into the
cytosol. A Ran-dependent process is also required for the nuclear export of ribosomal subunits through NPCs once the protein and RNA components have been properly assembled in the nucleolus. Likewise, certain specific mRNAs that associate with particular hnRNP proteins can be exported by a Ran-dependent mechanism.

## Most mRNAs Are Exported from the Nucleus by a Ran-Independent Mechanism

Once the processing of an mRNA is completed in the nucleus, it remains associated with specific proteins in a messenger ribonuclear protein complex, or $m R N P$. The principal transporter of mRNPs out of the nucleus is the mRNP exporter, a heterodimeric protein composed of a large subunit called nuclear export factor 1 (NXF1) and a small subunit called nuclear export transporter 1 (NXT1). Multiple NXF1/NXT1 dimers bind to nuclear mRNPs through cooperative interactions with the RNA and other mRNP adapter proteins that associate with nascent pre-mRNAs during transcriptional elongation and pre-mRNA processing. In many respects, the subunits of NXF1/NXT1 act like a nuclear transport receptor that binds to an NLS or NES, since both subunits interact with the FG-repeats of FG-nucleoporins, and this interaction allows them to diffuse through the central channel of the NPC.

The process of mRNP export does not require Ran, and thus the unidirectional transport of mRNA out of the nucleus requires a source of energy other than GTP hydrolysis by Ran. Once the mRNP-NXF1/NXT1 complex reaches the cytoplasmic side of the NPC, NXF1 and NXT1 dissociate from the mRNP with the help of the RNA helicase Dbp5, which is associated with cytoplasmic NPC filaments. Recall that RNA helicases use the energy derived from hydrolysis of ATP to move along RNA molecules, separating doublestranded RNA chains and dissociating RNA-protein complexes (see Chapter 5). This leads to the simple idea that Dpb5, which is associated with the cytoplasmic side of the nuclear pore complex, acts as an ATP-driven motor to remove NXF1/NXT1 from the mRNP complexes as they emerge on the cytoplasmic side of the NPC. The assembly of NXF1/NXT1 onto mRNPs on the nucleoplasmic side of the NPC and the subsequent ATP-dependent removal of NXF1/ NXT1 from mRNPs on the cytoplasmic side of the NPC creates a concentration gradient of mRNP-NXF1/NXT1, which drives unidirectional export. After being removed from the mRNP, the free NXF1 and NXT1 subunits that have been stripped from the mRNA by Dbp5 helicase are imported back into the nucleus by a process that depends on Ran and a nuclear transport receptor (Figure 13-37b).

In Ran-dependent nuclear export (discussed in the previous subsection), hydrolysis of GTP by Ran on the cytoplasmic side of the NPC causes dissociation of the nuclear transport receptor from its cargo. In basic outline, the Ranindependent nuclear export discussed here operates by a similar mechanism, except that Dbp5p on the cytosolic side of the NPC uses hydrolysis of ATP to dissociate the mRNP exporter from mRNA.

## KEY CONCEPTS OF SECTION 13.6

## Transport Into and Out of the Nucleus

- The nuclear envelope contains numerous nuclear pore complexes (NPCs), which are large, complex structures composed of multiple copies of 30 different proteins called nucleoporins (see Figure 13-33). FG-nucleoporins, which contain multiple repeats of a short hydrophobic sequence (FG-repeats), line the central transporter channel and play a role in the transport of all macromolecules through nuclear pores.
- Transport of macromolecules larger than 40 kDa through nuclear pores requires the assistance of nuclear transport receptors that interact with both the transported molecule and FG-repeats of FG-nucleoporins.
- Proteins imported into or exported from the nucleus contain a specific amino acid sequence that functions as a nuclear-localization signal (NLS) or a nuclear-export signal (NES). Nucleus-restricted proteins contain an NLS but not an NES, whereas proteins that shuttle between the nucleus and cytoplasm contain both signals.
- Several different types of NESs and NLSs have been identified. Each type of nuclear-transport signal is thought to interact with a specific nuclear transport receptor belonging to a family of homologous proteins.
- A cargo protein bearing an NES or NLS translocates through nuclear pores bound to its cognate nuclear transport receptor. The transient interactions between nuclear transport receptors and FG-repeats allow very rapid diffusion of nuclear transport receptor-cargo complexes through the central channel of the NPC, which is filled with a hydrophobic matrix of FG-repeats.
- The unidirectional nature of protein export and import through nuclear pores results from the participation of Ran, a monomeric G protein that exists in different conformations when bound to GTP or GDP. Localization of the Ran guanine nucleotide exchange factor (Ran-GEF) in the nucleus and of the Ran GTPase-activating protein (Ran-GAP) in the cytoplasm creates a gradient with high concentrations of Ran-GTP in the nucleoplasm and of Ran-GDP in the cytoplasm. The interaction of a cargo complex with Ran-GTP in the nucleoplasm causes dissociation of the complex, releasing the cargo into the nucleoplasm (see Figure 13-36), whereas the assembly of an export cargo complex is stimulated by interaction with Ran-GTP in the nucleoplasm (see Figure 13-37).
- Most mRNPs are exported from the nucleus by binding to a heterodimeric mRNP exporter in the nucleoplasm that interacts with FG-repeats. The direction of transport (nucleus to cytoplasm) results from the action of an RNA helicase associated with the cytoplasmic filaments of the NPC that removes the heterodimeric mRNP exporter once the transport complex has reached the cytoplasm.


## LaunchPad

macmillan learning

Visit LaunchPad to access study tools and to learn more about the content in this chapter.

- Perspectives for the Future
- Analyze the Data
- Additional study tools, including videos, animations, and quizzes


## Key Terms

cotranslational
translocation 588
dislocation 607
dolichol phosphate 601
FG-nucleoporin 622
general import
pore 611
hydropathy profile 599
microsome 586
multipass membrane
protein 593
N -linked
oligosaccharide 601
nuclear pore complex
(NPC) 622
nuclear transport
receptor 625
O-linked
oligosaccharide 601
post-translational translocation 591
protein disulfide isomerase (PDI) 603
Ran protein 625
rough ER 586
signal-anchor sequence 595
signal recognition particle
(SRP) 588
single-pass membrane protein 593
stop-transfer anchor sequence 594
targeting sequence 585
topogenic sequence 593
topology 593
translocon 590
unfolded-protein
response 606

## Review the Concepts

1. The following results were obtained in early studies on the translation of secretory proteins. Based on what we now know of this process, explain the reason why each result was observed.
a. An in vitro translation system consisting only of mRNA and ribosomes resulted in secretory proteins that were larger than the identical protein when translated in a cell.
b. A similar system that also included microsomes produced secretory proteins that were identical in size to those found in a cell.
c. When the microsomes were added after in vitro translation, the synthesized proteins were again larger than those made in a cell.
2. Describe the source or sources of energy needed for unidirectional translocation across the membrane in (a) cotranslational translocation into the endoplasmic reticulum (ER);

(b) post-translational translocation into the ER; (c) translocation into the mitochondrial matrix.
3. Translocation into most organelles usually requires the activity of one or more cytosolic proteins. Describe the basic functions of three different cytosolic factors required for translocation into the ER, mitochondria, and peroxisomes, respectively.
4. Describe the typical principles used to identify topogenic sequences within proteins and how these principles can be used to develop computer algorithms. How does the identification of topogenic sequences lead to prediction of the membrane arrangement of a multipass protein? What is the importance of the arrangement of positive charges relative to the membrane orientation of a signal-anchor sequence?
5. An abundance of misfolded proteins in the ER can result in the activation of the unfolded-protein response (UPR) and ER-associated degradation (ERAD) pathways. UPR decreases the abundance of unfolded proteins by altering gene expression of what type of genes? What is one manner in which ERAD may identify misfolded proteins? Why is dislocation of these misfolded proteins to the cytoplasm necessary?
6. Temperature-sensitive yeast mutants have been isolated that block each of the enzymatic steps in the synthesis of the dolichol-linked oligosaccharide precursor for N -linked glycosylation. Propose an explanation for why mutations that block synthesis of the intermediate with the structure dolichol-PP-(GlcNAc) $)_{2} \mathrm{Man}_{5}$ completely prevent addition of N -linked oligosaccharide chains to secretory proteins, whereas mutations that block conversion of this intermediate into the completed precursor-dolichol-PP-(GlcNAc) $)_{2} \mathrm{Man}_{9} \mathrm{Glc}_{2}$ —allow the addition of N -linked oligosaccharide chains to secretory glycoproteins.
7. Name four different proteins that facilitate the modification or folding of secretory proteins within the lumen of the ER. Indicate which of these proteins covalently modifies substrate proteins and which brings about only conformational changes in substrate proteins.
8. Describe what would happen to the precursor of a mitochondrial matrix protein in the following types of mitochondrial mutants: (a) a mutation in the Tom22 signal receptor; (b) a mutation in the Tom70 signal receptor; (c) a mutation in the matrix Hsp70; and (d) a mutation in the matrix signal peptidase.
9. Describe the similarities and differences between the mechanism of import into the mitochondrial matrix and the chloroplast stroma.
10. Design a set of experiments using chimeric proteins, composed of a mitochondrial precursor protein fused to dihydrofolate reductase (DHFR), that could be used to determine how much of the precursor protein must protrude into the mitochondrial matrix in order for the matrixtargeting sequence to be cleaved by the matrix-processing protease.
11. Peroxisomes contain enzymes that use molecular oxygen to oxidize various substrates, but in the process,
hydrogen peroxide-a compound that can damage DNA and proteins-is formed. What is the name of the enzyme responsible for the breakdown of hydrogen peroxide to water? What is the mechanism of the import of this protein into the peroxisome, and what other proteins are involved? 12. Suppose that you have identified a new mutant cell line that lacks functional peroxisomes. Describe how you could determine experimentally whether the mutant is primarily defective for insertion/assembly of peroxisomal membrane proteins or matrix proteins.
13. The nuclear import of proteins larger than 40 kDa requires the presence of what amino acid sequence? Describe the mechanism of nuclear import. How are nuclear transport receptors able to get through the nuclear pore complex?
14. Why is localization of Ran-GAP in the nucleus and RanGEF in the cytoplasm necessary for unidirectional transport of cargo proteins containing an NES?

## References

## Targeting Proteins To and Across the ER Membrane

Egea, P. F., R. M. Stroud, and P. Walter. 2005. Targeting proteins to membranes: structure of the signal recognition particle. Curr. Opin. Struct. Biol. 15:213-220.

Osborne, A. R., T. A. Rapoport, and B. van den Berg. 2005. Protein translocation by the Sec61/SecY channel. Annu. Rev. Cell Dev. Biol. 21:529-550.

Wickner, W., and R. Schekman. 2005. Protein translocation across biological membranes. Science 310:1452-1456.

## Insertion of Membrane Proteins into the ER

Englund, P. T. 1993. The structure and biosynthesis of glycosylphosphatidylinositol protein anchors. Annu. Rev. Biochem. 62:121-138.

Mothes, W., et al. 1997. Molecular mechanism of membrane protein integration into the endoplasmic reticulum. Cell 89:523-533.

Shao, S., and R. S. Hegde. 2011. Membrane protein insertion at the endoplasmic reticulum. Annu. Rev. Cell Dev. Biol. 27:25-56.

Wang, F., et al. 2011. The mechanism of tail-anchored protein insertion into the ER membrane. Mol. Cell 43:738-750.

## Protein Modifications, Folding, and Quality Control in the ER

Braakman, I., and N. J. Bulleid. 2011. Protein folding and modification in the mammalian endoplasmic reticulum. Annu. Rev. Biochem. 80:71-99.

Hegde, R. S., and H. L. Ploegh. 2010. Quality and quantity control at the endoplasmic reticulum. Curr. Opin. Cell Biol. 22:437-446.

Helenius, A., and M. Aebi. 2004. Roles of N-linked glycans in the endoplasmic reticulum. Annu. Rev. Biochem. 73:1019-1049.

Kornfeld, R., and S. Kornfeld. 1985. Assembly of asparaginelinked oligosaccharides. Annu. Rev. Biochem. 45:631-664.

Patil, C., and P. Walter. 2001. Intracellular signaling from the endoplasmic reticulum to the nucleus: the unfolded protein response in yeast and mammals. Curr. Opin. Cell Biol. 13:349-355.

Sevier, C. S., and C. A. Kaiser. 2002. Formation and transfer of disulphide bonds in living cells. Nat. Rev. Mol. Cell Biol. 3:836-847.

Smith, M. H., H. L. Ploegh, and J. S. Weissman. 2011. Road to ruin: targeting proteins for degradation in the endoplasmic reticulum. Science 334:1086-1090.

## Targeting of Proteins to Mitochondria and Chloroplasts

Dalbey, R. E., and A. Kuhn. 2000. Evolutionarily related insertion pathways of bacterial, mitochondrial, and thylakoid membrane proteins. Annu. Rev. Cell Dev. Biol. 16:51-87.

Dolezal, P., et al. 2006. Evolution of the molecular machines for protein import into mitochondria. Science 313:314-318.

Koehler, C. M. 2004. New developments in mitochondrial assembly. Annu. Rev. Cell Dev. Biol. 20:309-335.

Li, H.-M., and C.-C. Chiu. 2010. Protein transport into chloroplasts. Ann. Rev. Plant Biol. 61:157-180.

Matouschek, A., N. Pfanner, and W. Voos. 2000. Protein unfolding by mitochondria: the Hsp70 import motor. EMBO Rep. 1:404-410.

Neupert, W., and M. Brunner. 2002. The protein import motor of mitochondria. Nat. Rev. Mol. Cell Biol. 3:555-565.

Robinson, C., and A. Bolhuis. 2001. Protein targeting by the twin-arginine translocation pathway. Nat. Rev. Mol. Cell Biol. 2:350-356.

Schmidt, O., N. Pfanner, and C. Meisinger. 2010. Mitochondrial protein import: from proteomics to functional mechanisms. Nat. Rev. Mol. Cell Biol. 11:655-667.

## Targeting of Peroxisomal Proteins

Dammai, V., and S. Subramani. 2001. The human peroxisomal targeting signal receptor, Pex5p, is translocated into the peroxisomal matrix and recycled to the cytosol. Cell 105:187-196.

Gould, S. J., and D. Valle. 2000. Peroxisome biogenesis disorders: genetics and cell biology. Trends Genet. 16:340-345.

Hoepfner, D., et al. 2005. Contribution of the endoplasmic reticulum to peroxisome formation. Cell 122:85-95.

Ma, C., G. Agrawal, and S. Subramani. 2011. Peroxisome assembly: matrix and membrane protein biogenesis. J. Cell Biol. 193:7-16.

Purdue, P. E., and P. B. Lazarow. 2001. Peroxisome biogenesis. Annu. Rev. Cell Dev. Biol. 17:701-752.

Smith, J. J., and J. D. Aitchison. 2013. Peroxisomes take shape. Nat. Rev. Mol. Cell Biol. 14:803-817.

## Transport Into and Out of the Nucleus

Chook, Y. M., and G. Blobel. 2001. Karyopherins and nuclear import. Curr. Opin. Struct. Biol. 11:703-715.

Cole, C. N., and J. J. Scarcelli. 2006. Transport of messenger RNA from the nucleus to the cytoplasm. Curr. Opin. Cell Biol. 18:299-306.

Johnson, A. W., E. Lund, and J. Dahlberg. 2002. Nuclear export of ribosomal subunits. Trends Biochem. Sci. 27:580-585.

Ribbeck, K., and D. Gorlich. 2001. Kinetic analysis of translocation through nuclear pore complexes. EMBO J. 20:1320-1330.

Rout, M. P., and J. D. Aitchison. 2001. The nuclear pore complex as a transport machine. J. Biol. Chem. 276:16593-16596.

Schwartz, T. U. 2005. Modularity within the architecture of the nuclear pore complex. Curr. Opin. Struct. Biol. 15:221-226.

Stewart, M. 2010. Nuclear export of mRNA. Trends Biochem. Sci. 35:609-617.

Terry, L. J., and S. R. Wente. 2009. Flexible gates: dynamic topologies and functions for FG nucleoporins in nucleocytoplasmic transport. Eukaryot. Cell 8:1814-1827.

![li1584542480e](li1584542480e.jpg)

Schematic representation of the formation of a clathrin-coated vesicle. The process starts with the recruitment of AP2 complexes (green) to the inner surface of the plasma membrane, which in turn capture a threelegged clathrin triskelion. The curvature of the underlying membrane increases as assembly of the lattice proceeds by the recruitment of additional AP2 and clathrin triskelions, until scission results in a coated vesicle formation. (Courtesy of Ema Cocucci, Janet Iwasa, and Tom Kirchhausen.)

In the previous chapter, we explored how proteins are targeted to and translocated across the membranes of several different intracellular organelles, including the endoplasmic reticulum, mitochondria, chloroplasts, peroxisomes, and the nucleus. In this chapter, we turn our attention to the secretory pathway and the mechanisms of vesicular traffic that allow proteins to be secreted from the cell or delivered to the plasma membrane and the lysosomes. We also discuss the related processes of endocytosis and autophagy, which deliver proteins and small molecules either from outside the cell or from the cytoplasm to the interior of the lysosome for degradation.

The secretory pathway is so named because it was initially studied in dedicated secretory cells that produce and secrete large quantities of proteins such as insulin or digestive

## OUTLINE

14.1 Techniques for Studying the Secretory Pathway
14.2 Molecular Mechanisms of Vesicle Budding and Fusion
14.3 Early Stages of the Secretory Pathway

## Vesicular Traffic, <br> Secretion, and <br> Endocytosis

enzymes to the outside of the cell. It was later discovered that the same pathway used for extracellular secretion of proteins is used to distribute all soluble and membrane proteins that enter the ER to their final destinations at the cell surface or in the lysosomes. Proteins delivered to the plasma membrane include cell-surface receptors, transporters for nutrient uptake, and ion channels that maintain the proper ionic and electrochemical balance across the plasma membrane. Such membrane proteins, once they reach the plasma membrane, remain embedded within it. Soluble secreted proteins also follow the secretory pathway to the cell surface, but instead of remaining embedded in the membrane, they are released into the aqueous extracellular environment. Examples of secreted proteins are digestive enzymes, peptide hormones, serum proteins, and collagen. The lysosome, as described

### 14.4 Later Stages of the Secretory Pathway

### 14.5 Receptor-Mediated Endocytosis

14.6 Directing Membrane Proteins and Cytosolic Materials to the Lysosome

in Chapter 4, is the organelle with an acidic interior that is used for degradation of unneeded proteins and storage of small molecules such as amino acids. Accordingly, the types of proteins delivered to the lysosomal membrane include subunits of the V-class proton pump that pumps $\mathrm{H}^{+}$from the cytosol into the acidic lumen of the lysosome, as well as transporters that release small molecules stored in the lysosome into the cytoplasm. Soluble proteins delivered by this pathway include lysosomal digestive enzymes such as proteases, glycosidases, phosphatases, and lipases.

In contrast to the secretory pathway, which allows proteins to be targeted to the cell surface, the endocytic pathway is used to take up substances from the cell surface and move them into the interior of the cell. The endocytic pathway can selectively remove proteins from the plasma membrane and thus has a part in regulating the protein composition of the plasma membrane. In addition, the endocytic pathway is used to ingest certain nutrients that are too large to be transported across the plasma membrane by one of the transport mechanisms discussed in Chapter 11. For example, the endocytic pathway is used in the uptake of cholesterol carried in LDL particles and of iron atoms carried by the iron-binding protein transferrin. In addition, the endocytic pathway can be used to remove receptor proteins from the cell surface as a way to down-regulate their activity.

A single unifying principle governs all protein trafficking in the secretory and endocytic pathways: transport of membrane and soluble proteins from one membrane-bounded compartment to another is mediated by transport vesicles that collect cargo proteins in buds arising from the membrane of one compartment and then deliver these cargo proteins to the next compartment by fusing with the membrane of that compartment. Importantly, as transport vesicles bud from one membrane and fuse with the next, the same face of the membrane remains oriented toward the cytosol. Therefore, once a protein has been inserted into the membrane or the lumen of the ER, that protein can be carried along the secretory pathway, moving from one organelle to the next without being translocated across another membrane or altering its orientation within the membrane. Similarly, the endocytic pathway uses vesicle traffic to transport proteins from the plasma membrane to the endosome and lysosome and thus preserves their orientation in the membranes of these organelles. Figure 14-1 outlines the main secretory and endocytic pathways in the cell.

Reduced to its simplest elements, the secretory pathway operates in two stages. The first stage takes place in the rough endoplasmic reticulum (ER) (Figure 14-1, step $\mathbf{B}$ ). As described in Chapter 13, newly synthesized soluble and membrane proteins are translocated into the ER, where they fold into their proper conformation and receive covalent modifications such as N -linked and O -linked carbohydrates and disulfide bonds. Once newly synthesized proteins are properly folded and have received their correct modifications in the ER lumen, they progress to the second stage of the secretory pathway: transport to and through the Golgi complex.

The second stage of the secretory pathway can be summarized as follows. In the ER, cargo proteins are packaged into anterograde (forward-moving) transport vesicles (Figure 14-1, step $\mathbf{B}$ ). These vesicles fuse with one another to form a flattened membrane-bounded compartment known as the cis-Golgi network or cis-Golgi cisterna (a "cistern" is a container for holding water or other liquid). Certain proteins, mainly proteins that function in the ER, can be retrieved from the cis-Golgi cisterna and returned to the ER via a different set of retrograde (backwardmoving) transport vesicles (step $\mathbf{B}$ ). In a manner reminiscent of an assembly line, the new cis-Golgi cisterna, with its cargo of proteins, physically moves from the cis position (nearest the ER) to the trans position (farthest from the ER), successively becoming first a medial-Golgi cisterna and then a trans-Golgi cisterna (step $\mathbf{B}$ ). This process, known as cisternal maturation, primarily involves retrograde transport vesicles (step $\mathbf{B}$ ), which retrieve enzymes and other Golgi-resident proteins from later to earlier Golgi cisternae, thereby "maturing" the cis-Golgi cisternae to medial-Golgi cisternae, and medial-Golgi cisternae to trans-Golgi cisternae. As secretory proteins move through the Golgi, their linked carbohydrates may be further modified by specific glycosyl transferases that are housed in the different Golgi compartments.

Proteins in the secretory pathway are eventually delivered to a complex network of membranes and vesicles termed the trans-Golgi network. The trans-Golgi network is a major branch point in the secretory pathway. It is at this stage that proteins are loaded into different kinds of vesicles and thereby trafficked to different destinations. Depending on which kind of vesicle the protein is loaded into, it will be transported to the plasma membrane and secreted immediately, stored for later release, or shipped to the lysosome (steps $\mathbf{B}-\mathbf{B}$ ). The process by which a vesicle moves to and fuses with the plasma membrane and releases its contents is known as exocytosis. In all cell types, at least some proteins are secreted continuously (a process commonly called constitutive secretion), while others are stored inside the cell until a signal for exocytosis causes them to be released (regulated secretion). Secretory proteins destined for lysosomes are first transported by vesicles from the trans-Golgi network to a compartment usually called the late endosome; the proteins are then transferred to the lysosome by direct fusion of the late endosome with the lysosomal membrane.

Endocytosis is related mechanistically to the secretory pathway. In the endocytic pathway, vesicles bud inward from the plasma membrane, bringing membrane proteins and their bound ligands into the cell (see Figure 14-1, right). After being internalized by endocytosis, some proteins are transported to lysosomes via the late endosome, whereas others are recycled back to the cell surface.

In this chapter, we first discuss the experimental techniques that have contributed to our knowledge of the secretory pathway and endocytosis. Then we focus on the general mechanisms of membrane budding and fusion. We will see that although different kinds of transport vesicles use distinct sets of proteins for their formation and fusion,

![li1594542480e](li1594542480e.jpg)

FIGURE 14-1 Overview of the secretory and endocytic pathways of protein sorting. Secretory pathway: Synthesis of proteins bearing an ER signal sequence is completed on the rough ER $\mathbf{B}$, and the newly made polypeptide chains are inserted into the ER membrane or cross it into the ER lumen (see Chapter 13). Some proteins (e.g., ER enzymes or structural proteins) remain within the ER. The remainder are packaged into transport vesicles $\mathbf{B}$ that bud from the ER and fuse to form new cis-Golgi cisternae. Missorted ER-resident proteins and vesicle membrane proteins that need to be reused are retrieved to the ER by vesicles $\mathbf{B}$ that bud from the cis-Golgi and fuse with the ER. Each cis-Golgi cisterna, with its protein content, physically moves from the cis to the trans face of the Golgi complex by a nonvesicular process called cisternal maturation. Retrograde transport vesicles move Golgi-resident proteins to the proper Golgi compartments. In all cells, certain soluble proteins move to the cell surface in transport vesicles $\mathbf{B}$ and are secreted continuously (constitutive secretion). In certain cell types, some soluble proteins are stored in secretory vesicles $\mathbf{B}$ and are released only after the cell receives an appropriate neuronal or hormonal signal (regulated secretion). Lysosome-destined membrane and soluble proteins, which are transported in vesicles that bud from the trans-Golgi $\mathbf{B}$, first move to the late endosome and then to the lysosome. Endocytic pathway: Membrane and soluble extracellular proteins taken up in vesicles that bud from the plasma membrane $\mathbf{B}$ can also move to the lysosome via the endosome.

all vesicles use the same general mechanism for budding, selection of particular sets of cargo molecules, and fusion with the appropriate target membrane. In the remaining sections of the chapter, we discuss both the early and late stages of the secretory pathway, including how specificity of targeting to different destinations is achieved, and conclude with a discussion of how proteins are transported to the lysosome by the endocytic pathway.

### 14.1 Techniques for Studying the Secretory Pathway

The key to understanding how proteins are transported through the organelles of the secretory pathway has been to develop a basic description of the function of transport vesicles. Many components required for the formation and fusion of transport vesicles have been identified by a remarkable convergence of the genetic and biochemical approaches described in this section. All studies of intracellular protein trafficking employ some method for assaying the transport of a given protein from one compartment to another. We begin by describing how intracellular protein transport can be followed in live cells and then consider genetic and in vitro systems that have proved useful in elucidating the secretory pathway.

## Transport of a Protein Through the Secretory Pathway Can Be Assayed in Live Cells

The classic studies of G. Palade and his colleagues in the 1960s first established the order in which proteins move from one organelle to the next in the secretory pathway (see Classic Experiment 14-1). These early studies also showed that secretory proteins are never released into the cytosolthe first indication that transported proteins are always associated with some type of membrane-bounded intermediate. In these experiments, which combined pulse-chase labeling (see Figure 3-42) and autoradiography, radioactively labeled amino acids were injected into the pancreas of hamsters. At different times after injection, the animals were sacrificed and the pancreatic cells were immediately fixed with glutaraldehyde, sectioned, and subjected to autoradiography to visualize the locations of the radiolabeled proteins. Because the radioactive amino acids were administered in a short pulse, only those proteins synthesized immediately after injection were labeled, forming a distinct cohort of labeled proteins whose transport could be followed. In addition, because pancreatic acinar cells are dedicated secretory cells, almost all of the labeled amino acids in these cells were incorporated into secretory proteins, facilitating the observation of transported proteins.

Although autoradiography is rarely used today to localize proteins within cells, these early experiments illustrate the two basic requirements for any assay of intercompartmental transport. First, it is necessary to label a cohort of
proteins in an early compartment so that their subsequent transfer to later compartments can be followed over time. Second, it is necessary to have a way to identify the compartment in which a labeled protein resides. Here we describe two modern experimental procedures for observing the intracellular trafficking of a secretory protein in almost any type of cell.

In both procedures, a gene encoding an abundant membrane glycoprotein (G protein) from vesicular stomatitis virus (VSV) is introduced into cultured mammalian cells either by transfection or simply by infecting the cells with the virus. The treated cells, even those that are not specialized for secretion, rapidly synthesize the VSV G protein on the ER as they would normal cellular secretory proteins. Use of a mutant gene encoding a temperature-sensitive VSV G protein allows researchers to turn subsequent transport of this protein on and off. At the restrictive temperature of $40^{\circ} \mathrm{C}$, newly made VSV G protein is misfolded and is therefore retained within the ER by the quality-control mechanisms discussed in Chapter 13, whereas at the permissive temperature of $32^{\circ} \mathrm{C}$, the protein is correctly folded and is transported through the secretory pathway to the cell surface. Importantly, the misfolding of the temperature-sensitive VSV G protein is reversible; thus when cells synthesizing mutant VSV G protein are grown at $40^{\circ} \mathrm{C}$ and then shifted to $32^{\circ} \mathrm{C}$, the misfolded mutant VSV G protein that had accumulated in the ER will refold and be transported normally. This clever use of a temperature-sensitive mutation in effect defines a protein cohort whose subsequent transport can be followed.

In two variations of this basic procedure, transport of VSV G protein is monitored by different techniques. Studies using both of these modern trafficking assays came to the same conclusion as Palade's early experiments: in mammalian cells, vesicle-mediated transport of a protein molecule from its site of synthesis on the rough ER to its arrival at the plasma membrane takes from 30 to 60 minutes.

Microscopy of GFP-Labeled VSV G Protein One approach for observing the transport of VSV G protein employs a hybrid gene in which the viral gene is fused to the gene encoding green fluorescent protein (GFP), a naturally fluorescent protein (see Chapter 4). The hybrid gene is transfected into cultured cells by techniques described in Chapter 6. When cells expressing the temperature-sensitive form of the hybrid protein (VSVG-GFP) are grown at the restrictive temperature, VSVG-GFP accumulates in the ER, which appears as a lacy network of membranes when the cells are observed in a fluorescent microscope. When the cells are subsequently shifted to a permissive temperature, the VSVG-GFP can be seen to move first to the membranes of the Golgi complex, which are densely concentrated at the edge of the nucleus, and then to the cell surface (Figure 14-2a). By observing the distribution of VSVG-GFP at different times after shifting cells to the permissive temperature, researchers have determined how long VSVG-GFP resides in each organelle of the secretory pathway (Figure 14-2b).

![li1604542480e](li1604542480e.jpg)

EXPERIMENTAL FIGURE 14-2 Protein transport through the secretory pathway can be visualized by fluorescence microscopy of cells producing a GFP-tagged membrane protein. Cultured cells were transfected with a hybrid gene encoding the viral membrane glycoprotein VSV G linked to the gene for green fluorescent protein (GFP). A temperature-sensitive mutant version of the viral gene was used so that newly made hybrid protein (VSVG-GFP) was retained in the ER at $40^{\circ} \mathrm{C}$, but was released for transport at $32^{\circ} \mathrm{C}$. (a) Fluorescence micrographs of cells just before and at two times after they were shifted to the lower temperature. Movement of VSVG-GFP from
the ER to the Golgi and finally to the cell surface occurred within 180 minutes. The scale bar is $5 \mu \mathrm{~m}$. (b) Plot of the amount of VSVG-GFP in the endoplasmic reticulum (ER), Golgi, and plasma membrane (PM) at different times after the shift to the permissive temperature. The kinetics of transport from one organelle to another can be reconstructed from computer analysis of these data. The decrease in total fluorescence that occurs at later times probably results from slow inactivation of GFP fluorescence. (Jennifer Lippincott-Schwartz and Koret Hirschberg, Metabolism Branch, National Institute of Child Health and Human Development.)
Detection of Compartment-Specific Oligosaccharide Modifications A second way to follow the transport of secretory proteins takes advantage of modifications to their carbohydrate side chains that occur at different stages of the secretory pathway. To understand this approach, recall that many secretory proteins leaving the ER are carrying one or more copies of the N -linked oligosaccharide $\mathrm{Man}_{5}(\mathrm{GlcNAc})_{2}$, which are synthesized and attached to secretory proteins in the ER (see Figure 13-18). As a protein moves through the Golgi complex, different enzymes localized to the cis-, medial-, and trans-Golgi cisternae catalyze an ordered series of modifications to these core $\mathrm{Man}_{5}(\mathrm{GlcNAc})_{2}$ chains, as discussed in a later section of this chapter. For instance, glycosidases that reside specifically in the cis-Golgi compartment sequentially trim mannose residues off the core oligosaccharide to yield a "trimmed" form, $\mathrm{Man}_{5}(\mathrm{GlcNAc})_{2}$. Scientists can use a specialized carbohydrate-cleaving enzyme known as endoglycosidase D to distinguish glycosylated proteins that remain in the ER from those that have entered the cis-Golgi: trimmed cis-Golgi-specific oligosaccharides are cleaved from proteins by endoglycosidase D , whereas the core (untrimmed) oligosaccharide chains on secretory proteins within the ER are resistant to digestion by this enzyme (Figure 14-3a). Because a deglycosylated protein produced by endoglycosidase D digestion moves faster on an SDS gel than the corresponding glycosylated protein, these proteins can be readily distinguished (Figure 14-3b).

This type of assay can be used to track movement of VSV G protein in virus-infected cells pulse-labeled with radioactive amino acids. Immediately after labeling, all the labeled VSV G protein is still in the ER and, upon extraction, is resistant to digestion by endoglycosidase $D$, but over time,
the fraction of the extracted glycoprotein that is sensitive to digestion increases. This conversion of VSV G protein from an endoglycosidase D-resistant form to an endoglycosidase D-sensitive form corresponds to vesicular transport of the protein from the ER to the cis-Golgi. Note that transport of VSV G protein from the ER to the Golgi takes about 30 minutes, as measured either by the assay based on oligosaccharide processing or by fluorescence microscopy of VSVG-GFP (Figure 14-3c). A variety of assays based on specific carbohydrate modifications that occur in later Golgi compartments have been developed to measure progression of VSV G protein through each stage of the Golgi complex.

## Yeast Mutants Define Major Stages and Many Components in Vesicular Transport

The general organization of the secretory pathway and many of the molecular components required for vesicle trafficking are similar in all eukaryotic cells. Because of this conservation, genetic studies with yeast have been useful in confirming the sequence of steps in the secretory pathway and in identifying many of the proteins that participate in vesicular traffic. For yeast cells, as for all cells, the secretory pathway is essential for transport and delivery of new protein and membrane to the cell surface. Thus genes encoding important components of the secretory pathway are essential for cell growth and can be studied only as conditional mutants, as described in Chapter 8. Although yeasts secrete few proteins into the growth medium, they continuously secrete a number of enzymes that remain localized in the narrow space between the plasma membrane and the cell wall. The best studied of these, invertase, hydrolyzes the disaccharide sucrose to glucose and fructose.

![li1614542480e](li1614542480e.jpg)

EXPERIMENTAL FIGURE 14-3 Transport of a membrane glycoprotein from the ER to the Golgi can be assayed based on sensitivity to cleavage by endoglycosidase D. Cells expressing a temperature-sensitive VSV G protein were labeled with a pulse of radioactive amino acids at the nonpermissive temperature so that the labeled protein was retained in the ER. At periodic times after a return to the permissive temperature of $32^{\circ} \mathrm{C}$, VSV G protein was extracted from cells and digested with endoglycosidase D. (a) As proteins move to the cis-Golgi from the ER, the core oligosaccharide $\mathrm{Man}_{6}(\mathrm{GlcNAc})_{2}$ is trimmed to $\mathrm{Man}_{5}(\mathrm{GlcNAc})_{2}$ by enzymes that reside in the cis-Golgi compartment. Endoglycosidase D cleaves the oligosaccharide chains from proteins processed in the cis-Golgi, but not from proteins in the ER. (b) SDS-polyacrylamide gel electrophoresis of the digestion mixtures resolves the resistant, uncleaved (slower-migrating) and
sensitive, cleaved (faster-migrating) forms of labeled VSV G protein. Initially, as this gel shows, all of the VSV G protein was resistant to digestion, but over time, an increasing fraction was sensitive to digestion, reflecting transport of the protein from the ER to the Golgi and its processing there. In control cells kept at $40^{\circ} \mathrm{C}$, only slow-moving, digestion-resistant VSV G protein was detected after 60 minutes (not shown). (c) A plot of the percentage of VSV G protein that is sensitive to digestion, derived from electrophoretic data, reveals the time course of ER-to-Golgi transport. (Part (b) republished with permission of Elsevier, from Becckers, C. J., et al., "Semi-intact cells permeable to macromolecules: use in reconstitution of protein transport from the endoplasmic reticulum to the Golgi complex," 1987, Cell 50(4):523-34; permission conveyed through the Copyright Clearance Center.)
functions, proteins accumulate in the rough ER, not in the Golgi cisternae. Because proteins accumulate at the earliest blocked step, this finding shows that class B mutations must act at an earlier point in the secretory pathway than class D mutations do. These studies confirmed that as a secreted protein is synthesized and processed, it moves sequentially from the cytosol to the rough ER, to ER-to-Golgi transport vesicles, to Golgi cisternae, to secretory vesicles, and finally is exocytosed.

The three methods outlined in this section have delineated the major steps of the secretory pathway and have contributed to the identification of many of the proteins responsible for vesicle budding and fusion. Each of the individual steps in the secretory pathway is currently being studied in mechanistic detail, and increasingly, biochemical assays and molecular genetic studies are being used to study each of these steps in terms of the function of individual protein molecules.

## Cell-Free Transport Assays Allow Dissection of Individual Steps in Vesicular Transport

In vitro assays for intercompartmental transport are powerful complementary approaches to studies with yeast sec mutants for identifying and analyzing the cellular components responsible for vesicular trafficking. In one application of this approach, cultured mutant cells lacking one of the enzymes that modify N -linked oligosaccharide chains in the

Golgi are infected with vesicular stomatitis virus, and the fate of the VSV G protein is followed. For example, if infected cells lack N -acetylglucosamine transferase I, they produce abundant amounts of VSV G protein but cannot add N -acetylglucosamine residues to the oligosaccharide chains in the medial-Golgi as wild-type cells do (Figure 14-5a). When Golgi membranes isolated from such mutant cells are mixed with Golgi membranes from wild-type, uninfected cells, the addition of N -acetylglucosamine to VSV G protein
![li1624542480e](li1624542480e.jpg)

Fate of Normal secretion proteins

Defective function

Accumulation in the cytosol

Transport into the ER

## Class B

![li1634542480e](li1634542480e.jpg)

Accumulation in ER-to-Golgi transport vesicles

Fusion of transport vesicles with Golgi

Class D
![li1644542480e](li1644542480e.jpg)

Accumulation in Golgi

Acctumulation in secretory vesicles

Transport from
Golgi to secretory vesicles

Transport from secretory vesicles to cell surface
when cells are shifted from the permissive temperature to the higher, nonpermissive one. Analysis of double mutants permitted the sequential order of the steps to be determined. See P. Novick et al., 1981, Cell 25:461, and C. A. Kaiser and R. Schekman, 1990, Cell 61:723.
(a)
![li1654542480e](li1654542480e.jpg)
![li1664542480e](li1664542480e.jpg)
![li1674542480e](li1674542480e.jpg)
![li1684542480e](li1684542480e.jpg)
![li1694542480e](li1694542480e.jpg)
![li1704542480e](li1704542480e.jpg)
![li1714542480e](li1714542480e.jpg)
![li1724542480e](li1724542480e.jpg)

## VSV-infected mutant cells

(no $N$-acetylglucosamine transferase I)

$$
\begin{aligned}
& \text { (a) }=\text { N-Acetylglucosamine } \\
& \text { (b) }=\text { Galactose } \\
& \text { (c) }=\text { N-Acetylneuraminic acid }
\end{aligned}
$$

(b)
Golgi isolated from uninfected wild-type cells
G protein in Golgi from infected mutant cells
![li1734542480e](li1734542480e.jpg)

Incubation
![li1744542480e](li1744542480e.jpg)

## EXPERIMENTAL FIGURE 14-5 A cell-free assay demonstrates

protein transport from one Golgi cisterna to another. (a) A mutant line of cultured fibroblasts is essential in this type of assay. In this example, the cells lack the enzyme N -acetylglucosamine transferase I (see step $\mathbf{B}$ in Figure 14-14 below). In wild-type cells, this enzyme is localized to the medial-Golgi and modifies $N$-linked oligosaccharides by the addition of one $N$-acetylglucosamine. In VSV-infected wildtype cells, the oligosaccharide on the viral G protein is modified to a typical complex oligosaccharide, as shown in the trans-Golgi panel. In infected mutant cells, however, the G protein reaches the cell surface
with a simpler high-mannose oligosaccharide containing only two N -acetylglucosamine and five mannose residues. (b) When Golgi cisternae isolated from infected mutant cells are incubated with Golgi cisternae from normal, uninfected cells, the VSV G protein produced in vitro contains the additional N -acetylglucosamine. This modification is carried out by transferase enzyme that is moved by transport vesicles from the wild-type medial-Golgi cisternae to the mutant cis-Golgi cisternae in the reaction mixture. See W. E. Balch et al., 1984, Cell 39:405 and 525; W. A. Braell et al., 1984, Cell 39:511; and J. E. Rothman and T. Söllner, 1997, Science 276:1212.

is restored (Figure 14-5b). This modification is the consequence of vesicular transport of N -acetylglucosamine transferase I from the wild-type medial-Golgi to the cis-Golgi isolated from virally infected mutant cells. Successful intercompartmental transport in this cell-free system depends on requirements that are typical of a normal physiological process, including a cytosolic extract, a source of chemical energy in the form of ATP and GTP, and incubation at physiological temperatures.

In addition, under appropriate conditions, a uniform population of the transport vesicles that move N -acetylglucosamine transferase I from the medial- to cis-Golgi can be separated from the donor wild-type Golgi membranes by centrifugation. By examining the proteins that are enriched in these vesicles, scientists have been able to identify many of the integral membrane proteins and peripheral vesicle coat proteins that are the structural components of this type of vesicle. Moreover, fractionation of the cytosolic extract required for transport in cell-free reaction mixtures has permitted isolation of the various proteins required for formation of transport vesicles and of proteins required for the targeting and fusion of vesicles with appropriate acceptor membranes. In vitro assays similar in general design to the one shown in Figure 14-5 have been used to study various transport steps in the secretory pathway.

## KEY CONCEPTS OF SECTION 14.1

## Techniques for Studying the Secretory Pathway

- All assays for following the trafficking of proteins through the secretory pathway in live cells require a way to label a cohort of secretory proteins and a way to identify the compartments where the labeled proteins are subsequently located.
- Pulse labeling with radioactive amino acids can specifically label a cohort of newly made proteins in the ER. Alternatively, a temperature-sensitive mutant protein that is retained in the ER due to misfolding at the nonpermissive temperature will be released as a cohort for transport when cells are shifted to the permissive temperature.
- Transport of a fluorescently labeled protein along the secretory pathway can be observed by microscopy (see Figure 14-2). Transport of a radiolabeled protein is commonly tracked by following compartment-specific covalent modifications to the protein.
- Many of the components required for intracellular protein trafficking have been identified in yeast by analysis of temperature-sensitive sec mutants defective for the secretion of proteins at the nonpermissive temperature (see Figure 14-4).
- Cell-free assays for intercompartmental protein transport have allowed the biochemical dissection of individual steps of the secretory pathway. Such in vitro reactions can be used to produce pure transport vesicles and to test the biochemical function of individual transport proteins.


### 14.2 Molecular Mechanisms of Vesicle Budding and Fusion

Small membrane-bounded vesicles that transport proteins from one organelle to another are common elements in the secretory and endocytic pathways (see Figure 14-1). These vesicles bud from the membrane of a particular "parent" (donor) organelle and fuse with the membrane of a particular "target" (destination) organelle. Although each step in the secretory and endocytic pathways employs a different type of vesicle, studies employing genetic and biochemical techniques have revealed that each of the different vesicular transport steps is simply a variation on a common theme. In this section, we explore the basic mechanisms underlying vesicle budding and fusion that all vesicle types have in common, before discussing the details unique to each pathway.

## Assembly of a Protein Coat Drives Vesicle Formation and Selection of Cargo Molecules

The budding of a vesicle from its parent membrane is driven by the polymerization of soluble protein complexes on the membrane to form a proteinaceous vesicle coat (Figure 14-6a). Interactions between the cytosolic portions of integral membrane proteins and the vesicle coat gather the appropriate cargo proteins into the forming vesicle. Thus the coat gives curvature to the membrane to form a vesicle and acts as the filter to determine which proteins are admitted into the vesicle.

Proteins responsible for the eventual fusion of a vesicle with the target membrane, known as v-SNAREs, are incorporated into the vesicle membrane during assembly of the vesicle coat. After the coat is shed from a completed vesicle, v-SNARE proteins embedded in the vesicle membrane become accessible to join with cognate t-SNAREs in the target membrane to which the vesicle is docked. This joining brings the membranes into close apposition, allowing the two bilayers to fuse (Figure 14-6b). Regardless of target organelle, all transport vesicles use v-SNAREs and t-SNAREs to fuse.

Three major types of coated vesicles have been characterized, each with a different type of protein coat and each formed by reversible polymerization of a distinct set of protein subunits (Table 14-1). Each type of vesicle, named for its primary coat proteins, transports cargo proteins from particular parent organelles to particular target organelles:

- COPII vesicles transport proteins from the ER to the Golgi.
- COPI vesicles mainly transport proteins in the retrograde direction between Golgi cisternae and from the cis-Golgi back to the ER.
- Clathrin-coated vesicles transport proteins from the plasma membrane (cell surface) and the trans-Golgi network to late endosomes.

Every vesicle-mediated trafficking step is thought to use some kind of vesicle coat; however, a specific coat protein complex has not been identified for every type of vesicle. For

(a) Coated vesicle budding
![li1754542480e](li1754542480e.jpg)
(b) Uncoated vesicle fusion
![li1764542480e](li1764542480e.jpg)

FIGURE 14-6 Overview of vesicle budding and fusion with a target membrane. (a) Budding is initiated by recruitment of a small GTP-binding protein to a patch of donor membrane. Complexes of coat proteins in the cytosol then bind to the cytosolic domain of membrane cargo proteins, some of which also act as receptors that bind soluble proteins in the lumen, thereby recruiting luminal cargo proteins into the budding vesicle. (b) After being released and shedding its coat, a vesicle fuses with its target membrane in a process that involves interaction of cognate SNARE proteins.
example, vesicles that move proteins from the trans-Golgi to the plasma membrane during either constitutive or regulated secretion exhibit a uniform size and morphology, which suggests that their formation is driven by assembly of a regular coat structure, yet researchers have not identified specific coat proteins surrounding these vesicles.

The general scheme of vesicle budding shown in Figure 14-6a applies to all three known types of coated vesicles. Experiments with isolated or artificial membranes and purified coat proteins have shown that polymerization of the coat proteins on the cytosolic face of the parent membrane is necessary to produce the high curvature of the membrane that is typical of a transport vesicle about 50 nm in diameter. Electron micrographs of in vitro budding reactions often reveal structures that exhibit discrete regions of the parent membrane bearing a dense coat accompanied by the curvature characteristic of a completed vesicle (Figure 14-7). Such structures, usually called vesicle buds, appear to be intermediates that are visible after the coat has begun to polymerize but before the completed vesicle pinches off from the parent membrane. The polymerized coat proteins are thought to form a curved lattice that drives the formation of a vesicle bud by adhering to the cytosolic face of the membrane.

## A Conserved Set of GTPase Switch Proteins Controls the Assembly of Different Vesicle Coats

Using in vitro vesicle-budding reactions among isolated membranes and purified coat proteins, scientists have determined the minimum set of coat components required to form each of the three major types of vesicles. Although most of the coat proteins differ considerably from one type of vesicle to another, the coats of all three vesicles contain a small GTP-binding protein that acts as a regulatory subunit to control coat assembly (see Figure 14-6a). A GTP-binding protein known as ARF protein plays this role in COPI and clathrin-coated vesicles. A different but related GTP-binding protein known as Sar1 protein is present in the coat of COPII vesicles. Both ARF and Sar1 are monomeric proteins with a

TABLE 14-1 Coated Vesicles Involved in Protein Trafficking

| Vesicle Type | Transport Step Mediated | Coat Proteins | Associated GTPase |
| :--: | :--: | :--: | :--: |
| COPII | ER to cis-Golgi | Sec23/Sec24 and Sec13/Sec31 complexes, Sec16 | Sar1 |
| COPI | cis-Golgi to ER Later to earlier Golgi cisternae | Coatomers containing seven different COP subunits | ARF |
| Clathrin and adapter proteins* | trans-Golgi to endosome | Clathrin + AP1 complexes | ARF |
|  | trans-Golgi to endosome | Clathrin + GGA | ARF |
|  | Plasma membrane to endosome | Clathrin + AP2 complexes | ARF |
|  | Golgi to lysosome, melanosome, or platelet vesicles | AP3 complexes | ARF |

[^0]
[^0]:    *Each type of AP complex consists of four different subunits. It is not known whether the coat of AP3 vesicles contains clathrin.

![li1774542480e](li1774542480e.jpg)

EXPERIMENTAL FIGURE 14-7 Vesicle buds can be visualized during in vitro budding reactions. When purified COPII coat components are incubated with isolated ER vesicles or artificial phospholipid vesicles (liposomes), polymerization of the coat proteins on the vesicle surface induces emergence of highly curved buds. In this electron micrograph of an in vitro budding reaction, note the distinct membrane coat, visible as a dark protein layer, present on the vesicle buds. (Republished with permission of Elsevier, from Matsuoka, K. et al., "COPII-coated vesicle formation reconstituted with purified coat proteins and chemically defined liposomes," 1998, Cell 93(2):263-275; permission conveyed through the Copyright Clearance Center, Inc.)
structure generally similar to that of Ras, a key intracellular signal-transducing protein (see Figure 16-23). ARF and Sar1 proteins, like Ras, belong to the GTPase superfamily of switch proteins that cycle between GDP-bound and GTPbound forms (see Figure 3-34 to review the mechanism of GTPase switch proteins).

The cycle of GTP binding and hydrolysis by ARF and Sar1 is thought to control the initiation of coat assembly, as schematically depicted for the assembly of COPII vesicles in Figure 14-8. First, an ER membrane protein known as Sec12 catalyzes the release of GDP from cytosolic Sar1-GDP and the binding of GTP. This guanine nucleotide exchange factor (GEF) apparently receives and integrates multiple as yet unknown signals, probably including the presence in the ER membrane of cargo proteins that are ready to be transported. Binding of GTP causes a conformational change in Sar1 that exposes its amphipathic N -terminus, which then becomes embedded in the phospholipid bilayer and tethers Sar1-GTP to the ER membrane (Figure 14-8, step $\mathbf{D}$ ). The membrane-attached Sar1-GTP drives the polymerization of cytosolic complexes of COPII subunits on the membrane, eventually leading to formation of vesicle buds (step $\mathbf{D}$ ). Once COPII vesicles are released from the donor membrane, the Sar1 GTPase activity hydrolyzes Sar1-GTP in the vesicle membrane to Sar1-GDP with the assistance of one of the coat subunits (step $\mathbf{D}$ ). This hydrolysis triggers disassembly of the COPII coat (step $\mathbf{D}$ ). Thus Sar1 couples a cycle of GTP binding and hydrolysis to the formation and then dissociation of the COPII coat.

ARF protein undergoes a similar cycle of nucleotide exchange and hydrolysis coupled to the assembly of vesicle coats composed either of COPI or of clathrin and other coat

![li1784542480e](li1784542480e.jpg)

FIGURE 14-8 Model for the role of Sar1 in the assembly and disassembly of the COPII coat. Step $\mathbf{D}$ : Interaction of soluble GDPbound Sar1 with the GEF Sec12, an ER integral membrane protein, catalyzes exchange of GTP for GDP on Sar1. The hydrophobic N -terminus of the GTP-bound form of Sar1 extends outward from the protein's surface and anchors Sar1 to the ER membrane. Step $\mathbf{D}$ : Sar1 attached to the membrane serves as a binding site for the Sec23/ Sec24 coat protein complex. Membrane cargo proteins are recruited to the forming vesicle bud by binding of specific short sequences (sorting signals) in their cytosolic regions to sites on the Sec23/Sec24 complex. Some membrane cargo proteins also act as receptors that bind soluble proteins in the lumen. The coat is completed by assembly of a second type of coat complex composed of Sec13 and Sec31 (not shown). Step $\mathbf{D}$ : After the vesicle coat is complete, the Sec23 coat subunit promotes GTP hydrolysis by Sar1. Step $\mathbf{D}$ : Release of Sar1-GDP from the vesicle membrane causes disassembly of the coat. See S. Springer et al., 1999, Cell 97:145.

proteins (AP complexes), discussed later. A covalent protein modification known as a myristate anchor on the N -terminus of the ARF protein weakly tethers ARF-GDP to the Golgi membrane. When GTP is exchanged for the bound GDP by a GEF attached to the Golgi membrane, the resulting conformational change in ARF allows hydrophobic residues in its N-terminal segment to insert into the membrane bilayer. The resulting tight association of ARF-GTP with the membrane serves as the foundation for further coat assembly.

Drawing on the structural similarities of Sar1 and ARF to other small GTPase switch proteins, researchers have constructed genes encoding mutant versions of the two proteins that have predictable effects on vesicular traffic when transfected into cultured cells. For example, in cells expressing mutant versions of Sar1 or ARF that cannot hydrolyze GTP, vesicle coats form and vesicle buds pinch off. However, because the mutant proteins cannot trigger disassembly of the coat, all available coat subunits eventually become permanently assembled into coated vesicles that are unable to fuse with target membranes. Addition of a nonhydrolyzable GTP analog to in vitro vesicle-budding reactions causes a similar blocking of coat disassembly. The vesicles that form in such reactions have coats that never dissociate, allowing their composition and structure to be more readily analyzed. The purified COPI vesicles shown in Figure 14-9 were produced in such a budding reaction.

A second general function of small GTPases in vesicle formation is the pinching off of a completed vesicle from
![li1794542480e](li1794542480e.jpg)

EXPERIMENTAL FIGURE 14-9 Coated vesicles accumulate during in vitro budding reactions in the presence of a nonhydrolyzable analog of GTP. When isolated Golgi membranes are incubated with a cytosolic extract containing COPI coat proteins, vesicles form and bud off from the membranes. Inclusion of a nonhydrolyzable analog of GTP in the budding reaction prevents disassembly of the coat after vesicle release. This micrograph shows COPI vesicles generated in such a reaction and separated from membranes by centrifugation. Coated vesicles prepared in this way can be analyzed to determine their components and properties. [Courtesy of L. Orci (University of Geneva, Switzerland).]
the parent membrane. In vitro budding experiments show that the Sar1 GTPase is required for the pinching off of COPII vesicles and that the ARF GTPase drives the pinching off of COPI vesicles. The mechanism by which these small GTPases convert the energy from GTP hydrolysis to a mechanical force to complete the pinching off of the membrane is not understood. As we will see in Section 14.4, a large polymeric GTPase known as dynamin plays this role in clathrin-coated vesicles.

## Targeting Sequences on Cargo Proteins Make Specific Molecular Contacts with Coat Proteins

In order for transport vesicles to move specific proteins from one compartment to the next, vesicle buds must be able to discriminate among potential membrane and soluble cargo proteins, accepting only those cargo proteins that should advance to the next compartment and excluding those that should remain as residents in the donor compartment. In addition to sculpting the curvature of a donor membrane, the vesicle coat functions in selecting specific proteins as cargo. The primary mechanism by which the vesicle coat selects cargo molecules is by directly binding to specific sequences, or sorting signals, in the cytosolic portion of membrane cargo proteins (see Figure 14-6a). The polymerized coat thus acts as an affinity matrix to cluster selected membrane cargo proteins into forming vesicle buds. Because soluble proteins within the lumen of the parent organelle cannot contact the coat directly, they require a different kind of sorting signal. Soluble luminal proteins often contain what can be thought of as luminal sorting signals, which bind to the luminal domains of certain membrane cargo proteins. The properties of some of the known sorting signals in membrane and soluble proteins are summarized in Table 14-2. We describe the role of these signals in more detail in later sections.

## Rab GTPases Control Docking of Vesicles on Target Membranes

A second set of small GTP-binding proteins, known as Rab proteins, associate with transport vesicles and act as key regulators of vesicle trafficking to and fusion with the appropriate target membrane. Like Sar1 and ARF, Rab proteins belong to the GTPase superfamily of switch proteins. Rab proteins also contain an isoprenoid anchor that allows them to become tethered to the vesicle membrane. Association of an activated Rab protein with a specific vesicle type is generally a two-step process. In the first step, cytosolic Rab-GDP is targeted to the appropriate vesicle, becoming attached there by insertion of its isoprenoid anchor into the vesicle membrane. Often this attachment step is facilitated by a protein that can associate with Rab-GDP along with its isoprenoid anchor, usually known as a guanine nucleotide dissociation inhibitor (GDI). In the second step, a specific GEF located in the vesicle membrane converts membranebound Rab-GDP to Rab-GTP. Once localized and activated

TABLE 14-2 Known Sorting Signals That Direct Proteins to Specific Transport Vesicles

| Signal Sequence* | Signal-Bearing Protein | Proteins with Signal | Vesicles That Incorporate Signal Receptor |
| :--: | :--: | :--: | :--: |
| LUMINAL SORTING SIGNALS |  |  |  |
| Lys-Asp-Glu-Leu (KDEL) | ER-resident soluble proteins | KDEL receptor in cis-Golgi membrane | COPI |
| Mannose 6-phosphate (M6P) | Soluble lysosomal enzymes after processing in cis-Golgi | M6P receptor in trans-Golgi membrane | Clathrin/AP1 |
|  | Secreted lysosomal enzymes | M6P receptor in plasma membrane | Clathrin/AP2 |
| CYTOPLASMIC SORTING SIGNALS |  |  |  |
| Lys-Lys-X-X (KKXX) | ER-resident membrane proteins | COPI $\alpha$ and $\beta$ subunits | COPI |
| Di-arginine (X-Arg-Arg-X) | ER-resident membrane proteins | COPI $\alpha$ and $\beta$ subunits | COPI |
| Di-acidic (e.g., Asp-X-Glu) | Cargo membrane proteins in ER | COPII Sec24 subunit | COPII |
| Asn-Pro-X-Tyr (NPXY) | LDL receptor in plasma membrane | AP2 complex | Clathrin/AP2 |
| Tyr-X-X- $\Phi$ (YXX $\Phi$ ) | Membrane proteins in trans-Golgi | AP1 ( $\mu 1$ subunit) | Clathrin/AP1 |
|  | Plasma membrane proteins | AP2 ( $\mu 2$ subunit) | Clathrin/AP2 |
| Leu-Leu (LL) | Plasma membrane proteins | AP2 complexes | Clathrin/AP2 |

* $\mathrm{X}=$ any amino acid; $\Phi=$ hydrophobic amino acid. Single-letter amino acid abbreviations are in parentheses.
in this way, Rab-GTP is enabled to bind to a variety of different proteins, known as Rab effectors. Binding of Rab-GTP to a Rab effector can ultimately lead to docking of the vesicle on an appropriate target membrane (Figure 14-10a, step $\mathbf{\square}$ ). After vesicle fusion occurs, the GTP bound to the Rab protein is hydrolyzed to GDP, triggering the release of Rab-GDP, which can then undergo another cycle of GDPGTP exchange, binding, and hydrolysis.

A well-understood example of a Rab protein that enables vesicle fusion with the correct target membrane is the Sec4 protein of yeast, which specifically tags secretory vesicles, enabling them to fuse with the plasma membrane. Accordingly, yeast cells expressing mutant Sec4 proteins accumulate secretory vesicles that are unable to fuse with the plasma membrane (class E mutants in Figure 14-4). Sec4-GDP binds to secretory vesicles, where it is activated to Sec4-GTP by its cognate GEF, which is itself located on secretory vesicles. Sec4-GTP, in turn, binds to its effector, a large tethering complex composed of eight subunits, known as the exocyst. Tethering of secretory vesicles to the exocyst by binding of Sec4-GTP ultimately leads to vesicle fusion with the plasma membrane.

In mammalian cells, Rab5 protein is localized to endocytic vesicles, also known as early endosomes. These uncoated vesicles form from clathrin-coated vesicles just after they bud from the plasma membrane during endocytosis (see Figure 14-1, step $\mathbf{\square}$ ). The fusion of early endosomes with one another in cell-free systems requires the presence of Rab5, and addition of Rab5 and GTP to cell-free extracts
accelerates the rate at which these vesicles fuse with one another. A long coiled protein known as EEA1 (early endosome antigen 1), which resides on the membrane of the early endosome, functions as the effector for Rab5. In this case, Rab5-GTP on one endocytic vesicle is thought to bind specifically to EEA1 on the membrane of another endocytic vesicle, setting the stage for fusion of the two vesicles.

Other Rab proteins have motor proteins as effectors. For example, the Rab proteins Ypt31 and Ypt32, like Sec4, associate with secretory vesicles, but when in the activated GTP-bound state, recruit the effector myosin V to secretory vesicles. Myosin V enables secretory vesicles to move along actin filaments to the site of fusion with the plasma membrane.

Every type of transport vesicle appears to be labeled with one or more specific Rab proteins. These Rab proteins, through their specific association with effectors that are membrane tethers and molecular motors, ensure that the vesicles are directed to the correct target membrane address.

## Paired Sets of SNARE Proteins Mediate Fusion of Vesicles with Target Membranes

As noted previously, shortly after a vesicle buds off from the donor membrane, the vesicle coat disassembles to uncover a vesicle-specific membrane protein, a v-SNARE (see Figure 14-6b). Likewise, each type of target membrane in a cell contains t-SNARE membrane proteins, which interact specifically with v-SNAREs. After Rab-mediated docking of

![li1804542480e](li1804542480e.jpg)
a vesicle on its target membrane, the interaction of cognate SNAREs brings the two membranes close enough together that they can fuse.

One of the best-understood examples of SNARE-mediated fusion occurs during exocytosis of secreted proteins (see Figure 14-10a, steps $\mathbf{\square}$ and $\mathbf{\square}$ ). In this case, the $v$-SNARE, known as VAMP (vesicle-associated membrane protein), is incorporated into secretory vesicles as they bud from the trans-Golgi network. The t-SNAREs are syntaxin, an integral membrane protein in the plasma membrane, and SNAP-25, which is attached to the plasma membrane by a hydrophobic lipid anchor in the middle of the protein. The cytosolic region
(b) SNARE complex
![li1814542480e](li1814542480e.jpg)

FIGURE 14-10 Model for docking and fusion of transport vesicles with their target membranes. (a) The proteins shown in this example participate in fusion of secretory vesicles with the plasma membrane, but similar proteins mediate all vesicle-fusion events. Step $\mathbf{\square}$ A Rab protein tethered via a lipid anchor to a secretory vesicle binds to an effector protein complex on the plasma membrane, thereby docking the transport vesicle on the appropriate target membrane. Step $\mathbf{\square}$ A v-SNARE protein (in this case, VAMP) interacts with the cytosolic domains of the cognate t-SNAREs (in this case, syntaxin and SNAP-25). The very stable coiled-coil SNARE complexes that are formed hold the vesicle close to the target membrane. Step $\mathbf{\square}$ Fusion of the two membranes immediately follows formation of SNARE complexes, but precisely how this occurs is not known. Step $\mathbf{\square}$ Following membrane fusion, NSF, in conjunction with $\alpha$-SNAP, binds to the SNARE complexes. The NSF-catalyzed hydrolysis of ATP then drives dissociation of the SNARE complexes, freeing the SNARE proteins for another round of vesicle fusion. Also at this time, Rab-GTP is hydrolyzed to Rab-GDP and dissociates from the Rab effector (not shown). (b) The SNARE complex. Numerous noncovalent interactions between four long $\alpha$ helices, two from SNAP-25 and one each from syntaxin and VAMP, stabilize the coiled-coil structure. See J. E. Rothman and T. Söllner, 1997, Science 276:1212, Y. A. Chen and R. H. Scheller, 2001, Nat. Rev. Mol. Cell Biol. 2:98, and W. Weis and R. Scheller, 1998, Nature 395:328. [Part (b) data from I. Fernandez et al., 1998, Cell 94:841-849, PDB ID 1br0, and R. B. Sutton et al., 1998, Nature 395:347-353, PDB ID 1sfc.]
in each of these three SNARE proteins contains a repeating heptad sequence that allows four $\alpha$ helices-one from VAMP, one from syntaxin, and two from SNAP-25-to coil around one another to form a four-helix bundle (Figure 14-10b). The unusual stability of this bundled SNARE complex is conferred by the arrangement of hydrophobic and charged amino acid residues in the heptad repeats. The hydrophobic amino acids are buried in the central core of the bundle, and amino acids of opposite charge are aligned to form favorable electrostatic interactions between helices. As multiple four-helix bundles form, the embedded transmembrane domains of VAMP and syntaxin pull the vesicle

and target membranes together into very close apposition. The energetically favorable formation of four-helix bundles can overcome the electrostatic repulsion of the generally negatively charged phospholipid head groups in the vesicle and target membranes, allowing the hydrophobic interiors of the two membranes to come into contact, creating an opening between the two membranes, and ultimately causing the vesicle membrane to fuse with the target membrane.

In vitro experiments have shown that when liposomes containing purified VAMP are incubated with other liposomes containing syntaxin and SNAP-25, the two classes of membranes fuse, albeit slowly. This finding is strong evidence that the close apposition of membranes resulting from formation of SNARE complexes is sufficient to bring about membrane fusion. Fusion of a vesicle and target membrane occurs more rapidly and efficiently in the cell than it does in liposome experiments in which fusion is catalyzed only by SNARE proteins. The likely explanation for this difference is that in the cell, other proteins, such as Rab proteins and their effectors, are involved in targeting vesicles to the correct membrane.

Yeast cells, like all eukaryotic cells, express more than 20 different related v-SNARE and t-SNARE proteins. Analyses of yeast mutants defective in each of the SNARE genes have identified specific membrane-fusion events in which each SNARE protein participates. For all fusion events that have been examined, the SNAREs form four-helix bundled complexes similar to the VAMP/syntaxin/SNAP-25 complexes that mediate fusion of secretory vesicles with the plasma membrane. However, in other fusion events (e.g., fusion of COPII vesicles with the cis-Golgi network), each participating SNARE protein contributes only one $\alpha$ helix to the bundle (unlike SNAP-25, which contributes two helices); in these cases, the SNARE complexes comprise one v-SNARE and three t-SNARE molecules.

Using the in vitro liposome fusion assay, researchers have tested the ability of various combinations of individual v-SNARE and t-SNARE proteins to mediate fusion of donor and target membranes. Of the very large number of different combinations tested, only a small number could efficiently mediate membrane fusion. To a remarkable degree, the functional combinations of v-SNAREs and t-SNAREs revealed in these in vitro experiments correspond to the actual SNARE protein interactions that mediate known membrane-fusion events in the yeast cell. Thus, together with the specificity of interaction between Rab and Rab effector proteins, the specificity of the interaction between SNARE proteins can account for most, if not all, of the specificity of fusion between a particular vesicle type and its target membrane.

## Dissociation of SNARE Complexes After Membrane Fusion Is Driven by ATP Hydrolysis

After a vesicle and its target membrane have fused, the SNARE complexes must dissociate to make the individual SNARE proteins available for additional fusion events. Because of the stability of SNARE complexes, which are
held together by numerous noncovalent intermolecular interactions, their dissociation depends on additional proteins and the input of energy.

The first clue that dissociation of SNARE complexes required the assistance of other proteins came from in vitro transport reactions depleted of certain cytosolic proteins. The observed accumulation of vesicles in these reactions indicated that vesicles could form under these conditions, but were unable to fuse with a target membrane. Eventually two proteins, designated NSF and $\alpha-S N A P$, were found to be required for ongoing vesicle fusion in the in vitro transport reaction. The function of NSF in vivo can be blocked selectively by N -ethylmaleimide (NEM), a chemical that reacts with an essential -SH group on NSF (hence the name, NEM-sensitive factor).

Yeast mutants have also contributed to our understanding of SNARE function. Among the class C yeast sec mutants are strains that lack functional Sec18 or Sec17, the yeast counterparts of mammalian NSF and $\alpha$-SNAP, respectively. When these class C mutants are kept at the nonpermissive temperature, they accumulate ER-to-Golgi transport vesicles; when the cells are shifted to the lower, permissive temperature, the accumulated vesicles are able to fuse with the cis-Golgi.

Subsequent to the initial biochemical and genetic studies that identified NSF and $\alpha$-SNAP, more sophisticated in vitro transport assays were developed. Using these newer assays, researchers have shown that NSF and $\alpha$-SNAP proteins are not necessary for actual membrane fusion, but rather are required for regeneration of free SNARE proteins. NSF, a hexamer of identical subunits, associates with a SNARE complex with the aid of $\alpha$-SNAP (soluble NSF attachment protein). The bound NSF then hydrolyzes ATP, releasing sufficient energy to dissociate the SNARE complex (see Figure 14-10a, step $\mathbf{\square}$ ). Evidently, the defects in vesicle fusion observed in the earlier in vitro fusion assays and in the yeast mutants after a loss of Sec17 or Sec18 were a consequence of free SNARE proteins rapidly becoming sequestered in undissociated SNARE complexes and thus being unavailable to mediate membrane fusion.

## KEY CONCEPTS OF SECTION 14.2

## Molecular Mechanisms of Vesicle Budding and Fusion

- The three well-characterized types of transport vesiclesCOPI, COPII, and clathrin-coated vesicles-are distinguished by the proteins that form their coats and the transport routes they mediate (see Table 14-1).
- All types of coated vesicles are formed by polymerization of cytosolic coat proteins on a parent (donor) membrane to form vesicle buds that eventually pinch off from the membrane to release a complete vesicle. Shortly after vesicle release, the coat is shed, exposing proteins required for fusion with the target membrane (see Figure 14-6).

- Small GTP-binding proteins (ARF or Sar1) belonging to the GTPase superfamily control polymerization of coat proteins, the initial step in vesicle budding (see Figure 14-8). After vesicles are released from the donor membrane, hydrolysis of GTP bound to ARF or Sar1 triggers disassembly of the vesicle coats.
- Specific sorting signals in membrane and luminal proteins in donor organelles interact with coat proteins during vesicle budding, thereby recruiting cargo proteins to vesicles (see Table 14-2).
- A second set of GTP-binding proteins, the Rab proteins, label specific vesicle types and enable their targeting to the appropriate membrane. Activated Rab-GTP in a vesicle can bind to a specific type of effector protein. One type of effector is a filamentous tethering protein or large protein complex that enables tethering of the vesicle to the target membrane. Another type of effector is a motor protein that enables vesicles to move along cytoskeletal filaments to their correct destination.
- Each v-SNARE in a vesicular membrane specifically binds to a complex of cognate t-SNARE proteins in the target membrane, inducing fusion of the two membranes. After fusion is completed, the SNARE complex is disassembled in an ATP-dependent reaction mediated by other cytosolic proteins (see Figure 14-10).


### 14.3 Early Stages of the Secretory Pathway

In this section, we take a closer look at vesicular traffic between the ER and the Golgi and at some of the evidence supporting the general mechanisms discussed in the previous section. Recall that anterograde transport from the ER to the Golgi, the first vesicle trafficking step in the secretory pathway, is mediated by COPII vesicles. These vesicles contain newly synthesized proteins destined for the Golgi, cell surface, or lysosomes as well as vesicle components such as vSNAREs that are required to target vesicles to the cis-Golgi membrane. Proper sorting of proteins between the ER and Golgi also requires retrograde transport from the cis-Golgi to the ER, which is mediated by COPI vesicles (Figure 14-11). This retrograde vesicle transport serves to retrieve v-SNARE proteins and components of the membrane itself to provide the necessary material for additional rounds of vesicle budding from the ER. COPI-mediated retrograde transport also retrieves missorted ER-resident proteins from the cis-Golgi to correct sorting mistakes.

We also discuss in this section the process by which proteins that have been correctly delivered to the Golgi advance through successive compartments of the Golgi, from the cis- to the trans-Golgi network. This process of cisternal maturation involves budding and fusion of retrograde rather than anterograde transport vesicles.
![li1824542480e](li1824542480e.jpg)

FIGURE 14-11 Vesicle-mediated protein trafficking between the ER and cis-Golgi. Steps - Forward (anterograde) transport is mediated by COPII vesicles, which are formed by polymerization of soluble COPII coat protein complexes (green) on the ER membrane. v-SNAREs (orange) and other cargo proteins (blue) in the ER membrane are incorporated into the vesicle by interacting with coat proteins. Soluble cargo proteins (magenta) are recruited by binding to appropriate receptors in the membrane of budding vesicles. Dissociation of the coat recycles free coat complexes and exposes v-SNARE proteins on the vesicle surface. After the uncoated vesicle becomes tethered to the cis-Golgi membrane in a Rab-mediated process, pairing between the exposed v-SNAREs and cognate t-SNAREs in the Golgi membrane allows membrane fusion, releasing the contents of the vesicle into the cis-Golgi compartment (see Figure 14-10). Steps ■- Reverse (retrograde) transport, mediated by vesicles coated with COPI proteins (purple), recycles the membrane bilayer and certain proteins, such as v-SNAREs and missorted ER-resident proteins (not shown), from the cis-Golgi to the ER. All SNARE proteins are shown in orange, although v-SNAREs and t-SNAREs are distinct proteins.

## COPII Vesicles Mediate Transport from the ER to the Golgi

COPII vesicles were first recognized when cell-free extracts of yeast rough ER membranes were incubated with cytosol and a nonhydrolyzable analog of GTP. The vesicles that formed from the ER membranes had a distinct coat similar to that on COPI vesicles, but composed of different proteins, designated COPII proteins. Yeast cells with mutations

in the genes for COPII proteins are class B sec mutants and accumulate proteins in the rough ER (see Figure 14-4). Analysis of such mutants has revealed a set of proteins required for formation of COPII vesicles, including the proteins that make up the COPII vesicle coat.

As described previously, formation of COPII vesicles is triggered when Sec12, a GEF in the ER membrane, catalyzes the exchange of bound GDP for GTP on cytosolic Sar1. This exchange induces binding of Sar1 to the ER membrane, followed by binding of a complex of Sec23 and Sec24 proteins (see Figure 14-8). The resulting ternary complex formed between Sar1-GTP, Sec23, and Sec24 is shown in Figure 14-12. This core coat protein complex then provides binding sites for the recruitment of a second complex of Sec13 and Sec31 proteins to complete the coat structure. Since pure Sec13 and Sec31 proteins can spontaneously assemble into cagelike lattices, it is thought that Sec13 and Sec31 form the structural scaffold for COPII vesicles. Finally, a large fibrous protein called Sec16, which is bound to the cytosolic surface of the ER, interacts with Sar1-GTP and the Sec13/31 and Sec23/24 complexes to organize the other coat proteins, increasing the efficiency of coat polymerization. Like Sec 13/31, clathrin has the ability to self-assemble into a coat-like structure, as will be discussed in Section 14.4.
![li1834542480e](li1834542480e.jpg)

FIGURE 14-12 Three-dimensional structure of the ternary complex comprising the COPII coat proteins Sec23 and Sec24 and Sar1-GTP. Early in the formation of the COPII coat, Sec23 (orange)/ Sec24 (green) complexes are recruited to the ER membrane by Sar1 (red) in its GTP-bound state. In order to form a stable ternary complex in solution for structural studies, the nonhydrolyzable GTP analog GppNHp is used. A cargo protein in the ER membrane can be recruited to COPII vesicles by the interaction of a tripeptide di-acidic sorting signal (purple) in the cargo protein's cytosolic domain with Sec24. The likely positions of the COPII vesicle membrane and the transmembrane segment of the cargo protein are indicated. The N-terminal segment of Sar1 that tethers it to the membrane is not shown. [Republished with permission of Nature, from Bi, X. et al., "Structure of the Sec23/24-Sar1 pre-budding complex of the COPII vesicle coat," 2002, Nature 419(6904): 271-7; permission conveyed through the Copyright Clearance Center, Inc.]

Certain integral ER membrane proteins are specifically recruited into COPII vesicles for transport to the Golgi. The cytosolic segments of many of these proteins contain a diacidic sorting signal (the key residues in this sequence are Asp-X-Glu, or DXE in the one-letter code) (see Table 14-2). This sorting signal, which binds to the Sec24 subunit of the COPII coat, is essential for the selective export of certain membrane proteins from the ER (see Figure 14-12). Biochemical and genetic studies have identified additional signals that help direct membrane cargo proteins into COPII vesicles. All of the known sorting signals bind to one or another site on the Sec24 subunit of COPII. Ongoing studies seek to determine how soluble cargo proteins are selectively loaded into COPII vesicles. For example, TANGO1 is an ER membrane protein that acts as a cargo receptor for collagen by simultaneously binding to collagen in the lumen and to the Sec24 subunit of the coat.

2
The inherited disease cystic fibrosis is characterized by an imbalance in chloride and sodium ion transport in the epithelial cells of the lungs, leading to fluid buildup and difficulty breathing. Cystic fibrosis is caused by mutations in a protein known as CFTR, which is synthesized as an integral membrane protein in the ER and is transported to the Golgi before being transported to the plasma membranes of epithelial cells, where it functions as a chloride channel. Researchers have recently shown that the CFTR protein contains a di-acidic sorting signal that binds to the Sec24 subunit of the COPII vesicle coat and is necessary for transport of the CFTR protein out of the ER. The most common CFTR mutation is a deletion of a phenylalanine at position 508 in the protein sequence (known as $\Delta \mathrm{F} 508$ ). This mutation prevents normal transport of CFTR to the plasma membrane by blocking its packaging into COPII vesicles budding from the ER. Although the $\Delta \mathrm{F} 508$ mutation is not in the vicinity of the di-acidic sorting signal, this mutation may change the conformation of the cytosolic portion of CFTR so that the signal is unable to bind to Sec24. Interestingly, a folded CFTR with this mutation will still function properly as a normal chloride channel. However, it never reaches the membrane; the disease state is therefore caused by the absence of the channel, rather than by a defective channel.

The experiments described previously in which the transit of VSVG-GFP in cultured mammalian cells was followed by fluorescence microscopy (see Figure 14-2) provided insight into the intermediates in ER-to-Golgi transport. In some cells, small fluorescent vesicles containing VSVG-GFP could be seen to form from the ER, move less than $1 \mu \mathrm{~m}$, and then fuse directly with the cis-Golgi. In other cells, in which the ER was located several micrometers from the Golgi complex, several ER-derived vesicles were seen to fuse with one another shortly after their formation, forming what is termed the ER-to-Golgi intermediate compartment or the cis-Golgi network. These larger structures were then transported along microtubules to the cis-Golgi, much in the way vesicles in neurons are transported from the cell body, where

they are formed, down the long axon to the axon terminus (see Chapter 18). Microtubules function like "railroad tracks," enabling these large aggregates of transport vesicles to move long distances to their cis-Golgi destination. At the time the ER-to-Golgi intermediate compartment is formed, some COPI vesicles bud off from it, recycling some proteins back to the ER.

## COPI Vesicles Mediate Retrograde Transport Within the Golgi and from the Golgi to the ER

COPI vesicles were first discovered when isolated Golgi fractions were incubated in a solution containing cytosol and a nonhydrolyzable analog of GTP (see Figure 14-9). Subsequent analysis of these vesicles showed that the coat is formed from large cytosolic complexes, called coatomers, composed of seven polypeptide subunits. Yeast cells containing temperature-sensitive mutations in COPI proteins accumulate proteins in the rough ER at the nonpermissive temperature and thus are categorized as class B sec mutants (see Figure 14-4). Although discovery of these mutants initially suggested that COPI vesicles mediate ER-to-Golgi transport, subsequent experiments showed that their main function is retrograde transport, both between Golgi cisternae and from the cis-Golgi to the rough ER (see Figure 14-11, right). Because COPI mutants cannot recycle key membrane proteins back to the rough ER, the ER gradually becomes depleted of ER proteins, such as v-SNAREs, that are necessary for COPII vesicle function. Eventually, vesicle formation from the rough ER grinds to a halt; secretory proteins continue to be synthesized but accumulate in the ER-the defining characteristic of class B sec mutants. The general ability of sec mutations involved in either COPI or COPII vesicle function to eventually block both anterograde and retrograde transport illustrates the fundamental interdependence of these two transport processes.

As discussed in Chapter 13, the ER contains several soluble proteins dedicated to the folding and modification of newly synthesized secretory proteins. They include the chaperone BiP and the enzyme protein disulfide isomerase, which are necessary for the ER to carry out its functions. Although such ER-resident luminal proteins are not specifically selected by COPII vesicles, their sheer abundance causes them to be continuously loaded passively into vesicles destined for the cis-Golgi. The transport of these soluble proteins back to the ER, mediated by COPI vesicles, prevents their eventual depletion.

Most soluble ER-resident proteins carry a Lys-Asp-Glu-Leu (KDEL in the one-letter code) sequence at their C-terminus (see Table 14-2). Several experiments have demonstrated that this KDEL sorting signal is both necessary and sufficient to cause a protein bearing this sequence to be located in the ER. For instance, when a mutant protein disulfide isomerase lacking these four residues is synthesized in cultured fibroblasts, the protein is secreted. Moreover, if a protein that is normally secreted is altered so that it contains the KDEL sorting signal at its C-terminus, the protein is located in the

ER. The KDEL sorting signal is recognized and bound by the KDEL receptor, a transmembrane protein found primarily on small transport vesicles shuttling between the ER and the cis-Golgi and on the cis-Golgi membrane. In addition, soluble ER-resident proteins that carry the KDEL signal have oligosaccharide chains bearing modifications that are catalyzed by enzymes found only in the cis-Golgi or cis-Golgi network; thus at some time these proteins must have left the ER and been transported at least as far as the cis-Golgi network. These findings indicate that the KDEL receptor acts mainly to retrieve soluble proteins containing the KDEL sorting signal that have escaped to the cis-Golgi network and return them to the ER (Figure 14-13). The KDEL receptor binds
![li1844542480e](li1844542480e.jpg)

FIGURE 14-13 Role of the KDEL receptor in retrieval of ER-resident luminal proteins from the Golgi. ER luminal proteins, especially those present at high concentrations, can be passively incorporated into COPII vesicles and transported to the Golgi (steps $\mathbf{\square}$ and $\mathbf{\square}$ ). Many such proteins bear a C-terminal KDEL (Lys-Asp-Glu-Leu) sequence (red) that allows them to be retrieved. The KDEL receptor, located mainly in the cis-Golgi network and in both COPII and COPI vesicles, binds proteins bearing the KDEL sorting signal and returns them to the ER (steps $\mathbf{\square}$ and $\mathbf{\square}$ ). This retrieval system prevents depletion of ER luminal proteins such as those needed for proper folding of newly made secretory proteins. The binding affinity of the KDEL receptor is very sensitive to pH . The small difference between the pH of the ER and that of the Golgi favors binding of KDEL-bearing proteins to the receptor in Golgi-derived vesicles and their release in the ER. See J. Semenza et al., 1990, Cell 61:1349.

more tightly to its ligand at low pH , and it is thought that the receptor is able to bind KDEL peptides in the cis-Golgi and release these peptides in the ER because the pH of the Golgi is slightly lower than that of the ER.

The KDEL receptor and other membrane proteins that are transported back to the ER from the Golgi contain a Lys-Lys-X-X sequence at the very end of their C-terminal segment, which faces the cytosol (see Table 14-2). This KKXX sorting signal, which binds to a complex of the COPI $\alpha$ and $\beta$ subunits (two of the seven polypeptide subunits in the COPI coatomer), is both necessary and sufficient to incorporate membrane proteins into COPI vesicles for retrograde transport to the ER. Temperature-sensitive yeast mutants lacking $\mathrm{COPI} \alpha$ or $\mathrm{COPI} \beta$ are not only unable to bind the KKXX signal, but are also unable to transport proteins bearing this signal back to the ER, indicating that COPI vesicles mediate retrograde Golgi-to-ER transport.

A second sorting signal that targets proteins to COPI vesicles and thus enables recycling from the Golgi to the ER is a di-arginine sequence. Unlike the KKXX sorting signal, which must be located at the cytoplasmically oriented C-terminus of a protein, the di-arginine sorting signal can reside in any segment of a membrane protein that is on the cytoplasmic face of the membrane.

The partitioning of proteins between the ER and Golgi complex is a highly dynamic process depending on both COPII (anterograde) and COPI (retrograde) vesicles, with each type of vesicle responsible for recycling the components necessary for the function of the other type of vesicle. The organization of this partitioning process raises an interesting puzzle: how do vesicles preferentially use the v-SNAREs that specify fusion with the correct target membrane instead of the v-SNAREs that are being recycled and specify fusion with the donor membrane?

This basic question concerning correct membrane partitioning has recently been answered for COPII vesicles. After these vesicles form, the COPII coat proteins remain assembled long enough for the Sec23/Sec24 complex to interact with a specific tethering factor attached to the cisGolgi membrane. Vesicle uncoating to expose the v-SNAREs is completed only after the COPII vesicle is already closely associated with the cis-Golgi membrane and the COPII v-SNAREs are in position to form complexes with their cognate t-SNAREs. Although COPII vesicles also carry COPI-specific v-SNARE proteins, which are being recycled back to the cis-Golgi, these COPI v-SNARE proteins never have the opportunity to form SNARE complexes with cognate ER-localized t-SNARE proteins.

## Anterograde Transport Through the Golgi Occurs by Cisternal Maturation

The Golgi complex is organized into three compartments, often arranged in a stacked set of flattened sacs, called cisternae. The compartments of the Golgi differ from one another according to the enzymes they contain. Many of the enzymes are glycosidases and glycosyltransferases
that are involved in modifying the N -linked or O -linked carbohydrates attached to secretory proteins as they transit the Golgi complex. On the whole, the Golgi complex operates much like an assembly line, with proteins moving in sequence through the compartments, in which the modified carbohydrate chains in one compartment serve as the substrates for the modifying enzymes of the next compartment (Figure 14-14 shows a representative sequence of modification steps).
![li1854542480e](li1854542480e.jpg)

FIGURE 14-14 Processing of $N$-linked oligosaccharide chains on glycoproteins within cis-, medial-, and trans-Golgi cisternae in vertebrate cells. The enzymes catalyzing each step are localized to the indicated compartments. After removal of three mannose residues in the cis-Golgi (step $\mathbf{\square}$ ), the protein moves by cisternal maturation to the medial-Golgi. Here three $N$-acetylglucosamine (GlcNAc) residues are added (steps $\mathbf{\square}$ and $\mathbf{\square}$ ), two more mannose residues are removed (step $\mathbf{\square}$ ), and a single fucose is added (step $\mathbf{\square}$ ). Processing is completed in the trans-Golgi by addition of three galactose residues (step $\mathbf{\square}$ ) and finally by linkage of an N -acetylneuraminic acid residue to each of the galactose residues (step $\mathbf{\square}$ ). Specific transferase enzymes add sugars to the oligosaccharide, one at a time, from sugar nucleotide precursors imported from the cytosol. This pathway represents the Golgi processing events for a typical mammalian glycoprotein. Variations in the structure of N-linked oligosaccharides can result from differences in processing steps in the Golgi. See R. Kornfeld and S. Kornfeld, 1985, Annu. Rev. Biochem. 45:631.

For many years, it was thought that the Golgi complex was an essentially static set of compartments with small transport vesicles carrying secretory proteins forward, from the cis- to the medial-Golgi and from the medial- to the trans-Golgi. Indeed, electron microscopy reveals many small vesicles associated with the Golgi complex that appear to move proteins from one Golgi compartment to another (Figure 14-15). However, these vesicles are now known to mediate retrograde transport, retrieving ER or Golgi enzymes from a later compartment and transporting them to an earlier compartment in the secretory pathway. Thus the Golgi appears to have a highly dynamic organization, continually forming transport vesicles, though only in the retrograde direction. To see the effect of this retrograde transport on the organization of the Golgi, consider the net effect on the medial-Golgi compartment as enzymes from the trans-Golgi move to the medial-Golgi while enzymes from the medial-Golgi are transported to the cis-Golgi. As this process continues, the medial-Golgi acquires enzymes from the trans-Golgi while losing medial-Golgi enzymes to the cis-Golgi and thus progressively becomes a new transGolgi compartment. In this way, secretory cargo proteins acquire carbohydrate modifications in the proper sequential order without being moved from one cisterna to another via anterograde vesicle transport.

The first evidence that the forward transport of cargo proteins from the cis- to the trans-Golgi occurs by this progressive mechanism of cisternal maturation came from careful microscopic analysis of the synthesis of algal scales. These cell-wall glycoproteins are assembled in the cis-Golgi
into large complexes visible in the electron microscope. Like other secretory proteins, newly made scales move from the cis- to the trans-Golgi, but they can be 20 times larger than the usual transport vesicles that bud from Golgi cisternae. Similarly, in the synthesis of collagen by fibroblasts, large aggregates of the procollagen precursor often form in the lumen of the cis-Golgi (see Figure 20-25). The procollagen aggregates are too large to be incorporated into small transport vesicles, and investigators could never find such aggregates in transport vesicles. These observations show that the forward movement of these, and perhaps all, secretory proteins from one Golgi compartment to another does not occur via small vesicles.

A particularly elegant demonstration of cisternal maturation in yeast takes advantage of different-colored fluorescent labels to image two different Golgi proteins simultaneously. Figure 14-16 shows how a cis-Golgi resident protein labeled with a green fluorescent protein and a trans-Golgi resident protein labeled with a red fluorescent protein behave in the same yeast cell. At any given moment, individual Golgi cisternae appear to have a distinct compartmental identity, in the sense that they contain either the cis-Golgi protein or the trans-Golgi protein, but only rarely contain both proteins. However, over time, an individual cisterna labeled with the cis-Golgi protein can be seen to progressively lose this protein and acquire the trans-Golgi protein. This behavior is exactly that predicted by the cisternal maturation model, in which the composition of an individual cisterna changes as Golgi resident proteins move from later to earlier Golgi compartments.
![li1864542480e](li1864542480e.jpg)

EXPERIMENTAL FIGURE 14-15 Electron micrograph of the Golgi complex in an pancreatic acinar cell reveals secretory and retrograde transport vesicles. A large secretory vesicle can be seen forming from the trans-Golgi network. Elements of the rough ER are on the bottom and left in this micrograph. Adjacent to the rough ER are transitional elements from which smooth protrusions appear to be budding. These buds form the small vesicles that transport secretory proteins from the rough ER to the Golgi complex. Interspersed among the Golgi cisternae are other small vesicles now known to function in retrograde, not anterograde, transport. [George Palade.]

![li1874542480e](li1874542480e.jpg)

EXPERIMENTAL FIGURE 14-16 Fluorescence-tagged fusion proteins demonstrate Golgi cisternal maturation in a live yeast cell. Yeast cells expressing the early Golgi protein Vrg4 fused to GFP (green fluorescence) and the late Golgi protein Sec7 fused to DsRed (red fluorescence) were imaged by time-lapse microscopy. The top series of images, taken approximately 1 minute apart, shows a collection of Golgi cisternae, which at any one time are labeled with either Vrg4 or Sec7. The bottom series of images show just one Golgi cisterna, isolated by digital processing of the image. First only Vrg4-GFP is

Although most protein traffic appears to move through the Golgi complex by a cisternal maturation mechanism, there is evidence that at least some of the COPI transport vesicles that bud from Golgi membranes contain cargo proteins (rather than Golgi enzymes) and move in an anterograde (rather than retrograde) direction.

## KEY CONCEPTS OF SECTION 14.3

## Early Stages of the Secretory Pathway

- COPII vesicles transport proteins from the rough ER to the cis-Golgi; COPI vesicles transport proteins in the reverse direction (see Figure 14-11).
- COPII coats comprise three components: the small GTPbinding protein Sar1, a Sec23/Sec24 complex, and a Sec13/ Sec31 complex.
- Components of the COPII coat bind to membrane cargo proteins containing a di-acidic or other sorting signal in their cytosolic regions (see Figure 14-12). Soluble cargo proteins are probably targeted to COPII vesicles by binding to a membrane protein receptor.
- Many soluble ER-resident proteins contain a KDEL sorting signal. Binding of this retrieval sequence to a specific receptor protein in the cis-Golgi membrane recruits missorted ER proteins into retrograde COPI vesicles (see Figure 14-13).
- Membrane proteins needed to form COPII vesicles can be retrieved from the cis-Golgi by COPI vesicles. One of the sorting signals that directs membrane proteins into COPI vesicles is a KKXX sequence, which binds to subunits of the
located in the isolated cisterna, and later only Sec7-DsRed is located in the isolated cisterna, following a brief period in which both proteins are co-localized in this compartment. This experiment is a direct demonstration of the cisternal maturation hypothesis, showing that the composition of individual cisternae follows a process of maturation characterized by loss of early Golgi proteins and gain of late Golgi proteins. [Republished with permission of Nature, from Losev, E. et al., "Golgi maturation visualized in living yeast," 2006, Nature 441:1002-1006; permission conveyed through the Copyright Clearance Center, Inc.]

COPI coat. A distinct di-arginine sorting signal operates by a similar mechanism.

- COPI vesicles also carry Golgi-resident proteins from later to earlier compartments in the Golgi complex.
- Soluble and membrane proteins advance through the Golgi complex by cisternal maturation, a process of anterograde transport that depends on resident Golgi enzymes moving by COPI vesicular transport in a retrograde direction.


### 14.4 Later Stages of the Secretory Pathway

As cargo proteins move from the cis- to the trans-Golgi by cisternal maturation, modifications to their oligosaccharide chains are carried out by Golgi-resident enzymes. The retrograde trafficking of COPI vesicles from later to earlier Golgi compartments maintains sufficient levels of these carbohydrate-modifying enzymes in the appropriate compartments. Eventually, properly processed cargo proteins reach the trans-Golgi network, the most distal Golgi compartment. Here they are sorted into a number of different kinds of vesicles for delivery to their final destination. Each of the target destinations, such as the plasma membrane, endosomes, and lysosomes, has a unique composition of lipids and membrane proteins, and it is primarily the sorting in the trans-Golgi network that gives each of these organelles its unique identity. In this section, we discuss the different kinds of vesicles that bud from the transGolgi network, the mechanisms that segregate cargo proteins among them, and key processing events that occur late in the secretory pathway. The various types of vesicles that bud from the trans-Golgi are summarized in Figure 14-17.

![li1884542480e](li1884542480e.jpg)

FIGURE 14-17 Vesicle-mediated protein trafficking from the trans-Golgi network. COPI (purple) vesicles mediate retrograde transport within the Golgi ( $\mathbf{\square}$ ). Proteins that function in the lumen or in the membrane of the lysosome are first transported from the trans-Golgi network via clathrin-coated (red) vesicles ( $\mathbf{\square}$ ); after uncoating, these vesicles fuse with late endosomes, which deliver their contents to the lysosome. The coats on most clathrin-coated vesicles contains additional proteins (AP complexes) not shown here. Some vesicles from the trans-Golgi carrying cargo destined for the lysosome fuse with the lysosome directly ( $\mathbf{\square}$ ), bypassing the late endosome. These vesicles are coated with a type of AP complex (blue); it is unknown whether these vesicles also contain clathrin. The coat proteins surrounding constitutive ( $\mathbf{\square}$ ) and regulated ( $\mathbf{\square}$ ) secretory vesicles have not yet been characterized; these vesicles carry secreted proteins and plasmamembrane proteins from the trans-Golgi network to the cell surface.

## Vesicles Coated with Clathrin and Adapter Proteins Mediate Transport from the trans-Golgi

The best-characterized vesicles that bud from the trans-Golgi network have a two-layered coat: an outer layer composed of the fibrous protein clathrin and an inner layer composed of adapter protein (AP) complexes. Purified clathrin molecules, which have a three-limbed shape, are called triskelions, from the Greek for "three-legged" (Figure 14-18a). Each limb contains one clathrin heavy chain (180,000 MW) and one clathrin light chain ( $\sim 35,000-40,000 \mathrm{MW}$ ). Triskelions polymerize to form a polygonal lattice with an intrinsic curvature (Figure 14-18b). When clathrin polymerizes on a donor membrane, it does so in association with AP complexes, which fill the space between the clathrin lattice and the membrane. Each AP complex (340,000 MW) contains one copy each of four different adapter subunit proteins. A specific association between the globular domain at the end of each clathrin heavy chain in a triskelion and one subunit of the AP complex both promotes the co-assembly of clathrin triskelions with AP complexes and adds to the stability of the completed vesicle coat.

By binding to the cytosolic face of membrane proteins, adapter proteins determine which cargo proteins are specifically included in (or excluded from) a budding transport vesicle. Three different AP complexes are known (AP1, AP2, AP3), each with four subunits of different, though related, proteins. Recently, a second general type of adapter protein, known as GGA, has been shown to contain in a single

70,000-MW polypeptide both clathrin- and cargo-binding elements similar to those found in the much larger heterotetrameric AP complexes. Vesicles containing each type of adapter complex (AP or GGA) have been found to mediate specific transport steps (see Table 14-1). All vesicles whose coats contain one of these complexes use ARF to initiate coat assembly on the donor membrane. As discussed previously, ARF also initiates assembly of COPI coats. The additional membrane features or protein factors that determine which type of coat will assemble after ARF attachment are not well understood at this time.

Vesicles that bud from the trans-Golgi network en route to the lysosome by way of the late endosome (see Figure 14-17, step $\mathbf{\square}$ ) have clathrin coats associated with either AP1 or GGA. Both AP1 and GGA bind to the cytosolic domain of cargo proteins in the donor membrane. Membrane proteins containing a Tyr-X-X- $\Phi$ sequence, where X is any amino acid and $\Phi$ is a bulky hydrophobic amino acid, are recruited into clathrin/AP1-coated vesicles budding from the trans-Golgi network. This YXXФ sorting signal interacts with one of the AP1 subunits in the vesicle coat. As we discuss in the next section, vesicles with clathrin/AP2 coats, which bud from the plasma membrane during endocytosis, can also recognize the YXXФ sorting signal. Vesicles coated with GGA proteins and clathrin bind cargo molecules with a different kind of sorting sequence. Cytosolic sorting signals that specifically bind to GGA adapter proteins include Asp-X-Leu-Leu and Asp-Phe-Gly-X- $\Phi$ sequences (where X and $\Phi$ are defined as above).

FIGURE 14-18 Structure of clathrin coats. (a) A clathrin molecule, called a triskelion, is composed of three heavy and three light chains. It has an intrinsic curvature due to the bend in the heavy chains. (b) Clathrin coats were formed in vitro by mixing purified clathrin heavy and light chains with AP2 complexes in the absence of membranes. Cryoelectron micrographs of more than 1000 assembled hexagonal clathrin barrel particles were analyzed by digital image processing to generate an average structural representation. The processed image shows only the clathrin heavy chains in a structure composed of 36 triskelions. Three representative triskelions are highlighted in red, yellow, and green. Some of the AP2 complexes packed into the interior of the clathrin cage are also visible in this representation. See B. Pishvaee and G. Payne, 1998, Cell 95:443. (Part (b) republished with permission of Nature, from Fotin, A. et al., "Molecular model for a complete clathrin lattice from electron cryomicroscopy," 2004, Nature 432:573-9; permission conveyed through the Copyright Clearance Center, Inc.)

Some vesicles that bud from the trans-Golgi network have coats composed of the AP3 complex. Although the AP3 complex does contain a binding site for clathrin similar to those in the AP1 and AP2 complexes, it is not clear whether clathrin is necessary for the functioning of AP3-containing vesicles because mutant versions of AP3 that lack the clathrin binding site appear to be fully functional. AP3-coated vesicles mediate trafficking to the lysosome, but they appear to bypass the late endosome and fuse directly with the lysosomal membrane (see Figure 14-17, step ■). In certain types of cells, such AP3 vesicles mediate protein transport to specialized storage compartments related to the lysosome. For example, AP3 is required for delivery of proteins to melanosomes, which contain the black pigment melanin in skin cells, and to platelet storage vesicles in megakaryocytes, large cells that fragment into dozens of platelets. Mice with mutations in either of two different subunits of AP3 not only have abnormal skin pigmentation but also exhibit bleeding disorders. The latter occur because platelets require normal storage vesicles in order to repair tears in blood vessels.

## Dynamin Is Required for Pinching Off of Clathrin-Coated Vesicles

A fundamental step in the formation of a transport vesicle that we have not yet considered is how a vesicle bud is pinched off from the donor membrane. In the case of clathrin-coated vesicles, a cytosolic protein called dynamin is essential for the release of complete vesicles. At the later stages of bud formation, dynamin polymerizes around the neck portion of the bud and then hydrolyzes GTP. The energy derived from GTP hydrolysis is thought to drive a conformational change in dynamin that stretches the neck until the vesicle pinches off (Figure 14-19).

Incubation of cell extracts with a nonhydrolyzable derivative of GTP provides dramatic evidence for the importance of dynamin in the pinching off of clathrin/AP2-coated vesicles during endocytosis. Such treatment leads to accumulation of clathrin-coated vesicle buds with excessively long necks that are surrounded by polymeric dynamin but do not pinch off (Figure 14-20). Likewise, cells expressing
(a) Triskelion structure
![li1894542480e](li1894542480e.jpg)
(b)
![li1904542480e](li1904542480e.jpg)
mutant forms of dynamin that cannot bind GTP do not form clathrin-coated vesicles and instead accumulate similar longnecked vesicle buds encased with polymerized dynamin.

Like COPI and COPII vesicles, clathrin-coated vesicles normally lose their coat soon after their formation. Cytosolic Hsp70, a constitutive chaperone protein found in all eukaryotic cells, is thought to use energy derived from the hydrolysis of ATP to drive depolymerization of the clathrin coat
![li1914542480e](li1914542480e.jpg)

FIGURE 14-19 Model for dynamin-mediated pinching off of clathrin-coated vesicles. After a vesicle bud forms, dynamin polymerizes over the neck. By a mechanism that is not well understood, dynamin-catalyzed hydrolysis of GTP leads to release of the vesicle from the donor membrane. Note that membrane proteins in the donor membrane are incorporated into vesicles by interacting with AP complexes in the coat. See K. Takel et al., 1995, Nature 374:186.

![li1924542480e](li1924542480e.jpg)
into triskelions. In the case of endocytic vesicles, uncoating not only releases triskelions for reuse in the formation of additional vesicles, but also exposes $v$-SNAREs for use in fusion with target membranes. Vesicle uncoating by cytosolic Hsp70 appears to be activated by a co-chaperone, auxillin, that contains a domain that stimulates the ATP hydrolysis by Hsp70. Conformational changes that occur when ARF switches from the GTP-bound to the GDP-bound state are thought to regulate the timing of clathrin coat depolymerization, but how the action of Hsp70 and auxillin is coupled to ARF switching is not well understood.

## Mannose 6-Phosphate Residues Target Soluble Proteins to Lysosomes

As we have seen, many of the sorting signals that direct cargo-protein trafficking in the secretory pathway are short amino acid sequences in the targeted protein. In contrast, the sorting signal that directs soluble lysosomal enzymes from the trans-Golgi network to the late endosome is a carbohydrate residue, mannose 6-phosphate (M6P), which is formed in the cis-Golgi. The addition and initial processing of one or more preformed N -linked oligosaccharide precursors in the rough ER is the same for lysosomal enzymes as for membrane and secreted proteins, yielding core

EXPERIMENTAL FIGURE 14-20 GTP hydrolysis by dynamin is required for the pinching off of clathrin-coated vesicles in cell-free extracts. A preparation of nerve terminals, which undergo extensive endocytosis, was lysed by treatment with distilled water and incubated with GTP- $\gamma$-S, a nonhydrolyzable derivative of GTP. After sectioning, the preparation was treated with gold-tagged anti-dynamin antibody and viewed in the electron microscope. This image, which shows a long-necked clathrin/AP-coated bud with polymerized dynamin lining the neck, reveals that buds can form in the absence of GTP hydrolysis, but vesicles cannot pinch off. The extensive polymerization of dynamin that occurs in the presence of GTP- $\gamma$-S probably does not occur during the normal budding process. [Republished with permission of Nature, from Taker, K. et al., "Tubular membrane invaginations coated by dynamin rings are induced by GTP-gamma S in nerve terminals," 1995, Nature 374 (6518):186-90; permission conveyed through the Copyright Clearance Center, Inc.]
$\operatorname{Man}_{9}(\mathrm{GlcNAc})_{2}$ chains (see Figure 13-18). In the cis-Golgi, the N -linked oligosaccharides present on most lysosomal enzymes undergo a two-step reaction sequence that generates M6P residues (Figure 14-21). The addition of M6P residues to the oligosaccharide chains of soluble lysosomal enzymes prevents these proteins from undergoing the further processing reactions characteristic of secreted and membrane proteins (see Figure 14-14).

As shown in Figure 14-22, the segregation of M6Pbearing lysosomal enzymes from secreted and membrane proteins occurs in the trans-Golgi network. Here transmembrane mannose 6-phosphate receptors bind the M6P residues on lysosome-destined proteins very tightly and specifically. Clathrin/AP1-coated vesicles containing the M6P receptor and bound lysosomal enzymes then bud from the trans-Golgi network, lose their coats, and subsequently fuse with a late endosome by mechanisms described previously. Because M6P receptors can bind M6P at the slightly acidic $\mathrm{pH}(\sim 6.5)$ of the trans-Golgi network, but not at a pH of less than 6 , the bound lysosomal enzymes are released within late endosomes, which have an internal pH of 5.0-5.5. Furthermore, a phosphatase within late endosomes usually removes the phosphate from M6P residues on lysosomal enzymes, preventing any rebinding to the M6P receptor that might occur in spite of the low pH there. Vesicles budding
![li1934542480e](li1934542480e.jpg)

FIGURE 14-21 Formation of mannose 6-phosphate (M6P) residues that target soluble enzymes to lysosomes. The M6P residues that direct proteins to lysosomes are generated in the cis-Golgi by two Golgi-resident enzymes. Step $\mathbf{B}$ An N-acetylglucosamine (GlcNAc) phosphotransferase transfers a phosphorylated GlcNAc group to carbon atom 6 of one or more mannose residues. Because only lysosomal enzymes contain
sequences (red) that are recognized and bound by this enzyme, phosphorylated GlcNAc groups are added specifically to lysosomal enzymes. Step After release of the modified protein from the phosphotransferase, a phosphodiesterase removes the GlcNAc group, leaving a phosphorylated mannose residue on the lysosomal enzyme. See A. B. Cantor et al., 1992, J. Biol. Chem. 267:23349, and S. Kornfeld, 1987, FASEBJ. 1:462.

from late endosomes, known as retromers, recycle the M6P receptor back to the trans-Golgi network. Eventually, mature late endosomes fuse with lysosomes, delivering the lysosomal enzymes to their final destination.

The sorting of soluble lysosomal enzymes in the transGolgi network (Figure 14-22, steps $\mathbf{B}-\mathbf{B}$ ) shares many
features with the trafficking of proteins between the ER and cis-Golgi compartments mediated by COPII and COPI vesicles. First, M6P acts as a sorting signal by interacting with the luminal domain of a receptor protein in the donor membrane. Second, the membrane-embedded receptors with their bound ligands are incorporated into the
![li1944542480e](li1944542480e.jpg)

FIGURE 14-22 Trafficking of soluble lysosomal enzymes from the trans-Golgi network and cell surface to lysosomes. Newly synthesized lysosomal enzymes, produced in the ER, acquire mannose 6-phosphate (M6P) residues in the cis-Golgi (see Figure 14-21). For simplicity, only one phosphorylated oligosaccharide chain is depicted, although lysosomal enzymes typically have many such chains. In the trans-Golgi network, proteins that bear the M6P sorting signal interact with M6P receptors in the membrane and thereby are directed into clathrin/AP1-coated vesicles (step $\mathbf{B}$ ). The coat surrounding released vesicles is rapidly depolymerized (step $\mathbf{B}$ ), and the uncoated transport vesicles fuse with late endosomes (step $\mathbf{B}$ ). After the phosphorylated
enzymes dissociate from the M6P receptors and are dephosphorylated, late endosomes subsequently fuse with a lysosome (step $\mathbf{B}$ ). Note that coat proteins and M6P receptors are recycled (steps $\mathbf{B}$ and $\mathbf{B}$ ), and that some receptors are delivered to the cell surface (step $\mathbf{B}$ ). Phosphorylated lysosomal enzymes are occasionally sorted from the trans-Golgi to the cell surface and secreted. These secreted enzymes can be retrieved by receptor-mediated endocytosis (steps $\mathbf{B}-\mathbf{B}$ ), a process that closely parallels trafficking of lysosomal enzymes from the trans-Golgi network to lysosomes. See G. Griffiths et al., 1988, Cell 52:329; S. Kornfeld, 1992, Annu. Rev. Biochem. 61:307; and G. Griffiths and J. Gruenberg, 1991, Trends Cell Biol. 1:5.

appropriate vesicles-in this case, either GGA- or AP1containing clathrin-coated vesicles-by interacting with the vesicle coat. Third, these transport vesicles fuse with only one specific organelle, here the late endosome, as the result of interactions between specific v-SNAREs and t-SNAREs. And finally, intracellular transport receptors dissociated from their bound ligand are recycled by retrograde vesicle trafficking.

## Study of Lysosomal Storage Diseases Revealed Key Components of the Lysosomal Sorting Pathway

A group of genetic disorders termed lysosomal storage diseases are caused by the absence of one or more lysosomal enzymes. As a result, undigested glycolipids and extracellular components that would normally be degraded by lysosomal enzymes accumulate in lysosomes as large inclusions. Patients with lysosomal storage diseases can have a variety of developmental, physiological, and neurological abnormalities depending on the type and severity of the storage defect. I-cell disease is a particularly severe type of lysosomal storage disease in which multiple enzymes are missing from the lysosomes. Cells from affected individuals lack the N -acetylglucosamine phosphotransferase that is required for formation of M6P residues on lysosomal enzymes in the cisGolgi (see Figure 14-21). Biochemical comparison of lysosomal enzymes from normal individuals with those from patients with I-cell disease led to the initial discovery of M6P as the lysosomal sorting signal. Lacking this signal, the lysosomal enzymes of affected individuals are secreted rather than being sorted to and sequestered in lysosomes.

When fibroblasts from patients with I-cell disease are grown in a medium containing lysosomal enzymes bearing M6P residues, the diseased cells acquire a nearly normal intracellular content of lysosomal enzymes. This finding indicates that the plasma membrane of these cells contains M6P receptors, which can internalize extracellular phosphorylated lysosomal enzymes by receptor-mediated endocytosis. This process, used by many cell-surface receptors to bring bound proteins or particles into the cell, is discussed in detail in the next section. It is now known that even in normal cells, some M6P receptors are transported to the plasma membrane and some phosphorylated lysosomal enzymes are secreted (see Figure 14-22). The secreted enzymes can be retrieved by receptor-mediated endocytosis and directed to lysosomes. This pathway thus scavenges any lysosomal enzymes that escape the usual M6P sorting pathway.

Hepatocytes from patients with I-cell disease contain a normal complement of lysosomal enzymes and no inclusions, even though these cells are defective in mannose phosphorylation. This finding implies that hepatocytes (the most abundant type of liver cell) employ an M6P-independent pathway for sorting lysosomal enzymes. The nature of this pathway, which may also operate in other cell types, is unknown.

## Protein Aggregation in the trans-Golgi May Function in Sorting Proteins to Regulated Secretory Vesicles

As noted in the chapter introduction, all eukaryotic cells continuously secrete certain proteins (constitutive secretion). Specialized secretory cells store other proteins in vesicles and secrete them only when triggered by a specific stimulus. One example of such regulated secretion occurs in pancreatic $\beta$ cells, which store newly made insulin in specialized secretory vesicles and secrete it in response to an elevation in blood glucose (see Figure 16-39). These and other secretory cells simultaneously use two different types of secretory vesicles to move proteins from the trans-Golgi network to the cell surface: unregulated transport vesicles (also called constitutive secretory vesicles) and regulated transport vesicles.

A common mechanism appears to sort regulated proteins as diverse as ACTH (adrenocorticotropic hormone), insulin, and trypsinogen into regulated secretory vesicles. Evidence for such a common mechanism comes from experiments in which recombinant DNA techniques were used to induce the synthesis of insulin and trypsinogen in pituitary tumor cells already synthesizing ACTH. In these cells, which do not normally express insulin or trypsinogen, all three proteins segregate into the same regulated secretory vesicles and are secreted together when a hormone binds to a receptor on the pituitary cells and causes a rise in cytosolic $\mathrm{Ca}^{2+}$. Although these three proteins share no identical amino acid sequences that might serve as a sorting sequence, they must have some common feature that signals their incorporation into regulated secretory vesicles.

Morphologic evidence suggests that sorting into the regulated secretory pathway is controlled by selective protein aggregation. For instance, immature vesicles in this pathway-those that have just budded from the trans-Golgi network-contain diffuse aggregates of secretory protein that are visible in the electron microscope. These aggregates are also found in vesicles that are in the process of budding, indicating that proteins destined for regulated secretory vesicles selectively aggregate together before their incorporation into the vesicles.

Other studies have shown that regulated secretory vesicles from mammalian secretory cells contain three acidic proteins, chromogranin A, chromogranin B, and secretogranin II, that together form aggregates when incubated at the ionic conditions ( pH 6.5 and $1 \mathrm{mM} \mathrm{Ca}^{2+}$ ) thought to occur in the trans-Golgi network; such aggregates do not form at the neutral pH of the ER. The selective aggregation of regulated secretory proteins together with chromogranin A, chromogranin B, or secretogranin II could be the basis for the sorting of these proteins into regulated secretory vesicles. Secretory proteins that do not associate with these proteins, and thus do not form aggregates, would be sorted into unregulated transport vesicles by default.

## Some Proteins Undergo Proteolytic Processing After Leaving the trans-Golgi

For some secretory proteins (e.g., growth hormone) and certain viral membrane proteins (e.g., VSV G protein), removal of the N-terminal ER signal sequence from the nascent chain is the only known proteolytic cleavage required to convert the polypeptide to the mature, active protein (see Figure 13-6). However, some membrane proteins and many soluble secretory proteins are initially synthesized as relatively long-lived, inactive precursors, termed proproteins, that require further proteolytic processing to generate the mature, active proteins. Examples of proteins that undergo such processing are soluble lysosomal enzymes; many membrane proteins, such as influenza hemagglutinin (HA); and secreted proteins such as serum albumin, insulin, glucagon, and the yeast a mating factor. In general, the proteolytic conversion of a proprotein to the corresponding mature protein occurs after the proprotein has been sorted in the trans-Golgi network to appropriate vesicles.

In the case of soluble lysosomal enzymes, the proproteins, called proenzymes, are sorted by the M6P receptor as catalytically inactive enzymes. In the late endosome or lysosome, a proenzyme undergoes a proteolytic cleavage that generates a smaller but enzymatically active polypeptide. Delaying the activation of lysosomal proenzymes until they reach the lysosome prevents them from digesting macromolecules in earlier compartments of the secretory pathway.

Normally, mature vesicles carrying secreted proteins to the cell surface are formed by fusion of several immature ones containing proprotein. Proteolytic cleavage of proproteins, such as proinsulin, occurs in vesicles after they move away from the trans-Golgi network (Figure 14-23). The proproteins of most constitutively secreted proteins (e.g., albumin) are cleaved only once at a site C-terminal to a dibasic recognition sequence such as Arg-Arg or Lys-Arg (Figure 14-24a). Proteolytic processing of proteins whose secretion is regulated generally entails additional cleavages. In the case of proinsulin, multiple cleavages of the single polypeptide chain yield the N-terminal B chain and the C-terminal A chain of mature insulin, which are linked by disulfide bonds, and the central C peptide, which is lost and subsequently degraded (Figure 14-24b).

## EXPERIMENTAL FIGURE 14-23 Proteolytic cleavage of proinsulin occurs in secretory vesicles after they have budded

from the trans-Golgi network. Serial sections of the Golgi region of an insulin-secreting cell were stained with (a) a monoclonal antibody that recognizes proinsulin, but not insulin, or (b) a different antibody that recognizes insulin, but not proinsulin. The antibodies, which were bound to electron-opaque gold particles, appear as dark dots in these electron micrographs (see Figure 4-33). Immature secretory vesicles (closed arrowheads) and vesicles budding from the trans-Golgi (arrows) stain with the proinsulin antibody, but not with the insulin antibody. These vesicles contain diffuse protein aggregates that include proinsulin and other regulated secreted proteins. Mature vesicles (open arrowheads) stain with insulin antibody, but not with proinsulin antibody, and have a dense core of almost crystalline insulin. Since budding
![li1954542480e](li1954542480e.jpg)
(b) Insulin antibody
![li1964542480e](li1964542480e.jpg)
and immature secretory vesicles contain proinsulin (not insulin), the proteolytic conversion of proinsulin to insulin must take place in these vesicles after they bud from the trans-Golgi network. The inset in (a) shows a proinsulin-rich secretory vesicle surrounded by a protein coat (dashed line). [Republished with permission of Elsevier, from Orci, L. et al., "Proteolytic maturation of insulin is a post-Golgi event which occurs in acidifying clathrin-coated secretory vesicles," 1987, Cell 49(6):865-868; permission conveyed through the Copyright Clearance Center, Inc.]

(a) Constitutive secreted proteins
![li1974542480e](li1974542480e.jpg)
(b) Regulated secreted proteins
![li1984542480e](li1984542480e.jpg)

FIGURE 14-24 Proteolytic processing of proproteins in the constitutive and regulated secretory pathways. The processing of proalbumin and proinsulin is typical of the constitutive and regulated pathways, respectively. The endoproteases that function in such processing cleave at the C-terminal end of a sequence of two consecutive amino acids. (a) The endoprotease furin acts on the precursors of constitutive secreted proteins. (b) Two endoproteases, PC2 and PC3, act on the precursors of regulated secreted proteins. The final processing of many such proteins is catalyzed by a carboxypeptidase that sequentially removes two basic amino acid residues at the C-terminus of a polypeptide. See D. Steiner et al., 1992, J. Biol. Chem. 267:23435.

The breakthrough in identifying the proteases responsible for such processing of secreted proteins came from analysis of yeast with a mutation in the KEX2 gene. These mutant cells synthesized the precursor of the $\alpha$ mating factor but could not proteolytically process it to the functional form and thus were unable to mate with cells of the opposite mating type. The wild-type KEX2 gene encodes an endoprotease that cleaves the $\alpha$-factor precursor at a site C-terminal to Arg-Arg and Lys-Arg residues. Mammals contain a family of endoproteases homologous to the yeast KEX2 protein, all of which cleave a protein chain on the C-terminal side of an Arg-Arg or Lys-Arg sequence. One, called furin, is found in all mammalian cells; it processes proteins such as albumin that are secreted constitutively. In contrast, the PC2
and PC3 endoproteases are found only in cells that exhibit regulated secretion; these enzymes are localized to regulated secretory vesicles and proteolytically cleave the precursors of many hormones at specific sites.

## Several Pathways Sort Membrane Proteins to the Apical or Basolateral Region of Polarized Cells

The plasma membrane of a polarized epithelial cell is divided into two domains: apical and basolateral. Tight junctions located between the two domains prevent the movement of plasma-membrane proteins between them (see Figure 20-11). Several sorting mechanisms direct newly synthesized membrane proteins to either the apical or the basolateral domain of epithelial cells, and any one protein may be sorted by more than one mechanism. As a result of this sorting and the restriction on protein movement within the plasma membrane by tight junctions, distinct sets of proteins are found in the apical and basolateral domains. This preferential localization of certain transport proteins is critical to a variety of important physiological functions, such as absorption of nutrients from the intestinal lumen and acidification of the stomach lumen (see Figures 11-30 and 11-31).

Microscopic and cell-fractionation studies indicate that proteins destined for either the apical or the basolateral membrane are initially transported together to the membranes of the trans-Golgi network. In some cases, proteins destined for the apical membrane are sorted into their own transport vesicles that bud from the trans-Golgi network and then move to the apical region, whereas proteins destined for the basolateral membrane are sorted into other vesicles that move to the basolateral region. The different vesicle types can be distinguished by their protein constituents, including distinct Rab and v-SNARE proteins, which apparently target them to the appropriate plasma-membrane domain. In this mechanism, segregation of proteins destined for the two domains occurs as cargo proteins are incorporated into particular types of vesicles budding from the trans-Golgi network.

Such direct basolateral-apical sorting has been investigated in cultured Madin-Darby canine kidney (MDCK) cells, a line of cultured polarized epithelial cells (see Figure 4-4). In MDCK cells infected with the influenza virus, progeny viruses bud only from the apical membrane, whereas in cells infected with vesicular stomatitis virus, progeny viruses bud only from the basolateral membrane. This difference occurs because the HA glycoprotein of influenza virus is transported from the Golgi complex exclusively to the apical membrane and the VSV G protein is transported only to the basolateral membrane (Figure 14-25).

Mutational studies on proteins, such as the VSV G protein, that are specifically targeted to the basolateral domain have defined targeting sequences in their cytosolic domains that fall into two major classes. These motifs, known as a tyrosine-based motif and a di-leucine-based motif, correspond to motifs described in the next section that are required for membrane proteins to associate with clathrin adapter protein complexes. These results strongly implicate

FIGURE 14-25 Sorting of proteins destined for the apical and basolateral plasma membranes of polarized cells. When cultured MDCK cells are infected simultaneously with VSV and influenza virus, the VSV G protein (purple) is found only on the basolateral membrane, whereas the influenza HA glycoprotein (green) is found only on the apical membrane. Some cellular proteins (orange circle), especially those with a GPI anchor, are likewise sorted directly to the apical membrane and others to the basolateral membrane (not shown) via specific transport vesicles that bud from the trans-Golgi network. In certain polarized cells, some apical and basolateral proteins are transported together to the basolateral surface; the apical proteins (yellow oval) then move selectively, by endocytosis and transcytosis, to the apical membrane. See K. Simons and A. Wandinger-Ness, 1990, Cell 62:207, and K. Mostovet al., 1992, J. Cell Biol. 116:577.
![li1994542480e](li1994542480e.jpg)
clathrin-coated vesicles in the sorting of proteins to the basolateral membrane.

Among the cellular proteins that undergo similar apical-basolateral sorting in the Golgi are those with a glycosylphosphatidylinositol (GPI) membrane anchor. In MDCK cells and most other types of epithelial cells, GPI-anchored proteins are targeted to the apical membrane. In membranes, GPI-anchored proteins are clustered into lipid rafts, which are rich in sphingolipids (see Chapter 7). This finding suggests that lipid rafts are localized to the apical membrane along with proteins that preferentially partition them in many cells. However, the GPI anchor is not an apical sorting signal in all polarized cells; in thyroid cells, for example, GPI-anchored proteins are targeted to the basolateral membrane. Other than GPI anchors, no unique sequences have been identified that are both necessary and sufficient to target proteins to either the apical or basolateral domain. Instead, each membrane protein may contain multiple sorting signals, any one of which can target it to the appropriate plasma-membrane domain. The identities of these complex signals and of the vesicle coat proteins that recognize them are currently being pursued for a number of different proteins that are sorted to specific plasma-membrane domains of polarized epithelial cells.

Another mechanism for sorting apical and basolateral proteins, also illustrated in Figure 14-25, operates in hepatocytes. The basolateral membranes of hepatocytes face the blood (like those of intestinal epithelial cells), and the apical membranes line the small intercellular channels into which bile is secreted. In hepatocytes, newly made apical and basolateral proteins are first transported in vesicles from the trans-Golgi network to the basolateral region and incorporated into the plasma membrane by exocytosis (i.e., fusion of the vesicle membrane with the plasma membrane). From there, both basolateral and apical proteins are endocytosed in the same vesicles, but then their paths diverge.

The endocytosed basolateral proteins are sorted into transport vesicles that recycle them to the basolateral membrane. In contrast, the apically destined endocytosed proteins are sorted into transport vesicles that move across the cell and fuse with the apical membrane, a process called transcytosis. This process is also used to move extracellular materials from one side of an epithelium to another. Even in epithelial cells, such as MDCK cells, in which apical-basolateral protein sorting occurs in the Golgi, transcytosis may provide an editing function by which an apical protein sorted incorrectly to the basolateral membrane is subjected to endocytosis and then correctly delivered to the apical membrane.

## KEY CONCEPTS OF SECTION 14.4

## Later Stages of the Secretory Pathway

- The trans-Golgi network is a major branch point in the secretory pathway where soluble secreted proteins, lysosomal proteins, and in some cells, membrane proteins destined for the basolateral or apical plasma membrane are segregated into different transport vesicles.
- Many vesicles that bud from the trans-Golgi network as well as endocytic vesicles bear a coat composed of AP (adapter protein) complexes and clathrin (see Figure 14-18).
- The pinching off of clathrin-coated vesicles requires dynamin, which forms a collar around the neck of the vesicle bud and hydrolyzes GTP (see Figure 14-19).
- Soluble enzymes destined for lysosomes are modified in the cis-Golgi by the addition of multiple mannose 6-phosphate (M6P) residues to their oligosaccharide chains.
- M6P receptors in the membrane of the trans-Golgi network bind proteins bearing M6P residues and direct them to late endosomes, where receptors and their ligand proteins

dissociate. The receptors are then recycled to the Golgi or plasma membrane, and the lysosomal enzymes are delivered to lysosomes (see Figure 14-22).

- Regulated secreted proteins are concentrated and stored in secretory vesicles to await a neuronal or hormonal signal for exocytosis. Protein aggregation within the trans-Golgi network may play a role in sorting secreted proteins to the regulated secretory pathway.
- Many proproteins transported through the secretory pathway undergo post-Golgi proteolytic cleavages that yield the mature, active proteins. This proteolytic maturation can occur in vesicles carrying proteins from the trans-Golgi network to the cell surface, in the late endosome, or in the lysosome.
- In polarized epithelial cells, membrane proteins destined for the apical and basolateral domains of the plasma membrane are sorted in the trans-Golgi network into different transport vesicles (see Figure 14-25). The GPI anchor is the only apical-basolateral sorting signal identified so far.
- In hepatocytes and some other polarized cells, all plasmamembrane proteins are directed first to the basolateral membrane. Apically destined proteins are then endocytosed and moved across the cell to the apical membrane (transcytosis).


### 14.5 Receptor-Mediated Endocytosis

In previous sections, we have explored the main pathways whereby soluble and membrane secretory proteins synthesized on the rough ER are delivered to the cell surface or other destinations. Cells can also internalize materials from their surroundings and sort these materials to particular destinations. A few cell types (e.g., macrophages) can take up whole bacteria and other large particles by phagocytosis, a nonselective actin-mediated process in which extensions of the plasma membrane envelop the ingested material, forming large vesicles called phagosomes (see Figure 17-19). All eukaryotic cells continually engage in endocytosis, a process in which a small region of the plasma membrane invaginates to form a membrane-limited vesicle about $0.05-0.1 \mu \mathrm{~m}$ in diameter. In one form of endocytosis, called pinocytosis, small droplets of extracellular fluid and any material dissolved in it are nonspecifically taken up. Our focus in this section, however, is on receptor-mediated endocytosis, in which a specific receptor on the cell surface binds tightly to an extracellular macromolecular ligand that it recognizes; the plasma-membrane region containing the receptor-ligand complex buds inward to form a pit and then pinches off, becoming a transport vesicle.

Among the common macromolecules that vertebrate cells internalize by receptor-mediated endocytosis are cholesterol-containing low-density lipoprotein (LDL) particles, the iron-carrying protein transferrin, many protein hormones (e.g., insulin), and certain glycoproteins. Receptor-mediated endocytosis of such ligands generally
occurs via clathrin/AP2-coated pits and vesicles in a process similar to the packaging of lysosomal enzymes by the binding of M6P in the trans-Golgi network (see Figure 14-22). As noted earlier, some M6P receptors are found on the cell surface, and these receptors participate in the receptor-mediated endocytosis of lysosomal enzymes that are mistakenly secreted. In general, the transmembrane receptor proteins that function in the uptake of extracellular ligands are internalized from the cell surface during endocytosis and are then sorted and recycled back to the cell surface, much as M6P receptors are recycled to the plasma membrane and transGolgi. The rate at which a ligand is internalized is limited by the amount of its corresponding receptor on the cell surface.

Clathrin/AP2-coated pits make up about 2 percent of the surface of cells such as hepatocytes and fibroblasts. Many internalized ligands have been observed in clathrin/AP2-coated pits and vesicles, which are thought to function as intermediates in the endocytosis of most (though not all) ligands bound to cell-surface receptors (Figure 14-26). Some receptors are clustered over clathrin-coated pits even in the absence of ligand. Other receptors diffuse freely in the plane of the plasma membrane but undergo a conformational change when they bind to ligand, so that when the receptor-ligand complex diffuses into a clathrin-coated pit, it is retained there. Two or more types of receptor-bound ligands, such as LDL and transferrin, can be seen in the same coated pit or vesicle.

## Cells Take Up Lipids from the Blood in the Form of Large, Well-Defined Lipoprotein Complexes

Lipids absorbed from the diet in the intestines or stored in adipose tissue can be distributed to cells throughout the body. To facilitate the mass transfer of lipids between cells, animals have evolved an efficient way to package from hundreds to thousands of lipid molecules into water-soluble, macromolecular carriers, called lipoproteins, that cells can take up from the circulation as an ensemble. A lipoprotein particle has a shell composed of proteins (apolipoproteins) overlying a cholesterol-containing phospholipid monolayer. The shell is amphipathic because its outer surface is hydrophilic, making the particle water soluble, and its inner surface is hydrophobic. Beneath the hydrophobic inner surface of the shell is a core of neutral lipids containing mostly cholesteryl esters, triglycerides, or both. Mammalian lipoproteins fall into different classes, defined by their differing buoyant densities. The class we will consider here is low-density lipoprotein (LDL). A typical LDL particle, depicted in Figure 14-27, is a sphere $20-25 \mathrm{~nm}$ in diameter. The amphipathic outer shell is composed of a phospholipid monolayer and a single molecule of a large protein known as apoB-100; the core of the particle is packed with cholesterol in the form of cholesteryl esters.

Two general experimental approaches have been used to study how LDL particles enter cells. The first method makes use of LDL that has been labeled by the covalent attachment of radioactive ${ }^{125} \mathrm{I}$ to the side chains of tyrosine residues in

EXPERIMENTAL FIGURE 14-26 The initial stages of receptor-mediated endocytosis of low-density lipoprotein (LDL) particles are revealed by electron microscopy. Cultured human fibroblasts were incubated in a medium containing LDL particles covalently linked to the electron-dense, iron-containing protein ferritin; each small iron particle in ferritin is visible as a small dot under the electron microscope. Cells were initially incubated at $4^{\circ} \mathrm{C}$; at this temperature LDL can bind to its receptor, but internalization does not occur. After excess LDL not bound to the cells was washed away, the cells were warmed to $37^{\circ} \mathrm{C}$ and then prepared for microscopy at periodic intervals. (a) A coated pit, showing the clathrin coat on the inner (cytosolic) surface of the pit, soon after the temperature was raised. (b) A pit containing LDL apparently closing on itself to form a coated vesicle. (c) A coated vesicle containing ferritintagged LDL particles. (d) Ferritin-tagged LDL particles in a smooth-surfaced early endosome 6 minutes after internalization began. See also M. S. Brown and J. Goldstein, 1986, Science 232:34. [Photographs republished with permission of Nature, from Goldstein, J. et al., "Coated pits, coated vesicles, and receptor-mediated endocytosis," 1979, Nature 279:679-685; permission conveyed through the Copyright Clearance Center, Inc.]
![li2004542480e](li2004542480e.jpg)

FIGURE 14-27 Model of low-density lipoprotein (LDL).
All classes of lipoproteins have the same general structure: an amphipathic shell composed of apolipoprotein, a phospholipid monolayer (not bilayer), and cholesterol, and a hydrophobic core composed mostly of cholesteryl esters or triglycerides, or both, but with minor amounts of other neutral lipids (e.g., some vitamins). This model of LDL is based on electron microscopy and other low-resolution biophysical methods. LDL is unique in that it contains only a single molecule of one type of apolipoprotein (ApoB), which appears to wrap around the outside of the particle as a band of protein. The other lipoproteins contain multiple apolipoprotein molecules, often of different types. See M. Krieger, 1995, in E. Haber, ed., Molecular Cardiovascular Medicine, Scientific American Medicine, pp. 31-47.
apoB-100 on the surfaces of the LDL particles. After cultured cells are incubated for several hours with the labeled LDL, it is possible to determine how much LDL is bound to the surfaces of cells, how much is internalized, and how much of the apoB-100 component of the LDL is degraded by enzymatic hydrolysis to individual amino acids. The degradation of apoB-100 can be detected by the release of ${ }^{125}$ I-tyrosine into the culture medium. Figure 14-28 shows the time course of these events in receptor-mediated cellular LDL processing, determined by pulse-chase experiments with a fixed concentration of ${ }^{125}$ I-labeled LDL. These experiments clearly demonstrate the order of events: surface binding of LDL $\rightarrow$ internalization $\rightarrow$ degradation. The second approach involves tagging LDL particles with an electrondense label that can be detected by electron microscopy. Such studies can reveal the details of how LDL particles first bind to the surface of cells at clathrin-coated endocytic pits, then remain associated with the coated pits as they invaginate and bud off to form coated vesicles and are finally transported to endosomes (see Figure 14-26).

## Receptors for Macromolecular Ligands Contain Sorting Signals That Target Them for Endocytosis

The key to understanding how LDL particles bind to the cell surface and are then taken up into endocytic vesicles was the discovery of the LDL receptor. The LDL receptor is an 839 -residue glycoprotein with a single transmembrane

![li2014542480e](li2014542480e.jpg)

EXPERIMENTAL FIGURE 14-28 Pulse-chase experiment demonstrates precursor-product relations in cellular uptake of LDL. Cultured normal human skin fibroblasts were incubated in a medium containing ${ }^{125} \mathrm{I}$-LDL for 2 hours at $4^{\circ} \mathrm{C}$ (the pulse). After excess ${ }^{125} \mathrm{I}$-LDL not bound to the cells was washed away, the cells were incubated at $37^{\circ} \mathrm{C}$ for the indicated amounts of time in the absence of external LDL (the chase). The amounts of surface-bound, internalized, and degraded (hydrolyzed) ${ }^{125} \mathrm{I}$-LDL were measured. Binding, but not internalization or hydrolysis, of LDL apoB-100 occurs during the $4^{\circ} \mathrm{C}$ pulse. The data show the very rapid disappearance of bound ${ }^{125} \mathrm{I}$-LDL from the surface as it is internalized after the cells have been warmed to allow membrane movements. After a lag period of 15-20 minutes, lysosomal degradation of the internalized ${ }^{125} \mathrm{I}$-LDL commences. See M. S. Brown and J. L. Goldstein, 1976, Cell 9:663.
segment; it has a short C-terminal cytosolic segment and a long N-terminal exoplasmic segment that contains the LDL-binding domain. Seven cysteine-rich repeats form the LDL-binding domain, which interacts with the apoB-100 molecule in an LDL particle. Figure 14-29 shows how the LDL receptor facilitates internalization of LDL particles by receptor-mediated endocytosis. After internalized LDL particles reach lysosomes, lysosomal proteases hydrolyze their surface apolipoproteins and lysosomal cholesteryl esterases hydrolyze their core cholesteryl esters. The unesterified cholesterol is then free to leave the lysosome and be used as necessary by the cell in the synthesis of membranes or various cholesterol derivatives.

■
The discovery of the LDL receptor and an understanding of how it functions came from studying cells from patients with familial hypercholesterolemia $(F H)$, a hereditary disease that is marked by elevated plasma LDL levels and is now known to be caused by mutations in the LDL receptor (LDLR) gene. In patients who have one normal and one defective copy of the $L D L R$ gene (heterozygotes), LDL in the blood is increased about twofold. Those with two defective $L D L R$ genes (homozygotes) have LDL levels that are from fourfold to sixfold higher than normal. Without medical intervention, FH
heterozygotes commonly develop cardiovascular disease about 10 years earlier than normal people do, and FH homozygotes usually die of heart attacks before reaching their late twenties.

A variety of mutations in the $L D L R$ gene can cause FH. Some mutations prevent the synthesis of the LDLR protein; others prevent proper folding of the receptor protein in the ER, leading to its premature degradation (see Chapter 13); and still other mutations reduce the ability of the LDL receptor to bind LDL tightly. A particularly informative group of mutant receptors are expressed on the cell surface and bind LDL normally but cannot mediate the internalization of bound LDL. In individuals with this type of defect, plasma-membrane receptors for other ligands are internalized normally, but the mutant LDL receptor is not recruited into coated pits. Analysis of this mutant receptor and other mutant LDL receptors generated experimentally and expressed in fibroblasts identified a four-residue motif in the cytosolic segment of the receptor that is crucial for its internalization: Asn-Pro-X-Tyr, where X can be any amino acid. This NPXY sorting signal binds to the AP2 complex, linking the clathrin/AP2 coat to the cytosolic segment of the LDL receptor in coated pits. A mutation in any of the conserved residues of the NPXY signal abolishes the ability of the LDL receptor to be incorporated into coated pits.

A small number of individuals who exhibit the usual symptoms associated with FH produce normal LDL receptors. In these individuals, the gene encoding the AP2 subunit protein that binds the NPXY sorting signal is defective. As a result, LDL receptors are not incorporated into clathrin/ AP2-coated vesicles, and endocytosis of LDL particles is compromised. Analyses of patients with this genetic disorder highlight the importance of adapter proteins in protein trafficking mediated by clathrin-coated vesicles.

Mutational studies have shown that other cell-surface receptors can be directed into budding clathrin/AP2-coated pits by a YXXФ sorting signal. Recall from our earlier discussion that this same sorting signal recruits membrane proteins into clathrin/AP1-coated vesicles that bud from the trans-Golgi network by binding to a subunit of AP1 (see Table 14-2). All these observations indicate that YXXФ is a widely used signal for sorting membrane proteins to clathrin-coated vesicles.

In some cell-surface proteins, however, other sequences (e.g., Leu-Leu) or covalently linked ubiquitin molecules act as signals for endocytosis. Among the proteins associated with clathrin/AP2-coated vesicles, several contain domains that bind specifically to ubiquitin, and it has been hypothesized that these vesicle-associated proteins mediate the selective incorporation of ubiquitinylated membrane proteins into endocytic vesicles. As described later, the ubiquitin tag on endocytosed membrane proteins is also recognized at a later stage in the endocytic pathway and plays a role in delivering these proteins into the interior of the lysosome, where they are degraded.

![li2024542480e](li2024542480e.jpg)

FIGURE 14-29 Endocytic pathway for internalizing low-density lipoprotein (LDL). Step A cell-surface LDL receptor binds to an ApoB protein embedded in the phospholipid outer layer of an LDL particle. Interaction between the NPXY sorting signal in the cytosolic tail of the LDL receptor and the AP2 complex incorporates the receptorligand complex into a forming endocytic vesicle. Step $\mathbf{B}$ Clathrincoated pits containing receptor-LDL complexes are pinched off by the same dynamin-mediated mechanism used to form clathrin/AP1-coated vesicles on the trans-Golgi network (see Figure 14-19). Step After the vesicle coat is shed, the uncoated endocytic vesicle (early endosome)

## The Acidic pH of Late Endosomes Causes Most Receptor-Ligand Complexes to Dissociate

The overall rate of endocytic internalization of the plasma membrane is quite high: cultured fibroblasts regularly internalize 50 percent of their cell-surface proteins and phospholipids each hour. Most cell-surface receptors that undergo endocytosis will repeatedly deposit their ligands within the cell and then recycle to the plasma membrane to mediate internalization of ligand molecules once again. The LDL receptor, for instance, makes one round trip into and out of the cell interior every $10-20$ minutes, for a total of about a hundred trips in its 20 -hour life span.
fuses with a late endosome. The acidic pH in this compartment causes a conformational change in the LDL receptor that leads to release of the bound LDL particle. Step $\mathbf{B}$ The late endosome fuses with a lysosome, and the proteins and lipids of the free LDL particle are broken down into their constituent parts by enzymes in the lysosome. Step The LDL receptor is recycled to the cell surface, where at the neutral pH of the exterior medium, the receptor undergoes a conformational change so that it can bind another LDL particle. See M. S. Brown and J. L. Goldstein, 1986, Science 232:34, and G. Rudenko et al., 2002, Science 298:2353.

Internalized receptor-ligand complexes commonly follow the pathway depicted for the M6P receptor in Figure 14-22 and the LDL receptor in Figure 14-29. Endocytosed cellsurface receptors typically dissociate from their ligands within late endosomes, which appear as spherical vesicles with tubular branching membranes located a few micrometers from the cell surface. The original experiments that defined the late endosome as a sorting compartment used the asialoglycoprotein receptor. This liver-specific receptor mediates the binding and internalization of abnormal glycoproteins whose oligosaccharides terminate in galactose rather than the normal sialic acid; hence the name asialoglycoprotein. Electron microscopy of liver cells perfused with

asialoglycoprotein reveal that 5-10 minutes after internalization, ligand molecules are found in the lumen of late endosomes, while the tubular membrane extensions are rich in receptor and rarely contain ligand. These findings indicate that the late endosome is the organelle in which receptors and ligands are uncoupled.

The dissociation of receptor-ligand complexes in late endosomes occurs not only in the endocytic pathway, but also in the delivery of soluble lysosomal enzymes via the secretory pathway (see Figure 14-22). As discussed in Chapter 11, the membranes of late endosomes and lysosomes contain Vclass proton pumps that act in concert with $\mathrm{Cl}^{-}$channels to acidify the vesicle lumen (see Figure 11-14). Most receptors, including the M6P receptor and cell-surface receptors for LDL and asialoglycoprotein, bind their ligands tightly at neutral pH , but release their ligands if the pH is lowered to 6.0 or below. The late endosome is the first organelle encountered by receptor-ligand complexes whose luminal pH is sufficiently acidic to promote dissociation of most endocytosed receptors from their tightly bound ligands.

The mechanism by which the LDL receptor releases bound LDL particles is now understood in detail (Figure 14-30). At the endosomal pH of $5.0-5.5$, histidine residues in a region known as the $\beta$-propeller domain of the receptor become protonated, forming a site that can bind with high affinity to the negatively charged cysteine-rich repeats in the LDL-binding domain. This intramolecular interaction sequesters the repeats in a conformation that cannot simultaneously bind to apoB-100, thus causing release of the bound LDL particle.

## The Endocytic Pathway Delivers Iron to Cells Without Dissociation of the TransferrinTransferrin Receptor Complex in Endosomes

The endocytic pathway involving the transferrin receptor and its ligand differs from the LDL pathway in that the receptor-ligand complex does not dissociate in late endosomes. Nonetheless, changes in pH also mediate the sorting of receptors and ligands in the transferrin pathway, which functions to deliver iron to cells.

Transferrin, a major glycoprotein in the blood, transports iron to all tissue cells from the liver (the main site of iron storage in the body) and from the intestine (the site of iron absorption). The iron-free form, apotransferrin, binds two $\mathrm{Fe}^{3+}$ ions very tightly to form ferrotransferrin. All mammalian cells contain cell-surface transferrin receptors that bind ferrotransferrin with high affinity at neutral pH , after which the receptor-bound ferrotransferrin is subjected to endocytosis. Like the components of an LDL particle, the two $\mathrm{Fe}^{3+}$ atoms remain in the cell, but the apotransferrin part of the ligand does not dissociate from the receptor in the late endosome, and within minutes after being endocytosed, apotransferrin is returned to the cell surface and secreted from the cell.

As depicted in Figure 14-31, the explanation for the behavior of the transferrin receptor-ligand complex lies in the unique ability of apotransferrin to remain bound to the transferrin receptor at the low $\mathrm{pH}(5.0-5.5)$ of late endosomes. At a pH of less than 6.0 , the two bound $\mathrm{Fe}^{3+}$ atoms dissociate from ferrotransferrin, are reduced to $\mathrm{Fe}^{2+}$ by a
![li2034542480e](li2034542480e.jpg)

![li2044542480e](li2044542480e.jpg)

FIGURE 14-31 The transferrin cycle operates in all growing mammalian cells. Step The transferrin dimer carrying two bound atoms of $\mathrm{Fe}^{3+}$, called ferrotransferrin, binds to the transferrin receptor at the cell surface. Step $\mathbf{B}$ Interaction between the tail of the transferrin receptor and the AP2 adapter complex incorporates the receptorligand complex into endocytic clathrin-coated vesicles. Steps $\mathbf{B}$ and $\mathbf{B}$ The vesicle coat is shed, and the endocytic vesicles fuse with the
membrane of the endosome. $\mathrm{Fe}^{3+}$ is released from the receptor-ferrotransferin complex in the acidic late endosome compartment. Step The apotransferrin protein remains bound to its receptor at this pH , and they are recycled to the cell surface together. Step The neutral pH of the exterior medium causes release of the iron-free apotransferin. See A. Ciechanover et al., 1983, J. Biol. Chem. 258:9681.
metalloreductase located in the endosome, and are then exported into the cytosol by an endosomal transporter specific for divalent metal ions. The receptor-apotransferrin complex remaining after dissociation of the iron atoms is recycled back to the cell surface. Although apotransferrin binds tightly to its receptor at a pH of 5.0 or 6.0 , it does not bind at neutral pH . Hence the bound apotransferrin dissociates from the transferrin receptor when the recycling vesicles fuse with the plasma membrane and the receptor-ligand complex encounters the neutral pH of the extracellular interstitial fluid or growth medium. The recycled receptor is then free to bind another molecule of ferrotransferrin, and the released apotransferrin is carried in the bloodstream to the liver or intestine to be reloaded with iron.

- The endocytic pathway delivers some ligands (e.g., LDL particles) to lysosomes, where they are degraded. Transport vesicles from the cell surface first fuse with late endosomes, which subsequently fuse with lysosomes.
- Most receptor-ligand complexes dissociate in the acidic milieu of the late endosome; the receptors are recycled to the plasma membrane, while the ligands are sorted to lysosomes (see Figure14-29).
- Iron is imported into cells by an endocytic pathway in which $\mathrm{Fe}^{3+}$ ions are released from ferrotransferrin in the late endosome. The receptor-apotransferrin complex that remains is recycled to the cell surface, where the complex dissociates, releasing both the receptor and apotransferrin for reuse.


### 14.6 Directing Membrane Proteins and Cytosolic Materials to the Lysosome

The major function of lysosomes is to degrade extracellular materials taken up by the cell and to degrade intracellular components under certain conditions. Materials to be degraded must be delivered to the lumen of the lysosome, where the various degradative enzymes reside. As we have just seen, endocytosed ligands (e.g., LDL particles) that dissociate from their receptors in the late endosome subsequently enter the lysosomal lumen when the membrane of the late endosome fuses with the membrane of the lysosome (see Figure 14-29). Likewise, phagosomes carrying bacteria or other particulate matter can fuse with lysosomes, releasing their contents into the lumen for degradation.

It is apparent how the general vesicular trafficking mechanism discussed in this chapter can be used to deliver the luminal contents of an endosomal organelle to the lumen of the lysosome for degradation. However, membrane proteins delivered to the lysosome by the typical vesicular trafficking process we have discussed in this chapter should ultimately be delivered to the membrane of the lysosome. How, then, are membrane proteins delivered to the interior of the lysosome for degradation? As we will see in this section, the cell has two different specialized pathways for delivery of materials to the lysosomal lumen for degradation, one for membrane proteins and one for cytosolic materials. The first pathway, used to degrade endocytosed membrane proteins, uses an unusual type of vesicle that buds into the lumen of the endosome to produce a multivesicular endosome. The second pathway, known as autophagy, involves the de novo formation of a double-membrane organelle known as an autophagosome that envelops cytosolic material, such as soluble cytosolic proteins, or sometimes organelles, such as peroxisomes or mitochondria. Both pathways lead to fusion of either the multivesicular endosome or autophagosome with the lysosome, depositing the contents of these organelles into the lysosomal lumen for degradation.

## Multivesicular Endosomes Segregate Membrane Proteins Destined for the Lysosomal Membrane from Proteins Destined for Lysosomal Degradation

Resident lysosomal membrane proteins, such as V-class proton pumps and amino acid transporters, can carry out their functions and remain in the lysosomal membrane, where they are protected from degradation by the soluble hydrolytic enzymes in the lumen. Such proteins are delivered to the lysosomal membrane by transport vesicles that bud from either the trans-Golgi network or the endosome by the same basic mechanisms described in earlier sections. In contrast, endocytosed membrane proteins that are to be degraded are transferred in their entirety to the interior of the lysosome by a specialized delivery mechanism. Lysosomal degradation of cell-surface receptors for extracellular signaling molecules is a common mechanism for controlling the sensitivity of cells to such signals (see Chapter 15). Receptors that become damaged are also targeted for lysosomal degradation.

Early evidence that membranes can be delivered to the lumen of a membrane-bounded compartment came from electron micrographs showing membrane vesicles and fragments of membranes within endosomes and lysosomes. Parallel experiments in yeast revealed that endocytosed receptor proteins targeted to the vacuole (the yeast organelle equivalent to the lysosome) were primarily associated with membrane fragments and small vesicles within the interior of the vacuole rather than with the vacuole surface membrane.

These observations suggest that endocytosed membrane proteins can be incorporated into specialized vesicles that form at the endosomal membrane (Figure 14-32). Although these vesicles are similar in size and appearance to transport vesicles, they differ topologically. Transport vesicles bud outward from the surface of a donor organelle into the cytosol, whereas vesicles within the endosome bud inward from the surface into the lumen (away from the cytosol). Mature endosomes containing numerous vesicles in their interior are usually called multivesicular endosomes (or bodies). The surface membrane of a multivesicular endosome then fuses with the membrane of a lysosome, thereby delivering its internal vesicles and the membrane proteins they contain into the lysosome interior for degradation. Thus the sorting of proteins in the endosomal membrane determines which ones will remain on the lysosome surface (e.g., pumps and transporters) and which ones will be incorporated into internal vesicles and ultimately degraded in lysosomes.

Many of the proteins required for inward budding of the endosomal membrane were first identified by mutations in yeast that blocked the delivery of membrane proteins to the interior of the vacuole. More than 10 such "budding" proteins have been identified in yeast, most of which have significant similarities to mammalian proteins that evidently perform the same function in mammalian cells. The current model of endosomal budding to form multivesicular endosomes in mammalian cells is based primarily on studies

![li2054542480e](li2054542480e.jpg)

FIGURE 14-32 Delivery of plasma-membrane proteins to the lysosome interior for degradation. Early endosomes carrying endocytosed plasma-membrane proteins (blue) and vesicles carrying lysosomal membrane proteins (green) from the trans-Golgi network fuse with the late endosome, transferring their membrane proteins to the endosomal membrane (steps $\mathbf{\square}$ and $\mathbf{\square}$ ). Proteins to be degraded, such as those from the early endosome, are incorporated into vesicles that bud into the interior of the late endosome, eventually forming a multivesicular
endosome containing many such internal vesicles (step $\mathbf{\square}$ ). Fusion of a multivesicular endosome directly with a lysosome releases the internal vesicles into the lumen of the lysosome, where they can be degraded (step $\mathbf{\square}$ ). Because proton pumps and other lysosomal membrane proteins normally are not incorporated into internal endosomal vesicles, they are delivered to the lysosomal membrane and are protected from degradation. See F. Reggiori and D. J. Klionsky, 2002, Eukaryot. Cell 1:11, and D. J. Katzmann et al., 2002, Nat. Rev. Mol. Cell Biol. 3:893.
in yeast (Figure 14-33). Most cargo proteins that enter a multivesicular endosome are tagged with ubiquitin. Cargo proteins destined to enter a multivesicular endosome usually receive their ubiquitin tags at the plasma membrane, the trans-Golgi network, or the endosomal membrane. We have already seen how ubiquitin tagging can serve as a signal for degradation of cytosolic or misfolded ER proteins by proteasomes (see Chapters 3 and 13). When used as a signal for proteasomal degradation, the ubiquitin tag usually consists of a chain of covalently linked ubiquitin molecules (polyubiquitin), whereas ubiquitin used to tag proteins for entry into the multivesicular endosome usually takes the form of a single (monoubiquitin) molecule. In the membrane of the endosome, a ubiquitin-tagged peripheral membrane protein, known as Hrs, facilitates recruitment of a set of at least three different protein complexes to the membrane. These ESCRT (endosomal sorting complexes required for transport) proteins include the ubiquitin-binding protein Tsg101. The membrane-associated ESCRT proteins act to drive vesicle budding directed into the interior of the endosome as well as the loading of specific monoubiquitinylated membrane cargo proteins into the vesicle buds. Finally, the ESCRT proteins pinch off the vesicle by forming a filamentous spiral inside the neck of a vesicle bud, releasing it and the specific membrane cargo proteins it carries into the interior of the endosome. An ATPase, known as Vps4, uses the energy from ATP hydrolysis to disassemble the ESCRT proteins, releasing
them into the cytosol for another round of budding. In the fusion event that pinches off a completed endosomal vesicle, the ESCRT proteins and Vps4 may function like SNAREs and NSF, respectively, in the typical membrane-fusion process discussed previously (see Figure 14-10).

## Retroviruses Bud from the Plasma Membrane by a Process Similar to Formation of Multivesicular Endosomes

The vesicles that bud into the interior of endosomes have a topology similar to that of enveloped virus particles that bud from the plasma membrane of virus-infected cells. Moreover, recent experiments have demonstrated that a common set of proteins is required for both types of mem-brane-budding events. In fact, the two processes so closely parallel each other in mechanistic detail as to suggest that enveloped viruses have evolved mechanisms to recruit the cellular proteins used in inward endosomal budding for their own purposes.

The human immunodeficiency virus (HIV) is an enveloped retrovirus that buds from the plasma membrane of infected cells in a process driven by viral Gag protein, the major structural component of completed virus particles. Gag protein binds to the plasma membrane of an infected cell, and some 4000 Gag molecules polymerize into a spherical shell, producing a structure that looks like a vesicle bud protruding

![li2064542480e](li2064542480e.jpg)

FIGURE 14-33 Model of the mechanism for formation of multivesicular endosomes. In endosomal budding, ubiquitinylated Hrs on the endosomal membrane directs the loading of specific membrane cargo proteins (blue) into vesicle buds and then recruits cytosolic ESCRT protein complexes to the membrane (step $\mathbf{E}$ ). Note that both Hrs and the recruited cargo proteins are tagged with ubiquitin. After the set of bound ESCRT complexes mediates the completion and pinching off of the inwardly budding vesicles (step $\mathbf{E}$ ), these complexes are disassembled by the ATPase Vps 4 and returned to the cytosol (step $\mathbf{E}$ ). See text for discussion. See O. Pornillos et al., 2002, Trends Cell Biol. 12:569.
outward from the plasma membrane. Mutational studies with HIV have revealed that the N -terminal segment of Gag protein is required for association with the plasma membrane, whereas the C-terminal segment is required for pinching off of complete HIV particles. For instance, if the portion of the viral genome encoding the C-terminus of Gag is removed, HIV buds will form in infected cells, but pinching off does not occur, and thus no free virus particles are released.

The first indication that HIV budding employs the same molecular machinery as vesicle budding into endosomes came from the observation that Tsg101, an ESCRT protein, binds to the C-terminus of Gag protein. Subsequent findings have clearly established the mechanistic parallels between the two processes. For example, Gag is ubiquitinylated as part of the process of virus budding, and in cells with mutations in Tsg101 or Vps4, HIV virus buds accumulate but cannot pinch off from the membrane (Figure 14-34). Moreover, when a segment from the cellular Hrs protein is added to a truncated Gag protein by construction of the appropriate hybrid gene, proper budding and release of virus particles is restored. Taken together, these results indicate that Gag protein mimics the function of Hrs, redirecting ESCRT proteins to the plasma membrane, where they can function in the budding of virus particles.

Other enveloped retroviruses, such as murine leukemia virus and Rous sarcoma virus, have also been shown to require ESCRT complexes for their budding, although each virus appears to have evolved a somewhat different mechanism to recruit ESCRT complexes to the site of virus budding.

## The Autophagic Pathway Delivers Cytosolic Proteins or Entire Organelles to Lysosomes

When cells are placed under stressful conditions, such as starvation, they have the capacity to recycle macromolecules
for use as nutrients in a process of lysosomal degradation known as autophagy ("eating oneself"). The autophagic pathway begins with the formation of a flattened doublemembrane cup-shaped structure that envelops a region of the cytosol or an entire organelle (e.g., a mitochondrion), forming an autophagosome, or autophagic vesicle (Figure 14-35). The outer membrane of an autophagosome can fuse with a lysosome, delivering a large vesicle, bounded by a single membrane bilayer, to the interior of the lysosome. Lipases and proteases within the lysosome degrade the autophagosome and its contents into their molecular components, just as they do when the contents of multivesicular endosomes are delivered to the lysosome. Amino acid permeases in the lysosomal membrane then allow for the transport of free amino acids back into the cytosol for use in synthesis of new proteins.

By studying mutants with defects in the autophagic pathway, scientists have identified processes other than recycling of cellular components during starvation that also depend on autophagy. Experiments carried out principally in Drosophila and mice have shown that autophagy participates in a type of quality-control mechanism that removes organelles that have ceased to function properly. In particular, the autophagic pathway can target dysfunctional mitochondria that have lost their integrity and no longer have an electrochemical gradient across their inner membrane. In certain cell types, pathogenic bacteria and viruses that are multiplying in the cytosol of host cells can be targeted to the autophagic pathway for destruction in the lysosome as part of a host defense mechanism against infection.

In each of these processes, and in all eukaryotic organisms, the autophagic pathway takes place in three basic steps. Although the mechanisms underlying each of these steps are relatively poorly understood, they are thought to be related to the basic mechanisms for vesicular trafficking discussed in this chapter.

![li2074542480e](li2074542480e.jpg)

FIGURE 14-34 Mechanism for budding of HIV from the plasma membrane. Proteins required for the formation of multivesicular endosomes are exploited by HIV for virus budding from the plasma membrane. (a) Budding of HIV particles from HIV-infected cells occurs by a mechanism similar to that shown in Figure 14-33, using the virally encoded Gag protein and cellular ESCRT and Vps4 proteins (steps ■-■). Ubiquitinylated Gag near a budding particle functions like Hrs. See text for discussion. (b) In wild-type cells infected with HIV, virus particles bud from the plasma membrane and are rapidly released into the extracellular space. (c) In cells that lack the functional ESCRT protein Tsg101, the viral Gag protein forms dense viruslike structures, but budding of these structures from the plasma membrane cannot be completed, and chains of incomplete viral buds still attached to the plasma membrane accumulate. [Wes Sundquist, University of Utah.]

Autophagosome Nucleation The autophagosome is thought to originate from a fragment of a membranebounded organelle. The origin of this membrane has been difficult to trace because no known integral membrane proteins, which might serve to identify the source of this membrane, are known to be required for the formation of the autophagosome. Studies in yeast have shown that some mutants defective in Golgi trafficking are also defective in autophagy, suggesting that the autophagosome is initially derived from a fragment of the Golgi. Autophagy that is induced by starvation appears to be a nonspecific process in which a random portion of the cytoplasm, including organelles, becomes enveloped by an autophagosome. In these cases, the site of nucleation is probably random. In cases in which defective organelles are enveloped by the autophagosome, some type of signal or binding site must be present on the surface of the organelle to target nucleation of the autophagosome.

Autophagosome Growth and Completion New membrane must be delivered to the autophagosome membrane in order for this cup-shaped organelle to grow. This growth probably
occurs by the fusion of transport vesicles with the membrane of the autophagosome. About 30 proteins that participate in the formation of autophagosomes have been identified in genetic screens for yeast mutants that are defective in autophagy. One of these proteins is Atg8, shown in Figure 14-35, which is covalently linked to the lipid phosphatidylethanolamine and thus becomes attached to the cytoplasmic face of the autophagosome. Association of Atg8 with a membrane vesicle appears to be the key step in enabling a vesicle to fuse with the growing autophagosome.

Fusion of Atg8-containing vesicles with the autophagosome involves the formation of a cytosolic assembly of Atg12, Atg5, and Atg16. Atg12 is similar in structure to ubiquitin, and a set of proteins related to ubiquitin-conjugating enzymes are responsible for covalently joining Atg12 to Atg5 by a process similar to that used for covalently joining ubiquitin to a target protein (see Figure 3-31). The covalently linked Atg12-Atg5 dimer then co-assembles with Atg16 to form a polymeric complex localized to the site of a growing autophagosome. By an unknown mechanism, this cytosolic complex is thought to bring about the fusion of Atg8-containing vesicles into a cup-shaped autophagosome.

![li2084542480e](li2084542480e.jpg)

FIGURE 14-35 The autophagic pathway. The autophagic pathway allows cytosolic proteins and organelles to be delivered to the lysosome interior for degradation. In the autophagic pathway, a cup-shaped structure forms around a portion of the cytosol (right) or an organelle such as a mitochondrion (left). Continued addition of membrane eventually leads to the formation of an autophagosome that envelops its contents in two complete membranes (step 1). Fusion of the outer membrane with the membrane of a

Autophagosome Targeting and Fusion The outer membrane of the completed autophagosome is thought to contain a set of proteins that target it for fusion with the membrane of a lysosome. Two vesicle-tethering proteins have been found to be required for autophagosome fusion with a lysosome, but the corresponding SNARE proteins have not been identified. Fusion of the autophagosome with the lysosome occurs after Atg8 has been released from the autophagosome membrane by proteolytic cleavage, and this proteolysis step occurs only after the autophagosome has completely formed a sealed double-membrane system. Thus Atg8 protein appears to mask fusion proteins and to prevent premature fusion of the autophagosome with the lysosome.

## KEY CONCEPTS OF SECTION 14.6

## Directing Membrane Proteins and Cytosolic Materials to the Lysosome

- Endocytosed membrane proteins destined for degradation in the lysosome are incorporated into vesicles that bud into the interior of the endosome. Multivesicular endosomes, which contain many of these internal vesicles, can fuse with the lysosome to deliver the vesicles to the interior of the lysosome (see Figure 14-32).
lysosome releases a single-membrane vesicle and its contents into the lysosome interior (step 1). After degradation of the protein and lipid components by hydrolases in the lysosome interior, the released amino acids are transported across the lysosomal membrane into the cytosol. Proteins known to participate in the autophagic pathway include Atg8, which forms a coat structure around the autophagosome.
- Some of the cellular components (e.g., ESCRT) that mediate inward budding of endosomal membranes are used in the budding and pinching off of enveloped viruses such as HIV from the plasma membrane of virus-infected cells (see Figures 14-33 and 14-34).
- A portion of the cytoplasm or an entire organelle (e.g., a mitochondrion) can be enveloped in a flattened membrane and eventually incorporated into a double-membrane autophagosome. Fusion of the outer vesicle membrane with the lysosome delivers the enveloped contents to the interior of the lysosome for degradation (see Figure 14-35).


## 2 LaunchPad

macmillan learning
Visit LaunchPad to access study tools and to learn more about the content in this chapter.

- Perspectives for the Future
- Classic Experiment 14-1: Following a Protein Out of the Cell
- Analyze the Data
- Additional study tools, including videos, animations, and quizzes

## Key Terms

| adapter protein (AP) complexes 651 | mannose 6-phosphate (M6P) 653 |
| :--: | :--: |
| anterograde 632 | multivesicular |
| ARF protein 639 | endosomes 665 |
| autophagy 667 | Rab proteins 641 |
| cisternal maturation 632 | receptor-mediated endocytosis 659 |
| clathrin 638 | regulated secretion 632 |
| constitutive | retrograde 632 |
| secretion 632 | secretion (sec) mutants 636 |
| COPI 638 | secretory pathway 631 |
| COPII 638 | sorting signals 641 |
| dynamin 652 | transcytosis 658 |
| endocytic pathway 632 | trans-Golgi network 632 |
| ESCRT proteins 666 | transport vesicles 632 |
| late endosome 632 | t-SNAREs 638 |
| low-density lipoprotein | v-SNAREs 638 |
| (LDL) 659 |  |

## Review the Concepts

1. The studies of Palade and colleagues used pulse-chase labeling with radioactively labeled amino acids and autoradiography to visualize the location of newly synthesized proteins in pancreatic acinar cells. These early experiments provided invaluable information on protein synthesis and intercompartmental transport. New methods have replaced these early approaches, but two basic requirements are still necessary for any assay to study this type of protein transport. What are they, and how do recent experimental approaches meet these criteria?
2. Vesicle budding is associated with coat proteins. What is the role of coat proteins in vesicle budding? How are coat proteins recruited to membranes? What kinds of molecules are likely to be included or excluded from newly formed vesicles? What is the best-known example of a protein likely to be involved in the pinching off of vesicles?
3. Treatment of cells with the drug brefeldin A (BFA) has the effect of uncoating Golgi membranes, resulting in a cell in which the vast majority of Golgi proteins are found in the ER. What inferences can be made from this observation regarding roles of coat proteins other than promoting vesicle formation? Predict what type of mutation in ARF might have the same effect as treating cells with BFA.
4. Microinjection of an antibody known as EAGE, which reacts with the "hinge" region of the $\beta$ subunit of COPI, causes accumulation of Golgi enzymes in transport vesicles and inhibits anterograde transport of newly synthesized vesicles from the ER to the plasma membrane. What effect does the antibody have on COPI activity? Explain the results. 5. Specificity in fusion between vesicles involves two discrete and sequential processes. Describe the first of the two processes and its regulation by GTPase switch proteins. What effect on the size of early endosomes might result from overexpression of a mutant form of Rab5 that is stuck in the GTP-bound state?
5. Sec18 is a yeast gene that encodes NSF. Mutations of this gene produce class C mutants. What is the mechanistic role of NSF in membrane trafficking? As indicated by its class C phenotype, why does an NSF mutation produce accumulation of vesicles at what appears to be only one stage of the secretory pathway?
6. What feature of procollagen synthesis provided early evidence for the Golgi cisternal maturation model?
7. Sorting signals that cause retrograde transport of a protein in the secretory pathway are sometimes known as retrieval sequences. List the two known examples of retrieval sequences for soluble and membrane proteins of the ER. How does the presence of a retrieval sequence on a soluble ER protein result in its retrieval from the cis-Golgi complex? Describe how the concept of a retrieval sequence is essential to the cisternal-maturation model.
8. Clathrin adapter protein (AP) complexes bind directly to the cytosolic face of membrane proteins and also interact with clathrin. What are the four known adapter protein complexes? What observation regarding AP3 suggests that clathrin is an accessory protein to a core coat composed of adapter proteins?
9. I-cell disease is a classic example of an inherited human defect in protein targeting that affects an entire class of proteins: the soluble enzymes of the lysosome. What is the molecular defect in I-cell disease? Why does it affect the targeting of an entire class of proteins? What other types of mutations might produce the same phenotype?
10. The trans-Golgi network is the site of multiple sorting processes as proteins and lipids exit the Golgi complex. Compare and contrast the sorting of proteins to lysosomes with the packaging of proteins into regulated secretory vesicles such as those containing insulin. Compare and contrast the sorting of proteins to the basolateral versus apical cell surfaces in MDCK cells and in hepatocytes.
11. What does the budding of influenza virus and vesicular stomatitis virus (VSV) from polarized MDCK cells reveal about the sorting of newly synthesized plasma membrane proteins to the apical or basolateral domains? Now consider the following result: a peptide with a sequence identical to that of the VSV G protein cytoplasmic domain inhibits targeting of the G protein to the basolateral surface and has no effect on HA targeting to the apical membrane, but a peptide

in which the single tyrosine residue is mutated to an alanine has no effect on G protein basolateral targeting. What does this tell you about the sorting process?
13. Describe the role of pH in regulating the interaction between mannose 6-phosphate and the M6P receptor. Why does a rise in endosomal pH lead to the secretion of newly synthesized lysosomal enzymes into the extracellular medium? 14. What mechanistic features are shared by (a) the formation of multivesicular endosomes by budding into the interior of an endosome and (b) the outward budding of HIV virus at the cell surface? You wish to design a peptide inhibitor/competitor of HIV budding and decide to mimic a portion of the HIV Gag protein in a synthetic peptide. Which portion of the HIV Gag protein would be a logical choice? What normal cellular process might this inhibitor block? 15. The phagocytic and autophagic pathways serve two fundamentally different roles, but both deliver their vesicles to the lysosome. What are the fundamental differences between the two pathways? Describe the three basic steps in the formation and fusion of autophagosomes.
16. Compare and contrast the location and pH sensitivity of receptor-ligand interaction in the LDL and transferrin receptor-mediated endocytosis pathways.
17. What do mutations in the cytoplasmic domain of the LDL receptor that cause familial hypercholesterolemia reveal about the receptor-mediated endocytosis pathway?

## References

## Techniques for Studying the Secretory Pathway

Beckers, C. J., et al. 1987. Semi-intact cells permeable to macromolecules: use in reconstitution of protein transport from the endoplasmic reticulum to the Golgi complex. Cell 50:523-534.

Kaiser, C. A., and R. Schekman. 1990. Distinct sets of SEC genes govern transport vesicle formation and fusion early in the secretory pathway. Cell 61:723-733.

Lippincott-Schwartz, J., et al. 2001. Studying protein dynamics in living cells. Nat. Rev. Mol. Cell Biol. 2:444-456.

Novick, P., et al. 1981. Order of events in the yeast secretory pathway. Cell 25:461-469.

Orci, L., et al. 1989. Dissection of a single round of vesicular transport: sequential intermediates for intercisternal movement in the Golgi stack. Cell 56:357-368.

Palade, G. 1975. Intracellular aspects of the process of protein synthesis. Science 189:347-358.

## Molecular Mechanisms of Vesicle Budding and Fusion

Bonifacino, J. S., and B. S. Glick. 2004. The mechanisms of vesicle budding and fusion. Cell 116:153-166.

Hutagalung, A. H., and P. J. Novick. 2011. Role of Rab GTPases in membrane traffic and cell physiology. Physiol. Rev. 91:119-149.

Jahn, R., et al. 2003. Membrane fusion. Cell 112:519-533.

Kirchhausen, T. 2000. Three ways to make a vesicle. Nat. Rev. Mol. Cell Biol. 1:187-198.

McNew, J. A., et al. 2000. Compartmental specificity of cellular membrane fusion encoded in SNARE proteins. Nature 407:153-159.

Ostermann, J., et al. 1993. Stepwise assembly of functionally active transport vesicles. Cell 75:1015-1025.

Schimmöller, F., I. Simon, and S. Pfeffer. 1998. Rab GTPases, directors of vesicle docking. J. Biol. Chem. 273:22161-22164.

Weber, T., et al. 1998. SNAREpins: minimal machinery for membrane fusion. Cell 92:759-772.

Wickner, W., and A. Haas. 2000. Yeast homotypic vacuole fusion: a window on organelle trafficking mechanisms. Anпu. Rev. Biochem. 69:247-275.

Zerial, M., and H. McBride. 2001. Rab proteins as membrane organizers. Nat. Rev. Mol. Cell Biol. 2:107-117.

## Early Stages of the Secretory Pathway

Behnia, R., and S. Munro. 2005. Organelle identity and the signposts for membrane traffic. Nature 438:597-604.

Bi, X., et al. 2002. Structure of the Sec23/24-Sar1 pre-budding complex of the COPII vesicle coat. Nature 419:271-277.

Glick, B. S., and A. Nakano. 2009. Membrane traffic within the Golgi apparatus. Anпu. Rev. Cell Dev. Biol. 25:113-132.

Gurkan, C., et al. 2006. The COPII cage: unifying principles of vesicle coat assembly. Nat. Rev. Mol. Cell Biol. 7:727-738.

Lee, M. C., et al. 2004. Bi-directional protein transport between the ER and Golgi. Anпu. Rev. Cell Dev. Biol. 20:87-123. batsnyd

Letourneur, F., et al. 1994. Coatomer is essential for retrieval of dilysine-tagged proteins to the endoplasmic reticulum. Cell 79:1199-1207.

Lord, C., S. Ferro-Novick, and E. A. Miller. 2013. The highly conserved COPII coat complex sorts cargo from the endoplasmic reticulum and targets it to the Golgi. Cold Spring Harb. Perspect. Biol. 5:a013367.

Losev, E., et al. 2006. Golgi maturation visualized in living yeast. Nature 441:1002-1006.

Pelham, H. R. 1995. Sorting and retrieval between the endoplasmic reticulum and Golgi apparatus. Curr. Opin. Cell Biol. 7:530-535.

## Later Stages of the Secretory Pathway

Bonifacino, J. S. 2004. The GGA proteins: adaptors on the move. Nat. Rev. Mol. Cell Biol. 5:23-32.

Bonifacino, J. S., and E. C. Dell'Angelica. 1999. Molecular bases for the recognition of tyrosine-based sorting signals. J. Cell Biol. 145:923-926.

Edeling, M. A., C. Smith, and D. Owen. 2006. Life of a clathrin coat: insights from clathrin and AP structures. Nat. Rev. Mol. Cell Biol. 7:32-44.

Fotin, A., et al. 2004. Molecular model for a complete clathrin lattice from electron cryomicroscopy. Nature 432:573-579.

Ghosh, P., et al. 2003. Mannose 6-phosphate receptors: new twists in the tale. Nat. Rev. Mol. Cell Biol. 4:202-213.

Mellman, I., and W. J. Nelson. 2008. Coordinated protein sorting, targeting and distribution in polarized cells. Nat. Rev. Mol. Cell Biol. 9:833-845.

Schmid, S. 1997. Clathrin-coated vesicle formation and protein sorting: an integrated process. Anпu. Rev. Biochem. 66:511-548.

Simons, K., and E. Ikonen. 1997. Functional rafts in cell membranes. Nature 387:569-572.

Steiner, D. F., et al. 1996. The role of prohormone convertases in insulin biosynthesis: evidence for inherited defects in their action in man and experimental animals. Diabetes Metab. 22:94-104.

Tooze, S. A., et al. 2001. Secretory granule biogenesis: rafting to the SNARE. Trends Cell Biol. 11:116-122.

## Receptor-Mediated Endocytosis

Brown, M. S., and J. L. Goldstein. 1986. Receptor-mediated pathway for cholesterol homeostasis. Science 232:34-47. (Nobel Prize lecture.)

Kaksonen, M., C. P. Toret, and D. G. Drubin. 2006. Harnessing actin dynamics for clathrin-mediated endocytosis. Nat. Rev. Mol. Cell Biol. 7:404-414.

Rudenko, G., et al. 2002. Structure of the LDL receptor extracellular domain at endosomal pH. Science 298:2353-2358.

## Directing Membrane Proteins and Cytosolic Materials to the Lysosome

Geng, J., and D. J. Klionsky. 2008. The Atg8 and Atg12 ubiquitin-like conjugation systems in macroautophagy. EMBO Rep. 9:859-864.

Henne, W. M., N. J. Buchkovich, and S. D. Emr. 2011. The ESCRT pathway. Dev. Cell 21:77-91.

Katzmann, D. J., et al. 2002. Receptor downregulation and multivesicular-body sorting. Nat. Rev. Mol. Cell Biol. 3:893-905.

Lemmon, S. K., and L. M. Traub. 2000. Sorting in the endosomal system in yeast and animal cells. Curr. Opin. Cell Biol. 12:457-466.

Shintani, T., and D. J. Klionsky. 2004. Autophagy in health and disease: a double-edged sword. Science 306:990-995.

Sundquist, W. I., and H.-G. Kräusslich. 2012. HIV-1 assembly, budding, and maturation. Cold Spring Harb. Perspect. Med. 2:a006924.

![li2094542480e](li2094542480e.jpg)

Structure of a cell surface G protein-coupled receptor (green) bound to $\beta$-arrestin (purple). G protein-coupled receptors that are in the active state for a long period of time become phosphorylated, triggering binding of an arrestin and inhibition of further signaling by the receptor. [Data from Y. Kang et al., 2015, Nature 523:561-567, PDB ID 4zwj, and custom PDB.]

No cell lives in isolation; life requires that all cells sense chemicals and physical stimuli in their environment and respond with changes that can affect their function or development. Many cells sense and respond to light: many single-celled algae are phototactic (see Chapter 1), and later in this chapter, we will see how cells in the human retina sense light and respond by sending signals to the brain. Other cells sense and respond to physical stimuli such as touch or heat.

Most signals are chemical molecules; many, such as odorants, tastants (substances that can be tasted), and even oxygen are in the environment of virtually every metazoan cell and organism. Many types of signaling molecules are released by one cell and induce a response in a different cell; this fundamental process, known as cellular communication, shapes the development and function of every living organism. Even single-celled eukaryotic microorganisms, such as yeasts, algae, slime molds, and protozoans, communicate through extracellular signals. In the next chapter,

## OUTLINE

### 15.1 Signal Transduction: From Extracellular Signal to Cellular Response

15.2 Studying Cell-Surface Receptors and Signal Transduction Proteins

### 15.3 G Protein-Coupled Receptors: Structure and Mechanism

we will see how secreted molecules called pheromones coordinate the aggregation of free-living yeast cells for sexual mating.

More important in multicellular plants and animals are hormones and other extracellular signaling molecules that function within an organism to control a variety of processes, including the metabolism of sugars, fats, and amino acids; the growth and differentiation of tissues; the synthesis and secretion of small-molecule hormones and many proteins; and the composition of intracellular and extracellular fluids. Many types of metazoan cells also respond to signals from the external environment, including light, oxygen, odorants, and tastants in food.

In any system, for a signal to have an effect on a target, it has to be received. In cells, a signal produces a specific response only in target cells with receptor proteins that bind that signal. For some receptors, this signal is light, touch, or heat. Many types of chemicals act as signals: small molecules

### 15.4 G Protein-Coupled Receptors That Regulate Ion Channels

### 15.5 G Protein-Coupled Receptors That Activate or Inhibit Adenylyl Cyclase

15.6 G Protein-Coupled Receptors That Trigger Elevations in Cytosolic and Mitochondrial Calcium

(e.g., amino acid and lipid derivatives, steroids, acetylcholine), gases (e.g., oxygen, nitric oxide), peptides (e.g., adrenocorticotropic hormone and vasopressin), soluble proteins (e.g., insulin and growth hormone), and proteins that are tethered to the surface of a cell or bound to the extracellular matrix. Many of these extracellular signaling molecules are synthesized, packaged into secretory vesicles, and released by specialized signaling cells within multicellular organisms. Like enzymes, receptors bind a single type of molecule or a group of closely related molecules; unlike enzymes, receptors do not catalyze a chemical transformation of the bound molecule.

One large class of hydrophobic signaling molecules, primarily molecules such as steroids, retinoids, and thyroxine, spontaneously diffuse through the plasma membrane and bind to receptors in the cytosol (Figure 15-1). In most cases, as we saw in Chapter 9, the receptor-hormone complex moves into the nucleus, binds to specific regulatory sequences in DNA, and activates or represses expression of specific target genes.

In this chapter and the next, we focus on extracellular signaling molecules that are too large and too hydrophilic to diffuse through the plasma membrane. How, then, can they affect intracellular processes? These signaling molecules bind to cell-surface receptors that are integral membrane proteins embedded in the plasma membrane. Cell-surface receptors generally consist of three discrete topological domains, or segments: an extracellular domain facing the extracellular fluid, a plasma-membrane-spanning (transmembrane) domain, and an intracellular domain facing the cytosol. The signaling molecule acts as a ligand, which binds to a structurally complementary site on the extracellular or the membrane-spanning domain of the receptor. Binding of the ligand to its site on the receptor induces a conformational change in the receptor that is transmitted through the membrane-spanning domain to the cytosolic domain. This conformational change can result in the receptor's binding to, and subsequent activation or inhibition of, other proteins in the cytosol or attached to the plasma membrane. In many cases, these activated proteins catalyze the synthesis of specific small molecules or change the concentration of an intracellular ion such as $\mathrm{Ca}^{2+}$. These intracellular small-molecule second messengers then carry the signal to one or more effector proteins, such as enzymes or transcription factors.

Many signaling proteins, including the GTP-binding "switch" proteins and protein kinases introduced in Chapter 3, are members of large classes of signal transduction proteins that have been highly conserved throughout evolution. The overall process of converting extracellular signals into intracellular responses, as well as the individual steps in this process, is termed signal transduction; the chain of intermediates is called a signal transduction pathway because it transduces, or converts, information from one form into another as a signal is relayed from a receptor to its targets. Some signal transduction pathways contain just two or three intermediates; others can involve over a dozen.

As shown in Figure 15-1, signal transduction, which begins when extracellular signaling molecules bind to cellsurface receptors, can induce two major types of cellular

Hydrophobic signals
Steroids
Retinoids
Thyroxine
![li2104542480e](li2104542480e.jpg)

FIGURE 15-1 Overview of cell signaling. Hydrophobic signaling molecules, such as steroids and related molecules, diffuse through the plasma membrane (step $\mathbf{\square}$ ) and bind to receptors in the cytosol (step $\mathbf{\square}$. The receptor-signal complex moves into the nucleus (step $\mathbf{\square}$ ), where it can bind to transcription-control regions in DNA and activate or repress gene expression. The majority of signaling molecules, including small molecules (adrenaline, acetylcholine), peptides (yeast mating factors, glucagon), and proteins (insulin, growth hormone), are hydrophilic and cannot diffuse across the cell membrane. These molecules bind to specific cell-surface receptor proteins, triggering a conformational change in the receptor, thus activating it (step $\mathbf{\square}$ ). The activated receptor then activates one or more downstream signal transduction proteins or small-molecule second messengers (step $\mathbf{\square}$ ), which eventually leads to activation of one or more effector proteins (step $\mathbf{\square}$ ). The end result of a signaling cascade can be modification of specific cytosolic proteins, predominantly enzymes, leading to short-term changes in cellular function, metabolism, or movement (step $\mathbf{\square}$ ). Alternatively, an effector can move into the nucleus, triggering a long-term change in gene expression (step $\mathbf{\square}$ ). Termination or down-modulation of the cellular response is often caused by negative feedback from intracellular signaling molecules (step $\mathbf{\square}$ ) and by removal of the extracellular signal (step $\mathbf{\square}$ ).
responses. The first (step $\mathbf{\square}$ ) involves short-term (seconds to minutes) changes in the activity or function of specific enzymes and other proteins that preexist in the cell, often by covalent modifications such as phosphorylation or ubiquitinylation or by binding of molecules such as cAMP or $\mathrm{Ca}^{2+}$. The second (step $\mathbf{\square}$ ) involves activation (or repression) of specific transcription factors, often by phosphorylation or other covalent modifications, leading to long-term (hours to

days) changes in the amounts of specific proteins contained in a cell.

In eukaryotes, there are about a dozen classes of cellsurface receptors, which activate several types of intracellular signal transduction pathways. Our knowledge of signal transduction has advanced greatly in recent years, in large measure because these receptors and pathways are highly conserved and function in essentially the same way in organisms as diverse as worms, flies, mice, and humans. Genetic studies combined with biochemical analyses have enabled researchers to trace many entire signaling pathways from binding of ligand to final cellular responses. In this chapter, we first review some general principles of signal transduction, such as the molecular basis for ligand-receptor binding, and certain evolutionarily conserved components of signal transduction pathways. Next we describe how cell-surface receptors and signal transduction proteins are identified and characterized biochemically.

We then turn to an in-depth discussion of a very large and evolutionarily conserved class of receptors, found in organisms from fungi to humans: the G protein-coupled receptors. As their name implies, G protein-coupled receptors (GPCRs) consist of an integral membrane receptor protein coupled to an intracellular G protein that transmits signals to the interior of the cell. The human genome encodes more than 800 G proteincoupled receptors, which constitute about 4 percent of the identified human proteins. Most bind specific small molecules or peptides, but a few bind proteins. GPCRs include receptors in the visual, olfactory (smell), and gustatory (taste) systems, many neurotransmitter receptors, and most of the receptors for hormones that control carbohydrate, amino acid, and fat metabolism and even behavior. Signal transduction through GPCRs initially induces short-term changes in cell function, such as a change in metabolism or movement, but many of these signal transduction pathways also lead to changes in gene expression. We show how these pathways affect many aspects of cell function, including glucose metabolism, muscle contraction, perception of light, and gene expression.

Other large classes of cell-surface receptors mainly bind protein ligands. Activation of these receptors primarily alters a cell's pattern of gene expression, leading to cell differentiation or division and other long-term consequences, but can also induce short-term effects on cells. These receptors and the intracellular signaling pathways they activate are explored in Chapter 16. In Chapter 22, we discuss other classes of receptors, mainly confined to the nervous system, that are coupled to ion channels.

### 15.1 Signal Transduction: From Extracellular Signal to Cellular Response

In this section, we provide an overview of the major steps in signal transduction, starting with the signaling molecules themselves. We explore the molecular basis for ligand-receptor binding and the chain of events initiated in the target cell by binding of the signal to its receptor, focusing on a few components that are central to many signal transduction pathways.

## Signaling Molecules Can Act Locally or at a Distance

As noted above, cells respond to many different types of signals-some originating from outside the organism, some internally generated. Those that are generated internally can be described by how they reach their target. Some signaling molecules are transported long distances by the blood; others have more local effects. In animals, signaling by extracellular molecules can be classified into three types based on the distance over which the signal acts (Figure 15-2a-c).
(a) Endocrine signaling
![li2114542480e](li2114542480e.jpg)
(b) Paracrine signaling
![li2124542480e](li2124542480e.jpg)
(c) Autocrine signaling
![li2134542480e](li2134542480e.jpg)
(d) Signaling by plasma-membrane-attached proteins
![li2144542480e](li2144542480e.jpg)

FIGURE 15-2 Types of extracellular signaling. (a-c) Cell-to-cell signaling by extracellular chemicals occurs over distances from a few micrometers in autocrine and paracrine signaling to several meters in endocrine signaling. (d) Proteins attached to the plasma membrane of one cell can interact directly with cell-surface receptors on adjacent cells.

In endocrine signaling, the signaling molecules are synthesized and secreted by signaling cells (for example, those found in endocrine glands), transported through the circulatory system of the organism, and finally act on target cells distant from their site of synthesis. The term hormone generally refers to signaling molecules that mediate endocrine signaling. Insulin secreted by the pancreas and epinephrine secreted by the adrenal glands are examples of hormones that travel through the blood and thus mediate endocrine signaling.

In paracrine signaling, the signaling molecules released by a cell affect only those target cells in close proximity. A neuron releasing a neurotransmitter (e.g., acetylcholine) that acts on an adjacent neuron or on a muscle cell (inducing or inhibiting muscle contraction) is an example of paracrine signaling. In addition to neurotransmitters, many of the protein growth factors that regulate development in multicellular organisms act at short range. Some of these proteins bind tightly to components of the extracellular matrix and are unable to signal to adjacent cells; subsequent degradation of these matrix components, triggered by injury or infection, releases these growth factors and enables them to signal. Many of the developmentally important signaling proteins that we discuss in Chapter 16 diffuse away from the signaling cell, forming a concentration gradient and inducing different responses in adjacent cells depending on the concentration of the signaling protein.

In autocrine signaling, cells respond to substances that they themselves release. Some growth factors act in this fashion, and cultured cells often secrete growth factors that stimulate their own growth and proliferation. This type of signaling is particularly characteristic of tumor cells, many of which overproduce and release growth factors that stimulate inappropriate, unregulated self-proliferation, a process that may lead to formation of a tumor.

Many integral membrane proteins located on the cell surface play important roles as signals (Figure 15-2d). In some cases, such membrane-bound signaling proteins on one cell directly bind receptors on the surface of an adjacent target cell, often triggering its proliferation or differentiation. In other cases, proteolytic cleavage of a membrane-bound signaling protein releases the extracellular segment, which functions as a soluble signaling molecule.

Some signaling molecules can act at both short and long ranges. For example, epinephrine (also known as adrenaline) functions as a hormone (endocrine signaling) as part of the "fight or flight" response to a sudden danger in the environment, and also as a neurotransmitter (paracrine signaling). Another example, discussed in Chapter 16, is epidermal growth factor (EGF), which is synthesized as an integral plasma-membrane protein. Membrane-bound EGF can bind to receptors on an adjacent cell. In addition, cleavage by an extracellular protease releases a soluble form of EGF, which can signal in either an autocrine or a paracrine manner.

## Receptors Bind Only a Single Type of Hormone or a Group of Closely Related Hormones

Receptor proteins for all hydrophilic extracellular smallmolecule, peptide, and protein signaling molecules are located
on the surface of the target cell. The signaling molecule, or ligand, binds to a site on the extracellular domain of the receptor with high specificity and affinity. Ligand binding depends on multiple weak, noncovalent forces (i.e., ionic, van der Waals, and hydrophobic interactions) and molecular complementarity between the interacting surfaces of a receptor and ligand (see Figure 2-12). Like an enzyme, each type of receptor binds only a single type of signaling molecule or a group of very closely related ones. For example, the growth hormone receptor binds to growth hormone, but not to other hormones with very similar, though not identical, structures. Similarly, acetylcholine receptors bind only this small molecule and not others that differ from it only slightly in chemical structure, while the insulin receptor binds insulin and related hormones called insulin-like growth factors 1 and 2 (IGF-1 and IGF-2), but no other hormones. The binding specificity of a receptor refers to its ability to bind or not bind closely related substances.

Binding of ligand to receptor causes a conformational change in the receptor that initiates a sequence of reactions leading to a specific response inside the cell. Organisms have evolved to be able to use a single ligand to stimulate different cells to respond in distinct ways. Different cell types often have different receptors for the same ligand, and activation of each receptor type induces a different intracellular signal transduction pathway. For instance, the surfaces of skeletal muscle cells, heart muscle cells, and the pancreatic acinar cells that produce hydrolytic digestive enzymes each have different types of receptors for acetylcholine. In a skeletal muscle cell, release of acetylcholine from a motor neuron innervating the cell triggers muscle contraction by activating an acetylcholinegated ion channel. In heart muscle, the release of acetylcholine by certain neurons activates a G protein-coupled receptor and slows the rate of contraction and thus the heart rate. Acetylcholine stimulation of pancreatic acinar cells triggers a rise in the concentration of cytosolic $\mathrm{Ca}^{2+}$ that induces secretion of the digestive enzymes stored in secretory granules to facilitate digestion of a meal. Thus the activation by acetylcholine of different types of acetylcholine receptors that are expressed in different cell types leads to different cellular responses.

Alternatively, the same receptor may be found on various cell types in an organism, but binding of a particular ligand to the receptor triggers a different response in each type of cell, given the particular complement of proteins expressed by the cell. The same epinephrine receptor (the $\beta$-adrenergic receptor) is found on liver, muscle, and fat (adipose) cells; as we will see in Section 15.5, it stimulates depolymerization of glycogen to glucose in the first two cell types, but hydrolysis and secretion of stored fat in adipose cells. In these ways, the same ligand can induce different cells to respond in a variety of ways, often in a manner that coordinates the overall response of the organism. This property is known as the effector specificity of the receptor-ligand complex.

## Protein Kinases and Phosphatases Are Employed in Many Signaling Pathways

Activation of virtually all cell-surface receptors leads directly or indirectly to changes in protein phosphorylation through

the activation of protein kinases, enzymes that add phosphate groups to specific residues of target proteins. Some receptors activate protein phosphatases, which remove phosphate groups from specific residues on target proteins. Phosphatases act in concert with kinases to switch the function of various target proteins on or off (Figure 15-3).

The human genome encodes about 600 protein kinases and 100 different phosphatases. In general, each protein kinase phosphorylates specific amino acid residues in a specific set of target, or substrate, proteins whose patterns of expression generally differ in different cell types. Animal cells contain two types of protein kinases: those that add phosphate to the hydroxyl group on tyrosine residues (protein tyrosine kinases) and those that add phosphate to the hydroxyl group on serine or threonine (or both) residues (protein serine/threonine kinases). We will see in this chapter and the next that the catalytic subunits of all known protein kinases have similar three-dimensional structures, including an N-terminal and a C-terminal lobe; highly conserved amino acids cluster around the catalytic site and are essential for binding ATP. All kinases recognize their specific substrates by binding not only to the side chain to be phosphorylated, but also to specific amino acids that surround the phosphorylated residue. Thus one can analyze the amino acid sequences surrounding tyrosine, serine, and threonine residues in a protein and make a good prediction as to which kinases might phosphorylate those residues.

Many proteins are substrates for multiple kinases, each of which usually phosphorylates different amino acids in the protein. Each phosphorylation event has the potential to modify the activity of a particular target protein in different
![li2154542480e](li2154542480e.jpg)

FIGURE 15-3 Regulation of protein activity by a kinase/phosphatase switch. The cyclic phosphorylation and de-phosphorylation of a protein is a common cellular mechanism for regulating protein activity. In this example, the target, or substrate, protein is inactive (light green) when not phosphorylated and active (dark green) when phosphorylated; some proteins have the opposite pattern. Both the protein kinase and the phosphatase act only on specific amino acids in specific target proteins, and their activities are usually highly regulated.
![li2164542480e](li2164542480e.jpg)

FIGURE 15-4 GTPase switch proteins cycle between active and inactive forms. The switch protein is active when it has bound GTP and inactive when it has bound GDP. Conversion of the active into the inactive form by hydrolysis of the bound GTP is accelerated by GAPs (GTPase-activating proteins), RGSs (regulators of G protein signaling), and other types of proteins. Reactivation is promoted by GEFs (guanine nucleotide exchange factors), which catalyze the dissociation of the bound GDP and its replacement by GTP.
ways, some activating its function, others inhibiting it. An example we will encounter later is glycogen phosphorylase kinase, a key regulatory enzyme in glucose metabolism (see Figure 15-29 below). In many cases, addition of a phosphate group to an amino acid creates a binding surface that allows a second protein to bind; in the following chapter, we will encounter many examples of such kinase-driven assembly of multiprotein complexes.

Commonly the catalytic activity of a protein kinase itself is modulated by phosphorylation by other kinases, by the binding of other proteins to it, and by changes in the concentrations of various small intracellular signaling molecules and metabolites. Importantly, the activity of all protein kinases is opposed by the activity of protein phosphatases, some of which themselves are regulated by extracellular signals. Thus the activity of a protein in a cell can be a complex function of the activities of the usually multiple kinases and phosphatases that act on it. Several examples of this phenomenon that occur in regulation of the cell cycle are described in Chapter 19.

## GTP-Binding Proteins Are Frequently Used in Signal Transduction Pathways as On/Off Switches

Many cellular processes utilize members of the GTPase superfamily of proteins, which are found in all prokaryotic and eukaryotic cells. All of these GTP-binding switch proteins exist in two forms (Figure 15-4): (1) an active ("on") form with bound GTP (guanosine triphosphate), which modulates the activity of specific target proteins, and (2) an inactive ("off") form with bound GDP (guanosine diphosphate), which cannot affect the activity of target proteins. Members of the GTPase superfamily switch between GTPbound "on" and GDP-bound "off" forms. These proteins are evolutionarily ancient, as evidenced by their widespread functions in protein synthesis [examples include the roles of the eIF2 initiation factor (Figure 5-23) and the EF1 $\alpha$ and EF2

![li2174542480e](li2174542480e.jpg)

FIGURE 15-5 Switching mechanism of monomeric G proteins. The ability of a G protein to interact with other proteins and thus transduce a signal differs between the GTP-bound "on" state and GDP-bound "off" state. (a) In the active "on" state, two domains, termed switch I (green) and switch II (blue), are bound to the terminal $y$ phosphate of GTP through interactions with the backbone amide groups of conserved threonine and glycine residues. When bound to GTP in this way, the two switch domains are in a conformation such that they can bind to and thus activate specific downstream effector proteins. (b) Removal of the $y$ phosphate by GTPase-catalyzed hydrolysis causes switch I and switch II to relax into a different conformation, the inactive "off" state; in this state, they are unable to bind to effector proteins. The three-dimensional models shown here represent both conformations of Ras, a monomeric G protein. A similar spring-loaded mechanism switches the alpha subunit in heterotrimeric G proteins between the active and inactive conformations by movement of three, rather than two, switch segments. [Part (a) data from E. F. Pai et al., 1990, EMBO J. 9:2351-2359, PDB ID 5p21. Part (b) data from M. V. Milburn et al., 1990, Science 247:939-945, PDB ID 4q21.]
proteins in protein elongation (Figure 5-24); the transport of proteins between the nucleus and the cytoplasm (Ran, Figure 13-36); the formation of transport vesicles (Sar proteins, Figure 14-8) and their fusion with target membranes (Rab proteins, Figure 14-10); and rearrangements of the actin cytoskeleton (Rho, Rac, and Cdc42 proteins, discussed in Chapter 17)].

Here we focus on members of this superfamily that function in signal transduction pathways, in which conversion between the active GTP-bound and inactive GDP-bound states is tightly regulated. Conversion of the inactive to the active state is usually triggered by a signal (e.g., a hormone binding to a receptor) and is mediated by a guanine nucleotide exchange factor (GEF), which causes the release of GDP from the switch protein. Subsequent binding of GTP, favored by its high intracellular concentration relative to that of GDP, induces a conformational change to the active form. The principal conformational changes involve two highly conserved segments of the GTP-binding protein, termed switch I and switch II, that allow the protein to bind to and activate downstream signaling proteins (Figure 15-5). Conversion of the active form back to the inactive form is mediated by a GTPase, which is often part of the switch protein itself and which slowly hydrolyzes the bound GTP to GDP and $\mathrm{P}_{\mathrm{i}}$, thus altering the conformation of the switch I and switch II segments so that they are unable to bind to the target effector protein.

The rate of GTP hydrolysis regulates the length of time the switch protein remains in the active conformation and is able to signal its downstream target proteins: the slower the
rate of GTP hydrolysis, the longer the protein remains in the active state. The rate of GTP hydrolysis is often modulated by other proteins. For instance, both GTPase-activating proteins (GAPs) and regulators of G protein signaling (RGSs) accelerate GTP hydrolysis (see Figure 15-4). Many regulators of G protein activity are themselves controlled by extracellular signals.

Two large classes of GTPase switch proteins are used in signaling. Heterotrimeric G proteins directly bind to and are activated by certain cell-surface receptors. As we will see in Section 15.3, G protein-coupled receptors function as guanine nucleotide exchange factors (GEFs), activating the heterotrimeric G protein to which they are coupled by triggering its release of GDP and binding of GTP. Monomeric (often called low-molecular-weight) G proteins, including Ras and various Ras-like proteins such as Ran and Sar, do not directly bind to receptors, but play crucial roles in many pathways that regulate cell division and cell motility, as is evidenced by the fact that mutations in genes encoding these G proteins frequently lead to cancer.

## Intracellular "Second Messengers" Transmit Signals from Many Receptors

The binding of ligands ("first messengers") to many cellsurface receptors leads to a short-lived increase (or decrease) in the concentration of certain nonprotein, low-molecularweight intracellular signaling molecules termed second messengers. These molecules, in turn, bind to proteins, modifying their activity.

One second messenger used in virtually all metazoan cells is calcium $\left(\mathrm{Ca}^{2+}\right)$ ions. We noted in Chapter 11 that the concentration of free $\mathrm{Ca}^{2+}$ in the cytosol is kept very low $\left(\sim 10^{-7} \mathrm{M}\right)$ in part by ATP-powered pumps that continually transport $\mathrm{Ca}^{2+}$ out of the cell or into the endoplasmic reticulum (ER). The cytosolic $\mathrm{Ca}^{2+}$ concentration can be increased from tenfold to a hundredfold by a signal-induced release of $\mathrm{Ca}^{2+}$ from the ER lumen or the extracellular environment by the opening of calcium channels in the respective membranes; this change can be detected by fluorescent dyes introduced into the cell (see Figure 4-12). In muscle, a signal-induced rise in cytosolic $\mathrm{Ca}^{2+}$ triggers contraction (see Figure 17-34). In endocrine cells, a similar increase in $\mathrm{Ca}^{2+}$ induces exocytosis of secretory vesicles containing hormones, which are thus released into the circulation. In neurons, an increase in cytosolic $\mathrm{Ca}^{2+}$ leads to the exocytosis of neurotransmittercontaining vesicles (see Chapter 22). In all cells, such a rise in cytosolic $\mathrm{Ca}^{2+}$ is sensed by $\mathrm{Ca}^{2+}$-binding proteins, particularly those of the EF hand family, such as calmodulin, all of which contain the helix-loop-helix motif (see Figure 3-10b). The binding of $\mathrm{Ca}^{2+}$ to calmodulin and other EF hand proteins causes a conformational change that permits those proteins to bind various target proteins, thereby switching their activities on or off (see Figure 3-33). Often a rise in $\mathrm{Ca}^{2+}$ is localized to specific regions of the cytosol, allowing a process-such as exocytosis of a secretory vesicle-to occur there without affecting other processes elsewhere in the cell.

Another nearly universal second messenger is cyclic adenosine monophosphate (cAMP). In many eukaryotic cells, a rise in cAMP triggers the activation of a particular protein kinase, protein kinase $A$, that in turn phosphorylates specific target proteins to induce specific changes in cell metabolism. In some cells, cAMP regulates the activity of certain ion channels. The structures of cAMP and three other common second messengers are shown in Figure 15-6. Later in this chapter, we examine the specific roles of second messengers in signaling pathways activated by various G protein-coupled receptors.

## Signal Transduction Pathways Can Amplify the Effects of Extracellular Signals

Multiple signal transduction proteins are frequently combined to form a signal transduction pathway, allowing multiple target proteins to be activated (or inhibited) by a single type of cell-surface receptor. Figure 15-7 depicts a typical signal transduction pathway downstream of many G protein-coupled receptors. Here, binding of a hormone triggers a conformational change in the receptor, leading to activation of a G protein by catalyzing the exchange of GTP for GDP. The activated G protein binds to and activates an enzyme that synthesizes a second messenger; this small molecule binds and activates a protein kinase that, in turn, phosphorylates and thus changes the activity of one or more target proteins. One specific example of this multiprotein pathway-the regulation of glycogen metabolism in the liver by the hormone epinephrine-is detailed in Section 15.5, and the molecules involved in that pathway are listed in parentheses in Figure 15-7.

One important advantage of a cascade of proteins in a signal transduction pathway is that it facilitates amplification of an extracellular signal. Activation of a single cellsurface receptor protein can result in an increase of perhaps thousands of cAMP molecules or $\mathrm{Ca}^{2+}$ ions in the cytosol. Each of these molecules, in turn, by activating its target protein kinase or other signal transduction protein, can affect the activity of multiple downstream proteins. In many signal transduction pathways, amplification is necessary because cell-surface receptors are typically low-abundance proteins, present in only a thousand or so copies per cell, while the cellular responses induced by the binding of a relatively small number of hormones to the available receptors often require generation of tens or hundreds of thousands of activated effector molecules per cell. In the case of G protein-coupled hormone receptors, signal amplification is possible in part because a single receptor can activate multiple G proteins during the time a hormone remains bound, each of which
![li2184542480e](li2184542480e.jpg)

FIGURE 15-6 Four common intracellular second messengers.
The major direct effect or effects of each compound are indicated below
its structural formula. Calcium ions $\left(\mathrm{Ca}^{2+}\right)$ and several membranebound phosphatidylinositol derivatives also act as second messengers.

![li2194542480e](li2194542480e.jpg)

FIGURE 15-7 A signal transduction pathway involving a G protein, a second messenger, a protein kinase, and several target proteins. The figure depicts a generalized signal transduction pathway; in parentheses are listed the molecules involved in the specific signaling pathway discussed in Section 15-5. Binding of the hormone to its cell-surface receptor $\boldsymbol{\square}$ triggers activation of a G protein $\boldsymbol{\square}$ by the receptor functioning as a GEF and causing loss of GDP and binding of GTP. The active G protein binds to and activates an enzyme $\boldsymbol{\square}$ that synthesizes a second messenger $\boldsymbol{\square}$, which in turn binds to and activates a protein kinase $\boldsymbol{\square}$. The kinase, in turn, phosphorylates, and thus changes the activity of, one or more target proteins. These proteins can either be cytosolic proteins $\boldsymbol{\square}$ that induce changes in cellular function, metabolism, or movement or transcription factors $\boldsymbol{\square}$ that induce changes in gene expression.
in turn activates an effector protein. In Section 15.5, we see how this amplification cascade allows blood levels of epinephrine as low as $10^{-10} \mathrm{M}$ to stimulate conversion of glycogen to glucose by the liver and release of glucose into the blood, and in Figure 15-30 we will see how this amplification can regulate expression of many genes.

## KEY CONCEPTS OF SECTION 15.1

## Signal Transduction: From Extracellular Signal to Cellular Response

- Many cells sense and respond to light or physical stimuli such as touch or heat.
- All cells communicate through extracellular signals. In unicellular organisms, extracellular signaling molecules regulate interactions between individuals, while in multicellular organisms, they mainly regulate physiology and development.
- External signals include membrane-anchored and secreted proteins or peptides (e.g., vasopressin and insulin), small hydrophobic molecules (e.g., steroid hormones and thyroxine), small hydrophilic molecules (e.g., epinephrine), gases (e.g., $\mathrm{O}_{2}$, nitric oxide), and physical stimuli (e.g., light).
- Hydrophobic signaling molecules interact with cytosolic receptors and mainly affect gene expression.
- Binding of a hydrophilic extracellular signaling molecule to a cell-surface receptor triggers a conformational change in the receptor, which in turn leads to activation of intracellular signal transduction pathways that ultimately modulate cellular metabolism, function, or gene expression (see Figure 15-1).
- Signals from one cell act on distant cells in endocrine signaling, on nearby cells in paracrine signaling, or on the signaling cell itself in autocrine signaling (see Figure 15-2).
- Protein phosphorylation and de-phosphorylation, catalyzed by protein kinases and phosphatases, are employed in virtually all signaling pathways. The activities of kinases and phosphatases are highly regulated by many receptors and signal transduction proteins (see Figure 15-3).
- GTP-binding proteins of the GTPase superfamily act as switches regulating many signal transduction pathways (see Figures 15-4 and 15-5).
- Some nonprotein, low-molecular-weight intracellular molecules, such as $\mathrm{Ca}^{2+}$ and cAMP (see Figure 15-6), act as "second messengers," relaying and often amplifying the signal of the "first messenger"; that is, the ligand. Binding of ligand to cell-surface receptors often results in a rapid increase (or, occasionally, decrease) in the intracellular concentration of these ions or molecules.
- Signal transduction pathways allow amplification of an extracellular signal, enabling activation of a relatively small number of cell-surface receptors to trigger major changes in cell metabolism, movements, or gene expression.

### 15.2 Studying Cell-Surface Receptors and Signal Transduction Proteins

The response of a cell to an external signal depends on the cell's complement of receptors that recognize the signal as well as the signal transduction pathways activated by those receptors. In this section, we explore the biochemical basis for the specificity of receptor-ligand binding as well as the ability of different concentrations of ligand to activate a pathway. We also examine experimental techniques used to characterize receptor proteins. Many of these methods are also applicable to receptors that mediate endocytosis (see Chapter 14) or cell adhesion (see Chapter 20). We conclude the section with a discussion of techniques commonly used to measure the activity of signal transduction pathway components, such as kinases and GTP-binding "switch" proteins.

## The Dissociation Constant Is a Measure of the Affinity of a Receptor for Its Ligand

Binding of a single ligand to a receptor can usually can be viewed as a simple reversible reaction, where the receptor is represented as $R$, the ligand as $L$, and the receptor-ligand complex as RL:

$$
\mathrm{R}+\mathrm{L} \quad \frac{k_{\text {off }}}{k_{\text {on }}} \quad \mathrm{RL}
$$

where $k_{\text {on }}$ is the rate constant for formation of a receptorligand complex from free ligand and receptor and $k_{\text {off }}$ is the rate constant for dissociation of a ligand from its receptor. We define $[\mathrm{R}]$ and $[\mathrm{L}]$ as the concentrations of free receptor (that is, receptor without bound ligand) and ligand, respectively, and $[\mathrm{RL}]$ as the concentration of the receptor-ligand complex.

At equilibrium, the rate of formation of the receptorligand complex, $[\mathrm{R}][\mathrm{L}] k_{\text {on }}$, is equal to the rate of its dissociation, $[\mathrm{RL}] k_{\text {off }}$ : thus at equilibrium, $[\mathrm{R}][\mathrm{L}] k_{\text {on }}=[\mathrm{RL}] k_{\text {off }}$. This situation can be described by the simple equilibriumbinding equation $K_{\mathrm{d}}=k_{\text {off }} / k_{\text {on }}$, where $K_{\mathrm{d}}$, the dissociation constant, is a measure of the affinity (or tightness of binding) of the receptor for its ligand (see Chapter 2).

Equivalently, we can write this equilibrium equation as

$$
K_{d}=\frac{[\mathrm{R}][\mathrm{L}]}{[\mathrm{RL}]}
$$

The smaller the dissociation constant, the more stable the receptor-ligand complex. Another way of seeing this key point is that when the concentration of ligand equals $K_{\mathrm{d}}\left([\mathrm{L}]=\mathrm{K}_{\mathrm{d}}\right)$, then the concentration of free receptor [R] must equal the concentration of the receptor-ligand complex [RL]. That is, when the system is at equilibrium, half of the receptors have a ligand bound. The smaller the $K_{\mathrm{d}}$, the lower the ligand concentration required to bind 50 percent of the cell-surface receptors. The $K_{\mathrm{d}}$ for a binding reaction here is
similar to the Michaelis constant, $K_{\mathrm{m}}$, which reflects the affinity of an enzyme for its substrate (see Chapter 3).

Most hormone-receptor systems are finely balanced: too much or too little of a hormone can cause trouble. Consider, for example, the hormone tumor necrosis factor alpha (TNF $\alpha$ ), which is secreted by a number of immunesystem cells. TNF $\alpha$ induces inflammation by binding to TNF $\alpha$ receptors on several types of immune-system cells and recruiting them to a site of injury or infection; thus the body needs to make sufficient TNF $\alpha$ to protect against infections, but abnormally high levels of TNF $\alpha$ can cause the excessive inflammation seen in patients with autoimmune diseases such as the blistering skin disease psoriasis or the joint disease rheumatoid arthritis. These diseases are being treated by drugs that deplete the amount of TNF $\alpha$ in the joint or circulating in the body. One such drug is on the principle that hormone receptors are characterized by their high affinity and specificity for their ligands. This drug is a chimeric "fusion" protein, generated by recombinant DNA techniques, that contains the extracellular TNF $\alpha$-binding domain of a TNF $\alpha$ receptor fused to the constant ( Fc ) region of a human immunoglobulin (see Figures 3-21 and 23-9). The watersoluble drug can be injected into the body, where it binds tightly to the potentially dangerous free TNF $\alpha$ and prevents it from binding to its cell-surface receptors and causing inflammation; the fused Fc domain causes the protein to be stable when injected into the body.

## Binding Assays Are Used to Detect Receptors and Determine Their Affinity and Specificity for Ligands

Receptors are usually detected and quantified by their ability to bind radioactively or fluorescently labeled ligands that have been added to the fluid surrounding intact cells or to cell fragments. This assay is based on the concept that the total number of receptors $\left[\mathrm{R}_{\mathrm{T}}\right]$ is equal to the number of free receptors [R] plus the number of ligand-bound receptors $[\mathrm{RL}]$.

$$
\left[\mathrm{R}_{\mathrm{T}}\right]=[\mathrm{RL}]+[\mathrm{R}]
$$

The addition of increasing concentrations of ligand results in increasing amounts of cell-surface receptor-ligand complexes [RL]; as the concentration of ligand increases, the number of receptor-ligand complexes approaches, but never actually reaches, the total number of cell-surface receptors $\left[\mathrm{R}_{\mathrm{T}}\right]$. In most cases, the amount of ligand added is vastly in excess of the amount of cell-surface receptors, so one can assume that the concentration of free ligand $[\mathrm{L}]$ is equal to the concentration of ligand added to the reaction. Thus, in these reactions, one need only measure the amount of ligand bound to surface receptors $[\mathrm{RL}]$ at each concentration of ligand added.

A plot of the amount of ligand bound to the receptor [RL] versus the amount of free ligand [L] generally follows equation 15-4, which is simply an algebraic transformation

of equations 15-2 and 15-3. A typical ligandbinding curve can be seen in the red line in Figure 15-8.

$$
[\mathrm{RL}]=\frac{\mathrm{R}_{\mathrm{T}}}{\frac{\mathrm{~K}_{\mathrm{d}}}{[\mathrm{~L}]}+1}
$$

Computer curve-fitting programs are typically used to calculate the $\mathrm{R}_{\mathrm{T}}$ and $\mathrm{K}_{\mathrm{d}}$ values. Using this approach, one can calculate from this binding curve that there are 1000 receptors per cell surface and that the $\mathrm{K}_{\mathrm{d}}$ for binding this ligand is $1 \times 10^{-9} \mathrm{M}$, or 1 nM .
![li2204542480e](li2204542480e.jpg)

Concentration of ligand added to binding reaction (nM)

FIGURE 15-8 Binding assays determine the $K_{d}$ and the number of receptors per cell, but the maximal physiological response to an external signal usually occurs when only a fraction of the receptors are occupied by ligand. In a typical experiment to determine the affinity of a receptor for a ligand, radioactively or otherwise labeled ligand is incubated with cells that do not express the receptor of interest and with cells that have been altered by recombinant DNA techniques to express that receptor on their surface. Incubation is generally for an hour at $4^{\circ} \mathrm{C}$; the low temperature is used to prevent endocytosis of the cell-surface receptors. The cells are then separated from unbound ligand, usually by centrifugation and washing with buffer, and the amount of radioactivity bound to the cells is measured. "Background" binding by control cells is subtracted from the binding to the receptor-expressing cells, and the amount of bound ligand per cell is calculated and plotted (red curve) as a function of the ligand concentration. Note that even at relatively high ligand concentrations, the number of receptor-bound ligand molecules approaches, but does not equal, the number of cell-surface receptors. Nonetheless, by analysis of the data using equation 15-4, one can determine that these cells express 1000 receptors for this ligand, and that the $K_{d}$ for binding of the ligand is 1 nM . In parallel experiments, the physiological response of the cell to increasing concentrations of ligand is also measured (blue line). Typically, the plots of the extent of ligand binding to the receptor and of physiological response at different ligand concentrations differ. In the example shown here, 50 percent of the maximal physiological response is induced at a ligand concentration at which only 18 percent of the receptors are occupied. Likewise, 80 percent of the maximal response is induced when the ligand concentration equals the $K_{d}$ value, at which 50 percent of the receptors are occupied.

# Near-Maximal Cellular Response to a Signaling Molecule Usually Does Not Require Activation of All Receptors 

Signaling systems have evolved such that a rise in the level of extracellular signaling molecules induces a proportional response in the responding cell. For this to happen, the binding affinity ( $K_{d}$ value) of a cell-surface receptor for a signaling molecule must be large compared with the signaling molecule's normal unstimulated level in the extracellular fluid, or in blood. We can see this principle in practice by comparing the levels of insulin present in the body and the $K_{\mathrm{d}}$ for binding of insulin to its receptor on liver cells, $1.4 \times 10^{-10} \mathrm{M}(0.14 \mathrm{nM})$. Suppose, for instance, that the normal concentration of insulin in the blood is $5 \times 10^{-12}$ M. By substituting this value of L and the $K_{\mathrm{d}}$ into equation 15-2, we can calculate the fraction of insulin receptors with bound insulin, $[\mathrm{RL}] /\left[\mathrm{R}_{\mathrm{T}}\right]$, at equilibrium as 0.0344 ; that is, about 3 percent of the total insulin receptors will have insulin bound to them. If the insulin concentration rises fivefold to $2.5 \times 10^{-11} \mathrm{M}$, as it does after a meal, the number of receptor-hormone complexes will rise proportionately, almost fivefold, so that about 15 percent of the total receptors will have insulin bound to them. If the extent of the induced cellular response parallels the number of insulinreceptor complexes [RL], as is often the case, then the cellular response will also increase by about fivefold.

On the other hand, suppose that the normal concentration of insulin in the blood were the same as the $K_{\mathrm{d}}$ value of $1.4 \times 10^{-10} \mathrm{M}$; in that case, 50 percent of the total receptors would have insulin bound to them. A fivefold increase in the insulin concentration to $7 \times 10^{-10} \mathrm{M}$ would result in 83 percent of all insulin receptors having insulin bound to them (only a 66 percent increase). Thus, in order for a rise in hormone concentration to cause a proportional increase in the fraction of receptors with bound ligand, the $K_{d}$ value must be much larger than the normal concentration of the hormone.

In general, the maximal cellular response to a particular ligand is induced when much less than 100 percent of its receptors are bound to the ligand. This phenomenon can be revealed by comparing the extent of the response and of
receptor-ligand binding at different concentrations of ligand (see Figure 15-8). For example, a cell in the bone marrow (called an erythroid progenitor cell) has 1000 surface receptors for erythropoietin, the protein hormone that induces these cells to proliferate and differentiate into red blood cells; the $\mathrm{K}_{\mathrm{d}}$ for erythropoietin binding is 1 nM . But only 180 of these receptors ( 18 percent of them) need to bind Epo to induce 50 percent of the maximal cellular response. Thus the ligand concentration needed for a significant cellular response (i.e., the division of the progenitor cell in this case) is considerably lower than the $\mathrm{K}_{\mathrm{d}}$ value. In such cases, a plot of the percentage of maximal binding versus ligand concentration is different from a plot of the percentage of maximal cellular response versus ligand concentration.

## Sensitivity of a Cell to External Signals Is Determined by the Number of Cell-Surface Receptors and Their Affinity for Ligand

Because the cellular response to a particular signaling molecule depends on the number of receptor-ligand complexes, the fewer receptors for a ligand present on the surface of a cell, the less sensitive the cell is to that ligand. As a consequence, a higher ligand concentration is necessary to induce the physiological response than would be the case if more receptors were present. In contrast, if the level of a receptor for a particular ligand is increased, the cell will become more sensitive to the ligand.

Epidermal growth factor (EGF), as its name implies, stimulates the proliferation of many types of epithelial cells (see Chapters 16 and 20), including those that line the ducts of the mammary gland. In about 25 percent of breast cancers, the tumor cells have elevated levels of one particular EGF receptor called HER2. The overproduction of HER2 makes the cells hypersensitive to ambient levels of EGF and related hormones, which are normally too low to stimulate cell proliferation; as a consequence, growth of these tumor cells is inappropriately stimulated by EGF. We will see in Chapter 24 that an understanding of the role of HER2 in certain breast cancers led to the development of monoclonal antibodies that bind HER2 and thereby block signaling by EGF; these antibodies have proved useful in treatment of breast cancer patients whose tumors overexpress HER2.

The HER2-breast cancer connection vividly demonstrates that regulation of the number of receptors for a given signaling molecule expressed by a cell plays a key role in directing physiological and developmental events. Such regulation can occur at the levels of transcription, translation, and post-translational processing or by control of the rate of receptor degradation. Alternatively, endocytosis of receptors on the cell surface can sufficiently reduce the number present that the cellular response is effectively eliminated. As we discuss in later sections, other mechanisms can reduce a receptor's affinity for ligand and so reduce the cell's response to a given concentration of ligand. Thus reduction of a cell's sensitivity to a particular ligand, called desensitization, which is critical to the ability of cells to respond appropriately to external signals, can result from various mechanisms.

## Hormone Analogs Are Widely Used as Drugs

Synthetic analogs of natural hormones are widely used both in research on cell-surface receptors and as drugs. These analogs fall into two major classes. Agonists mimic the function of a natural hormone by binding to its receptor and inducing the normal cellular response to the hormone. Many synthetic agonists bind much more tightly to the receptor than does the natural hormone. In contrast, antagonists bind to the receptor but induce no response. By occupying ligandbinding sites on a receptor, an antagonist can block binding of a natural hormone (or agonist) and thus reduce the usual physiological activity of the hormone. In other words, antagonists inhibit receptor signaling.

Consider, for instance, the drug isoproterenol, used to treat asthma. Isoproterenol is made by the chemical addition of two methyl groups to epinephrine (Figure 15-9). Isoproterenol, an agonist of $\beta_{2}$ epinephrine-responsive G protein-coupled receptors on bronchial smooth muscle cells, binds about tenfold more strongly (has a tenfold lower $K_{\mathrm{d}}$ ) than does epinephrine. Because activation of these receptors promotes relaxation of bronchial smooth muscle and thus opening of the air passages in the lungs, isoproterenol and other agonists are used in treating bronchial asthma, chronic bronchitis, and emphysema. In contrast, activation of a different type of epinephrine-responsive G protein-coupled receptor on cardiac muscle cells (called the $\beta_{1}$-adrenergic receptor) increases the heart contraction rate. Antagonists of this receptor, such as alprenolol (see Figure 15-9) and related compounds, are referred to as beta-blockers; such antagonists are used to slow heart contractions in the treatment of cardiac arrhythmias and angina.

## Receptors Can Be Purified by Affinity Chromatography Techniques

In order to fully understand how receptors function, it is necessary to purify them and characterize their structures and biochemical properties in detail. Determining their molecular structures with and without a bound ligand, for instance, can elucidate the conformational changes that occur upon ligand binding that activate downstream signal transduction proteins. But separation of cell-surface receptors from other cellular proteins is very challenging. A "typical" mammalian cell has 1000 to 50,000 copies of a single type of cell-surface receptor. This may seem like a large number,
![li2214542480e](li2214542480e.jpg)

FIGURE 15-9 Structures of the natural hormone epinephrine, the synthetic agonist isoproterenol, and the synthetic antagonist alprenolol. As discussed in the text, isoproterenol and alprenolol, both of which bind to receptors for epinephrine, are used as drugs to treat different conditions.

but when you consider that this same cell contains $\sim 10^{10}$ total protein molecules, with $\sim 10^{6}$ of them in the plasma membrane alone, you realize that these receptors constitute only 0.1 to 5 percent of plasma-membrane proteins. This low abundance complicates the isolation and purification of cell-surface receptors. Purification of these integral membrane proteins is also difficult because the membrane must first be solubilized with a non-ionic detergent (see Figure 7-23) under conditions in which the three-dimensional structure of the receptor and its ability to bind ligand are maintained. Then the receptors can be separated from other cellular molecules.

Recombinant DNA techniques can often be used to generate cells that express large amounts of receptor proteins. But even when such techniques are used, special techniques are necessary to isolate the receptors from other membrane proteins. One technique often used in purifying cell-surface receptors that retain their ligand-binding ability when they are solubilized by detergents is a type of affinity chromatography (see Figure 3-40c). An antibody that recognizes either the receptor or a ligand for the receptor is chemically linked to the beads used to form a column. A crude, detergent-solubilized preparation of membrane proteins is then passed through the column. Only the receptor protein, together with other proteins tightly bound to it, will specifically stick to the column; other proteins are washed away. Once the other proteins are removed, the receptors can be released ("eluted") from the column either by passage of an excess of the soluble ligand through the column (ligand affinity chromatography) or by using chemical conditions (e.g., changes in pH ) to release the receptor from the antibody (antibody affinity chromatography). In some cases, a receptor can be purified as much as 100,000 -fold in a single affinity-chromatographic step.

## Immunoprecipitation Assays and Affinity Techniques Can Be Used to Study the Activity of Signal Transduction Proteins

Following ligand binding, receptors activate one or more signal transduction proteins that, in turn, can affect the activity of multiple target effector proteins (see Figures 15-1 and 15-7). Understanding a signaling cascade requires the researcher to be able to quantitatively measure the activity of these signal transduction proteins. Kinases and GTPbinding proteins are found in many signaling cascades; in this section, we describe several assays used for measuring their activities.

Immunoprecipitation of Kinases Kinases function in virtually all signaling pathways. Typical mammalian cells contain a hundred or more different types of kinases, each of which is highly regulated and can phosphorylate many target proteins. Immunoprecipitation assays, a type of antibody-affinity chromatography (see Figure 3-40c), are frequently used to measure the activity of a particular
kinase in a cell extract. In one version of this method, an antibody specific for the desired kinase is first incubated with small beads coated with protein A; this causes the antibody to bind to the beads via its Fc segment (see Figure 4-33). The beads are then mixed with an extract of the whole cell or of an organelle, such as the nucleus, then recovered by centrifugation and washed extensively with a salt solution to remove weakly bound proteins that are unlikely to be binding specifically to the antibody. Thus only cell proteins that specifically bind to the antibodythe kinase itself and proteins tightly bound to the kinaseare present on the beads. The beads are then incubated in a buffered solution with a substrate protein and $\gamma-\left[{ }^{32} \mathrm{P}\right]$ ATP, in which only the $\gamma$ phosphate of the ATP is radiolabeled. The amount of $\left[{ }^{32} \mathrm{P}\right]$ transferred to the substrate protein is a measure of kinase activity and can be quantified either by polyacrylamide gel electrophoresis followed by autoradiography (see Figure 3-38) or by immunoprecipitation with an antibody specific for the substrate followed by measurement of the radioactivity in the immunoprecipitate. By comparing extracts from cells before and after ligand addition, for example, one can readily determine whether or not a particular kinase is activated in the signal transduction pathway triggered by that ligand.

## Western Blotting with Monoclonal Antibodies Specific for a Phosphorylated Peptide We noted above that many

proteins can be phosphorylated by several different kinases, usually on different serine, threonine, or tyrosine residues. Thus it is important to measure the extent of phosphorylation of a single amino acid side chain in a specific protein, usually before and after ligand addition. Antibodies play a crucial role in detecting such phosphorylation events. To generate an antibody that can recognize a specific phosphorylated amino acid in a specific protein, one first chemically synthesizes an approximately 15 -aminoacid peptide that has the amino acid sequence surrounding the phosphorylated amino acid of the specific protein, with a phosphate group chemically linked to the desired serine, threonine, or tyrosine. After this peptide is coupled to an adjuvant (see Chapter 23) to increase its immunogenicity, it is used to generate a set of monoclonal antibodies (see Figure 4-6). One then selects a particular monoclonal antibody that reacts only with the phosphorylated, but not the nonphosphorylated, peptide; such an antibody generally will bind to the parent protein only when the same amino acid is phosphorylated. This specificity is possible because the antibody binds simultaneously to the phosphorylated amino acid and to side chains of adjacent amino acids. As an example of the use of such antibodies, Figure 15-10 shows that three signal transduction proteins in erythroid progenitors become phosphorylated on specific amino acid residues within 10 minutes of stimulation by varying concentrations of the hormone erythropoietin; phosphorylation, which is the first step in triggering the differentiation of these cells into red blood cells, increases with Epo concentration.

![li2224542480e](li2224542480e.jpg)

EXPERIMENTAL FIGURE 15-10 Activation by the hormone erythropoietin (Epo) of three signal transduction proteins via their phosphorylation. Mouse erythrocyte progenitor cells were treated for 10 minutes with different concentrations of the hormone erythropoietin (Epo). Extracts of the cells were analyzed by Western blotting with three different antibodies specific for the phosphorylated forms of three signal transduction proteins and three other antibodies that recognize a nonphosphorylated segment of amino acids in each of the same proteins. The data show that with increasing concentrations of Epo, the three proteins become phosphorylated. Treatment with 1 unit Epo/ml is sufficient to maximally phosphorylate and thus activate all three pathways. Stat5 = transcription factor phosphorylated on tyrosine 694; Akt = kinase phosphorylated on serine 473; p42/p44 = p42/p44 MAP kinase phosphorylated on threonine 202 and tyrosine 204. See Zhang et al., 2003, Blood 102:3938. [Courtesy Jing Zhang.]

![li2234542480e](li2234542480e.jpg)

Pull-Down Assays of GTP-Binding Proteins We've seen that the GTP-binding switch proteins of the GTPase superfamily cycle between an active ("on") form with bound GTP, which modulates the activity of specific target proteins, and an inactive ("off") form with bound GDP. The principal assay for measuring activation of this class of proteins takes advantage of the fact that each such switch protein has one or more targets to which it binds only when it has a bound GTP; the target protein usually has a specific binding domain that binds to the switch segments of that GTP-binding protein. Pull-down assays used to quantify the activation of a specific GTP-binding protein are similar to immunoprecipitation assays, except that the specific binding domain of the target protein is immobilized on small beads (Figure 15-11a). The beads are mixed with a cell extract and then recovered by centrifugation; the amount of the GTP-binding protein on the beads is quantified by Western blotting. Figure 15-11b shows that the fraction of the small GTPase Rac that has a bound GTP increases markedly after stimulation by the hormone platelet-derived growth factor (PDGF), indicating that Rac is a signal transduction protein activated by the PDGF receptor.

## KEY CONCEPTS OF SECTION 15.2

## Studying Cell-Surface Receptors and Signal Transduction Proteins

- The concentration of ligand at which half the ligand's receptors are occupied, the dissociation constant $\left(K_{\mathrm{d}}\right)$, can be determined experimentally and is a measure of the affinity of the receptor for the ligand (see Figure 15-8).
- Because of receptors' high affinity for their target ligand, the extracellular domains of receptors often can be used as parts of drugs to reduce the level of free hormone.
- The near-maximal response of a cell to a particular ligand generally occurs at ligand concentrations at which less than 100 percent of its receptors are bound to the ligand (see Figure 15-8).
- Affinity chromatography techniques can be used to purify receptors even when they are present in low abundance.
- Immunoprecipitation using antibodies specific for protein kinases can be used to measure the activity of these kinases. Western blotting assays using antibodies specific for phosphorylated peptides can measure phosphorylation of a specific amino acid on any desired protein within a cell (see Figure 15-10).
- Pull-down assays using the protein-binding domain of a target protein can be used to quantify activation of a GTPbinding protein within a cell (see Figure 15-11).


### 15.3 G Protein-Coupled Receptors: Structure and Mechanism

As noted above, perhaps the most numerous class of receptors are the G protein-coupled receptors (GPCRs). In humans, GPCRs are used to detect and respond to many different types of signals, including neurotransmitters, hormones involved in glycogen and fat metabolism, and even photons of light. Many GPCRs are found primarily in cells of the central nervous system and are used in neuronal signaling, as we will learn in Chapter 22. GPCRs are of immense medical importance, as approximately 30 percent of all drugs used in humans are agonists or antagonists of specific GPCRs or groups of closely related GPCRs. Table 15-1 describes just a few of these drugs; note the wide variety of receptors targeted by these drugs and the diverse diseases that they treat.

Despite this diversity, all GPCR signal transduction pathways share the following common elements: (1) a receptor that contains seven membrane-spanning $\alpha$ helices; (2) a heterotrimeric G protein, which functions as a receptoractivated switch by cycling between active and inactive forms; (3) a membrane-bound effector protein; and (4) proteins that participate in desensitization of the signaling pathway. Second messengers are also a part of many GPCR pathways. GPCR pathways usually have short-term effects in the cell by quickly modifying existing proteins, either enzymes or ion channels. Thus these pathways allow cells to respond rapidly to a variety of signals, whether they are environmental stimuli such as light or hormonal stimuli such as epinephrine.

In this section, we discuss the basic structure and mechanism of GPCRs and their associated heterotrimeric G proteins. In Sections 15.4 through 15.6, we describe GPCR pathways that activate several different effector proteins.

## All G Protein-Coupled Receptors Share the Same Basic Structure

All G protein-coupled receptors have the same orientation in the membrane and contain seven transmembrane $\alpha$-helical regions (H1-H7), four extracellular segments, and four cytosolic segments (Figure 15-12). Invariably the N -terminus is on the exoplasmic face and the C -terminus is on the cytosolic face of the plasma membrane. The human genome encodes some 800 functional GPCRs, which are divided into several subfamilies; members of these subfamilies are especially similar in amino acid sequence and structure.

G protein-coupled receptors are stably anchored in the hydrophobic core of the plasma membrane by the many hydrophobic amino acids on the outer surfaces of the seven membrane-spanning segments. One group of G proteincoupled receptors whose structure is known in molecular

TABLE 15-1 Human G Protein-Coupled Receptors of Pharmaceutical Importance

| Receptor | Natural Ligand | Location | Physiological Function | Drug | Medical Use |
| :--: | :--: | :--: | :--: | :--: | :--: |
| Histamine H2 receptor | Histamine | Acid-secreting cells of the stomach | Stimulates acid secretion | Cimetidine (Tagamet) Ranitidine (Zantac) (antagonists) | Prevent acid stomach; treat peptic ulcers |
| Histamine H1 receptor | Histamine | Smooth muscles, vascular endothelial cells | Increases vascular permeability and causes symptoms of allergy | Fexofenadine (Allegra) Loratadine (Claritin) (antagonists) | Reduce symptoms of allergy |
| Serotonin $5 \mathrm{HT}_{2 A}$ | Serotonin | Central nervous system | Synaptic transmission between neurons | Clozapine, risperidone (antagonists) | Treat schizophrenia |
| Serotonin $5 \mathrm{HT}_{1 a}$ | Serotonin | Central nervous system | Synaptic transmission between neurons | Buspirone (BuSpar) (agonist) | Treat depression, general anxiety disorder |
| Angiotensin $\mathrm{AT}_{1}$ | Angiotensin II | Vascular smooth muscle cells | Constrict blood vessels and increase blood pressure | Losartan (Cozarr) (antagonist) | Reduce hypertension |
| $\beta_{2}$-adrenergic receptor | Epinephrine | Smooth muscle cells lining the airway | Facilitate respiration | Salmeterol (Severent) (agonist) | Treatment of asthma, chronic obstructive pulmonary disease |
| CysLT $_{1}$ | Leukotrienes | Lungs, bronchial tubes, mast cells | Contracts smooth muscles | Montelukast (Singulair) <br> (antagonist) | Treatment of asthma, seasonal allergies |

source: Data from A. Wise et al., 2002, Drug Discovery Today 7:235-246.
detail is the $\beta$-adrenergic receptors, which bind hormones such as epinephrine and norepinephrine (Figure 15-13a). In these and many other receptors, segments of several membranembedded $\alpha$ helices and extracellular loops form the ligandbinding site, which is open to the exoplasmic surface. The antagonist cyanopindolol, shown in Figure 15-13b, binds with a much higher affinity to the $\beta$-adrenergic receptor than do most agonists, and the resulting receptor-ligand complex has been crystallized and its structure determined. Side chains of 15 amino acids located in four transmembrane $\alpha$ helices and extracellular loop E2 make noncovalent contacts with the ligand.

Glucagon is a 29-amino-acid peptide hormone secreted by the $\alpha$ cells of the pancreatic islets; as we will see later in this chapter, it acts on the liver to trigger glycogen breakdown and secretion of glucose into the bloodstream. The glucagon receptor is in a different GPCR subfamily from the $\beta$-adrenergic receptors; while it has the standard seven transmembrane $\alpha$ helices, it also has a large exoplasmic domain that is connected to an extension of the first transmembrane $\alpha$ helix. This exoplasmic domain binds tightly to the C -terminus of glucagon, positioning the N -terminus to bind to the pocket formed, as in $\beta$-adrenergic and other GPCRs, by residues from several transmembrane helices (Figure 15-13c). The amino acids that form the interior of
![li2244542480e](li2244542480e.jpg)

FIGURE 15-12 General structure of G protein-coupled receptors. All receptors of this type have the same orientation in the membrane and contain seven transmembrane $\alpha$-helical regions (H1-H7), four extracellular segments (E1-E4), and four cytosolic segments (C1-C4).
different GPCRs are diverse, allowing different receptors to bind very different small molecules, whether they are hydrophilic, such as epinephrine and glucagon, or hydrophobic, such as many odorants. But in all cases, binding of the ligand causes a conformational change in the receptor that enables it to activate the GTP-binding $G_{i a}$ subunit of a heterotrimeric G protein.

![li2254542480e](li2254542480e.jpg)

FIGURE 15-13 Binding of ligands to GPCRs. (a) Structure of the turkey $\beta_{1}$-adrenergic receptor bound to its antagonist cyanopindolol. This side view shows the approximate location of the membrane phospholipid bilayer. The ribbon representation of the receptor structure is in rainbow coloration ( N -terminus, blue; C -terminus, red), with cyanopindolol as a gray space-filling molecule. The extracellular loop 2 (E2) and cytoplasmic loops 1, 2, and 4 (C1, C2, C4) 1 and 2 (C1, C2) are labeled. (b) The hormone-binding pocket formed by residues in several transmembrane segments. View from external face showing a close-up of the ligand-binding pocket that is formed by amino acids in helices $3,5,6$, and 7 , as well as extracellular loop 2, located between helices 4 and 5. Cyanopindolol atoms are colored gray (carbon), blue (nitrogen), and red (oxygen). The ligand-binding pocket comprises 15 side chains from amino acid residues in four transmembrane a helices and extracellular loop 2.

As examples of specific binding interactions, the positively charged N atom in the amino group found both in cyanopindolol and in epinephrine forms an ionic bond with the carboxylate side chain of aspartate 121 (D) in helix 3 and the carboxylate of asparagine 329 $(\mathrm{N})$ in helix 7 . (c) Model for glucagon binding to the glucagon receptor. The seven transmembrane a helices of the glucagon receptor and the exoplasmic extension of transmembrane helix 1 are colored dark green and the N -terminal exoplasmic domain light green. The C-terminus of the 29-amino-acid peptide glucagon (red) is bound to the receptor N -terminal domain, and the glucagon N -terminus is thought to insert into a binding pocket that is in the center of the seven transmembrane a helices. [Parts (a) and (b) data from T. Warne et al., 2008, Nature 454:486, PDB ID 2vt4. Part (c) data from P. Siu et al., 2014 Nature 499:444, custom PDB.]

## Ligand-Activated G Protein-Coupled Receptors Catalyze Exchange of GTP for GDP on the $\alpha$ Subunit of a Heterotrimeric G Protein

Heterotrimeric G proteins contain three subunits, designated $\alpha, \beta$, and $\gamma$. Only the $G_{s}$ subunit binds GTP or GDP; both the $G_{s}$ and $G_{\gamma}$ subunits are linked to the plasma membrane by covalently attached lipids. The $\beta$ and $\gamma$ subunits are always bound together and are usually referred to as the $\mathrm{G}_{\beta \gamma}$ subunit. In the resting state, when no ligand is bound to the receptor, the $G_{s}$ subunit has a bound GDP and is complexed with $\mathrm{G}_{\beta \gamma}$. Binding of a ligand (e.g., epinephrine) or an agonist (e.g., isoproterenol) to a G protein-coupled receptor changes the conformation of its transmembrane helices and enables the receptor to bind to the $G_{s}$ subunit of the intact heterotrimeric G protein (Figure 15-14, steps $\mathbf{\square}$ and $\mathbf{\square}$ ). This
binding releases the bound GDP; thus the activated ligandbound receptor functions as a guanine nucleotide exchange factor (GEF) for the $G_{s}$ subunit (step $\mathbf{\square}$ ). Next GTP rapidly binds to the "empty" guanine nucleotide site in the $G_{s}$ subunit, causing a change in the conformation of its switch segments (see Figure 15-5). These changes weaken the binding of $G_{s}$ to both the receptor and the $G_{\beta \gamma}$ subunit (step $\mathbf{\square}$ ). In most cases, $G_{s}$-GTP, which remains anchored in the plasma membrane, then interacts with and activates an effector protein (step $\mathbf{\square}$ ). In some cases, $G_{s}$-GTP inhibits, rather than activates, the effector. Moreover, depending on the type of cell and G protein involved, the $G_{\beta \gamma}$ subunit, freed from the $G_{s}$ subunit, will sometimes transduce a signal by interacting with an effector protein.

The active $G_{s}$-GTP state is relatively short-lived because hydrolysis of the bound GTP to GDP, catalyzed by the
![li2264542480e](li2264542480e.jpg)

FIGURE 15-14 General mechanism of the activation of effector proteins associated with G protein-coupled receptors. Light colors denote the inactive and dark colors the active conformations of each protein. The $G_{s}$ and $G_{\beta s}$ subunits of a heterotrimeric G protein are tethered to the membrane by covalently attached lipid molecules (wiggly black lines). Following ligand binding, exchange of GDP for GTP, and dissociation of the G protein subunits
(steps $\mathbf{\square}-\mathbf{\square}$ ), the free $G_{s}$-GTP binds to and activates an effector protein (step $\mathbf{\square}$ ). Hydrolysis of GTP terminates signaling and leads to reassembly of the heterotrimeric G protein, returning the system to the resting state (step $\mathbf{\square}$ ). Binding of another ligand molecule causes repetition of the cycle. In some pathways, the effector protein is activated by the free $G_{\beta s}$ subunit. See W. Oldham and H. Hamm, 2006, Quart. Rev. Biophys. 39:117.

intrinsic GTPase activity of the $\mathrm{G}_{\alpha}$ subunit, occurs in minutes (Figure 15-14, step $\mathbf{\square}$ ). The conformation of the $\mathrm{G}_{\alpha}$ subunit thus switches back to the inactive $\mathrm{G}_{\alpha}$-GDP state, blocking any further activation of effector proteins. The resulting $\mathrm{G}_{\alpha}$-GDP quickly reassociates with $\mathrm{G}_{\beta \gamma}$, and the complex becomes ready to interact with an activated receptor and start the process all over again.

The rate of GTP hydrolysis is sometimes further enhanced by binding of the $\mathrm{G}_{\alpha}$-GTP complex to the effector; in this case, the effector functions as a GTPase-activating protein (GAP). This feedback mechanism significantly reduces the duration of effector activation and avoids a cellular overreaction. In many cases, a second type of GAP protein, called a regulator of G protein signaling (RGS), also accelerates GTP hydrolysis by the $\mathrm{G}_{\alpha}$ subunit, further reducing the time during which the effector remains activated. Thus the GPCR signal transduction system contains built-in feedback mechanisms that ensure that the effector protein becomes activated for only a few seconds or minutes following receptor activation; continual activation of receptors via ligand binding, together with subsequent activation of the corresponding G protein, is essential for prolonged activation of the effector.

Early evidence supporting the model shown in Figure 15-14 came from studies with compounds that are structurally similar to GTP, and so can bind to $\mathrm{G}_{\alpha}$ subunits as well as GTP does, but cannot be hydrolyzed by the intrinsic GTPase. In some of these compounds, the $\mathrm{P}-\mathrm{O}-\mathrm{P}$ phosphodiester linkage connecting the $\beta$ and $\gamma$ phosphates of GTP is replaced by a nonhydrolyzable $\mathrm{P}-\mathrm{CH}_{2}-\mathrm{P}$ or $\mathrm{P}-\mathrm{NH}-\mathrm{P}$ linkage. Addition of such a GTP analog to a plasma-membrane preparation in the presence of an agonist for a particular receptor results in a much longer-lived activation of the $\mathrm{G}_{\alpha}$
protein and its associated effector protein than occurs with GTP. In this experiment, once the nonhydrolyzable GTP analog is exchanged for GDP bound to $\mathrm{G}_{\alpha}$, it remains permanently bound to $\mathrm{G}_{\alpha}$. Because the $\mathrm{G}_{\alpha}$-GTP-analog complex is as functional as the normal $\mathrm{G}_{\alpha}$-GTP complex in activating the effector protein, the effector remains permanently active.

GPCR-mediated dissociation of heterotrimeric G proteins can be detected in live cells. These studies have exploited the phenomenon of Förster resonance energy transfer (FRET), which changes the wavelength of emitted fluorescence when two fluorescent proteins interact (see Figure 4-24). Figure 15-15 shows how this experimental approach has demonstrated the dissociation of the $\mathrm{G}_{\alpha}-\mathrm{G}_{\beta \gamma}$ complex within a few seconds of ligand addition, providing evidence for the model of G protein cycling. This general experimental approach can be used to follow the formation and dissociation of other protein-protein complexes in live cells.

For many years, it was impossible to determine the structure of the same GPCR in the active and inactive states. This has now been accomplished with the $\beta_{2}$-adrenergic receptor (as well as with rhodopsin, discussed in Section 15.4). The initial cloning and characterization of the $\beta_{2}$-adrenergic receptor by Robert Lefkowitz and the three-dimensional structures obtained by Brian Kobilka and depicted in Figure 15-16 were rewarded with the 2012 Nobel Prize in Chemistry. The seven membrane-embedded $\alpha$ helices of the $\beta_{2}$-adrenergic receptor form the binding pocket to which an agonist or antagonist can be noncovalently bound (see Figure 15-13). Binding of an agonist to the receptor induces major conformational changes in which there are substantial movements of transmembrane helices 5 and 6
![li2274542480e](li2274542480e.jpg)

EXPERIMENTAL FIGURE 15-15 Activation of a G protein occurs within seconds of ligand binding to its cell-surface
G protein-coupled receptor. In the amoeba Dictyostelium discoideum, cAMP acts as an extracellular signaling molecule and binds to and signals via a G protein-coupled receptor; it is not a second messenger. Amoeba cells were transfected with genes encoding two fusion proteins: a $\mathrm{G}_{\alpha}$ fused to cyan fluorescent protein (CFP), and a $\mathrm{G}_{\beta}$ fused to yellow fluorescent protein (YFP). CFP normally fluoresces 490-nm light; YFP, 527-nm light. (a) When CFP and YFP are close to each other, as in the resting $\mathrm{G}_{\alpha}-\mathrm{G}_{\beta \gamma}$ complex, fluorescence energy transfer can occur between them (left). As a result, irradiation of resting cells with

440-nm light (which directly excites CFP but not YFP) causes emission of 527-nm (yellow) light, characteristic of YFP, because of fluorescence energy transfer from CFP to YFP. However, if ligand binding leads to dissociation of the $\mathrm{G}_{\alpha}$ and $\mathrm{G}_{\beta \gamma}$ subunits, then fluorescence energy transfer cannot occur. In this case, irradiation of cells at 440 nm causes emission of 490-nm (cyan) light, characteristic of CFP (right). (b) Plot of the emission of yellow light ( 527 nm ) from a single transfected amoeba cell before and after addition of extracellular cAMP (arrow). The drop in yellow fluorescence, which results from the dissociation of the $\mathrm{G}_{\alpha}$-CFP fusion protein from the $\mathrm{G}_{\beta}$ YFP fusion protein, occurs within seconds of cAMP addition. [Data from C. Janetopoulos et al., 2001, Science 291:2408.]
(b)

and changes in the structure of the C3 loop; together, these changes create a surface that can now bind to a segment of the $\mathrm{G}_{\alpha}$ subunit (Figure 15-16b). Note that when unbound by ligand, the receptor (Figure 15-16a) has no surface that is complementary to the G protein and thus cannot bind to it.

X-ray crystallographic studies of the complex of the activated $\beta_{2}$-adrenergic receptor and its coupled G protein, $\mathrm{G}_{\alpha}$ have also revealed how the subunits of a G protein interact with each other and have provided clues about how binding of GTP leads to dissociation of the $\mathrm{G}_{\alpha}$ from the $\mathrm{G}_{\beta \gamma}$ subunit. Upon binding to an activated receptor, the $\mathrm{G}_{\alpha}$ subunit undergoes a small conformational change, lengthening the $\alpha 5$ helix. This change creates a large surface, mainly consisting of the N -terminal $\alpha$-helical segments $\alpha \mathrm{N}$ and $\alpha 5$ of the $\mathrm{G}_{\alpha s}$ protein that bind mainly to transmembrane helices 5 and 6 of the activated receptor (see Figure 15-16b). Concomitantly, this conformational change in $\mathrm{G}_{\alpha}$ triggers the release of GDP. Note that a large surface of $\mathrm{G}_{\alpha}$-GDP, including $\alpha \mathrm{N}$, interacts with the $\mathrm{G}_{\beta}$ subunit, but $\mathrm{G}_{\alpha}$-GDP does not directly contact $\mathrm{G}_{\gamma}$. Binding to the GPCR is followed by opening of the $\mathrm{G}_{\alpha}$ subunit, eviction of the bound GDP, and its replacement with GTP; this is immediately followed by conformational changes within switches I and II that disrupt the molecular interactions between $\mathrm{G}_{\alpha}$ and $\mathrm{G}_{\beta \gamma}$, leading to the dissociation of $\mathrm{G}_{\alpha}$ from the $\mathrm{G}_{\beta \gamma}$ subunit.

## Different G Proteins Are Activated by Different GPCRs and In Turn Regulate Different Effector Proteins

All effector proteins in GPCR signal transduction pathways are either membrane-bound ion channels or membrane-bound enzymes that catalyze the formation of one or more of the second messengers shown in Figure 15-6. The variations on the theme of GPCR signaling that we examine in Sections 15.4 through 15.6 arise because multiple G proteins with distinct activities are encoded in eukaryotic genomes. Humans have 21 different $\mathrm{G}_{\alpha}$ subunits encoded by 16 genes, several of which undergo alternative splicing; $6 \mathrm{G}_{\beta}$ subunits; and $12 \mathrm{G}_{\gamma}$ subunits. So far as is known, the different $\mathrm{G}_{\beta \gamma}$ subunits are interchangeable in their functions, while the different $\mathrm{G}_{\alpha}$ subunits afford the various G proteins their specificity. Thus we can refer to the entire three-subunit G protein by the name of its alpha subunit.

Table 15-2 summarizes the functions of the major classes of G proteins with different $\mathrm{G}_{\alpha}$ subunits. To illustrate the versatility of these proteins, we will consider the set of G proteincoupled receptors for epinephrine found in different types of mammalian cells. The hormone epinephrine is particularly important in mediating the body's response to stress, also known as the fight-or-flight response. As we detail in Section 15.5, during moments of fear or heavy exercise, when tissues may have an increased need to catabolize glucose and fatty acids
(a) Inactive receptor
![li2284542480e](li2284542480e.jpg)

FIGURE 15-16 Structure of the $\beta_{2}$-adrenergic receptor in the inactive and active states and with its associated heterotrimeric G protein, $\mathbf{G}_{\alpha}$. (a) The three-dimensional structure of the $\beta_{2}$-adrenergic receptor bound to an antagonist (not shown) and thus in the inactive state. Placed next to it are the three-dimensional structures of the subunits of the heterotrimeric G protein $\mathrm{G}_{\alpha}: \mathrm{G}_{\alpha s}$ (dark purple), $\mathrm{G}_{\beta}$
(b) Active receptor
![li2294542480e](li2294542480e.jpg)
(light purple), and $\mathrm{G}_{\gamma}$ (pink), showing the inability of the resting receptor to bind to and activate $\mathrm{G}_{\alpha \alpha}$. (b) The overall structure of the active receptor complex shows the adrenergic receptor bound to an agonist (black and red spheres) and engaged in extensive interactions with a segment of $\mathrm{G}_{\alpha \alpha}$. (Data from S. Rasmussen et al., 2011, Nature 476:549, PDB ID 3sn6; and V. Cherezov et al., 2007, Science 318:1258, PDB ID 2rhl.)

| $\mathrm{G}_{\mathrm{o}}$ Class | Associated Effector | Second Messenger | Receptor Examples |
| :--: | :--: | :--: | :--: |
| $G_{\text {ox }}$ | Adenylyl cyclase | cAMP (increased) | $\beta$-Adrenergic (epinephrine) receptor; receptors for glucagon, serotonin, vasopressin |
| $\mathrm{G}_{\mathrm{isi}}$ | Adenylyl cyclase $\mathrm{K}^{+}$channel $\left(\mathrm{G}_{\mathrm{ps}}\right.$ activates effector) | cAMP (decreased) Change in membrane potential | $\alpha_{2}$-Adrenergic receptor <br> Muscarinic acetylcholine receptor |
| $G_{\text {sulf }}$ | Adenylyl cyclase | cAMP (increased) | Odorant receptors in nose |
| $G_{\text {nq }}$ | Phospholipase C | $\mathrm{IP}_{3}, \mathrm{DAG}$ (increased) | $\alpha_{1}$-Adrenergic receptor |
| $G_{\text {no }}$ | Phospholipase C | $\mathrm{IP}_{3}, \mathrm{DAG}$ (increased) | Acetylcholine receptor in endothelial cells |
| $G_{\text {tat }}$ | cGMP phosphodiesterase | cGMP (decreased) | Rhodopsin (light receptor) in rod cells |

${ }^{\star}$ A given $\mathrm{G}_{\mathrm{o}}$ subclass may be associated with more than one effector protein. To date, only one major $\mathrm{G}_{\mathrm{ox}}$ has been identified, but multiple $\mathrm{G}_{\mathrm{nq}}$ and $\mathrm{G}_{\mathrm{ss}}$ proteins have been described. Effector proteins commonly are regulated by $\mathrm{G}_{\mathrm{o}}$ but in some cases by $\mathrm{G}_{\mathrm{ps}}$ or the combined action of $\mathrm{G}_{\mathrm{o}}$ and $\mathrm{G}_{\mathrm{ps}}$. $\mathrm{IP}_{3}=$ inositol 1,4,5-trisphosphate; $\mathrm{DAG}=1,2$-diacylglycerol.
sources: See L. Birnbaumer, 1992, Cell 71:1069; Z. Farfel et al., 1999, New Engl. J. Med. 340:1012; and K. Pierce et al., 2002, Nat. Rev. Mol. Cell Biol. 3:639.
to produce ATP, epinephrine signals for the rapid breakdown of glycogen to glucose in hepatic (liver) cells and of triacylglycerols to fatty acids in adipose (fat) cells; within seconds, these principal metabolic fuels are supplied to the blood. In mammals, the liberation of glucose and fatty acids is triggered by the binding of epinephrine (or its derivative, norepinephrine) to $\beta_{2}$-adrenergic receptors on the surface of hepatic and adipose cells. Both subtypes of $\beta$-adrenergic receptors, termed $\beta_{1}$ and $\beta_{2}$, are coupled to a stimulatory G protein $\left(\mathrm{G}_{\mathrm{s}}\right)$ whose alpha subunit $\left(\mathrm{G}_{\mathrm{as}}\right)$ activates a membrane-bound effector enzyme called adenylyl cyclase. Once activated, this enzyme catalyzes synthesis of the second messenger cAMP.

Epinephrine has other physical effects as well. Epinephrine bound to $\beta_{1}$-adrenergic receptors on heart muscle cells, for example, increases the contraction rate, which increases the blood supply to the tissues. Yet another type of epinephrine GPCR, the $\alpha_{1}$-adrenergic receptor, is found on smooth muscle cells lining the blood vessels in the intestinal tract, skin, and kidneys. Binding of epinephrine to these receptors causes the arteries to constrict, cutting off circulation to these organs. The $\mathrm{G}_{\mathrm{nq}}$ subunit, which is coupled to the $\alpha_{1}$-adrenergic receptor, activates a different effector enzyme, phospholipase C, which generates two other second messengers, DAG and IP3 (see Figure 15-6). Yet another epinephrine receptor, the $\alpha_{2}$-adrenergic receptor, is found on many body cells and is coupled to the $\mathrm{G}_{\mathrm{si}}$ subunit, which causes inhibition of adenylyl cyclase in these target cells. These diverse effects of epinephrine help orchestrate integrated responses throughout the body, all directed to a common end: supplying energy to major locomotor muscles, while at the same time diverting it from other organs not as crucial in executing a response to physical stress.

ESome bacterial toxins contain a subunit that penetrates the plasma membrane of target mammalian cells and, once in the cytosol, catalyzes a chemical modification of $\mathrm{G}_{\mathrm{o}}$ proteins that prevents hydrolysis of bound GTP to GDP. For example, toxins produced by the bacterium Vibrio cholerae,
which causes cholera, or certain strains of E. coli enter intestinal epithelial cells and catalyze a covalent modification of the $\mathrm{G}_{\mathrm{as}}$ protein in these cells. As a result, $\mathrm{G}_{\mathrm{as}}$ remains in the active state, continuously activating the effector adenylyl cyclase in the absence of hormonal stimulation. The resulting excessive rise in intracellular cAMP leads to the loss of electrolytes and water into the intestinal lumen, producing the watery diarrhea characteristic of infection by these bacteria. The toxin produced by Bordetella pertussis, a bacterium that commonly infects respiratory tract cells and causes whooping cough, catalyzes a modification of $\mathrm{G}_{\mathrm{si}}$ that prevents release of bound GDP. As a result, $\mathrm{G}_{\mathrm{si}}$ is locked in the inactive state, reducing the inhibition of adenylyl cyclase. The resulting increase in cAMP in epithelial cells of the airways promotes loss of fluids and electrolytes and mucus secretion.

With some 800 members in total, the G protein-coupled receptors represent the largest protein family in the human genome. Approximately half of the genes encoding these proteins are thought to encode sensory receptors; of these, the majority are in the olfactory system and bind odorants. The natural ligand has not been identified for many so-called orphan GPCRs-that is, putative GPCRs without known cognate ligands. Many of these orphan receptors are likely to bind heretofore unidentified signaling molecules, including novel peptide hormones. One approach that has proved fruitful in identifying the ligands of orphan GPCRs involves expressing the gene encoding the receptor in transfected cells and using the cells as a reporter system to detect substances in tissue extracts that activate the receptor and its downstream signal transduction pathway. This approach led to the discovery of two novel peptides, termed orexin-A and orexin-B (from the Greek orexis, meaning "appetite"), that were identified as the ligands for two orphan GPCRs in the same family as the glucagon receptor. Further research showed that the orexin gene is expressed only in the hypothalamus, the part of the brain that regulates feeding. Injection of orexin into the brain

ventricles of animals caused them to eat more, and expression of the orexin gene increased markedly during fasting. Both of these findings are consistent with orexin's role in increasing appetite. Strikingly, mice deficient for orexins suffer from narcolepsy, a disorder characterized in humans by excessive daytime sleepiness (in mice, nighttime sleepiness). Moreover, very recent reports suggest that the orexin system is dysfunctional in a majority of human narcolepsy patients: orexin peptides cannot be detected in their cerebrospinal fluid (although there is no evidence of mutation in their orexin genes). These findings firmly link orexin neuropeptides and their receptors to both feeding behavior and sleep in both animals and humans.

## KEY CONCEPTS OF SECTION 15.3

## G Protein-Coupled Receptors: Structure and Mechanism

- G protein-coupled receptors (GPCRs) are a large and diverse family with a common structure of seven membranespanning $\alpha$ helices and an internal ligand-binding pocket that is specific for particular ligands (see Figures 15-12 and 15-13).
- GPCRs are coupled to heterotrimeric G proteins, which contain three subunits designated $\alpha, \beta$, and $\gamma$. The $G_{s}$ subunit is a GTPase switch protein that alternates between an active ("on") state with bound GTP and inactive ("off") state with bound GDP. The "on" form separates from the $\beta$ and $\gamma$ subunits and activates a membrane-bound effector. The $\beta$ and $\gamma$ subunits remain bound together and can also transduce signals (see Figure 15-14).
- Ligand binding causes a conformational change in certain membrane-spanning helices and intracellular loops of the GPCR, allowing it to bind to and function as a guanine nucleotide exchange factor (GEF) for its coupled $G_{s}$ subunit, catalyzing dissociation of GDP and allowing GTP to bind. The resulting change in the conformation of the switch region in $G_{s}$ causes it to dissociate from the $G_{\beta \gamma}$ subunit and the receptor and interact with an effector protein (see Figure 15-14).
- FRET experiments demonstrate receptor-mediated dissociation of coupled $G_{s}$ and $G_{\beta \gamma}$ subunits in live cells (see Figure 15-15).
- The effector proteins activated (or inactivated) by heterotrimeric G proteins are either enzymes that form second messengers (e.g., adenylyl cyclase, phospholipase C) or ion channels (see Table 15-2). In each case, it is the $G_{s}$ subunit that determines the function of the $G$ protein and affords its specificity.
- GPCRs can have a range of cellular effects depending on the subtype of receptor that binds a ligand. The hormone epinephrine, for example, which mediates the fight-or-flight response, binds to multiple subtypes of GPCRs in multiple cell types, with varying physiological effects.
- Efforts to identify orphan GPCRs led to the discovery of orexins, hormones that regulate feeding behavior and sleep in both animals and humans.


### 15.4 G Protein-Coupled Receptors That Regulate Ion Channels

One of the simplest cellular responses to a signal is the opening or closing of ion channels that are essential for transmission of nerve impulses. Nerve impulses are essential to the sensory perception of environmental stimuli such as light and odors, to the transmission of information to and from the brain, and to the stimulation of muscle movement. During transmission of nerve impulses, the opening and closing of ion channels causes changes in the membrane potential. Many neurotransmitter receptors are ligand-gated ion channels, which open in response to the binding of a ligand. Such receptors include some types of glutamate, serotonin, and acetylcholine receptors, including the acetylcholine receptor found at nerve-muscle synapses. Ligand-gated ion channels that function as neurotransmitter receptors are covered in Chapter 22.

Many neurotransmitter receptors, however, are G protein-coupled receptors whose effector proteins are $\mathrm{Na}^{+}$or $\mathrm{K}^{+}$channels. Neurotransmitter binding to these receptors causes the associated ion channel to open or close, leading to changes in the membrane potential. Still other neurotransmitter receptors, as well as odorant receptors in the nose and photoreceptors in the eye, are G protein-coupled receptors that indirectly modulate the activity of ion channels via the action of second messengers. In this section, we consider two G protein-coupled receptors that illustrate the direct and indirect mechanisms for regulating ion channels: the muscarinic acetylcholine receptor of the heart and the light-activated rhodopsin protein in the eye.

## Acetylcholine Receptors in the Heart Muscle Activate a G Protein That Opens $\mathrm{K}^{+}$Channels

Muscarinic acetylcholine receptors are a type of GPCR found in cardiac muscle. When activated, these receptors slow the rate of heart muscle contraction. Because muscarine, an acetylcholine analog, also activates these receptors, they are termed "muscarinic." This type of acetylcholine receptor is coupled to a $G_{s i}$ protein, and ligand binding leads to the opening of an associated $\mathrm{K}^{+}$channel (the effector protein) in the plasma membrane (Figure 15-17). The subsequent efflux of $\mathrm{K}^{+}$ions from the cytosol causes an increase in the magnitude of the usual inside-negative potential across the plasma membrane that lasts for several seconds. This state of the membrane, called the hyperpolarized state, reduces the frequency of muscle contraction. This effect can be shown experimentally by adding acetylcholine to isolated heart muscle cells and measuring the membrane potential using a microelectrode inserted into the cell (see Figure 11-19).

As shown in Figure 15-17, the signal from activated muscarinic acetylcholine receptors is transduced to the effector channel protein by the released $\mathrm{G}_{\beta \gamma}$ subunit, rather than by $\mathrm{G}_{\mathrm{si}} \cdot \mathrm{GTP}$. That $\mathrm{G}_{\beta \gamma}$ directly activates the $\mathrm{K}^{+}$channel was demonstrated by patch-clamping experiments, which can measure ion flow through one or a few ion channels in a

![li2304542480e](li2304542480e.jpg)

FIGURE 15-17 In heart muscle, the muscarinic acetylcholine receptor activates its effector $\mathrm{K}^{+}$channel via the $\mathrm{G}_{\mathrm{i}}$, subunit of a G protein. Binding of acetylcholine triggers activation of the $\mathrm{G}_{\mathrm{at}}$ subunit and its dissociation from the $\mathrm{G}_{\mathrm{By}}$ subunit in the usual way (see Figure 15-14). In this case, however, the released $\mathrm{G}_{\mathrm{By}}$ subunit (rather than $\mathrm{G}_{\mathrm{at}}$-GTP) binds to and opens the associated effector protein, a $\mathrm{K}^{+}$channel. The increase in $\mathrm{K}^{+}$permeability hyperpolarizes the membrane, which reduces the frequency of heart muscle contraction. Though not shown here, activation is terminated when the GTP bound to $\mathrm{G}_{\mathrm{at}}$ is hydrolyzed (by a GAP enzyme that is an intrinsic part of the $\mathrm{G}_{\mathrm{at}}$ subunit) to GDP and $\mathrm{G}_{\mathrm{at}}$-GDP recombines with $\mathrm{G}_{\mathrm{By}}$. See K. Ho et al., 1993, Nature 362:31, and Y. Kubo et al., 1993, Nature 362:127.
small patch of membrane (see Figure 11-22). When purified $\mathrm{G}_{\mathrm{By}}$ (but not $\mathrm{G}_{\mathrm{at}}$-GTP) protein was added to the cytosolic face of a patch of heart muscle plasma membrane, $\mathrm{K}^{+}$channels opened immediately, even in the absence of acetylcholine or other neurotransmitters-clearly indicating that it is the $\mathrm{G}_{\mathrm{By}}$ protein that is responsible for opening the effector $\mathrm{K}^{+}$channels.

## Light Activates Rhodopsin in Rod Cells of the Eye

The human retina contains two types of photoreceptor cells, rods and cones, which are the primary recipients of visual stimulation. Cones are involved in color vision, while rods are stimulated by weak light such as moonlight over a range of wavelengths. The photoreceptor cells signal to (synapse with) layer upon layer of interneurons that are innervated by different combinations of photoreceptor cells. All these signals are processed and relayed through the visual thalamus to the part of the brain called the visual cortex, where they are interpreted.

Rod cells sense light with the aid of a light-sensitive GPCR known as rhodopsin. Rhodopsin consists of the protein opsin, which has the usual seven-transmembrane segment GPCR structure, covalently linked to a light-absorbing pigment called retinal. Rhodopsin, found only in rod cells, is localized to the
![li2314542480e](li2314542480e.jpg)

FIGURE 15-18 Human rod cell. (a) Schematic diagram of an entire rod cell. At the synaptic body, the rod cell forms synapses with one or more interneurons. Rhodopsin, a light-sensitive G protein-coupled receptor, is located in the flattened membrane disks of the cell's outer segment. (b) Electron micrograph of the region of the rod cell indicated by the bracket in (a). This region includes the junction of the inner and outer segments. [Part (b) Don W. Fawcett/Science Source.]
thousand or so flattened membrane disks that make up the outer segment of each of these rod-shaped cells (Figure 15-18). A human rod cell contains about $4 \times 10^{7}$ molecules of rhodopsin. The heterotrimeric G protein coupled to rhodopsin, called transducin $\left(\mathrm{G}_{\mathrm{t}}\right)$, contains a $\mathrm{G}_{\mathrm{n}}$ unit referred to as $\mathrm{G}_{\mathrm{nt}}$; like rhodopsin, $\mathrm{G}_{\mathrm{nt}}$ is found only in rod cells.

Rhodopsin (R) differs from other GPCRs in that binding of a ligand is not what activates the receptor. Rather, absorption of a photon of light by the bound retinal is the activating signal. On absorption of a photon, the retinal moiety of rhodopsin is immediately converted from the cis isomeric form (known as 11-cis-retinal) to the all-trans isomeric form, causing a conformational change in the opsin protein (Figure 15-19). This change is equivalent to the activating conformational change that occurs upon ligand binding by other G protein-coupled receptors; this conformational change allows rhodopsin to bind the $\mathrm{G}_{\mathrm{nt}}$ subunit of its coupled G protein, transducin, triggering exchange of GTP for GDP on its $\mathrm{G}_{\mathrm{nt}}$ subunit. At the same time, transducin also dissociates into its $\mathrm{G}_{\mathrm{nt}}$ and $\mathrm{G}_{\mathrm{By}}$ subunits. The activated rhodopsin that is formed, termed $\mathrm{R}^{*}$, is unstable, as the covalent linkage to retinal is spontaneously cleaved. Since retinal-free opsin cannot bind transducin, initiation of

![li2324542480e](li2324542480e.jpg)

FIGURE 15-19 Vision depends on the light-triggered isomerization of the retinal moiety of rhodopsin. Rhodopsin consists of the light-absorbing pigment 11-cis-retinal covalently attached to the amino group of lysine residue 296 in the opsin protein. Absorption of light causes rapid photoisomerization of the bound cis-retinal to the all-trans isomer. This change triggers a conformational change in rhodopsin, forming the unstable intermediate meta-rhodopsin II, or activated rhodopsin (see Figure 15-21 below), which activates $G_{s}$ proteins. Within seconds, all-trans-retinal dissociates from opsin and is converted by a series of enzymes in the rod cell and pigmented epithelium back to the cis isomer, which then rebinds to another opsin molecule. See J. Nathans, 1992, Biochemistry 31:4923.
visual signaling is terminated at this point. In the dark, free all-trans-retinal is converted back to 11-cis-retinal in a series of steps involving enzymes from both rod cells and cells in the adjacent retinal pigment epithelium. The 11-cis-retinal that is generated is then transferred to the rod cells, where it rebinds opsin, forming rhodopsin and completing the rhodopsin visual cycle.

## Activation of Rhodopsin by Light Leads to Closing of cGMP-Gated Cation Channels

In the dark, the membrane potential of a rod cell is about -30 mV , considerably more positive (less negative) than the resting potential ( -60 to -90 mV ) typical of neurons and other electrically active cells (see Chapter 11). This state of the membrane, called the depolarized state, causes rod cells in the dark to secrete neurotransmitters constantly, and thus the neurons with which they synapse are constantly being
stimulated. The depolarized state of the plasma membrane of a resting rod cell is due to the presence of a large number of open nonselective ion channels that admit both $\mathrm{Na}^{+}$ and $\mathrm{Ca}^{2+}$ into the cell; recall from Chapter 11 that movement of cations (positively charged ions) such as $\mathrm{Na}^{+}$and $\mathrm{Ca}^{2+}$ from the outside of the cell to the inside will reduce the magnitude of the inside-negative membrane potential, or depolarize the membrane. These nonselective cation channels open in response to (are "gated" by) binding of the second messenger cyclic guanosine monophosphate, or cGMP (see Figure 15-6). Rod-cell outer segments contain an unusually high concentration $(\sim 0.07 \mathrm{mM})$ of cGMP, which is continuously formed from GTP in a reaction catalyzed by guanylyl cyclase, and thus the cGMP-gated channels are mostly kept in the open state.

Absorption of light by rhodopsin leads to the closing of the nonselective cation channels, causing the membrane potential to become more negative inside. Unlike those associated with the muscarinic acetylcholine receptor discussed earlier, G proteins activated by rhodopsin do not act directly on ion channels. The closing of the cation channels in the rod-cell plasma membrane is caused by a marked reduction in the level of cGMP (Figure 15-20). Light absorption by rhodopsin induces rapid activation of a cGMP phosphodiesterase (PDE), which hydrolyzes cGMP to $5^{\prime}$-GMP; this lightinduced drop in cGMP levels leads to channel closing. This, in turn, causes membrane hyperpolarization and a reduction in neurotransmitter release. The more photons absorbed by rhodopsin, the more cGMP is hydrolyzed, the more channels are closed, the fewer $\mathrm{Na}^{+}$and $\mathrm{Ca}^{2+}$ ions cross the membrane from the outside, the more negative the membrane potential becomes, and the less neurotransmitter is released. The reduction in neurotransmitter release is transmitted to the brain by a series of neurons, where it is perceived as light.

As depicted in Figure 15-20, both $\mathrm{G}_{\mathrm{ss}}{ }^{-}$-GTP and its effector, PDE, are localized via lipid anchors to the cytoplasmic face of the disk membranes of the rod cell. The $\mathrm{G}_{\mathrm{ss}}{ }^{-} \mathrm{GTP}$ complexes that are generated through activation of rhodopsin are able to move laterally along the membrane surface and bind to the two inhibitory $\gamma$ subunits of PDE (see Figure 15-20; note that the stoichiometry is $1: 1$, i.e., one $\mathrm{G}_{\mathrm{ss}}{ }^{-} \mathrm{GTP}$ binds to one $\gamma$ subunit). The binding of $\mathrm{G}_{\mathrm{ss}}{ }^{-} \mathrm{GTP}$ to the $\gamma$ subunits releases the catalytically active $\alpha \beta$ dimer, which then converts cGMP to GMP. This is a clear example of how signal-induced removal of an inhibitory subunit can quickly activate an enzyme, a common mechanism in signaling pathways.

Direct support for the role of cGMP in rod-cell activity has been obtained in patch-clamping studies using isolated patches of rod outer-segment plasma membrane, which contains abundant cGMP-gated cation channels. When cGMP is added to the cytosolic surface of these patches, there is a rapid increase in the number of open cation channels; cGMP binds directly to a site on the channel proteins to keep them open. Like the $\mathrm{K}^{+}$channels discussed in Chapter 11, the cGMP-gated channel protein contains four subunits (see Figure 11-20). In this case, each of the subunits is able to

![li2334542480e](li2334542480e.jpg)

FIGURE 15-20 The light-activated rhodopsin pathway and the closing of cation channels in rod cells. In dark-adapted rod cells, a high level of cGMP keeps cGMP-gated nonselective cation channels open, leading to depolarization of the plasma membrane and neurotransmitter release. Light absorption generates activated rhodopsin, $\mathrm{R}^{\mathrm{a}}$ (step $\mathbf{\square}$ ), which binds inactive, GDP-bound $G_{o i}$ protein and mediates the exchange of GDP for GTP (step $\mathbf{\square}$ ). The free $\mathrm{G}_{\mathrm{o} 1}$-GTP generated then activates PDE by binding to its inhibitory $\gamma$ subunits (step $\mathbf{\square}$ ) and dissociating them from the catalytic $\alpha$ and $\beta$ subunits (step $\mathbf{\square}$ ). Relieved of their inhibition, the $\alpha$ and $\beta$ subunits of PDE
bind a cGMP molecule. Three or four cGMP molecules must bind to each channel in order to open it; this cooperative allosteric interaction makes channel opening very sensitive to small changes in cGMP levels.
$\mathrm{G}_{\mathrm{o} 1}$ contains an intrinsic GTPase activity that hydrolyzes a bound GTP to GDP, producing an inactive $\mathrm{G}_{\mathrm{o} 1}$-GDP complex that can no longer associate with a PDE $\gamma$ subunit. The intrinsic GTPase activity of $\mathrm{G}_{\mathrm{o} 1}$ in rod cells is accelerated by a specific GTPase-activating protein (GAP). In mammals, $\mathrm{G}_{\mathrm{o} 1}$ normally remains in the active GTP-containing state for only a fraction of a second. The rapid hydrolysis of GTP to GDP and subsequent inactivation of $\mathrm{G}_{\mathrm{o} 1}$ triggers release of the bound PDE $\gamma$ subunits, which then rejoin the PDE $\alpha$ and $\beta$ subunits to regenerate an inactive PDE $\alpha \beta \gamma_{2}$ tetramer. Thus PDE rapidly loses its activity, and the cGMP concentration begins to rise to pre-light-induction levels. Through this process, the eye can respond quickly to changing patterns of light associated with moving objects or as the direction of gaze changes.
hydrolyze cGMP to GMP (step $\mathbf{\square}$ ). The resulting decrease in cytosolic cGMP leads to dissociation of cGMP from the cation channels in the plasma membrane and the closing of those channels (step $\mathbf{\square}$ ). The membrane then becomes transiently hyperpolarized, and neurotransmitter release is reduced. The complex of $\mathrm{G}_{\mathrm{o} 1}$-GTP and the PDE $\gamma$ subunits binds a GTPase-activating complex termed RGS9G $\beta 5$ (step $\mathbf{\square}$ ); by hydrolyzing the bound GTP, this complex triggers the physiologically important rapid inactivation of the PDE. See V. Arshavsky and E. Pugh, 1998, Neuron 20:11, and V. Arshavsky, 2002, Trends Neurosci. 25:124.

## Signal Amplification Makes the Rhodopsin Signal Transduction Pathway Exquisitely Sensitive

Remarkably, a single photon absorbed by a resting rod cell produces a measurable response in the form of a small hyperpolarization in the membrane potential of about 1 mV , which in amphibians lasts a second or two. Humans are able to detect a flash of as few as five photons with their hundreds to thousands of rods; such responses to single-photon absorption are highly relevant for night vision. The lightdetecting system is very sensitive because the signal is greatly amplified by the signal transduction pathway. During the time it is active, each rhodopsin molecule in the disk membrane of a rod cell can activate $\sim 500 \mathrm{G}_{\mathrm{o} 1}$ molecules, two of which in turn activate a single phosphodiesterase molecule. Each PDE molecule hydrolyzes hundreds of cGMP molecules during the fraction of a second it remains active. Thus absorption of a single photon-yielding a single activated

rhodopsin molecule-can trigger the closing of thousands of cation channels (or about 5 percent of the open channels) in the plasma membrane and cause a measurable change in the membrane potential of the cell.

## Rapid Termination of the Rhodopsin Signal Transduction Pathway Is Essential for the Temporal Resolution of Vision

As in all G protein-coupled signaling pathways, timely termination of the rhodopsin signaling pathway requires that all the activated intermediates be inactivated rapidly, restoring the system to its basal state, ready for signaling again. Thus the three protein intermediates, activated rhodopsin $\left(R^{*}\right), G_{m} \cdot G T P$, and activated PDE, must all be inactivated, and the concentration of cytoplasmic cGMP must be restored to its dark level by guanylyl cyclase, the enzyme that catalyzes production of cGMP from GMP. The entire process of rhodopsin activation and inactivation in a mammalian rod cell as it responds to a single photon of light takes only about 50 milliseconds; this enables the eye to respond very quickly to changing light conditions. Several mechanisms act together to make possible this very rapid response.

A GTPase Activating Protein (GAP) That Inactivates $\mathrm{G}_{\mathrm{ca}} \cdot \mathrm{GTP}$ First, the complex composed of the inhibitory $\gamma$ subunit of PDE and $\mathrm{G}_{\mathrm{m}} \cdot \mathrm{GTP}$ recruits two additional proteins, RGS9 and G $\beta 5$, that together act as a GAP to enhance the rate of hydrolysis of bound GTP to GDP (step 1, Figure 15-20). Hydrolysis of GTP, in turn, causes release of the PDE $\gamma$ subunit, which rejoins the PDE $\alpha$ and $\beta$ subunits, terminating PDE activation (step 1). Experiments with mice in which
the gene encoding RGS9 had been knocked out showed that this protein is essential for normal inactivation of the signaling cascade in vivo. In individual mouse rod cells, the time required for recovery from a single flash of light increased from 0.2 seconds in the normal mouse to about 9 seconds in the RGS9-deficient mouse. This represents a 45 -fold increase, attesting to the importance of RGS9 as a component of the $\mathrm{G}_{\mathrm{m}} \cdot$ GTP-GAP complex.

## $\mathrm{Ca}^{2+}$-Sensing Proteins That Activate Guanylate Cyclase

Second, light-triggered closing of the cGMP-gated cation channels causes a drop in the cytosolic $\mathrm{Ca}^{2+}$ concentration inside the rod cell. This happens because $\mathrm{Ca}^{2+}$ is continually being transported out of the cells by a type of antiporter protein (the $\mathrm{Na}^{+} / \mathrm{Ca}^{2+}-\mathrm{K}^{+}$exchanger) that is not affected by cGMP levels. The drop in the intracellular $\mathrm{Ca}^{2+}$ concentration is sensed by a class of $\mathrm{Ca}^{2+}$-binding proteins known as guanylate cyclase-activating proteins, which bind to guanylate cyclase and stimulate its activity, thereby elevating the level of cGMP and causing the cGMP-gated ion channels to reopen. This is another example of negative feedback, in which a downstream second message-in this case, low cytosolic $\mathrm{Ca}^{2+}$-acts to inhibit further signaling and thus prevent an overreaction by the cell.

Rhodopsin Phosphorylation and Binding of Arrestin Third, a major process that suppresses and helps to terminate the visual response involves phosphorylation of rhodopsin when it is in its activated ( $\mathrm{R}^{*}$ ), but not its inactivated, or dark (R), form. Rhodopsin kinase (Figure 15-21), a member of a class of GPCR kinases, is the enzyme that catalyzes this phosphorylation reaction. Each opsin molecule has two principal
![li2344542480e](li2344542480e.jpg)

FIGURE 15-21 Inhibition of rhodopsin signaling by rhodopsin kinase. Light-activated rhodopsin $\left(R^{*}\right)$, but not dark-adapted rhodopsin, is a substrate for rhodopsin kinase. The extent of rhodopsin phosphorylation is proportional to the amount of time each rhodopsin molecule spends in the light-activated form; the greater the extent of
the phosphorylation, the more the ability of $\mathrm{R}^{*}$ to activate transducin is reduced. Arrestin binds to the completely phosphorylated opsin, forming a complex that cannot activate transducin at all. See A. Mendez et al., 2000, Neuron 28:153, and V. Arshavsky, 2002, Trends Neurosci. 25:124.

serine phosphorylation sites on its cytosol-facing C-terminal C4 segment; the more sites that are phosphorylated by rhodopsin kinase, the less able $\mathrm{R}^{*}$ is to activate $\mathrm{G}_{\mathrm{ur}}$ and so induce the closing of cGMP-gated cation channels.

The protein arrestin, which binds tightly to rhodopsin only when two or three of the serines are phosphorylated, dramatically speeds up the inactivation process. Arrestin bound to the phosphorylated $\mathrm{R}^{*}$ completely prevents interaction with $\mathrm{G}_{\mathrm{ur}}$, blocking formation of the active $\mathrm{G}_{\mathrm{ur}}$-GTP complex and stopping further activation of PDE. The entire process of rhodopsin phosphorylation and inactivation by arrestin is completed very quickly, within 50 milliseconds. Meanwhile, the phosphates linked to rhodopsin are continuously being removed by a specific rhodopsin phosphatase; this removal causes the dissociation of arrestin and the restoration of rhodopsin to its original, light-sensitive state.

## Rod Cells Adapt to Varying Levels of Ambient Light by Intracellular Trafficking of Arrestin and Transducin

Whereas cone cells are insensitive to low levels of light, rodcell function is saturated by high levels of light. When we move from bright sunlight into a dimly lit room, we initially cannot see the objects around us. The reason is because it takes some time for our rod cells to become sensitized to the low light level in the room; gradually, we are able to see and distinguish objects. During this time interval, our rod cells "turn up" their sensitivity to light. On the other hand, when we move quickly from a dimly lit room out into bright sunshine, the opposite occurs: our rod cells "turn down" their sensitivity to light. As the result of this process, called visual adaptation, rod cells are able to perceive light contrast over a 100,000-fold range of ambient light levels, all the way from a dimly lit room to bright sunlight. This wide range of sensitivities is possible because it is the difference in light levels in the visual field, rather than the absolute amount of light absorbed, that is ultimately sensed by the brain and used to form visual images. As we will see below, it is the subcellular trafficking of two key proteins in the signaling pathway, namely transducin and arrestin, that contribute to the extraordinary range of sensitivity of rod cells.

In dark-adapted rod cells, 80-90 percent of the $\mathrm{G}_{\mathrm{ur}}$ and $\mathrm{G}_{\mathrm{By}}$ transducin subunits are in the outer segments, while fewer than 10 percent of arrestin molecules are localized there (Figure 15-22). This spatial distribution allows maximal activation of the downstream effector PDE and thus maximal sensitivity to small changes in light. But exposure for 10 minutes to moderate daytime intensities of light causes a complete redistribution of these proteins: over 80 percent of the $\mathrm{G}_{\mathrm{ur}}$ and $\mathrm{G}_{\mathrm{By}}$ subunits move out of the outer segment into other parts of the cell, while over 80 percent of the inhibitor arrestin moves into the outer segment.

The mechanism by which these proteins move within the cell is not yet known, but it probably involves microtubuleattached motors that carry proteins and other cargos into and out of different subcellular regions (see Chapter 18). In bright light, the reduction in the level of transducin, with its
![li2354542480e](li2354542480e.jpg)

FIGURE 15-22 Schematic illustration of transducin and arrestin distribution in dark-adapted and light-adapted rod cells. (a) In the dark, most transducin (blue circles) is localized to the outer segment, while most arrestin (green circles) is found in other parts of the cell; in this condition, vision is most sensitive to very low light levels. (b) In bright light, little transducin is found in the outer segment, and abundant arrestin is found there; in this condition, vision is relatively insensitive to small changes in light. The coordinated movement of these proteins contributes to our ability to perceive images over a 100,000-fold range of ambient light levels. See P. Calvert et al., 2006, Trends Cell Biol. 16:560.
$\mathrm{G}_{\mathrm{ur}}$ and $\mathrm{G}_{\mathrm{By}}$ subunits, in the outer segment means that $\mathrm{G}_{\mathrm{ur}}$ is simply not available for binding to activated rhodopsin. As a result, less PDE is activated. At the same time, the increase in the arrestin level in the outer segment means that any activated rhodopsin that is present will become rapidly inactivated. Together, the drop in transducin level and the increase in arrestin level greatly reduce the ability of small increases in light levels to activate the downstream effector PDE; thus only large changes in light levels will be sensed by the rod cells. These protein movements are reversed when the ambient light level is lowered.

## KEY CONCEPTS OF SECTION 15.4

## G Protein-Coupled Receptors That Regulate Ion Channels

- The cardiac muscarinic acetylcholine receptor is a GPCR whose effector protein is a $\mathrm{K}^{+}$channel. Receptor activation releases the $\mathrm{G}_{\beta \gamma}$ subunit, which binds to and opens $\mathrm{K}^{+}$channels (see Figure 15-17). The resulting hyperpolarization of the cell membrane slows the rate of heart muscle contraction.
- Rhodopsin, the photosensitive GPCR in rod cells, comprises the protein opsin linked to 11-cis-retinal. Light-induced isomerization of the 11-cis-retinal moiety produces activated opsin, which then activates the coupled heterotrimeric G protein transducin $\left(\mathrm{G}_{\mathrm{t}}\right)$ by catalyzing exchange of free GTP for bound GDP on the $\mathrm{G}_{\mathrm{nt}}$ subunit (see Figures 15-19 and 15-20).
- The effector protein in the rhodopsin pathway is PDE, which is activated by the $\mathrm{G}_{\mathrm{nt}}$-GTP-mediated release of inhibitory subunits. Reduction in the cGMP level by this enzyme leads to closing of cGMP-gated $\mathrm{Na}^{+} / \mathrm{Ca}^{2+}$ channels, hyperpolarization of the membrane, and decreased release of neurotransmitter (see Figure 15-20).
- Several mechanisms act to terminate visual signaling: GAP proteins inactivate $\mathrm{G}_{\mathrm{nt}}$-GTP, $\mathrm{Ca}^{2+}$-sensing proteins activate guanylate cyclase, and rhodopsin phosphorylation and binding of arrestin inhibit activation of transducin.
- Adaptation to a wide range of ambient light levels is mediated by movements of transducin and arrestin into and out of the rod-cell outer segment, which together modulate the ability of small increases in light levels to activate the downstream effector PDE, and thus the sensitivity of the rod cell in different ambient levels of light.


### 15.5 G Protein-Coupled Receptors That Activate or Inhibit Adenylyl Cyclase

GPCR signal transduction pathways that use adenylyl cyclase as an effector protein and cAMP as a second messenger are found in most mammalian cells, where they regulate cellular functions as diverse as metabolism of fats and sugars, synthesis and secretion of hormones, and muscle contraction. These pathways follow the general GPCR mechanism outlined in Figure 15-14: ligand binding to the receptor activates a coupled heterotrimeric G protein, termed $\mathrm{G}_{\mathrm{ns}}$, that activates an effector protein-in this case, adenylyl cyclase, which synthesizes the diffusible second messenger cAMP from ATP (Figure 15-23). The cAMP, in turn, activates a cAMP-dependent protein kinase that phosphorylates specific target proteins. In mammals, more than 30 different GPCRs activate $\mathrm{G}_{\mathrm{ns}}$ and adenylyl cyclase; most cell types express one or more such GPCRs. As detailed in Classic Experiment 15-1, early studies of the mechanism of activation of adenylyl cyclase led to the discovery of the role of the first GTPbinding protein in receptor signaling.
![li2364542480e](li2364542480e.jpg)

FIGURE 15-23 Synthesis and hydrolysis of cAMP by adenylyl cyclase and PDE. Similar reactions occur for production of cGMP from GTP and hydrolysis of cGMP.

To explore this GPCR/cAMP pathway, we focus on the first such pathway discovered: the hormone-stimulated generation of glucose-1-phosphate from glycogen, a storage polymer of glucose (Figure 15-24). The breakdown of glycogen (glycogenolysis), which occurs in muscle and liver cells in response to hormones such as epinephrine and glucagon, is a principal way in which glucose is made available to cells in need of energy. This example shows how activation of a GPCR can stimulate or inhibit several intracellular enzymes, all coordinated to carry out a physiologically important task: glycogen metabolism.

## Adenylyl Cyclase Is Stimulated and Inhibited by Different Receptor-Ligand Complexes

Under conditions where body demand for glucose is high, such as low blood sugar, glucagon is released by the $\alpha$ cells of the pancreatic islets; in case of sudden danger, epinephrine is released by the adrenal glands. Both glucagon and epinephrine signal liver and muscle cells to depolymerize glycogen, releasing individual glucose molecules. In the liver, glucagon

![li2374542480e](li2374542480e.jpg)

FIGURE 15-24 Synthesis and degradation of glycogen. Incorporation of glucose from GDP-glucose into glycogen is catalyzed by glycogen synthase. Removal of glucose units from glycogen is catalyzed
by glycogen phosphorylase. Because two different enzymes catalyze the formation and the degradation of glycogen, the two reactions can be independently regulated.
![li2384542480e](li2384542480e.jpg)

FIGURE 15-25 Hormone-induced activation and inhibition of adenylyl cyclase in adipose cells. Ligand binding to $\mathrm{G}_{\mathrm{int}}$-coupled receptors causes activation of adenylyl cyclase, whereas ligand binding to $\mathrm{G}_{\mathrm{ss}}$-coupled receptors causes inhibition of the enzyme. The $\mathrm{G}_{\mathrm{gl}}$, subunit in stimulatory and in inhibitory G proteins is identical; the $\mathrm{G}_{\mathrm{s}}$, subunits and their corresponding receptors differ. Ligand-stimulated formation of active $\mathrm{G}_{\mathrm{ss}}$-GTP complexes occurs by the same mechanism in both $\mathrm{G}_{\mathrm{int}}$ and $\mathrm{G}_{\mathrm{ss}}$ proteins (see Figure 15-14). However, $\mathrm{G}_{\mathrm{int}}$-GTP and $\mathrm{G}_{\mathrm{ss}}$-GTP interact differently with adenylyl cyclase, so that one stimulates and the other inhibits its catalytic activity. See A. G. Gilman, 1984, Cell 36:577.
and epinephrine bind to different G protein-coupled receptors, but both receptors interact with and activate the same stimulatory $\mathrm{G}_{\mathrm{int}}$ protein, which activates adenylyl cyclase (Figure 15-25). Hence both hormones induce the same metabolic responses. Activation of adenylyl cyclase, and thus the increase in the cAMP level, is proportional to the total concentration of $\mathrm{G}_{\mathrm{int}}$-GTP resulting from the binding of each hormone to its respective receptor.

Positive (activation) and negative (inhibition) regulation of adenylyl cyclase activity occurs in many cell types, providing fine-tuned control of the cAMP level and thus of the downstream cellular response (see Figure 15-25). For example, in adipose cells, the breakdown of triacylglycerols (page 49) to fatty acids and glycerol (lipolysis) for use as a fuel by other body cells is stimulated by binding of epinephrine, glucagon, or adrenocorticotropic hormone

(ACTH) to distinct GPCRs, all of which activate $\mathrm{G}_{\mathrm{ox}}$ and thus activate adenylyl cyclase. Conversely, binding of two other hormones, prostaglandin $\mathrm{E}_{1}\left(\mathrm{PGE}_{1}\right)$ and adenosine, to their respective G protein-coupled receptors inhibits adenylyl cyclase. The prostaglandin and adenosine receptors activate an inhibitory $\mathrm{G}_{\mathrm{i}}$ protein that contains a different $\alpha$ subunit $\left(\mathrm{G}_{\mathrm{ox}}\right)$. After the active $\mathrm{G}_{\mathrm{ox}}$-GTP complex dissociates from $\mathrm{G}_{\mathrm{py}}$, it binds to and inhibits (rather than stimulates) adenylyl cyclase, resulting in lower cAMP levels.

## Structural Studies Established How $\mathrm{G}_{\mathrm{ox}}$-GTP Binds to and Activates Adenylyl Cyclase

X-ray crystallographic analysis has pinpointed the regions in $\mathrm{G}_{\mathrm{ox}}$-GTP that interact with adenylyl cyclase. This enzyme is a multipass transmembrane protein with two large cytosolic domains, each of which is a catalytic domain that binds ATP and converts it to cAMP (Figure 15-26a). Because such transmembrane proteins are notoriously difficult to crystallize, scientists prepared two soluble protein fragments from different catalytic domains of adenylyl cyclase, which tightly associated with each other in a refolded and catalytically active adenylyl cyclase enzyme. When these segments were allowed to associate in the presence of both $\mathrm{G}_{\mathrm{ox}}$-GTP and forskolin (a plant chemical that binds to and activates adenylyl cyclase), they could be stabilized in their active catalytic conformations.

The resulting water-soluble complex (an adenylyl cyclase catalytic domain with $\mathrm{G}_{\mathrm{ox}}$-GTP and forskolin) had a cAMPsynthesizing enzymatic activity similar to that of intact, full-length adenylyl cyclase. In this complex, two regions of $\mathrm{G}_{\mathrm{ox}}$-GTP-the switch II helix and the $\alpha 3-\beta 5$ loop-contact the adenylyl cyclase domain (Figure 15-26b). These contacts are thought to be responsible for the activation of the enzyme by $\mathrm{G}_{\mathrm{ox}}$-GTP. Recall that switch II is one of the segments of a $\mathrm{G}_{\alpha}$ subunit whose conformation is different in the GTP-bound and GDP-bound states (see Figure 15-5). The GTP-induced conformation of $\mathrm{G}_{\mathrm{ox}}$ that favors its dissociation from $\mathrm{G}_{\mathrm{py}}$ is precisely the conformation essential for the binding of $\mathrm{G}_{\mathrm{ox}}$ to adenylyl cyclase.

## cAMP Activates Protein Kinase A by Releasing Inhibitory Subunits

The second messenger cAMP, synthesized by adenylyl cyclase, transduces a wide variety of physiological signals in different cell types in multicellular animals. Virtually all of the diverse effects of cAMP are mediated through activation of protein kinase A (PKA), also called cAMP-dependent protein kinase, which phosphorylates multiple intracellular target proteins expressed in different cell types. Inactive PKA is a tetramer consisting of two regulatory (R) subunits and two catalytic (C) subunits (Figure 15-27a). Each R subunit contains a pseudosubstrate domain whose sequence resembles that of a peptide substrate and binds to the active site in the catalytic domain but is not phosphorylated; thus the pseudosubstrate domain inhibits the activity of the catalytic
(a)
![li2394542480e](li2394542480e.jpg)

FIGURE 15-26 Activation of the catalytic domain of mammalian adenylyl cyclase by binding to $\mathrm{G}_{\mathrm{ox}}$-GTP. (a) Schematic diagram of mammalian adenylyl cyclase. The membrane-bound enzyme contains two similar catalytic domains, which convert ATP to cAMP, on the cytosolic face of the membrane, and two integral membrane domains, each of which is thought to contain six transmembrane $\alpha$ helices. (b) Model of the three-dimensional structure of $\mathrm{G}_{\mathrm{ox}}$-GTP complexed with two fragments of catalytic domains that reconstituted in vitro one functional adenylyl cyclase catalytic domain, as determined by x-ray crystallography. A newly-formed cAMP is shown in green. The $\alpha 3-\beta 5$ loop and the helix in the switch II region (blue) of $\mathrm{G}_{\mathrm{ox}}$-GTP interact simultaneously with a specific region of adenylyl cyclase. GTP (yellow) is bound to the GTP-binding domain, which is similar in structure to Ras (see Figure 15-5). The two adenylyl cyclase fragments are shown in red and pink. [Part (b) data from J. J. G. Tesmer et al., 1997, Science 278:1907, PDB ID 1azs.]
subunits. Inactive PKA is turned on by binding of cAMP. Each R subunit has two distinct cAMP-binding sites, called CNB-A and CNB-B (Figure 15-27b). Binding of cAMP to both sites causes a conformational change in the R subunit, including its pseudosubstrate domain, so that it can no longer bind to and inhibit the catalytic domain, and thus releases it, instantly activating its kinase activity (Figure 15-27c).

Binding of cAMP by an R subunit of PKA occurs in a cooperative fashion; that is, binding of the first cAMP molecule to CNB-B lowers the $K_{\mathrm{d}}$ for binding of the second cAMP to CNB-A. Thus small changes in the level of cytosolic cAMP can

![li2404542480e](li2404542480e.jpg)

FIGURE 15-27 Structure of PKA and its activation by cAMP. (a) PKA consists of two regulatory (R) subunits and two catalytic (C) subunits. When cAMP (red triangle) binds to the regulatory subunit, the catalytic subunit is released, thus activating PKA. (b) The two regulatory subunits form a dimer, joined by a dimerization/docking domain and a flexible linker to which an A kinase-associated protein (AKAP; see Figure 15-31) can bind. Each R subunit has two cAMP-binding domains, CNB-A and CNB-B, and a binding site for a catalytic subunit (arrow). (c) Binding of cAMP to the CNB-A domain causes a subtle conformational change that displaces the C subunit from the R subunit, leading to its activation. Without bound cAMP, one loop of the CNB-A domain (purple) is in a conformation that can bind the catalytic (C) subunit. A glutamate (E200) and arginine (R209) residue participate in binding of cAMP (red), which causes a conformational change (green) in the loop that prevents binding of the loop to the C subunit. (Part (b) cAMP, CNB-A, and CNB-B data, and part (c) cAMP bound data from Y. Su et al., 1995, Science 269:807, PDB ID 1gs. Part (b) docking domain data from P. Banky et al., 2003, J. Mol. Biol. 330:1117, PDB ID 2ezw. Part (c) catalytic subunit bound data from C. Kim, N. H. Xuong, and S. S. Taylor, 2005, Science 307:690, PDB ID 1u7e.)
cause proportionately large changes in the number of dissociated C subunits and, hence, in cellular kinase activity. Rapid activation of enzymes by signal-triggered dissociation of an inhibitor is a common feature of many signaling pathways.

## Glycogen Metabolism Is Regulated by Hormone-Induced Activation of PKA

Like all biopolymers, glycogen is synthesized by one set of enzymes and degraded by another (see Figure 15-24). Degradation of glycogen, or glycogenolysis, involves the stepwise removal of glucose residues from one end of the polymer by a phosphorolysis reaction, catalyzed by glycogen phosphorylase $(G P)$, yielding glucose-1-phosphate.

In both muscle and liver cells, glucose-1-phosphate produced from glycogen is converted by an enzyme to glucose-6-phosphate. In muscle cells, this metabolite enters the glycolytic pathway and is metabolized to generate ATP for use in powering muscle contraction (see Chapters 12 and 17). Unlike muscle cells, liver cells contain a phosphatase that hydrolyzes glucose-6-phosphate to glucose, which is exported from these cells mainly by a glucose transporter (GLUT2) in the plasma membrane (see Chapter 11). Thus glycogen stores in the liver are primarily broken down to glucose, which is immediately released into the blood and transported to other tissues, particularly the muscles and brain, to nourish them. Glycogenolysis in both types of cells is induced by rises in blood epinephrine as part of the fight-or-flight response.

Activated PKA enhances the conversion of glycogen to glucose-1-phosphate in two ways: by inhibiting glycogen synthesis and by stimulating glycogen degradation (Figure 15-28a). PKA directly phosphorylates and, in so doing, inactivates glycogen synthase (GS), the enzyme that synthesizes glycogen. PKA promotes glycogen degradation indirectly by phosphorylating and thus activating an intermediate kinase, glycogen phosphorylase kinase (GPK). In turn, active GPK phosphorylates and activates GP, the enzyme that degrades glycogen.

Skeletal muscle GPK is a huge protein of subunit composition $(\alpha \beta \gamma \delta)_{4}$. The $\gamma$ subunit contains the kinase catalytic activity and the others are regulatory; the $\delta$ subunit is the ubiquitous protein calmodulin, which has four calcium ion binding sites (see Figure 3-33). GPK enzyme activity is increased both by phosphorylation of the $\alpha$ and $\beta$ subunits by PKA and by $\mathrm{Ca}^{2+}$ binding to the $\delta$ subunit; maximal activity requires both stimulators.

All three enzymes-GS, GPK, and GP—are counteracted by a phosphatase called phosphoprotein phosphatase (PP). At high cAMP levels, PKA phosphorylates an inhibitor of phosphoprotein phosphatase (IP), which keeps this phosphatase in its inactive state (see Figure 15-28a, right).

The entire process is reversed when epinephrine or another hormone activating $\mathrm{G}_{\mathrm{as}}$ is removed and the level of
cAMP drops, inactivating PKA. When PKA is inactive, it can no longer phosphorylate IP, so PP becomes active (Figure 15-28b). PP removes the phosphate residues previously added by PKA to GS and GPK, as well as the phosphates on GP added by GPK. As a consequence, the synthesis of glycogen by GS is enhanced and the degradation of glycogen by GP is inhibited.

Epinephrine-induced glycogenolysis thus exhibits dual regulation: activation of the enzymes catalyzing glycogen degradation and inhibition of enzymes promoting glycogen synthesis. Such dual regulation provides an efficient mechanism for regulating a particular cellular response and is a common phenomenon in cell biology.

## cAMP-Mediated Activation of PKA Produces Diverse Responses in Different Cell Types

In adipose cells, epinephrine-induced activation of PKA promotes phosphorylation and activation of the lipase that hydrolyzes stored triglycerides to yield free fatty acids and glycerol. These fatty acids are released into the blood and taken up as an energy source by cells in other tissues, such as the kidney, heart, and muscles (see Chapter 12). Therefore, activation of PKA by epinephrine in two different cell types, hepatic and adipose cells, has different effects. Indeed, cAMP
(a) Increased cAMP; increased glycogen breakdown, and decreased glycogen synthesis
![li2414542480e](li2414542480e.jpg)

FIGURE 15-28 Regulation of glycogen metabolism by cAMP and PKA. Active enzymes are highlighted in darker shades; inactive forms, in lighter shades. (a) An increase in cytosolic cAMP activates PKA, which phosphorylates glycogen synthase (GS) and thus inhibits glycogen synthesis directly. Active PKA also promotes glycogen degradation via a protein kinase cascade. At high cAMP concentrations, PKA also
phosphorylates an inhibitor of phosphoprotein phosphatase (PP). Binding of the phosphorylated inhibitor to PP prevents this phosphatase from de-phosphorylating the activated enzymes in the kinase cascade or the inactive glycogen synthase. (b) A decrease in cAMP inactivates PKA, leading to release of the active form of PP. The activation of PP promotes glycogen synthesis and inhibits glycogen degradation.

TABLE 15-3 Cellular Responses to Hormone-Induced Rise in cAMP in Various Tissues*

| Tissue | Hormone Inducing Rise in cAMP | Cellular Response |
| :--: | :--: | :--: |
| Adipose | Epinephrine; ACTH; glucagon | Increase in hydrolysis of triglyceride; decrease in amino acid uptake |
| Liver | Epinephrine; norepinephrine; glucagon | Increase in conversion of glycogen to glucose; inhibition of glycogen synthesis; increase in amino acid uptake; increase in gluconeogenesis (synthesis of glucose from amino acids) |
| Ovarian follicle | FSH; LH | Increase in synthesis of estrogen, progesterone |
| Adrenal cortex | ACTH | Increase in synthesis of aldosterone, cortisol |
| Cardiac muscle | Epinephrine | Increase in contraction rate |
| Thyroid gland | TSH | Secretion of thyroxine |
| Bone | Parathyroid hormone | Increase in resorption of calcium from bone |
| Skeletal muscle | Epinephrine | Conversion of glycogen to glucose-1-phosphate |
| Intestine | Epinephrine | Fluid secretion |
| Kidney | Vasopressin | Resorption of water |
| Blood platelets | Prostaglandin I | Inhibition of aggregation and secretion |

*Nearly all the effects of cAMP are mediated through PKA, which is activated by binding of cAMP.
source: Data from E. W. Sutherland, 1972, Science 177:401.
and PKA mediate a large array of hormone-induced cellular responses in multiple tissues (Table 15-3).

Although PKA acts on different substrates in different types of cells, it always phosphorylates a serine or threonine residue that occurs within the same sequence motif: X -Arg-(Arg/Lys)-X-(Ser/Thr)- $\Phi$, where X denotes any amino acid and $\Phi$ denotes a hydrophobic amino acid. Other serine/ threonine kinases phosphorylate target residues within other sequence motifs.

## Signal Amplification Occurs in the cAMP-PKA Pathway

We've seen that receptors such as the $\beta$-adrenergic receptor are low-abundance proteins, typically present in only a few hundred or thousand copies per cell. Yet the cellular responses induced by a hormone such as epinephrine can require production of large numbers of cAMP and activated enzyme molecules per cell. As an example, following activation of $\mathrm{G}_{\mathrm{as}}$-coupled receptors, the intracellular concentration of cAMP rises to about $10^{-6} \mathrm{M}$. In a typical cell, that is roughly a cube about $15 \mu \mathrm{~m}$ on a side; this concentration is equivalent to 2 million molecules of cAMP per cell. Thus substantial amplification of the signal is necessary if it is to induce a significant cellular response. We have already seen how signal amplification occurs following photon absorption in rod cells. In the case of G protein-coupled hormone receptors, signal amplification is possible in part because both receptors and G proteins can diffuse rapidly in the plasma membrane. A single epinephrine-GPCR complex causes conversion of up to a hundred inactive $\mathrm{G}_{\mathrm{as}}$ molecules
to the active form before epinephrine dissociates from the receptor. Each active $\mathrm{G}_{\mathrm{as}}$-GTP, in turn, activates a single adenylyl cyclase molecule, which then catalyzes the synthesis of many cAMP molecules during the time $\mathrm{G}_{\mathrm{as}}$-GTP is bound to it.

The amplification that occurs in such a signal transduction cascade depends on the number of steps in it and on the relative concentrations of the various components. In the epinephrine-induced cascade shown in Figure 15-7 and Figure 15-29, for example, blood levels of epinephrine as low as $10^{-10} \mathrm{M}$ can stimulate liver glycogenolysis and release of glucose. An epinephrine stimulus of this magnitude generates an intracellular cAMP concentration of $10^{-6} \mathrm{M}$, an amplification of $10^{4}$-fold. Because three more catalytic steps precede the release of glucose, another $10^{4}$ amplification can occur, resulting in a $10^{8}$ amplification of the epinephrine signal. In striated muscle, the amplification is less dramatic because the concentrations of the three successive enzymes in the glycogenolytic cascade-PKA, GPK, and GP—are in a 1:10:240 ratio (a potential 240-fold maximal amplification).

## CREB Links cAMP and PKA to Activation of Gene Transcription

Activation of PKA also stimulates the expression of many genes, leading to long-term effects on cells. For instance, in hepatic cells, PKA induces the expression of several enzymes involved in gluconeogenesis-the conversion of three-carbon compounds such as pyruvate (see Figure 12-3) to glucose-thus increasing the concentration of

![li2424542480e](li2424542480e.jpg)

FIGURE 15-29 Amplification of an extracellular signal by a signal transduction pathway involving cAMP and PKA. The specific example here is the pathway depicted in Figure 15-7. Binding of a single epinephrine molecule to one G protein-coupled receptor induces activation of several molecules of adenylyl cyclase, the enzyme that catalyzes the synthesis of cyclic AMP, and each of these enzyme molecules synthesizes a large number of cAMP molecules, the first level of amplification. Two molecules of cAMP activate one PKA, but each activated PKA phosphorylates and activates multiple target proteins. This second level of amplification may involve several sequential reactions in which the product of one reaction activates the enzyme catalyzing the next reaction. The more steps in such a cascade, the greater the signal amplification possible.
glucose in the blood and enhancing the short-term effects of activated PKA.

All genes regulated by PKA contain a cis-acting DNA sequence, the cAMP-response element (CRE), that binds the phosphorylated form of a transcription factor called CREbinding (CREB) protein, which is found only in the nucleus. Following the elevation of cAMP levels and the release of active PKA catalytic subunits, some of the catalytic subunits translocate to the nucleus. There they phosphorylate serine-133 on the CREB protein. Phosphorylated CREB binds to CRE-containing target genes and also binds to a co-activator termed CBP/P300 (see Figure 9-31). CBP/P300 links CREB to RNA polymerase II and other gene regulatory proteins, thereby stimulating gene transcription (Figure 15-30).

## Anchoring Proteins Localize Effects of cAMP to Specific Regions of the Cell

In many cell types, a rise in the cAMP level may produce a response that is required in one part of the cell but is unneeded, or perhaps deleterious, in another. Anchoring proteins localize members of the PKA family to specific subcellular locations, thereby restricting cAMP-dependent responses to these locations. Each of the roughly 50 such proteins, referred to as A kinase-associated proteins (AKAPs), has a two-domain structure; one domain confers a specific subcellular location
![li2434542480e](li2434542480e.jpg)

FIGURE 15-30 Activation of CREB transcription factor following ligand binding to $\mathrm{G}_{\mathrm{sq}}$-coupled GPCRs. Receptor stimulation $\square$ leads to activation of PKA $\square$ Catalytic subunits of PKA translocate to the nucleus $\square$ and there phosphorylate and activate the CREB transcription factor $\square$. Phosphorylated CREB associates with the co-activator CBP/ P300 $\square$ and other proteins to stimulate transcription of the various target genes controlled by a CRE regulatory element. See K. A. Lee and N. Masson, 1993, Biochim. Biophys. Acta 1174:221, and D. Parker et al., 1996, Mol. Cell Biol. 16(2):694.
and the other binds to the regulatory (R) subunit of PKA. AKAPs regulate cAMP and PKA signaling within the cell both spatially and temporally.

One AKAP in heart muscle anchors both PKA and PDE-the enzyme that hydrolyzes cAMP to AMP (see Figure 15-23)-to the outer nuclear membrane (Figure 15-31). Because of the close proximity of these two proteins, negative feedback provides tight local control of the cAMP concentration and hence of local PKA activity. As cAMP levels rise in response to hormone stimulation, PKA becomes activated and phosphorylates and activates several target proteins, including PDE. Active PDE, in turn, hydrolyzes cAMP, thus quickly returning PKA to its inactive state. The localization of PKA near the nuclear membrane also facilitates entry of its catalytic subunits into the nucleus, where they phosphorylate and activate the CREB transcription factor (see Figure 15-30).

![li2444542480e](li2444542480e.jpg)

FIGURE 15-31 Localization of PKA and PDE to the nuclear membrane in heart muscle by an A kinase-associated protein (AKAP). This member of the AKAP family, designated mAKAP, anchors both PDE and the regulatory subunit ( $R$, see Figure 15-27b) of PKA to the nuclear membrane, maintaining them in a negative feedback loop that provides close local control of the cAMP level and PKA activity. Step $\mathbf{B}$ The basal level of PDE activity in the absence of hormone (resting state) keeps cAMP levels below those necessary for PKA activation. Steps $\mathbf{B}$ and $\mathbf{B}$ Activation of $\beta$-adrenergic receptors causes an increase in cAMP to a
level in excess of that which can be degraded by PDE. The resulting binding of cAMP to the R subunits of PKA releases the active catalytic (C) subunits into the cytosol. Some C subunits enter the nucleus, where they phosphorylate and thus activate certain transcription factors (see Figure 15-30). Other C subunits phosphorylate PDE, stimulating its catalytic activity. Active PDE hydrolyzes cAMP, thereby driving cAMP levels back to basal levels and causing re-formation of the inactive PKA C-R complex. Step $\mathbf{B}$ Subsequent de-phosphorylation of PDE returns the complex to the resting state. See K. L. Dodge et al., 2001, EMBO J. 20:1921.

In certain heart muscle cells, a different AKAP is tethered to the cytosolic face of the plasma membrane near a particular type of gated $\mathrm{Ca}^{2+}$ channel. In the heart, activation of $\beta$-adrenergic receptors by epinephrine (as part of the fight-or-flight response) leads to PKA-catalyzed phosphorylation of these $\mathrm{Ca}^{2+}$ channels, causing them to open; the resulting influx of $\mathrm{Ca}^{2+}$ increases the rate of heart muscle contraction. The binding of AKAP to PKA localizes the kinase next to these channels, thereby reducing the time that would otherwise be required for diffusion of PKA catalytic subunits from their sites of generation to their $\mathrm{Ca}^{2+}$-channel substrates.

## Multiple Mechanisms Suppress Signaling from the GPCR/cAMP/PKA Pathway

For cells to respond effectively to changes in their environment, they must not only activate a signaling pathway, but also down-modulate or terminate the response once it is no longer needed; otherwise, signal transduction pathways would remain "on" too long, or at too high a level, and the cell would become overstimulated.

Earlier we saw that multiple mechanisms can rapidly terminate the rhodopsin signal transduction pathway, including GAP proteins that stimulate the hydrolysis of GTP bound to $\mathrm{G}_{\mathrm{ssr}}, \mathrm{Ca}^{2+}$-sensing proteins that activate guanylate cyclase, and phosphorylation of active rhodopsin by rhodopsin kinase followed by binding of arrestin (see Figure 15-21). In fact, most G protein-coupled receptors are modulated by multiple mechanisms that down-regulate their activity, as is exemplified by $\beta$-adrenergic receptors and others coupled to $\mathrm{G}_{\mathrm{ss}}$ that activate adenylyl cyclase.

- First, the intrinsic GTPase activity of $\mathrm{G}_{\mathrm{ss}}$ converts the bound GTP to GDP, thus terminating its ability to activate its
downstream target adenylyl cyclase. Importantly, the rate of hydrolysis of GTP bound to $\mathrm{G}_{\mathrm{ss}}$ is enhanced when $\mathrm{G}_{\mathrm{ss}}$ binds to adenylyl cyclase, lessening the duration of cAMP production; thus adenylyl cyclase functions as a GAP for $\mathrm{G}_{\mathrm{ss}}$-GTP. More generally, binding of most, if not all, $\mathrm{G}_{\mathrm{ss}}$-GTP complexes to their respective effector proteins accelerates the rate of GTP hydrolysis.
- Second, PDE acts to hydrolyze cAMP to $5^{\prime}$-AMP, terminating the cellular response. Thus the continuous presence of hormone at a high enough concentration is required for continuous activation of adenylyl cyclase and maintenance of an elevated cAMP level. Once the hormone concentration falls sufficiently, the cAMP level falls and all cellular responses quickly terminate.

Most GPCRs are also down-regulated by feedback repression, in which an end product of a signaling pathway blocks an early step in that pathway. For instance, when a $\mathrm{G}_{\mathrm{ss}}$ protein-coupled receptor is exposed to hormonal stimulation for several hours, several serine and threonine residues in the cytosolic domain of the receptor become phosphorylated by PKA. The phosphorylated receptor can bind its ligand, but cannot efficiently activate $\mathrm{G}_{\mathrm{ss} 1}$ thus ligand bound to the phosphorylated receptor is less efficient in activating adenylyl cyclase then is ligand bound to the nonphosphorylated receptor. Because the activity of PKA is enhanced by the high cAMP level induced by any hormone that activates $\mathrm{G}_{\mathrm{ssr}}$ prolonged exposure to one such hormone-say, epinephrine-desensitizes not only $\beta$-adrenergic receptors, but also other $\mathrm{G}_{\mathrm{ss}}$ protein-coupled receptors that are phosphorylated by PKA, even though they bind different ligands (e.g., glucagon receptors in the liver). This cross-regulation is called heterologous desensitization.

Several residues in the cytosolic domain of the $\beta$-adrenergic receptor, different from those phosphorylated by PKA, are phosphorylated by the enzyme $\beta$-adrenergic receptor kinase (BARK), but only when epinephrine or an agonist is bound to the receptor and thus the receptor is in its active conformation. BARK is a member of the same kinase family as rhodopsin kinase, and its action is similar to the phosphorylation and down-modulation of activated rhodopsin by rhodopsin kinase (see Figure 15-21). This process is termed homologous desensitization because only those receptors that are in their active conformations are subject to deactivation by phosphorylation.

We noted that binding of arrestin to extensively phosphorylated opsin completely inhibits activation of coupled G proteins by activated opsin (see Figure 15-21). A related protein, termed $\beta$-arrestin, plays a similar role in silencing other G protein-coupled receptors, including $\beta$-adrenergic receptors (Figure 15-32).

An additional function of $\beta$-arrestin in regulating cellsurface receptors was initially suggested by the observation that disappearance of $\beta$-adrenergic receptors from the cell surface in response to ligand binding is stimulated by overexpression of BARK and $\beta$-arrestin. Subsequent studies revealed that $\beta$-arrestin binds not only to phosphorylated GPCRs, but also to clathrin and an associated protein termed AP2, two key components of the coated vesicles that are involved in endocytosis from the plasma membrane (see Figure 15-32; see also Chapter 14). These interactions promote the formation of coated pits and
(a)
![li2454542480e](li2454542480e.jpg)

FIGURE 15-32 Binding of $\beta$-arrestin to phosphorylated GPCRs triggers receptor desensitization and activation of several different signal transduction proteins. (a) $\beta$-Arrestin binds to specific phosphorylated serine and threonine residues in the C-terminal segment of G proteincoupled receptors (GPCRs). Clathrin and AP2, two other proteins bound by $\beta$-arrestin, promote endocytosis of the receptor (see Figure 14-29). $\beta$-Arrestin also functions in transducing signals from activated receptors by binding to and activating several cytosolic protein kinases. Src activates the MAP kinase pathway, leading to phosphorylation of key transcription
endocytosis of the associated receptors, thereby decreasing the number of receptors exposed on the cell surface. Eventually some of the internalized receptors are degraded intracellularly, and some are de-phosphorylated in endosomes. Following dissociation of $\beta$-arrestin, the resensitized (de-phosphorylated) receptors are recycled to the cell surface in a manner similar to the recycling of the LDL receptor (see Chapter 14).

In addition to its role in regulating receptor activity, $\beta$-arrestin functions as an adapter protein in transducing signals from G protein-coupled receptors to the nucleus (see Chapter 16). The GPCR-arrestin complex acts as a scaffold for the binding and activation of several cytosolic kinases (see Figure 15-32), which we discuss in detail in subsequent chapters. These kinases include Src, a cytosolic protein tyrosine kinase that activates the MAP kinase pathway and other pathways leading to the transcription of genes needed for cell division (see Chapters 16 and 19). A complex of three arrestin-bound proteins, including a Jun N-terminal kinase (JNK-3), initiates a kinase cascade that ultimately activates the Jun transcription factor, which promotes expression of certain growth-promoting enzymes and other proteins that help cells respond to stresses. Thus the BARK- $\beta$-arrestin pathway, originally just thought to suppress signaling by GPCRs, actually functions as a switch, turning off signaling by G proteins and turning on other signaling pathways. The multiple functions of $\beta$-arrestin illustrate the importance of adapter proteins in both regulating signaling and transducing signals from cell-surface receptors.
![li2464542480e](li2464542480e.jpg)
factors (see Chapter 16). Interaction of $\beta$-arrestin with three other proteins, including JNK-3 (a Jun N-terminal kinase), results in phosphorylation and activation of another transcription factor, Jun. See W. Miller and R. J. Lefkowitz, 2001, Curr. Opin. Cell Biol. 13:139, and K. Pierce et al., 2002, Not. Rev. Mol. Cell Biol. 3:639. (b) Three-dimensional structure of rhodopsin bound to arrestin. Arrestin binds to segments of the C-terminal cytosolic alpha helix of activated rhodopsin that includes the two phosphorylated resides as well as to parts of transmembrane helix 7. [Part (b) data from Y. Kang et al., 2015, Nature 523:561-567, PDB ID 4zwj, and custom PDB.]

## KEY CONCEPTS OF SECTION 15.5

## G Protein-Coupled Receptors That Activate or Inhibit Adenylyl Cyclase

- Ligand binding by G protein-coupled receptors that activate $\mathrm{G}_{\mathrm{ss}}$ results in the activation of the membrane-bound enzyme adenylyl cyclase, which converts ATP to the second messenger cAMP (see Figure 15-23). Ligand binding of G proteincoupled receptors that activate $\mathrm{G}_{\mathrm{ssi}}$ results in the inhibition of adenylyl cyclase and lower levels of cAMP (see Figure 15-25).
- $\mathrm{G}_{\mathrm{ss}}$-GTP and $\mathrm{G}_{\mathrm{ss}}$-GTP bind to the catalytic domain in adenylyl cyclase to activate or inhibit the enzyme, respectively (see Figures 15-25 and 15-26).
- cAMP binds cooperatively to a regulatory subunit of PKA, releasing the active kinase catalytic subunit (see Figure 15-27).
- In liver and muscle cells, activation of PKA induced by epinephrine and other hormones exerts a dual effect, inhibiting glycogen synthesis and stimulating glycogen breakdown via a kinase cascade (see Figure 15-28), leading to an increase in glucose for production of ATP.
- PKA mediates the diverse effects of cAMP in most cells (see Table 15-3). The substrates for PKA, and thus the cellular responses to hormone-induced activation of PKA, vary among cell types.
- The signal that activates the GPCR/adenylyl cyclase/ cAMP/PKA signaling pathway is amplified tremendously by second messengers and kinase cascades (see Figures 15-7 and15-29).
- Activation of PKA often leads to phosphorylation of nuclear CREB protein, which, together with the CBP/P300 co-activator, stimulates transcription of genes, thus initiating a long-term change in the cell's protein composition (see Figure 15-30).
- Localization of PKA to specific regions of the cell by anchoring proteins restricts the effects of cAMP to particular subcellular locations (see Figure 15-31).
- Signaling from $\mathrm{G}_{\mathrm{s}}$-coupled receptors is down-regulated by multiple mechanisms: first, the intrinsic GTPase activity of $\mathrm{G}_{\mathrm{ss}}$ that converts the bound GTP to GDP is enhanced when $\mathrm{G}_{\mathrm{ss}}$ binds to adenylyl cyclase (this occurs when many $\mathrm{G}_{\mathrm{s}}$-GTP complexes bind to their respective effector proteins); and second, PDE acts to hydrolyze cAMP to $5^{\prime}$-AMP, terminating the cellular response.
- Most GPCRs are also regulated by feedback repression, in which the end product of a pathway (e.g., PKA) blocks an early step in the pathway. As with rhodopsin, binding of $\beta$-arrestin to phosphorylated $\beta$-adrenergic receptors completely inhibits activation of coupled G proteins (see Figure 15-32).
- $\beta$-adrenergic receptors are deactivated by $\beta$-adrenergic kinase (BARK), which phosphorylates cytosolic residues of the receptor in its active conformation. BARK phosphorylation of ligand-bound $\beta$-adrenergic receptors also leads to the binding of $\beta$-arrestin and endocytosis of the receptors.

The consequent reduction in the number of cell-surface receptors renders the cell less sensitive to additional hormone.

- The GPCR-arrestin complex functions as a scaffold that activates several cytosolic kinases, initiating cascades that lead to transcriptional activation of many genes controlling cell growth (see Figure 15-32).


### 15.6 G Protein-Coupled Receptors That Trigger Elevations in Cytosolic and Mitochondrial Calcium

Calcium ions play an essential role in regulating cellular responses to many signals, and many GPCRs and other types of receptors exert their effects on cells by influencing the cytosolic concentration of $\mathrm{Ca}^{2+}$. As we saw in Chapter 11, the level of $\mathrm{Ca}^{2+}$ in the cytosol is tightly maintained at a submicromolar level $(0.1 \mu \mathrm{M}=100 \mathrm{nM})$ by the continuous action of ATP-powered $\mathrm{Ca}^{2+}$ pumps and $\mathrm{Na}^{+} / \mathrm{Ca}^{2+}$ antiporters, which transport $\mathrm{Ca}^{2+}$ ions against their concentration gradient across the plasma membrane to the cell exterior or into the lumen of the endoplasmic reticulum.

A small rise in cytosolic $\mathrm{Ca}^{2+}$ induces a variety of cellular responses, including hormone secretion by endocrine cells, secretion of digestive enzymes by pancreatic acinar cells, and contraction of muscle (Table 15-4). For example, acetylcholine stimulation of GPCRs in secretory cells of the pancreas and of the parotid (salivary) glands induces a rise in cytosolic $\mathrm{Ca}^{2+}$ that triggers the fusion of secretory vesicles with the plasma membrane and release of their protein contents into the extracellular space. Thrombin, an enzyme in the bloodclotting cascade, binds to a GPCR on blood platelets. This binding activates the receptor and triggers a rise in cytosolic $\mathrm{Ca}^{2+}$, which in turn causes a conformational change in the platelets that leads to their aggregation, an important step in blood clotting to prevent leakage of blood out of damaged blood vessels.

We learned in Chapter 12 that increases in the concentration of free $\mathrm{Ca}^{2+}$ in the mitochondrial matrix accelerate pyruvate oxidation and ATP production. Thus, in muscle, increases in the concentration of $\mathrm{Ca}^{2+}$ are used both to induce contractions and to coordinately increase mitochondrial ATP synthesis to provide the energy to fuel those contractions.

In this section, we first discuss experimental tools scientists use to measure the concentration of free $\mathrm{Ca}^{2+}$ ionsthat is, $\mathrm{Ca}^{2+}$ ions that are not tightly bound to proteins-in organelles such as the endoplasmic reticulum and the mitochondrion. Our main focus is on an important signal transduction mechanism that results in an elevation of cytosolic $\mathrm{Ca}^{2+}$ ion concentrations: the GPCR-stimulated activation of a phospholipase C (PLC). PLCs are a family of enzymes that hydrolyze a phosphoester bond in certain phospholipids, yielding two second messengers that function in elevating both the cytosolic and mitochondrial-matrix

TABLE 15-4 Cellular Responses to Hormone-Induced Rise in Cytosolic $\mathrm{Ca}^{2+}$ in Various Tissues*

| Tissue | Hormone Inducing <br> Rise in $\mathrm{Ca}^{2+}$ | Cellular Response |
| :-- | :-- | :-- |
| Pancreas <br> (acinar cells) | Acetylcholine | Secretion of digestive <br> enzymes, such as <br> amylase and trypsinogen |
| Parotid <br> (salivary) gland | Acetylcholine | Secretion of amylase |
| Vascular <br> or stomach <br> smooth muscle | Acetylcholine | Contraction |
| Liver | Vasopressin | Conversion of <br> glycogen to glucose |
| Blood platelets | Thrombin | Aggregation, shape <br> change, secretion of <br> hormones |
| Mast cells | Antigen | Histamine secretion |
| Fibroblasts | Peptide growth <br> factors | DNA synthesis, cell di- <br> vision (e.g., bombesin <br> and PDGF) |

*Hormone stimulation leads to production of inositol 1,4,5-trisphosphate $\left(\mathrm{IP}_{3}\right)$, a second messenger that promotes release of $\mathrm{Ca}^{2+}$ stored in the endoplasmic reticulum.
source: Data from M. J. Berridge, 1987, Annu. Rev. Biochem. 56:159, and M. J. Berridge and R. F. Irvine, 1984, Nature 312:315.
$\mathrm{Ca}^{2+}$ levels and in activating a family of cytosolic kinases known as protein kinases C (PKCs); PKCs, in turn, affect many important cellular processes such as growth and differentiation as well as altering the activity of many proteins.

Some PLCs are activated by GPCRs, as we describe here; others, covered in the following chapter, are activated by other types of receptors. Phospholipases C also produce second messengers that are important for remodeling the actin cytoskeleton (see Chapter 17) and for the binding of proteins important for endocytosis and vesicle fusion (see Chapter 14). Later in this section, we will see how second messengers such as $\mathrm{Ca}^{2+}$ are used to help cells integrate their responses to more than one extracellular signal. In the final part of the section, we see how one PLC pathway leads to the synthesis of a gas, nitric oxide (NO), which diffuses out of the cell into adjacent cells, where it can induce activation of a kinase that, in turn, alters several cellular activities.

## Calcium Concentrations in the Mitochondrial Matrix, ER, and Cytosol Can Be Measured with Targeted Fluorescent Proteins

We learned in Chapter 4 how the ester of the fluorescent small-molecule dye fura-2 can spontaneously diffuse into the cytosol from extracellular fluids, how fura-2 fluorescence at a certain wavelength increases when $\mathrm{Ca}^{2+}$ is bound, and how
this dye can be used to measure the concentration of free $\mathrm{Ca}^{2+}$ in the cytosol of live cells (see Figure 4-12). Several proteins also emit light when they have bound $\mathrm{Ca}^{2+}$ and can be used experimentally to determine the free $\mathrm{Ca}^{2+}$ concentration in other subcellular compartments.

One such protein is aequorin, a calcium-activated bioluminescent protein isolated from the hydrozoan Aequorea victoria. Aequorin consists of a protein subunit that can be expressed in cells by recombinant DNA technologies as a fusion protein with a signal sequence that targets it to a specific organelle, such as the ER lumen (see Figure 13-6) or the mitochondrial intermembrane space or matrix (see Figure 13-26). Aequorin contains three EF hands (see Chapter 3) that function as binding sites for $\mathrm{Ca}^{2+}$. When the small-molecule prosthetic group coelenterazine is added to the culture medium, it diffuses into the cell and binds to aequorin, and the cell emits light at a specific wavelength proportional to the concentration of free $\mathrm{Ca}^{2+}$ in that subcellular space.

The total calcium present in a subcellular compartment is the sum of the free $\mathrm{Ca}^{2+}$, which can be measured by aequorin and other calcium sensors whose fluorescence is proportional to the free $\mathrm{Ca}^{2+}$ concentration, and the bound $\mathrm{Ca}^{2+}$, the amounts of which in the ER and mitochondrion are thought to be much greater than those of free $\mathrm{Ca}^{2+}$. Measurements of free $\mathrm{Ca}^{2+}$ have indicated that, while the concentrations can vary considerably among different types of resting cells, typically $\mathrm{Ca}^{2+\text { cytosol }}=-100 \mathrm{nM}, \mathrm{Ca}^{2+\mathrm{ER}}=$ $-400 \mu \mathrm{M}$, and $\mathrm{Ca}^{2+\text { mitochondria }}=-100 \mathrm{nM}$. The ER lumen contains several $\mathrm{Ca}^{2+}$-binding proteins, including the chaperones calreticulin and calnexin (page 605), that have a low affinity and large capacity for $\mathrm{Ca}^{2+}$ binding and that buffer $\mathrm{Ca}^{2+\mathrm{ER}}$.

Stimulation of different types of cells by hormones or neuronal signals can result in considerable variation in the free $\mathrm{Ca}^{2+}$ concentration in organelles, but it invariably results in an increase in $\mathrm{Ca}^{2+\text { cytosol }}$ to about $1 \mu \mathrm{M}$ and in $\mathrm{Ca}^{2+}$ mitochondria to approximately $1-10 \mu \mathrm{M}$, as well as a decrease in $\mathrm{Ca}^{2+\mathrm{ER}}$ to about $100 \mu \mathrm{M}$. Transport of $\mathrm{Ca}^{2+}$ into and out of the ER lumen and the mitochondrial matrix plays an important role in controlling the nature of these changes and thus calcium signaling throughout the cell.

## Activated Phospholipase C Generates Two Key Second Messengers Derived from the Membrane Lipid Phosphatidylinositol 4,5-Bisphosphate

A number of important second messengers, used in several signal transduction pathways, are derived from the membrane lipid phosphatidylinositol (PI; Figure 15-33). The inositol group in this phospholipid, which always faces the cytosol, can be reversibly phosphorylated at one or more positions by the combined actions of various kinases and phosphatases discussed in Chapter 16. One derivative of PI, the lipid phosphatidylinositol 4,5-bisphosphate $\left[\mathrm{PI}(4,5) \mathrm{P}_{2}\right]$, is made by stepwise addition of two phosphates to PI. $\mathrm{PI}(4,5)$ $\mathrm{P}_{2}$ is then cleaved by activated phospholipase C into two important second messengers: 1,2-diacylglycerol (DAG), a

![li2474542480e](li2474542480e.jpg)

FIGURE 15-33 Synthesis of second messengers DAG and $\mathrm{IP}_{3}$ from phosphatidylinositol (PI). Each membrane-bound PI kinase places a phosphate (yellow circles) on a specific hydroxyl group on the inositol ring, producing the phosphorylated derivatives $\mathrm{PI}(4) \mathrm{P}$ and $\mathrm{PI}(4,5) \mathrm{P}_{2}$. Cleavage of $\mathrm{PI}(4,5) \mathrm{P}_{2}$ by phospholipase C yields the two important second
lipophilic molecule that remains associated with the membrane, and inositol 1,4,5-trisphosphate $\left(\mathrm{IP}_{3}\right)$, which can freely diffuse in the cytosol (see Figure 15-33). We refer to downstream events involving these two second messengers collectively as the $\mathrm{IP}_{3} /$ DAG pathway.
messengers DAG and $\mathrm{IP}_{3}$. Signaling is terminated when a phosphatase removes the 5-phosphate from $\mathrm{IP}_{3}$; a second phosphatase removes the 1-phosphate, and the inositol 4-phosphate is reused to synthesize PI 4-phosphate. See A. Toker and L. C. Cantley, 1997, Nature 387:673, and C. L. Carpenter and L. C. Cantley, 1996, Curr. Opin. Cell Biol. 8:153.

Phospholipase C is activated by G proteins containing either $\mathrm{G}_{\mathrm{res}}$ or $\mathrm{G}_{\mathrm{reg}}$ subunits. In response to hormone activation of their associated GPCR, the $\mathrm{G}_{\text {res }}$ or $\mathrm{G}_{\text {reg }}$ subunits bound to GTP separate from $\mathrm{G}_{\mathrm{IP}_{3}}$ and bind to and activate phospholipase C in the membrane (Figure 15-34a, step ■).

FIGURE 15-34 The $\mathrm{IP}_{3} /$ DAG pathway and the elevation of cytosolic $\mathrm{Ca}^{2+}$. (a) Opening of endoplasmic reticulum $\mathrm{Ca}^{2+}$ channels can be triggered by ligand binding to GPCRs that activate either the $\mathrm{G}_{\text {res }}$ or $\mathrm{G}_{\text {reg }}$ subunit, leading to activation of phospholipase C (step ■). Cleavage of $\mathrm{PI}(4,5) \mathrm{P}_{2}$ by phospholipase C yields $\mathrm{IP}_{3}$ and DAG (step ■). After diffusing through the cytosol, $\mathrm{IP}_{3}$ interacts with and opens $\mathrm{IP}_{3}$-gated $\mathrm{Ca}^{2+}$ channels in the membrane of the endoplasmic reticulum (step $\mathbf{\square}$ ), causing release of stored $\mathrm{Ca}^{2+}$ ions into the cytosol (step $\mathbf{\square}$ ). One of several cellular responses induced by a rise in cytosolic $\mathrm{Ca}^{2+}$ is recruitment of protein kinase C (PKC) to the plasma membrane (step $\mathbf{\square}$ ), where it is activated by DAG (step $\mathbf{\square}$ ). The activated membrane-associated kinase can phosphorylate various cellular enzymes and receptors, thereby altering their activity (step $\mathbf{\square}$ ). (b) Opening of plasma-membrane $\mathrm{Ca}^{2+}$ channels. Top: In the resting cell, $\mathrm{Ca}^{2+}$ levels in the ER lumen are high, and $\mathrm{Ca}^{2+}$ ions (blue circles) bind to the luminal EF hand domains of the transmembrane STIM proteins. Bottom: As $\mathrm{Ca}^{2+}$ stores in the ER are depleted and $\mathrm{Ca}^{2+}$ ions dissociate from the EF hands, STIMs undergo oligomerization and relocalization to areas of the ER membrane near the plasma membrane. There the STIM CAD domains (orange) bind to and trigger the opening of the store-operated $\mathrm{Ca}^{2+}$ channels (Orai1) in the plasma membrane, allowing influx of extracellular $\mathrm{Ca}^{2+}$. (c) Drawing of the three-dimensional structure of Orai1 in the closed state. The Orai1 $\mathrm{Ca}^{2+}$ channel is composed of six identical subunits arranged around a central $\mathrm{Ca}^{2+}$ pore. Each subunit contains four transmembrane $\alpha$ helices-M1 (blue), M2 (red), M3 (orange), and M4 (violet)—and a helix following M4 that extends into the cytosol (termed the M4 extension helix, violet). The M1 helices are drawn as ribbons, the M2-M4 helices as cylinders. The pore is lined by the six M1 helices; in the closed state, a $\mathrm{Ca}^{2+}$ ion is bound at the extracellular entrance to the pore, but cannot enter it. Binding of the CADs leads to channel opening, most likely by widening of the pore by the outward movement of the M1 helices. The intracellular ends of the M1 helices are thought to interact with a portion of the STIM CAD, as are the M4 extensions, and it is hypothesized that the CADs bridge the cytosolic portions of the M1 helices and the M4/M4 extension helices. See J. W. Putney, 1999, P. Natl. Acad. Sci. USA 96:14669; Y. Zhou, 2010, P. Natl. Acad. Sci. USA 107:4896; and M. Cahalan, 2010, Science 330:43. [Part (c) data from X. Hou et al. 2012, Science 338:1308, PDB ID 4hks.]

![li2484542480e](li2484542480e.jpg)

In turn, activated phospholipase C cleaves $\mathrm{PI}(4,5) \mathrm{P}_{2}$ into DAG and $\mathrm{IP}_{3}$ (see Figure 15-34a, step $\mathbf{B}$ ). The two second messengers trigger distinct, but sometimes interrelated, downstream effects.
$\mathrm{Ca}^{2+}$ Release from the ER Triggered by $\mathrm{IP}_{3}$ Ligand binding to G protein-coupled receptors that activate phospholipase C induces an elevation in cytosolic $\mathrm{Ca}^{2+}$ even when $\mathrm{Ca}^{2+}$ ions are absent from the surrounding extracellular fluid. In this case, $\mathrm{Ca}^{2+}$ is released into the cytosol from the ER lumen (where, as noted above, $\mathrm{Ca}^{2+}$ can accumulate to almost millimolar concentrations) through the operation of $\mathrm{IP}_{3}$-gated $\mathrm{Ca}^{2+}$ channels in the ER membrane, as depicted in Figure 15-34a (steps $\mathbf{B}$ and $\mathbf{B}$ ). (This family of channel proteins is similar in structure to the family of voltage-sensitive $\mathrm{Ca}^{2+}$ channels that are called ryanodine receptors in muscle cell sarcoplasmic reticulum membranes; see page 805.) Each of these large $\mathrm{IP}_{3}$-gated channels is composed of four identical subunits, each of which contains an $\mathrm{IP}_{3}$-binding site in its N -terminal cytosolic domain. $\mathrm{IP}_{3}$ binding induces opening of the channel, allowing $\mathrm{Ca}^{2+}$ to flow down its concentration gradient from the ER lumen into the cytosol. When different phosphorylated inositols normally found in cells were added to preparations of ER vesicles, only $\mathrm{IP}_{3}$ caused release of $\mathrm{Ca}^{2+}$ ions from the vesicles. This simple experiment demonstrates the specificity of the $\mathrm{IP}_{3}$ effect.

The $\mathrm{IP}_{3}$-mediated rise in the cytosolic $\mathrm{Ca}^{2+}$ level is transient because $\mathrm{Ca}^{2+}$ pumps located in the plasma membrane and in the ER membrane actively transport $\mathrm{Ca}^{2+}$ from the cytosol to the cell exterior and back into the ER lumen, respectively. Furthermore, within a second of its generation, the phosphate linked to carbon 5 of $\mathrm{IP}_{3}$ is hydrolyzed, yielding inositol 1,4-bisphosphate (see Figure 15-33). This compound cannot bind to the $\mathrm{IP}_{3}$-gated $\mathrm{Ca}^{2+}$ channel and thus does not stimulate $\mathrm{Ca}^{2+}$ release from the ER.
$\mathrm{Ca}^{2+}$ Transport from the ER to the Mitochondrial Matrix Triggered by IP ${ }_{3}$ In Chapter 12, we learned that direct contacts between specialized regions of the ER membrane, called mitochondria-associated membranes (MAMs), and the outer mitochondrial membrane affect mitochondrial structure, dynamics, and function. Regulated movement of $\mathrm{Ca}^{2+}$ from the ER lumen through the MAMs into the mitochondrial matrix is a major part of this regulation (Figure 15-35a). $\mathrm{IP}_{3}$-gated $\mathrm{Ca}^{2+}$ channels in the MAMs open in response to a rise in cytosolic $\mathrm{IP}_{3}$ (step $\mathbf{B}$, Figure 15-35a). Voltage-dependent anion channels (VDACs) in the outer mitochondrial membrane adjacent to the MAMs, which are physically linked to these $\mathrm{IP}_{3}{ }^{-}$ gated $\mathrm{Ca}^{2+}$ channels by the GRP75 protein, efficiently pass the $\mathrm{Ca}^{2+}$ released from the ER lumen into the intermembrane space (step $\mathbf{B}$ ). A mitochondrial calcium uniporter (MCU) in the inner mitochondrial membrane then transports $\mathrm{Ca}^{2+}$ into the mitochondrial matrix, where it increases ATP synthesis as well as enhancing mitochondrial activity in other ways.

MCUs open only in the presence of a high $\mathrm{Ca}^{2+}$ concentration in the intermembrane space. Regulatory subunits of the MCU that face the intermembrane space (Figure 15-35b)
![li2494542480e](li2494542480e.jpg)
(b)
![li2504542480e](li2504542480e.jpg)

FIGURE 15-35 Movement of $\mathrm{Ca}^{2+}$ between the cytosol, mitochondrion, and endoplasmic reticulum. (a) The ER is the main intracellular storage depot for $\mathrm{Ca}^{2+}$. Binding of $\mathrm{IP}_{3}$ to $\mathrm{IP}_{3}$-gated $\mathrm{Ca}^{2+}$ channels $\left(\mathrm{IP}_{3} \mathrm{R}\right)$ in the membrane of the endoplasmic reticulum releases $\mathrm{Ca}^{2+}$ into the cytosol (step $\mathbf{B}$ ); binding also opens $\mathrm{IP}_{3}$-gated $\mathrm{Ca}^{2+}$ channels in the mitochondria-associated membranes (MAMs) of the ER (step $\mathbf{B}$ ). Step $\mathbf{B}$ VDAC channels in the outer mitochondrial membrane adjacent to MAMs are physically linked to $\mathrm{IP}_{3} \mathrm{Rs}$ by the GRP75 protein; they efficiently pass the $\mathrm{Ca}^{2+}$ released from the MAMs into the intermembrane space. Step $\mathbf{B}$ The high concentration of $\mathrm{Ca}^{2+}$ in the intermembrane space induces the opening of MCUs or other $\mathrm{Ca}^{2+}$ channels in the inner membrane, resulting in the flow of $\mathrm{Ca}^{2+}$ into the mitochondrial matrix. Step Over time, $\mathrm{Ca}^{2+}$ is released from the mitochondria by $\mathrm{Ca}^{2+} / \mathrm{Na}^{+}$(NCLX) and $\mathrm{Ca}^{2+} / \mathrm{H}^{+}$(HCX) antiporters in the inner membrane, then transferred into the cytosol through VDAC or other $\mathrm{Ca}^{2+}$ channels in the outer membrane. Finally, pumping of $\mathrm{Ca}^{2+}$ from the cytosol by ATP-powered $\mathrm{Ca}^{2+}$ pumps in the ER membrane (step $\mathbf{B}$ ) or plasma membrane restores the high $\mathrm{ERCa}^{2+}$ and low cytosolic $\mathrm{Ca}^{2+}$ levels. (b) Model of the mitochondrial calcium uniporter complex. A multimer of MCU subunits forms the regulated $\mathrm{Ca}^{2+}$ pore. Additional subunits include the integral membrane protein EMRE and regulatory subunits MICU1 and MICU2. $\mathrm{Ca}^{2+}$ binding to the MICU subunits opens the MCU pore, resulting in the flow of the $\mathrm{Ca}^{2+}$ from the intermembrane space into the matrix. See M. Schäfer et al., 2014, Cell Tissue Res. 357:395 and K. Kamer and V. Mootha, 2015, Nat. Rev. Mol. Cell Biol. 16:545.

have $\mathrm{Ca}^{2+}$-binding EF hands (see Chapter 3) with relatively low binding affinity for $\mathrm{Ca}^{2+}$; these subunits must bind calcium for the MCU to open. Individuals with mutations in the gene encoding one of these subunits have skeletal muscle defects and learning disabilities, symptoms that can accompany other mitochondrial disorders, attesting to the importance of these uniporters in mitochondrial metabolism.

To avoid the buildup of excess, potentially toxic, intramitochondrial calcium, the mitochondrial matrix gradually releases $\mathrm{Ca}^{2+}$ into the cytosol. The calcium first moves across the inner mitochondrial membrane into the intermembrane space via $\mathrm{Na}^{+} / \mathrm{Ca}^{2+}$ and $\mathrm{H}^{+} / \mathrm{Ca}^{2+}$ antiporters, then crosses the outer mitochondrial membrane, most likely via VDACs (step (1). The calcium transport cycle is completed when cytosolic calcium enters the ER via an ATP-powered $\mathrm{Ca}^{2+}$ pump (step 2; see Figure 11-10) or is pumped out of the cell by plasma-membrane ATP-powered $\mathrm{Ca}^{2+}$ pumps.

The Store-Operated Plasma-Membrane Ca ${ }^{2+}$ Channel Continued opening of the $\mathrm{IP}_{3}$-gated $\mathrm{Ca}^{2+}$ channel in the ER membrane, coupled with operation of the plasma-membrane $\mathrm{Ca}^{2+}$ pump, would eventually deplete the intracellular stores of $\mathrm{Ca}^{2+}$, and a cell would soon be unable to increase the cytosolic $\mathrm{Ca}^{2+}$ level in response to hormone-induced $\mathrm{IP}_{3}$. Patch-clamping studies (see Figure 11-22) have revealed that a plasma-membrane $\mathrm{Ca}^{2+}$ channel called the store-operated channel opens in response to depletion of $\mathrm{ER} \mathrm{Ca}^{2+}$ stores and admits extracellular $\mathrm{Ca}^{2+}$ into the cytosol. Studies in which each potential $\mathrm{Ca}^{2+}$ channel protein was knocked down one at a time with shRNAs (see Figure 6-42) established the identity of this channel protein as Orai1; the sequence and three-dimensional structure of Orai1 is unlike that of any other known ion-channel protein (see Figure 15-34c), which partly explains why it took so long to identify it as the store-operated channel.

The ER $\mathrm{Ca}^{2+}$-sensing protein is STIM, a transmembrane protein in the ER membrane (see Figure 15-34b). An EF hand, similar to that in calmodulin (see Figure 3-33), on the luminal side of the ER membrane binds $\mathrm{Ca}^{2+}$ when its level in the lumen is high. As stores of $\mathrm{Ca}^{2+}$ in the ER are depleted, the STIM proteins lose their bound $\mathrm{Ca}^{2+}$, oligomerize, and in an unknown manner, relocalize to areas of the ER membrane near the plasma membrane (see Figure 15-34b). There the CAD domains of the STIM proteins bind to and trigger the opening of Orai1, allowing the influx of extracellular $\mathrm{Ca}^{2+}$ (see Figure 15-34c). Combined overexpression of Orai and STIM in cultured cells leads to a marked increase in $\mathrm{Ca}^{2+}$ influx, establishing that these two proteins are the key components of the store-operated $\mathrm{Ca}^{2+}$ pathway.

Feedback Loops That Trigger Spikes in the Cytosolic Ca ${ }^{2+}$ Concentration Continuous activation of certain G proteincoupled receptors induces rapid, repeated spikes in the level of cytosolic $\mathrm{Ca}^{2+}$ (Figure 15-36). These oscillations in cytosolic $\mathrm{Ca}^{2+}$ levels are caused by a complex feedback interaction between the cytosolic $\mathrm{Ca}^{2+}$ concentration and the $\mathrm{IP}_{3}$-gated $\mathrm{Ca}^{2+}$-channels. The submicromolar level of cytosolic $\mathrm{Ca}^{2+}$ in the resting cell potentiates the opening of
![li2514542480e](li2514542480e.jpg)

FIGURE 15-36 Oscillations in the cytosolic $\mathrm{Ca}^{2+}$ concentration following treatment of human HeLa cells with histamine. Like the LH receptors described in the text, the histamine GPCR activates the $\mathrm{IP}_{3}$-DAG signaling pathway. The feedback loops generating the spikes in the cytosolic $\mathrm{Ca}^{2+}$ concentration are detailed in the text. [Data from A. Miyawaki et al., 1997, Nature 388:882.]
these channels via a rise in $\mathrm{IP}_{3}$, thus facilitating the rapid rise in cytosolic $\mathrm{Ca}^{2+}$ that follows hormone stimulation of cell-surface G protein-coupled receptors. However, the high cytosolic $\mathrm{Ca}^{2+}$ levels reached at the peak of the spike inhibit further $\mathrm{IP}_{3}$-induced release of $\mathrm{Ca}^{2+}$ from ER stores by decreasing the affinity of the $\mathrm{Ca}^{2+}$ channels for $\mathrm{IP}_{3}$. As a result, the $\mathrm{IP}_{3}$-gated $\mathrm{Ca}^{2+}$ channels close, and the cytosolic $\mathrm{Ca}^{2+}$ level drops rapidly as $\mathrm{Ca}^{2+}$ is pumped into the ER lumen or out of the cell. Thus cytosolic $\mathrm{Ca}^{2+}$ is a feedback inhibitor of the $\mathrm{IP}_{3}$-gated $\mathrm{Ca}^{2+}$ channels that, when open, trigger a rise in cytosolic $\mathrm{Ca}^{2+}$. As an example, this mechanism produces calcium ion oscillations occur in the pituitary gland cells that secrete luteinizing hormone (LH), which plays an important role in controlling ovulation and thus female fertility. LH secretion is induced by the binding of luteinizing hormonereleasing hormone (LHRH) to its G protein-coupled receptors on the surfaces of these cells; LHRH binding induces repeated $\mathrm{Ca}^{2+}$ spikes. Each $\mathrm{Ca}^{2+}$ spike induces exocytosis of a few LH-containing secretory vesicles, presumably those close to the plasma membrane.

## The $\mathrm{Ca}^{2+}$-Calmodulin Complex Mediates Many Cellular Responses to External Signals

The ubiquitous small cytosolic protein calmodulin functions as a multipurpose switch protein that mediates many cellular effects of $\mathrm{Ca}^{2+}$ ions. Binding of $\mathrm{Ca}^{2+}$ to four sites on calmodulin triggers a major conformational change that allows calmodulin to bind to and modulate the activity of many enzymes and other proteins (see Figure 3-33). Because four $\mathrm{Ca}^{2+}$ ions bind to calmodulin in a cooperative fashion, a small change in the level of cytosolic $\mathrm{Ca}^{2+}$ leads to a large change in the level of active calmodulin. One well-studied

enzyme activated by the $\mathrm{Ca}^{2+}$-calmodulin complex is myosin light-chain kinase, which regulates the activity of myosin and thus contraction in smooth muscle cells (see Chapter 17). Another is PDE, the enzyme that degrades cAMP to $5^{\prime}$-AMP and terminates its effects (see Figure 15-31). This reaction thus links $\mathrm{Ca}^{2+}$ and cAMP, one of many examples in which two second messenger-mediated pathways interact to finetune a cellular response.

In many cells, the rise in cytosolic $\mathrm{Ca}^{2+}$ that follows receptor signaling via phospholipase C -generated $\mathrm{IP}_{3}$ leads to the activation of specific transcription factors. In some cases, $\mathrm{Ca}^{2+}$-calmodulin activates protein kinases that, in turn, phosphorylate transcription factors, thereby modifying their activity and regulating gene expression. In other cases, $\mathrm{Ca}^{2+}$-calmodulin activates a phosphatase that removes phosphate groups from a transcription factor, thus activating it. An important example of this mechanism involves the T cells of the immune system (see Chapter 23).

## DAG Activates Protein Kinase C

After its formation by phospholipase C-catalyzed hydrolysis of $\mathrm{PI}(4,5) \mathrm{P}_{2}$, the hydrophobic DAG (see Figure 15-34a) remains associated with the plasma membrane. The principal function of DAG is to activate a family of protein kinases collectively termed protein kinase C (PKC). In the absence of hormone stimulation, protein kinase C is present as a soluble cytosolic protein that is catalytically inactive. A rise in the cytosolic $\mathrm{Ca}^{2+}$ level causes protein kinase C to translocate to the cytosolic leaflet of the plasma membrane, where it can interact with membrane-associated DAG (see Figure 15-34a, steps $\mathbf{\square}$ and $\mathbf{\square}$ ). Activation of PKC thus depends on an increase of both $\mathrm{Ca}^{2+}$ ions and DAG, suggesting an interaction between the two branches of the $\mathrm{IP}_{3} /$ DAG pathway.

The activation of PKC in different cells results in a varied array of cellular responses, indicating that it plays a key role in many aspects of cellular growth and metabolism. In many cells, PKC phosphorylates transcription factors that are localized in the cytosol, triggering their movement into the nucleus, where they activate genes necessary for cell division. In liver cells, PKC helps regulate glycogen metabolism by phosphorylating and so inhibiting glycogen synthase, as we will see next.

## Integration of $\mathrm{Ca}^{2+}$ and cAMP Second Messengers Regulates Glycogenolysis

All cells constantly receive multiple signals from their environment, including changes in levels of hormones, metabolites, and gases such as NO and oxygen. All of these signals must be integrated. The breakdown of glycogen to glucose (glycogenolysis) provides an excellent example of how cells can integrate their responses to more than one signal. As discussed in Section 15.5, epinephrine stimulation of muscle and liver cells leads to a rise in the second messenger cAMP, which promotes glycogen breakdown (see Figure 15-28a). In both muscle and liver cells, other signaling pathways produce the same cellular response of enhanced glycogenolysis.

In striated muscle cells (see Figure 17-30), stimulation by nerve impulses causes the release of $\mathrm{Ca}^{2+}$ ions from the sarcoplasmic reticulum and an increase in the cytosolic $\mathrm{Ca}^{2+}$ concentration, which triggers muscle contraction. The rise in cytosolic $\mathrm{Ca}^{2+}$ also allows $\mathrm{Ca}^{2+}$ binding to the $\delta$ subunit of glycogen phosphorylase kinase (GPK)—which is calmodulin-thus activating the kinase catalytic activity of the $\gamma$ subunit and thereby stimulating the degradation of glycogen to glucose-1-phosphate, which fuels prolonged muscle contraction.

Rises in blood epinephrine concentrations lead to activation of adenylyl cyclase and an increase in cAMP concentrations. Recall that phosphorylation by cAMP-dependent PKA also activates GPK (see Figure 15-28) and that maximal activation of GPK, and therefore glycogenolysis, requires both phosphorylation and $\mathrm{Ca}^{2+}$. Thus this key regulator of glycogenolysis is subject to both neural and hormonal regulation in muscle (Figure 15-37a).

In liver cells, hormone-induced activation of the effector protein phospholipase C also regulates glycogen breakdown by generating the second messengers DAG and $\mathrm{IP}_{3}$. As we have just learned, $\mathrm{IP}_{3}$ induces an increase in cytosolic $\mathrm{Ca}^{2+}$, which activates GPK, as in muscle cells, leading to glycogen degradation. Moreover, the combined effect of DAG and increased $\mathrm{Ca}^{2+}$ activates protein kinase C (see Figure 15-34). This kinase then phosphorylates and thereby inhibits glycogen synthase, reducing the rate of glycogen synthesis. Thus we see how multiple intracellular signal transduction pathways interact to regulate an important metabolic process (Figure 15-37b).

Recall that GPK is maximally active when $\mathrm{Ca}^{2+}$ ions are bound to the $\delta$ (calmodulin) subunit and the $\alpha$ subunit has been phosphorylated by PKA. After neuronal stimulation of muscle cells, GPK is sufficiently active, even if it is nonphosphorylated, to allow degradation of glycogen in the absence of hormone stimulation. Binding of $\mathrm{Ca}^{2+}$ to the $\delta$ subunit may be essential to the enzymatic activity of GPK. Phosphorylation of the $\alpha$ and also the $\beta$ subunits by PKA increases the affinity of the $\delta$ subunit for $\mathrm{Ca}^{2+}$, allowing $\mathrm{Ca}^{2+}$ ions to bind to the enzyme at the submicromolar $\mathrm{Ca}^{2+}$ concentrations found in resting cells.

## Signal-Induced Relaxation of Vascular Smooth Muscle Is Mediated by a $\mathrm{Ca}^{2+}$-Nitric Oxide-cGMPActivated Protein Kinase G Pathway

In the late nineteenth century, Alfred Nobel (of the Nobel Prize) figured out how to improve nitroglycerine as an explosive that could be used in blasting rock and in mining, but for over a century nitroglycerin has also found an important use as a treatment for the intense chest pain of angina. It was known to slowly decompose in the body to nitric oxide (NO), which causes relaxation of the smooth muscle cells surrounding the blood vessels that "feed" the heart muscle itself, thereby increasing the diameter of the blood vessels and increasing the flow of oxygen-bearing blood to the heart muscle. One of the most intriguing discoveries in modern medicine is that NO, a toxic gas found in car exhaust, is in fact a natural signaling molecule.

![li2524542480e](li2524542480e.jpg)

Definitive evidence for the role of NO in inducing relaxation of smooth muscle came from a set of experiments in which acetylcholine was added to experimental preparations of the smooth muscle cells that surround blood vessels (see Figure 1-25). Direct application of acetylcholine to these cells caused them to contract, as expected for vascular muscle cells. But addition of acetylcholine to the lumen of small isolated blood vessels caused the smooth muscles in those vessels to relax, not contract. Subsequent studies showed that in response to acetylcholine, the endothelial cells that line the lumen of a blood vessel were releasing some substance that in turn triggered muscle-cell relaxation. That substance turned out to be NO.

FIGURE 15-37 Integrated regulation of glycogenolysis by $\mathrm{Ca}^{2+}$ and cAMP/PKA pathways. (a) Neuronal stimulation of striated muscle cells or epinephrine binding to $\beta$-adrenergic receptors on their surfaces leads to increased cytosolic concentrations of the second messengers $\mathrm{Ca}^{2+}$ or cAMP, respectively. The key regulatory enzyme glycogen phosphorylase kinase (GPK) is activated by binding $\mathrm{Ca}^{2+}$ ions and by phosphorylation by cAMP-dependent PKA. (b) In liver cells, hormonal stimulation of two $\beta$-adrenergic receptors leads to increased cytosolic concentrations of cAMP and two other second messengers, diacylglycerol (DAG) and inositol 1,4,5-trisphosphate $\left(\mathrm{IP}_{3}\right)$. Enzymes are marked by white boxes. $(+)=\mathrm{activation}$ of enzyme activity; $(-)=$ inhibition.

We now know that vascular endothelial cells contain a $\mathrm{G}_{\mathrm{ss}}$, protein-coupled GPCR that binds acetylcholine and activates phospholipase C, leading to an increase in the level of cytosolic $\mathrm{Ca}^{2+}$. After $\mathrm{Ca}^{2+}$ binds to calmodulin, the $\mathrm{Ca}^{2+}$ / calmodulin complex stimulates the activity of NO synthase, an enzyme that catalyzes formation of NO from $\mathrm{O}_{2}$ and the amino acid arginine. Because NO has a short half-life (2-30 seconds), it can diffuse only locally in tissues from its site of synthesis. In particular, NO diffuses from the vascular endothelial cell into neighboring smooth muscle cells, where it triggers muscle relaxation, which increases the diameter of the blood vessel (vasodilation) (Figure 15-38).
![li2534542480e](li2534542480e.jpg)

FIGURE 15-38 The $\mathrm{Ca}^{2+}$ /nitric oxide (NO)/ cGMP pathway and the relaxation of vascular smooth muscle. Nitric oxide is synthesized in endothelial cells in response to activation of acetylcholine GPCRs, phospholipase C, and the subsequent elevation in cytosolic $\mathrm{Ca}^{2+}$ (steps 1- 5). NO diffuses locally through tissues and activates an intracellular NO receptor with guanylyl cyclase activity in nearby smooth muscle cells. The resulting rise in cGMP activities protein kinase G 2, leading to relaxation of the muscle and thus vasodilation $\square \mathrm{PPi}=$ pyrophosphate. See C. S. Lowenstein et al., 1994, Ann. Intern. Med. 120:227, and H. K. Surks, 2007, Circ. Res. 101:1078.

The effect of NO on smooth muscle is mediated by the second messenger cGMP, which is formed by a cytosolic NO receptor expressed by smooth muscle cells. Binding of NO to the heme group in this receptor leads to a conformational change that increases its intrinsic guanylyl cyclase activity, leading to a rise in the cytosolic cGMP level. Most of the effects of cGMP are mediated by a cGMP-dependent protein kinase, also known as protein kinase G (PKG), that is regulated similarly to PKA except that the regulatory domain is part of the PKG polypeptide. This N-terminal domain contains a pseudosubstrate segment that binds to the kinase domain and inhibits its activity, as well as two cGMP-binding sites. Binding of cGMP induces a conformational change in the regulatory domain that prevents the pseudosubstrate from inhibiting the PKG kinase. In vascular smooth muscle, PKG activates a signaling pathway leading to relaxation of the cell and therefore dilation of the blood vessel. The dilation is caused in part by inhibition of $\mathrm{Ca}^{2+}$ channels in the endoplasmic reticulum (see Figure 15-34a) and a resulting decrease in cytosolic $\mathrm{Ca}^{2+}$ concentration; in Chapter 17 (page 808), we learn that in smooth muscle, a reduction in cytosolic $\mathrm{Ca}^{2+}$ causes a decrease in phosphorylation of a regulatory myosin light chain, disassembly of the actin-myosin contractile structure, and inhibition of muscle contraction. In vascular smooth muscle cells, cGMP acts indirectly via protein kinase G, in contrast to rod cells, in which cGMP acts directly by binding to and opening cation channels in the plasma membrane (see Figure 15-20).

5Inhibitors of the cGMP-hydrolyzing enzyme PDE, which cause an elevation of cGMP in vascular smooth muscle cells, were originally developed as a treatment for male pattern baldness. A large clinical trial showed that they failed to increase hair growth, but a major side effect was noted—prolonged erections. Then several PDE inhibitors were quickly developed to treat erectile dysfunction. We'll spare you the details here, but any interested reader can quickly find out how these widely used drugs function.

## KEY CONCEPTS OF SECTION 15.6

## G Protein-Coupled Receptors That Trigger Elevations in Cytosolic and Mitochondrial Calcium

- A small rise in cytosolic $\mathrm{Ca}^{2+}$ induces a variety of responses in different cells, including hormone secretion, contraction of muscle, and platelet aggregation (see Table 15-4).
- Many hormones bind GPCRs coupled to G proteins containing a $\mathrm{G}_{\mathrm{ss} \mathrm{\alpha}}$ or $\mathrm{G}_{\mathrm{sqq}}$ subunit. The effector protein activated by GTP-bound $\mathrm{G}_{\mathrm{ss} \mathrm{\alpha}}$ or $\mathrm{G}_{\mathrm{sqq}}$ is a phospholipase C enzyme.
- Phospholipase C cleaves a phospholipid known as $\mathrm{PI}(4,5)$ $\mathrm{P}_{2}$, generating two second messengers: diffusible $\mathrm{IP}_{3}$ and membrane-bound DAG (see Figure 15-33).
- $\mathrm{IP}_{3}$ triggers the opening of $\mathrm{IP}_{3}$-gated $\mathrm{Ca}^{2+}$ channels in the endoplasmic reticulum and elevation of cytosolic free $\mathrm{Ca}^{2+}$.
- Opening of $\mathrm{IP}_{3}$-gated $\mathrm{Ca}^{2+}$ channels in the endoplasmic reticulum also leads to an increase in $\mathrm{Ca}^{2+}$ in the mitochondrial matrix and an acceleration of ATP synthesis (see Figure 15-35).
- Depletion of ER $\mathrm{Ca}^{2+}$ stores leads to the opening of plasma-membrane store-operated $\mathrm{Ca}^{2+}$ channels and an influx of $\mathrm{Ca}^{2+}$ from the extracellular medium (see Figure 15-34b).
- The $\mathrm{Ca}^{2+}$-calmodulin complex regulates the activity of many different proteins, including PDE and protein kinases and phosphatases that control the activity of various transcription factors.
- In response to elevated cytosolic $\mathrm{Ca}^{2+}$, protein kinase C is recruited to the plasma membrane, where it is activated by DAG (see Figure 15-34a).
- Glycogen breakdown and synthesis is coordinately regulated by the second messengers $\mathrm{Ca}^{2+}$ and cAMP, whose levels are regulated by neural and hormonal stimulation, respectively (see Figure 15-37).
- Stimulation of acetylcholine GPCRs on endothelial cells induces an increase in cytosolic $\mathrm{Ca}^{2+}$ and subsequent synthesis of NO. After diffusing into surrounding smooth muscle cells, NO activates an intracellular guanylate cyclase to synthesize cGMP. The resulting increase in cGMP leads to activation of protein kinase G, which triggers a pathway resulting in muscle relaxation and vasodilation (see Figure 15-38).


## LaunchPad

macmillan learning
Visit LaunchPad to access study tools and to learn more about the content in this chapter.

- Perspectives for the Future
- Classic Experiment 15-1: The Infancy of Signal Transduction Studies: GTP Stimulation of cAMP Synthesis
- Analyze the Data
- Additional study tools, including videos, animations, and quizzes


## Key Terms

adenylyl cyclase 692
agonist 683
amplification 679
antagonist 683 arrestin 698
autocrine 676
$\beta$-adrenergic receptor 687
calmodulin 679

cyclic adenosine monophosphate (cAMP) 679
desensitization 683
endocrine 676
epinephrine 676
glucagon 687
glycogenolysis 699
G protein-coupled receptor (GPCR) 675
heterotrimeric G protein 678
hormone 676
$\mathrm{IP}_{3} /$ DAG pathway 710
kinase 677
muscarinic acetylcholine receptor 693
nitric oxide (NO) 714
paracrine 676
phosphatase 677
phospholipase C (PLC) 692
protein kinase A (PKA) 679
protein kinase C (PKC) 714
protein kinase G (PKG) 716
rhodopsin 694
second messenger 678
signal transduction 674
transducin 694

## Review the Concepts

1. What common features are shared by most cell signaling systems?
2. Signaling by soluble extracellular molecules can be classified as endocrine, paracrine, or autocrine. Describe how these three types of cellular signaling differ. Growth hormone is secreted from the pituitary, which is located at the base of the brain and acts through growth hormone receptors located on the liver. Is this an example of endocrine, paracrine, or autocrine signaling? Why?
3. A ligand binds two different receptors with a $K_{\mathrm{d}}$ value of $10^{-7} \mathrm{M}$ for receptor 1 and a $K_{\mathrm{d}}$ value of $10^{-9} \mathrm{M}$ for receptor 2 . For which receptor does the ligand show the greater affinity? Calculate the fraction of receptors that have a bound ligand $\left([\mathrm{RL}] / \mathrm{R}_{\mathrm{T}}\right)$ in the case of receptor 1 and receptor 2 if the concentration of free ligand is $10^{-8} \mathrm{M}$.
4. To understand how a signaling pathway works, it is often useful to isolate the cell-surface receptor and to measure the activity of downstream effector proteins under different conditions. How could you use affinity chromatography to isolate a cell-surface receptor? With what technique could you measure the amount of activated G protein (the GTP-bound form) in ligand-stimulated cells? Describe the approach you would take.
5. How do seven-transmembrane domain G protein-coupled receptors transmit a signal across the plasma membrane? In your answer, include the conformational changes that occur in the receptor in response to ligand binding.
6. Signal-transducing heterotrimeric G proteins consist of three subunits designated $\alpha, \beta$, and $\gamma$. The $G_{α}$ subunit is a GTPase switch protein that cycles between active and inactive states depending on whether it is bound to GTP or to GDP. Review the steps for ligand-induced activation of effector proteins mediated by the heterotrimeric G proteins. Suppose that you have isolated a mutant $G_{a}$ subunit that has an increased GTPase activity. What effect would this mutation have on the G protein and the effector protein?
7. Explain how FRET could be used to monitor the association of $\mathrm{G}_{\mathrm{as}}$ and adenylyl cyclase following activation of the epinephrine receptor.
8. Which of the following steps amplify the epinephrine signal response in cells: receptor activation of G protein, G protein activation of adenylyl cyclase, cAMP activation of PKA, or PKA phosphorylation of glycogen phosphorylase kinase (GPK)? Which change will have a greater effect on signal amplification: an increase in the number of epinephrine receptors or an increase in the number of $\mathrm{G}_{\mathrm{as}}$ proteins?
9. The cholera toxin, produced by the bacterium Vibrio cholerae, causes a watery diarrhea in infected individuals. What is the molecular basis for this effect of cholera toxin? 10. Both rhodopsin in vision and the muscarinic acetylcholine receptor in cardiac muscle are coupled to ion channels via G proteins. Describe the similarities and differences between these two systems.
10. Epinephrine binds to both $\beta$-adrenergic and $\alpha$-adrenergic receptors. Describe the opposite actions on the effector protein, adenylyl cyclase, elicited by the binding of epinephrine to these two types of receptors. Describe the effect of adding an agonist or antagonist to a $\beta$-adrenergic receptor on the activity of adenylyl cyclase.
11. In liver and muscle, epinephrine stimulation of the cAMP pathway activates glycogen breakdown and inhibits glycogen synthesis, whereas in adipose tissue, epinephrine activates hydrolysis of triglycerides, and in other cells, it causes a diversity of other responses. What step in the cAMP signaling pathways in these cells specifies the cell response?
12. Continuous exposure of a $\mathrm{G}_{\mathrm{as}}$ protein-coupled receptor to its ligand leads to a phenomenon known as desensitization. Describe several molecular mechanisms for receptor desensitization. How can a receptor be reset to its original sensitized state? What effect would a mutant receptor lacking serine or threonine phosphorylation sites have on a cell? 14. What is the purpose of A kinase-associated proteins (AKAPs)? Describe how AKAPs work in heart muscle cells. 15. Inositol 1,4,5-trisphosphate $\left(\mathrm{IP}_{3}\right)$ and diacylglycerol (DAG) are second messenger molecules derived from the cleavage of phosphatidylinositol 4,5-bisphosphate [PI(4,5) $\mathrm{P}_{2}$ ] by activated phospholipase C . Describe the role of $\mathrm{IP}_{3}$ in causing a rise in cytosolic $\mathrm{Ca}^{2+}$ concentration. How do cells restore resting levels of cytosolic $\mathrm{Ca}^{2+}$ ? What is the principal function of DAG?
13. In Chapter 3, the $K_{\mathrm{d}}$ of calmodulin EF hands for binding $\mathrm{Ca}^{2+}$ is given as $10^{-6} \mathrm{M}$. Many proteins have much higher affinities for their respective ligands. Why is the specific affinity of calmodulin important for $\mathrm{Ca}^{2+}$ signaling processes such as that initiated by production of $\mathrm{IP}_{3}$ ?
14. Most of the short-term physiological responses of cells to cAMP are mediated by activation of PKA. Another common second messenger is cGMP. What are the targets of cGMP in rod and smooth muscle cells?

## References

Studying Cell-Surface Receptors and Signal Transduction Proteins
Flock, T., et al. 2015. Universal allosteric mechanism for $\mathrm{G}_{\mathrm{a}}$ activation by GPCRs. Nature, doi:10.1038/nature14663.

Garland, S. 2013. Are GPCRs still a source of new targets? J. Biomol. Screen. 18:947-966.

Grecco, H., M. Schmick, and P. Bastiaens. 2011. Signaling from the living plasma membrane. Cell 144:897-909.

Gross, A., and H. F. Lodish. 2006. Cellular trafficking and degradation of erythropoietin and NESP. J. Biol. Chem. 281: 2024-2032.

Scott, J. D., et al., 2013. Creating order from chaos: cellular regulation by kinase anchoring. Annu. Rev. Pharmacol. 53:187-210.

Taylor, S. S., and A. Kornev. 2011. Protein kinases: evolution of dynamic regulatory proteins. Trends Biochem. Sci. 36:65-77.

## G Protein-Coupled Receptors: Structure and Mechanism

Audel, M., and M. Bouvier. 2012. Restructuring G-proteincoupled receptor activation. Cell 151:14-23.

Benovic, J. 2012. G-protein-coupled receptors signal victory. Cell 151:1-3. (A review of the research leading to the 2011 Nobel Prize.)

Irannejad, R., et al. 2013. Conformational biosensors reveal GPCR signalling from endosomes. Nature 495:534-538.

Tesmer, J. 2010. The quest to understand heterotrimeric G protein signalling. Nat. Struct. Mol. Biol. 17:650-652.

## G Protein-Coupled Receptors That Regulate Ion Channels

Calvert, P., et al. 2006. Light-driven translocation of signaling proteins in vertebrate photoreceptors. Trends Cell Biol. 16:560-568.

Hofmann, K. P., et al. 2009. A G protein-coupled receptor at work: the rhodopsin model. Trends Biochem. Sci. 34:540-552.

Pearring, J. N. 2013. Protein sorting, targeting and trafficking in photoreceptor cells. Prog. Retin. Eye Res. 36:24-61.

Smith, S. O. 2010. Structure and activation of the visual pigment rhodopsin. Annu. Rev. Biophys. 39:309-328.

## G Protein-Coupled Receptors That Activate or Inhibit Adenylyl Cyclase

Agius, L. 2010. Physiological control of liver glycogen metabolism: lessons from novel glycogen phosphorylase inhibitors. Mini-Rev. Med. Chem. 10:1175-1187.

DeWire, S., et al. 2007. $\beta$-Arrestins and cell signaling. Annu. Rev. Physiol. 69:483-510.

Lefkowitz, R. J., and S. K. Shenoy. 2005. Transduction of receptor signals by $\beta$-arrestins. Science 308:512-517.

Shula, A., et al. 2014. Visualization of arrestin recruitment by a G-protein-coupled receptor. Nature 512:218-222.

Somsak, L., et al. 2008. New inhibitors of glycogen phosphorylase as potential antidiabetic agents. Curr. Med. Chem. 15: 2933-2983.

Taylor, S. S., et al. 2012. Assembly of allosteric macromolecular switches: lessons from PKA. Nat. Rev. Mol. Cell Biol. 13:646-658.

## G Protein-Coupled Receptors That Trigger Elevations in Cytosolic and Mitochondrial Calcium

Hoflich, K. P., and M. Ikura. 2002. Calmodulin in action: diversity in target recognition and activation mechanisms. Cell 108:739-742.

Hogan, P. G., R. S. Lewis, and A. Rao. 2010. Molecular basis of calcium signaling in lymphocytes: STIM and ORAI. Annu. Rev. Immunol. 28:491-533.

Kaufman, R., and J. Amphora. 2014. Calcium trafficking integrates endoplasmic reticulum function with mitochondrial bioenergetics. Biochim. Biophys. Acta 1843:2233-2239.

Parekh, A. 2011. Decoding cytosolic $\mathrm{Ca}^{2+}$ oscillations. Trends Biochem. Sci. 36:78-87.

Soboloff, J., et al. 2012. STIM proteins: dynamic calcium signal transducers. Nat. Rev. Mol. Cell Biol. 13:549-565.

Zhou, Y., et al. 2010. Pore architecture of the ORAI1 store-operated calcium channel. P. Natl Acad. Sci. USA 107: 4896-4901.

![li2544542480e](li2544542480e.jpg)

A molecular valentine-dimerized extracellular domain of the epidermal growth factor receptor (red) bound to two molecules of epidermal growth factor (pink). [Data from H. Ogiso et al., 2002, Cell 110:775, PDB ID 1ivo.]

Extracellular signals can have both short- and long-term effects on cells. Short-term effects are usually triggered by modification of existing enzymes or other proteins, as we saw in Chapter 15. Many extracellular signals also affect gene expression and thus induce long-term changes in cell function. These changes include alterations in cell division and development, such as those that occur during cell fate determination and differentiation. The body's production of red blood cells, white blood cells, and platelets in response to the cytokines we discuss later in this chapter is a good example of signalinduced changes in gene expression that influence cell proliferation and differentiation. Changes in gene expression also enable differentiated cells to respond to their environment by changing their shape, metabolism, or movement. In immunesystem cells, for example, several hormones activate one type of transcription factor (NF- $\kappa \mathrm{B}$ ) that ultimately affects the expression of more than 150 genes involved in the immune response to infection. Given the extensive role of gene transcription in mediating critical aspects of development, metabolism, and movement, it is not surprising that mutations in

## G

## Signaling Pathways That Control Gene Expression

such signaling pathways cause many human diseases, including cancer, diabetes, and immune-system disorders.

In this chapter, we explore the main signaling pathways that cells use to influence gene expression. In eukaryotes, there are about a dozen classes of highly conserved cellsurface receptors, and these receptors activate several types of highly conserved intracellular signal transduction pathways. Many of these pathways consist of multiple kinases, GTP-binding proteins, other regulatory proteins, small intracellular molecules, and ions such as $\mathrm{Ca}^{2+}$, which together form a complex signaling cascade. Given this complexity, cell signaling can seem a daunting subject to learn for the first time; the many names and abbreviations of the molecules found in each pathway can indeed be challenging. The subject repays careful study, however: when one becomes familiar with these pathways, one understands in a profound way the regulatory mechanisms that control a vast array of biological processes.

For simplicity, signal transduction pathways can be grouped into several basic types, based on the sequence of

### 16.1 Receptor Serine Kinases That Activate Smads

### 16.2 Cytokine Receptors and the JAK/STAT Signaling Pathway

16.3 Receptor Tyrosine Kinases
16.4 The Ras/MAP Kinase Pathway
16.5 Phosphoinositide Signaling Pathways

### 16.6 Signaling Pathways Controlled by Ubiquitinylation and Protein Degradation: Wnt, Hedgehog, and NF- $\kappa B$

16.7 Signaling Pathways Controlled by Protein Cleavage: Notch/Delta, SREBP, and Alzheimer's Disease

### 16.8 Integration of Cellular Responses to Multiple Signaling Pathways: Insulin Action

intracellular events. In one very common type of signal transduction pathway (Figure 16-1a), the binding of a ligand to a receptor triggers activation of a receptor-associated kinase. These receptors generally have a single transmembrane domain and are activated by ligand-induced receptor dimerization. The kinase may be an intrinsic part of the cytosolic domain of the receptor protein or may be tightly bound to the cytosolic domain of the receptor. These kinases often directly phosphorylate and activate a variety of signal-transducing proteins, including transcription factors located in the cytosol (Figure 16-1a, step 1). Some receptor kinases also activate small GTP-binding "switch" proteins such as Ras (Figure 16-1a, step 1). Many signal transduction pathways, such as those activated by Ras, involve several kinases in which one kinase phosphorylates and thus activates (or occasionally inhibits) the activity of another kinase; one or more of these kinases eventually phosphorylate and activate transcription factors.

Most other receptors become activated by conformational changes induced by ligand binding. Some, mainly the receptors
with seven membrane-spanning segments introduced in Chapter 15, activate GTP-binding $\mathrm{G}_{\mathrm{sr}}$ proteins (Figure 16-1b). Their activation usually leads to activation of one or more protein kinases, which in turn phosphorylate and activate multiple target proteins, including transcription factors.

In yet other signaling pathways, binding of a ligand to a receptor triggers disassembly of a multiprotein complex in the cytosol, releasing a transcription factor that then translocates into the nucleus and affects gene expression (Figure 16-1c). Finally, in the last common type, proteolytic cleavage of an inhibitor or of the receptor itself releases an active transcription factor, which then travels into the nucleus (Figure 16-1d). While every signaling pathway has its own subtleties and distinctions, nearly every one can be grouped into one of these basic types.

The pathways we discuss in this chapter have been conserved throughout evolution and operate in much the same manner in flies, worms, planaria, and humans. The substantial homology exhibited among proteins in these
![li2554542480e](li2554542480e.jpg)

FIGURE 16-1 Several common types of cell-surface receptors and signal transduction pathways. (a) The cytosolic domains of many receptors are protein kinases or are tightly associated with a cytosolic kinase; commonly the kinases are activated by ligand binding followed by receptor dimerization. Some of these kinases directly phosphorylate and activate transcription factors $\boldsymbol{B}$ or other signaling proteins. Many of these receptors also activate small GTP-binding "switch" proteins such as Ras $\boldsymbol{B}$. Many signal transduction pathways, such as those activated by Ras, involve several kinases; in these pathways, one kinase phosphorylates and thus activates (or occasionally inhibits) the activity
of another kinase. Many of the kinases in these pathways phosphorylate multiple protein targets, including transcription factors, which are usually different in different cells. (b) Other receptors, mainly those with seven membrane-spanning segments, activate the larger GTPbinding $\mathrm{G}_{\mathrm{sr}}$ proteins, which in turn activate specific kinases or other signaling proteins. (c) Several signaling pathways involve disassembly of a multiprotein complex in the cytosol, releasing a transcription factor that then translocates into the nucleus. (d) Some signaling pathways are irreversible; in many cases, proteolytic cleavage of a receptor releases an active transcription factor.

pathways has enabled researchers to study them in a variety of experimental systems. For instance, the secreted signaling protein Hedgehog (Hh) and its receptor were first identified in Drosophila mutants that had impaired development. Subsequently, the human and mouse homologs of these proteins were cloned and shown to participate in a number of important signaling events during cell differentiation, resulting in the discovery that abnormal activation of the Hh pathway occurs in several human tumors. Such discoveries illustrate the importance of studying signaling pathways both genetically-in flies, mice, worms, yeasts, and other organisms-and biochemically.

Many receptors are expressed in multiple types of body cells, but activation of these receptors by the same hormone triggers induction (or repression) of very different sets of genes in each cell type. In other words, the same activated transcription factor will induce (or repress) the expression of different sets of genes in different types of cells. To understand this key point, recall from Chapter 9 that the expression of every gene in higher organisms is regulated by multiple transcription factors and chromatin-modifying enzymes, and that many genes are expressed in each of multiple types of cells at very precise but often different levels.

Whether a transcription factor activated by a cell-surface receptor induces (or represses) a gene in a particular cell depends, first, on the epigenetic state of the cell determined by its developmental history (Figure 16-2). As we learned in Chapter 9, the epigenetic state dictates whether a gene is in an active "open" chromatin conformation, and therefore
accessible to binding by the transcription factor, or in a silenced "closed" state that is not accessible for transcriptional regulation. In other words, a given transcription factor can potentially bind to multiple gene regulatory sites in chromosomal DNA, but in any given cell type only a fraction of these sites will be accessible for binding.

Second, other transcription factors, histone readers and modifiers, and chromatin remodelers that interact with the particular activated transcription factor determine what genes will be induced or repressed by the binding of that factor to gene regulatory sequences. In particular, many cell types express one or more master transcription factors that determine the identity and developmental fate of the cell; a recent finding is that many transcription factors activated by cellsurface receptors bind to chromosomal DNA at regulatory sites-mainly enhancers-adjacent to these master factors and thus induce (or repress) cell-specific genes (see Figure 16-2).

No signaling pathway acts in isolation. Many cells respond to multiple types of hormones and other signaling molecules; some mammalian cells express roughly a hundred different types of cell-surface receptors, each of which binds a different ligand. Since many genes are regulated by multiple transcription factors, which in turn are activated or repressed by different intracellular signaling pathways, expression of any one gene can be regulated by multiple extracellular signals. Especially during early development, such "cross talk" between signaling pathways and the resultant sequential alterations in the pattern of gene expression eventually can become so extensive that the cell assumes
![li2564542480e](li2564542480e.jpg)

FIGURE 16-2 Induction of a particular gene by a transcription factor depends not only on binding sites for the factor, but also on the gene's epigenetic state and on the presence of master transcription factors and other nuclear proteins. Any given activated transcription factor has multiple sites on the chromosomal DNA to which it can potentially bind (green), but in any given cell it will bind only to those
sites that are in an "open chromatin" conformation and in which specific master transcription factors or other cell-specific proteins (here colored blue and red, respectively) are bound to adjacent sites on the DNA. Other potential transcription factor binding sites are adjacent to binding sites for other master transcription factors (yellow) that are not expressed in this cell type, and thus the transcription factor will not bind to those sites.

a different developmental fate. In this chapter, we will see how multiple signaling pathways interact to regulate crucial aspects of metabolism, such as the level of glucose in the blood and the formation of adipose (fat-storing) cells.

We begin with a discussion of a large class of receptorsthe receptor serine kinases-that, when activated by their corresponding hormones, directly phosphorylate and activate one or more transcription factors that move directly into the nucleus. We will see precisely how these transcription factors activate very different genes in different cell types depending on the epigenetic state of the chromatin and the presence of master transcription factors and other cellspecific proteins.

### 16.1 Receptor Serine Kinases That Activate Smads

In this section, we discuss an evolutionarily conserved family of receptor serine kinases (the TGF- $\beta$ receptor family) and the conserved large family of signaling molecules (the TGF- $\beta$ family) that bind to them. These receptors phosphorylate, and thus trigger the activation of, one conserved class of transcription factors (the Smads) that regulate several growth and differentiation pathways. In unstimulated cells, Smads are located in the cytosol, but when activated, they move into the nucleus to regulate transcription. The TGF- $\beta$ pathway has widely diverse effects in different types of cells because different members of the TGF- $\beta$ family activate different members of the TGF- $\beta$ receptor family, which activate different members of the Smad class of transcription factors. In addition, we will see that the same activated Smad protein partners with different transcription factors, and thus activates different sets of genes, in different types of cells.

The transforming growth factor $\beta$ (TGF- $\beta$ ) family includes a number of related extracellular signaling molecules that play widespread roles in regulating development in both invertebrates and vertebrates. The founding member of the TGF- $\beta$ family, TGF- $\beta 1$, was identified on the basis of its ability to induce a malignant phenotype in several cultured early-stage mammalian cancer cell lines ("transforming growth factor"); in this case, TGF- $\beta 1$ promoted metastases, the spreading and invasion of primary tumors, as discussed in Chapter 24. However, the principal function of all three human TGF- $\beta$ isoforms, TGF- $\beta 1,2$, and 3 , in most normal (non-cancerous) mammalian cells is to prevent their proliferation by inducing the synthesis of proteins, including p $15^{\text {INK4B }}$, that inhibit the cyclin-dependent kinases (CDKs) that are essential for progression into the S phase of the cell cycle (see Figures 1-21 and 19-10).

TGF- $\beta$ is produced by many cells in the body and inhibits the growth of both the secreting cell (autocrine signaling) and neighboring cells (paracrine signaling). Loss of TGF- $\beta$ receptors, or of any of several intracellular signaltransducing proteins in the TGF- $\beta$ pathway, releases cells from this growth inhibition and is seen frequently in the early development of human tumors. TGF- $\beta$ proteins
also promote expression of cell-adhesion molecules and extracellular-matrix molecules, which play important roles in tissue organization (see Chapter 20). A Drosophila homolog of TGF- $\beta$, called Dpp, participates in dorsal-ventral patterning in fly embryos. Other mammalian members of the TGF- $\beta$ family, the activins and the inhibins, affect early development of the urogenital tract. Another member of this family, a bone morphogenetic protein (BMP), was initially identified by its ability to induce bone formation in cultured cells. Now called BMP7, it is used clinically to strengthen bone after severe fractures. Of the numerous BMP proteins subsequently recognized, many induce key steps in development, including formation of mesoderm and of the earliest blood-forming cells; some are important for maintaining the undifferentiated state in cultures of embryonic and adult stem cells (see Chapter 21). Most have nothing to do with bones.

## TGF- $\beta$ Proteins Are Stored in an Inactive Form in the Extracellular Matrix

TGF- $\beta$ is synthesized with a long N-terminal prodomain that is cleaved off in the Golgi complex. The monomeric form of TGF- $\beta$ contains three conserved intramolecular disulfide linkages. An additional cysteine in the center of each monomer links TGF- $\beta$ monomers into functional homodimers and heterodimers (see Figure 16-3a), which are formed in the endoplasmic reticulum. The prodomain remains noncovalently attached to the TGF- $\beta$ growth-factor domain as the protein is secreted and prevents binding of TGF- $\beta$ to its cellsurface receptors. The latent prohormone-TGF- $\beta$ complex is stored near the secreting cell, attached to specific components of the extracellular matrix.

Several mechanisms, including protease digestion, release the active TGF- $\beta$ from the extracellular matrix and lead to quick activation of cell signaling-an important feature of many signaling pathways. A second important and unusual way of activating TGF- $\beta$ emerged from the recognition that the prodomains of TGF- $\beta 1$ and TGF- $\beta 3$ contain a three-amino-acid motif, RGD (Arg-Gly-Asp), that binds to a class of cell-surface membrane proteins termed integrins (discussed in Chapter 20); integrins bind to RGD motifs in many extracellular-matrix proteins. Remarkably, binding of either of two integrins, termed $\alpha v \beta 6$ or $\alpha v \beta 8$, to the RGD motif in the TGF- $\beta$ prodomain, followed by contraction of the integrin-expressing cell, physically pulls the prodomain away from the latent TGF- $\beta$, freeing it to bind to cell-surface receptors on the same cell and initiating signaling.

## Three Separate TGF- $\beta$ Receptor Proteins Participate in Binding TGF- $\boldsymbol{\beta}$ and Activating Signal Transduction

Researchers soon identified TGF- $\beta 1$ as a key growth inhibitory factor, but to understand the way it worked, they had to find the receptors to which it bound. The logic of how they went about their search is representative of

typical biochemical approaches to identifying receptors (see Section 15.2). Investigators first reacted the purified growth factor with the radioisotope iodine-125 ( ${ }^{125} \mathrm{I}$ ) under conditions that caused the iodine to become covalently linked to exposed tyrosine residues, tagging them with a radioactive label. The ${ }^{125}$ I-labeled TGF- $\beta$ protein was then incubated with cultured cells, and the incubation mixture was treated with a chemical agent that covalently cross-linked the labeled TGF- $\beta$ to its receptors on the cell surface. Purification of the ${ }^{125}$ I-labeled TGF- $\beta$-receptor complexes revealed three different polypeptides with molecular weights of 55,85 , and 280 kDa , referred to as RI, RII, and RIII TGF- $\beta$ receptors, respectively.

Figure 16-3 (steps $\mathbf{\square}$ and $\mathbf{\square}$ ) depicts the relationship and function of the three TGF- $\beta$ receptor proteins. The most
abundant, RIII, also called $\beta$-glycan, is a cell-surface proteoglycan. A proteoglycan consists of a protein bound to glycosaminoglycan (GAG) chains such as heparan sulfate and chondroitin sulfate (see Figure 20-32). RIII, a transmembrane protein, binds and concentrates TGF- $\beta$ molecules near the cell surface, facilitating their binding to RII receptors. The RI and RII receptors are dimeric transmembrane proteins with serine/threonine kinases as part of their cytosolic domains. RII exhibits constitutive kinase activity; that is, it is active even when not bound to TGF- $\beta$. Binding of TGF- $\beta$ to RII generates a new molecular surface at the TGF- $\beta$-RII interface that docks to RI, inducing the formation of complexes containing two copies each of RI and RII-an example of ligand-induced receptor hetero-oligomerization, which we will encounter often in this chapter. An RII subunit then
![li2574542480e](li2574542480e.jpg)

FIGURE 16-3 TGF- $\beta$ /Smad signaling pathway. (a) Ribbon diagram structure of a mature TGF- $\beta$ dimer. The three intrachain disulfide linkages (yellow) in each monomer form a cystineknot domain; another disulfide bond (red) links the two monomers. (b) Step $\mathbf{B}$ In some cells, TGF- $\beta$ binds to the type III TGF- $\beta$ receptor (RIII), which presents TGF- $\beta$ to the type II receptor (RII). Step $\mathbf{B}$ In other cells, TGF- $\beta$ binds directly to RII, a constitutively active kinase. Step Ligand-bound RII recruits and phosphorylates the juxtamembrane segment of the type I TGF- $\beta$ receptor (RI), which does not directly bind TGF- $\beta$. This releases the inhibition of RI kinase activity. Step $\mathbf{B}$ Activated RI then phosphorylates Smad2 or Smad3 (shown here as Smad2/3), causing a conformational change that unmasks its nuclearlocalization signal (NLS). Step Two phosphorylated molecules of Smad2/3 bind to a co-Smad (Smad4) molecule, which is not phosphorylated, and to an importin, forming a large cytosolic complex. Steps $\mathbf{B}$ and $\mathbf{B}$ After the entire complex translocates into the nucleus, Ran-GTP causes dissociation of the importin, as discussed in Chapter 13. Step A nuclear transcription factor (e.g., TFE3) then associates with the Smad2/3/ Smad4 complex, forming an activation complex that cooperatively binds to regulatory sequences of a target gene. Step This complex then recruits transcriptional co-activators and induces gene transcription (see Chapter 9). Smad2/3 is dephosphorylated by a nuclear phosphatase (step ) and recycles through a nuclear pore to the cytosol (step $\mathbf{B}$ ), where it can be reactivated by another TGF- $\beta$ receptor complex. Shown at the bottom is the activation complex for the gene encoding plasminogen activator inhibitor (PAI-1); similar transcription complexes activate expression of genes encoding other extracellular-matrix proteins such as fibronectin. See A. Moustakas and C.-H. Heldin, 2009, Development 136:3699, and D. Clarke and X. Liu, 2008, Trends Cell Biol. 18:430. [Part (a) data from S. Daopin et al., 1992, Science 257:369, PDB ID 2tgi.]

phosphorylates serine and threonine residues in a highly conserved sequence of the RI subunit adjacent to the cytosolic face of the plasma membrane, thereby activating the RI kinase activity.

## Activated TGF- $\boldsymbol{\beta}$ Receptors Phosphorylate Smad Transcription Factors

Researchers identified the transcription factors downstream from the TGF- $\beta$ receptors in studies of Drosophila and C. elegans mutants. These transcription factors in Drosophila and the related vertebrate proteins are now called Smads. Three types of Smad proteins function in the TGF- $\beta$ signaling pathway: R-Smads (receptor-regulated Smads; Smads 2 and 3), co-Smads (Smad4), and I-Smads (inhibitory Smads; Smad7).

As illustrated in Figure 16-3, an R-Smad (Smad2 or Smad3) contains two domains, termed MH1 and MH2, separated by a flexible linker region. The N-terminal MH1 domain contains a DNA-binding segment as well as a domain called the nuclear-localization signal (NLS). NLSs are present in virtually all transcription factors found in the cytosol and are required for their transport into the nucleus (see Chapter 13). However, when R-Smads are in their inactive, nonphosphorylated state, the NLS is masked so that it cannot bind to an importin (see Figure 13-36), and the MH1 and MH2 domains associate in such a way that they cannot bind to DNA or to a co-Smad. Phosphorylation of two serine residues near the C-terminus of an R-Smad by activated TGF- $\beta$ RI receptors separates the two domains, exposing the NLS and permitting binding of an importin, which catalyzes entrance of the Smad into the nucleus.

Simultaneously, the two serines in each Smad3 that were phosphorylated by the RI receptor kinase bind to phosphoserine-binding sites in the MH2 domains of a Smad3 or a Smad4, forming a stable complex containing two molecules of phosphorylated Smad3 (or Smad2) and one molecule of the co-Smad (Smad4). The bound importin then mediates translocation of the heteromeric R-Smad/co-Smad complex into the nucleus. After importin dissociates inside the nucleus, the Smad3/Smad4 (or Smad2/Smad4) complex binds to other transcription factors to activate transcription of specific target genes.

Within the nucleus, R-Smads are further modified by phosphorylation of their linker domains, acetylation of their MH1 domains, monoubiquitinylation of their MH2 domains, and dephosphorylation of the C-terminal serines by nuclear phosphatases. Collectively, these many modifications result in regulation of transcriptional activity and ultimately in dissociation of the R-Smad/co-Smad complex and export of the Smads from the nucleus via exportins. Thus the concentration of active Smads within the nucleus closely reflects the levels of activated TGF- $\beta$ receptors on the cell surface, allowing transcriptional regulation to closely follow the level of active TGF- $\beta$ in the environment.

BMP proteins, which also belong to the TGF- $\beta$ family, bind to and activate a different set of receptors that are
similar to the TGF- $\beta$ RI and RII proteins, but phosphorylate other R-Smads. Two of these phosphorylated Smads then form a trimeric complex with Smad4, and this Smad complex activates different transcriptional responses than those induced by the TGF- $\beta$ receptor.

## The Smad3/Smad4 Complex Activates Expression of Different Genes in Different Cell Types

Virtually all mammalian cells secrete at least one TGF- $\beta$ isoform, and most have TGF- $\beta$ receptors on their surface. However, the cellular responses induced by TGF- $\beta$ vary among cell types. In epithelial cells and fibroblasts, for example, TGF- $\beta$ induces expression of extracellular-matrix proteins (e.g., fibronectins and collagens; see Chapter 20). It also induces expression of proteins that inhibit serum proteases, which otherwise would degrade these extracellular-matrix proteins. This inhibition stabilizes the matrix, allowing cells to form stable tissues. The protease-inhibitory proteins include plasminogen activator inhibitor 1 (PAI-1). Transcription of the PAI-1 gene requires formation of a complex of the transcription factor TFE3 with the R-Smad/co-Smad (Smad3/Smad4) complex and binding of all these proteins to specific sequences within the regulatory region of the PAI-1 gene (Figure 16-3, bottom). By partnering with other transcription factors expressed in fibroblasts, the R-Smad/coSmad complex promotes expression of genes encoding other proteins such as p15 $5^{\text {INK4B }}$, which arrests the cell cycle at the $\mathrm{G}_{1}$ stage and thus blocks cell proliferation (see Chapter 19).

More generally, in order for the Smad3/Smad4 complex to bind to a DNA regulatory region and activate a given gene, the DNA binding segment must be in an active "open" chromatin conformation. Equally important, binding of the Smad3/Smad4 complex requires binding of other transcription factors at adjacent sites in the DNA; often these transcription factors are master transcription factors that determine the identity of a cell during its development. For example, we will learn in Chapter 21 that the transcription factor Oct4 forms a complex with two other transcription factors, Sox2 and Nanog, that are essential for keeping embryonic stem (ES) cells in an undifferentiated state, and that expression of these three master transcription factors, together with one or two other transcription factors, is sufficient to reprogram a differentiated cell (e.g., a cell of the immune system) into an ES cell. TGF- $\beta$ activation of the Smad3/Smad4 complex is also required to maintain ES cells in an undifferentiated state.

The chromatin immunoprecipitation-DNA sequence experiment summarized in Figure 16-4a shows that in ES cells, the Smad3/Smad4 complex binds to DNA at sites adjacent to those occupied by Oct4; similar experiments showed that Sox2 and Nanog are also bound at most of these sites. In contrast, MyoD and the related transcription factor Myf5 are the master transcription factors in development of striated muscle, and Oct4, Sox2, and Nanog are not expressed; in muscle cells, the Smad3/Smad4 complex binds to gene

![li2584542480e](li2584542480e.jpg)

EXPERIMENTAL FIGURE 16-4 The Smad3/Smad4 complex binds to DNA at sites adjacent to those occupied by cell-specific master transcription factors. (a) In human embryonic stem cells (ES cells), the Smad3/Smad4 complex binds at sites near those occupied by the ES master transcription factor Oct4. Chromatin immunoprecipitation studies were performed using antibodies specific for Smad3 or Oct4 and the DNA sequences bound to these factors were determined; this process is often abbreviated "ChIP-Seq." Plotted on the $y$ axis are the top 1000 sites bound by Oct4; the intensity of the red color is proportional to the amount of DNA at this site bound by Oct4 within 2500 bases upstream (-) or downstream (+) of this binding site. The left panel shows binding of Smad3 to DNA sequences within 2500 bases of these 1000 Oct4 sites; it is apparent that most of the Oct4 sites have Smad3 bound near them. Similar plots centered around the top 1000 Smad3 sites showed that over 80 percent of the sites with bound Smad3 also had a bound Oct4. (b) Smad3 can regulate the same gene in different cell types by binding to regulatory DNA sequences occupied by different cell-specific master transcription factors. In this hypothetical scenario, the Smad3/Smad4 complex (green) activates the same gene in ES cells and in muscle cells. In ES cells, it co-occupies a regulatory site with Oct4 (red, binding to a site in DNA colored red), and in muscle, it co-occupies a different regulatory site with the muscle master transcriptional regulator, MyoD (blue). [Part (a) data from A. Mullen et al., 2011, Cell 147:565.]
![li2594542480e](li2594542480e.jpg)
regulatory sites adjacent to those occupied by MyoD, which are very different DNA sites from those that bind the Smad3/Smad4 complex in ES cells. About 13 percent of the genes in muscle whose regulatory regions are bound by the Smad3/Smad4 complex in muscle cells are also bound by the Smad3/Smad4 complex in ES cells, but detailed analyses of these binding sites showed that the Smad3/Smad4 complex regulates the same gene in these different cell types by binding to different regulatory DNA sequences, most presumably enhancer regions, which are occupied by different cell-specific master transcription factors (Figure 16-4b). As we noted earlier, other types of transcription factors activated by other types of cell-surface receptors also activate or repress different sets of genes in different types of cells. We will see in this chapter that those transcription factors regulate those genes in the same manner as the Smads activated by the TGF- $\beta$ receptors.

5Loss of TGF- $\beta$ signaling plays a key role in the early development of many cancers. Many human tumors contain inactivating mutations in either TGF- $\beta$ receptors or Smad proteins and thus are resistant to growth inhibition by TGF- $\beta$ (see Figure 24-24). Most human pancreatic cancers,
for instance, contain a deletion in the gene encoding Smad4 and thus cannot induce cell cycle inhibitors in response to TGF- $\beta$. In fact, Smad4 was originally called DPC (deleted in pancreatic cancer). Retinoblastoma, colon and gastric cancer, hepatoma, and some T- and B-cell malignancies are also unresponsive to TGF- $\beta$ growth inhibition. This loss of responsiveness correlates with loss of TGF- $\beta$ RI or RII; responsiveness to TGF- $\beta$ can be restored by recombinant expression of the "missing" protein. Mutations in Smad2 also commonly occur in several types of human tumors.

## Negative Feedback Loops Regulate TGF- $\beta$ /Smad Signaling

In most signaling pathways, the response to a growth factor or other signaling molecule decreases with time (a phenomenon called desensitization). This response is adaptive because it prevents overreaction and makes fine-tuned control of cellular responses possible. Two cytosolic proteins called SnoN and Ski (Ski stands for "Sloan-Kettering Cancer Institute") are induced by TGF- $\beta$ signaling in virtually all body cells and serve to down-modulate the TGF- $\beta / \mathrm{Smad}$ signaling pathway. These proteins were originally identified

![li2604542480e](li2604542480e.jpg)

FIGURE 16-5 Model of Ski-mediated down-regulation of Smad transcription-activating function. Ski represses Smad function by binding directly to Smad4. Since the Ski-binding domain on Smad4 significantly overlaps with the Smad4 MH2 domain required for binding the phosphorylated tail of Smad3, binding of Ski disrupts the normal interactions between Smad3 and Smad4 necessary for transcriptional activation. In addition, Ski recruits the protein N-CoR, which binds directly to mSin3A; in turn, mSin3A interacts with histone deacetylase (HDAC), an enzyme that promotes histone deacetylation on nearby promoters, repressing gene expression (see Chapter 9). As a result of both processes, transcription activation induced by TGF- $\beta$ and mediated by Smad complexes is shut down. The related protein SnoN functions similarly to Ski in repressing TGF- $\beta$ signaling. See J. Deheuninck and K. Luo, 2009, Cell Res. 19:47.
as cancer-causing oncoproteins because their expression is elevated in many cancers, including melanomas and certain breast cancers, because growth-inhibitory proteins normally induced by TGF- $\beta$ are not produced. Indeed, when overexpressed in cultured primary fibroblast cells, Ski or SnoN can cause abnormal cell proliferation, and down-regulation of Ski in pancreatic cancers reduces tumor growth.

How SnoN and Ski trigger abnormal cell proliferation was not understood until years later, when they were found to bind to both the co-Smad (Smad4) and phosphorylated R-Smads (Smad3) after TGF- $\beta$ stimulation. SnoN and Ski do not prevent formation of an R-Smad/co-Smad complex or affect the ability of a Smad complex to bind to DNA regulatory regions. Rather, they block transcription activation by a bound Smad complex, in part by inducing deacetylation of histones in adjacent chromatin segments. This renders the cell resistant to the growth-inhibitory effects of TGF- $\beta$ (Figure 16-5). The increased levels of these proteins induced by TGF- $\beta$ are thought to dampen long-term signaling effects due to continued exposure to TGF- $\beta$; this is another example of negative feedback, in which a gene induced by TGF signaling, in this case SnoN, inhibits further signaling by TGF- $\beta$.

Among the other proteins induced after TGF- $\beta$ stimulation are the I-Smads, especially Smad7. Smad7 blocks the ability of activated type I TGF- $\beta$ receptors (RI) to phosphorylate R-Smad proteins, and Smad7 may also target TGF- $\beta$ receptors for degradation. In these ways, Smad7, like Ski and SnoN, participates in a negative feedback loop: its induction inhibits intracellular signaling by long-term exposure to the stimulating hormone.

## KEY CONCEPTS OF SECTION 16.1

## Receptor Serine Kinases That Activate Smads

- The transforming growth factor $\beta$ (TGF- $\beta$ ) family includes a number of related extracellular signaling molecules that play widespread roles in regulating development.
- TGF- $\beta$ dimers are stored in an inactive form on the cell surface or in the extracellular matrix; release of active dimers (e.g., by mechanical stretching of the prodomain or protease digestion) initiates TGF- $\beta$ signaling.
- TGF- $\beta$ receptors consist of three types (RI, RII, RIII). Binding of members of the TGF- $\beta$ family to the RII receptor kinase allows RII to phosphorylate the cytosolic domain of the RI receptor and activate its intrinsic serine/threonine kinase activity. RI then phosphorylates an R-Smad, exposing a nuclear-localization signal (see Figure 16-3).
- After phosphorylated R-Smads bind a co-Smad, the resulting complex translocates into the nucleus, where it interacts with various transcription factors to induce expression of target genes (see Figure 16-3).
- The Smad3/Smad4 complex induces different genes in different cells by binding to regulatory DNA sequences at sites adjacent to those occupied by cell-specific master transcription factors (see Figure 16-4).
- TGF- $\beta$ signaling generally inhibits cell proliferation. Loss of various components of this signaling pathway contributes to abnormal cell proliferation and malignancy.
- Oncoproteins (e.g., Ski and SnoN) and I-Smads (e.g., Smad7) act as negative regulators of TGF- $\beta$ signaling by inhibiting transcription mediated by the Smad2/3/Smad4 complex (see Figure 16-5).


### 16.2 Cytokine Receptors and the JAK/STAT Signaling Pathway

In this section and the next, we discuss two large classes of receptors that activate protein tyrosine kinases. Protein tyrosine kinases, of which there are about 90 in the human genome, phosphorylate specific tyrosine residues on target proteins, usually in the context of a specific linear sequence of amino acids in which the tyrosine is embedded. The phosphorylated target proteins can then activate one or more signaling pathways. These pathways are noteworthy because they regulate most aspects of cell proliferation, differentiation, survival, and metabolism.

There are two broad categories of receptors that activate tyrosine kinases: (1) those in which the tyrosine kinase enzyme is an intrinsic part of the receptor's polypeptide chain, called the receptor tyrosine kinases (RTKs), which we discuss in Section 16.3, and (2) those, such as cytokine receptors, in which the receptor and kinase are separate

![li2614542480e](li2614542480e.jpg)

FIGURE 16-6 Overview of signal transduction pathways triggered by receptors that activate protein tyrosine kinases. Both receptor tyrosine kinases (RTKs) and cytokine receptors activate multiple signal transduction pathways that ultimately regulate transcription of genes. (a) In the most direct pathway, mainly employed by cytokine receptors, a STAT transcription factor binds to the activated receptor, becomes phosphorylated, moves to the nucleus, and directly activates transcription. (b) Binding of one type of adapter protein (GRB2 or Shc)
polypeptides, encoded by different genes, yet are bound tightly together. In cytokine receptors, the tightly bound kinase is known as a JAK kinase. Both classes of receptors activate similar intracellular signal transduction pathways (Figure 16-6). We begin with the cytokine receptors, since they mainly employ a short signal transduction pathway called the JAK/STAT pathway: a STAT transcription factor binds to the activated receptor, becomes phosphorylated by the JAK kinase, moves to the nucleus, and directly activates transcription.

## Cytokines Influence the Development of Many Cell Types

The cytokines form a family of relatively small, secreted signaling molecules (generally containing about 160-200 amino acids) that control growth and differentiation of specific types of cells. One large family of cytokines, the interleukins, are essential for proliferation and functioning of the T cells and antibody-producing B cells of the immune system (see Chapter 23). Another family of cytokines, the interferons, are produced and secreted by certain cell types following viral infection and act on nearby cells to induce enzymes that render those cells more resistant to viral infection.

Growth hormone (GH), as its name implies, is a 191-amino-acid protein that stimulates proliferation of many types of body cells; it is made and secreted by cells in the anterior pituitary gland in response to another hormone, growth hormone-releasing hormone, that is made by the part of the brain termed the hypothalamus. GH was one of the first protein drugs to be made by recombinant DNA; it is used clinically to treat growth disorders in children and GH deficiency in adults. The bovine version is used to increase milk production in dairy cows. During pregnancy, a related hormone, the cytokine prolactin, induces epithelial cells lining the immature ductules of the mammary gland to differentiate into the acinar cells that produce milk proteins and secrete them into the ducts.
to an activated receptor leads to activation of the Ras/MAP kinase pathway (see Section 16.4). (c, d) Two phosphoinositide pathways are triggered by recruitment of phospholipase $\mathrm{C}_{\gamma}$ and $\mathrm{PI}-3$ kinase to the membrane (see Section 16.4). Elevated levels of $\mathrm{Ca}^{2+}$ and activated protein kinase B modulate the activity of transcription factors as well as of cytosolic proteins that are involved in metabolic pathways or cell movement or shape.

GH and prolactin have three-dimensional structures very similar to those of several cytokines that induce the formation of important types of blood cells. All blood cells are derived from hematopoietic stem cells, which form a series of progenitor cells that then differentiate into the mature blood cells (see Figure 21-17). For instance, the cytokine granulocyte colony-stimulating factor (G-CSF) induces a granulocyte progenitor cell in the bone marrow to divide several times and then differentiate into granulocytes, the type of white blood cells that inactivate bacteria and other pathogens. A related cytokine, thrombopoietin, stimulates a different progenitor cell to divide and differentiate into megakaryocytes, huge cells that fragment into the platelets that are essential for blood clotting.

A structurally related cytokine, erythropoietin (Epo), triggers production of erythrocytes (red blood cells) by inducing the proliferation and differentiation of erythroid progenitor cells in the bone marrow (Figure 16-7). Erythropoietin is synthesized by certain kidney cells. A drop in blood oxygen, such as that caused by loss of blood from a large wound, signifies a lower than optimal level of erythrocytes, whose major function is to transport oxygen complexed to hemoglobin. The transcription factor HIF-1 $\alpha$ is degraded in ambient oxygen levels. The kidney cells respond to low oxygen by preventing HIF-1 $\alpha$ degradation; HIF-1 $\alpha$ transcribes the erythropoietin gene, and the cells synthesize more erythropoietin and secrete it into the blood. As the level of erythropoietin rises, more and more erythroid progenitors are induced to divide and differentiate; each progenitor produces 30 to 50 erythrocytes in only a few days. In this way, the body can respond to the loss of blood by accelerating the production of erythrocytes.

GH, prolactin, G-CSF, thrombopoietin, and Epo undoubtedly evolved from a common ancestral protein, since all of these cytokines have a similar tertiary structure consisting of four long conserved $\alpha$ helices folded together.

![li2624542480e](li2624542480e.jpg)

FIGURE 16-7 Erythropoietin and formation of red blood cells (erythrocytes). Erythroid progenitor cells, called colony-forming unitserythroid (CFU-E), are derived from hematopoietic stem cells, which also give rise to progenitors of other blood cell types (see Figure 21-18). In the absence of erythropoietin (Epo), CFU-E cells undergo apoptosis (programmed cell death). Binding of Epo to its receptors on a CFU-E cell induces transcription of several genes whose encoded proteins prevent apoptosis, allowing the cell to survive. Other Epo-induced proteins trigger a developmental program of three to six terminal cell divisions, induction of hemoglobin and many other erythroidimportant genes, reduction in cell and nuclear size, and finally, loss of the cell nucleus. If CFU-E cells are cultured with Epo in a semisolid medium (e.g., containing methylcellulose), daughter cells cannot move away, and thus each CFU-E cell produces a colony of 30-100 erythroid cells; hence its name. See M. Socolovsky et al., 2001, Blood 98:3261.

8Both Epo and G-CSF are produced commercially by recombinant expression in cultured mammalian cells. Patients with kidney disease, especially those undergoing dialysis, frequently are anemic (have a low red blood cell count) and therefore are treated with recombinant Epo to boost red cell levels. Epo and G-CSF are used as adjuncts to certain cancer therapies because many cancer treatments affect the bone marrow and reduce production of red cells and granulocytes.

## Binding of a Cytokine to Its Receptor Activates One or More Tightly Bound JAK Protein Tyrosine Kinases

GH, prolactin, G-CSF, thrombopoietin, and Epo all have similar structures, and they activate receptors of similar structure by forming dimers of two identical cytokine receptor proteins, a process termed receptor homodimerization. Each of these receptor proteins then activates JAK2, the JAK kinase to which it is bound. The extracellular domains of these cytokine receptors are constructed of two subdomains, each of which contains seven conserved $\beta$ strands folded together in a characteristic fashion.

Many other cytokines, including the interleukins that activate immune-system cells and the interferons that induce expression of proteins that trigger resistance to viral infection, bind simultaneously to two or more different cytokine receptors, a process called receptor hetero-oligomerization. Generally these cytokine receptors bind to and activate two members of the JAK kinase family. Nonetheless, the signaling pathways activated by all cytokine receptors are broadly similar (see Figure 16-6), and receptor dimerization induced by hormone binding is common to many other receptor types that activate tyrosine kinases.

9Interleukin 2 (IL-2) is essential for the formation of functional T cells, an essential component of the immune system (see Chapter 23). The IL-2 receptor is an oligomer of three different subunits termed alpha, beta, and gamma. The gamma chain is also an essential subunit of the receptors for several other interleukins, including IL-4, IL-7, IL-9, IL-15, and IL-21, all cytokines that are essential for formation of the antibody-producing B cells and other types of immune-system cells. Severe combined immunodeficiency (SCID) is a genetic disease in which neither T nor B cells are produced. People with SCID cannot cope with any bacterial or viral infection, so they must be kept in a sterile environment (like the famous "bubble boy"). Many cases of SCID are due to a deficiency in the IL-2 receptor gamma chain. These children can now be cured by gene therapy: a viral vector (see Figure 6-30) is used to introduce a functional gamma chain gene into the hematopoietic stem cells that generate all immune-system cells (see Chapter 23).

Here we focus on the simpler case of receptor homodimerization. The interaction of one erythropoietin molecule with two identical erythropoietin receptor (EpoR) proteins, as depicted in Figure 16-8, exemplifies the binding of this group of cytokines to their receptors. The structure of the GH-GH receptor complex is similar, and detailed mutagenesis studies have shown that only eight amino acids in growth hormone (green) contribute 85 percent of the energy that is responsible for tight receptor binding; these amino acids are distant from one another in the primary sequence but adjacent in the folded protein (Figure 16-9). Because of the multiple weak, noncovalent forces (i.e., ionic, van der Waals, and hydrophobic interactions) involved in this binding as well as

![li2634542480e](li2634542480e.jpg)

Membrane
molecular complementarity between the interacting surfaces of the receptor and ligand, the binding is extremely tight; a dissociation constant $\left(K_{\mathrm{d}}\right)$ of about $10^{-10} \mathrm{M}$ is characteristic of most cytokine receptors. Thus very low concentrations of GH and most other cytokines-about 1 microgram per liter
![li2644542480e](li2644542480e.jpg)

EXPERIMENTAL FIGURE 16-9 Growth hormone binds to its receptor through multiple weak, noncovalent forces. (a) As determined from the three-dimensional structure of the 1 growth hormone:2 growth hormone receptor complex, 28 amino acids in the hormone are at the binding interface with one receptor molecule. To determine which amino acids are important in ligand-receptor binding, researchers mutated each of these amino acids, one at a time, to alanine and measured the effect on receptor binding. From this study, it was found that only 8 amino acids on growth hormone (green) contribute 85 percent of the energy that is responsible for tight receptor binding; these amino acids are distant from one another in the primary sequence, but

FIGURE 16-8 Structure of erythropoietin bound to an erythropoietin receptor. Like other cytokines, erythropoietin (Epo) contains four conserved long $\alpha$ helices that are folded in a particular arrangement. The activated erythropoietin receptor (EpoR) is a dimer of identical subunits; the extracellular domain of each monomer is constructed of two subdomains, each containing seven conserved $\beta$ strands folded in a characteristic fashion. Side chains of residues on two of the $\alpha$ helices in Epo, termed site 1, contact loops on one EpoR monomer, while residues on the two other Epo $\alpha$ helices, termed site 2, bind to the same loop segments in a second receptor monomer, thereby stabilizing the dimeric receptor in a specific conformation. [Data from R. S. Syed et al., 1998, Nature 395:511, PDB ID 1eer.]
(roughly one part in a billion)—are sufficient to activate cytokine receptors.

Cytokine receptors do not possess intrinsic enzyme activity. Rather, a JAK kinase is tightly bound to the cytosolic domain of the receptor (Figure 16-10). Each of the four members of the JAK family of kinases contains an N-terminal receptor-binding domain, a C-terminal kinase domain that is normally poorly active catalytically, and a middle "pseudokinase" domain that regulates kinase activity by an unknown mechanism. (JAKs are so named because when they were first cloned and characterized, their function was unknown; they were termed just another kinase.) These kinases become activated after ligand binding and receptor dimerization (Figure 16-10, step ■).
(b)
![li2654542480e](li2654542480e.jpg)
adjacent in the folded protein. Similar studies showed that two tryptophan residues (blue) in the receptor contribute most of the energy responsible for tight binding of growth hormone, although other amino acids at the interface with the hormone (yellow) are also important. (b) As with the Epo receptor, binding of growth hormone to one receptor molecule is followed by (c) binding of a second receptor (purple) to the opposite side of the hormone; this binding involves the same set of yellow and blue amino acids on the receptor, but different residues on the hormone. See B. Cunningham and J. Wells, 1993, J. Mol. Biol. 234:554, and T. Clackson and J. Wells, 1995, Science 267:383. [Data from A. M. de Vos, M. Ultsch, and A. A. Kossiakoff, 1992, Science 255:306, PDB ID 3hhr.]

![li2664542480e](li2664542480e.jpg)

FIGURE 16-10 General structure and activation of cytokine receptors. The cytosolic domain of a cytokine receptor binds tightly and irreversibly to a JAK protein tyrosine kinase. In the absence of ligand (step 1), two receptors form a homodimer, but the JAK kinases are poorly active. Ligand binding causes a conformational change that brings together the JAK kinase domains, which then phosphorylate
each other on a tyrosine residue in a region called the activation loop, activating the kinases (step 1). The active JAK kinases then phosphorylate multiple tyrosine residues in the receptor cytosolic domain (step 1). The resulting phosphotyrosines function as docking sites for signal-transducing proteins, including the STAT proteins.

Some cytokine receptors, such as the Epo receptor, are homodimers in the absence of ligand; others dimerize only in the presence of ligand. In both cases, ligand binding triggers a conformational change in the JAKs such that they can phosphorylate each other on a critical tyrosine in a region called the activation loop (Figure 16-10, step 1). As with many other kinases, phosphorylation of the activation loop leads to a conformational change in the kinase that enhances the affinity of the enzyme for ATP or the substrate to be phosphorylated, thereby increasing kinase activity (Figure 16-10, step 1). One piece of evidence for this activation mechanism comes from study of a mutant JAK2 in which the critical tyrosine is mutated to phenylalanine. The mutant JAK2 binds normally to EpoR, but cannot be phosphorylated and is catalytically inactive. In erythroid progenitor cells, expression of this mutant JAK2 in greater than normal amounts totally blocks EpoR signaling because the mutant JAK2 binds to the majority of EpoR proteins, preventing binding and functioning by the wildtype JAK2 protein. This type of mutation, referred to as a dominant-negative mutation, causes loss of function even in cells that carry copies of the wild-type gene because the mutant protein prevents the normal protein from functioning (see Chapter 6).

## Phosphotyrosine Residues Are Binding Surfaces for Multiple Proteins with Conserved Domains

Once the JAK kinases become activated, they first phosphorylate several tyrosine residues on the cytosolic domain of the cytokine receptor (see Figure 16-10). Several of these phosphotyrosine residues then serve as binding sites for signal-transducing proteins that have conserved phosphotyrosine-binding domains. One such phosphotyrosine-binding domain is called the SH2 domain. The SH2 domain derived its full name, the Src homology 2 domain, from its homology with a region in the prototypical Src cytosolic tyrosine kinase encoded by the src gene. (Src is an acronym for sarcoma, and a mutant form of the cellular src gene was found in chickens with sarcomas, as Chapter 24 details.) The three-dimensional structures of the SH2 domains in different signal-transducing proteins are very similar, but each binds to a distinct short sequence of amino acids surrounding a phosphotyrosine residue. The unique amino acid sequence of each SH2 domain determines the specific phosphotyrosine residues it binds (Figure 16-11). Variations in the hydrophobic socket in the SH2 domains of different proteins allow them to bind to phosphotyrosines adjacent to different sequences, accounting for the differences in their binding partners. The SH2 domain of the Src tyrosine kinase, for example, binds strongly to any peptide containing a critical four-residue core sequence: phosphotyrosine-glutamic acid-glutamic acid-isoleucine (see Figure 16-11). These four amino acids make intimate contact with the peptide-binding site in the Src SH2 domain. Binding resembles the insertion of a twopronged "plug"-the phosphotyrosine and isoleucine side chains of the peptide-into a two-pronged "socket" in the SH2 domain. The two glutamic acids fit snugly onto the surface of the SH2 domain between the phosphotyrosine socket and the hydrophobic socket that accepts the isoleucine residue. This specificity plays an important role in determining which signal-transducing proteins bind to which receptors and thus what pathways are activated.

![li2674542480e](li2674542480e.jpg)

FIGURE 16-11 Surface model of an SH2 domain bound to a phosphotyrosine-containing peptide. The peptide bound by this SH2 domain from Src tyrosine kinase (blue backbone with red oxygen atoms) is shown in stick form. The SH2 domain binds strongly to short target peptides containing a critical four-residue core sequence: phosphotyrosine (Tyr0 and $\mathrm{OPO}_{3}{ }^{-}$)-glutamic acid (Glu1)-glutamic

## SH2 Domains in Action: JAK Kinases Activate STAT Transcription Factors

To illustrate how binding of SH2 domains to specific phosphotyrosine residues induces specific signaling pathways, let's discuss the straightforward mechanism by which all JAK kinases, and some RTKs, directly activate members of the STAT family of transcription factors. All STAT proteins contain an N-terminal DNA-binding domain, an SH2 domain that binds to one or more specific phosphotyrosines in a cytokine receptor's cytosolic domain, and a C-terminal domain with a critical tyrosine residue. Once a monomeric STAT is bound to a phosphotyrosine in the receptor via its SH2 domain, the C-terminal tyrosine is phosphorylated by the associated JAK kinase (Figure 16-12a). This arrangement ensures that in a particular cell, only those STAT proteins with an SH2 domain that can bind to a particular receptor will be activated, and only when that receptor is activated. For example, the erythropoietin receptor, as well as the receptors for GH, prolactin, G-CSF, and several more cytokines, activates STAT5, but not STATs 1, 2, 3, or 4; those STATs are activated by other cytokine receptors. A phosphorylated STAT dissociates spontaneously from the receptor, and two phosphorylated STAT proteins form a homodimer in which the SH2 domain on each binds to the phosphotyrosine in the other (Figure 16-12b). Because dimerization involves conformational changes that expose the nuclear-localization signal (NLS), the STAT dimers move into the nucleus, where they bind to specific enhancers or promoters (DNA regulatory sequences) controlling target genes (see Figure 16-12a) and thus alter gene expression.
acid (Glu2)-isoleucine (Ile3). Binding resembles the insertion of a two-pronged "plug"-the phosphotyrosine and isoleucine side chains of the peptide-into a two-pronged "socket" in the SH2 domain. The two glutamate residues are bound to sites on the surface of the SH2 domain between the two sockets. [Data from G. Waksman et al., 1993, Cell 72:779, PDB ID 1sps.]

Because different cell types have unique complements of transcription factors and unique epigenetic modifications of their chromatin, the genes that are available to be activated by any STAT are different in different cell types. For example, in mammary gland cells, STAT5, the same STAT that is activated by EpoR in erythroid progenitor cells, becomes activated following prolactin binding to the prolactin receptor and induces transcription of genes encoding milk proteins. In contrast, when STAT5 becomes activated in erythroid progenitor cells following binding of Epo to EpoR, it induces expression of the protein $\mathrm{Bcl}-\mathrm{x}_{\mathrm{L}} . \mathrm{Bcl}-\mathrm{x}_{\mathrm{L}}$ prevents the programmed cell death, or apoptosis, of these progenitors (see Chapter 21), allowing them to proliferate and differentiate into red blood cells. More generally, it is thought that in each type of cell, activated STAT proteins, like the Smads discussed earlier, bind only to DNA sites in open chromatin and mainly to sites that have master transcription factors or other cell-specific gene regulatory proteins bound at adjacent sites. This combinatorial diversity allows a relatively limited set of receptors, JAK kinases, and STAT proteins to control a vast array of cellular activities.

## Multiple Mechanisms Down-Regulate Signaling from Cytokine Receptors

In the last chapter, we saw several ways in which signaling from G protein-coupled receptors is terminated. For instance, phosphorylation of receptors and downstream signaling proteins suppresses signaling, and this suppression can be reversed by the controlled action of phosphatases. Here we

![li2684542480e](li2684542480e.jpg)

FIGURE 16-12 Activation and structure of STAT proteins.
(a) Phosphorylation and dimerization of STAT proteins. Step $\mathbf{B}$ Following activation of a cytokine receptor (see Figure 16-10), the SH2 domain of an inactive monomeric STAT transcription factor binds to a phosphotyrosine in the receptor, bringing the STAT close to the active JAK associated with the receptor. The JAK then phosphorylates the C-terminal tyrosine in the STAT. Steps $\mathbf{B}$ and $\mathbf{B}$ Phosphorylated STATs spontaneously dissociate from the receptor and spontaneously dimerize. Because the STAT homodimer has two phosphotyrosine-SH2 domain interactions, whereas the receptor-STAT complex is stabilized by only one such interaction, phosphorylated STATs tend not to rebind to the receptor. Step $\mathbf{B}$ The STAT dimer moves into the nucleus, where it can bind to promoter sequences and activate transcription of target genes. (b) Ribbon diagram of the STAT1 dimer bound to DNA (black). The STAT1 dimer forms a C-shaped clamp around DNA that is stabilized by reciprocal and highly specific interactions between the SH2 domain (purple) of one monomer and the phosphorylated tyrosine residue (yellow with red oxygens) on the C-terminal segment of the other. The phosphotyrosine-binding site of the SH2 domain in each monomer is coupled structurally to the DNA-binding domain (magenta), suggesting a potential role for the SH2-phosphotyrosine interaction in the stabilization of DNA interacting elements. [Part (b) data from X. Chen et al., 1998, Cell 93:827, PDB ID 1bf5.]
(b)
![li2694542480e](li2694542480e.jpg)
discuss two mechanisms by which cytokine receptor signaling is regulated; other mechanisms for down-regulating signaling by protein tyrosine kinases are detailed in the next section because they have been investigated mainly with RTKs.

Phosphotyrosine Phosphatases Phosphotyrosine phosphatases are dephosphorylating enzymes that specifically hydrolyze phosphotyrosine linkages on specific target proteins. An excellent example of how phosphotyrosine phosphatase enzymes function to suppress the activity of protein tyrosine kinases is provided by SHP1, a phosphatase that negatively regulates signaling by several types of cytokine receptors. Its role was first identified by analysis of mice lacking this
protein, which died because of excess production of several types of blood cells, including erythrocytes.

SHP1 dampens cytokine signaling by binding to a cytokine receptor and inactivating the associated JAK protein, as depicted in Figure 16-13a. In addition to a phosphatase catalytic domain, SHP1 has two SH2 domains. When cells are in the resting state, unstimulated by a cytokine, one of the SH2 domains in SHP1 physically binds to and masks the catalytic site in the enzyme's phosphatase domain. In the stimulated state, however, this blocking SH2 domain binds to a specific phosphotyrosine residue in the activated receptor. The conformational change that accompanies this binding unmasks the SHP1 catalytic site and also brings it adjacent to the

(a) Short-term regulation: JAK2 deactivation by SHP1 phosphatase
![li2704542480e](li2704542480e.jpg)
(b) Long-term regulation: signal blocking and protein degradation by SOCS proteins
![li2714542480e](li2714542480e.jpg)
phosphotyrosine residue in the activation loop of the JAK associated with the receptor. By removing this phosphate, SHP1 inactivates the JAK, so that it can no longer phosphorylate the receptor or other substrates (such as STATs) unless additional cytokine molecules bind to cell-surface receptors, initiating a new round of signaling.

SOCS Proteins In a classic example of negative feedback, among the genes whose transcription is induced by STAT proteins are those encoding a class of small proteins termed suppressor of cytokine signaling (SOCS) proteins, which terminate signaling by cytokine receptors. All SOCS proteins contain an SH2 domain and another domain, called the SOCS box, that recruits components of E3 ubiquitin ligases (see Figure 3-31). The SOCS SH2 domain binds to specific phosphotyrosines on an activated receptor (Figure 16-13b); as a result, the receptor itself, as well as the associated JAK kinase, becomes polyubiquitinylated (a polymer of ubiquitins is covalently attached to the side chain of a lysine) and is then degraded in proteasomes

FIGURE 16-13 Two mechanisms for terminating cytokine signal transduction as exemplified by the erythropoietin receptor (EpoR). (a) Short-term regulation: SHP1, a phosphotyrosine phosphatase, is present in an inactive form in the cytosol of unstimulated cells. Binding of an SH2 domain in SHP1 to a particular phosphotyrosine in the activated receptor unmasks its phosphatase catalytic site and positions it near the phosphorylated tyrosine in the activation loop region of JAK2. Removal of the phosphate from this tyrosine inactivates the JAK kinase. See S. Constantinescu et al., 1999, Trends Endocrin. Met. 10:18. (b) Long-term regulation: SOCS proteins, whose expression is induced by the STAT5 protein in erythropoietin-stimulated erythroid progenitor cells, inhibit or permanently terminate signaling over longer periods. Binding of SOCS to specific phosphotyrosine residues on EpoR or JAK2 blocks binding of other signaling proteins (left). The SOCS box also targets the receptor as well as JAK2 for degradation by the ubiquitin-proteasome pathway (right). Similar mechanisms regulate signaling from other cytokine receptors. See B. T. Kile and W. S. Alexander, 2001, Cell. Mol. Life Sci. 58:1627.
(see Chapter 3), thereby permanently turning off all JAK2mediated signaling pathways until new receptors and JAK2 proteins can be made. The observation that proteasome inhibitors prolong JAK2 signal transduction supports this mechanism. One SOCS protein, SOCS-1, also binds to the critical phosphotyrosine in the activation loop of an activated JAK2 kinase, thereby inhibiting its catalytic activity.

Studies with cultured mammalian cells have shown that the receptor for growth hormone, which belongs to the cytokine receptor family, is down-regulated by another SOCS protein, SOCS-2. Strikingly, mice deficient in SOCS-2 grow significantly larger than their wild-type counterparts; these mice have long bones and proportionate enlargement of most organs. Thus SOCS proteins play an essential negative role in regulating intracellular signaling from the receptors for erythropoietin, growth hormone, and other cytokines.

## KEY CONCEPTS OF SECTION 16.2

## Cytokine Receptors and the JAK/STAT Signaling Pathway

- Two broad classes of receptors activate tyrosine kinases: (1) receptor tyrosine kinases (RTKs), in which the kinase is an intrinsic part of the receptor, and (2) cytokine receptors, in which the kinase is bound tightly to the cytosolic domain of the receptor. Signaling from receptor tyrosine kinases and from cytokine receptors activates similar downstream signaling pathways (see Figure 16-6).
- Cytokines play numerous roles in development. Erythropoietin, a cytokine secreted by kidney cells, promotes proliferation and differentiation of erythroid progenitor cells in the bone marrow (see Figure 16-7) to increase the number of mature red cells in the blood.
- Cytokines such as GH, prolactin, Epo, and G-CSF have very similar tertiary structures, as do their receptors. These and related cytokines form a 1 cytokine: 2 receptor complex on the cell surface (see Figures 16-8 and 16-9).

- The cytosolic domains of cytokine receptors are tightly bound to a JAK protein tyrosine kinase, which becomes activated after cytokine binding and receptor dimerization and phosphorylates tyrosine residues in the receptor (see Figure 16-10).
- In both RTKs and cytokine receptors, short amino acid sequences containing a phosphotyrosine residue are bound by signal-transducing proteins with conserved SH2 domains. The sequence of amino acids surrounding the phosphorylated tyrosine determines which SH2 domain will bind to it (see Figure 16-11). Such protein-protein interactions are important in many signaling pathways (see Figure 16-12).
- The JAK/STAT pathway operates downstream from all cytokine receptors and some RTKs. STAT monomers bound to phosphotyrosines on receptors are phosphorylated by receptor-associated JAKs, then dimerize and move to the nucleus, where they activate transcription (see Figure 16-12).
- Signaling from cytokine receptors is terminated by the phosphotyrosine phosphatase SHP1 and several SOCS proteins (see Figure 16-13).


### 16.3 Receptor Tyrosine Kinases

The signaling molecules that activate receptor tyrosine kinases are soluble or membrane-bound peptide or protein hormones, including many that were initially identified as growth factors for specific types of cells. Many of these RTK ligands, such as nerve growth factor (NGF), platelet-derived growth factor (PDGF), fibroblast growth factor (FGF),
and epidermal growth factor (EGF), stimulate proliferation and differentiation of specific cell types. Others, such as insulin, regulate expression of multiple genes that control sugar and lipid metabolism in liver, muscle, and adipose (fat) cells. Many RTKs and their ligands were identified in studies of human cancers associated with mutant forms of growth-factor receptors that stimulate proliferation even in the absence of the growth factor. The mutation "tricks" the receptor into behaving as though the ligand is present at all times, so the receptor is constantly in an active state (constitutively active; see Chapter 24). Other RTKs have been uncovered during analysis of developmental mutations that lead to blocks in differentiation of specific cell types in C. elegans, Drosophila, and the mouse.

## Binding of Ligand Promotes Dimerization of an RTK and Leads to Activation of Its Intrinsic Tyrosine Kinase

All RTKs have three essential components: an extracellular domain containing a ligand-binding site, a single hydrophobic transmembrane $\alpha$ helix, and a cytosolic segment that includes a domain with protein tyrosine kinase activity (Figure 16-14). Most RTKs are monomeric, and ligand binding to the extracellular domain induces formation of receptor dimers. As with cytokine receptors, the formation of functional dimers is a necessary step in activation of all RTKs.

RTK activation can be summarized as follows: In the resting, unstimulated (no ligand bound) state, the intrinsic kinase activity of an RTK is very low (Figure 16-14, step 1). Like most other kinases, including the JAK kinases discussed earlier, RTKs contain a flexible domain termed the activation
![li2724542480e](li2724542480e.jpg)

FIGURE 16-14 General structure and activation of receptor tyrosine kinases (RTKs). The cytosolic domain of RTKs contains an intrinsic protein tyrosine kinase catalytic site. In the absence of ligand (step 1), RTKs generally exist as monomers with poorly active kinases. Binding of two ligands to the extracellular domains of two RTKs forms or stabilizes an activated dimeric receptor. This brings together two poorly active kinases such that each one phosphorylates the other
on a tyrosine residue in the activation loop (step 1). Phosphorylation causes the loop to move out of the kinase catalytic site, thus increasing the ability of ATP and/or the protein substrate to bind. The activated kinase then phosphorylates several tyrosine residues in the receptor's cytosolic domain (step 1). The resulting phosphotyrosines function as docking sites for SH 2 and other binding domains on downstream signal-transducing proteins.

![li2734542480e](li2734542480e.jpg)

Membrane surface
(b) Top-down view
![li2744542480e](li2744542480e.jpg)

FIGURE 16-15 Structure of the extracellular domains of the active dimeric fibroblast growth factor (FGF) receptor, stabilized by binding of two ligands and by heparan sulfate. Shown here are side and top-down views of the complex comprising the extracellular domains of two FGF receptor (FGFR) monomers (purple and violet), two bound FGF molecules (red), and two short heparan sulfate chains, which bind tightly to FGF. (a) In the side view, the upper domain of one receptor monomer (purple) is seen situated behind that of the other (violet); the plane of the plasma membrane is at the bottom. A small segment of the extracellular domain whose structure is not known connects to the membrane-spanning $\alpha$-helical segment of each of the two receptor monomers (not shown) that protrude downward into the membrane. (b) In the top view, the heparan sulfate chains are seen threading between and making numerous contacts with the upper domains of both receptor monomers. These interactions promote binding of the ligand to the receptor and receptor dimerization. [Data from J. Schlessinger et al., 2000, Mol. Cell 6:743, PDB ID 1fq9.]
loop. In the resting state, the activation loop is nonphosphorylated and assumes a conformation that blocks kinase activity. In some receptors (e.g., the insulin receptor), it prevents binding of ATP. In others (e.g., the FGF receptor), it prevents
binding of substrate. Binding of ligand causes a conformational change that promotes dimerization of the extracellular domains of RTKs, which brings their transmembrane segments-and therefore their cytosolic domains-close together. In a manner similar to the activation of the JAK kinases, the kinase in each subunit then phosphorylates a particular tyrosine residue in the activation loop of the other subunit (Figure 16-14, step ). This phosphorylation leads to a conformational change in the activation loop that unblocks kinase activity by reducing the $K_{\mathrm{D}}$ for ATP or the substrate to be phosphorylated. The resulting enhanced kinase activity can then phosphorylate additional tyrosine residues in the cytosolic domain of the receptor (Figure 16-14, step ) , which bind SH2 or other domains of signaling proteins, as well as phosphorylating other target proteins, leading to intracellular signaling.

Although dimerization is a necessary step in the activation of all RTKs, functional dimers can be formed in multiple ways. Many receptors are dimerized in a manner similar to the fibroblast growth factor (FGF) receptor (Figure 16-15), in which each of two FGF molecules binds simultaneously to the extracellular domains of two receptor subunits, stabilizing the dimer. FGF also binds tightly to heparan sulfate, a negatively charged polysaccharide component of some cell-surface proteins and of the extracellular matrix (see Chapter 20); this association enhances ligand binding and formation of a dimeric receptor-ligand complex (see Figure 16-15). The participation of the heparan sulfate is essential for efficient receptor activation.

Yet other RTKs, such as the insulin receptor, form disulfide-linked dimers even in the absence of hormone; binding of ligand to this type of RTK alters its conformation in such a way that the receptor kinase becomes activated. This last example highlights the fact that simply having two receptor monomers in close contact is not sufficient for receptor activation-the proper conformational changes must accompany receptor dimerization to lead to tyrosine kinase activation. Once an RTK is locked into a functional dimeric state, its associated tyrosine kinase becomes activated.

## Homo- and Hetero-oligomers of Epidermal Growth Factor Receptors Bind Members of the Epidermal Growth Factor Family

Four RTKs participate in signaling by the many members of the epidermal growth factor (EGF) family of signaling molecules; these receptors are termed Erb-B1, 2, 3, and 4. In humans these receptors are also called HER (human epidermal growth factor receptor) 1, 2, 3, and 4, respectively. Epidermal growth factors and their receptors have been studied intensively because of their involvement in many human diseases; drugs targeting receptors that are overproduced in tumors or have activating mutations are used to treat many cancers, as we learn in Chapter 24.

In the resting state, the majority of EGF receptor molecules are monomeric. HER1 (Erb-B1) directly binds EGF as well as six other members of the EGF family:

![li2754542480e](li2754542480e.jpg)

FIGURE 16-16 Ligand-induced dimerization of HER1, a human receptor for epidermal growth factor (EGF). (a) The extracellular region of all EGF receptors contains four domains: domains I (blue) and III (yellow) are closely related in sequence, as are domains II (green) and IV (red). In the absence of bound EGF, the receptor is mostly monomeric and the intracellular kinase is inactive. The extracellular region adopts a configuration in which the $\beta$-hairpin from domain II that forms the "dimerization
herparin-binding EGF (HB-EGF), transforming growth factor $\alpha$ (TGF- $\alpha$ ), amphiregulin (AREG), epiregulin (EREG), epigen (EPGN), and betacellulin (BTC). Binding of any of these ligands leads to homodimerization of the HER1 extracellular domain (Figure 16-16). In the resting, monomeric, state, the $\beta$-hairpin in the segment of receptor domain II, termed the dimerization arm, forms a tight intramolecular interaction with domain IV. Binding of EGF induces a dramatic conformational change to the extracellular domain of HER1 in which the receptor "clamps down" on the hormone such that the dimerization arm becomes exposed. The dimerization arms from two activated receptors then bind tightly together, forming a stable receptor homodimer. In contrast to the FGF receptors (see Figure 16-15), the ligand does not directly stabilize the active homodimer.

HER3 and HER4 also bind members of the EGF family. Neuregulins 1 and 2 (NRG1 and NRG2) bind to both HER3 and HER4; HB-EGF, NRG3, and NRG4 bind only to HER4. HER4, like HER1, can form stable homodimers after binding a growth factor, but HER3 cannot. So how can HER3 signal? The answer involves HER2, which does not bind a ligand. Rather, HER2 exists on the plasma membrane in a conformation that is very similar to that of HER1 with a bound EGF; the HER2 dimerization arm protrudes outward and is able to bind to and form heterodimers with ligand-bound HER3 as well as with ligand-bound HER1 and HER4 (Figure 16-17). HER2 can signal only by forming heterocomplexes with ligand-bound HER1, HER3, or HER4; thus it facilitates signaling by all EGF family members (see Figure 16-17). Even though HER3 lacks a functional kinase domain, it can still participate in signaling; after binding a ligand, it heterodimerizes with HER2 and
arm" binds to domain IV of the same receptor molecule. (b) EGF binds simultaneously to domains I and III; binding induces a major conformational change in the extracellular domain such that the dimerization arm of domain II is now exposed. (c) Dimerization of two identical ligand-bound receptor monomers in the plane of the membrane occurs primarily through interactions between the dimerization arms of the two receptors. (Data from H. Ogiso et al., 2002, Cell 110:775, PDB ID 1ivo.)
becomes phosphorylated by the HER2 kinase, activating downstream signal transduction pathways.

Thus an increase in HER2 on the cell surface makes a cell more sensitive to signaling by many EGF family members because the rate at which the signaling heterodimers are formed after ligand binding will be enhanced. As we learn in Chapter 24, understanding the HERs has helped explain why a common form of breast cancer, in which the HER2 gene is amplified and HER2 is overexpressed, is so dangerous and has led to important drug therapies. Overexpression of HER2 makes the tumor cells sensitive to growth stimulation by low levels of any member of the EGF family of growth factors-levels that would not stimulate proliferation of cells with normal HER2 levels.

## Activation of the EGF Receptor Results in the Formation of an Asymmetric Active Kinase Dimer

As is the case with most receptor tyrosine kinases, the kinase domain in the FGF receptor is activated by phosphorylation of a tyrosine in the activation loop. In contrast, activation of the EGF receptor kinases does not involve loop phosphorylation; their mechanism of activation was uncovered only recently through structural studies of the receptor cytosolic domain in both the active and inactive states. The kinase domains are separated from the transmembrane segment by a so-called juxtamembrane segment, colored blue in Figure 16-18. In the inactive, monomeric state, the activation loop is localized to the active site of the kinase, blocking its activity; in this way the kinase is maintained in the "off" state (Figure 16-18, left). Receptor dimerization generates an asymmetric kinase dimer

![li2764542480e](li2764542480e.jpg)

FIGURE 16-17 The HER family of receptors and their ligands.
Humans and mice express four receptor tyrosine kinases, denoted HER1, 2, 3, and 4 in humans and Erb-B1, 2, 3, and 4 in mice and other animals. Only the extracellular domains of these receptors are depicted here. These receptors bind epidermal growth factor (EGF) and the other EGF family members: heparin-binding EGF (HB-EGF), transforming growth factor $\alpha$ (TGF- $\alpha$ ), amphiregulin (AREG), epiregulin (EREG), epigen (EPGN), betacellulin (BTC), and the four neuregulins. Note that Erb-B2 (HER2), which does not directly bind a ligand, exists in a conformation that is
(Figure 16-18, right) in which one kinase domain-termed the donor-binds the second kinase domain-the acceptor. This binding changes the conformation of the top lobe of the receiver, causing the activation loop to move out of the kinase active site and activating the kinase.

Thus evolution has produced many variations on the theme of this simple ligand-RTK mechanism: RTKs are activated by dimerization, but different receptors use different mechanisms to accomplish this. Similarly, kinases become activated by movement of the activation loop away from
very similar to that of the activated Erb-B1 with a bound EGF. Erb-B2 can form a heterodimer with ligand-activated Erb-B1, 3, or 4; thus Erb-B2 facilitates signaling by all EGF family members. Erb-B3 (HER3) has a very poorly active kinase domain and can signal only when complexed with Erb-B2. [Erb-B1 data from K. M. Ferguson et al., 2003, Mol. Cell 11:507, PDB ID 1nql. Erb-B2 data from H.-S. Cho et al., 2003, Nature 421:756, PDB ID 1n8z. Erb-B3 data from H. S. Cho and D. J. Leahy, 2002, Science 297:1330, PDB ID 1m6b. Erb-B4 data from S. Bouyain et al., 2005, Proc. Natl. Acad. Sci. USA 102:15024, PDB ID Iahx. EFG and Erb-B1 data from H. Ogiso et al., 2002, Cell 110:775, PDB ID 1ivo.]
the kinase catalytic site, but different receptors use different mechanisms to accomplish this task. Down-regulation of signaling from RTKs is also common, and different mechanisms have evolved to accomplish this task as well.

## Multiple Mechanisms Down-Regulate Signaling from RTKs

Earlier in this chapter we discussed two mechanisms by which signaling from cytokine receptors is down-modulated-those
![li2774542480e](li2774542480e.jpg)

FIGURE 16-18 Activation of the EGF receptor by EGF results in the formation of an asymmetric kinase domain dimer. In the inactive, monomeric state, the activation loop is localized to the kinase active site and thus inhibits kinase activation. Receptor dimerization generates an asymmetric kinase dimer such that the C-terminal C-lobe of the donor kinase binds to the N -terminal N -lobe of the acceptor kinase in the opposite receptor; the dimer is stabilized by interactions between the juxtamembrane segments of the two receptors. These interactions cause a conformational change that removes the activation loop from the kinase site of the acceptor kinase, activating its kinase activity. The active kinase then phosphorylates tyrosine residues in the C-terminal segments of the receptor cytosolic domain. [EGF receptor data from H. Ogiso et al., 2002, Cell 110:775, PDB ID 1ivo. Asymmetric kinase dimer data from K. M. Ferguson et al., 2003, Mol. Cell. 11:507, PDB ID 1nql.]

involving phosphotyrosine phosphatases and SOCS proteins (see Figure 16-13). These proteins are also used to depress signaling by some RTKs, but two other mechanisms-receptor-mediated endocytosis and degradation in lysosomes-are more common.

Receptor-Mediated Endocytosis Prolonged treatment of cells with ligand often reduces the number of available cellsurface receptors, so that the cells have a less robust response to exposure to a given concentration of ligand than they did before ligand addition. This desensitization response helps prevent inappropriately prolonged receptor activity. In the absence of EGF, for instance, cell-surface HER1 receptors are relatively long-lived, with an average half-life of 10-15 hours. Unbound receptors are internalized via clathrin-coated pits into endosomes at a relatively slow rate, on average once every 30 minutes, and are often returned rapidly to the plasma membrane so that there is little reduction in total surface receptor numbers. Following binding of an EGF ligand, the rate of endocytosis of HER1 is increased about tenfold, and only a fraction of the internalized receptors return to the plasma membrane; the rest are degraded in lysosomes. Each time a HER1-EGF complex is internalized via the process of receptor-mediated endocytosis (see Figure 14-29), the receptor has about a 20 to 80 percent chance of being degraded, depending on the cell type. Exposure of a fibroblast cell to high levels of EGF for several hours induces several rounds of endocytosis, resulting in degradation of most cell-surface EGF receptor molecules and thus a reduction in the cell's sensitivity to EGF. In this way, prolonged treatment with a given concentration of EGF desensitizes the cell to that level of hormone, though the cell may respond if the level of EGF is increased.

HER1 mutants that lack kinase activity do not undergo accelerated endocytosis in the presence of ligand. It is likely that ligand-induced activation of the kinase activity in normal HER1 induces a conformational change in the cytosolic tail, exposing a sorting motif that facilitates receptor recruitment into clathrin-coated pits and subsequent internalization of the receptor-ligand complex. Despite extensive study of mutant HER1 cytosolic domains, the identity of these "sorting motifs" is controversial, and most likely multiple motifs function to enhance endocytosis. Interestingly, internalized receptors can continue to signal from endosomes or other intracellular compartments before their degradation, as evidenced by their binding to signaling proteins such as GRB2 and Sos, which are discussed in the next section.

Lysosomal Degradation Several processes influence recycling of surface receptors to the plasma membrane versus lysosomal degradation; one is covalent modification by the small protein ubiquitin (see Chapter 3). There is a strong correlation between monoubiquitinylation (addition of a single ubiquitin to a given lysine of a protein) of the HER1 cytosolic domain and HER1 degradation. The monoubiquitinylation, which also occurs in other ligand-activated RTKs, is mediated by the enzyme c-Cbl. This enzyme, which is an E3 ubiquitin ligase (see Figure 3-31), contains an EGF
receptor-binding domain, which binds directly to phosphorylated EGF receptors, and a RING finger domain, which recruits ubiquitin-conjugating enzymes and mediates transfer of ubiquitin to the receptor. The ubiquitin functions as a "tag" on the receptor that stimulates its incorporation from endosomes into multivesicular bodies (see Figure 14-33) that are ultimately degraded in lysosomes. A role for c-Cbl in EGF receptor trafficking emerged from genetic studies in C. elegans, which established that c-Cbl negatively regulates the function of the nematode EGF receptor (Let-23), probably by inducing its degradation. Similarly, knockout mice lacking c-Cbl show hyperproliferation of mammary gland epithelia, consistent with a role of c-Cbl as a negative regulator of EGF signaling.

Experiments with mutant cell lines have demonstrated that internalization of RTKs plays an important role in regulating cellular responses to EGF and other growth factors. For instance, a mutation in the HER1 EGF receptor that prevents it from being incorporated into coated pits makes it resistant to ligand-induced endocytosis. As a result, this mutation leads to substantially elevated numbers of EGF receptors on cells and thus increased sensitivity of cells to EGF as a mitogenic signal. Such mutant cells are prone to EGF-induced transformation into tumor cells (see Chapter 24). Interestingly, the other EGF family receptors-HER2, HER3, and HER4-do not undergo ligand-induced internalization, an observation that emphasizes how each receptor evolved to be regulated in its own appropriate manner.

## KEY CONCEPTS OF SECTION 16.3

## Receptor Tyrosine Kinases

- Receptor tyrosine kinases, which bind to peptides and signaling proteins such as growth factors and insulin, may exist as preformed dimers or dimerize during binding to ligands. Ligand binding triggers formation of functional dimeric receptors, a necessary step in activation of the receptor-associated kinase, and different receptors accomplish this function in different ways (see Figures 16-14, 16-15, and 16-16).
- Activation of many RTKs leads to phosphorylation of the activation loop in the protein tyrosine kinase that is an intrinsic part of the cytoplasmic domain, enhancing its catalytic activity (see Figure 16-14). The activated kinase then phosphorylates tyrosine residues in the receptor cytosolic domain and in other protein substrates.
- Humans express many RTKs, four of which (HER1HER4) define the epidermal growth factor receptor family that mediates signaling from different members of the epidermal growth factor family of signaling molecules (see Figure 16-17). One of these receptors, HER2, does not bind ligand; it forms active heterodimers with ligand-bound monomers of the other three HER proteins. Overexpression of HER2 is implicated in many breast cancers.

- Endocytosis of receptor-hormone complexes and their degradation in lysosomes is a principal way of reducing the number of receptor tyrosine kinases and cytokine receptors on the cell surface and thus decreasing the sensitivity of cells to many peptide hormones.


### 16.4 The Ras/MAP Kinase Pathway

Almost all receptor tyrosine kinases and cytokine receptors activate the Ras/MAP kinase pathway (see Figure 16-6b). The monomeric Ras protein belongs to the GTPase superfamily of intracellular switch proteins (see Figure 15-5). Activated Ras promotes the formation of a signal transduction complex, containing three sequentially acting protein kinases, at the cytosolic surface of the plasma membrane. This kinase cascade culminates in activation of certain members of the MAP kinase family, which can translocate into the nucleus and phosphorylate many different proteins. Among the target proteins for MAP kinase are transcription factors that regulate the expression of proteins with important roles in the cell cycle and in cell differentiation. Importantly, different cell-surface receptors can activate different signaling pathways that result in activation of different members of the MAP kinase family.

Because an activating mutation in an RTK, Ras, or a protein in the MAP kinase cascade is found in almost all types of human tumors, the RTK/Ras/MAP kinase pathway has been subjected to extensive study, and a great deal is known about the components of this pathway (see Chapter 24). We begin our discussion by reviewing how Ras cycles between the active and inactive state. We then describe how Ras is activated and passes a signal to the MAP kinase pathway. Finally, we examine recent studies indicating that both yeasts and cells of higher eukaryotes contain multiple MAP kinase pathways, and we consider the ways in which cells keep different MAP kinase pathways separate from one another through the use of scaffold proteins.

## Ras, a GTPase Switch Protein, Operates Downstream of Most RTKs and Cytokine Receptors

Like the $\mathrm{G}_{\mathrm{q}}$ subunits in trimeric G proteins discussed in Chapter 15 , monomeric Ras proteins are "switch" proteins that alternate between an active "on" state with a bound GTP and an inactive "off" state with a bound GDP (see Figures 15-4 and 15-5 to review this concept). Unlike trimeric G proteins, Ras is not directly linked to cell-surface receptors. Like that of trimeric G proteins, Ras activity is regulated by several other proteins. Ras activation is accelerated by a guanine nucleotide exchange factor (GEF), which binds to the Ras-GDP complex, causing dissociation of the bound GDP (see Figure 15-4). As with other G proteins, GTP binds spontaneously to "empty" Ras molecules, forming the active Ras-GTP.

Ras ( $\sim 170$ amino acids) is smaller than $\mathrm{G}_{\mathrm{q}}$ proteins ( $\sim 300$ amino acids), but the GTP-binding domains of the
two protein types have a similar structure (see Figure 15-5 to review the structure of Ras). Hydrolysis of the bound GTP to GDP deactivates Ras. Structural and biochemical studies show that $\mathrm{G}_{\mathrm{q}}$ proteins contain a GTPase-activating protein (GAP) domain that increases the intrinsic rate of GTP hydrolysis. Because this domain is not present in Ras, that protein has an intrinsically slower rate of GTP hydrolysis. Thus the average lifetime of a GTP bound to Ras is about 1 minute, which is much longer than the average lifetime of a $\mathrm{G}_{\mathrm{q}}$-GTP complex. Because the intrinsic GTPase activity of Ras-GTP is low compared with that of $\mathrm{G}_{\mathrm{q}}$-GTP, Ras-GTP requires the assistance of another protein, a GTPase-activating protein (GAP), to deactivate it. Binding of a GAP to Ras-GTP accelerates the intrinsic GTPase activity of Ras by more than a hundredfold; the actual hydrolysis of GTP to GDP and $\mathrm{P}_{\mathrm{i}}$ is catalyzed by amino acids from both Ras and GAP. In particular, insertion of one of GAP's arginine side chains into the Ras active site stabilizes an intermediate in the hydrolysis reaction.

8Mammalian Ras proteins have been studied in great detail because mutant Ras proteins are associated with many types of human cancer. These mutant proteins, which bind but cannot hydrolyze GTP, are permanently in the "on" state and contribute to oncogenic transformation (see Chapter 24). Determination of the three-dimensional structure of the RasGAP complex and tests of mutant forms of Ras explained the puzzling observation that most oncogenic, constitutively active Ras proteins ( $\mathrm{Ras}^{\mathrm{D}}$ ) contain a mutation at position 12. Replacement of the normal glycine 12 with any other amino acid (except proline) blocks the functional binding of GAP proteins and in essence "locks" Ras in the active GTP-bound state.

The first indication that Ras functions downstream from RTKs in a common signaling pathway came from experiments in which cultured fibroblast cells were induced to proliferate by treatment with a mixture of two protein hormones that activate RTKs: PDGF and EGF. Microinjection of anti-Ras antibodies into these cells blocked cell proliferation. Conversely, injection of Ras ${ }^{\mathrm{D}}$, a constitutively active mutant Ras protein that hydrolyzes GTP very inefficiently and thus persists in the active state, caused the cells to proliferate in the absence of the growth factors. These findings are consistent with studies, using the pull-down assay method detailed in Figure 15-11, in which addition of FGF to fibroblasts led to a rapid increase in the proportion of Ras present in the GTP-bound active form. However, as we will see, an activated RTK (or cytokine receptor) cannot directly activate Ras. Instead, other proteins must first be recruited to the activated receptor, where they serve as adapters.

## Genetic Studies in Drosophila Identified Key Signal-Transducing Proteins in the Ras/MAP Kinase Pathway

Our knowledge of the proteins involved in the Ras/MAP kinase pathway came principally from genetic analyses of mutant fruit flies (Drosophila) and roundworms (C. elegans)

![li2784542480e](li2784542480e.jpg)

FIGURE 16-19 The compound eye of Drosophila melanogaster. (a) Scanning electron micrograph showing the individual ommatidia that compose the fruit fly eye. (b) Longitudinal and cutaway views of a single ommatidium. Each of these tubular structures contains eight photoreceptors, designated R1-R8, which are long, cylindrically shaped light-sensitive cells. R1-R6 (yellow) extend throughout the depth of the retina, whereas R7 (brown) is located toward the surface of the eye and R8 (blue) toward the back side, where the axons exit. (c) Comparison
of eyes from wild-type and sevenless mutant flies viewed by a special technique that can distinguish the photoreceptors in an ommatidium. The plane of sectioning is indicated by the blue arrows in (b), and the R8 cell is out of the plane of these images. The seven photoreceptors in this plane are easily seen in the wild-type ommatidia (top), whereas only six are visible in the mutant ommatidia (bottom). The eyes of flies with the sevenless mutation lack the R7 cell. [Part (a) Cheryl Power/Science Source; part (c) courtesy of Utpal Banerjee, UCLA.]
![li2794542480e](li2794542480e.jpg)
(a) Wild type
![li2804542480e](li2804542480e.jpg)
(b) Single mutant
![li2814542480e](li2814542480e.jpg)
(c) Double mutant
![li2824542480e](li2824542480e.jpg)
(d) Active
![li2834542480e](li2834542480e.jpg)
![li2844542480e](li2844542480e.jpg)

EXPERIMENTAL FIGURE 16-20 Genetic studies reveal that activation of Ras induces development of R7 photoreceptors in the Drosophila eye. (a) During larval development of wild-type flies, the R8 cell in each developing ommatidium expresses a cell-surface protein, called Boss, which binds to the Sev RTK on the surface of its neighboring R7 precursor cell. This interaction induces changes in gene expression that result in differentiation of the precursor cell into a functional R7 neuron. (b) In fly embryos with a mutation in the sevenless (sev) gene, R7 precursor cells cannot bind Boss and therefore do not differentiate normally into R7 cells. Rather, the precursor cell enters an alternative developmental pathway and eventually becomes a cone cell. (c) Doublemutant larvae (sev ${ }^{-/-} ;$Ras $^{D}$ ) express a constitutively active Ras (Ras ${ }^{\mathrm{D}}$ ) in the R7 precursor cell, which induces differentiation of R7 precursor cells in the absence of the Boss-mediated signal. This finding shows that activated Ras is sufficient to mediate induction of an R7 cell. See M. A. Simon et al., 1991, Cell 67:701, and M. E. Fortini et al., 1992, Nature 355:559.

The investigators reasoned that at this intermediate temperature, the signaling pathway would become defective (and thus no R7 cells would develop) if the level of another protein involved in the pathway was reduced, thereby reducing the activity of the overall pathway below the level required to form an R7 cell. A recessive mutation affecting such a protein would have this effect because, in diploid organisms such as Drosophila, a heterozygote containing one wild-type and one mutant allele of a gene will produce half the normal amount of the gene product; hence, even if such a recessive mutation is in an essential gene, the organism will usually be viable. However, a fly carrying a temperature-sensitive mutation in the sev gene and a second mutation affecting another protein in the signaling pathway would be expected to lack R7 cells at the intermediate temperature.

By use of this screen, researchers identified three genes encoding important proteins in the Sev pathway: an SH2containing protein exhibiting 64 percent amino acid sequence identity to human GRB2 (growth factor receptor-bound protein 2), a GEF called Sos (Son of Sevenless) exhibiting 45 percent identity with its mouse counterpart, and a Ras protein exhibiting 80 percent identity with its mammalian counterparts. These three proteins were later found to function in other signaling pathways initiated by ligand binding to other RTKs and to be used at different times and places in the developing fly.

In subsequent studies, researchers introduced a mutant ras ${ }^{D}$ gene into fly embryos carrying a mutation in the sevenless gene. As noted earlier, the ras ${ }^{D}$ gene encodes a constitutively active Ras protein that is present in the active GTP-bound form even in the absence of a hormone signal. Although no functional Sev RTK was expressed in these double mutants ( $s e v^{-/-} ; r a s^{D}$ ), R7 cells formed normally, indicating that the presence of an activated Ras protein is sufficient for induction of R7-cell development (Figure 16-20c). This finding, which is consistent with the results of experiments with cultured fibroblasts described earlier, supports the conclusion that activation of Ras is a principal step in intracellular signaling by most, if not all, RTKs and cytokine receptors.

## Receptor Tyrosine Kinases Are Linked to Ras by Adapter Proteins

In order for activated RTKs and cytokine receptors to activate Ras, two cytosolic proteins-GRB2 and Sos-must first be recruited to provide a link between the receptor and Ras (Figure 16-21). GRB2 is an adapter protein, meaning that

FIGURE 16-21 Activation of Ras following ligand binding to receptor tyrosine kinases (RTKs) or cytokine receptors. The receptors for fibroblast growth factor (FGF) and many other growth factors are RTKs. The SH2 domain of the cytosolic adapter protein GRB2 binds to a specific phosphotyrosine on an activated, ligand-bound receptor, and the SH3 domains bind to the cytosolic Sos protein, bringing it near the cytosolic surface of the plasma membrane and close to its substrate, the inactive Ras-GDP. The guanine nucleotide exchange factor (GEF) activity of Sos then promotes formation of active Ras-GTP. Note that Ras is tethered to the cytosolic surface of the plasma membrane by a hydrophobic farnesyl anchor (see Figure 7-19). See J. Schlessinger, 2000, Cell 103:211, and M. A. Simon, 2000, Cell 103:13.
![li2854542480e](li2854542480e.jpg)

it has no enzyme activity and serves as a link, or scaffold, between two other proteins-in this case, between the activated receptor and Sos. Sos is a guanine nucleotide exchange protein (GEF) that catalyzes conversion of inactive GDPbound Ras to the active GTP-bound form.

GRB2 is able to serve as an adapter protein because of its SH2 domain, which binds to a specific phosphotyrosine residue in the cytosolic domain of an activated RTK (or cytokine receptor). In addition to its SH2 domain, the GRB2 adapter protein contains two SH3 domains, which bind to Sos (see Figure 16-21). Like phosphotyrosinebinding SH2 domains, SH3 domains are present in a large number of proteins involved in intracellular signaling. Although the three-dimensional structures of various SH3 domains are similar, their specific amino acid sequences differ. The SH3 domains in GRB2 selectively bind to proline-rich sequences in Sos; different SH3 domains in other proteins bind to proline-rich sequences distinct from those in Sos.

Proline residues play two roles in the interaction between an SH3 domain in an adapter protein (e.g., GRB2) and a proline-rich sequence in another protein (e.g., Sos). First, the proline-rich sequence assumes an extended conformation that permits extensive contacts with the SH3 domain, thereby facilitating interaction. Second, a subset of the prolines fit into binding "pockets" on the surface of the SH3 domain (Figure 16-22). Several nonproline residues also interact with the SH3 domain and are responsible for determining the binding specificity. Hence the binding of proteins to SH3 and to SH2 domains follows a similar strategy: certain residues provide the key structural motif necessary for binding, and neighboring residues confer specificity to the binding.

## Binding of Sos to Inactive Ras Causes a Conformational Change That Triggers an Exchange of GTP for GDP

Following activation of an RTK (e.g., the FGF receptor), a complex containing the activated receptor, GRB2, and Sos is formed on the cytosolic face of the plasma membrane (see Figure 16-21). Complex formation depends on the ability of GRB2 to bind simultaneously to the receptor and to Sos. Thus receptor activation leads to relocalization of Sos from the cytosol to the membrane, which brings Sos close to its substrate, Ras-GDP, which is already bound to the plasma membrane by means of a covalently attached lipid. Binding of Sos to Ras-GDP leads to conformational changes in the switch I and switch II segments of Ras, thereby opening the binding pocket for GDP so it can diffuse out (Figure 16-23). In other words, Sos functions as a GEF for Ras. Binding of GTP to Ras, in turn, induces a specific conformation of switch I and switch II that allows Ras-GTP to activate the next protein in the Ras/MAP kinase pathway.
![li2864542480e](li2864542480e.jpg)

FIGURE 16-22 Surface model of an SH3 domain bound to a target peptide. The short, proline-rich target peptide is shown as a space-filling model. In this target peptide, two prolines (Pro4 and Pro7, dark blue) fit into binding pockets on the surface of the SH3 domain. Interactions involving an arginine (Arg1, red), two other prolines (gray), and other residues in the target peptide (green) determine the specificity of binding. [Data from S. Feng et al., 1995, Proc. Natl. Acad. Sci. USA 92:12408, PDB ID 1qwf.]

## Signals Pass from Activated Ras to a Cascade of Protein Kinases Ending with MAP Kinase

Biochemical and genetic studies in yeast, C. elegans, Drosophila, and mammals have revealed that downstream of Ras is a highly conserved cascade of three protein kinases, culminating in MAP kinase. Although activation of the kinase cascade does not yield the same biological results in all cells, a common set of sequentially acting kinases defines the Ras/MAP kinase pathway, as outlined in Figure 16-24: Ras $\rightarrow$ Raf $\rightarrow$ MEK $\rightarrow$ MAP kinase.

Ras is activated by the exchange of GDP for GTP (step $\mathbf{\square}$. Active Ras-GTP binds to the N-terminal regulatory domain of Raf, a serine/threonine (not tyrosine) kinase, thereby activating its kinase activity (step $\mathbf{\square}$. In unstimulated cells, Raf is phosphorylated and bound in an inactive state by the protein 14-3-3, which binds phosphoserine residues in a number of important signaling proteins. Hydrolysis of Ras $\cdot$ GTP to Ras $\cdot$ GDP releases active Raf from its complex with 14-3-3 (step $\mathbf{\square}$, and the now active Raf subsequently phosphorylates and thereby activates MEK (step $\mathbf{\square}$. MEK, a dual-specificity protein kinase, phosphorylates its target proteins on both tyrosine and serine/threonine residues. Active MEK mainly phosphorylates and activates MAP kinase, another serine/ threonine kinase also known as ERK (step $\mathbf{\square}$. In different cells, MAP kinase phosphorylates many different proteins, including nuclear transcription factors that mediate cellular responses (step $\mathbf{\square}$.

![li2874542480e](li2874542480e.jpg)

FIGURE 16-23 Structures of Ras bound to GDP, Sos protein, and GTP. (a) As with other G proteins bound to GDP, in Ras-GDP, the switch I (green) and switch II (blue) segments do not directly interact with GDP. (b) One $\alpha$ helix (yellow) in Sos binds to both switch segments of Ras-GDP, leading to a massive conformational change in Ras. In effect, Sos pries Ras open by displacing the switch I region, thereby allowing GDP to diffuse out. (c) GTP is thought to bind to the Ras-Sos complex first through its base (guanine); subsequent binding of the GTP phosphates completes the interaction. The resulting conformational change
in the switch I and switch II segments of Ras, allowing both to bind to the GTP $\gamma$ phosphate, displaces Sos and promotes interaction of Ras-GTP with its effectors (discussed later). Colored purple is the P loop, a sequence motif found in many ATP- and GTP-binding proteins, which binds the $\beta$ phosphate of the nucleotide. See Figure 15-5 for another depiction of Ras-GDP and Ras-GTP. (Part (a) data from M. V. Milburn et al., 1990, Science 247:939, PDB ID 4q21. Part (b) data from J. Sejbal et al., 1996, J. Med. Chem. 39:1281, PDB ID 1bdk. Part (c) data from M. E. Pacold et al., 2000, Cell 103:931, PDB ID The8.)
![li2884542480e](li2884542480e.jpg)

FIGURE 16-24 Ras/MAP kinase pathway. In unstimulated cells, most Ras is tethered to the cytosolic surface of the plasma membrane in the inactive form with bound GDP. Binding of a ligand to its RTK or cytokine receptor leads to formation of the active Ras-GTP complex (step $\mathbf{\square}$ see also Figure 16-21). Activated Ras triggers the downstream kinase cascade depicted in steps $\mathbf{\square}$- $\mathbf{\square}$. culminating in activation of MAP kinase (MAPK). In unstimulated cells, binding of a dimer of the 14-3-3 protein to Raf stabilizes it in an inactive conformation. Each 14-3-3 monomer binds to a phosphoserine residue in Raf, one to phosphoserine-259 in the N -terminal domain and the other to phosphoserine-621 in the kinase domain; binding to 14-3-3 maintains the kinase in a closed, inactive state. Interaction of the Raf N-terminal regulatory domain with Ras-GTP results in dephosphorylation of one of the serines that bind Raf to 14-3-3, phosphorylation of other residues, loss of 14-3-3 binding, and activation of Raf kinase activity. After inactive Ras-GDP dissociates from Raf, it can presumably be reactivated by signals from activated receptors, thereby recruiting additional Raf molecules to the membrane. See E. Kerkhoff and U. Rapp, 2001, Adv. Enzyme Regul. 41:261; J. Avruch et al., 2001, Recent Prog. Horm. Res. 56:127; and D. Matallanas et al., 2011, Genes Cancer 2:232.

Active MAP kinase translocates to nucleus; activates many transcription factors

Several types of experiments have demonstrated that Raf, MEK, and MAP kinase lie downstream from Ras and have revealed the sequential order of these proteins in the pathway. For example, cultured mammalian cells that express a mutant, nonfunctional Raf protein cannot be stimulated to proliferate uncontrollably by a constitutively active $\mathrm{Ras}^{\mathrm{D}}$ protein. This finding established a link between the Raf and Ras proteins and showed that Raf lies downstream of Ras in the signaling pathway. In vitro binding studies further showed that the purified Ras-GTP protein binds directly to the N-terminal regulatory domain of Raf and activates its catalytic activity.

That MAP kinase is activated in response to Ras activation was demonstrated in quiescent cultured cells expressing a constitutively active Ras ${ }^{\mathrm{D}}$ protein. In these cells, activated MAP kinase is generated in the absence of stimulation by growth-promoting hormones. More importantly, R7 photoreceptors develop normally in the developing eyes of Drosophila mutants that lack a functional Ras or Raf protein but express a constitutively active MAP kinase. This finding indicates that activation of MAP kinase is sufficient to transmit a proliferation or differentiation signal normally initiated by ligand binding to a receptor tyrosine kinase such as Sevenless (see Figure 16-20). Biochemical studies showed, however, that Raf cannot directly phosphorylate MAP kinase or otherwise activate its activity.

The final link in the kinase cascade activated by Ras-GTP emerged from studies in which scientists fractionated extracts of cultured cells to search for a kinase activity that could phosphorylate MAP kinase and that was present only in cells stimulated with growth factors, not in unstimulated cells. This work led to the identification of MEK, a kinase that specifically phosphorylates one threonine and one tyrosine residue on the activation loop of MAP kinase, thereby activating its catalytic activity. (The acronym MEK comes from MAP and ERK kinase.) Later studies showed that MEK binds to the C-terminal catalytic domain of Raf and is phosphorylated by the Raf serine/threonine kinase; this phosphorylation activates the catalytic activity of MEK.

Hence activation of Ras induces a kinase cascade that includes Raf, MEK, and MAP kinase: activated RTK $\rightarrow$ Ras $\rightarrow$ Raf $\rightarrow$ MEK $\rightarrow$ MAP kinase. Although we will not emphasize them here, the complexity of this pathway is increased by the multiple isoforms of each of its components. In humans, there are three RAS, three Raf, two MEK, and two ERK proteins, and each of these has overlapping but also nonredundant functions.

$\mathbf{R}$Activating mutations in the Raf gene occur in about 50 percent of melanomas, skin cancers that are often caused by exposure to the ultraviolet radiation in sunlight. Among these melanomas, one particular mutation, a glutamic acid substitution for the valine at position 600, occurs in over 90 percent. This mutant Raf stimulates MEK-ERK signaling in cells in the absence of growth factors, and mutant Raf transgenes induce melanoma in mice. Very potent and selective inhibitors of Raf have recently entered the
clinic and are producing excellent responses in patients with melanomas caused by this mutant Raf protein.

## Phosphorylation of MAP Kinase Results in a Conformational Change That Enhances Its Catalytic Activity and Promotes Its Dimerization

Biochemical and x-ray crystallographic studies have provided a detailed picture of how phosphorylation activates MAP kinase. As in JAK kinases and receptor tyrosine kinases, the catalytic site in the inactive, nonphosphorylated form of MAP kinase is blocked by an activation loop (Figure 16-25a). Binding of MEK to MAP kinase destabilizes the loop structure, resulting in exposure of tyrosine 185, which is buried in the inactive conformation. Following phosphorylation of this critical tyrosine, MEK phosphorylates the neighboring threonine 183 (Figure 16-25b). Both the phosphorylated tyrosine and the phosphorylated threonine residue in MAP kinase interact with additional amino acids, thereby conferring an altered conformation on the loop region, which in turn permits binding of ATP to the catalytic site, which, as in all kinases, is in the groove between the upper and lower kinase domains. The phosphotyrosine residue ( pY 185 ) also plays a key role in binding specific substrate proteins to the surface of MAP kinase. Phosphorylation promotes not only
![li2894542480e](li2894542480e.jpg)

FIGURE 16-25 Structures of inactive, nonphosphorylated MAP kinase and the active, phosphorylated form. (a) In inactive MAP kinase, the activation loop is in a conformation that blocks the kinase active site. (b) Phosphorylation by MEK at tyrosine 185 (Y-185) and threonine 183 (T-183) leads to a marked conformational change in the activation loop. This activating change promotes both binding of its substrates-ATP and its target proteins-to MAP kinase and its dimerization. A similar phosphorylation-dependent mechanism activates JAK kinases and the intrinsic kinase activity of RTKs. [Data from B. J. Canagarajah et al., 1997, Cell 90:859, PDB ID 2erk.]

the catalytic activity of MAP kinase, but also its dimerization. The dimeric form of MAP kinase is translocated to the nucleus, where it regulates the activity of many nuclear transcription factors.

## MAP Kinase Regulates the Activity of Many Transcription Factors Controlling Early Response Genes

Addition of a growth factor (e.g., EGF or PDGF) to quiescent (non-growing) cultured mammalian cells causes a rapid increase in the expression of as many as a hundred different genes. These genes are called early response genes because they are induced well before cells enter the $S$ phase and replicate their DNA (see Chapter 19). One important early response gene encodes the transcription factor c-Fos. Together with other transcription factors, such as c-Jun, c-Fos induces the expression of many genes that encode proteins necessary for cells to progress through the cell cycle. Most RTKs that bind growth factors use the MAP kinase pathway to activate genes encoding proteins such as c-Fos, which in turn propel the cell through the cell cycle.

The enhancer that regulates the c-fos gene contains a serum response element (SRE), so named because it is activated by many growth factors in serum. This complex enhancer contains DNA sequences that bind multiple transcription factors. As depicted in Figure 16-26, activated (phosphorylated) dimeric MAP kinase induces transcription of the c-fos gene by directly activating one transcription factor, ternary complex factor (TCF), and indirectly activating another, serum response factor (SRF). In the cytosol, MAP kinase phosphorylates and activates a kinase called $\mathrm{p} 90^{\mathrm{RSK}}$, which translocates to the nucleus, where it phosphorylates a specific serine in SRF. After translocating to the nucleus, MAP kinase directly phosphorylates specific serines in TCF. Association of phosphorylated TCF with two molecules of phosphorylated SRF forms an active trimeric factor that activates c-fos gene transcription.

Unlike the Raf and MEK kinases, which phosphorylate only a few target kinases in the MAP kinase cascade, MAP kinases are known to phosphorylate more than 175 proteins in the nucleus and cytosol. Many MAP kinase targets are regulators of gene expression, and different MAP kinase targets are expressed in different mammalian cell types. As with Stats, Smads, and other transcription factors activated
![li2904542480e](li2904542480e.jpg)

FIGURE 16-26 Induction of gene transcription by MAP kinase. Steps 1 In the cytosol, MAP kinase phosphorylates and activates the kinase $\mathrm{p} 90^{\mathrm{RSK}}$, which then moves into the nucleus and phosphorylates the SRF transcription factor. Steps $\boldsymbol{B}$ and $\boldsymbol{B}$ After translocating into the nucleus, MAP kinase directly phosphorylates the transcription
factor TCF, which is already bound to the promoter of the c-fos gene. Step $\boldsymbol{B}$ Phosphorylated TCF and SRF act together to stimulate transcription of c-fos and other genes that contain an SRE sequence in their promoter. See the text for details. See R. Marais et al., 1993, Cell 73:381, and V. M. Rivera et al., 1993, Mol. Cell Biol. 13:6260.

directly or indirectly by cell-surface receptors, the precise proteins induced by MAP kinase depend on the particular target proteins expressed in the cell, on epigenetic markers on DNA and chromatin proteins, and on the presence of other transcription factors (see Figure 16-2).

## G Protein-Coupled Receptors Transmit Signals to MAP Kinase in Yeast Mating Pathways

Although MAP kinase is often activated in multicellular animals by RTKs or cytokine receptors, signaling from other receptors can activate MAP kinase in other eukaryotic cells (see Figure 15-32). To illustrate, we consider the mating pathway in the yeast $S$. cerevisiae, a well-studied example of a MAP kinase cascade linked to G protein-coupled receptors (GPCRs), in this case for two secreted peptide pheromones, the $\mathbf{a}$ and $\alpha$ factors.

Haploid yeast cells are either of the $\mathbf{a}$ or $\alpha$ mating type. They secrete protein signals known as pheromones, which induce mating between haploid yeast cells of the opposite mating types. An a haploid cell secretes the a mating factor and has GPCRs for the $\alpha$ factor on its cell surface; an $\alpha$ cell secretes the $\alpha$ factor and has GPCRs for the a factor (see Figure 1-23). Thus each type of cell recognizes the mating factor produced by the opposite type. Activation of the MAP kinase pathway by either the $\mathbf{a}$ or the $\alpha$ GPCRs induces transcription of genes that inhibit progression of the cell cycle and others that enable cells of opposite mating type to fuse together and ultimately form a diploid cell.

Ligand binding to either of the two yeast pheromone GPCRs triggers the exchange of GTP for GDP on the $\mathrm{G}_{\alpha}$ subunit and dissociation of $\mathrm{G}_{\alpha}$-GTP from the $\mathrm{G}_{\beta \gamma}$ complex. This activation process is identical to that for the GPCRs discussed in the previous chapter (see Figure 15-14). In many mammalian GPCR-initiated pathways, the active $\mathrm{G}_{\alpha}$ transduces the signal. In contrast, the dissociated $\mathrm{G}_{\beta \gamma}$ complex mediates all the physiological responses induced by activation of the yeast pheromone GPCRs (Figure 16-27a). As evidence, in yeast cells that lack $\mathrm{G}_{\alpha}$, the $\mathrm{G}_{\beta \gamma}$ subunit is always free. Such cells can mate in the absence of mating factors; that is, the mating response is constitutively on. However, in cells defective for the $\mathrm{G}_{\beta}$ or $\mathrm{G}_{\gamma}$ subunit, the mating pathway cannot be induced at all. If dissociated $\mathrm{G}_{\alpha}$ were the transducer, the pathway would be expected to be constitutively active in these mutant cells.

In yeast mating pathways, $\mathrm{G}_{\beta \gamma}$ functions by triggering a kinase cascade that is analogous to the one downstream from Ras; each protein in the cascade has a yeast-specific name, but shares sequences with, and is analogous in structure and function to, the corresponding mammalian protein shown in Figure 16-24. The components of this cascade were uncovered mainly through analyses of mutants that possess functional a and $\alpha$ receptors and G proteins, but are sterile (Ste) or defective in mating responses. The physical interactions between the components of the cascade were assessed through immunoprecipitation experiments with extracts of yeast cells and other types of studies. Based on these studies, scientists have proposed the kinase cascade shown in Figure 16-27a. Free $\mathrm{G}_{\beta \gamma}$, which is tethered to the membrane via the lipid bound to the $\gamma$ subunit, binds the Ste 5 protein, thus recruiting it and its bound kinases to the plasma membrane. Ste5, which has no obvious catalytic function, acts as a scaffold for assembling other components in the cascade (Ste11, Ste7, and Fus3). $\mathrm{G}_{\beta \gamma}$ also activates cdc24, a GEF for the Ras-like protein Cdc42; GTP-Cdc42 in turn activates the Ste20 protein kinase. Ste20 phosphorylates and activates Ste11, a serine/threonine kinase analogous to Raf and other mammalian MEK kinase (MEKK) proteins. Activated Ste11 then phosphorylates Ste7, a dual-specificity MEK, which then phosphorylates and activates Fus3, a serine/threonine kinase equivalent to MAP kinase. After translocation to the nucleus, Fus3 phosphorylates two proteins, Dig1 and Dig2, relieving their inhibition of the Ste12 transcription factor. Activated Ste12, in turn, induces expression of proteins involved in mating-specific cellular responses. Fus3 also affects gene expression by phosphorylating other proteins.

## Scaffold Proteins Separate Multiple MAP Kinase Pathways in Eukaryotic Cells

As we have just seen, both yeasts and higher eukaryotic cells contain a Ras/MAP kinase signaling pathway that is activated by extracellular protein signals and culminates in the MAP kinase-mediated phosphorylation of transcription factors and other signaling proteins, which together trigger specific changes in cell behavior. Importantly, all eukaryotes possess multiple highly conserved MAP kinase pathways that are activated by different extracellular signals and that activate different MAP kinase proteins that phosphorylate different transcription factors; these transcription factors, in turn, trigger different changes in cell division, differentiation, or function. Mammalian MAP kinases include Jun N-terminal kinases (JNKs) and p38 kinases, which are activated by signaling pathways in response to various types of stresses and which phosphorylate various transcription factors and other types of signaling proteins that affect cell division.

Current genetic and biochemical studies in the mouse and Drosophila are aimed at determining which MAP kinases mediate which responses to which signals in higher eukaryotes. This goal has already been accomplished in large part for the simpler organism $S$. cerevisiae. Each of the six MAP kinases encoded in the $S$. cerevisiae genome has been assigned by genetic analyses to specific signaling pathways triggered by various extracellular signals, such as pheromones, high osmolarity, starvation, hypotonic shock, and carbon or nitrogen deprivation. As an example, one yeast MAP kinase cascade, known as the osmoregulatory pathway, is shown in Figure 16-27b; this pathway results in activation of the MAP kinase Hog1, which in turn phosphorylates and activates proteins that induce expression of genes essential for yeast to survive in a medium of high osmotic strength.

![li2914542480e](li2914542480e.jpg)

FIGURE 16-27 Scaffold proteins separate yeast MAP kinase cascades in the mating and osmoregulatory pathways. In yeast, different receptors activate different MAP kinase pathways, two of which are outlined here. The two MEKs depicted, like all MEKs, are dual-specificity threonine/tyrosine kinases; all of the other kinases are serine/threonine kinases. (a) Mating pathway: The receptors for yeast $\alpha$ and a mating factors are coupled to the same trimeric G protein. Following ligand binding and dissociation of the G protein subunits, the membrane-tethered $\mathrm{G}_{\mathrm{II}}$, subunit binds the Ste5 scaffold to the plasma membrane. $\mathrm{G}_{\mathrm{II}}$, also activates Cdc24, a GEF for the Ras-like protein Cdc42; the active, GTP-bound Cdc42, in turn, binds to and activates the Ste20 kinase. Ste20 then phosphorylates and activates Ste11, which is analogous to Raf and other mammalian MEK kinase (MEKK) proteins. Ste20 thus serves as a MAPKKK kinase. Ste11 initiates a kinase cascade in which the final component, Fus3, is functionally equivalent to MAP kinase (MAPK) in higher eukaryotes. Like other MAP kinases, activated Fus3 then translocates into the nucleus. There it phosphorylates two proteins, Dig1 and Dig2, relieving their inhibition of the Ste12

A complication arises because in both yeasts and higher eukaryotic cells, different MAP kinase cascades share some common components. For instance, the MEKK Ste11 functions in three yeast signaling pathways: the
transcription factor, allowing it to bind to DNA and initiate transcription of genes that inhibit progression of the cell cycle and others that enable cells of opposite mating type to fuse together and ultimately form a diploid cell. (b) Osmoregulatory pathway: Two plasma membrane proteins, Sho1 and Msb1, are activated in an unknown manner by exposure of yeast cells to media of high osmotic strength. Activated Sho1 recruits the Pbs2 scaffold protein, which contains a MEK domain, to the plasma membrane. At the plasma membrane, the Sho1-Msb1 complex activates Cdc42, which in turn activates the resident Ste20 kinase, as in the mating pathway. Ste20, in turn, phosphorylates and activates Ste11, initiating a kinase cascade that activates Hog1, a MAP kinase. In the cytosol, Hog1 phosphorylates specific protein targets, including ion channels; after translocating to the nucleus, Hog1 phosphorylates several transcription factors and chromatin-modifying enzymes. Hog1 also appears also to promote transcriptional elongation. Together, the newly synthesized and modified proteins support survival in high-osmoticstrength media. See N. Dard and M. Peter, 2006, BioEssays 28:146, and R. Chen and J. Thorner, 2007, Biochim. Biophys. Acta 1773:1311.
mating pathway, the osmoregulatory pathway, and the filamentous growth pathway, which is induced by starvation. Nevertheless, each pathway activates a distinct MAP kinase. Similarly, in mammalian cells, common upstream

signal-transducing proteins participate in activating multiple JNKs.

Once the sharing of components among different MAP kinase pathways was recognized, researchers wondered how the specificity of the cellular responses to particular signals is achieved. Studies with yeast provided the initial evidence that pathway-specific scaffold proteins enable the signaltransducing kinases in a particular pathway to interact with one another, but not with kinases in other pathways. For example, the scaffold protein Ste5 stabilizes a large complex that includes the kinases in the mating pathway; similarly, the Pbs 2 scaffold is used for the kinase cascade in the osmoregulatory pathway (see Figure 16-27). Ste11 participates in both pathways, but in each one, it is constrained within a large complex that forms in response to a specific extracellular signal, so that signaling downstream from Ste11 is restricted to the complex in which it is localized. As a result, exposure of yeast cells to mating factors induces activation of a single MAP kinase, Fus3, whereas exposure to a high osmolarity induces activation of a different MAP kinase, Hog1.

Scaffolds for MAP kinase pathways are well documented in yeast, fly, and worm cells, but their presence in mammalian cells has been difficult to demonstrate. Perhaps the bestdocumented scaffold protein in metazoans is Ksr (kinase suppressor of Ras). Ksr functions as a molecular scaffold by binding several signaling components of the MAP kinase cascade, including both MEK and MAP kinase; and thus can enhance MAP kinase activation by regulating the efficiency of these interactions. In Drosophila, loss of the Ksr homolog blocks signaling by a constitutively active Ras protein, suggesting a positive role for Ksr in the Ras/MAP kinase pathway in fly cells. In nematodes, Ksr is required for Ras-mediated signaling during several developmental pathways. The signal specificity of different MAP kinases in mammalian cells may arise from their association with Ksr proteins or other scaffold-like proteins, but much additional research is needed to test this possibility.

## KEY CONCEPTS OF SECTION 16.4

## The Ras/MAP Kinase Pathway

- Ras is an intracellular GTPase switch protein that acts downstream from most RTKs and cytokine receptors. Like $\mathrm{G}_{\mathrm{s}}$, Ras cycles between an inactive GDP-bound form and an active GTP-bound form. Ras cycling requires the assistance of two proteins: a guanine nucleotide exchange factor (GEF) and a GTPase-activating protein (GAP).
- RTKs are linked indirectly to Ras via two proteins: GRB2, an adapter protein, and Sos, which has GEF activity (see Figure 16-21).
- The SH2 domain in GRB2 binds to a phosphotyrosine in activated RTKs, while its two SH3 domains bind Sos, thereby bringing Sos close to membrane-bound Ras-GDP and activating its GEF activity.
- Binding of Sos to inactive Ras causes a large conformational change that permits release of GDP and binding of GTP, forming active Ras (see Figure 16-23).
- Activated Ras triggers a kinase cascade in which Raf, MEK, and MAP kinase are sequentially phosphorylated and thus activated. Activated MAP kinase then translocates to the nucleus (see Figure 16-24).
- Activation of MAP kinase following stimulation of a growth-factor receptor leads to phosphorylation and activation of two transcription factors, which associate into a trimeric complex that promotes transcription of various early response genes (see Figure 16-26).
- Different extracellular signals induce activation of different MAP kinase pathways, which regulate diverse cellular processes by phosphorylating different sets of transcription factors.
- The kinase components of each MAP kinase cascade assemble into a large pathway-specific complex stabilized by a scaffold protein (see Figure 16-27). This ensures that activation of one MAP kinase pathway by a particular extracellular signal does not lead to activation of other pathways containing shared components.


### 16.5 Phosphoinositide Signaling Pathways

In previous sections, we have seen how the transduction of signals from receptor tyrosine kinases (RTKs) and cytokine receptors begins with the formation of multiprotein complexes associated with the plasma membrane (see Figures 16-10 and 16-14) and how these complexes initiate the Ras/MAP kinase pathway. Here we discuss how these same receptors initiate signaling pathways that involve as intermediates special phosphorylated phospholipids derived from phosphatidylinositol. As discussed in Chapter 15, these membrane-bound lipids are collectively referred to as phosphoinositides. These phosphoinositide signaling pathways include several enzymes that synthesize different phosphoinositides on the cytosolic face of the plasma membrane as well as cytosolic proteins with domains that can bind to these molecules and that are thus recruited to the cytosolic surface of the plasma membrane. In addition to the short-term effects on cell metabolism we encountered in Chapter 15, these phosphoinositide pathways have long-term effects on patterns of gene expression. We will see that phosphoinositide pathways end with a variety of kinases, including protein kinase C (PKC) and protein kinase B (PKB), that play key roles in cell growth and metabolism. As an example, we will see later in the chapter how insulin activation of PKB plays a key role in stimulating glucose import into muscle.

## Phospholipase $\mathrm{C}_{\gamma}$ Is Activated by Some RTKs and Cytokine Receptors

As discussed in Chapter 15, hormonal stimulation of some G protein-coupled receptors leads to the activation of phospholipase C (PLC). This membrane-associated enzyme then cleaves phosphatidylinositol 4,5-bisphosphate [ $\left.\mathrm{PI}(4,5) \mathrm{P}_{2}\right]$ to generate two important second messengers: 1,2-diacylglycerol (DAG) and inositol 1,4,5-trisphosphate $\left(\mathrm{IP}_{3}\right)$. Signaling via the $\mathrm{IP}_{3} /$ DAG pathway leads to an increase in cytosolic $\mathrm{Ca}^{2+}$ and to activation of protein kinase C (see Figure 15-34).

Although we did not mention it during our discussion of phospholipase C in Chapter 15, it is specifically the $\beta$ isoform of this enzyme ( $\mathrm{PLC}_{\beta}$ ) that is activated by GPCRs. Many RTKs and cytokine receptors can also initiate the $\mathrm{IP}_{3} / \mathrm{DAG}$ pathway by activating another isoform of phospholipase C, the $\gamma$ isoform $\left(\mathrm{PLC}_{\gamma}\right)$, which contains SH2 domains and is localized to the cytosol. The SH2 domains of $\mathrm{PLC}_{\gamma}$ bind to specific phosphotyrosines on the activated receptors, thus positioning the enzyme close to its substrate, $\mathrm{PI}(4,5) \mathrm{P}_{2}$, on the cytosolic face of the plasma membrane. In addition, the kinase activity associated with receptor activation phosphorylates tyrosine residues on the bound $\mathrm{PLC}_{\gamma}$, enhancing its hydrolase activity. Thus activated RTKs and cytokine receptors promote PLC $\gamma$ activity in two ways: by localizing the enzyme to the membrane and by phosphorylating it. As seen in Chapter 15, the $\mathrm{IP}_{3} /$ DAG pathway initiated by PLC has multiple physiological effects.

## Recruitment of PI-3 Kinase to Activated Receptors Leads to Synthesis of Three Phosphorylated Phosphatidylinositols

Besides the $\mathrm{IP}_{3} / \mathrm{DAG}$ pathway, many activated RTKs and cytokine receptors initiate another phosphoinositide pathway by recruiting the enzyme phosphatidylinositol-3 kinase (PI-3 kinase) to the membrane. PI-3 kinase is recruited to the cytosolic surface of the plasma membrane by binding of its SH2 domain to phosphotyrosines on the cytosolic domain of many activated RTKs and cytokine receptors. This recruitment positions the catalytic domain of PI-3 kinase near its phosphoinositide substrates on the cytosolic face of the plasma membrane. Unlike kinases we have encountered earlier that phosphorylate proteins, PI-3 kinase adds a phosphate to the $3^{\prime}$ carbon in one of two separate phosphatidylinositol substrates, leading to formation of two phosphatidylinositol 3-phosphates: phosphatidylino-sitol 3,4-bisphosphate $\left[\mathrm{PI}(3,4) \mathrm{P}_{2}\right]$ or phosphatidylinositol 3,4,5-trisphosphate $\left[\mathrm{PI}(3,4,5) \mathrm{P}_{3}\right]$ (Figure 16-28). By acting as docking sites for various signal-transducing proteins, these membrane-bound PI 3-phosphate products of the PI-3 kinase reactions in turn transduce signals downstream in several important pathways.

In some cells, this PI-3 kinase pathway can trigger cell division and prevent apoptosis, thus ensuring cell survival.
![li2924542480e](li2924542480e.jpg)

FIGURE 16-28 Generation of phosphatidylinositol
3-phosphates. The enzyme phosphatidylinositol-3 kinase (PI-3 kinase) is recruited to the membrane by many activated receptor tyrosine kinases (RTKs) and cytokine receptors. The 3-phosphate added by this enzyme, to yield $\mathrm{PI}(3,4) \mathrm{P}_{2}$ or $\mathrm{PI}(3,4,5) \mathrm{P}_{3}$, is a binding site for various signal-transducing proteins, such as the PH domain of protein kinase B. Phosphatidylinositol 4,5-bisphosphate is also the substrate of phospholipase C (see Figure 15-33). See L. Rameh and L. C. Cantley, 1999, J. Biol. Chem. 274:8347.

In other cells, this pathway induces specific changes in cell metabolism. PI-3 kinase was first identified in studies of the polyoma virus, a DNA virus that transforms certain mammalian cells. Transformation requires several virally encoded oncoproteins, including one termed "middle T." In an attempt to discover how middle T functions, investigators found PI-3 kinase in partially purified preparations of middle T, which suggested a specific interaction between the two. Then they set out to determine how PI-3 kinase might affect cell behavior.

When an inactive, dominant-negative version of PI-3 kinase was expressed in polyoma virus-transformed cells, it inhibited the uncontrolled cell proliferation characteristic of virus-transformed cells. This finding suggested that the normal kinase is important in certain signaling pathways essential for cell proliferation or for the prevention of programmed cell death (apoptosis; see Chapter 21). Subsequent work showed that PI-3 kinases participate in many signaling pathways related to cell growth and apoptosis. Of the nine PI-3 kinase homologs encoded by the human genome, the best characterized contains a p110 subunit with catalytic activity and a p85 subunit with an SH2 phosphotyrosinebinding domain.

## Accumulation of PI 3-Phosphates in the Plasma Membrane Leads to Activation of Several Kinases

Many protein kinases become activated by binding to phosphatidylinositol 3-phosphates on the cytosolic face of the plasma membrane. In turn, these kinases affect the activity of many cellular proteins. One important kinase that binds to PI 3-phosphates is protein kinase B (PKB), a serine/threonine kinase that is also called Akt. In addition to its kinase domain, PKB contains a PH domain, a conserved protein domain present in a wide variety of signaling proteins that
binds with high affinity to the 3-phosphates in both $\operatorname{PI}(3,4)$ $\mathrm{P}_{2}$ and $\operatorname{PI}(3,4,5) \mathrm{P}_{3}$. Since these inositol phosphates are present on the cytosolic face of the plasma membrane, binding recruits the entire protein to the cell membrane. In unstimulated, resting cells, the concentration of these phosphoinositides (collectively called PI 3-phosphates) is low, and PKB is present in the cytosol in an inactive form (Figure 16-29). Following hormone stimulation and the resulting rise in PI 3-phosphates, PKB binds to these membrane-bound molecules via its PH domain and becomes localized at the plasma membrane. Binding of PKB to PI 3-phosphates not only recruits the enzyme to the plasma membrane, but also releases inhibition of the catalytic site by the PH domain. However, maximal activation of PKB depends on recruitment of two other kinases, named PDK1 and PDK2.

PDK1 is recruited to the plasma membrane via binding of its own PH domain to PI 3-phosphates. Anchored to PI 3-phosphates, both PKB and PDK1 diffuse randomly in the plane of the plasma membrane, eventually coming close enough together so that PDK1 can phosphorylate PKB on a critical threonine residue in its activation loopyet another example of kinase activation by phosphorylation. Phosphorylation of a second serine, not in the loop segment, by PDK2 is necessary for maximal PKB activity (see Figure 16-29). As in the regulation of Raf activity (see Figure 16-24), release of an inhibitory domain and phosphorylation by other kinases regulate the activity of protein kinase B.

## Activated Protein Kinase B Induces Many Cellular Responses

Once fully activated, PKB can dissociate from the plasma membrane and phosphorylate its many target proteins throughout the cell, which have a wide range of effects on cell behavior. In many cells, activated PKB directly
![li2934542480e](li2934542480e.jpg)

FIGURE 16-29 Recruitment and activation of protein kinase B (PKB) in PI-3 kinase pathways. In unstimulated cells (step 1), PKB is in the cytosol with its PH domain bound to its catalytic kinase domain, inhibiting its activity. Hormone stimulation leads to activation of PI-3 kinase and subsequent formation of PI 3-phosphates (see Figure 16-28). The 3-phosphate group serves as a docking site on the plasma membrane for the PH domain of PKB (step 1) and another kinase, PDK1. Full activation of PKB requires phosphorylation both in the activation loop by PDK1 and at the C-terminus by a second kinase, PDK2 (step 1). See A. Toker and A. Newton, 2000, Cell 103:185, and S. Sarbassov et al., 2005, Curr. Opin. Cell Biol. 17:596.

phosphorylates and inactivates pro-apoptotic proteins such as Bad, a short-term effect that prevents activation of an apoptotic pathway leading to cell death (see Figure 21-40). Activated PKB also promotes survival of many cultured cells by phosphorylating the Forkhead transcription factor FOXO3a on multiple serine/threonine residues, thereby reducing its ability to induce expression of several proapoptotic genes. In the absence of growth factors, FOXO3a is nonphosphorylated and mainly localizes to the nucleus, where it activates transcription of several genes encoding pro-apoptotic proteins. When growth factors are added to the cells, PKB becomes active and phosphorylates FOXO3a. This allows the cytosolic phosphoserine-binding protein 14-3-3 to bind FOXO3a and thus sequester it in the cytosol. (Recall that 14-3-3 also retains many other phosphorylated proteins, including Raf, in an inactive state in the cytosol; see Figure 16-24.) A FOXO3a mutant in which the three serine residues that are targets for PKB are mutated to alanines is constitutively active and initiates apoptosis even in the presence of activated PKB. This finding demonstrates the importance of FOXO3a and PKB in controlling apoptosis of cultured cells. Deregulation of PKB is implicated in the pathogenesis of both cancer and diabetes, and in Section 16.8 we will see how PKB, activated downstream of the insulin RTK, promotes glucose uptake and storage in muscle and liver. This is another example of one signaling pathway controlling different cellular functions in different cells.

## The PI-3 Kinase Pathway Is Negatively Regulated by PTEN Phosphatase

Like virtually all intracellular signaling events, phosphorylation by PI-3 kinase is reversible. The relevant phosphatase, termed PTEN, has an unusually broad specificity. Although PTEN can remove phosphate groups attached to serine, threonine, and tyrosine residues in proteins, its ability to remove the 3-phosphate from $\mathrm{PI}(3,4,5) \mathrm{P}_{3}$ is thought to be its major function in cells. Overexpression of PTEN in cultured mammalian cells promotes apoptosis by reducing the level of $\mathrm{PI}(3,4,5) \mathrm{P}_{3}$, and hence the activation and anti-apoptotic effect of PKB.

The PTEN gene is deleted in multiple types of advanced human cancers. The resulting loss of the PTEN protein contributes to the uncontrolled growth of cells. Indeed, cells lacking PTEN have elevated levels $\mathrm{PI}(3,4,5) \mathrm{P}_{3}$ and PKB activity. Because PKB exerts an anti-apoptotic effect, loss of PTEN reduces the programmed cell death that is the normal fate of many cells. In certain cells, such as neuronal stem cells, absence of PTEN not only prevents apoptosis, but also leads to stimulation of cell cycle progression and an enhanced rate of cell proliferation. Knockout mice lacking PTEN have big brains with an excess numbers of neurons, attesting to PTEN's importance in the control of normal development.

## KEY CONCEPTS OF SECTION 16.5

## Phosphoinositide Signaling Pathways

- Many RTKs and cytokine receptors can initiate the $\mathrm{IP}_{3}{ }^{\prime}$ DAG signaling pathway by activating phospholipase $\mathrm{C}_{\gamma}$ $\left(\mathrm{PLC}_{\gamma}\right)$, a different PLC isoform than the one activated by G protein-coupled receptors.
- Activated RTKs and cytokine receptors also can initiate another phosphoinositide pathway by binding a PI-3 kinase, thereby allowing the enzyme access to its membrane-bound phosphoinositide substrates, which then become phosphorylated at the 3 position (PI 3-phosphates; see Figure 16-28).
- The PH domain in various proteins binds to PI 3-phosphates, forming signaling complexes associated with the cytosolic face of the plasma membrane.
- Protein kinase B (PKB) becomes partially activated by binding to PI 3-phosphates with its PH domain. Full activation of PKB requires phosphorylation by the kinase PDK1, which is also recruited to the membrane by binding to PI 3-phosphates, and by a second kinase, PDK2 (see Figure 16-29).
- Activated PKB promotes survival of many cells by directly phosphorylating and inactivating several pro-apoptotic proteins and by phosphorylating and inactivating the FOXO3a transcription factor, which otherwise induces synthesis of pro-apoptotic proteins.
- Signaling via the PI-3 kinase pathway is terminated by the PTEN phosphatase, which hydrolyzes the 3-phosphate in PI 3-phosphates. Loss of PTEN, a common occurrence in human tumors, promotes cell survival and proliferation.


### 16.6 Signaling Pathways Controlled by Ubiquitinylation and Protein Degradation: Wnt, Hedgehog, and NF- $\kappa$ B

All the signaling pathways we have discussed so far are reversible and so can be turned off relatively quickly if the extracellular signal is removed. In this section, we discuss several irreversible or only slowly reversible pathways in which a critical component-either a transcription factor or an inhibitor of a transcription factor-is ubiquitinylated and then proteolytically cleaved. First we discuss signaling by Wnt and Hedgehog (Hh) proteins, two evolutionarily conserved families of signaling proteins that play key roles in many developmental pathways and often induce expression of genes required for a cell to acquire a new identify or fate. Although Wnt and Hedgehog signaling pathways use different sets of receptors and signaling proteins, they do share similarities, which is why we group them together:

- In the resting state, key transcription factors in both pathways are ubiquitinylated and targeted for proteolytic cleavage, which renders them inactive.

- Activation of each pathway involves disassembly of large cytosolic protein complexes, inhibition of ubiquitinylation, and release of the active transcription factor.
- Kinases, including glycogen synthase kinase 3 (GSK3), play key roles in both signaling pathways.

Next we examine the $N F-\kappa B$ pathway, a third signaling pathway controlled by ubiquitinylation. In this case, an inhibitor of a transcription factor, rather than a transcription factor itself, is degraded following ubiquitinylation. In the resting state, the transcription factor termed NF- $\kappa \mathbf{B}$ is sequestered in the cytosol and bound to an inhibitor. Several stress-inducing conditions cause ubiquitinylation and immediate degradation of the inhibitor, allowing cells to respond immediately and vigorously by activating gene transcription. In learning how the NF- $\kappa$ B pathway is activated by one class of surface receptors, we also see a very different function of polyubiquitinylation: the formation of a scaffold to assemble a key signal transduction complex.

## Wnt Signaling Triggers Release of a Transcription Factor from a Cytosolic Protein Complex

The components of the Wnt and Hedgehog signaling pathways have been conserved throughout the evolution of metazoan organisms and were elucidated mainly through genetic analysis of developmental mutants in Drosophila. In vertebrates, mutations in these pathways are thought to trigger several types of cancers. In fact, the first vertebrate Wnt gene to be discovered, the mouse Wnt-1 gene, attracted notice because it was overexpressed in certain mammary cancers because of insertion of a mouse retroviral DNA, the mammary tumor virus (MMTV) genome, near the Wnt-1 gene; the retrovirus LTR promoter (see Figure 8-13) activated inappropriate expression of the Wnt-1 gene.

The word Wnt is an amalgamation of wingless, the corresponding fly gene, with int for the retrovirus integration site in mice. The human genome encodes 19 different Wnt proteins, and Wnt proteins are essential for numerous critical developmental events, such as brain development, limb patterning, and organogenesis. A major role for Wnt signaling in bone formation was revealed by the finding that inactivating mutations in Wnt pathway components affect bone density in humans. Wnt signaling is now known to control formation of osteoblasts (bone-forming cells). Additionally, Wnt signals are essential for proliferation of many types of stem cells (see Chapter 21) and in many other aspects of development.

Wnt proteins are secreted signaling molecules that are modified by linkage of a monounsaturated fatty acid, palmitoleic acid, to a serine in the middle of the protein. Like other growth factors, Wnt proteins interact with several cell-surface proteins and activate multiple downstream signal transduction pathways. The principal signaling receptor for Wnt proteins is Frizzled (Fz), which contains seven transmembrane $\alpha$ helices. Like the glucagon receptor (see Figure 15-13c), Fz has a large extracellular domain that is
connected to the first membrane-spanning $\alpha$ helix and comprises the major ligand binding site, but Fz does not activate a G protein. The palmitate attached to the Wnt protein binds to a specific site on the Fz extracellular domain and stabilizes the Wnt-Fz complex. This lipid is central to receptor engagement by Wnt proteins, and is the only known example of post-translational modification by a lipid that mediates a ligand-receptor interaction.

At least three different signal transduction pathways are activated by the binding of different Wnt proteins to Fz. The most widespread, "canonical" Wnt signaling pathway uses a second transmembrane protein, LRP (called Arrow in Drosophila), that associates with Frizzled in a Wnt signaldependent manner (Figure 16-30). Inactivating mutations in the genes encoding Wnt proteins, Frizzled, or LRP all have similar effects on the development of embryos, indicating that all three proteins are essential for Wnt signaling.

The central player in the "canonical" Wnt intracellular signal transduction pathway is called $\beta$-catenin in vertebrates and Armadillo in Drosophila. This multi-talented protein functions both as a transcriptional activator and as a membrane-cytoskeleton linker protein (see Figure 20-14). In the absence of a Wnt signal, the $\beta$-catenin molecules that are not attached to cell-adhesion molecules are bound in a cytosolic complex based on the scaffold protein Axin. The complex contains the adenomatous polyposis coli (APC) protein, so named because its loss may result in colorectal cancer. In the resting state, two kinases in the complex, casein kinase 1 (CK1) and GSK3, sequentially phosphorylate $\beta$-catenin on multiple serine and threonine residues. Some of these phosphorylated residues serve as binding sites for a ubiquitin-ligase protein named TrCP. $\beta$-Catenin is then ubiquitinylated and rapidly degraded by the 26 S proteasome (Figure 16-30a; for more on ubiquitinylation, see Figures 3-31 and 3-36).

The complete pathway by which Wnt signaling blocks the degradation of $\beta$-catenin has not yet been identified. We do know that Wnt binding to the complex of Fz and LRP leads to the phosphorylation of the LRP cytosolic domain, probably by free GSK3 or CK1. This enables Axin to bind to the cytosolic domain of the LRP co-receptor. This shift in Axin localization disrupts the interactions that stabilize the cytosolic complex containing Axin, GSK3, CK1, and $\beta$-catenin and thus prevents phosphorylation of $\beta$-catenin by CK1 and GSK3. This change, in turn, prevents ubiquitinylation and subsequent degradation of $\beta$-catenin and stabilizes it in the cytosol (Figure 16-30b). This process requires the Dishevelled (Dsh) protein, which becomes bound to the cytosolic domain of the Frizzled receptor and stabilizes Axin binding to LRP. The freed $\beta$-catenin translocates to the nucleus, where it associates with a transcription factor (TCF) and functions as a coactivator to induce expression of particular target genes, often including those that promote cell proliferation. (The name is unfortunately confusing; this TCF is different from the TCF protein that functions in the MAP kinase pathway; see Figure 16-26.)

![li2944542480e](li2944542480e.jpg)

FIGURE 16-30 "Canonical" Wnt signaling pathway. (a) In the absence of Wnt, the transcription factor TCF is bound to promoters or enhancers of target genes, but its association with transcriptional repressors such as Groucho (Gro) inhibits gene activation. $\beta$-catenin is bound in a complex with Axin (a scaffold protein), APC, and the kinases CK1 and GSK3, which sequentially phosphorylate $\beta$-catenin at multiple serine and threonine residues. In particular, Axin-mediated formation of this complex facilitates phosphorylation of $\beta$-catenin by GSK3 by an estimated factor of 20,000. The E3 TrCP ubiquitin ligase then binds to two phosphorylated $\beta$-catenin residues, leading to $\beta$-catenin ubiquitinylation and degradation in proteasomes. (b) Binding of Wnt to its receptor Frizzled (Fz) and to the LRP co-receptor triggers phosphorylation of LRP by GSK3 and CK1, allowing subsequent binding of the Dishevelled scaffold protein. Binding of Axin to the phosphorylated LRP protein and to Dishevelled disrupts the Axin-APC-CK1-GSK3- $\beta$-catenin complex, preventing phosphorylation of $\beta$-catenin by CK1 and GSK3 and leading to accumulation of $\beta$-catenin in the cell. After translocation to the nucleus, $\beta$-catenin binds to TCF to displace the Gro repressor and recruits co-activator proteins including Pygo, LGS, and others to activate gene expression. See R. van Amerongen and R. Nusse, 2009, Development 136:3205; E. Verheyen and C. Gottardi, 2010, Dev. Dynam. 239:34; and J. Holland et al., 2013, Curr. Opin. Cell Biol. 25:254. See also the Wnt Homepage, http://web.stanford.edu/group/nusselab/cgi-bin/wnt.

5Aberrant hyperactive Wnt signaling is implicated in the progression of many cancers; more than 90 percent of human colon cancers display hyperactivity of the Wnt signaling pathway in that the level of free $\beta$-catenin is abnormally high (see Chapter 24). This observation provided one of the earliest clues that $\beta$-catenin can activate many growth-promoting genes. Inactivating mutations in genes encoding APC and Axin are found in multiple types of human cancers, as are mutations in $\beta$-catenin phosphorylation sites for GSK3 or CK1; these mutations reduce formation of the cytosolic complex that inactivates $\beta$-catenin (see Figure 16-30a), reduce $\beta$-catenin degradation, and allow $\beta$-catenin to activate gene expression in the absence of the normal Wnt signal.

Among the Wnt target genes are many that also control Wnt signaling, indicating a high degree of feedback regulation. The importance of $\beta$-catenin stability and location means that Wnt signals affect a critical balance between the three pools of $\beta$-catenin in the cell: at the membranecytoskeleton interface, in the cytosol, and in the nucleus.

In order to signal, Wnt must also bind to cell-surface proteoglycans. Evidence for the participation of proteoglycans in Wnt signaling comes from Drosophila Sugarless (Sgl) mutants, which lack a key enzyme needed to synthesize the glycosaminoglycans heparan and chondroitin sulfate. These mutants have greatly depressed levels of Wingless (the fly

Wnt protein) and exhibit other phenotypes associated with defects in Wnt signaling. How proteoglycans facilitate Wnt signaling is unknown, but perhaps binding of Wnt to specific glycosaminoglycan chains is required for it to bind to its receptor, Fz, or its co-receptor, LRP. This mechanism would be analogous to the binding of fibroblast growth factor to heparan sulfate, which enhances binding of FGF to its receptor tyrosine kinase (see Figure 16-15).

## Concentration Gradients of Wnt Protein Are Essential for Many Steps in Development

Wnts are secreted proteins; however, in part because of the hydrophobic lipid that is covalently linked to these proteins, they diffuse only a short distance from a signaling cell and generally have localized effects. As Wnt diffuses farther and farther away from secreting cells, its concentration decreases. Different Wnt concentrations induce different fates in target cells: cells that receive a large amount of Wnt turn on certain genes and form certain structures; cells that receive a smaller amount turn on different genes and so form different structures. Signals that induce different cell fates depending on their concentration at their target cells are referred to as morphogens.

Perhaps the most striking example of Wnt as a morphogen occurs during regeneration of the planarian Schmidtea mediterranea (see Figure 1-22e). If the planarian's head is cut off, a new one regenerates within 14 days; a normal, albeit

![li2954542480e](li2954542480e.jpg)

FIGURE 16-31 Gradients of Wnt are essential for normal regeneration of a head and a tail by planaria. (a) As indicated in the diagram, a small piece excised from the middle of the body of the planarian S. mediterranea was placed in culture. In situ mRNA hybridization performed after 4 days indicated that Wnt mRNA (pink dots) was expressed in cells at both the anterior and posterior wound sites, but that the Wnt inhibitor Notum (blue dots) was expressed only at the anterior wound site. Thus a posterior-to-anterior gradient of Wnt protein is formed. (b) As shown by in situ hybridization, the WntP-2 gene (pink dots) is expressed in a posterior-to-anterior gradient in adult planaria. See J. Witchley et al., 2013, Cell Rep. 4:633. (c) After 14 days, a normal, albeit smaller, worm has regenerated a head, easily visualized by the two eyes, from the anterior wound site and a tail from the posterior. Treatment of the excised body piece with an inhibitory RNA specific for $\beta$-catenin results in regeneration of a two-headed planarian, whereas treatment with an inhibitory RNA specific for Notum results in regeneration of a two-tailed planarian. Treatment of the excised body piece with two inhibitory RNAs, one specific for $\beta$-catenin and the other for Notum, results in a phenotype similar to that caused by the loss of $\beta$-catenin alone: a two-headed planarian is regenerated. These experiments give rise to the model depicted in part (c), in which $\beta$-catenin, stabilized by addition of Wnt to cells, causes expression of genes that promote tail formation; inhibition of Wnt/ $\beta$-catenin signaling by Notum causes a head to be formed. (Part (b) photo: Jessica Witchley and Peter Reddien. Part (c, left): Republished with permission of AAAS, from C. Petersen et al., "Smed- $\beta$-catenin-1 Is Required for Anteroposterior Blastema Polarity in Planarian Regeneration," Science (2008) 319:5861, pp. 327-330; photos courtesy of J. Witchley and Peter Reddien.)
smaller, worm is regenerated. Similarly, after removal of the tail, a new one regenerates. Most strikingly, a small body piece from the middle of the animal regenerates a normal head from the anterior (head-facing) wound and a tail from the posterior wound (Figure 16-31a). Gradients of Wnt control this polarity. Wnt proteins are expressed in a posterior-to-anterior gradient in normal adult planaria (Figure 16-31b) and predominantly at the posterior edge of the excised body piece (see Figure 16-31a). In contrast, the secreted extracellular enzyme Notum is produced only at the anterior wound. Notum inhibits Wnt signaling by cleaving off palmitoleic acid, the fatty acid that is covalently attached to Wnt and, as we learned above, is essential for Wnt signaling.

Figure 16-31c shows some elegant experiments establishing that signaling at the posterior wound by Wnt causes induction of a tail and that it is the absence of Wnt signaling at the anterior wound, rather than the presence of Notum, that induces head formation:

1. In the absence of all Wnt signaling, caused by treatment of the excised body fragment with an inhibitory RNA specific for $\beta$-catenin, a two-headed planarian is regenerated; in this case, Wnt signaling is absent at both the anterior and posterior wounds, and thus heads are induced at both wound sites.
2. Conversely, in the absence of Notum signaling, caused by treatment with an inhibitory RNA specific for Notum,
a two-tailed planarian is regenerated; in this case, Wnt is present and active at both the anterior and posterior wounds, and thus two tails are formed.
3. In the absence of both Wnt and Notum signaling, caused by treatment with inhibitory RNAs for both $\beta$-catenin and Notum, the phenotype is similar to that caused by the loss of Wnt alone: a two-headed planarian is regenerated. Wnt signaling is absent at both the anterior and posterior wounds, inducing head regeneration at both sites, and the absence of Notum is irrelevant in the absence of Wnt. Thus it is the absence of Wnt/ $\beta$-catenin signaling, not the expression of Notum, that causes a tail to be formed.

These experiments support the role of Wnt concentration gradients in planarian regeneration. In Chapter 21, we will learn that mature planaria contain multipotent stem cells termed neoblasts that can differentiate into any body cell type; clearly gradients of Wnt protein play a major role in instructing neoblasts to differentiate into the multiple cell types that constitute the planarian head or tail.

## Hedgehog Signaling Relieves Repression of Target Genes

The Hedgehog (Hh) signaling pathway is similar to the Wnt pathway in that two membrane proteins, one with seven

membrane-spanning segments, are required to receive and transduce a signal. The Hh pathway also involves the disassembly of an intracellular complex containing a transcription factor. Hh signaling differs from Wnt signaling in that its two membrane receptors move between the plasma membrane and intracellular vesicles, and that in mammals, Hh signaling is restricted to the primary cilium that protrudes from the cell surface. But, like Wnt proteins, Hh proteins can act as morphogens and signal nearby cells. Hh signaling plays essential roles in the development of nearly every organ system in vertebrates, from determining the fates of different segments of the nervous system to regulating lung morphogenesis and hair follicle formation. One of the three mammalian Hh proteins, Sonic Hedgehog (Shh), is essential for normal patterning of the limbs; abnormal expression of Shh in an anterior region of the developing limb, in addition to its normal expression in the posterior domain, leads to polydactyly (extra digits).

Processing of Hh Precursor Protein Hedgehog proteins are formed from a precursor protein with autoproteolytic activity that enables the protein to cut itself in half while still in the endoplasmic reticulum. The cleavage produces an N terminal fragment, which is subsequently secreted to signal other cells, and a C-terminal fragment, which is degraded. As shown in Figure 16-32, cleavage of the precursor is accompanied by covalent addition of the lipid cholesterol to the new carboxyl terminus of the N -terminal fragment. A second modification to Hedgehog, the addition of a palmitoyl group to the N -terminus, makes the protein even more hydrophobic.

Hh proteins can travel relatively long distances-up to $300 \mu \mathrm{~m}$ in the developing vertebrate limb—but how an Hh protein with two attached lipids can spread in the hydrophilic environment of the extracellular space is not well understood. Both Hh and Wnt are found anchored to the phospholipid monolayers of extracellular lipoprotein particles (see Figure 14-27 for the structure of a typical lipoprotein) via their attached lipid or cholesterol groups. In many cases, the majority of the Hh proteins produced by a cell remain bound to its plasma membrane; in such cases, Hh signals mainly by cell-cell contact. Extracellular aggregates of Hh proteins, stabilized by hydrophobic interactions between their lipid groups, have also been observed experimentally. Regardless of the detailed mechanism involved, the two attached hydrophobic groups limit the diffusion of Hh and thus its range of action in tissues. As with Wnt proteins, spatial restriction plays a crucial role in constraining the effects of Hh proteins.

Hedgehog Signaling Pathway in Drosophila Genetic studies in Drosophila indicated that two membrane proteins, Smoothened (Smo) and Patched (Ptc), are required to receive a Hedgehog signal and transduce it to the cell nucleus. Smoothened has seven membrane-spanning $\alpha$ helices and is related in sequence to the Wnt receptor Fz. Patched is predicted to contain twelve transmembrane $\alpha$ helices and is most similar structurally to the Niemann-Pick C1 (NPC1)
![li2964542480e](li2964542480e.jpg)

FIGURE 16-32 Processing of Hedgehog (Hh) precursor protein. Cells synthesize a $45-\mathrm{kDa}$ Hh precursor, which in the endoplasmic reticulum undergoes a nucleophilic attack by the thiol side chain of cysteine 258 (Cys-258) on the carbonyl carbon of the adjacent residue glycine 257 (Gly-257), forming a high-energy thioester intermediate. Enzyme activity in the C-terminal domain then catalyzes the formation of an ester bond between the hydroxyl group of cholesterol and glycine 257, cleaving the precursor into two fragments. The N-terminal signaling fragment (blue) retains the cholesterol group and is also modified by the addition of a palmitoyl group to the N -terminus. The two hydrophobic anchors may tether the secreted, processed Hh protein to the plasma membrane. See P. Thérond, 2012, Curr. Opin. Cell Biol. 24:173.
protein, a member of the ABC superfamily of membrane transport proteins (see Table 11-3).

Figure 16-33 depicts the current model of the Hedgehog pathway in Drosophila. Evidence supporting this model initially came from the study of fly embryos with loss-offunction mutations in the hedgehog (Hh) or smoothened (Smo) genes. Both types of mutant embryos have very similar developmental phenotypes; the name hedgehog came from the appearance of Hh mutant embryos, which were covered by an array of disorganized hairlike bristles that resembled hedgehog spines. Moreover, both the $H h$ and Smo genes are required to activate transcription of the same target genes (e.g., patched and wingless) during embryonic development. In contrast, loss-of-function mutations in the patched (Ptc) gene produce a quite different phenotype, one similar to the

![li2974542480e](li2974542480e.jpg)

FIGURE 16-33 Hedgehog signaling in Drosophila. (a) In the absence of Hedgehog (Hh), Patched (Ptc) protein inhibits Smoothened (Smo), which is present largely in the membranes of internal vesicles, and enhances its degradation. A complex containing the kinase Fused (Fu); other kinases including protein kinase A (PKA), glycogen synthase kinase $3 \beta$ (GSK3 $\beta$ ), and casein kinase 1 (CK1); the kinesin-related motor protein Costal-2 (Cos2); and Cubitis interruptus (Ci), a zinc-finger transcription factor, binds to microtubules. In this complex, Ci becomes phosphorylated in a series of steps catalyzed by PKA, GSK3 $\beta$, and CK1. The phosphorylated Ci is then proteolytically cleaved by the ubiquitinproteasome pathway, generating the N -terminal fragment Ci75, which is transported into the nucleus and functions as a transcriptional
effect of flooding the embryo with Hedgehog protein. These findings suggested that, in the absence of $\mathrm{Hh}, \mathrm{Ptc}$ represses target genes by inhibiting a signaling pathway needed for gene activation. The additional observation that Smo is required for the transcription of Hh target genes in mutants lacking patched function places Smo downstream of Ptc in the Hh pathway. Together with other experiments showing that Hh binds directly to Ptc, the evidence indicates that Hh binding to Ptc prevents Ptc from blocking Smo action, thus activating the transcription of target genes.

Subsequent biochemical and cell biological studies showed that, in the absence of $\mathrm{Hh}, \mathrm{Ptc}$ is enriched in the plasma membrane, but Smo is found in membranes of internal vesicles. Furthermore, the long cytosolic C-terminal segment of Smo is folded in such a way that it cannot bind downstream signaling proteins (Figure 16-33a). How Ptc inhibits Smo function and enhances its degradation is not clear; one theory is that Ptc transports a small molecule into the cell that inhibits Smo function.

A large cytosolic protein complex in the Hh pathway consists of several proteins including Fused (Fu), a serinethreonine kinase; protein kinases PKA, GSK3 $\beta$, and CK1,
![li2984542480e](li2984542480e.jpg)
repressor of Hh target genes. (b) Hh binds to Ptc, causing Ptc to be endocytosed from the cell surface and degraded, thereby relieving the inhibition of Smo. Smo then moves to the plasma membrane, is phosphorylated by PKA, CK1, and other kinases, binds Cos2, and is stabilized from degradation. Both Fu and Cos 2 become extensively phosphorylated, and most importantly, the Fu-Cos2-Ci complex becomes dissociated. This leads to the stabilization of the full-length Ci , which moves into the nucleus, displaces the repressor Ci75 from the promoter of target genes, recruits the CREB-binding activator protein (CBP), and induces expression of target genes. The exact membrane compartments in which Ptc and Smo respond to Hh and function are unknown. See S. Goetz and K. Anderson, 2010, Nat. Rev. Genet. 11:331.
which we have encountered previously in other signaling pathways; Costal-2 (Cos2), a microtubule-associated kinesin-like protein; and Cubitis interruptus (Ci), a zinc finger-containing transcription factor. The complex is bound to, and may move along, microtubules in the cytosol. Importantly, phosphorylation of Ci by at least three kinases in the complex causes binding of a component of a ubiquitin ligase complex, which in turn directs ubiquitinylation of Ci and its targeting to proteasomes. There Ci undergoes proteolytic cleavage; the resulting Ci fragment, designated Ci75, translocates to the nucleus and represses expression of Hh target genes.

Following binding of Hh to the receptor Ptc, the $\mathrm{Hh}-\mathrm{Ptc}$ complex, like other receptor-hormone complexes, is endocytosed from the cell surface into internal vesicles and is eventually degraded; the binding of Hh to Ptc also inhibits the ability of Ptc to inhibit Smo (see Figure 16-33a). Simultaneously, Smo moves from internal vesicles to the plasma membrane; the C-terminal segment of Smo becomes phosphorylated and adopts an "open" conformation. This change triggers several cellular responses, including an increase in phosphorylation of Fu and Cos2. Importantly, the complex

of $\mathrm{Fu}, \mathrm{Cos} 2$, and Ci dissociates from microtubules, and $\operatorname{Cos} 2$ becomes associated with the phosphorylated C-terminal tail of Smo. The resulting disruption of the Fus/Cos2/Ci complex causes a reduction in both phosphorylation and cleavage of Ci . As a result, full-length Ci is released and translocates to the nucleus, where it binds to the transcriptional co-activator CREB-binding protein (CBP), promoting the expression rather than repression of Hh target genes.

Regulation of Hh Signaling Feedback control of the Hh pathway is important because unrestrained Hh signaling can cause cancerous overgrowth or formation of the wrong cell types. In Drosophila, one of the genes induced by the Hh signal is patched. The subsequent increase in expression of Patched antagonizes the Hh signal in large measure by reducing the pool of active Smoothened protein. Thus the system is buffered: if during development too much Hh signal is made, a consequent increase in Ptc will compensate; if too little Hh signal is made, the amount of Ptc is decreased.

## Hedgehog Signaling in Vertebrates Requires Primary Cilia

The Hedgehog signaling pathway in vertebrates shares many conserved features with the Drosophila pathway, but there are also some striking differences. First, mammalian genomes contain three $H h$ genes and two $P t c$ genes, which are expressed differentially among various tissues. Second, mammals express three Gli transcription factors, which collectively perform the roles of the single Ci transcription factor in Drosophila. All other components of the Hh pathway in Drosophila also are conserved in mammals.

The most fascinating aspect of the mammalian Hh pathway is the importance of primary cilia. Cilia are long, plasma membrane-enveloped structures that protrude from the cell surface. The roles of cilia and flagella in specialized cell types-in tracheal cells in moving materials along the airway surface and in sperm in flagellum-powered locomotion-are well known (see Figure 1-14). Most vertebrate cells have a single cilium called the primary cilium (Figure 16-34), a slim, nonmotile structure that projects from the surface of nearly all vertebrate cells but is conspicuously absent in all invertebrate cell types that have been examined.

As we will learn in Chapter 18, a cilium is extended and maintained by the transport of proteins and particles along a bundle of microtubules in its center; different intraflagellar transport (IFT) motor proteins move proteins and particles from the base of the cilium to the tip and in the opposite direction. Some of the first evidence for a role of cilia in Hh signaling came from a screen for mutations that altered early mouse development in a manner similar to that seen in embryos with altered Hh signaling: the mutant phenotypes included losses of certain types of cells in the neural tube that require high concentrations of one Hh protein to develop. Many of these mutations were in genes encoding IFT proteins, indicating a role for cilia (or flagella) in Hh signaling.

Subsequent analysis showed that, in the absence of Hh signaling, Ptc is localized to the membrane of the primary cilium and Smo is located in internal vesicles near the base of the cilium (Figure 16-34a). As in Drosophila, a cytosolic complex of Gli, SUFU, and several kinases leads to phosphorylation of Gli, its proteolytic cleavage, and translocation of a Gli fragment termed GliR into the nucleus, where it binds to regulatory regions of Gli-responsive genes and blocks their induction.

After Hh addition, Smo moves to the ciliary membrane and then to the tip of the primary cilium, while the $\mathrm{Hh} / \mathrm{Ptc}$ complex is internalized and degraded (Figure 16-34b). These movements of Smo involve phosphorylation of the Smo C-terminal cytosolic domain by $\beta$-adrenergic receptor kinase (BARK), the same enzyme that modifies G protein-coupled receptors (see Figure 15-32). $\beta$-arrestin then binds to Smo. In turn, $\beta$-arrestin recruits the microtubule motor protein Kif7, which binds to the microtubules in the core of the cilium and moves Smo up the ciliary membrane. At the same time, the cytosolic complex containing Gli is disrupted, preventing Gli cleavage into a repressor fragment. Gli subsequently accumulates at the tip of the cilium, a process also requiring the Kif7 motor protein. There it becomes activated by Smo by a mechanism not yet known in detail, and then another motor protein, a dynein, moves the activated Gli, termed Gli*, to the base of the cilium (see Figure 16-34b) As in Drosophila, this active transcription factor then moves into the nucleus, where it activates expressions of multiple target genes.

Inappropriate activation of Hh signaling is the cause of several types of human tumors, including medulloblastomas (cerebellum tumors) and rhabdomyosarcomas (muscle tumors). Primary cilia are essential for this abnormal Hh signaling, and drugs that inhibit the function of primary cilia are being tested on animal models of these cancers. For instance, expression of a mutant activated form of Smoothened in the postnatal mouse brain causes medulloblastomas, but these tumors do not form if, simultaneously, a gene encoding an essential ciliary protein is inactivated.

## Degradation of an Inhibitor Protein Activates the NF-кB Transcription Factor

In the resting state of both the Wnt and Hedgehog pathways, a key transcription factor is ubiquitinylated and subjected to proteolytic degradation, generating a protein fragment that acts as a transcriptional repressor; activation of the signaling pathway involves blockage of ubiquitinylation and release of the transcription factor in its active state. The NF- $\kappa$ B pathway works in the opposite manner: in the resting state, the NF-кB transcription factor is retained in the cytosol bound to an inhibitor; activation of the signaling pathway involves ubiquitinylation of the inhibitor followed by its degradation, triggering release of the active transcription factor. This mechanism allows cells to respond to a variety of stress signals by immediately and vigorously activating gene transcription. The steps in the NF- $\kappa$ B pathway were revealed in studies with both mammalian cells and Drosophila.

![li2994542480e](li2994542480e.jpg)

FIGURE 16-34 Hedgehog signaling in vertebrates. Hedgehog (Hh) signaling occurs in primary cilia, but otherwise the overall process is similar to that in Drosophila. (a) In the absence of $\mathrm{Hh}, \mathrm{Ptc}$ is localized to the ciliary membrane and the base of the cilium. In an unknown manner, Ptc blocks the entry of Smo to the plasma membrane; Smo is present mainly in the membrane of internal vesicles. The kinesin Kif7 (the Cos2 homolog) binds to microtubules at the cilium base, where it prevents the transcription factor Gli (the vertebrate homolog of C ) from entering the cilium. Kif7 and Gli are part of a complex that includes SUFU and the kinases CK1, PKA, and GSK3 $\beta$, which phosphorylate Gli and promote its
(b) $\rightarrow \mathrm{Hh}$

Primary cilium
![li3004542480e](li3004542480e.jpg)
proteolytic cleavage to form the repressor GliR. (b) Hh binding triggers endocytosis and degradation of the $\mathrm{Hh} / \mathrm{Ptc}$ complex, movement of Smo to the plasma membrane, and then, together with several proteins bound to it, its movement to the tip of the cilium. There Smo triggers dissociation of the SUFU-Gli complex. Rather than being degraded, Gli accumulates, becomes modified by addition of several phosphate and acetyl groups, forming the active $\mathrm{Gli}^{*}$, and is then transported down the cilium by a dynein motor protein. Gli* is then released into the cytosol, translocates into the nucleus, and activates gene expression. See J. Briscoe and P. Thérond, 2013, Nat. Rev. Mol. Cell Biol. 14:416.

NF-кB (an acronym for the somewhat unwieldy descriptor "nuclear-factor kappa-light-chain enhancer of activated B cells") is rapidly activated in mammalian immune-system cells in response to bacterial and viral infection, inflammation, and a number of other stressful situations, such as ionizing radiation. The NF-кB pathway is activated in some cells of the immune system when components of bacterial or fungal cell walls bind to certain Toll-like receptors on the cell surface (see Figure 23-35). This pathway is also activated by so-called inflammatory cytokines, such as tumor
necrosis factor alpha (TNFa) and interleukin 1 (IL-1), which are released by nearby cells in response to infection. In all of these cases, binding of ligand to its receptor induces assembly of a multiprotein complex in the cytosol near the plasma membrane that triggers a signaling pathway resulting in activation of the NF-кB transcription factor.

NF-кB was originally discovered on the basis of its transcriptional activation of the gene encoding the light chains of antibodies (immunoglobulins) in B cells. It is now thought to be the master transcriptional regulator of the immune system

![li3014542480e](li3014542480e.jpg)

FIGURE 16-35 Activation of the NF-кB signaling pathway. (a) In resting cells, the dimeric transcription factor NF-кB, composed of p50 and p65 subunits, is sequestered in the cytosol, bound to the inhibitor I-кB $\alpha$. Step A Activation of the trimeric I-кB kinase is stimulated by many agents, including viral infection, ionizing radiation, binding of the pro-inflammatory cytokines TNF $\alpha$ or IL-1 to their respective receptors, or activation of any of several Toll-like receptors by components of invading bacteria or fungi. Step $\mathbf{B}$ The $\beta$ subunit of I-кB kinase then phosphorylates the inhibitor I-кB $\alpha$, which then binds an E3 ubiquitin ligase. Steps $\mathbf{B}$ and $\mathbf{B}$ Subsequent lysine 48-linked polyubiquitinylation of I-кB $\alpha$ targets it for degradation by proteasomes. Step $\mathbf{B}$ The removal of I-кB $\alpha$ unmasks the nuclear-localization signals in both subunits of NF-кB, allowing their translocation to the nucleus. Step $\mathbf{B}$ In the nucleus, NF-кB activates transcription of numerous target genes, including the gene encoding I-кB $\alpha$, which acts to terminate signaling, and genes encoding various inflammatory cytokines. See R. Khush et al., 2001, Trends Immunol. 22:260, and J-L Luo et al., 2005, J. Clin. Invest. 115:2625. (b) Binding of interleukin-1 $\beta$ (IL-1 $\beta$ ) to heterodimeric IL-1 receptors triggers receptor oligomerization and recruitment of several proteins to the receptor cytosolic domain, including TRAF6, an E3 ubiquitin ligase, which catalyzes synthesis of long lysine-63-linked polyubiquitin chains linked to TRAF6, NEMO, and other proteins in the complex. The polyubiquitin chains function as a scaffold to recruit the kinase TAK1 and the NEMO subunit of the trimeric I-кB kinase complex. TAK1 then phosphorylates itself and the $\beta$ subunit of I-кB kinase, activating its kinase activity and enabling it to phosphorylate I-кB $\alpha$. See B. Skaug et al., 2009, Annu. Rev. Biochem. 78:769, and J. Napetschnig and H. Wu, 2013 Annu. Rev. Biophys. 42:443. [IL-1 receptor data from B.J. Smith et al., 2010, Proc. Natl. Acad. Sci. USA 107:6771, PDB ID 3loh; Q. Li, Y. L. Wong, and C. Kang., 2014, Biochim. Biophys. Acta. 1838:1313, PDB ID 2mfr; and S. R. Hubbard, 1994, Nature 372:746, PDB ID 1irk.]

in mammals. Although flies do not make antibodies, NF- $\kappa$ B homologs in Drosophila induce synthesis of a large number of secreted antimicrobial peptides in response to bacterial and viral infection. This phenomenon indicates that the NF- $\kappa$ B regulatory system has been conserved during evolution and is more than half a billion years old.

Biochemical studies in mammalian cells and genetic studies in flies have provided important insights into the operation of the NF- $\kappa$ B pathway. The two subunits ( p 65 and p50) of the heterodimeric NF- $\kappa$ B transcription factor share a region of homology at their N -termini that is required for their dimerization and binding to DNA. In cells that are not undergoing a stress or responding to signs of an infection, direct binding to an inhibitor called I- $\kappa$ B $\alpha$ sequesters NF- $\kappa$ B in an inactive state in the cytosol. A single molecule of I- $\kappa$ B $\alpha$ binds to the paired N -terminal domains of the p50-p65 subunits, thereby masking their nuclear-localization signals (Figure 16-35a).

A three-protein complex termed $I-\kappa B$ kinase operates immediately upstream of NF- $\kappa$ B and is responsible for releasing it from sequestration. The $\beta$ subunit of I- $\kappa$ B kinase is the point of convergence of all of the extracellular signals noted above that activate NF- $\kappa$ B. Within minutes of stimulation of the cell by an infectious agent or inflammatory cytokine, the $\beta$ subunit of I- $\kappa$ B kinase becomes activated by phosphorylation and then phosphorylates two N -terminal serine residues on I- $\kappa \mathrm{B} \alpha$ (Figure 16-35a, steps $\mathbf{\square}$ and $\mathbf{\square}$ ). An E3 ubiquitin ligase then binds to these phosphoserines and polyubiquitinylates I- $\kappa \mathrm{B} \alpha$, triggering its immediate degradation by a proteasome (steps $\mathbf{\square}$ and $\mathbf{\square}$ ). In cells expressing mutant forms of I- $\kappa$ B $\alpha$ in which these two serines have been changed to alanine and so cannot be phosphorylated, NF- $\kappa$ B is permanently inactive, demonstrating that phosphorylation of I- $\kappa \mathrm{B} \alpha$ is essential for pathway activation.

The degradation of I- $\kappa$ B $\alpha$ exposes the nuclearlocalization signals on NF- $\kappa \mathrm{B}$, which then translocates into the nucleus and activates transcription of a multitude of target genes (Figure 16-35a, steps $\mathbf{\square}$ and $\mathbf{\square}$ ). Despite its activation by proteolysis, NF- $\kappa$ B signaling is eventually turned off by a negative feedback loop because one of the genes whose transcription is immediately induced by NF- $\kappa$ B encodes I- $\kappa \mathrm{B} \alpha$. The resulting increased levels of the I- $\kappa \mathrm{B} \alpha$ protein bind active NF- $\kappa$ B in the nucleus and return it to the cytosol.

In many immune-system cells, NF- $\kappa$ B stimulates transcription of more than 150 genes, including those encoding cytokines and chemokines; the latter attract other immunesystem cells and fibroblasts to sites of infection. NF- $\kappa$ B also promotes expression of receptor proteins that enable neutrophils (a type of white blood cell) to migrate from the blood into the underlying tissue (see Figure 20-40). In addition, NF- $\kappa$ B stimulates expression of iNOS, the inducible isoform of the enzyme that produces nitric oxide (see Figure 15-36), which is toxic to bacterial cells, as well as expression of several anti-apoptotic proteins, which prevent cell death. Thus this single transcription factor coordinates and activates the body's defense, either directly by responding to pathogens and stress or indirectly by responding to signaling molecules released from other infected or wounded tissues and cells.

## Polyubiquitin Chains Serve as Scaffolds Linking Receptors to Downstream Proteins in the NF-кB Pathway

As we have just seen, the $\beta$ subunit of I- $\kappa$ B kinase is the point of convergence for extracellular signals transmitted through multiple receptors, including Toll-like and IL-1 receptors. Since the cytosolic domains of the Toll-like and IL-1 receptors have no enzyme activity, it was a mystery for many years how activation of these receptors led to phosphorylation and activation of the $\beta$ subunit of I- $\kappa$ B kinase. Early work showed that the presence of IL-1 led to oligomerization of the two IL-1 receptor proteins and the binding of several proteins to its cytosolic domain, including TRAF6, an E3 ubiquitin ligase that synthesizes polyubiquitin chains. Since all polyubiquitinylation was then thought to signal degradation by proteasomes, researchers looked for ubiquitinylated target proteins that were quickly destroyed. Not finding these, scientists looked for other possible roles for polyubiquitin and soon found that, depending on the specific E3 ubiquitin ligase, ubiquitin forms multiple types of polymers that have different structures and biological functions.

The E3 ubiquitin ligase that ubiquitinylates I- $\kappa$ B $\alpha$ links the carboxyl terminus of one ubiquitin to lysine 48 (K48) on another; this poly-K48-linked ubiquitin targets the attached protein to the proteasome (see Figure 16-35a). The E3 ligase TRAF6, in contrast, links the carboxyl terminus of one ubiquitin to lysine 63 (K63) on another (see Figure 3-36). The resultant poly-K63 ubiquitin chain does not target proteins for degradation; rather, these ubiquitin chains act as scaffolds that bind proteins that have a poly-K63 ubiquitin-binding domain. One of these proteins is the protein kinase TAK1, which becomes activated by binding to the polyubiquitin chain; another is the NEMO subunit of I- $\kappa$ B kinase. Binding to poly-K63 ubiquitin thus brings the kinase and its target, the $\beta$ subunit of I- $\kappa$ B kinase, into proximity so that TAK1 can phosphorylate and thereby activate this downstream kinase (Figure 16-35b). As noted above, this kinase then phosphorylates I- $\kappa \mathrm{B} \alpha$. Thus different types of polyubiquitin chains participate in very different ways in transmitting the IL-1 signal to activation of the NF- $\kappa$ B transcription factors.

## KEY CONCEPTS OF SECTION 16.6

## Signaling Pathways Controlled by Ubiquitinylation and Protein Degradation: Wnt, Hedgehog, and NF-кB

- Many signaling pathways involve ubiquitinylation and proteolysis of target proteins and so are irreversible or only slowly reversible. These target proteins can be either a transcription factor or an inhibitor of a transcription factor.
- Wnt controls numerous critical developmental events, such as brain development, limb patterning, and organogenesis. Hedgehog also functions as a morphogen during development. Activating mutations in both pathways can cause cancer.

- Both Hedgehog and Wnt are secreted proteins that contain lipid anchors that reduce their signaling ranges. The fatty acid covalently attached to Wnt is essential for binding to its receptor.
- Wnt signals act through two cell-surface proteins, the receptor Frizzled and co-receptor LRP, and an intracellular complex containing $\beta$-catenin (see Figure 16-30). Binding of Wnt promotes the stability and nuclear localization of $\beta$-catenin, which either directly or indirectly promotes activation of the TCF transcription factor.
- Gradients of Wnt protein concentration are essential for many steps in development, including regeneration of a head and tail during planarian regeneration (see Figure 16-31).
- The Hedgehog signal also acts through two cell-surface proteins, Smoothened and Patched, and an intracellular complex containing the Cubitis interruptus (Ci) transcription factor (see Figure 16-33). An activating form of Ci is generated in the presence of Hedgehog; a repressing Ci fragment is generated in the absence of Hedgehog. Both Patched and Smoothened change their subcellular location in response to Hedgehog binding to Patched.
- Hh signaling in vertebrates requires primary cilia and intraflagellar transport proteins. Patched localizes to the ciliary membrane in the absence of Hh , and Smo moves to cilia when Hh is present (see Figure 16-34).
- The NF- $\kappa$ B transcription factor regulates many genes that permit cells to respond to infection and inflammation.
- In unstimulated cells, NF- $\kappa$ B is localized to the cytosol, bound to the inhibitor protein I- $\kappa$ B $\alpha$. In response to many types of extracellular signals, phosphorylation-dependent ubiquitinylation and degradation of I- $\kappa$ B $\alpha$ in proteasomes releases active NF- $\kappa \mathrm{B}$, which translocates to the nucleus (see Figure 16-35a).
- Polyubiquitin chains linked to the activated IL-1 receptor form a scaffold that brings the TAK1 kinase near its substrate, the $\beta$ subunit of the I- $\kappa$ B kinase, and thus allows signals to be transmitted from the receptor to downstream components of the NF- $\kappa$ B pathway (see Figure 16-35b).


### 16.7 Signaling Pathways Controlled by Protein Cleavage: Notch/Delta, SREBP, and Alzheimer's Disease

In this section, we consider signaling pathways activated by protein cleavage in an extracellular space-often at the surface of the cell-generally by members of the matrix metalloprotease (MMP) family and more specifically by the subclass of transmembrane ADAMs ( $a$-disintegrin-and-metalloproteases; a disintegrin is a conserved protein domain that binds integrins and disrupts cell-matrix interactions-see Chapter 20). In the Notch/Delta pathway, for instance, ADAM cleavage of the extracellular part of the

Notch receptor is followed by Notch cleavage within the plasma membrane by a different protease, releasing the cytosolic domain that functions as a transcription factor. This pathway determines the fates of many types of cells during development.

Earlier in the chapter, we saw that multiple growth factors signal through receptor tyrosine kinases. Many such growth factors, including members of the epidermal growth factor (EGF) family, are made as membrane-spanning precursors and can signal adjacent cells by binding to EGF receptors on their surfaces. But cleavage of these proteins by matrix metalloproteases releases the active growth factors into the extracellular medium, allowing them to signal cells much farther away, and even to the releasing cells themselves (autocrine signaling). Since this process involves a form of proteolytic cleavage similar to that which occurs in the Notch/Delta pathway, we consider it here as well. Inappropriate MMP cleavage of a membrane-spanning protein expressed in nerve cells has been implicated in the pathology of Alzheimer's disease, and we discuss this process as well.

Regulated protein cleavage is also used in some intracellular signaling pathways. Thus we conclude our discussion by describing one such pathway: the intramembrane cleavage of a transcription factor precursor within the Golgi membrane in response to low cholesterol levels. This pathway is essential for maintaining the proper balance of cholesterol and phospholipids for constructing cell membranes (see Chapter 7).

## On Binding Delta, the Notch Receptor Is Cleaved, Releasing a Component Transcription Factor

Both the receptor called Notch and its ligand Delta are single-spanning transmembrane proteins found on the cell surface. Notch also has other families of ligands, but the molecular mechanisms of activation are the same with each ligand. The extracellular domain of Delta on the signaling cell binds to Notch on an adjacent responding cell (but not on the same cell), activating Notch so that it undergoes two cleavage events; these events result in release of the Notch cytosolic domain, which functions as a transcription factor. Notch protein is synthesized as a monomeric membrane protein in the endoplasmic reticulum. In the Golgi complex, it undergoes a proteolytic cleavage that generates an extracellular subunit and a transmembrane-cytosolic subunit; the two subunits remain noncovalently associated with each other.

ADAM 10 is a matrix metalloprotease (MMP), a member of a class of metal-containing enzymes that cleave the extracellular segments of target proteins near the extracellular surface of the plasma membrane. ADAM 10 performs the initial cleavage of the Notch extracellular domain, but in the absence of Delta on an adjacent cell, the Notch extracellular domain is folded such that ADAM 10 cannot access the protease cleavage site (Figure 16-36).

Following the binding of Delta to a Notch protein on the responding cell, Delta in the signaling cell undergoes endocytosis. The force accompanying the movement of Delta into the signaling cell stretches the Notch protein, changing

![li3024542480e](li3024542480e.jpg)

FIGURE 16-36 Notch/Delta signaling pathway. In the absence of Delta, the transmembrane subunit of Notch on a responding cell is noncovalently associated with its extracellular subunit; the extracellular domain is folded so that it cannot be cleaved by the cell-surface protease ADAM 10. Binding of Notch to its ligand Delta on an adjacent signaling cell (step $\mathbf{\square}$ is followed by endocytosis of Delta by the signaling cell, stretching the Notch extracellular domain so that ADAM 10 can cleave it (step $\mathbf{\square}$. The released Notch extracellular domain remains bound to Delta and is endocytosed by the signaling cell (step $\mathbf{\square}$. Next
the nicastrin subunit (colored red) of the four-protein $\gamma$-secretase complex binds to the stump generated by ADAM 10 (step $\mathbf{\square}$ ), and then the protease, presenilin 1, catalyzes an intramembrane cleavage that releases the cytosolic segment of Notch (step $\mathbf{\square}$. Following translocation to the nucleus, this Notch segment interacts with several transcription factors to affect expression of genes that in turn influence the determination of cell fate during development (step $\mathbf{\square}$. See D. Seals and S. Courtneidge, 2003, Gene Dev. 17:7, and L. Meloty-Kapella et al., 2012, Dev. Cell 22:1299.
its conformation and allowing access by ADAM 10, which cleaves off the Notch extracellular domain. The Notch extracellular domain remains bound to Delta, is internalized by the signaling cell, and is probably degraded in lysosomes.

The second cleavage of Notch occurs within the hydrophobic membrane-spanning region of Notch and is catalyzed by a four-protein transmembrane complex termed $\gamma$-secretase. This cleavage releases the Notch cytosolic segment, which
immediately translocates to the nucleus, where it affects transcription of target genes. Its effect, like those of other transcription factors activated downstream of other cell-surface receptors, depends on the constellation of epigenetic chromatin marks and the presence of cell-specific transcription factors.

The four-protein $\gamma$-secretase complex contains presenilin 1 (PS1), the actual protease, and three other essential

subunits, aph-1, pen-2, and nicastrin. How peptide bond hydrolysis can occur within a hydrophobic intramembrane environment is not well understood because the molecular structure of presenilin is not known to sufficient resolution (see Figure 16-37b). But the molecular structure of a related archaeal protein, also with nine transmembrane segments, suggests that two aspartate residues, located near each other on the cytosolic side of two transmembrane helices, are essential for catalysis and are surrounded by water; thus proteolytic cleavage probably occurs in a partially aqueous environment.

Studies on cells and mice lacking nicastrin revealed why $\gamma$-secretase can cleave only proteins that have first been cleaved by an ADAM or other matrix metalloprotease. Nicastrin binds to the N-terminal extracellular stump of the membrane protein that is generated by the first protease (Figure 16-36, step 4). Without this stump, nicastrin, and thus the entire $\gamma$-secretase complex, cannot interact with its target protein. We examine the role of ADAM proteins and $\gamma$-secretase in the development of Alzheimer's disease below.

## Matrix Metalloproteases Catalyze Cleavage of Many Signaling Proteins from the Cell Surface

Many signaling molecules are synthesized as transmembrane proteins whose signal domain extends into the extracellular space. Such signaling proteins, like Delta described above, are often biologically active but can signal only by binding to receptors on adjacent cells. However, many growth factors and other protein signals are synthesized as transmembrane precursors whose cleavage releases the soluble, active signaling molecule into the extracellular space. This cleavage is often carried out by ADAMs. The human genome encodes 21 matrix metalloproteases in the ADAM family, but only 12 are known to be catalytically active; the rest may function as disintegrins. Many ADAMs are involved in cleaving the precursors of signaling proteins just outside their transmembrane segment.

$\square$ Medically important examples of the regulated cleavage of precursors of signaling proteins are members of the EGF family, including EGF, HB-EGF, TGF- $\alpha$, NRG1, and NRG2 (see Figure 16-17). Increased activity of one or more ADAMs, which is seen in many cancers, can promote cancer development in three ways. First, heightened ADAM activity can lead to high levels of extracellular EGF-family growth factors that stimulate the secreting cells (autocrine signaling) or adjacent cells (paracrine signaling) to proliferate inappropriately. Second, by destroying components of the extracellular matrix, increased ADAM activity is thought to facilitate metastasis, the movement of tumor cells to other sites in the body (see Chapter 24). Third, following metalloprotease cleavage of the extracellular domain, cleavage by $\gamma$-secretase releases the cytosolic fragment of these precursor proteins, in a manner similar to the release of the Notch intracellular domain. Several of these protein fragments migrate into the nucleus, where, like the Notch intracellular domain, they stimulate the transcription of growth-promoting genes.

ADAM proteases also are an important factor in heart disease. As we learned in the last chapter, epinephrine (adrenaline) stimulation of $\beta$-adrenergic receptors in heart muscle causes glycogenolysis and an increase in the rate of muscle contraction. Prolonged treatment of heart muscle cells with epinephrine, however, leads to activation of an ADAM by an unknown mechanism. This matrix metalloprotease cleaves the transmembrane precursor of HB-EGF. The released HBEGF then binds to EGF receptors on heart muscle cells and stimulates their inappropriate growth. This excessive proliferation can lead to an enlarged but weakened heart-a condition known as cardiac hypertrophy, which may cause early death.

## Inappropriate Cleavage of Amyloid Precursor Protein Can Lead to Alzheimer's Disease

Alzheimer's disease is another disorder marked by the inappropriate activity of matrix metalloproteases. A major pathological change associated with Alzheimer's disease is accumulation in the brain of amyloid plaques containing aggregates of a small 42 -amino-acid peptide termed $\mathrm{A} \beta_{42}$. This peptide is derived by proteolytic cleavage of amyloid precursor protein (APP), a transmembrane cell-surface protein of still mysterious function expressed by neurons.

Like Notch protein, APP undergoes one extracellular cleavage and one intramembrane cleavage (Figure 16-37a), but this can happen in two ways. In the first, APP is cleaved at a site in the extracellular domain by ADAM 10 (often called $\alpha$-secretase), and then by $\gamma$-secretase at a single intramembrane site, releasing the APP cytosolic domain and generating a 26 -amino-acid, partially membrane-embedded peptide that apparently does no harm. In contrast, if the extracellular domain is first cleaved at a different site by a different enzyme, $\beta$-secretase, and then by $\gamma$-secretase at the same intermembrane site, a 42 -amino-acid peptide, termed $\mathrm{A} \beta_{42}$, is generated. $\mathrm{A} \beta_{42}$ spontaneously forms oligomers and then the larger amyloid plaques found in the brain of patients with Alzheimer's disease.

APP was recognized as a major player in Alzheimer's disease through a genetic analysis of the small percentage of patients with a family history of the disease. Many had mutations in the APP protein, and intriguingly, these mutations are clustered around the cleavage sites of $\alpha-, \beta$-, or $\gamma$-secretase depicted in Figure 16-37a. Other cases of familial Alzheimer's disease involve missense mutations in presenilin 1 , the catalytic subunit of $\gamma$-secretase, that enhance the formation of the $\mathrm{A} \beta_{42}$ peptide, leading to plaque formation and eventually to the death of neurons.

## Regulated Intramembrane Proteolysis of SREBPs Releases a Transcription Factor That Acts to Maintain Phospholipid and Cholesterol Levels

Although this chapter is focused on signaling pathways initiated by extracellular molecules, signaling pathways that sense the levels of internal molecules and respond accordingly

![li3034542480e](li3034542480e.jpg)
sometimes share principles of molecular regulation and even mechanisms with pathways initiated from outside the cell. One such case is the control of cellular membrane lipids. A cell would soon face a crisis if it did not have enough phospholipids to make adequate amounts of membranes or had so much cholesterol that large crystals formed and damaged cellular structures (see Chapter 7). Cells sense the relative amounts of cholesterol and phospholipids in their membranes; they respond by adjusting the rates of cholesterol biosynthesis and import so that the cholesterol:phospholipid ratio is kept within a narrow desirable range. Regulated intramembrane proteolysis plays an important role in this cellular response to altered cholesterol levels.

As we learned in Chapter 14, low-density lipoprotein (LDL) is rich in cholesterol and functions in transporting this lipid through the aqueous circulatory system (see Figure 14-27). Both enzymes in the cholesterol biosynthetic pathway (see Figure 7-26) and cellular levels of the LDL receptors that mediate cellular uptake of LDL are downregulated when cellular cholesterol levels are adequate. Because LDL is imported into cells via receptor-mediated

FIGURE 16-37 Proteolytic cleavage of APP and Alzheimer's disease. (a, left) Sequential proteolytic cleavage by $\alpha$-secretase (ADAM 10 or ADAM 17) (step $\mathbf{\square}$ ) and $\gamma$-secretase (step $\mathbf{\square}$ ) produces an innocuous membrane-embedded peptide of 26 amino acids. (a, right) Cleavage in the extracellular domain by $\beta$-secretase (step $\mathbf{\square}$ ) followed by cleavage within the membrane by $\gamma$-secretase (step $\mathbf{\square}$ ) generates the 42-amino-acid $A \beta_{42}$ peptide, which spontaneously forms oligomers, and then the large amyloid plaques found in the brains of patients with Alzheimer's disease (inset). In both pathways, the cytosolic segment of APP is released into the cytosol, but its function is not known. See S. Lichtenthaler and C. Haass, 2004, J. Clin. Invest. 113:1384, and V. Wilquet and B. De Strooper, 2004, Curr. Opin. Neurobiol. 14:582. (b) Three-dimensional structure of human $\gamma$-secretase at 0.45 nm resolution. It contains a total of 19 transmembrane segments and a large extracellular domain from nicastrin. The protease catalytic site in PS1 is located near the cytosolic surface. See P. Lu et al., 2014, Nature 512:166. [Part (a) inset copyright © Pr. J. J. Hauw/ISM/Phototake. Part (b) data from L. Sun et al., 2015, Proc. Natl. Acad. Sci. USA 112:6003, PDB ID 4uis.]
endocytosis (see Figure 14-29), a decrease in the number of LDL receptors leads to reduced cellular import of cholesterol.

Both cholesterol biosynthesis and cholesterol import are regulated at the level of gene transcription. For example, when growing cultured cells that need new membrane for sustained division are incubated with an external source of cholesterol, such as LDL, the level and the activity of HMG-CoA reductase, the rate-controlling enzyme in cholesterol biosynthesis, are suppressed. In contrast, the activity of acyl:cholesterol acyl transferase (ACAT), the enzyme that converts cholesterol into the esterified storage form, is increased. Thus energy is not wasted making unnecessary additional cholesterol, and cholesterol homeostasis is achieved.

Genes whose expression is controlled by the level of sterols such as cholesterol often contain one or more 10-basepair sterol regulatory elements (SREs), or SRE half-sites, in their promoters. (These SREs differ from the serum response elements that control many early response genes, discussed in Section 16.4.) The interaction of cholesterol-dependent transcription factors called SRE-binding proteins (SREBPs) with these response elements modulates the expression of

the target genes. How do cells sense how much cholesterol they have, and how is this "signal" used to control the level of SREBPs in the nucleus and thus gene expression? The SREBP-mediated pathway begins in the membranes of the endoplasmic reticulum (ER) and includes at least two other proteins besides SREBP.

When cells have adequate concentrations of cholesterol, SREBP is found in the ER membrane complexed with SCAP (SREBP cleavage-activating protein), insig-1 (or its close homolog insig-2), and perhaps other proteins (Figure 16-38a). SREBP has three distinct domains: an N-terminal cytosolic domain, containing a basic helix-loophelix (bHLH) DNA-binding motif (see Figure 9-30d), that functions as a transcription factor when cleaved from the rest of SREBP; a central membrane-anchoring domain containing two transmembrane $\alpha$ helices; and a C-terminal cytosolic regulatory domain. SCAP has eight transmembrane $\alpha$ helices and a large C-terminal cytosolic domain that interacts with the regulatory domain of SREBP. Five of the transmembrane $\alpha$ helices in SCAP form a sterol-sensing domain similar to that in HMG-CoA reductase (Figure 16-38a; see

Section 7.3). When the sterol-sensing domain in SCAP is bound to cholesterol, the protein also binds to insig-1(2). When insig-1(2) is tightly bound to the SCAP-cholesterol complex, it blocks the binding of SCAP to the Sec24 coat protein subunit of COPII vesicles, thereby preventing incorporation of the SCAP-SREBP complex into ER-to-Golgi transport vesicles (see Chapter 14). This occurs when cholesterol concentrations in the ER membrane exceed 5 percent of total ER membrane lipids. Thus the cholesterol-dependent binding of insig-1(2) to the SCAP-cholesterol-SREBP complex traps that complex in the ER.

Cholesterol bound to SCAP is released when cellular cholesterol levels drop to less than 5 percent of ER lipids, a value that reflects total cellular cholesterol levels. Consequently, insig-1(2) no longer binds to the cholesterol-free SCAP, and the SCAP-SREBP complex moves from the ER to the Golgi complex via COPII vesicles (Figure 16-38b). SREBP is cleaved sequentially at two sites by two proteases in the Golgi membrane, S1P and S2P; the second cleavage represents an additional example of regulated intramembrane proteolysis. This second cleavage at site 2 releases the N -terminal
![li3044542480e](li3044542480e.jpg)

FIGURE 16-38 Cholesterol-sensitive control of SREBP activation. The cellular pool of cholesterol is monitored by the combined action of insig-1(2) and SCAP, both transmembrane proteins located in the ER membrane. Membrane-spanning helices 2-6 of SCAP (orange with black lines) form a sterol-sensing domain, and a C-terminal segment binds to SREBP. (a) When cholesterol levels are high enough that ER cholesterol exceeds 5 percent of total ER lipids, cholesterol binds to the sterol-sensing domain in SCAP, triggering a conformational change that enables the N -terminal SCAP domain to bind to insig-1(2), anchoring the SCAP-SREBP complex in the ER membrane. (b) At low cholesterol levels, cholesterol dissociates from the SCAP sterol-sensing domain,
triggering a reverse conformational change that dissociates SCAP from insig-1(2) and enables SCAP to bind to Sec24, a subunit of the COPII complex (see Figure 14-8). This binding initiates movement of the SCAP-SREBP complex to the Golgi complex by vesicular transport. In the Golgi, the sequential cleavage of SREBP by the site 1 and site 2 proteases (S1P, S2P) releases the N-terminal bHLH domain of SREBP, which translocates to the nucleus, and SCAP, which recycles to the ER. In the nucleus, the released SREBP domain, called nuclear SREBP (nSREBP), controls the transcription of genes containing sterol regulatory elements (SREs) in their promoters. See A. Radhakrishnan, 2008, Cell Metab. 8:451, and M. Brown and J. Goldstein, 2009, J. Lipid Res. 50:S15.

bHLH-containing domain into the cytosol. This fragment, called $n S R E B P$ (nuclear SREBP), is rapidly translocated into the nucleus. There it activates transcription of genes containing sterol regulatory elements (SREs) in their promoters, such as those encoding the LDL receptor and HMG-CoA reductase. Thus a reduction in cellular cholesterol, by activating the insig-1(2)/SCAP/SREBP pathway, triggers expression of genes encoding proteins that both import cholesterol into the cell (the LDL receptor) and synthesize cholesterol from small precursor molecules (HMG-CoA reductase).

After cleavage of SREBP in the Golgi, SCAP apparently recycles back to the ER, where it can interact with insig-1(2) and another intact SREBP molecule. High-level transcription of SRE-controlled genes requires the ongoing generation of new nSREBP because it is degraded fairly rapidly by the ubiquitin-mediated proteasomal pathway (see Chapter 3). The rapid generation and degradation of nSREBP help cells respond quickly to changes in levels of intracellular cholesterol.

Under some circumstances (e.g., during cell growth), cells need an increased supply of all the essential membrane lipids and their fatty acid precursors (which requires coordinate regulation). To make steroid hormones, cells sometimes need greater amounts of some lipids (such as cholesterol) than others, such as phospholipids (which requires differential regulation). How is such differential production achieved? Mammals express three known isoforms of SREBP: SREBP-1a and SREBP-1c, which are generated from alternatively spliced RNAs produced from the same gene, and SREBP-2, which is encoded by a different gene. Together, these intramembrane cleavage-activated transcription factors control expression of proteins that regulate availability not only of cholesterol, but also of fatty acids and the triglycerides and phospholipids made from fatty acids. In mammalian cells, SREBP-1a and SREBP-1c exert a greater influence on fatty acid metabolism than on cholesterol metabolism, whereas the reverse is the case for SREBP-2.

## KEY CONCEPTS OF SECTION 16.7

## Signaling Pathways Controlled by Protein Cleavage: Notch/Delta, SREBP, and Alzheimer's Disease

- Many important growth factors and other signaling proteins such as EGFs are synthesized as transmembrane proteins; regulated cleavage of the precursor near the plasma membrane by members of the matrix metalloprotease (MMP) family releases the active molecule into the extracellular space to signal distant cells.
- On binding to its ligand, Delta, on the surface of an adjacent cell, the receptor Notch protein undergoes two proteolytic cleavages (see Figure 16-36). The released Notch cytosolic segment then translocates into the nucleus and modulates transcription of target genes critical in determining cell fate during development.
- Cleavage of membrane-bound precursors of members of the EGF family of signaling molecules is catalyzed by ADAM metalloproteases. Inappropriate cleavage of these precursors can result in abnormal cell proliferation, potentially leading to cancer, cardiac hypertrophy, and other diseases.
- $\gamma$-Secretase, which catalyzes the regulated intramembrane proteolysis of Notch, also participates in the cleavage of amyloid precursor protein (APP) into a peptide that forms plaques characteristic of Alzheimer's disease (see Figure 16-37).
- In the insig-1(2)/SCAP/SREBP pathway, the active nSREBP transcription factor is released from the Golgi membrane by intramembrane proteolysis when cellular cholesterol is low (see Figure 16-38). It then stimulates the expression of genes encoding proteins that function in cholesterol biosynthesis (e.g., HMG-CoA reductase) and cellular import of cholesterol (e.g., LDL receptor). When cholesterol is high, SREBP is retained in the ER membrane complexed with insig-1(2) and SCAP (see Figure 16-38).


### 16.8 Integration of Cellular Responses to Multiple Signaling Pathways: Insulin Action

In the introduction to Chapter 15, we noted that the same hormone often acts on several types of body cells to coordinate specific physiological responses. And each cell in the body has multiple types of hormone receptors on its surface and in its cytosol; different hormones binding to these receptors can induce similar or dissimilar cellular responses. In this section, we consider how multiple hormones and signal transduction pathways interact, focusing on one of the most important physiological control systems: regulation of the body's needs for the metabolites glucose and fatty acids. Defects in these pathways lead to major diseases, including diabetes and cardiovascular disease; obesity itself can lead to these and other diseases, with dire consequences for the individual and increasingly for public health.

Cellular responses to changes in other nutrients, which are largely reflected in alterations in gene expression, are covered in Chapter 9.

## Insulin and Glucagon Work Together to Maintain a Stable Blood Glucose Level

During normal daily living, the maintenance of normal blood glucose concentrations depends on the balance between two peptide hormones, insulin and glucagon, which are made in distinct pancreatic islet cells and elicit different cellular responses. Insulin, which lowers blood glucose, contains two polypeptide chains linked by disulfide bonds and is synthesized by the $\beta$ cells in the islets (see

Figures 14-23 and 14-24). Glucagon, a monomeric peptide, is produced by the $\alpha$ islet cells and raises blood glucose. The availability of blood glucose is regulated during periods of abundance (following a meal) or scarcity (following fasting) by the adjustment of insulin and glucagon concentrations in the blood.

Our focus here will be on the key hormone insulin, which acts in several ways to reduce the level of blood glucose:

- Within seconds, insulin induces an increase in the uptake of glucose from the blood into muscle and fat cells, primarily by increasing the number of GLUT4 glucose transporters on the plasma membrane (see Figure 16-40 below).
- Within seconds to minutes, insulin stimulates glycogen synthesis from glucose in the liver.
- Over a longer time frame, insulin acts on the liver to inhibit synthesis of enzymes that catalyze the synthesis of glucose from smaller metabolites, a process termed gluconeogenesis.
- Insulin enhances the formation of adipocytes from progenitor cells, increasing the body's storage of fatty acids as triglycerides.
- Insulin acts on the nearby $\alpha$ cells in the pancreatic islets to inhibit glucagon synthesis.

A lowering of blood glucose stimulates glucagon release from pancreatic $\alpha$ cells. Like the epinephrine receptor, the glucagon receptor, found primarily on liver cells, is coupled to the $\mathrm{G}_{\alpha \mathrm{s}}$ protein, whose effector protein is adenylyl cyclase. The binding of glucagon to this receptor induces a rise in cAMP, leading to activation of protein kinase A, which inhibits glycogen synthesis and promotes glycogenolysis, yielding glucose 1-phosphate (see Figures 15-28a and 15-35b). Liver cells convert glucose 1-phosphate into glucose, which is released into the blood, thus raising blood glucose back toward its normal fasting level.

## A Rise in Blood Glucose Triggers Insulin Secretion from the $\beta$ Islet Cells

After a meal, when blood glucose rises above its normal level of 5 mM , the pancreatic $\beta$ cells respond to the rise in glucose (and the concurrent rise in amino acids) by releasing insulin into the blood (Figure 16-39). We saw in Chapter 14 that these cells store insulin in a dehydrated, almost crystalline form in secretory vesicles; as with all regulated secretory pathways, secretion is triggered by a rise in cytosolic $\mathrm{Ca}^{2+}$. Insulin secretion is triggered by a rise in extracellular glucose, which causes a proportionate increase in the rate of glucose entry into the cells and a corresponding increase in the rate of glycolysis. The resulting rise in the concentration of cytosolic ATP causes closing of an ion channel unique to the $\beta$ cells-an ATP-gated $\mathrm{K}^{+}$channel-reducing the efflux of $\mathrm{K}^{+}$ions from the cell. The resulting depolarization of the plasma membrane triggers the opening of voltage-sensitive
![li3054542480e](li3054542480e.jpg)

FIGURE 16-39 Secretion of insulin in response to a rise in blood glucose. The entry of glucose into pancreatic $\beta$ cells is mediated by the GLUT2 glucose transporter (step $\mathbf{\square}$ ). Because the $K_{m}$ for glucose of GLUT2 is 20 mM , a rise in extracellular glucose from 5 mM , characteristic of the fasting state, causes a proportional increase in the rate of glucose entry (see Figure 11-4). The conversion of glucose into pyruvate is thus accelerated, resulting in an increase in the concentration of ATP in the cytosol (step $\mathbf{\square}$ ). The binding of ATP to ATP-sensitive $\mathrm{K}^{+}$channels in the $\beta$ cells closes those channels (step $\mathbf{\square}$ ), thus reducing the efflux of $\mathrm{K}^{+}$ions from the cell. The resulting small depolarization of the plasma membrane (step $\mathbf{\square}$ ) triggers the opening of voltage-sensitive $\mathrm{Ca}^{2+}$ channels (step $\mathbf{\square}$ ). The influx of $\mathrm{Ca}^{2+}$ ions raises the cytosolic $\mathrm{Ca}^{2+}$ concentration, triggering the fusion of insulin-containing secretory vesicles with the plasma membrane and the secretion of insulin (step $\mathbf{\square}$ ). See J. Q. Henquin, 2000, Diabetes 49:1751.
$\mathrm{Ca}^{2+}$ channels, an increase in cytosolic $\mathrm{Ca}^{2+}$, and insulin secretion.

## In Fat and Muscle Cells, Insulin Triggers Fusion of Intracellular Vesicles Containing the GLUT4 Glucose Transporter to the Plasma Membrane

The released insulin circulates in the blood and binds to insulin receptors, which are present on many different kinds of cells, including muscle and adipocyte cells. The insulin receptor, a receptor tyrosine kinase, activates several signal transduction pathways, including the one leading to the activation of protein kinase B (PKB; see Figure 16-29). In this case, the main action of the PKB signaling pathway-an increase in uptake of glucose from the blood-is manifest within minutes. Since glucose uptake is the rate-limiting step in glucose utilization, this action results in rapid lowering of the blood glucose level.

Like those of most body cells, the plasma membranes of fat and muscle cells contain the GLUT1 glucose transporter, which allows the cell to import sufficient glucose

![li3064542480e](li3064542480e.jpg)

EXPERIMENTAL FIGURE 16-40 Insulin stimulation of fat cells induces translocation of GLUT4 from intracellular vesicles to the plasma membrane. (a) Cultured adipose cells engineered to express a chimeric protein comprising GLUT4 with a green fluorescent protein (GFP) fused to its C-terminus were visualized with a confocal fluorescence microscope. In the absence of insulin, virtually all of the GLUT4 is in intracellular membranes. Treatment with insulin triggers fusion of the GLUT4containing membranes with the plasma membrane. Arrows highlight GLUT4 present at the plasma membrane; $N$ indicates the position of the nucleus. (b) In fat and muscle cells, insulin signaling acts in multiple steps to increase the level of GLUT4 at the plasma membrane. In resting cells, the majority of the GLUT4 protein is localized to specialized GLUT4 storage vesicles (GSVs), tethered to Golgi matrix proteins by the TUG protein. Binding of insulin to the insulin receptor leads to activation of a protease (step 1) that cleaves the TUG protein, releasing GLUT4-containing vesicles (step 2), which then move along microtubules, powered by a kinesin motor (see Chapter 18), to the cell surface. Insulin also activates PKB (step 3, see Figure 16-29). PKB then phosphorylates the Rab GAP protein AS160 (step 4), inhibiting its ability to accelerate GTP hydrolysis by Rab8, Rab10, and Rab14. These Rab proteins accumulate in their active GTP-bound states (step 5) and allow the GLUT4 storage vesicles to move along microtubules to the cell surface (steps 6 and 10). Finally, these GSVs fuse with the plasma membrane (step 6). This step is catalyzed by the exocyst and also by
another monomeric GTP-binding protein, RALA. PKB stimulates this membrane fusion event by phosphorylating and thus inactivating the RALA GAP protein RGC (step 7), allowing RALA to accumulate in its active GTP-bound state (step 8). The resultant increase in plasma membrane GLUT4 allows the cell to incorporate glucose from the extracellular fluids at a rate about 10 times that of unstimulated cells (step 9). Following removal of insulin, the plasma membrane GLUT4 is internalized by endocytosis (step 11) and eventually transported to GSVs (step 12). Many other proteins, not shown here, participate in these signaling and vesicle budding and fusion events. See J. Bogan, 2012, Annu. Rev. Biochem. 81:507, D. Leto and A. Saltiel, 2012, Nat. Rev. Mol. Cell Biol. 13:383, and J. Belman et al., 2014, Rev. Endocr. Metab. Disord. 15:55. (Part (a) C. Yu et al., 2007, J. Biol. Chem 282:7710; ©2007 American Society for Biochemistry and Molecular Biology.)

for its basal metabolic needs. Fat and muscle cells also express large amounts of the insulin-responsive glucose transporter GLUT4; in resting (unstimulated) cells, virtually all of the GLUT4 is localized to small vesicles in the cytosol (Figure 16-40). While some GLUT4 is in endosomes, most is in a unique small organelle termed the GLUT4 storage vesicle (GSV). In these vesicles, much of the GLUT4 is tethered to the Golgi matrix, a network of coiled-coil proteins surrounding the Golgi complex, by a protein termed TUG.

In addition to activating the PI-3 kinase/PKB pathway, the insulin receptor phosphorylates several other target proteins. Together, these signals cause the movement of GLUT4-containing vesicles to the plasma membrane and then the fusion of these vesicles with the plasma membrane. The resulting immediate tenfold increase in the number of GLUT4 molecules on the cell surface increases glucose influx proportionally, thus lowering blood glucose (Figure 16-40b):

- Insulin triggers, by a signaling pathway that is only now being completely identified, activation of a protease that catalyzes a site-specific endoproteolytic cleavage of TUG, separating the N -terminal GLUT4-binding segment from the rest of the protein that is anchored to the Golgi matrix. This cleavage allows the GLUT4 vesicles to move to the plasma membrane.
- Recall that certain monomeric GTP-binding proteins are essential for the budding of intracellular transport vesicles (e.g., Sar proteins; see Figures 14-6 and 14-8); others, the Rabs, are essential for vesicle fusion (see Figure 14-10). PKB phosphorylates, and by so doing inactivates, two GAP proteins termed AS160 and RGC. In the basal unstimulated state, these GAPs inhibit Rab function by enhancing their rates of GTP hydrolysis, thus keeping the GLUT4 storage vesicles from moving to and fusing with the plasma membrane. Inhibition of these GAPs allows these monomeric GTP-binding proteins in fat and muscle cells to accumulate in their active GTP-bound state. These proteins catalyze multiple steps in the GLUT4 pathway, including transport of the GLUT4 storage vesicles along microtubules to the cell surface and, together with the exocyst (see Chapter 14), fusion of these vesicles with the plasma membrane (see Figure 16-40b).

As the blood glucose level drops, insulin secretion and insulin blood levels drop, and insulin receptors are no longer activated as strongly. In fat and muscle cells, plasmamembrane GLUT4 becomes internalized by endocytosis and stored in intracellular membranes, lowering the level of cellsurface GLUT4 and thus of glucose import.

## Insulin Inhibits Glucose Synthesis and Enhances Storage of Glucose as Glycogen

Within minutes, insulin stimulation of muscle cells enhances the conversion of glucose to glycogen, and PKB, activated downstream of the insulin receptor, plays a crucial role in
this process as well. Active PKB phosphorylates GSK3 (the same enzyme that functions in the Wnt and Hh pathways). In resting (non-insulin-stimulated) cells, GSK3 phosphorylates glycogen synthase and thus inhibits its activity. In contrast, in insulin-treated muscle, GSK3 is phosphorylated by PKB and cannot phosphorylate glycogen synthase; thus insulin-stimulated activation of PKB results in net short-term activation of glycogen synthase and glycogen synthesis.

Insulin also acts on hepatocytes (liver cells) to inhibit glucose synthesis from smaller molecules (gluconeogenesis), such as lactate, pyruvate, and acetate (see Chapter 12) and to enhance glycogen synthesis from glucose. Many of these effects are manifest at the level of gene transcription because insulin signaling reduces the expression of genes whose encoded enzymes simulate synthesis of glucose from small metabolites. The net effect of all these actions is to lower blood glucose to the fasting concentration of about 5 mM while storing the excess glucose intracellularly as glycogen for future use.

If the blood glucose level falls below about 5 mM-for example, due to sudden muscular activity-reduced insulin secretion from pancreatic $\beta$ cells induces pancreatic $\alpha$ cells to increase their secretion of glucagon into the blood and quickly trigger an increase in blood glucose levels.

$\square$ Unfortunately, these intricate and powerful control systems sometimes fail, causing serious, even lifethreatening, disease, mainly diabetes mellitus. In diabetes, the regulation of blood glucose is impaired, leading to persistent elevated blood glucose concentrations (hyperglycemia) that, if left untreated, lead to major complications, including blindness, kidney failure, and limb amputations. Type 1 diabetes mellitus, common in children and young adults, is caused by an autoimmune process that destroys the insulinproducing $\beta$ cells in the pancreas. Sometimes called insulindependent diabetes, this form of the disease is generally responsive to regulated lifelong insulin injections and constant monitoring of blood glucose levels.

Most adults in developed countries with diabetes mellitus have type 2, sometimes called non-insulin-independent diabetes; this condition results from a decrease in the ability of muscle, fat, and liver cells to respond to insulin and from a loss of insulin-producing cells as the body tries to compensate for an elevated glucose level by overproducing insulin. While the underlying causes of this form of the disease are not well understood, obesity is correlated with a huge increase in the incidence of diabetes. As we see in the next section, obesity also contributes to the malfunction of adipocytes, the cells that store fatty acids as triglycerides. The resulting accumulation of lipids (particularly diacylglycerols and sphingolipids) in muscle and liver impairs insulin action in these tissues. Further identification of the signaling pathways that control energy metabolism is expected to provide insight into the pathophysiology of diabetes, hopefully leading to new methods for its prevention and treatment.

## Multiple Signal Transduction Pathways Interact to Regulate Adipocyte Differentiation Through PPAR $\gamma$, the Master Transcriptional Regulator

Insulin is also a major inducer of the formation of white adipocytes, commonly called "fat cells." Adipocytes are the body's major fat storage depots; mature adipocytes contain a few triglyceride droplets that occupy the bulk of the cell. Adipocytes are also endocrine cells and secrete several signaling proteins that affect the metabolic functions of muscle, liver, and other organs. Adipocytes are the one type of cell in the body that can increase in number almost without limit. Readers in every country do not need to be reminded that obesity is a growing public health problem, and that it is a major risk factor not only for diabetes but also for cardiovascular diseases, such as heart attacks and stroke, and for certain cancers.

As we discuss in Chapter 21, several types of stem cells in vertebrates are used to generate specific types of differentiated cells. The mesenchymal stem cell, which resides in the bone marrow and other organs, gives rise to progenitor cells that in turn can form either adipocytes, cartilageproducing cells, or bone-forming osteoblasts. The adipocyte progenitor, called the preadipocyte, has lost the potential to differentiate into other cell types. When treated with insulin and other hormones, preadipocytes undergo terminal differentiation; they acquire the proteins that are necessary for lipid transport and synthesis, insulin responsiveness, and the secretion of adipocyte-specific proteins. Several lines of cultured preadipocytes can differentiate into adipocytes and express adipocyte-specific mRNAs and proteins, such as enzymes required for triglyceride synthesis.

The transcription factor PPAR $\gamma$, a member of the nuclear hormone receptor family, is the master transcriptional regulator of adipocyte differentiation. As evidence, recombinant expression of PPAR $\gamma$ in many fibroblast lines has been found sufficient to trigger the differentiation of these cells into adipocytes. Conversely, knocking down the gene for PPAR $\gamma$ in preadipocytes prevents their differentiation into adipocytes. Most hormones, such as insulin, that promote adipogenesis do so at least in part by activating expression of PPAR $\gamma$. PPAR $\gamma$, in turn, binds to the promoters of most adipocyte-specific genes, including genes encoding proteins needed in the insulin-signaling pathway, such as the insulin receptor and GLUT4, and induces their expression. Like other members of the nuclear hormone receptor family, such as steroid hormone receptors (see Chapter 9), which become activated when they bind their ligand, PPAR $\gamma$ is also thought to bind a ligand, possibly an oxidized derivative of a fatty acid.

Another transcription factor, $\mathrm{C} / \mathrm{EBP} \alpha$, is induced during adipocyte differentiation and also directly induces many adipocyte genes. Importantly, $\mathrm{C} / \mathrm{EBP} \alpha$ induces expression of the PPAR $\gamma$ gene, and PPAR $\gamma$ induces expression of $\mathrm{C} / \mathrm{EBP} \alpha$, leading to a rapid increase in both proteins during the first
two days of differentiation. Together, PPAR $\gamma$ and $\mathrm{C} / \mathrm{EBP} \alpha$ induce expression of all genes required for the differentiation of preadipocytes into mature fat cells.

Many signaling proteins, such as Wnt and TGF- $\beta$, oppose the action of insulin and prevent preadipocyte differentiation into adipocytes. As Figure 16-41 shows, transcription factors activated by receptors for these hormones prevent expression of the PPAR $\gamma$ gene, in part by blocking the ability of $\mathrm{C} / \mathrm{EBP} \alpha$ to induce PPAR $\gamma$ gene expression. Thus multiple extracellular signals act in concert to regulate adipogenesis, and the signal transduction pathways activated by them intersect at the regulation of expression of one key "master" gene, encoding PPAR $\gamma$.

## Inflammatory Hormones Cause Derangement of Adipose Cell Function in Obesity

The problem with obesity is not just the weight of the added adipose cells; the metabolism of these cells becomes abnormal, and insulin signaling is reduced. In many individuals, this leads to development of type 2 diabetes. Much of the problem comes from macrophages and other immune-system cells that invade and populate obese adipose tissue, probably attracted by the necrotic adipose cells that accumulate and the greasy lipid droplets they release. These macrophages secrete several so-called inflammatory hormones that profoundly affect the metabolism of the nearby adipose cells. Two of these hormones, TNF $\alpha$ and IL-1, bind to their respective receptors on adipose cells and induce activation of the transcription factor NF- $\kappa \mathrm{B}$ (see Figure 16-35a). NF- $\kappa \mathrm{B}$, in turn, induces the expression of several proteins that interfere with insulin signaling, including SOCS proteins; as with cytokine receptors (see Figure 16-14b), SOCS proteins bind to phosphorylated tyrosines in the insulin receptor cytosolic domain, inhibit kinase activity, and lead to degradation of the insulin receptor and associated signaling proteins.

Additionally, NF- $\kappa$ B represses the expression of several essential components of the insulin-signaling pathway, including GLUT4 and PKB. NF- $\kappa$ B also induces the expression of inflammatory cytokines such as TNF $\alpha$, which in turn act in an autocrine or paracrine manner on adipocytes and exacerbate the development of insulin resistance. Among the biochemical alterations in adipocyte metabolism induced by these "inflammatory" hormones is the hydrolysis of triglycerides into free fatty acids and glycerol; excess fatty acids are released into the blood, whence they can accumulate in and induce insulin resistance in liver and muscle. This in turn leads to an increase in blood glucose, and to overproduction of insulin by the $\beta$ islet cells in an attempt to overcome the increasing insulin insensitivity in fat, muscle, and liver cells-all leading to what has become a major public health problem in all developed countries in the twenty-first century, the type 2 diabetes epidemic.

![li3074542480e](li3074542480e.jpg)

FIGURE 16-41 Multiple signal transduction pathways interact to regulate adipocyte differentiation. The transcription factor PPAR $\gamma$ (purple) is the master regulator of adipocyte differentiation; together with $\mathrm{C} / \mathrm{EBP} \alpha$, it induces expression of all genes required for differentiation of preadipocytes into mature fat cells. Both PPAR $\gamma$ and $\mathrm{C} / \mathrm{EBP} \alpha$ are induced early in adipogenesis; each of them enhances the transcription of the gene encoding the other (an arrow at the end of a line means enhancement of expression of target genes), leading to a rapid increase in expression of both proteins during the first two days of differentiation. Signals from hormones such as insulin and from growth factors such as Wnt and TGF- $\beta$ that activate or repress adipogenesis are integrated in the nucleus by transcription factors that regulatedirectly or indirectly—expression of the PPAR $\gamma$ and $\mathrm{C} / \mathrm{EBP} \alpha$ genes. (A T shape at the end of a line indicates inhibition of expression of the target gene.) (a) Insulin activates adipogenesis by several pathways leading to activation of PPAR $\gamma$ expression, two of which are depicted here. Activation of protein kinase B (PKB) downstream of the IGF-1
and insulin receptor tyrosine kinases leads to repression of Necdin expression; Necdin, by modulating other transcription factors, would otherwise repress expression of the PPAR $\gamma$ gene. PKB also phosphorylates, and thus inactivates, the transcription factor GATA2, which when nonphosphorylated binds to the $\mathrm{C} / \mathrm{EBP} \alpha$ protein and prevents it from activating expression of the PPAR $\gamma$ gene. By inhibiting two repressors of the PPAR $\gamma$ gene, insulin thus stimulates PPAR $\gamma$ expression. (b) Wnt and TGF- $\beta$ inhibit adipogenesis by reducing expression of the PPAR $\gamma$ gene. Wnt signaling triggers release of $\beta$-catenin from a cytoplasmic complex, and free $\beta$-catenin binds the transcription factor TCF (see Figure 16-30). Active TCF blocks expression of the PPAR $\gamma$ and $\mathrm{C} / \mathrm{EBP} \alpha$ genes, probably by binding to their regulatory sequences. (c) Smad3, activated by phosphorylation following TGF- $\beta$ binding to the types I and II TGF- $\beta$ receptors, binds to the $\mathrm{C} / \mathrm{EBP} \alpha$ protein and prevents it from activating expression of the PPAR $\gamma$ gene. See E. Rosen and O. MacDougald, 2006, Nat. Rev. Mol. Cell Biol. 7:885.

## KEY CONCEPTS OF SECTION 16.8

## Integration of Cellular Responses to Multiple Signaling Pathways: Insulin Action

- A rise in blood glucose stimulates the release of insulin from pancreatic $\beta$ cells (see Figure 16-39).
- Subsequent binding of insulin to its receptor on muscle cells and adipocytes leads to the activation of protein kinase B, which activates several signal transduction pathways. Together with PKB-independent pathways, these pathways lead to translocation of the GLUT4 glucose transporter from intracellular vesicles to the plasma membrane and thus to an increase in glucose uptake and (in muscle) glycogen synthesis, resulting in a decrease in blood glucose (see Figure 16-40).
- A lowering of blood glucose stimulates glucagon release from pancreatic $\alpha$ cells. Binding of glucagon to its G proteincoupled receptor on liver cells promotes glycogenolysis by the cAMP-triggered kinase cascade (similar to epinephrine stimulation under stress conditions) and an increase in blood glucose (see Figures 15-28a and 15-35b).
- PPAR $\gamma$, a member of the nuclear receptor family, is the master transcriptional regulator of adipocyte differentiation.
- Extracellular hormones such as insulin that promote adipocyte differentiation induce signal transduction pathways that lead to enhanced production of PPAR $\gamma$. Conversely, signaling proteins such as Wnt and TGF- $\beta$ that prevent preadipocyte differentiation activate signaling pathways that prevent expression of the PPAR $\gamma$ gene (see Figure 16-41).


## LaunchPad

macmillan learning
Visit LaunchPad to access study tools and to learn more about the content in this chapter.

- Perspectives for the Future
- Analyze the Data
- Additional study tools, including videos, animations, and quizzes


## Key Terms

activation loop 730
adapter protein 741
constitutive 723
cytokines 727
diabetes mellitus 769
erythropoietin (Epo) 727
Hedgehog (Hh) 755
HER family 735
insig-1(2)/SCAP/SREBP pathway 766
insulin 766
JAK/STAT pathway 727
kinase cascade 739
MAP kinase 739
matrix metalloprotease
(MMP) family 761
NF-кB pathway 752
Notch/Delta pathway 761
nuclear-localization signal
(NLS) 724
phosphoinositides 748
PI-3 kinase pathway 749
PPAR $\gamma 770$
presenilin 1762
primary cilium 757
protein kinase B (PKB) 750
PTEN phosphatase 751
Ras protein 739
receptor tyrosine kinases (RTKs) 726
regulated intramembrane proteolysis 764
scaffold proteins 748
SH2 domains 730
Smads 722
SRE-binding proteins (SREBPs) 764
transforming growth factor $\beta$ (TGF- $\beta$ ) 722
Wnt 751

## Review the Concepts

1. Name three features common to the activation of cytokine receptors and receptor tyrosine kinases. Name one difference with respect to the enzyme activity of these receptors. 2. Erythropoietin (Epo) is a hormone that is produced naturally in the body in response to low $\mathrm{O}_{2}$ levels in the blood. The intracellular events that occur in response to Epo binding to its cell-surface receptor are well characterized. What molecule translocates from the cytosol to the nucleus after (a) JAK2 activates STAT5 and (b) GRB2 binds to the Epo receptor? Why did some endurance athletes use Epo to improve their performance ("blood doping") until it was banned by most sports?
2. Explain how expression of a dominant-negative mutant of JAK blocks the erythropoietin (Epo)-cytokine signaling pathway.
3. Even though GRB2 lacks intrinsic enzyme activity, it is an essential component of the epidermal growth factor (EGF) signaling pathway that activates MAP kinase. What is the function of GRB2? What roles do the SH2 and SH3 domains play in the function of GRB2? Many other signaling proteins possess SH2 domains. What determines the specificity of SH2 interactions with other molecules?
4. Once an activated signaling pathway has elicited the proper changes in target-gene expression, the pathway must be inactivated. Otherwise, pathological consequences may

result, as exemplified by persistent growth factor-initiated signaling in many cancers. Many signaling pathways possess intrinsic negative feedback by which a downstream event in a pathway turns off an upstream event. Describe the negative feedback that down-regulates signals induced by (a) erythropoietin and (b) TGF- $\beta$.
6. A mutation in the Ras protein renders Ras constitutively active $\left(\mathrm{Ras}^{\mathrm{D}}\right)$. What is constitutive activation? How does constitutively active Ras promote cancer? What type of mutation might render the following proteins constitutively active: (a) Smad3, (b) MAP kinase, and (c) NF- $\kappa$ B?
7. The enzyme Ste11 participates in several distinct MAP kinase signaling pathways in the budding yeast S. cerevisiae. What is the substrate for Ste11 in the mating factor signaling pathway? When a yeast cell is stimulated by mating factor, what prevents the induction of osmolytes required for survival in high-osmotic-strength media, given that Ste11 also participates in the MAP kinase pathway initiated by high osmolarity?
8. Describe the events required for full activation of protein kinase B. Name two effects of insulin mediated by PKB in muscle cells.
9. Describe the function of the PTEN phosphatase in the PI-3 kinase signaling pathway. Why does a loss-of-function mutation in PTEN promote cancer? Predict the effect of constitutively active PTEN on cell growth and survival.
10. Binding of TGF- $\beta$ to its receptors can elicit a variety of responses in different cell types. For example, TGF- $\beta$ induces plasminogen activator inhibitor 1 in epithelial cells and specific immunoglobulins in B cells. In both cell types, Smad3 is activated. Given the conservation of the signaling pathway, what accounts for the diversity of the response to TGF- $\beta$ in various cell types?
11. How is the signal generated by binding of TGF- $\beta$ to cellsurface receptors transmitted to the nucleus, where changes in target-gene expression occur? What activity in the nucleus ensures that the concentration of active Smads closely reflects the level of activated TGF- $\beta$ receptors on the cell surface?
12. The extracellular signaling protein Hedgehog can remain anchored to cell membranes. What modifications to Hedgehog enable it to be membrane bound? Why is this property useful?
13. Explain why loss-of-function hedgehog and smoothened mutations yield the same phenotype in flies, but a loss-offunction patched mutation yields the opposite phenotype.
14. Most mammalian cells have a single immobile cilium called the primary cilium, in which intraflagellar transport (IFT) motor proteins (discussed in greater detail in Chapter 18) move elements of the Hedgehog (Hh) signaling pathway along microtubules. What parts of the Hh signaling pathway would mutations in the IFT motor proteins Kif3A, Kif7, and dynein disrupt?
15. Why is the signaling pathway that activates NF- $\kappa \mathrm{B}$ considered to be relatively irreversible compared with cytokine or RTK signaling pathways? Nonetheless, NF- $\kappa$ B signaling must be down-regulated eventually. How is the NF- $\kappa$ B signaling pathway turned off?
16. Describe two roles for polyubiquitinylation in the NF- $\kappa$ B signaling pathway.
17. What feature of Delta ensures that only neighboring cells are signaled?
18. What biochemical reaction is catalyzed by $\gamma$-secretase? Why was it proposed that a chemical inhibitor of this activity might be a useful drug for treating Alzheimer's disease? What possible side effects of such a drug would complicate this use?

## References

Receptor Serine Kinases That Activate Smads
Deheuninck, J., and K. Luo. 2009. Ski and SnoN, potent negative regulators of TGF- $\beta$ signalling. Cell Res. 19:47-57.

Massagué, J. 2012. TGF $\beta$ signalling in context. Nat. Rev. Mol. Cell Biol. 13:617-630.

Xu, P., J. Liu, and R. Derynck. 2012. Post-translational regulation of TGF- $\beta$ receptor and Smad signaling. FEBS Lett. 586:18711884.

## Cytokine Receptors and the JAK/STAT Signaling Pathway

Hattangadi, S., et al. 2011. From stem cell to red cell: regulation of erythropoiesis at multiple levels by multiple proteins, RNAs and chromatin modifications. Blood 118:6258-6268.

Pfeifer, A. C., J. Timmer, and U. Klingmuller. 2008. Systems biology of JAK/STAT signalling. Essays Biochem. 45:109-120.

Schindler, C., D. E. Levy, and T. Decker. 2007. JAK-STAT signaling: from interferons to cytokines. J. Biol. Chem. 282:2005920063.

## Receptor Tyrosine Kinases

Lemmon, M. A., and J. Schlessinger. 2010. Cell signaling by receptor tyrosine kinases. Cell 141:1117-1134.

Lemmon, M., J. Schlessinger, and K. Fergusun, eds. 2014. The EGFR family: not so prototypical receptor tyrosine kinases. Cold Spring Harb. Perspect. Biol. doi: 10.1101/cshperspect.a020768.

Levitzki, A. 2013. Tyrosine kinase inhibitors: views of selectivity, sensitivity, and clinical performance. Annu. Rev. Pharmacol. 53:161-85.

Tomas, A., C. Futter, and E. Eden. 2014. EGF receptor trafficking: consequences for signaling and cancer. Trends Cell Biol. 24:26-34.

## The Ras/MAP Kinase Pathway

Chen, R., and J. Thorner. 2007. Function and regulation in MAPK signaling pathways: lessons learned from the yeast Saccharomyces cerevisiae. Biochim. Biophys. Acta 1773:1311-1340.

Gastel, M. 2006. MAPKAP kinases-MKs-two's company, three's a crowd. Nat. Rev. Mol. Cell Biol. 7:211-224.

Matallanas, D., et al. 2011. Raf family kinases: old dogs have learned new tricks. Genes Cancer 3:232-260.

## Phosphoinositide Signaling Pathways

Fayard, E., et al. 2010. Protein kinase B PKB/Akt, a key mediator of the PI3K signaling pathway. Curr. Top. Microbiol. 346:31-56.

Michell, R. H., et al. 2006. Phosphatidylinositol 3,5-bisphosphate: metabolism and cellular functions. Trends Biochem. Sci. 31:52-63.

Vogt, P. K., et al. 2010. Phosphatidylinositol 3-kinase: the oncoprotein. Curr. Top. Microbiol. Immunol. 347:79-104.

Signaling Pathways Controlled by Ubiquitinylation and Protein Degradation: Wnt, Hedgehog, and NF-кB

Briscoe, J., and P. Thérond. 2013. The mechanisms of Hedgehog signalling and its roles in development and disease. Nat. Rev. Mol. Cell Biol. 14:416-429.

Goetz, S., and K. Anderson. 2010. The primary cilium: a signalling centre during vertebrate development. Nat. Rev. Genet. 11:331-344.

Iwai, K., et al. 2014. Linear ubiquitin chains: NF-кB signalling, cell death and beyond. Nat. Rev. Mol. Cell Biol. 15:503-508.

Nehers, C. 2012. The complex world of WNT receptor signalling. Nat. Rev. Mol. Cell Biol. 13:767-779.

Wan, F., and M. Lenardo. 2010. The nuclear signaling of NF$\kappa B$ : current knowledge, new insights, and future perspectives. Cell Res. 20:24-33.

Signaling Pathways Controlled by Protein Cleavage: Notch/Delta, SREBP, and Alzheimer's Disease

Anderson, P., et al. 2012. Non-canonical Notch signaling: emerging role and mechanism. Trends Cell Biol. 22:257-265.

Brown, M. S., and J. L. Goldstein. 2009. Cholesterol feedback: from Schoenheimer's bottle to Scap's MELADL. J. Lipid Res. 50:S15-S27.

Musse, A., et al. 2012. Notch ligand endocytosis: mechanistic basis of signaling activity. Semin. Cell Dev. Biol. 23:429-436.

Integration of Cellular Responses to Multiple Signaling Pathways: Insulin Action

Bogan, J. 2012. Regulation of glucose transporter translocation in health and diabetes. Annu. Rev. Biochem. 81:507-532.

Leto, D., and A. Saltiel. 2012. Regulation of glucose transport by insulin: traffic control of GLUT4. Nat. Rev. Mol. Cell Biol. 1:383-396.

Rosen, E., and O. MacDougald. 2006. Adipocyte differentiation from the inside out. Nat. Rev. Mol. Cell Biol. 7:885-896.

![li3084542480e](li3084542480e.jpg)

A section of mouse intestine stained for actin (red), the extracellular matrix protein laminin (green), and DNA (blue). Each blue dot of DNA indicates the presence of a cell. Actin in the microvilli on the apical end of the epithelial cells can be seen lining the surface facing the lumen (top). Actin can also be seen prominently in the smooth muscle that surrounds the intestine (bottom). [Courtesy Thomas Deerinck and Mark Ellisman.]

When we look through a microscope at the wonderful diversity of cells in nature, the variety of cell shapes and movements we can discern is astonishing. At first we may notice that some cells, such as vertebrate sperm, ciliates such as Tetrahymena, or flagellates such as Chlamydomonas, swim rapidly, propelled by cilia and flagella. Other cells, such as amoebae and human macrophages, move more sedately, propelled not by external appendages, but by coordinated movement of the cell itself. We might also notice that some cells in tissues attach to one another, forming a pavementlike sheet, whereas other cells-neurons, for example-have long processes, up to 3 feet in length, and make selective contacts between cells. Looking more closely at the internal organization of cells, we see that organelles have characteristic locations; for example, the Golgi complex is generally near the central nucleus. How is this diversity of shape, cellular organization, and motility achieved? Why is it important for cells to have a distinct shape and clear internal organization?

## Cell Organization and Movement I: Microfilaments

Let us first consider two examples of cells with very different functions and organizations.

The epithelial cells that line the intestine form a tight, pavement-like layer of brick-shaped cells, known as an epithelium (Figure 17-1a, b). Their function is to import nutrients (such as glucose) from the intestinal lumen across the apical (top) plasma membrane and export them across the basolateral (bottom-side) plasma membrane toward the bloodstream. To perform this directional transport, the apical and basolateral plasma membranes of epithelial cells must have different protein compositions. Epithelial cells are attached and sealed together by cell junctions (discussed in Chapter 20), which create a physical barrier between the apical and basolateral domains of the membrane. This separation allows the cell to place the correct transport proteins in the plasma membranes of the two surfaces. In addition, the apical membrane has a unique morphology, with numerous fingerlike projections called microvilli that increase the

### 17.5 Myosins: Actin-Based Motor Proteins

### 17.6 Myosin-Powered Movements

17.7 Cell Migration: Mechanism, Signaling, and Chemotaxis

![li3094542480e](li3094542480e.jpg)

FIGURE 17-1 Overview of the cytoskeletons of an epithelial cell and a migrating cell. (a) Transmission electron micrograph of a thin section of an epithelial cell from the small intestine, showing the core bundles of microfilaments that provide support to the microvilli. (b) Epithelial cells are highly polarized, with distinct apical and basolateral domains. An intestinal epithelial cell transports nutrients into the cell through the apical domain and out of the cell across the basolateral domain. (c) Transmission electron micrograph of part of the leading edge of a migrating cell. The cell was treated with a mild detergent to dissolve the membranes, which also allows solubilization
of most cytoplasmic components. The remaining cytoskeleton was shadowed with platinum and visualized in the electron microscope. Note the network of actin filaments visible in this micrograph. (d) A migrating cell, such as a fibroblast or a macrophage, has morphologically distinct domains, with a leading edge at the front. Microfilaments are indicated in red, microtubules in green, and intermediate filaments in dark blue. The position of the nucleus (light blue oval) is also shown. (Part (a) ©1975 Mark Mooseker + T. Luey et al., The Journal of Cell Biology. 67:725-743. doi:10.1083/jcb.67.3.725.Part (c) ©1999 Tatyana Svitkina, Gary Borisy et al., The Journal of Cell Biology 145:1009-1026. doi: 10.1083/jcb.145.5.1009]
area of the plasma membrane available for nutrient absorption. To achieve this organization, epithelial cells must have an internal structure to give them shape and to deliver the appropriate proteins to the correct membrane surface.

Now consider the macrophage, a type of white blood cell that seeks out infectious agents and destroys them by an engulfing process called phagocytosis. Bacteria release chemicals that attract the macrophage and guide it to the infection. As the macrophage follows the chemical gradient, twisting and turning to get to the bacteria and phagocytose them, it must constantly reorganize its cell locomotion machinery.

As we will see, the internal motile machinery of macrophages and other crawling cells is always oriented in the direction in which they crawl (Figure 17-1c, d).

These are just two examples of cell polarity: the ability of cells to generate functionally distinct regions. In fact, as you think about all types of cells, you will realize that most of them have some form of cell polarity. An additional and fundamental example of cell polarity is the ability of cells to divide: they must first select an axis for cell division and then set up the machinery to segregate their organelles along that axis.

![li3104542480e](li3104542480e.jpg)

FIGURE 17-2 The components of the cytoskeleton. Each filament type is assembled from specific subunits in a reversible process so that cells can assemble and disassemble filaments as needed. Bottom panels show localization of the three filament systems in cultured cells
as seen by immunofluorescence microscopy of actin, tubulin, and an intermediate filament protein, respectively. (Actin and tubulin courtesy of Damien Garbett and Anthony Bretscher; intermediate filaments photo courtesy Molecular Expressions, Nikon \& FSU.)

A cell's shape, internal organization, and functional polarity are provided by a three-dimensional filamentous protein network called the cytoskeleton. The cytoskeleton can be isolated and visualized after treating cells with gentle detergents that solubilize the plasma membrane and internal organelles, releasing most of the cytoplasm (see Figure 17-1c). The cytoskeleton extends throughout the cell and is attached to the plasma membrane and internal organelles, thus providing a framework for cellular organization. The term cytoskeleton may imply a fixed structure like a bone skeleton. In fact, the cytoskeleton can be very dynamic, with components capable of reorganization in less than a minute, or it can be quite stable for hours at a time. As a result, the lengths and dynamics of cytoskeletal filaments can vary greatly, they can be assembled into diverse types of structures, and they can be regulated locally in the cell.

The cytoskeleton is composed of three major filament systems, shown in Figures 17-1b and 17-1d as well as in Figure 17-2. Each filament system is a polymer of assembled subunits that is organized and regulated in time and space. The subunits that make up the filaments undergo regulated assembly and disassembly, giving the cell the flexibility to assemble or disassemble different types of structures as needed.

- Microfilaments are polymers of the protein actin that are organized into functional bundles and networks by actinbinding proteins. Microfilaments are especially important in the organization of the plasma membrane, giving shape to surface structures such as microvilli. Microfilaments can function on their own or serve as tracks for ATP-powered myosin motor proteins, which provide a contractile function (as in muscle) or ferry cargo along microfilaments.
- Microtubules are long tubes formed by the protein tubulin and organized by microtubule-associated proteins. They often extend throughout the cell, providing an organizational framework for associated organelles and structural support to cilia and flagella. They also make up the structure of the mitotic spindle, the machine for separating duplicated chromosomes at mitosis. Motor proteins called kinesins and dyneins transport cargo along microtubules; like myosins, they are powered by ATP hydrolysis.
- Intermediate filaments are tissue-specific filamentous structures that serve a number of different functions, including lending structural support to the nuclear membrane, providing structural integrity to cells in tissues, and serving structural and barrier functions in skin, hair, and nails. Unlike microfilaments and microtubules, intermediate filaments are not used as tracks by motor proteins.

As we can see in Figure 17-1, different cells can construct very different arrangements of their cytoskeletons. To establish these arrangements, cells must sense signals-from soluble factors bathing the cell, from adjacent cells, or from the extracellular matrix-and interpret them (Figure 17-3). These signals are detected by cell-surface receptors that activate signal transduction pathways that ultimately converge on factors that regulate cytoskeletal organization.

The importance of the cytoskeleton for normal cell function and motility is evident when a defect in a cytoskeletal component-or in cytoskeletal regulation-causes a disease. For example, about 1 in 500 people has a defect that affects the contractile apparatus of the heart, which results in cardiomyopathies varying in degree of severity. Many diseases

![li3114542480e](li3114542480e.jpg)

FIGURE 17-3 Regulation of cytoskeleton function by cell signaling. Cells use cell-surface receptors to sense external signals from the extracellular matrix, other cells, or soluble factors. These signals are transmitted across the plasma membrane and activate specific cytosolic signaling pathways. Signals-often integrated from more than one receptor-lead to the organization of the cytoskeleton so as to provide cells with their shape as well as to determine organelle distribution and movement. In the absence of external signals, cells still organize their internal structure, but not in a polarized manner.
of red blood cells affect the cytoskeletal components that support these cells' plasma membranes. Metastatic cancer cells exhibit unregulated motility due to misregulation of the cytoskeleton, breaking away from their tissue of origin and migrating to new locations to form new colonies of uncontrolled growth.

In this and the following chapter, we discuss the structure, function, and regulation of the cytoskeleton. We see how a cell arranges its cytoskeleton to determine cell shape and polarity, to provide organization and motility to its organelles, and to be the structural framework for such processes as cell swimming and cell crawling. We discuss how cells assemble the three different filament systems and how signal transduction pathways regulate these structures both locally and globally. How the cytoskeleton is regulated during the cell cycle is discussed in Chapter 19, and how it participates in the functional organization of tissue is covered in Chapter 20. Our focus in this chapter is on microfilaments and actin-based structures. Although we initially examine the cytoskeletal systems separately, in the next chapter we will see that microfilaments cooperate with microtubules and intermediate filaments in the normal functioning of cells.

### 17.1 Microfilaments and Actin Structures

Microfilaments can assemble into a wide variety of different types of structures within a cell (Figure 17-4a). Each of these diverse structures underlies particular cellular functions.

Microfilaments can exist as a tight bundle of filaments making up the core of the slender, fingerlike microvilli, but they can also be found in a less ordered network beneath the plasma membrane, known as the cell cortex, where they provide support and organization. In epithelial cells, microfilaments form a contractile band around the cell, the adherens belt, that is intimately associated with adherens junctions (see Chapter 20) to provide strength to the epithelium. In migrating cells, a network of microfilaments is found at the front of the cell in the leading edge, or lamellipodium, from which bundles of filaments called filopodia may protrude. Many cells have contractile microfilaments called stress fibers, which attach to the external substratum as cells migrate (discussed in Chapter 20). Specialized cells such as macrophages use contractile microfilaments to engulf and internalize pathogens, which are then destroyed internally. Highly dynamic, short bursts of actin filament assembly can power the movement of endocytic vesicles away from the plasma membrane. At a late stage of cell division in animals, after all the organelles have been duplicated and segregated, a contractile ring forms and constricts to generate two daughter cells in a process known as cytokinesis. Thus cells use actin filaments in many ways: in a structural role, by harnessing the power of actin polymerization to do work, or as tracks for myosin motors. The electron micrograph in Figure 17-4b shows microfilaments in microvilli. Different arrangements of microfilaments often coexist within a single cell, as shown in Figure 17-4c for a migrating fibroblast.

The basic building block of microfilaments is actin, a protein that has the remarkable capacity for reversibly assembling into a polarized filament with functionally distinct ends. These filaments are then molded into the various structures described in the previous paragraph by actin-binding proteins. The name microfilament, which arose from the very thin filaments seen by electron microscopists in thinsection preparations of cells, refers to actin in its polymerized form, with its associated proteins. In this section, we look at the actin protein itself and the filaments into which it assembles.

## Actin Is Ancient, Abundant, and Highly Conserved

Actin is an abundant intracellular protein in eukaryotic cells. In muscle cells, for example, actin constitutes 10 percent by weight of the total cellular protein; even in nonmuscle cells, actin makes up $1-5$ percent of the cellular protein. The cytosolic concentration of actin in nonmuscle cells ranges from 0.1 to 0.4 mM ; in special structures such as microvilli, however, the local actin concentration can be as high as 5 mM . To grasp how much actin is present in cells, consider a typical liver cell, which has $2 \times 10^{4}$ insulin receptor molecules but approximately $5 \times 10^{8}$, or half a billion, actin molecules. Because they form structures that extend across large parts of the cell interior, cytoskeletal proteins are among the most abundant proteins in a cell.

![li3124542480e](li3124542480e.jpg)

FIGURE 17-4 Examples of microfilament-based structures. (a) In each panel, microfilaments are depicted in red. (b) The apical region of a polarized epithelial cell, showing the bundles of actin filaments that make up the cores of the microvilli. The sample was prepared using a rapid freeze, deep etch, rotary shadow protocol and
viewed by transmission electron microscopy. (c) A cell moving toward the top of the page, stained for actin with fluorescent phalloidin, a drug that specifically binds F-actin. Note how many different organizations of microfilaments can exist in one cell. [Part (b) courtesy Nobutaka Hirokawa; Part (c) courtesy J. Vic Small.]

Actin is encoded by a large family of genes that gives rise to some of the most conserved proteins within and across species. In those cases in which it has been examined, genetic analysis shows that actin plays an essential role in cells. The protein sequences of actins from amoebae and from animals are identical at 80 percent of their amino acid positions, despite about a billion years of evolution. The multiple actin genes found in modern eukaryotes are related to a bacterial gene, $M r e B$, whose product forms filaments that are important in bacterial cell-wall synthesis. Some single-celled eukaryotes, such as yeasts and amoebae, have one or two ancestral actin genes, whereas multicellular organisms often contain multiple actin genes. For instance, humans have six actin genes, and some plants have more than sixty actin genes (although most are pseudogenes, which do not encode functional actin proteins). Each functional actin gene encodes a different isoform of the protein. Actin isoforms have historically be classified into three groups based on their overall charge: the $\alpha$-actins, $\beta$-actins, and $\gamma$-actins. In vertebrates, four actin isoforms are present in specific types of muscle cells, and two isoforms are found in nonmuscle cells. These six isoforms differ at only about 25 of the 375 residues in the complete protein, or show about 93 percent identity. Although these differences may seem minor, the three types of isoforms have different functions: $\alpha$-actins are associated with various contractile structures, $\gamma$-actin accounts
for filaments in stress fibers, and $\beta$-actin is enriched in the cell cortex and leading edge of motile cells.

## G-Actin Monomers Assemble into Long, Helical F-Actin Polymers

Actin exists as a globular monomer called G-actin and as a filamentous polymer called F-actin, which is a linear chain of G-actin subunits. Each actin molecule contains a $\mathrm{Mg}^{2+}$ ion complexed with either ATP or ADP. In fact, actin is an ATPase, as it will hydrolyze ATP to ADP and $\mathrm{P}_{\mathrm{o}}$. The importance of the interconversion between the ATP and the ADP forms of actin is discussed below.

X-ray crystallographic analysis reveals that the G-actin monomer is separated into two lobes by a deep cleft (Figure 17-5a). At the base of the cleft is the ATPase fold, the site where ATP and $\mathrm{Mg}^{2+}$ are bound, which has structural similarity to the GTP-binding cleft of the GTPase molecular switches (see Figure 15-5). The floor of the cleft acts as a hinge that allows the lobes to flex relative to each other. When ATP or ADP is bound to G-actin, the nucleotide affects the conformation of the molecule; in fact, without a bound nucleotide, G-actin denatures very quickly. The addition of cations- $\mathrm{Mg}^{2+}, \mathrm{K}^{+}$, or $\mathrm{Na}^{+}$—to a solution of G-actin will induce the polymerization of G-actin into F-actin filaments. The process is reversible: F-actin depolymerizes into

![li3134542480e](li3134542480e.jpg)

FIGURE 17-5 Structures of monomeric G-actin and F-actin filaments. (a) Structure of the actin monomer (measuring $5.5 \times 5.5 \times 3.5 \mathrm{~nm}$ ), which is divided by a central cleft into two approximately equal-sized lobes and four subdomains, numbered I-IV. ATP (red) binds at the bottom of the cleft and contacts both lobes (the yellow ball represents $\mathrm{Mg}^{2+}$ ). The N -and C -termini lie in subdomain I. (b) An actin filament appears as two strands of subunits. One repeating unit consists of 28 subunits ( 14 in each strand, indicated by * for one strand), covering a distance of 72 nm . The ATP-binding cleft of every actin subunit is oriented toward the same end

G-actin when the ionic strength of the solution is lowered. The F-actin filaments that form in vitro are indistinguishable from microfilaments seen in cells, indicating that F-actin is the major component of microfilaments.

From the results of x-ray diffraction studies of actin filaments and from the actin monomer structure shown in Figure 17-5a, scientists have determined that the subunits in an actin filament are arranged in a helical structure (Figure 17-5b). In this arrangement, the filament can be considered as two helical strands wound around each other. Each subunit in the structure contacts one subunit above it and one below it in its own strand as well as two subunits in the other strand. The subunits in a single strand wind around the back of the other strand and repeat after 72 nm , or 14 actin subunits. Since there are two strands, the actin filament appears to repeat every 36 nm (see Figure 17-5b). When F-actin is viewed by electron microscopy after negative staining with uranyl acetate, it appears as a twisted string whose diameter varies between 7 and 9 nm (Figure 17-5c).

## F-Actin Has Structural and Functional Polarity

All subunits in an actin filament are oriented the same way. A consequence of this subunit orientation is that the filament as a whole exhibits polarity; that is, one end differs from the
![li3144542480e](li3144542480e.jpg)
of the filament. The end of a filament with an exposed binding cleft is the $(-)$ end; the opposite end is the $(+)$ end. (c) In the electron microscope, negatively stained actin filaments appear as long, flexible, and twisted strands of beaded subunits. Because of the twist, the filament appears alternately thinner ( $7-\mathrm{nm}$ diameter) and thicker ( $9-\mathrm{nm}$ diameter) (arrows). (The microfilaments visualized in a cell by electron microscopy are F-actin filaments plus any bound proteins.) [Part (a) data from C. E. Schutt et al., 1993, Nature 365:810, PDB ID 2brf, courtesy of M. Rozycki. Part (c) courtesy of Roger Craig, University of Massachusetts Medical School.]
other. As we will see, a result of this subunit orientation is that one end of the filament is favored for the addition of actin subunits and is designated the $(+)$ end, whereas the other end is favored for subunit dissociation and is designated the $(-)$ end. At the $(+)$ end, the ATP-binding cleft of the terminal actin subunit contacts the neighboring subunit, whereas on the $(-)$ end, the cleft is exposed to the surrounding solution (see Figure 17-5b).

Without the atomic resolution afforded by x-ray crystallography, the cleft in an actin subunit, and therefore the polarity of a filament, is not detectable. However, the polarity of actin filaments can be demonstrated by electron microscopy in "decoration" experiments, which exploit the ability of the motor protein myosin, discussed in section 17.5, to bind specifically to actin filaments. In this type of experiment, an excess of myosin S1, containing the actinbinding head domain of myosin, is mixed with actin filaments under conditions where binding takes place. Myosin attaches to the sides of a filament with a slight tilt. When all the actin subunits are bound by myosin, the filament appears "decorated" with arrowheads that all point toward one end of the filament (Figure 17-6).

The ability of myosin S1 to bind to and coat F-actin is very useful experimentally-it has allowed researchers to identify the polarity of actin filaments, both in vitro and in cells.

![li3154542480e](li3154542480e.jpg)

The arrowhead points to the $(-)$ end, and so the $(-)$ end is often called the "pointed" end of an actin filament; the $(+)$ end is known as the "barbed" end. Because myosin binds to actin filaments and not to microtubules or intermediate filaments, arrowhead decoration is one criterion by which actin filaments can be definitively identified among the other cytoskeletal fibers in electron micrographs of cells.

## KEY CONCEPTS OF SECTION 17.1

## Microfilaments and Actin Structures

- Microfilaments can be assembled into diverse structures, many of which are associated with the plasma membrane (see Figure 17-4a).
- Actin, the basic building block of microfilaments, is a major protein of eukaryotic cells and is highly conserved.
- Actin can reversibly assemble into filaments that consist of two helices of actin subunits.
- The actin subunits in a filament are all oriented in the same direction, with the nucleotide-binding site exposed on the $(-)$ end (see Figure 17-5).


### 17.2 Dynamics of Actin Filaments

The actin cytoskeleton is not a static, unchanging structure consisting of bundles and networks of actin filaments. Both the assembly and disassembly of microfilaments is exquisitely regulated. As a consequence, microfilaments are stable for hours in some structures, whereas in others they are highly dynamic, growing or shrinking in length over a period of seconds. These changes in the organization of actin filaments can generate forces that cause large changes in the shape of a cell or drive intracellular movements. In this section, we consider how the actin protein itself polymerizes and depolymerizes, which is responsible for the dynamic nature of filaments in cells. We will see that several actin-binding proteins make important contributions to the stability and disassembly of filaments. In the subsequent section, we turn to the mechanisms that cells use to assemble filaments and how the location of assembly is regulated by signaling pathways.

## Actin Polymerization In Vitro Proceeds in Three Steps

The in vitro polymerization of G-actin monomers to form F-actin filaments can be monitored by viscometry, sedimentation, fluorescence spectroscopy, or fluorescence microscopy (see Chapter 4). When actin filaments grow long enough to become entangled, the viscosity of the solution, which is measured as a decrease in its flow rate in a viscometer, increases. The basis of the sedimentation assay is the ability of ultracentrifugation ( $100,000 \mathrm{~g}$ for 30 minutes) to sediment F-actin, but not G-actin. The third assay makes use of G-actin covalently labeled with a fluorescent dye; the fluorescence spectrum of the labeled G-actin monomer changes when it is polymerized into F-actin. Finally, growth of the fluorescently labeled filaments can be imaged with fluorescence video microscopy. These assays are useful for kinetic studies of actin polymerization and for characterization of actin-binding proteins to determine how they affect actin dynamics or how they cross-link actin filaments.

The mechanism of actin assembly has been studied extensively. Remarkably, one can purify G-actin at a high protein concentration without it forming filaments-provided it is maintained in a buffer with ATP and low levels of cations. However, as we saw earlier, if the cation level is increased (e.g., to $100 \mathrm{mM} \mathrm{K}^{+}$and $2 \mathrm{mM} \mathrm{Mg}^{2+}$ ), the G-actin will polymerize, with the kinetics of the reaction depending on the starting concentration of G-actin. The polymerization of pure G-actin in vitro proceeds in three sequential phases (Figure 17-7a):

1. The nucleation phase is marked by a lag period in which G-actin subunits combine into an oligomer of two or three subunits. When the oligomer reaches three subunits in length, it can act as a seed, or nucleus, for the next phase.
2. During the elongation phase, the short oligomer rapidly increases in length by the addition of actin monomers to both of its ends. As F-actin filaments grow, the concentration of G-actin monomers decreases until equilibrium is reached between the filament ends and monomers, and a steady state is reached.
3. In the steady-state phase, G-actin monomers exchange with subunits at the filament ends, but there is no net change in the total length of filaments.
The kinetic curves in Figure 17-7b, c show the filament mass during each phase of polymerization. In Figure 17-7c we see

![li3164542480e](li3164542480e.jpg)

FIGURE 17-7 The three phases of in vitro G-actin polymerization. (a) In the initial nucleation phase, ATP-G-actin monomers (red) slowly form stable complexes of actin (purple). These nuclei are rapidly elongated in the second phase by the addition of subunits to both ends of the filament. In the third phase, the ends of actin filaments are
that the lag period is due to nucleation because it can be eliminated by the addition of a small number of F-actin nucleiconsisting of very short filaments-to a solution of G-actin.

How much G-actin is required for spontaneous filament assembly? Scientists have placed various concentrations of G-actin under polymerizing conditions and found that, below a certain concentration, filaments cannot assemble (Figure 17-8). Above this concentration, filaments begin to form, and when steady state is reached, the incorporation of more free subunits is balanced by the dissociation of subunits from filament ends to yield a mixture of filaments and monomers. The concentration at which filaments are formed is known
in equilibrium with monomeric G-actin. (b) Time course of the in vitro polymerization reaction reveals the initial lag period associated with nucleation, the elongation phase, and the steady state. (c) If some short, stable actin filament fragments are added at the start of the reaction to act as nuclei, elongation proceeds immediately, without any lag period.
as the overall critical concentration, $C_{\mathrm{c}}$. Below $C_{\mathrm{c}}$, filaments will not form; above $C_{\mathrm{c}}$, filaments form. At steady state, the concentration of monomeric actin remains at the critical concentration (see Figure 17-8).

## Actin Filaments Grow Faster at (+) Ends Than at ( - ) Ends

We saw earlier that myosin S1 decoration experiments revealed an inherent structural polarity of F-actin that is due to the uniform subunit orientation in the filament (see Figure 17-6). If free ATP-G-actin is added to a preexisting

FIGURE 17-8 Determination of filament formation by actin concentration. The critical concentration $\left(C_{c}\right)$ is the concentration at which G-actin monomers are in equilibrium with actin filaments. At monomer concentrations below the $C_{c}$, no polymerization takes place. When polymerization is induced at monomer concentrations above the $C_{c}$, filaments assemble until steady state is reached and the monomer concentration falls to $C_{c}$.
![li3174542480e](li3174542480e.jpg)

![li3184542480e](li3184542480e.jpg)

EXPERIMENTAL FIGURE 17-9 The two ends of a myosindecorated actin filament grow at different rates. When short actin filaments are decorated with myosin S1 heads to reveal the orientation of the filaments and then used to nucleate actin polymerization, G-actin monomers are added much more efficiently to the $(+)$ ends than to the $(-)$ ends of the nucleating filaments. [Courtesy Thomas Pollard.]
myosin-decorated filament, the two ends grow at very different rates (Figure 17-9). In fact, the rate of addition of ATP-G-actin is nearly 10 times faster at the $(+)$ end than at the $(-)$ end. The rate of addition is, of course, determined by the concentration of free ATP-G-actin. Kinetic experiments have shown that the rate of addition is about $12 \mu \mathrm{M}^{-1} \mathrm{~s}^{-1}$ at the $(+)$ end and about $1.3 \mu \mathrm{M}^{-1} \mathrm{~s}^{-1}$ at the $(-)$ end (Figure 17-10a). This means that if $1 \mu \mathrm{M}$ of free ATP-G-actin is added to preformed filaments, 12 subunits, on average, will be added to the $(+)$ end every second, whereas only 1.3
will be added to the $(-)$ end every second. What about the rate of subunit loss from each end? By contrast, the rates of dissociation of ATP-G-actin subunits from the two ends are quite similar, about $1.4 \mathrm{~s}^{-1}$ from the $(+)$ end and $0.8 \mathrm{~s}^{-1}$ from the $(-)$ end. Since this dissociation is simply the rate at which subunits leave ends, it does not depend on the concentration of free ATP-G-actin.

What implications do these association and dissociation rates have for actin dynamics? First, let's consider just one end, the $(+)$ end. As we noted above, the rate of addition of subunits depends on the free ATP-G-actin concentration, whereas the rate of loss of subunits does not. Thus subunits will be added at high free ATP-G-actin concentrations, but as the concentration is lowered, a point will be reached at which the rate of addition is balanced by the rate of loss, and no net growth occurs at that end. This point is called the critical concentration $C_{c}^{+}$for the $(+)$ end, and we can calculate it by setting the rate of assembly equal to the rate of disassembly. Thus, at the critical concentration, the rate of assembly is $C_{c}^{+}$times the measured rate of addition of $12 \mu \mathrm{M}^{-1} \mathrm{~s}^{-1}\left(\mathrm{C}_{c}^{+} 12 \mathrm{~s}^{-1}\right)$, whereas the rate of disassembly is independent of the free actin concentration, namely, $1.4 \mathrm{~s}^{-1}$. Setting these two rates equal to each other yields $C_{c}^{+}=1.4$ $\mathrm{s}^{-1} / 12 \mu \mathrm{M}^{-1} \mathrm{~s}^{-1}$, or $0.12 \mu \mathrm{M}$, for the $(+)$ end. Above this free ATP-G-actin concentration, subunits are added to the $(+)$ end and net growth occurs, whereas below it, there is a net loss of subunits, and shrinkage occurs.

Now let's consider just the $(-)$ end. Because the rate of addition is much lower, $1.3 \mu \mathrm{M}^{-1} \mathrm{~s}^{-1}$, yet the rate of dissociation is about the same, $0.8 \mathrm{~s}^{-1}$, we expect the critical concentration $C_{c}^{-}$at the $(-)$ end to be higher than $C_{c}^{+}$. Indeed, as
![li3194542480e](li3194542480e.jpg)
(b)
![li3204542480e](li3204542480e.jpg)

FIGURE 17-10 Actin treadmilling. ATP-actin subunits are added faster at the $(+)$ end than at the $(-)$ end of an actin filament, resulting in a lower critical concentration at the $(+)$ end and treadmilling at steady state. (a) The rate of addition of ATP-G-actin is much faster at the $(+)$ end than at the $(-)$ end, whereas the rate of dissociation of ADP-G-actin is similar at the two ends. This difference results in a lower critical concentration at the $(+)$ end. At steady state, ATP-actin
is added preferentially at the $(+)$ end, giving rise to a short region of the filament containing ATP-actin and regions containing ADP-P,-actin and ADP-actin toward the $(-)$ end. (b) At steady state, ATP-G-actin subunits add preferentially to the $(+)$ end, while ADP-G-actin subunits disassemble from the $(-)$ end, giving rise to treadmilling of subunits. Two actin subunits are colored blue as a reference point within the filament.

we just did for the $(+)$ end, we can calculate $C_{c}^{-}$to be about $0.8 \mathrm{~s}^{-1} / 1.3 \mu \mathrm{M}^{-1} \mathrm{~s}^{-1}$, or $0.6 \mu \mathrm{M}$. Thus, at less than $0.6 \mu \mathrm{M}$ free ATP-G-actin-say, $0.3 \mu \mathrm{M}$-the $(-)$ end will lose subunits. But notice that at this concentration the $(+)$ end will grow, because $0.3 \mu \mathrm{M}$ is above $C_{c}^{-}$. Because the critical concentrations for the two ends are different, at steady state the free ATP-G-actin will be intermediate between $C_{c}^{-}$and $C_{c}$, so the $(+)$ end will grow and the $(-)$ end will lose subunits. This phenomenon is known as treadmilling because particular subunits, such those shown in blue in Figure 17-10b, appear to move through the filament.

The treadmilling of actin filaments is powered by hydrolysis of ATP. When ATP-G-actin binds to a (+) end, ATP is hydrolyzed to ADP and $\mathrm{P}_{\mathrm{i}}$. The $\mathrm{P}_{\mathrm{i}}$ is slowly released from the subunits in the filament, so that the filament becomes asymmetric, with ATP-actin subunits at the $(+)$ end followed by a region with ADP- $\mathrm{P}_{\mathrm{i}}$-actin and then, after $\mathrm{P}_{\mathrm{i}}$ release, ADP-actin subunits toward the $(-)$ end (see Figure 17-10a). During hydrolysis of ATP and the subsequent release of $\mathrm{P}_{\mathrm{i}}$ from subunits in a filament, actin undergoes a conformational change that is responsible for the different association and dissociation rates at the two ends. Here we have considered only the kinetics of ATP-G-actin, but in reality it is ADP-G-actin that dissociates from the $(-)$ end. Our analysis also relies on a plentiful supply of ATP-G-actin, which, as we will see, turns out to be the case in vivo. Thus actin can use the power generated by hydrolysis of ATP to treadmill, and treadmilling filaments can do work in vivo, as we will see later.

How are the amounts of ATP-actin, ADP- $\mathrm{P}_{\mathrm{i}}$-actin, and ADP-actin in a filament determined? The actin ATPase activity and $\mathrm{P}_{\mathrm{i}}$ release are slow, so when the rate of actin assembly is enhanced, the amount of ATP-actin at the $(+)$ end is also enhanced. As we will see in the following section, cells have mechanisms to regulate the assembly, as well as the disassembly, of filaments, thereby regulating the nucleotide distribution along a filament.

## Actin Filament Treadmilling Is Accelerated by Profilin and Cofilin

Measurements of the rate of actin treadmilling in vivo show that it can be several times higher than can be achieved with pure actin in vitro under physiological conditions. Consistent with the treadmilling model, growth of actin filaments in vivo occurs only at the $(+)$ end. How is enhanced treadmilling achieved, and how does the cell recharge the ADP-actin dissociating from the $(-)$ end to ATP-actin for assembly at the $(+)$ end? Two different actin-binding proteins make important contributions to these processes.

The first of these proteins is profilin, a small protein that binds G-actin on the side opposite the nucleotide-binding cleft. When profilin binds ADP-actin, it opens the cleft and greatly enhances the loss of ADP, which is replaced by the more abundant cellular ATP, yielding a profilin-ATP-actin complex. This complex cannot bind to the $(-)$ end because profilin blocks the sites on G-actin for $(-)$ end assembly.
![li3214542480e](li3214542480e.jpg)

FIGURE 17-11 Regulation of filament turnover by actin-binding proteins. Actin-binding proteins regulate the rate of assembly and disassembly of actin filaments as well as the availability of G-actin for polymerization. In the profilin cycle $\mathbf{\square}$, profilin binds ADP-G-actin and catalyzes the exchange of ADP for ATP. The ATP-G-actin-profilin complex can deliver actin to the $(+)$ end of a filament with dissociation and recycling of profilin. In the cofilin cycle $\mathbf{\square}$ cofilin binds preferentially to filaments containing ADP-actin, inducing them to fragment and thus enhancing depolymerization by making more filament ends. In the thymosin- $\beta_{4}$ cycle $\mathbf{\square}$ ATP-G-actin made available by the profilin cycle is bound by thymosin- $\beta_{4}$, which sequesters it from polymerization. As the free G-actin concentration is lowered by polymerization, G-actin-thymosin- $\beta_{4}$ dissociates to make free G-actin available for association with profilin and further polymerization.

However, the profilin-ATP-actin complex can bind efficiently to the $(+)$ end, and profilin dissociates after the new actin subunit is bound (Figure 17-11). This function of profilin on its own does not enhance the treadmilling rate, but it does provide a supply of ATP-actin from released ADP-actin; as a consequence, essentially all the free G-actin in a cell has bound ATP.

Profilin has another important property: it can bind other proteins with sequences rich in proline residues at the same time it is binding actin. We will see later how this property functions in actin filament assembly.

Cofilin is another small protein involved in actin treadmilling, but it binds specifically to F-actin in which the subunits contain ADP, which are the older subunits in the filament toward the $(-)$ end (see Figure 17-10a). Cofilin

binds by bridging two actin monomers and inducing a small change in the twist of the filament. This small twist destabilizes the filament, breaking it into short pieces. By breaking the filament in this way, cofilin generates many more free $(-)$ ends and therefore greatly enhances the net disassembly of the filament (see Figure 17-11). The released ADP-actin subunits are then recharged by profilin and added to the (+) end as described above. In this way, profilin and cofilin can enhance treadmilling in vitro more than tenfold, up to levels seen in vivo. As might be anticipated, the cell uses signal transduction pathways to regulate both profilin and cofilin, and thereby the turnover of actin filaments.

## Thymosin- $\boldsymbol{\beta}_{4}$ Provides a Reservoir of Actin for Polymerization

It has long been known that cells often have a very large pool of unpolymerized actin, sometimes constituting as much as half the actin in the cell. Since cellular actin levels can be as high as $100-400 \mu \mathrm{M}$, this means that there can be as much as $50-200 \mu \mathrm{M}$ unpolymerized actin in cells. Since the critical concentration in vitro is about $0.2 \mu \mathrm{M}$, why doesn't all this actin polymerize? The answer lies, at least in part, in the presence of actin monomer sequestering proteins. One of these proteins is thymosin- $\beta_{4}$, a small protein that binds to ATPG -actin in such a way that it inhibits addition of the actin subunit to either end of the filament. Thymosin- $\beta_{4}$ can be very plentiful, for example, in human blood platelets. These disk-shaped cell fragments are very abundant in the blood, and when they are activated during blood clotting, they undergo a burst of actin assembly. Platelets are rich in actin: they are estimated to have a total concentration of about $550 \mu \mathrm{M}$ actin, of which about $220 \mu \mathrm{M}$ is in the unpolymerized form. They also contain about $500 \mu \mathrm{M}$ thymosin- $\beta_{4}$, which sequesters much of the free actin. However, as in any protein-protein interaction, free actin and free thymosin- $\beta_{4}$ are in a dynamic equilibrium with actin-thymosin- $\beta_{4}$. If some of the free actin is used up for polymerization, more actin-thymosin- $\beta_{4}$ will dissociate, providing more free actin for polymerization (see Figure 17-11). Thus thymosin- $\beta_{4}$ behaves as a buffer of unpolymerized actin, making it available when it is needed.

## Capping Proteins Block Assembly and Disassembly at Actin Filament Ends

The treadmilling and dynamics of actin filaments are further regulated in cells by capping proteins that specifically bind to the ends of the filaments. If this were not the case, actin filaments would continue to grow and disassemble in an uncontrolled manner. As one might expect, two classes of proteins have been discovered: ones that bind the $(+)$ end and ones that bind the $(-)$ end (Figure 17-12).

A protein known as CapZ, consisting of two closely related subunits, binds with a very high affinity $(\sim 0.1 \mathrm{nM})$ to the $(+)$ end of an actin filament, thereby inhibiting subunit addition or loss. The concentration of CapZ in cells
![li3224542480e](li3224542480e.jpg)

FIGURE 17-12 Filament capping proteins. Capping proteins block assembly and disassembly at filament ends. CapZ blocks the (+) end, which is where filaments normally grow, so its function is to limit actin dynamics to the $(-)$ end. The capping protein tropomodulin blocks $(-)$ ends, where filament disassembly normally occurs; thus the major function of tropomodulin is to stabilize filaments.
is generally sufficient to rapidly cap any newly formed (+) ends. So how can filaments grow at their $(+)$ ends? At least two mechanisms regulate the activity of CapZ. First, the capping activity of CapZ is inhibited by the regulatory phospholipid phosphatidylinositol 4,5-bisphosphate $\left[\mathrm{PI}(4,5) \mathrm{P}_{2}\right]$, found in the plasma membrane (see Chapter 16). Second, recent work has shown that certain regulatory proteins are able to bind the $(+)$ end and protect it from CapZ while still allowing assembly there. Thus cells have evolved an elaborate mechanism to block assembly of actin filaments at their $(+)$ ends except when and where assembly is needed.

Another protein, called tropomodulin, binds to the $(-)$ end of an actin filament, also inhibiting its assembly and disassembly. This protein is found predominantly in cells in which actin filaments need to be highly stabilized. Two examples of such filaments we will encounter later in this chapter are the short actin filaments in the cortex of the red blood cell and the actin filaments in muscle. In both cases, tropomodulin works with another protein, tropomyosin, which lies along the filament, to stabilize it. Tropomodulin binds to both tropomyosin and actin at the $(-)$ end to greatly stabilize the filament.

In addition to CapZ, another class of proteins can cap the $(+)$ ends of actin filaments. These proteins can also sever actin filaments. One member of this family, gelsolin, is regulated by $\mathrm{Ca}^{2+}$ ion concentrations. On binding $\mathrm{Ca}^{2+}$, gelsolin undergoes a conformational change that allows it to bind to the side of an actin filament and then insert itself between subunits of the helix, thereby breaking the filament. It then remains bound to and caps the $(+)$ end, generating a new $(-)$ end that can disassemble. As we discuss in a later section, actin cross-linking proteins can provide linkages between individual actin filaments to turn a solution of F-actin into a gel. If gelsolin is added to such a gel and the level of $\mathrm{Ca}^{2+}$ is elevated, gelsolin will sever the actin filaments and turn the gel back into a liquid solution. This ability to turn a gel into a sol is why the protein was named gelsolin.

## KEY CONCEPTS OF SECTION 17.2

## Dynamics of Actin Filaments

- The rate-limiting step in actin filament assembly is the formation of a short actin oligomer (nucleus) that can then be elongated into filaments.
- The critical concentration $\left(C_{c}\right)$ is the concentration of free G-actin at which the addition of monomers to a filament end is balanced by loss from that end.
- When the concentration of G-actin is above the $C_{c}$, the filament end will grow; when it is less than the $C_{c}$, the filament will shrink (see Figure 17-8).
- ATP-G-actin is added much faster at the $(+)$ end than at the $(-)$ end, resulting in a lower critical concentration at the $(+)$ end than at the $(-)$ end.
- At steady state, actin subunits treadmill through a filament. ATP-actin is added at the $(+)$ end, ATP is then hydrolyzed to ADP and $\mathrm{P}_{\mathrm{i}}, \mathrm{P}_{\mathrm{i}}$ is lost, and ADP-actin dissociates from the $(-)$ end.
- The length and rate of turnover of actin filaments is regulated by specialized actin-binding proteins (see Figure 17-11). Profilin enhances the exchange of ADP for ATP on G-actin; cofilin enhances the rate of loss of ADP-actin from the filament $(-)$ end, and thymosin- $\beta_{4}$ binds G-actin to provide reserve actin when it is needed. Capping proteins bind to filament ends, blocking assembly and disassembly.


### 17.3 Mechanisms of Actin Filament Assembly

The rate-limiting step of actin polymerization is the formation of an initial actin nucleus from which a filament can grow (see Figure 17-7a). In cells, this step is used as a control point to determine where actin filaments are assembled and what types of actin structures are generated (see Figures 17-1 and 17-4). Two major classes of actin-nucleating proteins, the formin protein family and the Arp2/3 complex, nucleate actin assembly under the control of signal transduction pathways. Moreover, they nucleate the assembly of different actin structures: formins lead to the assembly of long actin filaments, whereas the Arp2/3 complex leads to branched networks. Here we discuss each separately and see how the power of actin polymerization can drive motile processes in a cell. We then touch on the recent discoveries of new, specialized actin-nucleating factors.

## Formins Assemble Unbranched Filaments

Formins are found in essentially all eukaryotic cells as a diverse family of proteins: seven different classes are present
in vertebrates. Although they are diverse, all formin family members have two adjacent domains in common, the socalled FH1 and FH2 domains (formin-homology domains 1 and 2). Two FH2 domains from two individual formin monomers associate to form a doughnut-shaped complex (Figure 17-13a). This complex has the ability to nucleate actin assembly by binding two actin subunits, holding them so that the $(+)$ end of the nascent filament is toward the FH2 domains. The filament can now grow at the $(+)$ end while the FH2 domain dimer remains attached to it. How is this possible? As we saw earlier, an actin filament can be thought of as two intertwined strands of subunits. The FH2 dimer can bind to the two terminal subunits. It then probably rocks between the two end subunits, letting go of one to allow addition of a new subunit and then binding the newly added subunit and freeing up space for the addition of another subunit to the other strand. In this way, rocking between the two subunits on the end, it can remain attached while simultaneously allowing growth at the $(+)$ end (see Figure 17-13a).

TheFH1 domain adjacent to theFH2 domain also makes an important contribution to actin filament growth (Figure 17-14). This domain is rich in proline residues, which serve as sites for the binding of several profilin molecules. We discussed earlier how profilin can exchange the ADP nucleotide on G-actin for ATP to generate profilin-ATP-actin. The FH1 domain behaves as a landing site for profilin-ATP-G-actin to increase the local concentration of these complexes. The actin from these profilin-actin complexes is then fed into the FH2 domain to add actin to the $(+)$ end of the filament, and the profilin is released (see Figure 17-14). Since the formin allows rapid addition of actin subunits to the $(+)$ end, long filaments with a formin at their $(+)$ end are generated (Figure 17-13b). In this manner, formins not only nucleate actin assembly, but have the remarkable ability to remain bound to the $(+)$ end while also allowing rapid assembly there. To ensure the continued growth of the filament, formins bind to the $(+)$ end in such a way that precludes binding of a $(+)$ end capping protein such as CapZ, which would normally terminate assembly.

To be useful to a cell, formin activity must be regulated. Many formins exist in a folded, inactive conformation as a result of an interaction between the first half of the protein and its C-terminal tail. These formins are activated by membranebound Rho-GTP, a Ras-related small GTPase (discussed in Section 17.7). When Rho is switched from its inactive Rho-GDP form to its activated membrane-bound Rho-GTP state, it can bind and activate these formins (see Figure 17-14).

Recent studies have shown that formins are responsible for the assembly of long actin filaments such as those found in muscle cells, stress fibers, filopodia, and the contractile ring that forms during cytokinesis (see Figure 17-4). The actin-nucleating role of formins was discovered only recently, so the roles performed by this diverse protein family are only now being uncovered. Since there are many different formin classes in animals, it is likely that formins will be found to assemble additional actin-based structures.

![li3234542480e](li3234542480e.jpg)

FIGURE 17-13 Actin nucleation by the formin FH2 domain.
(a) Formins have a domain called FH2 that can form a dimer and nucleate filament assembly. The dimer binds two actin subunits step $\mathbf{B}$ and, by rocking back and forth steps $\mathbf{B}-\mathbf{B}$ can allow insertion of additional subunits between the FH2 domain and the (+) end of the growing filament. The FH2 domain protects the (+) end from being capped by capping proteins. (b) The FH2 domain of a formin was labeled with colloidal gold (black dot) and used to nucleate assembly of an actin filament. The resulting filament was visualized by electron microscopy after staining with uranyl acetate. Formins assemble long unbranched filaments. [Part (b) republished with permission of AAAS, from Pruyne et al., "Role of Formins in Actin Assembly: Nucleation and Barbed-End Association," Science 297:612, 2002; permission conveyed through Copyright Clearance Center, Inc.]

## Exterior

![li3244542480e](li3244542480e.jpg)

FIGURE 17-14 Regulation of formin activity by an intramolecular interaction. Some of the formin classes found in vertebrates are regulated by an intramolecular interaction. In its inactive state, the formin folds back on itself to inhibit the activity of the FH2 domain. The inactive formin is activated when its Rho-binding domain (RBD) binds to membrane-bound active Rho-GTP, resulting in exposure of the formin's FH2 domain, which can then nucleate the assembly of a new actin filament. All formins have an FH1 domain adjacent to the FH2 domain; the proline-rich FH1 domain is a site for recruitment of profilin-ATP-G-actin complexes that can then be "fed" into the growing (+) end. For simplicity of representation, a single formin protein is shown, but as shown in Figure 17-13, the FH2 domain functions as a dimer to nucleate actin assembly. Regulation of the Rho family of small GTPases is detailed in Figures 17-41 and 17-43.

## The Arp2/3 Complex Nucleates Branched Filament Assembly

The Arp2/3 complex is a protein machine consisting of seven subunits, two of which are actin-related proteins (Arp), explaining its name (Figure 17-15a). It is found in essentially all eukaryotes, including plants, yeasts, and animal cells. The Arp2/3 complex alone is a very poor nucleator. To nucleate the assembly of branched actin, Arp2/3 needs to be activated by interacting with a nucleation promoting factor (NPF), in addition to associating with the side of a preexisting actin filament. Although there are many different NPFs, the major NPF family is characterized by the presence of a region called WCA (WH2, connector, acidic). Experiments have shown that if the WCA domain is added to an actin assembly assay together with preformed actin filaments, Arp2/3 becomes a potent nucleator of further actin assembly.

How do the Arp2/3 complex and NPFs nucleate the assembly of actin filaments? Two NPFs each bind an actin subunit at their WH2 domains, and together, they activate the Arp2/3 complex through its interaction with their connector and acidic domains. In the inactive Arp2/3 complex, the two actin-related polypeptides-Arp2 and Arp3-are in the wrong configuration to nucleate filament assembly (see Figure 17-15b, step $\mathbf{B}$ ). When activated by the NPFs, Arp2 and Arp3 move into the correct configuration, and the complex binds the side of a preexisting actin filament. The actin subunits brought in by the WH2 domains of the NPFs

![li3254542480e](li3254542480e.jpg)

FIGURE 17-15 Actin nucleation by the Arp2/3 complex. (a) X-ray crystallographic structure of the Arp2/3 complex, with five of the subunits in gray and the Arp2 and Arp3 subunits in green and blue. (b) To nucleate actin assembly efficiently, Arp2/3 must interact with the activating part of an NPF, shown here with its W (WH2), C (connector), and A (acidic) domains. The first step involves binding of an actin subunit to the W domain of each NPF. Two NPF-actin complexes then bind the Arp2/3 complex step. This interaction induces a conformational change in the Arp2/3 complex. After binding of the Arp2/3 complex to the side of an actin filament, the actin subunits delivered by the W domains bind to the Arp2/3 complex step. which then initiates the
assembly of an actin filament at the available (+) end step. The Arp2/3 branch makes a characteristic $70^{\circ}$ angle between the old and new filaments. (c) Averaged image compiled from several electron micrographs of Arp2/3 at an actin branch. (d) Image of actin filaments in the leading edge of a motile cell, with a magnification and coloring of individual branched filaments. [Part (a) data from B. J. Nolen and T. D. Pollard, 2007, Mol. Cell 26:449-457, PDB ID 2p9l; part (c) from Egile C., Rouiller I., Xu X-P, Volkmann N., Li R., et al., 2005, Mechanism of filament nucleation and branch stability revealed by the structure of the Arp2/3 complex at actin branch junctions, PLoS Biol. 3(11): e383; part (d): 01999 Tatyana Svitkina, Gary Borisy et al., J. Cell Biol. 145:1009-1026. doi: 10.1083/jcb.145.5.1009]
binds to the Arp2/3 template to nucleate filament assembly at the $(+)$ end (Figure 17-15b). The NPFs are released, and the new $(+)$ end then grows as long as ATP-G-actin is available or until it is capped by a $(+)$ end capping protein such as CapZ. The angle between the old filament and the new one is $70^{\circ}$ (Figure 17-15c). This is also the angle observed experimentally in branched filaments at the leading edges of motile cells, which are believed to be formed by the action of the activated Arp2/3 complex (Figure 17-15d). As we discuss in subsequent sections, the Arp2/3 complex can be used to drive actin polymerization to power intracellular motility.

Actin nucleation by the Arp2/3 complex is finely controlled, and the NPFs are part of that regulatory process. One NPF is called WASp, as it is defective in patients with Wiskott-Aldrich syndrome, an X-linked disease characterized by eczema, low platelet count, and immune deficiency. WASp exists in a folded inactive conformation that makes
the WCA domain unavailable (Figure 17-16). It is important that this protein be activated only at the plasma membrane, and its activation requires two signals. One signal is the presence of the regulatory phospholipid $\mathrm{PI}(4,5) \mathrm{P}_{2}$, which is characteristically enriched in the plasma membrane (Chapter 16). WASp binds $\mathrm{PI}(4,5) \mathrm{P}_{2}$ through its basic domain. The second signal is binding of the activated form of the small GTPbinding protein Cdc42, which is itself activated in response to signaling pathways (discussed in Section 17.7). This type of two-signal input, called coincidence detection, ensures that the protein is activated only at the right place-at the plasma membrane-and by the right signaling pathway. Once bound to the two input signals, WASp is opened, and the WCA domain becomes accessible.

Another important NPF is a large protein complex called WAVE, which also has a WCA domain that can activate the Arp2/3 complex. WAVE is also activated by binding of

![li3264542480e](li3264542480e.jpg)

FIGURE 17-16 Regulation of the Arp2/3 complex by WASp and $\mathrm{PI}(4,5) \mathrm{P}_{3}$. The NPF WASp is inactive due to an intramolecular interaction that masks the WCA domain. It is activated by a coincidence detection mechanism: it must bind both the regulatory phospholipid $\mathrm{PI}(4,5) \mathrm{P}_{3}$ though its basic domain (B) and the membrane-bound active small G protein Cdc42-GTP (a member of the Rho family) through its Rho-binding domain (RBD). When activated in this way, the intramolecular interaction in WASp is relieved, allowing the W domain to bind actin and the acidic A domain to activate the Arp2/3 complex. For simplicity, only a single NFP-Arp2/3 interaction is shown. Regulation of the Rho family of small GTPases is detailed in Figures 17-41 and 17-43.
acidic phospholipids and by the active form of another small GTP-binding protein, Rac1. As we discuss in Section 17.7, activation of the Arp2/3 complex by Cdc42 through WASp and by Rac1 through WAVE induces the formation of different microfilament-based structures.

Although formins and the Arp2/3 complex are found in fungi, plants, and animals, additional actin nucleators have recently been discovered in animal cells. One of these, called Spire, has four tandem WH2 domains, so it can bind four actin monomers. It does this in a manner that allows the actins to assemble into a filament, although the detailed mechanism remains to be understood. Given that actin filaments perform so many functions in cells, it is not surprising that additional NPF proteins and actin nucleators have recently been described.

## Intracellular Movements Can Be Powered by Actin Polymerization

How can actin polymerization be harnessed to do work? As we have seen, actin polymerization involves the hydrolysis of actin-ATP to actin-ADP, which allows actin to grow preferentially at the $(+)$ end and disassemble at the $(-)$ end. If an actin filament were to become fixed in the network of the cytoskeleton and you could bind to and ride on the assembling $(+)$ end, you would be transported across the cell. This is just what the intracellular bacterial parasite Listeria monocytogenes does to get around the cell. The study of Listeria motility was, in fact, the way the nucleating activity of the Arp2/3 protein was discovered. As we will see shortly, Listeria has hijacked a normal cell motility process for its own purposes. We discuss Listeria first, as it is currently much better understood than the normal processes that employ similar mechanisms.

Listeria is a food-borne pathogen that causes mild gastrointestinal symptoms in most adults but can be fatal to elderly or immunocompromised individuals. It enters animal cells and divides in the cytoplasm. To move from one host cell to
another, it moves around the cell by polymerizing actin into a comet tail like the plume behind a rocket (Figure 17-17a, b), and when it runs into the plasma membrane, it pushes its way into the adjacent cell to infect it. To do this, it needs to direct the assembly of host-cell actin locally at its back and at the same time confine assembly there so that it efficiently pushes the bacterium forward. How does it do it? Listeria has on its surface a protein called ActA, which mimics an NPF by having an actin-binding site and an acidic region that efficiently activates the Arp2/3 complex (Figure 17-17c). The ActA protein also binds a host protein known as VASP, which has three important properties. First, VASP has a proline-rich region that can bind profilin-ATP-actin and thus enhance ATP-actin assembly into the newly formed barbed ends generated by the Arp2/3 complex. Second, it can hold onto the end of the newly formed filament. Third, it can protect the $(+)$ end of the growing filament from capping by CapZ. These properties allow VASP to enhance actin assembly and confine it to the rear of the bacterium. The assembling filaments then push on the bacterium. Since the filaments are embedded in the stationary cytoskeletal matrix of the cell, the Listeria cell is pushed forward, ahead of the polymerizing actin. Researchers have reconstituted Listeria motility in the test tube using purified proteins to see what the minimal requirements for Listeria motility are. Remarkably, the bacteria will move when just four proteins are added: ATP-G-actin, the Arp2/3 complex, CapZ, and cofilin (see Figure 17-17b, c). We have discussed the role of actin and Arp2/3, but why are CapZ and cofilin needed? As we have seen earlier, CapZ rapidly caps the free $(+)$ end of actin filaments, so when a growing filament no longer contributes to bacterial movement, it is rapidly capped and inhibited from further elongation. In this way, assembly occurs mostly adjacent to the bacterium, where ActA is stimulating the Arp2/3 complex. Cofilin is necessary to accelerate the disassembly of the $(-)$ end of the actin filament, regenerating free actin to keep the polymerization cycle going (see Figure 17-11). This minimal rate of motility

![li3274542480e](li3274542480e.jpg)

EXPERIMENTAL FIGURE 17-17 Listeria uses the power of actin polymerization for intracellular movement. (a) Fluorescence microscopy of a cultured cell stained with an antibody to a bacterial surface protein (red) and fluorescent phalloidin to localize F-actin (green). Behind each Listeria bacterium is an actin "comet tail" that propels the bacterium forward by actin polymerization. When the bacterium runs into the plasma membrane, it pushes the membrane out into a structure like a filopodium, which protrudes into a neighboring cell. (b) Listeria motility can be reconstituted in vitro with bacteria and just four proteins: ATP-G-actin, Arp2/3 complex, CapZ, and cofilin. This phase-contrast micrograph shows bacteria (black), behind which are the phase-dense actin tails. (c) A model of how Listeria moves
using just four proteins. The ActA protein on the bacterial cell surface activates the Arp2/3 complex to nucleate new filament assembly from preexisting filaments. Filaments grow at their ( + ) end until capped by CapZ. Actin is recycled through the action of cofilin, which enhances depolymerization at the ( - ) end of the filaments. In this way, polymerization is confined to the back of the bacterium and propels it forward. Although not essential for motility in vitro, the protein VASP (not shown) binds ActA in vivo to enhance motility, as described in the text. [Part (a) courtesy Julie Theriot and Timothy Michison; part (b) reprinted by permission from Macmillan Publishers Ltd: from Loisel, T. et al., "Reconstitution of actin-based motility of Listeria and Shigella using pure proteins," Nature, 1999, 401:613-616.]
can be increased by the presence of other proteins, such as VASP and profilin, as mentioned above.

To move inside cells, the Listeria bacterium, as well as other opportunistic pathogens such as the Shigella species that cause dysentery, hijacks a normal, regulated cellular process involved in cell locomotion. As we discuss in more detail later (Section 17.7), moving cells have a thin sheet of cytoplasm at the front of the cell called the leading edge (see Figures 17-1c, 17-4, and 17-15d). This thin sheet of cytoplasm consists of a dense network of actin filaments that are continually elongating at the front of the cell to push the membrane forward. Factors in the leading-edge membrane activate the Arp2/3 complex to nucleate these filaments.

Thus the power of actin assembly pushes the membrane forward to contribute to cell locomotion.

## Microfilaments Function in Endocytosis

As we saw in Chapter 14, endocytosis is the processes that cells use to take up particles, molecules, or fluid from the external medium by enclosing them in plasma membrane and then internalizing them. The uptake of molecules or liquid is called receptor-mediated or fluid-phase endocytosis, and the uptake of large particles is called phagocytosis ("cell eating"). Microfilaments participate in both of these processes.

![li3284542480e](li3284542480e.jpg)

FIGURE 17-18 Arp2/3-dependent actin assembly during endocytosis. (a) Clathrin-mediated endocytosis is a rapid and ordered process. It has been best studied in yeast, in which the temporal order of specific steps has been delineated. In vivo imaging has shown that endocytosis assembly factors recruit NPFs that activate the Arp2/3 complex. A burst of Arp2/3-dependent actin assembly drives internalized endocytic vesicles away from the plasma membrane in a manner

Fluid-phase endocytosis is a very highly organized process, and recent studies have shown that the power of actin assembly contributes to this mechanism. Endocytosis assembly factors recruit NPFs so that as the endocytic vesicles invaginate and pinch off from the membrane, they are driven into the cytoplasm, powered by a rapid and very short-lived burst (a few seconds in duration) of actin polymerization driven by the Arp2/3 complex (Figure 17-18a). This actinbased movement of endocytic vesicles can be reconstituted in vitro (Figure 17-18b) and is mechanistically very similar to leading-edge formation and Listeria motility.

Phagocytosis is a vital process in the recognition and removal of pathogens, such as bacteria, by leukocytes (white blood cells). The immune system identifies a bacterium as foreign material and makes antibodies that recognize components on its surface. As we discussed in Chapter 3, each antibody has a region called the Fab domain that binds specifically to its antigen, in this case a component on the bacterial cell surface. As antibodies coat the bacterium through interaction between their Fab domains and the cell-surface antigen, a second antibody domain, known as the Fc domain, is exposed. This process is known as opsonization (Figure 17-19, step $\mathbf{\square}$, see Chapter 23). The leukocytes have a receptor on their cell surface, called the Fc receptor, that recognizes the antibodies on the bacterium; this interaction signals the cells to bind and engulf the pathogen (steps $\mathbf{\square}$ and $\mathbf{\square}$ ). The signal also tells the cells to assemble microfilaments at the site of interaction with the bacterium, and the
(b)
![li3294542480e](li3294542480e.jpg)
similar to the movement of Listeria. (b) Endosome movement can be reconstituted in vitro. Endosomes isolated from cells that had taken up fluorescently labeled transferrin (red) were added to a cell extract containing fluorescently labeled actin (green). The endosomes bind WASp, which then activates the Arp2/3 complex to assemble actin tails that propel them through the cytoplasm. [Part (b) ©2000 Taunton et al., J. Cell Biol., 148:519-530. doi: 10.1083/jcb.148.3.519.]
assembled microfilaments, together with myosin motor proteins, provide the force necessary to draw the bacterium into the cell, ultimately fully enclosing the pathogen in plasma membrane (step $\mathbf{\square}$ ). Once internalized, the newly formed phagosome fuses with lysosomes, and the pathogen is killed and degraded by lysosomal enzymes.

Recently, actin assembly has been implicated in other steps of the endocytic pathway as well as in the secretory pathway. For example, an NPF called WASH has been implicated in the Arp2/3-dependent nucleation of actin filament on endosomes, regulating their shape and contributing to transport events. Another NPF, called WHAMM, localizes to the Golgi complex and is believed to direct the Arp2/3-dependent assembly of actin participating in membrane traffic from the endoplasmic reticulum to the Golgi. The emerging view is that actin assembly is not essential for several steps in membrane trafficking, but facilitates transport between different compartments of the secretory and endocytic pathways.

## Toxins That Perturb the Pool of Actin Monomers Are Useful for Studying Actin Dynamics

Certain fungi and sponges have developed toxins that target the polymerization cycle of actin and are therefore toxic to animal cells but are useful for the study of actin dynamics. Two types of these toxins have been characterized. The first class is represented by two unrelated toxins, cytochalasin D

![li3304542480e](li3304542480e.jpg)

FIGURE 17-19 Phagocytosis and actin dynamics. Actin assembly and contraction drives the phagocytic internalization of particles. Shown here is the phagocytosis and degradation of a bacterium by a leukocyte. An invading bacterium is coated by specific antibodies to a cell-surface protein in a process known as opsonization step $\mathbf{B}$. The Fc region of the bound antibodies is displayed on the bacterial surface and recognized by a specific receptor, the Fc receptor, on the leukocyte surface step $\mathbf{B}$. This interaction signals the cell to assemble a contractile actin structure that results in the internalization and engulfment of the bacterium step $\mathbf{B}$. Once it has been internalized into a phagosome, the bacterium is killed and degraded by enzymes delivered from lysosomes step $\mathbf{B}$.
and latrunculin, that promote the depolymerization of filaments, though by different mechanisms. Cytochalasin D, a fungal alkaloid, depolymerizes actin filaments by binding to the $(+)$ end of F-actin, where it blocks further addition of subunits. Latrunculin, a toxin secreted by sponges, binds and sequesters G-actin, inhibiting it from adding to a filament end. Exposure to either toxin thus increases the monomer pool. When cytochalasin D or latrunculin is added to live cells, the actin cytoskeleton disassembles and cell movements such as locomotion and cytokinesis are inhibited. These observations were among the first to implicate actin filaments in cell motility. Latrunculin is especially useful because it binds actin monomers and prevents any new actin assembly. Thus, if latrunculin is added to a cell, the rate at which actin-based structures disappear reflects their normal rate of turnover. This method has revealed that some actin structures have half-lives of less than a minute, whereas others are much more stable. For example, experiments with latrunculin show that the leading edges of motile cells turn over every $30-180$ seconds, and that stress fibers turn over every 5-10 minutes.

In contrast, the monomer-polymer equilibrium is shifted in the direction of filaments by jasplakinolide, another sponge toxin, and by phalloidin, which is isolated from Amanita phalloides (the "angel of death" mushroom). Jasplakinolide enhances nucleation by binding and stabilizing actin dimers and thereby lowering the critical concentration. Phalloidin binds at the interface between subunits in F-actin, locking adjacent subunits together and preventing actin filaments from depolymerizing. Even when actin is diluted below its critical concentration, phalloidin-stabilized filaments will not depolymerize. Because many actin-based processes depend on actin filament turnover, the introduction of phalloidin into a cell paralyzes all these systems, and the cell dies. However, phalloidin has been very useful to researchers, as fluorescent-labeled phalloidin, which binds only to F-actin, is commonly used to stain actin filaments for light microscopy (see Figure 17-4).

## KEY CONCEPTS OF SECTION 17.3

## Mechanisms of Actin Filament Assembly

- Actin assembly is nucleated by two classes of proteins: formins nucleate the assembly of unbranched filaments (see Figure 17-13), whereas the Arp2/3 complex nucleates the assembly of branched actin networks (see Figure 17-15). The activities of formins and Arp2/3 are regulated by signal transduction pathways.
- Functionally different actin-based structures are assembled by formins and by Arp2/3 nucleators. Formins drive the assembly of stress fibers and the contractile ring, whereas Arp2/3 nucleates the assembly of the branched actin filaments found in the leading edge of motile cells.
- The power of actin polymerization can be harnessed to do work, as is seen in the Arp2/3-dependent intracellular movement of pathogenic bacteria (see Figure 17-17) and the inward movement of endocytic vesicles (see Figures 17-18 and 17-19).
- Several toxins affect the dynamics of actin polymerization; some, such as latrunculin, bind and sequester actin monomers, whereas others, such as phalloidin, stabilize filamentous actin. Fluorescently labeled phalloidin is useful for staining actin filaments.


### 17.4 Organization of Actin-Based Cellular Structures

We have seen that actin filaments are assembled into a wide variety of different arrangements and that many associated proteins nucleate actin assembly and regulate filament turnover. Dozens of proteins in a vertebrate cell organize these filaments into diverse functional structures. Here we discuss just a few of these proteins, giving examples of typical types of actin cross-linking proteins found in cells as well as proteins involved in making functional links between actin and membrane proteins. One fascinating problem, about which very little is known, is how cells assemble different actinbased structures within the cytoplasm of the same cell. Some of this organization must be due to local regulation, a topic we come to at the end of the chapter.

## Cross-Linking Proteins Organize Actin Filaments into Bundles or Networks

When one assembles actin filaments in a test tube, they form a tangled network. In cells, however, actin filaments are found in a variety of distinct structures, such as the highly ordered filament bundles in microvilli or the network characteristic of the leading edge of a motile cell (see Figure 17-4a).

These different structures are determined by the filament assembly mechanisms discussed above and by the presence of actin cross-linking proteins. To be able to organize actin filaments, an actin cross-linking protein must have two F-actin-binding sites (Figure 17-20a).

Cross-linking of F-actin can be achieved by having two actin-binding sites within a single polypeptide, as with fimbrin, a protein found in microvilli that builds bundles of filaments all having the same polarity (Figure 17-20b). Other actin cross-linking proteins have a single actin-binding site in a polypeptide chain, and two chains associate to form a dimer that brings together two actin-binding sites. These dimeric cross-linking proteins can assemble to generate a rigid rod connecting the two binding sites, as happens with $\alpha$-actinin. Like fimbrin, $\alpha$-actinin bundles parallel actin filaments, but keeps them farther apart than fimbrin. Another protein, called spectrin, is a tetramer with two actin-binding sites; spectrin spans an even greater distance between actin filaments and makes networks under the plasma membrane (shown in Figure 17-21 and discussed in the next section). Other types of cross-linking proteins, such as filamin, have a highly flexible region between the two actin binding sites that functions like a molecular leaf spring, so they can make stabilizing cross-links between filaments in a network (Figure 17-20c) such as that found in the leading edge of a motile cell. The Arp2/3 complex, which we discussed in terms of its ability to nucleate actin filament assembly, is also an important crosslinking protein, attaching the $(-)$ end of one filament to the side of another filament (see Figure 17-15).

## Adapter Proteins Link Actin Filaments to Membranes

To contribute to the structure of cells and to harness the power of actin polymerization, actin filaments are very often attached to membranes or associated with intracellular structures. Actin filaments are especially abundant in the cell cortex underlying the plasma membrane, to which they give support. Actin filaments can interact with membranes either laterally or at their ends.

Our first example of actin filaments attached to a membrane involves the human erythrocyte-the red blood cell. The erythrocyte consists essentially of a plasma membrane enclosing a high concentration of the protein hemoglobin. It functions in the transport of oxygen from the lungs to tissues and of carbon dioxide from tissues back to the lungs-all powered by the magnificent muscle known as the heart. Erythrocytes must be able to survive the raging torrents of blood flow in the heart, then flow down arteries and survive squeezing through narrow capillaries before being cycled through the lungs via the heart. To survive this grueling process for thousands of cycles, erythrocytes have a based network underlying the plasma membrane that gives them both the tensile strength and the flexibility necessary for their journey. This network is based on short actin filaments about 14 subunits in length, stabilized on their sides

![li3314542480e](li3314542480e.jpg)

FIGURE 17-20 Actin cross-linking proteins. Actin cross-linking proteins mold F-actin filaments into diverse structures. (a) Examples of four F-actin cross-linking proteins, all of which have two domains (blue) that bind F-actin. Some have a $\mathrm{Ca}^{2+}$-binding site (purple) that inhibits their activity at high levels of free $\mathrm{Ca}^{2+}$. Also shown is dystrophin, which has an actin-binding site on its N -terminal end and a C-terminal domain that binds the membrane protein dystroglycan. (b) Transmission electron micrograph of a thin section of a stereocilium (an unfortunate
name, since it is really a giant microvillus) on a sensory hair cell in the inner ear. This structure contains a bundle of actin filaments crosslinked by fimbrin, a small cross-linking protein that allows for close and regular interaction of actin filaments. (c) Long cross-linking proteins such as filamin are flexible and can thus cross-link actin filaments into a loose network. [Part (b) ©1983 Tilney et al., J. Cell Biol., 96:822-834. doi: 10.1083/jcb.96.3.822; part (c) courtesy of John Hartwig, Harvard Brigham and Women's Hospital.]
by tropomyosin (discussed in more detail in Section 17.6) and by the capping protein tropomodulin on the ( - ) end. These short filaments serve as hubs for binding about six flexible spectrin molecules, generating a fishnet-like structure (Figure 17-21a) that provides both strength and flexibility. The spectrin molecules are attached to membrane proteins
through two mechanisms: through a protein called ankyrin to the bicarbonate transporter (a transmembrane protein also known as band 3), and through a spectrin- and F-actinbinding protein called band 4.1 to another transmembrane protein called glycophorin C (Figure 17-21b). Although this spectrin-based network is highly developed in erythrocytes,

![li3324542480e](li3324542480e.jpg)

FIGURE 17-21 Lateral attachment of microfilaments to membranes. (a) Electron micrograph of the erythrocyte membrane showing the spoke-and-hub organization of the cortical cytoskeleton supporting the plasma membrane in human erythrocytes. The long spokes are composed mainly of spectrin and can be seen to intersect at the hubs, or membrane-attachment sites. The darker spots along the spokes are ankyrin molecules, which link spectrin to integral membrane proteins. (b) Diagram of the erythrocyte cytoskeleton, showing the two main types of membrane attachments: $\mathbf{\square}$ through ankyrin to band 3 and $\mathbf{\square}$ through band 4.1 to glycophorin C. (c) Actin is incorporated into the tips of stereocilia (giant microvilli). Cells with stereocilia were transfected to express GFP-labeled actin for a short period of time and then
counterstained with rhodamine-phalloidin to stain all the F-actin. The experiment shows that new actin is incorporated at the tips of the stereocilia. (d) Ezrin, a member of the ezrin-radixin-moesin (ERM) family, links actin filaments laterally to the plasma membrane in surface structures such as microvilli; attachment can be direct or indirect. Ezrin, activated by phosphorylation (P), links directly to the cytoplasmic region of transmembrane proteins (right) or indirectly through a scaffolding protein such as EBP50 (left). See R. G. Fehon et al., 2010, Nature Rev. Mol. Cell Biol. 11:276, S. E. Lux, 1979, Nature 281:426, and E. J. Luna and A. L. Hitt, 1992, Science 258:955. [Part (a) from Byers T. J., Branton, D., 1985, Proc. Natl. Acad. Sci. USA 82:6153, courtesy Daniel Branton; part (c) ©2004 Rzadzinska, A. K. et al., J. Cell Biol. 164:887-897. doi: 10.1083/200310055.]
similar types of linkages occur in many cell types. For example, a related type of ankyrin-spectrin attachment links the $\mathrm{Na}^{+} / \mathrm{K}^{+}$ATPase to the actin cytoskeleton on the basolateral membrane of epithelial cells.

$\square$Genetic defects in proteins of the red blood cell cytoskeleton can result in cells that rupture easily, giving rise to diseases known as hereditary spherocytic anemias (spherocytic because the cells are rounder, anemias because there is a shortage of red blood cells) and hence a shorter life span. In human patients, mutations in spectrin, band 4.1, and ankyrin can cause these diseases.

Microfilaments also provide the support for cell-surface structures such as microvilli and membrane ruffles. If we look at a microvillus, it is clear that its actin filaments must have an end-on attachment at the tip and lateral attachments down its length. What is the orientation of actin filaments in microvilli? Decoration experiments show that the (+) end is at the tip. Moreover, when fluorescent actin is added to a cell, it is incorporated at the tip of a microvillus, showing not only that the (+) end is there, but also that actin filament assembly occurs there (Figure 17-21c). It is not yet known how actin filaments are attached at the microvillus tip or how assembly is regulated there. This (+) end orientation

of actin filaments with respect to the plasma membrane is found almost universally-not just in microvilli, but also, for example, in the leading edges of motile cells. The lateral attachments to the plasma membrane are provided, at least in part, by the ERM (ezrin-radixin-moesin) family of proteins. This family consists of regulated proteins that exist in a folded, inactive form. When locally activated by phosphorylation in response to an external signal, the F-actin-binding and membrane protein-binding sites of the ERM protein are exposed to provide a lateral linkage to actin filaments (Figure 17-21d). ERM proteins can link the actin filaments to the cytoplasmic domain of membrane proteins directly or indirectly through scaffolding proteins.

The types of actin-membrane linkages we have discussed so far do not involve areas of the plasma membrane attached directly to other cells in a tissue or to the extracellular matrix, but such linkages do exist. Contact between epithelial cells is mediated by highly specialized regions of the plasma membrane called adherens junctions (see Figure 17-1b). Other specialized regions of association called focal adhesions mediate attachment of cells to the extracellular matrix. In turn, these specialized types of attachments connect to the cytoskeleton, as will be described in more detail when we discuss cell migration (Section 17.7) and cells in the context of tissues (see Chapter 20).

Muscular dystrophies are genetic diseases that are often characterized by the progressive weakening of skeletal muscle. One of these genetic diseases, Duchenne muscular dystrophy, affects the protein dystrophin, whose gene is located on the X chromosome, so the disease is much more prevalent in males than in females. Dystrophin is a modular protein whose function is to link the cortical actin network of muscle cells to a complex of membrane proteins that link to the extracellular matrix. Thus dystrophin has an N-terminal actin-binding domain, followed by a series of spectrin-like repeats, and terminates in a domain that binds the transmembrane dystroglycan complex to the extracellular matrix protein laminin (see Figure 17-20a and Figure 1-31). In the absence of dystrophin, the plasma membrane of muscle cells becomes weakened by cycles of muscle contraction and eventually ruptures, resulting in death of the muscle fibers.

## KEY CONCEPTS OF SECTION 17.4

## Organization of Actin-Based Cellular Structures

- Actin filaments are organized by cross-linking proteins that have two F-actin-binding sites. Actin cross-linking proteins can be long or short, rigid or flexible, depending on the type of structure involved (see Figure 17-20).
- Actin filaments are attached laterally to the plasma membrane by specific classes of proteins, as seen in erythrocytes and in cell-surface structures such as microvilli (see Figure 17-21).
- The (+) ends of actin filaments can also be attached to membranes, with assembly mediated between the filament end and the membrane.
- Several diseases have been traced to defects in the microfilament-based cortical cytoskeleton that underlies the plasma membrane.


### 17.5 Myosins: Actin-Based Motor Proteins

In Section 17.3 we discussed how actin polymerization nucleated by the Arp2/3 complex can be harnessed to do work, as in the movement of vesicles during endocytosis, at the leading edges of motile cells, and the propulsion of the Listeria bacterium across the eukaryotic cell. In addition to this actin polymerization-based motility, cells have a large family of motor proteins called myosins that can move along actin filaments. The first myosin discovered, myosin II, was isolated from skeletal muscle. For a long time, biologists thought that this was the only type of myosin found in nature. However, they then discovered other types of myosins and began to ask how many different functional classes might exist. Today we know that there are several different classes of myosins, in addition to the myosin II of skeletal muscle, that move along actin. Indeed, with the discovery and analysis of all these microfilament-based motors and the corresponding microtubule-based motors described in the next chapter, our former relatively static view of cells has been replaced with the realization that the cytoplasm is incredibly dynamiclike an organized but busy freeway system with motors busily ferrying components around.

Myosins have the amazing ability to convert the energy released by ATP hydrolysis into mechanical work (movement along actin). All myosins convert energy from ATP hydrolysis into work, yet different myosins perform very different types of functions. For example, many molecules of myosin II pull together on actin filaments to bring about muscle contraction, whereas myosin V binds to vesicular cargo to transport it along actin filaments. The other classes of myosin provide a myriad of functions, from moving organelles around cells to contributing to cell migration.

To begin to understand myosins, we first discuss their general organization. Armed with this information, we explore the diversity of myosin classes in different organisms and describe in more detail some of those that are common in eukaryotes. To understand how the diverse functions of these myosin classes can be accommodated by one type of motor mechanism, we investigate the basic mechanism that converts the energy released by ATP hydrolysis into

work, and then see how this mechanism is modified to tailor the properties of specific myosin classes to their specific functions.

## Myosins Have Head, Neck, and Tail Domains with Distinct Functions

Much of what we know about myosins comes from studies of myosin II isolated from skeletal muscle. In skeletal muscle, hundreds of individual myosin II molecules are assembled into bipolar bundles called thick filaments (Figure 17-22a). In a later section, we will discuss how these myosin filaments interdigitate with actin filaments to bring about muscle contraction. Here we first investigate the properties of the myosin molecule itself.

It is possible to dissolve the myosin thick filament in a solution of ATP and a high concentration of salt, generating a pool of individual myosin II molecules. The soluble myo$\sin$ II molecule is actually a protein complex consisting of six polypeptide subunits. Two of the subunits are identical high-molecular-weight polypeptides known as myosin heavy chains. Each consists of a globular head domain and a long tail domain, connected by a flexible neck domain. The tails of the two myosin heavy chains intertwine, so that the head regions are in close proximity. The remaining four subunits of the myosin complex are smaller in size and are known as the light chains. There are two types of light chains, the essential light chain and the regulatory light chain. One light chain of each type associates with the neck region of each heavy chain (Figure 17-22b, top). The myosin heavy chain
![li3334542480e](li3334542480e.jpg)

FIGURE 17-22 Structure of myosin II. (a) Organization of myosin II in filaments isolated from skeletal muscle. Myosin II assembles into bipolar filaments in which the tails form the shaft of the filament and the heads are exposed at the ends. Treatment of bipolar filaments with high salt concentrations and ATP disassembles the filament into individual myosin II molecules. (b) A myosin II molecule consists of two identical heavy chains (light blue) and four light chains (green and dark blue). The tails of the heavy chains form a coiled coil to dimerize; the neck region of each heavy chain has two light chains associated with
it. Limited proteolytic cleavage of myosin II generates tail fragmentsLMM and S2—and the S1 motor domain. (c) Three-dimensional model of a single S1 head domain shows that it has a curved, elongated shape and is bisected by a cleft. The nucleotide-binding pocket lies on one side of this cleft, and the actin-binding site lies on the other side near the tip of the head. Wrapped around the shaft of the $\alpha$-helical neck are two light chains. These chains stiffen the neck so that it can act as a lever arm for the head. Shown here is the ADP-bound conformation. [Part (c) data from S. Gourinath et al., 2003, Structure 11:1621-1627, PDB ID 1qvi.]

and the two types of light chains are encoded by three different genes.

The soluble myosin II molecule has ATPase activity, reflecting its ability to power movements by hydrolysis of ATP. But which part of the myosin complex is responsible for this activity? A standard approach to identifying functional domains in a protein is to cleave the protein into fragments with specific proteases and then ask which fragments have the activity. Soluble myosin II can be cleaved by gentle treatment with the protease chymotrypsin to yield two fragments, one called heavy meromyosin (HMM; mero means "part of") and the other, light meromyosin (LMM) (Figure 17-22b, middle). The heavy meromyosin can be further cleaved by the protease papain to yield subfragment 1 (S1) and subfragment 2 (S2) (Figure 17-22b, bottom). By analyzing the properties of the various fragments-S1, S2, and LMM-it was found that the intrinsic ATPase activity of myosin resides in the S 1 fragment, as does its F-actinbinding site. Moreover, it was found that the ATPase activity of the S 1 fragment is greatly enhanced by the presence of filamentous actin, so that fragment is said to have an actinactivated ATPase activity, which is a hallmark of all myosins. The S1 fragment of myosin II consists of the head and neck domains with associated light chains, whereas the S2 and LMM regions make up the tail domain.

X-ray crystallographic analysis of the head and neck domains revealed the shapes of the subunits, the positions of the light chains, and the locations of the ATP-binding and actin-binding sites (Figure 17-22c). At the base of the myosin head is the $\alpha$-helical neck, where two light-chain molecules wrap around the heavy chain like C -clamps. In this position, the light chains stiffen the neck region. The actin-binding site is an exposed region at the tip of the head domain; the ATP-binding site is also in the head domain, within a cleft opposite the actin-binding site.

How much of the myosin II molecule is necessary and sufficient for its "motor" activity? To answer this question, one needs a simple in vitro motility assay. In one such assay,
![li3344542480e](li3344542480e.jpg)
the sliding-filament assay, myosin molecules are tethered to a coverslip to which is added stabilized, fluorescently labeled actin filaments. Because the myosin molecules are tethered, they cannot slide; thus any force generated by the interaction of myosin heads with actin filaments forces the filaments to move relative to the myosin (Figure 17-23a). If ATP is present, the added actin filaments can be seen to glide along the surface of the coverslip; if ATP is absent, no filament movement is observed. Using this assay, one can show that the S1 fragment of myosin II is sufficient to bring about movement of actin filaments. This movement is caused by the tethered myosin S1 fragments (bound to the coverslip) trying to "move" toward the ( + ) end of a filament; thus the filaments move with the ( - ) end leading. The rate at which myosin moves an actin filament can be determined from video recordings of sliding-filament assays (Figure 17-23b).

All myosins have a domain related to the S 1 domain of myosin II, comprising the head and neck domains, that is responsible for their motor activity. However, as we will see in a later section, the length of the neck domain and the numbers and types of light chains associated with it vary among myosin classes. The tail domain does not contribute to motility, but rather defines what is moved by the S1-related domain. Thus, as might be expected, the tail domains can be very different and are tailored to bind specific cargoes.

## Myosins Make Up a Large Family of Mechanochemical Motor Proteins

Since all myosins have related S1 domains with considerable similarity in their primary amino acid sequence, it is possible to determine how many myosin genes, and how many different classes of myosins, exist in a sequenced genome. There are about forty myosin genes in the human genome (Figure 17-24), nine in Drosophila, and five in budding yeast. Computer analysis of the sequence relationships between myosin head domains suggests that about 20 distinct classes of myosins have evolved in eukaryotes, with greater

## EXPERIMENTAL FIGURE 17-23 The sliding-filament assay

is used to detect myosin-powered movement. (a) After myosin molecules are adsorbed onto the surface of a glass coverslip, excess unbound myosin is removed; the coverslip is then placed myosin-side-down on a glass slide to form a chamber through which solutions can flow. A solution of actin filaments, made visible and stable by staining with rhodamine-labeled phalloidin, is allowed to flow into the chamber. In the presence of ATP, the myosin heads "walk" toward the ( + ) ends of the actin filaments by the mechanism discussed later and illustrated in Figure 17-26. Because the myosin tails are immobilized, walking of the heads toward the ( + ) ends causes sliding of the filaments, which appear to be moving with their ( - ) ends leading the way. Movement of individual filaments can be observed in a fluorescence light microscope. (b) These photographs show the positions of three actin filaments (numbered 1, 2, 3) at 30 -second intervals recorded by video microscopy. The rate of filament movement can be determined from such recordings. [Part (b) courtesy James Spudich.]

![li3354542480e](li3354542480e.jpg)

FIGURE 17-24 The myosin superfamily in humans. Results of a computer analysis of the relatedness of the S1 head domains of all of the approximately 40 myosins encoded by the human genome. Each myosin is indicated by a blue dot. The lengths of the black lines indicate phylogenetic distance relationships: myosins connected by short lines are closely related, whereas those separated by longer lines are more distantly related. Among these myosins, three classes-myosins I, II, and V—are widely represented among eukaryotes; others have more specialized functions. Indicated are examples in which loss of a specific myosin causes a disease. (Data from R. E. Cheney, 2001, Mol. Biol. Cell 12:780.)
sequence similarity within a class than between. As indicated in Figure 17-24, the genetic bases for some conditions have been traced to genes encoding myosins. All myosin head domains convert ATP hydrolysis into mechanical work using the same general mechanism. However, as we will see, subtle variations in this mechanism can have profound effects on the functional properties of different myosin classes. How do these different classes relate with respect to their tail domains? Amazingly, if one takes just the protein sequences of the tail domains of the myosins and uses this information to place them in classes, they fall into the same groupings as the motor domains. This finding implies that head domains with specific properties have coevolved with specific classes of tail domains, which makes a lot of sense, suggesting that each class of myosin has evolved to carry out a specific function.

Among all these classes of myosins are three especially well-studied ones, which are commonly found in animals and fungi: the so-called myosin I, myosin II, and myosin V families (Figure 17-25). In humans, eight genes encode heavy

![li3364542480e](li3364542480e.jpg)

FIGURE 17-25 Three common classes of myosins. Myosin I molecules consist of a head domain and a neck domain; a variable number of light chains is associated with the neck domain. Members of the myosin I class are the only myosins to have a single head domain. Some of these myosins are believed to associate directly with membranes through lipid interactions.

Myosin II molecules have two head domains and two light chains per neck and are the only class that can assemble into bipolar filaments. Myosin V molecules have two head domains and six light chains per neck. They bind specific receptors (brown box) on organelles, which they transport. All myosins in these three classes move toward the ( + ) end of actin filaments.

chains for the myosin I family, fourteen for the myosin II family, and three for the myosin V family (see Figure 17-24).

Myosin II molecules assemble into bipolar filaments with opposite orientations in each half of the filament, so that there is a cluster of head domains at each end of the filament. This organization is important for its involvement in contraction; indeed, this is the only class of myosins involved in contractile functions. The large number of members in this class reflects the need for myosin II filaments with the slightly different contractile properties seen in different muscles (e.g., skeletal, cardiac, and various types of smooth muscle) as well as in nonmuscle cells.

The myosin II class is the only one that assembles into bipolar filaments. All myosin II members have a relatively short neck domain and have two light chains per heavy chain. The myosin I class is quite large, has a variable number of light chains associated with the neck region, and is the only one in which the two heavy chains are not associated through their tail domains and so are single-headed. The large size and diversity of the myosin I class suggests that these myosins perform many functions, most of which remain to be determined, but some members of this family connect actin filaments to membranes, and others are implicated in endocytosis. Since these myosin molecules have only one head each, several of them must work together to generate movement, and at least one must remain attached to an actin filament. Members of the myosin V class have two heavy chains, resulting in a motor with two heads, long neck regions with six light chains each, and tail regions that dimerize and terminate in domains that bind to specific organelles to be transported. As we will see shortly, the length of the neck region affects the rate of myosin movement.

In every case that has been tested so far, myosins move toward the $(+)$ end of an actin filament-with one exception, the myosin VI found in animals. This remarkable myosin has an insert in its head domain to make it work in the opposite direction, and so movement is toward the $(-)$ end of an actin filament. Myosin VI is believed to contribute to endocytosis by moving the endocytic vesicles along actin filaments away from the plasma membrane. Recall that membrane-associated actin filaments have their $(+)$ ends toward the membrane, so a motor directed toward the $(-)$ end would take them away from the membrane toward the center of the cell.

## Conformational Changes in the Myosin Head Couple ATP Hydrolysis to Movement

Studies of muscle contraction provided the first evidence that myosin heads slide or walk along actin filaments. The unraveling of the mechanism of muscle contraction was greatly aided by the development of in vitro motility assays and single-molecule force measurements. On the basis of information obtained with these techniques and the
three-dimensional structure of the myosin head (see Figure 17-22c), researchers developed a general model for how myosin harnesses the energy released by ATP hydrolysis to move along an actin filament (Figure 17-26). Because all myosins are thought to use the same basic mechanism to generate movement, we will ignore for the moment whether the myosin tail is bound to a vesicle or is part of a thick filament, as it is in muscle. The most important aspect of this model is that the hydrolysis of a single ATP molecule is coupled to each step taken by a myosin molecule along an actin filament.

How can myosin convert the chemical energy released by ATP hydrolysis into mechanical work? As mentioned above, the S 1 head of myosin is an ATPase and thus has the ability to hydrolyze ATP into ADP and $\mathrm{P}_{\mathrm{i}}$. Biochemical analysis has revealed the mechanism of myosin movement (Figure 17-26a). In the absence of ATP, the head of myosin binds very tightly to F-actin. When ATP binds to the head, its affinity for F-actin is greatly reduced, and it releases from actin. The myosin head then hydrolyzes the ATP, and the hydrolysis products, ADP and $\mathrm{P}_{\mathrm{i}}$, remain bound to it. The energy provided by the hydrolysis of ATP induces a conformational change in the head that results in the head domain rotating with respect to the neck, assuming what is known as the "cocked" position (Figure 17-26b, top). In the absence of F-actin, release of $\mathrm{P}_{\mathrm{i}}$ is exceptionally slow-the slowest part of the ATPase cycle. However, in the presence of actin, the head binds F-actin tightly, inducing both release of $P_{i}$ and rotation of the head back to its original position, thus moving the actin filament relative to the neck domain (Figure 17-26b, bottom). In this way, binding to F-actin induces the movement of the head and the release of $\mathrm{P}_{\mathrm{i}}$, thereby coupling the two processes. This step is known as the power stroke. The head remains bound until the ADP leaves and a fresh ATP binds the head, releasing it from the actin filament. The cycle then repeats, and the myosin can move again against the filament.

How is the hydrolysis of ATP in the nucleotide-binding pocket of the head converted into force? The hydrolysis of ATP causes a small conformational change in the head domain. This small movement is amplified by a "converter" region at the base of the head, which acts like a fulcrum and causes the rodlike neck, also known as the lever arm, to rotate. This rotation is amplified by the neck domain, so the actin filament moves by a few nanometers (see Figure 17-26b).

This model makes a strong prediction: the distance a myosin moves along actin during hydrolysis of one ATPthe myosin step size-should be proportional to the length of the neck domain. To test this prediction, mutant myosin molecules were constructed with neck domains of different lengths, and the rate at which they moved down an actin filament was determined. Remarkably, there was an excellent correlation between the length of the neck domain and the rate of movement (Figure 17-27).

(a)
![li3374542480e](li3374542480e.jpg)
(b)
![li3384542480e](li3384542480e.jpg)

FIGURE 17-26 ATP-driven myosin movement along actin
filaments. (a) In the absence of ATP, the myosin head is firmly attached to the actin filament. Although this state is very short-lived in living muscle, it is the state responsible for muscle stiffness in death (rigor mortis). Step 1: On binding ATP, the myosin head releases from the actin filament. Step 1: The head hydrolyzes the ATP to ADP and $\mathrm{P}_{\mathrm{i}}$, which induces a rotation in the head with respect to the neck. This "cocked state" stores the energy released by ATP hydrolysis as elastic energy, like a stretched spring. Step 1: Myosin in the "cocked" state is stable until it binds actin. Step 1: When it is bound to actin, the myosin head couples release of $P_{i}$ with release of the elastic energy to move the actin filament. This movement is known as the "power stroke," as it involves moving the actin filament with respect to the end of the myosin neck domain. Step 1: The head remains tightly bound to the actin filament until ADP is released and fresh ATP is bound by the head. See R. D. Vale and R. A. Milligan, 2002, Science 288:88. (b) Molecular models of the conformational changes in the myosin head involved in "cocking" the head (upper panel) and after the power stroke (lower panel). The myosin light chains are shown in dark blue and green; the rest of the myosin head and neck are colored light blue, and actin is red. See S. Fischer et al., 2005, Proc. Natl. Acad. Sci. USA 102:6873-6876. [Part (b) data from Ken Holmes.]

![li3394542480e](li3394542480e.jpg)

EXPERIMENTAL FIGURE 17-27 The length of the myosin II neck domain determines the rate of movement. To test the leverarm model of myosin movement, investigators used recombinant DNA techniques to make myosin heads attached to neck domains of different lengths. The rate at which these myosins moved on actin filaments was determined. The longer the lever arm, the faster the myosin moved, supporting the proposed mechanism. [Data from K. A. Ruppel and J. A. Spudich, 1996, Ann. Rev. Cell Dev. Biol. 12:543-573.]

## Myosin Heads Take Discrete Steps Along Actin Filaments

The most critical feature of myosin is its ability to generate a force that powers movements. Researchers have used optical traps to measure the forces generated by single myosin molecules (Figure 17-28a). In this approach, myosin is immobilized on beads at a low density. An actin filament, held between two optical traps, is lowered toward the bead until it contacts a myosin molecule on the bead. When ATP is added, the myosin pulls on the actin filament. Using a mechanical feedback mechanism controlled by a computer, one can measure the distance pulled and the forces and duration of the movement. The results of optical trap studies show that myosin II takes discrete steps, which average out to about 8 nm (Figure 17-28b), and generates 3-5 piconewtons $(\mathrm{pN})$ of force, approximately the same force as that exerted by gravity on a single bacterium. We can also see that myosin II does not interact with the actin filament continuously, but rather binds, moves, and releases it (see Figure 17-28b). In fact, myosin II spends on average only about 10 percent of each ATPase cycle in contact with F-actin-it is said to have a duty ratio of 10 percent. This observation will be important later when we consider that in contracting muscle, hundreds of myosin heads pull on actin filaments, so that at any one time, 10 percent of the heads are engaged to provide a smooth contraction.

Now let us examine how myosin V moves. Scientists have managed to attach a fluorescent probe to just one of the two neck regions of a myosin V molecule and watch the fluorescent image as the molecule moves along an actin
filament (Figure 17-29a). The labeled head takes many 72-nm steps without releasing from the actin-it is said to move processively (Figure 17-29b). This step size is twice the 36-nm length between helical repeats in the actin filament (see Figures 17-5b and 17-29a). So each 36-nm helical repeat site on the filament presents binding sites as each head alternately takes 72-nm steps-like someone walking on stepping stones across a river and placing each foot on every other stone. Notice that although the step size for each individual head is 72 nm , a cargo attached to the myosin V tail region will move 36 nm per step as the head moves from behind the
(a)
![li3404542480e](li3404542480e.jpg)

EXPERIMENTAL FIGURE 17-28 Measuring myosin step size and force with actin held by optical traps. (a) Optical trap techniques can be used to determine the step size and force generated by a single myosin molecule. In an optical trap, the beam of an infrared laser is focused by a light microscope on a latex bead (or any other object that does not absorb infrared light) to capture and hold the bead in the center of the beam. The strength of the force holding the bead is adjusted by increasing or decreasing the intensity of the laser beam. In this experiment, an actin filament is held between two optical traps. The actin filament is then lowered onto a third bead coated with a dilute concentration of myosin molecules. If the actin filament encounters a myosin molecule in the presence of ATP, the myosin will pull on the actin filament, which allows the investigators to measure both the force generated and the step size the myosin takes. (b) Using an optical trap setup, investigators have analyzed the behavior of myosin II. As shown by the peaks in the trace, myosin II takes erratic small steps ( $5-15 \mathrm{~nm}$ ), which means that it binds the actin filament, moves, and then lets go. It is therefore a nonprocessive motor. [Part (b) data from Finer et al., 1994, Nature 368:113.]

(a)
![li3414542480e](li3414542480e.jpg)
(b)
![li3424542480e](li3424542480e.jpg)

EXPERIMENTAL FIGURE 17-29 Myosin V has a step size of 36 nm, with each head stepping hand-over-hand in 72-nm steps. (a) Researchers have managed to label the neck of just one head of myosin V and follow its movement down an actin filament with nanometer accuracy (see Section 4.2 for how this can be achieved). (b) Trace of the label on one myosin V molecule as it walks down an actin filament. The labeled myosin head takes successive 72-nm steps. When the label was attached to the tail, the myosin V motor as a whole was found to take 36-nm steps (not shown). Thus myosin V heads step hand-over-hand down an actin filament in 72-nm steps for each head, but with the motor as a whole moving 36 nm per step. As can be seen from the trace, myosin V takes many successive steps along a filament, so it is said to be processive. As shown in panel (a), the step size corresponds to equivalent sites on the helical structure of the actin filament. [Data from A. Yildiz et al., 2003, Science 300:2061.]
cargo to in front of it. Thus the step size of the motor overall is said to be 36 nm , as indicated in Figure 17-29. Myosin V has presumably evolved to have a long neck domain-the lever arm-to take large steps to match the size of the helical repeat of the filament. Moreover, its ATPase cycle has been modified to have a much higher duty ratio ( $>70$ percent) by slowing the rate of ADP release; thus the head remains in contact with the actin filament for a much larger percentage of the cycle. Since a single myosin V molecule has two heads, a duty ratio greater than 50 percent ensures that it maintains contact at all times as it moves down an actin filament, so that it does not fall off. These are exactly the properties one would expect for a motor designed to transport cargo along an actin filament.

## KEY CONCEPTS OF SECTION 17.5

## Myosins: Actin-Based Motor Proteins

- Myosins are actin-based motors powered by ATP hydrolysis.
- Myosins have a motor head domain, a lever-arm neck domain, and a cargo-binding tail domain (see Figure 17-22).
- There are many classes of myosins, three of which are present in many eukaryotes: myosin I, which has a single head domain; myosin II, which has two heads and assembles into bipolar filaments; and myosin V, which has two heads but does not assemble into filaments (see Figure 17-25).
- Myosins convert ATP hydrolysis to mechanical work by amplifying a small conformational change in the head through the neck domain when the head is bound to F-actin (see Figure 17-26).
- Myosin heads take discrete steps along an actin filament, which can be small ( 8 nm ) and nonprocessive in the case of myosin II, or large ( 36 nm for the motor overall, 72 nm for each head) and processive for myosin V.


### 17.6 Myosin-Powered Movements

We have already discussed the head and neck domains that are responsible for the motor properties of myosins. We now come to the tail domains, which define the cargoes that myosins move. The functions of many of the newly discovered classes of myosins found in metazoans are not yet known. In this section, we give just two examples for which we have a good idea of specific myosin functions. Our first example is skeletal muscle, in which myosin II was discovered. In muscle, many myosin II heads, each with a low duty ratio, are bundled into bipolar filaments, in which they work together to bring about contraction. Similarly organized contractile machineries function in the contraction of smooth muscle, in stress fibers, and in the contractile ring during cytokinesis. We then turn to the myosin V class, whose higher duty ratio allows these myosins to transport cargoes over relatively long distances without dissociating from actin filaments.

## Myosin Thick Filaments and Actin Thin Filaments in Skeletal Muscle Slide Past Each Other During Contraction

Muscle cells have evolved to carry out one highly specialized function: contraction. Muscle contractions must occur quickly and repetitively, and they must occur over long distances and with enough force to move large loads. A typical skeletal muscle cell is cylindrical, large ( $1-40 \mathrm{~mm}$ in length and $10-50 \mu \mathrm{~m}$ in width), and multinucleated (containing as many as 100 nuclei) (Figure 17-30a). Within each muscle cell are many myofibrils consisting of a regular repeating array

FIGURE 17-30 Structure of the skeletal muscle sarcomere.
(a) Skeletal muscles consist of muscle fibers made of bundles of multinucleated cells. Each cell contains a bundle of myofibrils, which consist of thousands of repeating contractile structures called sarcomeres.
(b) Electron micrograph of mouse skeletal muscle in longitudinal section, showing one sarcomere. On either side of the $Z$ disks are the lightly stained I bands, composed entirely of actin thin filaments. These thin filaments extend from both sides of the $Z$ disk to interdigitate with the dark-stained myosin thick filaments in the A band. (c) Diagram of the arrangement of myosin and actin filaments in a sarcomere. [Part (b) © James Dennis/Phototake.]
of a specialized structure called a sarcomere (Figure 17-30b). A sarcomere, which is about $2 \mu \mathrm{~m}$ long in resting muscle, shortens by about 70 percent of its length during contraction. Electron microscopy and biochemical analysis have shown that each sarcomere contains two major types of filaments: thick filaments, composed of myosin II, and thin filaments, containing actin and associated proteins (Figure 17-30c).

The thick filaments are myosin II bipolar filaments, in which the heads on each half of the filament have opposite orientations (see Figure 17-22a). The thin actin filaments are assembled with their $(+)$ ends embedded in a densely staining structure known as the $Z$ disk, so that the two sets of actin filaments in a sarcomere have opposite orientations (Figure 17-31). To understand how a muscle contracts, consider the interactions between one myosin head (among the hundreds in a thick filament) and one thin (actin) filament, as diagrammed in Figure 17-26. During these cyclical interactions, also called the cross-bridge cycle, the hydrolysis of ATP is coupled to the movement of a myosin head toward the Z disk, which corresponds to the (+) end of the actin thin filament. Because the thick filament is bipolar, the action of the myosin heads at opposite ends of the thick filament draws the thin filaments toward the center of the thick filament and therefore toward the center of the sarcomere (see Figure 17-31). This movement shortens the sarcomere until the ends of the thick filaments abut the Z disk. Contraction of an intact muscle results from the activity of hundreds of myosin heads on a single thick filament, amplified by the hundreds of thick and thin filaments in a sarcomere and thousands of sarcomeres in a muscle fiber. We can now see why myosin II is both nonprocessive and needs to have a low duty ratio: each head pulls a short distance on the actin filament and then lets go to allow other heads to pull, and so many heads working together allow the smooth contraction of the sarcomere. The first experimental basis for the sliding-filament model of muscle contraction is highlighted in Classic Experiment 17-1.

$\pm$The human heart is an amazing contractile organ-it contracts without interruption about 3 million times a year, or a fifth of a billion times in a lifetime. The muscle cells of the heart contain contractile machinery very similar to that of skeletal muscle, except that they are mono- and bi-nucleated cells. In each cell, the end sarcomeres insert into structures at the plasma membrane called intercalated disks, which link
![li3434542480e](li3434542480e.jpg)
the cells into a contractile chain. Heart muscle cells are generated only early in life, so they cannot be replaced in response to damage, such as occurs during a heart attack. Many different mutations in proteins of the heart contractile machinery

![li3444542480e](li3444542480e.jpg)

FIGURE 17-31 The sliding-filament model of contraction in skeletal muscle. The arrangement of thick myosin and thin actin filaments in the relaxed state is shown in the top diagram. In the presence of ATP and $\mathrm{Ca}^{2+}$, the myosin heads extending from the thick filaments walk toward the $(+)$ ends of the thin filaments. Because the thin filaments are anchored at the Z disks (purple), movement of myosin pulls the actin filaments toward the center of the sarcomere, shortening its length in the contracted state, as shown in the bottom diagram.
give rise to hypertrophic cardiomyopathies-thickening of the heart wall muscle, which compromises its function. For example, many mutations have been documented in the cardiac myosin heavy-chain gene that compromise the protein's contractile function even in heterozygous individuals. In such individuals, the heart tries to compensate by hypertrophy (enlargement), which often results in fatal heart arrhythmia (irregular beating). In addition to myosin heavy-chain defects, defects that result in cardiomyopathies have been traced to mutations in other components of the contractile machinery, including actin, myosin light chains, tropomyosin and troponin, and structural components such as titin (discussed below).

## Skeletal Muscle Is Structured by Stabilizing and Scaffolding Proteins

The structure of the sarcomere is maintained by a number of accessory proteins (Figure 17-32). The actin filaments are stabilized on their (+) ends by CapZ and on their (-) ends by tropomodulin. A giant protein known as nebulin extends along the thin actin filament all the way from the Z disk to tropomodulin, to which it binds. Nebulin consists of repeating domains that bind to the actin in the filament, and it is believed that the number of actin-binding repeats, and therefore the length of nebulin, determines the length of the thin filaments. Another giant protein, called titin (because it is so large), has its head associated with the Z disk and extends to the middle of the thick filament, where another titin molecule extends to the subsequent Z disk. Titin is believed
![li3454542480e](li3454542480e.jpg)

FIGURE 17-32 Accessory proteins found in skeletal muscle. To stabilize the actin filaments, CapZ caps the (+) end of the thin filaments at the Z disk, whereas tropomodulin caps the (-) end. The giant protein titin extends through the thick filaments and attaches to the Z disk. Nebulin binds actin subunits and determines the length of the thin filament.
to be an elastic molecule that holds the thick filaments in the middle of the sarcomere and also prevents overstretching to ensure that the thick filaments remain interdigitated between the thin filaments.

## Contraction of Skeletal Muscle Is Regulated by $\mathrm{Ca}^{2+}$ and Actin-Binding Proteins

Like many cellular processes, skeletal muscle contraction is initiated by an increase in the cytosolic $\mathrm{Ca}^{2+}$ concentration. As described in Chapter 11, the $\mathrm{Ca}^{2+}$ concentration of the cytosol is normally kept low, below $0.1 \mu \mathrm{M}$. In skeletal muscle cells, a low cytosolic $\mathrm{Ca}^{2+}$ level is maintained primarily by a unique $\mathrm{Ca}^{2+}$ ATPase that continually pumps $\mathrm{Ca}^{2+}$ ions from the cytosol containing the myofibrils into the sarcoplasmic reticulum (SR), a specialized endoplasmic reticulum of muscle cells (Figure 17-33). This activity establishes a reservoir of $\mathrm{Ca}^{2+}$ in the SR.

The arrival of a nerve impulse (or action potential; see Chapter 22) at a neuromuscular junction triggers an action potential in the muscle-cell plasma membrane (also known as the sarcolemma). The action potential travels down invaginations of the plasma membrane known as transverse tubules, which penetrate the cell to lie around each myofibril. The arrival of the action potential in the transverse tubules stimulates the opening of voltage-gated $\mathrm{Ca}^{2+}$ channels in the SR membrane, and the ensuing release of $\mathrm{Ca}^{2+}$ from the SR raises the cytosolic $\mathrm{Ca}^{2+}$ concentration in the myofibrils. This elevated $\mathrm{Ca}^{2+}$ concentration induces changes in two accessory proteins, tropomyosin and troponin, which are bound to the actin thin filaments and normally block myosin binding. Changes in the positions of these proteins on the actin thin filaments in turn permit the myosin-actin

(a)
![li3464542480e](li3464542480e.jpg)

FIGURE 17-33 The sarcoplasmic reticulum regulates the level of free $\mathrm{Ca}^{2+}$ in myofibrils. (a) When a nerve impulse stimulates a muscle cell, the action potential is transmitted down a transverse tubule (yellow), which is continuous with the plasma membrane (sarcolemma), leading
interactions and hence contraction. This type of regulation is very rapid and is known as thin-filament regulation.

Tropomyosin (TM) is a ropelike molecule, about 40 nm in length, that binds to seven actin subunits in an actin
![li3474542480e](li3474542480e.jpg)

FIGURE 17-34 $\mathrm{Ca}^{2+}$-dependent thin-filament regulation of skeletal muscle contraction. (a) Model and the corresponding structure of the tropomyosin-troponin regulatory complex on a thin filament. Troponin is a protein complex that is bound to the long $\alpha$-helical tropomyosin molecule. (b) Three-dimensional electron-microscope reconstructions of the tropomyosin helix (yellow) on a muscle thin filament. Tropomyosin in the relaxed state (top) shifts to a new position (arrows) in the state inducing contraction (bottom) when the $\mathrm{Ca}^{2+}$ concentration in the sarcoplasm increases. This movement exposes myosin-binding sites (red) on actin. (Troponin is not shown in this representation, but it remains bound to tropomyosin in both states.) (c) Summary of the regulation of skeletal muscle contraction by $\mathrm{Ca}^{2+}$ binding to troponin. [Part (a) data from S. Wu et al., 2012, Plos One 7:39422, PDB ID 2w4u. Part (b) ©1993 from Lehman et al., J. Cell Biol. 123:313-321. doi: 10.1083/jcb.123.2.313]
(b)
![li3484542480e](li3484542480e.jpg)
to release of $\mathrm{Ca}^{2+}$ from the adjacent sarcoplasmic reticulum (blue) into the myofibrils. (b) Thin-section electron micrograph of skeletal muscle, showing the intimate relationship of the sarcoplasmic reticulum to the muscle fibers. [Part (b) courtesy of Keith R. Porter and Clara Franzini-Armstrong.]
filament. TM molecules are strung together head to tail, forming a continuous chain along each side of the actin thin filament (Figure 17-34a, b). Associated with each tropomyosin molecule is troponin (TN), a complex of three
(b)
![li3494542480e](li3494542480e.jpg)
(c)

## Relaxation

Myosin-binding site masked
![li3504542480e](li3504542480e.jpg)

Myosin-binding site exposed Contraction

subunits, TN-T, TN-I, and TN-C. TN-C is the calciumbinding subunit of troponin; it controls the position of TM on the surface of an actin filament through the TN-I and TN-T subunits.

Under the control of $\mathrm{Ca}^{2+}$ and TN , TM can occupy either of two positions on a thin filament. In the absence of $\mathrm{Ca}^{2+}$, TM blocks myosin's interaction with F-actin, and the muscle is relaxed. Binding of $\mathrm{Ca}^{2+}$ ions to TN-C triggers movement of TM to a new position on the filament, thereby exposing the myosin-binding sites on actin (see Figure 17-34b). Thus, at $\mathrm{Ca}^{2+}$ concentrations greater than $1 \mu \mathrm{M}$, the inhibition exerted by the TM-TN complex is relieved, and contraction occurs. The $\mathrm{Ca}^{2+}$-dependent cycling between relaxation and contraction states in skeletal muscle is summarized in Figure 17-34c.

5
Heart muscle, like skeletal muscle, is subject to thinfilament regulation, using cardiac-specific tropomyosin and troponin. During a heart attack (myocardial infarction), heart cells are deprived of sufficient oxygen (cardiac ischemia) and may subsequently die. The plasma membrane of the dead cells ruptures and releases cellular components into the bloodstream. Blood tests that specifically measure the level of cardiac-specific troponins are used by physicians to determine the severity of a heart attack.

## Actin and Myosin II Form Contractile Bundles in Nonmuscle Cells

In skeletal muscle, as we have seen, actin thin filaments and myosin II thick filaments assemble into contractile structures. Nonmuscle cells contain several types of related contractile bundles composed of actin and myosin II filaments, which are similar to skeletal muscle fibers but much less organized. Moreover, they lack the troponin regulatory system
and are instead regulated by myosin phosphorylation, as we discuss below.

In epithelial cells, contractile bundles are most commonly found as an adherens belt, also known as the circumferential belt, that encircles the inner surface of the cell at the level of the adherens junction (see Figure 17-4a). These bundles are important in maintaining the integrity of the epithelium (discussed in Chapter 20). Stress fibers, which are seen along the lower surfaces of cells cultured on artificial (glass or plastic) surfaces or on extracellular matrices, are a second type of contractile bundle (see Figure 17-4a, c) important in cell adhesion, especially on deformable substrata. The ends of stress fibers terminate at integrin-containing focal adhesions, special structures that attach a cell to the underlying substratum (see Figure 17-40 below and Chapter 20). Circumferential belts and stress fibers contain several proteins found in the contractile apparatus of smooth muscle and exhibit some organizational features resembling those of muscle sarcomeres. A third type of contractile bundle, referred to as a contractile ring, is a transient structure in animal cells that assembles at the equator of a dividing cell, encircling the cell midway between the poles of the mitotic spindle (Figure 17-35a). As the ring contracts, pulling the plasma membrane in, the cytoplasm is divided and eventually pinched into two parts in a process known as cytokinesis, giving rise to two daughter cells. Dividing cells stained with antibodies against myosin I and myosin II show that myosin II is localized to the contractile ring, whereas myosin I is at the distal regions, where it links the cell cortex to the plasma membrane (Figure 17-35b). Cells from which the gene encoding the heavy chain of myosin II has been deleted are unable to undergo cytokinesis. Instead, these cells form a multinucleated syncytium because cytokinesis, but not nuclear division, is inhibited.
(a)
![li3514542480e](li3514542480e.jpg)
(b)
![li3524542480e](li3524542480e.jpg)

EXPERIMENTAL FIGURE 17-35 Fluorescent antibodies reveal the localization of myosin I and myosin II during cytokinesis. (a) Diagram of a cell going through cytokinesis, showing the mitotic spindle (microtubules green, chromosomes blue) and the contractile ring with
actin filaments (red). (b) Fluorescence micrograph of a Dictyostelium amoeba expressing GFP-myosin-II reveals myosin-II enrichment in the cleavage furrow cortex, as the cell pinches into two during cytokinesis. [Part (b) courtesy of Douglas N. Robinson]

## Myosin-Dependent Mechanisms Regulate Contraction in Smooth Muscle and Nonmuscle Cells

Smooth muscle is a specialized tissue composed of contractile cells that is found in many internal organs. For example, smooth muscle surrounds blood vessels to regulate blood pressure, surrounds the intestine to move food through the gut, and restricts airway passages in the lung. Smooth muscle cells contain large, loosely aligned contractile bundles that resemble those in epithelial cells. The contractile apparatus of smooth muscle and its regulation constitute a valuable model, as its contractile activity is regulated in a manner similar to that in nonmuscle cells. As we have just seen, skeletal muscle contraction is regulated by the switching of the tropomyosin-troponin complex bound to the actin thin filament between the contraction-inducing state in the presence of $\mathrm{Ca}^{2+}$ and the relaxed state in its absence. In contrast, smooth muscle contraction is regulated by the cycling of myosin II between on and off states. Myosin II cycling, and thus contraction of smooth muscle and nonmuscle cells, is regulated in response to many extracellular signaling molecules.

Contraction of vertebrate smooth muscle is regulated primarily by a pathway in which the myosin regulatory light chain (LC) associated with the myosin II neck domain (see Figure 17-22b) undergoes phosphorylation and dephosphorylation. When the regulatory LC is not phosphorylated, the smooth muscle myosin II adopts a folded conformation, and its ATPase cycle is inactive. When the regulatory LC is phosphorylated by the enzyme myosin light-chain kinase (MLC kinase), whose activity is regulated by the level of cytosolic free $\mathrm{Ca}^{2+}$, the myosin II unfolds, assembles into active bipolar filaments, and becomes active to induce contraction (Figure 17-36). The $\mathrm{Ca}^{2+}$-dependent regulation of MLC kinase activity is mediated through the $\mathrm{Ca}^{2+}$-binding protein calmodulin (see Figure 3-33). Calcium first binds to calmodulin, which induces a conformational change in the protein, and the $\mathrm{Ca}^{2+} /$ calmodulin complex then binds to MLC kinase and activates it. When the $\mathrm{Ca}^{2+}$ returns to its resting level, MLC kinase becomes inactive, and myosin light-chain phosphatase removes the phosphates to allow the system to return to its relaxed state. This mode of regulation relies on the diffusion of $\mathrm{Ca}^{2+}$ over greater distances than in sarcomeres and on the action of protein kinases, so contraction is much slower in smooth muscle than in skeletal muscle. Because this regulation involves myosin, it is known as thick-filament regulation.

Unlike skeletal muscle, which is stimulated to contract solely by nerve impulses, smooth muscle cells and nonmuscle cells are regulated by many types of external signals. For example, norepinephrine, angiotensin, endothelin, histamine, and other signaling molecules can modulate or induce the contraction of smooth muscle or elicit changes in the shape and adhesion of nonmuscle cells by triggering various signal transduction pathways. Some of these pathways lead to an increase in the cytosolic $\mathrm{Ca}^{2+}$ concentration; as

![li3534542480e](li3534542480e.jpg)

FIGURE 17-36 Myosin light-chain phosphorylation regulates smooth muscle contraction. In vertebrate smooth muscle, phosphorylation of the myosin regulatory light chain (LC) activates contraction. At $\mathrm{Ca}^{2+}$ concentrations of less than $10^{-6} \mathrm{M}$, the regulatory light chain is not phosphorylated, and the myosin adopts a folded conformation. When the $\mathrm{Ca}^{2+}$ level rises, $\mathrm{Ca}^{2+}$ binds calmodulin (CaM), which undergoes a conformational change $\left(\mathrm{CaM}^{*}\right)$. The $\mathrm{CaM}^{*}-\mathrm{Ca}^{2+}$ complex binds and activates myosin light-chain kinase (MLC kinase), which then phosphorylates the myosin LC. This phosphorylation event unfolds the myosin II, which is now active and can assemble into bipolar filaments to participate in contraction. When the $\mathrm{Ca}^{2+}$ levels drop, the myosin LC is dephosphorylated by myosin light-chain (MLC) phosphatase, which is not dependent on $\mathrm{Ca}^{2+}$ for activity, causing muscle relaxation.
previously described, this increase can stimulate myosin activity by activating MLC kinase (see Figure 17-36). As we will see below, other pathways activate Rho kinase, which is also able to activate myosin activity by phosphorylating the regulatory light chain, although in a $\mathrm{Ca}^{2+}$-independent manner.

## Myosin V-Bound Vesicles Are Carried Along Actin Filaments

In contrast to the contractile functions of myosin II filaments, the myosin V family of proteins, the most processive myosin motors known, transport cargo along actin filaments. In the next chapter we discuss how they can work together with microtubule motors to bring about transport of organelles. Although little is known about their functions

![li3544542480e](li3544542480e.jpg)
![li3554542480e](li3554542480e.jpg)

FIGURE 17-37 Cargo movement by myosin V. (a) The yeast Saccharomyces cerevisiae (used in making bread, beer, and wine) grows by budding. Secretory vesicles are transported into the bud, which swells to about the size of the mother cell. Before cell division can occur, all the organelles must be segregated between the mother and the bud. The cell then undergoes cytokinesis to form two daughter cells. As shown in (b), all these transport processes depend on myosin V. This diagram of a medium-sized bud shows how myosin V transports secretory vesicles (SV) down actin cables nucleated by formins (purple) located at the bud tip and bud neck. Myosin V is also used to segregate organelles such as the vacuole (the yeast equivalent of a lysosome), peroxisomes, mitochondria, endoplasmic reticulum (ER), trans-Golgi network (TGN), and even selected mRNAs into the bud. Myosin V also binds the ends of cytoplasmic microtubules (green) to orient the nucleus in preparation for mitosis. See D. Pruyne et al., 2004, Ann. Rev. Cell Dev. Biol. 20:559. (c) Myosin V is regulated by binding cargo. In the inactive folded state, the tails of myosin V bind and inactivate the motor head domains. Upon cargo binding, the head-to-tail interaction is alleviated, and the motor is now active to transport cargo. (d) Electron micrograph of inactive myosin V. Myosin V molecules were negatively stained and viewed in a transmission electron microscope. The image shown is the composite average from several individual images. (Part (d) reprinted by permission from Macmillan Publishers Ltd: from Thirumurugan, K. et al., "The cargo-binding domain regulates structure and activity of myosin 5," Nature, 2006, 442;70998:212-215.)
in mammalian cells, myosin V motors are not unimportant: defects in a specific myosin V protein can cause severe diseases, such as seizures (see Figure 17-24).

Much more is known about myosin V motors in more experimentally accessible and simpler systems such as the budding yeast. This well-studied organism grows by budding, which requires its secretory machinery to target newly synthesized materials to the growing bud (Figure 17-37a). Myosin V transports secretory vesicles along actin filaments at $3 \mu \mathrm{~m} / \mathrm{s}$ into the bud. However, this is not the only function of myosin V proteins in yeast. At a later stage of the cell cycle, all the organelles have to be distributed between the mother and daughter cells. Remarkably, myosin V molecules in yeast provide the transport system for segregation of many organelles, including peroxisomes, mitochondria,
vacuoles, endoplasmic reticulum, the trans-Golgi network, and even transport the ends of microtubules and some specific messenger RNAs into the bud (Figure 17-37b). Each of these organelles has a receptor to which the myosin V binds. The motor makes many delivery cycles, so it has to have a way to pick up, transport, and then deliver its organelle cargo. Recent work has shown that myosin V can exist in an inactive folded state in which the tail binds and inhibits the activity of the motor head domain, and an active state in which binding to cargo relieves the head-to-tail interaction (Figure 17-37c,d). How the motor releases its cargo upon delivery is poorly understood-in one case, the organelle receptor becomes degraded upon delivery of the organelle to its destination. Whereas budding yeast uses myosin V and polarized actin filaments in the transport of

![li3564542480e](li3564542480e.jpg)

FIGURE 17-38 Cytoplasmic streaming in cylindrical giant algae. (a) Cells of Nitella, a freshwater alga commonly found in ponds in the summer. The cytoplasmic movement, described below, can be readily observed with a simple microscope, so go find some Nitella (or related algae) and watch this amazing phenomenon! (b) The center of a Nitella cell is filled with a single large water-filled vacuole, which is surrounded by a layer of moving cytoplasm (blue arrows). A nonmoving layer of cortical cytoplasm filled with chloroplasts lies just under the plasma membrane (enlarged in bottom figure). On the inner side of this layer
![li3574542480e](li3574542480e.jpg)
are bundles of stationary actin filaments (red), all oriented with the same polarity. A motor protein (blue), a plant myosin V, carries parts of the endoplasmic reticulum (ER) along the actin filaments. The movement of the ER network propels the entire viscous cytoplasm, including organelles that are enmeshed in the ER network. (c) Electron micrograph of the cortical cytoplasm showing a large vesicle connected to an underlying bundle of actin filaments. (Part (a) courtesy of James C. French; part (c) courtesy of Bechara Kachar.)
many organelles, animal cells, which are much larger, employ microtubules and their motors to transport many of the same organelles over the relatively longer distances involved. We discuss those transport mechanisms in the next chapter.

Perhaps the most dramatic use of myosin V is seen in giant green algae, such as Nitella and Chara. These algae can be readily found in ponds during the summer and their movement easily observed using a simple microscope. In their large cells, which can be as much as 2 cm in length, cytosol flows rapidly, at a rate approaching 4.5 mm per
minute, in an endless loop around the inner circumference of the cell (Figure 17-38). This cytoplasmic streaming is a principal mechanism for distributing cellular metabolites, especially in large cells such as plant cells and amoebae. The algal cells have bundles of actin filaments aligned along the length of the cell, lying just above the stationary chloroplasts located adjacent to the membrane. The bulk cytosol is propelled by myosin V (also known as myosin XI in plants) attached to parts of the ER adjacent to the actin filaments. The flow rate of the cytosol in Nitella is about 15 times as fast as the movement produced by any other known myosin.

## KEY CONCEPTS OF SECTION 17.6

## Myosin-Powered Movements

- In skeletal muscle, contractile myofibrils are composed of thousands of repeating units called sarcomeres. Each sarcomere consists of two interdigitating filament types: myosin thick filaments and actin thin filaments (see Figure 17-30).
- Skeletal muscle contraction involves the ATP-dependent sliding of myosin thick filaments along actin thin filaments to shorten the sarcomere and hence the myofibril (see Figure 17-31).
- The ends of the actin thin filaments in skeletal muscle are stabilized by CapZ at the $(+)$ end and by tropomodulin at the $(-)$ end. Two large proteins, nebulin associated with the thin filaments and titin with the thick filaments, also contribute to skeletal muscle organization.
- Skeletal muscle contraction is subject to thin-filament regulation. At low levels of free $\mathrm{Ca}^{2+}$, tropomyosin blocks the interaction of myosin and F-actin, and the muscle is relaxed. At elevated levels of free $\mathrm{Ca}^{2+}$, the troponin complex associated with tropomyosin binds $\mathrm{Ca}^{2+}$ and moves the tropomyosin to uncover the myosin-binding sites on actin, allowing contraction (see Figure 17-34).
- Smooth muscle and nonmuscle cells have contractile bundles of actin and myosin filaments with an organization similar to that in skeletal muscle but less well ordered.
- Contractile bundles are subject to thick-filament regulation. A myosin light chain is phosphorylated by myosin light-chain kinase, which activates myosin and hence induces contraction. The MLC kinase is activated by binding of $\mathrm{Ca}^{2+}$-calmodulin when the free $\mathrm{Ca}^{2+}$ concentration rises (see Figure 17-36).
- Inactive myosin V is activated by binding cargo, which it transports processively along actin filaments.


### 17.7 Cell Migration: Mechanism, Signaling, and Chemotaxis

We have now examined the different mechanisms used by cells to create movement-from the assembly of actin filaments and the formation of actin filament bundles and networks to the contraction of bundles of actin and myosin and the transport of organelles by myosin molecules along actin filaments. Some of these same mechanisms constitute the major processes whereby cells generate the forces needed to migrate. Cell migration results from the coordination of motions generated in different parts of a cell, integrated with a directed endocytic cycle.

The study of cell migration is important to many fields of biology and medicine. For example, an essential feature of animal development is the migration of specific cells along predetermined paths. Epithelial cells in an adult animal
migrate to heal a wound, and white blood cells migrate to sites of infection. Less obvious are the continuous slow migration of intestinal epithelial cells along the villi in the intestine and of endothelial cells that line the blood vessels. The inappropriate migration of cancer cells after breaking away from their normal tissue results in metastasis.

Cell migration is initiated by the formation of a large, broad membrane protrusion at the leading edge of a cell. Video microscopy reveals that a major feature of this movement is the polymerization of actin at the membrane. Actin filaments at the leading edge are rapidly cross-linked into bundles and networks in a protruding region, called a lamellipodium in vertebrate cells. In some cases, slender, fingerlike membrane projections, called filopodia, also extend from the leading edge. These structures form stable contacts with the underlying surface (such as the extracellular matrix) that the cell moves across. In this section, we take a closer look at how cells coordinate various microfilament-based processes with endocytosis to move across a surface. We also consider the role of signaling pathways in coordinating and integrating the actions of the cytoskeleton, a major focus of current research.

## Cell Migration Coordinates Force Generation with Cell Adhesion and Membrane Recycling

A moving fibroblast (connective tissue cell) displays a characteristic sequence of events: initial extension of a membrane protrusion, attachment to the substratum, forward flow of cytosol, and retraction of the rear of the cell (Figure 17-39). These events occur in an ordered pattern in a slowly moving cell such as a fibroblast, but in rapidly moving cells, such as macrophages, all of them are occurring simultaneously in a coordinated manner. We first consider the role of the actin cytoskeleton in cell movement, involving assembly at the leading edge as well as attachment to the substratum via stress fibers (Figure 17-40a, b), and then discuss the role of the endocytic cycle.

Membrane Extension The network of actin filaments at the leading edge is a type of cellular engine that pushes the membrane forward in a manner very similar to the propulsion of Listeria by actin polymerization (Figure 17-40d; for Listeria, see Figure 17-17c). At the membrane of the leading edge, actin is nucleated by the activated Arp2/3 complex, and filaments are elongated by assembly onto $(+)$ ends adjacent to the plasma membrane. Because the actin network is fixed with respect to the substratum, the front membrane is pushed out as the filaments elongate. This process is very similar to the movement of Listeria, which "rides" on the polymerizing actin tail, which is also fixed within the cytoplasm. Actin turnover, and thus treadmilling, is mediated, as it is in the comet tails of Listeria, by the action of profilin and cofilin (see Figure 17-40d).

Cell-Substratum Adhesions When the membrane has been extended and the actin network has been assembled, the

FIGURE 17-39 Steps in cell locomotion. Movement begins with the extension of one or more lamellipodia from the leading edge of the cell $\mathbf{\square}$ some lamellipodia adhere to the substratum by focal adhesions $\square$ Then the bulk of the cytoplasm in the cell body flows forward due to contraction at the rear of the cell $\boldsymbol{\square}$. The trailing edge of the cell remains attached to the substratum until the tail eventually detaches and retracts into the cell body. During this cytoskeleton-based cycle, the endocytic cycle internalizes membrane and integrins at the rear of the cell and transports them to the front of the cell (arrows) for reuse in making new adhesions $\boldsymbol{\square}$
plasma membrane becomes firmly attached to the substratum. Time-lapse microscopy shows that actin bundles in the leading edge become anchored to structures known as focal adhesions (Figure 17-40c). The attachment serves two purposes: it prevents the lamellipodium from retracting, and it attaches the cell to the substratum, allowing the cell to move forward. Given the importance of focal adhesions and their regulation during cell locomotion, it is not surprising that they have been found to be very rich in molecules involved in signal transduction pathways. Focal adhesions are discussed in more detail in Chapter 20, where we discuss cell-matrix interactions.

The cell-adhesion molecules that mediate most cellmatrix interactions are membrane proteins called integrins. These proteins have an external domain that binds to specific components of the extracellular matrix, such as fibronectin and collagen, and a cytoplasmic domain that links them to the actin cytoskeleton (see Figure 17-40c and Chapter 20). The cell makes adhesions at the leading edge, and as the cell migrates forward, the adhesions eventually assume positions toward the rear.

Cell-Body Translocation After the forward adhesions have been made, the bulk contents of the cell body are translocated forward (see Figure 17-39, step $\mathbf{\square}$ ). It is believed that the nucleus and the other organelles embedded in the cytoskeleton are moved forward by myosin II-dependent cortical contraction in the rear part of the cell, like toothpaste when the lower half of the tube is squeezed. Consistent with this model, myosin II is localized to the rear cell cortex.

Breaking Cell Attachments Finally, in the last step of movement (de-adhesion), the focal adhesions at the rear of the cell are broken, the integrins recycled, and the freed tail brought forward. In the light microscope, the tail is often seen to "snap" loose from its connections-perhaps by the contraction of stress fibers in the tail or by elastic tension-and it sometimes leaves a little bit of its membrane behind, still firmly attached to the substratum.

Cells cannot move if they are either too strongly attached or not attached to a surface. The ability of a cell to move
![li3584542480e](li3584542480e.jpg)
corresponds to a balance between the mechanical forces generated by the cytoskeleton and the resisting forces generated by cell adhesions. This relationship can be demonstrated by measuring the rate of movement in cells that express varying levels of integrins. Such measurements show that the fastest migration occurs at an intermediate level of adhesion, with the rate of movement falling off at high and low levels of adhesion. Cell locomotion thus results from traction forces exerted by the cell on the underlying substratum.

Recycling of Membrane and Integrins by Endocytosis The dynamic changes in the actin cytoskeleton alone are not sufficient to drive cell migration; it is also dependent on endocytic recycling of membranes. The membrane needed during lamellipodium extension is provided by internal endosomes following their exocytosis. Adhesion molecules in focal adhesions at the rear of the cell are internalized as those adhesions are disassembled and transported by an endocytic cycle to the front to make new substratum attachments (Figure 17-39, step $\mathbf{\square}$ ). This cycling of adhesion molecules in a migrating cell resembles the way a tank uses its treads to move forward. The movement of membrane internally through the cell also generates a rearward membrane flow across the surface of the cell. Indeed, this type of flow may contribute to the mechanics of cell locomotion, as it has recently been found that white blood cells can move in a liquid ("swim") in the absence of attachment to a substratum, presumably as surface structures operating like paddles move backward across the cell surface.

![li3594542480e](li3594542480e.jpg)

FIGURE 17-40 Actin-based structures involved in cell locomotion. (a) Localization of actin in a fibroblast expressing GFP-actin. (b) Diagram of the classes of microfilaments involved in cell migration. The network of actin filaments in the leading edge advances the cell forward. Contractile fibers in the cell cortex squeeze the cell body forward, and stress fibers terminating in focal adhesions also pull the bulk of the cell body up as the rear adhesions are released.

## The Small GTP-Binding Proteins Cdc42, Rac, and Rho Control Actin Organization

A striking feature of a moving cell is its polarity: it has a front and a back. When a cell makes a turn, a new leading edge forms in the new direction of movement. If these extensions formed in all directions at once, the cell would be unable to pick a new direction of movement. To sustain movement in a particular direction, a cell requires signals to coordinate events at the front of the cell with events at the back and, indeed, signals to tell the cell where its front is. Our understanding of how this coordination occurs emerged from studies with growth factors.

Growth factors, such as epidermal growth factor (EGF) and platelet-derived growth factor (PDGF), bind to specific cell-surface receptors (see Chapter 16) and stimulate cells to
(b)
![li3604542480e](li3604542480e.jpg)
(c) The structure of focal adhesions involves the attachment of the ends of stress fibers through integrins to the underlying extracellular matrix. Focal adhesions also contain many signaling molecules important for cell locomotion. (d) The dynamic actin network in the leading edge is nucleated by the Arp2/3 complex and employs the same set of factors that control assembly and disassembly of actin filaments in the Listeria tail (see Figure 17-17). [Part (a) courtesy of J. Vic Small.]
move and then to divide. For example, in a wound, blood platelets become activated by being exposed to collagen in the extracellular matrix at the wound edge, which helps the blood to clot. Activated platelets also secrete PDGF to attract fibroblasts and epithelial cells to enter the wound and repair it. It is possible to watch part of this process in vitro. If you grow cells in a culture dish and, after starving them of growth factors, you add some fresh growth factor, within a minute or two the cells respond by forming membrane ruffles. Membrane ruffles are very similar to the lamellipodia of migrating cells: they are a result of activation of the machinery that controls exocytosis of endosomes coupled with actin assembly.

Scientists knew that growth factors bind to very specific receptors on the cell surface and induce a signal transduction

pathway on the inner surface of the plasma membrane (see Chapter 15), but how that process linked up to the actin machinery was mysterious. Research then revealed that the signal transduction pathway activates Rac, a member of the small GTPase superfamily of Ras-related proteins (see Chapter 15). Rac is one member of a family of proteins that regulate microfilament organization; two others are Cdc42 and Rho. Unfortunately, due to the history of their discovery, the family of proteins of which Cdc42, Rac, and Rho are members has also been collectively named "Rho proteins." To understand how these proteins work, we first have to recall the way small GTP-binding proteins function.

Like all small GTPases of the Ras superfamily, Cdc42, Rac, and Rho act as molecular switches, inactive in the GDP-bound state and active in the GTP-bound state (Figure 17-41). In their GDP-bound state, they exist free in the cytoplasm in an inactive form bound to a protein known as guanine nucleotide dissociation inhibitor (GDI). Growth factors can bind and activate their receptors to turn on specific membrane-bound regulatory proteins, called guanine nucleotide exchange factors (GEFs), which activate Rho proteins at the membrane by releasing them from GDI and catalyzing the exchange of GDP for GTP. The GTP-bound active Rho protein associates with the plasma membrane, where it binds effector proteins to transmit the biological response. The small GTPase remains active until the GTP is hydrolyzed to GDP, a process that is stimulated by specific GTPase-activating proteins (GAPs).

An important approach to unraveling the functions of the Rho proteins has been to introduce into cells mutant proteins
![li3614542480e](li3614542480e.jpg)

FIGURE 17-41 Regulation of the Rho family of small GTPases. The small GTPases of the Rho family are molecular switches regulated by accessory proteins. Rho proteins exist in the Rho-GDP bound form complexed with a protein known as GDI (guanine nucleotide dissociation inhibitor), which keeps them in an inactive state in the cytosol. Membrane-bound signaling pathways bring Rho proteins to the membrane and, through the action of a GEF (guanine nucleotide exchange factor), exchange the GDP for GTP, thus activating them. Membrane-bound activated Rho-GTP can then bind effector proteins that cause changes in the actin cytoskeleton. The Rho protein remains in the active Rho-GTP state until acted on by a GAP (GTPase-activating protein), which allows it to interact with the GDI and be returned to the cytoplasm. See S. Etienne-Manneville and A. Hall, 2002, Nature 420:629.
that are locked either in the active-Rho-GTP—state or in the inactive-Rho-GDP—state. A mutant small GTPase that is locked in the active state, called a dominant-active protein, binds the effector molecules constitutively, and one can then assess the biological outcomes. Alternatively, one can introduce a different mutant that is dominant negative, which binds and inhibits the relevant GEF protein. Thus introduction of a dominant-negative protein, even in the presence of the endogenous wild-type protein, interferes with the signal transduction pathway, so one can now assess what processes are blocked.

Cdc42, Rac, and Rho were implicated in the regulation of microfilament organization because introduction of dominant-active mutant proteins had dramatic effects on the actin cytoskeleton, even in the absence of growth factors. It was discovered that dominant-active Cdc42 results in the appearance of filopodia, dominant-active Rac results in the appearance of membrane ruffles, and dominant-active Rho results in the formation of stress fibers, which then contract (Figure 17-42). How can one tell if dominant-active Rac and growth factor stimulation, both of which stimulate
![li3624542480e](li3624542480e.jpg)

Dominant-active Rho
![li3634542480e](li3634542480e.jpg)

Dominant-active Rac
![li3644542480e](li3644542480e.jpg)

Dominant-active Cdc42

EXPERIMENTAL FIGURE 17-42 Dominant-active Rac, Rho, and Cdc42 induce different actin-containing structures. To look at the effects of constitutively active Rac, Rho, and Cdc42, growth-factor-starved fibroblasts were microinjected with plasmids to express dominant-active versions of the three proteins. The cells were then treated with fluorescent phalloidin, which stains filamentous actin. Dominant-active Rac induces the formation of peripheral membrane ruffles, whereas dominant-active Rho induces abundant contractile stress fibers, and dominant-active Cdc42 induces filopodia. [Republished with permission of AAAS, from Hall, A., "Rho GTPases and the actin cytoskeleton," Science, 1998, 279 (5350):509-14; permission conveyed through Copyright Clearance Center, Inc.]

![li3654542480e](li3654542480e.jpg)

FIGURE 17-43 Summary of signal-induced changes in the actin cytoskeleton. Specific signals, such as growth factors and lysophosphatidic acid (LPA), are detected by cell-surface receptors. Detection leads to the activation of the small GTP-binding proteins, which then interact with effectors to bring about cytoskeletal changes as indicated.
membrane ruffle formation, operate in the same signal transduction pathway? If growth factor stimulation leads to Rac activation, then introduction of a dominant-negative Rac protein into a cell should block the ability of a growth factor to induce membrane ruffling. This is precisely what was found. Using this and many other biochemical strategies, scientists have identified the signaling pathways involving Cdc42, Rac, and Rho (Figure 17-43).

Some of the pathways that these proteins regulate contain proteins we are familiar with. Activation of Cdc42 stimulates actin assembly by Arp2/3 through activation of WASp, a nucleation promoting factor (NPF) (see Figure 17-16), resulting in the formation of filopodia. Activation of Rac also induces Arp2/3, mediated by the WAVE complex, leading to the assembly of branched actin filaments in the leading edge. Activation of Rho has at least two effects. First, it can activate a formin for unbranched actin filament assembly. Second, through activation of Rho kinase, it can phosphorylate the myosin light chain to activate nonmuscle myosin II and can also inhibit light-chain dephosphorylation by phosphorylating myosin light-chain phosphatase to inhibit its activity. Both actions of Rho kinase lead to a higher level of myosin light-chain phosphorylation and therefore higher myosin activity and contraction. The three Rho proteins, Cdc42, Rac, and Rho, are also linked by activation and inhibition pathways, as shown in Figure 17-43.

## Cell Migration Involves the Coordinate Regulation of Cdc42, Rac, and Rho

How does each of these small GTP-binding proteins contribute to the regulation of cell migration? To answer this question, researchers developed an in vitro wound-healing
assay (Figure 17-44a). Cells in culture are grown in a petri dish with growth factors until they are confluent and form a tight monolayer, at which point they stop dividing. The cell monolayer is then scratched with a needle to remove a swath of cells, generating a "wound" containing a free edge of cells. The cells on the edge sense the loss of their neighbors and, in response to components of the extracellular matrix now exposed on the dish surface, move to fill up the empty wound area (Figure 17-44b). To do this, they orient themselves toward the empty area, first extending a lamellipodium and then moving in that direction. In this way, one can study the induction of directed cell migration in vitro.

Using this wounded-cell monolayer assay, researchers have introduced dominant-negative Rac into cells on the wound edge to see how it affects the ability of the cells to migrate and fill the wound. Since Rac is needed for activation of the Arp2/3 complex to form the lamellipodium, it is not surprising that the cells fail to form this structure and do not migrate, and so the wound does not close (Figure 17-44c). A very interesting result is obtained when dominant-negative Cdc42 is introduced into the cells at the wound edge: they can form a lamellipodium, but they do not orient in the correct direction-in fact, they try to migrate in random directions. This observation suggests that Cdc42 is critical for regulating the overall polarity of the cell. Studies from yeast (in which Cdc42 was first described), wounded-cell monolayers, epithelial cells, and neurons reveal that Cdc42 is a master regulator of polarity in many different systems. Part of this regulation in animals involves the binding of Cdc42 to its effector, Par6, a polarity protein that functions in nematodes (in which it was first discovered), neurons, and epithelial cells. We explore these polarity pathways in more detail in Chapter 21.

![li3664542480e](li3664542480e.jpg)

EXPERIMENTAL FIGURE 17-44 The wounded-cell monolayer assay can be used to dissect signaling pathways in directed cell movement. (a) A confluent layer of cells is scratched to remove a swath about three cells wide to generate a free cell border. The remaining cells detect the free space and newly exposed extracellular matrix and, over a period of hours, fill in the area. (b) Localization of actin in a wounded monolayer 5 minutes and 3 hours after scratching; after 3 hours, cells have migrated into the wounded area. (c) Effect of introducing dominant-negative Cdc42, Rac, and Rho into cells at the wound edge; all affect wound closure. (Part (b) 61999 Nobes \& Hall et al., J. Cell Biol. 144:1235-1244. doi:10.1083/jcb.144.6.1235. Part (c) data from C. D. Nobes and A. Hall, 1999, J. Cell Biol. 144:1235.)

Studies such as these suggest a general model of how cell migration is controlled (Figure 17-45). Signals from the environment are transmitted to Cdc42, which orients the cell. The oriented cell has high Rac activity at the front, which induces the formation of the leading edge; Rho activity is high in the rear, inducing the assembly of contractile structures and activating the myosin-II-based contractile machinery. It is important to notice that different regions of the cell can have different levels of active Cdc42, Rac, or Rho, so these regulators are controlled locally within the cell. Part of this spatial regulation occurs because some small G proteins can work antagonistically. For example, active
![li3674542480e](li3674542480e.jpg)

FIGURE 17-45 Contributions of Cdc42, Rac, and Rho to cell movement. The overall polarity of a migrating cell is controlled by Cdc42, which is activated at the front of a cell. Cdc42 activation leads to active Rac in the front of the cell, which generates the leading edge, and active Rho at the back of the cell, which causes myosin II activation and contraction. Active Rho inhibits Rac activation, ensuring the asymmetry of the two active G-proteins.

Rho can stimulate pathways that lead to the inactivation of Rac. This process might help ensure that no leading-edge structures form at the rear of the cell.

## Migrating Cells Are Steered by Chemotactic Molecules

Under certain conditions, extracellular chemical cues guide the locomotion of a cell in a particular direction. In some cases, the movement is guided by insoluble molecules in the underlying substratum, as in the wounded-cell monolayer assay described above. In other cases, the cell senses soluble molecules and follows them, along a concentration gradient, to their source-a process known as chemotaxis. For example, leukocytes are guided toward an infection by a tripeptide that is secreted by many bacterial cells (Figure 17-46). In another example, during the development of skeletal muscle, a secreted protein signal called scatter factor guides the migration of myoblasts to the proper locations in limb buds. One of the best-studied examples of chemotaxis is the migration of Dictyostelium amoebae during their starvation response. When these soil amoebae are stressed, they begin to secrete cAMP, which is an extracellular chemotactic agent in this organism. Other Dictyostelium cells move up the cAMP concentration gradient toward its source (see Figure 17-46). Thus the amoebae move toward one another, aggregate into a migratory slug, and then differentiate into a fruiting body in which starvation-resistant spores are formed.

Despite the variety of different chemotactic moleculessugars, peptides, cell metabolites, cell-wall or membrane lipids-they all work through a common and familiar mechanism: binding to cell-surface receptors, activation of intracellular signaling pathways, and remodeling of the cytoskeleton through the activation or inhibition of various actin-binding proteins. What is quite amazing is that just a

![li3684542480e](li3684542480e.jpg)

EXPERIMENTAL FIGURE 17-46 Chemotactic molecules guide migrating cells by signaling the actin cytoskeleton. Dictyostelium cells migrate toward a pipette of cAMP (left), and human neutrophils (a type of leukocyte) migrate toward a pipette of fMLP (formylated Met-Leu-Phe), a chemotactic peptide produced by bacteria (right). In the lower two panels are individual chemotaxing Dictyostelium and neutrophil cells, which look remarkably similar, despite about 800 million years of evolution separating them. [Republished with permission of Elsevier, from Parent, C. A., "Making all the right moves: chemotaxis in neutrophils and Dictyostelium," Current Opinion in Cell Biology, 2004;16(1):4-13, 2004; permission conveyed through Copyright Clearance Center, Inc.]

2 percent difference in the concentration of chemotactic molecules between the front and back of the cell is sufficient to induce directed cell migration (see Classic Experiment 17-2). Equally amazing is the finding that the internal signal transduction pathways used in chemotaxis have been conserved between Dictyostelium amoebae and human leukocytes despite almost a billion years of evolution.

## KEY CONCEPTS OF SECTION 17.7

## Cell Migration: Mechanism, Signaling, and Chemotaxis

- Cell migration involves the extension of an actin-rich leading edge at the front of the cell, the formation of focal adhesions that move backward with respect to the cell, and their subsequent release, combined with rear contraction to push the cell forward (see Figure 17-39).
- Cell migration also involves a directed endocytic cycle, taking membrane and adhesion molecules from the rear of the cell and inserting them at the front.
- The assembly and function of actin filaments is controlled by signaling pathways through small GTP-binding proteins of the Rho family. Cdc42 regulates overall polarity and the formation of filopodia, Rac regulates actin network formation through the Arp2/3 complex, and Rho regulates actin filament formation by formins as well as contraction through regulation of myosin II (see Figure 17-43).
- Chemotaxis, the directed movement of cells toward extracellular chemical cues, involves signaling pathways that contribute to the regulation of the actin cytoskeleton and direction of cell migration.


## LaunchPad <br> maomillan learning

Visit LaunchPad to access study tools and to learn more about the content in this chapter.

- Perspectives for the Future
- Classic Experiment 17-1: Looking at Muscle Contraction
- Classic Experiment 17-2: Sensing Chemotactic Gradients
- Analyze the Data
- Extended References
- Additional study tools, including videos, animations, and quizzes


## Key Terms

actin 777
actin cross-linking proteins 793
Arp2/3 complex 786
CapZ protein 785
Cdc42 protein 814
cell migration 811
cell polarity 776
chemotaxis 816
cofilin 784
contractile bundles 807
critical concentration
$\left(C_{\mathrm{c}}\right) \quad 782$
cytoskeleton 777
duty ratio 802
F-actin 779
filopodia 778
formin 786
G-actin 779
intermediate filaments 777
lamellipodium 778
leading edge 778
microfilaments 777
microtubules 777
microvilli 775
motor protein 777
myosin 796
myosin light-chain
kinase 808
nucleation promoting factor
(NPF) 787
power stroke 800
profilin 784
Rac protein 814
Rho protein 814
step size 800
stress fibers 778
thick filaments 804
thin filaments 804
thymosin $\beta_{4} \quad 785$
treadmilling 784
tropomodulin 785
tropomyosin 806
WASp 788

# Review the Concepts 

1. Three systems of cytoskeletal filaments exist in most eukaryotic cells. Compare them in terms of composition, function, and structure.
2. Actin filaments have a defined polarity. What is filament polarity? How is it generated at the subunit level? How is filament polarity detectable?

3. In cells, actin filaments form bundles or networks. How do cells form such structures, and what specifically determines whether actin filaments will form a bundle or a network?
4. Much of our understanding of actin assembly in the cell is derived from experiments using purified actin in vitro. What techniques can be used to study actin assembly in vitro? Explain how each of these techniques works. Which of these techniques would tell you whether the mass of actin filaments is made up of many short actin filaments or fewer longer filaments?
5. The predominant forms of actin inside a cell are ATP-G-actin and ADP-F-actin. Explain how the interconversion of the nucleotide state is coupled to the assembly and disassembly of actin subunits. What would be the consequence for actin filament assembly/disassembly if a mutation prevented actin's ability to bind ATP? What would be the consequence if a mutation prevented actin's ability to hydrolyze ATP?
6. Actin filaments at the leading edge of a crawling cell are believed to undergo treadmilling. What is treadmilling, and what accounts for this assembly behavior?
7. Although purified actin can assemble reversibly in vitro, various actin-binding proteins regulate the assembly of actin filaments in the cell. Predict the effect on a cell's actin cytoskeleton if function-blocking antibodies against each of the following were independently microinjected into cells: profilin, thymosin- $\beta_{4}$, CapZ, and the Arp2/3 complex.
8. Predict how actin would polymerize on a myosindecorated short actin filament (as shown in Figure 17-9) in the presence of CapZ, tropomodulin, or profilin-actin.
9. Compare and contrast the ways in which formin and WASp are activated and explain how each stimulates actin filament formation.
10. There are at least 20 different types of myosin. What properties do all types share, and what makes them different? Why is myosin II the only myosin capable of producing contractile force?
11. The ability of myosin to walk along an actin filament may be observed with the aid of an appropriately equipped microscope. Describe how such assays are typically performed. Why is ATP required in these assays? How can such assays be used to determine the direction of myosin movement or the force produced by myosin?
12. Contractile bundles occur in nonmuscle cells; these structures are less organized than the sarcomeres of muscle cells. What is the purpose of nonmuscle contractile bundles? Which type of myosin is found in contractile bundles?
13. How does myosin convert the chemical energy released by ATP hydrolysis into mechanical work?
14. Myosin II has a duty ratio of 10 percent, and its step size is 8 nm . In contrast, myosin V has a much higher duty ratio (about 70 percent) and takes $36-\mathrm{nm}$ steps as it walks down an actin filament. What differences between myosin II and myosin V account for their different properties? How do the
different structures and properties of myosin II and myosin V reflect their different functions in cells?
15. Contraction of both skeletal and smooth muscle is triggered by an increase in cytosolic $\mathrm{Ca}^{2+}$. Compare the mechanisms by which each type of muscle converts a rise in $\mathrm{Ca}^{2+}$ concentration into contraction.
16. Phosphorylation of myosin light-chain kinase (MLC kinase) by protein kinase A (PKA) inhibits MLC kinase activation by $\mathrm{Ca}^{2+}$ /calmodulin. Drugs such as albuterol bind to the $\beta$-adrenergic receptor, which causes a rise in cAMP in cells and activation of PKA. Explain why albuterol is useful for treating the severe contraction of the smooth muscle cells surrounding airway passages involved in an asthma attack.
17. Several types of cells use the actin cytoskeleton to power their locomotion across surfaces. How are different assemblies of actin filaments involved in locomotion?
18. To move in a specific direction, a migrating cell must use extracellular cues to establish which portion of the cell will act as the front and which will act as the back. Describe how small GTPase proteins appear to be involved in the signaling pathways used by migrating cells to determine direction of movement.
19. Cell motility has been described as being like the motion of tank treads. At the leading edge, actin filaments form rapidly into bundles and networks that make protrusions and move the cell forward. At the rear, cell adhesions are broken and the tail end of the cell is brought forward. What provides the traction for moving cells? How does cell-body translocation happen? How are cell adhesions released as cells move forward?

## References

## Microfilaments and Actin Structures

Holmes, K. C., et al. 1990. Atomic model of the actin filament. Nature 347:44-49.

Kabsch, W., et al. 1990. Atomic structure of the actin:DNase I complex. Nature 347:37-44.

Pollard, T. D., and J. A. Cooper. 2009. Actin, a central player in cell shape and movement. Science 326:1208-1212.

## Dynamics of Actin Filaments

Paavilainen, V. O., et al. 2004. Regulation of cytoskeletal dynamics by actin-monomer-binding proteins. Trends Cell Biol. 14:386-394.

Theriot, J. A. 1997. Accelerating on a treadmill: ADF/cofilin promotes rapid actin filament turnover in the dynamic cytoskeleton. J. Cell Biol. 136:1165-1168.

## Mechanisms of Actin Filament Assembly and Disassembly

Brieher, W. 2013. Mechanisms of actin disassembly. Mol. Biol. Cell 24:2299-2302.

Campellone, K. G., and M. D. Welch. 2010. A nucleator arms race: cellular control of actin assembly. Nature Rev. Mol. Cell Biol. 11:237-251.

Chen, B. et al. 2014. The WAVE regulatory complex links diverse receptors to the actin cytoskeleton. Cell 156:195-207.

Gouin, E., M. D. Welch, and P. Cossart. 2005. Actin-based motility of intracellular pathogens. Curr. Opin. Microbiol. 8:35-45.

Rotty, J. D., C. Wu, and J. E. Bear. 2013. New insights into the regulation and cellular functions of the Arp2/3 complex. Nature Rev. Mol. Cell Biol. 14:7-12.

## Organization of Actin-Based Cellular Structures

Bennett, V., and D. N. Lorenzo. 2013. Spectrin- and ankyrinbased membrane domains and the evolution of vertebrates. Curr. Top. Memb. 72:1-37.

Fehon, R. G., A. I. McClatchey, and A. Bretscher. 2010. Organizing the cell cortex: the role of ERM proteins. Nature Rev. Mol. Cell Biol. 11:276-287.

Myosins: Actin-Based Motor Proteins
Berg, J. S., B. C. Powell, and R. E. Cheney. 2001. A millennial myosin census. Mol. Biol. Cell 12:780-794.

Vale, R. D. 2003. The molecular motor toolbox for intracellular transport. Cell 112:467-480.

Vale, R. D., and R. A. Milligan. 2000. The way things move: looking under the hood of molecular motor proteins. Science 288:88-95.

Myosin-Powered Movements
Clark, K. A., et al. 2002. Striated muscle cytoarchitecture: an intricate web of form and function. Ann. Rev. Cell Dev. Biol. 18:637-706.

Hammer, J. A. III and J. R. Sellers. 2012. Walking to work: roles for class V myosins as cargo transporters. Nature Rev. Mol. Cell Biol. 13:13-26.

Cell Migration: Signaling and Chemotaxis
Artemenko, Y., T. J. Lampert, and P. N. Devreotes. 2014. Moving towards a paradigm: common mechanisms of chemotactic signaling in Dictyostelium and mammalian leukocytes. Cell Mol. Life Sci. 71:3711-3747.

Etienne-Manneville, S. 2004. Cdc42—the centre of polarity. J. Cell Sci. 117:1291-1300.

Etienne-Manneville, S., and A. Hall. 2002. Rho GTPases in cell biology. Nature 420:629-635.

Pollard, T. D., and G. G. Borisy. 2003. Cellular motility driven by assembly and disassembly of actin filaments. Cell 112:453-465.

Ridley, A. J., et al. 2003. Cell migration: integrating signals from the front to back. Science 302:1704-1709.

this page left intentionally blank

![li3694542480e](li3694542480e.jpg)

Newt lung cell in mitosis stained for centrosomes (magenta), microtubules (green), chromosomes (blue), and keratin intermediate filaments (red). [Reprinted by permission from Macmillan Publishers Ltd: A. Khodjakov, "Olympus/Nature competition: A 1, 2, 3 in light microscopy," Nature 408, 423-424. Courtesy Alexey Khodjakov.]

As we learned in the previous chapter, three types of filaments make up the animal-cell cytoskeleton: microfilaments, microtubules, and intermediate filaments. Why have these three distinct types of filaments evolved? It seems likely that their physical properties are suited to different functions. In Chapter 17, we described how actin microfilaments are often crosslinked into networks or bundles to form flexible and dynamic structures and to serve as tracks for the many different classes of myosin motors. Similarly, microtubules are stiff tubes that can exist as a single structure, extending up to $20 \mu \mathrm{~m}$ in cells, or in bundled arrangements such as those seen in specialized cell-surface structures like cilia and flagella. As a consequence of their tubular design, microtubules are able to generate pulling and pushing forces without buckling, a property that allows single tubules to extend long distances within a cell and bundles to slide past each other, as occurs in flagella and in the mitotic spindle. Microtubules' ability to extend long distances in the cell, together with their intrinsic polarity, is exploited by microtubule-dependent motors, which use microtubules as

## OUTLINE

18.1 Microtubule Structure and Organization
18.2 Microtubule Dynamics
18.3 Regulation of Microtubule Structure and Dynamics
18.4 Kinesins and Dyneins: Microtubule-Based Motor Proteins

## Cell Organization and Movement II: <br> Microtubules and Intermediate Filaments

tracks for long-range transport of organelles. Microtubules can be highly dynamic-assembling and disassembling rapidly from their ends-and can thus provide the cell with the flexibility to alter microtubule organization as needed.

In contrast to microfilaments and microtubules, intermediate filaments have great tensile strength and have evolved to withstand much larger stresses and strains. With properties akin to strong molecular ropes, they are ideally suited to endow both cells and tissues with structural integrity and to contribute to cellular organization. Intermediate filaments do not have an intrinsic polarity as microfilaments and microtubules do, so it is not surprising that there are no known motor proteins that use intermediate filaments as tracks. Although we discuss microtubules and intermediate filaments together in this chapter-and although their localization in the cytosol can look superficially quite similar-we will see that their dynamics and functions are very different. A summary of the similarities and differences among these three filament systems is shown in Figure 18-1.

### 18.5 Cilia and Flagella: Microtubule-Based Surface Structures

18.6 Mitosis

### 18.7 Intermediate Filaments

18.8 Coordination and Cooperation Between Cytoskeletal Elements

FIGURE 18-1 Overview of the physical properties and functions of the three filament systems in animal cells.
(a) Biophysical and biochemical properties (orange) and biological properties (green) are shown for each filament type. The micrographs (b-d) show examples of each filament type in a particular cellular context, but note that microtubules also make up other structures, and that intermediate filaments also line the inner surface of the nucleus. (b) Cultured cells stained for actin (green) and sites of actin attachment to the substratum (orange). (c) Localization of microtubules (green) and the Golgi complex (yellow). Notice the central location of the Golgi complex, which is collected there by transport along microtubules. (d) Localization of cytokeratins (red), a type of intermediate filament, and a component of desmosomes (yellow) in epithelial cells. Cytokeratins from individual cells are attached to each other through the desmosomes. (Part (b) courtesy of Keith Burridge. Part (c) courtesy of William J. Brown, Cornell University. Part (d) courtesy of Elaine Fuchs.)
![li3704542480e](li3704542480e.jpg)

This chapter covers five main topics. First, we discuss the structure and dynamics of microtubules and their motor proteins. Second, we examine how microtubules and their motors contribute to the movement of cilia and flagella. Third, we discuss the role of microtubules in the mitotic spindle-a molecular machine that has evolved to accurately segregate duplicated chromosomes. Fourth, we explore the roles of the different classes of intermediate filaments that provide structure to the nuclear envelope as well as strength and organization to cells and tissues. Although we consider microtubules, microfilaments, and intermediate filaments individually, the three filament systems do not act independently of one another, and we consider some examples of this interdependence in the last section of the chapter.

### 18.1 Microtubule Structure and Organization

In the early days of electron microscopy, cell biologists noted long tubules in the cytoplasm that they called microtubules. Morphologically similar microtubules were seen making up the fibers of the mitotic spindle, as components of axons, and as the structural elements in cilia and flagella (Figure 18-2a, b). Careful examination of single
microtubules from various sources in transverse section indicated that they are all made up of 13 longitudinal repeating units (Figure 18-2c), now called protofilaments, suggesting that these various microtubules all had a common structure. Microtubules purified from brain were then found to consist of a major protein, tubulin, and associated proteins, microtubule-associated proteins (MAPs). Purified tubulin alone can assemble into a microtubule under favorable conditions, proving that tubulin is the structural component of the microtubule wall. MAPs, as we will see, help mediate the assembly, dynamics, and function of microtubules. In this section, we consider the general structure and organization of microtubules before turning to a more detailed discussion of their dynamics and regulation in Sections 18.2 and 18.3.

## Microtubule Walls Are Polarized Structures Built from $\alpha \beta$-Tubulin Dimers

Tubulin isolated in a pure and soluble form consists of two closely related subunits called $\alpha$ - and $\beta$-tubulin, each with a molecular weight of about $55,000 \mathrm{Da}$. Genomic analyses reveal that genes encoding both $\alpha$ - and $\beta$-tubulins are present in all eukaryotes, and that the number of genes has expanded considerably in multicellular organisms. For example, budding yeast has two genes specifying $\alpha$-tubulin and

![li3714542480e](li3714542480e.jpg)

FIGURE 18-2 Microtubules are found in many different locations, and all have similar structures. (a) Surface of the ciliated epithelium lining a rabbit oviduct viewed in a scanning electron microscope. Beating cilia, each of which has a core of microtubules, propel eggs down the oviduct. (b) Microtubules and intermediate filaments in a quick-frozen and deep-etched frog axon visualized in a transmission
![li3724542480e](li3724542480e.jpg)
electron microscope. (c) High-magnification view of a single microtubule showing the 13 repeating units known as protofilaments. (Part (a) NIBSC/ Science Source. Part (b) ©1982 N. Hirokawa et al., The Journal of Cell Biology, 94:129-142. doi: 10.1083/jcb.94.1.129. Part (c) Sosa, H. and Chetrien, D., "Relationship between moiré patterns, tubulin shape, and microtubule polarity," CYTOSKELETON, Vol. 40, Issue 1, pages 38-43 © 1998 Wiley.)
one for $\beta$-tubulin, whereas the soil nematode Caenorhabditis elegans has nine genes encoding $\alpha$-tubulin and six for $\beta$-tubulin. In addition to $\alpha$ - and $\beta$-tubulin, all eukaryotes also have genes specifying a third tubulin, $\gamma$-tubulin, which is involved in microtubule assembly, as we will see shortly. Additional isoforms of tubulin have also been discovered that are present only in organisms that possess cellular structures called centrioles and basal bodies, suggesting that these tubulin isoforms are important for those structures. As we'll learn in this chapter, centrioles and basal bodies are specialized structures that some organisms use to nucleate and organize microtubule assembly.

The $\alpha$ - and $\beta$-subunits of the tubulin dimer can each bind one molecule of GTP (Figure 18-3a). The GTP in the $\alpha$-tubulin subunit is never hydrolyzed and is trapped by the interface between the $\alpha$ - and $\beta$-subunits. By contrast, the GTP-binding site on the $\beta$-subunit is at the surface of the dimer. GTP bound by the $\beta$-subunit can be hydrolyzed, and the resulting GDP can be exchanged for free GTP. Under appropriate conditions, soluble tubulin dimers can assemble into microtubules (Figure 18-3b). As we saw in Chapter 17 for the polymerization of actin, ATP-G-actin is preferentially added to one end of the filament, designated the $(+)$ end because it is the end favored for assembly. Once incorporated into the filament, the bound ATP is hydrolyzed to ADP and $\mathrm{P}_{\mathrm{i}}$. In a similar manner, tubulin dimers in which the $\beta$-subunit has bound GTP are added preferentially to one end of the microtubule, also designated the (+) end. As we will see, the GTP is hydrolyzed once tubulin is incorporated into the microtubule, but in contrast to ATP hydrolysis in an actin filament, this GTP hydrolysis has dramatic effects on the behavior of the microtubule $(+)$ end.

A microtubule is composed of 13 laterally associated protofilaments, which form a tubule whose external diameter is about 25 nm (see Figure 18-3b). Each of the 13 protofilaments is a string of $\alpha \beta$-tubulin dimers, longitudinally arranged so that the subunits alternate down a protofilament, with each subunit type repeating every 8 nm . Because the

![li3734542480e](li3734542480e.jpg)

FIGURE 18-3 Structure of tubulin dimers and their organization into microtubules. (a) Ribbon diagram of the tubulin dimer. The GTP bound to the $\alpha$-tubulin monomer is nonexchangeable, whereas the GDP bound to the $\beta$-tubulin monomer is exchangeable with free GTP. (b) The organization of tubulin subunits in a microtubule. The dimers are aligned end to end into protofilaments, which pack side by side to form the wall of the microtubule. The protofilaments are slightly staggered so that $\alpha$-tubulin in one protofilament is in contact with $\alpha$-tubulin in the neighboring protofilaments, except at the seam, where an $\alpha$-subunit contacts a $\beta$-subunit. The microtubule displays a structural polarity in that subunits are added preferentially at the end where $\beta$-tubulin monomers are exposed. This end of the microtubule is known as the ( + ) end. [Part (a) data from E. Nogales et al., 1998, Nature 391:199, PDB 1D 1tub.]
$\alpha \beta$-tubulin dimers in a protofilament are all oriented in the same way, each protofilament has an $\alpha$-subunit at one end and a $\beta$-subunit at the other-thus the protofilaments have an
intrinsic polarity. In a microtubule, all the laterally associated protofilaments have the same polarity, so the microtubule also has an overall polarity. The end with exposed $\beta$-subunits is the $(+)$ end, while the end with exposed $\alpha$-subunits is the $(-)$ end. In microtubules, the heterodimers in adjacent protofilaments are staggered slightly, forming tilted rows of $\alpha$ - and $\beta$-tubulin monomers in the microtubule wall. If you follow a row of $\beta$-subunits, for example, spiraling around a microtubule for one full turn, you will end up precisely three subunits up the protofilament, abutting an $\alpha$-subunit. Thus all microtubules have a single longitudinal seam, where an $\alpha$-subunit in one protofilament meets a $\beta$-subunit in the adjacent protofilament.

Most microtubules in a cell consist of a simple tube, a singlet microtubule, built from 13 protofilaments. In rare cases, singlet microtubules contain more or fewer protofilaments; for example, certain microtubules in the neurons of nematode worms contain 11 or 15 protofilaments. In addition to this simple singlet structure, doublet or triplet microtubules are found in specialized structures such as cilia and flagella (doublet microtubules) and centrioles and basal bodies (triplet microtubules), structures we will explore later in the chapter. Each doublet or triplet contains one complete 13-protofilament microtubule (called the A tubule) and one or two additional tubules (B and C) consisting of 10 protofilaments each (Figure 18-4).

## Microtubules Are Assembled from MTOCs to Generate Diverse Configurations

Once tubulin had been identified as the major structural component of microtubules, antibodies to tubulin could be generated and used in immunofluorescence microscopy to localize microtubules in cells (Figure 18-5a, b). This approach, coupled with electron microscopy, showed that microtubules are assembled from specific sites in the cell to generate many different configurations.

The nucleation phase of microtubule assembly is such an energetically unfavorable reaction that spontaneous nucleation does not play a significant role in microtubule assembly
![li3744542480e](li3744542480e.jpg)

FIGURE 18-4 Singlet, doublet, and triplet microtubules. In cross section, a typical microtubule, a singlet, is a simple tube built from 13 protofilaments. In a doublet microtubule, an additional set of 10 protofilaments forms a second tubule (B) by fusing to the wall of
a singlet (A) microtubule. Attachment of another 10 protofilaments to the (B) tubule of a doublet microtubule creates a (C) tubule and a triplet structure.

![li3754542480e](li3754542480e.jpg)

FIGURE 18-5 Microtubules are assembled from microtubule organizing centers (MTOCs). (a-b) The distribution of microtubules in cultured cells, as seen by immunofluorescence microscopy using antibodies to tubulin, in an interphase cell (a) and a cell in mitosis (b). (c-f) Diagrams of the distributions of microtubules in various cells and structures. All of these microtubules are assembled from distinct MTOCs. (c) In an interphase cell, the MTOC is called a centrosome (the nucleus is indicated by a blue oval). (d) In a mitotic cell, the two MTOCs
are called spindle poles (the chromosomes are shown in blue). (e) In a neuron, microtubules in both axons and dendrites are assembled from an MTOC in the cell body and then released from it. (f) The microtubules that make up the shaft of a cilium or flagellum are assembled from an MTOC known as a basal body. The polarity of microtubules is indicted by $(+)$ and $(-)$. (Part (a) courtesy Anthony Bretscher. Part (b) courtesy of Torsten Wittmann.)
in vivo. Rather, all microtubules are nucleated from structures known as microtubule-organizing centers, or MTOCs. In most cases, the $(-)$ end of the microtubule stays anchored in the MTOC while the $(+)$ end extends away from it.

There are several types of MTOCs. The centrosome is the main MTOC in animal cells. During interphase, the centrosome is generally located near the nucleus, producing an array of microtubules with their $(+)$ ends radiating toward the cell periphery (Figure 18-5c). This radial array provides tracks for microtubule-based motor proteins to organize and transport membrane-bounded compartments, such as those constituting the secretory and endocytic pathways. During mitosis, cells completely reorganize their microtubules to form a bipolar spindle extending from two centrosomes, also known as spindle poles, that can accurately segregate copies of the duplicated chromosomes (Figure 18-5d). Neurons, in another example, have long processes called axons, in which organelles are transported in both directions along microtubules (Figure 18-5e). The microtubules in axons, which can be as long as 1 m , are not continuous and have been released from the MTOC, but nevertheless all have the same polarity. The microtubules in shorter processes, called dendrites, have mixed polarity, although the functional significance of this difference is not clear. In cilia and flagella (Figure 18-5f), microtubules are assembled from an MTOC called a basal body. As we mention later, plants do not have centrosomes and basal bodies, but use other mechanisms to nucleate the assembly of microtubules.

Electron microscopy shows that each centrosome in an animal cell consists of a pair of orthogonally arranged cylindrical centrioles surrounded by apparently amorphous material called pericentriolar material (Figure 18-6a). The centrioles, which are about $0.5 \mu \mathrm{~m}$ long and $0.2 \mu \mathrm{~m}$ in diameter, are highly organized and stable structures that consist of nine sets of triplet microtubules (Figure 18-6b). They are closely related in structure to the basal bodies found at the bases of cilia and flagella. It is not the centrioles themselves that nucleate the cytoplasmic microtubule array, but rather factors in the pericentriolar material. A critical component is the $\gamma$-tubulin ring complex ( $\gamma$-TuRC) (Figure 18-6c and Figure 18-7), which is located in the pericentriolar material and consists of many copies of $\gamma$-tubulin associated with several other proteins. It is believed that $\gamma$-TuRC acts like a split-washer template to bind $\alpha \beta$-tubulin dimers for the formation of a new microtubule, whose $(-)$ end is associated with $\gamma$-TuRC and whose $(+)$ end is free for further assembly. In addition to nucleating the assembly of microtubules, centrosomes anchor and regulate the dynamics of the $(-)$ ends of the microtubules, which are located there.

Basal bodies, which have a structure similar to that of a centriole, are the MTOCs found at the bases of cilia and flagella. The A and B tubules of their triplet microtubules provide a template for the assembly of the microtubules making up the core structure of cilia and flagella.

Recent work has uncovered an additional mechanism for the nucleation of microtubules in animal cells, also involving

![li3764542480e](li3764542480e.jpg)

FIGURE 18-6 Structure of centrosomes. (a) Thin section of an animal-cell centrosome showing the two centrioles at right angles to each other, surrounded by pericentriolar material (arrows). (b) Diagram of a centrosome showing the mother and daughter centrioles, each of which consists of nine linked triplet microtubules, embedded in pericentriolar material that contains $\gamma$-TuRC nucleating structures. The mother centriole is distinct from the daughter as it has distal appendages (blue spheres). (c) A tomographic image of a section through a daughter centriole of the alga Chlamydomonas. The daughter centriole is believed to be templated by a nine-fold symmetric cartwheel structure that is later removed. (d) Immunofluorescence micrograph showing the microtubule array (green) in a cultured animal cell and the location of the MTOC, using an antibody to a centrosomal protein (yellow). [Part (a) reprinted by permission from Macmillan Publishers Ltd: G. Sluder, "Two-way traffic centrosomes and the cell cycle," Nature Reviews Molecular Cell Biology, 6 743-748. 2005. Part (c) data from Guichard et al. 2013, Curr. Biol. 23:1620, EMD-2329 and EMD-2330. Part (d) courtesy of Ryoko Kuriyama.]

$\gamma$-TuRC. A protein complex called the augmin complex, consisting of eight polypeptides, can bind to the sides of existing microtubules, then recruit $\gamma$-TuRC and nucleate the assembly of new ones. As we discuss in a later section, the augmin complex contributes to microtubule assembly in the mitotic spindle.
![li3774542480e](li3774542480e.jpg)

FIGURE 18-7 The $\gamma$-tubulin ring complex ( $\gamma$-TuRC) that nucleates microtubule assembly. (a) Immunofluorescence micrograph in which microtubules assembled in vitro are labeled green and a component of the $\gamma$-TuRC is labeled red, showing that it is located specifically at one end of the microtubule. (b) Model of how $\gamma$-TuRC nucleates assembly of a microtubule by forming a template corresponding to the $(-)$ end. [Part (a) reprinted by permission from Macmillan Publishers Ltd: Keating, T.J. and Borisy, G.G., "Immunostructural evidence for the template mechanism of microtubule nucleation," Nature Cell Biology, 2:352, copyright 2000.]
![li3784542480e](li3784542480e.jpg)

## KEY CONCEPTS OF SECTION 18.1

## Microtubule Structure and Organization

- Tubulin is the major structural component of microtubules (see Figure 18-3). Microtubule-associated proteins (MAPs) associate with tubulin and help mediate the assembly, dynamics, and function of microtubules.
- Free tubulin exists as a dimer in which the $\alpha$-subunit contains a trapped and nonhydrolyzable GTP and the $\beta$-subunit binds an exchangeable and hydrolyzable GTP.
- $\alpha \beta$-Tubulin assembles into microtubules, each of which is made up of 13 laterally associated protofilaments, with an $\alpha$-subunit exposed at the $(-)$ end and a $\beta$-subunit at the $(+)$ end of each protofilament.
- In cilia and flagella, as well as in centrioles and basal bodies, doublet or triplet microtubules exist in which the additional microtubules have 10 protofilaments (see Figure 18-4).
- All microtubules are nucleated from microtubule-organizing centers (MTOCs), and many remain anchored with their ( - ) ends there. Thus the end away from the MTOC is always the $(+)$ end.

- The centrosome is the MTOC that nucleates the radial array of microtubules in interphase animal cells. Two centrosomes, or spindle poles, are the MTOCs that nucleate the microtubules of the mitotic spindle in animal cells. Basal bodies are the MTOCs that assemble the microtubules of cilia and flagella (see Figure 18-5).
- Centrosomes consist of two centrioles and the surrounding pericentriolar material, which contains the $\gamma$-TuRC microtubule-nucleating complex (see Figures 18-6 and 18-7).


### 18.2 Microtubule Dynamics

Microtubules are dynamic structures that can assemble or disassemble rapidly at their ends. Their lifetimes can vary enormously, averaging less than 1 minute for cells in mitosis and about 5-10 minutes for the microtubules that make up the radial array seen in interphase animal cells. Microtubule lifetime is longer in axons and much longer in cilia and flagella. To see how these differences occur, let's discuss the dynamic properties of microtubules and how these properties contribute to their cellular organization.

## Individual Microtubules Exhibit Dynamic Instability

Early experiments revealed that most microtubules in animal cells disassemble when the cells are cooled to $4^{\circ} \mathrm{C}$ and reassemble when the cells are rewarmed to $37^{\circ} \mathrm{C}$. Researchers realized that this intrinsic property of microtubules could be exploited to purify their components. Since brain tissue is rich in microtubules, soluble extracts of pig brains were prepared at $4^{\circ} \mathrm{C}$; these clarified extracts were then warmed to $37^{\circ} \mathrm{C}$ to induce microtubule assembly. The assembled microtubules were collected into a pellet by centrifugation, separated from the supernatant, and then disassembled by adding buffer at $4^{\circ} \mathrm{C}$. After another cycle of assembly by warming, collection, and disassembly by cooling, researchers recovered microtubular protein, a collective term for $\alpha \beta$-tubulin and microtubule-associated proteins (MAPs). They were then able to fractionate the microtubular protein into pure $\alpha \beta$-tubulin and MAPs to study their behaviors separately. The investigators found that polymerization of dimeric $\alpha \beta$-tubulin into microtubules is greatly catalyzed by the presence of the MAPs.

Although a tremendous amount of research effort was devoted to characterizing the bulk polymerization properties of microtubular proteins in solution, its general relevance was superseded by subsequent studies examining the properties of individual microtubules. Nevertheless, some lessons learned from these earlier in vitro studies are important to microtubule biology. First, for assembly to occur, the $\alpha \beta$-tubulin concentration must be above the critical concentration $\left(C_{\mathrm{c}}\right)$, just as we saw for actin polymerization (see Figure 17-8). Second, at $\alpha \beta$-tubulin concentrations higher than $C_{c}$, dimers are added faster to one end of the microtubule than to the other (Figure 18-8). As with F-actin assembly, the preferred end for assembly, which is the end
![li3794542480e](li3794542480e.jpg)

EXPERIMENTAL FIGURE 18-8 Microtubules grow preferentially at the ( + ) end. A fragment of a microtubule bundle from a flagellum was used as a nucleus for the in vitro addition of $\alpha \beta$-tubulin. The nucleating flagellar fragment is the thick bundle seen in this electron micrograph, with the newly formed microtubules (MT) radiating from its ends. The greater length of the microtubules at one end, the $(+)$ end, indicates that tubulin subunits are added preferentially to this end. [Courtesy of Gary G. Borisy.]
with $\beta$-tubulin exposed, is designated the $(+)$ end. The $(-)$ end has $\alpha$-tubulin exposed (see Figure 18-3b).

When studying the bulk properties of microtubule assembly, one might assume that all microtubules would behave similarly. However, when researchers examined the behavior of individual microtubules within a population, they found that this was not the case. Individual microtubule behavior was examined in a very simple experiment: microtubules were assembled in vitro and then sheared to break them into shorter pieces whose individual lengths could be analyzed by microscopy. Under these conditions, one might expect all the short microtubules either to grow or to shrink, depending on the free tubulin concentration. However, the investigators found that some of the microtubules grew in length, whereas others shortened very rapidly-thus indicating the existence of two distinct populations of microtubules. Further studies showed that individual microtubules could grow and then suddenly experience a catastrophe: an abrupt transition to a shrinking phase during which the microtubule would undergo rapid

![li3804542480e](li3804542480e.jpg)

FIGURE 18-9 Dynamic instability of microtubules in vitro. Individual microtubules can be observed in the light microscope and their lengths plotted at different times during assembly and disassembly. Assembly and disassembly each proceed at uniform rates, but there is a big difference between the rate of assembly and that of disassembly, as seen in the different slopes of the lines. Shortening of a microtubule is much more rapid ( $7 \mu \mathrm{~m} / \mathrm{min}$ ) than growth ( $1 \mu \mathrm{~m} / \mathrm{min}$ ). Notice the abrupt transitions to the shrinkage stage (catastrophe) and to the elongation stage (rescue). [Data from P. M. Bayley, K. K. Sharma, and S. R. Martin, 1994, in Microtubules, Wiley-Liss, p. 118.]
depolymerization. Moreover, sometimes a depolymerizing microtubule end could go through a rescue and begin growing again (Figure 18-9). Although this phenomenon was first seen in vitro, analysis of fluorescently labeled tubulin microinjected into live cells showed that microtubules in cells also undergo periods of growth and shrinkage (Figure 18-10). This alternation between growing and shrinking states is known as dynamic instability. Thus the dynamic life of a microtubule end is determined by the rate of growth, the frequency of catastrophes, the rate of depolymerization, and the frequency of rescues. As we will see later, these features of microtubule dynamics are controlled in vivo. Since the $(-)$ ends of microtubules in animal cells are generally anchored to an MTOC, this dynamism is most relevant to the $(+)$ end of the microtubule.

What is the molecular basis of dynamic instability? If you look carefully at the ends of growing and shrinking microtubules in the electron microscope, you can see that they are quite different. A growing microtubule has a relatively

## EXPERIMENTAL FIGURE 18-10 Fluorescence microscopy reveals growth and shrinkage of individual microtubules in

vivo. Fluorescently labeled tubulin was microinjected into cultured human fibroblasts. The cells were chilled to depolymerize preexisting microtubules into tubulin dimers and were then incubated at $37^{\circ} \mathrm{C}$ to allow repolymerization, which incorporated the fluorescent tubulin into all the cells' microtubules. A region of a cell periphery was viewed in the fluorescence microscope at 0 seconds, 27 seconds later, and 3 minutes 51 seconds later (left to right panels). During this period, several microtubules can be seen to have elongated and shortened. The dots labeled A, B, and C mark the positions of the ends of three microtubules. [Reprinted by permission from Macmillan Publishers Ltd: P.J. Sammak and G. Borisy, "Direct observation of microtubule dynamics in living cells," Nature, 1998, 332:724-726.]
blunt end, whereas a depolymerizing end has protofilaments peeling off like rams' horns (Figure 18-11). In fact, the growing microtubule end is not simply a blunt end, but rather a short and gently curved sheet-like structure, formed by the addition of tubulin dimers to the ends of protofilaments, that then rolls up along the seam to form the cylindrical microtubule with straight protofilaments.

A simple structural difference accounts for the morphology of these two different classes of microtubule (+) ends. As we noted above, the $\beta$-subunit of the $\alpha \beta$-tubulin dimer is exposed on the $(+)$ end of each protofilament. Using a GDP analog, researchers have found that artificially made single protofilaments-which are not exposed to lateral interactions-made up of repeating $\alpha \beta$-tubulin dimers containing GDP- $\beta$-tubulin are curved, like a ram's horn. However, artificially made single protofilaments made up of $\alpha \beta$-tubulin dimers in which $\beta$-tubulin has a bound GTP analog are only slightly curved. The assembling end of a microtubule contains these slightly curved nascent protofilaments, giving rise to the gently curved sheets and single protofilaments characteristic of growing ends (see Figure 18-11). The gently curved protofilaments containing GTP- $\beta$-tubulin then zip up by lateral interactions to form straight protofilaments within the microtubule. By contrast, shrinking microtubules with curled ends terminate in GDP- $\beta$-tubulin. Therefore, if the GTP molecules in the terminal $\beta$-tubulins become hydrolyzed on a microtubule that has stopped growing, a formerly blunt-ended microtubule will curl and a catastrophe will ensue. These relationships are summarized in Figure 18-11.

These results have an additional and fascinating implication, but to understand it we must consider the growing microtubule in more detail. The addition of a dimer to the $(+)$ end of a protofilament on a growing microtubule involves an interaction between the preexisting terminal $\beta$-subunit and the new $\alpha$-subunit. This interaction enhances the hydrolysis of the GTP in the formerly terminal $\beta$-subunit to GDP and $\mathrm{P}_{\mathrm{i}}$. The inorganic phosphate is then released to yield a microtubule with predominantly GDP- $\beta$-tubulin down its length. However, the $\beta$-tubulin in the newly added dimer contains GTP. Thus each protofilament in a growing microtubule has mostly GDP- $\beta$-tubulin down its length and is capped by a few terminal dimers containing GTP- $\beta$-tubulin and GDP- $\mathrm{P}_{\mathrm{i}}$ -$\beta$-tubulin. As we mentioned above, an isolated protofilament
![li3814542480e](li3814542480e.jpg)

![li3824542480e](li3824542480e.jpg)

FIGURE 18-11 Dynamic instability depends on the presence or absence of a GTP- $\beta$-tubulin cap. Images taken in the electron microscope of frozen samples of a growing microtubule (upper) and a shrinking microtubule (lower). Notice that the end of the growing microtubule has a blunter end, whereas the end of the shrinking one has curls like a ram's horns. As the diagram shows, a microtubule with GTP- $\beta$-tubulin on the end of each protofilament is strongly favored to grow. However, a microtubule with GDP- $\beta$-tubulin at the ends of the protofilaments forms a curved structure and will undergo rapid disassembly. Switches between growing and shrinking phases, called rescues and catastrophes, can occur, and the rate of switching is regulated by associated proteins. See A. Desai and T. J. Mitchison, 1997, Annu. Rev. Cell Dev. Bi. 13:83-117. (Photos ©1991 Mandelkow, E-M et al., The Journal of Cell Biology, 114:977-991. doi: 10.1083/jcb.114.5.977]
containing GDP- $\beta$-tubulin is curved along its length, so when it is present in a microtubule, why doesn't it break out and peel away? The lateral protofilament-protofilament interactions in the GTP- $\beta$-tubulin cap are sufficiently strong that they do not allow the microtubule to unpeel at its end-and so the protofilaments behind the GTP- $\beta$-tubulin cap are constrained from unpeeling (see Figure 18-11). The energy released by GTP hydrolysis in the subunits behind the cap is stored within the lattice as structural strain waiting to be released when the

GTP- $\beta$-tubulin cap is lost. If the GTP- $\beta$-tubulin cap is lost, the stored energy can do work if some structure, such as a chromosome, is attached to the disassembling microtubule end. As we will see, this stored energy contributes to the movement of chromosomes during the anaphase stage of mitosis.

How can a disassembling microtubule suddenly be rescued to grow again? A possible answer to this perplexing problem has recently been suggested. Using an antibody that recognizes only GTP- $\beta$-tubulin and not GDP- $\beta$-tubulin, researchers have found that "islands" of GTP- $\beta$-tubulin can persist along the length of an assembled microtubule. It seems likely that when a disassembling microtubule encounters one of these GTP- $\beta$-tubulin islands, disassembly pauses, and a rescue may be provoked.

## Localized Assembly and "Search and Capture" Help Organize Microtubules

We have now presented two major concepts relating to microtubule organization and $(+)$ end dynamics: microtubules are assembled from localized sites known as MTOCs, and individual microtubules can undergo dynamic instability. Together, these two processes contribute to the distribution of microtubules in cells.

In an interphase cell growing in culture, microtubules are constantly being nucleated from the centrosome and spreading out, randomly "searching" the cytoplasmic space. The frequency of catastrophes and rescues, together with growth and shrinkage rates, determines the length of each microtubule: if the microtubule is subject to a high catastrophe frequency and a low rescue frequency, it will shrink back to the centrosome and disappear, whereas if it has few catastrophes and is readily rescued, it will continue to grow. If the searching microtubule encounters an appropriate target on a cell structure or organelle, the microtubule end may become attached to that structure. Organelle or cell-structure "capture" by the microtubule can stabilize its $(+)$ end and protect it from catastrophes, whereas unattached microtubules have a greater frequency of disassembly. So the dynamics of the microtubule $(+)$ end is a very important determinant of microtubule life cycle and function. "Search and capture" is part of the mechanism determining the overall organization of microtubules in a cell. Moreover, by changing the rate of nucleation or local microtubule dynamics and capture sites, a cell can rapidly change its overall microtubule distribution. We will see later that this is what happens as cells enter mitosis.

## Drugs Affecting Tubulin Polymerization Are Useful Experimentally and in Treatment of Diseases

$\square$The conserved nature of tubulins and their essential involvement in critical processes such as mitosis make them prime targets for both naturally occurring and synthetic drugs that affect polymerization or depolymerization. Historically, the first such drug was colchicine, present in extracts of the meadow saffron, which binds tubulin dimers

![li3834542480e](li3834542480e.jpg)

EXPERIMENTAL FIGURE 18-12 Microtubules grow from the MTOC. To investigate the assembly of microtubules in vivo, a cultured fibroblast was treated with colchicine until almost all the cytoplasmic microtubules were disassembled. The cell was then stained with antibodies to tubulin and viewed by immunofluorescence microscopy (a). The colchicine was then washed out to allow the reassembly of microtubules. Panel (b) shows the first stages of reassembly, revealing microtubules growing from the MTOC in the central region above the nucleus (dark areas). Note in panel (a) the remaining primary cilium (arrowhead; discussed in Section 18.5) associated with the centrosome, which is not depolymerized by colchicine treatment under these conditions. Note also the fluorescence from the cytoplasm, which is from unpolymerized $\alpha \beta$-tubulin dimers. [©1991 Mandelkow, E-M et al., The Journal of Cell Biology, 114:977-991. doi: 10.1083/jcb.114.5.977]
so that they cannot polymerize into microtubules. Since most microtubules are in a dynamic state between dimers and polymers, the addition of colchicine sequesters all free dimers in the cytoplasm, resulting in loss of microtubules due to their natural turnover. Treatment of cultured cells with colchicine for a short time results in the depolymerization of all the cytoplasmic microtubules, leaving the more stable tubulin-containing centrosome (Figure 18-12a). When the colchicine is washed out to allow regrowth of the microtubules, they can be seen to grow from the centrosome, revealing its ability to nucleate the assembly of new microtubules (Figure 18-12b).

Colchicine has been used for hundreds of years to relieve the joint pain of acute gout-a famous patient was King Henry VIII of England, who was treated with colchicine to relieve this ailment. A low level of colchicine relieves the inflammation caused in gout by reducing the microtubule dynamics of white blood cells, rendering them unable to migrate efficiently to the site of inflammation.

In addition to colchicine, a number of other drugs bind the tubulin dimer and restrain it from forming polymers. These drugs include podophyllotoxin (from juniper) and nocodazole (a synthetic drug).

Taxol, a plant alkaloid from the Pacific yew tree, binds microtubules and stabilizes them against depolymerization. Because taxol stops cells from dividing by inhibiting mitosis, it has been used to treat some cancers, such as those of the breast and ovary, where the cells are especially sensitive to the drug.

## KEY CONCEPTS OF SECTION 18.2

## Microtubule Dynamics

- Individual microtubule (+) ends can undergo dynamic instability, showing alternating periods of growth or rapid shrinkage (see Figure 18-10).
- Most of the $\beta$-tubulin in microtubules contains bound GDP. In growing microtubules, the ( + ) ends have a cap of a few dimers containing GTP- $\beta$-tubulin and GDP-Pi- $\beta$-tubulin and are blunt or slightly splayed out. Shrinking microtubules have lost the cap, causing the protofilaments at the end, containing GDP- $\beta$-tubulin, to peel outward and disassemble (see Figure 18-11).
- Growing microtubules store the energy derived from GTP hydrolysis in the microtubule lattice, so they have the potential to do work when disassembling.
- Microtubules assembled from the centrosome and exhibiting dynamic instability can "search" the cytoplasm for structures or organelles with appropriate targets and "capture" them, which results in stabilization of the microtubule (+) end. In this way, assembly coupled with "search and capture" can contribute to the overall distribution of microtubules in a cell.


### 18.3 Regulation of Microtubule Structure and Dynamics

The wall of a microtubule is built from $\alpha \beta$-tubulin dimers, and highly purified $\alpha \beta$-tubulin will assemble in vitro into microtubules. But assembly of microtubules in vitro can be greatly enhanced by the presence of stabilizing microtubuleassociated proteins (MAPs). Stabilizing MAPs represent just one class of proteins that interacts with tubulin in microtubules; other classes of MAPs destabilize microtubules or modify their growth properties. We discuss the various classes of MAPs in this section. The regulation of microtubule structure and dynamics is critical for proper cell function. As we will see later, microtubules are the major organizers of organelles in animal cells, and their stability and dynamics are tailored for the specific function of the cell at any given time. For example, the dynamics of microtubules increase dramatically as a cell enters mitosis to allow the cell to build a new configuration of microtubules, the mitotic spindle.

## Microtubules Are Stabilized by Side-Binding Proteins

Several different classes of proteins stabilize microtubules, many of which show cell-type-specific expression. Among the beststudied MAPs are members of the tau family of proteins, which includes tau itself as well as proteins called MAP2 and MAP4. Tau and MAP2 are neuronal proteins, while MAP4 is expressed by other cell types and is generally not present in neurons.

These proteins have a modular design with two key domains. The first domain consists of a positively charged 18-residue sequence, repeated three to four times, that binds to the negatively charged microtubule surface. The second domain projects outward at a right angle from the microtubule (Figure 18-13). Tau proteins are believed to stabilize microtubules and also to act as spacers between them. MAP2 is found only in dendrites of neurons, where it forms fibrous cross-bridges between microtubules and links microtubules to intermediate filaments. Tau, which is much smaller than most other MAPs, is present in both axons and dendrites. The basis for this specificity is still a mystery.

When stabilizing MAPs coat the outer wall of a microtubule, they can increase the growth rate of microtubules or
(a)
![li3844542480e](li3844542480e.jpg)

EXPERIMENTAL FIGURE 18-13 Spacing of microtubules depends on the length of the projection domains of microtubuleassociated proteins. (a) Insect cells transfected to express MAP2, which has a long arm, or to express tau protein, which has a short arm, grow long axon-like processes. These electron micrographs show cross sections through the processes induced by the expression of MAP2 (left) and tau (right) in transfected cells. Note that the spacing between microtubules (MTs) in MAP2-containing cells is greater than in tau-containing cells. Both cell types contain approximately the same number of microtubules, but the effect of MAP2 is to enlarge the caliber of the axon-like process. (b) Diagrams of association between microtubules and MAPs. Note the difference between the lengths of the projection domains in MAP2 and in tau. [Part (a) reprinted by permission from Macmillan Publishers Ltd: J. Chen et al., "Projection domains of MAP2 and tau determine spacings between microtubules in dendrites and axons," Nature 1992, 360:6405, pp. 674-676.]
reduce catastrophe frequency. In many cases, the activity of the MAPs is regulated by the reversible phosphorylation of their projection domains. Phosphorylated MAPs are unable to bind to microtubules; thus phosphorylation promotes microtubule disassembly. For example, microtubule-affinityregulating kinase (MARK/Par-1) is a key modulator of tau proteins. Some MAPs, including MAP4, are also phosphorylated by a cyclin-dependent kinase (CDK) that plays a major role in controlling the activities of proteins in the course of the cell cycle (see Chapter 19).

## +TIPs Regulate the Properties and Functions of the Microtubule (+) End

In addition to the side-binding MAPs, MAPs that associate with the $(+)$ ends of microtubules have been identified. In many cases, they associate only with $(+)$ ends that are growing, not shrinking (Figure 18-14a, b). The MAPs in this class are known as +TIPs, for plus-end tracking proteins. Although there are various mechanisms by which +TIPs recognize a growing microtubule $(+)$ end, the association of a major + TIP called EB1 (end binding-1) with a growing microtubule $(+)$ end is believed to occur through interaction with the cap containing GTP- $\beta$-tubulin and GDP- $\mathrm{P}_{\mathrm{i}}-\beta$-tubulin at the end of a growing microtubule (Figure 18-14c): EB1 binds preferentially to this structure rather than to the highly curved disassembling end with GDP- $\beta$-tubulin or to the straight protofilaments in the body of the microtubule. Most other + TIPs associate with the $(+)$ end either by binding EB1 or by requiring EB1 for their association with the $(+)$ end, and are generally said to be "hitchhiking" on EB1 (Figure 18-14d).

Other +TIPs can promote microtubule growth either by enhancing assembly or by suppressing catastrophes. For example, a protein called XMAP215 contains four so-called TOG domains. These domains have the ability to bind free $\alpha \beta$-tubulin dimers as well as the gently curved regions of protofilaments at the growing end of a microtubule. By binding to the growing end and bringing more $\alpha \beta$-tubulin dimers there, XMAP215 effectively increases the local $\alpha \beta$ tubulin concentration to enhance microtubule assembly. Another class of proteins, called CLASPs, have related TOG domains but do not enhance assembly. Instead, they bind to the gently curved growing end and suppress catastrophes.
+TIPs are very important in the life of a microtubule, as they can modify its properties in several ways. First, proteins such as EB1 and XMAP215 promote microtubule growth by enhancing polymerization at the $(+)$ end. Second, other + TIPs, such as the CLASPs, can reduce the frequency of catastrophes, thereby also promoting microtubule growth. A third class of +TIPs links the microtubule $(+)$ end to other cellular structures, such as the cell cortex, F-actin, and as we will see later during our discussion of mitosis, chromosomes; a key feature of this dynamic system is that when a + TIP at the end of a "searching" microtubule encounters an appropriate target, the microtubule can become "captured" and stabilized. Yet other +TIPs link microtubule $(+)$ ends to membranes; for example, linkage to the endoplasmic reticulum transmembrane protein STIM

(a)
![li3854542480e](li3854542480e.jpg)
(c)
![li3864542480e](li3864542480e.jpg)

EXPERIMENTAL FIGURE 18-14 The +TIP protein EB1 associates dynamically with the ( + ) ends of microtubules. (a) A cultured cell stained with antibodies to tubulin (red) and the +TIP protein EB1 (green). EB1 is enriched in the region of the microtubule $(+)$ end. (b) Edge of a live cell expressing EB3-GFP (green) and mCherry$\alpha$-tubulin (red). EB3, which is closely related to EB1, is found at the ends of some microtubules. (c) EB3-GFP selectively associates with growing microtubules, as seen in this so-called kymograph. In this figure, the dynamics of a single microtubule (red) and EB3 (green) in a live cell like that shown in part (b) is followed by taking the same region from sequential frames of a movie and lining them up top to bottom.
promotes microtubule-dependent extension of the tubular endoplasmic reticulum.

## Other End-Binding Proteins Regulate Microtubule Disassembly

Mechanisms also exist for enhancing the disassembly of microtubules. Although most of the regulation of microtubule dynamics appears to happen at the $(+)$ end, in some situations, such as in mitosis, it can occur at both ends.

Various mechanisms for microtubule destabilization are known. One of these involves the kinesin-13 family of proteins.
(b)
![li3874542480e](li3874542480e.jpg)
(d)
![li3884542480e](li3884542480e.jpg)

At the top, one sees the start of the movie with the microtubule capped by EB3. Moving down the figure, one can track the dynamics of the microtubule over time as it grows and shrinks. When the microtubule grows, it remains capped by EB3. When microtubule growth pauses or the microtubule shrinks, EB3 is no longer associated with the end, but it becomes reassociated when growth resumes. A diagrammatic summary of the microtubule dynamics overlies the kymograph. (d) A possible mechanism for EB1 binding to a growing microtubule and "hitchhiking" by other proteins on EB1. (Parts (a)-(c) courtesy of Dr. A. Akhmanova, Cell Biology, Utrecht University, The Netherlands, and Dr. M. Steinmetz, Biomolecular Research, Paul Scherrer Institut, Villigen PSI, Switzerland.]

As we will see in Section 18.4, most kinesins are molecular motors, but the kinesin-13 proteins are a distinct class of kinesins that bind and curve the end of the tubulin protofilaments into the GDP- $\beta$-tubulin conformation. They then facilitate the removal of terminal tubulin dimers, thereby greatly enhancing the frequency of catastrophes (Figure 18-15a). They act catalytically in the sense that they need to hydrolyze ATP to remove sequential terminal tubulin dimers.

Another protein, known as Op18/stathmin, also enhances the rate of catastrophes. It was originally identified as a protein highly overexpressed in certain cancers; hence part of its name (Oncoprotein 18). Op18/stathmin is a small

![li3894542480e](li3894542480e.jpg)

FIGURE 18-15 Proteins that destabilize the ends of microtubules. (a) A member of the kinesin-13 family enriched at a microtubule end can enhance the disassembly of that end. (Although depolymerization of the $(+)$ end is shown, kinesin-13 can also depolymerize the $(-)$ end.) These proteins are ATPases, and ATP enhances their activity by dissociating them from the $\alpha \beta$-tubulin dimer. (b) Op18/stathmin binds selectively to curved protofilaments and enhances their dissociation from a microtubule end. This protein's activity is inhibited by phosphorylation.
protein that binds two tubulin dimers in a curved, GDP- $\beta$ tubulin-like conformation (Figure 18-15b). It may function by enhancing the hydrolysis of the GTP in the terminal tubulin dimer and aiding in its dissociation from the end of the microtubule. As might be expected for a regulator of microtubule ends, it is subject to negative regulation by phosphorylation by a large variety of kinases. In fact, it has been found that Op18/stathmin is inactivated by phosphorylation near the leading edge of a motile cell, which contributes to preferential growth of microtubules toward the front of the cell.

## KEY CONCEPTS OF SECTION 18.3

## Regulation of Microtubule Structure and Dynamics

- Microtubules can be stabilized by side-binding microtubuleassociated proteins (MAPs) (see Figure 18-13).
- Some MAPs, called +TIPs, bind selectively to growing (+) ends of microtubules and can alter the dynamic properties of the microtubule or attach components of the cell to the searching $(+)$ end of the microtubule (see Figure 18-14).
- Microtubule ends can be destabilized by some proteins, such as the kinesin-13 family of proteins and Op18/stathmin, to enhance the frequency of catastrophes (see Figure 18-15).


### 18.4 Kinesins and Dyneins: MicrotubuleBased Motor Proteins

Organelles in cells are frequently transported distances of many micrometers along well-defined routes in the cytoplasm and delivered to particular intracellular locations. Diffusion alone cannot account for the rate, directionality, and destinations of such transport processes. Findings from early experiments with fish-scale pigment cells and neurons first demonstrated that microtubules function as tracks in the intracellular transport of various types of "cargo."

As already discussed, polymerization and depolymerization of microtubules can do work using the energy provided by GTP hydrolysis. In addition, motor proteins move along microtubules, powered by ATP hydrolysis. Two main families of motor proteins-kinesins and dyneins-are known to mediate transport along microtubules. In this section, we discuss how these motor proteins work and the roles they perform in interphase cells. In subsequent sections, we discuss their functions in cilia and flagella, and in mitosis.

## Organelles in Axons Are Transported Along Microtubules in Both Directions

A neuron must constantly supply new materials-proteins and membranes-to its axon terminal to replenish those lost in the exocytosis of neurotransmitters at the junction (synapse) with another cell (see Chapter 22). Because proteins and membranes are primarily synthesized in the cell body, these materials must be transported down the axon, which can be as long as a meter in some neurons, to the synaptic region. This movement of materials is accomplished on microtubules, which are all oriented with their (+) ends toward the axon terminal (see Figure 18-5e).

The results of classic pulse-chase experiments, in which radioactive amino acids were microinjected into the dorsalroot ganglia near the spinal cord to allow for their incorporation into proteins in spinal neurons, and the radioactivity was then tracked along the axons of those cells, showed that axonal transport occurs from the cell body down the axon. Other experiments showed that transport can also occur in the reverse direction, toward the cell body. Anterograde transport proceeds from the cell body to the axon terminal and is associated with axonal growth and the delivery of synaptic vesicles. In the opposite, retrograde, direction, "old" membranes from the axon terminal move along the axon rapidly toward the cell body, where they may be degraded in lysosomes. Findings from such experiments also revealed that different materials move at different speeds (Figure 18-16). The fastest-moving material, consisting of membrane-limited vesicles, has a velocity of about $3 \mathrm{~mm} / \mathrm{s}$, or $250 \mathrm{~mm} /$ dayrequiring about 4 days to travel from a cell body in your back down an axon that terminates in your big toe. The slowestmoving material, comprising tubulin subunits and neurofilaments (the intermediate filaments found in neurons), moves only a fraction of a millimeter per day. Organelles such as mitochondria move down the axon at an intermediate rate.

![li3904542480e](li3904542480e.jpg)

EXPERIMENTAL FIGURE 18-16 The rate of axonal transport in vivo can be determined by radiolabeling and gel electrophoresis. The cell bodies of neurons in the sciatic nerve are located in dorsal-root ganglia (near the spinal cord). Radioactive amino acids injected into these ganglia in experimental animals are incorporated into newly synthesized proteins, which are then transported down the axon to the synapse. Animals are sacrificed at various times after injection and the dissected sciatic nerve is cut into small segments to see how far radioactively labeled proteins have been transported; these proteins can be identified after gel electrophoresis and autoradiography. The red, blue, and purple dots represent groups of proteins that are transported down the axon at different rates, red most rapidly, purple least rapidly.

Neurobiologists have long made extensive use of the squid giant axon for studying organelle movement along microtubules. Involved in regulating the squid's water propulsion system, the aptly named giant axon can be up to 1 mm in diameter, which is about 100 times wider than the average mammalian axon. Moreover, squeezing the axon like a tube of toothpaste results in the extrusion of the cytoplasm (also known as axoplasm), which can then be observed by video microscopy. The movement of vesicles along microtubules in this cell-free system requires ATP, its rate is similar to that of axonal transport in intact cells, and it can proceed in both the anterograde and the retrograde directions (Figure 18-17). Electron microscopy of the same region of squid giant axon cytoplasm revealed organelles attached to individual microtubules. These pioneering in vitro experiments established definitively that organelles move along individual microtubules and that their movement requires ATP.

Findings from experiments in which neurofilaments tagged with green fluorescent protein (GFP) were injected into cultured cells suggest that neurofilaments pause frequently as they move down an axon. Although the peak velocity of neurofilaments is similar to that of fast-moving vesicles, their numerous pauses lower the average rate of transport. These findings suggest that there is no fundamental difference between fast and slow axonal transport, although why neurofilament transport stops periodically is unknown.
![li3914542480e](li3914542480e.jpg)

EXPERIMENTAL FIGURE 18-17 DIC microscopy demonstrates microtubule-based vesicle transport in vitro. Cytoplasm was squeezed from a squid giant axon with a roller onto a glass coverslip. After buffer containing ATP was added to the preparation, it was viewed by differential interference contrast (DIC) microscopy, and the images were recorded on videotape. In the sequential images shown, the two organelles indicated by open and solid triangles move in opposite directions (indicated by colored arrows) along the same filament, pass each other, and continue in their original directions. Elapsed time in seconds appears at the upper-right corner of each video frame. [Republished with permission of Elsevier, from Schnapp, B. J., et al., "Single microtubules from squid axoplasm support bidirectional movement of organelles," Cell, 1985, 40:455-62.]

## Kinesin-1 Powers Anterograde Transport of Vesicles Down Axons Toward the (+) Ends of Microtubules

The protein responsible for anterograde organelle transport was first purified from axonal extracts. Researchers found that when they mixed three components-purified organelles from squid axons, an organelle-free cytosolic axonal extract, and taxol-stabilized microtubules-organelles could be seen moving on the microtubules in an ATP-dependent manner. However, if they omitted the axonal extract, the organelles neither bound nor moved along the microtubules. The researchers concluded that the extract contributes a protein that both attaches organelles to a microtubule and transports them along it-that is, a motor protein. Their strategy for purifying the motor protein was based on additional observations of organelles moving on microtubules. It was known that if ATP was hydrolyzed to ADP, the organelles fell off the microtubules. However, if the nonhydrolyzable ATP analog AMPPNP was added, the organelles remained associated with the microtubules, but did not move. These observations suggested that the motor protein linked the organelles to the microtubules very tightly in the presence of AMPPNP, but then was released from the microtubules when the AMPPNP was replaced by ATP, which was subsequently hydrolyzed to ADP. By looking for a protein that binds microtubules in the presence of AMPPNP and is released upon the addition of ATP, researchers were able to purify the motor protein, which they called kinesin.
![li3924542480e](li3924542480e.jpg)

FIGURE 18-18 Structure of kinesin-1. (a) Representation of kinesin-1 showing its two intertwined heavy chains, each with a motor domain in the head region. Each head is attached to the coiled-coil stalk by a flexible linker domain. Two light chains associate with the tail of the heavy chain. See R. D. Vale, 2003, Cell 112:467. (b) X-ray structure of the kinesin heads with the microtubule-binding and nucleotide-binding sites (containing ADP) indicated, including the linkers and the beginning of the stalk region. See M. Thormahlen et al., 1998, J. Struct. Biol. 122:30. [Data from F. Kozielski et al., 1997, Cell 91:985-994, PDB ID 3kin]
![li3934542480e](li3934542480e.jpg)

FIGURE 18-19 Model of kinesin-1-catalyzed vesicle transport. Kinesin-1 molecules, attached to receptors on the vesicle surface, transport the vesicles from the $(-)$ end to the $(+)$ end of a stationary microtubule. ATP is required for movement. See R. D. Vale et al., 1985, Cell 40:559, and T. Schroer et al., 1988, J. Cell Biol. 107:1785.

Kinesin-1 isolated from squid giant axons is a dimer of two heavy chains, each associated with a light chain, with a total molecular weight of about $380,000 \mathrm{Da}$. The molecule comprises a pair of globular N-terminal head domains connected by a short flexible linker domain to a long central stalk and terminating in a pair of small globular tail domains, which associate with the light chains (Figure 1818). Each domain carries out a particular function: the head domain binds microtubules and ATP and is responsible for the motor activity of kinesin; the linker domain is critical for forward movement; the stalk domain is involved in dimerization through a coiled-coil interaction (see Figure 3-9) of the two heavy chains; and the tail domain is responsible for binding to receptors on the membranes of cargoes.

Kinesin-1-dependent movement of vesicles can be tracked by in vitro motility assays similar to those used to study myosin-dependent movements (see Figure 17-23). In one type of assay, a vesicle or a bead coated with kinesin-1 is added to a glass slide along with a preparation of stabilized microtubules and observed in a microscope. In the presence of ATP, the beads can be observed to move along a microtubule in one direction. Researchers found that beads coated with kinesin-1 always moved from the $(-)$ to the $(+)$ end of a microtubule (Figure 18-19). Thus kinesin-1 is a $(+)$ end-directed microtubule motor protein, and additional evidence shows that it mediates anterograde transport of organelles along the axon.

## The Kinesins Form a Large Protein Superfamily with Diverse Functions

Following the discovery of kinesin-1, a number of proteins with similar motor domains were identified by both genetic screens and molecular biology approaches. There are now 14 known classes of kinesins in animals, defined as sharing amino acid sequence homology with the motor domain of kinesin-1. Proteins of the kinesin superfamily are encoded by about 45 genes in the human genome. Although the functions

of all these proteins have not yet been elucidated, some of the best-studied kinesins are involved in processes such as organelle, mRNA, and chromosome transport, microtubule sliding, and microtubule depolymerization.

As with the different classes of myosin motors, in the various kinesin families the conserved motor domain is fused to a variety of class-specific nonmotor domains (Figure 18-20). Whereas kinesin-1 has two identical heavy chains and two identical light chains, members of the kinesin-2 family (also involved in organelle transport) have two different related heavy-chain motor domains and a third polypeptide that associates with the tail and binds cargo. Members of the bipolar kinesin-5 family have four heavy chains, forming bipolar motors that can cross-link antiparallel microtubules and, by walking toward the $(+)$ end of each microtubule, slide them past each other. The kinesin-14 motor proteins are the only known class to move toward the $(-)$ end of a microtubule; this class functions in mitosis. Members of the kinesin-13 family have two subunits, but with the conserved kinesin domain in the middle of the polypeptide. Kinesin-13 proteins do not have motor activity; instead, they are special ATPhydrolyzing proteins that can enhance the depolymerization of microtubule ends (see Figure 18-15).

## Kinesin-1 Is a Highly Processive Motor

How does kinesin-1 move down a microtubule? Optical trap and fluorescence-labeling techniques similar to those used
to characterize myosin (see Figures 17-28 and 17-29) have been used to study how kinesin-1 moves down a microtubule and how ATP hydrolysis is converted into mechanical work. Such experiments demonstrate that it is a very processive motor, taking hundreds of "steps" walking "hand over hand" down a microtubule without dissociating from it. During this process, the double-headed molecule takes 8-nm steps from one tubulin dimer to the next, tracking down the same protofilament within the microtubule. This movement entails each individual head taking 16-nm steps. The two heads work in a highly coordinated manner so that one is always attached to the microtubule.

The ATP cycle of kinesin-1 movement is most easily understood if we begin just after the motor has taken a step (Figure 18-21a). At this point, the motor has a nucleotide-free leading head, which is strongly bound to a tubulin dimer in a protofilament, and an ADP-bound trailing head, which is weakly associated with the protofilament. ATP then binds to the leading head (Figure 18-21a, step $\mathbf{\square}$ ), and this binding induces a conformational change in the linker domain so that instead of pointing backward, it swings forward and "docks" into its associated head. This swinging motion results in the linker domain rotating forward, and because it is attached to the trailing head, it swings the trailing head-like throwing a ballet dancer-into position to become the leading head (Figure 18-21a, step $\mathbf{\square}$ ). The new leading head finds the next binding site on the microtubule (Figure 18-21a, step $\mathbf{\square}$ and Figure 18-21b). The binding of the leading head to the microtubule induces the leading head

FIGURE 18-20 Structure and function of selected members of the kinesin superfamily. Kinesin-1, which includes the original kinesin isolated from squid axons, is a (+) end-directed microtubule motor involved in organelle transport. The kinesin-2 family has two different, but closely related, heavy chains and a third cargo-binding subunit; this class also transports organelles in a (+) end-directed manner. Members of the kinesin-5 family have four heavy chains assembled in a bipolar configuration to interact with two antiparallel microtubules and also move toward the (+) ends. Kinesin-13 family members have the motor domain in the middle of their heavy chains and do not have motor activity, but they do destabilize microtubule ends (see also Figure 18-15a). Additional kinesin family members are mentioned in the text. Different kinesins have been given many different names; we use the unified nomenclature described in C. J. Lawrence et al., 2004, J. Cell Biol. 167:19-22. See R. D. Vale, 2003, Cell 112:467.
![li3944542480e](li3944542480e.jpg)

to release ADP while the trailing head hydrolyzes ATP to ADP and $P_{i}$, releasing $P_{i}$ (Figure 18-21a, step $\mathbf{\square}$ ). ATP can now bind to the leading head to repeat the cycle and allow the protein to take another step down the microtubule.

Two features of this cycle ensure that one head is always firmly bound to the microtubule. First, the head domain binds tightly to the microtubule in the nucleotide-free, ATP, and ADP $+P_{i}$ states, but weakly in the ADP state. Second, the two heads communicate: when the leading head binds the microtubule and releases ADP, it is converted from a weak to a tight binding state. This change is communicated to the ATP-bound trailing head, which is tightly associated with the microtubule. The trailing head is stimulated to hydrolyze ATP, releasing
![li3954542480e](li3954542480e.jpg)
$\mathrm{P}_{\mathrm{i}}$ and converting to a weak binding state. Because this cycle requires that one head always be firmly attached to a tubulin dimer in a protofilament, kinesin-1 can take thousands of steps along a microtubule without disassociating from it.

As a transporter of organelles, kinesin-1 must bind and transport the correct cargo. It does this through receptor proteins on the appropriate organelle that bind to the tail domain of the motor protein. Since kinesin-1 is an ATPase, it is important that this activity be inactivated to conserve energy when it is not needed, and also after releasing the organelle following transport. To accomplish this, kinesin-1 can fold into an inactive state in which the tail domain interacts with the motor head domain to inhibit both its ATPase activity and
(b)
![li3964542480e](li3964542480e.jpg)

FIGURE 18-21 Kinesin-1 uses ATP to "walk" down a
microtubule. (a) In this diagram, the two kinesin heads are shown with differently colored linker domains (yellow and red) to distinguish them. The cycle is shown starting after kinesin has taken a step, with the leading head tightly bound to the microtubule and not bound by any nucleotide, while the trailing head is weakly bound to the microtubule and has ADP bound. The leading head then binds ATP (step $\mathbf{\square}$ ), which induces a conformational change that causes the yellow linker region to swing forward and dock into its associated head domain, thereby thrusting the trailing head forward (step ). The new leading head now finds a binding site 16 nm down the microtubule, to which it binds weakly (step $\mathbf{\square}$ ). The leading head now releases ADP and binds tightly to the microtubule, which induces the trailing head to hydrolyze ATP to ADP and $P_{i}$ (step $\mathbf{\square}$ ). $P_{i}$ is released and the trailing head is converted into a weak binding state, and also releases the docked linker domain. The cycle now repeats itself for another step. See R. D. Vale and R. A. Milligan, 2000, Science 288:88. (b) Structural model of two kinesin heads (purple) bound to a protofilament in a microtubule. The trailing head, at left, has bound ATP and has thrust the other head into the leading position. Notice how the linker domain (yellow) is docked into the trailing head, whereas the linker domain (red) of the leading head is still free. (Part (b) data from E. P. Sablin and R. J. Fletterick, 2004, J. Biol. Chem. 279:15707 and custom PDB files based on 3kin, 1mkj, and 1jff.]

microtubule binding (Figure 18-22). When the tail binds the cargo-associated receptor protein, the motor unfolds and is activated (step $\mathbf{\square}$ ). When the cargo-activated motor complex encounters a microtubule, it transports the organelle (step $\mathbf{\square}$ ). Upon organelle delivery, kinesin-1 is released from the cargo and is inactivated by refolding into the head-to-tail inhibited state.

When the x-ray crystallographic structure of the kinesin head was determined, it revealed a major surprise-the catalytic core has the same overall structure as myosin's (Figure 18-23)! This is the case despite the fact that there is no amino acid sequence conservation between the two proteins, arguing strongly that convergent evolution has twice generated a fold that can use the hydrolysis of ATP to generate work. Moreover, the same type of three-dimensional structure is seen in small GTP-binding proteins, such as Ras, that undergo a conformational change upon GTP hydrolysis (see Figure 15-7).

## Dynein Motors Transport Organelles Toward the (-) Ends of Microtubules

In addition to kinesin motors, which primarily mediate anterograde (+) end-directed transport of organelles, cells use another motor protein, cytoplasmic dynein, to transport organelles in a retrograde direction toward the ( - ) ends of microtubules. This motor protein is very large, consisting of two large ( $>500 \mathrm{kDa}$ ), two intermediate, and two small subunits. It is responsible not only for the ATP-dependent retrograde transport of organelles toward the ( - ) ends of microtubules in axons, but for many other functions we will consider in the following sections. Compared with the myosins and the kinesins, the family of dynein-related proteins is not very diverse.
![li3974542480e](li3974542480e.jpg)

FIGURE 18-22 Kinesin-1 is regulated by a head-to-tail interaction. In its inhibited form, the head of kinesin-1 folds back and interacts with the tail. This interaction inhibits the ATPase activity of kinesin-1. When the motor encounters an appropriate receptor, here shown on a vesicle, it unfolds (step $\mathbf{\square}$ ) and is now able to transport the cargo toward the $(+)$ end of a microtubule (step $\mathbf{\square}$ ). It is not yet clear how the motor dissociates from the cargo when they arrive at the destination, but it involves cargo release and folding back into the inhibited state.

Like kinesin-1, cytoplasmic dynein is a two-headed molecule, built around two identical or nearly identical heavy chains. However, because of the enormous size of the motor domain, dynein has been less well characterized in terms of its mechanochemical activity. A single dynein heavy chain consists of a number of distinct domains (Figure 18-24a). The first is the stem, to which the other dynein subunits bind, and which associates with cargo through another protein complex, dynactin. The next part of the heavy chain is a
![li3984542480e](li3984542480e.jpg)

FIGURE 18-23
Convergent structural evolution of the ATPbinding cores of myosin and kinesin heads. The common catalytic cores of myosin and kinesin are shown in yellow, the nucleotide in red, and the lever arm (for myosin-II) and linker domain (for kinesin-1) in light purple. See R. D. Vale and R. A. Milligan, 2000, Science 288:88. [Data from F. Kozielski et al., 1997, Cell 91:985-994, custom PDB file (left) and PDB ID 3kin (right)]

![li3994542480e](li3994542480e.jpg)

FIGURE 18-24 The domain structure of cytoplasmic dynein.
(a) The dynein heavy chain, consisting of over 4000 amino acid residues, has several distinct domains. Following the stem and linker domains are six AAA repeats (peach, numbered 1-6), with the stalk and its microtubule-binding domain between repeats 4 and 5 . The protein ends in an $\alpha$-helical domain that supports the stalk. (b) The six AAA repeats assume a structure like petals on a flower. Emerging from this structure is a coiled-coil stalk domain with a microtubule-binding site at the end. A number of additional subunits associate with the stem region and can link dynein to cargo through dynactin. See R. D. Vale, 2003, Cell 112:467.
linker that plays a critical role during ATP-dependent motor activity. A large part of the heavy chain makes up the head containing the AAA ATPase domain, consisting of six repeats that assemble into a flowerlike structure, within which lies the ATPase activity. Embedded between the fourth and fifth AAA repeats is the stalk, which protrudes from the structure and contains the microtubule-binding region.

Electron microscopy, combined with recently acquired x-ray images of the structure of a dynein heavy chain, provides a glimpse of how dynein might work. The first AAA repeat is probably the only one involved in converting the hydrolysis of ATP into mechanical work. In the absence of a nucleotide, dynein binds to microtubules, and the linker is straight, lying across the AAA domain and associating with the first and fifth AAA repeats. Upon ATP binding, dynein dissociates from the microtubule, and the linker becomes bent to now cross between the second and third AAA repeats ("pre-stroke"; Figure 18-25a and b, left
panels). Upon interaction with a microtubule and ATP hydrolysis and release of $\mathrm{P}_{\mathrm{i}}$, the linker straightens. This straightening is the power stroke that moves the cargo toward the $(-)$ end of the microtubule ("post-stroke"; Figure 18-25a and b, right panels). The ADP is released, and the motor head remains bound to the microtubule. When ATP binds, the head is released, and the cycle repeats for another step.

Unlike kinesin-1, cytoplasmic dynein cannot mediate cargo transport by itself. Rather, dynein-related transport
![li4004542480e](li4004542480e.jpg)

FIGURE 18-25 The power stroke of dynein. (a) Multiple images of purified single-headed dynein molecules in their pre-stroke and poststroke states were recorded in an electron microscope and then averaged. The image at the left shows dynein in the ADP $+P_{i}$ state, which represents the pre-stroke state, and the image at the right shows it in a nucleotide-free post-stroke state. (b) A comparison of the microscopic images combined with recently acquired structural data shows that the force-generation mechanism involves a change in the position of the linker, which causes a movement of the microtubule-binding stalk. See G. Bhabha et al., 2014, Cell 159:857. (Part (a) reprinted by permission of Macmillan Publishers Ltd: from S.A. Burgess, "Dynein structure and power stroke," Nature, 2003, 421:6924, pp. 715-718.

requires regulators such as dynactin, a large protein complex that links dynein to its cargo and regulates its activity (Figure 18-26). Dynactin consists of 11 different types of subunits, functionally organized into two domains. One domain is built around one actin subunit and eight copies of the actin-related protein Arp1 assembled into a short filament. The end corresponding to the $(+)$ end of this filament is capped by CapZ, the same capping protein that binds the $(+)$ end of an actin filament (see Figure 17-12); a number of other subunits are associated with the $(-)$ end. This Arp1-containing domain is responsible for binding cargo. The second domain of dynactin consists of a long protein called p150 ${ }^{\text {Glued }}$, which contains the dyneinbinding site and also has a microtubule-binding site at one end. Holding the two dynactin domains together is a protein called dynamitin-so named because when it is overexpressed, it dissociates (or "blows apart") the two domains, making a nonfunctional complex. This feature
has been very useful experimentally because it has allowed researchers to identify processes that are dependent on the interaction of dynein and dynactin, which are disrupted in cells overexpressing dynamitin.

Recent work has shown that both dynein and dynactin can exist in an inactive form. However, when they encounter an appropriate cargo adapter, the three molecules form a tripartite complex in which dynactin is activated to bind microtubules and transport by dynein becomes highly processive, a critical property for transporting cargo over long distances (Figure 18-26b). In some circumstances, dynein must be transported in an inactive form to a specific location, where it is then activated. For example, it has been found that the dynactin p150 ${ }^{\text {Glued }}$ subunit binds EB1, allowing dynein to become associated with the growing (+) end of a microtubule. Why would dynein associate with the microtubule (+) end if it is a ( - ) end-directed motor? Recent work suggests that when dynein is associated with
![li4014542480e](li4014542480e.jpg)

FIGURE 18-26 The dynactin complex links dynein to cargo. (a) One domain of the complex, which binds cargo, is built around a short filament made up of eight subunits of the actin-related protein Arp1 and one actin subunit, capped by CapZ. Another domain consists of the protein p150 ${ }^{\text {Glued }}$, which has a microtubulebinding site on its distal end and is also involved in attaching cytoplasmic dynein to the complex. Dynamitin holds the two parts of the dynactin complex together. (b) Diagram of how the dynactin and dynein complex interact with each other and with a microtubule. See Urnavicius et al. 2015, Science 347:1441.

the $(+)$ end of a microtubule via the dynactin-EB1 interaction, it is held in an inactive conformation. When the growing microtubule reaches the cell cortex, the inactive dynein and dynactin encounter an activator localized there. The dynein now becomes active, associating with the cortex and pulling on the microtubule that delivered it there! This mechanism has been shown to help orient the mitotic spindle in yeast, and it probably applies to other situations as well.

5In addition to dynactin, other regulators of dynein exist. One group of proteins, two of which are LIS1 and NudE, is involved in regulating the activity of dynein during brain development. NudE links the dynein intermediate and light chains to LIS1 (Figure 18-27a). LIS1 then interacts with the ATPase domain of dynein to lengthen the duration of the power stroke, making the motor more processive under conditions of high load. Defects in LIS1 cause the fatal disease Miller-Dieker lissencephaly (from which the protein got its name); lissencephaly means "smooth brain," as cortical folds and grooves are lacking in the brains of patients with this condition (Figure 18-27b). Mutations in LIS1 result in defects in both neuronal mitoses and neuronal migration from the ventricular zone to the cortical plate in early development, resulting in the smooth-brain phenotype and defects in mental development as well as many other abnormalities.

## Kinesins and Dyneins Cooperate in the Transport of Organelles Throughout the Cell

Both dynein and kinesin family members play important roles in the microtubule-dependent organization of organelles in cells (Figure 18-28). Because the orientation of microtubules is fixed by the MTOC, the direction of transport-toward or away from the cell center-depends on the motor protein. For example, the Golgi complex collects in the vicinity of the centrosome, where the ( - ) ends of microtubules lie, and is driven there by dynein-dynactin. In addition, secretory cargo emerging from the endoplasmic reticulum is transported to the Golgi by dynein-dynactin. Conversely, the endoplasmic reticulum is spread throughout the cytoplasm and is transported there by kinesin-1, which moves toward the peripheral $(+)$ ends of microtubules. Some organelles of the endocytic pathway are associated with dynein-dynactin, including late endosomes and lysosomes. Kinesins have been shown to transport mitochondria as well as nonmembranous cargoes such as specific mRNAs encoding proteins that need to be localized during development.

We have seen how kinesin-1 transports organelles in an anterograde direction down axons. What happens to the motor when it gets to the end of the axon? The answer is that it is carried back in a retrograde direction on organelles transported by cytoplasmic dynein. Thus kinesin-1 and
(a)
![li4024542480e](li4024542480e.jpg)
(b)
![li4034542480e](li4034542480e.jpg)

Normal
![li4044542480e](li4044542480e.jpg)

LIS1 patient

EXPERIMENTAL FIGURE 18-27 The LIS1 protein, which regulates dynein, is required for brain development. (a) Model of how NudE associates with dynein to allow LIS1 to interact with the ATPase domain of the dynein heavy chain. See McKenny et al., 2010, Cell 141:304-314. (b) Magnetic resonance images (MRIs) of a normal brain (left) and a brain from a patient with Miller-Dieker lissencephaly (right), which results from a lack of LIS1 function. Notice the absence of folding in the patient's brain at the locations marked by the arrows. [Part (b) from Kato, M. \& Dobyns, W. B. "Lissencephaly and the molecular basis of neuronal migration," Hum. Mol. Genet. (2003) 12, R89-R96, by permission of Oxford University Press.]
dynein can associate with the same organelle, and a mechanism must exist that turns one motor off while activating the other, although such mechanisms are not yet fully understood.

Much of what we know about the regulation of microtubule-based organelle transport comes from studies using fish (e.g., angelfish) or frog melanophores. Melanophores are cells of the vertebrate skin that contain hundreds of dark melanin-filled pigment granules called melanosomes. Melanophores either have their melanosomes dispersed, in which case they make the skin darker, or aggregated at the

![li4054542480e](li4054542480e.jpg)

FIGURE 18-28 Organelle transport by microtubule motors. Cytoplasmic dyneins (red) mediate retrograde transport of organelles toward the ( - ) ends of microtubules (cell center); kinesins (purple) mediate anterograde transport toward the ( + ) ends (cell periphery). Most organelles have one or more microtubule-based motors associated with them. It should be noted that the association of motors with organelles varies by cell type, so some of these associations may not exist in all cells, whereas others not shown here also exist. ERGIC $=$ ER-to-Golgi intermediate compartment.
cell center, which makes the skin paler (Figure 18-29). These changes in skin color, mediated by neurotransmitters in the fish and regulated by hormones in the frog, serve to camouflage the fish and enhance social interactions in the frog. The movement of the melanosomes is mediated by changes in intracellular cAMP and is dependent on microtubules. Studies investigating which motors are involved have shown that melanosome dispersion requires kinesin-2, whereas melanosome aggregation requires cytoplasmic dynein-dynactin. The first hints of how these activities might be coordinated came from the finding that overexpression of dynamitin inhibited melanosome transport in both directions. This surprising result was explained when it was found that dynactin binds not only to cytoplasmic dynein, but also to kinesin-2-and may coordinate the activity of the two motors.

The association of dynein and kinesin-2 with the same organelle is not limited to melanosomes; it has recently been suggested that these motors may cooperate to appropriately localize late endosomes, lysosomes, and mitochondria in some cells. Thus the association of organelles with a number of distinct motors is not the exception, but an emerging theme.

## Tubulin Modifications Distinguish Different Classes of Microtubules and Their Accessibility to Motors

The stability and functions of different classes of microtubules are influenced by post-translational modifications. Although multiple types of modifications have been detected, we restrict our discussion here to those that are the best understood-lysine acetylation, detyrosylation,
![li4064542480e](li4064542480e.jpg)

FIGURE 18-29 Movement of pigment granules in frog melanophores. (a) Diagram of the microtubule-based reorganization of melanosomes according to the level of cAMP. Melanosomes are aggregated by cytoplasmic dynein and dispersed by kinesin-2. (b) Melanosomes (red) in the dispersed state as seen by immunofluorescence microscopy. Microtubules are labeled green, and the DNA in the nucleus is labeled blue. [Part (b) courtesy of Steve Rogers.]

polyglutamylation, and polyglycylation (Figure 18-30a)and their functional consequences.

Two of these modifications are found only on $\alpha$-tubulin and are absent from $\beta$-tubulin. The first is the acetylation of the $\varepsilon$-amino group of a specific lysine residue of $\alpha$-tubulin that lies on the inside of the microtubule; microtubules with this acetylated lysine are found in stable microtubule structures such as centrioles, basal bodies, and primary cilia (primary cilia are discussed in Section 18.5). Indeed, cells unable to acetylate tubulin have defective primary cilia, whereas cells in which the acetylation cannot be removed have unusually stable primary cilia. The second modification of $\alpha$-tubulin relates to its C-terminal tyrosine. This tyrosine can be specifically removed by a carboxypeptidase that functions only when bound to the surface of a microtubule, where it sequentially removes the C-terminal tyrosines from $\alpha$-tubulin subunits. Such detyrosylated microtubules are more stable, as they are more resistant to depolymerization by the kinesin-13 family of depolymerizers. Moreover, in migrating cells, these more stable microtubules are generally oriented toward the front of the cell. When a detyrosylated microtubule depolymerizes, the $\alpha$-tubulin subunit of the $\alpha \beta$-dimer has the Cterminal tyrosine added back by a tyrosine ligase that acts only on soluble tubulin, and the $\alpha \beta$-tubulin dimer can now be used during elongation of another growing microtubule.
![li4074542480e](li4074542480e.jpg)

EXPERIMENTAL FIGURE 18-30 Post-translational modifications of tubulin affect the stability and function of microtubules. (a) Structure of $\alpha$ - and $\beta$-tubulin, showing the sites of lysine acetylation on the inner surface of the microtubule and polyglutamylation, polyglycylation, and detyrosylation on the outer surface. Note that polyglutamylation and polyglycylation are believed to be mutually exclusive, so they would not normally occur at the same time. See J. W. Hamond, D. Cai, and K. J. Verhey, 2008, Curr. Opin. Cell Biol.

The C-terminal regions of both $\alpha$ - and $\beta$-tubulin are very rich in glutamic acid residues, and specific enzymes can modify these residues. These modifications occur only after assembly of the microtubule. The tubulin tails can be modified by polyglutamylation, in which a chain of glutamic acid residues is linked to a specific glutamate residue, or by polyglycylation, in which a chain of glycine residues is added to a different glutamate residue. At present it is believed that these two modifications are mutually exclusive, so that if a tubulin subunit is modified by polyglycylation, it is protected from polyglutamylation, and vice versa. Like detyrosylation, polyglutamylation can enhance microtubule stability.

These post-translational modifications of tubulin not only affect microtubule stability, but can also influence the ability of molecular motors to interact with microtubules (Figure 18-30b). Kinesin-1 associates preferentially with detyrosylated and acetylated microtubules, so these modifications may be important in recruiting this motor for axonal transport in neurons. As we mentioned in Figure 18-5e, neurons have different microtubule organizations in their dendrites and their axons. The microtubules in the axon are stabilized by acetylation and detyrosylation, which allows kinesin-1 to associate preferentially with them for axonal transport. Polyglutamylation has a key role in the beating of cilia and flagella, which we discuss in the next section.
(b)
![li4084542480e](li4084542480e.jpg)

20:71-76. (b) Detyrosylated microtubules are preferentially oriented toward the leading edge of a moving cell. A cell migrating toward the right is stained for total microtubules (red) and detyrosylated microtubules (green). The resulting merged image shows the detyrosylated microtubules enriched toward the front of the cell in yellow, which is a combination of red and green. [Part (a) data from E. Nogales, S. G. Wolf, and K. H. Downing, 1998, Nature 391:199-203, PDB ID 1 tub. Part (b) courtesy of Greg Gundersen.]

The research elucidating the effects of post-translational modifications of tubulin on microtubule function and microtubule-based motors is all quite recent; we can expect future studies to reveal multiple "codes" that distinguish different classes of microtubules and specialize them for specific functions.

## KEY CONCEPTS OF SECTION 18.4

## Kinesins and Dyneins: Microtubule-Based Motor Proteins

- Kinesin-1 is a microtubule (+) end-directed ATP-dependent motor protein that transports membrane-bounded organelles (see Figure 18-19).
- Kinesin-1 consists of two heavy chains, each with an N terminal motor domain, and two light chains that associate with cargo (see Figure 18-18).
- The kinesin superfamily includes motors that function in interphase and mitotic cells, transporting organelles and sliding antiparallel microtubules past each other. The superfamily includes one class, kinesin-13, that is not motile, but instead destabilizes microtubule ends (see Figure 18-20).
- Kinesin-1 is a highly processive motor because it coordinates ATP hydrolysis between its two heads so that one head is always firmly bound to a microtubule (see Figure 18-21).
- Kinesin-1 can exist in an inactive folded conformation or in a cargo-associated extended conformation (see Figure 18-22).
- Cytoplasmic dynein is a microtubule ( - ) end-directed ATP-dependent motor that associates with the dynactin complex and cargo adapters to transport cargo (see Figures 18-24 and 18-26).
- Kinesins and dyneins associate with many different organelles to organize their locations in cells (see Figure 18-28).
- Post-translational modifications of tubulin can affect the stability of microtubules and regulate their ability to interact with microtubule-based motors.


### 18.5 Cilia and Flagella: MicrotubuleBased Surface Structures

Cilia and flagella are related microtubule-based and membrane-enveloped extensions of the plasma membrane that project from many protozoan and most animal cells. Abundant motile cilia are found on the surfaces of specific epithelia, such as those that line the trachea (see Fig. 4-35), where they beat in an orchestrated wavelike fashion to move
fluids. Animal-cell flagella, which are longer than cilia but have a very similar structure, can propel a cell, such as a sperm, through liquid. Cilia and flagella contain many different microtubule-based motors: axonemal dyneins are responsible for the beating of flagella and cilia, whereas kinesin-2 and cytoplasmic dynein are responsible for flagellum and cilium assembly and turnover.

## Eukaryotic Cilia and Flagella Contain Long Doublet Microtubules Bridged by Dynein Motors

Cilia and flagella range in length from a few micrometers to more than 2 mm for some insect sperm flagella. They possess a central bundle of microtubules, called the axoneme, which consists of a so-called $9+2$ arrangement of nine doublet microtubules surrounding a central pair of singlet, yet ultrastructurally distinct, microtubules (Figure 18-31a, b). Each of the nine outer doublets consists of an A microtubule with 13 protofilaments and a B microtubule with 10 protofilaments (see Figure 18-4). All the microtubules in cilia and flagella have the same polarity: the $(+)$ ends are located at the distal tip. At its point of attachment in the cell, the axoneme connects with the basal body, a complicated structure containing nine triplet microtubules (Figure 18-31a).

The axoneme is held together by three sets of protein cross-links (Figure 18-31b). The two central singlet microtubules are connected to each other by periodic bridges, like rungs on a ladder. A second set of linkers, composed of the protein nexin, joins adjacent outer doublet microtubules to each other. Radial spokes project from each A tubule of the outer doublets toward the central pair.

The major motor protein present in cilia and flagella is axonemal dynein, a large, multisubunit protein related to cytoplasmic dynein. Two rows of axonemal dynein motors are attached periodically down the length of each A tubule of the outer doublet microtubules; these motors are called the inner-arm and outer-arm dyneins (see Figure 18-31b). It is the interaction of these dynein motors with the B tubule in the adjacent doublet that brings about ciliary and flagellar bending.

## Ciliary and Flagellar Beating Are Produced by Controlled Sliding of Outer Doublet Microtubules

Cilia and flagella are motile structures because activation of the axonemal dynein motors causes them to bend. A close examination of their movement using video microscopy reveals that a bend starts at the base of a cilium or flagellum and then propagates along the structure (Figure 18-32). A clue to how this occurs came from studies of isolated axonemes. In these classic experiments, axonemes were gently treated with a protease that cleaves only the nexin linkers. When ATP was added to the treated axonemes, the doublet microtubules slid past one another as dynein, attached to the A tubule of one doublet, "walked" down the B tubule of

![li4094542480e](li4094542480e.jpg)

FIGURE 18-31 Structural organization of cilia and flagella.
(a) Cilia and flagella are assembled from a basal body, a structure built around nine linked triplet microtubules. Continuous with the A and B microtubules of the basal body are the A and B tubules of the axoneme-the membrane-enveloped core of the cilium or flagellum. Between the basal body and axoneme is the transitional zone.
the adjacent doublet (Figure 18-33b, c). In an axoneme with intact nexin linkers, the action of dynein induces flagellar bending because the microtubule doublets are connected to one another (Figure 18-33a).

How specific subsets of dynein are activated and how a wave of activation is propagated down the axoneme are not yet fully understood, but post-translational modifications of tubulin may play a role. Recall from Section 18.4 that post-translational modifications of tubulin subunits can affect the interactions between microtubules and motor proteins. The B tubules of the outer axoneme doublets are often polyglutamylated, and this modification strongly affects the interaction of the inner-arm dynein with the B tubule. Because inner-arm dynein motors mainly affect the waveform of the ciliary beat, it is this aspect of ciliary
(b)
![li4104542480e](li4104542480e.jpg)

The diagram and accompanying transverse sections of the basal body, transitional zone, and axoneme show their intricate structures. (b) A transverse section of a cilium, to show the identity of the structures. See S. K. Dutcher, 2001, Curr. Opin. Cell Biol. 13:49-54. (Micrographs republished with permission of Elsevier, from Dutcher, S. K., "The tubulin fraternity: alpha to eta," Curr. Opin. Cell Bio., 2001, 13(1):49-54.)
function that is compromised in mutants unable to undergo polyglutamylation.

## Intraflagellar Transport Moves Material Up and Down Cilia and Flagella

Although axonemal dynein is involved in bending cilia and flagella, another type of motility has been observed in these structures more recently. Careful examination of flagella on the biflagellate green alga Chlamydomonas reinhardtii revealed cytoplasmic particles moving at a constant speed of about $2.5 \mu \mathrm{~m} / \mathrm{s}$ toward the tip of a flagellum (anterograde movement) and other particles moving at about $4 \mu \mathrm{~m} / \mathrm{s}$ from the tip to the base (retrograde movement). This movement, known as intraflagellar transport (IFT), occurs in both cilia and flagella.

(a)
![li4114542480e](li4114542480e.jpg)

EXPERIMENTAL FIGURE 18-32 Video microscopy shows flagellar movements that propel sperm and Chlamydomonas forward. In both cases, the cells are moving to the left. (a) In the typical sperm flagellum, successive waves of bending originate at the base and are propagated out toward the tip; these waves push against the surrounding fluid and propel the cell forward. Captured in this multiple-exposure sequence, a bend at the base of the flagellum in the first (top) frame has moved distally halfway along the flagellum by the last frame. A pair of gold beads attached to the flagellum are seen to slide apart as the bend moves through their region. (b) Beating of the two flagella on Chlamydomonas occurs in two stages, called the effective stroke (top three frames) and the recovery stroke (remaining frames). The effective stroke pulls the organism through the water. During the recovery stroke, a different wave of bending moves outward from the bases of the flagella, pushing the flagella along the surface of the cell until they reach the position to initiate another effective stroke. Beating commonly occurs 5-10 times per second. [Part (a) ©1991 C. Brokaw et al., The Journal of Cell Biology, 114:1201-1215. doi: 10.1083/jcb.114.6.1201. Part (b) courtesy of Stuart Goldstein.]
![li4124542480e](li4124542480e.jpg)
(b) Nexin linkers removed by protease
![li4134542480e](li4134542480e.jpg)
![li4144542480e](li4144542480e.jpg)
(c)
![li4154542480e](li4154542480e.jpg)

FIGURE 18-33 Ciliary and flagellar bending is mediated by axonemal dynein. (a) Axonemal dynein attached to an A tubule of an outer doublet pulls on the B tubule of the adjacent doublet trying to move toward the ( - ) end. Because the adjacent tubules are tethered by nexin, the force generated by dynein bends the cilium or flagellum. (b) Experimental evidence for the model in (a). When the nexin linkers are cleaved with a protease and ATP added to induce dynein activity, the microtubule doublets slide past one another. (c) Electron micrograph of two doublet microtubules in a protease-treated axoneme incubated with ATP. In the absence of cross-linking proteins, doublet microtubules slide excessively. The dynein arms can be seen projecting from A tubules and interacting with B tubules of the left microtubule doublet. [Part (c) courtesy of P. Satir.]

Light and electron microscopy revealed that the particles move between the outer doublet microtubules and the plasma membrane (Figure 18-34). Analysis of algal mutants demonstrated that the anterograde movement is powered by kinesin-2 and the retrograde movement by cytoplasmic dynein.

The anterograde and retrograde IFT particles transported in Chlamydomonas flagella have been isolated and their composition determined. They consist of two distinct protein complexes, called IFT complex A and IFT complex B. By analyzing the phenotypes of cells having mutations affecting these complexes, it has been found that complex B is necessary for anterograde IFT, whereas complex A is important for retrograde IFT. Despite this segregation of function, both complexes are transported in both directions. All the components of IFT particles have homologs in organisms containing cilia, such as nematodes, fruit flies, mice, and humans, but these particles are absent from the genomes of yeasts and plants that lack cilia, suggesting that they are specific to IFT.

What is the function of IFT? Because all the microtubules in a flagellum have their growing $(+)$ ends at the tip, that is the site where new tubulin subunits and flagellar structural proteins are added. Moreover, even in cells with flagella of uniform length, the microtubules are turning over, with assembly and disassembly occurring at the flagellar tip. In cells defective for
kinesin-2, flagella shrink, suggesting that IFT transports new material to the tip for growth. Since IFT is a continually occurring process, what happens to the kinesin-2 molecules when they get to the tip, and where do the dynein motors come from to transport the particles retrogradely? Remarkably, dynein is carried to the tip as cargo on the anterograde-moving particles, powered by kinesin-2, and then kinesin-2 becomes cargo as the particles are transported back to the base by dynein.

## Primary Cilia Are Sensory Organelles on Interphase Cells

Many vertebrate cells bear a solitary nonmotile cilium known as the primary cilium. The primary cilium is a stable structure that is resistant to drugs such as colchicine that disassemble most microtubules; after colchicine treatment, the only remaining microtubules are found in the centrioles and the primary cilium (see Figure 18-12). Moreover, the tubulin in the primary cilium is highly acetylated, so using antibodies that specifically recognize acetylated $\alpha$-tubulin readily identifies the single primary cilium on each interphase cell (Figure 18-35a).

Terminally differentiated cells and dividing cells in interphase contain primary cilia. In the latter case, the presence of the primary cilium is tied to the duplication cycle
![li4164542480e](li4164542480e.jpg)

FIGURE 18-34 Intraflagellar transport. (a) Particles are transported between the plasma membrane and the outer doublet microtubules. Transport of the particles to the tip is dependent on kinesin-2, whereas transport toward the base is mediated by cytoplasmic dynein.
(b) This thin-section electron micrograph shows IFT particles in a section of a Chlamydomonas flagellum. [Part (b) reprinted by permission from Macmillan Publishers Ltd: Rosenbaum J.L., and Witman, G.B., "Intraflagellar transport," Nature Reviews Molecular Cell Biology, 2002, 3, 813-825.]

![li4174542480e](li4174542480e.jpg)

FIGURE 18-35 Many interphase cells contain a nonmotile primary cilium. (a) Fluorescence micrograph of mouse epithelial cells stained with antibodies to acetylated $\alpha$-tubulin (green), which decorate the primary cilia; to pericentrin (magenta), which decorate the centrosome; and to ZO-1(red), which label the tight junctions that encircle each cell. (b) A diagram depicting how the presence of the primary cilium is tied to the centrioles, one of which serves as its basal body. (c) A diagram depicting a section through a nonmotile primary cilium, showing the lack of the central pair of microtubules and dynein arms typical of motile cilia and flagella. (d) Scanning electron micrographs of epithelial cells of a kidney collecting tubule from a wild-type mouse (left) and a mutant mouse defective in a component of the IFT particles. Arrows point to the primary cilia, which are short stubs in the mutant mouse. (la) Republished with permission of John Wiley \& Sons, from "A Cell-Based Screen for Inhibitors of Flagella-Driven Motility in Chlamydomonas Reveals a Novel Modulator of Ciliary Length and Retrograde Actin Flow," Enge, B.D. et al., Cytoskeleton, 68(3). Copyright (c) 2011. (d) ©1991 Douglas J. Cole, from Pazour, G. et al., The Journal of Cell Biology. 151:709-718. doi: 10.1083/jcb.151.3.709.]

of the centrioles (discussed in Section 18.6), with the older, "mother" centriole functioning as the basal body for the assembly of the primary cilium (Figure 18-35b).

The primary cilium is nonmotile because it lacks the central pair of microtubules and the dynein arms that are found in other cilia and in flagella (Figure 18-35c). Recent work has led to the discovery that the primary cilium is instead a sensory organelle, acting as the cell's "antenna" by detecting extracellular signals. For example, the sense of smell is due to binding of odorants by receptors located in the primary cilia of olfactory sensory neurons in the nose (see Chapter 22). In another example, the rod and cone cells of the eye have a primary cilium with a greatly expanded tip to accommodate the proteins involved in photoreception. The retinal protein opsin moves through the primary cilium at about 2000 molecules per minute, transported by kinesin-2 as part of the IFT system. Defects in this transport cause retinal degeneration.

The primary cilium has a diffusion barrier at its base so that only the appropriate proteins can enter it: globular proteins of 10 kDa have ready access, but proteins above
![li4184542480e](li4184542480e.jpg)

40 kDa are excluded. Remarkably, transport through this barrier has similarities to transport into the nucleus through nuclear pores, and indeed, these two forms of transport have some components in common. Recall that import though nuclear pores requires a gradient of the Ran GTPase and involves binding of cargo proteins to importins (see Section 13.6). Transport of at least some proteins, such as kinesin-2, across the base and into a primary cilium requires both a gradient of the Ran GTPase and importins.

## Defects in Primary Cilia Underlie Many Diseases

For many years, the existence and function of the primary cilium was ignored. However, this situation has changed dramatically over the last decade as it has become appreciated that defects in intraflagellar transport result in the loss of primary cilia in mice (Figure 18-35d), and as diseases have been traced to defects in primary cilia and IFT. One of the first clues came from the discovery that

loss of a mammalian homolog of a Chlamydomonas IFT protein results in defects in the primary cilia and causes autosomal dominant polycystic kidney disease (ADPKD). It is believed that the primary cilia on the epithelial cells of the kidney collecting tubule act as mechanochemical sensors to measure the rate of fluid flow by the degree to which they are bent.

In another example, patients with Bardet-Biedl syndrome have retinal degeneration, polydactyly (from the Greek for "many fingers"), and obesity. The syndrome can be caused by mutations in any one of 14 genes and has been traced to defects in the function of primary cilia. Many of these genes encode subunits of the BBsome, an octameric complex that forms a coat with structural elements in common with COPI, COPII, and clathrin (described in Chapter 14) and that traffics membrane proteins to cilia. While defects in many of the BBsome's components do not affect the structure of the primary cilium itself, they result in a lack of specific membrane receptors that would normally be delivered to primary cilia through the interaction of the BBsome with the IFT apparatus. For example, the polydactyly seen in patients with Bardet-Biedl syndrome is due to a loss of localized Hedgehog signaling (see Chapter 16) in the primary cilium that is necessary for patterning during embryogenesis.

## KEY CONCEPTS OF SECTION 18.5

## Cilia and Flagella: Microtubule-Based Surface Structures

- Cilia and flagella are microtubule-based cell-surface structures with a characteristic central pair of singlet microtubules and nine sets of outer doublet microtubules (see Figure 18-31).
- All cilia and flagella grow from basal bodies, structures with nine sets of outer triplet microtubules that are closely related to centrioles.
- Axonemal dyneins attached to the A tubule on one doublet microtubule interact with the B tubule of another to bend cilia and flagella.
- Cilia and flagella have a mechanism, intraflagellar transport (IFT), in which material is transported to the tip by kinesin-2 and from the tip back to the base by cytoplasmic dynein. IFT regulates the function and length of cilia and flagella.
- Many cells have on their surface a single nonmotile primary cilium, which lacks the normal central pair of microtubules and the dynein arms of motile cilia. The primary cilium functions as a sensory organelle, with receptors for extracellular signals localized to its plasma membrane. Due to its sensory function, many diseases result from defects in receptor localization or in the structure of the primary cilium itself.


### 18.6 Mitosis

Of all the processes that permit the existence and perpetuation of life, perhaps the most critical is the ability of cells to accurately duplicate and then faithfully segregate their chromosomes at each cell division. During the cell cycle, a highly regulated process discussed in Chapter 19, cells duplicate their chromosomes precisely once during a period known as $S$ phase (for DNA synthesis phase). Once the individual chromosomes have been duplicated, they are held together by proteins called cohesins. The cells then pass through a period called $\mathrm{G}_{2}$ (for gap 2) before entering mitosis, the process by which the duplicated chromosomes are segregated to the daughter cells. This process has to be very precise-loss or gain of a chromosome can either be lethal to the cell (in which case it is often not detected) or cause severe complications for the cell. It is estimated that yeast only mis-segregates one of its 16 chromosomes every 100,000 cell divisions, which makes mitosis one of the most accurate processes in biology. To achieve this accuracy, the process must be highly regulated so that it proceeds in an orderly series of steps and errors are not made. The timing and mechanisms that ensure the fidelity of mitosis are closely regulated by the cell cycle circuitry that we discuss in detail in Chapter 19. Here we limit our discussion of this circuitry as it applies to microtubules and the mechanics of mitosis.

## Centrosomes Duplicate Early in the Cell Cycle in Preparation for Mitosis

In order to separate the chromosomes during mitosis, cells duplicate their MTOCs-their centrosomes-in a manner coordinated with the duplication of their chromosomes in $S$ phase (Figure 18-36). As we discuss in Chapter 19, the cell
![li4194542480e](li4194542480e.jpg)

FIGURE 18-36 Relation of centrosome duplication to the cell cycle. Centrosome duplication, which is initiated by the $\mathrm{G}_{1} / \mathrm{S}$ phase CDKs and Plk4 (step $\mathbf{\square}$ ), results in the pair of centrioles (green) separating and a daughter centriole (blue) budding from each. By the $\mathrm{G}_{2}$ phase, growth of the daughter centrioles is complete, but the two pairs of centrioles remain within a single centrosomal complex. Early in mitosis, driven by the activation of M phase CDKs (step $\mathbf{\square}$ ), the centrosome splits, each half nucleates assembly of microtubules, and the two centriole pairs are moved to opposite sides of the nucleus. The amount of pericentriolar material and the microtubule nucleation activity of the centrosomes increases greatly in mitosis. In mitosis, each MTOC is called a spindle pole.

![li4204542480e](li4204542480e.jpg)

Interphase
![li4214542480e](li4214542480e.jpg)

Chromosome duplication and cohesion
Centrosome duplication
![li4224542480e](li4224542480e.jpg)

Prophase
![li4234542480e](li4234542480e.jpg)

Breakdown of interphase microtubule array and its replacement by mitotic asters Mitotic aster separation Chromosome condensation Kinetochore assembly
![li4244542480e](li4244542480e.jpg)

Prometaphase
![li4254542480e](li4254542480e.jpg)

Nuclear envelope breakdown Chromosomes captured, bi-oriented and brought to the spindle equator
![li4264542480e](li4264542480e.jpg)

Metaphase
![li4274542480e](li4274542480e.jpg)

Chromosomes aligned at the metaphase plate

FIGURE 18-37 The stages of mitosis. (a) Upper panels show stages in cultured cells stained blue for DNA and green for tubulin. Lower diagrams show the different stages and the events that occur in each. Mitosis is a continuous process that has been divided into stages for ease of description. (b) Parts of a condensed chromosome in mitosis. The duplicated chromosome consists of two sister chromatids (each is a single DNA duplex), held together by cohesins at a constricted region called the centromere. The centromere is also the site where the kinetochore, which will make attachments to the kinetochore microtubules, forms. [Part (a) micrographs courtesy of Torsten Wittmann.]
cycle is largely driven by the association of cell-cycle-specific cyclins with cyclin-dependent kinases (CDKs). Two kinases$\mathrm{G}_{1} / \mathrm{S}$ phase CDKs and another type called Plk4—promote centrosome duplication. The duplicated centrosomes separate as the cells enter mitosis in a process known as centrosome disjunction, and then nucleate the assembly of microtubules to become the two poles of the mitotic spindle. The number of centrosomes in animal cells has to be very carefully controlled. In fact, many tumor cells have more than two centrosomes, which contributes to genetic instability resulting from mis-segregation of chromosomes and hence aneuploidy (unequal numbers of chromosomes). The reasons why aneuploidy results in cancer are discussed in detail in Chapter 24.

As cells enter mitosis, the activity of the two MTOCstheir ability to nucleate microtubules-increases greatly as they accumulate more pericentriolar material. Because the assemblies of microtubules radiating from these two MTOCs now resemble stars, they are often called mitotic asters.

## Mitosis Can Be Divided into Six Stages

Mitosis has been divided up into several stages for ease of description (Figure 18-37a), but in reality it is a continuous process. Here we review the major events of each stage.

The first stage of mitosis, called prophase, is signaled by a number of coordinated and dramatic events. First, the interphase array of microtubules is replaced as the duplicated centrosomes become more active in microtubule nucleation. This activity provides two sites of assembly for dynamic microtubules, forming the mitotic asters. Additionally, the dynamics of the growing microtubules themselves increase due to changes in the activities of +TIPs at their $(+)$ ends. The two asters are then moved to opposite sides of the nucleus by the action of bipolar kinesin-5 motors (see Figure 18-20) that push the intermingling microtubules apart. The separated centrosomes will become the two poles of the mitotic spindle, the microtubule-based

![li4284542480e](li4284542480e.jpg)

Anaphase
![li4294542480e](li4294542480e.jpg)

APC/C activated and cohesins degraded Anaphase A: Chromosome movement to poles Anaphase B: Spindle pole separation
![li4304542480e](li4304542480e.jpg)

Telophase
![li4314542480e](li4314542480e.jpg)

Nuclear envelope reassembly Assembly of contractile ring
![li4324542480e](li4324542480e.jpg)

Cytokinesis
![li4334542480e](li4334542480e.jpg)

Nuclear envelope reassembly Assembly of contractile ring
![li4344542480e](li4344542480e.jpg)

Re-formation of interphase microtubule array Contractile ring forms cleavage furrow
structure that separates chromosomes. Second, protein synthesis is switched from being dependent to independent on the mRNA $5^{\prime}$ cap (see Section 5.4), and the internal membrane systems, normally dependent on the interphase array of microtubules, are disassembled. In addition, endocytosis and exocytosis are halted, and the microfilaments are generally rearranged to give rise to a rounded cell. In the nucleus, the nucleolus breaks down and chromosomes begin to condense. The cohesins holding together each pair of duplicated chromosomes, or sister chromatids as they are called at this stage, are degraded except at the centromeric region, where the two sister chromatids remain linked by intact cohesins (Figure 18-37b). Also during prophase, specialized structures called kinetochores, which will become sites of microtubule attachment, assemble at the centromeric region of each sister chromatid. As discussed in more detail in Chapter 19, all these events are coordinated by a rapid increase in the activity of the mitotic cyclin-CDK complex, which is a kinase that drives cell cycle progression by phosphorylating multiple proteins.

The next stage of mitosis, prometaphase, is initiated by the breakdown of the nuclear envelope and its retraction into the endoplasmic reticulum, the disassembly of nuclear pores, and the disassembly of the lamin-based nuclear lamina. Microtubules assembled from the spindle poles search for and "capture" chromosome pairs by associating with their kinetochores. Each chromatid has a kinetochore, so a sister chromatid pair has two kinetochores, each of which becomes attached through the microtubules to a different spindle pole during prometaphase in a critical process we
discuss in detail below. The chromosome pairs then become aligned at a point equidistant between the two spindle poles. Prometaphase continues until all chromosomes have become aligned, at which point the cell enters the next stage, metaphase.

When the cell detects that all chromosomes are attached to the spindle correctly, the next stage, anaphase, is induced by activation of the anaphase-promoting complex or cyclosome (APC/C) (discussed in Chapter 19). The activated APC/C (through several intermediary steps) ultimately leads to the destruction of the cohesins that were holding the sister chromatids together, so that each chromatid can be pulled to its respective pole by the microtubules attached to its kinetochore. This movement is known as anaphase $A$. A separate and distinct movement also occurs: the movement of the spindle poles farther apart in a process known as anaphase B. Now that the chromosomes have separated, the cell enters telophase, when the nuclear envelope re-forms, the chromosomes decondense, and the cell is pinched into two daughter cells by the contractile ring during cytokinesis.

## The Mitotic Spindle Contains Three Classes of Microtubules

Before we discuss the mechanisms involved in the remarkable process of mitosis, it is important to understand the three distinct classes of microtubules that emanate from the spindle poles, which is where their ( - ) ends are embedded. The astral microtubules extend from the spindle poles to the

![li4354542480e](li4354542480e.jpg)

FIGURE 18-38 Mitotic spindles have three distinct classes of microtubules. (a) In this high-voltage electron micrograph, microtubules were stained with biotin-tagged anti-tubulin antibodies to increase their size. The large cylindrical objects are chromosomes. (b) Schematic diagram corresponding to the metaphase cell in (a). Three sets of microtubules (MTs) make up the mitotic apparatus.
cell cortex (Figure 18-38). By interacting with the cortex, the astral microtubules perform the critical function of orienting the spindle to the axis of cell division. The kinetochore microtubules function by a search-and-capture mechanism to link the spindle poles to the kinetochores on sister chromatid pairs. During anaphase A, the kinetochore microtubules transport the newly separated chromosomes to their respective poles. The polar microtubules extend from each spindle pole toward the opposite one and interact in an antiparallel manner. These microtubules are responsible initially for pushing the duplicated centrosomes apart during prophase, then for maintaining the structure of the spindle, and then for pushing the spindle poles apart in anaphase B.

Note that all the microtubules in each half of the symmetrical spindle have the same orientation except for some polar microtubules, which extend beyond the midpoint and interdigitate with polar microtubules from the opposite pole.

## Microtubule Dynamics Increase Dramatically in Mitosis

Although we have drawn static images of the stages of mitosis, microtubules in all stages of mitosis are highly dynamic. As we have seen, as cells enter mitosis, the ability of their centrosomes to nucleate the assembly of microtubules increases significantly (see Figure 18-36). In addition, microtubules become much more dynamic. How was this determined? In principle, one could label microtubules with a fluorescent tag and watch their individual behaviors, but practically speaking,
(b)
![li4364542480e](li4364542480e.jpg)

All the microtubules have their (-) ends at the poles. Astral microtubules project toward the cortex and are linked to it. Kinetochore microtubules are connected to chromosomes. Polar microtubules project toward the cell center with their distal ( + ) ends overlapping. The spindle pole and associated microtubules is also known as a mitotic aster. (Part (a) courtesy of J. Richard McIntosh.)
there are too many microtubules in a mitotic spindle to follow. To get an average value for the dynamic instability of these microtubules, researchers can introduce fluorescently labeled tubulin into cells, which becomes incorporated randomly into all microtubules. They can then bleach the fluorescent label in a small region of the mitotic spindle and measure the rate at which fluorescence comes back using a technique known as fluorescence recovery after photobleaching (FRAP) (see Figure 4-23). Since the recovery of fluorescence is due to the assembly of new microtubules from soluble fluorescent tubulin dimers, its rate represents the average rate at which microtubules turn over. In a mitotic spindle, their half-life is about 15 seconds, whereas in an interphase cell, it is about 5 minutes. It should be noted that these are bulk measurements and that individual microtubules can be more stable or dynamic, as we will see.

What makes microtubules more dynamic in mitosis? As we discussed earlier, dynamic instability is a measure of relative contributions of growth rates, shrinkage rates, catastrophes, and rescues (see Figure 18-9). Analysis of microtubule dynamics in vivo shows that the enhanced instability of individual microtubules in mitosis is generated mainly by an increase in catastrophes and a decrease in rescues, with little change in rates of growth (i.e., lengthening) or shrinkage (i.e., shortening). Studies with extracts from frog oocytes have suggested that the main factor enhancing catastrophes in both interphase and mitotic extracts is depolymerization by kinesin-13 proteins. This can be seen in an in vitro assay in which microtubule assembly

![li4374542480e](li4374542480e.jpg)

EXPERIMENTAL FIGURE 18-39 Microtubule dynamics increase in mitosis due to loss of a stabilizing MAP. (a) These three panels reveal the ability of centrosomes to assemble microtubules under various conditions: with pure tubulin (left); with tubulin and the destabilizing protein kinesin-13 (middle); and with tubulin, kinesin-13, and the stabilizing protein XMAP215 (Xenopus MAP of 215 kDa ) (right). Further analysis shows that the major effect of XMAP215 is to suppress catastrophes induced by kinesin-13. (b) The increased dynamics of microtubules in mitosis is due to the inactivation of XMAP215 by phosphorylation. (c) Diagram comparing the stabilities of microtubules in interphase and in mitosis. Note that in addition to the decrease in stability in mitosis, the ability of MTOCs to nucleate microtubules increases dramatically in mitosis. [Part (a) republished with permission of AAAS, from Kinoshita et al., "Reconstitution of Physiological Microtubule Dynamics Using Purified Components," Science 294, no 5545,1340-1343 (2001). Part (b) data from Kinoshita et al, 2002, Trends Cell Biol. 12:267-273.]
from pure tubulin is nucleated from purified centrosomes (Figure 18-39a). If kinesin-13 is added to the assay, many fewer microtubules are formed. However, if the protein XMAP215, which enhances assembly at the $(+)$ end, is added with the kinesin-13, many microtubules are formed due to a dramatic reduction in catastrophe frequency. It turns out that the activity of kinesin-13 does not change significantly during the cell cycle, whereas the activity of XMAP215 is inhibited by its phosphorylation during mitosis (Figure 18-39b). This results in much more unstable (more dynamic) microtubules as the cell enters mitosis (Figure 18-39c).

## Mitotic Asters Are Pushed Apart by Kinesin-5 and Oriented by Dynein

As the two mitotic asters form, they generate interdigitated microtubules of opposite polarity between them. During prophase, the bipolar kinesin-5 interacts with the antiparallel microtubules and, by moving toward the $(+)$ end of each microtubule, slides them apart and thereby pushes the two asters apart. The $(-)$ end-directed motor, cytoplasmic dynein, can also contribute to aster separation as well as orienting the spindle appropriately in the cell. Dynein does this by associating with the cell cortex and pulling on microtubules nucleated from the mitotic asters. As we discuss
shortly, this same mechanism is used to elongate the spindle during anaphase B (see Figure 18-43 below).

## Chromosomes Are Captured and Oriented During Prometaphase

Kinetochores, the structures that mediate attachment between chromosomes and microtubules, assemble on each sister chromatid at a region called the centromere. The centromere is a constricted region of the condensed chromosome defined by a centromeric DNA sequence. Centromeric DNA can vary enormously in size; in budding yeast it is about 125 bp , whereas in humans it is on the order of 1 Mb (see Chapter 8). Kinetochores contain many protein complexes to facilitate the linkage between centromeric DNA and microtubules. In animal cells, the kinetochore consists of a centromeric DNA layer and inner and outer kinetochore layers, with the $(+)$ ends of kinetochore microtubules terminating in the outer layer (Figure 18-40). Yeast kinetochores are attached by a single microtubule to their spindle pole, human kinetochores are attached by about 30 microtubules, and plant chromosomes by hundreds.

How does a kinetochore become attached to microtubules in prometaphase? Microtubules nucleated from the spindle poles are very dynamic, and when they contact a

![li4384542480e](li4384542480e.jpg)

FIGURE 18-40 The structure of a mammalian kinetochore. Diagram and electron micrograph of a mammalian kinetochore. [Republished with permission of Springer, from McEwen et al., "A new look at kinetochore structure in vertebrate somatic cells using high-pressure freezing and freeze substitution," Chromosoma, 1998, 107: 6-7 pp. 366-375; permission conveyed through Copyright Clearance Center, Inc.]
kinetochore, either laterally or at their ends, this contact can lead to chromosomal attachment (Figure 18-41a, steps $\mathbf{B}$ and $\mathbf{B}$ ). Microtubules that "capture" kinetochores are selectively stabilized by a reduction in the rate of catastrophes, which increases the chance that the attachment will persist.

Recent studies have uncovered a mechanism involving Ran, a small GTPase, that enhances the chance that microtubules will encounter kinetochores. Recall from Chapter 13 that during interphase, the Ran GTPase cycle is involved in the transport of proteins into and out of the nucleus through nuclear pores (see Figure 13-37). During mitosis, when the nuclear membrane and pores have disassembled, a guanine nucleotide exchange factor for the Ran GTPase is bound to chromosomes, thereby generating a higher local concentration of Ran-GTP in the vicinity of the chromosomes. Because the enzyme that stimulates GTP hydrolysis on Ran-the Ran GAP-is evenly distributed in the cytosol, this generates a gradient of Ran-GTP centered on the chromosomes. Ran-GTP induces the association of cytosolic microtubule-stabilizing factors with the microtubule, resulting in enhanced microtubule growth, and in this way biases growth of microtubules nucleated from spindle poles toward chromosomes.

Once a kinetochore is attached laterally or terminally to a microtubule, dynein-dynactin associates with the kinetochore to move the duplicated chromosome down the microtubule toward the spindle pole. This movement eventually results in an end-on attachment of the microtubule to one kinetochore (Figure 18-41a, step $\mathbf{B}$ ). This movement helps orient the sister chromatid so that the unoccupied kinetochore on the opposite side is pointing toward the distal spindle pole. Eventually a microtubule from the distal pole will capture the free kinetochore; at this point the sister chromatid pair is said to be bi-oriented (Figure 18-41a, step $\mathbf{B}$ ). With the two kinetochores attached to opposite poles, the duplicated chromosome is now under tension, being pulled in both directions by the two sets of kinetochore microtubules. When one or a few chromosomes are bi-oriented, other chromosomes use these existing kinetochore microtubules to contribute to their orientation and movement to the spindle center. This orientation is mediated by kinesin-7
(also known as CENP-E) associated with the free kinetochore, which moves the chromosome to the $(+)$ end of the kinetochore microtubule (Figure 18-41a, step $\mathbf{B}$ ).

## Duplicated Chromosomes Are Aligned by Motors and Microtubule Dynamics

During prometaphase, the chromosomes come to lie at the midpoint between the two spindle poles, called the metaphase plate, in a process known as chromosome congression. During this process, bi-oriented chromosome pairs often oscillate backward and forward before arriving at the metaphase plate. Chromosome congression involves the coordinated activity of several microtubule-based motors together with regulators of microtubule assembly and disassembly (Figure 18-41b). These regulators are localized at the kinetochores, but how they are maintained there is poorly understood-they are not part of the stable kinetochore complexes described in the next section. The oscillating behavior of chromosomes involves lengthening of microtubules attached to one kinetochore and shortening of microtubules attached to the other kinetochore, all without losing their attachments. In metazoans, several microtubule-based motors associated with the kinetochore contribute to this process. First, dynein-dynactin provides the strongest force pulling the chromosome pair toward the more distant pole. This movement requires simultaneous shortening of the microtubule, which is enhanced by kinetochore-localized kinesin-13. The microtubules associated with the other kinetochore have to grow as the chromosome moves. Anchored at this kinetochore is the kinesin-related motor kinesin-7, which holds onto the growing $(+)$ end of the lengthening microtubule. Also contributing to congression is another kinesin, chromokinesin/kinesin-4, which associates with the chromosome arms. Kinesin-4, a (+) end-directed motor, interacts with the polar microtubules to pull the chromosomes toward the center of the spindle. When the chromosomes have congressed to the metaphase plate, dynein-dynactin is released from the kinetochores and streams down the kinetochore microtubules

![li4394542480e](li4394542480e.jpg)

FIGURE 18-41 Chromosome capture and congression in prometaphase. (a) In the first stage of prometaphase, chromosomes become attached either to the end of a microtubule (D) or to the side of a microtubule (D). The chromosome is then drawn toward the spindle pole by dynein-dynactin that becomes associated with one of the kinetochores of the chromosome and moves toward the ( - ) end of the microtubule (D). Eventually, a microtubule from the opposite pole finds and becomes attached to the free kinetochore, and the chromosome is now said to be bi-oriented (D). Once some chromosomes are bi-oriented, others, having established one kinetochore-pole interaction, use CENP-E/kinesin-7 on their free kinetochore to aid in orientation (D). The bi-oriented chromosomes then move to a central point between the spindle poles in a process known as chromosome congression. Note that during these
steps, chromosome arms point away from the closest spindle pole: this is due to chromokinesin/kinesin-4 motors on the chromosome arms moving toward the $(+)$ ends of the polar microtubules. In animal cells, many microtubules associate with each kinetochore. For ease of presentation, only single kinetochore microtubules are shown here. (b) Congression involves bidirectional oscillations of chromosomes, with one set of kinetochore microtubules shortening on one side of the chromosomes and the other set lengthening on the other. On the shortening side, a kinesin-13 protein stimulates microtubule disassembly and a dyneindynactin complex moves the chromosome toward the pole. On the side with lengthening microtubules, kinesin-7 protein holds on to the growing microtubule. The kinetochore also contains many additional protein complexes not shown here. See Cleveland et al., 2003, Cell 112:407-421.
to the poles. These different activities and opposing forces work together to bring all the chromosomes to the metaphase plate, at which point the cell is ready for anaphase.

## The Chromosomal Passenger Complex Regulates Microtubule Attachment at Kinetochores

We have noted that the segregation of chromosomes at mitosis must be very accurate, so it is crucial that all chromosomes are bi-oriented before anaphase begins. During the random kinetochore-to-microtubule attachment process, it is possible for mistakes to be made; for example, both kinetochores
of a sister chromatid pair might attach to microtubules from the same spindle pole. If such attachments persisted during metaphase, it would result in one cell missing a chromosome and the other having an extra one, which would either be lethal or very detrimental. Cells have two important mechanisms to ensure that all chromosomes are correctly bi-oriented before anaphase begins.

The first mechanism ensures that the kinetochoremicrotubule interactions are weak until bi-orientation occurs. When a chromosome is correctly bi-oriented, tension is produced across the chromosome, and this tension leads to the kinetochore-microtubule attachments becoming stabilized.

To understand how this works, we need to look a bit more closely at the molecular components that link a kinetochore to a microtubule. As we discussed in Chapter 8, kinetochores assemble on regions of chromosomal DNA marked by a centromere-specific H3 histone variant called CENP-A. This variant marks the site for kinetochore assembly, which is a very complicated process. About half a dozen distinct stable protein complexes, consisting of more than 40 different proteins, have been shown to associate with this centromeric region in yeast. Essentially all these protein complexes are conserved in humans, which is not surprising given the fundamental importance of kinetochores. One of these, the so-called Ndc80 complex, is long and flexible, and many copies of it link the inner kinetochore with the ( + ) end of the microtubule in a sleevelike arrangement (Figure 18-42a). The function of Ndc80 and many of the associated factors at the kinetochore is regulated by the chromosomal passenger complex (CPC). This complex associates with the inner kinetochore early in mitosis, and among its components is a protein kinase called Aurora B. Once the CPC is associated with the kinetochore, Aurora B can phosphorylate several components in the near vicinity, including the Ndc80 complex,
![li4404542480e](li4404542480e.jpg)

FIGURE 18-42 CPC regulation of microtubule-kinetochore attachment. The Ndc80 complex forms a critical and regulated attachment between the kinetochore and microtubule ( + ) end. (a) Diagram showing the sleevelike arrangement of the Ndc80 complex linking the inner kinetochore to the ( + ) end of a microtubule embedded in the outer kinetochore. See S. Santaguida and A. Musacchio, 2009, EMBO J. 28:2511-2531. (b) Diagram of the relationship between the chromosomal passenger complex (CPC), which is associated with the inner kinetochore and contains the kinase Aurora B, and the outer kinetochore,
which loosens the attachment of Ndc80 to the microtubule. The phosphorylation of these components is not stable, however: another protein, the phosphatase PP1, which is associated with the outer kinetochore, can dephosphorylate them. Thus when the kinetochores on a pair of sister chromatids are not under tension, Ndc80 is continually phosphorylated by Aurora B and dephosphorylated by PP1. The result is a weak interaction between the kinetochore and microtubule. However, when bi-orientation occurs, the tension generated pulls on both kinetochores and extends the flexible Ndc80 complex to increase the spacing between the inner and outer kinetochores (Figure 18-42b, c). As a result of these movements, Ndc80 cannot be phosphorylated by Aurora B, and the dephosphorylated state of Ndc80 renders it more firmly attached to the microtubule. In this way, microtubule attachments to bi-oriented chromosomes are selectively stabilized.

While the CPC is important for bi-orientation of each individual chromosome, it does not ensure that all chromosomes are bi-oriented before anaphase begins. The second mechanism to ensure correct chromosome segregation is the spindle assembly checkpoint pathway, a signaling pathway that stops the progression of the cell cycle into anaphase
![li4414542480e](li4414542480e.jpg)
where the phosphatase PP1 binds. Notice that when both kinetochores are under tension, the outer kinetochores move away from the CPC; as a result, Aurora B cannot phosphorylate components in the outer kinetochore, which include the microtubule-binding site of the Ndc80 complex. (c) Cell in metaphase stained for tubulin (red), DNA (blue), Aurora B kinase (green), and the outer kinetochore (magenta). Notice how the outer kinetochore is pulled away from Aurora B (inset). [Part (c) reprinted by permission from Macmillan Publishers Ltd: from Ruchaud S. et al., "Chromosomal passengers: conducting cell division," Nature Reviews Molecular Cell Biology, 2007, 8:798-812.]

until tension is present at all the kinetochores. Even a single unattached, or inappropriately attached, kinetochore can activate the spindle assembly checkpoint pathway and pause the cell cycle until the error is corrected. This mechanism, discussed in detail in Chapter 19, guarantees that all the chromosomes are correctly bi-oriented before the cell proceeds into anaphase.

## Anaphase A Moves Chromosomes to Poles by Microtubule Shortening

The onset of anaphase A is one of the most dramatic movements that can be observed in the light microscope. When the spindle assembly checkpoint has been passed, APC/C activation induces proteolysis of the remaining cohesins holding the sister chromatids together. Suddenly, the two paired sister chromatids separate from each other and are drawn to their respective poles. The movement is sudden because the kinetochore microtubules are under tension, and as
soon as the cohesin attachments between the chromatids are removed, the separated chromatids are free to move.

Experiments with isolated metaphase chromosomes have shown that anaphase A movement can be powered by microtubule shortening, using the stored structural strain released by removal of the GTP-bound tubulin subunits at the microtubule tip. This mechanism can be nicely demonstrated in vitro. When metaphase chromosomes are added to purified microtubules, they bind preferentially to the $(+)$ ends of the microtubules. Dilution of the mixture to reduce the concentration of free tubulin dimers results in the movement of the chromosomes toward the $(-)$ ends by microtubule depolymerization at the chromosome-bound $(+)$ ends. In addition, recent experiments have shown that in Drosophila, two members of the microtubuledepolymerizing kinesin-13 protein family (see Figure 18-15) also contribute to chromosome movement in anaphase A. One of these kinesin-13 proteins is localized at the kinetochore and enhances disassembly there (Figure 18-43, 18), and the other is localized at the spindle pole, enhancing depolymerization
![li4424542480e](li4424542480e.jpg)

FIGURE 18-43 Chromosome movement and spindle pole separation in anaphase. Anaphase A movement is powered by microtubule-shortening kinesin-13 proteins at the kinetochore (18) and at the spindle pole (18). Note that the chromosome arms still point away from the spindle poles due to associated chromokinesin/ kinesin-4 members, so the depolymerization force has to be able to
overcome the force pulling the arms toward the center of the spindle. Anaphase B also has two components: sliding of antiparallel polar microtubules powered by a kinesin-5 (+) end-directed motor (18), and pulling on astral microtubules by dynein-dynactin located at the cell cortex (18). Arrows indicate the direction of movement generated by the respective forces. See Cleveland et al., 2003, Cell 112:407-421.

![li4434542480e](li4434542480e.jpg)

EXPERIMENTAL FIGURE 18-44 Mitotic spindles can form in the absence of centrosomes. Centrosome-free extracts can be isolated from frog oocytes arrested in mitosis by centrifuging eggs to separate a soluble material from the organelles and yolk. When fluorescently labeled tubulin (green) is added to extracts of the soluble material together with beads covered with DNA (red), mitotic spindles spontaneously form
and the beads from randomly nucleated microtubules. See Kinoshita et al., 2002, Trends Cell Biol. 12:267-273, and Antonio et al., 2000, Cell 102:425. [Micrograph republished with permission of Nature, from Heald, R. et al., "Self-organization of microtubules into bipolar spindles around artificial chromosomes in Xenopus egg extracts," Nature, 1996, 382:6590, pp. 420-425.]
there (Figure 18-43, 18). Thus, at least in the fly, anaphase A is powered in part by kinesin-13 proteins specifically localized at the kinetochore and spindle pole to shorten the kinetochore microtubules at both their $(+)$ and $(-)$ ends, drawing the chromosomes to the poles.

## Anaphase B Separates Poles by the Combined Action of Kinesins and Dynein

The second part of anaphase involves separation of the spindle poles in a process known as anaphase B. A major contributor to this movement is the involvement of the bipolar kinesin-5 proteins (Figure 18-43, 18). These motors associate with the overlapping polar microtubules, and since they are $(+)$ end-directed motors, they push the poles apart. While this is happening, the polar microtubules have to grow to accommodate the increased distance between the spindle poles. Another motor-the microtubule ( - ) end-directed cytoplasmic dynein, localized and anchored on the cell cortex-pulls on the astral microtubules and thus helps separate the spindle poles (Figure 18-43, 18).

## Additional Mechanisms Contribute to Spindle Formation

There are a number of cases in vivo in which spindles form in the absence of centrosomes, including plant-cell mitosis and animal-cell meiosis in females. This observation implies that nucleation of microtubules from centrosomes is not the only way in which a spindle can form. Studies exploiting mitotic extracts from frog eggs-extracts that do not contain centrosomesshow that the addition of beads covered with DNA is sufficient to assemble a relatively normal mitotic spindle (Figure 18-44). In this system, the beads induce the assembly of microtubules, and factors in the extract cooperate to make a spindle. One of the factors necessary for this reaction is cytoplasmic dynein, which is proposed to bind to two microtubules and migrate to their $(-)$ ends, thereby drawing them together.

As mentioned in Section 18.1, a newly discovered $\gamma$-TuRC-associated complex, the augmin complex, also contributes microtubules to the mitotic spindle. In late
![li4444542480e](li4444542480e.jpg)

EXPERIMENTAL FIGURE 18-45 The chromosomal passenger complex (CPC) remains at the spindle midzone during anaphase and telophase. Micrographs of a cell in late anaphase (left) and telophase (right), showing microtubules (red), DNA (blue), Aurora B kinase (green), and kinetochores (magenta). Notice how the Aurora B, which is part of the CPC, concentrates in the region where the polar microtubules overlap and where the contractile ring will form. Scale bar $5 \mu \mathrm{~m}$. [Reprinted by permission from Macmillan Publishers Ltd: from Ruchaud S. et al., "Chromosomal passengers: conducting cell division," Nature Reviews Molecular Cell Biology, 2007, 8:798-812.]

exchange factor for Ran (Ran-GEF), is localized there by binding to chromatin. Ran-GTP binds to the importin, changing its conformation to release the transported protein, and then the importin-Ran-GTP complex leaves the nucleus to return to the cytosol through a nuclear pore. After nuclear envelope breakdown in mitosis, because the Ran-GEF is associated with chromatin, there is a gradient of Ran-GTP around chromosomes. This Ran-GTP releases a protein called TPX from importin. TPX binds the augmin complex and $\gamma$-TuRC to drive the assembly of microtubules in the vicinity of the chromatin, which can be used to make the spindle. In spindles with centrosomes, this pathway also exists to enhance the number of kinetochore and polar microtubules. It involves an association of the TPX-augmin- $\gamma$-TuRC complex with the side of an existing microtubule to nucleate assembly of new ones at a shallow angle to the mother microtubule, and thereby enhance the density of microtubules with the same polarity.

## Cytokinesis Splits the Duplicated Cell in Two

During late anaphase and telophase in animal cells, the cell assembles a microfilament-based contractile ring attached
to the plasma membrane that will eventually contract and pinch the cell into two, a process known as cytokinesis (see Figure 18-37). The contractile ring is a thin band of actin filaments of mixed polarity interspersed with myosin-II bipolar filaments (see Figure 17-35). On receiving a signal, the ring contracts, first to generate a cleavage furrow and then to pinch the cell into two.

Two aspects of the contractile ring are essential to its function. First, it has to be appropriately placed in the cell. It is known that this placement is determined by a signal provided by the spindle, so that the ring forms equidistant between the two spindle poles. The signal is provided, at least in part, by the chromosomal passenger complex (CPC) that regulates the attachment of microtubules to kinetochores during prometaphase (see Figure 18-42b). Up to anaphase, the CPC is associated with the inner kinetochores of unseparated chromatids. When anaphase begins, it leaves the centromeres and associates with the overlapping polar microtubules at the center of the spindle (Figure 18-45). There the CPC recruits another protein complex, centralspindlin, that includes a ( + ) enddirected kinesin motor protein, which concentrates at the middle of the spindle due to its motor activity. As anaphase B
![li4454542480e](li4454542480e.jpg)

FIGURE 18-46 Mitosis in a flowering plant cell. Immunofluorescence micrographs (top) and corresponding diagrams (bottom) showing arrangement of microtubules in interphase and mitotic plant cells. A cortical array of microtubules girdles a cell during interphase. As the cell enters prophase, the microtubules (green), together with actin filaments (red), assemble under the cell cortex into a preprophase band, which marks the future cortical division site. As the cell enters prometaphase and metaphase, a spindle similar to that seen in animal cells forms. However,
due to the cell wall, cytokinesis in plant cells is very different from that in in animal cells. Microtubules deliver vesicles whose membranes are used to assemble a membrane network called a phragmoplast, whose organization is defined by actin filaments linked to the cortical division site. Eventually, the phragmoplast becomes part of the plasma membranes of the two daughter cells. Enzymes secreted from the vesicles then build a cell wall between the two daughter cells. See G. Jürgens, 2005, Annu. Rev. Plant Biol. 56:281-299 [Micrographs courtesy of Susan M. Wick.]

continues, centralspindlin recruits a guanine nucleotide exchange factor for RhoA. Recall from Chapter 17 that Rho proteins are small GTP-binding proteins that are activated by exchange factors to catalyze the exchange of GDP for GTP (see Figure 17-41). Once activated, the RhoA-GTP activates a formin protein to drive the nucleation and assembly of actin filaments that make up the contractile ring (see Figure 17-43). In this way, the position of the spindle directly defines the site of contractile ring formation, and hence cytokinesis.

The second important aspect of the contractile ring is the timing of its contraction: if it were to contract before all chromosomes had moved to their respective poles, disastrous genetic consequences would ensue. As we discuss in Chapter 19, a signaling pathway has been discovered in budding yeast called the spindle position checkpoint, which pauses the cell cycle to ensure that cytokinesis does not occur until the spindle is appropriately oriented. The mechanism of this coordination in animal cells is still being unraveled.

## Plant Cells Reorganize Their Microtubules and Build a New Cell Wall in Mitosis

12Interphase plant cells lack a central MTOC that organizes microtubules into the radiating array typical of animal cells. Instead, numerous MTOCs containing $\gamma$-tubulin line the cortex of plant cells and nucleate the assembly of transverse bands of microtubules below the cell wall (Figure 18-46, left). These microtubules, which are of mixed polarity, are released from the cortical MTOCs by the action of katanin, a microtubule-severing protein; loss of katanin gives rise to very long microtubules and misshapen cells. The reason for this is that these cortical microtubules, which are cross-linked by plant-specific MAPs, aid in laying down extracellular cellulose microfibrils, the main component of the rigid cell wall (see Figure 20-40).

Although mitotic events in plant cells are generally similar to those in animal cells, formation of the spindle and cytokinesis have unique features in plants (see Figure 18-46). Plant cells bundle their cortical microtubules and actin filaments into a preprophase band and reorganize them into a spindle at prophase without the aid of centrosomes. The site of the preprophase band defines the later cortical division site. At metaphase, the mitotic apparatus appears much the same in plant and animal cells. Because plants have cell walls, the division of the cell into two is quite different from animal cells and requires the assembly of a new wall between the daughter cells. Golgi-derived vesicles, which appear at telophase, are transported along microtubules to form the nascent cell plate. The cell plate expands and is guided toward the division site by actin filaments to form the phragmoplast, a membrane structure that replaces the animal-cell contractile ring. The membranes of the vesicles forming the phragmoplast become the plasma membranes of the daughter cells. The contents of these vesicles, such as polysaccharide precursors of cellulose and pectin, form the early cell plate, which develops into the new cell wall between the daughter cells.

## KEY CONCEPTS OF SECTION 18.6

## Mitosis

- Mitosis-the accurate separation of duplicated chromosomes-involves a molecular machine comprising dynamic microtubules and microtubule-associated motors.
- The mitotic spindle has three classes of microtubules, all emanating from the spindle poles: kinetochore microtubules, which attach to chromosomes; polar microtubules, which extend from each spindle pole and overlap in the middle of the spindle; and astral microtubules, which extend to the cell cortex (see Figure 18-38).
- In the first stage of mitosis, prophase, the nuclear chromosomes condense and the spindle poles move to either side of the nucleus (see Figure 18-37).
- At prometaphase, the nuclear envelope breaks down and microtubules emanating from the spindle poles capture sister chromatid pairs at their kinetochores. The two kinetochores (one on each chromatid) become attached to opposite spindle poles (bi-oriented), which allows the chromosome to congress to the middle of the spindle.
- The chromosomal passenger complex (CPC) associated with the inner kinetochore keeps microtubule attachments weak by the activity of its kinase component Aurora B, which phosphorylates critical kinetochore proteins. When a chromosome is bi-oriented, tension is generated, and the Aurora B substrates are pulled away from the kinase (see Figure 18-42). Without phosphorylation of kinetochore proteins by Aurora B, the chromosome-kinetochore attachment becomes stable.
- At metaphase, chromosomes are aligned on the metaphase plate. The spindle assembly checkpoint pathway monitors unattached kinetochores and delays anaphase until all chromosomes are attached.
- At anaphase, duplicated chromosomes are separated and moved toward the spindle poles by shortening of the kinetochore microtubules at both the kinetochore and spindle pole (anaphase A). The spindle poles also move apart, pushed by bipolar kinesin-5 moving toward the (+) ends of the polar microtubules (anaphase B). Spindle separation is also facilitated by cortically located dynein pulling on astral microtubules (see Figure 18-43).
- Since the mitotic spindle has the ability to self-assemble in the absence of centrosomes, additional mechanisms contribute to the assembly of the mitotic spindle.
- The actin-myosin-based contractile ring, the position of which is determined by the position of the spindle, contracts to pinch the cell in two during cytokinesis.
- In plants, cell division involves the delivery of membranes by microtubules to assemble the phragmoplast, which becomes the plasma membrane of the two daughter cells.

### 18.7 Intermediate Filaments

The third major filament system of eukaryotes comprises intermediate filaments (IFs). This name reflects their diameter of about 10 nm , which is intermediate between the $6-8-\mathrm{nm}$ diameter of microfilaments and the myosin thick filaments of skeletal muscle. Intermediate filaments extend throughout the cytoplasm as well as lining the inner nuclear envelope of interphase animal cells (Figure 18-47). Intermediate filaments have several unique properties that distinguish them from microfilaments and microtubules. First, they are biochemically much more heterogeneousthat is, many different, but evolutionarily related, IF subunits exist-and are often expressed in a tissue-dependent manner. Second, they have great tensile strength, as is clearly demonstrated by hair and nails, which consist primarily of the intermediate filaments of dead cells. Third, they do not have an intrinsic polarity like microfilaments and microtubules, and their constituent subunits do not bind a nucleotide. Fourth, because they have no intrinsic polarity, it is not surprising that there are no known motors that use them as tracks. Fifth, although they are dynamic in terms of subunit exchange, they are much more
![li4464542480e](li4464542480e.jpg)

EXPERIMENTAL FIGURE 18-47 Localization of two types of intermediate filaments in an epithelial cell. Immunofluorescence micrograph of an epithelial cell doubly stained with antibodies to keratin (red) and lamin (blue). A meshwork of lamin intermediate filaments can be seen underlying the nuclear membrane, whereas the keratin filaments extend from the nucleus to the plasma membrane. (Courtesy of Robert D. Goldman.)
stable than microfilaments and microtubules because the exchange rate is much slower. Indeed, a standard way to purify intermediate filaments is to subject cells to harsh extraction conditions in a detergent so that all membranes, microfilaments, and microtubules are solubilized, leaving a residue that is almost exclusively intermediate filaments. Finally, intermediate filaments are not found in all eukaryotes. Fungi and plants do not have intermediate filaments, and insects have only one class, represented by two genes that express lamins A/C and B.

These properties make intermediate filaments unique and important structures of metazoans. Their importance is underscored by the identification of hundreds of clinical disorders, some of which are discussed here, associated with mutations in genes encoding IF proteins. To understand their contributions to cell and tissue structure, we first examine the structure of IF proteins and see how they assemble into filaments. Next we discuss their dynamics, then describe the different classes of intermediate filaments and the functions they perform.

## Intermediate Filaments Are Assembled from Subunit Dimers

Intermediate filaments are encoded in the human genome by 70 different genes in at least five subfamilies. The defining feature of IF proteins is the presence of a conserved $\alpha$-helical rod domain of about 310 residues that has the sequence features of a coiled-coil motif (see Figure 3-9a).

The primary building block of intermediate filaments is a dimer held together through the rod domains, which associate as a coiled coil (Figure 18-48a). These dimers then associate in an offset fashion to make tetramers, in which the two dimers have opposite orientations (Figure 18-48b). Tetramers are assembled end to end and interlocked into long protofilaments. Four protofilaments associate into a protofibril, and four protofibrils associate side to side to generate the $10-\mathrm{nm}$ filament. Thus an intermediate filament has 16 protofilaments in it (Figure 18-48c). Flanking the rod domain of each dimer are nonhelical $\mathrm{N}-$ and $\mathrm{C}-$ terminal domains of different sizes, characteristic of each IF class (Figure 18-48d). Because the tetramer is symmetric, intermediate filaments have no polarity. This description of the filament is based on its structure rather than its mechanism of assembly: at present it is not yet clear how intermediate filaments are assembled in vivo. Unlike microfilaments and microtubules, there are no known intermediate filament nucleating, sequestering, capping, or filament-severing proteins.

## Intermediate Filaments Are Dynamic

Although intermediate filaments are much more stable than microtubules and microfilaments, IF protein subunits have been shown to be in dynamic equilibrium with the existing IF cytoskeleton. In one experiment, biotin-labeled

![li4474542480e](li4474542480e.jpg)

FIGURE 18-48 Structure and assembly of intermediate filaments. Electron micrographs and drawings of IF protein dimers, tetramers, and mature intermediate filaments from Ascaris, an intestinal parasitic worm. (a) IF proteins form parallel dimers through a highly conserved coiled-coil core domain. The globular heads and tails are quite variable in length and sequence among IF classes. (b) A tetramer is formed by antiparallel, staggered, side-by-side aggregation of two identical dimers. (c) Tetramers aggregate end to end and laterally into a protofibril. In a mature filament, consisting of four protofibrils, the
globular domains form beaded clusters on the surface. See N. Geisler et al., 1998, J. Mol. Biol. 282:601; courtesy of Ueli Aebi. (d) Comparison of the structure of vimentin and lamin A. Notice that the lamin protein has a nuclear localization sequence to target it to the nucleus. See H. Hermann et al., 2007, Nat. Rev. Mol. Cell Biol. 8:562. [Micrographs reprinted with permission from Elsevier, from: N. Geisler et al., "Assembly and architecture of invertebrate cytoplasmic intermediate filaments reconcile features of vertebrate cytoplasmic and nuclear lamin-type intermediate filaments," J. Mol. Biol. 282:601 (1998).]

# Cytoplasmic Intermediate Filament Proteins Are Expressed in a Tissue-Specific Manner 

Sequence analysis of IF proteins reveals that they fall into five distinct homology classes, four of which are localized to the cytoplasm. These IF classes show a strong correspondence to the developmental origin of the cell type in which

(a) 20 minutes after injection
![li4484542480e](li4484542480e.jpg)
(b) 4 hours after injection
![li4494542480e](li4494542480e.jpg)
![li4504542480e](li4504542480e.jpg)

EXPERIMENTAL FIGURE 18-49
Keratin intermediate filaments are dynamic, as soluble keratin is incorporated into
filaments. Monomeric type I keratin was purified, chemically labeled with biotin, and microinjected into living epithelial cells. The cells were then fixed at different times after injection and stained with an antibody to biotin and with antibodies to keratin. (a) At 20 minutes after injection, the injected biotin-labeled keratin is concentrated in small foci scattered throughout the cytoplasm (left) and has not been integrated into the endogenous keratin cytoskeleton (right). (b) After 4 hours, the biotin-labeled keratin (left) and the keratin filaments (right) display identical patterns, indicating that the microinjected protein has become incorporated into the existing cytoskeleton. [©1991 R.K. Miller, K. Vistrom, and R .D. Goldman et al., The Journal of Cell Biology, 113:843-855. doi: 10.1083/jcb.113.4.843.]
the IF protein is expressed (Table 18-1). We discuss the fifth class-the nuclear lamins-separately, as they perform functions distinct from the cytoplasmic intermediate filaments.

The keratins that make up IF protein classes I and II are found in epithelia; class III IF proteins are generally found in cells of mesodermal origin; and class IV IF proteins compose the neurofilaments found in neurons. The lamins, which make up class V , are found lining the nuclei of all animal tissues. Here we briefly summarize the four homology classes found in the cytoplasm and discuss their roles in specific tissues.

Keratins Keratins provide strength to epithelial cells. The first two IF protein homology classes are the so-called acidic and basic keratins. There are about 50 genes in the human genome encoding keratins, about evenly split between the acidic and basic classes. These keratin subunits assemble into an obligate dimer, so that each dimer consists of one basic chain and one acidic chain; these dimers are then assembled into a filament as described in the previous section.

The keratins are by far the most diverse of the IF protein families, with basic and acidic keratin pairs showing different expression patterns between distinct epithelia as well as differentiation-specific regulation. Among these are the so-called hard keratins that make up hair and nails. These keratins are rich in cysteine residues that become oxidized to form disulfide bridges, thereby strengthening the proteins. This property is exploited by hair stylists: if you do not like the shape of your hair, the disulfide bonds in your hair keratin can be reduced, the hair reshaped, and the disulfide
bonds re-formed by oxidation-the result is "permanent" hair curling or hair straightening.

The so-called soft keratins, or cytokeratins, are found in epithelial cells. The epidermal-cell layers that make up the skin provide a good example of the function of these keratins. The lowest layer of cells, the basal layer, which is in contact with the basal lamina, proliferates constantly, giving rise to cells called keratinocytes. After they leave the basal layer, the keratinocytes differentiate and express abundant cytokeratins. The cytokeratins associate with specialized attachment sites between cells, thereby providing sheets of cells that can withstand abrasion. The keratinocytes eventually die, leaving dead cells from which all cell organelles have disappeared. This dead cell layer provides an essential barrier to water evaporation, without which we could not survive. The life of a skin cell, from birth to its loss from the animal as a skin flake, is about one month.

In all epithelia, keratin filaments associate with desmosomes, which link adjacent cells together, and hemidesmosomes, which link cells to the extracellular matrix, thereby giving cells and tissues their strength. These structures are described in more detail in Chapter 20.

In addition to simply providing structural support, there is increasing evidence that keratin filaments provide some organization to organelles and participate in signal transduction pathways. For example, in response to tissue injury, rapid cell growth is induced. It has been shown that in epithelial cells, the growth signal requires an interaction between a cell-growth-signaling molecule and a specific keratin.

TABLE 18-1 The Major Classes of Intermediate Filaments in Mammals

| Class | Protein | Distribution | Proposed Function |
| :--: | :--: | :--: | :--: |
| I | Acidic keratins | Epithelial cells | Tissue strength and integrity |
| II | Basic keratins | Epithelial cells |  |
| III | Desmin, GFAP, vimentin | Muscle, glial cells, mesenchymal cells | Sarcomere organization, integrity |
| IV | Neurofilaments (NFL, NFM, and NFH ) | Neurons | Axon organization |
| V | Lamins | Nucleus | Nuclear structure and organization |

![li4514542480e](li4514542480e.jpg)

Desmin The class III IF proteins include vimentin, found in mesenchymal cells; GFAP (glial fibrillary acidic protein), found in glial cells; and desmin, found in muscle cells. Desmin provides strength and organization to muscle cells (see cartoons in Table 18-1).

In smooth muscle, desmin filaments link cytoplasmic dense bodies, to which the contractile myofibrils are also attached, to the plasma membrane to ensure that cells resist overstretching. In skeletal muscle, a lattice composed of a band of desmin filaments surrounds the sarcomere. The desmin filaments encircle the Z disk and are cross-linked to the plasma membrane. Longitudinal desmin filaments cross to neighboring Z disks within the myofibril, and connections between desmin filaments around $Z$ disks in adjacent myofibrils serve to cross-link myofibrils into bundles within a muscle cell. The lattice is also attached to the sarcomere through interactions with myosin thick filaments. Because the desmin filaments lie outside the sarcomere, they do not actively participate in generating contractile forces. Rather, desmin plays an essential structural role in maintaining muscle integrity. In transgenic mice lacking desmin, for example, this supporting architecture is disrupted and Z disks are misaligned. The locations and morphology of mitochondria in these mice are also abnormal, suggesting that these intermediate filaments may also contribute to the organization of organelles.

Neurofilaments Type IV intermediate filaments consist of the three related subunits-NF-L, NF-M, and NF-H (for NF light, medium, and heavy)-that make up the neurofilaments found in the axons of neurons (see Figure 18-2). The three subunits differ mainly in the size of their C-terminal domains, and all form obligate heterodimers. Experiments with transgenic mice reveal that neurofilaments are necessary to establish the correct diameter of axons, which determines the rate at which nerve impulses are propagated down them.

The structural integrity of the skin is essential in order to withstand abrasion. In humans and mice, the K4 and K14 keratin isoforms form heterodimers that assemble into protofilaments. A mutant K14 with deletions in either the N - or the C -terminal domain can form heterodimers in vitro, but does not assemble into protofilaments. The expression of such mutant keratin proteins in cells causes IF networks to break down into aggregates. Transgenic mice that express a mutant K14 protein in the basal stem cells of the epidermis display gross skin abnormalities, primarily blistering of the epidermis, that resemble the human skin disease epidermolysis bullosa simplex (EBS). Histological examination of the blistered area reveals a high incidence of dead basal cells. The death of these cells appears to be caused by mechanical trauma from rubbing of the skin during

movement of the limbs. Without their normal bundles of keratin filaments, the mutant basal cells become fragile and easily damaged, causing the overlying epidermal layers to delaminate and blister (Figure 18-50). Like the role of desmin filaments in supporting muscle tissue, the general role of keratin filaments appears to be to maintain the structural integrity of epithelial tissues by mechanically reinforcing the connections between cells.

## Lamins Line the Inner Nuclear Envelope To Provide Organization and Rigidity to the Nucleus

The most widespread IF proteins are the class V proteins, the lamins. The lamins are the progenitors of all IF proteins, from which the cytoplasmic IF proteins arose by gene duplication and mutation. They are the major components of a
![li4524542480e](li4524542480e.jpg)

EXPERIMENTAL FIGURE 18-50 Transgenic mice carrying a mutant keratin gene exhibit blistering similar to that in the human disease epidermolysis bullosa simplex. Histological sections through the skin of a normal mouse and a transgenic mouse carrying a mutant K14 keratin gene are shown. In the normal mouse, the skin consists of a hard outer epidermal layer covering and in contact with the soft inner dermal layer. In the skin from the transgenic mouse, the two layers are separated (arrow) due to weakening of the cells at the base of the epidermis. [Republished with permission from Elsevier: from Coulombe et al., "Point mutations in human keratin 14 genes of epidermolysis bullosa simplex patients," Cell, 1991, 66:6, pp.1301-1311; permission conveyed through Copyright Clearance Center, Inc.]
two-dimensional meshwork called the nuclear lamina that lies between the nuclear envelope and the chromatin of the nucleus (Figure 18-51a). In humans, three genes encode lamins: one alternatively spliced gene encodes lamins A and C, and two other genes encode lamins B1 and B2. The B-type lamins appear to be the primordial lamin proteins and are expressed in essentially all cells, whereas lamins A and C are developmentally regulated. B lamins are post-translationally prenylated (see p. 288), which helps them associate with the inner nuclear envelope membrane. Lamin proteins contain the coiled-coil regions characteristic of intermediate filaments that is needed for dimerization, but also have a nuclear localization sequence that targets them to the nucleus as well as a conserved immunoglobulin-like fold (see Figure 18-48d).

Cells that are subject to mechanical stress have a nuclear lamin meshwork to maintain the integrity of the nucleus; cells with strong cell walls, such as plant and fungal cells, do not have such a meshwork. The lamin meshwork therefore provides both strength and support to the inner surface of the nuclear membrane. In fact, cells regulate the level of lamin A to match the stiffness of the tissue they are in. Thus neutrophils, for example, which need to move through thin capillaries and migrate through tight interstitial spaces, have a highly lobulated nucleus as a result of very low levels of lamin A. If they had large, rigid nuclei that were resistant to deformation, they might have trouble squeezing through the small spaces found in the extracellular matrix.

To provide rigidity, the lamin meshwork is associated with chromatin on one side and attached to the cytoskeleton on the other. Some proteins embedded in the inner nuclear membrane, such as the lamin-B receptor and emerin, can bind both chromatin-associated proteins and lamins (see Figure 18-51a). Interestingly, transcriptionally silent regions of the genome are preferentially associated with lamins, and recent evidence suggests that lamins play a role in genome organization and DNA repair. Attachment to the cytoskeleton through both the inner and outer nuclear membranes involves proteins with so-called SUN and KASH domains. The SUN domain proteins are synthesized on the endoplasmic reticulum as transmembrane proteins with their SUN domain in the lumen of the endoplasmic reticulum, and sorting signals in their cytoplasmic domain target them to the outer nuclear membrane, which is continuous with the endoplasmic reticulum. They are then transported as membrane proteins through nuclear pores and, upon reaching the inner nuclear membrane, associate with the nuclear lamina (Figure 18-51b). Nesprins are KASH domain-containing transmembrane proteins of the outer nuclear membrane oriented so that the KASH domain can associate with the SUN domain of another protein in the perinuclear space. The nesprins, in turn, associate, either directly or through adapters, with intermediate filaments, actin filaments, and microtubules, thereby providing physical linkage of the nucleus to the cytoskeleton (see Figure 18-51b). These attachments are used to move the nucleus to the correct location in a cell as well as transport it, for example, in the long processes of the vertebrate neuroepithelium.

![li4534542480e](li4534542480e.jpg)

FIGURE 18-51 The nuclear lamina is attached to chromatin and through LINC complexes to the cytoskeleton. (a) Diagram of part of a nucleus, showing the association of the lamin-containing nuclear lamina with chromatin and, through the two membranes of the nucleus, with the cytoskeleton. Proteins such as the membrane-associated lamin B receptor and emerin tether the lamin intermediate filaments to the inner nuclear membrane. Lamins are also tethered to the nuclear membrane by the prenylation of lamin B (not shown). Diverse linkages,

## Lamins Are Reversibly Disassembled by Phosphorylation During Mitosis

For the nuclear envelope to be broken down as cells go from prophase to prometaphase during mitosis, the nuclear lamina has to be disassembled. As we discuss in Chapter 19, protein kinases called mitotic CDKs drive cells into mitosis, and one of their substrates is the lamins. Phosphorylation of lamins A, B and C results in disassembly of the intermediate filament lattice into lamin dimers. Due to their C-terminal prenylation, lamin B dimers remain associated with the nuclear membrane. Depolymerization of the nuclear lamin filaments leads to the disintegration of the nuclear lamina meshwork and contributes to disassembly of the nuclear envelope. Later in mitosis (telophase), removal of the phosphates by specific phosphatases promotes lamin reassembly, which is critical to re-formation of a nuclear envelope around the daughter chromosomes. The opposing actions of kinases and phosphatases thus provide a rapid mechanism for controlling the assembly state of lamin intermediate filaments. Other intermediate filaments undergo similar disassembly and reassembly in the cell cycle.

$\square$There are over 200 known mutations located throughout the human gene for lamin A that are known to cause diseases, collectively called laminopathies. These diseases include cardiomyopathies, muscular dystrophies, lipodystrophy, and aging-related progeria. Some of these mutations cause Emery-Dreifuss muscular dystrophy (EDMD), most likely because the fragile nuclei cannot stand the stress and strains in
called LINC complexes, attach the lamins through the two nuclear membranes to the cytoskeleton. (b) A LINC complex consists of a SUN domain-containing protein that interacts with lamins and extends across the inner nuclear membrane, and a KASH domain-containing protein that interacts with a SUN domain-containing protein in the perinuclear space and crosses the outer nuclear membrane to interact with components of the cytoskeleton. See B. Burke and C. L. Stewart, 2013, Nat. Rev. Mol. Cell Biol. 14:13.
the muscle tissue, so these cells are the first to show symptoms. Other forms of EDMD have been traced to mutations in emerin, the lamin-binding membrane protein of the inner nuclear envelope, as well as a nesprin and a SUN protein. Yet other mutations in lamin A cause progeria-accelerated aging, such as Hutchison-Gilford progeria ("prematurely old"). It is still a mystery why different mutations in the same human gene can cause such a wide variety of diseases.

## KEY CONCEPTS OF SECTION 18.7

## Intermediate Filaments

- Intermediate filaments are the only nonpolar fibrous component of the cytoskeleton and are not associated with motor proteins. Intermediate filaments are built from coiledcoil dimers that associate in an antiparallel fashion into tetramers and then into protofilaments, 16 of which make up the filament (see Figure 18-48).
- There are five major classes of intermediate filament proteins, with the nuclear lamins (class V) being the most ancient and ubiquitous in animal cells. The other four classes show tissue-specific expression (see Table 18-1).
- Keratins (IF classes I and II) are found in animal hair and nails, as well as in cytokeratin filaments that associate with desmosomes in epithelial cells to provide the cells and tissue with strength.

- The class III filaments include vimentin, GFAP, and desmin, which provide structure and order to muscle $Z$ disks and restrain smooth muscle from overextension.
- The neurofilaments make up class IV and are important for the structure of axons.
- The lamins are major components of the nuclear lamina. They contribute to genome organization as well as to the rigidity of the nucleus through linkages to the cytoskeleton involving proteins with SUN and KASH domains.
- Many diseases are associated with defects in intermediate filaments, especially laminopathies, which include a variety of conditions, and mutations in keratin genes, which can cause severe defects in skin (see Figure 18-49).


### 18.8 Coordination and Cooperation Between Cytoskeletal Elements

So far, we have generally discussed the three cytoskeletal filament classes-microfilaments, microtubules, and intermediate filaments-as though they function independently of one another. However, the fact that the microtubulebased mitotic spindle determines the site of formation of the microfilament-based contractile ring is just one example of how these two cytoskeletal filament systems are coordinated. Here we mention some other examples of linkages, physical and regulatory, between cytoskeletal elements and their integration into other aspects of cellular organization.

## Intermediate Filament-Associated Proteins Contribute to Cellular Organization

A group of proteins collectively called intermediate filamentassociated proteins (IFAPs) that co-purify with intermediate filaments have been identified. Among these IFAPs are members of the plakin family, which are involved in attaching intermediate filaments to other structures. Some plakins associate with keratin filaments to link them to desmosomes, which are junctions between epithelial cells that provide stability to a tissue, and hemidesmosomes, which are located at regions of the plasma membrane where intermediate filaments are linked to the extracellular matrix (these topics are covered in detail in Chapter 20). Other plakins are found along intermediate filaments and have binding sites for microfilaments and microtubules. One of these proteins, called plectin, can be seen by immunoelectron microscopy to provide connections between microtubules and intermediate filaments (Figure 18-52).

## Microfilaments and Microtubules Cooperate to Transport Melanosomes

Studies of mutant mice with light-colored coats have uncovered a pathway in which microtubules and microfilaments
![li4544542480e](li4544542480e.jpg)

EXPERIMENTAL FIGURE 18-52 Gold-labeled antibody identifies plectin cross-links between intermediate filaments and microtubules. In this immunoelectron micrograph of a fibroblast cell, microtubules are highlighted in red, intermediate filaments in blue, and the short connecting fibers between them in green. Staining with gold-labeled antibodies to plectin (yellow) reveals that these connecting fibers contain plectin. [C] 1996 T. M. Svitkina, A.B. Verkhhovsky, and G.G. Borisy et al., The Journal of Cell Biology, 135:991-1007. doi: 10.1083/jcb.135.4.991.]
cooperate to transport melanosomes. The pigment in mammalian hair is produced in cells called melanocytes, cells that are very similar to the fish and frog melanophores discussed earlier (see Figure 18-29). Melanocytes are found in the hair follicle at the base of the hair shaft and contain pigment-laden granules called melanosomes. Melanosomes are transported to the dendritic extensions of melanocytes for subsequent exocytosis to the surrounding epithelial cells. Transport to the cell periphery is mediated, just as in frog melanophores, by a kinesin family member. At the periphery, they are then handed off to myosin V and delivered for exocytosis. If the myosin V system is defective, the melanosomes are not captured and stay in the melanocyte cell body. Thus microtubules are responsible for the long-range transport of melanosomes, whereas microfilament-based myosin V is responsible for their capture and delivery at the cell cortex. This type of division of labor-long-range transport by microtubules and short-range transport by microfilamentshas been found in many different systems, from transport in filamentous fungi to transport along axons.

## Cdc42 Coordinates Microtubules and Microfilaments During Cell Migration

In Chapter 17, we discussed how the polarity of a migrating cell is regulated by Cdc42, which results in the formation of an actin-based leading edge at the front of the cell and contraction at the back (see Figure 17-45 and Figure 18-53, step 1). It turns out that Cdc42 activation at the cell front also leads to polarization of the microtubule cytoskeleton. This phenomenon was originally studied in wound-healing assays (see Figure 17-44), in which it was noticed that when

![li4554542480e](li4554542480e.jpg)

FIGURE 18-53 Independent Cdc42 regulation of microfilaments and microtubules to polarize a migrating cell. Active Cdc42-GTP at the front of the cell leads to Rac and WASP activation, which results in the assembly of a microfilament-based leading edge (step (). Independently, Cdc42-GTP also leads to the capture of microtubule (+) ends and the activation of dynein (step (). Dynein pulls on microtubules to orient the centrosome (step ( ) toward the front of the cell. This reorientation reorganizes the secretory pathway for the delivery of adhesion molecules carried in secretory vesicles along microtubules to the front of the cell (step (). See S. Etienne-Manneville et al., 2005, J. Cell Biol. 170:895-901.
the cells at the edge of a scratch are induced to polarize and move to fill in the empty space, the Golgi complex is moved to the front of the nucleus toward the cell front. Golgi localization at the front of the cell indicates that the centrosome moves to lie in front of the nucleus (recall that Golgi localization is dependent on the location of the MTOC; see Figures 18-1c, 18-28). Recent studies have suggested how this happens. Active Cdc42 at the front of the cell binds the polarity factor Par6, which results in the recruitment of the dynein-dynactin complex (Figure 18-53, step ). Cortically localized dynein-dynactin then interacts with microtubules, pulling on them to orient the centrosome and hence the whole radial array of microtubules (Figure 18-53, step ). This reorientation of the microtubule system leads to the reorganization of the secretory pathway to deliver secretory products, especially integrins to bind the extracellular matrix, to the front of the cell for attachment to the substratum for cell migration (Figure 18-53, step ■).

## Advancement of Neural Growth Cones Is Coordinated by Microfilaments and Microtubules

The nervous system depends on the integration and transmission of signals by neurons. Neurons have specialized structures, called dendrites, that receive signals and a single axon that terminates in one or more synapses on a target
cell or cells (for example, another neuron or a muscle cell) (see Figure 18-2). It is critical that neurons make the right connections, so how are growing axons guided to their correct destinations? As an axon extends, its terminal growth cone senses signals from the extracellular matrix and from other cells that guide it along the right path. Therefore, how the growth cone receives and interprets cues that direct axon growth is critical to the function of the nervous system. Growth cones are very rich in actin, and they typically have a broad lamellipodium and multiple filopodia. Also essential for the guidance of growth cones are microtubules. Recall that axons have microtubules of uniform polarity along which materials for growth of the axon move by axonal transport (see Figure 18-5e). These microtubules extend into the growth cone and, together with actin, are involved in guiding its direction of advancement. While actin is necessary for the advancement of the growth cone, the microtubules and actin together are necessary to steer growth in the correct direction. Although the mechanisms involved have not been fully elucidated, it has been found that a local growth signal alters local actin dynamics, with the result that microtubules extend into the region of the signal. It has also been found that microtubules in the shaft of the axon have post-translational modifications, such as acetylation, that stabilize them, whereas the more dynamic microtubules in the growth cone often do not (Figure 18-54).
![li4564542480e](li4564542480e.jpg)

EXPERIMENTAL FIGURE 18-54 Localization of actin (red), microtubules (green), and acetylated microtubules (blue) in a small growth cone. Notice how the stable acetylated microtubules are localized to the shaft of the axon and do not penetrate into the dynamic growth cone. [Republished with permission of Elsevier: from Bent, E.W. and Gertler, F.B., "Cytoskeletal Dynamics and Transport in Growth Cone Motility and Axon Guidance," Neuron, 2003, 40:209-227]

## KEY CONCEPTS OF SECTION 18.8

## Coordination and Cooperation Between Cytoskeletal Elements

- Intermediate filaments are linked both to specific attachment sites on the plasma membrane (called desmosomes and hemidesmosomes) and to microfilaments and microtubules (see Figure 18-52).
- In animal cells, microtubules are generally used for the long-range delivery of organelles, whereas microfilaments handle their local delivery.
- The signaling molecule Cdc42 coordinately regulates microfilaments and microtubules during cell migration.
- The advancement of growth cones in neurons requires the interplay of microfilaments and microtubules.


## LaunchPad

macmillan learning

Visit LaunchPad to access study tools and to learn more about the content in this chapter.

- Perspectives for the Future
- Analyze the Data
- Extended References
- Additional study tools, including videos, animations, and quizzes


## Key Terms

anaphase 851
anterograde 833
asters 850
axonal transport 833
axoneme 844
basal body 844
centromere 853
centrosome 825
cilia 844
cytokinesis 851
desmin 864
dynamic instability 828
dyneins 833
flagella 844
$\gamma$-tubulin ring complex
$(\gamma-\mathrm{TuRC}) \quad 825$
intermediate filament 861
intermediate filamentassociated proteins (IFAPs) 867
intraflagellar transport (IFT) 845
keratins 863
kinesins 833
kinetochores 851
lamins 863
metaphase 851
microtubule 821
microtubule-associated proteins (MAPs) 822
microtubule-organizing center (MTOC) 825
mitosis 849
mitotic spindle 850
neurofilaments 863
primary cilium 847
prophase 850
protofilament 861
retrograde 833
telophase 851
tubulin 822

## Review the Concepts

1. Microtubules are polar filaments; that is, one end is different from the other. What is the basis for this polarity, how is polarity related to microtubule organization within the cell, and how is polarity related to the intracellular movements powered by microtubule-dependent motors?
2. Microtubules both in vitro and in vivo undergo dynamic instability, and this type of assembly is thought to be intrinsic to the microtubule. What is the current model that accounts for dynamic instability?
3. In cells, microtubule assembly depends on other proteins as well as tubulin concentration and temperature. What types of proteins influence microtubule assembly in vivo, and how does each type affect assembly?
4. Microtubules within a cell appear to be arranged in specific arrays. What cellular structure is responsible for determining the arrangement of microtubules within a cell? How many of these structures are found in a typical cell? Describe how such structures serve to nucleate microtubule assembly.
5. Many drugs that inhibit mitosis bind specifically to tubulin, microtubules, or both. What diseases are such drugs used to treat? Functionally speaking, these drugs can be divided into two groups based on their effect on microtubule assembly. What are the two mechanisms by which such drugs alter microtubule structure?
6. Kinesin-1 was the first member of the kinesin superfamily to be identified and therefore is perhaps the best-characterized superfamily member. What fundamental property of kinesin was used to purify it?
7. Certain cellular components appear to move bidirectionally on microtubules. Describe how this is possible given that microtubule orientation is fixed by the MTOC.
8. The movement of kinesin motor proteins involves both the motor domain and the linker domain. Describe the role of each domain in kinesin movement, direction of movement, or both. Could kinesin-1 with one inactive head efficiently move a vesicle along a microtubule?
9. What features of the dynactin complex enable cytoplasmic dynein to transport cargo toward the microtubule ( - ) end? What effect could inhibition of dynactin interaction with the +TIP EB1 have on spindle orientation?
10. Cell swimming depends on appendages containing microtubules. What is the underlying structure of these appendages, and how do these structures generate the force required to produce swimming?
11. What effect would dynein inactivation have on kinesin-2-dependent IFT?

12. The mitotic spindle is often described as a microtubulebased cellular machine. The microtubules that constitute the mitotic spindle can be classified into three distinct types. What are the three types of spindle microtubules, and what is the function of each?
13. Mitotic spindle function relies heavily on microtubule motors. For each of the following motor proteins, predict the effect on spindle formation, function, or both of adding a drug that specifically inhibits only that motor: kinesin-5, kinesin-13, and kinesin-4.
14. The poleward movement of kinetochores, and hence chromatids, during anaphase A requires that kinetochores maintain a hold on the shortening microtubules. How does a kinetochore hold onto shortening microtubules?
15. Anaphase B involves the separation of spindle poles. What forces have been proposed to drive this separation? What underlying molecular mechanisms are thought to provide these forces?
16. Cytokinesis, the process of cytoplasmic division, occurs shortly after the separated sister chromatids have neared the opposite spindle poles. How is the plane of cytokinesis determined? What are the respective roles of microtubules and actin filaments in cytokinesis?
17. The best strategy for treating a specific type of human tumor can depend on identifying the type of cell that became cancerous to give rise to the tumor. For some tumors that have colonized a distant location (metastasized), identifying the parental cell type can be difficult. Because the type of IF protein expressed is cell-type-specific, using monoclonal antibodies that react with only one type of IF protein can help in this identification. What IF proteins would you produce monoclonal antibodies against to identify (a) a sarcoma of muscle cell origin, (b) an epithelial cell carcinoma, and (c) an astrocytoma (glial cell tumor)?
18. Explain why there are no known motors that use intermediate filaments as tracks.
19. Growth cones are highly mobile regions of developing neurons. What prevents the growth cone from moving or collapsing back into the main cell body, as often occurs with lamellipodia?

## References

## Microtubule Structure and Organization

Badano, J. L., T. M. Teslovich, and N. Katsanis. 2005. The centrosome in human genetic disease. Nat. Rev. Mol. Cell Biol. 6:194-205.

Dutcher, S. K. 2001. The tubulin fraternity: alpha to eta. Curr. Opin. Cell Biol. 13:49-54.

Nogales, E., and H.-W. Wang. 2006. Structural intermediates in microtubule assembly and disassembly: how and why? Curr. Opin. Cell Biol. 18:179-184.

## Microtubule Dynamics

Brouhard, G. Y., and L. M. Rice. 2014. The contribution of $\alpha \beta$-tubulin curvature to microtubule dynamics. J. Cell Biol. 207:323-334.

Desai, A., and T. J. Mitchison. 1997. Microtubule polymerization dynamics. Annu. Rev. Cell Dev. Bi. 13:83-117.

## Regulation of Microtubule Structure and Dynamics

Galjart, N. 2010. Plus-end-tracking proteins and their interactions at microtubule ends. Curr. Biol. 20:R528-R537.

Hammond, J. W., D. Cai, and K. J. Verhey. 2008. Tubulin modifications and their cellular functions. Curr. Opin. Cell Biol. 20:71-76.

Wloga, D., and J. Gaertig. 2010. Post-translational modifications of microtubules. J. Cell Sci. 123:3447-3455.

Zheng, Y., and P. A. Iglesias. 2013. Nucleating new branches from old. Cell 152:669-670.

## Kinesins and Dyneins: Microtubule-Based Motor Proteins

Web site: Kinesin Home Page, http://www.cellbio.duke.edu/ kinesin/

Allan, V. 2014. One, two, three, cytoplasmic dynein is go! Nature 345:271-272.

Bhabha, G., et al. 2014. Allosteric communication in the dynein motor domain. Cell 159:857-868.

Carter, A. P., et al. 2011. The crystal structure of dynein. Science 331:1159-1165.

Cho, C., and R. D. Vale. 2012. The mechanism of dynein motility: insight from crystal structures of the motor domain. Biochim. Biophys. Acta 1823:182-191.

Endow, S. A., F. J Kull, and H. Liu. 2010. Kinesins at a glance. J. Cell Sci. 123:3420-3424.

Hirokawa, N., et al. 2009. Kinesin superfamily motor proteins and intracellular transport. Nat. Rev. Mol. Cell Biol. 10:682-696.

McKenney, R. J., et al. 2010. LIS1 and NudE induce a persistent dynein force-producing state. Cell 141:304-314.

McKenney, R. J., et al. 2014. Activation of cytoplasmic dynein motility by dynactin-cargo adapter complexes. Nature 345:337-341.

Roberts, A. J., et al. 2013. Functions and mechanics of dynein motor proteins. Nat. Rev. Mol. Cell Biol. 14:713-726.

Trokter, M., and T. Surrey. 2012. Lis1 clamps dynein to the microtubule. Cell 150:877-879.

Vale, R. D. 2003. The molecular motor toolbox for intracellular transport. Cell 112:467-480.

Vale, R. D., and R. A. Milligan. 2000. The way things move: looking under the hood of molecular motor proteins. Science 288:88-95.

Verhey, K. J., and J. W. Hammond. 2009. Traffic control: regulation of kinesin motors. Nat. Rev. Mol. Cell Biol. 10:765-777.

Cilia and Flagella: Microtubule-Based Surface Structures
Gerdes, J. M., E. E. Davis, and N. Katsanis. 2009. The vertebrate primary cilium in development, homeostasis, and disease. Cell 137:32-45.

Jin, H., et al. 2010. The conserved Bardet-Biedl syndrome proteins assemble a coat that traffics membrane proteins to cilia. Cell 141:1208-1218.

## Mitosis

Foley, E. A., and T. M. Kapoor. 2013. Microtubule attachment and spindle assembly checkpoint signalling at the kinetochore. Nat. Rev. Cell Mol. Biol. 14:25-37.

Goshima, G., et al. 2009. Augmin: a protein complex required for centrosome-independent microtubule generation within the spindle. J. Cell Biol. 181:421-429.

Lampert, F., and S. Westermann. 2011. A blueprint for kinetochores-new insights into the molecular mechanics of cell division. Nat. Rev. Mol. Cell Biol. 12:407-412.

London, N., and S. Biggins. 2014. Signalling dynamics in the spindle checkpoint response. Nat. Rev. Mol. Cell Biol. 15:735-747.

Mitchison, T. J., and E. D. Salmon. 2001. Mitosis: a history of division. Nat. Cell Biol. 3:E17-E21.

Santaguida, S., and A. Musacchio. 2009. The life and miracle of kinetochores. EMBO J. 28:2511-2531.

Wang, H., I. Brust-Mascher, and J. M. Scholey. 2014. Sliding filaments and mitotic spindle organization. Nat. Cell Biol. 16:737-739.

## Intermediate Filaments

Burke, B., and C. L. Stewart. 2013. The nuclear lamins: flexibility in function. Nat. Rev. Mol. Cell Biol. 14:13-23.

Isermann, P., and J. Lammerding. 2013. Nuclear mechanics and mechanotransduction in health and disease. Curr. Biol. 23:R1113-R1121.

Luxton, G. W. G, and D. A. Starr. 2014. KASHing up with the nucleus: novel functional roles of KASH proteins at the cytoplasmic surface of the nucleus. Curr. Opin. Cell Biol. 28:69-75.

Worman, H. J. 2012. Nuclear lamins and laminopathies. J. Pathol. 226:316-325.

## Coordination and Cooperation Between Cytoskeletal Elements

Etienne-Manneville, S., et al. 2005. Cdc42 and Par6-PKCz regulate the spatially localized association of Dlg1 and APC to control cell polarization. J. Cell Biol. 170:895-901.

Kodama, A., T. Lechler, and E. Fuchs. 2004. Coordinating cytoskeletal tracks to polarize cellular movements. J. Cell Biol. 167:203-207.

Schaefer, A. W., et al. 2008. Coordination of actin filament and microtubule dynamics during neurite outgrowth. Dev. Cell 15:146-162.

this page left intentionally blank

![li4574542480e](li4574542480e.jpg)

Micrograph of a human epithelial cell undergoing anaphase. Following DNA replication, cells undergo mitosis to segregate their replicated chromosomes. The cell in the center is in the process of segregating its chromosomes (blue) to opposite ends using the mitotic spindle apparatus (green). This process occurs during anaphase. Thereafter, the cytoplasm of the cell is divided to produce two identical daughter cells. [Dr. Torsten Wittmann/Science Source.]

Proper control of cell division is vital to all organisms. In unicellular organisms, cell division must be balanced with cell growth so that cell size is properly maintained. If several divisions occur before parent cells have reached the proper size, daughter cells eventually become too small to be viable. If cells grow too large before cell division, the cells function improperly and the number of cells increases slowly. In developing multicellular organisms, the replication of each cell must be precisely controlled and timed to faithfully and reproducibly complete the developmental program in every individual. Each type of cell in every tissue must control its replication precisely for normal development of complex organs such as the brain or the kidney. In a normal adult, cells divide only when and where they are needed. However, loss of normal controls on cell replication is the fundamental defect in cancer, an all-too-familiar disease that kills one in every six people in the developed world (see Chapter 24).

## The Eukaryotic Cell Cycle

The molecular mechanisms regulating eukaryotic cell division discussed in this chapter have gone a long way in explaining how replication control goes awry in cancer cells. Appropriately, Leland Hartwell, Tim Hunt, and Paul Nurse were awarded the Nobel Prize in Physiology or Medicine in 2001 for the initial experiments that elucidated the master regulators of cell division in all eukaryotes.

The term cell cycle refers to the ordered series of events that lead to cell division and the production of two daughter cells, each containing chromosomes identical to those of the parent cell. Two main molecular processes take place during the cell cycle, with resting intervals in between: during the S phase of the cycle, each parent chromosome is duplicated to form two identical sister chromatids; and in mitosis (M phase), the resulting sister chromatids are distributed to each daughter cell (Figure 19-1; see also Figure 1-16). Chromosome replication and segregation to daughter cells must

### 19.1 Overview of the Cell Cycle and Its Control

### 19.2 Model Organisms and Methods of Studying the Cell Cycle

19.3 Regulation of CDK Activity
19.4 Commitment to the Cell Cycle and DNA Replication

### 19.5 Entry into Mitosis

19.6 Completion of Mitosis: Chromosome Segregation and Exit from Mitosis

### 19.7 Surveillance Mechanisms in Cell Cycle Regulation

19.8 Meiosis: A Special Type of Cell Division

![li4584542480e](li4584542480e.jpg)

FIGURE 19-1 The fate of a single parent chromosome throughout the eukaryotic cell cycle. Following mitosis (M), daughter cells contain $2 n$ chromosomes in diploid organisms and $1 n$ chromosomes in haploid organisms. In proliferating cells, $G_{1}$ is the period between the "birth" of a cell following mitosis and the initiation of DNA synthesis, which marks the beginning of the S phase. At the end of the S phase, cells enter $\mathrm{G}_{2}$ containing twice the number of chromosomes they had as $G_{1}$ cells ( $4 n$ in diploid organisms, $2 n$ in haploid organisms). The end of $\mathrm{G}_{2}$ is marked by the onset of mitosis, during which numerous events leading to cell division occur. The $G_{1}, S$, and $G_{2}$ phases are collectively referred to as interphase, the period between one mitosis and the next. Most nonproliferating cells in vertebrates leave the cell cycle in $\mathrm{G}_{1}$, entering the $G_{0}$ state. Although chromosomes condense only during mitosis, here they are shown in condensed form throughout the cell cycle to emphasize the number of chromosomes at each stage. For simplicity, the nuclear envelope is not depicted.
occur in the proper order in every cell division. If a cell undergoes chromosome segregation before the replication of all chromosomes has been completed, at least one daughter cell will lose genetic information. Likewise, if a second round of replication occurs in one region of a chromosome before cell division occurs, the genes encoded in that region are increased in number out of proportion to other genes, a phenomenon that often leads to an imbalance of gene expression that is incompatible with viability.

High accuracy and fidelity are required to ensure that DNA replication is carried out correctly and that each daughter cell inherits the correct number of each chromosome. To achieve this, cell division is controlled by surveillance mechanisms known as checkpoint pathways, which prevent initiation of each step in cell division until
the earlier steps on which it depends have been completed and any mistakes that occurred during the process have been corrected. Mutations that inactivate or alter the normal operation of these checkpoint pathways contribute to the generation of cancer cells because they result in chromosomal rearrangements and abnormal numbers of chromosomes, which lead to further mutations and changes in gene expression that cause uncontrolled cell growth (see Chapter 24).

In the late 1980s, it became clear that the molecular processes regulating the two key events in the cell cyclechromosome replication and chromosome segregationare fundamentally similar in all eukaryotic cells. Initially, it was surprising to many researchers that cells as diverse as budding yeast and developing human neurons use nearly identical proteins to regulate their division. However, like transcription and protein synthesis, control of cell division appears to be a fundamental cellular process that evolved and was largely optimized early in eukaryotic evolution. Because of this similarity, research with diverse organisms, each with its own particular experimental advantages, has contributed to a growing understanding of how cell cycle events are coordinated and controlled. Biochemical, genetic, imaging, and micromanipulation techniques have all been employed in studying various aspects of the eukaryotic cell cycle. These studies have revealed that cell division is controlled primarily by regulation of the timing of entry into the cell division cycle, DNA replication, and mitosis.

The master controllers of the cell cycle are a small number of protein kinases, each of which contains a regulatory subunit (cyclin) and a catalytic subunit (cyclin-dependent kinase, or CDK). These heterodimeric kinases regulate the activities of multiple proteins involved in entry into the cell cycle, DNA replication, and mitosis by phosphorylating those proteins at specific regulatory sites, activating some and inhibiting others to coordinate their activities. Regulated degradation of proteins also plays a prominent role in important cell cycle transitions. Since protein degradation is irreversible, it ensures that the processes move in only one direction through the cell cycle.

In this chapter, we first present an overview of the cell cycle and then describe the various experimental systems that have contributed to our current understanding of it. We then discuss cyclin-dependent kinases (CDKs) and the many different ways in which these key cell cycle controllers are regulated. Next we examine each cell cycle phase in greater detail, with an emphasis on how control of CDK activity governs the events that take place in each phase. We then discuss the checkpoint pathways that establish the order of the cell cycle and ensure that each cell cycle phase occurs with accuracy. The chapter concludes with a discussion of meiosis, a special type of cell division that generates haploid germ cells (eggs and sperm), and the molecular mechanisms that distinguish it from mitosis. In our discussion, we emphasize the general principles governing cell cycle progression

and use a species-spanning nomenclature when discussing the factors controlling each cell cycle phase.

### 19.1 Overview of the Cell Cycle and Its Control

The cell cycle in eukaryotes is a highly conserved, well-ordered series of events. During each cycle, DNA replication leads to the creation of two identical DNA molecules, which are compacted and structured for their segregation into daughter cells. In this section, we begin our discussion by reviewing the stages of the eukaryotic cell cycle, then introduce the master regulators, the cyclin-dependent kinases, and conclude with an overview of the principles that govern the cell cycle.

## The Cell Cycle Is an Ordered Series of Events Leading to Cell Replication

The cell cycle is divided into four major phases (see Figure 19-1). Cycling (replicating) mammalian somatic cells grow in size and synthesize the RNAs and proteins required for DNA synthesis during the $G_{1}$ (first gap) phase. When cells have reached the appropriate size and have synthesized the required proteins, they enter the cell cycle by traversing a point in $\mathrm{G}_{1}$ known as $S T A R T$ in yeast and the restriction point in mammals. Once this point has been crossed, cells
are committed to cell division. The first step toward successful cell division is entry into the $\mathbf{S}$ (synthesis) phase, the period in which cells actively replicate their chromosomes. After progressing through a second gap phase, the $\mathrm{G}_{2}$ phase, cells begin the complicated process of mitosis, also called the M (mitotic) phase, which is divided into several stages (Figure 19-2).

In discussing mitosis, we commonly use the term chromosome for the replicated structures that condense and become visible in the light microscope during the early stages of mitosis. Thus each chromosome is composed of two identical DNA molecules resulting from DNA replication, plus histones and other chromosome-associated proteins (see Figure 8-35). The two identical DNA molecules and associated chromosomal proteins that form one chromosome are called sister chromatids. Sister chromatids are attached to each other by protein cross-links.

During interphase, the part of the cell cycle between the end of one M phase and the beginning of the next, the outer nuclear membrane is continuous with the endoplasmic reticulum. With the onset of mitosis in prophase, the nuclear envelope retracts into the endoplasmic reticulum in most cells from higher eukaryotes, and the membranes of the Golgi complex break down into vesicles. This is necessary so that the microtubules, nucleated by the centrosomes, can interact with the chromosomes to form the mitotic spindle, consisting of a football-shaped bundle of microtubules with a star-shaped cluster of microtubules radiating from each end,

FIGURE 19-2 The stages of mitosis.
During prophase, the nuclear envelope breaks down, microtubules form the mitotic spindle apparatus, and chromosomes condense. At metaphase, attachment of chromosomes to microtubules via their kinetochores is complete. During anaphase, motor proteins and the shortening of spindle microtubules pull the sister chromatids toward opposite spindle poles. After chromosome movement to the spindle poles, chromosomes decondense. Cells reassemble nuclear membranes around the daughter-cell nuclei and undergo cytokinesis.
![li4594542480e](li4594542480e.jpg)

or spindle pole. A multiprotein complex, the kinetochore, assembles at each centromere. After nuclear envelope breakdown, at metaphase, the kinetochores of sister chromatids associate with microtubules coming from opposite spindle poles (see Figure 18-37), and the chromosomes align in a plane in the center of the cell. During anaphase, sister chromatids separate. They are initially pulled by microtubules toward the spindle poles and are then further separated as the spindle poles move away from each other (see Figure 19-2).

Once chromosome separation is complete, the mitotic spindle disassembles and chromosomes decondense during telophase. The nuclear envelope re-forms around the segregated chromosomes as they decondense. The physical division of the cytoplasm, called cytokinesis, yields two daughter cells. Following mitosis, cycling cells enter the $G_{1}$ phase, embarking on another turn of the cycle.

The progression of cell cycle stages is the same for all eukaryotes, though the time it takes to complete one turn of the cycle varies considerably among organisms. Rapidly replicating human cells progress through the full cell cycle in about 24 hours: $\mathrm{G}_{1}$ takes 9 hours; the S phase, 10 hours; $\mathrm{G}_{2}$, 4.5 hours; and mitosis, 30 minutes. In contrast, the full cycle takes only 90 minutes in rapidly growing yeast cells. The cell divisions that take place during early embryonic development of the fruit fly Drosophila melanogaster are completed in as little as 8 minutes!

In multicellular organisms, most differentiated cells exit the cell cycle and survive for days, weeks, or in some cases (e.g., nerve cells and cells of the eye lens) even the lifetime of the organism without dividing again. Such postmitotic cells generally exit the cell cycle in $G_{1}$, entering a phase called $G_{0}$ (see Figure 19-1). Some $\mathrm{G}_{0}$ cells can return to the cell cycle and resume replicating; this re-entry is regulated, thereby providing control of cell proliferation.

## Cyclin-Dependent Kinases Control the Eukaryotic Cell Cycle

As mentioned in the chapter introduction, passage through the cell cycle is controlled by heterodimeric protein kinases that comprise a catalytic subunit and a regulatory subunit. The catalytic subunits, the cyclin-dependent kinases (CDKs), have no kinase activity unless they are associated with a regulatory cyclin subunit. Each CDK can associate with a small number of different cyclins, which determine the substrate specificity of the complex-that is, which proteins it phosphorylates. Each cyclin is only present and active during the cell cycle stage it promotes and hence restricts the kinase activity of the CDKs to which it binds to that cell cycle stage. Cyclin-CDK complexes activate or inhibit hundreds of proteins involved in cell cycle progression by phosphorylating them at specific regulatory sites. Thus proper progression through the cell cycle is governed by activation of the appropriate cyclin-CDK complex at the appropriate time. As we will see, restricting cyclin expression to the appropriate cell cycle stage is one of the many mechanisms cells employ to regulate the activities of each cyclin-CDK heterodimer.

## Several Key Principles Govern the Cell Cycle

The goal of each cell division is to generate two daughter cells of identical genetic makeup. To achieve this, cell cycle events must occur in the proper order. DNA replication must always precede chromosome segregation. Today we know that the activity of the key proteins that promote cell cycle progression, the CDKs, fluctuates during the cell cycle. For example, CDKs that promote $S$ phase are active during $S$ phase but are inactive during mitosis. CDKs that promote mitosis are active only during mitosis. These oscillations in CDK activity are a fundamental aspect of eukaryotic cell cycle control, and we have gained some understanding as to how these oscillations are generated. Oscillations are generated by positive feedback mechanisms, whereby specific CDKs promote their own activation. These positive feedback loops are coupled to subsequent negative feedback mechanisms by which, indirectly or with a built-in delay, CDKs promote their own inactivation. Their oscillations not only propel the cell cycle forward but also create abrupt transitions between different cell cycle states, which is essential to bring about distinct cell cycle states.

Overlaid on the cell cycle oscillator machinery is a system of surveillance mechanisms that further ensures that the next cell cycle event is not activated before the preceding one has been completed or before errors that occurred during the preceding step have been corrected. These surveillance mechanisms are called checkpoint pathways, and their job is to ensure the accuracy of the chromosome replication and segregation processes. The system that ensures that chromosomes are segregated accurately is so efficient that a mis-segregation event occurs only once in $10^{4}-10^{5}$ divisions! These multiple layers of control on the cell cycle control machinery ensure that the cell cycle is robust and error free.

## KEY CONCEPTS OF SECTION 19.1

## Overview of the Cell Cycle and Its Control

- The eukaryotic cell cycle is divided into four phases: $\mathrm{G}_{1}$ (the period between mitosis and the initiation of nuclear DNA replication), S (the period of nuclear DNA replication), $\mathrm{G}_{2}$ (the period between the completion of nuclear DNA replication and mitosis), and M (mitosis).
- Cells commit to a new cell division at a specific point in $\mathrm{G}_{1}$ known as START or the restriction point.
- Cyclin-CDK complexes, composed of a regulatory cyclin subunit and a catalytic cyclin-dependent kinase (CDK) subunit, drive the progression of a cell through the cell cycle.
- Cyclins activate CDKs and are present only in the cell cycle stage that they promote.
- CDK activities oscillate during the cell cycle. Positive and negative feedback loops drive these oscillations.
- Surveillance mechanisms, called checkpoint pathways, guarantee that each cell cycle step is completed correctly before the next one is initiated.

### 19.2 Model Organisms and Methods of Studying the Cell Cycle

The unraveling of the molecular mechanisms governing cell cycle progression in eukaryotes was remarkably rapid and was fueled by a powerful combination of genetic and biochemical approaches. In this section, we discuss several model systems and their contributions to the discovery of the molecular mechanisms of cell division. The three most important systems employed to study the cell cycle are the single-celled yeasts Saccharomyces cerevisiae (budding yeast) and Schizosaccharomyces pombe (fission yeast) and the oocytes and early embryos of the frog Xenopus laevis. We also discuss the fruit fly Drosophila melanogaster, which proved extremely powerful in the study of the interplay between cell division and development. The study of mammalian tissue culture cells led to the characterization of cell cycle control in mammals.

Studies of the cell cycle in many different experimental systems also led to two remarkable discoveries about the general control of the cell cycle. First, complex molecular processes such as initiation of DNA replication and entry into mitosis are all regulated and coordinated by a small number of master cell cycle regulatory proteins. Second, these master regulators and the proteins that control them are highly conserved, so that cell cycle studies in fungi, sea urchins, insects, frogs, and other species are directly applicable to all eukaryotic cells, including human cells.
(a)
![li4604542480e](li4604542480e.jpg)
(b)

FIGURE 19-3 The budding yeast S. cerevisiae. (a) Scanning electron micrograph of $S$. cerevisiae cells at various stages of the cell cycle. The larger the bud, which emerges at the end of the $G_{1}$ phase, the farther along in the cycle the cell is. (b) Main events in the S. cerevisiae cell cycle. Daughter cells are born smaller than parent cells and must grow to a greater extent in $G_{1}$ before they are large enough to enter the $S$ phase.

## Budding and Fission Yeasts Are Powerful Systems for Genetic Analysis of the Cell Cycle

Budding and fission yeasts have proved to be valuable systems for the study of the cell cycle. Although they both belong to the kingdom Fungi, they are only distantly related. Both organisms can exist in the haploid state, carrying only one copy of each chromosome. The fact that these yeasts can exist as haploid cells makes them powerful genetic systems. It is easy to generate mutations that inactivate genes in haploids because there is only one copy of each gene (a diploid would require an inactivating mutation in each of the two copies of the gene to render its activity nonfunctional). Haploid yeast can be easily employed to screen or select for mutants with specific defects, such as defects in cell proliferation. Additional advantages of these two systems are the relative ease with which one can manipulate the expression of individual genes, and the ease with which yeasts can be cultivated and manipulated so that cultures of cells progress through the cell cycle in a synchronous manner.

Budding yeast cells are ovoid in shape and divide by budding (Figure 19-3a). The bud, which is the future daughter cell, begins to form concomitant with the initiation of DNA replication and continues to grow throughout the cell cycle (Figure 19-3b). Cell cycle stage can therefore be inferred from the size of the bud, which makes S. cerevisiae a useful system for identifying mutants that are blocked at specific steps in the cell cycle. Indeed, it was in this organism that
![li4614542480e](li4614542480e.jpg)

START is the point in the cell cycle after which cells are irreversibly committed to undergoing a cell cycle. $G_{2}$ is not well defined in budding yeast and is therefore denoted in parentheses. Note that the nuclear envelope does not disassemble during mitosis in S. cerevisiae and other yeasts. The small S. cerevisiae chromosomes do not condense sufficiently to be visible by light microscopy. [Part (a) SOMAT/Science Source.]

![li4624542480e](li4624542480e.jpg)

FIGURE 19-4 The fission yeast S. pombe. (a) Scanning electron micrograph of S. pombe cells at various stages of the cell cycle. Long cells are about to enter mitosis; short cells have just passed through cytokinesis. (b) Main events in the S. pombe cell cycle. START is the
point in the cell cycle after which cells are irreversibly committed to cell division. As in S. cerevisiae, the nuclear envelope does not break down during mitosis. [Part (a) Steve Gschmeissner/Science Source.]
cycle) mutants. Identification of the genes mutated in these temperature-sensitive yeast strains provided a comprehensive list of genes critical for virtually all aspects of cell division.

## Frog Oocytes and Early Embryos Facilitate Biochemical Characterization of the Cell Cycle Machinery

Biochemical studies require the preparation of cell extracts from many cells. For biochemical studies of the cell cycle, the eggs and early embryos of amphibians and marine invertebrates are particularly suitable. These organisms typically have large eggs, and fertilization is followed by multiple synchronous cell cycles. By isolating large numbers of eggs from females and fertilizing them simultaneously by addition of sperm (or by treating them in ways that mimic fertilization), researchers can obtain extracts from cells at specific points in the cell cycle for analysis of proteins and enzymatic activities.

To understand how X. laevis oocytes and eggs can be used for the analysis of cell cycle progression, we must first lay out the events of oocyte maturation, which can be recapitulated in vitro. So far, we have discussed mitotic division. Oocytes, however, undergo a meiotic division (see Figure 19-35 for an overview of meiosis). As oocytes develop in the frog ovary, they replicate their DNA and become arrested in $\mathrm{G}_{2}$ for 8 months, during which time they grow in size to a diameter of 1 mm , stockpiling all the materials needed for the multiple

![li4634542480e](li4634542480e.jpg)

FIGURE 19-5 Progesterone stimulates maturation of Xenopus oocytes. (a) Step 1 Progesterone treatment of $\mathrm{G}_{2}$-arrested Xenopus oocytes surgically removed from the ovary of an adult female causes the oocytes to enter meiosis I. Two pairs of synapsed homologous chromosomes (blue) connected to meiotic spindle microtubules (green) are shown schematically to represent cells in metaphase of meiosis I. Step Segregation of homologous chromosomes and a highly asymmetric cell division expels half the chromosomes into a small cell called the first polar body. The oocyte immediately commences meiosis II and arrests in metaphase II to yield an egg. Two chromosomes connected to spindle microtubules are shown schematically to represent egg cells arrested in metaphase of meiosis II. Step Fertilization by sperm releases eggs from their metaphase arrest, allowing them to proceed through anaphase of meiosis II and undergo a second highly asymmetric cell division that expels one chromatid of each chromosome into a second polar body. The resulting haploid female pronucleus fuses with the haploid sperm pronucleus to produce a diploid zygote. Step 1 The zygote undergoes DNA replication and the first mitosis. Step 1 The first mitosis is followed by 11 more synchronous divisions to form a blastula. (b) Micrograph of Xenopus eggs. [Part (b) © MICHEL DELARUE/ISM/Phototake.]
cell divisions of the early embryo. When stimulated by a male, an adult female's ovarian cells secrete the steroid hormone progesterone, which induces the $\mathrm{G}_{2}$-arrested oocytes to mature and enter meiosis. As we will see in Section 19.8, meiosis consists of two consecutive chromosome segregation phases known as meiosis I and meiosis II. Progesterone triggers oocytes to undergo meiosis I and progress to the second meiotic metaphase, where they arrest and await fertilization (Figure 19-5). At this stage the cells are called eggs. When fertilized by sperm, the egg nucleus is released from its metaphase II arrest and completes meiosis. The resulting haploid egg nucleus then fuses with the haploid sperm nucleus, producing a diploid zygote nucleus. DNA replication follows, and the first mitotic division of embryogenesis begins. The resulting embryonic cells then proceed through 11 more rapid, synchronous cell cycles, generating a hollow sphere of cells called the blastula. Cell division then slows, and subsequent divisions are non-synchronous, with cells at different positions in the blastula dividing at different times.

The advantage of using $X$. laevis to study factors involved in mitosis is that large numbers of oocytes and eggs can be prepared that are all proceeding synchronously through the cell cycle events that follow progesterone treatment and
fertilization. This makes it possible to prepare sufficient amounts of extract for biochemical experiments from cells that were all at the same point in the cell cycle. It was in this system that the cyclin-CDK complexes that trigger mitosis and the oscillatory nature of their activity were first discovered. This activity was called maturation-promoting factor (MPF) because of its ability to induce entry into meiosis and oocyte maturation when injected into $\mathrm{G}_{2}$-arrested oocytes.

## Fruit Flies Reveal the Interplay Between Development and the Cell Cycle

The development of complex tissues often requires specific modifications to the cell cycle. Understanding the interplay between development and cell division is thus crucial if we want to understand how complex organisms are built. Drosophila melanogaster has established itself as the premier model system for studying the interplay between development and the cell cycle. Not only does the development of this organism involve several highly unusual cell cycles, but the powerful genetic techniques that can be applied to fruit flies facilitated the discovery of genes involved in the developmental control of the cell cycle.

![li4644542480e](li4644542480e.jpg)

FIGURE 19-6 Cell division patterns during the life cycle of Drosophila melanogaster. After fertilization, nuclei in the embryo undergo 13 rapid S phase-M phase cycles. These cycles are followed by three divisions that include a $G_{2}$ phase. All these nuclear divisions occur within a common cytoplasm and are therefore called the syncytial divisions. During late stages of embryogenesis and throughout larval development (with the exception of the nervous system), cells undergo endocycles. These cycles lead to an increase in cellular ploidy and size
and hence larval growth. In the pupa, during a process called metamorphosis, imaginal disks, the tissues that give rise to the adult organs, undergo mitotic divisions and then differentiate to form adult structures. Several types of divisions are seen in the adult fly. Stem cells undergo mitotic divisions, meiosis gives rise to sperm and eggs, and endocycles create polyploid cells in the ovary. See L. A. Lee and T. L. Orr-Weaver, 2003, Ann. Rev. Genet. 37:545-578.

The first 13 nuclear divisions of the fertilized Drosophila embryo are rapid cycles of DNA replication and mitosis (with no gap phases), fueled by key cell cycle regulators that were stockpiled in the egg cytoplasm as it matured. These divisions, which all occur in a common cytoplasm, are called the syncytial divisions and occur in unison (Figure 19-6). As maternal stockpiles run out, gap phases are introduced, first $\mathrm{G}_{2}$, followed by $\mathrm{G}_{1}$. Most cells in the embryo cease to divide at this point, form plasma membranes, and undergo a specialized cell cycle known as the endocycle. In the endocycle, cells replicate their DNA but do not undergo mitosis. This replication leads to an increase in gene dosage and fuels increased macromolecule biosynthesis, which allows individual cells to grow in size. Thus the embryo, which has now developed into a crawling larva, grows simply by an increase in cell size and not through cell multiplication. A select number of cells do not share this fate. These cells are in the imaginal disks, the organs that will give rise to the adult fly tissues during metamorphosis. Metamorphosis occurs during the pupa stage and transforms larvae into adult flies. The divisions that give rise to the adult fly are canonical cell cycles leading to a diploid organism.

## The Study of Tissue Culture Cells Uncovers Cell Cycle Regulation in Mammals

Cell cycle regulation in human cells is more complex than in other non-mammalian systems. To understand this increased level of complexity, and to understand the cell cycle alterations that are the cause of cancer, it is important to study the cell cycle not only in model organisms, but also in human cells. Researchers use normal or tumor cells grown in
plastic dishes to study the properties of the human cell cycle, a method called tissue culture or cell culture. It is, however, important to note that many of the cell types used to study the human cell cycle have altered cell cycle properties due to genetic alterations that occurred during their culturing or because they were isolated from human tumors. Furthermore, in vitro culture conditions do not resemble those found in the organism and could lead to altered behavior of cells. Although some aspects of mammalian cell division are not recapitulated in cell culture conditions-such as tissue organization and developmental signals governing cell cycle control—cell culture systems nevertheless provide critical insights into the mammalian cell's intrinsic mechanisms governing cell division. Researchers are also working toward establishing culture systems that more closely resemble the cell architecture in tissues. For example, polymer lattices are currently being developed that allow scientists to grow cells in three-dimensional culture mimicking cellular organization within a tissue.

Primary human cells and other mammalian cells have a finite life span when cultured in vitro. Normal human cells, for example, divide $25-50$ times, but thereafter proliferation slows and eventually stops. This process is called replicative senescence. Some cells can escape this process and become immortalized, allowing researchers to establish cell lines. Although these cell lines harbor genetic alterations that affect some aspects of their proliferation, they are nevertheless a useful tool for studying cell cycle progression in human cells. These cell lines provide an inexhaustible supply of cells that, as we will see next, can be manipulated to progress through the cell cycle in a synchronous manner, allowing for the analysis of protein levels and enzymatic activity at different stages of the cell cycle.

FIGURE 19-7 Human cells undergoing mitosis. HeLa Kyoto cells were filmed as they underwent mitosis. The images shown were taken every 20 minutes. Cells are flat during interphase, but as cells undergo mitosis, they round up and divide. Subsequently, they flatten out again. [Courtesy of Sejal Vyas and Paul Chang, MIT.]
![li4654542480e](li4654542480e.jpg)

## Researchers Use Multiple Tools to Study the Cell Cycle

The experimental analysis of cell cycle properties requires that we be able to determine the cell cycle stage of individual cells. Light microscopy provides some estimate of cell cycle progression. For example, light microscopy allows a researcher to determine whether cultured mammalian cells are in interphase $\left(G_{1}, S\right.$ phase, and $\left.G_{2}\right)$ or in mitosis. Mammalian tissue culture cells are flat and adhere to their growth surface during interphase, but round up and form spherical structures as they undergo mitosis (Figure 19-7). Fluorescence microscopy of cellular structures or analysis of specific cell cycle markers-that is, proteins that are present only in certain cell cycle stages-allows for a more accurate determination of cell cycle stage.

In addition to microscopic tools, cell cycle researchers use flow cytometry to determine the DNA content of a cell population (Figure 19-8; see also Figure 4-2). Cells are treated with a DNA-binding fluorescent dye, and the amount of dye that is incorporated into the DNA of cells can then be quantitatively assessed using a flow cytometer. Cells are then recorded by their DNA content, and the percentages of cells in $G_{1}, S$ phase, and $G_{2}$ or mitosis can be assessed in this manner. Cells in $G_{1}$ have half as much DNA as cells in $G_{2}$ or mitosis. Cells undergoing DNA synthesis in $S$ phase have an intermediate amount of DNA.

To characterize different cell cycle events, it is essential to examine cell populations that progress through the cell cycle in unison. Researchers can generate such populations by reversibly arresting cells in a particular cell cycle stage. This cell cycle arrest is usually accomplished by restricting nutrients or by adding anti-growth factors, which cause cells to arrest in $G_{1}$. In budding yeast, for example, cells treated with a mating pheromone arrest in $G_{1}$ (see Figures 16-23 and 16-24). When the pheromone is removed from the cells (usually by washing them extensively), the cells exit $G_{1}$ and progress through the cell cycle in a synchronous manner. In mammalian cells, removal of growth factors by removing serum from the culture medium (serum starvation) arrests cells in $G_{0}$. Re-addition of serum allows cells to re-enter the cell cycle. Other methods involve blocking a certain cell
cycle step with chemicals. Hydroxyurea inhibits DNA replication, leading to arrest in $S$ phase. On removal of the drug, cells will resume DNA synthesis in unison. Nocodazole disrupts the mitotic spindle and halts the cell cycle in mitosis. Once the drug is washed away, cells will resume progression through mitosis in a synchronous manner. In budding and fission yeasts, the conditional cdc mutants introduced earlier have proved to be a powerful tool for creating synchronous cultures. Temperature-sensitive cdc mutants, when incubated at the non-permissive temperature, arrest in a particular cell cycle stage because they are defective in a certain key cell cycle protein. Returning cells to the permissive temperature allows them to continue with the cell division cycle in a synchronous fashion.
![li4664542480e](li4664542480e.jpg)

EXPERIMENTAL FIGURE 19-8 Analysis of DNA content by flow cytometry. Haploid yeast cells were grown in culture and stained with propidium iodide, a fluorescent dye that is incorporated into DNA. The $x$ axis shows DNA content, the $y$ axis the number of cells. The DNA content analysis shows two predominant populations of cells: cells with unreplicated DNA (1C) and with replicated DNA (2C). The cells between the two peaks represent cells that are in the process of undergoing DNA replication.

## KEY CONCEPTS OF SECTION 19.2

## Model Organisms and Methods of Studying the Cell Cycle

- The ability to isolate mutants and the powerful genetic tools of budding and fission yeasts allowed for the isolation of key genetic factors important for cell cycle regulation.
- Frog eggs and early embryos from synchronously fertilized eggs, which provide sources of extracts for biochemical studies of cell cycle events, allowed the identification of the oscillatory nature of cyclin-CDK complexes.
- Fruit flies are a powerful system for investigating the interplay between cell division and the developmental programs responsible for building multicellular organisms.
- Human tissue culture cells are used to study the properties of the mammalian cell cycle.
- The generation of synchronized cell populations by reversibly arresting cells in a particular cell cycle stage allows researchers to examine the behavior of proteins and cellular processes during the cell cycle.


### 19.3 Regulation of CDK Activity

In the following sections, we describe the current model of eukaryotic cell cycle regulation, which is summarized in Figure 19-9. A key discovery in cell cycle studies was that cyclin-dependent kinases govern progression through the cell cycle. Three key features of these kinases are important to keep in mind throughout this chapter:

- Cyclin-dependent kinases (CDKs) are active only when bound to a regulatory cyclin subunit.
- Different types of cyclin-CDK complexes initiate different events. $G_{1}$ CDKs and $G_{1} / S$ phase CDKs promote entry into the cell cycle, $S$ phase CDKs trigger $S$ phase, and mitotic CDKs initiate the events of mitosis (Figure 19-10).
- Multiple mechanisms are in place to ensure that the different CDKs are active only in the stages of the cell cycle they trigger.

In this section, we first discuss the properties of CDKs and investigate the structural basis of their activation and regulation. We then describe how cyclins activate CDKs and investigate the multiple regulatory mechanisms that restrict the different cyclins to the appropriate cell cycle stage. We will see that protein degradation plays an essential part in this process. In addition, we will see how post-translational modifications to CDKs and inhibitory proteins that directly

FIGURE 19-9 Regulation of cell cycle transitions. Cell cycle transitions are regulated by cyclin-CDK protein kinases, protein phosphatases, and ubiquitin-protein ligases. Here the cell cycle is diagrammed, with the major stages of mitosis shown at the top. In early $G_{1}$, no cyclin-CDKs are active. In mid-G, $G_{1} / S$ phase CDKs activate transcription of genes required for DNA replication. $S$ phase is initiated by the SCF ubiquitin-protein ligase, which ubiquitinylates inhibitors of $S$ phase CDKs, marking them for degradation by proteasomes. The $S$ phase CDKs then activate DNA replication, and DNA synthesis commences. Once DNA replication is complete, cells enter $G_{2}$. In late $G_{2}$, mitotic CDKs trigger entry into mitosis. During prophase, the nuclear envelope breaks down and chromosomes align on the mitotic spindle, but they cannot separate until the anaphase-promoting complex (APC/C), a ubiquitin-protein ligase, ubiquitinylates the anaphase inhibitor protein securin, marking it for degradation by proteasomes. This results in degradation of protein complexes linking the sister chromatids and the onset of anaphase as the sister chromatids separate. APC/C also ubiquitinylates mitotic cyclins, causing their degradation by proteasomes. The resulting drop in mitotic CDK activity, along with the action of protein phosphatases, results in chromosome decondensation, reassembly of nuclear membranes around the daughter-cell nuclei, and cytokinesis.
![li4674542480e](li4674542480e.jpg)

![li4684542480e](li4684542480e.jpg)
bind to cyclin-CDK complexes serve as essential additional control mechanisms in restricting different CDK activities to the appropriate cell cycle stage.

## Cyclin-Dependent Kinases Are Small Protein Kinases That Require a Regulatory Cyclin Subunit for Their Activity

Cyclin-dependent kinases are a family of small (30-40 kD) serine/threonine kinases. They are not active in the monomeric form, but require an activating subunit to be active as protein kinases. In budding and fission yeasts, a single CDK controls progression through the cell cycle. Its activity is specified by cell-cycle-stage-specific cyclin subunits. Mammalian cells contain as many as nine CDKs, with four of them, CDK1, CDK2, CDK4, and CDK6, having clearly been shown to regulate cell cycle progression. They bind to different types of cyclins and, together with those cyclins, promote different cell cycle transitions. CDK4 and CDK6 are $G_{1}$ CDKs that promote entry into the cell cycle, CDK2 functions as a $\mathrm{G}_{1} / \mathrm{S}$ phase and S phase CDK, and CDK1 is the mitotic CDK. For historical reasons, the names of various cyclin-dependent kinases from yeasts and vertebrates differ. Whenever possible, we will use the general terms $G_{1}$, $G_{1} / S$ phase, $S$ phase, and mitotic $C D K s$ to describe CDKs instead of the species-specific terminology. Table 19-1 lists

FIGURE 19-10 An overview of how CDKs regulate cell cycle progression. Cells harbor different types of CDKs that initiate different events of the cell cycle. Importantly, the CDKs are active only in the stages of the cell cycle they trigger. $G_{1} / S$ phase CDKs are active at the $G_{1}-S$ phase transition to trigger entry into the cell cycle. S phase CDKs are active during $S$ phase and trigger $S$ phase. Mitotic CDKs are active during mitosis and trigger mitosis. The anaphase-promoting complex or cyclosome (APC/C) ubiquitin-protein ligase catalyzes two key cell cycle transitions by ubiquitinylating proteins, hence targeting them for degradation. APC/C initiates anaphase and exit from mitosis.
the different names of the various CDKs and indicates when in the cell cycle they are active.

CDKs are regulated not only by cyclin binding, but also by both activating and inhibitory phosphorylation. Together, these regulatory events ensure that CDKs are active only at the appropriate cell cycle stage. The three-dimensional structure of CDKs provides insight into how the activity of these protein kinases is regulated. Nonphosphorylated, inactive CDK contains a flexible region, called the T loop or activation loop, that is highly conserved among protein kinases and is discussed extensively in Chapter16. This loop blocks access of protein substrates to the active site where ATP is bound (Figure 19-11a), explaining why free CDK, unbound to cyclin, has little protein kinase activity. Nonphosphorylated CDK bound to one of its cyclin partners has minimal but detectable protein kinase activity in vitro, although it may be essentially inactive in vivo. Extensive interactions between the cyclin and the T loop cause a dramatic shift in the position of the T loop, thereby exposing the CDK active site (Figure 19-11b). As we will see shortly, high activity of the cyclin-CDK complex requires phosphorylation of the activating threonine in the T loop, causing additional conformational changes in the cyclin-CDK complex that greatly increase its affinity for protein substrates (Figure 19-11c). As a result, the kinase activity of the phosphorylated complex is a hundredfold greater than that of the nonphosphorylated complex.

TABLE 19-1 Cyclins and CDKs: Nomenclature and Their Roles in the Mammalian Cell Cycle

| CDK | Cyclin | Function | General Name |
| :-- | :-- | :-- | :-- |
| CDK1 | Cyclin A, cyclin B | Mitosis | Mitotic CDKs |
| CDK2 | Cyclin E, cyclin A | Entry into the cell cycle <br> S phase | $\mathrm{G}_{1} / \mathrm{S}$ phase CDKs <br> S phase CDKs |
| CDK4 | Cyclin D | $\mathrm{G}_{1}$ <br> Entry into the cell cycle | $\mathrm{G}_{1}$ CDKs |
| CDK6 | Cyclin D | $\mathrm{G}_{1}$ <br> Entry into the cell cycle | $\mathrm{G}_{1}$ CDKs |

![li4694542480e](li4694542480e.jpg)

FIGURE 19-11 Structural models of human CDK2. (a) Free, inactive CDK2 not bound to its cyclin subunit, cyclin A. In free CDK2, the T loop blocks access of protein substrates to the $y$ phosphate of the bound ATP, shown as a ball-and-stick model. The conformations of the T-loop and the region highlighted in yellow ( $\alpha 1$ Helix) are altered when CDK is bound to cyclin A. (b) Nonphosphorylated, low-activity cyclin A-CDK2 complex. Conformational changes induced by binding of a domain of cyclin A (blue) cause the T loop to pull away from the active site of CDK2 so that substrate proteins can bind. The $\alpha 1$ helix in CDK2, which interacts

## Cyclins Determine the Activity of CDKs

Cyclins are so named because their concentrations change during the cell cycle. They form a family of proteins that is defined by three key features:

- Cyclins bind to and activate CDKs. The activity and substrate specificity of any given CDK is primarily defined by the particular cyclin to which it is bound.
- Cyclins are present only during the cell cycle stage that they trigger and are absent in other cell cycle stages.
- Cyclins not only regulate a particular cell cycle stage, but also set in motion a series of events in preparation for the next cell cycle stage. In this way, they propel the cell cycle forward.

Cyclins are divided into four classes defined by their presence and activity during specific phases of the cell cycle: $\mathrm{G}_{1}$ cyclins, $\mathrm{G}_{2} / \mathrm{S}$ phase cyclins, S phase cyclins, and mitotic cyclins (see Table 19-1). The different types of cyclins are quite distinct from one another in protein sequence, but all of them contain a conserved 100-amino-acid region known as the cyclin box and possess similar three-dimensional structures.

The $\mathrm{G}_{1}$ cyclins are the linchpin in coordinating the cell cycle with extracellular events. Their activity is subject to regulation by signal transduction pathways that sense the presence of growth factors or cell proliferation inhibitory signals. In metazoans, $\mathrm{G}_{1}$ cyclins are known as cyclin Ds, and they bind to CDK4 and CDK6. $\mathrm{G}_{1}$ cyclins are unusual
extensively with cyclin A, moves several angstroms into the catalytic cleft, repositioning key catalytic side chains required for the phosphotransfer reaction. The black ball marks the position of the threonine (Thr-160) whose phosphorylation activates CDKs. (c) Phosphorylated, high-activity cyclin A-CDK2 complex. The conformational changes induced by phosphorylation of the activating threonine (red ball) alter the shape of the substrate-binding surface, greatly increasing the affinity for protein substrates. See P. D. Jeffrey et al., 1995, Nature 1995, 376:313-20. [Data from A. A. Russo et al., 1996, Nature Struc. Biol. 3:696, PDB ID 1jst.]
in that their levels do not show strong fluctuation, as levels of other cyclins do. Instead, in response to macromolecule biosynthesis and extracellular signals, their levels gradually increase throughout the cell cycle.

The $\mathrm{G}_{1} / \mathrm{S}$ cyclins accumulate during late $\mathrm{G}_{1}$, reach peak levels when cells enter $S$ phase, and decline during $S$ phase (see Figure 19-10). They are known as cyclin E in metazoans and bind to CDK2. The main function of cyclin E-CDK2 complexes, together with cyclin D-CDK4/6, is to trigger the $\mathrm{G}_{1}-\mathrm{S}$ phase transition. This transition, known as START in yeast and the restriction point in mammalian cells, is defined as the point at which cells are irreversibly committed to cell division and can no longer return to the $G_{1}$ state. In molecular terms, this means that cells initiate DNA replication as well as duplicating their centrosomes, which is the first step in the formation of the mitotic spindle that will be used during mitosis.

S phase cyclins are synthesized concomitantly with $\mathrm{G}_{1}$ cyclins, but their levels remain high throughout S phase and do not decline until early mitosis. Two types of $S$ phase cyclins trigger $S$ phase in metazoans: cyclin E, which can also promote entry into the cell cycle and is therefore also a $\mathrm{G}_{1} / \mathrm{S}$ cyclin, and cyclin A. Both cyclins bind CDK2 (see Table 19-1) and are directly responsible for DNA synthesis. As we will see in Section 19.4, these protein kinases phosphorylate proteins that activate DNA helicases and load polymerases onto DNA.

Mitotic cyclins bind CDK1 to promote entry into and progression through mitosis. The metazoan mitotic cyclins are cyclins A and cyclins B (note that cyclin A can also

trigger S phase when bound to CDK2). Mitotic cyclin-CDK complexes are synthesized during S phase and $\mathrm{G}_{2}$, but as we will see shortly, their activities are held in check until DNA synthesis is completed. In Section 19.5, we will see that once activated, mitotic CDKs promote entry into mitosis by phosphorylating and activating hundreds of proteins to promote chromosome condensation, nuclear envelope breakdown, mitotic spindle formation, and other aspects of mitosis. Their inactivation during anaphase prompts cells to exit mitosis, which involves the disassembly of the mitotic spindle, chromosome decondensation, the re-formation of the nuclear envelope, and eventually cytokinesis.

Mitotic cyclins were the first cyclins to be discovered, and it was their characterization that led to the discovery of the oscillatory nature of the activities that govern cell cycle progression (see Classic Experiments 19-1 and 19-2). Subsequent studies showed that $G_{1} / S$ phase cyclins had similar properties. Their expression is sufficient to promote entry into the cell cycle, and therefore all the other proteins needed for cell cycle entry are present in unlimited amounts. It is thus clear that the regulation of cyclin levels is an essential aspect of the eukaryotic cell cycle. As we will see in the following section, cells use multiple mechanisms to restrict cyclins to the appropriate cell cycle stage and to keep them at the right concentration.

## Cyclin Levels Are Primarily Regulated by Protein Degradation

Multiple mechanisms ensure that CDKs are active at the right stage of the cell cycle. Table 19-2 lists the key regulators of CDKs. The timely activation of CDKs depends, in part, on the presence of the appropriate cyclins in the cell cycle stage at which they are needed. In this section, we discuss how the regulation of cyclin levels is brought about. Transcriptional control of the cyclin subunits is one mechanism that ensures proper temporal expression of the cyclins. In somatic cells and yeast, waves of transcription factor activity help establish waves of cyclin activity. A general principle here is that an earlier wave of transcriptional activity helps produce the factors essential to generate a subsequent transcriptional wave. As we will see in Section 19.4, transcription of the $G_{1} / S$ phase cyclins is promoted by the E2F transcription factor complex. Among the many other genes whose transcription E2F promotes are those encoding the transcription factors that promote the synthesis of mitotic cyclins.

The most important regulatory control that restricts cyclins to the appropriate cell cycle stage is ubiquitinmediated, proteasome-dependent protein degradation. Because protein degradation is an irreversible process, in the

TABLE 19-2 Regulators of Cyclin-CDK Activity

| Type of Regulator | Function |
| :--: | :--: |
| Kinases and Phosphatases |  |
| CAK kinase | Activates CDKs |
| Wee1 kinase | Inhibits CDKs |
| Cdc25 phosphatase | Activates CDKs |
| Cdc14 phosphatase | Activates Cdh1 to degrade mitotic cyclins |
| Cdc25A phosphatase | Activates vertebrate S phase CDKs |
| Cdc25C phosphatase | Activates vertebrate mitotic CDKs |
| Inhibitory Proteins |  |
| Sic1 | Binds and inhibits S phase CDKs |
| CKIs p27 ${ }^{\text {KIP1 }}$, p $57^{\text {KIP2 }}$, and p21 ${ }^{\text {CIP }}$ | Bind and inhibit CDKs |
| INK4 | Binds and inhibits $\mathrm{G}_{1}$ CDKs |
| Rb | Binds E2Fs, preventing transcription of multiple cell cycle genes |
| Ubiquitin-Protein Ligases |  |
| SCF | Degradation of phosphorylated Sic1 or p27 ${ }^{\text {KIP1 }}$ to activate S phase CDKs |
| APC/C ${ }^{\text {Cdc20 }}$ | Degradation of securin, initiating anaphase. Induces degradation of B-type cyclins |
| APC/C ${ }^{\text {Cdh1 }}$ | Degradation of B-type cyclins in $\mathrm{G}_{1}$ and geminin in metazoans to allow loading of replicative helicases on DNA replication origins |

sense that the protein can be replenished only through de novo protein synthesis, this regulatory mechanism is ideal to ensure that the cell cycle engine is driven forward and that cells cannot "go backward" in the cell cycle. In other words, once a particular cyclin is degraded, the processes that it activated can no longer take place.

Recall that during ubiquitin-mediated protein degradation, ubiquitin-protein ligases ubiquitinylate substrate proteins, marking them for degradation by proteasomes (see Figure 3-31). Cyclins are degraded through the action of two different ubiquitin-protein ligases, SCF (named after the first letters of its constituents, $S k p 1$, Cullin, and $F$-box proteins) and the anaphase-promoting complex or cyclosome (abbreviated as APC/C in this chapter). SCF controls the $\mathrm{G}_{1}-\mathrm{S}$ phase transition by degrading $\mathrm{G}_{1} / \mathrm{S}$ phase cyclins and, as we will see in detail shortly, CDK inhibitory proteins. APC/C degrades $S$ phase and mitotic cyclins, thereby promoting the exit from mitosis. APC/C also controls the onset of chromosome segregation at the metaphase-anaphase transition by degrading an anaphase inhibitory protein (as discussed in Section 19.6).

SCF and APC/C are multisubunit ubiquitin-protein ligases that belong to the RING finger family of ubiquitinprotein ligases. Despite the fact that SCF and APC/C belong to the same ubiquitin-protein ligase family, their regulation is quite different. SCF recognizes its substrates only when they are phosphorylated. It is continuously active throughout the cell cycle, and cell-cycle-regulated phosphorylation of its substrates ensures that they are degraded only at certain stages of the cell cycle. In the case of APC/C-dependent protein degradation, the regulation is reversed: substrates are recognizable throughout the cell cycle, but the activity of APC/C is regulated. APC/C is activated by phosphorylation at the metaphase-anaphase transition through the action of mitotic CDKs and other protein kinases. APC/C is then active throughout the rest of mitosis and during $G_{1}$ to promote the degradation of cyclins and other mitotic regulators (see Figure 19-10). The substrate specificity of active, phosphorylated APC/C is determined in part by its association with one of two related substrate targeting factors called Cdc20 and Cdh1. During anaphase, APC/C bound to Cdc20 ubiquitinylates proteins that inhibit chromosome segregation, while during telophase and $\mathrm{G}_{1}, \mathrm{APC} / \mathrm{C}$ bound to Cdh1 targets different substrates for degradation.

The substrates of APC/C contain recognition motifs. The first to be identified was the destruction box. It is found in most $S$ phase and mitotic cyclins and is both necessary and sufficient to target proteins for degradation. The importance of cyclin degradation was again first demonstrated for mitotic cyclins. Deletion of the destruction box in mitotic cyclins prevented cells from exiting mitosis, demonstrating that mitotic exit requires the degradation of mitotic cyclin (see Classic Experiment 19-2). Later studies showed that inhibiting the degradation of other cyclins also severely affected cell cycle progression, indicating that ubiquitinmediated degradation of cyclins is an essential aspect of the eukaryotic cell cycle.

## CDKs Are Regulated by Activating and Inhibitory Phosphorylation

Regulation of the levels of cyclins is not the only mechanism that controls CDK activity. Activating and inhibitory phosphorylation events on the CDK subunit itself are essential to control cyclin-CDK activity. Phosphorylation of a threonine residue near the active site of the enzyme is required for CDK activity. This phosphorylation is mediated by the CDK-activating kinase (CAK). In some organisms, cyclin binding is a prerequisite for CAK phosphorylation, whereas in others this phosphorylation event occurs prior to cyclin binding. Although the sequence of assembling active CDKs differs among organisms, it is clear that CAK phosphorylation of CDK is not a rate-limiting step in CDK activation. CAK activity is constant throughout the cell cycle and phosphorylates the CDK as soon as a cyclin-CDK complex is formed.

Two inhibitory phosphorylations on CDK also play a critical role in controlling CDK activity. In contrast to the CAK-induced activating phosphorylation, these inhibitory phosphorylations are regulated. A highly conserved tyrosine (Y15 in human CDKs) and an adjacent threonine (T14 in humans) are subject to regulated phosphorylation. Both residues are situated in the ATP-binding pocket of the CDK, and their phosphorylation most likely interferes with positioning of ATP in the pocket. Changes in the phosphorylation of these sites are essential for the regulation of mitotic CDKs and have also been implicated in the control of $G_{1} / S$ and $S$ phase CDKs. As we will see in Section 19.5, a highly conserved kinase called Wee1 brings about this inhibitory phosphorylation, and a highly conserved phosphatase called Cdc25 mediates dephosphorylation.

## CDK Inhibitors Control Cyclin-CDK Activity

So far, we have discussed the importance of regulating cyclin levels and of CDK phosphorylation in controlling CDK activity. The final layer of control that is critical in the regulation of CDKs is a family of proteins known as CDK inhibitors, or CKIs, which bind directly to the cyclin-CDK complex and inhibit its activity. As we will see in Section 19.4, these proteins play an especially important role in the regulation of the $\mathrm{G}_{1}-\mathrm{S}$ phase transition and its integration with extracellular signals.

All eukaryotes harbor CKIs involved in regulating S phase and mitotic CDKs. Although these inhibitors display little sequence similarity, they are all essential to prevent premature activation of $S$ phase and $M$ phase CDKs. Inhibitors of $\mathrm{G}_{1}$ CDKs play an essential role in mediating a $\mathrm{G}_{1}$ arrest in response to proliferation inhibitory signals. A class of CKIs called INK4s (inhibitors of kinase 4) includes several small, closely related proteins that interact only with the $\mathrm{G}_{1}$ CDKs. Binding of INK4s to CDK4 and CDK6 blocks their interaction with cyclin D and hence their protein kinase activity. A second class of CKIs found in metazoan cells consists of three proteins- $\mathrm{p} 21^{\mathrm{CIP}}, \mathrm{p} 27^{\mathrm{KIP} 1}$, and $\mathrm{p} 37^{\mathrm{KIP} 2}$.

These CKIs inhibit $\mathrm{G}_{1} / \mathrm{S}$ phase CDKs and S phase CDKs and must be degraded before DNA replication can begin. As we will discuss in Section 19.7, p21 ${ }^{\text {CIP }}$ plays an important role in the response of metazoan cells to DNA damage. CKIs regulating $\mathrm{G}_{1}$ CDKs play a critical role in preventing tumor formation. For example, both copies of the INK4 gene that encodes p16 are found inactivated in a large fraction of human cancers (see Chapter 24).

## Genetically Engineered CDKs Led to the Discovery of CDK Functions

Different CDKs initiate different cell cycle phases by phosphorylating specific proteins. It is now clear that rather than phosphorylating a small number of proteins that in turn initiate a certain cell cycle stage, CDKs phosphorylate a myriad of substrates, thereby directly initiating all aspects of a given cell cycle phase. Analysis of a small number of substrates has provided examples that show how phosphorylation by mitotic CDKs mediates many of the early events of mitosis: chromosome condensation, formation of the mitotic spindle, and disassembly of the nuclear envelope. We discuss these events in detail in the sections that follow.

In recent years, systematic efforts to identify all CDK substrates have been initiated. The challenge in identifying the substrates of a particular kinase is to distinguish that kinase's phosphorylation events from those carried out by other kinases. A breakthrough in understanding which proteins are targets of CDKs was facilitated by the engineering of a CDK mutant that can use an analog of ATP that is not bound by other kinases. This ATP analog has a bulky benzyl group attached to $\mathrm{N}_{6}$ of the adenine, which makes the analog too large to fit into the ATP-binding pocket of wild-type protein kinases. However, the ATP-binding pocket of the mutant CDK was modified to accommodate this $\mathrm{N}_{6}$-benzyl ATP analog. Consequently, only the mutant CDK can use this ATP analog as a substrate for transferring its $\gamma$ phosphate to a protein side chain. When the $\mathrm{N}_{6}$-benzyl ATP analog with a labeled $\gamma$ phosphate was incubated with cell extracts containing a recombinant mitotic CDK with the altered ATP-binding pocket, multiple proteins were labeled. In yeast, this procedure identified most of the known CDK substrates plus more than 150 additional yeast proteins. Similar approaches have also been used in mammalian cells to identify CDK substrates. For example, a hunt for substrates of the S phase cyclin A-CDK2 complex revealed 180 potential substrates. These substrates are currently being analyzed for their functions in cell cycle processes.

## KEY CONCEPTS OF SECTION 19.3

## Regulation of CDK Activity

- Cyclin-dependent kinases are activated by cyclin subunits. Their activity is controlled at multiple levels.
- Different cyclin subunits activate CDKs at different cell cycle stages. Cyclins are present only in the cell cycle stages that they promote.
- Protein degradation is the key mechanism responsible for restricting cyclins to the appropriate cell cycle stage. This degradation is mediated by the ubiquitin-proteasome system and the ubiquitin-protein ligases APC/C and SCF.
- Activating and inhibitory phosphorylation of the CDK subunit contributes to the regulation of CDK activity.
- CDK inhibitors (CKIs) inhibit CDK activity by binding directly to the cyclin-CDK complex.
- CDKs initiate every aspect of each cell cycle stage by phosphorylating many different target proteins. Systematic efforts using protein kinases engineered to bind only modified forms of ATP have led to the identification of many of these substrates.


### 19.4 Commitment to the Cell Cycle and DNA Replication

The previous section described the multiple mechanisms that control the different cyclin-CDK complexes. In this and the following two sections, we examine each cell cycle stage carefully and discuss how it is induced and controlled. We examine how cells initiate DNA replication and mitosis and how chromosomes are segregated. We focus on how cyclinCDK complexes and other key cell cycle regulators affect each cell cycle phase, and we examine the mechanisms that coordinate their activities.

This section investigates how cells decide whether or not to undergo cell division and how DNA replication is initiated. The process of cell cycle entry is well understood in budding yeast, and it was in this organism that the molecular mechanisms underlying this cell cycle transition were initially elucidated. We therefore begin by examining the molecular events governing cell cycle entry in budding yeast. We then investigate the striking similarities between the pathways that govern cell cycle entry in yeast and in metazoan cells, and we discuss the realization that many genes involved in this decision are frequently found mutated in cancer. Next we see how the decision to enter the cell cycle is influenced by extracellular events and learn about the signaling mechanisms that convey these environmental cues to the cell cycle machinery. Finally, we discuss the molecular mechanisms that govern initiation of DNA replication. We see why degradation of an S phase CKI is essential for this process, discover how CDKs ensure that DNA replication occurs only during S phase and then only once, and see how proteins that associate with DNA during DNA replication lay the foundation for accurate chromosome segregation during mitosis.

## Cells Are Irreversibly Committed to Division at a Cell Cycle Point Called START or the Restriction Point

In most eukaryotic cells, the key decision that determines whether or not a cell will divide is whether or not to enter S phase. In most cases, once a cell has become committed to entering the cell cycle, it must complete it. The budding yeast Saccharomyces cerevisiae regulates its proliferation in this manner, and much of our current understanding of the molecular mechanisms controlling entry into the cell cycle originated with genetic studies of $S$. cerevisiae.

Recall that $S$. cerevisiae exists in two different mating types, a and $\alpha$ (see Fig. 1-23). These two haploid cell types can mate to form an a/ $\alpha$ diploid cell. Mating is initiated by secreted pheromones and ultimately results in the fusion of two cells of opposite mating type. It is essential for successful mating that both mating partners be arrested in $G_{1}$. Thus, in addition to inducing genes critical for mating, the pheromones cause cells to arrest in $G_{1}$. If cells are in $G_{1}$, addition of pheromones will keep cells arrested in $G_{1}$, but once
![li4704542480e](li4704542480e.jpg)

FIGURE 19-12 Control of the $\mathrm{G}_{1}$-S phase transition. (a) In budding yeast, Cln3-CDK activity rises during $G_{1}$ and is controlled by nutrient availability. Once sufficiently active, the kinase phosphorylates the transcriptional repressor Whi5, promoting its export from the nucleus. This causes the transcription factor complex SBF to induce the transcription of the $G_{1} / S$ phase cyclin genes CLN1 and CLN2 and of other genes whose products are needed for DNA replication. $G_{1} / S$ phase CDKs further phosphorylate Whi5, promoting further CLN1 and CLN2 transcription. Once sufficiently high levels of $G_{1} / S$ phase CDKs have been produced, START is traversed. Cells enter the cell cycle: they initiate DNA replication, bud formation, and spindle pole body duplication.
a cell has committed to divide, it must complete a whole cell cycle before it can be arrested in $G_{1}$ again by pheromones. The point in late $G_{1}$ when $S$. cerevisiae cells become irrevocably committed to entering and traversing the entire cell cycle even when pheromones are present is called START. A similar transition point, called the restriction point, exists in mammalian cells at which cells become refractory to growth factor signals and proliferation inhibitor signals.

CDK activity is essential for entry into S phase. This was first realized in budding yeast, in which temperature-sensitive CDK mutants arrest in $G_{1}$, failing to form a bud and to initiate DNA replication (budding yeast has only one CDK that triggers all cell cycle transitions and is known as $C D C 28$ ). We now know that a CDK cascade triggers entry into the cell cycle. $G_{1}$ cyclin-CDK complexes stimulate the formation of $G_{1} / S$ phase cyclin-CDKs, which then initiate bud formation, centrosome duplication, and DNA replication. In yeast, the $G_{1}$ cyclin gene is called CLN3 (Figure 19-12a). Its mRNA is produced at a nearly constant level throughout the cell cycle, but its translation is regulated in response to nutrient levels and, as we will see shortly, it is a linchpin in
(b) Metazoans
![li4714542480e](li4714542480e.jpg)
(b) In vertebrates, $G_{1}$ CDK activity rises during $G_{1}$ and is stimulated by the presence of growth factors. When signaling from growth factors is sustained, the resulting cyclin D-CDK4/6 complexes begin phosphorylating Rb, releasing some E2F, which stimulates transcription of the genes encoding cyclin E, CDK2, and E2F itself. The cyclin E-CDK2 complexes further phosphorylate Rb, resulting in a positive feedback loop that leads to a rapid rise in the expression and activity of both E2F and cyclin E-CDK2. Once $G_{1} / S$ phase CDKs are sufficiently high, cells pass through the restriction point. They commence DNA replication and centrosome duplication.

coupling cell cycle entry to nutrient signals. Once sufficient Cln3 is synthesized from its mRNA, Cln3-CDK complexes phosphorylate and inactivate the transcriptional repressor Whi5. Phosphorylation of Whi5 promotes its export out of the nucleus, allowing the transcription factor complex SBF to induce transcription of the $\mathrm{G}_{1} / \mathrm{S}$ phase cyclin genes $C L N 1$ and $C L N 2$ as well as other genes important for DNA replication. Once produced, Cln1/2-CDKs contribute to further Whi5 phosphorylation. This positive feedback loop ensures the rapid accumulation of $\mathrm{G}_{1} / \mathrm{S}$ phase cyclin-CDKs. The point in the cell cycle at which 50 percent of Whi5 has exited the nucleus is the point when cells are irreversibly committed to division. It is the molecular definition of START. Cln1/2CDKs then trigger bud formation, entry into S phase, and the duplication of the centrosome (also known as the spindle pole body in yeast), which later in the cell cycle will organize the mitotic spindle.

## The E2F Transcription Factor and Its Regulator Rb Control the $\mathrm{G}_{1}$-S Phase Transition in Metazoans

The molecular events governing entry into S phase in mammalian-and in fact all metazoan-cells are remarkably similar to those in budding yeast (Figure 19-12b). $\mathrm{G}_{1}$ cyclins are present throughout $\mathrm{G}_{1}$ and are often found to be expressed at increased levels in response to growth factors. In turn, the $\mathrm{G}_{1}$ CDKs activate members of a small family of related transcription factors, referred to collectively as E2F transcription factors (E2Fs). During $\mathrm{G}_{1}, \mathrm{E} 2 \mathrm{Fs}$ are held inactive through their association with the retinoblastoma protein ( Rb ) until $\mathrm{G}_{1}$ CDKs activate E2Fs by phosphorylating and inactivating Rb . E2Fs then activate genes encoding many of the proteins involved in DNA synthesis. They also stimulate transcription of genes encoding the $\mathrm{G}_{1} / \mathrm{S}$ phase cyclins and the S phase cyclins. Thus the E2Fs have a function in late $\mathrm{G}_{1}$ that is similar to that of the S. cerevisiae transcription factor complex SBF.

Key to the regulation of E2F function is the Rb (retinoblastoma) protein. When E2Fs are bound to Rb, they function as transcriptional repressors. This is because Rb recruits chromatin-modifying enzymes that promote deacetylation and methylation of specific histone lysines, causing chromatin to assume a condensed, transcriptionally inactive form. $R B$ was initially identified as the gene mutated in retinoblastoma, a childhood cancer of the retina. Subsequent studies found Rb to be inactivated in many cancers, either by mutations in both alleles of $R B$ or by abnormal regulation of Rb phosphorylation.

Rb protein regulation by $\mathrm{G}_{1}$ CDKs in mammalian cells is analogous to Whi5 regulation by Cln3-CDK in yeast. Phosphorylation on multiple sites by $\mathrm{G}_{1}$ CDKs prevents Rb from associating with E2F and promotes its export out of the nucleus. This allows E2F to activate the transcription of genes required for entry into S phase. Once the expression of genes coding for the $\mathrm{G}_{1} / \mathrm{S}$ cyclins and CDK has been induced by phosphorylation of some of the Rb molecules, the resulting $\mathrm{G}_{1} / \mathrm{S}$ phase CDK complexes further phosphorylate
Rb in late $\mathrm{G}_{1}$. This is one of the principal biochemical events responsible for passage through the restriction point. Since E2F stimulates its own expression as well as that of the $\mathrm{G}_{1} / \mathrm{S}$ cyclin-CDKs, positive cross-regulation of E2F and $\mathrm{G}_{1} / \mathrm{S}$ cyclin-CDKs produces a rapid rise of both activities in late $\mathrm{G}_{1}$.

As they accumulate, S phase CDKs and mitotic CDKs maintain Rb protein in the phosphorylated state throughout the $\mathrm{S}, \mathrm{G}_{2}$, and early M phases. After cells complete anaphase and enter early $G_{1}$ or $G_{0}$, a fall in all cyclin-CDK activities leads to dephosphorylation of Rb. As a consequence, hypophosphorylated Rb is available to inhibit E2F activity during early $G_{1}$ of the next cycle and in $G_{0}$-arrested cells. Thus $G_{1} / S$ phase CDK activity remains low until cells decide to enter a new cell cycle and $\mathrm{G}_{1}$ CDKs break the inhibitory grip of Rb on E2F.

## Extracellular Signals Govern Cell Cycle Entry

Whether or not cells enter the cell cycle is influenced by extracellular as well as intracellular signals. Unicellular organisms such as yeasts, for example, enter the cell cycle only when they have reached an appropriate size, known as the critical cell size. This critical size, in turn, is controlled by nutrients available in the environment. This coordination between cell size and cell cycle entry will be discussed in Section 19.7. Here we restrict our discussion to the fact that $\mathrm{G}_{1}$ cyclin synthesis is responsive to the rate of protein synthesis, which is in turn controlled by pathways that are regulated by nutrients in the environment. This link between the macromolecule biosynthesis machinery and the cell cycle machinery is best understood in budding yeast. In this organism, the $\mathrm{G}_{1}$ cyclin transcript CLN3 contains a short upstream open reading frame that inhibits translation initiation when nutrients are limited. This inhibition is diminished when nutrients are in abundance. In the presence of sufficient nutrients, the TOR signaling pathway, which senses nutrients and growth factor signals, is active and stimulates translational activity (see Figure 10-32). Since Cln3 is a highly unstable protein, its concentration fluctuates with the translation rate of its mRNA. Consequently, the amount and activity of Cln3CDK complexes, which depend on the concentration of Cln 3 protein, are regulated by nutrient levels.

In multicellular organisms, cells are surrounded by nutrients, and as such, nutrients do not usually limit the rate of cell proliferation. Rather, cell proliferation is controlled by the presence of growth-promoting factors (mitogens) and growth-inhibiting factors (anti-mitogens) in the cell surroundings. Addition of mitogens to $\mathrm{G}_{0}$-arrested mammalian cells induces-as discussed in Chapter 16-receptor tyrosine kinase-linked signal transduction pathways that initiate signal transduction cascades that ultimately influence transcription and cell cycle control. They do so in multiple ways.

Mitogens activate the transcription of multiple genes. Most of these genes fall into one of two classes-early response or delayed response genes-depending on how soon their encoded mRNAs appear. Transcription of early response genes is induced within a few minutes after addition

of growth factors by signal transduction cascades that activate preexisting transcription factors in the cytosol or nucleus (see Chapter 16). Many of the early response genes encode transcription factors, such as c-Fos and c-Jun, that stimulate transcription of the delayed response genes. The early response transcription factor Myc induces the transcription of $\mathrm{G}_{1}$ cyclin and CDK genes. In addition to being controlled by transcription, $\mathrm{G}_{1}$ CDKs are regulated by CKIs. The CKI p15INK4b is a potent CDK inhibitor. In some tissues, mitogens inhibit the production of this CKI by inhibiting its transcription.

Cell proliferation in many tissues is regulated not only by proliferation-promoting mitogens, but also by anti-mitogens, which prevent entry into the cell cycle. Similarly, during differentiation, cells cease to divide and enter $\mathrm{G}_{0}$. Some differentiated cells (e.g., fibroblasts and lymphocytes) can be stimulated to re-enter the cell cycle and replicate. Many postmitotic differentiated cells, however, never re-enter the cell cycle to replicate again. Anti-mitogens and differentiation pathways prevent the accumulation of $\mathrm{G}_{1}$ CDKs. They antagonize the production of $\mathrm{G}_{1}$ cyclins and induce the production of CKIs. Transforming growth factor $\beta$ (TGF- $\beta$ ) is an important anti-mitogen. This hormone induces a signaling cascade that brings about $\mathrm{G}_{1}$ arrest by inducing the expression of p15INK4b. As we will see in Chapter 24, the signaling pathways that regulate $\mathrm{G}_{1}$ CDKs are found mutated in many human cancers.

## Degradation of an S Phase CDK Inhibitor Triggers DNA Replication

Entry into S phase is defined by the unwinding of origins of DNA replication. The molecular events leading to this unwinding are best understood in S. cerevisiae. $\mathrm{G}_{1} / \mathrm{S}$ phase CDKs play an essential role in this process. They turn off the machinery that degrades $S$ phase cyclins during exit from mitosis and $\mathrm{G}_{1}$, and they induce the degradation of a CKI that inhibits $S$ phase CDKs.

One of the important substrates of the $\mathrm{G}_{1} / \mathrm{S}$ phase cyclin-CDK complexes is Cdh1. During late anaphase, this substrate targeting factor directs APC/C to ubiquitinylate substrate proteins, including $S$ phase and mitotic cyclins, marking them for proteolysis by proteasomes. APC/C ${ }^{\text {Cdh1 }}$ remains active throughout $\mathrm{G}_{1}$, preventing the premature accumulation of $S$ phase and mitotic cyclins. Phosphorylation of Cdh1 by $\mathrm{G}_{1} / \mathrm{S}$ cyclin-CDKs causes it to dissociate from the APC/C complex, inhibiting further ubiquitinylation of S phase and mitotic cyclins during late $\mathrm{G}_{1}$ (Figure 19-13). This inhibition, combined with the induced transcription of $S$ phase cyclins during late $\mathrm{G}_{1}$, allows $S$ phase cyclins to accumulate as $\mathrm{G}_{1} / \mathrm{S}$ cyclin-CDK levels rise. Later in the cell cycle, S phase and mitotic CDKs take over to maintain Cdh1 in the phosphorylated, and hence inactive, state. Only as mitotic CDKs decline and a protein phosphatase known as Cdc14 is activated are these inhibitory phosphates removed from Cdh1, leading to its reactivation. In mammalian cells, similar mechanisms are responsible for stabilizing $S$ phase

Exit from mitosis and $\mathrm{G}_{1}$
![li4724542480e](li4724542480e.jpg)

FIGURE 19-13 Regulation of S phase and mitotic cyclin levels in budding yeast. In late anaphase, the anaphase-promoting complex (APC/C) ubiquitinylates S phase and mitotic cyclins. The activity of this ubiquitin-protein ligase is directed toward mitotic cyclins by a specificity factor called Cdh1. Cdh1 activity is regulated by phosphorylation. During exit from mitosis and $\mathrm{G}_{1}, \mathrm{Cdh} 1$ is dephosphorylated and active; during S phase and mitosis, Cdh1 is phosphorylated and dissociates from APC/C, and APC/C becomes inactive. The $\mathrm{G}_{1} / \mathrm{S}$ phase CDKs, which themselves are not APC/C ${ }^{\text {Cdh1 }}$ substrates, phosphorylate Cdh1 at the $\mathrm{G}_{1}-\mathrm{S}$ phase transition. A specific phosphatase called Cdc14 removes the regulatory phosphate from the specificity factor late in anaphase.
and mitotic cyclins, but the phosphatase(s) involved in dephosphorylation of Cdh1 have not been identified.

In S. cerevisiae, as S phase cyclin-CDK heterodimers accumulate in late $\mathrm{G}_{1}$ following the inactivation of APC/ $\mathrm{C}^{\mathrm{Cdh1}}$, they are immediately inactivated by binding of a CKI called Sic1 that is expressed late in mitosis and early in $\mathrm{G}_{1}$ (Figure 19-14). Because Sic1 specifically inhibits S phase and M phase CDK complexes, but has no effect on the $\mathrm{G}_{1} \mathrm{CDK}$ and $\mathrm{G}_{1} / \mathrm{S}$ phase CDK complexes, it functions as an S phase inhibitor. Initiation of DNA replication occurs when the Sic1 inhibitor is precipitously degraded following its ubiquitinylation by the SCF ubiquitin-protein ligase.

Degradation of Sic1 is induced by its phosphorylation by $\mathrm{G}_{1} / \mathrm{S}$ phase CDKs (see Figure 19-14). It must be phosphorylated at no fewer than six sites, which are relatively poor substrates for the $\mathrm{G}_{1} / \mathrm{S}$ phase CDKs, before it is bound sufficiently well by SCF to be ubiquitinylated. These multiple, poor $\mathrm{G}_{1} / \mathrm{S}$ phase CDK phosphorylation sites lead to an ultrasensitive, switch-like response in Sic1 degradation and hence precipitous activation of S phase CDKs (Figure 19-15). If Sic1 were inactivated following the phosphorylation of a single site, Sic1 molecules would start to get phosphorylated as soon as the levels of $\mathrm{G}_{1} / \mathrm{S}$ phase CDK activity begin to rise, leading to a gradual decrease in Sic1 levels. In contrast, because several sites need to be phosphorylated, at low levels of $\mathrm{G}_{1} / \mathrm{S}$ phase CDK activity only a few sites are phosphorylated and Sic1 is not destroyed. Only when $\mathrm{G}_{1} / \mathrm{S}$ phase CDK levels are high is Sic1 sufficiently phosphorylated at multiple sites to target it for degradation. Thus Sic1 degradation occurs only when $\mathrm{G}_{1} / \mathrm{S}$ phase CDK activity has reached its peak and virtually all other $\mathrm{G}_{1} / \mathrm{S}$ phase CDK substrates have been phosphorylated.

![li4734542480e](li4734542480e.jpg)

FIGURE 19-14 Control of $S$ phase onset in $S$. cerevisiae by regulated proteolysis of the $S$ phase inhibitor Sic1. The $S$ phase cyclinCDK complexes begin to accumulate in $\mathrm{G}_{1}$, but are inhibited by Sic1. This inhibition prevents initiation of DNA replication until the cells have completed all $\mathrm{G}_{1}$ events. $\mathrm{G}_{1} / \mathrm{S}$ phase CDKs assembled in late $\mathrm{G}_{1}$ phosphorylate Sic1 at multiple sites step $\mathbf{D}$, marking it for
ubiquitinylation by the SCF ubiquitin-protein ligase and subsequent proteasomal degradation step $\mathbf{D}$. The active $S$ phase CDKs then trigger initiation of DNA synthesis step $\mathbf{D}$ by phosphorylating and recruiting MCM helicase activators to DNA replication origins. See R. W. King et al., 1996, Science 274:1652.
(a) One optimal $\mathrm{G}_{1} / \mathrm{S}$ phase CDK site in Sic1
![li4744542480e](li4744542480e.jpg)
(b) Six sub-optimal $\mathrm{G}_{1} / \mathrm{S}$ phase CDK site in Sic1
![li4754542480e](li4754542480e.jpg)

FIGURE 19-15 Six suboptimal phosphorylation sites in Sic1 create a switch-like cell cycle entry. (a) A single optimal $\mathrm{G}_{1} / \mathrm{S}$ phase CDK phosphorylation site in Sic1 would result in a sluggish $\mathrm{G}_{1}$ - S phase transition. As $\mathrm{G}_{1} / \mathrm{S}$ phase CDKs accumulate during $\mathrm{G}_{1}$, Sic1 would be progressively degraded. As a result, S phase CDKs would slowly rise, and initiation of $S$ phase would be a drawn-out event. (b) Six suboptimal phosphorylation sites ensure that Sic1 is fully phosphorylated, and hence recognized by SCF, only when $\mathrm{G}_{1} / \mathrm{S}$ phase CDKs have reached high levels. This also ensures that Sic1 degradation occurs rapidly and when $\mathrm{G}_{1} / \mathrm{S}$ phase CDKs have accomplished all their other $\mathrm{G}_{1}$ tasks. See P. Nash et al., 2001, Nature 414:514-521, and D. O. Morgan, 2006.

Once Sic1 is degraded, the S phase cyclin-CDK complexes induce DNA replication by, as we will see shortly, phosphorylating several proteins involved in activating the replicative helicases. This mechanism for activating the S phase cyclin-CDK complexes-that is, inhibiting them as the cyclins are synthesized and then precipitously degrading the inhibitor-permits the sudden initiation of replication at large numbers of replication origins. An obvious advantage of proteolysis for controlling passage through this critical point in the cell cycle is that protein degradation is an irreversible process, ensuring that cells proceed in one direction through the cycle.

Entry into S phase in metazoan cells is regulated by a mechanism similar to that in budding yeast. Like Sic1, the CKI p27 prevents the premature activation of S phase CDKs during $\mathrm{G}_{1}$. Unlike Sic1, however, this CKI inhibits both S phase CDKs and $\mathrm{G}_{1} / \mathrm{S}$ phase CDKs and has additional cell cycle functions as well. For example, while inhibiting $\mathrm{G}_{1} / \mathrm{S}$ phase CDKs and S phase CDKs, p27 helps assemble and hence activate $\mathrm{G}_{1}$ CDKs. Like Sic1 in yeast, however, p27 is removed from cyclin-CDK complexes by ubiquitin-dependent protein degradation. Two pathways contribute to its degradation. Upon stimulation by mitogens, mitogen-activated protein kinases phosphorylate p27, promoting its export from the nucleus into the cytoplasm, where one of the cellular ubiquitin-protein ligases, KPC, is found. A second pathway, analogous to the one operating on Sic1, targets p27 for degradation at the $\mathrm{G}_{1}-\mathrm{S}$ phase transition. As $\mathrm{G}_{1} / \mathrm{S}$ phase CDKs and S phase CDKs reach high levels during late $G_{1}$ and early $S$ phase, they begin to phosphorylate p27, targeting it for ubiquitinylation by SCF. Degradation of p27 causes activation of $\mathrm{G}_{1} / \mathrm{S}$ phase and S phase CDKs. These kinases then initiate S phase by phosphorylating proteins important for the initiation of DNA replication.

## Replication at Each Origin Is Initiated Once and Only Once During the Cell Cycle

As discussed in Chapter 5, eukaryotic chromosomes are replicated from multiple replication origins. Initiation of replication from these origins occurs throughout $S$ phase. However, no eukaryotic origin initiates replication more than once per S phase. Moreover, S phase continues until replication from multiple origins along the length of each chromosome has resulted in complete replication of the entire chromosome. These two factors ensure that the correct gene copy number is maintained as cells proliferate.

S phase CDKs play an essential role in the regulation of DNA replication. The kinases initiate DNA replication only at the $\mathrm{G}_{1}-\mathrm{S}$ phase transition and prevent re-initiation from origins that have already fired. We first discuss how initiation of DNA replication is controlled, and the role of S phase CDKs in the process, before turning to the mechanisms whereby these kinases prevent re-initiation.

The mechanisms underlying the initiation of DNA replication are best understood in budding yeast, so we focus our discussion on this organism. However, it is important to note that the proteins and mechanisms controlling the initiation of DNA synthesis are essentially the same in all eukaryotic species. A protein complex known as the originrecognition complex (ORC) is associated with all DNA replication origins. In budding yeast, replication origins contain an 11-bp conserved core sequence to which ORC binds. In multicellular organisms, DNA replication origins lack a recognizable consensus sequence. Instead, chromatin-associated factors target ORC to the DNA. ORC and two additional replication initiation factors, Cdc6 and Cdt1, associate with the ORC at origins during $G_{1}$ to load the replicative helicases known as the MCM helicase complex onto DNA (Figure 19-16, step $\mathbf{\square}$ ). The MCM helicases function to unwind the DNA during the initiation of DNA replication.

To ensure that origins fire only once at the beginning of $S$ phase, the loading of MCM helicase complex and its activation occur at two opposing phosphorylation states. The MCM helicases can be loaded onto the DNA only in a state of low CDK activity that occurs when CDKs are inactivated during exit from mitosis and during early $G_{1}$. In other words, MCM helicases are loaded onto DNA when they are nonphosphorylated. In contrast, activation of the MCM helicases and the recruitment of DNA polymerases to the unwound origin DNA are triggered by S phase CDKs. Recall that $S$ phase CDKs become active only when $G_{1} / S$ phase CDK levels reach their peak and the CKIs of $S$ phase CDKs are destroyed. It is then that phosphorylation by S phase CDKs and a second heterodimeric protein kinase, DDK, activates the MCM helicases and recruits DNA polymerases to the sites of replication initiation (Figure 19-16, steps $\mathbf{\square}$ and $\mathbf{\square})$.

So how do S phase CDKs and DDK collaborate to initiate DNA replication? ORC and the two other initiation factors, Cdc6 and Cdt1, recruit the MCM helicases to sites of
replication initiation during $G_{1}$, when CDK activity is low (see Figure 19-16, step $\mathbf{\square}$ ). When DDK and S phase CDKs are activated in late $G_{1}$, DDK phosphorylates two subunits of the MCM helicase. The S phase CDKs phosphorylate two proteins called Sld2 and Sld3. These phosphorylation events (shown in green in Figure 19-16, steps $\mathbf{\square}$ and $\mathbf{\square}$ ) have an activating effect, promoting the recruitment of MCM helicase activators to sites of replication initiation. The helicase activators are called the Cdc45-Sld3 complex and the GINS complex. Exactly how they promote activation of the MCM helicases is not yet clear. In addition to activating the MCM helicase to unwind DNA, they recruit polymerases to the DNA: polymerase $\varepsilon$ to synthesize the leading strand and polymerase $\delta$ to synthesize the lagging strand (see Figure 19-16, step $\mathbf{\square}$ ). The replication machinery then initiates DNA synthesis.

S phase CDKs are not only essential for initiating DNA replication, but are also responsible for ensuring that each origin fires only once during $S$ phase. Re-firing of origins during $S$ phase is prevented by phosphorylation of several components of the MCM helicase loading machinery and the MCM helicase complex itself. To distinguish these phosphorylation events from the ones required for the initiation of DNA replication, they are depicted in yellow in Figure 19-16. Concomitant with activation of the MCM helicases, Cdc6 and Cdt1 dissociate from the sites of DNA replication initiation. Once they have done so, their phosphorylation leads to their degradation by the SCF ubiquitin-protein ligase. Phosphorylation of the MCM helicases leads to the export of these proteins from the nucleus after they dissociate from the DNA on completion of DNA replication. Thus only after CDK activity is lowered by APC/C ${ }^{\mathrm{Cdh} 1}$ during exit from mitosis can the MCM helicases be reloaded onto DNA. As a result, helicase loading is restricted to late stages of mitosis and early $G_{1}$ (see Figure 19-16, step $\mathbf{\square}$ ).

The general mechanisms governing the initiation of DNA replication in metazoan cells parallel those in S. cerevisiae, although small differences are found in vertebrates. The helicases are loaded in $G_{1}$, when CDK activity is low. Phosphorylation of MCM helicase activators by $\mathrm{G}_{1} / \mathrm{S}$ phase CDKs and S phase CDKs activates the helicases and promotes polymerase loading. As in yeast, phosphorylation of MCM helicase loading factors prevents reloading of MCM helicases until the cell passes through mitosis, thereby ensuring that replication from each origin occurs only once during each cell cycle. Here Cdt1 phosphorylation by multiple protein kinases, including CDKs, is especially critical for preventing reloading of the MCM helicases. In addition, a small protein, geminin, contributes to the inhibition of re-initiation at origins until cells complete a full cell cycle. Geminin is expressed in late $G_{1}$; it binds to and inhibits MCM helicase loading factors as they are released from origins once DNA replication is initiated during S phase (see Figure 19-16, step $\mathbf{\square}$ ). Geminin contains a destruction box at its N -terminus that is recognized by APC/C ${ }^{\mathrm{Cdh} 1}$, causing it to be ubiquitinylated in late anaphase and degraded by proteasomes. Its degradation

![li4764542480e](li4764542480e.jpg)

FIGURE 19-16 The molecular mechanisms governing the initiation of DNA replication. Step During exit from mitosis and early $\mathrm{G}_{1}$, when CDK activity is low, the MCM loading factors ORC, Cdc6, and Cdt1 load the replicative helicase, the MCM complex, onto DNA at replication origins. Step Activation of S phase CDKs and DDK marks the onset of S phase. They phosphorylate the MCM helicase, Sld2, and Sld3 (depicted as green phosphorylation events), to facilitate the loading of MCM helicase activators-the Cdc45-Sld3 and GINS complexes-onto sites of replication initiation. Loading of these activators leads MCM
helicases to unwind DNA. S phase CDKs also prevent reloading of MCM helicases by phosphorylating Cdc6 and Cdt1 (shown as yellow phosphorylation events), promoting their release from the replication origins and their degradation by SCF. S phase CDKs also phosphorylate MCM helicases, which leads to their export from the nucleus when the helicases disengage from the DNA when replication is complete. Step DNA polymerases are recruited to origins, which leads to the initiation of DNA synthesis (see Figure 5-30).
frees the MCM helicase loading factors, which are also dephosphorylated as CDK activity declines, to bind to ORC on replication origins and load MCM helicases during the following $\mathrm{G}_{1}$ phase.

## Duplicated DNA Strands Become Linked During Replication

During S phase, as chromosomes are duplicated to form sister chromatids, they become tethered to each other by protein links. The linkages between sister chromatids established during $S$ phase will be essential for their accurate segregation during mitosis.

The protein complexes that establish these linkages between sister chromatids are called cohesins. They are composed of four subunits: Smc1, Smc3, Scc1 (sometimes called Rad21), and Scc3 (Figure 19-17a). Smc1 and Smc3 are members of the SMC protein family, which is characterized by long coiled-coil domains that are flanked by a globular domain containing ATPase activity. The ATPase domains interact with Scc1 and Scc3 and, together, form a ring structure. These rings of cohesin embrace one or both copies of the replicated DNA. When cohesins are inactivated, sister chromatids do not associate properly with each other.

Cohesin-mediated cohesion between sister chromatids is established by a two-step process and is tightly tied to

![li4774542480e](li4774542480e.jpg)

FIGURE 19-17 Model for establishment of cohesin linkage of sister chromatids. Cohesin complexes form rings that link sister chromatids by embracing the two sister DNA molecules. (a) Schematic structure of the cohesin complex. (b) Mechanism whereby cohesins are loaded onto DNA and acquire their cohesive properties. Step $\mathbf{B}$ Cohesins are loaded onto chromosomes during $G_{1}$ by the cohesinloading complex Scc2-Scc4, but they do not possess cohesive properties (indicated as cohesins laterally associated with chromosomes). In this state, cohesins are dynamic and can dissociate from the DNA with the help of the Pds5-Wapl complex, which associates with cohesins. Step C Concomitant with DNA replication, closely behind the replication fork, cohesins are converted into cohesive molecules, able to hold sister chromatids together (indicated as cohesin rings encircling the
replicated sister chromatids), through acetylation of Smc3 by cohesin acetyltransferases (CoATs). Acetylation is accompanied by the binding of sororin to cohesin, which helps stabilize cohesins on chromosomes. During $G_{2}$, sister chromatids are replicated and linked along their entire length by cohesins. During this time, the Mei-S332/Shugoshin proteins recruit the protein phosphatase 2A (PP2A) to centromeric regions. Step $\mathbf{B}$ In vertebrate cells, cohesins are released from chromosome arms during prophase and early metaphase by the action of the Pds5Wapl complex and phosphorylation of cohesins by Polo kinase and Aurora B kinase. By the end of metaphase, cohesins are retained only in the region of the centromere, where Mei-S332/Shugoshin prevents cohesin phosphorylation, and hence dissociation, by recruiting PP2A.

DNA replication. Cohesins associate with chromosomes during $\mathrm{G}_{1}$; this association requires the cohesin loading factors Scc2 and Scc4 (Figure 19-17b, step $\mathbf{\square}$ ). In this phase of the cell cycle, cohesins are quite dynamic on chromosomes. They are continuously unloaded by a cohesin-associated complex composed of the Pds5 and Wapl proteins. This interphase dynamicity of cohesins is likely to be important for their role in regulating interphase chromatin structure and gene expression. Cohesins acquire their cohesive properties during DNA replication. The two duplicated DNA strands become entrapped within the cohesin rings as replication forks replicate the DNA (Figure 19-17b, step $\mathbf{\square}$ ). Converting DNA-bound $\mathrm{G}_{1}$ cohesins into cohesive complexes requires acetylation of the Smc3 subunit by cohesin acetyl transferases (CoATs). This acetylation prevents unloading of cohesins by Pds5-Wapl, thereby stabilizing the cohesins on chromosomes. In vertebrates, this stabilization of cohesins requires the cohesin-associated factor sororin (Figure 19-17b, step $\mathbf{\square}$ ). As we will see in Section 19.6, cohesins are essential for accurate attachment of the replicated sister chromatids to the mitotic spindle and for their segregation during mitosis. Cells lacking cohesins or the factors that load them onto chromosomes segregate chromosomes randomly.

Cohesins are not only critical for establishing linkages between replicated DNA molecules, and hence for their accurate segregation during mitosis, but also regulate gene expression. The functions of cohesins in this process appear to be diverse. In some instances, cohesins promote gene expression, whereas in others they restrain it. The mechanism whereby cohesins accomplish gene expression control is, however, the same in both cases: cohesins promote chromatin loop formation, thereby bringing enhancer or repressive elements close to the transcriptional start site. Defects of cohesin's gene expression regulation function are the cause of a group of diseases collectively called cohesinopathies. In these diseases, mutations in cohesin subunits or cohesin loading factors disrupt the expression of genes that are critical for development, causing limb and craniofacial abnormalities and intellectual disabilities. Cohesin's sister chromatid cohesion function, however, appears intact in these diseases. In contrast, as we will see in Section 19.8, defects in cohesin's cohesion function during meiosis cause miscarriages and intellectual disabilities.

## KEY CONCEPTS OF SECTION 19.4

## Commitment to the Cell Cycle and DNA Replication

- In yeast, START defines a stage in $\mathrm{G}_{1}$ after which cells are irreversibly committed to the cell cycle. Molecularly, it is defined as the point when 50 percent of Whi5 has exited the nucleus.
- The molecular events promoting entry into the cell cycle are conserved across species. $\mathrm{G}_{1}$ CDKs inhibit a transcriptional repressor. This allows the transcription of $\mathrm{G}_{1} / \mathrm{S}$ phase cyclin genes and other genes important for S phase.
- Extracellular signals such as nutritional state (in yeast) and the presence of mitogens and anti-mitogens (in vertebrates) regulate entry into the cell cycle.
- Various polypeptide growth factors called mitogens stimulate cultured mammalian cells to proliferate by inducing expression of early response genes. Many of these genes encode transcription factors that stimulate expression of genes encoding the $\mathrm{G}_{1} / \mathrm{S}$ phase cyclins and E2F transcription factors.
- The $\mathrm{G}_{1} / \mathrm{S}$ phase CDKs phosphorylate and inhibit Cdh1, the specificity factor that directs the anaphase-promoting complex (APC/C) to ubiquitinylate S phase and M phase cyclins. This allows $S$ phase cyclins to accumulate in late $G_{1}$.
- In yeast, S phase CDKs are initially inhibited by Sic1. Phosphorylation marks Sic1 for ubiquitinylation by the SCF ubiquitin-protein ligase and proteosomal degradation, releasing activated $S$ phase CDKs that trigger onset of the S phase (see Figure 19-14).
- DNA replication is initiated from helicase loading sites known as replication origins.
- Loading and activation of MCM helicases occur in mutually exclusive cell cycle states: MCM helicase loading can occur only when CDK activity is low (during early $\mathrm{G}_{1}$ ); MCM helicases are activated when CDK activity is high.
- S phase CDKs and DDK trigger the initiation of DNA replication by recruiting MCM helicase activators to origins (see Figure 19-16).
- Initiation of DNA replication occurs at each origin only once during the cell cycle because S phase CDKs activate the helicases and at the same time prevent additional helicases from loading onto DNA.
- Cohesins establish linkages between the replicated DNA molecules, which are essential for their accurate segregation later in the cell cycle. This linking mechanism is coupled to DNA replication.


### 19.5 Entry into Mitosis

Once $S$ phase has been completed and the entire genome has been duplicated, the pairs of duplicated DNA chromosomesthe sister chromatids-are segregated to the future daughter cells. This process requires not only the formation of the apparatus that facilitates this segregation-the mitotic spindle-but essentially a complete remodeling of the cell. Chromosomes condense and attach to the mitotic spindle, the nuclear envelope is disassembled, and almost all organelles are rebuilt or modified. All these events are triggered by mitotic CDKs.

This section first discusses how the mitotic CDKs are precipitously activated after the completion of DNA replication, during $\mathrm{G}_{2}$. We then describe how these protein kinases bring about the dramatic changes in the cell necessary to facilitate sister chromatid segregation during anaphase, focusing on the events as they occur in metazoans.

## Precipitous Activation of Mitotic CDKs Initiates Mitosis

Mitotic cyclin-CDKs initiate mitosis. Whereas levels of the catalytic CDK subunit are constant throughout the cell cycle, mitotic cyclins gradually accumulate during $S$ phase. Most eukaryotes contain multiple mitotic cyclins, which for historical reasons are subdivided into the cyclin A and cyclin B families. As they assemble, mitotic CDK complexes are maintained in an inactive state through inhibitory phosphorylation of the CDK subunit. Recall from Section 19.3 that two highly conserved tyrosine and threonine residues in mammalian CDKs are subject to regulated phosphorylation. In CDK1, the mitotic CDK, phosphorylation of tyrosine 15 and threonine 14 Imaintains mitotic cyclin-CDK complexes in an inactivate state. The phosphorylation state of T14 and Y15 is controlled by a dual-specificity protein kinase known as Wee1 and a dual-specificity phosphatase, Cdc25 (Figure 19-18). Such kinases and phosphatases can phosphorylate and dephosphorylate both serines/threonines and tyrosines, respectively. The regulation of mitotic CDKs by these activities underlies the abrupt activation of their kinase activity at the $\mathrm{G}_{2}-\mathrm{M}$ phase transition and explains the observation that although mitotic cyclins gradually accumulate during $S$ phase and $\mathrm{G}_{2}$, mitotic CDKs are not active until cells enter mitosis.

Studies in the fission yeast Schizosaccharomyces pombe unraveled the mechanisms that lead to the precipitous activation of mitotic CDKs during $\mathrm{G}_{2}$. The dual-specificity protein kinase Wee1 phosphorylates CDKs on the inhibitory tyrosine 15. (Threonine 14 is not phosphorylated in S. pombe CDK1.) Yeast cells with a defective wee $1^{+}$gene activate mitotic CDKs prematurely and hence experience premature entry into mitosis. These wee1 mutants not only enter mitosis prematurely, but are also smaller. This is because unlike most other eukaryotes, which coordinate cell size and cell division during $\mathrm{G}_{1}$, this coordination occurs during $\mathrm{G}_{2}$ in fission yeast. Fission yeast cells carrying a CDK1 mutation in which the tyrosine 15 residue is replaced by phenylalanine (which is structurally similar to tyrosine but cannot be phosphorylated) show the same premature mitotic CDK activation and entry into mitosis. Fission yeast cells carrying mutations in the $c d c 25^{+}$gene arrest in $\mathrm{G}_{2}$, indicating that the phosphatase that opposes Wee1 is essential for entry into mitosis.

Vertebrates contain multiple Wee1 protein kinases and multiple Cdc25 phosphatases, which collaborate to control not only mitotic CDK activity but also the activity of $\mathrm{G}_{1} / \mathrm{S}$ phase CDKs. One member of the Cdc25 family of phosphatases, Cdc25A, is activated in late $\mathrm{G}_{1}$. It removes the inhibitory phosphorylation on tyrosine 15 of the $\mathrm{G}_{1} / \mathrm{S}$ phase CDKs
![li4784542480e](li4784542480e.jpg)

FIGURE 19-18 Phosphorylation of the CDK subunit restrains mitotic CDK activity during $S$ phase and $\mathbf{G}_{2}$. Mitotic cyclins are synthesized during $S$ phase and $\mathrm{G}_{2}$ and bind to CDK1. However, the cyclinCDK complex is not active because threonine 14 and tyrosine 15 of the CDK1 subunit are phosphorylated by the protein kinase Wee1. Once DNA replication has been completed, the protein phosphatase Cdc25 is activated and dephosphorylates CDK1. Active mitotic CDKs further stimulate Cdc25. At the same time, mitotic CDKs inhibit Wee1, the protein kinase that places the inhibitory phosphorylation on the CDK subunit. Ongoing DNA replication inhibits Cdc25 activity. How Cdc25 is initially activated upon completion of DNA replication to put these feedback loops in motion is not yet known. Cell size also affects this regulatory loop. Once cells reach the appropriate size, Cdr2 inhibits Wee1, allowing Cdc25 to activate mitotic CDKs.
and S phase CDK catalytic subunit to activate the kinases. Another family member, Cdc25C, is active during $\mathrm{G}_{2}$ and removes the inhibitory phosphorylation on mitotic CDKs.

Activation of mitotic CDKs is the consequence of rapid inactivation of Wee1 and activation of Cdc25. Central to this rapid transition are feedback loops, whereby mitotic CDKs activate Cdc25 and inactivate Wee1 (see Figure 19-18). Phosphorylation of Cdc25 by mitotic CDKs stimulates its phosphatase activity; phosphorylation of Wee1 by mitotic CDKs inhibits its kinase activity. Ongoing DNA replication inhibits Cdc25 activity. A critical question that we know little about is how this positive feedback loop is started once DNA replication has been completed. CDKs that function earlier in the cell cycle have been suggested to start the positive feedback loop.

Although it is not yet known how the precipitous activation of mitotic CDKs is initiated, it is clear that once active, these protein kinases set in motion all the events necessary to ready the cell for chromosome segregation. The activation of mitotic CDKs is associated with changes in the subcellular localization of these kinases. Mitotic CDKs initially associate with centrosomes, where they are thought to facilitate

centrosome maturation. They then enter the nucleus, where they bring about chromosome condensation and nuclear envelope breakdown. In what follows, we discuss how the mitotic CDKs accomplish the coordinated execution of mitosis.

During DNA replication initiation, S phase CDKs work together with DDK to promote MCM helicase activation. In a similar manner, mitotic CDKs collaborate with other protein kinases to bring about the mitotic events. The Polo kinase family is critical for formation of the mitotic spindle as well as for chromosome segregation. The Aurora kinase family plays key roles in mitotic spindle formation and in ensuring that chromosomes attach to the mitotic spindle in the correct way so that they are segregated accurately during mitosis. Their contributions to the various mitotic events will also be discussed.

## Mitotic CDKs Promote Nuclear Envelope Breakdown

During interphase, chromosomes are surrounded by the nuclear envelope. The centrosomes that nucleate the mitotic spindle are located in the cytoplasm. For chromosomes to interact with the microtubules nucleated by the centrosomes, the nuclear envelope needs to be dismantled.

The nuclear envelope is a double-membrane extension of the endoplasmic reticulum containing many nuclear pore complexes (see Figures 1-12, 13-33, and 1-15). The lipid bilayer of the inner nuclear membrane is associated with the nuclear lamina, a meshwork of lamin filaments adjacent to the inside face of the nuclear envelope (Figure 19-19; see also Figure 1-15). The three nuclear lamins (A, B, and C) present in vertebrate cells belong to a class of cytoskeletal proteins, the intermediate filaments, that are critical in supporting cellular membranes. Once mitotic CDKs are activated at the end of $\mathrm{G}_{2}$, they phosphorylate specific serine residues in all three nuclear lamins. This phosphorylation causes depolymerization of the lamin intermediate filaments.
![li4794542480e](li4794542480e.jpg)

FIGURE 19-19 Electron micrograph of the nuclear lamina from a Xenopus oocyte. The regular mesh-like network of lamin intermediate filaments lies adjacent to the inner nuclear membrane (see Figure 18-47). [Republished with permission of Nature, from "Electron micrograph of the nuclear lamina from a Xenopus oocyte," U. Aebi et al., p. 323, 1986; permission conveyed through Copyright Clearance Center, Inc.]

Depolymerization of the nuclear lamins leads to disintegration of the nuclear lamina and contributes to disassembly of the nuclear envelope.

Mitotic CDKs also affect other nuclear envelope components. The CDKs phosphorylate specific nucleoporins, which causes nuclear pore complexes to dissociate into subcomplexes during prophase. Phosphorylation of integral membrane proteins of the inner nuclear membrane is thought to decrease their affinity for chromatin and further contributes to the disassembly of the nuclear envelope. The weakening of the associations between the inner nuclear membrane proteins and the nuclear lamina and chromatin allows sheets of inner nuclear membrane to retract into the endoplasmic reticulum, which is continuous with the outer nuclear membrane.

## Mitotic CDKs Promote Mitotic Spindle Formation

A key function of mitotic CDKs is to induce the formation of the mitotic spindle, also known as the mitotic apparatus. As we saw in Chapter 18, the mitotic spindle is made of microtubules that attach to chromosomes via specialized protein structures associated with the chromosomes, known as kinetochores. In most organisms, the mitotic spindle is organized by centrosomes, sometimes called spindle pole bodies. The centrosomes contain a specialized tubulin, $\gamma$ tubulin, which, together with associated proteins, nucleates microtubules. Notable exceptions to these centrosome-based spindle assembly mechanisms are higher plants and metazoan oocytes. In these cells, ( - ) ends of microtubules are cross-linked, and the microtubules selfassemble into a spindle.

The function of the mitotic spindle is to segregate chromosomes so that the sister chromatids separate from each other and are moved to opposite poles of the cell (see Figure 18-38). To achieve this, the mitotic spindle must attach to the chromosomes so that one kinetochore of each sister chromatid pair attaches to microtubules emanating from opposite spindle poles. Sister chromatids are then said to be bi-oriented. In what follows, we describe how the mitotic spindle forms, how chromosomes attach to it, and how cells correct faulty attachments.

During $\mathrm{G}_{1}$, cells contain a single centrosome, which functions as the major microtubule nucleating center of the cell. Mitotic spindle formation begins at the $\mathrm{G}_{1}-\mathrm{S}$ phase transition with the duplication of the centrosome. The mechanism whereby this duplication occurs is poorly understood, but at the heart of this process is the duplication of the pair of centrioles, short microtubules arranged orthogonally to each other. As discussed in Chapter 18, $\mathrm{G}_{1}$ cells contain a single pair of centrioles. Concomitant with entry into S phase and triggered by the $\mathrm{G}_{1} / \mathrm{S}$ phase CDKs, the two centrioles split apart, and each centriole begins to grow a daughter centriole (see Figure 18-36). The new centrioles grow and mature during S phase, each centriole pair begins to assemble centrosomal material, and by $\mathrm{G}_{2}$ the two centrosomes have formed. Several additional protein kinases

![li4804542480e](li4804542480e.jpg)

FIGURE 19-20 Chromosome attachment to the mitotic spindle. Chromosomes attach to the mitotic spindle and assemble at the spindle center. They then attach, via their kinetochores, to the ends of microtubules (called end-on attachments), and these attachments are stabilized by additional microtubules. The final chromosome attachment, in which the chromosome is stably bi-oriented on the mitotic spindle, is shown. " $(-)$ end" indicates the minus end of the microtubule, " $(+)$ end" the plus end.
have been identified that control centrosome duplication. Chief among them is a member of the conserved Polo kinase family, Plk4. How $\mathrm{G}_{1} / \mathrm{S}$ phase CDKs and Plk4 promote centrosome duplication is not yet understood, but is thought to involve the phosphorylation of multiple centrosome components, which facilitates their duplication and growth. As we will see, the Polo kinases not only play a key role in centrosome duplication, but also participate in essentially all aspects of mitosis.

The key initiating step of mitotic spindle formation is the severing of the ties that link the duplicated centrosomes. This centrosome disjunction occurs in $\mathrm{G}_{2}$ and is triggered by mitotic CDKs (see Figure 18-36). As soon as this separation occurs, microtubules extend from both centrosomes, and the two centrosomes move away from each other, pulled by the motor protein dynein. The specifics of microtubule array formation and mitotic spindle assembly were discussed in Chapter 18. Here we briefly consider how chromosomes attach to the mitotic spindle and how mistakes in the process are corrected.

For chromosomes to be accurately segregated during mitosis, the sister chromatid pair must be stably bi-oriented on the mitotic spindle (Figure 19-20). How is this accomplished? Once centrosomes have moved apart from each other, microtubules, in a search-and-capture mechanism, begin to interact with the kinetochores of sister chromatid pairs. Initially, chromosomes glide along the length of microtubules, propelled by motor proteins. When a chromosome reaches the $(+)$ end of a microtubule, the kinetochores attach to microtubules in an end-on attachment, the final configuration in which chromosomes are linked to the mitotic spindle (Figure 19-21, see also Figure 18-41). Kinetochores
of sister chromatids then bind microtubules emanating from the opposite spindle poles.

The ultimate goal of chromosome attachment to the mitotic spindle is that each and every chromosome be attached to the mitotic spindle in a bi-oriented manner (also known as amphitelic attachment; Figure 19-22a). How does the cell "know" that this has occurred? Microscopic analysis of chromosome attachment has shown that initially, many chromosomes attach to microtubules in faulty ways. A kinetochore can attach to microtubules emanating from both poles, a situation called merotelic attachment (Figure 19-22b). Kinetochores of a sister chromatid pair can attach to microtubules from the same pole (syntelic attachment;
![li4814542480e](li4814542480e.jpg)
(b)
![li4824542480e](li4824542480e.jpg)

FIGURE 19-21 Electron micrograph of purified yeast kinetochores bound to taxol-stabilized microtubules. (a) End-on attachment of a yeast kinetochore at a microtubule. (b) A cartoon schematizing the key features of the image. The ring structure embracing the microtubule most likely represents the outer kinetochore Dam1 complex and part of the Ndt80 complex, also an outer kinetochore component. The globular structure at the end of the complex most likely reflects the inner kinetochore and protein complexes that link the inner kinetochore to the outer kinetochore. [Part (a) from S. Gonen et al., 2012, Nature Struct. Mol. Biol. 19:925-929, Fig. 2d.]

![li4834542480e](li4834542480e.jpg)

Figure 19-22c), or only one kinetochore can attach to microtubules (monotelic attachment; Figure 19-22d). Clearly none of these attachments would result in accurate chromosome segregation. Thus mechanisms must be in place that detect and correct such faulty attachments.

The sensing mechanism used by cells to detect incorrect attachments is based on tension. When sister chromatids are correctly attached to microtubules, their kinetochores are under tension (see Figure 19-22a). Microtubules attached to the kinetochores pull at them, and the cohesin molecules that hold the sister chromatids together withstand these forces, creating tension at kinetochores. Merotelic, syntelic, or monotelic attachment leads to insufficient tension at kinetochores, allowing the cell to distinguish these faulty forms of attachment from the correct amphitelic one.

How does the cell sense whether or not kinetochores are under tension? The protein kinase Aurora B and its associated regulatory factors, together known as the chromosomal passenger complex (CPC), sense kinetochores that are not under tension and sever these microtubule attachments, giving cells a second chance to get the attachment right. The molecular basis for this sensing mechanism is partly understood. Recall that outer kinetochore components, especially the Ndc80 complex, bind microtubules (see Figure 18-41). Aurora B phosphorylates Ndc80. When phosphorylated, the protein loses its microtubule-binding activity. Aurora B localizes to the inner kinetochore. When kinetochores are not under tension, Ndc80 is in close proximity to Aurora B, and the protein kinase can phosphorylate the protein, destabilizing any kinetochore-microtubule attachments (see Figure 19-22b-d). When microtubules are attached correctly to kinetochores, microtubule forces pull Ndc80 away from Aurora B, and the kinase can no longer phosphorylate Ndc80 (see Figure 19-22a). Protein phosphatase 1 (PP1) localizes

FIGURE 19-22 Stable and unstable chromosome attachments. When sister kinetochores attach to microtubules emanating from opposite spindle poles, they are stably attached. This configuration is called amphitelic attachment (a). Microtubules (green) pull kinetochores; cohesins resist this pulling force. The resulting tension leads to the outer kinetochore component Ndc80 (yellow) being pulled away from the protein kinase Aurora B (red), which localizes to the inner kinetochore. As a result, Aurora B can no longer phosphorylate Ndc80, and kinetochore-microtubule attachments are stable. When one kinetochore attaches to microtubules emanating from two opposite spindle poles (merotelic attachment, b), or both sister kinetochores attach to microtubules emanating from the same spindle pole (syntelic attachment, c), or only one of the two sister kinetochores attaches to microtubules (monotelic attachment, d), Ndc80 is not pulled away from Aurora B. As a result, Aurora B phosphorylates Ndc80, and Ndc80 can no longer bind to microtubules.
to the outer kinetochore and continuously dephosphorylates Ndc80. Thus, when kinetochores are under tension and pulled away from Aurora B, Ndc80 is quickly dephosphorylated by PP1 and microtubule-kinetochore attachments are stabilized.

Microtubules continuously pull on chromosomes. Once all chromosomes have attached to microtubules in an amphitelic manner, the only thing that prevents chromosomes from segregating to the poles is the cohesins that hold them back in the middle of the spindle (see Figure 19-22a). As we will see in Section 19.6, it is the severing of these cohesins that initiates anaphase chromosome segregation.

## Chromosome Condensation Facilitates Chromosome Segregation

Chromosome segregation not only requires the building of the apparatus that segregates chromosomes, but also requires that the DNA be compacted into travel-friendly structures. Any attempt to segregate the long and intertwined DNAprotein complexes present in interphase cells would lead to breakage of the DNA and hence loss of genetic material. To avoid this fate, cells compact their chromosomes during prophase into the dense structures we have become acquainted with through light or electron microscopy (Figure 19-23).

Chromosome condensation results in a dramatic reduction in chromosome length, up to 10,000-fold in vertebrates. The second key aspect of the compaction process is the untangling of the intertwined sister chromatids. This process, called sister chromatid resolution, is mediated in part by the decatenation activity of topoisomerase II and goes hand in hand with the condensation process.

Recent studies have shown that chromosome compaction occurs via the generation of consecutive loops that lead to the

![li4844542480e](li4844542480e.jpg)

FIGURE 19-23 Scanning electron micrograph of a metaphase chromosome. During metaphase, the chromosomes are fully condensed, and the two individual sister chromatids are visible. [Biophoto Associates/Science Source.]
formation of a fiber at the bases of the loops (Figure 19-24a). These fibers are then compressed, leading to further chromosome compaction (Figure 19-24b). Central to the process
![li4854542480e](li4854542480e.jpg)

FIGURE 19-24 A model for chromosome compaction during mitosis. (a) The folding of chromosomes into consecutive loops leads to the formation of proteinaceous fibers at loop bases. (b) These chromosome axes are then further compressed to generate highly condensed chromosomes. The proteins responsible for loop formation are likely to be condensins. (Republished with permission of AAAS, from Science, N. Naumova et al., 342, 6161, 2013; permission is conveyed through Copyright Clearance Center, Inc.]
of chromosome condensation is a protein complex known as condensin. This protein complex, which is related to the cohesins that link sister chromatids together after DNA replication, was first identified based on its ability to promote chromosome condensation in frog extracts. Like cohesins, condensins are composed of two large coiled-coil SMC protein subunits that associate through their ATPase domains with non-SMC subunits. When condensin function is lost in cells, chromosomes do not condense and sister chromatid tangles are not resolved. Condensins are likely to create the intrachromosomal linkages that package chromosomes into consecutive loops. Their association with chromosomes is facilitated by mitotic CDKs and Aurora B. These two protein kinases phosphorylate histone H 2 A , allowing condensins to bind chromatin.

Finally, dissociation of cohesins leads to further compaction of chromosomes. A large fraction of cohesins are removed from chromosomes during prophase (see Figure 19-17, step ■). This process is mediated by phosphorylation of cohesins by Polo kinase and Aurora B kinase. In most organisms, cohesins are maintained only around centromeres. Cohesins around centromeres are protected from phosphorylation-dependent removal by protein phosphatase 2A (PP2A). This phosphatase is recruited to centromeric regions by a member of a family of PP2A targeting factors known as the Mei-S332/Shugoshin family of proteins. The protected pool of cohesins provides the resistance to the pulling force exerted by microtubules necessary to establish tension at bi-oriented kinetochores. As we will see in Section 19.8, this protection mechanism also plays an essential role in establishing the meiotic chromosome segregation pattern.

## KEY CONCEPTS OF SECTION 19.5

## Entry into Mitosis

- Mitotic CDKs induce entry into mitosis in all eukaryotes.
- Mitotic CDKs are held inactive until the completion of DNA replication by inhibitory phosphorylation of the CDK subunit.
- Mitotic CDKs promote their own activation through positive feedback loops leading to the rapid inactivation of Wee1 kinase and activation of Cdc25 phosphatase.
- Mitotic CDKs induce nuclear envelope breakdown in most eukaryotes by phosphorylating lamins.
- Centrosome duplication occurs during S phase. Mitotic CDKs induce the separation of the duplicated centrosomes, which initiates mitotic spindle formation.
- Sister chromatids attach to the mitotic spindle via their kinetochores in a bi-oriented manner, with one sister kinetochore attaching to microtubules emanating from one spindle pole and the other one to microtubules nucleated by the other spindle pole.

- Cells sense bi-orientation of sister chromatids through a tension-based mechanism. When kinetochores are not under tension, the protein kinase Aurora B phosphorylates the microtubule-binding subunits of the kinetochore, which decreases their microtubule binding affinity.
- Chromosomes must be compacted for segregation.
- Condensins, protein complexes that are related to cohesins, facilitate chromosome condensation and are activated by mitotic CDKs.


### 19.6 Completion of Mitosis: Chromosome Segregation and Exit from Mitosis

Once all chromosomes have condensed and have correctly attached to the mitotic spindle, chromosome segregation commences. In this section, we discuss how cleavage of cohesins by a protease known as separase triggers anaphase chromosome movement and how this cleavage is regulated. We then see how the same machinery that initiates cohesin cleavage at the metaphase-anaphase transition, APC/C, also initiates mitotic CDK inactivation. Next we describe how phosphatases activated at the end of mitosis participate in mitotic CDK inactivation, bringing about the disassembly of mitotic structures and the resetting of the cell to the $\mathrm{G}_{1}$ state. We end this section with a discussion of cytokinesis, the process that produces two daughter cells.

## Separase-Mediated Cleavage of Cohesins Initiates Chromosome Segregation

As mentioned in the previous section, each sister chromatid of a metaphase chromosome is attached to microtubules via its kinetochore (see Figure 19-20). At metaphase, the mitotic spindle is in a state of tension, with forces pulling the two
kinetochores toward the opposite spindle poles, but sister chromatids do not separate because they are held together at their centromeres by cohesins. In all organisms analyzed to date, loss of cohesins from chromosomes triggers anaphase chromosome movement. The mechanism that brings about this loss of cohesins from chromosomes is conserved as well. A protease known as separase cleaves the cohesin subunit Scc1 (Rad21), breaking the protein circles linking sister chromatids (Figure 19-25). Once this link is broken, anaphase begins as poleward force exerted on the kinetochores moves the split sister chromatids toward opposite spindle poles.

Cohesin cleavage was discovered in budding yeast. Analysis of the Scc1 subunit by Western blot analysis showed that from $G_{1}$ until metaphase, the protein migrated in polyacrylamide electrophoresis gels according to its predicted molecular weight, but during anaphase, the protein ran considerably faster in the gels, indicating that it had somehow become smaller. Subsequent studies showed that the faster-migrating form of Scc1 was indeed a cleavage product. Insight into the identity of the protein that was responsible for the cleavage of cohesin came from the analysis of previously identified yeast mutants that failed to segregate chromosomes during anaphase. A mutant form of the gene encoding Esp1-what we now know to be separase-failed to produce the cleavage fragment. Subsequent analyses revealed not only that separase is a protease, but that cleavage of cohesin is essential for chromosome segregation. Cells expressing a form of Scc1 with its cleavage sites mutated fail to segregate their chromosomes. Given the irreversible nature of Scc1 cleavage, it is absolutely essential that separase activity be tightly controlled. In what follows, we discuss its regulation.

## APC/C Activates Separase Through Securin Ubiquitinylation

Prior to anaphase, a protein known as securin binds to and inhibits separase (see Figure 19-25). Once all kinetochores
![li4864542480e](li4864542480e.jpg)

FIGURE 19-25 Regulation of cohesin cleavage. Separase, a protease that can cleave the Scc1 subunit of cohesin complexes, is inhibited before anaphase by the binding of securin. Mitotic CDKs also inhibit separase by phosphorylating it. When all the kinetochores have attached to spindle microtubules and the spindle apparatus is properly assembled and oriented, the Cdc20 specificity factor associated with

APC/C directs it to ubiquitinylate securin and mitotic cyclins. Following securin degradation and a decrease in mitotic CDK activity, the released and dephosphorylated separase cleaves the Scc1 subunit, breaking the cohesin circles and allowing sister chromatids to be pulled apart by the spindle apparatus that is pulling them toward opposite spindle poles.

have attached to spindle microtubules in the correct bi-oriented manner, the APC/C ubiquitin-protein ligase, directed by specificity factor Cdc20 (together, the complex is known as APC/C ${ }^{\text {Cdc20 }}$ ), ubiquitinylates securin (note that this specificity factor is distinct from Cdh1, which targets APC/C substrates for degradation later during mitosis and $\mathrm{G}_{1}$ ). Polyubiquitinylated securin is rapidly degraded by proteasomes, thereby releasing separase.

APC/C ${ }^{\text {Cdc20 }}$ is phosphorylated in prophase by mitotic CDK phosphorylation of several APC/C subunits. However, this phosphorylated APC/C ${ }^{\text {Cdc20 }}$ is not active until all chromosomes have become bi-oriented on the mitotic spindle. As we will see in Section 19.7, APC/C ${ }^{\text {Cdc20 }}$ is inhibited by a checkpoint pathway that ensures that anaphase is not initiated until all chromosomes have achieved proper attachment to the mitotic apparatus. Cdc20 is inhibited until every kinetochore has attached to microtubules and tension has been applied to the kinetochores of all sister chromatids, pulling them toward opposite spindle poles. In vertebrate cells, separase is also regulated by phosphorylation. Mitotic CDK activity inhibits separase during prophase and metaphase. Only when mitotic CDK activity begins to decline at the metaphase-anaphase transition through APC/C ${ }^{\text {Cdc20 }}$. mediated protein degradation can separase become active and trigger chromosome segregation.

Once cohesins are cleaved, anaphase chromosome movement ensues. As discussed in Chapter 18, chromosome segregation is mediated by microtubule depolymerization and motor proteins as the spindle poles move away from each other. Decline in mitotic CDK activity is important for these anaphase chromosome movements. When mitotic CDK inactivation is inhibited, anaphase does occur, but it is abnormal. Dephosphorylation of a number of microtubule-associated proteins that affect microtubule dynamics appears important for this process. In budding yeast, this dephosphorylation is brought about by the protein phosphatase Cdc14, which, as we will see, plays an essential role in the final cell cycle stage, exit from mitosis.

## Mitotic CDK Inactivation Triggers Exit from Mitosis

Anaphase spindle elongation and the events associated with exit from mitosis-mitotic spindle disassembly, chromosome decondensation, and nuclear envelope re-formation-are brought about by the dephosphorylation of CDK substrates. In other words, exit from mitosis can be viewed as a reversal of entry into mitosis. The phosphorylation events that triggered the different mitotic events need to be undone for the cell to revert to the $\mathrm{G}_{1}$ state.

Dephosphorylation of mitotic CDK substrates is caused by the inactivation of mitotic CDKs. In most organisms, mitotic CDK inactivation is triggered by APC/C ${ }^{\text {Cdc20 }}$-mediated degradation of mitotic cyclins. As mitotic CDKs activate APC/C ${ }^{\text {Cdc20 }}$, they initiate their own demise. In budding yeast, only about 50 percent of mitotic cyclins are degraded by APC/C ${ }^{\text {Cdc20 }}$. As we will see in Section 19.7, a pool of mitotic cyclins is protected from APC/C ${ }^{\text {Cdc20 }}$ to allow for enough
time to position the mitotic spindle accurately within the cell. How a fraction of mitotic cyclins is protected from APC/C ${ }^{\text {Cdc20 }}$ is not known, but it is clear that a second mitotic CDK-inactivating step is needed for exit from mitosis to occur. The conserved protein phosphatase Cdc14 brings about this second step in mitotic CDK inhibition.

In budding yeast, complete inactivation of mitotic CDKs requires the destruction of mitotic cyclins by APC/C ${ }^{\text {Cdh1 }}$ and the accumulation of the CDK inhibitor Sic1, whichrecall—holds S phase CDKs in check until cells enter the cell cycle. Both APC/C ${ }^{\text {Cdh1 }}$ and Sic1 are inhibited by mitotic CDKs. Conversely, APC/C ${ }^{\text {Cdh1 }}$ and Sic1 inhibit mitotic CDKs (Figure 19-26). The protein phosphatase Cdc14 throws the switch between these two mutually antagonistic states during anaphase. Cdc14 is kept inactive during most of the cell cycle, but is activated during anaphase by a GTPase signaling pathway known as the mitotic exit network. This signaling cascade, as we will see in Section 19.7, is responsive to spindle position and becomes active only in anaphase when the anaphase spindle is properly positioned within the cell. Once activated during anaphase, Cdc14 dephosphorylates APC/C ${ }^{\text {Cdh1 }}$ and Sic1 to promote mitotic cyclin degradation and mitotic CDK inactivation, respectively. This process leads to exit from mitosis.

Phosphatase activity is also essential for exit from mitosis in vertebrates. Simple inactivation of mitotic CDKs is not sufficient to trigger a timely exit from mitosis. It is not yet clear which phosphatase dephosphorylates CDK substrates to reset the cell to the $\mathrm{G}_{1}$ stage. Both protein phosphatase 1 and protein phosphatase 2A have been implicated in the process.

Ultimately, reversal of mitotic CDK phosphorylation changes the activities of many proteins back to their interphase states. Dephosphorylation of condensins, histone H1, and other chromatin-associated proteins leads to the decondensation of mitotic chromosomes in telophase. The targets of CDKs whose dephosphorylation is important for mitotic spindle disassembly are not known, but it is likely that
![li4874542480e](li4874542480e.jpg)

FIGURE 19-26 The protein phosphatase Cdc14 triggers exit from mitosis in budding yeast. During mitosis, mitotic CDK activity inhibits its inhibitors, APC/C ${ }^{\text {Cdh1 }}$ and Sic1. During G1, APC/C ${ }^{\text {Cdh1 }}$ and Sic1 inhibit mitotic CDKs. During exit from mitosis, the protein phosphatase Cdc14 throws the switch between these two antagonistic states. The mitotic exit network activates the phosphatase during anaphase, allowing it to dephosphorylate APC/C ${ }^{\text {Cdh1 }}$, thereby activating it. The phosphatase also promotes the accumulation of Sic1. In addition, Cdc14 dephosphorylates the many mitotic CDK substrates, which leads to rapid exit from mitosis.

multiple proteins are targets. More is known about how the nuclear envelope re-forms. Dephosphorylated inner nuclear membrane proteins are thought to bind to chromatin once again. As a result, multiple projections of regions of the ER membrane containing these proteins are thought to associate with the surfaces of the decondensing chromosomes and then fuse with one another, directed by an unknown mechanism to form a continuous double membrane around each chromosome (Figure 19-27). Dephosphorylation of nuclear pore subcomplexes allows them to reassemble into complete NPCs traversing the inner and outer membranes soon after fusion of the ER projections. Ran-GTP, required for driving most nuclear import and export (see Chapter 13), stimulates both fusion of the ER projections to form daughter nuclear envelopes and assembly of NPCs (see Figure 19-27). The Ran-GTP concentration is highest in the microvicinity of the decondensing chromosomes because the Ran-guanine nucleotide-exchange factor (Ran-GEF) is bound to
![li4884542480e](li4884542480e.jpg)

FIGURE 19-27 Model for reassembly of the nuclear envelope during telophase. Extensions of the endoplasmic reticulum (ER) associate with each decondensing chromosome and then fuse with one another, forming a double membrane around the chromosome. Dephosphorylated nuclear pore subcomplexes reassemble into nuclear pores, forming individual mini-nuclei called karyomeres. The enclosed chromosome further decondenses, and subsequent fusion of the nuclear envelopes of all the karyomeres at each spindle pole forms a single nucleus containing a full set of chromosomes. NPC, nuclear pore complex. See B. Burke and J. Ellenberg, 2002, Nature Rev. Mol. Cell Biol. 3:487.
chromatin. Consequently, membrane fusion is stimulated at the surfaces of decondensing chromosomes. Sheets of nuclear membrane with inserted NPCs then fuse with one another to form one nuclear membrane around all chromosomes.

## Cytokinesis Creates Two Daughter Cells

When chromosome segregation is completed, the cytoplasm and organelles are distributed between the two future daughter cells. This process is called cytokinesis. With the exception of higher plants, the division of the cell is brought about by a contractile ring made of actin and the actin motor myosin (see Figure 17-35). During cytokinesis, the ring contracts in a manner similar to muscle contraction, pulling the membrane inward and eventually closing the connection between the two daughter cells.

Cytokinesis must be coordinated with other cell cycle events in space and time. For cell division to produce two daughter cells each containing the components necessary for survival, the division plane must be placed so that each cell receives approximately half of the parent cell's cytoplasmic content and exactly half of the genetic content. Cytokinesis must also be coordinated with the completion of mitosis. In what follows, we explore both these aspects of cytokinesis regulation.

In animal cells, the contractile ring forms during anaphase and is placed in the middle of the anaphase spindle. This placement ensures that each daughter cell receives half of the genetic material. Despite the importance of this coordination, surprisingly little is known about it. Some experiments support the idea that signals sent from the spindle midzone, the area of the anaphase spindle between the segregated DNA masses, to the cell cortex are important for coordinating the site of cytokinesis with spindle position. Other research suggests that microtubules of the spindle interact with the cell cortex, positioning the cleavage furrow with respect to spindle pole position. Most likely, a combination of these pathways governs the formation of the cleavage furrow during cytokinesis.

As we will see in Section 19.7, cells have developed surveillance mechanisms that ensure that the site of cytokinesis is coordinated with spindle position. This is especially important during asymmetric cell divisions, which give rise to cells with different sizes or fates. Such cell divisions are essential during development and in stem cell divisions (see Chapter 21). Cytokinesis must also be coordinated with other cell cycle events. The major signal for cytokinesis is the inactivation of mitotic CDKs. Cells expressing a stabilized version of mitotic cyclins progress through anaphase but do not undergo cytokinesis. The CDK targets in the cytokinesis machinery have not yet been discovered.

This concludes our discussion of the molecular events of cell division. As we have seen, cyclin-dependent kinases and ubiquitin-mediated protein degradation are at the center of its control (Figure 19-28). In what follows, we discuss the mechanisms that ensure that a cell cycle stage is not initiated until the previous one has been completed and that each cell cycle step occurs accurately.

![li4894542480e](li4894542480e.jpg)

FIGURE 19-28 Fundamental processes in the eukaryotic cell cycle. See text for discussion.

## KEY CONCEPTS OF SECTION 19.6

## Completion of Mitosis: Chromosome Segregation and Exit from Mitosis

- Cleavage of cohesin by separase induces chromosome segregation during anaphase.
- At the onset of anaphase, APC/C is directed by Cdc20 to ubiquitinylate securin, which is subsequently degraded by proteasomes. The degradation of securin activates separase.
- Exit from mitosis is triggered by mitotic CDK inactivation mainly brought about by mitotic cyclin degradation.
- Exit from mitosis requires the activity of protein phosphatases such as Cdc14 to remove mitotic phosphates from many different proteins, permitting mitotic spindle disassembly, the decondensation of chromosomes, and the reassembly of the nuclear envelope.
- Cytokinesis finalizes cell fission and must be coordinated with the site of nuclear division. This coordination is especially important in cells undergoing an asymmetric division.


### 19.7 Surveillance Mechanisms in Cell Cycle Regulation

Surveillance mechanisms known as checkpoint pathways operate to ensure that the next cell cycle event is not initiated until the previous one has been completed. Checkpoint pathways consist of sensors that monitor a particular cellular event, a signaling cascade that initiates the response, and an effector that halts cell cycle progression and activates repair pathways when necessary. Cell cycle events monitored by checkpoint pathways include growth, DNA replication, DNA damage, kinetochore attachment to the mitotic spindle, and positioning of the spindle within the cell. These pathways are responsible for the extraordinarily high fidelity of cell division, ensuring that each daughter cell receives the correct number of accurately replicated chromosomes. They function by controlling the protein kinase activities of the cyclin-CDKs through a variety of mechanisms: regulation of the synthesis and degradation of cyclins, phosphorylation of CDKs at inhibitory sites, regulation of the synthesis and stability of CDK inhibitors (CKIs)

that inactivate cyclin-CDK complexes, and regulation of the APC/C ubiquitin-protein ligase.

## Checkpoint Pathways Establish Dependencies and Prevent Errors in the Cell Cycle

The experiments that led to the idea of surveillance mechanisms or checkpoint pathways establishing dependencies in the cell cycle were simple and beautiful in their interpretation (see Classic Experiment 19-3). Recall that Lee Hartwell and colleagues isolated temperature-sensitive cdc mutants in S. cerevisiae. It was the characterization of one of these cdc mutants that led Hartwell and colleagues to formulate the checkpoint pathway concept: a checkpoint pathway ensures that a cell cycle phase does not commence before the previous one has been completed. In addition to establishing dependencies, the checkpoint pathways ensure that each aspect of chromosome replication and division occurs with accuracy.

Today we know that cells harbor multiple checkpoint pathways and that each is built in the same manner. A sensor detects a defect in a particular cellular process and, in response to this defect, activates a signal transduction pathway. Effectors activated by the signaling pathway initiate repair of the defect and halt cell cycle progression until the defect is corrected. If the defect cannot be repaired, the checkpoint pathway induces apoptosis. In what follows, we discuss the major checkpoint pathways that govern cell cycle progression.

## The Growth Checkpoint Pathway Ensures That Cells Enter the Cell Cycle Only After Sufficient Macromolecule Biosynthesis

Cell proliferation requires that cells multiply through the process of cell division and that individual cells grow through macromolecule biosynthesis. Cell growth and cell division are separate processes, but for cells to maintain a constant size as they multiply, cell growth and cell division must be tightly coordinated. For example, when nutrients are limited, cells reduce their growth rate, and cell division must be down-regulated accordingly. This type of coordination between cell growth and division is especially important in unicellular organisms that experience changes in nutrient availability as part of their natural life cycle. It is therefore not surprising that surveillance mechanisms exist that adjust cell division rate according to growth rate.

In budding yeast, cell growth and division are coordinated in $G_{1}$. In this stage of the cell cycle, the growth and division cycles are linked by the dependence of the activity of $\mathrm{G}_{1} \mathrm{CDKs}$ on cell growth. Which aspect of growth is linked to the cell cycle? Classic experiments using protein synthesis inhibitors indicate that growth rate, and hence cell cycle control by growth, is determined by protein synthesis. How protein synthesis controls $\mathrm{G}_{1} \mathrm{CDK}$ activity is an area of active investigation. The current thinking is that a "sizer" protein, whose abundance is tightly controlled by growth rate, accumulates in a specific compartment of the cell. When the
sizer protein reaches a certain concentration, it serves as the signal that the appropriate cell size for cell cycle entry has been reached. This cell size is known as the critical cell size. In budding yeast, the $\mathrm{G}_{1}$ cyclin Cln 3 has been proposed as the sizer protein. Cln3 is subject to translational control and is highly unstable, which makes levels of this cyclin especially sensitive to protein synthesis rate. Cln3 localizes to the nucleus. It is thought that once the protein reaches a critical concentration in the nucleus, the process of cell cycle entry outlined in Figure 19-12 is set in motion. Importantly, size control is highly plastic: the length of $G_{1}$ and the critical cell size change with nutrient availability.
S. pombe grows as a rod-shaped cell that increases in length as it grows, then divides in the middle during mitosis to produce two daughter cells of equal size (see Figure 19-4). Unlike budding yeast and most metazoan cells, which grow primarily during $G_{1}$, this yeast does most of its growing during the $\mathrm{G}_{2}$ phase of the cell cycle, and its entry into mitosis is carefully regulated in response to cell size. Recall that entry into mitosis is regulated by the protein kinase Wee1, which inhibits CDK1 by phosphorylating tyrosine 15 and threonine 14. When nutrients are limited, Wee1 phosphorylates CDK1; hence cells remain in $\mathrm{G}_{2}$ until they reach the critical size for mitotic entry. Cdr2 appears to be the sizer protein in this yeast. Upon its synthesis, Cdr2 localizes in patches to the cell cortex in the middle of the cell. Cdr2 is an inhibitor of Wee1 (see Figure 19-18). As cells grow, the local concentration of Cdr2 in the middle of the cell rises, and inhibition of Wee1 occurs in the vicinity of Cdr2. Indeed, the centrosome, where CDK activation commences during $\mathrm{G}_{2}$, localizes in the middle of the cell, close to the cell cortex.

Nutrients are usually not limiting in multicellular organisms. Instead, cell growth is controlled by growth factor signaling pathways such as the Ras, AMPK, and TOR pathways (see Chapters 10 and 16). These pathways also appear to be important for coordinating cell growth and division. Mutations in key components of growth factor signaling pathways, such as Myc, cause dramatic changes in cell size in Drosophila. Myc regulates the transcription of many genes important for macromolecule biosynthesis and also, more indirectly, regulates $\mathrm{G}_{1}$ CDKs. Thus this transcription factor appears to integrate cell growth and division. However, the details of this coordination remain to be elucidated.

## The DNA Damage Response System Halts Cell Cycle Progression When DNA Is Compromised

The complete and accurate duplication of the genetic material is essential for cell division. If cells enter mitosis when DNA is incompletely replicated or otherwise damaged, genetic changes occur. In many instances, those changes will lead to cell death, but as we will see in Chapter 24, they can also lead to genetic alterations that result in loss of control over cell growth and division and, eventually, cancer. This risk is underscored by the finding that many proteins involved in sensing DNA damage and its repair are frequently found mutated in human cancers.

The enzymes that replicate DNA are highly accurate, but their exactness is not enough to ensure complete accuracy during DNA synthesis. Furthermore, environmental insults such as x-rays and UV light can cause DNA damage, and this damage must be repaired before a cell's entry into mitosis. Cells have a DNA damage response system in place that senses many different types of DNA damage and responds by activating repair pathways and halting cell cycle progression until the damage has been repaired. Cell cycle arrest can occur in $\mathrm{G}_{1}, \mathrm{~S}$ phase, or $\mathrm{G}_{2}$, depending on whether DNA damage occurred before cell cycle entry or during DNA replication. In multicellular organisms, the strategy to deal with particularly severe DNA damage is different. Rather than attempting to repair the damage, cells undergo programmed cell death or apoptosis, a mechanism that we will discuss in detail in Chapter 21.

DNA damage exists in many different forms and degrees of severity. A break of the DNA helix, known as a doublestrand break, is perhaps the most severe form of damage because such a lesion would almost certainly lead to DNA loss if mitosis ensued in its presence. More subtle defects include single-strand breaks, structural changes in nucleotides, and DNA mismatches. For our discussion here, it is important to note that cells have sensors for all these different types of damage. These sensors scan the genome and, when they detect a lesion, assemble signaling and repair factors at the site of the lesion.

Central to the detection of these different types of lesions is a pair of homologous protein kinases called ATM (for ataxia telangiectasia mutated) and ATR (for ataxia telangiectasia and Rad3-related protein). These proteins were identified and characterized because mutations in their encoding genes cause ataxia telangiectasia and Seckel syndrome, respectively. Both protein kinases are recruited to sites of DNA damage. They then initiate the sequential recruitment of adapter proteins and another set of protein kinases called Chk1 and Chk2. Those kinases then activate repair mechanisms and cause cell cycle arrest or apoptosis in animals (Figure 19-29). ATR and ATM recognize different types of DNA damage. ATM is very specialized in that it responds only to double-strand breaks. ATR is able to recognize more diverse types of DNA damage, such as stalled replication forks, DNA mismatches, damaged nucleotides, and doublestrand breaks. ATR recognizes these diverse types of damage because all of them contain some amount of single-stranded DNA, either as part of the damage itself or because repair enzymes create single-stranded DNA as part of the repair process. The association of ATR with single-stranded DNA is thought to activate its protein kinase activity, leading to the recruitment of adapter proteins whose function is to recruit and help activate the Chk1 kinase. Active Chk1 then induces repair pathways and inhibits cell cycle progression.

Chk1 and Chk2 halt the cell cycle. These protein kinases inhibit Cdc25 by phosphorylating that phosphatase on sites that are distinct from the CDK-activating phosphorylation sites (see Figure 19-29). When the DNA damage
![li4904542480e](li4904542480e.jpg)

FIGURE 19-29 The DNA damage response system. The protein kinases ATM and ATR are activated by damaged DNA. ATR responds to a variety of DNA damage-most likely to the single-stranded DNA that exists either as a result of the damage itself or as a result of repair. ATM is specifically activated by double-strand breaks. Because doublestrand breaks are converted into single-stranded DNA as a part of the repair process, they also, albeit indirectly (as depicted by a dashed line), activate ATR. ATM and ATR, once activated by DNA damage, activate another pair of related protein kinases, Chk1 and Chk2. These kinases then induce the DNA repair machinery and cause cell cycle arrest by inhibiting Cdc25. In metazoan cells, Chk1 and Chk2 also activate the transcription factor p53, which induces cell cycle arrest by inducing transcription of the CKI p21. When the DNA damage is severe, p53 induces apoptosis.
occurs during $\mathrm{G}_{1}, \mathrm{Cdc} 25 \mathrm{~A}$ inhibition results in inhibition of $\mathrm{G}_{1} / \mathrm{S}$ phase CDKs and S phase CDKs (Figure 19-30; see also Figure 19-18). As a result, these kinases cannot initiate DNA replication. When the DNA damage occurs during S phase or in $\mathrm{G}_{2}, \mathrm{Cdc} 25 \mathrm{C}$ inhibition by $\mathrm{Chk} 1 / 2$ results in the inhibition of mitotic CDKs and hence arrest in $\mathrm{G}_{2}$. Active DNA replication also inhibits entry into mitosis. ATR continues to inhibit Cdc25C via Chk1 until all replication forks complete DNA replication and disassemble. This mechanism makes the initiation of mitosis dependent on the completion of chromosome replication. Finally, cells also sense DNA replication stress that results in stalling or slowing of replication forks. Such stress triggers activation of the ATR-Chk1 checkpoint pathway and results in downregulation of S phase CDK activity and prevents the firing of late-replicating origins.

Chk1-mediated inhibition of the Cdc25 family of phosphatases is not the only mechanism whereby DNA damage or incomplete replication inhibits cell cycle progression. As we will see below, DNA damage leads to the activation of p53, a transcription factor that induces the expression of the gene encoding the CDK inhibitor p21. The p21 binds to and inhibits all metazoan cyclin-CDK complexes. As a result, cells are arrested in $\mathrm{G}_{1}$ and $\mathrm{G}_{2}$ (see Figure 19-30).

![li4914542480e](li4914542480e.jpg)

FIGURE 19-30 Overview of DNA damage checkpoint controls in the cell cycle. During $G_{1}$, the p53-p21 pathway inhibits $G_{1}$ CDKs. During ongoing DNA replication and in response to replication stress (slow DNA replication fork movement or DNA replication fork collapse), the ATR-Chk1 protein kinase cascade phosphorylates and inactivates Cdc25C, thereby preventing the activation of mitotic CDKs and inhibiting entry into mitosis. In response to DNA damage, the ATM or ATR
protein kinases (ATM/R) inhibit Cdc25 via the Chk1/2 protein kinases. They also activate p53, which induces production of the CKI p21. During $G_{1}$, the DNA damage checkpoint pathway inhibits Cdc25A, inhibiting $G_{1} / S$ phase CDKs and $S$ phase CDKs, and thereby blocking entry into or passage through S phase. During $G_{2}$, ATM/R and Chk1/2 inhibit Cdc25C. The p53-p21 pathway is also activated. Red symbols indicate pathways that inhibit progression through the cell cycle.

As mentioned above, ATM recognizes only double-strand breaks (see Figure 19-29). This protein kinase is directly recruited to the DNA ends by a complex known as the MRN complex, which binds to the broken ends and holds them together. Activated ATM then phosphorylates and activates Chk2 and recruits repair proteins. These repair proteins initiate homologous recombination, as described in Chapter 5. This process involves the creation of single-stranded overhangs, which in turn recruit and activate ATR and its effectors, further enhancing the DNA damage response. ATM can also recruit an alternative repair pathway by which two double-strand breaks are directly fused with each other in a repair process known as nonhomologous end joining. Like ATR, activated ATM also halts cell cycle progression by Chk2-mediated inhibition of Cdc25, thus preventing activation of CDKs. This inhibition can occur in $G_{1}$ or $G_{2}$.
A key effector of the DNA damage response in metazoan cells is the transcription factor p53 (see Figure 19-29). It is known as a tumor suppressor because its normal function is to limit cell proliferation in the face of DNA damage. This protein is extremely unstable and generally does not accumulate to high enough levels to stimulate transcription under normal conditions. The instability of p53 results from its ubiquitinylation by a ubiquitin-protein ligase called $M d m 2$ and its subsequent proteasomal degradation. Rapid degradation of p53 is inhibited by ATM and ATR, which phosphorylate p53 at a site that interferes with Mdm2 binding. This and other modifications of p53 in response to DNA damage greatly increase its ability to activate transcription of specific genes that help the cell cope with DNA damage. One of these genes encodes the CKI p21 (see Figure 19-29).

Under some circumstances, such as when DNA damage is extensive, p53 also activates expression of genes that lead to apoptosis, the process of programmed cell death that normally occurs in specific cells during the development of multicellular animals. In metazoans, the p53 response evolved to induce apoptosis (see Chapter 21) in the face of extensive DNA damage, presumably to prevent the accumulation of multiple mutations that might convert a normal cell into a cancer cell. The dual role of p53 in both cell cycle arrest and the induction of apoptosis may account for the observation that nearly all cancer cells have mutations in both alleles of the $p 53$ gene or in the pathways that stabilize p53 in response to DNA damage (see Chapter 24). The consequences of mutations in $p 53, A T M$, and $C h k 2$ provide dramatic examples of the significance of cell cycle checkpoint pathways to the health of a multicellular organism.

## The Spindle Assembly Checkpoint Pathway Prevents Chromosome Segregation Until Chromosomes Are Accurately Attached to the Mitotic Spindle

The spindle assembly checkpoint pathway prevents entry into anaphase until every kinetochore of every chromatid is properly attached to spindle microtubules. If even a single kinetochore is unattached or not under tension, anaphase is inhibited, as such a defect would almost certainly lead to chromosome loss if mitosis ensued in its presence. To achieve this, cells harbor a surveillance mechanism that prevents anaphase entry in the presence of tensionless or unattached kinetochores. The spindle assembly checkpoint pathway recognizes unattached kinetochores. However, as
we saw in Section 19.5, kinetochores of sister chromatids often attach to microtubules emanating from the same pole (syntelic attachment), or a single kinetochore attaches to microtubules that originate from two different poles (merotelic attachment). How are such faulty attachments recognized? Microtubule-kinetochore interactions that are under insufficient tension are destabilized by Aurora B phosphorylation of the microtubule-binding component of the kinetochore, the Ndc80 complex. This leads to the generation of unattached kinetochores, which are then recognized by the spindle assembly checkpoint pathway. In this manner, Aurora B and the spindle assembly checkpoint pathway collaborate during every cell cycle to accurately attach every single pair of sister chromatids to the mitotic spindle in the correct, bioriented manner.

Components of the spindle assembly checkpoint pathway bind to unoccupied microtubule binding sites at kinetochores and create an anaphase inhibitory signal that ultimately inhibits APC/C ${ }^{\mathrm{Cdc} 20}$. Recall that APC/C ${ }^{\mathrm{Cdc} 20}$ mediated ubiquitinylation of securin and its subsequent degradation are required for the activation of separase and entry into anaphase (see Figure 19-25). When a kinetochore is not attached to microtubules, the outer kinetochore component Knl1 is phosphorylated by the spindle assembly checkpoint pathway kinase Mps1 (Figure 19-31). This phosphorylation, in turn, recruits other checkpoint pathway components to the unattached kinetochore. Critical for the shutting down of APC/C ${ }^{\mathrm{Cdc} 20}$ is the recruitment and activation of the Mad1Mad2 complex by unattached kinetochores. Importantly, the activated Mad1-Mad2 complex can convert inactive Mad2 molecules in the cytoplasm to an active conformation that is capable of binding to and inhibiting APC/C ${ }^{\mathrm{Cdc} 20}$. Mad2 bound to APC/C ${ }^{\mathrm{Cdc} 20}$ recruits additional checkpoint

FIGURE 19-31 The spindle assembly checkpoint pathway. The spindle assembly checkpoint pathway is active until every single kinetochore has attached properly to spindle microtubules. When a kinetochore is unattached, the outer kinetochore component Knl1 is phosphorylated by the checkpoint kinase Mps1 step. Phosphorylated Knl1 then binds the checkpoint kinase Bub1-Bub3 and the checkpoint protein Mad3. These three proteins, in turn, recruit the Mad1-Mad2 complex to the kinetochore. The Mad1-Mad2 complex bound to a kinetochore is in the active form (shown as Mad2-A; step ) and has the ability to convert inactive Mad2 (Mad2-I) in the cytoplasm into active Mad2 that is able to bind APC/C ${ }^{\mathrm{Cdc} 20}$ and inhibit it. Complete inhibition of APC/C ${ }^{\mathrm{Cdc} 20}$ requires the recruitment of the checkpoint factors Bub1-Bub3 and Mad3 into the complex. Together, these proteins form the mitotic checkpoint complex (MCC) that prevents APC/C ${ }^{\mathrm{Cdc} 20}$ from recognizing and ubiquitinylating its substrates. Silencing of the spindle assembly checkpoint pathway occurs once all kinetochores have attached to microtubules in a tension-generating manner. Protein phosphatase 1 then dephosphorylates Knl1, thereby eliminating checkpoint protein binding sites at kinetochores. In addition, p31 $1^{\text {catset }}$ disassembles the MCC. See E. Foley and T. Kapoor, 2013, Nature Rev. Mol. Cell Biol. 14:25.
![li4924542480e](li4924542480e.jpg)

factors to the complex to form the mitotic checkpoint complex (MCC). The MCC then prevents APC/C ${ }^{\text {Cdc20 }}$ from recognizing and ubiquitinylating substrates. This elegant model for the spindle assembly checkpoint pathway can account for the ability of a single unattached kinetochore to inhibit all the cellular Cdc20 until the kinetochore becomes properly associated with spindle microtubules.

Once all chromosomes have attached to the mitotic spindle in the correct amphitelic manner, the spindle assembly checkpoint pathway must be silenced to allow APC/C ${ }^{\text {Cdc20 }}$ to degrade securin and initiate anaphase. Silencing of the spindle assembly checkpoint pathway occurs through multiple mechanisms. Protein phosphatase 1 dephosphorylates Knl1, thereby eliminating checkpoint protein binding sites at kinetochores. In addition, a protein known as p31 ${ }^{\text {comet }}$ disassembles the MCC, thereby activating APC/C ${ }^{\text {Cdc20 }}$ (see Figure 19-31).

The spindle assembly checkpoint pathway is essential for viability in mice, highlighting the importance of this quality-control pathway during every cell division. If anaphase is initiated before both kinetochores of a replicated chromosome become attached to microtubules from opposite spindle poles, chromosomes are mis-segregated in a process called nondisjunction. The resulting condition, in which cells have either lost or gained whole chromosomes, is called aneuploidy. Aneuploidy has profound effects on human health and fitness. It leads to the misregulation of genes and can contribute to the development of cancer. When nondisjunction occurs during the meiotic division that generates a human egg or sperm, trisomy (gain of a chromosome) or monosomy (loss of a chromosome) can be the result. As we will see in Section 19.8, meiosis is especially prone to nondisjunction, which can lead to spontaneous abortions or Down syndrome.

## The Spindle Position Checkpoint Pathway Ensures That the Nucleus Is Accurately Partitioned Between Two Daughter Cells

The coordination of the site of nuclear division with that of cytokinesis is essential for the production of two identical daughter cells. If cytokinesis occurred in such a way that each daughter cell failed to receive a complete genetic complement, chromosome loss or gain would ensue. A surveillance mechanism that prevents cytokinesis when the mitotic spindle is not correctly positioned in the cell has been described in many model systems. This surveillance mechanism, known as the spindle position checkpoint pathway, is best understood in budding yeast. In this organism, the site of bud formation, and therefore the site of cytokinesis, is determined during $G_{1}$. Thus the axis of division is defined prior to mitosis, and the mitotic spindle must be aligned along this parent-bud axis during every cell division (Figure 19-32, step ). When this process fails, the spindle position checkpoint prevents mitotic CDK inactivation, giving the cell an opportunity to reposition the spindle prior to spindle
![li4934542480e](li4934542480e.jpg)

FIGURE 19-32 The spindle position checkpoint pathway in budding yeast. Cdc14 phosphatase activity is required for exit from mitosis. Top: In S. cerevisiae, during interphase and early mitosis, Cdc14 (red dots) is sequestered and inactivated in the nucleolus. Inactive Tem1-GDP (purple) associates with the spindle pole body (SPB) nearest to the bud as soon as the mitotic spindle forms. If chromosome segregation occurs properly step $\mathbf{B}$, extension of the spindle microtubules inserts the daughter SPB into the bud, causing Tem1 to be activated by an unknown mechanism. Tem1-GTP activates a protein kinase cascade, which then promotes the release of active Cdc14 from the nucleolus and exit from mitosis. If the spindle apparatus fails to place the daughter SPB in the bud step $\mathbf{B}$, Kin4 (cyan), an inhibitor of Tem1, is recruited from the parent cell cortex to the parent-cell-located SPB and maintains Tem1 in the GDP-bound form, and mitotic exit does not occur. Lte1 (orange) is an inhibitor of Kin4 and is localized to the bud. Lte1 prevents Kin4 that leaks into the bud from inhibiting Tem1. If the checkpoint fails step $\mathbf{B}$, cells with mispositioned spindles inappropriately exit mitosis and produce anucleate and multinucleate cells.

disassembly and cytokinesis (Figure 19-32, step ■). If the spindle position checkpoint fails, cells that misposition their spindles give rise to mitotic products with too many or too few nuclei (Figure 19-32, step ■).

Recall that in budding yeast, a pool of mitotic cyclins is spared from degradation by $\mathrm{APC} / \mathrm{C}^{\mathrm{Cdc} 20}$ to facilitate the sometimes difficult process of aligning the mitotic spindle in such a way that half the nucleus squeezes through the tiny bud neck during anaphase spindle elongation. Recall further that inactivation of this protected pool of mitotic cyclin-CDK complexes is triggered by the protein phosphatase Cdc14, which in turn is activated by a signal transduction pathway known as the mitotic exit network (see Figure 19-26). The mitotic exit network is controlled by a small (monomeric) GTPase called Tem1. This member of the GTPase superfamily of switch proteins controls the activity of a protein kinase cascade in a manner similar to the way Ras controls MAP kinase pathways (see Chapter 16). Tem1 associates with spindle pole bodies as soon as they form. An inhibitor of the GTPase, called Kin4, localizes to the parent cell but is absent from the bud (see Figure 19-32). An inhibitor of Kin4, called Lte1, localizes to the bud but is absent from the parent cell; it inhibits any residual Kin4 that leaks into the bud. When spindle microtubule elongation at the end of anaphase has correctly positioned segregating daughter chromosomes in
the bud, Tem1 is released from inhibition by Kin4. As a consequence, Tem1 is converted into its active GTP-bound state, activating the protein kinase signaling cascade. The terminal kinase in the cascade then phosphorylates the nucleolar anchor that binds and inhibits Cdc14, releasing Cdc14 phosphatase into the cytoplasm and nucleoplasm in both the bud and the parent cell (see Figure 19-32, step ■). Once active Cdc14 is available, mitotic CDKs are inactivated and cells exit mitosis. When the spindle fails to be positioned correctly, the Tem1-bearing spindle pole body fails to enter the bud, the mitotic exit network cannot be activated, and the cell is arrested in anaphase. Thus spatial restriction of the inhibitors and activators of a signal transduction pathway allows the cell to sense a spatial cue, spindle position, and translate it into regulation of a signal transduction pathway.

The mitotic exit network that localizes to spindle pole bodies and whose activity is regulated by spindle position belongs to the family of signaling pathways known as the Hippo pathway in metazoans and the septation initiation network in fission yeast (Figure 19-33). This pathway consists of a highly conserved core kinase signaling network, but its signaling input and output have diverged during evolution. The conserved core of the Hippo signaling pathways consists of the Hippo protein kinase, which activates the Lats-Mob1 kinase. The kinases are organized by a scaffolding molecule
![li4944542480e](li4944542480e.jpg)

FIGURE 19-33 The Hippo signaling pathway family. The core kinase signaling module of the Hippo pathway is conserved across species (homologous proteins are shown in the same color), but input signals as well as pathway effectors have diverged during evolution. In budding and fission yeasts, cell cycle signals such as anaphase onset and nuclear position regulate exit from mitosis and cytokinesis
through effector molecules such as the protein phosphatase Cdc14 (Clp1 in fission yeast). In metazoans, tissue organization cues, such as pathways that establish cell polarity, mechanical cues, and signals from G protein-coupled receptors, activate the Hippo pathway. The Hippo pathway then inhibits the transcription factor TAZ to prevent cell proliferation and tissue growth.

that, at least in budding and fission yeasts, targets the kinase cascade to centrosomes. As in many kinase signaling cascades, a GTPase controls the activity of the mitotic exit network and the septation initiation network. Whether the Hippo pathway is controlled by a GTPase is not yet known.

The Hippo pathway is a great example of how signaling pathways have been repurposed during evolution. In budding and fission yeasts, the pathway regulates CDK activity and cytokinesis in response to anaphase and nuclear position signals. In metazoans, the Hippo pathway restrains cell proliferation during $G_{1}$. Inactivation of the Hippo pathway leads to tissue overgrowth because in the absence of Hippo pathway function, the transcriptional activator TAZ (known as YAP in the mouse) constitutively promotes the expression of growth- and proliferation-promoting genes. The signals controlling Hippo pathway function appear diverse. Apicalbasal cell polarity via cell-junction components, G proteincoupled receptors, and mechanical forces activate the Hippo pathway and hence cause $G_{1}$ arrest. Apical-basal polarity is characteristic of differentiated epithelial sheets. The exact mechanism whereby these polarity cues regulate the Hippo pathway is an area of active investigation, and the actin cytoskeleton appears to be a central node in this control. Cell junctions, mechanical forces, and G protein-coupled receptors all control the actin cytoskeleton and their regulators, such as the rho GTPases.

Given its central role in proliferation control, it is not surprising that the Hippo pathway plays a key role in stem cell maintenance and tissue regeneration. In fact, the pathway was first discovered because mutations in the Hippo pathway lead to tissue overgrowth and increased organ size in Drosophila. Its central function in restraining growth also means that the pathway is critically important for preventing tumor development. Mice harboring mutations in the Hippo pathway are cancer prone, and elevated levels of TAZ are frequently observed in human cancers.

This concludes our discussion of the surveillance mechanisms that ensure that cell cycle progression moves forward and occurs without error. As we will see in Chapter 24, cancer is a disease of failed cell cycle surveillance. Mutations in the pathways that ensure accurate chromosome replication, segregation, and nuclear partitioning are a major cause of cancerous transformation.

## KEY CONCEPTS OF SECTION 19.7

## Surveillance Mechanisms in Cell Cycle Regulation

- Surveillance mechanisms known as checkpoint pathways establish dependencies among cell cycle events and ensure that the next cell cycle event does not occur prior to the completion of a preceding event.
- Checkpoint pathways consist of sensors that monitor a particular cellular event, a signaling pathway, and an
effector that halts cell cycle progression and activates repair pathways when necessary.
- Growth and cell division are integrated during $G_{1}$ in most organisms. Reduced macromolecule biosynthesis delays cell cycle entry.
- Cells are able to detect and respond to a wide variety of DNA damage, and their response differs depending on the cell cycle stage that cells are in.
- In response to DNA damage, two related protein kinases, ATM and ATR, are recruited to the site of the damage, where they activate signaling pathways that lead to cell cycle arrest, repair, and under some circumstances, apoptosis.
- The spindle assembly checkpoint pathway, which prevents premature initiation of anaphase, utilizes Mad2 and other proteins to regulate APC/C ${ }^{\mathrm{cdc} 20}$, which targets securin and mitotic cyclins for ubiquitinylation.
- The spindle position checkpoint pathway prevents mitotic CDK inactivation when the spindle is mispositioned. In this pathway, localized activators and inhibitors and a sensor that shuttles between them allow cells to sense spindle position.
- The Hippo family of signal transduction pathways has been repurposed during evolution. Its function in coordinating exit from mitosis and cytokinesis with chromosome segregation in fungi has been replaced with coordination of tissue growth with tissue organization in metazoans.


### 19.8 Meiosis: A Special Type of Cell Division

In nearly all diploid eukaryotes, meiosis generates haploid germ cells (eggs and sperm), which can then fuse with a germ cell from another individual to generate a diploid zygote that develops into a new individual. Meiosis is a fundamental aspect of the biology and evolution of all eukaryotes because it results in the reassortment of the chromosome sets received from an individual's two parents. Chromosome reassortment and homologous recombination between parent DNA molecules during meiosis guarantee that each haploid germ cell generated will receive a unique combination of alleles that is distinct from each parent as well as from every other haploid germ cell formed.

The mechanisms of meiosis are analogous to those of mitosis. However, several key features of meiosis allow this process to generate genetically diverse haploid cells (see Figure 6-3). In the mitotic cell cycle, each S phase is followed by chromosome segregation and cell division. In contrast, during meiotic cell division, one round of DNA replication is followed by two consecutive chromosome segregation phases. This process leads to the formation of haploid, rather than diploid, daughter cells. During the two

divisions, maternal and paternal chromosomes are shuffled and divided so that the daughter cells are different in genetic makeup from the parent cell. In this section, we discuss the similarities between mitosis and meiosis as well as the meiosis-specific mechanisms that transform the canonical mitotic cell cycle machinery so that it brings about the unusual cell division that leads to the formation of haploid daughter cells.

## Extracellular and Intracellular Cues Regulate Germ Cell Formation

The signals triggering entry into meiosis in metazoans are a very active area of research, and much is still unknown. However, the same basic principles govern the decision to enter the meiotic program in all organisms in which this transition has been studied. Extracellular signals induce a transcriptional program that produces meiosis-specific cell cycle factors that bring about the unusual meiotic cell divisions. This modification of the cell cycle goes hand in hand with a developmental program that induces the features characteristic of gametes, such as the development of a flagellum in sperm or the production of a stress-resistant cell wall during spore formation in fungi. At least one of the extracellular signals inducing entry into meiosis in mammals is retinoic acid, a signaling molecule that, by binding to the transcription factor retinoic acid receptor (RAR), functions in many different developmental processes (see Figure 9-43). The cellular targets of this hormone, and how it functions to specify the meiotic fate, remain to be discovered.

The molecular mechanisms underlying the decision to enter meiosis are well understood in S. cerevisiae. The decision to enter the meiotic divisions is made in $\mathrm{G}_{1}$. Depletion of nitrogen and carbon sources induces diploid cells to undergo meiosis instead of mitosis, yielding haploid spores (see Figure 1-23). During the meiotic divisions, budding is repressed. Pre-meiotic $S$ phase and the two meiotic divisions thus occur within the confines of the parent cell. Spore walls are then produced around the four meiotic products. Recall that budding and the initiation of DNA replication are induced by $\mathrm{G}_{1} / \mathrm{S}$ phase CDKs. Their expression needs to be inhibited to prevent budding. Nutritional starvation represses expression of $\mathrm{G}_{1} / \mathrm{S}$ phase cyclins, thereby inhibiting budding. However, DNA replication also relies on $\mathrm{G}_{1} / \mathrm{S}$ phase CDKs. How can pre-meiotic DNA replication occur in the absence of $\mathrm{G}_{1} / \mathrm{S}$ phase CDKs? The sporulation-specific protein kinase Ime 2 takes over the role of $\mathrm{G}_{1} / \mathrm{S}$ phase CDKs in promoting DNA replication. Ime2 promotes (1) phosphorylation of the APC/C specificity factor Cdh1, inactivating it so that S phase and M phase cyclins can accumulate (see Figure 19-13); (2) phosphorylation of transcription factors to induce expression of genes whose products are required for $S$ phase, including DNA polymerases and S phase cyclins (see Figure 19-14); and (3) phosphorylation of the S phase CDK inhibitor Sic1, leading to release of active S phase CDKs and the onset of pre-meiotic DNA replication.

## Several Key Features Distinguish Meiosis from Mitosis

The meiotic divisions differ in several fundamental aspects from the mitotic divisions. These differences are summarized in Figure 19-34. During the meiotic cell division, a single round of DNA replication is followed by two cycles of cell division, termed meiosis I and meiosis II (Figure 19-35). Meiosis II resembles mitosis in that sister chromatids are segregated. However, meiosis I is very different. During this division, homologous chromosomes-the chromosome inherited from your mother and the same chromosome inherited from your father-are segregated. This unusual chromosome segregation requires three meiosis-specific modifications to the chromosome segregation machinery. In what follows, we discuss these modifications and explain why they are needed.

The tension-based sensing mechanism responsible for accurately attaching chromosomes to the spindle during mitosis is also responsible for segregating chromosomes during meiosis I. Thus homologous chromosomes must be linked so that this tension-based mechanism can function. Homologous recombination between homologous chromosomes creates these linkages (see Figure 19-34). The molecular mechanisms of homologous recombination are discussed in detail in Chapter 4. Here we restrict our discussion to the importance of homologous recombination to successful meiotic divisions.

In $\mathrm{G}_{2}$ and prophase of meiosis I, the two replicated chromatids of each chromosome are linked together by cohesin complexes along the full length of the chromosome arms, just as they are following DNA replication in a mitotic cell cycle. In prophase of meiosis I, homologous chromosomes (i.e., the maternal and paternal chromosome 1, the maternal and paternal chromosome 2, etc.) pair with each other and undergo homologous recombination. Significantly, at least one recombination event occurs between a maternal and a paternal chromosome. The crossing over of chromatids produced by recombination can be observed microscopically in the first meiotic prophase and metaphase as structures called chiasmata (singular, chiasma). In contrast, no pairing between homologous chromosomes occurs during mitosis, and recombination between nonsister chromatids is rare. Concomitant with homologous recombination, homologous chromosomes associate with each other in a process known as synapsis. In most organisms, this synapsis is mediated by a proteinaceous complex known as the synaptonemal complex (SC). Homologous chromosomes linked through chiasmata are called bivalents (see Figure 19-34). The chiasmata and the cohesin molecules distal to them now provide the resistance to the pulling force exerted by microtubules on the metaphase I spindle (see Figure 19-35).

The recombination between homologous chromosomes that occurs in prophase of meiosis I has at least two functional consequences: First, it connects homologous chromosomes during meiosis I metaphase. Second, it contributes to genetic diversity among individuals of a species by ensuring new

![li4954542480e](li4954542480e.jpg)

FIGURE 19-34 Comparison of the main features of mitosis and meiosis.
combinations of alleles in different individuals. (Note, however, that genetic diversity primarily arises from the independent reassortment of maternal and paternal homologs during the meiotic divisions.) The homologs, now connected through at least one chiasma, must align on the spindle in metaphase of meiosis I so that maternal and paternal chromosomes are segregated away from each other during anaphase of meiosis I. This requires that the kinetochores of sister chromatids attach to microtubules emanating from the same spindle pole rather than from opposite spindle poles, as in mitosis (Figure 19-36). Sister chromatids attached in this way are said to be co-oriented. However, the kinetochores of the maternal and paternal chromosomes of each bivalent attach to spindle microtubules from opposite spindle poles; they are bi-oriented.

Finally, to facilitate two consecutive chromosome segregation phases, cohesins must be lost from chromosomes in a stepwise manner. Recall that during mitosis, all cohesins are lost by the onset of anaphase (Figure 19-37a). In contrast, during meiosis, cohesins are lost from chromosome arms by the end of meiosis I, but a pool of cohesins around kinetochores is protected from removal (Figure 19-37b). This pool of cohesins persists throughout meiosis I, but is removed at the onset of anaphase II. As we will see next, loss of cohesins from chromosome arms is required for homologous chromosomes to segregate away from each other during meiosis I.

The mechanisms that remove cohesins during meiosis are the same as during mitosis. Securin degradation releases separase, which then cleaves the cohesins holding the chromosome

FIGURE 19-35 Meiosis. Pre-meiotic cells have two copies of each chromosome (2n), one derived from the paternal parent and one from the maternal parent. For simplicity, the paternal and maternal homologs of only one chromosome are diagrammed. Step All chromosomes are replicated during $S$ phase before the first meiotic division, giving a $4 n$ chromosomal complement. Cohesin complexes (not shown) link the sister chromatids composing each replicated chromosome along their full lengths. Step $\mathbf{E}$ As chromosomes condense during the first meiotic prophase, the replicated homologs pair and undergo homologous recombination, leading to at least one crossover event. At metaphase I, shown here, both chromatids of one chromosome associate with microtubules emanating from one spindle pole, but each member of a homologous chromosome pair associates with microtubules emanating from opposite poles. Step During anaphase of meiosis I, the homologous chromosomes, each consisting of two chromatids, are pulled to opposite spindle poles. Step $\mathbf{D}$ Cytokinesis yields two daughter cells (now $2 n$ ), which enter meiosis II without undergoing DNA replication. At metaphase of meiosis II, shown here, the sister chromatids associate with spindle microtubules from opposite spindle poles, as they do in mitosis. Steps $\mathbf{E}$ and Segregation of sister chromatids to opposite spindle poles during anaphase of meiosis II, followed by cytokinesis, generates haploid gametes ( $1 n$ ) containing one copy of each chromosome. Micrographs on the left show meiotic metaphase I and metaphase II in developing gametes from Lilium (lily) ovules. Chromosomes are aligned at the metaphase plate. [Photos courtesy of Ed Reschke/Peter Arnold, Inc./Photolibrary/Getty Images.]
![li4964542480e](li4964542480e.jpg)

![li4974542480e](li4974542480e.jpg)

FIGURE 19-36 Chiasmata and cohesins distal to them link homologous chromosomes in meiosis I metaphase. Connections between chromosomes during meiosis I are most easily visualized in organisms with acrocentric centromeres, such as the grasshopper. The kinetochores at the centromeres of sister chromatids attach to spindle microtubules emanating from the same spindle pole, with the kinetochores of the maternal (red) and paternal (blue) chromosomes attaching to spindle microtubules from opposite spindle poles. The maternal and paternal chromosomes are attached to each other by chiasmata, which are formed by recombination between them, and by the cohesion between sister chromatid arms that persists until metaphase I. Note that elimination of cohesion between sister chromatid arms is all that is required for the homologous chromosomes to separate at anaphase. See L. V. Paliulis and R. B. Nicklas, 2000, J. Cell Biol. 150:1223.
arms together, but leaves the cohesins around centromeres intact. This allows the recombined maternal and paternal chromosomes to separate, but each pair of chromatids remains associated at the centromere. During metaphase II, sister chromatids align on the metaphase II spindle, and separase is activated yet again, cleaving the residual cohesin around centromeres, facilitating anaphase II (see Figure 19-37b).

## Recombination and a Meiosis-Specific Cohesin Subunit Are Necessary for the Specialized Chromosome Segregation in Meiosis I

As we have seen, in metaphase of meiosis I, both sister chromatids in one (replicated) chromosome associate with microtubules emanating from the same spindle pole, rather than from opposite poles as they do in mitosis (see Figure 19-36). Two physical links between homologous chromosomes resist the pulling force of the spindle until anaphase: (1) the chiasmata that result from crossing over between chromatids, and (b) cohesins distal to the crossover point (see Figure 19-37b, top). Evidence for the linking function of recombination during meiosis comes from the observation that when recombination is blocked by mutations in proteins essential
![li4984542480e](li4984542480e.jpg)

FIGURE 19-37 Cohesin function during mitosis and meiosis.
(a) During mitosis, sister chromatids generated by DNA replication in S phase are initially linked by cohesin complexes along the length of the chromatids. During chromosome condensation, cohesin complexes (yellow) become restricted to the region of the centromere at metaphase. Mei-S332/Shugoshin (purple) recruits PP2A to centromeres, where they antagonize Polo kinase and Aurora B, preventing the dissociation of cohesins from centromeric regions. Dissociation of Mei-S332/Shugoshin from centromeres and activation of separase leads to removal of cohesins at the centromere. Sister chromatids now separate, marking the onset of anaphase. (b) In prophase of meiosis I, maternal and paternal chromatids establish linkages between each other by homologous recombination. By metaphase I, the chromatids of each replicated chromosome are cross-linked by cohesin complexes along their full length. Rec8, a meiosis-specific homolog of Scc1, is cleaved along chromosome arms but not around the centromere, allowing homologous chromosome pairs to segregate to daughter cells. Centromeric Rec8 is protected from cleavage by PP2A, recruited to centromeric regions by the PP2A regulator Mei-S332/Shugoshin (shown in purple). By metaphase II, the Mei-S332/Shugoshin-PP2A complex dissociates from chromosomes. Cohesins can now be cleaved during meiosis II, allowing sister chromatids to segregate. See F. Uhlmann, 2001, Curr. Opin. Cell Biol. 13:754.
for the process, chromosomes segregate randomly during meiosis I; that is, homologous chromosomes do not necessarily segregate to opposite spindle poles.

At the onset of meiotic anaphase I, cohesins between chromosome arms are cleaved by separase. This cleavage is required for homologous chromosomes to segregate. If cohesins were not lost from chromosome arms, the recombined

sister chromatids would be torn apart during anaphase I. The maintenance of centromeric cohesion during meiosis I is necessary for the proper segregation of sister chromatids during meiosis II.

Studies in many organisms have shown that a specialized cohesin subunit, Rec8, is necessary for the stepwise loss of cohesins from chromosomes during meiosis. Expressed only during meiosis, Rec8 is homologous to Scc1, the cohesin subunit that closes the cohesin ring in the cohesin complex of mitotic cells (see Figure 19-25). Immunolocalization experiments revealed that during early anaphase of meiosis I, Rec8 is lost from chromosome arms but is retained at centromeres. However, during early anaphase of meiosis II, centromeric Rec8 is cleaved by separase, so the sister chromatids can segregate, as they do in mitosis (see Figure 19-37b, bottom). Consequently, understanding the regulation of Rec8-cohesin complex cleavage is central to understanding chromosome segregation in meiosis I.

The mechanism that protects Rec8 from cleavage at centromeres during meiosis I is similar to the mechanism that protects Scc1 at centromeres during mitosis. Recall that during mitotic prophase, protein kinases, chief among them Polo kinase, phosphorylate cohesins in the chromatid arms, causing them to dissociate and eliminating cohesion on chromatid arms by metaphase. However, cohesion at the centromeres is maintained because a specific isoform of protein phosphatase 2A (PP2A) is localized to centromeric chromatin by members of a family of proteins known as Mei-S332/ Shugoshin. PP2A keeps cohesin in a hypophosphorylated state that does not dissociate from chromatin until meiosis II (see Figure 19-37a). During metaphase II, Mei-S332/Shugoshin dissociates from chromosomes. In addition, when the last kinetochore is properly associated with spindle microtubules, APC/C ${ }^{\mathrm{Cdc} 20}$ is derepressed, causing ubiquitinylation of securin. The inactivation of securin releases separase activity, which cleaves Scc1 whether it is phosphorylated or not, eliminating cohesion at the centromere and allowing chromatid separation in anaphase (see Figure 19-37a).

Cohesin removal differs for meiosis I because when Rec8 replaces Scc1 in the cohesin complex, the complex does not dissociate in prophase when it is phosphorylated. The meiotic cohesin complex can be removed from chromatin only by the action of separase. Rec8 also differs from Scc1 in that it must be phosphorylated by several protein kinases to be cleaved by separase. During meiosis I, the centromerespecific isoform of PP2A targeted to centromeric chromatin by Mei-S332/Shugoshin prevents this phosphorylation. The PP2A targeting factor and PP2A then dissociate from chromosomes by metaphase II, allowing separase to cleave Rec8.

8Meiosis I is much more error-prone than mitosis. It is estimated that 10 percent of all conceptions in humans are aneuploid. These aneuploidies are largely caused by chromosome mis-segregation in meiosis I, also known as chromosome nondisjunction. When recombination fails to take place between homologous chromosomes, or when the chiasmata are too close to the end of the chromosomes,
either no tension or too little tension is exerted on kinetochore-microtubule attachments, and they are destabilized, leading to nondisjunction. As a result, eggs or sperm receive too few or too many chromosomes. All monosomies (lack of a chromosome) and most trisomies (gain of a chromosome) result in embryonic lethality or death shortly after birth. Only trisomy 21, also known as Down syndrome, results in viable offspring, but it causes developmental abnormalities and intellectual disabilities.

Nondisjunction in meiosis I increases dramatically with maternal age. Whereas fewer than 0.1 percent of babies born to women under the age of 30 have Down syndrome, this rate increases to 3.5 percent at age 45 . This increase in chromosome nondisjunction with maternal age is not specific to chromosome 21. The incidence of trisomy in clinically recognized pregnancies is less than 5 percent in women under 30 , but as high as 35 percent in women over the age of 42 (Figure 19-38)! The reason for this dramatic increase in nondisjunction with maternal age lies in the biology of female meiosis in vertebrates. In all vertebrates, pre-meiotic DNA replication and recombination occur in the female embryo. The oocytes then become arrested in $\mathrm{G}_{2}$ of meiosis I until the female reaches sexual maturity, which in humans is between the ages of 12 and 16. It is at this time that the first oocyte enters meiosis and progresses to the second meiotic metaphase, where it arrests and awaits fertilization (see Figure 19-5). The oocytes that enter the meiotic divisions in a 40-year-old woman have been arrested in $\mathrm{G}_{2}$ for 40 years, during which time chiasmata close to the ends of chromosomes can slip off, or cohesins that hold the homologous chromosomes together during this very long $\mathrm{G}_{2}$ phase can deteriorate, causing the homologous chromosomes to dissociate from each other. Both events can cause the homologous chromosomes to mis-segregate, leading to the formation of aneuploid eggs.
![li4994542480e](li4994542480e.jpg)

FIGURE 19-38 Fetal aneuploidy increases with maternal age. The percentage of trisomic embryos identified among clinically recognized pregnancies is shown as a function of maternal age. [Data from T. Hassold and P. Hunt, 2001, Nature Rev. Genet. 2:280.]

## Co-orienting Sister Kinetochores Is Critical for Meiosis I Chromosome Segregation

As discussed earlier, at metaphase in mitosis and meiosis II, sister kinetochores attach to spindle microtubules emanating from opposite spindle poles; the kinetochores are said to be bi-oriented. This is essential for segregation of sister chromatids to different daughter cells. In contrast, at meiosis I metaphase, sister kinetochores attach to spindle microtubules emanating from the same spindle pole; these sister kinetochores are said to be co-oriented (see Figure 19-36). Obviously, attachment of sister kinetochores to the proper microtubules in meiosis I and II is critical for correct meiotic segregation of chromosomes.

Proteins required for meiosis I sister kinetochore coorientation were first identified in S. cerevisiae. In this organism, a single microtubule attaches to each kinetochore. We now know that a protein complex known as the monopolin complex associates with sister kinetochores during meiosis I and fuses them into a single kinetochore unit, to which one microtubule attaches. In all other organisms, kinetochores attach to multiple microtubules. In these organisms, Rec8containing cohesins are essential for sister kinetochore coorientation. These meiosis-specific cohesins impose a rigid kinetochore structure, restricting the movement of sister kinetochores and thereby favoring their attachment to microtubules from the same spindle pole.

Correct attachment of meiosis I chromosomes is mediated by a tension-based mechanism, as it is during mitosis and meiosis II. During meiotic metaphase I, kinetochoreassociated microtubules are under tension (even though the co-oriented kinetochores of sister chromatids attach to microtubules coming from the same spindle pole) because chiasmata generated by recombination between homologous chromosomes and the cohesins distal to the chiasmata prevent them from being pulled to the poles (see Figure 19-36). Since kinetochore-microtubule attachments are unstable in the absence of tension (due to Aurora B-mediated phosphorylation), kinetochores that attach to microtubules from the wrong spindle release those microtubules, which enables them to bind microtubules again until attachments are made that generate tension. As in mitosis, once tension is generated, microtubule attachment to the kinetochores is stabilized.

## DNA Replication Is Inhibited Between the Two Meiotic Divisions

The mechanism by which DNA replication is suppressed between meiosis I and II is currently an active area of investigation, but it is thought that a change in the regulation of CDK activity is at least in part responsible for this suppression. The same S phase CDKs that promote DNA replication prior to mitosis are needed for pre-meiotic DNA replication. And the same mitotic CDKs that promote mitosis also promote the meiotic divisions, except that in this case we call mitotic CDKs meiotic CDKs since they promote the meiotic divisions and not mitosis.

So how is DNA replication prevented between the two meiotic divisions? Following anaphase of meiosis I, meiotic CDK activity does not fall as low as it does following mitotic anaphase. This partial drop in CDK activity is thought to be sufficient to promote the disassembly of the meiosis I spindle, but insufficient to promote MCM helicase loading (recall that a state of very low or no CDK activity is needed to load MCM helicases). During prophase of meiosis II, meiotic CDK activity rises again, and the meiosis II spindle forms. After all sister kinetochores have attached to microtubules from opposite spindle poles, separase is activated and cells proceed through meiosis II anaphase, telophase, and cytokinesis to generate haploid germ cells.

## KEY CONCEPTS OF SECTION 19.8

## Meiosis: A Special Type of Cell Division

- Meiosis is a specialized type of cell division in which meiosis-specific gene products modulate the mitotic cell division program (see Figure 19-34).
- The meiotic division comprises one cycle of chromosome replication followed by two cycles of cell division to produce haploid germ cells from a diploid pre-meiotic cell. During meiosis I, homologous chromosomes are segregated; during meiosis II, sister chromatids separate.
- Specialized environmental conditions induce a developmental program that leads to the meiotic divisions.
- During prophase of meiosis I, homologous chromosomes undergo recombination. At least one recombination event occurs between the chromatids of each pair of homologous chromosomes.
- Chiasmata and cohesins distal to them are responsible for holding the homologous chromosomes together during prophase and metaphase of meiosis I.
- At the onset of anaphase of meiosis I, cohesins on chromosome arms are phosphorylated and, as a result, cleaved by separase, but cohesins in the region of the centromere are protected from phosphorylation and cleavage. This protection is brought about by a meiosis-specific cohesin subunit and a protein phosphatase that associates with centromeres. As a result, the sister chromatids remain linked to each other during segregation in meiosis I.
- Cleavage of centromeric cohesins during anaphase of meiosis II allows individual chromatids to segregate into germ cells.
- Meiotic cohesins facilitate the attachment of sister kinetochores to microtubules emanating from the same pole during meiosis I.
- Incomplete CDK inactivation between the two meiotic divisions inhibits DNA replication.

## LaunchPad

macmillan learning
Visit LaunchPad to access study tools and to learn more about the content in this chapter.

- Perspectives for the Future
- Analyze the Data
- Classic Experiment 19-1: How Cyclins Were Discovered
- Classic Experiment 19-2: Synthesis and Degradation of Mitotic Cyclin Are Required for Progression Through Mitosis
- Classic Experiment 19-3: The Formulation of the Checkpoint Concept
- Additional study tools, including videos, animations, and quizzes


## Key Terms

anaphase-promoting complex/cyclosome (APC/C) 886
aneuploidy 909
ATM/ATR 906
Aurora B 899
Cdc14 phosphatase 910
Cdc25 phosphatase 896
CDK-activating kinase (CAK) 886
CDK inhibitor (CKI) 886
checkpoint
pathway 904
cohesin 893
condensin 909
critical cell size 905
cyclin 876
cyclin-dependent kinase (CDK) 876
E2F transcription factor complex 889

G $_{1}$ CDKs 882
maturation-promoting factor (MPF) 879
meiosis 911
mitogen 889
mitosis 875
mitotic CDKs 882
monopolin complex 917
p53 protein 907
Polo kinases 897
Rb protein 889
restriction point 888
SCF (Skp1, Cullin, F-box proteins) 886
sensor 904
sister chromatids 875
S phase CDKs 882
START 888
synaptonemal complex 912
Wee1 896

## Review the Concepts

1. What cellular mechanism(s) ensure that passage through the cell cycle is unidirectional and irreversible? What molecular machinery underlies these mechanism(s)?
2. What types of experimental strategies do researchers employ to study cell cycle progression? How do genetic and biochemical approaches to this topic differ?
3. Tim Hunt shared the 2001 Nobel Prize in Physiology or Medicine for his work in the discovery and characterization
of cyclin proteins in eggs and embryos (see Classic Experiment 19-1). Describe the experimental steps that led him to his discovery of cyclins.
4. What experimental evidence indicates that cyclin B is required for a cell to enter mitosis? What evidence indicates that cyclin B must be destroyed for a cell to exit mitosis?
5. What physiological differences between $S$. pombe and S. cerevisiae make them useful yet complementary tools for studying the molecular mechanisms involved in cell cycle regulation and control?
6. In Xenopus, one of the substrates of mitotic CDKs is the phosphatase Cdc25. When phosphorylated by mitotic CDKs, Cdc25 is activated. What is the substrate of Cdc25? How does this information help to explain the rapid rise in mitotic CDK activity as cells enter mitosis?
7. Explain how CDK activity is modulated by the following proteins: (a) cyclin, (b) CAK, (c) Wee1, (d) p21.
8. Explain the role of CDK inhibitors. If cyclin-CDK complexes are necessary to allow regulated progression through the eukaryotic cell cycle, what would be the physiological rationale for CDK inhibitors?
9. Cancer cells typically lose cell cycle entry control. Explain how the following mutations, which are found in some cancer cells, lead to a bypass of these controls: (a) overexpression of cyclin D, (b) loss of Rb function, (c) loss of p16 function, (d) hyperactive E2F.
10. The Rb protein has been called the "master brake" of the cell cycle. Describe how the Rb protein acts as a cell cycle brake. How is the brake released in mid- to late $G_{1}$ to allow the cell to proceed to $S$ phase?
11. A common feature of cell cycle regulation is that the events of one phase ensure progression to a subsequent phase. In S. cerevisiae, $\mathrm{G}_{1}$ and $\mathrm{G}_{1} / \mathrm{S}$ phase CDKs promote S phase entry. Name two ways in which they promote the activation of $S$ phase.
12. For $S$ phase to be completed in a timely manner, DNA replication must be initiated from multiple origins in eukaryotes. In S. cerevisiae, what role do S phase CDKs and DDKs play to ensure that the entire genome is replicated once and only once per cell cycle?
13. In 2001, the Nobel Prize in Physiology or Medicine was awarded to three cell cycle scientists. Paul Nurse was recognized for his studies with the fission yeast $S$. pombe, in particular for the discovery and characterization of the wee $1^{+}$ gene. What did the characterization of the wee $1^{+}$gene tell us about cell cycle control?
14. Describe how cells know whether sister kinetochores are properly attached to the mitotic spindle.
15. Describe the series of events by which APC/C promotes the separation of sister chromatids at anaphase.
16. Leland Hartwell, the third recipient of the 2001 Nobel Prize in Physiology or Medicine, was acknowledged for his characterization of cell cycle checkpoint pathways in the budding yeast $S$. cerevisiae. What is a cell cycle checkpoint pathway? When during the cell cycle do checkpoint

pathways function? How do cell cycle checkpoint pathways help to preserve the genome?
17. What role do tumor suppressors, including p53, play in mediating cell cycle arrest for cells with DNA damage?
18. Individuals with the hereditary disorder ataxia telangiectasia suffer from neurodegeneration, immunodeficiency, and an increased incidence of cancer. The genetic basis for ataxia telangiectasia is a loss-of-function mutation in the gene encoding ATM (ATM; ataxia telangiectasia mutated). Besides p53, what other substrate is phosphorylated by ATM? How does the phosphorylation of this substrate lead to inactivation of CDKs to enforce cell cycle arrest?
19. Overall, meiosis and mitosis are analogous processes involving many of the same proteins. However, some proteins function uniquely in each of these cell-division events. Explain the meiosis-specific function of the following: (a) Ime2, (b) Rec8, (c) monopolin.
20. Explain why the incidence of Down syndrome increases with maternal age.

## References

## Overview of the Cell Cycle and Its Control

Morgan, D. O. 2006. The Cell Cycle: Principles of Control. New Science Press.

## Regulation of CDK Activity

Bloom, J., and F. R. Cross. 2007. Multiple levels of cyclin specificity in cell-cycle control. Nature Rev. Mol. Cell Biol. 8(2):149-160.

Ferrell, J. E. Jr, T. Y. Tsai, and Q. Yang. 2011. Modeling the cell cycle: why do certain circuits oscillate? Cell 144(6):874-885.

Lim, S., and P. Kaldis. 2013. CDKs, cyclins and CKIs: roles beyond cell cycle regulation. Development 140(15):3079-3093.

## Commitment to the Cell Cycle and DNA Replication

Costa, A., I. V. Hood, and J. M. Berger. 2013. Mechanisms for initiating cellular DNA replication. Ann. Rev. Biochem. 82:25-54.

Johnson, A., and J. M. Skotheim. 2013. Start and the restriction point. Curr. Opin. Cell Biol. 25(6):717-723.

Wood, A. J., A. F. Severson, and B. J. Meyer. 2010. Condensin and cohesin complexity: the expanding repertoire of functions. Nature Rev. Genet. 11(6):391-404.

## Entry into Mitosis

DeLuca, J. G., and A. Musacchio. 2012. Structural organization of the kinetochore-microtubule interface. Curr. Opin. Cell Biol. 24(1):48-56.

Hirano, T. 2012. Condensins: universal organizers of chromosomes with diverse functions. Genes \& Dev. 26(15): $1659-1678$.

Ohta, S., et al. 2011. Building mitotic chromosomes. Curr. Opin. Cell Biol. 23(1):114-121.

Smoyer, C. J., and S. L. Jaspersen. 2014. Breaking down the wall: the nuclear envelope during mitosis. Curr. Opin. Cell Biol. 26:1-9.

## Completion of Mitosis: Chromosome Segregation and Exit from Mitosis

Craney, A., and M. Rape. 2013. Dynamic regulation of ubiquitin-dependent cell cycle control. Curr. Opin. Cell Biol. 25(6):704-710.

Sullivan, M., and D. O. Morgan. 2007. Finishing mitosis, one step at a time. Nature Rev. Mol. Cell Biol. 8(11):894-903.

Wirth, K. G., et al. 2006. Separase: a universal trigger for sister chromatid disjunction but not chromosome cycle progression. J. Cell Biol. 172:847-860.

## Surveillance Mechanisms in Cell Cycle Regulation

Aguilera, A., and T. García-Muse. 2013. Causes of genome instability. Ann. Rev. Genet. 47:1-32.

Burke, D. J. 2009. Interpreting spatial information and regulating mitosis in response to spindle orientation. Genes \& Dev. 23(14):1613-1618.

Davie, E., and J. Petersen. 2012. Environmental control of cell size at division. Curr. Opin. Cell Biol. 24(6):838-844.

Foley, E. A., and T. M. Kapoor. 2013. Microtubule attachment and spindle assembly checkpoint signalling at the kinetochore. Nature Rev. Mol. Cell Biol. 14(1):25-37.

Shiloh, Y., and Y. Ziv. 2013. The ATM protein kinase: regulating the cellular response to genotoxic stress, and more. Nature Rev. Mol. Cell Biol. 14(4):197-210.

## Meiosis: A Special Type of Cell Division

Lesch, B. J., and D. C. Page. 2012. Genetics of germ cell development. Nature Rev. Genet. 13(11):781-794.

Lichten, M., and B. de Massy. 2011. The impressionistic landscape of meiotic recombination. Cell 147(2):267-270.

Miller, M. P., A. Amon, and E. Ünal. 2013. Meiosis I: when chromosomes undergo extreme makeover. Curr. Opin. Cell Biol. 25(6):687-696.

Zickler, D., and N. Kleckner. 1999. Meiotic chromosomes: integrating structure and function. Ann. Rev. Genet. 33:603-754.

this page left intentionally blank

![li5004542480e](li5004542480e.jpg)

The cochlea of the inner ear uses mechanotransduction to convert the energy in sound waves into neuronal signals. The distribution of type IV collagen in the extracellular matrix of the cochlear duct of a mouse was visualized by scanning thin-sheet laser imaging microscopy after removing the cells with the detergent SDS and the calcium with the chelator EDTA. The sample was then stained first with an anti-type IV collagen antibody and then a fluorescently labeled secondary antibody. The false colors in the image represent the relative intensities of fluorescence (red $>$ yellow $>$ blue), and thus the relative local amounts of type IV collagen, in the basal lamina of the blood vessels (red), other basement membranes (yellow), and the cochlear wall (blue). [Courtesy of Shane Johnson and Peter Santi, University of Minnesota.]

In the development of complex multicellular organisms such as plants and animals, progenitor cells differentiate into distinct "types" that have characteristic compositions, structures, and functions. Cells of a given type often aggregate into a tissue to cooperatively perform a common function: muscle contracts; neural tissue conducts electric impulses; xylem tissue in plants transports water. Different tissues can be organized into an organ, again to perform one or more specific functions. For instance, the muscles, valves, and blood vessels of a heart work together to pump blood. The coordinated functioning of many types of cells and tissues permits the organism to move, metabolize, reproduce, and carry out other essential activities. Indeed, the complex and diverse morphologies of plants and animals are examples of the whole being

## 20

## Integrating Cells into Tissues

greater than the sum of the individual parts, more technically described as the emergent properties of a complex system.

Vertebrates have hundreds of different cell types, including leukocytes (white blood cells) and erythrocytes (red blood cells), photoreceptors in the retina, fat-storing adipocytes, fibroblasts in connective tissue, and the hundreds of different subtypes of neurons in the human brain. Even simple animals exhibit complex tissue organization. The adult form of the roundworm Caenorhabditis elegans contains a mere 959 cells, yet these cells fall into 12 different general cell types and many distinct subtypes. But despite their diverse forms and functions, all animal cells can be classified as components of just five main classes of tissue: epithelial tissue, connective tissue, muscular tissue, neural tissue, and

### 20.1 Cell-Cell and Cell-Extracellular Matrix Adhesion: An Overview

### 20.2 Cell-Cell and Cell-Extracellular Matrix Junctions and Their Adhesion Molecules

20.3 The Extracellular Matrix I: The Basal Lamina

### 20.4 The Extracellular Matrix II: Connective Tissue

### 20.5 Adhesive Interactions in Motile and Nonmotile Cells

### 20.6 Plant Tissues