Boldface terms within a definition are also defined in this glossary. Figures and tables that illustrate defined terms are noted in parentheses.

AAA ATPase family A group of proteins that couple hydrolysis of ATP with large molecular movements usually associated with unfolding of protein substrates or the disassembly of multisubunit protein complexes.

ABC superfamily A large group of integral membrane proteins that often function as ATP-powered membrane transport proteins to move diverse molecules (e.g., phospholipids, cholesterol, sugars, ions, peptides) across cellular membranes. (Figure 11-15)
acetylcholine (ACh) Neurotransmitter that functions at vertebrate neuromuscular junctions and at various neuron-neuron synapses in the brain and peripheral nervous system. (Figure 22-25)
acetyl CoA Small, water-soluble metabolite comprising an acetyl group linked to coenzyme $\mathrm{A}(\mathrm{CoA})$. The acetyl group is transferred to citrate in the citric acid cycle and is used as a carbon source in the synthesis of fatty acids, steroids, and other molecules. (Figure 12-15)
acid Any compound that can donate a proton $\left(\mathrm{H}^{+}\right)$. The carboxyl and phosphate groups are the primary acidic groups in biological macromolecules.
actin Abundant structural protein in eukaryotic cells that interacts with many other proteins. The monomeric globular form ( $G$-actin) polymerizes to form actin filaments ( $F$-actin). In muscle cells, F-actin interacts with myosin during contraction. See also microfilament. (Figure 17-5)
action potential Rapid, transient, all-or-none electrical activity propagated in the plasma membrane of excitable cells (e.g., neurons and muscle cells) as the result of the selective opening and closing of voltage-gated $\mathrm{Na}^{+}$and $\mathrm{K}^{+}$channels. (Figures 22-2 and 22-9)
activation domain A region of an activator transcription factor that will stimulate transcription when fused to a DNA-binding domain.
activation energy The input of energy required to (overcome the barrier to) initiate a chemical reaction. By reducing the activation energy, an enzyme increases the rate of a reaction. (Figure 2-30)
activation loop A region in most protein-tyrosine kinases, containing a tyrosine residue that, when phosphorylated, increases kinase activity.
activator Specific transcription factor that stimulates transcription.
active site Specific region of an enzyme that binds a substrate molecule(s) and promotes a chemical change in the bound substrate. (Figure 3-23)
active transport Protein-mediated movement of an ion or small molecule across a membrane against its concentration gradient or electrochemical gradient driven by the coupled hydrolysis of ATP. (Figure 11-2, [1]; Table 11-1)
adapter proteins Adapter proteins physically link one protein to another protein by binding to both of them. Adapter proteins directly or indirectly (via additional adapters) connect cell-adhesion molecules or adhesion receptors to elements of the cytoskeleton or to intracellular signaling proteins.
adenosine triphosphate (ATP) See ATP.
adenylyl cyclase One of several enzymes that is activated by binding of certain ligands to their cell-surface receptors and catalyzes formation of cyclic AMP (cAMP) from ATP; also called adenylate cyclase. (Figures 15-25 and 15-26)
adhesion receptor Protein in the plasma membrane of animal cells that binds components of the extracellular matrix, thereby mediating cell-matrix adhesion. The integrins are major adhesion receptors. (Figure 20-1, [5])
ADP (adenosine diphosphate) The product, along with inorganic phosphate, of ATP hydrolysis by ATPases.
aequorin A bioluminescent protein, isolated from Aequorea victoria, that is activated by binding calcium ions.
aerobic Referring to a cell, organism, or metabolic process that utilizes gaseous oxygen $\left(\mathrm{O}_{2}\right)$ or that can grow in the presence of $\mathrm{O}_{2}$.
aerobic oxidation Oxygen-requiring metabolism of sugars and fatty acids to $\mathrm{CO}_{2}$ and $\mathrm{H}_{2} \mathrm{O}$ coupled to the synthesis of ATP.
aerobic respiration See aerobic oxidation.
afferent neurons Nerves that transmit signals from peripheral tissues to the central nervous system.
agonist A molecule, often synthetic, that mimics the biological function of a natural molecule (e.g., a hormone).
Agrin A glycoprotein synthesized by developing motor neurons that increases MuSK kinase activity in a muscle cell, facilitating development of a neuromuscular junction. (Figure 22-23)
Akt A cytosolic serine/threonine kinase that is activated following binding to PI 3,4-bisphosphate and PI 3,4,5-trisphosphate; also called protein kinase B.
allele One of two or more alternative forms of a gene. Diploid cells contain two alleles of each gene, located at the corresponding site (locus) on homologous chromosomes.

allosteric Referring to proteins and cellular processes that are regulated by allostery.
allostery Change in the tertiary and/or quaternary structure of a protein induced by binding of a small molecule to a specific regulatory site, causing a change in the protein's activity.
alpha carbon atom ( $\mathbf{C} \boldsymbol{\alpha}$ ) In amino acids, the central carbon atom that is bonded to four different chemical groups (except in glycine) including the side chain, or R group. (Figure 2-4)
alpha ( $\boldsymbol{\alpha}$ ) helix Common protein secondary structure in which the linear sequence of amino acids is folded into a right-handed spiral stabilized by hydrogen bonds between carboxyl and amide groups in the backbone. (Figure 3-4)
alternative splicing Process by which the exons of one premRNA are spliced together in different combinations, generating two or more different mature mRNAs from a single pre-mRNA. (Figure 5-16)
amino acid An organic compound containing at least one amino group and one carboxyl group. In the amino acids that are the monomers for building proteins, an amino group and carboxyl group are linked to a central carbon atom, the $\alpha$ carbon, to which a variable side chain is attached. (Figures 2-4 and 2-14)
aminoacyl-tRNA Activated form of an amino acid, used in protein synthesis, consisting of an amino acid linked via a highenergy ester bond to the $3^{\prime}$-hydroxyl group of a tRNA molecule. (Figure 5-19)
amphipathic Referring to a molecule or structure that has both a hydrophobic and a hydrophilic part.
amphiphilic See amphipathic.
amphitelic attachment Describes the correct attachment of chromosomes to the mitotic spindle, where sister kinetochores attach to microtubules emanating from opposite poles. (Figure 19-22)
amplification An increase in signal intensity as a cellular signal is transduced.
amyotrophic lateral sclerosis (ALS) Lou Gehrig's disease, characterized by progressive death of the motor neurons connecting the central nervous system to muscles.
anaerobic Referring to a cell, organism, or metabolic process that functions in the absence of gaseous oxygen $\left(\mathrm{O}_{2}\right)$.
anaphase Mitotic stage during which the sister chromatids (or duplicated homologs in meiosis I) separate and move apart (segregate) toward the spindle poles. (Figure 18-37)
anchoring junctions Specialized regions on the cell surface containing cell-adhesion molecules or adhesion receptors; includes adherens junctions and desmosomes, which mediate cell-cell adhesion, and hemidesmosomes, which mediate cell-matrix adhesion. (Figures 20-14 and 20-16)
anaerobic respiration Respiration in which molecules other than oxygen, such as sulfate or nitrate, are used as the final recipient of the electrons transported via the electron-transport chain.
anaphase-promoting complex or cyclosome (APC/C) A ubiquitin ligase that targets securin, mitotic cyclins, and other proteins for proteasomal degradation from the onset of anaphase until entry into the subsequent cell cycle.
aneuploidy Any deviation from the normal diploid number of chromosomes in which extra copies of one or more chromosomes are present or one of the normal copies is missing.
anion A negatively charged ion.
antagonist A molecule, often synthetic, that blocks the biological function of a natural molecule (e.g., hormone).
antibody A protein (immunoglobulin), normally produced in response to an antigen, that interacts with a particular site (epitope) on the same antigen and facilitates its clearance from the body. (Figure 3-21)
anticodon Sequence of three nucleotides in a tRNA that is complementary to a codon in an mRNA. During protein synthesis, base pairing between a codon and anticodon aligns the tRNA carrying the corresponding amino acid for addition to the growing polypeptide chain. (Figure 5-20)
antigen Any material (usually foreign) that elicits an immune response. For B cells, an antigen elicits formation of antibody that specifically binds the same antigen; for T cells, an antigen elicits a proliferative response, followed by production of cytokines or the activation of cytotoxic activity.
antigen-presenting cell (APC) Any cell that can digest an antigen into small peptides and display the peptides in association with class II MHC molecules on the cell surface where they can be recognized by T cells. Professional APCs (dendritic cells, macrophages, and B cells) constitutively express class II MHC molecules. (Figures 23-25 and 23-26)
antiport A type of cotransport in which a membrane protein (antiporter) transports two different molecules or ions across a cell membrane in opposite directions. See also symport. (Figure $11-2,[3 C])$
apical Referring to the tip (apex) of a cell, an organ, or other body structure. In the case of epithelial cells, the apical surface is exposed to the exterior of the body or to an internal open space (e.g., intestinal lumen, duct). (Figure 20-10)
apoptosis A genetically regulated process, occurring in specific tissues during development and disease, by which a cell destroys itself; marked by the breakdown of most cell components and a series of well-defined morphological changes; also called programmed cell death. See also caspases. (Figures 21-33 and 21-40)
apoptosome Large, disk-shaped heptamer of mammalian Apaf-1, a protein that assembles in response to apoptosis signals and serves as an activation machine for initiator and effector caspases. (Figure 21-41)
aptamer Region of single-stranded RNA or DNA $\sim 70-120 \mathrm{bp}$ long that folds into a complex tertiary structure that binds a small molecule specifically. (Figure 9-8)
aquaporins A family of membrane transport proteins that allow water and a few other small uncharged molecules, such as glycerol, to cross biomembranes. (Figure 11-8)
archaea Class of prokaryotes that constitutes one of the three distinct evolutionary lineages of modern-day organisms; also called archaebacteria and archaeans. In some respects, archaeans are more similar to eukaryotes than to bacteria (eubacteria). (Figure 1-1)

associated constant $\left(K_{\mathrm{a}}\right)$ See equilibrium constant.
aster Structure composed of microtubules (astral fibers) that radiate outward from a centrosome during mitosis. (Figure 18-37)
astrocytes Star-shaped glial cells in the brain and spinal cord that perform many functions, including support of endothelial cells that form the blood-brain barrier, maintain extracellular ion composition, and provide nutrients to neurons.
asymmetric carbon atom A carbon atom bonded to four different atoms or chemical groups; also called chiral carbon atom. The bonds can be arranged in two different ways, producing stereoisomers that are mirror images of each other. (Figure 2-4)
asymmetric cell division Any cell division in which the two daughter cells receive the same genes but otherwise inherit different components (e.g., mRNAs, proteins) from the parental cell. (Figure 21-23b)
ATM/ATR Two related proteins kinases that are activated by DNA damage. Once active, they phosphorylate other proteins to initiate the cell's response to DNA damage.
ATP (adenosine triphosphate) A nucleotide that is the most important molecule for capturing and transferring free energy in cells. Hydrolysis of each of the two phosphoanhydride bonds in ATP releases a large amount of free energy that can be used to drive energy-requiring cellular processes. (Figure 2-31)
ATPase One of a large group of enzymes that catalyze hydrolysis of ATP to yield ADP and inorganic phosphate with release of free energy. See also $\mathrm{Na}^{+} / \mathrm{K}^{+}$ATPase and ATP-powered pump.
ATP-powered pump Any transmembrane protein that has ATPase activity and couples hydrolysis of ATP to the active transport of an ion or small molecule across a biomembrane against its electrochemical gradient; often simply called pump. (Figure 11-9)
ATP synthase Multimeric protein complex, bound to inner mitochondrial membranes, thylakoid membranes of chloroplasts, and the bacterial plasma membrane, that catalyzes synthesis of ATP during oxidative phosphorylation and photosynthesis; also called $F_{0} F_{1}$ complex. (Figure 12-26a)

## ATR See ATM/ATR.

Aurora B kinase Destabilizes faulty microtubule-kinetochore interactions by phosphorylating microtubule-binding components within the kinetochore.
Aurora kinases Serine/threonine kinases that play a crucial role in cell division by controlling chromatid segregation. Aurora B kinase destabilizes faulty microtubule-kinetochore interactions by phosphorylating microtubule-binding components within the kinetochore.
autocrine Referring to signaling mechanism in which a cell produces a signaling molecule (e.g., growth factor) and then binds and responds to it.
autophagosome A large region of cytoplasm including multiple ribosomes and mitochondria engulfed in a closed membrane during periods of cell starvation for amino acids. The autophagosomes fuse with lysosomes, where the autophagosome constituents are broken down into amino acids and other nutrients that are transported into the cytoplasm.
autophagy Literally, "eating oneself"; the process by which cytosolic proteins and organelles are delivered to the lysosome, degraded, and recycled. Autophagy involves the formation of a double-membrane vesicle called an autophagosome or autophagic vesicle. (Figure 14-35)
autoradiography Technique for visualizing radioactive molecules in a sample (e.g., a tissue section or electrophoretic gel) by exposing a photographic film (emulsion) or two-dimensional electronic detector to the sample. The exposed film is called an autoradiogram or autoradiograph.
autosome Any chromosome other than a sex chromosome.
axon Long process extending from the cell body of a neuron that is capable of conducting an electric impulse (action potential), generated at the junction with the cell body, toward its distal, branching end (the axon terminus). (Figure 22-1)
axonal transport Motor protein-mediated transport of organelles and vesicles along microtubules in axons of nerve cells. Anterograde transport occurs from cell body toward axon terminal); retrograde transport, from axon terminal toward cell body. (Figures 18-16 and 18-17)
axoneme Bundle of microtubules and associated proteins present in cilia and flagella and responsible for their structure and movement. (Figure 18-31)
bacteria Class of prokaryotes that constitutes one of the three distinct evolutionary lineages of modern-day organisms; also called eubacteria. Phylogenetically distinct from archaea and eukaryotes. (Figure 1-1)
bacteriophage (phage) Any virus that infects bacterial cells. Some phages are widely used as vectors in DNA cloning.
basal See basolateral.
basal body Structure at the base of a cilium or flagellum from which microtubules forming the axoneme assemble; structurally similar to a centriole. (Figure 18-31)
basal lamina (pl. basal laminae) A thin sheet-like network of extracellular-matrix components that underlies most animal epithelia and other organized groups of cells (e.g., muscle), separating them from connective tissue or other cells. (Figures 20-21 and 20-22)
base Any compound, often containing nitrogen, that can accept a proton $\left(\mathrm{H}^{+}\right)$from an acid. Also, commonly used to denote the purines and pyrimidines in DNA and RNA.
base pair Association of two complementary nucleotides in a DNA or RNA molecule stabilized by hydrogen bonding between their base components. Adenine pairs with thymine or uracil ( $\mathrm{A} \cdot$ $\mathrm{T}, \mathrm{A} \cdot \mathrm{U}$ ) and guanine pairs with cytosine $(\mathrm{G} \cdot \mathrm{C})$. (Figure 4-3b)
basic helix-loop-helix See helix-loop-helix, basic.
basolateral Referring to the base (basal) and side (lateral) of a polarized cell, organ, or other body structure. In the case of epithelial cells, the basolateral surface abuts adjacent cells and the underlying basal lamina. (Figure 20-10)
B cell A lymphocyte that matures in the bone marrow and expresses antigen-specific receptors (membrane-bound immunoglobulin). After interacting with antigen, a B cell proliferates and differentiates into antibody-secreting plasma cells.

B-cell receptor Complex composed of an antigen-specific mem-brane-bound immunoglobulin molecule and associated signaltransducing $\operatorname{Ig} \alpha$ and $\operatorname{Ig} \beta$ chains. (Figure 23-18)
benign Referring to a tumor containing cells that closely resemble normal cells. Benign tumors stay in the tissue where they originate but can be harmful due to continued growth. See also malignant.
beta ( $\boldsymbol{\beta}$ )-adrenergic receptors Seven spanning G protein-coupled receptors that bind adrenaline and related molecules, leading to activation of adenylyl cyclase.
beta $(\boldsymbol{\beta})$ sheet A flat secondary structure in proteins that is created by hydrogen bonding between the backbone atoms in two different polypeptide chains or segments of a single folded chain. (Figure 3-5)
beta $(\boldsymbol{\beta})$ turn A short U-shaped secondary structure in proteins. (Figure 3-6)
bi-oriented Indicates that the kinetochores of sister chromatids have attached to microtubules emanating from opposite spindle poles.
blastocyst Stage of mammalian embryo composed of $\approx 64$ cells that have separated into two cell types-trophectoderm, which will form extra-embryonic tissues, and the inner cell mass, which gives rise to the embryo proper; stage that implants in the uterine wall and corresponds to the blastula of other animal embryos. (Figure 21-3)
blastopore The first opening that forms during embryogenesis of bilaterally symmetric animals, which later becomes the gut. This opening may become either the mouth or the anus.
bromodomain A protein domain of $\sim 120$ amino acids that binds acetylated lysine; found in chromosome-associated proteins involved in transcriptional activation.
buffer A solution of the acid (HA) and base $\left(\mathrm{A}^{-}\right)$form of a compound that undergoes little change in pH when small quantities of strong acid or base are added at pH values near the compound's $\mathrm{p} K_{\mathrm{a}}$.
cadherins A family of dimeric cell-adhesion molecules that aggregate in adherens junctions and desmosomes and mediate $\mathrm{Ca}^{2+}$-dependent cell-cell homophilic interactions. (Figure 20-2)
calmodulin A small cytosolic regulatory protein that binds four $\mathrm{Ca}^{2+}$ ions. The $\mathrm{Ca}^{2+} /$ calmodulin complex binds to many proteins, thereby activating or inhibiting them. (Figure 3-33)
Calvin cycle See carbon fixation.
cancer General term denoting any of various malignant tumors, whose cells grow and divide more rapidly than normal, invade surrounding tissue, and sometimes spread (metastasize) to other sites.
capsid The outer proteinaceous coat of a virus, formed by multiple copies of one or more protein subunits and enclosing the viral nucleic acid.
CAP site The DNA sequence in bacteria bound by catabolite activator protein, also known as the cyclic AMP regulatory protein. (Figure 9-4)
carbohydrate General term for certain polyhydroxyaldehydes, polyhydroxyketones, or compounds derived from these usually having the formula $\left(\mathrm{CH}_{2} \mathrm{O}\right)_{n}$. Primary type of compound used for storing and supplying energy in animal cells. (Figure 2-18)
carbon fixation The major metabolic pathway that fixes $\mathrm{CO}_{2}$ into carbohydrates during photosynthesis; also called the Calvin cycle. It is indirectly dependent on light but can occur both in the dark and light. (Figure 12-48)
carcinogen Any chemical or physical agent that can cause cancer when cells or organisms are exposed to it.
caretaker gene Any gene whose encoded protein helps protect the integrity of the genome by participating in the repair of damaged DNA. Loss of function of a caretaker gene leads to increased mutation rates and promotes carcinogenesis.
caspases A class of vertebrate protein-degrading enzymes (proteases) that function in apoptosis and work in a cascade with each type activating the next. (Figures 21-35 and 21-40)
catabolism Cellular degradation of complex molecules to simpler ones usually accompanied by the release of energy. Anabolism is the reverse process in which energy is used to synthesize complex molecules from simpler ones.
catalyst A substance that increases the rate of a chemical reaction without undergoing a permanent change in its structure. Enzymes are proteins with catalytic activity, and ribozymes are RNAs that can function as catalysts. (Figure 3-22)
cation A positively charged ion.
Cdc14 phosphatase A dual-specificity protein phosphatase that triggers mitotic CDK inactivation at the end of mitosis.
Cdc25 phosphatase A protein phosphatase that dephosphorylates CDKs on threonine 14 and tyrosine 15, thereby activating CDKs.
CDK-activating kinase (CAK) Phosphorylates CDKs on a threonine residue near the active site. This phosphorylation is essential for CDK activity.
CDK inhibitor (CKI) Binds to cyclin-CDK complex and inhibits its activity.
cDNA (complementary DNA) DNA molecule copied from an mRNA molecule by reverse transcriptase and therefore lacking the introns present in the DNA of the genome.
cell-adhesion molecules (CAMs) Proteins in the plasma membrane of cells that bind similar proteins on other cells, thereby mediating cell-cell adhesion. Four major classes of CAMs include the cadherins, IgCAMs, integrins, and selectins. (Figures 20-1 and 20-2)
cell-adhesion proteins See cell adhesion molecules (CAMs).
cell cycle Ordered sequence of events in which a eukaryotic cell duplicates its chromosomes and divides into two. The cell cycle normally consists of four phases: $\mathrm{G}_{1}$ before DNA synthesis occurs; S when DNA replication occurs; $\mathrm{G}_{2}$ after DNA synthesis; and M when cell division occurs, yielding two daughter cells. Under certain conditions, cells exit the cell cycle during $\mathrm{G}_{1}$ and remain in the $\mathrm{G}_{0}$ state as nondividing cells. (Figures 1-16 and 19-1)
cell division Separation of a cell into two daughter cells. In higher eukaryotes, it involves division of the nucleus (mitosis) and of the cytoplasm (cytokinesis); mitosis often is used to refer to both nuclear and cytoplasmic division.

cell junctions Specialized regions on the cell surface through which cells are joined to each other or to the extracellular matrix. (Figure 20-11; Table 20-3)
cell line A population of cultured cells, of plant or animal origin, that has undergone a genetic change allowing the cells to grow indefinitely. (Figure 4-1b)
cell polarity The ability of cells to organize their internal structure, resulting in changes of cell shape and generating regions of the plasma membrane with different protein and lipid compositions.
cell strain A population of cultured cells, of plant or animal origin, that has a finite life span and eventually dies, commonly after $25-50$ generations. (Figure 4-1a)
cell-surface receptor Protein embedded in the plasma membrane that has an extracellular domain that binds an extracellular molecule(s), called a ligand. Many cell surface receptors bind to extracellular signaling molecules; such binding induces conformational changes in the receptor, altering the activity of the receptor's intracellular domain, which transmits the signal to the interior of the cell.
cellular communication The transfer of information via signaling molecules or ions from one cell to another.
cellular respiration See respiration.
cellulose A structural polysaccharide made of glucose units linked together by $\beta(1 \rightarrow 4)$ glycosidic bonds. It forms long microfibrils, which are the major component of the cell wall in plants.
cell wall A specialized, rigid extracellular matrix that lies next to the plasma membrane, protecting a cell and maintaining its shape; prominent in most fungi, plants, and prokaryotes but absent in most multicellular animals. (Figure 20-41)
centriole Either of two cylindrical structures within the centrosome of animal cells and containing nine sets of triplet microtubules; structurally similar to a basal body. (Figure 18-6)
centromere DNA sequence required for proper segregation of chromosomes during mitosis and meiosis; the region of mitotic chromosomes where the kinetochore forms and that appears constricted. (Figure 6-39)
centrosome (cell center) Structure located near the nucleus of animal cells that is the primary microtubule-organizing center (MTOC); it contains a pair of centrioles embedded in a protein matrix and duplicates before mitosis, with each centrosome becoming a spindle pole. (Figures 18-6 and 18-35)
centrosome disjunction Describes the process of centrosome segregation during prophase.
channels Membrane proteins that transport water, ions, or small hydrophilic molecules across membranes down their concentration or electric potential gradients.
chaperone Collective term for two types of proteins-molecular chaperones and chaperonins-that prevent misfolding of a target protein or actively facilitate proper folding of an incompletely folded target protein, respectively. (Figures 3-17 and 3-18)
chaperonin See chaperone.
checkpoint Any of several points in the eukaryotic cell cycle at which progression of a cell to the next stage can be halted until conditions are suitable.
checkpoint pathway Surveillance mechanism that prevents initiation of each step in cell division until earlier steps on which it depends have been completed and mistakes that occurred during the process have been corrected.
chemical equilibrium The state of a chemical reaction in which the concentration of all products and reactants is constant because the rates of the forward and reverse reactions are equal.
chemical potential energy The energy stored in the bonds connecting atoms in molecules.
chemiosmosis Process whereby an electrochemical proton gradient ( pH plus electric potential) across a membrane is used to drive an energy-requiring process such as ATP synthesis; also called chemiosmotic coupling. See proton-motive force. (Figure 12-2)
chemokine Any of numerous small, secreted proteins that function as chemotatic cues for leukocytes.
chemotaxis Movement of a cell or organism toward or away from certain chemicals.
chlorophylls A group of light-absorbing porphyrin pigments that are critical in photosynthesis. (Figure 12-39)
chloroplast A specialized organelle in plant cells that is surrounded by a double membrane and contains internal chlorophyllcontaining membranes (thylakoids) where the light-absorbing reactions of photosynthesis occur. (Figure 12-37)
cholesterol A lipid containing the four-ring steroid structure with a hydroxyl group on one ring; a component of many eukaryotic membranes and the precursor of steroid hormones, bile acids, and vitamin D. (Figure 7-8c)
chromatid One copy of a replicated chromosome, formed during the $S$ phase of the cell cycle, that is joined to the other copy; also called sister chromatid. During mitosis, the two chromatids separate, each becoming a chromosome of one of the two daughter cells. (Figure 8-35)
chromatin Complex of DNA, histones, and nonhistone proteins from which eukaryotic chromosomes are formed. Condensation of chromatin during mitosis yields the visible metaphase chromosomes. (Figures 8-23 and 8-25)
chromatography, liquid Group of biochemical techniques for separating mixtures of molecules (e.g., different proteins) based on their mass (gel filtration chromatography), charge (ion exchange chromatography), or ability to bind specifically to other molecules (affinity chromatography). (Figure 3-38)
chromosome In eukaryotes, the structural unit of the genetic material consisting of a single, linear double-stranded DNA molecule and associated proteins. In most prokaryotes, a single, circular double-stranded DNA molecule constitutes the bulk of the genetic material. See also chromatin and karyotype.
cilium (pl. cilia) Short, membrane-enclosed structure extending from the surface of eukaryotic cells and containing a core bundle of microtubules. Cilia usually occur in groups and beat rhythmically to move a cell (e.g., single-celled organism) or to move small particles or fluid along a surface (e.g., trachea cells). See also axoneme and flagellum.
cisterna (pl. cisternae) Flattened membrane-bounded compartment, as found in the Golgi complex and endoplasmic reticulum.

citric acid cycle A set of nine coupled reactions occurring in the matrix of the mitochondrion in which acetyl groups are oxidized, generating $\mathrm{CO}_{2}$ and reduced intermediates used to produce ATP; also called Krebs cycle and tricarboxylic acid (TCA) cycle. (Figure 12-16)
clathrin A fibrous protein that with the aid of assembly proteins polymerizes into a lattice-like network at specific regions on the cytosolic side of a membrane, thereby forming a clathrin-coated pit that buds off to form a vesicle. (Figure 14-18; Table 14-1)
cleavage In embryogenesis, the series of rapid cell divisions that occurs following fertilization and with little cell growth, producing progressively smaller cells; culminates in formation of the blastocyst in mammals or blastula in other animals. Also used as a synonym for the hydrolysis of molecules. (Figure 21-3)
cleavage furrow Indentation in the plasma membrane that represents the initial steps in cytokinesis.
cleavage/polyadenylation complex Large, multiprotein complex that catalyzes the cleavage of pre-mRNA at a $3^{\prime}$ poly $(\mathrm{A})$ site and the initial addition of adenylate (A) residues to form the poly(A) tail. (Figure 10-15)
clone (1) A population of genetically identical cells, viruses, or organisms descended from a common ancestor. (2) Multiple identical copies of a gene or DNA fragment generated and maintained via DNA cloning.
co-activator A protein or protein complex required for transcription activation that does not bind directly to DNA. In contrast, an activator has a DNA-binding domain and binds directly to a DNA transcription control sequence.
codon Sequence of three nucleotides in DNA or mRNA that specifies a particular amino acid during protein synthesis; also called triplet. Of the 64 possible codons, three are stop codons, which do not specify amino acids and cause termination of synthesis. (Table 5-1)
cohesin Protein complex that holds the replicated sister chromatids together.
cohesin complex Protein complex that establishes cohesion between sister chromatids.
coiled coil A protein structural motif marked by amphipathic $\alpha$ helical regions that can self-associate to form stable, rodlike structures in proteins; commonly found in fibrous proteins and certain transcription factors. (Figure 3-10a)
collagen A triple-helical glycoprotein rich in glycine and praline that is a major component of the extracellular matrix and connective tissues. The numerous subtypes differ in their tissue distribution and the extracellular components and cell-surface proteins with which they associate. (Figure 20-24; Table 20-4)
complement A group of constitutive serum proteins that bind directly to microbial or fungal surfaces, thereby activating a proteolytic cascade that culminates in formation of the cytolytic membrane attack complex. (Figure 23-5)
complementary (1) Referring to two nucleic acid sequences or strands that can form perfect base pairs with each other. (2) Describing regions on two interacting molecules (e.g., an enzyme and its substrate) that fit together in a lock-and-key fashion.
complementary DNA (cDNA) See cDNA.
complementation See genetic complementation and functional complementation.
concentration gradient In cell biology, a difference in the concentration of a substance in different regions of a cell or embryo or on different sides of a cellular membrane.
condensin Protein complex that promotes chromosome condensation.
condensin complex Protein complex related to cohesins that compacts chromosomes and is necessary for their segregation during mitosis.
conformation The precise shape of a protein or other macromolecule in three dimensions resulting from the spatial location of the atoms in the molecule. (Figure 3-8)
connexins A family of transmembrane proteins that form gap junctions in vertebrates. (Figure 20-21)
constitutive Referring to the continuous production or activity of a cellular molecule or the continuous operation of a cellular process (e.g., constitutive secretion) that is not regulated by internal or external signals.
contractile bundles Bundles of actin and myosin in nonmuscle cells that function in cell adhesion (e.g., stress fibers) or cell movement (contractile ring in dividing cells).
contractile ring Composed of actin and myosin; located beneath the plasma membrane. During cytokinesis, its contraction pulls the membrane inward, eventually closing the connection between the two daughter cells.
contractile vacuole A vesicle found in many protozoans that takes up water from the cytosol and periodically discharges its contents through fusion with the plasma membrane.
coordinately regulated Genes whose expression are induced and repressed at the same time, as for the genes in a single bacterial operon. (Figure 5-13)
co-oriented Indicates that sister kinetochores attach to microtubules emanating from the same spindle pole rather than from opposite spindle poles.
COPI A class of proteins that coat transport vesicles in the secretory pathway. COPI-coated vesicles move proteins from the Golgi to the endoplasmic reticulum and from later to earlier Golgi cisternae. (Table 14-1)
COPII A class of proteins that coat transport vesicles in the secretory pathway. COPII-coated vesicles move proteins from the endoplasmic reticulum to the Golgi. (Table 14-1)
co-repressor A protein or protein complex required for transcription repression that does not bind directly to DNA. In contrast, a repressor has a DNA-binding domain and binds directly to a DNA transcription control sequence.
cotranslational translocation Simultaneous transport of a secretory protein into the endoplasmic reticulum as the nascent protein is still bound to the ribosome and being elongated. (Figure 13-6)
cotransport Protein-mediated movement of an ion or small molecule across a membrane against a concentration gradient driven by coupling to movement of a second molecule down its concentration gradient in the same (symport) or opposite (antiport) direction. (Figure 11-2, [3B, C]; Table 11-1)

covalent bond Stable chemical force that holds the atoms in molecules together by sharing of one or more pairs of electrons. See also noncovalent interaction. (Figures 2-2 and 2-6)
CpG islands Regions in vertebrate DNA $\sim 100$ to $\sim 1000 \mathrm{bp}$ that have an unusually high occurrence of the sequence CG. Many CpG islands function as promoters for transcription initiation, usually in both directions.
CRISPR Named for clustered regularly interspaced short palindromic repeats, a mechanism used by many bacterial cells to protect against invasion by foreign DNA. The mechanism has been utilized in the lab to edit the genomic DNA of metazoan organisms.
cristae Sheet-like and tubelike invaginations that extend from the boundary membrane into the center of the mitochondrion.
critical cell size The cell size at which cells can enter the cell cycle. The critical cell size varies with nutrient availability.
cross-exon recognition complex Large assembly including RNA-binding SR proteins and other components that helps delineate exons in the pre-mRNAs of higher eukaryotes and ensure correct RNA splicing. (Figure 10-13)
crossing over Exchange of genetic material between maternal and paternal chromatids during meiosis to produce recombined chromosomes. See also recombination. (Figure 6-10)
cyclic AMP (cAMP) A second messenger, produced in response to hormonal stimulation of certain $G$ protein-coupled receptors, that activates protein kinase A. (Figure 15-6; Table 15-2)
cyclic GMP (cGMP) A second messenger that opens cation channels in rod cells and activates protein kinase G in vascular smooth muscle and other cells. (Figures 15-6, 15-23, and 15-38)
cyclin Any of several related proteins whose concentrations rise and fall during the course of the eukaryotic cell cycle. Cyclins form complexes with cyclin-dependent kinases, thereby activating and determining the substrate specificity of these enzymes.
cyclin-dependent kinase (CDK) A protein kinase that is catalytically active only when bound to a cyclin. Various cyclin-CDK complexes trigger progression through different stages of the eukaryotic cell cycle by phosphorylating specific target proteins. (Table 19-1)
cytochromes A group of colored, heme-containing proteins, some of which function as electron carriers during cellular respiration and photosynthesis. (Figure 12-20a)
cytokine Any of numerous small, secreted proteins (e.g., erythropoietin, G-CSF, interferons, interleukins) that bind to cellsurface receptors on blood and immune-system cells to trigger their differentiation or proliferation.
cytokine receptor Member of major class of cell-surface signaling receptors, including those for erythropoietin, growth hormone, interleukins, and interferons. Ligand binding leads to activation of cytosolic JAK kinases associated with the receptor, thereby initiating intracellular signaling pathways. (Figures 16-6 and 16-13)
cytokinesis The division of the cytoplasm following mitosis to generate two daughter cells, each with a nucleus and cytoplasmic organelles. (Figure 17-35)
cytoplasm Viscous contents of a cell that are contained within the plasma membrane but, in eukaryotic cells, outside the nucleus.
cytoskeletal proteins Proteins that confer cell strength and rigidity, including microtubules, microfilaments, and intermediate filaments.
cytoskeleton Network of fibrous elements, consisting primarily of microtubules, microfilaments, and intermediate filaments, found in the cytoplasm of eukaryotic cells. The cytoskeleton provides organization and structural support for the cell and permits directed movement of organelles, chromosomes, and the cell itself. (Figures 17-1, 17-2, and 18-1)
cytoskeletal proteins See cytoskeleton.
cytosol Unstructured aqueous phase of the cytoplasm excluding organelles, membranes, and insoluble cytoskeletal components.
cytosolic face The face of a cell membrane directed toward the cytosol. (Figure 7-5)

## DAG See diacylglycerol.

dalton Unit of molecular mass approximately equal to the mass of a hydrogen atom $\left(1.66 \times 10^{-24} \mathrm{~g}\right)$.
denaturation Drastic alteration in the conformation of a protein or nucleic acid due to disruption of various noncovalent interactions caused by heating or exposure to certain chemicals; usually results in loss of biological function.
dendrite Process extending from the cell body of a neuron that is relatively short and typically branched and receives signals from axons of other neurons. (Figure 22-1)
dendritic cells Phagocytic professional antigen-presenting cells that reside in various tissues and can detect broad patterns of pathogen markers via their Toll-like receptors. After internalizing antigen at a site of tissue injury or infection, they migrate to lymph nodes and initiate activation of T cells. (Figure 23-7)
deoxyribonucleic acid See DNA.
depolarization Decrease in the cytosolic-face negative electric potential that normally exists across the plasma membrane of a cell at rest, resulting in a less inside-negative or an inside-positive membrane potential.
destruction box Recognition motif in APC/C substrates.
determinant In the context of antibody recognition of an antigen, a region on a protein to which the antibody binds. In this context, it is synonymous with epitope.
deuterosomes A group of bilaterally symmetric animals whose anus develops close to the blastopore and has a dorsal nerve cord. This group includes all chordates (fish, amphibians, reptiles, birds, and mammals) and echinoderms (sea stars, sea urchins).
diacylglycerol (DAG) Membrane-bound second messenger that can be produced by cleavage of phosphoinositides in response to stimulation of certain cell-surface receptors. (Figures 15-6 and 15-33)
diploid Referring to an organism or cell having two full sets of homologous chromosomes and hence two copies (alleles) of each gene or genetic locus. Somatic cells contain the diploid number of chromosomes $(2 n)$ characteristic of a species. See also haploid.
dipole A positive charge separated in space from an equal but opposite negative charge.
dipole moment A quantitative measure of the extent of charge separation, or strength, of a dipole, which for a chemical bond is the product of the partial charge on each atom and the distance between the two atoms.

disaccharide A small carbohydrate (sugar) composed of two monosaccharides covalently joined by a glycosidic bond. (Figure 2-19)
dissociation constant $\left(K_{\mathrm{d}}\right)$ See equilibrium constant.
disulfide bond (-S-S-) A common covalent linkage between the sulfur atoms on two cysteine residues in different polypeptides or in different parts of the same polypeptide.
diversity The entire set of antigen-specific receptors encoded by an immune system.
DNA (deoxyribonucleic acid) Long linear polymer, composed of four kinds of deoxyribose nucleotides, that is the carrier of genetic information. See also double helix, DNA. (Figure 5-3)
DNA-binding domain The domain of a transcription factor that binds specific, closely related DNA sequences.
DNA cloning Recombinant DNA technique in which specific cDNAs or fragments of genomic DNA are inserted into a cloning vector, which then is incorporated into cultured host cells and maintained during growth of the host cells; also called gene cloning. (Figure 6-14)
DNA damage response system Pathway that senses DNA damage and induces cell cycle arrest and DNA repair pathways.
DNA library Collection of cloned DNA molecules consisting of fragments of the entire genome (genomic library) or of DNA copies of all the mRNAs produced by a cell type (cDNA library) inserted into a suitable cloning vector.
DNA ligase An enzyme that links together the $3^{\prime}$ end of one DNA fragment with the $5^{\prime}$ end of another, forming a continuous strand.
DNA microarray An ordered set of thousands of different nucleotide sequences arrayed on a microscope slide or other solid surface; can be used to determine patterns of gene expression in different cell types or in a particular cell type at different developmental stages or under different conditions. (Figures 5-29 and 5-30)
DNA polymerase An enzyme that copies one strand of DNA (the template strand) to make the complementary strand, forming a new double-stranded DNA molecule. All DNA polymerases add deoxyribonucleotides one at a time in the $5^{\prime} \rightarrow 3^{\prime}$ direction to the $3^{\prime}$ end of a short preexisting primer strand of DNA or RNA.
DNA recombination The process by which two DNA molecules with similar sequences are subject to double-stranded breaks and then rejoined to generate two recombinant DNA molecules with sequences comprised of portions of each parent. (Figures 5-41 and 5-42)
domain A region of protein that has a distinct, and often independent, function or structure, or that has a distinct topology relative to the rest of the protein.
dominant In genetics, referring to that allele of a gene expressed in the phenotype of a heterozygote; the nonexpressed allele is recessive; also refers to the phenotype associated with a dominant allele. Mutations that produce dominant alleles generally result in a gain of function. (Figure 6-2)
dominant-negative In genetics, an allele that acts in a dominant manner but produces an effect similar to a loss of function; generally is an allele encoding a mutant protein that blocks the function of the normal protein by binding either to it or to a protein upstream or downstream of it in a pathway. double helix, DNA The most common three-dimensional structure for cellular DNA in which the two polynucleotide strands are antiparallel and wound around each other with complementary bases hydrogen-bonded. (Figure 5-3)
double-strand break Form of DNA damage where both phos-phate-sugar backbones of the DNA are severed.
downstream (1) For a gene, the direction RNA polymerase moves during transcription, which is toward the end of the template DNA strand with a $5^{\prime}$-hydroxyl group. Nucleotides downstream from the +1 position (the first transcribed a nucleotide) are designated $+2,+3$, etc. (2) Events that occur later in a cascade of steps (e.g., signaling pathway). See also upstream.
dyneins A class of motor proteins that use the energy released by ATP hydrolysis to move toward the $(-)$ end of microtubules. Dyneins can transport vesicles and organelles, are responsible for the movement of cilia and flagella, and play a role in chromosome movement during mitosis. (Figures 18-24 and 18-25)
ectoderm Outermost of the three primary cell layers of the animal embryo; gives rise to epidermal tissues, the nervous system, and external sense organs. See also endoderm and mesoderm.
effector Ultimate component of a signal transduction pathway that elicits a response to the transmitted signal.
endoderm Innermost of the three primary cell layers of the animal embryo; gives rise to the gut and most of the respiratory tract. See ectoderm and mesoderm.
EF hand A type of helix-loop-helix structural motif that occurs in many $\mathrm{Ca}^{2+}$-binding proteins such as calmodulin. (Figure 3-10b)
efferent neurons Nerves that transmit signals from the central nervous system to peripheral tissues such as muscles and endocrine cells.
electric potential The energy associated with the separation of positive and negative charges. An electric potential is maintained across the plasma membrane of nearly all cells.
electrochemical gradient The driving force that determines the energetically favorable direction of transport of an ion (or charged molecule) across a membrane. It represents the combined influence of the ion's concentration gradient across the membrane and the membrane potential.
electron carrier Any molecule or atom that accepts electrons from donor molecules and transfers them to acceptor molecules in coupled oxidation and reduction reactions. (Table 12-4)
electron transport Flow of electrons via a series of electron carriers from reduced electron donors (e.g., NADH) to $\mathrm{O}_{2}$ in the inner mitochondrial membrane, or from $\mathrm{H}_{2} \mathrm{O}$ to $\mathrm{NADP}^{+}$in the thylakoid membrane of plant chloroplasts. (Figures 12-19 and 12-38)
electron-transport chain Set of four large multiprotein complexes in the inner mitochondrial membrane plus diffusible cytochrome $c$ and coenzyme Q through which electrons flow from reduced electron donors (e.g., NADH) to $\mathrm{O}_{2}$. Each member of the chain contains one or more bound electron carriers. (Figure 12-22)
electrophoresis Any of several techniques for separating macromolecules based on their migration in a gel or other medium subjected to a strong electric field. (Figure 3-38)

elongation, transcription Addition of nucleotides to a growing polynucleotide chain, as templated by a complementary DNA coding strand. (Figure 5-11)
elongation factor (EF) One of a group of nonribosomal proteins required for continued translation of mRNA (protein synthesis) following initiation. (Figure 5-25)
embryonic stem (ES) cells A line of cultured cells derived from very early embryos that can differentiate into a wide range of cell types either in vitro or after reinsertion into a host embryo. (Figure 21-5)
ENCODE (Encyclopedia of DNA Elements) A comprehensive, publicly available database of human DNA control elements and the transcription factors that bind to them in different cell types, histone post-translational modifications mapped by ChIP-seq and other related methods, DNase I hypersensitive sites, and regulatory lncRNAs and their sites of association in the genome, as well as newly discovered regulatory elements "that control cells and circumstances in which a gene is active."
endergonic Referring to reactions and processes that have a positive $G$ and thus require an input of free energy in order to proceed; opposite of exergonic.
endocrine Referring to signaling mechanism in which target cells bind and respond to a hormone released into the blood by distant specialized secretory cells usually present in a gland (e.g., pituitary or thyroid gland).
endocytic pathway Cellular pathway involving. receptor-mediated endocytosis that internalizes extracellular materials too large to be imported by membrane transport proteins and to remove receptor proteins from the cell surface as a way to down-regulate their activity. (Figure 14-29)
endocytosis General term for uptake of extracellular material by invagination of the plasma membrane; includes receptormediated endocytosis, phagocytosis, and pinocytosis.
endoplasmic reticulum (ER) Network of interconnected membranous structures within the cytoplasm of eukaryotic cells contiguous with the outer nuclear envelope. The rough $E R$, which is associated with ribosomes, functions in the synthesis and processing of secreted and membrane proteins; the smooth $E R$, which lacks ribosomes, functions in lipid synthesis. (Figure 1-12)
endosome One of two types of membrane-bounded compartments: early endosomes (or endocytic vesicles), which bud off from the plasma membrane during receptor-mediated endocytosis, and late endosomes, which have an acidic internal pH and function in sorting of proteins to lysosomes. (Figures 14-1 and 14-29)
endosymbiont Bacterium that resides inside a eukaryotic cell in a mutually beneficial partnership. According to the endosymbiont hypothesis, both mitochondria and chloroplasts evolved from endosymbionts. (Figure 12-7)
endothelium The thin layer of cells that lines the interior surface of blood and lymphatic vessels.
endothermic Referring to reactions and processes that have a positive change in enthalpy, $\Delta H$, and thus must absorb heat in order to proceed; opposite of exothermic.
energy charge A measure of the fraction of total adenosine phosphates that have "high-energy" phosphoanhydride bonds, which is equal to ([ATP] + 0.5 [ADP])/([ATP] + [ADP] + [AMP]).
enhancer A regulatory sequence in eukaryotic DNA that may be located at a great distance from the gene it controls or even within the coding sequence. Binding of specific proteins to an enhancer modulates the rate of transcription of the associated gene. (Figure 9-23)
enhancesome Large nucleoprotein complex that assembles from transcription factors (activators and repressors) as they bind cooperatively to their binding sites in an enhancer with the assistance of DNA-bending proteins. (Figure 9-34)
enthalpy (H) Heat; in a chemical reaction, the enthalpy of the reactants or products is equal to their total bond energies.
entropy ( $\boldsymbol{S}$ ) A measure of the degree of disorder or randomness in a system; the higher the entropy, the greater the disorder.
envelope See nuclear envelope or viral envelope.
enzyme A protein that catalyzes a particular chemical reaction involving a specific substrate or small number of related substrates.
epidermal growth factor (EGF) One of a family of secreted signaling proteins (the EGF family) that is used in the development of most tissues in most or all animals. EGF signals are bound by receptor tyrosine kinases. Mutations in EGF signal transduction components are implicated in human cancer, including brain cancer. See HER family.
epigenetic Referring to a process that affects the expression of specific genes and is inherited by daughter cells but does not involve a change in DNA sequence.
epinephrine A catecholamine secreted by the adrenal gland and some neurons in response to stress; also called adrenaline. It functions as both a hormone and neurotransmitter, mediating "fight or flight" responses (e.g., increased blood glucose levels and heart rate).
epithelial-to-mesenchymal transition (EMT) Describes a developmental program during which epithelial cells acquire the characteristics of mesenchymal cells. Cells lose adhesive properties and acquire motility.
epithelium (pl. epithelia) Sheet-like covering, composed of one or more layers of tightly adhering cells, on external and internal body surfaces. (Figure 20-10)
epitope The part of an antigen molecule that binds to an anti-gen-specific receptor on B or T cells or to antibody. Large protein antigens usually possess multiple epitopes that bind to antibodies of different specificity.
equilibrium constant $\left(K_{\mathrm{eq}}\right)$ Ratio of forward and reverse rate constants for a reaction. For a binding reaction, $\mathrm{A}+\mathrm{B} \rightleftarrows \mathrm{AB}$, the association constant $\left(K_{\mathrm{a}}\right)$ equals $K$, and the dissociation constant $\left(K_{\mathrm{d}}\right)$ equals $1 / K$.
erythropoietin (Epo) A cytokine that triggers production of red blood cells by inducing the proliferation and differentiation of erythroid progenitor cells in the bone marrow. (Figures 16-8 and 21-23)
E2F transcription factor complex Transcription factor that promotes the transcription of $\mathrm{G}_{1} / \mathrm{S}$ phase cyclins and many other genes whose function is required for $S$ phase.
euchromatin Less condensed portions of chromatin present in interphase chromosomes; includes most transcriptionally active regions. See also heterochromatin. (Figure 8-28a)

eukaryotes Class of organisms, composed of one or more cells containing a membrane-enclosed nucleus and organelles, that constitutes one of the three distinct evolutionary lineages of modernday organisms; also called eukarya. Includes all organisms except viruses and prokaryotes. (Figure 1-1)
eukaryotic translation initiation factors (eIFs) Proteins required to initiate protein synthesis in eukaryotic cells. (Figure 5-24)
excision-repair system, DNA One of several mechanisms for repairing DNA damage due to spontaneous depurination or deamination or exposure to carcinogens. These repair systems normally operate with a high degree of fidelity and their loss is associated with increased risk for certain cancers.
excitatory synapse A synapse in which the neurotransmitter induces a depolarization of the postsynaptic cell, favoring generation of an action potential.
exergonic Referring to reactions and processes that have a negative $\Delta G$ and thus release free energy as they proceed; opposite ofendergonic.
exocytosis Release of intracellular molecules (e.g., hormones, matrix proteins) contained within a membrane-bounded vesicle by fusion of the vesicle with the plasma membrane of a cell.
exon Segment of a eukaryotic gene (or of its primary transcript) that reaches the cytoplasm as part of a mature mRNA, rRNA, or tRNA molecule. See also intron.
exon-junction complex A protein complex (EJC) that assembles at exon-exon junctions following pre-mRNA splicing. EJCs stimulate nuclear export of fully processed nuclear mRNPs and participate in the process of nonsense-mediated decay of improperly processed mRNAs.
exon shuffling Evolutionary process for creating new genes (i.e., new combinations of exons) from preexisting ones by recombination between introns of two separate genes or by transposition of mobile DNA elements. (Figures 8-18 and 8-19)
exoplasmic face The face of a cell membrane directed away from the cytosol. (Figure 7-5)
exosome Large exonuclease-containing complex that degrades spliced-out introns and improperly processed pre-mRNAs in the nucleus or mRNAs with shortened poly(A) tails in the cytoplasm. (Figure 10-1)
exothermic Referring to reactions and processes that have a negative change in enthalpy, $\Delta H$, and thus release heat as they proceed; opposite of endothermic.
expression vector A modified plasmid or virus that carries a gene or cDNA into a suitable host cell and there directs synthesis of the encoded protein; used to screen a DNA library for a gene of interest or to produce large amounts of a protein from its cloned gene (Figures 6-28 and 6-29)
extracellular matrix (ECM) A complex interdigitating meshwork of proteins and polysaccharides secreted by cells into the spaces between them. It provides structural support in tissues and can affect the development and biochemical functions of cells. (Table 20-1)

## $\mathrm{F}_{0} \mathrm{~F}_{1}$ complex See ATP synthase.

facilitated transport Protein-aided transport of an ion or small molecule across a cell membrane down its concentration gradient
at a rate greater than that obtained by simple diffusion; also called facilitated diffusion. (Table 11-1)
FAD (flavin adenine dinucleotide) A small organic molecule that functions as an electron carrier by accepting two electrons from a donor molecule and two $\mathrm{H}^{+}$from the solution. (Figure 2-33b)
fatty acid Any long hydrocarbon chain that has a carboxyl group at one end; a major source of energy during metabolism and a precursor for synthesis of phospholipids, triglycerides, and cholesteryl esters. (Figure 2-21; Table 2-4)
fermentation The conversion of some of the energy in organic molecule nutrients such as glucose into ATP via their oxidation into organic molecule "waste" products such as lactic acid or ethanol, typically involving the simultaneous cyclical reduction and oxidation of $\mathrm{NAD}^{+} / \mathrm{NADH}$.
FG-nucleoporins Proteins on the inner surface of the nuclear pore complex with a globular domain that forms part of the pore structure and a random coil domain of hydrophilic amino acids punctuated by short repeats rich in phenylalanine and glycine. (Figure 8-20)
fibroblast A common type of connective tissue cell that secretes collagen and other components of the extracellular matrix; migrates and proliferates during wound healing and in tissue culture.
fibronectin An abundant multi-adhesive matrix protein that occurs in numerous isoforms, generated by alternative splicing, in various cell types. Binds many other components of the extracellular matrix and to integrin adhesion receptors. (Figure 20-33)
FISH See fluorescence in situ hybridization.
flagellum (pl. flagella) Long locomotory structure (usually one per cell) extending from the surface of some eukaryotic cells (e.g.,sperm), whose whiplike bending propels the cell through a fluid medium. Bacterial flagella are smaller and much simpler structures. See also axoneme and cilium. (Figure 18-31)
flavin adenine dinucleotide See FAD.
flippase Protein that facilitates the movement of membrane lipids from one leaflet to the other leaflet of a phospholipid bilayer. (Figure 11-15)
fluorescence in situ hybridization (FISH) Any of several related techniques for detecting specific DNA or RNA sequences in cells and tissues by treating samples with fluorescent probes that hybridize to the sequence of interest and observing the samples by fluorescence microscopy.
fluorescent staining General technique for visualizing cellular components by treating cells or tissues with a fluorescent dyelabeled agent (e.g., antibody) that binds specifically to a component of interest and observing the sample by fluorescence microscopy.
free energy ( $G$ ) A measure of the potential energy of a system, which is a function of the enthalpy $(H)$ and entropy $(S)$.
functional complementation Procedure for screening a DNA library to identify the wild-type gene that restores the function of a defective gene in a particular mutant. (Figure 6-16)
$\mathrm{G}_{0}, \mathrm{G}_{1}, \mathrm{G}_{2}$ phase See cell cycle.
$\mathrm{G}_{1} \mathrm{CDKs}$ Cyclin-CDK complexes that promote entry into the cell cycle.

$\mathrm{G}_{1} / \mathrm{S}$ phase CDKs Cyclin-CDK complexes that promote entry into the cell cycle together with $\mathrm{G}_{1}$ CDKs.
gamete Specialized haploid cell (in animals either a sperm or an egg) produced by meiosis of precursor germ cells; in sexual reproduction, union of a sperm and an egg initiates the development of a new individual.
gap junction Protein-lined channel connecting the cytoplasms of adjacent animal cells that allows passage of ions and small molecules between the cells. See also plasmodesmata. (Figure 20-21)
gene Physical and functional unit of heredity, which carries information from one generation to the next. In molecular terms, it is the entire DNA sequence-including exons, introns, and tran-scription-control regions-necessary for production of a functional polypeptide or RNA. See also transcription unit.
gene conversion A type of DNA recombination in which one DNA sequence is converted to the sequence of a second homologous DNA sequence in the same cell.
gene control All of the mechanisms involved in regulating gene expression. Most common is regulation of transcription, although mechanisms influencing the processing, stabilization, and translation of mRNAs help control expression of some genes.
gene expression Overall process by which the information encoded in a gene is converted into an observable phenotype (most commonly production of a protein).
gene family Set of genes that arose by duplication of a common ancestral gene and subsequent divergence due to small changes in the nucleotide sequence. (Figure 8-21)
gene knockout Selective inactivation of a specific gene by replacing it with a nonfunctional (disrupted) allele in an otherwise normal organism.
genetic code The set of rules whereby nucleotide triplets (codons) in DNA or RNA specify amino acids in proteins. (Table 5-1)
genetic complementation Restoration of a wild-type function in diploid heterozygous cells generated from haploid cells, each of which carries a mutation in a different gene whose encoded protein is required for the same biochemical pathway. Complementation analysis can determine if recessive mutations in two mutants with the same mutant phenotype are in the same or different genes. (Figure 6-7)
genome Total genetic information carried by a cell or organism.
genome maintenance genes Genes that detect or repair DNA damage.
genome-wide association study (GWAS) A statistical method based on linkage disequilibrium of identifying genes for human diseases or other traits that exhibit genetic heterogeneity or may be polygenic.
genomics Comparative analyses of the complete genomic sequences from different organisms and determination of global patterns of gene expression; used to assess evolutionary relations among species and to predict the number and general types of RNAs produced by an organism.
genotype Entire genetic constitution of an individual cell or organism, usually with emphasis on the particular alleles at one or more specific loci.
germ cell In sexually reproducing organisms, any cell that can potentially contribute to the formation of offspring including gametes and their immature precursors; also called germ-line cell. See also somatic cell.
glia Supporting cells of nervous tissue that, unlike neurons, do not conduct electrical impulses; also called glial cells. Of the four types, Schwann cells and oligodendrocytes produce myelin sheaths, astrocytes function in synapse formation, and microglia make trophic factors and serve in immune responses. (Figure 22-17)
glucagon A peptide hormone produced in the cells of pancreatic islets that triggers the conversion of glycogen to glucose by the liver; acts with insulin to control blood glucose levels.
GLUT proteins A family of transmembrane proteins, containing 12 membrane-spanning $\alpha$ helices, that transport glucose (and a few other sugars) across cell membranes down its concentration gradient. (Figure 11-5)
GLUT4 storage vesicle (GSV) An intracellular vesicle with GLUT4 transporters in its membrane. Upon insulin stimulation, GSVs fuse with the cell membrane, exposing GLUT4s to the extracellular space, from which they can transport glucose into the cytosol.
glycogen A very long, branched polysaccharide, composed exclusively of glucose units, that is the primary storage carbohydrate in animals; found primarily in liver and muscle cells.
glycolipid Any lipid to which a short carbohydrate chain is covalently linked; commonly found in the plasma membrane.
glycolysis Metabolic pathway in which sugars are degraded anaerobically to lactate or pyruvate in the cytosol with the production of ATP. (Figure 12-3)
glycoprotein Any protein to which one or more oligosaccharide chains are covalently linked. Most secreted proteins and many membrane proteins are glycoproteins.
glycosaminoglycan (GAG) A long, linear, highly charged polymer of a repeating disaccharides in which many residues often are sulfated. GAGs are major components of the extracellular matrix, usually as components of proteoglycans. (Figure 20-29)
glycosidic bond The covalent linkage between two monosaccharide residues formed when a carbon atom in one sugar reacts with a hydroxyl group on a second sugar with the net release of a water molecule (dehydration). (Figure 2-13)
G protein-coupled receptor (GPCR) Member of a large class of cell-surface signaling receptors, including those for epinephrine, glucagon, and yeast mating factors. All GPCRs contain seven transmembrane $\alpha$ helices. Ligand binding leads to activation of a coupled trimeric G protein, thereby initiating intracellular signaling pathways. (Figures 15-14 and 15-15)
Golgi complex Stacks of flattened, interconnected membranebounded compartments (cisternae) in eukaryotic cells that function in processing and sorting of proteins and lipids destined for other cellular compartments or for secretion; also called Golgi apparatus. (Figure 4-32)
granulocyte colony-stimulating factor (G-CSF) A cytokine that induces a granulocyte progenitor cell in the bone marrow to divide and differentiate into granulocytes.

growth factor An extracellular polypeptide molecule that binds to a cell-surface receptor, triggering an intracellular signaling pathway generally leading to cell proliferation.
growth hormone (GH) A cytokine secreted by the anterior pituitary gland that stimulates proliferation of a variety of cells.
GTPase superfamily Group of intracellular switch proteins that cycle between an inactive state with bound GDP and an active state with bound GTP. Includes the G $\alpha$ subunit of trimeric (large) G proteins, monomeric (small) G proteins (e.g., Ras, Rab, Ran, and Rac), and certain elongation factors used in protein synthesis. (Figure 3-34)
haploid Referring to an organism or cell having only one memberof each pair of homologous chromosomes and hence only one copy (allele) of each gene or genetic locus. Gametes and bacterial cells are haploid. See also diploid.
Hedgehog (Hh) A family of secreted signaling proteins that are important regulators of the development of most tissues and organs in diverse animal species. Mutations in Hh signal transduction components are implicated in human cancer and birth defects. The receptor is the Patched transmembrane protein. (Figures 16-32, 16-33 and 16-34)
helicase (1) Any enzyme that moves along a DNA duplex using the energy released by ATP hydrolysis to separate (unwind) the two strands; required for DNA replication. (2) Activity of certain initiation factors that can unwind the secondary structures in mRNA during initiation of translation.
helix-loop-helix, basic (bHLH) A conserved DNA-binding structural motif, consisting of two $\alpha$ helices connected by a short loop, that is found in many dimeric eukaryotic transcription factors. (Figure 9-30d)
helix-turn-helix A structural motif in which two alpha helices are connected by a short stretch of connecting residues, sometimes also called a "loop." Helix-turn-helix/helix-loop-helix structural motifs can perform various functions, including binding calcium and binding DNA.
HER family Group of receptors, belonging to the receptor tyrosine kinase (RTK) class, that bind to members of the epidermal growth factor (EGF) family of signaling molecules in humans. Overexpression of HER2 protein is associated with some breast cancers. (Figure 16-17)
heredity The transfer of genetically determined characteristics from one generation to the next.
heterochromatin Regions of chromatin that remain highly condensed and transcriptionally inactive during interphase. (Figure 8-28a)
heterotrimeric G proteins A class of GTPase switch proteins, composed of alpha, beta, and gamma polypeptides, that bind to and are activated by certain cell-surface receptors. When activated, heterotrimeric G proteins release GDP and bind GTP.
heterozygous Referring to a diploid cell or organism having two different alleles of a particular gene.
hexose A six-carbon monosaccharide.
high-energy bond Covalent bond that releases a large amount of energy when hydrolyzed under the usual intracellular conditions.

Examples include the phosphoanhydride bonds in ATP, thioester bond in acetyl CoA, and various phosphate ester bonds.
Hippo pathway A signal transduction pathway that controls cell growth in the context of tissues.
histone One of several small, highly conserved basic proteins, found in the chromatin of all eukaryotic cells, that associate with DNA in the nucleosome. (Figure 8-24)
Holliday structure An intermediate in DNA recombination with four DNA strands. (Figure 5-42)
homeodomain Conserved DNA-binding structural motif (a helix-turn-helix) found in many developmentally important transcription factors.
homologous See homologs.
homologous chromosome One of the two copies of each morphologic type of chromosome present in a diploid cell; also called homolog. Each homolog is derived from a different parent.
homologous recombination See recombination.
homolog A protein that shares a common ancestor, and therefore is similar in sequence and/or structure, with another protein.
homologs Maternal and paternal copies of each morphologic type of chromosome present in a diploid cell; also called homologues.
homology Similarity in characteristics (e.g., protein and nucleic acid sequences or the structure of an organ) that reflects a common evolutionary origin. Proteins or genes that exhibit homology are said to be homologous and sometimes are called homologs. In contrast, analogy is a similarity in structure or function that does not reflect a common evolutionary origin.
homozygous Referring to a diploid cell or organism having two identical alleles of a particular gene.
hormone Generally, any extracellular substance that induces specific responses in target cells; specifically, those signaling molecules that circulate in the blood and mediate endocrine signaling.
hyaluronan A large, highly hydrated glycosaminoglycan (GAG) that is a major component of the extracellular matrix; also called hyaluronic acid and hyaluronate. It imparts stiffness and resilience as well as a lubricating quality to many types of connective tissue. (Figure 20-29a)
hybridization, nucleic acid Association of two complementary nucleic acid strands to form double-stranded molecules, which can contain two DNA strands, two RNA strands, or one DNA and one RNA strand. Used experimentally in various ways to detect specific DNA or RNA sequences.
hybridoma A clone of hybrid cells that are immortal and produce monoclonal antibody; formed by fusion of a normal anti-body-producing B cell with a myeloma cell. (Figure 4-6)
hydrocarbon Any compound containing only carbon and hydrogen atoms.
hydrogen bond A noncovalent interaction between an atom (commonly oxygen or nitrogen) carrying a partial negative charge

and a hydrogen atom carrying a partial positive charge. Important in stabilizing the conformation of proteins and in formation of base pairs between nucleic acid strands. (Figure 2-8)
hydrophilic Interacting effectively with water. See also polar.
hydrophobic Not interacting effectively with water; in general, poorly soluble or insoluble in water. See also nonpolar.
hydrophobic effect The tendency of nonpolar molecules or parts of molecules to associate with each other in aqueous solution so as to minimize their direct interactions with water; commonly called a hydrophobic interaction or bond. (Figure 2-11)
hyperpolarization Increase in the magnitude of the cytosolicface negative electric potential that normally exists across the plasma membrane of a cell at rest, resulting in a more negative membrane potential.
hypertonic Referring to an external solution whose solute concentration is high enough to cause water to move out of cells due to osmosis.
hypotonic Referring to an external solution whose solute concentration is low enough to cause water to move into cells due to osmosis.

IgCAMs A family of cell-adhesion molecules that contain multiple immunoglobulin (Ig) domains and mediate $\mathrm{Ca}^{2+}$-independent cell-cell interactions. IgCAMs are produced in a variety of tissues and are components of tight junctions. (Figure 20-2)
immunoblotting Technique in which proteins separated by electrophoresis are attached to a nitrocellulose or other membrane, and specific proteins then are detected by use of labeled antibodies; also called Western blotting.
immunoglobulin (Ig) Any of the serum proteins, produced by fully differentiated B cells, that can function as antibodies; also occur in membrane-bound form as part of the B-cell receptor. Immunoglobulins are divided into five main classes (isotypes) that exhibit distinct functional properties. See also antibody. (Figures 23-9 and 23-10)
immunoglobulin (Ig) fold Evolutionarily ancient structural motif found in antibodies, the T-cell receptor, and numerous other eukaryotic proteins not directly involved in antigen-specific recognition; also called Ig domain. (Figure 23-13b)
immunoprecipitation (IP) A technique that uses antibodies to separate a target molecule of interest from other molecules in a complex mixture in solution by cross-linking the target molecule into a large aggregate, resulting in the formation of an insoluble solid (precipitate) that can be easily separated and analyzed.
induced pluripotent stem (iPS) cells A mammalian cell with properties of an embryonic stem cell that is formed from a differentiated cell type by expression of one or more transcription factors or other genes that confer pluripotency.
inflammation Localized response to injury or infection that leads to the activation of immune-system cells and their recruitment to the affected site; marked by the four classical signs of redness, swelling, heat, and pain. (Figure 23-7)
inhibitory synapse A synapse in which the neurotransmitter induces a hyperpolarization of the postsynaptic cell, inhibiting generation of an action potential.
initiation, transcription The process by which an RNA polymerase separates DNA strands and synthesizes the first phosphodiester bond of an RNA chain as templated by the DNA strand that enters the RNA polymerase active site. (Figure 5-11)
initiation factor (IF) One of a group of nonribosomal proteins that promote the proper association of ribosomes and mRNA and are required for initiation of translation (protein synthesis). (Figure 5-24)
initiator A DNA sequence that specifies transcription initiation within the sequence.
inner cell mass (ICM) The part of an early embryo that will form the embryo proper but not the extra-embryonic tissues, including the placenta.
inner mitochondrial membrane The highly invaginated membrane that lies immediately underneath the outer mitochondrial membrane, and that comprises the boundary membrane, cristae, and crista junctions.
inositol 1,4,5-trisphosphate $\left(\mathrm{IP}_{3}\right)$ Intracellular second messenger produced by cleavage of the membrane lipid phosphatidylinositol 4,5-bisphosphate in response to stimulation of certain cell-surface receptors. $\mathrm{IP}_{3}$, which triggers release of $\mathrm{Ca}^{2+}$ stored in the endoplasmic reticulum, is one of several biologically active phosphoinositides. (Figure 15-6; Table 15-4)
in situ hybridization Any technique for detecting specific DNA or RNA sequences in cells and tissues by treating samples with single-stranded RNA or DNA probes that hybridize to the sequence of interest. (Figure 6-25)
insulin A protein hormone produced in the $\beta$ cells of the pancreatic islets that stimulates uptake of glucose into muscle and fat cells; acts with glucagon to help regulate blood glucose levels. Insulin also functions as a growth factor for many cells.
integral membrane protein Any protein that contains one or more hydrophobic segments embedded within the core of the phospholipids bilayer; also called transmembrane protein. (Figure 13-10)
integrins A large family of heterodimeric transmembrane proteins that function as adhesion receptors, promoting cell-matrix adhesion, or as cell-adhesion molecules, promoting cell-cell adhesion. (Table 20-4)
interferons (IFNs) Small group of cytokines that bind to cellsurface receptors on target cells inducing changes in gene expression that lead to an antiviral state or other cellular responses important in immune responses.
interleukins (ILs) Large group of cytokines, some released in response to inflammation, that promote proliferation and functioning of T cells and antibody-producing B cells of the immune system.
intermediate filament Cytoskeletal fiber ( 10 nm in diameter) formed by polymerization of related, but tissue-specific, subunit proteins, including keratins, lamins, and neurofilaments. (Figure 18-47; Table 18-1)
intermembrane space The mitochondrial compartment between the inner and outer mitochondrial membranes, which is continuous with the spaces inside the cristae.
interneurons Nerves that receive signals from other nerve cells and that in turn transmit signals to other nerve cells.

interphase Long period of the cell cycle, including the $\mathrm{G}_{1}, \mathrm{~S}$, and $\mathrm{G}_{2}$ phases, between one M (mitotic) phase and the next. (Figures $1-16$ and 19-1)
interspersed repeats Sequences from transposons that occur at multiple sites throughout the genomes of multicellular animals and plants. (Figure 8-8; Table 8-1)
intron Part of a primary transcript (or the DNA encoding it) that is removed by splicing during RNA processing and is not included in the mature, functional mRNA, rRNA, or tRNA.
in vitro Referring to experiments or manipulations performed outside a cell (including cell fragments, lysates, or purified molecules) or to cells placed in an artificial environment such as in a petri dish or test tube; literally, in glass.
in vivo Referring to experiments or manipulations performed in the context of an intact organism or intact cell, in contrast to experiments using cell fragments, lysates, or purified molecules; literally, in the living.
ionic interaction A noncovalent interaction between a positively charged ion (cation) and negatively charged ion (anion); commonly called ionic bond.
$\mathrm{IP}_{3}$ See inositol 1,4,5-trisphosphate.
isoelectric point ( $\mathbf{p l}$ ) The pH of a solution at which a dissolved protein or other potentially charged molecule has a net charge of zero and therefore does not move in an electric field. (Figure 3-39)
isoform One of several forms of the same protein whose amino acid sequences differ slightly and whose general activities are similar. Isoforms may be encoded by different genes or by a single gene whose primary transcript undergoes alternative splicing.
isotonic Referring to a solution whose solute concentration is such that it causes no net movement of water in or out of cells.

JAK kinase A class of protein tyrosine kinases that are bound to the cytosolic domain of cytokine receptors and are activated following cytokine binding.
JAK/STAT pathway A cell signaling pathway used by several cytokine receptors, in which a JAK kinase phosphorylates a receptor-bound STAT transcription factor, inducing its movement into the nucleus, where it activates transcription.
karyopherin One of a family of nuclear transport proteins that functions as an importin, exportin, or occasionally both. Each karyopherin binds to a specific signal sequence in cargo proteins moving in or out of the nucleus.
karyotype Number, sizes, and shapes of the entire set of metaphase chromosomes of a eukaryotic cell. (Chapter 8 opening figure)
keratins A group of intermediate filament proteins found in epithelial cells that assemble into heteropolymeric filaments. (Figure 18-49)
kinase An enzyme that transfers the terminal ( $\gamma$ ) phosphate group from ATP to a substrate. Protein kinases, which phosphorylate specific serine, threonine, or tyrosine residues, play a critical role in regulating the activity of many cellular proteins. See also phosphatases. (Figure 3-33)
kinesins A class of motor proteins that use energy released by ATP hydrolysis to move toward the $(+)$ end of a microtubule. Kinesins can transport vesicles and organelles and play a role in chromosome movement during mitosis. (Figures 18-18 through 18-20)
kinetic energy Energy of movement, such as the motion of molecules.
kinetochore A multilayer protein structure located at or near the centromere of each mitotic chromosome from which microtubules extend toward the spindle poles of the cell; plays an active role in movement of chromosomes toward the poles during anaphase. (Figure 18-40)
$K_{\mathrm{m}} \quad$ A parameter that describes the affinity of an enzyme for its substrate and equals the substrate concentration that yields the half-maximal reaction rate; also called the Michaelis constant. A similar parameter describes the affinity of a transport protein for the transported molecule or the affinity of a receptor for its ligand. (Figure 3-24)
knockdown, siRNA See siRNA knockdown.
knockout, gene See gene knockout.
lagging strand One of the two daughter DNA strands formed at a replication fork as short, discontinuous segments (Okazaki fragments), which are synthesized in the $5^{\prime} \rightarrow 3^{\prime}$ direction and later joined. See also leading strand. (Figure 5-29)
laminin Large heterotrimeric multi-adhesive matrix protein that is found in all basal lamina. (Figure 20-23)
lamins A group of intermediate filament proteins that form a fibrous network, the nuclear lamina, on the inner surface of the nuclear envelope.
late endosome See endosome.
lateral See basolateral.
lateral inhibition Important signal-mediated developmental process that results in adjacent equivalent or near-equivalent cells assuming different fates.
leading strand One of the two daughter DNA strands formed at a replication fork by continuous synthesis in the $5^{\prime} \rightarrow 3^{\prime}$ direction. The direction of leading-strand synthesis is the same as movement of the replication fork. See also lagging strand. (Figure 5-29)
lectin Any protein that binds tightly to specific sugars. Lectins assist in the proper folding of some glycoproteins in the endoplasmic reticulum and can be used in affinity chromatography to purify glycoproteins or as reagents to detect them in situ.
leucine zipper A type of coiled-coil structural motif composed of two a helices that form specific homo- or heterodimers; common motif in many eukaryotic transcription factors. See coiled coil. (Figures 9-30c and 3-10)
ligand Any molecule, other than an enzyme substrate, that binds tightly and specifically to a macromolecule, usually a protein, forming a macromolecule-ligand complex.
linkage In genetics, the tendency of two different loci on the same chromosome to be inherited together. The closer two loci are, the lower the frequency of recombination between them and the greater their linkage.

lipid Any organic molecule that is poorly soluble or virtually insoluble in water but is soluble in nonpolar organic solvents. Major classes include fatty acids, phospholipids, steroids, and triglycerides.
lipid-anchored membrane protein Any protein that is tethered to a cellular membrane by one or more covalently attached lipid groups, which are embedded in the phospholipids bilayer. (Figure 10-19)
lipid raft Microdomain in the plasma membrane that is enriched in cholesterol, sphingomyelin, and certain proteins.
lipoprotein Any large, water-soluble protein and lipid complex that functions in mass transfer of lipids throughout the body. See also low-density lipoprotein (LDL).
liposome Artificial spherical phospholipid bilayer structure with an aqueous interior that forms in vitro from phospholipids and may contain membrane proteins. (Figure 7-3c)
long interspersed elements (LINEs) Abundant mobile elements in mammals generated by retrotransposons lacking long-terminal repeats. (Figure 8-17)
long noncoding RNA (lncRNAs) RNA molecules of many kb in length that do not encode open reading frames. Some lncRNAs function in repression of gene transcription by forming a scaffold to which several proteins bind, forming an RNA-protein complex that affects chromatin structure.
long terminal repeats (LTRs) Direct repeat sequences, containing up to 600 base pairs, that flank the coding region of integrated retroviral DNA and viral retrotransposons.
low-density lipoprotein (LDL) A class of lipoprotein, containing apolipoprotein B-100, that is a primary transporter of cholesterol in the form of cholesteryl esters between tissues, especially to the liver. (Figure 14-27)
lumen The aqueous interior of an organelle.
lymphocytes Two classes of white blood cells that can recognize foreign molecules (antigens) and mediate immune responses. B lymphocytes (B cells) are responsible for production of antibodies; T lymphocytes ( T cells) are responsible for destroying virus and bacteria-infected cells, foreign cells, and cancer cells.
lysis Destruction of a cell by rupture of the plasma membrane and release of the contents.
lysogeny Phenomenon in which the DNA of a bacterial virus (bacteriophage) is incorporated into the host-cell genome and replicated along with the bacterial DNA but is not expressed. Subsequent activation leads to formation of new viral particles, eventually causing lysis of the cell.
lysosome Small organelle that has an internal pH of $4-5$, contains hydrolytic enzymes, and functions in degradation of materials internalized by endocytosis and of cellular components in autophagy. (Figures 1-12 and 4-13)

## M (mitotic) phase See cell cycle.

macromolecule Any large, usually polymeric molecule (e.g., a protein, nucleic acid, polysaccharide) with a molecular mass greater than a few thousand daltons.
macrophages Phagocytic leukocytes that can detect broad patterns of pathogen markers via Toll-like receptors. They function
as professional antigen-presenting cells and are a major source of cytokines.
major histocompatibility complex (MHC) Set of adjacent genes that encode class I and class II MHC molecules and other proteins required for antigen presentation, as well as some complement proteins; called the H-2 complex in mice and the HLA complex in humans. (Figure 23-21)
malignant Referring to a tumor or tumor cells that can invade surrounding normal tissue and/or undergo metastasis. See also benign.
MAP kinase Any of a family of protein kinases that are activated in response to cell stimulation by many different growth factors and that mediate cellular responses by phosphorylating specific transcription factors and other target proteins. (Figures $16-25$ and $16-26$ )
matrix The lumen of the innermost compartment of the mitochondrion; also the fibrous proteins and carbohydrates external to a cell (called the extracellular matrix).
matrix metalloproteases (MMPs) Matrix metalloproteases (MMPs) are proteolytic enzymes that employ the metal zinc in their active sites. They operate in the extracellular space, where they cut proteins in the extracellular matrix and sometimes other proteins (e.g., some cell-surface receptors).
maturation-promoting factor (MPF) Cyclin-CDK complex that has the ability to induce entry into meiosis when injected into $\mathrm{G}_{2}$-resting oocytes.
maximal velocity See $V_{\max }$.
mechanosensor Any of several types of sensory structures that are embedded in various tissues and respond to touch, the positions and movements of the limbs and head, pain, and temperature.
Mediator A very large multiprotein complex that forms a molecular bridge between transcriptional activators bound to an enhancer and to RNA polymerase II bound at a promoter; functions as a coactivator in stimulating transcription. (Figures 9-39 and 9-40)
meiosis In eukaryotes, a special type of cell division that occurs during maturation of germ cells; comprises two successive nuclear and cellular divisions with only one round of DNA replication. Results in production of four genetically nonequivalent haploid cells (gametes) from an initial diploid cell. (Figure 6-3)
melting See denaturation.
membrane potential Electric potential difference, expressed in volts, across a membrane due to the slight excess of positive ions (cations) on one side and negative ions (anions) on the other. (Figures 11-18 and 11-19)
membrane transport protein Collective term for any integral membrane protein that mediates movement of one or more specific ions or small molecules across a cellular membrane regardless of the transport mechanism. (Figure 11-2)
memory The ability of an antigen-experienced immune system to respond more rapidly to a reexposure to that same antigenic stimulus.
meristem Organized group of undifferentiated, dividing cells that are maintained at the tips of growing shoots and roots in plants. All the adult structures arise from meristems.

merotelic attachment Indicates that a single kinetochore attaches to microtubules emanating from two opposite spindle poles.
mesenchymal stem cell A class of stem cells in the bone marrow that can differentiate into fat cells, osteoblasts (bone-forming cells), and cartilage-producing cells; some may also produce muscle and other types of differentiated cells.
mesenchyme Immature embryonic connective tissue, composed of loosely organized and loosely attached cells, derived from either the mesoderm or ectoderm in animals.
mesoderm The middle of the three primary cell layers of the animal embryo, lying between the ectoderm and endoderm; gives rise to the notochord, connective tissue, muscle, blood, and other tissues.
messenger RNA See mRNA.
metaphase Stage of mitosis at which condensed chromosomes are aligned equidistant between the poles of the mitotic spindle but have not yet started to segregate toward the spindle poles. (Figure 18-37)
metastasis Spread of cancer cells from their site of origin and establishment of areas of secondary growth.
metazoans A subset of the animal kingdom that includes all multicellular animals with differentiated tissues, such as nerves and muscles.

## MHC See major histocompatibility complex.

MHC molecules Glycoproteins that display peptides, derived from foreign (and self) proteins, on the surface of cells and are required for antigen presentation to T cells. Class I molecules are expressed constitutively by nearly all nucleated cells; class II molecules, by professional antigen-presenting cells. (Figures 23-21 and 23-22)
micelle A water-soluble spherical aggregate of phospholipids or other amphipathic molecules that form spontaneously in aqueous solution. (Figure 10-3c)
Michaelis constant See $K_{\mathrm{m}}$.
microfilament Cytoskeletal fiber ( $\approx 7 \mathrm{~nm}$ in diameter) that is formed by polymerization of monomeric globular (G) actin; also called actin filament. Microfilaments play an important role in muscle contraction, cytokinesis, cell movement, and other cellular functions and structures. (Figure 17-4)
micro-RNA See miRNA.
microsatellites Simple-sequence repeated DNA sequences 1-13 bases long (most are $<5 \mathrm{bp}$ ) that occur in the human genome in tandem repeats of 150 repeats or fewer.
microtubule Cytoskeletal fiber ( $\approx 25 \mathrm{~nm}$ in diameter) that is formed by polymerization of $\alpha, \beta$-tubulin monomers and exhibits structural and functional polarity. Microtubules are important components of cilia, flagella, the mitotic spindle, and other cellular structures. (Figures 18-2 and 18-3)
microtubule-associated protein (MAP) Any protein that binds to microtubules and regulates their stability. (Figures 18-13, 18-14, and 18-15)
microtubule-organizing center See MTOC.
microvillus (pl. microvilli) Small, membrane-covered projection on the surface of an animal cell containing a core of actin
filaments. Numerous microvilli are present on the absorptive surface of intestinal epithelial cells, increasing the surface area for transport of nutrients. (Figures 17-4 and 20-10)
miRNA (micro-RNA) Any of numerous small, endogenous cellular RNAs, 20-30 nucleotides long, that are processed from double-stranded regions of hairpin secondary structures in long precursor RNAs. A single strand of the mature miRNA associates with several proteins to form an RNA-induced silencing complex (RISC) that inhibits translation of a target mRNA to which the miRNA hybridizes imperfectly. Several miRNAs must hybridize to a single mRNA to inhibit its translation. See also siRNA. (Figures 10-28a and 10-29)
mitochondria-associated membranes (MAMs) Portions of the endoplasmic reticulum that closely contact regions of mitochondria, influencing mitochondrial shape, function, and fission.
mitochondrion (pl. mitochondria) Large organelle that is surrounded by two phospholipid bilayer membranes, contains DNA, and carries out oxidative phosphorylation, thereby producing most of the ATP in eukaryotic cells. (Figures 1-20 and 12-6)
mitogen Any extracellular molecule, such as a growth factor, that promotes cell proliferation.
mitosis In eukaryotic cells, the process whereby the nucleus divides, producing two genetically equivalent daughter nuclei with the diploid number of chromosomes. See also cytokinesis and meiosis. (Figure 18-36)
mitotic CDKs Cyclin-CDK complexes that promote entry into and progression through mitosis.
mitotic spindle A specialized temporary structure, present in eukaryotic cells during mitosis, that captures the chromosomes and then pushes and pulls them to opposite sides of the dividing cell; also called mitotic apparatus. (Figure 18-37)
mobile DNA element See transposable DNA element.
model organism A non-human species used to study genes, proteins, and cellular functions. Model organisms are chosen based on the expectation that discoveries made via the model organism will provide insight into other organisms.
molecular chaperone See chaperone.
molecular complementarity Lock-and-key kind of fit between the shapes, charges, hydrophobicity, and/or other physical properties of two molecules or portions thereof that allow formation of multiple oncovalent interactions between them at close range. (Figure 2-12)
molecular markers, DNA-based DNA sequences that vary among individuals (DNA polymorphisms) of the same species and are useful in genetic linkage studies; includes SNPs and SSRs.
monoclonal antibody Antibody produced by the progeny of a single B cell and thus a homogeneous protein that recognizes a single antigen (epitope). It can be produced experimentally by use of a hybridoma. (Figure 4-6)
monomer Any small molecule that can be linked chemically with others of the same type to form a polymer. Examples include amino acids, nucleotides, and monosaccharides.
monomeric (small) G protein A monomeric GTPase with a structure similar to that of the Ras protein that changes

conformation when a bound GTP is hydrolyzed to GDP and phosphate. (Figure 15-5)
monopolin complex Protein complex that promotes the co-orientation of sister chromatids during meiosis I in budding yeast.
monosaccharide Any simple sugar with the formula $\left(\mathrm{CH}_{2} \mathrm{O}\right)_{n}$ where $n=3-7$.
monotelic attachment Indicates that only one of the sister kinetochore pair attached to microtubules.
monoubiquitination The covalent addition of a single ubiquitin molecule to a target protein.
morphogen A signaling molecule whose concentration determines the fate of target-differentiating cells.
motor protein Any member of a special class of mechanochemical enzymes that use energy from ATP hydrolysis to generate either linear or rotary motion; also called molecular motor. See also dyneins, kinesins, and myosins.
mRNA (messenger RNA) Any RNA that specifies the order of amino acids in a protein (i.e., the primary structure). It is produced by transcription of DNA by RNA polymerase. In eukaryotes, the initial RNA product (primary transcript) undergoes processing to yield functional mRNA. See also translation. (Figure 5-15)
mRNP-exporter A heterodimeric protein that binds to mRNAcontaining ribonucleoprotein particles (mRNPs) and directs their export from the nucleus to cytoplasm by interacting transiently with nucleoporins in the nuclear pore complex. (Figure 10-24)
MTOC (microtubule-organizing center) General term for any structure (e.g., centrosome, spindle pole, basal body) that organizes microtubules in cells. (Figure 18-5)
mRNA surveillance The processes that lead to the degradation of a pre-mRNA or mRNA that has been improperly processed.
multi-adhesive matrix proteins Group of long flexible proteins that bind to other components of the extracellular matrix and to cell-surface receptors, thereby cross-linking matrix components to the cell membrane. Examples include laminin, a major component of the basal lamina, and fibronectin, present in many tissues.
multimeric For proteins, containing several polypeptide chains (or subunits).
multiubiquitination The covalent addition of several single ubiquitin molecules, each at a distinct site, on a single target protein.
MuSK A receptor tyrosine kinase localized in the myotube plasma membrane that both induces clustering of acetylcholine receptors and serves to attract the termini of growing motor neuron axons.
mutagen A chemical or physical agent that induces mutations.
mutation In genetics, a permanent, heritable change in the nucleotide sequence of a chromosome, usually in a single gene; commonly causes an alteration in the function of the gene product.
myelin sheath Stacked specialized cell membrane that forms an insulating layer around vertebrate axons and increases the speed of impulse conduction. (Figure 22-18)
myofibril Long, slender structures within cytoplasm of muscle cells consisting of a regular repeating array of sarcomeres
composed of thick (myosin) filaments and thin (actin) filaments. (Figure 17-30)
myosins A class of motor proteins that have actin-stimulated ATPase activity. Myosins move along actin microfilaments during muscle contraction and cytokinesis and also mediate vesicle translocation. (Figure 17-22)

NAD $^{+}$(nicotinamide adenine dinucleotide) A small organic molecule that functions as an electron carrier by accepting two electrons from a donor molecule and one $\mathrm{H}^{+}$from the solution. (Figure 2-33a)
$\mathrm{NADP}^{+}$(nicotinamide adenine dinucleotide phosphate) Phosphorylated form of $\mathrm{NAD}^{+}$that is used extensively as an electron carrier in biosynthetic pathways and during photosynthesis.
$\mathrm{Na}^{+} / \mathrm{K}^{+}$ATPase A P-class ATP-powered pump that couples hydrolysis of one ATP molecule to export of $\mathrm{Na}^{+}$ions and import of $\mathrm{K}^{+}$ions; is largely responsible for maintaining the normal intracellular concentrations of $\mathrm{Na}^{+}$(low) and $\mathrm{K}^{+}$(high) in animal cells; commonly called $\mathrm{Na}^{+} / \mathrm{K}^{+}$pump. (Figure 11-13)
natural killer (NK) cells Components of the innate immune system that nonspecifically detect and kill virus-infected cells and tumor cells. (Figure 23-6)
necrosis Cell death resulting from tissue damage or other pathology; usually marked by swelling and bursting of cells with release of their contents. Contrast with apoptosis.
necroptosis A type of necrosis triggered by extracellular hormones that causes inflammation.
neoblasts Stem cells in planaria that can proliferate and regenerate many or all body cells.
negative feedback mechanism Process where the output of a pathway inhibits its own production.
neurofilaments (NFs) A group of intermediate filament proteins, found only in neurons, that contribute to axonal structure and rate of transmission of action potentials down axons. (Figure 18-2b)
neuron (nerve cell) Any of the impulse-conducting cells of the nervous system. A typical neuron contains a cell body; multiple short, branched processes (dendrites); and one long process (axon). (Figure 22-1)
neurotransmitter Extracellular signaling molecule that is released by the presynaptic neuron at a chemical synapse and relays the signal to the postsynaptic cell. The response elicited by a neurotransmitter, either excitatory or inhibitory, is determined by its receptor on the postsynaptic cell. (Figures 22-25 and 22-26)
neurotrophins Family of structurally and functionally related trophic factors that bind to receptors called Trks and are required for survival of neurons; include nerve growth factor (NGF) and brain-derived neurotrophic factor (BDNF).
neutrophils Phagocytic leukocytes that are attracted to sites of tissue damage and migrate into the tissue. Once activated, neutrophils secrete various chemokines, cytokines, bacteria-destroying enzymes (e.g., lysozyme), and other products that contribute to inflammation and help clear invading pathogens.
NF- $\kappa \mathrm{B}$ A transcription factor that is sequestered in the cytosol until its inhibitory protein is degraded.
nicotinamide adenine dinucleotide See NAD ${ }^{+}$.

nicotinamide adenine dinucleotide phosphate See NADP ${ }^{+}$.
N-linked oligosaccharide A branched oligosaccharide chain attached to the side-chain amino group of an asparagine residue in a glycoprotein. See also O-linked oligosaccharide.
nociceptor Mechanosensor that responds to pain associated with injury to body tissues caused by mechanical trauma, heat, electricity, or toxic chemicals.
noncovalent interaction Any relatively weak chemical interaction that does not involve an intimate sharing of electrons. (Figures 2-6 and 2-12)
nondisjunction Mis-segregation of chromosomes during mitosis, resulting in aneuploidy or trisomy.
nonhomologous end joining (NHEJ) A pathway that repairs double-strand breaks in DNA in which the break ends are directly ligated without the need for a homologous template.
nonpolar Referring to a molecule or structure that lacks any net electric charge or asymmetric distribution of positive and negative charges. Nonpolar molecules generally are less soluble in water than polar molecules and are often water insoluble.
nonsense-mediated decay (NMD) The surveillance mechanism by which cells identify and destroy improperly processed or mutant mRNAs with a stop codon that occurs before the last exonexon junction.
Northern blotting Technique for detecting specific RNAs separated by electrophoresis by hybridization to a labeled DNA probe. See also Southern blotting.
nuclear body Roughly spherical, functionally specialized region in the nucleus, containing specific proteins and RNAs; many function in the assembly of ribonucleoprotein (RNP) complexes. The most prominent type is the nucleolus.
nuclear envelope Double-membrane structure surrounding the nucleus; the outer membrane is continuous with the endoplasmic reticulum and the two membranes are perforated by nuclear pore complexes. (Figure 1-12a)
nuclear lamina Fibrous network on the inner surface of the nuclear envelope composed of lamin intermediate filaments. (Figure 19-19)
nuclear pore complex (NPC) Large, multiprotein structure, composed largely of nucleoporins, that extends across the nuclear envelope. Ions and small molecules freely diffuse through NPCs; large proteins and ribonucleoprotein particles are selectively transported through NPCs with the aid of soluble proteins. (Figure 13-33a and b)
nuclear receptor Member of a class of intracellular receptors that bind lipid-soluble molecules (e.g., steroid hormones), forming ligand-receptor complexes that activate transcription; also called steroid receptor superfamily. (Figure 9-45d)
nucleic acid A polymer of nucleotides linked by phosphodiester bonds. DNA and RNA are the primary nucleic acids in cells.
nucleic acid hybridization See hybridization, nucleic acid.
nucleocapsid A viral capsid plus the enclosed nucleic acid.
nucleolus Large structure in the nucleus of eukaryotic cells where rRNA synthesis and processing occurs and ribosome subunits are assembled. (Figure 8-28a)
nucleoporins Large group of proteins that make up the nuclear pore complex. One class (FG-nucleoporins) participates in nuclear import and export.
nucleoside A small molecule composed of a purine or pyrimidine base linked to a pentose (either ribose or deoxyribose). (Table 2-3)
nucleosome Structural unit of chromatin consisting of a diskshaped core of histone proteins around which a 147-bp segment of DNA is wrapped. (Figure 8-24)
nucleotide A nucleoside with one or more phosphate groups linked via an ester bond to the sugar moiety, generally to the $5^{\prime}$ carbon atom. DNA and RNA are polymers of nucleotides containing deoxyribose and ribose, respectively. (Figure 2-16 and Table 2-3)
nucleus Large membrane-bounded organelle in eukaryotic cells that contains DNA organized into chromosomes; synthesis and processing of RNA and ribosome assembly occur in the nucleus.

O-GlcNAcylation The reversible, post-translational, covalent modification of an intracellular protein by the addition to the hydroxyl group of the side chain of a serine or threonine of the sugar N -acetylglucosamine (GlcNAc). O-GlcNAcylation can alter the activity of the modified protein.
Okazaki fragments Short ( $<1000$ bases), single-stranded DNA fragments that are formed during synthesis of the lagging strand in DNA replication and are rapidly joined by DNA ligase to form a continuous DNA strand. (Figure 5-29)
oligopeptide A small to medium-size linear polymer composed of amino acids connected by peptide bonds. The terms peptide and oligospeptide are often used interchangeably.
O-linked oligosaccharide Oligosaccharide chain that is attached to the side-chain hydroxyl group in a serine or threonine residue in a glycoprotein. See also N-linked oligosaccharides.
oncogene A gene whose product is involved either in transforming cells in culture or in inducing cancer in animals. Generally is a mutant form of a normal gene (proto-oncogene) for a protein involved in the control of cell growth or division. (Figure 24-11)
oncogene addiction Describes the observation that some cancers, despite containing numerous genetic abnormalities, depend on only a few genetic alteration to maintain their malignant phenotype. It is said that these cancers are "addicted" to certain oncogenic mutations.
oncoprotein A protein encoded by an oncogene that causes abnormal cell proliferation; may be a mutant unregulated form of a normal protein or a normal protein that is produced in excess or in the wrong time or place in an organism.
oocyte The metazoan egg cell, containing one set of chromosomes from the maternal parent.
open reading frame (ORF) Region of sequenced DNA that is not interrupted by stop codons in one of the triplet reading frames. An ORF that begins with a start codon (usually AUG) and extends for 100 or more codons is called a long open reading frame and has a high probability of encoding a protein.
operator Short DNA sequence in a bacterial or bacteriophage genome that binds a repressor protein and controls transcription of an adjacent gene. (Figure 7-3)

operon In bacterial DNA, a cluster of contiguous genes transcribed from one promoter that gives rise to an mRNA containing coding sequences for multiple proteins. (Figure 5-13a)
optogenetics A technique in which channelrhodopsins are expressed in electrically excitable cells, allowing their membrane potential to be manipulated using light.
organelle Any membrane-limited subcellular structure found in eukaryotic cells. (Figures 1-12, 1-17, 1-19, and 1-20)
osmosis Net movement of water across a semipermeable membrane (permeable to water but not to solute) from a solution of lesser to one of greater solute concentration. (Figure 11-6)
outer mitochondrial membrane The smooth outside boundary of the mitochondrion.
oxidation Loss of electrons from an atom or molecule as occurs when a hydrogen atom is removed from a molecule or oxygen is added; opposite of reduction.
oxidation potential The voltage change when an atom or molecule loses an electron; a measure of the tendency of a molecule to loose an electron. For a given oxidation reaction, the oxidation potential has the same magnitude but opposite sign as the reduction potential for the reverse (reduction) reaction.
oxidative phosphorylation The phosphorylation of ADP to form ATP driven by the transfer of electrons to oxygen $\left(\mathrm{O}_{2}\right)$ in bacteria and mitochondria. Involves generation of a proton-motive force during electron transport and its subsequent use to power ATP synthesis.
p53 protein The product of a tumor-suppressor gene that plays a critical role in the arrest of cells with damaged DNA. Inactivating mutations in the $p 53$ gene are found in many human cancers. (Figure 24-26)
paracrine Referring to signaling mechanism in which a target cell responds to a signaling molecule (e.g., growth factor, neurotransmitter) that is produced by a nearby cell(s) and reaches the target by diffusion.
patch clamping Technique for determining ion flow through a single ion channel or across the membrane of an entire cell by use of a micropipette whose tip is applied to a small patch of the cell membrane. (Figure 11-22)
patterning genes Genes involved in metazoan development that determine the general organization of the animal, including the major body axes and segmentation.
P body Dense cytoplasmic domain, containing no ribosomes or translation factors, that functions in repression of translation and degradation of associated mRNAs; also called cytoplasmic RNAprocessing body.
PCR (polymerase chain reaction) Technique for amplifying a specific DNA segment in a complex mixture by multiple cycles of DNA synthesis from short oligonucleotide primers followed by brief heat treatment to separate the complementary strands. (Figure 6-18)
pentose A five-carbon monosaccharide. The pentoses ribose and deoxyribose are present in RNA and DNA, respectively. (Figure 2-16)
peptide A small linear polymer composed of amino acids connected by peptide bonds. The terms peptide and oligopeptide are often used interchangeably. See also polypeptide.
peptide bond The covalent amide linkage between amino acids formed between the amino group of one amino acid and the carboxyl group of another with the net release of a water molecule (dehydration). (Figure 2-13)
peptidoglycan A polysaccharide chain cross-linked by peptide cross-bridges in a bacterial cell wall that confers rigidity and helps to determine the cell's shape.
pericentriolar material Amorphous material seen by thin-section electron microscopy surrounding the centrioles of animal cells. Pericentriolar material contains many components, including the gamma-tubulin ring complex (gamma-TuRC), which promotes nucleation of microtubule assembly. (Figure 18-6)
peripheral membrane protein Any protein that associates with the cytosolic or exoplasmic face of a membrane but does not enter the hydrophobic core of the phospholipid bilayer. See also integral membrane protein. (Figure 7-1)
perlecan A large multidomain proteoglycan component of the extracellular matrix (ECM) that binds to many ECM components, cell-surface molecules, and growth factors; a major component of the basal lamina.
peroxisome Small organelle that contains enzymes for degrading fatty acids and amino acids by reactions that generate hydrogen peroxide, which is converted to water and oxygen by catalase.
pH A measure of the acidity or alkalinity of a solution defined as the negative logarithm of the hydrogen ion concentration in moles per liter: $\mathrm{pH}=-\log \left[\mathrm{H}^{+}\right]$. Neutrality is equivalent to a pH of 7 ; values below this are acidic and those above are alkaline.
phagocyte Any cell that can ingest and destroy pathogens and other particulate antigens. The primary phagocytes are neutrophils, macrophages, and dendritic cells.
phagocytosis Process by which relatively large particles (e.g., bacterial cells) are internalized by certain eukaryotic cells in a process that involves extensive remodeling of the actin cytoskeleton; distinct from receptor-mediated endocytosis. (Figure 17-19)
phenotype The detectable physical and physiological characteristics of a cell or organism determined by its genotype; also, the specific trait associated with a particular allele.
pheromone A signaling molecule released by an individual that can alter the behavior or gene expression of other individuals of the same species. The yeast $\alpha$ and a mating-type factors are wellstudied examples.
phosphatase An enzyme that removes a phosphate group from a substrate by hydrolysis. Phosphoprotein phosphatases act with protein kinases to control the activity of many cellular proteins. (Figure 3-35)
phosphoanhydride bond A type of high-energy bond formed between two phosphate groups, such as the $\gamma$ and $\beta$ phosphates and the $\beta$ and $\alpha$ phosphates in ATP. (Figure 2-31)
phosphodiester bond Chemical linkage between adjacent nucleotides in DNA and RNA; consists of two phosphoester bonds, one on the $5^{\prime}$ side of the phosphate and another on the $3^{\prime}$ side. (Figure 5-2)
phosphoglycerides Amphipathic derivatives of glycerol 3-phosphate that generally consist of two hydrophobic fatty acyl chains esterified to the hydroxyl groups in glycerol and a polar head

group attached to the phosphate; the most abundant lipids in biomembranes. (Figures 2-20 and 10-8a)
phosphoinositides A group of membrane-bound lipids containing phosphorylated inositol derivatives; some function as second messengers in several signal transduction pathways. (Figures 15-33 and 16-28)
phospholipase One of several enzymes that cleave various bonds in the hydrophilic end of phospholipids. (Figures 7-12)
phospholipase C (PLC) A membrane-associated phospholipase, activated by either $\mathrm{G}_{\mathrm{eq}}$ or $\mathrm{G}_{\mathrm{me}}$, that cleaves the membrane lipid phosphatidylinositol 4,5-bisphosphate to generate two second messengers, DAG and $\mathrm{IP}_{3}$. (Figures 15-33 and 15-34a)
phospholipid The major class of lipids present in biomembranes, including phosphoglycerides and sphingolipids. (Figures $7-8 \mathrm{a}, \mathrm{b}$ and 2-20)
phospholipid bilayer A two-layer, sheet-like structure in which the polar head groups of phospholipids are exposed to the aqueous media on either side and the nonpolar fatty acyl chains are in the center; the foundation for all biomembranes. (Figure 7-3a, b)
phosphorylation The covalent addition of a phosphate group to a molecule such as a sugar or a protein. The hydrolysis of ATP often accompanies phosphorylation, providing energy to drive the reaction and the phosphate group that is covalently added to the target molecule. Enzymes that catalyze phosphorylation are called kinases.
photoelectron transport Light-driven electron transport that generates a charge separation across the thylakoid membrane that drives subsequent events in photosynthesis. (Figure 12-35)
photorespiration A reaction pathway that competes with $\mathrm{CO}_{2}$ fixation (Calvin cycle) by consuming ATP and generating $\mathrm{CO}_{2}$, thus reducing the efficiency of photosynthesis. (Figure 12-49)
photosynthesis Complex series of reactions occurring in some bacteria and in plant chloroplasts in which light energy is used to generate carbohydrates from $\mathrm{CO}_{2}$, usually with the consumption of $\mathrm{H}_{2} \mathrm{O}$ and evolution of $\mathrm{O}_{2}$.
photosystems Multiprotein complexes, present in all photosynthetic organisms, that consist of light-harvesting complexes containing chlorophylls and a reaction center where photoelectron transport occurs. (Figure 12-44a)
phragmoplast In plants, a temporary structure, formed during telophase, whose membranes become the plasma membranes of the daughter cells and whose contents develop into the new cell wall between them. (Figure 18-46)
pI See isoelectric point.
plakins A family of proteins that help attach intermediate filaments to other structures.
plaque assay Technique for determining the number of infectious viral particles in a sample by culturing a diluted sample on a layer of susceptible host cells and then counting the clear areas of lysed cells (plaques) that develop. (Figure 5-44)
plasma membrane The membrane surrounding a cell that separates the cell from its external environment; consists of a phospholipids bilayer and associated membrane lipids and proteins. (Figures 1-12a, 7-1, and 7-5)
plasmid Small, circular extrachromosomal DNA molecule capable of autonomous replication in a cell; commonly used as a vector in DNA cloning.
plasmodesmata (sing. plasmodesma) Tube-like cell junctions that interconnect the cytoplasms of adjacent plant cells and are functionally analogous to gap junctions in animal cells. (Figure 20-41)
point mutation Change of a single nucleotide in DNA, especially in a region coding for protein; can result in formation of a codon specifying a different amino acid or a stop codon. Addition or deletion of a single nucleotide will cause a shift in the reading frame.
polar Referring to a molecule or structure with a net electric charge or asymmetric distribution of positive and negative charges. Polar molecules are usually soluble in water.
polarity In cell biology, the presence of functional and/or structural differences in distinct regions of a cell or cellular component. See also cell polarity.
polarized In cell biology, referring to any cell or subcellular structure marked by functional and structural asymmetries.
Polo kinases Family of protein kinases that are critical for many aspects of mitosis, such as centrosome duplication and cohesin removal from chromosomes.
polymer Any large molecule composed of multiple identical or similar units (monomers) linked by covalent bonds. (Figure 2-13)
polymerase chain reaction See PCR.
polypeptide Linear polymer of amino acids connected by peptide bonds, usually containing 20 or more residues. See also protein.
polyribosome A complex containing several ribosomes, all translating a single messenger RNA; also called polysome. (Figure $5-27$ )
polysaccharide Linear or branched polymer of monosaccharides linked by glycosidic bonds and usually containing more than 15 residues. Those with fewer than 15 residues are often called oligosaccharides.
polytene chromosome Enlarged chromosome composed of many parallel copies of itself formed by multiple cycles of DNA replication without chromosomal separation; found in the salivary glands and some other tissues of Drosophila and other dipteran insects. (Figure 8-40)
polyubiquitination The covalent addition of a chain of covalently linked ubiquitin molecules to a site on a target protein.
polyunsaturated Referring to a compound (e.g., fatty acid) in which two or more of the carbon-carbon bonds are double or triple bonds.
porins Class of trimeric transmembrane proteins through which small water-soluble molecules can cross the membrane; present in outer mitochondrial and chloroplast membranes and in the outer membrane of gram-negative bacteria. (Figure 7-18)
positive feedback mechanism Process where the output of a pathway promotes its own production.
potential energy Stored energy. In biological systems, the primary forms of potential energy are chemical bonds, concentration gradients, and electric potentials across cellular membranes.

pre-mRNA Precursor messenger RNA; the primary transcript and intermediates in RNA processing. (Figures 5-15 and 10-2)
pre-rRNA Large precursor ribosomal RNA that is synthesized in the nucleolus of eukaryotic cells and processed to yield three of the four RNAs present in ribosomes. (Figures 10-40 and 10-41)
primary cilium The single nonmotile cilium present on almost all vertebrate cells that serves as a sensory organelle to detect extracellular signals.
primary structure In proteins, the linear arrangement (sequence) of amino acids within a polypeptide chain.
primary transcript In eukaryotes, the initial RNA product, containing introns and exons, produced by transcription of DNA. Many primary transcripts must undergo RNA processing to form the physiologically active RNA species.
primase A specialized RNA polymerase that synthesizes short stretches of RNA used as primers for DNA synthesis. (Figure 5-30).
primer A short nucleic acid sequence containing a free $3^{\prime}$-hydroxyl group that forms base pairs with a complementary template strand and functions as the starting point for addition of nucleotides to copy the template strand.
probe Defined RNA or DNA fragment, radioactively, fluorescently, or chemically labeled chemically labeled, that is used to detect specific nucleic acid sequences by hybridization.
progenitor cells A type of undifferentiated cell that, when provided with the appropriate signals, will divide and differentiate into one or a few cell types.
programmed cell death See apoptosis.
prokaryotes Class of organisms, including the bacteria (eubacteria) and archaea, that lack a true membrane-limited nucleus and other organelles. See also eukaryotes. (Figure 1-1)
prolactin A cytokine released during pregnancy of mammals that induces the development of mammary glands to produce and secrete milk proteins.
prometaphase Second stage in mitosis, during which the nuclear envelope and nuclear lamina break down and microtubules assembled from the spindle poles "capture" chromosome pairs at specialized structures called kinetochores. (Figure 18-37)
promoter DNA sequence that determines the site of transcription initiation for an RNA polymerase. (Figure 5-11)
promoter-proximal element Any regulatory sequence in eukaryotic DNA that is located within $\sim 200$ base pairs of the transcription start site. Transcription of many genes is controlled by multiple promoter-proximal elements. (Figure 9-23)
prophase Earliest stage in mitosis, during which the chromosomes condense, the duplicated centrosomes separate to become the spindle poles, and the mitotic spindle begins to form. (Figure 18-37)
protease Any enzyme that cleaves one or more peptide bonds in target proteins.
proteasome Large multifunctional protease complex in the cytosol that degrades intracellular proteins marked for destruction by attachment of multiple ubiquitin molecules. (Figure 3-31)
protein A macromolecule composed of one or more linear polypeptide chains and folded into a characteristic three-dimensional shape (conformation) in its native, biologically active state.
protein family Set of homologous proteins encoded by a gene family.
protein domain Distinct regions of a protein's three-dimensional structure. A functional domain exhibits a particular activity characteristic of the protein; a structural domain is $\approx 40$ or more amino acids in length, arranged in a distinct secondary or tertiary structure; a topological domain has a distinctive spatial relationship to the rest of the protein.
protein kinase A (PKA) Cytosolic enzyme that is activated by cyclic AMP (cAMP) and functions to phosphorylate and thus regulate the activity of numerous cellular proteins; also called cAMP-dependent protein kinase. (Figure 15-29)
protein kinase B (PKB) Cytosolic enzyme that is recruited to the plasma membrane by signal-induced phosphoinositides and subsequently activated; also called Akt. (Figure 16-29)
protein kinase C (PKC) Cytosolic enzyme that is recruited to the plasma membrane in response to signal-induced rise in cytosolic $\mathrm{Ca}^{2+}$ level and is activated by membrane-bound diacylglycerol (DAG). (Figure 15-34a)
protein kinase G (PKG) A cytosolic protein kinase activated by cyclic AMP.
proteoglycans A group of glycoproteins (e.g., perlecan and aggrecan) that contain a core protein to which is attached one or more glycosaminoglycan (GAG) chains. They are found in nearly all animal extracellular matrices, and some are integral membrane proteins. (Figure 20-32)
proteome All the proteins in a cellular compartment, intact cell, organ, or organism.
proteomics The systematic study of the amounts, modifications, interactions, localization, and functions of all or subsets of proteins at the whole-organism, tissue, cellular, and subcellular levels.
proton-motive force The energy equivalent of the proton $\left(\mathrm{H}^{+}\right)$ concentration gradient and electric potential gradient across a membrane; used to drive ATP synthesis by ATP synthase, transport of molecules against their concentration gradient, and movement of bacterial flagella. (Figure 12-2)
proto-oncogene A normal cellular gene that encodes a protein usually involved in regulation of cell growth or differentiation and that can be mutated into a cancer-promoting oncogene, either by changing the protein-coding segment or by altering its expression. (Figure 24-15)
protostomes A group of bilaterally symmetric animals whose mouth develops close to the blastopore and has a ventral nerve cord. This group includes worms, insects, and mollusks.
provirus The DNA of an animal virus that is integrated into a host-cell genome; during replication of the cell, the proviral DNA is replicated and appears in both daughter cells. Activation of proviral DNA leads to production and release of progeny virions.
pseudogene DNA sequence that is similar to that of a functional gene but does not encode a functional product; probably arose by sequence drift of duplicated genes.

pseudosubstrate domain A protein domain whose sequence or structure resembles that of an enzyme's substrate and thus binds to the enzyme's active site, but which cannot be modified by the enzyme (i.e., phosphorylated), resulting in inhibition of the enzyme.
pulse-chase A type of experiment in which a radioactive small molecule is added to a cell for a brief period (the pulse) and then is replaced with an excess of the unlabeled form of the same small molecule (the chase). Used to detect changes in the cellular location of a molecule or its metabolic fate over time. (Figure 3-42)

## pump See ATP-powered pump.

purines A class of nitrogenous compounds containing two fused heterocyclic rings. Two purines, adenine (A) and guanine (G), are base components of nucleotides found in DNA and RNA. See also base pair. (Figure 2-17)
pyrimidines A class of nitrogenous compounds containing one heterocyclic ring. Two pyrimidines, cytosine (C) and thymine (T), are base components of nucleotides found in DNA; in RNA, uracil (U) replaces thymine. See also base pair. (Figure 2-17)
quaternary structure The number and relative positions of the polypeptide chains in multimeric (multisubunit) proteins. (Figure 3-11b)
radioisotope Unstable form of an atom that emits radiation as it decays. Several radioisotopes are commonly used experimentally as labels in biological molecules. (Table 3-1)
Ras protein A monomeric member of the GTPase superfamily of switch proteins that is tethered to the plasma membrane by a lipid anchor and functions in intracellular signaling pathways; activated by ligand binding to receptor tyrosine kinases and some other cell-surface receptors. (Figures 16-21 and 16-23)
rate constant A constant that relates the concentrations of reactants to the rate of a chemical reaction.
Rb protein Inhibitor of the E2F transcription factor family and thus a key regulator of cell cycle entry.
reading frame The sequence of nucleotide triplets (codons) that runs from a specific translation start codon in an mRNA to a stop codon. Some mRNAs can be translated into different polypeptides by reading in two different reading frames. (Figure 4-18)
receptor Any protein that specifically binds another molecule to mediate cell-cell signaling, adhesion, endocytosis, or other cellular process. Commonly denotes a protein located in the plasma membrane, cytosol, or nucleus that binds a specific extracellular molecule (ligand), which often induces a conformational change in the receptor, thereby initiating a cellular response. See also adhesion receptor and nuclear receptor. (Figures 15-1 and 16-1)
receptor-mediated endocytosis Uptake of extracellular materials bound to specific cell-surface receptors by invagination of the plasma membrane to form a small membrane-bounded vesicle (early endosome). (Figure 14-29)
receptor tyrosine kinase (RTK) Member of a large class of cellsurface receptors, usually with a single transmembrane domain, including those for insulin and many growth factors. Ligand binding activates tyrosine-specific protein kinase activity in the
receptor's cytosolic domain, thereby initiating intracellular signaling pathways. (Figures 16-14 and 16-15)
recessive In genetics, referring to that allele of a gene that is not expressed in the phenotype when the dominant allele is present; also refers to the phenotype of an individual (homozygote) carrying two recessive alleles. Mutations that produce recessive alleles generally result in a loss of the gene's function. (Figure 6-2)
recombinant DNA Any DNA molecule formed in vitro by joining DNA fragments from different sources.
recombination Any process in which chromosomes or DNA molecules are cleaved and the fragments are rejoined to give new combinations. Homologous recombination occurs during meiosis, giving rise to crossing over of homologous chromosomes. Homologous recombination and nonhomologous recombination (i.e., between chromosomes of different morphologic type) also occur during several DNA-repair mechanisms and can be carried out in vitro with purified DNA and enzymes. (Figure 6-10)
redox reaction An oxidation-reduction reaction in which one or more electrons are transferred from one reactant to another.
reduction Gain of electrons by an atom or molecule as occurs when a hydrogen atom is added to a molecule or oxygen is removed. The opposite of oxidation.
reduction potential $(E)$ The voltage change when an atom or molecule gains an electron; a measure of the tendency of a molecule to gain an electron. For a given reduction reaction, $E$ has the same magnitude but opposite sign as the oxidation potential for the reverse (oxidation) reaction.
release factor (RF) One of two types of nonribosomal proteins that recognize stop codons in mRNA and promote release of the completed polypeptide chain, thereby terminating translation (protein synthesis). (Figure 5-26)
replication fork Y-shaped region in double-stranded DNA at which the two strands are separated and replicated during DNA synthesis; also called growing fork. (Figure 5-29)
replication origin Unique DNA segment present in an organism's genome at which DNA replication begins. Eukaryotic chromosomes contain multiple origins, whereas bacterial chromosomes and plasmids usually contain just one.
reporter gene A gene encoding a protein that is easily assayed (e.g., $\beta$-galactosidase, luciferase). Reporter genes are used in various types of experiments to indicate activation of the promoter to which the reporter gene is linked.
repression domain A region of a repressor transcription factor that will inhibit transcription when fused to a DNA-binding domain.
repressor Specific transcription factor that inhibits transcription.
residue General term for the repeating units in a polymer that remain after covalent linkage of the monomeric precursors.
resolution The minimum distance between two objects that can be distinguished by an optical apparatus; also called resolving power.
respiration The conversion of energy in nutrients into ATP via a set of reactions involving a series of protein-catalyzed,

membrane-associated oxidation and reduction reactions, called an electron transport chain, that are ultimately coupled to the addition of $\mathrm{P}_{\mathrm{i}}$ to ADP (oxidative phosphorylation) to form ATP and transfer of electrons to oxygen or other inorganic electron acceptors.
respiratory chain See electron transport chain.
respiratory control Dependence of mitochondrial oxidation of NADH and $\mathrm{FADH}_{2}$ on the supply of ADP and $\mathrm{P}_{\mathrm{i}}$ for ATP synthesis.
resting $\mathrm{K}^{+}$channels Nongated $\mathrm{K}^{+}$ion channels in the plasma membrane that, in conjunction with the high cytosolic $\mathrm{K}^{+}$concentration produced by the $\mathrm{Na}^{+} / \mathrm{K}^{+}$ATPase, are primarily responsible for generating the inside-negative resting membrane potential in animal cells.
restriction enzyme Any enzyme that recognizes and cleaves a specific short sequence, the restriction site, in double-stranded DNA molecules; used extensively to produce recombinant DNA in vitro; also called restriction endonuclease. (Figure 6-11; Table 6-1)
restriction point Point in the mammalian cell cycle after which cells are no longer responsive to proliferation regulatory signals.
retrotransposon Type of eukaryotic transposable DNA element whose movement in the genome is mediated by an RNA intermediate and involves a reverse-transcription step. See also transposon. (Figure 8-8b)
retrovirus Type of eukaryotic virus containing an RNA genome that replicates in cells by first making a DNA copy of the RNA. This viral DNA is inserted into cellular chromosomal DNA, forming a provirus, and gives rise to further genomic RNA as well as the mRNAs for viral proteins. (Figure 5-48)
reverse transcriptase Enzyme found in retroviruses that catalyzes a complex reaction in which a double-stranded DNA is synthesized from a single-stranded RNA template. (Figure 8-14)
R group A portion of a molecule, or a chemical group within a larger molecule, that is covalently bonded as an appendage to the main body or core of the molecule. In an amino acid, the R group is the side chain that is attached to the alpha carbon atom and that confers the distinct characteristics of the amino acid.
ribonucleic acid (RNA) See RNA.
ribonucleoprotein (RNP) complex General term for any complex composed of proteins and RNA. Most RNA molecules are present in the cell in the form of RNPs.
ribosome A large complex comprising several different rRNA molecules and as many as 83 proteins, organized into a large subunit and small subunit; the engine of translation (protein synthesis). (Figures 5-22 and 5-23)
ribosomal RNA See rRNA.
ribozyme An RNA molecule with catalytic activity. Ribozymes function in RNA splicing and protein synthesis.
ribulose 1,5-bisphosphate carboxylase Enzyme located in chloroplasts that catalyzes the first reaction in the Calvin cycle, the addition of $\mathrm{CO}_{2}$ to a five-carbon sugar (ribulose 1,5-bisphosphate) to form two molecules of 3-phosphoglycerate; also called rubisco. (Figure 12-47)
RISC See RNA-induced silencing complex.

RNA (ribonucleic acid) Linear, single-stranded polymer, composed of ribose nucleotides. mRNA, rRNA, and tRNA play different roles in protein synthesis; a variety of small RNAs play roles in controlling the stability and translation of mRNAs and in controlling chromatin structure and transcription. (Figure 5-17)
RNA editing Unusual type of RNA processing in which the sequence of a pre-mRNA is altered.
RNA-induced silencing complex (RISC) Large multiprotein complex associated with a short single-stranded RNA (siRNA or miRNA) that mediates degradation or translational repression of a complementary or near-complementary mRNA.
RNA interference (RNAi) Functional inactivation of a specific gene by a corresponding double-stranded RNA that induces either inhibition of translation or degradation of the complementary single-stranded mRNA encoded by the gene but not that of mRNAs with a different sequence. (Figures 6-42)
RNA polymerase An enzyme that copies one strand of DNA (the template strand) to make the complementary RNA strand using as substrates ribonucleoside triphosphates. (Figure 5-11)
RNA splicing A process that results in removal of introns and joining of exons in pre-mRNAs. See also spliceosome. (Figures $10-8$ and 10-9)
rRNA (ribosomal RNA) Any one of several large RNA molecules that are structural and functional components of ribosomes. Often designated by their sedimentation coefficient: 28S, 18S, 5.8 S , and 5 S rRNA in higher eukaryotes. (Figure 5-22)
rubisco See ribulose 1,5-bisphosphate carboxylase.

## S (synthesis) phase See cell cycle.

S-adenosylmethionine (S-Ado-Met) A methyl donor derived from ATP and methionine, used in the synthesis of multiple metabolic intermediates. It is the methyl donor for methylation of DNA and RNA, including $5^{\prime}$ cap synthesis and internal m6A in mRNAs, methylation at multiple sites in rRNAs and tRNAs, and methylation of cytosines in DNA.
sarcomere Repeating structural unit of striated (skeletal) muscle composed of organized, overlapping thin (actin) filaments and thick (myosin) filaments and extending from one Z disk to an adjacent one; shortens during contraction. (Figures 17-30 and 17-31)
sarcoplasmic reticulum Network of membranes in cytoplasm of a muscle cell that sequesters $\mathrm{Ca}^{2+}$ ions; release of stored $\mathrm{Ca}^{2+}$ induced by muscle stimulation triggers contraction. (Figure 17-33)
satellite DNA See simple-sequence DNA.
saturated Referring to a compound (e.g., fatty acid) in which all the carbon-carbon bonds are single bonds.
$\mathrm{SC}-\boldsymbol{\beta}$ cells Insulin-secreting $\beta$-islet cells, derived in the laboratory from human iPS and ES cells, which have the potential to treat patients with diabetes.
SCF (Skp1, Cullin, F-box proteins) Ubiquitin-protein ligase that ubiquitinylates inhibitors of S-phase CDKs and many other proteins, marking them for degradation by proteasomes.
second messenger A small intracellular molecule (e.g., cAMP, cGMP, $\mathrm{Ca}^{2+}$, DAG, and $\mathrm{IP}_{3}$ ) whose concentration increases

(or decreases) in response to binding of an extracellular signal and that functions in signal transduction. (Figure 15-6)
secondary structure In proteins, local folding of a polypeptide chain into regular structures including the $\alpha$ helix, $\beta$ sheet, and $\beta$ turns.
secretory pathway Cellular pathway for synthesizing and sorting soluble and membrane proteins localized to the endoplasmic reticulum, Golgi, and lysosomes; plasma membrane proteins; and proteins eventually secreted from the cell. (Figure 14-1)
securin A protein that prevents the onset of chromosome segregation by inhibiting separase, the protease that cleaves cohesin.
segregation The process that distributes an equal complement of chromosomes to daughter cells during mitosis and meiosis.
selectins A family of cell-adhesion molecules that mediate $\mathrm{Ca}^{2+}-$ dependent interactions with specific oligosaccharide moieties in glycoproteins and glycolipids on the surface of adjacent cells or in extracellular glycoproteins. (Figures 20-2 and 20-40)
self-renewal The ability of a stem cell to reproduce itself during cell division without differentiating.
sensor Measures an intra- or extracellular property and converts into a signal.
separase A protease that cleaves cohesin, thereby initiating chromosome segregation.
short interspersed elements (SINEs) In mammals, repeated sequences related to a 150 (in mice)-300 (in humans) base-pair sequence that has been transposed throughout the genome over evolutionary time by proteins encoded by a long interspersed element (LINE) in the same organism. (Figure 8-8; Table 8-1)
shuttle vector Plasmid vector capable of propagation in two different hosts. (Figure 6-15)
side chain In amino acids, the variable substituent group attached to the alpha $(\alpha)$ carbon atom that largely determines the particular properties of each amino acid; also called $R$ group. (Figure 2-14)
signaling cascade Pathway that is activated by an intracellular or extracellular event and then transmits the signal to an effector.
signaling molecule General term for any extracellular or intracellular molecule involved in mediating the response of a cell to its external environment or to other cells.
signal peptide See targeting sequence.
signal-recognition particle (SRP) A cytosolic ribonucleoprotein particle that binds to the ER signal sequence in a nascent secretory protein and delivers the nascent chain/ribosome complex to the ER membrane, where synthesis of the protein and translocation into the ER are completed. (Figure 13-5)
signal sequence See targeting sequence.
signal transduction Conversion of a signal from one physical or chemical form into another. In cell biology commonly refers to the sequential process initiated by binding of an extracellular signal to a receptor and culminating in one or more specific cellular responses.
signal transduction pathway The set of proteins or small molecules involved in relaying a chemical message from the receptor at the cell surface to the target molecule(s) inside the cell.
silencer A sequence in eukaryotic DNA that promotes formation of condensed chromatin structures in a localized region, thereby blocking access of proteins required for transcription of genes within several hundred base pairs of the silencer; also called silencer sequence.
simple diffusion Net movement of a molecule across a membrane down its concentration gradient at a rate proportional to the gradient and the permeability of the membrane; also called passive diffusion.
simple-sequence DNA Short, tandemly repeated sequences that are found at centromeres and telomeres as well as at other chromosomal locations and are not transcribed; also called satellite DNA.
siRNA A small, double-stranded RNA, 21-23 nucleotides long with two single-stranded nucleotides at each end. A single strand of the siRNA associates with several proteins to form an RNAinduced silencing complex (RISC) that cleaves target RNAs to which the siRNA base pairs perfectly; variously called short or small interfering RNA and small inhibitory RNA. siRNAs can be designed to experimentally inhibit expression of specific genes. See also miRNA. (Figure 10-28b)
siRNA knockdown Technique for experimentally inhibiting translation of a specific mRNA by use of siRNA; useful for reducing the activity of a protein, particularly in organisms that are not amenable to classical genetic methods for isolating loss-of-function mutants.
sister chromatid resolution The process of untangling of the intertwined sister chromatids during prophase.
sister chromatids The two identical DNA molecules created during DNA replication and the associated chromosomal proteins. After DNA replication, each chromosome is composed of two sister chromatids.
Smads Class of transcription factors that are activated by phosphorylation following binding of members of the transforming growth factor $\boldsymbol{\beta}$ (TGF $\boldsymbol{\beta}$ ) family of signaling molecules to their cell-surface receptors. (Figure 16-3)
SM proteins Sec1/Munc18-like proteins, which bind syntaxin and are required for SNARE-mediated fusion of synaptic vesicles to a cell membrane.
SMC proteins Structural maintenance of chromosome proteins; a small family of nonhistone chromatin proteins that are critical for maintaining the morphological structure of chromosomes and their proper segregation during mitosis. Members of this family include condensins, which help condense chromosomes during mitosis, and cohesins, which link sister chromatids until their separation in anaphase. Bacterial SMC proteins function in the proper segregation of bacterial chromosomes to daughter cells. (Figures 8-32 and 19-25)
SNAREs Cytosolic and integral membrane proteins that promote fusion of vesicles with target membranes. Interaction of $\mathbf{v}$-SNAREs on a vesicle with cognate t-SNAREs on a target membrane forms very stable complexes, bringing the vesicle and target membranes into close apposition. (Figure 14-10)
snoRNA (small nucleolar RNA) A type of small, stable RNA that functions in rRNA processing and base modification in the nucleolus.
snRNA (small nuclear RNA) One of several small, stable RNAs localized to the nucleus. Five snRNAs are components of the

spliceosome and function in splicing of pre-mRNA. (Figures 10-9 and 10-11)
somatic cell Any plant or animal cell other than a germ cell.
somatic-cell nuclear transfer A procedure in which the nucleus of an adult somatic cell is transferred into an enucleated egg, a step in producing a cloned animal.
sorting signal A relatively short amino acid sequence within a protein that directs the protein to particular transport vesicles as they bud from a donor membrane in the secretory or endocytic pathway. (Table 14-2)
Southern blotting Technique for detecting specific DNA sequences separated by electrophoresis by hybridization to a labeled nucleic acid probe. (Figure 6-24)
specificity The ability of immune cells or their products to distinguish between structurally closely related molecules.
sperm The male gamete-a motile haploid cell that can bind to and fuse with an egg cell, forming a zygote.
S-phase CDKs Cyclin-CDK complexes that promote the initiation of DNA replication.
sphingolipid Major group of membrane lipids, derived from sphingosine, that contain two long hydrocarbon chains and either a phosphorylated head group (sphingomyelin) or carbohydrate head group (cerebrosides, gangliosides). (Figure 7-8b)
spindle assembly checkpoint pathway Pathway that senses incorrect attachment of chromosomes to the mitotic spindle and induces cell cycle arrest in metaphase.
spindle midzone Middle portion of the mitotic spindle that plays an important role in cleavage furrow positioning in some organisms.
spindle pole bodies Functionally analogous structure of centrosomes in yeast.
spindle position checkpoint pathway Pathway that senses incorrect position of the mitotic spindle within the cell and induces cell cycle arrest in anaphase.
spliceosome Large ribonucleoprotein complex that assembles on a pre-mRNA and carries out RNA splicing. (Figure 10-11)
SRE-binding proteins (SREBPs) Cholesterol-dependent transcription factors, localized in the ER membrane, that are activated in response to low cellular cholesterol levels and then stimulate expression of genes encoding proteins involved in cholesterol synthesis and import as well as synthesis of other lipids. (Figure 16-38)
starch A very long, branched polysaccharide, composed exclusively of glucose units, that is the primary storage carbohydrate in plant cells. (Figure 12-36)
START Point in the cell cycle at which cells are irreversibly committed to cell division and can no longer return to the $\mathrm{G}_{1}$ state.
steady state In cellular metabolic pathways, the condition when the rate of formation and rate of consumption of a substance are equal, so that its concentration remains constant. (Figure 2-23)
stem cell A self-renewing cell that can divide symmetrically to give rise to two daughter cells whose developmental potential is identical to that of the parental stem cell or asymmetrically to
generate daughter cells with different developmental potentials. (Figure 21-10 and 21-11)
stem-cell niche A set of cells, extracellular matrices, and hormones that surrounds a stem cell and that maintains its stem-cell properties
stereoisomers Two compounds with identical molecular formulas whose atoms are linked in the same order but in different spatial arrangements. In optical isomers, designated $t 0$ and t , the atoms bonded to an asymmetric carbon atom are arranged in a mirror-image fashion. Geometric isomers include the cis and trans forms of molecules containing a double bond.
steroids A group of four-ring hydrocarbons including cholesterol and related compounds. Many important hormones (e.g., estrogen and progesterone) are steroids. Sterols are steroids containing one or more hydroxyl groups. (Figure 10-8c)
strand invasion An early step in DNA recombination catalyzed by RecA in bacteria and Rad51 in eukaryotes in which a singlestranded region of DNA with a free $3^{\prime}$ end hybridizes to its complementary strand in a second double-stranded DNA molecule. The complementary strand of this target double-stranded DNA is displaced as a single-stranded loop of DNA over the region of hybridization to the invading strand. (Figures 5-40 and 5-41)
structural motif A particular combination of two or more secondary structures that form a distinct three-dimensional structure that appears in multiple proteins and that often, but not always, is associated with a specific function.
substrate Molecule that undergoes a charge in a reaction catalyzed by an enzyme.
substrate-level phosphorylation Formation of ATP from ADP and $P_{i}$ catalyzed by cytosolic enzymes in reactions that do not depend on a proton-motive force or molecular oxygen.
sulfhydryl group (-SH) A substituent group present in the amino acid cysteine and other molecules consisting of a hydrogen atom covalently bonded to a sulfur atom; also called a thiol group.
SWI/SNF chromatin-remodeling complex A multiprotein chromatin-remodeling co-activator complex with at least one subunit that is homologous to DNA helicases that facilitates nucleosomes sliding along DNA and decondensation of condensed chromatin structures.
symmetric cell division See cell division.
symport A type of cotransport in which a membrane protein (symporter) transports two different molecules or ions across a cell membrane in the same direction. See also antiport. (Figure $11-2,[3 B])$
synapse Specialized region between an axon terminal of a neuron and an adjacent neuron or other excitable cell (e.g., muscle cell) across which impulses are transmitted. At a chemical synapse, the impulse is conducted by a neurotransmitter; at an electric synapse, impulse transmission occurs via gap junctions connecting the pre- and postsynaptic cells. (Figure 22-3)
synaptic plasticity The modification of synaptic connectivity based on neuronal experience, which is a biological manifestation of forming and storing memories.
synaptic vesicles Small vesicles in axon termini that contain a neurotransmitter and that undergo exocytosis following arrival of an action potential.

synaptonemal complex (SC) Proteinaceous structure that mediates the association (synapsis) between homologous chromosomes during prophase of meiosis I.
syndecans A class of cell-surface proteoglycans that function in cell-matrix adhesion, interact with the cytoskeleton, and may bind external signals, thereby participating in cell-cell signaling.
syntelic attachment Indicates that kinetochores of a sister chromatid pair attach to microtubules emanating from the same pole.
synteny Occurrence of genes in the same order on a chromosome in two or more different species.
targeting sequence A relatively short amino acid sequence within a protein that directs the protein to a specific location within the cell; also called signal peptide, signal sequence, and uptake-targeting sequence. (Table 13-1)
Tat An HIV-encoded protein that prevents Pol II termination when synthesizing the polycistronic HIV mRNA, thereby allowing transcription of the complete proviral DNA genome.
TATA box A conserved sequence in the promoter of many eukaryotic protein-coding genes where the transcription-initiation complex assembles. (Figure 9-16)
T cell A lymphocyte that matures in the thymus and expresses antigen-specific receptors that bind antigenic peptides complexed to MHC molecules. There are two major classes: cytotoxic T cells (CD8 surface marker, class I MHC restricted, kill virus-infected and tumor cells) and helper T cells (CD4 marker, class II MHC restricted, produce cytokines, required for activation of B cells). (Figures 23-34 and 23-37)
T-cell receptor A heterodimeric antigen-binding transmembrane protein containing variable and constant regions and associated with the signal-transducing multimeric CD3 complex. (Figure 23-27)
telomere Region at each end of a eukaryotic chromosome containing multiple tandem repeats of a short telomeric (TEL) sequence. Telomeres are required for proper chromosome segregation and are replicated by a special process that prevents shortening of chromosomes during DNA replication. (Figure 8-44)
telophase Final mitotic stage, during which the nuclear envelope re-forms around the two sets of separated chromosomes, the chromosomes decondense, and division of the cytoplasm (cytokinesis) is completed. (Figure 18-37)
temperature-sensitive (ts) mutation A mutation that produces a wild-type phenotype at one temperature (the permissive temperature) but a mutant phenotype at another temperature (the nonpermissive temperature). This type of mutation is especially useful in identification of genes essential for life. (Figure 6-6)
termination, transcription Cessation of the synthesis of an RNA chain. (Figure 5-11)
tertiary structure In proteins, overall three-dimensional form of a polypeptide chain, which is stabilized by multiple noncovalent interactions between side chains. (Figure 3-11a)
thylakoids Flattened membranous sacs in a chloroplast that can be arranged in stacks and contain the photosynthetic pigments and photosystems. (Figure 12-37)
thrombopoietin A cytokine that stimulates the development of megakaryocytes, cells that form the platelets involved in blood clotting.
tight junction A type of cell-cell junction between the plasma membranes of adjacent epithelial cells that prevents diffusion of macromolecules and many small molecules and ions in the spaces between cells and diffusion of membrane components between the apical and basolateral regions of the plasma membrane. (Figure 20-17)
tolerance The absence of an immune response to a particular antigen or set of antigens.
Toll-like receptor (TLR) Member of a class of cell-surface and intracellular receptors that recognize a variety of microbial products. Ligand binding initiates a signaling pathway that induces various responses depending on the cell type. (Figure 23-35)
topogenic sequences Segments within a protein whose sequence, number, and arrangement direct the insertion and orientation of various classes of transmembrane proteins into the endoplasmic reticulum membrane. (Figure 13-14)
transcription Process in which one strand of a DNA molecule is used as a template for synthesis of a complementary RNA by RNA polymerase. (Figures 5-10 and 5-11)
transcription-control region Collective term for all the DNA regulatory sequences that regulate transcription of a particular gene.
transcription factor (TF) General term for any protein, other than RNA polymerase, required to initiate or regulate transcription in eukaryotic cells. General factors, required for transcription of all genes, participate in formation of the transcription-preinitiation complex near the start site. Specific factors stimulate (activators) or inhibit (repressors) transcription of particular genes by binding to their regulatory sequences.
transcription unit A region in DNA, bounded by an initiation (start) site and termination site, that is transcribed into a single primary transcript.
transcytosis Mechanism for transporting certain substances across an epithelial sheet that combines receptor-mediated endocytosis and exocytosis. (Figures 14-25 and 23-11)
transfection Experimental introduction of foreign DNA into cells in culture, usually followed by expression of genes in the introduced DNA. (Figure 6-29)
transfer RNA See tRNA.
transformation (1) Permanent, heritable alteration in a cell resulting from the uptake and incorporation of a foreign DNA into the host-cell genome; also called stable transfection. (2) Conversion of a "normal" mammalian cell into a cell with cancer-like properties usually induced by treatment with a virus or other cancer-causing agent.
transforming growth factor beta (TGF $\boldsymbol{\beta}$ ) A family of secreted signaling proteins that are used in the development of most tissues in most or all animals. Members of the TGF $\beta$ family more often inhibit growth than stimulate it. Mutations in TGF $\beta$ signal transduction components are implicated in human cancer, including breast cancer. (Figure 16-3)

transgene A cloned gene that is introduced and stably incorporated into a plant or animal and is passed on to successive generations.
trans-Golgi network (TGN) Complex network of membranes and vesicles that serves as a major branch point in the secretory pathway. Vesicles budding from this most-distal Golgi compartment carry membrane and soluble proteins to the cell surface or to lysosomes. (Figures 14-1 and 14-17)
transition state State of the reactants during a chemical reaction when the system is at its highest energy level; also called the transition-state intermediate.
translation The ribosome-mediated assembly of a polypeptide whose amino acid sequence is specified by the nucleotide sequence in an mRNA. (Figure 5-17)
translocon Multiprotein complex in the membrane of the rough endoplasmic reticulum through which a nascent secretory protein enters the ER lumen as it is being synthesized. (Figure 13-7)
transporters Membrane proteins that undergo conformational changes as they move a wide variety of ions and molecules across cell membranes at a slower rate than channels. See uniporter, symporter, and antiporter in Figure 11-3.
transport protein See membrane transport protein.
transport vesicle A small membrane-bounded compartment that carries soluble and membrane "cargo" proteins in the forward or reverse direction in the secretory pathway. Vesicles form by budding off from the donor organelle and release their contents by fusion with the target membrane.
transposable DNA element Any DNA sequence that is not present in the same chromosomal location in all individuals of a species and can move to a new position by transposition; also called mobile DNA element and interspersed repeat. (Table 8-1)
transposition Movement of a transposable DNA element within the genome; occurs by a cut-and-paste mechanism or copy-and-paste mechanism depending on the type of element. (Figure 8-8)
transposon, DNA A transposable DNA element present in prokaryotes and eukaryotes that moves in the genome by a mechanism involving DNA synthesis and transposition. See also retrotransposon. (Figures 8-9 and 8-10)
triacylglycerol See triglyceride.
triglyceride Major form in which fatty acids are stored and transported in animals; consists of three fatty acyl chains esterified to a glycerol molecule.
trimeric (large) G protein A regulatory membrane-associated GTPase consisting of a catalytic $\alpha$ subunit and a $\beta$ and $\gamma$ subunit. When the $\alpha$ subunit is bound to GTP, the $\beta$ and $\gamma$ subunits dissociate as a heterodimer. The free $\alpha$ subunit and the free $\beta-\gamma$ heterodimer can then interact with other proteins and transduce a signal across the membrane. When the $\alpha$ subunit hydrolyzes the GTP to GDP and phosphate, the $\alpha$-GDP associates with the $\beta-\gamma$ heterodimer, terminating signaling. (Figure 15-14)
tRNA (transfer RNA) A group of small RNA molecules that function as amino acid donors during protein synthesis. Each
tRNA becomes covalently linked to a particular amino acid, forming an aminoacyl-tRNA. (Figures 5-19 and 5-20)
trophic factor Any of numerous signaling proteins required for the survival of cells in multicellular organisms; in the absence of such signals, cells often undergo "suicide" by apoptosis.
trophoectoderm (TE) The part of an early mammalian embryo that will form the extra-embryonic tissues, including the placenta but not the embryo proper.

## t-SNAREs See SNAREs.

tubulin A family of globular cytoskeletal proteins that polymerize to form the cylindrical wall of microtubules. (Figure 18-3)
TUG A protein that tethers GLUT4 storage vesicles to the Golgi matrix.
tumor A mass of cells, generally derived from a single cell, that arises due to loss of the normal regulators of cell growth; may be benign or malignant.
tumor-suppressor gene Any gene whose encoded protein directly or indirectly inhibits progression through the cell cycle and in which a loss-of-function mutation is oncogenic. Inheritance of a single mutant allele of many tumor-suppressor genes (e.g., $R B$, $A P C$, and BRCA1) greatly increases the risk for developing colorectal and other types of cancer. (Figures 24-12 and 24-15)
tunneling nanotubes Tubelike projections of the plasma membrane that form a continuous channel connecting the cytosols of animal cells and can transfer chemical and electrical signals between cells in a manner analogous to plasmodesmata in plants.
ubiquitin A small protein that can be covalently linked to other intracellular proteins, thereby tagging these proteins for degradation by the proteasome, sorting to the lysosome, or alteration in the function of the target protein. (Figure 3-31)
uncoupler Any natural substance (e.g., the protein thermogenin) or chemical agent (e.g., 2,4-dinitrophenol) that dissipates the pro-ton-motive force across the inner mitochondrial membrane or thylakoid membrane of chloroplasts, thereby inhibiting ATP synthesis.
uniporter A transmembrane protein that transports a single type of molecule down its concentration gradient.
unsaturated Referring to a compound (e.g., fatty acid) in which one of the carbon-carbon bonds is a double or triple bond.
upstream (1) For a gene, the direction opposite to that in which RNA polymerase moves during transcription. Nucleotides upstream from the +1 position (the first transcribed nucleotide) are designated $-1,-2$, etc. (2) Events that occur earlier in a cascade of steps (e.g., signaling pathway). See also downstream.
upstream activating sequence (UAS) Any protein-binding regulatory sequence in the DNA of yeast and other simple eukaryotes that is necessary for maximal gene expression; equivalent to an enhancer or promoter-proximal element in higher eukaryotes. (Figure 9-23)
vaccine An innocuous preparation derived from a pathogen and designed to elicit an immune response in order to provide immunity against a future challenge by a virulent form of the same pathogen.

vacuole A membrane-limited plant organelle that stores water, ions, and small-molecule nutrients and may have a degradative function similar to that of lysosomes in animal cells.
van der Waals interaction A weak noncovalent interaction due to small, transient asymmetric electron distributions around atoms (dipoles). (Figure 2-10)
vector In cell biology, an autonomously replicating genetic element used to carry a cDNA or fragment of genomic DNA into a host cell for the purpose of gene cloning. Commonly used vectors are bacterial plasmids and modified bacteriophage genomes. See also expression vector and shuttle vector. (Figure 6-13)
viral envelope A phospholipid bilayer forming the outer covering of some viruses (e.g., influenza and rabies viruses); is derived by budding from a host-cell membrane and contains virusencoded glycoproteins. (Figure 5-46)
virion An individual viral particle.
virus A small intracellular parasite, consisting of nucleic acid (RNA or DNA) enclosed in a protein coat, that can replicate only in a susceptible host cell; widely used in cell biology research. (Figure 5-43)
$V_{\text {max }}$ Parameter that describes the maximal velocity of an enzyme-catalyzed reaction or other process such as protein-mediated transport of molecules across a membrane. (Figures 3-24 and 11-4)
v-SNAREs See SNAREs.

Warburg effect Named after its discoverer, Otto Warburg, describes the observation that most cancer cells predominantly produce energy by glycolysis followed by fermentation of pyruvate to lactic acid. While normal cells use this form of energy
production only under the oxygen-limiting (anaerobic) condition, cancer cells metabolize glucose in this manner even in the presence of sufficient oxygen. The process is thus also referred to as aerobic glycolysis.
Wee1 protein-tyrosine kinase; phosphorylates CDKs on threonine 14 and tyrosine 15 to inhibit CDK activity.
wild type Normal, nonmutant form of a gene, protein, cell, or organism.
Wnt A family of secreted signaling proteins used in the development of most tissues in most or all animals. Mutations in Wnt signal transduction components are implicated in human cancer, especially colon cancer. Receptors are Frizzled-class proteins with seven transmembrane segments. (Figure 16-30)
xeroderma pigmentosum A rare inherited disease due to inactivating mutations in any of seven genes encoding proteins involved in DNA nucleotide excision repair. Patients are abnormally sensitive to the UV in sunlight and have a high incidence of multiple types of skin cancer.
x-ray crystallography Commonly used technique for determining the three-dimensional structure of macromolecules (particularly proteins and nucleic acids) by passing x-rays through a crystal of the purified molecules and analyzing the diffraction pattern of discrete spots that results. (Figure 3-45)
zinc finger Several related DNA-binding structural motifs composed of secondary structures folded around a zinc ion; present in numerous eukaryotic transcription factors. (Figures $3-10 \mathrm{c}$ and $9-30 \mathrm{a}$ and b)

# INDEX 

Page numbers followed by " f " indicate figures.
Page numbers followed by " $t$ " indicate tables.

## A

A box, 413
A kinase-associated proteins (AKAPs), 705-706
AAA ATPase family, 608
ABC superfamily proteins, 484f, 485, 492f, $492 t$
drug and toxin export by, 491-493
phospholipids and, 493-494
ABCB1. See Multidrug-resistance transport protein
ABCB4, 493f, 494
ABL kinase, 1158
ABO blood groups, 290 t
ABO blood-group antigens, 289-290, 290f
Ac elements. See Activator (Ac) elements
ACAT. See Acylcholesterol acyl transferase
Acceptor stem, 185
Accessory proteins, 805
Acetabularia, 185
Acetate, 293
Acetic acid, 56, 56 f
Acetyl CoA, 293, 1053
citric acid cycle and, 533-535, 535f
structure of, 533, 534f
Acetyl groups, 44
Acetylation, 45
Acetylcholine, 676, 715, 1026, 1050, 1052f, 1053, 1055
muscle contraction and, 1057-1058
Acetylcholine receptors (AChR), 1050, 1050f, 1051
muscarinic, 693-694, 694f
nicotinic, 1057, 1058-1059, 1059f
Acetylcholinesterase (AChE), 1050f, 1057
Acetyl-CoA carboxylase, 293
AChE. See Acetylcholinesterase
Acid hydrolases, 17
Acid-base catalysis, 95
Acidic activation domains, 386
Acidic keratins, 863, 864t
Acids, 55, 55 f
Acquired immune deficiency syndrome (AIDS), 218, 967, 1152
Acrosome, 977, 978f
ACTH. See Adrenocorticotropic hormone
Actin, 778-779
capping proteins and, 785, 785f
in contractile bundles, 807
critical concentration of, 782
cytoskeleton, 815 f
dynamics of, 791-792, 792f
filament assembly mechanisms, 786-792
filament growth, 782-784, 783f
filament turnover regulation, 784 f
GTP-binding proteins and organization of, 813-815
intracellular movements and, 789-790, 790 f
membranes linked to, 793-796, 794f
myosin steps and, 802-803
myosin V-bound vesicles and, 808-810
in myosins, 796-803
nucleation by Arp2/3 complex, 787-789, 788 f
nucleation by formins, 786, 787f
polymerization of, 781-782, 782f, 789-790, 790 f
structure organization, 793-796
thymosin- $\beta_{4}$ and, 785
toxins perturbing monomer pool, 791-792, 792 f
treadmilling, 783f, 784-785
Actin cross-linking proteins, 793, 794f
Actin structures, 778-781
Actin-activated ADPase activity, 798
Actin-binding proteins, skeletal muscle contraction and, 805-807, 806f
Actin-nucleating proteins, 786
Action potential, 805, 1026, 1027-1028, 1027 f
conduction in myelinated axons, 1043, 1043 f
generation of, 1035-1037, 1059-1060, 1060 f
$\mathrm{Na}^{+}$influx and, 1034-1035
propagation, 1037-1039, 1038f
Activation, 1151, 1151f
of B cells, 1128
feed-forward, 517
by G proteins, 691-693
gene, during cellular differentiation, 395-396
of integrins, 963 f
of JAK kinase, 728-730, 729f
of mitotic CDKs, 896-897
of PKA, 701-702, 702f, 703-704
receptor, EGF, and, 736-737, 737f
of receptor proteins, 682
of rhodopsin, 695-696, 696f
of RTKs, 734-735, 734f
separase, by APC/C, 901-902
splicing, 436 f

SREBP activation, cholesterol and, 765, 765 f
of STAT proteins, 731, 732f
of T cells, 1122, 1122 f
of TLRs, 1126 f
Activation domain (AD), 382, 386-387, 387f, 403f
Mediator complex and, 396-397, 397f
Activation energy, 60-61, 60f, 90, 90f
Activation loop, 730, 730f, 734-735, 883
Activation-induced deaminase (AID), 1099-1100, 1103, 1137
Activator (Ac) elements, 314
Activator proteins, 354, 359, 364
CAP, 357
histone acetylation directed by, 394-395
Activators, 384f
DNA-binding, 358-359
functional domains, 381-383
splicing, 437-439
TPA, 77-78
Active immunization, 1089
Active site, 91-92, 91f
DNA transcription, 177
serine proteases and, 92-96, 94f
Active transport, 476, 477t
Acute inflammatory demyelinating polyneuropathy, 1045
Acylation, 288
Acylcholesterol acyl transferase (ACAT), 764
AD. See Activation domain
ADAM 10, 761-762, 762f
ADAMs, 763, 960
ADAMTSs, 960
Adapter protein complexes (AP complexes), $641,651-652$
Adapter proteins, 741-742, 793-796, 924
Adaptive immune system, 1079, 1080, 1080f, 1087f, 1088-1089
Ig production of B cells in, 1101-1102
lymph nodes initiating, 1083 f
ADCC. See Antibody-dependent cell-mediated cytotoxicity
Adenine, 45, 46f
post-transcription modifications of, 447
Adenoma, 1146, 1155
Adenosine, 701
Adenosine diphosphate (ADP), 6, 6f, 514
mitochondria and, 18
mitochondrial oxidation rate and, 558
Adenosine monophosphate (AMP/cAMP), $517,679,699,699 \mathrm{f}$
anchoring proteins localizing effects of, $705-706$

cellular responses to rise in, 703-704, 704f
glycogen metabolism regulation by, 703f
PKA activation by, 701-702, 702f, $703-704$
signal amplification and, 704
signal suppression, 706-707
Adenosine triphosphate (ATP), 6, 6f, 513, 514
axonal transport and, 834, 835
chloroplasts and, 18
generation of, 62-63
glycolysis regulation and demand for, $516-518$
glycolytic pathway and, 516, 517 f
hydrolysis of, 61-62, 61f
kinesin-1 cycle, 836-837, 837f
mitochondria and, 18
mitochondrial protein import and, 613
myosin movement and, 800, 801f
in photosynthesis, 563
post-translational translocation and, 591-592
proton-motive force and, 551-559
synthesis of, 515f, 516, 551-559, 551f, $552 \mathrm{f}, 563$
Adenovirus, 431-432, 967
Adenylyl cyclase, 692, 699f, 700f
$\mathrm{G}_{\mathrm{s},}$ binding to, 701, 701f
GPCRs activating or inhibiting, 699-707
stimulation and inhibition by receptor-
ligand complexes, 699-701, 700f
Adherens belt, 778
Adherens junctions, 796, 933, 935f
Adhesion molecules. See also Cell-adhesion molecules
evolution of, 928-929
JAMs, 940, 940f, 965
NCAM, 965, 1072
plant, 971-972, 971f
Adhesion receptors, 923, 925
Adhesive interactions, 922f, 961-967
leukocyte movement and, 966-967, 966f
Adipocytes, differentiation of, 770, 771f
Adipose cells
abnormal metabolism in, 770
adenylyl cyclase and, 700f
in obesity, 770
ADP. See Adenosine diphosphate
ADPKD. See Autosomal dominant polycystic kidney disease
Adrenocorticotropic hormone (ACTH), 655, $700-701$
AE1 antiporter, 506-507
Aequorea victoria, 146
Aerobic glycolysis, 1138, 1138f
Aerobic metabolism, 519 f
Aerobic oxidation, 513, 514f, 515, 533, 534f
Aerobic reactions, 63
Aerobic respiration, 513, 515
Afferent neurons, 1028
Affinity, 89, 681-682
signal sensitivity and, 683
Affinity chromatography, 111, 683-684
Aflatoxin, 1145
Agarose gels, 246

Aggrecan, 956, 957f
AGO2 protein, 450-451
Agonists, 683
Agrin, 1050f, 1051
AID. See Activation-induced deaminase
AIDS. See Acquired immune deficiency syndrome
AIRE, 1120
AKAPs. See A kinase-associated proteins
Akt, 750
Alanine, 43
Alb1 gene, 396
Alcoholic fermentation, 519 f
Aldehyde, 47
Algae, 4f, 22. See also Chlamydomonas reinhardtii
Alleles, 224
autosomal dominant, 255
dominant, 224-225
dominant-negative, 262-264, 265f
recessive, 224-225, 255
Allelic exclusion, 1100
Allogenic tissue transplantation, 1120
Allosteric regulation, 100-101, 518f
Allostery, 100
All-trans-retinal, 1046, 1046f
Alpha carbon atom $\left(\mathrm{C}_{\alpha}\right), 42$
Alpha helix, 70-71, 71f
$\alpha$-actinin, 793, 794f
$\alpha_{1}$-adrenergic receptor, 692
$\alpha_{1}$-antitrypsin, 607
$\alpha_{2}$-adrenergic receptor, 692
$\alpha$-like globins, 306, 307
$\alpha$-SNAP, 644
$\alpha \beta$-tubulin, 822-824, 827
Alport's syndrome, 950
Alprenolol, 683, 683f
ALS. See Amyotrophic lateral sclerosis
Alternative pathway, 1085, 1085f
Alternative polyadenylation, 450
Alternative RNA splicing, 181-182, 182f mRNA, 417, 438
neurological disorders linked to, 438, $438 \mathrm{t}-439 \mathrm{t}$
Alternative splicing, 304
in pre-mRNA processing regulation, 435
Alternatively spliced mRNAs, 417, 438
Alu elements, 320-321
ALV. See Avian leukosis virus
Alzheimer's disease, 87, 88f, 763, 764f, 984
Amanita phalloides, 792
Amino acids, 7, 42-45
codons to, 184, 184f
DNA information converted to, 9 f essential, 44
import against high concentration gradients, 503-504
linear arrangement of, 69-70
protein folding and sequence of, 81-82
proteins composed from, 42-45, 43f
side chains, $42-45,45 f$
symporters, 504, 505f
transport across epithelia, 508-509, 508f
tRNA activation of, 188

Aminoacyl-tRNA, 185, 186f, 191
Aminoacyl-tRNA synthetases, 185, 188
Aminoglycoside antibiotics, 527
Amoeba dubia, 309
Amoebidium parasiticum, 526
AMP kinase (AMPK), 453f, 454
AMPA receptors, 1073-1074, 1074f
Amphipathic, 32, 75
Amphipathic molecules, 273
Amphiphilic, 32
Amphitelic attachment, 898, 899f
Amplification, 679, 1151
of extracellular signaling, 679
in rhodopsin signal transduction pathway, 696-697
by signal transduction pathways, $679-680$
AMPPNP, 835
Amyloid fibrils, 87, 88f
Amyloid plaques, 87, 763
Amyloid precursor protein (APP), 763, 764f
Amyloids, 87, 88f
Amyotrophic lateral sclerosis (ALS), 984-986
Anaerobic metabolism, 519 f
Anaerobic respiration, 515
Anaphase, 851, 857f, 859, 876
Anaphase A, 857-858, 857f
Anaphase B, 857f, 858
Anaphase II, 913
Anaphase-promoting complex (APC/C), 886, 901f
separase activation by, 901-902
Anchoring junctions, 932, 932f
Anchoring proteins, 705-706
Androstenone, 1067
Anemia, 795
sickle-cell, 225
Aneuploid organisms, 225
Aneuploidy, 850, 909, 916, 1137
Anfinsen, Christian, 82
Angiogenesis, 1141
Angiotensin, 808
Animal cloning, 983
Anion antiporter, 506-507
Aniridia, 364
Ankyrin, 794, 795
Annular phospholipids, 287, 287f
Antagonistic interactions, 1004
Antagonists, 683
Antenna complex, 563
Anterograde transport vesicles, 632
Antibiotics
resistance to, 322-323
ribosomes and, 190
Antibodies, 89, 1081, 1088-1089, 1088f. See also Immunoglobulins
antigen-specific, 1091
in immunoelectron microscopy, 159, 159f
monoclonal, 135-136, 136f, 144, 684, 1080, 1094
for oncoproteins, 1158
organelle-specific, 162-164, 164f
phosphorylated peptide-specific, 684
polyclonal, 144

production of, 1128, 1129 f
Antibody assays, 111-112
Antibody-affinity chromatography, 110f, 111. See also Immunoprecipitation
Antibody-dependent cell-mediated cytotoxicity (ADCC), 1094
Anticodon, 183
in nonstandard base pairing, 186, 187f, 188
Antidepressant drugs, 1057
Antidiuretic hormone, 483
Antigen processing and presentation, $1109-1115$
Antigen-presenting cells (APCs), 1084, $1127-1128$
professional, 1105-1106
Antigens, 1081, 1083f, 1089
ABO blood group, 289-290, 290f
in antibody production process, 1128
large T-antigens, 199, 200f
MHCs binding, 1107-1109, 1108f
PCNA, 200
SC40 large T-antigen, 624
Sialyl Lewis-x, 966
Antigen-specific antibodies, 1091
Antigen-specific receptors, 1118, 1120
Anti-growth factors, 881
Anti-mitogens, 889
Antiporters, 476, 502-507, 558, 1052-1054, 1053 f
Antisera, 1089
Antitermination factors, 377, 377f
Antithrombin III, 955, 955f
AP complexes. See Adapter protein complexes
AP1, 388
Apaf-1, 1020f
APC/C. See Anaphase-promoting complex
APCs. See Antigen-presenting cells
Apical Par complex, 1007
Apical surface, 24, 508, 657, 658f, 931
aPKC, 1007
Aplysia californica, 1070, 1072f
APOB gene, 439, 440f
Apoptosis, 731, 751, 906, 1011
$\mathrm{Ca}^{2+}$ in, 1022
caspases in, 1015
evolutionarily conserved pathways in, 1013-1015, 1014f
genes regulating, 1163
mitochondria and, 1017, 1018
neurons and, 1015-1017
phagocytosis in, 1012, 1012f
programmed cell death through, 1012-1013, 1012f
regulation genes for, as tumor-suppressor genes, 1163
regulation in vertebrates, 1020-1021
signaling pathways in regulation of, 1019f
Apoptosome, 1018, 1020f
Apotransferrin, 663, 664f
APP. See Amyloid precursor protein
Aquaporins, 286-287, 481-483, 482f
Arabidopsis thaliana, 20f, 525, 561
meristems of, 998, 998f, 999f

Archaeans, 4f, 10
ARF protein, 639, 640
Arginine, 43, 184
Argonaute protein, 448
Arp2/3 complex, 786, 787-789, 788f, 789f, 790f, 791f
Arrestin, 697-698, 698f
ARSs. See Autonomously replicating sequences
Ascorbic acid, 951-952
Asialoglycoprotein receptor, 600, 600f
Asparagine, 44
Aspartate, 44, 488f
in $\mathrm{Ca}^{2+}$ ATPase, 487 f
in P-class pumps, 488
Aspartic acid, 43
Assays, 111-114, 113f. See also specific assays
Asters, 850, 853
Asthma, 683
Astral microtubules, 851
Astrocytes, 985-986, 1030-1031, 1031f, 1032, 1044
gap junctions connecting, 1060
synapses and, 1049f
Asymmetric carbon atom, 34
Asymmetric cell division, 27, 903, 975, 976f, 1002 f
Ataxia telangiectasia, 906
ATF6, 607
Atherosclerosis, 40, 296, 439-440, 1022
ATM, 906, 908
Atoms
bonding properties in biomolecules, 34t
electron structure of, 33-34
ATP. See Adenosine triphosphate
ATP synthase, 534f, 551, 551f, 553-554, 553f, 557 f
protons passing through, 555-556
ATP/ADP antiporters, 558
ATP/ADP transport system, 556, 558, 558f
ATPase, 486
AAA family, 608
actin-activated activity of, 798
$\mathrm{Ca}^{2+}, 486-489,487 \mathrm{f}, 488 \mathrm{f}, 489 \mathrm{f}$
$\mathrm{H}^{+} / \mathrm{K}^{+}, 509$
$\mathrm{Na}^{+} \mathrm{K}^{+}, 489,489 \mathrm{f}, 490 \mathrm{f}$
ATPase fold, 779
ATP-powered pumps, 475f, 476, 483-494
classes of, 484-485, 484f
ATR, 906
Atractylis gummifera, 558
Atractyloside, 558
Attenuation, 361, 1130
AUG codon, 184, 190
and translation initiation, 191, 193
Augmin complex, 826, 858
AU-rich elements, 447
Aurora B, 899
Aurora kinase, 897
Autocrine signaling, 675f, 676, 763
Autoimmune diseases, 986, 1141. See also specific diseases
desmoglein and, 938
TNF $\alpha$ and, 681

Autoimmunity, 1079
Autonomously replicating sequences (ARSs), 345
Autophagic pathway, 667-669, 669f
Autophagic vesicle, 667
Autophagosome, 453, 667
growth and completion, 668
nucleation of, 668
targeting and fusion, 669
Autophagy, 16, 452, 529, 667
Autoproteolytic cleavage, 1014
Autoradiography, 114-116, 634, 684
Autosomal dominant alleles, 255
Autosomal dominant polycystic kidney disease (ADPKD), 849
Autosomal recessive disorders, 255f, 621
Autosomal recessive mutations, 621
Autosomes, 341
Auxins, 969
Avian leukosis virus (ALV), 1152
Avidity, 1090, 1120
Axon termini, 1027
neurotransmitter and synaptic vesicle cycling in, 1053 f
synaptic vesicles in, 1049 f
Axonal transport, 833-834, 834f, 835
Axonemal dynein, 844, 846f
Axoneme, 844
Axons, 1016, 1027
anterograde transport down, 835
growth cones of, 868, 868f
information moving along, 1027-1028
myelinated, 1043, 1043f
neurofilaments in, 864

## B

B box, 413
B cells, 1081, 1082, 1083f, 1104
activation of, 1128
antibody production and, 1128, 1129f
antigen-specific receptors triggering, 1118,1120
class switching, 1102-1103, 1103f
development of, 1095, 1099, 1100-1101, 1121 f
Ig production in adaptive response, $1101-1102$
Ig production types, 1102-1103
Bacille Calmette-Geurin (BCG), 1130
Bacillus subtilis, 357
$x p t-p b u X$ operon, 362, 363 f
Backward slippage, 310, 310f
Bacteria, 10. See also specific bacteria
ATP synthesis in, 552-553
commensal, 1081, 1083
cyanobacteria, 10
eubacteria, 4f, 10
G proteins and toxins from, 692
gene expression control in, 356-362
genes in, 8
innate and adaptive immune responses to, 1087, 1087f
$\mathrm{K}^{+}$channels in, 498
mycobacteria, 1130

$\mathrm{Na}^{+} /$amino acid symporters in, 504, 505 f operons, expression of, 361-362, 362 f purple, 567, 568f release factors, 195 replication of, 1081 ribosomes, 189 f
RNA polymerase classes, 368, 368f
Bacterial insertion sequences, 314, 314f, 315f
Bacteriochlorophyll, 567
Bacteriophages, 212
temperate, 216
Bacteriorhodopsin, 286, 286f
Bad, 1018-1019, 1019f
bag of marbles gene (bam gene), 990
Bak, 1018, 1019f
Balbiani rings, 442-443, 443f
bam gene. See bag of marbles gene
Band 4.1, 794, 795
Band-shift assay. See Electrophoretic mobility shift assay
Bardet-Biedl syndrome, 849
BARK. See $\beta$-adrenergic receptor kinase
Barth's syndrome, 547
Basal body, 825, 844
Basal lamina, 24, 134, 931, 931f, 945-950, 946 f
collagens in, 945, 948-950
Basal layer, 863
Basal surface, 24, 931
Base excision repair, 205, 205f
Base pairs, 171
nonstandard, 186, 187f, 188
RNA splicing and, 424, 425f, 426
Basement membrane, 947
metastasis and, 1141, 1142f
Bases, 8, 55
damaged, 205
of nucleic acids, 46 f
Basic helix-loop-helix (bHLH), 75-76, 386, 387-388
Basic keratins, 863, 864t
Basic local alignment search tool (BLAST), $324-325,324 f$
Basic zipper (bZIP), 386, 387-388
Basolateral surfaces, 24, 508, 657, 658f, 931
Bax, 1018, 1019f
B-cell malignancies, 725
B-cell receptor (BCR), 1101, 1118
signal transduction from, 1119 f
B-cell tumors, 1091-1092
BCG. See Bacille Calmette-Geurin
bcl-2 gene, 1013
Bcl-2 homology domains, 1017
Bcl-2 protein family, 1017, 1017f, 1019f
BCR. See B-cell receptor
BCR-ABL protein kinase, 1156-1157, 1156f
Beige-fat cells, 559
Bence-Jones proteins, 1092
Benzo(a)pyrene, 1144, 1144f
Beta sheet, 71, 72 f
Beta strands, 71
Beta turn, 70, 71, 72 f
Beta-blockers, 683
$\beta$ islet cells, 480, 767, 767f, 986, 987f
$\beta$-adrenergic receptor kinase (BARK), 707, 757
$\beta$-adrenergic receptors, 687, 688f
$\beta$-arrestin, 707, 707f
$\beta$-galactosidase, 992
$\beta$-globins, 307
$\beta$-like globins, 306
BH3-only protein, 1014, 1018-1019, 1019f, 1020-1021
bHLH. See Basic helix-loop-helix
Bicarbonate, 506, 510
Bile acids, 278
Bim, 1019 f
Binding assays, 681-682, 682f
Binding specificity, 676
Binding-change mechanism, 554-555, 555f
Biochemical energetics, 57-64
Bioinformatics, 324
Biological fluids, pH values of, 54-55
Biology
common functional groups and linkages, 35 t
evolution and, 1
Biomarkers, 122
Biomembranes, 48-50, 271
bilayer structure of, 273f, 274
cell types and, 276 f
cholesterol and sphingolipid clustering in microdomains of, 282-283
fluid mosaic model of, 272, 272 f
lipid and protein mobility in, 278-279, 280 f
lipid classes in, 276, 277, 278
physical properties and lipid composition of, 279-281, 281f, 281t
protein removal from, 290-292, 291f, 292f proteins in, 284-292
Biomolecules, bonding properties of common atoms in, 34t
Bi-oriented chromatids, 897
Bi-oriented kinetochores, 913, 917
Biosynthetic pathways, ordering of, 230-231, 231f
BiP chaperones, 591-592
Bitter taste, 1065-1066
Bivalents, 912
BLAST. See Basic local alignment search tool
Blastocyst, 980f, 981
Blastopore, 26
Blastula, 879
Blood cells, 4f, 922
formation of, 728f, 994-995, 995f white, 1082
Blood glucose, 479, 766-767, 767f diabetes mellitus and, 769
Blood transfusions, 1104
Blood-brain barrier, 1031, 1031f
B-lymphocytes, monoclonal antibodies from, 135
BMP. See Bone morphogenic protein
BMP4. See Bone morphogenic protein 4
BNDF. See Brain-derived neurotrophic factor
Bone marrow
as stem-cell niche, 997f
transplants, 995, 996f
Bone morphogenic protein (BMP), 722, 1011f
Bone morphogenic protein 4 (BMP4), 982
Bone resorption, 510, 510f
Bordatella pertussis, 692
Botulinum toxin, 1055
Botulism, 1055
Boundary elements, 333, 338
Boundary membrane, 522
Boveri, Theodor, 1137
Brain Research through Advancing Innovative Neurotechnologies Initiative (BRAIN Initiative), 1025
Brain-derived neurotrophic factor (BNDF), 1015
Branch migration, 209
Branch-point A, 423
BRCA1 gene, 1153-1154, 1168
BRCA2 gene, 1168
BRE. See TFIIB recognition element
Breast cancer, 683, 1157
Bright-field light microscopy, 139, 140f, 142f
Bromodomain, 333, 394
Brown-fat tissue, 558-559
Budding yeast, 877-878, 877f, 890f
Buffers, 55-56, 56f
Bundle sheath cells, 577
Burkitt's lymphoma, 1161, 1161f, 1164
Buspirone, 687t
bZIP. See Basic zipper

## C

$\mathrm{C}_{3}$ pathway, 573
C3 protein, 1085-1086, 1085f
$\mathrm{C}_{4}$ pathway, 576-578, 577f
$\mathrm{C}_{\mathrm{n}}$. See Alpha carbon atom
$\mathrm{Ca}^{2+}, 437,437 \mathrm{f}, 485,486$
in apoptosis, 1022
cytosol concentrations of, 489, 713, 713f
GPCRs and, 708-709
$\mathrm{IP} / \mathrm{DAG}$ pathway and cytosolic, 710, 711f, 712
MAMs and, 532
movement of, 712 f
neurotransmitter release and, 1054-1055
plasma-membrane store-operated channel for, 713
responses to hormone-induced rise in, 708, 709t
as second messenger, 679
skeletal muscle contraction and, 805-807, 806f
transport to mitochondrial matrix, 712-713
$\mathrm{Ca}^{2+}$ ATPases, 486-489, 487f, 488f, 489f
$\mathrm{Ca}^{2+}$ pump, 486-489
$\mathrm{Ca}^{2+}$-activated $\mathrm{K}^{+}$channels, 437, 437f
$\mathrm{Ca}^{2+}$-calmodulin complex, 713-714
$\mathrm{Ca}^{2+} / \mathrm{NOcGMP}$ pathway, 714-716, 715f
Cadherins, 924f, 933-938, 965
extracellular domains of, 936
tissue differentiation and, 936
Caenorbabditis elegans, 20f, 24-25, 921, 976

apoptotic pathway in, 1013-1015, 1013f
cell lineage in, 1005 f
Par proteins and, 1003-1006
tubulins in, 823
Cajal bodies, 468-469, 468f
CAK. See CDK-activating kinase
Calcium. See also $\mathrm{Ca}^{2+}$
GPCRs and levels of, 708-716
noncovalent binding of, 101-102, 101f
synaptic plasticity and, 1072-1073
Calcium indicators, 1054-1055, 1055f
Calcium ions, 679
Calcium signaling, 532
Callilepis laureola, 558
Calmodulin, 101, 101f, 489, 679, 713-714, 715, 1072
Calnexin, 605
Calories, 58
Calreticulin, 605
Calvin cycle, 573, 574, 575f, 617
CamKII $\alpha, 1072-1073$
cAMP. See Adenosine monophosphate
cAMP receptor protein (CRP), 357
cAMP-dependent protein kinase. See Protein kinase A
cAMP-response element (CRE), 705
CAMs. See Cell-adhesion molecules
Cancer. See also Tumors; specific cancers
cell cycle and maintenance pathway deregulation in, 1163-1168
cells, mutations in, 1137
cellular housekeeping in cells of, 1137-1139
diagnosis and treatment, 1157-1158
DNA-repair system loss and, 1166-1168, $1167 t$
epigenetic changes in development of, 1155
escaping tissue boundaries, 1140
genes in onset of, 1150 t
genetic basis of, 1149-1158
genome sequencing, 1157
Hh signaling and, 757
immune system defenses, 1131
incidence of, 1145, 1145 f
karyotypes, 1137, 1138f
metabolic pathways, 1137-1139, 1138f
misregulation in, 1159-1163
mTORC1 pathway in, 454
multi-hit model of, 1145-1146
mutations and development of, 1135-1136, 1150f
origins and development of, 1143-1148
p53 gene and, 908
PTEN gene and, 751
Ras proteins and, 739, 1146, 1148, 1155, 1160
signaling pathway aberrations and, $1161-1163$
skin, 744
studying development of, 1146-1148, $1148 f$
telomerase and, 348-349
TGF- $\beta 1$ and, 722, 725
transcription factors and, 1160-1161
uncontrolled proliferation in, 1139-1140
viruses causing, 1152
Wnt signaling and, 753
Cancer stem cells, 1140-1141
CAP. See Catabolite activator protein
CAP site, 357
Capacitors, 496
Cap-binding complex (CBC), 441
Capping proteins, 785, 785f
Capsaicin, 1062
Capsids, 213
CapZ, 785, 789, 790f
CAR. See Coxsackievirus and adenovirus receptor
Carbohydrates, 46, 578
Carbon dioxide transport, 505-506, 506f
Carbon fixation, 63, 561, 576f
Carbonic acid, 505
Carbonic anhydrase, 505
Carboxy-terminal domain (CTD), 370, 370f, $420-421,428,428 f$
Carcinogens, 1136, 1143, 1144-1145, 1144f
Carcinomas, 1136
hepatocellular, 1141
CARD, 1020f
Cardiac muscle, 504-505, 807, 986
Cardiolipin, 547
Cardiomyocytes, 986
Cargo proteins, 632, 646f
targeting sequences on, 641
Carotenoids, 564
Carriers, 476
Cartilage, 956, 957f
Cas genes, 266, 267f
Cax9, 266, 267f
Caspase-1, 1126
Caspase-8, 1021-1022
Caspases, 1015, 1018
Catabolism, 63, 516
Catabolite activator protein (CAP), 357. See also cAMP receptor protein
Catalase, 17, 619
Catalysts, 52, 90
enzymes as, 90-91, 90f, 91f, 93f
protein-folding, 606-607
Catalytic RNA, 466
Catalytic site, 91
Cataracts, 944
Catecholamines, 1052f
Cation antiporter, 504
CBC. See Cap-binding complex; Nuclear cap-binding complex
CBP. See CREB binding protein
CBP/P300, 705
CD4, 1121-1122
CD8, 1121-1123
CD44, 956
cdc mutants, 228, 229f, 878, 881
Cdc14, 902, 902f
Cdc14 phosphatase, 910
Cdc25, 896, 896f
inhibition of, 906
Cdc42, 813-816, 816f, 867-868, 868f, 1007
cell asymmetry and, 1006
cell polarity and, 1003, 1004f
intrinsic polarity program and, 1000-1002, 1001f
Cdc45-Sld3 complex, 892
CDK inhibitors (CKIs), 886-887, 904
CDK2, 884f
CDK-activating kinase (CAK), 886
CDKs. See Cyclin-dependent kinases
cDNA libraries, 238-239, 240f
cDNAs. See Complementary DNAs
CDRs. See Complementarity-determining regions
C/EBP $\alpha, 770,771 f$
CED-3, 1013-1015, 1018, 1020f
ced-3 gene, 1013-1014, 1013f
CED-3 protease, 1014, 1015f
CED-4, 1013-1015, 1018, 1020f
CED-9, 1013-1015, 1017, 1017f
ced-9 gene, 1013-1014
Celexa, 34
Cell asymmetry, 1006
Cell attachments, breaking, 812
Cell body, 1027
Cell cortex, 778
Cell cultures
establishment of, 131, 131f
primary, 131-132
tissue, 880
Cell cycle
CDKs controlling, 876, 883f
commitment to, 887-895
deregulation in cancer of, 1163-1168
DNA damage response system halting, 806f, 905-908
DNA replication initiation in, 890-891, 891f, 892-893, 893f
eukaryotic cells, 18-19, 19f, 873-874, 874f, 904f
extracellular signals and entry to, 889-890
key principles of, 876
mammal regulation of, 880
model organisms and methods of study of, 877-881
phases of, 874f, 875-876
proteins in, 21-22
surveillance mechanisms in regulation of, 904-911
Cell death
apoptosis, 1012-1013, 1012f
programmed, 906, 977, 1011, 1011f, 1012-1013, 1012f
regulated, 1011-1022
Cell division
asymmetric, 27, 903, 975, 976f, 1002f
cell polarization before, 1002-1003, 1002f
eukaryotic cell regulation of, 18-19, 19f
symmetric, $975,976 f$
Cell exterior, targeting to, 585
Cell fates, 977
cell location and, 980 f
Cell junctions, 923, 924
types of, 932-933, 932f, 933t

Cell lineage, 975, 976f, 1005f
Cell lines, 132
HeLa, 132, 468f, 881f
Cell locomotion, 811-812, 812f, 813f
Cell membrane, 5 f
Cell migration, 811-817, 867-868
Cell polarity, 776, 776f, 1000-1010
cell division and, 1002-1003, 1002f
Cell signaling, 674f
Cell strains, 131
Cell walls
in bacteria, 10
of eukaryotic cells, 13 f
plant, 968-969, 968f
plant cell mitosis and, 860
rigid, 968
Cell-adhesion molecules (CAMs), 24, 131, 603, 922f, 923-925, 925f, 928f
evolution of, 928-929
families of, 924 f
mechanotransduction and, 929
Cell-attached patch clamping, 500 f
Cell-body translocation, 812
Cell-cell adhesive interactions, 922f, 925f
Cell-ECM adhesions, 938-939
Cell-free assay, 610, 610f
Cell-free protein synthesis, 587, 587f
Cell-free translocation, 612
Cell-free transport assays, 637-638, 637f
Cell-matrix adhesive interactions, 922 f
Cells
ABC proteins exporting toxins from, 491-493
ancestral, 2 f
biomembranes and types of, 276 f
blood, 4 f
chemical building blocks of, 41-51, 42f
chemical reactions in, 52-53
cytokines influencing development of, 727
endothelial, 931
energy transformation in, 58
epithelial, 775-776, 776f
excitable, 1027
lipid storage in, 283, 283f
lipid uptake by, 659-660
mast, 1087
memory, 1123
mesenchymal, 936
migrating, 776 f
organelle release from, 162
pancreas, 18 f
postmitotic, 876
splitting duplicated, 859-860
variety of, 4 f
Cell-substratum adhesions, 811-812
Cell-surface receptors, 674
affinity for ligands, 681-682, 682f
binding assays for detecting, 681-682
master transcription factors and, 721
purification with affinity chromatography, 683-684
signal sensitivity and, 683
studying, 681-686
types of, 720 f

Cellular communication, 673
Cellular differentiation
DNase I hypersensitive sites and, 398, 399f, 400
gene activation during, 395-396
Cellular membranes, 9, 9f, 473
in eukaryotes, 271, 272f
faces of, 275, 275f, 276f
water permeability of, 481-483
Cellular oxidative stress, 548
Cellular respiration, 63
Cellulose, 968-969
Cellulose synthase, 969
Centimorgan, 233
Central dogma, 168
Central nervous system, 1026, 1044
Centralspindlin, 859
Centrifugation, 106, 107f
density-gradient, 124f, 163f
organelle separation, $162,163 \mathrm{f}$
Centrioles, 825, 897
Centromeres, 329, 345, 853
of S. pombe, 346-347
of Saccharomyces cerevisiae, 346-347, 346 f
sequences length and complexity, 345-347
Centromeric heterochromatin, 311
Centrosome disjunction, 850, 898
Centrosomes, 825, 826f, 897
duplication of, 849-850, 849f
Cervical cancer, 218, 1130
CFI. See Cleavage factor I
CFII. See Cleavage factor II
CFTR. See Cystic fibrosis transmembrane conductance regulator protein
CFU-E. See Colony-forming units-erythroid cGAS, 1085
cGMP. See Cyclic guanosine monophosphate
cGMP phosphodiesterase (PDE), 695, 699f, $705,706 f$
Chain elongation, 193-195, 193f, 428, 428f
Chance, Britton, 546
Channel-inactivating segment, 1036, 1042-1043
Channels, 475f, 476. See also Ion channels; specific channels
gated, 476,497
half, 556
piezo, 1063, 1064f
Chaperones, 82-83, 328, 591
BiP, 591-592
mitochondrial protein import and, 610
protein folding mediated by, 83-85, 84f, 604-606, 605f
protein folding promoted by, 82-86
Chaperonins, 83, 85-86, 86f
Chara, 810
Charcot-Marie-Tooth disease, 944, 1045
subtype 2A, 529
Checkpoint pathways, 874, 876, 904, 905, 1149
growth, 905
spindle assembly, 856-857, 908-909, 908f
spindle position, 909-911, 909f
Chemical adducts, 206-207
Chemical bond energy, 32 f
Chemical building blocks, 32 f
of cells, 41-51, 42 f
Chemical defenses, 1083-1084
Chemical equilibrium, 32f, 51-57, 52, 53f
Chemical libraries, 137-138
Chemical potential energy, 57
Chemical reactions, 51-57
in cells, $52-53$
in equilibrium, 52
time dependence, 52 f
Chemical synapses, 1060
Chemiluminescence, 112
Chemiosmosis, 514, 552f
Chemiosmotic coupling, 514
Chemiosmotic hypothesis, 551, 553
Chemokines, 1087, 1124
Chemotactic signals, 1086, 1124
Chemotaxis, 816-817, 817f
Chemotherapy, 1157
Chiasmata, 912, 915f
nondisjunction and, 916
Chimeric proteins, 130, 612-613, 612f
Chiral carbon atoms, 34
Chirality, 34
Chironomus tentans, 442, 443f
Chk1, 906
Chk2, 906, 908
Chlamydomonas reinhardtii, 20f, 22, 570, $572,845,846 f, 847,1046,1046 f$
Chloramphenicol, 527
Chloride channels, 510, 510f
Chlorophylls, 560, 563f
Chloroplasts, 13f, 18, 514, 524f, 569f
ATP synthesis in, 552-553
DNA in, 523, 560-561
photosystems in, 567-568
protein import, 617, 618f
protein targeting to, 608-617
structure of, 561 f
thylakoid membranes in, 560
Cholera, 692, 942
Cholesterol, 39-40, 50, 277f, 278, 280
biosynthetic pathway, 295-296, 295f
clustering with proteins, 282-283
regulation of levels of, 763-766
SCAP binding, 765
SREBP activation and, 765, 765f
synthesis and intracellular movement, 293-297
transport mechanisms, 296-297, 296f
Cholesterol esters, 50
Chromatids, 341
bi-oriented, 897
crossing over, 912
middle prophase, 339
sister, 15f, 851, 875, 893, 894f, 895, 899, 913
Chromatin, 302, 327, 392f
condensation of, 354, 364
condensed, 356f, 391, 393, 395f

conservation of structure, 329
decondensation of, 364, 394, 395, 395f
DHS patterns in, 398
eukaryotic transcription regulation and, 364
forms of, 328-329, 328f
histone tail modification and, 330-335
loops of, 335-339, 335f
nuclear lamina and, 865, 866f
nucleosome structure and, 328
30-nm fiber, 328-329, 330f
Chromatin immunoprecipitation, 373, 374f, 376
Chromatin-mediated repression, 390
Chromatin-remodeling complexes, 395, 1155
Chromium release assay, 1106f
Chromodomain, 333
Chromogenic enzyme reactions, 111
Chromogranin A, 655
Chromogranin B, 655
Chromonema fiber, 339
Chromoshadow domain, 333
Chromosomal passenger complex (CPC), 855-857, 858f, 859, 899
Chromosomal translocations, 1151
Chromosome painting, 341-342, 342f
evolution revealed by, 342-343, 343f
Chromosomes, 14, 15f, 169. See also
Metaphase chromosomes;
X chromosome
aligning duplicate, 854-855, 855f
capture and orientation, 853-854, 855f
compaction, 900, 900f
condensation, 340f, 899-900
conformation capture, 336-337, 338f
congression, 854, 855f
elements for replication and stable inheritance, 345, 346f
evolution of primate, 342-343, 344f
gene organization in, 309-312
homologous, 912
interphase, 335, 335f, 343-345, 345f
interphase territories, 336, 337f
mitotic spindle attachment, 898, 898f
morphology and functional elements of eukaryotic, 341-349
movement to poles, 857-858, 857f
Philadelphia, 1156, 1156f
polytene, 343-345
in prometaphase, 853-854
reassortment, 911
replication, 874,875
segregation, 874, 899-900, 901
shortening of, 347, 348f
structural organization of eukaryotic, 327-340
structure of, 302 f
telomeres preventing shortening of, 347-349
topological domains within territories, 336-339
Chromothripsis, 1137
Chronic bronchitis, 683

Chronic lymphocytic leukemia (CLL), 1155, 1158
Chronic myelogenous leukemia (CML), $1156-1157$
Chronic progressive external ophthalmoplegia, 528
Cigarette smoking, 1144, 1144f
Cilia, 12, 276, 844-849
beating, 844-845, 846f
primary, 757, 847-848, 848f
structure of, 845 f
Ciliated epithelium, 14 f
Cimetidine, 687 t
Circulating tumor cells, 1142
Circulatory system, 1082 f
Circumferential belt, 807
cis-Golgi, 649
cis-Golgi cisterna, 632, 633f, 635
cis-Golgi network, 646-647, 647f
Cisternae, 16, 632
Cisternal maturation, 648-650, 650f
Cistrons, 303
Citalopram, 34
Citric acid cycle, 516, 532-536
CKIs. See CDK inhibitors
Clamp domain, 369, 369f
Clamp loader, 201
CLASPs, 831
Class C rhinoviruses (RV-C), 936-937
Class I MHC molecules, 1105-1107, 1107f
antigen processing and presentation path-
way, 1110-1112, 1110f
polypeptide antigen binding, 1108-1109
Class II MHC molecules, 1105-1107, 1107f
antigen processing and presentation path-
way, 1112-1115, 1112f
polypeptide antigen binding, 1109
Class switching, 1102-1103, 1103f
Classical cadherins, 933, 934-937, 935f
Classical genetics, 223
Classical pathway, 1085, 1085f
Clathrin, 163, 646, 651-652
coat structure, 652
dynamin and, 652-653, 652f
Clathrin-coated vesicles, 638
Claudin, 940
Claudin-1, 942
claudin14 gene, 941
Cleavage
autoproteolytic, 1014
of cohesins, $901,901 \mathrm{f}$
of mammalian embryo, 979
of pre-mRNAs, 430-432, 431f
$3^{\prime}, 419,430-432,431 f$
Cleavage and polyadenylation specificity factor (CPSF), 430, 452
Cleavage factor I (CFI), 430
Cleavage factor II (CFII), 430
Cleavage furrow, 859, 903
Cleavage stimulatory factor (CStF), 430
Cleavage/polyadenylation complex, 431
$\mathrm{Cl}^{-} / \mathrm{HCO}_{3}^{-}$antiporter, 506, 506f
CLL. See Chronic lymphocytic leukemia
Clonal selection theory, 1091, 1092f

Cloned DNA molecules
disease genes in, 257
sequencing, 243-244, 244f, 245 f
studying gene expression with, 246-253
Clones, 129
genes, protein production from, 250-251, 250 f
viral, 213
Cloning, 236-237
animal, 983
differentiation reversal and, 983
DNA, 234-244
plasmid vectors, 236-237
Clotting factor IX, 321
Clozapine, 687 t
Cluster analysis, 248-249, 250f
CMC. See Critical micelle concentration
CML. See Chronic myelogenous leukemia
$\mathrm{CO}_{2}, 505-506,506 f, 509$
erythrocyte transport of, 506-507
metabolism in photosynthesis, 573-578, 575 f
CoA. See Coenzyme A
Co-activators, 386, 387f, 395, 705
Coated pit, 16
Coated vesicles, 638, 639-641, 639t
Coatomers, 647
CoATs. See Cohesin acetyltransferases
Cocaine, 1057
Cochlear K+ channel, 438
Cockayne syndrome, 376
Codons, 181f, 183
to amino acids, 184, 184f
anticodon, 183, 186, 187f, 188
in nonstandard base pairing, 186, 187f, 188
start (initiation), 184, 190-191
stop (termination), 184, 195
Coenzyme A (CoA), 293, 533, 534f
Coenzyme Q (CoQ), 541, 541f, 542f
Coenzymes, 63, 95
Cofactor, 95
Cofilin, 784-785, 789, 790f
Cognate proteins, 380
Cohesin acetyltransferases (CoATs), 894f
Cohesins, 849, 893, 894f, 895, 913, 915f
cleavage of, 901, 901f
dissociation of, 900
in meiosis, 915-916, 915f
in mitosis, 915 f
nondisjunction and, 916
Co-immunoprecipitation (Co-IP), 113f, 114
Coincidence detection, 788
Co-IP. See Co-immunoprecipitation
Colchicine, 137, 829-830
Collagenases, 1084
Collagens, 925, 945, 946f, 959f
fibrillar, 951-952, 952f
triple helix, 948, 949f
type I, 952-953
type II, 952-953
type IV, 945, 948-950, 949f
types of, 948 t
Collapsed replication fork, 209-210, 210f

Colon cancer, 205-206, 725, 753, 1146, 1167
Colony-forming units-erythroid (CFU-E), 728 f
Colorectal cancer, 1147f
Combinatorial diversity, 938
Combinatorial transcription regulation, 387-388, 388f
Commensal bacteria, 1081, 1083
Common cold, 936
Compacted morula, 979
Compaction, 979
of chromosomes, 900, 900f
Compartment-specific oligosaccharide modifications, 635, 636f
Complement system, 1085-1086, 1085f
Complementarity-determining regions (CDRs), 89, 90f, 1093
Complementary, 171
Complementary DNAs (cDNAs), 239
cloned gene protein production with, 250-251, 250 f
Complementary matching, 8
Complementation tests, 229-230, 230f
Concentration gradient, 58
glucose and amino acid import against, 503-504
Condensin, 339, 340f, 900
Conditional mutations, 227-228, 878
Cone cells, 694, 1061
Confocal microscopy, 147-149, 148f, 149f
Conformations, 67
Congestive heart failure, 505
Connective tissue, 921, 951-960, 959f
Connexins, 942, 943f, 944
Connexons, 942, 943f, 944
Constant regions, 1092, 1094-1095
Constitutive expression, 383
Constitutive secretion, 632
Constitutive transport element (CTE), 444
Contact sites, 613
Contractile bundles, 807
Contractile ring, 778, 859, 903
Contractile vacuoles, 481
Control regions, 364, 365f, 366-367
Convergent evolution, 997
Convergent neural circuits, 1029, 1030f
Coomassie blue, 112
Cooperative DNA binding, 388, 389f
Cooperativity, 100, 101f
Coordinate expression, 179
Coordinately regulated genes, 357
Co-oriented sister chromatids, 913
Co-oriented sister kinetochores, 917
COPI vesicles, 638, 645
COPII vesicles, 638, 640, 640f, 645-647
CoQ. See Coenzyme Q
$\mathrm{CoQH}_{2}$-cytochrome $c$ reductase, 542, 544
Co-receptors, 942, 1122-1123
Co-regulated genes, 248-249, 250f
Co-repressor proteins, 387, 393
Cortical granules, 979
co-Smads, 724
Co-transcriptional processing, 419
Cotranslational translocation, 588-589, 589f

Cotransport, 502-507
Cotransporters, 476, 477t
cytosolic pH regulation by, 505-506
Covalent bonds, 33-40
defining, 33, 33f
equal or unequal sharing in, 34-36
relative energies, $36,36 \mathrm{f}$
strength of, 36
Covalently attached lipids, 288-289
coxII gene, 526
Coxsackievirus and adenovirus receptor (CAR), 940
CPC. See Chromosomal passenger complex
CPE. See Cytoplasmic polyadenylation element
CPEB. See CPE-binding protein
CPE-binding protein (CPEB), 451
CpG island promoters, 372-373
CpG islands, 372
CPSF. See Cleavage and polyadenylation specificity factor
Crambin, 38 f
CRE. See cAMP-response element
CREB. See CRE-binding protein
CREB binding protein (CBP), 386, 387f, 395
CREB protein, 386, 387f
CRE-binding protein (CREB), 705, 705f
Crick, Francis, 7, 170
CRISPR, 266
CRISPR-Cas9, 266, 267f, 268, 1158
Crista junction, 522
Cristae, 18, 522
Critical cell size, 889, 905
Critical concentration, 782
Critical micelle concentration (CMC), 291
Crohn's disease, 1141
Cross-bridge cycle, 804
Cross-exon recognition complex, 428
Cross-linking proteins, 793, 794f
Cross-presentation, 1111
CRP. See cAMP receptor protein
Crumbs complex, 1007
Cryoelectron microscopy, 119, 120f, 121, $160-161$
Cryoelectron tomography, 160, 160f
Crypts, 991, 991f, 993f, 994f
CStF. See Cleavage stimulatory factor
CTD. See Carboxy-terminal domain
CTE. See Constitutive transport element
CTLA4, 1131
CTLs. See Cytotoxic T cells
Culturing, 129
biological processes studied with, 136
ES cells, 981 f
media and surfaces for, 130-131
in vivo environment mimicked by, 133-135
Cyanide, 546
Cyanobacteria, 10
Cyanopindolol, 687, 688f
Cyclic electron flow, 568f, 570-571, 571f
Cyclic guanosine monophosphate (cGMP), 695-696, 696f
Cyclic photophosphorylation, 570-571
Cyclin D, 1164

Cyclin-dependent kinases (CDKs), 202, 722, 874, 888, 1164
activity regulation, 882-887, 882f
cell cycle control by, 876, 883f
cyclins and activity of, 884-885
fluctuation of, 876
genetically engineered, 887
nomenclature, 883, 883t
phosphorylation, 883, 886
regulators of activity of, 885-886, 885t
roles of, 883 t
Cyclins, 100, 874, 890f, 1164
CDK activity and, 884-885
regulators of activity of, 885-886, 885t
Cycloheximide, 527
Cyclosome, 886
Cyclosporine, 1120
Cysteine, 44
Cystic fibrosis, 255, 255f, 261, 494, 494f, 646
Cystic fibrosis transmembrane conductance regulator protein (CFTR), 485, 494, 494f, 646
Cytochalasin D, 791-792
Cytochrome $c, 1018$
Cytochrome $c$ oxidase, 542, 545-546, 549, 549 f
Cytochrome P-450 enzymes, 1143
Cytochromes, 540-541, 567
Cytokeratins, 863
Cytokine receptors, 726-733, 727f
binding to, 728-730, 729f
down-regulation of signaling from, 731-733, 733f
Cytokines, 727, 1086, 1087
binding to receptors, 728-730, 729f
HSC regulation by, 994
inflammatory, 758
T cells production of, 1123-1124
Cytokinesis, 778, 807f, 859-860, 876
daughter cells created by, 903
Cytoplasm, 8
mRNA degradation in, 445-447, 446g
post-transcription control mechanisms, $445-460$
Cytoplasmic dense bodies, 864
Cytoplasmic dynein, 838
Cytoplasmic inheritance, 524, 525f
Cytoplasmic mRNPs, 419
Cytoplasmic poly(A)-binding protein (PABPC/PABPC1), 195, 451
Cytoplasmic polyadenylation, 451-452, 452f
Cytoplasmic polyadenylation element (CPE), 451
Cytoplasmic sorting signals, 642 t
Cytoplasmic streaming, 810, 810f
Cytosine, 45, 46f
Cytoskeletal fibers, 13f
Cytoskeletal filaments, 14f
Cytoskeletal proteins, 7
Cytoskeleton, 271, 777, 777f, 778f
actin, 815 f
coordination and cooperation between elements of, 867-868

of epithelial cells, 776 f
functions of, 12
Cytosol
$\mathrm{Ca}^{2+}$ concentrations, $489,713,713 \mathrm{f}$
calcium concentrations in, 709
cholesterol synthesis in, 295-296
membrane protein N -terminus in, 597-598
misfolded protein degradation in, 607-608
muscle relaxation and, 486
$\mathrm{NAD}^{+}$and NADH concentrations, $535-536,536 f$
pH regulation, 485, 505-506
plasmodesmata connecting, 969-970, 970f
unassembled protein subunits degradation in, 607-608
Cytosolic face, 275
Cytosolic leaflets, 281-282
Cytosolic proteins, 99-100
autophagic pathway transporting, 667-669
lysosomes receiving, 667-669
nuclear transport and, 624 f
Cytotoxic T cells (CTLs), 1104, 1105, 1106f
CD8 co-receptor and, 1122-1123
cell killing by, 1123 f

## D

DAG. See 1,2-diaglycerol
Daltons, 70
Darvon, 34
Darwin, Charles, 1, 3f
DBD. See DNA-binding domains
DDK, 892, 897
De Materia Medica (The Materials of Medicine) (Dioscorides), 558
Deadenylation-dependent pathway, 446, 446f
Deadenylation-independent decapping pathway, 446, 446f
Deamination, 204, 204f
Death signals, 1021-1022
Debranching enzymes, 432
Deconvolution microscopy, 147, 147f
Defensins, 1087
Degradation
ECM, 960
Edman, 118
glycogen, 699, 700f
of misfolded proteins, 607-608
mRNA, 445-450, 446f
neurotransmitters, 1057
plasma membrane, 665, 666f
pre-mRNAs, 432
protein, 97-99, 99-100, 751-760
RNA, 432
RTKs, 738
of unassembled protein subunits, 607-608
Dehydration reaction, 41
Delayed response genes, 889
Deletional joining, 1096f, 1097f, 1098
Delta ligand, 761, 762 f
Dementia, 984

Demyelinating disease, 1045
Denaturation, 82, 172, 174
Dendrites, 1016, 1027, 1029f
Dendritic cells, 1082, 1087, 1125
Dendritic spines, 1071f
Density gradient, 106
Density-gradient centrifugation, 124f, 163 f
Deoxynucleotides (dNTPs), 241
Deoxyribonucleic acid (DNA), 2, 7-9, 8f, 45,167
A-form and B-form, 171-172, 172f
chemical and radiation damage to, 203-204
in chloroplasts, 523, 560-561
double helix of, 170-172, 171f
genome and, 14, 25
information encoded in, 9 f
intergenic, 312
intermediate-repeat, 312
mitochondrial, 18, 523-526, 524f, 525f, 526 f
moderately repeated, 312
noncoding, 301-302
nonfunctional, 309-310
nuclear eukaryotic, major classes of, 306 t in prokaryotes, 10
protein interaction with, 172, 172 f
repetitious, 301, 310
satellite, 310-311
sequences of, 8
simple-sequence, 310-311
strand separation, 172-174
torsional stress on, 174, 174f
transposable (mobile) elements, 312-323
unwinding duplex, 199
Deoxyribonucleotide triphosphates (dNTPs), 244, 245f
Dephosphorylation, 103f
in $\mathrm{Ca}^{2+}$ ATPase, 487 f
mRNP export and, 441-442, 442f
in protein regulation, 102-103
Depolarization, 1028
membrane, 1039-1040, 1042
plasma membrane, 1034, 1035f
Depolarized state, 695
Depurination, 205
Dermal tissue, 968
Desensitization, 683
Desipramine, 1057
Desmin, 864, 864t
Desmocollin, 937
Desmoglein, 937, 938
Desmosomal cadherins, 933, 937
Desmosomes, 5f, 863, 933, 937f
Desmotubule, 970
Destruction box, 886, 892
Detergents, 290-292, 291f, 292f
Detyrosylated microtubules, 843, 843f
Deubiquitinylation, 103-104
Deuterostomes, 26
Developmental defects, 353
DGC. See Dystrophin glycoprotein complex
DHFR. See Dihydrofolate reductase
DHSs. See DNase I hypersensitive sites

Diabetes insipidus, 483
Diabetes mellitus, 769
obesity and, 770
type I, 769, 986
type II, 504, 769, 770, 986
Diacidic sorting signal, 646, 646f
DIC microscopy. See Differential-interferencecontrast microscopy
dicer gene, 449, 449f
Dicer protein, 451
Dictyostelium amoebae, 816-817
Differential centrifugation, 106, 107f
Differential-interference-contrast microscopy (DIC microscopy), 141-142, 142f
Differentiation, 975, 981f
of adipocytes, $770,771 \mathrm{f}$
DNase I hypersensitive sites and cellular, 398, 399f, 400
of ES and iPS cells, 986
gene activation during cellular, 395-396
of model organisms, 24-29
reversal of, 983
of stem cells, 988, 989f
tissue, 936
Diffusion, 474-475, 474f, 476
DiGeorge syndrome, 448
Digoxin, 489, 505
Dihydrofolate reductase (DHFR), 612, 612f
Dimerization
receptor homodimerization, 728
of RTKs, 734-735
Dimerization arm, 736
Dimethyl sulfate (DMS), 1143
Dimethylallyl pyrophosphate (DMPP), 296
Dioscorides, 558
Diphtheria, 1088
Diploid cells, yeast, 21, 21f, 1003
Diploid organisms, 224
recessive lethal mutations in, 228
Dipole, 35, 35f
Dipole moment, 35
Direct repeat sequence, 314
Direct-acting carcinogens, 1143
Disaccharides, 47, 48f
Disease genes
human, locating and identifying, 254-259
linkage studies mapping, 256, 257f
multiple defect diseases, 257-259
Dishevelled gene, 1008, 1009f
Dislocation, 607-608
Disruption constructs, 260
Dissociation constants, 53-54, 55f, 681
Dissociation (Ds) elements, 314
Disulfide bonds, 44, 601
formation of, 603-604, 604f
Divergent neural circuits, 1029, 1030f
Divergent transcription, 372-373
Diversity, 1080
combinatorial, 938
DMD. See Duchenne muscular dystrophy
DMD gene, 370, 426
DMPP. See Dimethylallyl pyrophosphate
DMS. See Dimethyl sulfate
DNA. See Deoxyribonucleic acid

DNA amplification, 343-345, 1151, 1151f
DNA chips, 248
DNA cloning, 234-244
DNA damage checkpoint controls, 907f
DNA damage response system, 905-908, 906f
DNA Data Bank, 323
DNA fingerprinting, 311-312, 312f
DNA glycosylases, 205, 205f
DNA library, 237, 245f
DNA ligase, 199, 234-236
DNA looping, 360, 360f
DNA methylation, transcription repression through, 404-405
DNA microarray, 247-248, 249f
DNA polymerase $\beta$ (Pol $\beta$ ), 1168
DNA polymerase $\delta$ (Pol $\delta$ ), 200
DNA polymerase e (Pol e), 200
DNA polymerasees, 197-199, 200f
copying errors from, 203
error correction by, 203
proofreading function of, 203, 204f
DNA polymorphisms, 311
linkage mapping of human mutations with, 255-256
in multiple defect diseases, 258
DNA recombination, 203-212
double-strand break repair with, 207-211
homologous, 209-211, 211f
DNA repair, 203-212
base excision repair, 205
cancer and loss of, 1166-1168, 1167t
double-strand breaks, 207-211
excision-repair systems, 204-205
homologous recombination, 209-211, 211f
NHEJ, 208-209, 208f
transcription-coupled repair, 207
DNA replication, 197-202
bidirectional, 201-202, 201f, 202f
commitment to, 887-895
duplicate strand linking in, 893, 894f, 895
inhibition between meiotic divisions, 917
initiation of, 890-891, 892-893, 893f
proteins in, 199-201
semiconservative mechanism, 197, 198f
DNA response elements, 401f
DNA sequencing
cancer analysis with, 1157
cloned DNA molecules, 243-244, 244f, 245 f
evolution revealed by, 342-343
DNA synthesis, leading-strand and laggingstrand, 199, 199f
DNA transposons, 242-243, 243f, 313, 313f, 314-315, 315f, 322
DNA viruses, 212
DNA-binding domains (DBD), 382, 403f
of repressor proteins, 394f
structural types, 384-386, 385f
DNA-binding repressors and activators, $358-359$
DNase I, 331, 332f
DNase I footprinting, 380-381, 381f
DNase I hypersensitive sites (DHSs), 398, 399f, 400

DNP. See 2,4-dinitrophenol
dNTPs. See Deoxynucleotides; Deoxyribonucleotide triphosphates
Dolichol phosphate, 601-602, 602f
Domains, 76-78
Dominant alleles, 224-225
monogenic diseases and, 255
Dominant mutant alleles, 225, 225f
Dominant-active proteins, 814
Dominant-negative alleles, 262-264, 265f
Dominant-negative mutations, 225, 730
Dominant-negative proteins, 814
Dopamine, 1026, 1052f, 1057
Dopaminergic signaling, 1057
Dosage compensation, 334
Double helix, 8, 8f, 170
Double mutants, 230-231
Double-label fluorescence microscopy, 145, 146 f
Doublesex gene, 436
Double-strand breaks, 906
DNA-repair system loss and, 1168
repair of, 209-211, 211f
Doublet microtubules, 844-845
Down syndrome, 342, 909, 916
Downstream, 364
Downstream promoter element (DPE), 376
Doxycycline, 1148
DPE. See Downstream promoter element
Drosophila melanogaster, 20f, 24, 877
Dscam isoforms in, 439, 439f
gene tagging in, 243
germarium of, 990, 990f
Hh signaling in, 755-757, 756f
life cycle of, 879-880, 880f
mobile elements in, 315
mutation experiments with, 228
Polycomb and Trithorax complexes, 406-407, 408f
polytene chromosomes in, 343-345, 345f
Ras/MAP kinase pathway in, 739-741, 740 f
retinal neurons, 439
RNA splicing regulation in, 435-436
sexual differentiation control in, $435-436,435 f$
Wnt signaling in, 752-753
Drugs. See also Antibiotics; specific drugs antidepressants, 1057
in cell biological research, 136-138, 137t, 138 f
multidrug resistance, 491
tubulin proteins and, 829-830
Ds elements. See Dissociation (Ds) elements
Dscam isoforms, 439, 439f
DSIF, 369
Duchenne muscular dystrophy (DMD), 29, 29f, 255, 255f, 321, 426, 796, 964
Duplex DNA, unwinding, 199
Duplicated genes, 303-306, 304f
Duty ratio, 802
Dynactin, 840, 840f, 841
Dynamic instability, 827-829, 828f, 829f

Dynamin, 529, 652-653, 652f, 653f
synaptic vesicle recycling and, 1056-1057
Dynamitin, 840
Dyneins, 833-844, 838-841, 839f, 840f, 841-842, 841f, 853
spindle pole separation and, 858
Dysentery, 790
Dystroglycan, 964-965, 964f
Dystrophin, 794f, 796, 964
Dystrophin glycoprotein complex (DGC), 29, 29f, 964f, 965

## E

E. coli. See Escherichia coli

E2F transcription factor complex, 885
E2F transcription factors (E2Fs), 889
E3 ligases, 100
Early mammalian development, 977, 979
Early response genes, 745-746, 889
Ebola, 1130
EBS. See Epidermolysis bullosa simplex
E-cadherin, 924f, 934, 934f, 935f, 937f, 990
ECM. See Extracellular matrix
Ectoderm, 981f
Edman degradation, 118
EDMD. See Emery-Dreifuss muscular dystrophy
EF hand, 75
EF hand family, 679
Effector proteins, 689f, 814, 904
Effector specificity, 676
Efferent neurons, 1029
EFs. See Elongation factors
EGF. See Epidermal growth factor
EGF receptors, 736-737, 737f
Eggs, 977, 978f, 979
EGL-1, 1013-1015, 1014, 1014f
eIF2 complex, 191, 192f
eIF2 kinases, 454-455
eIF4 complex, 191, 192f
eIF4E-binding proteins (4E-BPs), 453
eIFs. See Eukaryotic translation initiation factors
80S initiation complex, 191, 192f
Elastase, 607
Elastic fibers, 959-960, 959f
Elastin, 951, 960
Electric energy, 57
Electric potential, 58, 495
across plasma membrane, 497, 498f
Electrical signals, 1026
Electrical synapses, 1060
Electrochemical gradient, 475
Electrogenic pumps, 490
Electron density map, 119
Electron microscopy, 130, 156-161, 157f, 161, 161f
metal shadowing for, 157-158, 158f
negative staining for, 157-158, 157f
sample preparation for, 158-159, 159f
Electron shuttles, 536, 536f
Electron transfer flavoprotein (ETF), 544
Electron transfer flavoprotein:ubiquinone oxidoreductase (ETF:QO), 544

Electronegativity, 34
Electron-transport chain, 514, 515, 516, $539-550,547 \mathrm{f}$
Electrophoresis, 107-109
Electrophoretic mobility shift assay (EMSA), 380, 382f
Electrospray (ES), 116, 117f
Electrostatic interactions, 36, 37f
Elongation, 353
Elongation factors (EFs), 193, 377, 428
Embryo, cleavage of, 979
Embryogenesis, recessive lethal mutations and, 228
Embryoid bodies, 981, 981f
Embryonic development, master transcription factors and, 25-27
Embryonic stem cell (ES cells), 131, 261, 977, 980-983, 986
$\beta$ islet cells from, 987 f
control of pluripotency of, 981-983, 982f
culturing, 981f
DHS pattern of, 398, 399f
differentiated cells from, 986, 987f
ICM as source of, 980-981
Emerson, Robert, 567
Emerson effect, 568
Emery-Dreifuss muscular dystrophy (EDMD), 866
Emphysema, 607, 683
EMS. See Ethylmethane sulfonate
EMSA. See Electrophoretic mobility shift assay
EMT. See Epithelial-to-mesenchymal transition
en passant synapses, 1048
ENaC channels, 1066
Encyclopedia of DNA Elements (ENCODE), 411
End-binding proteins, 832-833
Endergonic reactions, 58, 59f
Endocrine signaling, 675f, 676
Endocycle, 880
Endocytic pathway, 632, 633f
iron in, 663-664, 664f
LDL internalization, 662, 662f
Endocytic vessels, 778
Endocytosis, 632, 812
AMPA receptor removal by, 1073
fluid-phase, 790-791
microfilament function in, 790-791, 791f
receptor-mediated, 659-664, 660f, 738, 790
sorting signals for targeting, 660-661
Endoderm, 981f
Endoglycosidase D, 635, 636f
Endonuclease-mediated mRNA decay, 446f
Endonucleolytic pathway, 446, 446f
Endoplasmic reticulum (ER), 13f, 14, 16, 195, 486. See also Rough endoplasmic reticulum; Smooth endoplasmic reticulum
$\mathrm{Ca}^{2+}$ release from, $712,712 \mathrm{f}$
calcium concentrations in, 709
cholesterol synthesis and, 295-296
improper protein folding in, 606-607
membrane protein insertion into, 593-600
membrane protein topology, 598 f
mitochondria influenced by, 529, 531f, 532
PEK and, 455
phospholipid synthesis in, 279, 294, 294f
protein modification, folding, and quality control in, 601-608
protein targeting to, 583-585, 585-592
pulse-chase experiments with membrane of, 586
secretory pathway and, 645-648
secretory protein targeting to, 586-588, 587 f
sphingolipid synthesis and, 295
Endosomes, 16, 17f, 632
budding from membrane of, 665
multivesicular, 665-666, 667f
pH of, 662-663
receptor-ligand complex dissociation in, 662-663
Endosymbiont hypothesis, 18, 275, 524f, 552
Endosymbionts, 523
Endothelial cells, 931
Endothelin, 808
Endothelium, 24
Endothermic reactions, 59
End-product inhibition, 100
Energy
cells transforming, 58
forms of, 57-58
reaction rate and, 60-61
Energy charge, 517
Energy coupling, 62
Enhanceosome, 388, 389f
Enhancers, 303, 359-360, 367, 731
distant, 379
multiprotein complex formation on, 388-389
Entactin. See Nidogen
Entamoeba histolytica, 22
Enthalpy, 59
Entropy, 59
Envelope, 213
Enveloped viruses, 1083-1084
Enzyme catalysis, 89-96
Enzyme inhibitors, 95
Enzymes, 7, 68
active sites, 91-92, 91f
as catalysts, 90-91, 90f, 91f, 93f
in common pathways, 96
debranching, 432
pH dependence of activity, 95, 95f
proofreading activity, 188
serine proteases and active site, 92-96, 94f
torsional stress relieved by, 174, 174f
Epidermal growth factor (EGF), 76, 76f, $676,683,734-735,736 f, 761$, $763,813,1142 \mathrm{f}$
receptor activation, 736-737, 737f
Epidermolysis bullosa simplex (EBS), 864-865, 865f
Epigenetic code, 333

Epigenetic marks, 404, 406
Epigenetic memory, 333-334
Epigenetics
lncRNAs in, 409-411
Polycomb and Trithorax complexes, 406-407, 409
transcription regulation, 404-411
tumorigenesis and, 1155
X chromosome inactivation and, 334-335
Epilepsies, 1037
Epimerases, 47
Epinephrine, 676, 679, 683, 683f, 687, 691, 699-700, 1052f
glycogenolysis induced by, 703
Epithelia, 133, 922
barrier function of, 1081
keratin filaments in, 863
transport proteins in, 508-509, 508f
types of, 931f
Epithelial cells, 775-776
cytoskeleton of, 776 f
polarity in, 1007-1008, 1007f
Epithelial tissue, 921
Epithelial-to-mesenchymal transition (EMT), 936, 937f, 1142
Epithelium, 24, 133
Epitope, 135
Epitope tagging, 112, 145, 253
Epo. See Erythropoietin
EpoR. See Erythropoietin receptor
Equilibrium constant, 52
Equilibrium density-gradient centrifugation, 106
ER. See Endoplasmic reticulum; Estrogen receptor
ER signal sequences, 586-588
ER targeting sequence, 586-588
ERAD complex, 608
ER-Cre recombinase chimera, 992, 993f
ERE. See Estrogen receptor response element
Ergosterol, 277f
ERM. See Ezrin-radixin-moesin
ERMES complex, 529, 532
ER-to-Golgi intermediate compartment, 646-647
Erythrocytes, 4f, 1082
actin filaments and, 793-795
anion antiporter in, 506-507
aquaporins in, 483
$\mathrm{CO}_{2}$ transport, 506-507
GLUT-mediated glucose uptake, 478f, $479,480$
Erythropoietin (Epo), 682, 684, 685f, 727, 728, 728f, 729f, 994, 996, 1157
Erythropoietin receptor (EpoR), 728
ES. See Electrospray
ES cells. See Embryonic stem cell
Escherichia coli, 11-12
DNA polymerases of, 203
expression systems, 249-251, 250f
lac operon transcription, 357-358, 358f
plasmid cloning vectors and, 236-237, 236f, 237f

promoters, 359
ribosome components, 189t
sigma factors, $357,359,359 t$
transcription initiation in, 357
trp operon, 361-362, 362f, 363f
ESCRT proteins, 666, 667
Essential amino acids, 44
Essential light chain, 797
EST. See Expressed sequence tag
Esterification, 46
Estrogen receptor (ER), 992, 993f
Estrogen receptor response element (ERE), 400, 401f
ETF. See Electron transfer flavoprotein
ETF:QO. See Electron transfer flavoprotein:ubiquinone oxidoreductase
Ethylmethane sulfonate (EMS), 225, 1143
Eubacteria, 4f, 10
Euchromatin, 332, 333f, 390
epigenetic memory and, 333-334
Euglena gracilis, 523
Eukaryotes, 9 f
alternative RNA splicing, 181-182
cell cycle, 18-19, 19f, 904f
cell structure and function, 12-19
cellular membranes, 271, 272f
chain elongation, 193-195, 193f
chromodomain, 333
cilia and flagella in, 844
defining, 12
DNA transposons in, 314-315
facultative anaerobes, 518
gene density, 307f, 309
gene expression control in, 363-370
gene organization, 179, 180f
gene structure, 303-309
genes in, 8
internal membrane structures, 14, 16-18
model organisms, 20 f
morphology and functional elements of chromosomes, 341-349
mRNA processing in, 420 f
obligate aerobes, 518
parasitic, 1082
precursor mRNAs, 180-181, 181f
pre-mRNA processing, 419-433
regulation of division, 18-19, 19f
regulatory elements, 364, 365f, 366-367
release factors, 195
ribosome recycling in, 195
RNA polymerase classes, 367-370, 367f, $367 t, 368 f$
scaffold proteins separating MAP kinase pathways in, 746-748, 747f
specific gene inactivation in, 259-268
structural organization of chromosomes, $327-340$
subcellular organization of, 13 f
transcription units, 303-305, 305f
transcription-control elements in, $379-380,379 f$
translation initiation, 191, 192f, 193
translation termination in, 194f
unicellular model organisms, 19-24
Eukaryotic translation initiation factors (eIFs), 191
European Bioinformatics Institute, 323
European Molecular Biology Laboratory, 323
Evanescent wave, 151
Evolution
of adhesion molecules, 928-929
biology and, 1
chromosome painting and, 342-343
convergent, 997
of diverse animal tissues, 928-929
DNA sequencing and, 342-343
lineages of, 2 f
mobile DNA elements influencing, 321-323
of organisms, 2
signaling pathway repurposing in, 911
timeline for, 3 t
Evolutionary tree, 26 f
Excision-repair systems, 204-205
base excision, 205, 205f
mismatch excision, 205-206, 206f
nucleotide excision, 206-207, 207f
Excitable cells
neurons, 1027
non-neuron, 1028
Excitatory receptors, 1059
Excitatory synapses, 1026, 1048
Exergonic reactions, 58, 59f
Exocytosis, 632
of synaptic vesicles, 1056
Exon definition, 428-429, 429f
Exon recognition, 429f
Exon shuffling, 321f, 322, 322f
Exonic splicing enhancers, 428, 429f, 436
mRNP exporter and, 441
Exonic splicing silencers, 437
Exon-junction complex, 427
Exons, 179, 181f, 303, 1016
alternative splicing and, 435
gene duplication and, 303, 304f
noncoding DNA and, 309
RNA splicing and, 423-424, 425f
SR proteins and definition of, 428-429, 429 f
Exonucleases, 432
Exoplasmic face, 275
Exoplasmic space, membrane protein N -terminus in, 597-598
Exosome, 432, 433f
Exothermic reactions, 59
Experimental systems, 28
Experimentally regulated promoters, 260-261
Exportin 5, 448
Expressed sequence tag (EST), 326
Extensin, 969
Extracellular fluids, 55-56
Extracellular matrix (ECM), 24, 922f, 923, 925-928, 926f, 928f, 930f
basal lamina, 945-950
connective tissue, 951-960
functions of, 927 t
integrins mediating adhesion in, 961-962
metastasis and, 1142 f
proteins in, 926 t
remodeling and degrading of, 960
TGF- $\beta$ proteins in, 722
Extracellular signaling, 719
amplification of, 679
cell cycle entry and, 889-890
germ cell formation and, 912
types of, 675 f
Extravasation, 966f, 1124
Extrinsic apoptosis pathway, 1021f
Ezrin-radixin-moesin (ERM), 796

## F

$\mathrm{F}_{1} \gamma$ subunit, 554-555, 555f
FABPs. See Fatty acid-binding proteins
Facilitated diffusion, 476
Facilitated transport, 476, 477t
of glucose and water, 477-483
Facilitated transporters, 476
FACS. See Fluorescence-activated cell sorter
F-actin, 779, 780-781, 780f
Facultative anaerobes, 518
FAD. See Flavin adenine dinucleotide
FADD, 1021-1022
$\mathrm{FADH}_{2}, 63,64 \mathrm{f}, 533$
electron transfer from, 539-540
oxidation of, 539
Familial adenomatous polyposis (FAP), 1146
Familial ALS, 984
Familial hypercholesterolemia (FH), 661
Familial Parkinson's disease, 985f
FAP. See Familial adenomatous polyposis
Farnesyl, 288, 289f, 295f, 296, 741f
Fas ligand, 1021-1022
Fat cells
beige, 559
differentiation of, 770, 771f
insulin stimulation of, 767, 768f, 769
Fatty acid synthase, 293
Fatty acid-binding proteins (FABPs), 293-294, 294f
Fatty acids, 48, 49t, 50, 51f
oxidation of, 534f, 536-537, 538f
peroxisomal oxidation of, 537, 538f
phospholipids incorporating, 294-295
proteins facilitating movement of, 293-294
synthesis of, 293
Fatty acyl-CoA dehydrogenase, 544
Fc receptors, 1094
F-class pumps, 484f, 485
Feedback inhibition, 100
Feedback loops
cytosolic $\mathrm{Ca}^{2+}$ and, $713,713 \mathrm{f}$
intrinsic polarity program and, 1000-1002, 1001f
in nervous system, 1028
TGF- $\beta$ /Smad regulation, 725-726, 726f
Feedback neural circuits, 1029, 1030f
Feedback repression, 706
Feed-forward activation, 517
Fermentation, 518, 519f, 520
Ferrotransferrin, 663, 664f
Fertilization, 977, 978f, 979

Fexofenadine, 687 t
FGF. See Fibroblast growth factor
FGF receptors, $735,735 f$
FG-nucleoporins, 622, 623f, 627
FG-repeats, 625
FH. See Familial hypercholesterolemia
FH1. See Formin-homology domain 1
FH2. See Formin-homology domain 2
Fibril-associated collagens, 951
Fibrillar adhesions, 961
Fibrillar collagens, 951-952, 952f
fibrillin-1 gene, 960
Fibrils, 951
Fibroblast growth factor (FGF), 734, 735, 735f, 1141
Fibroblasts, 12, 131, 182, 182f, 655, 951, 983
adhesions on, 962 f
Fibronectin, 182, 182f, 303, 928f, 930f, 956-959, 958f, 959f, 965
alternative splicing and, 304
Fibrous proteins, 72-73
Filamin, 793, 794f
Fimbrin, 793, 794f
Fischer, Emil, 91
FISH. See Fluorescence in situ hybridization
$5^{\prime}$ capping, 419, 420-421, 421f
protection of, 432
5-hydroxytryptamine, 1052f
Flagella, 12, 276, 844-849
beating, 844-845, 846f
structure of, 845 f
Flavin adenine dinucleotide (FAD), 63, 64f, 95,516
Flavin mononucleotide (FMN), 542 f
Fleming, Alexander, 137
Flippases, 282, 295, 493-494
Flow cytometry, 132-133, 881, 881f
Fluid mosaic model, 272, 272f
Fluid-phase endocytosis, 790-791
Fluorescence in situ hybridization (FISH), 258f, 342, 342f
Fluorescence light microscopy, 140f, 143
Fluorescence recovery after photobleaching (FRAP), 151, 152f, 279, 280f, 852
Fluorescence staining, 112
Fluorescence-activated cell sorter (FACS), 132, 133 f
Fluorescent proteins, 146-147, 146f
Fluorescent staining, 143
Fluorochromes, 143, 144f
Fluoxetine, 1057
FMN. See Flavin mononucleotide
FMRP. See Fragile X mental retardation protein
Focal adhesions, 796, 933, 962f
Focal complexes, 961
Focal contacts, 933
Food poisoning, 1055
Foot-and-mouth disease virus, 967
Formaldehyde, 33 f
Formin-homology domain 1 (FH1), 786
Formin-homology domain 2 (FH2), 786, 787f
Formins, 786, 787f, 1004f

Frster resonance energy transfer (FRET), 152-153, 153f, 690
40 S subunit, 192f, 464, 465f
48 S preinitiation complex, 191, 192f
43 S preinitiation complex, 191, 192f
Fossil record, 3t
4E-BPs. See eIF4E-binding proteins
FOXO3a, 751
Fragile X mental retardation protein (FMRP), 1075
Fragile X syndrome (FXS), 1075
Frame shifting, 184
Frameshift mutations, 225
Franklin, Rosalind, 170
FRAP. See Fluorescence recovery after photobleaching
Free energy, 58-60
Free radicals, 547-548
FRET. See Frster resonance energy transfer
FRET biosensor, 152, 153f
Frizzled gene, 1008, 1009f
Frontotemporal dementia, 984
Fructose, 48f, 480
Fruit fly. See Drosophila melanogaster
Functional complementation, 237
Functional domain, 76
activators composed of, 381-383
Functional RNAs, 308-309
Fura-2, 143, 144f
Furin, 657
Fusion proteins, 403f, 681
Fusion yeast, 877-878, 878f
FXS. See Fragile X syndrome

## G

G protein-coupled receptors (GPCRs), 675
adenylyl cyclase activation or inhibition by, 699-707
calcium levels and, 708-716
CREB transcription factor and, 705f
effector protein activation and, 689 f
G protein activation and, 690
general structure, 686-687, 687f
ion channel regulation by, 693-698
ligand binding to, 688 f
ligand-activated, 689-691
MAP kinase and, 746
as neurotransmitter receptors, 1048
olfactory, 1067 f
orphan, 692
pharmaceutically important human, 687 t
signal suppression, 706-707, 707f
structure and mechanism of, 686-693
as taste receptors, 1064-1066
G proteins
activation of and regulation by, 691-693
classes of mammalian, 692 t
heterotrimeric, 678, 689-691
trimeric, 454
$\mathrm{G}_{1}$ CDKs, 882, 883
$\mathrm{G}_{1}$ cyclins, 884
$\mathrm{G}_{1}$ phase, $19,875,881$
loss of regulation of, 1164-1165
$\mathrm{G}_{1} / \mathrm{S}$ phase CDKs, 882, 883, 890
$\mathrm{G}_{1} / \mathrm{S}$ phase cyclins, 884,885
$\mathrm{G}_{1} / \mathrm{S}$ phase transition, 888, 888f, 889
$\mathrm{G}_{2}$ phase, 19,875
GABA. See Gamma-amino butyric acid
G-actin, 779-780, 780f, 783f, 789, 790f
GAGs. See Glycosaminoglycans
Gain of function, 225, 1149, 1151
Gal4, 381-382, 383f
Galactose, 48f, 479
$\mathrm{G}_{\mathrm{av}}, 699-701,701 \mathrm{f}$
Gametes, 976
in fertilization, 977, 978f, 979
fusion of, 977, 979
Gamma-amino butyric acid (GABA), 1048, 1051, 1052f
$\gamma$-secretase, 762-763
$\gamma$-tubulin ring complex ( $\gamma$-TuRC), 825, 826f
Ganglion mother cell (GMC), 1010f
Gangliosides, 278
GAP. See GTPase-activating protein
Gap junctions, 932, 932f, 942, 943f, 944, 1031, 1060
Gastric cancer, 725
Gated channels, 476, 497. See also Voltage-gated channels
Gaucher's disease, 561
GBS. See Guillain-Barr syndrome
GCN2 eIF2 kinase, 455
G-CSG. See Granulocyte colony-stimulating factor
GDI. See Guanine nucleotide dissociation inhibitor
GDNF. See Glia-derived neurotrophic factor
GDP- $\beta$-tubulin, 828-829, 829f
GEF. See Guanine nucleotide exchange factor
Gel electrophoresis, 246
Gel filtration chromatography, 110f, 111
Gel-shift assay. See Electrophoretic mobility shift assay
Gelsolin, 785
Geminin, 892
GenBank, 324
Gene complementation, 304
Gene control, 355
Gene expression, 168
cloned DNA molecules in studying, 246-253
cohesins and, 895
control in bacteria, 356-362
eukaryotic control, 363-370
heterochromatin silenced by, 390-391, 393
long-term memory formation and, 1074-1075
promoter-proximal elements regulating, $378-379$
regulation of, 353
TGF- $\beta$ and, 1162, 1162f
Gene Expression Omnibus (GEO), 411
Gene family, 306
Gene knockout, 261, 262f, 263f
Gene position, 232-233, 233f
Gene tagging, 242-243, 243f
General import pore, 611

General transcription factors, 373-376
Genes, 8. See also specific genes
activation during cellular differentiation, 395-396
in cancer onset, 1150 t
chromosomal organization of, 309-312
conditional mutations in studying, 227-228
coordinately regulated, 357
co-regulated, 248-249, 250f
density in eukaryotes, 307f, 309
dominant-negative alleles inhibiting, 262-264, 265f
duplicated, 303-306, 304f
exons and duplication of, 303, 304f
experimentally regulated promoters controlling, 260-261
histone acetylation directed at, 394-395
histone deacetylation directed at, 393-394
human disease, 254-259
identifying within sequences, 326
immunoglobulin, 1116-1118
inactivating function of specific, 259-268
inactivating with somatic cell recombination, 261-262
insertion mutation tagging, 242-243, 243 f
knockout of, 261
nonprotein-coding, 308-309, 308t
organization differences in prokaryotes and eukaryotes, 179, 180f
protein-coding, 176-182
recombination in mapping, 233
replacing with homologous recombination, 260, 260f
sequencing, 243-244, 244f, 245f
solitary, 305
structure in eukaryotes, 303-309
structure of, 302 f
studying by analyzing mutations, 224-233
tandemly repeated, 307-308
TCR, 1116-1118
Genetic code, 8, 183-185, 184t, 185f, 185t
Genetic diseases, 28-29. See also specific diseases and disorders
mtDNA mutations and, 528
X-linked, 547
Genetic linkage, 232
Genetic map unit, 233
Genetic mapping studies, 232-233
Genetic markers, 233
Genetic recombination, 203, 232
Genetic screens, 227-228
Genetic suppression, 231-232, 232f
Genome editing, 266, 267f, 268, 1158
Genome maintenance genes, 1135, 1150t, 1167
Genome maintenance pathways, deregulation in cancer, 1163-1168
Genomes, 14, 25
completely sequenced, 11 t
intergenic DNA in, 312
mitochondrial, 525-526
nonfunctional DNA in, 309-310
number of protein-coding genes in, 326-327, 326f
Genome-wide association study (GWAS), 258-259
Genomic DNA
gene identification within sequences, 326
isolation of segments of, 241-242, 242f
retroposed RNAs in, 321
Genomic libraries, 237-238, 238f
Genomics, 24-25, 302, 323-327
Genotype, 224
GEO. See Gene Expression Omnibus
Gephyrin, 1051, 1051f
Germ cells, regulation of formation, 912
Germ line, 990
Germarium, 990
Germ-line stem cells (GSCs), 990
Get3, 597
GFAP. See Glial fibrillary acidic protein
GFP. See Green fluorescent protein
GH. See Growth hormone
Giant green algae, 810, 810f
Gibbs, Josiah Willard, 58
Gibbs free energy, 58
GINS complex, 892
Gleevec, 1158
Glia. See Glial cells
Glia-derived neurotrophic factor (GDNF), 1159
Glial cells, 985, 986, 1026-1033, 1029-1031
formation of, 1031-1033
gap junctions connecting, 1060
synapse formation and, 1049
types of, 1044-1046, 1044f
Glial fibrillary acidic protein (GFAP), 864, 864t, 1032-1033
Gliomedin, 1046
Globin proteins, 79, 79f, 306, 307, 444
Globular proteins, 72-73
Glomeruli, 1067
Glucagon, 687, 688f, 699-700, 766-767
Glucocorticoid receptor (GR), 402, 403f
Glucocorticoid receptor response element (GRE), 400, 401f
Gluconeogenesis, 704
Glucopyranose, 47
Glucoronyltransferase 1, 972
Glucose, 48f, 480
aerobic metabolism of, 519 f
allosteric regulation of metabolism of, 518 f
anaerobic metabolism of, 519 f
blood, 479, 766-767, 767f, 769
cellular uptake of, $478,478 \mathrm{f}$
diabetes mellitus and, 769
facilitated transport of, 477-483
fermentation of, 518,520
GLUT-mediated uptake, 478f, 479, 480
import against high concentration gradients, 503-504
insulin and, 769
oxidation of, 514f, 515-516, 533-535, 534 f
transport across epithelia, 508-509, 508f

GLUT proteins, 480, 597
GLUT1, 478-479, 479f, 480, 600, 600f
GLUT2, 480, 702
GLUT3 glucose transporter, 479, 480
GLUT4, 480
insulin and, 767, 768f, 769
GLUT4 storage vesicle (GSV), 768f, 769
GLUT5, 480
Glutamate, 44, 488f, 1048, 1052f, 1058-1059
Glutamic acid, 43
Glutamine, 44
Glycine, 44, 1052f
Glycogen, 47, 480, 687
degradation of, 699, 700f
insulin and, 769
regulation of metabolism of, 702-703, 703f
synthesis and degradation of, 699, 700f
Glycogen phosphorylase (GP), 702-703
Glycogen phosphorylase kinase (GPK), 703, 714
Glycogen synthase (GS), 703
Glycogenolysis, 699, 702
epinephrine-induced, 703
second messengers regulating, 714
Glycolipids, 278, 289-290
Glycolysis, 63, 515-520
aerobic, 1138, 1138f
rate of, 516-518
stage I, 516
Glycolytic pathway, 516, 517f, 535t
Glycophorin A, 285f
Glycophorin C, 794
Glycoproteins, 289, 601
membrane, 636 f
in plant cell walls, 968-969
Glycosaminoglycans (GAGs), 48, 723, 951, 953-956
chain modifications, 954-955
disaccharides of, 954 f
Glycosidic bonds, 41
Glycosphingolipids, 295
Glycosylases, 205
Glycosylation, 601
Glycosylphosphatidylinositol (GPI), 289, 289f, 598-599, 940
Glycosylphosphatidylinositol membrane anchor, 658
Glyoxisomes, 17
GMC. See Ganglion mother cell
Golgi, Camillo, 1070
Golgi complex, 13f, 16, 16f
anterograde transport through, 648-650
cisternal maturation and, 648-650, 650f
N -linked oligosaccharide chains in, 648 f
retrograde transport through, 647-648
in secretory pathway, 632, 633f
secretory pathway and, 645-647
sphingolipid synthesis in, 279, 295
targeting to, 585
transport vesicles in, 649 f
Golgi stain, 1070, 1071f
Goodpasture's syndrome, 950
Gout, 830
GP. See Glycogen phosphorylase

GPCR-arrestin complex, 707
GPCRs. See G protein-coupled receptors
GPI. See Glycosylphosphatidylinositol
GPI anchors, 289
GPI-anchored proteins, 599, 599f
GPK. See Glycogen phosphorylase kinase
GR. See Glucocorticoid receptor
Grafts, 1104-1105
Gram staining, 10
Granulocyte colony-stimulating factor (G-CSF), 249-250, 727, 728, 1157
Granzymes, 1094, 1123, 1123f
GRB2, 741-742, 741f
GRE. See Glucocorticoid receptor response element
Green fluorescent protein (GFP), 112, 146, 252-253, 253f, 364, 634
GroEL, 85, 86f
GroES, 85, 86f
Ground tissue, 968
Group I introns, 429, 466, 466f
Group II introns, 429-430, 430f, 466, 466f
Growth checkpoint pathway, 905
Growth factor signaling pathways, 905
Growth hormone (GH), 727, 729f
GS. See Glycogen synthase
GSCs. See Germ-line stem cells
GSV. See GLUT4 storage vesicle
GTP
hydrolysis of, by dynamin, 653 f
hydrolysis of, in chain elongation, 193-194, 193f
noncovalent binding of, 101-102
GTPase, 102, 102f, 263-264, 265f
GTPase superfamily, 195-196, 910
anchor proteins, 289
mitochondrial fusion and fission and, 529
in signal transduction pathways, 677-678, 677f, 678 f
switch proteins, 639-641
vesicle formation and, 641
GTPase-activating protein (GAP), 324-325, $454,625,625 f, 678,697,739,814$
effector proteins as, 690
loss-of-function mutation in, 1160
GTP-binding proteins, 686
actin organization and, 813-815
GTP-binding switch, 677-678, 677f, 685f, 686
GTP-hydrolyzing proteins, 588-589
GU-AG rule, 427
Guanine, 45, 46 f
Guanine nucleotide dissociation inhibitor (GDI), 641, 678, 689, 814, 903
Guanine nucleotide exchange factor (GEF), $455,625,640,739,742$
Guanine nucleotide-binding proteins, 102
Guanylate cyclase-activating proteins, 697
Guillain-Barr syndrome (GBS), 1045
GWAS. See Genome-wide association study

## H

H-2 complex, 1104, 1104f
H3K9 demethylases, 984
H3K9 HMT, 333

HA. See Hemagglutinin
Hair cells, 437-438
Half-channels, 556
Half-life, 114
Halichondria panicea, 927f
Halophiles, 10
Haploid cells, yeast, 21, 21f, 1003
Haploid organisms, 224
Haploinsufficiency, 225, 1152
Haplotype, 256
Hartwell, Lee, 228, 873, 878, 905
HATS. See Histone acetyl transferases
HB-EGF, 763
HCAs. See Heterocyclic amines
HCl . See Hydrochloric acid
HDACs. See Histone deacetylases
HDL. See High-density lipoprotein
HDR. See Homology-directed repair
Hearing, 437-438, 437f
Heart, 804-805
Heart attacks, 807
Heart disease, ADAM proteases and, 763
Heart malformations, 944
Heart muscle. See Cardiac muscle
Heat-shock genes, 402
Heat-shock transcription factor (HSTF), 402
Heavy meromyosin (HMM), 798
Hedgehog protein (Hh), 278, 751, 990, $1161-1162$
signaling pathway, 754-757, 756f, 758 f
HeLa cell line, 132, 881f
nuclear bodies in, 468 f
Helicase, 191, 192f, 199, 202f
general transcription factors and, 375
MCM, 892-893
Helicobacter pylori, 1141
Helix-loop-helix, 75
Helix-turn-helix, 75, 384
Helper T cells, 1104, 1124, 1128
Hemagglutinin (HA), 604
folding and assembly, 605f, 606
Hematopoietic stem cells (HSCs), 994-995, 995f, 996f, 997f
Heme, 540-541, 541f
Heme-regulated inhibitor (HRI), 455
Hemicellulose, 969
Hemidesmosomes, 5f, 933, 938
Hemoglobin, 79, 79f, 101, 306
Hemophilia, 319, 321
Henderson-Hasselbalch equation, 55
Henry VIII (King), 830
Heparin, 954
Hepatitis B virus, 1141
Hepatitis C virus, 942, 1141
Hepatocellular carcinoma, 1141
Hepatocytes, 655
Hepatoma, 725
Her family, 735-736, 737f
HER1, 735-736, 736f
HER2, 683, 736, 1157, 1159, 1159f
Hereditary nonpolyposis colorectal cancer, 1167
Hereditary spherocytic anemias, 795
Heredity, 8

Herpes simplex type I virus (HSV-I), 378
Heterochromatin, 14, 332, 333f, 392f
centromeric, 311
epigenetic memory and, 333-334
formation of, 333, 334f
gene expression silencing, 390-391, 393
histone H3 lysine 9 methylation in, 405, 406f, 407f
Heterochromatin protein 1 (HP1), 333
Heterocyclic amines (HCAs), 1145
Heterodimeric nuclear receptors, 401f, 402
Heterogeneous nuclear RNAs (hnRNA), $420 \mathrm{t}, 421,443 \mathrm{f}$
functions of, 422, 422 f
Heterogeneous ribonucleoprotein particles (hnRNPs), 421, 423f, 438
Heterologous desensitization, 706
Heterophilic binding, 924
Heteroplasmy, 528
Heterotrimeric G proteins, 678, 689-691
Heterotypic adhesion, 924
Heterozygotes, 228
Heterozygous, 225
Hexokinase, 516
Hexon, 423
Hexoses, 46, 47f, 560
HGH. See Human growth hormone
HGH receptor, 600, 600f
Hh. See Hedgehog protein
Hh precursor protein, 755, 755 f
HIF-1. See Hypoxia-inducible factor 1
High-density lipoprotein (HDL), 40
High-mobility group proteins (HMG proteins), 339
High-throughput LC-MS/MS, 123
Hippo pathway, 910-911, 910f
Hippocampus, 1071-1072, 1073f
Histamine, 808, 1052f
Histidine, 44, 1052f
Histidine kinase sensor, 361, 361f
Histone acetyl transferases (HATs), 332
Histone acetylase, 394
Histone code, 331, 333, 405
Histone deacetylases (HDACs), 332, 391
Histone H3 lysine 9 methylation, 405, 406f, 407 f
Histone locus bodies, 469
Histone lysine demethylases, 405
Histone methyl transferase (HMT), 333
Histone tails, 330-335, 390
hypoacetylation of, 393
Histone-modifying enzymes, 1155
Histones, 327, 329, 390, 392f
acetylation, 331-332, 394-395, 394f
deacetylation, 393-394, 394f
methylation of specific lysines, 405
modification of tails, 330-335
post-translational modification of, 405, 406 t
HIV-1. See Human immunodeficiency virus
$\mathrm{H}^{+} / \mathrm{K}^{+}$ATPase, 509
$\mathrm{H}^{+}$-linked antiport proteins, 1052-1054, 1053 f

HMG proteins. See High-mobility group proteins
HMG-CoA reductase, 296, 764
HMM. See Heavy meromyosin
HMT. See Histone methyl transferase
hnRNA. See Heterogeneous nuclear RNAs
hnRNPs. See Heterogeneous ribonucleoprotein particles
Holliday, Robin, 210
Holliday structures, 210, 211
Holoenzyme, 397
Homeodomain proteins, 385
Homeostasis, 53
Homeostatic chemokines, 1124
Homogeneously staining region (HSR), 1151, 1151f
Homologous chromosomes, 912
Homologous desensitization, 707
Homologous DNA recombination, 209-211
Homologous recombination, 260, 260f, 907, 911, 912
Homologous sequences, 325
Homologs, 80
Homology
among proteins, 80
sequence, 600
Homology-directed repair (HDR), 267f
Homophilic binding, 924
Homotypic adhesion, 923-924
Homozygous, 225
Homozygous knockout mutants, 263
Hooke, Robert, 129
Hormone analogs, 683
Hormones. See also specific hormones
balance of systems, 681
inflammatory, 770
nuclear receptor regulation by, 400, 400f, 402, 403f
receptor binding specificity and, 676
Horvitz, Robert, 1013
Host defense collagens, 951
Host defenses, 1081-1089
layers of, 1079-1080, 1080f
Host range, 212-213
HOTAIR. See Hox Antisense Intergenic RNA
HOTTIP, 410-411
Hox Antisense Intergenic RNA (HOTAIR), 410, 411
Hox genes, 407
HOXC locus, 410
HOXD locus, 410
HOXD11 gene, 354
HP1. See Heterochromatin protein 1
HPV vaccine, 1130
HPVs. See Human papillomaviruses
HRI. See Heme-regulated inhibitor
HSCs. See Hematopoietic stem cells
Hsp70, 83, 84f, 85, 402, 610-611, 611f, 613
Hsp90, 84f, 85, 402, 610, 611f
HSR. See Homogeneously staining region
HSTF. See Heat-shock transcription factor
HSV-I. See Herpes simplex type I virus
HTLV. See Human T-cell lymphotrophic virus
Human growth hormone (HGH), 595

Human immunodeficiency virus (HIV-1), 218, 272f, 967, 1083, 1128, 1130-1131, 1152
budding, 666-667, 668f
mRNA transport, 444-445, 444f
transcription of, 377, 377f
Human papillomaviruses (HPVs), 218, 1130, 1152, 1164, 1166
Human T-cell lymphotrophic virus (HTLV), 218, 1152
Humans
disease genes in, locating and identifying, 254-259
GLUT proteins in genome of, 480
GPCRs of pharmaceutical importance, 687 t
inherited diseases, common, 254 t
Mediator complex, 396f
mitosis in cells of, 881, 881f
mutation linkage mapping, 255-256
$\mathrm{Na}^{+}$/glucose symporters, 504
ribosome components, 189 t
ribosome structure, 190 f
Hunt, Tim, 873
Huntington's disease, 255f, 311
Hutchison-Gilford progeria, 866
HXMS. See Hydrogen/deuterium exchange mass spectrometry
Hyaluronan, 954, 956
Hybridization, 174, 246-247
Hybridomas, 135-136, 136f
Hydration shell, 36
Hydrocarbons, 39
Hydrochloric acid $(\mathrm{HCl}), 509-510$
Hydrogen bonds, 37-38, 38f
Hydrogen ions, 55
Hydrogen peroxide, 549
Hydrogen/deuterium exchange mass spectrometry (HXMS), 121
Hydrolysis, 41
of ATP, 61-62, 61f
of GTP, 193-194, 193f, 653f
of RNA, 172, 173f
serine protease-mediated, 94, 94f
Hydropathic index, 599
Hydropathy profile, 599-600, 600f
Hydrophilic, 32, 273
Hydrophilic signals, 674f
Hydrophobic, 32, 273
Hydrophobic core, 586-588
Hydrophobic effect, 39-40, 39f
Hydrophobic signals, 674f
Hydrophobicity, 475
Hydroxyurea, 881
Hyperacetylated chromatin, 331
Hyperchromicity, 172
Hyperglycemia, 769
Hyperpolarized state, 693
Hypertonic solution, 481
Hypertrophic cardiomyopathies, 805
Hypervariable regions, 1092-1093, 1093f
Hypoacetylated chromatin, 331
Hypoacetylation, 393
Hypotonic solution, 481, 482f
Hypoxia-inducible factor 1 (HIF-1), 1141

I
ICAMs. See Intercellular CAMs
I-cell disease, 655
ICM. See Inner cell mass
IDH. See Isocitrate dehydrogenase
I-domain, 938
IEF. See Isoelectric focusing
IFN- $\gamma$. See Interferon $\gamma$
IFs. See Intermediate filaments
IFT. See Intraflagellar transport
IgA, 1090
IgCAM. See Immunoglobulin cell-adhesion molecule
IgG, 1090-1091
IgM, 1090
Igs. See Immunoglobulins
I- $\kappa$ B kinase, 760
I- $\kappa B \alpha, 760$
IL-1. See Interleukin-1
IL-1 receptors, 760
IL-2. See Interleukin-2
IL-4. See Interleukin-4
IL-6. See Interleukin-6
IL-7. See Interleukin-7
IL-15. See Interleukin-15
Imatiniv, 1158
Immune system, 1079. See also Adaptive immune system
cancer defenses, 1131
innate, 1079, 1080f, 1084-1086, 1087f
Immunity, 1079
vaccines and, 1130-1131
Immunization, 1089
Immunoaffinity chromatography, 111
Immunoblotting, 112, 113f, 1081
Immunoelectron microscopy, 159, 159f
Immunofluorescence microscopy, 130, 144-146, 145f, 1081
Immunogenic molecules, 1128
Immunoglobulin cell-adhesion molecule (IgCAM), 924f, 965, 1046
Immunoglobulin fold, 1093-1094, 1093f
Immunoglobulins (Igs), 1081, 1088
classes of, 1089
constant regions, 1092, 1094-1095
D gene segments, 1099
genes, 1116-1118
heavy-chain locus rearrangement, 1099
hypervariable regions, 1092-1093, 1093f
isotypes, 1090-1091, 1091f
V and J gene segments, 1096, 1099
junctional imprecision, 1098-1099
membrane-bound, 1101-1102, 1102f
RSSs, 1098, 1099
secreted, 1101-1102, 1102f
structure of, 1089-1090, 1090f, 1093-1094, 1094f
variable regions, 1092
Immunoprecipitation (IP), 112, 113f, 114
chromatin, 373, 374f, 376
of kinases, 684
signal transduction proteins studied with, 684, 685f, 686
Immunoproteasome, 1111

Immunosuppressants, 1120
Import receptor, 610
Importin, 625
Impulse conduction, 1043
In situ hybridization, 247, 248f
In vitro, 82
In vitro liposome fusion assay, 644
In vivo transfection assays, 382, 382f
Indirect immunofluorescence microscopy, 145, 145 f
Indirect-acting carcinogens, 1143
Induced fit, 91
Induced pluripotent stem cells (iPS cells), 977, 980f, 983-986
ALS studied with, 984-986
$\beta$ islet cells from, 987 f
differentiated cells from, 986, 987f
medical applications of, 985 f
somatic cells generating, 983-986
Induced regulatory T cells, 1124
Induced Tregs, 1124
Inflammasomes, 1084-1085, 1126-1127, 1127 f
Inflammation, 681, 1086-1087
colchicine and, 830
in necroptosis, 1022
Inflammatory chemokines, 1124
Inflammatory cytokines, 758
Inflammatory hormones, 770
Inflammatory T cells, 1124
Influenza virus, 596, 1083
Inherited diseases
human, 254 t
mtDNA mutations and, 528
multiple-defect, 257-259
Inhibitory receptors, 1059
Inhibitory synapses, 1026, 1048
Initiation, 353
Initiation factors, eukaryotic, 191
Initiator, 372
Initiator sequences, 372
INK4s, 886, 1165
Innate immune system, 1079, 1080f, 1084-1086, 1087f
Inner cell mass (ICM), 979, 980f culturing ES cells from, 981f
ES cells from, 980-981
Inner mitochondrial membrane, 18 pathways to, 615
Inner nuclear membrane, 14
Inner-membrane proteins, 613-615
Innexins, 944
Inositol 1,4,5-trisphosphate $\left(\mathrm{IP}_{3}\right), 710,712$
Insertion sequences, 314, 314f, 315f
Inside-out patch clamping, 500 f
Insig-1(2)/SCAP/SREBP pathway, 766
Insula, 1064
Insulin, 480, 986
blood glucose level stabilization and, $766-767$
concentration of, 682
diabetes mellitus and, 769
GLUT4 and, 767, 768f, 769
secretion of, $767,767 \mathrm{f}$
signaling pathways and, 766-770, 768f, 771 f
Insulin-like growth factor 1 (IGF-1), 676
Insulin-like growth factor 2 (IGF-2), 676
Integral membrane proteins, 72-73, 284, 292
Integrin adhesion receptors, signaling pathways mediated by, 929 f
Integrins, 722, 812, 929f, 930f, 938-939, 938t, 958f, 959f
activation, 963 f
adhesion mediated by, 961-962, 962f
binding, 962-964
diseases and, 967
expression of, 964
Interacting proteins, 231-232, 232f
Intercellular CAMs (ICAMs), 965, 967
Interferon $\gamma$ (IFN- $\gamma$ ), 1086, 1086f
Interferons, 1086
Intergenic DNA, 312
Interleukin-1 (IL-1), 758, 770, 1087
Interleukin-2 (IL-2), 388, 728
Interleukin-4 (IL-4), 1123-1124
Interleukin-6 (IL-6), 1087
Interleukin-7 (IL-7), 1123-1124
Interleukin-15 (IL-15), 1123-1124
Interleukins, 727, 1087, 1123-1124
Intermediate filament-associated proteins (IFAPs), 867
Intermediate filaments (IFs), 12, 14f, 777, 777f, 821, 822f, 861-866
classes of, 864 t
dynamism of, 861-862
structure and assembly of, 861, 862f
subunit dimers, 861
tissue-specific expressions of, 862-865
Intermediate progenitor cells (IPCs), 1032f
Intermediate-repeat DNA, 312
Intermembrane space, 18
mitochondria, 523
mitochondrial proteins, 615-616
Internal antennas, 566
Internal ribosome entry site (IRES), 191
Interneurons, 1026, 1029, 1030f
Interphase, 875
Interphase chromosomes, 335, 335f
polytene, 343-345, 345f
Interspersed repeats, 310
Inter-ubiquitin isopeptide bonds, 104f
Intestinal stem cells, 991-992, 991f, 992f, 993f, 994
Intracellular fluids, 55-56
Intracellular movements, 293-297, 294f, 789-790, 790f
Intraflagellar transport (IFT), 757, 845, 847, 847 f
defects in, 848
Intrinsic polarity program, 1000-1002, 1001f
Intrinsically disordered proteins, 72-73, 74f
Intronic splicing enhancers, 437
Intronic splicing silencer, 436
Introns, 179, 181f, 303
group I, 429, 466, 466f
group II, 429-430, 430f, 466, 466f
noncoding DNA and, 309

RNA splicing and, 423-424, 424f, 427 self-splicing, 429-430, 430f, 466, 466f
Invadopodium, 1142
Inversional joining, 1098
Inverted repeat, 314
Ion channels. See also $\mathrm{K}^{+}$channel; $\mathrm{Na}^{+}$ channels
characterizing, 501, 501f
current flow through, 500-501, 500f, 501f
GPCRs regulating, 693-698
ligand-gated, 1048, 1057
light-activated, 1046-1047
selectivity of, 497-500, 499f
Ion concentrations, 143-144
ATP-powered ion pumps maintaining, 485-486, 485f
transmembrane electric potential and, 495-497, 496f
Ion-exchange chromatography, 110f, 111
Ionic detergents, 291
Ionic gradients, 485-486, 485f, 495
Ionic interactions, 36-37, 37f
Ionophores, 550
IP. See Immunoprecipitation
$\mathrm{IP}_{3}$. See Inositol 1,4,5-trisphosphate
$\mathrm{IP}_{3} / \mathrm{DAG}$ pathway, 710, 711f, 712, 713, 749
$\mathrm{IP}_{3}$-gated $\mathrm{Ca}^{2+}$ channels, 712, 712 f
IPCs. See Intermediate progenitor cells
IPP. See Isopentenyl pyrophosphate
iPS. See Induced pluripotent stem cells
IRAK, 1127
IRE-BP. See Iron-response element-binding protein
IREs. See Iron response elements
IRES. See Internal ribosome entry site
Iron
control of intracellular concentrations, $455-456,456 f$
in endocytic pathway, 663-664, 664f
Iron response elements (IREs), 455-456, 456f
Iron-response element-binding protein (IRE-BP), 455-456, 456f
Iron-sulfur clusters, 541, 541f, 542f
IS elements, 314, 314f
I-Smads, 724
Isocitrate dehydrogenase (IDH), 1139, 1139f
Isoelectric focusing (IEF), 109
Isoforms, 182, 439, 439f, 928
Isoleucine, 43
Isomerizations, 606
Isopentenyl pyrophosphate (IPP), 296
Isoproterenol, 683, 683f
Isotonic medium, 481, 482f
Isotypes, 1090-1091, 1091f
ITAM, 1100, 1118

## J

JAK kinase, 727, 730f
activation of, 728-730, 729f
STAT transcription factor activation by, 731
JAK2, 728, 730
JAK/STAT signaling pathway, 726-733, 1124
JAMs. See Junction adhesion molecules

Jasplakinolide, 792
JNKs. See Jun N-terminal kinases
Jun N-terminal kinases (JNKs), 746
Junction adhesion molecules (JAMs), 940, 940f, 965
Junctional imprecision, 1098-1099
Junk DNA, 301

## K

$\mathrm{K}^{+}, 485$
$\mathrm{K}^{+}$channels, 476, 476f, 499f, 1040, 1041f
acetylcholine receptors and, 693-694
in bacteria, 498
resting, $497,498 \mathrm{f}$
resting membrane potential and, 495
voltage-gated, 1035-1037, 1037f, 1042, 1042 f
$\mathrm{K}^{+}$pump, 476f
$\mathrm{K}^{+}$-channel proteins, 437-438, 437f
Kandel, Eric, 1070
Karyotype, 341, 342
cancer, 1137, 1138f
Kataegis, 1137
KATs. See Nuclear lysine acetyl transferases
KDEL receptor, 647-648, 647f
KDEL sorting signal, 647
Kearns-Sayre syndrome, 528
Kendrew, John, 119
Keratinocytes, 863, 983
Keratins, 863, 863f, 864-865, 865f
Ketamine, 34
Keto groups, 47
KEX2 gene, 657
KH motif, 422
Kidneys, 950
$\mathrm{Na}^{+} /$glucose symporters in, 504
Kilocalories, 58
Kinase cascade, 739
Kinase-phosphatase switch, 677f
Kinases, 103. See also specific kinases immunoprecipitation of, 684
in signaling pathways, 676-677
in synaptic plasticity, 1072-1073
Kindlin, 963
Kinesin-1, 835, 835f, 837f, 838, 838f
ATP cycle, 836-837, 837f
regulation of, 838 f
Kinesin-5, 853
Kinesin-13, 853f
Kinesins, 833-844, 836f
spindle pole separation and, 858
Kinetic energy, 57
Kinetochore microtubules, 852
Kinetochores, 347f, 851, 854f, 897
bi-oriented, 913, 917
co-oriented, 917
microtubule attachment, 854, 855-857, 856f, 898, 898f
spindle assembly checkpoint pathway and, 908-909, 908f
Kitasato, Shibasaburo, 1088, 1088f
KKXX sorting signal, 647
Kleisins, 336
Knee-jerk reflex, 1029, 1030f

Knockout mice, 261, 262f, 263f
Kobilka, Brian, 690
Koshland, Daniel, 91
Kozak, Marilyn, 191
Kozak sequence, 191
K-ras gene, 1146
Krebs cycle, 533. See also Citric acid cycle

## L

L cells, 934
L1 elements, 319
lac operon transcription, 357-358, 358f
Lactic acid fermentation, 519 f
Lactose, 48 f
Lagging strand, 199, 199f
Lamellipodium, 778
Laminin, 796, 945, 946f, 947, 947f
Lamins, 14, 863, 864t, 865, 866
Langerhans cells, 1125
Large rRNA, 190
Large T-antigen, 199, 200f
Laser-scanning confocal microscope (LSCM), 147, 148f
Late endosome, 632
Lateral surface, 24, 931
Latrunculin, 792
LBD. See Ligand-binding domain
LC. See Liquid chromatography; Myosin regulatory light chain
LC-MS/MS, 118, 123
LDL. See Low-density lipoprotein
LDL receptor, 660-661
pH -dependent binding of, 663, 663f
LDLR gene, 661

Leading strand, 199, 199f
Leber's hereditary optic neuropathy, 528
Lectins, 289
Lefkowitz, Robert, 690
Leghemoglobin, 79, 79f
Lentiviruses, 252, 252 f
Leprosy, 965
Lesion-bypass DNA polymerases, 1168
let-7, 982-983
Leucine, 43, 184
Leucine zipper proteins, 386
Leucine zippers, 75
Leukemia, 218, 995, 1136, 1144 promyelocytic, 469
Leukemia inhibitory factor (LIF), 982
Leukocyte-adhesion deficiency, 967
Leukocytes, 4f, 792f, 1082-1083, 1124 movement of, 966-967, 966f
Levi-Montalcini, Rita, 1016
LG domains, 947
Lgr5-expressing stem cells, 992, 993f, 994, 994 f
LH. See Luteinizing hormone
LHCI, 568
LHCII, 568, 572, 572f
LHCs. See Light-harvesting complexes
LHRH. See Luteinizing hormone-releasing hormone
Lidocaine, 1042
LIF. See Leukemia inhibitory factor

Ligand, 53, 54f, 89, 90f
binding to GPCRs, 688f
cell sensitivity to, 683
delta, 761, 762f
fas, 1021-1022
maximal cellular response to, 682
receptor affinity for, 681-682, 682f
Ligand-activated G protein-coupled receptors, 689-691
Ligand-binding domain (LBD), 403f
Ligand-binding site, 89
Ligand-gated ion channels, 1048, 1057
Ligand-induced dimerization, 1090
Light chains, 797, 1096
Light meromyosin (LMM), 798
Light microscopy, 139-156
bright-field, 139, 140f, 142f
fixed samples for, 143
resolution of, 139,141
Light-absorbing pigments, 560-566
Light-activated ion channels, 1046-1047
Light-harvesting complexes (LHCs), 563, 565f, 566
Light-sheet microscopy, 155-156, 155f
Lignin, 969
Lincoln, Abraham, 960
LINE retrotransposons, 322
LINE RNA, 319
Linear electron flow, 568-569, 569f
LINEs. See Long interspersed elements
Linkage, 232
mapping, 255-256, 257f
Linkage disequilibrium, 256, 257f
Linker scanning mutagenesis, 378, 378f
Linkers, 239
Lipid bilayer, 273-283
Lipid droplets, 283, 283f
Lipid rafts, 282-283
Lipid-anchored membrane proteins, 284, 288-289
Lipid-binding motifs, 290
Lipids
biomembrane mobility of, 278-279, 280f
biomembrane properties and, 279-281, 281f, 281t
in biomembranes, 276, 277, 278
cell storage of, 283, 283f
cell uptake of, 659-660
covalently attached, 288-289
diabetes mellitus and, 769
in exoplasmic and cytosolic leaflets, 281-282
Lipid-soluble substrates, ABC proteins flipping, 493-494
Lipmann, Fritz, 61
Lipoproteins, 659-660. See also High-density lipoprotein; Low-density lipoprotein
Liposomes, 273, 644
Liquid chromatography (LC), 109-111, 110f, 123f
LIS1, 841, 841f
Listeria monocytogenes, 789-790, 790f
Liver, GLUT2 in, 480

Living organisms, 2 f
systems of, 5 f
L-LDP, 1074-1075
L-LTP, 1074-1075
LMM. See Light meromyosin
lncRNAs. See Long noncoding RNAs
Localization, mRNA, 457, 459-460
LOH. See Loss of heterozygosity
Long interspersed elements (LINEs), 318, 319-320, 319f, 320f
Long noncoding RNAs (lncRNAs), 308-309, $354,409-411$
cis activation by, 410-411
post-transcription modification of adenines, 447
trans repression by, 410
X chromosome inactivation, 409-410, 410 f
Long terminal repeats (LTRs), 316, 316f, $317 \mathrm{f}, 318,1152$
Long-term depression (LTD), 1072, 1073f, 1074-1075
Long-term memory formation, 1074-1075
Long-term potentiation (LTP), 1072, 1073f, 1074-1075
Loratadine, 687 t
Losartan, 687 t
Loss of function, 225, 1152-1153, 1160
Loss of heterozygosity (LOH), 1154, 1154f
Lou Gehrig's disease. See Amyotrophic lateral sclerosis
Low-density lipoprotein (LDL), 40, 440, 440f, 659-660, 660f, 764
cellular uptake of, $660,661 \mathrm{f}$
endocytic pathway for internalization of, $662,662 \mathrm{f}$
FH and, 661
pH -dependent binding of, $663,663 \mathrm{f}$
loxP-Cre recombination system, 262, 264f
LPA. See Lysophosphatidic acid
LRP4, 1050f, 1051
LSCM. See Laser-scanning confocal microscope
LTD. See Long-term depression
LTP. See Long-term potentiation
LTR retrotransposons, 316, 316f, 318
LTRs. See Long terminal repeats
Lumen, 16
Luminal sorting signals, 641, 642t
Lung cancer, 1137
Luteinizing hormone (LH), 713
Luteinizing hormone-releasing hormone (LHRH), 713
Lymph, 1082
Lymph nodes, 1082-1083, 1083f, 1128
Lymphatic system, 1082, 1082f
Lymphocytes, 1082-1083, 1083f
cancer defenses, 1131
naive, 1091-1093
Lymphoma, 1161, 1161f, 1164
Lynch syndrome, 1167
Lys-48, 103
Lysine, 43, 104f
Lysis, 213

Lysogeny, 216
Lysophosphatidic acid (LPA), 815f
Lysophospholipids, 278
Lysosomal storage diseases, 655
Lysosomes, 13f, 16-17, 17f, 792 f
acidity regulation in, 489-491
autophagic pathway and, 667-669
cytosolic proteins directed to, 667-669
enzyme trafficking, 653-655, 654f
membrane proteins directed to, 665-667
plasma membrane protein degradation in, $665,666 f$
protein targeting to, 653-655, 653f
RTK degradation by, 738
targeting to, 585
Lysozyme, 994, 1083
Lytic viral growth cycles, 213-216, 215f

## M

M (mitotic) phase, 19, 875
$\mathrm{m}^{\mathrm{a}} \mathrm{A}, 447$
M6P. See Mannose 6-phosphate
M6P receptors. See Mannose 6-phosphate receptors
Macromolecules, 6, 7-9, 41
Macronucleus, 345
Macrophages, 776, 1086
in obesity, 770
polarization, 1003
Mad1-Mad2 complex, 908-909, 908f
Madin-Darby canine kidney cells (MDCK), 133, 133f, 281, 657, 934, 934f, 936f, 937f
Major histocompatibility complex (MHC), 1081, 1104-1115, 1104f
antigen presentation and, 1109-1115
classes of molecules, 1105-1107, 1107f
CTL activity restricted by, 1105
graft rejection/acceptance and, 1104-1105
peptide antigens bound by, 1107-1109, 1108 f
T cell development and, 1120
T cells guided by, 1105-1107
TCR variable residues and, 1118
Malaria, 22, 23f, 24, 1082, 1130
Malate-aspartate shuttle, 536, 536f
MALDI. See Matrix-assisted laser desorption/ionization time-of-flight
Malignant tumors, 1141
Mammalian embryo, cleavage of, 979
Mammalian MEK kinase (MEKK), 746
Mammalian TOR (mTOR), 452
MAMs. See Mitochondria-associated membranes
Mannose, 479
Mannose 6-phosphate (M6P), 653-655, 653f
Mannose 6-phosphate receptors (M6P receptors), 653
Mannose-binding lectin pathway, 1085, 1085f
Map distance, 232
MAP kinase family, 739, 742, 744
GPCRs and, 746
phosphorylation of, 744-745, 744f
scaffold proteins separating, 746-748, 747f
transcription factor regulation by, $745-746,745 f$
MAPs. See Microtubule-associated proteins
Marfan syndrome, 960
Mass spectrometry, 116, 116f, 117f, 118
HXMS, 121
in proteomics, 123-124, 123f, 124f
Mast cells, 1087
Master transcription factors, 25-27, 27f, $721,721 f, 725 f$
Master transcriptional regulator, 770
Maternal mRNA, 979
Mating pathway, yeast, 746, 747f
Matriglycan, 965
Matrisome, 945
Matrix, 18, 608
mitochondrial, 523
$\mathrm{NAD}^{+}$and NADH concentrations in, $535-536,536 f$
Matrix metalloprotease (MMP), 761, 960
signaling protein cleavage and, 763
Matrix-assisted laser desorption/ionization time-of-flight (MALDI), 116, 116 f
Matrix-targeting sequences, 609, 616
Maturases, 430
Maturation-promoting factor (MPF), 879
Maximal velocity $\left(V_{\max }\right), 91$
MBP. See Myelin basic protein
MCC. See Mitotic checkpoint complex
McClintock, Barbara, 313, 314, 315
MCM helicase, 892-893
MCU. See Mitochondrial calcium uniporter
MDA5, 1085
MDCK. See Madin-Darby canine kidney cells
Mdm2, 907
MDM2, 1166
MDR. See Multidrug resistance
MDR1. See Multidrug-resistance transport protein
Measles virus, 217 f
MEC-4 touch-receptor complex, 1062, 1062 f
Mechanical defenses, 1083-1084
Mechanical energy, 57
Mechanoreceptors, 1061-1062
Mechanosensors, 929, 930f, 1061-1062
Mechanotransducers, 1063-1064, 1064f
Mechanotransduction, 929
medial-Golgi, 649
medial-Golgi cisterna, 632, 635
Mediator, 390
Mediator complex, 396-397, 396f, 397f
Medulloblastoma, 757
Meiosis, 226f, 879, 911-917
cohesins in, 915-916, 915f
DNA replication inhibited between divisions, 917
errors in, 916
key features of, 912-913, 913f, 914f, 915
nondisjunction in, 909, 916
recombination during, 233
trisomies and, 916

Meiosis I, 912, 913
error rates, 916
sister kinetochore co-orientation in, 917
Meiosis II, 912
Mei-S332/Shugoshin, 916
MEK, 742, 744
MEKK. See Mammalian MEK kinase
Melanoma, 744, 1131, 1137, 1144
Melanophores, 841, 842f
Melanosomes, 841
transport of, 867
Melting, 172
Melting temperature, 172, 173f
Membrane attack complex, 1085, 1085f
Membrane depolarization, 1039-1040, 1042
Membrane extension, 811
Membrane fusion
SNARE complex dissociation, 644
SNARE protein mediating, 642-644, 643 f
Membrane glycoproteins, 636f
Membrane nucleoporins, 622, 623f
Membrane potential, 475, 1027, 1028f
Membrane proteins
hydropathy profiles of, 599-600, 600f
insertion into ER, 593-600
lysosomes receiving, 665-667
multipass, 593, 597-598
sequence and deduction of topology of, 599-600
single-pass, 593, 594-597
sorting, 657-658, 658f
tail-anchored, 597
topological classes, 593-594, 594f, 598f
type I, 594-595, 594f
type II, 594f, 595-597, 596f
type III, 594f, 595-597, 596f
type IV, 594f, 597-598
Membrane synthesis, 9
Membrane transport proteins, 68, 271, 474, 476 f
classes of, 475-477, 475f
ion binding to, 487
Membrane-bound Ig, 1101-1102, 1102f
Memory
epigenetic, 333-334
forming, 1070-1072, 1074-1075
gene expression and, 1074-1075
in immune system, 1080
long term, 1074-1075
storage of, 1072 f
Memory cells, 1123
Mental retardation, 1075
Meristems, 996-999, 998f, 999f
Meromyosin, 798
Merotelic attachment, 898, 899f
Merozoites, 23f
Meselson, Matthew, 197
Meselson-Stahl experiment, 198f
Mesenchymal cell, 936
Mesenchymal stem cell, 770
Mesenchyme, 979
Mesoderm, 981f
Mesophyll cells, 577

Messenger ribonuclear protein complex (mRNP), 627
cytoplasmic, 419
directionality in export of, 441-442, 442 f
export of, 440-441, 443f
nuclear, 419
Messenger RNA (mRNA), 8, 9f, 14, 81, 168, 169f, 240f, 420t
alternatively spliced, 417, 438
bacterial, 10
cytoplasm degradation of, 445-447, 446f
decoding by tRNAs, 183-188
eukaryotic genes and, 303
gene regulation and, 419
IRES in, 191
localization of, 457, 459-460
maternal, 979
Mediator complex and expression of, 396f
miRNAs inducing degradation of, $447-450$
nuclear export of, 627
point mutations and, 257
post-transcription modification of adenines, 447
processing in eukaryotes, 420 f
protein synthesis role, 183, 183f
quality control in, 441
reading frames in sequence of, 185 f
RNAi and, 450-451
synthesis of, 353
TOP, 453
translation efficiency and, 195, 196f
transport across nuclear envelope, $440-445$
transport in HIV, 444-445, 444f
vertebrate neurons, 438
Metabolic cooperation, 942
Metabolic coupling, 78, 96, 942
Metabolic intermediates, 516
Metabolic pathways, allosteric regulation of, 100
Metabolites, 507
Metal shadowing, 157-158, 158f
Metalloproteases, 960
Metaphase, 851, 876
Metaphase chromosomes, 327, 335, 900f
banding patterns, 341-342
chromosome painting, 341-342, 342f
number, size, and shape, 341
structure of, 339, 339f
Metaphase plate, 854
Metastases
ADAM activity and, 763
TGF- $\beta 1$ and, 722
Metastasis, 1136, 1141-1143, 1142f, 1147f
Metazoans, 4-5
cell function, 24-25
DNA replication initiation in, 892
evolution of, 24-25
genes in, 8
lncRNAs and epigenetic repression in, 409-411
RNA polymerase II regulation in, 402
RNA processing in, 432-433
structure, differentiation, and model organisms, 24-29
Methane, 33 f
Methionine, 43, 44, 184, 190-191
Methionyl-tRNA, ${ }^{\text {Met }}, 190-191$. See also Met-tRNA, ${ }^{\text {Met }}$
Methotrexate, 612, 612f
Methylation
of histone lysines, 405, 406f, 407f
$\mathrm{N}^{\mathrm{e}}, 447$
transcription repressed by, 404-405
Met-tRNA, ${ }^{\text {Met }}, 191,193,194$
Mevalonate, 296
MG123, 99
MHC. See Major histocompatibility complex
MHC restriction, 1105
Mice, 20 f
Micelles, 273
Michaelis constant, 92, 478, 681
Michaelis-Menten equation, 92, 479
MICOS complex, 522
Microciona prolifera, 927 f
Microfibrils, 951
Microfilaments, 12, 14f, 777, 777f, 778-781, 779f, 822f
attachment to membranes, 795 f
cell migration and, 867-868
endocytosis and function of, 790-791, 791f
melanosome transport and, 867
neural growth cones and, 868, 868f
Microglia, 1030, 1044
Micro-RNA (miRNA), 176, 308, 367, 420t, $445,448 f$
alternative polyadenylation and, 450
gene regulation and, 419
limb development and, 449, 449f
mRNA degradation induced by, 447-450
processing of, 448, 449f
translation repression by, 447-450
tumorigenesis and, 1155-1156
Microsatellite repeats, 438
Microsatellites, 256, 310-311, 310f
Microscopy, 130
electron, 156-161, 157f
fixed samples for, 143
light, 139-156, 140f, 142f
organelle isolation, 161-164
subcellular detail imaging, 142-143, 143f
video, 139
Microsomes, 586, 587-588, 587f
Microtubular protein, 827
Microtubule-associated proteins (MAPs), 822, 827, 830-831, 831f
Microtubule-organizing centers (MTOCs), 824-826, 825f, 868
Microtubules, 12, 14f, 347f, 777, 777f, 821, 822f, 823f
in axonal transport, 833-834, 834f
cell migration and, 867-868
ciliar and flagellar beating produced by, 844-845, 846f
configurations of, 824-826, 825f, 826f
detyrosylated, 843, 843f
disassembly, 832-833
doublet, 844-845

dynamic instability in, 827-829, 828f, 829 f
dynamics, 827-830
dynamics in mitosis, 852-853, 853f
dynein transporting along, 838-841, 839f, 840f
growth of, 827 f
kinesin-1 walking down, 836-837, 837f
kinetochores attaching to, 854, 855-857, 856f, 898, 898f
melanosome transport and, 867
in mitotic spindle, 851-852
motor proteins based on, 833-844
neural growth cones and, 868, 868f
organelle transport by, 842 f
organization of, 829
regulation of structure and dynamics, 830-833
spindle assembly checkpoint pathway and, 908-909, 908f
structure and organization, 822-826
tubulin dimers in walls of, 822-824, 824f
tubulin modifications and classes of, 842-844, 843f
Microvilli, 13f, 795
Middle prophase chromatids, 339
Migrating cells, 776 f
Miller syndrome, 523
Miller-Dieker lissencephaly, 841
Minsatellites, 311
Miranda protein, 1010
miRNA. See Micro-RNA
Misfolded proteins, 87, 88f
Mismatch excision repair, 205-206, 206f
Missense mutations, 204, 225
Mitochondria, 13f, 18, 18f, 514
apoptosis regulation and, 1017, 1018
ATP synthesis in, 552-553
in brown-fat tissue, 558-559
$\mathrm{Ca}^{2+}$ transport to, 712-713
calcium concentrations in, 709
cryoelectron microscopy of, 120 f
diseases from defective, 523
electron transport in, 539-540, 540f, 542f
endosymbiont hypothesis and, 523, 524f
energy inputs for protein import, 613
ER influencing, 529, 531f, 532
evolutionary roots of, 527
fatty acid oxidation in, 536-537, 538f
functions of, 520, 522t
fusion and fission, 529, 530f
genomes of, 525-526
inner-membrane proteins, 613-615
interaction of, 528-529
intermembrane-space proteins, 615-616
internal structure, 521 f
membranes, 520, 522-523
outer-membrane proteins, 616
oxidation rate in, 558
pro-apoptotic proteins and, 1018
protein import, 610-613, 611f, 612f
protein targeting to, 608-617
proton-motive force in, 550
ribosomes, 526-527
structure and functions of, 520-532
subcompartment protein targeting, $613-616,614 f$
Mitochondria-associated membranes (MAMs), 529, 531f, 712
Mitochondrial calcium uniporter (MCU), 712
Mitochondrial DNA (mtDNA), 18, 523-526, 524f, 525f, 526f
differences from standard nuclear code, $527-528,527 \mathrm{t}$
mutations in, 528
Mitogens, 889, 1164
Mitophagy, 529
Mitosis, 226f, 849-860, 859f
cohesins in, 915 f
completion of, 901-903
entry into, 895-900
initiation of, 896-897
key features of, 913 f
microtubule dynamics in, 852-853, 853f
mitotic CDK activation and exit from, 902-903
stages of, 850-851, 850f, 851f, 875, 875f
Mitotic asters, 853
Mitotic CDKs, 882, 883
exit from mitosis and, 902-903
nuclear envelope breakdown and, 897
precipitous activation of, 896-897
Mitotic checkpoint complex (MCC), 909
Mitotic cyclins, 884-885, 890f
Mitotic exit network, 902
Mitotic spindle, 850, 851-852, 858-859, 875
chromosome attachment to, 898, 898f
mitotic CDKs promoting formation of, 897-899
Mitral neurons, 1067
MLC kinase. See Myosin light-chain kinase
MLKL, 1022
MMP. See Matrix metalloprotease
Mobile DNA elements, 312-323
movement of, 313-314, 313f
Model organism, 12
cancer study in, 1146-1148, 1148f
eukaryotic, 20 f
metazoans, 24-29
unicellular eukaryotic, 19-24
Moderately repeated DNA, 312
Molecular chaperones, 83-85, 84f, 591
Molecular complementarity, 32f, 40, 40f, 89,676
Molecules
centrifugation separating, 106, 107f
electrophoresis separating, 107-109
evolution and, 1
of life, 5-9
small, 5, 6, 6f
Monastrol, 138
Monoclonal antibodies, 135-136, 136f, 144, 163, 1080, 1094
for phosphorylated peptides, 684
Monocytes, 1082
Monogenic diseases, 254-255
inheritance patterns, 255 f
Monomeric G protein, 454, 678, 678f

Monomers, 6-7, 41, 169
Monopolin complex, 917
Monosaccharides, 46-48
Monotelic attachment, 899, 899f
Monoubiquitinylation, 103
Montelukast, 687t
Morgan, Thomas Hunt, 232, 233
Morphogenesis, 927, 928f
Morphogens, 753
Morula, 979
Mosaic development strategy, 976
Motor neurons, 1016, 1016f, 1026, 1029
Motor proteins, 68
microtubule-based, 833-844
myosins, 796-810
Movement
ATP hydrolysis and, 800, 801f
of $\mathrm{Ca}^{2+}, 712 \mathrm{f}$
of chromosomes, to poles, 857-858, 857f
intracellular, 293-297, 294f, 789-790, 790f
of leukocytes, 966-967, 966f
of mobile DNA elements, 313-314, 313f
myosin, ATP and, 800, 801f
myosin-powered, 803-810
MPF. See Maturation-promoting factor
mRNA. See Messenger RNA
mRNA surveillance, 456-457, 458f
mRNP. See Messenger ribonuclear protein complex
mRNP exporter, 440-441, 442f, 627
mRNP remodeling, 441, 441f
MS. See Multiple sclerosis (MS)
mtDNA. See Mitochondrial DNA
MTOCs. See Microtubule-organizing centers
mTOR. See Mammalian TOR
mTOR complex 1 (mTORC1), 452-454, 453 f
mTOR complex 2 (mTORC2), 452-454
mTORC1. See mTOR complex 1
mTORC2. See mTOR complex 2
Muller, Hermann Joseph, 228
Multi-adhesive matrix proteins, 925, 945, 947, 947f
Multi-adhesive proteins, 951
Multicellular organisms, 2
niches in, 987-999
stem cells in, 987-999
Multi-color FISH, 342
Multidrug resistance (MDR), 491
Multidrug-resistance transport protein (MDR1/ABCB1), 491, 492f
Multienzyme complexes, 96 f
Multi-hit model, 1145-1146
Multimeric membrane proteins, 287, 287f, 601
assembly of, 601
Multimeric proteins, 78
Multipass membrane proteins, 593, 597-598
Multipass transmembrane proteins, 286, 286f
Multiple defect diseases, 257-259
Multiple endocrine neoplasia type 2, 1159
Multiple expression experiments, cluster analysis for co-regulated genes in, 248-249, 250f

Multiple myeloma, 99
Multiple sclerosis (MS), 1045
Multipotent somatic stem cells, 988
Multipotent stem cells, 977, 987
Multiprotein complexes, 388-389
Multiubiquitinylation, 103
Multivesicular endosomes, 665-666, 667f
Muscarinic acetylcholine receptors, 693-694, 694f
Muscle cells, 804f. See also Cardiac muscle; Skeletal muscle; Smooth muscle contraction, 803-804
heart, 804-805
insulin and, 767, 769
NT-3 produced by, 1017f
Muscle contraction, 800, 808, 1057-1058
Muscle relaxation, 486
Muscular dystrophies, 796
Muscular tissue, 921
MuSK, 1050, 1050f
Mutagens, 21, 224, 375, 1143, 1144
Mutations, 2, 28, 203
autosomal recessive, 621
in cancer cells, 1137
in cancer development, 1135-1136, $1150 f$
cdc mutants, 228, 229f, 878, 881
from chemical and radiation damage, 203-204
complementation tests of, 229-230, 230f
conditional, 227-228, 878
in DMD, 426
dominant-negative, 225, 730
double mutants, 230-231
frameshift, 225
gain-of-function, 1149, 1151
gene tagging by inserting, 242-243, 243f
in genes encoding transcription factors, $355 f$
genetic analysis of, 224-233
homozygous knockout, 263
linkage mapping, 255-256, 257f
loss-of-function, 1152-1153, 1160
missense, 204, 225
mtDNA, 528
nonsense, 196-197, 204, 225
oncogenic, 1146
peroxisome defects from, 621
petite, 524, 525f
point, 204, 204f, 225, 257, 1151
recessive, $229-230$
recessive lethal, 228
sec mutants, 636, 637f
segregation of, in breeding experiments, 225, 227, 227f, 228f
shaker, 1039, 1042, 1042f
shiverer, 1045
silent, 204, 225
Smad proteins and, 725
somatic, 1153, 1153f
somatic hypermutation, 1099-1100
SR proteins and, 428-429
suppressor, 231, 232f
synthetic lethal, 231-232, 232f, 1158
temperature-sensitive, 21, 228, 229f, 878, 881, 905
TGF- $\beta$ receptors and, 725
transcription factor encodings, 355 f
in tumor-suppressor genes, 1152-1155, $1154 f$
unregulated $\mathrm{G}_{1}-\mathrm{S}$ phase passage, $1164-1165,1165 f$
in yeast, 21-22
Mutually antagonistic complexes, 1000
Myc, 905
MYC proto-oncogene, 1145, 1148, 1149f, 1152
Mycobacteria, 1130
Mycobacterium leprae, 965
Myelin basic protein (MBP), 1044, 1045, $1045 f$
Myelin sheath, 1027, 1029-1031
impulse conduction speed and, 1043
production of, 1044-1046, 1044f, 1045f
Myelination, 1043, 1043f
Myeloma cells, 135
Myocardial infarction, 807
Myofibrils, 803-804, 864
Myoglobin, 79, 79f
Myosin I, 799-800, 799f, 807f
Myosin II, 796, 799-800, 799f, 802, 807f in contractile bundles, 807
structure of, 797-798, 797f
Myosin light-chain kinase (MLC kinase), 808
Myosin regulatory light chain (LC), 808, 808f
Myosin S1, 780-781, 781f
Myosin V, 799-800, 799f, 802-803, 803f, 810 f
Myosin VI, 800
Myosins, 796-803
classes of, 799 f
mechanical work by, 800, 801f
movements powered by, 803-810
step size, 800, 802, 802f
steps along actin filaments, 802-803
structure of, 797-798, 797f
superfamily, 798-800, 799f
Myotonic dystrophy, 319, 438
Myotonic dystrophy type 1, 311

## $\mathbf{N}$

$\mathrm{N}^{6}$ methylation, 447
$\mathrm{Na}^{+}, 485$
action potential magnitude and, 1034-1035
$\mathrm{Na}^{+}$channels, 1034-1035, 1035f
ion flux through, 501, 501f
$\mathrm{Na}^{+} /$ amino acid symporters, 504, 505f
N -acetylglucosamine, 637f
N -acetylglucosamine phosphotransferase, 655
NaCl . See Sodium chloride
$\mathrm{NAD}^{+}$. See Nicotinamide adenine dinucleotide
NADH, 63, 64f, 516, 518, 519f, 532, 533
cytosol and matrix concentrations of, 535-536, 536f
electron transfer from, 539-540, 540f
oxidation of, 539

NADH dehydrogenase-like complex-dependent pathway, 571
NADH-CoQ reductase, 542-543
$\mathrm{NADP}^{+}$. See Nicotinamide adenine dinucleotide phosphate
$\mathrm{Na}^{+} / \mathrm{H}^{+}$antiporter, 505
$\mathrm{Na}^{+} \mathrm{HCO}_{3}^{-} / \mathrm{Cl}^{-}$antiporter, 505
Naive lymphocytes, 1091-1093
$\mathrm{Na}^{+} \mathrm{K}^{+}$ATPase, 489, 489f, 490f
$\mathrm{Na}^{+} \mathrm{K}^{+}$pump, 476, 476f, 486
$\mathrm{Na}^{+}$-linked $\mathrm{Ca}^{2+}$ antiporters, 504-505
$\mathrm{Na}^{+}$-linked symporters, 502-504, 503f
NALP. See Neuronal inhibitors of apoptosis
$\mathrm{Na}^{+} /$lysine symporter channel, 476 f
Nascent cell plate, 860
National Center for Biotechnology Information (NCBI), 323, 411
National Human Genome Research Institute, 411
National Institutes of Health, 323
Natural killer cells (NK cells), 1082, 1086, 1086f, 1123
Natural selection, 1
Natural Tregs, 1124
N -cadherins, 934, 944
NCAM. See Neural cell adhesion molecule
Ndc80, 899
NE. See Neuroepithelium
Near-maximal cellular response, 682, 682f
Nebulin, 805
Necroptosis, 1011, 1022
Necrosis, 1011
NECs. See Neuroepithelial cells
Negative feedback mechanisms, 876
Negative staining, 157-158, 157f
NELF, 377
NEM. See N-ethylmaleimide
Neoblasts, 988
Nernst equation, 497, 1034
Nerve growth factor (NGF), 734, 1016, 1017f
Nervous system, 1025-1026
central, 1026, 1044
feedback circuits in, 1028
peripheral, 1026, 1044, 1045f
signaling circuits, 1028-1029
NES. See Nuclear-export signal
Nesprins, 865
N -ethylmaleimide (NEM), 644
NEU oncoprotein, 1159, 1159f
Neural cell adhesion molecule (NCAM), 965, 1072
Neural circuits, 1029, 1030f
Neural growth cones, 868, 868f
Neural stem cells (NSCs), 1031-1033, 1032f
Neural tissue, 921
Neural tube, 1031, 1032f
Neuraminidase, 596
Neurites, 1016
Neuroblasts, 1010f, 1027
Neurodegeneration, 1022
Neuroepithelial cells (NECs), 1031
Neuroepithelium (NE), 1032f
Neurofascin155, 1046
Neurofascin186, 1046

Neurofibromas, 1160
Neurofibromatosis 1, 324-325
Neurofilaments, 863, 864, 864t
Neurogenesis, 1033f
Neuroglia. See Glial cells
Neurological disorders, alternative RNA splicing abnormalities and, 438, $438 \mathrm{t}-439 \mathrm{t}$
Neuromodulatory inputs, 1026
Neuromuscular junction (NMJ), 1050, 1050f, 1058 f
Neuronal inhibitors of apoptosis (NALP), 1126
Neurons, 1025, 1026-1033
afferent, 1028
alternatively spliced mRNAs and, 438
apoptosis and, 1015-1017
axonal transport on, 833-834, 834f
dendritic spines in, 1071f
efferent, 1029
formation of, 1031-1033, 1032f
gap junctions connecting, 942, 1060
glial cells supporting, 1029-1031
GLUT3 in, 480
information flow between, 1028
information flow through, 1027
interneurons, 1026, 1029, 1030f
key features, 1027
memory formation and, 1070-1071
mitral, 1067
morphology of, 1027 f
motor, 1016, 1016f, 1026, 1029
nociceptive, 1017
olfactory receptor, 1067, 1067f, 1068-1069, 1068f
presynaptic, 1048, 1049-1050
projection, 1067
proprioceptive, 1017, 1017f
Purkinje, 4f, 1071f
receptor, 1028
sensory, 1016, 1017f, 1026, 1028, 1071
signaling circuits composed of, 1028-1029
Neurotransmitter receptors, 693
Neurotransmitters, 693, 1028, 1029f
cycling of, 1053 f
degradation and reuptake of, 1057
duration of signal, 1048
release of, 1054-1055
structures of, 1052 f
transport into synaptic vesicles, 1052-1054
Neurotrophin-3 (NT-3), 1015, 1017f
Neurotrophins, 1015-1017
Neutrophils, 1082, 1087, 1157
cancer defenses, 1131
Nexin, 844
NF1 gene, 324-325, 1160
NFAT, 388, 1120
NF-B pathway, 752, 757-758, 759f, 760
NF-B transcription factor, 757-758, 760, 770
NGF. See Nerve growth factor
NHEJ. See Nonhomologous end joining

Nicotinamide adenine dinucleotide $\left(\mathrm{NAD}^{+}\right)$, $63,64 \mathrm{f}, 95,516,518,519 \mathrm{f}, 532,533$
cytosol and matrix concentrations of, 535-536, 536 f
Nicotinamide adenine dinucleotide phosphate $\left(\mathrm{NADP}^{+}\right), 563$
Nicotinic acetylcholine receptor, 1057, 1058-1059, 1059f
Nidogen, 945
Nitella, 810, 810f
Nitric oxide (NO), 714-716, 715f
Nitrogen mustards, 1143
Nitroglycerine, 714
NK cells. See Natural killer cells
N -linked oligosaccharides, 605, 635, 637, 954
in Golgi cisternae, 648 f
rough ER and, 601-602, 602f, 603f
NLSs. See Nuclear-localization signals
NMD. See Nonsense-mediated decay
NMJ. See Neuromuscular junction
NMR. See Nuclear magnetic resonance
NMR spectroscopy, 121
NO. See Nitric oxide
Nobel, Alfred, 714
Nociceptive neurons, 1017
Nociceptors, 1062
Nocodazole, 830, 881
Node of Ranvier, 1043, 1045
Nomarsky interference microscopy. See Differential-interference-contrast microscopy
Noncoding DNA, 301-302, 309
Noncovalent binding
allosteric regulation and, 100-101
of calcium and GTP, 101-102, 101f
Noncovalent interactions, 33-40
defining, 33
hydrogen bonds, 37-38
ionic, 36-37, 37f
molecular complementarity due to, $40,40 \mathrm{f}$
of phospholipids, 48-50
relative energies, $36,36 \mathrm{f}$
strength of, 36
van der Waals interactions, 38-39, 39f
Nondisjunction, 909, 916
Nonfunctional DNA, 309-310
Nongated channels, 476, 495-501, 497
Nonhomologous end joining (NHEJ), 208-209, 208f, 267f, 907
Non-ionic detergents, 291-292, 292f
Non-LTR retrotransposons, 318-321
Nonpermissive temperature, 21, 228, 229f
Nonpolar bonds, 34
Nonprotein-coding genes, 308-309, 308t
Nonsense mutations, 196-197, 204, 225
Nonsense suppression, 196-197
Nonsense-mediated decay (NMD), 456, 457f
Non-TLR nucleic acid sensors, 1084-1085
Non-viral retrotransposons, 318-321
Noonan syndrome, 102
Norepinephrine, 687, 808, 1026, 1052f, 1057
Northern blotting, 247
Notch receptor, 761, 762f

Notch signaling pathway, 607
Notch/Delta pathway, 761-763, 762f
Notum, 754
Novrad, 34
Noxa, 1020-1021
NPCs. See Nuclear pore complexes
NPF. See Nucleation promoting factor
NPXY sorting signal, 661
NrCAM, 1046
NRG1, 763
NRG2, 763
NSCs. See Neural stem cells
NSF, 644
NSREBP. See Nuclear SREBP
NT-3. See Neurotrophin-3
N-terminal acyl anchors, 289
N -terminal stromal-import sequence, 617
N -terminal targeting sequences, 609-610, 616
Nuclear basket, 441
Nuclear bodies, 468-469, 468f
Nuclear cap-binding complex (CBC), 432
Nuclear envelope, 13f, 623f
breakdown of, 897
lamins in, 865
mitotic CDKs and, 897
mRNA transport across, 440-445
reassembly in telophase, 903, 903f
Nuclear eukaryotic DNA, major classes of, 306 t
Nuclear exoribonucleases, 432
Nuclear export, 441, 441f
of mRNA, 627
nuclear transport receptors in, 625-627, 626 f
Nuclear export factor 1 (NXF1), 441, 627
Nuclear export transporter 1 (NXT1), 441, 627
Nuclear lamina, 14, 865, 866f, 897, 897f
Nuclear lysine acetyl transferases (KATs), 332
Nuclear magnetic resonance (NMR), 121
Nuclear membrane, 14
Nuclear mRNPs, 419
Nuclear paraspeckles, 469
Nuclear pore complexes (NPCs), 14, 440, $442-443,443 \mathrm{f}, 464,622,623 \mathrm{f}$
mRNP export directionality across, $441-442,442 \mathrm{f}$
Nuclear pores, 622
Nuclear receptors, 386
domain structure, 400
heterodimeric, 401f, 402
hormones regulating, 400, 400f, 402, 403f
regulation of, 400, 400f
response elements, 400, 401f, 402
Nuclear speckles, 468f, 469
Nuclear SREBP (nSREBP), 766
Nuclear transcription factors, 1160-1161
Nuclear transport receptors
in nuclear export, 625-627, 626f
in nuclear import, 624-625, 625f
Nuclear-export signal (NES), 626-627, 626f
Nuclear-localization signals (NLSs), 622, $624-625,624 \mathrm{f}, 724,731$
Nuclear-receptor superfamily, 400, 401f

Nucleation promoting factor (NPF), 787-788, 791f, 815
Nucleic acids, 7-9, 41
bases of, 46 f
nucleotides in, 45-46, 46t
strands, 170, 170f
structure of, 169-175
Nucleocapsid, 213
Nucleolar organizers, 336, 461-462, 462f
Nucleolus, 13f
pre-rRNA and organization of, 461-462, 462 f
Nucleoplasm, 14
Nucleoporins, 622, 623f
Nucleosides, 46, 46t
Nucleosomes, structure of, 328, 329f
Nucleotides, 8, 169, 194
excision of, 206-207, 207f
in nucleic acids, 45-46, 45f, 46t
Nucleus, 12, 13f, 14
pre-tRNA modification in, 466-468
spindle position checkpoint pathway and division of, 909-911, 909f
structure of, 15 f
transport into and out of, 622-627
NudE, 841
Nurse, Paul, 22, 873
Nsslein-Volhard, Christiane, 228
NXF1. See Nuclear export factor 1
NXT1. See Nuclear export transporter 1

## 0

$\mathrm{O}_{2}$, electron transport to, 539-540, 540f
Obesity, 770
Obligate aerobes, 518
Occludin, 940, 942
Octylglucoside, 291, 291f
Odorants, 1067, 1068-1069
Okazaki, Reiji, 199
Okazaki fragments, 199, 199f, 201, 202f
Oleic acid, 51f, 283
Olfactory receptor neurons (ORNs), 1067, 1067f, 1068-1069, 1068f
Olfactory receptors, 1066-1067
types of, 1069 f
Oligodendrocytes, 986, 1029, 1032, 1044-1045, 1044f
Oligonucleotide microarrays, 248
Oligopeptide, 70
Oligosaccharides
side chains, 602-603
synthesis of precursor, 602, 602f
Oligosaccharyl transferase, 602
$O$-linked oligosaccharides, 601, 954, 955f
Ommatidia, 740, 740f
Oncogenes, 217, 1135, 1146
nuclear transcription factors and, $1160-1161$
proto-oncogenes becoming, 1149, 1151
signal-transducing proteins encoded by, 1160
viruses carrying, 1152
Oncogenesis, 1136
Oncogenic mutations, 1146

Oncogenic receptors, 1159-1160, 1159f
Oncogenic retroviruses, 217
Oncogenic transformation, 132, 1140
Oncometabolites, 1139
Oncoprotein, 1151
1,2-diaglycerol (DAG), 709-710, 712, 714
1000 Genomes Project, 323
Oocyte expression assay, 501f
Oocytes, 975, 979
Open reading frames (ORFs), 326
Operator, 357
Operons, 179, 180f
coordinately regulated genes and, 357
expression of bacterial, 361-362, 362f
genes clustered into, 357
Opsonization, 1094, 1113f
Optical traps, 802
Optogenetics, 1046-1047, 1047f
ORC. See Origin-recognition complex
orexin gene, 692-693
Orexin-A, 692
Orexin-B, 692
ORFs. See Open reading frames
Organelle membranes, 473
Organelles, 12
autophagic pathway transporting, 667-669
cell disruption to release, 162
centrifugal separation of, 162, 163f
cholesterol and phospholipid transport between, 296-297, 296f
isolation of, 161-164
proteins in, 124 f
proteomics of, 164
transport of, 833-834, 838-842, 842f
Organelle-specific antibodies, 162-164, 164f
Organisms
with completely sequenced genomes, 11 t
evolution of, 2
as experimental systems, 28
lineages of, 2 f
multicellular, 2
Organizing center, 998
Organs, 921
tissues organized into, 24, 25 f
Origin-recognition complex (ORC), 892
ORNs. See Olfactory receptor neurons
Oropharyngeal cancers, 1164
Orphan GPCRs, 692
Orthologous sequences, 325
Osmoregulatory pathway, 747 f
Osmosis, 481
Osmotic flow, 481
Osmotic pressure, 481, 481f
Osteoclasts, 510, 510f
Osteogenesis imperfecta, 953
Osteopetrosis, 510
Ouabain, 489, 505
Outer mitochondrial membrane, 18
Outer nuclear membrane, 14
Outer-membrane proteins, 616
Outer-membrane receptors, mitochondrial protein import and, 610-612, 611f
Outside-out patch clamping, 500f
Oxa1, 613

Oxaloacetate, 533
Oxidases, 17
Oxidation, 63-64
aerobic, 513, 514f, 515, 533, 534f
of $\mathrm{FADH}_{2}, 539$
of fatty acids, 534f, 536-537, 538f
of glucose, 514f, 515-516, 533-535, 534f
mitochondrial rate of, 558
of NADH, 539
peroxisomal, 537, 538f
Oxidation potential, 64
Oxidative phosphorylation, 515
Oxidative stress, 548
Oxyanion hole, 94
Oxygen, hemoglobin binding, 101f
Oxygen-evolving complex, 569-570
Oxyntic cells. See Parietal cells

## $\mathbf{P}$

P bodies, 447, 457
P element, 243
P segments, 498
P16, 1164
p38 kinases, 746
p53, 907-908, 1020-1021, 1145, 1146, $1165-1166$
P54 protein, 588, 588f
PABP. See Poly(A)-binding protein
PABPC. See Cytoplasmic poly(A)-binding protein
PABPC1. See Cytoplasmic poly(A)-binding protein
PAF. See Platelet-activating factor
PAGE. See Polyacrylamide gel electrophoresis
Pain receptors, 1062-1064
Palade, George Emil, 634
PALM. See Photo-activated localization microscopy
Palmitic acid, 51f
Palmitoyl CoA, 293, 295
Pancreas cells, 18 f
Pancreatic $\beta$ cells, 767, 767f
Pancreatic eIF2 kinase (PEK), 455
Paneth cells, 992, 994
Pannexins, 944
PAP. See Poly(A) polymerase
Pap smear, 1152
Papillae, 1064
Par proteins, 1006f
epithelial cell polarity and, 1007-1008, 1007 f
nematode embryos and, 1003-1006
stem cells and, 1008, 1010, 1010f
Paracellular pathway, 941, 941f
Paracrine signaling, 675f, 676, 763
Paralogous sequences, 325
Parasites, 22, 24
eukaryotic, 1082
Parental types, 232
Parietal cells, 509-510, 509f
Parkin, 529
Parkinson's disease, 87, 985f
Paroxetine, 1057
Passive immunization, 1089

Passive transport, 476
Patch clamps, 500-501, 500f, 501f
Pathogens, 1079
entry routes, 1081-1082
mechanical and chemical boundaries against, 1083-1084
replication sites, 1081-1082
Patterning genes, 26
Pax6 gene, 364, 365f, 366
Paxil, 1057
PC. See Phosphatidylcholine
PC3 endoproteases, 657
P-class pumps, 484, 484f, 488
PCNA. See Proliferating cell nuclear antigen
PCP. See Planar cell polarity
PCR. See Polymerase chain reaction
PD-1, 1131
PDE. See cGMP phosphodiesterase
PDE inhibitors, 716
PDGF. See Platelet-derived growth factor
PDI. See Protein disulfide isomerase
PD-L1, 1131
PDZ domains, 940
PE. See Phosphatidylethanolamine
Pectin, 969, 972
PEK. See Pancreatic eIF2 kinase
Pelizaeus-Merzbacher disease, 1045
Pemphigus vulgaris, 938
Penicillin, 137
Peptide bonds, 41, 69
planar, 81, 81f
synthesis of, 194
Peptide groups, 38 f
planar, 81f
Peptide mass fingerprint, 118
Peptides
MHCs binding, 1107-1109, 1108f
phosphorylated, 684, 685f
Peptidoglycan, 47
Peptidyl-prolyl isomerases, 606
Peptidyltransferase reaction, 194
Perforins, 1086f, 1094, 1122-1123, 1123f
Pericentriolar material, 825
Peripheral membrane proteins, 284, 292
Peripheral nervous system, 1026, 1044, 1045f
Perlecan, 945, 950
Permissive temperature, 21, 228, 229f
Peroxisomal oxidation, 537, 538f
Peroxisomal proteins
import of, 620 f
incorporation pathways, 620f, 621
targeting of, 619-621
Peroxisomal-targeting sequence 1 (PTS1), 619-621, 620f
Peroxisomes, 13f, 17
biogenesis and division, 621, 621f
fatty acid oxidation in, 537, 538f
Perutz, Max, 119
Pervasive transcription, 432-433
Petite mutations, 524, 525f
PH domain, 750
pH values, 55 f
of biological fluids, 54-55, 54f
buffers and, 55-56, 56f
cytosolic, 485, 505-506
of endosomes, 662-663
enzyme activity dependent on, $95,95 f$
in lysosomes, 489-491
in vacuoles, 489-491
Phages, 212
temperate, 216
Phagocytes, 1084
Phagocytosis, 16, 659, 776, 790-791
actin dynamics and, 792 f
in apoptosis, 1012, 1012f
Phagosomes, 792 f
Phalloidin, 792
Pharming, 561
Phase-contrast light microscopy, 140f, 141-142, 142 f
Phenotype, 224
mutant allele effects on, 225 f
of transcription factor mutations, 355 f
Phenylalanine, 43
Pheromones, 673
Philadelphia chromosome, 1156, 1156f
Phosphatases, 103, 856
exit from mitosis and, 902
in signaling pathways, 676-677
Phosphate, 44
Phosphatidylcholine (PC), 49f, 277f
Phosphatidylethanolamine (PE), 277f
Phosphatidylinositol (PI), 277f, 282, 709, 710f
Phosphatidylinositol-3 kinase (PI-3 kinase), 749-750, 749f, 750 f
Phosphatidylserine (PS), 277f, 282, 1015
Phosphoanhydride bonds, 61
Phosphodiester bonds, 41, 170, 170f
Phosphoenolpyruvate carboxylase, 577
Phosphofructokinase-1, 516-517
Phosphofructokinase-2, 517
Phosphoglycerides, 49, 49f, 50t, 276, 278
Phosphohistidine, 361
Phosphoinositide signaling pathways, 748-751
Phosphoinositides, 278, 748
Phospholipase A2, 290, 291f
Phospholipase C (PLC), 289, 692, 708, 710, 714,749
second messengers from, 709-710, 710f, 712
Phospholipases, 282, 282f
Phospholipid anchors, 598-599
Phospholipid bilayers
biomembrane properties and composition of, 279
formation of, 273-274, 273f
gel and liquid phases of, 278-279, 279f
permeability of, $474,474 \mathrm{f}$
sealed compartment formed by, 274-276
Phospholipid molecules, 5f
Phospholipids, 9, 9f, 48-50, 49t
ABC proteins flipping, 493-494
annular, 287, 287f
fatty acid incorporation into, 294-295
flippases moving, 295
regulation of levels of, 763-766
synthesis and intracellular movement, 293-297, 294f
transport mechanisms, 296-297, 296f
Phosphoprotein phosphatase (PP), 703
Phosphoproteins, 103, 114, 1054
Phosphoproteomics, 124
Phosphoric acid, 56, 56f
Phosphorylated peptides, 684, 685f
Phosphorylated peptide-specific antibodies, 684
Phosphorylation, 103f
in $\mathrm{Ca}^{2+}$ ATPase, 487 f
CDKs and, 883, 886
of lamins, 866
of MAP kinase family, 744-745, 744f
mRNP export and, 441-442, 442f
oxidative, 515
in protein regulation, 102-103
of rhodopsin, 697-698
of Smad transcription factors, 724
substrate-level, 516
Phosphotyrosine, 730, 731f
Phosphotyrosine phosphatases, 732-733, 733f
Photo-activated localization microscopy (PALM), 154f, 155
Photoelectron transport, 564-566, 564f
ROS protection in, 570
Photoinhibition, 570
Photomotility, 1046
Photophobia, 1046
Photophosphorylation, cyclic, 570-571
Photoreceptors, 740-741, 740f, 1061
Photorespiration, 576-578, 576f
Photosynthesis, 6, 62-63, 513, 514f, 560-566
ATP in, 563
$\mathrm{CO}_{2}$ metabolism during, 573-578, 575f
rates of, 564 f
stages of, 561-563, 562f
Photosystem I (PSI), 563, 568, 569f
cyclic electron flow through, 570-571, 571f
regulation of, 571-572, 572f
Photosystem II (PSII), 563, 568, 569f
oxygen-evolving complex in, 569-570
regulation of, 571-572, 572f
Photosystems, 563-564
molecular analysis of, 567-572
proton-motive force in, 568-569
Phragmoplast, 860
PI. See Phosphatidylinositol
PI-3 kinase. See Phosphatidylinositol-3 kinase
PI-3 kinase pathway, 749-750
PIC. See Preinitiation complex
Piezo channels, 1063, 1064f
Piezo1, 1063, 1064f
Piezo2, 1063, 1064f
PINK1, 529
Pioneer transcription factors, 395-396
PKA. See Protein kinase A
PKB. See Protein kinase B
PKCs. See Protein kinase Cs
PKG. See Protein kinase G
PKR. See Protein kinase RNA-activated
Placental alkaline phosphatase (PLAP), 289
Plakins, 867
Planar cell polarity (PCP), 1008, 1009f

Planaria, 27-28, 988
Plant adhesion molecules, 971-972, 971f
Plant cell walls, 968-969, 968f
Plant cells, 4 f
mitosis of, 860
stem cells, 996-999, 998f, 999f
Plant tissues, 968-972
Plant vacuoles, 17-18, 17f, 507
PLAP. See Placental alkaline phosphatase
Plaque assay, 213, 215 f
Plaques, amyloid, 87, 88f
Plasma, 1082
Plasma cells, 1102
Plasma membrane, 9f, 13f, 271, 272f, 473
in bacteria, 10
depolarization of, 1034, 1035f
domains of, 657-658
electric potential across, $497,498 \mathrm{f}$
in myocardial infarction, 807
PI 3-phosphates in, 750
protein anchoring in, 289 f
protein degradation, 665, 666f
protein sorting and, 657-658, 658f
retroviruses budding from, 666-667, 668f
store-operated $\mathrm{Ca}^{2+}$ channel, 713
targeting to, 585
Plasmalogens, 278
Plasma-membrane-attached protein signaling, 675f, 676
Plasmid cloning vectors, 236-237
Plasmid expression vectors, 251-253
gene and protein tagging, 252-253
retroviral systems, 252, 252 f
transfection, 251-252
Plasmids, 10, 236
Plasmodesmata, 13f, 24, 969-970, 970f
Plasmodium falciparum, 22, 24, 525
Plasmodium species, 23 f
Plastocyanin, 569
Platelet-activating factor (PAF), 966-967
Platelet-derived growth factor (PDGF), 685f, $686,734,813,1159$
Platelets, 964
PLC. See Phospholipase C
Pleated sheet, 71
Plectin, 867, 868f
PLP. See Proteolipid protein

Pluripotent cells, 976
Pluripotent stem cells, 988. See also Induced pluripotent stem cells
PML. See Promyelocytic leukemia
P-nucleotides, 1099
Podophyllotoxin, 830
Podosomes, 961
Point mutations, 204, 204f, 225, 1151
disease gene location and, 257
mRNA and, 257
Point-scanning confocal microscope, 147, 148f
Pol I. See RNA polymerase I
Pol II. See RNA polymerase II
Pol III. See RNA polymerase III
Pol $\beta$. See DNA polymerase $\beta$
Pol $\delta$. See DNA polymerase $\delta$
Pol $\varepsilon$. See DNA polymerase $\varepsilon$

Polar bonds, 34
Polar microtubules, 852
Polarity, 824
cell, 776, 776f, 1000-1010
cell division and, 1002-1003, 1002f
hyperpolarized state, 693
membrane depolarization, 1039-1040, 1042
plasma membrane depolarization, 1034, 1035 f
Polarized cells, 931, 977
Polarized membrane traffic, 1003, 1007
Poliovirus, 1130
Polo kinase, 897, 916
Poly(A) polymerase (PAP), 180, 431, 452
Poly(A) sites, 303
Poly(A) tail, 180
Poly(A)-binding protein (PABP), 195, 196f, 431
in mRNP remodeling, 441, 441f
Polyacrylamide gel electrophoresis (PAGE), 108
kinase study with, 684
Polyacrylamide gels, 246
Polyadenylation
alternative, 450
cytoplasmic, 451-452, 452f
of pre-mRNAs, 430-432, 431f
$3^{\prime}$ cleavage and, 419, 430-432, 431f
Polyclonal antibodies, 144
Polycomb proteins, 406-407, 408f, 409, 982
Polycystic kidney disease, autosomal dominant, 849
Polydactyly, 354
Polygenic diseases, 258
Polyglutamine, 311
Polyglutamylation, 843, 843f
Polyglycylation, 843, 843f
Poly-K63 ubiquitin-binding domain, 760
Polylinkers, 237
Polymerase chain reaction (PCR), 234, 239-243, 241f
gene sequencing and, 243-244, 244f, 245 f
isolation of specific segments of DNA, 241-242, 242f
Polymers, 6, 41, 169
Polypeptides, 70
binding to ER, 589
chaperonins and folding, 85, 86f
in ER, 590-591
MHC molecules binding to, 1108-1109
structure of, 69 f
synthesizing chains of, 194
translocon allowing passage of, 590, 590f
Polyploid organisms, 224
Polyribosomes, 195
Polysaccharides, 7, 41, 46-48
Polysomes, 195
Polyspermy, 979
Polytene chromosomes, 343-345, 345f
Polytenization, 344
Polyubiquitiny chains, 760
Polyubiquitinylation, 103, 104f
Polyunsaturated fatty acids, 49

Porins, 288, 288f, 523
Positive feedback mechanisms, 876
intrinsic polarity program and, 1000-1002, 1001f
Postmitotic cells, 876
Postsynaptic cell dendrites, 1028
Postsynaptic density (PSD), 1050
Postsynaptic target cells, 1048
Post-transcriptional gene control, 418f, 419
cytoplasmic mechanisms, 445-460
Post-translational translocation, 591-592, 592f
Potential energy, 57
Power stroke
dynein, 839 f
myosins, 800, 801f
PP. See Phosphoprotein phosphatase
PP1. See Protein phosphatase 1
PP2A. See Protein phosphatase 2A
PPAR $\gamma, 770,771 f$
PRC1 complex, 407, 408f
PRC2 complex, 407, 408f, 982
Pre-B cells, 1100
Pre-B-cell receptor, 1100-1101, 1101f
Precancerous polyps, 1153
Precursor cells, 988
Precursor mRNAs (pre-mRNAs), 180-181, 181f, 417, 420t
nuclear exoribonucleases degrading, 432
processing of eukaryotic, 419-433
proteins associating with, 421-422
regulation of processing, 435-440
snRNAs base pairing with, 424, 425f, 426
spliceosome and, 426-427, 443
SR proteins and exon definition in, $428-429,429 \mathrm{f}$
$3^{\prime}$ cleavage and polyadenylation of, 430-432, 431f
Precursor rRNA (pre-rRNA), 367, 420t
nucleolar organization and, 461-462, 462f
processing of, 462-465, 464f
transcription units, 462, 462f
Precursor tRNA (pre-tRNA), 420t
modification in nucleus, 466-468
processing of, 467, 467f
Preinitiation complex (PIC), 191, 192f, 374, 375f, 376f
pre-mRNAs. See Precursor mRNAs
pre-mRNPs, 419
Prenyl anchors, 289
Prenylation, 288
Preprophase band, 860
pre-rRNA. See Precursor rRNA
Pre-rRNA genes, 461-462
Presenilin 1 (PS1), 762
Presynaptic neurons, 1048, 1049-1050
Presynaptic terminal, 1054
pre-tRNA. See Precursor tRNA
Primary cell cultures, 131-132
Primary cell walls, 969
Primary cilium, 757, 847-848, 848f
defects in, 848-849
Primary electron acceptor, 563
Primary lymphoid organs, 1082
Primary structure, 70

Primary transcripts, 417
Primase, 199
Primer, 198, 200f
Prions, 87
Pro-apoptotic proteins, 1018
Procaspases, 1015, 1018, 1126
Processed pseudogenes, 321
Procollagen molecules, 951
Proenzymes, 656
Professional APCs, 1105-1106
Profilin, 784-785
Progenitor cells, 988
Progeria, 866
Progesterone, 879, 879f, 1157
Programmed cell death, 906, 977, 1011, 1011f, 1012-1013, 1012f
Prohormone, 104
Proinsulin, 656f
Projection neurons, 1067
Prokaryotic cells
DNA transposons in, 314, 315f
gene organization, 179, 180f
structure and function of, 10-12, 10f
Prolactin, 727
Proliferating cell nuclear antigen (PCNA), 200
Proline, 44
Proline isomerases, 86, 87f
Prometaphase, 851, 853-854, 855f
Promoter fusion, 253
Promoter-pressional elements, 378-379
Promoters, 176, 364, 731
experimentally regulated, 260-261
RNA polymerase II, 371-373
Promyelocytic leukemia (PML), 469
Promyelocytic leukemia nuclear bodies, 469
Proofreading, 188, 203, 204f, 528, 528f
Prophage, 216
Prophase, 850, 875
Proprioceptive neurons, 1017, 1017f
Proproteins, 656
Prostaglandin $\mathrm{E}_{1}, 701$
Prostate cancer, 1155
Prosthetic groups, 95, 540, 540t
Proteases, 76
ADAM and, 763
metalloproteases, 960
MMP, 761, 763, 960
serine, 92-96, 94f
Proteasome, 97-99, 98f
immunoproteasome, 1111
ubiquitin and, 99-100
Proteasome inhibitors, 99
Protein binding, 89-96
ligands in, $89,90 \mathrm{f}$
Protein cleavage-controlled signaling pathways, 761-766
Protein coats, 638-639
Protein Data Bank, 119, 121
Protein degradation, 97
plasma membrane, 665, 666f
proteasome and, 97-99
signaling pathways controlled by, $751-760$
ubiquitin and, 99-100

Protein disulfide isomerase (PDI), 603-604, 604 f
Protein expression profiling, 122
Protein family, 306
Protein folding, 81-89, 82f
abnormal, 87, 88f
amino acid sequence and, 81-82
catalysts in, 606-607
chaperone-mediated, 83-85, 84f, $604-606,605 f$
chaperones promoting, 82-86
chaperonin-mediated, 85-86, 86f
in ER, 601-608
ER proteins and, 604-606, 605f
improper, 606-607
isomerizations and, 606
oil drop model for, $72,73 \mathrm{f}$
planar peptide bonds and, 81, 81f
proline isomerases promoting, 86, 87f
Protein fusion, 253
Protein kinase A (PKA), 679
activation of, 701-702, 702f, 703-704
anchoring proteins localizing, 705-706, 706 f
cell types and activation of, 703-704, 704 t
gene transcription and, 704-705, 705f
glycogen metabolism regulation by, 702-703, 703f
signal amplification and, 704
signal suppression, 706-707
structure of, 702 f
Protein kinase B (PKB), 748, 750-751, 750f, $767,1019 f$
Protein kinase Cs (PKCs), 709, 714, 748
Protein kinase G (PKG), 714-716, 715f
Protein kinase RNA-activated (PKR), 455
Protein kinases, 874. See also specific protein kinases
in signaling pathways, 676-677
Protein modification, in ER, 601-608
Protein phosphatase 1 (PP1), 899
Protein phosphatase 2A (PP2A), 900, 916
Protein phosphatases, in signaling pathways, $676-677$
Protein regulation, 97-105
allosteric, 100-101
high-order, 105
phosphorylation and dephosphorylation in, 102-103, 103f
proteolytic cleavage in, 104-105
ubiquitinylation and deubiquitinylation in, 103-104
Protein self-splicing, 105
Protein sorting, 583
pathways for, 583-584, 584f
plasma membrane and, 657-658, 658f
trans-Golgi and, 657-658, 658f
Protein structure
categories of, 72-73
comparing, 79-80
determining, 118
function and, 68 f
hierarchical, 69-80
motifs, $75-76,76 f, 325$
primary, 69-70
quaternary, 78
secondary, 70-71, 71f, 72 f
tertiary, 72
Protein synthesis
AUG start codon in, 190-191
cell-free, 587, 587f
chain elongation, 193-195, 193f
completion of, 195
global regulation of, 452-455
GTPase superfamily role in, 195-196
microsomes and, 587-588, 587f
nonsense mutations and, 196-197
ribosomes and, 188-197
RNA roles, 183, 183f
secretory proteins, 587, 588-589, 589f
translation efficiency, 195, 196f
translation initiation, 191, 192f, 193
translation termination, 194f, 195
Protein targeting, 583
to endoplasmic reticulum, 585-592
to lysosomes, 653-655, 653f
to mitochondria and chloroplasts, 608-617
Protein transport
assaying, through secretory pathway, $634-635,635 f$
COPI vesicle-mediated, 647-648
COPII vesicle-mediated, 645-647
from trans-Golgi, 651-652
Protein-coding genes, 305-307
number of, in genome, 326-327, 326f
transcription of, 176-182
Protein-folding catalysts, 606-607
Proteins, 2, 7f, 41. See also specific proteins accessory, 805
alternative RNA splicing and, 181-182
amino acids in formation of, 7, 42-45, 43f
anchoring, 705-706
biomembrane, 284-292
bonding, 40, 40f
cell cycle, 21-22
in cell structure and tasks, 7
chimeric, 130, 612-613, 612f
cholesterol and sphingolipids clustering with, 282-283
cognate, 380
cross-linking, 793, 794f
cytoskeletal, 7
cytosolic, 99-100
delivery of, 583
depicting conformation of, 74-75, 75f
determining conformation, 119, 120f, 121
DNA interaction with, 172, 172 f
in DNA replication, 199-201
domains, $76-78$
double mutants in studying function order, 230-231
enzyme and antibody assays detecting, $111-114,113 f$
evolutionary relationships among, 325
fluorescent, 146-147, 146f
functions of, 68
GPI-anchored, 599, 599f

GTP-hydrolyzing, 588-589
homology among, 80
integral membrane, 72-74, 284, 292
interacting, 231-232, 232f
lipid-anchored membrane, 284, 288-289
liquid chromatography of, 109-111
mass spectrometry of, 116, 116f, 117f, 119
membrane interaction with, 284-285
membrane transport, 68
misfolded, 87, 88f
motor, 68
multimeric membrane, 287, 287f, 601
order of functioning of, 230-231, 231f
peripheral membrane, 284
production from cloned genes, 250-251, 250 f
purifying, detecting, and characterizing, 105-121
redundant, 231-232, 232f
regulation of activity of, $677,677 \mathrm{f}$
regulation of concentrations of, 354, 354f
regulation of transcription and replication, 339
regulatory, 68
removal from membranes, 290-292, 291f, 292f
scaffold, 67, 805
sequence comparison, $79-80$
signaling, 68
stabilizing, 805
stepwise synthesis on ribosomes, 188-197
tail-anchored, 597
transmembrane, 285-287, 285f, 286f, 289-290
Proteoglycans, 723, 925, 945, 950, 951, 953-956
aggregate, $956,957 \mathrm{f}$
diversity of, 955-956
Proteolipid protein (PLP), 1044
Proteolysis, 98f, 1111, 1113-1114
Proteolytic cleavage, 104-105, 656f
Proteolytic processing, 656-657, 657f
Proteomics, 122-124
mass spectrometry in, 123-124, 123f, 124f of organelles, 164
Protofibril, 861
Protofilaments, 823-824, 828, 861
Proton pumping, 539-540, 540f, 542-546
stoichiometry of, 549-550
Proton translocation, 556
Proton-motive force, 514, 515f, 539, 550, 613
ATP synthesis and, 551-559
ATP-ADP exchange and, 556, 558, 558f
heat generation with, 558-559
in mitochondria, 550
photosystem electron flow generating, 568-569
Proton/sucrose antiporter, 507, 507f
Proto-oncogenes, 1135, 1150t, 1161
apoptosis regulation genes as, 1163
gain-of-function mutations converting, 1149,1151
MYC, 1145
viruses activating, 1152

Protostomes, 26
Protozoans, 22, 23f, 24
splicing in, 428
Provirus, 217, 444
Prozac, 1057
PS1. See Presenilin 1
PSD. See Postsynaptic density
PSD95, 1051, 1051f
Pseudogenes, 307
processed, 321
Pseudomonas fluorescens, 215 f
Pseudosubstrate domain, 701
PSI. See Photosystem I
PSII. See Photosystem II
Psoriasis, 681
PTEN gene, 751, 1163
PTEN phosphatase, 751, 1156
PTS1. See Peroxisomal-targeting sequence 1
Pull-down assays, 685f, 686
Pulse-chase experiments, 115, 115f, 586, 661f
Puma, 1019f, 1020-1021
Pumps, 476
ATP-powered, 475f, 476, 483-494, 484f
$\mathrm{Ca}^{2+}, 486-489$
electrogenic, 490
F-class, 484f, 485
$\mathrm{K}^{+}, 476 \mathrm{f}$
$\mathrm{Na}^{+} \mathrm{K}^{+}, 476,476 \mathrm{f}, 486$
P-class, 484, 484f, 488
regulation of, 489
V-class, 484f, 485, 489-491, 491f, 510, 510 f
Purines, 45, 46f
Purkinje neurons, 4f, 1071f
Purple bacteria, 567, 568f
P-values, 324
Pyranoses, 47
Pyrimidines, 45, 46f, 170
Pyrophosphate, 46
Pyruvate, 518, 519f, 533
Pyruvate dehydrogenase, 533
Pyruvate kinase, 516

## Q

Q cycle, 544-545, 545f
Quaternary structure, 78
Query sequence, 324
Quiescent center, 999
Quinone, 567

## $\mathbf{R}$

R group, 42, 43f
R7 photoreceptors, 740-741, 740f
Rab effector proteins, 644
Rab GTPases, 641-642
Rab proteins, 289, 641, 644
Rab-GDP, 641, 643f
Rabies virus, 1083
Rac, 813-816, 816f
Radial glial cells (RGCs), 1032f
Radiant energy, 57
Radiation
DNA damage from, 203-204
double-strand breaks from, 209
p53 and exposure to, 1165
thymine-thymine dimers from, 206, 206f
Radioisotopes, 114-116, 114f
Radiolabeling, 684
Raf, 742, 744
RAG1 recombinase, 1098, 1098f, 1118
RAG2 recombinase, 1098, 1098f, 1118
"Ragged-red" muscle fibers, 528
Ramn y Cajal, Santiago, 1070, 1071f
Ran protein, 624-625, 854
Random coil, 70
Ran-GAP, 625, 627
Ran-GEF, 625, 903
Ran-GTP, 626f, 627, 903
Ran-independent nuclear export, 627
Ranitidine, 687t
RAP1 protein, 391, 393
RAR. See Retinoic acid receptor
RARE. See Retinoic acid response element
Ras, 289, 325, 739, 743f
cancer development and, 739, 1146, $1148,1155,1160$
RTKs and, 741-742, 741f
signals passing from, 742, 744
Ras/MAP kinase pathway, 739-748, 743f
Rate constant, 52
Rate-zonal centrifugation, 106, 107f
$R B$ gene, 1153
Rb (retinoblastoma) protein, 889
RBD. See Rho-binding domain
Reaction center, 563
Reactions
aerobic, 63
chemical, 51-57
chromogenic enzyme, 111
dehydration, 41
endergonic, $58,59 \mathrm{f}$
endothermic, 59
exergonic, 58, 59f
exothermic, 59
peptidyltransferase, 194
rate of, 60-61
redox, 63
transesterification, 423, 425f, 466
Reactive oxygen species (ROS), 547-549, 548f
protection in photoelectron transport from, 570
Reading frame, 184, 185f
Rec8, 916
Receptor hetero-oligomerization, 728
Receptor homodimerization, 728
Receptor neurons, 1028
Receptor proteins, 271
activation of, 682
affinity for ligands, 681-682, 682f
binding specificity of, 676
purification with affinity chromatography, 683-684
response to signaling molecules and activation of, 682
Receptor serine kinases, 722-726
Receptor tyrosine kinases (RTKs), 726, 727f, $734-738,1159$
dimerization of, 734-735

down-regulation of signaling from, 737-738
general structure and activation of, 734-735, 734f
intrinsic tyrosine kinase activation, 734-735
lysosomal degradation of, 738
Ras and, 741-742, 741f
Receptor-ligand complexes
adenylyl cyclase stimulating and inhibiting, 699-701, 700f
binding assays and, 681-682
dissociation constant of, 681
endosome pH and dissociation of, 662-663
stability of, 681
Receptor-mediated endocytosis, 659-664, 660f, 738, 790
Receptors. See also Cell-surface receptors; G protein-coupled receptors
cytokine, 726-733, 727f, 729f, 733f
nuclear, 386, 400-402, 400f, 401f, 403f
Recessive alleles, 224-225
monogenic diseases and, 255
Recessive lethal mutations, 228
Recessive mutant alleles, 225, 225f
Recessive mutations
autosomal, 621
complementation tests of, 229-230
lethal, 621
Reclinomonas americana, 527
Recognition helix, 384
Recombinant DNA, 234
techniques, 28
Recombinant types, 232
Recombination, 209, 232
DNA, 203-212, 211f
exon shuffling by, 321 f
homologous, 260, 260f, 907, 911, 912
during meiosis, 233
somatic, 1095
somatic cell, 261-262
Recombination signal sequences (RSS), 1098, 1098f, 1099
Red blood cells, formation of, 728 f
Redox reactions, 63
Reduction, 63
Reduction potential, 64, 546
Redundant proteins, 231-232, 232f
REF. See RNA export factor
Reflex arc, 1029
Reflexes, 1029, 1030f
Refractive index, 141
Refractory period, 1036, 1038
Regulated cell death, 1011-1022
Regulated intramembrane proteolysis, 607, 764
Regulated secretion, 632
Regulators of G protein signaling (RGSs), 678,690
Regulatory light chain, 797
Regulatory proteins, 68
Regulatory sequences, 378-389
Regulatory signals
extrinsic, 989
intrinsic, 988
stem cells and, 988-989
Rehydration therapy, 509
Release factors (RFs), 195
Remodeling, 510
chromatin-remodeling complexes, 395
ECM, 960
mRNP, 441, 441f
Renaturation, 173-174
Reoviruses, 967
Repetitious DNA, 301, 310
Replication
of bacteria, 1081
chromosome, 345, 346f
regulation of, 339
of viruses, 1081
Replication factor C (RFC), 200-201
Replication fork, 199
collapsed, repair of, 209-210, 210f
Replication origins, 199, 345
Replication protein A (RPA), 201
Replicative helicase, 199
Replicative senescence, 880
Repolarization, 1028
Reporter gene, 364
Repression domain, 384, 386-387
Repressor proteins, 354, 364
histone deacetylation directed by, 393-394
Repressors, 383-384, 384f
splicing, 437-439
Residues, 42
Resonance hybrids, 36
Resonance transfer, 566
Respiration, 62-63, 515
aerobic, 513, 515
anaerobic, 515
cellular, 63
photorespiration, 576-578, 576f
Respiratory chain, 539
Respiratory control, 558
Response elements, 400, 401f, 402
Response regulator, 361, 361f
Resting $\mathrm{K}^{+}$channel, 497, 498f
Resting membrane potential, 495-501, 1034
$\mathrm{K}^{+}$channels and, 495
Resting potential, 1027
Restriction elements, 1105
Restriction enzymes, 234-236, 235f, 235t
Restriction fragments, 235, 236, 236f
Restriction point, 875, 888-889, 1164, 1164f
Retinitis pigmentosa, 258
Retinoblastoma, 725, 1153, 1153f, 1164
Retinoblastoma protein. See Rb (retinoblastoma) protein
Retinoic acid receptor (RAR), 912
Retinoic acid response element (RARE), 401f, 402
Retrograde transport vesicles, 632
Retroposed RNAs, 321
Retrotransposons, 313, 313f, 322
LTR, 316, 316f, 317f, 318
non-LTR, 318-321
Retroviral DNA, 316, 316f
Retroviral expression systems, 252, 252f

Retroviral genomic RNA, 316, 316f
Retroviruses, 216-218, 217f, 445
budding, 666-667, 668f
oncogenic, 217
slow-acting, 1152
Retrovirus-like elements, 316
Rev protein, 444-445, 444f
Reverse transcriptase, 217, 313, 316, 317f
Reverse transcriptase-PCR (RT-PCR), 242
Rev-response element (RRE), 444, 444f
RFC. See Replication factor C
RFs. See Release factors
RGCs. See Radial glial cells
RGD motif, 938
RGG box, 422
RGSs. See Regulators of G protein signaling
Rhabdomyosarcoma, 757
Rheb GTPase, 453f
Rheb-GAP, 453f, 454
Rheumatoid arthritis, 681
Rhinoviruses (RV), 936-937
Rho, 813-816, 816f
Rho kinase, 808
Rho-binding domain (RBD), 787f
Rhodopsin, 694f, 695f
activation of, 695-696, 696f
arrestin binding and, 697-698
phosphorylation of, 697-698
signal amplification in transduction pathway, 696-697
Rhodopsin kinase, 697, 697f
Rhodopsin signal transduction pathway amplification in, 696-697
termination of, 697-698, 697f
Rho-GEF, 1006, 1006f
Ribonucleic acid (RNA), 8-9, 45, 168, 420t
alternative splicing, 181-182, 182f
conformations and functions of, 174-175
$5^{\prime}$ capping, 420-421, 421f
hydrolysis of, 172, 173 f
noncoding, 14
nuclear exoribonucleases degrading, 432
retroposed, 321
roles in protein synthesis, 183, 183f
secondary and tertiary structures, $175,175 f$
self-splicing, 175
synthesis of, 177 f
as transposition intermediate, 318, 318f
Ribonucleoprotein complexes (RNP complexes), 419
Ribonucleoside triphosphate (rNTP), 177f
Ribonucleoside triphosphate monomers (rNTPs), 169f
Ribosomal RNA (rRNA), 14, 167f, 168, 169f, 183
processing of, 461-469
protein synthesis role, 183, 183f
tandemly repeated genes for, 307-308
Ribosomes, 8
antibiotics and, 190
bacterial, 189 f
common components, 189 t
common core structure, 190 f
mitochondrial, 526-527

protein synthesis on, 188-197
recycling, 195
subunit assembly, 464-465, 465f
Riboswitches, 362, 363f
Ribozymes, 8, 90, 175
Ribulose 1,5-bisphosphate carboxylase, 573
Rickettsiaceae group, 527
RIG-I, 1085
Rigid cell walls, 968
RIM, 1056
RIM-BP, 1056
RIP1, 1022
RIP3, 1022
RISC. See RNA-induced silencing complex
Risperidone, 687 t
RNA. See Ribonucleic acid
RNA editing, 439-440, 440f
RNA export factor (REF), 427, 441
RNA helicase, 191, 192f
RNA interference (RNAi), 95, 264-266, 265f, 419, 450-451, 1158
RNA knockdown, 419
RNA polymerase B (RPB), 368f, 369, 369f
RNA polymerase I (Pol I), 367, 367f, 461
transcription initiation by, 412-413, 412f
RNA polymerase II (Pol II), 367-370, 367f, 391
chain elongation by, 428, 428f
distant enhancers and transcription of, 379
$5^{\prime}$ capping and, 420-421
general transcription factors, 373-376
Mediator complex and, 396-397
metazoan regulation of, 402
polyadenylation signal and, 431-432
preinitiation complex, 374, 375f
promoters, 371-373
transcription initiation, 371
RNA polymerase III (Pol III), 367, 367f, 391, 461
transcription initiation by, 413-414, 413f
RNA polymerases, 8, 9f
bacterial classes of, 368, 368f
eukaryote classes of, 367-370, 367f, $367 \mathrm{t}, 368 \mathrm{f}$
mRNA synthesis initiation, 353
precursor mRNAs and, 180
structure of, 178-179, 179f
in transcription, 176-179, 177f
transcription initiation by, 357
RNA processing, 169f, 180, 181f, 418f, 419
metazoans and, 432-433
RNA recognition motif (RRM), 422, 423f
RNA splicing, 181, 419, 423-424, 424f, 426f
alternative, 181-182, 182f, 417, 438, $438 \mathrm{t}-439 \mathrm{t}$
base pairing during, 424, 425f, 426
pre-mRNA processing regulation and, $435-436$
snRNAs in, 424, 425f, 426
spliceosome carrying out, 426-428, 427f
RNA viruses, 212, 967
RNA world hypothesis, 8
RNA-binding domains, 421-422
RNA-binding motifs, 422

RNA-binding proteins, 455-456
RNAi. See RNA interference
RNA-induced silencing complex (RISC), 448-449, 451
RNA-processing factors, 428
RNase P, 467
RNP complexes. See Ribonucleoprotein complexes
rNTP. See Ribonucleoside triphosphate
Rod cells, 694-695, 694f, 1061
visual adaptation and, 698
Root apical meristem, 997, 998f, 999
ROS. See Reactive oxygen species
Rough endoplasmic reticulum, 13f, 16, 16f
N -linked oligosaccharide synthesis and, 601-602, 602f, 603f
in secretory pathway, 632
secretory proteins and, 587 f
structure of, 586, 586f
Roundworm. See Caenorhabditis elegans
Rous sarcoma virus (RSV), 1152
RPA. See Replication protein A
RPB. See RNA polymerase B
RPB1, 369, 369f
RPB2, 369
RRE. See Rev-response element
RRM. See RNA recognition motif
rRNA. See Ribosomal RNA
R-Smads, 724
RSS. See Recombination signal sequences
RSV. See Rous sarcoma virus
RTK/RAS/MAP kinase pathway, 1160
RTKs. See Receptor tyrosine kinases
RT-PCR. See Reverse transcriptase-PCR
$\mathrm{rtTA}, 1148$
Rubisco, 573, 574f
Rubisco activase, 574, 576
RV. See Rhinoviruses
RV-C. See Class C rhinoviruses
RXR-VDR, 401f

## S

S. cerevisiae. See Saccharomyces cerevisiae
S. mediterranea. See Schmidtea mediterranea
S (synthesis) phase, 19, 875
CDK and entry into, 888
loss of regulation of, 1164-1165
onset of, 890-891, 891f
S phase CDK inhibitor, 890-891, 891f
S phase CDKs, 882, 883, 897
S phase cyclins, 884, 890f
S. pombe. See Schizosaccharomyces pombe

S4 helices, 1039-1040, 1042
SA sequence. See Signal-anchor sequence
Saccharomyces cerevisiae, 19, 20f, 21, 21f, $22,124,224,877,877 f$
cell cycle commitment, 888
centromeres of, 346-347, 346f
DNA replication initiation in, 890
mating pathways, 746, 747f
mRNA localization in, 457, 459, 459f, 460f
myosin V cargo movement in, 809f
ribosome components, 189 t
ribosome structure, 190 f

RNA polymerases, 368, 368f
segregation in breeding, 227, 228f
temperature-sensitive mutation in, 228, 905
$S$-adenosylmethionine, 421
SAGA complex, 394
SALL1 gene, 366, 366f
Salmeterol, 687 t
Saltatory conduction, 1043
Salty taste, 1066
SAM complex, 616
Sar1 protein, 639, 640, 640f, 646f
Sarcomas, 1136, 1166
Sarcomeres, 804, 804f
Sarcoplasmic reticulum (SR), 486, 487f, 805, 806f
$\mathrm{Ca}^{2+}$ regulation by, 805, 806f
Satellite DNA, 310-311
Saturated fatty acids, 48, 50, 293
SC. See Synaptonemal complex
SCA. See Stigma/stylar cysteine-rich adhesin
Scaffold proteins, 67, 746-748, 747f, 805
Scanning electron microscope (SEM), 157, 157f, 161, 161f
SCAP. See SREBP cleavage-activating protein
Scatter factor, 816
Scavenger receptors, 1084
SCF. See Stem cell factor
Schizosaccharomyces pombe, 877, 878f
centromeres of, 346-347
growth cycle of, 905
mitotic CDKs in, 896
Schleiden, Matthias, 129
Schmidtea mediterranea, 20f, 27-28, 753, 754 f
Schwann, Theodore, 129
Schwann cells, 1030, 1044, 1044f, 1045-1046, 1045f
SCID. See Severe combined immunodeficiency
SCNT. See Somatic-cell nuclear transfer
Scribble complex, 1007, 1010
Scurvy, 952
$\mathrm{SC}-\beta$ cells, 986
SDS. See Sodium dodecylsulfate
SDS-PAGE. See SDS-polyacrylamide gel electrophoresis
SDS-polyacrylamide gel electrophoresis (SDS-PAGE), 107-108, 108f, 115
sec mutants. See Secretion mutants
Sec61 complex, 590, 591f
Sec61 $\alpha, 590,590 f$
Sec63 complex, 591
Seckel syndrome, 906
Second messengers, 678-679, 942
glycogenolysis regulation with, 714
from PLC, 709-710, 710f, 712
Secondary active transport, 476
Secondary cell walls, 969
Secondary lymphoid organs, 1082
Secondary structure
protein, $70-71,71 \mathrm{f}, 72 \mathrm{f}$
RNA, 175, 175f
Secreted Ig, 1101-1102, 1102f
Secretion mutants (sec mutants), 636, 637f

Secretogranin II, 655
Secretory pathway, 584-585, 631, 633f
assaying protein transport through, $634-635,635 f$
early stages of, 645-650
ER and, 645-648
Golgi complex in, 632, 633f
later stages of, 650-658
rough endoplasmic reticulum in, 632
study techniques for, 634-638
Secretory proteins, 585
ER targeting by, 586-588, 587f
synthesis of, 587, 588-589, 589f
Secretory vesicles, 13 f
Securin, 901-902, 901f, 913
Sediment, 106
Segmental duplication, 307
Segregation of mutations, 225, 227, 227f, 228f
Selectins, 966
Selection medium, 135
Selective serotonin reuptake inhibitors (SSRIs), 1057
Selectivity filter, 497-500, 499f
Self-splicing
introns, 429-430, 430f, 466, 466f
protein, 105
RNA, 175
SEM. See Scanning electron microscope
Senses, 1061
Sensory neurons, 1016, 1017f, 1026, 1028, 1071
Separase, 901f
APC/C activating, 901-902
cohesin cleavage and, 901, 901f
Septation initiation network, 910
Sequence drift, 306
Sequence homology, 600
Sequence motif, 76
Sequence-reading helix, 384
Sequence-specific DNA affinity chromatography, 381
Serine, 44, 184
Serine proteases, 92-96, 94f
Serotonin, 1026, 1052f, 1057
Sertraline, 1057
Serum, 1088, 1088f
Serum response element (SRE), 745
Serum response factor (SRF), 745
Serum starvation, 881
SET domain, 407, 408f
Severe combined immunodeficiency (SCID), 728
Sex-lethal gene, 435-436
SH2 domain, 730, 731, 731f
SH3 domains, 742, 742f
shaker mutation, 1039, 1042, 1042f
Sheet-forming and anchoring collagens, 951
Shh. See Sonic Hedgehog
Shigella species, 790
shiverer mutation, 1045
Shmoos, 1003, 1004f
Shoot apical meristem, 997, 998-999, 998f, 999 f

Short interfering RNAs (siRNAs), 419, 420t, 445
Short interspersed elements (SINEs), $318-319,320-321$
Short tandem repeats (STRs), 256
shRNA. See Small hairpin RNA
Sialyl Lewis-x antigen, 966
Sic1, 890-891, 891f
Sickle-cell disease, 225, 995
Side chains
amino acid, 42-45, 43f, 45 f
oligosaccharide, 602-603
Side-binding proteins, 830-831
Sigma factors. See $\sigma$ (sigma) factors
$\sigma^{34}$-RNA polymerase, alternative, 359-360
$\sigma$ (sigma) factors, 357, 359t
alternative, 359
Signal amplification
in cAMP-PKA pathway, 704
in rhodopsin signal transduction pathway, 696-697
Signal peptidase, 591
Signal peptides, 585
Signal recognition particle (SRP), 195, 367, 588-589, 588f, 589f
Signal sequences, 585
Signal transduction, 674, 675-680
Signal transduction pathways, 674, 680f
adipocyte differentiation and, 770, 771f
amplification by, 679-680
types of, 719-720, 720f
Signal transduction proteins, 685 f
studying, 681-686
Signal-anchor sequence (SA sequence), 594-597, 598f
Signaling cascade, 904
Signaling molecules, 674, 675-676
near-maximal cellular response to, 682, 682 f
Signaling pathways
in apoptosis regulation, 1019 f
cancer and aberrations in, 1161-1163
growth factor, 905
Hh, 754-757, 756f, 758f
insulin and, 766-770, 768f, 771f
integration of responses to multiple, $766-770,768 \mathrm{f}, 771 \mathrm{f}$
integrin adhesion receptors and, 929 f
JAK/STAT, 726-733
kinases in, 676-677
mitochondrial membrane permeability and, 1019 f
Notch, 607
ordering of, 231, 231f
phosphatases in, 676-677
phosphoinositide, 748-751
protein cleavage-controlled, 761-766
repurposing in evolution, 911
ubiquitinylation controlling, 751-760
Signaling proteins, 68, 674, 674f
Signal-transducing proteins, 1160
SILAC. See Stable isotope labeling with amino acids in cell culture
Silencer sequences, 391

Silent mutations, 204, 225
Silver staining, 112
SIM. See Structured illumination microscopy
Simple diffusion, 474-475, 477t
Simple-sequence DNA, 310-311
SINEs. See Short interspersed elements
Single-nucleotide polymorphisms (SNPs), 256, 256 f
Single-pass membrane proteins, 593, 594-597
Single-pass transmembrane proteins, 285-286, 285f
Single-stranded DNA, 906
SIR proteins, 391, 392f, 393
siRNA knockdown, 451
siRNAs. See Short interfering RNAs; Small inhibitory RNAs; Small interfering RNAs
sis oncogene, 1159
Sister chromatids, 15f, 851, 875, 893, 894f, 895
co-orientation of, 913
resolution of, 899
Sister kinetochores, co-orientation of, 917
60 S subunit, 192 f
Skeletal muscle, 804f
accessory proteins, 805
contraction of, 803-804, 805f
regulation of contraction of, 805-807, 806f
Ski, 725-726, 726f
Skin, structural integrity of, 864-865
Skin cancer, 744
SKL sequence, 619-621
SLE. See Systemic lupus erythematosus
Sleeping sickness, 1082
Sliding clamp, 200
Sliding-filament assay, 798, 798f
Sliding-filament contraction model, 803-805, 805 f
Slow-acting retroviruses, 1152
SM. See Sphingomyelins
SM proteins, synaptic vesicle fusion and, 1055
SMAC/DIABLO family, 1018
Smad3/Smad4 complex, 724-725, 725f
Smads, 722-726
loss of TGF- $\beta$ and, 1162
negative feedback loops regulating, $725-726,726 f$
transcription factors, 724
Small hairpin RNA (shRNA), 266, 1158
Small inhibitory RNAs (siRNAs), 264-265
Small interfering RNAs (siRNAs), 367, 368, 448 f
Small nuclear ribonucleoprotein particles (snRNPs), 424
in spliceosome, 426-427
splicing, 426
Small nuclear RNAs (snRNAs), 308, 321, 367, 420t
pre-mRNA splicing with, 424, 425f, 426
Small nucleolar RNAs (snoRNAs), 308, 420t pre-rRNA processing and, 462-465, 464f
Small rRNA, 190

Small subunit processome, 464
Small-molecule proteasome inhibitors, 99
Smallpox, 1079, 1130
SMC. See Structural maintenance of chromosome proteins
smewdi-1 gene, 988
Smooth endoplasmic reticulum, 13f, 16
Smooth muscle
contraction regulation in, 808
vascular, 714-716, 715f
SNAP-25, 643, 643f, 644, 1055
SNARE proteins, 643f. See also t-SNAREs; v-SNAREs
dissociation of, 644
synaptic vesicle fusion and, 1055
vesicle fusion and, 642-644, 643f
SnoN, 725-726
snoRNAs. See Small nucleolar RNAs
SNPs. See Single-nucleotide polymorphisms
snRNAs. See Small nuclear RNAs
snRNPs. See Small nuclear ribonucleoprotein particles
SOCS proteins, 733, 770, 1124
SOD. See Superoxide dismutase
SOD1, 984, 986
Sodium chloride $(\mathrm{NaCl}), 36,37 \mathrm{f}$
Sodium deoxycholate, 291, 291f
Sodium dodecylsulfate (SDS), 108, 291, 291f
Solitary genes, 303
Somatic cell recombination, 261-262
Somatic cells, 977
iPS cells from, 983-986
Somatic gene rearrangement, 1095, 1096f
Somatic hypermutation, 1099-1100
Somatic mutations, 1153, 1153f
Somatic recombination, 1095
Somatic-cell nuclear transfer (SCNT), 983, 984f
Sonic Hedgehog (Shh), 755, 1162
Sorting signals, 641
cytoplasmic, 642 t
diacidic, 646, 646f
endocytosis targeting, 660-661
KDEL, 647
KKXX, 647
luminal, 641, 642t
NPXY, 661
transport vesicles and, 642 t
YXX, 651
Sos protein, 741-742, 741f, 743f
Sound, perception of, 437-438, 437f
Sour taste, 1066
Southern, Edwin Mellor, 246
Southern blotting, 246-247
Specific activity, 114
Specificity, 89, 681-682, 1080, 1088
Spectrin, 793, 794f, 795
Sperm, 975, 977, 978f
Sphingolipids, 277f
clustering with proteins, 282-283
synthesis and intracellular movement, 293-297
Sphingomyelins (SM), 277f, 278, 279, 295
Sphingosine, 295

Spinal muscular atrophy, 428
Spindle assembly checkpoint pathway, 856-857, 908-909, 908f
Spindle midzone, 903
Spindle pole bodies, 897
Spindle poles, 825, 857f, 876
Spindle position checkpoint, 860
Spindle position checkpoint pathway, 909-911, 909f
Spinning disk confocal microscope, 147, 148-149, 149f
Splice sites, 303, 423
globin-gene, 444
Spliceosomes, 426-428, 426f, 427f, 443, 466f
Splicing activation, 436 f
Splicing activators, 437-439
Splicing factors, 427, 468f
Splicing repressors, 437-439
Sporogenous tissue, 968
Sporozoites, 23f
Spot desmosomes, 933
SR. See Sarcoplasmic reticulum
SR proteins, 428-429, 429f
mRNP export directionality and, 441-442, 442f
mRNP exporter and, 441
Src gene, 730
SRE. See Serum response element
SRE-binding proteins (SREBPs), 763-766, 765f
SREBP cleavage-activating protein (SCAP), 765-766, 765f
SREBs. See SRE-binding proteins
SREs. See Sterol regulatory elements
SRF. See Serum response factor
SRP. See Signal recognition particle
SSRIs. See Selective serotonin reuptake inhibitors
STA sequence. See Stop-transfer anchor sequence
Stabilizing proteins, 805
Stable isotope labeling with amino acids in cell culture (SILAC), 118
Stable transfection, 252
Stahl, Franklin William, 197
Starch, 47, 560
structure of, 560 f
START, 888-889
Start (initiation) codons, 184, 190-191
STAT pathway, 1124
STAT proteins, activation of, 731, 732f
STAT transcription factors, 731
Statins, 296
Steady state, 52-53, 53f
STED microscopy. See Stimulated emission depletion microscopy
Stem cell, mesenchymal, 770
Stem cell factor (SCF), 886, 996, 997f
Stem cells, 27, 976. See also Embryonic stem cell; Induced pluripotent stem cells
cancer, 1140-1141
differentiation of, 988, 989f
germ-line, 990
hematopoietic, 994-995, 995f, 996f, 997f
intestinal, 991-992, 991f, 992f, 993f, 994
Lgr5-expressing, 992
meristems and, 996-999, 998f, 999f
multicellular organism niches and, 987-999
multipotent, 977, 987
multipotent somatic, 988
neural, 1031-1033, 1032f
Par proteins and, 1008, 1010, 1010f pluripotent, 988
Stem-cell niche, 989
Stereoisomers, 34, 34f
Steroid receptor superfamily, 386
Sterol regulatory elements (SREs), 764, 766
Sterols, 277f, 278
Sterol-sensing domain, 296, 765
Stigmasterol, 277f, 278
Stigma/stylar cysteine-rich adhesin (SCA), 971
STIM proteins, 713
Stimulated emission depletion microscopy (STED microscopy), 153, 154f
Stimulatory G protein, 692
STING protein, 1085
Stomach, acidification of contents, 509-510, 509 f
Stop (termination) codons, 184, 195
Stop-transfer anchor sequence (STA sequence), 594-597, 598f
Stored sequences, 324-325, 324f
Store-operated channels, 713
Strahismus gene, 1008, 1009f
Strand invasion, 209
Streptomyces coelicolor, 359
Stress fibers, 778
Stromal cells, 963, 996, 997f, 1123, 1131
Stromal proteins, 617
Strong promoters, 358
STRs. See Short tandem repeats
Structural domain, 77
Structural maintenance of chromosome proteins (SMC), 336, 336f
Structural motifs, 75-76, 76f, 325
Structural nucleoporins, 622, 623f
Structural proteins, 67
Structured illumination microscopy (SIM), 153, 154f
Sturtevant, Alfred, 232, 233
Subcellular details, imaging, 142-143, 143f
Submitochondrial compartments, protein targeting to, 613-616, 614f
Substrate binding, 93-94, 93f
Substrate-binding site, 91
Substrate-level phosphorylation, 516
Substrates, 90
Subventricular zone (SVZ), 1032, 1032f
Succinate dehydrogenase, 534
Succinate-CoQ reductase, 542, 544
Sucrose, 48f, 560
synthesis of, 573-574, 575f
Sugars, 7, 560
Sulfhydryl group, 44, 603
Supercomplexes, 546-547, 548f
Superoxide dismutase (SOD), 548f, 549
Super-resolution microscopy, 130, 153, 154f, 155

Suppressor mutations, 231, 232f
SURF complex, 458 f
Surveillance mechanisms
in cell cycle regulation, 904-911
checkpoint pathways, 856-857, 874, 876, 904, 905, 908-911, 908f, 909f, 1149
mRNA, 456-457, 458f
SV40 DNA virus, 199, 200f, 201f, 431
NLS discovery and, 624
SV40 large T-antigen, 624
SVZ. See Subventricular zone
Sweet taste, 1066
SWI/SNF chromatin-remodeling complex, 395
Switch proteins, 101
GTPase, 639-641
Symmetric cell division, 975, 976f
Symplekin, 430
Symporters, 476, 502-507
two- $\mathrm{Na}^{+} /$one-glucose, 503-504, 504f
Synapses, 25, 1016, 1029f
astrocytes and, 1049 f
chemical, 1060
communication at, 1048-1060
electrical, 1060
elimination of, 1051-1052
excitatory, 1026, 1048
formation of, 1048-1052
glial cells and formation of, 1049
information flows in, 1028
inhibitory, 1026
memory formation and, 1070-1071
mRNA localization in, 459-460, 460f
neuromodulatory inputs, 1026
neurotransmitter transport to, 1052-1054
Synapsins, 1054, 1072
Synapsis, 226f
Synaptic plasticity, 1070, 1072-1074, 1072f, 1073 f
Synaptic vesicles, 1028, 1049f
cycling of, 1053 f
neurotransmitter transport into, 1052-1054
in presynaptic terminal, 1054
recycling, 1056-1057
regulation of fusion on, 1055-1056
Synaptonemal complex (SC), 912
Synaptotagmin, 1056, 1056f
Syncytial divisions, 880
Syntaxin, 643, 644
Syntelic attachment, 898, 899f
Synteny, 25, 26f, 342
Synthetic lethal mutations, 231-232, 232f, 1158
Synthetic lethality, 231-232
Systemic lupus erythematosus (SLE), 425f

## T

T cells, 1081, 1082, 1083f, 1095, 1104. See also Cytotoxic T cells
activation of, 1122, 1122f
antibody production and, 1128, 1129f
antigen-specific receptors triggering, 1118,1120
cancer defenses and, 1131

CD4 or CD8 lineage commitment, $1121-1122$
cytokine production by, 1123-1124
development of, 1120, 1121f
helper, 1124
MHC molecules interacting with, $1107-1109$
T loop, 883
TAFs. See TBP-associated factors
Tail-anchored proteins, 397
Talin, 930f, 963
Tamoxifen, 993f, 1157, 1158
Tandemly repeated arrays, 307-308
Target of rapamycin (TOR), 452
Targeting sequences, 585, 609-610, 609t
on cargo proteins, 641
coat proteins and, 641
TARPs. See Transmembrane AMPA receptor regulatory proteins
Tastes, 1064-1066, 1065f
Tat protein, 377, 377f, 617
TATA box, 371-372, 371f, 413f
TATA box-binding protein (TBP), 374
Tau proteins, 830-831
Taxol, 830
TAZ, 911
TBP. See TATA box-binding protein
TBP-associated factors (TAFs), 374
TCA cycle. See Tricarboxylic acid cycle
T-cell malignancies, 725
T-cell receptors (TCRs)
genes, 1116-1118, 1117f
signal transduction from, 1119 f
structure of, 1115-1116, 1116f
variable residues of, 1118
T-cells, formation of, 728
TCF. See Ternary complex factor
TCRs. See T-cell receptors
TdT. See Terminal deoxynucleotidyltransferase
TE. See Trophectoderm
Telomerase, 347-349, 348f, 1140
Telomere terminal transferase. See Telomerase
Telomeres, 345, 347-349, 391, 392f
Telophase, 851, 859, 876
nuclear envelope reassembly in, 903, 903f
TEM. See Transmission electron microscope
Tem1, 910
Temperate phages, 216
Temperature-sensitive mutation, 21, 228, 229f, 878, 881, 905
Template DNA strands
duplex DNA unwound for, 199
transcription of, 176-179
Terminal deoxynucleotidyltransferase (TdT), $1116-1117$
Termination, 353
Ternary complex, 646 f
Ternary complex factor (TCF), 745
Tertiary structure, 72
RNA, 175, 175 f
Tetrahymena, 345, 347, 348f, 394, 526
Tetrahymena thermophila, 466
Tetramers, 861
TFIIA, 374

TFIIB, 374, 376
TFIIB recognition element (BRE), 376
TFIID, 333, 373-376, 394
TFIIE, 374
TFIIF, 374
TFIIH, 374, 375
TFIIIB, 413
TFIIIC, 413
TfR. See Transferrin receptor
T-G mismatches, 205, 205f
TGF- $\alpha$. See Transforming growth factor $\alpha$
TGF- $\beta$. See Transforming growth factor $\beta$
TGF- $\beta$ receptor proteins, 722-724
Smad transcription factors and, 724
TGF- $\beta$ binding and signal transduction, $722-724,723 f$
TGF- $\beta /$ Smad signaling pathway, 723, 723f
Thalassemia, 444
Thermal energy, 57
Thermoacidophiles, 10
Thermogenesis, 559
Thermogenin, 559
Thermosynechococcus elongatus, 570
Thermus aquaticus, 241, 589
Thermus thermophilus, ribosome of, 189f, 190 f
Thick filaments, 804
Thick-filament regulation, 808
Thin filaments, 804
Thin-filament regulation, 806, 806f
Thiol groups, 603
Thioredoxin, 574
3 T 3 cells, $1140,1140 \mathrm{f}$
$3^{\prime}$ cleavage and polyadenylation, 419, $430-432,431 \mathrm{f}$
$3^{\prime}$ poly(A) tail, 430
3-D adhesions, 961, 962f
Three- $\mathrm{Na}^{+} /$one- $\mathrm{Ca}^{2+}$ antiporter, 504-505
Threonine, 44
Threshold potential, 1059, 1060f
Thrombopoietin, 727
Thrombospondin (TSP), 1049
Thylakoids, 18, 560, 617, 618f
Thymine, 45, 46f
Thymine-thymine dimers, 206, 206f, 375-376
Thymocytes, 1111
Thymosin- $\beta_{4}, 785$
Thymus, 1111, 1121-1122
Tight junctions, 508, 932, 932f, 939-942, 939f, 940f, 941f
Tim proteins, 611
Time-lapse microscopy, 142
+TIPs, 831-832, 832f
TIRF microscopy. See Total internal reflection fluorescence microscopy
Tissue, classical cadherins and differentiation of, 936
Tissue culture cells, 880
Tissue grafts, 1104-1105
Tissue plasminogen activator (TPA), 77-78
Tissues, 921
allogenic transplantation of, 1120
basal lamina and, 945-947, 946f
cancer cells escaping, 1140

evolution of, 928-929
organization into organs, $24,25 \mathrm{f}$
plant, 968-972
Titration curves, 56 f
TLRs. See Toll-like receptors
TM. See Tropomyosin
TN. See Troponin
TNF $\alpha$. See Tumor necrosis factor alpha
Tobacco smoke, 1144, 1144f, 1157
Tolerance, 1080
Toll-like receptors (TLRs), 758, 760, 1084, $1122,1125-1127$
activation of, 1126 f
diversity of, 1126
engagement of, 1127-1128
signaling cascade, 1127
structure of, 1125-1126
Tom proteins, 610
TOP mRNAs, 453
Topogenic sequences, 593, 599
Topoisomerase I, 174, 174f
Topoisomerase II, 174
Topological domains, 78
within chromosome territories, 336-339
Topology
deduction of, 599-600
of membrane proteins, 593-594, 594f
TOR. See Target of rapamycin
TOR pathway, 452-454
Torsional stress, 174, 174f
Total internal reflection fluorescence microscopy (TIRF microscopy), $150-151,151 f$
Totipotent cells, 976
Touch-receptor complex, 1062, 1062f
TPA. See Tissue plasminogen activator
Tra protein, 436, 436f
TRADD, 1021-1022
Trans fats, 50
Transcellular pathway, 941, 941f
Transcellular transport, 508-510
Transcription, 8, 81, 169f
activated, 356, 357-358
alternative splicing and, 435
chromatin-mediated repression of, 390
chromatin-remodeling complexes and, 395
divergent, 372-373
DNA methylation repressing, 404-405
epigenetic regulation of, 404-411
eukaryote regulatory elements and, 364, $365 f, 366-367$
MAP kinase regulating, 745-746, 745f
molecular mechanisms of repression and activation, 390-397
pervasive, 432-433
of protein-coding genes, 176-182
regulation of, 339,354
repressed, 356, 357-358
stages of, 176-178, 178f
of template DNA strands, 176-179
termination of, 402
Transcription bubble, 177
Transcription elongation factors, 369
Transcription factors, 9, 9f, 168, 333, 339.

See also specific transcription factors
cancer and, 1160-1161
ES cell pluripotency and, 982, 982f
in eukaryotes, 364
gene induction by, 721 f
general, 373-376
HOXD13 encoding, 354
interactions, 387-388, 388f, 389f
iPC cells and, 983
master, 25-27, 27f, 721, 721f, 725f
master transcriptional regulators, 770
mutations in genes encoding, 355 f
nuclear-receptor superfamily, 400, 401f
oncogenes and, 1160-1161
Pol II, 373-376
regulation of activity of, 398-403
Smad, 724
transcription-control elements as binding sites for, 380
Wnt signaling releasing, 752-753, 753f
Transcription initiation complex, 78-79, 78f
Transcription unit, 303-305, 305f
microsatellites in, 310
pre-rRNA, 462, 462f
Transcriptional control, eukaryotic, 356 f
Transcriptional elongation, regulation of, $361-362,362 \mathrm{f}$
Transcriptional machine, 78-79, 78f
Transcriptional profiling, 122
Transcription-control elements, 379-380, 379f
Transcription-control regions, 354, 378, 378f
Transcription-coupled repair, 207
Transcytosis, 658, 1008, 1090, 1091, 1092f
Transducin, 694, 698, 698f
Transesterification reactions, 423, 425f, 466
Transfection, 251, 382f
stable, 252
transient, 251-252
Transfer RNA (tRNA), 168, 169f
amino acid activation, 188
folded structure and decoding functions of, 185-186, 186f
mRNA decoding, 183-188
nonsense mutations and, 196-197
processing of, 461-469
protein synthesis role, 183, 183f
structure of, 187 f
tandemly repeated genes for, 308
wobble position, 186, 187f
Transferrin receptor (TfR), 456, 456f
Transferrin receptor complex, 663-664, 664f
Transformation, 236
oncogenic, 1140
Transformed cells, 132
Transformer gene, 436
Transforming growth factor $\alpha$ (TGF- $\alpha$ ), 1141
Transforming growth factor $\beta$ (TGF- $\beta$ ), 722, 990
binding and signal transduction, $722-724,723 \mathrm{f}$
cancer and, 722, 725
gene expression and, 1162, 1162 f
loss of signaling, 1162
negative feedback loops regulating, $725-726,726 f$
synthesis and protein storage, 722
Transgenes, 263
Transgenic mice, 263, 264f
trans-Golgi, 632, 649
M6P receptors and, 653-654
protein aggregation in, 655
protein sorting and, 657-658, 658f
proteolytic processing and, 656-657
vesicle-mediated protein transport from, $651-652,651 f$
Transient amplifying cells, 988
Transient transfection, 251-252
Transition state, 60
Transition-state intermediate, 60
Translation, 8, 81, 168, 169f, 176
error rates, 188
GTPase superfamily role in, 195-196
initiation in eukaryotes, 191, 192f, 193
miRNA repression of, 447-450
mRNA structure and efficiency of, 195, 196 f
polysomes and, 195
ribosome recycling and, 195
termination of, 194f, 195
Translocation, 193, 194, 343f
cell-body, 812
cell-free, 612
chromosomal, 1151
cotranslational, 588-589, 589f
post-translational, 591-592, 592f
proton, 556
termination of, 194f
Translocation channel, 585
Translocon, 589-591, 590f
mitochondrial protein import and, $610-612,611 f$
Transmembrane ADAMs, 761
Transmembrane AMPA receptor regulatory proteins (TARPs), 1074
Transmembrane channels, 556
Transmembrane collagens, 951
Transmembrane electric gradient, 495-497, 496 f
Transmembrane proteins, 285-287, 285f, 286f, 289-290
Transmembrane transport, 474-477
Transmissible spongiform encephalopathy, 87
Transmission electron microscope (TEM), $157,157 \mathrm{f}, 167 \mathrm{f}$
Transport proteins
in epithelia, 508-509, 508f
MDR1/ABCB1, 491, 492f
membrane, 68, 271, 474-477, 475f, 476f, 487
in plant vacuoles, 507
study approaches, 480
Transport vesicles, 632
budding, 665
in Golgi complex, 649 f
lysosomes and, 665
retrograde, 632
sorting signals and, 642 t

Transporters, 475f, 476
Transposable (mobile) DNA elements, $312-323$
Transposase, 314, 315f, 322f
Transposition, 312-313, 318f
exon shuffling by, 322 f
Trans-splicing, 428
Transverse tubules, 805
TRE, 401f, 402
Treadmilling, actin, 783f, 784-785
Treg cells, 1131
Triacylglycerols, 49, 536-537
Tricarboxylic acid cycle (TCA cycle), 533, 1137-1138. See also Citric acid cycle
Tricellulin, 940
Trichomonas vaginalis, 22
Tricyclic antidepressants, 1057
Triglycerides, 49, 50, 536-537
Trimeric G protein, 454
Triplet codes, 183
Triskelions, 651
tri-SNRP complex, 426, 426f
Trisomies, 916
Trisomy 21, 342, 916
Trithorax proteins, 406-407, 408f, 409
Triton X-100, 291, 291f, 292
TrkA, 1016-1017, 1017f, 1019f
TrkC, 1016-1017, 1017f
Trks, 1016
tRNA. See Transfer RNA
tRNA ${ }^{\text {Met }}, 190-191,192 \mathrm{f}$
Trophectoderm (TE), 979, 980f
Trophic factors, 1011, 1015, 1016, 1018-1019, 1019f
Tropoelastin, 960
Tropomodulin, 785, 785f
Tropomyosin (TM), 806, 806f, 807
Troponin (TN), 806-807
TRPV1, 1062, 1063f
True-breeding strains, 227
Trypanosoma brucei, 22, 526
Trypsin, 607
Tryptophan, 43, 44, 179, 180f, 184, 1052f
t-SNAREs, 638, 639f, 642-644, 643f, 1055
TSP. See Thrombospondin
tTA, 1148
Tuberculosis, 1130
Tubulin proteins, 325, 325f, 822
dimers of, in microtubule walls, 822-824, 824 f
drugs affecting, 829-830
microtubule modifications and classes of, 842-844, 843f
polymerization of, 829-830
Tumor microenvironment, 1141
Tumor necrosis factor alpha (TNF $\alpha$ ), 681, $758,763,770,1011,1021-1022$
Tumorigenesis, 1136
Tumors. See also specific cancers and tumors
angiogenesis in, 1141
blood supply, 1137
cell differences, 1136-1143, 1136f
epigenetic changes and, 1155
as heterogeneous organs, 1140-1141
invasive, 1141-1143, 1142f
malignant, 1141
miRNA and, 1155-1156
protein composition, 1137-1139
Tumor-suppressor genes, 1135, 1150t, $1164-1165$
apoptosis regulation genes as, 1163
inherited mutations in, 1153-1155, 1154f
loss-of-function mutations, 1152-1153
mutations in, 1162
Tunicamycin, 602
Tunneling nanotubes, 970-971, 971f
Turgor pressure, 481, 956
Turnover number, 92
Two-component regulatory systems, $360-361,361 f$
Two-dimensional gel electrophoresis, 108-109, 109f
2,4-dinitrophenol (DNP), 559
2-hydroxyglutarate, 1139, 1139f
Two-Na ${ }^{+}$/one-glucose symporter, 503-504, 504 f
Two-photon excitation microscopy, $149-150,150 f$
Ty transposition, 318 f
Type 1 diabetes mellitus, 769, 986
Type 2 diabetes mellitus, 504, 769, 770, 986
Type I collagen, 952-953
Type I transmembrane proteins, 594-595, 595f
Type II collagen, 952-953
Type II transmembrane proteins, 595-597, 596 f
Type III transmembrane proteins, 595-597, 596 f
Type IV collagen, 945, 948-950, 949f
Type IV transmembrane proteins, 597-598
Tyrosine, 43, 467

## U

UASs. See Upstream activating sequences
Ubiquinone, 541. See also Coenzyme Q
Ubiquitin, 98f, 99-100
Ubiquitinylation
protein regulation and, 103-104
securin, 901-902
signaling pathways controlled by, 751-760
UCPs. See Uncoupling proteins
Ultraviolet radiation (UV radiation), 206, 206f, 207, 375, 906
Umami taste, 1066
Uncoupling proteins (UCPs), 559
Unfolded-protein response, 606-607, 606f
Unicellular eukaryotic model organisms, $19-24$
Uniport transport, 477-478
Uniporters, 476
Universal code, 184f, 185
deviations from, 185 t
Unsaturated fatty acids, 48-49, 50, 293
Untranslated regions (UTRs), 181
Upstream, 364
Upstream activating sequences (UASs), 380

Upstream control elements, 412-413
Uracil, 45, 46f
UTRs. See Untranslated regions
UV radiation. See Ultraviolet radiation

## V

Vaccines, 1079, 1089, 1130-1131
Vacuoles, 13 f
acidity regulation in, 489-491
plant, 17-18, 17f
transport proteins, 507
Valine, 43
Valinomycin, 550
VAMP, 643, 643f, 644, 1053f
van der Waals interactions, 38-39, 39f, 171
van Leeuwenhoek, Antoni, 129
Variable regions, 1092
Vascular CAM-1, 964
Vascular endothelial growth factor (VEGF), 1141
Vascular plants, 481
Vascular smooth muscle, 714-716, 715f
Vascular tissue, 968
Vasopressin, 483
VASP, 789-790, 790f
V-class pumps, 484f, 485, 489-491, 491f, $510,510 f$
VDACs. See Voltage-dependent anion channels
VDRE. See Vitamin D3 response element
Vector plasmid, 252, 252f
VEGF. See Vascular endothelial growth factor
Ventricular zone (VZ), 1031-1032, 1032f
Vertebrate hair cells, 437-438
Vesicle buds, 639, 640f
Vesicle-mediated protein transport
COPI vesicles, 647-648
COPII vesicles, 645-647
from trans-Golgi network, 651-652, 651f
Vesicles, 296, 296f. See also Synaptic vesicles; Transport vesicles; specific vesicles
AP coats, 651-652
autophagic, 667
axonal transport of, 835
budding and fusion mechanisms, $638-644,639 \mathrm{f}$
clathrin coats, 651-652, 652f
coated types, 638, 639t
GTPase switch proteins and coats of, $639-641$
myosin V-bound, 808-810, 809f
protein coats, 638-639
secretory, 13 f
SNARE proteins and fusion of, 642-644, 643 f
Vesicular GABA transporters (VGATs), 1054
Vesicular glutamate transporters (VGLUTs), 1054
Vesicular stomatitis virus (VSV), 634, 637
Vesicular transport
cell-free assays for studying, 637-638, 637f
yeast mutants defining, 635-636, 637f

Vestibule, 498
VGATs. See Vesicular GABA transporters
VGLUTs. See Vesicular glutamate transporters
Vibrio cholerae, 692, 942
Video microscopy, 139
Villi, 991f, 992, 994f
Vimentin, 864, 864t
Virions, 212
progeny, 217 f
structures, 214 f
Viruses, 212-218
cancer-causing, 1152
capsids, 213
cloning and counting, 213, 215 f
host ranges, 212-213
lytic growth cycles, 213-216, 215f
replication of, 1081
Visual adaptation, 698
Visual cortex, 694, 1061
Vitamin C, 951-952
Vitamin D, 278
Vitamin $\mathrm{D}_{3}$ response element (VDRE), 401f, 402
Vitamins, coenzymes and, 95
$\mathrm{V}_{\text {max }}$. See Maximal velocity
Voltage gradients, 495
Voltage-dependent anion channels (VDACs), 523, 712
Voltage-dependent $\mathrm{Na}^{+}$channels, 1054
Voltage-gated $\mathrm{Ca}^{2+}$ channels, 1054, 1058 f
Voltage-gated channels, 1034-1047
common structures, 1039, 1040f
Voltage-gated $\mathrm{K}^{+}$channels, 1035-1037, 1037f, 1042, 1042f
Voltage-gated $\mathrm{Na}^{+}$channels, 1035-1037, 1036f, 1038f, 1042
Voltage-sensing $\alpha$ helices, 1035, 1039-1040, 1041f, 1042
von Behring, Emil, 1088, 1088f
von Hansemann, David, 1137
v-SNAREs, 638, 639f, 642-644, 643f, 645, 647, 1053 f
VSV. See Vesicular stomatitis virus
VSV G protein, 634, 635, 636f
VSVG-GFP, 634, 635
VZ. See Ventricular zone

## W

WAKs. See Wall-associated kinases
Wall-associated kinases (WAKs), 971
Warburg, Otto, 1138
Warburg effect, 1138
WASp, 788
Water, 5, 31-32
dipole nature of, 35 f
facilitated transport of, 477-483
hydrogen bonding, 37, 38f
osmotic pressure and, 481
Watson, James, 7, 170
WAVE complex, 788-789, 815
Weak promoters, 358
Wee1, 896, 905
Western blotting, 112, 113f, 1081
with monoclonal antibodies for phosphorylated peptides, 684, 685f
White blood cells, 1082
White-fat tissue, 558
Whole genome shotgun sequencing, 244
Whole-cell patch clamping, 500 f
Whooping cough, 692
Wieschaus, Eric (Francis), 228
Wild type, 224
Wild-type autosomes, 255 f
Wilkins, Maurice, 170
Wilson, Henry Van Peters, 926
Wiskott-Aldrich syndrome, 788
Wnt, 751, 752-754, 753f, 754f, 990, 992
Wobble position, 186, 187f
Wounded-cell monolayer assay, 815, 816f

## $X$

X chromosome
Duchenne muscular dystrophy inheritance and, 796
inactivation in mammals, 334-335, 409-410, 410f
XACT, 411
Xenopus laevis, 877, 878-879
Xenopus oocytes, 451-452, 878-879, 879f nuclear bodies in, 468 f nuclear envelopes of, 623 f
Xeroderma pigmentosum, 207, 207f, 376, 1167
XIAP, 1018
X-inactivation center, 409-410, 410f

Xist gene, 409, 410f
X-linked genetic diseases, 547
X-linked mental retardation, 102
X-linked recessive diseases, 255, 255 f
XMAP215, 831, 853, 853f
X-ray crystallography, 119, 119f, 691
X-rays, 906

## Y

Yamanaka, Shinya, 983
Y-complex, 622, 623f
Yeast, 19, 21. See also Saccharomyces cerevisiae
asymmetric growth in mating, 1003
budding, 877-878, 877f, 890f
fusion, 877-878, 878f
genomic libraries, 237-238, 238f
GPCR-MAP kinase signals in, 746
homologous recombination in, 260, 260f
mating pathways, 746, 747f
mating-type loci, 391, 391f
Mediator complex, 396 f
mutations in, 21-22
polarization, 1003
preinitiation complex, 376 f
pre-rRNA processing in, 462, 463f
shmoos in, 1003, 1004f
tRNA in, 467, 467f
tubulins in, 822-823
vesicular transport in, 635-636, 637f
$\mathrm{YXX} \Phi$ sorting signal, 651

## Z

Z disk, 804, 805
in desmin filaments, 864
Zebrafish, 20f
Zellweger syndrome, 621
Zinc finger, 76
Zinc finger proteins, 385-386
ZO. See Zonula occludens
ZO-1, 944
ZO-2, 944
Zoloft, 1057
Zona pellucida, 977, 978f
Zonula occludens (ZO), 941
Zwitterion, 55
Zygotes, 879
Zymogens, 104

this page left intentionally blank

# ABOUT THE COVER 

Imaging of the intracellular organelles of a live human HeLa cell shows the dramatic morphological changes that accompany the process of cell division. The membrane of the endoplasmic reticulum (ER) is labeled green by a fluorescently tagged component of the translocon (GFP-Sec61 $\beta$ ) and chromatin is labeled red by a fluorescently tagged histone (H2B-mRFP). Front: An interphase cell showing uncondensed chromatin filling the nucleus, with the ER as a reticulum of cisternae surrounding the nucleus and interconnected with lace-like tubules at the cell periphery. Back: Prior to cell division the chromatin condenses to reveal the worm-like structure of individual chromosomes, the nuclear envelope breaks down, and the ER condenses into an array of cisternae surrounding the condensed chromosomes. As cell division proceeds the replicated chromosomes will segregate equally into two daughter cells, nuclear envelopes will form in the daughter cells, and the ER will return to its characteristic reticular organization.

This book is available in hardcover (1-4641-8339-2) and loose-leaf (1-4641-8743-6) versions. This book is also available in digital format through LaunchPad (1-4641-8745-2), featuring a full e-Book and all instructor and student resources.
(Cover Image: Dr. Tomas Kirchhausen \& Dr. Lei Lu)

## w.h.freeman

Macmillan Learning
macmillanlearning.com
![li0120777555](li0120777555.jpg)